
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>adenovirus</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA327937</h3>Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a.
Abstract
Attempts to prevent the urea conversion of a 200-230,000 molecular weight DNA polymerase alpha to a 150-170,000 molecular weight form by the inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70,000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70,000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit.<br><br><br><br>
 Volume 6 Number 10 1979 <br><br> Nucleic Acids Research <br><br> Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a Keith McKune and Andrew M.Holmes* <br><br> Biochemistry Department, Strathclyde University, The Todd Centre, 31 Taylor Street, Glasgow, G4 ONR, UK <br><br> Received I June 1979 ABSTRACT <br><br> Attempts to prevent the urea conversion of a 200-230, 000 molecular weight DNA polymerase a to a 150-170, 000 molecular weight form by the <br><br> inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70, 000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70, 000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit. <br><br> INTRODUCTION <br><br> In mammals DNA polymerase a is thought to be the replicative <br><br> enzyme, but due to low levels of activity, even in tissues actively making DNA, and to enzyme heterogeneity it has been found difficult to purify <br><br> However, small samples of DNA polymerase a have been highly purified <br><br> 2, 3, 4, 5 from several sources and partially characterised <br><br> Heterogeneity has been observed in DNA polymerase a from a var<br><br> 6           7 <br><br> iety of sources, including <span class="yellow">rat</span> liver and spleen , Hela cells, baby <span class="yellow">hamster</span> <br><br> 8                9, 10                   11 <br><br> kidney cells , <span class="yellow">mouse</span> myeloma    , <span class="yellow">Drosophila</span> embryos  and <span class="yellow">calf</span> thymus <br><br> 1,6, 12, 13 . We have previously observed several species of <span class="yellow">calf</span> thymus <br><br> DNA polymerase a differing in size and charge  . In order of elution from DEAE cellulose they are enzyme A1 (200-230, 000 molecular weight), A2 (200-230, 000), B (100-110, 000) and C (150-170, 000). A poly (dA). oligo <br><br> (dT)10 preferring enzyme, enzyrrme D (140-150, 000 molecular weight) elutes just after enzyme B. The A enzymes seem identical in all properties except their charge. 5.0-5. 5S enzymes, analagous to B, have been observed to <br><br> C Information Retrieval Limited 1 Falconberg Court London Wl V 5FG England <br><br> 3341 <br><br> Nucleic Acids Research <br><br> arise as the result of proteolytic action  ' , similarly the B enzyme above 14 <br><br> ,but the relationship of the other observed species is not clear, nor is <br><br> the problem of which, if any, of these species is the replicative enzyme, <br><br> although circumstantial evidence has implicated a DNA polymerase a to be <br><br> 15 the replicative enzyme in <span class="blue">adenovirus</span> infected KB cells <br><br> We have previously shown that several of these enzymes are inter<br><br> 16 <br><br> convertible . In particular, mild urea treatment can convert both A <br><br> enzymes to C enzyme, with the loss of a subunit or fragment of 50-70, 000 molecular weight. The C enzyme retains DNA polymerase activity, but <br><br> does show differences compared to A enzyme in heat sensitivity and sensitivity to N-ethylmaleimide. <br><br> Highly purified samples of A and C enzymes when subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis have shown a corre<br><br> lation of DNA polymerase activity with a polypeptide band 155, 000 molecular weight . Contaminating material of 50-70, 000 molecular weight appeared <br><br> to obscure the putative 50-70, 000 subunit in gels of A enzyme. The conclusions drawn were that DNA polymerase a is a 155, 000 molecular weight <br><br> polypeptide (C enzyrrme) which can and does associated with material of 50<br><br> 70, 000 molecular weight to give A enzyme, and that enzyme B is a proteol<br><br> 14 <br><br> ytic degradation product . The relationship of enzyme D to the others is <br><br> not clear. <br><br> It is possible that during the enzyme isolation procedure proteolytic action may have taken place on a 200-230, 000 molecular weight enzyme releasing 155, 000 and 50-70, 000 fragments which remain in association <br><br> until urea treatment separates them. It is also possible that urea treatment itself renders the enzyme susceptible to contaminating proteases. Further experiments to ascertain whether or not this is the case and, if so, to prevent it, have been carried out. <br><br> A enzyme can be reconstituted from C enzyme formed by the action <br><br> of urea on A enzyme by concentrating it with the 50-70, 000 molecular weight material. The A enzyrme obtained in this manner is highly purified, as is <br><br> the C enzyme formed by the urea treatment, and has been used in attempts to show differences in synthetic template-initiator complex utilisation <br><br> 3342 <br><br> Nucleic Acids Research <br><br> b y the se two enzyme s . <br><br> MATERIALS AND METHODS <br><br> <span class="yellow">Calf</span> thymus was obtained from 10-16 week old <span class="yellow">calves</span> and frozen at <br><br> -20?C until required. Chromatographic media and chemicals were obtained <br><br> 6 <br><br> from sources previously referred to . Radioactive deoxynucleoside tri<br><br> phosphates were obtained from the Radiochemical Centre, Amersham, Bucks. Synthetic oligo and polynucleotides were obtained from P. L. Biochemicals <br><br> Inc., except for poly (dC) which was a gift from Dr. I.R. Johnston and pre<br><br> 17 <br><br> pared from oligo d(C)5 as described  . N-a-p Tosyl-L-lysine chloromethy<br><br> lketone HCl and phenylmethylsulphonylfluoride were obtained from Sigma and Trasylol from Bayer. <br><br> Except where indicated all buffers contained 20% w/v glycerol and <br><br> 1 mM dithiothreitol. Standard linear phosphate gradients were run between 0.03 M and 0.25 M potassium phosphate, pH 7.8. Gradient salt concentr<br><br> 2 <br><br> ations were measured using a conductivity meter as described . Urea was <br><br> prepared as a 4. 8 M solution in 20% w/v glycerol, stirred with Amberlite <br><br> MB3 and filtered. Dithiothreitol was added to a final concentration of 1 mM before use. <br><br> 6 DNA polymerase was assayed using activated DNA as described <br><br> except that the buffer was 50 mM tris HC1, pH 7.8. One unit of DNA polymerase activity incorporates ln mol [ H] dTMP into an acid insoluble form in one hour at 370C. Assays using synthetic template-initiator complexes were carried out at 30?C in 0. 12 ml. The template-initiator complexes <br><br> 6 <br><br> were prepared and assays processed as described . All assays contained <br><br> 1 mM dithiothreitol, 62. 5 p.g <span class="yellow">bovine</span> serum albumin, 1 ,ug template-initiator complex, enzyme protein and the relevant [ H] deoxynucleoside triphosphate at 0.1 mM and 12-15 cpm/pmol. These assays were carried out at either pH 6.4 in 20 mM sodium-potassium phosphate, or at pH 7.8 in 50 mM tris HCl and contained either 10 mM MgCl2 or 1 mM MnCl2 as indicated. <br><br> Preliminary purification of DNA polymerase a to Fraction IV was as <br><br> 6 <br><br> described , the purification steps being phosphocellulose chromatography, <br><br> ammonium sulphate precipitation and gel filtration on Sepharose 6B. <br><br> 3343 <br><br> Nucleic Acids Research <br><br> Enzyme obtained from the DEAE cellulose step is referred to as Fraction <br><br> V enzyme. Samples were prepared for sodium dodecyl sulphate polyacrylamide gel electrophoresis and scanned as previously described2 <br><br> RESULTS AND DISCUSSION <br><br> (a)   Interconversion Studies <br><br> A enzyme was routinely converted to C enzyme by incubating 200500 units/ml of Fraction V A enzyme in 2.4 M urea in 0. 02 M potassium <br><br> phosphate, pH 7.8, for 60 minutes at 0?C. The mixture was then loaded on to a DEAE cellulose, washed with 0. 03 M potassium phosphate pH 7. 8, and enzyrme eluted either with the standard phosphate gradient or batchwise. Under these conditions usually about 50-60% of recovered activity was <br><br> enzyme C (Fig 1 a). Overall recovery was 70-80%. A enzyme was recon<br><br> 14 <br><br> stituted essentially as described . The flow through material from the <br><br> DEAE cellulose column after the urea treatment was loaded on to a 1 x 0. 8 cm phosphocellulose column, washed with 0.03 M potassium phosphate, pH 7.8, and the protein eluted with 0.25 M potassium phosphate, pH 7. 8. This material, the putative subunit, was vacuum dialysed with the C enzyme produced by the urea treatment, rechromatographed on DEAE cellulose and enzyme eluted batchwise (Fig 1 b). Recovery from this procedure was <br><br> usually 50-70% A enzyme. Overall recovery was 30-40% of the C enzyme dialysed. If the material eluted from the phosphocellulose by the 0. 25 M potassium phosphate was heated to 90?C for 5 minutes prior to vacuum <br><br> dialysis with the C enzyme the recovery from the DEAE cellulose column <br><br> was significantly higher (60-70% of the original C enzyme activity), but all recoverable DNA polymerase activity was C enzyme. This, together with the fact that the 60 minute treatment with urea has, on occasions, given <br><br> rise to a 50% increase in DNA polymerase activity prior to loading on to the DEAE cellulose column, would indicate that A enzyme is less active on <br><br> activated DNA than is C enzyme. If the DEAE cellulose flow through material came from urea treatment of A2 enzyme then A2 was produced on reconstitution; if from A1 then A1 was produced. <br><br> Although the mild urea treatment of A has been used to prepare C of <br><br> 3344 <br><br> Nucleic Acids Research <br><br> b <br><br> 120       180 C <br><br> I         I    ^ <br><br> Figure la  DEAE cellulose chromatography after 2.4 M urea treatment. <br><br> 4, 5000 units, 4.4 mg of Fraction II A2 enzyme were incubated with urea at a final concentration of 2.4 M for 60 minutes at 0?C, loaded on to a 5 x 1.4 cm DEAE cellulose column, the column was washed with 0. 03 M potassium phosphate, pH 7.8 and a 200 ml standard phosphate gradient applied. 2.3 ml fractions were collected and 10 Rl assayed for 5 minutes. (o-o) no <br><br> phenylmethylsulphonylfluoride (*-*) 3 mM phenylmethylsulphonylfluoride ( - ) phosphate gradient. <br><br> Figure lb  The reconstitution of A2 enzyme. 3000 units for C enzyme <br><br> derived by urea treatment of A2 were vacuum dialysed with the DEAE cellulose flow through material after phosphocellulose chromatography and <br><br> chromatographed on a 2 x 1. 2 cm DEAE cellulose column. After washing with 0.03 M potassium phosphate, pH 7.7, the enzymes were eluted batchwise with the above concentrations of potassium phosphate, pH 7.8. 1 ml fractions were collected and 10 .l assayed for 10 minutes. <br><br> 3345 <br><br> Nucleic Acids Research <br><br> high specific activity it is possible that conversion of the 200-230, 000 molecular weight enzymes to a 155- 170, 000 species maybe due to unfolding of the molecule to allow limited attack by contaminating proteases. Accord<br><br> ingly the urea conversion of A to C was investigated in the presence of certain protease inhibitors. The presence of the serine protease inhibitor <br><br> phenylmethylsulphonyl-fluoride in the incubation and chromatography buffers did not affect the conversion of A2to C (Fig 1 a). Likewise trasylol and Na-p Tosyl-L-lysine chloromethylketone HC1 had no effect. However, protease action could have occurred earlier in the purification procedure and the urea could be separating two fragments. Usually the DEAE cellulose profile shows that the majority of the enzyme activity is present as A <br><br> enzyme (Fig 2). In this instance A1 and A2 have not been separated. When the temperature of the material in the original blending procedure was kept below 0?C or phenylmethylsulphonylfluoride, N-a-p Tosyl-L-lysine chloromethylketone HCl or trasylol was included in the isolation buffers the DEAE cellulose profile was similar. The A enzymes were still capable of conversion by mild urea treatment to C enzyme, indicating that if protease activity is involved then it is not susceptible to these inhibitors. When the <span class="yellow">calf</span> <br><br> 20FI: Now ~ ~      ~      20 20Fr Na      06 <br><br> Figure 2  DEAE cellulose elution profile of <span class="yellow">calf</span> thymus DNA polymerase a. 47, 000 units, 97 mg of Fraction IV enzyme prepared from 415 g of <span class="yellow">calf</span> <br><br> thymus were loaded on to a 10 x 1.8 cm DEAE cellulose column, after washing with 0. 03 M potassium phosphate, pH 7. 8 a 400 ml standard phosphate gradient was applied. 5 ml fractions were collected and 10 ,ul assayed for 5 minutes. <br><br> 3346 <br><br> Nucleic Acids Research <br><br> thymus was allowed to warm up during the blending procedure, or the supernatant prior to phosphocellulose chromatography was heated to 37 ?C for 30 minutes, there was a marked decrease in the amount of A enzyme with a concomitant increase in the amount of B and C (unpublished observation). The presence of phenylmethylsulphonylfluoride, N- a-p T osyl- L-lysine <br><br> chloromethylketone HCl or trasylol under these conditions only had a marginal effect on the appearance of C enzyme, but did reduce the amount of B <br><br> enzyme. Heating the enzyme to 37?C after the phosphocellulose step had no effect on the DEAE cellulose elution profile. Attempts to convert A enzyme to C using trypsin have not been successful. A enzyme activity is lost without the appearance of any other species (unpublished observation), although <br><br> 14 the action of trypsin on C enzyme can give rise to small amounts of B <br><br> Although the conversion of A enzyme to C does not appear to be the result of serine protease action, proteases other than serine proteases <br><br> 18 <br><br> could have been responsible  . Also, the fact that A enzyme can be recon<br><br> stituted from C plus the flow through material from the DEAE cellulose after urea treatment does not necessarily mean we are dealing with two subunits as fragments produced by proteases mray be reassembled to give active <br><br> enzyme 9    . However, the fact that a 200-230, 000 molecular weight polypeptide band has never been observed in sodium dodecylsulphate polyacrylamride gels of highly purified A enzyme , or even cruder fractions of A <br><br> enzyrre (unpublished observation), may be significant. One might expect <br><br> some of the enzyme not to have been attacked by whatever is responsible for cleaving the molecule, if, indeed, this does happen. It would appear, therefore, that the A enzyme consists of subunits of 155, 000 and 50-70, 000 molecular weight with the small subunit having a slightly different charge in the <br><br> case of A  and A2. It has also been concluded that the heterogeneity in the <br><br> 1                                     ~~~~~~~~~~~~~~~~~3 <span class="yellow">mouse</span> myeloma DNA polymerase a fraction is not due to proteolysis . However, the 50-70, 000 molecular weight subunit has not yet been identified. <br><br> Sodiurrm dodecylsulphate polyacrylamide gels of reconstituted A enzyme have shown polypeptide bands at 150-160, 000 and 50-70, 000 molecular weight <br><br> (Fig. 3), but the ratio of staining intensity of the bands does not correspond to a 1: 1 relationship. The ratio of the two bands is variable but is usually between 1:2 and 1:3, indicating, perhaps, that more than one subunit of 50<br><br> 3347 <br><br> Nucleic Acids Research <br><br> TOP      123   4 5   6 <br><br> Figure 3  Scan of a 5% sodium dodecyl sulphate polyacrylamide gel of <br><br> reconstituted A2 enzyme. 1500 units of reconstituted A2 enzyme were subjected to polyacrylamide gel electrophoresis under non-denaturing conditions. The gels were sliced, enzyme extracted and assayed and the peak <br><br> fraction of DNA polymerase activity from three gels were pooled, subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis on a single <br><br> gel, stained and scanned at 2 volt sensitivity as described2. The molecular weight standards were: (1) <span class="yellow">bovine</span> serum albumin dimer (134, 000), (2) ,3 galactosidase (130, 000), (3) phosphorylase a (94, 000), (4) <span class="yellow">bovine</span> serum <br><br> albumin (68, 000), (5) pyruvate kinase (57, 000) and (6) lactate dehydrogenase (35, 000). <br><br> 70, 000 molecular weight can associate with the 155, 000 subunit. The proportion of the lower molecular weight polypeptide band is higher in the reconstituted A enzyrrme than in the C enzyme preparation from which it was formed, but even the C enzyme contained some material in the region. <br><br> Breakdown of material from 155, 000 to 50-70, 000 may be partly responsible for the contamination, but the presence of phenylmethylsulphonylfluoride in samples in preparation for sodium dodecylsulphate polyacrylamide gel electrophoresis has not been successful in preventing it. The <span class="yellow">human</span> KB cell <br><br> DNA polymerase a, equivalent to the C enzyme, has been reported to consist <br><br> 4 <br><br> of subunits of 76, 000 and 66, 000 molecular weight , but at no time have we <br><br> observed polypeptide bands at 76, 000 and 66, 000 rising and falling with <br><br> enzyme activity in any gels of <span class="yellow">calf</span> thyrrmus enzymes A1, A  and C (and <br><br> 2 unpublished observations). (b)    Template Studies <br><br> Use has been made of the urea conversion of A to C and of the reconstitution of A enzyme to obtain samples of highly purified DNA polymerase, <br><br> specific activity in excess of 50, 000 units/mg, in order to study the effect of this 50-70, 000 molecular weight subunit on the DNA polymerase subunit. Previous results have indicated that A enzyrrme is stabler to heat and less <br><br> 3348 <br><br> Nucleic Acids Research <br><br> 14, 21 <br><br> susceptible to N-ethylmaleimide than C enzyme  . It has been reported <br><br> that highly purified DNA polymerase a can be associated with a protein capable of binding to DNA containing no 3' OH ends and capable of being released during the DNA polymerase assay  . Attempts to dissociate the 50<br><br> 70, 000 molecular weight subunit from the polymerase subunit by incubating A2 with DNA polymerase reaction mixes containing activated DNA, poly <br><br> (dA-T) and poly (dT). oligo (A)10 followed by ultracentrifugation in high salt have been unsuccessful. Similarly A absorbed and eluted from either native or denatured DNA cellulose remained A enzyme. Both A and C enzymes <br><br> were eluted from the DNA celluloses by less than 0. 1 M NaCl so it does not appear that the subunit enhances the binding of the DNA polymerase subunit to DNA. However, there are differences in the response of A and C <br><br> enzymes to synthetic template-initiator complexes (Table 1). Even if one takes into account the fact that A enzyme is less active on activated DNA <br><br> than C enzyme (the addition of the 50-70, 000 subunit to the polymerase subunit appears to result in a decrease of about 30% of polymerase activity on activated DNA) the A enzyme is still more active on these templates. <br><br> Although there is a variation in activity on these template-initiator comp<br><br> lexes each time assays are carried out on them depending on the method of preparing the complexes and the base ratio of template to initiator, A enzyme always seems to be significantly more active than C. The A2 <br><br> enzyme at pH 7.8, with extra subunit(s) is clearly more effective on the <br><br> oligoribonucleotide initiator, oligo(A)10 than is enzyme C. In view of the proposed RNA initiation of Okazaki pieces this may indicate a role of this subunit in Okazaki piece synthesis, in that it may aid the DNA polymerase to'take over' from the RNA polymerase. Using poly (dA). oligo (dT) 10 <br><br> (A:T=20: 1) and following the incorporation of [ H] dTMP as a function of time a short lag was observed for C enzyme, but not for A2 (Fig. 4). <br><br> Similar re sult s we re obtained when poly (dC). oligo (dG) 1 0 (C: G= 5: 1) wa s <br><br> used as temrrplate-initiator, but not when poly (dT). oligo (A)10 (T:A=1: 1) was used. Neither enzyme showed a lag on this template-initiator complex or on activated DNA. Incorporation versus enzyme concentration also showed this lag for C enzyme on poly (dA). oligo (dT)10. It is not certain what causes <br><br> 3349 <br><br> Nucleic Acids Research <br><br> TABLE 1 Template utilisation by reconstituted A and C derived from A. <br><br> 3H] dNTP Divalent cation  A2 at pH    C at pH <br><br> Template                          _      6.4   7.8    6.4   7.8 Activated DNA         dTTP         Mg++            100          100 Activated DNA         dATP         Mg++             91          85 Activated DNA         dGTP         Mg++             85          82 poly(dA).oligo(dT)10  dTTP         Mg      66.0    4.0    20    91 <br><br> (A:T = 20: 1) <br><br> poly(dA). oligo(dT)   dTTP         Mu      19.0    4. 5  6. 5  2.5 <br><br> poly(dA. oig~10 (A:T = 20:1) <br><br> poly(dT). oligo(dA)10  dATP        Mg       2.5   <1 (1       (1 <br><br> (T:A = 5:1) <br><br> poly(dT) .oligo(dA)1  dATP         Mn ++   15.5   23.0   9. 0  3. 0 <br><br> poly(dT). oligo(A)10  dATP         Mg      15.0  230.0  10.0  77. 5 <br><br> (T:A = 1:1) <br><br> poly(dT) * oligo(A) 10  dATP       Mn      41.5   75. 5 19.5  21.5 <br><br> (T:A = 1:1) <br><br> poly(dC). oligo(dG)   dGTP         Mg++     6. 0   15.5  4. 5  7. 5 <br><br> (C:G = 5:1)   10 <br><br> poly(dC). oligo(dG)10  dGTP        Mn +     6. 0  29.5   5. 0  6. 5 <br><br> (C:G  = 5: 1)                                 .    . <br><br> Values given are relative to incorporation of [ H] dTMP at pH 7. 8 on activated DNA. For A2 this was 176.5 pmol, for C 232 pmol. dATP, dCTP, dGTP and dTTP were included for assays using activated DNA, only the deoxynucleoside triphosphate stated was used in the synthetic template<br><br> initiator complex assays. Assays were for 10 minutes. The buffers and concentrations of the divalent cations were as in Materials and Methods. <br><br> this lag, but the annealing of template to initiator is only transient '  and the DNA polymerase subunit may have difficulty in stabilising the complex and the 50-70, 000 molecular weight subunit may be able to help the poly<br><br> merase subunit to overcome this. Addition of the 50-70, 000 subunit to the polymerase subunit prior to assaying with these templates had no effect on activity and it may be that a preincubation period is required before the two 3350 <br><br> Nucleic Acids Research <br><br> Figure 4 Activity of A2 and C enzymes on poly (dA). oligo (dT)jO (A:T = <br><br> 20:1) as a function of time. 50 ,ul samples were withdrawn at various times from a 0. 6 inl incubation mix at 30?C and added to 0. 5 ml 0. 1 M sodium <br><br> pyrophosphate containing 100 ,ug/ml native <span class="yellow">calf</span> thymus DNA and processed for counting in the usual manner6. The mix contained 1 mM dithiothreitol, 50 mM tris HC1, pH 7.8 10 mM MgC12, 0. 1 mM [3H] dTTP (15 cpm/pmol), 312.5 ,ug <span class="yellow">bovine</span> serum albumin, 5 Fg poly (dA). oligo (dT)  (A:T = 20:1) and enzyme protein (*-*) 3.3 units reconstituted A2, (o-o) 16. 0 units urea derived C enzyme. <br><br> subunits become fully associated. After the lag phase is over the C enzyme is still less active on these template-initiators than A enzyme . That is the A enzyme seems capable of elongating the initiator faster than the C enzyme. Experiments to determine whether the differences in rates of elongation of these template-initiator complexes are differences in processivity of the <br><br> enzymes under the different pH and divalent cation conditions are under way. <br><br> ACKNOWLEDGEMENTS <br><br> We thank the Medical Research Council for a research grant. <br><br> Communications concerning the paper should be sent to:<br><br> Dr. A. M. Holmes, Department of Biochemistry, University of Strathclyde, The Todd Centre, 31 Taylor Street, Glasgow G4 ONR, U.K. <br><br> 3351 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> (1)    Bollum, F.J. (1975). Prog. Nucleic Acid Res. Mol. Biol. 15, <br><br> 109-144. <br><br> (2)    Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1976). Eur. <br><br> J. Biochem. 62, 229-235. <br><br> (3)    Matsukage, A., Sivarajan, M. and Wilson, S.H. (1976). Biochem<br><br> istry 15, 5305-5314. <br><br> (4)    Fisher, P.A. and Korn, D. (1977). J. Biol. Chem. 252, 6528<br><br> 6535. <br><br> (5)    Fichot, O., Pascal, M., Mechali, M. and De Recondo, A, -M. <br><br> (1979). Biochim. Biophys. Acta, 561, 28-41. <br><br> (6)    Holmes, A.M., Hesslewood, I.R. and Johnston, I.R. (1974). Eur. <br><br> J. Biochemn. 43, 487-499. <br><br> (7)    Noy, G.P. and Weissbach, A. (1977). Biochim. Biophys. Acta, <br><br> 447, 70-83. <br><br> (8)    Craig, R.K. and Keir, H. M. (1975). Biochem. J. 145, 225-232. <br><br> (9)    Matsukage, A., Bohn, E.W. and Wilson, S.H. (1974). Proc. Natl. <br><br> Acad. Sci. U.S.A. 71, 578-582. <br><br> 110)   Hachmann, H.J. and Lezius, A.G. (1975). Eur. J. Biochem. 50, <br><br> 357- 366. <br><br> (11)   Brakel, C.L. and Blumenthal, A.B. (1977). Biochemistry, 16, <br><br> 3137-3143. <br><br> (12)   Momparler, R.L., Rossi, M. and Labitan, A. (1973). J. Biol. <br><br> Chem. 248, 285-293. <br><br> (13)   Yoshida, S., Konda, T. and Ando, T. (1974). Biochim. Biophys. <br><br> Acta, 353, 463-474. <br><br> (14)   Holmes, A.M., Hesslewood, I.P., Wickremasinghe, R.G. and <br><br> Johnston, I.R. (1977). Biochem. Soc. Symp. 42, 17-36. <br><br> (15)   De Jong, A., Van der Vliet, P. and Jansz, H.S. (1977). Biochim. <br><br> Biophys. Acta, 476, 156-165. <br><br> (16)   Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1975). <br><br> Nature (London) 255, 420-422. <br><br> (17)   Bollum, F. J. (1966). Procedures in Nucleic Acid Research <br><br> (Cantoni, G.L. and Davis, D.R. eds). pp 577-583, Harper and Row, New York. <br><br> (18)   Barrett, A.J. (1975). Proteases and Biological Control pp 467-482, <br><br> Cold Spring Harbor Laboratory, Cold Spring Harbor. <br><br> (19)   Richards, F. M. and Vithayathil, P.J. (1960). Brookhaven Symp. <br><br> Biol. 13, 115-134. <br><br> (20)   Lowe, P.A., and Malcolm, A.B.D. (1976). Eur. J. Biochem. 64, <br><br> 177- 188. <br><br> (21)   Hesslewood, I.P., Holmes, A.M., Wakeling, W.F. and Johnston, <br><br> I.R. (1978). Eur. J. Biochenr.. 84, 123-131. <br><br> (22)   Mechali, M. and De Recondo, A. -M. (1978). Biochim. Biophys. <br><br> Res. Comm, 82, 255-264. <br><br> (23)   Chang, L.M.S., Cassani, G.R. and Bollum, F.J. (1972). J. Biol. <br><br> Che. 247, 7718-7723. <br><br> (24)   Wickremasinghe, R.G. and Johnston, I.R. (1974). Biochim. <br><br> Biophys. Acta, 361, 37-52. <br><br> 3352 <h3>pmcA411032</h3>Down-regulation of the M6P/IGF-II receptor increases cell proliferation and reduces apoptosis in neonatal <span class="yellow">rat</span> cardiac myocytes
Abstract
Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a multi-functional protein that has been implicated in regulation of cell growth and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R, and cardiomyocyte apoptosis has been identified in a variety of cardiovascular disorders, such as myocardial infarction and heart failure. However, involvement of M6P/IGF2R in the pathogenesis of these conditions has not been determined. Thus, the objective of this study was to determine the role of M6P/IGF2R in regulation of cardiac myocyte growth and apoptosis.<br><br>Results
We down-regulated the expression of M6P/IGF2R in neonatal <span class="yellow">rat</span> cardiac myocytes and examined the effect on cell proliferation and apoptosis. Infection of neonatal cardiomyocytes with an <span class="red">adenovirus</span> expressing a ribozyme targeted against the M6P/IGF2R significantly reduced the level of M6P/IGF2R mRNA, as determined by RT-PCR and Ribonuclease Protection Assay (RPA). M6P-containing protein binding and endocytosis as well as the M6P/IGF2R-mediated internalization of 125I-IGF-II were lower in the ribozyme-treated cells than the control myocytes, indicating that the number of functional M6P/IGF2R in the ribozyme treated cells was reduced. Accordingly, a marked increase in cell proliferation and a reduced cell susceptibility to hypoxia- and TNF-induced apoptosis were observed in the ribozyme-treated cells.<br><br>Conclusions
These findings suggest that M6P/IGF2R may play a role in regulation of cardiac myocyte growth and apoptosis. Down regulation of this gene in cardiac tissues might be a new approach to prevention of cell death or promotion of mitogenesis for certain heart diseases.<br><br><br><br>Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a unique protein that interacts with multiple ligands, some of which are important growth regulatory factors [1]. The M6P/IGF2R participates in internalization and lysosomal degradation of IGF-II, a mitogen normally acting through the IGF-I receptor to stimulate cell proliferation [2]. The M6P/IGF2 receptor is required for the activation of TGF-β [3], a potent growth inhibitor for many cell types. This receptor is also involved in the binding, transport and activation of newly-synthesized lysosomal enzymes, such as cathepsins [4,5], which have been recently implicated in the induction of apoptosis [6]. On the basis of these functions, the M6P/IGF2R has been proposed to play a significant role in regulation of cell growth and apoptosis [7].
Apoptosis, or programmed cell death, is a tightly regulated process used to remove excess, hazardous or damaged somatic cells, and is crucial for the development, maintenance and survival of an organism. However, alterations in the control of apoptosis have also been shown to contribute to <span class="yellow">human</span> diseases. In fact, morphological and biochemical markers of apoptosis have been identified in a wide variety of cardiovascular disorders, including myocardial infarction and heart failure. This suggests that activation of apoptotic pathways contributes to cardiomyocyte loss and subsequent cardiac dysfunction in these conditions. A number of factors involved in cardiomyocyte apoptosis are currently known and include insulin-like growth factor-I (IGF-I), stress-activated protein kinases (SAPKs) and the anti-apoptotic Bcl-2 family [8]. There are indications that other factors may be involved in induction and regulation of cardiac apoptosis. However, these potential factors and their corresponding mechanisms have not been identified.
Several lines of evidence point to the potential involvement of M6P/IGF2R in cardiac myocyte proliferation and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R and transgenic <span class="yellow">mice</span> containing a homologous deletion of the M6P/IGF2R gene manifest ventricular hyperplasia due to an increase in cell number [9,10], suggesting that the M6P/IGF2R normally acts to suppress cardiac myocyte cell growth. It has also been shown that TGF-β, a potent growth suppressor whose activation requires the binding of latent TGF-β to M6P/IGF2R [3], is commonly upregulated in chronic heart failure [11]. Additional evidence for the involvement of M6P/IGF2R in regulation of apoptosis comes from studies of tumorigenesis. It has been shown that M6P/IGF2R expression is significantly reduced in a variety of tumors and loss of heterozygocity (LOH) at the M6P/IGF2R gene locus 6q26 have been found in breast, liver cancers and squamous cell carcinoma of the lung [12-15]. Although several studies have examined the effect of M6P/IGF2R over-expression on cell growth [7], it is not known whether down-regulation of this receptor protein leads to cellular protection against apoptosis.
Ribozymes are catalytic RNA molecules that cleave a complementary mRNA sequence [16], thereby inactivating specific mRNAs and suppressing gene expression in vitro and in vivo [17,18]. Ribozymes have been shown to be highly specific, efficient and stable. They can be packaged into viral vectors to enhance transfer into cells and to achieve longer expression compared with naked oligonucleotides. In the present study, we employed ribozyme technology to study the role of M6P/IGF2R in regulation of cardiac myocyte cell growth. A hammerhead ribozyme against the M6P/IGF2R mRNA was constructed and packaged in an adenoviral vector. We then examined the effect of ribozyme-mediated down-regulation of M6P/IGF2R expression on cell growth and hypoxia- and TNF-induced apoptosis.<br><br>Results
Cleavage reaction of the ribozyme in vitro
The M6P/IGF2R ribozyme we constructed has 13-bp binding arms complementary to the target site of M6P/IGF2R mRNA, and a catalytic core (Fig. 1A). To evaluate the bioactivity of the ribozyme and the accessibility of the target site, a cleavage reaction was performed in vitro. The substrates, [α-32P] labeled RNA transcripts containing 45 bp of M6P/IGF2R mRNA or an unmatched sequence, were incubated with the ribozyme as described (see Materials and Methods). The ribozyme cleaved only the specific M6P/IGF2R mRNA into the expected products. In the assay of time course, the hammerhead ribozyme was able to cleave 24.2% of the M6P/IGF2R target within 10 minutes of incubation, 50.3% of the M6P/IGF2R target within 40 minutes of incubation, and by 640 minutes, 80.8% of the M6P/IGF2R target was converted to the expected products (Fig. 1B). This ribozyme did not digest the unmatched sequence (Fig. 1B). These results indicate a high efficiency and specificity of the ribozyme in vitro.<br><br>Ribozymes down-regulate M6P/IGF2R expression in cardiac myocytes
To examine the ability of the ribozyme to reduce levels of M6P/IGF2R mRNA in cultured cardiac myocytes, total RNA was extracted from cells infected with Ad-GFP/IGF2R-Rz or Ad-GFP, and subjected to RT-PCR using M6P/IGF2R-specific primers. Primers specific for β-actin were added to a parallel reaction to serve as an internal standard. Cells were used 4 days after infection, with average infection efficiency of 70–80% (for which a viral dose used had minimal cytotoxicity). The RT-PCR product of M6P/IGF2R was 856 bp, and the β-actin product was 285 bp. As shown in Fig. 2A, the Ad-GFP/IGF2R-Rz-infected cells exhibited a significantly lower level of M6P/IGF2R mRNA than Ad-GFP-infected cells, with a reduction of about 50%. This result was confirmed by ribonuclease protection assay (RPA), in which GAPDH was used as a control (Fig. 2C &2D). There was no significant difference in the level of M6P/IGF2R mRNA between Ad-GFP-infected cells and uninfected cells (data not shown), indicating that infection with the <span class="red">adenovirus</span> itself did not alter the endogenous M6P/IGF2R mRNA level. The results demonstrated that the ribozyme was highly effective in suppressing M6P/IGF2R expression in cultured cardiac myocytes.<br><br>Effect of ribozyme expression on the functional activity of M6P/IGF2R
To determine the effect of the ribozyme on the functional activity of M6P/IGF2R, binding and internalization of exogenous 125I-IGF-II was measured in cells infected with Ad-GFP/IGF2R-Rz. As shown in Fig. 3A, cells infected with Ad-GFP/IGF2R-Rz showed a 54% reduction in 125I-IGF-II internalization when compared with the control cells (infected with Ad-GFP). We also examined the effect of the ribozyme on the M6P-binding activity of the M6P/IGF2R using the M6P-bearing lysosomal enzyme, β-glucuronidase, as a probe. The results showed that the maximal M6P-binding capacity of cells treated with the ribozyme was about 50% less than that of controls (Fig. 3B). Furthermore, we assessed the ability of cells to internalize exogenous β-glucuronidase after treatment with ribozyme. Similarly, the M6P-inhibitable endocytosis of β-glucuronidase by ribozyme-treated cells was about 52% less than that of control cells (Fig. 3C). These results confirm that the number of functional M6P/IGF2R in ribozyme-treated cells was reduced.<br><br>Adenoviral delivery of ribozymes increases the proliferation of cardiac myocytes
We examined the effects of the ribozyme on the growth of cultured neonatal <span class="yellow">rat</span> cardiac myocytes. Morphological evaluation showed a remarkable difference in growth pattern between Ad-GFP/IGF2R-Rz-infected cells and the control cells: the ribozyme-expressing cells formed larger and more spread colonies (Fig. 4). Assessment of cell proliferative activity by the MTT assay and counts of viable cells showed that the number of cardiac myocytes in ribozyme-expressing cultures was significantly higher than in control cultures (Fig. 5). These results indicate that treatment with M6P/IGF2R-ribozyme can promote cardiac myocyte proliferation.<br><br>Effect of M6P/IGF2R-ribozyme expression on apoptosis of cardiac myocytes
We examined the effects of ribozyme expression on TNF-α and hypoxia-induced apoptosis of cultured cardiac myocytes. After a 24 hr challenge with hypoxia, the number of apoptotic cells in M6P/IGF2R-Rz expressing cultures was 38% lower than in control cultures as determined by Hoechst staining (which highlights the nuclei of apoptotic cells) and ELISA (Fig. 6A, 7A). MTT analysis showed that the number of viable cells in ribozyme-treated cultures was 40% higher than in control cultures (Fig. 7A).
After treatment with TNF-α, as shown in Fig. 6B, a large number of control cells underwent apoptosis, as indicated by morphological changes (small round shape) and bright blue nuclear staining. There were significantly more apoptotic cells in control cultures than in cultures expressing the Ad-GFP/IGF2R-Rz. The number of apoptotic cells, as measured by the cell death ELISA assay, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 40%) lower than in cultures infected with Ad-GFP (Fig. 7B). Accordingly, the number of viable cells, as measured by MTT analysis, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 45%) higher than in cultures infected with Ad-GFP (Fig. 7B). These results are consistent with the hypothesis that decreasing M6P/IGF2R expression by ribozyme treatment can reduce cell apoptosis.<br><br>
Discussion
Some 62,000,000 Americans have one or more types of cardiovascular disease (CVD) and CVD is the leading cause (40.1%) of death in the United States. Myocardial infarction and heart failure, conditions accompanied by cardiac myocyte apoptosis, represent 23% of all CVDs and are a growing clinical challenge in need of novel therapeutic strategies. In this study, we investigated the M6P/IGF2R as a potential new therapeutic target for reduction of cardiac apoptosis and cardiac injury in these conditions.
Using ribozyme technology we down-regulated the expression of the M6P/IGF2R in neonatal cardiac myocytes. We then examined cell proliferation and apoptosis under normal conditions and post challenge with either hypoxia, a model of ischemia-reperfusion, or TNF-α, a cytokine implicated in the pathogenesis of chronic heart failure [19]. Our results demonstrate an association of a decrease in the expression and function of the M6P/IGF2R with increased cell proliferation and decreased cell susceptibility to hypoxia- and TNF-induced apoptosis. Expression of the ribozyme targeted against the M6P/IGF2R in cardiomyocytes resulted in down-regulation of M6P/IGF2R expression, as measured by RT-PCR and RPA, and of M6P/IGF2R function, as indicated by a decrease in internalization of 125I-IGF-II, and β-glucuronidase binding and endocytosis.
MTT analysis and viable cell counts showed that ribozyme-mediated down-regulation of M6P/IGF2R resulted in a marked increase in cell proliferation of cardiomyocytes, which normally express high levels of M6P/IGF2R [20] and have limited proliferative capabilities [21]. These results are consistent with the findings of previous knockout studies [9,10]. Since the M6P/IGF2R has multiple actions on cell growth, its proliferative effect on the heart cells observed in this study might involve multiple mechanisms. However, it is likely that unchecked IGF-II stimulation plays a key role in the effect. Because the M6P/IGF2R is believed to sequester and degrade IGF-II [2], a decrease in M6P/IGF2R expression and function could result in decreased degradation and hence increased bioavailability of IGF-II to the IGF-I receptor, which mediates the growth-promoting effect of IGF-II. Supporting evidence for the involvement of IGF-II in the proliferative effect resulting from loss of M6P/IGF2R function comes from studies of M6P/IGF2R knock-out <span class="yellow">mice</span>. M6P/IGF2R-null <span class="yellow">mice</span> display global hyperplasia that coincides with elevated levels of IGF-II. Most importantly, however, the lethal nature of an M6P/IGF2R-null phenotype is reversed in an IGF-II-null background [9]. Our results showing that ribozyme-mediated down-regulation of M6P/IGF2R lead to a decrease in IGF-II internalization support the above possibility. However, further investigation to confirm this mechanism is warranted.
More importantly, our results also showed that M6P/IGF2R down-regulation resulted in decreased sensitivity of cardiomyocytes to hypoxia- and TNF-induced apoptosis. There is evidence that lysosomal enzymes, such as cathepsins B and D contribute to hypoxia- and TNF-induced apoptosis in vitro [22-25] and in vivo [26,27]. The M6P/IGF2R has been shown to be involved in binding, transport and activation of lysosomal enzymes, including cathepsins [4,5]. Therefore, it is possible that down-regulation of the M6P/IGF2R results in improper trafficking and activation of cathepsins. This, in turn would eliminate the apoptotic cascades triggered by these enzymes under hypoxia and TNF stimulation and result in decreased sensitivity of cardiomyocytes to apoptosis.
It has also been shown that TNF stimulation involves the activation of TGF-β [28-30], a ligand of M6P/IGF2R that has been implicated in the progression of chronic heart failure [11,31]. Therefore, down-regulation of M6P/IGF2R expression could also lead to a decreased bioavailability of activated TGF-β, thereby decreasing the sensitivity of cardiomyocytes to the TNF/TGF-β apoptotic pathway. The detailed mechanism of the observed effects is unknown and requires further investigation.<br><br>Conclusions
The present study demonstrates that ribozyme-mediated down-regulation of expression and functional activity of the M6P/IGF2R results in a decrease in the susceptibility of cardiac myocytes to apoptotic stimuli. These findings suggest that this receptor might be involved in cardiac cell growth and apoptosis. The ability of the M6P/IGF2R ribozyme to reduce M6P/IGF2R expression and function in transfected cells verifies the utility of the ribozyme in studying the role of M6P/IGF2R in cardiomyocyte growth and apoptosis. In addition to its utility as a research tool, the ribozyme, with further exploration and development, might have potential application as a therapeutic agent to prevent cell death or promote mitogenesis for certain clinical conditions, such as, myocardial infarction and chronic heart failure.<br><br>Methods
Construction of recombinant M6P/IGF2R-RZ adenoviral vector
The nucleotide numbers of the <span class="yellow">rat</span> M6P/IGF2R sequence targeted by the hammerhead ribozyme is 1147–1160 after coding site (exon 9). The structure of the M6P/IGF2R hammerhead ribozyme is shown in Fig. 1. A 49 bp M6P/IGF2R ribozyme oligonucleotide, 5'-GAATTCCCC ACACTG ATGAGCCGCTTCGGCGGCGAAACATTCAAC GCGT-3' and the corresponding reverse complementary strand were synthesized. The fragments were subcloned to produce a plasmid containing a ribozyme against M6P/IGF2R. For construction of the recombinant <span class="red">adenovirus</span> containing the M6P/IGF2R-ribozyme (pAd-GFP/IGF2R-Rz), the segments containing the ribozymes were amplified by PCR and cloned into a pAdTrack-CMV vector and then recombined homologously with an adenoviral backbone pAdEasy 1 vector to generate (pAd-GFP/IGF2R-Rz), following the protocol described by He et al. [32]. The pAd-GFP/IGF2R-Rz carries both the IGF2R-Rz and GFP (as reporter) genes, each under the control of separate cytomegalovirus (CMV) promoters. Another viral vector, pAd-GFP, which carries the GFP gene only under the control of the CMV promoter, was generated and used as a control vector. The adenoviral vector DNA were linerized with Pac I and transfected into the replication-permissive 293 cells (E1A transcomplementing cell line) by using Lipofectamine (Life Technologies) to produce E1-deleted, replication-defective recombinant <span class="red">adenovirus</span> as described previously [33]. Large-scale amplification of recombinant <span class="red">adenovirus</span> in 293 cells was followed by purification using a discontinuous CsCl gradient. The constructs were confirmed by enzymatic digestion and DNA sequencing.<br><br>Transcription and cleavage reaction of ribozyme in vitro
Plasmids containing the ribozyme or the substrate (either 45 bp of M6P/IGF2R mRNA or an unmatched sequence 5'-GTGCTGTCTGTATG-3') were linearized with MluI, respectively. All transcripts were generated with T7 RNA polymerase (Promega). Substrate transcripts were labeled by incorporation of [α-32P] UTP (NEN Life Science Products, Inc.). Specific activity of the [α-32P] UTP (10 μCi/μl) and the base composition of each substrate molecule were used to calculate the substrate concentration. Ribozyme transcripts were quantified spectrophotometrically. (The half-life of the M6P/IGF2R target is about 280 minutes).
Cleavage reaction mixture contained substrate RNA (40 nM), increasing amounts of ribozyme (60 nM), 20 mM MgCl2 and 20 mM Tris-HCl, pH8.0, in a final volume of 10 μl. The mixture was incubated at 37°C for a time-course of cleavage reaction from 0, 5, 10, 20, 40, 80, 160, 320, to 640 minutes and the cleavage reaction was stopped by addition of loading buffer (80% formamide, 10 mM Na2EDTA, pH 8.0, and 1 mg/ml each bromophenol blue and xylene cyanol). Cleavage products were analyzed on a 15% polyacrylamide and 8M urea gel. Product and substrate fragments were quantitated by using NIH Imager.<br><br>Cell cultures and infection with Ad-GFP/Rz-IGF2R and Ad-GFP
Cardiac myocytes were isolated from 1-day-old newborn <span class="yellow">rats</span> using the Neonatal Cardiomyocyte Isolation System (Worthington). The isolated cells were plated in 6-well plates and cultured in F-10 medium containing 5% (vol/vol) FBS and 10% (vol/vol) <span class="yellow">horse</span> serum at 37°C in a tissue culture incubator with 5% CO2 and 98% relative humidity. Cells were used for experiments after 2–3 days of culture. Viral infections were carried out by adding viral particles at various concentrations (usually, 2 × 108 virus particles/ml) to culture medium containing 2% (vol/vol) FBS. Initially, optimal viral concentration was determined by using Ad-GFP to achieve an optimal balance of high gene expression and low viral titer to minimize cytotoxicity. After 24 hours of incubation, the infection medium was replaced with normal (15% vol/vol serum) culture medium. For treatment with IGF-II, cells were incubated with 50 ng/ml IGF-II after 24 hours infection with Ad-GFP/IGF2R-Rz or Ad-GFP. Four days after infection, cells were used for analysis of gene expression of M6P/IGF2R and its effect on cell growth and apoptosis.<br><br>Analysis of gene expression in cardiac myocytes
The M6P/IGF2R transcripts were determined by both RT-PCR and Ribonuclease Protection Assay (RPA). RT-PCR was performed using the GeneAmp EZ rTth RNA PCR kit (Roche). Total RNA was extracted from cultured cells using an RNA isolation kit (Qiagen,), according to the manufacturer's protocol. M6P/IGF2R transcripts were amplified using the primers (5'-GACAGGCTCGTTCTGACTTA-3') and (5'-CTTCCACTCTTATCCACAGC-3') specific to the M6P/IGF2R. Each RT-PCR assay was performed in triplicate and product levels varied by less than 3.2% for each RNA sample. Primers specific for β-actin cDNA were added to a parallel reaction to standardize for variations in PCR between samples. PCR products were resolved on a 1.0% agarose gel, visualized under UV light and quantitated using NIH Imager.
RPA was performed using the RPA III kit (Ambion, Austin, TX). Briefly, total RNA was extracted from cultured cells using a total RNA isolation reagent (TRIzol, Gibco BRL) according to the manufacturer's protocol. The plasmid containing the <span class="yellow">rat</span> M6P/IGF2R gene was linearized and used as a transcription template. Antisense RNA probes were transcribed in vitro using [33P]-UTP, T7 polymerase (Riboprobea System T7 kit, Promega), hybridized with the total RNA extracted from the <span class="yellow">rat</span> cardiomyocytes, and digested with ribonuclease to remove non-hybridized RNA and probe. The protected RNA·RNA was resolved on a denaturing 5% sequence gel and subjected to autoradiography. A probe targeting the GAPDH gene was used as an internal control.<br><br>Measurement of 125I-IGF-II internalization
Cells were incubated at 37°C for 2 hrs in serum-free F-10 culture medium containing 125I-labeled IGF-II (0.5 ng/ml) with or without excess unlabeled IGF-II (2 μg/ml). Following the incubation, the cells were washed three times with ice-cold PBS, and cell-associated radioactivity was determined by a γ counter. Specific internalized 125I-IGF-II was calculated by subtracting the count of samples with excessive unlabeled IGF-II from that without unlabeled IGF-II, and normalized to protein contents.<br><br>Beta-glucuronidase binding assay
Binding of β-glucuronidase was assayed as described previously [34,35]. Briefly, cells were permeabilized with 0.25% saponin in 50 mM Hepes (pH 7.0), 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="yellow">human</span> serum albumin, and 10 mM mannose-6-phosphate (M6P) for 30 minutes on ice. The cells were washed three times with ice-cold PBS containing 0.05% saponin. They were incubated with 20,000 units/ml β-glucuronidase from <span class="yellow">bovine</span> liver (Sigma) in 50 mM Hepes (pH 7.5) containing 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="yellow">human</span> serum albumin, 0.5% saponin with or without 10 mM M6P overnight on ice. Cells were washed five times with ice-cold PBS containing 0.05% saponin and sonicated in 100 mM sodium acetate (pH 4.6). The protein concentration of solubilized cell extract was measured and enzyme activity was assayed as follows: for each reaction 50 ul cell extract were added to 500 ul of 100 mM sodium acetate (pH 4.0) containing 1 mM paranitrophenyl (PNP)-β-glucuronide (Sigma) as substrate. After an incubation period of 3 hours at 37°C, 500 ul 1 M Na2CO3 were added to each reaction and the absorbance was measured at 400 nm. Experimental values were compared to a standard curve that was constructed using 1–100 nM solutions of PNP (Sigma) in 500 ul 100 mM sodium acetate and 500 u1 1 M Na2CO3. Specific activity was calculated as nM of PNP produced/hour/mg of protein.<br><br>Beta-glucuronidase endocytosis assay
Beta-glucuronidase endocytosis assay was carried out as described previously [36]. Briefly, confluent cell cultures were washed twice with pre-warmed serum-free DMEM followed by incubation with DMEM containing 5 mg/ml <span class="yellow">human</span> serum albumin and 10 mM M6P for 20 minutes. Following incubation cells were washed 3 times with pre-warmed DMEM. Cells were then incubated in DMEM containing 5 mg/ml <span class="yellow">human</span> serum albumin alone or 4000 units β-glucuronidase with or without 10 mM M6P for 2 hours at 37°C. Following the incubation, the cells were washed 5 times with ice-cold PBS and subjected to enzyme activity assay as described above.<br><br>Cell proliferation assay (MTT assay and cell counts)
Cardiac myocytes were grown in culture plates (tissue culture grade, 12 wells, flat bottom) in a final volume of 1 ml serum-containing culture medium per well, in a humidified atmosphere (37°C and 5% C02) for 3 days. After infection with Ad-GFP/IGF2R-Rz or Ad-GFP, cells were incubated with or without 50 ng/ml IGF-II for 4 days. Following supplementation with IGF-II, 100 μl MTT labeling reagent (Roche) were added to each well and cells were incubated for 4 hours, followed by addition of 1 ml solubilization solution into each well. The plate was placed in an incubator at 37°C overnight. Spectrophotometrical absorbency of the samples was measured using an UV-visible Recording Spectrophotometer with wavelength of 550–690 nm. In addition, the total number of viable cells in each treatment was counted by trypan blue exclusion method using a hemocytometer.<br><br>Induction and analysis of cell apoptosis
Cells were infected with Ad-GFP or Ad-GFP/IGF2R-Rz. Seventy-two hours post infection, cells were treated with TNF (0.1 ng/ml) for 24 hrs or subjected to hypoxia. For induction of apoptosis by hypoxia, cell culture medium was changed to serum-free F-10 saturated with 95% N2/5% CO2 and cells were placed in a 37°C airtight box saturated with 95% N2/5% CO2 for 24 hrs. For normoxic controls, culture medium was changed to F-10/5%F BS/10% HS and cells were placed in a 37°C/5% CO2 incubator for 24 hrs before analysis.
Apoptotic cells were identified by Hoechst staining using the Vybrant™ Apoptosis Kit #5 (Molecular Probes) according to the manufacturer's protocol. In addition, after infection with Ad-GFP or Ad-GFP/IGF2R-Rz and challenge with either TNF or hypoxia, cell viability was assessed using the MTT assay Kit (Roche Molecular Biochemicals) and cell apoptosis was determined using the Cell Death Detection ELISA Kit assay (Roche Molecular Biochemicals) according to the manufacturer's protocol.<br><br>Statistical analysis
Students' t-test was used to evaluate the difference between two values. Each experiment was repeated at least three times. Statistical significance was accepted at the level of p < 0.05.<br><br>
List of abbreviations used
Ad-GFP, <span class="red">adenovirus</span> carrying GFP gene; Ad-GFP/IGF2R-Rz, <span class="red">adenovirus</span> carrying both the ribozyme against M6P/IGF2R and the GFP gene; GFP, green fluorescent protein; IGF-II, insulin-like growth factor II; M6P/IGF2R, mannose 6-phosphate/insulin-like growth factor II receptor; Rz, ribozyme.<br><br>Authors' contributions
ZC carried out construction of the ribozyme, production of the viruses, cellular experiments, biochemical assays and data analysis.
YG carried out the RPA assay and participated in the molecular biological studies.
JXK conceived of the study, participated in its design and coordination, and drafted the manuscript.<br><br>
<h3>pmcA324075</h3>Structural organization of a 17 KB segment of the alpha 2 collagen gene: evaluation by R loop mapping.
Abstract
A recombinant phage, SpC3, containing a 17 kb genomic DNA insert representing approximately 60% of the 3' portion of the <span class="yellow">sheep</span> collagen alpha 2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 alpha 2 coding sequences (exons) were mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region required for coding the interspersed 3 kb of alpha 2 collagen genetic information was 5.6 fold longer than the corresponding alpha 2 mRNA coding sequences.Images<br><br><br><br><br><br>
 Nucleic Acids Research <br><br> Structural organization of a 17 KB segment of the a2 collagen gene: evaluation by R loop mapping Millie P.Schafer, Charles D.Boyd, Paul Tolstoshev and Ronald G.Crystal <br><br> Pulmonary Branch, National Heart, Lung and Blood Institute, National Institutes of Health, Bethesda, MD 20205, USA <br><br> Received 13 February 1980 <br><br> ABSTRACT <br><br> A recombinant phage, SpC3, containing a 17 kb genomic DNA insert rep<br><br> resenting approximately 60% of the 3' portion of the <span class="yellow">sheep</span> collagen a2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 a2 coding sequences (exons) were <br><br> mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region <br><br> required for coding the interspersed 3 kb of a2 collagen genetic information was 5.6 fold longer than the corresponding a2 mRNA coding sequences. <br><br> INTRODUCTION <br><br> Type I collagen, composed of two al(I) and one a2 polypeptide chains, is the most abundant of the five known mammalian collagen types. It is a major extracellular constituent of tissues such as bone, tendon, skin and lung where, because of its great tensile strength, it plays an important <br><br> role in tissue structure and function. The polypeptides comprising type I <br><br> collagen are synthesized in long, precursor forms, referred to as pro al(I) and pro a2 chains,1 each containing approximately 1500 amino acid residues (1). <br><br> 1 The terminology for the primary translation products of the various <br><br> collagen messenger RNAs is still in a state of flux. It is known that al(I) and a2 chains are synthesized in precursor forms; these are currently termed pro al(I) and pro a2 chains (1). However, recent studies (43,44) have shown that the actual primary translation product of al(I) mRNA is somewhat larger <br><br> than the pro al(I) chain; this translation product has been termed a "pre-pro al(I) chain." Although the corresponding "pre-pro a2 chain" likely exists, it has not yet been characterized. For simplicity, therefore, until the <br><br> proper terminology for the primary translation product is clarified, we have <br><br> chosen to use the terms "al(I)" and "a2" to refer to the type I collagen mRNAs and collagen structural gene sequences. <br><br> ?) IRL Press Umited, 1 Falconberg Court, London W1V 5FG, U.K. <br><br> Volurne 8 Number 10 1980 <br><br> 2241 <br><br> Nucleic Acids Research <br><br> There is increasing evidence that the quantities of al(I) and a2 <br><br> chains produced by cells may be regulated, in part, at the genomic level <br><br> (1). However, to understand the processes which mediate their expression, <br><br> it is necessary to understand the structural organization of these collagen genes and the various enzymes that modify the collagen chains after its <br><br> synthesis (1). As an initial approach to this problem, we have recently <br><br> isolated a recombinant <span class="yellow">bacteriophage</span>, termed SpC3, containing approximately 60% of the a2 gene for type I collagen. Partial characterization of this <br><br> a2 gene demonstrated that, like most other structurally evaluated eukaryotic genes, the coding sequences are interspersed with intervening sequences (2). The present study further characterizes this portion of the a2 <br><br> gene by utilizing electron microscopic R loop analysis to map the coding regions (exons) and intervening sequences (introns) contained within the <span class="yellow">sheep</span> genomic insert of SpC3. The data suggests that the structural <br><br> organization of this portion of the a2 collagen gene represents 3 kb of coding information that is interspersed in a complex fashion with 17 introns over 17 kb of genome. <br><br> MATERIALS AND METHODS <br><br> Isolation and characterization of a2 collagen recombinant clone, SpC3. <br><br> The <span class="yellow">sheep</span> a2 collagen recombinant clone, SpC3, containing a portion of the <span class="yellow">sheep</span> a2 gene, was isolated as previously described (2). Briefly, <br><br> high molecular weight fetal <span class="yellow">sheep</span> liver DNA was extracted by the method of Blin and Stafford (3), and 15-20 kb DNA fragments, resulting from partial Eco RI digestion, were isolated and ligated to the left and right arms of Charon 4A. The recombinant DNA was then packaged in vitro and the <span class="yellow">sheep</span> <br><br> genomic library amplified (4-6). Screening of the <span class="yellow">sheep</span> genomic library was <br><br> conducted utilizing [32 P]-labeled fetal <span class="yellow">sheep</span> tendon type I collagen cDNA and <br><br> several positive recombinant phages were isolated. One recombinant, containing a 17 kb <span class="yellow">sheep</span> genomic insert, was demonstrated to have coding sequences corresponding to 60% of the 3' end of a2 mRNA. Prior studies showed that the insert in SpC3 was not due to ligation of noncontiguous restriction fragments and/or genetic rearrangement during the cloning process (2). Electron mi croscopy. <br><br> To visualize the recombinant <span class="yellow">bacteriophage</span> DNA structure complementary to a2 collagen mRNA, 1 iig of SpC3 DNA was lyophilized with 1 ig or 100 ng of mRNA <br><br> to give DNA:mRNA ratios of 1:1 and 10:1, respectively. These were then dissolved in 10 i1 of R loop buffer [70% (v/v) deionized formamide, 0.1 N-[tris (hydroxy<br><br> 2242 <br><br> Nucleic Acids Research <br><br> methyl)methyl] glycine (tricine)-NaOH, pH 8.0, 0.5 M NaCl, and 0.01 M ethyl<br><br> enediaminetetraacetate (EDTA)], incubated in sealed capillary tubes at 520 for 14 hr (7), and then diluted 1:10 with R loop buffer. Two spreading methods were used to subsequently prepare the hybrids for visualization in the electron microscope. <br><br> 70% formamide method. Immediately before spreading onto a deionized <br><br> water hypophase, the nucleic acid mixtures were diluted an additional 10-fold in R loop buffer and cytochrome c, 100 vg/ml, was added. <br><br> Urea-formamide method.   The nucleic acid solutions were spread using a modification of the method of Westphal and Lai (8,9). The hyperphase contained 55% formamide, 2.6 M urea, 0.009 M EDTA, 0.09 M tricine-NaOH, pH 8.0, and the nucleic acid mixture (approximately 0.1 to 1.0 ig/ml). <br><br> The mixture was heated at 40? for 30 sec, placed in ice water, and then allowed to reach room temperature. Cytochrome c, 100 pg/ml, was added immediately before spreading onto a deionized water hypophase. <br><br> The nucleic acid-protein films from both methods were absorbed onto <br><br> parlodion-coated grids, stained with uranyl acetate (10), dehydrated in 90% ethanol and rotary-shadowed with platinum-palladium (80:20) at an angle of 50. Micrographs were taken with a Siemens Elmiskop 101 electron microscope at an original magnification of 10,000 and an accelerating voltage of 60 kv. Nucleic acid lengths were measured at a final magnification of 43,000 with a <br><br> Hewlett-Packard 9810A calculator equipped with a 9864A digitizer using pBR322 and fX-174 double and single stranded DNA, respectively, as internal length standards. <br><br> Approximately 10,000 hybrid molecules were screened for these studies; approximately 200 represented molecules containing unambiguous regions <br><br> appropriate for quantitative analysis. The mean lengths of each intron and exon were determined for each method and the data expressed as mean ? <br><br> standard deviation. Nucleic acid lengths of < 50 base pairs could not be <br><br> accurately determined using the methods outlined above. Thus, in the cases <br><br> where introns or exons were of this length or less, accurate error estimates could not be made. <br><br> Bi ohazard precauti ons. <br><br> The construction and screening of the <span class="yellow">sheep</span> genomic library together with amplification of pCg45 and preparation of high titre lysates of chimaeric <br><br> Charon 4A were performed under the physical and biological containment levels specified by the NIH guidelines for recombinant DNA research (11). <br><br> 2243 <br><br> Nucleic Acids Research <br><br> RESULTS <br><br> Location and orientation of the intervening and a2 mRNA coding sequences in the SpC3 recombinant DNA. <br><br> Hybridization of fetal <span class="yellow">sheep</span> tendon type I collagen mRNA to SpC3 DNA <br><br> yielded a complex R loop pattern in the region where the <span class="yellow">sheep</span> genomic insert was expected to be ligated to the arms of the lambda vector, Charon 4A. To <br><br> confirm that these hybridization events were restricted to the inserted <span class="yellow">sheep</span> DNA fragment, duplex DNA strands on both sides of this hybridization area were measured. Using pBR322 as an internal length standard, the left arm of the phage was found to contain 21,780 ? 1500 bp, and the right arm, 11,950 ? <br><br> 1030 bp, corresponding to the report values for the left and right arms, respectively, of Charon 4A DNA (5,6). <br><br> The hybridization of SpC3 to a2 mRNA resulted in the destabilization of <br><br> the inserted helical duplex DNA fragment such that one DNA strand was displaced (Figure 1). Occasionally, the displaced DNA strand was mostly absent, probably due to fragmentation induced by accidental mechanical shearing. This event considerably enhanced the visualization and ordering of the introns. <br><br> As the a2 mRNA annealed to complementary DNA regions present in one <br><br> DNA strand, numerous single-stranded loops resulted. These loops, referred to as introns, represented DNA sequences not complementary to the mRNA. Although, by convention, introns are usually sequentially labeled 5' to 3', an opposite order had to be employed in this case as SpC3 does not contain the sequences coding for the 5' end of <span class="yellow">sheep</span> a2 mRNA. Infrequently, all introns <br><br> were clearly observed within the same hybrid molecule (Figure 1). The largest introns were towards the left arm of Charon 4A and were more clustered than the introns at the opposite end of the hybrid. Thus, much of the genetic <br><br> information coding for the a2 mRNA was contained in that half of the hybrid attached to the right arm. <br><br> A small non-hybridized tail was consistently found in the region where <br><br> the right arm of the phage was ligated to the <span class="yellow">sheep</span> genomic insert (Figure 1). This tail likely corresponded to the 3' poly A sequence of the a2 mRNA. At <br><br> the extreme 5' end of the insert, a variable length was often observed for the remaining unhybridized mRNA strand. This was most likely due to the size heterogeneity of the a2 mRNA used in these studies. <br><br> Size determination and comparative analysis of SpC3 introns. <br><br> Although localization of exon and intron sequences was possible with both the 70% formamide and urea-formamide spreading methods, the probability of <br><br> detecting unambiguous regions in a hybrid molecule was far greater with the <br><br> 2244 <br><br> Nucleic Acids Research <br><br> A                                B ',:  '14 .0. <br><br> 13 <br><br> 7 1 <br><br> -5 13 <br><br> .  .   . .   ,4                     x2%.   N1 <br><br> 9 <br><br> >   10          R       4 <br><br> Figure 1.    Electron microscopic visualization of all 17 introns and 18 exons <br><br> from hybrid molecules formed between recombinant clone SpC3 DNA and fetal <span class="yellow">sheep</span> <br><br> tendon a2 mRNA. (A,B) Shown are the double-stranded segment of the right arm CR) of Charon 4A; double-stranded segment of the left arm CL) of Charon 4A; the <br><br> displaced single-stranded DNA segment of the insert (5); single-stranded DNA loops representing introns sequentially labeled 1 through 17 in the 3' to 5' direction; and 18 regions of insert DNA sequences hybridized to ct2 mRNA <br><br> ---      - ).  (C,D) Similar to (A,B) but the single-stranded DNA segment (5) of the insert is mostly absent. The urea-formamide spreading method, as described in Materials and Methods, was employed. <br><br> 2245 <br><br> Nucleic Acids Research <br><br> urea-formamide method than with the 70% formamide procedure. For this reason, most measurements were made with the urea-formamide method (Tables 1,2). For <br><br> example, the lowest probability event, the detection of intron 17 and the surrounding small exons, "q" and "r", was only observed by way of the ureaformamide spreading procedure. <br><br> The smallest introns, 7 and 12, were found to contain approximately <br><br> the same number of bases. Introns 1 through 8, representing the DNA sequences which split the a2 coding sequence corresponding to the 3' end of the a2 mRNA, varied in length over approximately a 2.4-fold range. With the exception of <br><br> Table 1. Electron microscopic R loop analysis <br><br> present in SpC3 DNA1. <br><br> of the <span class="yellow">sheep</span> a2 introns <br><br> SpC3 was hybridized to <span class="yellow">sheep</span> a2 mRNA as described in Materials and Methods; intron lengths were determined by comparison with *X-174 DNA lengths. <br><br> 2 Introns are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 Intron 17 was visualized only by the urea-formamide method. <br><br> 2246 <br><br> 70% Formamide Method              Urea-Formamide Method <br><br> Intron2         Number of       Intron length       Number of        Intron length <br><br> introns analyzed       (bases)       introns analyzed      (bases) <br><br> 1              10             819 t 063              35            806 t 76 2              12             771 ? 89               46            757 t 78 3              11             458 ? 58               47            464 ? 66 4               6             627? 53               41             611 ? 73 5               5             554 ? 51              34             562 ? 76 6               6             675 ? 19               40            684 ? 76 7               5             386 ? 48               50            342 ? 54 8                7            771 ? 84               49            757 ? 82 9               8           1,181 ? 94               52          1,124 ?122 10               8           1,133 ?106               53          1,124 ?137 11              10             964 t?79              61             952 ?105 12               9             362 ? 77              62             342 ? 71 13               4             723 ?222               53            806 ?105 14               4           1,085 ?142               53          1,075 ?105 15               3             795 ? 60              47             806 ?100 16               2           1,229 ? 48              43           1,319 ?146 174              -                 -                  12          1,539 ?166 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Table 2.   Electron microscopic R loop analysis of the <span class="yellow">sheep</span> a2 exons <br><br> present in SpC3 DNA1. <br><br> 70% Formamide Method                Urea-Formamide Method <br><br> Exon2          Number of        Exon length       Number of       Exon length <br><br> exons analyzed     (base pairs)    exons analyzed    (base pairs) <br><br> a                 7             807 ? 883           26            883 ?132 b                10             252 ? 35            41            234 ? 49 c                11             151 ? 25            48            182 ? 42 d                 6             252 ? 60            44            234 ? 62 e                 5             126 ? 45            37            129 ? 47 f                 5             126 ? 38            35            104 ? 41 9                 5             176 ? 48            40            182 ? 54 h                 5             100 ? 43            47            104 ? 31 <br><br> 7             100 ? 35            50            104 ? 28 j                 7             126 ? 48            45            130 ? 47 k                 8              50 ? 48            51             52 ? 39 1                 9              75? 10             60             52 ? 44 m                 6             126 ? 43            51            182 ? 54 n                 4             100 ? 45            53             52 ? 42 o                 3             <504                50             78 ? 36 p                 2             126 ? 71            47            104 ? 36 q5                - _                               33            130?47 r5 r                                                 6             78?36 <br><br> SpC3 was hybridized to <span class="yellow">sheep</span> a2 mRNA as described in M4aterials and <br><br> Methods; exon lengths were determined by comparison with pBR322 DNA lengths. <br><br> 2 Exons are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 5 <br><br> No statistical evaluation was possible because of the size of the exon (see (Materials and Methods). <br><br> Exons q and r were only analyzed by the urea-formamide method. <br><br> intron 1, the nucleotide size of these introns was less than 800 bases. In contrast, introns 9 through 17 varied over a 4.5-fold range. Except for intron 12, all of the introns 9 through 17 contained 800 bases or more. <br><br> Summation of introns 1 through 16 for the 70% formamide and urea<br><br> formamide spreading methods yielded 12,533 and 12,531 bases, respectively. <br><br> However, detection of intron 17 by the urea-formamide method indicates that <br><br> the actual summed value for the intervening sequences of SpC3 is 14,070 bases. <br><br> 2247 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Size determination and the comparative analysis of SpC3 exons. <br><br> Eighteen exons, "a" through "r", were mapped (Table 2). The largest exon, "a", located on the extreme 3' end of the <span class="yellow">sheep</span> genomic insert, contained <br><br> approximately 800 bp. With the exception of exon "m", exons larger than 150 <br><br> base pairs were confined to the genetic region corresponding to the far 3' end of the a2 mRNA. Exon "a" and exon "b", the two exons situated on either side of intron 1, coded for around 1000 bases of a2 collagen information; these two <br><br> exons contained one-third of the mapped 3,014 DNA bases complementary to a2 mRNA Length distribution and organization of the introns and exons in the <span class="yellow">sheep</span> genomi c i nsert. <br><br> There was a sharp contrast in the size distribution of the exons and introns contained within SpC3 (Figure 2). While the intron sizes varied <br><br> considerably, the exons, in general, showed a very narrow distribution of <br><br> lengths, and thus tended to cluster at the lowest end of the length distri<br><br> A. <br><br> (I) <br><br> z 6 0 <br><br> z <br><br> ZL4 0 <br><br> 021               <br><br> z <br><br> 10 <br><br> B. <br><br> 8 <br><br> z 0 <br><br> 16 0 z <br><br> 2 <br><br> 20   400  600  800 1000 1200 1400 1600 <br><br> LENGTH IN NUCLEOTIDES <br><br> Figure 2. Length distribution of the intron and exon mean lengths measured from micrographs prepared by the urea-formamide method. (A) Introns; (b) Exons. See Tables 1 and 2 for details. <br><br> 2248 <br><br> Nucleic Acids Research bution graph. The exon in the 800-900 bp region was unique and was the only exon which exhibited length overlap with introns. <br><br> The maps of the a2 collagen gene obtained with both electron micro<br><br> scopic spreading methods were almost identical (Figure 3), except that the <br><br> region at the extreme 5' end of the insert could only be mapped with the ureaformamide method. From these measurements we concluded that the length of the a2 collagen genetic regi?on in SpC3 was 17,084 bases long with only 3,014 bases actually coding for a2 mRNA. Therefore, only 18% of the mapped 17 kb <span class="yellow">sheep</span> genetic insert actually contained a2 structural information. <br><br> DISCUSSION <br><br> The organizational structure of the a2 collagen gene is complex. A minimum of 17 introns interrupt the 3 kb of ca2 genetic information interspersed as 18 exons over 17 kb of <span class="yellow">sheep</span> DNA. With the exception of exon "a" located at the extreme 3' end of the inserted a2 gene, all other mapped exons are rather small, exhibiting a size distribution range of 50 to 250 bp. In <br><br> contrast, the intron sizes range from 340 to 1540 bases occupying a total <br><br> of 14 kb. Thus, the DNA region containing the 3 kb of a2 collagen genetic information is 5.6 fold longer than the corresponding co2 mRNA sequences. <br><br> Although the total number of introns present within the 60% 3' portion <br><br> of the a2 <span class="yellow">sheep</span> gene was determined to be 17, variation in the precise number <br><br> TRANSRIPTlO <br><br> A                          5w                r <br><br> 1 "2a4  133  112s  11.17  0. 3O 2  715  6.2  313  04  4.2B  3.51  2.2  12 <br><br> EXON0            p  o    n         5  Ik         h     f     d 4  .  b   <br><br> WPM300       16   16  14   13  12  11  10  9   6  7  6  5  4  3   2   1 <br><br> B <br><br> 110   16     4.2  13111  112n  1113104  3.6    gm74  2 6.  4 4 2  3.2  22  in <br><br> 1  a1m03  133  1.2 11   10  01 341  6. 1   117  6.2 4  12  417  3I5'm 2  <br><br> DM00  1    q5    p  0    n   m  I  k.   j    i   h  g  f     d  c   b     a 147306   17   16   15  14  13  12  11  10  9   a  7  6  5  4  3   2   1 <br><br> 175 17.0 16.6& 16'5.5 15.0 14.5 14. 13.5 13.0 12.5 12.0 11.5 11 10.5M 10.0 9.5 9.0 8. 5.0 71 7.0 6.5 6. 5.5 50 4.5 4.0 3.5 3.0 2.5 2.0 1.5 1.0 <br><br> KILOBASE <br><br> Fiue3      Organization of 60% of the 3' end of the <span class="yellow">sheep</span> a2 collagen gene. (A)}The exon ( ) and intron (CDJ) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules visualized by the electron microscopic 70% formamide spreading method.  (B) The exon Um) <br><br> and intron (Efl) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules evaluated by the electron microscopic <br><br> urea-formamide spreading method. Arrows represent the approximate location of intron-exon junctions. All values are presented as kilobase pairs. <br><br> 2249 <br><br> Nucleic Acids Research <br><br> of introns present in individual hybrid molecules was observed. The reason <br><br> for this may be due to size heterogeneity of the a2 mRNA used as a complementary probe. Although the a2 mRNA was extensively purified (2), some degradation was likely present. It is also possible that additional introns exist that are too small to be detected by electron microscopic R loop analysis. The most likely a2 genetic regions containing such introns are exons "g" and "m". Small <br><br> "protrusions" were sometimes observed within these exons that could represent small introns. Very small introns have been reported for tRNA genes (12,13), <br><br> and it will be necessary to sequence these regions of SpC3 to determine if such introns are present within the a2 <span class="yellow">sheep</span> genome. <br><br> The existence of mRNAs comprised of sequences complementary to noncontiguous regions in a DNA genome was first described for <span class="blue">adenovirus</span> (14,15) and <br><br> simian virus (16,17) genes. Similar observations were soon reported for eukaryotic genes, including serum albumin (18), conalbumin (19), fibroin (20), s<br><br> globin (21,22), growth hormones (23,24), immunoglobulin (25), lysozyme (26), <br><br> ovalbumin (27-29), ovomucoid (30), rRNA (31-34), tRNA (12,13), and vitellogenin (35). The precise number or occurrence of introns within these genes does not seem to follow any clear pattern. However, the genes for albumin, conalbumin, <br><br> lysozyme, ovalbumin, and ovomucoid exhibit a structural organization comparable to the a2 collagen gene. All of these genes are composed of numerous small <br><br> exons, usually less than 250 base pairs, and introns which show a large size variation within an individual gene. Within all of these genomes, the DNA <br><br> regions required for coding structural formation are 4 to 7 times longer than the corresponding mRNA coding sequences. <br><br> As with the a2 collagen gene, a large unique exon at the extreme 3' end <br><br> is also present in the ovalbumin gene (36,37). In the a2 gene, this exon was 800 to 900 nucleotides in length, while in the ovalbumin gene the exon contained 1,030 base pairs. Interestingly, this ovalbumin exon contained 634-650 base pairs ot DNA sequence to the 3' side of the terminator <br><br> triplet, suggesting the ovalbumin mRNA has a large untranslated region at the 3' end. <br><br> The consistent appearance of a small non-hybridized tail, approximately 100 nucleotides in length, in the region where the right arm of the phage was ligated to the <span class="yellow">sheep</span> genomic insert of SpC3 very likely corresponded to the <br><br> 3' poly A sequence of the a2 mRNA. This would suggest that the a2 mRNA coding sequence terminates very near or at this point in the genome. Previously published biochemical evidence (2) obtained from restriction mapping and Southern blot analysis using [32P]-labeled <span class="yellow">chick</span> a2 cDNA probe supports <br><br> 2250 <br><br> Nucleic Acids Research <br><br> this observation. <br><br> It is unclear what role introns play in genetic expression. However, <br><br> it is known that these sequences are tranncribed as part of larger precursor mRNAs and the introns are then excised and the fragmented mRNA chains <br><br> covalently rejoined (38). Some evidence has recently accumulated demonstrating the existence of precursor a2 mRNAs (39,40). However, if the introns in the a2 gene are transcribed in their entirety, as has been <br><br> demonstrated for the introns in ovalbumin and globin (41,42), much larger precursor a2 mRNAs may exist than those reported to date. For if the re<br><br> mainder of the a2 collagen gene has a structural organization comparable to the 60% 3' portion that has been analyzed, it is probable that the a2 <br><br> collagen gene in its entirety is dispersed throughout a 30 kb DNA segment. <br><br> ACKNOWLEDGMENTS <br><br> We thank Dr. John Dahlberg and Frances Loebenstein, Laboratory of Cellular and Molecular Biology, National Cancer Institute, for the generous use of a <br><br> Siemens Elmiskop 101 electron microscope; Dr. Ursula Heine and Benjamin Elliott, Laboratory of Viral Carcinogenesis, National Cancer Institute, for the use of a Hewlett-Packard calculator equipped with a digitizer and a Denton vacuum <br><br> evaporator, and Drs. Victor Ferrans and Oichi Kawanami, Laboratory of Pathology, <br><br> National Heart, Lung, and Blood Institute, for expert assistance with photography. We are also grateful to Margery Sullivan, Laboratory of Molecular Genetics, <br><br> National Institute of <span class="yellow">Child</span> Health and <span class="yellow">Human</span> Development, for helpful discussions and to Dr. Helga Boedtker, Department of Chemistry, Harvard University, for making available the recombinant plasmid pCg45. <br><br> References <br><br> 1.   Prockop, D. J., Kivirikko, k. I., Tuderman, L., and Guzman, N. A. <br><br> (1979) New Engl. J. Med. 301, 13-23 and 77-85. <br><br> 2.   Boyd, C. D., Tolstoshev, P., Schafer, M. P., Trapnell, B. C., Coon, <br><br> H. C., Kretschmer, P. J., Nienhuis, A. W., and Crystal, R. G. (1980) J. Biol. Chem., in press. <br><br> 3.   Blin, N., and Stafford, D. W. (1976) Nucl. Acids Res. 3, 2303-2308. 4.   Kretschmer, P. J., Kaufman, R. E., Coon, H. C., Chen, M. JY., Geist <br><br> C. E., and Nienhuis, A. W. (1980) J. Biol. Chem., in press. <br><br> 5.   Maniatis, T., Hardison, R. C., Lacey, E., Laver, H., O'Connell, C., <br><br> Quon, D., Sim, G. K., and Efstratiadis, A. (1975) Cell 15, 687-701. <br><br> 6.   Blattner, F. R., Blechl, A. E., Denniston-Thompson, K., Faber, H. E., <br><br> Richards, J. E., Slightom, H. L., Tucker, P. W., and Smithies, 0. (1978) Science 202, 1279-1284. <br><br> 7.   Meyer, J., Neuwald, P. D., Lai, S., Maizel, J. V., Jr., and Westphal, <br><br> H. (1977) J. Virol. 21, 1010-1014. <br><br> 2251 <br><br> Nucleic Acids Research <br><br> 8.   Westphal, H., and Lai, S. P. (1977) J. Mol. Biol. 116, 525-548. 9.   Schafer, M. P., Rohrmann, G., Heine, U., and Beaudreau, G. S. <br><br> (1979) Virol. 95, 176-184. <br><br> 10. Davis, R. W., Simon, M. M., and Davison, N. (1971) in Methods in <br><br> Enzymology (Grossman, L. and Moldave, K., eds) Vol. 21, pp. 413428, Academic Press, New York. <br><br> 11.  Federal Register (1978) 43, 60108-60131. <br><br> 12. Goodman, H. M., Olson, M. V., Hall, B. D. (1977) Pro. Natl. Acad. <br><br> Sci. U.S.A. 74, 5453-5457. <br><br> 13. Valenzuela, P., Venegas, A., Weinberg, F., Bishop, R., Rutter, <br><br> W. J. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 190-194. <br><br> 14. Berget, S. M., Moore, C., and Sharp, P. A. (1977) Proc. Natl. Acad. <br><br> Sci. U.S.A. 74, 3171-3175. <br><br> 15. Chow, L. T., Gelinas, R. E., Broker, T., and Roberts, R. J. (1977) <br><br> Cell 12, 1-8. <br><br> 16. Berk, A. J., and Sharp, P. A. (1978) Proc. Natl. Acad. Sci. U.S.A. <br><br> 75, 1274-1278. <br><br> 17. Bratosin, S., Horowitz, M., Laub, 0., and Aloni, Y. (1978) <br><br> Cell 13, 783-790. <br><br> 18.  Sargent, T. D., Wu, J. R., Sala-Trepat, J. M., Wallace, R. B., <br><br> Reyes, A. A., and Bonner, J. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 3256-3260. <br><br> 19.  Perrin, F., Cochet, M., Gerlinger, P., Cami, B., LePennec, J. P., <br><br> and Chambon, P. (1979) Nucl. Acids Res. 6 2731-2748. <br><br> 20. Tsujimoto, Y., and Suzuki, Y. (1979) Cell 16 425-436. <br><br> 21. Jeffreys, A. J., and Flavell, R. A. (1977) Cell 12, 1097-1108. <br><br> 22. Tilghman, S. M., Tiemeier, D. C., Seidman, J. C., Matija Peterlin, B., <br><br> Sullivan, M., Maizel, J. V., and Leder, P. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 725-729. <br><br> 23. Fiddes, J. C., Seeburg, P. H., DeNoto, F. M., Hallewell, R. A., <br><br> Baxter, J. D., and Goodman, H. M. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 4294-4298. <br><br> 24.  Soreq, H., Harpold, M., Evans, R., Darnell, Jr., J. E., and <br><br> Bancroft, F. C. (1979) Nucl. Acids Res. 6, 2471-2482. <br><br> 25.  Tonegawa, S., Maxam, A. M., Tizard, R., Bernard, 0., and Gilbert, W. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1485-1489. <br><br> 26. Nguyen-Huu, M. C., Stratmann, M., Groner, B., Wurtz, T., Land, M., <br><br> Glesecke, K., Sippel, A. F., and Schutz, G. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 76-80. <br><br> 27. Breathnach, R., Mandel, J. L., and Chambon, P. (1977) Nature 270, <br><br> 314-319. <br><br> 28. Dugaiczyk, A., Woo, S. L. C., Lai, E. C., Mace, M. L., Jr., <br><br> McReynolds, L., and O'Malley, B. W. (1978) Nature 274, 328-333. 29. Weinstock, R., Sweet, R., Weiss, M., Cedar, M., and Axel, R. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1299-1303. <br><br> 30.  Catterall, J. F., Stein, J. P., Lai, E. C., Woo, S. L. C., <br><br> Dugaiczyk, Mace, M. L., Means, A. R., and O'Malley, B. W. (1979) Nature 278, 323-327. <br><br> 31. White, R. L., and Hogness, D. S. (1977) Cell 10, 177-192. <br><br> 32.  Glover, D. M., and Hogness, D. S. (1977) Cell 16, 167-176. 33. Wellauer, P. K., and David, I. B. (1977) Cell 10, 193-212. <br><br> 34. Pellegrini, M., Manning, J., and Davidson, N. (1977) Cell 10, <br><br> 213-214. <br><br> 35. Wahli, W., David, I. B., Wyler, T., Jaggi, R. B., Weber, R., and <br><br> Ryffel, G. V. (1979) Cell 16, 535-549. <br><br> 36. McReynolds, L., O'Malley, B. W., Nisbet, A. D., Fothergill, J. E., <br><br> 2252 <br><br> Nucleic Acids Research <br><br> Givol, D., Fields, S., Robertson, M., Brownlee, G. G. (1978) Nature 273, 723-728. <br><br> 37. O'Hare, K., Breathnach, R., Benoist, C., Chambon, P. (1979) <br><br> Nucl. Acids Res. 7, 321-334. <br><br> 38. Abelson, J. (1979) Ann. Rev. Biochem. 48, 1035-1069. <br><br> 39. Adams, S. L., Alwine, J. C., de Chrombrugghe, B., and Pastan, I. <br><br> (1979) J. Biol. Chem. 254, 4935-4938, <br><br> 40. Rave, N., Crkvenjakow, R., and Boedtker, H. (1979) Nucl. Acids Res. <br><br> 6, 3559-3567. <br><br> 41. Roop, D. R., Nordstrom, J. L., Tsai, S. Y., Tsai, M. J., and <br><br> O'Malley, B. W. (1978) Cell 15, 671-685. <br><br> 42. Tilghman, S. M., Curtis, P. J., Tiemeier, D. C., Leder, P., and <br><br> Weissmann, C. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1309-1313. 43.  Palmiter, R. D., Davidson, J. M., Gagnon, H., Rowe, D. W., and <br><br> Bornstein, P. (1979) J. Biol. Chem. 254, 1433-1436. <br><br> 44. Graves, P. N., Olsen, B. R., Feitzig, P. P., Monson, J. M. and <br><br> Prockop, D. J. (1979) Fed. Proc. 38, 620. <br><br> 2253 </body></html>