
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Arabidopsis thaliana</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA140510</h3>Enp1, a <span class="yellow">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="yellow">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="yellow">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="yellow">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="yellow">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="yellow">human</span> Enp1, expressed in <span class="yellow">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="yellow">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="yellow">human</span> homolog of <span class="yellow">yeast</span> Enp1 was PCR amplified from a <span class="yellow">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="yellow">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="yellow">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="yellow">donkey</span>-anti-<span class="yellow">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="yellow">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="yellow">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="yellow">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="yellow">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="yellow">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="yellow">yeast</span> Enp1 (19). In contrast, we identified a <span class="yellow">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="yellow">human</span> bystin (19) showed that they were encoded by the same gene on <span class="yellow">human</span> chromosome 6. The reported sequence for <span class="yellow">human</span> bystin is a fragment of the 437 amino acid <span class="yellow">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="yellow">human</span> bystin, as confirmed by available <span class="yellow">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="blue">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="yellow">Drosophila melanogaster</span> and <span class="yellow">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="yellow">human</span> Enp1 homolog was cloned and expressed in <span class="yellow">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="yellow">human</span> Enp1 homolog did not complement a <span class="yellow">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="yellow">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="yellow">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="yellow">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="yellow">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="yellow">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="yellow">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="yellow">human</span> homolog of Enp1 did not complement a <span class="yellow">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="yellow">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA1312363</h3>Gene expression studies in isolated mitochondria: <span class="yellow">Solanum tuberosum</span> rps10 is recognized by cognate <span class="yellow">potato</span> but not by the transcription, splicing and editing machinery of <span class="yellow">wheat</span> mitochondria
Abstract
The complex gene expression mechanisms that occur in plant mitochondria, such as RNA editing and splicing, are not yet well understood. RNA editing in higher plant mitochondria is a highly specific process which modifies mRNA sequences by C-to-U conversions. It has been suggested that in some cases this process is required for splicing. Here, we use an experimental model based on the introduction of DNA into isolated mitochondria by electroporation to study organellar gene expression events. Our aim was to compare processing and editing of <span class="yellow">potato</span> small ribosomal protein 10 gene (rps10) transcripts in heterologous (<span class="yellow">wheat</span> mitochondria) and homologous (<span class="yellow">potato</span> mitochondria) contexts. rps10 is a suitable model because it contains a group II intron, is absent in <span class="yellow">wheat</span> mitochondria but is actively expressed in <span class="yellow">potato</span> mitochondria, where transcripts are spliced and undergo five C-to-U editing events. For this purpose, conditions for electroporating isolated <span class="yellow">potato</span> mitochondria were established. rps10 was placed under the control of either <span class="yellow">potato</span> or <span class="yellow">wheat</span> cox2 promoters. We found that rps10 was only transcribed under the control of a cognate promoter. In <span class="yellow">wheat</span> mitochondria, rps10 transcripts were neither spliced nor edited while they are correctly processed in <span class="yellow">potato</span> mitochondria. Interestingly, a <span class="yellow">wheat</span> editing site grafted into rps10 was not recognized by <span class="yellow">wheat</span> mitochondria but was correctly edited in <span class="yellow">potato</span> mitochondria. Taken together, these results suggest that editing might occur only when the transcripts are engaged in processing and that they would not be available to editing factors outside of a putative RNA maturation machinery complex.<br><br>INTRODUCTION
Gene expression in plant mitochondria is a complex process involving multiple steps such as transcription, cis- and trans-splicing, RNA trimming and RNA editing on the way to translation (1). RNA editing has been found in a variety of organisms and occurs through different mechanisms such as insertion or deletion of nucleotides, or base conversions [for details see reference (2)]. In higher plant organelles, RNA editing is an important post-transcriptional event characterized by C-to-U changes via a deamination mechanism (3–5). In <span class="blue">Arabidopsis thaliana</span> 456 C-to-U editing events have been described (6). RNA editing occurs mostly in the coding regions which alter the identity of the encoded amino acid, but some editing events occur in highly structured regions of introns (7,8). Like other maturation processes, RNA editing is an essential post-transcriptional event in plant mitochondrial gene expression (9–11) required for the synthesis of functional proteins (12,13).
The introduction of foreign DNA into isolated mitochondria is a novel experimental model which has provided important information on RNA editing and RNA splicing enabling the use of a site-directed mutagenesis approach (14,15). Using a cognate cox2 chimeric gene construct, the cis-recognition elements required for plant mitochondria RNA editing have been determined (16,17). Using the same approach, Staudinger and Kempken (18) have reported that transcripts from <span class="yellow">A.thaliana</span> cox2, but not <span class="yellow">Sorghum bicolor</span> atp6, are edited when the genes are introduced into <span class="yellow">maize</span> mitochondria.
Interestingly, higher plant mitochondrial genomes differ in their gene contents due to an evolutionary information transfer from the organelle to the nucleus (19). Of particular interest is the situation of the small ribosomal protein 10 gene (rps10), a group II intron-bearing gene which is encoded in <span class="yellow">Solanum tuberosum</span> mitochondrial DNA but is absent from the <span class="yellow">wheat</span> mitochondrial genome (20). Higher plant mitochondria contain group II introns either in cis- or trans- configuration (1) which can be folded in a characteristic secondary structure (21). Intron removal in plant mitochondrial mRNAs is not well documented because such introns are unable to self-splice. Previously, we described that electroporated cox2 constructs were a good model for the study of the splicing process. Using this model, we found that editing and splicing of cox2 transcripts were not linked in <span class="yellow">wheat</span> mitochondria (15).
To challenge the ability of mitochondrial gene expression machinery to recognize genetic information which has been lost during evolution, we decided to introduce the <span class="yellow">S.tuberosum</span> non-cognate rps10 gene into <span class="yellow">wheat</span> mitochondria. Five C residues are changed to U in <span class="yellow">potato</span> mitochondria rps10 transcripts by editing. Two of them have been postulated as being necessary for acquisition of a proper secondary and/or tertiary structure for splicing. To test this hypothesis, it was necessary to set up the conditions for electroporation of foreign DNA into <span class="yellow">S.tuberosum</span> mitochondria. Here, we show that a <span class="yellow">potato</span> rps10 construct is transcribed when introduced into <span class="yellow">wheat</span> mitochondria, but transcripts are not recognized by the post-transcriptional processing machinery. In contrast, a rps10 construct is correctly expressed and processed in cognate <span class="yellow">potato</span> mitochondria. Moreover, we present evidence that transcript editing might be linked to overall RNA processing. This is the first report on DNA electroporation into <span class="yellow">potato</span> mitochondria.<br><br>MATERIALS AND METHODS
Plasmids
All plasmids used in this study are based on the previously described pCOXII vector (14). An NsiI restriction site was introduced at the initiation codon of the <span class="yellow">wheat</span> cox2 open reading frame (ORF). Then, NsiI was used in combination with a SpeI restriction site present in the original vector after the stop codon, to produce the chimeric vectors.
<span class="yellow">S.tuberosum</span> cox2 gene, including a 727 bp non-coding upstream region, was isolated by PCR from total DNA using the primer A designed from partial sequences reported by Löessl et al. (22), accession no. AF096321, containing the KpnI restriction sequence, and primer B derived from <span class="yellow">Triticum timopheevi</span> cox2 (AF336134). A fragment of 2888 bp was obtained and cloned on pGEM-T vector. The complete sequence of the <span class="yellow">S.tuberosum</span> cox2 was determined (accession no. DQ18064). To generate the plasmid pCOXIISt containing the <span class="yellow">S.tuberosum</span> cox2 gene, a KpnI/SpeI fragment containing the 727 bp upstream region and the complete coding region was used to replace the <span class="yellow">wheat</span> gene from pCOXII. As for pCOXII, a NsiI site was inserted at the ATG start codon and a 23 bp fragment was inserted at position −20. This 23 bp insertion provides a specific sequence allowing isolation of <span class="yellow">potato</span> cox2 transcripts originating from the introduced DNA by RT–PCR. The chimeric <span class="yellow">wheat</span>/<span class="yellow">potato</span> cox2 gene was constructed by replacing 727 bp KpnI/NsiI promoter sequence from pCOXIISt with the 880 bp <span class="yellow">wheat</span> upstream region from pCOXII.
The vectors pRPS10W and the pRPS10St derivative, containing <span class="yellow">wheat</span> and <span class="yellow">potato</span> cox2 promoters, respectively, were constructed by inserting the 1178 bp rps10 sequence containing two exons separated by a 777 bp intronic sequence [(23), accession no. X74826]. The coding region was isolated from total <span class="yellow">S.tuberosum</span> DNA by PCR using primers D1 and D2 containing the restriction sites NsiI and SpeI, respectively. The fragment NsiI/SpeI was purified and cloned into pCOXII and pCOXIISt, replacing the respective cox2 coding regions.
Since all vectors used here were based on pCOXII, they contain the downstream region from the <span class="yellow">wheat</span> cob gene (Ir-cob) (accession no. AF337547) (14). This sequence, combined with the 23 bp upstream insert sequence served to distinguish, using primers 1a and 1b, foreign from endogenous transcripts. All mutants were obtained using QuickChange® Site-Directed Mutagenesis kit (Stratagene). PCR product purifications were carried out with the Wizard® Clean-Up System (Promega).<br><br>PCR primers used
Oligonucleotides sequences are in 5′ to 3′ orientation. (1a), GCGGTGCAGTCATACAGATCTGC; (1b), TATCCAGATTTGGTACCAAAC; (2a), GCAGTCATACAGATCTGCCAAAG; (2b), AGATTTGGTACCAAACCCGGG; (A), TATAGGTACCTCTCAGGTGTCAAAGTGTGGATTT; (B), TATAACTAGTTTAAGCTTCCCCG; (D1), TATAATGCATAGACAAAGGAGAGCACTTA; (D2), TATAACTAGTTCAGGAAAGGGTCAACGCAA. Restriction sites are underlined.<br><br>Mutagenesis primers (only sense primers in 5′–3′ orientation are indicated)
Single C2, C3 and C2+C3 double mutant plasmids were constructed with primers (C2) AAGAAGTTCTTTTGGTTAAAACGCC and (C3) CGCCGTGCGACTTGGAGGACATAAG. NsiI-pCOXII: GGAAATCCAATGCATCTTCGTTCATT and NsiI-pCOXIISt: CCAAACCAAATGCATGTTCTAGAATG. Construct pCOXIISt containing a 23 nt insertion in the promoter region, was carried out into two consecutive insertions using primers: TGGGGGGAGCAGAGCAGTGCGGTGCAGTCACAAAGAATGAACCAAACC and GCAGTGCGGTGCAGTCATACAGATCTGCCAAAGAATGAACCAAACC. For constructs MA and MAB containing the <span class="yellow">wheat</span> cox2 C259 editing site, two consecutive insertions were carried out using primers: GGAAGATTGGATTACTATCGAAATTGCCCTGAATCA and TACTATCGAAATTATTCGGACCATGCCCTGAATCA. For ME6 and ME6b constructs, primers CCGCGAGGAATCAACTACTATCGAAATTATTGCCGGTGCTGAC and CAACTACTATCGAAATTATTCGGACCATATTGCCGGTGCTGAC were used. Inserted nucleotides are underlined and the modified residues are indicated in bold.<br><br>Mitochondria purification
<span class="yellow">S.tuberosum</span> cv. Rosevalt tubers and <span class="yellow">T.aestivum var Fortal</span> seeds were used. <span class="yellow">Potato</span> mitochondria were prepared from 2 kg of tubers in batches of 200 g with 200 ml of a homogenization buffer containing 0.4 M mannitol, 25 mM MOPS (pH 7.8), 1 mM EGTA, 8 mM cysteine and 1 mg/ml fatty acid-free BSA. Homogenization was carried out for 15 s in a Waring blendor at full speed. Homogenate was centrifuged in a Sorvall GSA rotor at 1500 g for 10 min at 4°C. Supernatant was centrifuged in a GSA rotor at 12 000 g for 15 min. The mitochondrial pellet was resuspended in 50 ml of homogenization buffer and centrifuged at 1500 g. The supernatant was centrifuged in a Sorvall SS-34 rotor at 15 000 g for 10 min, the pellet was resuspended in 12 ml of homogenization buffer and mitochondria were purified by centrifugation on a sucrose gradient essentially as described for <span class="yellow">wheat</span> embryo mitochondria (14).<br><br>Electroporation
Electrotransfer experiments were carried out with 1 mg of purified <span class="yellow">wheat</span> embryo or <span class="yellow">potato</span> tuber mitochondria in 50 µl of 0.33 M sucrose and 1 µg of recombinant plasmid as described (14). The electroporated mitochondria were incubated for 18 h at 25°C, then recovered by centrifugation in a Sigma N° 12024 rotor (Sigma 1K15 refrigerated centrifuge) at 15 000 g for 15 min at 4°C. In the case of <span class="yellow">potato</span> mitochondria the incubation mixture was supplemented with 1 mg/ml of fatty acid-free BSA. RNA was purified with 200 µl of TRIzol™ reagent (Gibco-BRL) according to the supplier's protocol.<br><br>DNAse I protection assay and DNA purification
After electroporation and centrifugation, the mitochondrial pellet was resuspended in 100 µl of buffer [10 mM Tris–HCl (pH 7.5), 2 mM magnesium acetate and 0.33 M sucrose] containing 60 U of DNase I (Gibco-BRL) and incubated for 1 h at room temperature. The DNase reaction was stopped by adding 4 µl of 0.5 M EDTA and then heating for 10 min at 65°C. Mitochondria were incubated with 100 µg of Proteinase K (Merck) for 4 h at 37°C. One microliter of 20% SDS was added to achieve mitochondrial lysis and the DNA was extracted with phenol/chloroform, precipitated with 0.1 vol of 3 M Sodium Acetate (pH 5.2), 3 vol of 100% ethanol and 100 ng of carrier <span class="yellow">yeast</span> tRNA and left overnight at −20°C. After centrifugation, the DNA pellet was resuspended in TE buffer [10 mM Tris–HCl (pH 8) and 1 mM EDTA].<br><br>RT–PCR
RNA (1 µg) was treated with 2 U of Amplification grade DNase I (Promega). cDNA synthesis was performed with 200 U of Superscript II RT using 100 ng of random hexamers. The PCR were performed with primers 1a and 1b using Advantage® 2 Polymerase Mix (Clontech) as follows: 95°C, 1 min; 5 cycles at 95°C for 30 s and 68°C for 1 min; 30 cycles at 95°C for 30 s, 58°C for 30 s and 68°C for 30 s, and finally 68°C for 1 min. Primers 2a and 2b were used for nested PCR from 1 µl of the first PCR. No RT–PCR amplification products were obtained with RNA from non-electroporated mitochondria.<br><br>DNA sequencing
Sequence analyses were performed directly on the RT–PCR product using an automatic DNA sequencing equipment with the BigDye® Terminator Cycle Sequencing Kit (Applied Biosystem).<br><br>
RESULTS
<span class="yellow">S.tuberosum</span> rps10 transcripts are not processed in <span class="yellow">wheat</span> mitochondria
The <span class="yellow">S.tuberosum</span> rps10 construct under control of a <span class="yellow">wheat</span> cox2 promoter (Figure 1A) was incorporated into purified <span class="yellow">wheat</span> mitochondria by electroporation as indicated in Materials and Methods. After electroporation and incubation in expression medium, mitochondrial RNA was extracted and analysed by RT–PCR. The primers used allowed detection of the transcripts generated by the constructs introduced and excluded any product from endogenous cox2 (or rps10 in the <span class="yellow">potato</span> system). Two bands of 1204 and 427 bp were expected from precursor and mature rps10 mRNAs, respectively. Only precursor rps10 molecules were detected (Figure 1B). As a control, a cognate cox2 construct (16) was used. In this case, the 2048 and 827 bp RT–PCR bands representing the precursor and spliced cox2 products, respectively were observed (Figure 1B). In all cases, the PCR bands actually represent transcription products since when PCR was performed without the RT step we observed no amplification products.
Previously, five C residues, two in exon 1, one in the intron and two in exon 2 have been reported to be changed to U by editing in <span class="yellow">potato</span> mitochondria (23). To determine if the non-cognate transcript could be recognized by the <span class="yellow">wheat</span> RNA editing machinery, the 1204 bp RT–PCR product was sequenced. While <span class="yellow">wheat</span> cox2 editing sites were correctly edited in control as expected (Figure 1C, only site C77 is shown), all five editable residues in <span class="yellow">potato</span> rps10 transcript remain unchanged. cox2 C77 editing sites are identical in <span class="yellow">potato</span> and <span class="yellow">wheat</span>.<br><br>Failure of rps10 transcript splicing in <span class="yellow">wheat</span> mitochondria is not linked to the absence of editing
It has been suggested that residues C2 and C3 participate in the secondary structure of the intron necessary for splicing (23). A possible explanation to the failure observed in precursor rps10 splicing could be that the absence of edition of C2 and C3 prevent the intron from organizing itself in a catalytically competent conformation. To test this hypothesis, single C2 and C3 mutants and a C2+C3 double mutant were constructed by changing the C residues to T. Neither single nor double mutants were able to undergo splicing (Figure 2).<br><br>Electroporation of <span class="yellow">S.tuberosum</span> mitochondria
Since two important post-transcriptional processes, RNA editing and splicing, were inoperative when <span class="yellow">S.tuberosum</span> rps10 was expressed in <span class="yellow">wheat</span> mitochondria, we decided to verify whether the negative results were inherent to the rps10 chimeric constructs or due to the lack of trans-recognition elements in <span class="yellow">wheat</span> mitochondria. For this purpose, it was necessary to set up an electroporation protocol adapted to <span class="yellow">S.tuberosum</span> mitochondria. Purified organelles were prepared from <span class="yellow">potato</span> tubers as indicated in Materials and Methods. Electric pulses in the range between 8 and 20 kV were performed. Internalization of exogenous DNA was measured by DNAse protection assays (14). <span class="yellow">Potato</span> mitochondria show a broad range response with a maximum around 13 kV (Figure 3). This voltage setting was therefore used for further experiments.<br><br><span class="yellow">S.tuberosum</span> mitochondria does not recognize the <span class="yellow">wheat</span> cox2 promoter
To ascertain the ability of electroporated mitochondria to perform expression of the exogenous gene construct, we used a plasmid containing the <span class="yellow">potato</span> cox2 gene controlled either by <span class="yellow">T.aestivum</span> or <span class="yellow">S.tuberosum</span> promoters. As shown in Figure 4, the construct containing the <span class="yellow">wheat</span> or <span class="yellow">potato</span> promoters was transcribed only in cognate mitochondria. In <span class="yellow">potato</span>, the mature product is barely detectable. In contrast to <span class="yellow">wheat</span> mitochondria, the 821 nt mature transcript was only detected when the electroporated <span class="yellow">potato</span> mitochondria were incubated in the presence of fatty acid-free BSA.<br><br>A cognate rps10 construct is correctly expressed, edited and processed in <span class="yellow">potato</span> mitochondria
The construct expressing <span class="yellow">potato</span> rps10 under the control of <span class="yellow">potato</span> cox2 promoter was introduced into <span class="yellow">S.tuberosum</span> isolated mitochondria. After incubation, the precursor and mature transcripts were amplified by RT–PCR (Figure 5A) and sequenced. As shown in Figure 5B, the four editing sites C1, C2, C4 and C5, described previously in endogenous transcripts, were found edited in mature mRNA. The presence of spliced molecules containing unedited residues (Figure 5B) indicates that rps10 transcripts could be spliced before editing. To confirm this observation, the PCR product was cloned and sequenced. One half of the individual clones were found edited at sites C1 and C2; 65% were edited at site C4 and 25% were edited at site C5.<br><br>A cognate editing site in a non-native context is not recognized by <span class="yellow">wheat</span> mitochondria but is recognized in heterologous mitochondria
To test whether <span class="yellow">wheat</span> mitochondria are able to edit a cognate site when placed in the context of a <span class="yellow">potato</span> transcript, we introduced the C259 −16/+6 region from <span class="yellow">wheat</span> cox2 into <span class="yellow">potato</span> rps10 exon 1 or intron (Figure 6A, construct MA and ME6, respectively). As reported previously, the 23 nt region forming the C259 editing site from <span class="yellow">wheat</span> cox2 was efficiently edited when grafted into a different context in a <span class="yellow">wheat</span> transcript (17). Wild-type rps10 (pRPS10W) and the MA and ME6 mutants were expressed in <span class="yellow">wheat</span> mitochondria but no splicing was observed (Figure 6B). Unexpectedly, <span class="yellow">wheat</span> mitochondria appear to be unable to edit the cognate C259 site when the −16/+6 region is located on a <span class="yellow">potato</span> rps10 precursor. The control CM construct, containing the C259 site grafted in a different context but in its own cox2 gene, was expressed, spliced and edited as expected [(17), Figure 6B and D]. To compare the editing status of RNAs at the same stage of processing, the editing status of the inserted C259 site in precursor cox2 transcripts is shown (Figure 6D). The analogous MAb and ME6b constructs, containing the <span class="yellow">S.tuberosum</span> cox2 promoter and the C259 region inserted into rps10 exon 1 and intron were used to electroporate <span class="yellow">potato</span> mitochondria. As shown in Figure 6C, MAb and ME6b were expressed and spliced at the same level as the control in the homologous context, indicating that insertion does not affect the splicing process. Interestingly, while C259 insertion was not recognized in <span class="yellow">wheat</span> mitochondria (Figure 6D), the <span class="yellow">potato</span> RNA editing machinery edited the inserted C259 region (Figure 6E). The C259 region inserted in the rps10 intron (construct ME6b) was also edited, although at a very low level, showing that editing may precede intron removal (not shown).<br><br>
DISCUSSION
Plant mitochondria undergo complex expression mechanisms which are poorly understood. Most studies are based on analysis of in vivo mature or intermediate gene products giving clues on possible mechanisms and suggesting pathways operating in gene expression. Direct tests using in vitro approaches have been fruitful (4,5,24), but are hampered by the difficulty of obtaining active mitochondrial extracts. Previously, we devised an experimental model based on electroporation of isolated mitochondria that allows us to test gene expression of wild-type and mutant genes. This approach has been very useful in elucidating the process of RNA maturation in plant mitochondria, in particular splicing and editing (14,15). The aim of this work was to analyse different gene expression events when a foreign gene was expressed in a heterologous context. rps10 was chosen precisely because this genetic information is lacking in <span class="yellow">wheat</span> mitochondria but is active in <span class="yellow">potato</span> mitochondria (20,23,25). Moreover, rps10 transcripts undergo five C-to-U editing events in <span class="yellow">potato</span> mitochondria and rps10 contains an intron, thus facilitating the analysis of post-transcriptional processing.
To best evaluate the expression of rps10 in the non-cognate mitochondria, it was necessary to set up electroporation conditions for introducing foreign DNA into <span class="yellow">S.tuberosum</span> mitochondria. <span class="yellow">Potato</span> tuber mitochondria were able to incorporate DNA essentially under the same conditions described for <span class="yellow">wheat</span> embryo organelles (14), except that the optimal voltage range was larger. A major difference to <span class="yellow">wheat</span> was that isolated <span class="yellow">potato</span> mitochondria were viable for 3–4 h as measured by oxygen consumption (data not shown). Thus, to observe transcript maturation after electroporation the incubation mixture needed to be supplemented with fatty acid-free BSA (see Figure 4). This behaviour probably reflects the uncoupling of oxidative phosphorylation by fatty acids during incubation of <span class="yellow">potato</span> mitochondria (26). In fact, previously we found that proper gene expression in isolated mitochondria requires a functional organelle able to generate ATP from ADP and succinate (14).
An interesting observation was that neither the <span class="yellow">wheat</span> nor the <span class="yellow">potato</span> cox2 promoters were recognized in a heterologous context (Figure 4A and B). This situation might by explained by the fact that <span class="yellow">potato</span> and <span class="yellow">wheat</span> promoter sequences have no recognizable homologous motifs. Plant mitochondria promoters are characterized by a conserved core CRTA sequence with differences in the extent and the composition of sequences around the consensus motif (27,28). While the transcription initiation site in <span class="yellow">wheat</span> mitochondria is located at position −170 (29), the <span class="yellow">potato</span> promoter has not yet been described. Transcription may be initiated at numerous sites suggesting a relaxed promoter recognition by the transcription machinery (28). However, the lack of crossed recognition of cox2 promoters in <span class="yellow">wheat</span> and <span class="yellow">potato</span> is not a general situation since the <span class="yellow">A.thaliana</span> cox2 gene is expressed and spliced when introduced into <span class="yellow">maize</span> mitochondria, indicating that the <span class="yellow">Arabidopsis</span> gene shares some signals with <span class="yellow">maize</span> cox2 promoter that are sufficient for transcription (18). The sites required for transcript initiation have been recently described in <span class="yellow">A.thaliana</span> mitochondrial genes (28). Three regions were described as important for transcription initiation. We found no such homologous sequences in the <span class="yellow">potato</span> cox2 upstream region indicating that the two dicot promoters do not have the same origin. This in turn may reflect the natural history of this particular mitochondrial gene evolving in its own context. It should be mentioned that the presence of a conserved sequence is not sufficient for expression since a region that acts as promoter in <span class="yellow">Arabidopsis</span>, <span class="yellow">potato</span> and Oenothera is inactive in vivo in <span class="yellow">pea</span> (30). Further studies will be required to understand the transcriptional events in plant mitochondria. Electroporation of foreign DNA into isolated mitochondria provides an interesting functional model, complementary to in vitro transcription assay, for answering these questions.
The <span class="yellow">potato</span> rps10 gene controlled by a <span class="yellow">T.aestivum</span> promoter (Figure 1A) was transcribed as a 1204 nt precursor in <span class="yellow">wheat</span> mitochondria, but no traces of mature RNA were observed. Moreover, the five C residues reported as RNA editing targets in vivo (23) remain unchanged, indicating that rps10 transcript was not recognized by the <span class="yellow">wheat</span> splicing and editing machinery. A control using the cognate cox2 construct demonstrates that electroporated organelles were competent for splicing and editing (Figure 1B and C).
Some editing events occur in highly structured domains of introns. Because in some cases, editing corrects A-C mispairing improving conformation of the intron, it has been proposed that the C-to-U change might be necessary for efficient splicing (8,9,23,31). Based on the canonical structure of group II mitochondrial introns (21), two editing sites, C2 located at the Intron Binding Site 2 (IBS2) and C3 located in the intron, nine residues downstream from the end of exon 1 in <span class="yellow">S.tuberosum</span> rps10 are of particular interest. It has been predicted that both edited residues participate in base-pair interactions in the putative secondary structure. Of particular interest is the site C3 located in intron domain I (23). This position may be crucial for splicing as inferred from mutants of a <span class="yellow">yeast</span> mitochondrial intron (32). We addressed this question by introducing rps10 mutant genes presenting C2, C3 or both positions in the edited form into mitochondria. As shown in Figure 2, transcription was as efficient as for the cognate cox2 construct but neither wild-type rps10 nor the C-U mutants underwent splicing. These results clearly demonstrate that C2 and C3 editing is not sufficient for splicing of rps10 precursor in <span class="yellow">wheat</span> mitochondria and lead us to conclude that splicing failure is probably linked to the lack of trans-recognition elements. One may speculate that these trans-acting factors, for instance nuclear-encoded maturases, were lost after rps10 was transferred to the nucleus in monocots (20,25). The C-to-U mutants were also tested in <span class="yellow">potato</span> mitochondria; no significant differences were detected in splicing efficiency when compared to the unedited construct (D. Choury, unpublished data). It should be noted that the lack of splicing in <span class="yellow">wheat</span> mitochondria is not a general feature of <span class="yellow">potato</span> introns, since the <span class="yellow">potato</span> cox2 intron is removed efficiently (Figure 4B). This is consistent with the hypothesis that <span class="yellow">potato</span> and <span class="yellow">wheat</span> mitochondria have similar trans-acting factors for cox2 intron removal.
A possible link between intron removal and editing has been proposed for splicing of nad1e and nad5III trans-introns where domain six may require editing to be structured in a catalytic competent secondary conformation (8). In <span class="yellow">rice</span>, failure of RNA maturation and editing has been correlated with cytoplasmic male sterile phenotype. In this model, the absence of the nuclear gene Rf-1 affects B-atp6 RNA cleavage and editing (33). In organello studies have shown that <span class="yellow">S.bicolor</span> atp6-1 gene was transcribed but not edited when introduced into <span class="yellow">maize</span> mitochondria (18). However, partially edited molecules were detected in <span class="yellow">sorghum</span>-<span class="yellow">maize</span> atp6 chimeric transcripts when they included the <span class="yellow">maize</span> atp6 5′-untranslated sequence, suggesting that the 5′ non-coding region provides a structural motif or binding site for a transcript-specific editing factor (34). Unfortunately, no data on atp6 processing was reported to determine whether this region is directly involved in editing or some other maturation event.
Since <span class="yellow">wheat</span> mitochondria do not have rps10 encoded in the mitochondrial genome (20), it was interesting to test whether <span class="yellow">wheat</span> mitochondria were able to recognize editing sites which had been lost during evolution. In other words, whether the mitochondrial trans-recognition elements are specific for each site or whether they are operating on a subset of editing sites. This question is important since to date the factors responsible for RNA editing in plant mitochondria remain unknown. Solving this issue may provide clues to uncover such factors. As described above, the rps10 transcript was not processed in <span class="yellow">wheat</span> but it was correctly spliced in cognate mitochondria (Figure 5A). Indeed, the mature transcript had identical exon1 and exon2 junction as found in endogenous rps10 mRNA (data not shown). More importantly, all editing sites, C1, C2, C4 and C5 were significantly converted to Us (Figure 5B). The fact that the four editing sites were found either edited or unedited in spliced rps10 mRNA is an indication that editing does not precede splicing as was previously found for cox2 in <span class="yellow">wheat</span> (15).
It may be argued that the absence of splicing precluded editing. This possibility can be discarded since <span class="yellow">potato</span> rps10 precursor mRNA was found to be edited in vivo (23). These data clearly show that splicing is not required for rps10 editing, similar to previous findings for cox2 mRNA and also in cox2 mutant derivatives unable to remove the intron (15). This led us to postulate that the inability of <span class="yellow">wheat</span> mitochondria to recognize rps10 editing sites is likely due to the fact that <span class="yellow">wheat</span> mitochondria have lost the editing trans-recognition elements which become dispensable after transfer of rps10 to the nucleus. To test this possibility, rps10 chimeric plasmids containing editing site C259 from <span class="yellow">wheat</span> cox2 inserted either in exon1 or the intron were constructed. Site C259 is formed by 23 nt corresponding to the −16/+6 sequence embedding the target C. Previously we found that this small region could be recognized by the RNA editing machinery when placed outside of its natural context (16,17). The <span class="yellow">wheat</span> C259 editing site in the chimeric construct was correctly edited by <span class="yellow">potato</span> mitochondria. This result is not unexpected since the corresponding region in endogenous <span class="yellow">potato</span> cox2 mRNA, which presents two differences at positions −3 (C instead of A) and −7 (G instead of A), is edited. Furthermore, these positions were not crucial for editing of C259 (17). Surprisingly, the C259 editing site grafted in rps10 was not recognized by the <span class="yellow">wheat</span> editing machinery. We cannot exclude the possibility that editing efficiency in precursor molecules is very low and so is undetectable when sequencing RT–PCR products representing a pool of transcripts. However, analysis of the same region in cox2 precursors indicates that this was not the case since significant C-to-U conversion was observed (Figure 6D). These observations lead us to postulate that editing is occurring only when the transcript is engaged in post-transcriptional processing, suggesting that rps10 transcripts are not available to editing factors independently of the RNA maturation machinery. The fact that the <span class="yellow">wheat</span> editing site C259 inserted in chimeric rps10 transcripts was not recognized by <span class="yellow">wheat</span> mitochondria is a strong argument for this hypothesis. One might speculate that in plant mitochondria, transcripts have to be engaged in a kind of multiprotein processing complex. A failure to be recognized at some early stage will lead to their accumulation as unmodified precursors.<br><br>
</body></html>