
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>bacteriophage M13</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1190194</h3>Structural organization and interactions of transmembrane domains in tetraspanin proteins
Abstract
Background
Proteins of the tetraspanin family contain four transmembrane domains (TM1-4) linked by two extracellular loops and a short intracellular loop, and have short intracellular N- and C-termini. While structure and function analysis of the larger extracellular loop has been performed, the organization and role of transmembrane domains have not been systematically assessed.<br><br>Results
Among 28 <span class="yellow">human</span> tetraspanin proteins, the TM1-3 sequences display a distinct heptad repeat motif (abcdefg)n. In TM1, position a is occupied by structurally conserved bulky residues and position d contains highly conserved Asn and Gly residues. In TM2, position a is occupied by conserved small residues (Gly/Ala/Thr), and position d has a conserved Gly and two bulky aliphatic residues. In TM3, three a positions of the heptad repeat are filled by two leucines and a glutamate/glutamine residue, and two d positions are occupied by either Phe/Tyr or Val/Ile/Leu residues. No heptad motif is apparent in TM4 sequences. Mutations of conserved glycines in <span class="yellow">human</span> CD9 (Gly25 and Gly32 in TM1; Gly67 and Gly74 in TM2) caused aggregation of mutant proteins inside the cell. Modeling of the TM1-TM2 interface in CD9, using a novel algorithm, predicts tight packing of conserved bulky residues against conserved Gly residues along the two helices. The homodimeric interface of CD9 was mapped, by disulfide cross-linking of single-cysteine mutants, to the vicinity of residues Leu14 and Phe17 in TM1 (positions g and c) and Gly77, Gly80 and Ala81 in TM2 (positions d, g and a, respectively). Mutations of a and d residues in both TM1 and TM2 (Gly25, Gly32, Gly67 and Gly74), involved in intramolecular TM1-TM2 interaction, also strongly diminished intermolecular interaction, as assessed by cross-linking of Cys80.<br><br>Conclusion
Our results suggest that tetraspanin intra- and intermolecular interactions are mediated by conserved residues in adjacent, but distinct regions of TM1 and TM2. A key structural element that defines TM1-TM2 interaction in tetraspanins is the specific packing of bulky residues against small residues.<br><br><br><br>Background
Tetraspanins constitute a large family of integral membrane proteins, characteristically containing 4, 6 or 8 conserved cysteine residues in the large extracellular loop (including the CCG and PxxCC motifs), which form disulfide bonds, and several conserved polar residues in the intracellular loop and transmembrane regions [1,2]. There are 32 putative tetraspanin family members in mammals, 37 in <span class="yellow">Drosophila melanogaster</span> and 20 in <span class="yellow">Caenorhabditis elegans</span>. Tetraspanins play diverse roles in cell adhesion, migration and fusion processes, cellular activation and signaling (reviewed in refs. [2-4]). Mammalian tetraspanins such as CD9, CD63, CD81, CD82, CD151, rds/peripherin, and uroplakins Ia and Ib have been most extensively studied, with <span class="yellow">mouse</span> knock-out models available for CD9 [5-7], CD81 [8,9], CD151 [10] and a few others. However, the majority of tetraspanins are characterized very little, if at all, at genetic, biochemical or structural levels.
The large extracellular loop (LEL) of tetraspanins has received most attention, since it contains functionally important sites. Sequence QRD (194–196) in CD151 is important for association with integrins, which has functional consequences for integrin-dependent cell spreading and multicellular cable formation [11]. A site in the LEL of CD9, SFQ (residues 173–175), is essential for CD9 function in sperm-egg fusion [12]. The crystal structure of tetraspanin CD81 LEL revealed five α-helixes, A-E [13]. Helices A, B and E form a relatively conserved region in tetraspanins, whereas the region between helices B and E is the most variable [14]. Interestingly, the variable region contains most of the functionally important sites involved in tetraspanin protein-protein interactions.
A remarkable biochemical property of tetraspanin molecules is their ability to associate with a large number of other transmembrane proteins, including integrins, membrane-associated growth factors and receptors, MHC class II molecules, Ig superfamily proteins, and each other [2,3,15]. Several of these lateral associations of tetraspanins are detected in "mild" detergents (Brij series, CHAPS), but are disrupted by "strong" detergents such as Triton X-100 or SDS. Multiprotein complexes of tetraspanins and associated molecules, also called the "tetraspanin web" [16], may represent a distinct tetraspanin-enriched membrane microdomain [17,18]. The formation of this microdomain is influenced by palmitoylation of several conserved juxtamembrane cysteine residues in tetraspanins [19-21].
The transmembrane domains, encompassing nearly half of a tetraspanin protein, are the most conserved part of the molecule (Stipp et al. [1] and this study). However, very little functional information is available on these domains. The differential detergent sensitivity of tetraspanin-tetraspanin associations suggests that hydrophobic interactions between TM helices may play a role. Indeed, when the large extracellular loop (LEL) of CD151 is deleted, the molecule is still able to associate with other tetraspanins [22]. Thus, TM domains are strong candidates for mediating tetraspanin-tetraspanin interactions.
The importance of TM domain interactions in intramolecular organization was demonstrated in a study showing that CD82 fragment TM2-4, lacking TM1, was retained in the endoplasmic reticulum, but was transported to the cell surface upon co-expression of TM1 [23]. This in vivo reconstitution experiment demonstrated a strong interaction between TM1 and the rest of the molecule. Expression of a truncated CD9 molecule (TM3-LEL-TM4) results in intracellular accumulation of the protein and significant misfolding of the LEL, as judged by inappropriate disulfide formation and diminished antibody reactivity (our unpublished data). Similarly, a CD9 epitope in the LEL is lost in molecules lacking either TM2+TM3 or just TM4 [24]. Thus, TM domain interactions and packing are crucial for proper folding, stability and transport of tetraspanin molecules.
In a previous study, we showed that covalent cross-linking of membrane-proximal cysteine residues can be used as a tool for detection of tetraspanin-tetraspanin associations [25]. Inhibition of cysteine palmitoylation by 2-bromopalmitate (2-BP) made cysteines available for cross-linking and enabled demonstration of specific tetraspanin homodimerization and low levels of heterodimerization. We concluded that tetraspanin homodimers, formed in the Golgi, may be a fundamental structural unit within tetraspanin microdomains.
In this study, we carried out detailed sequence analysis of <span class="yellow">human</span> tetraspanin TM domains. We show that a heptad repeat containing conserved glycine, asparagine and large hydrophobic residues occurs in TM1 and TM2 domains, and predict tight intramolecular association of these two domains by packing of the large residues against the small residues. Moreover, by using cysteine cross-linking we map a dimerization interface in the <span class="yellow">human</span> CD9 protein, and show that conserved heptad motif glycine residues are also important for intermolecular CD9 associations.<br><br>Results
Sequence analysis of tetraspanin transmembrane domains: presence of the heptad repeat motif
We focused our attention on 28 <span class="yellow">human</span> tetraspanins identified from the SWISS-PROT and GenBank databases. All tetraspanins have in common four hydrophobic stretches (TM domains) of 20–25 residues, and contain highly conserved residues in the second extracellular loop, in particular the Cys-Cys-Gly (CCG) motif. Detailed analysis of the large extracellular loop sequences [14], and dendrograms based on full-length alignment can be found in earlier studies [26,27]. The length of each transmembrane domain was established based on previous sequence analysis of tetraspanin sequences [27,28], and on annotations to the database entries. Manual adjustments based on sequence homology and hydrophobicity profiles were done to fully delineate the TM domains. The resulting lengths of TM domains were: TM1 – 23 residues; TM2 – 21 residues; TM3 – 25 residues; TM4 – 25 residues. Two more residues could be added onto the N-terminal part of TM2; however, relatively small sequence conservation of these residues among tetraspanins and occurrence of polar/charged side chains in some tetraspanins precluded us from doing so for the global alignment.
Figures 1 and 2 show a multiple sequence alignment of four TM domains of 28 <span class="yellow">human</span> tetraspanins. For each position within the domains, consensus residues were determined and classified (with individual color code) in 4 categories: 1) large hydrophobic residues (including Val, Met, Leu, Ile, Phe, Tyr, Trp), 2) small residues (Gly, Ala, Ser and Thr), 3) Cys, and 4) Asn. When more than two types of residues occupied a given position in a TM, a dual-color pattern that reflected the prevalence of the particular residue type was used (Figure 1). Cysteine residues were shown separately due to their importance as palmitoylation target sites. The highly conserved asparagine residue in TM1 was considered separately. No proline residues are found in TM domains 1–3 of <span class="yellow">human</span> tetraspanins.
An inspection of the multiple sequence alignment reveals a repeating heptad amino acid pattern, (abcdefg)n, in TM1, 2 and 3 (Figure 1, 2). Heptad repeats promote helical coiled coil interactions in multiple soluble and membrane-spanning proteins [29-31]. In the heptad repeat, hydrophobic residues in positions a and d are of special importance, as they directly mediate interhelical contacts by creating a tight knobs-into-holes packing in the coiled coil structure [32]. For instance, in the leucine zipper of the <span class="yellow">yeast</span> transcription factor GCN4, positions a and d contain Val and Leu residues, respectively, with an Asn residue in a single a position forming a hydrogen bond across the GCN4 dimer interface [33].
In TM1 of tetraspanins, highly conserved Asn, Gly and Gly residues (numbers 18, 25 and 32 in the CD9 sequence) appear at d positions of the heptad repeats, and residues 14, 21 and 28 are at a positions (Figure 1). In TM2, residues 67, 74 and 81 (consensus Gly, Gly and Ala, respectively) occupy a positions, whereas residues 63, 70 and 77 are at d positions. Another highly conserved glycine, Gly80, occupies the 3rd g position in TM2. In TM3, the conserved pattern consists of two leucine residues (Leu89 and Leu96) and a glutamate/glutamine residue (Glu/Gln103) in a positions (Figure 2). Two d positions are also conserved – Phe/Tyr92 and Ile/Val/Leu99. TM4 lacks a conserved heptad pattern and has only a single conserved position, Glu/Gln209 (with four exceptions). These features of TM1-4 of tetraspanins are displayed on helical wheel diagrams (Figure 3).<br><br>Analysis of TM1 sequences
The conserved Asn-Gly-Gly motif, occupying designated d positions of the heptad repeat, is the most prominent structural feature of TM1. We also compared sequences of CD9 orthologs from 10 different organisms (the most available for any tetraspanin) to gain further insight into conservation and variability of the TM1 sequence. As shown in Figure 4, positions a, d and g in TM1 are among the most conserved (0, 1 and 1 substitution, respectively), while interspecies variability tends to occur in other positions: b (5 substitutions), c (4 substitutions), e (4 substitutions) and f (4 substitutions). Thus, the positions typically involved in coiled coil interactions (a and d) are the most conserved.
When residues of TM1 are plotted as a helical wheel, additional structural features are revealed (Figure 3). There are highly conserved aliphatic and aromatic residues in the first three a positions of the heptad motif (Phe15, Trp22 and Leu29 in CD9), as well as in g positions (Leu14, Phe21, Val28 in CD9). The "ridges" formed by these bulky residues are flanking the "groove"-forming Gly residues of the Asn-Gly-Gly position d motif. In contrast, b, c, e and f positions show an overall higher variability among tetraspanins, as also seen in the comparison of CD9 orthologs described above.<br><br>Analysis of TM2 sequences
A landmark feature of TM2 in tetraspanins is the presence of highly conserved glycine residues (Gly67, 74, 77 and 80 in CD9, Figure 1). Other substitutions at these positions are almost exclusively small residues, such as Ala or Ser. In addition, Ala, Ser or Thr occupy position 81. This residue, together with Gly67 and Gly74, forms face a of the helix. Residue Gly77 (position d) is preceded by conserved, chiefly large hydrophobic residues on the same helical face (Leu63 and Met70 in CD9). Extremely conserved Gly80 falls into heptad position g (Figure 3). Among CD9 orthologs, heptad positions a and d are absolutely conserved, whereas other positions have the following number of substitutions: b – 3; c – 2; e – 1; f – 3; g – 1 (Figure 4). Two of the f position residues in TM2 (65 and 79) also show higher variability among different tetraspanins (Figures 1, 3). Cysteine residues are frequently found near the cytoplasmic end of TM2 helix at positions 78 and 79; these cysteines are likely to be palmitoylated.<br><br>Analysis of TM3 and TM4 sequences
The TM3 domain provides another example of the heptad repeat pattern. Position a is occupied by two highly conserved leucine and a glutamate/glutamine residue (Leu89, Leu96 and Glu/Gln103 in CD9). Furthermore, two d positions are conserved – Phe/Tyr92 (aromatic residue) and Ile/Val99 (β-branched aliphatic residue; Figures 2, 3). In addition, residue 100 in position e is generally Phe or Leu. Among CD9 orthologs, position a has 1 substitution, positions b, c and f each have 6, positions d and e each have 2, and g has 4. Thus, as for TM1 and TM2, positions a and d are among the most conserved, but overall TM3 has more variable positions than TM1 or TM2 (Figure 3). Less than half of TM3 sequences contain cysteine residues, and those tend to occur at the internal positions of the helix (Figure 2).
TM4 shows less conservation among various tetraspanin family members than the other TM domains (Figures 2, 3). The only highly conserved feature is the glutamate/glutamine residue in position 209. In addition, one or two cysteine residues can be found at the C-terminal end of TM4 in some tetraspanins (e.g. CD9, CD81, CD151), and many sequences contain additional polar residues (Arg, Lys, His, Asn, Gln). No conserved heptad motif was identified in TM4, as also confirmed by analysis of substitutions in CD9 orthologs (data not shown).<br><br>Mutational analysis of conserved glycine residues in TM1 and TM2
The conserved nature of the Asn and Gly residues in TM1 and TM2 prompted an analysis of their functional role. To this end, we have probed whether mutations of these residues destabilize the protein molecule. We expressed a construct of the first and second TMs of CD9, connected by the small extracellular loop, and tagged with a C-terminal green fluorescent protein (TM(1+2)-GFP molecule). In <span class="yellow">human</span> rhabdomyosarcoma RD cells, the wild-type fusion protein localized mostly in a reticular, intracellular pattern, without forming any large aggregates (Figure 5, panel A). Remarkably, when double mutants Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu were expressed, the protein formed distinct large aggregates in a high proportion of cells (Figure 5, panels C and E). In contrast, double mutant Gly77Leu + Gly80Leu did not form such aggregates (Figure 5, panel G). Results with respective single mutants were similar to that with double mutants, with the aggregation being somewhat more pronounced for Leu substitutions of Gly67 and Gly74 compared to Gly25 and Gly32 mutations. No aggregation was observed for Asn18Ser and Asn18Tyr mutants (data not shown). Also, nearly identical results were obtained with <span class="yellow">human</span> HT1080 cells (data not shown).
We interpret these results as an indication that aggregating mutants are destabilized or misfolded while non-aggregating mutants retain the wild-type conformation. Intriguingly, mutations to the conserved GG7 motifs caused protein aggregation while the mutation of other glycines had no detectable effect. These results also suggest that wild-type GFP, which has weak tendency to self-associate, could enhance non-specific interactions of destabilized mutant TM(1+2) CD9 moieties, leading to their aggregation. Consistent with this hypothesis, the aggregation of Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu double mutants was suppressed when monomeric GFP molecule, Leu221Lys [34] was used (Figure 5, panels D and F). The use of monomeric GFP did not affect intercellular localization of wild-type CD9 TM(1+2) (Figure 5, panel B), or a Gly77Leu + Gly80Leu double mutant (Figure 5, panel H).
In summary, Leu substitutions of Gly residues that are part of the Asn-Gly-Gly (NGG7) motif in TM1, or Gly-Gly-Ala (GGA7) motif in TM2, resulted in destabilization and aggregation of GFP-fused TM(1+2) proteins, whereas substitutions of Gly77 or Gly80, which are not part of these motifs (Figure 3), failed to show such aggregation.<br><br>Prediction and modelling of interaction between TM1 and TM2
Consecutive helices in polytopic membrane proteins frequently interact [35]. Sequence analysis of TM1 and TM2 helices of tetraspanins reveals a remarkable complementarity in the distribution of large and small residues at heptad positions a and d along the helical axis (Figure 3), suggesting that these residues may interact. To further elucidate the potential for TM1-TM2 interaction, the putative interface was modeled using a novel algorithm that considers mutational data during each step of a Monte Carlo simulated annealing cycle (see Methods for details). Specifically, Gly25Leu, Gly32Leu, Gly67Leu and Gly74Leu were scored as disruptive mutations, while Asn18Ser, Gly77Leu and Gly80Leu were scored as silent mutations, based on their effects on protein stability (Figure 5 and data not shown).
The resulting model predicts left-handed crossing of TM1 and TM2 helices at an angle of +28°. The key element of the structure is the apposition of bulky and small heptad position a and d residues, as follows: Gly32-Leu63; Gly67-Leu29; Gly25-Met70; Gly74-Trp22; Asn18-Gly77; Ala81-Phe15 (Figure 6). Our model predicts that each of these residue pairs are in van der Waals contact. Additionally, two potential H-bonds are predicted in this model, indicating close packing: Gly67 Cα to Gly25 carbonyl oxygen, and Trp22 Cα to Met70 carbonyl oxygen. The packing is tighter in the ectodomain-proximal portion of the helices (Figure 6, panel B), as determined by Cα-Cα distances between interacting residue pairs.
The key elements of the model are corroborated by the presence of apparently complementary substitutions in TM1 and TM2 sequences of different tetraspanins (Figure 1, boxed residues). For example, Gly74 is predicted to interact with Trp22. In 8 of the 10 tetraspanins that contain a substitution for Gly74, a compensatory substitution occurs at the Trp22 position (Figure 1). Thus, a larger non-glycine side chain at position 74 may necessitate a less bulky non-Trp side chain in position 22. Likewise, the presence of a Cβ at position 25, typically occupied by glycine, necessitates a non-β-branched amino acid at position 70, which is occupied by a β-branched residue in nearly half of all cases. Indeed, we find that in each of 5 cases in which position 25 contains a Cβ, a leucine residue occurs in position 70. This analysis is consistent with our molecular model that suggests Leu70 will pack most favorably against a Cβ at position 25 than a β-branched residue or a methionine.<br><br>Role of TM1 and TM2 heptad motif residues in CD9 dimerization
To probe CD9 dimerization, we used a cysteine-mediated cross-linking approach. We established previously a simple and efficient method for cysteine-mediated cross-linking [25]. After cells are pre-treated with 2-BP for 16–24 hours to expose normally palmitoylated cysteines, the cysteines can be cross-linked using any of the following methods: a) Spontaneous oxidation in Brij97 lysates (a condition that preserves tetraspanin-tetraspanin associations), b) In situ cross-linking, by pre-lysis oxidation of cells with Cu2+-phenanthroline (CuP) to promote disulfide bond formation. c) In situ cross-linking with thiol-reactive cross-linking agents of defined length (e.g. DTME, BMB). The first two approaches produce in essence "zero-length" disulfides, indicative of close proximity of target cysteines and presumably high specificity of interaction. In contrast, chemical cross-linkers with 6–20 Å spacer arm may cross-link with higher efficiency, but not necessarily higher specificity. However, they provide advantages such as variable membrane permeability, and potential linkage cleavability. For tetraspanins such as CD9, membrane-permeable cross-linker DTME (13.3 Å-long, reducible) provides highly specific and efficient cross-linking [25]. Here we have used a cysteine cross-linking strategy, in combination with cysteine-scanning mutagenesis, to map the residues from TM1 and TM2 contributing to the CD9 dimerization interface.
For subsequent cross-linking experiments using CD9 TM(1+2)-GFP protein, the non-dimerizing form of GFP was used. This avoids potential GFP-dependent dimerization and aggregation that can be observed with wild-type GFP, especially when fusions with transmembrane proteins are studied [36]. Importantly, the Leu221Lys mutation in GFP prevented aggregation of mutant forms of CD9 TM(1+2), which was observed with wild-type GFP fusion (Figure 5). The TM(1+2) fragment of CD9 contains three native cysteines – Cys9, Cys78 and Cys79. Single-cysteine mutants of TM(1+2) were constructed, in which a cysteine was placed at various faces of TM1 or TM2 while all of the wild-type cysteines were simultaneously replaced by serines. The mutant proteins were transiently expressed in RD cells (having little endogenous CD9), which were then treated for 16–18 hours with 2-BP. To achieve maximal specificity in cross-linking we used a "zero-length" agent, CuP.
First, single-cysteine replacements were constructed for residues Leu14, Phe15, Gly16, Phe17 and Asn18, covering just over one complete helical turn at the beginning of TM1. While residue Asn18 is highly conserved, positions 14, 15 and 17 are occupied by bulky hydrophobic residues in most tetraspanins, whereas position 16 shows less conservation (Figures 1, 4). All of the single-cysteine mutants showed diffused pattern of protein localization, without any signs of aggregation. As shown in Figure 7A, the highest level of intermolecular cross-linking was observed for Leu14Cys and Phe17Cys mutants, a lower level for Phe15Cys and Gly16Cys mutants, and very little cross-linking for Asn18Cys substitution. These results indicate that: a) the first two transmembrane domains of CD9 alone can mediate its dimerization, and b) the g and c residues of TM1 (e.g. Leu14 and Phe17, Figure 3) are likely to be part of the intermolecular interface.
Similarly, single-cysteine substitutions were made for residues Gly77, Gly80 and Ala81 in TM2; in addition, proteins carrying a single wild-type cysteine, Cys9, Cys78 or Cys79, were tested. No protein aggregation was observed for any of these single-cysteine mutants. As shown in Figure 7B, the relatively low level of intermolecular cross-linking of wild-type CD9 TM(1+2)-GFP protein was enhanced dramatically in single-cysteine TM2 mutants Gly80Cys and Ala81Cys. The Gly77Cys mutant also had an elevated level of cross-linking. In contrast, any of the three native cysteines (9, 78 and 79) produced level of cross-linking not much greater than the wild-type TM(1+2) protein. Similar results were obtained with cysteine-reactive cross-linker BMB (data not shown). Likewise, comparable results were obtained with single-cysteine mutants of untagged, full-length CD9, using CuP (Figure 7C) as well as DTME cross-linker (data not shown).
These cross-linking results for TM1 and TM2 are consistent with our model that places residues Leu14, Phe17 and Gly80 on the same side of the TM1-TM2 pair (Figure 6, panel C). The strong cross-linking with Leu14Cys, Phe17Cys and Gly80Cys places the intermolecular interface toward the c and g phases of the TM1 helix, and the g phase of the TM2 helix, away from its e and f faces containing wild-type cysteines 78 and 79.
Critical residues at the TM1-TM2 interface also affect dimerization indirectly. To assess specific CD9 dimerization, we used a Gly80Cys substitution at the intermolecular interface for cross-linking. As shown in Figure 8A, single replacements of conserved heptad residues in positions 18, 25, 32, 67 and 74 (Asn18Ser, Gly25/32/67/74→Leu) strongly decreased the cross-linking mediated by Cys80. The effect was most pronounced for mutations of residues, Gly32 and Gly67, located in the tightly packed extracellular end of TM helices (Figure 6). In contrast, mutations of residues closer to the cytoplasmic end of TM2 (Gly74 and especially Ala81) had only modest to very little effect on cross-linking.
Relatively low efficiency of intermolecular cross-linking via native residues Cys9, 78, and 79 (Figures 7B,C) correlates well with the predicted location of Cys78 and 79 away from the dimeric interface (Figure 3), and suggests that the extramembrane N-terminal part of CD9 (residues 1–13) does not self-associate. We next examined whether mutations of conserved Asn and Gly residues in TM1 and TM2 decreased low-level background cross-linking via native cysteines. As expected, these mutations had virtually no effect on dimer formation of CD9 TM(1+2)-GFP (Figure 8B). The level of covalent dimer formed was not diminished for triple Asn18Ser + Gly25Leu + Gly32Leu and double Gly67Leu + Gly74Leu mutants, compared to wild-type TM(1+2) CD9 molecule. Similarly, the same triple and double mutations in the context of full-length CD9-GFP protein (with six cysteines) produced wild-type levels of cross-linking (Figure 8C). We interpret these findings as evidence for at least two types of associations between CD9 molecules: primary, involving residues 14, 17 and 80, and dependent on integrity of conserved heptad residues in TM1 and TM2, and less efficient secondary interactions, probably representing random collision events, and independent of the heptad residues (see Discussion for more details).<br><br>TM3 and TM4 cysteine residues in CD9 dimerization
After identifying the roles of conserved TM1 and TM2 residues in CD9 dimerization, we next probed whether residues proximal to TM domains 3 and 4 are also involved. To this end, disulfide cross-linking of full-length CD9 molecules containing 3 C-terminal cysteines (87, just before TM3; 218 and 219 in TM4) or 3 N-terminal cysteines (9, 78 and 79) was compared (Figure 9). We found that the C-terminal cysteines were only slightly better than N-terminal cysteines with respect to detection of CD9 dimers. However, markedly more trimers and tetramers were detected using C-terminal cysteines. Thus, residues 87, 218 and 219 at TM3 and TM4 in CD9 can together form contacts across the dimeric interface and also additional contacts with other neighboring CD9 molecules.<br><br>
Discussion
Here we provide the first detailed analysis of tetraspanin protein transmembrane domains. First, we show 1) the presence of a heptad repeat motif in TM1 and TM2, containing highly conserved Asn and Gly residues, 2) a leucine and glutamate/glutamine-containing heptad motif in TM3, and 3) high variability and absence of heptad repeats in TM4 sequences. Second, we provide evidence for a specific, intramolecular interaction between TM1 and TM2 domains, in which bulky hydrophobic residues pack against GG7 motif, and present a molecular model for this interaction. Third, experimental mapping of the CD9 dimerization interface firmly establishes an additional role for conserved TM1 and TM2 residues in dimeric intermolecular interactions. Fourth, preliminary evidence is provided to suggest that TM3 and TM4 domains contribute to expansion of CD9 dimers into higher order multimers.
Conserved residues in TM1 and TM2 of tetraspanins: role in intramolecular packing
We hypothesized that the first two transmembrane domains of tetraspanins might interact with each other because: a) consecutive TM domains frequently associate in known protein 3D structures [35], and b) they both contain a series of highly conserved amino acids – several Gly residues and an Asn residue (Figure 1). Conserved Gly residues are a frequent theme in the organization of interacting transmembrane domains. Analysis of 3D helix packing in polytopic membrane proteins reveals that Gly residues tend to localize in buried positions, especially at the helix-helix interfaces and helix crossing points [37,38]. Due to the absence of a side chain, Gly provides a flat surface for packing of a side chain from another residue, without loss of side-chain entropy upon interaction. The most common Gly-containing motif is GxxxG [39,40]. In glycophorin A (GpA), the major glycoprotein in erythrocyte cell membranes, Gly79 and Gly83 are part of the LIxxGVxxGVxxT sequence that promotes homodimerization of parallel transmembrane α-helixes [41,42]. In the GpA dimerization motif, Gly residues allow for tight packing in the right-handed helical crossing [43]. There are also examples of left-handed helical crossing in the context of a GxxxG motif [44]. Other membrane proteins that use the GxxxG motif for homo- or heterodimerization include <span class="blue">bacteriophage M13</span> coat proteins [45], <span class="yellow">yeast</span> alpha factor receptor [46], integrin α IIb subunit [47], and ErbB1 receptor tyrosine kinase [48]. Other small residues, such as Ala and Ser, can substitute for Gly in this motif [49].
A protein motif in which Gly residues are separated by 6 other residues (GG7) is also common in transmembrane helices, especially in transporter/channel-like membrane proteins [50]. However, the structural features associated with this motif are not well known. In particular, it is unclear whether left-handed GG7 heptad repeat motif (as opposed to the "classic" right-handed GxxxG motif) can drive membrane helix association. In a recent work addressing this issue, Lear et al. [51] showed that a synthetic peptide containing Gly at heptad positions a and d could self-associate in vitro, likely in an antiparallel orientation. Heptad repeats containing conserved Gly residues occur in TM domains of α and β chains of MHC class II proteins, and mutations of the Gly residues disrupt the αβ heterodimer [52]. These examples demonstrate that Gly-based heptad motifs may be used for both intra- and intermolecular associations.
In this work, we identified a highly conserved GG7 motif in the first two tetraspanin TM domains. The GG7 sequence in tetraspanins is a part of a larger motif that also includes a conserved Asn residue in TM1 (NGG7) and an Ala/Ser/Thr residue in TM2 (GGA7). The seven-residue periodicity of these motifs strongly suggests their involvement in left-handed coiled coil packing reminiscent of the leucine zipper, rather than right-handed packing of the GpA-like GxxxG motif. For antiparallel helices, the left-handed crossing is in fact predominant over the right-handed in known TM domain structures [44].
In our model, heptad Gly residues in NGG7 and GGA7 sequences provide specific packing between antiparallel tetraspanin TM1 and TM2 helices by allowing tight van der Waals interactions with large hydrophobic residues (Figure 6). Highly efficient packing of bulky side chains against glycine residues is observed in known transmembrane protein 3D structures [38,53,54]. An example includes packing of helices M1 and M2 in potassium channel KcsA, where Val91 in M2 is paired with Gly43 in M1, and Leu36 in M1 contacts Ala98 and Gly99 in helix M2 [54,55]. In addition to facilitating helix-helix packing, Gly residues frequently provide additional CαH...O hydrogen bonds between two helices [44]. In our model, two Cα-backbone carbonyl H-bonds are predicted – between residues Gly27-Gly67, and Trp22-Met70.
Although polar and charged amino acid residues (such as Asn in the TM1 heptad motif) are infrequent in transmembrane domains, they are functionally important. Polar residues such as glutamine, glutamic acid, aspartic acid and asparagine can promote strong oligomerization of model membrane-associated helices [56-58]. Ruan et al. [59] used asparagine scanning mutagenesis to probe the interface of self-associating polyleucine helices by detecting their enhanced self-interaction in vitro and in the <span class="yellow">E. coli</span>-based ToxR assay. Thus, a hydrogen bond in an apolar environment can result in strong, though not necessarily specific, association of transmembrane helices. In fact, mutations to polar residues in transmembrane proteins are commonly associated with disease [60]. Because of this potential for non-specific interactions, polar residues tend to localize at buried positions in TM domains.
In our case, the conserved Asn18 residue in CD9 is predicted to be a part of the TM1-TM2 interface, though our model does not predict any electrostatic interaction between Asn18 and TM2 (Figure 6). Consistently, substitution such as Asn18Tyr (and Gly77Leu) in TM(1+2)-GFP protein was not destabilizing as analyzed by protein aggregation. Curiously, the full-length Asn18Ser CD9 migrated slightly slower on SDS-PAGE gel (data not shown), suggesting that Asn18 does play a role in maintaining conformation of the molecule. The Asn18Cys single-cysteine mutant shows very little intermolecular cross-linking (Figure 7A), supporting the proposed location of this residue at the intramolecular interface. It is tempting to speculate that the "pocket" between TM1 and TM2 lined by Asn18 and Gly77 might be important for accommodating palmitate moieties that target Cys78 and Cys79 residues, and/or important for access by the putative palmitoyl transferase to those residues. Understanding the exact role of these highly conserved Asn18 and Gly77 residues in tetraspanins awaits further investigation.
In summary, we identified conserved glycine residues of TM1 and TM2 of tetraspanins as key elements required for intramolecular packing. Mutations of these key residues (Gly25, Gly32, Gly67 and Gly74 in CD9) resulted in protein destabilization and aggregation. There is ample evidence in the literature for mutations in transmembrane proteins that lead to protein destabilization, misassembly and pathologic conditions [61]. Thus, we have identified conserved heptad Gly residues in TM1 and TM2 of tetraspanins as plausible targets of destabilizing mutations with potential functional consequences.<br><br>Intermolecular interactions in tetraspanins
Tetraspanin CD9 forms mostly homodimers, but also a low level of heterodimers with CD81 and CD151 [25]. Thus, mapping the dimerization interface is an important next step in structure-function analysis of tetraspanins. Disulfide-mediated cross-linking, often in combination with cysteine-scanning mutagenesis, is a common strategy to probe oligomerization or intersubunit interactions of transmembrane proteins such as histidine kinase EnvZ [62], M(3) muscarinic acetylcholine receptor [63], <span class="yellow">E. coli</span> lactose permease [64], synaptobrevin [65], integrins [66] and many others. In tetraspanins such as CD9, membrane-proximal cysteine residues are especially useful targets for disulfide trapping, as their linkage can be enhanced by pre-treating cells with 2-BP. While the ability of wild-type cysteines in CD9 to be cross-linked may indicate that they are close to the dimerization interface, more precise mapping was achieved here using cysteine-scanning mutagenesis.
Our data clearly identify regions, near the cytoplasmic face of TM1 and TM2, important for dimerization. Intermolecular zero-length cross-linking was highest when single cysteines were placed in positions 14, 17, 77, 80 and 81 in TM(1+2)-GFP molecule, or at positions 77, 80 or 81 in the full-length CD9 protein. Positions 14, 17 and 80 are distinct from the intramolecular interface and are on the same side of the TM1-TM2 pair (Figure 6). Thus, they are well-positioned to participate in an interaction with another molecule. At the same time, the model predicts that wild-type cysteines (Cys78, 79), which do not yield very efficient zero-length cross-linking, are on the other side of TM1-TM2 pair.
While using the cysteine at position 80 as the dimeric interface probe, mutations of conserved residues in TM1 and TM2 (especially Gly32 and Gly67 to Leu) clearly reduced intermolecular cross-linking. We do not suggest that those residues are directly involved in intermolecular interaction. Rather, we propose that destabilization of the intramolecular TM1-TM2 interaction by Gly to Leu substitutions (discussed above) causes an overall conformational change that reduces dimer formation.
An Ala81Leu mutation did not reduce cross-linking via Cys80, even though single-cysteine Ala81Cys molecules themselves produced a high level of cross-linking. These results, together with data on Gly32Leu and Gly67Leu mutations, are consistent with our model predicting that helices 1 and 2 interact more tightly near the extracellular end and less at the cytoplasmic end. This would give more flexibility to a cysteine at position 81 and also limit the effect of an Ala81Leu mutation. Location of this residue at the membrane/cytoplasmic border could also make it more accessible to CuP reagent as compared to residues buried in TM domain, thus elevating the efficiency of disulfide formation of the Ala81Cys mutant.<br><br>Multiple interfaces in tetraspanin molecules
In the full-length CD9 molecules, the 3 C-terminal cysteines (Cys87, 218 and 219) located at or in TM3 and TM4 promoted efficient dimer and even more efficient oligomer formation compared to the 3 N-terminal cysteines (Figure 9). Cys87 alone can be used to capture CD9 dimers [25]. These results suggest the existence of two dimeric interfaces in CD9 molecule – the TM 1-2/1-2 interface and the TM 3-4/3-4 interface (Figure 10). In a TM(1+2) molecule, the destabilization of 1–2 interaction, e.g. by Gly→Leu mutations, would affect the 1-2/1-2 interface, as discussed above. However, these mutations would not interfere with the 3-4/3-4 interface in a full-length molecule, which includes Cys87, 218 and 219. Thus, cross-linking of full-length molecules, containing all 6 cysteine residues, would be unaffected, as seen in Figure 8C. Furthermore, wild-type Cys9, Cys78 and Cys79 are apparently not at the primary 1-2/1-2 interface. Their relative inefficiency in cross-linking CD9 TM(1+2) protein likely reflects weak secondary contacts between the molecules, or possibly random collision events. Such events should be independent of mutations in the conserved Gly residues in TM1 and TM2, as was demonstrated in Figure 8B. The potential existence of two interfaces in tetraspanin molecules, 1-2/1-2 and 3-4/3-4, should provide enhanced flexibility for forming additional intermolecular contacts. Current understanding of tetraspanin microdomains assumes a few strong, primary homotypic and heterotypic tetraspanin complexes (e.g. CD9-CD9, CD9-CD81, CD151-α3 integrin, CD81-EWI2) that help bring together various other proteins, forming secondary-type associations. Such properties of tetraspanins may bring signaling molecules such as protein kinase C or phosphatidylinositol 4-kinase to the vicinity of integrins [67,68].
The organization of the TM3 domain points to a potential role in protein-protein interactions. A motif of Leu-Leu-Glu(Gln) spaced 7 residues apart (heptad positions a), with highly conserved residues in two consecutive positions d, poses as a likely interaction module. If responsible for heterologous protein-protein interactions, it would form another distinct interface of tetraspanin molecule. Our preliminary data indicate that replacing the Leu and Glu residues in TM3 of CD9 with Ala has no effect on cell surface expression of the protein and its dimerization (data not shown). It remains to be tested if interactions with other proteins will be affected. Similarly, the TM4 domain may provide additional contributions to lateral tetraspanin associations. Much higher sequence variability, and the lack of a distinct heptad pattern suggests that TM4 is a major contributor to diversity among tetraspanin complexes. Structure-function analysis of TM3 and TM4 domains in tetraspanins is the subject of ongoing investigation.<br><br>
Conclusion
We have defined the TM1-TM2 intramolecular interface in tetraspanin CD9, providing evidence for glycines (Gly25 and Gly32 in TM1, Gly67 and Gly74 in TM2) packing against apposing bulky aliphatic residues. Second, we mapped an intermolecular CD9 interface (involved in CD9 homodimer formation) to the vicinity of residues Leu14 and Phe17 in TM1 and Gly77, Gly80 and Ala81 in TM2. Finally, we provide preliminary evidence that TM3 and TM4 in CD9 may contribute to a second intermolecular interface. Key CD9 residues involved in intra- and intermolecular interactions are highly conserved throughout the tetraspanin family, thus suggesting that our findings will apply to most tetraspanins.<br><br>Methods
Materials
Cell culture reagents were from Invitrogen (Carlsbad, CA). 2-bromopalmitate was from Fluka (Milwaukee, WI), N-ethylmaleimide (NEM) and 1,10-phenanthroline were from Sigma-Aldrich (St. Louis, MO), and chemical cross-linkers dithio-bis-maleimidoethane (DTME) and 1,4-Bis-maleimidobutane (BMB) were purchased from Pierce Endogen (Rockford, IL). Triton X-100, protease inhibitor cocktail and FuGENE 6 transfection reagent were obtained from Roche (Indianapolis, IN). Restriction endonucleases and Pfu DNA polymerase were obtained New England Biolabs (Beverly, MA) and Stratagene (Carlsbad, CA), respectively. All other chemicals were purchased from Sigma-Aldrich or Fisher Scientific (Pittsburg, PA).<br><br>Sequence analysis
Tetraspanin sequences were obtained from SWISS-PROT and GenBank databases. Locus designations, accession numbers and the most commonly used protein names are summarized in Tables 1 and 2. TM segments were delineated by inspection of hydrophobicity profiles, using database annotations and previous analyses of TM sequences as a guide ([28], M. Hemler, unpublished), and aligned manually. Residue numbers in <span class="yellow">human</span> CD9 sequence are used a reference point throughout the study.<br><br>DNA cloning and mutagenesis
Sequence encoding CD9 protein was cloned into vector pcDNA3 (Invitrogen, Carlsbad, CA) and pEGFP-N1 (Clontech, Palo Alto, CA), for expression of untagged and C-terminally GFP-tagged CD9, respectively. pEGFP-N1 encoding CD9 TM(1+2) -GFP fusion protein was constructed by subcloning DNA for residues 1–83 of CD9 into HindIII and PstI sites of the vector; to introduce the PstI site, codon GTG for Val82 was changed to CTG (coding for Ala). In the resulting fusion protein, there is a 13-amino acid linker (with no cysteines) between CD9 and GFP. To minimize the low inherent ability of GFP to homodimerize, which could potentially influence the results of CD9 cross-linking, we used a monomeric GFP mutant, Leu221→Lys [34], for cross-linking experiments.
Mutations were introduced in full-length and TM(1+2) CD9 proteins by a PCR-based strategy using mutagenic primers and Pfu DNA polymerase. All mutations were confirmed by DNA sequencing.<br><br>Protein expression, microscopy, cysteine disulfide cross-linking and Western blotting
DNA constructs encoding TM(1+2)-GFP or full-length CD9 proteins were transfected into <span class="yellow">human</span> rhabdomyosarcoma RD cells using the FuGENE 6 reagent. Cells expressing GFP fusion proteins were analysed by fluorescence microscopy 18–28 hours post-transfection. Images were captured using Spot 1.4.0 camera (Diagnostic Instruments, Sterling Heights, MI) attached to Nikon Eclipse TE300 microscope.
For experiments involving cysteine-mediated cross-linking, cells were treated with 50 μM 2-BP starting 24–26 hours post-transfection and continuing for 16–18 hours. Cross-linking was carried out by incubating cells in HBSM buffer (25 mM Hepes-NaOH, pH 7.2, 150 mM NaCl, 2 mM MgCl2) containing either a) 0.6 mM CuSO4 and 1.8 mM 1,10-phenanthroline (CuP complex) or b) 0.2 mg/ml homobifunctional cysteine-reactive cross-linker (e.g. DTME), diluted from fresh 10 mg/ml solution in DMSO. After incubation for 10–15 minutes (with CuP) or 30–45 minutes (with cross-linker), cells were washed twice for 10 minutes with HBSM containing 10 mM NEM to block residual free cysteines. Cells were lysed in HBSM containing 1% Triton X-100, 0.1% SDS and a cocktail of protease inhibitors with 1 mM EDTA at 4°C for 45–60 minutes. Cell lysate was clarified by centrifugation at 14,000 × g for 15 minutes, an aliquot was removed, and proteins from it were precipitated by addition of trichloroacetic acid to 10% on ice followed by centrifugation at 14,000 × g for 10 minutes. After two washes with ice-cold acetone, protein pellet was solubilized in SDS-PAGE sample buffer without a reducing agent (50 mM Tris-HCl, pH 6.8, 1% SDS, 8% glycerol).
In some experiments, CD9 protein was immunoprecipitated using monoclonal antibody Alb6 (Immunotech, Marseille, France). Proteins were separated by SDS-PAGE and analyzed by Western blotting using monoclonal antibody JL-8 (Clontech) for GFP or Alb6 for CD9. Bands from X-ray films were quantitated using GeneTools™ software from Syngene (Frederick, MD).<br><br>Modeling of TM1-TM2 interaction
An atomic model of the CD9 TM1-TM2 dimer was constructed with a Monte Carlo-simulated annealing (MCSA) algorithm [69]. Two idealized α-helices corresponding to TM1 residues Tyr12 through Leu35 and TM2 residues Gly59 through Val82 were docked with six orthogonal parameters: three rigid body translations and three rotations. During each step of a MCSA cycle, there was an equal probability of changing either one parameter or all six parameters to random values. A conformation's energy was calculated in vacuo with the AMBER united-atom force field for van der Waals interactions [70]. The van der Waals term was modified as described by Kuhlman and Baker [71]. If a structure had favorable dimerization energy, the energies of select mutants were calculated. Structures were selected with a novel scoring function that maximizes the Boltzmann probability of dimerization for silent mutations while minimizing the probability for disruptive mutations. Asn18Ser, Gly77Leu, and Gly80Leu were scored as silent mutations while Gly25Leu, Gly32Leu, Gly67Leu, and Gly74Leu were considered to be disruptive. Each MCSA cycle consisted of 50,000 steps with an exponential temperature decay from 10,000 to 10 K.
Ten MCSA cycles through global sample space were used to restrict the search area. Parameters were restricted to ± 2 standard deviations from their mean value for structures within 10 kcal of the best structure. MCSA cycles were repeated as described above with additional optimization of χ values: rotamers at the protein-protein interface were optimized with Dead End Elimination [72], and χ values were further optimized with Monte Carlo. All MCSA cycles converged upon structures that were within a root mean squared deviation (RMSD) of 1.5 Å with the best structure, and structures that scored within 5 kcal of the best score had an RMSD of less than 0.5 Å with the best structure.<br><br>
List of abbreviations
2-BP, 2-bromopalmitate; CuP, Cu2+-phenanthroline; DTME, dithio-bis-maleimidoethane; LEL, large extracellular loop; NEM, N-ethylmaleimide; TM, transmembrane (domain).<br><br>Authors' contributions
OVK carried out sequence comparisons, mutational analysis and cross-linking experiments, and drafted the manuscript. DGM built the TM1-TM2 interaction model and contributed to the manuscript. WFD supervised DGM's work. MEH coordinated the whole study and prepared the final manuscript.<br><br>
<h3>pmcA328326</h3>Enzymatic multiplex DNA sequencing.
Abstract
The problem of reading DNA sequence films has been reformulated using an easily implemented, multiplex version of enzymatic DNA sequencing. By utilizing a uniquely tagged primer for each base-specific sequencing reaction, the four reactions can be pooled and electrophoresed in a single lane. This approach has been previously proposed for use with fluorescently labelled probes (1), and is analogous to the principle used in four-dye fluorescence sequencing except that the signals are resolved following electrophoresis (2). After transfer to a nylon membrane, images are obtained separately for each of the four reactions by hybridization using oligonucleotide probes. The images can then be superimposed to reconstitute a complete sequence pattern. In this way the correction of gel distortion effects and accurate band registration are considerably simplified, as each of the four base-specific ladders require very similar corrections. The methods therefore provide the basis for a second generation of more accurate and reliable film reading programs, as well as being useful for conventional multiplex sequencing. Unlike the original multiplex protocol (3), the approach described is suitable for small projects, as multiple cloning vectors are not used. Although more than one vector can be utilized, only a library of fragments cloned into any single phage, phagemid or plasmid vector is actually required, together with a set of tagged oligonucleotide primers.Images<br><br><br><br><br><br>
 Nucleic Acids Research, Vol. 19, No. 12 3301 <br><br> Enzymatic multiplex DNA sequencing <br><br> Mark Chee <br><br> Medical Research Council Laboratory of Molecular Biology, Hills Road, Cambridge CB2 20H, UK <br><br> Received March 15, 1991; Revised and Accepted May 2, 1991 <br><br> ABSTRACT <br><br> The problem of reading DNA sequence films has been reformulated using an easily implemented, multiplex version of enzymatic DNA sequencing. By utilizing a uniquely tagged primer for each base-specific sequencing reaction, the four reactions can be pooled and electrophoresed in a single lane. This approach has been previously proposed for use with fluorescently labelled probes (1), and is analogous to the principle used in four-dye fluorescence sequencing except that the signals are resolved following electrophoresis (2). After transfer to a nylon membrane, images are obtained separately for each of the four reactions by hybridization using oligonucleotide probes. The images can then be superimposed to reconstitute a complete sequence pattern. In this way the correction of gel distortion effects and accurate band registration are considerably simplified, as each of the four basespecific ladders require very similar corrections. The methods therefore provide the basis for a second generation of more accurate and reliable film reading programs, as well as being useful for conventional multiplex sequencing. Unlike the original multiplex protocol (3), the approach described is suitable for small projects, as multiple cloning vectors are not used. Although more than one vector can be utilized, only a library of fragments cloned into any single phage. phagemid or plasmid vector is actually required, together with a set of tagged oligonucleotide primers. <br><br> INTRODUCTION <br><br> The community of biologists is undertaking the sequencing of representative genomes of various free-living organisms, ranging in size from Mycoplasma (800kb) to mammals (3 Gb) (4). However, the largest contiguous DNA sequences which have been determined so far are the genomes of several dsDNA eukaryotic viruses (5, 6, 7, 8, 9) and plant chloroplasts (10, 11, 12). The largest of these is the 229kb genome of <span class="yellow">human cytomegalovirus</span> (8). The difficulty in sequencing millions of base pairs of DNA is that several steps in the methods are relatively labour intensive, although the sequencing reactions themselves are rapid and easily performed. Two limiting steps in conventional procedures are the size fractionation of sequencing reaction products by gel electrophoresis and the subsequent reading of sequence ladders. The former problem can be overcome by multiplexing, which theoretically allows an enormous amount of <br><br> data to be obtained from a single gel by processing clones as mixtures rather than individually (3). Each sequence in the mixture is labelled by a unique short oligonucleotide 'tag' sequence. This allows the mixture to be resolved following electrophoresis: the superimposed sequence ladders are blotted from the gel to a nylon membrane, and detected one at a time by hybridization using tag-specific oligonucleotide probes. In practice, at least 50 sets of sequences can be obtained from a single gel (3). <br><br> Unfortunately, a bottleneck in the multiplex procedure is the reading of sequence films. In previous large-scale sequencing projects this task has been performed with the aid of a sonic digitizer (13, 14). Although film reading programs have been under development for some time (15), and some programs are commercially available, their error rates are presently more variable and unpredictable than that of a skilled <span class="yellow">person</span> and the accurate interpretation of film-imaged sequence ladders by computer programs is difficult to achieve in routine practice. Programs specifically designed to read multiplex films have an advantage. This is because a sequence image can be used as an 'internal standard' to help interpret other images derived from the same membrane (3). However, the original implementation of the multiplex strategy used chemical sequencing (16), which yields a more complex sequence ladder than the enzymatic dideoxynucleotide chain-termination method (17). Most successful large scale sequencing projects have used the chaintermination method and <span class="blue">bacteriophage M13</span> vectors, which allows the routine production of clean and easily interpretable sequences (18). It was therefore decided to adapt the original multiplex protocol for use with enzymatic sequencing, using tagged primers. <br><br> MATERIALS AND METHODS <br><br> Eight oligonucleotide sequencing primers were synthesized, each 37 nucleotides in length. The 3' end of each primer consists of the 17 nucleotide M13 universal priming sequence [GTAAAACGACGGCCAGT3']. The 5' ends of the primers bear different 20mer tag sequences (Figure 1). In four of the primers, UEO1C, UPOIC, UE02C and UP02C, these tags are complementary to the EO1, PO1, E02 and P02 probe sequences respectively (copied from the original 'plex' vectors (3)). A second set of four primers, UJOL14C, UJOL15C, UJOL16C and UJOL17C, have the following tag sequences: 5' CAAGTTTGAAGGTACTCATT, TATCAATTAAATTGTllTGAC, GTGTTGCTACCCAAGAAGCA, and TGTCACTAGAGCTGTCACTT, respectively. The <br><br> ?=) 1991 Oxford University Press <br><br> 3302 Nucleic Acids Research, Vol. 19, No. 12 <br><br> oligonucleotides were gel-purified (19) and used to sequence ssDNA templates prepared by phenol extraction (20) or SDS denaturation (21). Conventional sequencing reactions were performed as previously described (20). <br><br> For hybridization experiments, radioactively labelled nucleotides were omitted from the sequencing reactions. Instead, the 21d of each nucleotide mix added to the reaction mixture consisted of the following: 'A' mix: 6.25MtM dATP, 62.5lM ddATP; 'C' mix: 6.25MM dCTP, 40MtM ddCTP; 'G' mix: 6.25MtM dGTP, 80MtM ddGTP; 'T' mix: 6.25MM dTTP, 250yM ddTTP; as well as 125MM of each of the three other dNTPs in each mix. Apart from the use of these modified mixes, no changes were made to the conventional sequencing procedure (20). <br><br> Sequencing reactions were pooled and ethanol precipitated as appropriate. Precipitation in microtitre trays was carried out as follows: a mixture of 3.2M1 3M NaAc pH 5.0 and 112Mi1 EtOH was dispensed to individual wells of a microtitre plate (Falcon 3911 or Corning 25855) using an 8-channel pipettor. Each set of four reactions was added to the EtOH/NaAc mixture, and the tray sealed using a Falcon 3073 plate sealer. The samples were mixed by inversion and stored at -20?C for 30 minutes. The DNA was collected by a 20 minute centrifugation at 4 000 rpm in an IEC Centra 3C centrifuge. The sealer was removed, and the plate inverted to discard the supernatant. After blotting the tray on tissue paper, 200MI of 95 % EtOH was added to each well. The plate was covered with a plastic lid and recentrifuged for 2 minutes. The EtOH was discarded and the plate inverted for several minutes on tissue paper, then left for 20 minutes to air dry. Precipitated samples were resuspended in 6M1 deionized water by vortexing on an SMI multi-tube vortexer for 1 minute. Samples were denatured and electrophoresed on 6 % polyacrylamide buffer gradient gels as previously described (20). <br><br> Following electrophoresis, the gel was transferred to a dry piece of Whatman 3MM blotting paper, and placed on a second sheet of blotting paper supported on a glass plate and saturated in 4 x SSC (SSC: 150mM NaCl, l5mM trisodium citrate). This sheet was wicked in a tray containing 1 litre of 4 x SSC. The DNA was transferred to a nylon membrane (Amersham Hybond N) by capillary blotting overnight (22). DNA was fixed to the membranes by U.V. crosslinking (23). <br><br> Plex oligonucleotide probes were a kind gift of Dr.George Church. Probes were tailed at their 3' ends using [a-32p] dCTP as previously described (3). For the preparation of digoxigenin (DIG) labelled probes, identical tailing reactions were carried out substituting  I0pmols of DIG-II dUTP (Boehringer Mannheim) for [a-32P] dCTP. Membranes were prehybridized for at least 10 minutes in 4 x SSC, 5 x Denhardts' (0.1 % (w/v) each of BSA (heated at 80?C for 30 minutes to inactivate any alkaline phosphatase activity), Ficoll (Pharmacia) and polyvinylpyrrolidone), 0.5% (w/v) SDS, 5mM NaHPO4 (23). Hybridization was carried out in 25-50M1 of prehybridization buffer per cm2 of membrane. The probe concentration was approximately lnM. After lh at 42?C, unbound probe was removed by five 1 minute washes at room temperature in 1 x SSC, 0.5% SDS (200MI/cm2 membrane). Radioactive blots were covered in Saran wrap and exposed to film immediately. Detection of DIG labelled probes used an anti-DIG antibodyalkaline phosphatase conjugate (Boehringer Mannheim) according to the manufacturer's instructions, except that all volumes were reduced by 70% and the conjugate was used at a 1:10 000 dilution. Blots were developed in 25M1 of 100mM Tris.Cl pH9.5, <br><br> mantane4-methoxy4(3 "-phosphoryloxy)phenyl-1 ,2-dioxetane); Tropix)/cm2 for 30 minutes at 37?C, prior to exposure to film. Probes and dioxetane were stripped from the membranes by two 10 minute washes at 700C with 0.2% SDS, 2mM EDTA (200,ul/cm2 membrane). <br><br> The hybridization and washing procedures were carried out in plastic bags. However, washing steps have also been performed with gentle agitation in a perspex tub (43 x 27 x 15cm) mounted on a reciprocal shaker, with equivalent results. In the latter case a minimum wash volume of 500mls was used. The use of a tub is more convenient for batch processing and should be straightforward to automate. <br><br> RESULTS <br><br> Autoradiograms revealed no difference in sequence quality when tagged primers were used instead of the 17mer universal primer in conventional [a-35S] dATP labelled sequencing reactions and in multiplex hybridization experiments using [a-32P] dCTP-tailed probes (results not shown). Experiments were then conducted to determine the feasibility of pooling the four base reactions for each clone and fractionating them in a single lane to obtain a superimposed but interpretable set of sequence ladders. The question addressed was whether or not difficulties in band registration might arise as a result of mobility differences between the different primer sequences and/or distortion of the membrane between probings. It is relevant that an automated film reader employing an internal standard requires that the nylon membrane does not undergo significant distortions between probings (George Church, personal communication). Clones were sequenced using the four tagged primers UEOlC, UPOIC, UE02C, and UP02C, one for each base reaction (Figure 1). The A, C, G and T reactions for each clone were pooled, and processed as described above. A complete set of sequence autoradiograms was obtained from four consecutive rounds of probing with [a-32P] dCTP-labelled oligonucleotides. Alignment of the films showed that sequence-specific mobility effects and distortion of the membrane between probings were sufficiently minor to allow accurate registration of the bands, and hence accurate reading of the sequence. At least 200 nucleotides of sequence could be read accurately from a single clone by simply tracing the four sets of bands using different colours, overlaying the tracings, and reading the bands sequentially. In order to assess the practicality of reading the sequences by machine, the images were scanned to provide optical density profiles (Figure 2). These profiles were overlaid, and were found to be sufficiently in register to allow accurate interpretation of the sequence for at least 300 nucleotides. This was essentially the limit of resolution of the gel for accurate manual reading. <br><br> In order to ensure that the relatively minor mobility differences observed between the four primers were not coincidental to the oligonucleotides used, a second set of four tagged M13 universal primers was synthesised, this time incorporating 20mer sequences derived from the genome of <span class="yellow">murine</span> herpesvirus-68 (UJOL14C, 15C, 16C, 17C). Sequencing reactions were performed using [cx-35S] dATP to label the DNA directly. Various templates were sequenced, and in all cases correctly ordered sequence ladders were obtained following conventional electrophoresis in which the four reactions were run side-by-side (results not shown). <br><br> Initial hybridization experiments were conducted using [f -32p] <br><br> dCTP tailed oligonucleotide probes. However, the use of <br><br> lOOmM NaCl, 5OmM MgC12, 0.15mM AMPPD ([3-(2'-ada<br><br> B <br><br> C <br><br> 'Ordinary'                       Plex'      4-CJ3              4-rn     + <br><br> Primeir                        Primer <br><br> "All      "C"       "G"   "   T" <br><br> 'Plex'                       'Ordinary'                                        Pool and <br><br> Vector                          Vector                                          fractionate <br><br> Resolve by sequential <br><br> hybridizations <br><br> Sequencing reactions <br><br> SequencingreactIo product <br><br> Sequencing reaction product <br><br> -           n;   <br><br> =      = <br><br> -   =   <br><br> -   n:  = <br><br> Figure 1. Approaches to enzymatic multiplex DNA sequencing. a) A set of sequence-tagged vectors can be used. The tag site is shown in red, and the insert to be sequenced in blue. However, the original plex vectors (3) are plasmids, and therefore amenable only to dsDNA sequencing. Sets of <span class="blue">bacteriophage M13</span> vectors have been constructed bearing either one (32) or two [Chee, unpublished] of the plex tag sites flanking the polylinker, which can be used for this approach. b) The strategy used in this paper. In this case the tag site is carried on the primer. c) If tagged primers are used, there is no practical impediment to performing each base reaction using a different primer, as depicted. The reactions can then be pooled in any combination desired. The configuration shown, in which the four reactions are electrophoresed in a single lane, is designed to facilitate accurate band registration and reading by an automatic film reader. In order to read the sequence manually, base reactions would be run side-by-side. The logistics of processing the reactions are essentially the same with either configuration; the same number of probings are required. <br><br> Figure 2. Four overlaid one-dimensional optical density profiles for a single clone shown in two overlapping sections. The optical density profiles are unprocessed, except for a simple transform to correct for the relative displacement (translation and rotation) of the four images from which they are extracted. The profiles read 5' to 3' from right to left. Nucleotides positions 66 to 214 from the start of the universal priming site are shown. The sequence is that of Bluescribe M13+ (template DNA obtained by rescue with M13K07 helper phage (30)), and was determined using the primers UEOIC, UPOIC, UE02C, and UP02C for the T, C, G and A <br><br> specific reactions respectively. Detection was by autoradiography following hybridization with [a- 32p] dCTP tailed plex probes. <br><br> A <br><br> Nucleic Acids Research, Vol. 19, No. 12 3303 <br><br> I <br><br> 3304 Nucleic Acids Research, Vol. 19, No. 12 <br><br> a) <br><br> 175-... <br><br> -w<br><br> .m; <br><br> b) <br><br> 182<br><br> a :...... <br><br> F.VW <br><br> an <br><br> - w <br><br> -1 <br><br> S              -  K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~i <br><br> 'F C C; A                                I1 'CG A <br><br> Figure 3. Four separate base-specific reactions imaged from a single lane using chemiluminescent detection. The clones sequenced are: a) Bluescribe M13 + (obtained by rescue with M13K07 helper phage (30)) and b) an M13 recombinant clone prepared in a microtitre tray (21) (a kind gift of Victoria Smith). Nucleotide positions shown on the figure are numbered from the start of the universal priming site. The clones were sequenced using UEOlC, UPOIC, UE02C, and UP02C for the T, C, G and A specific reactions respectively. The blot was probed with corresponding DIG-11-dUTP labelled oligonucleotides. <br><br> radioactivity on the scale envisioned for a large sequencing project is undesirable for reasons of safety. The relatively long exposure times required (6 to 24 hours) and the short half lives of the probes might also be inconvenient. It has been shown that a biotin/streptavidin/alkaline phosphatase based detection system used in conjunction with a chemiluminescent dioxetane substrate overcomes these disadvantages (24, 25, 26). We utilized a different bridging system with similar results. Digoxigenin (DIG) labelled oligonucleotide probes were detected using anti-DIG antibody-alkaline phosphatase conjugates and a chemiluminescent dioxetane substrate. Exposure times of 10 to 15 minutes were typically required, following a one hour preincubation period (Figure 3). In our hands the DIG bridging system was similar in sensitivity to the streptavidin based system (24), and the practical lower limit of template ssDNA required in order to obtain an easily interpretable sequencing ladder was estimated to be in the range of 20 to 50fmols per reaction. However, the sensitivity of detection was limited only by enzymaticallytriggered background luminescence, and not by the level of signal obtained. The nonradioactive methods described have been used successfully in an 8-plex system. <br><br> DISCUSSION <br><br> Although the original multiplex protocol was based on a set of tagged vectors (3), tagged primers have also been used or proposed for various forms of multiplex DNA sequencing (George Church, personal communication; 2, 27). For example, a proposal was recently put forward for multiplex sequencing using sequence-labelled primers and fluorophor-labelled probes <br><br> (1), similar in principle to the methods used here. However, we use tagged primers and the superposition of the four sequencing reactions to address the problem of reading DNA sequence films; a part of this solution is to utilize M13 dideoxynucleotide sequencing, thereby improving the quality of the data to be analyzed. In addition, the proposal for fluorophor-labelled probes does not take into consideration any of the practical sequencing problems addressed here, and, in the version described, remains a promising but unproven scheme for large scale DNA sequencing. <br><br> There are several advantages to tagging primers instead of vectors. Firstly, there is no need to prepare multiple libraries of clones in special vectors. This means that workers can use vector/host combinations that yield good results in their hands, and an increased depth of multiplexing can easily be accomodated by synthesizing more primers. This should make multiplexing more accessible to workers undertaking smaller projects. A theoretical disadvantage of tagged primers is that the procedure can only be multiplexed following primer annealing (1), or following the sequencing reactions (this paper; in practice, pooling immediately after the annealing step might lead to increased backgrounds if one or more primers were present in excess over their template DNAs). This is a relatively late stage. In the original procedure (3), clones were pooled prior to amplification by growth, an early step. However, we do not believe the sacrifice to be of practical importance when using phage vectors. In our experience, recombinant M13 phage have variable growth rates and the effects of competition are likely to severely limit the number of clones that can usefully be pooled for growth. In contrast, by growing clones individually, the depth of multiplexing is only really limited by probe sensitivity. We have not investigated the factors influencing variability in phage growth rates. <br><br> It is worth noting that reliable protocols have been developed for growing large numbers of individual M13 clones and preparing high quality ssDNA templates in microtitre trays (28, 21). It is relatively simple to prepare manually two microtitre trays of ssDNA templates (192 clones) in a day. Sufficient clones can be prepared in a week to sequence a 20kb fragment to a redundancy of 10 (Victoria Smith, personal communication). In this laboratory, ssDNA is now prepared with the aid of a commercially available robotic workstation (21). As sequencing reactions are also carried out in microtitre trays, manually or robotically (20, 29), the entire M13-based dideoxynucleotide sequencing procedure is amenable to automation (29). For these reasons we see little practical advantage in pooling clones early. Finally, by not pooling clones early, the ability to easily retrieve individual clones is retained, which may facilitate directed sequencing later in a project should this become necessary. <br><br> Multiplex DNA sequencing is currently limited by the lack of a robust computer program which can correct for the large variety of gel and sequencing artefacts that are normally encountered. The foundation of a film reading program is the ability to bring into register precisely vertical arrays of base-specific bands. This requires the ability to track lanes, correct for distortions, and order bands based on their relative spacing. A method of sequencing which has successfully overcome the problem of sequence reading uses real-time detection of fluorescently labelled DNA samples migrating through the gel (2). This system also utilizes the principle of running the four base reactions down the same lane (2). However, bands are detected at a fixed location <br><br> in space, and their detection is separated in time. Hence the <br><br> Nucleic Acids Research, Vol. 19, No. 12 3305 <br><br> problem of gel distortion is essentially avoided, although corrections for the different mobilities of the four dyes must be carried out. In contrast, we utilize the advantages of single lane electrophoresis to address the problem of superimposing four relatively large and complex two-dimensional images. Furthermore, by using sequence-tagged oligonucleotides which are detected by hybridization, a much greater depth of multiplexing can realistically be achieved than by real-time detection. <br><br> The use of two-dimensional colour traces to depict the processed output of a film reader is consistent with the method of displaying fluorescence traces, and should facilitate the checking and editing of sequence databases in which both kinds of data have been entered. The sequence compilation programs used in this laboratory, which are already capable of handling large shotgun databases (8, 31), have recently undergone extensive improvements (Rodger Staden, personal communication). There is now an interactive database editor which allows the graphical display of fluorescence traces, and it is envisaged that this feature could be extended to allow the handling of data from a film reader when a suitable machine is developed. <br><br> ACKNOWLEDGEMENTS <br><br> I am particularly grateful to George Church for thought-provoking discussions and gifts of vectors and oligonucleotide probes and to John Sulston for advice. I also thank Victoria Smith for the gift of DNA samples, Bart Barrell for long-term support, Tom O'Keefe of Milligen/Biosearch for lessons in multiplexing and Amersham International for the optical density overlays shown in Figure 2. M.C. is supported by a fellowship from Applied Biosystems. <br><br> 12. Hiratsuka, J., Shimada, H., Whittier, R., Ishibashi, T., Sakamoto, M., Mori, <br><br> M., Kondo, C., Honju, Y., Sun, C. -R, Meng, B. -Y, Li, Y. -Q, Kanno, A., Nisizawa, Y., Hirai, A., Shinozaki, K. and Sugiura, M. (1989) Molecular and General Genetics, 217, 185-194. <br><br> 13. Komaromy, M. and Govan, H. (1984) Nucleic Acids Research, 12, 675-678. 14. Staden, R. (1984) Nucleic Acids Research, 12, 499-503. <br><br> 15. Elder, J. K., Green, D. K. and Southern, E. M. (1986) Nucleic Acids <br><br> Research, 14, 417-424. <br><br> 16. Maxam, A. M. and Gilbert, W. (1977) Proceedings of the National Academy <br><br> of Sciences, U.S.A., 74, 560-564. <br><br> 17. Sanger, F., Nicklen, S. and Coulson, A. R. (1977) Proceedings of the National <br><br> Academy of Sciences, U.S.A., 74, 5463-5467. <br><br> 18. Sanger, F., Coulson, A. R., Barrell, B. G., Smith, A. J. H. and Roe, B. <br><br> A. (1980) Journal of Molecular Biology, 143, 161-178. 19. Applied Biosystems User Bulletin (1987) 13, 11-16. <br><br> 20. Bankier, A. T., Weston, K. M. and Barrell, B. G. (1987) Methods in <br><br> Enzymology, 155, 51-93. <br><br> 21. Smith, V., Brown, C. M., Bankier, A. T. and Barrell, B. G. (1990) DNA <br><br> Sequence, 1, 73-78. <br><br> 22. Southern, E. M. (1975) Journal of Molecular Biology, 98, 503-517. <br><br> 23. Church, G. M. and Gilbert, W. (1984) Proceedings of the National Academy <br><br> of Sciences, U.S.A., 81, 1991-1995. <br><br> 24. Beck, S., O'Keefe, T., Coull, J. M. and Koster, H. (1989) Nucleic Acids <br><br> Research, 17, 5115-5123. <br><br> 25. Tizard, R., Cate, R. L., Ramachandran, K. L., Wysk, M., Voyta, J. C., <br><br> Murphy, 0. J. and Bronstein, I. (1990) Proceedings of the National Academy of Sciences, U.S.A., 87, 4514-4518. <br><br> 26. Beck, S. and Koster, H. (1990) Analytical Chemistry, 62, 2558-2570. <br><br> 27. Jacobson, K. B., Arlinghaus, H. F., Schmitt, H. W., Sachleben, R. A., <br><br> Brown, G. M., Thonnard, N., Sloop, F. V., Foote, R. S., Larimer, F. W., Woychik, R. P., England, M. W., Burchett, K. L. and Jacobson, D. A. (1991) Genomics, 9, 51-59. <br><br> 28. Eperon, I. C. (1986) Analytical Biochemistry, 56, 406-412. <br><br> 29. Bankier, A. T. and Barrell, B. G. (1989) In Howe, C. J. and Ward, E. <br><br> S. (ed), Nucleic acids sequencing: a practical approach. IRL Press, Oxford, Vol. 1, pp. 37-78. <br><br> 30. Vieira, J. and Messing, J. (1987) Methods in Enzymology, 153, 3-11. 31. Davison, A. DNA Sequence, in press. <br><br> 32. Heller, C., Radley, E., Khurshid, F. A. and Beck, S. Gene, in press. <br><br> REFERENCES <br><br> 1. Yang, M. M. and Youvan, D. C. (1989) Biotechnology, 7, 576-580. <br><br> 2. Smith, L. M., Sanders, J. Z., Kaiser, R. J., Hughes, P., Dodd, C., Connell, <br><br> C. R., Heiner, C., Kent, S. B. H. and Hood, L. E. (1986) Nature, 321, 674-679. <br><br> 3. Church, G. M. and Kieffer-Higgins, S. (1988) Science, 240, 185-188. 4. Watson, J. D. (1990) Science, 248, 44-49. <br><br> 5. Baer, R., Bankier, A. T., Biggin, M. D., Deininger, P. L., Farrell, P. J., <br><br> Gibson, T. J., Hatfull, G., Hudson, G. S., Satchwell, S. C., Seguin, C., Tuffnell, P. S. and Barrell, B. G. (1984) Nature, 310, 207-211. <br><br> 6. Davison, A. J. and Scott, J. E. (1986) Journal of General Virology, 67, <br><br> 1759-1816. <br><br> 7. McGeoch, D. J., Dalrymple, M. A., Davison, A. J., Dolan, A., Frame, <br><br> M. C., McNab, D., Perry, L. J., Scott, J. E. and Taylor, P. (1988) Journal of General Virology, 69, 1531-1574. <br><br> 8. Chee, M. S., Bankier, A. T., Beck, S., Bohni, R., Brown, C. M., Cerny, <br><br> R., Horsnell, T., Hutchison III, C. A., Kouzarides, T., Martignetti, J. A., Satchwell, S. C., Tomlinson, P., Weston, K. M. and Barrell, B. G. (1990) Current Topics in Microbiology and Immunology, 154, 125-169. <br><br> 9. Goebel, S. J., Johnson, G. P., Perkus, M. E., Davis, S. W., Winslow, J. <br><br> P. and Paoletti, E. (1990) Virology, 179, 247-266. <br><br> 10. Ohyama, K., Fukuzawa, H., Kohchi, T., Shirai, H., Sano, T., Sano, S., <br><br> Umesono, K., Shiki, Y., Takeuchi, M., Chang, Z., Aota, S. -I, Inokuchi, H. and Ozeki, H. (1986) Nature, 322, 572-574. <br><br> 11. Shinozaki, K., Ohme, M., Tanaka, M., Wakasugi, T., Hayashida, N., <br><br> Matsubayashi, T., Zaita, N., Chunwongse, J., Obokata, J., YamaguchiShinozaki, K., Ohto, C., Torazawa, K., Meng, B. Y., Sugita, M., Deno, H., Kamogashira, T., Yamada, K., Kusuda, J., Takaiwa, F., Kato, A., Tohdoh, N., Shimada, H. and Sugiura, M. (1986) EMBO Journal, 5, 2043-2049. </body></html>