
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>bacteriophage</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1851977</h3>Molecular Basis for a Lack of Correlation between Viral Fitness and Cell Killing Capacity
Abstract
The relationship between parasite fitness and virulence has been the object of experimental and theoretical studies often with conflicting conclusions. Here, we provide direct experimental evidence that viral fitness and virulence, both measured in the same biological environment provided by host cells in culture, can be two unrelated traits. A biological clone of <span class="yellow">foot-and-mouth disease virus</span> acquired high fitness and virulence (cell killing capacity) upon large population passages in cell culture. However, subsequent plaque-to-plaque transfers resulted in profound fitness loss, but only a minimal decrease of virulence. While fitness-decreasing mutations have been mapped throughout the genome, virulence determinants—studied here with mutant and chimeric viruses—were multigenic, but concentrated on some genomic regions. Therefore, we propose a model in which viral virulence is more robust to mutation than viral fitness. As a consequence, depending on the passage regime, viral fitness and virulence can follow different evolutionary trajectories. This lack of correlation is relevant to current models of attenuation and virulence in that virus de-adaptation need not entail a decrease of virulence.
Virulence expresses the harm that parasites inflict upon their hosts. Many studies have addressed the basis of virulence and its effect on host and parasite survival. It has generally been accepted that one of the components of parasite virulence is fitness, or the capacity of the parasite to multiply in its host. Some models have equated virulence with fitness. In the present study, we use <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) to document that virulence and fitness—measured in the same biological environment provided by cells in culture—can be unrelated traits. This has been achieved by multiplying the virus in a manner that mutations accumulated in its genome. Mutations decreased fitness dramatically, but not virulence. Chimeric and mutant viruses were constructed to show that virulence is influenced by only some of the <span class="yellow">FMDV</span> genes, while fitness is influenced by the entire genome. For this reason, virulence is more robust (“resistant”) than fitness to the effects of deleterious mutations. The fact that virulence can be unrelated to fitness has implications for the design of anti-viral vaccines because it suggests that it may be possible to design high fitness, low virulence strains to stimulate the host immune response. Furthermore, in modelling studies it cannot be assumed that virulence is equal to fitness.<br><br><br><br>Introduction
The relationship between fitness and virulence is an unsettled question, and sometimes fitness is considered a component of the virulence phenotype of parasites. RNA viruses are ideal systems to address this important question because of their high mutability and fecundity, which result in a potential for rapid evolution, and also because of the availability of quantitative fitness and virulence assays.
RNA viruses replicate as complex and dynamic mutant spectra, termed viral quasispecies. Key to quasispecies dynamics are mutation rates in the range of 10−3 to 10−5 substitutions per nucleotide copied, and competition among continuously arising variant genomes [1–4], which prompt rapid movements in sequence space, with corresponding changes of position in the fitness landscape [5]. Indeed, large population passages of RNA viruses in cell culture permit competitive optimization of mutant distributions that generally result in fitness gain [6,7], while repeated bottleneck events (experimentally realized as plaque-to-plaque transfers) lead to random accumulation of deleterious mutations (operation of Muller's ratchet [8]) and result in average fitness decreases [9–13]. Fitness recovery of low fitness <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) clones occurs mainly with introduction of mutations along the genome, with very few true reversions.
An understanding of the consequences of fitness variation for viral virulence is a key question for viral pathogenesis and evolution. Here, we approach this issue with <span class="yellow">FMDV</span>, an important viral pathogen in veterinary medicine [14], and one that fully participates of quasispecies dynamics. Our laboratory has characterized multiple <span class="yellow">FMDV</span> variants that derive from one original biological clone, and that occupy widely different fitness levels when replicating in a defined environment in cell culture. We define fitness as the replication capacity of a mutant <span class="yellow">FMDV</span>, relative to a reference <span class="yellow">FMDV</span>, in direct growth-competition upon coinfection of baby <span class="yellow">hamster</span> kidney 21 (BHK-21) cells [15–17]. Fitness of <span class="yellow">FMDV</span> in BHK-21 cells is a multigenic trait [7].
In the present study, we define virulence of <span class="yellow">FMDV</span> as the capacity of the virus to kill BHK-21 cells under a standard set of cell culture conditions [18]. Thus, the <span class="yellow">FMDV</span>–BHK-21 system offered a means to investigate in a direct and comparative fashion the relationship between fitness and virulence of a virus, measured in the same biological environment provided by BHK-21 cells. We describe the behavior of an <span class="yellow">FMDV</span> clone (
				), which has a history of repeated serial plaque-to-plaque transfers in BHK-21 cells [11], that attained a very low fitness value relative to its parental reference virus (C-S8c1), and yet, its virulence for BHK-21 cells was significantly higher than that of C-S8c1. A comparative study of the capacity to kill BHK-21 cells of chimeric <span class="yellow">FMDVs</span> constructed with cDNA copies of the two parental <span class="yellow">FMDVs</span> indicates that the enhanced virulence for BHK-21 cells of the low fitness clone is a polygenic trait, with participation of the regions encoding capsid proteins and non-structural proteins 2A, 2B, and 2C as virulence determinants. Three specific amino acid replacements in 2C have been identified as redundant virulence determinants of <span class="yellow">FMDV</span> for BHK-21 cells. Thus, while large population passages of the virus resulted in a gain of both fitness and virulence, subsequent bottleneck passages resulted in a decrease of fitness but not of virulence.
			
The results suggest that fitness is very vulnerable to mutation in any genomic region. In contrast, because of the involvement of several (but not all) viral genes in virulence, and the redundant effect of three 2C substitutions, virulence is a more robust phenotypic trait than fitness, and less vulnerable to accumulation of mutations. Therefore, we provide direct evidence that viral fitness and capacity to kill cells can (in some cases) be unrelated traits. Furthermore, the relationship between fitness and virulence, of being either linked or unrelated traits, depends on the evolutionary history of the virus. This observation has implications for viral pathogenesis, and sheds light on models of virulence proposed on the basis of theoretical and experimental studies with cellular organisms.<br><br>Results
Inability of <span class="yellow">FMDV</span> 
					 to Establish a Persistent Infection in BHK-21 Cells
				
Several biological clones and populations were obtained by passaging <span class="yellow">FMDV</span> biological clone C-S8c1 [19–22] in BHK-21 cells, either as large population passages or plaque-to-plaque transfers (Figure 1). The biological clones and populations differed up to 236-fold in relative fitness (Table 1). The fitness differences found are expected from previous results on fitness gain upon large population passages of RNA viruses [6,7] and fitness decrease upon plaque-to-plaque (bottleneck) transfers [9–13]. The initial experiment was aimed at testing whether 
					, because of its low fitness (0.11 times that of its parental C-S8c1 [12,23,24] [Table 1]), had an advantage in establishing a persistent non-cytopathic infection in BHK-21 cells as compared with its parental clone, C-S8c1 (Figure 1). A persistent <span class="yellow">FMDV</span> infection is established by growing the cells that survive a standard cytolytic infection with <span class="yellow">FMDV</span> [25]. Confluent monolayers of BHK-21 cells were infected either with C-S8c1 or with 
					 at a multiplicity of infection (MOI) of 0.02–0.1 plaque-forming units (PFU)/cell (2 × 106 cells infected with 4 × 104 −2 × 105 PFU). Unexpectedly, at 24 h postinfection, the cells infected with 
					 showed extensive cytopathology, and at 48 h postinfection, no surviving cells were observed. The frequency of surviving cells in parallel infections with C-S8c1 was 5 × 10−3–9 × 10−3, which is consistent with previous determinations [25]. No persistently infected BHK-21 cell cultures could be established with 
					, despite several attempts. Thus, C-S8c1, which displays a 9-fold higher relative fitness than 
					 in BHK-21 cells, showed a capacity to kill BHK-21 cells that was at least 103-fold lower than the killing capacity of 
					 in the infectivity assay intended to establish a persistent <span class="yellow">FMDV</span> infection.
				<br><br><span class="yellow">FMDV</span> Fitness and Capacity to Kill BHK-21 Cells May Not Be Positively Correlated
The capacity of 
					 to kill BHK-21 cells despite its low fitness in BHK-21 cells led us to quantitatively examine the relationship between fitness of <span class="yellow">FMDV</span> and its capacity to kill BHK-21 cells. To this aim, <span class="yellow">FMDV</span> clones or populations were compared in a cell killing assay, consisting in determining the time required to kill 104 BHK-21 cells as a function of the PFU added (described in Materials and Methods). The results (Figure 2A) indicate that over the time range of 12 h to 48 h postinfection, the number of PFUs needed to kill 104 BHK-21 cells varied logarithmically as a function of time. Similar quantifications of relative virulence were obtained by measuring the PFU needed to kill 104 cells in 24 h, and then by extrapolating the PFU values to 0 h postinfection (Tables 1 and S1). Virulence of 
					 was 29 to 35 times higher than virulence of C-S8c1, despite the latter displaying a 9-fold higher fitness (Tables 1 and S1). The high virulence of 
					 was not due to the plaque-to-plaque transfers, since a high virulence was also quantitated for its parental clone, 
					, and for population C-S8p113 (Figure 2B; Tables 1, 2, and S1). 
					 deviated from a line that correlated relative fitness of <span class="yellow">FMDV</span> and the logarithm of cell killing capacity, as reflected in the decrease of the regression coefficient (R2) (inset in Figure 2A). Probably, this deviation is due to the fact that 
					 lost fitness due to plaque-to-plaque transfers, and the other viruses were not subjected to plaque-to-plaque transfers. On the other hand, virulence determinants were acquired during the large population passages done between C-S8c1 and C-S8c1p113. The 29- to 35-fold higher virulence of 
					 with respect to C-S8c1 (Tables 1 and S1), despite its low fitness, indicates that viral fitness and virulence can be two unrelated traits.
				<br><br>Mapping Virulence Determinants in the 
					 Genome
				
The comparison of the consensus nucleotide sequence of the 
					 genome with that of C-S8c1 revealed a total of 47 mutations (Table S2), leading to 21 amino acid replacements affecting structural and non-structural proteins (Figure 3). To identify the genomic regions associated with the increased virulence of 
					 with respect to C-S8c1, we measured the BHK-21 cell killing capacity of nine chimeric viruses rescued from constructs obtained by introducing fragments of cDNA of the 
					 genome into plasmid pMT28, which encodes infectious C-S8c1 RNA [21] (Figure 4). The results (Figure 5; Tables 2 and S1) show that several genomic regions contribute to the virulence of 
					 for BHK-21 cells, and that the major contributors map within genomic positions 2046 to 3760 (residues encoding part of VP2, VP3, and part of VP1, Figure 5A) and 3760 to 5839 (residues encoding 2A, 2B, 2C, and 3A, Figure 5B). The results exclude the internal ribosome entry site and the 3C- and 3D-coding regions as significant virulence determinants of 
					 for BHK-21 cells (virulence of the relevant chimeric viruses ≤ 2.5, relative to C-S8c1; Tables 2 and S1). Infectious progeny production by each chimeric virus was intermediate between the production of the parental viruses pMT28 and 
					, with no significant differences that could be correlated with virulence (Table 2).
				<br><br>Non-Structural Protein 2C Is a Determinant of the Virulence of <span class="yellow">FMDV</span> for BHK-21 Cells
Amino acid substitutions in <span class="yellow">human rhinovirus</span> protein 2C promoted cytopathology for <span class="yellow">mouse</span> L cells [26]. Remarkably, 
					 shares with other <span class="yellow">FMDV</span> clones and populations, notably, MARLS and C-S8p260p3d (the two viruses showing the highest virulence for BHK-21 cells; Figure 2A; Tables 1 and S1), three amino acid substitutions in 2C: S80N, T256A, and Q263H. In addition, MARLS and CS8p260p3d include replacement M283V in 2C, relative to C-S8c1 [27,28]. To test whether any (or a combination) of the three shared amino acid substitutions in 2C contributed to the increased virulence of <span class="yellow">FMDV</span>, each of the mutations was introduced individually into plasmid pMT28 by site-directed mutagenesis, as described in Materials and Methods. Transcripts of the three mutants, termed pMT28 (SN), pMT28 (TA), and pMT28 (QH) (Figure 4), were used to transfect BHK-21 cells, and the viruses obtained were tested with the BHK-21 cell killing assay. Viruses having any of the substitutions in 2C have a virulence intermediate between that of C-S8c1 and 
					 (Figure 6A). To test whether the combination of the three substitutions in 2C could produce an additional increase of virulence, the three mutations were introduced in pMT28 to rescue the triple mutant pMT28 (SN, TA, QH) (Figure 4). The results (Figure 6B) show that the virulence of the triple 2C mutant is similar to the virulence of the individual 2C mutants. The 2C mutations did not significantly affect the infectious progeny production (Table 2). A testable prediction of this result is that the introduction of the wild-type 2C-3A-coding region in the genetic background of 
					 should produce a virus with lower virulence than 
					. Indeed, the results with such a chimeric virus (Figure 5D) indicate that the presence of the 2C- and 3A-coding region as the only genetic region of the pMT28 in the genetic background of 
					 resulted in an <span class="yellow">FMDV</span> with a 2.4- to 4.8-fold lower virulence than 
					. We conclude that mutations in 2C contribute to virulence of <span class="yellow">FMDV</span> for BHK-21 cells.
				
Thus, a virus that evolves towards low fitness levels due to the operation of Muller's ratchet may nevertheless maintain its capacity to kill the same cells in which it displays low fitness. In <span class="yellow">FMDV</span>, the enhanced capacity to kill BHK-21 cells was multigenic, including participation of non-structural protein 2C with three amino acid substitutions acting in a redundant fashion. In conclusion, the results provide a molecular interpretation of why fitness and virulence of an animal virus can follow disparate evolutionary trajectories, culminating in two unrelated traits.<br><br>
Discussion
The capacity of a virus to kill cells is probably influenced by several steps in the virus life cycle, including receptor affinity (which may trigger signalling pathways and alter cell functions) and intracellular viral replication that may lead to metabolic alterations such as transcriptional or translational shut-off [29]. A parallel increase of virulence and fitness as a virus improves its adaptation to a host cell type is expected, since a key parameter that should contribute to the fitness level of a cytophatic virus is the accumulation of infectious particles and its release from cells, which are events often associated with cell killing. This expectation was fulfilled in our experiments (increase of both fitness and cell killing following large population passages of C-S8c1 [Figure 2]), and also in other virus–host systems. In a comparison of two genetically divergent isolates of the whispovirus white spot syndrome virus, whose virulence for the <span class="yellow">shrimp</span> host <span class="yellow">Penaeus monodon</span> was measured by in vivo cumulative mortality rates, virulence correlated with competitive fitness in vivo [30]. The onset of type 1 diabetes by coxsackievirus B strains was linked to the viral replication rate and to the infectious dose [31]. In engineered alphavirus replicons, a direct correlation between the level of RNA replication and cytopathogenicity was observed [32]. At an epidemiological level, a greater replicative fitness of historical versus current <span class="yellow">human immunodeficiency virus type 1</span> (<span class="yellow">HIV-1</span>) isolates was taken as evidence of <span class="yellow">HIV-1</span> attenuation over time, assuming a direct connection between fitness and virulence [33]. In vivo, viral fitness may vary among specific organs, and virulence may be affected only when fitness for some specific target tissues is affected [34].
Replicative fitness is, however, but one of several factors which influence the progression of a viral infection in vivo. In a comparative analysis, R5-tropic and X4-tropic clones of <span class="yellow">HIV-1</span> showed similar replication capacity in mitogen-activated T cells. However, X4 clones were transferred more efficiently than R5 clones from dendritic cells to CD4(+) T cells, a fact that can contribute to the competitive advantage of X4 viruses in AIDS <span class="yellow">patients</span> [35]. <span class="yellow">Simian immunodeficiency virus</span> <span class="yellow">SIVmac239</span> infects both the <span class="yellow">sooty mangabey</span> and the <span class="yellow">rhesus macaque</span>, reaching high viral loads in both hosts, yet it is only virulent for the <span class="yellow">rhesus macaque</span> [36]. Deviations of a positive correlation between viral fitness and virulence were observed also in the plant viruses <span class="yellow">cucumber mosaic virus</span> [37] and <span class="yellow">barley stripe mosaic virus</span> [38]. A study of the effect of lysis timing on <span class="blue">bacteriophage</span> fitness revealed that a strain with an intermediate lysis time had the highest fitness [39]. The time of transmission may also affect virulence. Nuclear polyhedrosis virus transmitted early to its host, the <span class="yellow">moth</span> <span class="yellow">Lymantria dispar</span>, was more virulent than virus transmitted late, although the latter was more productive because the virus could use more host tissue for replication [40]. In a study of the susceptibility of North American and non–North American breeds of Lymantria to several isolates of the fungus <span class="yellow">Entomaphaga maimaiga</span>, mortality was scored in all cases. However, virulence of the fungus, quantitated by the time of death of Lymantria, was, in some cases, inversely proportional to fitness, quantitated by fungal reproduction in the <span class="yellow">moth</span> [41]. In all these cases, the molecular basis of the lack of positive correlation between fitness and virulence is not understood.
Model for a Lack of Correlation between Viral Fitness and Virulence
The results with <span class="yellow">FMDV</span> clones H5 have documented that both fitness-enhancing and virulence-enhancing mutations can be incorporated in the viral genome in such a fashion that subsequent fitness-decreasing mutations associated with bottleneck (plaque-to-plaque) transfers produce only minimal effects on virulence (Figure 2). The dissection of accompanying molecular events, achieved through quantification of virulence of recombinant and mutant genomes (Tables 2 and S1), provides an interpretation of the lack of positive correlation between virulence and fitness. Multiple fitness-decreasing mutations occur in the course of plaque-to-plaque transfers, distributed throughout the <span class="yellow">FMDV</span> genome [11]. In contrast, determinants of virulence for BHK-21 cells are multigenic, but concentrated mainly in some <span class="yellow">FMDV</span> genomic regions. Similar multigenic but discrete virulence determinants have been described also in other virus–host systems [42,43]. To decrease virulence, mutations occurring randomly in the course of plaque-to-plaque transfers should affect specific genomic sites, and this will occur with a lower probability than fitness-decreasing mutations, which can hit any of the multifunctional picornaviral proteins and regulatory regions [11]. This model is reinforced by the observation that three amino acid substitutions in 2C (S80N, T256A, and Q263H) had a similar effect in enhancing <span class="yellow">FMDV</span> virulence, and the three mutations in the same genome had an effect comparable to each mutation individually (Figure 6; Tables 2 and S1). It is not clear what the basis of the contribution of 2C to virulence for BHK-21 could be. 2C is involved in RNA synthesis and contains a nucleotide-binding domain, although none of the substitutions found in 
					 and 
					 lie within such a domain. An unlikely triple reversion would be required to eliminate the virulence-enhancing effect of the three mutations in 2C. We propose that a higher robustness of the <span class="yellow">FMDV</span> genome with regard to virulence for BHK-21 cells, rather than to replicative fitness in the same cells, underlies the different trajectories followed by fitness and virulence upon subjecting the virus to repeated bottleneck transfers. Obviously, we cannot exclude that parameters of the virus life cycle, other than fitness as measured in our experiments, could correlate with virulence for BHK-21.
				
The comparative analysis of <span class="yellow">FMDV</span> clones and populations shows that shifts in virulence can occur even through the evolution of a single viral clone (C-S8c1), with its restricted genetic diversity prompted by different replication regimes in the same host cells, which also have a clonal origin (see Materials and Methods). We conjecture that the demonstration that fitness and virulence can follow different evolutionary courses has been possible thanks to the consequences of the extreme passage regimes to which the viral populations were subjected: competitive evolution of an ample mutant spectra during repeated large population passages, and accumulation of deleterious (with regard to fitness, but not with regard to virulence) mutations upon plaque-to-plaque transfers (predominance of genetic drift and operation of Muller's ratchet) [12,15].<br><br>Implications for Models of Virulence
It must be emphasized that fitness and virulence are relative values that pertain to a defined physical and biological environment. Virulence determinants of <span class="yellow">FMDV</span>, identified here for BHK-21 cells, need not apply to virulence for the natural animal hosts of <span class="yellow">FMDV</span> [44]. However, the observation of a lack of correlation between fitness and virulence in a <span class="yellow">FMDV</span> clone is relevant to current models of attenuation and virulence, since it shows that more virulent forms of a virus need not have a reproductive advantage, and that viral virulence is not necessarily a byproduct of viral fitness. Even if virulence is regarded as an unavoidable consequence of parasite adaptation [45], virus de-adaptation (fitness loss) need not entail a decrease of virulence.
Most current definitions of virulence include both the ability of the pathogen to multiply and to cause harm to its host; some authors, however, assume a direct relationship between fitness and capacity to produce disease [46–48]. In relating the results with <span class="yellow">FMDV</span> to general models of virulence in host–parasite systems, it must be considered that in the <span class="yellow">FMDV</span> system, evolution of the host BHK-21 cells could not influence <span class="yellow">FMDV</span> evolution, because clonal cell populations with a controlled passage history were supplied in constant numbers at each infection event (see Materials and Methods). Therefore, changes in host density, or mobility, as well as pathogen survival in the external environment, all of which are relevant parameters in virulence models [48,49], cannot play a role in our system. A consistent finding in serial passage experiments is that virulence of a parasite increases with passage number in a new host [50]. The results with <span class="yellow">FMDV</span> infecting BHK-21 cells cytolytically imply that the increase of virulence can be conditioned to the history of passage regimes undergone by a virus.
The invariance of BHK-21 cells in the course of serial cytolytic passages of <span class="yellow">FMDV</span> is in contrast with the parallel system consisting of BHK-21 cells persistently infected with <span class="yellow">FMDV</span> C-S8c1 [25], in which the cells are passaged and coevolve with the resident virus [51]. Host–virus coevolution has generally favored a decrease of viral virulence in the field, a classical example being <span class="yellow">myxoma virus</span> and myxomatosis in <span class="yellow">rabbits</span> [52].
Our comparison of <span class="yellow">FMDV</span> clones did not provide evidence of clones with high fitness and low virulence, which, with regard to natural hosts, is an aim of biomedicine to obtain vaccine strains. Yet, the existence of specific mutations that differentially affect fitness and virulence opens the way to engineer candidate vaccine strains unable to kill the host, while maintaining replicative competence. Virulence is, however, a feature of the host–parasite relationship [46], and the mutations needed to impair virulence are expected to be host-dependent [53,54].<br><br>
Materials and Methods
Cells, viruses, and infections.
The BHK-21 cells used in the present study were cloned by end-point dilution, followed by preparation of a cell stock from a single cell; they were passaged a maximum of 30 times before being used for <span class="yellow">FMDV</span> infection [25,51]. Procedures for cell growth, infection of BHK-21 cell monolayers with <span class="yellow">FMDV</span> in liquid medium, and plaque assays in semi-solid agar medium were carried out as previously described [11,19,25,27]. Mock-infected cells were handled in parallel in all infectivity and plaque assays to monitor absence of viral contamination. The <span class="yellow">FMDVs</span> used in the present study (Figure 1) are (i) the reference clone C-S8c1, which has been assigned a relative fitness of 1.0 [11]. (ii) MARLS, a monoclonal antibody escape mutant isolated from population C-S8c1p213 [55]; MARLS has a fitness of 25 relative to C-S8c1 [24]. (iii) C-S8p260p3d, a standard <span class="yellow">FMDV</span> virus rescued by low MOI passage of C-S8p260. The latter is a virus that evolved by passage of C-S8c1 at a high MOI, which resulted in dominance of two defective <span class="yellow">FMDV</span> genomes (both including internal deletions) that were infectious by complementation, in the absence of standard virus [22,24,28]; C-S8p260p3d has a relative fitness of 20 [24]. (iv) REDpt60, obtained after 60 successive plaque-to-plaque transfers of RED (a monoclonal antibody escape mutant isolated from population C-S8c1p100) [20]; REDpt60 has a fitness of 1.9 relative to C-S8c1. (v) C-S8c1p113, a viral population obtained after 113 serial cytolytic passages of C-S8c1 at a high MOI in BHK-21 cells (2 × 106 BHK-21 cells infected with the virus contained in 200 μl of the supernatant from the previous infection). (vi) Clone 
					, a biological clone isolated from population C-S8c1p113 [11]; its relative fitness is 26 (unpublished data). (vii) Clone 
					, obtained after 95 successive plaque-to-plaque transfers of 
					 [11]; its relative fitness is 0.11 [23].
				<br><br>Cell killing assay.
The capacity of <span class="yellow">FMDV</span> to kill BHK-21 cells was measured as previously described [18,22]. The assay consists in determining the minimum number of PFU required to kill 104 BHK-21 cells after variable times of infection. The assay was performed in M96 multiwell plates with monolayers of 104 BHK-21 cells per well infected with serial dilutions of virus. At different times postinfection, cells were fixed with 2% formaldehyde and stained with 2% crystal violet in 2% formaldehyde. Results are expressed as the logarithm of the number of PFUs needed for complete cell killing (as judged by cell staining with crystal violet, with series of control wells with known numbers of cells) as a function of time postinfection [18,22].<br><br>Determination of relative fitness of 
					.
				
The relative fitness of <span class="yellow">FMDV</span> 
					 was determined by growth competition in BHK-21 cells as previously described [7,10,11,24,56]. <span class="yellow">FMDV</span> 
					 was mixed with appropriate proportions of 
					, which was used as reference virus. This virus has a fitness 8.5-fold higher than that of the reference clone of C-S8c1 in BHK-21 cells [7,10,11,24,56]. Four serial infections were carried out at MOI of 0.1 PFU/cell. The proportion of the two competing genomes at different passages was determined by real-time reverse transcription (RT)–PCR, employing primers 5531wtnew and 
					, which are able to discriminate <span class="yellow">FMDV</span> 
					 RNA from 
					 RNA. The nucleotide sequences of the primers will be provided upon request. The fitness vector obtained for 
					 corresponded to the equation y = 0,0206e1,1074x; R2 = 0.9507. The antilogarithm (base e) of the vector slope is the fitness of the assayed virus relative to the reference virus [56].
				<br><br>cDNA synthesis, molecular cloning, and nucleotide sequencing.
Viral RNA was extracted by treatment with Trizol as previously described [57]. Reverse transcription of <span class="yellow">FMDV</span> RNA was carried out with <span class="yellow">avian myeloblastosis virus</span> reverse transcriptase (Promega, http://www.promega.com) or Transcriptor reverse transcriptase (Roche, http://www.roche.com), and PCR amplification was performed by using either Ampli-Taq polymerase (PerkinElmer, http://las.perkinelmer.com) or an Expand High Fidelity polymerase system (Roche), as instructed by the manufacturers. The <span class="yellow">FMDV</span> genome-specific oligonucleotide primers used have been previously described [22,58]. In all RT-PCR amplifications, negative amplification controls, including all reaction components except template RNA, were run in parallel to monitor absence of contamination.
Chimeric viruses containing selected regions of 
					 in the genetic background of C-S8c1 (Figure 4) were obtained by replacing the corresponding DNA fragment of pMT28 by a cDNA copy of 
					 RNA, using specific restriction sites. To obtain pMT28/
					 (436-2046), a chimera that included nucleotides 436 to 2046 of 
					 (the residue numbering of the <span class="yellow">FMDV</span> genome is as in [11]), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28 DNA digested with the same enzymes. To obtain pMT28/
					 (2046–3760), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760). To obtain pMT28/
					 (3760–5839), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839). To obtain pMT28/
					 (5839–7427), 
					 RNA was amplified by RT-PCR using primers 5531 wt new and C-Not-Pol, and then the cDNA was digested with Rsr II (5839) and Bam HI (7427). To obtain pMT28/
					 (436-3760), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and then the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–3760) DNA digested with the same enzymes. To obtain pMT28/
					 (3760–7427), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839), and ligated into pMT28/
					 (5839–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (2046–7427), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760), and ligated into pMT28/
					 (3760–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (436-7427), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2; the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–7427) DNA digested with the same enzymes. To obtain 
					/2C-3A(pMT28), pMT28 was digested with Bgl II (4201) and Rsr II (5839), and a DNA fragment including wild-type 2C-3A-coding region was inserted into pMT28/
					 (436-7427) DNA digested with the same enzymes. DNA ligation, transformation of <span class="yellow">Escherichia coli</span> DH5α, isolation of DNA from bacterial colonies, and characterization of DNA by restriction enzyme digestion were performed by standard procedures [59]. The primers used for molecular cloning and site-directed mutagenesis are described in Table S3.
				
To obtain <span class="yellow">FMDV</span> C-S8c1 containing the mutations found in gene 2C of 
					, plasmid pMT28 was subjected to site-directed mutagenesis using an oligonucleotide including the required nucleotide replacement, and 3R2New or 3CD1 as external oligonucleotide primer (Table S3; Figure 4). A DNA fragment termed A was obtained by subjecting plasmid pMT28 to site-directed mutagenesis using primers (reverse) mutSNu, mutTAu, and mutQHu (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3R2New, forward). A DNA fragment termed B was obtained amplifying pMT28 with primers (forward) mutSNd, mutTAd, and mutQHd (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3CD1, reverse). DNA fragments A and B, including the desired mutations, were recombined by shuffling PCR using equimolar amounts of DNA fragments and two external primers (3R2New and 3CD1). The DNA with the desired mutation(s) in the 2C gene was digested with Avr II (genomic position 3760) and Rsr II (position 5839), and cloned into pMT28 to generate pMT28 (SN), pMT28 (TA), and pMT28 (QH). To obtain pMT28 (SN, TA, QH), plasmid pMT28 (SN) was subjected to site-directed mutagenesis to introduce mutation T256A in a similar way as described above, and then, plasmid pMT28 (SN, TA) was subjected to site-directed mutagenesis to introduce mutation Q263H. All chimeric viruses and mutants were analyzed by nucleotide sequencing using Big Dye Terminator Cycle Sequencing kit (Abi Prism; PerkinElmer) and sequencer ABI373 as previously described [58]. Sequences were analyzed using DNASTAR 4.0 (http://www.dnastar.com), GeneDoc, and GCC (University of Wisconsin). Each sequence was determined at least twice, with products obtained using different oligonucleotide primers.
				
DNA from pMT28 or its recombinant and mutant derivatives was linearized with Nde I and transcribed with SP6 RNA polymerase as previously described [22,27]. Transcript RNA integrity and concentration were estimated by agarose gel electrophoresis, in parallel runs with known amounts of standard C-S8c1 RNA. BHK-21 cell monolayers (70% confluent, about 1 × 106 cells) were transfected with RNA transcripts (1 μg RNA) using lipofectin as previously described [59]. Virus was collected from the culture supernatant at 72 h post-transfection. The virus obtained by transfection was passaged twice before using it in biological studies. RNA was extracted and sequenced to ascertain that the virus maintained the genomic structure and mutations of the initial transcript.
Consensus genomic nucleotide sequences of <span class="yellow">FMDV</span> clones were obtained by RT-PCR amplification of virion RNA using specific primers [7,22,28].<br><br>
Supporting Information
Accession Numbers
The GenBank accession numbers for the C-S8c1, 
					, 
					, CS8p260p3d, and MARLS genomic sequences are AJ133357, AM409190, AM409325, DQ409185, and AF274010, respectively. Nucleotide and amino acid sequences for picornaviruses can be found at http://www.iah.bbsrc.ac.uk/virus/picornaviridae/SequenceDatabase/3Ddatabase/3D.HTM.
				<br><br><br><br><h3>pmcA320464</h3>Organization and structure of the <span class="yellow">mouse</span> interleukin-2 gene.
Abstract
We have cloned a chromosomal DNA segment which covers the entire sequence for the <span class="yellow">murine</span> interleukin-2 gene and analysed the structure of the gene. The coding regions are separated into four blocks by three introns each of which is located similarly to the corresponding <span class="yellow">human</span> gene. The exon sequences can be aligned perfectly with the previously cloned cDNA sequence. Of particular interests is the presence of sequences within the 5'-flanking region which are highly conserved between <span class="yellow">mouse</span> and <span class="yellow">man</span>. The conserved region which spans more than 400 base pairs may play a role in the regulation of IL-2 gene expression.Images<br><br><br><br><br><br>
 Volume 12 Number 24 1984                                   Nucleic Acids Research <br><br> Organization and structure of the <span class="yellow">mouse</span> interleukin-2 gene <br><br> Akira Fuse*, Takashi Fujita, Hidetaro Yasumitsu, Nobukazu Kashima+, Katsushige Hasegawa and Tadatsugu Taniguchi? <br><br> Department of Biochemistry, Cancer Institute, Japanese Foundation for Cancer Research, Toshimaku, Tokyo 170 and Institute for Molecular and Cellular Biology, Osaka University, Suita-shi, Osaka 565, Japan <br><br> Received 10 October 1984; Revised and Accepted 20 November 1984 <br><br> ABSTRACT <br><br> We have cloned a chromosomal DNA segment which covers the entire sequence for the <span class="yellow">murine</span> interleukin-2 gene and analysed the structure of the gene. The coding regions are separated into four blocks by three introns each of which is located similarly to the corresponding <span class="yellow">human</span> gene.    The exon sequences can be aligned perfectly with the previously cloned cDNA sequence.  Of particular interests is the presence of sequences within the 5'flanking region which are highly conserved between <span class="yellow">mouse</span> and <span class="yellow">man</span>. The conserved region which spans more than 400 base pairs may play a role in the regulation of IL-2 gene expression. <br><br> INTRODUCTION <br><br> Interleukin-2 (IL-2) is a lymphokine produced by T cells upon antigenic or mitogenic stimulation and is required for the proliferation of T cells (1,2).      Several other biological activities of IL-2 which appear to be crucial in the immune regulation have also been reported (3, 4. 5. 6. 7.) .        We previously reported isolation and sequence analysis of the cDNA for <span class="yellow">human</span> IL-2 (8), as well as the chromosomal gene (9). More recently, we have isolated a cDNA which encodes <span class="yellow">murine</span> IL-2 (Kashima et al., submitted for publication).  The cDNA contains a unique tandem repeat of CAG sequence which would encode 12 consecutive glutamine residues in the active IL-2 molecule. <br><br> In order to study the structure of the <span class="yellow">murine</span> IL-2 chromosomal gene and its controlling region, we isolated and analysed a A phage clone containing the gene and its flanking sequences. <br><br> MATERIALS AND METHODS <br><br> Southern blotting of total <span class="yellow">mouse</span> DNA <br><br> <span class="yellow">Mouse</span> chromosomal DNA was extracted from liver of BALB/c6 <br><br> ? I R L Press Limited, Oxford, England. <br><br> Nucleic Acids Research <br><br> Volume 12 Number 24 1984 <br><br> 9323 <br><br> Nucleic Acids Research <br><br> <span class="yellow">mouse</span> as described before (10). High molecular genomic DNA was digested with various restriction enzymes and electrophoresed on 0.8% agarose gel. Blotting analysis of DNA was carried out by the method of Southern (11). Hybridization was carried out as described previously and filters were washed either in 3 x SSC at 650C (lower stringent condition) or in 0.1 x SSC at 650C (higher stringent condition). <br><br> Screening of genomic DNA library <br><br> A <span class="red">bacteriophage</span> XCharon 4A/<span class="yellow">mouse</span> genomic DNA library prepared with partial EcoRI digests of <span class="yellow">mouse</span> DNA from MPC 11 plasmacytoma cells was kindly provided by Dr. T. Honjo. <span class="yellow">Mouse</span> IL-2-specific clones were screened by the method of Benton and Davis (12), using 700 bp PstI-AccI fragment of a cDNA clone, pMIL2-45   as  the probe   (Kashima   et  al.,   submitted   for publication).    Hybridization was performed as described previously(13). Positive clones were rescreened at least twice. Subcloninq and sequencing of the <span class="yellow">mouse</span> IL-2 gene <br><br> Two EcoRI fragments of 3.3 Kbp and 2.8 Kbp from the positive recombinant X phage were subcloned into EcoRI site of plasmid pBR322.  DNA segments derived from subcloned 3.3 Kbp and 2.8 Kbp fragments were labelled at either 3' end or 5' end, and subjected to sequence analysis by the chemical degradation method (14).  The 0.8 Kbp EcoRI fragment from the same X phage clone was directly subjected to sequence analysis by the dideoxy chain termination method (15). <br><br> RESULTS <br><br> Total DNA blotting analysis <br><br> In order to study structural organization of the <span class="yellow">mouse</span> IL-2 gene, we first subjected total <span class="yellow">mouse</span> DNA to the blotting analysis by using various probes specific for IL-2 gene. When <span class="yellow">mouse</span> DNA was digested with various restriction endonucleases and then probed with a 7 kb <span class="yellow">human</span> chromosomal DNA segment which contains the <span class="yellow">human</span> IL-2 gene and its flanking region (Fig. 1 lane 1-8, ref. 9.), single positive band appeared at lower but not at higher stringent condition for washing the filters (see Figure legend).    Additional bands corresponding to those observed by using <span class="yellow">mouse</span> IL-2 cDNA probes (lane 13-17) also appeared by longer <br><br> 9324 <br><br> Nucleic Acids Research <br><br> M 1 2 3 4     M 5 6 7 8      M 9 101112    M 13 141516     17 <br><br> a~ ~ 3 ..  <br><br> I pMIL2-45 SacI           Acc I <br><br> CZI~~1      1t L2- 20 <br><br> Acc I <br><br> 1 00 tp lX i <br><br> Fig. 1.  Blot hybridization analysis of <span class="yellow">mouse</span> chromosomal DNA. High molecular DNA prepared from Liver BALB/C6 <span class="yellow">mouse</span> was digested with various restriction endonucleases (BamHI for lanes 1, 5, 9, 13 ; EcoRI for lanes 2, 6, 10, 14, 17 ; HindIII for lanes 3, 7, 11, 15 ; XbaI for lanes 4, 8, 12, 16 ). The resulting digests were fractionated on 0.8 % agarose gel and transferred to a nitrocellulose filter.  Filters were hybridized by the published procedure (13) either with the nick-translated chromosomal DNA containing <span class="yellow">human</span> IL-2 gene and its flanking region (total length, 7.0 kb, ref. 9) (lane 1-8) or with the nick-translated cDNA for <span class="yellow">mouse</span> IL-2 (see figure). Filters were then washed either in 3 x SSC at 65 % (lower stringent condition) (lane 1-4, 9-12) or 0.1 x SSC at 65 OC (higher stringent condition) (lane 5-8, 13-17). Lane M each contains 7 size markers with their size being 23.7 kb,  9.5 kb,   6.7 kb,  4.3 kb,   2.3 kb,  2.0 kb and 0.6 kb, respectively.  Brief restriction endonuclease cleavage map for the <span class="yellow">mouse</span> IL-2 cDNAs is presented in the lower part of the figure. <br><br> exposure of the film (data not shown).    Those results suggest the presence of highly conserved sequences between <span class="yellow">human</span> and <span class="yellow">mouse</span> DNA either in the flanking regions or in the introns of the IL-2 gene, since the coding regions apparently show lower degree of sequence homology as evidenced in this series of blotting analysis (see below).  When the PstI insert of a <span class="yellow">mouse</span> IL-2 cDNA clone, pMIL2-20, was used as the probe, a simple pattern was <br><br> 9325 <br><br> Nucleic Acids Research <br><br> obtained at higher stringent condition (lane 13-17).   While the EcoRI-digested DNA gave rise to two positive bands (2.8 kb and 0.8 kb)(Fig. 1, lane 14) by this analysis, one additional band of 3.3 kb also appeared when the same DNA was probed with a longer cDNA insert from another clone, pMIL2-45 (Fig. 1, lane 17).  The 3.3 kb band was similar in its size with the positive band which became detectable by probing the same DNA with the 7.0 kb <span class="yellow">human</span> DNA probe (Fig. 1, lane 2). Since this band appeared with the cDNA probe extending further upstream, it is likely that the 5' region of the gene is located within this DNA segment (see below). Indeed, this 3.3 kb band did not appear even after longer exposure of lane 14 (result not shown). BamHI digest of the <span class="yellow">mouse</span> DNA (Fig. 1, lane 1, 13) constantly gave a very faint signal which would correspond to a DNA larger than 15 kb.         Taken together, the results suggested the presence of a single copy gene for <span class="yellow">murine</span> IL-2.    On the other hand appearance of the multiple positive bands at lower stringent washing condition (lane 9-12) indicates the presence of IL-2 related sequences within the <span class="yellow">mouse</span> genome. <br><br> Screening of recombinant phaqe libraries <br><br> We next screened a gene library from partial EcoRIdigested DNA from MPC 11 cells and by using 0.8 Kbp SacI-AccI cDNA fragment as the probe and isolated 14 positive clones containing sequences specific to the <span class="yellow">mouse</span> IL-2 gene. Three of the clones analysed all contained three EcoRI fragments whose size is in agreement with the result of blotting analysis of the chromosomal DNA as shown in Fig. 1 (lane 17). One of these is designated MIL-2G70. <br><br> Nucleotide sequence analysis <br><br> Two DNA fragments of 3.3 Kbp and 2.8 Kbp were excised from the phage clone MIL-2G70 by EcoRI digestion and they were subcloned into pBR322.     DNA sequences were determined for selected regions of both inserts and compared to the known cDNA sequences (Kashima et al., submitted for publication).       The strategy used for sequence analysis of the genomic DNA is presented in Fig. 2.    Comparison of the <span class="yellow">mouse</span> genomic IL-2 sequence with <span class="yellow">mouse</span> IL-2 cDNA sequence revealed that, like the <span class="yellow">human</span> gene, the gene is divided into four exons.     A putative <br><br> 9326 <br><br> Nucleic Acids Research <br><br> E               E           E    E   E <br><br> 500bp s    7S P HBH  H   \A <br><br> 200 bp  *     ,            -   4<br><br> I             4-4----*    *-    --4 <br><br> Fig. 2. Restriction map and sequencing strategy of <span class="yellow">mouse</span> IL-2 gene. Horizontal lines indicate the length of <span class="yellow">mouse</span> DNA inserted into the X phage Charon 4A or plasmid subclones.   Filled blocks, dashed blocks and open blocks indicate protein coding regions, untranslated regions and introns, respectively. Horizontal arrows indicate the direction and extent of sequence determination without ambiguity.   Dashed arrows:   determination was done by the chain termination method (15) after subcloning the       0.8 kb fragment into M13.       Rest of the sequence determination was      carried out by the method of Maxam and Gilbert (14).   A AccI site, B; BamHI site, E; EcoRI site, H; HindIII site, P; PstI site, S; SacI site. <br><br> capping site or the transcription initiation site was located 32 bp downstream from a TATAAA consensus promotor sequence (Fig. 3). The first ATG triplet was located 79 bp downstream from the TATA box. As seen also in the <span class="yellow">murine</span> IL-2 cDNA, there is an unusual repeat of CAG triplet coding for 12 glutamine residues in a row in the first exon. The second exon (60 bp) is separated from the first exon by a short intron consisting of 97 bp.     The second, the third and the fourth exons are interrupted by longer introns whose size is about 2.3 Kbp and 1.6 Kbp, respectively.  As far as the available sequence data are concerned, it seems that, despite their identical location, intron sequences are distinctly dissimilar except for the junction regions between the <span class="yellow">human</span> and <span class="yellow">mouse</span> IL-2 genes.    There are two potential poly (A) addition signals within the <span class="yellow">mouse</span> gene (nucleotide positions 793 - 798 and 924 - 929 in Fig. 3 ) and, based on our sequence data for various cDNA clones, both signals seem to function and give rise to heterogeneous termini of the mRNA in the LBRM-33 cells (16). <br><br> We have also determined the sequence of about 500 bp of 5'-flanking region of <span class="yellow">mouse</span> IL-2 gene, since (i) promoter/regulatory sequences are located in this region in many other genes of eukaryotes and (ii) this region appeared to contain sequences which show strongest cross-hybridization between <span class="yellow">human</span> and <span class="yellow">mouse</span> DNA around the IL-2 gene (Fig. 1). Comparison of the nucleotide sequences for the 5' flanking region <br><br> 9327 <br><br> Nucleic Acids Research <br><br> -4.0 <br><br> TAGGAGGTAAACCATCTCGAAACGGAAACCAATATCCTTCCTGTCTAATCAACAAATCTAAAG -400                                               -350 <br><br> GATTTATTCTTTTCATCTATCTCCTCTTGCGCCCGTCCACCACAACAGGCTGCTTACAGGTTCAGGATGGTTTTGACAAAGAGAACATTTT <br><br> -300                                               -250 <br><br> CATGAGTTACTTTTGTGTCTCCACCCCAAAGAGGAAAATTTTGTTCATACGAAAGGCGTTCATTGTATGAATTAAAACTGCCACCTAAGAG <br><br> -200                                               -150 <br><br> TGGGCTAACCCGACCAAGAGGGATTTCACCTAAATCCATTCAGTCAGTATATGGGGTTTAAACAAATTCCAGAGAGTCATCAGAAGAGGAA <br><br> -100                                                -50 <br><br> AAACAAAGGTAATACTTTCTGCCACACAGGTAGACTCTTTTGAAAATATGTGTAATATGTAAAACATCGTGACACCCCCATATTATTTTTC CAGCATTAACAGTATAAATTGCCTCCCATGCTGAAGAGCTGCC TCACCCTTGCTAATCACTCCTCACAGTGACCTCAAGTCCTGCAGGC <br><br> 50                                                                   100 <br><br> ATG TAC AGC ATG CAG CTC GCA TCC TGT GTC ACA TTG ACA CTT GTG CTC CTT GTC AAC AGC GCA CCC ACT Met Tyr Ser Met Gln Leu Ala Ser Cys Val Thr Leu Thr Leu Val Leu Leu Val Asn Ser Ala Pro Thr <br><br> 150 <br><br> TCA AGC TCC ACT TCA AGC TCT ACA GCG GAA GCA CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG Ser Ser Ser Thr Ser Ser Ser Thr Ala Glu Ala Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln <br><br> 200 <br><br> CAC CTG GAG CAG CTG TTG ATG GAC CTA CAG GAG CTC CTG AGC AGG ATG GAG    GTAAGTGCACAGCCATCCCATC His Leu Glu Gln Leu Leu Met Asp Leu Gln Glu Leu Leu Ser Arg Met Glu <br><br> TATAGGCAATACCTTTAGCTTTCTTGCCAAAGGCTGTGTTTAATAACCTTTAATAATAATGTTACGCTTTCTCAG AAT TAC AGG AAC <br><br> lAsn Tyr Arg Asn 250 <br><br> CTG AAA CTC CCC AGG ATG CTC ACC TTC AAA TTT TAC TTG CCC AAG CAG GTGAGTGAGTTTCTGTTTAACTGGTGC Leu Lys Leu Pro Arg Met Leu Thr Phe Lys Phe Tyr Leu Pro Lys Gln <br><br> TCTAATG----------------------------------------------------------__ <br><br> ?_______________----------------------- 2000 bp --------------------?---------------------------------------------------------- AATGTGAACCTTGTAGTTTCTTTGTAGATTGGAACAATAGTCTGAACTTGTGT TATGCATTGGTAGAGAAACACAGACTTTTTACAAAGTCTAATGAAGCCAGAAGAGCTCACTAGAGTGGCCTAACAGTTAAGGTGACCCTTA TGGTTGTGAGCTCTTGCTCTTCTAGATTTATGGCATCGATTACCTCAGTCCCCCTTTACAGAGGACAGGGAGTGGTAAAAGCTATGTGCTG CCTTCTCTTGATTAGAGAGACTGCAGACTAACTTTCTGGCTCTTCAGTATGTGGTGAGCTGAGCTGATGGTTAAGCTTATTACTCCTCTAG <br><br> 300                                                                 350 <br><br> GCC ACA GAA TTG AAA GAT CTT CAG TGC CTA GAA GAT GAA CTT GGA CCT CTG CGG CAT GTT CTG GAT TTG Ala Thr Glu Leu Lys Asp Leu Gln Cys Leu Glu Asp Glu Leu Gly Pro Leu Arg His Val Leu Asp Leu <br><br> 490 <br><br> ACT CAA AGC AAA AGC TTT CAA TTG GAA GAT GCT GAG AAT TTC ATC AGC AAT ATC AGA GTA ACT GTT GTA Thr Gln Ser Lys Ser Phe Gln Leu Glu Asp Ala Glu Asn Phe Ile Ser Asn Ile Arg Val Thr Val Val AAA CTA AAG GTAAGGTGTTGCTTTATTTGCTAATCTGGAAATAAAATAGAGAAGAAATGCATTTTTAAGTGGCTTGCCATTTCTGGT Lys Leu Lys <br><br> CTTTGATGGGTTCTGTGCATTTAGTCAACCAAAGTTTAAAGTCACTGTGCAAGTGAATCC --------------------------------------------------------------------    -1000   bp  ------ --------------------------------------------------------------- GAATTCTACAGAAGTGTTCAGTGTTCCCATCAAATGCTCGTTGGTAATCAACTCTGGAGAGCTT TATTTTTTATGCTTTACCATTTGATATCATAATAAAACATAACCCAATAATTGTGATCATTTCAGAAATGTGAATGTTCAATATATTTAAC <br><br> CAGTGTTAAAATAAATGCCTAAAAGCTCCGTTGAAGAATAATTATATGCAAAGTAAGCTACCTTAGCCTACAATTTTATATTCTTTTTTAG <br><br> 450                                                                 500 <br><br> GGC TCT GAC AAC ACA TTT GAG TGC CAA TTC GAT GAT GAG TCA GCA ACT GTG GTG GAC TTT CTG AGG AGA Gly Ser Asp Asn Thr Phe Glu Cys Gln Phe Asp Asp GlU Ser Ala Thr Val Val Asp Phe Leu Arg Arg <br><br> 550 <br><br> TGG ATA GCC TTC TGT CAA AGC ATC ATC TCA ACA AGC CCT CAA TAACTATGTACCTCCTGCTTACAACACATAAGGC Trp Ile Ala Phe Cys Gln Ser Ile Ile Ser Thr Ser Pro Gln <br><br> 600                                                650 <br><br> TCTCTATTTAT4TAAATATTTAACTTTAATTTATTTTTGGATGTATTGTTTACTATCTTTTGTAACTACTAGTCTTCAGATGATAAATATG <br><br> 700                                                750 <br><br> GATCTTTAAAGATTCTTTTTGTAAGCCCCAAGGGCTCAAAAATGTTTTAAACTATTTATCTGAAATTATT+ATTATATTGAATTGTTAAAT <br><br> < 4>800                                                 850 <br><br> ATCATGTGTAGGTAGACTcATTCA TAAAAGTAmAGATGATTCAAATATAAATAAGCTCAGATGTCTGTCATTTTTAGGACAGCACAAAG <br><br> 900                         0" <br><br> TAAGCGCTAAAATAACTTCTCAGTTATTCCTGTGAACTcTATGTTAATCAGTGTTTTCAAGAAATAAAGCTCTCCTCT           AAA GATGGCTTGTGGGAAAAGATCTCCTCTCCAGGGAGCTAACATCAGCTCAGAGTTTACTCAAGAATTC <br><br> Fig.      3.      Nucleotide sequence of <span class="yellow">mouse</span> IL-2 gene.                               Four exons are framed.          Numbers refer to nucleotide positions of exons from the presumed cap site. Dots and open circles indicate TATA box and poly A additional signals, respectively. <br><br> 9328 <br><br> Nucleic Acids Research <br><br> -4 5 0                               -400 <br><br> TAGGAGGT AAACCA T CTCGAAAC GGAAACCAATATCCTTCCTGTCTAATCAACAAATCTAAAGGATTTATTCTTTTCATCTATCTCC <br><br> .0     * .0 .- *. * ..* --.  -- .*-@ .. ............  --  -*-* * <br><br> TAAAAAGGTAAAACCAGTTCT GAAACAGGAAACCAATACACTTCCTGTTTAATCAACAAATCTAAA CATTTATTCTTTTCATCTGTTTAC <br><br> -350                         -300 <br><br> TCTTGCGCCCGTCCACCACAACAGGCTGCT T ACAGGTTCAGGATGGTTTTGACAAAGAGAACATTTTCATGAGTTACTTTTGTGTCTC TCTTGCTCTTGTTCACCACAATA  TGCTATTCACATGTTCAGTGTAGTTTTATGACAAAGAAAATTTTC TGAGTTACTTTTGTATCCC <br><br> -250 <br><br> CA CCCC  AAAG  AGGAAAATTTGTTTCATACGAAAGGCGTTCATTGTATGAATTAAAACT CCACCTAAGAGTGGGCTAACCCGA <br><br> *~~~ ~~   ..... *   .-- *..-  * .-- .**  . *-.   --.   *- *- -  -  *-   0 @  *-- 0 .. .   <br><br> CACCCCCTTAAAGAAAGGAGGAAAAACTGTTTCATACAGAAGGCGTTAATTGCATGAATTAGAGCTATCACCTAAGTGTGGGCTAATGTAA <br><br> -200                          -150 <br><br> CCAAGAGGGATTTCACCTAAATCCATTCAGTCAGTATAT GGGGTTTAAACAAATTCCAGAGAGTCATCAGAAGAGGAAAAACAAAGGTAA CAAAGAGGGATTTCACCTACATCCATTCAGTCAGTCTTTGGGGGTTTAAA AAATTCCAAAGAGTCATCAGAAGAGGAAAAATGAAGGTAA <br><br> -100                          -50 <br><br> TACTTTCTGCCACACAGGTAGACTCTTTTGAAAATATGTGTXATATTAAAACATCGTGACACCCCCATATiXrTCCAGCATTAACAG TGTTTTTT CAGACAGGTAAAGTC TTTGAAAATATGTGTAATATGTAAAACATTTTGACACCCCCATAATATTTTTCCAGAATTAACAG ATAAGCCTCCCATGCTGAAGAGCTGCCTATCACCCTTGCTAATCACTCCTCACAGTGACCTCAAGTCCTGCAGGCATG  <span class="yellow">Mouse</span> rTATGCATCTCTTGTTCAAGAGTTCCCTATCACTCTCTTTAATCACTACACACAGTAACCTCAACTCCTGCCACTATG  <span class="yellow">Human</span> <br><br> Fig.  4.   Comparison of 5'-flanking regions of <span class="yellow">mouse</span> and <span class="yellow">human</span> IL-2 genes.   In aligning the sequences for both genes, gaps were introduced   to maximize   homology.    Dots   indicate  identical nucleotide sequences.   Number 1 indicates putative transcription initiation site. Inverted repeats are indicated by bars and dashed lines. TATA box is framed. <br><br> of both genes is illustrated in Fig. 4.    The nucleotide sequence homology from the TATA box to -470 is 85%. The highest region of homology was observed between the TATA box and position -97 (60 bp matches out of 64 bp) of both genes. DISCUSSION <br><br> We have isolated recombinant clones for <span class="yellow">mouse</span> IL-2 gene from a phage Charon 4A/<span class="yellow">mouse</span> genomic DNA library and determined the entire sequence of the gene except for the sequence of the internal portion of the second and third introns.         The <span class="yellow">mouse</span> IL-2 cDNA sequence was aligned with the genomic sequence and both sequences matched completely each other. The unusual CAG repeats encoding 12 glutamines which was found previously in the cloned cDNA was shown to be present also in the chromosomal gene. This finding further excludes the possibility that the unique repeat is generated by artifacts during the cDNA cloning process. Although we can not rule out the possibility for the deletion of this sequence in the <span class="yellow">human</span> IL-2 gene, it is more likely that the CAG repeat has been generated in the <span class="yellow">mouse</span> genome rather recently.    The repeat could have been generated either by a direct insertion of the sequence or by the duplication after insertion of a unit sequence. <br><br> 9329 <br><br> Nucleic Acids Research <br><br> Organization of the <span class="yellow">mouse</span> IL-2 gene resembles to that of the <span class="yellow">human</span> gene (Fig. 2., ref. 9).   There seems to be little sequence homology between corresponding introns of <span class="yellow">mouse</span> and <span class="yellow">human</span> IL-2 genes, except for the intron-exon junctions part of which is thought to be necessary for the RNA splicing (17, 18). Dissimilarity of the intron sequences among the genes which are derived from a common ancestor has been reported in other genes (19, 20).   In spite of the divergence in sequence of introns, the size and position of the introns are very similar between the <span class="yellow">murine</span> and <span class="yellow">human</span> IL-2 genes. <br><br> Of particular interests is the presence of highly conserved sequences in the 5' -flanking region of the <span class="yellow">human</span> and <span class="yellow">mouse</span> IL-2 gene (Fig. 4).    Whereas the coding region shows nucleotide sequence homology of 72% between the two genes, the 5' upstream region spanning about 500 bp (Fig. 4) shows 85% homology which was readily detectable by the blotting analysis (Fig. 1, lane 1-4).  Since we have not yet determined the nucleotide sequence further upstream of the <span class="yellow">mouse</span> gene, we do not know whether or not this similarity extends further.  It is  likely that such sequences are involved in the controlled expression of the IL-2 genes in activated T-lymphocytes.  Work is in progress to identify such DNA sequences by introducing the cloned genes into various lymphocytic cell lines.   Our preliminary results indicate that the 5 '-flanking sequence of the <span class="yellow">human</span> IL-2 gene mediates mitogen induced expression of the gene in T-lymphocytic cells (Fujita & Taniguchi, unpublished observation). <br><br> ACKNOWLEDGEMENTS <br><br> We thank Dr. T. Honjo for <span class="yellow">mouse</span> gene library.    We are also indepted to Ms. M. Nagatsuka for typing the manuscript. This work was supported in part by Grant-in-Aid for Special Project Research, Cancer-Bioscience from the Ministry of Education, Science and Culture, Japan. <br><br> ?To whom correspondence should be addressed <br><br> *Present address: Department of Microbiology, School of Medicine, Chiba University, Chiba 280, Japan <br><br> +Present address: Central Research Laboratory, Ajinomoto Co.Inc., Totsuka-ku, Yokohama 244, Japan <br><br> 9330 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> 1. Morgan, D. A., Ruscetti, F. W. and Gallo, R. (1976) Science, <br><br> 193, 1007-1008. <br><br> 2. Gil lis, S. Ferm, M. M., Ou, W. and Smith, K. (1 978) J. <br><br> Immunol., 120, 2023-2027. <br><br> 3. Chen, B. M. and Di Sabato, G. (1976) Cell. Immunol., 22, 211<br><br> 224. <br><br> 4. Henney, C. S., Kuribayashi, K., Kern, D. E. and Gillis, S. <br><br> (1981) Nature, 291, 335-338. <br><br> 5. Wagner, H., Hardt, C.,   Heeg, K., Rollinghoff, M. and <br><br> Pfizenmaier, K. (1980) Nature, 284, 278-280. <br><br> 6. Farrar, J. J., Benjamin, W. R., Hilfikr M. L., Howard, M., <br><br> Farrar,W. L. and Fuller-Farrar, J. (1982) Immunol. Rev. 63, 129-166. <br><br> 7. Pearlstein, K., Palladino, M. A., Welte, K. and Vilcek, J. <br><br> (1983) Cell. Immunol., 80, 1-9. <br><br> 8. Taniguchi, T., Matsui, H., Fujita, T., Takaoka, C., Kashima, <br><br> N., Yoshimoto, R. and Hamuro, J. (1983) Nature 302, 305-310. <br><br> 9. Fujita,T.,Takaoka,C., Matsui, H. and Taniguchi T. (1983) <br><br> Proc. Natl. Acad. Sci. USA 80, 7437-7411. <br><br> 10. Ohno, S. and Taniguchi, T. (1982) Nucleic Acids Res. 10, 967<br><br> 977. <br><br> 11. Southern, E. M. (1975) J. Mol. Biol. 98, 503-517. <br><br> 12. Benton, W. D. and Davis, R. W. (1977) Science 196, 180-182. <br><br> 13. Ohno, S. and Taniguchi, T. (1981) Proc. Natl. Acad. Sci. USA <br><br> 78, 5305-5309. <br><br> 14. Maxam, A. M. and Gilbert, W. (1980)  Meth. Enzymol. 65, 560<br><br> 580. <br><br> 15. Sanger, F., Nicklen, S. and Coulson, A. R. (1977) Proc. Natl. <br><br> Acad. Sci. USA, 74, 5463-5467. <br><br> 16. Gillis, S., Scheid, M. and Watson, J. D.(1980) J. Immunol. <br><br> ,125, 2570-2580. <br><br> 17. Breathnach, R., Benoist, C., O'Hare, K., Gannon, F. and <br><br> Chambon, P.(1978) Proc. Natl. Acad. Sci. USA, 75, 4853-4857. <br><br> 18. Lerner, M. R., Boyle, J. A., Mount, S. M., Wolin, S. L. and <br><br> Steitz, J. A. (1980) Nature, 283, 220-224. <br><br> 19. Van Ooyen, A., Van den Berg, J., Mantei, N. and Weissmann, <br><br> C.(1979) Science, 206, 337-344. <br><br> 20. Searle, P. F., Davison, B. L., Stuart, G. W., Wilkie, T. M., <br><br> Norstedt, G. and Palmiter, R. D. (1984) Mol. Cell. Biol. 4, 1221-1230. <br><br> 9331 <h3>pmcA1885664</h3>The recombination-associated protein RdgC adopts a novel toroidal architecture for DNA binding
Abstract
RecA plays a central role in the nonmutagenic repair of stalled replication forks in bacteria. RdgC, a recombination-associated DNA-binding protein, is a potential negative regulator of RecA function. Here, we have determined the crystal structure of RdgC from <span class="yellow">Pseudomonas aeruginosa</span>. The J-shaped monomer has a unique fold and can be divided into three structural domains: tip domain, center domain and base domain. Two such monomers dimerize to form a ring-shaped molecule of approximate 2-fold symmetry. Of the two inter-subunit interfaces within the dimer, one interface (‘interface A’) between tip/center domains is more nonpolar than the other (‘interface B’) between base domains. The structure allows us to propose that the RdgC dimer binds dsDNA through the central hole of ∼30 Å diameter. The proposed model is supported by our DNA-binding assays coupled with mutagenesis, which indicate that the conserved positively charged residues on the protein surface around the central hole play important roles in DNA binding. The novel ring-shaped architecture of the RdgC dimer has significant implications for its role in homologous recombination.<br><br>INTRODUCTION
Homologous recombination systems contribute to the maintenance of genome integrity and are essential for the repair of stalled replication forks (1). Genetic studies of the recombination-dependent growth C (rdgC) gene in bacterial systems have suggested that it is involved in DNA replication and recombination (2,3). In <span class="yellow">Neisseria gonorrhoeae</span> (the gonococcus), RdgC is required for efficient pilin antigenic variation and plays some role in cell growth (4,5). The <span class="yellow">Escherichia coli</span> rdgC gene encodes a DNA-binding protein of 34 kDa, whose expression level was at its highest during exponential phase, reaching its maximum at ∼1000 dimers per cell (2). Its level decreased sharply to ∼50 dimers per cell in stationary phase. This profile suggests that RdgC might function during the period of DNA replication (2).
The RecA proteins and their homologs play a key role in recombinational DNA repair in bacteria and eukaryotes. Their aberrant reactions could result in gross chromosomal rearrangements that lead to <span class="yellow">human</span> diseases, including cancer. Therefore, their functions must be tightly regulated, and understanding how the RecA family of recombinases is regulated is of utmost importance (1). The <span class="yellow">E. coli</span> RdgC protein is a potential negative regulator of the function of RecA (1). It inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). Sedimentation equilibrium data indicated that <span class="yellow">E. coli</span> RdgC exists in solution as a mixture of oligomeric states in equilibrium, most likely monomers, dimers and tetramers (1). This concentration-dependent change in the oligomeric state appears to affect its mode of binding to DNA and its capacity to inhibit RecA (1). The primary mechanism of RdgC inhibition appears to involve a simple competition for DNA-binding sites, especially on double-stranded DNA (dsDNA) (1).
Deletion of the <span class="yellow">E. coli</span> rdgC gene alone causes no obvious phenotype but is highly deleterious in strains lacking certain enzymes involved in recombination and replication restart (2,3). RdgC is essential for the growth of an <span class="yellow">E. coli</span> strain lacking PriA, indicating that it might affect replication fork progression or fork rescue (2). PriA provides a means to load the DnaB replicative helicase at DNA replication fork and D loop structures (2). RdgC is, therefore, a key factor in the rescue of stalled or broken forks and subsequent replication restart. dnaC suppressors of PriA can overcome this inviability, especially when RecF, RecO or RecR is inactivated, which indicates that RdgC avoids or counters the toxic effect of these proteins by limiting inappropriate RecA loading on SSB-coated ssDNA (2).
The DNA-binding ability of the RdgC protein has been demonstrated clearly using electrophoretic mobility shift assays (2). <span class="yellow">Escherichia coli</span> RdgC binds non-specifically to dsDNA and with higher affinity than ssDNA (2). RdgC from <span class="yellow">N. meningitidis</span> also binds DNA with little specificity for sequence or structure, like the <span class="yellow">E. coli</span> protein (4). RdgC exhibits no preference for DNA replication or recombination intermediates (2). However, no detectable domains or nuclease activity could be identified for the <span class="yellow">E. coli</span> RdgC protein (2,6). The complexes of <span class="yellow">E. coli</span> RdgC with both linear and supercoiled circular plasmid DNA were imaged using atomic force microscopy (7). RdgC was found to have an increased affinity to DNA ends and to promote bending of DNA (7). RdgC has the effect on DNA superstructure; the promotion of DNA condensation was observed at high protein concentrations (7). Recombination is largely enhanced by close contacts of distant regions along the DNA strands through condensation (7).
<span class="yellow">Pseudomonas aeruginosa</span> is a ubiquitous environmental Gram-negative bacterium that belongs to the γ subdivision of the Proteobacteria. Orthologs of the rdgC gene are found only in β and γ subdivisions of the Proteobacteria (2). The amino acid sequence of <span class="yellow">P. aeruginosa</span> RdgC, a 306-residue protein, shows 46 and 36% identities against those of <span class="yellow">E. coli</span> and <span class="yellow">N. meningitidis</span>, respectively. Despite the importance of DNA-binding ability of RdgC in its biological roles, the structural details of RdgC and the mechanism of its action remain unclear. Here, we report the crystal structure of <span class="yellow">P. aeruginosa</span> RdgC as determined by the multiwavelength anomalous diffraction (MAD) method of X-ray crystallography. It reveals that RdgC dimer is ring-shaped, with a central hole of ∼30 Å diameter. The inside surface around the central hole is rich in conserved, positively charged residues, which are implicated in DNA-binding by mutagenesis. We propose that dsDNA binds to RdgC through this central hole. Our structure provides a solid structural framework for a better understanding of the role of RdgC in homologous recombination.<br><br>MATERIALS AND METHODS
Protein expression and purification
The rdgC gene (PA3263) was amplified by the polymerase chain reaction (PCR) using the genomic DNA of <span class="yellow">P. aeruginosa</span> strain PAO1 as a template. The oligonucleotide primers designed using the published genome sequence (8) were 5′-G GAA TTC CAT ATG TGG TTC CGC AAT CTG CTC G-3′ (forward) and 5′-CCG CCG CTC GAG GAC GCC CTG GGG GAT TTC TTC-3′ (reverse). The bases in bold denote the NdeI and XhoI cleavage sites, respectively. The PCR product was digested with NdeI and XhoI, and was then inserted into the NdeI/XhoI-digested expression vector pET-21a(+) (Novagen). This vector construction adds an eight-residue tag (LEHHHHHH) to the carboxyl terminus of the gene product to facilitate protein purification.
The protein was expressed in <span class="yellow">E. coli</span> C41 (DE3) cells (9). The cells were grown at 37°C up to OD600 of 0.6 in Luria–Bertani medium that contained 50 μg ml−1 ampicillin, and the protein expression was induced by 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG). The cells were allowed to grow at 20°C for 22 h after IPTG induction and were harvested by centrifugation at 4200 g (6000 rev min−1; Sorvall GSA rotor) for 10 min at 4°C. The cell pellet was resuspended in an ice-cold lysis buffer [20 mM Tris-HCl (pH 7.9), 500 mM sodium chloride, 5 mM imidazole] and was then homogenized with an ultrasonic processor. The crude cell extract was centrifuged at 36 000 g (18 000 rev min−1; Hanil Supra 21 K rotor) for 60 min at 4°C, and the recombinant protein in the supernatant fraction was purified in three chromatographic steps.
The first purification step utilized the C-terminal hexahistidine-tag by Ni2+-chelated HiTrap chelating column (GE Healthcare). The eluent was diluted 3-fold with 50 mM Tris-HCl at pH 7.4. The diluted solution was applied to a 20-ml heparin-Sepharose column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol and 1 mM EDTA. The protein was eluted with a linear gradient of 0–2.0 M sodium chloride in the same buffer. The first peak corresponded to high molecular aggregates of RdgC and the second peak to RdgC dimers, which was also consistent with our dynamic light scattering measurements. When the aggregates were not separated from the dimers, the protein readily aggregated within a day. The dimer fractions were concentrated to ∼6 mg ml−1 concentration using an YM10 ultrafiltration membrane (Millipore-Amicon) and further purified by gel filtration on a HiLoad XK 16 Superdex 200 prep-grade column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 200 mM sodium chloride, 1 mM dithiothreitol and 1 mM EDTA. The first peak corresponding to the high molecular aggregates of RdgC was only a minor component, and the second peak corresponding to RdgC dimers was stable for several weeks when frozen at −70°C. The procedure for purifying the SeMet-substituted protein was the same, except for the presence of 10 mM dithiothreitol in all buffers used during purification steps. Dynamic light scattering experiments were performed at 24°C, with the protein (at 2 mg ml−1 concentration) dissolved in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride on a Model DynaPro-801 instrument (Wyatt, Santa Barbara, CA, USA). The protein concentration was estimated by measuring the absorbance at 280 nm, employing the calculated extinction coefficient of 20 910 M−1 cm−1 (SWISS-PROT; http://www.expasy.ch/).<br><br>Mutagenesis and electrophoretic mobility shift assay
Point mutants were prepared using the QuikChange Site-Directed Mutagenesis kit (Stratagene), and the mutations were confirmed by DNA sequencing. The seven mutants R4A, F120A, K146A, R198A, R208A, Q212A and R252A were well expressed and soluble. The three mutants K70A, R81A and R211A were not expressed, while R122A mutant was expressed but was not soluble. Therefore, we mutated Lys70, Arg81, Arg122 and Arg211 into aspartate. The K70D, R81D, R122D and R211D mutants were well expressed and soluble. The soluble mutants were expressed and purified under the conditions identical to those for the wild type, except that the heparin-Sepharose column step was omitted. All of the soluble mutants had similar elution profiles as the wild type upon gel filtration, and the dimer fractions were used for DNA-binding assays. We mixed 5 μl of RdgC (1.1 μg μl−1) and 5 μl of a linear dsDNA with 414 bp (63 ng μl−1), corresponding to the molar ratio of 64:1 for RdgC dimer to dsDNA. The reaction mixtures were incubated for 10 min at room temperature. RdgC–DNA complexes were analyzed by 1.5% (w/v) agarose gel electrophoresis in 1× TAE buffer (45 mM Tris-acetate at pH 8.3, 1 mM EDTA). DNA bands were visualized by ethidium bromide staining. For DNA-binding assays, the protein concentration was measured with a Bio-Rad protein assay kit.<br><br>Crystallization and X-ray data collection
Crystals were grown by the hanging-drop vapor diffusion method at 24°C by mixing equal volumes (2 μl each) of the protein solution (at 19 mg ml−1 concentration in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride) and the reservoir solution. To grow the native crystals, we used a reservoir solution consisting of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate and 1% (w/v) Anapoe®35 as the detergent. Native crystals grew to approximate dimensions of 0.2 mm × 0.1 mm × 0.1 mm within a few days. The SeMet-substituted protein was crystallized under conditions identical to those for the native crystals except for the presence of 10 mM dithiothreitol in the protein solution.
A crystal of the SeMet-substituted protein was dipped into a cryoprotectant solution for a few seconds and was flash-cooled in the cold nitrogen gas stream at 100 K. The cryoprotectant solution consisted of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate, 1% (w/v) Anapoe®35 as the detergent and 10% (v/v) glycerol. A set of Se MAD data was collected at 100 K at three different wavelengths using an Area Detector System Corporation Quantum 210 charge-coupled device detector at the beamline NW12A of Photon Factory, Tsukuba, Japan. The crystal was rotated through a total of 180° with a 1° oscillation range per frame. The raw data were processed and scaled using the program suit HKL2000 (10). The SeMet-substituted crystal belongs to the space group P21212, with unit cell parameters of a = 87.04 Å, b = 115.60 Å and c = 76.07 Å.
The native crystals were soaked for 10 min in 10 mM ethyl mercury thiosalicylate (EMTS) to collect a set of Hg MAD data at four different wavelengths. X-ray diffraction data from the EMTS-derivatized crystal, as well as a native crystal, were collected on a Bruker charge-coupled device detector at the beamline BL-6B of Pohang Light Source, Pohang, Korea. The EMTS-soaked crystal belongs to the space group P21212, with unit cell parameters of a = 87.16 Å, b = 116.02 Å and c = 74.97 Å. The native crystal belongs to the space group P21212, with unit cell parameters of a = 87.49 Å, b = 116.04 Å and c = 76.03 Å. If the presence of a dimeric molecule in the asymmetric unit is assumed, the calculated crystal volume per protein mass (VM) is 2.75 Å3 Da−1 and the solvent content is 55.3%. Supplementary Table 1 summarizes the statistics of X-ray diffraction data collection.<br><br>Structure solution and refinement
Heavy atom sites were located with the program SOLVE (11). Four Se sites from the SeMet-substituted crystal and two Hg sites from the EMTS-soaked crystal were located. Two sets of the initial Se and Hg MAD phases were separately improved using the program RESOLVE (12). Two electron density maps were independently interpreted, and the fragments of the two models were combined into a single model, since the two maps were complementary to each other. Non-crystallographic symmetry (NCS) matrices were found from a partial model built from the initial electron density map, and phases were further improved by the 2-fold NCS averaging, solvent flattening and histogram matching with the program DM (13). The model was built with O (14). The model was refined with the program CNS (15), including the bulk solvent correction. 10% of the data were randomly set aside as the test data for the calculation of Rfree (16). Several rounds of model building, simulated annealing, positional refinement and individual B-factor refinement were performed. Subsequently, this model was used to refine the structure of the native protein. The model has excellent stereochemistry, as evaluated by the program PROCHECK (17). Refinement statistics are summarized in Supplementary Table 1.<br><br>
RESULTS AND DISCUSSION
Model quality and monomer structure
The structure of <span class="yellow">P. aeruginosa</span> RdgC was solved using two sets of MAD data collected from a crystal of the selenomethionine (SeMet)-substituted protein and from a mercury derivative crystal of the native protein (Supplementary Table 1). The model has subsequently been refined using the 20.0–2.50 Å data from a native crystal to crystallographic Rwork and Rfree values of 23.4 and 27.9%, respectively, with no sigma cut-off. The refined model contains 612 residues of the two monomers in the asymmetric unit of the crystal (residues 1–306 for both chains A and B) and 87 water molecules. The C-terminal eight-residue fusion tag has no electron density in both subunits and has not been modeled. Conformations of the two monomers in the asymmetric unit are similar. The root-mean-square (r.m.s.) difference between the two monomers is 0.69 Å for 306 Cα atom pairs, with a maximum deviation of 8.8 Å for the Cα atom of Gly305. The residues giving r.m.s. differences larger than 2.0 Å are 14–16, 94–96, 201–202 and 303–306, which correspond to loops connecting secondary structure elements or the C-terminus. They are located on the surface of the dimer. Subunit A has a better map quality than subunit B, and thus has a lower mean B-factor than subunit B (40.3 Å2 versus. 56.1 Å2 for all 4788 atoms) (Supplementary Figure 1). Therefore, subunit A is chosen for discussions, unless otherwise stated.
The <span class="yellow">P. aeruginosa</span> RdgC monomer is J-shaped and has approximate dimensions of 72 Å × 60 Å × 40 Å (Figure 1). It can be divided into three structural domains: center domain, tip domain and base domain. Center domain (residues 1–73 and 121–167) includes a six-stranded anti-parallel β-sheet with two flanking α helices (α1 and α4). The helix α1 runs nearly parallel to the two outermost strands β2 and β3, which in turn lie adjacent to β4. Strands β2 and β3 are separated by a loop. The helix α4 is nearly orthogonal to the β-strands. Tip domain (residues 74–120) contains two α helices (α2 and α3) and is inserted between strands β4 and β5 of the center domain. It sticks out from the center domain. The two connections between the center domain and the tip domain are rich in conserved residues, hinting at an important functional role of these joints. They could act as a hinge for a possible conformational change. The C-terminal region of the polypeptide chain is folded into the base domain (residues 168–306), which consists of a five-stranded anti-parallel β-sheet with four α-helices, three of which (α5, α6 and α8) cover one side of the β-sheet. The long helix α8 is slightly bent.<br><br>
Structural similarity searches
When we searched for structural similarity against the Protein Data Bank database using the program DALI (18), the <span class="yellow">P. aeruginosa</span> RdgC monomer showed no significant similarity with a Z score above 5. Therefore, we conclude that the overall fold of the RdgC polypeptide chain is unique. When we elaborated the structural comparisons with individual domains of <span class="yellow">P. aeruginosa</span> RdgC, only the center domain gave a Z score above 5.
Using the center domain (residues 1–73 and 121–167) alone, the highest structural similarity is found with the <span class="yellow">human</span> β2-adaptin appendage domain (PDB code 1E42, Z score = 7.1, r.m.s. deviation = 2.5 Å for 85 structurally aligned residues, residues 1–12, 14–30, 44–47, 58–72, 122–133 and 135–160 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 838–841, 848–880, 885–906 and 912–967 of the <span class="yellow">human</span> β2-adaptin appendage domain chain A) (19). There does not appear to be any functional relatedness between these two proteins. The next highest similarity is found with the <span class="yellow">yeast</span> TATA-box-binding protein (PDB code 1YTB, Z score = 5.7, r.m.s. deviation = 2.6 Å for 79 structurally aligned residues, residues 4–12, 14–19, 21–27, 56–70, 124–132 and 134–166 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 69–96, 100–126 and 132–155 of TATA-box-binding protein, N-terminal half of chain A) (20). The TATA-box-binding protein is composed of a ten-stranded antiparallel β-sheet with four flanking α-helices on the convex side of the β-sheet (20). It has an internal quasi 2-fold symmetry, and its β-strands are involved in DNA binding. There is, however, no structural similarity of functional significance, because the two center domains of the RdgC dimer are well separated and do not associate into a single unit (Figure 1C and D).
Using the tip domain (residues 74–120) alone, the ATPase domain of <span class="yellow">bovine</span> Hsc70 chaperone shows highest structural similarity (PDB code 1BA1, Z score = 4.8, r.m.s. deviation = 1.7 Å for 45 structurally aligned residues, residues 76–120 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 230–253 and 256–276 of the Hsc70 ATPase domain) (21). There does not seem to be any significant functional relationship between these two proteins. The α-helix-loop-α-helix structure of the tip domain is reminiscent of the helix–hairpin–helix (HhH) DNA-binding motif that is present in a number of DNA repair enzymes (22–25). The HhH motifs are characterized by the conserved sequence motif hxxhxGhGxxxAxxxhh, where h is any hydrophobic residue (VILMWFYA) and x any residue (23). Two conserved glycines allow a tight turn between two α-helices. However, the tip domain of <span class="yellow">P. aeruginosa</span> RdgC protein does not have such a sequence motif, and the loop in the RdgC tip domain is much more extended than that of the HhH motifs. Therefore, the RdgC tip domain is uniquely folded.
With the base domain (residues 168–306) alone, the highest similarity is found with <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase (unpublished deposition; PDB code 2H9C, Z score = 4.0, r.m.s. deviation = 4.8 Å for 51 structurally aligned residues, residues 186–191, 237–240, 257–272 and 274–298 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 36–41, 50–53, 55–66 and 68–96 of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase chain A). There does not appear to be any functional relationship between the two proteins, and the base domain of RdgC is uniquely folded. The 101-residue chain of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase is folded into three α-helices, two of which overlap with two α-helices (α2 and α3) of the base domain of RdgC.<br><br>Ring-shaped dimer structure and inter-subunit interface
Our structure reveals that <span class="yellow">P. aeruginosa</span> RdgC forms a ring-shaped dimer of approximate 2-fold symmetry in the crystal (Figure 1). This observation is consistent with the results of previous gel filtration and sedimentation equilibrium experiments, which suggested the presence of dimeric species of <span class="yellow">E. coli</span> RdgC in solution (1,2). Depending on the concentration, RdgC could also exist in solution as monomers, tetramers and higher oligomers (1). In the crystal lattice, no monomeric, trimeric or tetrameric species exist. Our structure unequivocally rules out the monomer–trimer model, which fits the sedimentation equilibrium data approximately, as well as the monomer–dimer–tetramer model (1). The RdgC dimer has approximate dimensions of 80 Å × 60 Å × 40 Å, with a central hole of ∼30 Å diameter.
Only a moderate amount of surface area is buried upon the formation of the RdgC dimer. The monomer–monomer interface buries a solvent-accessible surface area of ∼1360 Å2 per monomer, corresponding to ∼8% of the total surface area of the monomer (Protein–Protein Interaction Server at http://www.biochem.ucl.ac.uk/bsm/PP/server/). Polar and nonpolar atoms in interface are 44 and 56%, respectively. The interface can be divided into two kinds: ‘interface A’ involving both the tip domains and center domains, and ‘interface B’ involving the base domains (Figure 2A). These two interfaces have significantly different characters.<br><br>The interface A is formed largely by the crossing over of the two tip domains in the dimer (Figure 1). This interface covers a surface area of ∼565 Å2 and is largely nonpolar, with the polar and nonpolar atoms in the interface being 37.5 and 2.5%, respectively. Phe120 contributes considerably to the nonpolar character of the interface between the tip domains, accounting for ∼150 Å2 of the interface area. Phe120 is part of a highly conserved sequence segment and lies at the boundary between the tip domain and the center domain. The aromatic ring of Phe120′ inserts into a hydrophobic pocket, which is formed by the hydrophobic residues Ile74, Leu75, Pro76, Val79, Leu115, Ala119 and Phe120 of the neighboring monomer (Figure 2B). The primed residue belongs to the adjacent subunit. These residues are well conserved in the RdgC family (indicated by green triangles below the aligned sequences in Figure 3). As these hydrophobic residues are located near the molecular 2-fold axis, the two clusters of hydrophobic side chains are merged into a single, large cluster in the dimer (Figure 2B). Of the 16 hydrogen bonds that exist between the two subunits, only one is made in this interface between Arg118 and Arg118′.<br><br>The interface B is contributed to exclusively by the two base domains. This interface covers a surface area of ∼795 Å2. The polar and nonpolar atoms in this interface are 50% each. The two β-sheets of the two base domains merge into a single ten-stranded antiparallel β-sheet in this interface (Figure 2A). Thus, 15 of 16 inter-subunit hydrogen bonds are clustered in this interface. The hydrogen bond networks can be divided into two types. The first type involves the main chain atoms of Val206, Arg208 and Lys210 in strand β9; this strand associates with the equivalent strand from the other subunit in an antiparallel manner (Figure 2C). The other type involves both the side chain and main chain atoms. The side chain of Arg211′ interacts with the backbone of Gly204, while the side chain of Lys227 hydrogen bonds to those of both Gln212′ and Glu218 (Figure 2D). In addition, the side chain of Gln212′ interacts with that of Asp199 and the backbone of Arg211′ (Figure 2D). Arg211, Gln212, Glu218 and Lys227 are highly conserved among the RdgC family members. The residues participating in the hydrogen bond networks are indicated by red circles below the aligned sequences in Figure 3. The observation that conserved residues play important roles in forming the dimer implies that dimerization of RdgC is critical for its function.
If one or both of interfaces A and B are to be broken for functional reasons such as for allowing dsDNA to enter into the central hole of the RdgC dimer, we can conceive of three scenarios. The first possibility would be a simultaneous breakage of the two interfaces A and B. The second would be the sequential disruption of the interfaces A and B. The third would be the disruption of only one of the two interfaces A and B. Sedimentation equilibrium indicated the presence of monomers in solution (1), which is consistent with the first two scenarios. If the third scenario holds, RdgC dimers could possibly exist in two conformations in solution, i.e. a closed form as observed in the present crystal structure, and an open form, in which one of the two interfaces A and B is broken.<br><br>A model for DNA binding by RdgC and DNA-binding assay
The net charge of <span class="yellow">P. aeruginosa</span> RdgC is highly negative, since each monomer contains 12 aspartate, 13 glutamate, 10 lysine, 9 arginine and 2 histidine residues. The net charge of a dimer would be −12 (or −8), if we assume none (or all) of histidines to be positively charged. The electrostatic potential at the molecular surface of RdgC dimer shows that the charge distribution on the protein surface is highly nonuniform (Figure 4A). Since the inner surface of the RdgC dimer is rich in conserved, positively charged residues and the diameter of the central hole (∼30 Å) is approximately similar to that of other toroidal DNA-binding proteins (26), it appears likely that RdgC dimers bind dsDNA through the central hole. Despite extensive co-crystallization and soaking trials with linear dsDNAs of different lengths (14-mer, 16-mer and 18-mer), we could not locate the bound DNA experimentally. We attribute this difficulty to the nature of sequence-nonspecific DNA binding by RdgC. Therefore, we built a crude model of a DNA complex of the RdgC dimer by fitting dsDNA into the central hole (Figure 4B). This model is supported by the DNA-binding assays coupled with mutagenesis, as described below. It is also consistent with the suggested binding site size of ∼16 nt for the <span class="yellow">E. coli</span> RdgC dimer (1) and the approximate thickness (∼40 Å) of the <span class="yellow">P. aeruginosa</span> RdgC dimer.<br><br>When we performed electrophoretic mobility shift assay for dsDNA binding, <span class="yellow">P. aeruginosa</span> RdgC did not require Mg2+ ions (Supplementary Figure 2), unlike <span class="yellow">Deinococcus radiodurans</span> RecR, which required high concentrations of Mg2+ ions (27). ATP was not required for dsDNA binding by <span class="yellow">P. aeruginosa</span> RdgC, either (Supplementary Figure 2). This is consistent with an apparent lack of an ATP-binding pocket in the RdgC dimer structure.
In order to identify the residues that are important for dsDNA binding, we carried out mutational analyses. The residues for mutagenesis were selected on the basis of the modeled complex between RdgC dimer and dsDNA, together with sequence conservation (Figure 4C). We prepared eleven point mutants (R4A, K70D, R81D, F120A, R122D, K146A, R198A, R208A, R211D, Q212A and R252A). These eleven mutated residues are well conserved among RdgC proteins. Eight mutants (R4A, K70D, R81D, R122D, K146A, R198A, R208A and R252A) had a lower affinity for dsDNA than the wild-type RdgC (Figure 4D). These eight mutated residues are indicated by blue squares below the aligned sequences in Figure 3. This result indicates that dsDNA binds to the inside surface around the central hole of the RdgC dimer, since the mutated residues are located on the inner surface of the RdgC dimer. It is worth mentioning that a single mutation introduces two alterations per RdgC dimer.
The three mutants (F120A, R211D and Q212A) were prepared to investigate whether dimerization of RdgC is crucial for dsDNA binding. Phe120 is a key residue located in the inter-subunit interface A (Figure 2B), while Arg211 makes hydrogen bonds in the interface B (Figure 2D) and Gln212 is involved in both intra- and inter-subunit interactions (Figure 2D). dsDNA binding by the F120A mutant is considerably weakened, compared to either the R211D or Q212A mutant (Figure 4D). This result suggests that destabilization of the ring-shaped architecture of the RdgC dimer is deleterious for tight dsDNA binding and that interface A is probably more important than interface B for dsDNA binding. dsDNA-binding affinity of the Q212A mutant was lower than the wild type (Figure 4D), whereas that of the R211D mutant was comparable to the wild type (Figure 4D). This result suggests that disruption of the inter-subunit interface B is also detrimental for dsDNA binding but to a smaller extent than disruption of the interface A. It further suggests that the side chain of Arg211 is not involved in recognizing dsDNA. It is interesting that the DNA complex for wild type forms a distinct band (lane 3 in Figure 4D), as opposed to a more diffuse band as one would expect if the complexes contained different numbers of RdgC dimers. Assuming a binding site size of ∼16 nt (1), the 414-bp duplex DNA would bind ∼25 RdgC dimers if it is fully saturated.<br><br>RdgC is unique among ring-shaped DNA-binding proteins
<span class="yellow">Pseudomonas aeruginosa</span> RdgC adds a unique member to the repertoire of toroidal DNA-binding proteins (26) such as <span class="yellow">D. radiodurans</span> RecR (27) and DNA polymerase processivity factors, including the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29) and PCNA of eukaryotic DNA polymerase δ (30,31) (Figure 5). These proteins do not have an overall sequence similarity among themselves, and their quaternary structures also vary. <span class="yellow">P. aeruginosa</span> RdgC and <span class="yellow">E. coli</span> β clamp are homodimers, T4 gp45 and eukaryotic PCNAs are homotrimers and <span class="yellow">D. radiodurans</span> RecR is a homotetramer.<br><br>In <span class="yellow">P. aeruginosa</span> RdgC, the inner surface of the ring-shaped dimer is largely formed by β-strands, while the outer surface of the ring is mostly formed by α-helices (Figure 5). The predicted DNA-binding site of <span class="yellow">P. aeruginosa</span> RdgC dimer is primarily formed by β-strands and loops. In contrast, in other ring-shaped DNA-binding proteins such as the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29), PCNA of eukaryotic DNA polymerases δ (30,31) and <span class="yellow">D. radiodurans</span> RecR (27), α-helices are located toward the center of the ring and are surrounded by β-strands in the outermost side of the ring (Figure 5). Despite a superficial resemblance of the overall ring-shaped architecture of the RdgC dimer with those of other toroidal DNA-binding proteins, the internal arrangement of the secondary structure elements in the RdgC dimer is strikingly different from those of other toroidal DNA-binding proteins. Thus, RdgC is unique among ring-shaped DNA-binding proteins.
A number of other proteins involved in DNA metabolism also adopt a ring-shaped structure despite diversity in the molecular functions (26). They include NAD+-dependent DNA ligase (25), λ exonuclease (32), hexameric helicases, topoisomerases, the trp RNA-binding attenuation protein, the <span class="red">bacteriophage</span> head-to-tail connector and translin (26). There are also many examples of the proteins in DNA recombination pathways that assemble into multi-subunit toroids containing a central channel (26). Electron microscopy showed that the <span class="yellow">E. coli</span> RuvB protein, in the presence of ATP, forms a dodecamer on double-stranded DNA in which two stacked hexameric rings encircle the DNA and are oriented in opposite directions with D6 symmetry (33). The <span class="yellow">human</span> RAD52 protein was found to assemble into heptameric rings with a large funnel-shaped channel, 40–60 Å in diameter (34). The <span class="yellow">human</span> DMC1 protein has so far been detected only as an octameric ring (35).<br><br>Inhibition mechanism of RecA function by RdgC
RdgC inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). It was observed that the effects of RdgC on RecA-promoted DNA strand exchange were much more pronounced than the effects of RdgC on other RecA functions, when RdgC was added to the reaction late (1). By examining the effects of the DinI protein on RdgC-mediated inhibition of RecA activities, it was concluded that the potent effect of RdgC in the inhibition of strand exchange did not primarily reflect a displacement of RecA in the nucleoprotein filament by RdgC, but instead a binding of RdgC to the dsDNA substrate so as to make it unavailable to the filaments for strand exchange (1).
Our work contributes to a better understanding of the inhibition mechanism of RecA function by RdgC. The ring-shaped architecture with a central hole that is suitable for the binding of dsDNA lends RdgC with a capacity to hold dsDNA very tightly. Therefore, we suggest that RdgC dimers probably function as a tight dsDNA gripper and physically block access to the target dsDNA by RecA filaments, because tight encircling of dsDNA by RdgC dimers, possibly as a linear aggregate, would prevent the two strands from dissociating. However, it remains unclear how the closed-ring structure of the RdgC dimer might allow binding of a large number of RdgC dimers on the duplex DNA. It may be conceivable that each RdgC dimer can open temporarily at one or both of its two inter-subunit interfaces so that dsDNA can enter the central hole.
RdgC was at its highest level during exponential phase, reaching its maximum of ∼1000 dimers per <span class="yellow">E. coli</span> cell. Its level decreased sharply to ∼50 dimers per cell in stationary phase (2). Under normal cellular conditions, RecA is maintained at ∼1000 molecules per cell (http://www.els.net/). Following LexA repressor cleavage, the level of RecA protein in the cell increases by as much as 20-fold. RdgC inhibition of the RecA function by a simple competition mechanism at the cellular concentration of RdgC clearly requires a much higher affinity for DNA by RdgC than RecA. Possible protein–protein interactions between RdgC dimers might have a profound effect on the affinity for DNA, when a large number of RdgC dimers are bound on a duplex DNA. If only one of the two inter-subunit interfaces can open up and the 2-fold symmetry axes of the RdgC dimers are misaligned sufficiently, the escape of DNA from a linear array of interacting RdgC dimers would be extremely difficult. Alternatively, both inter-subunit interfaces of each dimer may open up simultaneously. A line of indirect evidence for the possible protein–protein interactions between RdgC dimers is provided by the cooperative binding of RdgC to DNA (7). Cooperativity of RdgC binding may be caused by local and distant protein–protein interactions (7). A kind of side-by-side interaction between two adjacent molecules of RdgC dimer is present in the crystal, burying an accessible surface area of ∼750 Å2 at the interface. The interaction involves mostly the less polar side surface of the dimer (left side of Figure 4A), with 2-fold symmetry axes of the two interacting RdgC molecules making an angle of ∼90° when viewed down the central holes and ∼15° when viewed perpendicular to the holes. This may explain the observed DNA bending by RdgC, which was speculated to be a necessary step in the working mechanism of RdgC, if one can assume that the strong protein–protein interactions in the crystal were relevant for protein–protein interactions in solution (7).
RdgC protein was shown to have a potent effect on RecA protein-promoted DNA strand exchange in <span class="yellow">E. coli</span> (1). In a series of experiments, RdgC was added after RecA filaments had formed on the ssDNA and 5 min after the linear dsDNA was added. With 3 μM RecA, a sharp reduction in DNA strand exchange products was seen with 0.4 μM RdgC, and the generation of products was abolished at 0.8 μM RdgC (1). This suggests that the binding affinity of RdgC to dsDNA is considerably higher than that of RecA. The high affinity of RdgC could be explained by its ring shape and the suggested mode of DNA binding. Tetrameric species of <span class="yellow">E. coli</span> RdgC, which started to form at around 2 µM concentrations, were speculated to be the species with the most potent inhibitory effect, possibly by having a higher affinity for DNA (1). Higher affinity for DNA of the tetrameric species could be understood, if the formation of a tetramer in solution involves a protein–protein interaction similar to the side-by-side interaction between two adjacent RdgC dimers in the crystal, although there is no experimental evidence. Obviously, there are still many open questions that need to be addressed in future biochemical experiments.<br><br>
SUMMARY
The crystal structure of RdgC, a recombination-associated DNA-binding protein, determined in this study reveals that the polypeptide chain is uniquely folded and two monomers associate to form a ring-shaped dimer with a central hole of ∼30 Å diameter. The inner surface around the central hole is rich in positively charged residues. These structural features suggest that dsDNA runs through the central hole when it binds to RdgC dimers. This idea is supported by our mutational studies, which indicate that the conserved, positively charged residues around the central hole are important for dsDNA binding. The toroidal architecture of the RdgC dimer provides a sound framework for a better understanding of its role in homologous recombination.
Coordinates
Atomic coordinates and the structure factor data have been deposited in the Protein Data Bank (accession code 2OWY).<br><br>
SUPPLEMENTARY DATA
Supplementary Data are available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA324075</h3>Structural organization of a 17 KB segment of the alpha 2 collagen gene: evaluation by R loop mapping.
Abstract
A recombinant phage, SpC3, containing a 17 kb genomic DNA insert representing approximately 60% of the 3' portion of the <span class="yellow">sheep</span> collagen alpha 2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 alpha 2 coding sequences (exons) were mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region required for coding the interspersed 3 kb of alpha 2 collagen genetic information was 5.6 fold longer than the corresponding alpha 2 mRNA coding sequences.Images<br><br><br><br><br><br>
 Nucleic Acids Research <br><br> Structural organization of a 17 KB segment of the a2 collagen gene: evaluation by R loop mapping Millie P.Schafer, Charles D.Boyd, Paul Tolstoshev and Ronald G.Crystal <br><br> Pulmonary Branch, National Heart, Lung and Blood Institute, National Institutes of Health, Bethesda, MD 20205, USA <br><br> Received 13 February 1980 <br><br> ABSTRACT <br><br> A recombinant phage, SpC3, containing a 17 kb genomic DNA insert rep<br><br> resenting approximately 60% of the 3' portion of the <span class="yellow">sheep</span> collagen a2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 a2 coding sequences (exons) were <br><br> mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region <br><br> required for coding the interspersed 3 kb of a2 collagen genetic information was 5.6 fold longer than the corresponding a2 mRNA coding sequences. <br><br> INTRODUCTION <br><br> Type I collagen, composed of two al(I) and one a2 polypeptide chains, is the most abundant of the five known mammalian collagen types. It is a major extracellular constituent of tissues such as bone, tendon, skin and lung where, because of its great tensile strength, it plays an important <br><br> role in tissue structure and function. The polypeptides comprising type I <br><br> collagen are synthesized in long, precursor forms, referred to as pro al(I) and pro a2 chains,1 each containing approximately 1500 amino acid residues (1). <br><br> 1 The terminology for the primary translation products of the various <br><br> collagen messenger RNAs is still in a state of flux. It is known that al(I) and a2 chains are synthesized in precursor forms; these are currently termed pro al(I) and pro a2 chains (1). However, recent studies (43,44) have shown that the actual primary translation product of al(I) mRNA is somewhat larger <br><br> than the pro al(I) chain; this translation product has been termed a "pre-pro al(I) chain." Although the corresponding "pre-pro a2 chain" likely exists, it has not yet been characterized. For simplicity, therefore, until the <br><br> proper terminology for the primary translation product is clarified, we have <br><br> chosen to use the terms "al(I)" and "a2" to refer to the type I collagen mRNAs and collagen structural gene sequences. <br><br> ?) IRL Press Umited, 1 Falconberg Court, London W1V 5FG, U.K. <br><br> Volurne 8 Number 10 1980 <br><br> 2241 <br><br> Nucleic Acids Research <br><br> There is increasing evidence that the quantities of al(I) and a2 <br><br> chains produced by cells may be regulated, in part, at the genomic level <br><br> (1). However, to understand the processes which mediate their expression, <br><br> it is necessary to understand the structural organization of these collagen genes and the various enzymes that modify the collagen chains after its <br><br> synthesis (1). As an initial approach to this problem, we have recently <br><br> isolated a recombinant <span class="red">bacteriophage</span>, termed SpC3, containing approximately 60% of the a2 gene for type I collagen. Partial characterization of this <br><br> a2 gene demonstrated that, like most other structurally evaluated eukaryotic genes, the coding sequences are interspersed with intervening sequences (2). The present study further characterizes this portion of the a2 <br><br> gene by utilizing electron microscopic R loop analysis to map the coding regions (exons) and intervening sequences (introns) contained within the <span class="yellow">sheep</span> genomic insert of SpC3. The data suggests that the structural <br><br> organization of this portion of the a2 collagen gene represents 3 kb of coding information that is interspersed in a complex fashion with 17 introns over 17 kb of genome. <br><br> MATERIALS AND METHODS <br><br> Isolation and characterization of a2 collagen recombinant clone, SpC3. <br><br> The <span class="yellow">sheep</span> a2 collagen recombinant clone, SpC3, containing a portion of the <span class="yellow">sheep</span> a2 gene, was isolated as previously described (2). Briefly, <br><br> high molecular weight fetal <span class="yellow">sheep</span> liver DNA was extracted by the method of Blin and Stafford (3), and 15-20 kb DNA fragments, resulting from partial Eco RI digestion, were isolated and ligated to the left and right arms of Charon 4A. The recombinant DNA was then packaged in vitro and the <span class="yellow">sheep</span> <br><br> genomic library amplified (4-6). Screening of the <span class="yellow">sheep</span> genomic library was <br><br> conducted utilizing [32 P]-labeled fetal <span class="yellow">sheep</span> tendon type I collagen cDNA and <br><br> several positive recombinant phages were isolated. One recombinant, containing a 17 kb <span class="yellow">sheep</span> genomic insert, was demonstrated to have coding sequences corresponding to 60% of the 3' end of a2 mRNA. Prior studies showed that the insert in SpC3 was not due to ligation of noncontiguous restriction fragments and/or genetic rearrangement during the cloning process (2). Electron mi croscopy. <br><br> To visualize the recombinant <span class="red">bacteriophage</span> DNA structure complementary to a2 collagen mRNA, 1 iig of SpC3 DNA was lyophilized with 1 ig or 100 ng of mRNA <br><br> to give DNA:mRNA ratios of 1:1 and 10:1, respectively. These were then dissolved in 10 i1 of R loop buffer [70% (v/v) deionized formamide, 0.1 N-[tris (hydroxy<br><br> 2242 <br><br> Nucleic Acids Research <br><br> methyl)methyl] glycine (tricine)-NaOH, pH 8.0, 0.5 M NaCl, and 0.01 M ethyl<br><br> enediaminetetraacetate (EDTA)], incubated in sealed capillary tubes at 520 for 14 hr (7), and then diluted 1:10 with R loop buffer. Two spreading methods were used to subsequently prepare the hybrids for visualization in the electron microscope. <br><br> 70% formamide method. Immediately before spreading onto a deionized <br><br> water hypophase, the nucleic acid mixtures were diluted an additional 10-fold in R loop buffer and cytochrome c, 100 vg/ml, was added. <br><br> Urea-formamide method.   The nucleic acid solutions were spread using a modification of the method of Westphal and Lai (8,9). The hyperphase contained 55% formamide, 2.6 M urea, 0.009 M EDTA, 0.09 M tricine-NaOH, pH 8.0, and the nucleic acid mixture (approximately 0.1 to 1.0 ig/ml). <br><br> The mixture was heated at 40? for 30 sec, placed in ice water, and then allowed to reach room temperature. Cytochrome c, 100 pg/ml, was added immediately before spreading onto a deionized water hypophase. <br><br> The nucleic acid-protein films from both methods were absorbed onto <br><br> parlodion-coated grids, stained with uranyl acetate (10), dehydrated in 90% ethanol and rotary-shadowed with platinum-palladium (80:20) at an angle of 50. Micrographs were taken with a Siemens Elmiskop 101 electron microscope at an original magnification of 10,000 and an accelerating voltage of 60 kv. Nucleic acid lengths were measured at a final magnification of 43,000 with a <br><br> Hewlett-Packard 9810A calculator equipped with a 9864A digitizer using pBR322 and fX-174 double and single stranded DNA, respectively, as internal length standards. <br><br> Approximately 10,000 hybrid molecules were screened for these studies; approximately 200 represented molecules containing unambiguous regions <br><br> appropriate for quantitative analysis. The mean lengths of each intron and exon were determined for each method and the data expressed as mean ? <br><br> standard deviation. Nucleic acid lengths of < 50 base pairs could not be <br><br> accurately determined using the methods outlined above. Thus, in the cases <br><br> where introns or exons were of this length or less, accurate error estimates could not be made. <br><br> Bi ohazard precauti ons. <br><br> The construction and screening of the <span class="yellow">sheep</span> genomic library together with amplification of pCg45 and preparation of high titre lysates of chimaeric <br><br> Charon 4A were performed under the physical and biological containment levels specified by the NIH guidelines for recombinant DNA research (11). <br><br> 2243 <br><br> Nucleic Acids Research <br><br> RESULTS <br><br> Location and orientation of the intervening and a2 mRNA coding sequences in the SpC3 recombinant DNA. <br><br> Hybridization of fetal <span class="yellow">sheep</span> tendon type I collagen mRNA to SpC3 DNA <br><br> yielded a complex R loop pattern in the region where the <span class="yellow">sheep</span> genomic insert was expected to be ligated to the arms of the lambda vector, Charon 4A. To <br><br> confirm that these hybridization events were restricted to the inserted <span class="yellow">sheep</span> DNA fragment, duplex DNA strands on both sides of this hybridization area were measured. Using pBR322 as an internal length standard, the left arm of the phage was found to contain 21,780 ? 1500 bp, and the right arm, 11,950 ? <br><br> 1030 bp, corresponding to the report values for the left and right arms, respectively, of Charon 4A DNA (5,6). <br><br> The hybridization of SpC3 to a2 mRNA resulted in the destabilization of <br><br> the inserted helical duplex DNA fragment such that one DNA strand was displaced (Figure 1). Occasionally, the displaced DNA strand was mostly absent, probably due to fragmentation induced by accidental mechanical shearing. This event considerably enhanced the visualization and ordering of the introns. <br><br> As the a2 mRNA annealed to complementary DNA regions present in one <br><br> DNA strand, numerous single-stranded loops resulted. These loops, referred to as introns, represented DNA sequences not complementary to the mRNA. Although, by convention, introns are usually sequentially labeled 5' to 3', an opposite order had to be employed in this case as SpC3 does not contain the sequences coding for the 5' end of <span class="yellow">sheep</span> a2 mRNA. Infrequently, all introns <br><br> were clearly observed within the same hybrid molecule (Figure 1). The largest introns were towards the left arm of Charon 4A and were more clustered than the introns at the opposite end of the hybrid. Thus, much of the genetic <br><br> information coding for the a2 mRNA was contained in that half of the hybrid attached to the right arm. <br><br> A small non-hybridized tail was consistently found in the region where <br><br> the right arm of the phage was ligated to the <span class="yellow">sheep</span> genomic insert (Figure 1). This tail likely corresponded to the 3' poly A sequence of the a2 mRNA. At <br><br> the extreme 5' end of the insert, a variable length was often observed for the remaining unhybridized mRNA strand. This was most likely due to the size heterogeneity of the a2 mRNA used in these studies. <br><br> Size determination and comparative analysis of SpC3 introns. <br><br> Although localization of exon and intron sequences was possible with both the 70% formamide and urea-formamide spreading methods, the probability of <br><br> detecting unambiguous regions in a hybrid molecule was far greater with the <br><br> 2244 <br><br> Nucleic Acids Research <br><br> A                                B ',:  '14 .0. <br><br> 13 <br><br> 7 1 <br><br> -5 13 <br><br> .  .   . .   ,4                     x2%.   N1 <br><br> 9 <br><br> >   10          R       4 <br><br> Figure 1.    Electron microscopic visualization of all 17 introns and 18 exons <br><br> from hybrid molecules formed between recombinant clone SpC3 DNA and fetal <span class="yellow">sheep</span> <br><br> tendon a2 mRNA. (A,B) Shown are the double-stranded segment of the right arm CR) of Charon 4A; double-stranded segment of the left arm CL) of Charon 4A; the <br><br> displaced single-stranded DNA segment of the insert (5); single-stranded DNA loops representing introns sequentially labeled 1 through 17 in the 3' to 5' direction; and 18 regions of insert DNA sequences hybridized to ct2 mRNA <br><br> ---      - ).  (C,D) Similar to (A,B) but the single-stranded DNA segment (5) of the insert is mostly absent. The urea-formamide spreading method, as described in Materials and Methods, was employed. <br><br> 2245 <br><br> Nucleic Acids Research <br><br> urea-formamide method than with the 70% formamide procedure. For this reason, most measurements were made with the urea-formamide method (Tables 1,2). For <br><br> example, the lowest probability event, the detection of intron 17 and the surrounding small exons, "q" and "r", was only observed by way of the ureaformamide spreading procedure. <br><br> The smallest introns, 7 and 12, were found to contain approximately <br><br> the same number of bases. Introns 1 through 8, representing the DNA sequences which split the a2 coding sequence corresponding to the 3' end of the a2 mRNA, varied in length over approximately a 2.4-fold range. With the exception of <br><br> Table 1. Electron microscopic R loop analysis <br><br> present in SpC3 DNA1. <br><br> of the <span class="yellow">sheep</span> a2 introns <br><br> SpC3 was hybridized to <span class="yellow">sheep</span> a2 mRNA as described in Materials and Methods; intron lengths were determined by comparison with *X-174 DNA lengths. <br><br> 2 Introns are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 Intron 17 was visualized only by the urea-formamide method. <br><br> 2246 <br><br> 70% Formamide Method              Urea-Formamide Method <br><br> Intron2         Number of       Intron length       Number of        Intron length <br><br> introns analyzed       (bases)       introns analyzed      (bases) <br><br> 1              10             819 t 063              35            806 t 76 2              12             771 ? 89               46            757 t 78 3              11             458 ? 58               47            464 ? 66 4               6             627? 53               41             611 ? 73 5               5             554 ? 51              34             562 ? 76 6               6             675 ? 19               40            684 ? 76 7               5             386 ? 48               50            342 ? 54 8                7            771 ? 84               49            757 ? 82 9               8           1,181 ? 94               52          1,124 ?122 10               8           1,133 ?106               53          1,124 ?137 11              10             964 t?79              61             952 ?105 12               9             362 ? 77              62             342 ? 71 13               4             723 ?222               53            806 ?105 14               4           1,085 ?142               53          1,075 ?105 15               3             795 ? 60              47             806 ?100 16               2           1,229 ? 48              43           1,319 ?146 174              -                 -                  12          1,539 ?166 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Table 2.   Electron microscopic R loop analysis of the <span class="yellow">sheep</span> a2 exons <br><br> present in SpC3 DNA1. <br><br> 70% Formamide Method                Urea-Formamide Method <br><br> Exon2          Number of        Exon length       Number of       Exon length <br><br> exons analyzed     (base pairs)    exons analyzed    (base pairs) <br><br> a                 7             807 ? 883           26            883 ?132 b                10             252 ? 35            41            234 ? 49 c                11             151 ? 25            48            182 ? 42 d                 6             252 ? 60            44            234 ? 62 e                 5             126 ? 45            37            129 ? 47 f                 5             126 ? 38            35            104 ? 41 9                 5             176 ? 48            40            182 ? 54 h                 5             100 ? 43            47            104 ? 31 <br><br> 7             100 ? 35            50            104 ? 28 j                 7             126 ? 48            45            130 ? 47 k                 8              50 ? 48            51             52 ? 39 1                 9              75? 10             60             52 ? 44 m                 6             126 ? 43            51            182 ? 54 n                 4             100 ? 45            53             52 ? 42 o                 3             <504                50             78 ? 36 p                 2             126 ? 71            47            104 ? 36 q5                - _                               33            130?47 r5 r                                                 6             78?36 <br><br> SpC3 was hybridized to <span class="yellow">sheep</span> a2 mRNA as described in M4aterials and <br><br> Methods; exon lengths were determined by comparison with pBR322 DNA lengths. <br><br> 2 Exons are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 5 <br><br> No statistical evaluation was possible because of the size of the exon (see (Materials and Methods). <br><br> Exons q and r were only analyzed by the urea-formamide method. <br><br> intron 1, the nucleotide size of these introns was less than 800 bases. In contrast, introns 9 through 17 varied over a 4.5-fold range. Except for intron 12, all of the introns 9 through 17 contained 800 bases or more. <br><br> Summation of introns 1 through 16 for the 70% formamide and urea<br><br> formamide spreading methods yielded 12,533 and 12,531 bases, respectively. <br><br> However, detection of intron 17 by the urea-formamide method indicates that <br><br> the actual summed value for the intervening sequences of SpC3 is 14,070 bases. <br><br> 2247 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Size determination and the comparative analysis of SpC3 exons. <br><br> Eighteen exons, "a" through "r", were mapped (Table 2). The largest exon, "a", located on the extreme 3' end of the <span class="yellow">sheep</span> genomic insert, contained <br><br> approximately 800 bp. With the exception of exon "m", exons larger than 150 <br><br> base pairs were confined to the genetic region corresponding to the far 3' end of the a2 mRNA. Exon "a" and exon "b", the two exons situated on either side of intron 1, coded for around 1000 bases of a2 collagen information; these two <br><br> exons contained one-third of the mapped 3,014 DNA bases complementary to a2 mRNA Length distribution and organization of the introns and exons in the <span class="yellow">sheep</span> genomi c i nsert. <br><br> There was a sharp contrast in the size distribution of the exons and introns contained within SpC3 (Figure 2). While the intron sizes varied <br><br> considerably, the exons, in general, showed a very narrow distribution of <br><br> lengths, and thus tended to cluster at the lowest end of the length distri<br><br> A. <br><br> (I) <br><br> z 6 0 <br><br> z <br><br> ZL4 0 <br><br> 021               <br><br> z <br><br> 10 <br><br> B. <br><br> 8 <br><br> z 0 <br><br> 16 0 z <br><br> 2 <br><br> 20   400  600  800 1000 1200 1400 1600 <br><br> LENGTH IN NUCLEOTIDES <br><br> Figure 2. Length distribution of the intron and exon mean lengths measured from micrographs prepared by the urea-formamide method. (A) Introns; (b) Exons. See Tables 1 and 2 for details. <br><br> 2248 <br><br> Nucleic Acids Research bution graph. The exon in the 800-900 bp region was unique and was the only exon which exhibited length overlap with introns. <br><br> The maps of the a2 collagen gene obtained with both electron micro<br><br> scopic spreading methods were almost identical (Figure 3), except that the <br><br> region at the extreme 5' end of the insert could only be mapped with the ureaformamide method. From these measurements we concluded that the length of the a2 collagen genetic regi?on in SpC3 was 17,084 bases long with only 3,014 bases actually coding for a2 mRNA. Therefore, only 18% of the mapped 17 kb <span class="yellow">sheep</span> genetic insert actually contained a2 structural information. <br><br> DISCUSSION <br><br> The organizational structure of the a2 collagen gene is complex. A minimum of 17 introns interrupt the 3 kb of ca2 genetic information interspersed as 18 exons over 17 kb of <span class="yellow">sheep</span> DNA. With the exception of exon "a" located at the extreme 3' end of the inserted a2 gene, all other mapped exons are rather small, exhibiting a size distribution range of 50 to 250 bp. In <br><br> contrast, the intron sizes range from 340 to 1540 bases occupying a total <br><br> of 14 kb. Thus, the DNA region containing the 3 kb of a2 collagen genetic information is 5.6 fold longer than the corresponding co2 mRNA sequences. <br><br> Although the total number of introns present within the 60% 3' portion <br><br> of the a2 <span class="yellow">sheep</span> gene was determined to be 17, variation in the precise number <br><br> TRANSRIPTlO <br><br> A                          5w                r <br><br> 1 "2a4  133  112s  11.17  0. 3O 2  715  6.2  313  04  4.2B  3.51  2.2  12 <br><br> EXON0            p  o    n         5  Ik         h     f     d 4  .  b   <br><br> WPM300       16   16  14   13  12  11  10  9   6  7  6  5  4  3   2   1 <br><br> B <br><br> 110   16     4.2  13111  112n  1113104  3.6    gm74  2 6.  4 4 2  3.2  22  in <br><br> 1  a1m03  133  1.2 11   10  01 341  6. 1   117  6.2 4  12  417  3I5'm 2  <br><br> DM00  1    q5    p  0    n   m  I  k.   j    i   h  g  f     d  c   b     a 147306   17   16   15  14  13  12  11  10  9   a  7  6  5  4  3   2   1 <br><br> 175 17.0 16.6& 16'5.5 15.0 14.5 14. 13.5 13.0 12.5 12.0 11.5 11 10.5M 10.0 9.5 9.0 8. 5.0 71 7.0 6.5 6. 5.5 50 4.5 4.0 3.5 3.0 2.5 2.0 1.5 1.0 <br><br> KILOBASE <br><br> Fiue3      Organization of 60% of the 3' end of the <span class="yellow">sheep</span> a2 collagen gene. (A)}The exon ( ) and intron (CDJ) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules visualized by the electron microscopic 70% formamide spreading method.  (B) The exon Um) <br><br> and intron (Efl) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules evaluated by the electron microscopic <br><br> urea-formamide spreading method. Arrows represent the approximate location of intron-exon junctions. All values are presented as kilobase pairs. <br><br> 2249 <br><br> Nucleic Acids Research <br><br> of introns present in individual hybrid molecules was observed. The reason <br><br> for this may be due to size heterogeneity of the a2 mRNA used as a complementary probe. Although the a2 mRNA was extensively purified (2), some degradation was likely present. It is also possible that additional introns exist that are too small to be detected by electron microscopic R loop analysis. The most likely a2 genetic regions containing such introns are exons "g" and "m". Small <br><br> "protrusions" were sometimes observed within these exons that could represent small introns. Very small introns have been reported for tRNA genes (12,13), <br><br> and it will be necessary to sequence these regions of SpC3 to determine if such introns are present within the a2 <span class="yellow">sheep</span> genome. <br><br> The existence of mRNAs comprised of sequences complementary to noncontiguous regions in a DNA genome was first described for <span class="yellow">adenovirus</span> (14,15) and <br><br> simian virus (16,17) genes. Similar observations were soon reported for eukaryotic genes, including serum albumin (18), conalbumin (19), fibroin (20), s<br><br> globin (21,22), growth hormones (23,24), immunoglobulin (25), lysozyme (26), <br><br> ovalbumin (27-29), ovomucoid (30), rRNA (31-34), tRNA (12,13), and vitellogenin (35). The precise number or occurrence of introns within these genes does not seem to follow any clear pattern. However, the genes for albumin, conalbumin, <br><br> lysozyme, ovalbumin, and ovomucoid exhibit a structural organization comparable to the a2 collagen gene. All of these genes are composed of numerous small <br><br> exons, usually less than 250 base pairs, and introns which show a large size variation within an individual gene. Within all of these genomes, the DNA <br><br> regions required for coding structural formation are 4 to 7 times longer than the corresponding mRNA coding sequences. <br><br> As with the a2 collagen gene, a large unique exon at the extreme 3' end <br><br> is also present in the ovalbumin gene (36,37). In the a2 gene, this exon was 800 to 900 nucleotides in length, while in the ovalbumin gene the exon contained 1,030 base pairs. Interestingly, this ovalbumin exon contained 634-650 base pairs ot DNA sequence to the 3' side of the terminator <br><br> triplet, suggesting the ovalbumin mRNA has a large untranslated region at the 3' end. <br><br> The consistent appearance of a small non-hybridized tail, approximately 100 nucleotides in length, in the region where the right arm of the phage was ligated to the <span class="yellow">sheep</span> genomic insert of SpC3 very likely corresponded to the <br><br> 3' poly A sequence of the a2 mRNA. This would suggest that the a2 mRNA coding sequence terminates very near or at this point in the genome. Previously published biochemical evidence (2) obtained from restriction mapping and Southern blot analysis using [32P]-labeled <span class="yellow">chick</span> a2 cDNA probe supports <br><br> 2250 <br><br> Nucleic Acids Research <br><br> this observation. <br><br> It is unclear what role introns play in genetic expression. However, <br><br> it is known that these sequences are tranncribed as part of larger precursor mRNAs and the introns are then excised and the fragmented mRNA chains <br><br> covalently rejoined (38). Some evidence has recently accumulated demonstrating the existence of precursor a2 mRNAs (39,40). However, if the introns in the a2 gene are transcribed in their entirety, as has been <br><br> demonstrated for the introns in ovalbumin and globin (41,42), much larger precursor a2 mRNAs may exist than those reported to date. For if the re<br><br> mainder of the a2 collagen gene has a structural organization comparable to the 60% 3' portion that has been analyzed, it is probable that the a2 <br><br> collagen gene in its entirety is dispersed throughout a 30 kb DNA segment. <br><br> ACKNOWLEDGMENTS <br><br> We thank Dr. John Dahlberg and Frances Loebenstein, Laboratory of Cellular and Molecular Biology, National Cancer Institute, for the generous use of a <br><br> Siemens Elmiskop 101 electron microscope; Dr. Ursula Heine and Benjamin Elliott, Laboratory of Viral Carcinogenesis, National Cancer Institute, for the use of a Hewlett-Packard calculator equipped with a digitizer and a Denton vacuum <br><br> evaporator, and Drs. Victor Ferrans and Oichi Kawanami, Laboratory of Pathology, <br><br> National Heart, Lung, and Blood Institute, for expert assistance with photography. We are also grateful to Margery Sullivan, Laboratory of Molecular Genetics, <br><br> National Institute of <span class="yellow">Child</span> Health and <span class="yellow">Human</span> Development, for helpful discussions and to Dr. Helga Boedtker, Department of Chemistry, Harvard University, for making available the recombinant plasmid pCg45. <br><br> References <br><br> 1.   Prockop, D. J., Kivirikko, k. I., Tuderman, L., and Guzman, N. A. <br><br> (1979) New Engl. J. Med. 301, 13-23 and 77-85. <br><br> 2.   Boyd, C. D., Tolstoshev, P., Schafer, M. P., Trapnell, B. C., Coon, <br><br> H. C., Kretschmer, P. J., Nienhuis, A. W., and Crystal, R. G. (1980) J. Biol. Chem., in press. <br><br> 3.   Blin, N., and Stafford, D. W. (1976) Nucl. Acids Res. 3, 2303-2308. 4.   Kretschmer, P. J., Kaufman, R. E., Coon, H. C., Chen, M. JY., Geist <br><br> C. E., and Nienhuis, A. W. (1980) J. Biol. Chem., in press. <br><br> 5.   Maniatis, T., Hardison, R. C., Lacey, E., Laver, H., O'Connell, C., <br><br> Quon, D., Sim, G. K., and Efstratiadis, A. (1975) Cell 15, 687-701. <br><br> 6.   Blattner, F. R., Blechl, A. E., Denniston-Thompson, K., Faber, H. E., <br><br> Richards, J. E., Slightom, H. L., Tucker, P. W., and Smithies, 0. (1978) Science 202, 1279-1284. <br><br> 7.   Meyer, J., Neuwald, P. D., Lai, S., Maizel, J. V., Jr., and Westphal, <br><br> H. (1977) J. Virol. 21, 1010-1014. <br><br> 2251 <br><br> Nucleic Acids Research <br><br> 8.   Westphal, H., and Lai, S. P. (1977) J. Mol. Biol. 116, 525-548. 9.   Schafer, M. P., Rohrmann, G., Heine, U., and Beaudreau, G. S. <br><br> (1979) Virol. 95, 176-184. <br><br> 10. Davis, R. W., Simon, M. M., and Davison, N. (1971) in Methods in <br><br> Enzymology (Grossman, L. and Moldave, K., eds) Vol. 21, pp. 413428, Academic Press, New York. <br><br> 11.  Federal Register (1978) 43, 60108-60131. <br><br> 12. Goodman, H. M., Olson, M. V., Hall, B. D. (1977) Pro. Natl. Acad. <br><br> Sci. U.S.A. 74, 5453-5457. <br><br> 13. Valenzuela, P., Venegas, A., Weinberg, F., Bishop, R., Rutter, <br><br> W. J. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 190-194. <br><br> 14. Berget, S. M., Moore, C., and Sharp, P. A. (1977) Proc. Natl. Acad. <br><br> Sci. U.S.A. 74, 3171-3175. <br><br> 15. Chow, L. T., Gelinas, R. E., Broker, T., and Roberts, R. J. (1977) <br><br> Cell 12, 1-8. <br><br> 16. Berk, A. J., and Sharp, P. A. (1978) Proc. Natl. Acad. Sci. U.S.A. <br><br> 75, 1274-1278. <br><br> 17. Bratosin, S., Horowitz, M., Laub, 0., and Aloni, Y. (1978) <br><br> Cell 13, 783-790. <br><br> 18.  Sargent, T. D., Wu, J. R., Sala-Trepat, J. M., Wallace, R. B., <br><br> Reyes, A. A., and Bonner, J. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 3256-3260. <br><br> 19.  Perrin, F., Cochet, M., Gerlinger, P., Cami, B., LePennec, J. P., <br><br> and Chambon, P. (1979) Nucl. Acids Res. 6 2731-2748. <br><br> 20. Tsujimoto, Y., and Suzuki, Y. (1979) Cell 16 425-436. <br><br> 21. Jeffreys, A. J., and Flavell, R. A. (1977) Cell 12, 1097-1108. <br><br> 22. Tilghman, S. M., Tiemeier, D. C., Seidman, J. C., Matija Peterlin, B., <br><br> Sullivan, M., Maizel, J. V., and Leder, P. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 725-729. <br><br> 23. Fiddes, J. C., Seeburg, P. H., DeNoto, F. M., Hallewell, R. A., <br><br> Baxter, J. D., and Goodman, H. M. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 4294-4298. <br><br> 24.  Soreq, H., Harpold, M., Evans, R., Darnell, Jr., J. E., and <br><br> Bancroft, F. C. (1979) Nucl. Acids Res. 6, 2471-2482. <br><br> 25.  Tonegawa, S., Maxam, A. M., Tizard, R., Bernard, 0., and Gilbert, W. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1485-1489. <br><br> 26. Nguyen-Huu, M. C., Stratmann, M., Groner, B., Wurtz, T., Land, M., <br><br> Glesecke, K., Sippel, A. F., and Schutz, G. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 76-80. <br><br> 27. Breathnach, R., Mandel, J. L., and Chambon, P. (1977) Nature 270, <br><br> 314-319. <br><br> 28. Dugaiczyk, A., Woo, S. L. C., Lai, E. C., Mace, M. L., Jr., <br><br> McReynolds, L., and O'Malley, B. W. (1978) Nature 274, 328-333. 29. Weinstock, R., Sweet, R., Weiss, M., Cedar, M., and Axel, R. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1299-1303. <br><br> 30.  Catterall, J. F., Stein, J. P., Lai, E. C., Woo, S. L. C., <br><br> Dugaiczyk, Mace, M. L., Means, A. R., and O'Malley, B. W. (1979) Nature 278, 323-327. <br><br> 31. White, R. L., and Hogness, D. S. (1977) Cell 10, 177-192. <br><br> 32.  Glover, D. M., and Hogness, D. S. (1977) Cell 16, 167-176. 33. Wellauer, P. K., and David, I. B. (1977) Cell 10, 193-212. <br><br> 34. Pellegrini, M., Manning, J., and Davidson, N. (1977) Cell 10, <br><br> 213-214. <br><br> 35. Wahli, W., David, I. B., Wyler, T., Jaggi, R. B., Weber, R., and <br><br> Ryffel, G. V. (1979) Cell 16, 535-549. <br><br> 36. McReynolds, L., O'Malley, B. W., Nisbet, A. D., Fothergill, J. E., <br><br> 2252 <br><br> Nucleic Acids Research <br><br> Givol, D., Fields, S., Robertson, M., Brownlee, G. G. (1978) Nature 273, 723-728. <br><br> 37. O'Hare, K., Breathnach, R., Benoist, C., Chambon, P. (1979) <br><br> Nucl. Acids Res. 7, 321-334. <br><br> 38. Abelson, J. (1979) Ann. Rev. Biochem. 48, 1035-1069. <br><br> 39. Adams, S. L., Alwine, J. C., de Chrombrugghe, B., and Pastan, I. <br><br> (1979) J. Biol. Chem. 254, 4935-4938, <br><br> 40. Rave, N., Crkvenjakow, R., and Boedtker, H. (1979) Nucl. Acids Res. <br><br> 6, 3559-3567. <br><br> 41. Roop, D. R., Nordstrom, J. L., Tsai, S. Y., Tsai, M. J., and <br><br> O'Malley, B. W. (1978) Cell 15, 671-685. <br><br> 42. Tilghman, S. M., Curtis, P. J., Tiemeier, D. C., Leder, P., and <br><br> Weissmann, C. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1309-1313. 43.  Palmiter, R. D., Davidson, J. M., Gagnon, H., Rowe, D. W., and <br><br> Bornstein, P. (1979) J. Biol. Chem. 254, 1433-1436. <br><br> 44. Graves, P. N., Olsen, B. R., Feitzig, P. P., Monson, J. M. and <br><br> Prockop, D. J. (1979) Fed. Proc. 38, 620. <br><br> 2253 </body></html>