
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>bovine</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="blue">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="blue">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1239921</h3>Epigenetic inactivation and aberrant transcription of CSMD1 in squamous cell carcinoma cell lines
Abstract
Background
The p23.2 region of <span class="yellow">human</span> chromosome 8 is frequently deleted in several types of epithelial cancer and those deletions appear to be associated with poor prognosis. Cub and Sushi Multiple Domains 1 (CSMD1) was positionally cloned as a candidate for the 8p23 suppressor but point mutations in this gene are rare relative to the frequency of allelic loss. In an effort to identify alternative mechanisms of inactivation, we have characterized CSMD1 expression and epigenetic modifications in head and neck squamous cell carcinoma cell lines.<br><br>Results
Only one of the 20 cell lines examined appears to express a structurally normal CSMD1 transcript. The rest express transcripts which either lack internal exons, terminate abnormally or initiate at cryptic promoters. None of these truncated transcripts is predicted to encode a functional CSMD1 protein. Cell lines that express little or no CSMD1 RNA exhibit DNA methylation of a specific region of the CpG island surrounding CSMD1's first exon.<br><br>Conclusion
Correlating methylation patterns and expression suggests that it is modification of the genomic DNA preceding the first exon that is associated with gene silencing and that methylation of CpG dinucleotides further 3' does not contribute to inactivation of the gene. Taken together, the cell line data suggest that epigenetic silencing and aberrant splicing rather than point mutations may be contributing to the reduction in CSMD1 expression in squamous cancers. These mechanisms can now serve as a focus for further analysis of primary squamous cancers.<br><br><br><br>Background
CUB and Sushi Multiple Domains 1 (CSMD1) was cloned as a candidate tumor suppressor or progression gene from a region of <span class="yellow">human</span> chromosome 8 deleted in tumors of the upper aerodigestive tract, prostate, ovary and bladder [1-7]. Deletion of 8p23.2 or reduced expression of CSMD1 has been associated with poor prognosis in head and neck squamous cell carcinomas and in prostate cancers [2,5,8].
CSMD1, consisting of 70 exons spread over two megabases of 8p23.2, encodes a rare 11.5 kb transcript most abundantly expressed in the brain [1]. It is the founding member of a novel, evolutionarily highly conserved gene family whose proteins contain multiple domains thought to be sites of protein-protein or protein-ligand interactions and whose structure suggests that they may be transmembrane receptors or adhesion proteins [9,10].
Tumor suppressor genes are expected to be inactivated in cancers either genetically by mutations or epigenetically by modification of their promoters. While CSMD1 transcripts are detectable in upper aerodigestive tract epithelium, preliminary analysis of several head and neck squamous cell carcinoma cell lines suggested that CSMD1 expression was lost in these lines [1]. Although the region containing CSMD1 is frequently deleted in head and neck squamous cell carcinomas and prostatic adenocarcinomas [3,11-15], point mutations in the gene are relatively rare in primary squamous cancers [16] and in squamous cell carcinoma cell lines (Schmidt, Richter and Scholnick, unpublished). Nonsense or splice junction mutations in CSMD1 have not been reported and not enough is known about the function of the protein to accurately assess the effect of the few missense mutations that have been detected. Thus, if CSMD1 is inactivated in tumors, alternative mechanisms for gene silencing must be operating.
In this paper, we demonstrate that while most squamous cell carcinoma cell lines do not express full length CSMD1 transcripts, nearly all produce abnormal transcripts unlikely to encode functional CSMD1 proteins. Methylation of the DNA preceding CSMD1's first exon is correlated with reduction in the level of expression and cell lines expressing at low levels do not appear to elongate the full 11.5 kb transcript. Other anomalies of expression include incorrect splicing and the use of cryptic promoters. Our data suggest that activation of these promoters may result from the global demethylation of the genome associated with tumorigenesis (reviewed by Ehrlich [17]).
Taken together these data demonstrate that mechanisms other than point mutation are responsible for the aberrant CSMD1 expression in head and neck squamous cell carcinoma cell lines, and these data suggest potential targets for further investigation in primary tumors.<br><br>Results
CSMD1 promoter methylation in HNSCC cell lines is correlated with expression levels
Preliminary evidence suggested that CSMD1 expression is lost in head and neck squamous cell carcinomas [1] but that point mutations were rare [[16], and Schmidt, Richter and Scholnick, unpublished]. To date, only two of the 20 cell lines we have tested for CSMD1 expression, UPCI:SCC066 and PCI-13, express large transcripts initiated at the normal CSMD1 promoter. These data suggest that a mechanism(s) other than point mutation must be responsible for the loss of expression. CSMD1's first exon is embedded in a 3.7 kb CpG island (data from the UCSC genome browser [18]) suggesting that promoter methylation might epigenetically silence the gene. To test this hypothesis, we surveyed 32 head and neck cancer cell lines for CSMD1 promoter methylation using the Combined Bisulfite Restriction Analysis (COBRA) technique described by Xiong and Laird (Methods) [19]. COBRA analysis of the three amplicons diagrammed in Figure 1 suggested that 28 of the cell lines (87%) had more promoter methylation than did normal upper aerodigestive epithelium (data not shown).
We selected nine of these cell lines for high resolution analysis of promoter methylation by sequencing of clones from bisulfite converted genomic DNA. This approach has the distinct advantage of allowing determination of the state of all the CpG dinucleotides within an amplicon on an allele by allele basis. Amplicons 1 and 2 have 19 and 20 CpG dinucleotides, respectively. Amplicon 3 could not be examined by this technique because it is unclonable after bisulfite conversion. The methylation data were correlated to CSMD1 expression levels as measured by quantitative RT-PCR using an amplicon spanning exons 1 and 2 (Methods). A pool of cDNA from five normal oropharyngeal epithelium specimens served as a basis for comparison to the cell lines.
Our data from amplicon 1 demonstrate a clear relationship between methylation and the level of expression (Figure 2). The bisulfite sequencing data confirm that there is relatively little promoter methylation in normal tissue (clones 1–20, Figure 2A). This is also the case in cell line UPCI:SCC066 which expresses a large CSMD1 transcript from the normal promoter at a level approximately 33% of that of normal tissue (clones 32–39, Figure 2B). PCI-13, our highest expressing line at 125% of normal epithelium, displays two distinct patterns of promoter methylation with some clones heavily methylated (clones 21–24, and 30) and others with no methylation (clones 25–39 and 31; Figure 2B). This pattern is consistent with either heterozygosity for methylation or the co-existence of 2 distinct populations within the cell line, one heavily methylated and one unmethylated. We cannot distinguish between these two possibilities using the currently available data.
The remaining cell lines express CSMD1 at a level half that of UPCI:SCC066 or less (ranging from 17% to 1% of normal epithelium) and they exhibit considerably greater methylation of amplicon 1 (clones 40–76, Figure 2C). Cell lines with more amplicon 1 methylation tend to express the gene at lower levels but the relationship is not strictly quantitative (Figure 2C).
In contrast, our data revealed no relationship between expression level and methylation of amplicon 2 (located towards the 3' end of exon 1, Figure 1). For example, all of the 10 clones of amplicon 2 sequenced from PCI-13 were methylated at 19 or 20 of their 20 CpG dinucleotides. UPCI:SCC066, on the other hand, has nearly no methylation in amplicon 2 with only a single methylated CpG dinucleotide detected in one clone out of the nine sequenced. Amplicon 2 ranges from completely unmethylated to heavily methylated in the seven remaining cell lines (data not shown).<br><br>Low transcript levels are accompanied by a failure to elongate the full CSMD1 transcript
On the surface, the quantitative RT-PCR data presented in Figure 2 suggest that the cell lines we consider low expressing might still have up to 17% of the normal level of CSMD1 transcript. A survey of 20 cell lines using a battery of RT-PCR primer pairs located throughout the 11.5 kb transcript reveals that this is not the case. These lines included OKF6-TERT1, a TERT immortalized, p16 deficient but untransformed oral keratinocyte cell line [20].
Our data suggest that the low expressing cell lines shown in Figure 2C express considerably more of the 5' end of the 11.5 kb transcript than they do exons further 3', a phenomenon well illustrated by cell line PCI-100. This line expresses the exon 1/exon 2 amplicon at approximately 15% of the level of normal epithelium. In contrast, we had previously reported that CSMD1 transcripts were not detectable in this line by combined RT-PCR and Southern blotting using three sets of intron spanning primers [1]. The most 5' of those amplicons spans exons 9 through 26.
Analysis with additional primer pairs resolves this apparent paradox by demonstrating that the amount of transcript declines sharply and reproducibly as one examines progressively more 3' exons (Figure 3). The same effect is seen using either oligo-dT or random hexamer primed cDNA. No transcript of this structure has been detected in normal epithelium nor have we detected any sequence alterations in PCI-100 that would explain why the full transcript is not expressed. It is not clear whether PCI-100 produces a small number of discrete size classes of transcript, if individual transcripts terminate at random points within the very large introns in this part of the gene (the first 10 introns average ~150 kb in length), or if the short transcripts result from the elevated activity of a previously undetected posttranscriptional control mechanism.<br><br>Inactivation of CSMD1 by aberrant splicing
Two cell lines, UPCI:SCC066 and PCI-13, express large CSMD1 transcripts initiated at the normal promoter. Subsequent finer scale analysis demonstrates that PCI-13's transcript lacks exons 4 and 5, resulting in a frameshift-induced nonsense codon in exon 6 (Figure 4). Sequencing of the PCI-13 RT-PCR product demonstrates the direct juxtaposition of wildtype exons 3 and 6 and that the transcript contains no novel sequences or splices that would prevent the frameshift. UPCI:SCC066 produces two transcripts, a normal one that includes exons 4 and 5 and another that lacks them (Figure 4). RT-PCR of <span class="yellow">human</span> fetal brain cDNA reveals very low levels of an RT-PCR product corresponding in size to that expected from the internally deleted transcript. This band is not readily visible at the exposure used for Figure 4A. We have not detected a similar sized PCR product in RNA from oropharyngeal epithelium but this may reflect the fact that CSMD1 transcripts are ~10x less abundant in oropharyngeal epithelium than they are in fetal brain (data not shown).
The transcripts lacking exons 4 and 5 appear to result from aberrant splicing rather than somatic deletion of these two exons or mutations of their splicing consensus sequences. Exons 4 and 5 can be amplified from PCI-13 genomic DNA and sequencing of those PCR products demonstrates that both their coding sequences and consensus splice sites are wildtype.<br><br>Activation of cryptic promoters in cancer cell lines
The RT-PCR survey revealed a second transcriptional anomaly exhibited by 4 cell lines: SCC9, 041, PCI-1 and PCI-2. Like PCI-100, these lines express low levels of the very 5' end of the transcript and even lower levels of more 3' exons within the first half of the transcript. However, these lines are distinct in expressing higher levels of the 3' half of the transcript, suggesting that alternative promoters in the middle of the gene may be used. SCC9 was chosen for further study because it expresses the 3' half of the transcript at a level dramatically higher than normal for oropharyngeal epithelium. Northern blotting detects a comparatively abundant 6.4 kb truncated transcript as well as smaller amounts of an 8.7 kb transcript in SCC9 (Figure 5A). The other three cell lines express their truncated CSMD1 transcripts at lower levels (data not shown). Sequence analysis of CSMD1 cDNA clones from SCC9 demonstrates that many transcripts are improperly spliced, resulting in retention of intronic sequences and/or deletion of exonic sequences (data not shown). In particular, retention of sequences from intron 40 is common. The high frequency of faulty splicing may explain the broadness of the 6.4 kb CSMD1 band in Figure 5A and suggests that the 8.7 kb transcript may also be incompletely or improperly spliced.
5' RACE [21] reveals that the SCC9 message is initiated just upstream of an Alu element in intron 36 (Figure 5B). Only the 5'-most 120 base pairs of the Alu element are present in the genome. RT-PCR using a forward primer specific for the novel sequences of the SCC9 transcript (prm1904, cgtttagttcgacacacttcatgt) demonstrates that cell lines 041, PCI-1 and PCI-2 do not initiate their CSMD1 transcripts at the same point, suggesting that other cryptic promoters are active in these lines. The sequence of this novel exon has been entered in Genbank as accession number DQ093422.<br><br>DNA methyltransferase inhibitors activate the same cryptic promoter used in cell line SCC9
Expression from epigenetically silenced promoters can sometimes be restored by treatment with inhibitors of DNA methyltransferase or histone deacetylase activity ([22]). We selected two low expressing cell lines with promoter methylation, UPCI:SCC104 and 094, for treatment with various concentrations of 5-azacytidine or 5-aza-2'-deoxycytidine (5-aza-dC) as well as combinations of either of those drugs with the histone deacetylase inhibitor trichostatin A. These treatments did not reactivate the silenced CSMD1 promoter. COBRA analysis of genomic DNA from the treated cells suggested that the drugs did not robustly affect methylation of the CSMD1 promoter even at levels high enough to be toxic to the cells. These experiments did however shed light on the cryptic promoter used in SCC9 cells and on the interpretation of experiments using methyltransferase inhibitors.
Treatment of cell line 094 with relatively high doses of 5aza-dC results in the expression of the 3' end of the CSMD1 transcript. This transcript was not detected in control 094 cells undergoing mock drug treatment (Figure 6) nor was it detected in 094 cells growing under normal culture conditions (data not shown). RT-PCR of cDNA from drug-treated 094 cells using the primer developed from the novel 5' exon of SCC9's truncated transcript (prm1904, see above) yielded a product identical in size to that amplified from SCC9 cDNA (Figure 6). The identity of the product was confirmed by DNA sequencing which also revealed that the drug-induced 094 transcript was more faithfully spliced than the transcript expressed in SCC9 (data not shown).<br><br>
Discussion
Our data clearly demonstrate that expression of normal CSMD1 transcripts is rare in head and neck squamous cell carcinoma cell lines. Of the HNSCC cell lines examined, only UPCI:SCC066 appears to express a normal transcript from the expected promoter. Even that cell line produces a second species of aberrantly spliced transcript lacking internal exons. Our data suggest that epigenetic modification of the DNA 5' of the transcription start site may contribute to the down-regulation of CSMD1. In addition, a low level of expression appears to be associated with production of prematurely terminated transcripts. This degree of complexity might be expected from a 2 megabase, 70 exon gene.
Methylation of a specific region of the CpG island, -395 to -112 bp relative to the transcriptional start site (amplicon 1), appears to be correlated with the activity of the normal CSMD1 promoter. In contrast, methylation of amplicon 2, located within the first exon, shows no such relationship. Our data suggest that the relationship between the amount of methylation in amplicon 1 and the level of expression may not be strictly quantitative. Differences between cell lines with amplicon 1 methylation could arise through a number of mechanisms, for example, variations in the levels of transcription factors between cell lines. In cases where there is considerable heterogeneity in the methylation pattern within a cell line like PCI-100, alleles with less methylation may be expressed at higher levels than those more heavily methylated (compare clones 51 and 52 to clone 58 in Figure 2C). Alternatively, the presence of methylation in amplicon 1 could be a qualitative but not strictly quantitative indicator of methylation of a critical segment of the promoter not discovered in this study.
The normal CSMD1 promoter was not reactivated by drugs that inhibit DNA methyltransferases and histone deacetylases, nor did the drugs abolish CSMD1 promoter methylation, even at toxic doses. Not all genes with promoter methylation respond to such treatments [23]. These drug treatments did, however, provide a potential explanation for the use of a normally cryptic promoter by cell line SCC9. The CSMD1 transcript in this line is initiated near a partial Alu element. 5-aza-dC treatment of cell line 094 activates the same cryptic promoter. This suggests that cryptic promoters may be naturally activated by general hypomethylation of the genome in cancer cells and the subsequent release of repetitive elements from epigenetic repression (reviewed by Ehrlich [17]). The SCC9 transcript does not appear to encode a functional protein but, with a very large gene like CSMD1, there is a potential for some abnormally initiated transcripts to encode truncated proteins with dominant negative properties.
The second ramification of this finding is for the interpretation of data obtained by treating cells with methyltransferase inhibitors. Detection of CSMD1 transcripts solely with primers mapping to the 3' end of the gene could have been erroneously interpreted as representing reactivation of the normal promoter. It seems imperative that such experiments demonstrate that transcripts detected after drug treatment are actually initiated at the normal promoter.
Aberrant splicing also appears to play a role in the production of defective CSMD1 transcripts. Loss of splicing fidelity has been proposed as a characteristic of cancer cells [24,25] and this would be consistent with the variety of misspliced transcripts we detected from SCC9. However, the removal of exons 4 and 5 from the CSMD1 transcript in PCI-13 may reflect a more specific phenomenon than a general inability to splice large introns; this line is still capable of splicing large introns as evidenced by its successful splicing of exon 3 to exon 6, eliminating an intron of over 666 kb. The failure to include exons 4 and 5 may be due to inactivation of a splicing enhancer in intron 3, or to less efficient splicing due to the fact that exons 4 and 5 do not begin with the consensus G residue (Figure 4B) [26].<br><br>Conclusion
Taken together, our data suggest that CSMD1 function is lost in head and neck squamous cell carcinoma cell lines through a variety of mechanisms other than point mutagenesis. Epigenetic modifications of amplicon 1 and defective splicing appear to be fruitful areas to explore in primary head and neck squamous cancers.<br><br>Methods
Cell lines and tissue samples
DNA from HNSCC cell lines UMSCC9, UMSCC35, UMSCC37, UMSCC38, UMSCC45, UMSCC49, UMSCC65, UMSCC68, and UMSCC76 was provided by Dr. Thomas Carey, University of Michigan [27]. Dr. Ruud Brakenhoff, Vrije Universitat, provided cell lines 040, 041, and 094 [28]; Dr. Theresa Whiteside, University of Pittsburgh, provided cell lines PCI-1, PCI-2, PCI-4B, PCI-13, PCI-30, PCI-50, PCI-51, PCI-52, PCI-100 [29], SCC4, SCC9 [30,31], and UPCI:SCC068, UPCI:SCC74, UPCI:SCC104, UPCI:SCC182, UPCI:SCC203, and UPCI:SCC220 [developed by Dr. Susanne Gollin, University of Pittsburgh, 32]. Dr. Gollin provided cell lines UPCI:SCC056, UPCI:SCC066, and UPCI:SCC114 [14,16,32]. The immortal but untransformed keratinocyte line OKF6-TERT1 was obtained from Dr. James Rheinwald, Harvard University [20].
Normal oropharyngeal epithelium was isolated from discarded tissue from uvulopalatopharyngoplasties (UPPP) collected anonymously with the approval of the Washington University <span class="yellow">Human</span> Studies Committee.<br><br>Cell Culture and Tissue Preparation
Squamous cell carcinoma cell lines were grown in DMEM or DMEM:F-12, 1:1 Mixture (BioWhittaker) containing 10% fetal <span class="blue">bovine</span> serum (Sigma). DMEM medium was supplemented with 1X MEM Nonessential Amino Acids (BioWhittaker). Upper aerodigestive tract epithelium was separated from the rest of the UPPP specimen by digestion with Dispase II (Roche) using a protocol adapted from Oda and Watson [33].<br><br>Nucleic acid preparation and bisulfite conversion
Genomic DNA was isolated by using either Nucleospin Tissue kits (Clontech), QIAamp DNA Blood Mini kits (Qiagen) or Trizol (Invitrogen) according to the manufacturers' instructions. Total RNA isolation, synthesis of first strand cDNA, RT-PCR and 5' RACE PCR were performed essentially as previously described [1]. Poly-A+ RNA for Northern blotting was selected from total RNA using Oligotex beads (Qiagen). Northern blotting and hybridization were performed as previously described [1]. cDNA synthesis was primed using oligo dT or random primers and extended by either Thermoscript or Superscript III reverse transcriptase (Invitrogen). PCR was run in Perkin-Elmer 480 or Applied Biosystems 9700 thermal cyclers for 35 cycles unless otherwise noted. Images of ethidium bromide stained gels were captured with a Gel-Doc imaging station (Biorad). Quantitative PCR was run in an Applied Biosystems 5700 thermal cycler using SYBR Green Master Mix (Applied Biosystems). Primers prm2426 (gtgtggagtatctgcagacatga) and prm2427 (ctggactaagcctccacagttct) were used to amplify a 132 base segment spanning CSMD1's first and second exons. An amplicon from <span class="yellow">human</span> 18S RNA was used as a basis for comparisons across cell lines (primers prm2396, ttcggaactgaggccatgat and prm2397, tttcgctctggtccgtcttg). Calculations were performed using the ΔΔCt method in GeneAmp 5700 SDS software (version 1.3) and Microsoft Excel. Quantitation was based on the average values obtained from duplicate reactions. The level of CSMD1 expression in normal oropharyngeal epithelium was determined from pooled cDNA from five UPPP specimens.
We used the CpGenome DNA Modification kit (Intergen) for bisulfite conversion of the genomic DNA according to the manufacturer's protocol, with the following exception. Incubation of the conversion reaction was carried out in a thermal cycler for six cycles each consisting of three minutes at 94°C followed by three hours at 50°C (Christina Menke and Paul Goodfellow, personal communication).<br><br>Analysis of CSMD1 Promoter Methylation
Methylation of three segments of the CSMD1 CpG island was examined using the Combined Bisulfite Restriction Analysis technique (COBRA) [19]. All three segments were amplified by using the nested primers and PCR conditions listed in Table 1. Amplicon 1 extends from -395 to -112 bp, amplicon 2 from +175 to +396 bp, and amplicon 3 from +398 to +718 bp relative to the first base of the transcript. A small region surrounding the transcription start site (-111 to +174 bp) could not be examined because no PCR primers could be designed from its extremely GC-rich sequence. The first round PCR used 2 μl of bisulfite converted genomic DNA in a final volume of 10 μl. Subsequent amplifications with nested primers used 4 μl of first round product as template in reactions with a final volume of 20 μl. All PCR was carried out for 35 cycles. A portion of the second round PCR product was run on a 1.5% agarose gel, stained with ethidium bromide, and quantified using the ImageQuant software package (v1.2 for Macintosh, Molecular Dynamics) so that equal amounts of each could be used in restriction digests.
Restriction digests for COBRA were performed with either BstU I or Taqα I (5 or 10 units per reaction, respectively; New England Biolabs) for 4 hours in a final volume of 10 μl. Taqα I digests were performed only when no methylation was detected with BstU I. BstU I digests also included an internal control DNA fragment to confirm complete digestion. This DNA fragment contains a single BstU I site and was amplified from a cloned CSMD1 cDNA using primers prm2020 (agatcccccagtgtctccctgtgt) and prm2021 (actgctggtgccgtggtaatgact). The control PCR product is 1019 bp long and is digested to two fragments of 605 and 414 bp by BstU I. Digestion products were fractionated on a 10% polyacrylamide gel, stained with ethidium bromide, and visualized with a Gel-Doc video imaging workstation (Bio-Rad).
High resolution analysis of methylation was performed by sequence analysis of individual clones from amplicons 1 and 2. DNA from amplicon 3 proved unclonable and gel electrophoresis suggests that its very AT rich sequence results in a bent DNA configuration. PCR products were purified using the Nucleospin Extraction columns (Clontech) and inserted into the pCR2.1-TOPO vector using the TOPO TA Cloning kit (Invitrogen) according to the manufacturer's instructions. Plasmid DNA from individual colonies was isolated using the Nucleospin Plus Plasmid Miniprep kit (Clontech) and sequenced with a reverse vector primer (agcggataacaatttcacacagga) using fluorescence based sequencing with Big Dye Terminator mix (Applied Biosystems).<br><br>Treatment of cultured cells with DNA methyltransferase inhibitors
Cell line 094 was treated with 5-aza-2'-deoxycytidine (5aza-dC) (Sigma) dissolved in DMSO. Two 100 mm cell culture dishes containing 5 × 105 cells were established for each of the drug concentrations tested. Cells were grown for 72 hours in media containing DMEM:F-12, 1:1 Mixture (BioWhittaker) with 1X MEM Nonessential Amino Acids (BioWhittaker) and 10% fetal <span class="blue">bovine</span> serum and then switched to media containing 5aza-dC at concentrations of 0 μm, 5 μM, 25 μM, or 100 μM. Cells were fed daily for 4–5 days and then both plates were harvested in 3 ml of Trizol (Invitrogen) for isolation of RNA and DNA according to the manufacturer's instructions. RT-PCR used for detection of CSMD1 transcripts in these treated cells was run for 40 cycles.<br><br>
Abbreviations
5aza-dC = 5-aza-2'deoxycytidine, COBRA = Combined Bisulfite Restriction Analysis, CSMD1 = Cub and sushi multiple domains 1, HNSCC = head & neck squamous cell carcinoma, RT-PCR = reverse transcription – polymerase chain reaction, SCC = squamous cell carcinoma, UPPP = uvulopalatopharyngoplasty.<br><br>Competing interests
The author(s) declare that they have no competing interests<br><br>Authors' contributions
TMR performed the DNA methylation analysis, and parts of the transcript survey. BDT cloned and characterized the novel first exon expressed in cell line SCC9. SBS performed parts of the transcript survey and characterized the deletion of exons 4 and 5 in PCI-13. All three authors participated in the analysis of the data and in the writing of the manuscript.<br><br>
<h3>pmcA147033</h3>Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the ribonuclease A gene family.
Abstract
We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophil-associated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this 'mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing.
� 1997 Oxford University Press <br><br> Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> 4235�4239 <br><br> Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the Ribonuclease A gene family Dean Batten+, Kimberly D. Dyer, Joseph B. Domachowske� and Helene F. Rosenberg* The Laboratory of Host Defenses, Building 10, Room 11N104, National Institute of Allergy and Infectious Diseases, National Institutes of Health, 9000 Rockville Pike, Bethesda, MD 20892, USA Received August 4, 1997; Revised and Accepted September 17, 1997 DDBJ/EMBL/GenBank accession numbers: U72031, U72032, AF017258�AF017261 <br><br> ABSTRACT We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophilassociated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this `mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing. INTRODUCTION Ribonuclease A (RNase A, <span class="blue">bovine</span> pancreatic ribonuclease) is the prototype of an enlarging family of proteins defined by distinctive elements of conserved primary structure and enzymatic activity (1,2). The features shared by all members of the RNase A family include six to eight cysteines and specific histidine and lysine residues that form the ribonuclease active site. To date, several distinct lineages within the RNase A superfamily have been identified. Pancreatic ribonucleases (RNases 1) have been isolated from an extensive array of mammalian species (3,4), and mRNA encoding <span class="yellow">human</span> pancreatic ribonuclease has been detected in <br><br> numerous somatic tissues in addition to pancreas (5). RNases 2 and 3, eosinophil-derived neurotoxin (EDN) and eosinophil cationic protein (ECP), respectively, have been characterized primarily with respect to eosinophil function (6,7), although expression of EDN [also known as RNase Us (8) or <span class="yellow">human</span> liver ribonuclease (9)] is also widespread (5). Angiogenin (RNase 5), a structurally atypical member of this family, was originally identified as an agent promoting blood vessel growth and development (10,11). RNase 4 (12,13) and RNase k6 (14) are also members of the RNase A superfamily, although their functions are currently obscure. Recently, Larson and colleagues (15) described cDNAs encoding two highly homologous <span class="yellow">murine</span> ribonucleases, the <span class="yellow">murine</span> eosinophil-associated ribonucleases (mEAR) 1 and 2, which were cloned and identified via tryptic peptides isolated from <span class="yellow">murine</span> eosinophil proteins. Although Southern analysis demonstrated that multiple copies of sequence homologous to mEAR-1 and mEAR-2 were present in <span class="yellow">murine</span> genomic DNA, the precise nature of these copies--as pseudogenes, polymorphisms, or distinct functional genes--was unclear. In this work, we have identified four of these homologous sequences, three encoding novel ribonucleases, and one encoding a pseudogene. Together with mEAR-1 and mEAR-2, these six ribonuclease genes form an `mR cluster,' whose members are more closely related to one another than they are to other <span class="yellow">murine</span> ribonucleases, yet whose position with respect to the defined RNase A lineages remains unclear. MATERIALS AND METHODS Isolation of genomic fragments encoding <span class="yellow">murine</span> ribonucleases by polymerase chain reaction (PCR) Genomic DNA was isolated from cells of the <span class="yellow">murine</span> 3T3 fibroblast cell line. The mR-3, mR-5 and mR-6P sequences were amplified using oligonucleotide primers derived from the published coding sequence of mEAR-1 as follows: 5-CAA ACC CTT TCC CAG AAG TTT GCC-3 (amino acids 28�33); 5-AAA TGT CCC ATC CAA GTG AAC TGG ACC-3 (amino acids 156�148) (15). The PCR reactions were performed as described previously (14), and the multiple products present in the single 400 bp product were identified by dideoxy<br><br> *To whom correspondence should be addressed. Tel: +1 301 402 9131; Fax: +1 301 402 4369; Email: hr2k@nih.gov Present addresses: +Duke University School of Medicine, Durham, NC 27701, USA and �Department of Pediatrics, State University of New York-Health Sciences Center at Syracuse, Syracuse, NY 13210, USA <br><br> 4236 Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> Figure 1. (A) Alignment of amino acid sequences encoded by six related <span class="yellow">murine</span> ribonuclease genes. Regions of amino acid sequence identity among the five functional genes are enclosed in boxes; the boxes enclosing the active site residues (His11, Lys35, His124) and eight cysteines conserved in all members of the RNase A superfamily are shaded. (B) Percent similarities between amino acid sequences determined via the BESTFIT algorithm of the Wisconsin Genetics Computer Group (WGCG) program on-line at the National Institutes of Health. GenBank accession numbers: mEAR (<span class="yellow">murine</span> eosinophil-associated ribonuclease)-1, U72032; mEAR-2, U72031; mR (<span class="yellow">murine</span> ribonuclease)-3, AF017258; mR-4, AF017259; mR-5, AF017260; mR-6P (pseudogene), AF017261. <br><br> sequencing of individual plasmids after subcloning into the pCR II TA cloning vector (Invitrogen, San Diego, CA). The complete open reading frames were obtained by extension in both 5 and 3 directions by uni-directional PCR (Genome Walker kit, Clontech, Palo Alto, CA), involving two amplifications with nested primers and Tth polymerase as described previously (14). The nested gene-specific primer pairs were as follows: for 5 extension of mR-3, 5-CTG GAA CCA CTG GAT ACG TGG GAC TGT CCT-3 and 5-ACG TGG GAC TGT CCT GTG GAG TTC TGG GTT-3; for 3 extension of mR-3, 5-ATG CTG TTG GTG TGT GTG GAA ATC CAA GTG-3 and 5-GCT TGT GCA GTG ACA ATA TAA GTC AAA ACT-3; for 5 extension of mR-5, 5-GGT GGG ACG GTC CTT TGG AGT TCT GGG GTT ACA-3 and 5-GGG GTT ACA GGC AAC TGT GTA GGA CTT CTT TCC-3; for 3 extension of mR-5, 5-GAT GTT GTC CGT GTG TGT CAC AAT CCA CCC-3 and 5-AAG ACT TGC AAA GAC GGG ACA AGT CCA AAT-3; for 5 extension of mR-6P, 5-TTG TTG TTT TGC ATC TGC ATT GTG CAT AAC TGC-3 and 5-TTC TGC ATA ACT GCT TGC TGA ACT TGT GAG TGA-3; for 3 extension of mR-6P, 5-ATG TGA TAA TGC AAT GCT GTC TCT TAG CAG TTA-3 and 5-TAC AAG <br><br> AGT ATG TAA GCC ATT GAA TCA TTT TCT-3. The products of these reactions were likewise subcloned into the PCR II TA vector and evaluated by dideoxy-sequencing. The mR-4 sequence was amplified with a 36 bp primer derived from sequence encoding the amino-terminal signal sequence of mEAR-1; 5-ATG GGT CCG AAG CTG CTT GAG TCC CGA CTT TGT CTC-3 with the carboxy-primer described above. <br><br> Northern analysis The <span class="yellow">murine</span> multi-tissue Northern blot was purchased from Clontech, and pre-hybridized and hybridized as per manufacturer's instructions with radiolabelled oligonucleotide probes. The membrane was washed with 5� SSPE with 0.1% SDS for 1 h at 37_C and the autoradiogram developed after exposure to film at �80_C. The mEAR-1/mEAR-2/mR-3 probe: 5-CTC TTG TCA CTG CAC AAG CCA CTT GGA TTT CC-3, the mR-5 probe: 5-GTC CCG TCT TTG CAA GTC TTG GGT GGA TTG TG-3, and the actin probe: 5-GCA CAT GCC GGA GCC GTT GTC GAC GAC GAG CGC GGC GAT ATC ATC ATC-3 (16). <br><br> 4237 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 4237 <br><br> Figure 2. (A) Dendrogram depicting relationships among the eight characterized <span class="yellow">murine</span> ribonucleases as determined by a modified UPGMA method (17). Abbreviations are as in Figure 1A, also mPR (<span class="yellow">murine</span> pancreatic ribonuclease, sw:rnp_<span class="yellow">mouse</span>), mANG (<span class="yellow">murine</span> angiogenin, U72672), hEDN (<span class="yellow">human</span> eosinophil-derived neurotoxin, M24157), hECP (<span class="yellow">human</span> eosinophil cationic protein, X15161), hRK6 (<span class="yellow">human</span> ribonuclease k6, U64998), pRK6 (porcine kidney ribonuclease (24), and bRK6 (<span class="blue">bovine</span> kidney ribonuclease, sw:rnkd_bovin). (B) Calculated isoelectric points from amino acid sequences in (A) as determined via the PEPTIDESORT algorithm of WGCG. Table 1. Amino acid sequence comparisons of <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases <br><br> Sequence analysis All DNA sequence analysis and comparisons were performed with the assistance of the Wisconsin Genetics Computer Group and Sequencher (Gene Codes Corporation) programs available at the National Institutes of Health. The dendrogram in Figure 2A was constructed by a modified version of the unweighted pair group method with arithmetic mean (UPGMA) (17), constructed using four initial pairings, mEAR-2 to mR-3, mR-5 to mR-6P, hEDN to hECP and bRK6 to pRK6. RESULTS Isolation of genomic fragments encoding novel <span class="yellow">murine</span> ribonucleases The alignment in Figure 1A displays the amino acid sequences encoded by the six related ribonuclease genes. <span class="yellow">Murine</span> eosinophil-associated ribonucleases-1 and -2 (mEAR-1 and mEAR-2) are the predicted amino acid sequences from two genes described previously by Larson and colleagues (15); <span class="yellow">murine</span> ribonucleases 3, 4, 5 and 6-pseudogene (mR-3, mR-4, mR-5 and mR-6P), are the predicted amino acid sequences encoded by novel DNA <br><br> hEDN mEAR-1 mEAR-2 mR-3 mR-4 mR-5 mANG mPR 56 56 54 60 60 43 53 <br><br> hECP 59 59 57 59 58 49 52 <br><br> hRK6 58 59 59 53 56 49 59 <br><br> hPR 47 48 49 48 57 55 79 <br><br> hR4 50 53 51 52 54 57 66 <br><br> hANG 49 48 51 47 47 80 60 <br><br> Values are expressed as percent similarity between pairs of amino acid sequences as determined by the BESTFIT algorithm of the Wisconsin Genetics Computer Group program on-line at the National Institutes of Health. Value representing the highest degree of sequence homology in each row is indicated in boldface. Abbreviations are as defined in Figures 1 and 2, and also include hR4 (<span class="yellow">human</span> ribonuclease 4, sw:rnl4_<span class="yellow">human</span>), hANG (<span class="yellow">human</span> angiogenin, M11567), and hPR (<span class="yellow">human</span> pancreatic ribonuclease, X62946). <br><br> 4238 Nucleic Acids Research, 1997, Vol. 25, No. 21 fragments amplified from <span class="yellow">murine</span> genomic DNA. All five functional genes encode amino acid sequences with eight cysteines and appropriately spaced catalytic histidines and lysine that are conserved among the members of the RNase A superfamily. The predicted coding sequence of the non-functional pseudogene (mR-6P) includes two aberrant stop codons (positions 19 and 100) as well as a point mutation resulting in the destruction of the cysteine at position 78. Each genomic fragment also encodes a 28 residue amino terminal signal sequence (not shown). The amino acid sequences are displayed in order of decreasing similarity to mEAR-1 (Fig. 1B). Genomic fragments encoding mEAR-1 and mEAR-2 were also detected. Relationships among the <span class="yellow">murine</span> ribonucleases A dendrogram depicting the relationships among several of the <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases is shown in Figure 2A. The mEAR-1, mEAR-2, and mR-3 sequences form a closely related sub-group, with only nine amino acid sequence differences noted between mEAR-2 and mR-3, and 12 and 17 between these two and mEAR-1, respectively. The mR-4, mR-5, and mR-6P sequences diverge from this subgroup and from one another as well. These six sequences are more closely related to one another than they are to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or to <span class="yellow">murine</span> angiogenin (mANG) or to any of the three <span class="yellow">human</span> ribonucleases shown (hEDN, hECP or hRK6). Isoelectric points The isoelectric points of all six related ribonucleases range from 8�11, as is typical for members of the RNase A superfamily (Fig. 2B). Interestingly, the isoelectric points within the mEAR-1/mEAR-2/mR-3 subgroup vary by a full 1.0 unit (mEAR-1 at 9.2 to mEAR-2 at 10.2) in response to only 12 discrepancies in amino acid sequence. As increased charge has been associated with increased toxicity in other branches of the RNase A gene family (18), this finding has potential consequences with respect to the function of these proteins (see Discussion). Northern analysis Tissue-specific expression of mRNA encoding these <span class="yellow">murine</span> ribonucleases is shown in Figure 3. The probe complementary to all three sequences of the mEAR-1/mEAR-2/mR-3 subgroup hybridizes to an 1.4 kb mRNA isolated from renal tissue (Fig. 3A). As the probe cannot distinguish among the three genes, it is not possible to conclude whether the band detected represents mRNA encoding one, two, or all three of the ribonucleases of this subgroup. In contrast, the probe complementary to the mR-5 genomic fragment hybridizes to two prominent mRNA species in hepatic tissue, one at 1.4 kb, and a second at 2.0 kb. A prominent mRNA of 1.8 kb was also detected in mRNA isolated from <span class="yellow">murine</span> testicular tissue; fainter hybridizing mRNAs of varying size were detected in lung and skeletal muscle. The size variation may represent differential splicing of the mR-5-encoding mRNA, or may represent mRNAs encoding one or more as yet undiscovered <span class="yellow">murine</span> ribonucleases that are more closely related to mR-5 than those whose sequences are reported here. <br><br> Figure 3. (A) <span class="yellow">Murine</span> RNA probed with a 32 bp oligonucleotide complementary to mEAR-1, mEAR-2 and mR-3 coding sequences. Tissue source of each RNA is indicated above each lane. (B) Same as in (A), probed with a 32 bp oligonucleotide complementary to mR-5. (C) Same as in (A), probed with a 48 bp oligonucleotide complementary to <span class="yellow">human</span> actin (16), demonstrating relative loading of each lane. <br><br> Relationship to <span class="yellow">human</span> ribonucleases The data in Table 1 denote the amino acid sequence similarity between pairs of <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases of the RNase A superfamily. Overall, the similarities within the identified cluster range from 47 to 60%, with remarkably little variation among the pairs. Although the sequences of the mEAR-1/mEAR-2/mR-3 subgroup are slightly more similar to ECP and RNase k6, and mR-4 and mR-5, to EDN and ECP, the orthologous relationships of these <span class="yellow">murine</span> ribonucleases cannot be discerned from these data. This stands in direct contrast to mPR and mANG, whose <span class="yellow">human</span> orthologs can be clearly distinguished on the basis of their amino acid sequence homologies. <br><br> DISCUSSION In this work, we have identified four novel <span class="yellow">murine</span> ribonuclease genes that, together with mEAR-1 and mEAR-2 defined by Larson and colleagues (15), form an unusual cluster within the RNase A gene superfamily. The six genes within this cluster have varying amino acid sequence homologies to one another, but are clearly more closely related to one another than to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or angiogenin (mANG), or to any of the <span class="yellow">human</span> ribonucleases of the RNase A family. These results suggest that the `mR cluster' emerged via multiple duplications of a gene that had already diverged from those encoding the other <span class="yellow">murine</span> ribonucleases. Although this type of expansion is unusual in this gene family, it is actually not unique; similar expansion has occurred in <span class="blue">bovine</span> species, with gene duplication resulting in the three closely-related <span class="blue">bovine</span> pancreatic, <span class="blue">bovine</span> brain (19,20) and <span class="blue">bovine</span> seminal (21) ribonucleases. One interesting feature of the mR cluster is that its relationship to any of to the six known <span class="yellow">human</span> RNase A-type genes cannot be <br><br> 4239 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 determined from their respective amino acid sequence homologies. Although the amino acid sequences encoded by mEAR-1 and mEAR-2 match those of tryptic peptides derived from <span class="yellow">murine</span> eosinophil proteins (15), the homology data do not stand in overwhelming support of a unique relationship between any of the mR cluster ribonucleases and the <span class="yellow">human</span> eosinophil ribonucleases EDN and ECP. The existence of an additional, as yet unidentified <span class="yellow">human</span> ribonuclease (or ribonucleases) more closely related to those of the mR cluster cannot be ruled out. Perhaps the most important issue raised by this work is the question of why so much evolutionary energy has been devoted to enlarging and diversifying the RNase A gene superfamily. To date, this superfamily now includes several distinct lineages, two species-limited clusters, and the two most rapidly evolving functional coding sequences known among primates (22). The question of evolutionary energy takes on particular significance here, where five functional ribonucleases have emerged in what appears to be a remarkably short period of evolutionary time (15). It is conceivable that each of these ribonucleases might serve a unique but related function. On this point, a number of studies have suggested a distinct role for the <span class="yellow">human</span> ribonuclease ECP in eosinophil-mediated host defense (6,7); in addition, we have recently shown that eosinophil ribonucleases can inhibit retroviral transduction of <span class="yellow">human</span> target cells (23). The possibility that certain ribonucleases have diverged to promote distinct and specific host defense-related activities remains an intriguing hypothesis. ACKNOWLEDGEMENTS We would like to thank Dr Jianzhi Zhang for helpful discussions, and Drs Harry L. Malech and John I. Gallin for their ongoing suppport of the work in our laboratory. D.B. is a Howard Hughes Medical Institute-National Institutes of Health Research Scholar REFERENCES 1 D'Alessio, G. and Riordan, J.F. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. 2 Sorrentino, S. and Libonati, M. (1997) FEBS Lett. 404, 1�5. 21 22 23 24 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br><br> 4239 <br><br> Beintema, J.J., Breukelman, H.J, Carsana, A., and Furia, A. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. Confalone, E., Beintema, J.J., Sasso, M.P. Carsana, A., Palmieria, M., Vento, M.T. and Furia A. (1995) J. Mol. Evol. 41, 850�858. Futami, J., Tsushima, Y., Murato, U, Tada, H., Sasaki, J., Seno, M., and Yamada, H. (1997) DNA Cell Biol. 16, 413�419. Ackerman, S.J. (1993) In Makino, S. and Fukuda, T. (eds) Eosinophils: biological and clinical aspects. CRC Press, Inc., Boca Raton, FL Spry, C.J.F. (1988) Eosinophils: a comprehensive review and guide to the scientific and medical literature. Oxford University Press, Oxford, UK. Beintema, J.J., Hofsteenge, J., Iwama, M., Morita, T., Ohgi, K., Irie, M., Sugiyama, R.H., Schieven G.L., Dekker, C.A., and Glitz, D.G. (1988) Biochemistry 27, 4530�4538. Sorrentino, S., Tucker, G.K., and Glitz, D.G. (1988) J. Biol. Chem. 263, 16125�16131. Vallee, B.L. and Riordan, J.F. (1988) Adv. Exp. Med. Biol. 234, 41�53. Moroianu, J. and Riordan, J.F. (1994) Proc. Natl. Acad. Sci. USA 91, 1677�1681. Seno, M., Futami, J., Tsushima, Y., Akutagawa, K., Kosaka, M., Tada, H. and Yamada, H. (1995) Bioch. Biophys. Acta 1261, 424�426. Rosenberg, H.F. and Dyer, K.D. (1995) Nucleic Acids Res. 23, 4290�4295. Rosenberg, H.F. and Dyer, K.D. (1996) Nucleic Acids Res. 24, 3507�3513. Larson, K.A., Olson, E.V., Madden, B.J., Gleich, G.J., Lee, N.A. and Lee, J.J. (1997) Proc. Natl. Acad. Sci. USA 93, 12370�12375. Nakajima-Iyima, S., Hamada, H., Reddy, P. and Kakanaga, T. (1985) Proc. Natl. Acad. Sci. USA 82, 6133�6137. Li, W.-H. and Graur, D. (1991) Fundamentals of Molecular Evolution. Sinauer Associates, Sunderland, MA. Rosenberg, H.F. and Dyer, K.D. (1995) J. Biol. Chem. 270, 21539�21544. Watanabe, H., Katoh, H, Ishii, M., Komoda, Y., Sanda, A., Takizawa, Y., Ohgi, K. and Irie, M. (1988) J. Biochem. 104, 939�945. Sasso, M.P., Carsana, A., Confalone, E., Cosi, C., Sorrentino, S., Viola, M., Palmieri, M., Russo, E. and Furia, A. (1991) Nucleic Acids Res. 19, 6469�6474. Palmieri, M., Carsana, A., Furia, A. and Libonati, M. (1985) Eur. J. Biochem. 152, 275�277. Rosenberg, H.F., Dyer, K.D., Tiffany, H.L. and Gonzalez, M. (1995) Nature Genet. 10, 219�223. Domachowske, J.B. and Rosenberg, H.F. (1997) J. Leukoc. Biol., 62, 363�368. Iwama, M., Sanda, A., Ohgi, K., Hofsteenge, J., and Irie, M. (1993) Biosci. Biotech. Biochem. 57, 2133�2138. <br><br>  <h3>pmcA2377325</h3>Purinergic signalling in the subretinal space: a role in the communication between the retina and the RPE
Abstract
The retinal pigment epithelium (RPE) is separated from the photoreceptor outer segments by the subretinal space. While the actual volume of this space is minimal, the communication that occurs across this microenvironment is important to the visual process, and accumulating evidence suggests the purines ATP and adenosine contribute to this communication. P1 and P2 receptors are localized to membranes on both the photoreceptor outer segments and on the apical membrane of the RPE which border subretinal space. ATP is released across the apical membrane of the RPE into this space in response to various triggers including glutamate and chemical ischemia. This ATP is dephosphorylated into adenosine by a series of ectoenzymes on the RPE apical membrane. Regulation of release and ectoenzyme activity in response to light-sensitive signals can alter the balance of purines in subretinal space, and thus coordinate communication across subretinal space with the visual process.<br><br>Introduction
The retinal pigment epithelium (RPE) lies between the outer segments of the photoreceptors and the choroidal blood supply (Fig. 1). The RPE combines the functions of epithelial and glial cells, providing a barrier while also supporting the neural photoreceptors and modulating their function. Tight communication between photoreceptors and the RPE is critical to coordinate the multiple levels of interaction, and the purinergic contribution to this communication is becoming apparent. The relevance of this purinergic input is emphasized by the many functional effects of P1 and P2 receptor stimulation and by the multiple mechanisms in place to regulate subretinal levels of purine agonists. As the dynamics of ATP release and extracellular conversion into adenosine will modify agonist availability, the modulation of these processes can exert a temporal control on purinergic signaling. The following review will outline the main interactions between the RPE and photoreceptors, describe the effects of stimulating purinergic receptors on both sides of subretinal space, and summarize how levels of ATP, ADP, and adenosine are manipulated in this microenvironment.
Fig. 1Schematic illustration of the key components of purinergic signaling in the subretinal microenvironment. Stimulation of P2 receptors on the RPE can enhance transepithelial fluid absorption while P1 receptors can modulate phagocytosis. ATP released through CFTR and other Cl− channels can stimulate P2 receptors or be converted to ADP, AMP, and adenosine (Ado) by a series of ectonucleotidases present on the apical membrane of the RPE. By controlling the balance of extracellular purines available to stimulate these receptors these mechanisms can control levels of endogenous purines available to activate the receptors. While theoretically possible, it remains to be determined whether these subretinal purines can actually stimulate photoreceptors
Purines and subretinal space
RPE-photoreceptor interactions across the subretinal space
The outer segments of the rods and cones are responsible for the initial stages of vision, converting photon energy into a series of enzymatic reactions that close the light-sensitive channels on the photoreceptor plasma membrane, hyperpolarize the cells, and reduce the release of glutamate from the synaptic terminals [1, 2]. Efficient photoreceptor function depends upon both short-term and long-term support from the RPE. The critical nature of these interactions is evident from the rapid degeneration of photoreceptors in the absence of a healthy RPE layer and by the RPE localization of defective gene product in some forms of hereditary photoreceptor degeneration [3].
The apical membrane of the RPE is separated from the plasma membrane of the outer segments by an extracellular space of only 10–20 nm [4]. Although small, this subretinal space contains a highly structured matrix which ensheathes the outer segments and extends to the RPE [5, 6]. The presence of enzymes within this interphotoreceptor matrix emphasizes that this extracellular space itself is functionally active [7, 8]. This intimate anatomical relationship between photoreceptors and the RPE reflects multiple functional interactions. For example, the RPE accepts, recycles, and exports central components of the phototransduction process [9]. The outer segments are continuously resynthesized, and the phagocytosis, degradation, and processing of shed outer segment tips by the RPE cells is central to this renewal [10]. The ion channels and transporters on the apical membrane of the RPE help regulate the ionic composition of the subretinal space [11]. As extracellular levels of ions can modify the ionic driving forces across the photoreceptor plasma membrane, these RPE transporters can influence the state of neural activity. The transport of fluid and ions from the apical membrane to basolateral membrane of the RPE is also one of the main forces keeping the retina attached [12].
The control of photoreceptor activity by light gives a rapid temporal dependence to some interactions between the photoreceptors and the RPE. The volume of subretinal space increases in response to light [13], linking small changes in the ionic composition of the subretinal space with activity of the RPE transport mechanisms which maintain this volume [14, 15]. Other processes are controlled on a diurnal cycle. The shed tips of the outer segments are phagocytosed by the RPE soon after the onset of light [16, 17]. These processes can both be modulated by purine levels in subretinal space, indicating purinergic regulation is important over multiple time scales.<br><br>Purinergic receptors on photoreceptors
A2 adenosine receptors were localized to both the inner and outer segments of photoreceptor outer segments over a decade ago by Blazynski and colleagues [18], with more recent reports emphasizing their functional role. A2 agonists inhibit the L-type Ca2+ channel on rod outer segments [19] and can inhibit the synaptic release of glutamate from rods, suggesting changes in adenosine levels in subretinal space could modulate light sensitivity [20]. The A2 agonist DPMA and the adenosine deaminase inhibitor EHNA reduce mRNA for opsin in rods, suggesting that endogenous levels of adenosine can downregulate opsin message at night [21]. EHNA and the A2A receptor agonist CGS21680 also increase the survival of <span class="yellow">chick</span> embryonic photoreceptors in culture [22], indicating a long-term neuroprotective role for adenosine.
P2 receptors are also present in the photoreceptors. mRNA for the P2X2 receptor is expressed in the photoreceptor cell bodies, with immunohistochemical localization of the protein to outer segments [23]. In situ hybridization indicates the photoreceptor layer has the highest level of P2Y2 receptor of any region in the <span class="yellow">rabbit</span> retina, although staining was not pronounced in monkey [24]. P2X7 receptors have recently been localized to photoreceptor synaptic terminals, with evidence for ecto-ATPase activity in the synapse, and functional evidence suggesting ATP augments transmission of the light response by rods [25]. It was suggested that ATP might be co-released from photoreceptors with glutamate, although this remains to be tested directly.<br><br>Purinergic receptors on the RPE
Stimulation of P1 receptors can have a considerable impact on RPE cells. A2 receptors have been recognized on cultured and fresh RPE cells for some time [26, 27], with in situ hybridization confirming the presence of A2A receptors in <span class="yellow">rat</span> RPE [28]. Stimulation of A2 receptors reduces the rate of rod outer segment phagocytosis by RPE cells [29], while application of adenosine to the apical membrane of <span class="yellow">chick</span> RPE cells increases the basolateral Cl- conductance, the transepithelial potential, and the c-wave, and decreases the hyperpolarization of the basal membrane in response to light [30]. Although adenosine alone does not increase intracellular Ca2+ levels [31], adenosine acts synergistically with ATP to elevate Ca2+ levels in <span class="yellow">human</span> RPE cells by stimulating both A1 and A2A receptors [32, 33]. Stimulation of A1 receptors with high doses of NECA increases the active transport of fluorescein across the RPE, while activation of A2A receptors decreases this transport, and by extension, transport of the ions that underlie fluid movement [34]. Stimulation of A1 and A2A receptors produces analogous increases and decreases, respectively, in the absorption of subretinal fluid blebs. This is consistent with the negative coupling of the A1 receptor and the positive coupling of the A2 receptors to adenylate cyclase, as increasing cAMP inhibits the transport of fluid across the RPE towards the choroid [35–37]. The agonist 2-Cl adenosine reverses the deficit in phosphoinositide metabolism found in diabetic RPE cells [38], suggesting effects on metabolism in addition to transport and phagocytosis.
Multiple P2 receptors have been localized to the RPE. The P2Y2 receptor was initially characterized in cultured <span class="yellow">human</span> RPE [31], with subsequent reports localizing transcript for P2Y1, P2Y2, P2Y4, and P2Y6 in the <span class="yellow">rat</span> RPE/choroid [39] and for P2Y1 and P2Y12 receptors in ARPE-19 cells [40], and functionally identifying a P2X receptor in <span class="yellow">rat</span> RPE cells [41]. ATP, ADP, and UTP induce numerous effects on RPE physiology [32, 33, 42, 43]. While these effects likely involve multiple different receptor types, the contributions of the P2Y2 receptor have been explored in most detail to date. The P2Y2 receptor has been specifically localized to the apical membrane of fresh <span class="blue">bovine</span> RPE cells, and addition of ATP to this membrane transiently elevates Ca2+, activates a basolateral Cl- conductance, inhibits an apical K+ conductance, and increases the apical to basolateral flow of fluid [43]. This increased absorption of fluid from the subretinal space suggests P2Y2 receptor stimulation could reduce the excess fluid that accumulates in retinal edemas, and several reports have supported this theory. ATP, UTP, and the P2Y2 receptor agonist INS37217 decrease the size of subretinal fluid blebs when injected into subretinal space of <span class="yellow">rats</span> [44]. In both normal and rds +/- <span class="yellow">mice</span> with experimentally induced detachment, INS31217 improves the ERG recovery and decreased cell death [45]. INS37217 also reduces subretinal blebs in <span class="yellow">rabbits</span> [46]. Injection of another P2Y2 agonist, INS542, increases the active transport of fluorescein across the RPE, consistent with this upregulation of ion and fluid transport across the tissue [47]. Together these experiments emphasize the clinical potential of treating retinal edema with P2Y2 agonists.<br><br>
Regulation of purine levels in subretinal space
While synthesized purinergic agonists may prove useful in treating some ocular disorders, the endogenous activation of P1 and P2 receptors in the subretinal microenvironment will ultimately be determined by availability of agonists. These levels are largely controlled by the release of ATP into the subretinal space, its conversion into other purines including adenosine, and the manipulation of adenosine by enzymes or transporters. Recent work has increased our understanding of both the stimuli that initiate changes in subretinal purine levels and the mechanisms that mediate these changes.
Release of ATP by the RPE
At least some of the ATP capable of stimulating the purinergic receptors on RPE cells is released from the RPE itself. The resulting autocrine stimulation ensures local delivery, and control, of purines to initiate the physiologic changes in the RPE. The release of ATP by RPE cells is triggered by multiple stimuli including osmotic stress, bFGF, UTP, NMDA, glutamate, and ATP [39, 40, 48–51]. The ATP release following activation of NMDA receptors by glutamate may have the most interesting implications for communication across subretinal space, given that glutamate confers the light signal from photoreceptors to the rest of the visual system. Glutamate and the specific receptor agonist NMDA triggers ATP release from ARPE-19 cells, with the release inhibited by NMDA antagonist MK-801, and by DCKA, which inhibits the glycine B binding site on NMDA receptors [51, 52]. Although NMDA raises intracellular Ca2+ levels, this increase is prevented by eliminating ATP with apyrase, indicating autostimulation through released ATP is responsible for this Ca2+ signal. NMDA also triggers a release of ATP when applied to the intact <span class="blue">bovine</span> RPE eyecup [51]. The NMDA receptors and the ATP release sites have been functionally identified to the apical membrane of the <span class="blue">bovine</span> RPE, suggesting the neurotransmitter interactions could amplify the signal from any glutamate reaching subretinal space.
The ability of both UTP and ATP to stimulate release of ATP from the RPE supports the theory that the system acts to amplify signals. When applied at greater than 1 μM, ATP triggers a secondary release of ATP peaking 10 min after the initial stimuli [40]. UTP also initiates a release in extracellular ATP with a similar delay [48]. The rise in ATP triggered by UTP is inhibited by the Cl- channel blocker NPPB, and UDP is much less effective at triggering release than UTP; both observations suggest the P2Y2 receptor contributes to the increase in ATP more than diphosphokinase, although influence from the enzyme cannot be ruled out [53].
Recent evidence suggests that ischemia may lead to the release of ATP from RPE cells. Chemical ischemia triggers a substantial ATP release from cardiac myocytes [54], while changes in oxygen levels trigger ATP release in central chemoreceptors [55]. We found that exposure to sodium cyanide led to a rapid release of ATP from ARPE-19 cells (Fig. 2). As hypoxic and/or ischemic challenge may lead to changes in the expression of growth factors in RPE cells during certain ocular disorders such as macular degeneration [56], and as purines can induce expression of VEGF in other cells [57], this ATP release may contribute to growth factor signaling by the diseased RPE.
Fig. 2Chemical ischemia triggers ATP release from ARPE-19 cells. ATP release was measured in the bath directly from cells plated in 96-well plates to which the luciferin- luciferase reaction mixture was added [51]. Left Levels of ATP in the bath after addition of 5 mM NaCN to the cells. Measurement began 1 min after addition of NaCN or control solution to wells. Right Levels of ATP measured at the peak, 3 min after addition of NaCN (n = 12). Levels were normalized to concurrent levels in control (n = 14). Symbols and bars represent mean ± SE, *p < 0.001
The particular conduit for ATP release varies with the stimuli. The release in response to hypotonic challenge is largely dependent upon CFTR, as it was prevented by the specific CFTR inhibitor CFTR172 in addition to the more general blocker glybenclamide [50]. While the precise mechanisms by which CFTR contributes to this release are not yet known, a role for CFTR in ATP release into subretinal space is consistent with the reduction of certain ERG components in cftr -/- <span class="yellow">mice</span> [58] and with the ability of apical ATP to activate conductances associated with these ERG components [43]. The release of ATP is also largely blocked by the vesicular transport inhibitor brefeldin A, suggesting the two processes occurred in series whereby ATP efflux follows the insertion of vesicles containing CFTR into the plasma membrane. Although the Ca2+ chelator BAPTA blocks this ATP release [50], raising Ca2+ alone with ionophore ionomycin does not itself initiate release [48]. This necessary but not sufficient contribution of Ca2+ also supports a role for vesicular insertion.
In contrast to the release following hypotonic challenge, the ATP release in response to NMDA does not involve CFTR [51]. Release is blocked by NPPB, however, suggesting another type of anion channel could serve as a conduit for ATP release. The presence of parallel mechanisms coexisting on the same cell for ATP release triggered by either agonists or by cell swelling has also been reported in astrocytes [59] and may reflect the multiple roles of purinergic signaling within a given tissue. As both stimuli lead to release across the apical membrane into subretinal space, both are expected to influence signaling in the microenvironment.<br><br>Interconversion of purines in subretinal space
The interconversion of nucleotides and nucleosides each capable of stimulating distinct receptors makes the purinergic signaling system of particular interest in a confined region such as the subretinal space. The main enzymes responsible for dephosphorylating extracellular ATP on the RPE cells have been analyzed and a basic understanding of their regulation has begun. This section first describes the enzymes that act on ATP and ADP, followed by enzymes which convert AMP into adenosine.
The dephosphorylation of extracellular ATP by RPE cells involves enzymes from multiple families [40], as found in airway epithelial cells [60]. Degradation of ATP by the apical membrane of the fresh <span class="blue">bovine</span> eyecup and by ARPE-19 cells is inhibited by ARL67156 or βγmATP. Message for eNPP1, eNPP2, and eNPP3 is present in ARPE-19 cells, consistent with the preference of βγmATP for members of the eNPP family [61]. The cells also express NTPDase2, and NTPDase3, although the intermittent presence of NTPDase1 likely reflects a regulated process [40]. Ecto-alkaline phosphatase has no effect on ATP degradation in RPE cells, in contrast to its considerable contribution in airway epithelium [62]. The putative contribution from diphosphokinases to interconversion of subretinal purines is presently unknown.
Extracellular AMP is rapidly dephosphorylated into adenosine in subretinal space. The production of adenosine from ATP at the apical membrane of the <span class="blue">bovine</span> RPE eyecup is inhibited by the ecto-5′-nucleotidase inhibitor αβmADP, confirming a role for this enzyme [63]. The enzyme is localized to <span class="yellow">rat</span> RPE and ARPE-19 cells immunohistochemically. Degradation of 5′AMP is highest near the subretinal space of <span class="yellow">rat</span> retina [63], although localization in <span class="yellow">mouse</span> indicated larger amounts of ecto-5′-nucleotidase at the tips of adjacent Müller cells [64]. Levamisole does not inhibit the dephosphorylation of 5′AMP by the RPE, consistent with the absence of substantial ecto-alkaline phosphatase in subretinal space.
The presence of light may alter the levels of adenosine in subretinal space. Epinephrine is released at the onset of light [65] and stimulation of the RPE with epinephrine can decrease activity of ecto-5′-nucleotidase [63]. While norepinephrine and phenylephrine lead to similar decreases in enzyme activity, prazosin and corynanthine block the effects of norepinephrine, implicating the α1 epinephrine receptor in the inhibition of ecto-5′-nucleotidase [63]. The kinetics of inhibition are consistent with cleavage of the nucleotidase from its GPI anchor. The phagocytosis of rod outer segments is maximal shortly after light onset [16], and this phagocytosis is inhibited by adenosine [29]. The ability of epinephrine released by the illuminated retina to reduce ecto-5′-nucleotidase activity and consequently adenosine levels may relieve this inhibition and enhance the rate of phagocytosis at light onset.<br><br>
Physiologic effects of subretinal purines on the RPE and photoreceptors
The number of purinergic receptors on both photoreceptor and RPE membranes suggests purines make multiple contributions to the physiology of the outer retina. Our increased understanding of how agonist levels in subretinal space are controlled has begun to indicate how and when this contribution may occur. Future research will involve applying these findings from isolated systems to intact RPE-photoreceptor models, and pursuing the role of defective purinergic regulation in ocular disease. While it is unlikely that ATP released across the apical membrane of the RPE can diffuse to these P2 receptors in the outer plexiform layer given the ecto-ATPase activity in the synaptic clef [25], stimulation of receptors elsewhere on the photoreceptor membrane is possible. It would be interesting to determine whether ATP released from the RPE and converted to adenosine by ecto-nucleotidases can actually modulate the response to light by stimulating the A2A receptors on photoreceptor outer segments. The impact of purinergic signaling on chronic ocular diseases is also of interest, such as the role of ischemia-driven ATP release in VEGF production. While the small size of subretinal space can complicate pharmacologic manipulation within the intact RPE-photoreceptor complex, molecular approaches may provide new insight into how endogenous purines in subretinal space affect the physiology, and pathophysiology, of both RPE and photoreceptors.<br><br><br><br><h3>pmcA2453154</h3>A Novel Role of the NRF2 Transcription Factor in the Regulation of Arsenite-Mediated Keratin 16 Gene Expression in <span class="yellow">Human</span> Keratinocytes
Abstract
Background
Inorganic sodium arsenite (iAs) is a ubiquitous environmental contaminant and is associated with an increased risk of skin hyperkeratosis and cancer.<br><br>Objectives
We investigated the molecular mechanisms underlying the regulation of the keratin 16 (K16) gene by iAs in the <span class="yellow">human</span> keratinocyte cell line HaCaT.<br><br>Methods
We performed reverse transcriptase polymerase chain reaction, luciferase assays, Western blots, and electrophoretic mobility shift assays to determine the transcriptional regulation of the K16 gene by iAs. We used gene overexpression approaches to elucidate the nuclear factor erythroid-derived 2 related factor 2 (NRF2) involved in the K16 induction.<br><br>Results
iAs induced the mRNA and protein expression of K16. We also found that the expression of K16 was transcriptionally induced by iAs through activator protein-1–like sites and an antioxidant response element (ARE) in its gene promoter region. Treatment with iAs also enhanced the production and translocation of the NRF2 transcription factor, an ARE-binding protein, into the nucleus without modification of its mRNA expression. In addition, iAs elongated the half-life of the NRF2 protein. When overexpressed in HaCaT cells, NRF2 was also directly involved in not only the up-regulation of the detoxification gene thioredoxin but also K16 gene expression.<br><br>Conclusions
Our data clearly indicate that the K16 gene is a novel target of NRF2. Furthermore, our findings also suggest that NRF2 has opposing roles in the cell—in the activation of detoxification pathways and in promoting the development of skin disorders.<br><br><br><br>Inorganic sodium arsenite (iAs), a ubiquitous element, is one of the most toxic metals present in the natural environment (Bagla and Kaiser 1996). Arsenicals are found as naturally occurring constituents of soil, food, and drinking water (Wu et al. 1989; Yoshida et al. 2004), and exposure to iAs has been associated with a variety of disease outcomes, including disorders of the skin, urinary bladder, liver, and lung (Tchounwou et al. 2004). In particular, skin hyperkeratosis is a characteristic dermatologic lesion associated with ingestion of arsenic from contaminated groundwater (McLellan 2002; Yoshida et al. 2004). There is also a significant association between hyperkeratosis, nonmelanoma skin cancer (e.g., basal cell carcinoma and squamous cell carcinoma), and Bowen disease (Col et al. 1999; Rossman et al. 2004). Furthermore, the pathologic features associated with arsenic-induced hyperkeratosis present as typical acanthotic types of psoriasis-like keratosis, characterized by the aberrant proliferation and terminal differentiation of epidermal keratinocytes (Lee et al. 2006). Many epidemiologic studies have shown that hyperkeratoses are the most frequent precursor lesions of some skin cancers (Bagla and Kaiser 1996; Col et al. 1999).
The keratins are the most prominent cytoskeletal proteins in keratinocytes and comprise a large family of proteins that form intermediate filament networks in all epithelial cell types (Moll et al. 1982). Keratin 16 (K16 ) and keratin 6 (K6) genes are constitutively expressed in a number of stratified epithelial levels, including the palmar and plantar epidermis (Moll et al. 1982). In skin diseases characterized by aberrant proliferation and differentiation, such as psoriasis and cancer, K16 is detectable at higher levels compared with normal tissue (Haider et al. 2006). Furthermore, the tissue-specific overexpression of wild-type K16 in the epidermis of transgenic <span class="yellow">mice</span> results in the hyperproliferation of keratinocytes and aberrant keratinization of cornified layers, leading to hyperkeratosis of the skin (Takahashi et al. 1994).
Nuclear factor erythoid-derived 2 related factor 2 (NRF2), a “cap ’n’ collar” basic leucine zipper transcription factor, regulates a transcriptional program that maintains cellular redox homeostasis and protects cells from oxidative stress and xenobiotic agents (Ishii et al. 2000; Moi et al. 1994). Several detoxifying and antioxidant genes, including glutathione-S-transferases (GSTs), heme oxygenase-1 (HMOX1), and thioredoxin (TXN), are regulated by NRF2 through the antioxidant responsive element (ARE) in the respective promoter regions of these genes (McMahon et al. 2001; Wakabayashi et al. 2004). NRF2 is held in the cytoplasm by a cytoskeletal-associated specific inhibitory protein (kelch-like ECH-associated protein 1; KEAP1) under normal cellular redox state conditions, where it is continuously targeted by the proteasomal degradation pathway (McMahon et al. 2003). Upon exposure of the cell to oxidative stress or electrophiles, NRF2 can escape this KEAP1-mediated repression, translocate to the nucleus, and activate the expression of its target genes (Dinkova-Kostova et al. 2002; McMahon et al. 2003).
Recently, studies of Keap1–/–<span class="yellow">mice</span> have shown that NRF2 accumulates in the nucleus and constitutively activates the transcription of its target genes, even in the absence of stress signals (Wakabayashi et al. 2003). Most interestingly, however, the skin, esophagus, and forestomach of Keap1-deficient <span class="yellow">mice</span> show cornified layer and hyperkeratosis phenotypes. In addition, previous studies have also shown that the expression of NRF2 and ARE-controlled genes is induced by iAs in some cell types (Pi et al. 2003; Sakurai et al. 2005). Furthermore, histochemical analyses have indicated that the expression of K16 is increased in Bowen disease, basal cell carcinoma, and squamous cell carcinoma induced by arsenicals (Yu et al. 1993). However, it remains to be determined whether NRF2 can regulate the transcriptional activation of K16 upon iAs exposure in <span class="yellow">human</span> keratinocytes. Hence, these findings prompted us to investigate the molecular mechanisms underlying the regulation of the K16 gene by iAs-induced NRF2 mediation.
Materials and Methods
Chemicals and reagents
A purified preparation of inorganic sodium arsenite (iAs; NaAsO2; Merck, Darmstadt, Germany) was dissolved in phosphate-buffered saline (PBS) and added directly to the culture medium. A fresh iAs solution was prepared for each new experiment. Cycloheximide (CHX), dimethyl-sulfoxide (DMSO), and a protease inhibitor cocktail were purchased from Sigma (St. Louis, MO, USA). CHX was dissolved in DMSO and stored –20°C until use.<br><br>Cells and culture conditions
The <span class="yellow">human</span> keratinocyte HaCaT cell line was obtained from N.E. Fusenig (German Cancer Research Center, Heidelberg, Germany). Cells were maintained in monolayer cultures in 95% air and 5% CO2 at 37°C in Dulbecco’s modified Eagles medium (DMEM) supplemented with 10% fetal <span class="red">bovine</span> serum (FBS), 50 U/mL penicillin and 50 mg/mL streptomycin and nonessential amino acids (Gibco BRL, Paisley, UK).<br><br>RNA preparation and semiquantitative reverse transcriptase-polymerase chain reaction (RT-PCR) analysis
We determined RNA expression levels by semiquantitative RT-PCR analysis as described previously (Sugioka et al. 2004). Total RNA was isolated from HaCaT cells using the GeneElute Mammalian Total RNA Kit (Sigma). The specific primers used for first-strand cDNA synthesis and PCR were as follows: K16 [forward, 5′-GAT GCT TGC TCT GAG AGG TC-3′, and reverse, 5′-CCA GCA AGA TCT GGT ACT CC-3′; Gene Bank accession no. NM_005557 (National Center for Biotechnology Information 2007)]; c-Jun (forward, 5′-CCT GTT GCG GCC CCG AAA CT-3′, and reverse, 5′-ACC ATG CCT GCC CCG TTG AC-3′; NM_002228); c-Fos (forward, 5′-TTT GCC TAA CCG CCA CGA TGA T-3′, and reverse, 5′-TTG CCG CTT TCT GCC ACC TC-3′; NM_005252); NRF2 (forward, 5′-AGA TTC ACA GGC CTT TCT CG-3′, and reverse, 5′-CAG CTC TCC CTA CCG TTG GA-3′; AF323119); KEAP1 (forward, 5′-CAG AGG TGG TGG TGT TGC TTA T-3′, and reverse, 5′-AGC TCG TTC ATG ATG CCA AAG-3′; NM_012289); TXN (forward, 5′-CAG GGG AAT GAA AGA AAG G-3′, and reverse, 5′-CAA GGT GAA GCA GAT CG-3′; NM_003329), and glyceraldehyde 3-phosphate dehydrogenase (GAPDH) as a loading control (forward, 5′-ACC ACA GTC CAT GCC ATC AC-3′, and reverse, 5′-TCC ACC ACC CTG TTG CTG TA-3′, NM_002046). PCR products were separated on a 1.8% agarose gel and stained with ethidium bromide.<br><br>Western blot analysis
We performed Western blot analysis as described previously (Sugioka et al. 2004). Briefly, nuclear and cytoplasmic proteins were extracted using the NE-PER nuclear and cytoplasmic extraction kit (Pierce, Rockford, IL, USA) according to the manufacturer’s protocol. For protein extraction, the cells were lysed in a buffer containing complete protease inhibitor cocktail. We measured protein concentrations using the DC Protein Assay Kit (Bio-Rad, Richmond, CA, USA). Equal amounts of protein were then resolved by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to a polyvinylidene fluoride membrane (Amersham Biosciences, Bucks, UK). Immunoblotting was carried out with specific antibodies in Tris-buffered saline with 0.05% Tween 20. The primary antibodies were as follows: K16 (Neomarkers, Fremont, CA, USA), NRF2 and KEAP1 (Santa Cruz Biotechnology, Santa Cruz, CA, USA), c-Jun (Cell Signaling, Beverly, MA, USA), and β-actin (Sigma). After washing, the membranes were probed with <span class="yellow">horseradish</span> peroxidase-conjugated secondary antibodies and developed by chemiluminescence using the ECL Plus Detection Kit (Amersham Biosciences).<br><br>Plasmids, transfections, and luciferase assays
<span class="yellow">Human</span> K16 promoter regions of varying lengths (pXK-1, 3, 4, 5-1, and 5-2) were provided by Y-N Wang (National Cheng Kung University, Taiwan). These DNA fragments were prepared from HaCaT cells and were ligated into the pXP-1 luciferase vector (Wang and Chang 2003). The p3xARE/Luc vector, harboring three tandem repeats of ARE, was donated by X.L. Chen (Discovery Research, AtheroGenics Inc., Alpharetta, GA, USA) (Chen et al. 2003). The wild-type NRF2 expression vector (WT-NRF2) was a gift from H.S. So (Wonkwang University School of Medicine, Korea) (So et al. 2006). NRF2 cDNA was subcloned into a pcDNA3.1(+) vector (Invitrogen, San Diego, CA, USA). For the transfection of reporter plasmids, we seeded HaCaT cells into six-well plates at a density of 80% the previous day. Cells were then transfected with a total of each luciferase reporter construct (2.5 μg) using LipofectAMINE plus (Invitrogen). To control for the efficiency of transfection, Renilla luciferase gene expression was monitored using either the pRL-CMV or pRL-TK vectors (Promega, Madison, WI, USA). For overexpression of WT-NRF2, we normalized the total plasmid concentration using the pcDNA3.1(+) empty vector. Thirty-six hours after transfection, the medium was replaced with fresh medium containing either vehicle (PBS) or iAs for 6 hr. After iAs exposure, we harvested cells and analyzed them for luciferase activity using a Dual-Luciferase Reporter Assay System (Promega).
For the investigation of the role of NRF2 in regulating K16 gene expression, transfection of an NRF2 expression plasmid into HaCaT cells was carried out using LipofectAMINE 2000 (Invitrogen). Cells were cultured in 100-mm plates 24 hr before transfection. The expression plasmid WT-NRF2 (15 μg) was then transfected into the cells for 48–60 hr. As a negative control, we used 15 μg of the pcDNA3.1(+) empty vector.<br><br>Electrophoretic mobility-shift assays (EMSA)
We extracted and measured nuclear proteins as described above. Nuclear protein/ DNA complexes were subjected to electrophoresis in nondenaturing 5% polyacrylamide gels containing 2% glycerol in 0.25% Tris-borate/EDTA buffer and transferred to Hybond-N+ nylon transfer membranes (Amersham Biosciences) for detection using the Light-Shift EMSA kit (Pierce) according to the manufacturer’s protocol, with minor modifications. We incubated 10-μg aliquots of nuclear extract with the DNA probe in a binding reaction buffer containing 10 mM Tris/HCl (pH 7.6), 50 mM KCl, 0.5 mM dithiothreitol, 0.25 mM EDTA, 5% glycerol, 2.5 mM MgCl2, 0.05% NP-40 detergent, and 2 μg of poly(dI-dC)·poly(dI-dC) for 30 min at room temperature. For supershift assays, 2 μg of either a polyclonal anti-NRF2 or an anti-c-Jun antibody (Santa Cruz Biotechnology) was added with the nuclear protein for 2 hr at 4°C before the labeled oligonucleotide probe was added. Biotin-labeled, double-stranded oligonucleotides WT-K16ARE (–157/–132, 5′-GGGGAACCTGGAGTCAGCAGT-TAGGA-3′), containing an ARE site (–148/ –140, underlined) in the <span class="yellow">human</span> K16 promoter region, and Mut-K16ARE (5′-GGGGAA-CCTGGAGTCAaaAGTTAGGA-3′, mutated GC box in the ARE) were prepared by Fasmac (Kanagawa, Japan). A consensus ARE probe was purchased from Panomics, Inc. (Redwood City, CA, USA). For competition binding of the K16 ARE-complexes, we used an unlabeled AP-1 consensus oligonucleotide (5′-TATC-GATAAGCTATGAGTCATCCGGG-3′). The binding specificity was confirmed in each case by the addition of a 100-fold molar excess of unlabeled oligonucleotide.<br><br>CHX chase experiment
We investigated the posttranscriptional regulation of both the steady-state levels and half-life of the NRF2 protein by CHX chase analysis. Cells were incubated in serum-free medium in the absence or presence of iAs for 6 hr. The culture medium was then replaced with serum-free medium containing CHX (100 μg/mL). We prepared cell lysates at 0, 10, 30, 60, 120, and 240 min after iAs treatment. Whole-cell lysates were resolved by SDS-PAGE and immunoblotted with antibodies against NRF2.<br><br>Statistics
All the data generated from at least three independent experiments and expressed as the mean ± SD were analyzed by the Student’s t-test. Statistical comparisons were made by logarithmic transformation of the normalized values. We considered p-values < 0.01 to be statistically significant.<br><br>
Results
K16 expression is induced by iAs in HaCaT cells
We wanted to determine whether the K16 mRNA is transcriptionally regulated by iAs, and treated HaCaT cells with this compound for various time periods over a range of doses. After treatment of HaCaT cells with 1–20 μM iAs, the expression of K16 mRNA was increased compared with the control at 6 hr (Figure 1A) but had declined to basal levels at 24 hr. The increase in the K16 protein levels after 6 hr of iAs exposure was just detectable at 10–20 μM, but a dose-dependent increase was more evident at 10 hr (Figure 1B). This enhancement of K16 expression had declined to basal levels at 24 hr.<br><br>Identification of the iAs responsive region in the K16 gene promoter
To investigate the mechanisms underlying the transactivation of the K16 gene by iAs, we first examined the response of the K16 regulatory region to this compound using a luciferase reporter gene assay. The dose-dependent activation of K16 transcription after iAs treatment was observed with a construct containing a 515-bp fragment of the K16 promoter (Figure 2A). To further elucidate the region containing the iAs responsive element, we examined a series of deletions of the 5′-flanking region of K16 gene. The ARE sequence in the pXK-5–1 vector contains an activator protein-1 (AP-1)–like element followed by a GC box. As shown in Figure 2B, an enhancement in the reporter activity levels was observed for the promoter constructs, pXK-1, 3, 4, and 5-1, in response to 20 μM iAs. A decline in reporter activity, however, depended on the number of AP-1–like sites, and the results for the pXK-5-1 construct show also that ARE is activated by iAs. In contrast, no significant activation was observed using a pXK-5-2 construct in response to 20 μM iAs.<br><br>Expression of AP-1 transcriptions factor and c-Jun production following iAs treatment
We examined AP-1 transcription factors c-Jun and c-Fos expression in iAs-treated HaCaT cells by semiquantitative RT-PCR. iAs-induced c-Jun expression was observed during the first 3 hr after treatment (data not shown). An appreciable induction of c-Jun was also confirmed after 6 hr, but this was down-regulated by 24 hr after iAs treatment (Figure 3A). In contrast, the expression of c-Fos was only transiently detectable at 3 hr (data not shown) but was not observed during the 6–24 hr period of this experiment. As shown in Figure 3B, iAs-enhanced c-Jun production can be observed in a dose-dependent manner at 6 hr, but it declines from 10 to 24 hr.<br><br>iAs potently induces the translocation of NRF2 and activates the ARE of the K16 promoter
The results of our reporter assays suggested that iAs stimulates not only the AP-1–like sites but also the ARE site within the K16 gene promoter in HaCaT cells (Figure 3). In addition, several oxidative stress agents and toxic chemicals, including iAs, have been reported to induce the expression of ARE-dependent genes in several cell types (Pi et al. 2003; Sakurai et al. 2005). On the basis of our observations and some recent reports, we thus hypothesized that iAs would have the ability to activate the ARE of the K16 gene promoter directly, resulting in the induction of K16 expression in HaCaT cells. To confirm that the K16 ARE indeed functions as an iAs-responsive transcriptional control element, we performed transient transfections of HaCaT cells with a p3xARE/Luc construct and then subjected these cells to iAs for 6 hr. As shown in Figure 4A, treatment of HaCaT cells with iAs results in a dramatic increase in ARE-driven promoter activity. Likewise, EMSA using a consensus ARE probe show that iAs-induced ARE-binding complexes increase markedly, in a dose-dependent fashion (Figure 4B). These results indicate that iAs has the ability to activate the ARE-driven genes. We performed further EMSA experiments using an ARE probe specific to the K16 proximal promoter region (WT-K16ARE) and found that K16ARE–nuclear protein complexes formation is augmented by iAs in a dose-dependent manner (Figure 4C). Moreover, the formation of these complexes is specifically inhibited by the addition of excess unlabeled oligonucleotide competitor (Figure 4B,C), whereas an excess of an unlabeled AP-1 probe competes only marginally for K16ARE binding (Figure 4C).
The NRF2 transcription factor has been shown to bind to AREs upon translocation into the nucleus, resulting in the induction of ARE-mediated genes (Wakabayashi et al. 2004). To examine whether iAs induces and translocates NRF2 into the nucleus in HaCaT cells, we treated these cells with iAs for either 3 or 6 hr. As shown in Figure 4D, a dose-dependent accumulation of NRF2 protein was observed in the nucleus upon treatment with iAs for 6 hr. This was not observed in the parallel experiment performed over the 3-hr time course.
Supershift EMSA analysis using an NRF2 antibody showed that the iAs-induced and iAs-translocated NRF2 protein binds to the WT-K16ARE probe containing the ARE sequence of the K16 proximal promoter region (5′-GGAGTCAGC-3′) that comprises an AP-1–like site and a GC box, whereas the supershift of c-Jun was not observed (Figure 4E). To identify whether the GC box is dispensable for the iAs-stimulated binding activity of NRF2, we next performed EMSA analyses with either WT- or a Mut-K16ARE probe containing an intact AP-1–like element but a mutated GC box. As shown in Figure 4F, the K16ARE–nuclear protein complexes and supershifted bands that were enhanced by iAs treatment were largely abolished by the addition of the Mut-K16ARE probe.<br><br>iAs stabilizes the NRF2 protein
We examined the effects of iAs treatment on the function of KEAP1 in HaCaT cells. Treatment with iAs did not alter the expression levels of KEAP1 mRNA or protein over either a 3 or 6 hr time course (Figure 5A). Next, we examined the effects of iAs on the expression of NRF2 mRNA in HaCaT cells. Exposure to iAs did not significantly alter the steady-state levels of NRF2 mRNA (data not shown). Production of NRF2 protein, however, was observed to increase in both a dose-and time-dependent manner (Figure 4D). To further examine the stabilization of NRF2 protein by iAs, we monitored the decay of basal and iAs-induced NRF2 proteins after inhibition of protein synthesis by CHX (Figure 5B). The results of this analysis revealed that the NRF2 protein levels decrease by approximately 50% within 30 min of treatment with CHX in cells that had not been exposed to iAs. Only trace amounts of NRF2 are then detectable after 60 min of exposure to CHX in these cells. The HaCaT cells were then pretreated with iAs for 6 hr before their exposure to CHX in a similar timecourse experiment. The levels of NRF2 in these iAs-treated cells were again found to decrease by about 50%, but only after 120 min of CHX exposure.<br><br>NRF2 plays a crucial role in the regulation of K16 gene expression in HaCaT cells
To confirm the functional role of NRF2 in the induction of K16 gene expression by iAs, we investigated whether the expression of K16 mRNA is induced by the overexpression of NRF2 (WT-NRF2) in HaCaT cells. We also investigated the expression of the detoxification gene TXN, which is highly induced by a variety of oxidative stimuli through NRF2-mediated ARE transactivation (Kim et al. 2001). The expression of TXN gene in untransfected cells after treatment with iAs was stronger than that of the control cells (Figure 6A). When the cells were transfected with WT-NRF2 and then treated with or without iAs, the expression of TXN mRNA was augmented markedly compared with the empty-vector control. Similarly, the expression of K16 mRNA was also induced in cells transfected with WT-NRF2 in the absence or presence of iAs. We next performed a transient transfection of HaCaT cells with the pXK-5–1 luciferase vector together with the WT-NRF2 vector. The overexpression of NRF2 in increasing concentrations resulted in significant enhancement of the ARE-mediated K16 promoter activation (Figure 6B).<br><br>
Discussion
In the present study, we showed for the first time that iAs induces the transcriptional activation of K16 in the <span class="yellow">human</span> keratinocyte cell line, HaCaT, through the ARE present in its gene promoter. It has been reported previously that treatment with iAs enhances the production and translocation of NRF2 into the nucleus in several cell types. However, until now it has remained uncertain whether the induction of NRF2 by iAs mediates the transcriptional activation of the K16 gene in keratinocytes. In our current experiments, we demonstrated that iAs elongates the half-life of the NRF2 protein, which results in its increased expression levels. Furthermore, this iAs-induced NRF2 protein was shown to bind to the ARE sequences in the promoter region of the K16 gene. Finally, by overexpressing NRF2, we have clarified that its induction is involved in not only the gene expression of the detoxification gene TXN, but also in the upregulation of K16 expression in HaCaT cells through the ARE in the K16 gene promoter. These experiments indicate an important and novel function for NRF2 in the regulation of K16 in keratinocytes and also help to further explain the molecular mechanisms underlying arsenic-mediated epidermal hyperkeratosis.
In our present experiments, the expression levels of K16 mRNA and protein were indeed found to be enhanced by iAs in a dose-dependent manner (Figure 1). In addition, luciferase assays of the K16 promoter revealed that iAs enhances its activity in a dose-dependent fashion, which is stimulated by AP-1–like sites and an ARE (Figure 2). The promoter of the <span class="yellow">human</span> K16 gene was recently cloned and sequenced, and several AP-1–like sites were found within the –515-bp region of the gene (Wang and Chang 2003). AP-1 transcription factor can be formed by the dimerization of either Jun or Jun/Fos family members (Eferl and Wagner 2003). In the present study, the increased expression of c-Jun, but not c-Fos was evident in the nuclei of HaCaT cells after iAs treatment (Figure 3). Our findings thus suggest that the activation of c-Jun/AP-1 is one of the essential steps in the regulation of K16 gene expression by iAs exposure in HaCaT cells.
It has been well documented that the ARE core sequence includes an AP-1–like binding site (TGAC/GTCA), followed by a GC box (Rushmore et al. 1991; Xie et al. 1995). We have found in our current analyses that the AP-1–like site within the K16 promoter region from –148 to –140 bp (5′-GGAGTCAGC-3′) resembles a consensus ARE sequence. Recent studies have shown that AREs can be specifically bound by complexes of several basic-leucine zipper transcription factors, including NRF2 (Ishii et al. 2000; Moi et al. 1994). NRF2 heterodimerizes with either AP-1 or small MAF (MAFG, MAFK, and MAFF) proteins (MAF, v-maf musculoaponeurotic fibrosarcoma oncogene homolog) and binds to the ARE to induce the transcription of ARE-mediated genes (Motohashi et al. 2002). In the present investigation, EMSA and supershift assays showed that the NRF2 proteins in the nuclei bind to the ARE sequences of K16 promoter region after iAs exposure. iAs-induced c-Jun, however, does not bind to this ARE (Figure 4E). c-Jun may thus act on other AP-1 sites within the K16 promoter region. These results also suggest that other heterodimer partners of NRF2 are involved in the ARE regulation of K16 promoter region underlying iAs-mediated the K16 gene expression. Gel shifts with an K16Mut-ARE probe (harboring a mutation in the ARE GC box) clearly show that the ARE sequence in the K16 promoter, particularly the terminal GC dinucleotide, is essential for mediating iAs-induced K16 transactivation and NRF2 binding (Figure 4F). Several investigations have suggested that the GC nucleotides within the ARE are essential for both the basal and oxidative stress–induced activities of the ARE-related genes, NAD(P)H dehydrogenase quinone 1 (NQO1) and glutamate-cysteine ligase catalytic subunit (GCLC) (Wasserman and Fahl 1997; Wild et al. 1998). Our current results are consistent with these earlier studies in showing that the formation of the iAs-responsive NRF2/ARE complexes is reduced by a mutation in the GC box. Collectively, our present observations reveal a new molecular mechanism in which iAs-induced K16 gene expression is also regulated by activation NRF2/ARE pathways.
It has been widely accepted that oxidative stress disrupts sequestration of NRF2 by KEAP1, permits NRF2 translocation to the nucleus, and transactivates the expression of various NRF2-mediated genes (Dinkova-Kostova et al. 2002; McMahon et al. 2003). Our present study showed that iAs elongates the half-life of the NRF2 protein but has no effects upon KEAP1 expression (Figure 5). Other studies have also demonstrated that the production of NRF2 is increased by various inducers via posttranscriptional control (Nguyen et al. 2003; Stewart et al. 2003). Several earlier reports also indicated that either oxidative stress or antioxidant substances stabilize the expression of the NRF2 protein, either by directly modifying the cysteine residues on KEAP1 to disrupt the NRF2/KEAP1 complex (Dinkova-Kostova et al. 2002) or by facilitating the release of NRF2 through the phosphorylation of the NRF2/KEAP1 complex (Bloom and Jaiswal 2003). These findings are largely consistent with our present finding that iAs stabilizes the expression of NRF2 in HaCaT cells by elongating the protein half-life.
Recently, Wakabayashi et al. (2003) demonstrated that NRF2 accumulates in the nucleus at constitutively high levels and produces various cytoprotective genes in embryonic fibroblast- and liver-derived Keap1-null <span class="yellow">mice</span>. Surprisingly, these Keap1-deficient <span class="yellow">mice</span> also show a thicker stratum corneum epidermis, abnormal keratinization, and cornification in the esophagus and forestomach (hyperkeratosis). K6 was found to be strongly expressed in the esophageal epithelium of these <span class="yellow">mice</span>. These results indicate that K6 is also a target gene of NRF2. In addition, the promoter of the K6 gene bears a remarkable sequence similarity to the K16 promoter (Jiang et al. 1993). Therefore, we examined whether K16 gene expression is also regulated by NRF2. In the present study, the gene expression and transactivation of K16 were dramatically induced by transfection with WT-NRF2 via in HaCaT cells, clearly demonstrating that NRF2 acts as a direct transcriptional regulator of the K16 gene (Figure 6). In addition, we also showed that transfection of HaCaT cells with WT-NRF2 induces the expression of detoxification gene TXN (Figure 6A). NRF2 may thus have a major role to play in the development of hyperkeratosis, whereas the expression and induction of NRF2 is implicated in cell protection against a variety of genotoxic and cytotoxic effects. Hence, based on these results and on the findings from studies of Keap1 knockout <span class="yellow">mice</span>, iAs may both cause hyperkeratosis and induce detoxification enzymes via the modification of NRF2. Given that there are both beneficial and adverse effects of NRF2 activity, caution will therefore be needed when using antioxidants for prevention and therapy. Although further investigations are needed, we believe that our findings provide important clues for the design of future therapies for arsenic-mediated hyperkeratosis and for treatments involving the molecular targeting of NRF2.<br><br>Conclusion
Our findings clearly demonstrate that the induction of the K16 gene in <span class="yellow">human</span> keratinocytes by iAs depends on NRF2 activation. Our results thus represent a valuable initial effort to elucidate the relationship between the K16 gene and the NRF2 transcription factor, which may be responsible for hyperkeratosis.<br><br>
<h3>pmcA327937</h3>Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a.
Abstract
Attempts to prevent the urea conversion of a 200-230,000 molecular weight DNA polymerase alpha to a 150-170,000 molecular weight form by the inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70,000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70,000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit.<br><br><br><br>
 Volume 6 Number 10 1979 <br><br> Nucleic Acids Research <br><br> Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a Keith McKune and Andrew M.Holmes* <br><br> Biochemistry Department, Strathclyde University, The Todd Centre, 31 Taylor Street, Glasgow, G4 ONR, UK <br><br> Received I June 1979 ABSTRACT <br><br> Attempts to prevent the urea conversion of a 200-230, 000 molecular weight DNA polymerase a to a 150-170, 000 molecular weight form by the <br><br> inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70, 000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70, 000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit. <br><br> INTRODUCTION <br><br> In mammals DNA polymerase a is thought to be the replicative <br><br> enzyme, but due to low levels of activity, even in tissues actively making DNA, and to enzyme heterogeneity it has been found difficult to purify <br><br> However, small samples of DNA polymerase a have been highly purified <br><br> 2, 3, 4, 5 from several sources and partially characterised <br><br> Heterogeneity has been observed in DNA polymerase a from a var<br><br> 6           7 <br><br> iety of sources, including <span class="yellow">rat</span> liver and spleen , Hela cells, baby <span class="yellow">hamster</span> <br><br> 8                9, 10                   11 <br><br> kidney cells , <span class="yellow">mouse</span> myeloma    , <span class="yellow">Drosophila</span> embryos  and <span class="yellow">calf</span> thymus <br><br> 1,6, 12, 13 . We have previously observed several species of <span class="yellow">calf</span> thymus <br><br> DNA polymerase a differing in size and charge  . In order of elution from DEAE cellulose they are enzyme A1 (200-230, 000 molecular weight), A2 (200-230, 000), B (100-110, 000) and C (150-170, 000). A poly (dA). oligo <br><br> (dT)10 preferring enzyme, enzyrrme D (140-150, 000 molecular weight) elutes just after enzyme B. The A enzymes seem identical in all properties except their charge. 5.0-5. 5S enzymes, analagous to B, have been observed to <br><br> C Information Retrieval Limited 1 Falconberg Court London Wl V 5FG England <br><br> 3341 <br><br> Nucleic Acids Research <br><br> arise as the result of proteolytic action  ' , similarly the B enzyme above 14 <br><br> ,but the relationship of the other observed species is not clear, nor is <br><br> the problem of which, if any, of these species is the replicative enzyme, <br><br> although circumstantial evidence has implicated a DNA polymerase a to be <br><br> 15 the replicative enzyme in <span class="yellow">adenovirus</span> infected KB cells <br><br> We have previously shown that several of these enzymes are inter<br><br> 16 <br><br> convertible . In particular, mild urea treatment can convert both A <br><br> enzymes to C enzyme, with the loss of a subunit or fragment of 50-70, 000 molecular weight. The C enzyme retains DNA polymerase activity, but <br><br> does show differences compared to A enzyme in heat sensitivity and sensitivity to N-ethylmaleimide. <br><br> Highly purified samples of A and C enzymes when subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis have shown a corre<br><br> lation of DNA polymerase activity with a polypeptide band 155, 000 molecular weight . Contaminating material of 50-70, 000 molecular weight appeared <br><br> to obscure the putative 50-70, 000 subunit in gels of A enzyme. The conclusions drawn were that DNA polymerase a is a 155, 000 molecular weight <br><br> polypeptide (C enzyrrme) which can and does associated with material of 50<br><br> 70, 000 molecular weight to give A enzyme, and that enzyme B is a proteol<br><br> 14 <br><br> ytic degradation product . The relationship of enzyme D to the others is <br><br> not clear. <br><br> It is possible that during the enzyme isolation procedure proteolytic action may have taken place on a 200-230, 000 molecular weight enzyme releasing 155, 000 and 50-70, 000 fragments which remain in association <br><br> until urea treatment separates them. It is also possible that urea treatment itself renders the enzyme susceptible to contaminating proteases. Further experiments to ascertain whether or not this is the case and, if so, to prevent it, have been carried out. <br><br> A enzyme can be reconstituted from C enzyme formed by the action <br><br> of urea on A enzyme by concentrating it with the 50-70, 000 molecular weight material. The A enzyrme obtained in this manner is highly purified, as is <br><br> the C enzyme formed by the urea treatment, and has been used in attempts to show differences in synthetic template-initiator complex utilisation <br><br> 3342 <br><br> Nucleic Acids Research <br><br> b y the se two enzyme s . <br><br> MATERIALS AND METHODS <br><br> <span class="yellow">Calf</span> thymus was obtained from 10-16 week old <span class="yellow">calves</span> and frozen at <br><br> -20?C until required. Chromatographic media and chemicals were obtained <br><br> 6 <br><br> from sources previously referred to . Radioactive deoxynucleoside tri<br><br> phosphates were obtained from the Radiochemical Centre, Amersham, Bucks. Synthetic oligo and polynucleotides were obtained from P. L. Biochemicals <br><br> Inc., except for poly (dC) which was a gift from Dr. I.R. Johnston and pre<br><br> 17 <br><br> pared from oligo d(C)5 as described  . N-a-p Tosyl-L-lysine chloromethy<br><br> lketone HCl and phenylmethylsulphonylfluoride were obtained from Sigma and Trasylol from Bayer. <br><br> Except where indicated all buffers contained 20% w/v glycerol and <br><br> 1 mM dithiothreitol. Standard linear phosphate gradients were run between 0.03 M and 0.25 M potassium phosphate, pH 7.8. Gradient salt concentr<br><br> 2 <br><br> ations were measured using a conductivity meter as described . Urea was <br><br> prepared as a 4. 8 M solution in 20% w/v glycerol, stirred with Amberlite <br><br> MB3 and filtered. Dithiothreitol was added to a final concentration of 1 mM before use. <br><br> 6 DNA polymerase was assayed using activated DNA as described <br><br> except that the buffer was 50 mM tris HC1, pH 7.8. One unit of DNA polymerase activity incorporates ln mol [ H] dTMP into an acid insoluble form in one hour at 370C. Assays using synthetic template-initiator complexes were carried out at 30?C in 0. 12 ml. The template-initiator complexes <br><br> 6 <br><br> were prepared and assays processed as described . All assays contained <br><br> 1 mM dithiothreitol, 62. 5 p.g <span class="blue">bovine</span> serum albumin, 1 ,ug template-initiator complex, enzyme protein and the relevant [ H] deoxynucleoside triphosphate at 0.1 mM and 12-15 cpm/pmol. These assays were carried out at either pH 6.4 in 20 mM sodium-potassium phosphate, or at pH 7.8 in 50 mM tris HCl and contained either 10 mM MgCl2 or 1 mM MnCl2 as indicated. <br><br> Preliminary purification of DNA polymerase a to Fraction IV was as <br><br> 6 <br><br> described , the purification steps being phosphocellulose chromatography, <br><br> ammonium sulphate precipitation and gel filtration on Sepharose 6B. <br><br> 3343 <br><br> Nucleic Acids Research <br><br> Enzyme obtained from the DEAE cellulose step is referred to as Fraction <br><br> V enzyme. Samples were prepared for sodium dodecyl sulphate polyacrylamide gel electrophoresis and scanned as previously described2 <br><br> RESULTS AND DISCUSSION <br><br> (a)   Interconversion Studies <br><br> A enzyme was routinely converted to C enzyme by incubating 200500 units/ml of Fraction V A enzyme in 2.4 M urea in 0. 02 M potassium <br><br> phosphate, pH 7.8, for 60 minutes at 0?C. The mixture was then loaded on to a DEAE cellulose, washed with 0. 03 M potassium phosphate pH 7. 8, and enzyrme eluted either with the standard phosphate gradient or batchwise. Under these conditions usually about 50-60% of recovered activity was <br><br> enzyme C (Fig 1 a). Overall recovery was 70-80%. A enzyme was recon<br><br> 14 <br><br> stituted essentially as described . The flow through material from the <br><br> DEAE cellulose column after the urea treatment was loaded on to a 1 x 0. 8 cm phosphocellulose column, washed with 0.03 M potassium phosphate, pH 7.8, and the protein eluted with 0.25 M potassium phosphate, pH 7. 8. This material, the putative subunit, was vacuum dialysed with the C enzyme produced by the urea treatment, rechromatographed on DEAE cellulose and enzyme eluted batchwise (Fig 1 b). Recovery from this procedure was <br><br> usually 50-70% A enzyme. Overall recovery was 30-40% of the C enzyme dialysed. If the material eluted from the phosphocellulose by the 0. 25 M potassium phosphate was heated to 90?C for 5 minutes prior to vacuum <br><br> dialysis with the C enzyme the recovery from the DEAE cellulose column <br><br> was significantly higher (60-70% of the original C enzyme activity), but all recoverable DNA polymerase activity was C enzyme. This, together with the fact that the 60 minute treatment with urea has, on occasions, given <br><br> rise to a 50% increase in DNA polymerase activity prior to loading on to the DEAE cellulose column, would indicate that A enzyme is less active on <br><br> activated DNA than is C enzyme. If the DEAE cellulose flow through material came from urea treatment of A2 enzyme then A2 was produced on reconstitution; if from A1 then A1 was produced. <br><br> Although the mild urea treatment of A has been used to prepare C of <br><br> 3344 <br><br> Nucleic Acids Research <br><br> b <br><br> 120       180 C <br><br> I         I    ^ <br><br> Figure la  DEAE cellulose chromatography after 2.4 M urea treatment. <br><br> 4, 5000 units, 4.4 mg of Fraction II A2 enzyme were incubated with urea at a final concentration of 2.4 M for 60 minutes at 0?C, loaded on to a 5 x 1.4 cm DEAE cellulose column, the column was washed with 0. 03 M potassium phosphate, pH 7.8 and a 200 ml standard phosphate gradient applied. 2.3 ml fractions were collected and 10 Rl assayed for 5 minutes. (o-o) no <br><br> phenylmethylsulphonylfluoride (*-*) 3 mM phenylmethylsulphonylfluoride ( - ) phosphate gradient. <br><br> Figure lb  The reconstitution of A2 enzyme. 3000 units for C enzyme <br><br> derived by urea treatment of A2 were vacuum dialysed with the DEAE cellulose flow through material after phosphocellulose chromatography and <br><br> chromatographed on a 2 x 1. 2 cm DEAE cellulose column. After washing with 0.03 M potassium phosphate, pH 7.7, the enzymes were eluted batchwise with the above concentrations of potassium phosphate, pH 7.8. 1 ml fractions were collected and 10 .l assayed for 10 minutes. <br><br> 3345 <br><br> Nucleic Acids Research <br><br> high specific activity it is possible that conversion of the 200-230, 000 molecular weight enzymes to a 155- 170, 000 species maybe due to unfolding of the molecule to allow limited attack by contaminating proteases. Accord<br><br> ingly the urea conversion of A to C was investigated in the presence of certain protease inhibitors. The presence of the serine protease inhibitor <br><br> phenylmethylsulphonyl-fluoride in the incubation and chromatography buffers did not affect the conversion of A2to C (Fig 1 a). Likewise trasylol and Na-p Tosyl-L-lysine chloromethylketone HC1 had no effect. However, protease action could have occurred earlier in the purification procedure and the urea could be separating two fragments. Usually the DEAE cellulose profile shows that the majority of the enzyme activity is present as A <br><br> enzyme (Fig 2). In this instance A1 and A2 have not been separated. When the temperature of the material in the original blending procedure was kept below 0?C or phenylmethylsulphonylfluoride, N-a-p Tosyl-L-lysine chloromethylketone HCl or trasylol was included in the isolation buffers the DEAE cellulose profile was similar. The A enzymes were still capable of conversion by mild urea treatment to C enzyme, indicating that if protease activity is involved then it is not susceptible to these inhibitors. When the <span class="yellow">calf</span> <br><br> 20FI: Now ~ ~      ~      20 20Fr Na      06 <br><br> Figure 2  DEAE cellulose elution profile of <span class="yellow">calf</span> thymus DNA polymerase a. 47, 000 units, 97 mg of Fraction IV enzyme prepared from 415 g of <span class="yellow">calf</span> <br><br> thymus were loaded on to a 10 x 1.8 cm DEAE cellulose column, after washing with 0. 03 M potassium phosphate, pH 7. 8 a 400 ml standard phosphate gradient was applied. 5 ml fractions were collected and 10 ,ul assayed for 5 minutes. <br><br> 3346 <br><br> Nucleic Acids Research <br><br> thymus was allowed to warm up during the blending procedure, or the supernatant prior to phosphocellulose chromatography was heated to 37 ?C for 30 minutes, there was a marked decrease in the amount of A enzyme with a concomitant increase in the amount of B and C (unpublished observation). The presence of phenylmethylsulphonylfluoride, N- a-p T osyl- L-lysine <br><br> chloromethylketone HCl or trasylol under these conditions only had a marginal effect on the appearance of C enzyme, but did reduce the amount of B <br><br> enzyme. Heating the enzyme to 37?C after the phosphocellulose step had no effect on the DEAE cellulose elution profile. Attempts to convert A enzyme to C using trypsin have not been successful. A enzyme activity is lost without the appearance of any other species (unpublished observation), although <br><br> 14 the action of trypsin on C enzyme can give rise to small amounts of B <br><br> Although the conversion of A enzyme to C does not appear to be the result of serine protease action, proteases other than serine proteases <br><br> 18 <br><br> could have been responsible  . Also, the fact that A enzyme can be recon<br><br> stituted from C plus the flow through material from the DEAE cellulose after urea treatment does not necessarily mean we are dealing with two subunits as fragments produced by proteases mray be reassembled to give active <br><br> enzyme 9    . However, the fact that a 200-230, 000 molecular weight polypeptide band has never been observed in sodium dodecylsulphate polyacrylamride gels of highly purified A enzyme , or even cruder fractions of A <br><br> enzyrre (unpublished observation), may be significant. One might expect <br><br> some of the enzyme not to have been attacked by whatever is responsible for cleaving the molecule, if, indeed, this does happen. It would appear, therefore, that the A enzyme consists of subunits of 155, 000 and 50-70, 000 molecular weight with the small subunit having a slightly different charge in the <br><br> case of A  and A2. It has also been concluded that the heterogeneity in the <br><br> 1                                     ~~~~~~~~~~~~~~~~~3 <span class="yellow">mouse</span> myeloma DNA polymerase a fraction is not due to proteolysis . However, the 50-70, 000 molecular weight subunit has not yet been identified. <br><br> Sodiurrm dodecylsulphate polyacrylamide gels of reconstituted A enzyme have shown polypeptide bands at 150-160, 000 and 50-70, 000 molecular weight <br><br> (Fig. 3), but the ratio of staining intensity of the bands does not correspond to a 1: 1 relationship. The ratio of the two bands is variable but is usually between 1:2 and 1:3, indicating, perhaps, that more than one subunit of 50<br><br> 3347 <br><br> Nucleic Acids Research <br><br> TOP      123   4 5   6 <br><br> Figure 3  Scan of a 5% sodium dodecyl sulphate polyacrylamide gel of <br><br> reconstituted A2 enzyme. 1500 units of reconstituted A2 enzyme were subjected to polyacrylamide gel electrophoresis under non-denaturing conditions. The gels were sliced, enzyme extracted and assayed and the peak <br><br> fraction of DNA polymerase activity from three gels were pooled, subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis on a single <br><br> gel, stained and scanned at 2 volt sensitivity as described2. The molecular weight standards were: (1) <span class="blue">bovine</span> serum albumin dimer (134, 000), (2) ,3 galactosidase (130, 000), (3) phosphorylase a (94, 000), (4) <span class="blue">bovine</span> serum <br><br> albumin (68, 000), (5) pyruvate kinase (57, 000) and (6) lactate dehydrogenase (35, 000). <br><br> 70, 000 molecular weight can associate with the 155, 000 subunit. The proportion of the lower molecular weight polypeptide band is higher in the reconstituted A enzyrrme than in the C enzyme preparation from which it was formed, but even the C enzyme contained some material in the region. <br><br> Breakdown of material from 155, 000 to 50-70, 000 may be partly responsible for the contamination, but the presence of phenylmethylsulphonylfluoride in samples in preparation for sodium dodecylsulphate polyacrylamide gel electrophoresis has not been successful in preventing it. The <span class="yellow">human</span> KB cell <br><br> DNA polymerase a, equivalent to the C enzyme, has been reported to consist <br><br> 4 <br><br> of subunits of 76, 000 and 66, 000 molecular weight , but at no time have we <br><br> observed polypeptide bands at 76, 000 and 66, 000 rising and falling with <br><br> enzyme activity in any gels of <span class="yellow">calf</span> thyrrmus enzymes A1, A  and C (and <br><br> 2 unpublished observations). (b)    Template Studies <br><br> Use has been made of the urea conversion of A to C and of the reconstitution of A enzyme to obtain samples of highly purified DNA polymerase, <br><br> specific activity in excess of 50, 000 units/mg, in order to study the effect of this 50-70, 000 molecular weight subunit on the DNA polymerase subunit. Previous results have indicated that A enzyrrme is stabler to heat and less <br><br> 3348 <br><br> Nucleic Acids Research <br><br> 14, 21 <br><br> susceptible to N-ethylmaleimide than C enzyme  . It has been reported <br><br> that highly purified DNA polymerase a can be associated with a protein capable of binding to DNA containing no 3' OH ends and capable of being released during the DNA polymerase assay  . Attempts to dissociate the 50<br><br> 70, 000 molecular weight subunit from the polymerase subunit by incubating A2 with DNA polymerase reaction mixes containing activated DNA, poly <br><br> (dA-T) and poly (dT). oligo (A)10 followed by ultracentrifugation in high salt have been unsuccessful. Similarly A absorbed and eluted from either native or denatured DNA cellulose remained A enzyme. Both A and C enzymes <br><br> were eluted from the DNA celluloses by less than 0. 1 M NaCl so it does not appear that the subunit enhances the binding of the DNA polymerase subunit to DNA. However, there are differences in the response of A and C <br><br> enzymes to synthetic template-initiator complexes (Table 1). Even if one takes into account the fact that A enzyme is less active on activated DNA <br><br> than C enzyme (the addition of the 50-70, 000 subunit to the polymerase subunit appears to result in a decrease of about 30% of polymerase activity on activated DNA) the A enzyme is still more active on these templates. <br><br> Although there is a variation in activity on these template-initiator comp<br><br> lexes each time assays are carried out on them depending on the method of preparing the complexes and the base ratio of template to initiator, A enzyme always seems to be significantly more active than C. The A2 <br><br> enzyme at pH 7.8, with extra subunit(s) is clearly more effective on the <br><br> oligoribonucleotide initiator, oligo(A)10 than is enzyme C. In view of the proposed RNA initiation of Okazaki pieces this may indicate a role of this subunit in Okazaki piece synthesis, in that it may aid the DNA polymerase to'take over' from the RNA polymerase. Using poly (dA). oligo (dT) 10 <br><br> (A:T=20: 1) and following the incorporation of [ H] dTMP as a function of time a short lag was observed for C enzyme, but not for A2 (Fig. 4). <br><br> Similar re sult s we re obtained when poly (dC). oligo (dG) 1 0 (C: G= 5: 1) wa s <br><br> used as temrrplate-initiator, but not when poly (dT). oligo (A)10 (T:A=1: 1) was used. Neither enzyme showed a lag on this template-initiator complex or on activated DNA. Incorporation versus enzyme concentration also showed this lag for C enzyme on poly (dA). oligo (dT)10. It is not certain what causes <br><br> 3349 <br><br> Nucleic Acids Research <br><br> TABLE 1 Template utilisation by reconstituted A and C derived from A. <br><br> 3H] dNTP Divalent cation  A2 at pH    C at pH <br><br> Template                          _      6.4   7.8    6.4   7.8 Activated DNA         dTTP         Mg++            100          100 Activated DNA         dATP         Mg++             91          85 Activated DNA         dGTP         Mg++             85          82 poly(dA).oligo(dT)10  dTTP         Mg      66.0    4.0    20    91 <br><br> (A:T = 20: 1) <br><br> poly(dA). oligo(dT)   dTTP         Mu      19.0    4. 5  6. 5  2.5 <br><br> poly(dA. oig~10 (A:T = 20:1) <br><br> poly(dT). oligo(dA)10  dATP        Mg       2.5   <1 (1       (1 <br><br> (T:A = 5:1) <br><br> poly(dT) .oligo(dA)1  dATP         Mn ++   15.5   23.0   9. 0  3. 0 <br><br> poly(dT). oligo(A)10  dATP         Mg      15.0  230.0  10.0  77. 5 <br><br> (T:A = 1:1) <br><br> poly(dT) * oligo(A) 10  dATP       Mn      41.5   75. 5 19.5  21.5 <br><br> (T:A = 1:1) <br><br> poly(dC). oligo(dG)   dGTP         Mg++     6. 0   15.5  4. 5  7. 5 <br><br> (C:G = 5:1)   10 <br><br> poly(dC). oligo(dG)10  dGTP        Mn +     6. 0  29.5   5. 0  6. 5 <br><br> (C:G  = 5: 1)                                 .    . <br><br> Values given are relative to incorporation of [ H] dTMP at pH 7. 8 on activated DNA. For A2 this was 176.5 pmol, for C 232 pmol. dATP, dCTP, dGTP and dTTP were included for assays using activated DNA, only the deoxynucleoside triphosphate stated was used in the synthetic template<br><br> initiator complex assays. Assays were for 10 minutes. The buffers and concentrations of the divalent cations were as in Materials and Methods. <br><br> this lag, but the annealing of template to initiator is only transient '  and the DNA polymerase subunit may have difficulty in stabilising the complex and the 50-70, 000 molecular weight subunit may be able to help the poly<br><br> merase subunit to overcome this. Addition of the 50-70, 000 subunit to the polymerase subunit prior to assaying with these templates had no effect on activity and it may be that a preincubation period is required before the two 3350 <br><br> Nucleic Acids Research <br><br> Figure 4 Activity of A2 and C enzymes on poly (dA). oligo (dT)jO (A:T = <br><br> 20:1) as a function of time. 50 ,ul samples were withdrawn at various times from a 0. 6 inl incubation mix at 30?C and added to 0. 5 ml 0. 1 M sodium <br><br> pyrophosphate containing 100 ,ug/ml native <span class="yellow">calf</span> thymus DNA and processed for counting in the usual manner6. The mix contained 1 mM dithiothreitol, 50 mM tris HC1, pH 7.8 10 mM MgC12, 0. 1 mM [3H] dTTP (15 cpm/pmol), 312.5 ,ug <span class="blue">bovine</span> serum albumin, 5 Fg poly (dA). oligo (dT)  (A:T = 20:1) and enzyme protein (*-*) 3.3 units reconstituted A2, (o-o) 16. 0 units urea derived C enzyme. <br><br> subunits become fully associated. After the lag phase is over the C enzyme is still less active on these template-initiators than A enzyme . That is the A enzyme seems capable of elongating the initiator faster than the C enzyme. Experiments to determine whether the differences in rates of elongation of these template-initiator complexes are differences in processivity of the <br><br> enzymes under the different pH and divalent cation conditions are under way. <br><br> ACKNOWLEDGEMENTS <br><br> We thank the Medical Research Council for a research grant. <br><br> Communications concerning the paper should be sent to:<br><br> Dr. A. M. Holmes, Department of Biochemistry, University of Strathclyde, The Todd Centre, 31 Taylor Street, Glasgow G4 ONR, U.K. <br><br> 3351 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> (1)    Bollum, F.J. (1975). Prog. Nucleic Acid Res. Mol. Biol. 15, <br><br> 109-144. <br><br> (2)    Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1976). Eur. <br><br> J. Biochem. 62, 229-235. <br><br> (3)    Matsukage, A., Sivarajan, M. and Wilson, S.H. (1976). Biochem<br><br> istry 15, 5305-5314. <br><br> (4)    Fisher, P.A. and Korn, D. (1977). J. Biol. Chem. 252, 6528<br><br> 6535. <br><br> (5)    Fichot, O., Pascal, M., Mechali, M. and De Recondo, A, -M. <br><br> (1979). Biochim. Biophys. Acta, 561, 28-41. <br><br> (6)    Holmes, A.M., Hesslewood, I.R. and Johnston, I.R. (1974). Eur. <br><br> J. Biochemn. 43, 487-499. <br><br> (7)    Noy, G.P. and Weissbach, A. (1977). Biochim. Biophys. Acta, <br><br> 447, 70-83. <br><br> (8)    Craig, R.K. and Keir, H. M. (1975). Biochem. J. 145, 225-232. <br><br> (9)    Matsukage, A., Bohn, E.W. and Wilson, S.H. (1974). Proc. Natl. <br><br> Acad. Sci. U.S.A. 71, 578-582. <br><br> 110)   Hachmann, H.J. and Lezius, A.G. (1975). Eur. J. Biochem. 50, <br><br> 357- 366. <br><br> (11)   Brakel, C.L. and Blumenthal, A.B. (1977). Biochemistry, 16, <br><br> 3137-3143. <br><br> (12)   Momparler, R.L., Rossi, M. and Labitan, A. (1973). J. Biol. <br><br> Chem. 248, 285-293. <br><br> (13)   Yoshida, S., Konda, T. and Ando, T. (1974). Biochim. Biophys. <br><br> Acta, 353, 463-474. <br><br> (14)   Holmes, A.M., Hesslewood, I.P., Wickremasinghe, R.G. and <br><br> Johnston, I.R. (1977). Biochem. Soc. Symp. 42, 17-36. <br><br> (15)   De Jong, A., Van der Vliet, P. and Jansz, H.S. (1977). Biochim. <br><br> Biophys. Acta, 476, 156-165. <br><br> (16)   Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1975). <br><br> Nature (London) 255, 420-422. <br><br> (17)   Bollum, F. J. (1966). Procedures in Nucleic Acid Research <br><br> (Cantoni, G.L. and Davis, D.R. eds). pp 577-583, Harper and Row, New York. <br><br> (18)   Barrett, A.J. (1975). Proteases and Biological Control pp 467-482, <br><br> Cold Spring Harbor Laboratory, Cold Spring Harbor. <br><br> (19)   Richards, F. M. and Vithayathil, P.J. (1960). Brookhaven Symp. <br><br> Biol. 13, 115-134. <br><br> (20)   Lowe, P.A., and Malcolm, A.B.D. (1976). Eur. J. Biochem. 64, <br><br> 177- 188. <br><br> (21)   Hesslewood, I.P., Holmes, A.M., Wakeling, W.F. and Johnston, <br><br> I.R. (1978). Eur. J. Biochenr.. 84, 123-131. <br><br> (22)   Mechali, M. and De Recondo, A. -M. (1978). Biochim. Biophys. <br><br> Res. Comm, 82, 255-264. <br><br> (23)   Chang, L.M.S., Cassani, G.R. and Bollum, F.J. (1972). J. Biol. <br><br> Che. 247, 7718-7723. <br><br> (24)   Wickremasinghe, R.G. and Johnston, I.R. (1974). Biochim. <br><br> Biophys. Acta, 361, 37-52. <br><br> 3352 <h3>pmcA411032</h3>Down-regulation of the M6P/IGF-II receptor increases cell proliferation and reduces apoptosis in neonatal <span class="yellow">rat</span> cardiac myocytes
Abstract
Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a multi-functional protein that has been implicated in regulation of cell growth and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R, and cardiomyocyte apoptosis has been identified in a variety of cardiovascular disorders, such as myocardial infarction and heart failure. However, involvement of M6P/IGF2R in the pathogenesis of these conditions has not been determined. Thus, the objective of this study was to determine the role of M6P/IGF2R in regulation of cardiac myocyte growth and apoptosis.<br><br>Results
We down-regulated the expression of M6P/IGF2R in neonatal <span class="yellow">rat</span> cardiac myocytes and examined the effect on cell proliferation and apoptosis. Infection of neonatal cardiomyocytes with an <span class="yellow">adenovirus</span> expressing a ribozyme targeted against the M6P/IGF2R significantly reduced the level of M6P/IGF2R mRNA, as determined by RT-PCR and Ribonuclease Protection Assay (RPA). M6P-containing protein binding and endocytosis as well as the M6P/IGF2R-mediated internalization of 125I-IGF-II were lower in the ribozyme-treated cells than the control myocytes, indicating that the number of functional M6P/IGF2R in the ribozyme treated cells was reduced. Accordingly, a marked increase in cell proliferation and a reduced cell susceptibility to hypoxia- and TNF-induced apoptosis were observed in the ribozyme-treated cells.<br><br>Conclusions
These findings suggest that M6P/IGF2R may play a role in regulation of cardiac myocyte growth and apoptosis. Down regulation of this gene in cardiac tissues might be a new approach to prevention of cell death or promotion of mitogenesis for certain heart diseases.<br><br><br><br>Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a unique protein that interacts with multiple ligands, some of which are important growth regulatory factors [1]. The M6P/IGF2R participates in internalization and lysosomal degradation of IGF-II, a mitogen normally acting through the IGF-I receptor to stimulate cell proliferation [2]. The M6P/IGF2 receptor is required for the activation of TGF-β [3], a potent growth inhibitor for many cell types. This receptor is also involved in the binding, transport and activation of newly-synthesized lysosomal enzymes, such as cathepsins [4,5], which have been recently implicated in the induction of apoptosis [6]. On the basis of these functions, the M6P/IGF2R has been proposed to play a significant role in regulation of cell growth and apoptosis [7].
Apoptosis, or programmed cell death, is a tightly regulated process used to remove excess, hazardous or damaged somatic cells, and is crucial for the development, maintenance and survival of an organism. However, alterations in the control of apoptosis have also been shown to contribute to <span class="yellow">human</span> diseases. In fact, morphological and biochemical markers of apoptosis have been identified in a wide variety of cardiovascular disorders, including myocardial infarction and heart failure. This suggests that activation of apoptotic pathways contributes to cardiomyocyte loss and subsequent cardiac dysfunction in these conditions. A number of factors involved in cardiomyocyte apoptosis are currently known and include insulin-like growth factor-I (IGF-I), stress-activated protein kinases (SAPKs) and the anti-apoptotic Bcl-2 family [8]. There are indications that other factors may be involved in induction and regulation of cardiac apoptosis. However, these potential factors and their corresponding mechanisms have not been identified.
Several lines of evidence point to the potential involvement of M6P/IGF2R in cardiac myocyte proliferation and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R and transgenic <span class="yellow">mice</span> containing a homologous deletion of the M6P/IGF2R gene manifest ventricular hyperplasia due to an increase in cell number [9,10], suggesting that the M6P/IGF2R normally acts to suppress cardiac myocyte cell growth. It has also been shown that TGF-β, a potent growth suppressor whose activation requires the binding of latent TGF-β to M6P/IGF2R [3], is commonly upregulated in chronic heart failure [11]. Additional evidence for the involvement of M6P/IGF2R in regulation of apoptosis comes from studies of tumorigenesis. It has been shown that M6P/IGF2R expression is significantly reduced in a variety of tumors and loss of heterozygocity (LOH) at the M6P/IGF2R gene locus 6q26 have been found in breast, liver cancers and squamous cell carcinoma of the lung [12-15]. Although several studies have examined the effect of M6P/IGF2R over-expression on cell growth [7], it is not known whether down-regulation of this receptor protein leads to cellular protection against apoptosis.
Ribozymes are catalytic RNA molecules that cleave a complementary mRNA sequence [16], thereby inactivating specific mRNAs and suppressing gene expression in vitro and in vivo [17,18]. Ribozymes have been shown to be highly specific, efficient and stable. They can be packaged into viral vectors to enhance transfer into cells and to achieve longer expression compared with naked oligonucleotides. In the present study, we employed ribozyme technology to study the role of M6P/IGF2R in regulation of cardiac myocyte cell growth. A hammerhead ribozyme against the M6P/IGF2R mRNA was constructed and packaged in an adenoviral vector. We then examined the effect of ribozyme-mediated down-regulation of M6P/IGF2R expression on cell growth and hypoxia- and TNF-induced apoptosis.<br><br>Results
Cleavage reaction of the ribozyme in vitro
The M6P/IGF2R ribozyme we constructed has 13-bp binding arms complementary to the target site of M6P/IGF2R mRNA, and a catalytic core (Fig. 1A). To evaluate the bioactivity of the ribozyme and the accessibility of the target site, a cleavage reaction was performed in vitro. The substrates, [α-32P] labeled RNA transcripts containing 45 bp of M6P/IGF2R mRNA or an unmatched sequence, were incubated with the ribozyme as described (see Materials and Methods). The ribozyme cleaved only the specific M6P/IGF2R mRNA into the expected products. In the assay of time course, the hammerhead ribozyme was able to cleave 24.2% of the M6P/IGF2R target within 10 minutes of incubation, 50.3% of the M6P/IGF2R target within 40 minutes of incubation, and by 640 minutes, 80.8% of the M6P/IGF2R target was converted to the expected products (Fig. 1B). This ribozyme did not digest the unmatched sequence (Fig. 1B). These results indicate a high efficiency and specificity of the ribozyme in vitro.<br><br>Ribozymes down-regulate M6P/IGF2R expression in cardiac myocytes
To examine the ability of the ribozyme to reduce levels of M6P/IGF2R mRNA in cultured cardiac myocytes, total RNA was extracted from cells infected with Ad-GFP/IGF2R-Rz or Ad-GFP, and subjected to RT-PCR using M6P/IGF2R-specific primers. Primers specific for β-actin were added to a parallel reaction to serve as an internal standard. Cells were used 4 days after infection, with average infection efficiency of 70–80% (for which a viral dose used had minimal cytotoxicity). The RT-PCR product of M6P/IGF2R was 856 bp, and the β-actin product was 285 bp. As shown in Fig. 2A, the Ad-GFP/IGF2R-Rz-infected cells exhibited a significantly lower level of M6P/IGF2R mRNA than Ad-GFP-infected cells, with a reduction of about 50%. This result was confirmed by ribonuclease protection assay (RPA), in which GAPDH was used as a control (Fig. 2C &2D). There was no significant difference in the level of M6P/IGF2R mRNA between Ad-GFP-infected cells and uninfected cells (data not shown), indicating that infection with the <span class="yellow">adenovirus</span> itself did not alter the endogenous M6P/IGF2R mRNA level. The results demonstrated that the ribozyme was highly effective in suppressing M6P/IGF2R expression in cultured cardiac myocytes.<br><br>Effect of ribozyme expression on the functional activity of M6P/IGF2R
To determine the effect of the ribozyme on the functional activity of M6P/IGF2R, binding and internalization of exogenous 125I-IGF-II was measured in cells infected with Ad-GFP/IGF2R-Rz. As shown in Fig. 3A, cells infected with Ad-GFP/IGF2R-Rz showed a 54% reduction in 125I-IGF-II internalization when compared with the control cells (infected with Ad-GFP). We also examined the effect of the ribozyme on the M6P-binding activity of the M6P/IGF2R using the M6P-bearing lysosomal enzyme, β-glucuronidase, as a probe. The results showed that the maximal M6P-binding capacity of cells treated with the ribozyme was about 50% less than that of controls (Fig. 3B). Furthermore, we assessed the ability of cells to internalize exogenous β-glucuronidase after treatment with ribozyme. Similarly, the M6P-inhibitable endocytosis of β-glucuronidase by ribozyme-treated cells was about 52% less than that of control cells (Fig. 3C). These results confirm that the number of functional M6P/IGF2R in ribozyme-treated cells was reduced.<br><br>Adenoviral delivery of ribozymes increases the proliferation of cardiac myocytes
We examined the effects of the ribozyme on the growth of cultured neonatal <span class="yellow">rat</span> cardiac myocytes. Morphological evaluation showed a remarkable difference in growth pattern between Ad-GFP/IGF2R-Rz-infected cells and the control cells: the ribozyme-expressing cells formed larger and more spread colonies (Fig. 4). Assessment of cell proliferative activity by the MTT assay and counts of viable cells showed that the number of cardiac myocytes in ribozyme-expressing cultures was significantly higher than in control cultures (Fig. 5). These results indicate that treatment with M6P/IGF2R-ribozyme can promote cardiac myocyte proliferation.<br><br>Effect of M6P/IGF2R-ribozyme expression on apoptosis of cardiac myocytes
We examined the effects of ribozyme expression on TNF-α and hypoxia-induced apoptosis of cultured cardiac myocytes. After a 24 hr challenge with hypoxia, the number of apoptotic cells in M6P/IGF2R-Rz expressing cultures was 38% lower than in control cultures as determined by Hoechst staining (which highlights the nuclei of apoptotic cells) and ELISA (Fig. 6A, 7A). MTT analysis showed that the number of viable cells in ribozyme-treated cultures was 40% higher than in control cultures (Fig. 7A).
After treatment with TNF-α, as shown in Fig. 6B, a large number of control cells underwent apoptosis, as indicated by morphological changes (small round shape) and bright blue nuclear staining. There were significantly more apoptotic cells in control cultures than in cultures expressing the Ad-GFP/IGF2R-Rz. The number of apoptotic cells, as measured by the cell death ELISA assay, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 40%) lower than in cultures infected with Ad-GFP (Fig. 7B). Accordingly, the number of viable cells, as measured by MTT analysis, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 45%) higher than in cultures infected with Ad-GFP (Fig. 7B). These results are consistent with the hypothesis that decreasing M6P/IGF2R expression by ribozyme treatment can reduce cell apoptosis.<br><br>
Discussion
Some 62,000,000 Americans have one or more types of cardiovascular disease (CVD) and CVD is the leading cause (40.1%) of death in the United States. Myocardial infarction and heart failure, conditions accompanied by cardiac myocyte apoptosis, represent 23% of all CVDs and are a growing clinical challenge in need of novel therapeutic strategies. In this study, we investigated the M6P/IGF2R as a potential new therapeutic target for reduction of cardiac apoptosis and cardiac injury in these conditions.
Using ribozyme technology we down-regulated the expression of the M6P/IGF2R in neonatal cardiac myocytes. We then examined cell proliferation and apoptosis under normal conditions and post challenge with either hypoxia, a model of ischemia-reperfusion, or TNF-α, a cytokine implicated in the pathogenesis of chronic heart failure [19]. Our results demonstrate an association of a decrease in the expression and function of the M6P/IGF2R with increased cell proliferation and decreased cell susceptibility to hypoxia- and TNF-induced apoptosis. Expression of the ribozyme targeted against the M6P/IGF2R in cardiomyocytes resulted in down-regulation of M6P/IGF2R expression, as measured by RT-PCR and RPA, and of M6P/IGF2R function, as indicated by a decrease in internalization of 125I-IGF-II, and β-glucuronidase binding and endocytosis.
MTT analysis and viable cell counts showed that ribozyme-mediated down-regulation of M6P/IGF2R resulted in a marked increase in cell proliferation of cardiomyocytes, which normally express high levels of M6P/IGF2R [20] and have limited proliferative capabilities [21]. These results are consistent with the findings of previous knockout studies [9,10]. Since the M6P/IGF2R has multiple actions on cell growth, its proliferative effect on the heart cells observed in this study might involve multiple mechanisms. However, it is likely that unchecked IGF-II stimulation plays a key role in the effect. Because the M6P/IGF2R is believed to sequester and degrade IGF-II [2], a decrease in M6P/IGF2R expression and function could result in decreased degradation and hence increased bioavailability of IGF-II to the IGF-I receptor, which mediates the growth-promoting effect of IGF-II. Supporting evidence for the involvement of IGF-II in the proliferative effect resulting from loss of M6P/IGF2R function comes from studies of M6P/IGF2R knock-out <span class="yellow">mice</span>. M6P/IGF2R-null <span class="yellow">mice</span> display global hyperplasia that coincides with elevated levels of IGF-II. Most importantly, however, the lethal nature of an M6P/IGF2R-null phenotype is reversed in an IGF-II-null background [9]. Our results showing that ribozyme-mediated down-regulation of M6P/IGF2R lead to a decrease in IGF-II internalization support the above possibility. However, further investigation to confirm this mechanism is warranted.
More importantly, our results also showed that M6P/IGF2R down-regulation resulted in decreased sensitivity of cardiomyocytes to hypoxia- and TNF-induced apoptosis. There is evidence that lysosomal enzymes, such as cathepsins B and D contribute to hypoxia- and TNF-induced apoptosis in vitro [22-25] and in vivo [26,27]. The M6P/IGF2R has been shown to be involved in binding, transport and activation of lysosomal enzymes, including cathepsins [4,5]. Therefore, it is possible that down-regulation of the M6P/IGF2R results in improper trafficking and activation of cathepsins. This, in turn would eliminate the apoptotic cascades triggered by these enzymes under hypoxia and TNF stimulation and result in decreased sensitivity of cardiomyocytes to apoptosis.
It has also been shown that TNF stimulation involves the activation of TGF-β [28-30], a ligand of M6P/IGF2R that has been implicated in the progression of chronic heart failure [11,31]. Therefore, down-regulation of M6P/IGF2R expression could also lead to a decreased bioavailability of activated TGF-β, thereby decreasing the sensitivity of cardiomyocytes to the TNF/TGF-β apoptotic pathway. The detailed mechanism of the observed effects is unknown and requires further investigation.<br><br>Conclusions
The present study demonstrates that ribozyme-mediated down-regulation of expression and functional activity of the M6P/IGF2R results in a decrease in the susceptibility of cardiac myocytes to apoptotic stimuli. These findings suggest that this receptor might be involved in cardiac cell growth and apoptosis. The ability of the M6P/IGF2R ribozyme to reduce M6P/IGF2R expression and function in transfected cells verifies the utility of the ribozyme in studying the role of M6P/IGF2R in cardiomyocyte growth and apoptosis. In addition to its utility as a research tool, the ribozyme, with further exploration and development, might have potential application as a therapeutic agent to prevent cell death or promote mitogenesis for certain clinical conditions, such as, myocardial infarction and chronic heart failure.<br><br>Methods
Construction of recombinant M6P/IGF2R-RZ adenoviral vector
The nucleotide numbers of the <span class="yellow">rat</span> M6P/IGF2R sequence targeted by the hammerhead ribozyme is 1147–1160 after coding site (exon 9). The structure of the M6P/IGF2R hammerhead ribozyme is shown in Fig. 1. A 49 bp M6P/IGF2R ribozyme oligonucleotide, 5'-GAATTCCCC ACACTG ATGAGCCGCTTCGGCGGCGAAACATTCAAC GCGT-3' and the corresponding reverse complementary strand were synthesized. The fragments were subcloned to produce a plasmid containing a ribozyme against M6P/IGF2R. For construction of the recombinant <span class="yellow">adenovirus</span> containing the M6P/IGF2R-ribozyme (pAd-GFP/IGF2R-Rz), the segments containing the ribozymes were amplified by PCR and cloned into a pAdTrack-CMV vector and then recombined homologously with an adenoviral backbone pAdEasy 1 vector to generate (pAd-GFP/IGF2R-Rz), following the protocol described by He et al. [32]. The pAd-GFP/IGF2R-Rz carries both the IGF2R-Rz and GFP (as reporter) genes, each under the control of separate cytomegalovirus (CMV) promoters. Another viral vector, pAd-GFP, which carries the GFP gene only under the control of the CMV promoter, was generated and used as a control vector. The adenoviral vector DNA were linerized with Pac I and transfected into the replication-permissive 293 cells (E1A transcomplementing cell line) by using Lipofectamine (Life Technologies) to produce E1-deleted, replication-defective recombinant <span class="yellow">adenovirus</span> as described previously [33]. Large-scale amplification of recombinant <span class="yellow">adenovirus</span> in 293 cells was followed by purification using a discontinuous CsCl gradient. The constructs were confirmed by enzymatic digestion and DNA sequencing.<br><br>Transcription and cleavage reaction of ribozyme in vitro
Plasmids containing the ribozyme or the substrate (either 45 bp of M6P/IGF2R mRNA or an unmatched sequence 5'-GTGCTGTCTGTATG-3') were linearized with MluI, respectively. All transcripts were generated with T7 RNA polymerase (Promega). Substrate transcripts were labeled by incorporation of [α-32P] UTP (NEN Life Science Products, Inc.). Specific activity of the [α-32P] UTP (10 μCi/μl) and the base composition of each substrate molecule were used to calculate the substrate concentration. Ribozyme transcripts were quantified spectrophotometrically. (The half-life of the M6P/IGF2R target is about 280 minutes).
Cleavage reaction mixture contained substrate RNA (40 nM), increasing amounts of ribozyme (60 nM), 20 mM MgCl2 and 20 mM Tris-HCl, pH8.0, in a final volume of 10 μl. The mixture was incubated at 37°C for a time-course of cleavage reaction from 0, 5, 10, 20, 40, 80, 160, 320, to 640 minutes and the cleavage reaction was stopped by addition of loading buffer (80% formamide, 10 mM Na2EDTA, pH 8.0, and 1 mg/ml each bromophenol blue and xylene cyanol). Cleavage products were analyzed on a 15% polyacrylamide and 8M urea gel. Product and substrate fragments were quantitated by using NIH Imager.<br><br>Cell cultures and infection with Ad-GFP/Rz-IGF2R and Ad-GFP
Cardiac myocytes were isolated from 1-day-old newborn <span class="yellow">rats</span> using the Neonatal Cardiomyocyte Isolation System (Worthington). The isolated cells were plated in 6-well plates and cultured in F-10 medium containing 5% (vol/vol) FBS and 10% (vol/vol) <span class="yellow">horse</span> serum at 37°C in a tissue culture incubator with 5% CO2 and 98% relative humidity. Cells were used for experiments after 2–3 days of culture. Viral infections were carried out by adding viral particles at various concentrations (usually, 2 × 108 virus particles/ml) to culture medium containing 2% (vol/vol) FBS. Initially, optimal viral concentration was determined by using Ad-GFP to achieve an optimal balance of high gene expression and low viral titer to minimize cytotoxicity. After 24 hours of incubation, the infection medium was replaced with normal (15% vol/vol serum) culture medium. For treatment with IGF-II, cells were incubated with 50 ng/ml IGF-II after 24 hours infection with Ad-GFP/IGF2R-Rz or Ad-GFP. Four days after infection, cells were used for analysis of gene expression of M6P/IGF2R and its effect on cell growth and apoptosis.<br><br>Analysis of gene expression in cardiac myocytes
The M6P/IGF2R transcripts were determined by both RT-PCR and Ribonuclease Protection Assay (RPA). RT-PCR was performed using the GeneAmp EZ rTth RNA PCR kit (Roche). Total RNA was extracted from cultured cells using an RNA isolation kit (Qiagen,), according to the manufacturer's protocol. M6P/IGF2R transcripts were amplified using the primers (5'-GACAGGCTCGTTCTGACTTA-3') and (5'-CTTCCACTCTTATCCACAGC-3') specific to the M6P/IGF2R. Each RT-PCR assay was performed in triplicate and product levels varied by less than 3.2% for each RNA sample. Primers specific for β-actin cDNA were added to a parallel reaction to standardize for variations in PCR between samples. PCR products were resolved on a 1.0% agarose gel, visualized under UV light and quantitated using NIH Imager.
RPA was performed using the RPA III kit (Ambion, Austin, TX). Briefly, total RNA was extracted from cultured cells using a total RNA isolation reagent (TRIzol, Gibco BRL) according to the manufacturer's protocol. The plasmid containing the <span class="yellow">rat</span> M6P/IGF2R gene was linearized and used as a transcription template. Antisense RNA probes were transcribed in vitro using [33P]-UTP, T7 polymerase (Riboprobea System T7 kit, Promega), hybridized with the total RNA extracted from the <span class="yellow">rat</span> cardiomyocytes, and digested with ribonuclease to remove non-hybridized RNA and probe. The protected RNA·RNA was resolved on a denaturing 5% sequence gel and subjected to autoradiography. A probe targeting the GAPDH gene was used as an internal control.<br><br>Measurement of 125I-IGF-II internalization
Cells were incubated at 37°C for 2 hrs in serum-free F-10 culture medium containing 125I-labeled IGF-II (0.5 ng/ml) with or without excess unlabeled IGF-II (2 μg/ml). Following the incubation, the cells were washed three times with ice-cold PBS, and cell-associated radioactivity was determined by a γ counter. Specific internalized 125I-IGF-II was calculated by subtracting the count of samples with excessive unlabeled IGF-II from that without unlabeled IGF-II, and normalized to protein contents.<br><br>Beta-glucuronidase binding assay
Binding of β-glucuronidase was assayed as described previously [34,35]. Briefly, cells were permeabilized with 0.25% saponin in 50 mM Hepes (pH 7.0), 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="yellow">human</span> serum albumin, and 10 mM mannose-6-phosphate (M6P) for 30 minutes on ice. The cells were washed three times with ice-cold PBS containing 0.05% saponin. They were incubated with 20,000 units/ml β-glucuronidase from <span class="blue">bovine</span> liver (Sigma) in 50 mM Hepes (pH 7.5) containing 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="yellow">human</span> serum albumin, 0.5% saponin with or without 10 mM M6P overnight on ice. Cells were washed five times with ice-cold PBS containing 0.05% saponin and sonicated in 100 mM sodium acetate (pH 4.6). The protein concentration of solubilized cell extract was measured and enzyme activity was assayed as follows: for each reaction 50 ul cell extract were added to 500 ul of 100 mM sodium acetate (pH 4.0) containing 1 mM paranitrophenyl (PNP)-β-glucuronide (Sigma) as substrate. After an incubation period of 3 hours at 37°C, 500 ul 1 M Na2CO3 were added to each reaction and the absorbance was measured at 400 nm. Experimental values were compared to a standard curve that was constructed using 1–100 nM solutions of PNP (Sigma) in 500 ul 100 mM sodium acetate and 500 u1 1 M Na2CO3. Specific activity was calculated as nM of PNP produced/hour/mg of protein.<br><br>Beta-glucuronidase endocytosis assay
Beta-glucuronidase endocytosis assay was carried out as described previously [36]. Briefly, confluent cell cultures were washed twice with pre-warmed serum-free DMEM followed by incubation with DMEM containing 5 mg/ml <span class="yellow">human</span> serum albumin and 10 mM M6P for 20 minutes. Following incubation cells were washed 3 times with pre-warmed DMEM. Cells were then incubated in DMEM containing 5 mg/ml <span class="yellow">human</span> serum albumin alone or 4000 units β-glucuronidase with or without 10 mM M6P for 2 hours at 37°C. Following the incubation, the cells were washed 5 times with ice-cold PBS and subjected to enzyme activity assay as described above.<br><br>Cell proliferation assay (MTT assay and cell counts)
Cardiac myocytes were grown in culture plates (tissue culture grade, 12 wells, flat bottom) in a final volume of 1 ml serum-containing culture medium per well, in a humidified atmosphere (37°C and 5% C02) for 3 days. After infection with Ad-GFP/IGF2R-Rz or Ad-GFP, cells were incubated with or without 50 ng/ml IGF-II for 4 days. Following supplementation with IGF-II, 100 μl MTT labeling reagent (Roche) were added to each well and cells were incubated for 4 hours, followed by addition of 1 ml solubilization solution into each well. The plate was placed in an incubator at 37°C overnight. Spectrophotometrical absorbency of the samples was measured using an UV-visible Recording Spectrophotometer with wavelength of 550–690 nm. In addition, the total number of viable cells in each treatment was counted by trypan blue exclusion method using a hemocytometer.<br><br>Induction and analysis of cell apoptosis
Cells were infected with Ad-GFP or Ad-GFP/IGF2R-Rz. Seventy-two hours post infection, cells were treated with TNF (0.1 ng/ml) for 24 hrs or subjected to hypoxia. For induction of apoptosis by hypoxia, cell culture medium was changed to serum-free F-10 saturated with 95% N2/5% CO2 and cells were placed in a 37°C airtight box saturated with 95% N2/5% CO2 for 24 hrs. For normoxic controls, culture medium was changed to F-10/5%F BS/10% HS and cells were placed in a 37°C/5% CO2 incubator for 24 hrs before analysis.
Apoptotic cells were identified by Hoechst staining using the Vybrant™ Apoptosis Kit #5 (Molecular Probes) according to the manufacturer's protocol. In addition, after infection with Ad-GFP or Ad-GFP/IGF2R-Rz and challenge with either TNF or hypoxia, cell viability was assessed using the MTT assay Kit (Roche Molecular Biochemicals) and cell apoptosis was determined using the Cell Death Detection ELISA Kit assay (Roche Molecular Biochemicals) according to the manufacturer's protocol.<br><br>Statistical analysis
Students' t-test was used to evaluate the difference between two values. Each experiment was repeated at least three times. Statistical significance was accepted at the level of p < 0.05.<br><br>
List of abbreviations used
Ad-GFP, <span class="yellow">adenovirus</span> carrying GFP gene; Ad-GFP/IGF2R-Rz, <span class="yellow">adenovirus</span> carrying both the ribozyme against M6P/IGF2R and the GFP gene; GFP, green fluorescent protein; IGF-II, insulin-like growth factor II; M6P/IGF2R, mannose 6-phosphate/insulin-like growth factor II receptor; Rz, ribozyme.<br><br>Authors' contributions
ZC carried out construction of the ribozyme, production of the viruses, cellular experiments, biochemical assays and data analysis.
YG carried out the RPA assay and participated in the molecular biological studies.
JXK conceived of the study, participated in its design and coordination, and drafted the manuscript.<br><br>
<h3>pmcA1257442</h3>Identification of kinectin as a novel Behçet's disease autoantigen
Abstract
There has been some evidence that Behçet's disease (BD) has a significant autoimmune component but the molecular identity of putative autoantigens has not been well characterized. In the initial analysis of the autoantibody profile in 39 Chinese BD <span class="yellow">patients</span>, autoantibodies to cellular proteins were uncovered in 23% as determined by immunoblotting. We have now identified one of the major autoantibody specificities using expression cloning. Serum from a BD <span class="yellow">patient</span> was used as a probe to immunoscreen a λZAP expression cDNA library. Candidate autoantigen cDNAs were characterized by direct nucleotide sequencing and their expressed products were examined for reactivity to the entire panel of BD sera using immunoprecipitation. Reactivity was also examined with normal control sera and disease control sera from <span class="yellow">patients</span> with lupus and Sjögren's syndrome. Six independent candidate clones were isolated from the cDNA library screen and were identified as overlapping partial <span class="yellow">human</span> kinectin cDNAs. The finding that kinectin was an autoantigen was verified in 9 out of 39 (23%) BD <span class="yellow">patient</span> sera by immunoprecipitation of the in vitro translation products. Sera from controls showed no reactivity. The significance of kinectin as a <span class="yellow">participant</span> in autoimmune pathogenesis in BD and the potential use of autoantibody to kinectin in serodiagnostics are discussed.<br><br>Introduction
Behçet's disease (BD) is a systemic vasculitic disease typified by a triad of symptoms including recurrent oral ulcers, genital ulcers and uveitis. In addition, skin, joint, large vessels, nervous system and gastrointestinal systems may be involved. BD is a global disease but has the highest prevalence in the region along the ancient 'Silk Road' in China. The etiopathogenesis of the disease remains unclear but microbial agent triggers, environmental factors, genetic predisposition, neutrophil hyperfunction, endothelial cell dysfunction and immunological abnormalities involving both T and B cells have been implicated. Increasing amounts of research evidence supports the possibility that it is an immune-mediated vasculitis, and that abnormal T-cell and B-cell reactions and autoantigen-driven autoimmunity play pivotal roles [1]. Systemic lupus erythematosus (SLE) is the prototypic systemic autoimmune rheumatic disease with autoantibodies against cellular (particularly nuclear) antigens, some of which are critically implicated in the autoimmune pathology while others provide valuable serodiagnostic markers for the disease. Unlike the picture in SLE and other related rheumatic diseases, in BD, antinuclear antibodies and antibodies to neutrophil cytoplasmic antigens etc. are not present. To date, since neither a specific autoantibody nor pathognomonic pathological index is available to help establish the diagnosis of BD, it is largely or solely based on clinical manifestations [2], and a dilemma in diagnosis is not a rare occurrence in clinical practice. Nevertheless, since the 1960s, there have been reports of autoantibodies against certain unknown components of <span class="yellow">human</span> oral mucosa in sera of <span class="yellow">patients</span> with BD. Since then, sporadic reports on findings of autoantibodies in this disease have been described, such as antibodies to retinal antigen(s), heat shock protein (HSP) of some strains of <span class="yellow">Streptococcus sanguis</span> cross-reactive with <span class="yellow">human</span> HSP polypeptide [3], antibodies to endothelial cell antigens (AECA) and antibodies to α-tropomyosin [4,5], attesting to the complicated humoral immune disorders in this disease.
This investigation was aimed at defining target cellular autoantigens using time-tested and well-established molecular techniques. Immunoscreening of expression libraries using BD sera was used since this approach has been successfully employed in the characterization of many clinically relevant antigens in systemic rheumatic diseases such as SS-A/Ro [6-9] and SS-B/La [10] antigens in Sjögren's syndrome (SjS) and centromere antigen CENP-B [11] in scleroderma. In addition, we have been successful in using this strategy to identify interesting autoantigens that have other biological significance. Examples of these include NOR90/hUBF [12], p80-coilin [13], Golgi autoantigens [14-16] and, more recently, GW182 [17].<br><br>Materials and methods
<span class="yellow">Patients</span> and sera
The currently used empirical criteria for the diagnosis of BD in this study were the criteria proposed by the International Study Group for BD (abbreviated as 'International Criteria') [2]. The study subjects of 39 Chinese BD <span class="yellow">patients</span> comprised 17 males and 22 females, mean age 37 ± 11.3 years old, who were divided into two subgroups: 25 typical BD <span class="yellow">patients</span> (Group I, satisfying the International Criteria) and 14 clinically diagnosed BD <span class="yellow">patients</span> who had recurrent oral ulcers and one of the symptoms of genital ulcers, eye symptoms or skin lesions as defined by the International Criteria, as well as additional symptom(s) closely related to BD as listed in the International Criteria, that is, gastrointestinal ulcerations, deep vein thrombosis or arthralgia/arthritis without evidence that the latter symptoms might be related to any other disease (Group II, defined as 'probable BD' in this study). Disease controls included 10 <span class="yellow">patients</span> with SLE and 10 with SjS, all satisfying corresponding international classification criteria. All BD <span class="yellow">patients</span> and disease controls involved in the study were <span class="yellow">patients</span> treated at the Rheumatology Department of Ren Ji Hospital, Shanghai, China, where their clinical data and serum samples were collected. Twenty normal control sera were randomly selected from healthy blood donors working in the same hospital. This study was approved by the institution review board of Ren Ji Hospital which is affiliated with Shanghai Second Medical University, and each <span class="yellow">patient</span> involved gave informed consent. All serum samples were preserved at -20°C or -70°C until use.<br><br>Cell lines and cell extracts
HeLa (ATCC CCL 2.2) and T24 (<span class="yellow">human</span> transitional cell bladder carcinoma) were obtained from the American Type Culture Collection (Rockville, MD, USA). A <span class="blue">bovine</span> aortic endothelial cell line was kindly provided by Dr Eugene G Levin from the Scripps Research Institute (La Jolla, CA, USA). Cells were cultured in DMEM containing 10% <span class="yellow">calf</span> serum, harvested and extracted in Buffer A (150 mM NaCl, 10 mM Tris-HCl, pH7.2, 0.5% Nonidet P-40) with protease inhibitor (Complete™; Boehringer Mannheim, Indianapolis, IN, USA). For the preparation of whole cell extract, 10 volumes of Laemmli gel sample buffer [18] were added to the cell pellet, boiled for 3 min and stored at -20°C until use.<br><br>Western blot
Whole cell lysates from <span class="blue">bovine</span> aortic endothelial cell, HeLa and T24 cells were resolved individually by discontinuous 7.5% gel SDS-PAGE according to Laemmli's method [18]. Immunoblotting was performed as described by Towbin et al. [19] with modifications. Nitrocellulose strips were blocked with 3% nonfat milk in PBS containing 0.05% Tween-20 (PBS-T) and then incubated with BD <span class="yellow">patient</span> sera and normal control sera (1:100 dilution) at room temperature for 1 h. Filters were washed extensively with PBS-T to remove any unbound antibodies. Bound antibodies were detected with polyvalent, peroxidase-conjugated <span class="yellow">goat</span> anti-<span class="yellow">human</span> Ig and visualized by incubating the nitrocellulose strips in chemiluminescent reagents (NEN Life Science Products Inc., Boston, MA, USA) and exposing to Kodak XAR-5 films.<br><br>Screening of phage cDNA expression library with antibody probes
Serum from a BD <span class="yellow">patient</span> showing the highest antibody titer in immunoblotting was selected as a probe and used at a dilution of 1:300 for initial immunoscreening of approximately 106 recombinants from a T24 cDNA expression library. The latter was constructed in λZAPExpress vector (Stratagene, La Jolla, CA, USA) and screened as previously described [20-22]. All screenings were performed on duplicate isopropyl β-D-thiogalactoside (IPTG) pre-impregnated nitrocellulose filters, and immunoreactive clones were detected by chemiluminescence. Positive phages were subsequently plaque purified to 100% by two repeated rounds of screening at low plaque densities. Before screening the cDNA library, the BD serum was extensively adsorbed against bacteria and wild-type λZAP phage mixture to reduce background binding.<br><br>Analysis of candidate cDNAs
Purified candidate plaques were subcloned in vivo into pBK-CMV plasmids using ExAssist™ helper phage as recommended in the manufacturer's instructions (Stratagene). The recombinant pBK-CMV plasmids were then purified using QIAprep Spin Minprep Kit (Qiagen, Valencia, CA, USA). Restriction enzyme digestion of plasmids with EcoRI and XhoI and electrophoresis in a standard 1.0% agarose gel was used to analyze the length of cDNA insert of each candidate plasmid. The complete nucleotide sequence was determined using Bigdye terminator sequencing and a semi-automated sequencer model 377 (ABI, Foster City, CA, USA). Both nucleotide and deduced amino acid sequences were analyzed for similarity with known sequences using BLAST search [23] and ExPASy Proteomics tools . Secondary structure analysis for coiled-coil motifs was conducted with the software program COILS [24].<br><br>Immunoprecipitation of in vitro translation products
Candidate cDNA clones were used as templates for in vitro transcription and translation and the products were used as substrates for immunoprecipitation to confirm the specificity of reaction with BD sera. In brief, 1 μg of the pBK-CMV plasmid identified in the screening outlined above was added as template in a 50-μl reaction for the coupled in vitro transcription and translation reaction with a <span class="yellow">rabbit</span> reticulocyte lysate system (Promega, Madison, WI, USA) in the presence of 35S-methionine (Trans-35S label; ICN Biochemicals, Costa Mesa, CA, USA) and RNasin® Ribonuclease Inhibitor (Stratagene) as recommended by the manufacturer (Promega). Translation was carried out at 30°C for 1.5 h. Products were analyzed in a 12.5% gel SDS-PAGE and stored at -80°C for further immunoprecipitation analysis. The in vitro translation proteins were examined for reactivity by sera using immunoprecipitation described [8,25].<br><br>
Results and discussion
Autoantibody detection in sera from BD <span class="yellow">patients</span>
Initial examination of a group of 39 BD <span class="yellow">patients</span> using indirect immunofluorescence (IIF) on a HEp-2 cell substrate did not yield any characteristic nuclear or cytoplasmic staining patterns. BD is thought by some to be a vasculitic disease involving pathophysiology of endothelial cells, and antibody to endothelial cell antigen (AECA) has been reported. Reports on the prevalence of AECA have varied largely and alpha-enolase was reported as one of the putative target antigens [26]. In this study, the use of <span class="blue">bovine</span> aortic endothelial cells as substrate for IIF did not provide any additional data. However, Western blot analysis of the BD sera began to show some interesting autoreactivity using cell lysates from both HeLa and <span class="blue">bovine</span> aortic endothelial cells. HeLa cells were initially used for this analysis because they are commonly used in the laboratory as Western blot substrate. Fig. 1 illustrates the common reactivity to 49 kDa and 120 kDa proteins in the endothelial cell lysates. These antigens were also detected in HeLa and T24 cells; the latter cell line was analyzed because our laboratory at The Scripps Research Institute has produced an excellent expression cDNA library from the T24 line and the positive result with the T24 cell extracts allowed us to screen the T24 library. Ig isotype analysis showed that all reactivity was largely IgG antibodies. Since the 49 kDa and 120 kDa bands were observed in cell extracts from <span class="blue">bovine</span> as well as <span class="yellow">human</span> cell lines, these autoantigens might be evolutionarily conserved.
In total, nine out of 39 BD sera (23%) had autoantibody to the 49 kDa antigen and eight (20%) to the 120 kDa antigen. Four BD sera (10%) reacted with both proteins. Additionally, sera that showed common reactivity to the 120 kDa protein also demonstrated a common band that migrated at ~150 kDa, although it appeared weaker than the 120 kDa band. These antigens appeared to have different molecular weights than those of the known autoantigens in systemic rheumatic diseases. In addition, other reactive bands were detected but they were not as commonly shared as the 49 kDa and 120 kDa bands. The 49 kDa protein was shown to be distinct from 48 kDa SS-B/La or 50 kDa Jo-1 proteins (Fig. 1). The 120 kDa antigen was also shown to migrate differently from alanyl tRNA synthetase in another Western blot analysis (data not shown) and did not share any apparent crossreactive epitopes with the 49 kDa antigen. Western blot analyses of 20 normal control sera did not show the reactivities observed with BD sera. In order to further characterize these autoreactivities, a serum sample from the Group I definitive BD <span class="yellow">patients</span> with the strongest reactivity to 49 kDa and 120 kDa antigens (Fig. 1, lane 2) was selected as antibody probe for expression library screening.<br><br>Kinectin identified as a novel BD autoantigen
After screening 500,000 clones from the T24 cell λZAPExpress expression library, seven immunoreactive clones were isolated and plaque purified in two to three rounds to achieve 100% homogeneity. The cDNA inserts were subcloned in vivo into pBK-CMV plasmids, analyzed by restriction digestion using EcoRI and XhoI enzymes, and submitted to direct nucleotide sequencing across the polylinker arms. The cDNA inserts represented six independent clones designated BD41 (identical to BD44), BD481, BD42, BD47, BD482 and BD49. Their identities were established as overlapping partial cDNAs of <span class="yellow">human</span> kinectin, ranging from 1.9 kb to 3 kb (Fig. 2a). The full-length <span class="yellow">human</span> kinectin (GenBank accession number NM_182926[27]) has 4,816 bases containing an open reading frame coding 1,357 amino acid residues with molecular mass 156 kDa. All six cDNAs lacked the 5' portion of the kinectin sequence to different degrees but spanned a sequence of kinectin that extended to the 3'-untranslated region. Secondary structure analysis of kinectin protein using the program COILS identified a long region of α-helical coiled-coil domain that extended from amino acid residue 327 to the C-terminus (Fig. 2a, hatched boxes). In vitro coupled transcription and translation of BD44 and BD42 clones directed the synthesis of [35S]-methionine-labeled polypeptides that migrated at 95 and 60 kDa, respectively, in addition to smaller polypeptides (Fig. 2b). These products had predicted molecular weights of 103 kDa and 75 kDa.
Kinectin was initially identified in <span class="yellow">chick</span> embryo brain microsome as an integral membrane protein anchored in endoplasmic reticulum and involved in kinesin-driven vesicle motility along microtubules [28,29]. Kinectin consists of a 120-kDa and a 160-kDa polypeptide interacting through the α-helical coiled-coil domain to form a heterodimer [30]. The full-length kinectin is the 160 kDa polypeptide containing an N-terminal transmembrane helix followed by a bipartite nuclear localization sequence and two C-terminal leucine zipper motifs. We presume that the 120 kDa polypeptide detected in Western blot (Fig. 1) is the truncated version of the 160-kDa polypeptide, lacking the N-terminal first 232 amino acids [30]. The N-terminus of the 160-kDa polypeptide consists of a transmembrane domain that anchors kinectin to endoplasmic reticulum [30,31]. This 120 kDa polypeptide is probably the predominant form detected in the Western blot analysis (Fig. 1) because of its preferential solubility due to the omission of the N-terminal transmembrane domain.
Other functions for kinectin have been reported. <span class="yellow">Yeast</span> two-hybrid screen studies from several laboratories have revealed the interaction of the Rho family of GTPase with kinectin, and have shown the functional links among RhoG, kinectin and kinesin, with kinectin as a key effector of RhoG microtubule-dependent cellular activity [32]. Kinectin was also identified as an important constituent of integrin-based adhesion complexes, which link integrins to the cytoskeleton and recruit signaling molecules [33]. A new study reported that a kinectin isoform lacking a major portion of the kinesin-binding domain is very probably the most conservative form of kinectin; it does not bind kinesin but act as a membrane anchor for the translation elongation factor-1 delta in the endoplasmic reticulum [34].<br><br>Prevalence and specificity of anti-kinectin autoantibodies
The in vitro [35S]-methionine-labeled translation product of BD44, representing the largest recombinant kinectin fragment available, was used as the antigen substrate in an immunoprecipitation assay. Out of 39 BD <span class="yellow">patient</span> sera, nine (23%) recognized the BD44 translation product (Fig. 3), whereas sera from 20 normal controls, 10 SLE and 10 SjS <span class="yellow">patients</span> did not show reactivity. Among the nine anti-kinectin positive <span class="yellow">patients</span>, six (6/25, 24%) were from Group I (definitive BD) including the BD <span class="yellow">patient</span> whose serum was used in the immunoscreening of expression cDNA library, and three (3/14, 21.4%) <span class="yellow">patients</span> were from the Group II (probable BD) in this study. According to the Fisher Exact Probability calculation (P = 1.00), there is no statistically significant difference for antibody to kinectin between the two groups. The combined data substantiated the finding that kinectin is an autoantigen that can be recognized by sera from 23% of Chinese BD <span class="yellow">patients</span> in this study with at least one immunoreactive region or autoepitope residing within the BD44 encoded polypeptide.
Currently, there are more than six diagnostic/classification criteria for BD, among which the International Criteria have been applied most extensively due to its relatively high sensitivity (91%) and specificity (96%) [2]. As discussed above, differential diagnosis of BD might be confusing in clinical practice since no specific laboratory test is available, and some <span class="yellow">patients</span> may have symptoms and signs strongly suggestive of BD but do not fully satisfy the International Criteria, as in the Group II (probable BD) <span class="yellow">patients</span> in our study group. A number of investigators have pointed out that a comprehensive analysis of the clinical data for a given <span class="yellow">patient</span> is very important for correct clinical diagnosis of BD, and that classification/diagnosis criteria, including the International Criteria, should be followed but should not be exclusive. The observation that three out of 14 <span class="yellow">patients</span> in the probable BD group also had antibody to kinectin and the similar percentage of positive reactors between this group and Group I (21.4% versus 24%) supports this notion. The further use of non-clinical parameters such as immunological biomarkers as adjuncts to identify BD <span class="yellow">patients</span> could be of help in the classification of this disease entity
While our work was ongoing, anti-kinectin antibodies were reported in sera from <span class="yellow">patients</span> with hepatocellular carcinoma (HCC) [35,36] and aplastic anemia [37,38]. The first HCC report [35] identified kinectin as a tumor-associated antigen from the screening of an autologous cDNA library constructed from the cancer of a 30-year-old <span class="yellow">patient</span> from Guangxi, China. This report stated that four out of five HCC <span class="yellow">patients</span> tested were positive for anti-kinectin antibody [35]. In 2004, another laboratory also reported the cloning of kinectin as a tumor-associated antigen from a (presumably) different 30-year-old Chinese HCC <span class="yellow">patient</span> [36]. In contrast, anti-kinectin antibodies were not detected in other studies of HCC <span class="yellow">patients</span> associated with our laboratory [39,40]. The reports of anti-kinectin antibodies in aplastic anemia are also very interesting [37,38]. The initial report by Hirano et al. identified kinectin by screening an aplastic anemia <span class="yellow">patient</span> for candidate antigens using a Clontech <span class="yellow">human</span> fetal liver cDNA expression library and it was concluded that seven out of 18 aplastic anemia <span class="yellow">patients</span> were positive for anti-kinectin while none of the normal or disease controls had this antibody [37]. In their recent report, Hirano et al. reported that anti-kinectin antibodies were found in 39% of aplastic anemia <span class="yellow">patients</span> from the United States but only in three out of 30 (10%) cases in Japan [38]. In our study reported here, kinectin antibodies were only detected in BD <span class="yellow">patients</span> and not in normal controls and SLE and SjS disease controls. None of the BD <span class="yellow">patients</span> with anti-kinectin had signs of HCC or aplastic anemia at the time of diagnosis and at up to 4 years of follow-up. Mapping of epitope(s) recognized by anti-kinectin antibodies may shed light on the question of whether different autoepitopes reside within the kinectin molecule recognized by sera from different diseases.<br><br>Kinectin – a new member of coiled-coil cytoplasmic autoantigens
We have recently reviewed the literature on the growing number of cytoplasmic autoantigens rich in α-helical coiled-coil domains as typified from our study of Golgi autoantigens [41]. Golgi autoantigens are generally high molecular weight proteins between 100 and 350 kDa and rich in coiled-coil domains in the central region with non-coiled-coil or globular domains at both N and C termini. Golgi autoantigens are displayed on the cytoplasmic face of the Golgi complex and are not localized to apoptotic blebs during apoptosis [42]. Giantin, the highest molecular weight Golgi autoantigen reported, is the predominant target of <span class="yellow">human</span> anti-Golgi complex antibodies and multiple non-cross-reactive epitopes have been mapped spanning the 350 kDa protein [43]. Other high molecular weight autoantigens with similar features have been reported in cytoplasmic and mitotic organelles suggesting that these selected proteins become autoimmunogenic based on their subcellular association and molecular features [41]. For example, in the endosomal compartment, the two known autoantigens are early endosomal protein EEA1 (180 kDa) [44] and CLIP-170 (170 kDa) [45]. There is also a series of centrosomal autoantigens identified as coiled-coil-rich proteins including pericentrin, a 220 kDa protein [46], ninein, a protein with alternatively spliced products of 245 and 249 kDa [47], Cep250 (250 kDa) and Cep110 (110 kDa) [48]. Centromere autoantigens have been described but the two interesting ones related to this discussion are CENP-E [49] and CENP-F [50]; both are high molecular weight proteins (312 to 400 kDa) and have the same type of overall structure as discussed above. NuMA is another large coiled-coil protein located at the mitotic spindle pole and is the most common target autoantigen in sera with mitotic spindle apparatus staining [51]. Non-muscle myosin (~200 kDa) is a cytoskeletal autoantigen [52] that falls in the same group of high molecular weight and coiled-coil-rich autoantigens. These endosomal, centrosomal, mitotic apparatus and intracellular autoantigens are, like the golgins, proteins with high molecular weights and an overall high content of coiled-coil domains. The combination of these two physical features in autoantigens may contribute to the induction and production of autoimmune antibodies in certain disease states. Kinectin is an integral membrane protein largely confined to the endoplasmic reticulum [28,31] and it fits into this new category of autoantigens that are large coiled-coil rich proteins (≥100 kDa) in the cytoplasm.<br><br>
Conclusion
Here we report the detection of kinectin autoantibody in 23% of Chinese <span class="yellow">patients</span> with BD. The identity of kinectin as a BD-related autoantigen has not been reported to date. Autoantibody reaction against kinectin in BD observed in this study further confirms the autoimmune involvement in BD and may provide new inroads into elucidating the immunopathogenesis of the disease. In an effort to clarify the association of BD with antibody to kinectin, it is essential to measure antibody to kinectin in larger <span class="yellow">patient</span> populations including both BD, probable BD and important autoimmune rheumatic diseases such as SLE, SjS, rheumatoid arthritis etc., as well as those diseases not easily differentiated from BD, such as recurrent aphthous oral ulcer, Reiter's syndrome, inflammatory bowel diseases etc. On the other hand, further analysis of the association of anti-kinectin antibody with different manifestations or disease 'subtypes' of BD is another important project. Anti-kinectin is clearly only one of the antigen-antibody systems identified because there were many other antibodies observed in the Western blot analysis of BD sera. Using other sera for immunoscreening would probably lead to the identification of other potentially important antigen-antibody systems.<br><br>Abbreviations
AECA = antibody to endothelial cell antigen; BD = Behçet's disease; DMEM = Dulbecco's modified Eagle's medium; HCC = hepatocellular carcinoma; HSP = heat shock protein; IIF = indirect immunofluorescence; PBS = phosphate buffered saline; SjS = Sjögren's syndrome; SLE = systemic lupus erythematosus.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YL performed the study and drafted the manuscript. PY provided technical help throughout the study. SLC and EMT conceived the study, participated in the design and helped in the analysis of the data. EKLC participated in the design of the study, interpreted data and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1779441</h3>Identification of a <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts
Abstract
Increased bone resorption mediated by osteoclasts causes various diseases such as osteoporosis and bone erosion in rheumatoid arthritis (RA). Osteoclasts are derived from the monocyte/macrophage lineage, but the precise origin remains unclear. In the present study, we show that the purified CD16- <span class="yellow">human</span> peripheral blood monocyte subset, but not the CD16+ monocyte subset, differentiates into osteoclast by stimulation with receptor activator of NF-κB ligand (RANKL) in combination with macrophage colony-stimulating factor (M-CSF). Integrin-β3 mRNA and the integrin-αvβ3 heterodimer were only expressed on CD16- monocytes, when they were stimulated with RANKL + M-CSF. Downregulation of β3-subunit expression by small interfering RNA targeting β3 abrogated osteoclastogenesis from the CD16- monocyte subset. In contrast, the CD16+ monocyte subset expressed larger amounts of tumor necrosis factor alpha and IL-6 than the CD16- subset, which was further enhanced by RANKL stimulation. Examination of RA synovial tissue showed accumulation of both CD16+ and CD16- macrophages. Our results suggest that peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct responses to RANKL. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for osteoclastogenesis. Blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Introduction
Rheumatoid arthritis (RA) is an autoimmune disease characterized by chronic inflammation and proliferation of the synovium in multiple joints. A large number of inflammatory cells, including T cells, B cells, macrophages and dendritic cells, accumulate in the affected synovium, and these inflammatory cells, together with fibroblast-like synoviocytes, express various cytokines, such as tumor necrosis factor alpha (TNFα), IL-6 and receptor activator of NF-κB ligand (RANKL), which are known to induce differentiation and activation of osteoclasts. The inflammatory synovial tissue, known as pannus, invades the articular bone and causes focal bone erosion, which is the hallmark of RA. Histopathologically, osteoclasts are present at the interface of the pannus and bone. Interestingly, the deletion of RANKL or c-Fos gene, which is important for osteoclastogenesis, results in minimal bone destruction in <span class="yellow">mouse</span> models of arthritis [1,2]. Furthermore, other studies indicated that inhibition of osteoclastogenesis by osteoprotegerin, a decoy receptor for RANKL, limits bone destruction in experimental models of arthritis. These studies suggest that osteoclasts are involved in focal bone erosion in RA [3].
Osteoclasts are derived from the monocyte/macrophage lineage. It is reported that osteoclast precursors reside in <span class="yellow">human</span> peripheral blood monocytes [4,5]. A marked increase of the circulating osteoclast precursors was demonstrated in <span class="yellow">patients</span> with erosive psoriatic arthritis as well as in arthritic TNFα transgenic <span class="yellow">mice</span> [6,7]. It was also shown that peripheral monocytes differentiate into osteoclasts when seeded on RANKL/osteoclast differentiation factor-producing RA synovial fibroblasts [8]. In addition, RA synovial macrophages thought to originate from peripheral blood monocytes were shown to differentiate into osteoclasts [9,10]. Monocytes are therefore involved not only in synovial inflammation, but also in bone remodeling as potential precursors for synovial macrophages and osteoclasts.
<span class="yellow">Human</span> peripheral blood monocytes consist of two major subsets, CD16+ and CD16-, comprising 5–10% and 90–95% of the monocytes, respectively. These two subsets exhibit different chemotaxis activities and potential of cytokine production [11,12]. Moreover, activation of the Toll-like receptor induces distinct subsets, CD1b+ dendritic cells and DC-SIGN+ (dendritic cell-specific C-type lectin ICAM-3-grabbing nonintegrin) macrophages from CD16+ and CD16- monocytes, respectively [13]. It has not been revealed, however, which monocyte subset develops into osteoclasts.
In the present study, we determined the <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts, and revealed that each subset exhibits a different response for osteoclastogenic stimuli.<br><br>Materials and methods
Purification of peripheral blood monocytes
Peripheral blood monocytes from healthy donors were collected using Ficoll-Conray (Imuuno-Biological Laboratories, Gunma, Japan) gradient centrifugation. Negative selection of monocytes was performed using MACS microbeads (Miltenyi Biotec, Auburn, CA, USA) according to the protocol supplied by the manufacturer.
The purified monocytes were separated into two subsets, CD16+ and CD16- monocytes, using CD16 MicroBeads (Miltenyi Biotec). Flow cytometry analysis using FITC-conjugated <span class="yellow">mouse</span> anti-CD14 mAb (MY4; Bechman Coulter, Fullerton, CA, USA) and phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8; BD Biosciences, San Jose, CA, USA) showed that the purities of the CD16+ and CD16- monocytes were more than 90% and 92%, respectively.
For the other experiment, monocytes were purified using CD14 MicroBeads (Miltenyi Biotec), and then stained either with FITC-conjugated <span class="yellow">mouse</span> anti-CD33 mAb (MY9; Bechman Coulter) or phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8). Cell sorting of the stained cells was performed using a FACS Vantage cytometer (BD Biosciences) or a MoFlo cell sorter (Dako, Glostrup, Denmark).<br><br>Osteoclast differentiation
Purified CD16+ and CD16- monocytes (5 × 104 cells/well) were incubated in 96-well plates in αMEM (Sigma, St Louis, MO, USA) with heat-inactivated 10% fetal <span class="blue">bovine</span> serum (FBS) (Sigma) or with Ultra-Low IgG FBS (IgG < 5 μg/ml; Invitrogen, Carlsbad, CA, USA), and where indicated with M-CSF + RANKL (Peprotech, Rocky Hill, NJ, USA).
For the other experiments, varied numbers of CD16+ monocytes (1 × 103, 2.5 × 103, 5 × 103) were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured in 96-well plates in αMEM with heat-inactivated 10% FBS. The medium was replaced with fresh medium 3 days later, and after incubation for 7 days the cells were stained for tartrate-resistant acid phosphatase (TRAP) expression using a commercial kit (Hokudo, Sapporo, Japan). The number of TRAP-positive multinucleated cells (MNC) in three randomly selected fields examined at 100× magnification or the total number of TRAP-positive MNC per well was counted under light microscopy.<br><br>Resorption assay
Monocytes were seeded onto plates coated with calcium phosphate thin films (Biocoat Osteologic; BD Biosciences) and were incubated with RANKL (40 ng/ml) + M-CSF (25 ng/ml) for 7 days. The cells were then lysed in bleach solution (6% NaOCl, 5.2% NaCl). The resorption lacunae were examined under light microscopy.<br><br>Enzyme-linked immunosorbent assay
Purified monocytes were cultured in 96-well plates where indicated either with RANKL or M-CSF for 24 hours. Concentrations of TNFα and IL-6 in the culture supernatant were measured with an ELISA kit (BioSourse International, Camarillo, CA, USA). For experiments of matrix metalloproteinase (MMP)-9 and TRAP-5b, culture supernatants were collected on day 7 and the concentrations of these enzymes were measured using an MMP-9 ELISA kit (Amersham Biosciences, Piscataway, NJ, USA) or a TRAP-5b ELISA kit (Suomen, Turku, Finland).<br><br>Reverse transcriptase-polymerase chain reaction
Monocytes (1 × 106 cells/well) were cultured in six-well plates with M-CSF alone or with M-CSF + RANKL for 3 days. Total RNA was extracted using RNeasy Micro (Qiagen, Valencia, CA, USA). The RNA was then treated with DNase I (Qiagen). The oligo(dT)-primed cDNA was synthesized using Superscript II reverse transcriptase (Invitrogen). The amount of cDNA for amplification was adjusted by the amount of RNA measured by an optical density meter and also by β-actin or GAPDH PCR products. One microliter of cDNA was amplified in a 50 μl final volume containing 25 pmol appropriate primer pair, 10 pmol each of the four deoxynucleotide triphosphates, and 5 units FastStart Taq DNA Polymerase (Roche, Manheim, Germany) in a thermal cycler (PTC-200; MJ GeneWorks, Waltham, MA, USA).
The PCR conditions were 25–40 cycles of denaturation (95°C for 30 s), annealing (60–62°C for 1 min) and extension (72°C for 1 min). The sequences of the primers are presented in Table 1. The PCR products were separated by electrophoresis through 2% agarose gel.<br><br>Western immunoblot analysis
Purified monocytes were cultured for 3 days in the presence of 40 ng/ml M-CSF with or without 25 ng/ml RANKL. Cells were lysed in RIPA Lysis buffer (upstate, Lake Placid, NY, USA) containing protease inhibitors (Roche) for 15 min at 4°C. A total of 20 μg protein was boiled in the presence of 6 × sodium dodecyl sulfate sample buffer, and was separated on 7.5% or 10% sodium dodecyl sulfate-polyacrylamide gel (ATTO, Tokyo, Japan). Proteins were then electrotransferred to a polyvinylidene fluoride microporous membrane (Millipore, Billerica, MA, USA) in a semidry system. Membranes were incubated in 10% skim milk prepared in phosphate-buffered saline (PBS) containing 0.1% Tween 20, and were subjected to immunoblotting. Antibodies used were <span class="yellow">goat</span> anti-RANK antibody (Techne Corporation, Minneapolis, MN, USA), <span class="yellow">goat</span> anti-c-fms antibody (R&D systems, Minneapolis, MN, USA), and <span class="yellow">mouse</span> anti-β-actin mAb (AC-15; Sigma). Peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">goat</span> IgG antibody (Dako) or peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">mouse</span> IgG antibody (Dako) was used as the second antibody. The signals were visualized by chemiluminescence reagent (ECL; Amersham Biocsiences, Little Chalfont, UK).<br><br>Cell surface expression of c-fms
The following mAbs were used for analysis of c-fms expression: Alexa 647-conjugated anti-CD14 mAb (UCHM1; Serotec, Oxford, UK), FITC-conjugated anti-CD16 mAb (3G8; Bechman Coulter) and phycoerythin-conjugated anti-c-fms mAb (61708; R&D systems). Alexa 647-conjugated <span class="yellow">mouse</span> IgG2a (Serotec), FITC-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) and phycoerythin-conjugated <span class="yellow">mouse</span> IgG1 (Bechman Coulter) were used as isotype controls. Peripheral blood monocytes (1 × 105 cells) were incubated with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and were then stained with three fluorochrome-labeled mAbs for 45 minutes on ice. The stained cells were analyzed with a FACS Calibur (BD Biosciences).<br><br>Immunofluorescent staining
Monocytes (8 × 104 cells/well) were allowed to adhere on 96-well plates overnight or were cultured with M-CSF and RANKL for 2–4 days. The cells were fixed in acetone and then stained with anti-αvβ3 mAb (LM609; Chemicon, Temecula, CA, USA) or <span class="yellow">mouse</span> IgG1 (11711; R&D Systems) as an isotype-matched control. Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes, Eugene, OR, USA) was used as the second antibody. TOTO-3 (Molecular Probes) was used for nuclear staining.<br><br>Flow cytometric analysis of p38 MAPK and ERK1/2 phosphorylation
Purified monocytes were cultured in the presence of 25 ng/ml M-CSF for 3 days, and were either left unstimulated or were stimulated with 40 ng/ml RANKL at 37°C. Stimulations were stopped by adding an equal volume of PhosFlow Fix Buffer I solution (BD Biosciences) to the cell culture. After incubation for 10 minutes at 37°C, the cells were permeabilized by washing twice at room temperature in PhosFlow Perm/Wash Buffer I (BD Biosciences). A total of 1 × 105 cells was then Fc blocked with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and was stained with Alexa Fluor 647-conjugated mAb either to phospho-p38 MAPK (T180/Y182) or to phospho-ERK1/2 (T202/Y204) (BD Biosciences) for 30 minutes at room temperature. Alexa Fluor 647-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) was used as an isotype control. The cells were washed in PhosFlow Perm/Wash Buffer I, and were analyzed by flow cytometry, as already described.<br><br>RNA interference
RNA oligonucleotides (iGENE, Tsukuba, Japan) were designed based on the algorithm that incorporates single nucleotide polymorphism and homology screening to ensure a target-specific RNA interference effect. The following sense and antisense oligonucleotides were used: integrin β3, 5'-GCU UCA AUG AGG AAG UGA AGA AGC A-AG and 3'-UA-CGA AGU UAC UCC UUC ACU UCU UCG U; randomized control, 5'-CGA UUC GCU AGA CCG GCU UCA UUG C-AG and 3'-UA-GCU AAG CGA UCU GGC CGA AGU AAC G; and lamin, 5'-GAG GAA CUG GAC UUC CAG AAG AAC A-AG and 3'-UA-CUC CUU GAC CUG AAG GUC UUC UUG U.
CD16- monocytes (8 × 104 cells/well) were incubated in 96-well plates in optimem (Invitrogen). After 1 hour, siRNAs were transfected into the cells using oligofectamine (Qiagen) based on the method recommended by the manufacturer. After 2 hours, the cells were washed once with PBS, followed by the addition of αMEM supplemented with 10% FBS, M-CSF and RANKL. After a 2-day incubation, the β3 mRNA expression was analyzed by RT-PCR with different PCR cycles, as described earlier. Immunofluorescent staining for the αvβ3 heterodimer was also performed as described above, and numbers of αvβ3-positive cells were counted in randomly selected three fields at 100× magnification. Seven days after the transfection of siRNAs, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Inhibition of osteoclastogenesis with cyclic RGDfV peptide
CD16- monocytes were incubated in 96-well plates with M-CSF + RANKL for 2 days. A medium containing either cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) (Calbiochem, San Diego, CA, USA) or dimethyl sulfoxide was then added. After incubation for a further 5 days, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Immunohistochemistry
Synovial tissue samples were obtained during total knee joint replacement surgery from four RA <span class="yellow">patients</span>. Signed consent forms were obtained before the operation. The experimental protocol was approved by the ethics committee of the Tokyo Medical and Dental University. RA was diagnosed according to the American College of Rheumatology criteria [14].
Double immunofluorescent staining for CD68 and CD16 antigens was conducted on optimal cutting temperature-embedded sections of frozen synovial samples. Eight-micrometer-thick cryostat sections of RA synovium were fixed in acetone for 3 minutes and were then rehydrated in PBS for 5 minutes. The samples were incubated in 5 μg/ml proteinase K (Roche), 50 mM ethylenediamine tetraacetic acid, 100 mM Tris–HCl, pH 8.0, for 15 minutes at room temperature followed by a wash in PBS. The samples were then blocked with 10% <span class="yellow">goat</span> serum in PBS for 60 minutes at room temperature, and were incubated with anti-CD16 mAb (3G8; Immunotech, Marseille, France) or <span class="yellow">mouse</span> IgG1 (11711) as an isotype-matched control in 1% <span class="blue">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then washed three times in PBS, for 5 minutes each, and incubated with Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes) in 1% <span class="blue">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then sequentially stained for CD68 antigen in a manner similar to that used for CD16 staining. The samples were stained with anti-CD68 mAb (PGM1; Immunotech) or <span class="yellow">mouse</span> IgG3 (6A3; MBL, Nagoya, Japan) followed by labeling with Alexa fluor488-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG3 antibody (Molecular Probes). The samples were examined by confocal laser scanning microscope (Olympus, Tokyo, Japan).<br><br>Statistical analysis
Data are expressed as the mean ± standard error of the mean. A nonpaired Student's t test was used for comparison, using the StatView program (Abacus Concepts, Berkeley, CA, USA). P < 0.05 was considered statistically significant.<br><br>
Results
Induction of osteoclasts from CD16- peripheral blood monocytes
To identify the monocyte subset that differentiates into osteoclasts, we examined osteoclast formation from CD16+ and CD16- <span class="yellow">human</span> peripheral blood monocytes. The monocyte subsets were purified using magnetic beads. Incubation with M-CSF alone did not induce osteoclast formation from either subset (Figure 1a). Culture with M-CSF + RANKL induced a significant number of TRAP-positive MNC from the CD16- subset, whereas only few CD16+ monocytes differentiated into TRAP-positive MNC (Figure 1a,b). We then assessed the bone resorptive ability by culturing cells on calcium phosphate-coated plates with M-CSF + RANKL. Resorption lacunae were detected only in the CD16- subset (Figure 1c), indicating the TRAP-positive CD16--derived MNC possessed the osteoclast phenotype.
Similar results were obtained using purified monocytes by FACS sorting (purities: CD16+, 96%; CD16-, 97%). The number of TRAP-positive MNC induced were 36 ± 3 cells/well and 348 ± 13 cells/well from CD16+ and CD16- monocytes, respectively. To exclude the possibility that the anti-CD16 antibody used for isolation of CD16+ monocytes inhibits osteoclast formation, we separated the two subsets, CD33low monocytes and CD33high monocytes, using anti-CD33 mAb and a fluorescent cell sorter, since it was reported that CD33low monocytes correspond to CD16+, and that CD33high correspond to CD16- monocytes [15]. On average, the CD33low population contained CD16- (10.2%)/CD16+ (89.8%) monocytes, and the CD33high population contained CD16- (86.3%)/CD16+ (13.7%) monocytes.
Culture with M-CSF + RANKL induced TRAP-positive MNC from CD33high monocytes, whereas no or few CD33low monocytes differentiated into TRAP-positive MNC (CD33low vs CD33high, 2 ± 1 vs 192 ± 71 cells/well; n = 3). TRAP-5b and MMP-9 in the culture supernatants, both of which are known to be produced by osteoclasts, were measured by ELISA. The concentrations of both enzymes were significantly higher in the culture supernatant of CD16- monocytes than in that of CD16+ monocytes (Figure 1d). These results suggest that the CD16- peripheral blood monocyte subset, but not the CD16+ subset, differentiate into osteoclasts by incubation with RANKL + M-CSF.<br><br>CD16+ monocytes do not affect the osteoclastogenesis from CD16- monocytes
To examine whether CD16+ monocytes affect osteoclastogenesis from CD16- monocytes, varied numbers of CD16+ monocytes were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured for 7 days in the presence of M-CSF + RANKL. The number of TRAP-positive MNC was not altered by the presence of CD16+ monocytes (Figure 2). The results indicated that CD16+ monocytes did not hamper or enhance the osteoclastogenesis from CD16- monocytes.<br><br>Differences in cytokine production by RANKL-stimulated or M-CSF-stimulated CD16+ and CD16- monocytes
To compare the biological response of CD16+ and CD16- subsets with either RANKL or M-CSF stimulation, we measured the amount of TNFα and IL-6 production after exposure of cells to various concentrations of RANKL or M-CSF with an ELISA. Without RANKL the CD16+ subset produced a significant amount of TNFα and IL-6, whereas the CD16- subset produced undetectable levels (Figure 3a). RANKL stimulation increased TNFα production from both subsets in a dose-dependent manner, although the CD16+ subset produced more TNFα than did the CD16- subset. RANKL stimulation also enhanced IL-6 production from the CD16+ subset, but not from the CD16- subset. M-CSF stimulation increased TNFα and IL-6 production from both subsets, although the CD16+ subset produced more than the CD16- subset (Figure 3b).
These results suggest that CD16+ monocytes also respond both to RANKL and M-CSF stimulation, although such stimulation does not result in differentiation into osteoclasts. CD16+ monocytes were also noted to express higher amounts of inflammatory cytokines compared with CD16- monocytes with or without RANKL or M-CSF stimulation.<br><br>Comparison of expression levels of molecules involved in osteoclastogenesis between CD16+ and CD16- monocytes
Diverse molecules are involved in RANKL/RANK and its costimulatory signal transduction pathways [16]. The different response to RANKL + M-CSF stimulation between the CD16+ monocyte subset and the CD16- monocytes subset might be explained by the expression profiles of these molecules. We therefore examined the mRNA levels of the following molecules: receptor activator of NF-κB (RANK), the receptor for RANKL; c-fms, the receptor for M-CSF; tumor necrosis factor receptor-associated factor 6 (TRAF-6), the adaptor protein for RANK; c-Fos and nuclear factor of activated T cells c1 (NFATc1), transcription factors that are essential for osteoclastogenesis; DNAX-activation protein 12 (DAP12) and Fc receptor γ chain (FcRγ), adaptor proteins known to deliver costimulatory signals in RANKL-induced osteoclastogenesis; signal regulatory protein β1 (SIRP-β1), triggering receptor expressed on myeloid cells 2 (TREM-2) and osteoclast-associated receptor (OSCAR), transmembrane receptors that associate with either DAP12 or FcRγ; and αv and β3, integrins known to be expressed as the αvβ3 heterodimer on osteoclasts.
The mRNA levels of RANK, c-fms, TRAF-6, DAP12 and SIRP-β1 under the baseline condition (no stimulation) varied between the donors; however, we did not find consistent differences in the mRNA levels of these molecules between the CD16+ monocyte subset and the CD16- monocyte subset among three to six donors (Figure 4a). The mRNA levels of other molecules, apart from integrin β3, were similar between the two subsets under the no-stimulation condition. Although the mRNA levels of RANK, c-fms, DAP12, FcRγ, TREM-2 and OSCAR increased in response to M-CSF alone or M-CSF + RANKL in both subsets, the expression levels were not significantly different between the two subsets. Expressions of TRAF-6, c-Fos and SIRP-β1 mRNA did not change following stimulation with M-CSF + RANKL. Of note, the expression of NFATc1 mRNA was enhanced by M-CSF + RANKL treatment only in the CD16- subset. Furthermore, expression of integrin αv in both subsets was enhanced by M-CSF with or without RANKL; however, the expression level was greater in the CD16- subset. It was noted that integrin-β3 mRNA was detected only in the CD16- subset and was increased by M-CSF + RANKL stimulation, but not by M-CSF alone. The protein expression of RANK under the baseline condition was weakly detected in both subsets, and the levels were varied between donors by western immunoblotting.
The protein expression of c-fms was weakly detected in unstimulated CD16+ monocytes, but not in CD16- monocytes (Figure 4b). Flow cytometry analysis of c-fms in fresh monocytes, however, showed that both subsets express the molecule on the cell surface (Figure 4c). Expressions of both RANK and c-fms were upregulated by M-CSF alone and by M-CSF + RANKL, and we did not find consistent differences in the protein levels of these molecules between the two monocyte subsets. The profiles of expression levels of molecules involved in RANKL/RANK and its costimulatory pathways are similar between the two subsets, except for NFATc1, integrin αv and integrin β3. We therefore assumed that the distinct induction of NFATc1, integrin αv and integrin β3 in response to RANKL stimulation among the two monocyte subsets might explain the differences in their abilities to differentiate into osteoclasts.<br><br>RANKL stimulation induces αvβ3 expression on CD16- monocytes
The integrin-β3 subunit binds to integrin αv only and is expressed as the heterodimeric protein αvβ3 on monocytes and osteoclasts [17]. We examined the expression of αvβ3 on CD16+ and CD16- monocytes by immunofluorescent staining. Neither unstimulated nor M-CSF-stimulated monocyte subsets expressed αvβ3 (Figure 4d and data not shown). After 48 and 72 hours of treatment with M-CSF + RANKL, αvβ3-positive mononuclear cells were observed in CD16- monocyte cultures but not in CD16+ monocyte cultures. At 96 hours, both αvβ3-positive mononuclear cells and multinucleated cells were present in the CD16- monocyte culture. The results indicated that αvβ3 was selectively expressed on CD16- monocytes in the presence of M-CSF + RANKL, and the expression was revealed before the cells differentiate into typical multinucleated osteoclasts.<br><br>RANKL activates ERK and p38 kinases only in CD16- monocytes
Since ERK and p38 MAPK are essential in RANKL-induced osteoclastogenesis [18-20], we next examined whether these kinases were activated differently in CD16+ monocytes and in CD16- monocytes. Purified monocytes were precultured with 25 ng/ml M-CSF for 3 days to enhance RANK expression, and were then treated with RANKL. The RANKL treatment induced phosphorylation of both ERK and p38 MAPK in CD16- monocytes at 5 minutes postexposure, although the p38 MAPK phosphorylation was weak. Both phosphorylations declined to a basal level within 20 minutes (Figure 5). In contrast, ERK and p38 MAPK were not detectably phosphorylated in CD16+ monocytes with RANKL.<br><br>siRNA targeting integrin β3 inhibits osteoclastogenesis from CD16- monocytes
The integrin-β3 cytoplasmic domain is essential for activation of intracellular signals from αvβ3 heterodimers [17]. We therefore examined the involvement of αvβ3 in RANKL + M-CSF-induced osteoclastogenesis in <span class="yellow">human</span> CD16- monocytes using siRNA targeting the integrin-β3 subunit. The integrin-β3 siRNA or control randomized siRNA were transfected into CD16- monocytes. At 48 hours post-transfection, we determined the integrin-β3 mRNA level and αvβ3 heterodimer protein expression. The integrin-β3 mRNA level was reduced in the integrin-β3 siRNA-transfected monocytes compared with control siRNA-transfected monocytes (Figure 6a). The αvβ3 heterodimer expression was evaluated by immunofluorescent staining. The number of αvβ3-positive cells was significantly decreased in integrin-β3 siRNA-transfected monocytes compared with that in control siRNA (Figure 6b).
After 7 days of incubation, the number of TRAP-positive MNC was counted. Transfection with integrin-β3 siRNA significantly reduced the number of TRAP-positive MNC in a dose-dependent manner compared with control siRNA transfection (Figure 6c). In addition, the use of siRNA directed toward a different site of integrin-β3 mRNA also inhibited osteoclast formation from CD16- monocytes (data not shown). On the other hand, siRNA that targeted lamin, which was used as a negative control, did not inhibit the induction of osteoclasts (data not shown). These results indicate the importance of integrin β3 in RANKL-induced osteoclast formation from CD16- peripheral blood monocytes.<br><br>Cyclic RGDfV peptide inhibits the osteoclastogenesis from CD16- monocytes
Integrin αvβ3 recognizes a common tripeptide sequence, RGD (Arg-Gly-Asp), which is present in bone matrix proteins such as vitronectin and fibronectin [21]. Cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) inhibits binding of the RGD-containing molecules to αvβ3 [22]. To investigate the role of ligand binding to the αvβ3 heterodimer in the osteoclastogenesis, we examined whether cyclic RGDfV peptide inhibits the formation of osteoclasts. Cyclic RGDfV peptide significantly reduced the number of TRAP-positive MNC in a dose-dependent manner (Figure 6d). The results imply possible involvement of ligand bindings to αvβ3 in the osteoclastogenesis.<br><br>Knockdown of integrin β3 did not affect the expression of NFATc1 mRNA
In the next step, we determined whether integrin-β3-siRNA-induced inhibition of the osteoclastogenesis reflects downregulation of NFATc1, which is a key transcription factor in osteoclastogenesis [23]. For this purpose, we compared NFATc1 mRNA levels between integrin β3 and control siRNA-transfected monocytes. Interestingly, integrin-β3 knockdown did not alter the NFATc1 mRNA level (Figure 7), suggesting that signal transduction mediated by integrin β3 does not affect the expression of NFATc1.<br><br>Detection of CD16+ and CD16- macrophages in synovium of RA <span class="yellow">patients</span>
RA synovial macrophages are derived from peripheral blood monocytes, and their recruitment into the synovium is facilitated by various adhesion molecules and chemokines [24]. To analyze CD16 expression on synovial macrophages, RA synovial tissues were double-stained for CD16 and a macrophage marker, CD68. CD16-/CD68+ macrophages were widespread in the synovium. Although less frequent, CD16+/CD68+ macrophages were also observed both in the synovial intima and subintima (Figure 8). The presence of two subsets of macrophages, CD16+ and CD16-, in RA synovium indicates that both CD16+ and CD16- peripheral blood monocytes are recruited into the synovium.<br><br>
Discussion
<span class="yellow">Human</span> peripheral blood monocytes are a heterogeneous population, and they are divided into two subsets based on the expression of CD16. The CD16+ and CD16- monocyte subsets show functional differences in migration, cytokine production and differentiation into macrophages or dendritic cells [11-13,15]. We focused on the heterogeneity of the monocytes, and the primary question addressed in this study was which monocyte subset could be the source of osteoclasts. The results demonstrated that CD16- peripheral blood monocytes, but not CD16+ monocytes, differentiated in vitro into osteoclasts by treatment with RANKL + M-CSF.
To investigate the molecular mechanisms of the different response to RANKL and the differentiation into osteoclasts between CD16+ and CD16- monocytes, we examined the expression of molecules known to be involved in osteoclastogenesis. The expression profiles of integrin αv, integrin β3 and NFATc1 were different between the two subsets. Integrin αvβ3 heterodimer was expressed only on RANKL and M-CSF-stimulated CD16- monocytes. It is known that αvβ3 expressed on osteoclasts is important in bone resorption as well as in attachment of osteoclasts to the bone matrix [25].
It was recently reported that bone marrow macrophages of integrin-β3-deficient <span class="yellow">mice</span> could not differentiate into mature osteoclasts in vitro, suggesting that αvβ3 is involved not only in activation, but also in differentiation, of osteoclasts in <span class="yellow">mice</span> [26,27]. The authors also showed that αvβ3 and c-fms share a common intracellular signaling pathway, including the activation of ERK and the induction of c-Fos [27], both of which are essential for osteoclastogenesis [28,29]. In addition, it was reported that echistatin, an αvβ3 antagonist, inhibited osteoclast formation of <span class="yellow">mouse</span> bone marrow cells [30].
In accordance with these reports, our data showed that knockdown of integrin-β3 expression resulted in downregulation of the αvβ3 heterodimer, and abrogated osteoclastogenesis from <span class="yellow">human</span> peripheral blood CD16- monocytes. We also showed that blocking of adhesive ligands to bind to αvβ3 by RGDfV peptide inhibited osteoclast formation from CD16- monocytes. Taken together, the process of ligand binding to αvβ3 may be involved in the osteoclastogenesis. Blockade of αvβ3 could therefore be a therapeutically beneficial approach to modulate osteoclastogenesis. Indeed, integrin αvβ3 antagonists effectively treated osteoporosis in <span class="yellow">mice</span>, <span class="yellow">rats</span> and <span class="yellow">humans</span>, and protected bone destruction in <span class="yellow">rat</span> adjuvant-induced arthritis in vivo [31-34]. Of note, it is reported that <span class="yellow">patients</span> with Iraqi-Jewish-type Glanzmann thrombasthenia who are deficient in integrin β3 do not develop osteopetrosis because of the upregulation of α2β1 expression on osteoclasts, although the bone-resorptive ability of the osteoclasts was decreased in vitro [35]. The function of αvβ3 in vivo in osteoclast formation and resorptive function could therefore be partially compensated by other integrins.
Although all the multinucleated osteoclasts expressed αvβ3 (Figure 4d) [36], a small number of M-CSF + RANKL-stimulated mononuclear CD16- monocytes expressed αvβ3 (Figure 4d). Multinucleated osteoclasts are formed by fusion of osteoclast precursor cells [37]. It was reported that αvβ3 is involved in the migration of osteoclast precursors [30]. The αvβ3-positive cells could therefore be forced to migrate by the ligands and may fuse with closed αvβ3-negative cells. Alternatively, only αvβ3-positive cells may be fused with each other.
It is possible to consider that signaling from CD16 by anti-CD16 mAb-coated magnetic beads, which were used for the cell separation, or by IgG contained in FBS might inhibit osteoclastogenesis from CD16+ monocytes. We therefore separated the two subsets using anti-CD33 mAb and a fluorescent cell sorter, and stimulated the cells with M-CSF + RANKL. The results showed that CD33low monocytes, which correspond to CD16+ monocytes, still could not differentiate into osteoclasts. CD16 is a heterodimer consisting of FcγIIIa and Fcγ, and has low affinity for the Fc region of IgG. Aggregation of CD16 by immune complexes leads to transmission of activating signals via the immunoreceptor tyrosine-based activation motif in the γ chain [38]. We also assessed osteoclastogenesis from the two monocyte subsets using IgG-depleted <span class="blue">bovine</span> serum. Even in the IgG-free medium, CD16- monocytes but not CD16+ monocytes differentiated into osteoclasts (data not shown). We could therefore exclude the possibility that signal transduction through CD16 inhibits osteoclastogenesis from CD16+ monocytes.
NFATc1 is a key transcription factor in osteoclastogenesis [16]. In the present study, stimulation with M-CSF + RANKL increased the NFATc1 mRNA expression in the CD16- subset only, similar to integrin αv and integrin β3. The differences in NFATc1 induction might therefore also explain the difference in osteoclastogenesis between the two monocyte subsets. It is of interest that knockdown of integrin β3 did not lower the mRNA level of NFATc1. This result supports the notion that NFATc1 is located upstream of integrin-β3 expression [39]. It is also possible that parallel activation of two signaling pathways mediated by integrin β3 and NFATc1 contributes to osteoclastogenesis independently or cooperatively. Further studies are needed to determine the mechanisms of integrin β3 involvement in RANKL/RANK-mediated osteoclast differentiation.
It has been demonstrated that MAPK families, ERK and p38 MAPK, were activated by RANKL-induced intracellular signalings in osteoclasts and osteoclast precursors [18,19]. In addition, these kinases are involved in the differentiation of osteoclasts [20]. We showed that RANKL stimulation induced phosphorylation of ERK and p38 MAPK only in CD16- monocytes. It is suggested that differential activation of these kinases may partially explain the distinct properties of the two monocyte subsets upon RANKL stimulation.
Our results showed that CD16+ monocytes produce higher levels of inflammatory cytokines including TNFα and IL-6 compared with CD16- monocytes. These results are consistent with the previous report showing that CD16+ monocytes produced larger amounts of TNFα upon lipopolysaccharide or lipopeptide stimulation than did CD16- monocytes [40]. Interestingly, we showed that stimulation either with RANKL or M-CSF upregulated the TNFα and IL-6 production by CD16+ monocytes. A marked increase of CD16+ monocytes in peripheral blood is reported in inflammatory diseases, such as infection, malignancy, Kawasaki disease and RA [41-44]. Taken together, CD16+ monocytes may be an important source of inflammatory cytokines.
In <span class="yellow">mice</span>, peripheral blood Ly-6Chigh monocytes, which are thought to correspond to <span class="yellow">human</span> CD16- monocytes, increase in inflammatory conditions, and these cells are recruited into sites of inflammation [45]. In contrast, Ly-6Clow monocytes, which are thought to correspond to <span class="yellow">human</span> CD16+, migrate into noninflamed tissues [12]. These data on <span class="yellow">mouse</span> monocytes seem to be in contrast to the data on <span class="yellow">human</span> monocytes, which show expansion of CD16+ monocytes in inflammatory conditions where they produce larger amounts of inflammatory cytokines. At present, it is not clear whether <span class="yellow">mouse</span> monocyte subsets, Ly-6Clow/Ly-6Chigh, represent <span class="yellow">human</span> monocyte subsets, CD16+/CD16- monocytes, and whether the biologic functions of <span class="yellow">mouse</span> monocytes are analogous to those of <span class="yellow">human</span> monocytes.
In <span class="yellow">mice</span>, blood monocytes newly released from the bone marrow are exclusively Ly-6Chigh and the level of Ly-6C is downregulated while in circulation [45]. It is thus suggested that in <span class="yellow">mice</span> the two monocyte subsets differing in Ly-6C expression represent different stages in the maturation pathway. In the <span class="yellow">human</span>, transition from CD16- monocytes to CD16+ monocytes is observed upon culture with IL-10, M-CSF and transforming growth factor beta in vitro [42,46]. Similar to <span class="yellow">mouse</span> monocytes, therefore, <span class="yellow">human</span> peripheral blood CD16- monocytes may also maturate into CD16+ monocytes.
It is reported that a significant number of RA synovial cells in the intima express CD16, suggesting that CD16+ cells are synovial macrophages [47]. We confirmed that both CD16+ and CD16- macrophages accumulate in the RA synovium by double-color immunohistochemical staining for CD68 and CD16. A number of chemokines are abundantly expressed in the RA synovium [24,48]. Among these cytokines, MCP-1, MIP-1α, SDF-1, RANTES and fractalkine can induce migration of CD16- monocytes in vitro ([11,12] and unpublished data). On the other hand, migration of CD16+ monocytes is induced only by fractalkine. These chemokines therefore seem to play an important role in recruitment of CD16+ and CD16- monocytes from the circulating pool into the RA synovium.
The osteoclast inducers are also produced in the RA synovium. RANKL is expressed by synovial fibroblasts and activated T cells [49-51], while M-CSF is expressed on RA synovial macrophages and fibroblasts [52,53].
TNFα and IL-6, which are mainly expressed on RA synovial macrophages and fibroblasts, respectively, could also enhance osteoclast differentiation [54]. Collectively, it is probable that the recruited CD16- monocytes/macrophages differentiate into osteoclasts in the RA synovium, and contribute to bone destruction. On the other hand, CD16+ monocytes/macrophages might also be involved in RA pathogenesis by producing inflammatory cytokines including TNFα and IL-6. Since TNFα and IL-6 enhance osteoclast formation [54,55], CD16+ monocytes/macrophages may also contribute to osteoclastogenesis in RA synovium.<br><br>Conclusion
We have shown that <span class="yellow">human</span> peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct response to osteoclastogenic stimuli. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for the osteoclastogenesis. The blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Abbreviations
DAP = DNAX-activation protein; ELISA = enzyme-linked immunosorbent assay; FBS = fetal <span class="blue">bovine</span> serum; FcRγ = Fc receptor γ chain; IL = interleukin; FITC = fluorescein isothiocianate; mAb, monoclonal antibody; M-CSF = macrophage colony-stimulating factor; MEM = modified Eagle's medium; MMP = matrix metalloproteinase; MNC = multinucleated cells; NF = nuclear factor; OSCAR = osteoclast-associated receptor; PBS = phosphate-buffered saline; PCR = polymerase chain reaction; RA = rheumatoid arthritis; RANK = receptor activator of NF-κB; RANKL = receptor activator of NF-κB ligand; RT = reverse transcriptase; siRNA = small interfering RNA; SIRP-β1 = signal regulatory protein-β1; TNFα = tumor necrosis factor alpha; TRAF = tumor necrosis factor receptor-associated factor; TRAP = tartrate-resistant acid phosphatase; TREM = triggering receptor expressed on myeloid cells.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YK participated in the design of the study, carried out the experiments and statistical analysis, and drafted the manuscript. KH and KT participated in the design of the study and its coordination. TN and NM conceived of the study, participated in its design and coordination, and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2365090</h3>Carbonic Anhydrase Inhibitors. Part 541: Metal Complexes of Heterocyclic Sulfonamides: A New Class of Antiglaucoma Agents
Abstract
Metal complexes of heterocyclic sulfonamides possessing carbonic anhydrase (CA) inhibitory properties were recently shown to be useful as intraocular pressure (IOP) lowering agents in experimental animals, and might be developed as a novel class of antiglaucoma drugs. Here we report the synthesis of a heterocyclic sulfonamide CA inhibitor and of the metal complexes containing main group metal ions, such as Be(II), Mg(II), Al(III), Zn(II), Cd(II) and Hg(II) and the new sulfonamide as well as 5-amino-1,3,4-thiadiazole-2-sulfonamide as ligands. The new complexes were characterized by standard physico-chemical procedures, and assayed as inhibitors of three CA isozymes, CA I, II and IV. Some of them (but not the parent sulfonamides) strongly lowered IOP in <span class="yellow">rabbits</span> when administered as a 2% solution into the eye.
CARBONIC ANHYDRASE INHIBITORS. Part 54 METAL COMPLEXES OF HETEROCYCLIC SULFONAMIDES: A NEW CLASS OF ANTIGLAUCOMA AGENTS Claudiu T. <br><br> Supuran*, Andrea Scozzafava <br><br> and Andrei Jitianu 2 <br><br> Universit& degli Studi, Dipartimento di Chimica, Laboratorio di Chimica Inorganica e Bioinorganica, Via Gino Capponi 7, 1-50121, Florence, Italy 2 "I.G. Murgulescu" Institute of Physical Chemistry, Academia Romana, Spl. Independentei 202, R-77208 Bucharest, Roumania Abstract: Metal complexes of heterocyclic sulfonamides possessing carbonic anhydrase (CA) inhibitory properties were recently shown to be useful as intraocular pressure (IOP) lowering agents in experimental animals, and might be developed as a novel class of antiglaucoma drugs. Here we report the synthesis of a heterocyclic sulfonamide CA inhibitor and of the metal complexes containing main group metal ions, such as Be(II), Mg(II), AI(III), Zn(II), Cd(II) and Hg(II) and the new sulfonamide as well as 5-amino-l,3,4thiadiazole-2-sulfonamide as ligands. The new complexes were characterized by standard physico-chemical procedures, and assayed as inhibitors of three CA isozymes, CA I, II and IV. Some of them (but not the parent sulfonamides) strongly lowered IOP in <span class="yellow">rabbits</span> when administered as a 2% solution into the eye. Introduction Sulfonamides possessing carbonic anhydrase (CA, EC 4.2.1.1) inhibitory properties [2] such as acetazolamide 1, methazolamide 2, ethoxzolamide 3 and dichlorophenamide 4 have been used for more than 40 years as pressure lowering systemic drugs in the treatment of open-angle glaucoma [3,4]. Their effect is due to inhibition of at least two CA isozymes present within cilliary processes of the eye, ie, CA II and CA IV, which is followed by lowered bicarbonate formation and reduction of aqueous humor secretion [5-7]. Their main drawback is constituted by side effects such as fatigue, augmented diuresis, or paresthesias, due to CA inhibition in other tissues/organs than the target one, ie, the eye [8]. <br><br> N--N <br><br> XN--N EtO S <br><br> NH 2 <br><br> O <br><br> NH O:S----O <br><br> NHEt <br><br> The above-mentioned side effects are absent in the case in which the inhibitor has topical activity, and is applied directly into the eye. This route has been demonstrated only in 1983 by Maren's group [9] and was followed by the development of the first clinical agent of this type, dorzolamide 5 [10,11 ]. Dorzolamide (Trusopt) has been introduced in clinical use in 1995 in USA and Europe and it constituted the beginning of a radically new treatment of glaucoma, devoid of the severe side effects observed with the systemic inhibitors [4-6]. The success of topical antiglaucoma CA inhibitors fostered much research in the synthesis and clinical evaluation of other types of such compounds [12-15]. 307 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides: A New Class ofAntiglaucoma Agents <br><br> On the other hand, metal complexes of heterocyclic sulfonamides of type 1-5 have been recently prepared by two groups 16-20], and it was proved that they possess much stronger CA inhibitory properties than the sulfonamides from which they were prepared 18-22]. Although the mechanism of CA inhibition of the metal complexes is presently unknown, it was hypothesized that their increased inhibitory power might be due to two processes, occurring separately or in concert, ie, (i) dissociation of the complex inhibitor in sulfonamide anions and metal ions (in diluted solution), which in turn both interact thereafter with the enzyme, at different binding sites, and (ii) direct interaction of the undissociated complex with the enzyme, and more specifically with the hydrophilic patch at the entrance of CA II active site [23], this being the isozyme most susceptible to inhibition with this class of compounds [2,22]. Whether initially the first mechanism of action mentioned above was favored by us [22], recent evidences suggested that the undissociated complex might be the inhibitory species, at least for some isozymes [24]. Since metal complexes are much more inhibitory than the parent sulfonamide from which they were prepared, it appeared of interest to test whether this property might be useful for their use in lowering IOP in experimental animals and whence as a possible glaucoma therapy. Recently we proved [25, 26] that some metal complexes of heterocyclic sulfonamides (which themselves do not possess IOP lowering properties) act as very powerful such agents when administered as diluted solutions directly into the eye of experimental animals, and would thus offer the possibility of developing such totally novel drugs. Here we report the synthesis of a heterocyclic sulfonamide possessing strong CA inhibitory properties, ie, 5-(chloroacetamido)-l,3,4-thiadiazole-2-sulfonamide, and of the metal complexes of this sulfonamide and of 5-amino-1,3,4-thiadiazole-2-sulfonamide, containing some main group metal ions. The new compounds have been characterized by standard physico-chemical procedures, and were assayed as inhibitors of three CA isozymes, hCA I, hCA II and bCA IV (h <span class="yellow">human</span>; b <span class="blue">bovine</span>; these are the isozymes considered to play a critical role in aqueous humour secretion within the eye of higher vertebrates [2-5]). Materials and Methods <br><br> Melting points were recorded with a heating plate microscope and are not corrected. IR spectra were recorded in KBr pellets with a Carl Zeiss IR-80 instrument. 1H-NMR spectra were recorded in DMSO-d6 as solvent, with a Bruker CPX200 instrument. Chemical shifts are reported as values, relative to Me4Si as internal standard. Conductimetric measurements were done at room temperature (1 mM concentration of complex) in DMSO solution with a Fisher conductimeter. Elemental analyses were done by combustion for C, H, N with an automated Carlo Erba analyzer, and gravimetrically for the metal ions, and were 0.4% of the theoretical values. Thermogravimetric measurements were done in air, at a heating rate of 10C/min., with a Perkin Elmer 3600 thermobalance. Sulfonamides used as standards in the enzymatic assay (except for 5), acetazolamide, pyridine, and chloroacetyl chloride used for the preparation of compound 7, solvents as well as inorganic reagents were from Sigma, Merck and Carlo Erba. 5-Amino-l,3,4-thiadiazole-2-sulfonamide 6 was prepared from acetazolamide by literature procedures [27], by desacetylation with concentrated hydrochloric acid, followed by neutralization with sodium bicarbonate of the corresponding hydrochloride (Scheme 1). Dorzolamide hydrochloride 5 was from Merck, Sharp and Dohme or was prepared as described by Ponticello et al 10,11 ]. <span class="yellow">Human</span> CA and CA II cDNAs were expressed in <span class="yellow">Escherichia coli</span> strain BL21 (DE3) from the plasmids pACA/hCA I and pACAdaCA II described by Forsman et al. [28] (the two plasmids were a gift from Prof. Sven Lindskog, Umea University, Sweden). Cell growth conditions were those described by Lindskog's group [29], and enzymes were purified by affinity chromatography according to the method of Khalifah et al [30]. Enzyme concentrations were determined spectrophotometrically at 280 nm, utilizing a molar absorptivity of 49 mM-l.cm-1 for hCA and 54 mM-l.cm-1 for hCA II, respectively, based on M 28.85 kDa for hCA I, and 29.3 kDa for hCA II, respectively [31,32]. bCA IV was isolated from <span class="blue">bovine</span> lung microsomes as described by Maren et al, and its concentration has been determined by titration with ethoxzolamide [33]. Synthesis of 5-(chloroacetamido)- l, 3, 4-thiadiazole-2-sulfonamide 7 An amount of 1.80 g (10 mmol) of 5-amino-1,3,4-thiadiazole-2-sulfonamide 6 was suspended in 20 mL of anhydrous acetonitrile and 0.9 mL (0.87g, 11 mmol) of pyridine added. The mixture was magnetically stirred at 4 C for 10 minutes, then 10.5 mmol of monochloroacetyl chloride, dissolved in 3 mL acetonitrile, were added dropwise for 5 min, and stirring was continued for other 2 hours at room temperature. After an additional 30 min of refluxation, followed by cooling, the precipitated crystals were filtered and recrystallized from ethanol. Yield of 62% white crystals, mp 246-248 o lit [34] mp IR (KBr), cm-l: 590, 610, 660, 790, 935, 1090, 1115, 1170, 1350, 1400, 1550, 1650, 1720, 2870, 3280 3370 (broad); UV spectrum, ,max, nm (lg): 255 (3.50); 288 (4.37) H-NMR (DMSO-d6), i, ppm: 2.96 (s, 2H, CH2); 8.20 (s, 2H, SOzNH2); 12.22 (s, 1H, CONH). Analysis, found: C, 18.56; H, 1.88; N, 21.76; S, 24.62 %; C4HsC1N403S2 requires" C, 18.72; H, 1.96; N, 21.83; S, 24.98 %. <br><br> General procedure for the preparation of compounds 8-20 An amount of 6 mmol of sodium salt of sulfonamides 6 or 7 was prepared by reacting the corresponding sulfonamide with the required amount of an alcoholic 1N NaOH solution, in ethanol as solvent. To this 308 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> solution was added the aqueous metal salt (Zn(II), Mg(II), AI(III), Cd(II) chlorides, and Be(II), Pb(II) and Hg(II) nitrate) solution, working in molar ratios RSOzNH- Mn+ of 2:1 for the divalent cations and 3:1 for the trivalent cation, respectively. The aqueous-alcoholic reaction mixture was heated on a steam bath for one hour, adjusting the pH at 7 if necessary, and after being cooled at 0 C the precipitated complexes were filtered and thoroughly washed with alcohol-water 1:1 (v/v) and air dried. Yields were in the range of 85-90 %. The obtained white powders of compounds 8-20 melted with decomposition at temperatures higher than 350 C, and were poorly soluble in water and alcohol, but had good solubilities in DMSO, DMF as well as mixtures of DMSO-water, DMF-water. <br><br> Pharmacology Carbonic anhydrase inhibition Initial rates of 4-nitrophenyl acetate hydrolysis catalysed by different CA isozymes were monitored spectrophotometrically, at 400 rim, with a Cary 3 instrument interfaced with an IBM compatible PC [35]. Solutions of substrate were prepared in anhydrous acetonitrile; the substrate concentrations varied between 2.10-2 and 1.10-6 M, working at 25C. A molar absorption coefficient of 18,400 M-l.cm-1 was used for the 4-nitrophenolate formed by hydrolysis, in the conditions of the experiments (pH 7.40), as reported in the literature [35]. Non-enzymatic hydrolysis rates were always subtracted from the observed rates. Duplicate experiments were done for each inhibitor concentration, and the values reported throughout the paper are the mean of such results. Stock solutions of inhibitor (1 mM) were prepared in distilled-deionized water with 1020% (v/v) DMSO (which is not inhibitory at these concentrations [2]) and dilutions up to 0.01 nM were done thereafter with distilled-deionized water. Inhibitor and enzyme solutions were preincubated together for 10 min at room temperature prior to assay, in order to allow for the formation of the E-I complex. The inhibition constant KI was determined as described by Pocket and Stone [35]. Enzyme concentrations were 3.3 nM for hCA II, l0 nM for hCA and 34 nM for bCA IV (this isozyme has a decreased esterase activity [36] and higher concentrations had to be used for the-measurements). <br><br> Measurement of tonometric lOP Adult male New Zealand albino <span class="yellow">rabbits</span> weighing 2-3 kg were used in the experiments (three animals were used for each inhibitor studied). The experimental procedures conform to the Association for Research in Vision and Ophthalmology Resolution on the use of animals. The <span class="yellow">rabbits</span> were kept in individual cages with food and water provided ad libitum. The animals were maintained on a 12 h: 12 h light/dark cycle in a temperature controlled room, at 22-26 C. Solutions of inhibitors (2 %, by weight) were obtained in DMSOwater (2:3, v/v) due to the low water solubility of some of these derivatives. Control experiments with DMSO (at the same concentration as that used for obtaining the inhibitors solutions showed that it does not possess IOP lowering or increasing effects. IOP was measured using a Digilab 30R pneumatonometer (BioRad, Cambridge, MA, USA) as described by Maren's group [37-39]. The pressure readings were matched with two-point standard pressure measurements at least twice each day using a Digilab Calibration verifier. All IOP measurements were done by the same investigator with the same tonometer. One drop of 0.2 % oxybuprocaine hydrochloride (novesine, Sandoz) diluted 1"1 with saline was instilled in each eye immediately before each set of pressure measurements. IOP was measured three times at each time interval, and the means reported. IOP was measured first immediately before drug administration, then at 30 min after the instillation of the pharmacological agent, and then each 30 minutes for a period of several hours. For all IOP experiments drug was administered to only one eye, leaving the contralateral eye as an untreated control. The ocular hypotensive activity is expressed as the average difference in IOP between the treated and control eye, in this way minimizing the diurnal, seasonal and interindividual variations commonly observed in the <span class="yellow">rabbit</span> [37-39]. All data are expressed as mean SE, using a one-tailed test. Results and Discussion Reaction of 5-amino-l,3,4-thiadiazole-2-sulfonamide 6 [19b] with chloroacteyl chloride in the presence of pyridine afforded 5-(chloroacetamido)-l,3,4-thiadiazole-2-sulfonamide 7, by the procedure already reported by Young et al. [34] (Scheme 1). The sulfonamide 7 has been characterized by elemental analysis and spectroscopic methods which confirmed its structure (only its m.p. has been reported in ref. [34]). The sodium salt of sulfonamides 6 and 7, obtained in situ from the corresponding sulfonamide and sodium hydroxide, were then used for the preparation of coordination compounds, containing the following metal ions: Be(II), Mg(II), Al(III), Zn(II), Cd(II) and Hg(II). Mention should be made that although 5-amino-l,3,4-thiadiazole-2-sulfonamide 6 is the parent compound of important sulfonamide CA inhibitors, such as acetazolamide, benzolamide, methazolamide, etc., its coordination chemistry has been scarcely investigated up to now [22, 40]. The new complexes prepared in this work are shown in Table I. Both compounds containing the sulfonamide-deprotonated species of sulfonamide 7 (LH), as well as complexes in which the anion of 5amino-1,3,4-thiadiazole-2-sulfonamide (tda) act as ligands, have been prepared. In fact in another work [40] it was documented that in some cases, sulfonamides derived from this ring system may undergo hydrolysis to <br><br> 309 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides." A New Class ofAntiglaucoma Agents <br><br> the moiety substituting the 5 position, with the formation of 5-amino-l,3,4-thiadiazole-2-sulfonamide 6, which thereafter coordinates metal ions present in solution. <br><br> N CIH N <br><br> I <br><br> N +NaHCO 2 <br><br> N 2 <br><br> S' <br><br> -NaCl <br><br> I S <br><br> N <br><br> II <br><br> SO2NH <br><br> 1_12 O Py/MeCN <br><br> 6Htda +CIOCCH2CI <br><br> 0 Cl CH 2 Scheme 1 <br><br> HN <br><br> .I <br><br> N <br><br> N <br><br> S 7: LH <br><br> II <br><br> SqNH <br><br> Thus, the X-ray crystal structure of the complex [Zn(tda)2(NH3)].H20 prepared in this way has recently been reported by this group [40]. On the other hand, when the ligand 7 has not been hydrolyzed (during the preparation of the coordination compounds) in the presence of the metal ion to 5-amino-l,3,4-thiadiazole-2sulfonamide and chloroacetate, the metal complexes contining 6 as ligand have been prepared from the last (pure) compound (as sodium salt) and the corresponding metal salt, by the general procedure described in the Experimental part. Table I: Prepared complexes 8-20, containing the conjugate bases of sulfonamides 6 and 7 as ligands and their elemental analysis data. L stands for the sulfonamide deprotonated species of 7, whereas tda for the sulfonamide deprotonated species of 5-amino-1,3,4-thiadiazole-2-sulfonamide 6. <br><br> No. 8 9 10 11 12 13 14 15 16 17 18 19 20 <br><br> Complex <br><br> Yield <br><br> (%) <br><br> %Ma <br><br> Analysis (calculated/found) %H b %C b 13.0/13.1 11.0/10.8 11.3/11.4 10.2/10.1 8.5/8.1 7.9/7.9 12.7/12.8 18.4/18.1 18.1/17.9 16.6/16.2 15.3/14.9 13.4/13.3 12.7/12.5 1.6/1.3 2.7/2.3 1.4/1.1 1.2/1.2 1.0/1.2 1.6/1.3 1.6/1.6 1.5/1.5 1.5/1.3 1.3/1.4 1.2/1.1 1.1/1.2 1.0/1.0 <br><br> %N b 30.4/30.2 25.6/25.5 26.4/26.4 13.7/23.3 20.0/19.8 18.6/18.5 29.7/29.6 21.5/21.3 21.1/20.8 19.4/19.3 17.9/17.8 15.7/15.6 14.8/14.6 <br><br> [Be(tda)2] [Mg(tda)2].3 H20 [Zn(tda)2] [Cd(tda)2] [Hg(tda)2] [Pb(tda)z(OH2)2] [Al(tda)3] [BeL2] [ALL3] [ZnL2] [CdL2] [HgL2] [PbLz(OH2)2] <br><br> 78 76 83 90 95 84 72 75 59 87 88 92 95 <br><br> 2.4/2.5 5.5/5.1 15.4/15.0 23.8/24.1 35.8/35.7 34.4/34.7 4.7/4.4 1.7/1.6 3.4/3.5 11.3/11.5 18.0/18.1 28.1/28.3 27.4/27.2 <br><br> aBy gravimetry; bBy combustion. The new complexes have also been characterized by spectroscopic, conductimetric and thermogravimetric measurements (Table II). By comparing the IR spectra of the complexes and the corresponding ligands, the following observations should be made: (i) the shift of the two sulfonamido vibrations (both the symmetric as well as the the assymetric one), towards lower wavenumbers in the spectra of the complexes, as compared to the spectra of the corresponding ligand (Table II), as already documented previously for similar complexes [13-22]. This is a direct indication that the deprotonated sulfonamido 310 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> moieties of the ligands interacts with the metal ions in the newly prepared coordination compounds; (ii) the amide vibrations (the most intense such bands at 1670-1680 cm-1) of ligand 7 appear unchanged in the IR spectra of complexes 15-20 (data not shown), suggesting that these moieties do not participate in coordination of the metal ions; (iii) the C-N stretching vibration in the spectra of the prepared complexes is shifted with 5-20 cm-1 towards lower wavenumbers, as compared to the same vibration in the spectra of sulfonamides 6 and 7, indicating that one of the endocyclic nitrogens of the thiadiazolic ring (presumably N3) acts as donor atom, as already documented by X-ray crystallographic and spectroscopic determinations on complexes of other sulfonamides (such as 1-3) with divalent metal ions [13-22] (Table II); (iv) changes in the region 3100-3160 cm-, as the bands present in the spectra of sulfonamides 6, 7 are present in the spectra of complexes 8-20 too, but they are not well resolved, and have a smaller intensity. This is probably due to deprotonation of the SO2NH2 moiety and participation in the binding of cations; (v) the amino vibrations from 3320 cm- in the spectra of 6 appear unchanged in the spectra of its complexes 8-14 (data not s.hown). In the 1H-NMR spectra of compound 6 and its metal complexes, the signal of the amino group has been evidenced as a broad singlet centered at 4.54 ppm (Table II), which is not exchangeable by addition of D20 into the NMR tube, in contrast to the sulfonamido NH2 protons (which readily exchange). This proves that the 5-amino moiety is not involved in binding the metal ions, as already shown in the X-ray crystallographic work of the complex [Zn(tda)z(NH3)].H20 previously reported [40]. For sulfonamide 7 the CONH proton resonates as a singlet at 12.22 ppm. In complexes 15-20 only very minor shifts of this signal were evidenced (Table II), proving basically that the CONH moiety does not interact with the metal ions in these complexes. <br><br> Table II: Spectroscopic, thermogravimetric and conductimetric data for compounds 6-20. <br><br> 1H-NMR Spectrab Comp. IR Spectraa ,cm-1 as v(C=N) CONH, i (ppm) v(SO2)S; v(SO2) 6 8 9 10 11 <br><br> TG analysisc calc./found <br><br> Conductimetryd AM (-1 x cm2x mol-) 2 7 4 5 3 2 2 6 3 4 2 9 3 2 8 <br><br> 12 13 14 7 15 16 17 18 19 20 <br><br> 1170; 1150; 1150; 1145; 1145; 1140; 1145; 1150; 1170; 1130; 1140; 1140; 1150; 1140; 1145; <br><br> 1350 1300 1305 1300 1305 1300 1300 1300 1350 1335 1330 1330 1330 1335 1330 <br><br> 1610 1600 1600 1600 1600 1590 1605 1605 1610 1605 1610 1610 1605 1600 1600 <br><br> A A A A A A A <br><br> e e <br><br> 12.3/12.1f e e e 5.9/5.7g <br><br> A 12.22 (1H) 12.19 (2H) 12.18 (3H) 12.20 (2H) 12.18 (2H) 12.19 (2H) 12.21 (2H) <br><br> e e e e e e e 4.7/4.8g <br><br> a In KBr; bin DMSO-d6; A the signal of the 5-amino group of 6 (appearing in the ligand at 4.54 ppm as a broad singlet) appears at the same chemical shift (4.50 4.55 ppm) in complexes 8-14; cWeight loss between 70-250 C; d mM solution, in DMF, at 25C; e No weight loss seen under 250 C; fCorresponding to three lattice water molecules lost at 70-110C, and gCorresponding to two coordinated water molecules, lost at 160-180 C. <br><br> Thermogravimetric analysis showed the presence of uncoordinated water molecules in the molecule of complex 9 (the three waters were lost in a single step, between 70-110 C) and of coordinated water in the molecules of the lead(II) derivatives 13 and 20. All these compounds behaved as non-electrolytes in DMF as solvent (Table II). Mention should be made that the Mg(II) complex of sulfonamide 7 could not be isolated. Instead, only the correponding complex of 5-amino-l,3,4-thiadiazole has been obtained from reaction mixtures containing magnesium salts and the sodium salt of 7, probably due to a metal ion assisted hydrolysis of 7 to 6 and chloroacetate. Generally such hydrolytic processes involve highly acidic conditions and prolonged heating of the 5-alkylamido-1,3,4-thiadiazole-2-sulfonamide derivatives [41 ], but they might become milder by taking into account the putative catalytic effect of Mg2+ ions reported here. The data shown above lead to the conclusion that ligand 7 shares a common coordination chemistry with acetazolamide 1 with which it is structurally related, whereas 6 probably also behaves similarly to acetazolamide in the sense that the 5-amino group seems not to be involved in coordinating metal ions, at 311 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides: A New Class ofAntiglaucoma Agents <br><br> least in the complexes reported by us here (and also in the compound characterized by X-ray crystallography mentioned above [40]). Thus, in all complexes reported here these sulfonamides (as monodeprotonated species at the SO2NH 2 moieties) act as bidentate ligands, through the endocyclic N-3 and the NH- groups. The proposed formulae of the new complxes are shown below. Except for the two Pb(II) complexes 13 and 20, as well as the AI(III) derivatives 14 and 16, which presumably are pseudo-octahedral, the other derivatives are supposed to contain tetrahedral M(II) ions. <br><br> R <br><br> R 13"R = 20" R= <br><br> H2N <br><br> CICH2CONH <br><br> 14: R = 16: R = <br><br> H2N CICH2CONH <br><br> The compounds 6-20 together with the standard CA inhibitors 1-5 were assayed for inhibition against three isozymes, hCA I, hCA II and bCA IV (Table III). As seen from the above data, the chloracetamido derivative is more inhibitory than acetazolamide, methazolamide and dichlorophenamide, whereas the unacylated compound 6 is less inhibitory than the above sulfonamides. The metal complexes 820 are much more inhibitory than the sulfonamides from which they derive 6, 7 and than all other simple sulfonamides assayed. They behave similarly to the metal complexes of acetazolamide, methazolamide or dorzolamide previously reported by this group, which were all more inhibitory than the parent sulfonamide from which were prepared [16-22, 40]. Particularly strong inhibition was observed for the Zn(II), Hg(II), Pb(II) and Cd(II) complexes, especially against CA II and CA IV, the isozymes critical for aqueous humor formation. In vivo IOP lowering experiments were done in <span class="yellow">rabbits</span> with some of the new compounds prepared in the present work, such as the sulfonamides 6 and 7, and their Zn(II) complexes, which were among the strong CA II and CA IV inhibitors in the obtained series. Some of the IOP lowering data at half an hour and one hour after the instillation of one drop of 2 % solution of inhibitor within the <span class="yellow">rabbit</span> eye are shown in 312 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> Table IV, with dorzolamide (at the same concentration) as standard. In Fig. the time dependence of IOP lowering with dorzolamide 5 and the two Zn(II) complexes 10 and 17 is presented. Table III. CA inhibition data with the standard inhibitors 1-5, the sulfonamides 6 and 7, and their metal complexes 8-20. <br><br> No 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 a <br><br> Inhibitor <br><br> KI hCA Ia 900 780 25 1200 >50,000 1550 640 1050 350 50 40 12 80 240 120 80 40 40 9 15 <br><br> (nM) bCA IVb 220 240 13 380 43 780 24 540 220 25 19 10 26 110 12 16 9 10 5 10 <br><br> hCA IIa 12 14 8 38 9 230 5 190 110 15 14 7 10 76 5 4 3 3 2 5 <br><br> Acetazolamide Methazolamide Ethoxzolamide <br><br> Dichlorophenamide Dorzolamide <br><br> <span class="yellow">Human</span> (cloned) isozymes; b From <span class="blue">bovine</span> lung microsomes. <br><br> Table IV: IOP lowering following topical application of CA inhibitors, half an hour and one hour after instillation into the eye of a drop (50 L) of 2 % solution of inhibitor. Inhibitor <br><br> lOP SE 1/2 h <br><br> a <br><br> (mm Hg) lh 4.1 0.15 0 0.09 0 0.09 5.0 0.12 8.1 0.21 Dorzolamide 5 6 7 10 17 a <br><br> 2.2 0.10 0 0.10 0 0.10 2.00.09 8.0 0.14 <br><br> IOP <br><br> IOP control eye" IOP treated eye (N 3). <br><br> As seen from the above data, the sulfonamides 6 and 7 are totally ineffective as lOP lowering agents, similarly to the classical clinically used inhibitors of type 1-5 [2,3]. On the other hand, dorzolamide, the first topical sulfonamide used clinically in the treatment of glaucoma is an effective such agent, with a decrease of lOP of around 4 mm Hg, one hour after administration directly into the eye (Table IV). From the data of this table, it is obvious that the metal complexes of heterocyclic sulfonamides investigated by us behave as much more effective IOP lowering agents than dorzolamide, and their effect is generally longerlasting (Fig. 1). A last remark should be made about the possible mechanism of action of the new class of IOP lowering agents. Obviously, their activity is due to inhibition of CA isozymes present in the cilliary processes within the eye, similarly to other topically active sulfonamides [2-6]. The fact that the sulfonamide per se is inactive via the topical route, whereas the metal complexes result much better than the drug <br><br> 313 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides." A New Class ofAntiglaucoma Agents <br><br> dorzolamide, indicates that the presence of metal ions in the molecules of these CA inhibitors is essential and confers them completely new properties. <br><br> Lowering of IOP (ram Hg) <br><br> "||, <br><br> / <br><br> ", , <br><br> '",., <br><br> ,< <br><br> .7, / <br><br> / <br><br> .6<br><br> -0 <br><br> -10 0 <br><br> 2 <br><br> Time (hours) <br><br> Fig l" Time dependence of IOP lowering with dorzolamide (curve 1); the zinc complex 10 (curve 3) and the zinc complex 17 (curve 3), after topical administration of one drop of 2 % solution of inhibitor in <span class="yellow">rabbit</span>. <br><br> Preliminary results from this laboratory indicate that the metal complexes of topically active sulfonamides show also increased lOP lowering effects with respect to the complexes prepared in the present study [42]. Our hypothesis is that the presence of the metal ion in the molecules of these complex inhibitors induces a dramatic change in their physico-chemical properties as compared to those of the parent sulfonamide. This phenomenon is certainly governed by the strong polarization induced by the metal ions. In this way, it is quite probable that the right balance between the lipo- and hydrosolubility of these compounds is achieved, which has been considered to be the critical factor for not observing topical activity in the classical CA inhibitors, such as acetazolamide, methazolamide and ethoxzolamide, which were either too lipophilic or too hy.drosoluble [2,3]. So, by choosing different metal ions and diverse sulfonamides, much larger possibilities arise to finely tune the pharmacological properties which strongly influence the value of a drug. In conclusion we describe here a novel class of lOP lowering agents, ie, the metal complexes of sulfonamide CA inhibitors. These derivatives appear to be very active and longer lasting than the drug dorzolamide, and might constitute the premises for a new generation of antiglaucoma drugs. <br><br> Acknowledgments. We are grateful to Prof. S. Lindskog (Umea Univ., Sweden) for the gift of the hCA and II plasmids. References Preceding part of this series" Supuran CT, Scozzafava A, Ilies MA, Iorga B, Cristea T, Chiraleu F, Banciu MD (1997) Eur J Med Chem, submitted. 2 Supuran CT (1994) "Carbonic anhydrase inhibitors" In Carbonic Anhydrase and Modulation of Physiologic and Pathologic Processes in the Organism, (Puscas I, Ed) Helicon, Timisoara, pp. 29-111. 3 Maren TH (1991) "The links among biochemistry, physiology and pharmacology in carbonic anhydrase mediated systems". In Carbonic Anhydrase From Biochemistry and Genetics to Physiology and Clinical Medicine, (Botr6 F, Gros G, Storey BT Eds) VCH, Weinheim, pp. 186-207. 4 Bayer A, Ferrari F, Maren TH, Erb C (1996) dFr Ophtalrnol 19, 3:57-362. :5 Maren TH, Conroy CW, Wynns GC, Levy NS (1997) d Ocul Pharrnacol Therapeut 13, 23-30. 6 Sugrue MF (1996) d Ocul Pharmacol Ther 12,363-376. 7 Bartlett JD, Jaanus SD (1989) "Carbonic anhydrase inhibitors". In Clinical Ocular Pharmacology, Second Edition, Butterworths Publishers, Boston, pp. 2:54-263. 8 Alward PD, Wilensky JT (1981) Arch Ophthalmo199, 1973-1976. <br><br> 314 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> 9 Maren TH, Jankowska L, Sanyal G, Edelhauser HF (1983) Exp Eye Res 36, 457-480. 10 a) Ponticello GS, Freedman MB, Habecker CN, Lyle PA, Schwam H, Varga SL, Christy ME, Randall WC, Baldwin JJ(1987) J Med Chem 30, 591-597; b) Greer J, Erickson JW, Baldwin JJ, Varney MD (1994) J Med Chem 37, 1035-1054. 11 Baldwin JJ, Ponticello GS, Anderson GS, Christy ME, Murcko MA, Randall WC, Schwam H, Sugrue MF, Springer JP, Gautheron P, Grove J, Mallorga P, Viader MP, McKeever BM, Navia MA (1989) J Med Chem 32, 2510-2513. 12 Chow K, Lai R, Holmes JM, Wijono M, Wheeler LA, Garst ME (1996) EurJMed Chem 31, 175-186. 13 Supuran CT, Nicolae A, Popescu A (1996) Eur JMed Chem 31, 431-438. 14 Supuran CT, Popescu A, Ilisiu M, Costandache A, Banciu MD (1996) Eur JMed Chem 31, 439-448. 15 Supuran CT, Scozzafava A, Popescu A, Bobes-Tureac R, Banciu A, Creanga A, Bobes-Tureac G, Banciu MD (1997) Eur JMed Chem 32, 445-452. 16 Alzuet G, Casanova J, Ramirez JA, Borras J, Carugo O (1995) J Inorg Bioehem 57, 219-234. 17 Sumalan SL, Casanova J, Alzuet G, Borras J, Castifieiras A, Supuran CT (1996) J Inorg Biochem 62, 3139. 18 a) Supuran CT (1995) Metal Based Drugs 2, 327-330; b) Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 19-26. 19 a) Borras J, Cristea T, Supuran CT (1996), Main Group Met Chem 19, 339-346; b) Jitianu A, Ilies MA, Briganti F, Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 1-7. 20 a) Supuran CT, Scozzafava A (1997) J Enzyme Inhibition 12, 37-51; b) Supuran CT, Briganti F, Scozzafava A (1997) J Enzyme Inhibition 12, 175-190. 21 Mincione G, Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 27-34. 22 Alzuet G, Ferrer S, Borras J, Supuran CT (1994) Roum Chem Quart Rev 2, 283-300. 23 Briganti F, Mangani S, Orioli P, Scozzafava A, Vernaglione G, Supuran CT (1997) Biochemistry 36, 10384-10392. 24 Borras J, Casanova J, Cristea T, Gheorghe A, Scozzafava A, Supuran CT, Tudor V (1996) Metal Based Drugs 3, 143-148. 25 Supuran CT, Mincione F, Scozzafava A, Briganti F, Mincione G, Ilies MA (1997) Eur J Med Chem, in <br><br> press. 26 Supuran CT, Scozzafava A, Saramet I, Banciu MD (1997) J Enzyme Inhibition, in press. 27 Jitianu A, Ilies MA, Scozzafava A, Supuran CT (1997) Main Group Met Chem 20, 151-156. 28 Forsman C, Behravan G, Osterman A, Jonsson BH (1988) Acta Chem Scand B42, 314-318. 29 Behravan G, Jonasson P, Jonsson BH, Lindskog S (1991) Eur JBiochem 198, 589-592. 30 Khalifah RG, Strader DJ, Bryant SH, Gibson SM (1977) Biochemistry 16, 2241-2247. 31 Nyman PO, Lindskog S (1964) Biochim Biophys Acta 85, 141-151. 32 Henderson LE, Henriksson D, Nyman PO (1976)JBiol Chem 251, 5457-5463. 33 Maren TH, Wynns GC, Wistrand PJ (1993) Mol Pharmaco144, 901-906. 34 Young RW, Wood KH, Vaughan JR, Anderson GW (1956) J Am Chem Soc 78, 4649-4654. 35 Pocker Y, Stone JT (1967) Biochemistry 6, 668-678. 36 Baird TT, Waheed A, Okuyama T, Sly WS, Fierke CA (1997) Biochemistry 36, 2669-2678. 37 Maren TH, Bar-Ilan A, Conroy CW, Brechue WF (1990) Exp Eye Res 50, 27-36. 38 Maren TH, Brechue WF, Bar-Ilan A (1992) Exp Eye Res 55, 73-79. 39 Brechue WF, Maren TH (1993) Invest Ophthalmol Vis Sci 34, 2581-2587. 40 Borja P, Alzuet G, Server-Carri6 J, Borras J, Supuran CT (1997) J Biol Inorg Chem (JBIC), in press. 41 Supuran CT, Balaban AT, Gheorghiu MD, Schiketanz A, Dinculescu A, Puscas I (1990) Rev Roum Chim 35, 399-405. 42 Supuran CT, Scozzafava A (1997) unpublished results. <br><br> Received: September 24, 1997 Accepted: October 17, 1997 Received in revised camera-ready format" October 23, 1997 <br><br> 315 <br><br>  <h3>pmcA2398786</h3>Genomic-Bioinformatic Analysis of Transcripts Enriched in the Third-Stage Larva of the Parasitic Nematode <span class="yellow">Ascaris suum</span>
Abstract
Differential transcription in <span class="yellow">Ascaris suum</span> was investigated using a genomic-bioinformatic approach. A cDNA archive enriched for molecules in the infective third-stage larva (L3) of <span class="yellow">A. suum</span> was constructed by suppressive-subtractive hybridization (SSH), and a subset of cDNAs from 3075 clones subjected to microarray analysis using cDNA probes derived from RNA from different developmental stages of <span class="yellow">A. suum</span>. The cDNAs (n = 498) shown by microarray analysis to be enriched in the L3 were sequenced and subjected to bioinformatic analyses using a semi-automated pipeline (ESTExplorer). Using gene ontology (GO), 235 of these molecules were assigned to ‘biological process’ (n = 68), ‘cellular component’ (n = 50), or ‘molecular function’ (n = 117). Of the 91 clusters assembled, 56 molecules (61.5%) had homologues/orthologues in the free-living nematodes <span class="yellow">Caenorhabditis elegans</span> and <span class="yellow">C. briggsae</span> and/or other organisms, whereas 35 (38.5%) had no significant similarity to any sequences available in current gene databases. Transcripts encoding protein kinases, protein phosphatases (and their precursors), and enolases were abundantly represented in the L3 of <span class="yellow">A. suum</span>, as were molecules involved in cellular processes, such as ubiquitination and proteasome function, gene transcription, protein–protein interactions, and function. In silico analyses inferred the <span class="yellow">C. elegans</span> orthologues/homologues (n = 50) to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%), or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway. Small numbers of these 50 molecules were predicted to be secreted (10%), anchored (2%), and/or transmembrane (12%) proteins. Functionally, 17 (34%) of them were predicted to be associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority being embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%). A genetic interaction network was predicted for these 17 <span class="yellow">C. elegans</span> orthologues, revealing highly significant interactions for nine molecules associated with embryonic and larval development (66.9%), information storage and processing (5.1%), cellular processing and signaling (15.2%), metabolism (6.1%), and unknown function (6.7%). The potential roles of these molecules in development are discussed in relation to the known roles of their homologues/orthologues in <span class="yellow">C. elegans</span> and some other nematodes. The results of the present study provide a basis for future functional genomic studies to elucidate molecular aspects governing larval developmental processes in <span class="yellow">A. suum</span> and/or the transition to parasitism.<br><br>Introduction
Parasitic nematodes are of major socio-economic importance in animals. For example, hundreds of millions of <span class="yellow">people</span> are infected with geohelminths (soil-transmitted worms), such as blood-feeding hookworms <span class="yellow">Ancylostoma duodenale</span> and/or <span class="yellow">Necator americanus</span>, <span class="yellow">Trichuris trichiura</span> and Ascaris spp. [1], causing serious adverse effects on <span class="yellow">human</span> health, particularly in <span class="yellow">children</span>. Similarly, parasitic nematodes of livestock, such as <span class="yellow">pigs</span>, also cause substantial economic losses due to subclinical and clinical diseases, with billions of dollars spent annually on the treatment and control of gastro-intestinal nematodes. In addition to the socioeconomic impact that these parasites have, there is potential for the emergence of resistance in them against all of the main classes of (nematocidal) compounds used to treat the diseases they cause [2]–[5]. Therefore, there is a significant need to work toward discovering new compounds to control these parasites. Gaining an improved understanding of the molecular basis of parasite development provides such an avenue.
Compared with the free-living nematode <span class="yellow">Caenorhabditis elegans</span>, there is very little information on fundamental molecular aspects of development in parasitic nematodes [6]–[8]. Since the genome sequence of <span class="yellow">C. elegans</span> was published in 1998 [9], many aspects of the molecular biology of this nematode have been elucidated. For instance, microarray analyses have been used to examine developmental and gender-enriched gene expression [10],[11], and the functions of more than 96% of the <span class="yellow">C. elegans</span> genes have been assessed by double-stranded RNA interference (RNAi, or gene silencing; [12]) [13]–[18]. Comparative analyses of genetic data sets have shown that parasitic nematodes usually share ∼50–70% of genes with <span class="yellow">C. elegans</span> (e.g., [19],[20]). There is similarity in other features (such as basic body plan and moulting) between <span class="yellow">C. elegans</span> and parasitic nematodes, suggesting that some molecular pathways are relatively conserved [8],[21]. Understanding the pathways linked to basic nematode biology and development could have important implications for finding new ways of disrupting these pathways and thus facilitate the identification of new drug targets.
Despite the advances in genomic technologies [7], [22]–[29] and the study of <span class="yellow">C. elegans</span>, there is a paucity of information on the genomics of parasitic nematodes of animals, particularly in relation to development. Also considering the major socioeconomic impact of Ascaris and ascariasis in <span class="yellow">humans</span> and <span class="yellow">pigs</span> [30]–[32], several characteristics, including the large size of the adult worm (providing the opportunity of investigating individual organ systems and tissues), the ability to maintain Ascaris in the <span class="yellow">pig</span>, store eggs and culture larvae in vitro for relatively long periods of time (months to years) [32] as well as the discovery that RNAi achieves “cross-species” gene silencing for a selected number of genes [33],[34] and the imminent genome sequence (http://www.sanger.ac.uk/Projects/Helminths/) all indicate that Ascaris could serve as a powerful model system for investigating reproductive and developmental processes in nematodes.
In the present study, Ascaris from <span class="yellow">pigs</span> was used to study molecules abundantly transcribed in the infective third-stage larva (L3). Following the oral ingestion of Ascaris eggs by the host (<span class="yellow">human</span> or <span class="yellow">pig</span>), L3s are released and then invade/penetrate predominantly the caecal wall [35] to then undergo hepato-pulmonary migration, after which ultimately the adult females and males establish and develop in the small intestine [36],[37]. The molecular mechanisms linked to host invasion and parasite development are largely unknown. Here, we constructed an L3-enriched cDNA library using the method of suppressive-subtractive hybridization (SSH), explored transcription of a representative subset of molecules by microarray analysis and conducted bioinformatic analyses to characterize these molecules, map them to biochemical pathways and predict genetic interactions based on comparisons with <span class="yellow">C. elegans</span> and/or other organisms.<br><br>Materials and Methods
Production of Different Developmental Stages of Ascaris
Experimental <span class="yellow">pigs</span> (8–12 weeks of age) were purchased from and maintained in the Experimental Animal Center of South China Agricultural University. These <span class="yellow">pigs</span> were treated humanely, according to the Animal Ethics procedures and guidelines of the People's Republic of China. Adult <span class="yellow">worms</span> (males and females) of <span class="yellow">A. suum</span> were collected from the small intestines of <span class="yellow">pigs</span> from an abattoir in Shenzhen, China. Infective eggs and infective L3s of <span class="yellow">A. suum</span> were produced according to the methods described previously [38]. In brief, eggs from the uteri of adult females of <span class="yellow">A. suum</span> were collected and incubated at 28°C for 28 days to allow them to develop to infective eggs (containing infective L3s). To obtain pure infective L3s, 7.5% v/v sodium hypochlorite was used to treat the larvated eggs at 37°C overnight and then the eggs were shaken with glass-beads; then, the exsheathed L3s and shells were separated by density gradient centrifugation using lymphocyte separating medium (LSM) [38]. Following the experimental infection of helminth-free <span class="yellow">pigs</span> with infective Ascaris eggs as described previously [39], the L3s from livers and in lungs as well as L4s in intestines were isolated according to an established method [40]. All parasite materials were snap-frozen in liquid nitrogen prior to storage at −70°C.<br><br>Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)
Total RNA was isolated from adult females and males, different larval stages or eggs of <span class="yellow">A. suum</span> using TriPure reagent (Roche) as recommended by the manufacturer. Equal amounts of total RNA from each stage or sex were pooled. The mRNA was isolated using the Oligotex mRNA Kit (Qiagen), following the manufacturer's protocol. SSH was carried out using the PCR-Select cDNA Subtraction kit (Clontech), according to the manufacturer's protocol. In brief, cDNA synthesized from mRNAs from infective L3s was subtracted against cDNA synthesized from the pooled mRNA from all other stages included herein. The SSH library was constructed using infective L3s as the tester and pooled cDNAs from all other stages as the driver. The effectiveness of this subtraction process has already been demonstrated in previous studies [41],[42]. The cDNA obtained following SSH was cloned into the pGEM-T Easy plasmid vector (Promega) and competent <span class="yellow">Escherichia coli</span> (JM109) transformed. Positive clones, picked randomly (based on blue/white selection), were grown overnight in Luria Bertani (LB) medium (shaking, 37°C). Individual inserts were PCR-amplified using “nested primers” 1 and 2R from the Subtraction kit (Clontech) and examined by agarose electrophoresis.<br><br>Preparation of Microarray Slides
Clones (n = 3075) from the subtracted library were picked and cultured overnight in LB containing ampicillin (1000 IU/ml) in sealed 96-well blocks. Five µl of culture suspension from each well were transferred into individual wells thermocycling (96-well) plates and the inserts PCR-amplified using primers 1 and 2R. Following a 10 min denaturation step at 94°C, the amplification proceeded for 25 cycles of 10 s at 94°C, 30 s at 68°C and 1.5 min at 72°C, with a final extension for 5 min at 72°C. Products were resolved in agarose gels, ethanol precipitated, re-suspended in 16 µl of “spotting solution” (Shanghai BioStar Genechip, Inc) to a final concentration of ∼500 ng per µl, before being printed on to glass slides (in duplicate) using a robotic arrayer. Sixteen blanks (using spotting solution only) and the same number of negative (irrelevant cDNAs with no relationship to Ascaris) were also printed on to slides and served as negative controls; β-actin of <span class="yellow">A. suum</span> served as a positive control to assess the efficiency of labeling and hybridization. The slides were air-dried for 2 h, and cDNA in the spots were cross-linked at 254 mJ. The printed slides were stored at 4°C.<br><br>Labeling of cDNA Probes with Fluorescent Dyes, and Microarray Analysis
The cDNAs produced from total RNA from <span class="yellow">A. suum</span> eggs, infective L3s, L3s isolated from <span class="yellow">pig</span> liver or lung, fourth-stage larvae (L4s), adult males or females [as described in the section ‘Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)’] were labeled with cyanine dyes. Cy3 or Cy5-dCTP was incorporated into cDNA produced from 30 µg of total RNA by direct labeling in a reverse transcription reaction using an oligo (dT) primer. Labeled cDNA was purified using DyeEx columns (Qiagen).
Microarray slides were incubated with a pre-hybridization solution [5×SSC, 1% <span class="blue">bovine</span> serum albumin (BSA), 0.1% sodium dodecyl-sulphate (SDS)] for 6 h at 42°C. After pre-hybridization, the microarray slides were incubated with ‘pooled’ Cy3 and Cy5-labeled probes in hybridization solution (5×SSC, 1% BSA, 0.1% SDS), in the dark at 42°C for 18 h, and then washed in solution I (1×SSC, 0.2% SDS) for 10 min, followed by solution II (0.1×SSC, 0.2% SDS) for 10 min at 60°C, according to the protocols provided by Shanghai BioStar Genechip, Inc. A “dye flip” was carried out to control for any bias in hybridization signal between the Cy-labeled cDNA probes (produced for two distinct mRNA populations). The slides were dried and scanned (ScanArray 4000 scanner) using image acquisition software (Shanghai BioStar Genechip Inc.) and a range of laser power and photo-multiplier tube intensities. The mean hybridization signal (derived from four replicates of the same array) were corrected for background, normalized [43], log2-transformed and then subjected to statistical analysis employing the students t-test in a spreadsheet (Excel, Microsoft, USA). The microarray data were analysed for differential cDNA hybridization (>2.0-fold to 114.3-fold) between L3 and each of the other stages (eggs, lung and liver L3s, L4, adult female and adult male).<br><br>Verification of Differential Hybridization by Reverse Transcription-Coupled Polymerase Chain Reaction (RT-PCR) Analysis
For a subset (n = 17) of representative ESTs (rESTs), RT-PCR was used to verify the differential transcription recorded by microarray analysis. Double-stranded cDNA was synthesized from total RNA (separately) from each stage or sex of <span class="yellow">A. suum</span> using reverse transcriptase (Superscript III, Invitrogen). Briefly, 5 µg of total RNA were added to 14 µl of H2O and 1 µl of oligo d(T)n = 12–18 primer (0.5 µg/µl), heated to 70 °C for 10 min and chilled on ice. First- and second-strand cDNAs were synthesized via the addition of 4 µl of first-strand cDNA buffer (250 mM Tris-HCl, pH 8.3, 375 mM KCl and 15 mM MgCl2), 2 µl of 0.1 M dithiothreitol, and 1 µl of 10 mM of each dNTP, followed by an incubation at 25 °C (10 min), 42 °C (50 min) and 70 °C (15 min). One-tenth of each double-stranded cDNA produced was then used as a template in the PCR. The transcripts were amplified from individual cDNAs by PCR using oligonucleotide primers (sequences available upon request) designed to each EST. The PCR amplification of a portion (209 bp) of the β-actin gene (accession no. BI594141) using forward primer (5′-CTCGAAACAAGAATACGATG-3′) and reverse primer (5′- ACATGTGCCGTTGTATGATG-3′), previously determined to be present in all developmental stages and both sexes of <span class="yellow">A. suum</span> [44], served as a positive control. Samples without template (no-DNA controls) were included in each PCR run. The following cycling conditions were employed: one cycle at 94 °C (5 min), 94 °C (30 s), 60 °C (30 s) and 72 °C (30 s) for 30 cycles, followed by a final extension of 70 °C (7 min). Following the PCR, 5 µl of individual amplicons were resolved in ethidium bromide-stained agarose gels (2%) and then photographed upon transillumination. The relative band intensities were analyzed using UVIsoft Image Acquisition and Analysis software (UVITEC). The specificity and identity of individual amplicons were confirmed by direct sequencing using the same primers (separately) as employed for their amplification.<br><br>Sequencing and Bioinformatics Analyses
Clones from the SSH cDNA library with increased hybridization in microarray analysis to the infective L3 compared with other stages were sequenced using standard technology [45]. The nucleotide sequences have been deposited in the GenBank database under accession numbers ES290984-ES291074. Following the processing of the sequences (i.e., removal of vector sequences, quality assurance and clustering), contigs or singletons from individual clusters were subjected to BLASTx (NCBI: www.ncbi.nlm.nih.gov) and BLASTn (EMBL-EBI Parasite Genome Blast Server: www.ebi.ac.uk) analysis to identify putative homologues in <span class="yellow">C. elegans</span>, other nematodes and other organisms (e-value of ≤1e-05). Peptides inferred from ESTs were classified functionally using Interproscan (available at http://www.ebi.ac.uk/InterProScan/) employing the default search parameters. WormBase (www.wormbase.org) was interrogated extensively for relevant information on <span class="yellow">C. elegans</span> homologues/orthologues, including RNAi phenotypic, transcriptomic, proteomic and interactomic data. ESTs with homologues/orthologues in <span class="yellow">C. elegans</span> and other nematodes were also subjected to analysis employing the KEGG Orthology-Based Annotation System (KOBAS) (www.kobas.cbi.pku.edu.cn), which predicts the biochemical pathways in which molecules are involved. The open reading frames (ORFs) inferred from selected ESTs with orthologues in <span class="yellow">C. elegans</span> were also subjected to “secretome analysis” using the program SignalP v.2.0 www.cbs.dtu.dk/services/SignalP/), employing both the neural network and hidden Markov models to predict signal peptides and/or anchors [46]–[48]. Also, transmembrane domains were predicted using the program TMHMM (www.cbs.dtu.dk/services/TMHMM/; [49]–[51]), and subcellular localization inferred employing the program WoLF PSORT (http://wolfpsort.org/; [52]).
The method established by Zhong and Sternberg [53] was used to predict the interactions for <span class="yellow">C. elegans</span> orthologues of the L3-enriched molecules from Ascaris. In brief, interaction, phenotypic, expression and gene ontology data from <span class="yellow">fruitfly</span>, <span class="yellow">yeast</span>, <span class="yellow">mouse</span> and <span class="yellow">human</span> were integrated using a naïve Bayesian model to predict genetic interactions among <span class="yellow">C. elegans</span> genes ([45],[53]; Zhong and Sternberg, unpublished). The predicted networks resulting from the analyses were saved in a graphic display file (gdf) format and examined using the graph exploration system available at http://graphexploration.cond.org/. Images were labeled and saved in the joint photographic experts group (jpeg) format.<br><br>
Results
To identify molecules transcribed abundantly in the L3 of <span class="yellow">A. suum</span>, an enriched cDNA library was constructed by SSH. From a total of 3075 clones from this library, 2921 (95%) were shown to contain an insert (which could be amplified by PCR). From 2671 (92%) of these clones, amplicons representing single bands of ∼400 to 600 bp in size were produced. These latter amplicons were arrayed (in duplicate) on to slides and then hybridized with Cy3-labeled L3-cDNA or with Cy5-labeled cDNA from eggs, liver/lung L3s, L4s, adult female or adult male of Ascaris. Dye flip was conducted to verify the hybridization data. Of the 2671 (duplicate) spots, 1526 had a significant difference in hybridization between infective L3 cDNA and cDNAs from all other stages or sexes of <span class="yellow">A. suum</span>, of which 515 had a >2.0-fold increased hybridization for the L3.
In order to independently verify the hybridization results in the microarray, a PCR-based analysis of a selected subset (n = 17) clones was conducted using specific primer pairs. Having verified the specificity and identity of individual amplicons by sequencing, PCR results were reproducible (based on multiple runs on different days) and ∼94% (16 of 17) concordant with those of the microarray analysis (not shown). There was complete concordance for representative clones associated with a differential signal of ≥3.0-fold in the microarray.
The clones linked to the 515 spots representing increased transcription (>2.0-fold) in infective L3 compared with the other developmental stages or sexes included were subjected to sequencing. The 498 sequences (length: 550±115 bp) determined were then subjected to detailed bioinformatic analyses. There were 91 unique clusters (accession numbers ES290984-ES291074), of which 55 were singletons (sequences determined once). Of 56 molecules (61.5%) with significant similarity to sequences other than <span class="yellow">A. suum</span> in the databases interrogated, 50 (54.9%) had <span class="yellow">C. elegans</span> or <span class="yellow">C. briggsae</span> homologues, and six had similarity to ESTs already sequenced from ascaridoid and/or other parasitic nematodes, and/or other organisms (Table 1). A significant proportion (38.4%) did not have any similarity in sequence to any organisms for which data are presently available. Comparative analysis specifically against <span class="yellow">A. suum</span> EST data sets (n∼42,000) available in public databases confirmed independently that the majority of molecules (>60%) were present exclusively in the infective L3 stage or were orphans.
As gene ontology (GO) provides a hierarchy that unifies the descriptions of biological, cellular and molecular functions [54], this approach was employed to predict the classification and gene function of molecules enriched in infective L3 of <span class="yellow">A. suum</span>. A summary of the GO categories of these molecules is displayed in Fig. 1. Of the 91 contigs, 32 (35%) could be functionally assigned to ‘biological process’ (n = 38), ‘cellular component’ (n = 17) and ‘molecular function’ (n = 64). The most common subcategories were gluconeogenesis (13%) and metabolic process (13%) within ‘biological process’, extracellular region (24%) within ‘cellular component’, and catalytic activity (11%) and phosphoenolpyruvate carboxykinase activity (8%) within ‘molecular function’ (Table S1).
A focused KOBAS analysis inferred the 50 <span class="yellow">C. elegans</span> orthologues/homologues to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%) or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway (Table 2). Of these 50 molecules, small numbers were predicted to be secreted (10%), anchored (2%) and/or transmembrane (12%) proteins (Table 2). Functionally, 17 (34%) of the 50 molecules were associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority displaying embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%) (Table 2).
Extending this analysis, a relatively complex genetic interaction network was predicted for the 17 <span class="yellow">C. elegans</span> orthologues (i.e., with non-wild-type RNAi phenotypes) (see Table S2). Statistically highly significant interactions were predicted for nine of the <span class="yellow">C. elegans</span> genes; the top five interactors are displayed in Fig. 2. The gene ontology categories for eight selected <span class="yellow">C. elegans</span> genes (F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2) included: embryonic development, egg hatching, larval development and/or growth. The other categories included: positive regulation of growth rate (F55A12.8, kin-2, mup-2, pab-1, rpl-22 and T21B10.2) and gamete generation and locomotory behaviour (kin-2, mup-2, pab-1 and F55A12.8, kin-2, mup-2, respectively). The <span class="yellow">C. elegans</span> homologue egl-3 was predicted to be involved in proteolysis (see www.wormbase.org). All nine <span class="yellow">C. elegans</span> orthologues were predicted to interact directly with a total of 296 (range: 5–75) other genes and, in particular, a direct genetic interaction was predicted between pab-1 and T21B10.2 (Fig. 2). The 296 interactors were associated with embryonic and larval development (n = 198; 66.9%), information storage and processing (n = 15; 5.1%), cellular processes and signalling (n = 45; 15.2%) and metabolism (n = 18; 6.1%); the precise function of some of the interactors (n = 20; 6.7%) is presently unknown (Table S2).<br><br>Discussion
The present study investigated transcripts in infective L3s of <span class="yellow">A. suum</span> using a genomic-bioinformatic platform. The focus was on comparisons with <span class="yellow">C. elegans</span> homologues/orthologues, because the entire genome sequence of this nematode is known [9] and because there is a wealth of information on the localization and functionality of its molecules (www.wormbase.org; http://elegans.bcgsc.bc.ca/knockout.shtml). The functions of most genes in <span class="yellow">C. elegans</span> have been assessed using RNAi (e.g., [14],[15],[17],[55],[56]) in the hermaphroditic stage, whereas there is a paucity of functional information available for Ascaris and other parasitic nematodes of animals [57],[58].
Following the microarray analysis of >2500 ESTs from the SSH library, 498 cDNAs inferred to be enriched in the L3, based on hybridization signal, were sequenced and subjected to comprehensive in silico analyses. Of the 91 clusters of molecules categorized, 50 (54.9%) had <span class="yellow">C. elegans</span> homologues/orthologues with loss-of-function phenotypes could be mapped to key pathways. The statistically significant genetic interactions predicted for 9 of the 50 <span class="yellow">C. elegans</span> orthologues [namely egl-3, F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2 ( = enol-1)] and the interaction network included genes encoding kinases, alpha-tubulins, enolases, troponin and other named and unnamed proteins. Eight of these molecules (enol-1, pab-1, F33D11.10, rpl-22, F55A12.8 mec-12, mup-2 and kin-2) have known or predicted roles in embryonic and larval growth and development, gamete generation, locomotory behaviour or other biological processes in <span class="yellow">C. elegans</span> (see www.wormbase.org).
The enolase encoded by enol-1 is predicted to play a role in glycolysis, gluconeogenesis, phenylalanine, tyrosine and tryptophan biosynthesis (cf. [59]). Since glucose is the main source for ATP production, the alteration in these key glycolytic enzymes may lead to cellular dysfunction, such as impaired ion-motive ATPase required to maintain potential gradients, operate pumps and maintain membrane lipid asymmetry [60]. Bioinformatic analysis for transmembrane helices (TMHMM) and peptide signal sequences (SignalP) predicted ENOL-1 to be a non-secreted protein localized to the cytoplasm (cf. Table 2). Nonetheless, enolases are often detected in the excretory/secretory (ES) products of parasitic helminths, including adult <span class="yellow">A. suum</span> [61], and appear to play a role in the triggering of nitric oxide production by host cells. The enol-1 orthologue of <span class="yellow">C. elegans</span> has been predicted to interact specifically with the polyadenylate binding protein gene, pab-1, inferred to be involved in coordinated gene transcription and expression during normal larval development [16]. Poly(A)-binding proteins (PABPs) are recognized to be central to the regulation of mRNA translation and stability [62]. Present evidence suggests that the expression of PAB-1 is regulated by an oligo-pyrimidine tract in response to cell growth and relates to coordinated growth regulation in <span class="yellow">C. elegans</span> [62]. Furthermore, gene silencing of pab-1 and its selected interactors (see Fig. 2) leads to embryonic lethal (Emb), slow growth (Slo) and sterile progeny (Stp) phenotypes (see www.wormbase.org).
Another gene (F33D11.10; EST code 4F10; see Table 2) which encodes an ATP-dependent RNA helicase and is associated with embryonic lethal (Emb) and larval lethal (Lvl) RNAi phenotypes, was shown to be highly transcribed in infective L3s of <span class="yellow">A. suum</span>. Helicases are involved in a variety of RNA metabolic processes, including translation initiation, pre-mRNA splicing, pre-rRNA processing, rRNA maturation and RNA degradation [63], and are crucial for life cycle progression, sex determination and early embryogenesis in <span class="yellow">C. elegans</span> [60]. The high transcription levels of a homologue/orthologue in the L3 of <span class="yellow">A. suum</span> might suggest a similar role in this ascaridoid. Similarly, the coordination of the expression of a large number of genes is required for normal growth and cell proliferation during larval development. The high transcription level for the ribosomal protein gene homologue rpl-22 (large subunit family member; EST code 26G12, see Table 2) in the infective L3 of <span class="yellow">A. suum</span> compared with other developmental stages is likely to reflect the substantial rate of cell growth in this stage [64].
The gene (F55A12.8, EST code 4G11; see Table 2) encoding an acetyl-transferase with a putative ATPase domain, shown to be enriched in the L3 of <span class="yellow">A. suum</span>, was predicted to interact with 75 other genes all involved in energy production and/or RNA processing (see Table S2). Several molecules involved in ATP synthesis and mitochondrial pathways (e.g., cytochrome oxidase c subunits 1, 2 and 3, ADP/ATP translocases, NADH dehydrogenases, ATPases and ATP synthetases) have been reported previously to be highly represented in the L3 stage of <span class="yellow">Anisakis simplex</span> [65], thus supporting the proposal that substantial energy is required for larval development as well as the transition from the free-living to the parasitic stage and the invasion of the host. There is also likely to be a substantial energy requirement for muscle contraction linked to larval motility in <span class="yellow">A. suum</span>, as the L3s penetrate the caecal wall in the porcine host, before undergoing hepato-pulmonary migration [35]. Accordingly, genes encoding a specialized tubulin expressed in mechanoreceptors (mec-12, EST code 13E09) and a troponin (mup-2, EST code 01G03; see Table 2), both predicted to interact with a total number of 32 tubulin- and myosin-encoding genes, also supported a link to extensive muscle contraction and motility in <span class="yellow">A. suum</span> L3s. Also, neuroactive peptides are required to regulate the responsiveness of nematode larvae to mechanical stimuli [66]. A homologue encoded by egl-3 was shown to be highly transcribed in the L3 of <span class="yellow">A. suum</span>; EGL-3 is predicted to be a pro-hormone convertase involved in the maturation of neuropeptides and could be associated with mechano-sensory responses and touch sensitivity linked to the host invasion.
A regulatory subunit of a cAMP-dependent protein kinase (kin-2, EST code 22H01; see Table 2) was predicted to interact with 72 other genes all involved in diverse cellular processes, such as nuclear trafficking, and DNA replication and repair (see Table S2). Based on gene ontology terms, kin-2 is implicated in gamete generation, growth, larval development, post-embryonic body morphogenesis, signal transduction and/or protein amino acid phosphorylation (see Table S2). Gene silencing of kin-2 in <span class="yellow">C. elegans</span> leads to phenotypes, such as larval lethal (Lvl), larval arrest (Lva), body morphology defect (Bmd), dumpy (Dpy), uncoordinated (Unc) and sterile progeny (Stp) (www.wormbase.org), suggesting that its homologue in <span class="yellow">A. suum</span> is central to larval maturation. The KOBAS analysis predicted the protein KIN-2 to be involved in the insulin-signaling pathway, previously implicated in controlling the exit from dauer in <span class="yellow">C. elegans</span> and the activation of L3s of the <span class="yellow">canine hookworm</span>, <span class="yellow">Ancylostoma caninum</span>, following exsheathment [67]. In a recent study, Brand and Hawdon [68] were able to inhibit (with a phosphoinositide-3-OH-kinase inhibitor) the activation of infective L3s of both of the hookworms <span class="yellow">Ancylostoma caninum</span> and <span class="yellow">Ancylostoma ceylanicum</span> via the insulin signaling pathway, thus lending some credence to the hypothesis that this pathway plays an critical role in regulating the transition from the free-living to the parasitic stage [68]. Recently, it has been proposed that transcriptional and feeding responses to serum-stimulation in <span class="yellow">Ancylostoma caninum</span> are regulated by parallel systems, with the insulin signaling pathway playing a significant role in the ‘resumption of feeding’ in activated larvae [69].
Protein kinases are also likely to be involved in pathways linked to sexual maturation in developing larvae. As already proposed for adult stages of <span class="yellow">H. contortus</span> [45], the protein kinase gene cdk-1 is predicted to play a pivotal role in the germline, oogenesis and spermiogenesis pathways of this parasitic nematode. Other protein kinases, such as PEPCK, and phosphatases, were shown herein to be transcribed at high levels in the L3 stage compared with other developmental stages of <span class="yellow">A. suum</span> (see Table 2), which is in accordance to findings reported recently for <span class="yellow">Anisakis simplex</span> [65]. Due to their major regulatory effects in eukaryotic signaling events and regulatory and sensory functions, protein kinases have been considered interesting targets for anti-parasitic drugs [70].
In conclusion, this study has given some interesting insights into early molecular processes in the L3 of <span class="yellow">A. suum</span>. Approximately 60% of the transcripts enriched in the L3 stage of <span class="yellow">A. suum</span> have homologues/orthologues in <span class="yellow">C. elegans</span>. The bioinformatic analyses of selected molecules suggest that a complex genetic network regulates or controls larval growth and development in <span class="yellow">A. suum</span> L3s, and some of these might be involved in or regulate the switch from the free-living to the parasitic stage. Some caution is warranted in drawing conclusions regarding molecular mechanisms regulating the transition to parasitism in parasitic nematodes from information on <span class="yellow">C. elegans</span>, as latter is a free-living nematode. Also, while the method of data integration is essential for the reliable prediction of genetic interactions, it might limit the capacity of the approach somewhat to infer nematode-specific interactions. As additional datasets of genes and gene functions become available for various parasitic nematodes, more informed inferences can be made regarding the functions of nematode-specific genes, particularly those involved in the transition to parasitism. The imminent genome sequence of <span class="yellow">A. suum</span> (http://www.sanger.ac.uk/Projects/Helminths/) should all assist in this endeavour. Also, functional analysis of selected molecules representing selected ESTs identified herein, utilizing gene silencing approaches established recently [33],[34], could provide some insights into developmental processes in Ascaris and related ascaridoid nematodes and provide avenues for the development of novel approaches for their control.<br><br>Supporting Information<br><br>
<h3>pmcA1885664</h3>The recombination-associated protein RdgC adopts a novel toroidal architecture for DNA binding
Abstract
RecA plays a central role in the nonmutagenic repair of stalled replication forks in bacteria. RdgC, a recombination-associated DNA-binding protein, is a potential negative regulator of RecA function. Here, we have determined the crystal structure of RdgC from <span class="yellow">Pseudomonas aeruginosa</span>. The J-shaped monomer has a unique fold and can be divided into three structural domains: tip domain, center domain and base domain. Two such monomers dimerize to form a ring-shaped molecule of approximate 2-fold symmetry. Of the two inter-subunit interfaces within the dimer, one interface (‘interface A’) between tip/center domains is more nonpolar than the other (‘interface B’) between base domains. The structure allows us to propose that the RdgC dimer binds dsDNA through the central hole of ∼30 Å diameter. The proposed model is supported by our DNA-binding assays coupled with mutagenesis, which indicate that the conserved positively charged residues on the protein surface around the central hole play important roles in DNA binding. The novel ring-shaped architecture of the RdgC dimer has significant implications for its role in homologous recombination.<br><br>INTRODUCTION
Homologous recombination systems contribute to the maintenance of genome integrity and are essential for the repair of stalled replication forks (1). Genetic studies of the recombination-dependent growth C (rdgC) gene in bacterial systems have suggested that it is involved in DNA replication and recombination (2,3). In <span class="yellow">Neisseria gonorrhoeae</span> (the gonococcus), RdgC is required for efficient pilin antigenic variation and plays some role in cell growth (4,5). The <span class="yellow">Escherichia coli</span> rdgC gene encodes a DNA-binding protein of 34 kDa, whose expression level was at its highest during exponential phase, reaching its maximum at ∼1000 dimers per cell (2). Its level decreased sharply to ∼50 dimers per cell in stationary phase. This profile suggests that RdgC might function during the period of DNA replication (2).
The RecA proteins and their homologs play a key role in recombinational DNA repair in bacteria and eukaryotes. Their aberrant reactions could result in gross chromosomal rearrangements that lead to <span class="yellow">human</span> diseases, including cancer. Therefore, their functions must be tightly regulated, and understanding how the RecA family of recombinases is regulated is of utmost importance (1). The <span class="yellow">E. coli</span> RdgC protein is a potential negative regulator of the function of RecA (1). It inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). Sedimentation equilibrium data indicated that <span class="yellow">E. coli</span> RdgC exists in solution as a mixture of oligomeric states in equilibrium, most likely monomers, dimers and tetramers (1). This concentration-dependent change in the oligomeric state appears to affect its mode of binding to DNA and its capacity to inhibit RecA (1). The primary mechanism of RdgC inhibition appears to involve a simple competition for DNA-binding sites, especially on double-stranded DNA (dsDNA) (1).
Deletion of the <span class="yellow">E. coli</span> rdgC gene alone causes no obvious phenotype but is highly deleterious in strains lacking certain enzymes involved in recombination and replication restart (2,3). RdgC is essential for the growth of an <span class="yellow">E. coli</span> strain lacking PriA, indicating that it might affect replication fork progression or fork rescue (2). PriA provides a means to load the DnaB replicative helicase at DNA replication fork and D loop structures (2). RdgC is, therefore, a key factor in the rescue of stalled or broken forks and subsequent replication restart. dnaC suppressors of PriA can overcome this inviability, especially when RecF, RecO or RecR is inactivated, which indicates that RdgC avoids or counters the toxic effect of these proteins by limiting inappropriate RecA loading on SSB-coated ssDNA (2).
The DNA-binding ability of the RdgC protein has been demonstrated clearly using electrophoretic mobility shift assays (2). <span class="yellow">Escherichia coli</span> RdgC binds non-specifically to dsDNA and with higher affinity than ssDNA (2). RdgC from <span class="yellow">N. meningitidis</span> also binds DNA with little specificity for sequence or structure, like the <span class="yellow">E. coli</span> protein (4). RdgC exhibits no preference for DNA replication or recombination intermediates (2). However, no detectable domains or nuclease activity could be identified for the <span class="yellow">E. coli</span> RdgC protein (2,6). The complexes of <span class="yellow">E. coli</span> RdgC with both linear and supercoiled circular plasmid DNA were imaged using atomic force microscopy (7). RdgC was found to have an increased affinity to DNA ends and to promote bending of DNA (7). RdgC has the effect on DNA superstructure; the promotion of DNA condensation was observed at high protein concentrations (7). Recombination is largely enhanced by close contacts of distant regions along the DNA strands through condensation (7).
<span class="yellow">Pseudomonas aeruginosa</span> is a ubiquitous environmental Gram-negative bacterium that belongs to the γ subdivision of the Proteobacteria. Orthologs of the rdgC gene are found only in β and γ subdivisions of the Proteobacteria (2). The amino acid sequence of <span class="yellow">P. aeruginosa</span> RdgC, a 306-residue protein, shows 46 and 36% identities against those of <span class="yellow">E. coli</span> and <span class="yellow">N. meningitidis</span>, respectively. Despite the importance of DNA-binding ability of RdgC in its biological roles, the structural details of RdgC and the mechanism of its action remain unclear. Here, we report the crystal structure of <span class="yellow">P. aeruginosa</span> RdgC as determined by the multiwavelength anomalous diffraction (MAD) method of X-ray crystallography. It reveals that RdgC dimer is ring-shaped, with a central hole of ∼30 Å diameter. The inside surface around the central hole is rich in conserved, positively charged residues, which are implicated in DNA-binding by mutagenesis. We propose that dsDNA binds to RdgC through this central hole. Our structure provides a solid structural framework for a better understanding of the role of RdgC in homologous recombination.<br><br>MATERIALS AND METHODS
Protein expression and purification
The rdgC gene (PA3263) was amplified by the polymerase chain reaction (PCR) using the genomic DNA of <span class="yellow">P. aeruginosa</span> strain PAO1 as a template. The oligonucleotide primers designed using the published genome sequence (8) were 5′-G GAA TTC CAT ATG TGG TTC CGC AAT CTG CTC G-3′ (forward) and 5′-CCG CCG CTC GAG GAC GCC CTG GGG GAT TTC TTC-3′ (reverse). The bases in bold denote the NdeI and XhoI cleavage sites, respectively. The PCR product was digested with NdeI and XhoI, and was then inserted into the NdeI/XhoI-digested expression vector pET-21a(+) (Novagen). This vector construction adds an eight-residue tag (LEHHHHHH) to the carboxyl terminus of the gene product to facilitate protein purification.
The protein was expressed in <span class="yellow">E. coli</span> C41 (DE3) cells (9). The cells were grown at 37°C up to OD600 of 0.6 in Luria–Bertani medium that contained 50 μg ml−1 ampicillin, and the protein expression was induced by 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG). The cells were allowed to grow at 20°C for 22 h after IPTG induction and were harvested by centrifugation at 4200 g (6000 rev min−1; Sorvall GSA rotor) for 10 min at 4°C. The cell pellet was resuspended in an ice-cold lysis buffer [20 mM Tris-HCl (pH 7.9), 500 mM sodium chloride, 5 mM imidazole] and was then homogenized with an ultrasonic processor. The crude cell extract was centrifuged at 36 000 g (18 000 rev min−1; Hanil Supra 21 K rotor) for 60 min at 4°C, and the recombinant protein in the supernatant fraction was purified in three chromatographic steps.
The first purification step utilized the C-terminal hexahistidine-tag by Ni2+-chelated HiTrap chelating column (GE Healthcare). The eluent was diluted 3-fold with 50 mM Tris-HCl at pH 7.4. The diluted solution was applied to a 20-ml heparin-Sepharose column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol and 1 mM EDTA. The protein was eluted with a linear gradient of 0–2.0 M sodium chloride in the same buffer. The first peak corresponded to high molecular aggregates of RdgC and the second peak to RdgC dimers, which was also consistent with our dynamic light scattering measurements. When the aggregates were not separated from the dimers, the protein readily aggregated within a day. The dimer fractions were concentrated to ∼6 mg ml−1 concentration using an YM10 ultrafiltration membrane (Millipore-Amicon) and further purified by gel filtration on a HiLoad XK 16 Superdex 200 prep-grade column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 200 mM sodium chloride, 1 mM dithiothreitol and 1 mM EDTA. The first peak corresponding to the high molecular aggregates of RdgC was only a minor component, and the second peak corresponding to RdgC dimers was stable for several weeks when frozen at −70°C. The procedure for purifying the SeMet-substituted protein was the same, except for the presence of 10 mM dithiothreitol in all buffers used during purification steps. Dynamic light scattering experiments were performed at 24°C, with the protein (at 2 mg ml−1 concentration) dissolved in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride on a Model DynaPro-801 instrument (Wyatt, Santa Barbara, CA, USA). The protein concentration was estimated by measuring the absorbance at 280 nm, employing the calculated extinction coefficient of 20 910 M−1 cm−1 (SWISS-PROT; http://www.expasy.ch/).<br><br>Mutagenesis and electrophoretic mobility shift assay
Point mutants were prepared using the QuikChange Site-Directed Mutagenesis kit (Stratagene), and the mutations were confirmed by DNA sequencing. The seven mutants R4A, F120A, K146A, R198A, R208A, Q212A and R252A were well expressed and soluble. The three mutants K70A, R81A and R211A were not expressed, while R122A mutant was expressed but was not soluble. Therefore, we mutated Lys70, Arg81, Arg122 and Arg211 into aspartate. The K70D, R81D, R122D and R211D mutants were well expressed and soluble. The soluble mutants were expressed and purified under the conditions identical to those for the wild type, except that the heparin-Sepharose column step was omitted. All of the soluble mutants had similar elution profiles as the wild type upon gel filtration, and the dimer fractions were used for DNA-binding assays. We mixed 5 μl of RdgC (1.1 μg μl−1) and 5 μl of a linear dsDNA with 414 bp (63 ng μl−1), corresponding to the molar ratio of 64:1 for RdgC dimer to dsDNA. The reaction mixtures were incubated for 10 min at room temperature. RdgC–DNA complexes were analyzed by 1.5% (w/v) agarose gel electrophoresis in 1× TAE buffer (45 mM Tris-acetate at pH 8.3, 1 mM EDTA). DNA bands were visualized by ethidium bromide staining. For DNA-binding assays, the protein concentration was measured with a Bio-Rad protein assay kit.<br><br>Crystallization and X-ray data collection
Crystals were grown by the hanging-drop vapor diffusion method at 24°C by mixing equal volumes (2 μl each) of the protein solution (at 19 mg ml−1 concentration in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride) and the reservoir solution. To grow the native crystals, we used a reservoir solution consisting of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate and 1% (w/v) Anapoe®35 as the detergent. Native crystals grew to approximate dimensions of 0.2 mm × 0.1 mm × 0.1 mm within a few days. The SeMet-substituted protein was crystallized under conditions identical to those for the native crystals except for the presence of 10 mM dithiothreitol in the protein solution.
A crystal of the SeMet-substituted protein was dipped into a cryoprotectant solution for a few seconds and was flash-cooled in the cold nitrogen gas stream at 100 K. The cryoprotectant solution consisted of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate, 1% (w/v) Anapoe®35 as the detergent and 10% (v/v) glycerol. A set of Se MAD data was collected at 100 K at three different wavelengths using an Area Detector System Corporation Quantum 210 charge-coupled device detector at the beamline NW12A of Photon Factory, Tsukuba, Japan. The crystal was rotated through a total of 180° with a 1° oscillation range per frame. The raw data were processed and scaled using the program suit HKL2000 (10). The SeMet-substituted crystal belongs to the space group P21212, with unit cell parameters of a = 87.04 Å, b = 115.60 Å and c = 76.07 Å.
The native crystals were soaked for 10 min in 10 mM ethyl mercury thiosalicylate (EMTS) to collect a set of Hg MAD data at four different wavelengths. X-ray diffraction data from the EMTS-derivatized crystal, as well as a native crystal, were collected on a Bruker charge-coupled device detector at the beamline BL-6B of Pohang Light Source, Pohang, Korea. The EMTS-soaked crystal belongs to the space group P21212, with unit cell parameters of a = 87.16 Å, b = 116.02 Å and c = 74.97 Å. The native crystal belongs to the space group P21212, with unit cell parameters of a = 87.49 Å, b = 116.04 Å and c = 76.03 Å. If the presence of a dimeric molecule in the asymmetric unit is assumed, the calculated crystal volume per protein mass (VM) is 2.75 Å3 Da−1 and the solvent content is 55.3%. Supplementary Table 1 summarizes the statistics of X-ray diffraction data collection.<br><br>Structure solution and refinement
Heavy atom sites were located with the program SOLVE (11). Four Se sites from the SeMet-substituted crystal and two Hg sites from the EMTS-soaked crystal were located. Two sets of the initial Se and Hg MAD phases were separately improved using the program RESOLVE (12). Two electron density maps were independently interpreted, and the fragments of the two models were combined into a single model, since the two maps were complementary to each other. Non-crystallographic symmetry (NCS) matrices were found from a partial model built from the initial electron density map, and phases were further improved by the 2-fold NCS averaging, solvent flattening and histogram matching with the program DM (13). The model was built with O (14). The model was refined with the program CNS (15), including the bulk solvent correction. 10% of the data were randomly set aside as the test data for the calculation of Rfree (16). Several rounds of model building, simulated annealing, positional refinement and individual B-factor refinement were performed. Subsequently, this model was used to refine the structure of the native protein. The model has excellent stereochemistry, as evaluated by the program PROCHECK (17). Refinement statistics are summarized in Supplementary Table 1.<br><br>
RESULTS AND DISCUSSION
Model quality and monomer structure
The structure of <span class="yellow">P. aeruginosa</span> RdgC was solved using two sets of MAD data collected from a crystal of the selenomethionine (SeMet)-substituted protein and from a mercury derivative crystal of the native protein (Supplementary Table 1). The model has subsequently been refined using the 20.0–2.50 Å data from a native crystal to crystallographic Rwork and Rfree values of 23.4 and 27.9%, respectively, with no sigma cut-off. The refined model contains 612 residues of the two monomers in the asymmetric unit of the crystal (residues 1–306 for both chains A and B) and 87 water molecules. The C-terminal eight-residue fusion tag has no electron density in both subunits and has not been modeled. Conformations of the two monomers in the asymmetric unit are similar. The root-mean-square (r.m.s.) difference between the two monomers is 0.69 Å for 306 Cα atom pairs, with a maximum deviation of 8.8 Å for the Cα atom of Gly305. The residues giving r.m.s. differences larger than 2.0 Å are 14–16, 94–96, 201–202 and 303–306, which correspond to loops connecting secondary structure elements or the C-terminus. They are located on the surface of the dimer. Subunit A has a better map quality than subunit B, and thus has a lower mean B-factor than subunit B (40.3 Å2 versus. 56.1 Å2 for all 4788 atoms) (Supplementary Figure 1). Therefore, subunit A is chosen for discussions, unless otherwise stated.
The <span class="yellow">P. aeruginosa</span> RdgC monomer is J-shaped and has approximate dimensions of 72 Å × 60 Å × 40 Å (Figure 1). It can be divided into three structural domains: center domain, tip domain and base domain. Center domain (residues 1–73 and 121–167) includes a six-stranded anti-parallel β-sheet with two flanking α helices (α1 and α4). The helix α1 runs nearly parallel to the two outermost strands β2 and β3, which in turn lie adjacent to β4. Strands β2 and β3 are separated by a loop. The helix α4 is nearly orthogonal to the β-strands. Tip domain (residues 74–120) contains two α helices (α2 and α3) and is inserted between strands β4 and β5 of the center domain. It sticks out from the center domain. The two connections between the center domain and the tip domain are rich in conserved residues, hinting at an important functional role of these joints. They could act as a hinge for a possible conformational change. The C-terminal region of the polypeptide chain is folded into the base domain (residues 168–306), which consists of a five-stranded anti-parallel β-sheet with four α-helices, three of which (α5, α6 and α8) cover one side of the β-sheet. The long helix α8 is slightly bent.<br><br>
Structural similarity searches
When we searched for structural similarity against the Protein Data Bank database using the program DALI (18), the <span class="yellow">P. aeruginosa</span> RdgC monomer showed no significant similarity with a Z score above 5. Therefore, we conclude that the overall fold of the RdgC polypeptide chain is unique. When we elaborated the structural comparisons with individual domains of <span class="yellow">P. aeruginosa</span> RdgC, only the center domain gave a Z score above 5.
Using the center domain (residues 1–73 and 121–167) alone, the highest structural similarity is found with the <span class="yellow">human</span> β2-adaptin appendage domain (PDB code 1E42, Z score = 7.1, r.m.s. deviation = 2.5 Å for 85 structurally aligned residues, residues 1–12, 14–30, 44–47, 58–72, 122–133 and 135–160 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 838–841, 848–880, 885–906 and 912–967 of the <span class="yellow">human</span> β2-adaptin appendage domain chain A) (19). There does not appear to be any functional relatedness between these two proteins. The next highest similarity is found with the <span class="yellow">yeast</span> TATA-box-binding protein (PDB code 1YTB, Z score = 5.7, r.m.s. deviation = 2.6 Å for 79 structurally aligned residues, residues 4–12, 14–19, 21–27, 56–70, 124–132 and 134–166 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 69–96, 100–126 and 132–155 of TATA-box-binding protein, N-terminal half of chain A) (20). The TATA-box-binding protein is composed of a ten-stranded antiparallel β-sheet with four flanking α-helices on the convex side of the β-sheet (20). It has an internal quasi 2-fold symmetry, and its β-strands are involved in DNA binding. There is, however, no structural similarity of functional significance, because the two center domains of the RdgC dimer are well separated and do not associate into a single unit (Figure 1C and D).
Using the tip domain (residues 74–120) alone, the ATPase domain of <span class="blue">bovine</span> Hsc70 chaperone shows highest structural similarity (PDB code 1BA1, Z score = 4.8, r.m.s. deviation = 1.7 Å for 45 structurally aligned residues, residues 76–120 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 230–253 and 256–276 of the Hsc70 ATPase domain) (21). There does not seem to be any significant functional relationship between these two proteins. The α-helix-loop-α-helix structure of the tip domain is reminiscent of the helix–hairpin–helix (HhH) DNA-binding motif that is present in a number of DNA repair enzymes (22–25). The HhH motifs are characterized by the conserved sequence motif hxxhxGhGxxxAxxxhh, where h is any hydrophobic residue (VILMWFYA) and x any residue (23). Two conserved glycines allow a tight turn between two α-helices. However, the tip domain of <span class="yellow">P. aeruginosa</span> RdgC protein does not have such a sequence motif, and the loop in the RdgC tip domain is much more extended than that of the HhH motifs. Therefore, the RdgC tip domain is uniquely folded.
With the base domain (residues 168–306) alone, the highest similarity is found with <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase (unpublished deposition; PDB code 2H9C, Z score = 4.0, r.m.s. deviation = 4.8 Å for 51 structurally aligned residues, residues 186–191, 237–240, 257–272 and 274–298 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 36–41, 50–53, 55–66 and 68–96 of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase chain A). There does not appear to be any functional relationship between the two proteins, and the base domain of RdgC is uniquely folded. The 101-residue chain of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase is folded into three α-helices, two of which overlap with two α-helices (α2 and α3) of the base domain of RdgC.<br><br>Ring-shaped dimer structure and inter-subunit interface
Our structure reveals that <span class="yellow">P. aeruginosa</span> RdgC forms a ring-shaped dimer of approximate 2-fold symmetry in the crystal (Figure 1). This observation is consistent with the results of previous gel filtration and sedimentation equilibrium experiments, which suggested the presence of dimeric species of <span class="yellow">E. coli</span> RdgC in solution (1,2). Depending on the concentration, RdgC could also exist in solution as monomers, tetramers and higher oligomers (1). In the crystal lattice, no monomeric, trimeric or tetrameric species exist. Our structure unequivocally rules out the monomer–trimer model, which fits the sedimentation equilibrium data approximately, as well as the monomer–dimer–tetramer model (1). The RdgC dimer has approximate dimensions of 80 Å × 60 Å × 40 Å, with a central hole of ∼30 Å diameter.
Only a moderate amount of surface area is buried upon the formation of the RdgC dimer. The monomer–monomer interface buries a solvent-accessible surface area of ∼1360 Å2 per monomer, corresponding to ∼8% of the total surface area of the monomer (Protein–Protein Interaction Server at http://www.biochem.ucl.ac.uk/bsm/PP/server/). Polar and nonpolar atoms in interface are 44 and 56%, respectively. The interface can be divided into two kinds: ‘interface A’ involving both the tip domains and center domains, and ‘interface B’ involving the base domains (Figure 2A). These two interfaces have significantly different characters.<br><br>The interface A is formed largely by the crossing over of the two tip domains in the dimer (Figure 1). This interface covers a surface area of ∼565 Å2 and is largely nonpolar, with the polar and nonpolar atoms in the interface being 37.5 and 2.5%, respectively. Phe120 contributes considerably to the nonpolar character of the interface between the tip domains, accounting for ∼150 Å2 of the interface area. Phe120 is part of a highly conserved sequence segment and lies at the boundary between the tip domain and the center domain. The aromatic ring of Phe120′ inserts into a hydrophobic pocket, which is formed by the hydrophobic residues Ile74, Leu75, Pro76, Val79, Leu115, Ala119 and Phe120 of the neighboring monomer (Figure 2B). The primed residue belongs to the adjacent subunit. These residues are well conserved in the RdgC family (indicated by green triangles below the aligned sequences in Figure 3). As these hydrophobic residues are located near the molecular 2-fold axis, the two clusters of hydrophobic side chains are merged into a single, large cluster in the dimer (Figure 2B). Of the 16 hydrogen bonds that exist between the two subunits, only one is made in this interface between Arg118 and Arg118′.<br><br>The interface B is contributed to exclusively by the two base domains. This interface covers a surface area of ∼795 Å2. The polar and nonpolar atoms in this interface are 50% each. The two β-sheets of the two base domains merge into a single ten-stranded antiparallel β-sheet in this interface (Figure 2A). Thus, 15 of 16 inter-subunit hydrogen bonds are clustered in this interface. The hydrogen bond networks can be divided into two types. The first type involves the main chain atoms of Val206, Arg208 and Lys210 in strand β9; this strand associates with the equivalent strand from the other subunit in an antiparallel manner (Figure 2C). The other type involves both the side chain and main chain atoms. The side chain of Arg211′ interacts with the backbone of Gly204, while the side chain of Lys227 hydrogen bonds to those of both Gln212′ and Glu218 (Figure 2D). In addition, the side chain of Gln212′ interacts with that of Asp199 and the backbone of Arg211′ (Figure 2D). Arg211, Gln212, Glu218 and Lys227 are highly conserved among the RdgC family members. The residues participating in the hydrogen bond networks are indicated by red circles below the aligned sequences in Figure 3. The observation that conserved residues play important roles in forming the dimer implies that dimerization of RdgC is critical for its function.
If one or both of interfaces A and B are to be broken for functional reasons such as for allowing dsDNA to enter into the central hole of the RdgC dimer, we can conceive of three scenarios. The first possibility would be a simultaneous breakage of the two interfaces A and B. The second would be the sequential disruption of the interfaces A and B. The third would be the disruption of only one of the two interfaces A and B. Sedimentation equilibrium indicated the presence of monomers in solution (1), which is consistent with the first two scenarios. If the third scenario holds, RdgC dimers could possibly exist in two conformations in solution, i.e. a closed form as observed in the present crystal structure, and an open form, in which one of the two interfaces A and B is broken.<br><br>A model for DNA binding by RdgC and DNA-binding assay
The net charge of <span class="yellow">P. aeruginosa</span> RdgC is highly negative, since each monomer contains 12 aspartate, 13 glutamate, 10 lysine, 9 arginine and 2 histidine residues. The net charge of a dimer would be −12 (or −8), if we assume none (or all) of histidines to be positively charged. The electrostatic potential at the molecular surface of RdgC dimer shows that the charge distribution on the protein surface is highly nonuniform (Figure 4A). Since the inner surface of the RdgC dimer is rich in conserved, positively charged residues and the diameter of the central hole (∼30 Å) is approximately similar to that of other toroidal DNA-binding proteins (26), it appears likely that RdgC dimers bind dsDNA through the central hole. Despite extensive co-crystallization and soaking trials with linear dsDNAs of different lengths (14-mer, 16-mer and 18-mer), we could not locate the bound DNA experimentally. We attribute this difficulty to the nature of sequence-nonspecific DNA binding by RdgC. Therefore, we built a crude model of a DNA complex of the RdgC dimer by fitting dsDNA into the central hole (Figure 4B). This model is supported by the DNA-binding assays coupled with mutagenesis, as described below. It is also consistent with the suggested binding site size of ∼16 nt for the <span class="yellow">E. coli</span> RdgC dimer (1) and the approximate thickness (∼40 Å) of the <span class="yellow">P. aeruginosa</span> RdgC dimer.<br><br>When we performed electrophoretic mobility shift assay for dsDNA binding, <span class="yellow">P. aeruginosa</span> RdgC did not require Mg2+ ions (Supplementary Figure 2), unlike <span class="yellow">Deinococcus radiodurans</span> RecR, which required high concentrations of Mg2+ ions (27). ATP was not required for dsDNA binding by <span class="yellow">P. aeruginosa</span> RdgC, either (Supplementary Figure 2). This is consistent with an apparent lack of an ATP-binding pocket in the RdgC dimer structure.
In order to identify the residues that are important for dsDNA binding, we carried out mutational analyses. The residues for mutagenesis were selected on the basis of the modeled complex between RdgC dimer and dsDNA, together with sequence conservation (Figure 4C). We prepared eleven point mutants (R4A, K70D, R81D, F120A, R122D, K146A, R198A, R208A, R211D, Q212A and R252A). These eleven mutated residues are well conserved among RdgC proteins. Eight mutants (R4A, K70D, R81D, R122D, K146A, R198A, R208A and R252A) had a lower affinity for dsDNA than the wild-type RdgC (Figure 4D). These eight mutated residues are indicated by blue squares below the aligned sequences in Figure 3. This result indicates that dsDNA binds to the inside surface around the central hole of the RdgC dimer, since the mutated residues are located on the inner surface of the RdgC dimer. It is worth mentioning that a single mutation introduces two alterations per RdgC dimer.
The three mutants (F120A, R211D and Q212A) were prepared to investigate whether dimerization of RdgC is crucial for dsDNA binding. Phe120 is a key residue located in the inter-subunit interface A (Figure 2B), while Arg211 makes hydrogen bonds in the interface B (Figure 2D) and Gln212 is involved in both intra- and inter-subunit interactions (Figure 2D). dsDNA binding by the F120A mutant is considerably weakened, compared to either the R211D or Q212A mutant (Figure 4D). This result suggests that destabilization of the ring-shaped architecture of the RdgC dimer is deleterious for tight dsDNA binding and that interface A is probably more important than interface B for dsDNA binding. dsDNA-binding affinity of the Q212A mutant was lower than the wild type (Figure 4D), whereas that of the R211D mutant was comparable to the wild type (Figure 4D). This result suggests that disruption of the inter-subunit interface B is also detrimental for dsDNA binding but to a smaller extent than disruption of the interface A. It further suggests that the side chain of Arg211 is not involved in recognizing dsDNA. It is interesting that the DNA complex for wild type forms a distinct band (lane 3 in Figure 4D), as opposed to a more diffuse band as one would expect if the complexes contained different numbers of RdgC dimers. Assuming a binding site size of ∼16 nt (1), the 414-bp duplex DNA would bind ∼25 RdgC dimers if it is fully saturated.<br><br>RdgC is unique among ring-shaped DNA-binding proteins
<span class="yellow">Pseudomonas aeruginosa</span> RdgC adds a unique member to the repertoire of toroidal DNA-binding proteins (26) such as <span class="yellow">D. radiodurans</span> RecR (27) and DNA polymerase processivity factors, including the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29) and PCNA of eukaryotic DNA polymerase δ (30,31) (Figure 5). These proteins do not have an overall sequence similarity among themselves, and their quaternary structures also vary. <span class="yellow">P. aeruginosa</span> RdgC and <span class="yellow">E. coli</span> β clamp are homodimers, T4 gp45 and eukaryotic PCNAs are homotrimers and <span class="yellow">D. radiodurans</span> RecR is a homotetramer.<br><br>In <span class="yellow">P. aeruginosa</span> RdgC, the inner surface of the ring-shaped dimer is largely formed by β-strands, while the outer surface of the ring is mostly formed by α-helices (Figure 5). The predicted DNA-binding site of <span class="yellow">P. aeruginosa</span> RdgC dimer is primarily formed by β-strands and loops. In contrast, in other ring-shaped DNA-binding proteins such as the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29), PCNA of eukaryotic DNA polymerases δ (30,31) and <span class="yellow">D. radiodurans</span> RecR (27), α-helices are located toward the center of the ring and are surrounded by β-strands in the outermost side of the ring (Figure 5). Despite a superficial resemblance of the overall ring-shaped architecture of the RdgC dimer with those of other toroidal DNA-binding proteins, the internal arrangement of the secondary structure elements in the RdgC dimer is strikingly different from those of other toroidal DNA-binding proteins. Thus, RdgC is unique among ring-shaped DNA-binding proteins.
A number of other proteins involved in DNA metabolism also adopt a ring-shaped structure despite diversity in the molecular functions (26). They include NAD+-dependent DNA ligase (25), λ exonuclease (32), hexameric helicases, topoisomerases, the trp RNA-binding attenuation protein, the <span class="yellow">bacteriophage</span> head-to-tail connector and translin (26). There are also many examples of the proteins in DNA recombination pathways that assemble into multi-subunit toroids containing a central channel (26). Electron microscopy showed that the <span class="yellow">E. coli</span> RuvB protein, in the presence of ATP, forms a dodecamer on double-stranded DNA in which two stacked hexameric rings encircle the DNA and are oriented in opposite directions with D6 symmetry (33). The <span class="yellow">human</span> RAD52 protein was found to assemble into heptameric rings with a large funnel-shaped channel, 40–60 Å in diameter (34). The <span class="yellow">human</span> DMC1 protein has so far been detected only as an octameric ring (35).<br><br>Inhibition mechanism of RecA function by RdgC
RdgC inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). It was observed that the effects of RdgC on RecA-promoted DNA strand exchange were much more pronounced than the effects of RdgC on other RecA functions, when RdgC was added to the reaction late (1). By examining the effects of the DinI protein on RdgC-mediated inhibition of RecA activities, it was concluded that the potent effect of RdgC in the inhibition of strand exchange did not primarily reflect a displacement of RecA in the nucleoprotein filament by RdgC, but instead a binding of RdgC to the dsDNA substrate so as to make it unavailable to the filaments for strand exchange (1).
Our work contributes to a better understanding of the inhibition mechanism of RecA function by RdgC. The ring-shaped architecture with a central hole that is suitable for the binding of dsDNA lends RdgC with a capacity to hold dsDNA very tightly. Therefore, we suggest that RdgC dimers probably function as a tight dsDNA gripper and physically block access to the target dsDNA by RecA filaments, because tight encircling of dsDNA by RdgC dimers, possibly as a linear aggregate, would prevent the two strands from dissociating. However, it remains unclear how the closed-ring structure of the RdgC dimer might allow binding of a large number of RdgC dimers on the duplex DNA. It may be conceivable that each RdgC dimer can open temporarily at one or both of its two inter-subunit interfaces so that dsDNA can enter the central hole.
RdgC was at its highest level during exponential phase, reaching its maximum of ∼1000 dimers per <span class="yellow">E. coli</span> cell. Its level decreased sharply to ∼50 dimers per cell in stationary phase (2). Under normal cellular conditions, RecA is maintained at ∼1000 molecules per cell (http://www.els.net/). Following LexA repressor cleavage, the level of RecA protein in the cell increases by as much as 20-fold. RdgC inhibition of the RecA function by a simple competition mechanism at the cellular concentration of RdgC clearly requires a much higher affinity for DNA by RdgC than RecA. Possible protein–protein interactions between RdgC dimers might have a profound effect on the affinity for DNA, when a large number of RdgC dimers are bound on a duplex DNA. If only one of the two inter-subunit interfaces can open up and the 2-fold symmetry axes of the RdgC dimers are misaligned sufficiently, the escape of DNA from a linear array of interacting RdgC dimers would be extremely difficult. Alternatively, both inter-subunit interfaces of each dimer may open up simultaneously. A line of indirect evidence for the possible protein–protein interactions between RdgC dimers is provided by the cooperative binding of RdgC to DNA (7). Cooperativity of RdgC binding may be caused by local and distant protein–protein interactions (7). A kind of side-by-side interaction between two adjacent molecules of RdgC dimer is present in the crystal, burying an accessible surface area of ∼750 Å2 at the interface. The interaction involves mostly the less polar side surface of the dimer (left side of Figure 4A), with 2-fold symmetry axes of the two interacting RdgC molecules making an angle of ∼90° when viewed down the central holes and ∼15° when viewed perpendicular to the holes. This may explain the observed DNA bending by RdgC, which was speculated to be a necessary step in the working mechanism of RdgC, if one can assume that the strong protein–protein interactions in the crystal were relevant for protein–protein interactions in solution (7).
RdgC protein was shown to have a potent effect on RecA protein-promoted DNA strand exchange in <span class="yellow">E. coli</span> (1). In a series of experiments, RdgC was added after RecA filaments had formed on the ssDNA and 5 min after the linear dsDNA was added. With 3 μM RecA, a sharp reduction in DNA strand exchange products was seen with 0.4 μM RdgC, and the generation of products was abolished at 0.8 μM RdgC (1). This suggests that the binding affinity of RdgC to dsDNA is considerably higher than that of RecA. The high affinity of RdgC could be explained by its ring shape and the suggested mode of DNA binding. Tetrameric species of <span class="yellow">E. coli</span> RdgC, which started to form at around 2 µM concentrations, were speculated to be the species with the most potent inhibitory effect, possibly by having a higher affinity for DNA (1). Higher affinity for DNA of the tetrameric species could be understood, if the formation of a tetramer in solution involves a protein–protein interaction similar to the side-by-side interaction between two adjacent RdgC dimers in the crystal, although there is no experimental evidence. Obviously, there are still many open questions that need to be addressed in future biochemical experiments.<br><br>
SUMMARY
The crystal structure of RdgC, a recombination-associated DNA-binding protein, determined in this study reveals that the polypeptide chain is uniquely folded and two monomers associate to form a ring-shaped dimer with a central hole of ∼30 Å diameter. The inner surface around the central hole is rich in positively charged residues. These structural features suggest that dsDNA runs through the central hole when it binds to RdgC dimers. This idea is supported by our mutational studies, which indicate that the conserved, positively charged residues around the central hole are important for dsDNA binding. The toroidal architecture of the RdgC dimer provides a sound framework for a better understanding of its role in homologous recombination.
Coordinates
Atomic coordinates and the structure factor data have been deposited in the Protein Data Bank (accession code 2OWY).<br><br>
SUPPLEMENTARY DATA
Supplementary Data are available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA2435603</h3>Potentiation of Nerve Growth Factor-Induced Neurite Outgrowth by Fluvoxamine: Role of Sigma-1 Receptors, IP3 Receptors and Cellular Signaling Pathways
Abstract
Background
Selective serotonin reuptake inhibitors (SSRIs) have been widely used and are a major therapeutic advance in psychopharmacology. However, their pharmacology is quite heterogeneous. The SSRI fluvoxamine, with sigma-1 receptor agonism, is shown to potentiate nerve-growth factor (NGF)-induced neurite outgrowth in PC 12 cells. However, the precise cellular and molecular mechanisms underlying potentiation by fluvoxamine are not fully understood. In this study, we examined the roles of cellular signaling pathways in the potentiation of NGF-induced neurite outgrowth by fluvoxamine and sigma-1 receptor agonists.<br><br>Methods and Findings
The effects of three SSRIs (fluvoxamine, sertraline, paroxetine) and three sigma-1 receptor agonists (SA4503, 4-phenyl-1-(4-phenylbutyl) piperidine (PPBP), and dehydroepiandrosterone (DHEA)-sulfate) on NGF-induced neurite outgrowth in PC12 cells were examined. Also examined were the effects of the sigma-1 receptor antagonist NE-100, inositol 1,4,5-triphosphate (IP3) receptor antagonist, and specific inhibitors of signaling pathways in the potentiation of NGF-induced neurite outgrowth by selective sigma-1 receptor agonist SA4503. Fluvoxamine (but not sertraline or paroxetine) and the sigma-1 receptor agonists SA4503, PPBP, and DHEA-sulfate significantly potentiated NGF-induced neurite outgrowth in PC12 cells in a concentration-dependent manner. The potentiation by fluvoxamine and the three sigma-1 receptor agonists was blocked by co-administration of the selective sigma-1 receptor antagonist NE-100, suggesting that sigma-1 receptors play a role in blocking the enhancement of NGF-induced neurite outgrowth. Moreover, the potentiation by SA4503 was blocked by co-administration of the IP3 receptor antagonist xestospongin C. In addition, the specific inhibitors of phospholipase C (PLC-γ), phosphatidylinositol 3-kinase (PI3K), p38MAPK, c-Jun N-terminal kinase (JNK), and the Ras/Raf/mitogen-activated protein kinase (MAPK) signaling pathways blocked the potentiation of NGF-induced neurite outgrowth by SA4503.<br><br>Conclusion
These findings suggest that stimulation of sigma-1 receptors and subsequent interaction with IP3 receptors, PLC-γ, PI3K, p38MAPK, JNK, and the Ras/Raf/MAPK signaling pathways are involved in the mechanisms of action of sigma-1 receptor agonists such as fluvoxamine and SA4503.<br><br><br><br>Introduction
Selective serotonin (5-HT; 5-hydroxytryptamine) reuptake inhibitors (SSRIs) have emerged as a major therapeutic advance in psychopharmacology. SSRIs are the treatment of choice for many indications, including major depressive disorder, dysthymia, panic disorder, obsessive-compulsive disorder, eating disorders, and premenstrual dysphoric disorder. In contrast, it is well known that their pharmacology is quite heterogeneous, although all of them block 5-HT transporters, thus increasing 5-HT levels throughout the central nervous system (CNS) [1]–[9].
Accumulating evidence suggests that sigma-1 receptors, which are intracellular endoplasmic reticulum (ER) proteins, are involved in both the neuroplasticity and pathophysiology of neuropsychiatric diseases such as major depressive disorder, anxiety, schizophrenia, and Alzheimer's disease [10]–[18]. Previously, we reported that some SSRIs possess high to moderate affinities for sigma-1 receptors in the <span class="yellow">rat</span> brain. The rank order of SSRIs affinities for sigma-1 receptors is fluvoxamine (Ki = 36 nM)>sertraline (Ki = 57 nM)>>paroxetine (Ki = 1893 nM) [19]. Recently, we reported that fluvoxamine, but not paroxetine, significantly ameliorated cognitive deficits in <span class="yellow">mice</span> after repeated phencyclidine administration, and that the effects of fluvoxamine were antagonized by co-administration of the selective sigma-1 receptor antagonist NE-100 [20], suggesting that sigma-1 receptors are involved in the mechanism of action of fluvoxamine [9]. Interestingly, it has been demonstrated that sigma-1 receptor agonists including fluvoxamine could potentiate nerve growth factor (NGF)-induced neurite outgrowth in PC12 cells, and that NE-100 blocked the potentiation by sigma-1 receptor agonists, suggesting sigma-1 receptors are involved in neuroplasticity [21]. However, the precise cellular mechanisms underlying the potentiation by sigma-1 receptor agonists are not fully understood [13], [21].
It is therefore of great interest to study the precise cellular mechanisms underlying the enhancement by fluvoxamine on NGF-induced neurite sprouting in PC12 cells. In the present study, we examined the effects of three SSRIs (fluvoxamine, sertraline, paroxetine), as well as the effects of a sigma-1 receptor agonist (4-phenyl-1-(4-phenylbutyl) piperidine (PPBP), dehydroepiandrosterone-sulphate (DHEA)-sulfate) [9], [22]–[28] and the selective sigma-1 receptor agonist SA4503 [29], [30], on NGF-induced neurite outgrowth in PC12 cells. Furthermore, it is also known that sigma-1 receptors have been shown to interact with IP3 receptors (17,18). Therefore, we examined the effects of NE-100 and xestospongin C (a selective inositol 1,4,5-triphosphate (IP3) receptor antagonist) [31] in order to investigate the roles of sigma-1 receptors and IP3 receptors in the mechanisms underlying the enhancement of NGF-induced neurite outgrowth by SA4503. Moreover, we examined the effects of specific inhibitors of several cellular signaling targets on the enhancement of NGF-induced neurite outgrowth by SA4503, since several signal transduction molecules have been implicated in NGF-induced neurite outgrowth [32].<br><br>Materials and Methods
Drugs
The drugs were obtained from the following sources: fluvoxamine maleate (Solvay Seiyaku K.K., Tokyo, Japan); paroxetine hydrochloride, dehydroepiandosterone-sulfate (DHEA-sulfate), LY294002 (Sigma-Aldrich, St Louis, MO, USA); sertraline (Toronto Research Chemicals Inc., North York, ON, Canada); SA4503 (M's Science Corporation, Kobe, Japan); NGF (Promega, Madison, WI); xestospongin C, lovastatin, PD98059, GW5074, SB203580, MEK 1/2 inhibitor (SL327), and SP600125 (Calbiochem-Novabiochem, San Diego, CA). The selective sigma-1 receptor antagonists NE-100 and 4-phenyl-1-(4-phenylbutyl) piperidine (PPBP) were synthesized in our laboratory. Other drugs were purchased from commercial sources.<br><br>Cell culture
PC12 sells (RIKEN Cell Bank, Tsukuba, Japan) were cultured at 37°C, 5% CO2 with Dulbecco's modified Eagle's medium (DMEM) supplemented with 5% heat-inactivated fetal <span class="blue">bovine</span> serum (FBS), 10% heat-inactivated <span class="yellow">horse</span> serum, and 1% penicillin. The medium was changed two or three times a week. PC12 cells were plated onto 24-well tissue culture plates coated with poly-D-lysine/laminin. Cells were plated at relatively low density (0.25×104 cells/cm2) in DMEM medium containing 0.5% FBS, 1% penicillin streptomycin. Medium containing a minimal level of serum (0.5% FBS) was used, since serum is known to contain steroid hormones that bind to sigma-1 receptors [21]. First, we examined the optimal concentration of NGF for NGF-induced neurite outgrowth in PC12 cells. NGF (2.5, 5, 10, 20, 40 ng/ml) increased the number of cells with neurite outgrowth in PC12 cells in a concentration-dependent manner (Figure 1). In the subsequent studies, 2.5 ng/ml of NGF was used to study the potentiating effects of sigma-1 receptor agonists on NGF-induced neurite outgrowth. Twenty-four hours after plating, the medium was replaced with DMEM medium containing 0.5% FBS and 1% penicillin streptomycin with NGF (2.5 ng/ml) with or without several drugs.<br><br>Quantification of neurite sprouting
Five days after incubation with NGF (2.5 ng/ml) with or without the several drugs, morphometric analysis was performed on digitized images of live cells taken under phase-contrast illumination with an inverted microscope linked to a camera. Images of three fields per well were taken, with an average of 100 cells per field. Differentiated cells were counted by visual examination of the field; only cells that had at least one neurite with a length equal to the cell body diameter were counted, and were then expressed as a percentage of the total cells in the field. The counting was performed in a blinded manner.<br><br>Immunocytochemistry
Cells were fixed for 30 min at room temperature with 4% paraformaldehyde then permeabilized with 0.2% Triton and blocked with 1.5% normal <span class="yellow">goat</span> serum, 0.1% <span class="blue">bovine</span> serum albumin (BSA) in 0.1 M phosphate-buffer saline for 1 h to reduce nonspecific binding. Cells were incubated overnight at 4°C with anti-microtubule-associated protein 2 (MAP-2) antibodies (1∶1000 dilution in blocking solution, Chemicon International, Temecula, CA, USA). The immunolabeling was visualized with secondary antibodies conjugated to Alexa-488 (1∶1000; Invitrogen, Carlsbad, CA, USA). MAP-2 immuncytochemistry was visualized with a fluorescence microscope (Axiovert 200, Carl Zeiss, Oberkocken, Germany).<br><br>Statistical analysis
Data are expressed as means±S.E.M. Statistical analysis was performed by using one-way analysis of variance (ANOVA) and the post hoc Bonferroni/Dunn test. P values less than 0.05 were considered statistically significant.<br><br>
Results
Effects of SSRIs on NGF-induced neurite outgrowth
Fluvoxamine (0.1, 1.0, or 10 µM) significantly increased the number of cells with neurite outgrowth by NGF (2.5 ng/ml) in PC12 cells, in a concentration-dependent manner (Figure 2 and 4). In contrast, sertraline (0.1, 1.0, or 10 µM) and paroxetine (0.1, 1.0, or 10 µM) did not increase the number of cells with neurite outgrowth by NGF (2.5 ng/ml). A higher concentration (10 µM) of paroxetine and sertraline significantly decreased the number of cells with NGF-induced neurite outgrowth, suggesting the cellular toxicity of these drugs in PC12 cells (Figure 2).<br><br>Role of sigma-1 receptors in the potentiation of NGF-induced neurite outgrowth by fluvoxamine and SA4503
The selective and potent sigma-1 receptor agonist SA4503 (0.01, 0.1, or 1.0 µM) significantly increased the number of cells with neurite outgrowth by NGF (2.5 ng/ml) in PC12 cells, in a concentration-dependent manner (Figure 3 and 4). In addition, other sigma-1 receptor agonists–PPBP (0.1, 1.0, or 10 µM) [22]–[26] and DHEA-sulphate (0.1, 1.0, or 10 µM) [9], [27], [28]–significantly increased the number of such cells, also in a concentration-dependent manner (Figure 3). In the absence of NGF, sigma-1 receptor agonists did not produce neurite outgrowth in PC12 cells (data not shown).
To investigate the role of sigma-1 receptors, we examined the effects of NE-100 (a selective sigma-1 receptor antagonist) on the potentiation of NGF-induced neurite outgrowth by fluvoxamine, SA4503, PPBP, and DHEA-sulphate. Co-administration of NE-100 (1.0 µM) significantly blocked such potentiation by fluvoxamine (10 µM), SA4503 (1.0 µM), PPBP (10 µM), or DHEA-sulphate (10 µM) (Figure 2 and 3). Furthermore, administration of NE-100 (1.0 µM) alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 2).<br><br>Role of IP3 receptors in the potentiation of NGF-induced neurite outgrowth by SA4503
Sigma-1 receptors have been shown to interact with IP3 receptors (17,18,33–35). To investigate the role of IP3 receptors in the effects of SA4503 on NGF-induced neurite outgrowth, we examined the effects of xestospongin C (a selective, reversible, and membrane-permeable inhibitor of IP3 receptors) [31] on the effects of SA4503 on NGF-induced neurite outgrowth. Co-administration of xestospongin C (1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 5). Furthermore, administration of xestospongin C (1.0 µM) alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 5).<br><br>Role of signaling molecules proximal to TrkA in the potentiation of NGF-induced neurite outgrowth by SA4503
Next, we examined the effects of the specific inhibitors of PLC-γ, PI3K, p38 MAPK, and c-Jun N-terminal kinase (JNK), since these signaling molecules are activated upon the addition of NGF [36]. The PLC-γ inhibitor (U73122; 1.0 µM), PI3K inhibitor (LY294002; 1.0 µM), p38 MAPK inhibitor (SB203580; 1.0 µM), and JNK inhibitor (SP600125; 1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 6). In contrast, these inhibitors alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 6).<br><br>Role of Raf/Ras/ERK/MAPK pathway in the potentiation of NGF-induced neurite outgrowth by SA4503
The Raf/Ras/ERK/MAPK pathway is known to be involved in NGF-induced outgrowth [32]. Therefore, we examined the effects of this pathway's specific inhibitors. The Ras inhibitor (GW5074; 1.0 µM), Raf inhibitor (lovastatin; 1.0 µM), MEK1/2 inhibitor (SL327; 1.0 µM), and MAPK inhibitor (PD98059; 1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 7). In contrast, these inhibitors alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 7).<br><br>
Discussion
In the present study, we found that fluvoxamine (but not sertraline or paroxetine) and the sigma-1 receptor agonists (SA4503, PPBP, DHEA-sulfate) could potentiate NGF-induced neurite outgrowth in PC12 cells, and that the effects of these drugs were blocked by co-incubation with the selective sigma-1 receptor antagonist NE-100. These findings suggest that agonism at sigma-1 receptors for these drugs is involved in the mechanisms underlying the drugs' potentiation of NGF-induced neurite outgrowth.
As shown in Figure 1, NGF increased the number of cells with neurite outgrowth in PC12, in a concentration dependent manner. To examine whether or not the NGF levels are altered by incubation with sigma-1 receptor agonists, we measured the levels of NGF in PC12 cells with or without sigma-1 receptor agonist SA4503. No differences of NGF levels were shown in the between control group and SA4503-treated group (data not shown). It is, therefore, unlikely that the potentaition of NGF-induced neurite outgrowth by sigma-1 receptor agonists might be due to increased levels of NGF.
Unlike fluvoxamine, sertraline, which has a moderate affinity for sigma-1 receptors, did not alter NGF-induced neurite outgrowth. The reasons underlying this discrepancy between these two SSRIs are currently unclear. One possibility may involve the difference in pharmacological actions (agonist vs. antagonist) between these SSRIs at sigma-1 receptors. Interestingly, we recently found that, in a novel object recognition test, phencyclidine-induced cognitive deficits could be significantly improved by subsequent subchronic (14 days) administration of fluvoxamine, but not of sertraline, suggesting that sigma-1 receptor agonism is involved in fluvoxamine's mechanism of action [9], Ishima et al., submitted. Taken together, these findings suggest that fluvoxamine and sertraline may function as an agonist and an antagonist at sigma-1 receptors, respectively, although further study is necessary. Another possibility may be that other pharmacological activities of sertraline mask the effects of sigma-1 receptor agonism. In this study, we also found that high concentration (10 µM) of paroxetine and sertraline, but not fluvoxamine, showed cytotoxicity in PC12 cells, suggesting that fluvoxamine may be a safe drug than paroxetine and sertraline.
Sigma-1 receptors have been shown to affect intracellular Ca2+ signaling, although the precise molecular and cellular mechanisms underlying this effect are unknown. Sigma-1 receptors bind to IP3 receptors in ER, and sigma-1 receptors regulate Ca2+ release from intracellular Ca2+ storage sites [12]. Very recently, Hayashi and Su [17] reported that sigma-1 receptors function as novel ligand-operated chaperones that specifically target mitochondrion-associated ER membrane. Furthermore, sigma-1 receptors form Ca2+-sensitive chaperone machinery with another chaperone, BiP, and prolong Ca2+ signaling from ER into mitochondria by stabilizing IP3 receptors at mitochondrion-associated ER membrane [17]. In this study, we found that the IP3 receptor antagonist xestospongin C significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503, suggesting the role of IP3 receptors on sigma-1 receptor-mediated potentiation of NGF-induced neurite outgrowth. Therefore, it is likely that stimulation at sigma-1 receptors by sigma-1 receptor agonists and subsequent interaction with IP3 receptors are involved in the mechanism underlying the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists.
NGF binds to the high-affinity tyrosine receptor TrkA, initiating several signaling pathways affecting both morphological and transcriptional targets [32], [36], [37]. The signaling molecules, including PLC-γ, PI3K, p38 MAPK, and JNK, are activated upon the addition of NGF [38]. PLC-γ catalyzes the hydrolysis of phosphatidylinositol-4,5-bisphosphate (PIP2) to diacylglycerol (DAG) and inositol triphosphate (IP3). DAG activates protein kinase C, and IP3 promotes transient release of Ca2+ from the ER [39]. The pathway via PLC-γ is responsible for NGF-induced cell differentiation [40] and neurite outgrowth [41]. Furthermore, stimulation of PI3K is reported to be involved in the promotion of neurite outgrowth in PC12 cells [42]. In this study, we found that the PLC-γ inhibitor U73122 and the PI3K inhibitor LY294002 significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503. Moreover, we found that both the p38MAPK inhibitor SB203580 and the JNK inhibitor SP600125 significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503. In addition, it is also interesting that neurite outgrowth induced by low concentration (2.5 ng/ml) of NGF was not blocked by these inihibitors, consistent with a previous report [43], [44], suggesting the existence of novel pathway(s) for NGF-induced neurite outgrowth. These findings suggest that the PLC-γ, PI3K, p38MAPK, and JNK signaling pathways are involved in the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists (Figure 8). In addition, we found that the specific inhibitors for the Raf/Ras/MEK/MAPK pathways significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503, suggesting that these pathways are involved in the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists (Figure 8).
Our recent positron emission tomography (PET) study demonstrated that, after a single oral administration, fluvoxamine bound to sigma-1 receptors in the living <span class="yellow">human</span> brain [45]. This finding suggests that sigma-1 receptors are involved in the mechanism of action of fluvoxamine in the <span class="yellow">human</span> brain [45]. Taken together, the past and present findings suggest that, unlike paroxetine and sertraline, the SSRI fluvoxamine, with its sigma-1 receptor agonistic activity, might be a unique therapeutic drug for neuropsychiatric diseases.
In conclusion, the present results suggest that, as a sigma-1 receptor agonist, fluvoxamine could potentiate the NGF-induced neurite outgrowth in PC12 cells. Furthermore, it is likely that interaction with IP3 receptors and several subsequent signaling molecules are involved in the mechanism underlying the pharmacological action of sigma-1 receptor agonists. Therefore, it is likely that sigma-1 receptor agonists such as fluvoxamine and DHEA-sulfate, which are available around the world, would be unique therapeutic drugs for neuropsychiatric diseases.<br><br>
<h3>pmcA2562362</h3>Host immunity in the protective response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span>
Abstract
Background
We performed initial cell, cytokine and complement depletion studies to investigate the possible role of these effectors in response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span> in a susceptible BALB/c <span class="yellow">mouse</span> model of infection.<br><br>Results
While protection with heat-killed bacilli did not result in sterilizing immunity, limited protection was afforded against an otherwise lethal infection and provided insight into potential host protective mechanisms. Our results demonstrated that <span class="yellow">mice</span> depleted of either B cells, TNF-α or IFN-γ exhibited decreased survival rates, indicating a role for these effectors in obtaining partial protection from a lethal challenge by the intraperitoneal route. Additionally, complement depletion had no effect on immunoglobulin production when compared to non-complement depleted controls infected intranasally.<br><br>Conclusion
The data provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br><br><br>Background
<span class="yellow">Burkholderia mallei</span>, the etiologic agent of glanders, is a gram-negative, capsulated, non-motile, facultative intracelluar bacterium. Most known members of the Burkholderiaceae are resident in the soil; however, <span class="yellow">B. mallei</span> is thought to be an obligate mammalian pathogen. <span class="yellow">Horses</span> are highly susceptible to infection and considered the natural reservoir for infection, although <span class="yellow">mules</span> and <span class="yellow">donkeys</span> are also susceptible [1]. Clinically, glanders in solipeds can present as either a chronic (<span class="yellow">horses</span>) or acute (<span class="yellow">mules</span> and <span class="yellow">donkeys</span>) form. Naturally acquired <span class="yellow">human</span> infection with <span class="yellow">B. mallei</span>, although not seen in the United States since 1945, has occurred rarely and sporadically among laboratory workers and those in direct contact with infected animals [2]. However, glanders is endemic among domestic animals in Africa, Asia, the Middle East, and Central and South America. The course of infection is dependent on the route of exposure. Direct contact with the skin can lead to a systemic infection. Inhalation of aerosol or dust containing <span class="yellow">B. mallei</span> can lead to septicemic, pulmonary, or chronic infections of the muscle, liver and spleen. The disease has a 95% case fatality rate for untreated septicemic infections and a 50% case fatality rate in antibiotic-treated individuals [3].
There is no <span class="yellow">human</span> or animal vaccine available for glanders, and development of a partial or fully protective adaptive host response to the organism has not been well-defined. Previous studies with <span class="yellow">B. mallei</span> and the host response have shown that a mixed immune response consisting of both Th1 and Th2-associated cytokines with a predominant IgG1 subclass does not correlate with protection [4]. Additional studies with passive transfer of monoclonal antibodies specific for <span class="yellow">B. mallei</span> have correlated with early protection from infection [5]. Recent studies have also shown the Th1 cytokine IL-12 to mediate partial protection to non-viable <span class="yellow">B. mallei</span>-vaccinated <span class="yellow">mice</span> [6]. Thus, full correlates of protection mediated by the adaptive immune system against <span class="yellow">B. mallei</span> remain to be fully elucidated.
In this series of studies, we sought to address the impact of depletion of the major effector lymphoid cell populations (B220+ B cells, CD4+ or CD8+ T cells) and key pro-inflammatory/Type 1 cytokines (IFN-γ or TNF-α) on survival in BALB/c <span class="yellow">mice</span> vaccinated with heat killed (HK) bacilli followed by an intraperitoneal (i.p.) challenge with live organism. In addition, studies investigating the effect of complement on opsonization of organism and antibody production were assessed. Heat killed bacteria were used as a model of vaccination to allow evaluation of <span class="yellow">B. mallei</span> specific immune responses. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br>Results
Heat-killed <span class="yellow">B. mallei</span> vaccination mediates partial protection from lethal challenge
To begin to address this issue in an animal model of acute infection, we established that immunologically naive BALB/c <span class="yellow">mice</span> challenged i.p. with 2 × 107 CFU resulted in death by day 4–6, while i.p. immunization with 1 × 105 heat killed (HK) bacteria provided partial protection against a subsequent challenge. Two independent experiments resulted in similar findings of 40% survival for HK-vaccinated <span class="yellow">mice</span> with a mean survival time (MST) of 8 days versus 4 days in naïve <span class="yellow">mice</span> (Fig. 1). The administration of vaccines for <span class="yellow">B. mallei</span> during an outbreak would mandate relatively rapid onset of protection for <span class="yellow">human</span> or veterinary use. Based on non-routine use and vaccine implementation in the course of an outbreak, a 14 day window was chosen for assessment of protection. Our results indicate that HK vaccination can afford partial protection to an otherwise lethal challenge of <span class="yellow">B. mallei</span> by the i.p. route.<br><br>Effects of cell depletion on HK-vaccinated survival
To dissect the cellular basis for protection mediated by HK vaccination, 13 days after immunization with HK bacteria (day -1), and at day of challenge, <span class="yellow">mice</span> were dosed with antibodies to deplete CD4+, CD8+ or B220+ cells. Antibody depletion of CD4+, CD8+, or B220+ cells in these <span class="yellow">mice</span> was confirmed by flow cytometric analysis with depletion efficiencies for CD4, CD8, and B220 populations at 99.7%, 96%, and 95%, respectively, relative to <span class="yellow">mice</span> treated with isotype control monoclonal antibodies (data not shown). Our results demonstrated decreased survival rates in B220 (p = 0.3418), CD4+ (p = 0.5417) and CD8+ (p = 0.4684) antibody depleted <span class="yellow">mice</span>, compared to isotype control antibody, a finding that indicated a possible role for vaccine induced antibody production. When challenged with 2 × 107 CFU/<span class="yellow">mouse</span> by the i.p. route, loss of T cells resulted in reduced survival (50%) relative to the non-specific isotype control (Fig. 2). In contrast to the loss of T cells, depletion of B220+ cells resulted in 100% mortality relative to the non-specific isotype control (Fig. 2). To further evaluate the necessity of these effector cells in providing protection following HK vaccination, relatively resistant C57BL/6 <span class="yellow">mice</span>, deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) were subjected to an identical HK vaccination and challenge regimen. Mature, B-cell-deficient <span class="yellow">mice</span> demonstrated a 50% decreased survival (p = 0.0888) compared to the wild-type <span class="yellow">mice</span> with an MST of 35.5 days (Fig. 3). CD4-/- and CD8-/- <span class="yellow">mice</span> exhibited a 60% (p = 0.1343) and 0% reduced survival, respectively (Fig. 3).<br><br>Effects of cytokine depletion on HK vaccination
Similar studies were performed to determine the role of IFN-γ or TNF-α in acute infection in BALB/c <span class="yellow">mice</span> immunized with HK bacteria. Six hours before challenge, <span class="yellow">mice</span> were dosed with antibodies that neutralize IFN-γ or TNF-α. Individual depletion of either TNF-α (p = 0.0145) or IFN-γ (p = 0.0446) resulted in 100% mortality with an MST of 3 and 2 days, respectively, compared to the HK-vaccinated isotype control <span class="yellow">mice</span> (Fig. 4). In contrast, 40% of HK-vaccinated, isotype control <span class="yellow">mice</span> survived to at least 12 days post-challenge (Fig 4). To further evaluate the host TNF-α response during an established <span class="yellow">B. mallei</span> chronic infection, we infected 12 BALB/c <span class="yellow">mice</span> by the i.p. route with 1 × 106 CFU <span class="yellow">B. mallei</span>. One animal was terminally ill on day 37 post-infection. On day 42 post-infection, the remaining 11 <span class="yellow">mice</span> were dosed with either anti-TNF-α (n = 6), or control mAb (AFRC Mac 49) (n = 5). No further deaths were observed in the control mAb-treated <span class="yellow">mice</span>. Rapid mortality was observed in the anti-TNF-α-treated group, with all <span class="yellow">mice</span> dying within 7 days of treatment (p = 0.0023) relative to the isotype-treated controls (Fig. 5).<br><br>J774A.1 uptake of serum treated <span class="yellow">B. mallei</span>
Complement mediated uptake assays were performed to evaluate opsonization. Results indicated enhanced bacterial uptake in J774A.1 phagocytes inoculated with serum treated <span class="yellow">B. mallei</span> (p = .0082), compared to <span class="yellow">B. mallei</span> alone, while heat-inactivated serum produced uptake percentages similar to those prior to serum addition (Fig. 6). Taken together, these results imply an active role for complement components in the uptake of organism by macrophages.<br><br>Immunoglobulin production in HK vaccinated BALB/c <span class="yellow">mice</span>
We further characterized the ability of HK vaccination to induce a predominant IgG isotype by determining IgG2a/IgG1 ratios in i.p. and i.n. vaccinated BALB/c <span class="yellow">mice</span>. Pre (day 14 post vaccination) and post (day 2 post infection) exposure serum samples were obtained and evaluated for IgG isotype concentrations (Table 1). No appreciable differences in IgG pre-exposure levels were seen when comparing i.n. to i.p. vaccination. In addition, cobra venom factor-treated animals showed no significant differences to non-cobra venom factor-treated animals in IgG pre-exposure (challenge) levels. Conversely, isotype switching in the cobra venom factor treated animals was enhanced in post-exposure serum IgG2a (Table 1).<br><br>
Discussion
Recent studies have shown a key role in protection from lethal challenge for IFN-γ in non-vaccinated <span class="yellow">mice</span> from either NK and/or NKT cells following experimental exposure to <span class="yellow">B. mallei</span> and <span class="yellow">B. pseudomallei</span> [7,8]. A similar protective role in the innate response to infection has been demonstrated for TNF-α in <span class="yellow">B. pseudomallei</span> infection [8]. The studies presented here are consistent with the essential role of these factors in the relative levels of protection conferred by vaccination with heat-killed <span class="yellow">B. pseudomallei</span> and would appear to be viable early markers for protection from lethal acute infection [9]. Currently, there are no fully protective vaccines against <span class="yellow">B. mallei</span> or <span class="yellow">B. pseudomallei</span> in a <span class="yellow">murine</span> model, particularly for the sensitive BALB/c versus C57BL6 models. Previous studies have also demonstrated that both the humoral and cell-mediated arms are essential for protection from <span class="yellow">B. pseudomallei</span> infection [10]. Thus, loss or reduction of TNF-α and IFN-γ levels result in significantly reduced survival rates, substantiating previous reports of the role of these factors in protection against <span class="yellow">B. mallei</span> [7]. Moreover, we demonstrate a role for sustained TNF-α production in the maintenance of host survival throughout the course of <span class="yellow">B. mallei</span> infection. <span class="yellow">Mice</span> with an established <span class="yellow">B. mallei</span> chronic infection rapidly lost the ability to control the growth of the bacillus upon neutralization of TNF-α. This would suggest a potential role for TNF-α in the maintenance of productive granulomas which may limit the spread of bacteria in chronically infected hosts, or, alternatively, in direct or indirect microbicidal or bacteriostatic activities at the sites of infection. Additional studies are underway to determine more precisely the role of TNF-α in host protection to <span class="yellow">B. mallei</span>.
Multiple innate and adaptive cell types may contribute to the production of IFN-γ in response to infection with <span class="yellow">B. mallei</span> following vaccination. Our results with individual depletion of CD4+ and CD8+ T cells suggests that both cell types may compensate for the functional loss of the other effector cell type in the production of this key cytokine. The effector role for IFN-γ in mediating protection against <span class="yellow">B. mallei</span> may include both immunoregulatory and non-regulatory functions. Regardless, the requirement of IFN-γ, as demonstrated by administration of neutralizing antibody prior to infection, indicates that stimulation of IFN-γ response is a desirable goal for a <span class="yellow">B. mallei</span> vaccine.
Similarly, B220-positive cells appear to play a role in protection following vaccination with heat-killed <span class="yellow">B. mallei</span>. Interestingly, this protective immunity, occurring in other intracellular pathogens, is not exclusively dependent on B cells [11]. Passive protection has been demonstrated against acute Burkholderia infection by monoclonal antibodies [5,12]. Protection against <span class="yellow">B. pseudomallei</span> infection by anti-LPS, capsular polysaccharide and proteins has been short-lived, suggesting that antibody production offers limited protection in the initial stages of infection by an as-yet-undefined mechanism [12]. We have shown that following depletion of B220+ cells, survival rates decreased as much as 100% relative to non-depleted controls and individual CD4/CD8-depleted <span class="yellow">mice</span> via the intraperitoneal route. Results from C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) substantiate the requirement for B-cell involvement by evidence of μMT and CD4-/- decreased survival. The lack of an effective CTL response to vaccination did not appear to alter survival in what would appear to be a CD4/B-cell (humoral)-driven response. In CD4-deficient <span class="yellow">mice</span>, we have the additional potential variable that a CD4-dependent antibody response might also be inhibited during the vaccination phase relative to <span class="yellow">mice</span> treated with antibody immediately prior to and during the early phases of infection. Although not statistically significant, we did observe a decrease in survival in μMT (mature B cell) deficient <span class="yellow">mice</span> as early as day 9 post challenge, whereas CD4-deficient <span class="yellow">mice</span> produced similar results at day 32 post challenge, indicating a role for B cells independent of CD4 T cell help, perhaps through a T-independent mechanism of antibody production. Although CD8-/- C57BL/6 demonstrated no decreased survival in our HK-vaccinated model, a lack of potential endogenous protein production by HK <span class="yellow">B. mallei</span> may have contributed to limited MHC-I presentation.
Complement associated studies revealed increased J774A.1 uptake of serum-treated <span class="yellow">B. mallei</span>. Complement-mediated uptake studies of <span class="yellow">B. pseudomallei</span> by polymorphonuclear leukocytes (PMNs) suggest that capsule production contributes to resistance of phagocytosis by reducing C3b bacterial deposition [13]. Previous studies have demonstrated that a polysaccharide capsule is present in <span class="yellow">B. mallei</span>, [14,15] although in the present study enhanced uptake with serum-treated <span class="yellow">B. mallei</span> was observed. Intracellular survival assays of complement mediated uptake of organisms were not performed in the present study, thus, the role of complement opsonization on intracellular survival is not fully resolved. Previous reports have demonstrated the ability of <span class="yellow">B. mallei</span> to survive within macrophage without the aid of serum coating organisms [16]. Conversely, the idea of antibody mediated opsonization to facilitate macrophage activation and clearance of intracellular organisms may offer support to the role of B cells in an effective immune response. A possible protective mechanism may include HK vaccination induced production of opsonizing antibodies which may aid in complement mediated uptake, thereby limiting the initial bacterial threshold below a lethal level.
Immunoglobulin responses to HK vaccination resulted in modest levels of IgG1 following 2 weeks post vaccination, while post-exposure levels were indicative of efficient class switching to a favorable IgG2a isotype. Importantly, cobra venom factor treatment of animals at time of vaccination did not alter their ability to produce immunoglobulin. In fact, cobra venom factor treated animals resulted in higher IgG2a levels when compared to non-treated. Complement activation can modulate both the primary and secondary immune responses and has been shown to enhance secondary immune responses to vaccination [17]. The current results suggest that cobra venom factor treatment may affect the modulation of the immune response to <span class="yellow">B. mallei</span> infection through B cell activation and/or memory B cell generation.<br><br>Conclusion
In summary, our results provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. Understanding and defining the role of B cells in adaptive <span class="yellow">B. mallei</span> immunity will likely be fundamental to the design of an efficacious vaccine and important goals of future research.<br><br>Methods
Bacterial strain and <span class="yellow">mice</span>
<span class="yellow">B. mallei</span> strain ATCC 23344 (China 7) was cultured on Luria-Bertani agar supplemented with 4% glycerol (LB+4%G) agar plates for 48 h at 37°C. Isolated colonies were sub-cultured to LB+4%G broth, and cultures were incubated at 37°C until optical density readings at 600 nm (OD600) reached an exponential phase of growth. Bacteria were pelleted by centrifugation, washed and re-suspended in sterile 1× phosphate-buffered saline (PBS, pH 7.4) to obtain the desired CFU/ml. To obtain HK inoculums, bacterial suspensions were incubated at 85°C for 3 h and stored at 4°C until use. The absence of live <span class="yellow">B. mallei</span> organisms in the HK preparations was confirmed after plating 10% of the total inoculums (v/v) and incubating these at 37°C for 48 h. All procedures were performed under a class II biosafety cabinet in a biosafety level 3 laboratory. Female, 6- to 8-week-old, BALB/c <span class="yellow">mice</span> (n = 5–7) were obtained from Harlan Sprague Dawley, Inc. (Indianapolis, Indiana). Female, 6- to 8-week-old, C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) and CD8 T-cells (CD8-/-) and wild-type <span class="yellow">mice</span> were obtained from The Jackson Laboratory (Bar Harbor, Maine).<br><br>Vaccination and challenge
BALB/c and C57BL/6 <span class="yellow">mice</span> were grouped and vaccinated with 0.5 μg of HK <span class="yellow">B. mallei</span> (without adjuvant) by i.p. injection using a 25-gauge syringe. Two weeks post HK vaccination <span class="yellow">mice</span> were injected i.p. with 2 × 107 CFU/100 μl of live <span class="yellow">B. mallei</span> (~20 LD50) [18]. Complement depleted animals were challenged with 2.5 × 104 CFU/50 μl (~0.25 LD50) by intranasal (i.n.) route. Aliquots from the inoculums were plated to confirm the infecting dose. All procedures and animal protocols used in this study were approved by the Biosafety and IACUC committees at UTMB and conducted in either BSL-3 or ABSL-3 laboratories.<br><br>Cell and cytokine depletions
Acute in vivo cell/cytokine depletion was performed with monoclonal <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD4 (GK1.5), CD8α (53-6.7) or B220 (RA3-6B2) obtained from R&D Systems, Inc. (Minneapolis, MN) by methods similar to those we have previously described [19]. Functional grade purified <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> interferon-gamma (IFN-γ, AN-18) was obtained from eBioscience (San Diego, CA) and purified anti-<span class="yellow">mouse</span> tumor necrosis factor (TNF-α, MP6-XT3) from BD Pharmingen (San Diego, CA). IFN-γ and TNF-α antibodies were injected i.p. 6 h prior to challenge, 200 μg per <span class="yellow">mouse</span> in 200 μl PBS or at later time points as indicated. <span class="yellow">Rat</span> IgG isotype control was obtained from Southern Biotech (Birmingham, AL) and administered i.p. on day of challenge, 200 μg/<span class="yellow">mouse</span>. <span class="yellow">Rat</span> anti-<span class="yellow">mouse</span> CD4, CD8α and B220 were injected i.p. twice, 1 day prior to challenge and on day of challenge, with an equivalent dosage sufficient to deplete T or B cells from 6 × 108 bone marrow cells per injection. The efficiency of depletion at time of infection for CD4+, CD8+, and B220+ cells was confirmed by flow cytometry analysis immediately prior to infection.<br><br>Complement depletion with cobra venom factor
<span class="yellow">Mice</span>, six to seven per group, were vaccinated i.p. with 1 × 105 CFU of nonviable <span class="yellow">B. mallei</span> cell preparation in a total volume of 0.1 ml. Two weeks later, 24 h and 1 h before challenge, complement depleted <span class="yellow">mice</span> were treated i.p. with 12.5 units total cobra venom factor (Quidel Corporation Speciality Products, San Diego, CA) in 0.1 ml of PBS. Complement depletion was confirmed prior to challenge by micro-titer hemolytic complement activity (CH50) assay as previously described [20].<br><br><span class="yellow">B. mallei</span> J774A.1 uptake assays
J774A.1 cells were seeded (5 × 105) onto Corning costar 24 well plates (Corning, NY) with DMEM and incubated overnight at 37°C with 5% CO2. Bacterial suspensions were incubated at 37°C for 45 minutes supplemented with 2% <span class="yellow">mouse</span> serum from Sigma-Aldrich (St. Louis, MO.), heat inactivated <span class="yellow">mouse</span> serum (56°C 30 minutes), or bacteria alone and then added at an MOI of 10:1 to J774A.1 cells in triplicate. Inoculated wells were centrifuged at 800 g for 2 minutes and incubated for 2 hours at 37°C with 5% CO2 followed by a PBS wash (×2) and 2 hour incubation with 250 μg/ml kanamycin. Wells were washed twice with PBS and lysed with 0.1% Triton X-100, followed by serial 10-fold dilutions plated on LBG plates and incubated at 37°C for 2 days. Colony forming units were enumerated and uptake expressed as a percentage of initial inoculating dose ± SEM.<br><br>Antibodies and flow cytometry
Flow cytometric analysis was performed on 0.1-ml blood samples transferred to micro centrifuge tubes containing 90 μl of acid citrate dextrose (ACD) solution. Red blood cells were lysed using ACK-lysing buffer (Biosource International, Inc., Camarillo, CA) according to the manufacturer's instruction. Antibodies used for analysis of surface markers included: FITC-conjugated <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD45R/B220 (RA3-6B2, BD Pharmingen San Diego, CA) for B cells; FITC-conjugated <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD8α (53-6.7) and CD4 (GK1.5, BD Pharmingen, San Diego, CA) for CD8+ or CD4+ cells, respectively. Samples evaluated for CD4+ and CD8α+ cells were also incubated with biotin-conjugated <span class="yellow">hamster</span> anti-<span class="yellow">mouse</span> CD3e (145-2C11) monoclonal antibody (BD Pharmingen, San Diego, CA) and subsequently with streptavidin APC Cy7. Isotype-matched, non-specific controls were assayed in parallel (BD Pharmingen, San Diego, CA). Surface staining was performed according to previously published protocols [21]. Following cell staining, the samples were fixed with 2% buffered paraformaldehyde overnight prior to analysis by flow cytometry. Samples were analyzed using a FACSCalibur flow cytometer with BD CellQuest Pro software.<br><br>Antibody assays
Immunoglobulin subclass IgG1 and IgG2a titers in <span class="yellow">mice</span> were determined by a whole bacterial cell ELISA performed in 96-well, Immulon 2 HB, round-bottom plates (Dynex Technologies). <span class="yellow">B. mallei</span> antigen was diluted in 0.1 M carbonate buffer (pH 9.5) and 50 μl of diluted cells placed into wells. Plates were stored overnight at 4°C. The plates were washed with washing solution (1× PBS, 0.05% Tween 20), and incubated with 100 μl of blocking solution (1× PBS, 1% <span class="blue">bovine</span> serum albumin, 0.05% Tween 20) for 1 h at 37°C. Dilutions of <span class="yellow">mouse</span> sera were made with blocking solution in duplicate and plates were incubated for 1 h at 37°C. Following incubation, plates were washed and 50 μl of anti-Ig-<span class="yellow">horseradish</span> peroxidase subclass conjugate, diluted accordingly to manufacturer's instructions (Southern Biotechnology Associates, Inc. Birmingham, Ala.), was added to each well and incubated for 1 h at 37°C. After washing, 50 μl of 2,2'-azino-di-(3-ethylbenzthizoline)-6-sulfonate (ABTS) peroxidase substrate (KPL, Inc., Gaithersburg, Maryland) was added to each well and plates incubated for 25 min at room temperature. The amount of bound antibody was determined colorimetrically by absorbance at 405 nm.<br><br>Statistical analysis
Survival curves were calculated by Kaplan Meier survival analysis with log-rank tests between groups using GraphPad Prism (V.4.03 for windows). Statistical analysis was generally performed with the paired Student's t-test. P value ≤ 0.05 was considered significant.<br><br>
Abbreviations
HK: Heat-killed; i.p.: intraperitoneal; i.n.: intranasal.<br><br>Authors' contributions
GCW designed and conducted experiments and drafted the manuscript. BMJ carried out the immunoassays and animal work. SP provided analysis of data and contributed to design and animal work. RAL participated in the generation and analysis of chronic TNF-α data. DME conceived the study, and participated in its design and coordination and helped to draft the manuscript. AGT participated in the bacterial work and drafting of the manuscript. All authors read and approved the final manuscript.<br><br>
</body></html>