
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>chicken</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="blue">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1805739</h3>Embryonic sympathoblasts transiently express TrkB in vivo and proliferate in response to brain-derived neurotrophic factor in vitro
Abstract
Background
Nerve growth factor and neurotrophin-3 are involved in the development of sympathetic neurons; however, whether brain derived neurotrophic factor also plays a role is not known. The purpose of this study was to determine whether BDNF and its receptor, TrkB, are expressed during the development of paravertebral sympathetic ganglia in vivo and to determine the effect of BDNF in vitro.<br><br>Results
As neural crest cells coalesce to form sympathetic ganglia, TrkB-positive cells are seen in both <span class="blue">chicken</span> and <span class="yellow">mouse</span> embryos. In <span class="blue">chicken</span> embryos, TrkB-expressing cells first appear at Hamburger-Hamilton Stage (St) 27 and they co-express HNK-1, confirming that they are migrating neural crest cells. The TrkB-positive cells lack neural markers at this stage; however, they migrate with other neurally differentiating cells that are TrkA and TrkC-positive. By St. 29/30, TrkB-positive cells begin to express the neural specific markers Hu C/D and Islet-1; eventually, all TrkB positive cells commence neural differentiation. By St. 34, TrkB and TrkC staining are lost. BDNF transcript expression parallels that of TrkB. In the <span class="yellow">mouse</span>, TrkB-positive cells surround newly formed sympathetic ganglia and a small number of TrkB positive cells that co-express tyrosine hydroxylase are seen within ganglia between E13.5-15. In cell culture, many cells from St. 29–30 <span class="blue">chicken</span> lumbar sympathetic ganglia express neural markers and are dividing, indicating that they are sympathoblasts. Sympathoblasts and neurons require both nerve growth factor and neurotrophin-3 for survival. BDNF increases the number of cells expressing neural markers in culture by increasing number of cells that incorporate bromodeoxyuridine. In contrast, most TrkB-positive sympathetic cells in vivo are not actively proliferating between E6–E8.<br><br>Conclusion
Developing paravertebral sympathetic ganglia in avian and <span class="yellow">murine</span> embryos contain a subpopulation of sympathoblasts that transiently express TrkB and ultimately commence neuronal differentiation. These TrkB expressing sympathoblasts are not actively dividing in vivo; yet, when placed in vitro, will divide in response to BDNF. This suggests that the availability of BDNF in vivo fails to reach a threshold necessary to induce proliferation. We suggest that excess TrkB stimulation of sympathoblasts in vivo may lead to the genesis of neuroblastoma.<br><br><br><br>Background
Neural crest cells destined to become paravertebral sympathetic neurons proliferate and differentiate during migration and gangliogenesis. In <span class="blue">chicken</span> embryos, migrating neural crest cells express catecholamines at Hamburger/Hamilton Stage (St.) 19, and these cells form the primary sympathetic chain dorsolateral to the aorta at St. 22 (E3.5) [1]. Between St. 23 (E4) and St. 28 (E6), these cells disperse and undergo a secondary migration to form the paravertebral sympathetic chain that resides ventral to the spinal cord and dorsal root ganglion [1]. After ganglia coalesce, sympathoblasts express markers of neuronal differentiation, such as Q211 and tyrosine hydroxylase (TH), at a time when they also incorporate [3H]-thymidine [2]. Time lapse photography has shown that cultured E15.5–E16.5 sympathetic neurons from <span class="yellow">rat</span> embryos extend axons while they divide [3-5]. Although proliferation appears to be an important process to expand the sympathetic neuron population during differentiation, the mechanisms that guide sympathoblast proliferation have not been identified.
The development of sympathetic neurons is guided by neurotrophins. Neurotrophin-3 (NT-3) binds to its receptor, TrkC, to promote the survival of cultured sympathoblasts from early lumbar paravertebral ganglia [6]. Nerve growth factor (NGF) signals through its receptor, TrkA, to promote the survival of sympathetic neurons upon target innervation [7]. There are severe sympathetic defects in the superior cervical ganglion of individual NT-3 and NGF knockout <span class="yellow">mice</span> [8-10]. Furthermore, there is no additional cell death in the superior cervical ganglion of NT-3 and NGF double knockout <span class="yellow">mouse</span> embryos, suggesting that all of the neurons are dependent on both neurotrophins for survival [11]. There is also an increase in sympathetic neuron cell death in TrkA knockout <span class="yellow">mice</span> [12]. However, in TrkB and BDNF knockout <span class="yellow">mice</span>, there is no apparent phenotype in the superior cervical ganglion, and there is little evidence that TrkB or BDNF is expressed in sympathetic ganglia. Thus, it is generally thought that TrkB and BDNF have little or no roles in guiding the development of sympathetic neurons.
In addition to their developmental functions, neurotrophin receptors regulate cell behavior in neuroblastoma, a tumor found in sympathetic ganglia and adrenal medulla. Tumors that express TrkA often spontaneously regress, while those that express TrkB and its ligand, brain-derived neurotrophic factor (BDNF), grow aggressively, are invasive, and fail to respond to chemotherapeutic agents [13]. The presence of TrkA in neuroblastoma tumors is consistent with its expression in developing sympathetic neurons, and suggests that regressive neuroblastoma tumors arise from early sympathetic neurons that express TrkA. The function of TrkB in early sympathetic development is unknown, which makes understanding the etiology of aggressive neuroblastoma tumors difficult. Based on its function in neuroblastoma tumors, we hypothesize that BDNF and TrkB expression in differentiating sympathoblasts is responsible for expanding the neuronal population through proliferation.
We sought to determine whether BDNF and TrkB are involved in sympathetic development. We report that during early embryonic development, TrkB is expressed in a subset of differentiating sympathoblasts in both avian and <span class="yellow">murine</span> embryos. We also find that BDNF promotes the proliferation of TrkB-positive sympathoblasts in cell culture. However, the majority of TrkB positive cells in vivo fail to take up bromodeoxyuridine (BrdU) over a 24 hr period, suggesting that endogenous BDNF concentrations do not reach a threshold necessary to stimulate proliferation of sympathoblasts. Shortly after all of the TrkB positive cells commence neuronal differentiation, TrkB immunoreactivity is lost. These results suggest that prolonged expression and/or activation of TrkB signaling at these early stages may be an early event triggering the formation of neuroblastoma.<br><br>Results
TrkB is expressed during migration of neural crest cells to sympathetic ganglia
We first determined whether TrkB is expressed in neural crest-derived cells in the region ventral to the spinal cord and dorsal root ganglia where sympathetic ganglia coalesce between Hamburger/Hamilton Stages (St.) 25–28/29. To identify cells that have commenced neuronal differentiation, transverse sections of the lumbar spinal column region were stained with antibodies against Hu C/D [14], a neuronal-specific RNA-binding protein, or Islet-1, a transcription factor found in sympathetic neurons [15]. We found that Hu C/D and Islet-1 are expressed in the same cells both in vivo and in vitro throughout sympathetic development. In experiments done between St. 25 and 28, Islet-1 staining appeared weaker than Hu C/D staining, and thus we used Hu C/D to identify differentiating neurons at these stages. At later stages, Islet-1 was used to facilitate the identification of neurons because of the nuclear location of its immunoreactivity.
Cells expressing Hu C/D are first detected at St. 25 ventral to the spinal cord and dorsal root ganglion and lateral to the dorsal aorta (Figure 1A, 1B). By St. 26, the number of cells that express Hu C/D in this region increases dramatically (Figure 1C). TrkB-expressing cells first appear at St. 27 in the same region and are adjacent to Hu C/D-positive cells (Figure 1D, 2A, 2H). TrkB-positive cells co-localize with a neural crest marker, HNK-1 (Figures 2B, 2C, 2D). The Hu C/D-positive cells in this region are likely to be sympathetic neurons, since they appear in the region where sympathetic ganglia form and express tyrosine hydroxylase, a rate-limiting enzyme in the synthesis of catecholamines (Figures 2E, 2F, 2G). At St. 28/29, the cells begin to coalesce ventral to the dorsal root ganglion and the Hu C/D-positive cells and TrkB-positive cells remain as two separate cell populations (Figure 1E); however, shortly afterwards, all of the TrkB-positive cells begin to express Islet-1 (Figure 3B) and Hu C/D (data not shown).<br><br>Developmental regulation of TrkA, TrkB, TrkC, and BDNF expression
In contrast to TrkB, the other neurotrophin receptors, TrkA and TrkC, are co-expressed in both Hu C/D-positive and Hu C/D-negative cells at St. 27 (Figures 2I, 2J). We find that approximately 30% of the Islet-1-positive cells express TrkA (Figure 3A), while 50% express TrkB (Figure 3B) and 100% express TrkC (Figure 3C) at St. 29/30 (E6.5). Thus, all developing neurons express TrkC in combination with either TrkA or TrkB. By St. 31 (E7), the number of TrkA-positive, Islet-1-positive cells increases to 100% (Figure 3D) and immunoreactivities for both TrkB and TrkC appear dispersed (Figure 3E, 3F). By St. 34 (E8), TrkA expression is well-sustained (Figure 3G) and TrkB and TrkC immunoreactivities are lost (Figure 3H, 3I). We also examined the early development of <span class="yellow">murine</span> sympathetic ganglia (Figure 4). At E13, the newly formed lumbar sympathetic ganglia can be observed ventral to the spinal cord and notochord by their staining for Hu C/D and TH (Figure 4A). TrkB-positive cells can be seen surrounding developing ganglia, as well as in occasional cells within the ganglia (Figure 4C, D). These TrkB-positive cells within the ganglia co-express TH and are seen at a frequency of 1–2 cells per section starting at E13 (Figure 4A–D) and are still present at E15.5 (data not shown).
In neuroblastoma cells, BDNF is co-expressed with TrkB, suggesting that autocrine stimulation is a means by which proliferation is sustained in the transformed cells. To test whether BDNF, the ligand for TrkB, was present in embryonic <span class="yellow">chick</span> sympathetic ganglia, we used quantitative real-time PCR with TaqMan probes to determine the relative abundance of BDNF transcripts in total RNA extracted from lumbar sympathetic ganglia at St. 29/30 (E6.5), St. 31 (E7), St. 34 (E8), and E9. BDNF expression within the ganglia parallels that of TrkB: BDNF mRNA expression levels are highest at St 29/30 (E6.5), and these levels decrease 2-fold at St. 31 (E7) and St. 34 (E8; Figure 5). By E9, BDNF levels are 7 times lower than at St. 29/30 (E6.5; Figure 5).<br><br>NT3 and NGF promote survival of differentiating sympathetic neurons in culture
To determine the effect of neurotrophins, we cultured cells dispersed from lumbar sympathetic ganglia at St. 29/30 (E6.5) because, at this stage, ganglion formation is complete, the number of TrkA-, TrkB-, and TrkC-positive cells have peaked, and all Trk-expressing cells have initiated neural differentiation. First, we identified markers expressed by acutely isolated cells. As shown in Table 1, 80–91% of the cells are p75 neurotrophin receptor (NTR)-positive, indicating that most of the cells are neural crest-derived and little mesenchymal contamination is introduced by the isolation procedure. In addition, 28–33% of the cultured cells express the neural marker Hu C/D. Approximately half of these Hu C/D-positive cells express TrkB. Conversely, all of the TrkB positive cells express Hu C/D. These TrkB-positive cells comprise approximately 14–17% of the total cell population.
We then determined how many of the acutely isolated cells were proliferating by incubating them for 12 hrs in BrdU-containing medium. For these experiments, we identified differentiating neurons with the transcription factor Islet-1 because this marker labels nuclei, thus it co localizes with any BrdU that has been incorporated into the DNA, allowing us to determine whether the cell had undergone S-phase of the cell cycle. After 12 hrs in BrdU, 59% of Islet-1-positive nuclei stain for BrdU immunoreactivity. Thus, cultures of St. 29/30 sympathetic ganglia contain many cells that proliferate while exhibiting markers of neuronal differentiation, confirming previous observations [2]. We call these dividing neuronal precursors sympathoblasts. The remaining non-BrdU incorporating, Islet-1 positive cells are likely to be post mitotic neurons.
Finally, we determined the trophic requirements of St. 29/30 (E6.5) sympathetic neurons and sympathoblasts. We monitored cultures over a three day period after plating and counted the number of phase bright cells with neurites, a morphological feature of both neurons and sympathoblasts. In the absence of trophic factors, more than 2/3 of the cells die by 24 hours in culture and BDNF, NT-3, or NGF alone is not sufficient to promote survival (Figure 6). However, NGF together with NT-3 supports the survival of a significantly larger number of cells (Figure 6). For the subsequent experiments, all neurons were cultured with 25 ng/ml NT-3 and 1 μg/ml 7S NGF to optimize survival.<br><br>BDNF promotes proliferation of TrkB-positive sympathetic neurons in culture
To determine the effects of BDNF, cultures of cells from St. 29/30 (E6.5) sympathetic ganglia were supplemented with 200 ng/ml BDNF and the number of neurons and sympathoblasts were counted at 24, 48, and 72 hours using phase microscopy (Figure 7A). A 1.6-fold increase in the number of neurons due to BDNF is observed by 24 hours and this number does not increase further at 48 or 72 hours. This effect of BDNF is concentration-dependent with an EC50 of 75 ng/ml (Figure 7B).
To test whether the increase in the number of neurons and sympathoblasts caused by BDNF is due to the differentiation of pluripotent neural crest cells, we quantified the effects of BDNF on the number of neurally differentiating cells (Hu C/D-positive) versus the number of non-neuronal cells (Hu C/D-negative) after identifying all neural crest-derived cells by staining for p75NTR in St. 29/30 (E6.5) cultures. If BDNF increases the number of neurons and sympathoblasts by inducing a non-neuronal cell to express Hu C/D, then we expected that the total cell number would remain the same and that there would be a decrease in the number of non-neuronal cells as well as a corresponding increase in the number of neurons. After 24 hours, BDNF significantly increases the number of p75NTR-positive cells as well as the number of Hu C/D-positive cells (Figure 8A). However, there was no statistically significant change in the number of non-neuronal cells. Thus, it is unlikely that BDNF increases the number of neurons and sympathoblasts by inducing differentiation of non-neuronal cells.
To determine whether the increase in the total number of neurons and sympathoblasts is caused by BDNF-induced proliferation, control and BDNF-treated sympathetic cultures were exposed to BrdU for 12 hours after plating, and the number of cells that incorporated BrdU into their DNA was determined after 24 hours in culture. Even in the control condition, a number of cells in the culture are dividing, giving a high baseline of BrdU incorporation (Figure 8B). When BDNF is added, the total number of BrdU positive cells increases approximately 1.6-fold (Figure 8B). This BDNF-induced increase in the total number of BrdU positive cells occurs in sympathoblasts because the number of Islet-1-positive nuclei from control cultures that label with BrdU is 268 ± 59 and BDNF treatment raises this number to 424 ± 80, which corresponds to an increase of 1.6-fold. This accounts for the 1.6-fold increase in total neuron number and total BrdU-positive cells described above. We then confirmed that BDNF acts on TrkB-positive cells: BDNF increases the number of TrkB-expressing cells that incorporate BrdU 2.6 – 4-fold over control (Table 2) and it also increases the overall number of TrkB-positive cells 2 – 2.5-fold over control (Table 2). BDNF does not increase the number of BrdU-positive, TrkB-negative cells or the overall number of TrkB-negative cells (Table 2). In further support that BDNF acts directly on TrkB-expressing cells, an antibody directed against the extracellular domain of TrkB completely prevents the effect of BDNF in promoting proliferation of TrkB-positive, but not TrkB-negative cells (compare Figure 9A to 9B). Thus, the effect of BDNF is restricted to the population that expresses TrkB, which are developing sympathoblasts.
To determine whether TrkB-positive cells are actively proliferating in vivo, embryos were injected with BrdU at St. 27 and harvested at St. 29, approximately 24 hrs later. The majority (85–90%) of TrkB-positive cells do not incorporate BrdU into their nuclei under basal conditions in vivo (Figure 10), although a few TrkB positive cells with labeled nuclei could be observed (arrows). This contrasts with our observation that 71–76% of TrkB-positive cells incorporate BrdU in culture after treatment with BDNF (Table 2), suggesting that endogenous BDNF does not achieve a threshold sufficient to support a high level of sympathoblast proliferation in vivo.<br><br>
Discussion
We report that the neurotrophin receptor TrkB is expressed in a subset of embryonic sympathoblasts during the early development of lumbar paravertebral sympathetic ganglia in <span class="blue">chicken</span> and <span class="yellow">mouse</span> embryos. In the <span class="blue">chicken</span>, TrkB expression is transient, and completely lost by St 34 (E8). Since BDNF induces the proliferation of sympathoblasts in cell culture, yet in vivo there is little proliferation observed in TrkB-positive cells in nascent ganglia, we propose that if TrkB activation becomes unregulated by excess BDNF or constitutive phosphorylation of TrkB [16], this transient population of TrkB-positive sympathoblasts may trigger the genesis of neuroblastoma, a childhood tumor found in the paravertebral chain and adrenal medulla.
The two populations of sympathoblasts that we observe support previous findings of heterogeneity among developing sympathetic neurons and neural crest cells. Early sympathetic ganglia contain at least two subpopulations: early differentiating neurons that lack TrkB expression and express TrkA and TrkC, and late differentiating sympathoblasts that express TrkB. Explant cultures of sympathetic ganglia from E16 <span class="yellow">chick</span> embryos give rise to two neuronal populations: one that remains close to the explant, and one that migrates away from the explant [1]. In addition, early neuronal subpopulations have been observed in cultures of neural crest cells from St. 13/14 quail embryos as evidenced by the expression of neuronal cell type-specific gangliosides [17]. Perhaps these different subpopulations will ultimately give rise to the two neurochemically distinct populations found in lumbar sympathetic ganglia: the noradrenergic, NPY-containing neurons that innervate internal organs and enteric ganglia and the cholinergic, VIP-containing neurons that innervate vasculature in the hind limbs.
The effects of BDNF and TrkB deletion and over expression have been studied on superior cervical ganglion and preganglionic neurons in thoracic segments of the spinal column, but not on paravertebral sympathetic neurons. In the superior cervical ganglion, an increase in the number of neurons of BDNF null <span class="yellow">mice</span> is likely due to apoptosis induced by BDNF via p75NTR [18]. In contrast, the responses of paravertebral sympathetic neurons to BDNF are complex and subtype dependent. Over expression of BDNF leads to an increase in the number of noradrenergic fibers innervating the erector pilli muscles of hair follicles, while noradrenergic fibers innervating blood vessels were unaffected [19]. If our results indicating that BDNF promotes proliferation of TrkB-positive sympathoblasts in the <span class="blue">chicken</span> embryo can be extrapolated to the subset of TrkB-positive sympathoblasts in <span class="yellow">murine</span> ganglia, then these TrkB-positive cells may be neurons destined to innervate the erector pilli. In other studies, TrkB null <span class="yellow">mice</span> showed no changes in morphology or cell number in superior cervical ganglia [12] or in the intermediolateral column [20]; but this may not be predictive of a phenotype in the lumbar paravertebral chain. It is thus possible that BDNF/TrkB signaling could play a specific role in other regions of the paravertebral sympathetic chain, such as the lumbar region. However, if TrkB-positive cells are not normally actively proliferating in vivo, then it would not be surprising that the development of the paravertebral sympathetic chain is not disrupted in TrkB or BDNF null <span class="yellow">mice</span>. It may be more informative to examine <span class="yellow">mice</span> that over express BDNF on a promoter that targets expression to embryonic lumbar ganglia. Unfortunately, such <span class="yellow">mice</span> do not exist.
Our findings that the St. 29/30 (E6.5) sympathoblasts are dependent on both NT-3 and NGF for survival in culture are consistent with previous work on <span class="yellow">mouse</span> sympathoblasts from the superior cervical ganglion [11]. In these studies, NT-3 and NGF deletion separately led to a decrease in the number of sympathetic neurons at E17.5 compared to control. Deletion of both NT-3 and NGF together did not enhance cell death. In contrast, cultured <span class="yellow">rat</span> superior cervical ganglion sympathetic neurons respond to NT-3 at E14.5 and then to NGF at E19.5, although time points in between were not analyzed [6].
In addition to promoting survival, NT-3, NGF, and BDNF also induce proliferation of various neuronal precursors at different stages of development. NT-3 can promote the incorporation of [3H]-thymidine into cultured quail neural crest cells from the trunk region [21,22], Later in <span class="yellow">rat</span> sympathetic development, NT-3 supports survival of neurons, but does not promote proliferation [6], which is consistent with our results. NGF promotes an increase in BrdU incorporation from 25% to 35% in the DRG cervical segment 2 in the <span class="yellow">chick</span> embryo [23]. In <span class="blue">chicken</span> embryos that are treated with NGF in ovo at St. 18 and 21, there is an increase in BrdU uptake after formation of the primary sympathetic chain at St. 23 [24]. Since NGF does not appear to affect proliferation of St. 29/30 (E6.5) <span class="yellow">chick</span> sympathoblasts, NGF may only promote proliferation in primary, but not secondary chain sympathoblasts. Motor neuron progenitors in the ventral neural tube from the <span class="yellow">chick</span> embryo express TrkB and when ventral neural tube explants are treated with BDNF, there is an increase in the number of motor neurons produced and BrdU incorporation [25]. BDNF also promotes the proliferation of cultured neuroblastoma cells [13]. Taken together, these results are consistent with our findings that NT-3 and NGF do not promote proliferation of St. 29/30 (E6.5) sympathoblasts, and support the assertion that BDNF promotes proliferation of TrkB-positive sympathoblasts in culture.
Our observations suggest a transient function of TrkB during early sympathetic development in supporting proliferation of this early subpopulation of sympathoblasts. However, the in vivo labeling suggests that only a minority (10–20%) of this population is dividing during the window that TrkB is expressed. In light of the very strong proliferative effect produced in cell culture, these TrkB expressing cells could respond more strongly if endogenous BDNF rises to higher levels, or if the mechanism that down regulates TrkB expression becomes nonfunctional. Such events could trigger an early proliferative event that leads to a cascade of changes that initiates transformation of cells to neuroblastoma. Thus, these early TrkB expressing cells help solve the puzzle as to why TrkB is expressed in aggressive and invasive forms of neuroblastoma, particularly because BDNF induces cultured neuroblastoma cells to become more proliferative, invasive, angiogenic, and resistant to chemotherapeutic reagents than untreated cultures [13]. Future studies will determine whether constitutive expression of BDNF and TrkB in the <span class="yellow">chick</span> embryo sustains proliferation of differentiating sympathoblasts.<br><br>Conclusion
We have identified a time point during development when differentiating lumbar sympathetic neurons transiently express TrkB and proliferate in response to high concentrations of BDNF in culture. These studies suggest that elevated BDNF expression above basal levels and signaling through TrkB may be a mechanism that contributes to the onset of neuroblastoma. A further understanding of the two populations of sympathetic neurons and the fate of the TrkB-positive cells will provide additional insight into the development of paravertebral sympathetic ganglia and the genesis of neuroblastoma.<br><br>Methods
Preparation of tissue for immunohistochemistry
The lumbar spinal column and surrounding tissues were dissected from <span class="blue">chicken</span> embryos at the indicated stages and placed in Zamboni's fixative (4% (w/v) paraformaldehyde, 15% (v/v) picric acid in 0.1 M sodium phosphate buffer, pH 7.4) for two hours at room temperature. <span class="yellow">Mouse</span> embryos at 13–15 days post-coitus were collected according to an IACUC-approved protocol to Dr. L. Sherman at the Oregon Health and Science University. The <span class="yellow">mouse</span> embryos were immersion-fixed in Zamboni's fixative overnight at 4 degrees C then washed with phosphate buffered saline (PBS; 130 mM NaCl, 20 mM sodium phosphate buffer, pH 7.4). Fixed tissues were equilibrated in 30% sucrose in 1× phosphate-buffered saline (PBS). Fixed <span class="yellow">mouse</span> embryos were shipped to Vermont in sucrose. Transverse 30 μM sections of the spinal columns were cut at on a Microm HM cryostat (knife temperature: 16°C; object temperature: 23°C) and collected on Superfrost Plus slides (Fisher). Sections were dried at room temperature, washed in 1× PBS and incubated overnight in blocking buffer (1× PBS consisting of 10% (v/v) heat-inactivated <span class="yellow">horse</span> serum (Invitrogen/Gibco), 0.5% Triton X-100 (Sigma), and 0.1% sodium azide (Fisher)).<br><br>Immunohistochemistry
Sections were incubated with primary antibodies overnight at 4°C, followed by incubation with secondary antibodies for 2 hours at room temperature. Primary antibodies used were: <span class="yellow">rabbit</span> anti-p75 (1:1500, generous gift from Louis Reichardt, UCSF [26]), <span class="yellow">mouse</span> IgG2b anti-Hu C/D, (1:250, Molecular Probes); <span class="yellow">mouse</span> IgG1 anti-Islet-1, (1:10, Developmental Studies Hybridoma Bank); <span class="yellow">rabbit</span> anti-<span class="blue">chicken</span> TrkA (1:500); <span class="yellow">rabbit</span> anti-<span class="blue">chicken</span> TrkB (1:500); <span class="yellow">rabbit</span> anti-<span class="blue">chicken</span> TrkC (1:500) (all Trk antibodies were generous gifts of Dr. Louis Reichardt, UCSF [26-28]); <span class="yellow">mouse</span> anti-HNK-1 (1:50, Developmental Studies Hybridoma Bank); <span class="yellow">mouse</span> IgG2a anti-tyrosine hydroxylase (1:10, Developmental Studies Hybridoma Bank), <span class="yellow">sheep</span> anti-BrdU (1:100, Biodesign International), <span class="yellow">rabbit</span> anti-tyrosine hydroxylase (1:100, Chemicon), and <span class="yellow">goat</span> anti-TrkB (1:1000, R&D Systems). Immunofluorescence was imaged using a Nikon C1 confocal mounted on a Nikon Eclipse E800 microscope with a 10× Plan Apo (NA 0.785) air objective or a 60× Plan Apo (NA 1.4) oil objective lens, E7-C1 software, and UV, Argon, and He/Ne lasers exciting at 408, 488, and 543 nm and emitting at 404 500–530, and 555–615 nm, respectively. A Nikon Eclipse E800 microscope in the nearby COBRE Molecular/Cellular Core Facility was used for counting immunofluorescent cells at 200× using epifluorescence optics.<br><br>RNA Extraction/cDNA synthesis
Sympathetic ganglia were removed from <span class="yellow">chick</span> embryos and RNA was isolated using TriReagent (Molecular Research Center), an acidified guanidinium with phenol extraction method [29]. RNA was transcribed to cDNA using oligo-dT with Superscript II Reverse Transcriptase (Invitrogen) at 42°C for 1 hour.<br><br>Real-time PCR
Relative RNA levels were determined using quantitative real-time PCR with an ABI 7500 Fast Real Time PCR System. TaqMan probes were used to quantify the progression of the PCR reaction and reactions were normalized using the constitutively expressed gene <span class="yellow">chick</span> ribosomal binding protein s17 (CHRPS). The sequences were used for primer/probes sets: for BDNF: forward: 5'-AGCCCAGTGAGGAAAACAAG-3', reverse: 5'-ACTCCTCGAGCAGAAAGAGC-3', probe: 5'-[6-FAM]-TACACATCCCGAGTCATGCTGAGCA-[BHQ]-3'; for CHRPS (<span class="yellow">chick</span> ribosomal binding protein S-17): 5'AACGACTTCCACACCAACAA3', reverse: 5'CTTCATCAGGTGGGTGACAT3', probe: 5'-[6-FAM]-CGCCATCATCCCCAGCAAGA [BHQ]-3'. Primers and probes were synthesized by Operon Technologies, Inc (Alameda, CA). The primers for BDNF were validated against primers for CHRPS according to an Applied BioSystems protocol by serially diluting the target cDNA 1:10, determining the cycle threshold (Ct) for each reaction, and plotting the Ct versus log concentration. Slopes of the resulting lines were calculated and primers were accepted if their Ct slopes were between -3.2 and -3.4 (a perfect efficiency of 1.0 yields a slope of -3.3). To analyze the data, the delta Ct method of relative quantification was used, where the Ct of Chrps was subtracted from the Ct of the gene of interest (Delta Ct) and the arbitrary units of mRNA were expressed as 10000/2^(Delta Ct).<br><br>Cell culture
Sympathetic neurons were cultured as previously described [30] with a few modifications. Sympathetic ganglia were removed from the lumbar region of the paravertebral chain of St. 29/30 (E6.5) <span class="yellow">chick</span> embryos and placed in Modified Puck's solution with glucose (MPG). The cells were dissociated by incubation of sympathetic ganglia with 0.1% trypsin in MPG at 37°C for 10 minutes followed by triturating with a fire polished 9" Pasteur pipette. Cells were then resuspended in Dulbecco's Modified Eagle Medium (DMEM) consisting of 10% <span class="yellow">horse</span> serum, 2% fetal <span class="yellow">calf</span> serum, and 10 mg/ml penicillin/streptomycin. For neurotrophin studies, the culture medium was supplemented with 25 ng/ml NT-3 (R & D Systems) and 1 μg/ml 7S NGF (Alomone Labs) upon plating, and 50 ng/ml, 100 ng/ml, or 200 ng/ml BDNF (R & D Systems) once the cells adhered to the wells. Cells were plated on poly-D-lysine/laminin coated wells or cover slips (Fisher) as previously described [30].<br><br>Quantification of neurons and sympathoblasts using phase microscopy
Embryonic sympathoblasts and neurons are small, phase bright cells with neurites. The total number of cells with neurites the length of two cell bodies were counted in 10 non-overlapping fields of view evenly spaced in a grid-like pattern across the bottom of a well from a 24 well plate at 200× using a Nikon Eclipse TE200 microscope.<br><br>BrdU labeling
For in vitro studies, approximately 2 hours after plating cells from St. 29/30 (E6.5) sympathetic ganglia, cells were labeled with 10 μM bromodeoxyuridine (BrdU, Sigma) for 12 hours at 37°C. Following this labeling period, cells were incubated in complete medium without BrdU for an additional 10 hrs. Cells were then fixed in Zamboni's fixative for 30 min at room temperature and rinsed with 1× PBS. For in vivo studies, 25 μg BrdU was injected into the amnion of <span class="yellow">chick</span> embryos at St. 27. The cells and sections were denatured with 2 N HCl at 37°C for 1 hr, and were then neutralized with 0.1 M borate buffer, pH 8.5, for 10 min at room temperature. Immunochemistry was performed as described above.<br><br>
Abbreviations
BDNF, brain-derived neurotrophic factor; BrdU, Bromodeoxyuridine; DA, dorsal aorta; DMEM, Dulbecco's Modified Eagle's Medium; DRG, dorsal root ganglion; E, embryonic day; HS, <span class="yellow">horse</span> serum; MPG, Modified Puck's solution with glucose; NGF, nerve growth factor; NC, notochord; NT, neural tube; NT-3, neurotrophin-3; NTR, neurotrophin receptor; PBS, phosphate-buffered saline; SC, spinal cord; SCG, superior cervical ganglion; SEM, standard error of the mean; SG, sympathetic ganglion; St., stage; w/v, weight/volume; v/v, volume/volume.<br><br>Authors' contributions
JAS designed the experiments, performed the experiments, analyzed the data, and wrote the manuscript. GLSS contributed intellectually to the conception and design of this study, and assisted in the interpretation of the results. RN supervised the study, participated in the design of experiments, edited the manuscript, and obtained funding for the project. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2636797</h3>Efficacy of intra-articular hyaluronan (Synvisc®) for the treatment of osteoarthritis affecting the first metatarsophalangeal joint of the foot (hallux limitus): study protocol for a randomised placebo controlled trial
Abstract
Background
Osteoarthritis of the first metatarsophalangeal joint (MPJ) of the foot, termed hallux limitus, is common and painful. Numerous non-surgical interventions have been proposed for this disorder, however there is limited evidence for their efficacy. Intra-articular injections of hyaluronan have shown beneficial effects in case-series and clinical trials for the treatment of osteoarthritis of the first metatarsophalangeal joint. However, no study has evaluated the efficacy of this form of treatment using a randomised placebo controlled trial. This article describes the design of a randomised placebo controlled trial to evaluate the efficacy of intra-articular hyaluronan (Synvisc®) to reduce pain and improve function in <span class="yellow">people</span> with hallux limitus.<br><br>Methods
One hundred and fifty community-dwelling <span class="yellow">men</span> and <span class="yellow">women</span> aged 18 years and over with hallux limitus (who satisfy inclusion and exclusion criteria) will be recruited.
<span class="yellow">Participants</span> will be randomised, using a computer-generated random number sequence, to receive a single intra-articular injection of up to 1 ml hyaluronan (Synvisc®) or sterile saline (placebo) into the first MPJ. The injections will be performed by an interventional radiologist using fluoroscopy to ensure accurate deposition of the hyaluronan in the joint. <span class="yellow">Participants</span> will be given the option of a second and final intra-articular injection (of Synvisc® or sterile saline according to the treatment group they are in) either 1 or 3 months post-treatment if there is no improvement in pain and the <span class="yellow">participant</span> has not experienced severe adverse effects after the first injection. The primary outcome measures will be the pain and function subscales of the Foot Health Status Questionnaire. The secondary outcome measures will be pain at the first MPJ (during walking and at rest), stiffness at the first MPJ, passive non-weightbearing dorsiflexion of the first MPJ, plantar flexion strength of the toe-flexors of the hallux, global satisfaction with the treatment, health-related quality of life (assessed using the Short-Form-36 version two questionnaire), magnitude of symptom change, use of pain-relieving medication and changes in dynamic plantar pressure distribution (maximum force and peak pressure) during walking. Data will be collected at baseline, then 1, 3 and 6 months post-treatment. Data will be analysed using the intention to treat principle.<br><br>Discussion
This study is the first randomised placebo controlled trial to evaluate the efficacy of intra-articular hyaluronan (Synvisc®) for the treatment of osteoarthritis of the first MPJ (hallux limitus). The study has been pragmatically designed to ensure that the study findings can be implemented into clinical practice if this form of treatment is found to be an effective treatment strategy.<br><br>Trial registration
Australian New Zealand Clinical Trials Registry: ACTRN12607000654459<br><br><br><br>Background
Osteoarthritis (OA) is a degenerative joint disease that commonly presents within the first metatarsophalangeal joint (MPJ) of the foot. The terms hallux limitus and hallux rigidus have frequently been used interchangeably to describe differing severities of pain and limitation of motion associated with OA at the first MPJ [1]. Hallux limitus is a progressive osteoarthritic condition of the first MPJ that may advance to an end-stage presentation of hallux rigidus where the joint fuses and there is a complete restriction of motion [1]. First MPJ OA is the second most common disorder affecting the foot after hallux valgus [2]. The prevalence of the condition increases with age, and it has been reported that radiographic changes in the first MPJ affect are evident in approximately 46% of <span class="yellow">women</span> and 32% of <span class="yellow">men</span> at 60 years of age [3]. Osteoarthritis at the first MPJ is characterised by the symptoms of pain and stiffness at the joint [1]. Secondary painful symptoms relate to compensations during gait that may occur due to the reduced motion of the first MPJ [1]. The presence of pain associated with first MPJ OA impacts on normal walking and quality of life [4].
Treatment of hallux limitus involves conservative measures (such as physical therapy, foot orthoses, footwear modification, joint manipulation and injection with corticosteroid) [5], or surgical intervention (either joint-salvage or joint-destructive procedures) [6]. Pharmacological treatment is also often undertaken as an adjunct for pain relief in the management of hallux limitus [6]. However, although non-steroidal anti-inflammatory drugs (NSAIDs) and cyclooxygenase-2 inhibitors have been found to be effective in the management of various forms of OA, gastrointestinal complications remain a concern [7]. In light of these limitations with existing treatments, an alternative treatment termed 'viscosupplementation' – the intra-articular injection of hyaluronan into arthritic joints with the aim of restoring the viscoelasticity of the synovial fluid [8] – has been proposed and has attracted considerable attention in the medical literature as a treatment for OA [9]. In particular, both the American College of Rheumatology (ACR) and European League Against Rheumatism (EULAR) recommend hyaluronan in the management of OA of the knee [10,11]. Although the results of systematic reviews investigating the effectiveness of this type of treatment for knee OA are controversial, the most recent update of the Cochrane systematic review evaluating viscosupplementation for the treatment of knee OA concluded that viscosupplementation was both safe and effective for the treatment of OA and was superior or equivalent to any form of systemic intervention or intra-articular corticosteroids [9,12].
Despite there being a large number of studies investigating the effectiveness of hyaluronan for knee OA, few studies have investigated the effects of this form of treatment for OA at the first MPJ [13]. In a case-series retrospective study, 14 <span class="yellow">patients</span> with radiographically confirmed OA at the first MPJ that received up to 3 intra-articular injections of 1 ml hyaluronan (Ostenil® Mini) (sodium hyaluronate) reported a statistically significant reduction in pain (reported using a visual analogue scale) after 6 months [14]. The treatment was well tolerated, with 3/14 (21%) <span class="yellow">participants</span> reporting mild adverse reactions at the injection site. In another study, Pons et al[13] compared a single intra-articular injection of 1 ml Ostenil® Mini (sodium hyaluronate) with 1 ml Trigon depot® (triamcinolone acetonide, a corticosteroid) for the treatment of painful, grade 1 hallux limitus (Karasick and Wapner [15] scale) in 37 <span class="yellow">participants</span> (40 feet) [13]. Both treatment groups showed statistically significant reductions in pain at rest or on palpation for up to 12 weeks post-injection. However, hyaluronan treatment resulted in a statistically significant greater reduction in pain during walking and greater improvement in the American Orthopaedic Foot and Ankle Society (AOFAS) hallux MPJ score compared to treatment with triamcinolone acetonide. The treatment with hyaluronan was well tolerated, with 2/20 (10%) <span class="yellow">participants</span> reporting mild adverse reactions at the injection site.
Although both of these studies suggest that intra-articular hyaluronan is safe and effective for the treatment of hallux limitus, neither used a placebo control group [13,14]. This limitation is significant as a placebo effect can account for 79% of the efficacy of intra-articular hyaluronan treatment [16]. Further, both studies are limited in that neither of the studies used blinding of both the <span class="yellow">participants</span> and assessors in their protocols. It is therefore possible that the positive effects of hyaluronan may have been overestimated. Accordingly, the aims of this project are to conduct a double blind randomised controlled trial to determine the effectiveness of intra-articular hyaluronan (Synvisc®) on (i) foot pain and function; (ii) the range of motion of the first MPJ; (iii) the strength of the plantarflexor muscles of the first MPJ; (iv) the health related quality of life; and (v) the use of pain-relieving medications in <span class="yellow">people</span> with hallux limitus. The study protocol is presented in this paper, consistent with the recommendations of Editorial Board of BioMed Central [17].<br><br>Methods
Design
This study is a parallel group, <span class="yellow">participant</span> and assessor blinded, randomised controlled trial with a 6 month follow-up (Figure 1). It has been developed using the principles described by Osteoarthritis Research Society International (OARSI) Clinical Trials Task Force guidelines [18]. <span class="yellow">Participants</span> will be randomised to receive a single intra-articular injection of up to 1 ml hyaluronan (Synvisc®) or sterile saline (placebo) into the first MPJ. Allocation to either the Synvisc® or placebo groups will be achieved using a computer-generated random number sequence. The allocation sequence will be generated and held by an external <span class="yellow">person</span> not directly involved in the trial. Concealment of the allocation sequence will be ensured as each <span class="yellow">participant</span>'s allocation will be contained in a sealed opaque envelope. Envelopes will be made opaque by using a sheet of aluminium foil inside the envelope. In addition, a system using carbon paper will be employed so the details (name and date of recruitment) are transferred from the outside of the envelope to the paper inside the envelope containing the allocation prior to opening the seal. Assessors and <span class="yellow">participants</span> will be blinded to group allocation. <span class="yellow">Participants</span> will be given the option of a second and final intra-articular injection (of Synvisc® or sterile saline according to the treatment group they are in) on days 30 or 90 if there is no improvement in pain and the <span class="yellow">participant</span> has not experienced severe adverse effects after the first injection).<br><br><span class="yellow">Participants</span>
The <span class="yellow">Human</span> Studies Ethics Committee at La Trobe University (Human Ethics Committee Application No. 07-45) and the Radiation Advisory Committee of the Victorian Department of <span class="yellow">Human</span> Services have given approval for the study. Written informed consent will be obtained from all <span class="yellow">participants</span> prior to their participation. <span class="yellow">People</span> with hallux limitus will be recruited from a number of sources:
(i) advertisements in relevant Melbourne (Australia) newspapers;
(ii) mail-out advertisements to health-care practitioners in Melbourne;
(iii) advertisements using relevant internet web-sites (including );
(iv) posters displayed in local retirement villages, community centres and universities located in Melbourne.
Respondents will initially be screened by telephone interview to ensure they are suitable for the study. Suitable individuals will then be invited to participate in the study and attend an initial assessment.
To be included in the study, <span class="yellow">participants</span> must meet the following inclusion criteria:
(i) be aged at least 18 years;
(ii) report having symptoms of pain, during walking or rest, in the first MPJ for at least 3 months;
(iii) report having pain rated at least 20 mm on a 100 mm visual analogue pain scale (VAPS);
(iv) have pain upon palpation of the dorsal aspect of the first MPJ;
(v) radiographic evidence of OA (score 1 or 2 for either osteophytes or joint space narrowing using a previously published radiographic classification) [19] at the first MPJ.
(vi) able to walk household distances (>50 meters) without the aid of a walker, crutches or cane;
(vii) be willing to attend the La Trobe University Medical Centre (Melbourne, Australia) for treatment with either Synvisc® or placebo (single intra-articular injection) and attend the Health Sciences Clinic at La Trobe University (Melbourne, Australia) for the initial assessment and the outcome measurements (at baseline and 1, 3 and 6 months post-treatment);
(viii) not receive other intra-articular injections into the first MPJ during the course of the study, apart from those dictated by the study;
(ix) be willing to discontinue taking all pain-relieving medications (analgesics and non-steroidal anti-inflammatory medications (NSAIDs), except paracetamol up to 4 g/day, taken by mouth or applied topically):
- for at least 14 days prior to the baseline assessment;
- during the study period (6 months after the final treatment with Synvisc®).
<span class="yellow">Participants</span> who do take paracetamol need to discontinue its use at least 24 hours prior to the baseline assessment and follow-up assessments at 1, 3 and 6 months after the treatment;
(x) be willing to not receive any physical therapy on the involved MPJ or trial of shoe modifications or foot orthoses during the study period.
Exclusion criteria for <span class="yellow">participants</span> in this study will be:
(i) Severe radiographic evidence of OA (score 3 for either osteophytes or joint space narrowing) at the first MPJ using a previously published radiographic classification [19];
(ii) previous surgery on the first MPJ;
(iii) intra-articular steroid, or any other intra-articular injection at the first MPJ in the previous 6 months;
(iv) treatment with systemic steroid (excluding inhalation or topical steroids), immunosuppressives or anticoagulants (except for acetylsalicylic acid at dosages of up to 325 mg/day);
(v) presence of joint infection(s) of the foot;
(vi) significant deformity of the first MPJ including hallux abducto valgus (grade of 3 or 4 scored using the Manchester Scale [20];
(vii) presence of peripheral vascular disease. Peripheral vascular disease will be considered to be present if any of the following are present [21];
▪ past history of, vascular surgery, Raynaud's phenomenon, vasculitis associated with connective tissue diseases, Buerger's disease, arterial emboli, deep vein thrombosis or lower limb ulcers;
▪ history of intermittent claudication or rest pain;
▪ presence of atrophy, ulcers or significant oedema;
▪ inability to palpate at least one pedal pulse;
▪ Ankle Brachial Pressure Index <0.9;
(viii) presence of one or more conditions that can confound pain and functional assessments of the first MPJ, such as metatarsalgia, plantar fasciitis, pre-dislocation syndrome, sprains of the foot, Achilles tendinopathy, degenerative joint disease of the foot (other than the first MPJ) or painful corns and callus;
(ix) planning to undergo any surgical procedure or receive any injections, apart from those dictated by the study, at the involved first MPJ during the study period;
(x) presence of systemic inflammatory condition or infection, such as inflammatory arthritis, diagnosed with rheumatoid arthritis, ankylosing spondylitis, psoriatic arthritis, reactive arthritis, septic arthritis, acute pseudogout, or any other connective tissue disease;
(xi) evidence of gout or other musculoskeletal disease other than OA within the feet. Gout will be screened for using clinical history and physical assessment (painful joint, abrupt onset, swelling), radiographic assessment (asymmetrical joint swelling, subcortical cysts without erosion and tophi) as well as serum uric acid levels (hyperuricaemia = serum uric acid > mean + 2 SD from normal population) [22];
(xii) active skin disease or infection in the area of the injection site;
(xiii) any medical condition that, in the opinion of the investigators, makes the <span class="yellow">participant</span> unsuitable for inclusion (e.g., severe progressive chronic disease, malignancy, bleeding disorder, clinically important pain in a part of the musculoskeletal system other than the first MPJ, or fibromyalgia);
(xiv) pregnant or lactating <span class="yellow">women</span>, or <span class="yellow">women</span> who are of <span class="yellow">child</span> bearing age or have not undergone menopause (Synvisc® has not been tested in pregnant <span class="yellow">women</span> or <span class="yellow">women</span> who are nursing);
(xv) cognitive impairment (defined as a score of < 7 on the Short Portable Mental Status Questionnaire) [23];
(xvi) known hypersensitivity (allergy) to hyaluronan preparations, or to avian proteins, feathers or egg products;
(xvii) involvement in any clinical research study in the previous 3 months that could be considered to affect the results of this study.<br><br>Intra-articular injections for the treatment groups
<span class="yellow">Participants</span> will be randomised to receive a single intra-articular injection of up to 1 ml of hyaluronan (Synvisc®; Genzyme Biosurgery, Genzyme Corporation, NJ, USA) or sterile saline (placebo) into the first MPJ. Each 2 ml ampoule of Synvisc® contains 16 mg of hylan G-F 20 (cross-linked hylan polymers; hylan A and B), 17 mg sodium chloride, 0.32 mg disodium hydrogen phosphate, 0.08 mg sodium dihydrogen phosphate monohydrate. The hyaluronan is extracted from <span class="blue">chicken</span> combs and the purified material has an average molecular weight of 6,000 kDa.
The injections will be performed by the same experienced interventional radiologist (AEZ) using fluoroscopic imaging to ensure accurate deposition of the hyaluronan within the joint. As the Synvisc® is provided in ampoules that are labelled with the product name, it will not be possible to blind the injector, however this <span class="yellow">person</span> is not involved in generation of the allocation order, recruitment, assessment or data analysis. The intra-articular injection will be performed using a 21 gauge (0.80 × 19 mm) Surflo® (Terumo® Corp., Tokyo, Japan) winged infusion set under aseptic procedures. Either a dorso-lateral or dorso-medial approach for injection will be used at the discretion of the injector (depending on which approach provides minimum interference from the osteophytes at the first MPJ joint margins). No anaesthetic will be used. If the <span class="yellow">participant</span> has bilateral painful first MPJs, only one side (the most painful side) will be treated and used for data collection. The injector will record the volume of the agent that is injected.
<span class="yellow">Participants</span> will be given the option of a second and final intra-articular injection (of Synvisc® or sterile saline according to the treatment group they are in) on days 30 or 90 if there is no improvement in pain (assessed using the VAPS for pain during walking or at rest) and the <span class="yellow">participant</span> has not experienced severe adverse effects after the first injection).<br><br>Assessments
Initial assessments
An initial assessment will be performed to determine the eligibility of <span class="yellow">participants</span> for this study. Demographic data will be collected including the age, gender, height and weight of <span class="yellow">participants</span>. Data will also be obtained concerning the presentation of symptoms (foot affected, duration of symptoms). If the <span class="yellow">participant</span> has bilateral painful first MPJs, the most painful side will be used for data collection and subsequent treatment. To establish eligibility for the study, <span class="yellow">participants</span> will undergo a clinical assessment, have one set of dorso-plantar and lateral weight-bearing x-rays taken of their feet to grade the severity of first MPJ OA as well as undergo a blood test to assess serum uric acid levels (to exclude gout).
Weightbearing dorso-plantar and lateral radiographic views will be obtained from both feet with the <span class="yellow">participant</span> standing in a relaxed bipedal stance position. All x-rays will be taken by the same medical imaging department using a Shimadzu UD150LRII 50 kw/30 kHz Generator and 0.6/1.2 P18DE-80S high speed x-ray tube from a ceiling suspended tube mount. AGFA MD40 CR digital phosphor plates in a 24 cm × 30 cm cassette will be used. For dorso-plantar projections, the x-ray tube will be angled 15° cephalad and centered at the base of the third metatarsal. For lateral projections, the tube will be angled 90° and centered at the base of the third metatarsal. The film focus distance will be set at 100 cm [19].<br><br>Baseline assessments and outcome measures
<span class="yellow">Participants</span> who are eligible for the study will be invited to attend a baseline assessment. During the baseline assessment, <span class="yellow">participants</span> will undergo primary and secondary outcome measurements prior to receiving their injection. The outcome measurements have been developed in accordance of the recommendations of the OARSI Clinical Trials Task Force guidelines [18].<br><br>Primary outcome measures
Outcome measurements (primary and secondary) will occur at four time-points at baseline, 1, 3 and 6 months post-treatment (after the intra-articular injection of Synvisc® or placebo). The assessor performing the measurements will be blinded as to which treatment group <span class="yellow">participants</span> have been allocated to. <span class="yellow">Participants</span> who receive a second treatment at day 30 or 90 will be followed for a further 30 days or 90 days respectively and undergo outcome measurements at 7 or 9 months respectively.
The primary outcome measures will be the Pain and Function subscales of the Foot Health Status Questionnaire (FHSQ) [24]. The FHSQ includes 13 questions that assess four domains of foot health, Foot pain, Foot function, Footwear and General foot health. The FHSQ has been subjected to an extensive validation (content, criterion and construct validity) process. It has a high test-retest reliability (intraclass correlation coefficients ranging from 0.74 to 0.92) and a high degree of internal consistency (Cronbach's α ranging from 0.85 to 0.88) [24]. Rigorous reviews have rated it as one of the highest quality foot health status measures currently available [25-27].<br><br>Secondary outcome measures
The secondary outcome measures will be:
(i) Severity of pain
Severity of pain at the first MPJ during walking, and during rest, over the past week will be assessed using a 100 mm visual analogue pain scale. The left side of the scale (0 mm) will be labelled "no pain" and the right side of the scale (100 mm) will be labelled "worst pain possible" for each question [25,28].<br><br>(ii) Severity and duration of stiffness at the first metatarsophalangeal joint
The severity of stiffness at the first MPJ during walking over the past week will be assessed using a 100 mm visual analogue scale. The left side of the scale (0 mm) will be labelled "not stiff at all" and the right side of the scale (100 mm) will be labelled "most stiff possible". The average duration of stiffness at the first MPJ over the past week will be assessed using a four category scale response. The responses are: "none", "1–15 minutes", "16–30 minutes" and "greater than 30 minutes" [29].<br><br>(iii) Passive, non-weightbearing dorsiflexion range of motion of the first metatarsophalangeal joint
First MPJ dorsiflexion range of motion will be measured using a goniometer as the maximum angle at which the hallux cannot be passively moved into further extension in a non-weightbearing position (Figure 2) [30]. The test will be performed two times and the average will be used for analysis. This measurement technique shows high intra-reliability (ICC = 0.95, standard error of mean = 1.3°) [30].<br><br>(iv) Plantar flexion strength of the toe-flexors of the hallux
Plantar flexion strength of the toe-flexors of the hallux will be measured using the Mat Scan® plantar pressure measurement device [31]. <span class="yellow">Participants</span> will be seated with the hip, knee, and ankle at 90 degrees and their foot placed over the Mat Scan® plantar pressure measurement device (Tekscan, Boston, MA, USA) (Figure 3a). This system consists of a 5-mm thick floor mat (432 × 368 mm) incorporating 2288 resistive sensors (1.4 sensors/cm2) sampling at a rate of 40 Hz. The mat will be calibrated for each <span class="yellow">participant</span> using his or her own bodyweight before each testing session. <span class="yellow">Participants</span> will be instructed to use their toe-flexor muscles to maximally push their hallux down on the MatScan® device and forces under the hallux will be recorded (Figure 3b). The test will be performed three times for the hallux and the maximal force will be used for analysis. The test-retest reliability of this measurement technique has previously been shown to be high, with intraclass correlation coefficients (ICCs) = 0.88 (95% CI 0.81 – 0.93) [31].<br><br>(vi) Plantar pressure measurement
Plantar pressures will be recorded during level barefoot walking using the MatScan® system (Tekscan®, Boston, MA, USA). The two-step gait initiation protocol will be used to obtain foot pressure data, as it requires fewer trials than the mid-gait protocol and has similar re-test reliability [32]. Three trials will be recorded, which has been found to be sufficient to ensure adequate reliability of pressure data [32,33]. Following data collection, the Research Foot® software (version 5.24) will be used to construct individual "masks" to determine maximum force (kg) and peak pressure (kg/cm2) under seven regions of the foot: hallux, lesser toes, 1st MPJ, 2nd MPJ, 3rd to 5th MPJs, midfoot and heel (Figure 4a). For each region, the median of the three trials will be used for analysis. Typical plantar pressure recordings from a <span class="yellow">participant</span> are shown in Figure 4b.<br><br>(vi) Global satisfaction with the treatment
Global satisfaction with the treatment will be assessed using a 5-point Likert scale, as well as a dichotomous (yes/no) scale. The five point-Likert scale will ask "How satisfied are you with the treatment you received for your big-toe joint pain?", and will have the following five responses: "Dissatisfied", "Only moderately satisfied", "Fairly satisfied", "Clearly satisfied" and "Very satisfied". The dichotomous scale of satisfaction will be answered as "Yes"' or "No" in response to the question: "Would you recommend the treatment that you received to someone else with big-toe joint pain".<br><br>(vii) Health related quality of life
The Short-Form-36 (version two) (SF-36) questionnaire will be used to assess health related quality of life. The SF-36 is a 36 question survey that measures eight health concepts most affected by disease and treatment. The eight health concepts can then be used to form two summary measures: Physical health and Mental health. The Short Form-36 (SF-36) has been extensively validated and is one of the most widely used instruments to measure health status. The SF-36 shows content, concurrent, criterion, construct, and predictive evidence of validity. The reliability of the eight concepts and two summary measures has been assessed using both internal consistency and test-retest methods. Reliability statistics have exceeded 0.80 [34-37].<br><br>(viii) Self-reported magnitude of symptom change
Self-reported magnitude of symptom change will be measured using a 15-point Likert scale. The scale will ask <span class="yellow">participants</span> "how much have your symptoms in your big-toe joint have changed from the beginning of the study to now?". The fifteen responses will range from "A very great deal better" to "A very great deal worse".<br><br>(ix) Use of rescue medications to relieve pain at the first metatarsophalangeal joint
The number of <span class="yellow">participants</span> who consumed rescue medication (e.g., paracetamol) and mean consumption of rescue medication to relieve pain at the first MPJ (mean grams of paracetamol/<span class="yellow">participant</span>/month] will be assessed using a medications diary that <span class="yellow">participants</span> will self-complete [38,39]. The diary will be returned to the assessor at monthly intervals for analysis.<br><br>(x) Frequency and severity of adverse events as safety variables
The frequency (number of <span class="yellow">participants</span> affected and number of cases) and types of adverse events (including adverse drug reactions) in each treatment group during the trial will be recorded using a questionnaire that <span class="yellow">participants</span> will complete during the follow-up appointments at 1, 3 and 6 months post-treatment [40]. To classify the 'type' of adverse event, a blinded assessor will classify the adverse event as being serious or non-serious [40]. Any serious adverse events, defined as adverse events leading to serious disability, hospital admission, or prolongation of hospitalisation, life-threatening events; or death) will be further classified using the International Classification of Diseases (ICD) codes [41]. Non-serious adverse events will include both local (pain, effusion and heat, with each classified as mild, moderate, severe) and systemic adverse events. An open-response type format will also be available for <span class="yellow">participant</span> responses.<br><br><br><br>Sample size
The sample size for the study has been pre-specified using an a priori power analysis using the primary outcome measure of the pain domain of the FHSQ [42]. One hundred and forty two <span class="yellow">participants</span> (i.e. 71 per group) will provide power of 90% to detect a minimally important difference in the pain domain of the FHSQ (i.e. 14 points on the FHSQ questionnaire) with the significance level set at p < 0.05. A difference of 14 points was determined to be a clinically significant difference worth detecting [43] and a standard deviation of 25 was derived from a previous report [44]. This calculation included a 5% drop-out rate [13]. However, we will aim to recruit 150 <span class="yellow">participants</span> (~75 <span class="yellow">participants</span> per intervention group). Further, we have conservatively ignored the extra precision provided by covariate analysis when estimating the sample size.<br><br>Statistical analysis
Statistical analysis will be undertaken using SPSS version 14.0 (SPSS Corp, Chicago, Ill, USA) and STATA 8 (Stata Corp, College Station, Tex., USA) statistical software. All analyses will be conducted on an intention-to-treat principle using all randomised <span class="yellow">participants</span> [45-47]. Missing data will be replaced with the last score carried forward [48]. Standard tests for normal distribution will be used and transformation carried out if required.
Demographic characteristics (gender, age, weight, height, body mass index) will be determined for the baseline visit for each treatment group. Summary statistics will be calculated for duration of symptoms, side affected (left, right, bilateral), grade of OA at the first MPJ [19] as well as all primary and secondary outcome measurements for each treatment group.
Analyses will be conducted on 1, 3 and 6 month outcome measures. The continuously-scored outcome measures at 1, 3 and 6 months will be compared using analysis of covariance with baseline scores and intervention group entered as independent variables [49,50]. The exception to this will be the plantar pressure measurements which will be analysed at baseline, 1, 3 and 6 months using two-way repeated measures analysis of variance statistics. Post-hoc comparisons will be performed using Bonferroni-adjusted t-tests. Nominal and ordinal scaled data will be compared at 1, 3 and 6 months using Mann-Whitney U-tests and chi-square analyses (or Fisher's exact test where appropriate) respectively. Effect sizes will be determined using Cohen's d (continuous scaled data) or odds ratios (nominal scaled data and ordinal scaled data) as appropriate.
The outcome measurements obtained at 7 or 9 months for <span class="yellow">participants</span> that receive a second and final intra-articular injection (of Synvisc® or sterile saline according to the treatment group they are in) on days 30 or 90 respectively, will also be analysed as described above. These analyses will be classified as secondary outcomes.<br><br>
Discussion
This study is a randomised placebo controlled trial designed to investigate the efficacy of intra-articular hyaluronan (Synvisc®) to reduce pain and improve function in <span class="yellow">people</span> with OA of the first MPJ (hallux limitus). Two studies have previously investigated the efficacy of intra-articular hyaluronan for the treatment of first MPJ OA [13,14]. However, neither of these studies used a placebo control group. To our knowledge, this is the first randomised controlled trial using intra-articular hyaluronan for OA of the first MPJ.
The use of a placebo control group is essential for studies evaluating the effects of intra-articular therapies as there is likely to be a large placebo response related to the injection procedure and this may inflate the results in uncontrolled evaluations [51]. Indeed, a recent meta-analysis of hyaluronan for knee OA concluded that a placebo effect accounted for 79% of the efficacy of intra-articular hyaluronan [16].
The study protocol and outcome measures have been developed in accordance of the recommendations of the OARSI Clinical Trials Task Force guidelines [18]. The outcome measures are pain and function subscales of the FHSQ, pain and stiffness at the first MPJ, range of motion (dorsiflexion) of the first MPJ, plantar flexion strength of muscles of the first MPJ, generic health related quality of life (SF-36), <span class="yellow">patient</span> satisfaction with treatment, consumption of rescue medication as well as frequency and nature of adverse effects. These outcomes will be measured at baseline then at 1, 3 and 6 months after treatment. Previous research suggests that the effects of intra-articular hyaluronan persist for up to 12 months following treatment [9,38]. Thus, the use of follow-up assessments at 6 month post-treatment will allow us to determine if the effects, if any, of intra-articular hyaluronan persist in the longer term.
<span class="yellow">Participants</span> will be given the option of a second and final intra-articular injection (of Synvisc® or sterile saline according to the treatment group they are in) on days 30 or 90 if there is no improvement in their symptoms. Although this has the potential to complicate the interpretation of the results of the study, this protocol was included as it is likely to be more reflective of clinical practice [14], and this is in keeping with the pragmatic nature of this trial.
In summary, this project is the first randomised controlled trial to be conducted to evaluate the efficacy of intra-articular hyaluronan for reducing pain and improving function in <span class="yellow">people</span> with hallux limitus. The study protocol, including interventions, have been pragmatically designed to ensure that the study findings are generaliseable to clinical practice. Recruitment for the study will commence in June 2008, and we expect final results to be available in mid-2010.<br><br>Competing interests
HBM and KBL are Editor-in-Chief and Deputy Editor-in-Chief, respectively, of Journal of Foot and Ankle Research. It is journal policy that editors are removed from the peer review and editorial decision making processes for papers they have co-authored.<br><br>Authors' contributions
SEM, HBM, KBL and CJH conceived the idea and obtained funding for the study. SEM, HBM, KBL, AEZ and JDL designed the trial protocol. SEM, HBM, KBL and GVZ drafted the manuscript. All authors have read and approved the final manuscript.<br><br>
<h3>pmcA150219</h3>Rap1p and other transcriptional regulators can function in defining distinct domains of gene expression
Abstract
Barrier elements that are able to block the propagation of transcriptional silencing in <span class="yellow">yeast</span> are functionally similar to chromatin boundary/insulator elements in metazoans that delimit functional chromosomal domains. We show that the upstream activating sequences of many highly expressed ribosome protein genes and glycolytic genes exhibit barrier activity. Analyses of these barriers indicate that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p may participate in barrier function. We also present evidence suggesting that Rap1p is directly involved in barrier activity, and its barrier function correlates with local changes in chromatin structure. We further demonstrate that tethering the transcriptional activation domain of Rap1p to DNA is sufficient to recapitulate barrier activity. Moreover, targeting the activation domain of Adr1p or Gcn4p also establishes a barrier to silencing. These results support the notion that transcriptional regulators could also participate in delimiting functional domains in the genome.<br><br>INTRODUCTION
The eukaryotic genome is organized into discrete functional domains in which gene expression is either permitted or repressed. Domains permissive for gene expression are usually defined as euchromatin or active chromatin, and domains that repress gene expression heterochromatin or silent chromatin. The HML and HMR loci in <span class="yellow">Saccharomyces cerevisiae</span> are silent chromatin domains (1). Compared to active chromatin, silent chromatin is more compact and its histone components are hypoacetylated (2). The SIR complex consisting of Sir2p through Sir4p is an integral part of <span class="yellow">yeast</span> silent chromatin. Sir3p and Sir4p interact with the N-terminal tails of histones H3 and H4 providing the structural basis of silent chromatin (1). Sir2p was recently found to be an NAD-dependent histone deacetylase and was suggested to be responsible for histone hypoacetylation in silent chromatin (3). Silent chromatin is initiated at small cis-acting DNA elements known as the E and I silencers flanking each of the HM loci (4).
Increasing evidence indicates that histone deacetylation is key to the propagation of silent chromatin (4). In vertebrates and <span class="yellow">fission yeast</span><span class="yellow">yeast</span>, spread of silent chromatin involves a chain of events of histone H3 deacetylation → H3 methylation → binding of methylated H3 by HP1 (heterochromatin protein 1) or swi6 (4). In <span class="yellow">S.cerevisiae</span>, there is evidence that Sir3p (hence the SIR complex) has much higher affinity to unacetylated histone H4 than acetylated H4 (5). Based on this and the fact that Sir2p is a histone deacetylase, a refined model for the propagation of silent chromatin can be proposed. In this model, Sir2p recruited to the silencer deacetylates histones in an adjacent nucleosome enabling it to bind another SIR complex with high affinity. The nucleosome-bound SIR complex in turn deacetylates the neighboring nucleosome allowing it to recruit a new SIR complex. In this manner, the SIR complex promotes its own propagation along an array of nucleosomes (4).
The fact that transcriptionally silent chromatin can encroach upon active chromatin poses the question of how interspersed domains of silent and active chromatin are demarcated. Studies of the <span class="yellow">Drosophila</span> and vertebrate genomes demonstrated that some domains are delimited by boundary or insulator elements (6). These elements are specialized DNA sequences that act as barriers to enhancers and/or silent chromatin. One of the best characterized insulators is the <span class="blue">chicken</span> HS4 insulator at the β-globin locus. Histones surrounding this insulator are hyperacetylated indicating the presence of histone acetyltransferase (HAT) activity at the insulator (6). Recently, sequences that could block the spread of silent chromatin have been discovered in <span class="yellow">S.cerevisiae</span> (7,8). These sequences (referred to as silent chromatin barriers) include sub-telomeric anti-silencing regions (STARs), the right boundary of HMR, and the upstream activating sequence (UAS) of the highly expressed TEF2 gene (9–11). The mechanisms underlying barrier functions in <span class="yellow">yeast</span> are not known.
TEF2-UAS contains three tandem repressor activator protein 1 (Rap1p)-binding sites that are necessary and sufficient for its barrier function (11). Rap1p is an abundant sequence-specific DNA-binding protein (12). Variants of the 13 bp consensus sequence for Rap1p binding (ACACC CRYACAYM) (13) lie not only in the UASs of numerous genes but also in the silencers of the HM loci and in telomeric C1–3A repeats. Accordingly, Rap1p functions as a global regulator of transcriptional activation and repression, silencing, as well as telomere length (12). Rap1p performs these diverse functions by interacting with different factors in respective contexts. As an activator, Rap1p binds to the promoters of 362 ORFs (13) and can function via at least three mechanisms. First, Rap1p binding can ‘open up’ local chromatin structure at a promoter to help another activator to bind (14). Secondly, Rap1p bound to DNA can help Gcr1p bind to an adjacent site in the promoter through physical interaction (15). Thirdly, Rap1p is involved in the recruitment of the NuA4 HAT complex and TFIID to ribosome protein genes (RPGs) (16,17). As a silencing factor, Rap1p binds to silencers or telomeric repeats and recruits Sir3p/Sir4p through direct interactions (18). Rap1p also interacts with telomere-specific proteins in executing its role in regulating telomere length (19). It is puzzling that a protein that helps establish silent chromatin could also act as a barrier to its propagation.
In this study, we found that in addition to TEF2-UAS, UASs of many <span class="yellow">yeast</span> genes exhibit barrier activity. Analyses of these UASs indicate that binding sites for transcriptional regulators Abf1p, Reb1p, Adr1p, Gcn4p, as well as Rap1p, may participate in barrier function. These UASs are dispersed across the genome and might play a role in defining functional chromosomal domains. Moreover, we demonstrated that targeting the activation domain of Rap1p, Adr1p or Gcn4p alone, led to barrier function. Since the activation domain of an activator was usually involved in interacting with co-activators and/or components of the transcriptional machinery, these data indicated that other factors might also be involved in barrier function.<br><br>MATERIALS AND METHODS
Plasmids and strains
Plasmids 1–53 were used to make strains 1–53, respectively (see Figs 1–6). Plasmid 1 was previously named pMB22-a that contained a HindIII–BamHI fragment of chromosome III (coordinates 14838–16263) with a URA3 gene inserted at its EcoRV site (20). Plasmid 2 was derived from 1 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SnaBI site. Plasmids 3–38 were identical to 2 except that different fragments from various promoters (see Figs 1 and 2) were inserted at the SnaBI site. Plasmid 29 had a sequence of chromosome III (295327–295713) containing the HMR-tRNA gene (21) inserted at the SnaBI site of plasmid 1. Plasmids 39 and 40 were previously named pYXB26 and pYXB59, respectively (11). Plasmid 41 was identical to pYXB59 except bearing mutations illustrated in Figure 3A. Plasmids 42–48 were previously described as pYXB29, 48, 28, 27, 59, 31 and 37, respectively (11). Plasmid 50 was derived from plasmid 1 by inserting two copies of a sequence bearing the consensus binding sequence of LexA (bold) (22), GGGGTACGTACTGTATGTACATACAGGATATCGGGG, at the SnaBI site. Plasmid 51 was identical to 50 except having only one copy of the LexA-binding sequence. Plasmid 52 was derived from pYXB26 (11) by inserting a SpeI-ADE2-AvrII sequence at the SpeI site. Plasmid 53 was derived from plasmid 52 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SpeI site.
Plasmid L1 (see Fig. 5A) was made by inserting LEU2 into pBTM116 (23) carrying the LexA gene flanked by the promoter and terminator of ADH1. L2 through L8 were derived from L1 by fusing various sequences to LexA (see Fig. 5A and B). Plasmid pRS425 (24) carried the 2 µm origin and LEU2.
Strain YXB76 was MATa ura3-52 leu2-3,112 ade2-1 lys1-1 his5-2 can1-100, E-HMLα-Iinverted (20). Y2047b was MATa HMRa HMLα EΔ79–113::SUP4-o IΔ242 LEU2-GAL10-FLP1 ura3-52 ade2-1 lys1-1 his5-1 can1-100 [cir0] (25). Strain 1 was made by transforming YXB76 to Ura+ with HindIII + BamHI digested plasmid 1. Strains 2–38, 50 and 51 were similarly made with corresponding plasmids, respectively (e.g. strain 2 was made with plasmid 2). Strains 39, 40 and 42–48 were previously described as YXB26, 48, 29, 48, 28, 27, 59, 31 and 37, respectively (11). Strains 41, 52 and 53 (see Figs 3C and 6) were made by transforming Y2047b to canavanine-resistant by BamHI-digested plasmids 41, 52 and 53, respectively. Strains 40′–48′ were sir3::URA3 derivatives of strains 40–48, respectively. The relevant genotypes of all the strains were confirmed by Southern blotting.<br><br>Quantitative mating assay
Quantitative mating was performed as described (11).<br><br>Electrophoresis mobility shift assay (EMSA)
Rap1p was expressed in <span class="yellow">Escherichia coli BL21 (DE3)</span> from pL3S5, a pET vector carrying the RAP1 gene downstream of a T7 promoter (26). Crude protein extracts were prepared from 40 ml cultures harvested after 3 h of IPTG induction and resuspended in 1 ml of lysis buffer (50 mM NaH2PO4, 300 mM NaCl, pH 8.0). A control extract was prepared from a culture without IPTG induction. Western blotting revealed that Rap1p protein was present in the induced extract but not the uninduced one (data not shown). DNA fragments of 26 bp in length (see Fig. 3A) were end-labeled with 32P-phosphate. Fifteen microliter binding reactions were prepared, each consisting of 20 000 c.p.m. radiolabeled DNA probe, 5 µg of crude protein extract, non-specific competitor DNAs (0.3 µg of <span class="yellow">yeast</span> tRNA, 0.3 µg of <span class="yellow">salmon</span> sperm DNA and 0.5 µg of <span class="yellow">E.coli</span> DNA), 10 µg of BSA, 10 mM Tris–HCl (pH 8.0), 10 mM MgCl2 and 8% glycerol. The reactions were incubated at 25°C for 20 min and loaded onto non-denaturing polyacrylamide gels (4–5%). Gels were run at 40 mA for 1 h in 0.5× TBE buffer.<br><br>Analysis of the supercoiling of DNA circles
Cells were grown in YPR (<span class="yellow">yeast</span> extract + bacto peptone + 2% raffinose) to early log phase. Galactose (2%) was then added to the culture to induce the expression of FLP1. After 2.5 h of incubation, nucleic acid was isolated using the glass bead method and fractionated on agarose gels with 30 µg/ml chloroquine. DNA circles were detected by Southern blotting.<br><br>
RESULTS
UASs of many highly expressed genes can function as barriers to the spread of silencing
We have previously shown that TEF2-UAS was able to block the spread of silencing in a silencer-blocking assay (11). Such an assay tested if a sequence had the ability to prevent a silencer from silencing a reporter gene when it was positioned between the silencer and the reporter. In this report we used a new silencer-blocking assay to identify more barrier elements. This assay employed the URA3 gene and an inverted HML-I that could silence sequences to the right of HML (20) (Fig. 1, strain 1). URA3 expression makes cells sensitive to 5-fluoroorotic acid (FOA) thus URA3 silencing could be measured by cell growth on FOA medium (Fig. 1, strain 1). As expected, TEF2-UAS and another known barrier element, HMR-tRNA gene (21) exhibited barrier activity in this assay as indicated by the lack of silencing of URA3 in strains 2 and 29 (Fig. 1).
TEF2 encoding translation elongation factor 1α is one of the most highly expressed genes in <span class="yellow">yeast</span> (27). Other highly expressed genes include ribosome protein genes (RPGs) and genes coding for glycolytic enzymes (27). The coordinated regulation of most of these genes requires the so-called general regulatory factors Rap1p, Abf1p and Reb1p (28,29). Using the above silencer-blocking assay, we tested if the UASs of other highly expressed genes could also block URA3 silencing (Fig. 1). Seventeen new barrier elements were identified from a total of 26 UASs tested. These 17 elements were UASs of eight RPGs (see Fig. 1, strains 3–10), eight glycolytic genes (strains 18–25) and the HIS3 gene (strain 26). All 17 elements except ADH2-UAS and HIS3-UAS exhibited very strong barrier activity as evidenced by the absence of URA3-silencing in strains 3–10 and 18–24 (Fig. 1).
Seven of the eight RPG-UASs that had barrier activity, contained a single or a pair of tandem Rap1p-binding sites (Fig. 1, strains 3–9). Three of them also contained an Abf1p or Reb1p site (strains 6, 8 and 9). One barrier RPG-UAS (strain 10) had only a predicted Abf1p site. Of the seven RPG-UASs that had no barrier activity (Fig. 1, 11–17), only two bore no site for Rap1p, Abf1p or Reb1p (11 and 12). The remaining five each contained one to three Rap1p sites (strains 13–17). RPL15A-UAS (17) also contained an Abf1p site. For most of the 15 RPG-UASs tested (3–17), the presence or absence of predicted Rap1p sites correlated with the presence or absence of Rap1p association in vivo as previously examined by chromatin immunoprecipitation assays (13). Exceptions were the UASs of RPS24B, RPS28A and RPL15A (13).
Of the eight UASs of glycolytic genes tested (Fig. 1, strains 18–25), seven exhibited strong barrier activity (18–24) and one had weaker but detectable barrier activity (strain 25). Glycolytic genes are among the most highly expressed genes and their high expression depends on the functions of Rap1p, Abf1p or Reb1p as well as Gcr1p (29,30). The seven strong glycolytic barriers all consist of one or two Rap1p sites plus one or two Reb1p or Abf1p sites (Fig. 1, strains 18–24). In addition, one or more Gcr1p-binding sites are found adjacent to each Rap1p site. It was proposed that the function of Rap1p was to facilitate the binding of Gcr1p to the UAS (15). Abf1p or Reb1p was shown to play a role similar to that of Rap1p in activating glycolytic genes (31–33).
Although the above data reinforced the notion that certain Rap1p-binding UASs could serve as barriers to silencing, they raised new questions about the essential components of a barrier element. For example, some of the UASs that contained only a single Rap1p site had barrier activity (e.g. RPL19B-UAS, strain 3) whereas others that contained two or three sites had no activity (e.g. RPL39, strain 16). One explanation was that sequences flanking the Rap1p site(s) in a particular UAS could positively or negatively influence barrier function. Moreover, the presence of Abf1p or Reb1p sites in 11 of the 17 new barriers made us wonder if they also participated in barrier function. We began to address these issues by analyzing three new barrier elements (Fig. 2). RPS10A-UAS containing an Abf1p-binding site and a pair of Rap1p sites was divided into three fragments that were tested in a silencer-blocking assay (Fig. 2, strains 30–32). It was clear that only the fragment containing Rap1p sites functioned as a barrier (strain 31). TPI1-UAS bearing a Reb1p site and a Rap1p site was divided into two parts with one containing the Rap1p site (Fig. 2, strain 34) and the other Reb1p site (strain 33). Neither of these fragments alone retained barrier activity indicating that concerted action of Reb1p and Rap1p was required for barrier activity. Duplicating the Reb1p-bearing part of TPI1-UAS didn’t restore barrier activity (strain 35). However, triplicating the Rap1p-containing part did re-create barrier function (strain 36). RPS28A-UAS bore, in addition to a Rap1p site, an Abf1p site and an adjacent T-rich region that are required for efficient transcription of RPS28A (34). A sequence of RPS28A-UAS deleted for a fragment bearing the Rap1p site lost the barrier activity (Fig. 2, strain 37). However, duplicating this sequence restored barrier activity (strain 38). The above results indicated again that Rap1p sites could form barriers (strains 31 and 36). On the other hand, they also suggested that synergistic actions of Abf1p sites (strain 38), Reb1p site + Rap1p site (strain 18), or Abf1p site + Rap1p site (strain 16) could all lead to barrier function. This was not very surprising since Rap1p, Abf1p and Reb1p could perform similar and sometimes interchangeable functions in gene regulation (16,35,36). Note, however, we haven’t ruled out the possibility that other factors (e.g. Gcr1p) or structure features of DNA (e.g. T-tracks) were also involved in barrier function.
It was interesting that ADH2-UAS and HIS3-UAS bearing no site for Rap1p, Abf1p or Reb1p exhibited detectable barrier activity (Fig. 1, strains 25 and 26). However, binding sites for other transcriptional activators existed in these UASs. The major regulator that binds to ADH2-UAS is Adr1p (37). On the other hand, Gcn4p binds to multiple sites in HIS3-UAS and activates HIS3 transcription (38). The possible involvement of Adr1p or Gcn4p in barrier function was examined below. Note Adr1p function was subject to glucose repression (37), which might explain why ADH2-UAS only exhibited limited barrier activity since glucose was used in the media used in our assay.<br><br>Effect of mutating Rap1p-binding sites in TEF2-UAS on barrier function
Although we had shown that the three Rap1p-binding sites in TEF2-UAS were necessary and sufficient for its barrier activity (11), we had not shown if binding of Rap1p to these sites was required. To address this we performed mutational analysis of TEF2-UAS. As shown in Figure 3A and B, the three Rap1p-binding sites in TEF2-UAS (designated as R1, R2 and R3) were all able to bind Rap1p in an EMSA (Fig. 3B). Single or double C→A mutations were introduced into R1, R2 and R3 resulting in m1, m2 and m3, respectively (Fig. 3A). The m2 and m3 sites had both lost the ability to bind Rap1p, whereas m1 still bound Rap1p (Fig. 3B), which could be predicted from the specific mutations in each site (Fig. 3A, compare m1, m2 and m3 to the consensus sequence). We then tested a mutated TEF2-UAS containing m1, m2 and m3 in a silencer-blocking assay using the HML-E silencer and the HMLα genes (Fig. 3C). In this assay, the HML-I silencer was deleted but the HML-E silencer was sufficient to silence the HMLα genes (Fig. 3C, strain 39) (11). Silencing in strain 39 (a-type) was measured by its ability to mate with an α-type strain (Fig. 3C). As predicted, the wild-type TEF2-UAS blocked HMLα silencing (11) (Fig. 3C, strain 40). On the other hand, the mutated TEF2-UAS containing m1, m2 and m3 had lost barrier activity (Fig. 3C, compare the mating efficiencies of strains 41 and 40). Therefore, mutations that prevented Rap1p from binding two of the three sites in TEF2-UAS abolished its barrier activity, indicating that association of Rap1p with TEF2-UAS at more than one site was required for barrier function.<br><br>The barrier TEF2-UAS alters local chromatin structure
In eukaryotic cells, the topology of local DNA reflects the chromatin structure in which it resides. We had previously developed a DNA topology-based assay to examine chromatin structure in vivo, which involved excising a chromosomal region of interest from its genomic location as a circle using site-specific recombination (39). Using this strategy, we demonstrated that DNA in silenced chromatin was more negatively supercoiled than that in active chromatin (39). In this report we used the topology assay to address if barrier elements altered local chromatin structure as <span class="yellow">Drosophila</span> or <span class="blue">chicken</span> insulators did (40,41).
Various sequences (designated X) from UASs of TEF2, AgTEF (TEF gene from <span class="yellow">Ashbya gossypii</span>) and ADE2 were inserted between HML-E and HMLα at HMLΔI (HML locus deleted for the I silencer) flanked by the FRT sites for the Flp1p site-specific recombinase (Fig. 4A and B). The FLP1 gene was under the control of the inducible GAL10 promoter (39). The mating efficiency of each strain measured silencing of HMLα (Fig. 4C). In strains 40, 42, 44 and 47, silencing was restricted to the left of the X insertions and HMLα was derepressed (Fig. 4C). Consistently, the proportion of silent chromatin in the region flanked by the FRT sites in each of these strains was decreased, as indicated by the reduced negative supercoiling of the excised circle (11) (data not shown). Note that the above effect of a barrier on HML chromatin reflected both change in the scope of silencing caused by the barrier activity and alteration (if any) in chromatin structure at the barrier independent of the silencing state of the locus. To exclusively examine the latter, we analyzed the topology of HML DNA in a silencing-deficient background. The SIR3 gene in strains described in Figure 4B was disrupted rendering them defective in silencing (strains 40′–48′ in Fig. 4D). Recombination mediated by Flp1p in these strains led to the excision of a group of chromosomal circles that differed only in the X insertion. The supercoiling of these circles was analyzed as a function of the length of the insertion (Fig. 4D). If an insertion simply lengthened a circle without causing abnormal changes in nucleosomal structure and/or density, the topoisomers of this circle would migrate slower relative to those of a similar circle with a smaller size. Otherwise, migration of the topoisomers would deviate from that predicted based on the size of the circle. As shown in Figure 4D, the circle from strain 40′ bearing a 104 bp barrier element migrated faster rather than slower than the circle from strain 46′ bearing a 91 bp non-barrier element. Circles bearing other barrier elements also exhibited unexpected faster migration (compare strains 44′ to 45′, 47′ to 43′, 42′ to 48′, respectively). Therefore, these data indicated that barrier function correlated with altered topology of local DNA. This was further demonstrated by that mutations in TEF2-UAS that abolished its barrier activity also abolished its effect on DNA topology (Fig. 4D, compare 41′ to 40′; note that circles from both strains had the same size). Since in our gel assay more negatively supercoiled DNA migrate slower, we concluded that every fragment with barrier activity caused a reduction in negative supercoiling of approximately one to two turns in HMLΔI DNA (Fig. 4D, compare the centers of distributions of topoisomers in pairs of lanes, e.g. 40′ and 41′).
The topology of eukaryotic DNA reflects mainly the wrapping of DNA into nucleosomes (approximately one negative superhelical turn is associated with each nucleosome) (42). Therefore, removal of one nucleosome would eliminate approximately one turn. On the other hand, histone acetylation can reduce negative supercoiling associated with a nucleosome by approximately 0.2 turns (43). Therefore, acetylating five nucleosomes would reduce negative supercoiling by approximately one turn. Consequently, the linking number change of one to two brought about by TEF2-UAS can be accounted for by either the loss of one to two nucleosomes or acetylation of five to 10 nucleosomes. Further experiments are under way to clarify the nature of change in chromatin induced by a barrier.<br><br>Targeting the transcription activation domain of Rap1p recapitulates its barrier activity
The multifunctional Rap1p can be divided into at least four functional domains (Fig. 5A) (12). We were interested in defining which of these domains might be involved in the barrier activity of Rap1p. To this end, we fused various parts of Rap1p to the bacterial LexA protein and tested if targeting any of the fusion proteins to LexA-binding sites could re-create a barrier to silencing (Fig. 5A). The LexA-RAP1 fusion genes were carried on 2 µm based plasmids marked by LEU2 (designated L1–L6, Fig. 5A). When introduced into strain 50 in which two LexA-binding sites had been inserted between the inverted HML-I and URA3, LexA alone had no effect on URA3 silencing (Fig. 5A, plasmid L1). Targeting the DNA binding domain, DNA bending domain, silencing domain, or the activating + silencing domain of Rap1p also did not affect URA3 silencing (L2–L5). On the other hand, targeting the activation domain alone recapitulated the barrier activity of Rap1p (L6). These results indicated that the activation domain of Rap1p was involved in barrier activity. The inability of the Rap1p-silencing domain to block URA3 silencing in our assay did not necessarily mean that this domain had no barrier function, since such a function might be suppressed by much stronger silencing activity associated with the silencing domain.
Was the lack of barrier function of fusion constructs L2–L5 due to a lack of expression of these proteins? To address this possibility, we examined the levels of L1–L6 proteins as well as L7 and L8 proteins described in Figure 5B by western blotting using an antibody against LexA. As shown in Figure 5C, L5 was expressed at a level comparable to L6, and L4 was expressed at a higher level than L6. Therefore, the lack of barrier function of L4 and L6 was not the result of their lack of expression. On the other hand, the level of L2 or L3 was significantly lower than that of L6, hence the lack of barrier function of L2 and L3 might be due to their insufficient expression.
As an important control for the above targeting experiments, we showed that plasmids L1–L6 (Fig. 5A) as well as L7 and L8 (Fig. 5B) bearing LexA-fusion genes had no effect on URA3 silencing in strain 1 (Fig. 1) in which there was no binding site for LexA present between HML-I and URA3 (data not shown). Hence, the effect of the fusion proteins on URA3 silencing was the result of their targeting to specific loci, not the consequence of non-specific, indirect functions (if any) of these proteins.<br><br>Targeted activation domain of Adr1p or Gcn4p can also act as a barrier factor
We have shown above that ADH2-UAS bearing Adr1p-binding sites and HIS3-UAS bearing Gcn4p sites both had detectable barrier activity (Fig. 1, strains 25 and 26). Given that tethering the activation domain of Rap1p is sufficient for barrier function, we tested if targeting the activation domain of Adr1p or Gcn4p could also establish a barrier. Adr1p contains four separate activation domains (TADI–TADIV) that can contact Ada2p, Gcn5p and/or TFIIB (45). TAD III (415–467) interacts with only Gcn5p but not Ada2p or TFIIB. We fused this domain of Adr1p and the activation domain of Gcn4p (102–149) to LexA, respectively (Fig. 5B, L7 and L8). Targeting these fusion proteins to two LexA-binding sites between HML-I and URA3 in strain 50 dramatically reduced URA3 silencing (Fig. 5B, compare L7 and L8 to L1). Therefore, like LexA-Rap1p (627–695), both LexA-Adr1p (415–467) and LexA-Gcn4p (102–149) also had barrier activity. Barrier function of these fusion proteins was significantly decreased in strain 51 in which only one LexA site was inserted between HML-I and URA3 (compare strains 50 and 51 carrying plasmid L7), indicating that the potency of such barriers was dependent on the amount of tethered fusion proteins.<br><br>
DISCUSSION
We have shown in this report that UASs of many highly expressed genes could function as barriers to the spread of transcriptional silencing. These newly identified barrier elements together with previously described barrier elements (STARs, HMR-tRNA gene, TEF2-UAS and CHA1-UAS) (9–11,21) are scattered across the <span class="yellow">yeast</span> genome and may play a role in dividing the genome into functional domains. Analyses of the barrier UASs suggest that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p can participate in barrier function. Barrier activity often requires concerted actions of more than one factor-binding site. Our mutational analysis of TEF2-UAS suggests a direct involvement of Rap1p in its barrier function. It is noteworthy that not all UASs of highly expressed genes have barrier activity. Many non-barrier UASs also consist of multiple regulator binding sites, therefore other unknown factors might determine whether a UAS could act as a barrier.
How Rap1p and the other transcriptional regulators act to block silencing is not clear. Since they can all act as transcriptional activators, one natural explanation is that they may overcome the silencing effect of a silencer and directly activate the reporter gene. However, it has been well documented that Rap1p and the other transcriptional regulators usually act at relatively short distances (less than ∼600 bp upstream of the translation start codon) (28,29,46), but the binding sites for these regulators were >800 bp from the translation start codon in most of the silencing-blocking tests in this report (Figs 1 and 2). In the test shown in Figure 6, TEF2-UAS blocked ADE2 silencing when it was ∼2 kb downstream from the translation start codon of the gene. Moreover, Fourel et al. (44) demonstrated that the barrier function (or anti-silencing function) of an activator was independent of its role in activating transcription. Therefore, it is unlikely that the ability of a UAS to prevent gene silencing is due to its direct activation of the gene. In light of this, our results are in agreement with the notion that transcriptional regulators can also participate in defining the boundaries of functional chromosomal domains.
A few models have been proposed for the barrier functions of transcriptional regulators. One model proposed that binding of a regulator to a barrier led to the formation of a nucleosome-free region thereby disrupting a continuous nucleosome array (11). Since the SIR complexes spread by self-interaction and interacting with adjacent nucleosmes in a stepwise fashion (4), a nucleosome-free gap could present a barrier to the spreading SIR complex. This ‘nucleosome gap’ hypothesis is consistent with that certain barrier factors such as Rap1p and Reb1p could indeed prevent nucleosome formation around their binding sites (14) and the barrier function of TEF2-UAS correlated with its ability to alter local chromatin structure (this report). In fact, many insulator/boundary elements in higher organisms have also been shown to alter local chromatin structure. For instance, the <span class="blue">chicken</span> β-globin insulator was linked to changes in nucleosome positioning, and the <span class="yellow">Drosophila</span> scs and scs′ boundary elements are associated with unusual chromatin structures that are hypersensitive to nucleases (40,41). However, it is not clear whether any chromatin change associated with barrier function in <span class="yellow">yeast</span> or insulator function in higher cells is the cause or effect of the function.
Another model for barrier action proposed by R. Kamakaka and colleagues (21) proposed that barrier factors recruit chromatin modifying and/or remodeling complexes to counteract the more compact, hypoacetylated silent chromatin. In agreement with this hypothesis, targeted HATs SAS2, GCN5 and ESA1 all had barrier activity (21; Y.-H. Chiu, Q. Yu and X. Bi, unpublished results). Interestingly, Rap1p, Adr1p, Gcn4p and Abf1p share the ability to recruit HAT complexes NuA4 and/or SAGA to target genes (16,45,47–49). Therefore, these factors that are capable of binding to the newly identified barriers may function as barrier factors by recruiting HAT complexes. In fact, recruitment of HAT activity has also been suggested to be the mechanism underlying the function of the <span class="blue">chicken</span> β-globin insulator (6), although it was not known what HAT might be involved. Our demonstration that tethering the activation domain of Rap1p, Adr1p or Gcn4p alone recapitulates barrier activity is consistent with the above HAT model for barrier function, since at least the activation domain of Adr1p has been shown to directly interact with HAT complexes (45). However, we haven’t ruled out the possibility that the barrier function of the targeted activation domain of Rap1p, Adr1p or Gcn4p is not related to the barrier function of the native protein. Note that the ‘nucleosome gap’ model and the HAT model are not mutually exclusive since the nucleosome gap could result from destabilization of nucelosomes caused by HAT function. A recent intriguing study showed that targeting proteins of the nuclear pore complex (NPC) could also establish a boundary for silent chromatin, and that tethering to NPC was required for boundary activity (50). This led to a revisit to the looping model for the function of boundary/insulator elements (51). However, an alternative explanation was that the NPC compartment could simply be rich in transcriptional activating factors or poor in silencing factors like the SIR proteins (50).
In summary, results from this and other studies indicate that factors previously identified as positive and/or negative regulators of gene activation could also function in demarcating distinct domains of gene activity. How these factors carry out their boundary function is not resolved but recruitment of other factors (e.g. chromatin modifying complexes) is likely to be involved.<br><br>NOTE ADDED IN PROOF
After this work was completed and submitted for publication the first time, Fourel et al. (52) reported that targeted activation domains of Rap1p and Abf1p had insulating capacity.<br><br>
</body></html>