
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>chick</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA2377325</h3>Purinergic signalling in the subretinal space: a role in the communication between the retina and the RPE
Abstract
The retinal pigment epithelium (RPE) is separated from the photoreceptor outer segments by the subretinal space. While the actual volume of this space is minimal, the communication that occurs across this microenvironment is important to the visual process, and accumulating evidence suggests the purines ATP and adenosine contribute to this communication. P1 and P2 receptors are localized to membranes on both the photoreceptor outer segments and on the apical membrane of the RPE which border subretinal space. ATP is released across the apical membrane of the RPE into this space in response to various triggers including glutamate and chemical ischemia. This ATP is dephosphorylated into adenosine by a series of ectoenzymes on the RPE apical membrane. Regulation of release and ectoenzyme activity in response to light-sensitive signals can alter the balance of purines in subretinal space, and thus coordinate communication across subretinal space with the visual process.<br><br>Introduction
The retinal pigment epithelium (RPE) lies between the outer segments of the photoreceptors and the choroidal blood supply (Fig. 1). The RPE combines the functions of epithelial and glial cells, providing a barrier while also supporting the neural photoreceptors and modulating their function. Tight communication between photoreceptors and the RPE is critical to coordinate the multiple levels of interaction, and the purinergic contribution to this communication is becoming apparent. The relevance of this purinergic input is emphasized by the many functional effects of P1 and P2 receptor stimulation and by the multiple mechanisms in place to regulate subretinal levels of purine agonists. As the dynamics of ATP release and extracellular conversion into adenosine will modify agonist availability, the modulation of these processes can exert a temporal control on purinergic signaling. The following review will outline the main interactions between the RPE and photoreceptors, describe the effects of stimulating purinergic receptors on both sides of subretinal space, and summarize how levels of ATP, ADP, and adenosine are manipulated in this microenvironment.
Fig. 1Schematic illustration of the key components of purinergic signaling in the subretinal microenvironment. Stimulation of P2 receptors on the RPE can enhance transepithelial fluid absorption while P1 receptors can modulate phagocytosis. ATP released through CFTR and other Cl− channels can stimulate P2 receptors or be converted to ADP, AMP, and adenosine (Ado) by a series of ectonucleotidases present on the apical membrane of the RPE. By controlling the balance of extracellular purines available to stimulate these receptors these mechanisms can control levels of endogenous purines available to activate the receptors. While theoretically possible, it remains to be determined whether these subretinal purines can actually stimulate photoreceptors
Purines and subretinal space
RPE-photoreceptor interactions across the subretinal space
The outer segments of the rods and cones are responsible for the initial stages of vision, converting photon energy into a series of enzymatic reactions that close the light-sensitive channels on the photoreceptor plasma membrane, hyperpolarize the cells, and reduce the release of glutamate from the synaptic terminals [1, 2]. Efficient photoreceptor function depends upon both short-term and long-term support from the RPE. The critical nature of these interactions is evident from the rapid degeneration of photoreceptors in the absence of a healthy RPE layer and by the RPE localization of defective gene product in some forms of hereditary photoreceptor degeneration [3].
The apical membrane of the RPE is separated from the plasma membrane of the outer segments by an extracellular space of only 10–20 nm [4]. Although small, this subretinal space contains a highly structured matrix which ensheathes the outer segments and extends to the RPE [5, 6]. The presence of enzymes within this interphotoreceptor matrix emphasizes that this extracellular space itself is functionally active [7, 8]. This intimate anatomical relationship between photoreceptors and the RPE reflects multiple functional interactions. For example, the RPE accepts, recycles, and exports central components of the phototransduction process [9]. The outer segments are continuously resynthesized, and the phagocytosis, degradation, and processing of shed outer segment tips by the RPE cells is central to this renewal [10]. The ion channels and transporters on the apical membrane of the RPE help regulate the ionic composition of the subretinal space [11]. As extracellular levels of ions can modify the ionic driving forces across the photoreceptor plasma membrane, these RPE transporters can influence the state of neural activity. The transport of fluid and ions from the apical membrane to basolateral membrane of the RPE is also one of the main forces keeping the retina attached [12].
The control of photoreceptor activity by light gives a rapid temporal dependence to some interactions between the photoreceptors and the RPE. The volume of subretinal space increases in response to light [13], linking small changes in the ionic composition of the subretinal space with activity of the RPE transport mechanisms which maintain this volume [14, 15]. Other processes are controlled on a diurnal cycle. The shed tips of the outer segments are phagocytosed by the RPE soon after the onset of light [16, 17]. These processes can both be modulated by purine levels in subretinal space, indicating purinergic regulation is important over multiple time scales.<br><br>Purinergic receptors on photoreceptors
A2 adenosine receptors were localized to both the inner and outer segments of photoreceptor outer segments over a decade ago by Blazynski and colleagues [18], with more recent reports emphasizing their functional role. A2 agonists inhibit the L-type Ca2+ channel on rod outer segments [19] and can inhibit the synaptic release of glutamate from rods, suggesting changes in adenosine levels in subretinal space could modulate light sensitivity [20]. The A2 agonist DPMA and the adenosine deaminase inhibitor EHNA reduce mRNA for opsin in rods, suggesting that endogenous levels of adenosine can downregulate opsin message at night [21]. EHNA and the A2A receptor agonist CGS21680 also increase the survival of <span class="red">chick</span> embryonic photoreceptors in culture [22], indicating a long-term neuroprotective role for adenosine.
P2 receptors are also present in the photoreceptors. mRNA for the P2X2 receptor is expressed in the photoreceptor cell bodies, with immunohistochemical localization of the protein to outer segments [23]. In situ hybridization indicates the photoreceptor layer has the highest level of P2Y2 receptor of any region in the <span class="yellow">rabbit</span> retina, although staining was not pronounced in monkey [24]. P2X7 receptors have recently been localized to photoreceptor synaptic terminals, with evidence for ecto-ATPase activity in the synapse, and functional evidence suggesting ATP augments transmission of the light response by rods [25]. It was suggested that ATP might be co-released from photoreceptors with glutamate, although this remains to be tested directly.<br><br>Purinergic receptors on the RPE
Stimulation of P1 receptors can have a considerable impact on RPE cells. A2 receptors have been recognized on cultured and fresh RPE cells for some time [26, 27], with in situ hybridization confirming the presence of A2A receptors in <span class="yellow">rat</span> RPE [28]. Stimulation of A2 receptors reduces the rate of rod outer segment phagocytosis by RPE cells [29], while application of adenosine to the apical membrane of <span class="red">chick</span> RPE cells increases the basolateral Cl- conductance, the transepithelial potential, and the c-wave, and decreases the hyperpolarization of the basal membrane in response to light [30]. Although adenosine alone does not increase intracellular Ca2+ levels [31], adenosine acts synergistically with ATP to elevate Ca2+ levels in <span class="yellow">human</span> RPE cells by stimulating both A1 and A2A receptors [32, 33]. Stimulation of A1 receptors with high doses of NECA increases the active transport of fluorescein across the RPE, while activation of A2A receptors decreases this transport, and by extension, transport of the ions that underlie fluid movement [34]. Stimulation of A1 and A2A receptors produces analogous increases and decreases, respectively, in the absorption of subretinal fluid blebs. This is consistent with the negative coupling of the A1 receptor and the positive coupling of the A2 receptors to adenylate cyclase, as increasing cAMP inhibits the transport of fluid across the RPE towards the choroid [35–37]. The agonist 2-Cl adenosine reverses the deficit in phosphoinositide metabolism found in diabetic RPE cells [38], suggesting effects on metabolism in addition to transport and phagocytosis.
Multiple P2 receptors have been localized to the RPE. The P2Y2 receptor was initially characterized in cultured <span class="yellow">human</span> RPE [31], with subsequent reports localizing transcript for P2Y1, P2Y2, P2Y4, and P2Y6 in the <span class="yellow">rat</span> RPE/choroid [39] and for P2Y1 and P2Y12 receptors in ARPE-19 cells [40], and functionally identifying a P2X receptor in <span class="yellow">rat</span> RPE cells [41]. ATP, ADP, and UTP induce numerous effects on RPE physiology [32, 33, 42, 43]. While these effects likely involve multiple different receptor types, the contributions of the P2Y2 receptor have been explored in most detail to date. The P2Y2 receptor has been specifically localized to the apical membrane of fresh <span class="yellow">bovine</span> RPE cells, and addition of ATP to this membrane transiently elevates Ca2+, activates a basolateral Cl- conductance, inhibits an apical K+ conductance, and increases the apical to basolateral flow of fluid [43]. This increased absorption of fluid from the subretinal space suggests P2Y2 receptor stimulation could reduce the excess fluid that accumulates in retinal edemas, and several reports have supported this theory. ATP, UTP, and the P2Y2 receptor agonist INS37217 decrease the size of subretinal fluid blebs when injected into subretinal space of <span class="yellow">rats</span> [44]. In both normal and rds +/- <span class="yellow">mice</span> with experimentally induced detachment, INS31217 improves the ERG recovery and decreased cell death [45]. INS37217 also reduces subretinal blebs in <span class="yellow">rabbits</span> [46]. Injection of another P2Y2 agonist, INS542, increases the active transport of fluorescein across the RPE, consistent with this upregulation of ion and fluid transport across the tissue [47]. Together these experiments emphasize the clinical potential of treating retinal edema with P2Y2 agonists.<br><br>
Regulation of purine levels in subretinal space
While synthesized purinergic agonists may prove useful in treating some ocular disorders, the endogenous activation of P1 and P2 receptors in the subretinal microenvironment will ultimately be determined by availability of agonists. These levels are largely controlled by the release of ATP into the subretinal space, its conversion into other purines including adenosine, and the manipulation of adenosine by enzymes or transporters. Recent work has increased our understanding of both the stimuli that initiate changes in subretinal purine levels and the mechanisms that mediate these changes.
Release of ATP by the RPE
At least some of the ATP capable of stimulating the purinergic receptors on RPE cells is released from the RPE itself. The resulting autocrine stimulation ensures local delivery, and control, of purines to initiate the physiologic changes in the RPE. The release of ATP by RPE cells is triggered by multiple stimuli including osmotic stress, bFGF, UTP, NMDA, glutamate, and ATP [39, 40, 48–51]. The ATP release following activation of NMDA receptors by glutamate may have the most interesting implications for communication across subretinal space, given that glutamate confers the light signal from photoreceptors to the rest of the visual system. Glutamate and the specific receptor agonist NMDA triggers ATP release from ARPE-19 cells, with the release inhibited by NMDA antagonist MK-801, and by DCKA, which inhibits the glycine B binding site on NMDA receptors [51, 52]. Although NMDA raises intracellular Ca2+ levels, this increase is prevented by eliminating ATP with apyrase, indicating autostimulation through released ATP is responsible for this Ca2+ signal. NMDA also triggers a release of ATP when applied to the intact <span class="yellow">bovine</span> RPE eyecup [51]. The NMDA receptors and the ATP release sites have been functionally identified to the apical membrane of the <span class="yellow">bovine</span> RPE, suggesting the neurotransmitter interactions could amplify the signal from any glutamate reaching subretinal space.
The ability of both UTP and ATP to stimulate release of ATP from the RPE supports the theory that the system acts to amplify signals. When applied at greater than 1 μM, ATP triggers a secondary release of ATP peaking 10 min after the initial stimuli [40]. UTP also initiates a release in extracellular ATP with a similar delay [48]. The rise in ATP triggered by UTP is inhibited by the Cl- channel blocker NPPB, and UDP is much less effective at triggering release than UTP; both observations suggest the P2Y2 receptor contributes to the increase in ATP more than diphosphokinase, although influence from the enzyme cannot be ruled out [53].
Recent evidence suggests that ischemia may lead to the release of ATP from RPE cells. Chemical ischemia triggers a substantial ATP release from cardiac myocytes [54], while changes in oxygen levels trigger ATP release in central chemoreceptors [55]. We found that exposure to sodium cyanide led to a rapid release of ATP from ARPE-19 cells (Fig. 2). As hypoxic and/or ischemic challenge may lead to changes in the expression of growth factors in RPE cells during certain ocular disorders such as macular degeneration [56], and as purines can induce expression of VEGF in other cells [57], this ATP release may contribute to growth factor signaling by the diseased RPE.
Fig. 2Chemical ischemia triggers ATP release from ARPE-19 cells. ATP release was measured in the bath directly from cells plated in 96-well plates to which the luciferin- luciferase reaction mixture was added [51]. Left Levels of ATP in the bath after addition of 5 mM NaCN to the cells. Measurement began 1 min after addition of NaCN or control solution to wells. Right Levels of ATP measured at the peak, 3 min after addition of NaCN (n = 12). Levels were normalized to concurrent levels in control (n = 14). Symbols and bars represent mean ± SE, *p < 0.001
The particular conduit for ATP release varies with the stimuli. The release in response to hypotonic challenge is largely dependent upon CFTR, as it was prevented by the specific CFTR inhibitor CFTR172 in addition to the more general blocker glybenclamide [50]. While the precise mechanisms by which CFTR contributes to this release are not yet known, a role for CFTR in ATP release into subretinal space is consistent with the reduction of certain ERG components in cftr -/- <span class="yellow">mice</span> [58] and with the ability of apical ATP to activate conductances associated with these ERG components [43]. The release of ATP is also largely blocked by the vesicular transport inhibitor brefeldin A, suggesting the two processes occurred in series whereby ATP efflux follows the insertion of vesicles containing CFTR into the plasma membrane. Although the Ca2+ chelator BAPTA blocks this ATP release [50], raising Ca2+ alone with ionophore ionomycin does not itself initiate release [48]. This necessary but not sufficient contribution of Ca2+ also supports a role for vesicular insertion.
In contrast to the release following hypotonic challenge, the ATP release in response to NMDA does not involve CFTR [51]. Release is blocked by NPPB, however, suggesting another type of anion channel could serve as a conduit for ATP release. The presence of parallel mechanisms coexisting on the same cell for ATP release triggered by either agonists or by cell swelling has also been reported in astrocytes [59] and may reflect the multiple roles of purinergic signaling within a given tissue. As both stimuli lead to release across the apical membrane into subretinal space, both are expected to influence signaling in the microenvironment.<br><br>Interconversion of purines in subretinal space
The interconversion of nucleotides and nucleosides each capable of stimulating distinct receptors makes the purinergic signaling system of particular interest in a confined region such as the subretinal space. The main enzymes responsible for dephosphorylating extracellular ATP on the RPE cells have been analyzed and a basic understanding of their regulation has begun. This section first describes the enzymes that act on ATP and ADP, followed by enzymes which convert AMP into adenosine.
The dephosphorylation of extracellular ATP by RPE cells involves enzymes from multiple families [40], as found in airway epithelial cells [60]. Degradation of ATP by the apical membrane of the fresh <span class="yellow">bovine</span> eyecup and by ARPE-19 cells is inhibited by ARL67156 or βγmATP. Message for eNPP1, eNPP2, and eNPP3 is present in ARPE-19 cells, consistent with the preference of βγmATP for members of the eNPP family [61]. The cells also express NTPDase2, and NTPDase3, although the intermittent presence of NTPDase1 likely reflects a regulated process [40]. Ecto-alkaline phosphatase has no effect on ATP degradation in RPE cells, in contrast to its considerable contribution in airway epithelium [62]. The putative contribution from diphosphokinases to interconversion of subretinal purines is presently unknown.
Extracellular AMP is rapidly dephosphorylated into adenosine in subretinal space. The production of adenosine from ATP at the apical membrane of the <span class="yellow">bovine</span> RPE eyecup is inhibited by the ecto-5′-nucleotidase inhibitor αβmADP, confirming a role for this enzyme [63]. The enzyme is localized to <span class="yellow">rat</span> RPE and ARPE-19 cells immunohistochemically. Degradation of 5′AMP is highest near the subretinal space of <span class="yellow">rat</span> retina [63], although localization in <span class="yellow">mouse</span> indicated larger amounts of ecto-5′-nucleotidase at the tips of adjacent Müller cells [64]. Levamisole does not inhibit the dephosphorylation of 5′AMP by the RPE, consistent with the absence of substantial ecto-alkaline phosphatase in subretinal space.
The presence of light may alter the levels of adenosine in subretinal space. Epinephrine is released at the onset of light [65] and stimulation of the RPE with epinephrine can decrease activity of ecto-5′-nucleotidase [63]. While norepinephrine and phenylephrine lead to similar decreases in enzyme activity, prazosin and corynanthine block the effects of norepinephrine, implicating the α1 epinephrine receptor in the inhibition of ecto-5′-nucleotidase [63]. The kinetics of inhibition are consistent with cleavage of the nucleotidase from its GPI anchor. The phagocytosis of rod outer segments is maximal shortly after light onset [16], and this phagocytosis is inhibited by adenosine [29]. The ability of epinephrine released by the illuminated retina to reduce ecto-5′-nucleotidase activity and consequently adenosine levels may relieve this inhibition and enhance the rate of phagocytosis at light onset.<br><br>
Physiologic effects of subretinal purines on the RPE and photoreceptors
The number of purinergic receptors on both photoreceptor and RPE membranes suggests purines make multiple contributions to the physiology of the outer retina. Our increased understanding of how agonist levels in subretinal space are controlled has begun to indicate how and when this contribution may occur. Future research will involve applying these findings from isolated systems to intact RPE-photoreceptor models, and pursuing the role of defective purinergic regulation in ocular disease. While it is unlikely that ATP released across the apical membrane of the RPE can diffuse to these P2 receptors in the outer plexiform layer given the ecto-ATPase activity in the synaptic clef [25], stimulation of receptors elsewhere on the photoreceptor membrane is possible. It would be interesting to determine whether ATP released from the RPE and converted to adenosine by ecto-nucleotidases can actually modulate the response to light by stimulating the A2A receptors on photoreceptor outer segments. The impact of purinergic signaling on chronic ocular diseases is also of interest, such as the role of ischemia-driven ATP release in VEGF production. While the small size of subretinal space can complicate pharmacologic manipulation within the intact RPE-photoreceptor complex, molecular approaches may provide new insight into how endogenous purines in subretinal space affect the physiology, and pathophysiology, of both RPE and photoreceptors.<br><br><br><br><h3>pmcA1257442</h3>Identification of kinectin as a novel Behçet's disease autoantigen
Abstract
There has been some evidence that Behçet's disease (BD) has a significant autoimmune component but the molecular identity of putative autoantigens has not been well characterized. In the initial analysis of the autoantibody profile in 39 Chinese BD <span class="yellow">patients</span>, autoantibodies to cellular proteins were uncovered in 23% as determined by immunoblotting. We have now identified one of the major autoantibody specificities using expression cloning. Serum from a BD <span class="yellow">patient</span> was used as a probe to immunoscreen a λZAP expression cDNA library. Candidate autoantigen cDNAs were characterized by direct nucleotide sequencing and their expressed products were examined for reactivity to the entire panel of BD sera using immunoprecipitation. Reactivity was also examined with normal control sera and disease control sera from <span class="yellow">patients</span> with lupus and Sjögren's syndrome. Six independent candidate clones were isolated from the cDNA library screen and were identified as overlapping partial <span class="yellow">human</span> kinectin cDNAs. The finding that kinectin was an autoantigen was verified in 9 out of 39 (23%) BD <span class="yellow">patient</span> sera by immunoprecipitation of the in vitro translation products. Sera from controls showed no reactivity. The significance of kinectin as a <span class="yellow">participant</span> in autoimmune pathogenesis in BD and the potential use of autoantibody to kinectin in serodiagnostics are discussed.<br><br>Introduction
Behçet's disease (BD) is a systemic vasculitic disease typified by a triad of symptoms including recurrent oral ulcers, genital ulcers and uveitis. In addition, skin, joint, large vessels, nervous system and gastrointestinal systems may be involved. BD is a global disease but has the highest prevalence in the region along the ancient 'Silk Road' in China. The etiopathogenesis of the disease remains unclear but microbial agent triggers, environmental factors, genetic predisposition, neutrophil hyperfunction, endothelial cell dysfunction and immunological abnormalities involving both T and B cells have been implicated. Increasing amounts of research evidence supports the possibility that it is an immune-mediated vasculitis, and that abnormal T-cell and B-cell reactions and autoantigen-driven autoimmunity play pivotal roles [1]. Systemic lupus erythematosus (SLE) is the prototypic systemic autoimmune rheumatic disease with autoantibodies against cellular (particularly nuclear) antigens, some of which are critically implicated in the autoimmune pathology while others provide valuable serodiagnostic markers for the disease. Unlike the picture in SLE and other related rheumatic diseases, in BD, antinuclear antibodies and antibodies to neutrophil cytoplasmic antigens etc. are not present. To date, since neither a specific autoantibody nor pathognomonic pathological index is available to help establish the diagnosis of BD, it is largely or solely based on clinical manifestations [2], and a dilemma in diagnosis is not a rare occurrence in clinical practice. Nevertheless, since the 1960s, there have been reports of autoantibodies against certain unknown components of <span class="yellow">human</span> oral mucosa in sera of <span class="yellow">patients</span> with BD. Since then, sporadic reports on findings of autoantibodies in this disease have been described, such as antibodies to retinal antigen(s), heat shock protein (HSP) of some strains of <span class="yellow">Streptococcus sanguis</span> cross-reactive with <span class="yellow">human</span> HSP polypeptide [3], antibodies to endothelial cell antigens (AECA) and antibodies to α-tropomyosin [4,5], attesting to the complicated humoral immune disorders in this disease.
This investigation was aimed at defining target cellular autoantigens using time-tested and well-established molecular techniques. Immunoscreening of expression libraries using BD sera was used since this approach has been successfully employed in the characterization of many clinically relevant antigens in systemic rheumatic diseases such as SS-A/Ro [6-9] and SS-B/La [10] antigens in Sjögren's syndrome (SjS) and centromere antigen CENP-B [11] in scleroderma. In addition, we have been successful in using this strategy to identify interesting autoantigens that have other biological significance. Examples of these include NOR90/hUBF [12], p80-coilin [13], Golgi autoantigens [14-16] and, more recently, GW182 [17].<br><br>Materials and methods
<span class="yellow">Patients</span> and sera
The currently used empirical criteria for the diagnosis of BD in this study were the criteria proposed by the International Study Group for BD (abbreviated as 'International Criteria') [2]. The study subjects of 39 Chinese BD <span class="yellow">patients</span> comprised 17 males and 22 females, mean age 37 ± 11.3 years old, who were divided into two subgroups: 25 typical BD <span class="yellow">patients</span> (Group I, satisfying the International Criteria) and 14 clinically diagnosed BD <span class="yellow">patients</span> who had recurrent oral ulcers and one of the symptoms of genital ulcers, eye symptoms or skin lesions as defined by the International Criteria, as well as additional symptom(s) closely related to BD as listed in the International Criteria, that is, gastrointestinal ulcerations, deep vein thrombosis or arthralgia/arthritis without evidence that the latter symptoms might be related to any other disease (Group II, defined as 'probable BD' in this study). Disease controls included 10 <span class="yellow">patients</span> with SLE and 10 with SjS, all satisfying corresponding international classification criteria. All BD <span class="yellow">patients</span> and disease controls involved in the study were <span class="yellow">patients</span> treated at the Rheumatology Department of Ren Ji Hospital, Shanghai, China, where their clinical data and serum samples were collected. Twenty normal control sera were randomly selected from healthy blood donors working in the same hospital. This study was approved by the institution review board of Ren Ji Hospital which is affiliated with Shanghai Second Medical University, and each <span class="yellow">patient</span> involved gave informed consent. All serum samples were preserved at -20°C or -70°C until use.<br><br>Cell lines and cell extracts
HeLa (ATCC CCL 2.2) and T24 (<span class="yellow">human</span> transitional cell bladder carcinoma) were obtained from the American Type Culture Collection (Rockville, MD, USA). A <span class="yellow">bovine</span> aortic endothelial cell line was kindly provided by Dr Eugene G Levin from the Scripps Research Institute (La Jolla, CA, USA). Cells were cultured in DMEM containing 10% <span class="yellow">calf</span> serum, harvested and extracted in Buffer A (150 mM NaCl, 10 mM Tris-HCl, pH7.2, 0.5% Nonidet P-40) with protease inhibitor (Complete™; Boehringer Mannheim, Indianapolis, IN, USA). For the preparation of whole cell extract, 10 volumes of Laemmli gel sample buffer [18] were added to the cell pellet, boiled for 3 min and stored at -20°C until use.<br><br>Western blot
Whole cell lysates from <span class="yellow">bovine</span> aortic endothelial cell, HeLa and T24 cells were resolved individually by discontinuous 7.5% gel SDS-PAGE according to Laemmli's method [18]. Immunoblotting was performed as described by Towbin et al. [19] with modifications. Nitrocellulose strips were blocked with 3% nonfat milk in PBS containing 0.05% Tween-20 (PBS-T) and then incubated with BD <span class="yellow">patient</span> sera and normal control sera (1:100 dilution) at room temperature for 1 h. Filters were washed extensively with PBS-T to remove any unbound antibodies. Bound antibodies were detected with polyvalent, peroxidase-conjugated <span class="yellow">goat</span> anti-<span class="yellow">human</span> Ig and visualized by incubating the nitrocellulose strips in chemiluminescent reagents (NEN Life Science Products Inc., Boston, MA, USA) and exposing to Kodak XAR-5 films.<br><br>Screening of phage cDNA expression library with antibody probes
Serum from a BD <span class="yellow">patient</span> showing the highest antibody titer in immunoblotting was selected as a probe and used at a dilution of 1:300 for initial immunoscreening of approximately 106 recombinants from a T24 cDNA expression library. The latter was constructed in λZAPExpress vector (Stratagene, La Jolla, CA, USA) and screened as previously described [20-22]. All screenings were performed on duplicate isopropyl β-D-thiogalactoside (IPTG) pre-impregnated nitrocellulose filters, and immunoreactive clones were detected by chemiluminescence. Positive phages were subsequently plaque purified to 100% by two repeated rounds of screening at low plaque densities. Before screening the cDNA library, the BD serum was extensively adsorbed against bacteria and wild-type λZAP phage mixture to reduce background binding.<br><br>Analysis of candidate cDNAs
Purified candidate plaques were subcloned in vivo into pBK-CMV plasmids using ExAssist™ helper phage as recommended in the manufacturer's instructions (Stratagene). The recombinant pBK-CMV plasmids were then purified using QIAprep Spin Minprep Kit (Qiagen, Valencia, CA, USA). Restriction enzyme digestion of plasmids with EcoRI and XhoI and electrophoresis in a standard 1.0% agarose gel was used to analyze the length of cDNA insert of each candidate plasmid. The complete nucleotide sequence was determined using Bigdye terminator sequencing and a semi-automated sequencer model 377 (ABI, Foster City, CA, USA). Both nucleotide and deduced amino acid sequences were analyzed for similarity with known sequences using BLAST search [23] and ExPASy Proteomics tools . Secondary structure analysis for coiled-coil motifs was conducted with the software program COILS [24].<br><br>Immunoprecipitation of in vitro translation products
Candidate cDNA clones were used as templates for in vitro transcription and translation and the products were used as substrates for immunoprecipitation to confirm the specificity of reaction with BD sera. In brief, 1 μg of the pBK-CMV plasmid identified in the screening outlined above was added as template in a 50-μl reaction for the coupled in vitro transcription and translation reaction with a <span class="yellow">rabbit</span> reticulocyte lysate system (Promega, Madison, WI, USA) in the presence of 35S-methionine (Trans-35S label; ICN Biochemicals, Costa Mesa, CA, USA) and RNasin® Ribonuclease Inhibitor (Stratagene) as recommended by the manufacturer (Promega). Translation was carried out at 30°C for 1.5 h. Products were analyzed in a 12.5% gel SDS-PAGE and stored at -80°C for further immunoprecipitation analysis. The in vitro translation proteins were examined for reactivity by sera using immunoprecipitation described [8,25].<br><br>
Results and discussion
Autoantibody detection in sera from BD <span class="yellow">patients</span>
Initial examination of a group of 39 BD <span class="yellow">patients</span> using indirect immunofluorescence (IIF) on a HEp-2 cell substrate did not yield any characteristic nuclear or cytoplasmic staining patterns. BD is thought by some to be a vasculitic disease involving pathophysiology of endothelial cells, and antibody to endothelial cell antigen (AECA) has been reported. Reports on the prevalence of AECA have varied largely and alpha-enolase was reported as one of the putative target antigens [26]. In this study, the use of <span class="yellow">bovine</span> aortic endothelial cells as substrate for IIF did not provide any additional data. However, Western blot analysis of the BD sera began to show some interesting autoreactivity using cell lysates from both HeLa and <span class="yellow">bovine</span> aortic endothelial cells. HeLa cells were initially used for this analysis because they are commonly used in the laboratory as Western blot substrate. Fig. 1 illustrates the common reactivity to 49 kDa and 120 kDa proteins in the endothelial cell lysates. These antigens were also detected in HeLa and T24 cells; the latter cell line was analyzed because our laboratory at The Scripps Research Institute has produced an excellent expression cDNA library from the T24 line and the positive result with the T24 cell extracts allowed us to screen the T24 library. Ig isotype analysis showed that all reactivity was largely IgG antibodies. Since the 49 kDa and 120 kDa bands were observed in cell extracts from <span class="yellow">bovine</span> as well as <span class="yellow">human</span> cell lines, these autoantigens might be evolutionarily conserved.
In total, nine out of 39 BD sera (23%) had autoantibody to the 49 kDa antigen and eight (20%) to the 120 kDa antigen. Four BD sera (10%) reacted with both proteins. Additionally, sera that showed common reactivity to the 120 kDa protein also demonstrated a common band that migrated at ~150 kDa, although it appeared weaker than the 120 kDa band. These antigens appeared to have different molecular weights than those of the known autoantigens in systemic rheumatic diseases. In addition, other reactive bands were detected but they were not as commonly shared as the 49 kDa and 120 kDa bands. The 49 kDa protein was shown to be distinct from 48 kDa SS-B/La or 50 kDa Jo-1 proteins (Fig. 1). The 120 kDa antigen was also shown to migrate differently from alanyl tRNA synthetase in another Western blot analysis (data not shown) and did not share any apparent crossreactive epitopes with the 49 kDa antigen. Western blot analyses of 20 normal control sera did not show the reactivities observed with BD sera. In order to further characterize these autoreactivities, a serum sample from the Group I definitive BD <span class="yellow">patients</span> with the strongest reactivity to 49 kDa and 120 kDa antigens (Fig. 1, lane 2) was selected as antibody probe for expression library screening.<br><br>Kinectin identified as a novel BD autoantigen
After screening 500,000 clones from the T24 cell λZAPExpress expression library, seven immunoreactive clones were isolated and plaque purified in two to three rounds to achieve 100% homogeneity. The cDNA inserts were subcloned in vivo into pBK-CMV plasmids, analyzed by restriction digestion using EcoRI and XhoI enzymes, and submitted to direct nucleotide sequencing across the polylinker arms. The cDNA inserts represented six independent clones designated BD41 (identical to BD44), BD481, BD42, BD47, BD482 and BD49. Their identities were established as overlapping partial cDNAs of <span class="yellow">human</span> kinectin, ranging from 1.9 kb to 3 kb (Fig. 2a). The full-length <span class="yellow">human</span> kinectin (GenBank accession number NM_182926[27]) has 4,816 bases containing an open reading frame coding 1,357 amino acid residues with molecular mass 156 kDa. All six cDNAs lacked the 5' portion of the kinectin sequence to different degrees but spanned a sequence of kinectin that extended to the 3'-untranslated region. Secondary structure analysis of kinectin protein using the program COILS identified a long region of α-helical coiled-coil domain that extended from amino acid residue 327 to the C-terminus (Fig. 2a, hatched boxes). In vitro coupled transcription and translation of BD44 and BD42 clones directed the synthesis of [35S]-methionine-labeled polypeptides that migrated at 95 and 60 kDa, respectively, in addition to smaller polypeptides (Fig. 2b). These products had predicted molecular weights of 103 kDa and 75 kDa.
Kinectin was initially identified in <span class="blue">chick</span> embryo brain microsome as an integral membrane protein anchored in endoplasmic reticulum and involved in kinesin-driven vesicle motility along microtubules [28,29]. Kinectin consists of a 120-kDa and a 160-kDa polypeptide interacting through the α-helical coiled-coil domain to form a heterodimer [30]. The full-length kinectin is the 160 kDa polypeptide containing an N-terminal transmembrane helix followed by a bipartite nuclear localization sequence and two C-terminal leucine zipper motifs. We presume that the 120 kDa polypeptide detected in Western blot (Fig. 1) is the truncated version of the 160-kDa polypeptide, lacking the N-terminal first 232 amino acids [30]. The N-terminus of the 160-kDa polypeptide consists of a transmembrane domain that anchors kinectin to endoplasmic reticulum [30,31]. This 120 kDa polypeptide is probably the predominant form detected in the Western blot analysis (Fig. 1) because of its preferential solubility due to the omission of the N-terminal transmembrane domain.
Other functions for kinectin have been reported. <span class="yellow">Yeast</span> two-hybrid screen studies from several laboratories have revealed the interaction of the Rho family of GTPase with kinectin, and have shown the functional links among RhoG, kinectin and kinesin, with kinectin as a key effector of RhoG microtubule-dependent cellular activity [32]. Kinectin was also identified as an important constituent of integrin-based adhesion complexes, which link integrins to the cytoskeleton and recruit signaling molecules [33]. A new study reported that a kinectin isoform lacking a major portion of the kinesin-binding domain is very probably the most conservative form of kinectin; it does not bind kinesin but act as a membrane anchor for the translation elongation factor-1 delta in the endoplasmic reticulum [34].<br><br>Prevalence and specificity of anti-kinectin autoantibodies
The in vitro [35S]-methionine-labeled translation product of BD44, representing the largest recombinant kinectin fragment available, was used as the antigen substrate in an immunoprecipitation assay. Out of 39 BD <span class="yellow">patient</span> sera, nine (23%) recognized the BD44 translation product (Fig. 3), whereas sera from 20 normal controls, 10 SLE and 10 SjS <span class="yellow">patients</span> did not show reactivity. Among the nine anti-kinectin positive <span class="yellow">patients</span>, six (6/25, 24%) were from Group I (definitive BD) including the BD <span class="yellow">patient</span> whose serum was used in the immunoscreening of expression cDNA library, and three (3/14, 21.4%) <span class="yellow">patients</span> were from the Group II (probable BD) in this study. According to the Fisher Exact Probability calculation (P = 1.00), there is no statistically significant difference for antibody to kinectin between the two groups. The combined data substantiated the finding that kinectin is an autoantigen that can be recognized by sera from 23% of Chinese BD <span class="yellow">patients</span> in this study with at least one immunoreactive region or autoepitope residing within the BD44 encoded polypeptide.
Currently, there are more than six diagnostic/classification criteria for BD, among which the International Criteria have been applied most extensively due to its relatively high sensitivity (91%) and specificity (96%) [2]. As discussed above, differential diagnosis of BD might be confusing in clinical practice since no specific laboratory test is available, and some <span class="yellow">patients</span> may have symptoms and signs strongly suggestive of BD but do not fully satisfy the International Criteria, as in the Group II (probable BD) <span class="yellow">patients</span> in our study group. A number of investigators have pointed out that a comprehensive analysis of the clinical data for a given <span class="yellow">patient</span> is very important for correct clinical diagnosis of BD, and that classification/diagnosis criteria, including the International Criteria, should be followed but should not be exclusive. The observation that three out of 14 <span class="yellow">patients</span> in the probable BD group also had antibody to kinectin and the similar percentage of positive reactors between this group and Group I (21.4% versus 24%) supports this notion. The further use of non-clinical parameters such as immunological biomarkers as adjuncts to identify BD <span class="yellow">patients</span> could be of help in the classification of this disease entity
While our work was ongoing, anti-kinectin antibodies were reported in sera from <span class="yellow">patients</span> with hepatocellular carcinoma (HCC) [35,36] and aplastic anemia [37,38]. The first HCC report [35] identified kinectin as a tumor-associated antigen from the screening of an autologous cDNA library constructed from the cancer of a 30-year-old <span class="yellow">patient</span> from Guangxi, China. This report stated that four out of five HCC <span class="yellow">patients</span> tested were positive for anti-kinectin antibody [35]. In 2004, another laboratory also reported the cloning of kinectin as a tumor-associated antigen from a (presumably) different 30-year-old Chinese HCC <span class="yellow">patient</span> [36]. In contrast, anti-kinectin antibodies were not detected in other studies of HCC <span class="yellow">patients</span> associated with our laboratory [39,40]. The reports of anti-kinectin antibodies in aplastic anemia are also very interesting [37,38]. The initial report by Hirano et al. identified kinectin by screening an aplastic anemia <span class="yellow">patient</span> for candidate antigens using a Clontech <span class="yellow">human</span> fetal liver cDNA expression library and it was concluded that seven out of 18 aplastic anemia <span class="yellow">patients</span> were positive for anti-kinectin while none of the normal or disease controls had this antibody [37]. In their recent report, Hirano et al. reported that anti-kinectin antibodies were found in 39% of aplastic anemia <span class="yellow">patients</span> from the United States but only in three out of 30 (10%) cases in Japan [38]. In our study reported here, kinectin antibodies were only detected in BD <span class="yellow">patients</span> and not in normal controls and SLE and SjS disease controls. None of the BD <span class="yellow">patients</span> with anti-kinectin had signs of HCC or aplastic anemia at the time of diagnosis and at up to 4 years of follow-up. Mapping of epitope(s) recognized by anti-kinectin antibodies may shed light on the question of whether different autoepitopes reside within the kinectin molecule recognized by sera from different diseases.<br><br>Kinectin – a new member of coiled-coil cytoplasmic autoantigens
We have recently reviewed the literature on the growing number of cytoplasmic autoantigens rich in α-helical coiled-coil domains as typified from our study of Golgi autoantigens [41]. Golgi autoantigens are generally high molecular weight proteins between 100 and 350 kDa and rich in coiled-coil domains in the central region with non-coiled-coil or globular domains at both N and C termini. Golgi autoantigens are displayed on the cytoplasmic face of the Golgi complex and are not localized to apoptotic blebs during apoptosis [42]. Giantin, the highest molecular weight Golgi autoantigen reported, is the predominant target of <span class="yellow">human</span> anti-Golgi complex antibodies and multiple non-cross-reactive epitopes have been mapped spanning the 350 kDa protein [43]. Other high molecular weight autoantigens with similar features have been reported in cytoplasmic and mitotic organelles suggesting that these selected proteins become autoimmunogenic based on their subcellular association and molecular features [41]. For example, in the endosomal compartment, the two known autoantigens are early endosomal protein EEA1 (180 kDa) [44] and CLIP-170 (170 kDa) [45]. There is also a series of centrosomal autoantigens identified as coiled-coil-rich proteins including pericentrin, a 220 kDa protein [46], ninein, a protein with alternatively spliced products of 245 and 249 kDa [47], Cep250 (250 kDa) and Cep110 (110 kDa) [48]. Centromere autoantigens have been described but the two interesting ones related to this discussion are CENP-E [49] and CENP-F [50]; both are high molecular weight proteins (312 to 400 kDa) and have the same type of overall structure as discussed above. NuMA is another large coiled-coil protein located at the mitotic spindle pole and is the most common target autoantigen in sera with mitotic spindle apparatus staining [51]. Non-muscle myosin (~200 kDa) is a cytoskeletal autoantigen [52] that falls in the same group of high molecular weight and coiled-coil-rich autoantigens. These endosomal, centrosomal, mitotic apparatus and intracellular autoantigens are, like the golgins, proteins with high molecular weights and an overall high content of coiled-coil domains. The combination of these two physical features in autoantigens may contribute to the induction and production of autoimmune antibodies in certain disease states. Kinectin is an integral membrane protein largely confined to the endoplasmic reticulum [28,31] and it fits into this new category of autoantigens that are large coiled-coil rich proteins (≥100 kDa) in the cytoplasm.<br><br>
Conclusion
Here we report the detection of kinectin autoantibody in 23% of Chinese <span class="yellow">patients</span> with BD. The identity of kinectin as a BD-related autoantigen has not been reported to date. Autoantibody reaction against kinectin in BD observed in this study further confirms the autoimmune involvement in BD and may provide new inroads into elucidating the immunopathogenesis of the disease. In an effort to clarify the association of BD with antibody to kinectin, it is essential to measure antibody to kinectin in larger <span class="yellow">patient</span> populations including both BD, probable BD and important autoimmune rheumatic diseases such as SLE, SjS, rheumatoid arthritis etc., as well as those diseases not easily differentiated from BD, such as recurrent aphthous oral ulcer, Reiter's syndrome, inflammatory bowel diseases etc. On the other hand, further analysis of the association of anti-kinectin antibody with different manifestations or disease 'subtypes' of BD is another important project. Anti-kinectin is clearly only one of the antigen-antibody systems identified because there were many other antibodies observed in the Western blot analysis of BD sera. Using other sera for immunoscreening would probably lead to the identification of other potentially important antigen-antibody systems.<br><br>Abbreviations
AECA = antibody to endothelial cell antigen; BD = Behçet's disease; DMEM = Dulbecco's modified Eagle's medium; HCC = hepatocellular carcinoma; HSP = heat shock protein; IIF = indirect immunofluorescence; PBS = phosphate buffered saline; SjS = Sjögren's syndrome; SLE = systemic lupus erythematosus.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YL performed the study and drafted the manuscript. PY provided technical help throughout the study. SLC and EMT conceived the study, participated in the design and helped in the analysis of the data. EKLC participated in the design of the study, interpreted data and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1805739</h3>Embryonic sympathoblasts transiently express TrkB in vivo and proliferate in response to brain-derived neurotrophic factor in vitro
Abstract
Background
Nerve growth factor and neurotrophin-3 are involved in the development of sympathetic neurons; however, whether brain derived neurotrophic factor also plays a role is not known. The purpose of this study was to determine whether BDNF and its receptor, TrkB, are expressed during the development of paravertebral sympathetic ganglia in vivo and to determine the effect of BDNF in vitro.<br><br>Results
As neural crest cells coalesce to form sympathetic ganglia, TrkB-positive cells are seen in both <span class="yellow">chicken</span> and <span class="yellow">mouse</span> embryos. In <span class="yellow">chicken</span> embryos, TrkB-expressing cells first appear at Hamburger-Hamilton Stage (St) 27 and they co-express HNK-1, confirming that they are migrating neural crest cells. The TrkB-positive cells lack neural markers at this stage; however, they migrate with other neurally differentiating cells that are TrkA and TrkC-positive. By St. 29/30, TrkB-positive cells begin to express the neural specific markers Hu C/D and Islet-1; eventually, all TrkB positive cells commence neural differentiation. By St. 34, TrkB and TrkC staining are lost. BDNF transcript expression parallels that of TrkB. In the <span class="yellow">mouse</span>, TrkB-positive cells surround newly formed sympathetic ganglia and a small number of TrkB positive cells that co-express tyrosine hydroxylase are seen within ganglia between E13.5-15. In cell culture, many cells from St. 29–30 <span class="yellow">chicken</span> lumbar sympathetic ganglia express neural markers and are dividing, indicating that they are sympathoblasts. Sympathoblasts and neurons require both nerve growth factor and neurotrophin-3 for survival. BDNF increases the number of cells expressing neural markers in culture by increasing number of cells that incorporate bromodeoxyuridine. In contrast, most TrkB-positive sympathetic cells in vivo are not actively proliferating between E6–E8.<br><br>Conclusion
Developing paravertebral sympathetic ganglia in avian and <span class="yellow">murine</span> embryos contain a subpopulation of sympathoblasts that transiently express TrkB and ultimately commence neuronal differentiation. These TrkB expressing sympathoblasts are not actively dividing in vivo; yet, when placed in vitro, will divide in response to BDNF. This suggests that the availability of BDNF in vivo fails to reach a threshold necessary to induce proliferation. We suggest that excess TrkB stimulation of sympathoblasts in vivo may lead to the genesis of neuroblastoma.<br><br><br><br>Background
Neural crest cells destined to become paravertebral sympathetic neurons proliferate and differentiate during migration and gangliogenesis. In <span class="yellow">chicken</span> embryos, migrating neural crest cells express catecholamines at Hamburger/Hamilton Stage (St.) 19, and these cells form the primary sympathetic chain dorsolateral to the aorta at St. 22 (E3.5) [1]. Between St. 23 (E4) and St. 28 (E6), these cells disperse and undergo a secondary migration to form the paravertebral sympathetic chain that resides ventral to the spinal cord and dorsal root ganglion [1]. After ganglia coalesce, sympathoblasts express markers of neuronal differentiation, such as Q211 and tyrosine hydroxylase (TH), at a time when they also incorporate [3H]-thymidine [2]. Time lapse photography has shown that cultured E15.5–E16.5 sympathetic neurons from <span class="yellow">rat</span> embryos extend axons while they divide [3-5]. Although proliferation appears to be an important process to expand the sympathetic neuron population during differentiation, the mechanisms that guide sympathoblast proliferation have not been identified.
The development of sympathetic neurons is guided by neurotrophins. Neurotrophin-3 (NT-3) binds to its receptor, TrkC, to promote the survival of cultured sympathoblasts from early lumbar paravertebral ganglia [6]. Nerve growth factor (NGF) signals through its receptor, TrkA, to promote the survival of sympathetic neurons upon target innervation [7]. There are severe sympathetic defects in the superior cervical ganglion of individual NT-3 and NGF knockout <span class="yellow">mice</span> [8-10]. Furthermore, there is no additional cell death in the superior cervical ganglion of NT-3 and NGF double knockout <span class="yellow">mouse</span> embryos, suggesting that all of the neurons are dependent on both neurotrophins for survival [11]. There is also an increase in sympathetic neuron cell death in TrkA knockout <span class="yellow">mice</span> [12]. However, in TrkB and BDNF knockout <span class="yellow">mice</span>, there is no apparent phenotype in the superior cervical ganglion, and there is little evidence that TrkB or BDNF is expressed in sympathetic ganglia. Thus, it is generally thought that TrkB and BDNF have little or no roles in guiding the development of sympathetic neurons.
In addition to their developmental functions, neurotrophin receptors regulate cell behavior in neuroblastoma, a tumor found in sympathetic ganglia and adrenal medulla. Tumors that express TrkA often spontaneously regress, while those that express TrkB and its ligand, brain-derived neurotrophic factor (BDNF), grow aggressively, are invasive, and fail to respond to chemotherapeutic agents [13]. The presence of TrkA in neuroblastoma tumors is consistent with its expression in developing sympathetic neurons, and suggests that regressive neuroblastoma tumors arise from early sympathetic neurons that express TrkA. The function of TrkB in early sympathetic development is unknown, which makes understanding the etiology of aggressive neuroblastoma tumors difficult. Based on its function in neuroblastoma tumors, we hypothesize that BDNF and TrkB expression in differentiating sympathoblasts is responsible for expanding the neuronal population through proliferation.
We sought to determine whether BDNF and TrkB are involved in sympathetic development. We report that during early embryonic development, TrkB is expressed in a subset of differentiating sympathoblasts in both avian and <span class="yellow">murine</span> embryos. We also find that BDNF promotes the proliferation of TrkB-positive sympathoblasts in cell culture. However, the majority of TrkB positive cells in vivo fail to take up bromodeoxyuridine (BrdU) over a 24 hr period, suggesting that endogenous BDNF concentrations do not reach a threshold necessary to stimulate proliferation of sympathoblasts. Shortly after all of the TrkB positive cells commence neuronal differentiation, TrkB immunoreactivity is lost. These results suggest that prolonged expression and/or activation of TrkB signaling at these early stages may be an early event triggering the formation of neuroblastoma.<br><br>Results
TrkB is expressed during migration of neural crest cells to sympathetic ganglia
We first determined whether TrkB is expressed in neural crest-derived cells in the region ventral to the spinal cord and dorsal root ganglia where sympathetic ganglia coalesce between Hamburger/Hamilton Stages (St.) 25–28/29. To identify cells that have commenced neuronal differentiation, transverse sections of the lumbar spinal column region were stained with antibodies against Hu C/D [14], a neuronal-specific RNA-binding protein, or Islet-1, a transcription factor found in sympathetic neurons [15]. We found that Hu C/D and Islet-1 are expressed in the same cells both in vivo and in vitro throughout sympathetic development. In experiments done between St. 25 and 28, Islet-1 staining appeared weaker than Hu C/D staining, and thus we used Hu C/D to identify differentiating neurons at these stages. At later stages, Islet-1 was used to facilitate the identification of neurons because of the nuclear location of its immunoreactivity.
Cells expressing Hu C/D are first detected at St. 25 ventral to the spinal cord and dorsal root ganglion and lateral to the dorsal aorta (Figure 1A, 1B). By St. 26, the number of cells that express Hu C/D in this region increases dramatically (Figure 1C). TrkB-expressing cells first appear at St. 27 in the same region and are adjacent to Hu C/D-positive cells (Figure 1D, 2A, 2H). TrkB-positive cells co-localize with a neural crest marker, HNK-1 (Figures 2B, 2C, 2D). The Hu C/D-positive cells in this region are likely to be sympathetic neurons, since they appear in the region where sympathetic ganglia form and express tyrosine hydroxylase, a rate-limiting enzyme in the synthesis of catecholamines (Figures 2E, 2F, 2G). At St. 28/29, the cells begin to coalesce ventral to the dorsal root ganglion and the Hu C/D-positive cells and TrkB-positive cells remain as two separate cell populations (Figure 1E); however, shortly afterwards, all of the TrkB-positive cells begin to express Islet-1 (Figure 3B) and Hu C/D (data not shown).<br><br>Developmental regulation of TrkA, TrkB, TrkC, and BDNF expression
In contrast to TrkB, the other neurotrophin receptors, TrkA and TrkC, are co-expressed in both Hu C/D-positive and Hu C/D-negative cells at St. 27 (Figures 2I, 2J). We find that approximately 30% of the Islet-1-positive cells express TrkA (Figure 3A), while 50% express TrkB (Figure 3B) and 100% express TrkC (Figure 3C) at St. 29/30 (E6.5). Thus, all developing neurons express TrkC in combination with either TrkA or TrkB. By St. 31 (E7), the number of TrkA-positive, Islet-1-positive cells increases to 100% (Figure 3D) and immunoreactivities for both TrkB and TrkC appear dispersed (Figure 3E, 3F). By St. 34 (E8), TrkA expression is well-sustained (Figure 3G) and TrkB and TrkC immunoreactivities are lost (Figure 3H, 3I). We also examined the early development of <span class="yellow">murine</span> sympathetic ganglia (Figure 4). At E13, the newly formed lumbar sympathetic ganglia can be observed ventral to the spinal cord and notochord by their staining for Hu C/D and TH (Figure 4A). TrkB-positive cells can be seen surrounding developing ganglia, as well as in occasional cells within the ganglia (Figure 4C, D). These TrkB-positive cells within the ganglia co-express TH and are seen at a frequency of 1–2 cells per section starting at E13 (Figure 4A–D) and are still present at E15.5 (data not shown).
In neuroblastoma cells, BDNF is co-expressed with TrkB, suggesting that autocrine stimulation is a means by which proliferation is sustained in the transformed cells. To test whether BDNF, the ligand for TrkB, was present in embryonic <span class="blue">chick</span> sympathetic ganglia, we used quantitative real-time PCR with TaqMan probes to determine the relative abundance of BDNF transcripts in total RNA extracted from lumbar sympathetic ganglia at St. 29/30 (E6.5), St. 31 (E7), St. 34 (E8), and E9. BDNF expression within the ganglia parallels that of TrkB: BDNF mRNA expression levels are highest at St 29/30 (E6.5), and these levels decrease 2-fold at St. 31 (E7) and St. 34 (E8; Figure 5). By E9, BDNF levels are 7 times lower than at St. 29/30 (E6.5; Figure 5).<br><br>NT3 and NGF promote survival of differentiating sympathetic neurons in culture
To determine the effect of neurotrophins, we cultured cells dispersed from lumbar sympathetic ganglia at St. 29/30 (E6.5) because, at this stage, ganglion formation is complete, the number of TrkA-, TrkB-, and TrkC-positive cells have peaked, and all Trk-expressing cells have initiated neural differentiation. First, we identified markers expressed by acutely isolated cells. As shown in Table 1, 80–91% of the cells are p75 neurotrophin receptor (NTR)-positive, indicating that most of the cells are neural crest-derived and little mesenchymal contamination is introduced by the isolation procedure. In addition, 28–33% of the cultured cells express the neural marker Hu C/D. Approximately half of these Hu C/D-positive cells express TrkB. Conversely, all of the TrkB positive cells express Hu C/D. These TrkB-positive cells comprise approximately 14–17% of the total cell population.
We then determined how many of the acutely isolated cells were proliferating by incubating them for 12 hrs in BrdU-containing medium. For these experiments, we identified differentiating neurons with the transcription factor Islet-1 because this marker labels nuclei, thus it co localizes with any BrdU that has been incorporated into the DNA, allowing us to determine whether the cell had undergone S-phase of the cell cycle. After 12 hrs in BrdU, 59% of Islet-1-positive nuclei stain for BrdU immunoreactivity. Thus, cultures of St. 29/30 sympathetic ganglia contain many cells that proliferate while exhibiting markers of neuronal differentiation, confirming previous observations [2]. We call these dividing neuronal precursors sympathoblasts. The remaining non-BrdU incorporating, Islet-1 positive cells are likely to be post mitotic neurons.
Finally, we determined the trophic requirements of St. 29/30 (E6.5) sympathetic neurons and sympathoblasts. We monitored cultures over a three day period after plating and counted the number of phase bright cells with neurites, a morphological feature of both neurons and sympathoblasts. In the absence of trophic factors, more than 2/3 of the cells die by 24 hours in culture and BDNF, NT-3, or NGF alone is not sufficient to promote survival (Figure 6). However, NGF together with NT-3 supports the survival of a significantly larger number of cells (Figure 6). For the subsequent experiments, all neurons were cultured with 25 ng/ml NT-3 and 1 μg/ml 7S NGF to optimize survival.<br><br>BDNF promotes proliferation of TrkB-positive sympathetic neurons in culture
To determine the effects of BDNF, cultures of cells from St. 29/30 (E6.5) sympathetic ganglia were supplemented with 200 ng/ml BDNF and the number of neurons and sympathoblasts were counted at 24, 48, and 72 hours using phase microscopy (Figure 7A). A 1.6-fold increase in the number of neurons due to BDNF is observed by 24 hours and this number does not increase further at 48 or 72 hours. This effect of BDNF is concentration-dependent with an EC50 of 75 ng/ml (Figure 7B).
To test whether the increase in the number of neurons and sympathoblasts caused by BDNF is due to the differentiation of pluripotent neural crest cells, we quantified the effects of BDNF on the number of neurally differentiating cells (Hu C/D-positive) versus the number of non-neuronal cells (Hu C/D-negative) after identifying all neural crest-derived cells by staining for p75NTR in St. 29/30 (E6.5) cultures. If BDNF increases the number of neurons and sympathoblasts by inducing a non-neuronal cell to express Hu C/D, then we expected that the total cell number would remain the same and that there would be a decrease in the number of non-neuronal cells as well as a corresponding increase in the number of neurons. After 24 hours, BDNF significantly increases the number of p75NTR-positive cells as well as the number of Hu C/D-positive cells (Figure 8A). However, there was no statistically significant change in the number of non-neuronal cells. Thus, it is unlikely that BDNF increases the number of neurons and sympathoblasts by inducing differentiation of non-neuronal cells.
To determine whether the increase in the total number of neurons and sympathoblasts is caused by BDNF-induced proliferation, control and BDNF-treated sympathetic cultures were exposed to BrdU for 12 hours after plating, and the number of cells that incorporated BrdU into their DNA was determined after 24 hours in culture. Even in the control condition, a number of cells in the culture are dividing, giving a high baseline of BrdU incorporation (Figure 8B). When BDNF is added, the total number of BrdU positive cells increases approximately 1.6-fold (Figure 8B). This BDNF-induced increase in the total number of BrdU positive cells occurs in sympathoblasts because the number of Islet-1-positive nuclei from control cultures that label with BrdU is 268 ± 59 and BDNF treatment raises this number to 424 ± 80, which corresponds to an increase of 1.6-fold. This accounts for the 1.6-fold increase in total neuron number and total BrdU-positive cells described above. We then confirmed that BDNF acts on TrkB-positive cells: BDNF increases the number of TrkB-expressing cells that incorporate BrdU 2.6 – 4-fold over control (Table 2) and it also increases the overall number of TrkB-positive cells 2 – 2.5-fold over control (Table 2). BDNF does not increase the number of BrdU-positive, TrkB-negative cells or the overall number of TrkB-negative cells (Table 2). In further support that BDNF acts directly on TrkB-expressing cells, an antibody directed against the extracellular domain of TrkB completely prevents the effect of BDNF in promoting proliferation of TrkB-positive, but not TrkB-negative cells (compare Figure 9A to 9B). Thus, the effect of BDNF is restricted to the population that expresses TrkB, which are developing sympathoblasts.
To determine whether TrkB-positive cells are actively proliferating in vivo, embryos were injected with BrdU at St. 27 and harvested at St. 29, approximately 24 hrs later. The majority (85–90%) of TrkB-positive cells do not incorporate BrdU into their nuclei under basal conditions in vivo (Figure 10), although a few TrkB positive cells with labeled nuclei could be observed (arrows). This contrasts with our observation that 71–76% of TrkB-positive cells incorporate BrdU in culture after treatment with BDNF (Table 2), suggesting that endogenous BDNF does not achieve a threshold sufficient to support a high level of sympathoblast proliferation in vivo.<br><br>
Discussion
We report that the neurotrophin receptor TrkB is expressed in a subset of embryonic sympathoblasts during the early development of lumbar paravertebral sympathetic ganglia in <span class="yellow">chicken</span> and <span class="yellow">mouse</span> embryos. In the <span class="yellow">chicken</span>, TrkB expression is transient, and completely lost by St 34 (E8). Since BDNF induces the proliferation of sympathoblasts in cell culture, yet in vivo there is little proliferation observed in TrkB-positive cells in nascent ganglia, we propose that if TrkB activation becomes unregulated by excess BDNF or constitutive phosphorylation of TrkB [16], this transient population of TrkB-positive sympathoblasts may trigger the genesis of neuroblastoma, a childhood tumor found in the paravertebral chain and adrenal medulla.
The two populations of sympathoblasts that we observe support previous findings of heterogeneity among developing sympathetic neurons and neural crest cells. Early sympathetic ganglia contain at least two subpopulations: early differentiating neurons that lack TrkB expression and express TrkA and TrkC, and late differentiating sympathoblasts that express TrkB. Explant cultures of sympathetic ganglia from E16 <span class="blue">chick</span> embryos give rise to two neuronal populations: one that remains close to the explant, and one that migrates away from the explant [1]. In addition, early neuronal subpopulations have been observed in cultures of neural crest cells from St. 13/14 quail embryos as evidenced by the expression of neuronal cell type-specific gangliosides [17]. Perhaps these different subpopulations will ultimately give rise to the two neurochemically distinct populations found in lumbar sympathetic ganglia: the noradrenergic, NPY-containing neurons that innervate internal organs and enteric ganglia and the cholinergic, VIP-containing neurons that innervate vasculature in the hind limbs.
The effects of BDNF and TrkB deletion and over expression have been studied on superior cervical ganglion and preganglionic neurons in thoracic segments of the spinal column, but not on paravertebral sympathetic neurons. In the superior cervical ganglion, an increase in the number of neurons of BDNF null <span class="yellow">mice</span> is likely due to apoptosis induced by BDNF via p75NTR [18]. In contrast, the responses of paravertebral sympathetic neurons to BDNF are complex and subtype dependent. Over expression of BDNF leads to an increase in the number of noradrenergic fibers innervating the erector pilli muscles of hair follicles, while noradrenergic fibers innervating blood vessels were unaffected [19]. If our results indicating that BDNF promotes proliferation of TrkB-positive sympathoblasts in the <span class="yellow">chicken</span> embryo can be extrapolated to the subset of TrkB-positive sympathoblasts in <span class="yellow">murine</span> ganglia, then these TrkB-positive cells may be neurons destined to innervate the erector pilli. In other studies, TrkB null <span class="yellow">mice</span> showed no changes in morphology or cell number in superior cervical ganglia [12] or in the intermediolateral column [20]; but this may not be predictive of a phenotype in the lumbar paravertebral chain. It is thus possible that BDNF/TrkB signaling could play a specific role in other regions of the paravertebral sympathetic chain, such as the lumbar region. However, if TrkB-positive cells are not normally actively proliferating in vivo, then it would not be surprising that the development of the paravertebral sympathetic chain is not disrupted in TrkB or BDNF null <span class="yellow">mice</span>. It may be more informative to examine <span class="yellow">mice</span> that over express BDNF on a promoter that targets expression to embryonic lumbar ganglia. Unfortunately, such <span class="yellow">mice</span> do not exist.
Our findings that the St. 29/30 (E6.5) sympathoblasts are dependent on both NT-3 and NGF for survival in culture are consistent with previous work on <span class="yellow">mouse</span> sympathoblasts from the superior cervical ganglion [11]. In these studies, NT-3 and NGF deletion separately led to a decrease in the number of sympathetic neurons at E17.5 compared to control. Deletion of both NT-3 and NGF together did not enhance cell death. In contrast, cultured <span class="yellow">rat</span> superior cervical ganglion sympathetic neurons respond to NT-3 at E14.5 and then to NGF at E19.5, although time points in between were not analyzed [6].
In addition to promoting survival, NT-3, NGF, and BDNF also induce proliferation of various neuronal precursors at different stages of development. NT-3 can promote the incorporation of [3H]-thymidine into cultured quail neural crest cells from the trunk region [21,22], Later in <span class="yellow">rat</span> sympathetic development, NT-3 supports survival of neurons, but does not promote proliferation [6], which is consistent with our results. NGF promotes an increase in BrdU incorporation from 25% to 35% in the DRG cervical segment 2 in the <span class="blue">chick</span> embryo [23]. In <span class="yellow">chicken</span> embryos that are treated with NGF in ovo at St. 18 and 21, there is an increase in BrdU uptake after formation of the primary sympathetic chain at St. 23 [24]. Since NGF does not appear to affect proliferation of St. 29/30 (E6.5) <span class="blue">chick</span> sympathoblasts, NGF may only promote proliferation in primary, but not secondary chain sympathoblasts. Motor neuron progenitors in the ventral neural tube from the <span class="blue">chick</span> embryo express TrkB and when ventral neural tube explants are treated with BDNF, there is an increase in the number of motor neurons produced and BrdU incorporation [25]. BDNF also promotes the proliferation of cultured neuroblastoma cells [13]. Taken together, these results are consistent with our findings that NT-3 and NGF do not promote proliferation of St. 29/30 (E6.5) sympathoblasts, and support the assertion that BDNF promotes proliferation of TrkB-positive sympathoblasts in culture.
Our observations suggest a transient function of TrkB during early sympathetic development in supporting proliferation of this early subpopulation of sympathoblasts. However, the in vivo labeling suggests that only a minority (10–20%) of this population is dividing during the window that TrkB is expressed. In light of the very strong proliferative effect produced in cell culture, these TrkB expressing cells could respond more strongly if endogenous BDNF rises to higher levels, or if the mechanism that down regulates TrkB expression becomes nonfunctional. Such events could trigger an early proliferative event that leads to a cascade of changes that initiates transformation of cells to neuroblastoma. Thus, these early TrkB expressing cells help solve the puzzle as to why TrkB is expressed in aggressive and invasive forms of neuroblastoma, particularly because BDNF induces cultured neuroblastoma cells to become more proliferative, invasive, angiogenic, and resistant to chemotherapeutic reagents than untreated cultures [13]. Future studies will determine whether constitutive expression of BDNF and TrkB in the <span class="blue">chick</span> embryo sustains proliferation of differentiating sympathoblasts.<br><br>Conclusion
We have identified a time point during development when differentiating lumbar sympathetic neurons transiently express TrkB and proliferate in response to high concentrations of BDNF in culture. These studies suggest that elevated BDNF expression above basal levels and signaling through TrkB may be a mechanism that contributes to the onset of neuroblastoma. A further understanding of the two populations of sympathetic neurons and the fate of the TrkB-positive cells will provide additional insight into the development of paravertebral sympathetic ganglia and the genesis of neuroblastoma.<br><br>Methods
Preparation of tissue for immunohistochemistry
The lumbar spinal column and surrounding tissues were dissected from <span class="yellow">chicken</span> embryos at the indicated stages and placed in Zamboni's fixative (4% (w/v) paraformaldehyde, 15% (v/v) picric acid in 0.1 M sodium phosphate buffer, pH 7.4) for two hours at room temperature. <span class="yellow">Mouse</span> embryos at 13–15 days post-coitus were collected according to an IACUC-approved protocol to Dr. L. Sherman at the Oregon Health and Science University. The <span class="yellow">mouse</span> embryos were immersion-fixed in Zamboni's fixative overnight at 4 degrees C then washed with phosphate buffered saline (PBS; 130 mM NaCl, 20 mM sodium phosphate buffer, pH 7.4). Fixed tissues were equilibrated in 30% sucrose in 1× phosphate-buffered saline (PBS). Fixed <span class="yellow">mouse</span> embryos were shipped to Vermont in sucrose. Transverse 30 μM sections of the spinal columns were cut at on a Microm HM cryostat (knife temperature: 16°C; object temperature: 23°C) and collected on Superfrost Plus slides (Fisher). Sections were dried at room temperature, washed in 1× PBS and incubated overnight in blocking buffer (1× PBS consisting of 10% (v/v) heat-inactivated <span class="yellow">horse</span> serum (Invitrogen/Gibco), 0.5% Triton X-100 (Sigma), and 0.1% sodium azide (Fisher)).<br><br>Immunohistochemistry
Sections were incubated with primary antibodies overnight at 4°C, followed by incubation with secondary antibodies for 2 hours at room temperature. Primary antibodies used were: <span class="yellow">rabbit</span> anti-p75 (1:1500, generous gift from Louis Reichardt, UCSF [26]), <span class="yellow">mouse</span> IgG2b anti-Hu C/D, (1:250, Molecular Probes); <span class="yellow">mouse</span> IgG1 anti-Islet-1, (1:10, Developmental Studies Hybridoma Bank); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkA (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkB (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkC (1:500) (all Trk antibodies were generous gifts of Dr. Louis Reichardt, UCSF [26-28]); <span class="yellow">mouse</span> anti-HNK-1 (1:50, Developmental Studies Hybridoma Bank); <span class="yellow">mouse</span> IgG2a anti-tyrosine hydroxylase (1:10, Developmental Studies Hybridoma Bank), <span class="yellow">sheep</span> anti-BrdU (1:100, Biodesign International), <span class="yellow">rabbit</span> anti-tyrosine hydroxylase (1:100, Chemicon), and <span class="yellow">goat</span> anti-TrkB (1:1000, R&D Systems). Immunofluorescence was imaged using a Nikon C1 confocal mounted on a Nikon Eclipse E800 microscope with a 10× Plan Apo (NA 0.785) air objective or a 60× Plan Apo (NA 1.4) oil objective lens, E7-C1 software, and UV, Argon, and He/Ne lasers exciting at 408, 488, and 543 nm and emitting at 404 500–530, and 555–615 nm, respectively. A Nikon Eclipse E800 microscope in the nearby COBRE Molecular/Cellular Core Facility was used for counting immunofluorescent cells at 200× using epifluorescence optics.<br><br>RNA Extraction/cDNA synthesis
Sympathetic ganglia were removed from <span class="blue">chick</span> embryos and RNA was isolated using TriReagent (Molecular Research Center), an acidified guanidinium with phenol extraction method [29]. RNA was transcribed to cDNA using oligo-dT with Superscript II Reverse Transcriptase (Invitrogen) at 42°C for 1 hour.<br><br>Real-time PCR
Relative RNA levels were determined using quantitative real-time PCR with an ABI 7500 Fast Real Time PCR System. TaqMan probes were used to quantify the progression of the PCR reaction and reactions were normalized using the constitutively expressed gene <span class="blue">chick</span> ribosomal binding protein s17 (CHRPS). The sequences were used for primer/probes sets: for BDNF: forward: 5'-AGCCCAGTGAGGAAAACAAG-3', reverse: 5'-ACTCCTCGAGCAGAAAGAGC-3', probe: 5'-[6-FAM]-TACACATCCCGAGTCATGCTGAGCA-[BHQ]-3'; for CHRPS (<span class="blue">chick</span> ribosomal binding protein S-17): 5'AACGACTTCCACACCAACAA3', reverse: 5'CTTCATCAGGTGGGTGACAT3', probe: 5'-[6-FAM]-CGCCATCATCCCCAGCAAGA [BHQ]-3'. Primers and probes were synthesized by Operon Technologies, Inc (Alameda, CA). The primers for BDNF were validated against primers for CHRPS according to an Applied BioSystems protocol by serially diluting the target cDNA 1:10, determining the cycle threshold (Ct) for each reaction, and plotting the Ct versus log concentration. Slopes of the resulting lines were calculated and primers were accepted if their Ct slopes were between -3.2 and -3.4 (a perfect efficiency of 1.0 yields a slope of -3.3). To analyze the data, the delta Ct method of relative quantification was used, where the Ct of Chrps was subtracted from the Ct of the gene of interest (Delta Ct) and the arbitrary units of mRNA were expressed as 10000/2^(Delta Ct).<br><br>Cell culture
Sympathetic neurons were cultured as previously described [30] with a few modifications. Sympathetic ganglia were removed from the lumbar region of the paravertebral chain of St. 29/30 (E6.5) <span class="blue">chick</span> embryos and placed in Modified Puck's solution with glucose (MPG). The cells were dissociated by incubation of sympathetic ganglia with 0.1% trypsin in MPG at 37°C for 10 minutes followed by triturating with a fire polished 9" Pasteur pipette. Cells were then resuspended in Dulbecco's Modified Eagle Medium (DMEM) consisting of 10% <span class="yellow">horse</span> serum, 2% fetal <span class="yellow">calf</span> serum, and 10 mg/ml penicillin/streptomycin. For neurotrophin studies, the culture medium was supplemented with 25 ng/ml NT-3 (R & D Systems) and 1 μg/ml 7S NGF (Alomone Labs) upon plating, and 50 ng/ml, 100 ng/ml, or 200 ng/ml BDNF (R & D Systems) once the cells adhered to the wells. Cells were plated on poly-D-lysine/laminin coated wells or cover slips (Fisher) as previously described [30].<br><br>Quantification of neurons and sympathoblasts using phase microscopy
Embryonic sympathoblasts and neurons are small, phase bright cells with neurites. The total number of cells with neurites the length of two cell bodies were counted in 10 non-overlapping fields of view evenly spaced in a grid-like pattern across the bottom of a well from a 24 well plate at 200× using a Nikon Eclipse TE200 microscope.<br><br>BrdU labeling
For in vitro studies, approximately 2 hours after plating cells from St. 29/30 (E6.5) sympathetic ganglia, cells were labeled with 10 μM bromodeoxyuridine (BrdU, Sigma) for 12 hours at 37°C. Following this labeling period, cells were incubated in complete medium without BrdU for an additional 10 hrs. Cells were then fixed in Zamboni's fixative for 30 min at room temperature and rinsed with 1× PBS. For in vivo studies, 25 μg BrdU was injected into the amnion of <span class="blue">chick</span> embryos at St. 27. The cells and sections were denatured with 2 N HCl at 37°C for 1 hr, and were then neutralized with 0.1 M borate buffer, pH 8.5, for 10 min at room temperature. Immunochemistry was performed as described above.<br><br>
Abbreviations
BDNF, brain-derived neurotrophic factor; BrdU, Bromodeoxyuridine; DA, dorsal aorta; DMEM, Dulbecco's Modified Eagle's Medium; DRG, dorsal root ganglion; E, embryonic day; HS, <span class="yellow">horse</span> serum; MPG, Modified Puck's solution with glucose; NGF, nerve growth factor; NC, notochord; NT, neural tube; NT-3, neurotrophin-3; NTR, neurotrophin receptor; PBS, phosphate-buffered saline; SC, spinal cord; SCG, superior cervical ganglion; SEM, standard error of the mean; SG, sympathetic ganglion; St., stage; w/v, weight/volume; v/v, volume/volume.<br><br>Authors' contributions
JAS designed the experiments, performed the experiments, analyzed the data, and wrote the manuscript. GLSS contributed intellectually to the conception and design of this study, and assisted in the interpretation of the results. RN supervised the study, participated in the design of experiments, edited the manuscript, and obtained funding for the project. All authors read and approved the final manuscript.<br><br>
<h3>pmcA324075</h3>Structural organization of a 17 KB segment of the alpha 2 collagen gene: evaluation by R loop mapping.
Abstract
A recombinant phage, SpC3, containing a 17 kb genomic DNA insert representing approximately 60% of the 3' portion of the <span class="yellow">sheep</span> collagen alpha 2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 alpha 2 coding sequences (exons) were mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region required for coding the interspersed 3 kb of alpha 2 collagen genetic information was 5.6 fold longer than the corresponding alpha 2 mRNA coding sequences.Images<br><br><br><br><br><br>
 Nucleic Acids Research <br><br> Structural organization of a 17 KB segment of the a2 collagen gene: evaluation by R loop mapping Millie P.Schafer, Charles D.Boyd, Paul Tolstoshev and Ronald G.Crystal <br><br> Pulmonary Branch, National Heart, Lung and Blood Institute, National Institutes of Health, Bethesda, MD 20205, USA <br><br> Received 13 February 1980 <br><br> ABSTRACT <br><br> A recombinant phage, SpC3, containing a 17 kb genomic DNA insert rep<br><br> resenting approximately 60% of the 3' portion of the <span class="yellow">sheep</span> collagen a2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 a2 coding sequences (exons) were <br><br> mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region <br><br> required for coding the interspersed 3 kb of a2 collagen genetic information was 5.6 fold longer than the corresponding a2 mRNA coding sequences. <br><br> INTRODUCTION <br><br> Type I collagen, composed of two al(I) and one a2 polypeptide chains, is the most abundant of the five known mammalian collagen types. It is a major extracellular constituent of tissues such as bone, tendon, skin and lung where, because of its great tensile strength, it plays an important <br><br> role in tissue structure and function. The polypeptides comprising type I <br><br> collagen are synthesized in long, precursor forms, referred to as pro al(I) and pro a2 chains,1 each containing approximately 1500 amino acid residues (1). <br><br> 1 The terminology for the primary translation products of the various <br><br> collagen messenger RNAs is still in a state of flux. It is known that al(I) and a2 chains are synthesized in precursor forms; these are currently termed pro al(I) and pro a2 chains (1). However, recent studies (43,44) have shown that the actual primary translation product of al(I) mRNA is somewhat larger <br><br> than the pro al(I) chain; this translation product has been termed a "pre-pro al(I) chain." Although the corresponding "pre-pro a2 chain" likely exists, it has not yet been characterized. For simplicity, therefore, until the <br><br> proper terminology for the primary translation product is clarified, we have <br><br> chosen to use the terms "al(I)" and "a2" to refer to the type I collagen mRNAs and collagen structural gene sequences. <br><br> ?) IRL Press Umited, 1 Falconberg Court, London W1V 5FG, U.K. <br><br> Volurne 8 Number 10 1980 <br><br> 2241 <br><br> Nucleic Acids Research <br><br> There is increasing evidence that the quantities of al(I) and a2 <br><br> chains produced by cells may be regulated, in part, at the genomic level <br><br> (1). However, to understand the processes which mediate their expression, <br><br> it is necessary to understand the structural organization of these collagen genes and the various enzymes that modify the collagen chains after its <br><br> synthesis (1). As an initial approach to this problem, we have recently <br><br> isolated a recombinant <span class="yellow">bacteriophage</span>, termed SpC3, containing approximately 60% of the a2 gene for type I collagen. Partial characterization of this <br><br> a2 gene demonstrated that, like most other structurally evaluated eukaryotic genes, the coding sequences are interspersed with intervening sequences (2). The present study further characterizes this portion of the a2 <br><br> gene by utilizing electron microscopic R loop analysis to map the coding regions (exons) and intervening sequences (introns) contained within the <span class="yellow">sheep</span> genomic insert of SpC3. The data suggests that the structural <br><br> organization of this portion of the a2 collagen gene represents 3 kb of coding information that is interspersed in a complex fashion with 17 introns over 17 kb of genome. <br><br> MATERIALS AND METHODS <br><br> Isolation and characterization of a2 collagen recombinant clone, SpC3. <br><br> The <span class="yellow">sheep</span> a2 collagen recombinant clone, SpC3, containing a portion of the <span class="yellow">sheep</span> a2 gene, was isolated as previously described (2). Briefly, <br><br> high molecular weight fetal <span class="yellow">sheep</span> liver DNA was extracted by the method of Blin and Stafford (3), and 15-20 kb DNA fragments, resulting from partial Eco RI digestion, were isolated and ligated to the left and right arms of Charon 4A. The recombinant DNA was then packaged in vitro and the <span class="yellow">sheep</span> <br><br> genomic library amplified (4-6). Screening of the <span class="yellow">sheep</span> genomic library was <br><br> conducted utilizing [32 P]-labeled fetal <span class="yellow">sheep</span> tendon type I collagen cDNA and <br><br> several positive recombinant phages were isolated. One recombinant, containing a 17 kb <span class="yellow">sheep</span> genomic insert, was demonstrated to have coding sequences corresponding to 60% of the 3' end of a2 mRNA. Prior studies showed that the insert in SpC3 was not due to ligation of noncontiguous restriction fragments and/or genetic rearrangement during the cloning process (2). Electron mi croscopy. <br><br> To visualize the recombinant <span class="yellow">bacteriophage</span> DNA structure complementary to a2 collagen mRNA, 1 iig of SpC3 DNA was lyophilized with 1 ig or 100 ng of mRNA <br><br> to give DNA:mRNA ratios of 1:1 and 10:1, respectively. These were then dissolved in 10 i1 of R loop buffer [70% (v/v) deionized formamide, 0.1 N-[tris (hydroxy<br><br> 2242 <br><br> Nucleic Acids Research <br><br> methyl)methyl] glycine (tricine)-NaOH, pH 8.0, 0.5 M NaCl, and 0.01 M ethyl<br><br> enediaminetetraacetate (EDTA)], incubated in sealed capillary tubes at 520 for 14 hr (7), and then diluted 1:10 with R loop buffer. Two spreading methods were used to subsequently prepare the hybrids for visualization in the electron microscope. <br><br> 70% formamide method. Immediately before spreading onto a deionized <br><br> water hypophase, the nucleic acid mixtures were diluted an additional 10-fold in R loop buffer and cytochrome c, 100 vg/ml, was added. <br><br> Urea-formamide method.   The nucleic acid solutions were spread using a modification of the method of Westphal and Lai (8,9). The hyperphase contained 55% formamide, 2.6 M urea, 0.009 M EDTA, 0.09 M tricine-NaOH, pH 8.0, and the nucleic acid mixture (approximately 0.1 to 1.0 ig/ml). <br><br> The mixture was heated at 40? for 30 sec, placed in ice water, and then allowed to reach room temperature. Cytochrome c, 100 pg/ml, was added immediately before spreading onto a deionized water hypophase. <br><br> The nucleic acid-protein films from both methods were absorbed onto <br><br> parlodion-coated grids, stained with uranyl acetate (10), dehydrated in 90% ethanol and rotary-shadowed with platinum-palladium (80:20) at an angle of 50. Micrographs were taken with a Siemens Elmiskop 101 electron microscope at an original magnification of 10,000 and an accelerating voltage of 60 kv. Nucleic acid lengths were measured at a final magnification of 43,000 with a <br><br> Hewlett-Packard 9810A calculator equipped with a 9864A digitizer using pBR322 and fX-174 double and single stranded DNA, respectively, as internal length standards. <br><br> Approximately 10,000 hybrid molecules were screened for these studies; approximately 200 represented molecules containing unambiguous regions <br><br> appropriate for quantitative analysis. The mean lengths of each intron and exon were determined for each method and the data expressed as mean ? <br><br> standard deviation. Nucleic acid lengths of < 50 base pairs could not be <br><br> accurately determined using the methods outlined above. Thus, in the cases <br><br> where introns or exons were of this length or less, accurate error estimates could not be made. <br><br> Bi ohazard precauti ons. <br><br> The construction and screening of the <span class="yellow">sheep</span> genomic library together with amplification of pCg45 and preparation of high titre lysates of chimaeric <br><br> Charon 4A were performed under the physical and biological containment levels specified by the NIH guidelines for recombinant DNA research (11). <br><br> 2243 <br><br> Nucleic Acids Research <br><br> RESULTS <br><br> Location and orientation of the intervening and a2 mRNA coding sequences in the SpC3 recombinant DNA. <br><br> Hybridization of fetal <span class="yellow">sheep</span> tendon type I collagen mRNA to SpC3 DNA <br><br> yielded a complex R loop pattern in the region where the <span class="yellow">sheep</span> genomic insert was expected to be ligated to the arms of the lambda vector, Charon 4A. To <br><br> confirm that these hybridization events were restricted to the inserted <span class="yellow">sheep</span> DNA fragment, duplex DNA strands on both sides of this hybridization area were measured. Using pBR322 as an internal length standard, the left arm of the phage was found to contain 21,780 ? 1500 bp, and the right arm, 11,950 ? <br><br> 1030 bp, corresponding to the report values for the left and right arms, respectively, of Charon 4A DNA (5,6). <br><br> The hybridization of SpC3 to a2 mRNA resulted in the destabilization of <br><br> the inserted helical duplex DNA fragment such that one DNA strand was displaced (Figure 1). Occasionally, the displaced DNA strand was mostly absent, probably due to fragmentation induced by accidental mechanical shearing. This event considerably enhanced the visualization and ordering of the introns. <br><br> As the a2 mRNA annealed to complementary DNA regions present in one <br><br> DNA strand, numerous single-stranded loops resulted. These loops, referred to as introns, represented DNA sequences not complementary to the mRNA. Although, by convention, introns are usually sequentially labeled 5' to 3', an opposite order had to be employed in this case as SpC3 does not contain the sequences coding for the 5' end of <span class="yellow">sheep</span> a2 mRNA. Infrequently, all introns <br><br> were clearly observed within the same hybrid molecule (Figure 1). The largest introns were towards the left arm of Charon 4A and were more clustered than the introns at the opposite end of the hybrid. Thus, much of the genetic <br><br> information coding for the a2 mRNA was contained in that half of the hybrid attached to the right arm. <br><br> A small non-hybridized tail was consistently found in the region where <br><br> the right arm of the phage was ligated to the <span class="yellow">sheep</span> genomic insert (Figure 1). This tail likely corresponded to the 3' poly A sequence of the a2 mRNA. At <br><br> the extreme 5' end of the insert, a variable length was often observed for the remaining unhybridized mRNA strand. This was most likely due to the size heterogeneity of the a2 mRNA used in these studies. <br><br> Size determination and comparative analysis of SpC3 introns. <br><br> Although localization of exon and intron sequences was possible with both the 70% formamide and urea-formamide spreading methods, the probability of <br><br> detecting unambiguous regions in a hybrid molecule was far greater with the <br><br> 2244 <br><br> Nucleic Acids Research <br><br> A                                B ',:  '14 .0. <br><br> 13 <br><br> 7 1 <br><br> -5 13 <br><br> .  .   . .   ,4                     x2%.   N1 <br><br> 9 <br><br> >   10          R       4 <br><br> Figure 1.    Electron microscopic visualization of all 17 introns and 18 exons <br><br> from hybrid molecules formed between recombinant clone SpC3 DNA and fetal <span class="yellow">sheep</span> <br><br> tendon a2 mRNA. (A,B) Shown are the double-stranded segment of the right arm CR) of Charon 4A; double-stranded segment of the left arm CL) of Charon 4A; the <br><br> displaced single-stranded DNA segment of the insert (5); single-stranded DNA loops representing introns sequentially labeled 1 through 17 in the 3' to 5' direction; and 18 regions of insert DNA sequences hybridized to ct2 mRNA <br><br> ---      - ).  (C,D) Similar to (A,B) but the single-stranded DNA segment (5) of the insert is mostly absent. The urea-formamide spreading method, as described in Materials and Methods, was employed. <br><br> 2245 <br><br> Nucleic Acids Research <br><br> urea-formamide method than with the 70% formamide procedure. For this reason, most measurements were made with the urea-formamide method (Tables 1,2). For <br><br> example, the lowest probability event, the detection of intron 17 and the surrounding small exons, "q" and "r", was only observed by way of the ureaformamide spreading procedure. <br><br> The smallest introns, 7 and 12, were found to contain approximately <br><br> the same number of bases. Introns 1 through 8, representing the DNA sequences which split the a2 coding sequence corresponding to the 3' end of the a2 mRNA, varied in length over approximately a 2.4-fold range. With the exception of <br><br> Table 1. Electron microscopic R loop analysis <br><br> present in SpC3 DNA1. <br><br> of the <span class="yellow">sheep</span> a2 introns <br><br> SpC3 was hybridized to <span class="yellow">sheep</span> a2 mRNA as described in Materials and Methods; intron lengths were determined by comparison with *X-174 DNA lengths. <br><br> 2 Introns are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 Intron 17 was visualized only by the urea-formamide method. <br><br> 2246 <br><br> 70% Formamide Method              Urea-Formamide Method <br><br> Intron2         Number of       Intron length       Number of        Intron length <br><br> introns analyzed       (bases)       introns analyzed      (bases) <br><br> 1              10             819 t 063              35            806 t 76 2              12             771 ? 89               46            757 t 78 3              11             458 ? 58               47            464 ? 66 4               6             627? 53               41             611 ? 73 5               5             554 ? 51              34             562 ? 76 6               6             675 ? 19               40            684 ? 76 7               5             386 ? 48               50            342 ? 54 8                7            771 ? 84               49            757 ? 82 9               8           1,181 ? 94               52          1,124 ?122 10               8           1,133 ?106               53          1,124 ?137 11              10             964 t?79              61             952 ?105 12               9             362 ? 77              62             342 ? 71 13               4             723 ?222               53            806 ?105 14               4           1,085 ?142               53          1,075 ?105 15               3             795 ? 60              47             806 ?100 16               2           1,229 ? 48              43           1,319 ?146 174              -                 -                  12          1,539 ?166 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Table 2.   Electron microscopic R loop analysis of the <span class="yellow">sheep</span> a2 exons <br><br> present in SpC3 DNA1. <br><br> 70% Formamide Method                Urea-Formamide Method <br><br> Exon2          Number of        Exon length       Number of       Exon length <br><br> exons analyzed     (base pairs)    exons analyzed    (base pairs) <br><br> a                 7             807 ? 883           26            883 ?132 b                10             252 ? 35            41            234 ? 49 c                11             151 ? 25            48            182 ? 42 d                 6             252 ? 60            44            234 ? 62 e                 5             126 ? 45            37            129 ? 47 f                 5             126 ? 38            35            104 ? 41 9                 5             176 ? 48            40            182 ? 54 h                 5             100 ? 43            47            104 ? 31 <br><br> 7             100 ? 35            50            104 ? 28 j                 7             126 ? 48            45            130 ? 47 k                 8              50 ? 48            51             52 ? 39 1                 9              75? 10             60             52 ? 44 m                 6             126 ? 43            51            182 ? 54 n                 4             100 ? 45            53             52 ? 42 o                 3             <504                50             78 ? 36 p                 2             126 ? 71            47            104 ? 36 q5                - _                               33            130?47 r5 r                                                 6             78?36 <br><br> SpC3 was hybridized to <span class="yellow">sheep</span> a2 mRNA as described in M4aterials and <br><br> Methods; exon lengths were determined by comparison with pBR322 DNA lengths. <br><br> 2 Exons are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 5 <br><br> No statistical evaluation was possible because of the size of the exon (see (Materials and Methods). <br><br> Exons q and r were only analyzed by the urea-formamide method. <br><br> intron 1, the nucleotide size of these introns was less than 800 bases. In contrast, introns 9 through 17 varied over a 4.5-fold range. Except for intron 12, all of the introns 9 through 17 contained 800 bases or more. <br><br> Summation of introns 1 through 16 for the 70% formamide and urea<br><br> formamide spreading methods yielded 12,533 and 12,531 bases, respectively. <br><br> However, detection of intron 17 by the urea-formamide method indicates that <br><br> the actual summed value for the intervening sequences of SpC3 is 14,070 bases. <br><br> 2247 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Size determination and the comparative analysis of SpC3 exons. <br><br> Eighteen exons, "a" through "r", were mapped (Table 2). The largest exon, "a", located on the extreme 3' end of the <span class="yellow">sheep</span> genomic insert, contained <br><br> approximately 800 bp. With the exception of exon "m", exons larger than 150 <br><br> base pairs were confined to the genetic region corresponding to the far 3' end of the a2 mRNA. Exon "a" and exon "b", the two exons situated on either side of intron 1, coded for around 1000 bases of a2 collagen information; these two <br><br> exons contained one-third of the mapped 3,014 DNA bases complementary to a2 mRNA Length distribution and organization of the introns and exons in the <span class="yellow">sheep</span> genomi c i nsert. <br><br> There was a sharp contrast in the size distribution of the exons and introns contained within SpC3 (Figure 2). While the intron sizes varied <br><br> considerably, the exons, in general, showed a very narrow distribution of <br><br> lengths, and thus tended to cluster at the lowest end of the length distri<br><br> A. <br><br> (I) <br><br> z 6 0 <br><br> z <br><br> ZL4 0 <br><br> 021               <br><br> z <br><br> 10 <br><br> B. <br><br> 8 <br><br> z 0 <br><br> 16 0 z <br><br> 2 <br><br> 20   400  600  800 1000 1200 1400 1600 <br><br> LENGTH IN NUCLEOTIDES <br><br> Figure 2. Length distribution of the intron and exon mean lengths measured from micrographs prepared by the urea-formamide method. (A) Introns; (b) Exons. See Tables 1 and 2 for details. <br><br> 2248 <br><br> Nucleic Acids Research bution graph. The exon in the 800-900 bp region was unique and was the only exon which exhibited length overlap with introns. <br><br> The maps of the a2 collagen gene obtained with both electron micro<br><br> scopic spreading methods were almost identical (Figure 3), except that the <br><br> region at the extreme 5' end of the insert could only be mapped with the ureaformamide method. From these measurements we concluded that the length of the a2 collagen genetic regi?on in SpC3 was 17,084 bases long with only 3,014 bases actually coding for a2 mRNA. Therefore, only 18% of the mapped 17 kb <span class="yellow">sheep</span> genetic insert actually contained a2 structural information. <br><br> DISCUSSION <br><br> The organizational structure of the a2 collagen gene is complex. A minimum of 17 introns interrupt the 3 kb of ca2 genetic information interspersed as 18 exons over 17 kb of <span class="yellow">sheep</span> DNA. With the exception of exon "a" located at the extreme 3' end of the inserted a2 gene, all other mapped exons are rather small, exhibiting a size distribution range of 50 to 250 bp. In <br><br> contrast, the intron sizes range from 340 to 1540 bases occupying a total <br><br> of 14 kb. Thus, the DNA region containing the 3 kb of a2 collagen genetic information is 5.6 fold longer than the corresponding co2 mRNA sequences. <br><br> Although the total number of introns present within the 60% 3' portion <br><br> of the a2 <span class="yellow">sheep</span> gene was determined to be 17, variation in the precise number <br><br> TRANSRIPTlO <br><br> A                          5w                r <br><br> 1 "2a4  133  112s  11.17  0. 3O 2  715  6.2  313  04  4.2B  3.51  2.2  12 <br><br> EXON0            p  o    n         5  Ik         h     f     d 4  .  b   <br><br> WPM300       16   16  14   13  12  11  10  9   6  7  6  5  4  3   2   1 <br><br> B <br><br> 110   16     4.2  13111  112n  1113104  3.6    gm74  2 6.  4 4 2  3.2  22  in <br><br> 1  a1m03  133  1.2 11   10  01 341  6. 1   117  6.2 4  12  417  3I5'm 2  <br><br> DM00  1    q5    p  0    n   m  I  k.   j    i   h  g  f     d  c   b     a 147306   17   16   15  14  13  12  11  10  9   a  7  6  5  4  3   2   1 <br><br> 175 17.0 16.6& 16'5.5 15.0 14.5 14. 13.5 13.0 12.5 12.0 11.5 11 10.5M 10.0 9.5 9.0 8. 5.0 71 7.0 6.5 6. 5.5 50 4.5 4.0 3.5 3.0 2.5 2.0 1.5 1.0 <br><br> KILOBASE <br><br> Fiue3      Organization of 60% of the 3' end of the <span class="yellow">sheep</span> a2 collagen gene. (A)}The exon ( ) and intron (CDJ) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules visualized by the electron microscopic 70% formamide spreading method.  (B) The exon Um) <br><br> and intron (Efl) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules evaluated by the electron microscopic <br><br> urea-formamide spreading method. Arrows represent the approximate location of intron-exon junctions. All values are presented as kilobase pairs. <br><br> 2249 <br><br> Nucleic Acids Research <br><br> of introns present in individual hybrid molecules was observed. The reason <br><br> for this may be due to size heterogeneity of the a2 mRNA used as a complementary probe. Although the a2 mRNA was extensively purified (2), some degradation was likely present. It is also possible that additional introns exist that are too small to be detected by electron microscopic R loop analysis. The most likely a2 genetic regions containing such introns are exons "g" and "m". Small <br><br> "protrusions" were sometimes observed within these exons that could represent small introns. Very small introns have been reported for tRNA genes (12,13), <br><br> and it will be necessary to sequence these regions of SpC3 to determine if such introns are present within the a2 <span class="yellow">sheep</span> genome. <br><br> The existence of mRNAs comprised of sequences complementary to noncontiguous regions in a DNA genome was first described for <span class="yellow">adenovirus</span> (14,15) and <br><br> simian virus (16,17) genes. Similar observations were soon reported for eukaryotic genes, including serum albumin (18), conalbumin (19), fibroin (20), s<br><br> globin (21,22), growth hormones (23,24), immunoglobulin (25), lysozyme (26), <br><br> ovalbumin (27-29), ovomucoid (30), rRNA (31-34), tRNA (12,13), and vitellogenin (35). The precise number or occurrence of introns within these genes does not seem to follow any clear pattern. However, the genes for albumin, conalbumin, <br><br> lysozyme, ovalbumin, and ovomucoid exhibit a structural organization comparable to the a2 collagen gene. All of these genes are composed of numerous small <br><br> exons, usually less than 250 base pairs, and introns which show a large size variation within an individual gene. Within all of these genomes, the DNA <br><br> regions required for coding structural formation are 4 to 7 times longer than the corresponding mRNA coding sequences. <br><br> As with the a2 collagen gene, a large unique exon at the extreme 3' end <br><br> is also present in the ovalbumin gene (36,37). In the a2 gene, this exon was 800 to 900 nucleotides in length, while in the ovalbumin gene the exon contained 1,030 base pairs. Interestingly, this ovalbumin exon contained 634-650 base pairs ot DNA sequence to the 3' side of the terminator <br><br> triplet, suggesting the ovalbumin mRNA has a large untranslated region at the 3' end. <br><br> The consistent appearance of a small non-hybridized tail, approximately 100 nucleotides in length, in the region where the right arm of the phage was ligated to the <span class="yellow">sheep</span> genomic insert of SpC3 very likely corresponded to the <br><br> 3' poly A sequence of the a2 mRNA. This would suggest that the a2 mRNA coding sequence terminates very near or at this point in the genome. Previously published biochemical evidence (2) obtained from restriction mapping and Southern blot analysis using [32P]-labeled <span class="red">chick</span> a2 cDNA probe supports <br><br> 2250 <br><br> Nucleic Acids Research <br><br> this observation. <br><br> It is unclear what role introns play in genetic expression. However, <br><br> it is known that these sequences are tranncribed as part of larger precursor mRNAs and the introns are then excised and the fragmented mRNA chains <br><br> covalently rejoined (38). Some evidence has recently accumulated demonstrating the existence of precursor a2 mRNAs (39,40). However, if the introns in the a2 gene are transcribed in their entirety, as has been <br><br> demonstrated for the introns in ovalbumin and globin (41,42), much larger precursor a2 mRNAs may exist than those reported to date. For if the re<br><br> mainder of the a2 collagen gene has a structural organization comparable to the 60% 3' portion that has been analyzed, it is probable that the a2 <br><br> collagen gene in its entirety is dispersed throughout a 30 kb DNA segment. <br><br> ACKNOWLEDGMENTS <br><br> We thank Dr. John Dahlberg and Frances Loebenstein, Laboratory of Cellular and Molecular Biology, National Cancer Institute, for the generous use of a <br><br> Siemens Elmiskop 101 electron microscope; Dr. Ursula Heine and Benjamin Elliott, Laboratory of Viral Carcinogenesis, National Cancer Institute, for the use of a Hewlett-Packard calculator equipped with a digitizer and a Denton vacuum <br><br> evaporator, and Drs. Victor Ferrans and Oichi Kawanami, Laboratory of Pathology, <br><br> National Heart, Lung, and Blood Institute, for expert assistance with photography. We are also grateful to Margery Sullivan, Laboratory of Molecular Genetics, <br><br> National Institute of <span class="yellow">Child</span> Health and <span class="yellow">Human</span> Development, for helpful discussions and to Dr. Helga Boedtker, Department of Chemistry, Harvard University, for making available the recombinant plasmid pCg45. <br><br> References <br><br> 1.   Prockop, D. J., Kivirikko, k. I., Tuderman, L., and Guzman, N. A. <br><br> (1979) New Engl. J. Med. 301, 13-23 and 77-85. <br><br> 2.   Boyd, C. D., Tolstoshev, P., Schafer, M. P., Trapnell, B. C., Coon, <br><br> H. C., Kretschmer, P. J., Nienhuis, A. W., and Crystal, R. G. (1980) J. Biol. Chem., in press. <br><br> 3.   Blin, N., and Stafford, D. W. (1976) Nucl. Acids Res. 3, 2303-2308. 4.   Kretschmer, P. J., Kaufman, R. E., Coon, H. C., Chen, M. JY., Geist <br><br> C. E., and Nienhuis, A. W. (1980) J. Biol. Chem., in press. <br><br> 5.   Maniatis, T., Hardison, R. C., Lacey, E., Laver, H., O'Connell, C., <br><br> Quon, D., Sim, G. K., and Efstratiadis, A. (1975) Cell 15, 687-701. <br><br> 6.   Blattner, F. R., Blechl, A. E., Denniston-Thompson, K., Faber, H. E., <br><br> Richards, J. E., Slightom, H. L., Tucker, P. W., and Smithies, 0. (1978) Science 202, 1279-1284. <br><br> 7.   Meyer, J., Neuwald, P. D., Lai, S., Maizel, J. V., Jr., and Westphal, <br><br> H. (1977) J. Virol. 21, 1010-1014. <br><br> 2251 <br><br> Nucleic Acids Research <br><br> 8.   Westphal, H., and Lai, S. P. (1977) J. Mol. Biol. 116, 525-548. 9.   Schafer, M. P., Rohrmann, G., Heine, U., and Beaudreau, G. S. <br><br> (1979) Virol. 95, 176-184. <br><br> 10. Davis, R. W., Simon, M. M., and Davison, N. (1971) in Methods in <br><br> Enzymology (Grossman, L. and Moldave, K., eds) Vol. 21, pp. 413428, Academic Press, New York. <br><br> 11.  Federal Register (1978) 43, 60108-60131. <br><br> 12. Goodman, H. M., Olson, M. V., Hall, B. D. (1977) Pro. Natl. Acad. <br><br> Sci. U.S.A. 74, 5453-5457. <br><br> 13. Valenzuela, P., Venegas, A., Weinberg, F., Bishop, R., Rutter, <br><br> W. J. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 190-194. <br><br> 14. Berget, S. M., Moore, C., and Sharp, P. A. (1977) Proc. Natl. Acad. <br><br> Sci. U.S.A. 74, 3171-3175. <br><br> 15. Chow, L. T., Gelinas, R. E., Broker, T., and Roberts, R. J. (1977) <br><br> Cell 12, 1-8. <br><br> 16. Berk, A. J., and Sharp, P. A. (1978) Proc. Natl. Acad. Sci. U.S.A. <br><br> 75, 1274-1278. <br><br> 17. Bratosin, S., Horowitz, M., Laub, 0., and Aloni, Y. (1978) <br><br> Cell 13, 783-790. <br><br> 18.  Sargent, T. D., Wu, J. R., Sala-Trepat, J. M., Wallace, R. B., <br><br> Reyes, A. A., and Bonner, J. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 3256-3260. <br><br> 19.  Perrin, F., Cochet, M., Gerlinger, P., Cami, B., LePennec, J. P., <br><br> and Chambon, P. (1979) Nucl. Acids Res. 6 2731-2748. <br><br> 20. Tsujimoto, Y., and Suzuki, Y. (1979) Cell 16 425-436. <br><br> 21. Jeffreys, A. J., and Flavell, R. A. (1977) Cell 12, 1097-1108. <br><br> 22. Tilghman, S. M., Tiemeier, D. C., Seidman, J. C., Matija Peterlin, B., <br><br> Sullivan, M., Maizel, J. V., and Leder, P. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 725-729. <br><br> 23. Fiddes, J. C., Seeburg, P. H., DeNoto, F. M., Hallewell, R. A., <br><br> Baxter, J. D., and Goodman, H. M. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 4294-4298. <br><br> 24.  Soreq, H., Harpold, M., Evans, R., Darnell, Jr., J. E., and <br><br> Bancroft, F. C. (1979) Nucl. Acids Res. 6, 2471-2482. <br><br> 25.  Tonegawa, S., Maxam, A. M., Tizard, R., Bernard, 0., and Gilbert, W. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1485-1489. <br><br> 26. Nguyen-Huu, M. C., Stratmann, M., Groner, B., Wurtz, T., Land, M., <br><br> Glesecke, K., Sippel, A. F., and Schutz, G. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 76-80. <br><br> 27. Breathnach, R., Mandel, J. L., and Chambon, P. (1977) Nature 270, <br><br> 314-319. <br><br> 28. Dugaiczyk, A., Woo, S. L. C., Lai, E. C., Mace, M. L., Jr., <br><br> McReynolds, L., and O'Malley, B. W. (1978) Nature 274, 328-333. 29. Weinstock, R., Sweet, R., Weiss, M., Cedar, M., and Axel, R. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1299-1303. <br><br> 30.  Catterall, J. F., Stein, J. P., Lai, E. C., Woo, S. L. C., <br><br> Dugaiczyk, Mace, M. L., Means, A. R., and O'Malley, B. W. (1979) Nature 278, 323-327. <br><br> 31. White, R. L., and Hogness, D. S. (1977) Cell 10, 177-192. <br><br> 32.  Glover, D. M., and Hogness, D. S. (1977) Cell 16, 167-176. 33. Wellauer, P. K., and David, I. B. (1977) Cell 10, 193-212. <br><br> 34. Pellegrini, M., Manning, J., and Davidson, N. (1977) Cell 10, <br><br> 213-214. <br><br> 35. Wahli, W., David, I. B., Wyler, T., Jaggi, R. B., Weber, R., and <br><br> Ryffel, G. V. (1979) Cell 16, 535-549. <br><br> 36. McReynolds, L., O'Malley, B. W., Nisbet, A. D., Fothergill, J. E., <br><br> 2252 <br><br> Nucleic Acids Research <br><br> Givol, D., Fields, S., Robertson, M., Brownlee, G. G. (1978) Nature 273, 723-728. <br><br> 37. O'Hare, K., Breathnach, R., Benoist, C., Chambon, P. (1979) <br><br> Nucl. Acids Res. 7, 321-334. <br><br> 38. Abelson, J. (1979) Ann. Rev. Biochem. 48, 1035-1069. <br><br> 39. Adams, S. L., Alwine, J. C., de Chrombrugghe, B., and Pastan, I. <br><br> (1979) J. Biol. Chem. 254, 4935-4938, <br><br> 40. Rave, N., Crkvenjakow, R., and Boedtker, H. (1979) Nucl. Acids Res. <br><br> 6, 3559-3567. <br><br> 41. Roop, D. R., Nordstrom, J. L., Tsai, S. Y., Tsai, M. J., and <br><br> O'Malley, B. W. (1978) Cell 15, 671-685. <br><br> 42. Tilghman, S. M., Curtis, P. J., Tiemeier, D. C., Leder, P., and <br><br> Weissmann, C. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1309-1313. 43.  Palmiter, R. D., Davidson, J. M., Gagnon, H., Rowe, D. W., and <br><br> Bornstein, P. (1979) J. Biol. Chem. 254, 1433-1436. <br><br> 44. Graves, P. N., Olsen, B. R., Feitzig, P. P., Monson, J. M. and <br><br> Prockop, D. J. (1979) Fed. Proc. 38, 620. <br><br> 2253 </body></html>