
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>donkey</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA140510</h3>Enp1, a <span class="yellow">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="yellow">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="yellow">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="yellow">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="yellow">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="yellow">human</span> Enp1, expressed in <span class="yellow">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="yellow">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="yellow">human</span> homolog of <span class="yellow">yeast</span> Enp1 was PCR amplified from a <span class="yellow">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="yellow">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="yellow">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="blue">donkey</span>-anti-<span class="yellow">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="yellow">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="yellow">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="yellow">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="yellow">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="yellow">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="yellow">yeast</span> Enp1 (19). In contrast, we identified a <span class="yellow">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="yellow">human</span> bystin (19) showed that they were encoded by the same gene on <span class="yellow">human</span> chromosome 6. The reported sequence for <span class="yellow">human</span> bystin is a fragment of the 437 amino acid <span class="yellow">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="yellow">human</span> bystin, as confirmed by available <span class="yellow">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="yellow">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="yellow">Drosophila melanogaster</span> and <span class="yellow">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="yellow">human</span> Enp1 homolog was cloned and expressed in <span class="yellow">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="yellow">human</span> Enp1 homolog did not complement a <span class="yellow">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="yellow">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="yellow">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="yellow">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="yellow">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="yellow">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="yellow">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="yellow">human</span> homolog of Enp1 did not complement a <span class="yellow">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="yellow">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA2652712</h3>Neurogenin2 Directs Granule Neuroblast Production and Amplification while NeuroD1 Specifies Neuronal Fate during Hippocampal Neurogenesis
Abstract
The specification and differentiation of dentate gyrus granule neurons in the hippocampus require temporally and spatially coordinated actions of both intrinsic and extrinsic molecules. The basic helix-loop-helix transcription factor Neurogenin2 (Ngn2) and NeuroD1 are key regulators in these processes. Based on existing classification, we analyzed the molecular events occurring during hippocampal neurogenesis, primarily focusing on juvenile animals. We found that Ngn2 is transiently expressed by late type-2a amplifying progenitors. The Ngn2 progenies mature into hippocampal granule neurons. Interestingly, the loss of Ngn2 at early stages of development leads to a robust reduction in neurogenesis, but does not disturb granule neuron maturation per se. We found that the role of Ngn2 is to maintain progenitors in an undifferentiated state, allowing them to amplify prior to their maturation into granule neurons upon NeuroD1 induction. When we overexpressed Ngn2 and NeuroD1 in vivo, we found NeuroD1 to exhibit a more pronounced neuron-inductive effect, leading to granule neuron commitment, than that displayed by Ngn2. Finally, we observed that all markers expressed during the transcriptional control of hippocampal neurogenesis in rodents are also present in the <span class="yellow">human</span> hippocampus. Taken together, we demonstrate a critical role of for Ngn2 and NeuroD1 in controlling neuronal commitment and hippocampal granule neuroblast formation, both during embryonic development and in post-natal hippocampal granule neurogenesis.<br><br>Introduction
Neurons are born not only during development of the central nervous system, but neurogenesis also continues into adulthood. In both <span class="yellow">rodent</span> and <span class="yellow">human</span> adult brain, neurogenesis is active in two distinct zones of the forebrain: the subventricular zone (SVZ) of the lateral ventricle and the subgranular zone (SGZ) of the hippocampal dentate gyrus (DG) [1]–[5]. The molecular mechanism underlying neurogenesis in the DG is not fully understood. Clearly, a cascade of transcriptional events controls the specification of neuronal identity in the DG [3], [6]–[8], but details of the expression pattern and function of each transcription factor remain elusive.
The paired-box homeodomain transcription factor Pax6 and the bHLH transcription factors Ngn2 and NeuroD1 are important when cells acquire a pan-neuronal character and a specific neuronal subtype [9], [10]. In the developing neocortex, Pax6 is expressed in dividing radial glial cells at the ventricular surface [9]. In the adult hippocampus, Pax6 is expressed in astrocytes in the SGZ and are considered to be the true stem cells [9], [11]. Loss- and gain-of-function studies indicate that Pax6 is involved in regulating the proliferation of neocortical and hippocampal progenitors [9], [12]–[16]. During neocorticogenesis, high concentrations of Pax6 induce the expression of the bHLH transcription factor Neurogenin2 (Ngn2) [17]. In turn, Ngn2 causes cell cycle exit [18], an event that takes place when NeuroD1 starts to be expressed [19]. In the developing hippocampus, the absence of Ngn2 leads to a reduced number of granule neurons [20]. NeuroD1, on the other hand, is essential for the differentiation and survival of hippocampal granule neurons [10]. Interestingly, in the absence of Ngn2, NeuroD1 can still be activated and neuronal differentiation can still take place [20]. This has led to the idea that the primary role of Ngn2 is not to direct neuronal differentiation.
Only a small number of studies have addressed the role of Ngn2 and NeuroD1 in hippocampal neurogenesis [10], [20], [21]. Therefore, we now re-examine their roles in hippocampal neurogenesis in detail, using gain- and loss-of-function experiments. We first establish a hierarchy of transcriptional events that occur during neurogenesis in the DG and then define the place that Ngn2, NeuroD1 and other transcription factors have in this cascade. We find that a lack of Ngn2 expression result in a markedly smaller hippocampus and an almost complete absence of the DG. We show that Ngn2 is required for granule neuroblasts production/amplification. Gain-of-function of NeuroD1 during development of the DG results in an efficient generation of granule neuroblasts, an effect that we do not observe when we overexpress Ngn2. Finally, we demonstrate that the same transcription factors and cellular markers seen in <span class="yellow">mouse</span> and <span class="yellow">rat</span> tissue are also present in the <span class="yellow">human</span> hippocampus.<br><br>Results
Expression of transcription factors and cellular markers define different phases of postnatal hippocampal granule neuron maturation
We first analyzed in detail the chronology of expression of different transcription factors and cell-specific markers during DG granule neuron formation and correlated our findings to the previously established classification of adult hippocampal neurogenesis [3]. The current classification describes hippocampal stem cells and progenies into three categories (type-1, -2 and -3 cells) depending on the markers they express as they mature. In the adult <span class="yellow">mouse</span> DG, only a very small proportion of cells undergo mitosis at any one given time [22]. Consequently, in adult animals it is difficult to analyze which transcription factors are involved in the transition phase from one cell category to the next. Therefore, we studied 2 week-old rodents which have higher numbers of maturing granule neurons in the DG. We performed triple immunohistochemistry for all markers of interest and compared results from 2 week- and 2 month-old <span class="yellow">mice</span> (Figure 1 and figure S1).
We first confirmed that stem cells in the SGZ exhibit characteristics of radial glia [3], [11], [16]. They extend radial processes from the SGZ to the apex of the subgranular layer (SGL) and the basal molecular layer. Moreover, they express the intermediate filament protein nestin, glial fibrillary acidic protein (GFAP) and glial glutamate transporter (GLAST) [16] (Figure 1A and B and figure S1A and S1B). The GFAP- and GLAST+ cells are defined as type-1 cells [3]. They also express the paired-homeodomain transcription factor Pax6 (Figure 1B). These radial glia-like stem cells divide relatively infrequently, and are believed to undergo symmetric division (giving rise to two identical stem cells) under some conditions. They can also divide asymmetrically and give rise to a new stem cell and one neuronal progenitor, which usually is defined as a type-2 cell (Figure 2B) [3].
Type-2 cells are divided into two different populations: type-2a expressing nestin and type-2b co-expressing nestin and doublecortin (Dcx) [23]. Based on a previous report [24], we examined the expression pattern of the T-domain transcription factors Tbr1 and Tbr2. We found that Tbr2 is expressed in type-2a cells, in agreement with a recent report [25]. In the SGZ of 2 week-old <span class="yellow">mice</span>, we found an average of 50–70 Tbr2+ cells per 30 µm-thick section throughout the rostro-caudal axis of the dorsal hippocampus. This number was reduced by 35% in 2 month-old <span class="yellow">mice</span> (Figure 1C with figure S1C and data not shown). As opposed to Tbr1 (Figure 1D and figure S1D), Tbr2+ cells did not co-express Dcx or PSA-NCAM (Figure 1C and figure S1C). We observed that Tbr1-immuoreactivity gradually decreased as neurons matured and started to express NeuN (Figure 1H and figure S1H). Thus, Tbr2 labels type-2 cells, while Tbr1 is expressed by immature granule neurons.
We then examined the molecular phenotype of type-3 cells, a cell type that transiently expresses Calretinin [26]. We hypothesized that hippocampal type-3 cells might also express the bHLH transcription factor NeuroD1, known to partially overlap with Tbr2 and Tbr1 in different brain regions [24]. We found that NeuroD1 expression in the hippocampus starts in Tbr2+ cells and extends to post-mitotic Tbr1+ cells, which also express the hippocampal granule identity transcription factor Prox1 (Figure 1G and 1I and figure S1G and S1I). While Tbr1 expression ceases during granule maturation, that of NeuroD1 is weakly maintained when NeuN expression starts (Figure 1G and 1H and figure S1G and S1H). We found that calretinin expression decreased during granule neuron maturation, before NeuroD1 was reduced (Figure 1F). Thus NeuroD1 is expressed in type-2b and type-3 cells, as well as immature granule neurons. Finally, we found that NeuroD2 starts to be expressed just after NeuroD1 and, unlike NeuroD1, continues to be highly expressed in mature neurons (Figure 1J and data not shown).<br><br>Mash1 and Ngn2 define the early versus late type-2a stage in 2 weeks old <span class="yellow">mice</span>
Having established the hierarchy of transcription factors and cellular markers appearing during hippocampal neurogenesis, we set out to clarify the position of the two bHLH transcription factors Ngn2 and Mash1, in this process. We found that more than 90% of Ngn2-immunoreactive cells are Pax6 positive (Figure 2A), and almost all co-express Tbr2 (Figure 2B). Thus, Ngn2 appears to be expressed by type-2a cells. As our antibodies against Ngn2 and NeuroD1 were made in the same species, we could not determine if there was an overlap between these two proteins. However, by comparing their overlapping expression pattern with that of Tbr2, we propose that some Ngn2+ cells co-express NeuroD1, and that they appear at the onset of NeuroD1 expression and downregulation of Ngn2. We base this assumption on the facts that more than 50% of the Tbr2+ cells are Ngn2+ and more than 50% of Tbr2+ cells also express NeuroD1. The expression of Ngn2 in adult hippocampus was hard to detect using immunohistochemistry due to the low expression of the protein (data not shown). However, it is present, as previously described using reporter <span class="yellow">mice</span> and as revealed by in situ hybridization [21].
We next examined the expression of Mash1 and compared it with that of Ngn2, Pax6 and Tbr2. We found some Mash1+ cells (10–20 cells per section) in the dorsal hippocampus. Almost all of them co-expressed Pax6 (Figure 2C). Only few cells co-expressed Tbr2 (Figure 2D) or Ngn2 (Figure 2E), suggesting that Mash1 is downregulated when these two proteins are expressed. None of the Mash1-expressing cells were positive for NeuroD1 (Figure 2F). Mash1+ cells were still undergoing mitosis (Figure 2C) and consequently over 90% of them co-labeled with the cell-cycle marker Ki67 (Figure S1J).
Taken together, we show that Mash1 and Ngn2 are expressed in early and late stages of maturation of type2a progenitors, respectively, and that both transcription factors are co-expressed briefly when cells transit from early to late phase type2a cells.<br><br>Ngn2 progenies become hippocampal granule neurons
Previous work has shown that in Mash1 null mutant <span class="yellow">mice</span> hippocampal neurogenesis is not reduced and the hippocampus is not malformed [20]. Therefore we focused our initial analysis on the role of Ngn2 in hippocampal neurogenesis. We characterized the fate of Ngn2-expressing cells using Ngn2 Knock-in green fluorescent protein (GFP) <span class="yellow">mice</span> (Ngn2+/GFP) [27]. In the SGZ of 2 week-old hippocampi, we found bright GFP-expressing cells that were positive for Tbr2 (Figure 2H), the immature neuronal marker PSA-NCAM (Figure 2I) and NeuroD1 (Figure 2J). Furthermore, Prox1+, NeuroD2+ and calretinin+ cells were also weakly positive for GFP. We made similar observations in 2 month-old <span class="yellow">mice</span> (data not shown) confirming earlier published results [21]). We did not observe Mash1+/GFP+ and NeuN+/GFP+ cells (Figure 2G). These data from Ngn2+/GFP <span class="yellow">mice</span> are entirely consistent with our earlier observations using immunocytochemistry to label Ngn2-expressing cells and with GFP undergoing slow degradation after the expression of Ngn2 has ceased. Importantly, none of the GFP-expressing cells, including those only weakly fluorescent, residing in the SGL expressed the astrocytic marker GFAP or the oligodendrocytic markers CNPase (data not shown). Taken together, the data show that Ngn2 progenies become neurons and never generate astrocytes or oligodendrocytes.<br><br>Marked reduction of hippocampal granule neurons in absence of Ngn2
To determine the role of Ngn2 during the initiation of hippocampal granule neurogenesis, we analyzed the hippocampus of <span class="yellow">mice</span> lacking Ngn2 (Ngn2GFP/GFP). In contrast to the <span class="yellow">mice</span> use by Galichet and coworkers, our <span class="yellow">mice</span> have a lifespan of only 2 weeks after birth. Therefore, we analyzed 2 week-old and younger <span class="yellow">mice</span>. Consistent with previous observations [20], the hippocampus in Ngn2GFP/GFP mutants is clearly malformed (Figure 3A). In two week-old Ngn2GFP/GFP <span class="yellow">mice</span>, the ventral blade of the DG is completely absent. Already on postnatal day 1–2, the ventral blade of the DG is malformed along its whole rostro-caudal axis (Figure 3B).
As neither migration defects nor cell death cause the reduced number of hippocampal neurons in Ngn2 null <span class="yellow">mice</span> [20], we asked if Ngn2 is necessary for production of hippocampal granule neuroblasts. We first injected 2 day-old <span class="yellow">mouse</span> pups with BrdU 2 hours prior to sacrifice, allowing us to evaluate cell proliferation and ongoing neurogenesis. We also injected BrdU into female <span class="yellow">mice</span> on their final day of pregnancy and examined the brains of their pups 48 hours later (corresponding to P1–P2 old pups). This allowed us to assess the number of neurons derived from the last day of intrauterine development. Regardless of whether the <span class="yellow">mice</span> were WT or hetero/homozygous mutant pups, we found cells that had incorporated BrdU in the hippocampal subventricular zone (hSVZ), fimbria and the DG (Figure 4A and B; [28]), suggesting that progenitors originating in the hSVZ divide while migrating towards the DG. Most of them were organized in chains typical of migrating cells (Figure 4A and B and figure S2). In Ngn2GFP/GFP mutant <span class="yellow">mice</span> injected with BrdU 2 hours prior to sacrifice, we observed a decrease in the number of BrdU+ cells compared to heterozygous littermates. Thus, the numbers of newborn cells were reduced in the hSVZ/fimbria (133.3±7.5 for Ngn2+/GFP vs 69.7±5.0 for Ngn2GFP/GFP) and DG (132.0±9.1 for Ngn2+/GFP vs 42.6±2.2 for Ngn2GFP/GFP)(Figure 4C and D). We also examined the number of newborn cells differentiating into neurons, and identified them by the co-expression of BrdU and NeuroD1. As expected, they were relatively few in numbers because the short delay (2 hours) between BrdU administration and sacrifice in this first experimental paradigm. In Ngn2GFP/GFP null mutants, we found the number of cells differentiating into neurons to be reduced by 60% and 71% in the hSVZ/fimbria (8.6±1.4 for Ngn2+/GFP vs 3.4±0.5 for Ngn2GFP/GFP) and DG (9.7±1.1 for Ngn2+/GFP vs 2.8±0.6 for Ngn2GFP/GFP), respectively, (Figure 4E and F). As a result, the number of NeuroD1+ cells in the DG was reduced by 80% (580.3±27.7 for Ngn2+/GFP vs 116.2±7.3 for Ngn2GFP/GFP; Figure 4G).
We obtained similar results in the second paradigm, i.e. in <span class="yellow">mice</span> that we sacrificed 48 hours after BrdU administration and in which a larger number of the newborn cells had time to mature into neurons. In this case, the number of BrdU+/NeuroD1+ cells in the DG was reduced by 73% of Ngn2GFP/GFP mutant animals (91.5±24.1 for Ngn2+/GFP vs 26.4±3.3 for Ngn2GFP/GFP) (Figure 4J). We found no differences in the proportion of BrdU+ cells that expressed NeuroD1 in Ngn2+/GFP and Ngn2GFP/GFP <span class="yellow">mice</span> (14.2±1.0% and 17.4±1.8% respectively; Figure 4K). This shows that the few cells that manage to proliferate in the Ngn2 null <span class="yellow">mice</span> have the same ability to differentiate into neurons as those in <span class="yellow">mice</span> with one Ngn2 allele.
Interestingly, the absence of Ngn2 did not alter the identity of neurons in the DG granule layer in 2 week-old Ngn2 null <span class="yellow">mice</span>. The maturing granule progenitors sequentially expressed Pax6, Tbr2 (Figure S3A and S3B), NeuroD1 and Calretinin (Figure 4L), PSA-NCAM (Figure 4N), Tbr1 (data not shown) and Prox1 (Figure 4M). We confirmed that the same transcriptional cascade is active in the 20% (compared to <span class="yellow">mice</span> with one Ngn2 allele) residual granule neurons that are formed two day-old in Ngn2GFP/GFP <span class="yellow">mice</span> (Figure S3D–G and data not shown).
We next investigated if the cells that failed to develop into neurons in Ngn2GFP/GFP mutant <span class="yellow">mice</span>, became glial cells. We found that cells expressing GFP never co-labeled with the astrocyte marker GFAP (Figure 4N) or the oligodendrocyte marker CNPase (data not shown).
Altogether, our results show that Ngn2 plays an important role during the production/amplification of hippocampal granule neuroblasts, but not the acquisition of the granule neuron identity (Figure S7).<br><br>Ngn2 controls the amplification of granule neuron progenitors
We next monitored the mitogenic activity of Ngn2+ cells in the DG of 2 week-old WT <span class="yellow">mice</span>. We observed that over 50% of Ngn2+ cells in the DG of WT <span class="yellow">mice</span> co-expressed the mitosis marker Ki67 (Figure 5A). Inspired by this finding, we injected BrdU into 2 week-old WT and Ngn2GFP/GFP <span class="yellow">mice</span> and compared cell proliferation in the DG. We observed 92% reduction in number of proliferating cells in the SGZ of Ngn2 null mutant <span class="yellow">mice</span> (6.7±0.5) compared to WT (83.6±1.2) and heterozygotic (data not shown) littermates (Figure 5B–C). This data confirm the importance of Ngn2 during the amplification of granule progenitor.s.
To further explore whether Ngn2+ cells become post-mitotic or still proliferate after Ngn2 is downregulated, we performed immunohistochemistry on sections through the DG of Ngn2+/GFP <span class="yellow">mice</span>. We stained them with the mitosis marker Phospho-histone 3 (PH3) and NeuroD1, which is downstream of Ngn2 in the transcriptional cascade controlling neurogenesis. Thus, in the same sections we could identify whether cells that had initiated Ngn2 expression (GFP labeled), continued to divide (PH3+) or committed to neuronal differentiation (NeuroD1+). We observed some GFP+ cells that co-expressed PH3. They all exhibited morphological characteristics of one of the five mitotic phases: prophase, metaphase, anaphase, telophase and cytokinesis (Figure 5D and data not shown). Unexpectedly, cells that colabeled for GFP, PH3 and NeuroD1 (Figure 5E and figure S4A and S4B) were rare. This indicates that Ngn2+ cells undergo division/amplification and that they mature into post-mitotic neurons upon NeuroD1 expression. In <span class="yellow">mice</span> lacking Ngn2, we found the cells expressing GFP localized to the malformed ventral blade of the DG (Figure 5F and G). All of these cells were Ki67+ and they only very rarely expressed PH3 (Figure 5F and G, and figure S4C) in the absence of Ngn2 most of the cells are arrested in the cell cycle prior to entering the M phase and their mitosis is impaired. These data suggest that the mechanism of action of Ngn2 is conserved from development of the DG to postnatal hippocampal neurogenesis.
Our findings suggest that Ngn2 regulates amplification and cell cycle exit of DG granule progenitors. To examine this hypothesis, we compared the effects of Ngn2 and NeuroD1 on mitotic activity in embryonic cortico-hippocampal neurosphere-derived progenitors, 5 days upon transduction, in vitro. In cultures transduced with Ngn2 retrovirus, we found that 27.2% (±4.1%) of the PH3+ cells had been transduced (Figure I1–J). These cells were immunopositive for MAP2 (Figure 5I1 and I2) and therefore represented dividing neuroblasts. By contrast, in NeuroD1 transduced cultures, only 7.3% of PH3+ cells were GFP+ (Figure 5H1, H2 and J). All of the NeuroD1-transduced cells became MAP2+ (Figure 5H). To confirm this data we pulse-labeled transduced cultures with chlorodeoxyuridine (CldU) for 48 hours, five days after differentiation. In contrast to NeuroD1-transduced cultures, we observed dividing cells transduced with Ngn2 retrovirus that were positive for PH3 and that had incorporated CldU (Figure 5K–L2). As the number of cells PH3+/CldU+/GFP+ we observed was low, we did not quantify this finding.
Collectively, we have shown that Ngn2 is required for granule neuroblast production and amplification and that in the absence of Ngn2 the progenitors arrest in the cell cycle. NeuroD1, on the other hand, induces cell cycle exit and promotes rapid neuronal maturation.<br><br>NeuroD1 directs neuronal differentiation and maturation
Based on our previous observations we proposed that Ngn2 primarily controls amplification of granule neuroblasts and NeuroD1 directs neuronal differentiation. To test this hypothesis, we overexpressed Ngn2 or NeuroD1 in E14.5 cortico-hippocampal neurospheres and compared their effects after 5 days of differentiation of the progenitors. All cortico-hippocampal progenitors expressed Pax6 prior to differentiation (Figure S5). After 5 days, both factors suppressed Pax6 and Sox2 (Figure S5, figure 6C) and induced expression of Tbr1, Map2, NeuroD1 and PSA-NCAM (Figure 6D and E).
Interestingly, all Ngn2-overexpressing cells co-expressed NeuroD1. Based on these findings, we next explored whether Ngn2 overexpression induces NeuroD1 expression, and if NeuroD1 in turn directs neuronal differentiation. We compared the effects of both transcription factors in progenitors that either do or do not normally express them. Thus, we expressed Ngn2 in E14.5 neural progenitors isolated from three different brain regions: cortex/hippocampus, lateral ganglionic eminence (LGE) and ventral mesencephalon (VM). Ngn2 expression in the developing forebrain is normally limited to the neocortex, and it does not appear in LGE tissue [29]. In the developing VM, both Ngn2 and NeuroD1 are expressed, but not when neural progenitors from this region are cultured in vitro [30], [31]. In our experiments, cortico-hippocampal progenitors could differentiate into neurons, although they did not normally express NeuroD1 (Figure 6E). When we overexpressed Ngn2 in VM and LGE progenitors, the cells started to express both NeuroD1 and PSA-NCAM (Figure 6F and G). We then overexpressed NeuroD1 in the same types of cultured progenitors and found that the cells became immunoreactive for PSA-NCAM (Figure 6G and data not shown). Thus, NeuroD1 is sufficient to direct neuronal differentiation in cortico-hippocampal-, LGE- and VM-derived progenitors.
Our results from in vitro cultures and the analysis of the DG of Ngn2 mutant animals collectively show the neuron-inducing effect of NeuroD1 in hippocampal granule cell progenitors [32].<br><br>NeuroD1 directs exclusive neuronal differentiation of hippocampal granule neuron progenitors
As the next step, we tested the effects of NeuroD1 in vivo and compared them with those of Ngn2. We injected retroviruses carrying the gene for either Ngn2 [33] or NeuroD1, and the reporter gene eGFP into the ventricles E15.5 <span class="yellow">rat</span> embryos, in utero (Figure 7A). Three weeks later, when the <span class="yellow">rats</span> were about two weeks old, we examined their hippocampi and analyzed the eGFP+ cells. The <span class="yellow">rats</span> injected with control vector exhibited eGFP+ cells within the hippocampus that were either star-shaped, progenitor/glial-like cells (14.8±3.4%) or neuron-like cells with long neurites (85.3±3.4%) (Figure 7B). In <span class="yellow">rats</span> that we had injected with the vector encoding Ngn2, the eGFP-labeled, the transduced hippocampal cells were composed of 31.9% (±5.7%) progenitor/glia-like cells and 69.4% (±6.1%) neuron-like cells (Figure 7B). The glia-like subpopulation was immunopositive for the astrocytic marker GFAP (Figure 7C–D). In contrast, virtually all of the transduced cells in <span class="yellow">rats</span> injected with the NeuroD1 vector became neuron-like cells (99.9%±0.1; Figure 7B). None of these cells stained for GFAP (Figure 7E and F). They were positioned in the external layers of both the ventral and dorsal blades of the DG (Figure 7B3, E and I and figure S6A).
When we examined neuronal maturation of Ngn2- and NeuroD1-transduced cells, we observed that only a few of those transduced with the NeuroD1 vector expressed the early mature neuronal marker Tbr1 (3.1±1.4%; Figure 7I and J). They were located within the basal layers, near the SGL, of the ventral and dorsal blades of the DG. In contrast, a greater proportion of the cells transduced with Ngn2 still expressed Tbr1 (19.6%±3.3%; Figure 7H and J), indicating that they were less mature than the vast majority of the NeuroD1-transduced cells. Immunohistochemistry for NeuN confirmed these data (Figure 7M and O, and figure S6A). Indeed, when we overexpressed Ngn2 the proportion of hippocampal progenitors that became NeuN+ was no greater than in <span class="yellow">rats</span> transduced with the control virus (Figure 7F).
As a whole, our in utero injection experiments confirm that NeuroD1 has a stronger neuron-inducing effect than Ngn2 when overexpressed in hippocampal progenitors.<br><br>Involvement of Mash1, Ngn2, Tbr and NeuroD proteins during <span class="yellow">human</span> hippocampal neurogenesis
Finally, we examined whether the transcription factors and cellular markers that are expressed in rodents are also expressed in <span class="yellow">human</span> hippocampus. We performed immunohistochemistry on aged <span class="yellow">human</span> hippocampal DG and found GFAP-, Sox2-, Pax6-, Nestin-, Prox1- and NeuN-immunopositive cells. This indicates that radial glia-like stem cells, neural progenitors and mature granule neurons are present in the aged <span class="yellow">human</span> hippocampus (Figure 8A–E). While Prox1 is found mainly in granule neurons, Sox2 and Pax6 are exclusively localized to cells in the SGZ (Figure 8D and E).
As for aged rodents, we did not observe the presence of Mash1, Ngn2, Tbr2, Tbr1 and NeuroD1 proteins in aged <span class="yellow">human</span> hippocampus, using immunohistochemistry. However, we could identify the presence of the transcripts, using RT-PCR (Figure 8F). In addition, we detected mRNA for Prox1, Calbindin1 and Calbindin2/Calretinin (Figure 8F). Interestingly, we also observed Sonic Hedgehog and Wnt-3A transcripts, indicating that they are present in the <span class="yellow">human</span> DG and supporting previous claims that they are important for the regulation of adult hippocampal neurogenesis (Figure 8F; [34], [35]). Finally, we also found mRNA for GLAST, Pax6 and GFAP which suggests that radial glia-like stem cells are present in the adult <span class="yellow">human</span> hippocampus, and may play a role in adult <span class="yellow">human</span> hippocampal neurogenesis (Figure 8F; [8]).
Overall, our data show that different proteins known to play key roles in <span class="yellow">rodent</span> hippocampal granule neurogenesis are present in the adult <span class="yellow">human</span> hippocampus.<br><br>
Discussion
In this study we determine the functions of Ngn2 and NeuroD1 during hippocampal neurogenesis. First, we map the hierarchy of molecular markers of neurogenesis in the DG. Second, we describe that Ngn2 is necessary for granule progenitor production/amplification. Third, we demonstrate that NeuroD1 directs neuronal differentiation of granule progenitors. Finally, we show that different cellular markers expressed during hippocampal neurogenesis in rodents are present in <span class="yellow">human</span>.
Sequential expression of different transcription factors and cellular markers during hippocampal neurogenesis
We clarified the pattern of expression of various markers expressed during postnatal and adult hippocampal granule neurogenesis in detail. We found that the transcription factor Pax6 is initially expressed by both radial glia stem cells (type-1) and early amplifying progenitors (type-2a). The next transcription factor to appear in chronology is Mash1. Mash1 expression characterizes the early stage of hippocampal progenitor amplification (early type-2a). The role of Mash1 during hippocampal granule neurogenesis is still unclear. Mash1 null mutant <span class="yellow">mice</span> do not display any clear malformation of the DG [20]. However, overexpression of Mash1 alone in the DG leads to the generation of oligodendrocytes [36]. Based on these observations, we hypothesize that Mash1-expressing cells are not yet committed towards a granule cell fate. They may still have the potential to generate both neurons and oligodendrocytes, as is the case in the SVZ [37].
In agreement with different developing brain regions, we found that Ngn2 starts to be expressed in Mash1-positive transiently amplifying progenitors, i.e. “late type-2a cells” according to the classification we propose. We observed that Pax6 expression persisted longer than that of Mash1, in Ngn2-expressing cells (late type-2a cells), in juvenile but not adult DG (Fig. 1 and figure S1). Later on, Ngn2 is downregulated, whilst Tbr2 expression persists (type-2b cells). The transition from amplifying progenitor to neuroblast is defined by the expression of NeuroD1. Thus, Ngn2 is expressed at the beginning of the transiently amplifying progenitor phase while NeuroD1 marks the end of that period. This applies both to the juvenile and adult <span class="yellow">rat</span> brain [21], [25]. After NeuroD1 is turned on the progenitors leave the cell cycle, gradually mature, express PSA-NCAM, NeuroD2, Calretinin, Prox1, Tbr1 and, finally, NeuN. NeuroD1 expression persists at low levels in mature neurons (Figure 9; [10]). We found the sequential expression of these markers conserved in juvenile (P2 and 2 weeks) and adult <span class="yellow">rodent</span> brains. Importantly, we observed that the same transcription factors and neuronal markers are also present in the adult <span class="yellow">human</span> hippocampus and arranged spatially in a manner reminiscent with what we saw in rodents. It is possible, however, that some of these markers are expressed for longer or shorter periods at postnatal and adult stages of hippocampal progenitor maturation. Moreover, one can ask to which extent the number of divisions occurring during neuroblasts maturation is conserved at postnatal and adult stages of neurogenesis.<br><br>A new function for Ngn2 in maintenance of a progenitor state for granule neuron production/amplification during embryonic and postnatal hippocampal neurogenesis
In agreement with earlier work, we confirm that Ngn2 is indispensable for hippocampal development and plays a vital role in postnatal hippocampal granule neurogenesis. Ngn2 null mutant animals display a reduced size of the cornu ammonis and a malformed DG (Figure 3 and 4; [20]). We show that the number of newborn neurons (incorporated BrdU) is decreased in absence of Ngn2, but hippocampal granule neuron subtype specification is not affected, post-nataly. Likewise, neurons from other brain regions, e.g. ventral midbrain dopamine neurons, can be generated in the absence of Ngn2 [30], [31]. The reduced numbers of mature neurons in the hippocampus and ventral midbrain of Ngn2 null mutant animals does not appear to be due to cell death or migration defects [20], [31]. Instead it may be due to a defect in the generation and/or amplification of neuronal progenitors. Indeed, in the hippocampus of postnatal animals, we observed that cells lacking Ngn2 arrest in the cell cycle, maintain Pax6 expression and cease to proliferate.
When overexpressed in neural progenitors in vitro, Ngn2 lead to up-regulation of NeuroD1 and caused neuronal differentiation (Figure 6), which was not always the case when we overexpressed the same factor in vivo (Figure 7L, N and O). We also saw that Ngn2 does not always efficiently induce cell cycle exit. Typically some mitotically active eGFP+ cells were present in the DG of <span class="yellow">rats</span> injected with the Ngn2 retrovirus (Figure 7P), and we observed that cultured Ngn2-transduced cells are still capable of dividing. Thus, Ngn2 does not always promote cell cycle exit and neuronal commitment, but depending on the state of the cells, Ngn2 may instead promote alternate cellular fates [38]–[40]. Indeed, an earlier study has shown that Ngn2-overexpressing progenitors generate oligodendrocytes when grafted to the adult spinal cord [40]. We have also seen that Ngn2-overexpressing neural progenitors from the embryonic midbrain form astrocytes when grafted to the striatum (unpublished observations). Probably, Ngn2 cannot influence already committed cells, but rather would control neuroblasts production. One could speculate that Ngn2 oscillates during granule neuron formation, as recently demonstrated during neocorticogenesis [41], [42].<br><br>NeuroD1, but not Ngn2, is obligatory for granule neuron progenitor differentiation
Hippocampal granule progenitors can mature and express Tbr1 and Prox1, both during the development and postnatally, even in the absence of Ngn2. Because neuronal differentiation still occurs in Ngn2 null <span class="yellow">mice</span>, it is clear that compensatory, Ngn2-independent mechanisms induce NeuroD1 expression. One candidate is Ngn1, which is expressed during neocorticogenesis, the specification of olfactory sensory neurons and during embryonic <span class="yellow">rat</span> hippocampal development [43]–[47]. Both Ngn1 and Ngn2 regulate NeuroD1 [19], [48]. The introduction of two mutant forms of Ngn1, a deletion of the basic region of Ngn1 and a substitution of two amino acids in the C-terminal basic region, prevents NeuroD1 expression and neuronal differentiation [48]. The double null Ngn1/Ngn2 mutant displays a more severe phenotype than single gene (Ngn1 or Ngn2) mutant <span class="yellow">mice</span>. For example, the total brain size of the double mutants is much smaller [10], [32]. If the role of Ngn1 during hippocampal neurogenesis is to activate NeuroD1 and that of Ngn2 is to control granule neuroblasts production/amplification, the DG of double Ngn1/Ngn2 knockout <span class="yellow">mice</span> should resemble that of NeuroD1 null mutant <span class="yellow">mice</span>.
We demonstrated that NeuroD1 directs neuronal differentiation both in vitro and in vivo. In progenitors isolated from different embryonic brain regions and cultured in vitro, we found that overexpression of NeuroD1 induced neuronal differentiation. The neurons generated expressed PSA-NCAM, Dcx and MAP2. After in utero retroviral vector-mediated gene delivery, virtually all cells transduced with NeuroD1 became neurons. In contrast, progenitors transduced with Ngn2 or control retroviruses adopted a neuron-like morphology in only 70–85% of cases. Under in vitro cell culture conditions, when we overexpressed Ngn2 in progenitors derived from the LGE we observed robust expression of NeuroD1 and neuronal differentiation. These in vitro results differ from those we obtained when we transduced the embryonic brain with a viral vector expressing Ngn2. In this latter case, Ngn2 did not induce neurons and the transduced cells did not express NeuroD1. Therefore, Ngn2 appears to direct non-neuronal cell type specification in vivo [38]–[40]. Taken together, we have confirmed that NeuroD1 plays a key role in neuronal differentiation in the hippocampus, both during development and in the adult brain.<br><br>Concluding remark
We present a detailed classification of different stages in hippocampal neurogenesis. Our detailed molecular mapping of hippocampal neurogenesis allows for a more accurate analysis of how new factors stimulate neurogenesis at different steps in the development of granule neurons. Thereby we hope to facilitate the development of new agents, which stimulate endogenous progenitors in the treatment of diseases.<br><br>
Materials and Methods
Animal tissue preparation
The creation of the Ngn2 transgenic <span class="yellow">mice</span> was reported elsewhere (ref guillemot). Heterozygote male and female <span class="yellow">mice</span> were crossed to obtain WT, heterozygote (Ngn2+/GFP) and null mutant (Ngn2GFP/GFP) animals. Tails of the Ngn2 offspring were used to obtain DNA for determination of the genotype using a polymerase chain reaction (PCR) assay as previously reported [31]. As null mutant Ngn2GFP/GFP do not survive longer than 2.5–3 weeks after birth, Ngn2GFP/GFP and their littermates were sacrificed at the postnatal ages of two days (P2) or two weeks for this study. Neurogenesis was assessed in two weeks or two months old WT <span class="yellow">mice</span>. Sprague Dawley pregnant <span class="yellow">rats</span> were ordered from B&K Universal Ltd, Sollentuna, Sweden (hppt://www.bku.com). All animals were housed in groups with ad libitum access to food and water at a 12-h light/dark cycle. All experimental procedures conducted in this study had been approved by the Ethical Committee at Lund University.
For immunohistochemical analysis, <span class="yellow">mice</span> (from two weeks old and adult stage) and juvenile <span class="yellow">rats</span> were sacrificed by transcardial perfusion with saline for 5–10 minutes, followed by 4% paraformaldehyde (PFA) for 10 minutes. Brains were kept in PFA overnight at 4°C and subsequently cryopreserved in a 20–30% sucrose/0.1 M phosphate buffer solution until sectioning on a microtome apparatus (30 µm thickness sections, Microm Zeiss). Seven series of coronal sections were cut throughout the brain. Free-floating sections were preserved in antifreeze solution until immunohistochemistry was performed. Heads of postanatal two days old <span class="yellow">mice</span> were decapitated and soaked in PFA 4% for 24 hours, at 4°C and transferred into sucrose solution until sectioning on cryostat apparatus (16 µm thickness; Leica CM3000). Sections were mounted on Superfrost glass slides and stored at −80°C until immunohistochemistry was performed.<br><br>Cloning, subcloning, virus production and titer measurement
The Moloney leukemia-derived retroviral vectors used in this study, pCMMP-IRES2eGFP-WPRE and pCMMP-Ngn2-IRES2eGFP-WPRE were previously described [40], [49]. To generate the construct pCMMP-NeuroD1-IRES2eGFP-WPRE, <span class="yellow">mouse</span> NeuroD1 cDNA was amplified from a pCS2+mtNeuroD1 plasmid (kindly provided by Professor Jackie Lee, Denver university, Boulder, USA) by PCR to introduce the restriction sites PmeI in 5′ and XhoI in 3′. Amplification of cDNA was performed as previously described [49]. The construction was verified by enzymatic restriction and by DNA sequencing using BigDye 3.1 (ABI). All infectious particles were produced using the producer cell line 293VSV-G and as previously described [49]. The titer of each retrovirus was measured by flow-cytometry based on eGFP expression, four days following infection of HT1080 cells and ranged from 0.5×109−2.1×109 TU/ml (All details on how to produce infectious particles can be provided upon request).<br><br>Neurosphere generation, transduction and differentiation
Pregnant female <span class="yellow">Sprague-Dawley rats</span> (B & K Universal, Sollentuna, Sweden) were terminally anesthetized by an overdose of sodium pentobarbital (i.p., 60 ng/ml). Embryos at stage embryonic day E14.5 (Plug day as day 0) were collected and cortical-hippocampal neurospheres were generated following dissection of the dorso-posterior part of the cortical tissue, and generated as previously described [49]. In this study, second passage (P2) neurospheres were used to study the effect of the overexpression of Ngn2 and NeuroD1 on neuronal differentiation. Each well, containing an equal starting population of 200,000 cells/ml, corresponding to 15–25 neurospheres, was transduced independently with each retrovirus at a multiplicity of infection (MOI) of 1, in proliferation medium supplemented by protamine-sulfate (4 mg/ml, Sigma). To induce differentiation, the medium was replaced with normal basic differentiation medium two days post-transduction and subsequently changed every other day until fixation.<br><br>Immunocytochemistry, immunohistochemistry and microscopy
The antibodies used in this study are: <span class="yellow">rabbit</span> anti-GFAP (1∶1000; DAKO), <span class="yellow">mouse</span> anti-Nestin (1∶100; BD PharMingen), <span class="yellow">guinea pig</span> anti-Glast (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Prox1 (1∶1000; Covance), <span class="yellow">goat</span> anti-Ngn2 (1∶20; Santa Cruz), <span class="yellow">goat</span> anti-NeuroD1 (1∶200; Santa Cruz), <span class="yellow">rabbit</span> anti-Pax6 (1∶150; Covance), <span class="yellow">rabbit</span> anti-Trb2 (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Trb1: (1∶1000; Chemicon), <span class="yellow">goat</span> anti-Dcx: (1∶500; Santa Cruz), <span class="yellow">mouse</span> anti-PSA-NCAM: (1∶500; Chemicon), <span class="yellow">mouse</span> anti-NeuN (1∶300; Chemicon), <span class="yellow">rabbit</span> anti-Calretinin (1∶500; Swant); <span class="yellow">mouse</span> anti-Sox2 (1∶100; R&D systems), <span class="yellow">mouse</span> anti-Mash1 (1∶100; BD PharMingen), <span class="yellow">rabbit</span> anti-NeuroD2 (1∶300; ABCAM), <span class="yellow">mouse</span> anti-MAP2 (1∶500; Sigma), <span class="yellow">rabbit</span> anti-Ki67 (1∶150; NovaCastra) and <span class="yellow">rabbit</span> anti-phospho-Histone H3. The secondary antibodies (1∶200) Cy2, FITC, Cy3 and Cy5 were from Jackson IR laboratories, Alexa-fluor 488, 568, 595 and 647 from Invitrogen-Molecular Probes. DAPI (1∶1000) was purchased from Sigma.
For immunocytochemistry, cultures were fixed in 4% paraformaldehyde at day five, rinsed with PBS three times prior to pre-incubation with a blocking solution (10% <span class="blue">donkey</span> serum, 0.25% TritonX100 in PBS) for 1 hour. The remainder of the procedure was performed as previously described [49]. Specimen analyses were performed using a Leica confocal microscope (Leica software, equipped with a GreNe and a HeNe laser, using the following lines of excitation: 488 nm, 594 nm and 647 nm). Samples were analyzed using 20×, 40× and 63× objectives, sometimes zoomed. Figures were composed in CANVAS-X software.<br><br>Bromo-deoxyuridine (BrdU) and Cloro-deoxyuridine (CldU) pulse labeling and immunohisto- and immunocyto-chemistry
To assess cell proliferation and ongoing neurogenesis in vivo, animals were injected with BrdU (100 mg/kg, Sigma), two hours prior to sacrifice (for both P2 and two weeks old Ngn2 <span class="yellow">mice</span>). To assess neurogenesis, BrdU (100 mg/kg) was injected 48 hours prior to sacrifice (for P2 animals, BrdU was injected in pregnant dams half day prior to give birth). Immunohistochemistry was carried on as described above using a <span class="yellow">rat</span> anti-CldU/BrdU primary antibody (1∶200, monoclonal, Immunologicalsdirect, Oxfordshire, UK), with an additional denaturation in 1 M HCl for 30 minutes prior to pre-incubation with serum.
To assess the neuronal-inducing activity of Ngn2 and NeuroD1, transduced cultures were incubated with CldU (20 µM, Sigma) for 2 days, after a period of differentiation of five days. For immunocytochemistry, cultures were fixed with 4% paraformaldehyde at day 7, rinsed with PBS three times, treated with 1 M HCl at 65°C for 5–10 min, pre-incubated and then incubated with a <span class="yellow">rat</span> anti-CldU/BrdU antibody and other primary antibodies. The remainder of the procedure was performed according to the protocol for immunohistochemistry already mentioned.<br><br>In utero surgery
Timed pregnant female <span class="yellow">Sprague Dawley rats</span> with embryos at gestational age E15.5 were anesthetized with halothane. The mother was placed in the lower level of a two-level wooden stage. The abdomen was shaved with an electric razor and then cleaned with 70% alcohol. A 2–3 cm midline laparotomy was performed. Each uterine horn was carefully taken out individually and the number of embryos recorded. One horn was then placed back inside the mother, whilst the other horn was prepared for injection. The embryos were kept moist with constant application of warm saline to prevent dehydration. Approximately 2 µl of viral suspension (1×10∧9TU/ml) was injected into the lateral ventricle of each embryo, except for the embryo closest to the vagina. After the injections, the uterine horns were placed back into the abdomen. The abdominal wall and the overlying skin were then sutured. Care was taken not to damage abdominal muscles so that normal delivery of the pups was possible at term. The entire surgery generally took about 45 minutes to one hour. Each mother was allowed to recover in her cage before being returned to the animal stable. Shredded paper was added to each cage to encourage nesting and special care was taken not to stress the mothers. Following normal delivery, the pups were allowed to develop to adulthood up to two weeks.<br><br>Quantification
For the in vivo experiment, manual cell counting was performed on 18 and 30 µm thick brain sections for two days and two weeks old animals, respectively. The brain of P2 animals was cut into 10 series of coronal sections; the brain of two-weeks old animals was cut into 7–8 series of coronal sections. For each staining, one-two series from three to five different individuals were analyzed per genotype using a confocal microscope with 1, 2 or 3 lines of excitation, in sequential scanning in order to avoid false positives. When two series were analyzed, only one series was counted. The mean number of cells per hippocampus expressing the markers of interest, was calculated for each brain, based on the analysis of 6–8 consecutive dorsal hippocampal sections. The final mean number of cells per section was calculated by adding the mean number from each individual. We expressed the data as the mean number of positive cells±standard error of the mean (SEM). In this study, more than 25 Ngn2 null, >45 Ngn2 heterozygotes, and >60 WT individuals (including <span class="yellow">rats</span> injected with retroviruses) were analyzed, in total. Statistical comparison was performed using one-factor analysis of variance (ANOVA) with transcription factor, cellular marker, genotype or time as variables, followed by post-hoc analysis when significant differences were observed, using Statview 5.0 software (SAS institute Inc.). For cell culture experiments, three independent experiments were performed, each in duplicate. The counting was based on seven randomly chosen different fields of view. For each diagram, the level of significance (p-value) is represented as follows: P<0.05 = *; P<0.001 = **; P<0.0001 = ***. All data are expressed as±standard error of the mean (SEM).<br><br><span class="yellow">Human</span> sample and RT-PCR
The brain from a 64 year-old male was provided by the Harvard Brain Tissue Resource Center. The individual whose brain tissue was being analyzed gave written consent for storage and use of his tissue for research. Five millimeters thick fresh <span class="yellow">human</span> hippocampal tissue sample was snap frozen. The frozen tissue block was cut into 20 series of coronal sections, using a cryostat. Every 10 sections, one section was placed in an Eppendorf tube, on dry ice. Pooled sections were used for RT-PCR. Total RNA was prepared using Trizol and RNAeasy, supplemented with RNAGuard as previously described [49]. The RNA was digested with <span class="yellow">shrimp</span> DNAse before cDNA synthesis, which was performed using a mix of oligo-dT and random hexamer primers and SuperscriptII reverse transcriptase. Advantage2 polymerase mix (Clontech/BRL) was used for PCR, with the following cycling conditions: 10 cycles of 94° C for 30 seconds, 68°C for 2 minutes, 30 cycles of 94°C for 30 seconds, 60°C for 1 minute, 68°C for 1 minute and 30 seconds, one cycle of 68° C for 2 minutes, soak at 16°C. Primers marked * were designed by PrimerBank. hRPS18 (sense) 5′-GCCTTTGCCATCACTGCCATT and hRPS18 (antisense) 5′-GCCAGTGGTCTTGGTGTGCT, hPAX6 (sense) 5′-GCCCTGGAGAAAGAGTTTGAGAGAACCCATT hPAX6 (antisense) 5′-GGGGAAATGAGTCCTGTTGAAGTGGTGC, hTBR1 (sense) 5′-GCGGACACCAATGTGCAAGGAAATCG and hTBR1 (antisense) 5′-CGAGGGGGTCAGGCGGTCCATGTCACAGC, hTBR2 (sense) 5′-GACCTGTGGCAAAGCCGACAATAACATGC and hTBR2 (antisense) 5′-GGGGGTGTCTCTATCCAAGAAGAGCCAAT, hPDHX ( = PROX1) (sense) 5′-GGGACACTACGGTTCCGTTTAAGTCCAGC and hPDHX (antisense) 5′-CTCTCCATCCCAGCTTACATTAACATCTGGCATTTGT, hNGN2 (sense) * 5′-CATCAAGAAGACCCGTAGACTGA and hNGN2 (antisense)* 5′-CAACACTGCCTCGGAGAAG, hMASH1 (sense) 5′-CCTGGATCCGCATGGAAAGCTCTGCCAAGATGGAG and hMASH1 (antisense) 5′-CCTGGATCCCCCCTCAGAACCAGTTGGTGAAGTCGA, hNEUROD1 (sense) 5′-GCTCAGGACCTACTAACAACAAAGGAAATCGAAACATG and hNEUROD1 (antisense) 5′-CAAAGCGTCTGAACGAAGGAGACCAGGT, hGLAST (sense) 5′-CATCAGGGAAGATGGGAATGCGAGCTGTAGTCTATTAT and hGLAST (antisense) 5′-CCACGGGGGCATACCACATTATTACTGCTACC, hNEST (sense) 5′-CAGGAGCGGCTGCGGGCTACTGAAAAGTTCC and hNEST (antisense) 5′-CAGGGCTGAGGGGTGGTGCCAAGGAGG, hGFAP (sense) 5′-CCACGAGGAGGAGGTTCGGGAACTCCAGGAGC and hGFAP (antisense) 5-GGAATGGTGATCCGGTTCTCCTCGCCCTCTAGC, hCALB2 ( = Calretinin) (sense) 5′-GGCTCTGGCATGATGTCAAAGAGTGACAACTTT and hCALB2 (antisense) 5′-GGGCATCCAGCTCATGCTCGTCAATGTAGCC, hCALB1 (sense) 5′-GCGAAAGAAGGCTGGATTGGAGTTATCACC and hCALB1 (antisense) 5′-CCCTCCATCCGACAAAGCCATTATGTTCTTCTTGTATG, hSOX2 (sense) 5′-GGAGAACCCCAAGATGCACAACTCGGAGAT and hSOX2 (antisense) 5′-GAGGAAGAGGTAACCACAGGGGGGCTGGAGC, hWNT3A (sense) 5′-CCGAGGGCATCAAGATTGGCATCCAGGAGTGC and hWNT3A (antisense) 5′-TCGGGTTGCGACCACCAGCATGTCTTCACCTC, hSHH* (sense) 5′-ACTCCGAGCGATTTAAGGAACT and hSHH* (antisense) 5′-CAGACGTGGTGATGTCCACTG.<br><br>
Supporting Information<br><br>
</body></html>