
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Drosophila melanogaster</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="blue">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1131934</h3>HP1 modulates the transcription of cell-cycle regulators in <span class="blue">Drosophila melanogaster</span>
Abstract
Heterochromatin protein 1 (HP1) was originally described as a non-histone chromosomal protein and is required for transcriptional gene silencing and the formation of heterochromatin. Although it is localized primarily at pericentric heterochromatin, a scattered distribution over a large number of euchromatic loci is also evident. Here, we provide evidence that <span class="yellow">Drosophila</span> HP1 is essential for the maintenance of active transcription of euchromatic genes functionally involved in cell-cycle progression, including those required for DNA replication and mitosis. Depletion of HP1 in proliferating embryonic cells caused aberrant progression of the cell cycle at S phase and G2/M phase, linked to aberrant chromosome segregation, cytokinesis, and an increase in apoptosis. The chromosomal distribution of Aurora B, and the level of phosphorylation of histone H3 serine 10 were also altered in the absence of HP1. Using chromatin immunoprecipitation analysis, we further demonstrate that the promoters of a number of cell-cycle regulator genes are bound to HP1, supporting a direct role for HP1 in their active transcription. Overall, our data suggest that HP1 is essential for the maintenance of cell-cycle progression and the transcription of cell-cycle regulatory genes. The results also support the view that HP1 is a positive regulator of transcription in euchromatin.<br><br>INTRODUCTION
Chromatin in higher eukaryotes is subdivided into different functional compartments termed heterochromatin and euchromatin (1). Heterochromatin differs from euchromatin in its DNA composition, replication timing, condensation throughout the cell cycle, and its ability to silence euchromatic genes placed adjacent to or within its territory, often described as position-effect-variegation (PEV) (2).
Heterochromatin protein 1 (HP1) was the first protein identified in <span class="blue">Drosophila melanogaster</span> as a heterochromatin-associated protein (3); the corresponding gene has been cloned from a number of organisms and is highly conserved from <span class="yellow">yeast</span> to <span class="yellow">human</span> (4). Polytene chromosome staining showed that, in <span class="yellow">Drosophila</span>, HP1 is distributed mainly in pericentric heterochromatin, telomeric heterochromatin, the banded small fourth chromosome (5–8), as well as ∼200 individual loci scattered throughout the euchromatic chromosomal arms (5). The gene encoding HP1 in <span class="yellow">D.melanogaster</span>, Su(var)2-5, was isolated as a suppressor of PEV (9–11). The protein contains a highly conserved motif, the chromo (chromatin organization modifier) domain, similar to Polycomb (Pc), a repressor of homeotic genes (12). The association between HP1 and pericentric heterochromatin is believed to occur via the chromo domain of HP1 and the N-terminal tail of histone H3 methylated at lysine 9 (13,14), generated by histone methyltransferase-Su(var)3-9, a partner of HP1 in pericentric heterochromatin (15). The C-terminal chromo ‘shadow’ domain of HP1 interacts with other silencing complexes to suppress local transcriptional activity (15–18). However, studies of HP1 chromosomal distribution also showed that HP1 does not always co-localize with lysine 9 methylated histone H3 or Su(var)3-9, especially in euchromatic regions (19–21); in some cases, HP1 is found directly bound to DNA (22,23). All these features argue for distinct roles for HP1 in chromatin and in epigenetic gene regulation.
HP1 is believed to be an essential structural protein protecting the integrity of chromosomes during cell division (8,24). Swi6, the homolog of HP1 in <span class="yellow">fission yeast</span>, is dispensable for survival, but its deletion results in lagging chromosomes during anaphase, and a high rate of chromosome loss (25,26). Mutations of HP1 in <span class="yellow">D.melanogaster</span> result in late larval lethality, chromosome breakages/loss, telomere fusion and a high frequency of cells with abnormal anaphase (8,27). Null alleles of the HP1 functional partner in <span class="yellow">mice</span> (SUVAR39) also showed various chromosomal defects (28), supporting a conserved role for heterochromatin proteins in the regulation of chromosome dynamics during cell-cycle progression. However, the mechanism(s) involved remains to be understood.
In this study, we utilized <span class="yellow">Drosophila</span> embryonic Kc cells and an RNA interference (RNAi)-based approach to demonstrate that HP1 plays an important role at S phase and G2/M phases during the cell cycle. We further show that nearly one-third of known/predicted cell-cycle regulators require HP1 to maintain their active transcription. These genes include MCMs, Orc4, CDC45L, INCENP, Aurora B, CAF1, Bub1, Bub3 and a few other cell-cycle regulators. ChIP analysis suggests that HP1 plays a direct role in their transcription. Therefore, the results of this study provide an alternative explanation for the specific role of HP1 in the regulation of chromatin dynamics and in cell-cycle progression.<br><br>MATERIALS AND METHODS
RNAi in Kc cells
<span class="yellow">Drosophila</span> Kc cells were routinely cultured at 25°C in Schneider <span class="yellow">Drosophila</span> medium (GIBCO) supplemented with 10% fetal <span class="yellow">calf</span> serum, 160 μg/ml penicillin, 250 μg/ml streptomycin, and 4 mM l-glutamine. Double-stranded RNA (dsRNA) of HP1 was generated by incubation of single-stranded RNA in annealing buffer (100 mM potassium acetate, 30 mM HEPES-KOH, pH 7.4, 2 mM magnesium acetate) for 3 min at 95°C and then placed in a beaker with water at 75°C and allowed to cool slowly to room temperature. The detailed procedure of RNAi was carried out according to the established protocols (). Briefly, Kc cells were seeded in a six-well dish using serum-free medium at 1 × 106 cells/ml. HP1 dsRNA (5 μg/ml) was added to the cultured Kc cells. After 60 min at room temperature, 2 ml of medium containing 10% serum was added to each well and the plates transferred to 25°C for up to 8 days. Western blotting and RT–PCR were carried out using the extract/total RNA isolated from control and dsRNA-treated cells on days 2, 6 and 8.<br><br>Cell-cycle and apoptosis analysis
The procedure for flow cytometric analysis of Kc cells followed that in the manual provided with the BrdU flow kit (BD PharMingen). The cells were fed with BrdU for 4 h, then scraped and collected. Fluorescence was measured using a FACSCalibur (Becton Dickinson). Data collection and analysis were performed using CellQuest software.<br><br>Electrophoresis and immunoblotting
Cell extracts (15 μg) were fractionated by 10% SDS–PAGE, then transferred to Hybond-P PVDF membranes (Amersham) and probed with primary antibodies (CIA9), and secondary antibodies (anti-<span class="yellow">rabbit</span> or anti-<span class="yellow">mouse</span> <span class="yellow">horseradish</span> peroxidase-conjugated IgG), obtained from Jackson Immunoresearch Laboratories. Enhanced chemiluminescence reagents (Amersham Pharmacia Biotech) were used for signal detection.
For the analysis of H3 ser10 phosphorylation, we used whole-cell extracts from 700 000 Kc cells (control and RNAi at day 8). Western blotting was performed using polyclonal antibodies against ser10-phosphorylated histone H3 at a dilution of 1:1000 (Upstate). Kc control cells arrested in mitosis by incubation in 25 µM colchicine (Sigma) for 24 h were also analyzed for comparison.<br><br>Immunofluorescence
Kc cells were seeded onto polylysine slides, fixed with 4% formaldehyde for 15 min and permeabilized with 0.5% Triton X-100 for 5 min. The incubation with primary antibodies was carried out in blocking solution for 1 h.
For staining of mitotic cells, the cells were permeabilized using PBST (PBS containing 0.3% Triton X-100) and stained with polyclonal antibody against <span class="yellow">Drosophila</span> Aurora B at 1:200 dilution and monoclonal <span class="yellow">mouse</span> at anti-β-tubulin 1:300 dilution (Chemicon International) as primary antibodies. Secondary antibodies were anti-<span class="yellow">rabbit</span> coupled with Alexa 488 (1:500) and anti-<span class="yellow">mouse</span> coupled to Alexa 546 (1:500) (Molecular Probes, Eugene, Oregon). Images were acquired using a confocal LSM510 META microscope (Zeiss). Stacks of images were analyzed using the IMARIS 4.0 program (Media cybernetics, Carlsbad, CA).<br><br>Antibodies
Affinity-purified polyclonal antibodies of HP1 (<span class="yellow">rabbit</span> #192 and #187, 5 μg) and 5 μg of polyclonal anti-HA antibodies (Sigma) were used in each ChIP reaction. The specificity of the HP1 polyclonal antibodies was determined using various approaches, including western blotting assay, immunofluorescence staining and immunoprecipitation to pull down HP1 (data not shown). The monoclonal antibody HP1–CIA9 (5) was used at a dilution of 1:20 in immunoblotting assays.<br><br>Microarray analysis and RT–PCR
Total RNA was isolated from control and HP1-depleted Kc cells at day 8 using an RNeasy kit (Qiagen). RNA labeling and microarray data analysis followed the standard protocol from Affymetrix. We used ANOVA (P < 0.001) to assess the expression confidence for each gene.
For RT–PCR analysis, poly(A)+ mRNA was purified with the Oligotex Direct mRNA kit (Qiagen) according to the manufacturer's instructions. The purified poly(A)+ RNA was reverse transcribed using the Thermoscript kit (Invitrogen). The cDNA was then used for PCR amplification for 35 cycles with gene-specific primers. PCR products were scanned after electrophoretic separation with a Typhoon Scanner, quantified using ImageQuant software (Amersham Biosciences) and normalized for amplification of the Actin5c transcript. The sequence of primers used for RT–PCR and ChIP analysis are provided in the Supplementary Material.<br><br>ChIP
ChIP was performed according to Orlando et al. (29) and the protocol provided by Upstate () with some modifications. In brief, 1–2 × 108 Kc cells were prepared and fixed in 1% formaldehyde. Nuclei were isolated according to a standard procedure in Current Protocols (), then resuspended in 1.7 ml of lysis buffer (50 mM Tris, pH 8.0, 10 mM EDTA, 1% SDS and protease inhibitors) and sonicated using a Branson sonifier 250. Chromatin fractions in the size range 0.2–0.8 kb were used to perform immunoprecipitation experiments. We used 5 μg affinity-purified polyclonal antibodies (#192 and #182 for HP1; HA antibody for control) and 1 ml of <span class="yellow">salmon</span> sperm DNA/protein-A-agarose (Upstate) pre-cleared chromatin lysate in each reaction. The mixture was then rotated at 4°C overnight and the recovered beads were washed twice with 1 ml of Low salt buffer (Upstate), once with High salt buffer (Upstate), once with LiCl buffer (Upstate) and twice with TE at 4°C for 8 min. ChIP DNA was extracted according to the standard procedures (29).<br><br>
RESULTS
Depletion of HP1 in <span class="yellow">Drosophila</span> Kc cells
Various chromosomal defects in the cell cycle have been observed in embryos or larval tissues of <span class="yellow">Drosophila</span> HP1 mutants (8,27). However, the presence of maternally loaded HP1 in embryos and the lethality of HP1 mutants at late larval stages have so far precluded a systematic study of the role of HP1 in cell-cycle regulation. Therefore, we used <span class="yellow">Drosophila</span> Kc cells, a cell line derived from <span class="yellow">Drosophila</span> embryos, as a model system to address this problem. HP1 transcripts were depleted using an RNAi-based approach (see Materials and Methods). The reduction in HP1 expression was measured both by RT–PCR and by western blotting analysis (Figure 1A). A significant reduction in the HP1 expression was already evident after 2 days treatment with HP1 dsRNA. Cells at day 8 showed a reduction in HP1 of ∼90% (Figure 1A) and were therefore used in all subsequent experiments.<br><br>Cell-cycle progression at S and G2/M phase is altered in the absence of HP1
The impact of HP1 loss on the cell cycle of Kc cells was determined using cell-cycle profile analysis of HP1-depleted and control cells. The percentage of cells in S phase was determined by BrdU incorporation, and total DNA content by 7-amino-actinomycin (7-AAD). The results showed that the depletion of HP1 (day 8) caused a decrease in S-phase cells of at least 4-fold, and a 2-fold decrease in G2/M-phase cells (Figure 1B), although no significant effect was found at the G1 phase. In addition, depletion of HP1 caused a greater than 7-fold increase in the number of apoptotic cells. These results, therefore, confirm that HP1 is an important regulator during the cell cycle, especially at the S and G2/M phases.<br><br>Cell-cycle regulators require HP1 to maintain their active transcription
To ask whether the cell-cycle defects were due to changes in the transcription of genes functionally involved in S phase and the G2/M phase, we next assessed global changes in gene transcription following depletion of HP1. Expression profile analysis was performed using total RNA isolated from both HP1-depleted Kc cells and control Kc cells, and an Affymetrix <span class="yellow">Drosophila</span> chip. For each experiment, we used total RNA isolated from two independent HP1-depleted and control samples, and at least two independent experiments were performed.
The microarray analysis showed that loss of HP1 function in Kc cells resulted in alterations in transcription of >500 genes: ∼400 genes were down-regulated and ∼120 genes were up-regulated (>1.5-fold, ANOVA). The function of these genes ranged from cellular enzymes, signal transduction molecules, and membrane and cell structural proteins, to nucleic acid-binding proteins and cell-cycle regulators (Figure 2A). At the chromosomal level, the genes targeted by HP1 appeared to be distributed along all euchromatic chromosomal arms (data not shown), supporting a global role of HP1 in euchromatic gene regulation (20).
Among 60 known/predicted genes associated with DNA replication function, 15 were down-regulated in the absence of HP1 (Figure 2B). These included McM2, McM5, McM6 and CDC45L, which are required for processive DNA replication and correct chromosome condensation (30–32). Other genes involved in DNA replication, such as components of the origin recognition complex (Orc)—Orc4, Caf1, Gnf1, Dref1, DNA polymerase-γ and Tam—were also down-regulated (Figure 2B). Aurora B and inner centromere protein (INCENP), known to be required for kinetochore assembly, chromosome condensation and bipolar chromosome attachment during mitosis (33), also showed a reduction in transcription. A similar loss of transcription was observed in Bub1 and Bub3 (Figure 2B), encoding mitotic checkpoint control proteins (34,35). Loss function of Bub1 has been shown to cause chromatin bridges to extend between the two separating groups of chromosomes, and extensive chromosome fragmentation in anaphase cells (35).
We confirmed the changes in the transcription of cell-cycle regulators using semi-quantitative RT–PCR, which gave results consistent with the microarray analysis. In addition, cell-cycle regulator genes, such as McM3, McM7 and Asp (abnormal spindle), were also confirmed to be down-regulated (Figure 2C). Collectively, these results demonstrate that HP1 is indeed involved in the regulation of transcription of cell-cycle regulators.<br><br>HP1 is required for Aurora B distribution and histone H3 phosphorylation
INCENP is localized to the centromeric region of chromosomes at metaphase and the spindle midzone at anaphase, which then targets Aurora B, a kinase essential for histone H3 ser10 phosphorylation, to these sites (36). Loss of function of both these ‘chromosomal passenger’ proteins causes abnormal chromosomal segregation at metaphase, as well as certain cytokinesis defects (36,37). The loss of transcription of both INCENP and Aurora B after depletion of HP1, therefore, raised the possibility that localization of Aurora B (Figure 3 and data not shown) may be altered. Staining of HP1-depleted Kc cells with anti-Aurora B antibodies indeed revealed an altered localization of Aurora B and, in a number of cases, a complete loss of Aurora B (Figure 3A). Consistent with the loss function of Aurora B, the spindles in the metaphase cells were also disorganized, with a large number of cells showing an altered prometaphase chromosome alignment (Figure 3A). Some showed extensive chromosome fragmentation (Figure 3B), or the presence of a third spindle pole-like structure as indicated by beta-tubulin (Figure 3A). At telophase, we observed defective separating cells with an extra cell envelope-like structure without nuclei (Figure 3A). Chromatin bridges or lagging chromatids at telophase were also evident in some cells (Figure 3C); however, in some cases, localization of Aurora B appeared not to be affected, arguing that other pathways are possibly involved.
We next analyzed changes in histone H3 serine 10 phosphorylation, since the loss of transcription of INCENP is known to affect localization of Aurora B (33), which is essential in the regulation of histone H3 phosphorylation (36). Total cell extracts from HP1-depleted Kc cells were analyzed by western blotting (Figure 3D). The results indeed showed a severalfold reduction in H3 ser10 phosphorylation after depletion of HP1, consistent with the functional disruption of INCENP and Aurora B in the absence of HP1.<br><br>HP1 directly targets genes encoding cell-cycle regulators in euchromatin
To test whether the loss of transcription of genes involved in DNA replication and mitosis was a direct effect of the loss of HP1, we performed a ChIP analysis to determine whether HP1 is physically associated with these genes. Chromatin lysates from formaldehyde-fixed Kc cells were sonicated into small chromatin fragments (0.2–0.8 kb) and immunoprecipitated with polyclonal antibodies against <span class="yellow">Drosophila</span> HP1. As a control, we used a mock precipitation (beads only) and polyclonal antibodies against HA. Our ChIP results showed that known transposable elements distributed in heterochromatin, such as F-element, TART and 1360 (7,38), were all enriched in HP1 binding (Figure 4), which is also consistent with a previous study (20).
Using the same ChIP DNA material, we then attempted to determine whether HP1 was enriched in genes involved in DNA replication. Primers were designed to cover the promoter regions of selected genes. The results showed that McM3, McM5 and Tam were all enriched in HP1 binding (Figure 4). However, McM7 appeared to be HP1-negative, although its transcription was also affected by the loss of HP1 function. Genes essential for mitosis, such as Aurora B, were also HP1-positive (Figure 4). These results demonstrate that these cell-cycle regulator genes are directly targeted by HP1 in their promoter regions.<br><br>
DISCUSSION
In this study, we used microarray and RT–PCR techniques to demonstrate that transcription of cell-cycle regulators is misregulated in the absence of HP1. Certain defects in S phase may be a direct consequence of the loss of transcription of DNA replication genes such as McM2, McM5, McM6, CDC45L, Orc4 and others, since these genes have been functionally implicated in the initiation of DNA replication and/or the progression of replication forks (39). Depletion or mutation of these genes has been shown to result in DNA damage (32), the blockage of replication forks (39), increased chromosome loss/genome instability, and defective condensation (30).
The reduction in the number of cells in G2/M phase may be a consequence of the reduction in transcription or functional disruption of INCENP, Aurora B, Bub1, and Bub3 (34,36). Chromosome segregation defects, such as chromosome fragmentation and chromatin bridges in anaphase/telophase cells, and certain cytokinesis defects in HP1-depleted cells, mimic the phenotype of cells with loss function of INCENP, Aurora B or Bub1 (35–37). The mislocalization of Aurora B in the absence of HP1 is also consistent with the loss of transcription and functional disruption of INCENP (37), and the reduction in Aurora B transcription may be partially responsible for the observed chromosomal defects, including loss of histone H3 phosphorylation at serine 10.
HP1 is also known to physically interact with certain components of replication complexes such as ORCs and MCMs (30,40,41), with the inner centromere protein INCENP (42) and the chromatin assembly factor CAF1 (43) promoting delivery of HP1 to heterochromatin sites (44). Loss of HP1 is, therefore, expected to cause disruption to such HP1-associated complexes, and will partially contribute to the chromatin/chromosomal defects in HP1 mutants (8,27) and HP1-depleted Kc cells. It is therefore well possible that the loss of transcription of these cell-cycle regulator genes, and consequent disruption of HP1 functional complexes or heterochromatin structure, all contributed to the cell-cycle defects observed.
The ChIP assay supports the hypothesis that the loss of transcription of cell-cycle regulator genes is a direct effect of the lack of HP1. Aurora B, McM3, and McM5 were all bound by HP1 at their promoter regions, although other cell-cycle regulators, such as McM7, were HP1-negative, implying that the altered transcription in these genes might be a secondary effect of the loss of HP1.
A previous study in <span class="yellow">Drosophila</span> Kc cells (20) employed an approach based on the ectopic expression of a fusion protein of HP1 with a prokaryotic DNA adenine methyltransferase and identified a number of methylated targets in the genome. In this study, MCM3 and MCM5 were not found to be methylated, indicating lack of association with HP1. On the other hand, heterochromatin repeats, such F-element and 1360, were consistently found to be HP1-enriched both here and in the previous study. It remains to be determined whether these discrepancies are due to the different experimental systems used. However, we note that the previous study was performed using a cDNA array, while we observe binding of endogenous HP1 at the promoter of these genes. Similarly, another study using chromatin immunoprecipitation in larvae also showed few HP1-positive genes that were not detected in Kc cells by the Dam ID approach (21).
A large number of genes affected by the loss of <span class="yellow">D.melanogaster</span> HP1 in larval tissues (21) seem to be different from that in embryonic Kc cells. The change in the transcription of Aurora B and few cell-cycle regulators reported in this study is also not found among the HP1-affected genes at larval stage (21). This may be due to specific role(s) of HP1 in different stages of development. Alternatively, it is also possible that the impact of HP1 in the transcription of cell-cycle regulators in proliferating cells is underestimated when performing the analyses on larval tissues, and thus on mixed populations of both proliferating and differentiating/differentiated cells.
HP1 is generally known as a transcriptional repressor, as supported by several lines of evidence: silencing of a euchromatic reporter gene in heterochromatin requires HP1 (10,11), tethering of HP1 next to a euchromatic reporter gene causes silencing (45), and the repression of genes within euchromatic region 31 bound by HP1 is relieved in the absence of HP1 (46). In contrast, genes in heterochromatin, known as heterochromatic genes, such as light and rolled, seem to require HP1 to maintain their active transcription (47,48). The level of transcription of heterochromatic genes was dramatically reduced in a mutated HP1 background (47–49). It was therefore proposed that HP1 may function as a positive regulator of transcription of these genes (50,51), although the exact regulation mechanism remains unclear. A study of heat-shock genes found that HP1 is associated with RNA transcripts in the coding region, and is also a positive regulator of their transcription (52,53). The chromatin association of HP1 at the promoter region of active euchromatic genes demonstrated from this work and others, and its independence from histone H3K9 methylation (21), all suggest that mechanism whereby HP1 modulates transcription of euchromatic genes is potentially distinct from its role in heterochromatin formation.
Collectively, the results of this study demonstrate that HP1 plays an essential role in cell-cycle progression, and support the view that HP1, in addition to its role in heterochromatin, can act as a positive transcriptional regulator of euchromatic genes.<br><br>SUPPLEMENTARY MATERIAL
Supplementary Material is available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA1190194</h3>Structural organization and interactions of transmembrane domains in tetraspanin proteins
Abstract
Background
Proteins of the tetraspanin family contain four transmembrane domains (TM1-4) linked by two extracellular loops and a short intracellular loop, and have short intracellular N- and C-termini. While structure and function analysis of the larger extracellular loop has been performed, the organization and role of transmembrane domains have not been systematically assessed.<br><br>Results
Among 28 <span class="yellow">human</span> tetraspanin proteins, the TM1-3 sequences display a distinct heptad repeat motif (abcdefg)n. In TM1, position a is occupied by structurally conserved bulky residues and position d contains highly conserved Asn and Gly residues. In TM2, position a is occupied by conserved small residues (Gly/Ala/Thr), and position d has a conserved Gly and two bulky aliphatic residues. In TM3, three a positions of the heptad repeat are filled by two leucines and a glutamate/glutamine residue, and two d positions are occupied by either Phe/Tyr or Val/Ile/Leu residues. No heptad motif is apparent in TM4 sequences. Mutations of conserved glycines in <span class="yellow">human</span> CD9 (Gly25 and Gly32 in TM1; Gly67 and Gly74 in TM2) caused aggregation of mutant proteins inside the cell. Modeling of the TM1-TM2 interface in CD9, using a novel algorithm, predicts tight packing of conserved bulky residues against conserved Gly residues along the two helices. The homodimeric interface of CD9 was mapped, by disulfide cross-linking of single-cysteine mutants, to the vicinity of residues Leu14 and Phe17 in TM1 (positions g and c) and Gly77, Gly80 and Ala81 in TM2 (positions d, g and a, respectively). Mutations of a and d residues in both TM1 and TM2 (Gly25, Gly32, Gly67 and Gly74), involved in intramolecular TM1-TM2 interaction, also strongly diminished intermolecular interaction, as assessed by cross-linking of Cys80.<br><br>Conclusion
Our results suggest that tetraspanin intra- and intermolecular interactions are mediated by conserved residues in adjacent, but distinct regions of TM1 and TM2. A key structural element that defines TM1-TM2 interaction in tetraspanins is the specific packing of bulky residues against small residues.<br><br><br><br>Background
Tetraspanins constitute a large family of integral membrane proteins, characteristically containing 4, 6 or 8 conserved cysteine residues in the large extracellular loop (including the CCG and PxxCC motifs), which form disulfide bonds, and several conserved polar residues in the intracellular loop and transmembrane regions [1,2]. There are 32 putative tetraspanin family members in mammals, 37 in <span class="blue">Drosophila melanogaster</span> and 20 in <span class="yellow">Caenorhabditis elegans</span>. Tetraspanins play diverse roles in cell adhesion, migration and fusion processes, cellular activation and signaling (reviewed in refs. [2-4]). Mammalian tetraspanins such as CD9, CD63, CD81, CD82, CD151, rds/peripherin, and uroplakins Ia and Ib have been most extensively studied, with <span class="yellow">mouse</span> knock-out models available for CD9 [5-7], CD81 [8,9], CD151 [10] and a few others. However, the majority of tetraspanins are characterized very little, if at all, at genetic, biochemical or structural levels.
The large extracellular loop (LEL) of tetraspanins has received most attention, since it contains functionally important sites. Sequence QRD (194–196) in CD151 is important for association with integrins, which has functional consequences for integrin-dependent cell spreading and multicellular cable formation [11]. A site in the LEL of CD9, SFQ (residues 173–175), is essential for CD9 function in sperm-egg fusion [12]. The crystal structure of tetraspanin CD81 LEL revealed five α-helixes, A-E [13]. Helices A, B and E form a relatively conserved region in tetraspanins, whereas the region between helices B and E is the most variable [14]. Interestingly, the variable region contains most of the functionally important sites involved in tetraspanin protein-protein interactions.
A remarkable biochemical property of tetraspanin molecules is their ability to associate with a large number of other transmembrane proteins, including integrins, membrane-associated growth factors and receptors, MHC class II molecules, Ig superfamily proteins, and each other [2,3,15]. Several of these lateral associations of tetraspanins are detected in "mild" detergents (Brij series, CHAPS), but are disrupted by "strong" detergents such as Triton X-100 or SDS. Multiprotein complexes of tetraspanins and associated molecules, also called the "tetraspanin web" [16], may represent a distinct tetraspanin-enriched membrane microdomain [17,18]. The formation of this microdomain is influenced by palmitoylation of several conserved juxtamembrane cysteine residues in tetraspanins [19-21].
The transmembrane domains, encompassing nearly half of a tetraspanin protein, are the most conserved part of the molecule (Stipp et al. [1] and this study). However, very little functional information is available on these domains. The differential detergent sensitivity of tetraspanin-tetraspanin associations suggests that hydrophobic interactions between TM helices may play a role. Indeed, when the large extracellular loop (LEL) of CD151 is deleted, the molecule is still able to associate with other tetraspanins [22]. Thus, TM domains are strong candidates for mediating tetraspanin-tetraspanin interactions.
The importance of TM domain interactions in intramolecular organization was demonstrated in a study showing that CD82 fragment TM2-4, lacking TM1, was retained in the endoplasmic reticulum, but was transported to the cell surface upon co-expression of TM1 [23]. This in vivo reconstitution experiment demonstrated a strong interaction between TM1 and the rest of the molecule. Expression of a truncated CD9 molecule (TM3-LEL-TM4) results in intracellular accumulation of the protein and significant misfolding of the LEL, as judged by inappropriate disulfide formation and diminished antibody reactivity (our unpublished data). Similarly, a CD9 epitope in the LEL is lost in molecules lacking either TM2+TM3 or just TM4 [24]. Thus, TM domain interactions and packing are crucial for proper folding, stability and transport of tetraspanin molecules.
In a previous study, we showed that covalent cross-linking of membrane-proximal cysteine residues can be used as a tool for detection of tetraspanin-tetraspanin associations [25]. Inhibition of cysteine palmitoylation by 2-bromopalmitate (2-BP) made cysteines available for cross-linking and enabled demonstration of specific tetraspanin homodimerization and low levels of heterodimerization. We concluded that tetraspanin homodimers, formed in the Golgi, may be a fundamental structural unit within tetraspanin microdomains.
In this study, we carried out detailed sequence analysis of <span class="yellow">human</span> tetraspanin TM domains. We show that a heptad repeat containing conserved glycine, asparagine and large hydrophobic residues occurs in TM1 and TM2 domains, and predict tight intramolecular association of these two domains by packing of the large residues against the small residues. Moreover, by using cysteine cross-linking we map a dimerization interface in the <span class="yellow">human</span> CD9 protein, and show that conserved heptad motif glycine residues are also important for intermolecular CD9 associations.<br><br>Results
Sequence analysis of tetraspanin transmembrane domains: presence of the heptad repeat motif
We focused our attention on 28 <span class="yellow">human</span> tetraspanins identified from the SWISS-PROT and GenBank databases. All tetraspanins have in common four hydrophobic stretches (TM domains) of 20–25 residues, and contain highly conserved residues in the second extracellular loop, in particular the Cys-Cys-Gly (CCG) motif. Detailed analysis of the large extracellular loop sequences [14], and dendrograms based on full-length alignment can be found in earlier studies [26,27]. The length of each transmembrane domain was established based on previous sequence analysis of tetraspanin sequences [27,28], and on annotations to the database entries. Manual adjustments based on sequence homology and hydrophobicity profiles were done to fully delineate the TM domains. The resulting lengths of TM domains were: TM1 – 23 residues; TM2 – 21 residues; TM3 – 25 residues; TM4 – 25 residues. Two more residues could be added onto the N-terminal part of TM2; however, relatively small sequence conservation of these residues among tetraspanins and occurrence of polar/charged side chains in some tetraspanins precluded us from doing so for the global alignment.
Figures 1 and 2 show a multiple sequence alignment of four TM domains of 28 <span class="yellow">human</span> tetraspanins. For each position within the domains, consensus residues were determined and classified (with individual color code) in 4 categories: 1) large hydrophobic residues (including Val, Met, Leu, Ile, Phe, Tyr, Trp), 2) small residues (Gly, Ala, Ser and Thr), 3) Cys, and 4) Asn. When more than two types of residues occupied a given position in a TM, a dual-color pattern that reflected the prevalence of the particular residue type was used (Figure 1). Cysteine residues were shown separately due to their importance as palmitoylation target sites. The highly conserved asparagine residue in TM1 was considered separately. No proline residues are found in TM domains 1–3 of <span class="yellow">human</span> tetraspanins.
An inspection of the multiple sequence alignment reveals a repeating heptad amino acid pattern, (abcdefg)n, in TM1, 2 and 3 (Figure 1, 2). Heptad repeats promote helical coiled coil interactions in multiple soluble and membrane-spanning proteins [29-31]. In the heptad repeat, hydrophobic residues in positions a and d are of special importance, as they directly mediate interhelical contacts by creating a tight knobs-into-holes packing in the coiled coil structure [32]. For instance, in the leucine zipper of the <span class="yellow">yeast</span> transcription factor GCN4, positions a and d contain Val and Leu residues, respectively, with an Asn residue in a single a position forming a hydrogen bond across the GCN4 dimer interface [33].
In TM1 of tetraspanins, highly conserved Asn, Gly and Gly residues (numbers 18, 25 and 32 in the CD9 sequence) appear at d positions of the heptad repeats, and residues 14, 21 and 28 are at a positions (Figure 1). In TM2, residues 67, 74 and 81 (consensus Gly, Gly and Ala, respectively) occupy a positions, whereas residues 63, 70 and 77 are at d positions. Another highly conserved glycine, Gly80, occupies the 3rd g position in TM2. In TM3, the conserved pattern consists of two leucine residues (Leu89 and Leu96) and a glutamate/glutamine residue (Glu/Gln103) in a positions (Figure 2). Two d positions are also conserved – Phe/Tyr92 and Ile/Val/Leu99. TM4 lacks a conserved heptad pattern and has only a single conserved position, Glu/Gln209 (with four exceptions). These features of TM1-4 of tetraspanins are displayed on helical wheel diagrams (Figure 3).<br><br>Analysis of TM1 sequences
The conserved Asn-Gly-Gly motif, occupying designated d positions of the heptad repeat, is the most prominent structural feature of TM1. We also compared sequences of CD9 orthologs from 10 different organisms (the most available for any tetraspanin) to gain further insight into conservation and variability of the TM1 sequence. As shown in Figure 4, positions a, d and g in TM1 are among the most conserved (0, 1 and 1 substitution, respectively), while interspecies variability tends to occur in other positions: b (5 substitutions), c (4 substitutions), e (4 substitutions) and f (4 substitutions). Thus, the positions typically involved in coiled coil interactions (a and d) are the most conserved.
When residues of TM1 are plotted as a helical wheel, additional structural features are revealed (Figure 3). There are highly conserved aliphatic and aromatic residues in the first three a positions of the heptad motif (Phe15, Trp22 and Leu29 in CD9), as well as in g positions (Leu14, Phe21, Val28 in CD9). The "ridges" formed by these bulky residues are flanking the "groove"-forming Gly residues of the Asn-Gly-Gly position d motif. In contrast, b, c, e and f positions show an overall higher variability among tetraspanins, as also seen in the comparison of CD9 orthologs described above.<br><br>Analysis of TM2 sequences
A landmark feature of TM2 in tetraspanins is the presence of highly conserved glycine residues (Gly67, 74, 77 and 80 in CD9, Figure 1). Other substitutions at these positions are almost exclusively small residues, such as Ala or Ser. In addition, Ala, Ser or Thr occupy position 81. This residue, together with Gly67 and Gly74, forms face a of the helix. Residue Gly77 (position d) is preceded by conserved, chiefly large hydrophobic residues on the same helical face (Leu63 and Met70 in CD9). Extremely conserved Gly80 falls into heptad position g (Figure 3). Among CD9 orthologs, heptad positions a and d are absolutely conserved, whereas other positions have the following number of substitutions: b – 3; c – 2; e – 1; f – 3; g – 1 (Figure 4). Two of the f position residues in TM2 (65 and 79) also show higher variability among different tetraspanins (Figures 1, 3). Cysteine residues are frequently found near the cytoplasmic end of TM2 helix at positions 78 and 79; these cysteines are likely to be palmitoylated.<br><br>Analysis of TM3 and TM4 sequences
The TM3 domain provides another example of the heptad repeat pattern. Position a is occupied by two highly conserved leucine and a glutamate/glutamine residue (Leu89, Leu96 and Glu/Gln103 in CD9). Furthermore, two d positions are conserved – Phe/Tyr92 (aromatic residue) and Ile/Val99 (β-branched aliphatic residue; Figures 2, 3). In addition, residue 100 in position e is generally Phe or Leu. Among CD9 orthologs, position a has 1 substitution, positions b, c and f each have 6, positions d and e each have 2, and g has 4. Thus, as for TM1 and TM2, positions a and d are among the most conserved, but overall TM3 has more variable positions than TM1 or TM2 (Figure 3). Less than half of TM3 sequences contain cysteine residues, and those tend to occur at the internal positions of the helix (Figure 2).
TM4 shows less conservation among various tetraspanin family members than the other TM domains (Figures 2, 3). The only highly conserved feature is the glutamate/glutamine residue in position 209. In addition, one or two cysteine residues can be found at the C-terminal end of TM4 in some tetraspanins (e.g. CD9, CD81, CD151), and many sequences contain additional polar residues (Arg, Lys, His, Asn, Gln). No conserved heptad motif was identified in TM4, as also confirmed by analysis of substitutions in CD9 orthologs (data not shown).<br><br>Mutational analysis of conserved glycine residues in TM1 and TM2
The conserved nature of the Asn and Gly residues in TM1 and TM2 prompted an analysis of their functional role. To this end, we have probed whether mutations of these residues destabilize the protein molecule. We expressed a construct of the first and second TMs of CD9, connected by the small extracellular loop, and tagged with a C-terminal green fluorescent protein (TM(1+2)-GFP molecule). In <span class="yellow">human</span> rhabdomyosarcoma RD cells, the wild-type fusion protein localized mostly in a reticular, intracellular pattern, without forming any large aggregates (Figure 5, panel A). Remarkably, when double mutants Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu were expressed, the protein formed distinct large aggregates in a high proportion of cells (Figure 5, panels C and E). In contrast, double mutant Gly77Leu + Gly80Leu did not form such aggregates (Figure 5, panel G). Results with respective single mutants were similar to that with double mutants, with the aggregation being somewhat more pronounced for Leu substitutions of Gly67 and Gly74 compared to Gly25 and Gly32 mutations. No aggregation was observed for Asn18Ser and Asn18Tyr mutants (data not shown). Also, nearly identical results were obtained with <span class="yellow">human</span> HT1080 cells (data not shown).
We interpret these results as an indication that aggregating mutants are destabilized or misfolded while non-aggregating mutants retain the wild-type conformation. Intriguingly, mutations to the conserved GG7 motifs caused protein aggregation while the mutation of other glycines had no detectable effect. These results also suggest that wild-type GFP, which has weak tendency to self-associate, could enhance non-specific interactions of destabilized mutant TM(1+2) CD9 moieties, leading to their aggregation. Consistent with this hypothesis, the aggregation of Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu double mutants was suppressed when monomeric GFP molecule, Leu221Lys [34] was used (Figure 5, panels D and F). The use of monomeric GFP did not affect intercellular localization of wild-type CD9 TM(1+2) (Figure 5, panel B), or a Gly77Leu + Gly80Leu double mutant (Figure 5, panel H).
In summary, Leu substitutions of Gly residues that are part of the Asn-Gly-Gly (NGG7) motif in TM1, or Gly-Gly-Ala (GGA7) motif in TM2, resulted in destabilization and aggregation of GFP-fused TM(1+2) proteins, whereas substitutions of Gly77 or Gly80, which are not part of these motifs (Figure 3), failed to show such aggregation.<br><br>Prediction and modelling of interaction between TM1 and TM2
Consecutive helices in polytopic membrane proteins frequently interact [35]. Sequence analysis of TM1 and TM2 helices of tetraspanins reveals a remarkable complementarity in the distribution of large and small residues at heptad positions a and d along the helical axis (Figure 3), suggesting that these residues may interact. To further elucidate the potential for TM1-TM2 interaction, the putative interface was modeled using a novel algorithm that considers mutational data during each step of a Monte Carlo simulated annealing cycle (see Methods for details). Specifically, Gly25Leu, Gly32Leu, Gly67Leu and Gly74Leu were scored as disruptive mutations, while Asn18Ser, Gly77Leu and Gly80Leu were scored as silent mutations, based on their effects on protein stability (Figure 5 and data not shown).
The resulting model predicts left-handed crossing of TM1 and TM2 helices at an angle of +28°. The key element of the structure is the apposition of bulky and small heptad position a and d residues, as follows: Gly32-Leu63; Gly67-Leu29; Gly25-Met70; Gly74-Trp22; Asn18-Gly77; Ala81-Phe15 (Figure 6). Our model predicts that each of these residue pairs are in van der Waals contact. Additionally, two potential H-bonds are predicted in this model, indicating close packing: Gly67 Cα to Gly25 carbonyl oxygen, and Trp22 Cα to Met70 carbonyl oxygen. The packing is tighter in the ectodomain-proximal portion of the helices (Figure 6, panel B), as determined by Cα-Cα distances between interacting residue pairs.
The key elements of the model are corroborated by the presence of apparently complementary substitutions in TM1 and TM2 sequences of different tetraspanins (Figure 1, boxed residues). For example, Gly74 is predicted to interact with Trp22. In 8 of the 10 tetraspanins that contain a substitution for Gly74, a compensatory substitution occurs at the Trp22 position (Figure 1). Thus, a larger non-glycine side chain at position 74 may necessitate a less bulky non-Trp side chain in position 22. Likewise, the presence of a Cβ at position 25, typically occupied by glycine, necessitates a non-β-branched amino acid at position 70, which is occupied by a β-branched residue in nearly half of all cases. Indeed, we find that in each of 5 cases in which position 25 contains a Cβ, a leucine residue occurs in position 70. This analysis is consistent with our molecular model that suggests Leu70 will pack most favorably against a Cβ at position 25 than a β-branched residue or a methionine.<br><br>Role of TM1 and TM2 heptad motif residues in CD9 dimerization
To probe CD9 dimerization, we used a cysteine-mediated cross-linking approach. We established previously a simple and efficient method for cysteine-mediated cross-linking [25]. After cells are pre-treated with 2-BP for 16–24 hours to expose normally palmitoylated cysteines, the cysteines can be cross-linked using any of the following methods: a) Spontaneous oxidation in Brij97 lysates (a condition that preserves tetraspanin-tetraspanin associations), b) In situ cross-linking, by pre-lysis oxidation of cells with Cu2+-phenanthroline (CuP) to promote disulfide bond formation. c) In situ cross-linking with thiol-reactive cross-linking agents of defined length (e.g. DTME, BMB). The first two approaches produce in essence "zero-length" disulfides, indicative of close proximity of target cysteines and presumably high specificity of interaction. In contrast, chemical cross-linkers with 6–20 Å spacer arm may cross-link with higher efficiency, but not necessarily higher specificity. However, they provide advantages such as variable membrane permeability, and potential linkage cleavability. For tetraspanins such as CD9, membrane-permeable cross-linker DTME (13.3 Å-long, reducible) provides highly specific and efficient cross-linking [25]. Here we have used a cysteine cross-linking strategy, in combination with cysteine-scanning mutagenesis, to map the residues from TM1 and TM2 contributing to the CD9 dimerization interface.
For subsequent cross-linking experiments using CD9 TM(1+2)-GFP protein, the non-dimerizing form of GFP was used. This avoids potential GFP-dependent dimerization and aggregation that can be observed with wild-type GFP, especially when fusions with transmembrane proteins are studied [36]. Importantly, the Leu221Lys mutation in GFP prevented aggregation of mutant forms of CD9 TM(1+2), which was observed with wild-type GFP fusion (Figure 5). The TM(1+2) fragment of CD9 contains three native cysteines – Cys9, Cys78 and Cys79. Single-cysteine mutants of TM(1+2) were constructed, in which a cysteine was placed at various faces of TM1 or TM2 while all of the wild-type cysteines were simultaneously replaced by serines. The mutant proteins were transiently expressed in RD cells (having little endogenous CD9), which were then treated for 16–18 hours with 2-BP. To achieve maximal specificity in cross-linking we used a "zero-length" agent, CuP.
First, single-cysteine replacements were constructed for residues Leu14, Phe15, Gly16, Phe17 and Asn18, covering just over one complete helical turn at the beginning of TM1. While residue Asn18 is highly conserved, positions 14, 15 and 17 are occupied by bulky hydrophobic residues in most tetraspanins, whereas position 16 shows less conservation (Figures 1, 4). All of the single-cysteine mutants showed diffused pattern of protein localization, without any signs of aggregation. As shown in Figure 7A, the highest level of intermolecular cross-linking was observed for Leu14Cys and Phe17Cys mutants, a lower level for Phe15Cys and Gly16Cys mutants, and very little cross-linking for Asn18Cys substitution. These results indicate that: a) the first two transmembrane domains of CD9 alone can mediate its dimerization, and b) the g and c residues of TM1 (e.g. Leu14 and Phe17, Figure 3) are likely to be part of the intermolecular interface.
Similarly, single-cysteine substitutions were made for residues Gly77, Gly80 and Ala81 in TM2; in addition, proteins carrying a single wild-type cysteine, Cys9, Cys78 or Cys79, were tested. No protein aggregation was observed for any of these single-cysteine mutants. As shown in Figure 7B, the relatively low level of intermolecular cross-linking of wild-type CD9 TM(1+2)-GFP protein was enhanced dramatically in single-cysteine TM2 mutants Gly80Cys and Ala81Cys. The Gly77Cys mutant also had an elevated level of cross-linking. In contrast, any of the three native cysteines (9, 78 and 79) produced level of cross-linking not much greater than the wild-type TM(1+2) protein. Similar results were obtained with cysteine-reactive cross-linker BMB (data not shown). Likewise, comparable results were obtained with single-cysteine mutants of untagged, full-length CD9, using CuP (Figure 7C) as well as DTME cross-linker (data not shown).
These cross-linking results for TM1 and TM2 are consistent with our model that places residues Leu14, Phe17 and Gly80 on the same side of the TM1-TM2 pair (Figure 6, panel C). The strong cross-linking with Leu14Cys, Phe17Cys and Gly80Cys places the intermolecular interface toward the c and g phases of the TM1 helix, and the g phase of the TM2 helix, away from its e and f faces containing wild-type cysteines 78 and 79.
Critical residues at the TM1-TM2 interface also affect dimerization indirectly. To assess specific CD9 dimerization, we used a Gly80Cys substitution at the intermolecular interface for cross-linking. As shown in Figure 8A, single replacements of conserved heptad residues in positions 18, 25, 32, 67 and 74 (Asn18Ser, Gly25/32/67/74→Leu) strongly decreased the cross-linking mediated by Cys80. The effect was most pronounced for mutations of residues, Gly32 and Gly67, located in the tightly packed extracellular end of TM helices (Figure 6). In contrast, mutations of residues closer to the cytoplasmic end of TM2 (Gly74 and especially Ala81) had only modest to very little effect on cross-linking.
Relatively low efficiency of intermolecular cross-linking via native residues Cys9, 78, and 79 (Figures 7B,C) correlates well with the predicted location of Cys78 and 79 away from the dimeric interface (Figure 3), and suggests that the extramembrane N-terminal part of CD9 (residues 1–13) does not self-associate. We next examined whether mutations of conserved Asn and Gly residues in TM1 and TM2 decreased low-level background cross-linking via native cysteines. As expected, these mutations had virtually no effect on dimer formation of CD9 TM(1+2)-GFP (Figure 8B). The level of covalent dimer formed was not diminished for triple Asn18Ser + Gly25Leu + Gly32Leu and double Gly67Leu + Gly74Leu mutants, compared to wild-type TM(1+2) CD9 molecule. Similarly, the same triple and double mutations in the context of full-length CD9-GFP protein (with six cysteines) produced wild-type levels of cross-linking (Figure 8C). We interpret these findings as evidence for at least two types of associations between CD9 molecules: primary, involving residues 14, 17 and 80, and dependent on integrity of conserved heptad residues in TM1 and TM2, and less efficient secondary interactions, probably representing random collision events, and independent of the heptad residues (see Discussion for more details).<br><br>TM3 and TM4 cysteine residues in CD9 dimerization
After identifying the roles of conserved TM1 and TM2 residues in CD9 dimerization, we next probed whether residues proximal to TM domains 3 and 4 are also involved. To this end, disulfide cross-linking of full-length CD9 molecules containing 3 C-terminal cysteines (87, just before TM3; 218 and 219 in TM4) or 3 N-terminal cysteines (9, 78 and 79) was compared (Figure 9). We found that the C-terminal cysteines were only slightly better than N-terminal cysteines with respect to detection of CD9 dimers. However, markedly more trimers and tetramers were detected using C-terminal cysteines. Thus, residues 87, 218 and 219 at TM3 and TM4 in CD9 can together form contacts across the dimeric interface and also additional contacts with other neighboring CD9 molecules.<br><br>
Discussion
Here we provide the first detailed analysis of tetraspanin protein transmembrane domains. First, we show 1) the presence of a heptad repeat motif in TM1 and TM2, containing highly conserved Asn and Gly residues, 2) a leucine and glutamate/glutamine-containing heptad motif in TM3, and 3) high variability and absence of heptad repeats in TM4 sequences. Second, we provide evidence for a specific, intramolecular interaction between TM1 and TM2 domains, in which bulky hydrophobic residues pack against GG7 motif, and present a molecular model for this interaction. Third, experimental mapping of the CD9 dimerization interface firmly establishes an additional role for conserved TM1 and TM2 residues in dimeric intermolecular interactions. Fourth, preliminary evidence is provided to suggest that TM3 and TM4 domains contribute to expansion of CD9 dimers into higher order multimers.
Conserved residues in TM1 and TM2 of tetraspanins: role in intramolecular packing
We hypothesized that the first two transmembrane domains of tetraspanins might interact with each other because: a) consecutive TM domains frequently associate in known protein 3D structures [35], and b) they both contain a series of highly conserved amino acids – several Gly residues and an Asn residue (Figure 1). Conserved Gly residues are a frequent theme in the organization of interacting transmembrane domains. Analysis of 3D helix packing in polytopic membrane proteins reveals that Gly residues tend to localize in buried positions, especially at the helix-helix interfaces and helix crossing points [37,38]. Due to the absence of a side chain, Gly provides a flat surface for packing of a side chain from another residue, without loss of side-chain entropy upon interaction. The most common Gly-containing motif is GxxxG [39,40]. In glycophorin A (GpA), the major glycoprotein in erythrocyte cell membranes, Gly79 and Gly83 are part of the LIxxGVxxGVxxT sequence that promotes homodimerization of parallel transmembrane α-helixes [41,42]. In the GpA dimerization motif, Gly residues allow for tight packing in the right-handed helical crossing [43]. There are also examples of left-handed helical crossing in the context of a GxxxG motif [44]. Other membrane proteins that use the GxxxG motif for homo- or heterodimerization include <span class="yellow">bacteriophage M13</span> coat proteins [45], <span class="yellow">yeast</span> alpha factor receptor [46], integrin α IIb subunit [47], and ErbB1 receptor tyrosine kinase [48]. Other small residues, such as Ala and Ser, can substitute for Gly in this motif [49].
A protein motif in which Gly residues are separated by 6 other residues (GG7) is also common in transmembrane helices, especially in transporter/channel-like membrane proteins [50]. However, the structural features associated with this motif are not well known. In particular, it is unclear whether left-handed GG7 heptad repeat motif (as opposed to the "classic" right-handed GxxxG motif) can drive membrane helix association. In a recent work addressing this issue, Lear et al. [51] showed that a synthetic peptide containing Gly at heptad positions a and d could self-associate in vitro, likely in an antiparallel orientation. Heptad repeats containing conserved Gly residues occur in TM domains of α and β chains of MHC class II proteins, and mutations of the Gly residues disrupt the αβ heterodimer [52]. These examples demonstrate that Gly-based heptad motifs may be used for both intra- and intermolecular associations.
In this work, we identified a highly conserved GG7 motif in the first two tetraspanin TM domains. The GG7 sequence in tetraspanins is a part of a larger motif that also includes a conserved Asn residue in TM1 (NGG7) and an Ala/Ser/Thr residue in TM2 (GGA7). The seven-residue periodicity of these motifs strongly suggests their involvement in left-handed coiled coil packing reminiscent of the leucine zipper, rather than right-handed packing of the GpA-like GxxxG motif. For antiparallel helices, the left-handed crossing is in fact predominant over the right-handed in known TM domain structures [44].
In our model, heptad Gly residues in NGG7 and GGA7 sequences provide specific packing between antiparallel tetraspanin TM1 and TM2 helices by allowing tight van der Waals interactions with large hydrophobic residues (Figure 6). Highly efficient packing of bulky side chains against glycine residues is observed in known transmembrane protein 3D structures [38,53,54]. An example includes packing of helices M1 and M2 in potassium channel KcsA, where Val91 in M2 is paired with Gly43 in M1, and Leu36 in M1 contacts Ala98 and Gly99 in helix M2 [54,55]. In addition to facilitating helix-helix packing, Gly residues frequently provide additional CαH...O hydrogen bonds between two helices [44]. In our model, two Cα-backbone carbonyl H-bonds are predicted – between residues Gly27-Gly67, and Trp22-Met70.
Although polar and charged amino acid residues (such as Asn in the TM1 heptad motif) are infrequent in transmembrane domains, they are functionally important. Polar residues such as glutamine, glutamic acid, aspartic acid and asparagine can promote strong oligomerization of model membrane-associated helices [56-58]. Ruan et al. [59] used asparagine scanning mutagenesis to probe the interface of self-associating polyleucine helices by detecting their enhanced self-interaction in vitro and in the <span class="yellow">E. coli</span>-based ToxR assay. Thus, a hydrogen bond in an apolar environment can result in strong, though not necessarily specific, association of transmembrane helices. In fact, mutations to polar residues in transmembrane proteins are commonly associated with disease [60]. Because of this potential for non-specific interactions, polar residues tend to localize at buried positions in TM domains.
In our case, the conserved Asn18 residue in CD9 is predicted to be a part of the TM1-TM2 interface, though our model does not predict any electrostatic interaction between Asn18 and TM2 (Figure 6). Consistently, substitution such as Asn18Tyr (and Gly77Leu) in TM(1+2)-GFP protein was not destabilizing as analyzed by protein aggregation. Curiously, the full-length Asn18Ser CD9 migrated slightly slower on SDS-PAGE gel (data not shown), suggesting that Asn18 does play a role in maintaining conformation of the molecule. The Asn18Cys single-cysteine mutant shows very little intermolecular cross-linking (Figure 7A), supporting the proposed location of this residue at the intramolecular interface. It is tempting to speculate that the "pocket" between TM1 and TM2 lined by Asn18 and Gly77 might be important for accommodating palmitate moieties that target Cys78 and Cys79 residues, and/or important for access by the putative palmitoyl transferase to those residues. Understanding the exact role of these highly conserved Asn18 and Gly77 residues in tetraspanins awaits further investigation.
In summary, we identified conserved glycine residues of TM1 and TM2 of tetraspanins as key elements required for intramolecular packing. Mutations of these key residues (Gly25, Gly32, Gly67 and Gly74 in CD9) resulted in protein destabilization and aggregation. There is ample evidence in the literature for mutations in transmembrane proteins that lead to protein destabilization, misassembly and pathologic conditions [61]. Thus, we have identified conserved heptad Gly residues in TM1 and TM2 of tetraspanins as plausible targets of destabilizing mutations with potential functional consequences.<br><br>Intermolecular interactions in tetraspanins
Tetraspanin CD9 forms mostly homodimers, but also a low level of heterodimers with CD81 and CD151 [25]. Thus, mapping the dimerization interface is an important next step in structure-function analysis of tetraspanins. Disulfide-mediated cross-linking, often in combination with cysteine-scanning mutagenesis, is a common strategy to probe oligomerization or intersubunit interactions of transmembrane proteins such as histidine kinase EnvZ [62], M(3) muscarinic acetylcholine receptor [63], <span class="yellow">E. coli</span> lactose permease [64], synaptobrevin [65], integrins [66] and many others. In tetraspanins such as CD9, membrane-proximal cysteine residues are especially useful targets for disulfide trapping, as their linkage can be enhanced by pre-treating cells with 2-BP. While the ability of wild-type cysteines in CD9 to be cross-linked may indicate that they are close to the dimerization interface, more precise mapping was achieved here using cysteine-scanning mutagenesis.
Our data clearly identify regions, near the cytoplasmic face of TM1 and TM2, important for dimerization. Intermolecular zero-length cross-linking was highest when single cysteines were placed in positions 14, 17, 77, 80 and 81 in TM(1+2)-GFP molecule, or at positions 77, 80 or 81 in the full-length CD9 protein. Positions 14, 17 and 80 are distinct from the intramolecular interface and are on the same side of the TM1-TM2 pair (Figure 6). Thus, they are well-positioned to participate in an interaction with another molecule. At the same time, the model predicts that wild-type cysteines (Cys78, 79), which do not yield very efficient zero-length cross-linking, are on the other side of TM1-TM2 pair.
While using the cysteine at position 80 as the dimeric interface probe, mutations of conserved residues in TM1 and TM2 (especially Gly32 and Gly67 to Leu) clearly reduced intermolecular cross-linking. We do not suggest that those residues are directly involved in intermolecular interaction. Rather, we propose that destabilization of the intramolecular TM1-TM2 interaction by Gly to Leu substitutions (discussed above) causes an overall conformational change that reduces dimer formation.
An Ala81Leu mutation did not reduce cross-linking via Cys80, even though single-cysteine Ala81Cys molecules themselves produced a high level of cross-linking. These results, together with data on Gly32Leu and Gly67Leu mutations, are consistent with our model predicting that helices 1 and 2 interact more tightly near the extracellular end and less at the cytoplasmic end. This would give more flexibility to a cysteine at position 81 and also limit the effect of an Ala81Leu mutation. Location of this residue at the membrane/cytoplasmic border could also make it more accessible to CuP reagent as compared to residues buried in TM domain, thus elevating the efficiency of disulfide formation of the Ala81Cys mutant.<br><br>Multiple interfaces in tetraspanin molecules
In the full-length CD9 molecules, the 3 C-terminal cysteines (Cys87, 218 and 219) located at or in TM3 and TM4 promoted efficient dimer and even more efficient oligomer formation compared to the 3 N-terminal cysteines (Figure 9). Cys87 alone can be used to capture CD9 dimers [25]. These results suggest the existence of two dimeric interfaces in CD9 molecule – the TM 1-2/1-2 interface and the TM 3-4/3-4 interface (Figure 10). In a TM(1+2) molecule, the destabilization of 1–2 interaction, e.g. by Gly→Leu mutations, would affect the 1-2/1-2 interface, as discussed above. However, these mutations would not interfere with the 3-4/3-4 interface in a full-length molecule, which includes Cys87, 218 and 219. Thus, cross-linking of full-length molecules, containing all 6 cysteine residues, would be unaffected, as seen in Figure 8C. Furthermore, wild-type Cys9, Cys78 and Cys79 are apparently not at the primary 1-2/1-2 interface. Their relative inefficiency in cross-linking CD9 TM(1+2) protein likely reflects weak secondary contacts between the molecules, or possibly random collision events. Such events should be independent of mutations in the conserved Gly residues in TM1 and TM2, as was demonstrated in Figure 8B. The potential existence of two interfaces in tetraspanin molecules, 1-2/1-2 and 3-4/3-4, should provide enhanced flexibility for forming additional intermolecular contacts. Current understanding of tetraspanin microdomains assumes a few strong, primary homotypic and heterotypic tetraspanin complexes (e.g. CD9-CD9, CD9-CD81, CD151-α3 integrin, CD81-EWI2) that help bring together various other proteins, forming secondary-type associations. Such properties of tetraspanins may bring signaling molecules such as protein kinase C or phosphatidylinositol 4-kinase to the vicinity of integrins [67,68].
The organization of the TM3 domain points to a potential role in protein-protein interactions. A motif of Leu-Leu-Glu(Gln) spaced 7 residues apart (heptad positions a), with highly conserved residues in two consecutive positions d, poses as a likely interaction module. If responsible for heterologous protein-protein interactions, it would form another distinct interface of tetraspanin molecule. Our preliminary data indicate that replacing the Leu and Glu residues in TM3 of CD9 with Ala has no effect on cell surface expression of the protein and its dimerization (data not shown). It remains to be tested if interactions with other proteins will be affected. Similarly, the TM4 domain may provide additional contributions to lateral tetraspanin associations. Much higher sequence variability, and the lack of a distinct heptad pattern suggests that TM4 is a major contributor to diversity among tetraspanin complexes. Structure-function analysis of TM3 and TM4 domains in tetraspanins is the subject of ongoing investigation.<br><br>
Conclusion
We have defined the TM1-TM2 intramolecular interface in tetraspanin CD9, providing evidence for glycines (Gly25 and Gly32 in TM1, Gly67 and Gly74 in TM2) packing against apposing bulky aliphatic residues. Second, we mapped an intermolecular CD9 interface (involved in CD9 homodimer formation) to the vicinity of residues Leu14 and Phe17 in TM1 and Gly77, Gly80 and Ala81 in TM2. Finally, we provide preliminary evidence that TM3 and TM4 in CD9 may contribute to a second intermolecular interface. Key CD9 residues involved in intra- and intermolecular interactions are highly conserved throughout the tetraspanin family, thus suggesting that our findings will apply to most tetraspanins.<br><br>Methods
Materials
Cell culture reagents were from Invitrogen (Carlsbad, CA). 2-bromopalmitate was from Fluka (Milwaukee, WI), N-ethylmaleimide (NEM) and 1,10-phenanthroline were from Sigma-Aldrich (St. Louis, MO), and chemical cross-linkers dithio-bis-maleimidoethane (DTME) and 1,4-Bis-maleimidobutane (BMB) were purchased from Pierce Endogen (Rockford, IL). Triton X-100, protease inhibitor cocktail and FuGENE 6 transfection reagent were obtained from Roche (Indianapolis, IN). Restriction endonucleases and Pfu DNA polymerase were obtained New England Biolabs (Beverly, MA) and Stratagene (Carlsbad, CA), respectively. All other chemicals were purchased from Sigma-Aldrich or Fisher Scientific (Pittsburg, PA).<br><br>Sequence analysis
Tetraspanin sequences were obtained from SWISS-PROT and GenBank databases. Locus designations, accession numbers and the most commonly used protein names are summarized in Tables 1 and 2. TM segments were delineated by inspection of hydrophobicity profiles, using database annotations and previous analyses of TM sequences as a guide ([28], M. Hemler, unpublished), and aligned manually. Residue numbers in <span class="yellow">human</span> CD9 sequence are used a reference point throughout the study.<br><br>DNA cloning and mutagenesis
Sequence encoding CD9 protein was cloned into vector pcDNA3 (Invitrogen, Carlsbad, CA) and pEGFP-N1 (Clontech, Palo Alto, CA), for expression of untagged and C-terminally GFP-tagged CD9, respectively. pEGFP-N1 encoding CD9 TM(1+2) -GFP fusion protein was constructed by subcloning DNA for residues 1–83 of CD9 into HindIII and PstI sites of the vector; to introduce the PstI site, codon GTG for Val82 was changed to CTG (coding for Ala). In the resulting fusion protein, there is a 13-amino acid linker (with no cysteines) between CD9 and GFP. To minimize the low inherent ability of GFP to homodimerize, which could potentially influence the results of CD9 cross-linking, we used a monomeric GFP mutant, Leu221→Lys [34], for cross-linking experiments.
Mutations were introduced in full-length and TM(1+2) CD9 proteins by a PCR-based strategy using mutagenic primers and Pfu DNA polymerase. All mutations were confirmed by DNA sequencing.<br><br>Protein expression, microscopy, cysteine disulfide cross-linking and Western blotting
DNA constructs encoding TM(1+2)-GFP or full-length CD9 proteins were transfected into <span class="yellow">human</span> rhabdomyosarcoma RD cells using the FuGENE 6 reagent. Cells expressing GFP fusion proteins were analysed by fluorescence microscopy 18–28 hours post-transfection. Images were captured using Spot 1.4.0 camera (Diagnostic Instruments, Sterling Heights, MI) attached to Nikon Eclipse TE300 microscope.
For experiments involving cysteine-mediated cross-linking, cells were treated with 50 μM 2-BP starting 24–26 hours post-transfection and continuing for 16–18 hours. Cross-linking was carried out by incubating cells in HBSM buffer (25 mM Hepes-NaOH, pH 7.2, 150 mM NaCl, 2 mM MgCl2) containing either a) 0.6 mM CuSO4 and 1.8 mM 1,10-phenanthroline (CuP complex) or b) 0.2 mg/ml homobifunctional cysteine-reactive cross-linker (e.g. DTME), diluted from fresh 10 mg/ml solution in DMSO. After incubation for 10–15 minutes (with CuP) or 30–45 minutes (with cross-linker), cells were washed twice for 10 minutes with HBSM containing 10 mM NEM to block residual free cysteines. Cells were lysed in HBSM containing 1% Triton X-100, 0.1% SDS and a cocktail of protease inhibitors with 1 mM EDTA at 4°C for 45–60 minutes. Cell lysate was clarified by centrifugation at 14,000 × g for 15 minutes, an aliquot was removed, and proteins from it were precipitated by addition of trichloroacetic acid to 10% on ice followed by centrifugation at 14,000 × g for 10 minutes. After two washes with ice-cold acetone, protein pellet was solubilized in SDS-PAGE sample buffer without a reducing agent (50 mM Tris-HCl, pH 6.8, 1% SDS, 8% glycerol).
In some experiments, CD9 protein was immunoprecipitated using monoclonal antibody Alb6 (Immunotech, Marseille, France). Proteins were separated by SDS-PAGE and analyzed by Western blotting using monoclonal antibody JL-8 (Clontech) for GFP or Alb6 for CD9. Bands from X-ray films were quantitated using GeneTools™ software from Syngene (Frederick, MD).<br><br>Modeling of TM1-TM2 interaction
An atomic model of the CD9 TM1-TM2 dimer was constructed with a Monte Carlo-simulated annealing (MCSA) algorithm [69]. Two idealized α-helices corresponding to TM1 residues Tyr12 through Leu35 and TM2 residues Gly59 through Val82 were docked with six orthogonal parameters: three rigid body translations and three rotations. During each step of a MCSA cycle, there was an equal probability of changing either one parameter or all six parameters to random values. A conformation's energy was calculated in vacuo with the AMBER united-atom force field for van der Waals interactions [70]. The van der Waals term was modified as described by Kuhlman and Baker [71]. If a structure had favorable dimerization energy, the energies of select mutants were calculated. Structures were selected with a novel scoring function that maximizes the Boltzmann probability of dimerization for silent mutations while minimizing the probability for disruptive mutations. Asn18Ser, Gly77Leu, and Gly80Leu were scored as silent mutations while Gly25Leu, Gly32Leu, Gly67Leu, and Gly74Leu were considered to be disruptive. Each MCSA cycle consisted of 50,000 steps with an exponential temperature decay from 10,000 to 10 K.
Ten MCSA cycles through global sample space were used to restrict the search area. Parameters were restricted to ± 2 standard deviations from their mean value for structures within 10 kcal of the best structure. MCSA cycles were repeated as described above with additional optimization of χ values: rotamers at the protein-protein interface were optimized with Dead End Elimination [72], and χ values were further optimized with Monte Carlo. All MCSA cycles converged upon structures that were within a root mean squared deviation (RMSD) of 1.5 Å with the best structure, and structures that scored within 5 kcal of the best score had an RMSD of less than 0.5 Å with the best structure.<br><br>
List of abbreviations
2-BP, 2-bromopalmitate; CuP, Cu2+-phenanthroline; DTME, dithio-bis-maleimidoethane; LEL, large extracellular loop; NEM, N-ethylmaleimide; TM, transmembrane (domain).<br><br>Authors' contributions
OVK carried out sequence comparisons, mutational analysis and cross-linking experiments, and drafted the manuscript. DGM built the TM1-TM2 interaction model and contributed to the manuscript. WFD supervised DGM's work. MEH coordinated the whole study and prepared the final manuscript.<br><br>
<h3>pmcA140510</h3>Enp1, a <span class="yellow">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="yellow">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="yellow">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="yellow">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="yellow">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="yellow">human</span> Enp1, expressed in <span class="yellow">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="yellow">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="yellow">human</span> homolog of <span class="yellow">yeast</span> Enp1 was PCR amplified from a <span class="yellow">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="yellow">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="yellow">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="yellow">donkey</span>-anti-<span class="yellow">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="yellow">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="yellow">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="yellow">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="yellow">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="yellow">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="yellow">yeast</span> Enp1 (19). In contrast, we identified a <span class="yellow">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="yellow">human</span> bystin (19) showed that they were encoded by the same gene on <span class="yellow">human</span> chromosome 6. The reported sequence for <span class="yellow">human</span> bystin is a fragment of the 437 amino acid <span class="yellow">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="yellow">human</span> bystin, as confirmed by available <span class="yellow">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="yellow">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="blue">Drosophila melanogaster</span> and <span class="yellow">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="yellow">human</span> Enp1 homolog was cloned and expressed in <span class="yellow">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="yellow">human</span> Enp1 homolog did not complement a <span class="yellow">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="yellow">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="yellow">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="yellow">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="yellow">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="yellow">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="yellow">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="yellow">human</span> homolog of Enp1 did not complement a <span class="yellow">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="yellow">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="blue">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA280693</h3>p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells and its expression is induced through pathways involved in growth inhibition and repressed by factors promoting cell growth
Abstract
Background
p8 is a stress-induced protein with multiple functions and biochemically related to the architectural factor HMG-I/Y. We analyzed the expression and function of p8 in pancreatic cancer-derived cells.<br><br>Methods
Expression of p8 was silenced in the <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 by infection with a retrovirus expressing p8 RNA in the antisense orientation. Cell growth was measured in control and p8-silenced cells. Influence on p8 expression of the induction of intracellular pathways promoting cellular growth or growth arrest was monitored.<br><br>Results
p8-silenced cells grew more rapidly than control cells transfected with the empty retrovirus. Activation of the Ras→Raf→MEK→ERK and JNK intracellular pathways down-regulated p8 expression. In addition, the MEK1/2 inhibitor U0126 and the JNK inhibitor SP600125 up-regulates expression of p8. Conversely, p38 or TGFβ-1 induced p8 expression whereas the specific p38 inhibitor SB203580 down-regulated p8 expression. Finally, TGFβ-1 induction was in part mediated through p38.<br><br>Conclusions
p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells. p8 expression is induced through pathways involved in growth inhibition and repressed by factors that promote cell growth. These results suggest that p8 belongs to a pathway regulating the growth of pancreatic cancer cells.<br><br><br><br>Background
While studying the molecular response of the injured pancreas, we identified a new gene, called p8, whose expression is strongly induced during the acute phase of pancreatitis [1]. Further experiments have shown that p8 mRNA is activated in almost all cells in response to several stresses [2], including minimal stresses such as after routine change of the culture medium in the absence of any added substance [3], indicating that p8 is a ubiquitous protein induced by cellular stress. The p8 gene was cloned in <span class="yellow">human</span>, <span class="yellow">rat</span>, <span class="yellow">mouse</span>, and <span class="yellow">Xenopus laevis</span> [1,4-6], conceptually translated from the <span class="blue">Drosophila melanogaster</span> genome or deduced from EST libraries (<span class="yellow">Bos taurus</span>, <span class="yellow">Xenopus tropicalis</span>, <span class="yellow">Zebrafish</span>, <span class="yellow">Orzzias latipes</span>, <span class="yellow">Bombyx mori</span> and <span class="yellow">Paralichthys olivaceous</span>). The overall degree of homology with <span class="yellow">human</span> p8 ranged from 81 to 40%. Secondary structure prediction methods indicated that within the homologous region of the eleven proteins, there is a basic Helix-Loop-Helix secondary structure motif, characteristic of some classes of transcription factors [1]. Even though a small protein such as p8 would not need a nuclear localization signal (NLS) to be transported to the nucleus, a clear NLS can be predicted for the eleven proteins comprising a bipartite domain of positively charged aminoacids. In addition, a nuclear/cytoplasmic location has been demonstrated for <span class="yellow">human</span> p8 upon overexpression of the recombinant protein and immunohistochemistry [4], and for recombinant <span class="yellow">Xenopus laevis</span> p8 fused to green fluorescent protein [6]. Homology searching in databases did not reveal significant similarity of p8 with other proteins of known function. However, biochemical properties of the mammalian p8 proteins are shared by some high mobility group proteins (HMG) [7], particularly by the HMG-I/Y family. The overall identity of <span class="yellow">human</span> p8 with <span class="yellow">human</span> HMG-I/Y is only about 35%, but the molecular mass, isoelectric point, hydrophilicity plot, the resistance to denaturation after heating at 100°C and the charge separation are very similar [8]. The p8 protein seems to bind DNA weakly, as shown by electrophoretic mobility shift assay, without preference for DNA sequences. Finally, <span class="yellow">human</span> p8 has also been shown to be a substrate for protein kinase A in vitro and phosphorylated p8 has a higher content of secondary structure and binding to DNA is highly increased [8]. An architectural role in transcription has been proposed for this protein, in analogy with the HMG-I/Y proteins, and a recent work seems to confirm this hypothesis [9].
Functions of p8 appear to be multiple and complex. For example, p8 mRNA expression was strongly induced in 3T3 cells upon TGFβ-1 treatment which in turn enhances the Smad-transactivating function responsible for TGFβ-1 activity [10]. We also found that p8 is involved in cell cycle regulation since p8-deficient embryonic fibroblasts grew more rapidly and incorporated more [3H] thymidine and BrdU than p8-expressing cells [11]. Moreover, expression of p8 in breast cancer-derived cells seems to mediate the inhibition of cell growth induced by 1,25-Dihydroxyvitamin D3 [12]. On the contrary, we also reported that p8 may promote cell growth when overexpressed in Cos-7, AR42J and HeLa cells [1,4]. In addition, p8 seems to be involved in other intracellular functions such as apoptosis since p8-expressing fibroblasts are more sensitive than p8-deficient fibroblasts to the apoptosis induced by DNA damage. Also, p8 is required for endothelin-induced mesangial cell hypertrophy in diabetic kidney, in a mechanism involving ERK, JNK and PI3 kinase [13]. p8 seems to play a functional role in the initiation of LHβ gene expression during embryonic cell differentiation [14]. Moreover, the <span class="blue">Drosophila melanogaster</span> p8 homologue is involved in response to starvation and might be activated to stop cell growth in case of nutrient deprivation [15]. Finally, a particularly attractive role in tumour progression was recently proposed for p8 [16]. Fibroblasts obtained from p8-expressing or p8-deficient animals were transformed with a retroviral vector expressing both the rasV12 mutated protein and the E1A adenoviral oncogene. In soft-agar assays, transformed p8-expressing cells formed colonies at high frequency, as expected, but p8-deficient transformed fibroblasts were unable to form colonies. Similarly, transformed p8-expressing cells produced tumours in all athymic <span class="yellow">nude mice</span> when injected subcutaneously or intraperitoneally, whereas transformed p8-deficient fibroblasts did not. On the other hand, studies by another laboratory revealed that expression of the Com1 protein [17], which is identical to <span class="yellow">human</span> p8, mediates the growth of tumour cells after metastatic establishment in a secondary organ, indicating that activated expression of Com1/p8 in metastatic cells is required for tumour progression. These results strongly suggest that p8 is involved in the cellular pathway(s) required for tumour progression and metastasis.
Our aim is to check the relevance of p8 to cancer progression in <span class="yellow">human</span>. As a first step, we investigated in the present study the function of p8 in two cell lines derived from <span class="yellow">human</span> pancreatic cancer. We observed that inhibition of p8 expression increased the cells growth rate. In addition, activations of the Ras→Raf→MEK→ERK and JNK intracellular pathways, which promote the growth of pancreatic cells, down-regulated p8 expression, whereas activation of p38 or TGFβ-1, which inhibit cell growth, induced its expression. It was concluded that i/ p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cell lines, ii/ p8 expression is induced through pathways involved in growth inhibition and, conversely, repressed by factors that promote cell growth.<br><br>Results
p8 is silenced in pancreatic cancer cells by infection with a retrovirus expressing p8 RNA in the antisense orientation
Panc-1 and BxPc-3 pancreatic cells were chosen for this study because, on the one hand, both cells express higher level of p8 (Figure 1) and, on the other hand, because Panc-1 is wild-type for Smad4/DPC4 and mutated for K-ras, while BxPc-3 is Smad4/DPC4 mutated and K-ras wild-type [18,19], therefore representing different mechanisms of transformation and different genetic backgrounds. K-ras and Smad4/DPC4 mutations are the major mechanisms involved in pancreatic cancer development. We inhibited p8 expression in both Panc-1 and BxPc-3 pancreatic cells by infecting cells with a retrovirus expressing the p8 asRNA (antisense RNA) and carrying the puromycin resistance. The antibiotic-selected cells were analyzed by Western blotting to evaluate the intracellular amounts of p8 protein. As shown in Figure 2, the p8 protein was clearly visible in both Panc-1 and BxPc-3 pancreatic cells infected with the empty retrovirus but almost undetectable in cells infected with the retrovirus encoding the p8 asRNA showed, indicating that our anti-sense strategy is efficient to silence p8 gene expression in pancreatic cancer cells. Preliminary studies had been conducted to select the best strategy to inhibit p8 expression. We compared the efficacy of the stable transfection of a siRNA, using a retroviral expression vector to the asRNA strategy described above. In our hands, the antisense strategy worked best, as judged from Western blot assessment of p8 protein expression (data not shown).<br><br>p8-silenced pancreatic cells grow more rapidly
We compared in the two cell lines the influence on growth parameters of blocking p8 expression with the p8 asRNA. Figure 3 shows that both Panc-1 and BxPc-3 cells in which p8 has been silenced grew more rapidly than cells infected with the empty vector suggesting that inhibition by p8 of pancreatic cancer cell growth is independent from the mechanism of transformation and genetic background.<br><br>Serum-stimulated cellular growth down-regulates p8 expression
Fetal <span class="yellow">calf</span> serum, which contains a complex mix of growth factors, can be used as inductor of cell growth. As shown in Figure 4 expression of p8 mRNA was down-regulated in both Panc-1 and BxPc-3 when the cells were shifted from culture media containing 0.1% fetal <span class="yellow">calf</span> serum to media containing 10% FCS. p8 protein showed a similar behavior. These results show that p8 expression is down-regulated in growing pancreatic cells.<br><br>The Ras→Raf→MEK→ERK pathway down-regulates p8 expression in pancreatic cancer cells
Most <span class="yellow">human</span> pancreatic cancers harbor mutations in the K-ras oncogene, which happens relatively early in pancreatic tumorigenesis [20]. The oncogenic mutation of the K-ras gene stabilizes the Ras protein in a GTP-bound form, which is constitutively active and make the cells grow more rapidly. Contrary to the activated Ras protein, p8 inhibits cell growth (Figure 3). We looked whether the Ras→Raf→MEK→ERK pathway was also involved in the regulation of p8 expression, and which step(s) were critical. Figure 5 shown that expression of a mutated form of the Ras protein (rasV12) in BxPc-3 cells, which are wild-type for ras, resulted in decreased p8 mRNA concentration and protein level suggesting that the activated ras inhibits p8 expression. Figure 6 shows that overexpression of Raf, but not of Raf301 (a negative mutant of Raf), and of ERK also inhibited the expression of the p8-CAT construct in Panc-1 as well as in BxPc-3. Finally, the MEK1/2 specific inhibitor U0126 [21] activated p8 mRNA expression in pancreatic cells whether they carry mutated ras (Panc-1) or wild-type (BxPc-3). Similar results were observed when expression of the p8 protein was monitored by Western blotting (Figure 7).<br><br>Activation of the JNK pathway down-regulates p8 expression in pancreatic cancer cells
c-Jun NH2-terminal kinase (JNK) is another major MAPK pathway which converts extracellular signals into expression of specific target genes through phosphorylation and activation of transcription factors. JNK activation has been implicated in various, often opposite cellular responses, such as cell proliferation, transformation and apoptosis. As shown in Figure 8, overexpression of JNK down-regulates the gene reporter activity of the p8-CAT construct in Panc-1 cells. Similar results were found in BxPc-3 cells. Treatment of these cells with the JNK specific inhibitor SP600125 [22] up-regulates expression of the p8 mRNA and p8 protein (Figure 9). These results show that the JNK pathway is involved in the regulation of p8 expression.<br><br>The p38 pathway up-regulates p8 expression in pancreatic cancer cells
The p38 signal transduction pathway also plays an essential role in regulating several cell functions including growth, response to inflammation, differentiation and apoptosis. In fact, in pancreatic cancer cells, p38 is a strong inhibitor of proliferation [23] contrary to the Ras→Raf→MEK→ERK and JNK pathways. We therefore analyzed the putative role of the p38 pathway in regulating p8 expression in pancreatic cancer cells. Figure 10 shows that over-expression of the plasmid encoding p38 significantly increases p8-CAT activity in Panc-1 as well as in BxPc-3 cells. Then, cells were treated with SB203580, a specific inhibitor of p38 [24], and p8 expression was measured. p8 mRNA as well as the encoded protein were down-regulated after inhibition of the p38 activity (Figure 11). These results indicate that the p38 pathway is a positive regulator of p8 expression in pancreatic cancer cells.<br><br>TGFβ-1 up-regulates p8 expression in pancreatic cancer cells
The most prominent biological activity of TGFβ-1 is its potent inhibition of cell growth in a wide variety of cell types including pancreatic cells. TGFβ-1 signals are sent through two types of transmembrane serine/threonine kinase receptors. In fact, TGFβ-1 binds and brings together the type I and type II receptors. In the resulting complex, the constitutively active TGFβ-1 type II receptor phosphorylates the type I receptor, which then plays a major role in transducing the signal to downstream components to affect gene expression through phosphorylation of SMAD proteins. Phosphorylated receptor-regulated SMADs then form heteromeric complexes with the common partner SMAD4. These heteromeric complexes then move to the nucleus, where SMAD4 will bind DNA and contribute to transcriptional activation. In general, pancreatic cancer cells present with defects in TGFβ-1 signaling and are resistant to TGFβ-1-mediated growth suppression. Since TGFβ-1 and p8 are inhibitors of pancreatic cell growth we analyzed whether p8 could mediate, at least in part, the effect of TGFβ-1. First, we found that treatment of Panc-1 cells with TGFβ-1 increased p8 mRNA levels and p8 protein as judged by Western blot (Figure 12). Then, to confirm that overexpression is regulated at the transcriptional level, we analyzed the effect of some constructs expressing constitutively activated type I TGFβ receptor, dominant negative type II TGFβ receptor, a dominant negative of Smad4 and the wild-type Smad4 on the p8-CAT activity. As expected, the constitutively activated type I TGFβ receptor but not the dominant negative type II TGFβ receptor increased CAT activity. Also, expression of the Smad4, contrary to that of the negative mutant, induced p8 transcription (Figure 13). Together, these results indicate that p8 is positively regulated by TGFβ-1.
Beside the Smad proteins, TGFβ-1 also activates the p38 MAPK pathway in pancreas-derived cells, which may play an important role in TGFβ-1 induced genes [25]. Therefore, we analyzed the p38-dependent effect of TGFβ-1 on p8 transcription. As shown in Figure 13, inhibition of p38 activity with the SB203580 specific inhibitor decreased about 40% the activity of TGFβ-1 on the p8 promoter indicating that the effect of TGFβ-1 on p8 promoter is mediated by both p38-dependent and p38-independent pathways.<br><br>
Discussion
Pancreatic adenocarcinoma is the fourth leading cause of death from malignant diseases [26]. The aggressive nature of the neoplasia, the lack of early detection, and the limited response to treatments such as chemotherapy and radiotherapy contribute to the high mortality rate of the disease. Therefore, a better understanding of the molecular mechanism leading to pancreatic cancer remains a major goal because it may help proposing strategies for earlier diagnosis and better treatment. The most commonly altered genes in pancreatic adenocarcinoma are K-ras (75 to 100%), p16INK4a (95%), p53 (50 to 75%) and DPC4 (50%) [27-31]. Whereas K-ras is a proto-oncogene all the others are tumour suppressor genes. Additional genes have been found altered at lower frequency. Panc-1 and BxPc-3 pancreatic cells were chosen for this study because they both express high levels of p8 (Figure 1) and because they present with different mechanisms of transformation and genetic backgrounds, Panc-1 being wild-type for Smad4/DPC4 but mutated for K-ras and BxPc-3 mutated for Smad4/DPC4 and wild-type for K-ras [18,19]. This work presents evidences that p8 inhibits the growth rate of pancreatic cancer-derived cells and that the intracellular pathways promoting cell growth down-regulate p8 expression whereas those promoting growth arrest up-regulate its expression. Together, these results suggest that p8 is downstream of some cell growth regulators and therefore regulation of p8 expression or its activity could be used as a target for treating pancreatic cancer.
Silencing p8 expression was able to strongly promote cell growth in both cell types, Panc-1 and BxPc-3, suggesting that p8 may act downstream of the ras- or Smad4/DPC4-dependent ways. Also, we found that stimulating cell growth by the complex combination of growth factors contained in fetal <span class="yellow">calf</span> serum down-regulated expression of p8 whereas, on the contrary, treating the cells with TGFβ-1, which promotes cell cycle arrest, stimulates p8 expression. Therefore, p8 gene expression seems to be regulated in opposite directions by mechanisms promoting cell growth or cell cycle arrest. It is interesting to note that while p8 expression is under the control of cell growth regulatory pathways such as Ras→Raf→MEK→ERK, JNK, p38 and TGFβ-1, p8 can affect cell cycle progression, suggesting that p8 is a target for factors regulating pancreatic cell growth.
A mechanism by which p8 could regulate cell cycle progression in embryonic fibroblasts was previously proposed [11]. In fact, p8 seems to take action upstream from cyclin-dependent kinases because the intracellular levels and activities of Cdk2 and Cdk4 are decreased when p8 is expressed. Concomitantly, the cyclin-dependent kinase inhibitor p27 is expressed at a low level in p8-deficient cells which may explain the increased activity of Cdk2 and Cdk4. The mechanism by which p8 regulates the intracellular level of those proteins remains to be determined. However, because p8 is a transcriptional cofactor, it is possible that regulation of expression of these molecules takes place, at least in part, at the transcription level.
Interestingly, expression of p8 mRNA seems to be regulated in a cell type- and stimulus-specific manner since, for example, p38 can induce p8 expression in response to stress in fibroblasts [3] but not in renal mesangial cells treated with endothelin [13]. In pancreatic cancer-derived cells p38 seems to play a major role since it is involved in p8 activation as judged by transient transfection assays and using a specific p38 inhibitor (Figures 10 and 11). In addition, p38 is also involved in TGFβ-1-induced p8 expression because about 40% of the TGFβ-1 effect was abolished when p38 activity was specifically blocked (Figure 13). On the other hand, ERK and JNK are inducers of p8 expression in mesangial cells treated with endothelin, but not involved in the activation of p8 in response to stress in fibroblasts [3], and even repressors in pancreatic cells (Figures 5, 6, 7, 8 and 9). Finally, PI3 kinase is an inducer of p8 expression in both endothelin-mediated p8 activation in mesangial cells [13] and pancreatic cells (data not shown).
Based on these observations, overexpression of p8 could be considered a possible goal for treating pancreatic tumours, in order to limit their growth. However, we previously reported that p8 repression would prevent rasV12/E1A transformed fibroblasts from evolving as tumours in <span class="yellow">nude mice</span> [16]. This apparent contradiction needs to be resolved before considering p8 as a target for treating cancer progression.<br><br>Conclusions
In conclusion (see Figure 14), we report in this paper that inhibition of p8 expression by an anti-sense strategy increases the growth rate of both Panc-1 and BxPc-3 pancreatic cancer-derived cells. Moreover, ERK- and JNK-mediated pathways down-regulate p8 expression, whereas p38 and TGFβ-1 pathways induce p8 expression. Also, cell growth triggered by expression of a RAS mutated protein or by 10% fetal <span class="yellow">calf</span> serum induces down-regulation of p8 expression. Together, these results indicate that p8 is an intracellular cell growth inhibitor and that it is oppositely regulated by growth-promoting or growth-inhibiting factors in pancreatic cancer-derived cells.<br><br>Material and Methods
Cell lines and cell culture conditions
The <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 were a kind gift of Dr C. Susini (INSERM U.531, Toulouse) and A. Hajri (IRCAD, Strasbourg) respectively. Panc-1 cells were grown in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal <span class="yellow">calf</span> serum, 2 mM L-glutamine, 100 IU/ml penicilin G and 100 μg/ml streptomycin. BxPc-3 were cultivated in RPMI 1640 medium in the presence of 2 mM L-glutamine, 4.5 g/L glucose, 10 mM Hepes, 1.0 mM sodium pyruvate, 10% fetal <span class="yellow">calf</span> serum and 100 IU/ml penicilin G and 100 μg/ml streptomycin. <span class="yellow">Human</span> recombinant TGFβ-1 was obtained from Sigma, and specific SB203580, U0126 and SP600125 inhibitors were from Calbiochem and utilized at 10 μM.<br><br>Expression plasmids
Expression plamids encoding p38 (pCEFL HA p38), Erk2 (pcDNAIII HA ERK2), JNK (pcDNAIIIB HA JNK), the wild-type Raf (pcDNA RAF BXB) and the Raf dominant negative (pcDNA RAF 301 K375W) were obtained from O Coso (University of Buenos Aires). Plamids encoding the constitutively activated type I TGFβ receptor (RI ACT), the dominant negative type II TGFβ receptor (RII DN) and the Smad4 dominant negative (DPC4 1–514 a.a.) were obtained from R Urrutia (Mayo Clinic, Rochester) and the wild type Smad4 was from C Heldin (Ludwig Institute, Uppsala).<br><br>Pancreatic p8-deficient cells
To silence p8 expression in pancreatic cells, we infected these cells with a retrovirus expressing <span class="yellow">human</span> p8 in the antisense orientation. The retroviral vector was constructed as follows: <span class="yellow">human</span> p8 cDNA was subcloned in HindIII and XhoI restriction sites of the pLPC plasmid (obtained from S. Lowe) in the antisense orientation. Amphotrope <span class="yellow">human</span> p8 expressing retrovirus was then generated by transient transfection using Phoenix amphotrope packaging cells. Viral supernatant was used to infect Panc-1 and BxPc-3 pancreatic cells and the population of p8-silenced cells was isolated by selection in presence of puromycin (1 μg/ml). As control, cells were infected with the pLPC empty vector.<br><br>p8 expression in arrested and growing cells
One million of Panc-1 or BxPc-3 cells were cultivated on 10-cm Petri plates in standard conditions (with 10% FCS). After 48 h, culture media were changed for fresh media with FCS restricted to 0.1%, in order to stop growth. After 24 hours of growth arrest, culture medium was replaced either by medium with 10% fetal <span class="yellow">calf</span> serum to resume cell growth or, as control, by medium with 0.1% fetal <span class="yellow">calf</span> serum. Twenty four hours later cells were recovered and RNA and protein extracted.<br><br>p8 expression in TGFβ-1-treated Panc-1 cells
One million of Panc-1 cells were cultivated in 10-cm culture dishes for 48 hours under standard conditions before TGFβ-1 treatment. <span class="yellow">Human</span> recombinant TGFβ-1 (5 ng/ml) was added to cells, without changing the culture medium, and cells were collected 12 hours later for RNA and protein preparation.<br><br>BxPc-3 rasV12-expressing cells
pLPC-rasV12 and pLPC plasmids were obtained from S. Lowe. Phoenix amphotrope packaging cells (106) were plated in a 6-well plate, incubated for 24 hours, then transfected with PEI with 5 μg of retroviral plasmid. After 48 hours, the medium containing virus was filtered (0.45 μm filter, Millipore) to obtain the viral supernatant. Target BxPc-3 were plated at 2 × 105 cells per 35-mm dish and incubated overnight. For infections, the culture medium was replaced by an appropriate mix of the viral supernatant and culture medium (V/V), supplemented with 4 μg/ml polybrene (Sigma), and cells were incubated at 37°C. BxPc-3 rasV12-expressing cells were selected with puromycin (1 μg/ml). Cells infected with the pLPC empty vector were used as control.<br><br>Western-blot analyses
One hundred μg of total protein extracted from cells was separated with standard procedures on 15.0% SDS-PAGE using the Mini Protean System (Bio-Rad) and transferred to a nitrocellulose membrane (Sigma). The intracellular level of p8 was estimated by Western blot using a polyclonal antibody (1:1000) raised against <span class="yellow">human</span> p8 [4].<br><br>Growth curves
One hundred thousand cells per well were plated in a series of 35-mm culture dishes. The cell number was estimated daily in triplicate, during 1 to 5 days, in a haemocytometer. Within experiments, each point was determined at least two times.<br><br>Cell transfection and gene reporter assays
Panc-1 and BxPc-3 (105) were cultivated in 30 mm diameter culture dishes for 24 hours then transiently transfected with 0.5 μg of p8-CAT reporter plasmid and 0.5 μg of pCMV/βgal plasmid (to control transfection efficiency) using the Fugene reagent in accordance with the manufacturer's protocol (Roche Molecular Biochemicals). The p8-CAT plasmid is the previously reported p-1471/+37p8-CAT promoter construct [5]. Reporter activities were measured as previously described [5]. Briefly, cell extracts were prepared with the reporter lysis buffer (Promega) 24 hours after transfection and CAT activity was determined by the phase extraction procedure [32] and β-galactosidase assay was performed essentially as described in Sambrook et al. [33]. CAT activity was normalized to β-galactosidase activity. Experiments were carried out in triplicate and repeated at least two times. Expression plasmids (0.5 μg) were co-transfected with p8-CAT and pCMV/βgal plasmids as indicated.<br><br>RT-PCR analysis
RNA was extracted using the Trizol (Life Technologies) procedure. Total RNA (1 μg) was analyzed by RT-PCR with the SuperScript™ One-step RT-PCR System and the Platinum Taq kit (Life Technologies). RT-PCR was performed using different numbers of cycles to verify that the conditions chosen were within the linear range. The mRNA coding for p8 was specifically amplified with sense (5' GAAGAGAGGCAGGGAAGACA 3') and antisense (5' CTGCCGTGCGTGTCTATTTA 3') primers, in positions 72 and 643 of the cDNA (accession # NM_012385), respectively. As control, the transcript coding for the ribosomal protein RL3 was specifically amplified for 22 cycles with sense (5'GAAAGAAGTCGTGGAGGCTG3') and antisense (5'ATCTCATCCTGCCCAAACAC3') primers, in positions 216 and 637 of the cDNA, respectively.<br><br>
Author's contributions
CM prepared cells and retrovirus, carried out RNA purification, RT-PCR, Western blots, and cell growth experiments, NL carried out CAT assays, SV participated in the design of the study and analysis of data, JLI participated in the analysis of data and wrote the manuscript. All authors read and approved the final manuscript.<br><br>
</body></html>