
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>E. coli</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1190194</h3>Structural organization and interactions of transmembrane domains in tetraspanin proteins
Abstract
Background
Proteins of the tetraspanin family contain four transmembrane domains (TM1-4) linked by two extracellular loops and a short intracellular loop, and have short intracellular N- and C-termini. While structure and function analysis of the larger extracellular loop has been performed, the organization and role of transmembrane domains have not been systematically assessed.<br><br>Results
Among 28 <span class="yellow">human</span> tetraspanin proteins, the TM1-3 sequences display a distinct heptad repeat motif (abcdefg)n. In TM1, position a is occupied by structurally conserved bulky residues and position d contains highly conserved Asn and Gly residues. In TM2, position a is occupied by conserved small residues (Gly/Ala/Thr), and position d has a conserved Gly and two bulky aliphatic residues. In TM3, three a positions of the heptad repeat are filled by two leucines and a glutamate/glutamine residue, and two d positions are occupied by either Phe/Tyr or Val/Ile/Leu residues. No heptad motif is apparent in TM4 sequences. Mutations of conserved glycines in <span class="yellow">human</span> CD9 (Gly25 and Gly32 in TM1; Gly67 and Gly74 in TM2) caused aggregation of mutant proteins inside the cell. Modeling of the TM1-TM2 interface in CD9, using a novel algorithm, predicts tight packing of conserved bulky residues against conserved Gly residues along the two helices. The homodimeric interface of CD9 was mapped, by disulfide cross-linking of single-cysteine mutants, to the vicinity of residues Leu14 and Phe17 in TM1 (positions g and c) and Gly77, Gly80 and Ala81 in TM2 (positions d, g and a, respectively). Mutations of a and d residues in both TM1 and TM2 (Gly25, Gly32, Gly67 and Gly74), involved in intramolecular TM1-TM2 interaction, also strongly diminished intermolecular interaction, as assessed by cross-linking of Cys80.<br><br>Conclusion
Our results suggest that tetraspanin intra- and intermolecular interactions are mediated by conserved residues in adjacent, but distinct regions of TM1 and TM2. A key structural element that defines TM1-TM2 interaction in tetraspanins is the specific packing of bulky residues against small residues.<br><br><br><br>Background
Tetraspanins constitute a large family of integral membrane proteins, characteristically containing 4, 6 or 8 conserved cysteine residues in the large extracellular loop (including the CCG and PxxCC motifs), which form disulfide bonds, and several conserved polar residues in the intracellular loop and transmembrane regions [1,2]. There are 32 putative tetraspanin family members in mammals, 37 in <span class="yellow">Drosophila melanogaster</span> and 20 in <span class="yellow">Caenorhabditis elegans</span>. Tetraspanins play diverse roles in cell adhesion, migration and fusion processes, cellular activation and signaling (reviewed in refs. [2-4]). Mammalian tetraspanins such as CD9, CD63, CD81, CD82, CD151, rds/peripherin, and uroplakins Ia and Ib have been most extensively studied, with <span class="yellow">mouse</span> knock-out models available for CD9 [5-7], CD81 [8,9], CD151 [10] and a few others. However, the majority of tetraspanins are characterized very little, if at all, at genetic, biochemical or structural levels.
The large extracellular loop (LEL) of tetraspanins has received most attention, since it contains functionally important sites. Sequence QRD (194–196) in CD151 is important for association with integrins, which has functional consequences for integrin-dependent cell spreading and multicellular cable formation [11]. A site in the LEL of CD9, SFQ (residues 173–175), is essential for CD9 function in sperm-egg fusion [12]. The crystal structure of tetraspanin CD81 LEL revealed five α-helixes, A-E [13]. Helices A, B and E form a relatively conserved region in tetraspanins, whereas the region between helices B and E is the most variable [14]. Interestingly, the variable region contains most of the functionally important sites involved in tetraspanin protein-protein interactions.
A remarkable biochemical property of tetraspanin molecules is their ability to associate with a large number of other transmembrane proteins, including integrins, membrane-associated growth factors and receptors, MHC class II molecules, Ig superfamily proteins, and each other [2,3,15]. Several of these lateral associations of tetraspanins are detected in "mild" detergents (Brij series, CHAPS), but are disrupted by "strong" detergents such as Triton X-100 or SDS. Multiprotein complexes of tetraspanins and associated molecules, also called the "tetraspanin web" [16], may represent a distinct tetraspanin-enriched membrane microdomain [17,18]. The formation of this microdomain is influenced by palmitoylation of several conserved juxtamembrane cysteine residues in tetraspanins [19-21].
The transmembrane domains, encompassing nearly half of a tetraspanin protein, are the most conserved part of the molecule (Stipp et al. [1] and this study). However, very little functional information is available on these domains. The differential detergent sensitivity of tetraspanin-tetraspanin associations suggests that hydrophobic interactions between TM helices may play a role. Indeed, when the large extracellular loop (LEL) of CD151 is deleted, the molecule is still able to associate with other tetraspanins [22]. Thus, TM domains are strong candidates for mediating tetraspanin-tetraspanin interactions.
The importance of TM domain interactions in intramolecular organization was demonstrated in a study showing that CD82 fragment TM2-4, lacking TM1, was retained in the endoplasmic reticulum, but was transported to the cell surface upon co-expression of TM1 [23]. This in vivo reconstitution experiment demonstrated a strong interaction between TM1 and the rest of the molecule. Expression of a truncated CD9 molecule (TM3-LEL-TM4) results in intracellular accumulation of the protein and significant misfolding of the LEL, as judged by inappropriate disulfide formation and diminished antibody reactivity (our unpublished data). Similarly, a CD9 epitope in the LEL is lost in molecules lacking either TM2+TM3 or just TM4 [24]. Thus, TM domain interactions and packing are crucial for proper folding, stability and transport of tetraspanin molecules.
In a previous study, we showed that covalent cross-linking of membrane-proximal cysteine residues can be used as a tool for detection of tetraspanin-tetraspanin associations [25]. Inhibition of cysteine palmitoylation by 2-bromopalmitate (2-BP) made cysteines available for cross-linking and enabled demonstration of specific tetraspanin homodimerization and low levels of heterodimerization. We concluded that tetraspanin homodimers, formed in the Golgi, may be a fundamental structural unit within tetraspanin microdomains.
In this study, we carried out detailed sequence analysis of <span class="yellow">human</span> tetraspanin TM domains. We show that a heptad repeat containing conserved glycine, asparagine and large hydrophobic residues occurs in TM1 and TM2 domains, and predict tight intramolecular association of these two domains by packing of the large residues against the small residues. Moreover, by using cysteine cross-linking we map a dimerization interface in the <span class="yellow">human</span> CD9 protein, and show that conserved heptad motif glycine residues are also important for intermolecular CD9 associations.<br><br>Results
Sequence analysis of tetraspanin transmembrane domains: presence of the heptad repeat motif
We focused our attention on 28 <span class="yellow">human</span> tetraspanins identified from the SWISS-PROT and GenBank databases. All tetraspanins have in common four hydrophobic stretches (TM domains) of 20–25 residues, and contain highly conserved residues in the second extracellular loop, in particular the Cys-Cys-Gly (CCG) motif. Detailed analysis of the large extracellular loop sequences [14], and dendrograms based on full-length alignment can be found in earlier studies [26,27]. The length of each transmembrane domain was established based on previous sequence analysis of tetraspanin sequences [27,28], and on annotations to the database entries. Manual adjustments based on sequence homology and hydrophobicity profiles were done to fully delineate the TM domains. The resulting lengths of TM domains were: TM1 – 23 residues; TM2 – 21 residues; TM3 – 25 residues; TM4 – 25 residues. Two more residues could be added onto the N-terminal part of TM2; however, relatively small sequence conservation of these residues among tetraspanins and occurrence of polar/charged side chains in some tetraspanins precluded us from doing so for the global alignment.
Figures 1 and 2 show a multiple sequence alignment of four TM domains of 28 <span class="yellow">human</span> tetraspanins. For each position within the domains, consensus residues were determined and classified (with individual color code) in 4 categories: 1) large hydrophobic residues (including Val, Met, Leu, Ile, Phe, Tyr, Trp), 2) small residues (Gly, Ala, Ser and Thr), 3) Cys, and 4) Asn. When more than two types of residues occupied a given position in a TM, a dual-color pattern that reflected the prevalence of the particular residue type was used (Figure 1). Cysteine residues were shown separately due to their importance as palmitoylation target sites. The highly conserved asparagine residue in TM1 was considered separately. No proline residues are found in TM domains 1–3 of <span class="yellow">human</span> tetraspanins.
An inspection of the multiple sequence alignment reveals a repeating heptad amino acid pattern, (abcdefg)n, in TM1, 2 and 3 (Figure 1, 2). Heptad repeats promote helical coiled coil interactions in multiple soluble and membrane-spanning proteins [29-31]. In the heptad repeat, hydrophobic residues in positions a and d are of special importance, as they directly mediate interhelical contacts by creating a tight knobs-into-holes packing in the coiled coil structure [32]. For instance, in the leucine zipper of the <span class="yellow">yeast</span> transcription factor GCN4, positions a and d contain Val and Leu residues, respectively, with an Asn residue in a single a position forming a hydrogen bond across the GCN4 dimer interface [33].
In TM1 of tetraspanins, highly conserved Asn, Gly and Gly residues (numbers 18, 25 and 32 in the CD9 sequence) appear at d positions of the heptad repeats, and residues 14, 21 and 28 are at a positions (Figure 1). In TM2, residues 67, 74 and 81 (consensus Gly, Gly and Ala, respectively) occupy a positions, whereas residues 63, 70 and 77 are at d positions. Another highly conserved glycine, Gly80, occupies the 3rd g position in TM2. In TM3, the conserved pattern consists of two leucine residues (Leu89 and Leu96) and a glutamate/glutamine residue (Glu/Gln103) in a positions (Figure 2). Two d positions are also conserved – Phe/Tyr92 and Ile/Val/Leu99. TM4 lacks a conserved heptad pattern and has only a single conserved position, Glu/Gln209 (with four exceptions). These features of TM1-4 of tetraspanins are displayed on helical wheel diagrams (Figure 3).<br><br>Analysis of TM1 sequences
The conserved Asn-Gly-Gly motif, occupying designated d positions of the heptad repeat, is the most prominent structural feature of TM1. We also compared sequences of CD9 orthologs from 10 different organisms (the most available for any tetraspanin) to gain further insight into conservation and variability of the TM1 sequence. As shown in Figure 4, positions a, d and g in TM1 are among the most conserved (0, 1 and 1 substitution, respectively), while interspecies variability tends to occur in other positions: b (5 substitutions), c (4 substitutions), e (4 substitutions) and f (4 substitutions). Thus, the positions typically involved in coiled coil interactions (a and d) are the most conserved.
When residues of TM1 are plotted as a helical wheel, additional structural features are revealed (Figure 3). There are highly conserved aliphatic and aromatic residues in the first three a positions of the heptad motif (Phe15, Trp22 and Leu29 in CD9), as well as in g positions (Leu14, Phe21, Val28 in CD9). The "ridges" formed by these bulky residues are flanking the "groove"-forming Gly residues of the Asn-Gly-Gly position d motif. In contrast, b, c, e and f positions show an overall higher variability among tetraspanins, as also seen in the comparison of CD9 orthologs described above.<br><br>Analysis of TM2 sequences
A landmark feature of TM2 in tetraspanins is the presence of highly conserved glycine residues (Gly67, 74, 77 and 80 in CD9, Figure 1). Other substitutions at these positions are almost exclusively small residues, such as Ala or Ser. In addition, Ala, Ser or Thr occupy position 81. This residue, together with Gly67 and Gly74, forms face a of the helix. Residue Gly77 (position d) is preceded by conserved, chiefly large hydrophobic residues on the same helical face (Leu63 and Met70 in CD9). Extremely conserved Gly80 falls into heptad position g (Figure 3). Among CD9 orthologs, heptad positions a and d are absolutely conserved, whereas other positions have the following number of substitutions: b – 3; c – 2; e – 1; f – 3; g – 1 (Figure 4). Two of the f position residues in TM2 (65 and 79) also show higher variability among different tetraspanins (Figures 1, 3). Cysteine residues are frequently found near the cytoplasmic end of TM2 helix at positions 78 and 79; these cysteines are likely to be palmitoylated.<br><br>Analysis of TM3 and TM4 sequences
The TM3 domain provides another example of the heptad repeat pattern. Position a is occupied by two highly conserved leucine and a glutamate/glutamine residue (Leu89, Leu96 and Glu/Gln103 in CD9). Furthermore, two d positions are conserved – Phe/Tyr92 (aromatic residue) and Ile/Val99 (β-branched aliphatic residue; Figures 2, 3). In addition, residue 100 in position e is generally Phe or Leu. Among CD9 orthologs, position a has 1 substitution, positions b, c and f each have 6, positions d and e each have 2, and g has 4. Thus, as for TM1 and TM2, positions a and d are among the most conserved, but overall TM3 has more variable positions than TM1 or TM2 (Figure 3). Less than half of TM3 sequences contain cysteine residues, and those tend to occur at the internal positions of the helix (Figure 2).
TM4 shows less conservation among various tetraspanin family members than the other TM domains (Figures 2, 3). The only highly conserved feature is the glutamate/glutamine residue in position 209. In addition, one or two cysteine residues can be found at the C-terminal end of TM4 in some tetraspanins (e.g. CD9, CD81, CD151), and many sequences contain additional polar residues (Arg, Lys, His, Asn, Gln). No conserved heptad motif was identified in TM4, as also confirmed by analysis of substitutions in CD9 orthologs (data not shown).<br><br>Mutational analysis of conserved glycine residues in TM1 and TM2
The conserved nature of the Asn and Gly residues in TM1 and TM2 prompted an analysis of their functional role. To this end, we have probed whether mutations of these residues destabilize the protein molecule. We expressed a construct of the first and second TMs of CD9, connected by the small extracellular loop, and tagged with a C-terminal green fluorescent protein (TM(1+2)-GFP molecule). In <span class="yellow">human</span> rhabdomyosarcoma RD cells, the wild-type fusion protein localized mostly in a reticular, intracellular pattern, without forming any large aggregates (Figure 5, panel A). Remarkably, when double mutants Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu were expressed, the protein formed distinct large aggregates in a high proportion of cells (Figure 5, panels C and E). In contrast, double mutant Gly77Leu + Gly80Leu did not form such aggregates (Figure 5, panel G). Results with respective single mutants were similar to that with double mutants, with the aggregation being somewhat more pronounced for Leu substitutions of Gly67 and Gly74 compared to Gly25 and Gly32 mutations. No aggregation was observed for Asn18Ser and Asn18Tyr mutants (data not shown). Also, nearly identical results were obtained with <span class="yellow">human</span> HT1080 cells (data not shown).
We interpret these results as an indication that aggregating mutants are destabilized or misfolded while non-aggregating mutants retain the wild-type conformation. Intriguingly, mutations to the conserved GG7 motifs caused protein aggregation while the mutation of other glycines had no detectable effect. These results also suggest that wild-type GFP, which has weak tendency to self-associate, could enhance non-specific interactions of destabilized mutant TM(1+2) CD9 moieties, leading to their aggregation. Consistent with this hypothesis, the aggregation of Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu double mutants was suppressed when monomeric GFP molecule, Leu221Lys [34] was used (Figure 5, panels D and F). The use of monomeric GFP did not affect intercellular localization of wild-type CD9 TM(1+2) (Figure 5, panel B), or a Gly77Leu + Gly80Leu double mutant (Figure 5, panel H).
In summary, Leu substitutions of Gly residues that are part of the Asn-Gly-Gly (NGG7) motif in TM1, or Gly-Gly-Ala (GGA7) motif in TM2, resulted in destabilization and aggregation of GFP-fused TM(1+2) proteins, whereas substitutions of Gly77 or Gly80, which are not part of these motifs (Figure 3), failed to show such aggregation.<br><br>Prediction and modelling of interaction between TM1 and TM2
Consecutive helices in polytopic membrane proteins frequently interact [35]. Sequence analysis of TM1 and TM2 helices of tetraspanins reveals a remarkable complementarity in the distribution of large and small residues at heptad positions a and d along the helical axis (Figure 3), suggesting that these residues may interact. To further elucidate the potential for TM1-TM2 interaction, the putative interface was modeled using a novel algorithm that considers mutational data during each step of a Monte Carlo simulated annealing cycle (see Methods for details). Specifically, Gly25Leu, Gly32Leu, Gly67Leu and Gly74Leu were scored as disruptive mutations, while Asn18Ser, Gly77Leu and Gly80Leu were scored as silent mutations, based on their effects on protein stability (Figure 5 and data not shown).
The resulting model predicts left-handed crossing of TM1 and TM2 helices at an angle of +28°. The key element of the structure is the apposition of bulky and small heptad position a and d residues, as follows: Gly32-Leu63; Gly67-Leu29; Gly25-Met70; Gly74-Trp22; Asn18-Gly77; Ala81-Phe15 (Figure 6). Our model predicts that each of these residue pairs are in van der Waals contact. Additionally, two potential H-bonds are predicted in this model, indicating close packing: Gly67 Cα to Gly25 carbonyl oxygen, and Trp22 Cα to Met70 carbonyl oxygen. The packing is tighter in the ectodomain-proximal portion of the helices (Figure 6, panel B), as determined by Cα-Cα distances between interacting residue pairs.
The key elements of the model are corroborated by the presence of apparently complementary substitutions in TM1 and TM2 sequences of different tetraspanins (Figure 1, boxed residues). For example, Gly74 is predicted to interact with Trp22. In 8 of the 10 tetraspanins that contain a substitution for Gly74, a compensatory substitution occurs at the Trp22 position (Figure 1). Thus, a larger non-glycine side chain at position 74 may necessitate a less bulky non-Trp side chain in position 22. Likewise, the presence of a Cβ at position 25, typically occupied by glycine, necessitates a non-β-branched amino acid at position 70, which is occupied by a β-branched residue in nearly half of all cases. Indeed, we find that in each of 5 cases in which position 25 contains a Cβ, a leucine residue occurs in position 70. This analysis is consistent with our molecular model that suggests Leu70 will pack most favorably against a Cβ at position 25 than a β-branched residue or a methionine.<br><br>Role of TM1 and TM2 heptad motif residues in CD9 dimerization
To probe CD9 dimerization, we used a cysteine-mediated cross-linking approach. We established previously a simple and efficient method for cysteine-mediated cross-linking [25]. After cells are pre-treated with 2-BP for 16–24 hours to expose normally palmitoylated cysteines, the cysteines can be cross-linked using any of the following methods: a) Spontaneous oxidation in Brij97 lysates (a condition that preserves tetraspanin-tetraspanin associations), b) In situ cross-linking, by pre-lysis oxidation of cells with Cu2+-phenanthroline (CuP) to promote disulfide bond formation. c) In situ cross-linking with thiol-reactive cross-linking agents of defined length (e.g. DTME, BMB). The first two approaches produce in essence "zero-length" disulfides, indicative of close proximity of target cysteines and presumably high specificity of interaction. In contrast, chemical cross-linkers with 6–20 Å spacer arm may cross-link with higher efficiency, but not necessarily higher specificity. However, they provide advantages such as variable membrane permeability, and potential linkage cleavability. For tetraspanins such as CD9, membrane-permeable cross-linker DTME (13.3 Å-long, reducible) provides highly specific and efficient cross-linking [25]. Here we have used a cysteine cross-linking strategy, in combination with cysteine-scanning mutagenesis, to map the residues from TM1 and TM2 contributing to the CD9 dimerization interface.
For subsequent cross-linking experiments using CD9 TM(1+2)-GFP protein, the non-dimerizing form of GFP was used. This avoids potential GFP-dependent dimerization and aggregation that can be observed with wild-type GFP, especially when fusions with transmembrane proteins are studied [36]. Importantly, the Leu221Lys mutation in GFP prevented aggregation of mutant forms of CD9 TM(1+2), which was observed with wild-type GFP fusion (Figure 5). The TM(1+2) fragment of CD9 contains three native cysteines – Cys9, Cys78 and Cys79. Single-cysteine mutants of TM(1+2) were constructed, in which a cysteine was placed at various faces of TM1 or TM2 while all of the wild-type cysteines were simultaneously replaced by serines. The mutant proteins were transiently expressed in RD cells (having little endogenous CD9), which were then treated for 16–18 hours with 2-BP. To achieve maximal specificity in cross-linking we used a "zero-length" agent, CuP.
First, single-cysteine replacements were constructed for residues Leu14, Phe15, Gly16, Phe17 and Asn18, covering just over one complete helical turn at the beginning of TM1. While residue Asn18 is highly conserved, positions 14, 15 and 17 are occupied by bulky hydrophobic residues in most tetraspanins, whereas position 16 shows less conservation (Figures 1, 4). All of the single-cysteine mutants showed diffused pattern of protein localization, without any signs of aggregation. As shown in Figure 7A, the highest level of intermolecular cross-linking was observed for Leu14Cys and Phe17Cys mutants, a lower level for Phe15Cys and Gly16Cys mutants, and very little cross-linking for Asn18Cys substitution. These results indicate that: a) the first two transmembrane domains of CD9 alone can mediate its dimerization, and b) the g and c residues of TM1 (e.g. Leu14 and Phe17, Figure 3) are likely to be part of the intermolecular interface.
Similarly, single-cysteine substitutions were made for residues Gly77, Gly80 and Ala81 in TM2; in addition, proteins carrying a single wild-type cysteine, Cys9, Cys78 or Cys79, were tested. No protein aggregation was observed for any of these single-cysteine mutants. As shown in Figure 7B, the relatively low level of intermolecular cross-linking of wild-type CD9 TM(1+2)-GFP protein was enhanced dramatically in single-cysteine TM2 mutants Gly80Cys and Ala81Cys. The Gly77Cys mutant also had an elevated level of cross-linking. In contrast, any of the three native cysteines (9, 78 and 79) produced level of cross-linking not much greater than the wild-type TM(1+2) protein. Similar results were obtained with cysteine-reactive cross-linker BMB (data not shown). Likewise, comparable results were obtained with single-cysteine mutants of untagged, full-length CD9, using CuP (Figure 7C) as well as DTME cross-linker (data not shown).
These cross-linking results for TM1 and TM2 are consistent with our model that places residues Leu14, Phe17 and Gly80 on the same side of the TM1-TM2 pair (Figure 6, panel C). The strong cross-linking with Leu14Cys, Phe17Cys and Gly80Cys places the intermolecular interface toward the c and g phases of the TM1 helix, and the g phase of the TM2 helix, away from its e and f faces containing wild-type cysteines 78 and 79.
Critical residues at the TM1-TM2 interface also affect dimerization indirectly. To assess specific CD9 dimerization, we used a Gly80Cys substitution at the intermolecular interface for cross-linking. As shown in Figure 8A, single replacements of conserved heptad residues in positions 18, 25, 32, 67 and 74 (Asn18Ser, Gly25/32/67/74→Leu) strongly decreased the cross-linking mediated by Cys80. The effect was most pronounced for mutations of residues, Gly32 and Gly67, located in the tightly packed extracellular end of TM helices (Figure 6). In contrast, mutations of residues closer to the cytoplasmic end of TM2 (Gly74 and especially Ala81) had only modest to very little effect on cross-linking.
Relatively low efficiency of intermolecular cross-linking via native residues Cys9, 78, and 79 (Figures 7B,C) correlates well with the predicted location of Cys78 and 79 away from the dimeric interface (Figure 3), and suggests that the extramembrane N-terminal part of CD9 (residues 1–13) does not self-associate. We next examined whether mutations of conserved Asn and Gly residues in TM1 and TM2 decreased low-level background cross-linking via native cysteines. As expected, these mutations had virtually no effect on dimer formation of CD9 TM(1+2)-GFP (Figure 8B). The level of covalent dimer formed was not diminished for triple Asn18Ser + Gly25Leu + Gly32Leu and double Gly67Leu + Gly74Leu mutants, compared to wild-type TM(1+2) CD9 molecule. Similarly, the same triple and double mutations in the context of full-length CD9-GFP protein (with six cysteines) produced wild-type levels of cross-linking (Figure 8C). We interpret these findings as evidence for at least two types of associations between CD9 molecules: primary, involving residues 14, 17 and 80, and dependent on integrity of conserved heptad residues in TM1 and TM2, and less efficient secondary interactions, probably representing random collision events, and independent of the heptad residues (see Discussion for more details).<br><br>TM3 and TM4 cysteine residues in CD9 dimerization
After identifying the roles of conserved TM1 and TM2 residues in CD9 dimerization, we next probed whether residues proximal to TM domains 3 and 4 are also involved. To this end, disulfide cross-linking of full-length CD9 molecules containing 3 C-terminal cysteines (87, just before TM3; 218 and 219 in TM4) or 3 N-terminal cysteines (9, 78 and 79) was compared (Figure 9). We found that the C-terminal cysteines were only slightly better than N-terminal cysteines with respect to detection of CD9 dimers. However, markedly more trimers and tetramers were detected using C-terminal cysteines. Thus, residues 87, 218 and 219 at TM3 and TM4 in CD9 can together form contacts across the dimeric interface and also additional contacts with other neighboring CD9 molecules.<br><br>
Discussion
Here we provide the first detailed analysis of tetraspanin protein transmembrane domains. First, we show 1) the presence of a heptad repeat motif in TM1 and TM2, containing highly conserved Asn and Gly residues, 2) a leucine and glutamate/glutamine-containing heptad motif in TM3, and 3) high variability and absence of heptad repeats in TM4 sequences. Second, we provide evidence for a specific, intramolecular interaction between TM1 and TM2 domains, in which bulky hydrophobic residues pack against GG7 motif, and present a molecular model for this interaction. Third, experimental mapping of the CD9 dimerization interface firmly establishes an additional role for conserved TM1 and TM2 residues in dimeric intermolecular interactions. Fourth, preliminary evidence is provided to suggest that TM3 and TM4 domains contribute to expansion of CD9 dimers into higher order multimers.
Conserved residues in TM1 and TM2 of tetraspanins: role in intramolecular packing
We hypothesized that the first two transmembrane domains of tetraspanins might interact with each other because: a) consecutive TM domains frequently associate in known protein 3D structures [35], and b) they both contain a series of highly conserved amino acids – several Gly residues and an Asn residue (Figure 1). Conserved Gly residues are a frequent theme in the organization of interacting transmembrane domains. Analysis of 3D helix packing in polytopic membrane proteins reveals that Gly residues tend to localize in buried positions, especially at the helix-helix interfaces and helix crossing points [37,38]. Due to the absence of a side chain, Gly provides a flat surface for packing of a side chain from another residue, without loss of side-chain entropy upon interaction. The most common Gly-containing motif is GxxxG [39,40]. In glycophorin A (GpA), the major glycoprotein in erythrocyte cell membranes, Gly79 and Gly83 are part of the LIxxGVxxGVxxT sequence that promotes homodimerization of parallel transmembrane α-helixes [41,42]. In the GpA dimerization motif, Gly residues allow for tight packing in the right-handed helical crossing [43]. There are also examples of left-handed helical crossing in the context of a GxxxG motif [44]. Other membrane proteins that use the GxxxG motif for homo- or heterodimerization include <span class="yellow">bacteriophage M13</span> coat proteins [45], <span class="yellow">yeast</span> alpha factor receptor [46], integrin α IIb subunit [47], and ErbB1 receptor tyrosine kinase [48]. Other small residues, such as Ala and Ser, can substitute for Gly in this motif [49].
A protein motif in which Gly residues are separated by 6 other residues (GG7) is also common in transmembrane helices, especially in transporter/channel-like membrane proteins [50]. However, the structural features associated with this motif are not well known. In particular, it is unclear whether left-handed GG7 heptad repeat motif (as opposed to the "classic" right-handed GxxxG motif) can drive membrane helix association. In a recent work addressing this issue, Lear et al. [51] showed that a synthetic peptide containing Gly at heptad positions a and d could self-associate in vitro, likely in an antiparallel orientation. Heptad repeats containing conserved Gly residues occur in TM domains of α and β chains of MHC class II proteins, and mutations of the Gly residues disrupt the αβ heterodimer [52]. These examples demonstrate that Gly-based heptad motifs may be used for both intra- and intermolecular associations.
In this work, we identified a highly conserved GG7 motif in the first two tetraspanin TM domains. The GG7 sequence in tetraspanins is a part of a larger motif that also includes a conserved Asn residue in TM1 (NGG7) and an Ala/Ser/Thr residue in TM2 (GGA7). The seven-residue periodicity of these motifs strongly suggests their involvement in left-handed coiled coil packing reminiscent of the leucine zipper, rather than right-handed packing of the GpA-like GxxxG motif. For antiparallel helices, the left-handed crossing is in fact predominant over the right-handed in known TM domain structures [44].
In our model, heptad Gly residues in NGG7 and GGA7 sequences provide specific packing between antiparallel tetraspanin TM1 and TM2 helices by allowing tight van der Waals interactions with large hydrophobic residues (Figure 6). Highly efficient packing of bulky side chains against glycine residues is observed in known transmembrane protein 3D structures [38,53,54]. An example includes packing of helices M1 and M2 in potassium channel KcsA, where Val91 in M2 is paired with Gly43 in M1, and Leu36 in M1 contacts Ala98 and Gly99 in helix M2 [54,55]. In addition to facilitating helix-helix packing, Gly residues frequently provide additional CαH...O hydrogen bonds between two helices [44]. In our model, two Cα-backbone carbonyl H-bonds are predicted – between residues Gly27-Gly67, and Trp22-Met70.
Although polar and charged amino acid residues (such as Asn in the TM1 heptad motif) are infrequent in transmembrane domains, they are functionally important. Polar residues such as glutamine, glutamic acid, aspartic acid and asparagine can promote strong oligomerization of model membrane-associated helices [56-58]. Ruan et al. [59] used asparagine scanning mutagenesis to probe the interface of self-associating polyleucine helices by detecting their enhanced self-interaction in vitro and in the <span class="blue">E. coli</span>-based ToxR assay. Thus, a hydrogen bond in an apolar environment can result in strong, though not necessarily specific, association of transmembrane helices. In fact, mutations to polar residues in transmembrane proteins are commonly associated with disease [60]. Because of this potential for non-specific interactions, polar residues tend to localize at buried positions in TM domains.
In our case, the conserved Asn18 residue in CD9 is predicted to be a part of the TM1-TM2 interface, though our model does not predict any electrostatic interaction between Asn18 and TM2 (Figure 6). Consistently, substitution such as Asn18Tyr (and Gly77Leu) in TM(1+2)-GFP protein was not destabilizing as analyzed by protein aggregation. Curiously, the full-length Asn18Ser CD9 migrated slightly slower on SDS-PAGE gel (data not shown), suggesting that Asn18 does play a role in maintaining conformation of the molecule. The Asn18Cys single-cysteine mutant shows very little intermolecular cross-linking (Figure 7A), supporting the proposed location of this residue at the intramolecular interface. It is tempting to speculate that the "pocket" between TM1 and TM2 lined by Asn18 and Gly77 might be important for accommodating palmitate moieties that target Cys78 and Cys79 residues, and/or important for access by the putative palmitoyl transferase to those residues. Understanding the exact role of these highly conserved Asn18 and Gly77 residues in tetraspanins awaits further investigation.
In summary, we identified conserved glycine residues of TM1 and TM2 of tetraspanins as key elements required for intramolecular packing. Mutations of these key residues (Gly25, Gly32, Gly67 and Gly74 in CD9) resulted in protein destabilization and aggregation. There is ample evidence in the literature for mutations in transmembrane proteins that lead to protein destabilization, misassembly and pathologic conditions [61]. Thus, we have identified conserved heptad Gly residues in TM1 and TM2 of tetraspanins as plausible targets of destabilizing mutations with potential functional consequences.<br><br>Intermolecular interactions in tetraspanins
Tetraspanin CD9 forms mostly homodimers, but also a low level of heterodimers with CD81 and CD151 [25]. Thus, mapping the dimerization interface is an important next step in structure-function analysis of tetraspanins. Disulfide-mediated cross-linking, often in combination with cysteine-scanning mutagenesis, is a common strategy to probe oligomerization or intersubunit interactions of transmembrane proteins such as histidine kinase EnvZ [62], M(3) muscarinic acetylcholine receptor [63], <span class="blue">E. coli</span> lactose permease [64], synaptobrevin [65], integrins [66] and many others. In tetraspanins such as CD9, membrane-proximal cysteine residues are especially useful targets for disulfide trapping, as their linkage can be enhanced by pre-treating cells with 2-BP. While the ability of wild-type cysteines in CD9 to be cross-linked may indicate that they are close to the dimerization interface, more precise mapping was achieved here using cysteine-scanning mutagenesis.
Our data clearly identify regions, near the cytoplasmic face of TM1 and TM2, important for dimerization. Intermolecular zero-length cross-linking was highest when single cysteines were placed in positions 14, 17, 77, 80 and 81 in TM(1+2)-GFP molecule, or at positions 77, 80 or 81 in the full-length CD9 protein. Positions 14, 17 and 80 are distinct from the intramolecular interface and are on the same side of the TM1-TM2 pair (Figure 6). Thus, they are well-positioned to participate in an interaction with another molecule. At the same time, the model predicts that wild-type cysteines (Cys78, 79), which do not yield very efficient zero-length cross-linking, are on the other side of TM1-TM2 pair.
While using the cysteine at position 80 as the dimeric interface probe, mutations of conserved residues in TM1 and TM2 (especially Gly32 and Gly67 to Leu) clearly reduced intermolecular cross-linking. We do not suggest that those residues are directly involved in intermolecular interaction. Rather, we propose that destabilization of the intramolecular TM1-TM2 interaction by Gly to Leu substitutions (discussed above) causes an overall conformational change that reduces dimer formation.
An Ala81Leu mutation did not reduce cross-linking via Cys80, even though single-cysteine Ala81Cys molecules themselves produced a high level of cross-linking. These results, together with data on Gly32Leu and Gly67Leu mutations, are consistent with our model predicting that helices 1 and 2 interact more tightly near the extracellular end and less at the cytoplasmic end. This would give more flexibility to a cysteine at position 81 and also limit the effect of an Ala81Leu mutation. Location of this residue at the membrane/cytoplasmic border could also make it more accessible to CuP reagent as compared to residues buried in TM domain, thus elevating the efficiency of disulfide formation of the Ala81Cys mutant.<br><br>Multiple interfaces in tetraspanin molecules
In the full-length CD9 molecules, the 3 C-terminal cysteines (Cys87, 218 and 219) located at or in TM3 and TM4 promoted efficient dimer and even more efficient oligomer formation compared to the 3 N-terminal cysteines (Figure 9). Cys87 alone can be used to capture CD9 dimers [25]. These results suggest the existence of two dimeric interfaces in CD9 molecule – the TM 1-2/1-2 interface and the TM 3-4/3-4 interface (Figure 10). In a TM(1+2) molecule, the destabilization of 1–2 interaction, e.g. by Gly→Leu mutations, would affect the 1-2/1-2 interface, as discussed above. However, these mutations would not interfere with the 3-4/3-4 interface in a full-length molecule, which includes Cys87, 218 and 219. Thus, cross-linking of full-length molecules, containing all 6 cysteine residues, would be unaffected, as seen in Figure 8C. Furthermore, wild-type Cys9, Cys78 and Cys79 are apparently not at the primary 1-2/1-2 interface. Their relative inefficiency in cross-linking CD9 TM(1+2) protein likely reflects weak secondary contacts between the molecules, or possibly random collision events. Such events should be independent of mutations in the conserved Gly residues in TM1 and TM2, as was demonstrated in Figure 8B. The potential existence of two interfaces in tetraspanin molecules, 1-2/1-2 and 3-4/3-4, should provide enhanced flexibility for forming additional intermolecular contacts. Current understanding of tetraspanin microdomains assumes a few strong, primary homotypic and heterotypic tetraspanin complexes (e.g. CD9-CD9, CD9-CD81, CD151-α3 integrin, CD81-EWI2) that help bring together various other proteins, forming secondary-type associations. Such properties of tetraspanins may bring signaling molecules such as protein kinase C or phosphatidylinositol 4-kinase to the vicinity of integrins [67,68].
The organization of the TM3 domain points to a potential role in protein-protein interactions. A motif of Leu-Leu-Glu(Gln) spaced 7 residues apart (heptad positions a), with highly conserved residues in two consecutive positions d, poses as a likely interaction module. If responsible for heterologous protein-protein interactions, it would form another distinct interface of tetraspanin molecule. Our preliminary data indicate that replacing the Leu and Glu residues in TM3 of CD9 with Ala has no effect on cell surface expression of the protein and its dimerization (data not shown). It remains to be tested if interactions with other proteins will be affected. Similarly, the TM4 domain may provide additional contributions to lateral tetraspanin associations. Much higher sequence variability, and the lack of a distinct heptad pattern suggests that TM4 is a major contributor to diversity among tetraspanin complexes. Structure-function analysis of TM3 and TM4 domains in tetraspanins is the subject of ongoing investigation.<br><br>
Conclusion
We have defined the TM1-TM2 intramolecular interface in tetraspanin CD9, providing evidence for glycines (Gly25 and Gly32 in TM1, Gly67 and Gly74 in TM2) packing against apposing bulky aliphatic residues. Second, we mapped an intermolecular CD9 interface (involved in CD9 homodimer formation) to the vicinity of residues Leu14 and Phe17 in TM1 and Gly77, Gly80 and Ala81 in TM2. Finally, we provide preliminary evidence that TM3 and TM4 in CD9 may contribute to a second intermolecular interface. Key CD9 residues involved in intra- and intermolecular interactions are highly conserved throughout the tetraspanin family, thus suggesting that our findings will apply to most tetraspanins.<br><br>Methods
Materials
Cell culture reagents were from Invitrogen (Carlsbad, CA). 2-bromopalmitate was from Fluka (Milwaukee, WI), N-ethylmaleimide (NEM) and 1,10-phenanthroline were from Sigma-Aldrich (St. Louis, MO), and chemical cross-linkers dithio-bis-maleimidoethane (DTME) and 1,4-Bis-maleimidobutane (BMB) were purchased from Pierce Endogen (Rockford, IL). Triton X-100, protease inhibitor cocktail and FuGENE 6 transfection reagent were obtained from Roche (Indianapolis, IN). Restriction endonucleases and Pfu DNA polymerase were obtained New England Biolabs (Beverly, MA) and Stratagene (Carlsbad, CA), respectively. All other chemicals were purchased from Sigma-Aldrich or Fisher Scientific (Pittsburg, PA).<br><br>Sequence analysis
Tetraspanin sequences were obtained from SWISS-PROT and GenBank databases. Locus designations, accession numbers and the most commonly used protein names are summarized in Tables 1 and 2. TM segments were delineated by inspection of hydrophobicity profiles, using database annotations and previous analyses of TM sequences as a guide ([28], M. Hemler, unpublished), and aligned manually. Residue numbers in <span class="yellow">human</span> CD9 sequence are used a reference point throughout the study.<br><br>DNA cloning and mutagenesis
Sequence encoding CD9 protein was cloned into vector pcDNA3 (Invitrogen, Carlsbad, CA) and pEGFP-N1 (Clontech, Palo Alto, CA), for expression of untagged and C-terminally GFP-tagged CD9, respectively. pEGFP-N1 encoding CD9 TM(1+2) -GFP fusion protein was constructed by subcloning DNA for residues 1–83 of CD9 into HindIII and PstI sites of the vector; to introduce the PstI site, codon GTG for Val82 was changed to CTG (coding for Ala). In the resulting fusion protein, there is a 13-amino acid linker (with no cysteines) between CD9 and GFP. To minimize the low inherent ability of GFP to homodimerize, which could potentially influence the results of CD9 cross-linking, we used a monomeric GFP mutant, Leu221→Lys [34], for cross-linking experiments.
Mutations were introduced in full-length and TM(1+2) CD9 proteins by a PCR-based strategy using mutagenic primers and Pfu DNA polymerase. All mutations were confirmed by DNA sequencing.<br><br>Protein expression, microscopy, cysteine disulfide cross-linking and Western blotting
DNA constructs encoding TM(1+2)-GFP or full-length CD9 proteins were transfected into <span class="yellow">human</span> rhabdomyosarcoma RD cells using the FuGENE 6 reagent. Cells expressing GFP fusion proteins were analysed by fluorescence microscopy 18–28 hours post-transfection. Images were captured using Spot 1.4.0 camera (Diagnostic Instruments, Sterling Heights, MI) attached to Nikon Eclipse TE300 microscope.
For experiments involving cysteine-mediated cross-linking, cells were treated with 50 μM 2-BP starting 24–26 hours post-transfection and continuing for 16–18 hours. Cross-linking was carried out by incubating cells in HBSM buffer (25 mM Hepes-NaOH, pH 7.2, 150 mM NaCl, 2 mM MgCl2) containing either a) 0.6 mM CuSO4 and 1.8 mM 1,10-phenanthroline (CuP complex) or b) 0.2 mg/ml homobifunctional cysteine-reactive cross-linker (e.g. DTME), diluted from fresh 10 mg/ml solution in DMSO. After incubation for 10–15 minutes (with CuP) or 30–45 minutes (with cross-linker), cells were washed twice for 10 minutes with HBSM containing 10 mM NEM to block residual free cysteines. Cells were lysed in HBSM containing 1% Triton X-100, 0.1% SDS and a cocktail of protease inhibitors with 1 mM EDTA at 4°C for 45–60 minutes. Cell lysate was clarified by centrifugation at 14,000 × g for 15 minutes, an aliquot was removed, and proteins from it were precipitated by addition of trichloroacetic acid to 10% on ice followed by centrifugation at 14,000 × g for 10 minutes. After two washes with ice-cold acetone, protein pellet was solubilized in SDS-PAGE sample buffer without a reducing agent (50 mM Tris-HCl, pH 6.8, 1% SDS, 8% glycerol).
In some experiments, CD9 protein was immunoprecipitated using monoclonal antibody Alb6 (Immunotech, Marseille, France). Proteins were separated by SDS-PAGE and analyzed by Western blotting using monoclonal antibody JL-8 (Clontech) for GFP or Alb6 for CD9. Bands from X-ray films were quantitated using GeneTools™ software from Syngene (Frederick, MD).<br><br>Modeling of TM1-TM2 interaction
An atomic model of the CD9 TM1-TM2 dimer was constructed with a Monte Carlo-simulated annealing (MCSA) algorithm [69]. Two idealized α-helices corresponding to TM1 residues Tyr12 through Leu35 and TM2 residues Gly59 through Val82 were docked with six orthogonal parameters: three rigid body translations and three rotations. During each step of a MCSA cycle, there was an equal probability of changing either one parameter or all six parameters to random values. A conformation's energy was calculated in vacuo with the AMBER united-atom force field for van der Waals interactions [70]. The van der Waals term was modified as described by Kuhlman and Baker [71]. If a structure had favorable dimerization energy, the energies of select mutants were calculated. Structures were selected with a novel scoring function that maximizes the Boltzmann probability of dimerization for silent mutations while minimizing the probability for disruptive mutations. Asn18Ser, Gly77Leu, and Gly80Leu were scored as silent mutations while Gly25Leu, Gly32Leu, Gly67Leu, and Gly74Leu were considered to be disruptive. Each MCSA cycle consisted of 50,000 steps with an exponential temperature decay from 10,000 to 10 K.
Ten MCSA cycles through global sample space were used to restrict the search area. Parameters were restricted to ± 2 standard deviations from their mean value for structures within 10 kcal of the best structure. MCSA cycles were repeated as described above with additional optimization of χ values: rotamers at the protein-protein interface were optimized with Dead End Elimination [72], and χ values were further optimized with Monte Carlo. All MCSA cycles converged upon structures that were within a root mean squared deviation (RMSD) of 1.5 Å with the best structure, and structures that scored within 5 kcal of the best score had an RMSD of less than 0.5 Å with the best structure.<br><br>
List of abbreviations
2-BP, 2-bromopalmitate; CuP, Cu2+-phenanthroline; DTME, dithio-bis-maleimidoethane; LEL, large extracellular loop; NEM, N-ethylmaleimide; TM, transmembrane (domain).<br><br>Authors' contributions
OVK carried out sequence comparisons, mutational analysis and cross-linking experiments, and drafted the manuscript. DGM built the TM1-TM2 interaction model and contributed to the manuscript. WFD supervised DGM's work. MEH coordinated the whole study and prepared the final manuscript.<br><br>
<h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="blue">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="blue">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="blue">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="blue">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="blue">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="blue">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="blue">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="blue">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA2327290</h3><span class="yellow">Borrelia burgdorferi</span> membranes are the primary targets of reactive oxygen species
Abstract
Spirochetes living in an oxygen-rich environment or when challenged by host immune cells are exposed to reactive oxygen species (ROS). These species can harm/destroy cysteinyl residues, iron-sulphur clusters, DNA and polyunsaturated lipids, leading to inhibition of growth or cell death. Because <span class="yellow">Borrelia burgdorferi</span> contains no intracellular iron, DNA is most likely not a major target for ROS via Fenton reaction. In support of this, growth of <span class="yellow">B. burgdorferi</span> in the presence of 5 mM H2O2 had no effect on the DNA mutation rate (spontaneous coumermycin A1 resistance), and cells treated with 10 mM t-butyl hydroperoxide or 10 mM H2O2 show no increase in DNA damage. Unlike most bacteria, <span class="yellow">B. burgdorferi</span> incorporates ROS-susceptible polyunsaturated fatty acids from the environment into their membranes. Analysis of lipoxidase-treated <span class="yellow">B. burgdorferi</span> cells by Electron Microscopy showed significant irregularities indicative of membrane damage. Fatty acid analysis of cells treated with lipoxidase indicated that host-derived linoleic acid had been dramatically reduced (50-fold) in these cells, with a corresponding increase in the levels of malondialdehyde by-product (fourfold). These data suggest that <span class="yellow">B. burgdorferi</span> membrane lipids are targets for attack by ROS encountered in the various stages of the infective cycle.<br><br>Introduction
<span class="yellow">Borrelia burgdorferi</span>, the causative agent of Lyme disease, survives and proliferates in distinctly different niches, including its arthropod vector and various mammalian hosts. These ‘micro’ environments provide their own distinct sets of advantages and challenges to <span class="yellow">B. burgdorferi</span>. For example, during the initial stages of infection of the mammalian host, immune cells attempt to prevent <span class="yellow">B. burgdorferi</span> from establishing an infection using several systems including those generating bacteriocidal reactive oxygen species (ROS) [e.g. superoxide radicals (O2-), hydrogen peroxide (H2O2) and hydroxyl radicals (OH)] and reactive nitrogen species (RNS) [e.g. nitric oxide (NO) and peroxynitrite] (Storz and Imlay, 1999). In order for <span class="yellow">B. burgdorferi</span> to successfully colonize a new host and cause disease, they must overcome the challenges posed by the innate immune system including the deleterious effects of ROS/RNS compounds.
The effects of ROS/RNS on cells have been extensively investigated. These highly reactive compounds have been shown to damage cellular macromolecules including DNA, proteins and cellular membranes. The damage to membranes can arise through either lipid or membrane protein damage. In eukaryotes, membrane lipids are a major target of ROS. Free radicals attack polyunsaturated fatty acids in membranes and initiate lipid peroxidation. A primary effect of this is a decrease in membrane fluidity which affects the physical properties of the membrane altering the function of membrane-associated proteins. Once lipid peroxides form, they react with adjacent polyunsaturated lipids causing an amplification of the damage. Lipid peroxides undergo further oxidation to a variety of products, including aldehydes, which subsequently react with and damage membrane proteins. However, in bacteria, it is assumed that lipids are not subject to the oxidative damage observed in eukaryotic cells. Only certain polyunsaturated lipids, such as linoleic acid and linolenic acid, are susceptible to oxidation (Gutteridge and Halliwell, 1990), and it is clear that most bacteria do not synthesize or incorporate these types of lipids in their cell membrane. Two notable exceptions are the photosynthetic bacteria which synthesize and incorporate significant levels of linoleic acid in their membrane (Tasaka et al., 1996) and <span class="yellow">Helicobacter pylori</span> membranes which contain between 0.5% and 3% linoleic acid (Ursini and Bindoli, 1987).
Instead, it has been shown that the most damaging effects of ROS in bacteria result from the interactions of H2O2 with ‘free’ Fe2+ (Imlay, 2003), generating very reactive OH (Fenton reaction). Because of this reactivity, the effect on any given biomolecule will depend largely upon proximity to the target. Because Fe2+ localizes along the phosphodiester backbone of nucleic acid, DNA is a major target of OH. This reactive species can pull electrons from either the base or sugar moieties, producing a variety of lesions including single- and double-stranded breaks in the backbone and chemical cross-links to other molecules. These strand breaks and other lesions block DNA replication and contribute to OH toxicity and cell death. Other base damage, which does not hinder replication, may result in a significant increase in mutation rates.
The intracellular biochemistry of <span class="yellow">B. burgdorferi</span> suggests that the primary cellular target of ROS may be distinct from that described in other bacteria such as <span class="yellow">Escherichia coli</span>. In <span class="blue">E. coli</span>, the extent of DNA damage due to H2O2 and Fenton chemistry is directly proportional to Fe metabolism and the free Fe concentration within the cell (10 µM) (Keyer and Imlay, 1996). As the intracellular Fe concentrations of <span class="yellow">B. burgdorferi</span> are estimated to be < 10 atoms per cell (Posey and Gherardini, 2000), it seems unlikely that DNA is a primary target for ROS. Therefore, the purpose of this study is to determine the biochemical effects of ROS on <span class="yellow">B. burgdorferi</span>, including growth effects and biological/physical damage.<br><br>Results
Effect of ROS on <span class="yellow">B. burgdorferi</span> cells
In order to determine what the cellular targets of ROS in <span class="yellow">B. burgdorferi</span> are, we first needed to determine the sensitivity of Borrelia cells to various oxidants. Microaerobic cultures of <span class="yellow">B. burgdorferi</span> strain B31A3 were grown to a cell density of 5 × 107 cells ml−1, treated with varying concentrations of H2O2 (0–50 mM) or t-butyl hydroperoxide (0–50 mM) and the number of surviving cells determined by plating. The results are shown in Fig. 1. When cells were exposed to 10 mM t-butyl hydroperoxide, approximately 50% of the cells survive (Fig. 1A). In contrast, when <span class="blue">E. coli</span> cells were exposed to 1 mM t-butyl hydroperoxide, approximately 1% of the cells survive. <span class="yellow">B. burgdorferi</span> cells were much more resistant to t-butyl hydroperoxide than <span class="blue">E. coli</span> cells, with a survival rate of approximately 100% at 1 mM t-butyl hydroperoxide. This trend was observed when cells were exposed to H2O2. <span class="blue">E. coli</span> cells exposed to 1 mM H2O2 have approximately 10% survivability, while 100% of the <span class="yellow">B. burgdorferi</span> cells survive at this concentration, and approximately 80% survive when exposed to 10 mM H2O2 (Fig. 1B). Similar results were obtained when cells were exposed in HN (Hepes-NaCl) buffer, suggesting that the high-level resistance to ROS was not due to the interaction of ROS with components of Barbour-Stoenner-Kelly (BSK-II) growth medium. Taken together, these data indicated that <span class="yellow">B. burgdorferi</span> strain B31A3 was highly resistant to exposure to both t-butyl peroxide and H2O2.<br><br><span class="yellow">Borrelia burgdorferi</span> DNA is not the primary target of ROS
For most bacteria, DNA is the major target of ROS causing a wide variety of DNA lesions. This is in part due to the localization of ‘free’ Fe2+ along the phosphodiester backbone of nucleic acids, putting the DNA in close proximity to the active species formed via the Fenton reaction. However, <span class="yellow">B. burgdorferi</span> has been shown to harbour few genes encoding orthologues of known iron-containing proteins, does not require Fe for growth and has intracellular Fe concentrations estimated to be < 10 atoms per cell (Posey and Gherardini, 2000). Taken together, these observations strongly suggest that <span class="yellow">B. burgdorferi</span> DNA is not a major target for ROS. To test this hypothesis, different techniques were used to measure DNA damage in <span class="yellow">B. burgdorferi</span> cells after exposure to ROS.
One reliable indicator of DNA damage by ROS in a cell is an increase in the spontaneous mutation rate. In <span class="yellow">B. burgdorferi</span>, mutations that confer resistance to coumermycin A1, which targets the β subunit of DNA gyrase, have been mapped to gyrB, the gene encoding DNA gyrase B (Samuels et al., 1994). In each case, a single point mutation correlated with this drug resistance. To determine if exposure to oxidants increases DNA damage by increasing point mutations, <span class="yellow">B. burgdorferi</span> B31A3 cells grown under microaerobic conditions were treated with 5 mM H2O2 and plated in the presence and absence of 250 ng ml−1 coumermycin A1. The mutation frequency was calculated as the number of colonies that are CouR per total number of cells plated. The spontaneous resistance frequency of treated cells was approximately equivalent to that of untreated cells, 8.8 × 10−8 and 1.33 × 10−7, respectively, indicating no increase in the number of point mutations (Fig. 2A). Also, no increase in point mutations was observed when cells were treated with higher concentrations of H2O2 (10 mM) or when treated with t-butyl hydroperoxide (5 and 10 mM) (data not shown).
Another effective way to determine DNA damage is by measuring the number of apurinic/apyrimidinic sites (AP). AP sites in DNA, where the DNA base is lost, can be generated spontaneously under physiological conditions by hydrolysis of the N-glycosylic bond, or can also be formed by DNA-damaging agents (Lindahl and Nyberg, 1972), such as UV, alkylating agents or OH. They are also intermediates in the base excision repair pathway (BER) (Weiss and Grossman, 1987; Friedberg and Hanawalt, 1988; Wallace, 1988). Thus, the cellular steady state level of AP sites would increase as a consequence of base modifications and their subsequent repair. As AP sites are bypassed inefficiently by DNA polymerase in bacterial cells, DNA lesions can result in a significant block in DNA replication. Therefore, the number of AP sites can serve as a sensitive indicator of DNA damage resulting from oxidative stress (Kubo et al., 1992). To determine whether ROS can damage <span class="yellow">B. burgdorferi</span> DNA, strain B31A3 cells grown under microaerobic conditions were treated with H2O2 (1 or 10 mM), t-butyl hydroperoxide (1 or 10 mM) or lipoxidase (an enzyme which specifically catalyses the hydroperoxidation of lipids containing a cis, cis-pentadiene structure, such as linoleic acid). The DNA was isolated and assayed for AP sites. The results are shown in Fig. 2B. In all cases, the numbers of AP sites per 105 bp DNA were equivalent, indicating that the addition of oxidants did not increase the number of DNA base lesions. In contrast, when <span class="blue">E. coli</span> strain TA4315 cells (ahpCF) (Storz et al., 1989) were treated with 100 µM H2O2, the number of AP sites increased ∼10-fold over untreated cells. These data suggest that <span class="yellow">B. burgdorferi</span> DNA was not a major target for oxidative damage under these conditions.
It is of note to mention that the total number of AP sites per 105 bp DNA is approximately 10-fold higher in untreated Borrelia B31A3 DNA than in untreated <span class="blue">E. coli</span> DNA. The Borrelia genome consists of a linear chromosome and multiple linear and circular plasmids. Numerous observations indicate that functional telomeres require interaction with DNA damage repair proteins, suggesting that the DNA damage repair machinery, including the BER, is involved in replication of telomeres and protection of functional chromosome ends (Verdun and Karlseder, 2007). An important intermediate in BER is the apurinic or abasic site. Therefore, it was possible that the higher numbers of AP sites in untreated Borrelia DNA was due to the number of telomeres present in the genome. The <span class="yellow">B. burgdorferi</span> strain B31A3 harbours 11 linear plasmids and a linear chromosome with ∼100 unpaired bases in the telomere loops per genome or 10 bases per 105 bp DNA (Hinnebusch and Barbour, 1991; Chaconas, 2005). Untreated Borrelia B31A3 contains 11 ± 3.8 AP sites per 105 bp DNA, suggesting that the elevated number of AP sites in Borrelia was due to the number of telomeres. To test this hypothesis, the AP sites were measured in DNA isolated from untreated <span class="yellow">B. burgdorferi</span> strain B314 which harbours no linear plasmids (Sadziene et al., 1993). The number of unpaired bases in the telomere loops per genome was estimated to be 10, which corresponds to 1 base per 105 bp DNA, and only 3.6 ± 0.2 AP sites per 105 bp DNA were detected, supporting the hypothesis that the high number of AP sites in untreated Borrelia DNA was due to the number of unpaired bases in the telomeres. The analyses of the AP sites strongly suggested that <span class="yellow">B. burgdorferi</span> DNA was not the major target for oxidative damage.
In addition to the generation of abasic sites, oxygen radicals often damage DNA through the formation of 8-oxoguanine lesions (Nakamura et al., 2000). 8-Oxoguanine, through its ability to mispair with bases other than cytosine, likely plays a role in DNA mutagenesis. Consequently, 8-oxoguanine is often used as a marker of oxidized DNA damage. To determine whether ROS can damage <span class="yellow">B. burgdorferi</span> DNA and cause 8-oxoguanine lesions, strain B31A3 cells grown under microaerobic conditions were treated with H2O2 (5 or 10 mM), t-butyl hydroperoxide (5 or 10 mM) or lipoxidase. The DNA was isolated and assayed for 8-oxoguanine using an Enzyme-Linked ImmunoSorbent Assay. The results are shown in Fig. 2C. In all cases, the amount of 8-oxoguanine in Borrelia cells was below the detection limit of the assay. In contrast, when a MutM (the specific glycosylase that removes the 8-oxoguanine)-deficient <span class="blue">E. coli</span> strain was treated with 100 µM H2O2, the amount of 8-oxoguanine sites increased ∼fivefold over untreated cells. It is important to point out that no MutM homologue has been identified in the genome of <span class="yellow">B. burgdorferi</span>. Taken together, these data suggested that <span class="yellow">B. burgdorferi</span> DNA was not a major target for oxidative damage under these conditions.<br><br>The membranes of <span class="yellow">B. burgdorferi</span> are targeted during oxidative stress
In eukaryotes, membrane lipids are a major target of ROS. Free radicals can attack polyunsaturated fatty acids, such as linoleic acid and linolenic acid, in membranes and initiate lipid peroxidation. This reaction can cascade throughout the membrane to adjacent polyunsaturated fatty acids, decreasing membrane fluidity and generating more toxic products such as aldehydes (Imlay, 2003). Because most bacterial membranes contain saturated and monounsaturated fatty acids rather than ‘reactive’ polyunsaturated lipids, peroxidation of lipids in bacterial membranes is not considered a problem. As <span class="yellow">B. burgdorferi</span> cannot synthesize their own lipids, they must instead scavenge them. Therefore, it seems likely that their membrane composition would reflect the host's lipid profile or that of their growth medium (Barbour, 1984; Fraser et al., 1997) and would contain some polyunsaturated fatty acids. To determine if <span class="yellow">B. burgdorferi</span> contains polyunsaturated lipids, <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under anaerobic conditions and analysed for fatty acid composition by Lipid Technologies (Austin, MN). The results are shown in Table 1 and are reported as percentage of total fatty acid content. Linoleic acid comprised ∼10% of the total lipid content and the linolenic acid content was measured to be ∼1%, indicating that Borrelia cells do contain lipids that are susceptible to ROS damage. The amount of linoleic acid and linolenic acid present in Borrelia reflected the amount present in the media. Therefore, these results suggested that the amount of these fatty acids in the membranes would vary as availability varies.
To determine if Borrelia polyunsaturated lipids can undergo lipid peroxidation, <span class="yellow">B. burgdorferi</span> B31A3 cells grown microaerobically were treated with 1 mM t-butyl hydroperoxide or 250 mg of lipoxidase, and the cell pellets were analysed for fatty acid composition (Industrial Laboratory). The results are shown in Table 2 and are reported as the percentage of total cell mass. The per cent of linoleic acid in the total cell mass decreased with treatment, while the levels of oleic acid (c18:1n9) and pentadecanoic acid (c15:0) remained relatively constant. Untreated cells contained 0.04% linoleic acid, while cells treated with t-butyl hydroperoxide contained 10-fold less (0.004%) linoleic acid and cells treated with lipoxidase had no detectable linoleic acid present in the sample. These data indicated that the linoleic acid present in <span class="yellow">B. burgdorferi</span> membranes can be oxidized by ROS.
Malondialdehyde (MDA) is generated as a relatively stable end-product from the oxidative degradation of polyunsaturated fatty acids. MDA has thus been used as an indicator of lipid peroxidation (Gutteridge and Halliwell, 1990; Esterbauer et al., 1991). To further demonstrate that Borrelia lipids can undergo lipid peroxidation, B31A3 cells grown microaerobically were treated with 5 mM AAPH (free radical generator) or 250 mg of lipoxidase and MDA measured (Seljeskog et al., 2006). The results are shown in Fig. 3. Untreated cells contained ∼16.5 µM of MDA per mg of protein. When the cells were treated with AAPH or lipoxidase, the amount of MDA increased ∼1.5-fold (27.3 µM of MDA per mg of protein) and approximately twofold (33 µM of MDA per mg of protein) respectively (Fig. 3A). As a control, eukaryotic cells (<span class="yellow">mouse</span> myeloma cells SP2) were treated with AAPH and MDA measured (Fig. 3B). <span class="yellow">Mouse</span> myeloma cells have been used as a model system for the determination of phospholipid hydroperoxides (Chotimarkorn et al., 2005). In this case, the amount of MDA increased approximately threefold (36.3 µM of MDA per mg of protein) in the treated cells versus untreated cells (11.7 µM of MDA per mg of protein). Taken together, these data suggested that, like eukaryotic membranes, Borrelia membrane lipids were capable of undergoing lipid peroxidation.
As shown in Fig. 3A, there is a measurable quantity of MDA present even in untreated cells, suggesting that the membrane lipids are damaged without the addition of exogenous ROS. Oxidative damage is an unavoidable by-product of growth in an oxygen environment because superoxide anions and H2O2 are formed whenever molecular oxygen chemically oxidizes electron carriers. To determine if the Borrelia lipids are damaged from growth in an oxygen environment, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under anaerobic, microaerobic and aerobic conditions and MDA measured. Figure 3C demonstrates that as the oxygen concentration increased, the amount of MDA increased. Cells grown under anaerobic conditions contained the lowest amount of MDA (7.6 µM of MDA per mg of protein), approximately twofold less than the measured amount in microaerobic cells (16.5 µM of MDA per mg of protein). Aerobically grown cells contained the highest amount of MDA, ∼1.5-fold greater than that observed in microaerobic cells (28.8 µM of MDA per mg of protein) and ∼3.7-fold greater than that observed in anaerobically grown cells. These data suggested that Borrelia lipids can be damaged during aerobic growth.
Our High Performance Liquid Chromatography analyses of the MDA present in anaerobically grown <span class="yellow">B. burgdorferi</span> cells showed a small peak with a retention time similar to that of the MDA standard. This was puzzling as little or no lipid peroxidation should occur under these conditions. To further characterize this ‘MDA’ peak, a three-dimensional diode array spectra was generated by scanning each sample during the elution of the peak (Fig. 4, lower sections). An authentic MDA standard was also scanned as a control (Fig. 4A, lower section). In the anaerobically grown cells, the spectrum shows that two compounds with different absorbance maximums (Fig. 4B, lower section) comprised the single retention time peak from the HPLC chromatogram (Fig. 4B, upper section). Based on this spectrum, the amount of actual MDA contributed < 15% of the total amount of material detected in the HPLC peak while the second contaminating peak contributed > 85%. Therefore, the amount of MDA in untreated anaerobically grown cells was considerably less than the 7.6 µM of MDA per mg of protein actually measured. However, in cells grown under microaerobic conditions, the MDA peak contributed more to the overall retention time peak when compared with the anaerobic spectrum, while the second contaminating peak stays relatively constant (Fig. 4C, lower section). These spectra demonstrated that the increase in the MDA retention time peak between anaerobically and aerobically grown cells was due to the increase in the amount of authentic MDA present.
The fluorescent probe diphenyl-1-pyrenylphophine (DPPP) has been used for detection of lipid hydroperoxides in cell membranes (Okimoto et al., 2000; Takahashi et al., 2001). In this method, the hydroperoxides are reduced with DPPP, resulting in the formation of the fluorescent arylphosphine oxide. To visualize the lipid hydroperoxides, <span class="yellow">B. burgdorferi</span> B31A3 cells were labelled with DPPP and observed by fluorescence microscopy (Fig. 5B). <span class="blue">E. coli</span> Top10 cells and <span class="yellow">mouse</span> myeloma SP2 cells were also labelled and visualized to serve as negative and positive controls respectively (Fig. 5A and C). Red Fluorescent dye was used to visualize the cell membranes. A strong fluorescence of DPPP was observed for B31A3 cells (Fig. 5B) and for the <span class="yellow">mouse</span> myeloma cells (Fig. 5C), but not for the <span class="blue">E. coli</span> cells (Fig. 5A). An overlay of the two dyes demonstrates that the DPPP fluorescence of both the Borrelia and myeloma cells corresponds to the areas of red fluorescence. This indicated that lipid hydroperoxides were present on the Borrelia cell membranes and suggested that the membranes were damaged.
To further demonstrate Borrelia membrane damage, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under anaerobic and microaerobic conditions and visualized by negative stain using an electron microscope. Additionally, cells grown under microaerobic conditions were treated with 250 mg of lipoxidase and visualized by Electron Microscopy. Intact membranes were observed in cultures of B31A3 grown under anaerobic and microaerobic conditions (Fig. 6A and B respectively). However, in cultures treated with lipoxidase (Fig. 6C), a significant number of membrane blebs were seen surrounding the spirochetes, indicating membrane damage. Taken together, these data indicated that Borrelia membranes were a target for oxidative stress.<br><br>
Discussion
Most bacterial pathogens are faced with the challenge of overcoming ROS generated by the host immune system. These radicals can cause a great deal of damage to biological molecules both in vitro and in vivo. The potential cellular targets for ROS damage in bacteria include DNA, RNA, proteins and lipids, and extensive work has been done to determine the cellular targets of ROS that affect bacterial survival. To date, the most definitive work has been done on <span class="blue">E. coli</span>. It seems clear from several studies that the most physiologically relevant target of ROS in <span class="blue">E. coli</span> is DNA. The exposure to µM concentrations of ROS (i.e. H2O2) is sufficient to cause DNA damage, inhibit DNA replication, increase the mutation rate and often lead to cell death. This process leading to DNA damage begins with formation of O2- and H2O2 from the oxidation of flavoproteins and/or the diffusion of these reactive species from the extracellular milieu. The subsequent oxidation of Fe-S proteins by ROS (e.g. O2-) leads to an increase in intracellular ‘free’ Fe2+ which triggers the reduction of H2O2 to OH (Fenton reaction). The highly reactive nature of OH limits its diffusion so that it generally reacts with molecules in close proximity to its origin (e.g. the Fe2+ associated with DNA). In DNA, OH oxidizes sugar and base moieties, producing radicals which ultimately generate lesions, including base modifications, strand breaks and chemical cross-links to other molecules. Base modifications lead to mismatching and increased mutation frequencies while more severe damage, such as strand breaks, prevents DNA replication, contributing to OH toxicity and cell death. Critical to this chemical process in vivo is the presence of iron. No other metal or non-metal electron carrier appears to be able to univalently reduce H2O2 to OH in vivo (Macomber et al., 2007).
These observations are critical in beginning to understand the oxidative damage/targets in <span class="yellow">B. burgdorferi</span>. It has been shown in <span class="blue">E. coli</span> that the free-iron pool size determines the rate of oxidative DNA damage. For example, in wild-type <span class="blue">E. coli</span>, free iron levels are estimated to be 10 µM, yet H2O2 is only mildly genotoxic (Keyer and Imlay, 1996). However, in <span class="blue">E. coli</span> Fur- mutants, intracellular iron concentration increases eightfold while survival is 10-fold lower when cells are exposed to H2O2. Because <span class="yellow">B. burgdorferi</span> cells do not contain detectable levels of intracellular Fe, it seems unlikely that DNA is a major target for damage via the Fenton reaction in this bacterium (Posey and Gherardini, 2000). The experimental data present in this report suggested that this was the case. When <span class="yellow">B. burgdorferi</span> cells were exposed to high concentrations of ROS (e.g. H2O2), there was no effect on the spontaneous mutation rate (Fig. 2A) or the number of DNA lesions (AP sites or 8-oxoguanine) (Fig. 2B and C). It should be noted that in these experiments, a wild-type strain of <span class="yellow">B. burgdorferi</span> was used and presumably all of the endogenous oxidative stress enzymes were expressed. Therefore, it is possible that no DNA damage was observed because these enzymes are capable of detoxifying the cell and protecting nucleic acids from oxidation via the Fenton reaction. However, we do not believe this to be the case as the concentration of oxidants used in these experiments were significantly higher than concentrations known to cause DNA damage in <span class="blue">E. coli</span> and other pathogenic bacteria.
The lack of detectable DNA damage in <span class="yellow">B. burgdorferi</span> cells under the conditions tested could be the result of very efficient DNA repair systems. Bacteria, such as <span class="blue">E. coli</span>, harbour genes encoding repair enzymes (i.e. MutM, MutY, Ung, AlkA, MutS, MutL, ExoIII, EndoVIII, PolI, RecJ). Key enzymes in the repair of 8-oxoguanine lesions resulting from the oxidation of DNA are the bifunctional glycosylases, such as MutM, MutY or EndoVIII. The first activity of these enzymes is to remove oxidized or ring-saturated bases while the second activity is to remove the resulting deoxyribose residue, generating a 3′-phosphate end (Krwawicz et al., 2007). This 3′-P is converted to a 3′-OH by various enzymes/pathways and the lesion is repaired by enzymes (e.g. ExoIII, PolI, LigI etc.) in the short or long BER pathways (S-BER, L-BER). Interestingly, the <span class="yellow">B. burgdorferi</span> genome does not contain genes encoding homologues of MutM, MutY or EndoVIII, suggesting that it would be difficult for the cells to efficiently repair 8-oxoguanine sites using this pathway. <span class="yellow">B. burgdorferi</span> does harbour the genes encoding the enzymes Ung (monofunctional gylcosylase, BB0053), MutS (BB0797, BB0098), MutL (BB0211), ExoIII (BB0114), PolI (BB0548), LigI (BB0552) and RecJ (BB0254) (Fraser et al., 1997) which are involved in excision and repair, via S-BER or L-BER, of deaminated, alkylated, methylated or mismatched bases. Clearly, these repair systems in <span class="yellow">B. burgdorferi</span> do not seem as robust at defending against oxidation of DNA as those described in <span class="blue">E. coli</span>. The lack of key repair enzymes in this system may indicate that <span class="yellow">B. burgdorferi</span> DNA is not subjected to the same challenge from ROS as is <span class="blue">E. coli</span>.
In most bacteria, ROS-mediated damage to lipids (lipid peroxidation) is very unlikely because of the lack of polyunsaturated fatty acids (e.g. linoleic acid) (Imlay, 2003). When it does occur, lipid peroxidation is initiated by the attack of free radicals on polyunsaturated fatty acids which decreases the membrane fluidity and, if these reactions propagate, lipid peroxides and their degradation products (e.g. aldehydes) in turn could damage proteins (Gutteridge and Halliwell, 1990). This would dramatically affect the function of transmembrane proteins and membrane-bound lipoproteins involved in the maintenance of membrane potential and solute transport, decreasing cell survivability. In contrast to most bacteria, <span class="yellow">B. burgdorferi</span> membranes contained significant levels of unsaturated fatty acids, such as linoleic acid and linolenic acid, which were derived from the growth media (Table 1). Thus, it seemed possible that lipids and/or proteins rather than DNA are the primary targets of ROS in <span class="yellow">B. burgdorferi</span>. When Borrelia cells are treated with oxidants, the levels of linoleic acid decreased while other fatty acids remain unaffected (Table 2). In addition, HPLC-based assays demonstrated that, as linoleic acid concentrations decrease, MDA (a toxic lipid peroxide intermediate) increased (Fig. 3). When these cells were examined by electron microscopy, damage to the membranes (membrane ‘blebs’) was observed (Fig. 6). These data indicated that unlike most other bacteria, Borrelia membranes were damaged by oxygen radicals.
As <span class="yellow">B. burgdorferi</span> may be exposed to potentially harmful oxygen species at different stages of the infective cycle, the ability to protect its membrane lipids from ROS should be required for survival in the different host environments. In eukaryotic cells, where lipid peroxidation is a major consequence of oxidative attack, proteins which protect membranes have been well studied. For example, phospholipid hydroperoxide glutathione peroxidase, a member of the glutathione peroxidase family, has been identified in a variety of higher organisms. These enzymes are capable of reducing an assortment of hydroperoxy lipids, including oxidized phospholipids and cholesterol esters (Ursini and Bindoli, 1987), and protect complex membranes from oxidative damage. Much less is known about the enzyme(s) responsible for protecting unsaturated lipids from ROS in prokaryotes. In <span class="yellow">H. pylori</span>, lipid hydroperoxide levels in ahpC mutants are approximately three times higher than in wild-type cells, suggesting a role for AhpC in reducing organic peroxides (Wang et al., 2006a,b). In addition, purified AhpC has been shown to reduce linoleic acid hydroperoxide in vitro (Baker et al., 2001). Because these antioxidant enzymes promote the in vivo survival of cells when challenged with ROS, enzymes for the reduction of lipid peroxides in Borrelia need to be identified.
Interestingly, Borrelia cells grown aerobically showed signs of membrane damage similar to those observed in cells exposed to various oxidants. The amount of MDA in untreated aerobically grown cells was equivalent to that observed in treated cells (Fig. 3). Also, EM indicated that cells grown microaerobically or aerobically had significantly more membrane damage than cells grown anaerobically (Fig. 6). This indicated that Borrelia membranes can be damaged simply by exposure to physiologically relevant concentrations of dissolved oxygen. In exponentially growing <span class="blue">E. coli</span>, Imlay and Fridovich (1991) have shown that both O2- and H2O2 are generated by the autoxidation of components of the respiratory chain during oxygen metabolism. In contrast, <span class="yellow">B. burgdorferi</span> is very metabolically limited and has no enzymes for the TCA cycle or respiration. Therefore, it seems more likely that sources of ROS are exogenous (e.g. innate immune response in the mammalian host) rather than endogenous. In addition, the current practice of growing <span class="yellow">B. burgdorferi</span> under atmospheric oxygen could itself be unintentionally compromising cell integrity during in vitro manipulations.
Analyses of the <span class="yellow">B. burgdorferi</span> genome indicates that only a few genes encoding putative oxidative stress/intracellular redox proteins (SodA, NapA, BosR, CoADR, Trx and TrxR) (Fraser et al., 1997) are present, compared with other bacterial pathogens, including other pathogenic spirochetes (e.g. Treponema, Leptospira). Despite this apparent ‘deficiency’ in the number of ROS-protective enzymes, <span class="yellow">B. burgdorferi</span> cells appear to be able to cope with physiologically relevant levels of ROS. There are several factors that would contribute to this phenomenon: (i) as <span class="yellow">B. burgdorferi</span> does not harbour the genes encoding respiratory enzymes nor metabolize oxygen, it seems unlikely that significant levels of ROS are generated via the incomplete reduction of O2 during cellular metabolism. This would suggest that potential ROS challenges to <span class="yellow">B. burgdorferi</span> would come almost completely from extracellular sources with little contribution from intracellularly generated ROS; (ii) owing to a lack of understanding of the physiological conditions in vector and host tissues infected with <span class="yellow">B. burgdorferi</span>, it is difficult to assess the levels and/or sites of the potential ROS challenge during the infective cycle and (iii) most importantly, analyses of the potential targets for ROS in <span class="yellow">B. burgdorferi</span> strongly suggested that the major targets of oxidative damage are different and perhaps less extensive in this bacterium than in other bacterial pathogens (e.g. <span class="blue">E. coli</span>). Taken together, this suggests that <span class="yellow">B. burgdorferi</span> would be innately more resistant to ROS and require a less extensive repertoire of enzymes to protect the cells from oxidative damage.<br><br>Experimental procedures
Strains, growth conditions and reagents
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 and strain B314 (Sadziene et al., 1993) were grown in modified BSK-II (Barbour, 1984) medium at 34°C under an atmosphere of 0–20% O2 with 5% CO2 and the balance N2. Cells density was determined using a dark-field microscope. All reagents were purchased from Sigma Chemicals, St. Louis, MO unless stated otherwise. <span class="blue">E. coli</span> strain Top10, strain TA4315 (ahpCF) (Storz et al., 1989) and CM1319 (mutM) (Bridges et al., 1996) were grown in Luria–Bertani (LB) at 37°C with shaking.<br><br>Per cent survivability assays
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 was grown to a cell density of 5 × 107 cells ml−1 in BSK II under microaerobic (3% O2) conditions, the culture split and the cells treated with varying concentrations of oxidants (0–50 mM t-butyl hydroperoxide or H2O2) at 34°C for 4 h. After the incubation, cells were diluted in fresh BSK II, plated on BSK plates and incubated 7–14 days at 34°C. Per cent survivability was calculated as the number of colonies on the treated plates versus the number of colonies on the untreated plates.<br><br>Determination of the spontaneous mutation rate
To determine spontaneous resistance to coumermycin A1, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under microaerobic conditions to a cell density of 5 × 107 cells ml−1 and treated with 5 mM H2O2 for 1 h at 34°C. The cells were plated on BSK plates containing 0 or 250 ng ml−1 coumermycin A1 and incubated 7–14 days at 34°C. The resistance frequency was calculated as the number of colonies that are CouR per total number of cells plated.<br><br>Measurement of DNA base lesions
<span class="yellow">Borrelia burgdorferi</span> B31A3 cells were grown in BSK-II under microaerobic conditions to a cell density of 5 × 107 cells ml−1, treated with various oxidants (1 or 10 mM t-butyl hydroperoxide, 1 or 10 mM H2O2, 10 mM paraquat or lipoxidase) for 4 h and total DNA was isolated using Wizard Genomic DNA Purification Kit (Promega Corp., Madison, WI). The number of base lesions was determined using the DNA Damage Quantification Colorimetric Assay kit (Oxford Biomedical Research, Oxford, MI) following the manufacturer's protocol. Briefly, 500 ng of DNA was mixed with an equal volume of 10 mM biotinylated aldehyde reactive probe (ARP) reagent and incubated for 1 h at 37°C. The DNA-ARP product was ethanol-precipitated using glycogen as a carrier, washed three times with 70% ethanol and resuspended in Tris-EDTA to give a final concentration of 0.5 µg ml−1. The DNA-ARP product was allowed to bind to the wells of 96-well microplate overnight at 37°C. After the binding, the wells were washed four times with TPBS (137 mM NaCl, 2.7 mM KCl, 10 mM Na3HPO4, 2 mM KH2PO4, 0.5% Tween 20, pH 7.4). The HRP-streptavidin conjugate was diluted to 0.5 µg ml−1 in Assay Buffer (0.15 M NaCl, 10 mM NasHPO4, 1.5 mM KH2PO4, 2.5 mM KCl, 5 mg ml−1 BSA, 0.1% Tween, pH 7.5), 100 µl was added to each well and the plate incubated at 100 r.p.m. for 1 h at room temperature (RT). After incubation, the wells were washed four times with TPBS, 100 µl of substrate was added to each well and incubated for 1 h at 37°C. The reaction was then quenched with 100 µl of 1 M sulphuric acid and the reaction was monitored at 450 nm. The number of aldehyde reactive probe (DNA base lesions) per 105 bp DNA was determined using a standard curve. As a control, <span class="blue">E. coli</span> TA4315 cells were grown in minimal media to OD600 of 0.4, treated with 0 or 100 µM H2O2 for 30 min and DNA isolated. The number of base lesions was determined as described above. <span class="yellow">B. burgdorferi</span> strain B314 cells were grown under microaerobic conditions and DNA isolated. The number of DNA lesions was determined as described.
To determine the amount of 8-oxoguanine in <span class="yellow">B. burgdorferi</span> DNA, cells were grown and treated as described above and DNA isolated. The DNA was converted to single-stranded DNA by boiling the sample for 5 min followed by rapid chill on ice. The DNA was then digested with nuclease P1 for 2 h and then treated with alkaline phosphatase for 1 h following manufacturer's protocols. The resultant mixture was then centrifuged for 5 min at 6000 g and the supernatant used for the 8-oxoguanine ELISA assay (Oxford Biomedical Research, Oxford, MI). As a control, <span class="blue">E. coli</span> CM1319 (mutM) cells were grown in LB to OD600 of 0.4, treated with 0 or 100 µM H2O2 for 1 h and DNA isolated. The concentration of 8-oxoguanine was determined as above.<br><br>Lipid analyses
To determine fatty acid content in <span class="yellow">B. burgdorferi</span> total membranes, <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under anaerobic conditions to a cell density of 5 × 107 cells ml−1, harvested by centrifugation (5000 g, 15 min, 4°C) and washed two times with HN (20 mM NaCl, 50 mM Hepes, pH 7.6) buffer. Cell pellets were analysed for fatty acid composition by fatty acid methyl ester (FAME) gas chromatography (Lipid Technologies, Austin, MN) and results are reported as percentage of total fatty acid content. To determine the effects of oxidants on fatty acid composition, a 1.5 l culture of <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under microaerobic conditions to a cell density of 5 × 107 cells ml−1. The culture was split into 500 ml aliquots, the first was treated with 1 mM t-butyl hydroperoxide, the second with 0.25 mg of lipoxidase (17 700 units) and the third was untreated. All cultures were incubated for 12 h at 34°C. Cells were harvested by centrifugation (5000 g, 4°C, 15 min) and washed three times with HN buffer. The fatty acids present in the cell pellets were analysed by FAME gas chromatography (Industrial Laboratory, Wheat Ridge, CO). Fatty acids were reported as percentage of total cell mass.<br><br>Measurement of MDA
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 was grown aerobically, microaerobically and anaerobically as described above to a cell density of 5 × 107 cells ml−1. The microaerobic culture was split and treated with either 5 mM AAPH [2,2′-azobis(2-methylpropionamidine) dihydrochloride] or 250 mg of lipoxidase for 4 h at 34°C. To measure the amount of MDA, the cells were derivatized with thiobarbituric acid and analysed by HPLC as described by Seljeskog et al. (2006). After the incubation, all cells were harvested by centrifugation (1000 g, 5 min, 4°C) and washed three times with HN buffer. Each sample was resuspended in 50 µl of HN, mixed with 150 µl of 0.1 N perchloric acid, 150 µl of 40 mM thiobarbituric acid and 35 µl of 20% SDS, vortexed and heated at 97°C for 60 min. After cooling at −20°C for 20 min, 300 µl of methanol and 100 µl of 20% trichloroacetic acid was added and the samples were mixed vigorously and centrifuged (13 000 g, 6 min). The samples (10 µl) were then analysed with an Agilent Technologies 1200 series HPLC system using a C18 4.6 × 150 mm column with mobile phase 72:17:11 (50 mM KPO4, pH 6.8 : methanol : acetonitrile). Absorbance was monitored at 532 nm. Pure MDA standards (0–10 µM) were prepared in methanol for comparison. As a negative control, <span class="blue">E. coli</span> Top10 cells were grown to OD600 of 0.4, treated with 0 or 5 mM AAPH for 30 min and MDA measured as above. As a positive control, <span class="yellow">mouse</span> myeloma SP2/O cells were cultured with HYQ-CCM1 (HyClone) medium at 37°C in a humidified 5% CO2 atmosphere, treated with 1 mM AAPH at 37°C for 4 h (Chotimarkorn et al., 2005) and MDA measured as above.<br><br>Identification of lipid damage using Diphenyl-1-pyrenylphosphine fluorescent stain
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 cells were grown microaerobically as described above until a cell density of 5 × 107 cells ml−1 was obtained. The culture was divided into two equal aliquots and the cells treated with 5 mM AAPH for 4 h at 34°C. After the incubation, all cells were harvested by centrifugation (1000 g, 5 min, 4°C) and washed three times with HN buffer. To visualize Borrelia cells, the cells were stained with PKH26 Red Fluorescent Cell Linker Dye (Sigma Aldrich) following the manufacturer's protocol. To visualize the lipid hydroperoxides in the cell membrane, the cells were counterstained with DPPP (Cayman Chemicals, MI) (Okimoto et al., 2000). Briefly, after cells were stained with the Red Fluorescent dye, 2 ml of <span class="yellow">rabbit</span> serum was added to stop the reaction and the mixture incubated for 1 min at RT. Next, 4 ml of HN buffer was added, the cells harvested by centrifugation (1000 g, 10 min, RT) and washed three times with HN buffer. The cells were then resuspended in 1 ml of HN buffer, incubated at 34°C for 5 min and 30 µl of 2.5 mM DPPP was added. The incubation was then continued at 34°C for 5 min in the dark. After incubation, the mixture was centrifuged (1000 g, 10 min, RT) and the cells washed three times with HN buffer. The cells were then resuspended in 100 µl HN buffer and observed by fluorescence microscopy with excitation wavelengths 551 nm (Red Fluorescent) and 351 nm (DPPP), and emission wavelengths 567 nm (Red Fluorescent) and 380 nm (DPPP). As a positive control, <span class="yellow">mouse</span> myeloma SP2/O cells were cultured with HYQ-CCM1 (HyClone) medium at 37°C in a humidified 5% CO2 atmosphere, treated with 1 mM AAPH at 37°C for 4 h (Chotimarkorn et al., 2005) and stained as described above. As a negative control, <span class="blue">E. coli</span> Top10 cells were grown to OD600 of 0.4, treated with 0 or 5 mM AAPH for 30 min and stained as described above.<br><br>Electron microscopy
<span class="yellow">Borrelia burgdorferi</span> cells were grown under microaerobic or anaerobic conditions to a cell density of 5 × 107 cells ml−1. The microaerobic cultures were split and treated with lipoxidase for 4 h at 34°C. Cells were harvested by centrifugation (5000 g, 10 min, 4°C), washed and resuspended in HBSS (Lonza Group Ltd, Switzerland). The cells were fixed with Karnovsky's phosphate for 5 min at RT adsorbed to Formvar/carbon-coated grids (Ted Pella, Redding, CA) for 5 min and washed three times in H2O. The grids were stained with 1% ammonium molybdate and allowed to air-dry. Samples were examined using a Hitachi H7500 electron microscope (Hitachi High Technologies America, Pleasanton, CA).<br><br><br><br><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="yellow">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="yellow">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="yellow">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="yellow">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="yellow">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="yellow">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="yellow">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="yellow">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="yellow">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="yellow">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="yellow">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="yellow">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="yellow">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="yellow">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="yellow">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="blue">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="yellow">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="yellow">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="yellow">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="yellow">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
<h3>pmcA2658668</h3>A novel DNA-binding protein modulating methicillin resistance in <span class="yellow">Staphylococcus aureus</span>
Abstract
Background
Methicillin resistance in <span class="yellow">Staphylococcus aureus</span> is conferred by the mecA-encoded penicillin-binding protein PBP2a. Additional genomic factors are also known to influence resistance levels in strain specific ways, although little is known about their contribution to resistance phenotypes in clinical isolates. Here we searched for novel proteins binding to the mec operator, in an attempt to identify new factor(s) controlling methicillin resistance phenotypes.<br><br>Results
Analysis of proteins binding to a DNA fragment containing the mec operator region identified a novel, putative helix-turn-helix DNA-binding protein, SA1665. Nonpolar deletion of SA1665, in heterogeneously methicillin resistant <span class="yellow">S. aureus</span> (<span class="yellow">MRSA</span>) of different genetic backgrounds, increased methicillin resistance levels in a strain dependent manner. This phenotype could be fully complemented by reintroducing SA1665 in trans. Northern and Western blot analyses, however, revealed that SA1665 had no visible influence on mecA transcription or amounts of PBP2a produced.<br><br>Conclusion
SA1665 is a new chromosomal factor which influences methicillin resistance in <span class="yellow">MRSA</span>. Although SA1665 bound to the mecA promoter region, it had no apparent influence on mecA transcription or translation, suggesting that this predicted DNA-binding protein modulates resistance indirectly, most likely through the control of other genomic factors which contribute to resistance.<br><br><br><br>Background
Methicillin resistant <span class="yellow">S. aureus</span> (<span class="yellow">MRSA</span>) are an ever increasing threat, both in clinical settings and more recently as an emerging community acquired pathogen. Their invasiveness and pathogenesis relies on a variable arsenal of virulence factors, paired with resistance to virtually all β-lactams and their derivatives. Their ability to rapidly generate resistance to other unrelated classes of antibiotics, or to take up additional resistance determinants, severely hampers therapy and eradication.
In <span class="yellow">S. aureus</span>, methicillin resistance is conferred by an acquired, β-lactam-insensitive penicillin-binding protein (PBP), PBP2a [1-4]. PBP2a is encoded by mecA, which is divergently transcribed from its cognate regulators, mecR1 (sensor/signal transducer) and mecI (repressor). If mecR1-mecI are absent or truncated, transcriptional control of mecA is taken over by the structurally similar blaZ (penicillinase) regulatory elements blaR1/blaI, if present. In the absence of both regulatory loci, mecA is constitutively transcribed [5,6]. In the presence of β-lactams, the transmembrane sensor/signal transducers BlaR1/MecR1, undergo a conformational change, followed by autoproteolytic cleavage of the n-terminal cytoplasmic domain, leading to the activation of the cytoplasmic peptidase and subsequent dissociation of the repressor due to proteolytic degradation [7-9]. However, the signal transduction cascade of this regulatory system has still not been completely elucidated.
Oxacillin resistance levels conferred by mecA are strain specific and can vary greatly, with oxacillin minimal inhibitory concentrations (MICs) of different strains ranging from phenotypically susceptible levels, as low as 1 μg/ml up to extremely high values of > 500 μg/ml. Methicillin resistance is also generally expressed heterogeneously. Heterogeneously resistant <span class="yellow">MRSA</span>, when exposed to β-lactam antibiotics, segregate highly resistant subpopulations, which are much more resistant than the majority of the cells [10]. The frequency of highly resistant subclones generated is often well above the spontaneous mutation frequency, and once selected high level resistance often remains stable, even in the absence of selective pressure. There is currently no satisfactory genetic model which explains how these higher resistance levels are triggered or selected and exactly what factors are functionally responsible for the increased resistance in clinical isolates. Methicillin resistance levels are known to not directly correlate with mecA transcription or levels of PBP2a produced [11,12]. However, resistance levels can be manipulated by environmental conditions, such as temperature, pH, osmolarity, and medium composition [13,14].
It has been shown experimentally, that in addition to mecA, methicillin resistance depends on the correct interplay of a multitude of genomic factors, termed fem/aux factors, including genes involved in peptidoglycan precursor formation, composition and turnover; teichoic acid synthesis; and genes of unknown or poorly characterised functions [15-18]. In addition to structural genes, many regulatory loci have also been shown to influence resistance levels, including global regulators of virulence factor production such as the quorum sensing agr system, the staphylococcal accessory regulator SarA and the alternate sigma factor σB [19,20]; regulators of metabolism, such as the catabolite control protein A (CcpA) [21]; and the VraSR two-component sensor transducer, which induces the cell wall stress stimulon in response to cell wall active antibiotic challenge [22].
The vast MIC differences between <span class="yellow">MRSA</span> strains, the population heterogeneity within single strains and the dependence of resistance levels on external factors are reflected in these many structural genes and global regulators, which can influence resistance levels.
While typically considered nosocomial pathogens, new faster growing and apparently more virulent <span class="yellow">MRSA</span> have begun spreading in the community. Interestingly, these emerging strains often express very low methicillin resistance, e.g. the <span class="yellow">MRSA</span> clone spreading amongst intravenous drug users in the Zurich area, which has an in vitro doubling time of 25 min, but oxacillin MICs of only 0.5 to 4 μg/ml [23]. This particular clone's low-level resistance is partially due to a promoter mutation, leading to tight repression of mecA, but resistance levels appear to be mainly restricted by unknown factors within its genomic background [12].
To identify potential factors involved in mecA regulation or methicillin resistance levels in such an extremely low level resistant <span class="yellow">MRSA</span>, we performed DNA-binding protein purification assays, using the mecA operator region as bait. A novel, uncharacterized protein, SA1665, was found to bind to this DNA fragment, and shown to increase methicillin resistance levels when deleted.<br><br>Results
Identification of SA1665
<span class="yellow">MRSA</span> strain CHE482 is the type strain for the so-called "drug clone" spreading amongst intravenous drug users in the Zurich area [12,23]. This strain carries mecA and expresses PBP2a, but appears phenotypically methicillin susceptible by conventional phenotypic tests. However, like most other low-level resistant <span class="yellow">MRSA</span>, it can segregate a small proportion of higher resistant subclones in the presence of β-lactams. We hypothesized that regulation of methicillin resistance in such low-level resistant clonal lineages may differ qualitatively from classical heterogeneously- or highly-resistant <span class="yellow">MRSA</span>.
A DNA-binding protein purification assay was performed to identify new potential factors involved in the regulation of mecA/PBP2a. The mecA/mecR1 intergenic DNA region, including the 5' 9 bp of mecR1 and the first 52 bp of mecA, was used as bait against crude protein extract from strain CHE482. Proteins binding to this DNA fragment were analysed by SDS-PAGE. Even though CHE482 contained BlaI, which is known to bind to the mec operator, this band could not be identified on gels due to co-migrating, non-specific bands the same size as BlaI (14.9 KDa) that bound to both the DNA-coated and uncoated control beads. The most prominent protein band of ~16–20 kDa, isolated from DNA-labelled but not from control beads, was identified as the hypothetical protein SA1665 (N315 genome annotation [BA000018]) (Figure 1A). SA1665 encodes a predicted 17-kDa protein with an n-terminal helix-turn-helix (HTH) motif characteristic of DNA-binding transcriptional regulators. The amino acid sequence of SA1665 showed 100% identity amongst <span class="yellow">S. aureus</span> database sequences and 97–98% identity amongst other staphylococci, including <span class="yellow">S. haemolyticus</span>, <span class="yellow">S. epidermidis</span> and <span class="yellow">S. saprophyticus</span>, indicating that SA1665 is highly conserved. Conversely, there were no orfs highly similar to SA1665 found in other bacterial species, with the most similar sequences found in <span class="yellow">Bacillus licheniformis</span> DSM13 and <span class="yellow">Desulfitobacterium hafniense Y51</span>, which shared only 64% and 59% similarity, respectively.<br><br>Electro mobility shift assays (EMSA)
EMSA was used to confirm binding of SA1665 to the mec operator region. Crude protein extracts of <span class="yellow">E. coli strain BL21</span>, carrying the empty plasmid (pET28nHis6) or pME20 (pET28nHis6-SA1665) which expressed nHis6-SA1665 upon induction with IPTG, were incubated with the 161-bp biotinylated-DNA fragment previously used as bait in the DNA-binding protein assay. A band shift was observed with extracts from the strain expressing recombinant nHis6-SA1665 but not from the control strain carrying the empty plasmid. Several bands resulted from the shift, which is most likely due to protein oligomerisation (Figure 2A). The specificity of the gel shift was also demonstrated by the addition of increasing concentrations of purified nHis6-SA1665 protein to the biotinylated-DNA fragment (Figure 2B). Band-shift of the biotinylated DNA was inhibited in the presence of specific competitor DNA but not by the presence of the non-specific competitor DNA, confirming that nHis6-SA1665 had a specific binding affinity for the 161-bp DNA fragment.<br><br>Effect of SA1665 deletion on β-lactam resistance
To analyse the effect of SA1665 inactivation on methicillin resistance, nonpolar markerless deletions of SA1665 (Figure 1B) were constructed in a selection of clinical <span class="yellow">MRSA</span> isolates, which varied in their genetic background, SCCmec type, and mecA regulation [24]. Strain CHE482, belongs to clonal complex CC45 and sequence type ST45, and contains a novel SCCmec (SCCmecN1 [23]); while strains ZH37 (CC45/ST45) and ZH73 (CC22/ST22) contain type IV SCCmecs. All three of these strains have truncated mecI/mecR1 regulatory loci but intact BlaI/BlaR1 loci controlling mecA expression. Strain ZH44 (CCT8/ST8) contained a type A mec complex (mecI-mecR1-mecA) within a type II SCCmec, and had no β-lactamase locus; so mecA was only under the control of its cognate regulators MecI/MecR1.
Deletion of SA1665 increased oxacillin resistance in all mutants compared to their corresponding parent strains, as demonstrated on oxacillin gradient plates (Figure 3A); with mutants ΔCHE482 and ΔZH37 approximately doubling in resistance and ΔZH44 and ΔZH73 expressing considerably higher resistance. Population analysis resistance profiles of the mutants showed a distinct shift at the top of the curve, indicating that the higher resistance was due to increased basal oxacillin resistance levels (Figure 3B). Strains CHE482/ΔCHE482 and ZH37/ΔZH37 had very similar resistance profiles, despite having different SCCmec elements, suggesting that it was their common clonal background (CC45) that determined their resistance levels and the extent of resistance increase upon SA1665 deletion.
Growth curve analyses showed that deletion of SA1665 slightly reduced the growth rate of all strains tested (Figure 3C). Wild type growth rates were restored upon complementation (data not shown).<br><br>Resistance complementation
Plasmids pME26 and pME27 were constructed for complementation of the deletion mutants. Both plasmids contained the SA1665 orf along with its own promoter and transcriptional terminator. Strains ΔCHE482, ΔZH37, and ΔZH73 were complemented with pME26, and intrinsically kanamycin resistant strain ΔZH44 was complemented with pME27. Wild type-like resistance levels were restored in all mutants by introduction of the complementing plasmids, as shown by gradient plates (Figure 3A).<br><br>Transcriptional analyses
Primer extension, using the 5'-biotinylated primer me97, identified two potential SA1665 transcriptional start sites (TSS), 76-nt and 139-nt upstream of the SA1665 ATG start codon (Figure 4A). Predicted σA promoter consensus -10/-35 box sequences were located upstream of both TSS (Figure 4B). Identical TSS were also identified using the downstream primer me98 (data not shown).
Northern blot analysis was used to investigate SA1665 expression and the influence of SA1665 deletion on mecA and mecR1 transcription. RNA samples taken from different time points over the growth curve of CHE482 showed that SA1665 was expressed strongly in early exponential phase at OD600 nm 0.25 and 0.5, then transcript levels decreased and were almost undetectable in early stationary phase at OD600 nm 4.0 (Figure 5A). In addition to the main transcript of ~0.46 kb, a weaker, larger transcript of ~0.6 kb was also visible, especially at later growth stages. Figure 5B shows the transcriptional behaviour of SA1665 when CHE482 cells were challenged with sub-inhibitory (4 μg/ml) and inhibitory (120 μg/ml) concentrations of cefoxitin. These results showed that low levels of cefoxitin, such as those used to induce mecA/mecR1 transcription, appeared to slightly decrease SA1665 transcription after 30 min exposure, while larger, inhibitory concentrations caused even more significant alterations in the SA1665 transcriptional profile, making it similar to that normally seen in stationary phase growth. These results indicate that transcription of SA1665 may respond in some way to cell wall stress, rather than in direct response to the presence of β-lactams. This observation is based on relatively subtle changes in SA1665 transcription, especially at low concentrations of cefoxitin such as those required for mecA/mecR1 induction. Since deletion of SA1665 has been shown to increase β-lactam resistance, reduced SA1665 transcription in the presence of β-lactams may also provide some protection against β-lactam exposure.
Northerns also showed that, as expected, the SA1665 transcripts were absent from the deletion mutant (Figure 5C), and additional experiments demonstrated that wild type SA1665 transcription patterns were restored by complementation of ΔCHE482 with pME26 (data not shown). The effects of SA1665 deletion on directly up- and down-stream genes were also investigated. Northern blots of the neighbouring genes SA1664, SA1666 and SA1667, showed that expression of all three genes was very weak compared to that of SA1665. A weak transcript of about 3 kb was present in hybridizations probed with orfs SA1665-SA1667. This band decreased in size in the SA1665 mutant when probed with SA1666 and SA1667. One of the transcripts hybridising to the SA1664 probe also decreased in size by ~0.5 kb in the SA1665 mutant, suggesting that SA1665 was present on several transcripts of different lengths, including a high abundance monocistronic transcript and low abundance polycistronic transcripts (Figure 5C). Transcript abundance of both the upstream SA1666-SA1667 operon and the downstream SA1664-specific transcript all appeared to increase slightly in ΔCHE482. The significance of these subtle increases in transcription are unknown, however, polar effects from SA1665 deletion seem unlikely, based on the facts that all genes were still transcribed, their transcription levels all remained extremely low and the transcriptional terminator of SA1665 remained intact in the deletion mutant (Figure 1B).
Expression of mecR1 and mecA were analysed from RNA of uninduced and induced cultures of CHE482 and ΔCHE482. Cells were induced at OD600 nm 0.25 (Figure 5D) and 1.0 (data not shown) with sub-inhibitory concentrations of cefoxitin, to relieve BlaI-repression of mecA. mecR1, although truncated in CHE482, was still transcribed and had the same expression pattern as mecA, as both became derepressed over time and had the highest transcript levels after 30 min of induction. In the mutant ΔCHE482, transcripts of both mecA and mecR1' were unaffected by SA1665 deletion, indicating that SA1665 had no influence on their expression at either OD 0.25 (Figure 5D) or OD 1.0 (data not shown). SA1665 deletion also had no effect on mecA transcription or induction in strains ZH37, ZH44 and ZH73 (data not shown).<br><br>Western blot analysis
Mutants of CHE482 and of ZH44 and ZH73, which had the largest differences in oxacillin resistance levels, were analysed by Western blot analysis to determine if SA1665 affected production of PBP2a from mecA. As shown in Figure 5E, all pairs of wild type and mutant strains had similar amounts of PBP2a present both before and after induction with cefoxitin, indicating that SA1665 deletion did not alter amounts of PBP2a produced. Therefore it seems that SA1665 exerts no direct control over mecA or PBP2a expression.<br><br>
Discussion
Methicillin resistance in <span class="yellow">MRSA</span> is primarily dependent on the presence of the mecA gene, however, resistance levels are generally governed by strain-specific factors including mecA regulatory elements and other chromosomal fem/aux factors which either enhance or repress the expression of resistance. For instance, the very low-level methicillin resistance of the Zurich drug clone CHE482, was shown to be controlled by its genetic background [12] suggesting that it either contained or lacked certain fem/aux factors involved in controlling resistance expression. Many of the currently known fem/aux factors are directly or indirectly involved in cell wall synthesis and turnover, or envelope biogenesis, however there still remain factors of unknown function. Most of the currently known fem/aux factors reduce methicillin resistance levels when inactivated. A few genes, such as lytH, dlt, norG, sarV and cidA increase resistance levels upon inactivation or mutation. All of these genes, except norG, which is an efflux pump regulator, play a role in either autolysis or are important for cell physiology and growth [25-30]. Other genes increase β-lactam resistance upon overexpression, such as hmrA coding for a putative amidohydrolase, hmrB coding for a putative acyl carrier protein [31], or the NorG-controlled abcA multidrug efflux pump [28].
SA1665, a predicted DNA-binding transcriptional regulator, was found to bind to a DNA fragment containing the mecA promoter region. However, although this protein shifted the mecA operator/5' coding sequence, it did not appear to directly control mecA or mecR1 transcription or PBP2a production. Therefore its binding to the mecA region may have no specific regulatory function. Such interactions have been noted before, such as the HTH protein NorG, which was shown to bind specifically to norA, norB and norC promoters, but only transcription of norB was increased when NorG was overexpressed [28]. We have to postulate therefore that SA1665 may modulate β-lactam resistance in a mecA-independent manner, by controlling cellular functions affecting resistance levels. Experiments to determine the SA1665 regulon are ongoing. The impact of deleting SA1665 in <span class="yellow">MRSA</span> was extremely strain specific, underlining the importance of the genetic background in governing the final methicillin resistance levels of <span class="yellow">MRSA</span>, and demonstrating the large genomic variability between different strain lineages.<br><br>Conclusion
SA1665 is a previously uncharacterised DNA-binding protein that has a negative effect on β-lactam resistance in <span class="yellow">MRSA</span>. The SA1665 protein was identified in a DNA-binding protein purification assay, in which it bound to a DNA fragment covering the mec operator region. However, while nonpolar deletion of SA1665 was shown to increase oxacillin resistance levels in several heterogeneously resistant <span class="yellow">MRSA</span>, its deletion had no effect on mecA transcription or PBP2a production. Therefore the negative impact of SA1665 on methicillin resistance is most likely to be through the regulation of other chromosomal factors or cellular functions required for methicllin resistance.<br><br>Methods
Strains and growth conditions
Strains and plasmids used in this study are listed in Table 1. Clinical isolates are from the IMM collection in Zurich, Switzerland. Strains were grown at 37°C in Luria Bertani (LB) broth, shaking at 180 rpm, or on LB agar. Media were supplemented with the following antibiotics when appropriate: 25 or 50 μg/ml kanamycin, 10 μg/ml chloramphenicol, 5 or 10 μg/ml tetracycline, 100 μg/ml ampicillin. Concentrations of cefoxitin used for transcriptional induction were either sub-inhibitory (4 μg/ml) or inhibitory (120 μg/ml).<br><br>Susceptibility testing
Oxacillin resistance levels were compared by swabbing 0.5 McFarland cell suspensions across agar plates containing appropriate concentration gradients of oxacillin. For population analysis profiles, appropriate dilutions of an overnight culture, ranging from 100 to 108, were plated on increasing concentrations of oxacillin. Plates were incubated at 35°C and colony forming units per ml (cfu/ml) were determined after 48 h.<br><br>Binding-protein purification
Crude protein extracts were isolated from CHE482, grown under normal culture conditions until OD600 nm 1.5. Cells were harvested, resuspended in PBS (pH 7.4) and mechanically lysed using Lysing Matrix B (BIO 101 Systems) tubes and a FastPrep FP120 (BIO 101 Systems). Suspensions were clarified by centrifugation and supernatants, containing soluble cytoplasmic proteins, were transferred to Amicon Ultra centrifugal filter devices (Millipore) with a pore cut-off size of 10 kDa. Proteins were then washed and concentrated in 1× binding buffer (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, and 1 mM DTT, 0.5 M NaCl). Protein concentrations were measured by Bradford assay (BioRad Laboratories GmbH) [32]. Primer pair me36F/me36Rbiot (Table 2) were used to amplify a biotinylated mecA promoter/operator fragment, which was bound to streptavidin coated magnetic beads (Dynabeads M-280 Streptavidin, DYNAL BIOTECH) according to the manufacturer's instructions. Binding reactions, containing DNA-coated beads mixed with 100 μg of crude protein extract in 1× protein binding buffer (20 mM Hepes, pH 7.6, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20 (w/v), 30 mM KCl), 0.02 μg/μl poly d(I-C) and 2 ng/μl poly L-lysine, were incubated at room temperature for 30 min with constant rotation. Beads were then washed and binding-proteins eluted in elution buffer (1× protein binding buffer containing 2 M KCl). Eluted proteins were dialysed against water, concentrated by evaporation, and run on 15% SDS polyacrylamide gels. Gels were silver stained using the Protein Silver Staining kit (Amersham Biosciences AB) without the addition of glutaraldehyde. Protein bands were excised from gels and analysed by mass spectrometry (LC/ESI/MS/MS) at the Functional Genomics Centre, Zurich. The SA1665 protein sequence [BAB42933] was analysed by Blast search  and motif scan .<br><br>Expression of recombinant SA1665 protein
SA1665 was amplified using primer pair me65BamHI/me65XhoI (Table 2) and cloned in-frame into pET28nHis6 (unpublished, D. Frey). The resulting plasmid, pME20, was transformed into <span class="yellow">E. coli BL21</span> for expression of recombinant nHis6-SA1665 protein. To maximise the abundance of soluble protein produced, cultures were grown in osmotic shock medium at 37°C (1 g/l NaCl, 16 g/l tryptone, 10 g/l <span class="yellow">yeast</span>, 1 M sorbitol, 10 mM betaine, modified from [33]) to an OD600 nm of 0.5, cooled briefly on ice, then induced by adding 100 μM IPTG and growing overnight at 22°C. Crude soluble proteins were extracted using CelLyticB 2× cell lysis reagent (SIGMA). HIS-Select Cobalt Affinity Gel (SIGMA) was used to purify recombinant nHis6-SA1665 according to the manufacturer's instructions.<br><br>Electro mobility shift assay
For gel shift assays, 6 ng aliquots of the biotinylated-DNA fragment used for binding-protein purification were incubated with 0–250 ng of purified nHis6-SA1665 protein in 1× binding buffer (20 mM Hepes pH 7.6, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20 (w/v), 30 mM KCl) containing 0.05 μg/μl poly d(I-C) (Roche) and 5 ng/μl poly L-lysine (Roche). For control binding reactions, 130 × unlabelled mec operator DNA (amplified using primers me36F/me36R, Table 2) was used as a specific binding competitor and 6 ng of herring sperm DNA was used as unspecific competitor DNA. Binding was carried out at 22°C for 30 min. Samples were run on 6% native polyacrylamide gels, contact blotted onto positively charged nylon membrane and detected with the Biotin Chromogenic Detection Kit (Fermentas).<br><br>Primer extension
RNA was extracted from CHE482 cultures that were grown to OD600 nm 0.5, as previously described [12]. Primer extension reactions were performed using 20 μg of total RNA and 3 pmol of the 5'-biotin-labelled primers me97 and me98 (Table 2) using Superscript II reverse transcriptase (Invitrogen), according to the manufacturers instructions. Sequencing reactions were performed using the Thermo Sequenase cycle sequencing kit (U.S. Biochemicals). The Biotin Chromogenic Detection Kit (Fermentas) was used for biotin detection.<br><br>Markerless deletion of SA1665
In frame markerless deletions of SA1665, from the chromosomes of CHE482, ZH37, ZH44, and ZH73, were constructed using the pKOR1 allelic replacement system, as described by Bae et al. [34]. Primer pairs used to amplify the DNA fragments flanking SA1665, for recombination into pKOR1 were: me62attB1/me51BamHI and me62BamHI/me62attB2 (Table 2). All deletion mutants were confirmed by nucleotide sequencing over the deleted region, as well as by Southern blot analysis [35] and pulsed field gel electrophoresis (PFGE) [36].<br><br>Cloning of SA1665 for complementation
A 1533-bp DNA fragment, containing SA1665 together with 690-bp of upstream and 379-bp of downstream DNA, was amplified from strain CHE482 using primers me94BamHI/me94Asp718 (Table 2) and cloned into the <span class="blue">E. coli</span>/<span class="yellow">S. aureus</span> shuttle vectors pAW17 and pBUS1 [37], creating the complementing plasmids pME26 and pME27, respectively. Plasmids were electroporated into RN4220 [38] and then transduced into different strains using phage 80α.<br><br>Northern blot analysis
Strains were grown overnight in LB (Difco), diluted 1:200 and grown for another 3 h. This preculture was used to inoculate 150 ml (1:1000) of fresh prewarmed LB. Cells were then grown to OD600 nm 0.25 or 1.0 and either left uninduced or induced with cefoxitin 4 or 120 μg/ml. Cultures were sampled from both uninduced and induced cells at time point 0' before induction and at 10' and 30' (min) after induction. To monitor SA1665 expression over growth, separate cultures were also sampled at different growth stages corresponding to OD600 nm 0.25, 0.5, 1, 2, and 4. Total RNA was extracted as described by Cheung et al. [39]. RNA samples (10 μg) were separated in a 1.5% agarose-20 mM guanidine thiocyanate gel in 1× TBE running buffer [40], then transferred and detected as described previously [41]. Digoxigenin (DIG) labelled-probes were amplified using the PCR DIG Probe synthesis kit (Roche). Table 2 contains the list of primer pairs used for the amplification of SA1664, SA1665, SA1666, SA1667, mecR1 and mecA [42] probes. All Northern's were repeated at least two times, using independently isolated RNA samples.<br><br>Western blot analysis
Cells were cultured, as described for Northern blot analysis, to OD600 nm 1.0, then induced with cefoxitin 4 μg/ml. Samples were collected at time 0 (before induction), 10 and 30 min (after induction). Cells were harvested by centrifugation, resuspended in PBS pH 7.4 containing DNase, lysostaphin and lysozyme (150 μg/ml of each) and incubated for 1 h at 37°C. Suspensions were then sonicated and protein aliquots (15 μg) were separated on 7.5% SDS-polyacrylamide gels, blotted onto nitrocellulose membranes (Hybond) and stained with Ponceau to confirm equal protein loading. PBP2a detection was performed using monoclonal PBP2a antibody (1:20000) from the <span class="yellow">MRSA</span>-screen kit (Denka Seiken).<br><br>
Authors' contributions
ME carried out molecular genetic and microbiological studies and drafted the manuscript. BB participated in the design of the study and helped to draft the manuscript. NM participated in the design and coordination of the study, carried out molecular biological studies and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA334876</h3>The c1 genes of P1 and P7.
Abstract
The c1 genes of the heteroimmune phages P1 and P7 were sequenced and their products were compared. P7c1 expression was correlated with the translation in vitro of a protein whose predicted molecular weight (33,000 daltons) is indistinguishable from that of the P1c1 repressor. The c1 regions from both P1 and P7 were found to contain open reading frames capable of coding for a 283-amino acid protein whose predicted secondary structure lacks the helix-turn-helix motif commonly associated with repressor proteins. Two P1c1 amber mutations were localized to the 283-amino acid open reading frame. The P1c1 and P7c1 sequences were found to differ at only 18 positions, all but two of which alter the third position of the affected codon and do not alter the amino acid sequence of the protein. Plasmids expressing the c1 gene from either phage cause the repression of transcription from a cloned promoter situated upstream of P1c1.Images<br><br><br><br><br><br>
 Volme 7 Nmbe 19198             Nulei Acds    eserc <br><br> The cl genes of P1 and P7 <br><br> Francis A.Osborne, Sonja R.Stovall and Barbara R.Baumstark* <br><br> Department of Biology, Georgia State University, Atlanta, GA 30303, USA <br><br> Received July 11, 1989; Revised and Accepted August 29, 1989  EMBL accession nos X16005, X16006 <br><br> ABSTRACT <br><br> The cl genes of the heteroimmune phages P1 and P7 were sequenced and their products were compared. P7cl expression was correlated with the translation in vitro of a protein whose predicted molecular weight (33,000 daltons) is indistinguishable from that of the Plcl repressor. The cl regions from both P1 and P7 were found to contain open reading frames capable of coding for a 283-amino acid protein whose predicted secondary structure lacks the helix-tum-helix motif commonly associated with repressor proteins. Two Plcl amber mutations were localized to the 283-amino acid open reading frame. The Plcl and P7cl sequences were found to differ at only 18 positions, all but two of which alter the third position of the affected codon and do not alter the amino acid sequence of the protein. Plasmids expressing the cI gene from either phage cause the repression of transcription from a cloned promoter situated upstream of Plcl. <br><br> INTRODUCTION <br><br> The cl genes of the plasmid prophages P1 and P7 code for repressor proteins that are required for the establishment and maintenance of lysogeny (reviewed in 1). A protein corresponding to the Plcl repressor has been isolated and shown to be a sequence-specific DNA binding protein that recognizes several widely dispersed sites on the phage DNA (2-7). The consensus DNA sequence recognized by the Plcl repressor (ATTTATTAGAGCA[A/T]T) contains no discernable bilateral symmetry, a feature that is highly unusual among prokaryotic operator sites. <br><br> P1 and P7 are heteroimmune; that is, each phage is able to establish a lytic infection on a lysogen of the other phage. In this sense, their relationship is analogous to that of phage X and 434, which differ in the DNA specificity of their cI repressor proteins (8). However, genetic studies indicate that Plcl and P7cl can be crossed into the heterologous phage without affecting the immunity specificity of the recipient (9). The basis for P1/P7 heteroimmunity has been localized to a second regulatory gene, c4, that is unlinked to cl. The c4 gene products prevent the expression of antireb, a closely linked gene that interferes with cl-mediated repression (10, 11). According to current models, P1/P7 heteroimmunity results from the inability of the c4 repressor of one phage to prevent antireb expression from the heteroimmune phage genome (10, 11). <br><br> Because the cl genes of P1 and P7 are genetically interchangeable, it is anticipated that the two gene products carry out similar or identical regulatory functions. The studies presented in this paper were undertaken to investigate the biochemical basis for the apparent genetic identity of the two cl genes. In this paper, we present the DNA sequence of the cl genes of P1 and P7 and the predicted amino acid sequence of the cl repressor proteins. We report that Plcl and P7cl code for proteins of identical size (283 amino acids) and <br><br> Nucleic Acids Research <br><br> Volume 17 Number 19 1989 <br><br> 767 1 <br><br> (r IRL Press <br><br> Nucleic Acids Research <br><br> nearly identical sequence. We report further that both repressors prevent the expression of a promoter located immediately upstream of the Plc 1 open reading frame, an observation that confirms their functional similarity and suggests an autoregulatory role for the two proteins. Analysis of the secondary structure predicted by the open reading frames does not reveal the characteristic helix-turn-helix (12) or other motifs commonly associated with DNA binding proteins. <br><br> MATERIALS AND METHODS Bacterial and phage strains. <br><br> <span class="blue">E. coli</span> K336 is a SuO derivative of K140 (13). <span class="blue">E. coli</span> CB454 is a recA-, lacZderivative of K-12 (14). P1 + is described by Scott (13). P7+ is the strain of Smith (15), as described by Scott (16). P7cl. 1 contains a missense mutation in the cl gene (17). The P7 phage strains and the cl amber mutant phage strains PIci .245Cm, Plc1. 169 and P Ic .55 (11) were generously provided by June Scott. Enzymes and reagents. <br><br> Restriction enzymes, T4 DNA ligase and polymerase, and the Klenow fragment of <span class="blue">E. coli</span> DNA polymerase were purchased from Boehringer Biochemicals or New England Biolabs and reactions were carried out according to the manufacturers' instructions. DNA sequencing kits and in vitro transcription-translation kits were purchased from Bethesda Research Laboratories and Amersham Corporation, respectively. Synthetic oligonucleotides to be used as sequencing primers were prepared on an Applied Biosystems DNA synthesizer. Plasmid construction. <br><br> pBRB7.2. pBRB7.2 (2) contains a 3.2 kb EcoRI/PvuII fragment from the cl region of P1 (Figure 1) inserted into the 2.3 kb EcoRIlPvuII fragment of pBR322 that contains the origin of replication and the f3-lactamase gene. <br><br> pFA02. P7 plasmid DNA was digested with PvuII, ligated to similarly digested pBR322, and transformed into <span class="blue">E. coli</span> K336. Ampicillin-resistant colonies were screened for cl activity by cross-streak complementation analysis against P7cl.1 (18). pFAO2 contains a 3.5 kb insert of P7 DNA. The fragment was localized to the cl region of the P7 genome by Southern hybridization against P1 and P7 DNA that had been digested with BamHI and BglII (data not shown). <br><br> pBRBJ69. 1 and pBRB55. 1. The PI cI open reading frame was previously localized to a 2.6 kb EcoRlIBamHI fragment derived from PlEcoRI-7 (2). This fragment also contains the wildtype allele for the conditional lethal mutation am43 (19, 20). To clone the cl reading frame from the amber mutant phage Plc1.169 and P Ic .55, we digested phage DNA with EcoRI and BamHI, ligated the digestion products into similarly digested pBR322, and transformed the ligation mixture into <span class="blue">E. coli</span> K336. Ampicillin-resistant, tetracyclinesensitive colonies were screened by cross-streak complementation analysis for their ability to support the growth of Plam43. Plasmid DNA isolated from complementation-positive cells was shown by agarose gel electrophoresis to carry plasmids containing the 2.6 kb EcoRIlBamHI fragment from the cl region. The cl mutant open reading frames were placed under the control of normal regulatory signals present in the cI region by digesting the cl.55 and cl. 169-containing plasmids with BamHI and PvuII and inserting a 601 bp BamHI/PvuH fragment containing the cl promoter region (2). The resulting plasmids, pBRB55.1 and pBRB169.1, respectively, contain the 3.2 kb EcoRllPvuIl fragment analogous to that present in pBRB7.2 (Figure 1). <br><br> pBCB2. 13-2.18. To identify cl-repressible promoters, we introduced selected fragments <br><br> 7672 <br><br> Nucleic Acids Research <br><br> 4 f      -         -   + <br><br> I     4                        P1 4-    4-          4 <br><br> ECAR     H        C      B  R        P <br><br> 4   4.4,~   4,  1  1 4.  I  4,  I 11 <br><br> t        t              t  t        t E G R    N  E            B  R        P <br><br> 4           4  -  <br><br> 4<br><br> 1 '  '110z I  I  I  I  I     I   1 <br><br> 3.2    1.5       1.0       0.5        0 <br><br> P7 <br><br> pBRB7.2 <br><br> -------------------cl1-------------<br><br> r ~ / >'                                                                   pFAO2 <br><br> *-lacZ                           pBCB2.13 <br><br> <-lacZ Z          pBCB2.16 <-lacZX                          pBCB2.18 <br><br> Fig. 1. The cl regions of P1 and P7. A restriction map is indicated by the solid line in the upper part of the figure. Sites for EcoRI (E), Pvul (P), NnrI (N), Bgll (G), BamHI (B), and EcoRV (R) are shown. The sequencing strategy is indicated by the horizontal arrows. Letters and arrows above and below the map refer to sites and sequence analysis for P1 and P7, respectively. The size of this region (in kilobase pairs) is indicated below the map. The DNA fragments present in selected plasmids are illustrated by boxes at the bottom of the figure. The dashed line reveals the approximate position of the cl gene (2). The sites of the -yb mutations introduced into pFA02 are indicated by asterisks. pFA02.16 and pFA02.26 contain insertions located 0.9 kb and 1.4 kb, respectively, from the PvuIl site at the left side of the map. The direction of the lacZ open reading frame in pBCB2.13-18 is indicated by the adjacent arrow. <br><br> from the cI region of P1 into pCB192, a promoter-probe vector containing promoterless copies of lacZ and galK extending in opposite directions from a multiple cloning site (21). The source of P1 DNA for these constructions was pZHA3, a derivative of pBRB7.2 that contains a HindIlI linker at the single EcoRV site located about 200 bps upstream of the cl open reading frame (Figure 1). pBCB2.13 contains a 460 bp fragment of P1 DNA extending from the EcoRV site to a Bgll site within the cI open reading frame. pBCB2. 16 contains a 130 bp fragment extending from the EcoRV site to a BamHI site located about 100 bps upstream of the cl open reading frame, while pBCB2.18 contains the region extending from this BamHI site to the Bglll site within the open reading frame (Figure 1). The orientation of the P1 DNA fragments within these plasmids was confirmed by restriction mapping and DNA sequencing. <br><br> To test for the regulation of promoter expression by cl, we transformed pBCB2.13 and its derivatives into CB454(pBRB7.152) and CB454(pFAO2.152), two strains that express P7cl and Pll, respectively, from the pCB192-compatible kanamycin-resistant vector pDPT152 (22). Cells harboring both plasmids were selected by their resistance to both ampicillin and kanamycin. lacZ expression was measured by the procedure of Miller (23). pBRB7.152 was generated by introducing PlEcoRI-7 into pDPT152. pBRB7.152 has sustained a spontaneous deletion within the EcoRI-7 fragment that results in the loss of 2.5 kb of P1 DNA from the far left side of the P1 genetic map, but retains the 3.2 kb PvuHlEcoRI fragment required for cl expression that is present in pBRB7.2 (Figure 1). <br><br> 7673 <br><br> z <br><br> I                011Z                                                                                                                                                                                              I <br><br> e., <br><br> Nucleic Acids Research <br><br> Table 1. Complementation of PIci .245 by plasmids that contain cI genes. <br><br> Phenotype             Efficiency            Relative <br><br> of                    of                    Efficiency of Plasmid              cI gene               Lysogeny              Lysogeny pBR322                                     1.5x 10-6             6.0x 10-6 pBRB7.2              Plcl+                 2.5 x 10-'            1.0 <br><br> pBRB55.1             Plcl-am               1.4 x 10-6            5.6 x 10-6 pBRB169.1            PlCl-am               2.1 x 10-6            8.4 x 10-6 pFAO2                P7cl+                 8.7 x 10-2            0.35 <br><br> pFAO2.16             P7c I-,               4.7 x 10-6            1.9 X 10-5 pFAO2.26             P7cI -1               3.1 x 10-6            1.2 x 10-5 <br><br> Complementation for lysogeny was carried out as described by Devlin et al. (26). The plasmids were carried by <span class="blue">E. coli</span> K336. Cells were grown to mid-log phase at 370 in LB containing 50 tig/m1 sodium ampicillin and infected with Plcl.245 at a multiplicity of infection of 5 in the presence of 50 mM CaC12. After 10 minutes, non-absorbed phage were removed by centrifugation and the infection was allowed to proceed for 2 hours at 370. The infected cells were plated on LB plates containing 50 pLg/ml sodium ampicillin, 50 pLg/ml chloramphenicol, and 40 mM sodium citrate. The efficiency of lysogeny is defined as the number of ApRCmR cells at the end of infection divided by the number of ApR cells present at the start of the infection. <br><br> To construct pFA02.152, we introduced a 2.8 kb EcoRV fragment of P7 DNA containing the cl gene (Figure 1) into the single EcoRI site of pDPT152 after it had been rendered blunt-ended by extension with T4 DNA polymerase. cl expression by cells harboring either pBRB7.152    or pFA02.152      was confirmed    by   measuring  their ability to  form chloramphenicol-resistant lysogens when infected with Plcl.245Cm. ,yb insertional mutagenesis. <br><br> Insertional mutagenesis of P7cl was carried out using the 'yb transposon of F (24) as described by Devlin et al. (25). <span class="blue">E. coli</span> W1485(pFA02) was mated with the F- strain MX648 and subsequently plated on ampicillin (to select for the plasmid) and streptomycin sulfate (to select for the recipient strain). Transconjugants which could support only lytic growth upon infection by P7cl .1 (as scored by cross-streak analysis; 18) were assumed to have lost cl-complementing activity and were characterized further. The positions of two cl - insertional mutations, carried by pFA02.16 and pFA02.26, were identified by restriction mapping (Figure 1). DNA sequencing. <br><br> DNA sequence analysis was carried out using the M13-dideoxy technique of Sanger et al. (26). Selected DNA fragments containing the cl wildtype or mutant genes were introduced into M13 mp8 or mp9. 18-nucleotide oligomers complementary to defined sequences within the cl gene were extended using the Klenow fragment of DNA polymerase in the presence of dideoxynucleotide triphosphates and analyzed by polyacrylamide-urea gel electrophoresis. The sequencing strategy is shown in Figure 1. RESULTS <br><br> Localization of the P7cJ gene. <br><br> Initial localization of the P7cl gene was undertaken by subjecting pFA02 to 'yb mutagenesis and determining the map position of inserts which destroy the ability of the plasmids to complement a P7cl - mutation (as determined by cross-streak analysis). pFA02 and the -yb insertion mutants were tested further by comparing their ability to complement a PIcI amber mutation with the complementation activity of plasmids containing cl genes isolated <br><br> 7674 <br><br> Nucleic Acids Research <br><br> B    C    D    E    F   G <br><br> *.4  <br><br> 68<br><br> 43<br><br> 29-     _     -    =I   =   Am~  W    _ <br><br> 18<br><br> 14  p <br><br> Fig. 2. In vitro transcription-translation of plasmids carrying the cl region of P1 and P7. Proteins encoded by selected plasmids were labeled with 35S methionine according to the procedure of DeVries and Zubay (27), using a commercial in vitro transcription/tramnslation kit from Amersham Corporation. The reaction mixtures were subjected to electrophoresis on a 12.5% SDS-polyacrylamide gel and the labeled proteins were visualized by autoradiography. The migration of 14C-labeled protein molecular weight standards (Bethesda Research Laboratories) is indicated at the left side of the figure. Plasmids present in each lane are: A. pBRB55.1; B. pBRB169.1; C. pBRB7.2; D. pFAO2; E. pFAO2.16; F. pFAO2.26; G. pBR322. <br><br> from P1 wildtype and amber mutants. Lysogeny by cells infected with Plcl.245Cm was scored as the growth of infected cells on ampicillin (to select for the resident plasmid) and chloramphenicol (to select for the phage genome). The values observed for the two plasmids containing Plcl and P7cl (pBRB7.2 and pFA02, respectively) are very similar and significantly higher than those obtained for pBR322 or for any of the plasmids carrying <br><br> Table 2. Assay for lacZ expression from plasmids containing P1 DNA fragments. <br><br> ,3-galactosidase activity (units) <br><br> minus cI        plus Pici      plus P7cl            relative activity <br><br> (pDPT152)      (pBB7. 152)     (pFA02.152)     +PICJ           +P7cJ <br><br> pCB192            0.58            0.55           0.54          0.95            0.93 pBCB2.13        154              15.2            13.6          0.10            0.09 pBCB2.16          1.1             1.2             1.1          1.1             1.0 pBCB2.18          4.0             3.4            3.9           0.9             1.0 <br><br> Cells containing derivatives of the ApR promoter-probe plasmid pCB192 and the compatible KnR plasmid pDPT152 were grown in LB at 37?. When they reached mid-log phase, the cells were chilled, lysed, and assayed for (3-galactosidase activity according to the procedure of Miller (23). Plasmids derived from pCB192 are indicated at the left side of the Table. Plasmids derived from pDPT152 are indicated in parentheses across the top of the Table. The values reported are the average of two independent experiments. Relative activity is defined as the f3-galactosidase activity measured in cells harboring plasmids expressing cl divided by the activity measured in cells carrying only pDPT152. <br><br> 7675 <br><br> Nucleic Acids Research <br><br> GATATCCAATCAGGAGTACC GCATCACCCAAGACGACCTG GATGATCTCACTGACACAAT CGAATATCTCATGGCCACTA ACCAGCCAGACTCACAATAAATGCA             105 v-- <br><br> TtgAca                    TATAATG <br><br> CTAATAAATCTATTATTTTC GTTGGATCCTTCTATAATGG TGGCCAACAACTCCCAGTGT AATCCGCTGTGAGTTGTTGG CCATGTCAATTCTGGAGGAGGATCA            210 <br><br> b-----                              I          I                            GGAGGtG <br><br> ATG ATA AAT TAT GTC TAC GGC GAA CAA CTG TAC CAG GAG TTC GTC AGC TTC AGG GAT CTC TTT CTA AAA AAA GCT GTT GCA CGC GCC CAA        300 MET lIe Asn Tyr Vat Tyr Gly Glu Gin Leu Tyr Gin Gtu Phe Vat Ser Phe Arg Asp Leu Phe Leu Lys Lys Ala Val Ala Arg Ala Gtn <br><br> tag(cl .55) <br><br> CAC GTT GAT GCC GCC AGC GAC GGT CGT CCT GTT CGC CCG GTT GTC GTT CTG CCG TTC AM GM ACG GAC AGC ATT CAG GCT GMA ATT GAT         390 His Val Asp Ala Ala Ser Asp Gly Arg Pro Vat Arg Pro Vat Vat Vat Leu Pro Phe Lys Glu Thr Asp Ser lIe Gin Ala Glu lie Asp <br><br> T                            A                C                                         A   G <br><br> AAA TGG ACA TTA ATG GCG CGG GAA CTG GAG CAG TAC CCA GAT CTC MT ATC CCA MG ACT ATT TTA TAT CCT GTA CCT AAC ATC CTT CGC          480 <br><br> 9<br><br> Lys Trp Thr Leu MET Ala Arg Glu Leu Gtu Gin Tyr Pro Asp Leu Asn lIe Pro Lys Thr lie Leu Tyr Pro Vat Pro Asn lIe Leu Arg <br><br> A                        T                    C <br><br> GGT GTG CGT AAG GTT ACG ACT TAT CAG ACA GAA GCA GTG MC AGC GTC AAT ATG ACC GCT GGC CGC ATT ATT CAT CTG ATT GAT AAG GAC        570 Gly Vat Arg Lys Vat Thr Thr Tyr Gin Thr Glu Ala Vat Asn Ser Vat Asn MET Thr Ala Gly Arg lIe lIe His Leu lIe Asp Lys Asp <br><br> G <br><br> ATT CGC ATC CAA AM AGC GCG GGG ATC MT GAG CAC AGT GCG AAA TAC ATA GAG MC CTG GAA GCA ACA AM GAG CTA ATG AAG CAG TAC           660 lle Arg lIe Gin Lys Ser Ala Gly lIe Asn Gtu His Ser Ala Lys Tyr lIe Gtu Asn Leu Gtu Ala Thr Lys Gtu Leu MET Lys Gin Tyr <br><br> T                                  T <br><br> CCG GAG GAT GAA AAA TTC CGT ATG CGC GTA CAC GGC TTT AGC GAA ACA ATG CTG CGC GTC CAT TAC ATT TCC AGT AGC CCT AAC TAC AAT        750 Pro Glu Asp Glu Lys Phe Arg MET Arg Vat His Gly Phe Ser Gtu Thr MET Leu Arg Vat His Tyr lie Ser Ser Ser Pro Asn Tyr Asn <br><br> Phe <br><br> T                               C     G       T                                     T <br><br> I                      ~~~I   I       II <br><br> GAT GGC MA TCA GTT AGT TAC CAT GTG CTG CTA TGT GGC GTG TTT ATC TGC GAT GM ACT CTC CGA GAT GGA ATC ATC ATC AAC GGT GAA          840 <br><br> e.. Asp Gly Lys Ser Vat Ser Tyr His Vat Leu Leu Cys Gly Vat Phe lie Cys Asp Glu Thr Leu Arg Asp Gly lIe lie lIe Asn Gly Gtu <br><br> Pro <br><br> C                                                                                     tag(cl .169) <br><br> TTT GAG AM GCA AAA TTT AGC CTT TAT GAC TCT ATA GM CCG ATC ATC TGC GAC CGC TGG CCG CAG GCA AM ATA TAT CGC CTG GCA GAT          930 Phe Gtu Lys Ala Lys Phe Ser Leu Tyr Asp Ser lIe Glu Pro lie lie Cys Asp Arg Trp Pro Gin Ala Lys lIe Tyr Arg Leu Ala Asp <br><br> T <br><br> ATT GM MT GTA AM AM CM ATT GCC ATC ACT CGC GM GAG AAA           G GTC AM TCA GCC GCA TCA GTT ACG CGC AGC CGC AAA ACT AAG 1020 <br><br> n-----<br><br> lie Glu Asn Vat Lys Lys Gin lIe Ala lie Thr Arg Glu Glu Lys Lys Vat Lys Ser Ala Ala Ser Vat Thr Arg Ser Arg Lys Thr Lys <br><br> AAG GGG CAG CCA GTA AAC GAC MC CCC GAA AGC GCG CM TAG Lys Gly Gin Pro Val Asn Asp Asn Pro Glu Ser Ala Gin ter <br><br> Fig. 3. DNA sequence of PIcI and P7cl. The DNA sequence of PIcI is indicated. Positions where the sequence of P7cl differs from that of Plcl are indicated above the P1 sequence. The amino acid sequence predicted by the open reading frame is given below the sequence. The two amino acid substitutions present in P7cl are shown below the open reading frame. The locations of the amber mutant codons in cl.55 and ci. 169 are indicated by small letters above the sequence. Sites for selected restriction enzymes (EcoRV [v]; BamHI [b]; Bgll [g]; EcoRP [e]; and NruI [n]) are illustrated by dashed lines beneath the sequence. The cl repressor binding site is underlined. Inverted arrows beneath the sequence illustrate the inverted repeat sequence upstream of the open reading frame. Predicted promoter ribosome binding sites are indicated by the presence of the consensus sequences above and below the line, respectively. The DNA sequences of Plcl from bp 1-134 and bp 1-434 were reported previously (2,5). <br><br> mutant cl genes from either P1 or P7 (Table 1). The efficiency with which a cloned P7cl gene complements a PlcI mutation confirms previous genetic studies indicating that these two genes are functionally interchangeable (9). The location of the 'y6 mutations that destroy cl-complementing activity suggests that the P7cl open reading frame occupies a map position similar to that of the P1 open reading frame (Figure 1). 7676 <br><br> Nucleic Acids Research <br><br> Proteins produced by fragments containing Plcl. <br><br> As an initial step in the comparison of the P1 and P7 repressors, we analyzed the gene products expressed from the cloned cl regions. In an in vitro transcription-translation reaction, plasmids coding for the wildtpe alleles of either PIcI or P7cl direct the production of a protein with an estimated molecular weight of 33,000 daltons (Figure 2, Lanes C and D), a size that agrees closely with the predicted molecular weight of the PIcI repressor reported previously (3,28). The loss of the 33,000 dalton protein in the cl - 'ya-induced P7 mutant plasmids (Figure 2, Lanes E and F) is consistent with its designation as the P7cl repressor. As expected, the 33,000 dalton protein is not observed when reaction mixtures contain DNA from Plcl amber mutants (Figure 2, Lanes A and B). DNA sequence analysis of the cl genes. <br><br> To make a direct comparison between the Plcl and P7cl DNA sequences and to predict the amino acid sequences of the repressor proteins, we carried out M 13-dideoxy sequence analysis of cloned fragments carrying the cl genes. The sequences of about 1 kb of P1 and P7 DNA were determined starting from a common EcoRV site predicted to lie approximately 200 bps upstream of the cl genes. The P1 and P7 sequences (Figure 3) both contain an ATG initiation codon preceded by a putative ribosome binding sequence (29) situated 211 bps downstream of the EcoRV site. In each case, the initiation codon is followed by an open reading reading frame extending for 283 codons. The P1 and P7 open reading frames code for proteins with predicted molecular weights (32,515 and 32,499 daltons, respectively) that agree closely with the values of the proteins expressed from the cloned DNA fragments (Figure 2) and with results predicted independently for the purified PIcI repressor (3-4). The localization of two PIcI amber mutations to the P1 open reading frame confirms its identification as the cI coding sequence. cI. 169 contains an amber mutation that would result in a protein fragment of 26,680 daltons, a value that agrees well with the size of a protein fragment observed under the in vitro transcription/translation reaction conditions (Figure 2, Lane B). The cl.55 amber mutation lies close to the N-terminal region of the protein, resulting in the production of a fragment of 55 amino acids that is apparently too small to resolve under the electrophoretic conditions used for separation of the proteins. Over 60% of the amino acid sequence predicted for the PIcI open reading frame has been verified by amino acid sequence analysis of peptide fragments isolated from the purified repressor protein (see accompanying paper, reference 3). <br><br> The DNA sequences of P1 and P7 are identical for a 399-bp region that extends from the EcoRV site at the 5' side of the cl gene to a point 188 bps within the open reading frame. The sequences within the Plcl and the P7cl open reading frames differ at only 18 positions, all but two of which occur in the wobble position of the predicted codon. From these results, we conclude that the functional identity of the P1 and P7 cl genes is a consequence of their nearly identical amino acid sequence. Analysis of promoters upstream of the cl open reading frame. <br><br> Expression of Plcl was shown previously to require sequences on the distal side of a BamHI site (2, 5) located about 100 bps upstream of the open reading frame (Figure 3). A binding site for the cl repressor has also been shown to exist close to this BamHI site (2, 5, 6). To determine whether this region contains a promoter that is detectable in vivo and, further, to determine whether this promoter can be regulated by cl repressor proteins from either P1 or P7, we introduced several DNA fragments from this region into the promoter probe vector pCB192, screened for promoter activity (as monitored by lacZ expression) and checked for repression of this activity in the presence of a compatible <br><br> 7677 <br><br> Nucleic Acids Research <br><br> plasmid expressing Plcl or P7cl. Cells harboring pBCB2.13 (a plasmid which carries a 460 bp fragment of P1 DNA that extends across the BamHI site upstream of cl into the open reading frame) are dark blue in the presence of Xgal and produce significant levels of 3-galactosidase (Table 2). In contrast, pBCB2.16 and pBCB2.18 (which each contain DNA from only one side of the BamHI site located in pBCB2. 13) do not confer a blue color on their host cell in the presence of X-Gal and express negligible amounts of 3-galactosidase (Table 2). These observations suggest that expression from the promoter identified here requires sequences that span the BamHI site upstream of cl. Expression of cl from a compatible plasmid in the presence of pBCB2. 13 results in a 90% reduction in promoter strength (Table 2). This reduction is seen in the presence of either Plcl or P7cl, indicating that the two repressor proteins are both capable of repressing expression from this promoter. DISCUSSION. <br><br> The DNA sequences of Plcl and P7cl differ at only 18 sites, all but two of which occur at the third position of the affected codon. This observation provides biochemical confirmation of the functional identity predicted on the basis of previous genetic analysis (9). A number of DNA binding proteins exhibit a common structural motif in which two helices are separated by a glycine residue (12). This motif is not observed in the predicted secondary structures (30) of the Plcl and P7cl amino acid sequences. A sequence with some similarity to the XCro helix-turn-helix region was previously reported near the Nterminus of the PIcI protein (5); however, it was noted that the potential for helix formation is disrupted by the presence of several prolines within the region. The secondary structure predicted for the Plcl and P7cl repressor proteins (30) does not reveal other structural characteristics (e.g., Zn fingers (31), leucine zippers (32), or helix-loop-helix motifs (33)) that have been associated with DNA binding activity in other systems. A search of the GenBank and EMBL databases does not reveal any other known regulatory proteins with significant amino acid similarity to the Plcl or the P7cl repressor sequences. Since the Plcl repressor differs from most other repressors in DNA binding specificity (i.e., in its recognition of an asymmetric operator sequence), it is not unexpected to find that the protein does not exhibit common structural motifs at the amino acid level. <br><br> The cl-repressible promoter described in this report is located in a region just upstream of the cl open reading frame and is oriented in the direction of cl. Because the promoter is present on a multicopy plasmid, it is not possible to make a direct calculation of promoter strength; however, the values observed are about five-fold lower than the levels produced by a derivative of pCB192 that contains the plac promoter from pUC19 (34). Because sequences on both sides of the BamHI site located upstream of cl are required for promoter activity (Table 2), we suggest that the promoter spans this site. Less than 10 bps downstream of this BamHI site is a heptanucleotide sequence (TATAATG) that is identical to the -10 consensus sequence for RNA polymerase (35). If this sequence does indeed correspond to the -10 region of the promoter, the -35 region would be predicted to lie on the other side of the BamHI site in a region that overlaps a known cI repressor binding site (2-5). Analysis of this region does not reveal any sequences with significant similarity to the -35 consensus sequence. The best fit is the sequence TCTATT (Figure 3), which matches only two positions of the -35 consensus (TTGACA). The lack of a strong -35 region is often observed with genes that require an activator. Although a pentanucleotide sequence corresponding to the conserved portion of the CRP protein consensus binding site (36) <br><br> 7678 <br><br> Nucleic Acids Research <br><br> is located just upstream of the predicted -35 region (at position 91; Figure 3), a role for CRP-mediated activation in cl expression has not previously been described. The orientation of the promoter and its cl-repressible character raise the possibility that cl expression is autoregulatory. If this is so, one potential activator would be the cl repressor itself. Expression cannot be absolutely dependent on cl-mediated activation, however, because the cloned promoter exhibits significant activity in the absence of the cl gene (Table 2). Under the conditions reported here, the presence of the cl gene results in a decrease rather than an increase in lacZ expression; however, these observations do not rule out a potential activator role for the cl protein, since the ratios of repressor and operator provided by the multicopy plasmids may not be optimal for activation. Physiologically, the role of additional repressor binding sites in regulating cl expression also cannot be discounted. Three potential operator sites have been identified several hundred bps upstream of the cI open reading frame (2-5); one or more of these could be involved (possibly through a DNA looping mechanism; 37) in the activation or repression of cl expression during phage growth. <br><br> A cl-repressible promoter oriented in the direction of cl was previously reported (38) to be located entirely within PlBamHI-9, a fragment located upstream of cl which is bracketed by the BamHI site within pBCB2. 13. Because sequences on both sides of this BamHI site are required for the activity of the promoter in pBCB2.13, we suggest that the previously identified promoter is distinct from the one reported here. The promoter from BamHI-9 could correspond to a consensus promoter sequence that is situated about 500 bps upstream of cl and overlaps a cl repressor binding site (2). If so, cl expression is likely to be controlled by more than one promoter. Located between this promoter sequence and the promoter encoded on pBCB2. 13 is an open reading frame whose product (termed coi, or c-one inactivator) has been implicated in the establishment of lytic growth (1, 39; B.R. Baumstark, unpublished results). It has been suggested (2) that the decision to enter lytic or lysogenic growth is influenced by the level of transcription initiated from the distal promoter (which would transcribe coi prior to the transcription of cl) relative to that of the promoter located immediately upstream of the cl gene (which would transcribe only ci). <br><br> A 32-nucleotide hyphenated inverted repeat sequence is located just upstream of the cl open reading frame (positions 146-188; Figure 3). It is not currently known whether this sequence has any regulatory effect on cl expression. Conceivably, the sequence could serve as a recognition site for an as-yet-unidentified regulatory protein. Alternatively, it may affect the secondary structure of the messenger RNA. A transcript extending from a promoter located upstream of the putative coi open reading frame would be capable of forming a stable stem-loop structure containing 16 bps with a single bp mismatch (AG = -33.6 Kcal) of this inverted repeat sequence. Such a structure could potentially serve as a recognition site for a regulatory factor or, alternatively, could mask such a site. On the other hand, transcription originating from the promoter spanning the BamHI site just upstream of cl would start at a site within the inverted repeat sequence, forming a comparatively less stable stem-loop structure of about 8 bps. The role of the inverted repeat region in the regulation of cl expression is currently under investigation. <br><br> ACKNOWLEDGEMENTS <br><br> We thank Heinz Schuster for his review of the manuscript. This work was supported by National Science Foundation grant DMB-8704146. <br><br> 7679 <br><br> Nucleic Acids Research <br><br> Abbreviations: bp, basepairs; kb, kilobase pairs; X-Gal, 5-Bromo-4-Chloro-3-indolylbeta-D-galactopyranoside. <br><br> *To whom correspondence should be addressed <br><br> REFERENCES <br><br> 1. Yarmolinsky, M.B., and Steinberg, N. (1988). In Calendar, R., (ed.), The Bacteriophages, Plenum Publishing <br><br> Corp., NY, Vol. 1, pp. 291-438. <br><br> 2. Baumstark, B.R., Stovall, S.R., and Ashkar, S. (1987). Virology 156, 404-413. <br><br> 3. Dreiseikelmann, B., Velleman, M., and Schuster, H. (1988). J. Biol. Chem. 263, 1391-1397. <br><br> 4. Heinrich, J., Riedel, H.-D., Baumstark, B.R., Kimura, M., and Schuster, H. (1989). Nucleic Acids Res, <br><br> this volume. <br><br> 5. Eliason, J.L., and Stemnberg, N. (1987). J. Mol. Biol. 198, 281-293. <br><br> 6. Velleman, M., Dreiseikelmann, B., and Schuster, H. (1987). Proc. Natl. Acad. Sci. USA 84, 5570-5574. 7. Citron, M., Velleman, M., and Schuster, H. (1988). J. Biol. Chem. 264, 3611-3617. <br><br> 8. Chadwick, P., Pirotta, V., Steinberg, R., Hopkins, N., and Ptashne, M. (1970). Cold Spring Harbor Symp. <br><br> Quant. Biol. 35, 283-294. <br><br> 9. Chesney, R.H., and Scott, J.R. (1975). Virology 67, 375-384. <br><br> 10. Wandersman, C., and Yarmolinsky, M. (1977). Virology 78, 267-276. <br><br> 11. Scott, J.R., West, B.W., and Laping, J.L. (1978). Virology 85, 587-600. 12. Pabo, C.O., and Sauer, R. A. (1984). Ann. Rev. Biochem. 53, 293-321. 13. Scott, J.R. (1974). Virology 62, 344-349. <br><br> 14. Schneider, K., and Beck, C.F. (1987). Methods in Enzymol. 153, 452-461. 15. Smith, H.W. (1972). Nature New Biol. 238, 205-206. 16. Scott, J.R. (1975). Virology 65, 173-178. <br><br> 17. Scott, J.R., Kropf, M.M., and Mendelson, L. (1977). Virology 76, 39-46. 18. Scott, J.R. (1968). Virology 36, 564-574. <br><br> 19. Walker, D.H., Jr., and Walker, J.T. (1976). J. Virol. 20, 177-187. 20. Stemnberg, N. (1979). Virology 96, 129-142. <br><br> 21. Schneider, K., and Beck, C.F. (1986). Gene 42, 37-48. <br><br> 22. Taylor, D.P., and Cohen, S.N. (1979). J. Bacteriol. 137, 92-104. <br><br> 23. Miller, J.H. (1972). In Experiments in Molecular Genetics, Cold Spring Harbor Laboratory, Cold Spring <br><br> Harbor, N.Y. 352-355. <br><br> 24. Guyer, R.S. (1978). J. Mol. Biol. 126, 347-365. <br><br> 25. Devlin, B.H., Baumstark, B.R., and Scott, J.R. (1982). Virology 120, 360-375. <br><br> 26. Sanger, F., Nicklen, S., and Coulson, A.R. (1977). Proc. Natl. Acad. Sci. USA 74, 5463-5467. 27. DeVries, J.K., and Zubay, G. (1967). Proc. Natl. Acad. Sci. USA 57, 1010-1012. <br><br> 28. Heilmann, H., Reeve, J.R., and Puhler, A. (1980). Mol. Gen. Genet. 178, 149-154. 29. Shine, J., and Dalgarno, L. (1974). Proc. Natl. Acad. Sci. USA 71, 1342-1346. 30. Chou, P.Y., and Fasman, G.D. (1978). Adv. Enzymol. 47, 45-148. 31. Berg, J.M. (1986). Nature 319, 264-265. <br><br> 32. Landschultz, W.H., Johnson, P.F., and McKnight, S.L. (1988). Science 240, 1759-1764. 33. Murre, C., McCaw, P., and Baltimore, D. Cell 56, 777-783. <br><br> 34. Anderson, B.E., Baumstark, B.R., and Bellini, W.J. (1988). J. Bacteriol. 170, 4493-4500. 35. Rosenberg, M., and Court, D. (1979). Annu. Rev. Genet. 13, 319-353. <br><br> 36. Ebright, R.H., Cossart, P., Gicquel-Sanzey, B., and Beckwith, J. (1984). Nature 311, 232-235. 37. Ptashne, M. (1986). Nature 322, 697-701. <br><br> 38. Stemnberg, N., and Hoess, R. (1983). Annu. Rev. Genet. 17, 123-154. 39. Scott, J.R. (1980). Curr. Top. Microbiol. Immunol. 90, 49-65. <br><br> 7680 <br><br> This article, submitted on disc, has been automatically <br><br> converted into this typeset format by the publisher. <h3>pmcA1885853</h3>A global gene evolution analysis on Vibrionaceae family using phylogenetic profile
Abstract
Background
Vibrionaceae represent a significant portion of the cultivable heterotrophic sea bacteria; they strongly affect nutrient cycling and some species are devastating pathogens.
In this work we propose an improved phylogenetic profile analysis on 14 Vibrionaceae genomes, to study the evolution of this family on the basis of gene content.
The phylogenetic profile is based on the observation that genes involved in the same process (e.g. metabolic pathway or structural complex) tend to be concurrently present or absent within different genomes. This allows the prediction of hypothetical functions on the basis of a shared phylogenetic profiles. Moreover this approach is useful to identify putative laterally transferred elements on the basis of their presence on distantly phylogenetically related bacteria.<br><br>Results
Vibrionaceae ORFs were aligned against all the available bacterial proteomes. Phylogenetic profile is defined as an array of distances, based on aminoacid substitution matrixes, from single genes to all their orthologues. Final phylogenetic profiles, derived from non-redundant list of all ORFs, was defined as the median of all the profiles belonging to the cluster. The resulting phylogenetic profiles matrix contains gene clusters on the rows and organisms on the columns.
Cluster analysis identified groups of "core genes" with a widespread high similarity across all the organisms and several clusters that contain genes homologous only to a limited set of organisms. On each of these clusters, COG class enrichment has been calculated. The analysis reveals that clusters of core genes have the highest number of enriched classes, while the others are enriched just for few of them like DNA replication, recombination and repair.<br><br>Conclusion
We found that mobile elements have heterogeneous profiles not only across the entire set of organisms, but also within Vibrionaceae; this confirms their great influence on bacteria evolution even inside the same family. Furthermore, several hypothetical proteins highly correlate with mobile elements profiles suggesting a possible horizontal transfer mechanism for the evolution of these genes. Finally, we suggested the putative role of some ORFs having an unknown function on the basis of their phylogenetic profile similarity to well characterized genes.<br><br><br><br>Background
Over the past ten years, a great number of microbial genomes have been sequenced covering a wide representation of prokaryots as well as multiple strains of some species. The study of these genomes both by computational and experimental approaches has highly improved our understanding on physiology, phylogenetic relationship and pathogenicity of many organisms. Furthermore, it has provided new knowledge on microbial genome evolution, revealing a gene core shared by the great majority of bacteria, genes characteristic of particular groups and "novel" genes that possibly originated by lateral gene transfer from some unknown source.
Analysis performed on closely related genomes revealed that a substantial fraction of genes in any genome seem to be strain specific. These genes might sometime arise by gene duplication followed by a rapid divergence, or by lineage-specific loss of genes in one strain, resulting in a unique gene in other strains. However, there are several lines of evidence indicating that lateral gene transfer may be the main mechanism to acquire novel genes. Indeed, this could be one of the main forces driving bacterial adaptation and evolution. Phage DNA is thought to be one of the main vectors for lateral gene transfer among bacteria [1] and many virulence factors from bacterial pathogen are phage encoded [2]. For example, the genes for CT, the most important virulence factor of <span class="yellow">V. cholerae</span>, are encoded in the genome of phage CTXφ, integrated in the bacterial chromosome 1.
Since lateral gene transfer plays a relevant role in bacterial evolution, the reconstruction of phylogeny is very complex and phylogenetic trees built by standard sequence analysis may not lead to a reliable picture of the evolutionary history. In fact, alternative trees can be obtained when different proteins are considered.
For many aspects the classification of bacteria on the basis of their global gene content may give a better description of their evolutionary history. This may be particularly important when bacteria of the same group are compared, since newly acquired genes could be relevant to confer peculiar features that allows the exploitation of different ecological niches.
In this study we propose a bioinformatic procedure to investigate bacterial genome evolution, taking into account the global gene content, as well as sequence similarity. We based our analysis on modified phylogenetic profiles [3]; however, we do not consider only the presence/absence of orthologue genes, but also their distance, based on a substitution matrix.
A phylogenetic profile is a non-sequence-homology-based method developed to infer a possible functional relationship between genes. It is based on the idea that proteins that are involved in the same metabolic pathway or structural complex are likely to evolve in a correlate fashion and during evolution appear phylogenetically linked, showing a tendency to be either preserved or eliminated as a whole. Therefore, genes showing similar phylogenetic profiles are likely to be functionally related. We extended the use of phylogenetic profiles to produce an evolutionary tree based on a hierarchical clusterization of organisms with similar phylogenetic profiles.
For this study we took the whole gene dataset of 320 prokaryotic genomes, however, we limited the analysis to the orthologous groups that are present in at least one of the 14 considered species of the Vibrionaceae family. These bacteria belong to the Gammaproteobacteria group and are highly abundant in aquatic environment, they strongly influence nutrient cycling and various species are also devastating pathogens. Since we focused our analysis on this particular group, the aim of this study is not the construction of a global evolutionary tree, but rather a Vibrionaceae perspective of bacterial diversity, based on phylogenetic profiles.<br><br>Results and discussion
Phylogenetic matrix
The analysis was performed on 14 bacteria belonging to the Vibrionaceae family (Table 1). The redundant list of Vibrionaceae ORFs was clustered to reduce the number of proteins to analyze and the phylogenetic profile for each cluster was calculated as described in the Method section.
Many authors proposed and successfully applied different measure methods to calculate the phylogenetic profile values.
Pellegrini et al. [3] firstly proposed a phylogenetic profile described as a string of bits, each bit representing the absence or presence of an homologous gene in a given genome. This method lacks a weighting procedure, giving the same weight (value 1) to all the sequences that are considered homologous given a similarity threshold. Enault and colleagues proposed an improved phylogenetic profile based on a normalized Blastp bit score [4]. This method, compared to the approach implemented by Pellegrini, allows weighting each point of the profile proportionally to the length and the quality of the alignment. Jingchun and colleagues optimized the phylogenetic profiles method by integrating phylogenetic relationships among reference organisms and sequence homology information, based on E-value score, to improve prediction accuracy [5].
The measure index I proposed in this work is similar to the others described above, taking into account both the quality and the length of the alignment using a substitution matrix. Moreover our approach considers also the total length of the sequences, penalizing good alignments occurring between ORFs having different lengths and taking into consideration that ORFs could differentiate mainly for the presence of functional domains.
The final phylogenetic profile for each cluster was defined as the median of all the profiles belonging to the cluster, named "meta-profile", which describes the profile of conserved ORFs belonging to an entire family.<br><br>Hierarchical cluster analysis
A hierarchical cluster analysis was performed on the entire phylogenetic profile matrix and it was calculated a statistical support based on bootstrap method for the nodes of the columns tree (Fig 1). The branch tree colors represent the bootstrap percentage support. This constitutes a phylogenetic tree based on gene content using Vibrionaceae ORFs as a reference. Genomes belonging to the same taxonomic group tend to cluster together and the Vibrionaceae species are closely related. As expected, according to the Vibrionaceae branch lengths it is evident that variability within this group is higher compared to the other groups. The dataset used for phylogenetic matrix calculation is indeed composed by Vibrionaceae ORFs. This implies that the similarity measures between these ORFs and the corresponding orthologues will be nearly zero in most of the other species and significantly higher in the Vibrionaceae family, increasing the variability into this group. Moreover the average percentage of clusters shared by the Vibrionaceae members is only 47.5% (average number of shared clusters divided by the total number of clusters) that again indicates a high variability inside this family. It is also interesting to note that organisms belonging to the same or closely related taxa split into different subgroups. This highlights the existence of a high variability among lineages, due to genetic and evolutionary processes such as lateral gene transfer, concerted evolution and gene duplication [6]. In terms of gene content, the organisms more related to the Vibrionaceae belong to the gamma and beta proteobacteria. In particular Altermonadales, Enterobacteriales and Burkholderiales are closely related to Vibrionaceae, and share the higher number of cluster of genes (average percentage of 20%). As expected, Archea are the most distant group sharing just 3.8% of clusters.
Clusters and genes distribution, as shown in Fig 2, reveals that the number of clusters and genes shared by the organisms decreases as the number of organisms considered increases. The analysis was performed considering for each cluster profile the number of organisms sharing the same numbers of clusters (and genes). The majority of gene cluster groups no more than 21 species on a total of 320. The highest blue spike corresponds to the higher number of genes shared by 105 groups of 14 organisms. Among these groups, as expected, Vibrionaceae are highly represented. Other species represented are <span class="yellow">Colwellia psychrerythraea 34H</span> and <span class="yellow">Shewanella oneidensis</span>, that belong to the Alteromonadales family.
The cluster analysis performed on genes is shown in Fig. 3. From now on, to avoid confusing interpretation between clusters derived from the cluster analysis and cluster derived from the ORFs clustering we will use the term "gene" in place of cluster of ORFs.
The different gradient of color, from bright to dark red, represents decreasing similarity values. The cluster analysis allows the detection of three main groups of genes. The first one (Fig 3, panel B) contains the most conserved and established genes shared almost by all the organisms. These core genes can be defined as the set of all genes shared as orthologous by all members of an evolutionary coherent group. In our analysis we identify four clusters, for a total of 145 genes, shared by all the 320 organisms.
The ORFs belonging to these clusters are predicted to codify for the ATP binding subunit of ABC transporters (annotated as ABC-type polar amino acid transport system, ABC-type antimicrobial peptide transport system, ABC-type histidine transport system and ABC-type transport system involved in lysophospholipase L1 biosynthesis). This finding is surprising since this is the first report where these ORFs are assigned to the core genes. Anyway two different explanations can be traced. First, it is known that the ABC transporters represent an essential transport system in the prokaryotes and that their ATP binding subunits are apparently overrepresented compared to the other two subunits (ligand binding and permease subunit) in all genomes sequenced thus far [7]. Second, one organism, <span class="yellow">Buchnera aphidicola</span>, presents these genes with a similarity just below the cut-off used for the analysis, but they have been considered since it is well known that in this mutualistic endosymbiont the accelerated evolution and AT bias affect all its genes, including the 16S rRNA [8,9].
The dataset used for the analysis includes genomes in draft quality (<span class="yellow">Vibrio cholerae 0395</span>, <span class="yellow">Vibrio cholerae MO10</span>, <span class="yellow">Vibrio cholerae RC385</span>, <span class="yellow">Vibrio cholerae V51</span>, <span class="yellow">Vibrio cholerae V52</span>, <span class="yellow">Photobacterium profundum 3TCK</span>, <span class="yellow">Vibrio MED222</span>, <span class="yellow">Vibrio splendidus12B01</span>). Wrong ORFs prediction or missing genes due to incomplete genome sequences can explain the low number of core genes identified. To avoid such problems we repeated the analysis excluding the draft genomes and thus considering 312 genomes. The results, reported in Table 2, show an increased number of the core genes and in particular ribosomal proteins and tRNA synthetase, as reported by Charlesbois and Doolittle [10]. This could be considered as a sort of "minimal genome" containing the group of genes that are necessary to maintain a free-living organism.
The low number of genes shared by all the organisms can be due to many factors. First of all we used the Vibrionaceae ORFs as a reference, limiting the number of genes we were able to identify. It was further demonstrated that the core gene size decreases as more genome sequences are analyzed [10].
Genes that are considered to belong to the core set when close organisms are compared, are classified as flexible genes when distantly related genomes are analyzed [6]. Finally, genes within core genomes might be transferred or replaced, introducing new versions of existing genes into genomes. Such transfers can replace even highly conserved genes by non-homologous counterparts but the advantages provided are difficult to explain. It is also to take into consideration that many symbiotic and parasitic bacteria undergo a reduction of their genomes, loosing many genes required by free-living cell.
The second group (Fig 3, panel C) represents genes shared mainly among Vibrionaceae and other gamma proteobacteria (as Altermonadales, Burkholderiales and Enterobactidiales).
Finally, the third group (Fig. 3, panel D) is composed by genes that are mainly specific to the Vibrionaceae.<br><br>k-mean cluster analysis and cluster enrichment
We performed a k-means cluster analysis, setting the k value to 14. As shown in Fig. 4, the clusters 3, 4, 11, 13 and 14 contain the higher percentage of genes, accounting for more that 50% of the total genes, while clusters 9 and 10 contain the lower number of ORFs (3% of genes). The variance in each k-means cluster is very low (Fig. 4), meaning that the clusters contain genes with compact and similar profiles. As described in Fig. 4, the majority of the clusters (1, 2, 3, 4, 5, 6, 7, 9, 12, 13) contains genes with a similar profile, with the average values (red line) near zero, except for the presence of some spikes correspondent to an increasing similarity with some isolated organisms. As shown in Table 3, clusters 1, 3, 4, 5, and 9 contains genes that have a high similarity in a small subset of organisms. The majority of these ORFs are annotated as hypothetical proteins or phage related proteins. Clusters 8, 10 and 14 present genes shared among almost all the organisms. In particular cluster 10 is composed by the core genes described before having an high value of similarity widespread among all the organisms; cluster 8 contains genes shared mainly by gamma proteobacteria and cluster 14 is composed of genes in common between Vibrionaceae and Enterobacteriaceae.
A functional annotation has been performed on each gene cluster using COG (Cluster of Orthologous Genes), KEGG pathway map and GO databases. For each k-mean cluster the enrichment probability with respect to the total number of clusters has been obtained with the hypergeometric distribution.
Fig. 5 shows COG enrichment results for each cluster. As expected clusters represented by conserved genes (cluster 8, 10 and 14) have the higher number of enriched COG codes, while cluster specific of few organisms are characterized by a small number of enriched COGs. The majority of clusters presents COG codes enrichment for S (function unknown), R (poorly characterized) and – (absence of COG code) categories. This is due to the large abundance of unknown and hypothetical proteins presents in the Vibrionaceae proteomes.
It is worth noting that cluster 3, mainly represented by <span class="yellow">Photobacterium profundum SS9</span> ORFs, is enriched only by C (Energy production and conversion), L (DNA replication, recombination and repair) and M (Cell envelope biogenesis, outer membrane). Probably the L class overrepresentation is determined by the high number of transposons that are present in the SS9 genome [11]. The role played by these transposable elements in the survival of this deep-sea bacterium it is still a matter of debate [12].
In addition <span class="yellow">V. vulnificus YJ016</span> and <span class="yellow">V. vulnificus CMCP6</span> (cluster 13) seem to share genes belonging to the enriched COG classes K (Transcription), L and T (Signal transduction mechanisms). It was previously reported an enrichment in genes belonging to the transcription class in the genome of <span class="yellow">V. vulnificus</span> respect to the <span class="yellow">V. cholerae</span> genome [13]. This class is clearly related to the T class and seems to indicate that this organism is able to receive and translate in a transcriptional response specific environmental signals. Despite this, the large majority of the genes in clusters 3 and 13 lacks COG annotation.
Cluster 7, as shown in Table 3, accounts organisms with large genome size (see Table 1). This can explain the fact the this cluster contains almost all the COG class enriched and suggests a more complex and flexible life-style of these organisms compared to the other Vibrionaceae members.
KEGG annotation is limited to metabolic or structural complex network and so a reduced number of genes have a KEGG entry. This causes the presence of clusters without enriched map (cluster 2–5, 7, 13, see Fig 5). Also in this case, the clusters presenting the higher number of significant KEGG map are those containing the conserved genes. The most enriched KEGG clusters are cluster 14, 10 and 8 accounting for the majority of the metabolic pathways. Cluster 1 is enriched for map3080 (type IV secretion system). In fact <span class="yellow">V. fischeri</span> genome contains 10 separate pilus gene clusters, including eight type-IV pilus loci. The presence of multiple pilus gene clusters suggests that different pili may be expressed in different environments or during multiple stages of its development as a symbiont [14].
Cluster 11 is enriched for map3090 (type II secretion system). The type II pathway is conserved among gram-negative bacteria, including many pathogens, and secretes a variety of virulence factors and degradative enzymes [15].
Cluster 9 is enriched for map 00860 (Porphyrin and chlorophyll metabolism). These genes are involved in the cobalamin (coenzyme B12) biosynthetic pathway [16]. Some organisms, such as <span class="yellow">Salmonella typhimurium</span> and <span class="yellow">Klebsiella pneumoniae</span>, can synthesize cobalamin de novo [17], while <span class="blue">E. coli</span> and large part of the Vibrionaceae perform cobalamin biosynthesis only when provided with cobinamide. It is interesting to observe that the genes belonging to the de novo pathway are only shared by Archea, some other organisms like Salmonella, Pseudomonas and <span class="yellow">Vibrio MED222</span>.
Finally cluster 6 is enriched by map2010 (ABC transporter), map2020 (two-component system), map2030 and map2031 (bacterial chemotaxis), map2040 (Flagellar assembly) and map3090 (type II secretion system). This cluster contains genes shared with a high similarity by all Vibrio and with a lower similarity with <span class="yellow">Photobacterium profundum</span> species. Among the Vibrio species the organisms showing the highest similarity (Tab. 3) are <span class="yellow">V. cholerae</span> strains.<br><br>Vibrionaceae specific genes
We identify 1940 clusters specific to the Vibrionaceae. All the Vibrionaceae considered in the analysis share 108 clusters. Among these genes we identify ToxR and ToxS genes. ToxR gene encodes a transmembrane regulatory protein firstly identified in <span class="yellow">V. cholerae</span>, in which it co-ordinates many virulence factors in response to several environmental parameters [18]. <span class="yellow">V. cholerae</span> ToxR activity is enhanced by a second transmembrane protein, ToxS, encoded downstream toxR [19]. This family of proteins is involved in response to temperature, pH, osmolarity and in <span class="yellow">Photobacterium profundum SS9</span>, a piezophilic bacterium, to hydrostatic pressure [20]. The widespread presence of these genes among the Vibrionaceae suggests their importance in regulatory mechanisms.
We identify two other noteworthy groups of genes composed by 257 and 160 genes respectively shared just by two strains, mainly annotated as "hypothetical protein". The first group of genes is shared between <span class="yellow">Photobacterium profundum SS9</span> and <span class="yellow">Photobacterium profundum 3TCK</span>, while the second is shared between <span class="yellow">V. vulnificus CMCP6</span> and <span class="yellow">YJ016</span>. These strains are closely related and this explains the high number of shared genes; while, inside the Vibrionaceae family, the number of specific shared genes highly decreases, showing a high inter-species variability (Fig. 6)<br><br>Prophages and transposases
Prophages recover different biological roles both as quantitatively important genetic elements of the bacterial chromosome, and as vectors of lateral gene transfer among bacteria, due to their characters of mobile DNA elements. Indeed, numerous virulence factors from bacterial pathogens are phage encoded. It was postulated that this role of prophages is not limited to pathogenic bacteria but some adaptations of nonpathogenic strains to their ecological niche might also be mediated by prophages acquisition [21].
To better understand the importance of mobile elements within Vibrionaceae family, we performed a hierarchical cluster analysis using gene profiles annotated as "phage protein" and "transposase", for a total of 172 clusters of genes (Fig 7). We found that a high inter-strain genetic variability exists and phages and transposases are both shared by almost all Vibrionaceae, and specific to just some organisms. We identified five major clusters of mobile elements that are specific to a single organism. A group composed by 26 clusters containing both transposase and phage proteins seem to be unique to <span class="yellow">V. splendiduds 12B01</span> (Fig 7). Another one composed by 16 clusters is specific of <span class="yellow">V. vulnificus CMCP6</span> (Fig 7) while <span class="yellow">V. parahaemoliticus</span> has a cluster of 11 genes (Fig 7). Moreover there is another group of transposases and phage genes shared mainly by <span class="yellow">V. cholerae</span> 0395, <span class="yellow">Shewanella oneidensis</span> and <span class="yellow">V. cholerae</span> V51 (Fig 7). Finally a big cluster of almost 30 genes, all predicted to codify for transposases, was found in <span class="yellow">P. profundum SS9</span> genome (Fig. 7). The high presence of transposases in this bacterium seems to correlate with its deep-sea habitat, a feature presumably shared with other deep-sea microorganisms [12]. As shown in Fig 7, many of the clusters well conserved in an organism, are partially shared with a low similarity by other organisms. This agrees with the idea that prophages are not maintained in the genome over a long period of time and part of their genes may be deleted from the chromosome. Moreover, microarray analysis and PCR scanning demonstrated that prophages are frequently strain specific within a given bacterial species [22-24]. According to the modular theory of phage evolution, phage genomes are mosaics of modules, groups of genes functionally related, that are free to recombine in genetic exchanges between distinct phages infecting the same cell [21]. This can result in the occurrences of different part of phage distributed in far related genomes. Phylogenetic profile of some transposases is similar to the phage ones, suggesting a possible transfer mechanism phage-mediated for such mobile elements.<br><br>
Conclusion
In this work we propose an improved phylogenetic profile analysis on 14 Vibrionaceae genomes, to study this family on the basis of gene content. Using a phylogenetic profile for each cluster of genes defined as the median of all the profiles belonging to the cluster (meta-profile) we investigate the evolution of groups of ORFs belonging to the entire family. A two-way cluster analysis allows us to identify similarity structures on the entire phylogenetic matrix composed by 8,239 clusters of genes and 320 organisms.
The phylogenetic tree obtained with the cluster analysis does not reflect the global evolutionary tree because of the Vibrionaceae ORFs dataset used for the analysis, but rather can be considered as the Vibrionaceae perspective of bacterial diversity. The phylogenetic tree reflects the evolutionary processes that shape genomes, as lateral gene transfer, genes genesis and loss. In this context, the tree allows to group together genomes on the base of their global gene content.
We found that genomes belonging to the same taxonomic group tend to cluster together and that Vibrionaceae species are closely related. Moreover organisms belonging to the same or closely related taxa split into different subgroups, confirming the existence of a high variability among lineages, due to genetic and evolutionary process such as lateral gene transfer, concerted evolution and genes duplication.
On the other hand several groups of genes characterised by different homogeneous profiles have been identified. In particular we found, 1) a set of conserved genes (with a high similarity values across all organisms) that reflects the "minimal genome" composition defined in other previous works; 2) a set of genes mainly shared by Vibrionaceae and other Gamma proteobacteria and 3) genes specific to different sets of Vibrionaceae.
Finally a further analysis on prophage and transposase has confirmed the high inter-strain genetic variability even among closely related species.
The increasing number of genomes included in this type of analysis surely add new sorces of variability and noise, anyway we think that the use of meta-profiles can be useful for complexity reduction and data analysis to study global gene evolution.<br><br>Methods
Datasets
The Vibrionaceae species used in this analysis were selected among the freely available complete and draft genome sequences. The proteomes of <span class="yellow">V. cholerae N16961</span>, <span class="yellow">V. parahaemolyticus</span>, <span class="yellow">V. vulnificus YJ016</span>, <span class="yellow">V. vulnificus CMCP6</span>, <span class="yellow">V. fischeri ES114</span>, <span class="yellow">Photobacterium profundum SS9</span> were downloaded from the NCBI ftp site [25]. Protein sequences of <span class="yellow">Vibrio cholerae MO10</span>, <span class="yellow">Vibrio cholerae 0395</span>, <span class="yellow">Vibrio cholerae RC385</span>, <span class="yellow">Vibrio cholerae V51</span>, <span class="yellow">Vibrio cholerae V52</span> were downloaded from the NCBI genome database, while sequences of <span class="yellow">Vibrio MED222</span>, <span class="yellow">Vibrio splendidus 12B01</span>, <span class="yellow">Photobacterium profundum 3TCK</span> from the J. Craig Venter Institute web site.
The 320 complete genomes update at 03/06 were downloaded from the NCBI ftp site.<br><br>Similarity search and phylogenetic profile construction
All the Vibrionaceae ORFs were merged generating a redundant list of 59,669 proteins and were compared to all open reading frame from 320 bacterial and archeal genomes using Blastp. To determine the presence of an orthologous we used a combination of three different thresholds; a similarity value equal to or higher than 30%, an alignment length equal or higher than 70% and an Evalue score lower than or equal to e-6. After determining the presence of an orthologous gene, we computed a similarity index I for each pair of orthologous (a point of the phylogenetic profile) as follow:
I=SqsSqq⋅min⁡(lq,ls)max⁡(lq,ls)
 MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacqWGjbqscqGH9aqpdaWcaaqaaiabdofatnaaBaaaleaacqWGXbqCcqWGZbWCaeqaaaGcbaGaem4uam1aaSbaaSqaaiabdghaXjabdghaXbqabaaaaOGaeyyXIC9aaSaaaeaacyGGTbqBcqGGPbqAcqGGUbGBcqGGOaakcqWGSbaBdaWgaaWcbaGaemyCaehabeaakiabcYcaSiabdYgaSnaaBaaaleaacqWGZbWCaeqaaOGaeiykaKcabaGagiyBa0MaeiyyaeMaeiiEaGNaeiikaGIaemiBaW2aaSbaaSqaaiabdghaXbqabaGccqGGSaalcqWGSbaBdaWgaaWcbaGaem4CamhabeaakiabcMcaPaaaaaa@532D@
where lq and ls are the query and subject length sequence respectively and Sqs is the similarity score between the query and the subject sequence. Sqs is defined as follow:
Sqs=∑i=1MαAqi,Asi+GP
 MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacqWGtbWudaWgaaWcbaGaemyCaeNaem4Camhabeaakiabg2da9maaqahabaacciGae8xSde2aaSbaaSqaaiabdgeabjabdghaXjabdMgaPjabcYcaSiabdgeabjabdohaZjabdMgaPbqabaaabaGaemyAaKMaeyypa0JaeGymaedabaGaemyta0eaniabggHiLdGccqGHRaWkcqWGhbWrcqWGqbauaaa@4620@
where M is the match length between the query and subject sequence; Aqi and Asi respectively the query and subject amino acid in position i; α the BLOSUM substitution matrix value for amino acid pair Aqi, Asi and GP the gap penalty. GP is defined as follow:
GP = GOP+GEP(k-1)
where GOP is the Gap Open Penalty set to -11, GEP the Gap Extension Penalty set to -1 and k the gap length. Sqq represents the score of the self-aligned query sequence.
Sqs is always smaller than Sqq and the score S range between 0 and 1. In order to take into account also the different sequence lengths, we multiplied the score S by the ratio between the minimum length between query and subject and the maximum length between query and subjects. In this way the total score is weighted on the base of the length, resulting in a lower similarity value if the lengths of the sequences are different.
The phylogenetic profile for each ORF is an array of index I with length equal to the number of genomes considered (320).<br><br>ORFs clustering
The redundant list of 59,669 Vibrionaceae ORFs contained multiple copies of the same genes due to the presence of conserved genes in the considered genomes. In order to reduce the redundancy, we clustered proteins using a two-step approach. The first step is based on COG (Cluster of Othologous Genes) annotation. COG classifies conserved genes according to their homologous relationships. All the Vibrionaceae ORFs were annotated using COG clusters and proteins sharing the same COG code were considered belonging to the same cluster. In particular, the annotation process consists of a similarity search of all the ORFs against the COG proteins using blast and considering the best hit for each protein. 43,024 ORFs presented a similarity with a COG entry, producing 2,463 different clusters. In the second step, the remaining 16,645 ORFs without similarity with any COG entry were clustered using CD-HIT software [26]. CD-HIT program clusters protein sequence database at high sequence identity threshold and efficiently removes high sequence redundancy. This last clustering process produced 9,613 different groups of similar proteins.
Finally from the 12,076 total clusters obtained by this methodology, those composed by ORFs that do not have any ortologous genes (with a phylogenetic profile composed by an array with all zero values except for one position match with itself) were eliminated, resulting in a dataset composed by 8,239 distinct clusters.
The final phylogenetic profile for each cluster (meta-profile) was defined as the median of all the profiles belonging to the cluster. At the end of these procedures the final phylogenetic matrix was composed by 8,239 rows (cluster of genes) and 320 columns (organisms). In each cell the median of the index in the cluster was reported.<br><br>Cluster analysis
Several clustering techniques have been used to identify the similarity structure underneath our data. A k-means and a two-way hierarchical cluster analysis with Euclidean distance and complete linkage were performed on the phylogenetic matrix.
The goal of a cluster analysis is to partition the elements into subsets without any constrains or a priori information, so that two criteria are satisfied: homogeneity, elements inside a cluster are highly similar to each other; and separation, elements from different clusters have low similarity to each other.
The Figure of Merit (FOM) is a measure of fit of the expression patterns for the clusters produced by a particular algorithm that estimates the predictive power of a clustering algorithm. It is computed by removing each sample in turn from the data set, clustering genes based on the remaining data, and calculating the fit of the withheld sample to the clustering pattern obtained from the other samples. On our data FOM analysis identified the best number of k-means clusters between 10 and 15. We decided to set k (in the k-means analysis) equal to 14. In each of these 14 clusters subsequent hierarchical cluster analysis was performed with bootstrap cluster assessment. All the previous analyses were performed with TMEV software [27], freely available at [28].<br><br>Enrichment categories
Each cluster of genes has been annotated according to COG code, GO terms and KEGG pathway maps. Class enrichment (with respect to the entire matrix) has been calculated according to the hypergeometric distribution that was used to obtain the chance probability of observing the number of genes annotated with a particular COG, GO and KEGG category among the selected cluster. The probability P of observing at least k genes of a functional category within a group of n genes is given by:
P=∑i=kn(fi)(g−fn−i)(gn)
 MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacqWGqbaucqGH9aqpdaaeWbqaamaalaaabaWaaeWaaeaafaqabeGabaaabaGaemOzaygabaGaemyAaKgaaaGaayjkaiaawMcaamaabmaabaqbaeqabiqaaaqaaiabdEgaNjabgkHiTiabdAgaMbqaaiabd6gaUjabgkHiTiabdMgaPbaaaiaawIcacaGLPaaaaeaadaqadaqaauaabeqaceaaaeaacqWGNbWzaeaacqWGUbGBaaaacaGLOaGaayzkaaaaaaWcbaGaemyAaKMaeyypa0Jaem4AaSgabaGaemOBa4ganiabggHiLdaaaa@47C6@
where f is the total number of genes with the same category (in the matrix) and g is the total number of genes in our matrix.<br><br><br><br><h3>pmcA1885664</h3>The recombination-associated protein RdgC adopts a novel toroidal architecture for DNA binding
Abstract
RecA plays a central role in the nonmutagenic repair of stalled replication forks in bacteria. RdgC, a recombination-associated DNA-binding protein, is a potential negative regulator of RecA function. Here, we have determined the crystal structure of RdgC from <span class="yellow">Pseudomonas aeruginosa</span>. The J-shaped monomer has a unique fold and can be divided into three structural domains: tip domain, center domain and base domain. Two such monomers dimerize to form a ring-shaped molecule of approximate 2-fold symmetry. Of the two inter-subunit interfaces within the dimer, one interface (‘interface A’) between tip/center domains is more nonpolar than the other (‘interface B’) between base domains. The structure allows us to propose that the RdgC dimer binds dsDNA through the central hole of ∼30 Å diameter. The proposed model is supported by our DNA-binding assays coupled with mutagenesis, which indicate that the conserved positively charged residues on the protein surface around the central hole play important roles in DNA binding. The novel ring-shaped architecture of the RdgC dimer has significant implications for its role in homologous recombination.<br><br>INTRODUCTION
Homologous recombination systems contribute to the maintenance of genome integrity and are essential for the repair of stalled replication forks (1). Genetic studies of the recombination-dependent growth C (rdgC) gene in bacterial systems have suggested that it is involved in DNA replication and recombination (2,3). In <span class="yellow">Neisseria gonorrhoeae</span> (the gonococcus), RdgC is required for efficient pilin antigenic variation and plays some role in cell growth (4,5). The <span class="yellow">Escherichia coli</span> rdgC gene encodes a DNA-binding protein of 34 kDa, whose expression level was at its highest during exponential phase, reaching its maximum at ∼1000 dimers per cell (2). Its level decreased sharply to ∼50 dimers per cell in stationary phase. This profile suggests that RdgC might function during the period of DNA replication (2).
The RecA proteins and their homologs play a key role in recombinational DNA repair in bacteria and eukaryotes. Their aberrant reactions could result in gross chromosomal rearrangements that lead to <span class="yellow">human</span> diseases, including cancer. Therefore, their functions must be tightly regulated, and understanding how the RecA family of recombinases is regulated is of utmost importance (1). The <span class="blue">E. coli</span> RdgC protein is a potential negative regulator of the function of RecA (1). It inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). Sedimentation equilibrium data indicated that <span class="blue">E. coli</span> RdgC exists in solution as a mixture of oligomeric states in equilibrium, most likely monomers, dimers and tetramers (1). This concentration-dependent change in the oligomeric state appears to affect its mode of binding to DNA and its capacity to inhibit RecA (1). The primary mechanism of RdgC inhibition appears to involve a simple competition for DNA-binding sites, especially on double-stranded DNA (dsDNA) (1).
Deletion of the <span class="blue">E. coli</span> rdgC gene alone causes no obvious phenotype but is highly deleterious in strains lacking certain enzymes involved in recombination and replication restart (2,3). RdgC is essential for the growth of an <span class="blue">E. coli</span> strain lacking PriA, indicating that it might affect replication fork progression or fork rescue (2). PriA provides a means to load the DnaB replicative helicase at DNA replication fork and D loop structures (2). RdgC is, therefore, a key factor in the rescue of stalled or broken forks and subsequent replication restart. dnaC suppressors of PriA can overcome this inviability, especially when RecF, RecO or RecR is inactivated, which indicates that RdgC avoids or counters the toxic effect of these proteins by limiting inappropriate RecA loading on SSB-coated ssDNA (2).
The DNA-binding ability of the RdgC protein has been demonstrated clearly using electrophoretic mobility shift assays (2). <span class="yellow">Escherichia coli</span> RdgC binds non-specifically to dsDNA and with higher affinity than ssDNA (2). RdgC from <span class="yellow">N. meningitidis</span> also binds DNA with little specificity for sequence or structure, like the <span class="blue">E. coli</span> protein (4). RdgC exhibits no preference for DNA replication or recombination intermediates (2). However, no detectable domains or nuclease activity could be identified for the <span class="blue">E. coli</span> RdgC protein (2,6). The complexes of <span class="blue">E. coli</span> RdgC with both linear and supercoiled circular plasmid DNA were imaged using atomic force microscopy (7). RdgC was found to have an increased affinity to DNA ends and to promote bending of DNA (7). RdgC has the effect on DNA superstructure; the promotion of DNA condensation was observed at high protein concentrations (7). Recombination is largely enhanced by close contacts of distant regions along the DNA strands through condensation (7).
<span class="yellow">Pseudomonas aeruginosa</span> is a ubiquitous environmental Gram-negative bacterium that belongs to the γ subdivision of the Proteobacteria. Orthologs of the rdgC gene are found only in β and γ subdivisions of the Proteobacteria (2). The amino acid sequence of <span class="yellow">P. aeruginosa</span> RdgC, a 306-residue protein, shows 46 and 36% identities against those of <span class="blue">E. coli</span> and <span class="yellow">N. meningitidis</span>, respectively. Despite the importance of DNA-binding ability of RdgC in its biological roles, the structural details of RdgC and the mechanism of its action remain unclear. Here, we report the crystal structure of <span class="yellow">P. aeruginosa</span> RdgC as determined by the multiwavelength anomalous diffraction (MAD) method of X-ray crystallography. It reveals that RdgC dimer is ring-shaped, with a central hole of ∼30 Å diameter. The inside surface around the central hole is rich in conserved, positively charged residues, which are implicated in DNA-binding by mutagenesis. We propose that dsDNA binds to RdgC through this central hole. Our structure provides a solid structural framework for a better understanding of the role of RdgC in homologous recombination.<br><br>MATERIALS AND METHODS
Protein expression and purification
The rdgC gene (PA3263) was amplified by the polymerase chain reaction (PCR) using the genomic DNA of <span class="yellow">P. aeruginosa</span> strain PAO1 as a template. The oligonucleotide primers designed using the published genome sequence (8) were 5′-G GAA TTC CAT ATG TGG TTC CGC AAT CTG CTC G-3′ (forward) and 5′-CCG CCG CTC GAG GAC GCC CTG GGG GAT TTC TTC-3′ (reverse). The bases in bold denote the NdeI and XhoI cleavage sites, respectively. The PCR product was digested with NdeI and XhoI, and was then inserted into the NdeI/XhoI-digested expression vector pET-21a(+) (Novagen). This vector construction adds an eight-residue tag (LEHHHHHH) to the carboxyl terminus of the gene product to facilitate protein purification.
The protein was expressed in <span class="blue">E. coli</span> C41 (DE3) cells (9). The cells were grown at 37°C up to OD600 of 0.6 in Luria–Bertani medium that contained 50 μg ml−1 ampicillin, and the protein expression was induced by 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG). The cells were allowed to grow at 20°C for 22 h after IPTG induction and were harvested by centrifugation at 4200 g (6000 rev min−1; Sorvall GSA rotor) for 10 min at 4°C. The cell pellet was resuspended in an ice-cold lysis buffer [20 mM Tris-HCl (pH 7.9), 500 mM sodium chloride, 5 mM imidazole] and was then homogenized with an ultrasonic processor. The crude cell extract was centrifuged at 36 000 g (18 000 rev min−1; Hanil Supra 21 K rotor) for 60 min at 4°C, and the recombinant protein in the supernatant fraction was purified in three chromatographic steps.
The first purification step utilized the C-terminal hexahistidine-tag by Ni2+-chelated HiTrap chelating column (GE Healthcare). The eluent was diluted 3-fold with 50 mM Tris-HCl at pH 7.4. The diluted solution was applied to a 20-ml heparin-Sepharose column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol and 1 mM EDTA. The protein was eluted with a linear gradient of 0–2.0 M sodium chloride in the same buffer. The first peak corresponded to high molecular aggregates of RdgC and the second peak to RdgC dimers, which was also consistent with our dynamic light scattering measurements. When the aggregates were not separated from the dimers, the protein readily aggregated within a day. The dimer fractions were concentrated to ∼6 mg ml−1 concentration using an YM10 ultrafiltration membrane (Millipore-Amicon) and further purified by gel filtration on a HiLoad XK 16 Superdex 200 prep-grade column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 200 mM sodium chloride, 1 mM dithiothreitol and 1 mM EDTA. The first peak corresponding to the high molecular aggregates of RdgC was only a minor component, and the second peak corresponding to RdgC dimers was stable for several weeks when frozen at −70°C. The procedure for purifying the SeMet-substituted protein was the same, except for the presence of 10 mM dithiothreitol in all buffers used during purification steps. Dynamic light scattering experiments were performed at 24°C, with the protein (at 2 mg ml−1 concentration) dissolved in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride on a Model DynaPro-801 instrument (Wyatt, Santa Barbara, CA, USA). The protein concentration was estimated by measuring the absorbance at 280 nm, employing the calculated extinction coefficient of 20 910 M−1 cm−1 (SWISS-PROT; http://www.expasy.ch/).<br><br>Mutagenesis and electrophoretic mobility shift assay
Point mutants were prepared using the QuikChange Site-Directed Mutagenesis kit (Stratagene), and the mutations were confirmed by DNA sequencing. The seven mutants R4A, F120A, K146A, R198A, R208A, Q212A and R252A were well expressed and soluble. The three mutants K70A, R81A and R211A were not expressed, while R122A mutant was expressed but was not soluble. Therefore, we mutated Lys70, Arg81, Arg122 and Arg211 into aspartate. The K70D, R81D, R122D and R211D mutants were well expressed and soluble. The soluble mutants were expressed and purified under the conditions identical to those for the wild type, except that the heparin-Sepharose column step was omitted. All of the soluble mutants had similar elution profiles as the wild type upon gel filtration, and the dimer fractions were used for DNA-binding assays. We mixed 5 μl of RdgC (1.1 μg μl−1) and 5 μl of a linear dsDNA with 414 bp (63 ng μl−1), corresponding to the molar ratio of 64:1 for RdgC dimer to dsDNA. The reaction mixtures were incubated for 10 min at room temperature. RdgC–DNA complexes were analyzed by 1.5% (w/v) agarose gel electrophoresis in 1× TAE buffer (45 mM Tris-acetate at pH 8.3, 1 mM EDTA). DNA bands were visualized by ethidium bromide staining. For DNA-binding assays, the protein concentration was measured with a Bio-Rad protein assay kit.<br><br>Crystallization and X-ray data collection
Crystals were grown by the hanging-drop vapor diffusion method at 24°C by mixing equal volumes (2 μl each) of the protein solution (at 19 mg ml−1 concentration in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride) and the reservoir solution. To grow the native crystals, we used a reservoir solution consisting of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate and 1% (w/v) Anapoe®35 as the detergent. Native crystals grew to approximate dimensions of 0.2 mm × 0.1 mm × 0.1 mm within a few days. The SeMet-substituted protein was crystallized under conditions identical to those for the native crystals except for the presence of 10 mM dithiothreitol in the protein solution.
A crystal of the SeMet-substituted protein was dipped into a cryoprotectant solution for a few seconds and was flash-cooled in the cold nitrogen gas stream at 100 K. The cryoprotectant solution consisted of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate, 1% (w/v) Anapoe®35 as the detergent and 10% (v/v) glycerol. A set of Se MAD data was collected at 100 K at three different wavelengths using an Area Detector System Corporation Quantum 210 charge-coupled device detector at the beamline NW12A of Photon Factory, Tsukuba, Japan. The crystal was rotated through a total of 180° with a 1° oscillation range per frame. The raw data were processed and scaled using the program suit HKL2000 (10). The SeMet-substituted crystal belongs to the space group P21212, with unit cell parameters of a = 87.04 Å, b = 115.60 Å and c = 76.07 Å.
The native crystals were soaked for 10 min in 10 mM ethyl mercury thiosalicylate (EMTS) to collect a set of Hg MAD data at four different wavelengths. X-ray diffraction data from the EMTS-derivatized crystal, as well as a native crystal, were collected on a Bruker charge-coupled device detector at the beamline BL-6B of Pohang Light Source, Pohang, Korea. The EMTS-soaked crystal belongs to the space group P21212, with unit cell parameters of a = 87.16 Å, b = 116.02 Å and c = 74.97 Å. The native crystal belongs to the space group P21212, with unit cell parameters of a = 87.49 Å, b = 116.04 Å and c = 76.03 Å. If the presence of a dimeric molecule in the asymmetric unit is assumed, the calculated crystal volume per protein mass (VM) is 2.75 Å3 Da−1 and the solvent content is 55.3%. Supplementary Table 1 summarizes the statistics of X-ray diffraction data collection.<br><br>Structure solution and refinement
Heavy atom sites were located with the program SOLVE (11). Four Se sites from the SeMet-substituted crystal and two Hg sites from the EMTS-soaked crystal were located. Two sets of the initial Se and Hg MAD phases were separately improved using the program RESOLVE (12). Two electron density maps were independently interpreted, and the fragments of the two models were combined into a single model, since the two maps were complementary to each other. Non-crystallographic symmetry (NCS) matrices were found from a partial model built from the initial electron density map, and phases were further improved by the 2-fold NCS averaging, solvent flattening and histogram matching with the program DM (13). The model was built with O (14). The model was refined with the program CNS (15), including the bulk solvent correction. 10% of the data were randomly set aside as the test data for the calculation of Rfree (16). Several rounds of model building, simulated annealing, positional refinement and individual B-factor refinement were performed. Subsequently, this model was used to refine the structure of the native protein. The model has excellent stereochemistry, as evaluated by the program PROCHECK (17). Refinement statistics are summarized in Supplementary Table 1.<br><br>
RESULTS AND DISCUSSION
Model quality and monomer structure
The structure of <span class="yellow">P. aeruginosa</span> RdgC was solved using two sets of MAD data collected from a crystal of the selenomethionine (SeMet)-substituted protein and from a mercury derivative crystal of the native protein (Supplementary Table 1). The model has subsequently been refined using the 20.0–2.50 Å data from a native crystal to crystallographic Rwork and Rfree values of 23.4 and 27.9%, respectively, with no sigma cut-off. The refined model contains 612 residues of the two monomers in the asymmetric unit of the crystal (residues 1–306 for both chains A and B) and 87 water molecules. The C-terminal eight-residue fusion tag has no electron density in both subunits and has not been modeled. Conformations of the two monomers in the asymmetric unit are similar. The root-mean-square (r.m.s.) difference between the two monomers is 0.69 Å for 306 Cα atom pairs, with a maximum deviation of 8.8 Å for the Cα atom of Gly305. The residues giving r.m.s. differences larger than 2.0 Å are 14–16, 94–96, 201–202 and 303–306, which correspond to loops connecting secondary structure elements or the C-terminus. They are located on the surface of the dimer. Subunit A has a better map quality than subunit B, and thus has a lower mean B-factor than subunit B (40.3 Å2 versus. 56.1 Å2 for all 4788 atoms) (Supplementary Figure 1). Therefore, subunit A is chosen for discussions, unless otherwise stated.
The <span class="yellow">P. aeruginosa</span> RdgC monomer is J-shaped and has approximate dimensions of 72 Å × 60 Å × 40 Å (Figure 1). It can be divided into three structural domains: center domain, tip domain and base domain. Center domain (residues 1–73 and 121–167) includes a six-stranded anti-parallel β-sheet with two flanking α helices (α1 and α4). The helix α1 runs nearly parallel to the two outermost strands β2 and β3, which in turn lie adjacent to β4. Strands β2 and β3 are separated by a loop. The helix α4 is nearly orthogonal to the β-strands. Tip domain (residues 74–120) contains two α helices (α2 and α3) and is inserted between strands β4 and β5 of the center domain. It sticks out from the center domain. The two connections between the center domain and the tip domain are rich in conserved residues, hinting at an important functional role of these joints. They could act as a hinge for a possible conformational change. The C-terminal region of the polypeptide chain is folded into the base domain (residues 168–306), which consists of a five-stranded anti-parallel β-sheet with four α-helices, three of which (α5, α6 and α8) cover one side of the β-sheet. The long helix α8 is slightly bent.<br><br>
Structural similarity searches
When we searched for structural similarity against the Protein Data Bank database using the program DALI (18), the <span class="yellow">P. aeruginosa</span> RdgC monomer showed no significant similarity with a Z score above 5. Therefore, we conclude that the overall fold of the RdgC polypeptide chain is unique. When we elaborated the structural comparisons with individual domains of <span class="yellow">P. aeruginosa</span> RdgC, only the center domain gave a Z score above 5.
Using the center domain (residues 1–73 and 121–167) alone, the highest structural similarity is found with the <span class="yellow">human</span> β2-adaptin appendage domain (PDB code 1E42, Z score = 7.1, r.m.s. deviation = 2.5 Å for 85 structurally aligned residues, residues 1–12, 14–30, 44–47, 58–72, 122–133 and 135–160 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 838–841, 848–880, 885–906 and 912–967 of the <span class="yellow">human</span> β2-adaptin appendage domain chain A) (19). There does not appear to be any functional relatedness between these two proteins. The next highest similarity is found with the <span class="yellow">yeast</span> TATA-box-binding protein (PDB code 1YTB, Z score = 5.7, r.m.s. deviation = 2.6 Å for 79 structurally aligned residues, residues 4–12, 14–19, 21–27, 56–70, 124–132 and 134–166 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 69–96, 100–126 and 132–155 of TATA-box-binding protein, N-terminal half of chain A) (20). The TATA-box-binding protein is composed of a ten-stranded antiparallel β-sheet with four flanking α-helices on the convex side of the β-sheet (20). It has an internal quasi 2-fold symmetry, and its β-strands are involved in DNA binding. There is, however, no structural similarity of functional significance, because the two center domains of the RdgC dimer are well separated and do not associate into a single unit (Figure 1C and D).
Using the tip domain (residues 74–120) alone, the ATPase domain of <span class="yellow">bovine</span> Hsc70 chaperone shows highest structural similarity (PDB code 1BA1, Z score = 4.8, r.m.s. deviation = 1.7 Å for 45 structurally aligned residues, residues 76–120 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 230–253 and 256–276 of the Hsc70 ATPase domain) (21). There does not seem to be any significant functional relationship between these two proteins. The α-helix-loop-α-helix structure of the tip domain is reminiscent of the helix–hairpin–helix (HhH) DNA-binding motif that is present in a number of DNA repair enzymes (22–25). The HhH motifs are characterized by the conserved sequence motif hxxhxGhGxxxAxxxhh, where h is any hydrophobic residue (VILMWFYA) and x any residue (23). Two conserved glycines allow a tight turn between two α-helices. However, the tip domain of <span class="yellow">P. aeruginosa</span> RdgC protein does not have such a sequence motif, and the loop in the RdgC tip domain is much more extended than that of the HhH motifs. Therefore, the RdgC tip domain is uniquely folded.
With the base domain (residues 168–306) alone, the highest similarity is found with <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase (unpublished deposition; PDB code 2H9C, Z score = 4.0, r.m.s. deviation = 4.8 Å for 51 structurally aligned residues, residues 186–191, 237–240, 257–272 and 274–298 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 36–41, 50–53, 55–66 and 68–96 of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase chain A). There does not appear to be any functional relationship between the two proteins, and the base domain of RdgC is uniquely folded. The 101-residue chain of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase is folded into three α-helices, two of which overlap with two α-helices (α2 and α3) of the base domain of RdgC.<br><br>Ring-shaped dimer structure and inter-subunit interface
Our structure reveals that <span class="yellow">P. aeruginosa</span> RdgC forms a ring-shaped dimer of approximate 2-fold symmetry in the crystal (Figure 1). This observation is consistent with the results of previous gel filtration and sedimentation equilibrium experiments, which suggested the presence of dimeric species of <span class="blue">E. coli</span> RdgC in solution (1,2). Depending on the concentration, RdgC could also exist in solution as monomers, tetramers and higher oligomers (1). In the crystal lattice, no monomeric, trimeric or tetrameric species exist. Our structure unequivocally rules out the monomer–trimer model, which fits the sedimentation equilibrium data approximately, as well as the monomer–dimer–tetramer model (1). The RdgC dimer has approximate dimensions of 80 Å × 60 Å × 40 Å, with a central hole of ∼30 Å diameter.
Only a moderate amount of surface area is buried upon the formation of the RdgC dimer. The monomer–monomer interface buries a solvent-accessible surface area of ∼1360 Å2 per monomer, corresponding to ∼8% of the total surface area of the monomer (Protein–Protein Interaction Server at http://www.biochem.ucl.ac.uk/bsm/PP/server/). Polar and nonpolar atoms in interface are 44 and 56%, respectively. The interface can be divided into two kinds: ‘interface A’ involving both the tip domains and center domains, and ‘interface B’ involving the base domains (Figure 2A). These two interfaces have significantly different characters.<br><br>The interface A is formed largely by the crossing over of the two tip domains in the dimer (Figure 1). This interface covers a surface area of ∼565 Å2 and is largely nonpolar, with the polar and nonpolar atoms in the interface being 37.5 and 2.5%, respectively. Phe120 contributes considerably to the nonpolar character of the interface between the tip domains, accounting for ∼150 Å2 of the interface area. Phe120 is part of a highly conserved sequence segment and lies at the boundary between the tip domain and the center domain. The aromatic ring of Phe120′ inserts into a hydrophobic pocket, which is formed by the hydrophobic residues Ile74, Leu75, Pro76, Val79, Leu115, Ala119 and Phe120 of the neighboring monomer (Figure 2B). The primed residue belongs to the adjacent subunit. These residues are well conserved in the RdgC family (indicated by green triangles below the aligned sequences in Figure 3). As these hydrophobic residues are located near the molecular 2-fold axis, the two clusters of hydrophobic side chains are merged into a single, large cluster in the dimer (Figure 2B). Of the 16 hydrogen bonds that exist between the two subunits, only one is made in this interface between Arg118 and Arg118′.<br><br>The interface B is contributed to exclusively by the two base domains. This interface covers a surface area of ∼795 Å2. The polar and nonpolar atoms in this interface are 50% each. The two β-sheets of the two base domains merge into a single ten-stranded antiparallel β-sheet in this interface (Figure 2A). Thus, 15 of 16 inter-subunit hydrogen bonds are clustered in this interface. The hydrogen bond networks can be divided into two types. The first type involves the main chain atoms of Val206, Arg208 and Lys210 in strand β9; this strand associates with the equivalent strand from the other subunit in an antiparallel manner (Figure 2C). The other type involves both the side chain and main chain atoms. The side chain of Arg211′ interacts with the backbone of Gly204, while the side chain of Lys227 hydrogen bonds to those of both Gln212′ and Glu218 (Figure 2D). In addition, the side chain of Gln212′ interacts with that of Asp199 and the backbone of Arg211′ (Figure 2D). Arg211, Gln212, Glu218 and Lys227 are highly conserved among the RdgC family members. The residues participating in the hydrogen bond networks are indicated by red circles below the aligned sequences in Figure 3. The observation that conserved residues play important roles in forming the dimer implies that dimerization of RdgC is critical for its function.
If one or both of interfaces A and B are to be broken for functional reasons such as for allowing dsDNA to enter into the central hole of the RdgC dimer, we can conceive of three scenarios. The first possibility would be a simultaneous breakage of the two interfaces A and B. The second would be the sequential disruption of the interfaces A and B. The third would be the disruption of only one of the two interfaces A and B. Sedimentation equilibrium indicated the presence of monomers in solution (1), which is consistent with the first two scenarios. If the third scenario holds, RdgC dimers could possibly exist in two conformations in solution, i.e. a closed form as observed in the present crystal structure, and an open form, in which one of the two interfaces A and B is broken.<br><br>A model for DNA binding by RdgC and DNA-binding assay
The net charge of <span class="yellow">P. aeruginosa</span> RdgC is highly negative, since each monomer contains 12 aspartate, 13 glutamate, 10 lysine, 9 arginine and 2 histidine residues. The net charge of a dimer would be −12 (or −8), if we assume none (or all) of histidines to be positively charged. The electrostatic potential at the molecular surface of RdgC dimer shows that the charge distribution on the protein surface is highly nonuniform (Figure 4A). Since the inner surface of the RdgC dimer is rich in conserved, positively charged residues and the diameter of the central hole (∼30 Å) is approximately similar to that of other toroidal DNA-binding proteins (26), it appears likely that RdgC dimers bind dsDNA through the central hole. Despite extensive co-crystallization and soaking trials with linear dsDNAs of different lengths (14-mer, 16-mer and 18-mer), we could not locate the bound DNA experimentally. We attribute this difficulty to the nature of sequence-nonspecific DNA binding by RdgC. Therefore, we built a crude model of a DNA complex of the RdgC dimer by fitting dsDNA into the central hole (Figure 4B). This model is supported by the DNA-binding assays coupled with mutagenesis, as described below. It is also consistent with the suggested binding site size of ∼16 nt for the <span class="blue">E. coli</span> RdgC dimer (1) and the approximate thickness (∼40 Å) of the <span class="yellow">P. aeruginosa</span> RdgC dimer.<br><br>When we performed electrophoretic mobility shift assay for dsDNA binding, <span class="yellow">P. aeruginosa</span> RdgC did not require Mg2+ ions (Supplementary Figure 2), unlike <span class="yellow">Deinococcus radiodurans</span> RecR, which required high concentrations of Mg2+ ions (27). ATP was not required for dsDNA binding by <span class="yellow">P. aeruginosa</span> RdgC, either (Supplementary Figure 2). This is consistent with an apparent lack of an ATP-binding pocket in the RdgC dimer structure.
In order to identify the residues that are important for dsDNA binding, we carried out mutational analyses. The residues for mutagenesis were selected on the basis of the modeled complex between RdgC dimer and dsDNA, together with sequence conservation (Figure 4C). We prepared eleven point mutants (R4A, K70D, R81D, F120A, R122D, K146A, R198A, R208A, R211D, Q212A and R252A). These eleven mutated residues are well conserved among RdgC proteins. Eight mutants (R4A, K70D, R81D, R122D, K146A, R198A, R208A and R252A) had a lower affinity for dsDNA than the wild-type RdgC (Figure 4D). These eight mutated residues are indicated by blue squares below the aligned sequences in Figure 3. This result indicates that dsDNA binds to the inside surface around the central hole of the RdgC dimer, since the mutated residues are located on the inner surface of the RdgC dimer. It is worth mentioning that a single mutation introduces two alterations per RdgC dimer.
The three mutants (F120A, R211D and Q212A) were prepared to investigate whether dimerization of RdgC is crucial for dsDNA binding. Phe120 is a key residue located in the inter-subunit interface A (Figure 2B), while Arg211 makes hydrogen bonds in the interface B (Figure 2D) and Gln212 is involved in both intra- and inter-subunit interactions (Figure 2D). dsDNA binding by the F120A mutant is considerably weakened, compared to either the R211D or Q212A mutant (Figure 4D). This result suggests that destabilization of the ring-shaped architecture of the RdgC dimer is deleterious for tight dsDNA binding and that interface A is probably more important than interface B for dsDNA binding. dsDNA-binding affinity of the Q212A mutant was lower than the wild type (Figure 4D), whereas that of the R211D mutant was comparable to the wild type (Figure 4D). This result suggests that disruption of the inter-subunit interface B is also detrimental for dsDNA binding but to a smaller extent than disruption of the interface A. It further suggests that the side chain of Arg211 is not involved in recognizing dsDNA. It is interesting that the DNA complex for wild type forms a distinct band (lane 3 in Figure 4D), as opposed to a more diffuse band as one would expect if the complexes contained different numbers of RdgC dimers. Assuming a binding site size of ∼16 nt (1), the 414-bp duplex DNA would bind ∼25 RdgC dimers if it is fully saturated.<br><br>RdgC is unique among ring-shaped DNA-binding proteins
<span class="yellow">Pseudomonas aeruginosa</span> RdgC adds a unique member to the repertoire of toroidal DNA-binding proteins (26) such as <span class="yellow">D. radiodurans</span> RecR (27) and DNA polymerase processivity factors, including the β subunit of <span class="blue">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29) and PCNA of eukaryotic DNA polymerase δ (30,31) (Figure 5). These proteins do not have an overall sequence similarity among themselves, and their quaternary structures also vary. <span class="yellow">P. aeruginosa</span> RdgC and <span class="blue">E. coli</span> β clamp are homodimers, T4 gp45 and eukaryotic PCNAs are homotrimers and <span class="yellow">D. radiodurans</span> RecR is a homotetramer.<br><br>In <span class="yellow">P. aeruginosa</span> RdgC, the inner surface of the ring-shaped dimer is largely formed by β-strands, while the outer surface of the ring is mostly formed by α-helices (Figure 5). The predicted DNA-binding site of <span class="yellow">P. aeruginosa</span> RdgC dimer is primarily formed by β-strands and loops. In contrast, in other ring-shaped DNA-binding proteins such as the β subunit of <span class="blue">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29), PCNA of eukaryotic DNA polymerases δ (30,31) and <span class="yellow">D. radiodurans</span> RecR (27), α-helices are located toward the center of the ring and are surrounded by β-strands in the outermost side of the ring (Figure 5). Despite a superficial resemblance of the overall ring-shaped architecture of the RdgC dimer with those of other toroidal DNA-binding proteins, the internal arrangement of the secondary structure elements in the RdgC dimer is strikingly different from those of other toroidal DNA-binding proteins. Thus, RdgC is unique among ring-shaped DNA-binding proteins.
A number of other proteins involved in DNA metabolism also adopt a ring-shaped structure despite diversity in the molecular functions (26). They include NAD+-dependent DNA ligase (25), λ exonuclease (32), hexameric helicases, topoisomerases, the trp RNA-binding attenuation protein, the <span class="yellow">bacteriophage</span> head-to-tail connector and translin (26). There are also many examples of the proteins in DNA recombination pathways that assemble into multi-subunit toroids containing a central channel (26). Electron microscopy showed that the <span class="blue">E. coli</span> RuvB protein, in the presence of ATP, forms a dodecamer on double-stranded DNA in which two stacked hexameric rings encircle the DNA and are oriented in opposite directions with D6 symmetry (33). The <span class="yellow">human</span> RAD52 protein was found to assemble into heptameric rings with a large funnel-shaped channel, 40–60 Å in diameter (34). The <span class="yellow">human</span> DMC1 protein has so far been detected only as an octameric ring (35).<br><br>Inhibition mechanism of RecA function by RdgC
RdgC inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). It was observed that the effects of RdgC on RecA-promoted DNA strand exchange were much more pronounced than the effects of RdgC on other RecA functions, when RdgC was added to the reaction late (1). By examining the effects of the DinI protein on RdgC-mediated inhibition of RecA activities, it was concluded that the potent effect of RdgC in the inhibition of strand exchange did not primarily reflect a displacement of RecA in the nucleoprotein filament by RdgC, but instead a binding of RdgC to the dsDNA substrate so as to make it unavailable to the filaments for strand exchange (1).
Our work contributes to a better understanding of the inhibition mechanism of RecA function by RdgC. The ring-shaped architecture with a central hole that is suitable for the binding of dsDNA lends RdgC with a capacity to hold dsDNA very tightly. Therefore, we suggest that RdgC dimers probably function as a tight dsDNA gripper and physically block access to the target dsDNA by RecA filaments, because tight encircling of dsDNA by RdgC dimers, possibly as a linear aggregate, would prevent the two strands from dissociating. However, it remains unclear how the closed-ring structure of the RdgC dimer might allow binding of a large number of RdgC dimers on the duplex DNA. It may be conceivable that each RdgC dimer can open temporarily at one or both of its two inter-subunit interfaces so that dsDNA can enter the central hole.
RdgC was at its highest level during exponential phase, reaching its maximum of ∼1000 dimers per <span class="blue">E. coli</span> cell. Its level decreased sharply to ∼50 dimers per cell in stationary phase (2). Under normal cellular conditions, RecA is maintained at ∼1000 molecules per cell (http://www.els.net/). Following LexA repressor cleavage, the level of RecA protein in the cell increases by as much as 20-fold. RdgC inhibition of the RecA function by a simple competition mechanism at the cellular concentration of RdgC clearly requires a much higher affinity for DNA by RdgC than RecA. Possible protein–protein interactions between RdgC dimers might have a profound effect on the affinity for DNA, when a large number of RdgC dimers are bound on a duplex DNA. If only one of the two inter-subunit interfaces can open up and the 2-fold symmetry axes of the RdgC dimers are misaligned sufficiently, the escape of DNA from a linear array of interacting RdgC dimers would be extremely difficult. Alternatively, both inter-subunit interfaces of each dimer may open up simultaneously. A line of indirect evidence for the possible protein–protein interactions between RdgC dimers is provided by the cooperative binding of RdgC to DNA (7). Cooperativity of RdgC binding may be caused by local and distant protein–protein interactions (7). A kind of side-by-side interaction between two adjacent molecules of RdgC dimer is present in the crystal, burying an accessible surface area of ∼750 Å2 at the interface. The interaction involves mostly the less polar side surface of the dimer (left side of Figure 4A), with 2-fold symmetry axes of the two interacting RdgC molecules making an angle of ∼90° when viewed down the central holes and ∼15° when viewed perpendicular to the holes. This may explain the observed DNA bending by RdgC, which was speculated to be a necessary step in the working mechanism of RdgC, if one can assume that the strong protein–protein interactions in the crystal were relevant for protein–protein interactions in solution (7).
RdgC protein was shown to have a potent effect on RecA protein-promoted DNA strand exchange in <span class="blue">E. coli</span> (1). In a series of experiments, RdgC was added after RecA filaments had formed on the ssDNA and 5 min after the linear dsDNA was added. With 3 μM RecA, a sharp reduction in DNA strand exchange products was seen with 0.4 μM RdgC, and the generation of products was abolished at 0.8 μM RdgC (1). This suggests that the binding affinity of RdgC to dsDNA is considerably higher than that of RecA. The high affinity of RdgC could be explained by its ring shape and the suggested mode of DNA binding. Tetrameric species of <span class="blue">E. coli</span> RdgC, which started to form at around 2 µM concentrations, were speculated to be the species with the most potent inhibitory effect, possibly by having a higher affinity for DNA (1). Higher affinity for DNA of the tetrameric species could be understood, if the formation of a tetramer in solution involves a protein–protein interaction similar to the side-by-side interaction between two adjacent RdgC dimers in the crystal, although there is no experimental evidence. Obviously, there are still many open questions that need to be addressed in future biochemical experiments.<br><br>
SUMMARY
The crystal structure of RdgC, a recombination-associated DNA-binding protein, determined in this study reveals that the polypeptide chain is uniquely folded and two monomers associate to form a ring-shaped dimer with a central hole of ∼30 Å diameter. The inner surface around the central hole is rich in positively charged residues. These structural features suggest that dsDNA runs through the central hole when it binds to RdgC dimers. This idea is supported by our mutational studies, which indicate that the conserved, positively charged residues around the central hole are important for dsDNA binding. The toroidal architecture of the RdgC dimer provides a sound framework for a better understanding of its role in homologous recombination.
Coordinates
Atomic coordinates and the structure factor data have been deposited in the Protein Data Bank (accession code 2OWY).<br><br>
SUPPLEMENTARY DATA
Supplementary Data are available at NAR Online.<br><br>Supplementary Material<br><br>
</body></html>