
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Escherichia coli</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="blue">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1851977</h3>Molecular Basis for a Lack of Correlation between Viral Fitness and Cell Killing Capacity
Abstract
The relationship between parasite fitness and virulence has been the object of experimental and theoretical studies often with conflicting conclusions. Here, we provide direct experimental evidence that viral fitness and virulence, both measured in the same biological environment provided by host cells in culture, can be two unrelated traits. A biological clone of <span class="yellow">foot-and-mouth disease virus</span> acquired high fitness and virulence (cell killing capacity) upon large population passages in cell culture. However, subsequent plaque-to-plaque transfers resulted in profound fitness loss, but only a minimal decrease of virulence. While fitness-decreasing mutations have been mapped throughout the genome, virulence determinants—studied here with mutant and chimeric viruses—were multigenic, but concentrated on some genomic regions. Therefore, we propose a model in which viral virulence is more robust to mutation than viral fitness. As a consequence, depending on the passage regime, viral fitness and virulence can follow different evolutionary trajectories. This lack of correlation is relevant to current models of attenuation and virulence in that virus de-adaptation need not entail a decrease of virulence.
Virulence expresses the harm that parasites inflict upon their hosts. Many studies have addressed the basis of virulence and its effect on host and parasite survival. It has generally been accepted that one of the components of parasite virulence is fitness, or the capacity of the parasite to multiply in its host. Some models have equated virulence with fitness. In the present study, we use <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) to document that virulence and fitness—measured in the same biological environment provided by cells in culture—can be unrelated traits. This has been achieved by multiplying the virus in a manner that mutations accumulated in its genome. Mutations decreased fitness dramatically, but not virulence. Chimeric and mutant viruses were constructed to show that virulence is influenced by only some of the <span class="yellow">FMDV</span> genes, while fitness is influenced by the entire genome. For this reason, virulence is more robust (“resistant”) than fitness to the effects of deleterious mutations. The fact that virulence can be unrelated to fitness has implications for the design of anti-viral vaccines because it suggests that it may be possible to design high fitness, low virulence strains to stimulate the host immune response. Furthermore, in modelling studies it cannot be assumed that virulence is equal to fitness.<br><br><br><br>Introduction
The relationship between fitness and virulence is an unsettled question, and sometimes fitness is considered a component of the virulence phenotype of parasites. RNA viruses are ideal systems to address this important question because of their high mutability and fecundity, which result in a potential for rapid evolution, and also because of the availability of quantitative fitness and virulence assays.
RNA viruses replicate as complex and dynamic mutant spectra, termed viral quasispecies. Key to quasispecies dynamics are mutation rates in the range of 10−3 to 10−5 substitutions per nucleotide copied, and competition among continuously arising variant genomes [1–4], which prompt rapid movements in sequence space, with corresponding changes of position in the fitness landscape [5]. Indeed, large population passages of RNA viruses in cell culture permit competitive optimization of mutant distributions that generally result in fitness gain [6,7], while repeated bottleneck events (experimentally realized as plaque-to-plaque transfers) lead to random accumulation of deleterious mutations (operation of Muller's ratchet [8]) and result in average fitness decreases [9–13]. Fitness recovery of low fitness <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) clones occurs mainly with introduction of mutations along the genome, with very few true reversions.
An understanding of the consequences of fitness variation for viral virulence is a key question for viral pathogenesis and evolution. Here, we approach this issue with <span class="yellow">FMDV</span>, an important viral pathogen in veterinary medicine [14], and one that fully participates of quasispecies dynamics. Our laboratory has characterized multiple <span class="yellow">FMDV</span> variants that derive from one original biological clone, and that occupy widely different fitness levels when replicating in a defined environment in cell culture. We define fitness as the replication capacity of a mutant <span class="yellow">FMDV</span>, relative to a reference <span class="yellow">FMDV</span>, in direct growth-competition upon coinfection of baby <span class="yellow">hamster</span> kidney 21 (BHK-21) cells [15–17]. Fitness of <span class="yellow">FMDV</span> in BHK-21 cells is a multigenic trait [7].
In the present study, we define virulence of <span class="yellow">FMDV</span> as the capacity of the virus to kill BHK-21 cells under a standard set of cell culture conditions [18]. Thus, the <span class="yellow">FMDV</span>–BHK-21 system offered a means to investigate in a direct and comparative fashion the relationship between fitness and virulence of a virus, measured in the same biological environment provided by BHK-21 cells. We describe the behavior of an <span class="yellow">FMDV</span> clone (
				), which has a history of repeated serial plaque-to-plaque transfers in BHK-21 cells [11], that attained a very low fitness value relative to its parental reference virus (C-S8c1), and yet, its virulence for BHK-21 cells was significantly higher than that of C-S8c1. A comparative study of the capacity to kill BHK-21 cells of chimeric <span class="yellow">FMDVs</span> constructed with cDNA copies of the two parental <span class="yellow">FMDVs</span> indicates that the enhanced virulence for BHK-21 cells of the low fitness clone is a polygenic trait, with participation of the regions encoding capsid proteins and non-structural proteins 2A, 2B, and 2C as virulence determinants. Three specific amino acid replacements in 2C have been identified as redundant virulence determinants of <span class="yellow">FMDV</span> for BHK-21 cells. Thus, while large population passages of the virus resulted in a gain of both fitness and virulence, subsequent bottleneck passages resulted in a decrease of fitness but not of virulence.
			
The results suggest that fitness is very vulnerable to mutation in any genomic region. In contrast, because of the involvement of several (but not all) viral genes in virulence, and the redundant effect of three 2C substitutions, virulence is a more robust phenotypic trait than fitness, and less vulnerable to accumulation of mutations. Therefore, we provide direct evidence that viral fitness and capacity to kill cells can (in some cases) be unrelated traits. Furthermore, the relationship between fitness and virulence, of being either linked or unrelated traits, depends on the evolutionary history of the virus. This observation has implications for viral pathogenesis, and sheds light on models of virulence proposed on the basis of theoretical and experimental studies with cellular organisms.<br><br>Results
Inability of <span class="yellow">FMDV</span> 
					 to Establish a Persistent Infection in BHK-21 Cells
				
Several biological clones and populations were obtained by passaging <span class="yellow">FMDV</span> biological clone C-S8c1 [19–22] in BHK-21 cells, either as large population passages or plaque-to-plaque transfers (Figure 1). The biological clones and populations differed up to 236-fold in relative fitness (Table 1). The fitness differences found are expected from previous results on fitness gain upon large population passages of RNA viruses [6,7] and fitness decrease upon plaque-to-plaque (bottleneck) transfers [9–13]. The initial experiment was aimed at testing whether 
					, because of its low fitness (0.11 times that of its parental C-S8c1 [12,23,24] [Table 1]), had an advantage in establishing a persistent non-cytopathic infection in BHK-21 cells as compared with its parental clone, C-S8c1 (Figure 1). A persistent <span class="yellow">FMDV</span> infection is established by growing the cells that survive a standard cytolytic infection with <span class="yellow">FMDV</span> [25]. Confluent monolayers of BHK-21 cells were infected either with C-S8c1 or with 
					 at a multiplicity of infection (MOI) of 0.02–0.1 plaque-forming units (PFU)/cell (2 × 106 cells infected with 4 × 104 −2 × 105 PFU). Unexpectedly, at 24 h postinfection, the cells infected with 
					 showed extensive cytopathology, and at 48 h postinfection, no surviving cells were observed. The frequency of surviving cells in parallel infections with C-S8c1 was 5 × 10−3–9 × 10−3, which is consistent with previous determinations [25]. No persistently infected BHK-21 cell cultures could be established with 
					, despite several attempts. Thus, C-S8c1, which displays a 9-fold higher relative fitness than 
					 in BHK-21 cells, showed a capacity to kill BHK-21 cells that was at least 103-fold lower than the killing capacity of 
					 in the infectivity assay intended to establish a persistent <span class="yellow">FMDV</span> infection.
				<br><br><span class="yellow">FMDV</span> Fitness and Capacity to Kill BHK-21 Cells May Not Be Positively Correlated
The capacity of 
					 to kill BHK-21 cells despite its low fitness in BHK-21 cells led us to quantitatively examine the relationship between fitness of <span class="yellow">FMDV</span> and its capacity to kill BHK-21 cells. To this aim, <span class="yellow">FMDV</span> clones or populations were compared in a cell killing assay, consisting in determining the time required to kill 104 BHK-21 cells as a function of the PFU added (described in Materials and Methods). The results (Figure 2A) indicate that over the time range of 12 h to 48 h postinfection, the number of PFUs needed to kill 104 BHK-21 cells varied logarithmically as a function of time. Similar quantifications of relative virulence were obtained by measuring the PFU needed to kill 104 cells in 24 h, and then by extrapolating the PFU values to 0 h postinfection (Tables 1 and S1). Virulence of 
					 was 29 to 35 times higher than virulence of C-S8c1, despite the latter displaying a 9-fold higher fitness (Tables 1 and S1). The high virulence of 
					 was not due to the plaque-to-plaque transfers, since a high virulence was also quantitated for its parental clone, 
					, and for population C-S8p113 (Figure 2B; Tables 1, 2, and S1). 
					 deviated from a line that correlated relative fitness of <span class="yellow">FMDV</span> and the logarithm of cell killing capacity, as reflected in the decrease of the regression coefficient (R2) (inset in Figure 2A). Probably, this deviation is due to the fact that 
					 lost fitness due to plaque-to-plaque transfers, and the other viruses were not subjected to plaque-to-plaque transfers. On the other hand, virulence determinants were acquired during the large population passages done between C-S8c1 and C-S8c1p113. The 29- to 35-fold higher virulence of 
					 with respect to C-S8c1 (Tables 1 and S1), despite its low fitness, indicates that viral fitness and virulence can be two unrelated traits.
				<br><br>Mapping Virulence Determinants in the 
					 Genome
				
The comparison of the consensus nucleotide sequence of the 
					 genome with that of C-S8c1 revealed a total of 47 mutations (Table S2), leading to 21 amino acid replacements affecting structural and non-structural proteins (Figure 3). To identify the genomic regions associated with the increased virulence of 
					 with respect to C-S8c1, we measured the BHK-21 cell killing capacity of nine chimeric viruses rescued from constructs obtained by introducing fragments of cDNA of the 
					 genome into plasmid pMT28, which encodes infectious C-S8c1 RNA [21] (Figure 4). The results (Figure 5; Tables 2 and S1) show that several genomic regions contribute to the virulence of 
					 for BHK-21 cells, and that the major contributors map within genomic positions 2046 to 3760 (residues encoding part of VP2, VP3, and part of VP1, Figure 5A) and 3760 to 5839 (residues encoding 2A, 2B, 2C, and 3A, Figure 5B). The results exclude the internal ribosome entry site and the 3C- and 3D-coding regions as significant virulence determinants of 
					 for BHK-21 cells (virulence of the relevant chimeric viruses ≤ 2.5, relative to C-S8c1; Tables 2 and S1). Infectious progeny production by each chimeric virus was intermediate between the production of the parental viruses pMT28 and 
					, with no significant differences that could be correlated with virulence (Table 2).
				<br><br>Non-Structural Protein 2C Is a Determinant of the Virulence of <span class="yellow">FMDV</span> for BHK-21 Cells
Amino acid substitutions in <span class="yellow">human rhinovirus</span> protein 2C promoted cytopathology for <span class="yellow">mouse</span> L cells [26]. Remarkably, 
					 shares with other <span class="yellow">FMDV</span> clones and populations, notably, MARLS and C-S8p260p3d (the two viruses showing the highest virulence for BHK-21 cells; Figure 2A; Tables 1 and S1), three amino acid substitutions in 2C: S80N, T256A, and Q263H. In addition, MARLS and CS8p260p3d include replacement M283V in 2C, relative to C-S8c1 [27,28]. To test whether any (or a combination) of the three shared amino acid substitutions in 2C contributed to the increased virulence of <span class="yellow">FMDV</span>, each of the mutations was introduced individually into plasmid pMT28 by site-directed mutagenesis, as described in Materials and Methods. Transcripts of the three mutants, termed pMT28 (SN), pMT28 (TA), and pMT28 (QH) (Figure 4), were used to transfect BHK-21 cells, and the viruses obtained were tested with the BHK-21 cell killing assay. Viruses having any of the substitutions in 2C have a virulence intermediate between that of C-S8c1 and 
					 (Figure 6A). To test whether the combination of the three substitutions in 2C could produce an additional increase of virulence, the three mutations were introduced in pMT28 to rescue the triple mutant pMT28 (SN, TA, QH) (Figure 4). The results (Figure 6B) show that the virulence of the triple 2C mutant is similar to the virulence of the individual 2C mutants. The 2C mutations did not significantly affect the infectious progeny production (Table 2). A testable prediction of this result is that the introduction of the wild-type 2C-3A-coding region in the genetic background of 
					 should produce a virus with lower virulence than 
					. Indeed, the results with such a chimeric virus (Figure 5D) indicate that the presence of the 2C- and 3A-coding region as the only genetic region of the pMT28 in the genetic background of 
					 resulted in an <span class="yellow">FMDV</span> with a 2.4- to 4.8-fold lower virulence than 
					. We conclude that mutations in 2C contribute to virulence of <span class="yellow">FMDV</span> for BHK-21 cells.
				
Thus, a virus that evolves towards low fitness levels due to the operation of Muller's ratchet may nevertheless maintain its capacity to kill the same cells in which it displays low fitness. In <span class="yellow">FMDV</span>, the enhanced capacity to kill BHK-21 cells was multigenic, including participation of non-structural protein 2C with three amino acid substitutions acting in a redundant fashion. In conclusion, the results provide a molecular interpretation of why fitness and virulence of an animal virus can follow disparate evolutionary trajectories, culminating in two unrelated traits.<br><br>
Discussion
The capacity of a virus to kill cells is probably influenced by several steps in the virus life cycle, including receptor affinity (which may trigger signalling pathways and alter cell functions) and intracellular viral replication that may lead to metabolic alterations such as transcriptional or translational shut-off [29]. A parallel increase of virulence and fitness as a virus improves its adaptation to a host cell type is expected, since a key parameter that should contribute to the fitness level of a cytophatic virus is the accumulation of infectious particles and its release from cells, which are events often associated with cell killing. This expectation was fulfilled in our experiments (increase of both fitness and cell killing following large population passages of C-S8c1 [Figure 2]), and also in other virus–host systems. In a comparison of two genetically divergent isolates of the whispovirus white spot syndrome virus, whose virulence for the <span class="yellow">shrimp</span> host <span class="yellow">Penaeus monodon</span> was measured by in vivo cumulative mortality rates, virulence correlated with competitive fitness in vivo [30]. The onset of type 1 diabetes by coxsackievirus B strains was linked to the viral replication rate and to the infectious dose [31]. In engineered alphavirus replicons, a direct correlation between the level of RNA replication and cytopathogenicity was observed [32]. At an epidemiological level, a greater replicative fitness of historical versus current <span class="yellow">human immunodeficiency virus type 1</span> (<span class="yellow">HIV-1</span>) isolates was taken as evidence of <span class="yellow">HIV-1</span> attenuation over time, assuming a direct connection between fitness and virulence [33]. In vivo, viral fitness may vary among specific organs, and virulence may be affected only when fitness for some specific target tissues is affected [34].
Replicative fitness is, however, but one of several factors which influence the progression of a viral infection in vivo. In a comparative analysis, R5-tropic and X4-tropic clones of <span class="yellow">HIV-1</span> showed similar replication capacity in mitogen-activated T cells. However, X4 clones were transferred more efficiently than R5 clones from dendritic cells to CD4(+) T cells, a fact that can contribute to the competitive advantage of X4 viruses in AIDS <span class="yellow">patients</span> [35]. <span class="yellow">Simian immunodeficiency virus</span> <span class="yellow">SIVmac239</span> infects both the <span class="yellow">sooty mangabey</span> and the <span class="yellow">rhesus macaque</span>, reaching high viral loads in both hosts, yet it is only virulent for the <span class="yellow">rhesus macaque</span> [36]. Deviations of a positive correlation between viral fitness and virulence were observed also in the plant viruses <span class="yellow">cucumber mosaic virus</span> [37] and <span class="yellow">barley stripe mosaic virus</span> [38]. A study of the effect of lysis timing on <span class="yellow">bacteriophage</span> fitness revealed that a strain with an intermediate lysis time had the highest fitness [39]. The time of transmission may also affect virulence. Nuclear polyhedrosis virus transmitted early to its host, the <span class="yellow">moth</span> <span class="yellow">Lymantria dispar</span>, was more virulent than virus transmitted late, although the latter was more productive because the virus could use more host tissue for replication [40]. In a study of the susceptibility of North American and non–North American breeds of Lymantria to several isolates of the fungus <span class="yellow">Entomaphaga maimaiga</span>, mortality was scored in all cases. However, virulence of the fungus, quantitated by the time of death of Lymantria, was, in some cases, inversely proportional to fitness, quantitated by fungal reproduction in the <span class="yellow">moth</span> [41]. In all these cases, the molecular basis of the lack of positive correlation between fitness and virulence is not understood.
Model for a Lack of Correlation between Viral Fitness and Virulence
The results with <span class="yellow">FMDV</span> clones H5 have documented that both fitness-enhancing and virulence-enhancing mutations can be incorporated in the viral genome in such a fashion that subsequent fitness-decreasing mutations associated with bottleneck (plaque-to-plaque) transfers produce only minimal effects on virulence (Figure 2). The dissection of accompanying molecular events, achieved through quantification of virulence of recombinant and mutant genomes (Tables 2 and S1), provides an interpretation of the lack of positive correlation between virulence and fitness. Multiple fitness-decreasing mutations occur in the course of plaque-to-plaque transfers, distributed throughout the <span class="yellow">FMDV</span> genome [11]. In contrast, determinants of virulence for BHK-21 cells are multigenic, but concentrated mainly in some <span class="yellow">FMDV</span> genomic regions. Similar multigenic but discrete virulence determinants have been described also in other virus–host systems [42,43]. To decrease virulence, mutations occurring randomly in the course of plaque-to-plaque transfers should affect specific genomic sites, and this will occur with a lower probability than fitness-decreasing mutations, which can hit any of the multifunctional picornaviral proteins and regulatory regions [11]. This model is reinforced by the observation that three amino acid substitutions in 2C (S80N, T256A, and Q263H) had a similar effect in enhancing <span class="yellow">FMDV</span> virulence, and the three mutations in the same genome had an effect comparable to each mutation individually (Figure 6; Tables 2 and S1). It is not clear what the basis of the contribution of 2C to virulence for BHK-21 could be. 2C is involved in RNA synthesis and contains a nucleotide-binding domain, although none of the substitutions found in 
					 and 
					 lie within such a domain. An unlikely triple reversion would be required to eliminate the virulence-enhancing effect of the three mutations in 2C. We propose that a higher robustness of the <span class="yellow">FMDV</span> genome with regard to virulence for BHK-21 cells, rather than to replicative fitness in the same cells, underlies the different trajectories followed by fitness and virulence upon subjecting the virus to repeated bottleneck transfers. Obviously, we cannot exclude that parameters of the virus life cycle, other than fitness as measured in our experiments, could correlate with virulence for BHK-21.
				
The comparative analysis of <span class="yellow">FMDV</span> clones and populations shows that shifts in virulence can occur even through the evolution of a single viral clone (C-S8c1), with its restricted genetic diversity prompted by different replication regimes in the same host cells, which also have a clonal origin (see Materials and Methods). We conjecture that the demonstration that fitness and virulence can follow different evolutionary courses has been possible thanks to the consequences of the extreme passage regimes to which the viral populations were subjected: competitive evolution of an ample mutant spectra during repeated large population passages, and accumulation of deleterious (with regard to fitness, but not with regard to virulence) mutations upon plaque-to-plaque transfers (predominance of genetic drift and operation of Muller's ratchet) [12,15].<br><br>Implications for Models of Virulence
It must be emphasized that fitness and virulence are relative values that pertain to a defined physical and biological environment. Virulence determinants of <span class="yellow">FMDV</span>, identified here for BHK-21 cells, need not apply to virulence for the natural animal hosts of <span class="yellow">FMDV</span> [44]. However, the observation of a lack of correlation between fitness and virulence in a <span class="yellow">FMDV</span> clone is relevant to current models of attenuation and virulence, since it shows that more virulent forms of a virus need not have a reproductive advantage, and that viral virulence is not necessarily a byproduct of viral fitness. Even if virulence is regarded as an unavoidable consequence of parasite adaptation [45], virus de-adaptation (fitness loss) need not entail a decrease of virulence.
Most current definitions of virulence include both the ability of the pathogen to multiply and to cause harm to its host; some authors, however, assume a direct relationship between fitness and capacity to produce disease [46–48]. In relating the results with <span class="yellow">FMDV</span> to general models of virulence in host–parasite systems, it must be considered that in the <span class="yellow">FMDV</span> system, evolution of the host BHK-21 cells could not influence <span class="yellow">FMDV</span> evolution, because clonal cell populations with a controlled passage history were supplied in constant numbers at each infection event (see Materials and Methods). Therefore, changes in host density, or mobility, as well as pathogen survival in the external environment, all of which are relevant parameters in virulence models [48,49], cannot play a role in our system. A consistent finding in serial passage experiments is that virulence of a parasite increases with passage number in a new host [50]. The results with <span class="yellow">FMDV</span> infecting BHK-21 cells cytolytically imply that the increase of virulence can be conditioned to the history of passage regimes undergone by a virus.
The invariance of BHK-21 cells in the course of serial cytolytic passages of <span class="yellow">FMDV</span> is in contrast with the parallel system consisting of BHK-21 cells persistently infected with <span class="yellow">FMDV</span> C-S8c1 [25], in which the cells are passaged and coevolve with the resident virus [51]. Host–virus coevolution has generally favored a decrease of viral virulence in the field, a classical example being <span class="yellow">myxoma virus</span> and myxomatosis in <span class="yellow">rabbits</span> [52].
Our comparison of <span class="yellow">FMDV</span> clones did not provide evidence of clones with high fitness and low virulence, which, with regard to natural hosts, is an aim of biomedicine to obtain vaccine strains. Yet, the existence of specific mutations that differentially affect fitness and virulence opens the way to engineer candidate vaccine strains unable to kill the host, while maintaining replicative competence. Virulence is, however, a feature of the host–parasite relationship [46], and the mutations needed to impair virulence are expected to be host-dependent [53,54].<br><br>
Materials and Methods
Cells, viruses, and infections.
The BHK-21 cells used in the present study were cloned by end-point dilution, followed by preparation of a cell stock from a single cell; they were passaged a maximum of 30 times before being used for <span class="yellow">FMDV</span> infection [25,51]. Procedures for cell growth, infection of BHK-21 cell monolayers with <span class="yellow">FMDV</span> in liquid medium, and plaque assays in semi-solid agar medium were carried out as previously described [11,19,25,27]. Mock-infected cells were handled in parallel in all infectivity and plaque assays to monitor absence of viral contamination. The <span class="yellow">FMDVs</span> used in the present study (Figure 1) are (i) the reference clone C-S8c1, which has been assigned a relative fitness of 1.0 [11]. (ii) MARLS, a monoclonal antibody escape mutant isolated from population C-S8c1p213 [55]; MARLS has a fitness of 25 relative to C-S8c1 [24]. (iii) C-S8p260p3d, a standard <span class="yellow">FMDV</span> virus rescued by low MOI passage of C-S8p260. The latter is a virus that evolved by passage of C-S8c1 at a high MOI, which resulted in dominance of two defective <span class="yellow">FMDV</span> genomes (both including internal deletions) that were infectious by complementation, in the absence of standard virus [22,24,28]; C-S8p260p3d has a relative fitness of 20 [24]. (iv) REDpt60, obtained after 60 successive plaque-to-plaque transfers of RED (a monoclonal antibody escape mutant isolated from population C-S8c1p100) [20]; REDpt60 has a fitness of 1.9 relative to C-S8c1. (v) C-S8c1p113, a viral population obtained after 113 serial cytolytic passages of C-S8c1 at a high MOI in BHK-21 cells (2 × 106 BHK-21 cells infected with the virus contained in 200 μl of the supernatant from the previous infection). (vi) Clone 
					, a biological clone isolated from population C-S8c1p113 [11]; its relative fitness is 26 (unpublished data). (vii) Clone 
					, obtained after 95 successive plaque-to-plaque transfers of 
					 [11]; its relative fitness is 0.11 [23].
				<br><br>Cell killing assay.
The capacity of <span class="yellow">FMDV</span> to kill BHK-21 cells was measured as previously described [18,22]. The assay consists in determining the minimum number of PFU required to kill 104 BHK-21 cells after variable times of infection. The assay was performed in M96 multiwell plates with monolayers of 104 BHK-21 cells per well infected with serial dilutions of virus. At different times postinfection, cells were fixed with 2% formaldehyde and stained with 2% crystal violet in 2% formaldehyde. Results are expressed as the logarithm of the number of PFUs needed for complete cell killing (as judged by cell staining with crystal violet, with series of control wells with known numbers of cells) as a function of time postinfection [18,22].<br><br>Determination of relative fitness of 
					.
				
The relative fitness of <span class="yellow">FMDV</span> 
					 was determined by growth competition in BHK-21 cells as previously described [7,10,11,24,56]. <span class="yellow">FMDV</span> 
					 was mixed with appropriate proportions of 
					, which was used as reference virus. This virus has a fitness 8.5-fold higher than that of the reference clone of C-S8c1 in BHK-21 cells [7,10,11,24,56]. Four serial infections were carried out at MOI of 0.1 PFU/cell. The proportion of the two competing genomes at different passages was determined by real-time reverse transcription (RT)–PCR, employing primers 5531wtnew and 
					, which are able to discriminate <span class="yellow">FMDV</span> 
					 RNA from 
					 RNA. The nucleotide sequences of the primers will be provided upon request. The fitness vector obtained for 
					 corresponded to the equation y = 0,0206e1,1074x; R2 = 0.9507. The antilogarithm (base e) of the vector slope is the fitness of the assayed virus relative to the reference virus [56].
				<br><br>cDNA synthesis, molecular cloning, and nucleotide sequencing.
Viral RNA was extracted by treatment with Trizol as previously described [57]. Reverse transcription of <span class="yellow">FMDV</span> RNA was carried out with <span class="yellow">avian myeloblastosis virus</span> reverse transcriptase (Promega, http://www.promega.com) or Transcriptor reverse transcriptase (Roche, http://www.roche.com), and PCR amplification was performed by using either Ampli-Taq polymerase (PerkinElmer, http://las.perkinelmer.com) or an Expand High Fidelity polymerase system (Roche), as instructed by the manufacturers. The <span class="yellow">FMDV</span> genome-specific oligonucleotide primers used have been previously described [22,58]. In all RT-PCR amplifications, negative amplification controls, including all reaction components except template RNA, were run in parallel to monitor absence of contamination.
Chimeric viruses containing selected regions of 
					 in the genetic background of C-S8c1 (Figure 4) were obtained by replacing the corresponding DNA fragment of pMT28 by a cDNA copy of 
					 RNA, using specific restriction sites. To obtain pMT28/
					 (436-2046), a chimera that included nucleotides 436 to 2046 of 
					 (the residue numbering of the <span class="yellow">FMDV</span> genome is as in [11]), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28 DNA digested with the same enzymes. To obtain pMT28/
					 (2046–3760), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760). To obtain pMT28/
					 (3760–5839), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839). To obtain pMT28/
					 (5839–7427), 
					 RNA was amplified by RT-PCR using primers 5531 wt new and C-Not-Pol, and then the cDNA was digested with Rsr II (5839) and Bam HI (7427). To obtain pMT28/
					 (436-3760), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and then the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–3760) DNA digested with the same enzymes. To obtain pMT28/
					 (3760–7427), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839), and ligated into pMT28/
					 (5839–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (2046–7427), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760), and ligated into pMT28/
					 (3760–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (436-7427), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2; the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–7427) DNA digested with the same enzymes. To obtain 
					/2C-3A(pMT28), pMT28 was digested with Bgl II (4201) and Rsr II (5839), and a DNA fragment including wild-type 2C-3A-coding region was inserted into pMT28/
					 (436-7427) DNA digested with the same enzymes. DNA ligation, transformation of <span class="blue">Escherichia coli</span> DH5α, isolation of DNA from bacterial colonies, and characterization of DNA by restriction enzyme digestion were performed by standard procedures [59]. The primers used for molecular cloning and site-directed mutagenesis are described in Table S3.
				
To obtain <span class="yellow">FMDV</span> C-S8c1 containing the mutations found in gene 2C of 
					, plasmid pMT28 was subjected to site-directed mutagenesis using an oligonucleotide including the required nucleotide replacement, and 3R2New or 3CD1 as external oligonucleotide primer (Table S3; Figure 4). A DNA fragment termed A was obtained by subjecting plasmid pMT28 to site-directed mutagenesis using primers (reverse) mutSNu, mutTAu, and mutQHu (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3R2New, forward). A DNA fragment termed B was obtained amplifying pMT28 with primers (forward) mutSNd, mutTAd, and mutQHd (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3CD1, reverse). DNA fragments A and B, including the desired mutations, were recombined by shuffling PCR using equimolar amounts of DNA fragments and two external primers (3R2New and 3CD1). The DNA with the desired mutation(s) in the 2C gene was digested with Avr II (genomic position 3760) and Rsr II (position 5839), and cloned into pMT28 to generate pMT28 (SN), pMT28 (TA), and pMT28 (QH). To obtain pMT28 (SN, TA, QH), plasmid pMT28 (SN) was subjected to site-directed mutagenesis to introduce mutation T256A in a similar way as described above, and then, plasmid pMT28 (SN, TA) was subjected to site-directed mutagenesis to introduce mutation Q263H. All chimeric viruses and mutants were analyzed by nucleotide sequencing using Big Dye Terminator Cycle Sequencing kit (Abi Prism; PerkinElmer) and sequencer ABI373 as previously described [58]. Sequences were analyzed using DNASTAR 4.0 (http://www.dnastar.com), GeneDoc, and GCC (University of Wisconsin). Each sequence was determined at least twice, with products obtained using different oligonucleotide primers.
				
DNA from pMT28 or its recombinant and mutant derivatives was linearized with Nde I and transcribed with SP6 RNA polymerase as previously described [22,27]. Transcript RNA integrity and concentration were estimated by agarose gel electrophoresis, in parallel runs with known amounts of standard C-S8c1 RNA. BHK-21 cell monolayers (70% confluent, about 1 × 106 cells) were transfected with RNA transcripts (1 μg RNA) using lipofectin as previously described [59]. Virus was collected from the culture supernatant at 72 h post-transfection. The virus obtained by transfection was passaged twice before using it in biological studies. RNA was extracted and sequenced to ascertain that the virus maintained the genomic structure and mutations of the initial transcript.
Consensus genomic nucleotide sequences of <span class="yellow">FMDV</span> clones were obtained by RT-PCR amplification of virion RNA using specific primers [7,22,28].<br><br>
Supporting Information
Accession Numbers
The GenBank accession numbers for the C-S8c1, 
					, 
					, CS8p260p3d, and MARLS genomic sequences are AJ133357, AM409190, AM409325, DQ409185, and AF274010, respectively. Nucleotide and amino acid sequences for picornaviruses can be found at http://www.iah.bbsrc.ac.uk/virus/picornaviridae/SequenceDatabase/3Ddatabase/3D.HTM.
				<br><br><br><br><h3>pmcA2238978</h3>SuperCAT: a supertree database for combined and integrative multilocus sequence typing analysis of the <span class="yellow">Bacillus cereus group</span> of bacteria (including <span class="yellow">B. cereus</span>, <span class="yellow">B. anthracis</span> and <span class="yellow">B. thuringiensis</span>)
Abstract
The <span class="yellow">Bacillus cereus group</span> of bacteria is an important group including mammalian and insect pathogens, such as <span class="yellow">B. anthracis</span>, the <span class="yellow">anthrax</span> bacterium, <span class="yellow">B. thuringiensis</span>, used as a biological pesticide and <span class="yellow">B. cereus</span>, often involved in food poisoning incidents. To characterize the population structure and epidemiology of these bacteria, five separate multilocus sequence typing (MLST) schemes have been developed, which makes results difficult to compare. Therefore, we have developed a database that compiles and integrates MLST data from all five schemes for the <span class="yellow">B. cereus group</span>, accessible at http://mlstoslo.uio.no/. Supertree techniques were used to combine the phylogenetic information from analysis of all schemes and datasets, in order to produce an integrated view of the <span class="yellow">B. cereus group</span> population. The database currently contains strain information and sequence data for 1029 isolates and 26 housekeeping gene fragments, which can be searched by keywords, MLST scheme, or sequence similarity. Supertrees can be browsed according to various criteria such as species, isolate source, or genetic distance, and subtrees containing strains of interest can be extracted. Besides analysis of the available data, the user has the possibility to enter her/his own sequences and compare them to the database and/or include them into the supertree reconstructions.<br><br>INTRODUCTION
Multilocus sequence typing (MLST) is a tool that is widely used for phylogenetic typing of bacteria. MLST is based on polymerase chain reaction (PCR) amplification and sequencing of internal fragments of usually seven essential or housekeeping genes spread around the bacterial chromosome. The genetic relatedness among isolates is then determined by comparison of the nucleotide sequence types (1,2). MLST is thus a method that is unambiguous and truly portable among laboratories. Since the initial development of this technique for <span class="yellow">Neisseria meningitidis</span> in 1998, MLST schemes have been developed for about 30 species including some of the most important bacterial pathogens, e.g. <span class="yellow">Streptococcus pneumoniae</span>, <span class="yellow">Streptococcus pyogenes</span>, <span class="yellow">Haemophilus influenzae</span>, <span class="yellow">Staphylococcus aureus</span>, <span class="yellow">Campylobacter jejuni</span>, <span class="yellow">Enterococcus faecium</span>, <span class="yellow">Burkholderia pseudomallei</span>, <span class="blue">Escherichia coli</span>, <span class="yellow">Salmonella enterica</span> and the <span class="yellow">Bacillus cereus group</span> (see (1) for a recent review). These MLST schemes have been used successfully to explore the population structure of bacteria, to study the evolution of their virulence properties, to identify antibiotic-resistant strains and epidemic clones, and for epidemiological surveillance.
The <span class="yellow">B. cereus group</span> includes bacterial species that are of medical and/or economic importance, such as <span class="yellow">B. anthracis</span>, an obligate mammalian pathogen causing the lethal disease <span class="yellow">anthrax</span>, <span class="yellow">B. cereus</span>, an opportunistic <span class="yellow">human</span> pathogen involved in food-poisoning incidents and contaminations in hospitals, <span class="yellow">B. thuringiensis</span>, an insect pathogen and one of the world's most widely used biopesticide and <span class="yellow">B. weihenstephanensis</span>, a cold-tolerant species known for contaminating dairies. These species are genetically very closely related and may be considered as one species based on genetic and genomic evidence (3–5). Unlike other bacterial species that are typed using a single MLST scheme, five separate schemes have been developed for the <span class="yellow">B. cereus group</span>, based on different sets of genes and isolates (5–10). The Priest scheme (8) is currently the most widely used. Studies with the various schemes have independently indicated that the <span class="yellow">B. cereus group</span> population is divided into three main phylogenetic clusters and that species are usually intermixed within the groups. One cluster contains the monomorphic <span class="yellow">B. anthracis</span> isolates and a number of <span class="yellow">B. cereus</span> and <span class="yellow">B. thuringiensis</span> strains, many of which are from clinical sources. A second heterogeneous cluster includes <span class="yellow">B. cereus</span> and <span class="yellow">B. thuringiensis</span> isolates from various origins, while cold-tolerant <span class="yellow">B. weihenstephanensis</span> and <span class="yellow">B. cereus</span> isolates belong to the third group. The separate MLST analyses have also revealed that the <span class="yellow">B. cereus group</span> population is weakly clonal overall due to numerous clinical and virulent isolates emerging from different phylogenetic positions (5–8,11–14), with the exception of the ‘cold-tolerant’ cluster that seems to exhibit a panmictic (or sexual) population structure, i.e. with frequent genetic exchanges between strains (9).
Despite the overall congruence between the various MLST studies, the use of separate schemes with no gene overlap and very little strain overlap has produced a confusing situation and makes the results difficult to compare directly. Therefore, we recently proposed a combined scheme based on genes taken from three of the four schemes available by then and for which we created a web-based database accessible at the University of Oslo's MLST server, http://mlstoslo.uio.no/ (5). Here, in order to provide the <span class="yellow">B. cereus group</span> research community with a common MLST resource, we have developed on the same website a database, SuperCAT, that compiles and integrates MLST data from all the published <span class="yellow">B. cereus group</span> schemes. In addition, we applied supertree reconstruction methods to build an integrated view of the <span class="yellow">B. cereus group</span> population and phylogeny. Below we describe the content and main features of the new database as well as the process of supertree building.<br><br>DATABASE CONTENT AND IMPLEMENTATION
The SuperCAT database provides information, sequence and phylogenetic data for all bacterial isolates that have been typed using any of the five published MLST schemes for the <span class="yellow">B. cereus group</span> (Table 1). Strain information, when known, includes isolate description, source and geographical location of isolation, and the scheme(s) used for typing. The sequence data include the nucleotide sequences of the MLST loci examined in a given strain. SuperCAT also contains the phylogenetic supertree of the <span class="yellow">B. cereus group</span> reconstructed by combining the sequence data from all five schemes, as well as supertrees built for individual schemes. Information and sequences for isolates typed by the Priest and Tourasse–Helgason schemes were retrieved from the databases devoted to these schemes at http://pubmlst.org/bcereus and http://mlstoslo.uio.no/, respectively. MLST data for additional strains not available in the pubmlst.org repository (strains from (15) are missing therein) and for the Helgason, Ko, and Candelon–Sorokin schemes were taken from the published literature and the Genbank nucleotide sequence database (Table 1). In addition, sequences of all MLST loci were extracted from the complete genomes of the 21 sequenced <span class="yellow">B. cereus group</span> strains available in Genbank. Altogether, SuperCAT currently contains data for 1029 isolates and 26 gene fragments from 25 different genes. However, since most strains have been typed using only 6 or 7 of the 26 loci, about one-third of the complete set of sequences are included. The 26 loci, only available for the completely sequenced strains, sum up to 10 619 bp. All these genes are located on the chromosome, thus the database provides no information about extrachromosomal plasmids even though most of the strains do carry one or several small and/or large plasmids. Unlike scheme-specific MLST databases, SuperCAT does not contain allele and sequence type (ST) numbers. Since isolates in SuperCAT have been typed by different subsets of loci, complete allelic profiles are unavailable and therefore STs cannot be assigned for most strains, except the fully sequenced ones.
Table 1.The five MLST schemes designed for typing bacteria of the <span class="yellow">B. cereus group</span>SchemeGenesTotal sequence length (bp)Total number of isolateseUsed in (references)Helgasonadk, ccpA, ftsA, glpT, pyre, recF and sucC2 938120(6,12,46)Candelon–Sorokina,cclpC, dinB, gdpD, panC, purF and yhfL2 850149(9,10)Kob,cgyrB, mbl, mdh, mutS, pycA(1) and rpoB2 00265(7)Priesta,bglpF, gmk, ilvD, pta, purH, pycA(2) and tpi2 829721(8,11,13–15,46–48)Tourasse–Helgasona,b,dadk, ccpA, glpF, glpT, panC, pta and pycA(2)2 658172(5)aSpecific databases for the Priest and Tourasse–Helgason schemes are accessible at http://pubmlst.org/bcereus/ and http://mlstoslo.uio.no/, respectively. A BLAST database for the Candelon–Sorokin scheme is available at http://spock.jouy.inra.fr/cgi-bin/bacilliMLSopen.cgi.bWhile the Tourasse–Helgason and Priest schemes use the same gene fragment for the pycA gene, the Ko scheme is based on a different and non-overlapping gene region.cThe <span class="yellow">B. cereus group</span>-specific transcriptional regulator plcR was originally included in the Candelon–Sorokin and Ko schemes. However, plcR follows a phylogeny different from the other MLST loci (7,10) and is no longer used for MLST; therefore, it is not included in SuperCAT.dThe Tourasse–Helgason scheme is a combined scheme based on 3 genes from the Helgason scheme (adk, ccpA, and glpT), 3 genes from the Priest scheme (glpF, pta and pycA(2)), and the panC gene from the Candelon–Sorokin scheme.eIncluding strains with fully sequenced genomes.
SuperCAT is built as a relational database using the PostgreSQL management system, and data are accessible through a graphical web interface. User queries and results pages are processed and created on-the-fly via a highly modified version of the mlstdbNet software (16) written in PERL and based on the DataBase Interface (DBI) and Common Gateway Interface (CGI) modules. The database is implemented on a Linux Apache web server maintained through the facilities and support provided by the Norwegian EMBnet node. Some large supertree computations are run on a Linux supercomputer at the University of Oslo. The ATV (A Tree Viewer) Java applet is used for phylogenetic tree display (17). ATV notably supports horizontal and vertical zooming capabilities that are suitable for browsing large trees. The Jalview editor Java applet is also implemented in SuperCAT for advanced multiple sequence alignment display (18).<br><br>SUPERTREE RECONSTRUCTION
Supertree techniques allow to combine the phylogenetic information from different datasets into a common phylogenetic tree and several studies have shown that meaningful supertrees can be obtained even when taxon overlap is very sparse (see (19,20) for reviews). Supertree analysis has thus become increasingly popular for taking advantage and combining the massive amount of sequence data available in public databases for reconstructing large-scale organismal phylogenies with the ultimate goal of building the tree of life (21–24). In this study, the 21 <span class="yellow">B. cereus group</span> strains that have been completely sequenced, and for which the sequences at all 26 MLST loci are thus available, can be used to join all five schemes and provide the strain overlap necessary for supertree analysis. The global <span class="yellow">B. cereus group</span> supertree, containing 1029 isolates, was reconstructed according to the widely used matrix representation by parsimony (MRP) procedure (Figure 1; (19,25,26)). Scheme-specific supertrees were also reconstructed for each of the five MLST schemes by the same technique. Briefly, a phylogenetic tree is built for every gene separately by the maximum likelihood method with the PHYML_aLRT program (27). Then, each gene tree is recoded into a binary matrix representing the branching order (i.e. the phylogenetic groupings) following standard MRP coding using the SuperMRP.pl script (28). All gene tree matrices are concatenated into a supermatrix, in which isolates missing from a particular tree are coded using the ‘?’ character representing unknown data. In this supermatrix, the sequence of 0's, 1's and ?'s defines the branching profile of a strain. Closely related strains have similar branching profiles. Supertrees are then generated from the supermatrix by the maximum parsimony technique using the program MIX from the PHYLIP package (29) run with default parameters. The maximum parsimony step infers the trees that would require the minimum number of changes between the branching profiles of all isolates, where the unknown characters can take any of the two possible states 0 or 1 (they are not treated as missing gaps). As many trees were equally parsimonious, the final supertree was taken as the strict consensus of all parsimony trees with the CONSENSE program of PHYLIP. In order to obtain branch lengths that are proportional to the amount of nucleotide changes, we added an additional step in which branch lengths and statistical support for groupings are estimated from the concatenated sequences by the maximum likelihood method employing approximate likelihood-ratio tests (aLRTs) for branches using PHYML_aLRT with Shimodaira-Hasegawa-like support values (27,30). aLRTs provide a fast way of testing branch support without requirement of multiple replicates like traditional bootstrap procedures. The Felsenstein-1984 nucleotide substitution model supplemented with a gamma distribution (F84+Γ) was used in maximum likelihood computations for individual gene trees and the supertree (31). This model allows for unequal base frequencies, transition/transversion rate bias, and gamma-distributed substitution rate variation among sites. It was empirically chosen as a consensus from exploratory model testing using ModelTest (32,33), which indicated that models including these three factors were most appropriate for the MLST loci studied, although models for individual loci differed slightly. Note that the maximum likelihood technique also allows for uneven rates of nucleotide substitution between strains, which allows to accommodate slow- and fast-evolving isolates. To reduce the size of the binary supermatrix and speed up computations, individual gene trees and the supertree were built using only one representative from a set of strains having identical sequences. The remaining identical isolates were graphically added to the tree afterwards when drawing the final supertree.
Figure 1.Schematic overview of the <span class="yellow">B. cereus group</span> supertree reconstruction procedure using Matrix Representation by Parsimony (MRP). See text for details.
It should be noted that the global 1029-strain supertree retains the phylogenetic signals from the individual schemes and contains the three main clusters of the <span class="yellow">B. cereus group</span> population described in the section ‘Introduction’. The integrated SuperCAT system may also allow to infer new relationships between strains that were analyzed with different gene sets. Even though the 26 loci sequences are available for only 21 isolates, they apparently provide enough overlap information for building the main branches of the supertree. These 21 isolates cover all three clusters, although the majority of them are <span class="yellow">B. anthracis</span> strains or clinical strains closely related to <span class="yellow">B. anthracis</span> due to the focus of genome sequencing projects, making the part of the supertree containing these isolates likely to be more accurate than the rest of the tree. Furthermore, 111 other isolates have been typed by 10 genes or more, providing additional overlap (see the ‘Gene Distribution’ page). Although about two-thirds of the sequence data are missing overall, it has been shown for other organisms that relevant supertrees could be reconstructed with datasets containing more than 90% of missing data, especially when the characters that are present are informative (20,22,23,34). Empirical and simulation studies have indicated that this behavior may be due to the fact that the characters which are present are more important for the tree-building process than those which are absent (see (20,34) and references therein). Precise within-cluster groupings may contain more uncertainty, as indicated by the large number of unresolved multifurcations in the <span class="yellow">B. cereus group</span> supertree. Finally, it is also worth mentioning that the branching orders of the scheme-specific MRP supertrees are highly correlated to those of the published trees built with concatenated sequences and other phylogenetic algorithms.<br><br>DATA ACCESS AND MANIPULATION
The complete list of isolates included in SuperCAT (currently 1029) with strain description, source and country of origin is available at the database home page. By default all isolates in the database are used in the analysis tools provided, but the user can select strains of interest by keywords, MLST scheme, entering a list of strain identifiers, or choosing isolates individually via checkboxes. All subsequent analyses will be based on the selected strain subset and their loci. The keyword search will look for matches in any of the strain, description, source, location and scheme fields. Complex keyword queries with several logical operators can be formulated in the ‘advanced search’ page. Note that many isolates were referred by alternative names in different MLST schemes and publications, therefore synonyms have been included in the strain descriptions that allow a particular isolate to be looked up using any of its alternative identifiers. A sequence search is also possible using BLASTN (35), in order to select isolates that have allele sequences identical to user-entered query sequences.
Throughout SuperCAT, clicking on a strain name will pop up an isolate-specific window showing all relevant information and giving access to the nucleotide sequences of individual loci for that isolate. Detailed information about the MLST schemes (e.g. loci names and lengths, genomic coordinates, literature references) and their overlap, the distribution of available loci among the isolates, and the supertree reconstruction procedure can be obtained by clicking the relevant links in the header line present at the top of every page.
Apart from the basic functions for selecting and accessing strain information and sequence data for all five <span class="yellow">B. cereus group</span> MLST schemes, the main features of SuperCAT relate to the manipulation of the supertrees constructed by the MRP approach. The global supertree based on the combination of all five <span class="yellow">B. cereus group</span> MLST schemes as well as the five scheme-specific supertrees can be browsed according to various user-chosen criteria (Figure 2). Isolates in the supertrees can be colored by species or source of isolation. It is also possible to specifically mark in red the current subset of strains that has been selected by the user and to extract from the supertrees the subtree containing only those isolates. In the case of the multi-scheme supertree highlighting of the strains can be based on genetic distance. With this option the user can mark on and/or extract from the tree the isolates that are genetically closely related to strains of her/his choice. The user can either select strains that share one or several identical allele sequences with her/his query isolate(s) or that are at a specified genetic distance. Distances between isolates are computed by summing up the lengths of the branches (in average number of nucleotide substitutions per site) connecting the isolates in the supertree (known as patristic distances; (36)). The genetic relatedness search functions are also available in a separate page for the user to find closely related isolates without tree manipulation. SuperCAT allows to compare the scheme-specific MLST supertrees with each other and with the global supertree by using the subset of isolates that are common to all selected schemes. Common isolates can either be highlighted in red or be extracted as subtrees from each supertree, which can be used for comparing the positions of the common strains in the various MLST trees. For all supertree-related options, detailed tree navigation can be achieved using the various functions in the ATV tree window when the trees are displayed (17).
Figure 2.Examples of supertree browsing and manipulation in SuperCAT. A, supertree colored by species; B, specific highlighting of user-selected strains (in red); C, extracted subtree containing only the strains highlighted in B. Trees are displayed using ATV (17).
Besides the manipulation of the precomputed supertrees, SuperCAT offers the user the possibility to compute new supertrees by MRP using any combination of strains, schemes and genes. Supertree computations may be extremely time consuming, ranging from a few minutes to 2–3 days with the complete database. Users are therefore requested to enter their e-mail addresses and will receive a notification containing a link to the results page when the supertree is ready. Note that when building a supertree for a user-selected subset of strains, the computation will first include all database isolates. A subtree containing only the user-selected isolates will then be extracted from the supertree of all strains. Although more time consuming, this strategy allows: (i) to avoid sampling artefacts as phylogenies built with different isolate sets may vary and (ii) to obtain relationships even if the selected isolates have been typed using non-overlapping gene sets, as the supertree of all isolates can always be built owing to the completely sequenced strains that are common to all schemes.
Another main feature of the SuperCAT database is that the user can enter her/his own private sequences and conduct several sequence analyses (Figure 3). These analyses include: (a) building new supertrees containing user isolates and sequences; (b) finding database isolates having sequences most similar to the user's query sequences using an on-line BLASTN (35) service; and (c) aligning user sequences to database genes using the multiple sequence alignment program CLUSTALW (37). For the last option, the Jalview editor (18) is provided for advanced multiple alignment display. All user-entered data must be in FASTA format and can be either copied and pasted into the query forms or uploaded from text files stored locally on the user's computer.
Figure 3.Examples of query results in SuperCAT. A, multi-scheme BLAST search with sequence alignment; B, multi-scheme genetic search showing the list of isolates sharing one or more sequences with a query strain; C, multiple sequence alignment using Jalview (18).
All strain information data, sequences and phylogenetic trees, including user-made supertrees and extracted subtrees, can be saved and downloaded freely from the database. Users wishing to have their MLST data included as part of the SuperCAT release (and/or the Tourasse–Helgason scheme-specific database) are welcome to contact N.J.T. or A.-B.K. at the e-mail addresses given on the Oslo MLST server front page.<br><br>DISCUSSION AND FUTURE DEVELOPMENTS
SuperCAT is a newly created database devoted to the <span class="yellow">B. cereus group</span> of bacteria whose main objectives are to provide a common MLST repository and means for building a comprehensive genetic analysis of the group that has been typed by five separate schemes. The database is publicly and freely available at http://mlstoslo.uio.no/, along with the database specific for the combined scheme of (5). We plan to update SuperCAT quarterly.
Future developments of the database may deal with refining the supertree-building procedure. In particular, a new improved method has recently been developed for taking into account both nucleotide substitutions and recombination events in phylogenies, as part of the ClonalFrame software, which has been applied to the <span class="yellow">B. cereus group</span> and the Priest scheme (38). It would therefore be of interest to examine the suitability of ClonalFrame in the supertree context. It is also tempting to extend the supertree analysis beyond MLST data, by incorporating the phylogenies obtained previously from large-scale multilocus enzyme electrophoresis (MLEE; (3,39–41)) and amplified fragment length polymorphism (AFLP; (42–45)) studies. The MRP framework is ideal since it allows to integrate trees that can be built from different methods and data. As MLEE, AFLP and MLST have different levels of resolution, one can hope that combining them might provide an even more robust supertree for the <span class="yellow">B. cereus group</span>.<br><br>
<h3>pmcA2327290</h3><span class="yellow">Borrelia burgdorferi</span> membranes are the primary targets of reactive oxygen species
Abstract
Spirochetes living in an oxygen-rich environment or when challenged by host immune cells are exposed to reactive oxygen species (ROS). These species can harm/destroy cysteinyl residues, iron-sulphur clusters, DNA and polyunsaturated lipids, leading to inhibition of growth or cell death. Because <span class="yellow">Borrelia burgdorferi</span> contains no intracellular iron, DNA is most likely not a major target for ROS via Fenton reaction. In support of this, growth of <span class="yellow">B. burgdorferi</span> in the presence of 5 mM H2O2 had no effect on the DNA mutation rate (spontaneous coumermycin A1 resistance), and cells treated with 10 mM t-butyl hydroperoxide or 10 mM H2O2 show no increase in DNA damage. Unlike most bacteria, <span class="yellow">B. burgdorferi</span> incorporates ROS-susceptible polyunsaturated fatty acids from the environment into their membranes. Analysis of lipoxidase-treated <span class="yellow">B. burgdorferi</span> cells by Electron Microscopy showed significant irregularities indicative of membrane damage. Fatty acid analysis of cells treated with lipoxidase indicated that host-derived linoleic acid had been dramatically reduced (50-fold) in these cells, with a corresponding increase in the levels of malondialdehyde by-product (fourfold). These data suggest that <span class="yellow">B. burgdorferi</span> membrane lipids are targets for attack by ROS encountered in the various stages of the infective cycle.<br><br>Introduction
<span class="yellow">Borrelia burgdorferi</span>, the causative agent of Lyme disease, survives and proliferates in distinctly different niches, including its arthropod vector and various mammalian hosts. These ‘micro’ environments provide their own distinct sets of advantages and challenges to <span class="yellow">B. burgdorferi</span>. For example, during the initial stages of infection of the mammalian host, immune cells attempt to prevent <span class="yellow">B. burgdorferi</span> from establishing an infection using several systems including those generating bacteriocidal reactive oxygen species (ROS) [e.g. superoxide radicals (O2-), hydrogen peroxide (H2O2) and hydroxyl radicals (OH)] and reactive nitrogen species (RNS) [e.g. nitric oxide (NO) and peroxynitrite] (Storz and Imlay, 1999). In order for <span class="yellow">B. burgdorferi</span> to successfully colonize a new host and cause disease, they must overcome the challenges posed by the innate immune system including the deleterious effects of ROS/RNS compounds.
The effects of ROS/RNS on cells have been extensively investigated. These highly reactive compounds have been shown to damage cellular macromolecules including DNA, proteins and cellular membranes. The damage to membranes can arise through either lipid or membrane protein damage. In eukaryotes, membrane lipids are a major target of ROS. Free radicals attack polyunsaturated fatty acids in membranes and initiate lipid peroxidation. A primary effect of this is a decrease in membrane fluidity which affects the physical properties of the membrane altering the function of membrane-associated proteins. Once lipid peroxides form, they react with adjacent polyunsaturated lipids causing an amplification of the damage. Lipid peroxides undergo further oxidation to a variety of products, including aldehydes, which subsequently react with and damage membrane proteins. However, in bacteria, it is assumed that lipids are not subject to the oxidative damage observed in eukaryotic cells. Only certain polyunsaturated lipids, such as linoleic acid and linolenic acid, are susceptible to oxidation (Gutteridge and Halliwell, 1990), and it is clear that most bacteria do not synthesize or incorporate these types of lipids in their cell membrane. Two notable exceptions are the photosynthetic bacteria which synthesize and incorporate significant levels of linoleic acid in their membrane (Tasaka et al., 1996) and <span class="yellow">Helicobacter pylori</span> membranes which contain between 0.5% and 3% linoleic acid (Ursini and Bindoli, 1987).
Instead, it has been shown that the most damaging effects of ROS in bacteria result from the interactions of H2O2 with ‘free’ Fe2+ (Imlay, 2003), generating very reactive OH (Fenton reaction). Because of this reactivity, the effect on any given biomolecule will depend largely upon proximity to the target. Because Fe2+ localizes along the phosphodiester backbone of nucleic acid, DNA is a major target of OH. This reactive species can pull electrons from either the base or sugar moieties, producing a variety of lesions including single- and double-stranded breaks in the backbone and chemical cross-links to other molecules. These strand breaks and other lesions block DNA replication and contribute to OH toxicity and cell death. Other base damage, which does not hinder replication, may result in a significant increase in mutation rates.
The intracellular biochemistry of <span class="yellow">B. burgdorferi</span> suggests that the primary cellular target of ROS may be distinct from that described in other bacteria such as <span class="blue">Escherichia coli</span>. In <span class="yellow">E. coli</span>, the extent of DNA damage due to H2O2 and Fenton chemistry is directly proportional to Fe metabolism and the free Fe concentration within the cell (10 µM) (Keyer and Imlay, 1996). As the intracellular Fe concentrations of <span class="yellow">B. burgdorferi</span> are estimated to be < 10 atoms per cell (Posey and Gherardini, 2000), it seems unlikely that DNA is a primary target for ROS. Therefore, the purpose of this study is to determine the biochemical effects of ROS on <span class="yellow">B. burgdorferi</span>, including growth effects and biological/physical damage.<br><br>Results
Effect of ROS on <span class="yellow">B. burgdorferi</span> cells
In order to determine what the cellular targets of ROS in <span class="yellow">B. burgdorferi</span> are, we first needed to determine the sensitivity of Borrelia cells to various oxidants. Microaerobic cultures of <span class="yellow">B. burgdorferi</span> strain B31A3 were grown to a cell density of 5 × 107 cells ml−1, treated with varying concentrations of H2O2 (0–50 mM) or t-butyl hydroperoxide (0–50 mM) and the number of surviving cells determined by plating. The results are shown in Fig. 1. When cells were exposed to 10 mM t-butyl hydroperoxide, approximately 50% of the cells survive (Fig. 1A). In contrast, when <span class="yellow">E. coli</span> cells were exposed to 1 mM t-butyl hydroperoxide, approximately 1% of the cells survive. <span class="yellow">B. burgdorferi</span> cells were much more resistant to t-butyl hydroperoxide than <span class="yellow">E. coli</span> cells, with a survival rate of approximately 100% at 1 mM t-butyl hydroperoxide. This trend was observed when cells were exposed to H2O2. <span class="yellow">E. coli</span> cells exposed to 1 mM H2O2 have approximately 10% survivability, while 100% of the <span class="yellow">B. burgdorferi</span> cells survive at this concentration, and approximately 80% survive when exposed to 10 mM H2O2 (Fig. 1B). Similar results were obtained when cells were exposed in HN (Hepes-NaCl) buffer, suggesting that the high-level resistance to ROS was not due to the interaction of ROS with components of Barbour-Stoenner-Kelly (BSK-II) growth medium. Taken together, these data indicated that <span class="yellow">B. burgdorferi</span> strain B31A3 was highly resistant to exposure to both t-butyl peroxide and H2O2.<br><br><span class="yellow">Borrelia burgdorferi</span> DNA is not the primary target of ROS
For most bacteria, DNA is the major target of ROS causing a wide variety of DNA lesions. This is in part due to the localization of ‘free’ Fe2+ along the phosphodiester backbone of nucleic acids, putting the DNA in close proximity to the active species formed via the Fenton reaction. However, <span class="yellow">B. burgdorferi</span> has been shown to harbour few genes encoding orthologues of known iron-containing proteins, does not require Fe for growth and has intracellular Fe concentrations estimated to be < 10 atoms per cell (Posey and Gherardini, 2000). Taken together, these observations strongly suggest that <span class="yellow">B. burgdorferi</span> DNA is not a major target for ROS. To test this hypothesis, different techniques were used to measure DNA damage in <span class="yellow">B. burgdorferi</span> cells after exposure to ROS.
One reliable indicator of DNA damage by ROS in a cell is an increase in the spontaneous mutation rate. In <span class="yellow">B. burgdorferi</span>, mutations that confer resistance to coumermycin A1, which targets the β subunit of DNA gyrase, have been mapped to gyrB, the gene encoding DNA gyrase B (Samuels et al., 1994). In each case, a single point mutation correlated with this drug resistance. To determine if exposure to oxidants increases DNA damage by increasing point mutations, <span class="yellow">B. burgdorferi</span> B31A3 cells grown under microaerobic conditions were treated with 5 mM H2O2 and plated in the presence and absence of 250 ng ml−1 coumermycin A1. The mutation frequency was calculated as the number of colonies that are CouR per total number of cells plated. The spontaneous resistance frequency of treated cells was approximately equivalent to that of untreated cells, 8.8 × 10−8 and 1.33 × 10−7, respectively, indicating no increase in the number of point mutations (Fig. 2A). Also, no increase in point mutations was observed when cells were treated with higher concentrations of H2O2 (10 mM) or when treated with t-butyl hydroperoxide (5 and 10 mM) (data not shown).
Another effective way to determine DNA damage is by measuring the number of apurinic/apyrimidinic sites (AP). AP sites in DNA, where the DNA base is lost, can be generated spontaneously under physiological conditions by hydrolysis of the N-glycosylic bond, or can also be formed by DNA-damaging agents (Lindahl and Nyberg, 1972), such as UV, alkylating agents or OH. They are also intermediates in the base excision repair pathway (BER) (Weiss and Grossman, 1987; Friedberg and Hanawalt, 1988; Wallace, 1988). Thus, the cellular steady state level of AP sites would increase as a consequence of base modifications and their subsequent repair. As AP sites are bypassed inefficiently by DNA polymerase in bacterial cells, DNA lesions can result in a significant block in DNA replication. Therefore, the number of AP sites can serve as a sensitive indicator of DNA damage resulting from oxidative stress (Kubo et al., 1992). To determine whether ROS can damage <span class="yellow">B. burgdorferi</span> DNA, strain B31A3 cells grown under microaerobic conditions were treated with H2O2 (1 or 10 mM), t-butyl hydroperoxide (1 or 10 mM) or lipoxidase (an enzyme which specifically catalyses the hydroperoxidation of lipids containing a cis, cis-pentadiene structure, such as linoleic acid). The DNA was isolated and assayed for AP sites. The results are shown in Fig. 2B. In all cases, the numbers of AP sites per 105 bp DNA were equivalent, indicating that the addition of oxidants did not increase the number of DNA base lesions. In contrast, when <span class="yellow">E. coli</span> strain TA4315 cells (ahpCF) (Storz et al., 1989) were treated with 100 µM H2O2, the number of AP sites increased ∼10-fold over untreated cells. These data suggest that <span class="yellow">B. burgdorferi</span> DNA was not a major target for oxidative damage under these conditions.
It is of note to mention that the total number of AP sites per 105 bp DNA is approximately 10-fold higher in untreated Borrelia B31A3 DNA than in untreated <span class="yellow">E. coli</span> DNA. The Borrelia genome consists of a linear chromosome and multiple linear and circular plasmids. Numerous observations indicate that functional telomeres require interaction with DNA damage repair proteins, suggesting that the DNA damage repair machinery, including the BER, is involved in replication of telomeres and protection of functional chromosome ends (Verdun and Karlseder, 2007). An important intermediate in BER is the apurinic or abasic site. Therefore, it was possible that the higher numbers of AP sites in untreated Borrelia DNA was due to the number of telomeres present in the genome. The <span class="yellow">B. burgdorferi</span> strain B31A3 harbours 11 linear plasmids and a linear chromosome with ∼100 unpaired bases in the telomere loops per genome or 10 bases per 105 bp DNA (Hinnebusch and Barbour, 1991; Chaconas, 2005). Untreated Borrelia B31A3 contains 11 ± 3.8 AP sites per 105 bp DNA, suggesting that the elevated number of AP sites in Borrelia was due to the number of telomeres. To test this hypothesis, the AP sites were measured in DNA isolated from untreated <span class="yellow">B. burgdorferi</span> strain B314 which harbours no linear plasmids (Sadziene et al., 1993). The number of unpaired bases in the telomere loops per genome was estimated to be 10, which corresponds to 1 base per 105 bp DNA, and only 3.6 ± 0.2 AP sites per 105 bp DNA were detected, supporting the hypothesis that the high number of AP sites in untreated Borrelia DNA was due to the number of unpaired bases in the telomeres. The analyses of the AP sites strongly suggested that <span class="yellow">B. burgdorferi</span> DNA was not the major target for oxidative damage.
In addition to the generation of abasic sites, oxygen radicals often damage DNA through the formation of 8-oxoguanine lesions (Nakamura et al., 2000). 8-Oxoguanine, through its ability to mispair with bases other than cytosine, likely plays a role in DNA mutagenesis. Consequently, 8-oxoguanine is often used as a marker of oxidized DNA damage. To determine whether ROS can damage <span class="yellow">B. burgdorferi</span> DNA and cause 8-oxoguanine lesions, strain B31A3 cells grown under microaerobic conditions were treated with H2O2 (5 or 10 mM), t-butyl hydroperoxide (5 or 10 mM) or lipoxidase. The DNA was isolated and assayed for 8-oxoguanine using an Enzyme-Linked ImmunoSorbent Assay. The results are shown in Fig. 2C. In all cases, the amount of 8-oxoguanine in Borrelia cells was below the detection limit of the assay. In contrast, when a MutM (the specific glycosylase that removes the 8-oxoguanine)-deficient <span class="yellow">E. coli</span> strain was treated with 100 µM H2O2, the amount of 8-oxoguanine sites increased ∼fivefold over untreated cells. It is important to point out that no MutM homologue has been identified in the genome of <span class="yellow">B. burgdorferi</span>. Taken together, these data suggested that <span class="yellow">B. burgdorferi</span> DNA was not a major target for oxidative damage under these conditions.<br><br>The membranes of <span class="yellow">B. burgdorferi</span> are targeted during oxidative stress
In eukaryotes, membrane lipids are a major target of ROS. Free radicals can attack polyunsaturated fatty acids, such as linoleic acid and linolenic acid, in membranes and initiate lipid peroxidation. This reaction can cascade throughout the membrane to adjacent polyunsaturated fatty acids, decreasing membrane fluidity and generating more toxic products such as aldehydes (Imlay, 2003). Because most bacterial membranes contain saturated and monounsaturated fatty acids rather than ‘reactive’ polyunsaturated lipids, peroxidation of lipids in bacterial membranes is not considered a problem. As <span class="yellow">B. burgdorferi</span> cannot synthesize their own lipids, they must instead scavenge them. Therefore, it seems likely that their membrane composition would reflect the host's lipid profile or that of their growth medium (Barbour, 1984; Fraser et al., 1997) and would contain some polyunsaturated fatty acids. To determine if <span class="yellow">B. burgdorferi</span> contains polyunsaturated lipids, <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under anaerobic conditions and analysed for fatty acid composition by Lipid Technologies (Austin, MN). The results are shown in Table 1 and are reported as percentage of total fatty acid content. Linoleic acid comprised ∼10% of the total lipid content and the linolenic acid content was measured to be ∼1%, indicating that Borrelia cells do contain lipids that are susceptible to ROS damage. The amount of linoleic acid and linolenic acid present in Borrelia reflected the amount present in the media. Therefore, these results suggested that the amount of these fatty acids in the membranes would vary as availability varies.
To determine if Borrelia polyunsaturated lipids can undergo lipid peroxidation, <span class="yellow">B. burgdorferi</span> B31A3 cells grown microaerobically were treated with 1 mM t-butyl hydroperoxide or 250 mg of lipoxidase, and the cell pellets were analysed for fatty acid composition (Industrial Laboratory). The results are shown in Table 2 and are reported as the percentage of total cell mass. The per cent of linoleic acid in the total cell mass decreased with treatment, while the levels of oleic acid (c18:1n9) and pentadecanoic acid (c15:0) remained relatively constant. Untreated cells contained 0.04% linoleic acid, while cells treated with t-butyl hydroperoxide contained 10-fold less (0.004%) linoleic acid and cells treated with lipoxidase had no detectable linoleic acid present in the sample. These data indicated that the linoleic acid present in <span class="yellow">B. burgdorferi</span> membranes can be oxidized by ROS.
Malondialdehyde (MDA) is generated as a relatively stable end-product from the oxidative degradation of polyunsaturated fatty acids. MDA has thus been used as an indicator of lipid peroxidation (Gutteridge and Halliwell, 1990; Esterbauer et al., 1991). To further demonstrate that Borrelia lipids can undergo lipid peroxidation, B31A3 cells grown microaerobically were treated with 5 mM AAPH (free radical generator) or 250 mg of lipoxidase and MDA measured (Seljeskog et al., 2006). The results are shown in Fig. 3. Untreated cells contained ∼16.5 µM of MDA per mg of protein. When the cells were treated with AAPH or lipoxidase, the amount of MDA increased ∼1.5-fold (27.3 µM of MDA per mg of protein) and approximately twofold (33 µM of MDA per mg of protein) respectively (Fig. 3A). As a control, eukaryotic cells (<span class="yellow">mouse</span> myeloma cells SP2) were treated with AAPH and MDA measured (Fig. 3B). <span class="yellow">Mouse</span> myeloma cells have been used as a model system for the determination of phospholipid hydroperoxides (Chotimarkorn et al., 2005). In this case, the amount of MDA increased approximately threefold (36.3 µM of MDA per mg of protein) in the treated cells versus untreated cells (11.7 µM of MDA per mg of protein). Taken together, these data suggested that, like eukaryotic membranes, Borrelia membrane lipids were capable of undergoing lipid peroxidation.
As shown in Fig. 3A, there is a measurable quantity of MDA present even in untreated cells, suggesting that the membrane lipids are damaged without the addition of exogenous ROS. Oxidative damage is an unavoidable by-product of growth in an oxygen environment because superoxide anions and H2O2 are formed whenever molecular oxygen chemically oxidizes electron carriers. To determine if the Borrelia lipids are damaged from growth in an oxygen environment, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under anaerobic, microaerobic and aerobic conditions and MDA measured. Figure 3C demonstrates that as the oxygen concentration increased, the amount of MDA increased. Cells grown under anaerobic conditions contained the lowest amount of MDA (7.6 µM of MDA per mg of protein), approximately twofold less than the measured amount in microaerobic cells (16.5 µM of MDA per mg of protein). Aerobically grown cells contained the highest amount of MDA, ∼1.5-fold greater than that observed in microaerobic cells (28.8 µM of MDA per mg of protein) and ∼3.7-fold greater than that observed in anaerobically grown cells. These data suggested that Borrelia lipids can be damaged during aerobic growth.
Our High Performance Liquid Chromatography analyses of the MDA present in anaerobically grown <span class="yellow">B. burgdorferi</span> cells showed a small peak with a retention time similar to that of the MDA standard. This was puzzling as little or no lipid peroxidation should occur under these conditions. To further characterize this ‘MDA’ peak, a three-dimensional diode array spectra was generated by scanning each sample during the elution of the peak (Fig. 4, lower sections). An authentic MDA standard was also scanned as a control (Fig. 4A, lower section). In the anaerobically grown cells, the spectrum shows that two compounds with different absorbance maximums (Fig. 4B, lower section) comprised the single retention time peak from the HPLC chromatogram (Fig. 4B, upper section). Based on this spectrum, the amount of actual MDA contributed < 15% of the total amount of material detected in the HPLC peak while the second contaminating peak contributed > 85%. Therefore, the amount of MDA in untreated anaerobically grown cells was considerably less than the 7.6 µM of MDA per mg of protein actually measured. However, in cells grown under microaerobic conditions, the MDA peak contributed more to the overall retention time peak when compared with the anaerobic spectrum, while the second contaminating peak stays relatively constant (Fig. 4C, lower section). These spectra demonstrated that the increase in the MDA retention time peak between anaerobically and aerobically grown cells was due to the increase in the amount of authentic MDA present.
The fluorescent probe diphenyl-1-pyrenylphophine (DPPP) has been used for detection of lipid hydroperoxides in cell membranes (Okimoto et al., 2000; Takahashi et al., 2001). In this method, the hydroperoxides are reduced with DPPP, resulting in the formation of the fluorescent arylphosphine oxide. To visualize the lipid hydroperoxides, <span class="yellow">B. burgdorferi</span> B31A3 cells were labelled with DPPP and observed by fluorescence microscopy (Fig. 5B). <span class="yellow">E. coli</span> Top10 cells and <span class="yellow">mouse</span> myeloma SP2 cells were also labelled and visualized to serve as negative and positive controls respectively (Fig. 5A and C). Red Fluorescent dye was used to visualize the cell membranes. A strong fluorescence of DPPP was observed for B31A3 cells (Fig. 5B) and for the <span class="yellow">mouse</span> myeloma cells (Fig. 5C), but not for the <span class="yellow">E. coli</span> cells (Fig. 5A). An overlay of the two dyes demonstrates that the DPPP fluorescence of both the Borrelia and myeloma cells corresponds to the areas of red fluorescence. This indicated that lipid hydroperoxides were present on the Borrelia cell membranes and suggested that the membranes were damaged.
To further demonstrate Borrelia membrane damage, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under anaerobic and microaerobic conditions and visualized by negative stain using an electron microscope. Additionally, cells grown under microaerobic conditions were treated with 250 mg of lipoxidase and visualized by Electron Microscopy. Intact membranes were observed in cultures of B31A3 grown under anaerobic and microaerobic conditions (Fig. 6A and B respectively). However, in cultures treated with lipoxidase (Fig. 6C), a significant number of membrane blebs were seen surrounding the spirochetes, indicating membrane damage. Taken together, these data indicated that Borrelia membranes were a target for oxidative stress.<br><br>
Discussion
Most bacterial pathogens are faced with the challenge of overcoming ROS generated by the host immune system. These radicals can cause a great deal of damage to biological molecules both in vitro and in vivo. The potential cellular targets for ROS damage in bacteria include DNA, RNA, proteins and lipids, and extensive work has been done to determine the cellular targets of ROS that affect bacterial survival. To date, the most definitive work has been done on <span class="yellow">E. coli</span>. It seems clear from several studies that the most physiologically relevant target of ROS in <span class="yellow">E. coli</span> is DNA. The exposure to µM concentrations of ROS (i.e. H2O2) is sufficient to cause DNA damage, inhibit DNA replication, increase the mutation rate and often lead to cell death. This process leading to DNA damage begins with formation of O2- and H2O2 from the oxidation of flavoproteins and/or the diffusion of these reactive species from the extracellular milieu. The subsequent oxidation of Fe-S proteins by ROS (e.g. O2-) leads to an increase in intracellular ‘free’ Fe2+ which triggers the reduction of H2O2 to OH (Fenton reaction). The highly reactive nature of OH limits its diffusion so that it generally reacts with molecules in close proximity to its origin (e.g. the Fe2+ associated with DNA). In DNA, OH oxidizes sugar and base moieties, producing radicals which ultimately generate lesions, including base modifications, strand breaks and chemical cross-links to other molecules. Base modifications lead to mismatching and increased mutation frequencies while more severe damage, such as strand breaks, prevents DNA replication, contributing to OH toxicity and cell death. Critical to this chemical process in vivo is the presence of iron. No other metal or non-metal electron carrier appears to be able to univalently reduce H2O2 to OH in vivo (Macomber et al., 2007).
These observations are critical in beginning to understand the oxidative damage/targets in <span class="yellow">B. burgdorferi</span>. It has been shown in <span class="yellow">E. coli</span> that the free-iron pool size determines the rate of oxidative DNA damage. For example, in wild-type <span class="yellow">E. coli</span>, free iron levels are estimated to be 10 µM, yet H2O2 is only mildly genotoxic (Keyer and Imlay, 1996). However, in <span class="yellow">E. coli</span> Fur- mutants, intracellular iron concentration increases eightfold while survival is 10-fold lower when cells are exposed to H2O2. Because <span class="yellow">B. burgdorferi</span> cells do not contain detectable levels of intracellular Fe, it seems unlikely that DNA is a major target for damage via the Fenton reaction in this bacterium (Posey and Gherardini, 2000). The experimental data present in this report suggested that this was the case. When <span class="yellow">B. burgdorferi</span> cells were exposed to high concentrations of ROS (e.g. H2O2), there was no effect on the spontaneous mutation rate (Fig. 2A) or the number of DNA lesions (AP sites or 8-oxoguanine) (Fig. 2B and C). It should be noted that in these experiments, a wild-type strain of <span class="yellow">B. burgdorferi</span> was used and presumably all of the endogenous oxidative stress enzymes were expressed. Therefore, it is possible that no DNA damage was observed because these enzymes are capable of detoxifying the cell and protecting nucleic acids from oxidation via the Fenton reaction. However, we do not believe this to be the case as the concentration of oxidants used in these experiments were significantly higher than concentrations known to cause DNA damage in <span class="yellow">E. coli</span> and other pathogenic bacteria.
The lack of detectable DNA damage in <span class="yellow">B. burgdorferi</span> cells under the conditions tested could be the result of very efficient DNA repair systems. Bacteria, such as <span class="yellow">E. coli</span>, harbour genes encoding repair enzymes (i.e. MutM, MutY, Ung, AlkA, MutS, MutL, ExoIII, EndoVIII, PolI, RecJ). Key enzymes in the repair of 8-oxoguanine lesions resulting from the oxidation of DNA are the bifunctional glycosylases, such as MutM, MutY or EndoVIII. The first activity of these enzymes is to remove oxidized or ring-saturated bases while the second activity is to remove the resulting deoxyribose residue, generating a 3′-phosphate end (Krwawicz et al., 2007). This 3′-P is converted to a 3′-OH by various enzymes/pathways and the lesion is repaired by enzymes (e.g. ExoIII, PolI, LigI etc.) in the short or long BER pathways (S-BER, L-BER). Interestingly, the <span class="yellow">B. burgdorferi</span> genome does not contain genes encoding homologues of MutM, MutY or EndoVIII, suggesting that it would be difficult for the cells to efficiently repair 8-oxoguanine sites using this pathway. <span class="yellow">B. burgdorferi</span> does harbour the genes encoding the enzymes Ung (monofunctional gylcosylase, BB0053), MutS (BB0797, BB0098), MutL (BB0211), ExoIII (BB0114), PolI (BB0548), LigI (BB0552) and RecJ (BB0254) (Fraser et al., 1997) which are involved in excision and repair, via S-BER or L-BER, of deaminated, alkylated, methylated or mismatched bases. Clearly, these repair systems in <span class="yellow">B. burgdorferi</span> do not seem as robust at defending against oxidation of DNA as those described in <span class="yellow">E. coli</span>. The lack of key repair enzymes in this system may indicate that <span class="yellow">B. burgdorferi</span> DNA is not subjected to the same challenge from ROS as is <span class="yellow">E. coli</span>.
In most bacteria, ROS-mediated damage to lipids (lipid peroxidation) is very unlikely because of the lack of polyunsaturated fatty acids (e.g. linoleic acid) (Imlay, 2003). When it does occur, lipid peroxidation is initiated by the attack of free radicals on polyunsaturated fatty acids which decreases the membrane fluidity and, if these reactions propagate, lipid peroxides and their degradation products (e.g. aldehydes) in turn could damage proteins (Gutteridge and Halliwell, 1990). This would dramatically affect the function of transmembrane proteins and membrane-bound lipoproteins involved in the maintenance of membrane potential and solute transport, decreasing cell survivability. In contrast to most bacteria, <span class="yellow">B. burgdorferi</span> membranes contained significant levels of unsaturated fatty acids, such as linoleic acid and linolenic acid, which were derived from the growth media (Table 1). Thus, it seemed possible that lipids and/or proteins rather than DNA are the primary targets of ROS in <span class="yellow">B. burgdorferi</span>. When Borrelia cells are treated with oxidants, the levels of linoleic acid decreased while other fatty acids remain unaffected (Table 2). In addition, HPLC-based assays demonstrated that, as linoleic acid concentrations decrease, MDA (a toxic lipid peroxide intermediate) increased (Fig. 3). When these cells were examined by electron microscopy, damage to the membranes (membrane ‘blebs’) was observed (Fig. 6). These data indicated that unlike most other bacteria, Borrelia membranes were damaged by oxygen radicals.
As <span class="yellow">B. burgdorferi</span> may be exposed to potentially harmful oxygen species at different stages of the infective cycle, the ability to protect its membrane lipids from ROS should be required for survival in the different host environments. In eukaryotic cells, where lipid peroxidation is a major consequence of oxidative attack, proteins which protect membranes have been well studied. For example, phospholipid hydroperoxide glutathione peroxidase, a member of the glutathione peroxidase family, has been identified in a variety of higher organisms. These enzymes are capable of reducing an assortment of hydroperoxy lipids, including oxidized phospholipids and cholesterol esters (Ursini and Bindoli, 1987), and protect complex membranes from oxidative damage. Much less is known about the enzyme(s) responsible for protecting unsaturated lipids from ROS in prokaryotes. In <span class="yellow">H. pylori</span>, lipid hydroperoxide levels in ahpC mutants are approximately three times higher than in wild-type cells, suggesting a role for AhpC in reducing organic peroxides (Wang et al., 2006a,b). In addition, purified AhpC has been shown to reduce linoleic acid hydroperoxide in vitro (Baker et al., 2001). Because these antioxidant enzymes promote the in vivo survival of cells when challenged with ROS, enzymes for the reduction of lipid peroxides in Borrelia need to be identified.
Interestingly, Borrelia cells grown aerobically showed signs of membrane damage similar to those observed in cells exposed to various oxidants. The amount of MDA in untreated aerobically grown cells was equivalent to that observed in treated cells (Fig. 3). Also, EM indicated that cells grown microaerobically or aerobically had significantly more membrane damage than cells grown anaerobically (Fig. 6). This indicated that Borrelia membranes can be damaged simply by exposure to physiologically relevant concentrations of dissolved oxygen. In exponentially growing <span class="yellow">E. coli</span>, Imlay and Fridovich (1991) have shown that both O2- and H2O2 are generated by the autoxidation of components of the respiratory chain during oxygen metabolism. In contrast, <span class="yellow">B. burgdorferi</span> is very metabolically limited and has no enzymes for the TCA cycle or respiration. Therefore, it seems more likely that sources of ROS are exogenous (e.g. innate immune response in the mammalian host) rather than endogenous. In addition, the current practice of growing <span class="yellow">B. burgdorferi</span> under atmospheric oxygen could itself be unintentionally compromising cell integrity during in vitro manipulations.
Analyses of the <span class="yellow">B. burgdorferi</span> genome indicates that only a few genes encoding putative oxidative stress/intracellular redox proteins (SodA, NapA, BosR, CoADR, Trx and TrxR) (Fraser et al., 1997) are present, compared with other bacterial pathogens, including other pathogenic spirochetes (e.g. Treponema, Leptospira). Despite this apparent ‘deficiency’ in the number of ROS-protective enzymes, <span class="yellow">B. burgdorferi</span> cells appear to be able to cope with physiologically relevant levels of ROS. There are several factors that would contribute to this phenomenon: (i) as <span class="yellow">B. burgdorferi</span> does not harbour the genes encoding respiratory enzymes nor metabolize oxygen, it seems unlikely that significant levels of ROS are generated via the incomplete reduction of O2 during cellular metabolism. This would suggest that potential ROS challenges to <span class="yellow">B. burgdorferi</span> would come almost completely from extracellular sources with little contribution from intracellularly generated ROS; (ii) owing to a lack of understanding of the physiological conditions in vector and host tissues infected with <span class="yellow">B. burgdorferi</span>, it is difficult to assess the levels and/or sites of the potential ROS challenge during the infective cycle and (iii) most importantly, analyses of the potential targets for ROS in <span class="yellow">B. burgdorferi</span> strongly suggested that the major targets of oxidative damage are different and perhaps less extensive in this bacterium than in other bacterial pathogens (e.g. <span class="yellow">E. coli</span>). Taken together, this suggests that <span class="yellow">B. burgdorferi</span> would be innately more resistant to ROS and require a less extensive repertoire of enzymes to protect the cells from oxidative damage.<br><br>Experimental procedures
Strains, growth conditions and reagents
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 and strain B314 (Sadziene et al., 1993) were grown in modified BSK-II (Barbour, 1984) medium at 34°C under an atmosphere of 0–20% O2 with 5% CO2 and the balance N2. Cells density was determined using a dark-field microscope. All reagents were purchased from Sigma Chemicals, St. Louis, MO unless stated otherwise. <span class="yellow">E. coli</span> strain Top10, strain TA4315 (ahpCF) (Storz et al., 1989) and CM1319 (mutM) (Bridges et al., 1996) were grown in Luria–Bertani (LB) at 37°C with shaking.<br><br>Per cent survivability assays
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 was grown to a cell density of 5 × 107 cells ml−1 in BSK II under microaerobic (3% O2) conditions, the culture split and the cells treated with varying concentrations of oxidants (0–50 mM t-butyl hydroperoxide or H2O2) at 34°C for 4 h. After the incubation, cells were diluted in fresh BSK II, plated on BSK plates and incubated 7–14 days at 34°C. Per cent survivability was calculated as the number of colonies on the treated plates versus the number of colonies on the untreated plates.<br><br>Determination of the spontaneous mutation rate
To determine spontaneous resistance to coumermycin A1, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under microaerobic conditions to a cell density of 5 × 107 cells ml−1 and treated with 5 mM H2O2 for 1 h at 34°C. The cells were plated on BSK plates containing 0 or 250 ng ml−1 coumermycin A1 and incubated 7–14 days at 34°C. The resistance frequency was calculated as the number of colonies that are CouR per total number of cells plated.<br><br>Measurement of DNA base lesions
<span class="yellow">Borrelia burgdorferi</span> B31A3 cells were grown in BSK-II under microaerobic conditions to a cell density of 5 × 107 cells ml−1, treated with various oxidants (1 or 10 mM t-butyl hydroperoxide, 1 or 10 mM H2O2, 10 mM paraquat or lipoxidase) for 4 h and total DNA was isolated using Wizard Genomic DNA Purification Kit (Promega Corp., Madison, WI). The number of base lesions was determined using the DNA Damage Quantification Colorimetric Assay kit (Oxford Biomedical Research, Oxford, MI) following the manufacturer's protocol. Briefly, 500 ng of DNA was mixed with an equal volume of 10 mM biotinylated aldehyde reactive probe (ARP) reagent and incubated for 1 h at 37°C. The DNA-ARP product was ethanol-precipitated using glycogen as a carrier, washed three times with 70% ethanol and resuspended in Tris-EDTA to give a final concentration of 0.5 µg ml−1. The DNA-ARP product was allowed to bind to the wells of 96-well microplate overnight at 37°C. After the binding, the wells were washed four times with TPBS (137 mM NaCl, 2.7 mM KCl, 10 mM Na3HPO4, 2 mM KH2PO4, 0.5% Tween 20, pH 7.4). The HRP-streptavidin conjugate was diluted to 0.5 µg ml−1 in Assay Buffer (0.15 M NaCl, 10 mM NasHPO4, 1.5 mM KH2PO4, 2.5 mM KCl, 5 mg ml−1 BSA, 0.1% Tween, pH 7.5), 100 µl was added to each well and the plate incubated at 100 r.p.m. for 1 h at room temperature (RT). After incubation, the wells were washed four times with TPBS, 100 µl of substrate was added to each well and incubated for 1 h at 37°C. The reaction was then quenched with 100 µl of 1 M sulphuric acid and the reaction was monitored at 450 nm. The number of aldehyde reactive probe (DNA base lesions) per 105 bp DNA was determined using a standard curve. As a control, <span class="yellow">E. coli</span> TA4315 cells were grown in minimal media to OD600 of 0.4, treated with 0 or 100 µM H2O2 for 30 min and DNA isolated. The number of base lesions was determined as described above. <span class="yellow">B. burgdorferi</span> strain B314 cells were grown under microaerobic conditions and DNA isolated. The number of DNA lesions was determined as described.
To determine the amount of 8-oxoguanine in <span class="yellow">B. burgdorferi</span> DNA, cells were grown and treated as described above and DNA isolated. The DNA was converted to single-stranded DNA by boiling the sample for 5 min followed by rapid chill on ice. The DNA was then digested with nuclease P1 for 2 h and then treated with alkaline phosphatase for 1 h following manufacturer's protocols. The resultant mixture was then centrifuged for 5 min at 6000 g and the supernatant used for the 8-oxoguanine ELISA assay (Oxford Biomedical Research, Oxford, MI). As a control, <span class="yellow">E. coli</span> CM1319 (mutM) cells were grown in LB to OD600 of 0.4, treated with 0 or 100 µM H2O2 for 1 h and DNA isolated. The concentration of 8-oxoguanine was determined as above.<br><br>Lipid analyses
To determine fatty acid content in <span class="yellow">B. burgdorferi</span> total membranes, <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under anaerobic conditions to a cell density of 5 × 107 cells ml−1, harvested by centrifugation (5000 g, 15 min, 4°C) and washed two times with HN (20 mM NaCl, 50 mM Hepes, pH 7.6) buffer. Cell pellets were analysed for fatty acid composition by fatty acid methyl ester (FAME) gas chromatography (Lipid Technologies, Austin, MN) and results are reported as percentage of total fatty acid content. To determine the effects of oxidants on fatty acid composition, a 1.5 l culture of <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under microaerobic conditions to a cell density of 5 × 107 cells ml−1. The culture was split into 500 ml aliquots, the first was treated with 1 mM t-butyl hydroperoxide, the second with 0.25 mg of lipoxidase (17 700 units) and the third was untreated. All cultures were incubated for 12 h at 34°C. Cells were harvested by centrifugation (5000 g, 4°C, 15 min) and washed three times with HN buffer. The fatty acids present in the cell pellets were analysed by FAME gas chromatography (Industrial Laboratory, Wheat Ridge, CO). Fatty acids were reported as percentage of total cell mass.<br><br>Measurement of MDA
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 was grown aerobically, microaerobically and anaerobically as described above to a cell density of 5 × 107 cells ml−1. The microaerobic culture was split and treated with either 5 mM AAPH [2,2′-azobis(2-methylpropionamidine) dihydrochloride] or 250 mg of lipoxidase for 4 h at 34°C. To measure the amount of MDA, the cells were derivatized with thiobarbituric acid and analysed by HPLC as described by Seljeskog et al. (2006). After the incubation, all cells were harvested by centrifugation (1000 g, 5 min, 4°C) and washed three times with HN buffer. Each sample was resuspended in 50 µl of HN, mixed with 150 µl of 0.1 N perchloric acid, 150 µl of 40 mM thiobarbituric acid and 35 µl of 20% SDS, vortexed and heated at 97°C for 60 min. After cooling at −20°C for 20 min, 300 µl of methanol and 100 µl of 20% trichloroacetic acid was added and the samples were mixed vigorously and centrifuged (13 000 g, 6 min). The samples (10 µl) were then analysed with an Agilent Technologies 1200 series HPLC system using a C18 4.6 × 150 mm column with mobile phase 72:17:11 (50 mM KPO4, pH 6.8 : methanol : acetonitrile). Absorbance was monitored at 532 nm. Pure MDA standards (0–10 µM) were prepared in methanol for comparison. As a negative control, <span class="yellow">E. coli</span> Top10 cells were grown to OD600 of 0.4, treated with 0 or 5 mM AAPH for 30 min and MDA measured as above. As a positive control, <span class="yellow">mouse</span> myeloma SP2/O cells were cultured with HYQ-CCM1 (HyClone) medium at 37°C in a humidified 5% CO2 atmosphere, treated with 1 mM AAPH at 37°C for 4 h (Chotimarkorn et al., 2005) and MDA measured as above.<br><br>Identification of lipid damage using Diphenyl-1-pyrenylphosphine fluorescent stain
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 cells were grown microaerobically as described above until a cell density of 5 × 107 cells ml−1 was obtained. The culture was divided into two equal aliquots and the cells treated with 5 mM AAPH for 4 h at 34°C. After the incubation, all cells were harvested by centrifugation (1000 g, 5 min, 4°C) and washed three times with HN buffer. To visualize Borrelia cells, the cells were stained with PKH26 Red Fluorescent Cell Linker Dye (Sigma Aldrich) following the manufacturer's protocol. To visualize the lipid hydroperoxides in the cell membrane, the cells were counterstained with DPPP (Cayman Chemicals, MI) (Okimoto et al., 2000). Briefly, after cells were stained with the Red Fluorescent dye, 2 ml of <span class="yellow">rabbit</span> serum was added to stop the reaction and the mixture incubated for 1 min at RT. Next, 4 ml of HN buffer was added, the cells harvested by centrifugation (1000 g, 10 min, RT) and washed three times with HN buffer. The cells were then resuspended in 1 ml of HN buffer, incubated at 34°C for 5 min and 30 µl of 2.5 mM DPPP was added. The incubation was then continued at 34°C for 5 min in the dark. After incubation, the mixture was centrifuged (1000 g, 10 min, RT) and the cells washed three times with HN buffer. The cells were then resuspended in 100 µl HN buffer and observed by fluorescence microscopy with excitation wavelengths 551 nm (Red Fluorescent) and 351 nm (DPPP), and emission wavelengths 567 nm (Red Fluorescent) and 380 nm (DPPP). As a positive control, <span class="yellow">mouse</span> myeloma SP2/O cells were cultured with HYQ-CCM1 (HyClone) medium at 37°C in a humidified 5% CO2 atmosphere, treated with 1 mM AAPH at 37°C for 4 h (Chotimarkorn et al., 2005) and stained as described above. As a negative control, <span class="yellow">E. coli</span> Top10 cells were grown to OD600 of 0.4, treated with 0 or 5 mM AAPH for 30 min and stained as described above.<br><br>Electron microscopy
<span class="yellow">Borrelia burgdorferi</span> cells were grown under microaerobic or anaerobic conditions to a cell density of 5 × 107 cells ml−1. The microaerobic cultures were split and treated with lipoxidase for 4 h at 34°C. Cells were harvested by centrifugation (5000 g, 10 min, 4°C), washed and resuspended in HBSS (Lonza Group Ltd, Switzerland). The cells were fixed with Karnovsky's phosphate for 5 min at RT adsorbed to Formvar/carbon-coated grids (Ted Pella, Redding, CA) for 5 min and washed three times in H2O. The grids were stained with 1% ammonium molybdate and allowed to air-dry. Samples were examined using a Hitachi H7500 electron microscope (Hitachi High Technologies America, Pleasanton, CA).<br><br><br><br><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="yellow">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="yellow">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="yellow">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="yellow">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="yellow">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="yellow">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="yellow">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="yellow">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="yellow">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="yellow">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="yellow">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="yellow">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="yellow">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="yellow">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="blue">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="yellow">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="yellow">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="yellow">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="yellow">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="yellow">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
<h3>pmcA2365090</h3>Carbonic Anhydrase Inhibitors. Part 541: Metal Complexes of Heterocyclic Sulfonamides: A New Class of Antiglaucoma Agents
Abstract
Metal complexes of heterocyclic sulfonamides possessing carbonic anhydrase (CA) inhibitory properties were recently shown to be useful as intraocular pressure (IOP) lowering agents in experimental animals, and might be developed as a novel class of antiglaucoma drugs. Here we report the synthesis of a heterocyclic sulfonamide CA inhibitor and of the metal complexes containing main group metal ions, such as Be(II), Mg(II), Al(III), Zn(II), Cd(II) and Hg(II) and the new sulfonamide as well as 5-amino-1,3,4-thiadiazole-2-sulfonamide as ligands. The new complexes were characterized by standard physico-chemical procedures, and assayed as inhibitors of three CA isozymes, CA I, II and IV. Some of them (but not the parent sulfonamides) strongly lowered IOP in <span class="yellow">rabbits</span> when administered as a 2% solution into the eye.
CARBONIC ANHYDRASE INHIBITORS. Part 54 METAL COMPLEXES OF HETEROCYCLIC SULFONAMIDES: A NEW CLASS OF ANTIGLAUCOMA AGENTS Claudiu T. <br><br> Supuran*, Andrea Scozzafava <br><br> and Andrei Jitianu 2 <br><br> Universit& degli Studi, Dipartimento di Chimica, Laboratorio di Chimica Inorganica e Bioinorganica, Via Gino Capponi 7, 1-50121, Florence, Italy 2 "I.G. Murgulescu" Institute of Physical Chemistry, Academia Romana, Spl. Independentei 202, R-77208 Bucharest, Roumania Abstract: Metal complexes of heterocyclic sulfonamides possessing carbonic anhydrase (CA) inhibitory properties were recently shown to be useful as intraocular pressure (IOP) lowering agents in experimental animals, and might be developed as a novel class of antiglaucoma drugs. Here we report the synthesis of a heterocyclic sulfonamide CA inhibitor and of the metal complexes containing main group metal ions, such as Be(II), Mg(II), AI(III), Zn(II), Cd(II) and Hg(II) and the new sulfonamide as well as 5-amino-l,3,4thiadiazole-2-sulfonamide as ligands. The new complexes were characterized by standard physico-chemical procedures, and assayed as inhibitors of three CA isozymes, CA I, II and IV. Some of them (but not the parent sulfonamides) strongly lowered IOP in <span class="yellow">rabbits</span> when administered as a 2% solution into the eye. Introduction Sulfonamides possessing carbonic anhydrase (CA, EC 4.2.1.1) inhibitory properties [2] such as acetazolamide 1, methazolamide 2, ethoxzolamide 3 and dichlorophenamide 4 have been used for more than 40 years as pressure lowering systemic drugs in the treatment of open-angle glaucoma [3,4]. Their effect is due to inhibition of at least two CA isozymes present within cilliary processes of the eye, ie, CA II and CA IV, which is followed by lowered bicarbonate formation and reduction of aqueous humor secretion [5-7]. Their main drawback is constituted by side effects such as fatigue, augmented diuresis, or paresthesias, due to CA inhibition in other tissues/organs than the target one, ie, the eye [8]. <br><br> N--N <br><br> XN--N EtO S <br><br> NH 2 <br><br> O <br><br> NH O:S----O <br><br> NHEt <br><br> The above-mentioned side effects are absent in the case in which the inhibitor has topical activity, and is applied directly into the eye. This route has been demonstrated only in 1983 by Maren's group [9] and was followed by the development of the first clinical agent of this type, dorzolamide 5 [10,11 ]. Dorzolamide (Trusopt) has been introduced in clinical use in 1995 in USA and Europe and it constituted the beginning of a radically new treatment of glaucoma, devoid of the severe side effects observed with the systemic inhibitors [4-6]. The success of topical antiglaucoma CA inhibitors fostered much research in the synthesis and clinical evaluation of other types of such compounds [12-15]. 307 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides: A New Class ofAntiglaucoma Agents <br><br> On the other hand, metal complexes of heterocyclic sulfonamides of type 1-5 have been recently prepared by two groups 16-20], and it was proved that they possess much stronger CA inhibitory properties than the sulfonamides from which they were prepared 18-22]. Although the mechanism of CA inhibition of the metal complexes is presently unknown, it was hypothesized that their increased inhibitory power might be due to two processes, occurring separately or in concert, ie, (i) dissociation of the complex inhibitor in sulfonamide anions and metal ions (in diluted solution), which in turn both interact thereafter with the enzyme, at different binding sites, and (ii) direct interaction of the undissociated complex with the enzyme, and more specifically with the hydrophilic patch at the entrance of CA II active site [23], this being the isozyme most susceptible to inhibition with this class of compounds [2,22]. Whether initially the first mechanism of action mentioned above was favored by us [22], recent evidences suggested that the undissociated complex might be the inhibitory species, at least for some isozymes [24]. Since metal complexes are much more inhibitory than the parent sulfonamide from which they were prepared, it appeared of interest to test whether this property might be useful for their use in lowering IOP in experimental animals and whence as a possible glaucoma therapy. Recently we proved [25, 26] that some metal complexes of heterocyclic sulfonamides (which themselves do not possess IOP lowering properties) act as very powerful such agents when administered as diluted solutions directly into the eye of experimental animals, and would thus offer the possibility of developing such totally novel drugs. Here we report the synthesis of a heterocyclic sulfonamide possessing strong CA inhibitory properties, ie, 5-(chloroacetamido)-l,3,4-thiadiazole-2-sulfonamide, and of the metal complexes of this sulfonamide and of 5-amino-1,3,4-thiadiazole-2-sulfonamide, containing some main group metal ions. The new compounds have been characterized by standard physico-chemical procedures, and were assayed as inhibitors of three CA isozymes, hCA I, hCA II and bCA IV (h <span class="yellow">human</span>; b <span class="yellow">bovine</span>; these are the isozymes considered to play a critical role in aqueous humour secretion within the eye of higher vertebrates [2-5]). Materials and Methods <br><br> Melting points were recorded with a heating plate microscope and are not corrected. IR spectra were recorded in KBr pellets with a Carl Zeiss IR-80 instrument. 1H-NMR spectra were recorded in DMSO-d6 as solvent, with a Bruker CPX200 instrument. Chemical shifts are reported as values, relative to Me4Si as internal standard. Conductimetric measurements were done at room temperature (1 mM concentration of complex) in DMSO solution with a Fisher conductimeter. Elemental analyses were done by combustion for C, H, N with an automated Carlo Erba analyzer, and gravimetrically for the metal ions, and were 0.4% of the theoretical values. Thermogravimetric measurements were done in air, at a heating rate of 10C/min., with a Perkin Elmer 3600 thermobalance. Sulfonamides used as standards in the enzymatic assay (except for 5), acetazolamide, pyridine, and chloroacetyl chloride used for the preparation of compound 7, solvents as well as inorganic reagents were from Sigma, Merck and Carlo Erba. 5-Amino-l,3,4-thiadiazole-2-sulfonamide 6 was prepared from acetazolamide by literature procedures [27], by desacetylation with concentrated hydrochloric acid, followed by neutralization with sodium bicarbonate of the corresponding hydrochloride (Scheme 1). Dorzolamide hydrochloride 5 was from Merck, Sharp and Dohme or was prepared as described by Ponticello et al 10,11 ]. <span class="yellow">Human</span> CA and CA II cDNAs were expressed in <span class="red">Escherichia coli</span> strain BL21 (DE3) from the plasmids pACA/hCA I and pACAdaCA II described by Forsman et al. [28] (the two plasmids were a gift from Prof. Sven Lindskog, Umea University, Sweden). Cell growth conditions were those described by Lindskog's group [29], and enzymes were purified by affinity chromatography according to the method of Khalifah et al [30]. Enzyme concentrations were determined spectrophotometrically at 280 nm, utilizing a molar absorptivity of 49 mM-l.cm-1 for hCA and 54 mM-l.cm-1 for hCA II, respectively, based on M 28.85 kDa for hCA I, and 29.3 kDa for hCA II, respectively [31,32]. bCA IV was isolated from <span class="yellow">bovine</span> lung microsomes as described by Maren et al, and its concentration has been determined by titration with ethoxzolamide [33]. Synthesis of 5-(chloroacetamido)- l, 3, 4-thiadiazole-2-sulfonamide 7 An amount of 1.80 g (10 mmol) of 5-amino-1,3,4-thiadiazole-2-sulfonamide 6 was suspended in 20 mL of anhydrous acetonitrile and 0.9 mL (0.87g, 11 mmol) of pyridine added. The mixture was magnetically stirred at 4 C for 10 minutes, then 10.5 mmol of monochloroacetyl chloride, dissolved in 3 mL acetonitrile, were added dropwise for 5 min, and stirring was continued for other 2 hours at room temperature. After an additional 30 min of refluxation, followed by cooling, the precipitated crystals were filtered and recrystallized from ethanol. Yield of 62% white crystals, mp 246-248 o lit [34] mp IR (KBr), cm-l: 590, 610, 660, 790, 935, 1090, 1115, 1170, 1350, 1400, 1550, 1650, 1720, 2870, 3280 3370 (broad); UV spectrum, ,max, nm (lg): 255 (3.50); 288 (4.37) H-NMR (DMSO-d6), i, ppm: 2.96 (s, 2H, CH2); 8.20 (s, 2H, SOzNH2); 12.22 (s, 1H, CONH). Analysis, found: C, 18.56; H, 1.88; N, 21.76; S, 24.62 %; C4HsC1N403S2 requires" C, 18.72; H, 1.96; N, 21.83; S, 24.98 %. <br><br> General procedure for the preparation of compounds 8-20 An amount of 6 mmol of sodium salt of sulfonamides 6 or 7 was prepared by reacting the corresponding sulfonamide with the required amount of an alcoholic 1N NaOH solution, in ethanol as solvent. To this 308 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> solution was added the aqueous metal salt (Zn(II), Mg(II), AI(III), Cd(II) chlorides, and Be(II), Pb(II) and Hg(II) nitrate) solution, working in molar ratios RSOzNH- Mn+ of 2:1 for the divalent cations and 3:1 for the trivalent cation, respectively. The aqueous-alcoholic reaction mixture was heated on a steam bath for one hour, adjusting the pH at 7 if necessary, and after being cooled at 0 C the precipitated complexes were filtered and thoroughly washed with alcohol-water 1:1 (v/v) and air dried. Yields were in the range of 85-90 %. The obtained white powders of compounds 8-20 melted with decomposition at temperatures higher than 350 C, and were poorly soluble in water and alcohol, but had good solubilities in DMSO, DMF as well as mixtures of DMSO-water, DMF-water. <br><br> Pharmacology Carbonic anhydrase inhibition Initial rates of 4-nitrophenyl acetate hydrolysis catalysed by different CA isozymes were monitored spectrophotometrically, at 400 rim, with a Cary 3 instrument interfaced with an IBM compatible PC [35]. Solutions of substrate were prepared in anhydrous acetonitrile; the substrate concentrations varied between 2.10-2 and 1.10-6 M, working at 25C. A molar absorption coefficient of 18,400 M-l.cm-1 was used for the 4-nitrophenolate formed by hydrolysis, in the conditions of the experiments (pH 7.40), as reported in the literature [35]. Non-enzymatic hydrolysis rates were always subtracted from the observed rates. Duplicate experiments were done for each inhibitor concentration, and the values reported throughout the paper are the mean of such results. Stock solutions of inhibitor (1 mM) were prepared in distilled-deionized water with 1020% (v/v) DMSO (which is not inhibitory at these concentrations [2]) and dilutions up to 0.01 nM were done thereafter with distilled-deionized water. Inhibitor and enzyme solutions were preincubated together for 10 min at room temperature prior to assay, in order to allow for the formation of the E-I complex. The inhibition constant KI was determined as described by Pocket and Stone [35]. Enzyme concentrations were 3.3 nM for hCA II, l0 nM for hCA and 34 nM for bCA IV (this isozyme has a decreased esterase activity [36] and higher concentrations had to be used for the-measurements). <br><br> Measurement of tonometric lOP Adult male New Zealand albino <span class="yellow">rabbits</span> weighing 2-3 kg were used in the experiments (three animals were used for each inhibitor studied). The experimental procedures conform to the Association for Research in Vision and Ophthalmology Resolution on the use of animals. The <span class="yellow">rabbits</span> were kept in individual cages with food and water provided ad libitum. The animals were maintained on a 12 h: 12 h light/dark cycle in a temperature controlled room, at 22-26 C. Solutions of inhibitors (2 %, by weight) were obtained in DMSOwater (2:3, v/v) due to the low water solubility of some of these derivatives. Control experiments with DMSO (at the same concentration as that used for obtaining the inhibitors solutions showed that it does not possess IOP lowering or increasing effects. IOP was measured using a Digilab 30R pneumatonometer (BioRad, Cambridge, MA, USA) as described by Maren's group [37-39]. The pressure readings were matched with two-point standard pressure measurements at least twice each day using a Digilab Calibration verifier. All IOP measurements were done by the same investigator with the same tonometer. One drop of 0.2 % oxybuprocaine hydrochloride (novesine, Sandoz) diluted 1"1 with saline was instilled in each eye immediately before each set of pressure measurements. IOP was measured three times at each time interval, and the means reported. IOP was measured first immediately before drug administration, then at 30 min after the instillation of the pharmacological agent, and then each 30 minutes for a period of several hours. For all IOP experiments drug was administered to only one eye, leaving the contralateral eye as an untreated control. The ocular hypotensive activity is expressed as the average difference in IOP between the treated and control eye, in this way minimizing the diurnal, seasonal and interindividual variations commonly observed in the <span class="yellow">rabbit</span> [37-39]. All data are expressed as mean SE, using a one-tailed test. Results and Discussion Reaction of 5-amino-l,3,4-thiadiazole-2-sulfonamide 6 [19b] with chloroacteyl chloride in the presence of pyridine afforded 5-(chloroacetamido)-l,3,4-thiadiazole-2-sulfonamide 7, by the procedure already reported by Young et al. [34] (Scheme 1). The sulfonamide 7 has been characterized by elemental analysis and spectroscopic methods which confirmed its structure (only its m.p. has been reported in ref. [34]). The sodium salt of sulfonamides 6 and 7, obtained in situ from the corresponding sulfonamide and sodium hydroxide, were then used for the preparation of coordination compounds, containing the following metal ions: Be(II), Mg(II), Al(III), Zn(II), Cd(II) and Hg(II). Mention should be made that although 5-amino-l,3,4-thiadiazole-2-sulfonamide 6 is the parent compound of important sulfonamide CA inhibitors, such as acetazolamide, benzolamide, methazolamide, etc., its coordination chemistry has been scarcely investigated up to now [22, 40]. The new complexes prepared in this work are shown in Table I. Both compounds containing the sulfonamide-deprotonated species of sulfonamide 7 (LH), as well as complexes in which the anion of 5amino-1,3,4-thiadiazole-2-sulfonamide (tda) act as ligands, have been prepared. In fact in another work [40] it was documented that in some cases, sulfonamides derived from this ring system may undergo hydrolysis to <br><br> 309 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides." A New Class ofAntiglaucoma Agents <br><br> the moiety substituting the 5 position, with the formation of 5-amino-l,3,4-thiadiazole-2-sulfonamide 6, which thereafter coordinates metal ions present in solution. <br><br> N CIH N <br><br> I <br><br> N +NaHCO 2 <br><br> N 2 <br><br> S' <br><br> -NaCl <br><br> I S <br><br> N <br><br> II <br><br> SO2NH <br><br> 1_12 O Py/MeCN <br><br> 6Htda +CIOCCH2CI <br><br> 0 Cl CH 2 Scheme 1 <br><br> HN <br><br> .I <br><br> N <br><br> N <br><br> S 7: LH <br><br> II <br><br> SqNH <br><br> Thus, the X-ray crystal structure of the complex [Zn(tda)2(NH3)].H20 prepared in this way has recently been reported by this group [40]. On the other hand, when the ligand 7 has not been hydrolyzed (during the preparation of the coordination compounds) in the presence of the metal ion to 5-amino-l,3,4-thiadiazole-2sulfonamide and chloroacetate, the metal complexes contining 6 as ligand have been prepared from the last (pure) compound (as sodium salt) and the corresponding metal salt, by the general procedure described in the Experimental part. Table I: Prepared complexes 8-20, containing the conjugate bases of sulfonamides 6 and 7 as ligands and their elemental analysis data. L stands for the sulfonamide deprotonated species of 7, whereas tda for the sulfonamide deprotonated species of 5-amino-1,3,4-thiadiazole-2-sulfonamide 6. <br><br> No. 8 9 10 11 12 13 14 15 16 17 18 19 20 <br><br> Complex <br><br> Yield <br><br> (%) <br><br> %Ma <br><br> Analysis (calculated/found) %H b %C b 13.0/13.1 11.0/10.8 11.3/11.4 10.2/10.1 8.5/8.1 7.9/7.9 12.7/12.8 18.4/18.1 18.1/17.9 16.6/16.2 15.3/14.9 13.4/13.3 12.7/12.5 1.6/1.3 2.7/2.3 1.4/1.1 1.2/1.2 1.0/1.2 1.6/1.3 1.6/1.6 1.5/1.5 1.5/1.3 1.3/1.4 1.2/1.1 1.1/1.2 1.0/1.0 <br><br> %N b 30.4/30.2 25.6/25.5 26.4/26.4 13.7/23.3 20.0/19.8 18.6/18.5 29.7/29.6 21.5/21.3 21.1/20.8 19.4/19.3 17.9/17.8 15.7/15.6 14.8/14.6 <br><br> [Be(tda)2] [Mg(tda)2].3 H20 [Zn(tda)2] [Cd(tda)2] [Hg(tda)2] [Pb(tda)z(OH2)2] [Al(tda)3] [BeL2] [ALL3] [ZnL2] [CdL2] [HgL2] [PbLz(OH2)2] <br><br> 78 76 83 90 95 84 72 75 59 87 88 92 95 <br><br> 2.4/2.5 5.5/5.1 15.4/15.0 23.8/24.1 35.8/35.7 34.4/34.7 4.7/4.4 1.7/1.6 3.4/3.5 11.3/11.5 18.0/18.1 28.1/28.3 27.4/27.2 <br><br> aBy gravimetry; bBy combustion. The new complexes have also been characterized by spectroscopic, conductimetric and thermogravimetric measurements (Table II). By comparing the IR spectra of the complexes and the corresponding ligands, the following observations should be made: (i) the shift of the two sulfonamido vibrations (both the symmetric as well as the the assymetric one), towards lower wavenumbers in the spectra of the complexes, as compared to the spectra of the corresponding ligand (Table II), as already documented previously for similar complexes [13-22]. This is a direct indication that the deprotonated sulfonamido 310 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> moieties of the ligands interacts with the metal ions in the newly prepared coordination compounds; (ii) the amide vibrations (the most intense such bands at 1670-1680 cm-1) of ligand 7 appear unchanged in the IR spectra of complexes 15-20 (data not shown), suggesting that these moieties do not participate in coordination of the metal ions; (iii) the C-N stretching vibration in the spectra of the prepared complexes is shifted with 5-20 cm-1 towards lower wavenumbers, as compared to the same vibration in the spectra of sulfonamides 6 and 7, indicating that one of the endocyclic nitrogens of the thiadiazolic ring (presumably N3) acts as donor atom, as already documented by X-ray crystallographic and spectroscopic determinations on complexes of other sulfonamides (such as 1-3) with divalent metal ions [13-22] (Table II); (iv) changes in the region 3100-3160 cm-, as the bands present in the spectra of sulfonamides 6, 7 are present in the spectra of complexes 8-20 too, but they are not well resolved, and have a smaller intensity. This is probably due to deprotonation of the SO2NH2 moiety and participation in the binding of cations; (v) the amino vibrations from 3320 cm- in the spectra of 6 appear unchanged in the spectra of its complexes 8-14 (data not s.hown). In the 1H-NMR spectra of compound 6 and its metal complexes, the signal of the amino group has been evidenced as a broad singlet centered at 4.54 ppm (Table II), which is not exchangeable by addition of D20 into the NMR tube, in contrast to the sulfonamido NH2 protons (which readily exchange). This proves that the 5-amino moiety is not involved in binding the metal ions, as already shown in the X-ray crystallographic work of the complex [Zn(tda)z(NH3)].H20 previously reported [40]. For sulfonamide 7 the CONH proton resonates as a singlet at 12.22 ppm. In complexes 15-20 only very minor shifts of this signal were evidenced (Table II), proving basically that the CONH moiety does not interact with the metal ions in these complexes. <br><br> Table II: Spectroscopic, thermogravimetric and conductimetric data for compounds 6-20. <br><br> 1H-NMR Spectrab Comp. IR Spectraa ,cm-1 as v(C=N) CONH, i (ppm) v(SO2)S; v(SO2) 6 8 9 10 11 <br><br> TG analysisc calc./found <br><br> Conductimetryd AM (-1 x cm2x mol-) 2 7 4 5 3 2 2 6 3 4 2 9 3 2 8 <br><br> 12 13 14 7 15 16 17 18 19 20 <br><br> 1170; 1150; 1150; 1145; 1145; 1140; 1145; 1150; 1170; 1130; 1140; 1140; 1150; 1140; 1145; <br><br> 1350 1300 1305 1300 1305 1300 1300 1300 1350 1335 1330 1330 1330 1335 1330 <br><br> 1610 1600 1600 1600 1600 1590 1605 1605 1610 1605 1610 1610 1605 1600 1600 <br><br> A A A A A A A <br><br> e e <br><br> 12.3/12.1f e e e 5.9/5.7g <br><br> A 12.22 (1H) 12.19 (2H) 12.18 (3H) 12.20 (2H) 12.18 (2H) 12.19 (2H) 12.21 (2H) <br><br> e e e e e e e 4.7/4.8g <br><br> a In KBr; bin DMSO-d6; A the signal of the 5-amino group of 6 (appearing in the ligand at 4.54 ppm as a broad singlet) appears at the same chemical shift (4.50 4.55 ppm) in complexes 8-14; cWeight loss between 70-250 C; d mM solution, in DMF, at 25C; e No weight loss seen under 250 C; fCorresponding to three lattice water molecules lost at 70-110C, and gCorresponding to two coordinated water molecules, lost at 160-180 C. <br><br> Thermogravimetric analysis showed the presence of uncoordinated water molecules in the molecule of complex 9 (the three waters were lost in a single step, between 70-110 C) and of coordinated water in the molecules of the lead(II) derivatives 13 and 20. All these compounds behaved as non-electrolytes in DMF as solvent (Table II). Mention should be made that the Mg(II) complex of sulfonamide 7 could not be isolated. Instead, only the correponding complex of 5-amino-l,3,4-thiadiazole has been obtained from reaction mixtures containing magnesium salts and the sodium salt of 7, probably due to a metal ion assisted hydrolysis of 7 to 6 and chloroacetate. Generally such hydrolytic processes involve highly acidic conditions and prolonged heating of the 5-alkylamido-1,3,4-thiadiazole-2-sulfonamide derivatives [41 ], but they might become milder by taking into account the putative catalytic effect of Mg2+ ions reported here. The data shown above lead to the conclusion that ligand 7 shares a common coordination chemistry with acetazolamide 1 with which it is structurally related, whereas 6 probably also behaves similarly to acetazolamide in the sense that the 5-amino group seems not to be involved in coordinating metal ions, at 311 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides: A New Class ofAntiglaucoma Agents <br><br> least in the complexes reported by us here (and also in the compound characterized by X-ray crystallography mentioned above [40]). Thus, in all complexes reported here these sulfonamides (as monodeprotonated species at the SO2NH 2 moieties) act as bidentate ligands, through the endocyclic N-3 and the NH- groups. The proposed formulae of the new complxes are shown below. Except for the two Pb(II) complexes 13 and 20, as well as the AI(III) derivatives 14 and 16, which presumably are pseudo-octahedral, the other derivatives are supposed to contain tetrahedral M(II) ions. <br><br> R <br><br> R 13"R = 20" R= <br><br> H2N <br><br> CICH2CONH <br><br> 14: R = 16: R = <br><br> H2N CICH2CONH <br><br> The compounds 6-20 together with the standard CA inhibitors 1-5 were assayed for inhibition against three isozymes, hCA I, hCA II and bCA IV (Table III). As seen from the above data, the chloracetamido derivative is more inhibitory than acetazolamide, methazolamide and dichlorophenamide, whereas the unacylated compound 6 is less inhibitory than the above sulfonamides. The metal complexes 820 are much more inhibitory than the sulfonamides from which they derive 6, 7 and than all other simple sulfonamides assayed. They behave similarly to the metal complexes of acetazolamide, methazolamide or dorzolamide previously reported by this group, which were all more inhibitory than the parent sulfonamide from which were prepared [16-22, 40]. Particularly strong inhibition was observed for the Zn(II), Hg(II), Pb(II) and Cd(II) complexes, especially against CA II and CA IV, the isozymes critical for aqueous humor formation. In vivo IOP lowering experiments were done in <span class="yellow">rabbits</span> with some of the new compounds prepared in the present work, such as the sulfonamides 6 and 7, and their Zn(II) complexes, which were among the strong CA II and CA IV inhibitors in the obtained series. Some of the IOP lowering data at half an hour and one hour after the instillation of one drop of 2 % solution of inhibitor within the <span class="yellow">rabbit</span> eye are shown in 312 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> Table IV, with dorzolamide (at the same concentration) as standard. In Fig. the time dependence of IOP lowering with dorzolamide 5 and the two Zn(II) complexes 10 and 17 is presented. Table III. CA inhibition data with the standard inhibitors 1-5, the sulfonamides 6 and 7, and their metal complexes 8-20. <br><br> No 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 a <br><br> Inhibitor <br><br> KI hCA Ia 900 780 25 1200 >50,000 1550 640 1050 350 50 40 12 80 240 120 80 40 40 9 15 <br><br> (nM) bCA IVb 220 240 13 380 43 780 24 540 220 25 19 10 26 110 12 16 9 10 5 10 <br><br> hCA IIa 12 14 8 38 9 230 5 190 110 15 14 7 10 76 5 4 3 3 2 5 <br><br> Acetazolamide Methazolamide Ethoxzolamide <br><br> Dichlorophenamide Dorzolamide <br><br> <span class="yellow">Human</span> (cloned) isozymes; b From <span class="yellow">bovine</span> lung microsomes. <br><br> Table IV: IOP lowering following topical application of CA inhibitors, half an hour and one hour after instillation into the eye of a drop (50 L) of 2 % solution of inhibitor. Inhibitor <br><br> lOP SE 1/2 h <br><br> a <br><br> (mm Hg) lh 4.1 0.15 0 0.09 0 0.09 5.0 0.12 8.1 0.21 Dorzolamide 5 6 7 10 17 a <br><br> 2.2 0.10 0 0.10 0 0.10 2.00.09 8.0 0.14 <br><br> IOP <br><br> IOP control eye" IOP treated eye (N 3). <br><br> As seen from the above data, the sulfonamides 6 and 7 are totally ineffective as lOP lowering agents, similarly to the classical clinically used inhibitors of type 1-5 [2,3]. On the other hand, dorzolamide, the first topical sulfonamide used clinically in the treatment of glaucoma is an effective such agent, with a decrease of lOP of around 4 mm Hg, one hour after administration directly into the eye (Table IV). From the data of this table, it is obvious that the metal complexes of heterocyclic sulfonamides investigated by us behave as much more effective IOP lowering agents than dorzolamide, and their effect is generally longerlasting (Fig. 1). A last remark should be made about the possible mechanism of action of the new class of IOP lowering agents. Obviously, their activity is due to inhibition of CA isozymes present in the cilliary processes within the eye, similarly to other topically active sulfonamides [2-6]. The fact that the sulfonamide per se is inactive via the topical route, whereas the metal complexes result much better than the drug <br><br> 313 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides." A New Class ofAntiglaucoma Agents <br><br> dorzolamide, indicates that the presence of metal ions in the molecules of these CA inhibitors is essential and confers them completely new properties. <br><br> Lowering of IOP (ram Hg) <br><br> "||, <br><br> / <br><br> ", , <br><br> '",., <br><br> ,< <br><br> .7, / <br><br> / <br><br> .6<br><br> -0 <br><br> -10 0 <br><br> 2 <br><br> Time (hours) <br><br> Fig l" Time dependence of IOP lowering with dorzolamide (curve 1); the zinc complex 10 (curve 3) and the zinc complex 17 (curve 3), after topical administration of one drop of 2 % solution of inhibitor in <span class="yellow">rabbit</span>. <br><br> Preliminary results from this laboratory indicate that the metal complexes of topically active sulfonamides show also increased lOP lowering effects with respect to the complexes prepared in the present study [42]. Our hypothesis is that the presence of the metal ion in the molecules of these complex inhibitors induces a dramatic change in their physico-chemical properties as compared to those of the parent sulfonamide. This phenomenon is certainly governed by the strong polarization induced by the metal ions. In this way, it is quite probable that the right balance between the lipo- and hydrosolubility of these compounds is achieved, which has been considered to be the critical factor for not observing topical activity in the classical CA inhibitors, such as acetazolamide, methazolamide and ethoxzolamide, which were either too lipophilic or too hy.drosoluble [2,3]. So, by choosing different metal ions and diverse sulfonamides, much larger possibilities arise to finely tune the pharmacological properties which strongly influence the value of a drug. In conclusion we describe here a novel class of lOP lowering agents, ie, the metal complexes of sulfonamide CA inhibitors. These derivatives appear to be very active and longer lasting than the drug dorzolamide, and might constitute the premises for a new generation of antiglaucoma drugs. <br><br> Acknowledgments. We are grateful to Prof. S. Lindskog (Umea Univ., Sweden) for the gift of the hCA and II plasmids. References Preceding part of this series" Supuran CT, Scozzafava A, Ilies MA, Iorga B, Cristea T, Chiraleu F, Banciu MD (1997) Eur J Med Chem, submitted. 2 Supuran CT (1994) "Carbonic anhydrase inhibitors" In Carbonic Anhydrase and Modulation of Physiologic and Pathologic Processes in the Organism, (Puscas I, Ed) Helicon, Timisoara, pp. 29-111. 3 Maren TH (1991) "The links among biochemistry, physiology and pharmacology in carbonic anhydrase mediated systems". In Carbonic Anhydrase From Biochemistry and Genetics to Physiology and Clinical Medicine, (Botr6 F, Gros G, Storey BT Eds) VCH, Weinheim, pp. 186-207. 4 Bayer A, Ferrari F, Maren TH, Erb C (1996) dFr Ophtalrnol 19, 3:57-362. :5 Maren TH, Conroy CW, Wynns GC, Levy NS (1997) d Ocul Pharrnacol Therapeut 13, 23-30. 6 Sugrue MF (1996) d Ocul Pharmacol Ther 12,363-376. 7 Bartlett JD, Jaanus SD (1989) "Carbonic anhydrase inhibitors". In Clinical Ocular Pharmacology, Second Edition, Butterworths Publishers, Boston, pp. 2:54-263. 8 Alward PD, Wilensky JT (1981) Arch Ophthalmo199, 1973-1976. <br><br> 314 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> 9 Maren TH, Jankowska L, Sanyal G, Edelhauser HF (1983) Exp Eye Res 36, 457-480. 10 a) Ponticello GS, Freedman MB, Habecker CN, Lyle PA, Schwam H, Varga SL, Christy ME, Randall WC, Baldwin JJ(1987) J Med Chem 30, 591-597; b) Greer J, Erickson JW, Baldwin JJ, Varney MD (1994) J Med Chem 37, 1035-1054. 11 Baldwin JJ, Ponticello GS, Anderson GS, Christy ME, Murcko MA, Randall WC, Schwam H, Sugrue MF, Springer JP, Gautheron P, Grove J, Mallorga P, Viader MP, McKeever BM, Navia MA (1989) J Med Chem 32, 2510-2513. 12 Chow K, Lai R, Holmes JM, Wijono M, Wheeler LA, Garst ME (1996) EurJMed Chem 31, 175-186. 13 Supuran CT, Nicolae A, Popescu A (1996) Eur JMed Chem 31, 431-438. 14 Supuran CT, Popescu A, Ilisiu M, Costandache A, Banciu MD (1996) Eur JMed Chem 31, 439-448. 15 Supuran CT, Scozzafava A, Popescu A, Bobes-Tureac R, Banciu A, Creanga A, Bobes-Tureac G, Banciu MD (1997) Eur JMed Chem 32, 445-452. 16 Alzuet G, Casanova J, Ramirez JA, Borras J, Carugo O (1995) J Inorg Bioehem 57, 219-234. 17 Sumalan SL, Casanova J, Alzuet G, Borras J, Castifieiras A, Supuran CT (1996) J Inorg Biochem 62, 3139. 18 a) Supuran CT (1995) Metal Based Drugs 2, 327-330; b) Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 19-26. 19 a) Borras J, Cristea T, Supuran CT (1996), Main Group Met Chem 19, 339-346; b) Jitianu A, Ilies MA, Briganti F, Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 1-7. 20 a) Supuran CT, Scozzafava A (1997) J Enzyme Inhibition 12, 37-51; b) Supuran CT, Briganti F, Scozzafava A (1997) J Enzyme Inhibition 12, 175-190. 21 Mincione G, Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 27-34. 22 Alzuet G, Ferrer S, Borras J, Supuran CT (1994) Roum Chem Quart Rev 2, 283-300. 23 Briganti F, Mangani S, Orioli P, Scozzafava A, Vernaglione G, Supuran CT (1997) Biochemistry 36, 10384-10392. 24 Borras J, Casanova J, Cristea T, Gheorghe A, Scozzafava A, Supuran CT, Tudor V (1996) Metal Based Drugs 3, 143-148. 25 Supuran CT, Mincione F, Scozzafava A, Briganti F, Mincione G, Ilies MA (1997) Eur J Med Chem, in <br><br> press. 26 Supuran CT, Scozzafava A, Saramet I, Banciu MD (1997) J Enzyme Inhibition, in press. 27 Jitianu A, Ilies MA, Scozzafava A, Supuran CT (1997) Main Group Met Chem 20, 151-156. 28 Forsman C, Behravan G, Osterman A, Jonsson BH (1988) Acta Chem Scand B42, 314-318. 29 Behravan G, Jonasson P, Jonsson BH, Lindskog S (1991) Eur JBiochem 198, 589-592. 30 Khalifah RG, Strader DJ, Bryant SH, Gibson SM (1977) Biochemistry 16, 2241-2247. 31 Nyman PO, Lindskog S (1964) Biochim Biophys Acta 85, 141-151. 32 Henderson LE, Henriksson D, Nyman PO (1976)JBiol Chem 251, 5457-5463. 33 Maren TH, Wynns GC, Wistrand PJ (1993) Mol Pharmaco144, 901-906. 34 Young RW, Wood KH, Vaughan JR, Anderson GW (1956) J Am Chem Soc 78, 4649-4654. 35 Pocker Y, Stone JT (1967) Biochemistry 6, 668-678. 36 Baird TT, Waheed A, Okuyama T, Sly WS, Fierke CA (1997) Biochemistry 36, 2669-2678. 37 Maren TH, Bar-Ilan A, Conroy CW, Brechue WF (1990) Exp Eye Res 50, 27-36. 38 Maren TH, Brechue WF, Bar-Ilan A (1992) Exp Eye Res 55, 73-79. 39 Brechue WF, Maren TH (1993) Invest Ophthalmol Vis Sci 34, 2581-2587. 40 Borja P, Alzuet G, Server-Carri6 J, Borras J, Supuran CT (1997) J Biol Inorg Chem (JBIC), in press. 41 Supuran CT, Balaban AT, Gheorghiu MD, Schiketanz A, Dinculescu A, Puscas I (1990) Rev Roum Chim 35, 399-405. 42 Supuran CT, Scozzafava A (1997) unpublished results. <br><br> Received: September 24, 1997 Accepted: October 17, 1997 Received in revised camera-ready format" October 23, 1997 <br><br> 315 <br><br>  <h3>pmcA2398786</h3>Genomic-Bioinformatic Analysis of Transcripts Enriched in the Third-Stage Larva of the Parasitic Nematode <span class="yellow">Ascaris suum</span>
Abstract
Differential transcription in <span class="yellow">Ascaris suum</span> was investigated using a genomic-bioinformatic approach. A cDNA archive enriched for molecules in the infective third-stage larva (L3) of <span class="yellow">A. suum</span> was constructed by suppressive-subtractive hybridization (SSH), and a subset of cDNAs from 3075 clones subjected to microarray analysis using cDNA probes derived from RNA from different developmental stages of <span class="yellow">A. suum</span>. The cDNAs (n = 498) shown by microarray analysis to be enriched in the L3 were sequenced and subjected to bioinformatic analyses using a semi-automated pipeline (ESTExplorer). Using gene ontology (GO), 235 of these molecules were assigned to ‘biological process’ (n = 68), ‘cellular component’ (n = 50), or ‘molecular function’ (n = 117). Of the 91 clusters assembled, 56 molecules (61.5%) had homologues/orthologues in the free-living nematodes <span class="yellow">Caenorhabditis elegans</span> and <span class="yellow">C. briggsae</span> and/or other organisms, whereas 35 (38.5%) had no significant similarity to any sequences available in current gene databases. Transcripts encoding protein kinases, protein phosphatases (and their precursors), and enolases were abundantly represented in the L3 of <span class="yellow">A. suum</span>, as were molecules involved in cellular processes, such as ubiquitination and proteasome function, gene transcription, protein–protein interactions, and function. In silico analyses inferred the <span class="yellow">C. elegans</span> orthologues/homologues (n = 50) to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%), or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway. Small numbers of these 50 molecules were predicted to be secreted (10%), anchored (2%), and/or transmembrane (12%) proteins. Functionally, 17 (34%) of them were predicted to be associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority being embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%). A genetic interaction network was predicted for these 17 <span class="yellow">C. elegans</span> orthologues, revealing highly significant interactions for nine molecules associated with embryonic and larval development (66.9%), information storage and processing (5.1%), cellular processing and signaling (15.2%), metabolism (6.1%), and unknown function (6.7%). The potential roles of these molecules in development are discussed in relation to the known roles of their homologues/orthologues in <span class="yellow">C. elegans</span> and some other nematodes. The results of the present study provide a basis for future functional genomic studies to elucidate molecular aspects governing larval developmental processes in <span class="yellow">A. suum</span> and/or the transition to parasitism.<br><br>Introduction
Parasitic nematodes are of major socio-economic importance in animals. For example, hundreds of millions of <span class="yellow">people</span> are infected with geohelminths (soil-transmitted worms), such as blood-feeding hookworms <span class="yellow">Ancylostoma duodenale</span> and/or <span class="yellow">Necator americanus</span>, <span class="yellow">Trichuris trichiura</span> and Ascaris spp. [1], causing serious adverse effects on <span class="yellow">human</span> health, particularly in <span class="yellow">children</span>. Similarly, parasitic nematodes of livestock, such as <span class="yellow">pigs</span>, also cause substantial economic losses due to subclinical and clinical diseases, with billions of dollars spent annually on the treatment and control of gastro-intestinal nematodes. In addition to the socioeconomic impact that these parasites have, there is potential for the emergence of resistance in them against all of the main classes of (nematocidal) compounds used to treat the diseases they cause [2]–[5]. Therefore, there is a significant need to work toward discovering new compounds to control these parasites. Gaining an improved understanding of the molecular basis of parasite development provides such an avenue.
Compared with the free-living nematode <span class="yellow">Caenorhabditis elegans</span>, there is very little information on fundamental molecular aspects of development in parasitic nematodes [6]–[8]. Since the genome sequence of <span class="yellow">C. elegans</span> was published in 1998 [9], many aspects of the molecular biology of this nematode have been elucidated. For instance, microarray analyses have been used to examine developmental and gender-enriched gene expression [10],[11], and the functions of more than 96% of the <span class="yellow">C. elegans</span> genes have been assessed by double-stranded RNA interference (RNAi, or gene silencing; [12]) [13]–[18]. Comparative analyses of genetic data sets have shown that parasitic nematodes usually share ∼50–70% of genes with <span class="yellow">C. elegans</span> (e.g., [19],[20]). There is similarity in other features (such as basic body plan and moulting) between <span class="yellow">C. elegans</span> and parasitic nematodes, suggesting that some molecular pathways are relatively conserved [8],[21]. Understanding the pathways linked to basic nematode biology and development could have important implications for finding new ways of disrupting these pathways and thus facilitate the identification of new drug targets.
Despite the advances in genomic technologies [7], [22]–[29] and the study of <span class="yellow">C. elegans</span>, there is a paucity of information on the genomics of parasitic nematodes of animals, particularly in relation to development. Also considering the major socioeconomic impact of Ascaris and ascariasis in <span class="yellow">humans</span> and <span class="yellow">pigs</span> [30]–[32], several characteristics, including the large size of the adult worm (providing the opportunity of investigating individual organ systems and tissues), the ability to maintain Ascaris in the <span class="yellow">pig</span>, store eggs and culture larvae in vitro for relatively long periods of time (months to years) [32] as well as the discovery that RNAi achieves “cross-species” gene silencing for a selected number of genes [33],[34] and the imminent genome sequence (http://www.sanger.ac.uk/Projects/Helminths/) all indicate that Ascaris could serve as a powerful model system for investigating reproductive and developmental processes in nematodes.
In the present study, Ascaris from <span class="yellow">pigs</span> was used to study molecules abundantly transcribed in the infective third-stage larva (L3). Following the oral ingestion of Ascaris eggs by the host (<span class="yellow">human</span> or <span class="yellow">pig</span>), L3s are released and then invade/penetrate predominantly the caecal wall [35] to then undergo hepato-pulmonary migration, after which ultimately the adult females and males establish and develop in the small intestine [36],[37]. The molecular mechanisms linked to host invasion and parasite development are largely unknown. Here, we constructed an L3-enriched cDNA library using the method of suppressive-subtractive hybridization (SSH), explored transcription of a representative subset of molecules by microarray analysis and conducted bioinformatic analyses to characterize these molecules, map them to biochemical pathways and predict genetic interactions based on comparisons with <span class="yellow">C. elegans</span> and/or other organisms.<br><br>Materials and Methods
Production of Different Developmental Stages of Ascaris
Experimental <span class="yellow">pigs</span> (8–12 weeks of age) were purchased from and maintained in the Experimental Animal Center of South China Agricultural University. These <span class="yellow">pigs</span> were treated humanely, according to the Animal Ethics procedures and guidelines of the People's Republic of China. Adult <span class="yellow">worms</span> (males and females) of <span class="yellow">A. suum</span> were collected from the small intestines of <span class="yellow">pigs</span> from an abattoir in Shenzhen, China. Infective eggs and infective L3s of <span class="yellow">A. suum</span> were produced according to the methods described previously [38]. In brief, eggs from the uteri of adult females of <span class="yellow">A. suum</span> were collected and incubated at 28°C for 28 days to allow them to develop to infective eggs (containing infective L3s). To obtain pure infective L3s, 7.5% v/v sodium hypochlorite was used to treat the larvated eggs at 37°C overnight and then the eggs were shaken with glass-beads; then, the exsheathed L3s and shells were separated by density gradient centrifugation using lymphocyte separating medium (LSM) [38]. Following the experimental infection of helminth-free <span class="yellow">pigs</span> with infective Ascaris eggs as described previously [39], the L3s from livers and in lungs as well as L4s in intestines were isolated according to an established method [40]. All parasite materials were snap-frozen in liquid nitrogen prior to storage at −70°C.<br><br>Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)
Total RNA was isolated from adult females and males, different larval stages or eggs of <span class="yellow">A. suum</span> using TriPure reagent (Roche) as recommended by the manufacturer. Equal amounts of total RNA from each stage or sex were pooled. The mRNA was isolated using the Oligotex mRNA Kit (Qiagen), following the manufacturer's protocol. SSH was carried out using the PCR-Select cDNA Subtraction kit (Clontech), according to the manufacturer's protocol. In brief, cDNA synthesized from mRNAs from infective L3s was subtracted against cDNA synthesized from the pooled mRNA from all other stages included herein. The SSH library was constructed using infective L3s as the tester and pooled cDNAs from all other stages as the driver. The effectiveness of this subtraction process has already been demonstrated in previous studies [41],[42]. The cDNA obtained following SSH was cloned into the pGEM-T Easy plasmid vector (Promega) and competent <span class="blue">Escherichia coli</span> (JM109) transformed. Positive clones, picked randomly (based on blue/white selection), were grown overnight in Luria Bertani (LB) medium (shaking, 37°C). Individual inserts were PCR-amplified using “nested primers” 1 and 2R from the Subtraction kit (Clontech) and examined by agarose electrophoresis.<br><br>Preparation of Microarray Slides
Clones (n = 3075) from the subtracted library were picked and cultured overnight in LB containing ampicillin (1000 IU/ml) in sealed 96-well blocks. Five µl of culture suspension from each well were transferred into individual wells thermocycling (96-well) plates and the inserts PCR-amplified using primers 1 and 2R. Following a 10 min denaturation step at 94°C, the amplification proceeded for 25 cycles of 10 s at 94°C, 30 s at 68°C and 1.5 min at 72°C, with a final extension for 5 min at 72°C. Products were resolved in agarose gels, ethanol precipitated, re-suspended in 16 µl of “spotting solution” (Shanghai BioStar Genechip, Inc) to a final concentration of ∼500 ng per µl, before being printed on to glass slides (in duplicate) using a robotic arrayer. Sixteen blanks (using spotting solution only) and the same number of negative (irrelevant cDNAs with no relationship to Ascaris) were also printed on to slides and served as negative controls; β-actin of <span class="yellow">A. suum</span> served as a positive control to assess the efficiency of labeling and hybridization. The slides were air-dried for 2 h, and cDNA in the spots were cross-linked at 254 mJ. The printed slides were stored at 4°C.<br><br>Labeling of cDNA Probes with Fluorescent Dyes, and Microarray Analysis
The cDNAs produced from total RNA from <span class="yellow">A. suum</span> eggs, infective L3s, L3s isolated from <span class="yellow">pig</span> liver or lung, fourth-stage larvae (L4s), adult males or females [as described in the section ‘Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)’] were labeled with cyanine dyes. Cy3 or Cy5-dCTP was incorporated into cDNA produced from 30 µg of total RNA by direct labeling in a reverse transcription reaction using an oligo (dT) primer. Labeled cDNA was purified using DyeEx columns (Qiagen).
Microarray slides were incubated with a pre-hybridization solution [5×SSC, 1% <span class="yellow">bovine</span> serum albumin (BSA), 0.1% sodium dodecyl-sulphate (SDS)] for 6 h at 42°C. After pre-hybridization, the microarray slides were incubated with ‘pooled’ Cy3 and Cy5-labeled probes in hybridization solution (5×SSC, 1% BSA, 0.1% SDS), in the dark at 42°C for 18 h, and then washed in solution I (1×SSC, 0.2% SDS) for 10 min, followed by solution II (0.1×SSC, 0.2% SDS) for 10 min at 60°C, according to the protocols provided by Shanghai BioStar Genechip, Inc. A “dye flip” was carried out to control for any bias in hybridization signal between the Cy-labeled cDNA probes (produced for two distinct mRNA populations). The slides were dried and scanned (ScanArray 4000 scanner) using image acquisition software (Shanghai BioStar Genechip Inc.) and a range of laser power and photo-multiplier tube intensities. The mean hybridization signal (derived from four replicates of the same array) were corrected for background, normalized [43], log2-transformed and then subjected to statistical analysis employing the students t-test in a spreadsheet (Excel, Microsoft, USA). The microarray data were analysed for differential cDNA hybridization (>2.0-fold to 114.3-fold) between L3 and each of the other stages (eggs, lung and liver L3s, L4, adult female and adult male).<br><br>Verification of Differential Hybridization by Reverse Transcription-Coupled Polymerase Chain Reaction (RT-PCR) Analysis
For a subset (n = 17) of representative ESTs (rESTs), RT-PCR was used to verify the differential transcription recorded by microarray analysis. Double-stranded cDNA was synthesized from total RNA (separately) from each stage or sex of <span class="yellow">A. suum</span> using reverse transcriptase (Superscript III, Invitrogen). Briefly, 5 µg of total RNA were added to 14 µl of H2O and 1 µl of oligo d(T)n = 12–18 primer (0.5 µg/µl), heated to 70 °C for 10 min and chilled on ice. First- and second-strand cDNAs were synthesized via the addition of 4 µl of first-strand cDNA buffer (250 mM Tris-HCl, pH 8.3, 375 mM KCl and 15 mM MgCl2), 2 µl of 0.1 M dithiothreitol, and 1 µl of 10 mM of each dNTP, followed by an incubation at 25 °C (10 min), 42 °C (50 min) and 70 °C (15 min). One-tenth of each double-stranded cDNA produced was then used as a template in the PCR. The transcripts were amplified from individual cDNAs by PCR using oligonucleotide primers (sequences available upon request) designed to each EST. The PCR amplification of a portion (209 bp) of the β-actin gene (accession no. BI594141) using forward primer (5′-CTCGAAACAAGAATACGATG-3′) and reverse primer (5′- ACATGTGCCGTTGTATGATG-3′), previously determined to be present in all developmental stages and both sexes of <span class="yellow">A. suum</span> [44], served as a positive control. Samples without template (no-DNA controls) were included in each PCR run. The following cycling conditions were employed: one cycle at 94 °C (5 min), 94 °C (30 s), 60 °C (30 s) and 72 °C (30 s) for 30 cycles, followed by a final extension of 70 °C (7 min). Following the PCR, 5 µl of individual amplicons were resolved in ethidium bromide-stained agarose gels (2%) and then photographed upon transillumination. The relative band intensities were analyzed using UVIsoft Image Acquisition and Analysis software (UVITEC). The specificity and identity of individual amplicons were confirmed by direct sequencing using the same primers (separately) as employed for their amplification.<br><br>Sequencing and Bioinformatics Analyses
Clones from the SSH cDNA library with increased hybridization in microarray analysis to the infective L3 compared with other stages were sequenced using standard technology [45]. The nucleotide sequences have been deposited in the GenBank database under accession numbers ES290984-ES291074. Following the processing of the sequences (i.e., removal of vector sequences, quality assurance and clustering), contigs or singletons from individual clusters were subjected to BLASTx (NCBI: www.ncbi.nlm.nih.gov) and BLASTn (EMBL-EBI Parasite Genome Blast Server: www.ebi.ac.uk) analysis to identify putative homologues in <span class="yellow">C. elegans</span>, other nematodes and other organisms (e-value of ≤1e-05). Peptides inferred from ESTs were classified functionally using Interproscan (available at http://www.ebi.ac.uk/InterProScan/) employing the default search parameters. WormBase (www.wormbase.org) was interrogated extensively for relevant information on <span class="yellow">C. elegans</span> homologues/orthologues, including RNAi phenotypic, transcriptomic, proteomic and interactomic data. ESTs with homologues/orthologues in <span class="yellow">C. elegans</span> and other nematodes were also subjected to analysis employing the KEGG Orthology-Based Annotation System (KOBAS) (www.kobas.cbi.pku.edu.cn), which predicts the biochemical pathways in which molecules are involved. The open reading frames (ORFs) inferred from selected ESTs with orthologues in <span class="yellow">C. elegans</span> were also subjected to “secretome analysis” using the program SignalP v.2.0 www.cbs.dtu.dk/services/SignalP/), employing both the neural network and hidden Markov models to predict signal peptides and/or anchors [46]–[48]. Also, transmembrane domains were predicted using the program TMHMM (www.cbs.dtu.dk/services/TMHMM/; [49]–[51]), and subcellular localization inferred employing the program WoLF PSORT (http://wolfpsort.org/; [52]).
The method established by Zhong and Sternberg [53] was used to predict the interactions for <span class="yellow">C. elegans</span> orthologues of the L3-enriched molecules from Ascaris. In brief, interaction, phenotypic, expression and gene ontology data from <span class="yellow">fruitfly</span>, <span class="yellow">yeast</span>, <span class="yellow">mouse</span> and <span class="yellow">human</span> were integrated using a naïve Bayesian model to predict genetic interactions among <span class="yellow">C. elegans</span> genes ([45],[53]; Zhong and Sternberg, unpublished). The predicted networks resulting from the analyses were saved in a graphic display file (gdf) format and examined using the graph exploration system available at http://graphexploration.cond.org/. Images were labeled and saved in the joint photographic experts group (jpeg) format.<br><br>
Results
To identify molecules transcribed abundantly in the L3 of <span class="yellow">A. suum</span>, an enriched cDNA library was constructed by SSH. From a total of 3075 clones from this library, 2921 (95%) were shown to contain an insert (which could be amplified by PCR). From 2671 (92%) of these clones, amplicons representing single bands of ∼400 to 600 bp in size were produced. These latter amplicons were arrayed (in duplicate) on to slides and then hybridized with Cy3-labeled L3-cDNA or with Cy5-labeled cDNA from eggs, liver/lung L3s, L4s, adult female or adult male of Ascaris. Dye flip was conducted to verify the hybridization data. Of the 2671 (duplicate) spots, 1526 had a significant difference in hybridization between infective L3 cDNA and cDNAs from all other stages or sexes of <span class="yellow">A. suum</span>, of which 515 had a >2.0-fold increased hybridization for the L3.
In order to independently verify the hybridization results in the microarray, a PCR-based analysis of a selected subset (n = 17) clones was conducted using specific primer pairs. Having verified the specificity and identity of individual amplicons by sequencing, PCR results were reproducible (based on multiple runs on different days) and ∼94% (16 of 17) concordant with those of the microarray analysis (not shown). There was complete concordance for representative clones associated with a differential signal of ≥3.0-fold in the microarray.
The clones linked to the 515 spots representing increased transcription (>2.0-fold) in infective L3 compared with the other developmental stages or sexes included were subjected to sequencing. The 498 sequences (length: 550±115 bp) determined were then subjected to detailed bioinformatic analyses. There were 91 unique clusters (accession numbers ES290984-ES291074), of which 55 were singletons (sequences determined once). Of 56 molecules (61.5%) with significant similarity to sequences other than <span class="yellow">A. suum</span> in the databases interrogated, 50 (54.9%) had <span class="yellow">C. elegans</span> or <span class="yellow">C. briggsae</span> homologues, and six had similarity to ESTs already sequenced from ascaridoid and/or other parasitic nematodes, and/or other organisms (Table 1). A significant proportion (38.4%) did not have any similarity in sequence to any organisms for which data are presently available. Comparative analysis specifically against <span class="yellow">A. suum</span> EST data sets (n∼42,000) available in public databases confirmed independently that the majority of molecules (>60%) were present exclusively in the infective L3 stage or were orphans.
As gene ontology (GO) provides a hierarchy that unifies the descriptions of biological, cellular and molecular functions [54], this approach was employed to predict the classification and gene function of molecules enriched in infective L3 of <span class="yellow">A. suum</span>. A summary of the GO categories of these molecules is displayed in Fig. 1. Of the 91 contigs, 32 (35%) could be functionally assigned to ‘biological process’ (n = 38), ‘cellular component’ (n = 17) and ‘molecular function’ (n = 64). The most common subcategories were gluconeogenesis (13%) and metabolic process (13%) within ‘biological process’, extracellular region (24%) within ‘cellular component’, and catalytic activity (11%) and phosphoenolpyruvate carboxykinase activity (8%) within ‘molecular function’ (Table S1).
A focused KOBAS analysis inferred the 50 <span class="yellow">C. elegans</span> orthologues/homologues to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%) or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway (Table 2). Of these 50 molecules, small numbers were predicted to be secreted (10%), anchored (2%) and/or transmembrane (12%) proteins (Table 2). Functionally, 17 (34%) of the 50 molecules were associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority displaying embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%) (Table 2).
Extending this analysis, a relatively complex genetic interaction network was predicted for the 17 <span class="yellow">C. elegans</span> orthologues (i.e., with non-wild-type RNAi phenotypes) (see Table S2). Statistically highly significant interactions were predicted for nine of the <span class="yellow">C. elegans</span> genes; the top five interactors are displayed in Fig. 2. The gene ontology categories for eight selected <span class="yellow">C. elegans</span> genes (F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2) included: embryonic development, egg hatching, larval development and/or growth. The other categories included: positive regulation of growth rate (F55A12.8, kin-2, mup-2, pab-1, rpl-22 and T21B10.2) and gamete generation and locomotory behaviour (kin-2, mup-2, pab-1 and F55A12.8, kin-2, mup-2, respectively). The <span class="yellow">C. elegans</span> homologue egl-3 was predicted to be involved in proteolysis (see www.wormbase.org). All nine <span class="yellow">C. elegans</span> orthologues were predicted to interact directly with a total of 296 (range: 5–75) other genes and, in particular, a direct genetic interaction was predicted between pab-1 and T21B10.2 (Fig. 2). The 296 interactors were associated with embryonic and larval development (n = 198; 66.9%), information storage and processing (n = 15; 5.1%), cellular processes and signalling (n = 45; 15.2%) and metabolism (n = 18; 6.1%); the precise function of some of the interactors (n = 20; 6.7%) is presently unknown (Table S2).<br><br>Discussion
The present study investigated transcripts in infective L3s of <span class="yellow">A. suum</span> using a genomic-bioinformatic platform. The focus was on comparisons with <span class="yellow">C. elegans</span> homologues/orthologues, because the entire genome sequence of this nematode is known [9] and because there is a wealth of information on the localization and functionality of its molecules (www.wormbase.org; http://elegans.bcgsc.bc.ca/knockout.shtml). The functions of most genes in <span class="yellow">C. elegans</span> have been assessed using RNAi (e.g., [14],[15],[17],[55],[56]) in the hermaphroditic stage, whereas there is a paucity of functional information available for Ascaris and other parasitic nematodes of animals [57],[58].
Following the microarray analysis of >2500 ESTs from the SSH library, 498 cDNAs inferred to be enriched in the L3, based on hybridization signal, were sequenced and subjected to comprehensive in silico analyses. Of the 91 clusters of molecules categorized, 50 (54.9%) had <span class="yellow">C. elegans</span> homologues/orthologues with loss-of-function phenotypes could be mapped to key pathways. The statistically significant genetic interactions predicted for 9 of the 50 <span class="yellow">C. elegans</span> orthologues [namely egl-3, F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2 ( = enol-1)] and the interaction network included genes encoding kinases, alpha-tubulins, enolases, troponin and other named and unnamed proteins. Eight of these molecules (enol-1, pab-1, F33D11.10, rpl-22, F55A12.8 mec-12, mup-2 and kin-2) have known or predicted roles in embryonic and larval growth and development, gamete generation, locomotory behaviour or other biological processes in <span class="yellow">C. elegans</span> (see www.wormbase.org).
The enolase encoded by enol-1 is predicted to play a role in glycolysis, gluconeogenesis, phenylalanine, tyrosine and tryptophan biosynthesis (cf. [59]). Since glucose is the main source for ATP production, the alteration in these key glycolytic enzymes may lead to cellular dysfunction, such as impaired ion-motive ATPase required to maintain potential gradients, operate pumps and maintain membrane lipid asymmetry [60]. Bioinformatic analysis for transmembrane helices (TMHMM) and peptide signal sequences (SignalP) predicted ENOL-1 to be a non-secreted protein localized to the cytoplasm (cf. Table 2). Nonetheless, enolases are often detected in the excretory/secretory (ES) products of parasitic helminths, including adult <span class="yellow">A. suum</span> [61], and appear to play a role in the triggering of nitric oxide production by host cells. The enol-1 orthologue of <span class="yellow">C. elegans</span> has been predicted to interact specifically with the polyadenylate binding protein gene, pab-1, inferred to be involved in coordinated gene transcription and expression during normal larval development [16]. Poly(A)-binding proteins (PABPs) are recognized to be central to the regulation of mRNA translation and stability [62]. Present evidence suggests that the expression of PAB-1 is regulated by an oligo-pyrimidine tract in response to cell growth and relates to coordinated growth regulation in <span class="yellow">C. elegans</span> [62]. Furthermore, gene silencing of pab-1 and its selected interactors (see Fig. 2) leads to embryonic lethal (Emb), slow growth (Slo) and sterile progeny (Stp) phenotypes (see www.wormbase.org).
Another gene (F33D11.10; EST code 4F10; see Table 2) which encodes an ATP-dependent RNA helicase and is associated with embryonic lethal (Emb) and larval lethal (Lvl) RNAi phenotypes, was shown to be highly transcribed in infective L3s of <span class="yellow">A. suum</span>. Helicases are involved in a variety of RNA metabolic processes, including translation initiation, pre-mRNA splicing, pre-rRNA processing, rRNA maturation and RNA degradation [63], and are crucial for life cycle progression, sex determination and early embryogenesis in <span class="yellow">C. elegans</span> [60]. The high transcription levels of a homologue/orthologue in the L3 of <span class="yellow">A. suum</span> might suggest a similar role in this ascaridoid. Similarly, the coordination of the expression of a large number of genes is required for normal growth and cell proliferation during larval development. The high transcription level for the ribosomal protein gene homologue rpl-22 (large subunit family member; EST code 26G12, see Table 2) in the infective L3 of <span class="yellow">A. suum</span> compared with other developmental stages is likely to reflect the substantial rate of cell growth in this stage [64].
The gene (F55A12.8, EST code 4G11; see Table 2) encoding an acetyl-transferase with a putative ATPase domain, shown to be enriched in the L3 of <span class="yellow">A. suum</span>, was predicted to interact with 75 other genes all involved in energy production and/or RNA processing (see Table S2). Several molecules involved in ATP synthesis and mitochondrial pathways (e.g., cytochrome oxidase c subunits 1, 2 and 3, ADP/ATP translocases, NADH dehydrogenases, ATPases and ATP synthetases) have been reported previously to be highly represented in the L3 stage of <span class="yellow">Anisakis simplex</span> [65], thus supporting the proposal that substantial energy is required for larval development as well as the transition from the free-living to the parasitic stage and the invasion of the host. There is also likely to be a substantial energy requirement for muscle contraction linked to larval motility in <span class="yellow">A. suum</span>, as the L3s penetrate the caecal wall in the porcine host, before undergoing hepato-pulmonary migration [35]. Accordingly, genes encoding a specialized tubulin expressed in mechanoreceptors (mec-12, EST code 13E09) and a troponin (mup-2, EST code 01G03; see Table 2), both predicted to interact with a total number of 32 tubulin- and myosin-encoding genes, also supported a link to extensive muscle contraction and motility in <span class="yellow">A. suum</span> L3s. Also, neuroactive peptides are required to regulate the responsiveness of nematode larvae to mechanical stimuli [66]. A homologue encoded by egl-3 was shown to be highly transcribed in the L3 of <span class="yellow">A. suum</span>; EGL-3 is predicted to be a pro-hormone convertase involved in the maturation of neuropeptides and could be associated with mechano-sensory responses and touch sensitivity linked to the host invasion.
A regulatory subunit of a cAMP-dependent protein kinase (kin-2, EST code 22H01; see Table 2) was predicted to interact with 72 other genes all involved in diverse cellular processes, such as nuclear trafficking, and DNA replication and repair (see Table S2). Based on gene ontology terms, kin-2 is implicated in gamete generation, growth, larval development, post-embryonic body morphogenesis, signal transduction and/or protein amino acid phosphorylation (see Table S2). Gene silencing of kin-2 in <span class="yellow">C. elegans</span> leads to phenotypes, such as larval lethal (Lvl), larval arrest (Lva), body morphology defect (Bmd), dumpy (Dpy), uncoordinated (Unc) and sterile progeny (Stp) (www.wormbase.org), suggesting that its homologue in <span class="yellow">A. suum</span> is central to larval maturation. The KOBAS analysis predicted the protein KIN-2 to be involved in the insulin-signaling pathway, previously implicated in controlling the exit from dauer in <span class="yellow">C. elegans</span> and the activation of L3s of the <span class="yellow">canine hookworm</span>, <span class="yellow">Ancylostoma caninum</span>, following exsheathment [67]. In a recent study, Brand and Hawdon [68] were able to inhibit (with a phosphoinositide-3-OH-kinase inhibitor) the activation of infective L3s of both of the hookworms <span class="yellow">Ancylostoma caninum</span> and <span class="yellow">Ancylostoma ceylanicum</span> via the insulin signaling pathway, thus lending some credence to the hypothesis that this pathway plays an critical role in regulating the transition from the free-living to the parasitic stage [68]. Recently, it has been proposed that transcriptional and feeding responses to serum-stimulation in <span class="yellow">Ancylostoma caninum</span> are regulated by parallel systems, with the insulin signaling pathway playing a significant role in the ‘resumption of feeding’ in activated larvae [69].
Protein kinases are also likely to be involved in pathways linked to sexual maturation in developing larvae. As already proposed for adult stages of <span class="yellow">H. contortus</span> [45], the protein kinase gene cdk-1 is predicted to play a pivotal role in the germline, oogenesis and spermiogenesis pathways of this parasitic nematode. Other protein kinases, such as PEPCK, and phosphatases, were shown herein to be transcribed at high levels in the L3 stage compared with other developmental stages of <span class="yellow">A. suum</span> (see Table 2), which is in accordance to findings reported recently for <span class="yellow">Anisakis simplex</span> [65]. Due to their major regulatory effects in eukaryotic signaling events and regulatory and sensory functions, protein kinases have been considered interesting targets for anti-parasitic drugs [70].
In conclusion, this study has given some interesting insights into early molecular processes in the L3 of <span class="yellow">A. suum</span>. Approximately 60% of the transcripts enriched in the L3 stage of <span class="yellow">A. suum</span> have homologues/orthologues in <span class="yellow">C. elegans</span>. The bioinformatic analyses of selected molecules suggest that a complex genetic network regulates or controls larval growth and development in <span class="yellow">A. suum</span> L3s, and some of these might be involved in or regulate the switch from the free-living to the parasitic stage. Some caution is warranted in drawing conclusions regarding molecular mechanisms regulating the transition to parasitism in parasitic nematodes from information on <span class="yellow">C. elegans</span>, as latter is a free-living nematode. Also, while the method of data integration is essential for the reliable prediction of genetic interactions, it might limit the capacity of the approach somewhat to infer nematode-specific interactions. As additional datasets of genes and gene functions become available for various parasitic nematodes, more informed inferences can be made regarding the functions of nematode-specific genes, particularly those involved in the transition to parasitism. The imminent genome sequence of <span class="yellow">A. suum</span> (http://www.sanger.ac.uk/Projects/Helminths/) should all assist in this endeavour. Also, functional analysis of selected molecules representing selected ESTs identified herein, utilizing gene silencing approaches established recently [33],[34], could provide some insights into developmental processes in Ascaris and related ascaridoid nematodes and provide avenues for the development of novel approaches for their control.<br><br>Supporting Information<br><br>
<h3>pmcA2412862</h3>Minocycline attenuates lipopolysaccharide (LPS)-induced neuroinflammation, sickness behavior, and anhedonia
Abstract
Background
Activation of the peripheral innate immune system stimulates the secretion of CNS cytokines that modulate the behavioral symptoms of sickness. Excessive production of cytokines by microglia, however, may cause long-lasting behavioral and cognitive complications. The purpose of this study was to determine if minocycline, an anti-inflammatory agent and purported microglial inhibitor, attenuates lipopolysaccharide (LPS)-induced neuroinflammation, sickness behavior, and anhedonia.<br><br>Methods
In the first set of experiments the effect of minocycline pretreatment on LPS-induced microglia activation was assessed in BV-2 microglia cell cultures. In the second study, adult (3–6 m) BALB/c <span class="yellow">mice</span> received an intraperitoneal (i.p.) injection of vehicle or minocycline (50 mg/kg) for three consecutive days. On the third day, <span class="yellow">mice</span> were also injected (i.p.) with saline or <span class="blue">Escherichia coli</span> LPS (0.33 mg/kg) and behavior (i.e., sickness and anhedonia) and markers of neuroinflammation (i.e., microglia activation and inflammatory cytokines) were determined. In the final study, adult and aged BALB/c <span class="yellow">mice</span> were treated with the same minocycline and LPS injection regimen and markers of neuroinflammation were determined. All data were analyzed using Statistical Analysis Systems General Linear Model procedures and were subjected to one-, two-, or three-way ANOVA to determine significant main effects and interactions.<br><br>Results
Minocycline blocked LPS-stimulated inflammatory cytokine secretion in the BV-2 microglia-derived cell line and reduced LPS-induced Toll-like-receptor-2 (TLR2) surface expression on brain microglia. Moreover, minocycline facilitated the recovery from sickness behavior (i.e., anorexia, weight loss, and social withdrawal) and prevented anhedonia in adult <span class="yellow">mice</span> challenged with LPS. Furthermore, the minocycline associated recovery from LPS-induced sickness behavior was paralleled by reduced mRNA levels of Interleukin (IL)-1β, IL-6, and indoleamine 2, 3 dioxygenase (IDO) in the cortex and hippocampus. Finally, in aged <span class="yellow">mice</span>, where exaggerated neuroinflammation was elicited by LPS, minocycline pretreatment was still effective in markedly reducing mRNA levels of IL-1β, TLR2 and IDO in the hippocampus.<br><br>Conclusion
These data indicate that minocycline mitigates neuroinflammation in the adult and aged brain and modulates the cytokine-associated changes in motivation and behavior.<br><br><br><br>Background
The bi-directional communication between the immune system and the central nervous system (CNS) is necessary for mounting the appropriate immunological, physiological, and behavioral responses to immune stimulation [1]. CNS innate immune cells including microglia and macrophages play integral roles in receiving and propagating inflammatory signals that are initiated at the periphery. Activation of peripheral innate immune cells elicits the secretion of inflammatory cytokines, including interleukin (IL)-1, IL-6, and tumor necrosis factor-α (TNFα.), that use neural [2,3], humoral [4] and blood brain barrier pathways [5] to relay this signal to the CNS. This inflammatory signal, in turn, induces CNS macrophages and microglia to produce the same cytokines [6], which target neuronal substrates and elicit a sickness behavior syndrome that is normally adaptive and beneficial to the host [1]. An amplified or excessive inflammatory cytokine response in the brain, however, is associated with a myriad of complications including cognitive dysfunction [7-10], prolonged sickness behavior [11-14], and depressive-like behavior [15].
Microglia are primarily involved in immune surveillance [16,17], but when activated have macrophage-like capabilities including phagocytosis, inflammatory cytokine production, and antigen presentation [18]. Normally these neuroinflammatory changes are transient with microglia returning to a resting state as the immune stimulus is resolved. Aging or neurological disease, however, may provide a brain environment where microglia are more "reactive or primed" to a peripheral immune challenge [19]. Recent findings indicate that several markers of glial activation such as major histocompatibility complex (MHC) class II, complement receptors, and scavenger receptors are increased in brain during normal aging [13,20-26]. Furthermore, we and others have reported that a biological consequence of this reactive glial profile is an exaggerated neuroinflammatory response to innate immune challenge [9,10,12-14,27,28].
Active microglia and CNS macrophages also contribute to the production of oxidative and neuroactive mediators that may influence behavior. For instance, inflammatory cytokines in the CNS upregulate the enzyme IDO [29,30], which metabolizes tryptophan (TRP) into L-kynurenine (KYN) [31]. TRP degradation to KYN can reduce TRP levels that are required for serotonin synthesis [32] and can lead to the production of neuroactive mediators including 3-hydroxykynurenine (3HK) and quinolinic acid (QUIN) [31]. High levels of 3HK and QUIN induce neuronal damage through oxidative stress [33] and over stimulation of N-methyl-D-aspartate (NMDA) receptors [34,35]. A recent study indicates that while several cell types in the CNS express IDO, only microglia maintain all the enzymes required to produce 3HK and QUIN [36]. Because IDO mediated TRP degradation impacts both serotonergic and glutamatergic pathways, this may be an important mechanism underlying mood and behavior complications concomitant with inflammation [37-39].
Because activated microglia are suspected to cause or exacerbate several neurodegenerative diseases, pharmacological strategies to suppress microglial activity are being explored as therapies. Minocycline is a tetracycline derived antibiotic that has anti-inflammatory properties in the CNS that are separate from its antimicrobial action [40]. Minocycline readily crosses the blood brain barrier and attenuates inflammation associated with microglial activation. For example, minocycline blocks the deleterious effects of neuroinflammation on neurogenesis, long-term potentiation, and neuronal survival [41-43]. The mechanism of action is unclear, but recent studies indicate that minocycline abrogates MAPkinase and NFκB dependent signaling pathways in primary microglia and microglia cell cultures [44]. Moreover, in the brain of <span class="yellow">rats</span>, minocycline abrogates microglial expression of CD11b and MHC II through a protein kinase-c dependent mechanism [45]. This is relevant because minocycline attenuates neuroinflammation in several <span class="yellow">rodent</span> models of disease including Amyotrophic Lateral Sclerosis [46], Experimental Autoimmune Encephalomyelitis (EAE) [45] and MPTP-induced Parkinson's disease [47]. However, the extent to which minocycline facilitates the recovery from cytokine-mediated sickness behavior is unknown.
The present study investigated the degree to which minocycline–an anti-inflammatory agent and purported microglial inhibitor–reduced LPS-induced neuroinflammation and sickness behavior. We show that minocycline blocked LPS-stimulated inflammatory cytokine secretion in the BV-2 microglia-derived cell line and reduced LPS-induced Toll-like-receptor-2 (TLR2) surface expression on brain microglia. Moreover, our data show that minocycline pretreatment attenuated LPS-induced weight loss, social withdrawal, and anhedonia in adult <span class="yellow">mice</span>. The attenuation of sickness behavior was paralleled with minocycline dependent decrease in markers of neuroinflammation (IL-1β, TLR2, and IDO) in adult and aged <span class="yellow">mice</span>. These findings support our hypothesis that the ability to mitigate cytokine expression in the brain during systemic inflammatory events may be useful in preventing cognitive and behavioral deficits.<br><br>Methods
Animals
Male BALB/c <span class="yellow">mice</span>, adults (3 month old) and juvenile (3–4 week old) were purchased from Harlan (Indianapolis, IN). For age comparisons, male BALB/c <span class="yellow">mice</span> (3–4 and 20–22 month old) were purchased from the National Institute on Aging specific pathogen free colony. Upon arrival, <span class="yellow">mice</span> were individually housed in polypropylene cages and maintained at 21°C under a 12 h light: 12 h dark cycle with ad libitum access to water and <span class="yellow">rodent</span> chow. At the end of each study, <span class="yellow">mice</span> were examined postmortem for gross signs of disease (e.g., splenomeglia or tumors). Data from <span class="yellow">mice</span> determined to be unhealthy were excluded from analysis (< 5%). All procedures were in accordance with the National Institute of Health Guidelines for the Care and Use of Laboratory Animals and were approved by The Ohio State University Institutional Laboratory Animal Care and Use Committee.<br><br>Cell culture
BV-2 microglia cell lines were cultured in growth medium (DMEM supplemented with 10% FBS, sodium bicarbonate 3.7 g/l, 200 mM glutamine, 100 U/ml penicillin G, 100 μg/ml streptomycin, 0.25 μg/ml fungizone) as previously described [12]. Cultures were maintained at 37°C with 95% humidity and 5% CO2 and growth medium was replenished every third day until confluence. Cultures were washed twice and supplemented with warm growth medium containing experimental treatments. Cell viability was measured by the MTS cell proliferation assay according to the manufacturer's instructions (Promega, Madison, WI).<br><br>CNS macrophage/microglia isolation
CNS macrophages and microglia were collected from whole brain homogenates as described previously [48], but with several modifications. <span class="yellow">Mice</span> were euthanized by CO2 asphyxiation and whole brains were collected. Brains were homogenized in Hank's balanced salt solution (HBSS) pH 7.4. Brain homogenates were passed through a 70 μm nylon cell strainer and centrifuged at 400 × g for 10 min. Supernatants were removed and cell pellets were re-suspended in 70% isotonic Percoll (GE-healthcare, Uppsala, Sweden) at room temperature. A discontinuous Percoll density gradient was set up as follows: 70%, 35%, and 0% isotonic Percoll. This suspension was centrifuged for 30 minutes at 400 × g. A mixed population of CNS macrophages and microglia was collected from the interphase between the 70% and 35% Percoll layers. Cells were washed and then re-suspended in sterile HBSS. The number of viable cells was determined using a hemacytometer and 0.2% trypan blue staining.<br><br>Flow cytometry
Flow cytometric analysis of microglial cell surface markers was performed as described previously, but with a few modifications [48]. In brief, Fc receptors on macrophages and microglia were blocked with anti-CD16/CD32 antibody (eBiosciences, CA). Next, cells were incubated with either Panel-1 (anti-CD11b APC, anti-CD45 FITC, and anti-MHC II PE from eBiosciences, CA) or Panel-2 antibodies (anti-CD11b APC, anti-CD45 FITC, and anti-TLR2 PE from eBiosciences, CA). Expression of these surface receptors was determined by flow cytometry using a Becton-Dickinson FACSCaliber four color Cytometer. Thirty thousand events were collected and microglia were differentiated from macrophages based on the levels of CD11b and CD45 surface expression. Microglia stain CD11b+/CD45low and macrophages stain CD11b+/CD45high [48,49]. Flow data were analyzed using FlowJo software (Tree Star, San Carlos, CA).<br><br>Behavior tests
Social exploratory behavior
To assess the motivation to engage in social exploratory behavior, a novel juvenile conspecific was introduced into the test subject's home cage for a 10-min period. Behavior was video taped and the cumulative amount of time the subject engaged in social investigation was determined from the video records by a trained observer who was blind to the experimental treatments. Baseline social behavior was measured at time 0 for all experimental treatments. Social behavior was determined as the amount of time that the experimental subject spent investigating (e.g., anogenital sniffing, trailing) the juvenile. Results are expressed as percent decrease in time engaged in social behavior compared to respective baseline measures.<br><br>Sucrose preference
To assess sucrose preference, <span class="yellow">mice</span> were provided two solutions, water or water supplemented with 2% sucrose, in 50 ml conical tubes with stoppers fitted with ball-type sipper tubes. Prior to testing conditions, all <span class="yellow">mice</span> were acclimated to the two bottle test choice. All <span class="yellow">mice</span> drank both the water and the 2% sucrose solution, but preferred drinking the sucrose over the water (data not shown). On the day of testing, <span class="yellow">mice</span> were fluid and food deprived for 2 h prior to testing [50]. At the start of the dark phase of the photoperiod, drinking water and the 2% sucrose solution were placed in the home cage overnight (15 h). At the end of each testing period the fluid content of the conical tubes was measured and sucrose preference was determined using the equation: Sucrose intake/Total fluid intake (water + sucrose intake) × 100 [51].<br><br>
Plasma cytokine measurement
IL-6 and IL-1β were measured in the plasma as previously described [52]. In brief, <span class="yellow">mice</span> were euthanized by CO2 asphyxiation and blood was collected by cardiac puncture into EDTA coated syringes. Samples were centrifuged (6000 × g for 15 min at 4°C) and plasma was collected and stored frozen (-80°C) until assaying. Plasma samples were assayed for IL-6 using a customized ELISA that we have described in detail [52] and for IL-1β using a commercial ELISA kit (R&D Systems, Minneapolis, MN). Assays were sensitive to 8 pg/ml of IL-6 and 1.5 pg/ml of IL-1β, and inter- and intra-assay coefficients of variation were less than 10%.<br><br>Real time PCR
Total RNA was isolated from brain using the Tri Reagent protocol (Sigma, St. Louis, MO). RNA samples were subjected to a DNase I digestion procedure and then reverse transcribed to cDNA using a RT RETROscript kit (Ambion, Austin, TX). Quantitative real time PCR was performed using the Applied Biosystems (Foster, CA) Assay-on Demand Gene Expression protocol as previously described [13]. In brief, cDNA was amplified by real time PCR where a target cDNA (IL-1β, IL-6, MHC II, TLR2, or IDO) and a reference cDNA (glyceraldehyde-3-phosphate dehydrogenase) were amplified simultaneously using an oligonucleotide probe with a 5' fluorescent reporter dye (6-FAM) and a 3' quencher dye (NFQ). Fluorescence was determined on an ABI PRISM 7300-sequence detection system (Applied Biosystems, CA). Data were analyzed using the comparative threshold cycle (Ct) method and results are expressed as fold difference.<br><br>Experimental protocols
For the cell culture studies, minocycline was prepared in dimethyl sulfoxide (DMSO) and BV-2 cells were washed and replenished with growth mediumsupplemented with 0, 25, 50, 100, 200, or 400 μg/ml minocycline. After 30 min, LPS at 10 ng/ml was added to the culture medium. Supernatants were collected 4 h later and IL-6 and IL-1β concentrations were determined by ELISA. Total proteins were determined from cell culture homogenates by the Bio-Rad Dc protein assay according to the manufacturer's instructions (Bio-Rad Lboratories, Hercules, CA). Each treatment was replicated a minimum of four times. Cell viability was confirmed by the MTS cell proliferation assay according to the manufacturer's instructions (Promega, Madison, WI).
For all <span class="yellow">mouse</span> studies, minocycline (Sigma, St. Louis, MO) was dissolved in sterile water and sonicated to ensure complete solubilization. In the first <span class="yellow">mouse</span> study, adult male BALB/c <span class="yellow">mice</span> received an intraperitoneal (i.p.) injection of vehicle or minocycline (50 mg/kg) for three consecutive days. On the 3rd day, <span class="yellow">mice</span> were also injected i.p. with saline or <span class="blue">Escherichia coli</span> LPS (0.33 mg/kg; serotype 0127:B8, Sigma, St. Louis, MO) and were euthanized by CO2 asphyxiation 24 h later (n = 6). The LPS dosage was selected because it elicits a proinflammatory cytokine response in the brain resulting in mild transient sickness behavior in adult <span class="yellow">mice</span> [13,53]. Macrophage/microglial cells were isolated from whole brain homogenates and TLR2 and MHC II surface expression were determined by flow cytometry. The minocycline injection regimen and dosage was selected because a repeated pretreatment course with minocycline is necessary to attenuate neuroinflammation [41-43,45].
In the second study, adult male BALB/c <span class="yellow">mice</span> received an i.p. injection with vehicle or minocycline for three consecutive days. On the third day, motivation to engage in social behavior was determined immediately before i.p. injection of saline or LPS (0.33 mg/kg) and again 2, 4, 8, 12, and 24 h later (n = 8). Body weight and food intake were measured at each time point over the 24 h period. In a related, but separate study, adult <span class="yellow">mice</span> were treated with minocycline and LPS as described and anhedonia was assessed 24–39 h following i.p. injection of saline or LPS (0.33 mg/kg) (n = 15). Body weight, food intake, water intake, and sucrose intake were determined over the testing period.
In the third study, adult BALB/c <span class="yellow">mice</span> were treated with minocycline and then LPS as described. <span class="yellow">Mice</span> were euthanized by CO2 asphyxiation 4 later. Brains were removed and dissected to collect different brain regions. Brain regions were stored at -20°C in RNAlater (Qiagen, CA). Total RNA was isolated from brain samples and assayed using quantitative PCR (n = 8). Plasma was also collected and stored (-80°C) until assaying.
In a final study, adult (3–4 month old) or aged (20–22 month old) male BALB/c <span class="yellow">mice</span> were treated with minocycline and LPS as described and euthanized 4 h later. Brains were dissected to collect different brain regions and were stored at -20°C in RNAlater (Qiagen, CA). Total RNA was isolated from the hippocampus and assayed using quantitative PCR (n = 8).<br><br>Statistical analysis
All data were analyzed using Statistical Analysis Systems (SAS) General Linear Model procedures. Data were subjected to one, two- (Mino × LPS, Age × LPS, Mino × Age) or three-way (Mino × LPS × Time, Mino × LPS × Age) ANOVA to determine significant main effects and interactions between main factors. When appropriate, differences between treatment means were evaluated by an F-protected t-test using the Least-Significant Difference procedure of SAS. All data are expressed as treatment means ± standard error of the mean (SEM).<br><br>
Results
Minocycline attenuates LPS-induced cytokine production in BV-2 microglia
Minocycline is a tetracycline-type antibiotic that has anti-inflammatory properties in the CNS [41-43,45]. To determine the degree to which minocycline suppresses microglia activation, BV-2 microglia-derived cell lines were used. In the first experiment, BV-2 cells were treated with LPS and IL-6 production was determined 4 h later. Fig. 1A shows that LPS increased IL-6 production in a dose dependent manner F(5, 23) = 101, P < 0.001). In the second experiment, BV-2 cells were incubated with DMSO or minocycline and then stimulated with LPS. Minocycline reduced LPS-induced IL-6 secretion in a dose dependent manner (Mino × LPS interaction, F(4, 23) = 16.87, P < 0.001, Fig. 1B). Minocycline pretreatment had a similar anti-inflammatory effect on LPS-stimulated IL-1β secretion (Fig. 1C). In a third experiment, minocycline suppressed LPS-induced MHC II, TLR2, IL-1β, and IL-6 mRNA levels (P < 0.05, for each, Fig. 1D). The MTS assay verified that neither cell survival nor proliferation was affected by the experimental treatments (data not shown).<br><br>LPS-induced TLR2 surface expression on microglia is reduced by minocycline
Because minocycline attenuated LPS-induced cytokine secretion and TLR2 mRNA expression in BV-2 cells we next sought to determine if minocycline suppresses markers of microglial activation in the brain of <span class="yellow">mice</span>. <span class="yellow">Mice</span> were injected i.p. with vehicle or minocycline for 3 consecutive days then challenged with saline or LPS i.p. Markers of activation, TLR2 and MHC II, were determined on microglia collected 24 h later. The representative bivariate density plot in Fig. 2A shows that there were two populations of CD11b/CD45 positive cells and that more cells stained CD11b+/CD45low (microglia) than CD11b+/CD45high (CNS macrophages). ANOVA revealed that LPS injection increased TLR2 surface expression on microglia (F(1, 20) = 17.6, P < 0.004, Fig. 2B&D), but this induction was abrogated by minocycline pretreatment (Tendency for Mino × LPS interaction, F(1, 20) = 2.66, P = 0.10, Fig. 2C&D). It is important to note that because minocycline and saline controls did not differ in their TLR2 expression, these data were grouped together as the Control group (Fig. 2B&C). In addition, neither minocycline nor LPS treatment had a significant main effect on MHC class II surface expression on microglia (data not shown). These data indicate that minocycline attenuated LPS-induced TLR2 expression on microglia.<br><br>Minocycline facilitates the recovery from LPS-induced sickness behavior
CNS macrophages and microglia produce inflammatory cytokines and secondary messengers that modulate behavioral responses. Therefore, we next investigated if minocycline reduced the sickness response associated with peripheral LPS injection. In this experiment, adult <span class="yellow">mice</span> were treated with minocycline and LPS as described. Social exploratory behavior was measured before i.p. LPS injection and again 2, 4, 8, and 24 h later. Fig. 3A shows that LPS injection caused a reduction in social exploratory behavior (F(1,57) = 218, P < 0.001) that was time dependent (F(4,57) = 66.5, P < 0.001). Moreover, the LPS-associated reduction in social exploration was attenuated by minocycline (Mino × LPS interaction, F(1,57) = 7.5, P < 0.007). For example, at 8 h post LPS, social exploration was reduced by 35% in minocycline pretreated <span class="yellow">mice</span> given LPS compared to a 67% reduction in vehicle pretreated <span class="yellow">mice</span> given LPS (P < 0.001). While minocycline administration alone reduced food intake and body weight in control <span class="yellow">mice</span> (P < 0.05, for each), it also protected against LPS-associated anorexia (Mino × LPS interaction, F(1, 60) = 70.0, P < 0.001, Fig. 3B) and weight loss (Mino × LPS interaction, F(1, 60) = 29.7, P < 0.001, Fig. 3C).
Because sickness can also be associated with longer lasting changes in motivation [38], we next sought to determine if minocycline abrogated LPS-induced anhedonia [54,55]. In this experiment, <span class="yellow">mice</span> were subjected to the same minocycline injection regimen and LPS challenge as above and sucrose preference was assessed 24–39 h post LPS injection. By 24 h post LPS injection, food and water intake returned to baseline and LPS treated <span class="yellow">mice</span> still exhibited a marked reduction in sucrose preference from 24–39 h (F(1,59) = 14.3, P < 0.003). Moreover, this LPS-dependent reduction in sucrose preference was prevented by minocycline pretreatment (Mino × LPS interaction, F(1, 59) = 9.9, P < 0.004, Fig. 4). For example, minocycline pretreated <span class="yellow">mice</span> injected with LPS maintained the same strong preference for sucrose as saline and minocycline controls (i.e., approximately 85% preference). These data can be interpreted to indicate that minocycline blocks anhedonia associated with peripheral LPS challenge.<br><br>Minocycline reduces LPS-induced neuroinflammation
Pro-inflammatory cytokines in the CNS are partially responsible for the behavioral symptoms of sickness (e.g., anorexia, social withdrawal, and anhedonia) [1]. Therefore, we investigated the degree to which minocycline reduces neuroinflammation (IL-1β, IL-6, and IDO) after peripheral injection of LPS. In this experiment, <span class="yellow">mice</span> were subjected to the minocycline injection regimen and LPS challenge as above and cytokine mRNA levels were determined in the cortex and hippocampus 4 h after LPS injection. In <span class="yellow">mice</span> pretreated with vehicle, LPS markedly increased IL-1β mRNA levels in the hippocampus (F(1,31) = 62, P < 0.0001) and cortex (F(1,31) = 17.25, P < 0.0003). The LPS-induced IL-1β mRNA expression was reduced in both brain regions in <span class="yellow">mice</span> receiving minocycline prior to LPS injection: (hippocampus, F(1,31) = 9.63, P < 0.01) and cortex, F(1,31) = 7.23, P = 0.08, Fig. 5A). LPS caused a similar induction of IL-6 mRNA levels in the hippocampus (F(1,31) = 37.2, P < 0.001) and cortex (F(1,31) = 22.5, P < 0.001), but minocycline pretreatment only significantly attenuated LPS-induced IL-6 mRNA levels in the hippocampus (F(1,31) = 10.27, P < 0.004, Fig. 5B).
IDO mRNA levels were determined from the same RNA pool. Fig. 6D shows that LPS injection increased IDO mRNA expression in the hippocampus (F(1,31) = 11.69, P < 0.002) and cortex (F(1,31) = 5.26, P < 0.03). This LPS-induced IDO mRNA expression was attenuated by minocycline in the hippocampus (F(1,31) = 11.69, P < 0.002) and cortex (F(1,31) = 5.26, P < 0.03). It is important to note that IDO mRNA was undetected in saline treated <span class="yellow">mice</span>. Therefore, the fold IDO change was relative to the IDO mRNA levels in <span class="yellow">mice</span> receiving minocycline prior to LPS.<br><br>Minocycline reduces LPS-induced IL-6, but not IL-1β, in the plasma
Because cytokine signals can be relayed from the periphery to the brain by humoral pathways [56], plasma cytokine levels of IL-6 and IL-1β were determined 4 h post LPS injection. As expected, LPS injection caused a marked increase in plasma IL-1β (F(1,36) = 52.5, P < 0.001) and IL-6 levels (F(1,36) 34.01, P < 0.01). Minocycline pretreatment reduced LPS-induced IL-6 levels in the plasma (F(1,36) 6.68, P < 0.01) but had no significant main effect on LPS-induced IL-1β levels (Fig. 6).<br><br>Minocycline attenuates LPS-induced exaggerated neuroinflammation in aged <span class="yellow">mice</span>
Aged BALB/c <span class="yellow">mice</span> (22–24 m) have an exaggerated neuroinflammatory response to LPS injection [10,13,14]. Therefore, we next sought to determine if the heightened inflammatory response in the brain of aged <span class="yellow">mice</span> was reduced by minocycline. In this experiment, adult and aged <span class="yellow">mice</span> were subjected to the minocycline injection regimen and LPS challenge as above. As we have reported previously, MHC II mRNA expression was increased by age (P < 0.03, Fig. 7A)[13,14], but MHC II levels were unaffected by either LPS or minocycline treatment (not shown). Consistent with the data presented in Fig. 2, ANOVA revealed a significant main effect of LPS injection on TLR2 mRNA expression in the hippocampus (F(1,63) = 85.5, P < 0.001). Moreover, LPS caused a greater increase in TLR2 mRNA in the hippocampus of aged <span class="yellow">mice</span> compared to adults (LPS × Age interaction, F(1,63) = 12.70, P < 0.01). Furthermore, minocycline pretreatment attenuated LPS-induced TLR2 mRNA levels in both adult and aged <span class="yellow">mice</span> (Mino × LPS interaction, F(1,63) = 9.02, P < 0.004).
Parallel to the results for TLR2, LPS caused a greater increase in IL-1β and IDO mRNA levels in hippocampus of aged <span class="yellow">mice</span> compared to adults (Age × LPS, F(1,60) = 8.64, P < 0.01 for IL-1β and F(1,60) = 4.0, P < 0.05 for IDO). Minocycline pretreatment attenuated LPS-induced mRNA levels of IL-1β (Mino × LPS, F(1,60) = 8.76, P < 0.01, Fig. 7C) and IDO (Mino × LPS, F(1,60) = 9.7, P < 0.003, Fig. 7D). While LPS induced higher IL-6 mRNA levels in the hippocampus of both adult and aged <span class="yellow">mice</span> (F(1,59) = 44.5, P < 0.001), there was not an Age × LPS interaction. Minocycline pretreatment attenuated the LPS-induced increase in hippocampal IL-6 mRNA (Mino × LPS, F(1,59) = 5.4, P < 0.02, Fig. 7E). Taken together these data indicate that minocycline pretreatment was effective in attenuating the exaggerated neuroinflammation in aged <span class="yellow">mice</span>.<br><br>
Discussion
In the elderly, systemic infection is associated with an increased frequency of behavioral and cognitive complications [57,58]. We have reported that stimulation of the peripheral immune system in older (20–24 m) BALB/c <span class="yellow">mice</span> causes exaggerated neuroinflammation that is paralleled by prolonged sickness [13], impaired working memory [10], and depressive-like behaviors [15]. Therefore, it is important to understand the mechanisms that can modulate cytokine-mediated pathways in the brain. Here we show that minocycline treatment reduced LPS-induced TLR2 expression in BV-2 cells and on microglia isolated from adult <span class="yellow">mice</span>. Moreover, we demonstrate that minocycline was effective in facilitating the recovery from LPS-induced sickness and preventing anhedonia in adult <span class="yellow">mice</span>. Furthermore, we show that minocycline attenuated LPS-induced neuroinflammation in adults and normalized the exaggerated neuroinflammation in aged <span class="yellow">mice</span>.
Our findings, using cell culture and animal experiments, support the notion that minocycline attenuates microglial activation and limits production of inflammatory mediators. For instance, minocycline pretreatment of BV-2 cultures decreased LPS-stimulated cytokine production in a dose dependent manner (Fig. 1A). In BV-2 cells, minocycline also attenuated mRNA expression of inflammatory genes including IL-6, IL-1β, MHC II, and TLR2 (Fig. 1D). These data are consistent with other studies using minocycline and LPS in BV-2 cells [44,59]. Based on these data we next investigated if microglial activation could be attenuated in the brain. Because LPS increases brain cytokine production we expected that MHC II expression would also be increased. Contrary to our predictions, neither MHC II mRNA levels (Fig. 7) in the brain nor MHC II surface expression on microglia (CD11b+/CD45low) (data not shown) were increased by LPS injection. In an EAE model, minocycline reduced microglial expression of MHC II [45], but one key difference from our study is that the induction of EAE pathology requires functional antigen presentation on MHC II [60]. It is postulated that microglia have several activation states that depend on the specific inflammatory stimulus [61]. Thus, in situations of transient peripheral innate immune stimulation, markers in the CNS such as Toll-Like receptors [6] may be indicative of microglia activation. In support of this premise, our data show that LPS injection increases TLR2 surface expression on microglia (CD11b+/CD45low), which is inhibited by minocycline pretreatment (Fig. 2). These data are consistent with other studies showing that central or peripheral LPS challenge increases TLR2 mRNA in the brain [6,14]. Taken together our findings can be interpreted to suggest that minocycline attenuates pathways associated with microglia activation following peripheral LPS challenge.
One of the important findings of this study was that reduction of neuroinflammation by minocycline was associated with facilitated recovery from LPS-induced sickness behavior. These results are akin to our previous work with the anti-oxidant, α-tocopherol [52], and an NFKB decoy inhibitor [62]. Consistent with our previous studies [52,53,62,63], reductions in neuroinflammatory cytokines (Fig. 5) did not prevent the induction of the LPS-induced sickness response (2–4 h), but rather facilitated the recovery from sickness (8–24 h) (Fig. 3A). Recovery may be a critical issue because brain cytokines and the corresponding physiological and behavioral responses are beneficial to the host [1]. The potential risk for a maladaptive response occurs when the normally transient neuroinflammatory response is amplified or protracted [64]. Therefore pharmacological agents, similar to minocycline, that attenuate neuroinflammatory responses, but do not completely inhibit them, may be important in preventing the development of more severe and long-lasting cognitive and behavioral complications.
The results of the sucrose preference experiments support the idea that limiting exposure to neuroinflammation decreases the duration of behavioral responses. For example, while minocycline did not inhibit cytokine expression (Fig. 5) or the induction of sickness (Fig. 3A), minocycline pretreatment completely reversed the reduction in sucrose preference (i.e., anhedonia) associated with LPS injection (Fig. 4). It is also important to mention that while LPS-associated sickness and anhedonia are interrelated, these behaviors can be differentiated from one another. For instance, reduced social exploration was evident 2–24 h post injection (Fig. 3A), but only decreased sucrose preference was exhibited 24 to 39 h later (Fig. 4). This separation between behaviors is consistent with other studies investigating sickness and longer-lasting changes in motivation [15,65,66].
IDO mediated TRP metabolism represents a potential connection between activation of CNS innate immune cells and longer lasting behavioral responses. IDO mediated TRP metabolism in the brain may affect behavior by impacting both serotonin and glutamate pathways [39]. We have reported that IDO induction and activity is amplified in the brain of aged <span class="yellow">mice</span> and is associated with prolonged depressive-like behavior [15]. Here we show that IDO mRNA induction is blocked by minocycline in the brain of both adult and aged <span class="yellow">mice</span> (Figs. 5&7). These data are consistent with a recent report showing a causal relationship between IDO activity and acute depressive effects in adult CD-1 <span class="yellow">mice</span>. In this study, O'Connor et al. report that both 1-methyl tryptophan (a competitive inhibitor of IDO) and minocycline blocked IDO induction and prevented depressive-like immobility in the tail suspension and forced swimming tests [66]. Thus, in the present study, the minocycline blockade of IDO induction may explain the abrogation of LPS-induced anhedonia.
Another interesting finding was that while minocycline pretreatment in adult <span class="yellow">mice</span> attenuated LPS-induced brain IL-1β at 4 h (Fig. 5), it had no effect on plasma IL-1β levels (Fig. 6). Because IL-1β signals can be relayed from the periphery to the brain by humoral pathways [5], these findings suggest that minocycline has anti-inflammatory properties within the brain. These data are consistent with related findings that minocycline readily crosses the blood brain barrier to elicit an anti-inflammatory effect [41-43]. With regard to IL-6, minocycline pretreatment attenuated both brain and plasma levels at 4 h post LPS injection. Because circulating IL-6 levels can be increased by CNS mediated pathways including activation of the hypothalamus-pituitary-adrenal (HPA) axis [67] and the sympathetic nervous system [68], the specific reduction in plasma IL-6 by minocycline may reflect the reduction in brain inflammation at 4 h (Fig. 5). In support of this notion, we and others have reported that i.c.v. injection of LPS or IL-1β increase plasma IL-6 levels, but not IL-1β levels [14,68,69].
The final critical finding of this study was that minocycline was effective in attenuating neuroinflammation independent of age. Consistent with other aging and neuroinflammation studies, our data show that LPS caused exaggerated neuroinflammation in aged <span class="yellow">mice</span> compared to adults [10,13-15]. It is important to mention that while there was an age-related difference in MHC II expression in the hippocampus of saline treated <span class="yellow">mice</span> (Fig. 7A) there was not an age-related difference in IL-1β and IL-6 mRNA levels. These data differ from a previous report using BALB/c <span class="yellow">mice</span> showing an increase in IL-6 in older <span class="yellow">mice</span> [70]. This may be because the <span class="yellow">mice</span> used in the present study were approximately 4 months younger than the <span class="yellow">mice</span> used previously. Nonetheless, microglia can be primed or reactive with increased MHC II expression, but do not necessarily produce inflammatory cytokines in this state [19]. The key results are that peripheral LPS injection causes a greater induction of TLR2, IL-1β, and IDO mRNA in the aged brain than in the adult brain and that minocycline pretreatment normalizes this age-related exaggerated neuroinflammation (Fig. 7). These findings are also important because an amplified neuroinflammatory response in the aged brain is a precursor to complications including deficits in working memory, memory consolidation, and depressive-like behavior [9,10,15]. Based on the biochemical and behavioral data obtained from this study, we predict that minocycline will abrogate the prolonged LPS-induced sickness [13] and depressive-like behavior exhibited by aged BALB/c <span class="yellow">mice</span> [15]. We acknowledge, however, that future studies are necessary to test these predictions.<br><br>Conclusion
The present study demonstrates that minocycline reduces LPS-induced microglial activation, CNS cytokine production, and behavioral symptoms of sickness (e.g., social withdrawal and anhedonia). These findings are potentially important because they indicate that minocycline can be used to mitigate cytokine expression in the brain and have a beneficial affect on behavioral responses. Taken together, these data support the idea that pharmacological strategies aimed at decreasing neuroinflammation associated with microglial activation are important for improving recovery from sickness and reducing the frequency of neurobehavioral complications.<br><br>List of abbreviations
3-Hydroxy-L-Kynuriene (3HK), Allophycocyanin (APC), Analysis of variance (ANOVA), Central Nervous System (CNS), Dulbecco's Modified Eagle's Medium (DMEM), Dimethyl Sulfoxide (DMSO), Experimental Autoimmune Encephalomyelitis (EAE), Enzyme Linked Immmunosorbent Assay (ELISA), Fluorescein Isothiocyanate (FITC), Fetal <span class="yellow">Bovine</span> Serum (FBS), Hank's Balanced Salt Solution (HBSS), Indoleamine 2, 3 dioxygenase (IDO), Intraperitoneal (i.p.), Intracerebroventricular (i.c.v.), Interleukin (IL), Kynurenine (KYN), Lipopolysaccharide (LPS), Major Histocompatibility Complex class II (MHC II), Mitogen Activated Protein Kinase (MAP-kinase), Nuclear factor kappa B (NFκB), N-methyl-D-aspartate (NMDA), R-Phycoerthrin (PE), Quinolinic acid (QUIN), Statistical Analysis Systems (SAS), Standard Error of the Mean (SEM), Toll-like Receptor-2 (TLR2), and Tryptophan (TRP).<br><br>Competing interests
The authors of this manuscript declare that there are no actual or potential conflicts of interest. The authors affirm that there are no financial, personal or other relationships with other <span class="yellow">people</span> or organizations that have inappropriately influenced or biased their research.<br><br>Authors' contributions
CJH was involved in research experimentation, completion of statistical analysis, and writing of the manuscript. YH, AW, MH and JH assisted with experimentation and data analysis. MB and JFS contributed to the design of the experiments and assisted in editing the manuscript. JPG directed all aspects of this research project including the experimental design, research experimentation, completion of statistical analysis, and writing of the manuscript.<br><br>
<h3>pmcA2553092</h3>TOPS++FATCAT: Fast flexible structural alignment using constraints derived from TOPS+ Strings Model
Abstract
Background
Protein structure analysis and comparison are major challenges in structural bioinformatics. Despite the existence of many tools and algorithms, very few of them have managed to capture the intuitive understanding of protein structures developed in structural biology, especially in the context of rapid database searches. Such intuitions could help speed up similarity searches and make it easier to understand the results of such analyses.<br><br>Results
We developed a TOPS++FATCAT algorithm that uses an intuitive description of the proteins' structures as captured in the popular TOPS diagrams to limit the search space of the aligned fragment pairs (AFPs) in the flexible alignment of protein structures performed by the FATCAT algorithm. The TOPS++FATCAT algorithm is faster than FATCAT by more than an order of magnitude with a minimal cost in classification and alignment accuracy. For beta-rich proteins its accuracy is better than FATCAT, because the TOPS+ strings models contains important information of the parallel and anti-parallel hydrogen-bond patterns between the beta-strand SSEs (Secondary Structural Elements). We show that the TOPS++FATCAT errors, rare as they are, can be clearly linked to oversimplifications of the TOPS diagrams and can be corrected by the development of more precise secondary structure element definitions.<br><br>Software Availability
The benchmark analysis results and the compressed archive of the TOPS++FATCAT program for Linux platform can be downloaded from the following web site: <br><br>Conclusion
TOPS++FATCAT provides FATCAT accuracy and insights into protein structural changes at a speed comparable to sequence alignments, opening up a possibility of interactive protein structure similarity searches.<br><br><br><br>Background
Structural biology is one of the most successful fields of modern biology. Over 50,000 solved protein structures illustrate details of many specific biological processes. The same data also provide us with information about the global features of protein structure space and can be studied to discover the evolutionary, physical, and mathematical rules governing them. How many fundamentally different protein shapes (folds) are there? How do protein structures evolve? How do new structural features appear, and if they are coupled with changes in function, how does this process occur? Such questions can be studied by classifying, comparing and analyzing known protein structures. Two different, but synergistic strategies are typically used for this purpose. In classification systems such as SCOP [1] or CATH [2], <span class="yellow">human</span> intuition is used to simplify the description of protein structures to a manageable size, and a <span class="yellow">human</span> eye, sometimes supported by automated analysis, can recognize patterns and types of structures. In the second approach, specialized comparison algorithms, such as DALI [3], CE [4], or FATCAT [5] can be used to calculate a distance-like metric in the protein structure space. This in turn can be used to cluster proteins into groups. Many such algorithms have been developed over the past few decades and have been mostly used for the classification of protein structures into families.
An exact solution of an alignment between two structures is formally equivalent to a threading problem and is therefore NP-complete [6]. However, a practical solution can be obtained by heuristics reducing the problem to a manageable size [7]. In <span class="yellow">human</span> classification systems, the protein is usually reduced to a set of several structural elements, which obviously involve many arbitrary thresholds. Automated algorithms have the same problem and also suffer from inconsistencies between different numerical measures of protein structure similarity [8]. Interestingly, despite these problems, results of different approaches are broadly similar. They all identify approximately a few hundred general classes of protein structures, usually called folds [1] or topologies [2], distinguished by how the main chain of the protein folds around itself in the three-dimensional space. At the same time, the comparison of different approaches, both between and within the two classes, shows that fold/topologies (or cluster) definitions are somewhat fuzzy, with some proteins being occasionally difficult to classify and joining different groups depending on various assumptions. This lead some to question the concept of the fold [9], but practical application of protein structure comparison leaves little doubt that protein structure space has some natural granularity that overlaps well with the traditional fold classification.
Comparison and classification of protein structures is significantly simplified by the fact that proteins have naturally modular structures, being mostly composed of locally regular structures: alpha helices and beta strands. These two types of secondary structures constitute a little over 50% of an average protein's length. With the average length of a secondary structure being around 10 amino acids, this makes it possible to describe protein structure as an arrangement of a much smaller number of elements. Protein structures are often visualized in a simplified form, with the so-called ribbon diagram with secondary structures shown as helices and arrows being the most popular (see Figure 1). This picture can be simplified further by showing individual secondary structure elements as simple symbols (circles or boxes/triangles). These depictions, called fold diagrams, originally proposed in the 70s [10-12] are best captured by a TOPS (Topology of Protein Structures) algorithm, which attempts to automate the process of creation of the topology cartoon [13]. While useful in protein classification, such simplified descriptions are not used in the most popular automated protein structure comparison algorithms such as DALI [3] or CE [4]. Kleywegt and Jones developed a method for finding similar motifs based on comparing distance matrices that are constructed by representing protein as a set of SSEs with their directional vectors and angle between those vectors [14]. Programs that used SSEs either for structure comparison based on hierarchical superposition of both SSEs and atomic representation [15] or for finding common substructures in the comparison process based on subgraph isomorphism, such as [16,17] and recent applications of the TOPS diagram [18,19], usually struggle with translating the comparison results from the secondary structure to the individual residue level. Although the SSM method uses graph-matching procedures at the SSE level followed by an interactive 3D alignment of the protein C-alpha atom [20], it lacks the topological relationships between the SSEs, which are essential features in identifying common scaffolds in distantly related proteins. A TOPS pattern was used to guide the sequence alignment, for instance, to build multiple structural alignments of the distantly related family of beta-rich protein domains [21]. The Multiple Sequence Alignment Tool (MSAT) automates this approach, merging it with a popular ClustalW program [22]. DALI [3], CE [4] or FATCAT [5] introduce their own methods of decomposing the protein structure into smaller units, such as 7 × 7 dense distance map fragments (DALIs) or aligned fragment pairs (AFPs) (CE and FATCAT). The large number of such fragments and the combinations of the fragments that need to be evaluated by structure comparison programs is the main reason for the significant computational requirements of such algorithms. However, more importantly, TOPS+ method is used here to enable a structural comparison that takes into account flexibility in protein structures and not only classifies the differences, but also can recognize such rearrangements – which is a first such application using the SSEs language. In this contribution, we explore the question of whether it would be possible to combine insights provided by topology diagrams into automated protein structure alignment algorithms, focusing on the FATCAT program developed previously in our group.<br><br>Methods
Flexible structure alignment method FATCAT
FATCAT [5] is a unique structure alignment method that allows for flexibility in the structures being compared. It builds the alignment by chaining aligned fragment pairs (AFPs) [23] together using a unified scoring function where AFP extensions, gaps, and twists each have their specific scores (Figure 2). Introducing a twist into the alignment is penalized, but this penalty may be compensated for by the gain in the score of the resulting alignment (i.e., longer alignment and/or better RMSD). Rigid alignment can be treated as a special case, in which no twist is allowed in chaining AFPs. FATCAT program provides alignment in both, "rigid" mode and "flexible" (default) mode.
FATCAT, as well as most other protein structure comparison programs, is very slow when compared to sequence alignments. The computing time of FATCAT is determined by the size of the collection of AFPs detected between the two structures being compared. FATCAT is available from a server  with an option to search in SCOP or PDB databases for similar structures. This search typically takes between 8 to 16 hours of CPU time, and this is the main obstacle to broader use of this option. FATCAT has been used to construct a Flexible Structure Neighborhood (FSN) database that contains pre-computed results of structure similarity searches and it takes several weeks of CPU time to update the FSN database. Other protein structure comparison resources, such as DALI or CE have very similar problems.<br><br>TOPS cartoons and TOPS graph models
As discussed in the Background, TOPS cartoons capture the simplified, fold-level description of protein structure and at the same time can be automated [24]. The TOPS algorithm uses structural features such as hydrogen bonds and chirality of the beta strands to provide a scoring function to optimize the cartoon (see Figure 1(b)). In TOPS, the secondary structural elements (SSEs) are derived from the DSSP program [25]. Based on TOPS cartoons, a formal graph model and graph-based definitions of protein topology and pattern discovery and comparison methods were developed [26,27]. The TOPS database and comparison, pattern discovery and matching programs are accessible from .<br><br>Novel TOPS+ and TOPS+ strings models
The TOPS model was further enhanced to incorporate features such as protein-ligand interaction information and more detailed secondary structural segment information. This enhanced model is called TOPS+ model (see Figure 3a). This TOPS+ model can be described formally in a TOPS+ strings language (Figure 3b) at a reduced linear level. The enhanced TOPS+ strings models can be used in fast string-based structure matching and comparison, at the same time avoiding issues of NP-completeness associated with graph alignments.
In detail, each node (SSE segment) of the TOPS+ strings is described by its type, orientation, PDB start number, segment length, total number of incoming (InArc) and outgoing (OutArc) arcs (edges), total number of ArcTypes, and total number of ligand arcs (LigArc). The type of the segment (SSEType) could be one of [E, e, H, h, U, u], where, "E" and "e" represent the "up"- and "down"-oriented beta strands; "H" and "h" indicate the "up"- and "down"-oriented alpha helices; and "U" and "u" represent ligand-bound and ligand-free loops. The InArcType can be classified as an/a [R, L, P, A], where "R" and "L" represent right and left chiralities; and "P" and "A" represent parallel and anti-parallel hydrogen bonds, respectively. The OutArcType is represented in a similar manner by [R', L', P', A']. Ligand arcs are indicated by LT = AA, where LT is the ligand type and AA is the PDB number. For example, Figure 3(a) and 3(b) contain visual representations of TOPS+ and TOPS+ strings models, respectively, for the protein domain d1fnb_1. Here the triangles represent the beta strands; the red curve represents the alpha helix; gray ellipsoids indicate loops; and green arcs indicate hydrogen bonds between two beta strands, called anti-parallel beta sheets. The length of a TOPS+ strings model is defined by number of SSEs; thus, the length of d1fnb_1 is 19. For further details, see [28].<br><br>TOPS+ strings comparison method
TOPS+ is a comparison method that computes a distance between TOPS+ strings models of two proteins based on a dynamic programming approach and identifies the longest common subsequence (LCS), consisting of the list of the topologically equivalent SSEs between two proteins. For example, Figure 3(c) shows the TOPS+ strings alignment between Dihydropteridine reductase proteins from <span class="yellow">rat</span> (1dhr) and <span class="yellow">human</span> (1hdr). The TOPS+ strings models for 1dhr and 1hdr are represented by a linear string-model, where a yellow triangle and red curves indicate the beta strands and alpha helices in their "up" or "down" orientations, respectively. The grey line and purple stubs represent the loop regions and the NAD ligand interactions, respectively. Note that the ligand-interaction information is optional and in this work we have not used it. The incoming and outgoing arcs are depicted in the SSEs (top and bottom of the beta strands), where red and green arcs represent the parallel and anti-parallel hydrogen-bond interactions that show beta-sheet information, while yellow and blue arcs indicate the right and left chirality relationships between the SSEs. A pink arrow between the TOPS+ strings elements indicates the conserved SSE. The dotted arrows indicate the conserved alpha helices and beta strands, while the plain arrows indicate the conserved loop regions.<br><br>TOPS++FATCAT method
In this work, we want to test the general idea of pruning the search space of the FATCAT comparison process using topological constraints derived from the TOPS+ strings alignment. Many of the AFPs considered in the FATCAT alignment could be easily eliminated from the comparison by constraining the alignment region. Here we explore constraints obtained from the TOPS+ strings alignment, which identifies topologically equivalent secondary structure elements (alpha helices, beta strands, and loops) for this purpose. Such equivalences define blocks that restrict the alignment region; AFPs that fall outside these regions are simply not considered (see Figure 4(b)). We introduce a parameter r to control the strictness of constraints by TOPS+ strings alignments; r equals 0 if the alignment region is strictly restrained by TOPS+ strings alignment, and r is set to 1 by default in our program to allow certain flexibility to the constrained alignment region (Figure 4(c)). We then can speed up the FATCAT alignment by considering only the AFPs within the constrained alignment area (Figure 4(d)). The rigid structural alignment can be treated as a special case of TOPS++FATCAT, in which no twist is allowed in chaining AFPs. However, the TOPS++FATCAT program provides alignment in both, "rigid" mode and "flexible" mode (default).<br><br>Benchmarking
For benchmarking and comparison, we have used the PDB40 dataset of 1,901 protein domain pairs (DP) corresponding to SCOP version 1.61 from the ASTRAL database [29]. Table 1 provides the SCOP superfamily level homolog versus non-homolog statistics for the four main SCOP classes i.e., all-alpha, all-beta, alpha/beta, alpha+beta, and all proteins regardless of their structural classes.<br><br>Evaluation Analyses
We performed the Receiver Operating Characteristics (ROC) curve and the AUC (Area Under the ROC Curve) analyses to compare the performance of the TOPS++FATCAT method with the original FATCAT method, using SCOP classification at the superfamily level as a standard of comparison [30].<br><br>
Results
ROC and AUC Analyses
We have compared the performance of the TOPS++FATCAT method against the original FATCAT method using the SCOP classification information at the superfamily level. We have plotted the ROC curves based on P-values obtained from the FATCAT and the TOPS++FATCAT methods. We have plotted the ROC curves separately for the main SCOP classes, i.e., all-alpha, all-beta, alpha/beta, alpha+beta, and all proteins regardless of the class they belong to (see Figure 5(a) to 5(e)). In the graph, the x- and y-axes represent the false positive and true positive rates of the performance of the comparison methods respectively. In the legend, rF-pvalue and fF-pvalue indicate results from the rigid and flexible FATCAT methods, respectively; similarly, rT2F-pvalue and fT2F-pvalue represent the rigid and flexible TOPS++FATCAT methods, respectively. We have calculated the AUC values for all the SCOP classes based on ROC curves obtained from the FATCAT and TOPS++FATCAT methods with the flexible/rigid options (see Table 2).
For all protein classes, the rigid FATCAT performs best, usually followed by the flexible FATCAT, the rigid TOPS++FATCAT, and the flexible TOPS++FATCAT. The performance of all four methods is best for all alpha and all beta proteins, and all four perform markedly worse (but similar to each other) for alpha/beta proteins. Only alpha+beta proteins show a clear difference between the FATCAT and TOPS++FATCAT methods. It is important to note that the TOPS+ strings models consider the parallel and anti-parallel properties of the beta-sheet information in the form of total number of incoming and outgoing arcs with their ArcTypes. Thus, the TOPS++FATCAT method discriminates the protein domain pairs more efficiently compared to the original FATCAT method. For example, in the all-beta protein domain pairs, both the flexible and the rigid TOPS++FATCAT methods perform well. The flexible TOPS++FATCAT method covers nearly 84% of protein domains with 0% false positives, but the flexible and rigid FATCAT methods cover only 76% and 49% of the true positives, respectively, with 0% false positives. The zoomed-in version of the ROC curves with up to 10% false positives for all-beta rich protein families is shown in Figure 5(f); where both the rigid TOPS++FATCAT (green) and flexible (red) TOPS++FATCAT methods have coverage rates of 82% and 84% true positives respectively with 0% false positives. The overall results for all protein classes show that TOPS++FATCAT performance is only slightly lower (3%–7% AUC value difference (see Table 2)) as compared to FATCAT while providing a significant, more than 10-fold speedup (see next section).<br><br>AFP and Runtime Analyses
We tested both the FATCAT and TOPS++FATCAT methods using the Mac OS X version 10.4.10 computer system with a 2 × 2.66-GHz Dual-Core Intel Xeon processor and 1-GB 667 MHz memory. We have performed runtime analysis on 1,901 protein domain pairs and counted the total number of AFPs and the corresponding runtime from both the FATCAT and the TOPS++FATCAT methods. The results show an exponential increase in AFPs (Figure 6(b)) and corresponding runtime (Figure 6(a)) for the FATCAT method as compared to the TOPS++FATCAT method (see Table 3) For example, the average number of AFPs for the TOPS++FATCAT method is 530, but the average number of AFPs for the FATCAT method is 15,019. This represents the number of average AFPs used by the FATCAT method is increased by a factor of 28 (see Table 3). This result leads to the conclusion that TOPS++FATCAT is 22 times faster compared to the FATCAT because this method must take into account more number of AFPs in the comparison process (see Table 3).<br><br>Case Studies
While the overall accuracy of both rigid and flexible FATCAT methods is better than their TOPS++FATCAT equivalents, an interesting example where the opposite is true lies in the comparison of two proteins, d2trxa_ (108 aa) from <span class="blue">Escherichia coli</span> and d1kte__ (105 aa) from <span class="yellow">Sus scrofa</span> (<span class="yellow">pig</span>) from the thioredoxin-like superfamily. For this pair, the flexible_TOPS++FATCAT method provides an alignment with 88 equivalent positions with 1.67 Å chain RMSD and 3.06 Å of optimal RMSD without any twist, giving the alignment with 10% sequence identity (see Table 4). On the other hand, the flexible_FATCAT method provides an alignment with 86 aligned positions using a twist in the C-terminal region; it has a higher chain RMSD of 5.14 Å, and its optimal RMSD is 3.48 Å. For more information regarding the chain and optimal RMSDs refer [5]. The flexible_FATCAT method uses the twist to align a helix in the C-terminal region, which is positioned incorrectly with a beta-sheet core (see Table 4). Figure 7(a) shows the superposition of d2trxa_ (gray) and d1kte__ (orange) domains from the flexible_FATCAT method, where the blue color indicates the d1kte__ protein domain from the flexible_TOPS++FATCAT method. The incorrect alignment of the C-terminal domain alpha helix of the d1kte__ domain (orange) is visible in the core of the beta-sheet region. Figure 7(b) and 7(c) shows the AFPs from the flexible_FATCAT and flexible_TOPS++FATCAT methods, respectively. The hinge region provides a twist in the flexible_FATCAT method indicated by an arrow and the AFPs represented by a different color (see Figure 7(b)). In this case, the alignment constraints from the TOPS+ strings alignment allow the TOPS++FATCAT method to avoid a spurious alignment.
The Erythrocruorin protein domain d1eca__ (136 aa) from <span class="yellow">Chironomus thummi</span> and the Phycocyanin alpha subunit protein domain d1cpca_ (162 aa) from <span class="yellow">Fremyella diplosiphon</span> (Cyanobacterium) belong to the Globin-like superfamily. For these protein domain pairs, the FATCAT method provides a better alignment with 120 and 118 aligned positions with the chain RMSD of 4.02 Å based on the flexible and rigid options, respectively. The flexible_TOPS++FATCAT method gives an alignment of 63 aligned positions with the 3.23 Å optimal RMSD and the 6.28 Å chain RMSD. In this case, the flexible_TOPS++FATCAT method misses the N-terminal region helix and misaligns some helices. For example, Figure 8(a) shows the superposition of d1eca__ (gray) and d1cpca_ (orange) domains from the flexible_FATCAT method, while d1cpca_ (blue) domain is from the flexible_TOPS++FATCAT method. The AFP chaining alignment and the actual alignment from FATCAT are shown in Figure 8(b) and 8(e), respectively. Figure 8(c) shows the AFP alignment from TOPS++FATCAT, in which this method misses the N-terminal region and incorrectly aligns some of the C-terminal regions (see Figure 8(d)). However, the rigid_TOPS++FATCAT method produces an alignment of 108 aligned positions with optimal and chain RMSDs of 3.22 Å and 6.28 Å respectively. In general, TOPS comparison does not work well for alpha-rich proteins due to the lack of hydrogen bonds between SSEs [26]. The same is true for TOPS+ strings comparison to some extent; however, this method takes advantage of ligand-interaction information to compare protein domains more efficiently; for example the DNA binding motifs such as helix-turn-helix and helix-loop-helix can be easily recognized [28]. However, we have not explored that ligand pattern discovery option within the TOPS+ strings comparison in this paper. In addition, the TOPS+ strings alignment provides only a basic alignment; the scoring function to find the best alignment has not been optimized. These problems can be addressed in future development by considering the advanced TOPS+ and TOPS+ strings models based on helix-helix packing relationships and SSE-ligand interaction properties together with the right and left chiralities. Furthermore, the TOPS+ strings comparison can be optimized in both the comparison process as well as in the alignment process in order to take into account indels (insertion/deletion) of SSEs which exist in nature across the different members of the protein superfamilies [31].<br><br>
Discussion and conclusion
The overall results for all protein classes show that TOPS++FATCAT performance is only slightly lower (3%–7% AUC value difference) as compared to FATCAT while providing a significant, more than 10-fold speedup. The main reason for the discrepancies is that TOPS+ strings alignments occasionally misalign the secondary structure elements and subsequent FATCAT alignment, constrained by the TOPS+ strings alignment, cannot overcome the earlier errors. There is a clear trade-off between the runtime and the accuracy; limiting the pool of fragments being compared speeds up the algorithm but results in (slightly) lower accuracy. At the same time, these results offer clear suggestions for future development. Using a more advanced version of the TOPS+ strings comparison method would remove some of the false positives might be at a cost of significantly slowing the total performance of the TOPS++FATCAT method.<br><br>Authors' contributions
MV developed the TOPS++FATCAT algorithm, performed the calculations and prepared the figures, YY provided advice and oversight in the project, verified the code and provided FATCAT results for comparison, AG contributed to the original idea and to writing of the manuscript.<br><br>
<h3>pmcA1885664</h3>The recombination-associated protein RdgC adopts a novel toroidal architecture for DNA binding
Abstract
RecA plays a central role in the nonmutagenic repair of stalled replication forks in bacteria. RdgC, a recombination-associated DNA-binding protein, is a potential negative regulator of RecA function. Here, we have determined the crystal structure of RdgC from <span class="yellow">Pseudomonas aeruginosa</span>. The J-shaped monomer has a unique fold and can be divided into three structural domains: tip domain, center domain and base domain. Two such monomers dimerize to form a ring-shaped molecule of approximate 2-fold symmetry. Of the two inter-subunit interfaces within the dimer, one interface (‘interface A’) between tip/center domains is more nonpolar than the other (‘interface B’) between base domains. The structure allows us to propose that the RdgC dimer binds dsDNA through the central hole of ∼30 Å diameter. The proposed model is supported by our DNA-binding assays coupled with mutagenesis, which indicate that the conserved positively charged residues on the protein surface around the central hole play important roles in DNA binding. The novel ring-shaped architecture of the RdgC dimer has significant implications for its role in homologous recombination.<br><br>INTRODUCTION
Homologous recombination systems contribute to the maintenance of genome integrity and are essential for the repair of stalled replication forks (1). Genetic studies of the recombination-dependent growth C (rdgC) gene in bacterial systems have suggested that it is involved in DNA replication and recombination (2,3). In <span class="yellow">Neisseria gonorrhoeae</span> (the gonococcus), RdgC is required for efficient pilin antigenic variation and plays some role in cell growth (4,5). The <span class="blue">Escherichia coli</span> rdgC gene encodes a DNA-binding protein of 34 kDa, whose expression level was at its highest during exponential phase, reaching its maximum at ∼1000 dimers per cell (2). Its level decreased sharply to ∼50 dimers per cell in stationary phase. This profile suggests that RdgC might function during the period of DNA replication (2).
The RecA proteins and their homologs play a key role in recombinational DNA repair in bacteria and eukaryotes. Their aberrant reactions could result in gross chromosomal rearrangements that lead to <span class="yellow">human</span> diseases, including cancer. Therefore, their functions must be tightly regulated, and understanding how the RecA family of recombinases is regulated is of utmost importance (1). The <span class="yellow">E. coli</span> RdgC protein is a potential negative regulator of the function of RecA (1). It inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). Sedimentation equilibrium data indicated that <span class="yellow">E. coli</span> RdgC exists in solution as a mixture of oligomeric states in equilibrium, most likely monomers, dimers and tetramers (1). This concentration-dependent change in the oligomeric state appears to affect its mode of binding to DNA and its capacity to inhibit RecA (1). The primary mechanism of RdgC inhibition appears to involve a simple competition for DNA-binding sites, especially on double-stranded DNA (dsDNA) (1).
Deletion of the <span class="yellow">E. coli</span> rdgC gene alone causes no obvious phenotype but is highly deleterious in strains lacking certain enzymes involved in recombination and replication restart (2,3). RdgC is essential for the growth of an <span class="yellow">E. coli</span> strain lacking PriA, indicating that it might affect replication fork progression or fork rescue (2). PriA provides a means to load the DnaB replicative helicase at DNA replication fork and D loop structures (2). RdgC is, therefore, a key factor in the rescue of stalled or broken forks and subsequent replication restart. dnaC suppressors of PriA can overcome this inviability, especially when RecF, RecO or RecR is inactivated, which indicates that RdgC avoids or counters the toxic effect of these proteins by limiting inappropriate RecA loading on SSB-coated ssDNA (2).
The DNA-binding ability of the RdgC protein has been demonstrated clearly using electrophoretic mobility shift assays (2). <span class="blue">Escherichia coli</span> RdgC binds non-specifically to dsDNA and with higher affinity than ssDNA (2). RdgC from <span class="yellow">N. meningitidis</span> also binds DNA with little specificity for sequence or structure, like the <span class="yellow">E. coli</span> protein (4). RdgC exhibits no preference for DNA replication or recombination intermediates (2). However, no detectable domains or nuclease activity could be identified for the <span class="yellow">E. coli</span> RdgC protein (2,6). The complexes of <span class="yellow">E. coli</span> RdgC with both linear and supercoiled circular plasmid DNA were imaged using atomic force microscopy (7). RdgC was found to have an increased affinity to DNA ends and to promote bending of DNA (7). RdgC has the effect on DNA superstructure; the promotion of DNA condensation was observed at high protein concentrations (7). Recombination is largely enhanced by close contacts of distant regions along the DNA strands through condensation (7).
<span class="yellow">Pseudomonas aeruginosa</span> is a ubiquitous environmental Gram-negative bacterium that belongs to the γ subdivision of the Proteobacteria. Orthologs of the rdgC gene are found only in β and γ subdivisions of the Proteobacteria (2). The amino acid sequence of <span class="yellow">P. aeruginosa</span> RdgC, a 306-residue protein, shows 46 and 36% identities against those of <span class="yellow">E. coli</span> and <span class="yellow">N. meningitidis</span>, respectively. Despite the importance of DNA-binding ability of RdgC in its biological roles, the structural details of RdgC and the mechanism of its action remain unclear. Here, we report the crystal structure of <span class="yellow">P. aeruginosa</span> RdgC as determined by the multiwavelength anomalous diffraction (MAD) method of X-ray crystallography. It reveals that RdgC dimer is ring-shaped, with a central hole of ∼30 Å diameter. The inside surface around the central hole is rich in conserved, positively charged residues, which are implicated in DNA-binding by mutagenesis. We propose that dsDNA binds to RdgC through this central hole. Our structure provides a solid structural framework for a better understanding of the role of RdgC in homologous recombination.<br><br>MATERIALS AND METHODS
Protein expression and purification
The rdgC gene (PA3263) was amplified by the polymerase chain reaction (PCR) using the genomic DNA of <span class="yellow">P. aeruginosa</span> strain PAO1 as a template. The oligonucleotide primers designed using the published genome sequence (8) were 5′-G GAA TTC CAT ATG TGG TTC CGC AAT CTG CTC G-3′ (forward) and 5′-CCG CCG CTC GAG GAC GCC CTG GGG GAT TTC TTC-3′ (reverse). The bases in bold denote the NdeI and XhoI cleavage sites, respectively. The PCR product was digested with NdeI and XhoI, and was then inserted into the NdeI/XhoI-digested expression vector pET-21a(+) (Novagen). This vector construction adds an eight-residue tag (LEHHHHHH) to the carboxyl terminus of the gene product to facilitate protein purification.
The protein was expressed in <span class="yellow">E. coli</span> C41 (DE3) cells (9). The cells were grown at 37°C up to OD600 of 0.6 in Luria–Bertani medium that contained 50 μg ml−1 ampicillin, and the protein expression was induced by 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG). The cells were allowed to grow at 20°C for 22 h after IPTG induction and were harvested by centrifugation at 4200 g (6000 rev min−1; Sorvall GSA rotor) for 10 min at 4°C. The cell pellet was resuspended in an ice-cold lysis buffer [20 mM Tris-HCl (pH 7.9), 500 mM sodium chloride, 5 mM imidazole] and was then homogenized with an ultrasonic processor. The crude cell extract was centrifuged at 36 000 g (18 000 rev min−1; Hanil Supra 21 K rotor) for 60 min at 4°C, and the recombinant protein in the supernatant fraction was purified in three chromatographic steps.
The first purification step utilized the C-terminal hexahistidine-tag by Ni2+-chelated HiTrap chelating column (GE Healthcare). The eluent was diluted 3-fold with 50 mM Tris-HCl at pH 7.4. The diluted solution was applied to a 20-ml heparin-Sepharose column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol and 1 mM EDTA. The protein was eluted with a linear gradient of 0–2.0 M sodium chloride in the same buffer. The first peak corresponded to high molecular aggregates of RdgC and the second peak to RdgC dimers, which was also consistent with our dynamic light scattering measurements. When the aggregates were not separated from the dimers, the protein readily aggregated within a day. The dimer fractions were concentrated to ∼6 mg ml−1 concentration using an YM10 ultrafiltration membrane (Millipore-Amicon) and further purified by gel filtration on a HiLoad XK 16 Superdex 200 prep-grade column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 200 mM sodium chloride, 1 mM dithiothreitol and 1 mM EDTA. The first peak corresponding to the high molecular aggregates of RdgC was only a minor component, and the second peak corresponding to RdgC dimers was stable for several weeks when frozen at −70°C. The procedure for purifying the SeMet-substituted protein was the same, except for the presence of 10 mM dithiothreitol in all buffers used during purification steps. Dynamic light scattering experiments were performed at 24°C, with the protein (at 2 mg ml−1 concentration) dissolved in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride on a Model DynaPro-801 instrument (Wyatt, Santa Barbara, CA, USA). The protein concentration was estimated by measuring the absorbance at 280 nm, employing the calculated extinction coefficient of 20 910 M−1 cm−1 (SWISS-PROT; http://www.expasy.ch/).<br><br>Mutagenesis and electrophoretic mobility shift assay
Point mutants were prepared using the QuikChange Site-Directed Mutagenesis kit (Stratagene), and the mutations were confirmed by DNA sequencing. The seven mutants R4A, F120A, K146A, R198A, R208A, Q212A and R252A were well expressed and soluble. The three mutants K70A, R81A and R211A were not expressed, while R122A mutant was expressed but was not soluble. Therefore, we mutated Lys70, Arg81, Arg122 and Arg211 into aspartate. The K70D, R81D, R122D and R211D mutants were well expressed and soluble. The soluble mutants were expressed and purified under the conditions identical to those for the wild type, except that the heparin-Sepharose column step was omitted. All of the soluble mutants had similar elution profiles as the wild type upon gel filtration, and the dimer fractions were used for DNA-binding assays. We mixed 5 μl of RdgC (1.1 μg μl−1) and 5 μl of a linear dsDNA with 414 bp (63 ng μl−1), corresponding to the molar ratio of 64:1 for RdgC dimer to dsDNA. The reaction mixtures were incubated for 10 min at room temperature. RdgC–DNA complexes were analyzed by 1.5% (w/v) agarose gel electrophoresis in 1× TAE buffer (45 mM Tris-acetate at pH 8.3, 1 mM EDTA). DNA bands were visualized by ethidium bromide staining. For DNA-binding assays, the protein concentration was measured with a Bio-Rad protein assay kit.<br><br>Crystallization and X-ray data collection
Crystals were grown by the hanging-drop vapor diffusion method at 24°C by mixing equal volumes (2 μl each) of the protein solution (at 19 mg ml−1 concentration in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride) and the reservoir solution. To grow the native crystals, we used a reservoir solution consisting of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate and 1% (w/v) Anapoe®35 as the detergent. Native crystals grew to approximate dimensions of 0.2 mm × 0.1 mm × 0.1 mm within a few days. The SeMet-substituted protein was crystallized under conditions identical to those for the native crystals except for the presence of 10 mM dithiothreitol in the protein solution.
A crystal of the SeMet-substituted protein was dipped into a cryoprotectant solution for a few seconds and was flash-cooled in the cold nitrogen gas stream at 100 K. The cryoprotectant solution consisted of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate, 1% (w/v) Anapoe®35 as the detergent and 10% (v/v) glycerol. A set of Se MAD data was collected at 100 K at three different wavelengths using an Area Detector System Corporation Quantum 210 charge-coupled device detector at the beamline NW12A of Photon Factory, Tsukuba, Japan. The crystal was rotated through a total of 180° with a 1° oscillation range per frame. The raw data were processed and scaled using the program suit HKL2000 (10). The SeMet-substituted crystal belongs to the space group P21212, with unit cell parameters of a = 87.04 Å, b = 115.60 Å and c = 76.07 Å.
The native crystals were soaked for 10 min in 10 mM ethyl mercury thiosalicylate (EMTS) to collect a set of Hg MAD data at four different wavelengths. X-ray diffraction data from the EMTS-derivatized crystal, as well as a native crystal, were collected on a Bruker charge-coupled device detector at the beamline BL-6B of Pohang Light Source, Pohang, Korea. The EMTS-soaked crystal belongs to the space group P21212, with unit cell parameters of a = 87.16 Å, b = 116.02 Å and c = 74.97 Å. The native crystal belongs to the space group P21212, with unit cell parameters of a = 87.49 Å, b = 116.04 Å and c = 76.03 Å. If the presence of a dimeric molecule in the asymmetric unit is assumed, the calculated crystal volume per protein mass (VM) is 2.75 Å3 Da−1 and the solvent content is 55.3%. Supplementary Table 1 summarizes the statistics of X-ray diffraction data collection.<br><br>Structure solution and refinement
Heavy atom sites were located with the program SOLVE (11). Four Se sites from the SeMet-substituted crystal and two Hg sites from the EMTS-soaked crystal were located. Two sets of the initial Se and Hg MAD phases were separately improved using the program RESOLVE (12). Two electron density maps were independently interpreted, and the fragments of the two models were combined into a single model, since the two maps were complementary to each other. Non-crystallographic symmetry (NCS) matrices were found from a partial model built from the initial electron density map, and phases were further improved by the 2-fold NCS averaging, solvent flattening and histogram matching with the program DM (13). The model was built with O (14). The model was refined with the program CNS (15), including the bulk solvent correction. 10% of the data were randomly set aside as the test data for the calculation of Rfree (16). Several rounds of model building, simulated annealing, positional refinement and individual B-factor refinement were performed. Subsequently, this model was used to refine the structure of the native protein. The model has excellent stereochemistry, as evaluated by the program PROCHECK (17). Refinement statistics are summarized in Supplementary Table 1.<br><br>
RESULTS AND DISCUSSION
Model quality and monomer structure
The structure of <span class="yellow">P. aeruginosa</span> RdgC was solved using two sets of MAD data collected from a crystal of the selenomethionine (SeMet)-substituted protein and from a mercury derivative crystal of the native protein (Supplementary Table 1). The model has subsequently been refined using the 20.0–2.50 Å data from a native crystal to crystallographic Rwork and Rfree values of 23.4 and 27.9%, respectively, with no sigma cut-off. The refined model contains 612 residues of the two monomers in the asymmetric unit of the crystal (residues 1–306 for both chains A and B) and 87 water molecules. The C-terminal eight-residue fusion tag has no electron density in both subunits and has not been modeled. Conformations of the two monomers in the asymmetric unit are similar. The root-mean-square (r.m.s.) difference between the two monomers is 0.69 Å for 306 Cα atom pairs, with a maximum deviation of 8.8 Å for the Cα atom of Gly305. The residues giving r.m.s. differences larger than 2.0 Å are 14–16, 94–96, 201–202 and 303–306, which correspond to loops connecting secondary structure elements or the C-terminus. They are located on the surface of the dimer. Subunit A has a better map quality than subunit B, and thus has a lower mean B-factor than subunit B (40.3 Å2 versus. 56.1 Å2 for all 4788 atoms) (Supplementary Figure 1). Therefore, subunit A is chosen for discussions, unless otherwise stated.
The <span class="yellow">P. aeruginosa</span> RdgC monomer is J-shaped and has approximate dimensions of 72 Å × 60 Å × 40 Å (Figure 1). It can be divided into three structural domains: center domain, tip domain and base domain. Center domain (residues 1–73 and 121–167) includes a six-stranded anti-parallel β-sheet with two flanking α helices (α1 and α4). The helix α1 runs nearly parallel to the two outermost strands β2 and β3, which in turn lie adjacent to β4. Strands β2 and β3 are separated by a loop. The helix α4 is nearly orthogonal to the β-strands. Tip domain (residues 74–120) contains two α helices (α2 and α3) and is inserted between strands β4 and β5 of the center domain. It sticks out from the center domain. The two connections between the center domain and the tip domain are rich in conserved residues, hinting at an important functional role of these joints. They could act as a hinge for a possible conformational change. The C-terminal region of the polypeptide chain is folded into the base domain (residues 168–306), which consists of a five-stranded anti-parallel β-sheet with four α-helices, three of which (α5, α6 and α8) cover one side of the β-sheet. The long helix α8 is slightly bent.<br><br>
Structural similarity searches
When we searched for structural similarity against the Protein Data Bank database using the program DALI (18), the <span class="yellow">P. aeruginosa</span> RdgC monomer showed no significant similarity with a Z score above 5. Therefore, we conclude that the overall fold of the RdgC polypeptide chain is unique. When we elaborated the structural comparisons with individual domains of <span class="yellow">P. aeruginosa</span> RdgC, only the center domain gave a Z score above 5.
Using the center domain (residues 1–73 and 121–167) alone, the highest structural similarity is found with the <span class="yellow">human</span> β2-adaptin appendage domain (PDB code 1E42, Z score = 7.1, r.m.s. deviation = 2.5 Å for 85 structurally aligned residues, residues 1–12, 14–30, 44–47, 58–72, 122–133 and 135–160 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 838–841, 848–880, 885–906 and 912–967 of the <span class="yellow">human</span> β2-adaptin appendage domain chain A) (19). There does not appear to be any functional relatedness between these two proteins. The next highest similarity is found with the <span class="yellow">yeast</span> TATA-box-binding protein (PDB code 1YTB, Z score = 5.7, r.m.s. deviation = 2.6 Å for 79 structurally aligned residues, residues 4–12, 14–19, 21–27, 56–70, 124–132 and 134–166 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 69–96, 100–126 and 132–155 of TATA-box-binding protein, N-terminal half of chain A) (20). The TATA-box-binding protein is composed of a ten-stranded antiparallel β-sheet with four flanking α-helices on the convex side of the β-sheet (20). It has an internal quasi 2-fold symmetry, and its β-strands are involved in DNA binding. There is, however, no structural similarity of functional significance, because the two center domains of the RdgC dimer are well separated and do not associate into a single unit (Figure 1C and D).
Using the tip domain (residues 74–120) alone, the ATPase domain of <span class="yellow">bovine</span> Hsc70 chaperone shows highest structural similarity (PDB code 1BA1, Z score = 4.8, r.m.s. deviation = 1.7 Å for 45 structurally aligned residues, residues 76–120 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 230–253 and 256–276 of the Hsc70 ATPase domain) (21). There does not seem to be any significant functional relationship between these two proteins. The α-helix-loop-α-helix structure of the tip domain is reminiscent of the helix–hairpin–helix (HhH) DNA-binding motif that is present in a number of DNA repair enzymes (22–25). The HhH motifs are characterized by the conserved sequence motif hxxhxGhGxxxAxxxhh, where h is any hydrophobic residue (VILMWFYA) and x any residue (23). Two conserved glycines allow a tight turn between two α-helices. However, the tip domain of <span class="yellow">P. aeruginosa</span> RdgC protein does not have such a sequence motif, and the loop in the RdgC tip domain is much more extended than that of the HhH motifs. Therefore, the RdgC tip domain is uniquely folded.
With the base domain (residues 168–306) alone, the highest similarity is found with <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase (unpublished deposition; PDB code 2H9C, Z score = 4.0, r.m.s. deviation = 4.8 Å for 51 structurally aligned residues, residues 186–191, 237–240, 257–272 and 274–298 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 36–41, 50–53, 55–66 and 68–96 of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase chain A). There does not appear to be any functional relationship between the two proteins, and the base domain of RdgC is uniquely folded. The 101-residue chain of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase is folded into three α-helices, two of which overlap with two α-helices (α2 and α3) of the base domain of RdgC.<br><br>Ring-shaped dimer structure and inter-subunit interface
Our structure reveals that <span class="yellow">P. aeruginosa</span> RdgC forms a ring-shaped dimer of approximate 2-fold symmetry in the crystal (Figure 1). This observation is consistent with the results of previous gel filtration and sedimentation equilibrium experiments, which suggested the presence of dimeric species of <span class="yellow">E. coli</span> RdgC in solution (1,2). Depending on the concentration, RdgC could also exist in solution as monomers, tetramers and higher oligomers (1). In the crystal lattice, no monomeric, trimeric or tetrameric species exist. Our structure unequivocally rules out the monomer–trimer model, which fits the sedimentation equilibrium data approximately, as well as the monomer–dimer–tetramer model (1). The RdgC dimer has approximate dimensions of 80 Å × 60 Å × 40 Å, with a central hole of ∼30 Å diameter.
Only a moderate amount of surface area is buried upon the formation of the RdgC dimer. The monomer–monomer interface buries a solvent-accessible surface area of ∼1360 Å2 per monomer, corresponding to ∼8% of the total surface area of the monomer (Protein–Protein Interaction Server at http://www.biochem.ucl.ac.uk/bsm/PP/server/). Polar and nonpolar atoms in interface are 44 and 56%, respectively. The interface can be divided into two kinds: ‘interface A’ involving both the tip domains and center domains, and ‘interface B’ involving the base domains (Figure 2A). These two interfaces have significantly different characters.<br><br>The interface A is formed largely by the crossing over of the two tip domains in the dimer (Figure 1). This interface covers a surface area of ∼565 Å2 and is largely nonpolar, with the polar and nonpolar atoms in the interface being 37.5 and 2.5%, respectively. Phe120 contributes considerably to the nonpolar character of the interface between the tip domains, accounting for ∼150 Å2 of the interface area. Phe120 is part of a highly conserved sequence segment and lies at the boundary between the tip domain and the center domain. The aromatic ring of Phe120′ inserts into a hydrophobic pocket, which is formed by the hydrophobic residues Ile74, Leu75, Pro76, Val79, Leu115, Ala119 and Phe120 of the neighboring monomer (Figure 2B). The primed residue belongs to the adjacent subunit. These residues are well conserved in the RdgC family (indicated by green triangles below the aligned sequences in Figure 3). As these hydrophobic residues are located near the molecular 2-fold axis, the two clusters of hydrophobic side chains are merged into a single, large cluster in the dimer (Figure 2B). Of the 16 hydrogen bonds that exist between the two subunits, only one is made in this interface between Arg118 and Arg118′.<br><br>The interface B is contributed to exclusively by the two base domains. This interface covers a surface area of ∼795 Å2. The polar and nonpolar atoms in this interface are 50% each. The two β-sheets of the two base domains merge into a single ten-stranded antiparallel β-sheet in this interface (Figure 2A). Thus, 15 of 16 inter-subunit hydrogen bonds are clustered in this interface. The hydrogen bond networks can be divided into two types. The first type involves the main chain atoms of Val206, Arg208 and Lys210 in strand β9; this strand associates with the equivalent strand from the other subunit in an antiparallel manner (Figure 2C). The other type involves both the side chain and main chain atoms. The side chain of Arg211′ interacts with the backbone of Gly204, while the side chain of Lys227 hydrogen bonds to those of both Gln212′ and Glu218 (Figure 2D). In addition, the side chain of Gln212′ interacts with that of Asp199 and the backbone of Arg211′ (Figure 2D). Arg211, Gln212, Glu218 and Lys227 are highly conserved among the RdgC family members. The residues participating in the hydrogen bond networks are indicated by red circles below the aligned sequences in Figure 3. The observation that conserved residues play important roles in forming the dimer implies that dimerization of RdgC is critical for its function.
If one or both of interfaces A and B are to be broken for functional reasons such as for allowing dsDNA to enter into the central hole of the RdgC dimer, we can conceive of three scenarios. The first possibility would be a simultaneous breakage of the two interfaces A and B. The second would be the sequential disruption of the interfaces A and B. The third would be the disruption of only one of the two interfaces A and B. Sedimentation equilibrium indicated the presence of monomers in solution (1), which is consistent with the first two scenarios. If the third scenario holds, RdgC dimers could possibly exist in two conformations in solution, i.e. a closed form as observed in the present crystal structure, and an open form, in which one of the two interfaces A and B is broken.<br><br>A model for DNA binding by RdgC and DNA-binding assay
The net charge of <span class="yellow">P. aeruginosa</span> RdgC is highly negative, since each monomer contains 12 aspartate, 13 glutamate, 10 lysine, 9 arginine and 2 histidine residues. The net charge of a dimer would be −12 (or −8), if we assume none (or all) of histidines to be positively charged. The electrostatic potential at the molecular surface of RdgC dimer shows that the charge distribution on the protein surface is highly nonuniform (Figure 4A). Since the inner surface of the RdgC dimer is rich in conserved, positively charged residues and the diameter of the central hole (∼30 Å) is approximately similar to that of other toroidal DNA-binding proteins (26), it appears likely that RdgC dimers bind dsDNA through the central hole. Despite extensive co-crystallization and soaking trials with linear dsDNAs of different lengths (14-mer, 16-mer and 18-mer), we could not locate the bound DNA experimentally. We attribute this difficulty to the nature of sequence-nonspecific DNA binding by RdgC. Therefore, we built a crude model of a DNA complex of the RdgC dimer by fitting dsDNA into the central hole (Figure 4B). This model is supported by the DNA-binding assays coupled with mutagenesis, as described below. It is also consistent with the suggested binding site size of ∼16 nt for the <span class="yellow">E. coli</span> RdgC dimer (1) and the approximate thickness (∼40 Å) of the <span class="yellow">P. aeruginosa</span> RdgC dimer.<br><br>When we performed electrophoretic mobility shift assay for dsDNA binding, <span class="yellow">P. aeruginosa</span> RdgC did not require Mg2+ ions (Supplementary Figure 2), unlike <span class="yellow">Deinococcus radiodurans</span> RecR, which required high concentrations of Mg2+ ions (27). ATP was not required for dsDNA binding by <span class="yellow">P. aeruginosa</span> RdgC, either (Supplementary Figure 2). This is consistent with an apparent lack of an ATP-binding pocket in the RdgC dimer structure.
In order to identify the residues that are important for dsDNA binding, we carried out mutational analyses. The residues for mutagenesis were selected on the basis of the modeled complex between RdgC dimer and dsDNA, together with sequence conservation (Figure 4C). We prepared eleven point mutants (R4A, K70D, R81D, F120A, R122D, K146A, R198A, R208A, R211D, Q212A and R252A). These eleven mutated residues are well conserved among RdgC proteins. Eight mutants (R4A, K70D, R81D, R122D, K146A, R198A, R208A and R252A) had a lower affinity for dsDNA than the wild-type RdgC (Figure 4D). These eight mutated residues are indicated by blue squares below the aligned sequences in Figure 3. This result indicates that dsDNA binds to the inside surface around the central hole of the RdgC dimer, since the mutated residues are located on the inner surface of the RdgC dimer. It is worth mentioning that a single mutation introduces two alterations per RdgC dimer.
The three mutants (F120A, R211D and Q212A) were prepared to investigate whether dimerization of RdgC is crucial for dsDNA binding. Phe120 is a key residue located in the inter-subunit interface A (Figure 2B), while Arg211 makes hydrogen bonds in the interface B (Figure 2D) and Gln212 is involved in both intra- and inter-subunit interactions (Figure 2D). dsDNA binding by the F120A mutant is considerably weakened, compared to either the R211D or Q212A mutant (Figure 4D). This result suggests that destabilization of the ring-shaped architecture of the RdgC dimer is deleterious for tight dsDNA binding and that interface A is probably more important than interface B for dsDNA binding. dsDNA-binding affinity of the Q212A mutant was lower than the wild type (Figure 4D), whereas that of the R211D mutant was comparable to the wild type (Figure 4D). This result suggests that disruption of the inter-subunit interface B is also detrimental for dsDNA binding but to a smaller extent than disruption of the interface A. It further suggests that the side chain of Arg211 is not involved in recognizing dsDNA. It is interesting that the DNA complex for wild type forms a distinct band (lane 3 in Figure 4D), as opposed to a more diffuse band as one would expect if the complexes contained different numbers of RdgC dimers. Assuming a binding site size of ∼16 nt (1), the 414-bp duplex DNA would bind ∼25 RdgC dimers if it is fully saturated.<br><br>RdgC is unique among ring-shaped DNA-binding proteins
<span class="yellow">Pseudomonas aeruginosa</span> RdgC adds a unique member to the repertoire of toroidal DNA-binding proteins (26) such as <span class="yellow">D. radiodurans</span> RecR (27) and DNA polymerase processivity factors, including the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29) and PCNA of eukaryotic DNA polymerase δ (30,31) (Figure 5). These proteins do not have an overall sequence similarity among themselves, and their quaternary structures also vary. <span class="yellow">P. aeruginosa</span> RdgC and <span class="yellow">E. coli</span> β clamp are homodimers, T4 gp45 and eukaryotic PCNAs are homotrimers and <span class="yellow">D. radiodurans</span> RecR is a homotetramer.<br><br>In <span class="yellow">P. aeruginosa</span> RdgC, the inner surface of the ring-shaped dimer is largely formed by β-strands, while the outer surface of the ring is mostly formed by α-helices (Figure 5). The predicted DNA-binding site of <span class="yellow">P. aeruginosa</span> RdgC dimer is primarily formed by β-strands and loops. In contrast, in other ring-shaped DNA-binding proteins such as the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29), PCNA of eukaryotic DNA polymerases δ (30,31) and <span class="yellow">D. radiodurans</span> RecR (27), α-helices are located toward the center of the ring and are surrounded by β-strands in the outermost side of the ring (Figure 5). Despite a superficial resemblance of the overall ring-shaped architecture of the RdgC dimer with those of other toroidal DNA-binding proteins, the internal arrangement of the secondary structure elements in the RdgC dimer is strikingly different from those of other toroidal DNA-binding proteins. Thus, RdgC is unique among ring-shaped DNA-binding proteins.
A number of other proteins involved in DNA metabolism also adopt a ring-shaped structure despite diversity in the molecular functions (26). They include NAD+-dependent DNA ligase (25), λ exonuclease (32), hexameric helicases, topoisomerases, the trp RNA-binding attenuation protein, the <span class="yellow">bacteriophage</span> head-to-tail connector and translin (26). There are also many examples of the proteins in DNA recombination pathways that assemble into multi-subunit toroids containing a central channel (26). Electron microscopy showed that the <span class="yellow">E. coli</span> RuvB protein, in the presence of ATP, forms a dodecamer on double-stranded DNA in which two stacked hexameric rings encircle the DNA and are oriented in opposite directions with D6 symmetry (33). The <span class="yellow">human</span> RAD52 protein was found to assemble into heptameric rings with a large funnel-shaped channel, 40–60 Å in diameter (34). The <span class="yellow">human</span> DMC1 protein has so far been detected only as an octameric ring (35).<br><br>Inhibition mechanism of RecA function by RdgC
RdgC inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). It was observed that the effects of RdgC on RecA-promoted DNA strand exchange were much more pronounced than the effects of RdgC on other RecA functions, when RdgC was added to the reaction late (1). By examining the effects of the DinI protein on RdgC-mediated inhibition of RecA activities, it was concluded that the potent effect of RdgC in the inhibition of strand exchange did not primarily reflect a displacement of RecA in the nucleoprotein filament by RdgC, but instead a binding of RdgC to the dsDNA substrate so as to make it unavailable to the filaments for strand exchange (1).
Our work contributes to a better understanding of the inhibition mechanism of RecA function by RdgC. The ring-shaped architecture with a central hole that is suitable for the binding of dsDNA lends RdgC with a capacity to hold dsDNA very tightly. Therefore, we suggest that RdgC dimers probably function as a tight dsDNA gripper and physically block access to the target dsDNA by RecA filaments, because tight encircling of dsDNA by RdgC dimers, possibly as a linear aggregate, would prevent the two strands from dissociating. However, it remains unclear how the closed-ring structure of the RdgC dimer might allow binding of a large number of RdgC dimers on the duplex DNA. It may be conceivable that each RdgC dimer can open temporarily at one or both of its two inter-subunit interfaces so that dsDNA can enter the central hole.
RdgC was at its highest level during exponential phase, reaching its maximum of ∼1000 dimers per <span class="yellow">E. coli</span> cell. Its level decreased sharply to ∼50 dimers per cell in stationary phase (2). Under normal cellular conditions, RecA is maintained at ∼1000 molecules per cell (http://www.els.net/). Following LexA repressor cleavage, the level of RecA protein in the cell increases by as much as 20-fold. RdgC inhibition of the RecA function by a simple competition mechanism at the cellular concentration of RdgC clearly requires a much higher affinity for DNA by RdgC than RecA. Possible protein–protein interactions between RdgC dimers might have a profound effect on the affinity for DNA, when a large number of RdgC dimers are bound on a duplex DNA. If only one of the two inter-subunit interfaces can open up and the 2-fold symmetry axes of the RdgC dimers are misaligned sufficiently, the escape of DNA from a linear array of interacting RdgC dimers would be extremely difficult. Alternatively, both inter-subunit interfaces of each dimer may open up simultaneously. A line of indirect evidence for the possible protein–protein interactions between RdgC dimers is provided by the cooperative binding of RdgC to DNA (7). Cooperativity of RdgC binding may be caused by local and distant protein–protein interactions (7). A kind of side-by-side interaction between two adjacent molecules of RdgC dimer is present in the crystal, burying an accessible surface area of ∼750 Å2 at the interface. The interaction involves mostly the less polar side surface of the dimer (left side of Figure 4A), with 2-fold symmetry axes of the two interacting RdgC molecules making an angle of ∼90° when viewed down the central holes and ∼15° when viewed perpendicular to the holes. This may explain the observed DNA bending by RdgC, which was speculated to be a necessary step in the working mechanism of RdgC, if one can assume that the strong protein–protein interactions in the crystal were relevant for protein–protein interactions in solution (7).
RdgC protein was shown to have a potent effect on RecA protein-promoted DNA strand exchange in <span class="yellow">E. coli</span> (1). In a series of experiments, RdgC was added after RecA filaments had formed on the ssDNA and 5 min after the linear dsDNA was added. With 3 μM RecA, a sharp reduction in DNA strand exchange products was seen with 0.4 μM RdgC, and the generation of products was abolished at 0.8 μM RdgC (1). This suggests that the binding affinity of RdgC to dsDNA is considerably higher than that of RecA. The high affinity of RdgC could be explained by its ring shape and the suggested mode of DNA binding. Tetrameric species of <span class="yellow">E. coli</span> RdgC, which started to form at around 2 µM concentrations, were speculated to be the species with the most potent inhibitory effect, possibly by having a higher affinity for DNA (1). Higher affinity for DNA of the tetrameric species could be understood, if the formation of a tetramer in solution involves a protein–protein interaction similar to the side-by-side interaction between two adjacent RdgC dimers in the crystal, although there is no experimental evidence. Obviously, there are still many open questions that need to be addressed in future biochemical experiments.<br><br>
SUMMARY
The crystal structure of RdgC, a recombination-associated DNA-binding protein, determined in this study reveals that the polypeptide chain is uniquely folded and two monomers associate to form a ring-shaped dimer with a central hole of ∼30 Å diameter. The inner surface around the central hole is rich in positively charged residues. These structural features suggest that dsDNA runs through the central hole when it binds to RdgC dimers. This idea is supported by our mutational studies, which indicate that the conserved, positively charged residues around the central hole are important for dsDNA binding. The toroidal architecture of the RdgC dimer provides a sound framework for a better understanding of its role in homologous recombination.
Coordinates
Atomic coordinates and the structure factor data have been deposited in the Protein Data Bank (accession code 2OWY).<br><br>
SUPPLEMENTARY DATA
Supplementary Data are available at NAR Online.<br><br>Supplementary Material<br><br>
</body></html>