
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>flies</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="red">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA1634875</h3>Molecular polymorphism, differentiation and introgression in the period gene between <span class="yellow">Lutzomyia intermedia</span> and <span class="yellow">Lutzomyia whitmani</span>
Abstract
Background
<span class="yellow">Lutzomyia intermedia</span> and <span class="yellow">Lutzomyia whitmani</span> (Diptera: Psychodidae) are important and very closely related vector species of cutaneous leishmaniasis in Brazil, which are distinguishable by a few morphological differences. There is evidence of mitochondrial introgression between the two species but it is not clear whether gene flow also occurs in nuclear genes.<br><br>Results
We analyzed the molecular variation within the clock gene period (per) of these two species in five different localities in Eastern Brazil. AMOVA and Fst estimates showed no evidence for geographical differentiation within species. On the other hand, the values were highly significant for both analyses between species. The two species show no fixed differences and a higher number of shared polymorphisms compared to exclusive mutations. In addition, some haplotypes that are "typical" of one species were found in some individuals of the other species suggesting either the persistence of old polymorphisms or the occurrence of introgression. Two tests of gene flow, one based on linkage disequilibrium and a MCMC analysis based on coalescence, suggest that the two species might be exchanging alleles at the per locus.<br><br>Conclusion
Introgression might be occurring between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> in period, a gene controlling behavioral rhythms in <span class="yellow">Drosophila</span>. This result raises the question of whether similar phenomena are occurring at other loci controlling important aspects of behavior and vectorial capacity.<br><br><br><br>Background
The Phlebotominae sand <span class="red">flies</span> <span class="yellow">Lutzomyia intermedia</span> Lutz & Neiva 1912 and <span class="yellow">Lutzomyia whitmani</span> Antunes & Coutinho 1912 are vectors of cutaneous leishmaniasis in Brazil. These are closely related species that can be only distinguished by a few morphological differences [1] and both show high anthropophily and reported natural infections with Leishmania in different regions of Brazil [2].
Despite their importance as vectors, only a handful of studies have been carried out in these two species using molecular techniques [3-6]. One of the most important findings from an epidemiological perspective is the evidence obtained for introgression between the two species using mitochondrial DNA [4]. This was particularly interesting because apparently, only lineages of <span class="yellow">L. whitmani</span> sympatric with <span class="yellow">L. intermedia</span> have been involved in cutaneous leishmaniasis transmission in the peridomestic environment [4], which suggests that genes controlling aspects of vectorial capacity could be passing from one species to the other. In fact, mitochondrial introgression has been reported in other sand <span class="yellow">fly</span> species [7,8] suggesting that might be a common phenomenon in these insect vectors. However, because mitochondrial genes can introgress relatively easily between closely related species [9], it becomes important to examine whether introgression can occur with nuclear genes.
The <span class="yellow">Drosophila</span> period (per) gene homologue was isolated in sand <span class="red">flies</span> by Peixoto et al. [10]. This circadian clock gene was originally identified using mutagenesis by Konopka and Benzer [11], but is also known to control the differences in the "lovesong" rhythms between <span class="yellow">D. melanogaster</span> and <span class="yellow">D. simulans</span> [12], that are important to the sexual isolation between these two species [13-15]. In addition, per was implicated in the control of species-specific circadian mating rhythms in <span class="yellow">Drosophila</span> and Bractocera, which might also constitute a reproductive isolation mechanism [16-18]. Thus per may possibly represent an example of a <span class="yellow">Drosophila</span> speciation gene [19], and in fact it has been used as a molecular marker in a number of speciation and evolutionary studies, not only in <span class="yellow">Drosophila</span> (reviewed in [20]) but also in other insects (e.g. [21]) including sand <span class="red">flies</span> [22-24].
Because per controls the circadian clock in different insects [25], it is almost certainly involved in the rhythms of activity and biting of sand <span class="red">flies</span> [26], which are very important to leishmaniasis transmission. In addition, per might be involved in reproductive isolation in sand <span class="red">flies</span>, via mating rhythms, or via their "lovesongs" [2,27]. per is thus a particularly interesting marker, among the few available, for an introgression analysis in <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. Evidence for introgression in per might suggest that gene flow between these two vector species is occurring at other genes controlling important aspects of behavior and vectorial capacity. It might also suggest that per does not have a strong role in their reproductive isolation. In the current study, we analyzed the molecular variation within the per gene of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> in five different localities in Eastern Brazil.<br><br>Results
Polymorphism and divergence between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>
A total of 68 sequences from <span class="yellow">L. intermedia</span> and 53 from <span class="yellow">L. whitmani</span> homologue to a fragment of the period gene were analyzed from populations of five localities in Eastern Brazil (Fig 1). The alignment of 72 variable sites is shown in Fig 2. Although most of the changes are either synonymous or occur within the 58 bp intron, non-synonymous substitutions are observed causing 9 amino acid differences among the sequences (Fig 2).
Table 1 shows the number of sequences of each population of the two species, the number of polymorphic sites (S) and the estimates of molecular polymorphism θ (based on the total number of mutations) and π. Table 1 also shows the Tajima's [28] and Fu & Li's [29] statistics. Within each species, all populations present similar levels of polymorphism with the exception of <span class="yellow">L. whitmani</span> from Ilhéus, which seems to be less polymorphic than the others. This population was also the only one presenting a significant value in the Fu & Li test but only at the 5% level. Finally, the last column of Table 1 presents the recombination estimator γ [30] indicating that both species show evidence of intragenic recombination in the per gene.
To investigate the level of intra and interspecific differences, initially an AMOVA was carried out as shown in Table 2. The results show a non-significant within species and a significant between species molecular variation at the per locus. Table 3 shows a more detailed analysis of the intraspecific differentiation among populations of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. None of the pairwise and overall fixation indexes (Fst) are significant in the case of <span class="yellow">L. intermedia</span> and only one (Posse × Ilhéus) has a borderline significant value in <span class="yellow">L. whitmani</span>. The results therefore show that no significant geographical heterogeneity was detected among the populations of the two species. The estimated number of migrants per generation, based on the overall Fst values, is 20.683 for <span class="yellow">L. intermedia</span> and 23.125 for <span class="yellow">L. whitmani</span>.
Table 4 shows measures for DNA divergence between species (Dxy and Da), as well as the Fst and Nm values considering each species as a unique population. Dxy is the average number of nucleotide substitutions per site between alleles from two different populations and Da is the number of net nucleotide substitutions between two populations. Table 4 also shows the number of polymorphisms exclusive for each species (Sint and Swhit), the number of shared polymorphisms (Ss) and the number of fixed differences (Sf) between species. As one can note, there is a high number of shared polymorphisms between species, and no fixed differences between them suggesting either the persistence of ancestral polymorphisms or the occurrence of introgression. In fact, there is one shared haplotype between the two species (IPO13, WPO10 and WPO19) and three <span class="yellow">L. whitmani</span> sequences (WAC02, WPO13 and WPO14) which show only one nucleotide difference to "typical" <span class="yellow">L. intermedia</span> haplotypes (see also below).<br><br>Genealogy of period sequences
A phylogenetic analysis of the period gene sequences from <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> was carried out with the Minimum Evolution method using the Kimura 2-parameter distance (Fig 3). A sequence from <span class="yellow">L. umbratilis</span>, a related species from the same subgenus Nyssomyia, was used as outgroup [24]. The tree shows <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> as non-monophyletic. However, despite the low bootstrap values, which are below 50% in most cases, there is a large group that contains most <span class="yellow">L. intermedia</span> sequences and a second large group with most <span class="yellow">L. whitmani</span> sequences. A few other sequences are clustered outside these two main groups. It is interesting to note that there are three <span class="yellow">L. whitmani</span> alleles (WAC2, WPO13 and WPO14) inside <span class="yellow">L. intermedia</span> main group, as well as one <span class="yellow">L. intermedia</span> allele (ICP16) inside the <span class="yellow">L. whitmani</span> main group. In addition, a second <span class="yellow">L. intermedia</span> allele (IPO13) is a shared haplotype between the two species as mentioned above. Again, the results suggest either the persistence of ancestral polymorphisms or the occurrence of introgression between the two species. Very similar results were obtained using the maximum likelihood algorithm as implemented in PAUP 4.0b10 software [31] (data not shown).
As mentioned before, there is evidence of intragenic recombination in the per gene fragment of both species (see Table 1) and for that reason the bifurcating tree shown in Fig 3 has to be viewed with caution, as different regions of the gene might have different phylogenetic histories [32]. Therefore, we constructed Minimum Evolution trees with the two most polymorphic non-recombining blocks of the per gene fragment identified using the Hudson and Kaplan [33] method available in the DNAsp 4.1 program [34]. We did not observed major changes in the genealogy of the <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> per sequences, especially regarding the five haplotypes (ICP16, IPO13, WAC2, WPO13 and WPO14) that clearly cluster with sequences of the other species (data not shown).
Finally, a haplotype network was estimated from per sequences using statistical parsimony, as described by Templeton et al. [35] and implemented in the TCS1.21 software [36] (Fig 4). A small number of ambiguities were resolved as suggested by Crandall and Templeton [37]. The haplotype network shows connections between sequences from each species, separating most of the sequences of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> in two groups. No intraspecific geographical structuring was found. Once again, some of the <span class="yellow">L. whitmani</span> sequences (WAC2, WAC10, WPO13 and WPO14) appear more closely related to <span class="yellow">L. intermedia</span> haplotypes. In addition, one <span class="yellow">L. intermedia</span> allele (ICP16) is connected by a small number of mutations to some of the main <span class="yellow">L. whitmani</span> haplotypes and IPO13 is a shared haplotype between the two species. These results confirm the same putative introgressed sequences indicated by the phylogenetic reconstructions.<br><br>LD test of introgression
We tested the hypothesis of gene flow between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> using a method based on linkage disequilibrium (LD) developed by Machado et al. [38]. In this test, x is the difference between the average LD found among all pairs of shared polymorphisms (DSS) between the two species and the average LD among all pairs of sites for which one member is a shared polymorphism and the other is an exclusive polymorphism (DSX). In case of gene flow x should tend to be positive [see [38] for more details].
Because of limitations on the total number of sequences that could be handled by the WH program we could not perform the simulations with all sequences. Therefore, we carried out the LD test of introgression between each pair of sympatric populations of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> from the localities of Posse, Afonso Claudio and Corte de Pedra. The input files were prepared using the values of recombination and linkage disequilibrium calculated by the SITES program [30] for each population (data not shown). Although no significant values were found for the smaller samples of Afonso Claudio and Corte de Pedra, the results (Table 5) present evidence for introgression in the period gene in both directions (from <span class="yellow">L. intermedia</span> to <span class="yellow">L. whitmani</span> and vice-versa) in the locality of Posse.<br><br>Isolation with Migration model
To further examine the gene flow between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> we used the IM software [39]. The Isolation with Migration model has six demographic parameters that include two migration rates, one for each population. The IM software estimates the posterior probability for each of the model parameters, fitting the Isolation with Migration model to the data. One of the assumptions of this model is that the loci studied do not have internal recombination. Therefore, we identified four different non-recombining blocks of our fragment of per, which were then treated as different loci in the analysis. The four-gametes test [33] implemented in DnaSP4.1 was used for the identification of possible recombination events. Since the program estimates parameters for a pair of closely related populations or species, all sequences of each species were used in the analysis as a single population. We performed MCMC runs using the IM software with different seed numbers, in order to guarantee convergence of the sample.
Maximum likelihood estimates of migration parameters revealed a non-zero value for both species, m1 = 1.398 and m2 = 1.014 (m1 – from <span class="yellow">L. whitmani</span> towards <span class="yellow">L. intermedia</span>; m2 – from <span class="yellow">L. whitmani</span> towards <span class="yellow">L. intermedia</span>). Fig 5 shows the posterior distributions for migration rates and reveals a null probability for the absence of migration from <span class="yellow">L. whitmani</span> towards <span class="yellow">L. intermedia</span>. In addition, the absence of migration in the opposite direction is not included in the 95% confidence interval (values range from 0.222 to 8.898), thus supporting the presence of migration in both directions. The conversion of the migration rate estimate to population migration rate per generation (m1 and m2) is not accurate when the population size is based on a single locus. However, the average of the migrant number per generation for both species was very close to the Nm estimate based on Fst values (Nm ~0.49 in Table 4, m1 ~0.52 and m2 ~0.34).<br><br>
Discussion
There is some evidence that <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> might represent sibling-species complexes in Brazil. <span class="yellow">Lutzomyia neivai</span> Pinto 1926, a sibling of <span class="yellow">L. intermedia</span> is found in parts of Southern and Western Brazil and some other countries of South America [40]. The present study did not include populations of this species. In the case of <span class="yellow">L. whitmani</span>, mitochondrial data [3,6] indicates three main lineages in Brazil: an Amazonian group, a North-South group and a Northeast group. We did not find strong evidence of a geographical differentiation in the period gene among populations of <span class="yellow">L. whitmani</span> although one of the pairwise Fst comparisons (Posse × Ilhéus) was significant at the 5% level.
When we compare <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>, we find a highly significant Fst value (0.3373), which is however smaller than that observed for the period gene between sympatric siblings of <span class="yellow">Lutzomyia longipalpis</span> (Fst = 0.3952) [23], a complex of cryptic species that are vectors of American visceral leishmaniasis. Therefore, despite the presence of diagnostic morphological characters to identify <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> [1] the level of molecular divergence in period is not as high as the cryptic <span class="yellow">L. longipalpis</span> siblings.
Even though it is hard to distinguish introgression from the persistence of ancestral polymorphisms, a test of gene flow based on the signature introgression leaves on the patterns of linkage disequilibrium [38] as well as simulations that fit the "Isolation with Migration" model to the data suggest that <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> might be exchanging alleles at the per locus. This is further supported by the presence of shared haplotypes between the two species in Posse and very similar sequences in all sympatric populations. There is mounting evidence that introgression plays a major role in the evolution of closely related insect vector species. Introgression among vectors may have important epidemiological consequences. Gene flow in loci that affect vectorial capacity, such as those controlling host preference and susceptibility to parasite infection, can change the transmission patterns and consequently make the disease control a harder task. Introgression of genes that control adaptation to particular types of environment can also have a major impact on the spread of vector-borne diseases as was proposed for the major African malaria vector <span class="yellow">Anopheles gambiae</span> [41]. The same can be said about genes controlling insecticide resistance. For example, Weill et al. [42] found a kdr mutation responsible for pyrethroid resistance in the Mopti form of <span class="yellow">Anopheles gambiae</span>, a normally susceptible taxon of this species complex. Sequence analysis reveals that this resistant allele probably originates through introgression from the Savanna form.
Although <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> are closely related and only distinguished by a few morphological differences, they do show differentiation in some other important traits. For example, in Posse, one of the localities we studied, the two species show differences in abundance during the year. <span class="yellow">L. intermedia</span> is more abundant in the summer while <span class="yellow">L. whitmani</span> is more frequent in the winter months [2]. They also show differences in microhabitat preferences, <span class="yellow">L. intermedia</span> being more common in the peridomestic area while <span class="yellow">L. whitmani</span> is found mainly in the surrounding forest [2]. In addition, the two species show marked differences in their tendencies to bite <span class="yellow">humans</span> in the early morning, with <span class="yellow">L. whitmani</span> showing higher feeding rates than <span class="yellow">L. intermedia</span> [26]. Therefore, despite the evidence of introgression in the period gene in this locality, there are important ecological and behavioral differences between the two species in Posse suggesting that gene flow is probably rather limited in loci controlling these traits. Hence, it is yet not clear whether introgression has played an important role in the evolution of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. Further work with other genes might help clarify the issue.<br><br>Conclusion
Evidence for introgression between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> obtained using mitochondrial DNA [4] seems to be corroborated by our data on the period gene, a nuclear marker. Nevertheless, considering that period is potentially involved in reproductive isolation and might be, therefore, less prone to introgression than the "average" gene [43], it is possible that much higher levels of gene flow between the two species occur at other genes. It might, on the other hand, suggest that this behavioral gene, or at least the fragment we analyzed, did not play a role in speciation between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. In fact the same has been suggested for some <span class="yellow">Drosophila</span> species [44] despite per's role controlling lovesong and mating rhythm differences between <span class="yellow">D. melanogaster</span> and <span class="yellow">D. simulans</span> [13-16].
Although the evidence for introgression in the per gene between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> is not overwhelming, it does indicate the need to extend this analysis to other loci in the future. We are currently isolating new molecular markers in the two species to carry out a multi-locus approach [39] that might help determining how much variation in gene flow and differentiation there is across the genome of these two very important leishmaniasis vectors.<br><br>Methods
Sand <span class="yellow">fly</span> samples
Sand <span class="yellow">fly</span> samples used in this work were all the F1 generation from wild collected females from the Brazilian localities of Posse (Petrópolis, Rio de Janeiro State, 22°30'S 43°10'W), Jacarepaguá (Rio de Janeiro, Rio de Janeiro State, 22°55'S 43°21'W), Afonso Claudio (Espírito Santo State, 20°04'S 41°07'W), Corte de Pedra (Presidente Tancredo Neves, Bahia State, 13°27'S 39°25'W) and Ilhéus (Bahia State, 14°50'S 39°06'W). <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> were identified according to Young and Duncan [1]. The progeny of each wild caught female was raised separately according to Souza et al. [45] and only one F1 male of each female was used for the molecular analysis, which included 68 individuals of <span class="yellow">L. intermedia</span> (12 from Afonso Claudio, 18 from Posse, 20 from Corte de Pedra and 18 from Jacarepaguá) and 51 individuals of <span class="yellow">L. whitmani</span> (12 from Afonso Claudio, 17 from Posse, 3 from Corte de Pedra and 19 from Ilhéus). Note that, although the distribution of the two species shows considerable overlap in Eastern Brazil, in many localities only one species is found or is far more abundant than the other. There are also seasonal and microhabitat differences in abundance between them in areas of sympatry [2].<br><br>DNA methods
Genomic DNA was prepared according to Jowett [46] with slight modifications and the PCR was carried out for 30 cycles at 95°C for 30 sec, 60°C for 30 sec and 72°C for 30 sec, using Abgene, Amersham Biosciences or Biotools reagents according to manufacturers directions. The per primer sequences are: 5llper2: 5'-AGCATCCTTTTGTAGCAAAC-3' (forward) and 3llper2: 5'-TCAGATGAACTCTTGCTGTC-3' (reverse). These primers amplify a 486 bp fragment of the sand <span class="yellow">fly</span> per gene homologue that includes part of the PAS/CLD domain, an intron (58 bp) and the beginning of the perS domain [24]. The amplified fragments were cloned using the pMOSBlue blunt ended cloning kit (Amersham Biosciences) and plasmid DNA preparation was carried out using the "Flexiprep" Kit (Amersham Biosciences). Cloned PCR fragments were sequenced at Fundação Oswaldo Cruz and at University of Leicester using ABI 377 sequencers. With the exception of two <span class="yellow">L. whitmani</span> individuals from Corte de Pedra (see below), only one sequence of each sand <span class="yellow">fly</span> (representing one of the two possible alleles) was used in the analysis but an average of three sequences per individual were obtained in order to check possible PCR induced mutations. In addition, PCR fragments were also sequenced directly in some cases for the same reason. In the case of the two <span class="yellow">L. whitmani</span> mentioned above 6 and 9 clones were sequenced respectively from specimens WCP01 and WCP03 to determine both alleles simply to increase the size of this small sample.
Negative controls were performed for all amplification reactions. In addition, PCR, cloning and sequencing were repeated for two individuals to confirm putative introgressed sequences and to exclude the possibility that they were the result of PCR contamination. Finally, for at least two individuals with putative introgressed sequences, we could define the other allele from additional clones (not included in the analysis), which showed to be typical of the species, indicating no identification problems.
The sequences were submitted to GenBank (accession numbers AY927062 to AY927182).<br><br>Sequence analyses
The preliminary sequence editing was carried out using the Wisconsin Package Version 9.1, Genetics Computer Group (GCG), Madison, and ClustalX [47] was used to perform the multiple alignment. Analyses of population polymorphisms and differentiation between populations were carried out using DNAsp4.1 [34] and ProSeq [48] softwares, while Arlequin v. 2.0 [49] was used for an analysis of molecular variance (AMOVA) between populations. The Minimum Evolution phylogenetic tree was constructed using MEGA 3.1 software [50]. The haplotype network was estimated using TCS1.21 [36]. Recombination and linkage disequilibrium analyses were performed using the DNAsp4.1 and SITES program [30]. Linkage disequilibrium simulations were carried out by the WH program [51,52] and Markov Chain Monte Carlo (MCMC) simulations of the isolation with migration model were performed using the algorithm implemented in the IM program [39].<br><br>
Authors' contributions
CJM generate and analyzed all the data and drafted the manuscript. NAS and CAC collected and maintained sand <span class="yellow">fly</span> samples. CPK helped to write the manuscript and supervised CJM during her stay in Leicester. AAP is the principal investigator, participated in its design and coordination, and helped to write the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2636784</h3>Tissue remodeling: a mating-induced differentiation program for the <span class="yellow">Drosophila</span> oviduct
Abstract
Background
In both vertebrates and invertebrates, the oviduct is an epithelial tube surrounded by visceral muscles that serves as a conduit for gamete transport between the ovary and uterus. While <span class="yellow">Drosophila</span> is a model system for tubular organ development, few studies have addressed the development of the <span class="yellow">fly</span>'s oviduct. Recent studies in <span class="yellow">Drosophila</span> have identified mating-responsive genes and proteins whose levels in the oviduct are altered by mating. Since many of these molecules (e.g. Muscle LIM protein 84B, Coracle, Neuroglian) have known roles in the differentiation of muscle and epithelia of other organs, mating may trigger similar differentiation events in the oviduct. This led us to hypothesize that mating mediates the last stages of oviduct differentiation in which organ-specific specializations arise.<br><br>Results
Using electron- and confocal-microscopy we identified tissue-wide post-mating changes in the oviduct including differentiation of cellular junctions, remodeling of extracellular matrix, increased myofibril formation, and increased innervation. Analysis of once- and twice-mated females reveals that some mating-responsive proteins respond only to the first mating, while others respond to both matings.<br><br>Conclusion
We uncovered ultrastructural changes in the mated oviduct that are consistent with the roles that mating-responsive proteins play in muscle and epithelial differentiation elsewhere. This suggests that mating triggers the late differentiation of the oviduct. Furthermore, we suggest that mating-responsive proteins that respond only to the first mating are involved in the final maturation of the oviduct while proteins that remain responsive to later matings are also involved in maintenance and ongoing function of the oviduct. Taken together, our results establish the oviduct as an attractive system to address mechanisms that regulate the late stages of differentiation and maintenance of a tubular organ.<br><br><br><br>Background
Most internal organs, including the vascular and respiratory systems and the gastro-intestinal and urinary-genital tracts are comprised of a single epithelial tube or a network of tubes. Tubular organs serve as conduits for the transport of gases, liquids, or solutes, and serve as barriers between biological compartments. To create tubes with specific flow and barrier properties, the morphology of the tube must be precisely specified during development and modulated by physiology. To accommodate specific physiological roles, tissue-specific programs for differentiation are employed at the last stages of development. While much is known about the molecular and cellular basis of tube formation [1-6], little is known about the mechanisms that regulate the late stages of differentiation in which organ-specific specializations arise.
The conservation of genes and similarity in tubular organ design across taxa make <span class="yellow">Drosophila</span> an excellent model for understanding organogenesis in higher animals. In <span class="yellow">Drosophila</span>, the best understood tubular organs from a developmental point of view are the trachea and salivary gland. Studies of these organs reveal a general program for tubular organ development, in which combinatorial expression of global patterning genes specifies positions within the embryo for the subsequent activation of tissue-specific early genes and transcription factors. This program results in the activation of downstream genes involved in terminal differentiation of organ-specific specializations such as the cuticle that lines the tracheal lumen [1,2,4,7-9].
The <span class="yellow">Drosophila</span> female reproductive tract is another tubular system, consisting of the uterus and a common oviduct (the main tube) that branches into two lateral oviducts. Regional differences in function are observed along the length of the tract, with egg activation occurring largely at the proximal end, in the lateral oviducts, and fertilization occurring at the distal end, in the uterus [10,11]. Unlike other tubular organs, little is known about the development of the female reproductive tract. However, regional differences in function suggest the presence of region-specific differentiation programs within the female reproductive tract.
In <span class="yellow">Drosophila</span>, mating induces changes in female behavior and physiology via molecules transmitted in the seminal fluid. These changes are rapid and lead to a mated female state which is profoundly different from the unmated female state. While an unmated female lays few eggs and readily accepts the courtship efforts of a male, a mated female exhibits increased egg-laying and actively rejects males [12-19]. Microarray studies of whole <span class="blue">flies</span> reveal that the changes in egg-laying rate are accompanied by a change in gene expression. Within three hours of mating there is an increase in expression of a small number of genes [20,21]. Rapid changes in gene expression, as well as protein abundance, have also been observed in the female reproductive tract [22,23].
In the upper reproductive tract (lateral and common oviducts, hereafter, oviduct), mating induces an increase in immune related transcripts and down regulates transcription factors involved in cell growth and differentiation. At the protein level mating induces increased abundance of proteins associated with muscle assembly and function and cytoskeletal proteins associated with epithelial morphogenesis [23]. Since many of these mating-responsive proteins act in late differentiation pathways of muscle and epithelia elsewhere (e.g. Bent, Muscle LIM protein 84B(Mlp84B), Neuroglian (Nrg), Coracle (Cora)), we hypothesize that mating triggers similar differentiation in the oviduct. To test our hypothesis we characterized the ultrastructure of oviduct epithelia and muscle, as well as the pattern of innervation before and after mating. We then examined the effect of different mating regimes on oviduct mating-responsive cytoskeletal proteins and on female reproductive output. Our results suggest that active tissue remodeling takes place in the oviduct epithelia and musculature in response to mating. Furthermore, we found a striking increase in innervation of the oviduct after mating.
Our results show that the reproductive tract is an attractive system to address mechanisms that regulate the late stages of tissue differentiation in a tubular organ. Unlike other tubular organs, the last differentiation stage of the oviduct is triggered by an extrinsic cue (mating). This makes it possible to experimentally control the onset of differentiation, with an opportunity to independently examine the effects of mating and age. In addition, it allows us to examine processes essential for reproduction.<br><br>Results
Mating induces changes in oviduct lumen
Our previous molecular profiling showed that mating promotes changes in actin-based cytoskeletal molecules and suggests that mating triggers molecular changes and tissue remodeling in the female reproductive tract that mediate its progression to a mature functional stage [23]. To gain insight into the mechanisms that underlie this progression, we used light and electron microscopy to determine the morphological status of the oviduct in unmated and mated 3-day old females.
In nearly all mated reproductive tracts processed for microscopy (8/9), an egg was located in one of the lateral oviducts, whereas an egg was never observed in the oviduct of unmated reproductive tracts (5/5) (Figure 1). This observation is consistent with previous studies that report increased ovulation and egg-laying at 6 h post-mating [24]. In all unmated reproductive tracts examined, the region between the lateral oviducts and the middle of the common oviduct was either tapered or constricted, whereas this region appeared relaxed in the mated reproductive tract (Figure 1A and 1D). These observations raise the possibility that the lumen is narrow in the unmated oviduct and larger in the mated oviduct. To address this possibility, we collected serial 1 μm longitudinal sections through the reproductive tracts of unmated and mated females and stained these sections with toluidine blue to survey the appearance of the lumen along the entire length of the oviduct (Figure 1B and 1E). Our examination reveals that, in unmated reproductive tracts, the lateral oviduct lumen has an irregular shape, while the common oviduct lumen appears straight. Moreover, in all the unmated reproductive tracts sectioned, we detected patches of darkly stained material in the lumen of the lower common oviduct. In the mated reproductive tract, the lumen of the upper oviduct (defined as the lateral oviduct and upper part of the common oviduct) has an irregular shape, and the lumen of the lower oviduct (defined as the lower part of common oviduct) appears straight. In addition, the lumen of the lower oviduct appears wider in mated than unmated reproductive tracts (Figure 1C and 1F). Interestingly, darkly stained material was not detected in the oviduct lumen of mated females. This observation appears to be consistent with the description made by Mahowald et al. [25], who reported that the oviduct lumen of unmated females is nearly filled with an intima-like matrix and that this matrix is reduced after mating.
Because 3-day-old mated females lay eggs, it is unclear whether the lack of lumenal material and increase in lumen size in mated females occurred before or after the passage of eggs. We suggest that mating directly or indirectly induces morphological changes in the oviduct that facilitate egg passage through the duct. Taken together, our observations lead us to propose that the oviduct lumen is closed and/or obstructed in the unmated reproductive tract, and that mating induces changes in the epithelia and/or muscle that "open" the oviduct lumen.<br><br>Initial formation of cell-cell junctions in oviduct epithelia is mating-independent
To determine whether mating induces specific morphological changes in the oviduct epithelia post-mating, we next examined the ultrastructure of the oviduct epithelia in unmated and mated reproductive tracts. Since molecular profiling demonstrates that proteins associated with cellular junctions such as α- and β-Spectrin (Spec), Cora, and Nrg [23] increase post-mating, we first determined the status of the cellular junctions in the oviduct epithelia of unmated females, and whether these junctions change post-mating. In <span class="yellow">Drosophila</span>, most ectodermally derived epithelia (such as the epidermis and trachea), with a few exceptions, are joined apically by a belt-like adherens junction called the zonal adherens junction (ZA) followed basally by a septate junction (SJ) [26]. Our analysis reveals that the oviduct is lined, along its entire length, by a monolayered epithelium comprised of squamous-type cells. Although region-specific differences in morphology were observed, all oviduct epithelia examined, in both unmated and mated females, are joined along their lateral membranes by an extensive SJ and lack an apical ZA (Figure 2). SJs and ZAs form complete belts that surround the epithelial cell, thus making these junctions easily visible in transverse sections through the epithelium. Because ZAs were not detected in our transverse sections through the oviduct, this implies that ZAs never formed, or developed earlier and were lost (Figure 2D). Interestingly, we did not detect any ultrastructural differences in the SJs at 6 h post-mating, but we did uncover differences in SJ ultrastructure in different regions of the oviduct.
Based on their ultrastructure, two types of SJs, smooth and pleated, can be distinguished in <span class="yellow">Drosophila</span> [26]. Smooth SJs are distinguished by the lack of visible septae and the appearance of electron dense material in the intercellular space, while pleated SJs are distinguished by the ladder-like appearance of septae. In the lateral oviducts and upper common oviduct, septa were not detected in the SJ, thus these SJs represent smooth SJs or an immature stage of pleated SJ (Figure 2A, 2A', 2A"). In contrast, a ladder-like arrangement of septae was often visible in the SJs of the lower common oviduct (Figure 2C'), thus these SJs can be classified as pleated. Unlike the smooth-like SJs of the upper oviduct, the pleated SJs of the lower oviduct are followed basally by spot type adherens junction (spot AJs) (Figure 2B, 2B'; additional file 1). Further analysis is necessary to determine if the SJs of the upper and lower oviduct represent different types or different developmental stages. Our findings demonstrate that the initial formation of SJs, as well as spot AJs in the lower oviduct, are mating-independent. This raises an interesting question. Why are SJ proteins such as Cora and Nrg up-regulated post-mating if SJs are formed prior to mating? It is possible that the increased expression of SJ proteins is associated with functional changes in polarized secretion post-mating. Recent studies have shown that SJs play an unexpected role in regulating the apical secretion of specialized extracellular matrix molecules in the trachea [27,28], and that these molecules are important regulators of lumen size.<br><br>Mating modulates apical secretory activity in the oviduct
Given the presence of extensive SJs in the oviduct and the role of SJs in regulating apical secretion of extracellular matrix molecules in other epithelia (e.g. trachea), we asked if mating modulates apical secretion in the oviduct epithelia. Our ultrastructural analysis reveals that different regions of the oviduct display different apical membrane morphology (i.e microvilli or pleats) (see Figures 2 and additional file 2A and 2A), but all epithelia are covered by an electron dense apical extracellular matrix (AECM) and a thin layer of cuticle. We found that mating induces ultrastructural changes in the AECM and cuticle in both the upper and lower oviduct. In the upper oviduct of the unmated female, the AECM varies in thickness along the apical surface (Figure 3A). Some areas have little AECM, while other areas are covered by a distinct layer of AECM (~1–2 μm in thickness; Figure 3B). However, the AECM of mated females is more evenly distributed along the apical surface, (~2 μm in thickness; Figure 3E). Strikingly, the AECM and cuticle of mated females have a ruffled appearance, suggesting that the AECM and cuticle have increased in surface area (Figure 3F). Electron dense granules up to ~1.5 μm in diameter were occasionally observed in both the AECM and cell cytoplasm (Figure 3F). Although further analysis is needed to determine the role of these granules in the oviduct epithelia, it is possible that these granules participate in the secretion and deposition of the AECM. Taken together, our findings suggest that polarized secretion via the AECM, while ongoing in the upper oviduct of the unmated female is enhanced and/or modulated post-mating.
Post-mating changes in AECM ultrastructure are also observed in the lower common oviduct. However, unlike the AECM of the upper oviduct, the AECM of the lower oviduct is well developed prior to mating. In the lower oviduct of the unmated female, the AECM consists of an amorphous electron dense material and is unevenly distributed, forming a thick, bulbous layer above the plasma membrane in some regions (additional file 2). Matrix-like material is also observed in the lumen, but this material is more electron dense than the AECM (additional file 2A and 2A). In the mated female, the AECM is flattened against the plasma membrane and is uniformly distributed along the apical surface (additional file 2B and 2B). Matrix-like material was not observed in the center of the lumen, but small pools of very electron dense material were detected in the spaces between the epithelial folds (additional file 2C and 2C). This may explain why lumenal matrix was not detected at the light microscopic level in the mated female oviduct (see Figure 1E and 1F). Taken together, our observations suggest that the lower common oviduct is a site of active apical secretion in both mated and unmated females, and that matrix secretion, particularly in the lumen, is reduced post-mating. These findings raise the intriguing possibility that the AECM and lumenal matrix function as a plug in the lower oviduct, and that mating induces the breakdown of this plug.<br><br>Mating induces changes in hemi-adherens junctions in upper oviduct
In addition to modulating secretion at the apical membrane, mating induces changes at the basolateral membrane. In many epithelia, one of the last steps of differentiation is the development of a layer of extracellular matrix (ECM) called the basal lamina that covers the apical and/or basal membranes and the concomitant development of hemi-adherens junctions (HAJs). HAJs connect the cell cytoskeleton with the ECM and are formed at virtually all cell surfaces that contact an ECM. HAJs can be distinguished at the ultrastructural level as a patch-like, electron dense undercoat of the plasma membrane that opposes the basal lamina ([26]; Figure 3G, 3H). In the <span class="yellow">Drosophila</span> embryo, the HAJs and basal lamina are formed at the same time [26]. Because the basal lamina is established at a time when the majority of extracellular matrix molecules are actively secreted [26,29], this suggests that the formation of HAJs is tightly coordinated with the secretion of the ECM.
One of the most striking post-mating changes observed in the oviduct epithelia was the appearance of numerous HAJs along the basolateral membrane in the upper oviduct (Figure 3). The importance of HAJs, particularly in the upper oviduct, is underscored by the extensive infolding of the basolateral membrane that is observed in both unmated and mated females (Figure 3A and 3E). The infolded membrane gives rise to a highly branched intercellular space that is filled with an ECM (Figure 3C and 3G). This ECM is contiguous with the basal lamina that surrounds the epithelia. Few HAJs were observed in the upper oviduct of the unmated female, and these were largely restricted to the basal membrane, and not observed along the basolateral infolding (Figure 3C). In contrast, numerous HAJs appear along the basolateral infolding post-mating in this region of the oviduct (Figure 3F–3H). In addition, the intercellular space appears wider post-mating (Figure 3C–3D and 3G–3H), suggesting that mating induces increased secretion and/or deposition of the ECM in this cellular compartment and brings the ECM to a threshold concentration that can support the development of HAJs. HAJs were also detected along the basal membrane, but they were not detected along the apical membrane even though this membrane was covered by an ECM. Interestingly, while the basolateral membrane forms very shallow folds in the lower oviduct (see additional file 2), HAJs were observed along this membrane in unmated reproductive tracts (data not shown), thus suggesting that the epithelia is more differentiated in this region of the oviduct, and that the differentiation of the upper and lower oviduct may be under different control.<br><br>Muscle differentiation is enhanced post-mating
While we uncovered post-mating changes in the oviduct epithelia that might facilitate its transition to a high egg-laying state, this transition may also be mediated by changes in oviduct muscle properties and/or activity. The oviduct is lined by circular muscle fibers with supercontractile characteristics [30]. Our previous studies showed that mef2 and mlp84B genes that regulate muscle differentiation, are expressed and increased post-mating in the oviduct, as well as in the sperm storage regions of the reproductive tract [22,23]. This suggests that mating induces muscle differentiation in the reproductive tract. Muscle differentiation is characterized by the assembly of myofilaments into bundles called myofibrils. As muscles differentiate, myofibrils and z-bodies appear simultaneously, and increase in number until the cytoplasm is filled with myofibrils [31]. Like epithelia, one of the last steps of muscle differentiation is the secretion of a basal lamina that surrounds the muscle fiber. To determine if mating induces structural changes in muscles (such as increased myofibrils) we examined the ultrastructure of muscle fibers in the upper and lower parts of the oviduct. Our analysis revealed that, in both unmated and mated reproductive tracts, the muscles of the lower oviduct are highly differentiated as evidenced by the high density of myofibrils, well developed and aligned z-bodies, and secretion of a thick, electron dense basal lamina (Figure 4E and 4F). In contrast, muscle fibers in the lateral oviducts and upper common oviduct appear less differentiated than muscles in the lower common oviduct, as evidenced by the lesser density of myofibrils and z-bodies, and little or no basal lamina (Figure 4A and 4B). Moreover, the muscles of the upper oviduct appear more differentiated in the mated than in unmated reproductive tracts (Figure 4A–4D). Interestingly, we observed neighboring muscle fibers in different states of differentiation in the lateral oviducts in both unmated and mated reproductive tracts, but not elsewhere in the oviduct (Figure 4B). These results suggest that mating enhances the rate of muscle differentiation in the upper oviduct, and that muscle differentiation is delayed in the upper as compared to the lower oviduct. The increased muscle differentiation in the upper oviduct is not dramatic and likely reflects the short post-mating period examined in this study. The delayed differentiation of the upper oviduct muscles resembles the delay in the onset of development between the adult thoracic muscles and abdominal muscles during metamorphosis [32]. Since the ovaries and the other parts of the reproductive tract are known to have different segmental origins [33], we hypothesize that different parts of the oviduct develop at different rates or begin development at different times.<br><br>Increased innervation in the oviduct post-mating
Nerve-muscle interactions play an important role in regulating adult muscle development and refining the final pattern of innervation [34,35]. Given that oviduct muscle differentiation is enhanced post-mating, we predicted that mating either directly or indirectly induces changes in innervation. To address this prediction, we quantified the number of nerve terminals or boutons innervating the lateral oviducts and common oviduct in unmated and mated reproductive tracts. Studies of oviduct innervation in <span class="yellow">Drosophila</span> reveal that the <span class="yellow">fly</span>'s oviduct receives aminergic, peptidergic and glutamatergic input [30,36-39]. In both larval and adult <span class="yellow">Drosophila</span>, different types of boutons are formed by neurons that express different neurotransmitters and modulators [40-42]. By similarity to the boutons described at the larval and adult neuromuscular junction, Middleton et al. [30] report that the <span class="yellow">fly</span>'s oviduct is innervated by glutamatergic type I boutons and tyraminergic/octopaminergic type II boutons. Rodgriguez-Valentin et al. [43] further report that the oviduct type II boutons co-express octopamine and glutamate. The neurons that give rise to the type I innervation have not been identified. However, it is well established that type II innervation arises from octopaminergic neurons located in the abdominal ganglion [30,43]. In addition, it has been shown that some or all of these neurons express a GAL4 insertion line for the bullwinkle (bwk) gene [43]. Bwk encodes a HMG-box containing putative transcription factor [44]. To determine if mating induces any changes in the number of type I and II boutons innervating the oviduct muscles, we used the pan-neural marker, anti-HRP to label all oviduct boutons in unmated and mated females. To distinguish between type I and II boutons we used an antibody against the Disc Large (DLG) protein [45]. Type I boutons were identified by their DLG postsynaptic staining and large size (> 8 μm in diameter) (Figure 4G), while type II boutons were distinguished by their absence of DLG staining and smaller size (< 2 μm) (Figure 4H). We find that type I and II boutons innervate the lateral oviducts and common oviduct, and that the type I innervation is restricted to a few axons that run parallel to the length of the oviduct, while the type II innervation is more widespread. We quantified the number of boutons in the lateral oviducts and common oviduct and observed a 74% increase in bouton number in the lateral oviduct and a 66% increase in the common oviduct post-mating (Figure 4I). More over, we observed no significant change in the number of type I boutons in the lateral oviduct and common oviduct. However, we detected a 1.5-fold increase in the number of type II boutons in the lateral oviduct and a 1.8-fold increase in type II innervation in the common oviduct. Dramatic increases in bouton growth are also observed during development. For example, a ten-fold increase in bouton number is observed at the neuromuscular junction during the larval period [46]. To determine if the increase in type II innervation was specific to mating or reflected normal growth in 3 day-old females, we quantified type I and II innervation in the oviducts of 5 day-old unmated females. We found no significant difference in type I and II innervation in unmated 3 day-old and 5-day-old females, indicating that mating, either directly or indirectly, induces a dramatic increase in type II innervation (Figure 4I). To determine if the post-mating increase in innervation is unique to the oviduct, we asked if mating induces a global change in innervation. We quantified bouton number in the adult ventral midline muscles of the 5th abdominal segment. These muscles are innervated by boutons that increase in number during metamorphosis [47]. No significant difference in bouton number was detected at these muscles in unmated and mated females (additional file 3). Though further analysis is needed, this suggests that the post-mating increase in innervation is oviduct-specific. Because the type II boutons are octopaminergic, the increased type II innervation may result in increased octopamine (OA) release in the oviduct. In support of this possibility, we have preliminary evidence that OA is released in the oviduct post-mating (Heifetz and Wolfner, in preparation). Studies in locust and <span class="yellow">Drosophila</span> demonstrate that OA inhibits oviduct contraction, while glutamate activates oviduct contraction [30,43,48]. In <span class="yellow">Drosophila</span>, electrical stimulation of the posterior abdominal nerve gives rise to a series of muscle contractions in the oviduct followed by a period of muscle fatigue or relaxation [43]. This pattern of muscle contraction and relaxation may facilitate the proper movement of the egg through the oviduct. In their study of bwk expressing neurons that innervate the oviduct, Rodriguez-Valentin et al. [43] show that OA and glutamate interact to produce the pattern of oviduct contraction and relaxation described above. It is therefore possible that the post-mating increase in type II innervation in the oviduct plays an important role in the increased rate of ovulation and egg-laying observed post-mating.<br><br>Female mating history affects the enrichment of cytosekeletal proteins in the oviduct
To gain insights into the role of cytoskeletal protein enrichment ([23]; additional file 4) in mediating the morphological changes detected in this study, we examined the effect of different mating regimes on cytoskeletal protein abundance. We focused on a subset of mating-responsive cytoskeletal proteins with well established roles in the differentiation of muscle and epithelia. These include: (i) Mlp84B which regulates the late differentiation pathway of muscle [49]; (ii) Cora and Nrg which are required for the formation of septate junctions in epithelia [50], and (iii) Hu-li tai shao (Hts), also known as adducin-like protein, which functions in assembly of the cytoskeletal network. Na+ pump α subunit (ATPα), another protein associated with septate junctions in epithelia, is not a mating-responsive protein and was used as a control. Using western blots, we first determined the abundance of the cytoskeletal proteins in oviducts of 3-day-old unmated and mated females at 6 hrs post-mating. We confirmed the proteomic results of Kalpenikov et al. [23] and found that mating increases the abundance of all proteins, except ATPα in mated oviducts relative to their abundance in unmated oviducts (Figure 5A).
To determine whether the increased abundance of mating-responsive proteins persists for longer times post-mating, we examined oviducts of 10-day-old females that mated once at 3 days of age, and calculated the abundance of the mating-responsive proteins relative to their level in oviducts of 3-day-old unmated females. We found no change or a slight increase in the relative abundance of all cytoskeletal proteins except Mlp84B at 7 days post-mating (Figure 5A). Strikingly, the level of Mlp84B declines by 7 days post-mating to the level observed prior to mating. Thus Mlp84B levels rise and fall after mating, while the epithelial-related proteins rapidly rise and are maintained at a high level after mating. This raises the possibility that a second mating might trigger an increase in Mlp84B protein expression as observed in 3-day-old females at 6 h post-mating. To test this possibility, females were mated twice (once at day 3, and once on day 10 of age), and their oviducts were examined at 6 hrs after the second mating. We calculated the abundance of the cytoskeletal proteins in the twice mated oviducts relative to their abundance in oviducts of 3-day-old unmated females. Our results show that a second mating has little or no effect on Mlp84B abundance. Thus, Mlp84B may represent a class of mating-responsive proteins that is only needed after the first mating. Interestingly, the effect of the second mating on the epithelial-related mating-responsive proteins appears to be different for each protein. While Cora levels drop to the level observed in 3-day-old unmated females, Nrg and Hts are maintained at a high level.
To determine if the changes in cytoskeletal protein abundance are mating-dependent we measured their abundance in the oviducts of unmated 5- and 10-day-old females. We calculated their abundance relative to their level in oviducts of unmated 3-day-old females (Figure 5B). We rationalized that if the change in cytoskeletal protein abundance is mating-dependent we will not see similar changes in unmated females. We observed a slow increase in the relative abundance of all mating-responsive proteins with time post-eclosion (Figure 5B). Because unmated females lay more eggs as they age (see additional file 5C) one possible interpretation of the increased level of cytoskeletal proteins in unmated females is that these proteins are associated with an intrinsic program for oviduct maturation and that mating accelerates this process to maximize egg-laying efficacy. Alternatively, it is possible that the slow increase in protein abundance observed in unmated females is due to the passage of eggs through the oviduct.
Taken together, our results suggest that mating is essential to fine-tune the levels of the mating-responsive proteins examined in this study. Because the changes in cytoskeletal protein abundance are different in unmated and mated females, this suggests that the post-mating changes are mating-dependent. Furthermore, we suggest that these post-mating changes are linked to changes in oviduct function.<br><br>Early or prior mating increases fecundity
In <span class="yellow">Drosophila</span>, female fecundity decreases with age [51-54]. It has been proposed that this decrease is due, in part, to the loss of germline and somatic stem cells [55]. Since the expression of the oviduct cytoskeletal proteins examined in this study change with age and mating experience, the state of the oviduct may also play a role in fecundity. To separate the effects of age and mating, we measured the fecundity of females that mated twice, first at 3 days post-eclosion and again at 10 days, and compared that to the fecundity of females that mated once at 3 days and females that mated once at 10 days. Fecundity was measured as the number of eggs laid per day per female during the first three days after mating. Once-mated 3-day-old females laid nearly twice as many eggs as once-mated 10-day-old females during the three days examined (24.5 ± 0.7 versus 13.3 ± 0.9, p < 0.0001). Twice-mated 10-day-old females also laid about 50% more eggs than once-mated females of the same age (19.3 ± 0.9 versus 13.3 ± 0.9, p < 0. 0001), but about 20% fewer than laid by once-mated 3-day-old females during the three days examined (19.3 ± 0.9 versus 24.5 ± 0.7, p < 0.0001) (Figure 6A, see also additional file 5A and 5B). Thus, the difference in fecundity between once-mated 10-day-old and once-mated 3-day-old females is not a result of age alone. Rather the main determinant of fecundity at 10 days is whether there had been a prior mating at 3 days. We also calculated the fertility (number of adults eclosed) of once- and twice-mated females. Once-mated 3-day-old females are more fertile than once-mated 10-day-old females (69.5 ± 1.6% versus 56.1 ± 3.0%, p < 0.0001) and slightly more fertile than twice-mated 10-day-old females (69.5 ± 1.6% versus 62.7 ± 2.4%, p < 0.015) (Figure 6B, see also additional file 5D). Because there is no significant difference in fertility between once-mated and twice-mated 10-day-old females, this suggests that (1) a prior mating has no significant effect on the fertility of 10-day-old mated females and (2) fertility decreases with age. One intriguing interpretation of these results is that an early mating increases fecundity which partially compensates for the age-related decrease in fertility. Thus, cytoskeletal mating-responsive protein changes may be associated with structural changes in the oviduct that result in increased fecundity. This may counteract the effects of decreased fertility on the reproductive output.<br><br>
Discussion
Despite the use of <span class="yellow">Drosophila</span> as a model system for organ-level biology and the emerging parallels between mammalian and <span class="yellow">Drosophila</span> reproductive biology [56], this is the first integrative tissue-wide study of post-mating changes in the <span class="yellow">Drosophila</span> oviduct. Our results provide several lines of evidence at the molecular, morphological and physiological levels suggesting that mating induces tissue-wide differentiation in the oviduct. Moreover, we identify ultrastructural changes in the mated oviduct that are consistent with the roles that some of the mating-responsive proteins examined in this study (e.g. Mlp84B, Cora, Nrg) are reported to play in muscle and epithelial differentiation elsewhere. For example, the increased abundance of Mlp84B, a major regulator of the late differentiation pathway of muscle [49] is consistent with the increased muscle differentiation in the upper oviduct post-mating (Figure 4A–4F). Similarly, the increased abundance of Cora and Nrg, molecules that are essential for SJ development and function [50] is consistent with the observation that SJs in the upper oviduct are immature and/or that mating induces changes in the apical extracellular matrix whose secretion may be regulated, in part, by SJs as occurs in the trachea [27,28]. Other post-mating changes that indicate that mating induces tissue-wide differentiation include increased HAJs along the basolateral membrane and increased innervation.
Analysis of protein abundance following different mating regimes (unmated, once-mated, twice-mated) gave us further insights into the possible roles that mating-responsive proteins play in the oviduct. For example, Mlp84B is only responsive to the first mating, while the epithelial proteins examined (Cora, Nrg and Hts) are responsive to the first and second mating. Furthermore, the response to the second mating is different from the response to the first mating. Taken together, these results suggest that Mlp84B is required for the final maturation of the oviduct, while the epithelial proteins examined are required for both the maturation and maintenance of the oviduct at a high functional state. Moreover, the post-mating pattern of Mlp84B supports the idea that the first mating induces the final maturation of the oviduct. The rise and fall of Mlp84B abundance after the first mating (Figure 5) parallels the expression pattern of Mlp84B during development where peaks in Mlp84B transcription occur during periods of embryogenesis and metamorphosis when muscle is differentiating [49].
Using different mating regimes we tested whether the first/early mating is essential for maintenance of high reproductive output in the second mating. Our results suggest that mating at an early age is essential to achieve maximum reproductive output (i.e. high fecundity and fertility). Since the first mating increases reproductive output (evidence from our mating regime experiments), it is likely that the ultrastructural changes detected in mated 3-day-old females lead to a highly functional oviduct.
We suggest that the final maturation of the oviduct includes a mating-dependent stage. We propose that during the first few days post-eclosion, the oviduct undergoes the first phase of differentiation, after which the oviduct is developmentally poised for a rapid response to an extrinsic cue (mating). Mating then triggers the second phase of maturation (tissue remodeling and modulation) which is essential for proper oviduct function (Figure 7). We further propose that the second phase of maturation consists of processes that are mating-independent and -dependent, and that both of these pathways are essential to produce a functional oviduct. For example, initial formation of SJs occurs prior to mating (mating-independent) while the increased apical secretion and development of HAJs are mating-dependent. The oviduct musculature is an example where both mating-independent and -dependent processes play a role. Muscles are highly differentiated in the lower oviduct prior to mating, while muscle differentiation is ongoing in the upper oviduct and increases after mating. Although the onset of muscle differentiation in both regions is mating-independent, the further differentiation of muscle in the upper oviduct is mating-dependent. Another possible interpretation of the role of mating on oviduct maturation is that mating accelerates and synchronizes processes that are essential for the functional maturation of the oviduct. It will therefore be interesting to examine the oviducts of older females to determine the status of oviduct maturation.
What is the benefit of mating-induced differentiation of the oviduct tissues? Unmated females are capable of laying eggs, albeit at a reduced rate as compared to mated females of the same age. One possible interpretation is that reproduction is energetically costly, thus delaying oviduct maturation until sperm is available is advantageous to the female. This may reflect the evolution of a mechanism to optimize reproductive capacity in early adulthood in short-lived animals.
In summary, we have identified events at the cellular, molecular and physiological levels that are part of an efficient and specific program for reproduction. <span class="yellow">Drosophila</span> affords us the opportunity to uncover the signaling pathways that coordinate these events to produce a physiologically functional organ.<br><br>Methods
Flies
Wild-type Canton-S <span class="blue">flies</span> were used for the fecundity/fertility experiments and confocal analysis. Wild-type Canton-S5 [57] <span class="blue">flies</span> were used for electron microscopy. All <span class="blue">flies</span> were kept in a 12 hrs light/dark cycle at 23 ± 2°C. Upon eclosion, females and males were collected on ice and held separately until 3 (females and males) or 10 (females) days of age.<br><br>Sample preparation
For all assays (unless described differently) unmated females were placed with 3-day-old unmated males and observed until mating initiated. At the end of mating, females were aspirated into fresh vials and held for 6 hrs. At 6 hrs after the start of mating, females were placed on ice for dissection. Previous molecular studies indicate changes in protein abundance at 3 hrs post-mating. We hypothesize that these molecular changes will translate into morphological changes in the next few hours. We chose to analyze the morphology of mated females at 6 h post-mating as opposed to later times post-mating because the changes observed at later times may be due, in part, to the high rate of eggs passing through the oviduct.<br><br>Electron microscopy
Reproductive tracts were dissected in Schneider's <span class="yellow">Drosophila</span> medium (Sigma) on ice and processed for electron microscopy as described in [42]. Tracts were flat-embedded between two sheets of Aclar (Electron Microscopy Sciences), which allowed us to image the entire tract at the light microscopic level prior to sectioning. Sections were cut on a Reichart Ultracut microtome. One-μm thick sections were stained with 1% toluidine blue, and viewed with a Zeiss Axoplan microscope. Ultrathin sections (~100 nm) were mounted on formvar grids, stained with lead citrate, and viewed with a Philips/FEI Morgagni 268 TEM at 80 kV. Our analysis is based on 4 unmated samples and 3 mated samples. Two unmated samples were cut in the longitudinal plane and two additional unmated samples were cut in the transverse plane, while one mated sample was cut in the longitudinal plane and two additional mated samples were cut in the transverse plane. For longitudinal sections, the entire tract was re-embedded and cut. For samples cut in the transverse plane, the flat-embedded reproductive tract was divided into three regions: (1) lateral oviducts and upper common oviducts, (2) middle common oviduct, and (3) lower common oviduct. Each region was re-embedded and sectioned. It is beyond the scope of this paper to describe all three regions, and our analysis focuses on the uppermost and lowermost regions.<br><br>Immunocytochemistry
Reproductive tracts were dissected in Yamamoto's Ringer (10 mM MOPS; 80 mM NaCl; 10 mM KCL; 0.2 mM MgCl2; 0.1 mM CaCl2) with 5% (w/v) sucrose on ice, fixed in 4% paraphormaldehyde in PBS (phosphate-buffered saline; 0.85% NaCl, 1.4 mM KH2 PO4, 8 mM Na2 HPO4, pH 7.4) for 45 min and then washed in PBS. The reproductive tracts were then incubated in blocking solution (0.5% Triton x-100, 3% NGS, 0.1% BSA) for 2 hrs at room temperature. The following primary antibodies, reagents, and dilutions were used: Cy3-conjugated <span class="yellow">goat</span> anti-HRP, 1:200 (Jackson Immunochemicals, West Grove, PA); <span class="yellow">mouse</span> anti-Disc Large (DLG), 1:1000 (Developmental Hybridoma Bank), Alexa Fluor 488-phalloidin, 1:200 (Invitrogen, Molecular Probes, Scotland). Secondary antibodies were Alexa Flour 488-conjugated <span class="yellow">Goat</span> anti-<span class="yellow">mouse</span>, 1:200 and Alexa Flour 546-conjugated <span class="yellow">Goat</span> anti-<span class="yellow">rabbit</span>, 1:200 (Invitrogen, Molecular Probes, Scotland). Reproductive tracts were incubated with the different primary antibodies (diluted in PBS + 0.2% Triton x-100) for 2 hr at room temperature, washed with PBST, incubated with secondary antibodies for 2 hrs at room temperature and washed with PBS. Reproductive tracts of the different treatments were mounted with Antifade media [58] on a multi-well glass slide (Hendley-Essex, UK). For each treatment (unmated, mated) and antibody/reagent (HRP, DLG, phalloidin) a minimum of ten reproductive tracts from at least two independent biological replicates were prepared.<br><br>Confocal microscopy
Reproductive tracts were viewed with a Zeiss 510 laser scanning confocal microscope using 20× and 60× objective with additional zooming. Optical sections from different focal plans of each reproductive tract region (lateral oviducts, common oviduct, uterus) were collected and projected as a reconstructed three-dimensional image using LSM image browser (version 3,5,0,376) software. Image collections were identical for each of the different reproductive tract regions analyzed.<br><br>Quantitation of bouton number
To quantify the number of boutons in the lateral and common oviducts we used ImageJ software (1.37b, National Institutes of Health) to analyze confocal images of anti-HRP and anti-DLG labeled boutons in the oviduct. The number of boutons per unit area was quantified with the Particle Analysis Tool. Briefly, to differentiate between the boutons, the particle analysis tool requires the image to be a "binary" image (i.e., black or white), thus we first converted the images to gray scale. We then set a "threshold" range so that pixels in the image whose value lies in this range are converted to black; pixels with values outside this range are converted to white. We next defined a region of interest (ROI) within the oviduct to count particles (i.e. count boutons). This ROI was saved and served to measure the number of boutons per unit area in each treatment. For each oviduct we counted the number of anti-HRP and anti-DLG labeled boutons in two ROIs within the lateral oviducts and two ROIs in the common oviduct. One-way ANOVA (SPSS 15.0) was used to measure the difference in bouton number per unit area in different regions of the oviducts, in both unmated and mated females.<br><br>Quantitation of cytoskeleton proteins
Sample preparation
To evaluate the effect of mating on the abundance of the cytoskeleton proteins tested, females were: (i) aged for 3 days, mated with 3-day-old unmated males and their oviducts were dissected after 6 hrs post-mating (Once3); (ii) aged for 3 days, mated with 3-day-old unmated males and their oviducts were dissected after 7 days (Once3 day10); (iii) aged for 3 days, mated first with 3-day-old unmated males and held singly for 7 days. At 10 days of age, female were mated again with 3-day-old unmated males. Oviducts were dissected at 6 hrs post-second mating (Twice3&10). We also examined 5-day-old and 10-day-old unmated females (UM5, UM10 respectively).<br><br>SDS polyacrylamide gel electrophoresis (SDS-PAGE) and Western blotting
For each mating regime, sixty oviducts were pooled and 30 μl of SDS-PAGE sample buffer was added as described in [59]. Samples were boiled, and then frozen at -20°C until loading. SDS-PAGE was performed on 12% polyacrylamide gels and western blotted as in [60]. Proteins were cross-linked to the filter. The following primary antibodies and dilutions were used: <span class="yellow">mouse</span> anti-Neuroglian (kindly provided by M. Hortsch) 1:250; <span class="yellow">Guinea pig</span> anti-Coracle (kindly provided by R.G. Fehon) 1:2500; <span class="yellow">rabbit</span> anti-Mlp84B (kindly provided by M. Beckerle) 1:1000; <span class="yellow">mouse</span> anti-hts (1B1, Developmental Studies Hybridoma Bank, DSHB) 1:75; <span class="yellow">mouse</span> anti-Na, K-ATPase (α5, DSHB) 1:100. Secondary antibodies included: anti-<span class="yellow">Guinea pig</span> IgG (peroxidase conjugated), anti-<span class="yellow">Rabbit</span> IgG and anti-<span class="yellow">Mouse</span> IgG (developed in <span class="yellow">goat</span>, Sigma, Israel) 1:10,000. Proteins were visualized using an enhanced chemiluminescence (ECL) detection system (Amersham Piscataway, NJ).<br><br>Analysis
The developed film was scanned and the signal intensity (protein abundance) of each band was determined using ImageJ software (1.37d, National Institutes of Health). We evaluated protein abundance by measuring the mean gray value of a specific band and the background. The mean gray value of the background was then subtracted from that of the measured band. Relative protein abundance in mated oviduct vs. 3-day-old unmated oviduct was then calculated. Four independent biological replicates were prepared for each mating status. The reported abundance (see Figure 5) is the relative ratio (mated/unmated or unmated/unmated) of at least three replicates that showed the same trend.<br><br>
Examination of female reproductive output
Mating regimes
To evaluate the effect of mating on reproductive output females were treated as follows: (i) aged for 3 days and mated with 3-day-old unmated males (Once3); (ii) aged for 10 days and mated with 3-day-old unmated males (Once10); (iii) aged for 3 days, mated first with 3-day-old unmated males, held for 7 days and mated again with 3-day-old unmated males (Twice3&10). In all cases male and female pairs were observed to record mating initiation and termination.<br><br>Analysis
Following mating, females were aspirated into fresh vials, held singly and allowed to lay eggs for 6 hrs, then transferred daily (each 24 hrs) to fresh vials. The number of eggs laid and the number of eclosed adults were counted from vials created at 6 hrs, 1, 2 and 3 days post-mating. To ascertain the baseline of female egg-laying, we also included in our experiment unmated females that were kept in the same conditions as mated females. The number of eggs laid by unmated females was counted from vials created at 6 hrs, 1, 2 and 3 days after placing the females in the holding vials. In addition, we also recorded the pattern of unmated female egg-laying for 10 days.
To determine the effect of different mating regimes on female reproductive output (i.e. fecundity and fertility), we used One-way ANOVA (SPSS 15.0).<br><br><br><br>Abbreviations
Nrg: Neuroglian; Cora: Coracle; Spec: α- and β-Spectrin; SJ: septate junction; SSJ: smooth septate junction; PSJ: pleated septate junction; ZA: zonal adherens junction; AJ: adherens junction; AECM: apical extracellular matrix; ECM: extracellular matrix; HAJ: hemi-adherens junction; SAJ: spot adherens junction; OA: Octopamine; Hts: Hu-li tai shao; ATP α: Na+ pump α subunit; Mlp84B: Muscle LIM protein at 84B; DLG: Disc Large; HRP: <span class="yellow">horseradish</span> peroxidas; UM: unmated; M: mated.<br><br>Authors' contributions
AK and PKR contributed equally to this manuscript. AK, PKR and YH conceived and designed the project and analyzed the data. AK performed the confocal, Western blots and fertility assays. PKR and AK performed the light microscopy. PKR conducted the electron microscopy. AK, PKR and YH wrote the manuscript. RRH contributed to design of the study and revision of the manuscript. All authors participated in the discussion and approval of the final manuscript.<br><br>Supplementary Material<br><br>
</body></html>