
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>hamster</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1851977</h3>Molecular Basis for a Lack of Correlation between Viral Fitness and Cell Killing Capacity
Abstract
The relationship between parasite fitness and virulence has been the object of experimental and theoretical studies often with conflicting conclusions. Here, we provide direct experimental evidence that viral fitness and virulence, both measured in the same biological environment provided by host cells in culture, can be two unrelated traits. A biological clone of <span class="yellow">foot-and-mouth disease virus</span> acquired high fitness and virulence (cell killing capacity) upon large population passages in cell culture. However, subsequent plaque-to-plaque transfers resulted in profound fitness loss, but only a minimal decrease of virulence. While fitness-decreasing mutations have been mapped throughout the genome, virulence determinants—studied here with mutant and chimeric viruses—were multigenic, but concentrated on some genomic regions. Therefore, we propose a model in which viral virulence is more robust to mutation than viral fitness. As a consequence, depending on the passage regime, viral fitness and virulence can follow different evolutionary trajectories. This lack of correlation is relevant to current models of attenuation and virulence in that virus de-adaptation need not entail a decrease of virulence.
Virulence expresses the harm that parasites inflict upon their hosts. Many studies have addressed the basis of virulence and its effect on host and parasite survival. It has generally been accepted that one of the components of parasite virulence is fitness, or the capacity of the parasite to multiply in its host. Some models have equated virulence with fitness. In the present study, we use <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) to document that virulence and fitness—measured in the same biological environment provided by cells in culture—can be unrelated traits. This has been achieved by multiplying the virus in a manner that mutations accumulated in its genome. Mutations decreased fitness dramatically, but not virulence. Chimeric and mutant viruses were constructed to show that virulence is influenced by only some of the <span class="yellow">FMDV</span> genes, while fitness is influenced by the entire genome. For this reason, virulence is more robust (“resistant”) than fitness to the effects of deleterious mutations. The fact that virulence can be unrelated to fitness has implications for the design of anti-viral vaccines because it suggests that it may be possible to design high fitness, low virulence strains to stimulate the host immune response. Furthermore, in modelling studies it cannot be assumed that virulence is equal to fitness.<br><br><br><br>Introduction
The relationship between fitness and virulence is an unsettled question, and sometimes fitness is considered a component of the virulence phenotype of parasites. RNA viruses are ideal systems to address this important question because of their high mutability and fecundity, which result in a potential for rapid evolution, and also because of the availability of quantitative fitness and virulence assays.
RNA viruses replicate as complex and dynamic mutant spectra, termed viral quasispecies. Key to quasispecies dynamics are mutation rates in the range of 10−3 to 10−5 substitutions per nucleotide copied, and competition among continuously arising variant genomes [1–4], which prompt rapid movements in sequence space, with corresponding changes of position in the fitness landscape [5]. Indeed, large population passages of RNA viruses in cell culture permit competitive optimization of mutant distributions that generally result in fitness gain [6,7], while repeated bottleneck events (experimentally realized as plaque-to-plaque transfers) lead to random accumulation of deleterious mutations (operation of Muller's ratchet [8]) and result in average fitness decreases [9–13]. Fitness recovery of low fitness <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) clones occurs mainly with introduction of mutations along the genome, with very few true reversions.
An understanding of the consequences of fitness variation for viral virulence is a key question for viral pathogenesis and evolution. Here, we approach this issue with <span class="yellow">FMDV</span>, an important viral pathogen in veterinary medicine [14], and one that fully participates of quasispecies dynamics. Our laboratory has characterized multiple <span class="yellow">FMDV</span> variants that derive from one original biological clone, and that occupy widely different fitness levels when replicating in a defined environment in cell culture. We define fitness as the replication capacity of a mutant <span class="yellow">FMDV</span>, relative to a reference <span class="yellow">FMDV</span>, in direct growth-competition upon coinfection of baby <span class="blue">hamster</span> kidney 21 (BHK-21) cells [15–17]. Fitness of <span class="yellow">FMDV</span> in BHK-21 cells is a multigenic trait [7].
In the present study, we define virulence of <span class="yellow">FMDV</span> as the capacity of the virus to kill BHK-21 cells under a standard set of cell culture conditions [18]. Thus, the <span class="yellow">FMDV</span>–BHK-21 system offered a means to investigate in a direct and comparative fashion the relationship between fitness and virulence of a virus, measured in the same biological environment provided by BHK-21 cells. We describe the behavior of an <span class="yellow">FMDV</span> clone (
				), which has a history of repeated serial plaque-to-plaque transfers in BHK-21 cells [11], that attained a very low fitness value relative to its parental reference virus (C-S8c1), and yet, its virulence for BHK-21 cells was significantly higher than that of C-S8c1. A comparative study of the capacity to kill BHK-21 cells of chimeric <span class="yellow">FMDVs</span> constructed with cDNA copies of the two parental <span class="yellow">FMDVs</span> indicates that the enhanced virulence for BHK-21 cells of the low fitness clone is a polygenic trait, with participation of the regions encoding capsid proteins and non-structural proteins 2A, 2B, and 2C as virulence determinants. Three specific amino acid replacements in 2C have been identified as redundant virulence determinants of <span class="yellow">FMDV</span> for BHK-21 cells. Thus, while large population passages of the virus resulted in a gain of both fitness and virulence, subsequent bottleneck passages resulted in a decrease of fitness but not of virulence.
			
The results suggest that fitness is very vulnerable to mutation in any genomic region. In contrast, because of the involvement of several (but not all) viral genes in virulence, and the redundant effect of three 2C substitutions, virulence is a more robust phenotypic trait than fitness, and less vulnerable to accumulation of mutations. Therefore, we provide direct evidence that viral fitness and capacity to kill cells can (in some cases) be unrelated traits. Furthermore, the relationship between fitness and virulence, of being either linked or unrelated traits, depends on the evolutionary history of the virus. This observation has implications for viral pathogenesis, and sheds light on models of virulence proposed on the basis of theoretical and experimental studies with cellular organisms.<br><br>Results
Inability of <span class="yellow">FMDV</span> 
					 to Establish a Persistent Infection in BHK-21 Cells
				
Several biological clones and populations were obtained by passaging <span class="yellow">FMDV</span> biological clone C-S8c1 [19–22] in BHK-21 cells, either as large population passages or plaque-to-plaque transfers (Figure 1). The biological clones and populations differed up to 236-fold in relative fitness (Table 1). The fitness differences found are expected from previous results on fitness gain upon large population passages of RNA viruses [6,7] and fitness decrease upon plaque-to-plaque (bottleneck) transfers [9–13]. The initial experiment was aimed at testing whether 
					, because of its low fitness (0.11 times that of its parental C-S8c1 [12,23,24] [Table 1]), had an advantage in establishing a persistent non-cytopathic infection in BHK-21 cells as compared with its parental clone, C-S8c1 (Figure 1). A persistent <span class="yellow">FMDV</span> infection is established by growing the cells that survive a standard cytolytic infection with <span class="yellow">FMDV</span> [25]. Confluent monolayers of BHK-21 cells were infected either with C-S8c1 or with 
					 at a multiplicity of infection (MOI) of 0.02–0.1 plaque-forming units (PFU)/cell (2 × 106 cells infected with 4 × 104 −2 × 105 PFU). Unexpectedly, at 24 h postinfection, the cells infected with 
					 showed extensive cytopathology, and at 48 h postinfection, no surviving cells were observed. The frequency of surviving cells in parallel infections with C-S8c1 was 5 × 10−3–9 × 10−3, which is consistent with previous determinations [25]. No persistently infected BHK-21 cell cultures could be established with 
					, despite several attempts. Thus, C-S8c1, which displays a 9-fold higher relative fitness than 
					 in BHK-21 cells, showed a capacity to kill BHK-21 cells that was at least 103-fold lower than the killing capacity of 
					 in the infectivity assay intended to establish a persistent <span class="yellow">FMDV</span> infection.
				<br><br><span class="yellow">FMDV</span> Fitness and Capacity to Kill BHK-21 Cells May Not Be Positively Correlated
The capacity of 
					 to kill BHK-21 cells despite its low fitness in BHK-21 cells led us to quantitatively examine the relationship between fitness of <span class="yellow">FMDV</span> and its capacity to kill BHK-21 cells. To this aim, <span class="yellow">FMDV</span> clones or populations were compared in a cell killing assay, consisting in determining the time required to kill 104 BHK-21 cells as a function of the PFU added (described in Materials and Methods). The results (Figure 2A) indicate that over the time range of 12 h to 48 h postinfection, the number of PFUs needed to kill 104 BHK-21 cells varied logarithmically as a function of time. Similar quantifications of relative virulence were obtained by measuring the PFU needed to kill 104 cells in 24 h, and then by extrapolating the PFU values to 0 h postinfection (Tables 1 and S1). Virulence of 
					 was 29 to 35 times higher than virulence of C-S8c1, despite the latter displaying a 9-fold higher fitness (Tables 1 and S1). The high virulence of 
					 was not due to the plaque-to-plaque transfers, since a high virulence was also quantitated for its parental clone, 
					, and for population C-S8p113 (Figure 2B; Tables 1, 2, and S1). 
					 deviated from a line that correlated relative fitness of <span class="yellow">FMDV</span> and the logarithm of cell killing capacity, as reflected in the decrease of the regression coefficient (R2) (inset in Figure 2A). Probably, this deviation is due to the fact that 
					 lost fitness due to plaque-to-plaque transfers, and the other viruses were not subjected to plaque-to-plaque transfers. On the other hand, virulence determinants were acquired during the large population passages done between C-S8c1 and C-S8c1p113. The 29- to 35-fold higher virulence of 
					 with respect to C-S8c1 (Tables 1 and S1), despite its low fitness, indicates that viral fitness and virulence can be two unrelated traits.
				<br><br>Mapping Virulence Determinants in the 
					 Genome
				
The comparison of the consensus nucleotide sequence of the 
					 genome with that of C-S8c1 revealed a total of 47 mutations (Table S2), leading to 21 amino acid replacements affecting structural and non-structural proteins (Figure 3). To identify the genomic regions associated with the increased virulence of 
					 with respect to C-S8c1, we measured the BHK-21 cell killing capacity of nine chimeric viruses rescued from constructs obtained by introducing fragments of cDNA of the 
					 genome into plasmid pMT28, which encodes infectious C-S8c1 RNA [21] (Figure 4). The results (Figure 5; Tables 2 and S1) show that several genomic regions contribute to the virulence of 
					 for BHK-21 cells, and that the major contributors map within genomic positions 2046 to 3760 (residues encoding part of VP2, VP3, and part of VP1, Figure 5A) and 3760 to 5839 (residues encoding 2A, 2B, 2C, and 3A, Figure 5B). The results exclude the internal ribosome entry site and the 3C- and 3D-coding regions as significant virulence determinants of 
					 for BHK-21 cells (virulence of the relevant chimeric viruses ≤ 2.5, relative to C-S8c1; Tables 2 and S1). Infectious progeny production by each chimeric virus was intermediate between the production of the parental viruses pMT28 and 
					, with no significant differences that could be correlated with virulence (Table 2).
				<br><br>Non-Structural Protein 2C Is a Determinant of the Virulence of <span class="yellow">FMDV</span> for BHK-21 Cells
Amino acid substitutions in <span class="yellow">human rhinovirus</span> protein 2C promoted cytopathology for <span class="yellow">mouse</span> L cells [26]. Remarkably, 
					 shares with other <span class="yellow">FMDV</span> clones and populations, notably, MARLS and C-S8p260p3d (the two viruses showing the highest virulence for BHK-21 cells; Figure 2A; Tables 1 and S1), three amino acid substitutions in 2C: S80N, T256A, and Q263H. In addition, MARLS and CS8p260p3d include replacement M283V in 2C, relative to C-S8c1 [27,28]. To test whether any (or a combination) of the three shared amino acid substitutions in 2C contributed to the increased virulence of <span class="yellow">FMDV</span>, each of the mutations was introduced individually into plasmid pMT28 by site-directed mutagenesis, as described in Materials and Methods. Transcripts of the three mutants, termed pMT28 (SN), pMT28 (TA), and pMT28 (QH) (Figure 4), were used to transfect BHK-21 cells, and the viruses obtained were tested with the BHK-21 cell killing assay. Viruses having any of the substitutions in 2C have a virulence intermediate between that of C-S8c1 and 
					 (Figure 6A). To test whether the combination of the three substitutions in 2C could produce an additional increase of virulence, the three mutations were introduced in pMT28 to rescue the triple mutant pMT28 (SN, TA, QH) (Figure 4). The results (Figure 6B) show that the virulence of the triple 2C mutant is similar to the virulence of the individual 2C mutants. The 2C mutations did not significantly affect the infectious progeny production (Table 2). A testable prediction of this result is that the introduction of the wild-type 2C-3A-coding region in the genetic background of 
					 should produce a virus with lower virulence than 
					. Indeed, the results with such a chimeric virus (Figure 5D) indicate that the presence of the 2C- and 3A-coding region as the only genetic region of the pMT28 in the genetic background of 
					 resulted in an <span class="yellow">FMDV</span> with a 2.4- to 4.8-fold lower virulence than 
					. We conclude that mutations in 2C contribute to virulence of <span class="yellow">FMDV</span> for BHK-21 cells.
				
Thus, a virus that evolves towards low fitness levels due to the operation of Muller's ratchet may nevertheless maintain its capacity to kill the same cells in which it displays low fitness. In <span class="yellow">FMDV</span>, the enhanced capacity to kill BHK-21 cells was multigenic, including participation of non-structural protein 2C with three amino acid substitutions acting in a redundant fashion. In conclusion, the results provide a molecular interpretation of why fitness and virulence of an animal virus can follow disparate evolutionary trajectories, culminating in two unrelated traits.<br><br>
Discussion
The capacity of a virus to kill cells is probably influenced by several steps in the virus life cycle, including receptor affinity (which may trigger signalling pathways and alter cell functions) and intracellular viral replication that may lead to metabolic alterations such as transcriptional or translational shut-off [29]. A parallel increase of virulence and fitness as a virus improves its adaptation to a host cell type is expected, since a key parameter that should contribute to the fitness level of a cytophatic virus is the accumulation of infectious particles and its release from cells, which are events often associated with cell killing. This expectation was fulfilled in our experiments (increase of both fitness and cell killing following large population passages of C-S8c1 [Figure 2]), and also in other virus–host systems. In a comparison of two genetically divergent isolates of the whispovirus white spot syndrome virus, whose virulence for the <span class="yellow">shrimp</span> host <span class="yellow">Penaeus monodon</span> was measured by in vivo cumulative mortality rates, virulence correlated with competitive fitness in vivo [30]. The onset of type 1 diabetes by coxsackievirus B strains was linked to the viral replication rate and to the infectious dose [31]. In engineered alphavirus replicons, a direct correlation between the level of RNA replication and cytopathogenicity was observed [32]. At an epidemiological level, a greater replicative fitness of historical versus current <span class="yellow">human immunodeficiency virus type 1</span> (<span class="yellow">HIV-1</span>) isolates was taken as evidence of <span class="yellow">HIV-1</span> attenuation over time, assuming a direct connection between fitness and virulence [33]. In vivo, viral fitness may vary among specific organs, and virulence may be affected only when fitness for some specific target tissues is affected [34].
Replicative fitness is, however, but one of several factors which influence the progression of a viral infection in vivo. In a comparative analysis, R5-tropic and X4-tropic clones of <span class="yellow">HIV-1</span> showed similar replication capacity in mitogen-activated T cells. However, X4 clones were transferred more efficiently than R5 clones from dendritic cells to CD4(+) T cells, a fact that can contribute to the competitive advantage of X4 viruses in AIDS <span class="yellow">patients</span> [35]. <span class="yellow">Simian immunodeficiency virus</span> <span class="yellow">SIVmac239</span> infects both the <span class="yellow">sooty mangabey</span> and the <span class="yellow">rhesus macaque</span>, reaching high viral loads in both hosts, yet it is only virulent for the <span class="yellow">rhesus macaque</span> [36]. Deviations of a positive correlation between viral fitness and virulence were observed also in the plant viruses <span class="yellow">cucumber mosaic virus</span> [37] and <span class="yellow">barley stripe mosaic virus</span> [38]. A study of the effect of lysis timing on <span class="yellow">bacteriophage</span> fitness revealed that a strain with an intermediate lysis time had the highest fitness [39]. The time of transmission may also affect virulence. Nuclear polyhedrosis virus transmitted early to its host, the <span class="yellow">moth</span> <span class="yellow">Lymantria dispar</span>, was more virulent than virus transmitted late, although the latter was more productive because the virus could use more host tissue for replication [40]. In a study of the susceptibility of North American and non–North American breeds of Lymantria to several isolates of the fungus <span class="yellow">Entomaphaga maimaiga</span>, mortality was scored in all cases. However, virulence of the fungus, quantitated by the time of death of Lymantria, was, in some cases, inversely proportional to fitness, quantitated by fungal reproduction in the <span class="yellow">moth</span> [41]. In all these cases, the molecular basis of the lack of positive correlation between fitness and virulence is not understood.
Model for a Lack of Correlation between Viral Fitness and Virulence
The results with <span class="yellow">FMDV</span> clones H5 have documented that both fitness-enhancing and virulence-enhancing mutations can be incorporated in the viral genome in such a fashion that subsequent fitness-decreasing mutations associated with bottleneck (plaque-to-plaque) transfers produce only minimal effects on virulence (Figure 2). The dissection of accompanying molecular events, achieved through quantification of virulence of recombinant and mutant genomes (Tables 2 and S1), provides an interpretation of the lack of positive correlation between virulence and fitness. Multiple fitness-decreasing mutations occur in the course of plaque-to-plaque transfers, distributed throughout the <span class="yellow">FMDV</span> genome [11]. In contrast, determinants of virulence for BHK-21 cells are multigenic, but concentrated mainly in some <span class="yellow">FMDV</span> genomic regions. Similar multigenic but discrete virulence determinants have been described also in other virus–host systems [42,43]. To decrease virulence, mutations occurring randomly in the course of plaque-to-plaque transfers should affect specific genomic sites, and this will occur with a lower probability than fitness-decreasing mutations, which can hit any of the multifunctional picornaviral proteins and regulatory regions [11]. This model is reinforced by the observation that three amino acid substitutions in 2C (S80N, T256A, and Q263H) had a similar effect in enhancing <span class="yellow">FMDV</span> virulence, and the three mutations in the same genome had an effect comparable to each mutation individually (Figure 6; Tables 2 and S1). It is not clear what the basis of the contribution of 2C to virulence for BHK-21 could be. 2C is involved in RNA synthesis and contains a nucleotide-binding domain, although none of the substitutions found in 
					 and 
					 lie within such a domain. An unlikely triple reversion would be required to eliminate the virulence-enhancing effect of the three mutations in 2C. We propose that a higher robustness of the <span class="yellow">FMDV</span> genome with regard to virulence for BHK-21 cells, rather than to replicative fitness in the same cells, underlies the different trajectories followed by fitness and virulence upon subjecting the virus to repeated bottleneck transfers. Obviously, we cannot exclude that parameters of the virus life cycle, other than fitness as measured in our experiments, could correlate with virulence for BHK-21.
				
The comparative analysis of <span class="yellow">FMDV</span> clones and populations shows that shifts in virulence can occur even through the evolution of a single viral clone (C-S8c1), with its restricted genetic diversity prompted by different replication regimes in the same host cells, which also have a clonal origin (see Materials and Methods). We conjecture that the demonstration that fitness and virulence can follow different evolutionary courses has been possible thanks to the consequences of the extreme passage regimes to which the viral populations were subjected: competitive evolution of an ample mutant spectra during repeated large population passages, and accumulation of deleterious (with regard to fitness, but not with regard to virulence) mutations upon plaque-to-plaque transfers (predominance of genetic drift and operation of Muller's ratchet) [12,15].<br><br>Implications for Models of Virulence
It must be emphasized that fitness and virulence are relative values that pertain to a defined physical and biological environment. Virulence determinants of <span class="yellow">FMDV</span>, identified here for BHK-21 cells, need not apply to virulence for the natural animal hosts of <span class="yellow">FMDV</span> [44]. However, the observation of a lack of correlation between fitness and virulence in a <span class="yellow">FMDV</span> clone is relevant to current models of attenuation and virulence, since it shows that more virulent forms of a virus need not have a reproductive advantage, and that viral virulence is not necessarily a byproduct of viral fitness. Even if virulence is regarded as an unavoidable consequence of parasite adaptation [45], virus de-adaptation (fitness loss) need not entail a decrease of virulence.
Most current definitions of virulence include both the ability of the pathogen to multiply and to cause harm to its host; some authors, however, assume a direct relationship between fitness and capacity to produce disease [46–48]. In relating the results with <span class="yellow">FMDV</span> to general models of virulence in host–parasite systems, it must be considered that in the <span class="yellow">FMDV</span> system, evolution of the host BHK-21 cells could not influence <span class="yellow">FMDV</span> evolution, because clonal cell populations with a controlled passage history were supplied in constant numbers at each infection event (see Materials and Methods). Therefore, changes in host density, or mobility, as well as pathogen survival in the external environment, all of which are relevant parameters in virulence models [48,49], cannot play a role in our system. A consistent finding in serial passage experiments is that virulence of a parasite increases with passage number in a new host [50]. The results with <span class="yellow">FMDV</span> infecting BHK-21 cells cytolytically imply that the increase of virulence can be conditioned to the history of passage regimes undergone by a virus.
The invariance of BHK-21 cells in the course of serial cytolytic passages of <span class="yellow">FMDV</span> is in contrast with the parallel system consisting of BHK-21 cells persistently infected with <span class="yellow">FMDV</span> C-S8c1 [25], in which the cells are passaged and coevolve with the resident virus [51]. Host–virus coevolution has generally favored a decrease of viral virulence in the field, a classical example being <span class="yellow">myxoma virus</span> and myxomatosis in <span class="yellow">rabbits</span> [52].
Our comparison of <span class="yellow">FMDV</span> clones did not provide evidence of clones with high fitness and low virulence, which, with regard to natural hosts, is an aim of biomedicine to obtain vaccine strains. Yet, the existence of specific mutations that differentially affect fitness and virulence opens the way to engineer candidate vaccine strains unable to kill the host, while maintaining replicative competence. Virulence is, however, a feature of the host–parasite relationship [46], and the mutations needed to impair virulence are expected to be host-dependent [53,54].<br><br>
Materials and Methods
Cells, viruses, and infections.
The BHK-21 cells used in the present study were cloned by end-point dilution, followed by preparation of a cell stock from a single cell; they were passaged a maximum of 30 times before being used for <span class="yellow">FMDV</span> infection [25,51]. Procedures for cell growth, infection of BHK-21 cell monolayers with <span class="yellow">FMDV</span> in liquid medium, and plaque assays in semi-solid agar medium were carried out as previously described [11,19,25,27]. Mock-infected cells were handled in parallel in all infectivity and plaque assays to monitor absence of viral contamination. The <span class="yellow">FMDVs</span> used in the present study (Figure 1) are (i) the reference clone C-S8c1, which has been assigned a relative fitness of 1.0 [11]. (ii) MARLS, a monoclonal antibody escape mutant isolated from population C-S8c1p213 [55]; MARLS has a fitness of 25 relative to C-S8c1 [24]. (iii) C-S8p260p3d, a standard <span class="yellow">FMDV</span> virus rescued by low MOI passage of C-S8p260. The latter is a virus that evolved by passage of C-S8c1 at a high MOI, which resulted in dominance of two defective <span class="yellow">FMDV</span> genomes (both including internal deletions) that were infectious by complementation, in the absence of standard virus [22,24,28]; C-S8p260p3d has a relative fitness of 20 [24]. (iv) REDpt60, obtained after 60 successive plaque-to-plaque transfers of RED (a monoclonal antibody escape mutant isolated from population C-S8c1p100) [20]; REDpt60 has a fitness of 1.9 relative to C-S8c1. (v) C-S8c1p113, a viral population obtained after 113 serial cytolytic passages of C-S8c1 at a high MOI in BHK-21 cells (2 × 106 BHK-21 cells infected with the virus contained in 200 μl of the supernatant from the previous infection). (vi) Clone 
					, a biological clone isolated from population C-S8c1p113 [11]; its relative fitness is 26 (unpublished data). (vii) Clone 
					, obtained after 95 successive plaque-to-plaque transfers of 
					 [11]; its relative fitness is 0.11 [23].
				<br><br>Cell killing assay.
The capacity of <span class="yellow">FMDV</span> to kill BHK-21 cells was measured as previously described [18,22]. The assay consists in determining the minimum number of PFU required to kill 104 BHK-21 cells after variable times of infection. The assay was performed in M96 multiwell plates with monolayers of 104 BHK-21 cells per well infected with serial dilutions of virus. At different times postinfection, cells were fixed with 2% formaldehyde and stained with 2% crystal violet in 2% formaldehyde. Results are expressed as the logarithm of the number of PFUs needed for complete cell killing (as judged by cell staining with crystal violet, with series of control wells with known numbers of cells) as a function of time postinfection [18,22].<br><br>Determination of relative fitness of 
					.
				
The relative fitness of <span class="yellow">FMDV</span> 
					 was determined by growth competition in BHK-21 cells as previously described [7,10,11,24,56]. <span class="yellow">FMDV</span> 
					 was mixed with appropriate proportions of 
					, which was used as reference virus. This virus has a fitness 8.5-fold higher than that of the reference clone of C-S8c1 in BHK-21 cells [7,10,11,24,56]. Four serial infections were carried out at MOI of 0.1 PFU/cell. The proportion of the two competing genomes at different passages was determined by real-time reverse transcription (RT)–PCR, employing primers 5531wtnew and 
					, which are able to discriminate <span class="yellow">FMDV</span> 
					 RNA from 
					 RNA. The nucleotide sequences of the primers will be provided upon request. The fitness vector obtained for 
					 corresponded to the equation y = 0,0206e1,1074x; R2 = 0.9507. The antilogarithm (base e) of the vector slope is the fitness of the assayed virus relative to the reference virus [56].
				<br><br>cDNA synthesis, molecular cloning, and nucleotide sequencing.
Viral RNA was extracted by treatment with Trizol as previously described [57]. Reverse transcription of <span class="yellow">FMDV</span> RNA was carried out with <span class="yellow">avian myeloblastosis virus</span> reverse transcriptase (Promega, http://www.promega.com) or Transcriptor reverse transcriptase (Roche, http://www.roche.com), and PCR amplification was performed by using either Ampli-Taq polymerase (PerkinElmer, http://las.perkinelmer.com) or an Expand High Fidelity polymerase system (Roche), as instructed by the manufacturers. The <span class="yellow">FMDV</span> genome-specific oligonucleotide primers used have been previously described [22,58]. In all RT-PCR amplifications, negative amplification controls, including all reaction components except template RNA, were run in parallel to monitor absence of contamination.
Chimeric viruses containing selected regions of 
					 in the genetic background of C-S8c1 (Figure 4) were obtained by replacing the corresponding DNA fragment of pMT28 by a cDNA copy of 
					 RNA, using specific restriction sites. To obtain pMT28/
					 (436-2046), a chimera that included nucleotides 436 to 2046 of 
					 (the residue numbering of the <span class="yellow">FMDV</span> genome is as in [11]), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28 DNA digested with the same enzymes. To obtain pMT28/
					 (2046–3760), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760). To obtain pMT28/
					 (3760–5839), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839). To obtain pMT28/
					 (5839–7427), 
					 RNA was amplified by RT-PCR using primers 5531 wt new and C-Not-Pol, and then the cDNA was digested with Rsr II (5839) and Bam HI (7427). To obtain pMT28/
					 (436-3760), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and then the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–3760) DNA digested with the same enzymes. To obtain pMT28/
					 (3760–7427), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839), and ligated into pMT28/
					 (5839–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (2046–7427), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760), and ligated into pMT28/
					 (3760–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (436-7427), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2; the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–7427) DNA digested with the same enzymes. To obtain 
					/2C-3A(pMT28), pMT28 was digested with Bgl II (4201) and Rsr II (5839), and a DNA fragment including wild-type 2C-3A-coding region was inserted into pMT28/
					 (436-7427) DNA digested with the same enzymes. DNA ligation, transformation of <span class="yellow">Escherichia coli</span> DH5α, isolation of DNA from bacterial colonies, and characterization of DNA by restriction enzyme digestion were performed by standard procedures [59]. The primers used for molecular cloning and site-directed mutagenesis are described in Table S3.
				
To obtain <span class="yellow">FMDV</span> C-S8c1 containing the mutations found in gene 2C of 
					, plasmid pMT28 was subjected to site-directed mutagenesis using an oligonucleotide including the required nucleotide replacement, and 3R2New or 3CD1 as external oligonucleotide primer (Table S3; Figure 4). A DNA fragment termed A was obtained by subjecting plasmid pMT28 to site-directed mutagenesis using primers (reverse) mutSNu, mutTAu, and mutQHu (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3R2New, forward). A DNA fragment termed B was obtained amplifying pMT28 with primers (forward) mutSNd, mutTAd, and mutQHd (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3CD1, reverse). DNA fragments A and B, including the desired mutations, were recombined by shuffling PCR using equimolar amounts of DNA fragments and two external primers (3R2New and 3CD1). The DNA with the desired mutation(s) in the 2C gene was digested with Avr II (genomic position 3760) and Rsr II (position 5839), and cloned into pMT28 to generate pMT28 (SN), pMT28 (TA), and pMT28 (QH). To obtain pMT28 (SN, TA, QH), plasmid pMT28 (SN) was subjected to site-directed mutagenesis to introduce mutation T256A in a similar way as described above, and then, plasmid pMT28 (SN, TA) was subjected to site-directed mutagenesis to introduce mutation Q263H. All chimeric viruses and mutants were analyzed by nucleotide sequencing using Big Dye Terminator Cycle Sequencing kit (Abi Prism; PerkinElmer) and sequencer ABI373 as previously described [58]. Sequences were analyzed using DNASTAR 4.0 (http://www.dnastar.com), GeneDoc, and GCC (University of Wisconsin). Each sequence was determined at least twice, with products obtained using different oligonucleotide primers.
				
DNA from pMT28 or its recombinant and mutant derivatives was linearized with Nde I and transcribed with SP6 RNA polymerase as previously described [22,27]. Transcript RNA integrity and concentration were estimated by agarose gel electrophoresis, in parallel runs with known amounts of standard C-S8c1 RNA. BHK-21 cell monolayers (70% confluent, about 1 × 106 cells) were transfected with RNA transcripts (1 μg RNA) using lipofectin as previously described [59]. Virus was collected from the culture supernatant at 72 h post-transfection. The virus obtained by transfection was passaged twice before using it in biological studies. RNA was extracted and sequenced to ascertain that the virus maintained the genomic structure and mutations of the initial transcript.
Consensus genomic nucleotide sequences of <span class="yellow">FMDV</span> clones were obtained by RT-PCR amplification of virion RNA using specific primers [7,22,28].<br><br>
Supporting Information
Accession Numbers
The GenBank accession numbers for the C-S8c1, 
					, 
					, CS8p260p3d, and MARLS genomic sequences are AJ133357, AM409190, AM409325, DQ409185, and AF274010, respectively. Nucleotide and amino acid sequences for picornaviruses can be found at http://www.iah.bbsrc.ac.uk/virus/picornaviridae/SequenceDatabase/3Ddatabase/3D.HTM.
				<br><br><br><br><h3>pmcA327937</h3>Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a.
Abstract
Attempts to prevent the urea conversion of a 200-230,000 molecular weight DNA polymerase alpha to a 150-170,000 molecular weight form by the inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70,000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70,000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit.<br><br><br><br>
 Volume 6 Number 10 1979 <br><br> Nucleic Acids Research <br><br> Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a Keith McKune and Andrew M.Holmes* <br><br> Biochemistry Department, Strathclyde University, The Todd Centre, 31 Taylor Street, Glasgow, G4 ONR, UK <br><br> Received I June 1979 ABSTRACT <br><br> Attempts to prevent the urea conversion of a 200-230, 000 molecular weight DNA polymerase a to a 150-170, 000 molecular weight form by the <br><br> inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70, 000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70, 000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit. <br><br> INTRODUCTION <br><br> In mammals DNA polymerase a is thought to be the replicative <br><br> enzyme, but due to low levels of activity, even in tissues actively making DNA, and to enzyme heterogeneity it has been found difficult to purify <br><br> However, small samples of DNA polymerase a have been highly purified <br><br> 2, 3, 4, 5 from several sources and partially characterised <br><br> Heterogeneity has been observed in DNA polymerase a from a var<br><br> 6           7 <br><br> iety of sources, including <span class="yellow">rat</span> liver and spleen , Hela cells, baby <span class="red">hamster</span> <br><br> 8                9, 10                   11 <br><br> kidney cells , <span class="yellow">mouse</span> myeloma    , <span class="yellow">Drosophila</span> embryos  and <span class="yellow">calf</span> thymus <br><br> 1,6, 12, 13 . We have previously observed several species of <span class="yellow">calf</span> thymus <br><br> DNA polymerase a differing in size and charge  . In order of elution from DEAE cellulose they are enzyme A1 (200-230, 000 molecular weight), A2 (200-230, 000), B (100-110, 000) and C (150-170, 000). A poly (dA). oligo <br><br> (dT)10 preferring enzyme, enzyrrme D (140-150, 000 molecular weight) elutes just after enzyme B. The A enzymes seem identical in all properties except their charge. 5.0-5. 5S enzymes, analagous to B, have been observed to <br><br> C Information Retrieval Limited 1 Falconberg Court London Wl V 5FG England <br><br> 3341 <br><br> Nucleic Acids Research <br><br> arise as the result of proteolytic action  ' , similarly the B enzyme above 14 <br><br> ,but the relationship of the other observed species is not clear, nor is <br><br> the problem of which, if any, of these species is the replicative enzyme, <br><br> although circumstantial evidence has implicated a DNA polymerase a to be <br><br> 15 the replicative enzyme in <span class="yellow">adenovirus</span> infected KB cells <br><br> We have previously shown that several of these enzymes are inter<br><br> 16 <br><br> convertible . In particular, mild urea treatment can convert both A <br><br> enzymes to C enzyme, with the loss of a subunit or fragment of 50-70, 000 molecular weight. The C enzyme retains DNA polymerase activity, but <br><br> does show differences compared to A enzyme in heat sensitivity and sensitivity to N-ethylmaleimide. <br><br> Highly purified samples of A and C enzymes when subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis have shown a corre<br><br> lation of DNA polymerase activity with a polypeptide band 155, 000 molecular weight . Contaminating material of 50-70, 000 molecular weight appeared <br><br> to obscure the putative 50-70, 000 subunit in gels of A enzyme. The conclusions drawn were that DNA polymerase a is a 155, 000 molecular weight <br><br> polypeptide (C enzyrrme) which can and does associated with material of 50<br><br> 70, 000 molecular weight to give A enzyme, and that enzyme B is a proteol<br><br> 14 <br><br> ytic degradation product . The relationship of enzyme D to the others is <br><br> not clear. <br><br> It is possible that during the enzyme isolation procedure proteolytic action may have taken place on a 200-230, 000 molecular weight enzyme releasing 155, 000 and 50-70, 000 fragments which remain in association <br><br> until urea treatment separates them. It is also possible that urea treatment itself renders the enzyme susceptible to contaminating proteases. Further experiments to ascertain whether or not this is the case and, if so, to prevent it, have been carried out. <br><br> A enzyme can be reconstituted from C enzyme formed by the action <br><br> of urea on A enzyme by concentrating it with the 50-70, 000 molecular weight material. The A enzyrme obtained in this manner is highly purified, as is <br><br> the C enzyme formed by the urea treatment, and has been used in attempts to show differences in synthetic template-initiator complex utilisation <br><br> 3342 <br><br> Nucleic Acids Research <br><br> b y the se two enzyme s . <br><br> MATERIALS AND METHODS <br><br> <span class="yellow">Calf</span> thymus was obtained from 10-16 week old <span class="yellow">calves</span> and frozen at <br><br> -20?C until required. Chromatographic media and chemicals were obtained <br><br> 6 <br><br> from sources previously referred to . Radioactive deoxynucleoside tri<br><br> phosphates were obtained from the Radiochemical Centre, Amersham, Bucks. Synthetic oligo and polynucleotides were obtained from P. L. Biochemicals <br><br> Inc., except for poly (dC) which was a gift from Dr. I.R. Johnston and pre<br><br> 17 <br><br> pared from oligo d(C)5 as described  . N-a-p Tosyl-L-lysine chloromethy<br><br> lketone HCl and phenylmethylsulphonylfluoride were obtained from Sigma and Trasylol from Bayer. <br><br> Except where indicated all buffers contained 20% w/v glycerol and <br><br> 1 mM dithiothreitol. Standard linear phosphate gradients were run between 0.03 M and 0.25 M potassium phosphate, pH 7.8. Gradient salt concentr<br><br> 2 <br><br> ations were measured using a conductivity meter as described . Urea was <br><br> prepared as a 4. 8 M solution in 20% w/v glycerol, stirred with Amberlite <br><br> MB3 and filtered. Dithiothreitol was added to a final concentration of 1 mM before use. <br><br> 6 DNA polymerase was assayed using activated DNA as described <br><br> except that the buffer was 50 mM tris HC1, pH 7.8. One unit of DNA polymerase activity incorporates ln mol [ H] dTMP into an acid insoluble form in one hour at 370C. Assays using synthetic template-initiator complexes were carried out at 30?C in 0. 12 ml. The template-initiator complexes <br><br> 6 <br><br> were prepared and assays processed as described . All assays contained <br><br> 1 mM dithiothreitol, 62. 5 p.g <span class="yellow">bovine</span> serum albumin, 1 ,ug template-initiator complex, enzyme protein and the relevant [ H] deoxynucleoside triphosphate at 0.1 mM and 12-15 cpm/pmol. These assays were carried out at either pH 6.4 in 20 mM sodium-potassium phosphate, or at pH 7.8 in 50 mM tris HCl and contained either 10 mM MgCl2 or 1 mM MnCl2 as indicated. <br><br> Preliminary purification of DNA polymerase a to Fraction IV was as <br><br> 6 <br><br> described , the purification steps being phosphocellulose chromatography, <br><br> ammonium sulphate precipitation and gel filtration on Sepharose 6B. <br><br> 3343 <br><br> Nucleic Acids Research <br><br> Enzyme obtained from the DEAE cellulose step is referred to as Fraction <br><br> V enzyme. Samples were prepared for sodium dodecyl sulphate polyacrylamide gel electrophoresis and scanned as previously described2 <br><br> RESULTS AND DISCUSSION <br><br> (a)   Interconversion Studies <br><br> A enzyme was routinely converted to C enzyme by incubating 200500 units/ml of Fraction V A enzyme in 2.4 M urea in 0. 02 M potassium <br><br> phosphate, pH 7.8, for 60 minutes at 0?C. The mixture was then loaded on to a DEAE cellulose, washed with 0. 03 M potassium phosphate pH 7. 8, and enzyrme eluted either with the standard phosphate gradient or batchwise. Under these conditions usually about 50-60% of recovered activity was <br><br> enzyme C (Fig 1 a). Overall recovery was 70-80%. A enzyme was recon<br><br> 14 <br><br> stituted essentially as described . The flow through material from the <br><br> DEAE cellulose column after the urea treatment was loaded on to a 1 x 0. 8 cm phosphocellulose column, washed with 0.03 M potassium phosphate, pH 7.8, and the protein eluted with 0.25 M potassium phosphate, pH 7. 8. This material, the putative subunit, was vacuum dialysed with the C enzyme produced by the urea treatment, rechromatographed on DEAE cellulose and enzyme eluted batchwise (Fig 1 b). Recovery from this procedure was <br><br> usually 50-70% A enzyme. Overall recovery was 30-40% of the C enzyme dialysed. If the material eluted from the phosphocellulose by the 0. 25 M potassium phosphate was heated to 90?C for 5 minutes prior to vacuum <br><br> dialysis with the C enzyme the recovery from the DEAE cellulose column <br><br> was significantly higher (60-70% of the original C enzyme activity), but all recoverable DNA polymerase activity was C enzyme. This, together with the fact that the 60 minute treatment with urea has, on occasions, given <br><br> rise to a 50% increase in DNA polymerase activity prior to loading on to the DEAE cellulose column, would indicate that A enzyme is less active on <br><br> activated DNA than is C enzyme. If the DEAE cellulose flow through material came from urea treatment of A2 enzyme then A2 was produced on reconstitution; if from A1 then A1 was produced. <br><br> Although the mild urea treatment of A has been used to prepare C of <br><br> 3344 <br><br> Nucleic Acids Research <br><br> b <br><br> 120       180 C <br><br> I         I    ^ <br><br> Figure la  DEAE cellulose chromatography after 2.4 M urea treatment. <br><br> 4, 5000 units, 4.4 mg of Fraction II A2 enzyme were incubated with urea at a final concentration of 2.4 M for 60 minutes at 0?C, loaded on to a 5 x 1.4 cm DEAE cellulose column, the column was washed with 0. 03 M potassium phosphate, pH 7.8 and a 200 ml standard phosphate gradient applied. 2.3 ml fractions were collected and 10 Rl assayed for 5 minutes. (o-o) no <br><br> phenylmethylsulphonylfluoride (*-*) 3 mM phenylmethylsulphonylfluoride ( - ) phosphate gradient. <br><br> Figure lb  The reconstitution of A2 enzyme. 3000 units for C enzyme <br><br> derived by urea treatment of A2 were vacuum dialysed with the DEAE cellulose flow through material after phosphocellulose chromatography and <br><br> chromatographed on a 2 x 1. 2 cm DEAE cellulose column. After washing with 0.03 M potassium phosphate, pH 7.7, the enzymes were eluted batchwise with the above concentrations of potassium phosphate, pH 7.8. 1 ml fractions were collected and 10 .l assayed for 10 minutes. <br><br> 3345 <br><br> Nucleic Acids Research <br><br> high specific activity it is possible that conversion of the 200-230, 000 molecular weight enzymes to a 155- 170, 000 species maybe due to unfolding of the molecule to allow limited attack by contaminating proteases. Accord<br><br> ingly the urea conversion of A to C was investigated in the presence of certain protease inhibitors. The presence of the serine protease inhibitor <br><br> phenylmethylsulphonyl-fluoride in the incubation and chromatography buffers did not affect the conversion of A2to C (Fig 1 a). Likewise trasylol and Na-p Tosyl-L-lysine chloromethylketone HC1 had no effect. However, protease action could have occurred earlier in the purification procedure and the urea could be separating two fragments. Usually the DEAE cellulose profile shows that the majority of the enzyme activity is present as A <br><br> enzyme (Fig 2). In this instance A1 and A2 have not been separated. When the temperature of the material in the original blending procedure was kept below 0?C or phenylmethylsulphonylfluoride, N-a-p Tosyl-L-lysine chloromethylketone HCl or trasylol was included in the isolation buffers the DEAE cellulose profile was similar. The A enzymes were still capable of conversion by mild urea treatment to C enzyme, indicating that if protease activity is involved then it is not susceptible to these inhibitors. When the <span class="yellow">calf</span> <br><br> 20FI: Now ~ ~      ~      20 20Fr Na      06 <br><br> Figure 2  DEAE cellulose elution profile of <span class="yellow">calf</span> thymus DNA polymerase a. 47, 000 units, 97 mg of Fraction IV enzyme prepared from 415 g of <span class="yellow">calf</span> <br><br> thymus were loaded on to a 10 x 1.8 cm DEAE cellulose column, after washing with 0. 03 M potassium phosphate, pH 7. 8 a 400 ml standard phosphate gradient was applied. 5 ml fractions were collected and 10 ,ul assayed for 5 minutes. <br><br> 3346 <br><br> Nucleic Acids Research <br><br> thymus was allowed to warm up during the blending procedure, or the supernatant prior to phosphocellulose chromatography was heated to 37 ?C for 30 minutes, there was a marked decrease in the amount of A enzyme with a concomitant increase in the amount of B and C (unpublished observation). The presence of phenylmethylsulphonylfluoride, N- a-p T osyl- L-lysine <br><br> chloromethylketone HCl or trasylol under these conditions only had a marginal effect on the appearance of C enzyme, but did reduce the amount of B <br><br> enzyme. Heating the enzyme to 37?C after the phosphocellulose step had no effect on the DEAE cellulose elution profile. Attempts to convert A enzyme to C using trypsin have not been successful. A enzyme activity is lost without the appearance of any other species (unpublished observation), although <br><br> 14 the action of trypsin on C enzyme can give rise to small amounts of B <br><br> Although the conversion of A enzyme to C does not appear to be the result of serine protease action, proteases other than serine proteases <br><br> 18 <br><br> could have been responsible  . Also, the fact that A enzyme can be recon<br><br> stituted from C plus the flow through material from the DEAE cellulose after urea treatment does not necessarily mean we are dealing with two subunits as fragments produced by proteases mray be reassembled to give active <br><br> enzyme 9    . However, the fact that a 200-230, 000 molecular weight polypeptide band has never been observed in sodium dodecylsulphate polyacrylamride gels of highly purified A enzyme , or even cruder fractions of A <br><br> enzyrre (unpublished observation), may be significant. One might expect <br><br> some of the enzyme not to have been attacked by whatever is responsible for cleaving the molecule, if, indeed, this does happen. It would appear, therefore, that the A enzyme consists of subunits of 155, 000 and 50-70, 000 molecular weight with the small subunit having a slightly different charge in the <br><br> case of A  and A2. It has also been concluded that the heterogeneity in the <br><br> 1                                     ~~~~~~~~~~~~~~~~~3 <span class="yellow">mouse</span> myeloma DNA polymerase a fraction is not due to proteolysis . However, the 50-70, 000 molecular weight subunit has not yet been identified. <br><br> Sodiurrm dodecylsulphate polyacrylamide gels of reconstituted A enzyme have shown polypeptide bands at 150-160, 000 and 50-70, 000 molecular weight <br><br> (Fig. 3), but the ratio of staining intensity of the bands does not correspond to a 1: 1 relationship. The ratio of the two bands is variable but is usually between 1:2 and 1:3, indicating, perhaps, that more than one subunit of 50<br><br> 3347 <br><br> Nucleic Acids Research <br><br> TOP      123   4 5   6 <br><br> Figure 3  Scan of a 5% sodium dodecyl sulphate polyacrylamide gel of <br><br> reconstituted A2 enzyme. 1500 units of reconstituted A2 enzyme were subjected to polyacrylamide gel electrophoresis under non-denaturing conditions. The gels were sliced, enzyme extracted and assayed and the peak <br><br> fraction of DNA polymerase activity from three gels were pooled, subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis on a single <br><br> gel, stained and scanned at 2 volt sensitivity as described2. The molecular weight standards were: (1) <span class="yellow">bovine</span> serum albumin dimer (134, 000), (2) ,3 galactosidase (130, 000), (3) phosphorylase a (94, 000), (4) <span class="yellow">bovine</span> serum <br><br> albumin (68, 000), (5) pyruvate kinase (57, 000) and (6) lactate dehydrogenase (35, 000). <br><br> 70, 000 molecular weight can associate with the 155, 000 subunit. The proportion of the lower molecular weight polypeptide band is higher in the reconstituted A enzyrrme than in the C enzyme preparation from which it was formed, but even the C enzyme contained some material in the region. <br><br> Breakdown of material from 155, 000 to 50-70, 000 may be partly responsible for the contamination, but the presence of phenylmethylsulphonylfluoride in samples in preparation for sodium dodecylsulphate polyacrylamide gel electrophoresis has not been successful in preventing it. The <span class="yellow">human</span> KB cell <br><br> DNA polymerase a, equivalent to the C enzyme, has been reported to consist <br><br> 4 <br><br> of subunits of 76, 000 and 66, 000 molecular weight , but at no time have we <br><br> observed polypeptide bands at 76, 000 and 66, 000 rising and falling with <br><br> enzyme activity in any gels of <span class="yellow">calf</span> thyrrmus enzymes A1, A  and C (and <br><br> 2 unpublished observations). (b)    Template Studies <br><br> Use has been made of the urea conversion of A to C and of the reconstitution of A enzyme to obtain samples of highly purified DNA polymerase, <br><br> specific activity in excess of 50, 000 units/mg, in order to study the effect of this 50-70, 000 molecular weight subunit on the DNA polymerase subunit. Previous results have indicated that A enzyrrme is stabler to heat and less <br><br> 3348 <br><br> Nucleic Acids Research <br><br> 14, 21 <br><br> susceptible to N-ethylmaleimide than C enzyme  . It has been reported <br><br> that highly purified DNA polymerase a can be associated with a protein capable of binding to DNA containing no 3' OH ends and capable of being released during the DNA polymerase assay  . Attempts to dissociate the 50<br><br> 70, 000 molecular weight subunit from the polymerase subunit by incubating A2 with DNA polymerase reaction mixes containing activated DNA, poly <br><br> (dA-T) and poly (dT). oligo (A)10 followed by ultracentrifugation in high salt have been unsuccessful. Similarly A absorbed and eluted from either native or denatured DNA cellulose remained A enzyme. Both A and C enzymes <br><br> were eluted from the DNA celluloses by less than 0. 1 M NaCl so it does not appear that the subunit enhances the binding of the DNA polymerase subunit to DNA. However, there are differences in the response of A and C <br><br> enzymes to synthetic template-initiator complexes (Table 1). Even if one takes into account the fact that A enzyme is less active on activated DNA <br><br> than C enzyme (the addition of the 50-70, 000 subunit to the polymerase subunit appears to result in a decrease of about 30% of polymerase activity on activated DNA) the A enzyme is still more active on these templates. <br><br> Although there is a variation in activity on these template-initiator comp<br><br> lexes each time assays are carried out on them depending on the method of preparing the complexes and the base ratio of template to initiator, A enzyme always seems to be significantly more active than C. The A2 <br><br> enzyme at pH 7.8, with extra subunit(s) is clearly more effective on the <br><br> oligoribonucleotide initiator, oligo(A)10 than is enzyme C. In view of the proposed RNA initiation of Okazaki pieces this may indicate a role of this subunit in Okazaki piece synthesis, in that it may aid the DNA polymerase to'take over' from the RNA polymerase. Using poly (dA). oligo (dT) 10 <br><br> (A:T=20: 1) and following the incorporation of [ H] dTMP as a function of time a short lag was observed for C enzyme, but not for A2 (Fig. 4). <br><br> Similar re sult s we re obtained when poly (dC). oligo (dG) 1 0 (C: G= 5: 1) wa s <br><br> used as temrrplate-initiator, but not when poly (dT). oligo (A)10 (T:A=1: 1) was used. Neither enzyme showed a lag on this template-initiator complex or on activated DNA. Incorporation versus enzyme concentration also showed this lag for C enzyme on poly (dA). oligo (dT)10. It is not certain what causes <br><br> 3349 <br><br> Nucleic Acids Research <br><br> TABLE 1 Template utilisation by reconstituted A and C derived from A. <br><br> 3H] dNTP Divalent cation  A2 at pH    C at pH <br><br> Template                          _      6.4   7.8    6.4   7.8 Activated DNA         dTTP         Mg++            100          100 Activated DNA         dATP         Mg++             91          85 Activated DNA         dGTP         Mg++             85          82 poly(dA).oligo(dT)10  dTTP         Mg      66.0    4.0    20    91 <br><br> (A:T = 20: 1) <br><br> poly(dA). oligo(dT)   dTTP         Mu      19.0    4. 5  6. 5  2.5 <br><br> poly(dA. oig~10 (A:T = 20:1) <br><br> poly(dT). oligo(dA)10  dATP        Mg       2.5   <1 (1       (1 <br><br> (T:A = 5:1) <br><br> poly(dT) .oligo(dA)1  dATP         Mn ++   15.5   23.0   9. 0  3. 0 <br><br> poly(dT). oligo(A)10  dATP         Mg      15.0  230.0  10.0  77. 5 <br><br> (T:A = 1:1) <br><br> poly(dT) * oligo(A) 10  dATP       Mn      41.5   75. 5 19.5  21.5 <br><br> (T:A = 1:1) <br><br> poly(dC). oligo(dG)   dGTP         Mg++     6. 0   15.5  4. 5  7. 5 <br><br> (C:G = 5:1)   10 <br><br> poly(dC). oligo(dG)10  dGTP        Mn +     6. 0  29.5   5. 0  6. 5 <br><br> (C:G  = 5: 1)                                 .    . <br><br> Values given are relative to incorporation of [ H] dTMP at pH 7. 8 on activated DNA. For A2 this was 176.5 pmol, for C 232 pmol. dATP, dCTP, dGTP and dTTP were included for assays using activated DNA, only the deoxynucleoside triphosphate stated was used in the synthetic template<br><br> initiator complex assays. Assays were for 10 minutes. The buffers and concentrations of the divalent cations were as in Materials and Methods. <br><br> this lag, but the annealing of template to initiator is only transient '  and the DNA polymerase subunit may have difficulty in stabilising the complex and the 50-70, 000 molecular weight subunit may be able to help the poly<br><br> merase subunit to overcome this. Addition of the 50-70, 000 subunit to the polymerase subunit prior to assaying with these templates had no effect on activity and it may be that a preincubation period is required before the two 3350 <br><br> Nucleic Acids Research <br><br> Figure 4 Activity of A2 and C enzymes on poly (dA). oligo (dT)jO (A:T = <br><br> 20:1) as a function of time. 50 ,ul samples were withdrawn at various times from a 0. 6 inl incubation mix at 30?C and added to 0. 5 ml 0. 1 M sodium <br><br> pyrophosphate containing 100 ,ug/ml native <span class="yellow">calf</span> thymus DNA and processed for counting in the usual manner6. The mix contained 1 mM dithiothreitol, 50 mM tris HC1, pH 7.8 10 mM MgC12, 0. 1 mM [3H] dTTP (15 cpm/pmol), 312.5 ,ug <span class="yellow">bovine</span> serum albumin, 5 Fg poly (dA). oligo (dT)  (A:T = 20:1) and enzyme protein (*-*) 3.3 units reconstituted A2, (o-o) 16. 0 units urea derived C enzyme. <br><br> subunits become fully associated. After the lag phase is over the C enzyme is still less active on these template-initiators than A enzyme . That is the A enzyme seems capable of elongating the initiator faster than the C enzyme. Experiments to determine whether the differences in rates of elongation of these template-initiator complexes are differences in processivity of the <br><br> enzymes under the different pH and divalent cation conditions are under way. <br><br> ACKNOWLEDGEMENTS <br><br> We thank the Medical Research Council for a research grant. <br><br> Communications concerning the paper should be sent to:<br><br> Dr. A. M. Holmes, Department of Biochemistry, University of Strathclyde, The Todd Centre, 31 Taylor Street, Glasgow G4 ONR, U.K. <br><br> 3351 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> (1)    Bollum, F.J. (1975). Prog. Nucleic Acid Res. Mol. Biol. 15, <br><br> 109-144. <br><br> (2)    Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1976). Eur. <br><br> J. Biochem. 62, 229-235. <br><br> (3)    Matsukage, A., Sivarajan, M. and Wilson, S.H. (1976). Biochem<br><br> istry 15, 5305-5314. <br><br> (4)    Fisher, P.A. and Korn, D. (1977). J. Biol. Chem. 252, 6528<br><br> 6535. <br><br> (5)    Fichot, O., Pascal, M., Mechali, M. and De Recondo, A, -M. <br><br> (1979). Biochim. Biophys. Acta, 561, 28-41. <br><br> (6)    Holmes, A.M., Hesslewood, I.R. and Johnston, I.R. (1974). Eur. <br><br> J. Biochemn. 43, 487-499. <br><br> (7)    Noy, G.P. and Weissbach, A. (1977). Biochim. Biophys. Acta, <br><br> 447, 70-83. <br><br> (8)    Craig, R.K. and Keir, H. M. (1975). Biochem. J. 145, 225-232. <br><br> (9)    Matsukage, A., Bohn, E.W. and Wilson, S.H. (1974). Proc. Natl. <br><br> Acad. Sci. U.S.A. 71, 578-582. <br><br> 110)   Hachmann, H.J. and Lezius, A.G. (1975). Eur. J. Biochem. 50, <br><br> 357- 366. <br><br> (11)   Brakel, C.L. and Blumenthal, A.B. (1977). Biochemistry, 16, <br><br> 3137-3143. <br><br> (12)   Momparler, R.L., Rossi, M. and Labitan, A. (1973). J. Biol. <br><br> Chem. 248, 285-293. <br><br> (13)   Yoshida, S., Konda, T. and Ando, T. (1974). Biochim. Biophys. <br><br> Acta, 353, 463-474. <br><br> (14)   Holmes, A.M., Hesslewood, I.P., Wickremasinghe, R.G. and <br><br> Johnston, I.R. (1977). Biochem. Soc. Symp. 42, 17-36. <br><br> (15)   De Jong, A., Van der Vliet, P. and Jansz, H.S. (1977). Biochim. <br><br> Biophys. Acta, 476, 156-165. <br><br> (16)   Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1975). <br><br> Nature (London) 255, 420-422. <br><br> (17)   Bollum, F. J. (1966). Procedures in Nucleic Acid Research <br><br> (Cantoni, G.L. and Davis, D.R. eds). pp 577-583, Harper and Row, New York. <br><br> (18)   Barrett, A.J. (1975). Proteases and Biological Control pp 467-482, <br><br> Cold Spring Harbor Laboratory, Cold Spring Harbor. <br><br> (19)   Richards, F. M. and Vithayathil, P.J. (1960). Brookhaven Symp. <br><br> Biol. 13, 115-134. <br><br> (20)   Lowe, P.A., and Malcolm, A.B.D. (1976). Eur. J. Biochem. 64, <br><br> 177- 188. <br><br> (21)   Hesslewood, I.P., Holmes, A.M., Wakeling, W.F. and Johnston, <br><br> I.R. (1978). Eur. J. Biochenr.. 84, 123-131. <br><br> (22)   Mechali, M. and De Recondo, A. -M. (1978). Biochim. Biophys. <br><br> Res. Comm, 82, 255-264. <br><br> (23)   Chang, L.M.S., Cassani, G.R. and Bollum, F.J. (1972). J. Biol. <br><br> Che. 247, 7718-7723. <br><br> (24)   Wickremasinghe, R.G. and Johnston, I.R. (1974). Biochim. <br><br> Biophys. Acta, 361, 37-52. <br><br> 3352 <h3>pmcA328328</h3>Mapping irradiation hybrids to cosmid and <span class="yellow">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products.
Abstract
A direct hybridization protocol is described for screening cosmid and <span class="yellow">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual Alu-PCR product before library screening. A series of <span class="yellow">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis.Images<br><br><br><br><br><br>
 Nucleic Acids Research, Vol. 19, No. 12 3315 <br><br> Mapping irradiation hybrids to cosmid and <span class="yellow">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products <br><br> Anthony P.Monaco*, Veronica M.S.Lam2, Gunther Zehetner, Gregory G.Lennon, Christal Douglas, Dean Nizetic, Peter N.Goodfellow1 and Hans Lehrach <br><br> Genome Analysis Laboratory and 'Molecular <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, 44 Lincoln's Inn Fields, London WC2A 3PX, UK and 2Department of Biochemistry, Li Shu Fan Building, Sassoon Road, University of Hong Kong, Hong Kong <br><br> Received March 12, 1991; Revised and Accepted May 16, 1991 <br><br> ABSTRACT <br><br> A direct hybridization protocol is described for screening cosmid and <span class="yellow">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual AluPCR product before library screening. A series of <span class="yellow">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis. <br><br> INTRODUCTION <br><br> The generation of <span class="yellow">human</span> DNA probes specific for individual chromosomes and subregions of chromosomes has been advanced with Alu-sequence primed polymerase chain reaction amplification (Alu-PCR, 1-3). This method specifically amplifies sequences between Alu repeats from <span class="yellow">human</span> DNA in somatic cell hybrids and <span class="yellow">yeast</span> artificial chromosomes (YACs, 4). Individual Alu-PCR products can be purified from agarose gels or ligated into plasmid vectors to screen for single copy sequences. Unique Alu-PCR products are then localized to certain chromosome regions using DNA blots of somatic cell hybrid panels. Once localized, Alu-PCR fragments can be screened against genomic libraries to isolate longer DNA fragments from the region of interest. As an alternative to this multistep process we have developed a hybridization protocol for screening of cosmid and YAC libraries directly with pools of Alu-PCR products. <br><br> Two new <span class="yellow">human</span> specific Alu primers were used to generate DNA probes from a series of irradiation-reduced hybrids containing multiple <span class="yellow">human</span> X chromosome fragments of 1-2000 kb on a <span class="red">hamster</span> chromosome background (5; P.N.G., unpublished). The Alu-PCR products were hybridized as a pool of probes to X-specific cosmid and YAC libraries, after <br><br> competitive reassociation with an excess of <span class="yellow">human</span> DNA to both the library filters and radioactively labelled Alu-PCR products. Comparisons were made between clones identified by overlapping irradiation hybrids and single copy DNA probes hybridized to the cosmid and YAC libraries. <br><br> METHODS <br><br> Two <span class="yellow">human</span> Alu sequence primers were generated which were shown to be <span class="yellow">human</span> specific; 3144 from the 3' end of Alu: 5'-GAGCGAGACTCCGTCTCAAA-3' and 2729 from the 5' end of Alu: 5'-GTGGATCACCTGAGGTCAGGAGTTC-3'. All PCR reactions were carried out with 100 ng of hybrid DNA and 0.7 ,^g of a single Alu primer in 100 d41 of 0.01 M Tris-HCl pH 8.3, 0.0015 M MgCl2, 0.05 M KCI, 200 AtM each of dNTPs, 10% dimethlysulfoxide, and 2.5 units of Cetus Taq polymerase. Reactions were 30 cycles of 94?C for 2 min, 57?C for 2 min, and 74?C for 4 min followed by a final extension time at 74?C for 9 min. Reactions products were analyzed on 1 % agarose gels and shown to contain between five and twenty fragments, with sizes ranging from 0.1 to 2.0 kb. <span class="yellow">Chinese hamster</span> DNA and no DNA PCR reactions were done to control for non-<span class="yellow">human</span> products (data not shown). <br><br> Alu-PCR products were separated from Alu oligomers over Qiagen columns, and approximately 50-100 ng were labelled by random hexamer priming (6). The radioactively labelled pool of fragments was prehybridized with 37.5 ,^g of total <span class="yellow">human</span> DNA and 12.5 tsg of <span class="red">hamster</span> DNA immobilized on a cellulose support matrix, prepared as previously described (7). Reactions were at 65?C in 1 ml of 0.75M NaCl, 0.05M sodium phosphate pH 7.2, 0.005M  EDTA, 0.1 % sodium dodecyl sulphate (SDS), 0.5 mg/ml heparin, and 100 jig/ml denatured <span class="yellow">salmon</span> sperm DNA. The cellulose was pelleted and the supernatant boiled for 2 min every 12-16 hours (three times in two days). <br><br> Cosmid and YAC library filters (Hybond N+, Amersham) were prehybridized at 42?C for 16 hours with 100 jig/ml <br><br> * To whom correspondence should be addressed at <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford OX3 9DU, UK <br><br> 1991 Oxford University Press <br><br> 3316 Nucleic Acids Research, Vol. 19, No. 12 <br><br> Chromosome X <br><br> .,.., ., < <br><br> _ ., . < ., <br><br> *2.' 1- 7-_ <br><br> _-_ 1. J <br><br> ,.1 .. - 1 , _ i - _ 1 . 1 <br><br> 1 ' 4 - -1 + <br><br> 1 1 L L . i <br><br> . 1 . _ 1  <br><br> 1 1 s. 1 i <br><br> 1 i ;. --r' F 1 -<br><br> 1 -1-l <br><br> j i-_1. 1 <br><br> -   _ 1 .. <br><br> s- S . i n -. <br><br> .. : ,1' ,. -1 _ E I i ? <br><br> ,. . _ _ _ <br><br> G-<br><br> . <br><br> *.-.,., z <br><br> _ <br><br> _ , <br><br> z ;- [X1 <br><br> 21 , In <br><br> 27  38 45   48  54  74   86 <br><br> I  1      I <br><br> I <br><br> I <br><br> 107 <br><br> MD <br><br> I <br><br> :I                                                       I  <br><br> I        I <br><br> I I <br><br> Figur 2a and 2b: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 48 to duplicate copies of 22 x22cm filters containing 9216 <span class="yellow">human</span> X chromosome cosmids (8). 2c: Hybridization of Alu-PCR products generated with Alu primer 3144 from an independent hybrid (54) to a third identical cosmid filter. <br><br> Figure 1: A schematic diagram of the <span class="yellow">human</span> X chromosome alongside the approximate cytogenetic location of fragments identified in nine irradiation hybrids (numbers across the top). The <span class="yellow">human</span> X fragments were identified by hybridization of 27 known DNA markers (indicated by a black line; P.N.G, unpublished) or by cosmids in common with unique X chromosome probes in the reference library database (open boxes and Table 1). The size of the lines and boxes relate to the best cytogenetic location of the probes used according to <span class="yellow">Human</span> Gene Mapping 10.5 (14) and does not indicate the physical extent of the irradiation hybrid fragments. <br><br> denatured and sheared total <span class="yellow">human</span> DNA in 50% formamide, 4XSSC, 0.05 M sodium phosphate pH 7.2, 0.001 M EDTA, 10% dextran sulphate, 1.0% SDS, 50 isg/ml denatured <span class="yellow">salmon</span> sperm DNA and OxDenhardt's solution. The radioactively labelled Alu-PCR products were denatured and added to fresh <br><br> hybridization solution without <span class="yellow">human</span> DNA competitor at 1 x 106 <br><br> cpm/ml and hybridized at 42?C for 16 hours. Filters were washed in 0.1 xSSC and 1.0% SDS, twice at room temperature and twice at 65?C for 30 min each and exposed to Kodak X-OMAT film for 2-3 days at -70?C with an intensifying screen. <br><br> For each hybridization, two sets of duplicate cosmid filters were used from the ICRF reference library system (8), each containing 9216 flow-sorted <span class="yellow">human</span> X chromosome cosmid clones or approximately 2 chromosome equivalents on a 22 x 22 cm filter (9). The coordinates of signals positive on duplicate cosmid filters were entered into the reference library database (G. Z, unpublished) using a digitizing tablet. For the X chromosome specific YAC library (A.P.M. and H.L., unpublished), about 420 YAC colonies were spotted manually onto filters from 96 well microtiter dishes using a 96 prong device. After growth on selective media for 3 days, YAC filters were processed for hybridization as previously described (10). <br><br> RESULTS <br><br> A panel of 195 X chromosome irradiation hybrids was constructed (50,000 rads) and characterized by DNA hybridization using 27 X chromosome markers and flourescence in situ hybridization using total <span class="yellow">human</span> DNA as probe (Benham et al., 1989; P.N.G., unpublished). This analysis indicated that the hybrids contained multiple small fragments (4-10 fragments of 1000-5000 kb each) with a preferential retaining of centromere sequences (90%). From this panel, nine irradiation hybrids were chosen that contained less than five different regions by DNA probe hybridizations, mostly from the short arm of the X chromosome (Fig 1). All nine hybrids were used in PCR reactions with 3'-Alu primer 3144 and two were used with 5'-Alu <br><br> Figure 3: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 54 to a filter containing 420 YAC clones from the <span class="yellow">human</span> X chromosome. The positive YAC was also identified in a separate hybridization with the DMD probe P20 (12). <br><br> primer 2729. Example hybridizations to a <span class="yellow">human</span> X chromosome cosmid filter in Fig 2 shows the intensity and reliability of positive clones identified on duplicate filters with Alu-PCR products from the same irradiation hybrid (48) and the independence of clones identified with Alu-PCR products from a different hybrid (54). Fig 3 shows a single positive YAC clone after hybridization of Alu-PCR products from irradiation hybrid 54 to a filter containing about 420 YAC clones specific for the <span class="yellow">human</span> X chromosome. <br><br> The total number of cosmids identified with each pool of AluPCR products for each hybrid is shown in Table 1. From the average number of cosmids identified (24) in four chromosome equivalents screened and the estimated average DNA content in each hybrid (3000-15000 kb), the Alu-PCR products generated by a single primer were calculated on average to be 300-1500 kb apart, similar to published estimates for this method (1,2). Only 3-4 cosmids were identified in common using Alu-PCR products generated with either 3' or 5' Alu primers (3144 or 2729) from two hybrids (38 and 45). This shows that separate products were amplified with the two Alu primers since they prime synthesis from opposite ends of the Alu consensus sequence and Alu sequences are oriented in the genome either head to head, <br><br> A <br><br> B <br><br> C <br><br> El <br><br> I0 <br><br> Nucleic Acids Research, Vol. 19, No. 12 3317 <br><br> Table 1. Cosmids identified by hybrids and unique probes <br><br> hybrids       21    27     38    38    45    45    48     54    74    86     107   unique <br><br> primer 3144  3144  3144  2729  3144  2729   3144  3144  3144  33144 3144   probes <br><br> 21     3144  59                                                                   1 27     3144   7    14                                                             2 38     3144   3     1    28                                                       2 38     2729   0     0     4     25                                                0 45     3144  13     2      2     0    32                                          3 45     2729   1     0      0     0     3    31                                    0 48     3144  10     2      4     0     2     0    28                              3 54     3144  11     3     0      0     5     0     5     30                       2 74     3144   5     0     0      0     4     0     5     12    20                 3 86     3144   3     1     0      0     4     1     3      4     3    17           0 107     3144   1     0      1     1     0     0     3     0      0     0    13    0 <br><br> tail to tail or head to tail relative to each other. Therefore, by using the 3'- and 5'-Alu primers in separate PCR reactions with the same hybrid DNA, the total number of products and cosmid clones identified was almost doubled. <br><br> Table 1 also indicates how many cosmids were identified by Alu-PCR products from other hybrids, and 16 cosmids previously identified with unique DNA probes in the reference library database. As can be seen in Fig 1 and previous irradiation hybrid analysis, there is a preferential retention of centromere sequences (2,11). However, there were no cosmids identified in common from all the hybrids positive with centromere sequences. This is probably due to a paucity of Alu repeats in the correct orientation in alphoid centromere sequences and thus few or no Alu-PCR products would be amplified from the centromere. <br><br> Cosmids identified in common by several irradiation hybrids (Table 1) were most likely from regions of overlap outside the centromere area as shown by the previous DNA probe characterization (Fig 1). The overlap regions between hybrids were also seen by 16 cosmids (Table 1, Fig 1) that were hybridization targets of unique DNA markers in the reference library database that mapped in independent experiments to the overlap region. At least for several cosmids this showed that the Alu-PCR products identified target cosmids that were definitely from the expected region contained in the hybrids. For example, hybrids 21 and 54 were both shown to contain part of the Duchenne muscular dystrophy (DMD) locus (Fig 1; P.N.G., unpublished) and had 11 cosmids in common, including one identified by the probe P20 from the deletion hotspot region of the DMD gene (12). In Fig 3 the hybridization of Alu-PCR products from hybrid 54 identified a YAC clone which was also positive for the DMD probe P20 (data not shown). This method also identified fragments in the hybrids that were not seen in the initial DNA characterization (Fig 1 and Table 1). Since the 27 DNA probes were not close enough to each other along the chromosome to detect all possible hybrid fragments (1000-5000 kb), many regions would have been untested. For example, AluPCR products from hybrids 45 and 48 identified several cosmids, also seen by the cDNA for chronic granulomatous disease gene (CYBB, 13) in Xp2 1.1, although this region was not tested in the original hybrid characterization. <br><br> DISCUSSION <br><br> The direct hybridization of Alu-PCR products from somatic cell hybrids to genomic libraries can bypass gel purification or ligation of fragments into plasmid vectors and individual analysis for single copy sequences. Hybridization of Alu-PCR products as <br><br> a pool to ordered array libraries such as the flow-sorted X chromosome cosmid library (9), allows the direct comparison of overlapping hybrids to pinpoint cosmids most likely to be from the region of interest. In conjunction with the reference library database (G.Z, unpublished) with 183 X chromosome probe hybridization entries, cosmids identified with both Alu-PCR products and uniquely mapped X probes immediately map them to the region of interest and proove that the method has worked. Similar hybridization experiments using Alu-PCR products from four overlapping irradiation hybrids identified four cosmids in common that mapped to the region of overlap by independent experiments (F.Muscatelli, A.P.M., P.N.G., H.L. and M.Fontes, in preparation). Since only 27 probes from the X chromosome were used to initially characterize the hybrids and the length of individual <span class="yellow">human</span> fragments in the irradiation hybrids is about 1000-5000 kb, many regions could have been undetected in the original analysis. The direct hybridization of Alu-PCR products to the cosmid reference library detected such fragments since they identified cosmids in common with uniquely mapped probes in regions not tested originally (Table 1 and Fig 1). This should prove to be a sensitive and efficient method to determine content and overlap of irradiation hybrids in conjunction with DNA blot hybridization. However, since the exact length of the <span class="yellow">human</span> DNA fragments for each hybrid and the spacing of Alu-PCR products along the chromosome is not known, it is difficult to directly correlate the number of target cosmids to the DNA content of the hybrids. <br><br> The direct hybridization of Alu-PCR products from irradiation or somatic cell hybrids to total genomic YAC libraries will be especially useful to construct long range YAC contigs from specific subregions of chromosomes. The dissection of a total genomic YAC library by this method may be more efficient than generating chromosome specific YAC libraries from somatic cell hybrids (usually a haploid <span class="yellow">human</span> chromosome on a diploid or greater <span class="yellow">rodent</span> background) or flow-sorted chromosomes, because of the low transformation efficiency of <span class="yellow">yeast</span>. <br><br> ACKNOWLEDGEMENTS <br><br> We would like to thank Gert-Jan Van Ommen for the probe P20 and Stuart Orkin for the CYBB cDNA probe. A.P.M is supported in part by a research fellowship from the Muscular Dystrophy Association of America. V.M.S.L. is supported in part by the Medical Research Grant of the University of Hong Kong. Reference library filters and cosmids identified by Alu-PCR products can be requested from the Reference Library Database, ICRF, 44 Lincoln's Inn Fields, London WC2A 3PX, U.K. <br><br> 3318 Nucleic Acids Research, Vol. 19, No. 12 <br><br> REFERENCES <br><br> 1. Nelson, D.L., Ledbetter, S.A., Corbo, L., Victoria, M.F., Ramirez-Solis, <br><br> R., Webster, T.D., Ledbetter, D.H., and Caskey, C.T. (1989) Proc. Natl. Acad. Sci. USA, 86:6686-6690. <br><br> 2. Ledbetter, S.A., Nelson, D.L., Warren, S.T., and Ledbetter, D.H. (1990) <br><br> Genomics, 6:475-481. <br><br> 3. Cotter, F.E., Hampton, G.M., Nasipuri, S., Bodmer, W.F., and Young <br><br> B.D. (1990) Genomics, 7:257-263. <br><br> 4. Burke, D.T., Carle, G.F., and Olson, M.V. (1987) Science, 236:806-812. 5. Benham, F., Hart, K., Crolla, J., Bobrow, M., Francavilla, M., and <br><br> Goodfellow, P.N. (1989) Genomics, 4:509-517. <br><br> 6. Feinberg, A.P., and Vogelstein, B. (1984) Anal. Biochem., 137:266-267. 7. Hochgeschwender, U., Sutciffe, J.G., and Brennan, M.B. (1989) Proc. Natl. <br><br> Acad. Sci. USA, 86:8482-8486. <br><br> 8. Lehrach, H., Drmanac, R., Hoheisel, J., Larin, Z., Lennon, G., Monaco, <br><br> A.P., Nizetic, D., Zehetner, G., and Poustka, A. (1990) In Davies, K.E. & Tilghman, S.M. (eds.), Genome Analysis Volume 1: Genetic and Physical Mapping. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, pp. 39-81. <br><br> 9. Nizetic, D., Zehetner, G., Monaco, A.P., Gellen, L., Young, B.D., and <br><br> Lehrach, H. (1991) Proc. Natl. Acad. Sci. USA, 88:3233-3237. <br><br> 10. Larin, Z. and Lehrach, H. (1990) Genet. Res. Camb., 56:203-208. <br><br> 11. Cox, D,R., Pritchard, C.A., Uglum, E., Casher, D., Koborl. J. and Myers, <br><br> R.M. (1989) Genomics, 4:397-407. <br><br> 12. Wapenaar, M.C., Kievits, T., Hart, K.A., Abbs S., Blonden, L.A.J., <br><br> denDunnen, J.T., Grootscholten, P.M., Bakker, E., Verellen-Dumoulin, C., Bobrow, M., vanOmmen, G.J.B., and Pearson, P.L. (1988) Genomics, 2:101-108. <br><br> 13. Royer-Pokora, B., Kunkel, L.M., Monaco, A.P., Goff, S.C., Newburger, <br><br> P.E., Baehner, R.L., Cole F.S., Curnette J.T., and Orkin, S.A. (1986) Nature, 322: 32-38. <br><br> 14. Davies, K.E., Mandel, J.L., Monaco, A.P., Nussbaum, R.L. and Willard, <br><br> H.F. (1990) Cytogenet. Cell Genet. 55: 254-313. <h3>pmcA2562362</h3>Host immunity in the protective response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span>
Abstract
Background
We performed initial cell, cytokine and complement depletion studies to investigate the possible role of these effectors in response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span> in a susceptible BALB/c <span class="yellow">mouse</span> model of infection.<br><br>Results
While protection with heat-killed bacilli did not result in sterilizing immunity, limited protection was afforded against an otherwise lethal infection and provided insight into potential host protective mechanisms. Our results demonstrated that <span class="yellow">mice</span> depleted of either B cells, TNF-α or IFN-γ exhibited decreased survival rates, indicating a role for these effectors in obtaining partial protection from a lethal challenge by the intraperitoneal route. Additionally, complement depletion had no effect on immunoglobulin production when compared to non-complement depleted controls infected intranasally.<br><br>Conclusion
The data provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br><br><br>Background
<span class="yellow">Burkholderia mallei</span>, the etiologic agent of glanders, is a gram-negative, capsulated, non-motile, facultative intracelluar bacterium. Most known members of the Burkholderiaceae are resident in the soil; however, <span class="yellow">B. mallei</span> is thought to be an obligate mammalian pathogen. <span class="yellow">Horses</span> are highly susceptible to infection and considered the natural reservoir for infection, although <span class="yellow">mules</span> and <span class="yellow">donkeys</span> are also susceptible [1]. Clinically, glanders in solipeds can present as either a chronic (<span class="yellow">horses</span>) or acute (<span class="yellow">mules</span> and <span class="yellow">donkeys</span>) form. Naturally acquired <span class="yellow">human</span> infection with <span class="yellow">B. mallei</span>, although not seen in the United States since 1945, has occurred rarely and sporadically among laboratory workers and those in direct contact with infected animals [2]. However, glanders is endemic among domestic animals in Africa, Asia, the Middle East, and Central and South America. The course of infection is dependent on the route of exposure. Direct contact with the skin can lead to a systemic infection. Inhalation of aerosol or dust containing <span class="yellow">B. mallei</span> can lead to septicemic, pulmonary, or chronic infections of the muscle, liver and spleen. The disease has a 95% case fatality rate for untreated septicemic infections and a 50% case fatality rate in antibiotic-treated individuals [3].
There is no <span class="yellow">human</span> or animal vaccine available for glanders, and development of a partial or fully protective adaptive host response to the organism has not been well-defined. Previous studies with <span class="yellow">B. mallei</span> and the host response have shown that a mixed immune response consisting of both Th1 and Th2-associated cytokines with a predominant IgG1 subclass does not correlate with protection [4]. Additional studies with passive transfer of monoclonal antibodies specific for <span class="yellow">B. mallei</span> have correlated with early protection from infection [5]. Recent studies have also shown the Th1 cytokine IL-12 to mediate partial protection to non-viable <span class="yellow">B. mallei</span>-vaccinated <span class="yellow">mice</span> [6]. Thus, full correlates of protection mediated by the adaptive immune system against <span class="yellow">B. mallei</span> remain to be fully elucidated.
In this series of studies, we sought to address the impact of depletion of the major effector lymphoid cell populations (B220+ B cells, CD4+ or CD8+ T cells) and key pro-inflammatory/Type 1 cytokines (IFN-γ or TNF-α) on survival in BALB/c <span class="yellow">mice</span> vaccinated with heat killed (HK) bacilli followed by an intraperitoneal (i.p.) challenge with live organism. In addition, studies investigating the effect of complement on opsonization of organism and antibody production were assessed. Heat killed bacteria were used as a model of vaccination to allow evaluation of <span class="yellow">B. mallei</span> specific immune responses. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br>Results
Heat-killed <span class="yellow">B. mallei</span> vaccination mediates partial protection from lethal challenge
To begin to address this issue in an animal model of acute infection, we established that immunologically naive BALB/c <span class="yellow">mice</span> challenged i.p. with 2 × 107 CFU resulted in death by day 4–6, while i.p. immunization with 1 × 105 heat killed (HK) bacteria provided partial protection against a subsequent challenge. Two independent experiments resulted in similar findings of 40% survival for HK-vaccinated <span class="yellow">mice</span> with a mean survival time (MST) of 8 days versus 4 days in naïve <span class="yellow">mice</span> (Fig. 1). The administration of vaccines for <span class="yellow">B. mallei</span> during an outbreak would mandate relatively rapid onset of protection for <span class="yellow">human</span> or veterinary use. Based on non-routine use and vaccine implementation in the course of an outbreak, a 14 day window was chosen for assessment of protection. Our results indicate that HK vaccination can afford partial protection to an otherwise lethal challenge of <span class="yellow">B. mallei</span> by the i.p. route.<br><br>Effects of cell depletion on HK-vaccinated survival
To dissect the cellular basis for protection mediated by HK vaccination, 13 days after immunization with HK bacteria (day -1), and at day of challenge, <span class="yellow">mice</span> were dosed with antibodies to deplete CD4+, CD8+ or B220+ cells. Antibody depletion of CD4+, CD8+, or B220+ cells in these <span class="yellow">mice</span> was confirmed by flow cytometric analysis with depletion efficiencies for CD4, CD8, and B220 populations at 99.7%, 96%, and 95%, respectively, relative to <span class="yellow">mice</span> treated with isotype control monoclonal antibodies (data not shown). Our results demonstrated decreased survival rates in B220 (p = 0.3418), CD4+ (p = 0.5417) and CD8+ (p = 0.4684) antibody depleted <span class="yellow">mice</span>, compared to isotype control antibody, a finding that indicated a possible role for vaccine induced antibody production. When challenged with 2 × 107 CFU/<span class="yellow">mouse</span> by the i.p. route, loss of T cells resulted in reduced survival (50%) relative to the non-specific isotype control (Fig. 2). In contrast to the loss of T cells, depletion of B220+ cells resulted in 100% mortality relative to the non-specific isotype control (Fig. 2). To further evaluate the necessity of these effector cells in providing protection following HK vaccination, relatively resistant C57BL/6 <span class="yellow">mice</span>, deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) were subjected to an identical HK vaccination and challenge regimen. Mature, B-cell-deficient <span class="yellow">mice</span> demonstrated a 50% decreased survival (p = 0.0888) compared to the wild-type <span class="yellow">mice</span> with an MST of 35.5 days (Fig. 3). CD4-/- and CD8-/- <span class="yellow">mice</span> exhibited a 60% (p = 0.1343) and 0% reduced survival, respectively (Fig. 3).<br><br>Effects of cytokine depletion on HK vaccination
Similar studies were performed to determine the role of IFN-γ or TNF-α in acute infection in BALB/c <span class="yellow">mice</span> immunized with HK bacteria. Six hours before challenge, <span class="yellow">mice</span> were dosed with antibodies that neutralize IFN-γ or TNF-α. Individual depletion of either TNF-α (p = 0.0145) or IFN-γ (p = 0.0446) resulted in 100% mortality with an MST of 3 and 2 days, respectively, compared to the HK-vaccinated isotype control <span class="yellow">mice</span> (Fig. 4). In contrast, 40% of HK-vaccinated, isotype control <span class="yellow">mice</span> survived to at least 12 days post-challenge (Fig 4). To further evaluate the host TNF-α response during an established <span class="yellow">B. mallei</span> chronic infection, we infected 12 BALB/c <span class="yellow">mice</span> by the i.p. route with 1 × 106 CFU <span class="yellow">B. mallei</span>. One animal was terminally ill on day 37 post-infection. On day 42 post-infection, the remaining 11 <span class="yellow">mice</span> were dosed with either anti-TNF-α (n = 6), or control mAb (AFRC Mac 49) (n = 5). No further deaths were observed in the control mAb-treated <span class="yellow">mice</span>. Rapid mortality was observed in the anti-TNF-α-treated group, with all <span class="yellow">mice</span> dying within 7 days of treatment (p = 0.0023) relative to the isotype-treated controls (Fig. 5).<br><br>J774A.1 uptake of serum treated <span class="yellow">B. mallei</span>
Complement mediated uptake assays were performed to evaluate opsonization. Results indicated enhanced bacterial uptake in J774A.1 phagocytes inoculated with serum treated <span class="yellow">B. mallei</span> (p = .0082), compared to <span class="yellow">B. mallei</span> alone, while heat-inactivated serum produced uptake percentages similar to those prior to serum addition (Fig. 6). Taken together, these results imply an active role for complement components in the uptake of organism by macrophages.<br><br>Immunoglobulin production in HK vaccinated BALB/c <span class="yellow">mice</span>
We further characterized the ability of HK vaccination to induce a predominant IgG isotype by determining IgG2a/IgG1 ratios in i.p. and i.n. vaccinated BALB/c <span class="yellow">mice</span>. Pre (day 14 post vaccination) and post (day 2 post infection) exposure serum samples were obtained and evaluated for IgG isotype concentrations (Table 1). No appreciable differences in IgG pre-exposure levels were seen when comparing i.n. to i.p. vaccination. In addition, cobra venom factor-treated animals showed no significant differences to non-cobra venom factor-treated animals in IgG pre-exposure (challenge) levels. Conversely, isotype switching in the cobra venom factor treated animals was enhanced in post-exposure serum IgG2a (Table 1).<br><br>
Discussion
Recent studies have shown a key role in protection from lethal challenge for IFN-γ in non-vaccinated <span class="yellow">mice</span> from either NK and/or NKT cells following experimental exposure to <span class="yellow">B. mallei</span> and <span class="yellow">B. pseudomallei</span> [7,8]. A similar protective role in the innate response to infection has been demonstrated for TNF-α in <span class="yellow">B. pseudomallei</span> infection [8]. The studies presented here are consistent with the essential role of these factors in the relative levels of protection conferred by vaccination with heat-killed <span class="yellow">B. pseudomallei</span> and would appear to be viable early markers for protection from lethal acute infection [9]. Currently, there are no fully protective vaccines against <span class="yellow">B. mallei</span> or <span class="yellow">B. pseudomallei</span> in a <span class="yellow">murine</span> model, particularly for the sensitive BALB/c versus C57BL6 models. Previous studies have also demonstrated that both the humoral and cell-mediated arms are essential for protection from <span class="yellow">B. pseudomallei</span> infection [10]. Thus, loss or reduction of TNF-α and IFN-γ levels result in significantly reduced survival rates, substantiating previous reports of the role of these factors in protection against <span class="yellow">B. mallei</span> [7]. Moreover, we demonstrate a role for sustained TNF-α production in the maintenance of host survival throughout the course of <span class="yellow">B. mallei</span> infection. <span class="yellow">Mice</span> with an established <span class="yellow">B. mallei</span> chronic infection rapidly lost the ability to control the growth of the bacillus upon neutralization of TNF-α. This would suggest a potential role for TNF-α in the maintenance of productive granulomas which may limit the spread of bacteria in chronically infected hosts, or, alternatively, in direct or indirect microbicidal or bacteriostatic activities at the sites of infection. Additional studies are underway to determine more precisely the role of TNF-α in host protection to <span class="yellow">B. mallei</span>.
Multiple innate and adaptive cell types may contribute to the production of IFN-γ in response to infection with <span class="yellow">B. mallei</span> following vaccination. Our results with individual depletion of CD4+ and CD8+ T cells suggests that both cell types may compensate for the functional loss of the other effector cell type in the production of this key cytokine. The effector role for IFN-γ in mediating protection against <span class="yellow">B. mallei</span> may include both immunoregulatory and non-regulatory functions. Regardless, the requirement of IFN-γ, as demonstrated by administration of neutralizing antibody prior to infection, indicates that stimulation of IFN-γ response is a desirable goal for a <span class="yellow">B. mallei</span> vaccine.
Similarly, B220-positive cells appear to play a role in protection following vaccination with heat-killed <span class="yellow">B. mallei</span>. Interestingly, this protective immunity, occurring in other intracellular pathogens, is not exclusively dependent on B cells [11]. Passive protection has been demonstrated against acute Burkholderia infection by monoclonal antibodies [5,12]. Protection against <span class="yellow">B. pseudomallei</span> infection by anti-LPS, capsular polysaccharide and proteins has been short-lived, suggesting that antibody production offers limited protection in the initial stages of infection by an as-yet-undefined mechanism [12]. We have shown that following depletion of B220+ cells, survival rates decreased as much as 100% relative to non-depleted controls and individual CD4/CD8-depleted <span class="yellow">mice</span> via the intraperitoneal route. Results from C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) substantiate the requirement for B-cell involvement by evidence of μMT and CD4-/- decreased survival. The lack of an effective CTL response to vaccination did not appear to alter survival in what would appear to be a CD4/B-cell (humoral)-driven response. In CD4-deficient <span class="yellow">mice</span>, we have the additional potential variable that a CD4-dependent antibody response might also be inhibited during the vaccination phase relative to <span class="yellow">mice</span> treated with antibody immediately prior to and during the early phases of infection. Although not statistically significant, we did observe a decrease in survival in μMT (mature B cell) deficient <span class="yellow">mice</span> as early as day 9 post challenge, whereas CD4-deficient <span class="yellow">mice</span> produced similar results at day 32 post challenge, indicating a role for B cells independent of CD4 T cell help, perhaps through a T-independent mechanism of antibody production. Although CD8-/- C57BL/6 demonstrated no decreased survival in our HK-vaccinated model, a lack of potential endogenous protein production by HK <span class="yellow">B. mallei</span> may have contributed to limited MHC-I presentation.
Complement associated studies revealed increased J774A.1 uptake of serum-treated <span class="yellow">B. mallei</span>. Complement-mediated uptake studies of <span class="yellow">B. pseudomallei</span> by polymorphonuclear leukocytes (PMNs) suggest that capsule production contributes to resistance of phagocytosis by reducing C3b bacterial deposition [13]. Previous studies have demonstrated that a polysaccharide capsule is present in <span class="yellow">B. mallei</span>, [14,15] although in the present study enhanced uptake with serum-treated <span class="yellow">B. mallei</span> was observed. Intracellular survival assays of complement mediated uptake of organisms were not performed in the present study, thus, the role of complement opsonization on intracellular survival is not fully resolved. Previous reports have demonstrated the ability of <span class="yellow">B. mallei</span> to survive within macrophage without the aid of serum coating organisms [16]. Conversely, the idea of antibody mediated opsonization to facilitate macrophage activation and clearance of intracellular organisms may offer support to the role of B cells in an effective immune response. A possible protective mechanism may include HK vaccination induced production of opsonizing antibodies which may aid in complement mediated uptake, thereby limiting the initial bacterial threshold below a lethal level.
Immunoglobulin responses to HK vaccination resulted in modest levels of IgG1 following 2 weeks post vaccination, while post-exposure levels were indicative of efficient class switching to a favorable IgG2a isotype. Importantly, cobra venom factor treatment of animals at time of vaccination did not alter their ability to produce immunoglobulin. In fact, cobra venom factor treated animals resulted in higher IgG2a levels when compared to non-treated. Complement activation can modulate both the primary and secondary immune responses and has been shown to enhance secondary immune responses to vaccination [17]. The current results suggest that cobra venom factor treatment may affect the modulation of the immune response to <span class="yellow">B. mallei</span> infection through B cell activation and/or memory B cell generation.<br><br>Conclusion
In summary, our results provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. Understanding and defining the role of B cells in adaptive <span class="yellow">B. mallei</span> immunity will likely be fundamental to the design of an efficacious vaccine and important goals of future research.<br><br>Methods
Bacterial strain and <span class="yellow">mice</span>
<span class="yellow">B. mallei</span> strain ATCC 23344 (China 7) was cultured on Luria-Bertani agar supplemented with 4% glycerol (LB+4%G) agar plates for 48 h at 37°C. Isolated colonies were sub-cultured to LB+4%G broth, and cultures were incubated at 37°C until optical density readings at 600 nm (OD600) reached an exponential phase of growth. Bacteria were pelleted by centrifugation, washed and re-suspended in sterile 1× phosphate-buffered saline (PBS, pH 7.4) to obtain the desired CFU/ml. To obtain HK inoculums, bacterial suspensions were incubated at 85°C for 3 h and stored at 4°C until use. The absence of live <span class="yellow">B. mallei</span> organisms in the HK preparations was confirmed after plating 10% of the total inoculums (v/v) and incubating these at 37°C for 48 h. All procedures were performed under a class II biosafety cabinet in a biosafety level 3 laboratory. Female, 6- to 8-week-old, BALB/c <span class="yellow">mice</span> (n = 5–7) were obtained from Harlan Sprague Dawley, Inc. (Indianapolis, Indiana). Female, 6- to 8-week-old, C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) and CD8 T-cells (CD8-/-) and wild-type <span class="yellow">mice</span> were obtained from The Jackson Laboratory (Bar Harbor, Maine).<br><br>Vaccination and challenge
BALB/c and C57BL/6 <span class="yellow">mice</span> were grouped and vaccinated with 0.5 μg of HK <span class="yellow">B. mallei</span> (without adjuvant) by i.p. injection using a 25-gauge syringe. Two weeks post HK vaccination <span class="yellow">mice</span> were injected i.p. with 2 × 107 CFU/100 μl of live <span class="yellow">B. mallei</span> (~20 LD50) [18]. Complement depleted animals were challenged with 2.5 × 104 CFU/50 μl (~0.25 LD50) by intranasal (i.n.) route. Aliquots from the inoculums were plated to confirm the infecting dose. All procedures and animal protocols used in this study were approved by the Biosafety and IACUC committees at UTMB and conducted in either BSL-3 or ABSL-3 laboratories.<br><br>Cell and cytokine depletions
Acute in vivo cell/cytokine depletion was performed with monoclonal <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD4 (GK1.5), CD8α (53-6.7) or B220 (RA3-6B2) obtained from R&D Systems, Inc. (Minneapolis, MN) by methods similar to those we have previously described [19]. Functional grade purified <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> interferon-gamma (IFN-γ, AN-18) was obtained from eBioscience (San Diego, CA) and purified anti-<span class="yellow">mouse</span> tumor necrosis factor (TNF-α, MP6-XT3) from BD Pharmingen (San Diego, CA). IFN-γ and TNF-α antibodies were injected i.p. 6 h prior to challenge, 200 μg per <span class="yellow">mouse</span> in 200 μl PBS or at later time points as indicated. <span class="yellow">Rat</span> IgG isotype control was obtained from Southern Biotech (Birmingham, AL) and administered i.p. on day of challenge, 200 μg/<span class="yellow">mouse</span>. <span class="yellow">Rat</span> anti-<span class="yellow">mouse</span> CD4, CD8α and B220 were injected i.p. twice, 1 day prior to challenge and on day of challenge, with an equivalent dosage sufficient to deplete T or B cells from 6 × 108 bone marrow cells per injection. The efficiency of depletion at time of infection for CD4+, CD8+, and B220+ cells was confirmed by flow cytometry analysis immediately prior to infection.<br><br>Complement depletion with cobra venom factor
<span class="yellow">Mice</span>, six to seven per group, were vaccinated i.p. with 1 × 105 CFU of nonviable <span class="yellow">B. mallei</span> cell preparation in a total volume of 0.1 ml. Two weeks later, 24 h and 1 h before challenge, complement depleted <span class="yellow">mice</span> were treated i.p. with 12.5 units total cobra venom factor (Quidel Corporation Speciality Products, San Diego, CA) in 0.1 ml of PBS. Complement depletion was confirmed prior to challenge by micro-titer hemolytic complement activity (CH50) assay as previously described [20].<br><br><span class="yellow">B. mallei</span> J774A.1 uptake assays
J774A.1 cells were seeded (5 × 105) onto Corning costar 24 well plates (Corning, NY) with DMEM and incubated overnight at 37°C with 5% CO2. Bacterial suspensions were incubated at 37°C for 45 minutes supplemented with 2% <span class="yellow">mouse</span> serum from Sigma-Aldrich (St. Louis, MO.), heat inactivated <span class="yellow">mouse</span> serum (56°C 30 minutes), or bacteria alone and then added at an MOI of 10:1 to J774A.1 cells in triplicate. Inoculated wells were centrifuged at 800 g for 2 minutes and incubated for 2 hours at 37°C with 5% CO2 followed by a PBS wash (×2) and 2 hour incubation with 250 μg/ml kanamycin. Wells were washed twice with PBS and lysed with 0.1% Triton X-100, followed by serial 10-fold dilutions plated on LBG plates and incubated at 37°C for 2 days. Colony forming units were enumerated and uptake expressed as a percentage of initial inoculating dose ± SEM.<br><br>Antibodies and flow cytometry
Flow cytometric analysis was performed on 0.1-ml blood samples transferred to micro centrifuge tubes containing 90 μl of acid citrate dextrose (ACD) solution. Red blood cells were lysed using ACK-lysing buffer (Biosource International, Inc., Camarillo, CA) according to the manufacturer's instruction. Antibodies used for analysis of surface markers included: FITC-conjugated <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD45R/B220 (RA3-6B2, BD Pharmingen San Diego, CA) for B cells; FITC-conjugated <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD8α (53-6.7) and CD4 (GK1.5, BD Pharmingen, San Diego, CA) for CD8+ or CD4+ cells, respectively. Samples evaluated for CD4+ and CD8α+ cells were also incubated with biotin-conjugated <span class="red">hamster</span> anti-<span class="yellow">mouse</span> CD3e (145-2C11) monoclonal antibody (BD Pharmingen, San Diego, CA) and subsequently with streptavidin APC Cy7. Isotype-matched, non-specific controls were assayed in parallel (BD Pharmingen, San Diego, CA). Surface staining was performed according to previously published protocols [21]. Following cell staining, the samples were fixed with 2% buffered paraformaldehyde overnight prior to analysis by flow cytometry. Samples were analyzed using a FACSCalibur flow cytometer with BD CellQuest Pro software.<br><br>Antibody assays
Immunoglobulin subclass IgG1 and IgG2a titers in <span class="yellow">mice</span> were determined by a whole bacterial cell ELISA performed in 96-well, Immulon 2 HB, round-bottom plates (Dynex Technologies). <span class="yellow">B. mallei</span> antigen was diluted in 0.1 M carbonate buffer (pH 9.5) and 50 μl of diluted cells placed into wells. Plates were stored overnight at 4°C. The plates were washed with washing solution (1× PBS, 0.05% Tween 20), and incubated with 100 μl of blocking solution (1× PBS, 1% <span class="yellow">bovine</span> serum albumin, 0.05% Tween 20) for 1 h at 37°C. Dilutions of <span class="yellow">mouse</span> sera were made with blocking solution in duplicate and plates were incubated for 1 h at 37°C. Following incubation, plates were washed and 50 μl of anti-Ig-<span class="yellow">horseradish</span> peroxidase subclass conjugate, diluted accordingly to manufacturer's instructions (Southern Biotechnology Associates, Inc. Birmingham, Ala.), was added to each well and incubated for 1 h at 37°C. After washing, 50 μl of 2,2'-azino-di-(3-ethylbenzthizoline)-6-sulfonate (ABTS) peroxidase substrate (KPL, Inc., Gaithersburg, Maryland) was added to each well and plates incubated for 25 min at room temperature. The amount of bound antibody was determined colorimetrically by absorbance at 405 nm.<br><br>Statistical analysis
Survival curves were calculated by Kaplan Meier survival analysis with log-rank tests between groups using GraphPad Prism (V.4.03 for windows). Statistical analysis was generally performed with the paired Student's t-test. P value ≤ 0.05 was considered significant.<br><br>
Abbreviations
HK: Heat-killed; i.p.: intraperitoneal; i.n.: intranasal.<br><br>Authors' contributions
GCW designed and conducted experiments and drafted the manuscript. BMJ carried out the immunoassays and animal work. SP provided analysis of data and contributed to design and animal work. RAL participated in the generation and analysis of chronic TNF-α data. DME conceived the study, and participated in its design and coordination and helped to draft the manuscript. AGT participated in the bacterial work and drafting of the manuscript. All authors read and approved the final manuscript.<br><br>
</body></html>