
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>humans</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="blue">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1526545</h3>B lymphocyte stimulator (BLyS) isoforms in systemic lupus erythematosus: disease activity correlates better with blood leukocyte BLyS mRNA levels than with plasma BLyS protein levels
Abstract
Considerable evidence points to a role for B lymphocyte stimulator (BLyS) overproduction in <span class="yellow">murine</span> and <span class="yellow">human</span> systemic lupus erythematosus (SLE). Nevertheless, the correlation between circulating levels of BLyS protein and disease activity in <span class="yellow">human</span> SLE is modest at best. This may be due to an inadequacy of the former to reflect endogenous BLyS overproduction faithfully, in that steady-state protein levels are affected not just by production rates but also by rates of peripheral utilization and excretion. Increased levels of BLyS mRNA may better reflect increased in vivo BLyS production, and therefore they may correlate better with biologic and clinical sequelae of BLyS overexpression than do circulating levels of BLyS protein. Accordingly, we assessed peripheral blood leukocyte levels of BLyS mRNA isoforms (full-length BLyS and ΔBLyS) and plasma BLyS protein levels in <span class="yellow">patients</span> with SLE, and correlated these levels with laboratory and clinical features. BLyS protein, full-length BLyS mRNA, and ΔBLyS mRNA levels were greater in SLE <span class="yellow">patients</span> (n = 60) than in rheumatoid arthritis <span class="yellow">patients</span> (n = 60) or normal control individuals (n = 30). Although full-length BLyS and ΔBLyS mRNA levels correlated significantly with BLyS protein levels in the SLE cohort, BLyS mRNA levels were more closely associated with serum immunoglobulin levels and SLE Disease Activity Index scores than were BLyS protein levels. Moreover, changes in SLE Disease Activity Index scores were more closely associated with changes in BLyS mRNA levels than with changes in BLyS protein levels among the 37 SLE <span class="yellow">patients</span> from whom repeat blood samples were obtained. Thus, full-length BLyS and ΔBLyS mRNA levels are elevated in SLE and are more closely associated with disease activity than are BLyS protein levels. BLyS mRNA levels may be a helpful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>.<br><br>Introduction
B lymphocyte stimulator (BLyS; a trademark of <span class="yellow">Human</span> Genome Sciences, Inc., Rockville, MD, USA) is a 285-amino-acid member of the tumor necrosis factor ligand superfamily [1-3]. A causal relation between constitutive overproduction of BLyS and development of systemic lupus erythematosus (SLE)-like illness has incontrovertibly been established in <span class="yellow">mice</span>. BLyS-transgenic <span class="yellow">mice</span> often develop SLE-like features as they age [3-5], and SLE-prone (NZB × NZW)F1 (BWF1) and MRL-lpr/lpr <span class="yellow">mice</span> respond clinically to treatment with BLyS antagonists (decreased disease progression and improved survival) [3,6].
Considerable inferential evidence points to a role for BLyS overproduction in <span class="yellow">human</span> SLE as well. Cross-sectional studies have demonstrated elevated circulating levels of BLyS in 20–30% of <span class="yellow">human</span> SLE <span class="yellow">patients</span> tested at a single point in time [7,8]. Moreover, a 12-month longitudinal study documented persistently elevated serum BLyS levels in about 25% of SLE <span class="yellow">patients</span> and intermittently elevated serum BLyS levels in an additional 25% of <span class="yellow">patients</span> [9]. Remarkably, circulating BLyS levels did not correlate with disease activity (measured using the SLE Disease Activity Index [SLEDAI]) in these cross-sectional or longitudinal studies [7-9]. Although a statistically significant correlation between circulating BLyS levels and SLEDAI has been appreciated in a more recent 24-month longitudinal study of 245 SLE <span class="yellow">patients</span> (with >1,700 plasma samples analyzed) [10], the correlation remains weak.
The limited correlation between circulating BLyS protein levels and disease activity in these studies may have exposed an inadequacy of the former to reflect faithfully endogenous BLyS overproduction. In addition to the rate of BLyS protein production, several other factors (for example, utilization and excretion) can affect circulating BLyS protein levels. Although there are no practicable means of directly measuring in vivo BLyS production per se in <span class="blue">humans</span>, the level of BLyS mRNA may serve as a better surrogate marker of in vivo BLyS production than does the level of BLyS protein. Candidate BLyS mRNA isoforms include the full-length BLyS mRNA isoform, which encodes the full-length protein, and the alternatively spliced ΔBLyS mRNA isoform, which encodes a protein with a small peptide deletion [11]. (ΔBLyS does not bind to cells expressing BLyS receptors, and therefore it has no agonistic activity. Moreover, ΔBLyS can form heterotrimers with full-length BLyS, thereby actually functioning as a dominant-negative antagonist of BLyS activity.)
In this report we demonstrate that peripheral blood leukocytes from SLE <span class="yellow">patients</span> express elevated mRNA levels of both full-length BLyS and ΔBLyS relative to those levels expressed by <span class="yellow">patients</span> with rheumatoid arthritis (RA) or by normal control individuals. In the SLE <span class="yellow">patients</span>, both full-length BLyS and ΔBLyS mRNA levels are more closely associated with disease activity (SLEDAI) than are BLyS protein levels. Accordingly, BLyS mRNA levels may be a helpful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>.<br><br>Materials and methods
General details
This study was approved by the institutional review boards of the University of Southern California and the Scripps Research Institute. All <span class="yellow">participants</span> gave their written informed consent before participation in this study.<br><br><span class="yellow">Participants</span>
<span class="yellow">Patients</span> receiving outpatient medical care at the rheumatology clinics of the Los Angeles County + University of Southern California Medical Center were recruited into the study. Diagnoses of SLE (n = 60) or RA (n = 60) were based on established clinical criteria [12]. Healthy control individuals (n = 30) were recruited from Los Angeles County + University of Southern California Medical Center and University of Southern California Keck School of Medicine personnel. No exclusions were made on any basis other than an inability to give informed consent. Each <span class="yellow">patient</span>'s sex, race, age, and medications at the time of the phlebotomy were recorded (Table 1).
Based solely on the <span class="yellow">patient</span>'s willingness to donate a second blood sample, repeat blood samples were collected from 37 of the SLE <span class="yellow">patients</span> 147–511 days (median 371 days) after collection of the first samples. These <span class="yellow">patients</span> were not selected on the basis of any demographic, clinical, or laboratory feature.
Clinical disease activity for the SLE <span class="yellow">patients</span> was assessed using the SLEDAI [13] and using a modified SLEDAI that excludes the contribution of anti-double-stranded DNA (anti-dsDNA) antibodies from the total score. Each <span class="yellow">patient</span>'s medical chart was reviewed for results of standard clinical laboratory tests within the previous or subsequent 1-month period.<br><br>Plasma BLyS determination
Whole venous blood was centrifuged to yield plasma and a buffy coat. The plasma was harvested, stored at -70°C, and assayed for BLyS levels by ELISA [8,14] using Fab fragments of the capture antibody rather than the whole antibody to reduce assay interference by rheumatoid factor. The lower limit of detection in this assay is 0.3 ng/ml. For statistical purposes, plasma samples with BLyS concentrations below the lower limit of detection were assigned a value of 0.25 ng/ml.<br><br>Blood BLyS mRNA determination
The buffy coat from centrifuged whole blood was harvested, added to RNAlater™ (Ambion, Austin, TX, USA) at a 1:4 vol/vol ratio for RNA stabilization, stored at -70°C, and assayed for full-length BLyS and ΔBLyS mRNA levels by real-time PCR. Total RNA was purified from buffy coat samples using RNAeasy miniprep kits (Qiagen, Valencia, CA, USA), and contaminating genomic DNA was removed by DNAse-I digestion. One-tenth volume of total RNA was used as template in the first-strand cDNA reaction using oligo-dT and the Superscript III first-strand synthesis system (Invitrogen, Carlsbad, CA, USA). Duplicate samples of cDNA were amplified with primers against β-actin, full-length BLyS, or ΔBLyS: β-actin sense 5'-CGAGAAGATGACCCAGATCATGT-3'; β-actin anti-sense 5'-GGCATACCCCTCGTAGATGG-3'; full-length BLyS sense 5'-GCAGACAGTGAAACACCAACTATAC-3'; ΔBLyS sense 5'-CAGAAGAAACAGGATCTTACACAT-3'; and full-length BLyS/ΔBLyS anti-sense 5'-TGCCAGCTGAATAGCAGGAATTAT-3'.
A 165 bp amplicon for β-actin was PCR-amplified using the 7900 HT ABI Prism machine (Qiagen) with annealing at 65°C. A 296 bp amplicon for full-length BLyS was PCR-amplified, with annealing at 64°C. A 270 bp amplicon for ΔBLyS was PCR-amplified with annealing at 61°C. The annealing conditions for full-length BLyS and ΔBLyS were determined so that each primer set remained specific to the respective BLyS isoform and yielded a PCR efficiency similar to those of cloned cDNA standards. Melting curve analysis revealed a single peak for each gene amplified. The threshold cycle (Ct) values for each reaction were determined using Sequence Detection System software (Applied Biosystems, Foster City, CA, USA). Results are presented as ratios of full-length BLyS or ΔBLyS mRNA to β-actin mRNA, which were calculated using the following formulae:
2 exp(Ctβ-actin - Ctfull-length BLyS)
2 exp(Ctβ-actin - CtΔBLyS)<br><br>Determination of anti-BLyS autoantibodies
BLyS was bound to microtiter plates by first coating the plates with streptavidin and then adding biotinylated recombinant BLyS. Using these plates as the capture reagent, plasma samples were incubated, and <span class="yellow">horseradish</span> peroxidase-conjugated anti-<span class="yellow">human</span> IgA/IgM/IgG (Southern Biotechnology Associates, Birmingham, AL, USA; 1:20,000 final dilution) or <span class="yellow">horseradish</span> peroxidase-conjugated anti-<span class="yellow">human</span> IgG (Southern Biotechnology; 1:10,000 final dilution) were used as the detector reagents.<br><br>Statistical analysis
All analyses were performed using SigmaStat software (SPSS, Chicago, IL, USA). Results that did not follow a normal distribution were log-transformed to achieve normality. Parametric testing between two matched or unmatched groups was performed using the paired or unpaired t test, respectively. Parametric testing among three or more groups was performed using one-way analysis of variance. When log-transformation failed to generate normally distributed data or the equal variance test was not satisfied, nonparametric testing was performed using the Mann–Whitney rank sum test between two groups and by Kruskal–Wallis one-way analysis of variance on ranks among three or more groups. Correlations were determined using Pearson product moment correlation for interval data and using Spearman rank order correlation for ordinal data or for interval data that did not follow a normal distribution. Nominal data were analyzed using χ2 analysis-of-contingency tables.<br><br>
Results
Elevated plasma BLyS levels and blood levels of full-length BLyS and ΔBLyS mRNA isoforms in systemic lupus erythematosus <span class="yellow">patients</span>
Previous reports of elevated circulating BLyS levels in SLE <span class="yellow">patients</span> were based on a BLyS ELISA that utilized a whole (unfragmented) capture anti-BLyS monoclonal antibody [7-9]. Since the publication of these reports, it has been recognized that the presence of rheumatoid factor can potentially interfere with the assay and lead to spurious overestimation of the true circulating BLyS levels (<span class="yellow">Human</span> Genome Sciences, Inc.; unpublished observations). To mitigate potential interference from rheumatoid factor, the BLyS ELISA has been modified and the capture anti-BLyS monoclonal antibody is now utilized as a Fab fragment. Despite the changes in the ELISA format, our findings are entirely consistent with those of the previous reports. Plasma BLyS levels were significantly greater in the SLE group than in either RA or normal control group (P < 0.001; Figure 1a). Arbitrary assignment of the 95th percentile value among the normal control individuals as the upper limit of 'normal' revealed that two of the 30 normal control individuals, 15 of the 60 RA <span class="yellow">patients</span>, and 29 of the 60 SLE <span class="yellow">patients</span> harbored elevated plasma BLyS levels (P < 0.001).
Overexpression of BLyS in SLE <span class="yellow">patients</span> was also established by measuring BLyS mRNA levels normalized to β-actin mRNA levels in peripheral blood leukocytes (buffy coats). The geometric mean full-length BLyS mRNA and ΔBLyS mRNA levels among the SLE <span class="yellow">patients</span> were each significantly greater than those among the RA <span class="yellow">patients</span> and normal control individuals, respectively (P < 0.001 for each; Figure 1b,c). Arbitrary assignment of the 95th percentile values for full-length BLyS and ΔBLyS mRNA levels among the normal control individuals as the upper limits of 'normal' revealed that two of the 30 normal control individuals, four of the 60 RA <span class="yellow">patients</span>, and 20 of the 60 SLE <span class="yellow">patients</span> had elevated full-length BLyS mRNA levels (P < 0.001), and that two of the 30 normal control individuals, three of the 60 RA <span class="yellow">patients</span>, and 19 of the 60 SLE <span class="yellow">patients</span> had elevated ΔBLyS mRNA levels (P < 0.001). Levels of full-length BLyS and ΔBLyS mRNA strongly correlated with each other (r = 0.703; P < 0.001) in the SLE cohort, and plasma BLyS levels also correlated significantly with levels of each BLyS isoform (r = 0.429, P < 0.001; and r = 0.290, P = 0.024, respectively). Among these SLE <span class="yellow">patients</span>, none of the measured BLyS parameters correlated with <span class="yellow">patient</span> age, sex, race, or daily dose of corticosteroids (data not shown). Because the racial composition of the normal cohort was not as predominantly Hispanic as were those of the RA and SLE cohorts, we assessed the BLyS parameters in the respective Hispanic subpopulations. As for the entire populations, values for SLE were significantly greater than those for either RA or normal controls (P ≤ 0.004; data not shown).<br><br>Correlations between BLyS parameters and plasma immunoglobulin levels
BLyS is a potent B cell survival factor [15-21], and administration of exogenous BLyS to <span class="yellow">mice</span> leads to B cell expansion and hypergammaglobulinemia [1]. Previous studies with numbers of SLE <span class="yellow">patients</span> greater than were included in the present study documented a modest but significant correlation between serum levels of BLyS and IgG [8,10]. In our SLE cohort of limited size, plasma BLyS levels failed to show significant correlations with plasma levels of total immunoglobulin, IgG, or IgA. In contrast, full-length BLyS and ΔBLyS mRNA levels correlated significantly with each (Figure 2). (None of the BLyS parameters correlated with plasma IgM levels.) The absence of significant correlation between plasma BLyS levels and the immunoglobulin parameters also persisted when just the 53 <span class="yellow">patients</span> with detectable plasma BLyS levels were considered (r = -0.133, P = 0.346 for total immunoglobulin; r = -0.048, P = 0.734 for IgG; and r = 0.033, P = 0.817 for IgA).<br><br>Correlations between BLyS parameters and disease activity
Previous studies either have failed to demonstrate a significant correlation between disease activity and circulating BLyS levels [7-9] or have detected only a weak correlation between the two [10]. Consonant with those studies, we identified no significant correlation between plasma BLyS levels and SLEDAI in our cohort of 60 SLE <span class="yellow">patients</span> (Figure 3a). The failure to demonstrate a significant correlation cannot be attributed to a skewing of the results by the <span class="yellow">patients</span> in whom plasma BLyS levels were below the limit of detection, because no significant correlation was detected among the 53 SLE <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range (r = 0.185, P = 0.183). In contrast, a significant correlation between SLEDAI and full-length BLyS mRNA levels was readily discernible (Figure 3b). A trend toward a correlation between SLEDAI and ΔBLyS mRNA levels was also observed, although it did not achieve statistical significance (Figure 3c).
A component of the SLEDAI is the presence of circulating anti-dsDNA antibodies. Because circulating BLyS levels may affect the presence and/or titers of circulating anti-dsDNA antibodies [7-10], we assessed correlations between the individual BLyS parameters and a modified SLEDAI that excludes any consideration of anti-dsDNA antibodies. As with the unmodified SLEDAI, the modified SLEDAI did not correlate with plasma BLyS levels (Figure 3d) either among the SLE cohort overall or among the 53 <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range (r = 0.160, P = 0.252), but it significantly correlated with full-length BLyS mRNA levels (Figure 3e) and exhibited a trend toward correlation with ΔBLyS mRNA levels (Figure 3f). Thus, the stronger correlations between BLyS mRNA levels and disease activity cannot solely be explained by any effects that BLyS may have on anti-dsDNA antibodies per se.
Moreover, among the 37 SLE <span class="yellow">patients</span> who were evaluated on two separate occasions, trends toward correlation were appreciated between changes in the unmodified or modified SLEDAI and changes in full-length BLyS or ΔBLyS mRNA levels but not changes in plasma BLyS levels (Figure 4). These results cannot be ascribed to changes in medications taken by the <span class="yellow">patients</span>, because changes in neither disease activity nor in any of the BLyS parameters correlated with changes in the doses of corticosteroids or cytotoxics taken by the <span class="yellow">patients</span> (data not shown). The failure to demonstrate a meaningful association between changes in SLEDAI score and changes in plasma BLyS protein levels cannot be attributed to a skewing of the results by the <span class="yellow">patients</span> in whom plasma BLyS levels were below the limit of detection, because the absence of association between the two persisted among the 27 SLE <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range in both samples (r = -0.069, P = 0.727 for plasma BLyS versus unmodified SLEDAI; r = -0.020, P = 0.919 for plasma BLyS versus modified SLEDAI).<br><br>Lack of correlation between levels of BLyS mRNA isoforms and percentages of individual leukocyte cell types
Among cells in peripheral blood, BLyS is predominantly expressed by cells of the myeloid lineage (monocytes and neutrophils) [1,14,22,23]. Accordingly, a shift in the differential leukocyte count away from lymphocytes to monocytes and/or neutrophils could substantially alter BLyS mRNA results. Because of the limited amount of blood we were permitted to obtain from the SLE <span class="yellow">patients</span> (consequent to the high prevalence of anemia among these <span class="yellow">patients</span>), we were unable to purify the individual leukocyte populations for BLyS mRNA analysis. Nevertheless, to demonstrate that the elevated BLyS mRNA levels in SLE did not simply reflect a shift in differential leukocyte count, we assessed the correlations between the individual BLyS parameters on the one hand and the percentages of blood neutrophils, monocytes, and lymphocytes on the other. No correlations were appreciated (Figure 5).<br><br>Presence of anti-BLyS autoantibodies in <span class="yellow">patients</span> with systemic lupus erythematosus
The poorer correlation between plasma BLyS protein levels and disease activity compared with that between BLyS mRNA levels and disease activity was striking. <span class="yellow">Patients</span> with SLE frequently develop autoantibodies against self-antigens, and so some of the SLE <span class="yellow">patients</span> might have harbored autoantibodies to BLyS. Such autoantibodies could have complexed with BLyS and enhanced its clearance, thereby masking BLyS overproduction. Alternatively, such autoantibodies might have sterically blocked the epitopes recognized by the detecting antibodies in the in vitro ELISA. In this case, measured BLyS levels would have been spuriously reduced, again masking BLyS overproduction.
In our cohort, IgA/IgM/IgG anti-BLyS antibodies were detected in six out of the 60 SLE <span class="yellow">patients</span>. Such autoantibodies were also detected in two out of 60 RA <span class="yellow">patients</span> and in one out of 30 normal control individuals, demonstrating that anti-BLyS autoantibodies are not restricted to SLE <span class="yellow">patients</span>. IgG anti-BLyS autoantibodies were detected in 3 SLE <span class="yellow">patients</span> but in no RA <span class="yellow">patients</span> or normal control individuals.<br><br>
Discussion
Elevated blood levels of BLyS protein and mRNA are well described features of <span class="yellow">human</span> SLE [7-9]. We confirmed these observations in our study and extended them by documenting increases not just in levels of full-length BLyS mRNA but also in levels of ΔBLyS mRNA (Figure 1). Of note, BLyS mRNA levels were elevated in SLE but not in RA, raising the possibility that BLyS overproduction in SLE is systemic whereas BLyS overproduction in RA may be more focused to the affected arthritic joints [24]. The modestly elevated plasma BLyS protein levels in RA <span class="yellow">patients</span> may reflect, at least in part, release of locally overproduced BLyS into the circulation.
The relationship between circulating BLyS protein levels and disease activity was addressed in several previous studies, but significant correlations between the two measures did not emerge [7-9]. In the largest study to date, a 2-year longitudinal study of 245 <span class="yellow">patients</span> in which more than 1,700 plasma samples were analyzed, a significant but weak correlation between the two was appreciated [10]. In the present study, a significant correlation between plasma BLyS protein levels and disease activity was again not realized (Figure 3a,d).
The weak, at best, correlation between circulating BLyS levels and disease activity is seemingly rather surprising. There is a clear-cut association in BlyS transgenic <span class="yellow">mice</span> between BLyS overexpression and development of SLE-like features [3-5], and treatment of SLE-prone <span class="yellow">mice</span> with BLyS antagonists retards the progression of disease and improves survival [3,6]. Moreover, development of precocious glomerular pathology in autoimmune-prone <span class="yellow">mice</span> correlates strongly with circulating BLyS levels [25].
The likely explanation for the weak correlation between circulating BLyS levels and disease activity in <span class="yellow">human</span> SLE is not that disease activity in SLE <span class="yellow">patients</span> is insensitive to the degree of BLyS overproduction. Rather, a more tenable explanation is that circulating BLyS levels in <span class="yellow">human</span> SLE do not always accurately reflect excessive endogenous BLyS production. We can identify at least three nonmutually exclusive mechanisms to explain a dissociation between the two.
First, SLE <span class="yellow">patients</span> frequently develop autoantibodies to a myriad of self-targets (for example, erythrocytes, lymphocytes). Indeed, we detected circulating IgA/IgM/IgG anti-BLyS autoantibodies in 10% (6/60) of the tested SLE <span class="yellow">patients</span>, and we detected circulating IgG anti-BLyS autoantibodies in 5% (3/60). These percentages may be underestimates of the true prevalence of anti-BLyS autoantibodies, because some of these autoantibodies may be saturated in vivo with circulating BLyS, rendering them incapable of binding to BLyS in the in vitro detection assay. We do not yet know whether the anti-BLyS autoantibodies are functionally neutralizing but, regardless, such autoantibodies could enhance the clearance of BLyS and/or interfere with in vitro detection of BLyS, thereby masking endogenous BLyS overproduction.
Second, increased urinary excretion of BLyS has been reported in SLE <span class="yellow">patients</span>, especially among those with clinically overt renal involvement [26]. At least four of the <span class="yellow">patients</span> we studied manifested nephrotic-range proteinuria (≥3 g/24 hours), and so urinary loss of BLyS was probably substantial in at least these <span class="yellow">patients</span>. A validated assay for urinary BLyS detection has not yet been developed so we were unable to quantify urinary BLyS levels. Once an assay for urinary BLyS levels is validated, we should be able to assess the effect of urinary BLyS excretion on circulating BLyS levels.
Third, BLyS promotes in vivo expansion of B cells [1]. Freshly isolated SLE B cells, despite intact surface expression of BLyS receptors, bind less biotinylated BLyS ex vivo than do freshly isolated normal B cells [27]. Although other interpretations are possible, the most likely explanation is that BLyS receptors on B cells in SLE <span class="yellow">patients</span> are occupied in vivo by soluble BLyS. Accordingly, it is likely that BLyS receptors expressed by the expanded B cell population do bind BLyS and remove it from the circulation, resulting in a homeostatic pathway that modulates the effects of BLyS overproduction on circulating BLyS levels. Indeed, circulating levels of BLyS rise with peripheral blood B cell depletion and fall with re-emergence of peripheral blood B cells in rituximab-treated RA or SLE <span class="yellow">patients</span> [28,29], highlighting this inverse relationship between circulating BLyS levels and B cell load. Moreover, one of the hallmarks of active disease in <span class="yellow">human</span> SLE is the increased percentages of activated B cells and plasma cells in peripheral blood [30-34], probably reflecting increased systemic numbers of activated B cells and plasma cells. Although not yet formally tested, differential BLyS receptor expression by these cells compared with expression by nonactivated B cells may result in increased peripheral BLyS utilization, further dampening the effects of BLyS overproduction on circulating protein levels.
To circumvent these confounding processes, we used BLyS mRNA levels as a surrogate marker of endogenous BLyS production. Overall, the correlations between disease activity and either full-length BLyS or ΔBLyS mRNA levels were much stronger than that between disease activity and BLyS protein levels (Figures 3 and 4). This was the case regardless of whether we used the standard SLEDAI or the modified SLEDAI as a measure of disease activity. These correlations were not spurious ones consequent to shifts in percentages of leukocyte subpopulations in peripheral blood, because BLyS mRNA levels did not correlate with percentages of blood neutrophils, monocytes, or lymphocytes (Figure 5).
A similar pattern was observed between plasma immunoglobulin levels and the BLyS parameters, with plasma levels of total immunoglobulin, IgG, and IgA correlating significantly with full-length BLyS and ΔBLyS mRNA levels but not with plasma BLyS levels (Figure 2). These significant correlations between full-length BLyS or ΔBLyS mRNA levels and plasma immunoglobulin levels again highlight the greater ability of BLyS mRNA levels, compared with plasma BLyS protein levels, to reflect ongoing BLyS overproduction.
At present, it is not known whether soluble ΔBLyS protein is present in the circulation of SLE <span class="yellow">patients</span> or of normal individuals. Although full-length BLyS protein is readily cleaved and released from cells transfected with a vector containing <span class="yellow">murine</span> full-length BLyS, ΔBLyS protein is not cleaved or released from <span class="yellow">murine</span> ΔBLyS transfectants [11]. Given the strong similarities between <span class="yellow">murine</span> and <span class="yellow">human</span> full-length BLyS and ΔBLyS, it is likely that <span class="yellow">human</span> soluble ΔBLyS protein is also not cleaved from the cell surface and released into the circulation. Moreover, soluble ΔBLyS protein is not released from cells transfected with a vector containing just the extracellular domain of <span class="yellow">human</span> ΔBLyS (which encodes the soluble protein; A.L. Gavin, unpublished observations). Whether this reflects rapid intracellular degradation of soluble ΔBLyS or some other impediment to its release remains unknown. Regardless, if the inability to release soluble ΔBLyS in vitro faithfully recapitulates in vivo biology, then the stronger associations between SLE disease activity and full-length BLyS or ΔBLyS mRNA levels compared with that between SLE disease activity and BLyS protein levels could not be attributable to interference by biologically inactive (inhibitory) ΔBLyS protein in the BLyS protein detection ELISA. Importantly, even if soluble ΔBLyS protein is present in the circulation and is detected by the BLyS protein detection ELISA, then the stronger correlations between SLE disease activity and full-length BLyS or ΔBLyS mRNA levels than that between disease activity and total BLyS (including ΔBLyS) protein levels suggest that full-length BLyS and/or ΔBLyS mRNA levels may operationally serve as useful biomarkers of disease activity in SLE. Although the complexity and labor intensiveness associated with quantitative real-time PCR may render measurement of BLyS mRNA levels impracticable for routine clinical practice, such measurement could readily be incorporated into clinical trials and yield valuable information.
Longitudinal observations in large numbers of SLE <span class="yellow">patients</span> will be necessary to establish or refute the utility of full-length BLyS and/or ΔBLyS mRNA to subserve this clinically vital function.
Although expression of the two major BLyS isoforms was highly coordinate among SLE <span class="yellow">patients</span>, there were several <span class="yellow">patients</span> in whom ΔBLyS mRNA levels were markedly greater than or less than the expected values based on full-length BLyS mRNA levels (data not shown). This raises the possibility that dysregulation of ΔBLyS may contribute to overall BLyS dysregulation in at least some SLE <span class="yellow">patients</span>. It is known that interferon-γ, interleukin-10, interferon-α, and CD154 can upregulate full-length BLyS mRNA levels [14,22,35], but it is not known what effects these or other cytokines/cell-surface structures have on ΔBLyS expression. Further investigation of the regulation of ΔBLyS and the differential expression of BLyS isoforms is certainly warranted.
Although the associations between full-length BLyS and/or ΔBLyS mRNA levels and disease activity in SLE were usually strong when the SLE cohort was analyzed in aggregate, there were several SLE <span class="yellow">patients</span> in whom BLyS mRNA levels were quite high despite little objective ongoing disease activity, and there were several SLE <span class="yellow">patients</span> in whom BLyS mRNA levels were low despite considerable ongoing disease activity. One must recognize that the bulk of the pathogenic autoimmune responses probably takes place in the spleen and lymph nodes, rather than in the peripheral blood, where myeloid lineage cells (for example, dendritic cells) produce BLyS and support B cell survival and expansion [36]. Local BLyS production in the secondary lymphoid tissues will be more important to the development and maintenance of an autoimmune response than will remote BLyS levels in the circulation. Because at least some autoreactive B cells may be more sensitive to BLyS-mediated survival signals than non-autoreactive B cells [37,38], local increases in BLyS production could preferentially promote expansion of autoreactive B cells. These cells, in turn, could activate autoreactive T cells by presenting autoantigen to them, and some of the autoreactive B cells would respond to T cell derived signals and mature into (pathogenic) autoantibody secreting plasma cells. In contrast to <span class="yellow">murine</span> studies, in which investigators can readily harvest and analyze lymphoid and myeloid lineage cells from any site (for example, spleen, bone marrow), such is not the case for <span class="yellow">human</span> studies. Peripheral blood is the only site readily accessible for <span class="yellow">human</span> studies, and it is possible that, at least in some <span class="yellow">patients</span>, BLyS mRNA levels in circulating leukocytes do not reflect local BLyS production in the secondary lymphoid tissues.
One must also recognize that disease activity in SLE is not solely driven by B cells. Systemic inflammation and SLE flares can be triggered via B cell independent means. Not all SLE <span class="yellow">patients</span> treated with a B cell depleting course of rituximab experience clinical remission [39], strongly pointing to the importance of non-B cells in disease pathogenesis/maintenance. Conversely, not all pathogenic B cells necessarily require high levels of BLyS to effect their pathogenicity. <span class="yellow">Murine</span> studies have unequivocally documented B cell subpopulations that do not depend upon BLyS for their survival [40-42]. Although <span class="yellow">mice</span> completely devoid of BLyS have reduced numbers of mature B cells and harbor reduced levels of immunoglobulin, these reductions are incomplete. Thus, it is possible that some SLE <span class="yellow">patients</span> harbor pathogenic B cells that are relatively insensitive to BLyS and could drive considerable disease activity even in the context of low endogenous BLyS production. Conversely, <span class="yellow">patients</span> with high BLyS mRNA levels may be those <span class="yellow">patients</span> whose disease is strongly driven by BLyS and may be especially helped by BLyS antagonist therapy. Future clinical trials should be able to establish whether the BLyS mRNA levels are good predictors of response to such agents.<br><br>Conclusion
Plasma total immunoglobulin, IgG, and IgA levels and disease activity (as measured by SLEDAI) in SLE <span class="yellow">patients</span> correlate more closely with peripheral blood leukocyte levels of BLyS mRNA than with plasma levels of BLyS protein. These findings suggest that BLyS mRNA levels better reflect in vivo BLyS production than do circulating BLyS protein levels, and may be a useful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>. These findings also support the premise that BLyS overexpression not only promotes development of disease but also actively contributes to the ongoing maintenance of disease in SLE <span class="yellow">patients</span>. This reinforces the rationale underlying clinical trials with BLyS antagonists in SLE.<br><br>Abbreviations
anti-dsDNA = anti-double-stranded DNA; BLyS = B lymphocyte stimulator; bp = base pairs; Ct = threshold cycle; ELISA = enzyme-linked immunosorbent assay; PCR = polymerase chain reaction; RA = rheumatoid arthritis; SLE = systemic lupus erythematosus; SLEDAI = SLE Disease Activity Index.<br><br>Competing interests
TSM and DMH were employees of <span class="yellow">Human</span> Genome Sciences (HGS) at the time the investigation was conducted. (DMH has since left the company.) WS has received research support from HGS and has served as a consultant to HGS (<$10,000). CEC, ALG, and DN declare that they have no competing interests.<br><br>Authors' contributions
CEC identified and recruited all <span class="yellow">participants</span>; collected all the blood samples and reviewed all the medical charts; and wrote the initial working draft of this manuscript. ALG developed and performed all the real-time PCR assays and assisted in the interpretation of the results and in writing the final version of the manuscript. TSM performed the plasma BLyS protein and anti-BLyS assays and assisted in the interpretation of the results and in writing the final version of the manuscript. DMH assisted in the design in the study, in the interpretation of the results, and in writing the final version of the manuscript. DN assisted in the design in the study, in the interpretation of the results, and in writing the final version of the manuscript. WS conceived the study, supervised the recruitment of <span class="yellow">participants</span>, performed the statistical analyses, assisted in the interpretation of the results, and supervised the editing of the manuscript to its final form. All authors read and approved the final manuscript version.<br><br>
<h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="blue">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA2365769</h3>IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome
Abstract
This study was undertaken to evaluate the role of IL-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum IL-5 levels that might be essential for eosinophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-IL-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of IL-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce IL-5.
Research Paper Mediators of Inflammation 5, 24-31 (1996) <br><br> Tins study was undertaken to evaluate the role of 11-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum 11-5 levels that might be essential for eosi<br><br> IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome L. H. Faccioli, 1"cA V. F. Mokwa, C. L. Silva, G. M. Rocha, J. I. Araujo, M. A. Nahori 2 and B. B. Vargaftig 2 <br><br> nophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-11-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of 11-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing 11-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce 11-5. <br><br> 1Department of Parasitology, Microbiology and Immunology, School of Medicine of Ribeir,5o Preto, 14049-900, Ribeiro Preto, SP, Brazil. Fax: (+55) 16 633 6631 2Unit de Pharmacologie Cellulaire, Unit Associe Institut Pasteur/INSERM n. 285, Paris, <br><br> France. <br><br> CACorresponding Author <br><br> Key words: Eosinophil, Eosinophilia by <span class="yellow">Toxocara canis</span>, IL-5 in eosinophilia, <span class="yellow">Toxocara canis</span> <br><br> Introduction <br><br> treatment in <span class="yellow">mice</span> infected with <br><br> Eosinophilia has been associated with parasitic diseases, particularly when the parasites invade the tissues or injure the mucosal surfaces. <span class="yellow">Toxocara canis</span> is an intestinal parasite of <span class="yellow">dogs</span>, and is the most common aetiologic agent of visceral larva migrans syndrome (VLMS). In <span class="blue">humans</span>, VLMS results from the ingestion of embryonated eggs of <span class="yellow">T. canis</span>, that eclode in the small intesfine. The infective larvae invade the mucosa, move into the liver via the portal circulation, and from there to the lungs. 2 Beaver et al., 3 who were the first to describe this syndrome, noted the intense eosinophilia which reaches more than 90% of total leucocyte counts. However, there are few studies regarding the mechanisms involved in the blood and tissue eosinophilia obseeeed in VLMS. Several investigators have suggested a direct correlation between eosinophilia and interleukin5 (IL-5) in <span class="yellow">human</span> helminth infections4'5 and in experimental animal models. 67 Inhibition of eosinophilia has been demonstrated by anti-IL-5 24 Mediators of Inflammation Vol 5 <br><br> brasiliensis, 8 $chistosoma mansoni, <span class="yellow">Toxocara canis</span>7 and <span class="yellow">Heligmosomoidespolygyrus</span>. 1 IL-5 has also been shown to support the terminal differentiation, proliferation of eosinophil precursors 11'12 and eosinophil activation. 13 Although IL-5 does not demonstrate eosinophil chemotactic activity in vivo14 there is some evidence suggesting that this cytokine may modulate a selective eosinophil accumulation at the site of inflammation. Moreover, Sehmi et a/.15 reported that IL-5 has a selective priming effect on eosinophil migratory response to nonselective chemoattractant mediators in vitro. Also, Moser et a/. 16 have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with IL-5, IL-3 and GM-CSF. Thus, the involvement of IL-5 in eosinophilia is not fully understood. In the present study we have used a <span class="yellow">guinea-pig</span> model of VLMS to investigate the involvement of IL-5 in eosinophil migration and in the maintenance of eosinophilia in blood, bone marrow, lung and peritoneal cavity. (C) 1996 Rapid Science Publishers <br><br> Nipostrongylus <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> Materials and Methods Animals: Outbred albino weighing 300-400g at the ments were obtained from the School of Medicine of versity of So Paulo, Brazil. <br><br> female <span class="yellow">guinea-pigs</span> start of the experi<br><br> were routinely processed, embedded in paraffin, sectioned at 4-61.tm, and stained with Chromothope 2R and haematoxylin, for examination by light microscopy. Determination oflL-5 in serum: The IL-5 level in the serum of <span class="yellow">guinea-pigs</span> was measured using an enzyme-linked immunosorbent assay (ELISA). Briefly, ELISA plates (96-well Immunoplate MaxiSorp, Nunc, Roskilde, Denmark) were coated with IL-5-specific monoclonal antibody (TRFK-5, 5 lag/ml in phosphate buffered saline, pH 7.4, PBS, 100 l.tl/well). After 2 h of incubation at 37C, the wells were washed four times with PBS containing 0.1% Tween 20 (PBS-T). Then, 100 btl of samples or recombinant <span class="yellow">murine</span> IL-5 standards (0.15-200ng/ml) in PBS-T and 2% BSA (PBSTBSA) were added to each well. After incubation for l h at 37C, the wells were washed three times and 100 l.tl of biotinylated <span class="yellow">rat</span> anti-<span class="yellow">murine</span> IL-5 (biotinylated-TRFK-5, 1 lag/ml in PBS-TBSA) was added. After incubation for 1 h at 37C, the wells were washed again three times and 100 of peroxidase-labelled streptavidin (1/1000, Kirkegaard & Perry Laboratories Inc., Maryland, USA) were added to each well. Following incubation for l h at 37C and further washing, the enzyme was developed using the TMB substrate peroxidase for 5 min. The reaction was stopped by adding 501.tl of 2.0 N HCl, and the optical densities were read at 490nm using an automated plate reader. The sensitivity of the assay was 0.15 ng/ml and the upper limit 100ng/ml. <br><br> the animal house of Ribeiro Preto, Uni<br><br> Infection of animals: <span class="yellow">T. canis</span> eggs were obtained by the methods of Olson and Schulz, 17 with minor modifications. Briefly, gravid female <span class="yellow">worms</span> were recovered from <span class="yellow">dogs</span>, and the eggs were rescued from the uterus, washed and allowed to develop to the infective stage in shallow dishes containing 0.5% formalin at 37C. Under light ether anaesthesia, the animals were infected with I ml saline containing 500 <span class="yellow">T. canis</span> eggs, by gastric intubation using a metal cannula. Blood cell counts: <span class="yellow">Guinea-pigs</span> were anaesthetized with sodium pentobarbitone (30mg/kg, i.v.) and blood samples were collected by cardiac puncture with 10% EDTA. Total cell counts were carried out using diluting fluid in a Neubauer chamber. Differential countings were obtained using Rosenfeld-stained cytocentrifuge preparations, <br><br> Bronchoalveolar lavage fluid. The <span class="yellow">guinea-pigs</span> were killed by an overdose of sodium pentobarbitone and 5ml of phosphate-buffered saline (PBS) containing 0.5% sodium citrate (PBS/SC), at room temperature, were instilled through a polyethylene cannula introduced into the trachea. The cells present in the bronchoalveolar lavage fluid (BALF) were recovered immediately. The procedure was repeated once. The leucocyte counts in the BAI_ were determined as described above, Peritoneal cells: The cells from the peritoneal cavities were harvested by injection of 10ml of PBS/SC into the peritoneum. Only 5-8 ml of the <br><br> fluid was withdrawn for cell counts, as described above, <br><br> Monoclonal antibodies: The <span class="yellow">rat</span> monoclonal antibody TRFK-5 was a generous gift from Dr P. Minoprio, Institut Pasteur, Paris. The neutralizing antibody was purified by precipitation with ammonium sulfate (45%) from ascites prepared in CD1 <span class="yellow">nude mice</span> (Charles River, St Aubin les Elbeuf, France) inoculated 1 week before the injection of hybridoma cells, with I ml of pristane (Sigma). After precipitation and dialysis of the ascite fluid overnight against PBS, the dialysate was further purified on a Protein G1 column (HiTrapTM, Pharmacia Upsala, Sweden). <br><br> Bone marrow cells: Bone marrow cells were collected by flushing the contents of the <span class="yellow">guinea-pig</span> femur with 10 ml of PBS/SC. Total cell numbers were determined as above. In the differential cell counts the cell populations were divided into mature neutrophils, mature eosinophils and others (mainly precursors and mononuclear <br><br> cells), Histopathological studies: Tissues were removed from <span class="yellow">guinea-pigs</span> at various times post-infection and immediately fixed in 10% formalin. Tissues <br><br> Eosinophil and cytokine depletion: <span class="yellow">Guinea-pigs</span> were injected i.p. with TRFK-5 or with the irrelevant antibody (<span class="yellow">rat</span> IgG against total anti-<span class="yellow">human</span> IgG) once, 2mg/animal, at the time of infection or at different intervals (1, 3, 12 or 17 days) thereafter. The animals in this group were sacririced 18 days after infection. liver: One lobule of was used to determine the larval each liver counts from infected <span class="yellow">guinea-pigs</span>. Larval recovery <br><br> Recovery <br><br> of larvae from <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 25 <br><br> L. H. Faccioli et al. <br><br> was evaluated as described by Kayes and Oaks, 18 with minor modifications. Briefly, the tissue was chopped and digested with pepsin-HC1 (pH 1.5<br><br> 1.8) for 2h at 37C. Larval counts for each sample were performed after centrifugation and examination of three 100-l.tl samples under the light microscope. Statistical analysis: Data are presented as the mean _+ S.E.M. and were analysed statistically using the Mann-Whitney test for unpaired data. A p < 0.05 value was considered to be statistically <br><br> significant. <br><br> Results <br><br> of eosinophil counts in blood bone marrow, BALF and peritoneum: <span class="yellow">Guinea-pigs</span> infected with <span class="yellow">T. canis</span> eggs showed a timedependent blood, bone-marrow, BAUV and peritoneal eosinophilia (Fig. 1). The results represent the mean of nine animals obtained in three different experiments. The eosinophil number Kinetics 16<br><br> increased significantly from 0.55 +_ 0.37 x 105 at the beginning of experiment to 6.0 +__ 1.03 x 105 at 6 days post-infection, peaked at day 18 (12.0 _+ 2.31 x 105), and decreased by day 24 (8.11 2.85 x 105) (Fig. 1A). A rise in the percentage of mature eosinophils in bone marrow" was observed 12 days after infection (ranging from 6 2% to 14 2%) and peaked at 18 days (17 2%) (Fig. 1B). As in blood, the number of eosinophils in BALF increased significantly from 0.14 0.06 x 105 to 1.37 _+ 0.35 x 105 at 6 days after infection, reaching a peak at 18 days (10.23 2.62 x 105) with an increase in relative number of as much as 90% in eosinophil counts in relation to controls, and was still elevated at day 24 (9.07___ 3.47 x 10 s) (Fig. 1C). The remaining cells in the BALF were alveolar macrophages, lymphocytes, mast cells and ciliated cells. In contrast to blood and BALF, the number of eosinophils in the peritoneal cavity increased significantl only at day 12 post-infection (onset, 2.06 1.04 x 105; day 6, 3.68 _+ 0.82 x 105; day 12, 5.77 +_ 1.12 x 10>; and increased progres<br><br> __ _ _ _. (B) <br><br> (AI Blood <br><br> 24 20" <br><br> Bone Marrow <br><br> 1612 8 <br><br> 4 O0<br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 111 <br><br> 21 <br><br> 24 <br><br> 16<br><br> (c) BALF <br><br> 16<br><br> ID) Peritoneum <br><br> 12<br><br> __o <br><br> 12<br><br> 8 <br><br> 4 <br><br> 0 3 <br><br> O6 9 <br><br> 12 <br><br> 15 <br><br> 16 <br><br> 21 <br><br> 24 <br><br> Days postinfoction <br><br> FIG. 1. Number of eosinophils in blood, BALF and peritoneal cavity, and percentage, of eosinophils in bone marrow of <span class="yellow">T. canis</span>qnfected <span class="yellow">guinea-pigs</span>. Values are the mean -t-S.E.M. (n=8 to 9). Asterisks indicate a significant difference between infected and noninfected animals (n 5 6). *p < 0.05 and **p < 0.01. 26 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> sively until day 24, 12.44 4- 2.72 x 105) (Fig. 1D). The percentage of eosinophils in some animals reached 55% at the peak of infection. No increase in the number of mononuclear cells was seen in any compartment analysed. Larval counts: The percentage of inoculated <span class="yellow">T. canis</span> larvae recovered by peptic digestion of the liver of experimental animals 4 h and 1, 2, 3, 4, 9, 12 and 18 days after inoculation of 500 eggs per animal is shown in Fig. 2. Most of the larvae were recovered 2 to 4 days after infection and 10% recovery was also observed on day 18 in the liver of the animals. <br><br> 150 <br><br> 120 <br><br> IL-5 level in serum of infected animals: IL-5 was measured in the serum of infected and normal <span class="yellow">guinea-pigs</span>. Each time point in Fig. 3 represents the mean of results from three to five infected animals, and from six controls. Two peaks of IL5 were present in the serum of infected <span class="yellow">guineapigs</span> 1 day after infection (102 __+ 22 pg/ml), and 18 days later (59 7 pg/ml). The level of IL-5 in the controls was 31 4- 4 pg/ml. Eosinophil numbers in infected animals treated with TRFK-5: When <span class="yellow">guinea-pigs</span> received an i.p. injection of TRFK-5, the monoclonal antibody against IL-5, at the time of egg administration or 1 day later, the number of eosinophils in blood, BALF, peritoneal cavity and bone marrow was <br><br> _ <br><br> 30<br><br> 0 <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 12 <br><br> 18 <br><br> Days postinfection <br><br> FIG. 3. IL-5 concentration in serum of <span class="yellow">T. cani</span>infected <span class="yellow">guineapigs</span> (n 3-5). Basal IL-5 concentrations were of 31 4 pg/ml (n= 12). <br><br> _ <br><br> 24 <br><br> 60<br><br> 50<br><br> 40<br><br> 0 <br><br> "T" <br><br> 1" <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 18 <br><br> 21 <br><br> 24 <br><br> Days postinfection FIG. 2. Percentage of <span class="yellow">T. canis</span> larvae recovered from liver of <span class="yellow">guinea-pigs</span> studied at various times post-infection. Data obtained from five animals. <br><br> drastically reduced, even when determined 18 days after infection (Table 1). No inhibition of eosinophil counts was observed when the animals were inoculated with the irrelevant antibody at the time of infection (Table 1). Fig. 4 shows the comparative results of eosinophilia obtained when the antibody was given 3 days or 17 days after egg inoculation. The antibody given at 3 days after infection induced a high percentage of inhibition in eosinophil counts in all the compartments analysed 18 days after infection (Fig. 4A). However, when TRFK-5 was administered to the infected animals on day 17 post-infection (thus 1 day before sacrifice), a significant inhibition in number and percentage of eosinophils was observed only in the blood (p=0.030) (Fig. 4B). A small non-significant decrease was seen in BALF (p 0.790) and peritoneum (p= 0.222). Moreover, the number of mature eosinophils in bone marrow increased by 140% (p 0.038). As demonstrated in Fig. 4B, the behaviour of eosinotShilia in BALF was completely different from that observed in blood. Thus, to better understand the eosinophilia in the lungs of infected animals, we monitored eosinophil numbers in BALF after administration of TRFK-5 at the same time, or 1, 3, 12 or 17 days after infection. The animals were sacrificed 18 days after infection. In another group, TRFK-5 was administered 18 days post-infection and the animals were sacrificed 6 days later. When the mAb was administered at the same time or 1 or 3 days postinfection there was a significant inhibition in the number of eosinophils (Fig. 5). These data show Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 27 <br><br> L. H. Faccioli et al. Table 1. Eosinophils in T. caniinfected <span class="yellow">guinea-pigs</span> treated or untreated with TRFK-5 Compartment <br><br> Time of sacrifice <br><br> (days) <br><br> Non18 24 18 24 18 24 18 24 <br><br> treated <br><br> Blood <br><br> 12.18 5.28 10.08 -t- 2.85 16.50 4.42 6.57 +__ 1.68 <br><br> BALF Peritoneal cavity <br><br> 12.31 -I- 2.35 12.44 2.33 9 6 <br><br> Bone marrow <br><br> _ _ _ _ _ ___ _ __ __ _ __ Days of treatment with TRFK-5 after egg administration <br><br> Irrelevant Ab at the time of infection <br><br> 0 <br><br> 3 <br><br> 12 <br><br> 17 <br><br> 18 <br><br> (n 6) <br><br> (n=4) <br><br> (n=4) <br><br> (n 5) <br><br> (n 5) <br><br> (n=4) <br><br> 0.14 -t- 0.14" <br><br> 0.26 0.79 <br><br> 0.26* 0.24* <br><br> 0.17 _-t- 0.17" <br><br> 0.15 <br><br> 0.15" <br><br> 2.59 <br><br> 0.80* 4.36 2.21 <br><br> 13.32 <br><br> 4.54 <br><br> O. 15 <br><br> 0.15* <br><br> 0.36 <br><br> 0.10" <br><br> 1.22 -t- 0.50* <br><br> 7.07 -t- 2.39 <br><br> 15.24 <br><br> 29.31 _+ 15.90 16.09 <br><br> 10.33 _+ 4.10 3.79 <br><br> 1.47 -t- 0.76* 3 <br><br> 0.48 <br><br> _+ 0.35* <br><br> 0.46 2.6 <br><br> 0.27* 1.3" <br><br> 2.89 <br><br> 0.94* <br><br> 10.42 <br><br> 4.00 <br><br> 1.58" <br><br> -+ <br><br> 1" <br><br> 1.25 -I- 0.25* <br><br> + <br><br> 5 <br><br> 15 -t- 2 <br><br> 9_+1 <br><br> 2 <br><br> 0.7* <br><br> In blood BALF and peritoneal cavity the values represent mean -t-_ S.E.M. x 10 eosinophils. *p < 0.05. <br><br> ml-1 <br><br> and in bone marrow mean -t-_ S.E.M. of the percentage of mature <br><br> that the inhibition of the first peak of IL-5 which appeared at 1 to 3 days after infection as shown in Fig. 3, is also very important for the establishment of eosinophilia in the lungs. However, 24- (A) <br><br> when the mAb was administered 12, 17 or 18 days after infection there was no significant inhibition in the numbers of eosinophils in BALF (Fig. 5), showing that once established, the eosi18<br><br> 15x <br><br> 16<br><br> 12<br><br> ._ . 0 <br><br> 12 <br><br> o <br><br> BIo o d <br><br> BALF <br><br> P e rit oneum <br><br> Bone marrow <br><br> 24- (Is) <br><br> 1815<br><br> E x <br><br> 20<br><br> 16<br><br> o <br><br> 12///. <br><br> o = o <br><br> 12<br><br> 0 Blood <br><br> O" <br><br> BALF <br><br> Peritoneum <br><br> Bone marrow FIG. 4. Number of eosinophils in blood, BALF and peritoneum and percentage of eosinophils in bone marrow of <span class="yellow">T. cani</span>infected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) 2 mg/animal at 3 days post-infection; (B) 2 mg/animal 17 days post-infection. The treated and control animals were sacrificed 18 days after infection. Asterisks indicate a significant difference from infected controls (n=5-6) and from animals treated with TRFK-5 (n=4- 5). *p < 0.05 and **p < 0.01. 28 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> 24<br><br> E <br><br> 20<br><br> heart; data not shown) and muscle, as reported by other investigators, 8 were infiltrated. The factors responsible for in vivo eosinophil accumulation at inflammatory sites have been <br><br> 0 <br><br> TRFK-5 i.p. <br><br> 0 <br><br> 3 <br><br> 12 17 <br><br> A <br><br> B <br><br> FIG. 5. Number of eosinophils in BALF of T. caniinfected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) animals were sacrificed 18 days post-infection and (B) 24 days after infection. Asterisks indicate a significant difference from infected control and from animals treated with TRFK-5 (p< 0.01). <br><br> nophilia persists in lungs, probably by the secretion of IL-5 from cells localized in the lung microenvironment, <br><br> Histopathological analysis: The treatment of <span class="yellow">T. canis</span>-infected animals with irrelevant antibody showed a widespread eosinophilic infiltration as in untreated animals (Fig. 6A, B). However, the treatment of animals with TRFK-5 at the same time of infection, or 1 day or 3 days later ted to a complete inhibition of eosinophil infiltration in the lung parenchyma (Fig. 6C). By contrast, the mononuclear cell infiltration in the lungs was not modified. When the infected <span class="yellow">guinea-pigs</span> received TRFK-5 1 day before sacrifice (or 17 days post-infection), eosinophil infiltration in the lung parenchyma was also inhibited (Fig. 6D) but not to the same extent as observed in the group receiving TRFK-5 given at the time of infection or 3 days later. Thus, the histological determination of eosinophil infiltration in these lungs corroborates a reduction but not a sizeable inhibition of eosinophil numbers as observed in the BALF of the same infected animals, <br><br> poorly defined, although T lymphocytes and mast cells appear to be involved in eosinophilia. 9'2 IL-5, a T cell-derived factor that regulates B cell functions, is an eosinophil differentiation factor11 as well as a stimulating and survival-prolonging factor specific for eosinophils in vitro. 2 Also, several investigators have demonstrated that sTstemic eosinophilia in <span class="yellow">mice</span> infected with parasites is mediated by IL-5 produced in response to the infection. 2'22 In the present study, the i.p. administration of the TRFK-5 antibody markedly inhibited the widespread eosinophilia observed in <span class="yellow">T. canis</span>-infected <span class="yellow">guinea-pigs</span>, indicating that IL5 participated in a <span class="yellow">guinea-pig</span> model of VLMS eosinophJlia. Most of the <span class="yellow">T. canis</span> larvae which penetrated the intestinal wall had migrated into the liver within 72h after inoculation as demonstrated here and elsewhere. 2 It is apparently during this interval that the worm provides the signals to cytokine-producing cells, which in turn trigger increased serum levels of specific cytokine as demonstrated here for IL-5, 24 to 72 h after infection. The signals may be provided directly by the invading parasite or by cells in response to the parasite. The cytokine pattern that develops at this early stage, probably induced by a T-cell independent pathway, may also influence the pattern of T cell differentiation into a Th2 type, which may be responsible for the second peak of IL-5 observed in our experimental model (Fig. 3), although a second cycle of larval invasion (Fig. 2) with a rapid peak of IL-5 liberation cannot be ruled out. <br><br> Discussion The results of the present study show that in our experimental model widespread eosinophilia follows the infection of <span class="yellow">guinea-pigs</span> with second stage eggs from <span class="yellow">T. canis</span>, as also noted in <span class="blue">humans</span> and in other experimental animals. 7'7 <span class="yellow">T. canis</span> is a potent stimulus for systemic eosinophilia, since blood, BALF, peritoneum and all tissues examined (kidney, eyes, spleen, thymus, <br><br> Thus, our results suggest that the eosinophilia against helminth larvae may be initiated by the release of IL-5 when the parasites migrate from the intestine to the liver by stimulation of specific cell populations. Then, an early release of IL-5 quickly induces eosinophil recruitment, probably first from the stored mature eosinophil pool from vascular endothelium or by the mobilization of eosinophils from extravascular sites to the blood. This fact could explain why we found increased eosinophils first in blood and later in other compartments. The early IL-5 release may also serve as a signal for eosinophil differentiation and maturation in bone marrow. The time <br><br> inteeeal observed between the first peak of IL-5 release and the increase of eosinophils in blood coincides with that reported to be necessary for eosinophil differentiation and maturation in vitro. 12Increased eosinophil production and liberation into blood and other tissues occurs Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 29 <br><br> L. H. Faccioli et al. <br><br> FIG. 6. (A) photomicrographs of lung parenchyma from <span class="yellow">guinea-pigs</span> infected for 18 days with <span class="yellow">T. cani</span>,, (B) infected animals which were treated with irrelevant antibody at the time of infection; (C) infected animals which were treated with the mAb TRFK-5 at 3 days after infection; (D) mAb administration 17 days after infection. The animals were sacrificed 18 days after infection. Note the intense eosinophil infiltration into the lung in A and B, the inhibition of eosinophils in C and the reduction of eosinophils in D. <br><br> thereafter. Thus, early and later IL-5 release provides a necessary level of this cytokine, which is involved in the maintenance of eosinophilia. We may assume that the inhibition of the first peak of IL-5 release by TRFK-5 does not permit the subsequent T cell stimulation and differentiation. This may explain the long-lasting effect of TRFK5 treatment observed here and also reported by others. 8 In agreement with our results, there is an important observation of Svetic et al. 24 showing that a specific and highly reproducible IL-5 gene expression pattern is detectable in Peyer's patches by 6 to 12h after <span class="yellow">Heligmosomoides polygyrus</span> infection. The early increase in IL-5 gene expression after infection was probably T cell-independent, inasmuch as it was obseeeed in Peyer's patches of congenitally athymic <span class="yellow">mice</span> and of conventional <span class="yellow">mice</span> treated with anti-CD4 30 Mediators of Inflammation Vol 5 <br><br> and anti-CD8 mAb. Moreover, Kusama eta/. 25 have observed two peaks of eosinophilia in normal and athymic <span class="yellow">mice</span>, and suggested that IL-5 observed in the first peak was produced by cells other than CD4 T cells, since anti-CD4 and anti-CD3 mAb reduced only the second peak of eosinophilia in normal <span class="yellow">mice</span> and slightly reduced the first peak of eosinophilia in both normal and nu/nu <span class="yellow">mice</span>. The local lung cells producing IL-5 may also help us to explain the reason why 12, 17 or 18 days post-infection TRFK-5 treatment only partially inhibits, or does not inhibit eosinophil infiltration into the lungs, as demonstrated in Figs 5 and 6. We may suggest that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK<br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> 5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens have already stimulated some cell populations to produce IL-5, as demonstrated by Kusama et aL25These results suggest that eosinophilia in lungs is under the control of different factors when compared to that observed in blood and the peritoneal cavity. One of the most important results obtained here was the inhibition of circulating eosinophil numbers by the different mAb treatments, even when the antibody was given at the peak of blood eosinophilia, which was accompanied by an increase of mature eosinophils in bone marrow. This suggests that IL-5, apart from being required for the terminal differentiation of eosinophils in bone marrow, 26 is also likely to drive eosinophils from the bone marrow to the blood and then to the tissues, probably by upregulating VLA-4 expression in eosinophils. Moser et aL have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with cytokines such as IL-5, IL-3 or GM-CSF for expression of adhesion molecules such as VI-4. Recently, Pretolani et al. 27 have indeed shown that an anti-VLA-4 antibody suppresses eosinophil recruitment to lung in the <span class="yellow">guinea-pig</span> and, as a consequence, inhibits the accompanying bronchopulmonary hyperresponsiveness. <br><br> tant in protective immunity to a gastrointestinal nematode infection in <span class="yellow">mice</span>. Proc Natl Acad Sci USA 1991; 88= 5513-5517. 11. Yamaguchi A, Suda T, Suda J, et al. Purified interleukin (IL-5) supports the terminal differentiation and proliferation of <span class="yellow">murine</span> eosinophilic <br><br> precursors. J Exp Med 1988; 16'7: 43-56. 12. Yamaguchi Y, Hayashi Y, Sugama Y, et al. Highly purified <span class="yellow">murine</span> interleukin (IL-5) stimulates eosinophil function and prolongs in vitro survival. IL-5 as an eosinophil chemotactic factor. J Exp Med 1988; 16'7: 17371752. 13. Rothenberg ME, Petersen J, Stevens RL, Silberstein DS, McKenzie DT, Austen KF, Owen WF. IL-5-dependent conversion of normodense <span class="yellow">human</span> eosinophils to the hypodense phenotype uses 3T3 fibroblasts for enhanced viability, accelerated hypodensity, and sustained antibodydependent cytotoxicity. J Immuno11989; 143; 2311-2316. 14. Collins PD, Weg VB, Faccioli LH, Watson ML, Moqbel R, Williams TJ. Eosinophil accumulation induced by <span class="yellow">human</span> interleukin-8 in the <span class="yellow">guinea pig</span> in vivo. Immunology 1993; '7}; 312-318. 15. Sehmi R, Wardlavo AJ, Cromwell O, Kurihara K, Waltmann P, Kay AB. Interleukin-5 selectively enhances the chemotactic response of eosinophils obtained from normal but not eosinophilic subjects. Blood 1992; '79; 2952-2959. 16. Moser R, Fehr J, Bruijnzeel PLB. IL-4 controls the selective endotheliumdriven transmigration of eosinophils from allergic individuals. J Immunol 1992; 149: 1432-1438. 17. Olson LJ, Schulz CW. Nematode induced hypersensitivity reactions in <span class="yellow">guinea pigs</span>: onset of eosinophilia and positive Schultz-Dale reactions following graded infection with <span class="yellow">Toxocara canis</span>. Ann N Y Acad Sci 1963; 113; 440-455. 18. Kayes SG, Oaks <br><br> JA. Development <br><br> of the granulomatous response in <br><br> <span class="yellow">murine</span> toxocariasis. I. Initial events. Am J Patho11978; ,}3; <br><br> 277-294. <br><br> 19. Basten A, Beeson PB. Mechanisms of eosinophilia. II. Role of the lymphocyte. J Exp Med 1970; 131; 1288-1305. 20. Plaut M, Pierce JH, Watson CJ, Hanley-Hyde J, Nordan RP, Paul WE. Mast cell lines produce lymphokines in response to cross-linkage of Fc epsilon RI or to calcium ionophoras. Nature 1989; 339: 64-67. is 21. Sher A, Coffman RL, Hieny S, Scott P, Cheever AW. Interleukin required for the blood and tissue eosinophilia but not granuloma formation induced by infection with <span class="yellow">Schistosoma mansoni</span>. Proc Natl Acad Sci USA 1990; 8'7; 61-65. 22. Herndon FJ, Kayes SG. Depletion of eosinophils by anti-IL-5 monoclonal antibody treatment of <span class="yellow">mice</span> infected with <span class="yellow">Trichinella spiralis</span> does not alter parasite burden or immunologic resistance to reinfection. J Immuno11992; 149: 3642-3647. 23. Oshima T. Standardization of techniques for infecting <span class="yellow">mice</span> with Tox24. ocara canis and observations on the normal migration routes of the larvae. J Parasito11961; 4'7= 652. Svetic A, Madden KB, Zhou XD, et al. A primary intestinal helminthic infection rapidly induces a gut-associated elevation of Th2-associated cytokines and IL-3. J Immuno11993; 150: 3434-3441. Kusama Y, Takamoto M, Kasahara T, Takatsu K, Nariuchi H, Sugane K. Mechanisms of eosinophilia in BALB/c-nu/+ and congenitally athymic BALB/c-nu/nu <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Immunology 1995; 84; 461-468. Rennick DM, Thompson-Snipes L, Coffman RL, Seymour BWP, Jackson JD, Hudak S. In vivo administration of antibody to interleukin-5 inhibits increased generation of eosinophils and their progenitors in bone marrow of parasitized <span class="yellow">mice</span>. Blood 1990; '76: 312-316. Petrolani M, Ruffle C, Lapa e Silva JR, Joseph D, Lobb RR, Boris Vargaftig B. Antibody to very late activation antigen 4 prevents antigen-induced bronchial hyperreactivity and cellular infiltration in <span class="yellow">guinea pig</span> airways. J Exp Med 1994; 180= 795-805. <br><br> References 25. 1. Nutman <br><br> 2. <br><br> 3. 4. 5. <br><br> 6. 7. <br><br> 8. <br><br> 9. <br><br> TB, Ottesen EA, Cohen SG. The eosinophil, eosinophilia, and eosinophil-related disorders. Allergy Proc 1989; 10; 47-62. Glickman LT, Schantz PM. Epidemiology and pathogenesis of zoonotic toxocariasis. Epidem Rev 1981; 3= 230-250. Beaver P, Snyder H, Carrera G, Dent J, Lafferty J. Chronic eosinophilia due to visceral larva migrans. Pediatrics 1952; 9: 7-19. Limaye AP, Abrams JS, Silver JE, Ottesen EA, Nutman TB. Regulation of parasite-induced eosinophilia: selectively increased interleukin production in helminth-infected <span class="yellow">patients</span>. J Exp Med 1990; l'7:a; 399-402. Steel C, Nutman TB. Regulation of IL-5 in onchocerciasis: a critical role for IL-2. J Immuno11993; 15{}; 5511-5518. Yamaguchi Y, Matsui T, Kasahara T, et al. In vivo changes of hemapoietic progenitors and the expression of the interleukin gene in eosinophilic <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Exp Hemato11990; 18; 1152-1157. Parsons JC, Coffman RL, Grieve RB. Antibody to interleukin prevents blood and tissue eosinophilia but not liver trapping in <span class="yellow">murine</span> larval toxocariasis. Parasite Immuno11993; 15: 501-508. Coffman RL, Seymour BWP, Hudak S, Jackson J, Rennick D. Antibody to interleukin-5 inhibits helminth-induced eosinophilia in <span class="yellow">mice</span>. Science 1989; 245; 308-310. Sher A, Coffman RL, Hieny S, Cheever AW. Ablation of eosinophil and IgE responses with anti-IL-5 or anti-IL-4 antibodies fails to affect immunity against <span class="yellow">Schistosoma mansoni</span> in the <span class="yellow">mouse</span>. J Immuno11990; 145: 3911<br><br> 26. <br><br> 27. <br><br> ACKNOWLEDGEMENTS. This work was funded by grant 300652/85-2 from Conselho Nacional de Desenvolvimento Cientifico e Tecnol6gico (CNPq) and Grant 92/5105-7 from Fundago de Amparo ft Pesquisa do Estado de So Paulo (FAPESP). We wish to thank Mrs M.A. Fernandes for technical assistance, Mrs M.M.O. Rossi for the histological sections, M. Costa Gongalves for the photography artwork, and Dr P. Minoprio, Institut Pasteur, for kindly providing TRFK-5. <br><br> 3916. 10. Urban <br><br> Jr JF, <br><br> Katona IM, Paul WE, Finkelman FD. Interleukin 4 is impor<br><br> Received 12 October 1995; accepted 17 November 1995 <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 31 <br><br>  <h3>pmcA2430230</h3>Emergence of Delayed Methylmercury Toxicity after Perinatal Exposure in Metallothionein-Null and Wild-Type C57BL <span class="yellow">Mice</span>
Abstract
Background
Although a long latency period of toxicity after exposure to methylmercury (MeHg) is known to exist in <span class="blue">humans</span>, few animal studies have addressed this issue. Substantiation of delayed MeHg toxicity in animals would affect the risk evaluation of MeHg.<br><br>Objectives
Our goal in this study was to demonstrate the existence of a latency period in a <span class="yellow">rodent</span> model in which the toxicity of perinatal MeHg exposure becomes apparent only later in life. Our study included metallothionein (MT) knockout <span class="yellow">mice</span> because studies have suggested the potential susceptibility of this strain to the neurodevelopmental toxicity of MeHg.<br><br>Methods
Pregnant MT-null and wild-type C57Bl/6J <span class="yellow">mice</span> were exposed to MeHg through their diet containing 5 μg Hg/g during gestation and early lactation. We examined behavioral functions of the offspring using frequently used paradigms, including open field behavior (OPF), passive avoidance (PA), and the Morris water maze (MM), at ages of 12–13 and 52–53 weeks.<br><br>Results
At 12 weeks of age, behavioral effects of MeHg were not detected, except for OPF performance in MeHg-exposed MT-null females. At 52 weeks of age, the MeHg-exposed groups showed poorer performance both in PA and MM, and their OPF activity differed from controls. These effects of MeHg appeared exaggerated in the MT-null strain. The brain Hg concentration had leveled off by 13 weeks of age.<br><br>Conclusions
The results suggest the existence of a long latency period after perinatal exposure to low-level MeHg, in which the behavioral effects emerged long after the leveling-off of brain Hg levels. Hence, the initial toxicologic event responsible for the late effects should have occurred before this leveling-off of brain Hg.<br><br><br><br>Methylmercury (MeHg) poses serious and practical concerns for <span class="yellow">human</span> populations regarding perinatal exposure. Fish, especially large predator (carnivore) fish species, accumulate high concentrations of MeHg through the marine food chain, and exposure of pregnant <span class="yellow">women</span> to MeHg through the consumption of fish has evoked widespread concern due to potential effects on offspring. Two large-scale cohort studies in fish-eating populations of Seychelles and Faroe islanders are being conducted; although the former has not found consistent adverse developmental effects of MeHg (Myers et al. 2007), the latter has reported adverse effects (Debes et al. 2006). Fundamental reasons for this discrepancy have not been completely elucidated, and many questions remain regarding the neurotoxicity of MeHg, despite extensive study.
Among the unanswered questions is whether there is a long latency period for behavioral manifestations after exposure to MeHg (Clarkson and Magos 2006; Landrigan et al. 2005; Rice 1996; Weiss et al. 2005a, 2005b). Typical examples of latent toxicity in <span class="blue">humans</span>, including both acute and chronic MeHg exposures, have been described in detail elsewhere (Weiss et al. 2005a). Davidson et al. (2006) recently suggested that effects of perinatal exposure to MeHg may emerge 9 years after birth in the Seychelles cohort. Consequently, risk assessments of MeHg exposure could be inaccurate because studies (<span class="yellow">human</span> or animal) usually do not focus on later stages of life and therefore could miss delayed effects. The possibility of delayed toxicity is exemplified by the expanded Barker hypothesis, which posits that the origin of some neurodegenerative diseases such as Parkinson and Alzheimer diseases lies in early exposure to environmental chemicals (Landrigan et al. 2005). Although epidemiologic evidence would be ideal for exploring the possibility of delayed toxicity (and, indeed, data from epidemiologic studies form the basis of current risk assessment for developmental toxicity of MeHg), considering the complex effects of numerous potential confounders and the existence of multiple exposures in <span class="yellow">human</span> populations, animal models would likely make important contributions to this field.
Although numerous animal studies have described the developmental neurotoxicity of MeHg (Watanabe and Satoh, 1996), few have addressed the latency issue. Few studies have evaluated the neurobehavioral effects in rodents longitudinally beyond 6 months after perinatal exposure. Spyker (1975) addressed this issue in her pioneering work, reporting the late development of behavioral toxicity in <span class="yellow">mice</span> prenatally exposed to MeHg; it appeared, however, that the substantial mortality and retarded growth among the exposed <span class="yellow">mice</span> were apparent before weaning, indicating that the doses used (even though some lower dose levels were included) exerted severe toxicity. Using a relatively complex schedule-controlled operant behavior method, <span class="yellow">rats</span> whose parents were exposed to MeHg (0.5 or 6.4 mg/L) from 4 weeks before mating and continuing to postnatal day (PND) 16 were shown to be less sensitive to a change in the reinforcement schedule than were their nonexposed counterparts at 28–32 months of age (Newland et al. 2004). <span class="yellow">Mice</span> that were perinatally exposed to 1 or 3 mg/L MeHg in drinking water did not show significant deviation from controls in behavioral performance (motor performance, memory, and learning) at 5, 15, or 26 months of age, but the lifetime-exposed groups did show a significant deviation (Weiss et al. 2005c). The existence of a latency period (i.e., the absence of effects earlier in life followed by the emergence of effects at a later stage of life) has not been demonstrated in any <span class="yellow">rodent</span> study. In nonhuman primates, delayed emergence of the signs of neurotoxicity was observed several years after the cessation of a 7-year postnatal exposure (Rice 1996).
Metallothionein (MT) protects against the toxicities of a variety of metals. We examined the neurotoxicity and developmental toxicity of metallic Hg in MT I/II-knockout <span class="yellow">mice</span> (Yoshida et al. 2004) and demonstrated the susceptibility of this genetically manipulated strain to the toxicity of metallic Hg. In contrast to metallic Hg, MeHg does not induce MT, and MT would not substantially influence the kinetics of MeHg (Yasutake et al. 1998). Several reports, however, have demonstrated protective effects of MT against MeHg toxicity, which was ascribed to the radical scavenging effect of MT (Yao et al. 2000). We also showed that perinatal exposure to MeHg results in altered metabolism of thyroid hormones in neonates that was more distinct in MT-null strains than their wild-type counterparts (Mori et al. 2006). The vulnerability of the MT-null strain suggests that delayed neurobehavioral toxicity due to MeHg, if it does exist, might be more distinctive in this strain.
By utilizing the MT-null strain, we aimed to answer the following two questions: First, could we generate a model in which the toxicity of MeHg would emerge or at least become exaggerated later in life as opposed to earlier in life (i.e., at 3–6 months, which was the timing for most of the earlier studies that used behavioral evaluations)? Second, would the MT-null strain be affected more than its parent C57BL strain? Answering either of these questions not only could influence the risk evaluation of MeHg, but it could also lead to a better understanding of the mechanism of toxicity for perinatal MeHg exposure. To address these issues, we used three behavioral paradigms, the open field (OPF), passive avoidance (PA), and Morris (water) maze (MM) tests, which are often used in this field and which we used in our previous studies on the effects of Hg vapor (Yoshida et al. 2004, 2006). Performances in the MM and PA are said to be the most sensitive to aging (Gower and Lamberty 1993).
We used a dose of 5 μg MeHg/g in the diet, which resulted in a brain Hg level relevant to <span class="yellow">human</span> risk assessment. We evaluated the behavioral end points twice, once around 3 months of age and the other time around 1 year; the latter time roughly corresponds to the period when many behavioral performances, including OPF (Acevedoa et al. 2006; Carrie et al. 1999; Gower and Lamberty 1993), PA (Gower and Lamberty 1993), and MM (Bach et al. 1999; Carrie et al. 1999), show alterations in this <span class="yellow">mouse</span> strain.
Materials and Methods
Animals and MeHg exposure
OLA129/C57BL/6J strain <span class="yellow">mice</span> (wild type) and MT I/II-knockout <span class="yellow">mice</span> (MT-null) of this strain were provided by K.H. Choo of the Murdoch Institute, Parkville, Australia (Michalska and Choo 1993) and were of a mixed genetic background of 129/Ola and C57BL/6 strains. F1 hybrid <span class="yellow">mice</span> were mated with C57BL/6 <span class="yellow">mice</span> for six generations at the National Institute for Environmental Studies (Tsukuba, Japan). At 10 weeks of age, single male and female <span class="yellow">mice</span> were allowed to cohabit; every female <span class="yellow">mouse</span> was checked each morning for the presence of a vaginal plug. When a plug was confirmed, the day was designated as day 0 of gestation (GD0).
The diet, NIH-07PLD formula (CLEA Japan, Inc., Tokyo, Japan), contained vitamins and trace elements as follows (per kilogram diet): 3.2 mg CuSO4, 88 mg FeSO4, 149 mg MnSO4, 25 mg ZnCO3, 1.6 mg Ca(IO3)2, 11 mg vitamin B1, 4.7 mg vitamin B2, 1.9 mg vitamin B6, 44 mg vitamin E, in addition to 5 μg MeHg/g. This diet was fed to the pregnant <span class="yellow">mice</span> starting from GD0 through 10 days after delivery (i.e., PND10). Thereafter, we switched <span class="yellow">mice</span> to a diet that did not contain MeHg. We chose GD0 as the beginning of exposure because exposures that started before conception often resulted in fairly high Hg concentrations in fetal/neonatal brains (Kakita et al. 2003), and we chose PND10 to cover the early neonatal period, in which considerable brain growth occurs. In our experimental setting, the neonatal <span class="yellow">mice</span> started to eat from the diet bucket and drink from the water bottle from PND10 onward. Control <span class="yellow">mice</span> were kept on the same diet but without MeHg (< 0.01 μg Hg/g). On PND1, to avoid the confounding effects due to different litter size, we reduced each litter to six pups (three males and three females when possible), and on PND10, two males and two females from each litter were killed for chemical analyses.
The remaining male and female offspring per litter were weaned on PND28 and used for subsequent behavioral analyses (either at 12–13 weeks or 52–53 weeks, depending on the litter) as described below. We measured body weights of the weaned <span class="yellow">mice</span> every 2 weeks. Thus, four experimental groups were used (with or without MeHg exposure for two strains), and each group consisted of 12–13 litters. For half of the litters, the behavioral analyses were conducted at 12–13 weeks of age, and upon completion of the behavioral analyses, the animals were killed under ether anesthesia. We then dissected the organs (brain, liver, and kidneys) for Hg analyses. For the remaining half of the litters, we conducted the behavioral tests at 52–53 weeks of age. The <span class="yellow">mice</span> were treated humanely and with regard to alleviation of suffering according to the National Institute for Environmental Studies’ Guidelines for Animal Welfare and the guidelines of St. Marianna University.<br><br>Behavioral evaluations
The details of each behavioral procedure have been described elsewhere (Yoshida et al. 2006). Brief descriptions follow.
For the OPF test, we used an OPF apparatus (Ohara Co., Ltd., Tokyo, Japan) with a 60 × 60-cm floor surrounded by walls 60 cm high. The experimental room light was turned off, and a dim light of 80 lux was lit during the experiment. We placed a <span class="yellow">mouse</span> in the center of the floor and monitored its behavior for 10 min using a CCD camera connected to a computer. The position of the center of gravity was calculated by image-analyzing software, which was used to calculate the total distance traveled by the <span class="yellow">mouse</span> as well as the positional preference (either center or peripheral, where peripheral was defined as the area within 10 cm from the wall). We cleaned the OPF apparatus with 70% ethanol between trials.
The apparatus for the PA test (Ohara Co. Ltd.) consisted of a light compartment that illuminated by a 400-lux light and a dark compartment with black opaque walls and lids. The two compartments were separated by a mobile guillotine door. On the first day (training trial), we placed each <span class="yellow">mouse</span> in the light compartment facing away from the guillotine door, which was closed. After 30 sec of introduction, the door was opened; when the <span class="yellow">mouse</span> entered the dark compartment, a brief electric shock (4 mA for 2 sec) was delivered through the metal grid floor. This would force the <span class="yellow">mouse</span> back to the light component. The interval between the opening of the door and the entry to the dark room (in seconds) was recorded and defined as the latency. On the next day, the same procedure was repeated, but without the electric shock (retaining trail). In this PA paradigm, aversive learning was assumed to be established in the training trial, and we used its retention in the retaining trail as the index of learning. Between each individual trial, we cleaned the apparatus with ethanol. The cutoff time of the retention session was 300 sec.
The MM test apparatus (Ohara Co., Ltd.) was a round-shaped water pool with a diameter of 120 cm. A small platform was submerged in the water, which provided a place for <span class="yellow">mice</span> to escape from the water (i.e., an aversive stimulus, water temperature = 23 ± 1°C). The water was made opaque by adding white paint so that the <span class="yellow">mouse</span> could not see the submerged platform. In each trial, we released a <span class="yellow">mouse</span> into the pool from a determined position along the wall, and the performance of the <span class="yellow">mouse</span> was monitored by a CCD camera/image analyzer. The time required to reach the platform was recorded. If a <span class="yellow">mouse</span> could not find the platform within 60 sec after release, it was led to the platform and placed on it for 20 sec before being removed. In these cases, a latency of 60 sec was recorded. We conducted the trial once a day up to the fifth day for each <span class="yellow">mouse</span>, and the order of each <span class="yellow">mouse</span> was counterbalanced across the day. On the sixth day, a transfer test (or probe test), which is a trial without the platform, was conducted; in this procedure, we counted the number of times that the <span class="yellow">mouse</span> crossed the position where the platform had been.<br><br>Tissue Hg concentration
The tissue samples were homogenized (10% weight/volume) in distilled water using a Polytron homogenizer (Kinematica GmbH, Littau, Switzerland). We determined Hg levels in the homogenates by the oxygen combustion–gold amalgamation method (Ohkawa et al. 1977) using an atomic absorption Hg detector MD-1 (Nippon Instruments, Co. Ltd., Osaka, Japan). To ensure the accuracy of the measurement, we included reference material from a dogfish (DORM-2; National Research Council of Canada, Ottawa, Ontario, Canada) with a certified value of 4.64 ± 0.26 μg/g in the analyses; the observed values fell within the certified range. The detection limit of the measurement was 0.1 ng Hg.<br><br>Statistics
We analyzed behavioral data for OPF and PA by analysis of variance (ANOVA), taking sex, strain, and MeHg exposure as the factors. All the interactions among these factors were put into the model. When any of the interactions was highly significant, we analyzed the data separately in an appropriate way; for example, if sex × Hg was significant, the data for males and females were separately analyzed for the effects of strain and Hg. Whenever appropriate, ANOVAs were followed by Mann-Whitney U or Student’s t-tests, depending on the nature and distribution of the variables. We analyzed data for the MM by repeated-measures ANOVA, taking the exposure as between-group and trials as within-group variables. The test was performed for each of the four sex and strain combinations separately. The significance level was set at p < 0.05.<br><br>
Results
Body weight
Up to 20 weeks of age, the body weight values of the control and MeHg-exposed groups were not different, regardless of strain or sex. After 28 weeks, except for the wild-type female groups, the MeHg-exposed groups weighed significantly less than the controls (Figure 1).<br><br>OPF
At 12 weeks of age, a three-way ANOVA of the locomotion distance revealed that only strain was a significant factor (p < 0.001), reflecting the longer distance traveled by the MT-null <span class="yellow">mice</span> (Figure 2A). Strain was also a significant factor for the proportion of the central-area locomotion (Figure 3A), and Hg exposure marginally affected this outcome. In MT-null females, the proportion of central-area locomotion was higher in MeHg-exposed <span class="yellow">mice</span> than in the controls; this difference was not observed in any other strain–sex combination. At 52 weeks of age, the strain × Hg interaction was highly significant (p < 0.001) in an ANOVA of locomotion distance (Figure 2B); MeHg exposure was associated with decreased locomotion distance in wild-type <span class="yellow">mice</span> and with increased distance in MT-null <span class="yellow">mice</span>. A strain-wise two-way ANOVA (with sex and Hg as the factors) revealed that only Hg was significant in both strains (p < 0.01). Regarding the proportion of the central-area locomotion, the effects of MeHg appeared to depend on sex [i.e., sex × Hg was highly significant (p < 0.001) in the three-way ANOVA; Figure 3B]. Indeed, a sex-wise two-way ANOVA showed significant effects of Hg only in females (p < 0.001).<br><br>PA
At 12 weeks of age, all the groups showed prolonged latency in the second (retention) trial, and no consistent effect of MeHg was recognized regardless of strain or sex (Figure 4A).
At 52 weeks of age, a three-way ANOVA revealed a significant interaction between strain and Hg (p < 0.05); strain-wise two-way ANOVAs revealed a significant effect of Hg on learning in MT-null <span class="yellow">mice</span> of both sexes; these groups of <span class="yellow">mice</span> showed significantly shorter (less than half) latency times compared to control <span class="yellow">mice</span> (Figure 4B). A notable difference between the results at 52 weeks of age and those at 12 weeks was that many of the tested <span class="yellow">mice</span> exceeded the cutoff time in the retention trials at 52 weeks, except for the MT-null groups.<br><br>MM
At 13 weeks of age, repeated-measures ANOVA did not indicate any effects of MeHg (Figure 5A). At 52 weeks of age (Figure 5B), wild-type males and MT-null females shared the same statistical results; Hg as well as the Hg × trial interaction were statistically significant. Thus, in both cases, the MeHg groups showed a longer latency, hampering learning performance.<br><br>Tissue Hg concentration
At PND10, which was immediately after the exposure, brain Hg concentrations of the neonatal <span class="yellow">mice</span> were approximately ≤ 0.5 μg/g (Table 1). Although the MT-null <span class="yellow">mice</span> and females showed slightly higher brain Hg concentrations than the corresponding wild-type group and males, respectively, neither of these differences was significant. At 13 weeks, when the behavioral tests were completed, the brain Hg concentration was comparable to the control (nonexposed) level (approximately 5 ng/g in both the exposed and control brains). Interestingly, MT-null <span class="yellow">mice</span> had a significantly lower brain Hg concentration than the corresponding wild-type groups.<br><br>
Discussion
Results of the present study demonstrate the delayed emergence of neurobehavioral toxicity due to perinatal MeHg exposure, which presumably developed after brain MeHg concentrations had leveled off. This emergent toxicity was exaggerated in MT-null <span class="yellow">mice</span> and was more distinct in females. To our knowledge, our findings show the first clear-cut demonstrations of a long latency period of MeHg neurobehavioral toxicity in rodents and possible genetic susceptibility for the emergent toxicity.
The exposure level should be considered before discussing the end points. On PND10, immediately after the cessation of MeHg exposure, the brain Hg concentration was approximately 0.5 μg/g, regardless of the strain or sex. In a previous study, the brain Hg concentration in <span class="yellow">mice</span> perinatally exposed to 6 mg MeHg/L (via water) peaked between PND0 and PND4 and was approximately three times higher than on PND21 (Goulet et al. 2003). Therefore, the peak brain Hg concentration, which is presumably observed around birth, can be estimated as about 3-fold higher than that on PND10 and would be approximately 1.5 μg/g (0.5 μg/g × 3), which is one of the lowest levels among recent <span class="yellow">rodent</span> studies. As shown by Sakamoto et al. (2002) in their Figure 2, prenatal exposure of <span class="yellow">rats</span> to MeHg showed a peak brain Hg concentration around PND1 that was four to five times greater than the level on PND10. Also, Newland and Rasmussen (2000) reported a slight alteration of a complex operant behavior in <span class="yellow">rats</span> at ages < 2 years at brain Hg concentrations as low as 0.5 μg/g at birth, although statistical significance of this particular effect was not clear. It should be noted that <span class="yellow">rats</span> have different Hg kinetics (Hirayama and Yasutake 1986; Yasutake and Hirayama 1990) due to the high affinity of <span class="yellow">rat</span> hemoglobin for MeHg (Clarkson and Magos 2006).
The most important observation of the present study was that the effects of low-level MeHg exposure were detected only at later stages in the lives of the <span class="yellow">mice</span>. Except for the central-area occupancy in OPF in MT-null females, no statistically significant effects of MeHg were observed in any of the three behavioral tests at 12 weeks of age. In contrast, significant effects were observed in all three tests at 52 weeks of age. The brain Hg concentration of the exposed groups had leveled off and was not distinguishable from the non-exposed group at 13 weeks of age, immediately after the first phase of the behavioral testing. Therefore, in the present study, there was a latency period in which the dose and effects could not be detected, although effects were observed 9 months later. Another notable observation was that the emergent manifestation of toxicity was also recognized in the suppression of body weight (except for wild-type females), which only became apparent on or after 28 weeks of age.
The existence of a latency period of as long as several years after chronic (7 years from birth), low-level exposure to MeHg has been described in nonhuman primates (Rice 1996). In that case, however, the Hg concentration in the brain remained elevated, presumably as a result of the long exposure. Indeed, Rice (1996) argued that the minute amount of residual brain Hg could have caused the delayed toxicity. This was clearly not the case in the present study because the Hg concentration leveled off around the time of the first phase of the behavioral study. The absence of behavioral effects at 12 weeks of age ruled out the possibility that the residual behavioral effects were due to elevated Hg early in life. Therefore, the behavioral toxicity that surfaced at 52 weeks of age must have had its origin before the brain Hg concentration leveled off (at or before 13 weeks of age), although the redistribution of Hg to the brain from other sites of deposition, such as the liver, cannot be completely excluded. The long silent period before the manifestations of toxicity emerged suggests that a slow process plays a role in this latent toxicity.
Although an example of a slow process is aging, 52 weeks of age might not be sufficiently old for a <span class="yellow">mouse</span> to be considered aged in the physiologic sense because C57BL/6 <span class="yellow">mice</span> have a relatively long life span among <span class="yellow">mouse</span> strains [median survival of 27–31 months (Gower and Lamberty 1993)] and a survival rate at 18–19 months as high as 90% (Institute on Aging HP) (National Institute on Aging 2008). Nevertheless, various behavioral examinations have shown age-related changes in the performance of <span class="yellow">mice</span> at approximately 1 year of age in the OPF (Acevedoa et al. 2006; Carrie et al. 1999), MM (Bach et al. 1999; Carrie et al. 1999), and PA (Gower and Lamberty 1993) tests. The observed effects of MeHg, including the deterioration in the PA and MM and suppression in the OPF (in wild-type <span class="yellow">mice</span>), were consistent with these reported effects of aging on behavioral function (in the sense described above), except for the increased OPF activity in the MT-null <span class="yellow">mice</span>.
Regardless of its neural basis, the basis of neurobehavioral toxicity should be sought in early life stages when the brain Hg concentration is highly elevated (approximately 1.5 μg/g at its peak). Some in vivo experiments have demonstrated several candidate mechanisms of perinatal exposure to MeHg, including abnormal migration of neurons and/or glias (Kakita et al. 2003; Rodier et al. 1984), but at higher Hg concentrations. Using exactly the same exposure protocol as the present study, we found suppressed activity of type III iodo-thyronine deiodinase, a thyroid hormone-metabolizing enzyme, in the brains of PND10 <span class="yellow">mouse</span> neonates (Mori et al. 2006), consistent with our previous study of higher MeHg doses (Watanabe et al. 1999). This perturbation could be one of the candidate mechanisms responsible for the later anomalous behaviors because even a transient change in thyroid hormones during the critical period of perinatal life exerts long-term consequences (Auso et al. 2004).
The effects of MeHg at 52 weeks of age were influenced by two potential modifying factors, sex and strain. In the OPF, while the locomotion was affected in both strains (although the direction was opposite), center occupancy was significantly increased only in the MT-null <span class="yellow">mice</span>. In addition, the effects on PA were significant only in MT-null <span class="yellow">mice</span>, whereas MeHg at a higher dose was reported to worsen PA performance in <span class="yellow">rats</span> (6–8 weeks of age; Sakamoto et al. 2002). In addition, body weight gain was suppressed in both male and female MT-null <span class="yellow">mice</span>, whereas in wild-type <span class="yellow">mice</span> the suppression was observed only in males. Taken together, the MT-null strain appeared to be slightly more susceptible to the late-emergent effects of MeHg. Several lines of evidence have shown that MT-I,II is protective against the toxicity of MeHg (Leiva-Presa et al. 2004; Yao et al. 2000), and the present results were basically consistent with these reports. We have also reported the susceptibility of the MT-null strain to the neurotoxic effects of metallic Hg (Yoshida et al. 2004, 2006).
The difference in the susceptibility to MeHg between sexes is still debated (Clarkson and Magos 2006; National Research Council 2000; Vahter 2007). In the present study, some responses to MeHg were different between the sexes, including OPF performance at 12 and 52 weeks of age and MM performance at 52 weeks. The fact that the MT-null female group was the only group affected by MeHg at 12 weeks may suggest the particular susceptibility of females in this strain. This point needs to be clarified in further experiments.
The question remains of why MT-null <span class="yellow">mice</span> are susceptible to the delayed neurotoxicity of perinatal MeHg. Apparently kinetics play only a minor role because the strain did not show distinct effects on the brain Hg concentration at PND10. The significantly lower brain Hg concentration in MT-null <span class="yellow">mice</span> compared with corresponding wild-type <span class="yellow">mice</span> at 13 weeks of age indicated that MT-I,II might play a significant role in the retention of Hg (or MeHg). This is consistent with the results of studies of metallic Hg exposure (Yoshida et al. 2004, 2005); a lower brain Hg concentration may not guarantee lower toxicity, supporting the protective role of the protein. Earlier studies suggest that brain MT-I,II has an important role both in the response to oxidative injury (Potter et al. 2007) and in the process of aging (Kojima et al. 1999). Therefore, the lack of MT can exaggerate the toxicity of MeHg by enhancing the initial effects due to oxygen radicals and/or by accelerating functional aging. Apart from this, an intriguing possibility is that the brain-specific isoform, MT-III, might contribute to the results we obtained because the expression of MT-III, together with MT-I, is increased in the brain of old <span class="yellow">rats</span>, resulting in the low availability of free zinc for synapses (Mocchegiani et al. 2004). The age-dependent expressions of MT isoforms might be modified in MT-null <span class="yellow">mice</span>.
Results of the present study might allow the possibility of alternative interpretations due to some potentially confounding factors. For example, except for the wild-type females, we observed significant differences in body weight between MeHg-exposed and non-exposed groups. Because these differences only became clear later in life, they might be associated with the toxicity that also emerged later in life. Manipulation of body weight in rodents alters activity levels, although the reported results are not always consistent with each other (Harrison and Archer 1987; Samuelsson et al. 2008). Also, the differential performance in PA could be related with the potential effects of MeHg on (electric) shock sensitivity, which we did not examine. At least one high-dose study with adult <span class="yellow">rats</span> showed reduced electric sensitivity due to mercury exposure (Wu et al. 1985). These possibilities need to be addressed in future studies.
To summarize, the present results suggest that an initial (or triggering) toxicologic event occurs before the brain Hg concentration stabilizes and that the nature of this event should be either an acceleration of the aging process or interaction with the aging process. Thus, by identifying the physiologic events associated with the functional aging of the examined behavioral tasks, the fundamental toxicologic scar might be revealed.<br><br>
<h3>pmcA2526157</h3>Identification and Analysis of Co-Occurrence Networks with NetCutter
Abstract
Background
Co-occurrence analysis is a technique often applied in text mining, comparative genomics, and promoter analysis. The methodologies and statistical models used to evaluate the significance of association between co-occurring entities are quite diverse, however.<br><br>Methodology/Principal Findings
We present a general framework for co-occurrence analysis based on a bipartite graph representation of the data, a novel co-occurrence statistic, and software performing co-occurrence analysis as well as generation and analysis of co-occurrence networks. We show that the overall stringency of co-occurrence analysis depends critically on the choice of the null-model used to evaluate the significance of co-occurrence and find that random sampling from a complete permutation set of the bipartite graph permits co-occurrence analysis with optimal stringency. We show that the Poisson-binomial distribution is the most natural co-occurrence probability distribution when vertex degrees of the bipartite graph are variable, which is usually the case. Calculation of Poisson-binomial P-values is difficult, however. Therefore, we propose a fast bi-binomial approximation for calculation of P-values and show that this statistic is superior to other measures of association such as the Jaccard coefficient and the uncertainty coefficient. Furthermore, co-occurrence analysis of more than two entities can be performed using the same statistical model, which leads to increased signal-to-noise ratios, robustness towards noise, and the identification of implicit relationships between co-occurring entities. Using NetCutter, we identify a novel protein biosynthesis related set of genes that are frequently coordinately deregulated in <span class="yellow">human</span> cancer related gene expression studies. NetCutter is available at http://bio.ifom-ieo-campus.it/NetCutter/).<br><br>Conclusion
Our approach can be applied to any set of categorical data where co-occurrence analysis might reveal functional relationships such as clinical parameters associated with cancer subtypes or SNPs associated with disease phenotypes. The stringency of our approach is expected to offer an advantage in a variety of applications.<br><br><br><br>Introduction
Biological research has experienced a paradigm shift in the last decade catalyzed by the availability of genome sequences and the resulting development of high-throughput technologies. The large data volumes produced by these novel technologies are often published as supplementary material and/or stored in extensive data repositories [1]. Functional interpretation of these data is an ongoing challenge. Co-occurrence analysis, based on the hypothesis that co-occurring entities are functionally linked, is a technique that has been used in three main areas of biological research:
Co-occurrence of genes in sequenced genomes relies on the fact that proteins do not function in isolation and are dependent on other proteins, either as direct binding partners, or as catalysts of substrates. Thus, when two proteins significantly co-occur in a large number of genomes or can be observed as fusion proteins in a subset of genomes, they are likely to be binding partners or enzymes needed for a specific metabolic pathway. Examples of those studies have been reported by [2]–[7].
Text mining is a quickly evolving field that aims at developing technologies helping to cope with the functional interpretation of large volumes of publications. Co-occurrence of gene names in publication abstracts, entire publications, or other gene-related databases has been used to derive co-occurrence networks with clear evidence that edges in those networks are reflecting functionally relevant relationships [8]–[11]. Gene names have also been analyzed for co-occurrence with other entities such as mutations [12], chemical compounds [13], and disease related keywords [14]. From the resulting networks, hypotheses about candidate genes involved in inherited diseases and drug targets can be derived. Clustering of gene related publications using keywords has been applied to enhance the quality of gene expression clusters [15], [16]. More general (non gene-centric) approaches try to organize the literature into functional areas based on co-occurrence of MeSH terms, keywords, diseases, phenotypes, chemicals, and similar objects of biomedical research interest [17]–[21].
Co-occurrence analysis of transcription factor binding motifs has been carried out in a variety of slightly differing ways in a wide range of organisms, including <span class="blue">humans</span>. [22]–[33]. The underlying hypothesis is that co-regulated genes, identified usually by gene expression studies, should contain specific combinations of transcription factor binding motifs in their upstream regulatory regions, the identification of which would allow the reverse-engineering of transcription regulatory networks [34].
We have recently applied co-occurrence analysis to studying published gene expression signatures and showed that co-occurrence patterns of genes reflect cancer signaling pathways [35]. Although co-occurrence analysis has a respectable history, the methodologies used in the studies mentioned above could not be easily applied to studying gene expression signatures. There are three main reasons that dictated the use of a different approach. First, gene expression signatures can vary in size by orders of magnitude. Obviously, the larger a signature the more likely it is to find two or more genes co-occurring in that signature. Thus, the significance of co-occurrences must be evaluated in the presence of considerable heterogeneity of co-occurrence probabilities among gene lists. As a consequence, the statistics used to evaluate the significance of co-occurrence events must reflect this heterogeneity. In particular, it must be based on list-specific co-occurrence probabilities. Second, in the vast majority of previous studies, co-occurrence is analyzed for pair-wise combinations of co-occurring entities. We found that the resulting stringency of this approach is not adequate for the analysis of published gene expression signatures [35]. Third, the null-model against which the significance of co-occurrences is tested does not work well for gene expression signatures. A common procedure is to use generic randomization of the entire data set under analysis or to select subsets of data entries randomly for comparison purposes. However, gene expression signatures are composed of distinct gene sets and the null-model must maintain this property, which is not guaranteed using these approaches. Furthermore, the list-specific nature of co-occurrence probabilities cannot be dealt with properly.
NetCutter was developed to address these challenges and to provide a generic tool for generating and analyzing co-occurrence networks. Although NetCutter has been developed for the analysis of gene expression signatures, it is based on abstract concepts that make it applicable to a wide variety of problems. The input is represented by a bipartite graph that is composed of list-entry pairs, which are stored in tab-separated text format. Co-occurrence of entries in lists is analyzed using pair-wise or higher order combinations of entries. The significance of co-occurrence is tested using a novel bi-binomial approximation of Poisson-binomial statistics (which is a binomial distribution with trial specific probabilities) that handles list-length-heterogeneity properly and provides a novel measure of association that is found to be superior to the Jaccard and the uncertainty coefficients. Occurrence probabilities are obtained from an edge-swapping procedure that maintains vertex degrees in the underlying bipartite graph and distinct sets of entries per list. As we shall see below, this procedure has a number of advantages over other possible null-models and permits co-occurrence analysis with near maximum stringency. Last but not least, NetCutter is equipped with a number of algorithms for the identification of network communities, vertex ranking, and convenience tools needed in the analysis of co-occurrence networks, or any undirected graph. We illustrate the utility of NetCutter in the identification of corresponding clusters of genes and publications from the PubLiME data set. PubLiME (Published Lists of Microarray Experiments) is a repository of published cancer related gene expression signatures (http://bio.ifom-ieo-campus.it/Publime). The concept of cluster correspondence follows from the bipartite graph representation of the data. Reversing the list-entry order in the bipartite graph permits identifying communities of entries as well as communities of lists. We show that communities of publications corresponding to communities of genes in the PubLiME data set can be used to generate hypotheses about the putative function of gene communities.<br><br>Results
The bipartite graph model of co-occurrence analysis
Co-occurrence analysis using NetCutter is based on the abstraction of list-entry pairs. Any entity that co-occurs with some other entity must be confined to some sort of container where co-occurrence is observed. For example, in the case of gene name co-occurrence in PubMed abstracts, the abstract is the container and the gene names are the co-occurring entities. Similarly, co-occurrence of transcription factor binding motifs is observed in gene promoters. The promoters are the containers where motif entities co-occur. The containers generally host more than one entity (otherwise co-occurrence would be impossible) and can be conveniently interpreted as lists. The co-occurring entities are the list entries. Lists and entries form a bipartite graph with one part of the graph representing lists and the other part representing entries. The presence of a given entry in a given list is indicated by an edge between the corresponding list and entry vertices. It is required that each entry can be linked to the same list only once. Without loss of generality, let's consider genes as entries and PubMedID_listIDs as lists in the following, unless otherwise specified (Fig. 1A). This interpretation of lists and entries has been applied in the co-occurrence analysis of published gene expression signatures [35].<br><br>Occurrence probabilities and null-models
A prerequisite for co-occurrence analysis is the availability of occurrence probabilities of genes per list. The occurrence probabilities can be derived from randomizing the bipartite graph and are dependent on the choice of the null-model. A null-model creates an occurrence probability matrix where the occurrence probability for each list–gene pair is listed. As a general property of this matrix, the sum of all matrix elements must equal the number of edges in the bipartite graph. This is because each edge is linked to either side of the bipartite graph with certainty and therefore the sum of occurrence probabilities over all lists (which can be calculated as the row sum if genes are listed vertically or as the column sum if genes are listed horizontally) followed by summing the results over all genes must be 1 for every edge. The number of matrix elements is given by #genes*#lists and therefore the average occurrence probability for any null-model must be #edges/(#genes*#lists). As a consequence, different null-models will only be distinguished by the way they attribute occurrence probabilities to vertices with different vertex degrees but not by the average occurrence probability.
We consider six different strategies to randomize the bipartite graph. First, we could reconnect all edges of the graph randomly. The probability of being connected by an edge for a given list-gene pair is given by (1/#genes)*(1/#lists). Since there are #edges edges to be reconnected, the occurrence probability for a single list-gene pair is #edges/(#genes*#lists), i.e. equal to the average occurrence probability. Thus, this model provides equal occurrence probabilities for all gene-list pairs and does not consider vertex degrees. We call this model the generic randomization (GR) model in the following.
Second, we could disconnect the edges on only the list side of the bipartite graph and reconnect them randomly. The occurrence probability of a gene vertex would be given by (gene vertex degree)/#lists. The sum of these probabilities over all lists is equal to the gene vertex degree and the sum of all gene vertex degrees is equal to the total number of edges. Thus, the sum of all matrix elements is equal to the number of edges, as required. Since this model considers gene vertex degrees, we call it the gene vertex degree (GVD) model.
Third, we disconnect the edges on the gene side of the bipartite graph and reconnect them randomly. The probability of a list vertex being connected to a gene would be given by (list vertex degree)/#genes. The sum of these probabilities over all genes is equal to the list vertex degree and the sum of all list vertex degrees is equal to the total number of edges. Again, the sum of all matrix elements is equal to the total number of edges. Since this model considers list vertex degrees, we call it the list vertex degree (LVD) model.
In model four and five, we reconnect edges considering both gene and list vertex degrees and allow multiple edges between list-gene pairs. The occurrence probabilities in model four are calculated according to the binomial distribution. We calculate the probability of a list-gene pair for being connected as the cumulative binomial probability of the list-gene pair being chosen at least once in the process of randomly reconnecting the edges. This can be achieved by setting the number of trials equal to the gene vertex degree, the probability of success equal to the list vertex degree divided by the total number of edges, and the number of successes equal to 0. The occurrence probability of a list-gene pair is then given by the complement of this probability. This model is called the binomial (BN) model. In model five, we calculate occurrence probabilities according to the hypergeometric distribution. The number of successes in the sample is equal to 0, the sample size is equal to the gene vertex degree, the number of successes in the population is set to the list vertex degree, and the population size is the total number of edges. Again, the occurrence probability of a list-gene pair is obtained as the complement of this probability. We call this model the hypergeometric (HG) model. Calculating occurrence probabilities in this manner does not guarantee that the matrix elements add up to the total number of edges. Therefore, the matrices are normalized such that this condition is satisfied by multiplying each matrix element with the factor #edges/(observed matrix sum), which is generally quite close to 1, however.
In model six, we again consider vertex degrees, but we require that each list is composed of distinct sets of genes. Thus, multiple edges are forbidden. This condition is satisfied by applying an edge-swapping procedure during graph randomization. Edge-swapping works by randomly choosing two list-gene pairs from the bipartite graph and prior to performing the edge-swap, a test is performed to ensure that the two genes are not already linked to the respective target lists. This procedure is performed a large number of times. To ensure complete randomization of the graph, the number of swaps performed should be significantly larger than the number of edges. After performing R randomizations of the graph and counting the number of times a gene has been linked to a particular list, division of this number by R gives the occurrence probability of a gene in a given list. As will be shown below, edge-swapping produces occurrence probabilities that closely approximate occurrence probabilities obtained by generating a complete permutation set of the bipartite graph, counting the number of times a gene is found part of a list, and dividing this number by the total number of permutations. In the permutation model, the sum of occurrence probabilities of a gene over all lists equals the gene vertex degree (see below) and thus the sum of all matrix elements is the number of edges. Since permutation sets of bipartite graphs are difficult to calculate, we use the edge-swapping procedure as a close approximation and call this model the edge-swapping (ES) model.
Fig. 1 shows the occurrence probabilities of the different null-models for the bipartite graph shown in Fig. 1A. The GR model yields identical occurrence probabilities for all list-gene pairs, which is equal to the average occurrence probability in all models. In the other models, the occurrence probabilities deviate to varying extent from the average occurrence probability as a function of vertex degrees. In the GVD model, the deviations are a function of gene vertex degree and in the LVD model the deviations are dependent on list vertex degrees. In the remaining models, the deviations are functions of both the gene and the list vertex degrees. In all cases, larger than average occurrence probabilities are obtained for larger vertex degrees at the expense of smaller than average occurrence probabilities for smaller vertex degrees. From these data, it is difficult to choose the most effective null-model. A hint can be gleaned from gene1, however. Gene1 is present in all lists. Therefore, the co-occurrence probability of gene1 with other genes, which is calculated by multiplying the occurrence probabilities of gene1 and geneX for every list under study, should depend only on the occurrence probability of this other gene. In other words, the occurrence probability of gene1 in all lists should be 1.0. Only two models satisfy this constraint: The GVD and the ES models. Since the GVD model does not consider list vertex degrees, it seems that the ES model is the preferred null-model.<br><br>Expected number of co-occurrences
As a general criterion for comparing the effectiveness of different null-models, we have to compare them for the number of expected co-occurrences. The most effective null-model will be the one that maximizes the expected number of co-occurrences. If the expected number of co-occurrences is larger, an observed number of co-occurrences in a real bipartite graph will be less significant and thus such a null-model permits co-occurrence analysis with higher stringency. The expected number of co-occurrences depends in an obvious fashion on the list vertex degree. If pair-wise co-occurrences are considered, the number of co-occurrences in a list of vertex degree N is given by the binomial coefficient N over 2. Larger lists will give rise to more co-occurrences and the number increases quickly with list vertex degree. The dependency of the expected number of co-occurrences on the gene vertex degree is less obvious and depends strongly on the null-model. A gene that is part of a list with vertex degree N will give rise to N-1 co-occurrences in that list. The null-model permits calculating the probability to find this gene in a given list. Thus, the expected number of co-occurrences of a gene is given by the sum of expected co-occurrences in all lists where for a single list the expected co-occurrences are given by (Nl−1)*pl. Nl is the list vertex degree and pl is the occurrence probability of the gene in that list as determined by the null-model.
We used the PubLiME data set [35] to calculate the expected number of co-occurrences with different null-models. The results are shown in Fig 2A. The expected number of co-occurrences was calculated for all genes in all lists using all null-models and the sum of expected co-occurrences per gene is shown as a scatter plot with the gene vertex degree on the x-axis and the expected number of co-occurrences on the y-axis. The results in Fig. 2A suggest the following ranking of null-models: GR<GVD<LVD<BN = HG<ES. The BN and the HG models perform in an essentially identical way. However, the ES model is the model that yields the largest estimates of expected co-occurrences. The results are also in line with the intuitive expectation that genes with higher vertex degree give rise to more co-occurrences. However, it can be seen that this is not true for all null-models. In particular, it is not true for the GR and the LVD models, which do not consider gene vertex degrees.
As outlined above, it is expected that the null-model that yields the highest estimates of expected co-occurrences should permit co-occurrence analysis with the highest stringency. In Fig. 2B, this hypothesis is tested directly again using the PubLiME data set [35]. For all null-models, co-occurrence analysis was carried out using module size 3 and support 5 (co-occurrence modules must be present in at least five publications). The choice of these parameters has been discussed in [35]. The number of co-occurrence modules was then determined that have a Z-score higher or equal than the cut-off shown in Fig. 2B. The Z-score is calculated from the mean and variance of the Poisson-binomial distribution as shown in the Materials and Methods section and published in [35]. More details on the probability distribution will be provided below. The GR and GVD models perform very poorly and identify large numbers of modules with high Z-scores. The LVD model performs a little better and approximates the BN and HG models at higher Z-score cut-offs. The BN and HG models give essentially identical results. However, the ES model is the model that yields the fewest number of significant co-occurrence modules and is thus the most stringent. The increased stringency of the ES model over the BN and HG models is also reflected in a higher signal-to-noise ratio calculated as the number of significant co-occurrence modules in the real bipartite graph divided by the number of modules found in a randomized bipartite graph (Fig. 2C).
The reason for the superior stringency of the ES model over all other models can be explained by examining the average occurrence probability per gene and list vertex degree. Fig. 2D and E show the average occurrence probability of genes with the same gene vertex degree as a function of the gene vertex degree. It can be seen that the ES model yields higher occurrence probability estimates for genes with higher vertex degrees as compared to the BN and HG models. In GR and LVD models, gene vertex degrees are ignored and occurrence probabilities for genes with large vertex degree are very small, which is compensated by larger occurrence probabilities for genes with small vertex degree. The GVD model is identical to the ES model in this setting. Fig. 2E shows the average occurrence probability of all lists with the same vertex degree as a function of list vertex degree. It can be seen that the ES model provides higher occurrence probability estimates for large lists as compared to the BN and HG models. In this setting, the LVD model performs like the ES model while the GR and GVD models yield small occurrence probabilities for large lists. Since it has been shown above that long lists and genes with high vertex degree are responsible for a large part of the total number of co-occurrences for the most stringent null-models, the null-model that provides larger occurrence probability estimates for genes and lists with high vertex degree at the expense of lower estimates for smaller degrees will be the most stringent because large occurrence probabilities make co-occurrence more likely and thus less significant. By these criteria, the ES model is the most stringent of all models tested.<br><br>The ES model as an approximation of the permutation null-model
The data shown above have revealed that the ES model is the best of the models tested. One may wonder, however, whether yet more effective null-models can be found. An obvious choice would be the permutation model. In the permutation model, a complete permutation set of the bipartite graph is created such that each list is composed of distinct sets of genes. The number of graphs where a gene is present in a given list divided by the total number of permutations then provides the occurrence probability estimate. The permutation model is the ideal null-model because it is exhaustive. The problem is that a complete permutation set of bipartite graphs of some complexity is very time consuming to calculate. For example, the simple bipartite graph from Fig. 1A is part of a permutation set of 455 graphs. The number of permutations is increasing quickly as the numbers of genes and lists grow. However, since edge-swapping ensures that gene lists are composed of distinct sets of genes, each edge-swap produces a graph that is part of the permutation set of the bipartite graph. Edge-swapping can thus be viewed as a random sampling procedure from the permutation set of the bipartite graph. Therefore, occurrence probability estimates derived by edge-swapping should approximate those obtained from the permutation model.
We generated a complete permutation set of the graph shown in Fig. 1A to verify this hypothesis. The results are shown in Fig. 3. Fig. 3A shows how the number of possible permutations can be calculated. Gene1 is present in all lists and does not have an impact on the total number of permutations. Gene2, having vertex degree two, is present in two out of three lists in one out of three possible ways. The remaining genes have vertex degree 1 and can be freely chosen to fill the empty slots. We can now count exactly how many times a gene is linked to a list and divide these counts by 455, the size of the permutation set, to obtain exact occurrence probabilities. These numbers are shown in graphical form in Fig. 3B and in numerical form in Fig. 3C. Fig. 3B also shows the occurrence probability estimates obtained by edge-swapping side-by-side to the exact occurrence probabilities. The graph in Fig. 1A was subjected to edge-swapping 1000 times and the number of times a gene was found present in a list was divided by 1000 to obtain the occurrence probability. At each run, 100 random edge swaps were performed to ensure complete randomization of the graph. This procedure was repeated 10 times and the mean and standard deviation of occurrence probability estimates for each gene in each list are shown. In all cases, the mean differs from the real probability by less than two standard deviations, in most cases by less than one standard deviation. Thus, edge-swapping provides reliable estimates of exact occurrence probabilities as determined from a complete permutation set.
As an interesting observation, we provide evidence that occurrence probabilities are non-linear functions of vertex degrees in the edge-swapping model. This is illustrated in Fig. 3C. Individual and average occurrence probabilities are shown as a function of gene and list vertex degrees. Non-linearity of individual occurrence probabilities can be verified from the counts table underneath the plots. However, the average occurrence probability is found to depend on vertex degrees in a linear fashion instead. This is a consequence of the fact that occurrence probabilities of a gene over all lists add up to the gene vertex degree and that the occurrence probabilities of all genes for a given list add up to the list vertex degree. At the same time, since the most stringent permutation based null-model predicts non-linear dependencies of individual occurrence probabilities on vertex degrees, assuming such linearity in statistical models of co-occurrence will be linked to loss of stringency.
We conclude that the ES null-model is the null-model that permits co-occurrence analysis with the highest stringency among the models tested and that it closely approximates occurrence probabilities derived from an ideal permutation model. The increased stringency of the ES model over other models is a consequence of higher occurrence probabilities for genes and list with high vertex degrees, which are giving rise to a large part of all co-occurrences in the bipartite graph. Since large occurrence probabilities make co-occurrence more likely, the analysis becomes more stringent.<br><br>Co-occurrence probabilities
Co-occurrence analysis can be thought of as a Bernoulli experiment with a binomial outcome (a given combination of entries is either present or not present in a given list). Thus, the Binomial distribution (BD) is a natural choice for judging the significance of the number of co-occurrences. However, the BD is defined for a probability of success which is equal in all trials. The list-specific nature of occurrence probabilities is not compatible with this condition (analysis of each list represents one trial), which means that co-occurrence analysis in the presence of list-length-heterogeneity is better described as a series of Poisson trials, where the probability of success varies from trial to trial. Therefore, the significance of co-occurrences must be evaluated using a binomial distribution with trial-specific probabilities, i.e. the Poisson-binomial distribution (PBD). The probability of success in a single Poisson trial can be calculated by multiplying the list-specific occurrence probabilities for the combination of genes under study. The number of occurrence probabilities that need to be multiplied is equal to the module size, i.e. the number of genes whose combination is studied. An observed number of co-occurrences for a combination of genes can then be evaluated using the PBD, which is given by the formula [36]:(18)
The structure of this formula is very similar to the structure of the formula used to calculate the binomial distribution, except that multiplication with a binomial coefficient is replaced by summation over individual terms, which makes calculation of P-values using (18) inefficient (note that equation numbering starts in the Materials and Methods section). Here, Ak denotes the kth set of indices of the i lists where genes are co-occurring (“success”). There are  possible sets and summation is carried out accordingly.  denotes the set of indices of N−i lists where genes are not co-occurring (“failure”). [36] have reported two fast procedures for calculating exact PBD P-values. However, both procedures work with probability ratios and suffer from numerical overflow/underflow problems for large numbers of trials. NetCutter uses two workarounds to circumvent this problem. One is based on using Poisson-binomial Z-scores, which can be calculated very easily instead (see below). The other relies on a fast approximation procedure for calculating Poisson-binomial P-values, which we call bi-binomial approximation (BBA) or bi-binomial distribution (BBD).<br><br>Z-scores and P-values of BBD
Given the mean μ (1) and variance σ2 (2) of PBD (see Materials and Methods), the Z-score associated with a given number of co-occurrences x is obtained as:(19)
Considering the structure of formulae (1) and (2) (Materials and Methods section), PBD Z-scores can be calculated very easily and provide a simple estimate of the significance of co-occurrence modules. However, in contrast to normally distributed Z-scores, binomial and Poisson-binomial Z-scores do not correspond to the same P-value for different sets of probabilities of success. To see this, calculate for example the probability of success in a series of 100 Bernoulli trials with success probability 0.1 and 0.9 for the expectation of 10 and 90 successes, respectively. The Z-score will be 0 in both cases but the corresponding cumulative P-values are 0.5832 and 0.5487. Therefore, exact levels of significance cannot be derived from Z-scores alone. Thus, a fast and reliable procedure for calculating Poisson-binomial P-values is needed. The BBD approximation was developed to solve this problem.
The BBD approximation of PBD P-values follows from the relationship between the variance of PBD and the population variance of trial-specific probabilities of success. This relationship is shown in Materials and Methods to be described by (4):(4)
This equation shows that there is an inverse linear relationship between the population variance S2 of the N trial probabilities and the variance of PBD σ2, which means that PBD becomes increasingly narrow as the variance of trial probabilities grows. It also shows that, for constant mean μ and number of trials N, the shape of PBD depends only on the variance of trial probabilities. Therefore, relationship (4) suggests an easy way to approximate PBD P-values. The P-value can be obtained by constructing a set of trial probabilities with equal variance as the original set of trial probabilities, which, however, are not all different. In other words, the series of Poisson trials can be replaced by two sets of Bernoulli trials with trial probabilities p1 and p2 constructed such that the variance is equal to the original set of trial probabilities. This strategy is illustrated in Fig. 4 and explains why this approximation is called bi-binomial. The details on how to obtain the values of the two sets of Bernoulli trial probabilities and the number of trials with p1 and p2 as probabilities of success are provided in the Materials and Methods section. The precision of the BBD approximation is discussed in supplementary material Simulation S1.
In order to evaluate whether BBD P-values as a significance measure of co-occurrence offer an advantage over other measures such as the Jaccard coefficient or the uncertainty coefficient, pair-wise co-occurrence of two genes in 200 lists with and without list-length-heterogeneity was studied (Fig. 5). Each gene is assumed to occur in 100 lists. Therefore, the occurrence probabilities of both genes over all 200 lists must add up to 100, regardless of list-length-heterogeneity. For simplicity, occurrence probabilities of both genes are assumed to be equal in any particular list. The co-occurrence probability in a list is then given by the square of the occurrence probability in that list. For all possible co-occurrences from 0 to 100, the Jaccard and uncertainty coefficients were calculated as detailed in the Materials and Methods section. In addition, cumulative BBD P-values were calculated using the co-occurrence probabilities as trial probabilities. To illustrate the advantage of BBD over BD as co-occurrence probability distribution, cumulative BD P-values of a BD with the same mean as BBD but constant trial probabilities is shown. These trial probabilities can be obtained by dividing the mean of BBD by the number of lists.
Three different cases of list-length-heterogeneity are considered in Fig. 5: No heterogeneity (standard deviation 0), heterogeneity with standard deviation 0.283 and heterogeneity with standard deviation 0.401 in the occurrence probabilities. The Jaccard and uncertainty coefficients are by definition insensitive to list-length-heterogeneity because differences in co-occurrence probabilities in a given list cannot be considered in their calculation. This is because both coefficients are defined by the counts of the four list classes: both genes absent, both genes present, first gene absent second gene present, and first gene present second gene absent, i.e. by the corresponding contingency table, which does not change with different list-length-heterogeneity. In the absence of list-length-heterogeneity, the cumulative P-values of BD and BBD (which are perfectly overlapping as expected) assume 0.5 at 50 co-occurrences, which corresponds to the expected number of co-occurrences calculated as (50 = 100 occurrences per gene/200 lists) ˆ2*200 lists. The uncertainty coefficient is found to be 0 and the Jaccard coefficient is 0.33333 at that point. When there is modest list-length-heterogeneity (standard deviation 0.283), the mean of BBD is shifting to the right. This is because the sum of squares of varying occurrence probabilities (i.e. the sum of co-occurrence probabilities used as trial probabilities, which is equal to the mean of BBD) is always larger than the sum of squares of constant occurrence probabilities with the same average occurrence probability (0.5). The corresponding BD in the presence of list-length-heterogeneity is obtained by dividing the expected number of co-occurrences by the total number of lists, which means assuming equal co-occurrences in all lists. This visualization is shown to illustrate how BBD (which is narrower than the corresponding BD) gives rise to a steeper cumulative distribution of P-values and as a consequence to more significant P-values for numbers of co-occurrence that are far from the expectation. As the level of list-length-heterogeneity grows (standard deviation of occurrence probabilities 0.401), the mean of BBD is shifted even further to the right and BBD P-values are distributed in a still steeper fashion as compared to corresponding BD P-values and the interval of non-significant co-occurrences is shrinking further. With modest list-length-heterogeneity, the expected number of co-occurrences is 66, which is associated with a Jaccard coefficient of 0.49 and an uncertainty coefficient of 0.075. In the case of large list-length-heterogeneity, the expected number of co-occurrences is 82 with J = 0.69 and UC = 0.32.
Taken together, these data show that the expected number of co-occurrences varies strongly with the level of list-length-heterogeneity and that the expected number of co-occurrences is associated with different values of UC and J. To complicate matters further, 66 co-occurrences (J = 0.49, UC = 0.075) represent significant positive association (PBBD = 0.996) with equal list lengths, no significant association with modest differences in list length (PBBD = 0.536) and strongly negative association (meaning one gene excludes the other) with strong list-length-heterogeneity (PBBD = 0.00016). Thus, the same J and UC association measure is obtained for positive, negative, and absence of association. Therefore, the meaning of these measures cannot be interpreted properly in the absence of knowledge about the occurrence probabilities of the co-occurring entities. Furthermore, the data in Fig. 5 also show that neither J nor UC can distinguish between positive and negative association while this is easy with cumulative BBD P-values: Large P-values mean positive association and low P-values mean negative association. In summary, we conclude that BBD provides a novel association measure that offers a number of advantages over the existing contingency table based association measures Jaccard coefficient and uncertainty coefficient. The results in Fig. 5 also show that significance of association depends critically on the specific distribution of co-occurring entities over lists of varying length (because this distribution determines the occurrence probabilities) and that contingency table based methods (which cannot capture this distribution) should be avoided in the presence of significant list-length-heterogeneity.<br><br>Generation of co-occurrence networks and the identification of communities
The procedures outlined above allow the identification of significant co-occurrence modules in any type of bipartite graph. Three user defined parameters have an impact on the stringency of co-occurrence analysis: The module size, the support, and the Z-score/P-value cutoff. The module size determines how many entries will be tested for co-occurrence, the support sets a lower boundary on the required number of co-occurrences, and the Z-score/P-value cutoff sets the significance threshold. In general, higher module size leads to more stringent co-occurrence analysis at the cost of computational complexity. The support parameter allows limiting this complexity by filtering out co-occurrence modules which co-occur less frequently than required by the support. The significance cutoff permits adjusting the signal-to-noise ratio, which is calculated as the number of co-occurrence modules observed in the real versus a randomized bipartite graph. The impact of these parameters on the stringency of co-occurrence analysis has been reported previously for the PubLiME data set [35] and is illustrated in a simulation study provided as supplementary material Simulation S1. From the set of significant co-occurrence modules, a co-occurrence network is generated by considering each entry a vertex and drawing an edge between any two vertices, which have been part of the same significant co-occurrence module [35].
An important question in the analysis of co-occurrence networks regards the presence of network communities. Communities can be understood as groups of vertices with the property that the number of edges running within groups is larger than expected by chance and that the number of edges running between groups is lower than expected by chance [37]. This problem of partitioning a graph is often referred to as the graph-cut problem (hence the name NetCutter). NetCutter is built on the Java Universal Network and Graph framework (JUNG) software package (http://jung.sourceforge.net), which provides algorithms for solving this problem. In particular, NetCutter implements the Bicomponent clustering algorithm [38], the Edge-Betweenness clustering algorithm [39], and the Exact Flow Community algorithm [40]. Furthermore, there is a clustering tool that is not part of the JUNG package, namely an algorithm identifying communities using eigenvectors of the modularity matrix [37]. The code for this algorithm was kindly provided by Mark Newman in C++ and ported to Java. In addition to these tools, NetCutter provides a number of convenience functions for the analysis of co-occurrence networks, such as testing the significance of lists reporting a set of entries making up a network community, ranking of vertices, random graph generators for topological analysis of co-occurrence networks, and others. Details on all functions are provided in the NetCutter software documentation.
One of the possible applications of NetCutter is illustrated below. This application is tightly linked to the bipartite graph representation of the data. Namely, NetCutter can be used to perform co-occurrence analysis of genes or list derived from the same bipartite graph. The network communities identified in each both reflect the same underlying structure of the bipartite graph. In the case of gene expression signatures stored in PubLiME, clusters of genes correspond to clusters of publications, which can reveal possible functions of gene clusters.<br><br>Cluster correspondence and association studies
The co-occurrence analysis of the PubLiME data set published previously [35] identified 5 major network communities of genes with consistent functional annotations that are deregulated in cancer related gene expression signatures. This analysis was performed by considering all genes mentioned in a particular publication as a single signature, even though they might have been part of different tables and cluster analyses. Here we present an advanced analysis of the PubLiME data set where each table and/or cluster identified in a given publication is considered as a separate signature. This brings the total number of signatures to be analyzed to 1015 comprising a total of 7358 differentially regulated genes derived from 233 publications reporting cancer related signatures derived from <span class="yellow">human</span> samples. We use this analysis to illustrate three major points: First, the set of communities reported previously is reproduced by this more fine-grained analysis. Second, the set of gene communities corresponds to a set of publication communities. Third, associations between publications and gene communities can be calculated with higher stringency using the edge-swapping null-model in conjunction with bi-binomial P-values as compared to binomial or hypergeometric statistics.
The bipartite graph to be analyzed is composed of PubMedID_listID-gene pairs (see supplementary material Table S1). Co-occurrence analysis was carried out in two ways: First, gene co-occurrence was analyzed and communities of co-occurring genes were defined by edge-betweenness clustering as described in Materials and Methods. Second, co-occurrence of PubMedID_listIDs was analyzed. To this end, the order of PubMedID_listID-gene pairs was reversed to form GENE-PUBMEDID_LISTID pairs. Thus, the lists in the resulting bipartite graph are formed by genes and the entries are the PubMedID_listIDs where the genes are reported as differentially regulated. Occurrence probabilities for the reversed bipartite graph can be obtained by transposing the occurrence probability matrix of the original bipartite graph. Since the gene communities identified in gene co-occurrence analysis reflect the structure inherent in the bipartite graph (which is not affected by reversing the list-entry order), co-occurrence analysis of the reversed bipartite graph will result in PubMedID_listID communities that reflect the same underlying structure in the bipartite graph. In other words, PubMedID_listID communities correspond to gene communities. In less abstract terms, the PubMedID_listID communities should correspond to sets of publications that report similar sets of genes as differentially regulated. The identification of communities of publications can help the researcher to easily identify publications studying genes in a gene community that is of interest to the researcher.
The results of both types of co-occurrence analysis are displayed in Fig. 6. Fig. 6A shows the gene clusters identified. The clusters are named after significant enrichments of gene categories as determined by functional category enrichment using DAVID [41]. The P-values shown are Benjamini corrected for multiple testing as reported by DAVID. The clusters are very similar to the clusters published previously [35]. There is one new cluster that is strongly enriched for ribosomal proteins (“protein biosynthesis” cluster), which has not reached significance in our previous analysis. The “surface antigen” cluster contains many genes that had been reported as part of the “signal transduction” cluster. Altogether, however, these results strongly support the notion that the gene clusters in the PubLiME data set can be reproduced by the more fine-grained analysis that considers sublists in each publication as separate signatures.
The corresponding clusters of PubMedID_listIDs are shown in Fig. 6B. There are five clusters, which have been named after their corresponding gene cluster. Only one cluster (the “extracellular matrix-immune response cluster”) cannot be separated by edge-betweenness clustering at the point of maximal graph modularity. To see that this naming is indeed justified, we needed to investigate how strongly a given PubMedID_listID is associated with a given gene cluster, i.e. how significant is the overlap of the genes reported in a gene cluster and the genes reported in a PubMedID_listID. Binomial or hypergemetric statistics are generally used to calculate this significance. However, the bipartite graph model in conjunction with the edge-swapping null-model offers a more fine-grained approach based on bi-binomial statistics.
The edge-swapping null-model determines occurrence probabilities in such a way that the number of genes in a given PubMedID_listID is associated with insignificant P-values in the context of the complete bipartite graph. However, when a subset of genes is analyzed, e.g. all the genes that are reported in a particular list, the P-value associated with the number of genes contained in this list will likely be highly significant according to how unlikely it is to obtain all the genes contained in a given list in a random draw from all genes present in the bipartite graph. Thus, PubMedID_listID association with a set of genes in the bipartite graph model can be calculated in the following way: The set of genes that is used to analyze association is used to extract a subgraph from the original bipartite graph where occurrence probabilities for each gene-PubMedID_listID pair are identical to those in the original bipartite graph (i.e. they are not recalculated by edge-swapping). The vertex degree of the PubMedID_listID vertices in the subgraph indicates the number of genes contained in each PubMedID_listID overlapping with the set of genes used to extract the subgraph. From the occurrence probabilities of the genes in a given PubMedID_listID, the bi-binomial P-value can then be calculated for every list vertex degree observed in the subgraph. In Fig. 7, the significance of association of the PubMedID_listIDs (see Fig. 6B) with the cell cycle cluster of genes (Fig. 6A) is calculated. For comparison, binomial and hypergeometric P-values are also shown. It can be seen that the bi-binomial P-value is larger than the binomial and hypergeometric P-values, which means that the strength of association is evaluated in a more stringent manner using BBD statistics (see Discussion for an explanation of this observation).
The analysis of significant associations between PubMedID_listIDs and gene clusters now permit answering the question whether there is correspondence between gene clusters and PubMedID_listID clusters. The naming of PubMedID_listID clusters shown in Fig. 6B is based on the number of PubMedID_listID that are significantly associated with gene clusters shown in Fig. 6A. First, for each gene cluster, all the PubMedID_listIDs that are associated with that cluster with more than 95% confidence (i.e. cumulative bi-binomial P-values> = 0.95) were identified. Second, the number of significant PubMedID_listIDs in each PubMedID_listID cluster was counted for every gene cluster. The significance of this number was then calculated using binomial statistics. The results of this analysis are shown in Table 1. Negative decadic logarithms of the binomial P-value are displayed. It is apparent that each PubMedID_listID cluster is strongly associated with at least one gene cluster, except for the “extracellular matrix-immune response” cluster, which is associated with two gene clusters. The strength of these associations suggests that the PubMedID_listID clusters are indeed corresponding to the gene clusters and that both the gene and the PubMedID_listID clusters reflect the structure of the bipartite graph representing the PubLiME data set.
Details about all the lists analyzed are attached as supplementary material Table S2. Looking at these lists, some general conclusions about the gene clusters can be drawn. Cell cycle cluster genes have been found deregulated in a wide variety of tumor types such as colon cancer, breast cancer, in biliary tract cancer, pancreatic cancer, gastric cancer, prostate cancer, T-cell leukemia, glioma, acute lymphoblastic and myeloblastic leukemias, soft tissue sarcoma, neuroblastoma, as well as in a number of cellular model systems in response to different stimuli. Thus, the cell cycle cluster seems to consist of genes with a general role in oncogenesis. The surface antigen cluster instead seems to be derived preferentially from studies on leukemia. The interferon cluster genes are found deregulated in virus induced pathologies such as papilloma virus induced cervical cancer, and viral hepatitis. Immune response cluster genes were reported as differentially regulated in inflammatory conditions such as ulcerative colitis, Crohn's disease, and <span class="yellow">Helicobacter pylori</span> infections. Genes of the extracellular matrix cluster seem to be associated with cancer progression studies and metastatic potential. For the protein biosynthesis cluster, there are 15 signatures that are significantly enriched for those genes. The cancers studied comprise medulloblastoma, glioblastoma, pancreatic cancer, soft tissue sarcoma, lung carcinoma, breast carcinoma, prostate carcinoma, multiple myeloma, and lymphocytic leukemia. The genes are also found deregulated in response to DNA damage. Although the number of signatures is limited, the variation in conditions where the genes are deregulated is compatible with the hypothesis that protein biosynthesis genes, as cell cycle genes, are deregulated in many cancer types, which might reflect the general property of cancer cells to divide and grow in an uncontrolled fashion.<br><br>
Discussion
Here we have investigated basic aspects of co-occurrence analysis and present a software tool, NetCutter, which can be used to identify and analyze generic co-occurrence networks. In NetCutter, a co-occurrence data set is represented as a bipartite graph with one part representing lists and the other part list entries whose co-occurrence patterns are studied. The bipartite graph representation of co-occurrence data sets allows the efficacy of different null-models to be tested systematically. We have shown that an edge-swapping procedure used to randomize the bipartite graph generates a null-model that allows co-occurrence analysis with the highest stringency. The other null-models tested here tend to underestimate occurrence probabilities of entries per list for lists and genes with high vertex degrees, i.e. for lists and genes where most co-occurrences are observed. As a result, co-occurrences are judged more significant than they really are.
Co-occurrence data sets with exactly equal lists lengths are likely to be the exception from the rule. It can be assumed that some list-length-heterogeneity will be present in most circumstances. An important consequence of list-length-heterogeneity regards the co-occurrence probability distribution used to evaluate the significance of the observed number of co-occurrences. Co-occurrence analysis in the presence of list-length-heterogeneity is best performed using the Poisson-binomial distribution (a binomial distribution with trial specific probabilities). However, calculating Poisson-binomial P-values for large numbers of lists is difficult using existing procedures [36]. We have presented an approximation to the Poisson-binomial distribution, called bi-binomial distribution, which is based on replacing the set of Poisson trials by two sets of Bernoulli trials. The resulting distribution reproduces the Poisson-binomial distribution nearly exactly and its P-values can be calculated with ease even for thousands of lists (see also supplementary material Simulation S1 for details on the precision of BBD). Importantly, BBD provides a novel measure of association, which is shown to be superior to existing measures such as the Jaccard coefficient and the uncertainty coefficient, whose values cannot be interpreted properly in the absence of knowledge about the occurrence probabilities of co-occurring entities.
It is worth noting that Poisson-binomial Z-scores are distinguished from Gaussian Z-scores by the fact that they do not correspond to the same P-value for different PBDs, BBDs, and even BDs. This is because the Z-score is an explicit part of the function defining the normal probability density while it is not part of the definitions of BD, PBD, and BBD densities. As a consequence, the simple Poisson-binomial Z-score based approach to evaluating significance of co-occurrence must be complemented with the BBD to approximate Poisson-binomial P-values in order to enable multiple testing corrections and to allow calculation of confidence levels in association studies precisely. However, NetCutter is equipped with a bipartite graph randomization tool that permits measuring the number of false positives due to multiple testing directly by comparing the number of significant co-occurrence modules in the real bipartite graph to the corresponding number in a randomized version thereof. Randomization is performed by edge-swapping in order to preserve vertex degrees. The resulting signal-to-noise ratios that are plotted for each Z-score/P-value cutoff provide a highly reliable and visually intuitive defense mechanism against false positives (see also supplementary material Simulation S1).
In the vast majority of co-occurrence studies, pair-wise co-occurrences have been analyzed using different statistical models. We have observed that the stringency of pair-wise co-occurrence analysis is far below the stringency of co-occurrence analysis using higher order combinations of co-occurring entities [35]. In NetCutter, co-occurrence analysis is preceded by occurrence analysis, i.e. the occurrence probability of each entry in each list is determined. Starting from occurrence probabilities, co-occurrence probabilities for any size of co-occurrence modules under study can be obtained by multiplying the respective list-specific occurrence probabilities. Given the list-specific co-occurrence probabilities, bi-binomial P-values are then calculated in exactly the same way for any module size. As a consequence, NetCutter can perform co-occurrence analysis for higher order combinations of co-occurring entries (i.e. larger module sizes) using the same statistical model. One of the benefits of using higher module sizes is robustness of the analyses in the presence of noise. This is because each edge in the resulting co-occurrence network is evaluated many times since every pair of co-occurring entries can be part of many higher order co-occurrence modules [35]. Another advantage is that implicit relationships between entries, which have never occurred together [18], can be derived as a natural by-product of using module sizes larger than 2. As shown in a simulation study (supplementary material Simulation S1), the result is a dramatic reduction of misclassifications at higher module sizes.
NetCutter can be used to calculate the strength of association between a subset of entries and lists reporting those entries. In this case, the analysis is performed on a subgraph of the original bipartite graph. The subgraph can correspond to communities of entries in the co-occurrence network, or any set of entries of interest. NetCutter will then calculate the significance of observing a given number of occurrences of an entry in the user defined subset of lists using bi-binomal statistics. This analysis mode corresponds to association studies with the advantage that the structure of the underlying bipartite graph (i.e. list length heterogeneity) is considered and handled appropriately using the bi-binomial distribution. As a consequence, association studies can be performed with higher stringency.
This result can be understood by examining the occurrence probability matrix that is implicitly assumed in performing binomial or hypergeometric tests for the significance of overlaps. In both tests, a gene is assumed to have an equal opportunity to be present in a list. Therefore, the probability of success for a gene to be part of a list is given by the list vertex degree divided by the total number of genes. In other words, both tests are implicitly based on the list vertex degree model, which has been shown previously to underestimate the occurrence probability and the expected number of co-occurrences for genes with high vertex degree (see Fig. 2A). Since the BBD P-values are calculated from the ES-model, which assigns higher occurrence probabilities to genes with higher vertex degree and more expected co-occurrences, the observed number of overlaps between a set of genes of interest and the content of a given list (which can be viewed as co-occurrence of the overlapping genes in that list) will be judged less significant when the overlapping genes are of high vertex degree (and vice versa when the overlapping genes are of low vertex degree) as compared to binomial or hypergeometric tests. Since the BBD P-values are derived from the most stringent ES null-model, BBD P-values provide a more reliable estimate for the significance of overlap.
Co-occurrence analysis of data represented as bipartite graphs permits visualizing the structure of the bipartite graph either as communities of list entries (genes) or as communities of lists (PubMedID_ListID) in co-occurrence networks. We have analyzed the PubLiME data set for the presence of corresponding gene and list clusters. In addition to previously published clusters of genes, we describe a novel gene cluster that is composed of protein biosynthesis associated genes [35]. We found that the corresponding clusters of PubMedID_ListID (gene expression signatures) are in general strongly enriched for genes reported in the corresponding gene cluster and that interrogation of corresponding clusters can be used to deduct hypotheses about the putative function of gene clusters.
In addition to co-occurrence analysis, NetCutter offers a number of tools for the analysis of co-occurrence networks, or any undirected graph. In particular, community identification is supported by four different community identification algorithms. NetCutter also offers a range of convenience functions that are of help in network analysis. Worthy of mentioning are the random graph generators that can provide control graphs for topological studies. The complete set of options is described in the software documentation.
In summary, we present a general framework for co-occurrence analysis with many potential applications. We illustrate a number of advantages of using the bipartite graph representation of data and the associated statistics. In particular, the identification of corresponding clusters permits the identification of functional subunits such as gene clusters on the one hand, and the generation of hypotheses about the function of those units by analyzing the corresponding list clusters on the other hand. Future developments will be directed towards the analysis of data sets that are considerably larger than the data sets analyzed so far. For example, co-occurrence analysis might be of interest for the analysis of single nucleotide polymorphism (SNP) data sets and association studies of genome variability with disease. Each <span class="yellow">patient</span> is characterized by a specific range of SNPs. Co-occurrence patterns of <span class="yellow">patients</span> according to their SNPs could be compared to clinical parameters with the aim of identifying genomic regions associated with disease. The increased stringency of association studies offered by NetCutter may be of use in the analysis of polygenic diseases where conventional methods fail. For being useful in this setting, NetCutter must be capable of analyzing bipartite graphs with millions instead of thousands of vertices.<br><br>Materials and Methods
Implementation of NetCutter
NetCutter is written in Java using NetBeans6 software (http://www.netbeans.info/) and tested on the Java Runtime environment 1.6.0.0. on a Windows XP Professional computer. The Java Runtime environment, which can be downloaded from http://java.sun.com/, must be installed on a computer that is intended to run NetCutter. NetCutter is provided as a single jar file and should run by double clicking the jar file, provided that the Java runtime environment is properly installed. NetCutter makes use of the following software packages and classes: JUNG version 1.3 (http://jung.sourceforge.net/download.html), Apache Jakarta Commons Collections 3.1 (http://jakarta.apache.org/commons/collections/), Cern Colt Scientific Library 1.2.0 (http://dsd.lbl.gov/hoschek/colt/), Xerces (http://xerces.apache.org/xerces2-j/index.html), Jama (http://math.nist.gov/javanumerics/jama/), Netlib Java LAPACK (http://www.netlib.org/lapack/), JFreeChart (http://www.jfree.org/jfreechart/), partition.java (http://astro.u-strasbg.fr/fmurtagh/mda-sw/java/partition.java).<br><br>Bi-binomial approximation of Poisson-binomial distribution
The Poisson-binomial distribution (binomial distribution with trial specific probabilities) has recently been proposed as a statistic that properly handles largely differing sizes of gene expression signatures in meta-analysis of gene expression data [35]. Z-scores have been used to estimate the significance of co-occurrence because P-value calculation is cumbersome and error prone. Two methods reported by [36] suffer from numerical overflow/underflow problems when large numbers of Poisson trials with probabilities deviating significantly from 0.5 are being analyzed. Therefore, we propose a fast approximation of P-values based on a bi-binomial distribution. The bi-binomial distribution is a special case of the Poisson-binomial distribution where the probability of success can assume only two values. In order to achieve a good approximation of the underlying Poisson-binomial distribution, the values of these two probabilities and the number of trials where they are assumed must be determined carefully. As is shown in the following, the values of the two trial probabilities and their number of occurrences follow from the formula used to calculate the variance of the Poisson-binomial distribution and from the formula yielding the population variance of trial probabilities of the Poisson-binomial distribution to be approximated.
The mean μ and the variance σ2 of the Poisson-binomial distribution are given by equation (1) and (2), respectively.(1)(2)pi is the trial-specific probability of success and N is the total number of trials. For the sake of completeness, a formal proof of equation (1) is reported as supplementary material Proof S1 and the proof of equation (2) can be obtained in an analogous fashion.
The population variance S2 of trial probabilities pi is given by equation (3).(3)
Rearranging equation (3) considering (1) and (2) leads to (4) and (5), where pa denotes the average trial probability of success and qa its complement.(4)(5)Now let's define two trial probabilities p1 and p2, which are used N1 and N2 times during the Poisson trials, respectively. Thus, N1 and N2 add up to N.(6)
Considering (1), the average trial probability pa can then be obtained from (7).(7)
Using (7), p1 can thus be calculated as (8).(8)
Similarly, considering (2), the variance σ2 is given by (9).(9)
Substituting p1 in (9) using (8) followed by substituting σ2 in (5) by (9) leads to a quadratic equation for p2 as a function of pa, N, and S2, as shown in equation (10).(10)
The solution to (10) is given by (11).(11)
Setting p2 to(12)p1 can be obtained from (8) and shown to be given by formula (13):(13)Choosing p2 as(12a)leads to p1(13a)
Comparing (13a) to (12) and (12a) to (13), it can be seen that the formulae are identical except for the fact that N1 and N2 are reversed. Since the assignment of which set of trials is called N1 and which set of trials is called N2 is completely arbitrary, we can limit the remaining analysis on (12) and (13) without loss of generality.
Note that (12) and (13) do not guarantee that p1 and p2 are always confined between 0 and 1 for any combination of N1 and N2. While probabilities smaller than 0 or bigger than 1 would still result in a distribution with the same overall variance as the original distribution, P-value calculation will be imprecise because the tails of the distribution will deviate significantly from the original distribution. Thus, we need to define the values N1 and N2 in such a way that p2< = 1 and p1> = 0. This can be achieved by evaluating (12) and (13).
Evaluating (12) for the condition that p2< = 1, solving the resulting inequality for N2, and considering (5), which relates S2 and σ2, we obtain (14).(14)
Similarly, evaluating (13) for the condition p1> = 0, solving the resulting inequality for N2, considering (5), which relates S2 and σ2, and defining μf the expected number of failures as N * (1−pa) (15),(15)we obtain (16)(16)
The meaning of these boundaries is best illustrated by considering a Poisson-binomial distribution whose variance is 0, i.e. that assumes 1 at X = μ and 0 otherwise. In this case (14) requires N2< = μ while (16) requires N2> = μ. These conditions can only be fulfilled contemporaneously when N2 is set to μ. Intuitively, this means that there are μ trials with probability of success 1 and N−μ trials with probability of 0, resulting in a Poisson-binomial distribution with variance σ2 = 0 and mean μ. When σ2 is larger than 0, the choice of N1 and N2 is more flexible. However, since the choice of N2 = μ is valid for all possible values of σ2, this is how NetCutter determines N1 and N2. When μ is not an integer, N2 is set to the integer closest to μ.
Having determined p2 (12) and p1 (13) as well a N1 and N2 (14, 16, 6), we can now calculate the bi-binomial approximation of the Poisson-binomial distribution in a fashion that is very similar to calculate the binomial P-value. With q1 = 1−p1 and q2 = 1−p2 we obtain:(17)
The summation is necessary because i successes can be obtained from any combination of j p1 and i−j p2 trials, where j can assume any value from 0 to i.<br><br>Calculating Jaccard and uncertainty coefficients
For the purpose of comparing the efficacy of the bi-binomial distribution as a significance measure of co-occurrence, Jaccard and uncertainty coefficients (which are also called measures of association) were calculated using the formulae:
The Jaccard coefficient J is calculated as the number of times A and B occur together divided by the number of times A occurs without B plus the number of times B occurs without A plus the number of times A and B occur together [42].
The uncertainty coefficient [42] is calculated as:H is the entropy associated with A, B, and AB. For A, the entropy is calculated from the probabilities of A occurring in n1 out of N lists (n1/N) and A not occurring in n2 out of N lists (n2/N). Analogous calculations lead to the entropy associated with B. For H(A,B), the probabilities of A occurring without B, B occurring without A, A and B occurring together, and neither A nor B occurring in the lists are used.<br><br>Co-occurrence analysis of the PubLiME data set
The bipartite graph to be analyzed is composed of 27619 PubMedID_listID-gene pairs (see supplementary material Table S1). Edge-swapping (1000 simulations, see above) was used to determine occurrence probabilities and gene co-occurrence was analyzed using module size 3 (co-occurrence of three genes), bi-binomial Z-score cutoff 6, bi-binomial P-value cutoff 1.0E-5, and support 5. Supplementary material Simulation S1 provides details on why module size 3 is chosen. The support parameter ensures that each 3-gene co-occurrence module is present in at least 5 signatures. We identified 1654 significant modules in the test data compared to 5 modules in a randomized bipartite graph, corresponding to a signal-to-noise ratio of 331. The co-occurrence network was generated from the significant co-occurrence modules by drawing an edge between each pair wise combination of genes that are part of the same co-occurrence module. Gene communities were identified in this network by edge-betweenness clustering removing 4 edges, which resulted in a maximal network modularity of 0.63. Modularity is calculated as described by [43].
For the identification of PubMedID_listID clusters, the PubMedID_listID-gene pairs in the original bipartite graph were reversed to form gene-PubMedID_listID pairs. Occurrence probabilities were obtained by transposing the original occurrence probability matrix determined by edge-swapping as described above. PubMedID_listID co-occurrence was analyzed using module size 5, Z-score cutoff 6, bi-binomial P-value cutoff 1.0E-5, and support 3. Please note that the choice of these parameters is dictated by the parameters used in gene co-occurrence analysis. The reversal of the bipartite graph necessitates the support parameter used in gene co-occurrence analysis (5) to be used as module size in PubMedID_listID co-occurrence analysis and the module size used in gene co-occurrence analysis (3) to be used as the support parameter in PubMedID_listID co-occurrence analysis if the scope of the analysis is the identification of PubMedID_listID clusters that correspond to gene clusters identified before. The significance cutoffs remain unchanged. PubMedID_listID co-occurrence analysis revealed 448 significant co-occurrence modules in the real bipartite graph and 6 significant co-occurrence modules in the randomized bipartite graph with a signal-to-noise ratio of 75. Communities in the resulting co-occurrence network were identified by edge-betweenness clustering removing 130 edges. The resulting maximal network modularity was found to be 0.47.<br><br>
Supporting Information<br><br>
<h3>pmcA2610030</h3><span class="yellow">Human</span> genetic selection on the MTHFR 677C>T polymorphism
Abstract
Background
The prevalence of genotypes of the 677C>T polymorphism for the MTHFR gene varies among <span class="blue">humans</span>. In previous studies, we found changes in the genotypic frequencies of this polymorphism in populations of different ages, suggesting that this could be caused by an increase in the intake of folate and multivitamins by <span class="yellow">women</span> during the periconceptional period. The aim was to analyze changes in the allelic frequencies of this polymorphism in a Spanish population, including samples from spontaneous abortions (SA).<br><br>Methods
A total of 1305 subjects born in the 20th century were genotyped for the 677C>T polymorphism using allele specific real-time PCR with Taqman® probes. A section of our population (n = 276) born in 1980–1989 was compared with fetal samples (n = 344) from SA of unknown etiology from the same period.<br><br>Results
An increase in the frequency of the T allele (0.38 vs 0.47; p < 0.001) and of the TT genotype (0.14 vs 0.24; p < 0.001) in subjects born in the last quarter of the century was observed. In the 1980–1989 period, the results show that the frequency of the wild type genotype (CC) is about tenfold lower in the SA samples than in the controls (0.03 vs 0.33; p < 0.001) and that the frequency of the TT genotype increases in the controls (0.19 to 0.27) and in the SA samples (0.20 to 0.33 (p < 0.01)); r = 0.98.<br><br>Conclusion
Selection in favor of the T allele has been detected. This selection could be due to the increased fetal viability in early stages of embryonic development, as is deduced by the increase of mutants in both living and SA populations.<br><br><br><br>Background
The methylenetetrahydrofolate reductase enzyme (MTHFR) catalyzes a reaction that produces 5-methyltetrahydrofolate (5-methylTHF), the methyl donor for homocysteine in the synthesis of methionine. The 677C>T mutation of the MTHFR gene has been associated with a thermolabile enzyme with decreased activity that may cause an increase in plasma homocysteine concentrations [1] when folate status is poor. This polymorphism is one of the most widely studied clinically relevant polymorphisms in <span class="blue">humans</span>, as it is related to cardiovascular disease [2] and neural tube defects (NTD; 601634) [3].
A large number of studies have provided a broad overview of the prevalence of the 677C>T polymorphism in different <span class="yellow">human</span> populations, showing that the distribution of frequencies is diverse [4]. These differences have been also observed between groups of different ages in the same Spanish population (older and younger than 24 years) [5] and in a Swiss population (older and younger than 60 years) [6], as well as in a Japanese population [7].
In some populations, such the Toscanians in Italy [8] and Mexicans [9], the homozygous mutated genotype (TT) has reached frequencies greater than 30%. On the other hand, in Africans the frequency of the TT genotype is very low (less than 1%) [10,11], but, in African-Americans, it has already reached 2% [12]. Studies based on the distribution of genotypic and allelic frequencies of the 677C>T polymorphism and the 1298A>C polymorphism in the MTHFR gene in Israeli, Japanese and Ghanaian Africans populations [13] concluded that the 677T mutation in the MTHFR gene emerged as a founder haplotype with some selective advantage. Recently, preliminary evidence of genetic selection of this polymorphism related to folate intake has been reported [14].
The aim of the present study is to analyze the changes in frequencies of the 677C>T polymorphism during the 20th century and particularly the evolution of the frequencies during the decade of 1980–1989, by comparing the genotype frequencies between living subjects born in this period versus samples of spontaneous abortions (SA) that occurred during in the same time period.<br><br>Methods
Subjects
This study was approved by the Ethics Committee at the University Hospital "Virgen de la Victoria" (Málaga). One of the study groups consisted of 344 fetal tissue samples from SA, obtained from the Department of Pathology of the University Hospital Carlos Haya (Málaga). These samples were selected after checking the clinical history and by the inclusion criteria of containing histologically confirmed fetal tissue collected in the 1980s from SA at less than 3 months (11 ± 1.70 week) and of unknown etiology. These fetal samples were compared with a control population of 276 subjects born in the 1980s with an average age of 22 ± 4.58.
Another population of subjects born in the south of Spain in the 20th century were genotyped (1305 subjects, 697 <span class="yellow">women</span> and 608 <span class="yellow">men</span>) and divided into four groups according to birth date: 1900 to 1925 (n = 206); 1926 to 1950 (n = 320), 1951 to 1975 (n = 408), 1976 to 2000 (n = 371). Individuals were selected randomly from different areas of the province of Malaga, in southern Spain, and from different social statuses to avoid a selection bias. All the selected individuals were Caucasian and residents of the study area. The parents of all subjects included in the study were also Caucasian and born in Spain. The possibility of a founder effect or genetic drift was investigated and rejected. All the selected individuals were also genotyped for an insertion/deletion polymorphism in the angiotensin converting enzyme (ACE) gene and/or the 2756A>G polymorphism in the methionine synthase gene (MTR), in order to determine whether or not our adult and young populations were genetically homogeneous. No significant differences were observed in allelic or genotypic frequencies for these genes between the different groups.
The population studied was randomly selected according to age. Subjects 0–12 years old were selected from dried blood spots from neonatal screening papers; subjects 10–24 years old were recruited from students in primary and secondary schools and in university; subjects 25–50 years old and >51 years old were recruited using their Andalusia Health Service identity cards. After approval by the University Hospital Ethical Committee, all the subjects were contacted, and, from those whose written consent was obtained, 10 ml of blood was taken. The investigation in this study conforms to the principles outlined in the Declaration of Helsinki.<br><br>Genetic analysis
The fetal samples were extracted from the archived formalin-fixed, paraffin-embedded tissue sections. Genomic DNA was extracted from fetal tissue using the method described by Coombs et al. (1999) [15]. Genomic DNA of the second and third groups was extracted from peripheral leukocytes using the AquaPure Genomic DNA Blood Kit (Bio-Rad).
Genotyping was performed using Real Time PCR with allele specific Taqman® probes and primers described by Ulvik et al. (2001) [16] and the following optimized protocol for 45 cycles: 10 s – 94°C, 40 s – 54°C, 15 s – 72°C. The PCR mix (25 μl total volume) consisted of 5 μl of genomic DNA, 0.5 μl of sense primer, 0.62 μl of anti-sense primer, 0.85 μl Taqman® probe FAM, 0.43 μl Taqman® probe TET, 20 μl PCR-buffer iQ-SupermixTM (Bio-Rad) (containing 100 mM KCl, 40 mM Tris-HCl, (pH 8.4) 1.6 mM dNTP (dATP, dCTP, dGTP and dTTP), iTaq® polymerase (50 units/mL) and 6 mM MgCl2) and 17.75 μl H2O.<br><br>Statistical and mathematical analysis
All samples were genotyped, and the allelic and genotypic frequencies were compared. Differences were analyzed statistically using the chi-square test or Fisher's exact test. Correlations are expressed using Pearson's coefficient (r).
Compliance of genotype distributions with Hardy-Weinberg (HW) equilibrium was evaluated by chi-square analysis. For all tests, a p-value < 0.05 was considered to be statistically significant. Values are expressed as the mean ± SD.
The genetic selection model was calculated for the evolution of the 677C>T genotypes. The genetic selection could be classified as codominant or incompletely dominant and directional with the heterozygous genotype having an intermediate fitness. For this kind of selection, the most appropriate mathematical model is dq = [spq(2hp+q-h)]/[p2 + 2pq(1-hs) + q2 × (1-s)], where dq is the change of frequency of the allele with lower fitness, s is the fraction of that genotype lost to selection, h is the degree of dominance (between 0, for no dominance and 1, for complete dominance), and p is the frequency of the allele with higher fitness.<br><br>
Results
We analyzed the genotype frequencies of the 677C>T polymorphism in a population born during the 20th century. A total of 1305 subjects were divided into four groups of 25 years according to birth date. The genotype frequencies were compared between the four quarters of the century and showed very significant changes (p < 0.001) in the group born in the last quarter of the 20th century (1976–2000), when compared to any of the other groups. The changes show a decrease of the CC genotype and an increase of the TT genotype in the last 25 years of the 20th century. (Table 1)
Considering that each 25 year period corresponds to a generation, allelic frequencies did not change during the first 75 years of the century (HW equilibrium). However, we found that allelic and genotypic frequencies for the 677C>T polymorphism in the last quarter of the century are significantly different compared to the previous generation (1951–1975). The genotype frequencies in the last quarter of the century are not the expected by a HW calculation using the allelic frequencies of the previous generation. This could be described as a consequence of genetic selection found in this population, in the absence of other causes. Applying the mathematical model described above to our population, the calculated fitness (s) is 0.5, and it can be predicted that both alleles will be approximately at a frequency of 50% in the next generation and allele T will be at 90% after seven generations (Figure 1A). Another possibility is that a scenario could be predicted in which both alleles will have frequencies of about 50% in the next generation and that they will maintain this stability while conditions remain unchanged. (Figure 1B)
The comparison of the genotype frequencies between a group of fetal samples from SA that occurred during the 1980–1989 decade and living subjects born in the same decade showed significant differences in genotype frequencies (p < 0.001). CC genotypes were almost absent in abortion samples compared to living subjects (0.03 vs 0.33), while CT and TT genotypes were overrepresented in the same group. When 3-year periods are studied in the decade, we detected a significant increase of the mutated subjects during the decade (CT p < 0.05; TT p < 0.01). Allele frequencies showed the same pattern (p < 0.05). Controls showed the same tendency but without statistical significance. (Table 2)
The evolution of genotype frequencies during the 1980–1989 decade of the TT genotypes correlates well in both living populations as well as fetal samples r = 0.98 (p = 0.11).<br><br>Discussion
Different reports show that the prevalence of the 677C>T polymorphism of the MTHFR gene differs dramatically among <span class="yellow">human</span> populations. Evidence of this dynamism can be observed in many reports: frequency variations between populations that are geographically very close, even in the same country [8]; changes found in the same race or ethnic group such as Africans [10,11] and African-Americans [12]; the high prevalence of the 677C>T poymorphism in populations with special nutritional features such as Mexicans [9] and Japanese [13]; and changes in frequencies between generations of the same population, as has been observed in Spain [5], Switzerland [6] and Japan [7].
There are numerous interpretations of this great diversity, and most tend to be related to adaptation to external conditions such as climate or nutritional status. Dependence of folate degradation on skin pigmentation [17], nutritional habits or <span class="yellow">human</span> intervention periconceptional periods could explain this genetic variation. Definitely, external factors in combination with different levels of MTHFR enzyme activity, conditioned by polymorphisms, could influence the fetal viability of certain genotypes.
In 1998, we suggested the possibility of genetic selection in Spain in favor of the mutants of the 677C>T polymorphism in the MTHFR gene based on the fact that treatment with vitamins and folates during pregnancy increased the viability of fetuses with the TT homozygous genotype. This hypothesis was based on the increase in the number of mutated individuals found in our population since the mid-1970s [5] and the coincident increased intake of vitamins and folate by pregnant <span class="yellow">women</span> in Spain [18,19]. In 2002, a new study found changes in genotype frequencies for the 677C>T and 1298A>C polymorphisms in different age groups. Total homocysteine (tHcy) levels in plasma were also analyzed according to the different genotype interactions [20]. That study hypothesized about fetal viability and about a genetic selection model on the basis of non-linkage disequilibrium between both polymorphisms. Recently, a study with fetal and control populations showed the strong influence of these polymorphisms, though mainly of the 677C>T polymorphism, on spontaneous early abortion [21]. In the present study, significant changes in allelic and genotypic frequencies are detected, as is Hardy-Weinberg disequilibrium, at the 677C>T polymorphism. We hypothesize that there is a dynamic process of genetic selection that favors the T allele. This process of selection started during the last quarter of the 20th century, during which the frequency for mutant homozygous (TT) rose significantly from 14% to 24%. We propose that this increase in mutants is due to the inclusion of an external factor that enhances mutant fetal viability.
If we apply the mathematical model for dynamic selection developed for diploid organisms with sexual reproduction, the T allele could reach to 90% in seven generations in our population (Figure 1A). However, this model assumes selection in a constant environment that applies to all individuals in the population studied. In our case, we suggest that the external factor is related to an increase in folate and vitamin intake in <span class="yellow">women</span> in periconceptional period and does not affect to all individuals [18,19]. We assume that prediction of a classic selection model in this case is only theoretical.
On the basis of a competition between alleles in which an environmental factor favors one allele versus the other, the final result would be that predicted by the previous mathematical model. However in this case, the environment is not selecting against the wild type allele but rather allowing the survival of more mutated alleles. Therefore, the expected result would be not a systematic increase of the mutated allele but the creation of an allelic balance dependent on vitamin and folate abundance conditions. In this case, the mutation would have a lower influence on fetal viability (Figure 1B).
The results showed an increase in mutated genotypes (CT and TT) and a strong protection against abortion by the wild type genotype (CC), which is practically non-existent in the SA group. The frequency of the CC genotype shows no change over the decade studied (1980–1989), which indicates that folate does not exert a visible effect on this genotype. However, the frequency of the mutated allele increases during this decade, especially in fetuses from abortions, and this increase correlated with the increase of the T allele in the control population. This finding suggests that the effect of folate is crucial to viability during the early stages of embryonic development, but, even with folate, not all embryos will survive until birth.
In this population, the mutant allele with lower enzymatic activity has higher fitness than the wild type. In the folate cycle, it can be observed that 5,10-methyleneTHF availability may be important. 5,10-methyleneTHF is the substrate for several reactions in the cycle, but two of them (5-methylTHF and thymidilate synthesis) might be essential for embryo development in folate deficiency conditions.
In both cases, complete or limited MTHFR activity will produce higher or lower 5,10-methyleneTHF availability, which might be an essential factor for embryo development, such that a greater folate levels can compensate the lower enzymatic activity of the mutant.
The implications of this polymorphism in nucleotide synthesis have not yet been determined, but certain data, such as high levels of uric acid found in mutated subjects [22,23], suggest that there are different turnover rates associated with different polymorphisms.<br><br>Conclusion
We suggest that there is genetic selection in our population for the T allele of the MTHFR – 677C>T polymorphism, whose origin could be an increase in fetal viability during the early stages of embryonic development because of an increase in folate and vitamin intake by <span class="yellow">women</span> in the periconceptional period that began to be established in Spain in the last quarter of the 20th century [18,19]. Higher frequencies for the T allele and TT genotype in our population are observed in the living and SA populations.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
AMO performed the statistical analysis, helped to draft the manuscript and revised it for publication. GC is the corresponding author, participated in the acquisition of samples and carried out the genotyping. ARP carried out the bibliographic search and helped to draft the manuscript. AJJ participated in the selection and the processing of samples. MJG coordinated the laboratory work and selected the genotyping method. AR selected the control subjects and designed the consent form. MR helped in the interpretation of data and tables performance. ARE conceived the study and is the guarantor of this work and the general coordinator. All authors read and approved the final manuscript.<br><br>Pre-publication history
The pre-publication history for this paper can be accessed here:<br><br><br><br><h3>pmcA1075922</h3>Tissue-dependent isoforms of mammalian Fox-1 homologs are associated with tissue-specific splicing activities
Abstract
An intronic hexanucleotide UGCAUG has been shown to play a critical role in the regulation of tissue-specific alternative splicing of pre-mRNAs in a wide range of tissues. Vertebrate Fox-1 has been shown to bind to this element, in a highly sequence-specific manner, through its RNA recognition motif (RRM). In mammals, there are at least two Fox-1-related genes, ataxin-2 binding protein 1 (A2BP1)/Fox-1 and Fxh/Rbm9, which encode an identical RRM. Here, we demonstrate that both <span class="yellow">mouse</span> Fxh and A2BP1 transcripts undergo tissue-specific alternative splicing, generating protein isoforms specific to brain and muscle. These tissue-specific isoforms are characterized for their abilities to regulate neural cell-specific alternative splicing of a cassette exon, N30, in the non-muscle myosin heavy chain II-B pre-mRNA, previously shown to be regulated through an intronic distal downstream enhancer (IDDE). All Fxh and A2BP1 isoforms with the RRM are capable of binding to the IDDE in vitro through the UGCAUG elements. Each isoform, however, shows quantitative differences in splicing activity and nuclear distribution in transfected cells. All Fxh isoforms and a brain isoform of A2BP1 show a predominant nuclear localization. Brain isoforms of both Fxh and A2BP1 promote N30 splicing much more efficiently than do the muscle-specific isoforms. Skeletal muscles express additional isoforms that lack a part of the RRM. These isoforms are incapable of activating neural cell-specific splicing and, moreover, can inhibit UGCAUG-dependent N30 splicing. These findings suggest that tissue-specific isoforms of Fxh and A2BP1 play an important role in determining tissue specificity of UGCAUG-mediated alternative splicing.<br><br>INTRODUCTION
Alternative splicing of pre-mRNA is one of the fundamental mechanisms for the regulation of gene expression in higher eukaryotes (1,2). Developmentally regulated, cell type- or tissue-specific, and signal-induced alternative splicing of pre-mRNAs takes place in multicellular organisms throughout their lifetimes. Misregulation or abnormalities in pre-mRNA splicing, in some instances, leads to cellular dysfunctions found in <span class="yellow">human</span> and animal diseases (3–5). Using various model systems of regulated alternative splicing, a number of pre-mRNA features that influence alternative splice site selection have been defined (1,2). These include enhancer and repressor RNA sequences located in exons and introns. Identification of RNA-binding proteins targeting these cis-regulatory elements is currently in progress. In vertebrates, participation of the SR family proteins and hnRNP proteins, such as PTB and hnRNPA1, in alternative splicing regulation via binding to the cis-regulatory elements have been shown in many tissue-specific splicing models (6,7). Although these RNA-binding proteins are ubiquitously expressed, their different abundance in different cells, differences in their post-translational modifications in different cellular contexts and their different abilities to assemble multiprotein complexes in different pre-mRNA contexts are thought to contribute to the determination of cell type-specific patterns of alternative splicing. For the last few years, tissue-specific and tissue-enriched RNA-binding proteins have begun to be identified as splicing regulators. These include brain-specific (or enriched) Nova-1, nPTB (brPTB) and some of the CELF family proteins (8–12). Discovery of these proteins has had a great impact on studies aimed at understanding the molecular mechanisms of alternative splicing regulation.
One of the intronic cis-elements, which are involved in tissue-specific or differentiation stage-dependent regulation of alternative splicing, is the hexanucleotide UGCAUG. The importance of this element was originally recognized in fibronectin pre-mRNA by Huh and Hynes (13). Since then, alternative splicing specific to a variety of tissues or cell types, including neural cells, muscles, epithelial cells and erythrocytes, has been shown to be modulated via this element (14–21). Recently, Jin et al. (21) discovered that a <span class="yellow">zebrafish</span> homolog of <span class="yellow">Caenorhabditis elegans</span> Fox-1 (22) could bind specifically to the pentanucleotide GCAUG by in vitro selection from randomized RNA sequences. This pentanucleotide is almost identical to the hexanucleotide UGCAUG except for the first U. Moreover, the <span class="yellow">zebrafish</span> Fox-1 homolog, as well as the <span class="yellow">mouse</span> Fox-1 homolog, are capable of repressing the inclusion of an alternative cassette exon of the ATP synthase F1γ pre-mRNA via binding to GCAUG, which mimics muscle-specific exclusion of this exon. This <span class="yellow">mouse</span> homolog is identical to the ataxin-2 binding protein 1 (A2BP1), which has been previously cloned in <span class="blue">humans</span> and <span class="yellow">mice</span> as the cDNA encoding a protein, which interacts with ataxin-2, the product of the causative gene for spinocerebellar ataxia type 2 (23,24). In addition to A2BP1/Fox-1, another <span class="yellow">mouse</span> homolog of <span class="yellow">C.elegans</span> Fox-1, Fxh, has been independently cloned as a cDNA, which is induced by androgen in motor neurons (25). Of note is that A2BP1/Fox-1 and Fxh share an identical RNA recognition motif (RRM) at the amino acid level. Therefore, two genes in the <span class="yellow">mouse</span> genome encode homologs of nematode Fox-1. According to the names given by the first cDNA cloning, we used the nomenclature of A2BP1 and Fxh in this report. A2BP1 and Fxh have been named A2BP1 and Rbm9, respectively, in the <span class="yellow">human</span> and <span class="yellow">mouse</span> genomes.
We have been studying regulatory mechanisms of neural cell-specific alternative splicing using the non-muscle myosin heavy chain II-B (NMHC-B) gene as a model system (14,26). NMHC-B mRNA is expressed ubiquitously. However, an alternative exon, N30, which encodes a 30 nt coding sequence, is included in the mRNAs from some neural cells, but is skipped in those from all other cells in mammals and birds (27,28). In cultured cells, a switch in N30 splicing from exclusion to inclusion can be seen in neural retinoblastoma Y79 cells during the post-mitotic and differentiated stages triggered by butyrate treatment. We have previously defined an intronic distal downstream enhancer (IDDE), which confers neural cell specificity on N30 inclusion, using this cell line (14). The IDDE includes two copies of UGCAUG. Mutation of these hexanucleotides results in N30 skipping in post-mitotic differentiated Y79 cells.
In this study, we investigated the possible involvement of A2BP1 and Fxh in the regulation of N30 splicing. To this end, we have isolated cDNA clones for A2BP1 and Fxh from brain and muscles. cDNA cloning revealed the existence of tissue-specific (enriched) isoforms of both A2BP1 and Fxh. Of importance, different isoforms of A2BP1 and Fxh show different activities with respect to N30 splicing as well as different subcellular localizations.<br><br>MATERIALS AND METHODS
Database disposition
The sequences reported in this paper have been deposited in the GenBank database with accession numbers AY659951 (F011), AY659952 (F411), AY659953 (F402), AY659954 (A016), AY659955 (A030), AY659956 (A713), AY659957 (A715) and AY659958 (A704).<br><br>RNA preparation and RT–PCR
Total RNAs were isolated from <span class="yellow">mouse</span> tissues and cultured cells using an RNA isolation kit (Stratagene) or an RNeasy mini kit (Qiagen). To obtain the full-length coding regions of cDNAs for Fxh and A2BP1, RT–PCRs were performed using Superscript II RNase H− reverse transcriptase (Invitrogen) and Pfu Turbo DNA polymerase (Stratagene). The PCR primers used to obtain all Fxh cDNAs were 5′-ctcaggcctccactagttATGGAGAAAAAGAAAATGGTAACTC-3′ and 5′-ctcaggcctcctctagaaGTAGGGGGCAAATCGGCTGTA-3′. The upstream primers for the brain (A016 and A030) and muscle (A713, A715 and A704) A2BP1 cDNAs were 5′-ctcaggcctccactagtgATGAATTGTGAAAGAGAGCAGCT-3′ and 5′-ctcaggcctccactagtcATGTTGGCGTCGCAAGGAGTCC-3′, respectively, and the downstream primer for all A2BP1 cDNAs was 5′-ctcaggcctcctctagagATATGGAGCAAAACGGTTGTATCC-3′. Lower case letters represent adapter sequences including restriction enzyme sites. 5′ Rapid amplification of cDNA ends (RACE) was performed using Marathon-Ready cDNA (BD Biosciences Clontech). For the analysis of minigene and NMHC-B mRNAs, RT–PCRs were performed as described previously (14,26). Sequences of primers P1–P9 shown in Figures 1, 4 and 6 are as follows: P1, 5′-AATTCACCCAGCAACCAGAAT-3′; P2, 5′-TAGAGGGATGTAAGTGTTGATGCC-3′; P3, 5′-CAGAGGGCGGACAGTGTATGGT-3′; P4, 5′-GGCGGCAGGGGCGAGGGCAT-3′; P5, 5′-CCGTGGTCGCACCGTGTACAAC-3′; P6, 5′-CAGCGGCAGTGGCAGGGGTG-3′; P7, 5′-AGGAAGAAAGGACCATAATATTCC-3′; P8, 5′-CCTCCACCCAGCTCCAGTTGT-3′; and P9, 5′-CCTGTAGTTATTAAATCCTTCAAG-3′.<br><br>Preparation of expression constructs and minigenes
The cDNAs of Fxh and A2BP1 were introduced into a plasmid pCS3 + MT, which contains a myc-epitope, and its modified version pCS3 + MT + NLS, which in addition contains the nuclear localization signal (NLS) of the <span class="yellow">SV40</span> large T antigen (26). Minigenes G, J without the IDDE, and H with the wild-type IDDE are the same as minigenes W, D4 and Cm0, respectively in ref. (14). The 201 nt IDDEs with mutations were generated by recombinant PCR using the appropriate primers, which included mutated sequences. The hexanucleotide TGCATG sequences at the 5′ and 3′ sides were changed to GTTACT and ACCTAC, respectively.<br><br>Electrophoresis mobility shift assay
Template DNAs for in vitro RNA transcription were prepared by PCR using the wild-type and mutant IDDEs in the minigenes as templates and an upstream primer that included the T7 promoter sequence at the 5′ end. The probe and competitor RNAs were transcribed by T7 RNA polymerase in the presence of [α-32P]UTP and a trace amount of [35S]UTPαS, respectively, using a MAXIscript kit (Ambion). Mole concentrations of synthesized RNAs were estimated by radioactivities. Fxh and A2BP1 proteins with a myc tag were synthesized in vitro by using a TNT quick-coupled transcription/translation system (Promega) from pCS3 + MT constructs, which include the SP6 promoter. Binding reactions were carried out in a 10 μl mixture that contains 10 mM HEPES (pH 7.9), 2 mM MgCl2, 50 mM KCl, 5% glycerol, 0.5 mM DTT, 5 μg tRNA, 1 μl reticulocyte lysate reaction mixture and 15 fmol of probe, on ice for 20–30 min. An aliquot of 5 μg of heparin was added to the reaction 10 min before gel electrophoresis. The reaction mixtures were analyzed by electrophoresis in a 6% polyacrylamide gel using a 0.5× TBE buffer (Invitrogen).<br><br>Cell culture and transfection
The <span class="yellow">human</span> retinoblastoma cell line Y79 was cultured and transfected with DNAs as described previously (14,26). Total amounts of transfected DNAs were adjusted to be constant by addition of the empty vector. Either Lipofectin (Invitrogen) or Effectene transfection reagent (Qiagen) was used for the transfection. For stable transfection, the pCS3+MT expression constructs were co-transfected with a plasmid carrying a neomycin resistant gene and selected by 0.2 mM geneticin (Invitrogen). For differentiation of Y79 cells, cells were plated on the poly-d-lysine-coated plates and then treated with 2.0–2.5 mM sodium butyrate for 4–5 days. HeLa cells were cultured as described and transfected with DNA using Effectene reagent (14,26).<br><br>Immunoblot analysis
Samples that required both protein and mRNA analysis were split upon harvesting. Total cell proteins were subjected to SDS–PAGE and blotted as described previously (26). The primary antibodies used are monoclonal antibodies to a myc-epitope (Invitrogen) and green fluorescent protein (GFP) (Clontech). Binding of antibodies was detected with the SuperSignal System (Pierce) or ECL (Amersham).<br><br>Immunofluorescent microscopy
HeLa or Y79 cells grown in a four-chamber glass slide were transfected as described above. Cells were fixed with 10% formaldehyde 24–48 h after transfection, and permealized with 0.5% Triton X-100 then blocked with 5% <span class="yellow">goat</span> serum. Primary antibodies used were <span class="yellow">mouse</span> anti-myc (Invitrogen), <span class="yellow">rabbit</span> anti-NMHC-B (29). Secondary antibodies were Alexa Fluor 594 <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG and Alexa Fluor 488 <span class="yellow">goat</span> anti-<span class="yellow">rabbit</span> IgG (Molecular Probes). DAPI was used for DNA staining. Specimens were mounted in ProLong antifed kit (Molecular Probe). The images were collected using Leica SP confocal microscopy (Leica).<br><br>
RESULTS
Tissue-dependent isoforms of Fxh and A2BP1 and their subcellular distribution
The A2BP1 mRNAs are detected almost exclusively in brain and striated muscles (heart and skeletal muscles) in adult <span class="yellow">mice</span>, whereas the Fxh mRNAs are expressed in a wide variety of tissues with the highest expression in brain and heart [(21,25) and S. Kawamoto, unpublished data]. These expression profiles prompted us to characterize the mRNAs of A2BP1 and Fxh in brain and muscles. We have cloned cDNAs for two gene transcripts from brain, heart and skeletal muscles by RT–PCR and 5′ RACE. The isolated cDNAs are schematically presented with genomic structures in Figure 1 (for amino acid sequences see also Figure 8). Both gene transcripts are found to undergo tissue-specific alternative splicing. In both cases, brain and striated muscles express unique splice variants generated by mutually exclusive splicing of exons B40 and M43, respectively, which provide different coding sequences in the middle of the carboxyl half of the molecules. Southern blot analysis of the RT–PCR products of the Fxh mRNAs from the different tissues probed by oligonucleotides corresponding to B40 and M43 shows that M43 is exclusively used in heart and skeletal muscles, while B40 is predominantly used in brain (Figure 2A). Digestion of the RT–PCR products of the A2BP1 mRNAs with restriction enzymes unique to B40 and M43 demonstrates that B40 and M43 are used almost exclusively in brain and muscles, respectively (Figure 2B). A2BP1 contains an additional cassette type alternative exon A53, consisting of 53 nt. Inclusion and exclusion of exon A53 results in two different amino acid sequences at the C-terminal region owing to a frame shift. In the case of the A2BP1 gene, moreover, it appears that brain and striated muscle utilize alternative promoters, resulting in different amino acid sequences at the N-terminus. The 5′ RACE of skeletal muscle mRNAs yielded essentially a single species of sequence with 29 unique N-terminal amino acids. The 5′ RACE using brain mRNAs; however, yielded multiple products with multiple deduced amino acid sequences, all of which differ from the muscle amino acid sequence at the very N-terminus. Here, we focus our analysis on a clone containing nine unique amino acids at the N-terminus, which was obtained most frequently. Exon–intron organization of Fxh and A2BP1 shows remarkable similarities and a RRM is encoded in four exons (Figure 1B). Of note is that the significant amounts of Fxh and A2BP1 mRNAs from skeletal muscles are missing a part of the RRM by exon skipping. Typically, as shown in Figure 2C, they lack the 93 nt exon that encodes RNP1, one of the two most critical motifs of the RRM (30). Some of them lack almost the entire RRM (e.g. F402 in Figure 1A).
To examine the subcellular distribution of each isoform of Fxh and A2BP1, myc-tagged proteins were transiently expressed in cultured cells and immunostained with an anti-myc antibody. Initially, HeLa cells were used to investigate subcellular localization, since these cells are more suited for these studies. Representative confocal images are shown in Figure 3A. DAPI and anti-NMHC-B antibodies serve as markers for nuclei and cytoplasm, respectively. As noted, the ratio of protein distributed between nuclei and cytoplasm differs among the proteins. All isoforms of Fxh have a predominant nuclear localization. The brain isoform of A2BP1 without the A53 exon (A016) localizes to both nuclei and cytoplasm, whereas the other brain isoform with the A53 exon (A030) localizes predominantly to cytoplasm with only a minimum being in the nuclei. The relative amounts of both muscle isoforms of A2BP1 (A713 and A715) are somewhat between the amounts of the two brain isoforms. These data are summarized in Figure 1A. The subcellular distribution of representative isoforms (F011, A016 and A030) was also examined in retinoblastoma Y79 cells, which were used as host cells for the transfection experiments in order to characterize splicing activities of Fxh and A2BP proteins (see below). Although Y79 cells have a spherical shape and have only thin cytoplasm, exogenously expressed isoforms of Fxh and A2BP1 show a similar subcellular distribution as in HeLa cells (Figure 3B). F011 localizes predominantly to nuclei, A016 to both nuclei and cytoplasm and A030 predominantly to cytoplasm.<br><br>Specific interaction of Fxh and A2BP1 with IDDE via a hexanucleotide UGCAUG
Fxh and A2BP1 share an identical RRM and A2BP1 has been reported to bind specifically to the pentanucleotide GCAUG through this RRM (21). We have previously reported that the IDDE of the NMHC-B transcript, which is indispensable for the regulation of neural cell-specific cassette type exon N30 splicing, has two copies of GCAUG (14). Therefore, we investigated whether Fxh and/or A2BP1 bound to the IDDE. Electrophoretic mobility shift assays (EMSAs) were carried out using labeled IDDE and in vitro transcribed and translated Fxh and A2BP1, which include a myc-epitope. Since all Fxh and A2BP1 isoforms, except F402 and A704, contain the identical RRM, representative isoforms were analyzed. The expression of F011 in reticulocyte lysate causes the formation of a RNA–protein complex whose migration shift distinguishes it from those of the control reticulocyte lysate (C in Figure 4B, lanes 3 and 10). The unlabeled wild-type IDDE competes with the probe efficiently for the formation of the specific complex C (Figure 4B, lane 4). On the other hand, the mutant mc (Figure 4A), which has a mutation in both copies of UGCAUG, does not (Figure 4B, lane 7). The mutant ma, which has a mutation in the hexanucleotide at the 5′ side, shows less efficient competition, compared with mb, which has a mutation in the 3′ side of the hexanucleotide (Figure 4B, lanes 5 and 6), indicating that the nucleotides at the 5′ side are more important than those at the 3′ side. The presence of an anti-myc antibody, but not a non-specific antibody, inhibits the formation of the specific complex (Figure 4B, lane 8). Synthesis of the full-length F011 protein using a reticulocyte lysate and specificity of the myc antibody for the expressed protein are verified by immunoblot analysis (Figure 4B, lanes 11 and 12). A016 and A030 show essentially identical results to those with F011 (data not shown). These results indicate that Fxh and A2BP1 can bind to the IDDE and that the hexanucleotide UGCAUG is required for their binding.<br><br>Fxh and A2BP1 enhance N30 inclusion in a UGCAUG-dependent manner
To study whether Fxh and A2BP1 regulate neural cell-specific splicing via binding to UGCAUG, the Fxh and A2BP1 expression constructs were co-transfected into retinoblastoma Y79 cells with a number of the reporter minigene constructs, which include the wild-type or a mutant version of UGCAUG in the IDDE (Figure 4A). Minigenes H and J consist of the exons E5, N30 and E6 and their flanking introns with some deletions in the introns. The IDDE with or without mutations in the hexanucleotide is included or excluded between N30 and E6. As described above, each isoform of Fxh and A2BP1 enters the nucleus to a different extent. To see the effects of different proteins on N30 splicing itself, independent of their differential properties in nuclear localization, an exogenous NLS was added to the expressed proteins. Since Fxh and A2BP1 proteins contain the identical RRM, and F011, A016 and A030 all show the same UGCAUG-dependent binding to the IDDE in vitro, F011 and A030 were used for these experiments. Host cells Y79 at the proliferating stage exclude the N30 exon in ∼90% of the endogenous NMHC-B mRNAs (e.g. see Figure 6A, lane 1).
As shown in Figure 4C, the mRNAs derived from minigene J exclude N30 without exogenous expression of Fxh or A2BP1 in Y79 cells, similar to the endogenous NMHC-B mRNAs (Figure 4C, upper panel, lanes 1–5). However, in the presence of exogenous expression of F011 and A030, the N30 inclusion is increased (Figure 4C, upper panel, lanes 7 and 12). The N30 inclusion is absolutely dependent on the presence of the IDDE (Figure 4C, upper panel, lanes 6 and 11). Moreover, mutation of either one of the two copies of UGCAUG (ma, mb) abolishes the inclusion of N30 (Figure 4C, upper panel, lanes 8–10, 13–15). In the context of minigene H, which contains shorter introns, the larger extent of N30 inclusion is induced by either F011 or A030 overexpression (Figure 4C, middle panel, lanes 7 and 12). The N30 inclusion of the minigene H mRNAs also depends on the presence of the IDDE (Figure 4C, middle panel, lanes 6 and 11). Mutation of both copies of UGCAUG (mc) results in a complete loss of N30 inclusion (Figure 4C, middle panel, lanes 10 and 15). The mutant ma shows stronger inhibition of N30 inclusion compared with mb (Figure 4C, middle panel, lanes 8 and 9). This observation is consistent with the competition experiments of the EMSA shown in Figure 4B, indicating that the 5′ hexanucleotide is more important than the 3′ hexanucleotide for binding of Fxh or A2BP1 to the IDDE as well as an activation of N30 splicing. Comparable amounts of F011 and A030 are expressed in each transfection as verified by immunoblots using an anti-myc antibody (Figure 4C, lower panel, lanes 6–17).
Since minigenes J and H lack a portion of the intron between exons N30 and E6 and, therefore, the IDDE is located ∼100 nt downstream of N30, instead of 1.5 kb as in the native gene, we also analyzed the effects of Fxh and A2BP1 on the N30 splicing of the wild-type minigene G, which includes full-length introns among E5, N30 and E6 (Figure 4A). As shown in Figure 4C (right panel), both F011 and A030 are capable of promoting N30 inclusion, with F011 showing a higher activity (Figure 4C, lanes 16 and 17). Although A030 can promote N30 inclusion as efficiently as F011 in the contexts of the minigene J and H transcripts, it can do so less efficiently than F011 in the context of the minigene G transcript. Interpretation of this observation will be discussed below.
Taken together, Fxh and A2BP1 can activate N30 inclusion in an IDDE-dependent manner. The hexanucleotide motif UGCAUG is indispensable for this activation.<br><br>Differential activities of alternatively spliced isoforms of Fxh and A2BP1 in promoting N30 inclusion
Both Fxh and A2BP1 mRNAs are expressed in brain and A2BP1 is also expressed in striated muscles and Fxh is expressed in an even wider variety of tissues. As demonstrated above, however, both Fxh and A2BP1 transcripts undergo tissue-dependent alternative splicing, producing muscle-specific and brain-enriched isoforms. Therefore, the relative activity of individual isoform of Fxh and A2BP1 in promoting N30 inclusion was compared using minigene G.
First, in order to evaluate the relative specific activity of each isoform in the splicing reaction separately from its ability to localize to nuclei, an exogenous NLS was included in the expressed protein to equalize the nuclear concentration of the expressed protein in these experiments. Essentially, all of the expressed proteins with the exogenous NLS are localized to the nucleus (data not shown). Therefore, the relative nuclear concentrations of the expressed proteins can be estimated easily by immunoblots. As shown in immunoblots in Figure 5A, similar quantities of proteins are expressed in a dose-dependent manner. Expression of the brain isoform F011 causes a dose-dependent increase in N30 inclusion and the extent of N30 inclusion reaches a plateau with ∼85% of the mRNAs including N30. In contrast, inclusion of N30 promoted by F411, the predominant isoform from skeletal muscles, reaches a plateau with only 40% of the mRNAs. With respect to A2BP1, the brain isoform A030 shows higher activity in N30 inclusion than the muscle isoform A715, which shows almost no activation, as shown in Figure 5A. Including additional isoforms, the activities of individual isoforms with an exogenous NLS in promoting N30 inclusion are shown in Figure 5B (lanes 8–13) and are also summarized in Figure 1A (nls). Special care was taken to ensure that a similar quantity of each protein was expressed and, if not, different amounts were tested to obtain comparable expression. In the presence of the exogenous NLS (nls), F011 and A016 show the highest activities among all isoforms tested. A30 shows a considerably lower activity than A016. Each of the muscle isoforms (F411, A713 and A715) has a lower activity than that of their brain counterparts (F011, A016 and A030, respectively). As expected, isoforms lacking a part or all of the RRM (F402 and A704) have no activity for N30 inclusion.
Next, the splicing activities of the wild-type proteins without an exogenous NLS were examined. Representative data are shown in Figure 5B (lanes 2–7) and the relative activities are summarized in Figure 1A (wt). The protein amounts detected by immunoblots in these experiments represent the total amounts of the proteins distributed to both the nuclei and the cytoplasm. The splicing activities of the wild-type proteins are consistent with the activities that combine the splicing activities of the proteins with the exogenous NLS and the activities of the native proteins to localize to nuclei. In the absence of the exogenous NLS (wt), the brain isoforms F011, and A016 to a lesser extent, are still capable of activating N30 inclusion efficiently. The muscle isoforms for both Fxh and A2BP1 (F411, A713 and A715), as well as the brain isoform A030, show only minimal activities. Therefore, F011 and A016 appear to have the most physiological relevance to N30 splicing activation.<br><br>Overexpression of Fxh and A2BP1 activates N30 inclusion of endogenous NMHC-B mRNAs
The <span class="yellow">human</span> NMHC-B gene consists of 41 constitutive exons and 3 alternative exons. Its pre-mRNA is ∼156 kb in length and it is much more complex than the pre-mRNA from the minigenes. In addition, the minigene pre-mRNAs are driven by a heterologous promoter. Therefore, we next examined if Fxh and A2BP1 were capable of promoting N30 inclusion of the endogenous transcript. Y79 cells were stably transfected with the expression construct for F011 or A016. Both F011 and A016 are enriched in the brain and show higher activation of N30 inclusion in the minigene transcripts. mRNAs encoding endogenous NMHC-B were analyzed by using RT–PCR. As shown in Figure 6, the inclusion of exon N30 with and without another alternative exon, R18, in the endogenous mRNAs is markedly increased in the clones which were stably transfected with the construct for F011 or A016 containing an exogenous NLS (Figure 6, lanes 4 and 5). Although to a lesser extent, transfection of the wild-type constructs without an exogenous NLS also results in a significant increase in N30 inclusion (Figure 6, lanes 2 and 3). Thus, exogenously expressed Fxh and A2BP1 are capable of activating N30 inclusion not only in the transcripts from the minigenes, but also in those from the native NMHC-B gene in Y79 cells.<br><br>Fxh may cooperate with other factor(s) to promote N30 inclusion
To address the role of endogenous Fxh and its potential interaction with other proteins in promoting N30 inclusion, we made use of an isoform of Fxh, F402, which lacks the RRM. The mutant proteins lacking an RRM for other RNA-binding proteins have previously been reported to function in a dominant-negative fashion and inhibit the activities of the wild-type proteins (26,31). Therefore, the effects of F402 on the N30 inclusion of minigene H mRNAs were examined in the context of the Y79 cells treated with butyrate. Upon butyrate treatment, as reported previously, Y79 cells enter in a post-mitotic and differentiated stage, and importantly, the endogenous as well as the minigene mRNAs in those cells include N30 to a large extent, unlike those in the untreated cells that predominantly exclude N30 (14). As shown in Figure 7, in the absence of exogenous expression of F402, large quantities of the mRNAs derived from minigene H-wt, which contains the wild-type IDDE, include N30 (Figure 7, upper panel, lane 4). In contrast, only small quantities of N30 inclusion are detected in the mRNAs from minigene H-mc, which has mutations in both copies of UGCAUG in the IDDE (Figure 7, upper panel, lane 8). Therefore, the butyrate-treated Y79 cells contain factor(s), which are capable of activating the UGCAUG-dependent N30 inclusion. Co-transfection of the wild-type minigene H-wt with the F402 expression construct causes a dose-dependent inhibition of N30 inclusion (Figure 7, lanes 1–3), indicating that F402 has an antagonistic effect on the endogenous factors with respect to N30 inclusion. Notably, the UGCAUG-independent N30 inclusion seen in the mutant minigene H-mc is not affected by the co-expression of F402 (Figure 7, lanes 5–7), indicating that the inhibitory activity of F402 depends on the UGCAUG element. The parallel experiment with the F011 expression construct does not show a significant effect on N30 splicing in either the wild-type or mutant minigene (Figure 7, lanes 9–16). This may be due to the fact that N30 inclusion of the wild-type minigene mRNAs has already reached a maximal level and is consistent with the idea that butyrate-treated Y79 cells contain sufficient amounts of protein(s) functionally equivalent to F011, which can promote N30 inclusion in a UGCAUG-dependent manner. Since F402 does not bind to the UGCAUG element, and has a UGCAUG-dependent antagonistic effect on N30 splicing, it most probably disrupts protein–protein interactions of the endogenous proteins that are required for the UGCAUG-dependent activation of N30 splicing. Endogenous Fxh (and/or A2BP1) would be a good candidate whose function could be antagonized by exogenously expressed F402. Thus, this observation suggests that the endogenous Fxh has an effect on the activation of N30 splicing and that other proteins cooperate with Fxh for N30 activation. Since muscle cells express the RRM-defective isoforms to a significant extent (Figure 2C) and the wild-type F402 localize to nuclei efficiently, this also raises the possibility that the RRM-defective isoforms may have an inhibitory function on the N30 splicing in muscle cells.<br><br>
DISCUSSION
Two major findings are described in this report. First, Fxh and A2BP1 facilitate neural cell-specific inclusion of the cassette-type exon via binding to the specific intronic sequence UGCAUG. In addition to a minigene model system, Fxh and A2BP1 are capable of facilitating N30 inclusion of the endogenous pre-mRNA. This result provides an important demonstration of physiological relevance and supports the notion that the NMHC-B pre-mRNA is likely to be the true target for Fxh or A2BP1-mediated regulation. However, whether the endogenous Fxh or A2BP1 regulates endogenous NMHC-B pre-mRNA splicing needs to be determined in a future study. In vertebrates, small interfering RNAs and gene targeting strategies have recently been used successfully to address the roles of endogenous splicing regulators in alternative splicing of endogenous target pre-mRNAs (8,32–36). A second and more novel finding is the identification of tissue-specific isoforms of Fxh and A2BP1 with different splicing activities as well as different subcellular localizations. This finding raises the possibility that the products of the Fxh and A2BP1 genes can contribute to a mechanism as to how tissue specificity of alternative splicing is achieved.
Many splicing factors are detected not only in the nuclei, but also in the cytoplasm (37). They are shuttling between the nucleus and the cytoplasm and, in some instances, extracellular stimuli trigger changes in subcellular distribution of these proteins. Such translocations have been reported for hnRNPA1 and PTB (38,39). Moreover, a number of RNA-binding proteins have been demonstrated to play a role in multiple steps during gene expression in different subcellular compartments, such as pre-mRNA processing in nuclei, mRNA export from nuclei to cytoplasm and mRNA localization, stability and translation in cytoplasm (37,40). Therefore, not surprisingly, Fxh and A2BP1 isoforms were found to be distributed in both the nuclei and the cytoplasm in HeLa and Y79 cells. However, the relative ratios of proteins distributed between the two subcellular compartments at steady-state differ among the isoforms. In agreement with Jin et al. (21), substantial amounts of the brain isoform A016 are detected in nuclei. Other A2BP1 isoforms, the brain isoform A030 and the muscle isoforms A713 and A715, are only poorly detected in nuclei. This observation is consistent with the reports where endogenous A2BP1 in cerebellar Purkinje cells, hippocampus neurons and cardiac myocytes were shown to be localized essentially to the cytoplasm (23,24). Thus, inclusion and exclusion of A53 and differences in the very N-terminal sequences results in A2BP1 isoforms with a distinct subcellular localization. It is likely that A2BP1 proteins have multiple roles, involving both nuclear and cytoplasmic events. In contrast, all three Fxh isoforms predominantly localized to the nuclei. Therefore, in terms of their localization, Fxh proteins are better candidates for regulators of the pre-mRNA splicing that takes place in nuclei. Of note, however, our preliminary results of 5′ RACE, as well as the EST database, detect multiple 5′ end sequences for both Fxh and A2BP1 mRNAs, which are presumably generated by alternative promoters and alternative splicing. The diversity of the 5′ end cDNA sequences leads to the generation of a number of unique N-terminal amino acid sequences. Therefore, this study does not exclude the possible existence of other isoforms with different subcellular localizations for both Fxh and A2BP1. Our study also does not exclude the possibility that some of the isoforms translocate between the nucleus and the cytoplasm following stimuli.
The main aim of this study is to determine the relative activities of tissue-dependent isoforms of Fxh and A2BP1 in neural cell-specific and UGCAUG element-dependent alternative splicing. To obtain an indication of the relative specific activity of each isoform in transfected cells, the same amounts of the expressed proteins should be available for the splicing reaction in the nuclei. For this reason, an exogenous NLS was included in the expressed proteins. Essentially, all of the expressed proteins with the exogenous NLS localized to nuclei. Thus, the amounts of the expressed proteins determined by immunoblots represent the nuclear concentrations. The analysis using the proteins expressed with the exogenous NLS allowed us to compare directly the splicing activities of these proteins. Furthermore, this analysis also allowed us to define the critical regions of the proteins for splicing activation.
As shown in Figure 5, the splicing activities of the various isoforms of Fxh and A2BP1 are intrinsically different, regardless of the subcellular localization properties of the wild-type proteins. Among the isoforms tested in this study, F011 and A016, which include B40, are found to have higher activities in promoting N30 inclusion. When the primary amino acid sequences, outside of the RRM, of these two proteins are compared, the C-terminal regions (amino acids 190–377 of F011) show a higher homology with 71% identity, whereas the N-terminal regions (amino acids 1–112 of F011) show only 53% identity. The C-terminal region includes four subregions of nearly identical stretches of amino acids (Figure 8, I–IV). One subregion (II) includes 13 amino acids encoded by exon B40. Substitution of this subregion with exon M43 in Fxh causes substantial changes in amino acid sequences resulting in only a 21% identity in this region between F011 and F411. Another subregion (IV) is located at the C-terminal end and A030 lacks this homologous region by the inclusion of exon A53, which results in a frame shift. Since F411 and A030 show poor splicing activation compared with F011 and A016, respectively, these two subregions of F011 and A016 appear to serve as activation domains, presumably by interacting with other proteins. This notion is supported by the finding that the RRM-lacking isoform F402, which includes the same subregions II and IV as F011, functions apparently as a dominant-negative mutant to the wild-type Fxh, consistent with the interpretation that the mutant and wild-type are competing to interact with other protein(s). To date, Fyn tyrosine kinase and estrogen receptor-α have been reported to interact with Fxh, and ataxin-2 with A2BP1 (23,41,42). Whether these proteins participate in the regulation of pre-mRNA splicing is currently unknown. Of interest, A030 with the exogenous NLS activates N30 splicing as efficiently as F011 in the pre-mRNAs derived from minigenes J and H, which contain the shorter intron, whereas this isoform poorly activates N30 splicing in the pre-mRNA from minigene G, which contains the full-length intron. This observation implies that the interactions of A030 with different factors are required in the different pre-mRNA contexts. Therefore, the isoforms of Fxh and A2BP1 described here may have different effects on other UGCAUG-regulated alternative splicing.
The involvement of the hexanucleotide UGCAUG in regulated alternative splicing has been experimentally demonstrated in a number of neural cell-specific, as well as other tissue-specific, model systems (13–21). This hexanucleotide element plays a role, in most cases, as an enhancer in regulating alternative splicing of cassette-type exons as well as mutually exclusive exons. Furthermore, computational analysis has revealed that UGCAUG is over-represented in the introns in which splicing is regulated, compared with the constitutively spliced introns (43). This analysis also pointed to the UGCAUG element as playing a role in the regulation of tissue-specific alternative splicing in a wide range of tissues, but not in specific tissues.
To date, KH-type splicing regulatory protein (KSRP) (44), A2BP1 (21) and Fxh (this study) are known to be capable of binding to UGCAUG. KSRP is expressed ubiquitously and tissue-specific variants of this gene have not been described so far. In this study, we have described the existence of tissue-dependent isoforms of Fxh and A2BP1, which, while not identical in some areas of the molecule, may contain the same RRM. The physiological relevance of these isoforms is that they have different splicing activities and different subcellular localizations. The brain isoforms promote N30 inclusion more efficiently than the muscle isoforms of both Fxh and A2BP1. The isoforms lacking the RRM are normally expressed to a significant extent in skeletal muscles. This isoform is incapable of activating N30 splicing and, moreover, can inhibit N30 inclusion. The properties of these isoforms are consistent with Fxh and A2BP1 acting as regulators for N30 splicing, since N30 is included in neuronal cells, but excluded in muscles. Therefore, despite the tissue-independent occurrence of UGCAUG as a regulatory element, given the tissue-dependent isoforms of the UGCAUG-binding proteins (Fxh and A2BP1) with different activities, the hexanucleotide UGCAUG could confer tissue specificity on regulated splicing. One of the major problems in understanding the mechanisms responsible for alternative pre-mRNA splicing is the manner in which tissue specificity is determined. In vertebrates, to date, only a few tissue-specific proteins have been identified as splicing regulators (8–12). Here, we have shown that the tissue-dependent isoforms of the sequence-specific RNA-binding proteins, which themselves are generated by alternative splicing, have different activities in tissue-specific alternative splicing of target pre-mRNA. Therefore, these isoforms play a role in the determination of tissue specificity of target pre-mRNA splicing. The discovery of these tissue-dependent isoforms of the UGCAUG-binding proteins with different splicing activities adds an important new dimension to the molecular mechanisms responsible for regulating tissue-dependent alternative splicing mediated via UGCAUG.<br><br>
<h3>pmcA1347483</h3>EndoNet: an information resource about endocrine networks
Abstract
EndoNet is a new database that provides information about the components of endocrine networks and their relations. It focuses on the endocrine cell-to-cell communication and enables the analysis of intercellular regulatory pathways in <span class="blue">humans</span>. In the EndoNet data model, two classes of components span a bipartite directed graph. One class represents the hormones (in the broadest sense) secreted by defined donor cells. The other class consists of the acceptor or target cells expressing the corresponding hormone receptors. The identity and anatomical environment of cell types, tissues and organs is defined through references to the CYTOMER® ontology. With the EndoNet user interface, it is possible to query the database for hormones, receptors or tissues and to combine several items from different search rounds in one complex result set, from which a network can be reconstructed and visualized. For each entity, a detailed characteristics page is available. Some well-established endocrine pathways are offered as showcases in the form of predefined result sets. These sets can be used as a starting point for a more complex query or for obtaining a quick overview. The EndoNet database is accessible at .<br><br>INTRODUCTION
Theoretical analyses in the post-sequencing era, in particular in the context of systems biology approaches, increasingly investigate the properties of all kinds of pathways and networks such as metabolic and signaling pathways. It is commonly accepted that we need formal descriptions of these networks to make systematic use of the overwhelming body of facts gathered over decades of laboratory work both in narrow or global scale.
Corresponding databases have been created and are available now for metabolic networks (KEGG) (1,2), protein interaction networks (BIND and DIP) (3,4) and signaling pathways (CSNDB, Patika and TRANSPATH®) (5–8), just to name a few. So far, however, their main focus is on intracellular processes. Intercellular signaling is addressed only insofar as usually the pathways modeled start with extracellular ligands, and as there exist catalogs and databases about secreted proteins, or the ‘secretome’, of certain systems (9–12).
This shortcoming is part of the more comprehensive problem, the genotype–phenotype gap: from a certain genotype, we are able to infer a ‘molecular phenotype’, but for correlating it with a more complex phenotype such as a biological process, or even a certain disease and its clinical appearance, we still depend largely on the mere description of an observed correlation. There is no way to infer such a phenotype through all the different layers of increasing complexity between genomic DNA sequences and the physiological function of whole organs and their interplay within an organism.
There may be principal barriers preventing such an inference across different complexity levels, but even to explore these limits, we have to make attempts to bridge the genotype–phenotype gap. We have to do the next step towards modeling intercellular networks that are inextricably linked to the physiology of multicellular organisms (13). Being one of the most complex constructs in the body, the endocrine system comprises numerous cells and tissues that secrete hormones which pass through the body, activate specific receptors of target cells and initiate there multiple intracellular signaling pathways.
Here, we present a new database, EndoNet, which provides information about the components of endocrine networks and their relations, and enables the analysis of intercellular regulatory pathways in <span class="blue">humans</span>. The EndoNet database is accessible at .<br><br>RESULTS
The biological schema of endocrine actions
Development and function of different organs as well as the response of a whole multicellular organism to its environment is coordinated through a complex communication system between specialized cells that are part of its organs. This communication is mostly mediated by hormones. In a broader sense, this functional class of biomolecules also comprises growth factors, cytokines, chemokines and other signal transmitters. This generic view is supported, for instance, by the definition given for ‘hormone activity’ by Gene Ontology (GO) (14): ‘Any substance formed in very small amounts in one specialized organ or group of cells and carried (sometimes in the bloodstream) to another organ or group of cells, in the same organism, upon which it has a specific regulatory action’. This definition is also broad enough to include modes of hormonal actions as diverse as endocrine, paracrine and autocrine effects. Accordingly, with the term ‘hormone’ one might refer to any extracellular substance that induces specific responses in target cell and helps to coordinate growth, differentiation, gene expression and metabolic activities of various cells, tissues and organs in multicellular organisms (15).
Hormones can be classified based on their chemical nature, solubility, the distance over which the signal acts and so on (15,16). From the viewpoint of genome–phenotype relations, it is reasonable to distinguish between polypeptide, thus, genome-encoded hormones, on one hand, and those low-molecular weight hormones such as steroids, with only the machinery of their synthesis being genome-encoded, on the other hand. Another classification of hormones, which seems to be overlapping with the previous one refers to the intracellular location of their receptors and, thus, how the subsequent signal is further transduced: membrane-bound receptors usually trigger more or less complex signaling cascades towards the nucleus, whereas nuclear receptors, mainly bound by low-molecular weight hormones, have a very short signaling pathway downstream since they act as transcription factors themselves (15,16).
In intercellular communication, we can basically differentiate between two kinds of cells: donor cells which synthesize and secrete a hormone, and acceptor cells which express a hormone receptor (Figure 1a). Donor cells become active under the influence of an external, mostly environmental, stimulus. In the acceptor cell, binding of the hormone to a receptor triggers an intracellular signal transduction cascade with different kinds of end nodes and effects: transcription factors affecting the gene expression program of the acceptor cell, metabolic enzymes controlling the cell's metabolism, structural components which define the acceptor's morphological features, or components of the secretory apparatus regulating the release of other extracellular molecules. If synthesis and secretion of another hormone is among the effects exerted by receptor activation, the acceptor is turned into a donor cell, thus becoming an internal node of the organism's endocrine network. Acceptor cells which do not become producers of another hormone are called ‘terminal target cells’ of the endocrine network, but finally constitute the overall physiological effect of the respective hormonal pathway, or simply the phenotype (Figure 1a).<br><br>The EndoNet data model
In the EndoNet data model, two classes of entities—hormones (in the broadest sense) and their acceptor or target cells expressing the corresponding receptors—span a bipartite directed graph. Since one and the same hormone may be secreted by multiple cell types (donor cells), each such secretion event is represented by a hormone node on its own. Similarly, each cell type known to express a hormone receptor (acceptor cell) leads to an individual node. The graph's edges represent hormone transport and binding to a receptor (intercellular edges), on one hand, and triggering or inhibition of hormone secretion by a receptor activated by hormone binding (intracellular edges), on the other hand. Optionally, an edge representing the transport of a hormone can be subdivided by introducing the transport medium (usually blood) as an additional, intermediary node.
Thus, in the conceptual schema of the EndoNet database (Figure 1b), the links between cells/organs and hormone define donor cells (‘D’), those between cells/organs and receptors acceptor cells (‘A’). If an acceptor cell synthesizes another hormone in response to an incoming signal, it becomes an internal node in the emerging hormonal network.
In EndoNet, the pathway between a hormone receptor expressed in an acceptor cell and a hormone synthesized in the same cell (intracellular edge) is handled as a black box. In case of genome-encoded peptide hormones, cross-references to entries in the TRANSPATH® database, which describe the signaling cascade starting from the hormone's receptor and ending at the hormone's gene, are provided, if available. Datasets on non-peptide hormones will in future be enriched by a specific metabolic add-on which will include references to the databases KEGG (1) and BRENDA (17), allowing for further characterization of the steps performed during the hormone's synthesis and the regulation of both activity and expression of the enzymes involved.
By now, the EndoNet structure already allows for including descriptions of the physiological effects induced by hormone binding (see below, Future Developments).<br><br>The contents of EndoNet
In the present version of EndoNet, and as a first approach, we consider the endocrine (hormonal) network of the <span class="yellow">human</span> body. For each molecule (hormone or receptor), a primary name and synonyms are given. In case of peptide hormones, the sequence of the processed polypeptide, rather than that of the protein precursor, is specified. For a multimeric protein hormone, the subunit composition as well as the sequences of all subunits are stored; the same holds true for hormone receptors.
Additionally, all peptide hormone and receptor datasets have links to HumanPSD™ (18) and to the Swiss-Prot database, The structures of non-peptide hormones can be accessed through the corresponding hyperlinks to the KEGG COMPOUND section. Finally, all molecules may have links to the TRANSPATH® database.
As described, EndoNet utilizes data about the tissues from which hormones are secreted and in which receptors are expressed to define donor and acceptor cells, respectively. The identity and anatomical environment of cell types, tissues and organs is defined through references to the CYTOMER® ontology (19,20); in the numerous cases where the receptors are ubiquitously expressed, just the root term ‘<span class="yellow">human</span> body’ is linked.
Data on whether a hormone's synthesis is triggered or inhibited by another hormone through its respective receptor in a particular cell was obtained by manual selection from textbooks [e.g. (16), ], monographies (21), original literature, the EST library information and the linked databases (TRANSPATH®, HumanPSD™ and Swiss-Prot). The contents of EndoNet are summarized in Table 1.<br><br>Web interface, queries and visualization
EndoNet can be accessed through the WWW via a JSP-based web interface. Hormones, receptors and tissues can be queried for their names, and detailed information on all identified components is available through individual characteristics pages. Each hormone's individual entry page displays its source and target tissues (donor and acceptor cells) as well as its receptors, along with some molecular data. Similarly, each receptor entry exhibits the tissues in which the receptor is expressed, and the hormones it interacts with. Finally, each tissue entry lists the hormone receptors that are found in, as well as the hormones that are synthesized and secreted by the tissue. It is also indicated whether the corresponding tissue exerts gender-specific properties; additional information based on the CYTOMER® ontology is available through the corresponding link on the tissue detail page.
Instead of searching for a name of a hormone, one can also browse the hierarchical hormone classification featured by EndoNet (available at the ‘Search’ page).
Each query result can be used as starting point for an extended query. Different items of interest can be selected and added to a common result set. Since several search results can be combined, it is possible to create sets with multiple search parameters.
At any step of this incremental retrieval process, the sets of hormones, receptors and tissues obtained so far can be used as starting points for reconstructing a network by a depth-first graph traversal algorithm. The maximum number of steps can be selected separately for the upstream and downstream part of the reconstruction process. Subsequently, the graph will be displayed using a Graphviz-based visualization method [(22), ]. In the resulting image, hormones and receptors are represented as nodes grouped together into subgraphs representing the tissues (cells/organs) they are secreted from or expressed in, respectively (Figure 2). Autocrine loops (donor and acceptor cell being identical) are treated specially for visualization. On demand, the hormones' transport media can be included in the visualization of intercellular edges, enabling the user to choose between different complexities of output. Intracellular edges, which connect a receptor to a hormone, represent the influence of a receptor's activation on the secretion of a hormone from the same cell and are displayed differently, depending on whether this influence is triggering or inhibitory in nature.
The graph is displayed as a clickable image map, linking each entity to its detailed characteristics page, thus making the database entries accessible from the graphical overview of a network, too. The graph is available in two different formats: PNG and SVG. While virtually every browser supports PNG (a ‘pixel’ or ‘bitmap’ format), only a few of them provide a zoom functionality for bitmap pictures. Scalable vector graphics, (SVG) provides more functionality (including perfect image quality throughout all zoom factors), but until now most browsers do not support SVG natively and require an SVG plugin () for displaying this vector-based format.
Some well-established endocrine pathways are offered as showcases in the form of predefined result sets. For instance, sets representing the hypothalamic–hypophyseal axis with a focus on either thyroid hormones, adrenal hormones, growth hormones or prolactin are provided. These predefined sets can be used for obtaining a quick overview or as starting points for more complex queries.<br><br>
FUTURE DEVELOPMENTS
Among the important improvements of EndoNet which we are currently working on is the possibility to represent intercellular communication at different levels of the hierarchical organization of organs, tissues and cells in the organism, as well as to distinguish between such communications in male and female organisms. These options will be introduced by a tighter integration with the CYTOMER-based ontology (19,20).
The next step will be to expand the contents of EndoNet towards the details of the processes occurring in the transport medium, usually the blood. Since not all hormones are transported as free molecules and some hydrophobic hormones (e.g. steroids and thyroids) need to be bound to specific carrier proteins, proper description of such transporters and their interaction with the corresponding hormones will be required. It is planned to involve quantitative data about the regular or pathological levels of the hormones, the overall kinetics of each hormone in the blood (monotonous decay, oscillating concentrations, increase in response to certain stimuli, etc.), its turnover and metabolic products, etc. That will allow utilization of EndoNet contents for diagnostic purposes. In future, the EndoNet data model will be extended in order to incorporate external stimuli (e.g. light) and physiological states (stress, age, etc.) in a formalized manner, allowing to determine whether or not and in which quantity a hormone will be secreted under the given circumstances. EndoNet will then link the physiological effects of hormones with the intracellular molecular processes leading to its synthesis and secretion in the donor cells, and to the effects on its acceptor cells. At many places of these intracellular and intercellular networks, genetically determined aberrations may cause specific, sometimes pathological phenotypes. Thus, EndoNet will enable to bridge the gap between known genotypes and their molecular and clinical phenotypes in this area of medical research and its applications.<br><br>DISCUSSION
At its present state, EndoNet provides a high coverage of molecules that are conventionally considered as hormones as well as other molecules that are involved in intercellular communication, such as growth factors, lymphokines and chemokines and their known receptors. The aim of the database is to provide a useful resource for studying the principal features of hormonal networks in a comprehensive way, as it was done more exemplarily in the past for these kinds of networks (23), but was done globally for many other intracellular network types, such as metabolic, protein interaction and transcription networks [reviewed in (24,25)]. EndoNet database certainly is not yet complete but will grow rapidly.<br><br>
<h3>pmcA140144</h3>Gender differences in factors influencing insulin resistance in elderly hyperlipemic non-diabetic subjects
Abstract
Background
The increase in the prevalence of insulin resistance-related metabolic syndrome, a disorder that greatly increases the risk of diabetes, heart attack and stroke, is alarming. One of the most frequent and early symptoms of metabolic syndrome is hypertriglyceridemia. We examined the gender differences between various metabolic factors related to insulin resistance in elderly non-diabetic <span class="yellow">men</span> and postmenopausal <span class="yellow">women</span> of comparable age suffering from hypertriglyceridemia, and compared them with healthy subjects of equal age.<br><br>Results
The indexes of insulin resistance HOMA IR and QUICKI were significantly higher in both hyperlipemic <span class="yellow">men</span> and <span class="yellow">women</span> than in controls; 95% confidence limits of hyperlipemic subjects did not overlap with controls. In both normolipemic and hyperlipemic <span class="yellow">men</span> and <span class="yellow">women</span> serum leptin correlated significantly with insulin resistance, while HDL-cholesterol correlated inversely with HOMA-IR only in <span class="yellow">women</span> (both normo- and hyperlipemic), and serum tumor necrosis factor α (TNFα) only in hyperlipemic <span class="yellow">women</span>. According to results of multiple regression analysis with HOMA-IR as a dependent variable, leptin played a significant role in determining insulin resistance in both genders, but – aside from leptin – triglycerides, TNFα and decreased HDL-cholesterol were significant determinants in <span class="yellow">women</span>, while body mass index and decreased HDL-cholesterol were significant determinants in <span class="yellow">men</span>. The coefficient of determination (R2) of HOMA IR by above mentioned metabolic variables was in <span class="yellow">women</span> above 60%, in <span class="yellow">men</span> only about 40%.<br><br>Conclusion
The significant role of serum leptin in determination of insulin resistance in both elderly <span class="yellow">men</span> and postmenopausal <span class="yellow">women</span> of equal age was confirmed. However, the study also revealed significant gender differences : in <span class="yellow">women</span> a strong influence of triglycerides, TNFα and decreased HDL-cholesterol, in <span class="yellow">men</span> only a mild role of BMI and decreased HDL-cholesterol.<br><br><br><br>Background
In association with pandemic obesity the prevalence of the insulin resistance-related metabolic syndrome is constantly growing [1]. As a consequence of this fact, type 2 diabetes mellitus and cardiovascular mortality occurs in much younger age groups [2]. A typical hyperlipemia, consisting of an increase of serum triglycerides and a decrease of serum HDL-cholesterol, is a characteristic and an early symptom of this syndrome [3].
With increasing age, body mass index (BMI) and adiposity, insulin sensitivity declines and the number of cardiovascular risk factors increases in both genders [4-6]. It was repeatedly demonstrated that plasma concentration of leptin – a hormone produced mainly by adipose tissue – is substantially higher in all age groups of <span class="yellow">women</span> than in <span class="yellow">men</span> [7-10]. This may be caused by different size and/or distribution of fat tissue compartments influenced by hormones: estrogens stimulate, whereas testosterone inhibits leptin secretion. In <span class="yellow">women</span> subcutaneous fat mass prevails – and during augmentation of overweight it increases – while in <span class="yellow">men</span> intra-abdominal fat mass prevails [11-13]. Subcutaneous fat in particular serves as a substantial source of tumor necrosis factor α (TNFα), which represents one of the factors that interfere with insulin signal transduction into the cells [14-16]. Leptin, TNFα and some other factors are abundantly expressed in adipose tissue and contribute to the insulin resistance that accompanies overweight and obesity. Leptin correlates positively with hyperinsulinemia, BMI, fat mass and hypertriglyceridemia, respectively, and correlates inversely with HDL-cholesterol and lean body mass [17-25].
The incidence and mortality of ischemic heart disease and of other consequences of atherosclerosis increases with age in both genders, especially after the age of sixty. In premenopausal <span class="yellow">women</span>, however, the incidence of these disorders is considerably less frequent than in <span class="yellow">men</span> of appropriate age. After the menopause the prevalence of metabolic syndrome and cardiovascular mortality in <span class="yellow">women</span> gradually increases, attaining values comparable to <span class="yellow">men</span> at about the age of 70 [2,26]. Paradoxically, it takes place at the time when serum leptin concentration in <span class="yellow">women</span> has relatively decreased [27,28].
The aim of this study was to analyze the interrelations between several metabolic variables and factors related to insulin resistance in groups of both normal and hyperlipemic postmenopausal <span class="yellow">women</span> and <span class="yellow">men</span> of appropriate age, and to attempt to elucidate the gender differences and some pathophysiologic mechanisms of these differences. We compared homeostatic indexes of insulin resistance HOMA IR and QUICKI, serum lipid and insulin parameters, uric acid, leptin and TNFα between groups of subjects without apparent symptoms of metabolic syndrome, and groups showing mild hypertriglyceridemia with decreased HDL-cholesterol. In addition, serum concentration of the heart fraction of fatty acid binding protein (hFATP) was explored as a factor that might reflect the regulative role of PPAR gamma in lipid homeostasis [29,30], and serum IgG anticardiolipin (ACL-IgG) was investigated as an indirect indicator of oxidized lipid fractions related to atherosclerotic complications [31,32].<br><br>Methods
Subjects
The study was carried out on 70 out-<span class="yellow">patients</span> of the Metabolic Center at the hospital in Sternberk, Czech Republic. From these, 40 <span class="yellow">patients</span> (20 <span class="yellow">men</span> and 20 <span class="yellow">women</span>) were selected with mild hyperlipidemia, i.e. with plasma triglyceride concentration exceeding 2.0 mmol/l, total cholesterol exceeding 6.0 mmol/l, LDL cholesterol exceeding 4.0 mmol/l, and with HDL cholesterol concentration in <span class="yellow">men</span> under 1.0 mmol/l, and in <span class="yellow">women</span> under 1.2 mmol/l. These groups were denominated as "hyperlipemic". Two other groups (10 <span class="yellow">men</span> and 20 <span class="yellow">women</span>) with approximately normal serum values of these variables were taken as "controls". The average age in <span class="yellow">men</span> was 59.1 ± 10.6 y, and in <span class="yellow">women</span> 59.4 ± 10.1 y, respectively. The differences between lipid parameters of hyperlipemic and control groups were highly statistically significant, while the age differences were insignificant (see Table 1). None of the <span class="yellow">patients</span> had clinically apparent diabetes mellitus, but some of the hyperlipemic <span class="yellow">patients</span> exerted impaired glucose tolerance or impaired fasting glucose (values between 6.1 and 7.0 mmol/l, or between 6.1 and 7.8 mmol/l, respectively). None of the <span class="yellow">patients</span> was treated with insulin, peroral antidiabetics or antihyperlipemic drugs; some of them were treated with antihypertensive therapy. No signs of major clinical or laboratory symptoms of other diseases were present in any group of the explored <span class="yellow">patients</span>. Blood samples were obtained in the morning via a venipuncture after overnight fasting. After clotting the serum was separated and stored at -20° until used. An informed consent was obtained from all probands.
Body mass indexes (BMI), defined as weight in kilograms divided by the square of height in meters, were calculated.<br><br>Biochemical methods
Serum leptin concentrations were measured by a sandwich ELISA test kit (<span class="yellow">Human</span> Leptin ELISA, BioVendor Laboratory Medicine, Inc, Czech Republic). Its sensitivity limit was 0.2 ng/ml, intraassay CV 6.1% at the level of 7.5 ng/m, inter-assay CV 8.5% at the level of 4.8 ng/ml. Tetramethylbenzidine was used as a substrate; quality controls were <span class="yellow">human</span> based. Several other hormones and peptides were estimated by routine immunochemical tests: insulin, C-peptide, TNFα (IMMULITE, Diagnostic Products Corporation, Los Angeles, CA, U.S.A.), proinsulin intact (DAKO, Denmark), IgG anticardiolipin (ACL-IgG, IMMCO Diagnostics, Buffalo, NY, U.S.A.) and heart fatty acid binding protein (hFABP, Hbt HUMAN H-FABP, HyCult Biotechnology, Uden, the Netherlands). Serum concentration of glucose, total cholesterol, triglycerides, HDL-cholesterol, LDL-cholesterol, Apoprotein B and uric acid were measured on a ILAB-600 biochemical analyzer (Instrumentation Laboratory, Lexington, Ma, U.S.A.) using BioVendor sets. All samples were processed and examined according to principles of good laboratory practice and under constant intralaboratory and external quality control.
The homeostatic indexes of insulin resistance (HOMA IR and QUICKI) were calculated according to the homeostasis model of assessment [33-35] as follows:
HOMA IR = fasting insulin (μU/ml) * fasting glucose (mmol/l) / 22.5;
QUICKI = 1 / [log fasting insulin (μU/ml) + log fasting glucose (mg/100 ml)].<br><br>Statistics
Statistical analysis was performed using the Version 6 SAS/STAT software (SAS Institute, Inc., Cary, NC, U.S.A.). The Shapiro-Wilks tests were used in testing the normality of distribution. Some of the data obtained were not normally distributed. The statistical significance of differences between the means in the hyperlipemic and control groups were evaluated using the unpaired Student's T-test in the case of normal distribution of data sets, and using the Kolmogorov-Smirnov test when at least in one of the data sets the normal distribution was excluded. Spearman's rank-order correlation was used for correlation analysis. Multiple regression analysis was performed using HOMA IR indexes of insulin resistance as dependent variables, and other metabolic and hormonal factors (lipid parameters, BMI, leptin, TNFα, hFABP, ACL-IgG) as independent variables. The so-called step-down regression model was used to select dominant independent variables. Various four-member groups of independent (explanatory) variables were used for the analysis and the non-zero intercept was taken into account. The independent variables were then dropped, one at a time; at each stage one variable making the least contribution to the dependent variable (i.e. that showed the least p-value in the test of the regression coefficient being zero) was excluded. The coefficient of determination R2, which can be viewed as a percentage explaining the total variance, was simultaneously monitored. A great drop in R2 after excluding some independent variable enabled selection of those independent variables that could be thought to be the most important determinants of the dependent variable.<br><br>
Results
Table 1 demonstrates mean parameters in individual groups of subjects matched according to sex, lipid parameters and age. While the age of all four groups did not differ substantially, the concentrations of total serum cholesterol, triglycerides, HDL-cholesterol and LDL-cholesterol differ very significantly in both male and female hyperlipemic groups as compared with controls. In addition, the concentration of triglycerides in control <span class="yellow">women</span> was significantly higher than in control <span class="yellow">men</span>, the concentration of triglycerides in hyperlipemic <span class="yellow">women</span> was lower than in hyperlipemic <span class="yellow">men</span>, and the concentration of HDL-cholesterol in hyperlipemic <span class="yellow">women</span> was very significantly higher when compared with hyperlipemic <span class="yellow">men</span>.
Table 2 shows the values of other metabolic and insulin parameters, factors related to insulin resistance and indexes of insulin resistance, respectively. Body mass indexes and uric acid concentration were significantly higher in hyperlipemic <span class="yellow">men</span> as compared to controls, but not in hyperlipemic <span class="yellow">women</span>. Uric acid concentration was substantially lower in hyperlipemic <span class="yellow">women</span> than in hyperlipemic <span class="yellow">men</span>. Plasma concentrations of glycemia, insulin and intact proinsulin were significantly higher in both hyperlipemic <span class="yellow">men</span> and <span class="yellow">women</span> as compared with controls of identical gender, while the concentration of leptin increased only in hyperlipemic <span class="yellow">men</span>. However, serum leptin concentrations of both control and hyperlipemic <span class="yellow">women</span> were significantly higher than in corresponding groups of <span class="yellow">men</span>. Serum concentrations of TNFα, hFABP and ACL-IgG in hyperlipemic groups of both <span class="yellow">men</span> and <span class="yellow">women</span> were not significantly different from control groups. On the other hand, the indexes of insulin resistance HOMA IR and QUICKI differed very significantly in hyperlipemic groups of both <span class="yellow">men</span> and <span class="yellow">women</span> as compared with corresponding control groups, more distinctly in <span class="yellow">women</span>.
From Fig. 1, presenting 95% confidence limits of insulin resistance indexes HOMA IR and QUICKI, we concluded that in groups of hyperlipemic <span class="yellow">patients</span> of both genders the insulin resistance was substantially higher than in control groups; the groups did not overlap each other.
In Table 3 the results of Spearman's correlations between insulin resistance index HOMA IR and various metabolic parameters are presented. In the control group of <span class="yellow">men</span>, positive significant correlation between HOMA IR and serum leptin concentration, and inverse significant correlation between HOMA IR and ACL IgG, respectively, were found. In the control group of <span class="yellow">women</span>, the significance of Spearman's correlation between HOMA IR and leptin was more expressive; inverse correlation between HOMA IR and HDL-cholesterol was also present.
In the hyperlipemic group of <span class="yellow">men</span>, the significance of the correlation between HOMA IR was more expressive in relation to the control group, and no significant correlation between HOMA IR and ACL IgG was found. In the hyperlipemic group of <span class="yellow">women</span>, however, the significance of Spearman's correlation between HOMA IR and serum leptin concentration weakened, the inverse correlation between HOMA IR and HDL-cholesterol remained approximately unchanged, and a positive correlation between HOMA IR and serum concentration of TNFα appeared.
Table 4 shows results of multiple regression analysis, when data from both control and hyperlipemic groups of each gender were judged together. HOMA IR was considered as a dependent variable and differently changed constellations of metabolic and other factors were taken as independent variables.
In <span class="yellow">men</span>, BMI and leptin seemed to play a main role in influencing the insulin resistance index HOMA IR, while TGL, ACL IgG and LDL-cholesterol didn't play any significant role (see left columns of Table 4). The decreasing of HDL-cholesterol concentration may also have some influence (see a significant drop of R2 after exclusion of this factor in Table 4A, 4B). But in the presence of leptin in the group of independent factors, the drop of R2 after exclusion of HDL-cholesterol from these factors was minimal (see Table 4C). On the other hand, after the exclusion of TNFα from the group of independent variables (see Table 4B, 4D) the value of R2 has unexpectedly risen, which could reflect the interference of TNFα with factors increasing the insulin resistance.
In <span class="yellow">women</span> (see right columns of Table 4), the maximal values of R2 were achieved with combination of independent variables containing TGL, leptin and HDL-cholesterol (about 60% influence on HOMA IR! – see Table 4A, 4B, 4C). TNFα seemed to play quite a different role than in <span class="yellow">men</span>: after exclusion of this factor from the group of independent factors R2significantly decreased (see Table 4B, 4D). In contrast to <span class="yellow">men</span>, the role of BMI seemed to be minimal. As in <span class="yellow">men</span>, the role of ACL IgG and LDL-cholesterol in influencing HOMA IR was negligible, but in contrast to <span class="yellow">men</span>, hFABP might play a certain role in this process (see Table 4D).
Generally, the insulin resistance (represented by HOMA IR) was in <span class="yellow">men</span> much less influenced by metabolic variables than in <span class="yellow">women</span>; while in <span class="yellow">women</span> in some combinations of dependent variables R2 reached 64 %, in <span class="yellow">men</span> the maximal value of R2 was only 39 %.<br><br>Discussion
In our previous paper [36], the mean value of HOMA IR in healthy subjects of both genders and of age comparable with our controls was 1.57 ± 0.87, and the mean value of index QUICKI 0.366 ± 0.029, respectively. These values, as well as the 95 % confidence limits, correspond to values found in controls in this study.
In accordance with many previous papers, serum concentrations of leptin in <span class="yellow">women</span> (both control and hyperlipemic) were substantially higher than in <span class="yellow">men</span>. In the control group of <span class="yellow">women</span> the correlation between leptin and HOMA IR was highly significant. However, in hyperlipemic <span class="yellow">women</span> the significance of this correlation lessened, because HOMA IR increased considerably (and significantly) but serum concentration of leptin only slightly (insignificantly). In <span class="yellow">men</span> the significance of correlations between serum leptin and HOMA IR was high and approximately the same in both the control and hyperlipemic groups, because the values of HOMA IR as well as serum leptin have nearly doubled in hyperlipemic in relation to control groups. In non-hyperlipemic postmenopausal <span class="yellow">women</span> the high concentration of serum leptin was not associated with higher insulin resistance: HOMA IR did not differ substantially from <span class="yellow">men</span>. A significant increase of insulin resistance in hyperlipemic <span class="yellow">women</span> was associated by only slight and insignificant increase of leptin concentration. According to Spearman's correlations, an increase of serum TNFα and/or a decrease of HDL-cholesterol might also play a distinct role in this respect. (see Table 3). In contrast to <span class="yellow">women</span>, in hyperlipemic <span class="yellow">men</span> the increase of insulin resistance index was approximately proportional with the increase of leptin concentration.
Multiple regression analysis affirmed the importance of leptin serum in increasing of insulin resistance in both genders. In <span class="yellow">men</span>, only BMI and HDL-cholesterol from other factors studied seemed to play a certain role, but the maximal values of influencing HOMA IR reached only 39%, with leptin and BMI being the more important factors. On the other hand, in <span class="yellow">women</span> the maximal determination of HOMA IR as high as 60% was registered in combination of serum leptin, TGL and decreased HDL-cholesterol as independent factors; the role of BMI was insignificant.
It is not known how leptin is regulated. A strong correlation between plasma leptin and fasting insulin undoubtedly exists, but hyperleptinemia in both obese and lean <span class="blue">humans</span> is not likely the result of hyperinsulinemia [37]. A relationship between leptin and insulin dependent on sex or BMI was reported, but relationship between triglyceride concentrations and leptin was independent of sex, BMI, and insulin [18,24]. Hyperleptinemia, as an early sign of obesity, was closely linked to subcutaneous fat mass [39,40]. Percentage of body fat has been shown to be the strongest predictor of leptin levels even in lean <span class="yellow">women</span> [41]. Leptin was highly correlated with percentage of body fat and with fat mass in adults irrespective of gender and age; however, the mean determinant of leptin plasma concentration in <span class="yellow">men</span> and postmenopausal <span class="yellow">women</span> was BMI, while in premenopausal <span class="yellow">women</span> it was only the fat mass [42]. These findings contrast with our results showing minimal influence of BMI on HOMA IR in postmenopausal <span class="yellow">women</span>.
All factors mentioned are connected with fat tissue: leptin and TNFα are directly produced chiefly by adipocytes, BMI growth is obviously accompanied by fat mass increase, and the typical hypertriglyceridemia associated with a decrease of HDL-cholesterol goes along with obesity and fat mass growth. The gender differences in circulating leptin were best explained by percentage of body fat and – inversely – by lean body mass [25]. In both genders the intra-abdominal fat correlated with insulin resistance, while the subcutaneous fat correlated with circulating leptin [11,12]. In <span class="yellow">men</span> obesity led to a prevalent increase of intra-abdominal fat, while in <span class="yellow">women</span> of subcutaneous fat [13]. Influences of different compartments of adipose tissues could elucidate the variability of correlations between insulin resistance and high leptin concentrations in lean and obese subjects of both genders [43]. In our non-hyperlipemic postmenopausal <span class="yellow">women</span> the content of subcutaneous fat mass might be higher than in non-hyperlipemic <span class="yellow">men</span> of appropriate age, which indicated a higher serum concentration of leptin. However, the insulin resistance – related to intra-abdominal fat mass – did not differ from <span class="yellow">men</span>. The significant increase of insulin resistance and leptin concentration in hyperlipemic <span class="yellow">men</span> might reflect the growing content of both subcutaneous and intra-abdominal fat mass (see the significant increase of BMI). In hyperlipemic <span class="yellow">women</span> the significant increase of insulin resistance accompanying only minimal insignificant increase of leptin could be caused by prevalent growing of intra-abdominal fat mass.
In elderly postmenopausal <span class="yellow">women</span>, an association between leptin and plasma lipoprotein concentration was found which depended on adiposity [17], and inverse correlations between serum leptin and HDL-cholesterol were described [44]. In our study, insulin resistance in <span class="yellow">women</span> seemed to be more notably than in <span class="yellow">men</span> influenced by lipid disorders, i.e. positively by serum triglycerides and inversely by HDL-cholesterol. These findings might be important in considering the concept of treatment of insulin resistance-related disorders in postmenopausal <span class="yellow">women</span>.
The significant role of TNFα in insulin resistance, caused by inhibiting the transduction of insulin signaling and by down-regulation of glucose transporter GLUT-4 and insulin receptor substrate-1, has been repeatedly confirmed [45-48]. Our results supported these findings unambiguously only in <span class="yellow">women</span>, while in <span class="yellow">men</span> TNFα seemed paradoxically to interfere with other factors – mainly BMI and leptin – in influencing insulin resistance, thus playing a quite different role. Previously it was found [46] that correlation between serum TNFα on the one side, and insulin, HOMA IR, serum triglycerids, respectively, on the other side, was substantially more significant in <span class="yellow">women</span> than in <span class="yellow">men</span>. Serum concentration of TNFα in <span class="yellow">patients</span> with type 2 diabetes of both genders correlated only with the quantity of intra-abdominal fat compartment [50]. Visceral obesity correlated with plasmatic aldosterone and with insulin resistance only in premenopausal <span class="yellow">women</span>, but not in <span class="yellow">men</span> [51].
From all these data we might support our above mentioned conclusion – that rising of insulin resistance in hyperlipemic <span class="yellow">women</span> was associated with an increase of intra-abdominal fat, because this fat mass in particular is a source of TNFα, which interfered with insulin sensitivity only in <span class="yellow">women</span>. We came to this conclusion irrespective of the finding that the increase of serum TNFα in hyperlipemic <span class="yellow">women</span> was statistically insignificant; results of Spearman's correlation (Table 3) and multiple regression analysis confirm a distinct role of this factor. In hyperlipemic <span class="yellow">men</span> not only the serum concentration of TNFα has decreased instead of increasing, but according to multiple regression analysis it played a quite different role in influencing insulin sensitivity, interfering with factors that determined insulin resistance (leptin and BMI).
In the control group of <span class="yellow">men</span> IgG anticardiolipin was inversely correlated to HOMA IR. The significance of this finding is not clear. These antibodies indicate vascular and thrombotic complications and oxidative modification of lipoproteins [52,53] and may represent an increased risk of atherogenic and inflammatory complications. In this case, however, their growing might be connected with an increase in insulin sensitivity. Anyway, ACL IgG evidently did not participate significantly in influencing the increase of insulin resistance associated with hyperlipidemia, although other anti-cardiolipin correlations could be masked by the relatively large inter-individual variations in this parameter.
Neither serum concentration of hFABP, a factor ensuring transmembrane transport and oxidative metabolisation of long-chain fatty acids [54,55], was significantly changed in hyperlipemic and insulin resistant subjects of both genders. This factor was very weakly associated only with HOMA IR in <span class="yellow">women</span> (see Table 4D), indicating that enhanced metabolisation of fatty acids in cells might to some degree contribute to insulin resistance.<br><br>Conclusions
In postmenopausal <span class="yellow">women</span> as well as in <span class="yellow">men</span> of approximately equal age serum leptin plays a significant role as an important determinant of insulin resistance. In addition to this factor, in <span class="yellow">women</span> the grade of insulin resistance is very considerably influenced by serum triglycerides, tumor necrosis factor alpha, and by decreased concentration of HDL-cholesterol, while in <span class="yellow">men</span> only a mild influence of BMI and decreased HDL-cholesterol is observed. These findings are explained as a consequence of gender-related differences in adipose tissue composition and/or distribution in both normal-weight and over-weight subjects and should be taken into account in treatment of <span class="yellow">patients</span> with metabolic risk factors of cardiovascular diseases.<br><br>List of abbreviations
HDL-Cholesterol = high-density cholesterol
LDL-cholesterol = low-density cholesterol
HOMA IR = Homeostasis Assessment of Insulin Resistance
= fasting insulin (μU/ml) * fasting glucose (mmol/l) / 22,5
QUICKI = 1 / [log fasting insulin (μU/ml) + log fasting glucose (mg/100 ml) ]
TNFα = tumor necrosis factor alpha
BMI = body mass index
R2 = coefficient of determination
hFABP = heart fatty acid binding protein
ACL-IgG = IgG fraction of anticardiolipin
TGL = triglycerides
GLUT-4 = glucose transporter-4
PPARγ = Peroxisome Proliferator-Associated Receptor gamma
CV = coefficient of variation<br><br>Authors' contributions
Dr. Radka Lichnovská collected the clinical material, performed analysis of biochemical values and edited the manuscript.
Dr. Simona Gwozdziewiczová performed analysis of clinical and biochemical data and edited the manuscript.
Prof. Jirí Hrebícek initiated the study, participated in its design and coordination, and wrote and edited the manuscript.<br><br>
<h3>pmcA1634875</h3>Molecular polymorphism, differentiation and introgression in the period gene between <span class="yellow">Lutzomyia intermedia</span> and <span class="yellow">Lutzomyia whitmani</span>
Abstract
Background
<span class="yellow">Lutzomyia intermedia</span> and <span class="yellow">Lutzomyia whitmani</span> (Diptera: Psychodidae) are important and very closely related vector species of cutaneous leishmaniasis in Brazil, which are distinguishable by a few morphological differences. There is evidence of mitochondrial introgression between the two species but it is not clear whether gene flow also occurs in nuclear genes.<br><br>Results
We analyzed the molecular variation within the clock gene period (per) of these two species in five different localities in Eastern Brazil. AMOVA and Fst estimates showed no evidence for geographical differentiation within species. On the other hand, the values were highly significant for both analyses between species. The two species show no fixed differences and a higher number of shared polymorphisms compared to exclusive mutations. In addition, some haplotypes that are "typical" of one species were found in some individuals of the other species suggesting either the persistence of old polymorphisms or the occurrence of introgression. Two tests of gene flow, one based on linkage disequilibrium and a MCMC analysis based on coalescence, suggest that the two species might be exchanging alleles at the per locus.<br><br>Conclusion
Introgression might be occurring between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> in period, a gene controlling behavioral rhythms in <span class="yellow">Drosophila</span>. This result raises the question of whether similar phenomena are occurring at other loci controlling important aspects of behavior and vectorial capacity.<br><br><br><br>Background
The Phlebotominae sand <span class="yellow">flies</span> <span class="yellow">Lutzomyia intermedia</span> Lutz & Neiva 1912 and <span class="yellow">Lutzomyia whitmani</span> Antunes & Coutinho 1912 are vectors of cutaneous leishmaniasis in Brazil. These are closely related species that can be only distinguished by a few morphological differences [1] and both show high anthropophily and reported natural infections with Leishmania in different regions of Brazil [2].
Despite their importance as vectors, only a handful of studies have been carried out in these two species using molecular techniques [3-6]. One of the most important findings from an epidemiological perspective is the evidence obtained for introgression between the two species using mitochondrial DNA [4]. This was particularly interesting because apparently, only lineages of <span class="yellow">L. whitmani</span> sympatric with <span class="yellow">L. intermedia</span> have been involved in cutaneous leishmaniasis transmission in the peridomestic environment [4], which suggests that genes controlling aspects of vectorial capacity could be passing from one species to the other. In fact, mitochondrial introgression has been reported in other sand <span class="yellow">fly</span> species [7,8] suggesting that might be a common phenomenon in these insect vectors. However, because mitochondrial genes can introgress relatively easily between closely related species [9], it becomes important to examine whether introgression can occur with nuclear genes.
The <span class="yellow">Drosophila</span> period (per) gene homologue was isolated in sand <span class="yellow">flies</span> by Peixoto et al. [10]. This circadian clock gene was originally identified using mutagenesis by Konopka and Benzer [11], but is also known to control the differences in the "lovesong" rhythms between <span class="yellow">D. melanogaster</span> and <span class="yellow">D. simulans</span> [12], that are important to the sexual isolation between these two species [13-15]. In addition, per was implicated in the control of species-specific circadian mating rhythms in <span class="yellow">Drosophila</span> and Bractocera, which might also constitute a reproductive isolation mechanism [16-18]. Thus per may possibly represent an example of a <span class="yellow">Drosophila</span> speciation gene [19], and in fact it has been used as a molecular marker in a number of speciation and evolutionary studies, not only in <span class="yellow">Drosophila</span> (reviewed in [20]) but also in other insects (e.g. [21]) including sand <span class="yellow">flies</span> [22-24].
Because per controls the circadian clock in different insects [25], it is almost certainly involved in the rhythms of activity and biting of sand <span class="yellow">flies</span> [26], which are very important to leishmaniasis transmission. In addition, per might be involved in reproductive isolation in sand <span class="yellow">flies</span>, via mating rhythms, or via their "lovesongs" [2,27]. per is thus a particularly interesting marker, among the few available, for an introgression analysis in <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. Evidence for introgression in per might suggest that gene flow between these two vector species is occurring at other genes controlling important aspects of behavior and vectorial capacity. It might also suggest that per does not have a strong role in their reproductive isolation. In the current study, we analyzed the molecular variation within the per gene of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> in five different localities in Eastern Brazil.<br><br>Results
Polymorphism and divergence between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>
A total of 68 sequences from <span class="yellow">L. intermedia</span> and 53 from <span class="yellow">L. whitmani</span> homologue to a fragment of the period gene were analyzed from populations of five localities in Eastern Brazil (Fig 1). The alignment of 72 variable sites is shown in Fig 2. Although most of the changes are either synonymous or occur within the 58 bp intron, non-synonymous substitutions are observed causing 9 amino acid differences among the sequences (Fig 2).
Table 1 shows the number of sequences of each population of the two species, the number of polymorphic sites (S) and the estimates of molecular polymorphism θ (based on the total number of mutations) and π. Table 1 also shows the Tajima's [28] and Fu & Li's [29] statistics. Within each species, all populations present similar levels of polymorphism with the exception of <span class="yellow">L. whitmani</span> from Ilhéus, which seems to be less polymorphic than the others. This population was also the only one presenting a significant value in the Fu & Li test but only at the 5% level. Finally, the last column of Table 1 presents the recombination estimator γ [30] indicating that both species show evidence of intragenic recombination in the per gene.
To investigate the level of intra and interspecific differences, initially an AMOVA was carried out as shown in Table 2. The results show a non-significant within species and a significant between species molecular variation at the per locus. Table 3 shows a more detailed analysis of the intraspecific differentiation among populations of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. None of the pairwise and overall fixation indexes (Fst) are significant in the case of <span class="yellow">L. intermedia</span> and only one (Posse × Ilhéus) has a borderline significant value in <span class="yellow">L. whitmani</span>. The results therefore show that no significant geographical heterogeneity was detected among the populations of the two species. The estimated number of migrants per generation, based on the overall Fst values, is 20.683 for <span class="yellow">L. intermedia</span> and 23.125 for <span class="yellow">L. whitmani</span>.
Table 4 shows measures for DNA divergence between species (Dxy and Da), as well as the Fst and Nm values considering each species as a unique population. Dxy is the average number of nucleotide substitutions per site between alleles from two different populations and Da is the number of net nucleotide substitutions between two populations. Table 4 also shows the number of polymorphisms exclusive for each species (Sint and Swhit), the number of shared polymorphisms (Ss) and the number of fixed differences (Sf) between species. As one can note, there is a high number of shared polymorphisms between species, and no fixed differences between them suggesting either the persistence of ancestral polymorphisms or the occurrence of introgression. In fact, there is one shared haplotype between the two species (IPO13, WPO10 and WPO19) and three <span class="yellow">L. whitmani</span> sequences (WAC02, WPO13 and WPO14) which show only one nucleotide difference to "typical" <span class="yellow">L. intermedia</span> haplotypes (see also below).<br><br>Genealogy of period sequences
A phylogenetic analysis of the period gene sequences from <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> was carried out with the Minimum Evolution method using the Kimura 2-parameter distance (Fig 3). A sequence from <span class="yellow">L. umbratilis</span>, a related species from the same subgenus Nyssomyia, was used as outgroup [24]. The tree shows <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> as non-monophyletic. However, despite the low bootstrap values, which are below 50% in most cases, there is a large group that contains most <span class="yellow">L. intermedia</span> sequences and a second large group with most <span class="yellow">L. whitmani</span> sequences. A few other sequences are clustered outside these two main groups. It is interesting to note that there are three <span class="yellow">L. whitmani</span> alleles (WAC2, WPO13 and WPO14) inside <span class="yellow">L. intermedia</span> main group, as well as one <span class="yellow">L. intermedia</span> allele (ICP16) inside the <span class="yellow">L. whitmani</span> main group. In addition, a second <span class="yellow">L. intermedia</span> allele (IPO13) is a shared haplotype between the two species as mentioned above. Again, the results suggest either the persistence of ancestral polymorphisms or the occurrence of introgression between the two species. Very similar results were obtained using the maximum likelihood algorithm as implemented in PAUP 4.0b10 software [31] (data not shown).
As mentioned before, there is evidence of intragenic recombination in the per gene fragment of both species (see Table 1) and for that reason the bifurcating tree shown in Fig 3 has to be viewed with caution, as different regions of the gene might have different phylogenetic histories [32]. Therefore, we constructed Minimum Evolution trees with the two most polymorphic non-recombining blocks of the per gene fragment identified using the Hudson and Kaplan [33] method available in the DNAsp 4.1 program [34]. We did not observed major changes in the genealogy of the <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> per sequences, especially regarding the five haplotypes (ICP16, IPO13, WAC2, WPO13 and WPO14) that clearly cluster with sequences of the other species (data not shown).
Finally, a haplotype network was estimated from per sequences using statistical parsimony, as described by Templeton et al. [35] and implemented in the TCS1.21 software [36] (Fig 4). A small number of ambiguities were resolved as suggested by Crandall and Templeton [37]. The haplotype network shows connections between sequences from each species, separating most of the sequences of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> in two groups. No intraspecific geographical structuring was found. Once again, some of the <span class="yellow">L. whitmani</span> sequences (WAC2, WAC10, WPO13 and WPO14) appear more closely related to <span class="yellow">L. intermedia</span> haplotypes. In addition, one <span class="yellow">L. intermedia</span> allele (ICP16) is connected by a small number of mutations to some of the main <span class="yellow">L. whitmani</span> haplotypes and IPO13 is a shared haplotype between the two species. These results confirm the same putative introgressed sequences indicated by the phylogenetic reconstructions.<br><br>LD test of introgression
We tested the hypothesis of gene flow between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> using a method based on linkage disequilibrium (LD) developed by Machado et al. [38]. In this test, x is the difference between the average LD found among all pairs of shared polymorphisms (DSS) between the two species and the average LD among all pairs of sites for which one member is a shared polymorphism and the other is an exclusive polymorphism (DSX). In case of gene flow x should tend to be positive [see [38] for more details].
Because of limitations on the total number of sequences that could be handled by the WH program we could not perform the simulations with all sequences. Therefore, we carried out the LD test of introgression between each pair of sympatric populations of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> from the localities of Posse, Afonso Claudio and Corte de Pedra. The input files were prepared using the values of recombination and linkage disequilibrium calculated by the SITES program [30] for each population (data not shown). Although no significant values were found for the smaller samples of Afonso Claudio and Corte de Pedra, the results (Table 5) present evidence for introgression in the period gene in both directions (from <span class="yellow">L. intermedia</span> to <span class="yellow">L. whitmani</span> and vice-versa) in the locality of Posse.<br><br>Isolation with Migration model
To further examine the gene flow between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> we used the IM software [39]. The Isolation with Migration model has six demographic parameters that include two migration rates, one for each population. The IM software estimates the posterior probability for each of the model parameters, fitting the Isolation with Migration model to the data. One of the assumptions of this model is that the loci studied do not have internal recombination. Therefore, we identified four different non-recombining blocks of our fragment of per, which were then treated as different loci in the analysis. The four-gametes test [33] implemented in DnaSP4.1 was used for the identification of possible recombination events. Since the program estimates parameters for a pair of closely related populations or species, all sequences of each species were used in the analysis as a single population. We performed MCMC runs using the IM software with different seed numbers, in order to guarantee convergence of the sample.
Maximum likelihood estimates of migration parameters revealed a non-zero value for both species, m1 = 1.398 and m2 = 1.014 (m1 – from <span class="yellow">L. whitmani</span> towards <span class="yellow">L. intermedia</span>; m2 – from <span class="yellow">L. whitmani</span> towards <span class="yellow">L. intermedia</span>). Fig 5 shows the posterior distributions for migration rates and reveals a null probability for the absence of migration from <span class="yellow">L. whitmani</span> towards <span class="yellow">L. intermedia</span>. In addition, the absence of migration in the opposite direction is not included in the 95% confidence interval (values range from 0.222 to 8.898), thus supporting the presence of migration in both directions. The conversion of the migration rate estimate to population migration rate per generation (m1 and m2) is not accurate when the population size is based on a single locus. However, the average of the migrant number per generation for both species was very close to the Nm estimate based on Fst values (Nm ~0.49 in Table 4, m1 ~0.52 and m2 ~0.34).<br><br>
Discussion
There is some evidence that <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> might represent sibling-species complexes in Brazil. <span class="yellow">Lutzomyia neivai</span> Pinto 1926, a sibling of <span class="yellow">L. intermedia</span> is found in parts of Southern and Western Brazil and some other countries of South America [40]. The present study did not include populations of this species. In the case of <span class="yellow">L. whitmani</span>, mitochondrial data [3,6] indicates three main lineages in Brazil: an Amazonian group, a North-South group and a Northeast group. We did not find strong evidence of a geographical differentiation in the period gene among populations of <span class="yellow">L. whitmani</span> although one of the pairwise Fst comparisons (Posse × Ilhéus) was significant at the 5% level.
When we compare <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>, we find a highly significant Fst value (0.3373), which is however smaller than that observed for the period gene between sympatric siblings of <span class="yellow">Lutzomyia longipalpis</span> (Fst = 0.3952) [23], a complex of cryptic species that are vectors of American visceral leishmaniasis. Therefore, despite the presence of diagnostic morphological characters to identify <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> [1] the level of molecular divergence in period is not as high as the cryptic <span class="yellow">L. longipalpis</span> siblings.
Even though it is hard to distinguish introgression from the persistence of ancestral polymorphisms, a test of gene flow based on the signature introgression leaves on the patterns of linkage disequilibrium [38] as well as simulations that fit the "Isolation with Migration" model to the data suggest that <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> might be exchanging alleles at the per locus. This is further supported by the presence of shared haplotypes between the two species in Posse and very similar sequences in all sympatric populations. There is mounting evidence that introgression plays a major role in the evolution of closely related insect vector species. Introgression among vectors may have important epidemiological consequences. Gene flow in loci that affect vectorial capacity, such as those controlling host preference and susceptibility to parasite infection, can change the transmission patterns and consequently make the disease control a harder task. Introgression of genes that control adaptation to particular types of environment can also have a major impact on the spread of vector-borne diseases as was proposed for the major African malaria vector <span class="yellow">Anopheles gambiae</span> [41]. The same can be said about genes controlling insecticide resistance. For example, Weill et al. [42] found a kdr mutation responsible for pyrethroid resistance in the Mopti form of <span class="yellow">Anopheles gambiae</span>, a normally susceptible taxon of this species complex. Sequence analysis reveals that this resistant allele probably originates through introgression from the Savanna form.
Although <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> are closely related and only distinguished by a few morphological differences, they do show differentiation in some other important traits. For example, in Posse, one of the localities we studied, the two species show differences in abundance during the year. <span class="yellow">L. intermedia</span> is more abundant in the summer while <span class="yellow">L. whitmani</span> is more frequent in the winter months [2]. They also show differences in microhabitat preferences, <span class="yellow">L. intermedia</span> being more common in the peridomestic area while <span class="yellow">L. whitmani</span> is found mainly in the surrounding forest [2]. In addition, the two species show marked differences in their tendencies to bite <span class="blue">humans</span> in the early morning, with <span class="yellow">L. whitmani</span> showing higher feeding rates than <span class="yellow">L. intermedia</span> [26]. Therefore, despite the evidence of introgression in the period gene in this locality, there are important ecological and behavioral differences between the two species in Posse suggesting that gene flow is probably rather limited in loci controlling these traits. Hence, it is yet not clear whether introgression has played an important role in the evolution of <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. Further work with other genes might help clarify the issue.<br><br>Conclusion
Evidence for introgression between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> obtained using mitochondrial DNA [4] seems to be corroborated by our data on the period gene, a nuclear marker. Nevertheless, considering that period is potentially involved in reproductive isolation and might be, therefore, less prone to introgression than the "average" gene [43], it is possible that much higher levels of gene flow between the two species occur at other genes. It might, on the other hand, suggest that this behavioral gene, or at least the fragment we analyzed, did not play a role in speciation between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span>. In fact the same has been suggested for some <span class="yellow">Drosophila</span> species [44] despite per's role controlling lovesong and mating rhythm differences between <span class="yellow">D. melanogaster</span> and <span class="yellow">D. simulans</span> [13-16].
Although the evidence for introgression in the per gene between <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> is not overwhelming, it does indicate the need to extend this analysis to other loci in the future. We are currently isolating new molecular markers in the two species to carry out a multi-locus approach [39] that might help determining how much variation in gene flow and differentiation there is across the genome of these two very important leishmaniasis vectors.<br><br>Methods
Sand <span class="yellow">fly</span> samples
Sand <span class="yellow">fly</span> samples used in this work were all the F1 generation from wild collected females from the Brazilian localities of Posse (Petrópolis, Rio de Janeiro State, 22°30'S 43°10'W), Jacarepaguá (Rio de Janeiro, Rio de Janeiro State, 22°55'S 43°21'W), Afonso Claudio (Espírito Santo State, 20°04'S 41°07'W), Corte de Pedra (Presidente Tancredo Neves, Bahia State, 13°27'S 39°25'W) and Ilhéus (Bahia State, 14°50'S 39°06'W). <span class="yellow">L. intermedia</span> and <span class="yellow">L. whitmani</span> were identified according to Young and Duncan [1]. The progeny of each wild caught female was raised separately according to Souza et al. [45] and only one F1 male of each female was used for the molecular analysis, which included 68 individuals of <span class="yellow">L. intermedia</span> (12 from Afonso Claudio, 18 from Posse, 20 from Corte de Pedra and 18 from Jacarepaguá) and 51 individuals of <span class="yellow">L. whitmani</span> (12 from Afonso Claudio, 17 from Posse, 3 from Corte de Pedra and 19 from Ilhéus). Note that, although the distribution of the two species shows considerable overlap in Eastern Brazil, in many localities only one species is found or is far more abundant than the other. There are also seasonal and microhabitat differences in abundance between them in areas of sympatry [2].<br><br>DNA methods
Genomic DNA was prepared according to Jowett [46] with slight modifications and the PCR was carried out for 30 cycles at 95°C for 30 sec, 60°C for 30 sec and 72°C for 30 sec, using Abgene, Amersham Biosciences or Biotools reagents according to manufacturers directions. The per primer sequences are: 5llper2: 5'-AGCATCCTTTTGTAGCAAAC-3' (forward) and 3llper2: 5'-TCAGATGAACTCTTGCTGTC-3' (reverse). These primers amplify a 486 bp fragment of the sand <span class="yellow">fly</span> per gene homologue that includes part of the PAS/CLD domain, an intron (58 bp) and the beginning of the perS domain [24]. The amplified fragments were cloned using the pMOSBlue blunt ended cloning kit (Amersham Biosciences) and plasmid DNA preparation was carried out using the "Flexiprep" Kit (Amersham Biosciences). Cloned PCR fragments were sequenced at Fundação Oswaldo Cruz and at University of Leicester using ABI 377 sequencers. With the exception of two <span class="yellow">L. whitmani</span> individuals from Corte de Pedra (see below), only one sequence of each sand <span class="yellow">fly</span> (representing one of the two possible alleles) was used in the analysis but an average of three sequences per individual were obtained in order to check possible PCR induced mutations. In addition, PCR fragments were also sequenced directly in some cases for the same reason. In the case of the two <span class="yellow">L. whitmani</span> mentioned above 6 and 9 clones were sequenced respectively from specimens WCP01 and WCP03 to determine both alleles simply to increase the size of this small sample.
Negative controls were performed for all amplification reactions. In addition, PCR, cloning and sequencing were repeated for two individuals to confirm putative introgressed sequences and to exclude the possibility that they were the result of PCR contamination. Finally, for at least two individuals with putative introgressed sequences, we could define the other allele from additional clones (not included in the analysis), which showed to be typical of the species, indicating no identification problems.
The sequences were submitted to GenBank (accession numbers AY927062 to AY927182).<br><br>Sequence analyses
The preliminary sequence editing was carried out using the Wisconsin Package Version 9.1, Genetics Computer Group (GCG), Madison, and ClustalX [47] was used to perform the multiple alignment. Analyses of population polymorphisms and differentiation between populations were carried out using DNAsp4.1 [34] and ProSeq [48] softwares, while Arlequin v. 2.0 [49] was used for an analysis of molecular variance (AMOVA) between populations. The Minimum Evolution phylogenetic tree was constructed using MEGA 3.1 software [50]. The haplotype network was estimated using TCS1.21 [36]. Recombination and linkage disequilibrium analyses were performed using the DNAsp4.1 and SITES program [30]. Linkage disequilibrium simulations were carried out by the WH program [51,52] and Markov Chain Monte Carlo (MCMC) simulations of the isolation with migration model were performed using the algorithm implemented in the IM program [39].<br><br>
Authors' contributions
CJM generate and analyzed all the data and drafted the manuscript. NAS and CAC collected and maintained sand <span class="yellow">fly</span> samples. CPK helped to write the manuscript and supervised CJM during her stay in Leicester. AAP is the principal investigator, participated in its design and coordination, and helped to write the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1779441</h3>Identification of a <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts
Abstract
Increased bone resorption mediated by osteoclasts causes various diseases such as osteoporosis and bone erosion in rheumatoid arthritis (RA). Osteoclasts are derived from the monocyte/macrophage lineage, but the precise origin remains unclear. In the present study, we show that the purified CD16- <span class="yellow">human</span> peripheral blood monocyte subset, but not the CD16+ monocyte subset, differentiates into osteoclast by stimulation with receptor activator of NF-κB ligand (RANKL) in combination with macrophage colony-stimulating factor (M-CSF). Integrin-β3 mRNA and the integrin-αvβ3 heterodimer were only expressed on CD16- monocytes, when they were stimulated with RANKL + M-CSF. Downregulation of β3-subunit expression by small interfering RNA targeting β3 abrogated osteoclastogenesis from the CD16- monocyte subset. In contrast, the CD16+ monocyte subset expressed larger amounts of tumor necrosis factor alpha and IL-6 than the CD16- subset, which was further enhanced by RANKL stimulation. Examination of RA synovial tissue showed accumulation of both CD16+ and CD16- macrophages. Our results suggest that peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct responses to RANKL. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for osteoclastogenesis. Blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Introduction
Rheumatoid arthritis (RA) is an autoimmune disease characterized by chronic inflammation and proliferation of the synovium in multiple joints. A large number of inflammatory cells, including T cells, B cells, macrophages and dendritic cells, accumulate in the affected synovium, and these inflammatory cells, together with fibroblast-like synoviocytes, express various cytokines, such as tumor necrosis factor alpha (TNFα), IL-6 and receptor activator of NF-κB ligand (RANKL), which are known to induce differentiation and activation of osteoclasts. The inflammatory synovial tissue, known as pannus, invades the articular bone and causes focal bone erosion, which is the hallmark of RA. Histopathologically, osteoclasts are present at the interface of the pannus and bone. Interestingly, the deletion of RANKL or c-Fos gene, which is important for osteoclastogenesis, results in minimal bone destruction in <span class="yellow">mouse</span> models of arthritis [1,2]. Furthermore, other studies indicated that inhibition of osteoclastogenesis by osteoprotegerin, a decoy receptor for RANKL, limits bone destruction in experimental models of arthritis. These studies suggest that osteoclasts are involved in focal bone erosion in RA [3].
Osteoclasts are derived from the monocyte/macrophage lineage. It is reported that osteoclast precursors reside in <span class="yellow">human</span> peripheral blood monocytes [4,5]. A marked increase of the circulating osteoclast precursors was demonstrated in <span class="yellow">patients</span> with erosive psoriatic arthritis as well as in arthritic TNFα transgenic <span class="yellow">mice</span> [6,7]. It was also shown that peripheral monocytes differentiate into osteoclasts when seeded on RANKL/osteoclast differentiation factor-producing RA synovial fibroblasts [8]. In addition, RA synovial macrophages thought to originate from peripheral blood monocytes were shown to differentiate into osteoclasts [9,10]. Monocytes are therefore involved not only in synovial inflammation, but also in bone remodeling as potential precursors for synovial macrophages and osteoclasts.
<span class="yellow">Human</span> peripheral blood monocytes consist of two major subsets, CD16+ and CD16-, comprising 5–10% and 90–95% of the monocytes, respectively. These two subsets exhibit different chemotaxis activities and potential of cytokine production [11,12]. Moreover, activation of the Toll-like receptor induces distinct subsets, CD1b+ dendritic cells and DC-SIGN+ (dendritic cell-specific C-type lectin ICAM-3-grabbing nonintegrin) macrophages from CD16+ and CD16- monocytes, respectively [13]. It has not been revealed, however, which monocyte subset develops into osteoclasts.
In the present study, we determined the <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts, and revealed that each subset exhibits a different response for osteoclastogenic stimuli.<br><br>Materials and methods
Purification of peripheral blood monocytes
Peripheral blood monocytes from healthy donors were collected using Ficoll-Conray (Imuuno-Biological Laboratories, Gunma, Japan) gradient centrifugation. Negative selection of monocytes was performed using MACS microbeads (Miltenyi Biotec, Auburn, CA, USA) according to the protocol supplied by the manufacturer.
The purified monocytes were separated into two subsets, CD16+ and CD16- monocytes, using CD16 MicroBeads (Miltenyi Biotec). Flow cytometry analysis using FITC-conjugated <span class="yellow">mouse</span> anti-CD14 mAb (MY4; Bechman Coulter, Fullerton, CA, USA) and phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8; BD Biosciences, San Jose, CA, USA) showed that the purities of the CD16+ and CD16- monocytes were more than 90% and 92%, respectively.
For the other experiment, monocytes were purified using CD14 MicroBeads (Miltenyi Biotec), and then stained either with FITC-conjugated <span class="yellow">mouse</span> anti-CD33 mAb (MY9; Bechman Coulter) or phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8). Cell sorting of the stained cells was performed using a FACS Vantage cytometer (BD Biosciences) or a MoFlo cell sorter (Dako, Glostrup, Denmark).<br><br>Osteoclast differentiation
Purified CD16+ and CD16- monocytes (5 × 104 cells/well) were incubated in 96-well plates in αMEM (Sigma, St Louis, MO, USA) with heat-inactivated 10% fetal <span class="yellow">bovine</span> serum (FBS) (Sigma) or with Ultra-Low IgG FBS (IgG < 5 μg/ml; Invitrogen, Carlsbad, CA, USA), and where indicated with M-CSF + RANKL (Peprotech, Rocky Hill, NJ, USA).
For the other experiments, varied numbers of CD16+ monocytes (1 × 103, 2.5 × 103, 5 × 103) were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured in 96-well plates in αMEM with heat-inactivated 10% FBS. The medium was replaced with fresh medium 3 days later, and after incubation for 7 days the cells were stained for tartrate-resistant acid phosphatase (TRAP) expression using a commercial kit (Hokudo, Sapporo, Japan). The number of TRAP-positive multinucleated cells (MNC) in three randomly selected fields examined at 100× magnification or the total number of TRAP-positive MNC per well was counted under light microscopy.<br><br>Resorption assay
Monocytes were seeded onto plates coated with calcium phosphate thin films (Biocoat Osteologic; BD Biosciences) and were incubated with RANKL (40 ng/ml) + M-CSF (25 ng/ml) for 7 days. The cells were then lysed in bleach solution (6% NaOCl, 5.2% NaCl). The resorption lacunae were examined under light microscopy.<br><br>Enzyme-linked immunosorbent assay
Purified monocytes were cultured in 96-well plates where indicated either with RANKL or M-CSF for 24 hours. Concentrations of TNFα and IL-6 in the culture supernatant were measured with an ELISA kit (BioSourse International, Camarillo, CA, USA). For experiments of matrix metalloproteinase (MMP)-9 and TRAP-5b, culture supernatants were collected on day 7 and the concentrations of these enzymes were measured using an MMP-9 ELISA kit (Amersham Biosciences, Piscataway, NJ, USA) or a TRAP-5b ELISA kit (Suomen, Turku, Finland).<br><br>Reverse transcriptase-polymerase chain reaction
Monocytes (1 × 106 cells/well) were cultured in six-well plates with M-CSF alone or with M-CSF + RANKL for 3 days. Total RNA was extracted using RNeasy Micro (Qiagen, Valencia, CA, USA). The RNA was then treated with DNase I (Qiagen). The oligo(dT)-primed cDNA was synthesized using Superscript II reverse transcriptase (Invitrogen). The amount of cDNA for amplification was adjusted by the amount of RNA measured by an optical density meter and also by β-actin or GAPDH PCR products. One microliter of cDNA was amplified in a 50 μl final volume containing 25 pmol appropriate primer pair, 10 pmol each of the four deoxynucleotide triphosphates, and 5 units FastStart Taq DNA Polymerase (Roche, Manheim, Germany) in a thermal cycler (PTC-200; MJ GeneWorks, Waltham, MA, USA).
The PCR conditions were 25–40 cycles of denaturation (95°C for 30 s), annealing (60–62°C for 1 min) and extension (72°C for 1 min). The sequences of the primers are presented in Table 1. The PCR products were separated by electrophoresis through 2% agarose gel.<br><br>Western immunoblot analysis
Purified monocytes were cultured for 3 days in the presence of 40 ng/ml M-CSF with or without 25 ng/ml RANKL. Cells were lysed in RIPA Lysis buffer (upstate, Lake Placid, NY, USA) containing protease inhibitors (Roche) for 15 min at 4°C. A total of 20 μg protein was boiled in the presence of 6 × sodium dodecyl sulfate sample buffer, and was separated on 7.5% or 10% sodium dodecyl sulfate-polyacrylamide gel (ATTO, Tokyo, Japan). Proteins were then electrotransferred to a polyvinylidene fluoride microporous membrane (Millipore, Billerica, MA, USA) in a semidry system. Membranes were incubated in 10% skim milk prepared in phosphate-buffered saline (PBS) containing 0.1% Tween 20, and were subjected to immunoblotting. Antibodies used were <span class="yellow">goat</span> anti-RANK antibody (Techne Corporation, Minneapolis, MN, USA), <span class="yellow">goat</span> anti-c-fms antibody (R&D systems, Minneapolis, MN, USA), and <span class="yellow">mouse</span> anti-β-actin mAb (AC-15; Sigma). Peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">goat</span> IgG antibody (Dako) or peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">mouse</span> IgG antibody (Dako) was used as the second antibody. The signals were visualized by chemiluminescence reagent (ECL; Amersham Biocsiences, Little Chalfont, UK).<br><br>Cell surface expression of c-fms
The following mAbs were used for analysis of c-fms expression: Alexa 647-conjugated anti-CD14 mAb (UCHM1; Serotec, Oxford, UK), FITC-conjugated anti-CD16 mAb (3G8; Bechman Coulter) and phycoerythin-conjugated anti-c-fms mAb (61708; R&D systems). Alexa 647-conjugated <span class="yellow">mouse</span> IgG2a (Serotec), FITC-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) and phycoerythin-conjugated <span class="yellow">mouse</span> IgG1 (Bechman Coulter) were used as isotype controls. Peripheral blood monocytes (1 × 105 cells) were incubated with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and were then stained with three fluorochrome-labeled mAbs for 45 minutes on ice. The stained cells were analyzed with a FACS Calibur (BD Biosciences).<br><br>Immunofluorescent staining
Monocytes (8 × 104 cells/well) were allowed to adhere on 96-well plates overnight or were cultured with M-CSF and RANKL for 2–4 days. The cells were fixed in acetone and then stained with anti-αvβ3 mAb (LM609; Chemicon, Temecula, CA, USA) or <span class="yellow">mouse</span> IgG1 (11711; R&D Systems) as an isotype-matched control. Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes, Eugene, OR, USA) was used as the second antibody. TOTO-3 (Molecular Probes) was used for nuclear staining.<br><br>Flow cytometric analysis of p38 MAPK and ERK1/2 phosphorylation
Purified monocytes were cultured in the presence of 25 ng/ml M-CSF for 3 days, and were either left unstimulated or were stimulated with 40 ng/ml RANKL at 37°C. Stimulations were stopped by adding an equal volume of PhosFlow Fix Buffer I solution (BD Biosciences) to the cell culture. After incubation for 10 minutes at 37°C, the cells were permeabilized by washing twice at room temperature in PhosFlow Perm/Wash Buffer I (BD Biosciences). A total of 1 × 105 cells was then Fc blocked with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and was stained with Alexa Fluor 647-conjugated mAb either to phospho-p38 MAPK (T180/Y182) or to phospho-ERK1/2 (T202/Y204) (BD Biosciences) for 30 minutes at room temperature. Alexa Fluor 647-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) was used as an isotype control. The cells were washed in PhosFlow Perm/Wash Buffer I, and were analyzed by flow cytometry, as already described.<br><br>RNA interference
RNA oligonucleotides (iGENE, Tsukuba, Japan) were designed based on the algorithm that incorporates single nucleotide polymorphism and homology screening to ensure a target-specific RNA interference effect. The following sense and antisense oligonucleotides were used: integrin β3, 5'-GCU UCA AUG AGG AAG UGA AGA AGC A-AG and 3'-UA-CGA AGU UAC UCC UUC ACU UCU UCG U; randomized control, 5'-CGA UUC GCU AGA CCG GCU UCA UUG C-AG and 3'-UA-GCU AAG CGA UCU GGC CGA AGU AAC G; and lamin, 5'-GAG GAA CUG GAC UUC CAG AAG AAC A-AG and 3'-UA-CUC CUU GAC CUG AAG GUC UUC UUG U.
CD16- monocytes (8 × 104 cells/well) were incubated in 96-well plates in optimem (Invitrogen). After 1 hour, siRNAs were transfected into the cells using oligofectamine (Qiagen) based on the method recommended by the manufacturer. After 2 hours, the cells were washed once with PBS, followed by the addition of αMEM supplemented with 10% FBS, M-CSF and RANKL. After a 2-day incubation, the β3 mRNA expression was analyzed by RT-PCR with different PCR cycles, as described earlier. Immunofluorescent staining for the αvβ3 heterodimer was also performed as described above, and numbers of αvβ3-positive cells were counted in randomly selected three fields at 100× magnification. Seven days after the transfection of siRNAs, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Inhibition of osteoclastogenesis with cyclic RGDfV peptide
CD16- monocytes were incubated in 96-well plates with M-CSF + RANKL for 2 days. A medium containing either cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) (Calbiochem, San Diego, CA, USA) or dimethyl sulfoxide was then added. After incubation for a further 5 days, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Immunohistochemistry
Synovial tissue samples were obtained during total knee joint replacement surgery from four RA <span class="yellow">patients</span>. Signed consent forms were obtained before the operation. The experimental protocol was approved by the ethics committee of the Tokyo Medical and Dental University. RA was diagnosed according to the American College of Rheumatology criteria [14].
Double immunofluorescent staining for CD68 and CD16 antigens was conducted on optimal cutting temperature-embedded sections of frozen synovial samples. Eight-micrometer-thick cryostat sections of RA synovium were fixed in acetone for 3 minutes and were then rehydrated in PBS for 5 minutes. The samples were incubated in 5 μg/ml proteinase K (Roche), 50 mM ethylenediamine tetraacetic acid, 100 mM Tris–HCl, pH 8.0, for 15 minutes at room temperature followed by a wash in PBS. The samples were then blocked with 10% <span class="yellow">goat</span> serum in PBS for 60 minutes at room temperature, and were incubated with anti-CD16 mAb (3G8; Immunotech, Marseille, France) or <span class="yellow">mouse</span> IgG1 (11711) as an isotype-matched control in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then washed three times in PBS, for 5 minutes each, and incubated with Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes) in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then sequentially stained for CD68 antigen in a manner similar to that used for CD16 staining. The samples were stained with anti-CD68 mAb (PGM1; Immunotech) or <span class="yellow">mouse</span> IgG3 (6A3; MBL, Nagoya, Japan) followed by labeling with Alexa fluor488-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG3 antibody (Molecular Probes). The samples were examined by confocal laser scanning microscope (Olympus, Tokyo, Japan).<br><br>Statistical analysis
Data are expressed as the mean ± standard error of the mean. A nonpaired Student's t test was used for comparison, using the StatView program (Abacus Concepts, Berkeley, CA, USA). P < 0.05 was considered statistically significant.<br><br>
Results
Induction of osteoclasts from CD16- peripheral blood monocytes
To identify the monocyte subset that differentiates into osteoclasts, we examined osteoclast formation from CD16+ and CD16- <span class="yellow">human</span> peripheral blood monocytes. The monocyte subsets were purified using magnetic beads. Incubation with M-CSF alone did not induce osteoclast formation from either subset (Figure 1a). Culture with M-CSF + RANKL induced a significant number of TRAP-positive MNC from the CD16- subset, whereas only few CD16+ monocytes differentiated into TRAP-positive MNC (Figure 1a,b). We then assessed the bone resorptive ability by culturing cells on calcium phosphate-coated plates with M-CSF + RANKL. Resorption lacunae were detected only in the CD16- subset (Figure 1c), indicating the TRAP-positive CD16--derived MNC possessed the osteoclast phenotype.
Similar results were obtained using purified monocytes by FACS sorting (purities: CD16+, 96%; CD16-, 97%). The number of TRAP-positive MNC induced were 36 ± 3 cells/well and 348 ± 13 cells/well from CD16+ and CD16- monocytes, respectively. To exclude the possibility that the anti-CD16 antibody used for isolation of CD16+ monocytes inhibits osteoclast formation, we separated the two subsets, CD33low monocytes and CD33high monocytes, using anti-CD33 mAb and a fluorescent cell sorter, since it was reported that CD33low monocytes correspond to CD16+, and that CD33high correspond to CD16- monocytes [15]. On average, the CD33low population contained CD16- (10.2%)/CD16+ (89.8%) monocytes, and the CD33high population contained CD16- (86.3%)/CD16+ (13.7%) monocytes.
Culture with M-CSF + RANKL induced TRAP-positive MNC from CD33high monocytes, whereas no or few CD33low monocytes differentiated into TRAP-positive MNC (CD33low vs CD33high, 2 ± 1 vs 192 ± 71 cells/well; n = 3). TRAP-5b and MMP-9 in the culture supernatants, both of which are known to be produced by osteoclasts, were measured by ELISA. The concentrations of both enzymes were significantly higher in the culture supernatant of CD16- monocytes than in that of CD16+ monocytes (Figure 1d). These results suggest that the CD16- peripheral blood monocyte subset, but not the CD16+ subset, differentiate into osteoclasts by incubation with RANKL + M-CSF.<br><br>CD16+ monocytes do not affect the osteoclastogenesis from CD16- monocytes
To examine whether CD16+ monocytes affect osteoclastogenesis from CD16- monocytes, varied numbers of CD16+ monocytes were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured for 7 days in the presence of M-CSF + RANKL. The number of TRAP-positive MNC was not altered by the presence of CD16+ monocytes (Figure 2). The results indicated that CD16+ monocytes did not hamper or enhance the osteoclastogenesis from CD16- monocytes.<br><br>Differences in cytokine production by RANKL-stimulated or M-CSF-stimulated CD16+ and CD16- monocytes
To compare the biological response of CD16+ and CD16- subsets with either RANKL or M-CSF stimulation, we measured the amount of TNFα and IL-6 production after exposure of cells to various concentrations of RANKL or M-CSF with an ELISA. Without RANKL the CD16+ subset produced a significant amount of TNFα and IL-6, whereas the CD16- subset produced undetectable levels (Figure 3a). RANKL stimulation increased TNFα production from both subsets in a dose-dependent manner, although the CD16+ subset produced more TNFα than did the CD16- subset. RANKL stimulation also enhanced IL-6 production from the CD16+ subset, but not from the CD16- subset. M-CSF stimulation increased TNFα and IL-6 production from both subsets, although the CD16+ subset produced more than the CD16- subset (Figure 3b).
These results suggest that CD16+ monocytes also respond both to RANKL and M-CSF stimulation, although such stimulation does not result in differentiation into osteoclasts. CD16+ monocytes were also noted to express higher amounts of inflammatory cytokines compared with CD16- monocytes with or without RANKL or M-CSF stimulation.<br><br>Comparison of expression levels of molecules involved in osteoclastogenesis between CD16+ and CD16- monocytes
Diverse molecules are involved in RANKL/RANK and its costimulatory signal transduction pathways [16]. The different response to RANKL + M-CSF stimulation between the CD16+ monocyte subset and the CD16- monocytes subset might be explained by the expression profiles of these molecules. We therefore examined the mRNA levels of the following molecules: receptor activator of NF-κB (RANK), the receptor for RANKL; c-fms, the receptor for M-CSF; tumor necrosis factor receptor-associated factor 6 (TRAF-6), the adaptor protein for RANK; c-Fos and nuclear factor of activated T cells c1 (NFATc1), transcription factors that are essential for osteoclastogenesis; DNAX-activation protein 12 (DAP12) and Fc receptor γ chain (FcRγ), adaptor proteins known to deliver costimulatory signals in RANKL-induced osteoclastogenesis; signal regulatory protein β1 (SIRP-β1), triggering receptor expressed on myeloid cells 2 (TREM-2) and osteoclast-associated receptor (OSCAR), transmembrane receptors that associate with either DAP12 or FcRγ; and αv and β3, integrins known to be expressed as the αvβ3 heterodimer on osteoclasts.
The mRNA levels of RANK, c-fms, TRAF-6, DAP12 and SIRP-β1 under the baseline condition (no stimulation) varied between the donors; however, we did not find consistent differences in the mRNA levels of these molecules between the CD16+ monocyte subset and the CD16- monocyte subset among three to six donors (Figure 4a). The mRNA levels of other molecules, apart from integrin β3, were similar between the two subsets under the no-stimulation condition. Although the mRNA levels of RANK, c-fms, DAP12, FcRγ, TREM-2 and OSCAR increased in response to M-CSF alone or M-CSF + RANKL in both subsets, the expression levels were not significantly different between the two subsets. Expressions of TRAF-6, c-Fos and SIRP-β1 mRNA did not change following stimulation with M-CSF + RANKL. Of note, the expression of NFATc1 mRNA was enhanced by M-CSF + RANKL treatment only in the CD16- subset. Furthermore, expression of integrin αv in both subsets was enhanced by M-CSF with or without RANKL; however, the expression level was greater in the CD16- subset. It was noted that integrin-β3 mRNA was detected only in the CD16- subset and was increased by M-CSF + RANKL stimulation, but not by M-CSF alone. The protein expression of RANK under the baseline condition was weakly detected in both subsets, and the levels were varied between donors by western immunoblotting.
The protein expression of c-fms was weakly detected in unstimulated CD16+ monocytes, but not in CD16- monocytes (Figure 4b). Flow cytometry analysis of c-fms in fresh monocytes, however, showed that both subsets express the molecule on the cell surface (Figure 4c). Expressions of both RANK and c-fms were upregulated by M-CSF alone and by M-CSF + RANKL, and we did not find consistent differences in the protein levels of these molecules between the two monocyte subsets. The profiles of expression levels of molecules involved in RANKL/RANK and its costimulatory pathways are similar between the two subsets, except for NFATc1, integrin αv and integrin β3. We therefore assumed that the distinct induction of NFATc1, integrin αv and integrin β3 in response to RANKL stimulation among the two monocyte subsets might explain the differences in their abilities to differentiate into osteoclasts.<br><br>RANKL stimulation induces αvβ3 expression on CD16- monocytes
The integrin-β3 subunit binds to integrin αv only and is expressed as the heterodimeric protein αvβ3 on monocytes and osteoclasts [17]. We examined the expression of αvβ3 on CD16+ and CD16- monocytes by immunofluorescent staining. Neither unstimulated nor M-CSF-stimulated monocyte subsets expressed αvβ3 (Figure 4d and data not shown). After 48 and 72 hours of treatment with M-CSF + RANKL, αvβ3-positive mononuclear cells were observed in CD16- monocyte cultures but not in CD16+ monocyte cultures. At 96 hours, both αvβ3-positive mononuclear cells and multinucleated cells were present in the CD16- monocyte culture. The results indicated that αvβ3 was selectively expressed on CD16- monocytes in the presence of M-CSF + RANKL, and the expression was revealed before the cells differentiate into typical multinucleated osteoclasts.<br><br>RANKL activates ERK and p38 kinases only in CD16- monocytes
Since ERK and p38 MAPK are essential in RANKL-induced osteoclastogenesis [18-20], we next examined whether these kinases were activated differently in CD16+ monocytes and in CD16- monocytes. Purified monocytes were precultured with 25 ng/ml M-CSF for 3 days to enhance RANK expression, and were then treated with RANKL. The RANKL treatment induced phosphorylation of both ERK and p38 MAPK in CD16- monocytes at 5 minutes postexposure, although the p38 MAPK phosphorylation was weak. Both phosphorylations declined to a basal level within 20 minutes (Figure 5). In contrast, ERK and p38 MAPK were not detectably phosphorylated in CD16+ monocytes with RANKL.<br><br>siRNA targeting integrin β3 inhibits osteoclastogenesis from CD16- monocytes
The integrin-β3 cytoplasmic domain is essential for activation of intracellular signals from αvβ3 heterodimers [17]. We therefore examined the involvement of αvβ3 in RANKL + M-CSF-induced osteoclastogenesis in <span class="yellow">human</span> CD16- monocytes using siRNA targeting the integrin-β3 subunit. The integrin-β3 siRNA or control randomized siRNA were transfected into CD16- monocytes. At 48 hours post-transfection, we determined the integrin-β3 mRNA level and αvβ3 heterodimer protein expression. The integrin-β3 mRNA level was reduced in the integrin-β3 siRNA-transfected monocytes compared with control siRNA-transfected monocytes (Figure 6a). The αvβ3 heterodimer expression was evaluated by immunofluorescent staining. The number of αvβ3-positive cells was significantly decreased in integrin-β3 siRNA-transfected monocytes compared with that in control siRNA (Figure 6b).
After 7 days of incubation, the number of TRAP-positive MNC was counted. Transfection with integrin-β3 siRNA significantly reduced the number of TRAP-positive MNC in a dose-dependent manner compared with control siRNA transfection (Figure 6c). In addition, the use of siRNA directed toward a different site of integrin-β3 mRNA also inhibited osteoclast formation from CD16- monocytes (data not shown). On the other hand, siRNA that targeted lamin, which was used as a negative control, did not inhibit the induction of osteoclasts (data not shown). These results indicate the importance of integrin β3 in RANKL-induced osteoclast formation from CD16- peripheral blood monocytes.<br><br>Cyclic RGDfV peptide inhibits the osteoclastogenesis from CD16- monocytes
Integrin αvβ3 recognizes a common tripeptide sequence, RGD (Arg-Gly-Asp), which is present in bone matrix proteins such as vitronectin and fibronectin [21]. Cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) inhibits binding of the RGD-containing molecules to αvβ3 [22]. To investigate the role of ligand binding to the αvβ3 heterodimer in the osteoclastogenesis, we examined whether cyclic RGDfV peptide inhibits the formation of osteoclasts. Cyclic RGDfV peptide significantly reduced the number of TRAP-positive MNC in a dose-dependent manner (Figure 6d). The results imply possible involvement of ligand bindings to αvβ3 in the osteoclastogenesis.<br><br>Knockdown of integrin β3 did not affect the expression of NFATc1 mRNA
In the next step, we determined whether integrin-β3-siRNA-induced inhibition of the osteoclastogenesis reflects downregulation of NFATc1, which is a key transcription factor in osteoclastogenesis [23]. For this purpose, we compared NFATc1 mRNA levels between integrin β3 and control siRNA-transfected monocytes. Interestingly, integrin-β3 knockdown did not alter the NFATc1 mRNA level (Figure 7), suggesting that signal transduction mediated by integrin β3 does not affect the expression of NFATc1.<br><br>Detection of CD16+ and CD16- macrophages in synovium of RA <span class="yellow">patients</span>
RA synovial macrophages are derived from peripheral blood monocytes, and their recruitment into the synovium is facilitated by various adhesion molecules and chemokines [24]. To analyze CD16 expression on synovial macrophages, RA synovial tissues were double-stained for CD16 and a macrophage marker, CD68. CD16-/CD68+ macrophages were widespread in the synovium. Although less frequent, CD16+/CD68+ macrophages were also observed both in the synovial intima and subintima (Figure 8). The presence of two subsets of macrophages, CD16+ and CD16-, in RA synovium indicates that both CD16+ and CD16- peripheral blood monocytes are recruited into the synovium.<br><br>
Discussion
<span class="yellow">Human</span> peripheral blood monocytes are a heterogeneous population, and they are divided into two subsets based on the expression of CD16. The CD16+ and CD16- monocyte subsets show functional differences in migration, cytokine production and differentiation into macrophages or dendritic cells [11-13,15]. We focused on the heterogeneity of the monocytes, and the primary question addressed in this study was which monocyte subset could be the source of osteoclasts. The results demonstrated that CD16- peripheral blood monocytes, but not CD16+ monocytes, differentiated in vitro into osteoclasts by treatment with RANKL + M-CSF.
To investigate the molecular mechanisms of the different response to RANKL and the differentiation into osteoclasts between CD16+ and CD16- monocytes, we examined the expression of molecules known to be involved in osteoclastogenesis. The expression profiles of integrin αv, integrin β3 and NFATc1 were different between the two subsets. Integrin αvβ3 heterodimer was expressed only on RANKL and M-CSF-stimulated CD16- monocytes. It is known that αvβ3 expressed on osteoclasts is important in bone resorption as well as in attachment of osteoclasts to the bone matrix [25].
It was recently reported that bone marrow macrophages of integrin-β3-deficient <span class="yellow">mice</span> could not differentiate into mature osteoclasts in vitro, suggesting that αvβ3 is involved not only in activation, but also in differentiation, of osteoclasts in <span class="yellow">mice</span> [26,27]. The authors also showed that αvβ3 and c-fms share a common intracellular signaling pathway, including the activation of ERK and the induction of c-Fos [27], both of which are essential for osteoclastogenesis [28,29]. In addition, it was reported that echistatin, an αvβ3 antagonist, inhibited osteoclast formation of <span class="yellow">mouse</span> bone marrow cells [30].
In accordance with these reports, our data showed that knockdown of integrin-β3 expression resulted in downregulation of the αvβ3 heterodimer, and abrogated osteoclastogenesis from <span class="yellow">human</span> peripheral blood CD16- monocytes. We also showed that blocking of adhesive ligands to bind to αvβ3 by RGDfV peptide inhibited osteoclast formation from CD16- monocytes. Taken together, the process of ligand binding to αvβ3 may be involved in the osteoclastogenesis. Blockade of αvβ3 could therefore be a therapeutically beneficial approach to modulate osteoclastogenesis. Indeed, integrin αvβ3 antagonists effectively treated osteoporosis in <span class="yellow">mice</span>, <span class="yellow">rats</span> and <span class="blue">humans</span>, and protected bone destruction in <span class="yellow">rat</span> adjuvant-induced arthritis in vivo [31-34]. Of note, it is reported that <span class="yellow">patients</span> with Iraqi-Jewish-type Glanzmann thrombasthenia who are deficient in integrin β3 do not develop osteopetrosis because of the upregulation of α2β1 expression on osteoclasts, although the bone-resorptive ability of the osteoclasts was decreased in vitro [35]. The function of αvβ3 in vivo in osteoclast formation and resorptive function could therefore be partially compensated by other integrins.
Although all the multinucleated osteoclasts expressed αvβ3 (Figure 4d) [36], a small number of M-CSF + RANKL-stimulated mononuclear CD16- monocytes expressed αvβ3 (Figure 4d). Multinucleated osteoclasts are formed by fusion of osteoclast precursor cells [37]. It was reported that αvβ3 is involved in the migration of osteoclast precursors [30]. The αvβ3-positive cells could therefore be forced to migrate by the ligands and may fuse with closed αvβ3-negative cells. Alternatively, only αvβ3-positive cells may be fused with each other.
It is possible to consider that signaling from CD16 by anti-CD16 mAb-coated magnetic beads, which were used for the cell separation, or by IgG contained in FBS might inhibit osteoclastogenesis from CD16+ monocytes. We therefore separated the two subsets using anti-CD33 mAb and a fluorescent cell sorter, and stimulated the cells with M-CSF + RANKL. The results showed that CD33low monocytes, which correspond to CD16+ monocytes, still could not differentiate into osteoclasts. CD16 is a heterodimer consisting of FcγIIIa and Fcγ, and has low affinity for the Fc region of IgG. Aggregation of CD16 by immune complexes leads to transmission of activating signals via the immunoreceptor tyrosine-based activation motif in the γ chain [38]. We also assessed osteoclastogenesis from the two monocyte subsets using IgG-depleted <span class="yellow">bovine</span> serum. Even in the IgG-free medium, CD16- monocytes but not CD16+ monocytes differentiated into osteoclasts (data not shown). We could therefore exclude the possibility that signal transduction through CD16 inhibits osteoclastogenesis from CD16+ monocytes.
NFATc1 is a key transcription factor in osteoclastogenesis [16]. In the present study, stimulation with M-CSF + RANKL increased the NFATc1 mRNA expression in the CD16- subset only, similar to integrin αv and integrin β3. The differences in NFATc1 induction might therefore also explain the difference in osteoclastogenesis between the two monocyte subsets. It is of interest that knockdown of integrin β3 did not lower the mRNA level of NFATc1. This result supports the notion that NFATc1 is located upstream of integrin-β3 expression [39]. It is also possible that parallel activation of two signaling pathways mediated by integrin β3 and NFATc1 contributes to osteoclastogenesis independently or cooperatively. Further studies are needed to determine the mechanisms of integrin β3 involvement in RANKL/RANK-mediated osteoclast differentiation.
It has been demonstrated that MAPK families, ERK and p38 MAPK, were activated by RANKL-induced intracellular signalings in osteoclasts and osteoclast precursors [18,19]. In addition, these kinases are involved in the differentiation of osteoclasts [20]. We showed that RANKL stimulation induced phosphorylation of ERK and p38 MAPK only in CD16- monocytes. It is suggested that differential activation of these kinases may partially explain the distinct properties of the two monocyte subsets upon RANKL stimulation.
Our results showed that CD16+ monocytes produce higher levels of inflammatory cytokines including TNFα and IL-6 compared with CD16- monocytes. These results are consistent with the previous report showing that CD16+ monocytes produced larger amounts of TNFα upon lipopolysaccharide or lipopeptide stimulation than did CD16- monocytes [40]. Interestingly, we showed that stimulation either with RANKL or M-CSF upregulated the TNFα and IL-6 production by CD16+ monocytes. A marked increase of CD16+ monocytes in peripheral blood is reported in inflammatory diseases, such as infection, malignancy, Kawasaki disease and RA [41-44]. Taken together, CD16+ monocytes may be an important source of inflammatory cytokines.
In <span class="yellow">mice</span>, peripheral blood Ly-6Chigh monocytes, which are thought to correspond to <span class="yellow">human</span> CD16- monocytes, increase in inflammatory conditions, and these cells are recruited into sites of inflammation [45]. In contrast, Ly-6Clow monocytes, which are thought to correspond to <span class="yellow">human</span> CD16+, migrate into noninflamed tissues [12]. These data on <span class="yellow">mouse</span> monocytes seem to be in contrast to the data on <span class="yellow">human</span> monocytes, which show expansion of CD16+ monocytes in inflammatory conditions where they produce larger amounts of inflammatory cytokines. At present, it is not clear whether <span class="yellow">mouse</span> monocyte subsets, Ly-6Clow/Ly-6Chigh, represent <span class="yellow">human</span> monocyte subsets, CD16+/CD16- monocytes, and whether the biologic functions of <span class="yellow">mouse</span> monocytes are analogous to those of <span class="yellow">human</span> monocytes.
In <span class="yellow">mice</span>, blood monocytes newly released from the bone marrow are exclusively Ly-6Chigh and the level of Ly-6C is downregulated while in circulation [45]. It is thus suggested that in <span class="yellow">mice</span> the two monocyte subsets differing in Ly-6C expression represent different stages in the maturation pathway. In the <span class="yellow">human</span>, transition from CD16- monocytes to CD16+ monocytes is observed upon culture with IL-10, M-CSF and transforming growth factor beta in vitro [42,46]. Similar to <span class="yellow">mouse</span> monocytes, therefore, <span class="yellow">human</span> peripheral blood CD16- monocytes may also maturate into CD16+ monocytes.
It is reported that a significant number of RA synovial cells in the intima express CD16, suggesting that CD16+ cells are synovial macrophages [47]. We confirmed that both CD16+ and CD16- macrophages accumulate in the RA synovium by double-color immunohistochemical staining for CD68 and CD16. A number of chemokines are abundantly expressed in the RA synovium [24,48]. Among these cytokines, MCP-1, MIP-1α, SDF-1, RANTES and fractalkine can induce migration of CD16- monocytes in vitro ([11,12] and unpublished data). On the other hand, migration of CD16+ monocytes is induced only by fractalkine. These chemokines therefore seem to play an important role in recruitment of CD16+ and CD16- monocytes from the circulating pool into the RA synovium.
The osteoclast inducers are also produced in the RA synovium. RANKL is expressed by synovial fibroblasts and activated T cells [49-51], while M-CSF is expressed on RA synovial macrophages and fibroblasts [52,53].
TNFα and IL-6, which are mainly expressed on RA synovial macrophages and fibroblasts, respectively, could also enhance osteoclast differentiation [54]. Collectively, it is probable that the recruited CD16- monocytes/macrophages differentiate into osteoclasts in the RA synovium, and contribute to bone destruction. On the other hand, CD16+ monocytes/macrophages might also be involved in RA pathogenesis by producing inflammatory cytokines including TNFα and IL-6. Since TNFα and IL-6 enhance osteoclast formation [54,55], CD16+ monocytes/macrophages may also contribute to osteoclastogenesis in RA synovium.<br><br>Conclusion
We have shown that <span class="yellow">human</span> peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct response to osteoclastogenic stimuli. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for the osteoclastogenesis. The blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Abbreviations
DAP = DNAX-activation protein; ELISA = enzyme-linked immunosorbent assay; FBS = fetal <span class="yellow">bovine</span> serum; FcRγ = Fc receptor γ chain; IL = interleukin; FITC = fluorescein isothiocianate; mAb, monoclonal antibody; M-CSF = macrophage colony-stimulating factor; MEM = modified Eagle's medium; MMP = matrix metalloproteinase; MNC = multinucleated cells; NF = nuclear factor; OSCAR = osteoclast-associated receptor; PBS = phosphate-buffered saline; PCR = polymerase chain reaction; RA = rheumatoid arthritis; RANK = receptor activator of NF-κB; RANKL = receptor activator of NF-κB ligand; RT = reverse transcriptase; siRNA = small interfering RNA; SIRP-β1 = signal regulatory protein-β1; TNFα = tumor necrosis factor alpha; TRAF = tumor necrosis factor receptor-associated factor; TRAP = tartrate-resistant acid phosphatase; TREM = triggering receptor expressed on myeloid cells.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YK participated in the design of the study, carried out the experiments and statistical analysis, and drafted the manuscript. KH and KT participated in the design of the study and its coordination. TN and NM conceived of the study, participated in its design and coordination, and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2151861</h3>Recombinant activated protein C in sepsis: endothelium protection or endothelium therapy?
Abstract
Endothelium dysfunction is one of the hallmarks of sepsis. Looney and Mattay, in the previous issue of Critical Care, highlight the role of activated protein C (APC) as a protective endothelial drug in septic situations. Nevertheless, the results of in vivo studies are less explicit and it remains uncertain whether these properties are relevant in <span class="yellow">human</span> septic shock. Before considering recombinant APC (rAPC) as a therapeutic drug for the endothelium, we have to demonstrate its efficiency to protect or to reduce endothelium injury when infused a long time after the septic challenge. Nevertheless, if rAPC is efficient when infused in the early phase of septic challenge, we thus need to treat our <span class="yellow">patients</span> earlier. At the least, genetically engineered variants have been designed with greater anti-apoptotic activity and reduced anticoagulant activity relative to wild-type APC. Further studies are needed to demonstrate the usefulness of these variants in septic shock therapy.<br><br>
The use of recombinant activated protein C (rAPC) is one of the hottest topics in septic shock therapy. The pivotal phase 3 placebo-controlled Protein C Worldwide Evaluation in Severe Sepsis (PROWESS) clinical trial demonstrated a 19.4% relative risk reduction in 28-day mortality (6.1% absolute risk reduction) with an increased risk (3.5% versus 2.0%) of serious bleeding events compared with placebo.
Two recent and important articles have highlighted the role of APC as a protective endothelial drug [1] and as a cyto-protective drug [2]. Beneficial effects of rAPC in the PROWESS study were thought to be related to a reduction in coagulation and, to a lesser extent, to a reduction in inflammatory response to sepsis [3]. Post-PROWESS investigations have been associated with a myriad of cellular or animal studies demonstrating that rAPC, through reactions mediated by endothelial protein C receptor and the effector receptor, protease activated receptor-1, acts directly on cells to exert multiple cytoprotective effects including: down regulation of pro-inflammatory gene expression [4]; anti-inflammatory activities [5]; anti-apoptotic activity [6]; and protection of endothelial barrier function [1,2].
Endothelium dysfunction is one of the hallmarks of sepsis [7]. Sepsis, per se, may induce phenotypic modulations of the endothelium through direct or indirect interaction of the endothelial layer with components of the bacterial wall, inducing a myriad of host-derived factors from endothelial cells. Phenotypic modifications include changes in pro-coagulant and proadhesive properties, increased endothelial permeability, endothelial cell apoptosis and changes in vasomotor properties; the last of these is crucial since vasoplegia is directly related to septic shock mortality. Recent animal and <span class="yellow">human</span> data have suggested that rAPC may improve both vascular and myocardial dysfunction and vascular reactivity to catecholamine during endotoxin and/or septic challenge [8,9].<br><br>From bench to bedside
Experimental evidence supports a role of APC in maintaining the integrity of the endothelium through both direct and indirect mechanisms. Nevertheless, the results of in vivo studies are less explicit. In a retrospective study of septic shock in <span class="blue">humans</span>, Monnet and colleagues [9] demonstrated that APC infusion was associated with a decrease in the amount of delivered norepinephrine. Wiel and colleagues [10] demonstrated in a <span class="yellow">rabbit</span> model of endotoxin induced shock that APC decreased aorta endothelial injury. By contrast, in a lung model of endotoxin induced inflammation, Robriquet and colleagues [11] demonstrated a trend to an increased vascular permeability using high doses of <span class="yellow">human</span> APC. This last result was in sharp contrast with the results obtained by Nick and colleagues [12] in a <span class="yellow">human</span> model of pulmonary endotoxin administration. APC appears to improve mortality in septic shock with a high APACHE 2 score and is potentially detrimental in severe sepsis. In <span class="yellow">rats</span>, APC markedly decreased tumour necrosis factor concentrations whereas they remained unchanged in either <span class="yellow">human</span> septic shock or endotoxemia. The question arises, therefore, as to whether it is truly possible to reconcile all these discrepancies? Moreover, can these stirring laboratory data be translated into clinical practice?<br><br>Limitations of experimental studies: endothelium protection versus endothelium therapy
Clearly, in cellular and animal models, rAPC has been given either as a pre-treatment or concurrent with septic challenge. This mode of administration favours the anti-inflammatory effects of rAPC, which are particularly efficient in <span class="yellow">murine</span> models in protecting the endothelium from cytokine-mediated apoptosis or upregulation of endothelial adhesion molecules. Thus, studies using post-injury treatment are needed in models that mimic septic shock, such as experimental pneumonia or peritonitis treated by antibiotics and volume resuscitation, and where the effects of rAPC would be investigated 16 to 24 hours after septic challenge. If we can demonstrate the efficiency of rAPC to protect or to reduce endothelium injury in these conditions, we can ultimately postulate that rAPC is also a therapeutic drug for the endothelium.<br><br>The earlier the better
If rAPC is efficient when infused in the early phase of septic challenge, we thus need to treat our <span class="yellow">patients</span> earlier. At least two studies suggest that treatment with rAPC within 24 hours may carry a larger survival advantage for <span class="yellow">patients</span> with severe sepsis, compared with those treated more than 24 hours after organ dysfunction [13]. Interventions directed at specific endpoints, when initiated early in the 'golden hours' of a patient's condition, seem to be promising [14]. The beneficial effects of earlier administration of rAPC to appropriate <span class="yellow">patients</span> may fit into this paradigm.<br><br>The future
Extensive in vivo and in vitro studies have focused on the cytoprotective effects of APC and most authors agree that its anticoagulant and cytoprotective effects are mediated by distinct APC structural features. Positively charged residues in surface loops in the APC protease domain have been identified as participating in the anticoagulant activity but not in cellular effects. Hence, variants have been designed with greater anti-apoptotic activity and reduced anticoagulant activity relative to wild-type APC [2]. Whether these genetically engineered variants actually provide superior pharmacological properties remains to be elucidated in vivo. Such investigations may allow the design of therapeutic APC variants with decreased anticoagulant activity to reduce the risk of bleeding on the one hand, but also with normal cytoprotective properties in order to retain full beneficial effects on sepsis outcome.<br><br>Abbreviations
PROWESS = Protein C Worldwide Evaluation in Severe Sepsis; rAPC = recombinant activated protein C.<br><br>Competing interests
BL has received reimbursements and funding from Eli Lilly, France.<br><br>
<h3>pmcA2398786</h3>Genomic-Bioinformatic Analysis of Transcripts Enriched in the Third-Stage Larva of the Parasitic Nematode <span class="yellow">Ascaris suum</span>
Abstract
Differential transcription in <span class="yellow">Ascaris suum</span> was investigated using a genomic-bioinformatic approach. A cDNA archive enriched for molecules in the infective third-stage larva (L3) of <span class="yellow">A. suum</span> was constructed by suppressive-subtractive hybridization (SSH), and a subset of cDNAs from 3075 clones subjected to microarray analysis using cDNA probes derived from RNA from different developmental stages of <span class="yellow">A. suum</span>. The cDNAs (n = 498) shown by microarray analysis to be enriched in the L3 were sequenced and subjected to bioinformatic analyses using a semi-automated pipeline (ESTExplorer). Using gene ontology (GO), 235 of these molecules were assigned to ‘biological process’ (n = 68), ‘cellular component’ (n = 50), or ‘molecular function’ (n = 117). Of the 91 clusters assembled, 56 molecules (61.5%) had homologues/orthologues in the free-living nematodes <span class="yellow">Caenorhabditis elegans</span> and <span class="yellow">C. briggsae</span> and/or other organisms, whereas 35 (38.5%) had no significant similarity to any sequences available in current gene databases. Transcripts encoding protein kinases, protein phosphatases (and their precursors), and enolases were abundantly represented in the L3 of <span class="yellow">A. suum</span>, as were molecules involved in cellular processes, such as ubiquitination and proteasome function, gene transcription, protein–protein interactions, and function. In silico analyses inferred the <span class="yellow">C. elegans</span> orthologues/homologues (n = 50) to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%), or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway. Small numbers of these 50 molecules were predicted to be secreted (10%), anchored (2%), and/or transmembrane (12%) proteins. Functionally, 17 (34%) of them were predicted to be associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority being embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%). A genetic interaction network was predicted for these 17 <span class="yellow">C. elegans</span> orthologues, revealing highly significant interactions for nine molecules associated with embryonic and larval development (66.9%), information storage and processing (5.1%), cellular processing and signaling (15.2%), metabolism (6.1%), and unknown function (6.7%). The potential roles of these molecules in development are discussed in relation to the known roles of their homologues/orthologues in <span class="yellow">C. elegans</span> and some other nematodes. The results of the present study provide a basis for future functional genomic studies to elucidate molecular aspects governing larval developmental processes in <span class="yellow">A. suum</span> and/or the transition to parasitism.<br><br>Introduction
Parasitic nematodes are of major socio-economic importance in animals. For example, hundreds of millions of <span class="yellow">people</span> are infected with geohelminths (soil-transmitted worms), such as blood-feeding hookworms <span class="yellow">Ancylostoma duodenale</span> and/or <span class="yellow">Necator americanus</span>, <span class="yellow">Trichuris trichiura</span> and Ascaris spp. [1], causing serious adverse effects on <span class="yellow">human</span> health, particularly in <span class="yellow">children</span>. Similarly, parasitic nematodes of livestock, such as <span class="yellow">pigs</span>, also cause substantial economic losses due to subclinical and clinical diseases, with billions of dollars spent annually on the treatment and control of gastro-intestinal nematodes. In addition to the socioeconomic impact that these parasites have, there is potential for the emergence of resistance in them against all of the main classes of (nematocidal) compounds used to treat the diseases they cause [2]–[5]. Therefore, there is a significant need to work toward discovering new compounds to control these parasites. Gaining an improved understanding of the molecular basis of parasite development provides such an avenue.
Compared with the free-living nematode <span class="yellow">Caenorhabditis elegans</span>, there is very little information on fundamental molecular aspects of development in parasitic nematodes [6]–[8]. Since the genome sequence of <span class="yellow">C. elegans</span> was published in 1998 [9], many aspects of the molecular biology of this nematode have been elucidated. For instance, microarray analyses have been used to examine developmental and gender-enriched gene expression [10],[11], and the functions of more than 96% of the <span class="yellow">C. elegans</span> genes have been assessed by double-stranded RNA interference (RNAi, or gene silencing; [12]) [13]–[18]. Comparative analyses of genetic data sets have shown that parasitic nematodes usually share ∼50–70% of genes with <span class="yellow">C. elegans</span> (e.g., [19],[20]). There is similarity in other features (such as basic body plan and moulting) between <span class="yellow">C. elegans</span> and parasitic nematodes, suggesting that some molecular pathways are relatively conserved [8],[21]. Understanding the pathways linked to basic nematode biology and development could have important implications for finding new ways of disrupting these pathways and thus facilitate the identification of new drug targets.
Despite the advances in genomic technologies [7], [22]–[29] and the study of <span class="yellow">C. elegans</span>, there is a paucity of information on the genomics of parasitic nematodes of animals, particularly in relation to development. Also considering the major socioeconomic impact of Ascaris and ascariasis in <span class="blue">humans</span> and <span class="yellow">pigs</span> [30]–[32], several characteristics, including the large size of the adult worm (providing the opportunity of investigating individual organ systems and tissues), the ability to maintain Ascaris in the <span class="yellow">pig</span>, store eggs and culture larvae in vitro for relatively long periods of time (months to years) [32] as well as the discovery that RNAi achieves “cross-species” gene silencing for a selected number of genes [33],[34] and the imminent genome sequence (http://www.sanger.ac.uk/Projects/Helminths/) all indicate that Ascaris could serve as a powerful model system for investigating reproductive and developmental processes in nematodes.
In the present study, Ascaris from <span class="yellow">pigs</span> was used to study molecules abundantly transcribed in the infective third-stage larva (L3). Following the oral ingestion of Ascaris eggs by the host (<span class="yellow">human</span> or <span class="yellow">pig</span>), L3s are released and then invade/penetrate predominantly the caecal wall [35] to then undergo hepato-pulmonary migration, after which ultimately the adult females and males establish and develop in the small intestine [36],[37]. The molecular mechanisms linked to host invasion and parasite development are largely unknown. Here, we constructed an L3-enriched cDNA library using the method of suppressive-subtractive hybridization (SSH), explored transcription of a representative subset of molecules by microarray analysis and conducted bioinformatic analyses to characterize these molecules, map them to biochemical pathways and predict genetic interactions based on comparisons with <span class="yellow">C. elegans</span> and/or other organisms.<br><br>Materials and Methods
Production of Different Developmental Stages of Ascaris
Experimental <span class="yellow">pigs</span> (8–12 weeks of age) were purchased from and maintained in the Experimental Animal Center of South China Agricultural University. These <span class="yellow">pigs</span> were treated humanely, according to the Animal Ethics procedures and guidelines of the People's Republic of China. Adult <span class="yellow">worms</span> (males and females) of <span class="yellow">A. suum</span> were collected from the small intestines of <span class="yellow">pigs</span> from an abattoir in Shenzhen, China. Infective eggs and infective L3s of <span class="yellow">A. suum</span> were produced according to the methods described previously [38]. In brief, eggs from the uteri of adult females of <span class="yellow">A. suum</span> were collected and incubated at 28°C for 28 days to allow them to develop to infective eggs (containing infective L3s). To obtain pure infective L3s, 7.5% v/v sodium hypochlorite was used to treat the larvated eggs at 37°C overnight and then the eggs were shaken with glass-beads; then, the exsheathed L3s and shells were separated by density gradient centrifugation using lymphocyte separating medium (LSM) [38]. Following the experimental infection of helminth-free <span class="yellow">pigs</span> with infective Ascaris eggs as described previously [39], the L3s from livers and in lungs as well as L4s in intestines were isolated according to an established method [40]. All parasite materials were snap-frozen in liquid nitrogen prior to storage at −70°C.<br><br>Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)
Total RNA was isolated from adult females and males, different larval stages or eggs of <span class="yellow">A. suum</span> using TriPure reagent (Roche) as recommended by the manufacturer. Equal amounts of total RNA from each stage or sex were pooled. The mRNA was isolated using the Oligotex mRNA Kit (Qiagen), following the manufacturer's protocol. SSH was carried out using the PCR-Select cDNA Subtraction kit (Clontech), according to the manufacturer's protocol. In brief, cDNA synthesized from mRNAs from infective L3s was subtracted against cDNA synthesized from the pooled mRNA from all other stages included herein. The SSH library was constructed using infective L3s as the tester and pooled cDNAs from all other stages as the driver. The effectiveness of this subtraction process has already been demonstrated in previous studies [41],[42]. The cDNA obtained following SSH was cloned into the pGEM-T Easy plasmid vector (Promega) and competent <span class="yellow">Escherichia coli</span> (JM109) transformed. Positive clones, picked randomly (based on blue/white selection), were grown overnight in Luria Bertani (LB) medium (shaking, 37°C). Individual inserts were PCR-amplified using “nested primers” 1 and 2R from the Subtraction kit (Clontech) and examined by agarose electrophoresis.<br><br>Preparation of Microarray Slides
Clones (n = 3075) from the subtracted library were picked and cultured overnight in LB containing ampicillin (1000 IU/ml) in sealed 96-well blocks. Five µl of culture suspension from each well were transferred into individual wells thermocycling (96-well) plates and the inserts PCR-amplified using primers 1 and 2R. Following a 10 min denaturation step at 94°C, the amplification proceeded for 25 cycles of 10 s at 94°C, 30 s at 68°C and 1.5 min at 72°C, with a final extension for 5 min at 72°C. Products were resolved in agarose gels, ethanol precipitated, re-suspended in 16 µl of “spotting solution” (Shanghai BioStar Genechip, Inc) to a final concentration of ∼500 ng per µl, before being printed on to glass slides (in duplicate) using a robotic arrayer. Sixteen blanks (using spotting solution only) and the same number of negative (irrelevant cDNAs with no relationship to Ascaris) were also printed on to slides and served as negative controls; β-actin of <span class="yellow">A. suum</span> served as a positive control to assess the efficiency of labeling and hybridization. The slides were air-dried for 2 h, and cDNA in the spots were cross-linked at 254 mJ. The printed slides were stored at 4°C.<br><br>Labeling of cDNA Probes with Fluorescent Dyes, and Microarray Analysis
The cDNAs produced from total RNA from <span class="yellow">A. suum</span> eggs, infective L3s, L3s isolated from <span class="yellow">pig</span> liver or lung, fourth-stage larvae (L4s), adult males or females [as described in the section ‘Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)’] were labeled with cyanine dyes. Cy3 or Cy5-dCTP was incorporated into cDNA produced from 30 µg of total RNA by direct labeling in a reverse transcription reaction using an oligo (dT) primer. Labeled cDNA was purified using DyeEx columns (Qiagen).
Microarray slides were incubated with a pre-hybridization solution [5×SSC, 1% <span class="yellow">bovine</span> serum albumin (BSA), 0.1% sodium dodecyl-sulphate (SDS)] for 6 h at 42°C. After pre-hybridization, the microarray slides were incubated with ‘pooled’ Cy3 and Cy5-labeled probes in hybridization solution (5×SSC, 1% BSA, 0.1% SDS), in the dark at 42°C for 18 h, and then washed in solution I (1×SSC, 0.2% SDS) for 10 min, followed by solution II (0.1×SSC, 0.2% SDS) for 10 min at 60°C, according to the protocols provided by Shanghai BioStar Genechip, Inc. A “dye flip” was carried out to control for any bias in hybridization signal between the Cy-labeled cDNA probes (produced for two distinct mRNA populations). The slides were dried and scanned (ScanArray 4000 scanner) using image acquisition software (Shanghai BioStar Genechip Inc.) and a range of laser power and photo-multiplier tube intensities. The mean hybridization signal (derived from four replicates of the same array) were corrected for background, normalized [43], log2-transformed and then subjected to statistical analysis employing the students t-test in a spreadsheet (Excel, Microsoft, USA). The microarray data were analysed for differential cDNA hybridization (>2.0-fold to 114.3-fold) between L3 and each of the other stages (eggs, lung and liver L3s, L4, adult female and adult male).<br><br>Verification of Differential Hybridization by Reverse Transcription-Coupled Polymerase Chain Reaction (RT-PCR) Analysis
For a subset (n = 17) of representative ESTs (rESTs), RT-PCR was used to verify the differential transcription recorded by microarray analysis. Double-stranded cDNA was synthesized from total RNA (separately) from each stage or sex of <span class="yellow">A. suum</span> using reverse transcriptase (Superscript III, Invitrogen). Briefly, 5 µg of total RNA were added to 14 µl of H2O and 1 µl of oligo d(T)n = 12–18 primer (0.5 µg/µl), heated to 70 °C for 10 min and chilled on ice. First- and second-strand cDNAs were synthesized via the addition of 4 µl of first-strand cDNA buffer (250 mM Tris-HCl, pH 8.3, 375 mM KCl and 15 mM MgCl2), 2 µl of 0.1 M dithiothreitol, and 1 µl of 10 mM of each dNTP, followed by an incubation at 25 °C (10 min), 42 °C (50 min) and 70 °C (15 min). One-tenth of each double-stranded cDNA produced was then used as a template in the PCR. The transcripts were amplified from individual cDNAs by PCR using oligonucleotide primers (sequences available upon request) designed to each EST. The PCR amplification of a portion (209 bp) of the β-actin gene (accession no. BI594141) using forward primer (5′-CTCGAAACAAGAATACGATG-3′) and reverse primer (5′- ACATGTGCCGTTGTATGATG-3′), previously determined to be present in all developmental stages and both sexes of <span class="yellow">A. suum</span> [44], served as a positive control. Samples without template (no-DNA controls) were included in each PCR run. The following cycling conditions were employed: one cycle at 94 °C (5 min), 94 °C (30 s), 60 °C (30 s) and 72 °C (30 s) for 30 cycles, followed by a final extension of 70 °C (7 min). Following the PCR, 5 µl of individual amplicons were resolved in ethidium bromide-stained agarose gels (2%) and then photographed upon transillumination. The relative band intensities were analyzed using UVIsoft Image Acquisition and Analysis software (UVITEC). The specificity and identity of individual amplicons were confirmed by direct sequencing using the same primers (separately) as employed for their amplification.<br><br>Sequencing and Bioinformatics Analyses
Clones from the SSH cDNA library with increased hybridization in microarray analysis to the infective L3 compared with other stages were sequenced using standard technology [45]. The nucleotide sequences have been deposited in the GenBank database under accession numbers ES290984-ES291074. Following the processing of the sequences (i.e., removal of vector sequences, quality assurance and clustering), contigs or singletons from individual clusters were subjected to BLASTx (NCBI: www.ncbi.nlm.nih.gov) and BLASTn (EMBL-EBI Parasite Genome Blast Server: www.ebi.ac.uk) analysis to identify putative homologues in <span class="yellow">C. elegans</span>, other nematodes and other organisms (e-value of ≤1e-05). Peptides inferred from ESTs were classified functionally using Interproscan (available at http://www.ebi.ac.uk/InterProScan/) employing the default search parameters. WormBase (www.wormbase.org) was interrogated extensively for relevant information on <span class="yellow">C. elegans</span> homologues/orthologues, including RNAi phenotypic, transcriptomic, proteomic and interactomic data. ESTs with homologues/orthologues in <span class="yellow">C. elegans</span> and other nematodes were also subjected to analysis employing the KEGG Orthology-Based Annotation System (KOBAS) (www.kobas.cbi.pku.edu.cn), which predicts the biochemical pathways in which molecules are involved. The open reading frames (ORFs) inferred from selected ESTs with orthologues in <span class="yellow">C. elegans</span> were also subjected to “secretome analysis” using the program SignalP v.2.0 www.cbs.dtu.dk/services/SignalP/), employing both the neural network and hidden Markov models to predict signal peptides and/or anchors [46]–[48]. Also, transmembrane domains were predicted using the program TMHMM (www.cbs.dtu.dk/services/TMHMM/; [49]–[51]), and subcellular localization inferred employing the program WoLF PSORT (http://wolfpsort.org/; [52]).
The method established by Zhong and Sternberg [53] was used to predict the interactions for <span class="yellow">C. elegans</span> orthologues of the L3-enriched molecules from Ascaris. In brief, interaction, phenotypic, expression and gene ontology data from <span class="yellow">fruitfly</span>, <span class="yellow">yeast</span>, <span class="yellow">mouse</span> and <span class="yellow">human</span> were integrated using a naïve Bayesian model to predict genetic interactions among <span class="yellow">C. elegans</span> genes ([45],[53]; Zhong and Sternberg, unpublished). The predicted networks resulting from the analyses were saved in a graphic display file (gdf) format and examined using the graph exploration system available at http://graphexploration.cond.org/. Images were labeled and saved in the joint photographic experts group (jpeg) format.<br><br>
Results
To identify molecules transcribed abundantly in the L3 of <span class="yellow">A. suum</span>, an enriched cDNA library was constructed by SSH. From a total of 3075 clones from this library, 2921 (95%) were shown to contain an insert (which could be amplified by PCR). From 2671 (92%) of these clones, amplicons representing single bands of ∼400 to 600 bp in size were produced. These latter amplicons were arrayed (in duplicate) on to slides and then hybridized with Cy3-labeled L3-cDNA or with Cy5-labeled cDNA from eggs, liver/lung L3s, L4s, adult female or adult male of Ascaris. Dye flip was conducted to verify the hybridization data. Of the 2671 (duplicate) spots, 1526 had a significant difference in hybridization between infective L3 cDNA and cDNAs from all other stages or sexes of <span class="yellow">A. suum</span>, of which 515 had a >2.0-fold increased hybridization for the L3.
In order to independently verify the hybridization results in the microarray, a PCR-based analysis of a selected subset (n = 17) clones was conducted using specific primer pairs. Having verified the specificity and identity of individual amplicons by sequencing, PCR results were reproducible (based on multiple runs on different days) and ∼94% (16 of 17) concordant with those of the microarray analysis (not shown). There was complete concordance for representative clones associated with a differential signal of ≥3.0-fold in the microarray.
The clones linked to the 515 spots representing increased transcription (>2.0-fold) in infective L3 compared with the other developmental stages or sexes included were subjected to sequencing. The 498 sequences (length: 550±115 bp) determined were then subjected to detailed bioinformatic analyses. There were 91 unique clusters (accession numbers ES290984-ES291074), of which 55 were singletons (sequences determined once). Of 56 molecules (61.5%) with significant similarity to sequences other than <span class="yellow">A. suum</span> in the databases interrogated, 50 (54.9%) had <span class="yellow">C. elegans</span> or <span class="yellow">C. briggsae</span> homologues, and six had similarity to ESTs already sequenced from ascaridoid and/or other parasitic nematodes, and/or other organisms (Table 1). A significant proportion (38.4%) did not have any similarity in sequence to any organisms for which data are presently available. Comparative analysis specifically against <span class="yellow">A. suum</span> EST data sets (n∼42,000) available in public databases confirmed independently that the majority of molecules (>60%) were present exclusively in the infective L3 stage or were orphans.
As gene ontology (GO) provides a hierarchy that unifies the descriptions of biological, cellular and molecular functions [54], this approach was employed to predict the classification and gene function of molecules enriched in infective L3 of <span class="yellow">A. suum</span>. A summary of the GO categories of these molecules is displayed in Fig. 1. Of the 91 contigs, 32 (35%) could be functionally assigned to ‘biological process’ (n = 38), ‘cellular component’ (n = 17) and ‘molecular function’ (n = 64). The most common subcategories were gluconeogenesis (13%) and metabolic process (13%) within ‘biological process’, extracellular region (24%) within ‘cellular component’, and catalytic activity (11%) and phosphoenolpyruvate carboxykinase activity (8%) within ‘molecular function’ (Table S1).
A focused KOBAS analysis inferred the 50 <span class="yellow">C. elegans</span> orthologues/homologues to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%) or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway (Table 2). Of these 50 molecules, small numbers were predicted to be secreted (10%), anchored (2%) and/or transmembrane (12%) proteins (Table 2). Functionally, 17 (34%) of the 50 molecules were associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority displaying embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%) (Table 2).
Extending this analysis, a relatively complex genetic interaction network was predicted for the 17 <span class="yellow">C. elegans</span> orthologues (i.e., with non-wild-type RNAi phenotypes) (see Table S2). Statistically highly significant interactions were predicted for nine of the <span class="yellow">C. elegans</span> genes; the top five interactors are displayed in Fig. 2. The gene ontology categories for eight selected <span class="yellow">C. elegans</span> genes (F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2) included: embryonic development, egg hatching, larval development and/or growth. The other categories included: positive regulation of growth rate (F55A12.8, kin-2, mup-2, pab-1, rpl-22 and T21B10.2) and gamete generation and locomotory behaviour (kin-2, mup-2, pab-1 and F55A12.8, kin-2, mup-2, respectively). The <span class="yellow">C. elegans</span> homologue egl-3 was predicted to be involved in proteolysis (see www.wormbase.org). All nine <span class="yellow">C. elegans</span> orthologues were predicted to interact directly with a total of 296 (range: 5–75) other genes and, in particular, a direct genetic interaction was predicted between pab-1 and T21B10.2 (Fig. 2). The 296 interactors were associated with embryonic and larval development (n = 198; 66.9%), information storage and processing (n = 15; 5.1%), cellular processes and signalling (n = 45; 15.2%) and metabolism (n = 18; 6.1%); the precise function of some of the interactors (n = 20; 6.7%) is presently unknown (Table S2).<br><br>Discussion
The present study investigated transcripts in infective L3s of <span class="yellow">A. suum</span> using a genomic-bioinformatic platform. The focus was on comparisons with <span class="yellow">C. elegans</span> homologues/orthologues, because the entire genome sequence of this nematode is known [9] and because there is a wealth of information on the localization and functionality of its molecules (www.wormbase.org; http://elegans.bcgsc.bc.ca/knockout.shtml). The functions of most genes in <span class="yellow">C. elegans</span> have been assessed using RNAi (e.g., [14],[15],[17],[55],[56]) in the hermaphroditic stage, whereas there is a paucity of functional information available for Ascaris and other parasitic nematodes of animals [57],[58].
Following the microarray analysis of >2500 ESTs from the SSH library, 498 cDNAs inferred to be enriched in the L3, based on hybridization signal, were sequenced and subjected to comprehensive in silico analyses. Of the 91 clusters of molecules categorized, 50 (54.9%) had <span class="yellow">C. elegans</span> homologues/orthologues with loss-of-function phenotypes could be mapped to key pathways. The statistically significant genetic interactions predicted for 9 of the 50 <span class="yellow">C. elegans</span> orthologues [namely egl-3, F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2 ( = enol-1)] and the interaction network included genes encoding kinases, alpha-tubulins, enolases, troponin and other named and unnamed proteins. Eight of these molecules (enol-1, pab-1, F33D11.10, rpl-22, F55A12.8 mec-12, mup-2 and kin-2) have known or predicted roles in embryonic and larval growth and development, gamete generation, locomotory behaviour or other biological processes in <span class="yellow">C. elegans</span> (see www.wormbase.org).
The enolase encoded by enol-1 is predicted to play a role in glycolysis, gluconeogenesis, phenylalanine, tyrosine and tryptophan biosynthesis (cf. [59]). Since glucose is the main source for ATP production, the alteration in these key glycolytic enzymes may lead to cellular dysfunction, such as impaired ion-motive ATPase required to maintain potential gradients, operate pumps and maintain membrane lipid asymmetry [60]. Bioinformatic analysis for transmembrane helices (TMHMM) and peptide signal sequences (SignalP) predicted ENOL-1 to be a non-secreted protein localized to the cytoplasm (cf. Table 2). Nonetheless, enolases are often detected in the excretory/secretory (ES) products of parasitic helminths, including adult <span class="yellow">A. suum</span> [61], and appear to play a role in the triggering of nitric oxide production by host cells. The enol-1 orthologue of <span class="yellow">C. elegans</span> has been predicted to interact specifically with the polyadenylate binding protein gene, pab-1, inferred to be involved in coordinated gene transcription and expression during normal larval development [16]. Poly(A)-binding proteins (PABPs) are recognized to be central to the regulation of mRNA translation and stability [62]. Present evidence suggests that the expression of PAB-1 is regulated by an oligo-pyrimidine tract in response to cell growth and relates to coordinated growth regulation in <span class="yellow">C. elegans</span> [62]. Furthermore, gene silencing of pab-1 and its selected interactors (see Fig. 2) leads to embryonic lethal (Emb), slow growth (Slo) and sterile progeny (Stp) phenotypes (see www.wormbase.org).
Another gene (F33D11.10; EST code 4F10; see Table 2) which encodes an ATP-dependent RNA helicase and is associated with embryonic lethal (Emb) and larval lethal (Lvl) RNAi phenotypes, was shown to be highly transcribed in infective L3s of <span class="yellow">A. suum</span>. Helicases are involved in a variety of RNA metabolic processes, including translation initiation, pre-mRNA splicing, pre-rRNA processing, rRNA maturation and RNA degradation [63], and are crucial for life cycle progression, sex determination and early embryogenesis in <span class="yellow">C. elegans</span> [60]. The high transcription levels of a homologue/orthologue in the L3 of <span class="yellow">A. suum</span> might suggest a similar role in this ascaridoid. Similarly, the coordination of the expression of a large number of genes is required for normal growth and cell proliferation during larval development. The high transcription level for the ribosomal protein gene homologue rpl-22 (large subunit family member; EST code 26G12, see Table 2) in the infective L3 of <span class="yellow">A. suum</span> compared with other developmental stages is likely to reflect the substantial rate of cell growth in this stage [64].
The gene (F55A12.8, EST code 4G11; see Table 2) encoding an acetyl-transferase with a putative ATPase domain, shown to be enriched in the L3 of <span class="yellow">A. suum</span>, was predicted to interact with 75 other genes all involved in energy production and/or RNA processing (see Table S2). Several molecules involved in ATP synthesis and mitochondrial pathways (e.g., cytochrome oxidase c subunits 1, 2 and 3, ADP/ATP translocases, NADH dehydrogenases, ATPases and ATP synthetases) have been reported previously to be highly represented in the L3 stage of <span class="yellow">Anisakis simplex</span> [65], thus supporting the proposal that substantial energy is required for larval development as well as the transition from the free-living to the parasitic stage and the invasion of the host. There is also likely to be a substantial energy requirement for muscle contraction linked to larval motility in <span class="yellow">A. suum</span>, as the L3s penetrate the caecal wall in the porcine host, before undergoing hepato-pulmonary migration [35]. Accordingly, genes encoding a specialized tubulin expressed in mechanoreceptors (mec-12, EST code 13E09) and a troponin (mup-2, EST code 01G03; see Table 2), both predicted to interact with a total number of 32 tubulin- and myosin-encoding genes, also supported a link to extensive muscle contraction and motility in <span class="yellow">A. suum</span> L3s. Also, neuroactive peptides are required to regulate the responsiveness of nematode larvae to mechanical stimuli [66]. A homologue encoded by egl-3 was shown to be highly transcribed in the L3 of <span class="yellow">A. suum</span>; EGL-3 is predicted to be a pro-hormone convertase involved in the maturation of neuropeptides and could be associated with mechano-sensory responses and touch sensitivity linked to the host invasion.
A regulatory subunit of a cAMP-dependent protein kinase (kin-2, EST code 22H01; see Table 2) was predicted to interact with 72 other genes all involved in diverse cellular processes, such as nuclear trafficking, and DNA replication and repair (see Table S2). Based on gene ontology terms, kin-2 is implicated in gamete generation, growth, larval development, post-embryonic body morphogenesis, signal transduction and/or protein amino acid phosphorylation (see Table S2). Gene silencing of kin-2 in <span class="yellow">C. elegans</span> leads to phenotypes, such as larval lethal (Lvl), larval arrest (Lva), body morphology defect (Bmd), dumpy (Dpy), uncoordinated (Unc) and sterile progeny (Stp) (www.wormbase.org), suggesting that its homologue in <span class="yellow">A. suum</span> is central to larval maturation. The KOBAS analysis predicted the protein KIN-2 to be involved in the insulin-signaling pathway, previously implicated in controlling the exit from dauer in <span class="yellow">C. elegans</span> and the activation of L3s of the <span class="yellow">canine hookworm</span>, <span class="yellow">Ancylostoma caninum</span>, following exsheathment [67]. In a recent study, Brand and Hawdon [68] were able to inhibit (with a phosphoinositide-3-OH-kinase inhibitor) the activation of infective L3s of both of the hookworms <span class="yellow">Ancylostoma caninum</span> and <span class="yellow">Ancylostoma ceylanicum</span> via the insulin signaling pathway, thus lending some credence to the hypothesis that this pathway plays an critical role in regulating the transition from the free-living to the parasitic stage [68]. Recently, it has been proposed that transcriptional and feeding responses to serum-stimulation in <span class="yellow">Ancylostoma caninum</span> are regulated by parallel systems, with the insulin signaling pathway playing a significant role in the ‘resumption of feeding’ in activated larvae [69].
Protein kinases are also likely to be involved in pathways linked to sexual maturation in developing larvae. As already proposed for adult stages of <span class="yellow">H. contortus</span> [45], the protein kinase gene cdk-1 is predicted to play a pivotal role in the germline, oogenesis and spermiogenesis pathways of this parasitic nematode. Other protein kinases, such as PEPCK, and phosphatases, were shown herein to be transcribed at high levels in the L3 stage compared with other developmental stages of <span class="yellow">A. suum</span> (see Table 2), which is in accordance to findings reported recently for <span class="yellow">Anisakis simplex</span> [65]. Due to their major regulatory effects in eukaryotic signaling events and regulatory and sensory functions, protein kinases have been considered interesting targets for anti-parasitic drugs [70].
In conclusion, this study has given some interesting insights into early molecular processes in the L3 of <span class="yellow">A. suum</span>. Approximately 60% of the transcripts enriched in the L3 stage of <span class="yellow">A. suum</span> have homologues/orthologues in <span class="yellow">C. elegans</span>. The bioinformatic analyses of selected molecules suggest that a complex genetic network regulates or controls larval growth and development in <span class="yellow">A. suum</span> L3s, and some of these might be involved in or regulate the switch from the free-living to the parasitic stage. Some caution is warranted in drawing conclusions regarding molecular mechanisms regulating the transition to parasitism in parasitic nematodes from information on <span class="yellow">C. elegans</span>, as latter is a free-living nematode. Also, while the method of data integration is essential for the reliable prediction of genetic interactions, it might limit the capacity of the approach somewhat to infer nematode-specific interactions. As additional datasets of genes and gene functions become available for various parasitic nematodes, more informed inferences can be made regarding the functions of nematode-specific genes, particularly those involved in the transition to parasitism. The imminent genome sequence of <span class="yellow">A. suum</span> (http://www.sanger.ac.uk/Projects/Helminths/) should all assist in this endeavour. Also, functional analysis of selected molecules representing selected ESTs identified herein, utilizing gene silencing approaches established recently [33],[34], could provide some insights into developmental processes in Ascaris and related ascaridoid nematodes and provide avenues for the development of novel approaches for their control.<br><br>Supporting Information<br><br>
<h3>pmcA335194</h3>The <span class="yellow">Caenorhabditis elegans</span> genome contains monomorphic minisatellites and simple sequences.
Abstract
Many species have been shown to contain tandemly repeated short sequence DNA known as minisatellites and simple sequence motifs. Due to allelic variation in the copy number of the repeat unit these loci are usually highly polymorphic. Here we demonstrate the presence of sequences in the genome of the nematode <span class="yellow">Caenorhabditis elegans</span> which are homologous to two sets of short sequence DNA. However, when two independent strains were compared no polymorphism for these sequences could be detected.Images<br><br><br><br><br><br>
 Volume 17 Number 23 1989                                   Nucleic Acids Research <br><br> The <span class="yellow">Caenorhabds elegans</span> genome contains monomorphic minisatellites and simple sequences Andrd G.Uitterlinden, P.Eline Slagboom, Thomas E.Johnsonl and Jan Vijg <br><br> Department of Molecular Biology, TNO Institute for Experimental Gerontology, PO Box 5815, 2280 HV Rijswijk, The Netherlands and lInstitute for Behavioral Genetics, University of Colorado, Boulder, CO 80309, USA <br><br> Received October 27, 1989; Accepted November 3, 1989 <br><br> ABSTRACT <br><br> Many species have been shown to contain tandemly repeated short sequence DNA kinown as minisatellites and simple sequence motifs. Due to allelic variation in the copy number of the repeat unit these loci are usually highly polymorphic. Here we demonstrate the presence of sequences in the genome of the nematode <span class="yellow">Caenorhabditis elegans</span> which are homologous to two sets of short sequence DNA. However, when two independent strains were compared no polymorphism for these sequences could be detected. <br><br> INTRODUCTION <br><br> The genome of many species, including lower organisms, contain minisatellite sequences and so-called simple sequence motifs (1,2,3). Due to extensive variation in the number of repeat units, many of these loci have been shown useful as polymorphic markers, e.g. for genetic linkage studies and identification purposes. <br><br> Here we demonstrate that the genome of the nematode <span class="yellow">Caenorhabditis elegans</span> contains sequences that are homologous to the minisatellite core sequence 33.6 (1) and to the simple sequence motif (AGC)n. Surprisingly, these sequences did not display polymorphism when two independent <span class="yellow">C. elegans</span> strains, Bristol and Bergerac, were compared. <br><br> MATERIALS AND METHODS Genomic DNA from nematodes <br><br> Genomic DNA was isolated according to standard procedures from the two strains Bergerac (BO) and Bristol (strain N2). These strains are derived from two different individual <span class="yellow">worms</span> isolated in France and England, respectively. Southern blotting <br><br> Genomic DNA digests (5 Ag) were separated in a 1 % agarose gel in 1 x TAE (40 mM Tris.HCl, pH 7.4/20 mM sodium acetate/I mM NaEDTA) by electrophoresis at 75 V for 16 h. Separation patterns were transferred to Zetaprobe membrane (BIORAD) in 0.4 N NaOH, 0.6 M NaCl in a vacu-blot apparatus (LKB) according to the manufacturer's instructions. The minisatellite and simple sequence probes used were prepared essentially as described (5). The Tcl probe is a plasmid containing an insert of the transposable element Tcl (4). All probes were labelled by random-priming. Hybridization was performed for 12 h in 7 % SDS, 0.5 M phosphate buffer, 1 mM Na2EDTA at 650C. Blots were washed twice in 2.5 x SSC at 650C and exposed to Kodak XAR-5 film with intensifying screens. Exposure times are indicated in Fig. 1. <br><br> ( IRL Press <br><br> Nucleic Acids Research <br><br> Volume 17 Number 23 1989 <br><br> 9527 <br><br> Nucleic Acids Research <br><br> <span class="yellow">C. elegans</span> <br><br> N      0$       8 <br><br> _ _ <br><br> :  , s , ..e :*. ,~ :::::<br><br> . 4i <br><br> .t_:  -::?J?.: <br><br> i-  <br><br> .a <br><br> E?:< . :: y : <br><br> *01 <br><br> 4?j* <br><br> .-I <br><br> kb - 27 <br><br> - 9A - <br><br> : .::....... <br><br> Mi ....  - 4v3 <br><br> IC .3 <br><br> If.0r <br><br> MW <br><br> !5 c6 <br><br> I <br><br> " R  1d~ ! ;,<br><br> Figure 1. Southern hybridization analysis of Hae III, Rsa I and Eco RI digested genomic DNA isolated from the two <span class="yellow">C. elegans</span> strains Bristol (N) and Bergerac (B). The probes used and the exposure times of the autoradiograph are indicated below the figure. kb=kilo basepairs. 9528 <br><br> *:..?.;. .*. i...  :". <br><br> ANN& <br><br> 'ROW    :. lwmwww. <br><br> .Ammkw.l <br><br> 4w, <br><br> 9 <br><br> _       i ob <br><br> Ill       a <br><br> Nucleic Acids Research <br><br> RESULTS <br><br> In Figure 1 the hybridization patterns are shown of <span class="yellow">C. elegans</span> genomic DNA digested with Hae II, Rsa I and Eco RI, and subsequently hybridized to minisatellite core probe 33.6, the simple sequence probe (AGC)n and Tcl. The latter probe contains a member of the Tcl family of transposable elements which is present in different copy numbers in the two different strains (4). The hybridization patterns of probes 33.6 and (AGC)n obtained after Hae Im and Rsa I digestion and the differences in intensities of the hybridizing bands is reminiscent of DNA-fingerprint patterns obtained with these probes in other species (1,2,3). However, identical hybridization patterns for the two strains with the three enzymes tested indicated that in <span class="yellow">C. elegans</span> these sequences are monomorphic (Fig. 1). Based on the number of hybridizing bands we estimate the <span class="yellow">C. elegans</span> genome to contain about 30 33.6 homologous loci and about 20 (AGC)n homologous loci. <br><br> Rehybridization of the same blot with a Tcl probe revealed extensive RFLPs due to the different copy number of the transposon in the two strains, a phenomenon which has been described previously by others (4). <br><br> DISCUSSION <br><br> The findings presented in this paper indicate that polymorphism of minisatellite sequences and simple sequence motifs, is not a general phenomenon in animal species. So far, only some species of whales have displayed similar high levels of monomorphism (6). In other species thus far tested both minisatellites and simple sequences display high to very high levels of polymorphism. It should be noted, however, that at least in <span class="blue">humans</span>, a substantial part of the minisatellites detected by core probes also displays high levels of monomorphism as analysed by cloning (1) or by two-dimensional DNA fingerprinting (5). <br><br> The fact that nematodes are hermaphrodites, and thus inbred, might be a contributing factor to the observed lack of polymorphism. However, the high levels of polymorphism detected by the Tcl probe do not indicate a general absence of events causing genetic variation. Indeed, Eide and Anderson (7) showed that tandemly repeated duplications in the unc-54 gene of <span class="yellow">C. elegans</span> revert at high frequencies. Since in all cases the revertants had the normal genomic configuration this suggests that unequal crossing-over does occur in the nematode. <br><br> A more likely explanation for the monomorphic nature of the sequences detected with 33.6 and (AGC)n in <span class="yellow">C. elegans</span> is selection against sequence variants at these loci. This might be the result of the presence of a particular subset of minisatellites and/or simple sequences at sites in the genome of this organism, e.g. in coding sequences, in which variation in copy number of repeat units cannot be tolerated. An example of such a coding sequence could be the High Mobility Group proteins which usually have stretches of identical (acidic) amino acids (2,8). An interesting observation in this respect is the demonstration of the absence of any protein polymorphisms in electrophoretic comparisons for 24 different enzymes between the Bristol and Bergerac strains (9). An important step in understanding this phenomenon will therefore be the isolation and analysis of individual homologous minisatellite and simple sequence loci from a genomic library of <span class="yellow">C. elegans</span>. <br><br> REFERENCES <br><br> 1. Jeffreys, A.J., Wilson, V., and Thein, S.L. (1985) Nature 314, 67-73. 2. Tautz, D., Trick M., and Dover, G.A. (1986) Nature 322, 652-656. <br><br> 3. Rogstad, S.H., Herwaldt, B.L., Schlesinger, P.H., and Krogstad, D.J. (1989) Nucleic Acids Res. 17, 9, 3610. 4. Emmons, S.W., Yesner, L., Ruan, K. and Katzenberg, D. (1983) Cell 32, 55-65. <br><br> 9529 <br><br> Nucleic Acids Research <br><br> 5. Uitterlinden, A.G., Slagboom, P.E., Knook, D.L., and Vijg, J. (1989) Proc. Natl. Acad. USA 86, 2742-2746. 6. Rus Hoelzel, A., and Amos, W. (1988) Nature 333, 305. 7. Eide, D., and Anderson, P. (1985) Genetics 109, 67-79. <br><br> 8. Pentecost, B.T., Wright, J.M., and Dixon, G.H. (1985) Nucleic Acids Res. 13, 4871-4888. 9. Butler et al. (1981) J. Molec. Evolution 18, 18-23. <br><br> 9530 <br><br> This article, submitted on disc, has been automatically <br><br> converted into this typeset format by the publisher. </body></html>