
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>human</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="blue">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="blue">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="blue">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1131934</h3>HP1 modulates the transcription of cell-cycle regulators in <span class="yellow">Drosophila melanogaster</span>
Abstract
Heterochromatin protein 1 (HP1) was originally described as a non-histone chromosomal protein and is required for transcriptional gene silencing and the formation of heterochromatin. Although it is localized primarily at pericentric heterochromatin, a scattered distribution over a large number of euchromatic loci is also evident. Here, we provide evidence that <span class="yellow">Drosophila</span> HP1 is essential for the maintenance of active transcription of euchromatic genes functionally involved in cell-cycle progression, including those required for DNA replication and mitosis. Depletion of HP1 in proliferating embryonic cells caused aberrant progression of the cell cycle at S phase and G2/M phase, linked to aberrant chromosome segregation, cytokinesis, and an increase in apoptosis. The chromosomal distribution of Aurora B, and the level of phosphorylation of histone H3 serine 10 were also altered in the absence of HP1. Using chromatin immunoprecipitation analysis, we further demonstrate that the promoters of a number of cell-cycle regulator genes are bound to HP1, supporting a direct role for HP1 in their active transcription. Overall, our data suggest that HP1 is essential for the maintenance of cell-cycle progression and the transcription of cell-cycle regulatory genes. The results also support the view that HP1 is a positive regulator of transcription in euchromatin.<br><br>INTRODUCTION
Chromatin in higher eukaryotes is subdivided into different functional compartments termed heterochromatin and euchromatin (1). Heterochromatin differs from euchromatin in its DNA composition, replication timing, condensation throughout the cell cycle, and its ability to silence euchromatic genes placed adjacent to or within its territory, often described as position-effect-variegation (PEV) (2).
Heterochromatin protein 1 (HP1) was the first protein identified in <span class="yellow">Drosophila melanogaster</span> as a heterochromatin-associated protein (3); the corresponding gene has been cloned from a number of organisms and is highly conserved from <span class="yellow">yeast</span> to <span class="blue">human</span> (4). Polytene chromosome staining showed that, in <span class="yellow">Drosophila</span>, HP1 is distributed mainly in pericentric heterochromatin, telomeric heterochromatin, the banded small fourth chromosome (5–8), as well as ∼200 individual loci scattered throughout the euchromatic chromosomal arms (5). The gene encoding HP1 in <span class="yellow">D.melanogaster</span>, Su(var)2-5, was isolated as a suppressor of PEV (9–11). The protein contains a highly conserved motif, the chromo (chromatin organization modifier) domain, similar to Polycomb (Pc), a repressor of homeotic genes (12). The association between HP1 and pericentric heterochromatin is believed to occur via the chromo domain of HP1 and the N-terminal tail of histone H3 methylated at lysine 9 (13,14), generated by histone methyltransferase-Su(var)3-9, a partner of HP1 in pericentric heterochromatin (15). The C-terminal chromo ‘shadow’ domain of HP1 interacts with other silencing complexes to suppress local transcriptional activity (15–18). However, studies of HP1 chromosomal distribution also showed that HP1 does not always co-localize with lysine 9 methylated histone H3 or Su(var)3-9, especially in euchromatic regions (19–21); in some cases, HP1 is found directly bound to DNA (22,23). All these features argue for distinct roles for HP1 in chromatin and in epigenetic gene regulation.
HP1 is believed to be an essential structural protein protecting the integrity of chromosomes during cell division (8,24). Swi6, the homolog of HP1 in <span class="yellow">fission yeast</span>, is dispensable for survival, but its deletion results in lagging chromosomes during anaphase, and a high rate of chromosome loss (25,26). Mutations of HP1 in <span class="yellow">D.melanogaster</span> result in late larval lethality, chromosome breakages/loss, telomere fusion and a high frequency of cells with abnormal anaphase (8,27). Null alleles of the HP1 functional partner in <span class="yellow">mice</span> (SUVAR39) also showed various chromosomal defects (28), supporting a conserved role for heterochromatin proteins in the regulation of chromosome dynamics during cell-cycle progression. However, the mechanism(s) involved remains to be understood.
In this study, we utilized <span class="yellow">Drosophila</span> embryonic Kc cells and an RNA interference (RNAi)-based approach to demonstrate that HP1 plays an important role at S phase and G2/M phases during the cell cycle. We further show that nearly one-third of known/predicted cell-cycle regulators require HP1 to maintain their active transcription. These genes include MCMs, Orc4, CDC45L, INCENP, Aurora B, CAF1, Bub1, Bub3 and a few other cell-cycle regulators. ChIP analysis suggests that HP1 plays a direct role in their transcription. Therefore, the results of this study provide an alternative explanation for the specific role of HP1 in the regulation of chromatin dynamics and in cell-cycle progression.<br><br>MATERIALS AND METHODS
RNAi in Kc cells
<span class="yellow">Drosophila</span> Kc cells were routinely cultured at 25°C in Schneider <span class="yellow">Drosophila</span> medium (GIBCO) supplemented with 10% fetal <span class="yellow">calf</span> serum, 160 μg/ml penicillin, 250 μg/ml streptomycin, and 4 mM l-glutamine. Double-stranded RNA (dsRNA) of HP1 was generated by incubation of single-stranded RNA in annealing buffer (100 mM potassium acetate, 30 mM HEPES-KOH, pH 7.4, 2 mM magnesium acetate) for 3 min at 95°C and then placed in a beaker with water at 75°C and allowed to cool slowly to room temperature. The detailed procedure of RNAi was carried out according to the established protocols (). Briefly, Kc cells were seeded in a six-well dish using serum-free medium at 1 × 106 cells/ml. HP1 dsRNA (5 μg/ml) was added to the cultured Kc cells. After 60 min at room temperature, 2 ml of medium containing 10% serum was added to each well and the plates transferred to 25°C for up to 8 days. Western blotting and RT–PCR were carried out using the extract/total RNA isolated from control and dsRNA-treated cells on days 2, 6 and 8.<br><br>Cell-cycle and apoptosis analysis
The procedure for flow cytometric analysis of Kc cells followed that in the manual provided with the BrdU flow kit (BD PharMingen). The cells were fed with BrdU for 4 h, then scraped and collected. Fluorescence was measured using a FACSCalibur (Becton Dickinson). Data collection and analysis were performed using CellQuest software.<br><br>Electrophoresis and immunoblotting
Cell extracts (15 μg) were fractionated by 10% SDS–PAGE, then transferred to Hybond-P PVDF membranes (Amersham) and probed with primary antibodies (CIA9), and secondary antibodies (anti-<span class="yellow">rabbit</span> or anti-<span class="yellow">mouse</span> <span class="yellow">horseradish</span> peroxidase-conjugated IgG), obtained from Jackson Immunoresearch Laboratories. Enhanced chemiluminescence reagents (Amersham Pharmacia Biotech) were used for signal detection.
For the analysis of H3 ser10 phosphorylation, we used whole-cell extracts from 700 000 Kc cells (control and RNAi at day 8). Western blotting was performed using polyclonal antibodies against ser10-phosphorylated histone H3 at a dilution of 1:1000 (Upstate). Kc control cells arrested in mitosis by incubation in 25 µM colchicine (Sigma) for 24 h were also analyzed for comparison.<br><br>Immunofluorescence
Kc cells were seeded onto polylysine slides, fixed with 4% formaldehyde for 15 min and permeabilized with 0.5% Triton X-100 for 5 min. The incubation with primary antibodies was carried out in blocking solution for 1 h.
For staining of mitotic cells, the cells were permeabilized using PBST (PBS containing 0.3% Triton X-100) and stained with polyclonal antibody against <span class="yellow">Drosophila</span> Aurora B at 1:200 dilution and monoclonal <span class="yellow">mouse</span> at anti-β-tubulin 1:300 dilution (Chemicon International) as primary antibodies. Secondary antibodies were anti-<span class="yellow">rabbit</span> coupled with Alexa 488 (1:500) and anti-<span class="yellow">mouse</span> coupled to Alexa 546 (1:500) (Molecular Probes, Eugene, Oregon). Images were acquired using a confocal LSM510 META microscope (Zeiss). Stacks of images were analyzed using the IMARIS 4.0 program (Media cybernetics, Carlsbad, CA).<br><br>Antibodies
Affinity-purified polyclonal antibodies of HP1 (<span class="yellow">rabbit</span> #192 and #187, 5 μg) and 5 μg of polyclonal anti-HA antibodies (Sigma) were used in each ChIP reaction. The specificity of the HP1 polyclonal antibodies was determined using various approaches, including western blotting assay, immunofluorescence staining and immunoprecipitation to pull down HP1 (data not shown). The monoclonal antibody HP1–CIA9 (5) was used at a dilution of 1:20 in immunoblotting assays.<br><br>Microarray analysis and RT–PCR
Total RNA was isolated from control and HP1-depleted Kc cells at day 8 using an RNeasy kit (Qiagen). RNA labeling and microarray data analysis followed the standard protocol from Affymetrix. We used ANOVA (P < 0.001) to assess the expression confidence for each gene.
For RT–PCR analysis, poly(A)+ mRNA was purified with the Oligotex Direct mRNA kit (Qiagen) according to the manufacturer's instructions. The purified poly(A)+ RNA was reverse transcribed using the Thermoscript kit (Invitrogen). The cDNA was then used for PCR amplification for 35 cycles with gene-specific primers. PCR products were scanned after electrophoretic separation with a Typhoon Scanner, quantified using ImageQuant software (Amersham Biosciences) and normalized for amplification of the Actin5c transcript. The sequence of primers used for RT–PCR and ChIP analysis are provided in the Supplementary Material.<br><br>ChIP
ChIP was performed according to Orlando et al. (29) and the protocol provided by Upstate () with some modifications. In brief, 1–2 × 108 Kc cells were prepared and fixed in 1% formaldehyde. Nuclei were isolated according to a standard procedure in Current Protocols (), then resuspended in 1.7 ml of lysis buffer (50 mM Tris, pH 8.0, 10 mM EDTA, 1% SDS and protease inhibitors) and sonicated using a Branson sonifier 250. Chromatin fractions in the size range 0.2–0.8 kb were used to perform immunoprecipitation experiments. We used 5 μg affinity-purified polyclonal antibodies (#192 and #182 for HP1; HA antibody for control) and 1 ml of <span class="yellow">salmon</span> sperm DNA/protein-A-agarose (Upstate) pre-cleared chromatin lysate in each reaction. The mixture was then rotated at 4°C overnight and the recovered beads were washed twice with 1 ml of Low salt buffer (Upstate), once with High salt buffer (Upstate), once with LiCl buffer (Upstate) and twice with TE at 4°C for 8 min. ChIP DNA was extracted according to the standard procedures (29).<br><br>
RESULTS
Depletion of HP1 in <span class="yellow">Drosophila</span> Kc cells
Various chromosomal defects in the cell cycle have been observed in embryos or larval tissues of <span class="yellow">Drosophila</span> HP1 mutants (8,27). However, the presence of maternally loaded HP1 in embryos and the lethality of HP1 mutants at late larval stages have so far precluded a systematic study of the role of HP1 in cell-cycle regulation. Therefore, we used <span class="yellow">Drosophila</span> Kc cells, a cell line derived from <span class="yellow">Drosophila</span> embryos, as a model system to address this problem. HP1 transcripts were depleted using an RNAi-based approach (see Materials and Methods). The reduction in HP1 expression was measured both by RT–PCR and by western blotting analysis (Figure 1A). A significant reduction in the HP1 expression was already evident after 2 days treatment with HP1 dsRNA. Cells at day 8 showed a reduction in HP1 of ∼90% (Figure 1A) and were therefore used in all subsequent experiments.<br><br>Cell-cycle progression at S and G2/M phase is altered in the absence of HP1
The impact of HP1 loss on the cell cycle of Kc cells was determined using cell-cycle profile analysis of HP1-depleted and control cells. The percentage of cells in S phase was determined by BrdU incorporation, and total DNA content by 7-amino-actinomycin (7-AAD). The results showed that the depletion of HP1 (day 8) caused a decrease in S-phase cells of at least 4-fold, and a 2-fold decrease in G2/M-phase cells (Figure 1B), although no significant effect was found at the G1 phase. In addition, depletion of HP1 caused a greater than 7-fold increase in the number of apoptotic cells. These results, therefore, confirm that HP1 is an important regulator during the cell cycle, especially at the S and G2/M phases.<br><br>Cell-cycle regulators require HP1 to maintain their active transcription
To ask whether the cell-cycle defects were due to changes in the transcription of genes functionally involved in S phase and the G2/M phase, we next assessed global changes in gene transcription following depletion of HP1. Expression profile analysis was performed using total RNA isolated from both HP1-depleted Kc cells and control Kc cells, and an Affymetrix <span class="yellow">Drosophila</span> chip. For each experiment, we used total RNA isolated from two independent HP1-depleted and control samples, and at least two independent experiments were performed.
The microarray analysis showed that loss of HP1 function in Kc cells resulted in alterations in transcription of >500 genes: ∼400 genes were down-regulated and ∼120 genes were up-regulated (>1.5-fold, ANOVA). The function of these genes ranged from cellular enzymes, signal transduction molecules, and membrane and cell structural proteins, to nucleic acid-binding proteins and cell-cycle regulators (Figure 2A). At the chromosomal level, the genes targeted by HP1 appeared to be distributed along all euchromatic chromosomal arms (data not shown), supporting a global role of HP1 in euchromatic gene regulation (20).
Among 60 known/predicted genes associated with DNA replication function, 15 were down-regulated in the absence of HP1 (Figure 2B). These included McM2, McM5, McM6 and CDC45L, which are required for processive DNA replication and correct chromosome condensation (30–32). Other genes involved in DNA replication, such as components of the origin recognition complex (Orc)—Orc4, Caf1, Gnf1, Dref1, DNA polymerase-γ and Tam—were also down-regulated (Figure 2B). Aurora B and inner centromere protein (INCENP), known to be required for kinetochore assembly, chromosome condensation and bipolar chromosome attachment during mitosis (33), also showed a reduction in transcription. A similar loss of transcription was observed in Bub1 and Bub3 (Figure 2B), encoding mitotic checkpoint control proteins (34,35). Loss function of Bub1 has been shown to cause chromatin bridges to extend between the two separating groups of chromosomes, and extensive chromosome fragmentation in anaphase cells (35).
We confirmed the changes in the transcription of cell-cycle regulators using semi-quantitative RT–PCR, which gave results consistent with the microarray analysis. In addition, cell-cycle regulator genes, such as McM3, McM7 and Asp (abnormal spindle), were also confirmed to be down-regulated (Figure 2C). Collectively, these results demonstrate that HP1 is indeed involved in the regulation of transcription of cell-cycle regulators.<br><br>HP1 is required for Aurora B distribution and histone H3 phosphorylation
INCENP is localized to the centromeric region of chromosomes at metaphase and the spindle midzone at anaphase, which then targets Aurora B, a kinase essential for histone H3 ser10 phosphorylation, to these sites (36). Loss of function of both these ‘chromosomal passenger’ proteins causes abnormal chromosomal segregation at metaphase, as well as certain cytokinesis defects (36,37). The loss of transcription of both INCENP and Aurora B after depletion of HP1, therefore, raised the possibility that localization of Aurora B (Figure 3 and data not shown) may be altered. Staining of HP1-depleted Kc cells with anti-Aurora B antibodies indeed revealed an altered localization of Aurora B and, in a number of cases, a complete loss of Aurora B (Figure 3A). Consistent with the loss function of Aurora B, the spindles in the metaphase cells were also disorganized, with a large number of cells showing an altered prometaphase chromosome alignment (Figure 3A). Some showed extensive chromosome fragmentation (Figure 3B), or the presence of a third spindle pole-like structure as indicated by beta-tubulin (Figure 3A). At telophase, we observed defective separating cells with an extra cell envelope-like structure without nuclei (Figure 3A). Chromatin bridges or lagging chromatids at telophase were also evident in some cells (Figure 3C); however, in some cases, localization of Aurora B appeared not to be affected, arguing that other pathways are possibly involved.
We next analyzed changes in histone H3 serine 10 phosphorylation, since the loss of transcription of INCENP is known to affect localization of Aurora B (33), which is essential in the regulation of histone H3 phosphorylation (36). Total cell extracts from HP1-depleted Kc cells were analyzed by western blotting (Figure 3D). The results indeed showed a severalfold reduction in H3 ser10 phosphorylation after depletion of HP1, consistent with the functional disruption of INCENP and Aurora B in the absence of HP1.<br><br>HP1 directly targets genes encoding cell-cycle regulators in euchromatin
To test whether the loss of transcription of genes involved in DNA replication and mitosis was a direct effect of the loss of HP1, we performed a ChIP analysis to determine whether HP1 is physically associated with these genes. Chromatin lysates from formaldehyde-fixed Kc cells were sonicated into small chromatin fragments (0.2–0.8 kb) and immunoprecipitated with polyclonal antibodies against <span class="yellow">Drosophila</span> HP1. As a control, we used a mock precipitation (beads only) and polyclonal antibodies against HA. Our ChIP results showed that known transposable elements distributed in heterochromatin, such as F-element, TART and 1360 (7,38), were all enriched in HP1 binding (Figure 4), which is also consistent with a previous study (20).
Using the same ChIP DNA material, we then attempted to determine whether HP1 was enriched in genes involved in DNA replication. Primers were designed to cover the promoter regions of selected genes. The results showed that McM3, McM5 and Tam were all enriched in HP1 binding (Figure 4). However, McM7 appeared to be HP1-negative, although its transcription was also affected by the loss of HP1 function. Genes essential for mitosis, such as Aurora B, were also HP1-positive (Figure 4). These results demonstrate that these cell-cycle regulator genes are directly targeted by HP1 in their promoter regions.<br><br>
DISCUSSION
In this study, we used microarray and RT–PCR techniques to demonstrate that transcription of cell-cycle regulators is misregulated in the absence of HP1. Certain defects in S phase may be a direct consequence of the loss of transcription of DNA replication genes such as McM2, McM5, McM6, CDC45L, Orc4 and others, since these genes have been functionally implicated in the initiation of DNA replication and/or the progression of replication forks (39). Depletion or mutation of these genes has been shown to result in DNA damage (32), the blockage of replication forks (39), increased chromosome loss/genome instability, and defective condensation (30).
The reduction in the number of cells in G2/M phase may be a consequence of the reduction in transcription or functional disruption of INCENP, Aurora B, Bub1, and Bub3 (34,36). Chromosome segregation defects, such as chromosome fragmentation and chromatin bridges in anaphase/telophase cells, and certain cytokinesis defects in HP1-depleted cells, mimic the phenotype of cells with loss function of INCENP, Aurora B or Bub1 (35–37). The mislocalization of Aurora B in the absence of HP1 is also consistent with the loss of transcription and functional disruption of INCENP (37), and the reduction in Aurora B transcription may be partially responsible for the observed chromosomal defects, including loss of histone H3 phosphorylation at serine 10.
HP1 is also known to physically interact with certain components of replication complexes such as ORCs and MCMs (30,40,41), with the inner centromere protein INCENP (42) and the chromatin assembly factor CAF1 (43) promoting delivery of HP1 to heterochromatin sites (44). Loss of HP1 is, therefore, expected to cause disruption to such HP1-associated complexes, and will partially contribute to the chromatin/chromosomal defects in HP1 mutants (8,27) and HP1-depleted Kc cells. It is therefore well possible that the loss of transcription of these cell-cycle regulator genes, and consequent disruption of HP1 functional complexes or heterochromatin structure, all contributed to the cell-cycle defects observed.
The ChIP assay supports the hypothesis that the loss of transcription of cell-cycle regulator genes is a direct effect of the lack of HP1. Aurora B, McM3, and McM5 were all bound by HP1 at their promoter regions, although other cell-cycle regulators, such as McM7, were HP1-negative, implying that the altered transcription in these genes might be a secondary effect of the loss of HP1.
A previous study in <span class="yellow">Drosophila</span> Kc cells (20) employed an approach based on the ectopic expression of a fusion protein of HP1 with a prokaryotic DNA adenine methyltransferase and identified a number of methylated targets in the genome. In this study, MCM3 and MCM5 were not found to be methylated, indicating lack of association with HP1. On the other hand, heterochromatin repeats, such F-element and 1360, were consistently found to be HP1-enriched both here and in the previous study. It remains to be determined whether these discrepancies are due to the different experimental systems used. However, we note that the previous study was performed using a cDNA array, while we observe binding of endogenous HP1 at the promoter of these genes. Similarly, another study using chromatin immunoprecipitation in larvae also showed few HP1-positive genes that were not detected in Kc cells by the Dam ID approach (21).
A large number of genes affected by the loss of <span class="yellow">D.melanogaster</span> HP1 in larval tissues (21) seem to be different from that in embryonic Kc cells. The change in the transcription of Aurora B and few cell-cycle regulators reported in this study is also not found among the HP1-affected genes at larval stage (21). This may be due to specific role(s) of HP1 in different stages of development. Alternatively, it is also possible that the impact of HP1 in the transcription of cell-cycle regulators in proliferating cells is underestimated when performing the analyses on larval tissues, and thus on mixed populations of both proliferating and differentiating/differentiated cells.
HP1 is generally known as a transcriptional repressor, as supported by several lines of evidence: silencing of a euchromatic reporter gene in heterochromatin requires HP1 (10,11), tethering of HP1 next to a euchromatic reporter gene causes silencing (45), and the repression of genes within euchromatic region 31 bound by HP1 is relieved in the absence of HP1 (46). In contrast, genes in heterochromatin, known as heterochromatic genes, such as light and rolled, seem to require HP1 to maintain their active transcription (47,48). The level of transcription of heterochromatic genes was dramatically reduced in a mutated HP1 background (47–49). It was therefore proposed that HP1 may function as a positive regulator of transcription of these genes (50,51), although the exact regulation mechanism remains unclear. A study of heat-shock genes found that HP1 is associated with RNA transcripts in the coding region, and is also a positive regulator of their transcription (52,53). The chromatin association of HP1 at the promoter region of active euchromatic genes demonstrated from this work and others, and its independence from histone H3K9 methylation (21), all suggest that mechanism whereby HP1 modulates transcription of euchromatic genes is potentially distinct from its role in heterochromatin formation.
Collectively, the results of this study demonstrate that HP1 plays an essential role in cell-cycle progression, and support the view that HP1, in addition to its role in heterochromatin, can act as a positive transcriptional regulator of euchromatic genes.<br><br>SUPPLEMENTARY MATERIAL
Supplementary Material is available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA1190194</h3>Structural organization and interactions of transmembrane domains in tetraspanin proteins
Abstract
Background
Proteins of the tetraspanin family contain four transmembrane domains (TM1-4) linked by two extracellular loops and a short intracellular loop, and have short intracellular N- and C-termini. While structure and function analysis of the larger extracellular loop has been performed, the organization and role of transmembrane domains have not been systematically assessed.<br><br>Results
Among 28 <span class="blue">human</span> tetraspanin proteins, the TM1-3 sequences display a distinct heptad repeat motif (abcdefg)n. In TM1, position a is occupied by structurally conserved bulky residues and position d contains highly conserved Asn and Gly residues. In TM2, position a is occupied by conserved small residues (Gly/Ala/Thr), and position d has a conserved Gly and two bulky aliphatic residues. In TM3, three a positions of the heptad repeat are filled by two leucines and a glutamate/glutamine residue, and two d positions are occupied by either Phe/Tyr or Val/Ile/Leu residues. No heptad motif is apparent in TM4 sequences. Mutations of conserved glycines in <span class="blue">human</span> CD9 (Gly25 and Gly32 in TM1; Gly67 and Gly74 in TM2) caused aggregation of mutant proteins inside the cell. Modeling of the TM1-TM2 interface in CD9, using a novel algorithm, predicts tight packing of conserved bulky residues against conserved Gly residues along the two helices. The homodimeric interface of CD9 was mapped, by disulfide cross-linking of single-cysteine mutants, to the vicinity of residues Leu14 and Phe17 in TM1 (positions g and c) and Gly77, Gly80 and Ala81 in TM2 (positions d, g and a, respectively). Mutations of a and d residues in both TM1 and TM2 (Gly25, Gly32, Gly67 and Gly74), involved in intramolecular TM1-TM2 interaction, also strongly diminished intermolecular interaction, as assessed by cross-linking of Cys80.<br><br>Conclusion
Our results suggest that tetraspanin intra- and intermolecular interactions are mediated by conserved residues in adjacent, but distinct regions of TM1 and TM2. A key structural element that defines TM1-TM2 interaction in tetraspanins is the specific packing of bulky residues against small residues.<br><br><br><br>Background
Tetraspanins constitute a large family of integral membrane proteins, characteristically containing 4, 6 or 8 conserved cysteine residues in the large extracellular loop (including the CCG and PxxCC motifs), which form disulfide bonds, and several conserved polar residues in the intracellular loop and transmembrane regions [1,2]. There are 32 putative tetraspanin family members in mammals, 37 in <span class="yellow">Drosophila melanogaster</span> and 20 in <span class="yellow">Caenorhabditis elegans</span>. Tetraspanins play diverse roles in cell adhesion, migration and fusion processes, cellular activation and signaling (reviewed in refs. [2-4]). Mammalian tetraspanins such as CD9, CD63, CD81, CD82, CD151, rds/peripherin, and uroplakins Ia and Ib have been most extensively studied, with <span class="yellow">mouse</span> knock-out models available for CD9 [5-7], CD81 [8,9], CD151 [10] and a few others. However, the majority of tetraspanins are characterized very little, if at all, at genetic, biochemical or structural levels.
The large extracellular loop (LEL) of tetraspanins has received most attention, since it contains functionally important sites. Sequence QRD (194–196) in CD151 is important for association with integrins, which has functional consequences for integrin-dependent cell spreading and multicellular cable formation [11]. A site in the LEL of CD9, SFQ (residues 173–175), is essential for CD9 function in sperm-egg fusion [12]. The crystal structure of tetraspanin CD81 LEL revealed five α-helixes, A-E [13]. Helices A, B and E form a relatively conserved region in tetraspanins, whereas the region between helices B and E is the most variable [14]. Interestingly, the variable region contains most of the functionally important sites involved in tetraspanin protein-protein interactions.
A remarkable biochemical property of tetraspanin molecules is their ability to associate with a large number of other transmembrane proteins, including integrins, membrane-associated growth factors and receptors, MHC class II molecules, Ig superfamily proteins, and each other [2,3,15]. Several of these lateral associations of tetraspanins are detected in "mild" detergents (Brij series, CHAPS), but are disrupted by "strong" detergents such as Triton X-100 or SDS. Multiprotein complexes of tetraspanins and associated molecules, also called the "tetraspanin web" [16], may represent a distinct tetraspanin-enriched membrane microdomain [17,18]. The formation of this microdomain is influenced by palmitoylation of several conserved juxtamembrane cysteine residues in tetraspanins [19-21].
The transmembrane domains, encompassing nearly half of a tetraspanin protein, are the most conserved part of the molecule (Stipp et al. [1] and this study). However, very little functional information is available on these domains. The differential detergent sensitivity of tetraspanin-tetraspanin associations suggests that hydrophobic interactions between TM helices may play a role. Indeed, when the large extracellular loop (LEL) of CD151 is deleted, the molecule is still able to associate with other tetraspanins [22]. Thus, TM domains are strong candidates for mediating tetraspanin-tetraspanin interactions.
The importance of TM domain interactions in intramolecular organization was demonstrated in a study showing that CD82 fragment TM2-4, lacking TM1, was retained in the endoplasmic reticulum, but was transported to the cell surface upon co-expression of TM1 [23]. This in vivo reconstitution experiment demonstrated a strong interaction between TM1 and the rest of the molecule. Expression of a truncated CD9 molecule (TM3-LEL-TM4) results in intracellular accumulation of the protein and significant misfolding of the LEL, as judged by inappropriate disulfide formation and diminished antibody reactivity (our unpublished data). Similarly, a CD9 epitope in the LEL is lost in molecules lacking either TM2+TM3 or just TM4 [24]. Thus, TM domain interactions and packing are crucial for proper folding, stability and transport of tetraspanin molecules.
In a previous study, we showed that covalent cross-linking of membrane-proximal cysteine residues can be used as a tool for detection of tetraspanin-tetraspanin associations [25]. Inhibition of cysteine palmitoylation by 2-bromopalmitate (2-BP) made cysteines available for cross-linking and enabled demonstration of specific tetraspanin homodimerization and low levels of heterodimerization. We concluded that tetraspanin homodimers, formed in the Golgi, may be a fundamental structural unit within tetraspanin microdomains.
In this study, we carried out detailed sequence analysis of <span class="blue">human</span> tetraspanin TM domains. We show that a heptad repeat containing conserved glycine, asparagine and large hydrophobic residues occurs in TM1 and TM2 domains, and predict tight intramolecular association of these two domains by packing of the large residues against the small residues. Moreover, by using cysteine cross-linking we map a dimerization interface in the <span class="blue">human</span> CD9 protein, and show that conserved heptad motif glycine residues are also important for intermolecular CD9 associations.<br><br>Results
Sequence analysis of tetraspanin transmembrane domains: presence of the heptad repeat motif
We focused our attention on 28 <span class="blue">human</span> tetraspanins identified from the SWISS-PROT and GenBank databases. All tetraspanins have in common four hydrophobic stretches (TM domains) of 20–25 residues, and contain highly conserved residues in the second extracellular loop, in particular the Cys-Cys-Gly (CCG) motif. Detailed analysis of the large extracellular loop sequences [14], and dendrograms based on full-length alignment can be found in earlier studies [26,27]. The length of each transmembrane domain was established based on previous sequence analysis of tetraspanin sequences [27,28], and on annotations to the database entries. Manual adjustments based on sequence homology and hydrophobicity profiles were done to fully delineate the TM domains. The resulting lengths of TM domains were: TM1 – 23 residues; TM2 – 21 residues; TM3 – 25 residues; TM4 – 25 residues. Two more residues could be added onto the N-terminal part of TM2; however, relatively small sequence conservation of these residues among tetraspanins and occurrence of polar/charged side chains in some tetraspanins precluded us from doing so for the global alignment.
Figures 1 and 2 show a multiple sequence alignment of four TM domains of 28 <span class="blue">human</span> tetraspanins. For each position within the domains, consensus residues were determined and classified (with individual color code) in 4 categories: 1) large hydrophobic residues (including Val, Met, Leu, Ile, Phe, Tyr, Trp), 2) small residues (Gly, Ala, Ser and Thr), 3) Cys, and 4) Asn. When more than two types of residues occupied a given position in a TM, a dual-color pattern that reflected the prevalence of the particular residue type was used (Figure 1). Cysteine residues were shown separately due to their importance as palmitoylation target sites. The highly conserved asparagine residue in TM1 was considered separately. No proline residues are found in TM domains 1–3 of <span class="blue">human</span> tetraspanins.
An inspection of the multiple sequence alignment reveals a repeating heptad amino acid pattern, (abcdefg)n, in TM1, 2 and 3 (Figure 1, 2). Heptad repeats promote helical coiled coil interactions in multiple soluble and membrane-spanning proteins [29-31]. In the heptad repeat, hydrophobic residues in positions a and d are of special importance, as they directly mediate interhelical contacts by creating a tight knobs-into-holes packing in the coiled coil structure [32]. For instance, in the leucine zipper of the <span class="yellow">yeast</span> transcription factor GCN4, positions a and d contain Val and Leu residues, respectively, with an Asn residue in a single a position forming a hydrogen bond across the GCN4 dimer interface [33].
In TM1 of tetraspanins, highly conserved Asn, Gly and Gly residues (numbers 18, 25 and 32 in the CD9 sequence) appear at d positions of the heptad repeats, and residues 14, 21 and 28 are at a positions (Figure 1). In TM2, residues 67, 74 and 81 (consensus Gly, Gly and Ala, respectively) occupy a positions, whereas residues 63, 70 and 77 are at d positions. Another highly conserved glycine, Gly80, occupies the 3rd g position in TM2. In TM3, the conserved pattern consists of two leucine residues (Leu89 and Leu96) and a glutamate/glutamine residue (Glu/Gln103) in a positions (Figure 2). Two d positions are also conserved – Phe/Tyr92 and Ile/Val/Leu99. TM4 lacks a conserved heptad pattern and has only a single conserved position, Glu/Gln209 (with four exceptions). These features of TM1-4 of tetraspanins are displayed on helical wheel diagrams (Figure 3).<br><br>Analysis of TM1 sequences
The conserved Asn-Gly-Gly motif, occupying designated d positions of the heptad repeat, is the most prominent structural feature of TM1. We also compared sequences of CD9 orthologs from 10 different organisms (the most available for any tetraspanin) to gain further insight into conservation and variability of the TM1 sequence. As shown in Figure 4, positions a, d and g in TM1 are among the most conserved (0, 1 and 1 substitution, respectively), while interspecies variability tends to occur in other positions: b (5 substitutions), c (4 substitutions), e (4 substitutions) and f (4 substitutions). Thus, the positions typically involved in coiled coil interactions (a and d) are the most conserved.
When residues of TM1 are plotted as a helical wheel, additional structural features are revealed (Figure 3). There are highly conserved aliphatic and aromatic residues in the first three a positions of the heptad motif (Phe15, Trp22 and Leu29 in CD9), as well as in g positions (Leu14, Phe21, Val28 in CD9). The "ridges" formed by these bulky residues are flanking the "groove"-forming Gly residues of the Asn-Gly-Gly position d motif. In contrast, b, c, e and f positions show an overall higher variability among tetraspanins, as also seen in the comparison of CD9 orthologs described above.<br><br>Analysis of TM2 sequences
A landmark feature of TM2 in tetraspanins is the presence of highly conserved glycine residues (Gly67, 74, 77 and 80 in CD9, Figure 1). Other substitutions at these positions are almost exclusively small residues, such as Ala or Ser. In addition, Ala, Ser or Thr occupy position 81. This residue, together with Gly67 and Gly74, forms face a of the helix. Residue Gly77 (position d) is preceded by conserved, chiefly large hydrophobic residues on the same helical face (Leu63 and Met70 in CD9). Extremely conserved Gly80 falls into heptad position g (Figure 3). Among CD9 orthologs, heptad positions a and d are absolutely conserved, whereas other positions have the following number of substitutions: b – 3; c – 2; e – 1; f – 3; g – 1 (Figure 4). Two of the f position residues in TM2 (65 and 79) also show higher variability among different tetraspanins (Figures 1, 3). Cysteine residues are frequently found near the cytoplasmic end of TM2 helix at positions 78 and 79; these cysteines are likely to be palmitoylated.<br><br>Analysis of TM3 and TM4 sequences
The TM3 domain provides another example of the heptad repeat pattern. Position a is occupied by two highly conserved leucine and a glutamate/glutamine residue (Leu89, Leu96 and Glu/Gln103 in CD9). Furthermore, two d positions are conserved – Phe/Tyr92 (aromatic residue) and Ile/Val99 (β-branched aliphatic residue; Figures 2, 3). In addition, residue 100 in position e is generally Phe or Leu. Among CD9 orthologs, position a has 1 substitution, positions b, c and f each have 6, positions d and e each have 2, and g has 4. Thus, as for TM1 and TM2, positions a and d are among the most conserved, but overall TM3 has more variable positions than TM1 or TM2 (Figure 3). Less than half of TM3 sequences contain cysteine residues, and those tend to occur at the internal positions of the helix (Figure 2).
TM4 shows less conservation among various tetraspanin family members than the other TM domains (Figures 2, 3). The only highly conserved feature is the glutamate/glutamine residue in position 209. In addition, one or two cysteine residues can be found at the C-terminal end of TM4 in some tetraspanins (e.g. CD9, CD81, CD151), and many sequences contain additional polar residues (Arg, Lys, His, Asn, Gln). No conserved heptad motif was identified in TM4, as also confirmed by analysis of substitutions in CD9 orthologs (data not shown).<br><br>Mutational analysis of conserved glycine residues in TM1 and TM2
The conserved nature of the Asn and Gly residues in TM1 and TM2 prompted an analysis of their functional role. To this end, we have probed whether mutations of these residues destabilize the protein molecule. We expressed a construct of the first and second TMs of CD9, connected by the small extracellular loop, and tagged with a C-terminal green fluorescent protein (TM(1+2)-GFP molecule). In <span class="blue">human</span> rhabdomyosarcoma RD cells, the wild-type fusion protein localized mostly in a reticular, intracellular pattern, without forming any large aggregates (Figure 5, panel A). Remarkably, when double mutants Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu were expressed, the protein formed distinct large aggregates in a high proportion of cells (Figure 5, panels C and E). In contrast, double mutant Gly77Leu + Gly80Leu did not form such aggregates (Figure 5, panel G). Results with respective single mutants were similar to that with double mutants, with the aggregation being somewhat more pronounced for Leu substitutions of Gly67 and Gly74 compared to Gly25 and Gly32 mutations. No aggregation was observed for Asn18Ser and Asn18Tyr mutants (data not shown). Also, nearly identical results were obtained with <span class="blue">human</span> HT1080 cells (data not shown).
We interpret these results as an indication that aggregating mutants are destabilized or misfolded while non-aggregating mutants retain the wild-type conformation. Intriguingly, mutations to the conserved GG7 motifs caused protein aggregation while the mutation of other glycines had no detectable effect. These results also suggest that wild-type GFP, which has weak tendency to self-associate, could enhance non-specific interactions of destabilized mutant TM(1+2) CD9 moieties, leading to their aggregation. Consistent with this hypothesis, the aggregation of Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu double mutants was suppressed when monomeric GFP molecule, Leu221Lys [34] was used (Figure 5, panels D and F). The use of monomeric GFP did not affect intercellular localization of wild-type CD9 TM(1+2) (Figure 5, panel B), or a Gly77Leu + Gly80Leu double mutant (Figure 5, panel H).
In summary, Leu substitutions of Gly residues that are part of the Asn-Gly-Gly (NGG7) motif in TM1, or Gly-Gly-Ala (GGA7) motif in TM2, resulted in destabilization and aggregation of GFP-fused TM(1+2) proteins, whereas substitutions of Gly77 or Gly80, which are not part of these motifs (Figure 3), failed to show such aggregation.<br><br>Prediction and modelling of interaction between TM1 and TM2
Consecutive helices in polytopic membrane proteins frequently interact [35]. Sequence analysis of TM1 and TM2 helices of tetraspanins reveals a remarkable complementarity in the distribution of large and small residues at heptad positions a and d along the helical axis (Figure 3), suggesting that these residues may interact. To further elucidate the potential for TM1-TM2 interaction, the putative interface was modeled using a novel algorithm that considers mutational data during each step of a Monte Carlo simulated annealing cycle (see Methods for details). Specifically, Gly25Leu, Gly32Leu, Gly67Leu and Gly74Leu were scored as disruptive mutations, while Asn18Ser, Gly77Leu and Gly80Leu were scored as silent mutations, based on their effects on protein stability (Figure 5 and data not shown).
The resulting model predicts left-handed crossing of TM1 and TM2 helices at an angle of +28°. The key element of the structure is the apposition of bulky and small heptad position a and d residues, as follows: Gly32-Leu63; Gly67-Leu29; Gly25-Met70; Gly74-Trp22; Asn18-Gly77; Ala81-Phe15 (Figure 6). Our model predicts that each of these residue pairs are in van der Waals contact. Additionally, two potential H-bonds are predicted in this model, indicating close packing: Gly67 Cα to Gly25 carbonyl oxygen, and Trp22 Cα to Met70 carbonyl oxygen. The packing is tighter in the ectodomain-proximal portion of the helices (Figure 6, panel B), as determined by Cα-Cα distances between interacting residue pairs.
The key elements of the model are corroborated by the presence of apparently complementary substitutions in TM1 and TM2 sequences of different tetraspanins (Figure 1, boxed residues). For example, Gly74 is predicted to interact with Trp22. In 8 of the 10 tetraspanins that contain a substitution for Gly74, a compensatory substitution occurs at the Trp22 position (Figure 1). Thus, a larger non-glycine side chain at position 74 may necessitate a less bulky non-Trp side chain in position 22. Likewise, the presence of a Cβ at position 25, typically occupied by glycine, necessitates a non-β-branched amino acid at position 70, which is occupied by a β-branched residue in nearly half of all cases. Indeed, we find that in each of 5 cases in which position 25 contains a Cβ, a leucine residue occurs in position 70. This analysis is consistent with our molecular model that suggests Leu70 will pack most favorably against a Cβ at position 25 than a β-branched residue or a methionine.<br><br>Role of TM1 and TM2 heptad motif residues in CD9 dimerization
To probe CD9 dimerization, we used a cysteine-mediated cross-linking approach. We established previously a simple and efficient method for cysteine-mediated cross-linking [25]. After cells are pre-treated with 2-BP for 16–24 hours to expose normally palmitoylated cysteines, the cysteines can be cross-linked using any of the following methods: a) Spontaneous oxidation in Brij97 lysates (a condition that preserves tetraspanin-tetraspanin associations), b) In situ cross-linking, by pre-lysis oxidation of cells with Cu2+-phenanthroline (CuP) to promote disulfide bond formation. c) In situ cross-linking with thiol-reactive cross-linking agents of defined length (e.g. DTME, BMB). The first two approaches produce in essence "zero-length" disulfides, indicative of close proximity of target cysteines and presumably high specificity of interaction. In contrast, chemical cross-linkers with 6–20 Å spacer arm may cross-link with higher efficiency, but not necessarily higher specificity. However, they provide advantages such as variable membrane permeability, and potential linkage cleavability. For tetraspanins such as CD9, membrane-permeable cross-linker DTME (13.3 Å-long, reducible) provides highly specific and efficient cross-linking [25]. Here we have used a cysteine cross-linking strategy, in combination with cysteine-scanning mutagenesis, to map the residues from TM1 and TM2 contributing to the CD9 dimerization interface.
For subsequent cross-linking experiments using CD9 TM(1+2)-GFP protein, the non-dimerizing form of GFP was used. This avoids potential GFP-dependent dimerization and aggregation that can be observed with wild-type GFP, especially when fusions with transmembrane proteins are studied [36]. Importantly, the Leu221Lys mutation in GFP prevented aggregation of mutant forms of CD9 TM(1+2), which was observed with wild-type GFP fusion (Figure 5). The TM(1+2) fragment of CD9 contains three native cysteines – Cys9, Cys78 and Cys79. Single-cysteine mutants of TM(1+2) were constructed, in which a cysteine was placed at various faces of TM1 or TM2 while all of the wild-type cysteines were simultaneously replaced by serines. The mutant proteins were transiently expressed in RD cells (having little endogenous CD9), which were then treated for 16–18 hours with 2-BP. To achieve maximal specificity in cross-linking we used a "zero-length" agent, CuP.
First, single-cysteine replacements were constructed for residues Leu14, Phe15, Gly16, Phe17 and Asn18, covering just over one complete helical turn at the beginning of TM1. While residue Asn18 is highly conserved, positions 14, 15 and 17 are occupied by bulky hydrophobic residues in most tetraspanins, whereas position 16 shows less conservation (Figures 1, 4). All of the single-cysteine mutants showed diffused pattern of protein localization, without any signs of aggregation. As shown in Figure 7A, the highest level of intermolecular cross-linking was observed for Leu14Cys and Phe17Cys mutants, a lower level for Phe15Cys and Gly16Cys mutants, and very little cross-linking for Asn18Cys substitution. These results indicate that: a) the first two transmembrane domains of CD9 alone can mediate its dimerization, and b) the g and c residues of TM1 (e.g. Leu14 and Phe17, Figure 3) are likely to be part of the intermolecular interface.
Similarly, single-cysteine substitutions were made for residues Gly77, Gly80 and Ala81 in TM2; in addition, proteins carrying a single wild-type cysteine, Cys9, Cys78 or Cys79, were tested. No protein aggregation was observed for any of these single-cysteine mutants. As shown in Figure 7B, the relatively low level of intermolecular cross-linking of wild-type CD9 TM(1+2)-GFP protein was enhanced dramatically in single-cysteine TM2 mutants Gly80Cys and Ala81Cys. The Gly77Cys mutant also had an elevated level of cross-linking. In contrast, any of the three native cysteines (9, 78 and 79) produced level of cross-linking not much greater than the wild-type TM(1+2) protein. Similar results were obtained with cysteine-reactive cross-linker BMB (data not shown). Likewise, comparable results were obtained with single-cysteine mutants of untagged, full-length CD9, using CuP (Figure 7C) as well as DTME cross-linker (data not shown).
These cross-linking results for TM1 and TM2 are consistent with our model that places residues Leu14, Phe17 and Gly80 on the same side of the TM1-TM2 pair (Figure 6, panel C). The strong cross-linking with Leu14Cys, Phe17Cys and Gly80Cys places the intermolecular interface toward the c and g phases of the TM1 helix, and the g phase of the TM2 helix, away from its e and f faces containing wild-type cysteines 78 and 79.
Critical residues at the TM1-TM2 interface also affect dimerization indirectly. To assess specific CD9 dimerization, we used a Gly80Cys substitution at the intermolecular interface for cross-linking. As shown in Figure 8A, single replacements of conserved heptad residues in positions 18, 25, 32, 67 and 74 (Asn18Ser, Gly25/32/67/74→Leu) strongly decreased the cross-linking mediated by Cys80. The effect was most pronounced for mutations of residues, Gly32 and Gly67, located in the tightly packed extracellular end of TM helices (Figure 6). In contrast, mutations of residues closer to the cytoplasmic end of TM2 (Gly74 and especially Ala81) had only modest to very little effect on cross-linking.
Relatively low efficiency of intermolecular cross-linking via native residues Cys9, 78, and 79 (Figures 7B,C) correlates well with the predicted location of Cys78 and 79 away from the dimeric interface (Figure 3), and suggests that the extramembrane N-terminal part of CD9 (residues 1–13) does not self-associate. We next examined whether mutations of conserved Asn and Gly residues in TM1 and TM2 decreased low-level background cross-linking via native cysteines. As expected, these mutations had virtually no effect on dimer formation of CD9 TM(1+2)-GFP (Figure 8B). The level of covalent dimer formed was not diminished for triple Asn18Ser + Gly25Leu + Gly32Leu and double Gly67Leu + Gly74Leu mutants, compared to wild-type TM(1+2) CD9 molecule. Similarly, the same triple and double mutations in the context of full-length CD9-GFP protein (with six cysteines) produced wild-type levels of cross-linking (Figure 8C). We interpret these findings as evidence for at least two types of associations between CD9 molecules: primary, involving residues 14, 17 and 80, and dependent on integrity of conserved heptad residues in TM1 and TM2, and less efficient secondary interactions, probably representing random collision events, and independent of the heptad residues (see Discussion for more details).<br><br>TM3 and TM4 cysteine residues in CD9 dimerization
After identifying the roles of conserved TM1 and TM2 residues in CD9 dimerization, we next probed whether residues proximal to TM domains 3 and 4 are also involved. To this end, disulfide cross-linking of full-length CD9 molecules containing 3 C-terminal cysteines (87, just before TM3; 218 and 219 in TM4) or 3 N-terminal cysteines (9, 78 and 79) was compared (Figure 9). We found that the C-terminal cysteines were only slightly better than N-terminal cysteines with respect to detection of CD9 dimers. However, markedly more trimers and tetramers were detected using C-terminal cysteines. Thus, residues 87, 218 and 219 at TM3 and TM4 in CD9 can together form contacts across the dimeric interface and also additional contacts with other neighboring CD9 molecules.<br><br>
Discussion
Here we provide the first detailed analysis of tetraspanin protein transmembrane domains. First, we show 1) the presence of a heptad repeat motif in TM1 and TM2, containing highly conserved Asn and Gly residues, 2) a leucine and glutamate/glutamine-containing heptad motif in TM3, and 3) high variability and absence of heptad repeats in TM4 sequences. Second, we provide evidence for a specific, intramolecular interaction between TM1 and TM2 domains, in which bulky hydrophobic residues pack against GG7 motif, and present a molecular model for this interaction. Third, experimental mapping of the CD9 dimerization interface firmly establishes an additional role for conserved TM1 and TM2 residues in dimeric intermolecular interactions. Fourth, preliminary evidence is provided to suggest that TM3 and TM4 domains contribute to expansion of CD9 dimers into higher order multimers.
Conserved residues in TM1 and TM2 of tetraspanins: role in intramolecular packing
We hypothesized that the first two transmembrane domains of tetraspanins might interact with each other because: a) consecutive TM domains frequently associate in known protein 3D structures [35], and b) they both contain a series of highly conserved amino acids – several Gly residues and an Asn residue (Figure 1). Conserved Gly residues are a frequent theme in the organization of interacting transmembrane domains. Analysis of 3D helix packing in polytopic membrane proteins reveals that Gly residues tend to localize in buried positions, especially at the helix-helix interfaces and helix crossing points [37,38]. Due to the absence of a side chain, Gly provides a flat surface for packing of a side chain from another residue, without loss of side-chain entropy upon interaction. The most common Gly-containing motif is GxxxG [39,40]. In glycophorin A (GpA), the major glycoprotein in erythrocyte cell membranes, Gly79 and Gly83 are part of the LIxxGVxxGVxxT sequence that promotes homodimerization of parallel transmembrane α-helixes [41,42]. In the GpA dimerization motif, Gly residues allow for tight packing in the right-handed helical crossing [43]. There are also examples of left-handed helical crossing in the context of a GxxxG motif [44]. Other membrane proteins that use the GxxxG motif for homo- or heterodimerization include <span class="yellow">bacteriophage M13</span> coat proteins [45], <span class="yellow">yeast</span> alpha factor receptor [46], integrin α IIb subunit [47], and ErbB1 receptor tyrosine kinase [48]. Other small residues, such as Ala and Ser, can substitute for Gly in this motif [49].
A protein motif in which Gly residues are separated by 6 other residues (GG7) is also common in transmembrane helices, especially in transporter/channel-like membrane proteins [50]. However, the structural features associated with this motif are not well known. In particular, it is unclear whether left-handed GG7 heptad repeat motif (as opposed to the "classic" right-handed GxxxG motif) can drive membrane helix association. In a recent work addressing this issue, Lear et al. [51] showed that a synthetic peptide containing Gly at heptad positions a and d could self-associate in vitro, likely in an antiparallel orientation. Heptad repeats containing conserved Gly residues occur in TM domains of α and β chains of MHC class II proteins, and mutations of the Gly residues disrupt the αβ heterodimer [52]. These examples demonstrate that Gly-based heptad motifs may be used for both intra- and intermolecular associations.
In this work, we identified a highly conserved GG7 motif in the first two tetraspanin TM domains. The GG7 sequence in tetraspanins is a part of a larger motif that also includes a conserved Asn residue in TM1 (NGG7) and an Ala/Ser/Thr residue in TM2 (GGA7). The seven-residue periodicity of these motifs strongly suggests their involvement in left-handed coiled coil packing reminiscent of the leucine zipper, rather than right-handed packing of the GpA-like GxxxG motif. For antiparallel helices, the left-handed crossing is in fact predominant over the right-handed in known TM domain structures [44].
In our model, heptad Gly residues in NGG7 and GGA7 sequences provide specific packing between antiparallel tetraspanin TM1 and TM2 helices by allowing tight van der Waals interactions with large hydrophobic residues (Figure 6). Highly efficient packing of bulky side chains against glycine residues is observed in known transmembrane protein 3D structures [38,53,54]. An example includes packing of helices M1 and M2 in potassium channel KcsA, where Val91 in M2 is paired with Gly43 in M1, and Leu36 in M1 contacts Ala98 and Gly99 in helix M2 [54,55]. In addition to facilitating helix-helix packing, Gly residues frequently provide additional CαH...O hydrogen bonds between two helices [44]. In our model, two Cα-backbone carbonyl H-bonds are predicted – between residues Gly27-Gly67, and Trp22-Met70.
Although polar and charged amino acid residues (such as Asn in the TM1 heptad motif) are infrequent in transmembrane domains, they are functionally important. Polar residues such as glutamine, glutamic acid, aspartic acid and asparagine can promote strong oligomerization of model membrane-associated helices [56-58]. Ruan et al. [59] used asparagine scanning mutagenesis to probe the interface of self-associating polyleucine helices by detecting their enhanced self-interaction in vitro and in the <span class="yellow">E. coli</span>-based ToxR assay. Thus, a hydrogen bond in an apolar environment can result in strong, though not necessarily specific, association of transmembrane helices. In fact, mutations to polar residues in transmembrane proteins are commonly associated with disease [60]. Because of this potential for non-specific interactions, polar residues tend to localize at buried positions in TM domains.
In our case, the conserved Asn18 residue in CD9 is predicted to be a part of the TM1-TM2 interface, though our model does not predict any electrostatic interaction between Asn18 and TM2 (Figure 6). Consistently, substitution such as Asn18Tyr (and Gly77Leu) in TM(1+2)-GFP protein was not destabilizing as analyzed by protein aggregation. Curiously, the full-length Asn18Ser CD9 migrated slightly slower on SDS-PAGE gel (data not shown), suggesting that Asn18 does play a role in maintaining conformation of the molecule. The Asn18Cys single-cysteine mutant shows very little intermolecular cross-linking (Figure 7A), supporting the proposed location of this residue at the intramolecular interface. It is tempting to speculate that the "pocket" between TM1 and TM2 lined by Asn18 and Gly77 might be important for accommodating palmitate moieties that target Cys78 and Cys79 residues, and/or important for access by the putative palmitoyl transferase to those residues. Understanding the exact role of these highly conserved Asn18 and Gly77 residues in tetraspanins awaits further investigation.
In summary, we identified conserved glycine residues of TM1 and TM2 of tetraspanins as key elements required for intramolecular packing. Mutations of these key residues (Gly25, Gly32, Gly67 and Gly74 in CD9) resulted in protein destabilization and aggregation. There is ample evidence in the literature for mutations in transmembrane proteins that lead to protein destabilization, misassembly and pathologic conditions [61]. Thus, we have identified conserved heptad Gly residues in TM1 and TM2 of tetraspanins as plausible targets of destabilizing mutations with potential functional consequences.<br><br>Intermolecular interactions in tetraspanins
Tetraspanin CD9 forms mostly homodimers, but also a low level of heterodimers with CD81 and CD151 [25]. Thus, mapping the dimerization interface is an important next step in structure-function analysis of tetraspanins. Disulfide-mediated cross-linking, often in combination with cysteine-scanning mutagenesis, is a common strategy to probe oligomerization or intersubunit interactions of transmembrane proteins such as histidine kinase EnvZ [62], M(3) muscarinic acetylcholine receptor [63], <span class="yellow">E. coli</span> lactose permease [64], synaptobrevin [65], integrins [66] and many others. In tetraspanins such as CD9, membrane-proximal cysteine residues are especially useful targets for disulfide trapping, as their linkage can be enhanced by pre-treating cells with 2-BP. While the ability of wild-type cysteines in CD9 to be cross-linked may indicate that they are close to the dimerization interface, more precise mapping was achieved here using cysteine-scanning mutagenesis.
Our data clearly identify regions, near the cytoplasmic face of TM1 and TM2, important for dimerization. Intermolecular zero-length cross-linking was highest when single cysteines were placed in positions 14, 17, 77, 80 and 81 in TM(1+2)-GFP molecule, or at positions 77, 80 or 81 in the full-length CD9 protein. Positions 14, 17 and 80 are distinct from the intramolecular interface and are on the same side of the TM1-TM2 pair (Figure 6). Thus, they are well-positioned to participate in an interaction with another molecule. At the same time, the model predicts that wild-type cysteines (Cys78, 79), which do not yield very efficient zero-length cross-linking, are on the other side of TM1-TM2 pair.
While using the cysteine at position 80 as the dimeric interface probe, mutations of conserved residues in TM1 and TM2 (especially Gly32 and Gly67 to Leu) clearly reduced intermolecular cross-linking. We do not suggest that those residues are directly involved in intermolecular interaction. Rather, we propose that destabilization of the intramolecular TM1-TM2 interaction by Gly to Leu substitutions (discussed above) causes an overall conformational change that reduces dimer formation.
An Ala81Leu mutation did not reduce cross-linking via Cys80, even though single-cysteine Ala81Cys molecules themselves produced a high level of cross-linking. These results, together with data on Gly32Leu and Gly67Leu mutations, are consistent with our model predicting that helices 1 and 2 interact more tightly near the extracellular end and less at the cytoplasmic end. This would give more flexibility to a cysteine at position 81 and also limit the effect of an Ala81Leu mutation. Location of this residue at the membrane/cytoplasmic border could also make it more accessible to CuP reagent as compared to residues buried in TM domain, thus elevating the efficiency of disulfide formation of the Ala81Cys mutant.<br><br>Multiple interfaces in tetraspanin molecules
In the full-length CD9 molecules, the 3 C-terminal cysteines (Cys87, 218 and 219) located at or in TM3 and TM4 promoted efficient dimer and even more efficient oligomer formation compared to the 3 N-terminal cysteines (Figure 9). Cys87 alone can be used to capture CD9 dimers [25]. These results suggest the existence of two dimeric interfaces in CD9 molecule – the TM 1-2/1-2 interface and the TM 3-4/3-4 interface (Figure 10). In a TM(1+2) molecule, the destabilization of 1–2 interaction, e.g. by Gly→Leu mutations, would affect the 1-2/1-2 interface, as discussed above. However, these mutations would not interfere with the 3-4/3-4 interface in a full-length molecule, which includes Cys87, 218 and 219. Thus, cross-linking of full-length molecules, containing all 6 cysteine residues, would be unaffected, as seen in Figure 8C. Furthermore, wild-type Cys9, Cys78 and Cys79 are apparently not at the primary 1-2/1-2 interface. Their relative inefficiency in cross-linking CD9 TM(1+2) protein likely reflects weak secondary contacts between the molecules, or possibly random collision events. Such events should be independent of mutations in the conserved Gly residues in TM1 and TM2, as was demonstrated in Figure 8B. The potential existence of two interfaces in tetraspanin molecules, 1-2/1-2 and 3-4/3-4, should provide enhanced flexibility for forming additional intermolecular contacts. Current understanding of tetraspanin microdomains assumes a few strong, primary homotypic and heterotypic tetraspanin complexes (e.g. CD9-CD9, CD9-CD81, CD151-α3 integrin, CD81-EWI2) that help bring together various other proteins, forming secondary-type associations. Such properties of tetraspanins may bring signaling molecules such as protein kinase C or phosphatidylinositol 4-kinase to the vicinity of integrins [67,68].
The organization of the TM3 domain points to a potential role in protein-protein interactions. A motif of Leu-Leu-Glu(Gln) spaced 7 residues apart (heptad positions a), with highly conserved residues in two consecutive positions d, poses as a likely interaction module. If responsible for heterologous protein-protein interactions, it would form another distinct interface of tetraspanin molecule. Our preliminary data indicate that replacing the Leu and Glu residues in TM3 of CD9 with Ala has no effect on cell surface expression of the protein and its dimerization (data not shown). It remains to be tested if interactions with other proteins will be affected. Similarly, the TM4 domain may provide additional contributions to lateral tetraspanin associations. Much higher sequence variability, and the lack of a distinct heptad pattern suggests that TM4 is a major contributor to diversity among tetraspanin complexes. Structure-function analysis of TM3 and TM4 domains in tetraspanins is the subject of ongoing investigation.<br><br>
Conclusion
We have defined the TM1-TM2 intramolecular interface in tetraspanin CD9, providing evidence for glycines (Gly25 and Gly32 in TM1, Gly67 and Gly74 in TM2) packing against apposing bulky aliphatic residues. Second, we mapped an intermolecular CD9 interface (involved in CD9 homodimer formation) to the vicinity of residues Leu14 and Phe17 in TM1 and Gly77, Gly80 and Ala81 in TM2. Finally, we provide preliminary evidence that TM3 and TM4 in CD9 may contribute to a second intermolecular interface. Key CD9 residues involved in intra- and intermolecular interactions are highly conserved throughout the tetraspanin family, thus suggesting that our findings will apply to most tetraspanins.<br><br>Methods
Materials
Cell culture reagents were from Invitrogen (Carlsbad, CA). 2-bromopalmitate was from Fluka (Milwaukee, WI), N-ethylmaleimide (NEM) and 1,10-phenanthroline were from Sigma-Aldrich (St. Louis, MO), and chemical cross-linkers dithio-bis-maleimidoethane (DTME) and 1,4-Bis-maleimidobutane (BMB) were purchased from Pierce Endogen (Rockford, IL). Triton X-100, protease inhibitor cocktail and FuGENE 6 transfection reagent were obtained from Roche (Indianapolis, IN). Restriction endonucleases and Pfu DNA polymerase were obtained New England Biolabs (Beverly, MA) and Stratagene (Carlsbad, CA), respectively. All other chemicals were purchased from Sigma-Aldrich or Fisher Scientific (Pittsburg, PA).<br><br>Sequence analysis
Tetraspanin sequences were obtained from SWISS-PROT and GenBank databases. Locus designations, accession numbers and the most commonly used protein names are summarized in Tables 1 and 2. TM segments were delineated by inspection of hydrophobicity profiles, using database annotations and previous analyses of TM sequences as a guide ([28], M. Hemler, unpublished), and aligned manually. Residue numbers in <span class="blue">human</span> CD9 sequence are used a reference point throughout the study.<br><br>DNA cloning and mutagenesis
Sequence encoding CD9 protein was cloned into vector pcDNA3 (Invitrogen, Carlsbad, CA) and pEGFP-N1 (Clontech, Palo Alto, CA), for expression of untagged and C-terminally GFP-tagged CD9, respectively. pEGFP-N1 encoding CD9 TM(1+2) -GFP fusion protein was constructed by subcloning DNA for residues 1–83 of CD9 into HindIII and PstI sites of the vector; to introduce the PstI site, codon GTG for Val82 was changed to CTG (coding for Ala). In the resulting fusion protein, there is a 13-amino acid linker (with no cysteines) between CD9 and GFP. To minimize the low inherent ability of GFP to homodimerize, which could potentially influence the results of CD9 cross-linking, we used a monomeric GFP mutant, Leu221→Lys [34], for cross-linking experiments.
Mutations were introduced in full-length and TM(1+2) CD9 proteins by a PCR-based strategy using mutagenic primers and Pfu DNA polymerase. All mutations were confirmed by DNA sequencing.<br><br>Protein expression, microscopy, cysteine disulfide cross-linking and Western blotting
DNA constructs encoding TM(1+2)-GFP or full-length CD9 proteins were transfected into <span class="blue">human</span> rhabdomyosarcoma RD cells using the FuGENE 6 reagent. Cells expressing GFP fusion proteins were analysed by fluorescence microscopy 18–28 hours post-transfection. Images were captured using Spot 1.4.0 camera (Diagnostic Instruments, Sterling Heights, MI) attached to Nikon Eclipse TE300 microscope.
For experiments involving cysteine-mediated cross-linking, cells were treated with 50 μM 2-BP starting 24–26 hours post-transfection and continuing for 16–18 hours. Cross-linking was carried out by incubating cells in HBSM buffer (25 mM Hepes-NaOH, pH 7.2, 150 mM NaCl, 2 mM MgCl2) containing either a) 0.6 mM CuSO4 and 1.8 mM 1,10-phenanthroline (CuP complex) or b) 0.2 mg/ml homobifunctional cysteine-reactive cross-linker (e.g. DTME), diluted from fresh 10 mg/ml solution in DMSO. After incubation for 10–15 minutes (with CuP) or 30–45 minutes (with cross-linker), cells were washed twice for 10 minutes with HBSM containing 10 mM NEM to block residual free cysteines. Cells were lysed in HBSM containing 1% Triton X-100, 0.1% SDS and a cocktail of protease inhibitors with 1 mM EDTA at 4°C for 45–60 minutes. Cell lysate was clarified by centrifugation at 14,000 × g for 15 minutes, an aliquot was removed, and proteins from it were precipitated by addition of trichloroacetic acid to 10% on ice followed by centrifugation at 14,000 × g for 10 minutes. After two washes with ice-cold acetone, protein pellet was solubilized in SDS-PAGE sample buffer without a reducing agent (50 mM Tris-HCl, pH 6.8, 1% SDS, 8% glycerol).
In some experiments, CD9 protein was immunoprecipitated using monoclonal antibody Alb6 (Immunotech, Marseille, France). Proteins were separated by SDS-PAGE and analyzed by Western blotting using monoclonal antibody JL-8 (Clontech) for GFP or Alb6 for CD9. Bands from X-ray films were quantitated using GeneTools™ software from Syngene (Frederick, MD).<br><br>Modeling of TM1-TM2 interaction
An atomic model of the CD9 TM1-TM2 dimer was constructed with a Monte Carlo-simulated annealing (MCSA) algorithm [69]. Two idealized α-helices corresponding to TM1 residues Tyr12 through Leu35 and TM2 residues Gly59 through Val82 were docked with six orthogonal parameters: three rigid body translations and three rotations. During each step of a MCSA cycle, there was an equal probability of changing either one parameter or all six parameters to random values. A conformation's energy was calculated in vacuo with the AMBER united-atom force field for van der Waals interactions [70]. The van der Waals term was modified as described by Kuhlman and Baker [71]. If a structure had favorable dimerization energy, the energies of select mutants were calculated. Structures were selected with a novel scoring function that maximizes the Boltzmann probability of dimerization for silent mutations while minimizing the probability for disruptive mutations. Asn18Ser, Gly77Leu, and Gly80Leu were scored as silent mutations while Gly25Leu, Gly32Leu, Gly67Leu, and Gly74Leu were considered to be disruptive. Each MCSA cycle consisted of 50,000 steps with an exponential temperature decay from 10,000 to 10 K.
Ten MCSA cycles through global sample space were used to restrict the search area. Parameters were restricted to ± 2 standard deviations from their mean value for structures within 10 kcal of the best structure. MCSA cycles were repeated as described above with additional optimization of χ values: rotamers at the protein-protein interface were optimized with Dead End Elimination [72], and χ values were further optimized with Monte Carlo. All MCSA cycles converged upon structures that were within a root mean squared deviation (RMSD) of 1.5 Å with the best structure, and structures that scored within 5 kcal of the best score had an RMSD of less than 0.5 Å with the best structure.<br><br>
List of abbreviations
2-BP, 2-bromopalmitate; CuP, Cu2+-phenanthroline; DTME, dithio-bis-maleimidoethane; LEL, large extracellular loop; NEM, N-ethylmaleimide; TM, transmembrane (domain).<br><br>Authors' contributions
OVK carried out sequence comparisons, mutational analysis and cross-linking experiments, and drafted the manuscript. DGM built the TM1-TM2 interaction model and contributed to the manuscript. WFD supervised DGM's work. MEH coordinated the whole study and prepared the final manuscript.<br><br>
<h3>pmcA1239921</h3>Epigenetic inactivation and aberrant transcription of CSMD1 in squamous cell carcinoma cell lines
Abstract
Background
The p23.2 region of <span class="blue">human</span> chromosome 8 is frequently deleted in several types of epithelial cancer and those deletions appear to be associated with poor prognosis. Cub and Sushi Multiple Domains 1 (CSMD1) was positionally cloned as a candidate for the 8p23 suppressor but point mutations in this gene are rare relative to the frequency of allelic loss. In an effort to identify alternative mechanisms of inactivation, we have characterized CSMD1 expression and epigenetic modifications in head and neck squamous cell carcinoma cell lines.<br><br>Results
Only one of the 20 cell lines examined appears to express a structurally normal CSMD1 transcript. The rest express transcripts which either lack internal exons, terminate abnormally or initiate at cryptic promoters. None of these truncated transcripts is predicted to encode a functional CSMD1 protein. Cell lines that express little or no CSMD1 RNA exhibit DNA methylation of a specific region of the CpG island surrounding CSMD1's first exon.<br><br>Conclusion
Correlating methylation patterns and expression suggests that it is modification of the genomic DNA preceding the first exon that is associated with gene silencing and that methylation of CpG dinucleotides further 3' does not contribute to inactivation of the gene. Taken together, the cell line data suggest that epigenetic silencing and aberrant splicing rather than point mutations may be contributing to the reduction in CSMD1 expression in squamous cancers. These mechanisms can now serve as a focus for further analysis of primary squamous cancers.<br><br><br><br>Background
CUB and Sushi Multiple Domains 1 (CSMD1) was cloned as a candidate tumor suppressor or progression gene from a region of <span class="blue">human</span> chromosome 8 deleted in tumors of the upper aerodigestive tract, prostate, ovary and bladder [1-7]. Deletion of 8p23.2 or reduced expression of CSMD1 has been associated with poor prognosis in head and neck squamous cell carcinomas and in prostate cancers [2,5,8].
CSMD1, consisting of 70 exons spread over two megabases of 8p23.2, encodes a rare 11.5 kb transcript most abundantly expressed in the brain [1]. It is the founding member of a novel, evolutionarily highly conserved gene family whose proteins contain multiple domains thought to be sites of protein-protein or protein-ligand interactions and whose structure suggests that they may be transmembrane receptors or adhesion proteins [9,10].
Tumor suppressor genes are expected to be inactivated in cancers either genetically by mutations or epigenetically by modification of their promoters. While CSMD1 transcripts are detectable in upper aerodigestive tract epithelium, preliminary analysis of several head and neck squamous cell carcinoma cell lines suggested that CSMD1 expression was lost in these lines [1]. Although the region containing CSMD1 is frequently deleted in head and neck squamous cell carcinomas and prostatic adenocarcinomas [3,11-15], point mutations in the gene are relatively rare in primary squamous cancers [16] and in squamous cell carcinoma cell lines (Schmidt, Richter and Scholnick, unpublished). Nonsense or splice junction mutations in CSMD1 have not been reported and not enough is known about the function of the protein to accurately assess the effect of the few missense mutations that have been detected. Thus, if CSMD1 is inactivated in tumors, alternative mechanisms for gene silencing must be operating.
In this paper, we demonstrate that while most squamous cell carcinoma cell lines do not express full length CSMD1 transcripts, nearly all produce abnormal transcripts unlikely to encode functional CSMD1 proteins. Methylation of the DNA preceding CSMD1's first exon is correlated with reduction in the level of expression and cell lines expressing at low levels do not appear to elongate the full 11.5 kb transcript. Other anomalies of expression include incorrect splicing and the use of cryptic promoters. Our data suggest that activation of these promoters may result from the global demethylation of the genome associated with tumorigenesis (reviewed by Ehrlich [17]).
Taken together these data demonstrate that mechanisms other than point mutation are responsible for the aberrant CSMD1 expression in head and neck squamous cell carcinoma cell lines, and these data suggest potential targets for further investigation in primary tumors.<br><br>Results
CSMD1 promoter methylation in HNSCC cell lines is correlated with expression levels
Preliminary evidence suggested that CSMD1 expression is lost in head and neck squamous cell carcinomas [1] but that point mutations were rare [[16], and Schmidt, Richter and Scholnick, unpublished]. To date, only two of the 20 cell lines we have tested for CSMD1 expression, UPCI:SCC066 and PCI-13, express large transcripts initiated at the normal CSMD1 promoter. These data suggest that a mechanism(s) other than point mutation must be responsible for the loss of expression. CSMD1's first exon is embedded in a 3.7 kb CpG island (data from the UCSC genome browser [18]) suggesting that promoter methylation might epigenetically silence the gene. To test this hypothesis, we surveyed 32 head and neck cancer cell lines for CSMD1 promoter methylation using the Combined Bisulfite Restriction Analysis (COBRA) technique described by Xiong and Laird (Methods) [19]. COBRA analysis of the three amplicons diagrammed in Figure 1 suggested that 28 of the cell lines (87%) had more promoter methylation than did normal upper aerodigestive epithelium (data not shown).
We selected nine of these cell lines for high resolution analysis of promoter methylation by sequencing of clones from bisulfite converted genomic DNA. This approach has the distinct advantage of allowing determination of the state of all the CpG dinucleotides within an amplicon on an allele by allele basis. Amplicons 1 and 2 have 19 and 20 CpG dinucleotides, respectively. Amplicon 3 could not be examined by this technique because it is unclonable after bisulfite conversion. The methylation data were correlated to CSMD1 expression levels as measured by quantitative RT-PCR using an amplicon spanning exons 1 and 2 (Methods). A pool of cDNA from five normal oropharyngeal epithelium specimens served as a basis for comparison to the cell lines.
Our data from amplicon 1 demonstrate a clear relationship between methylation and the level of expression (Figure 2). The bisulfite sequencing data confirm that there is relatively little promoter methylation in normal tissue (clones 1–20, Figure 2A). This is also the case in cell line UPCI:SCC066 which expresses a large CSMD1 transcript from the normal promoter at a level approximately 33% of that of normal tissue (clones 32–39, Figure 2B). PCI-13, our highest expressing line at 125% of normal epithelium, displays two distinct patterns of promoter methylation with some clones heavily methylated (clones 21–24, and 30) and others with no methylation (clones 25–39 and 31; Figure 2B). This pattern is consistent with either heterozygosity for methylation or the co-existence of 2 distinct populations within the cell line, one heavily methylated and one unmethylated. We cannot distinguish between these two possibilities using the currently available data.
The remaining cell lines express CSMD1 at a level half that of UPCI:SCC066 or less (ranging from 17% to 1% of normal epithelium) and they exhibit considerably greater methylation of amplicon 1 (clones 40–76, Figure 2C). Cell lines with more amplicon 1 methylation tend to express the gene at lower levels but the relationship is not strictly quantitative (Figure 2C).
In contrast, our data revealed no relationship between expression level and methylation of amplicon 2 (located towards the 3' end of exon 1, Figure 1). For example, all of the 10 clones of amplicon 2 sequenced from PCI-13 were methylated at 19 or 20 of their 20 CpG dinucleotides. UPCI:SCC066, on the other hand, has nearly no methylation in amplicon 2 with only a single methylated CpG dinucleotide detected in one clone out of the nine sequenced. Amplicon 2 ranges from completely unmethylated to heavily methylated in the seven remaining cell lines (data not shown).<br><br>Low transcript levels are accompanied by a failure to elongate the full CSMD1 transcript
On the surface, the quantitative RT-PCR data presented in Figure 2 suggest that the cell lines we consider low expressing might still have up to 17% of the normal level of CSMD1 transcript. A survey of 20 cell lines using a battery of RT-PCR primer pairs located throughout the 11.5 kb transcript reveals that this is not the case. These lines included OKF6-TERT1, a TERT immortalized, p16 deficient but untransformed oral keratinocyte cell line [20].
Our data suggest that the low expressing cell lines shown in Figure 2C express considerably more of the 5' end of the 11.5 kb transcript than they do exons further 3', a phenomenon well illustrated by cell line PCI-100. This line expresses the exon 1/exon 2 amplicon at approximately 15% of the level of normal epithelium. In contrast, we had previously reported that CSMD1 transcripts were not detectable in this line by combined RT-PCR and Southern blotting using three sets of intron spanning primers [1]. The most 5' of those amplicons spans exons 9 through 26.
Analysis with additional primer pairs resolves this apparent paradox by demonstrating that the amount of transcript declines sharply and reproducibly as one examines progressively more 3' exons (Figure 3). The same effect is seen using either oligo-dT or random hexamer primed cDNA. No transcript of this structure has been detected in normal epithelium nor have we detected any sequence alterations in PCI-100 that would explain why the full transcript is not expressed. It is not clear whether PCI-100 produces a small number of discrete size classes of transcript, if individual transcripts terminate at random points within the very large introns in this part of the gene (the first 10 introns average ~150 kb in length), or if the short transcripts result from the elevated activity of a previously undetected posttranscriptional control mechanism.<br><br>Inactivation of CSMD1 by aberrant splicing
Two cell lines, UPCI:SCC066 and PCI-13, express large CSMD1 transcripts initiated at the normal promoter. Subsequent finer scale analysis demonstrates that PCI-13's transcript lacks exons 4 and 5, resulting in a frameshift-induced nonsense codon in exon 6 (Figure 4). Sequencing of the PCI-13 RT-PCR product demonstrates the direct juxtaposition of wildtype exons 3 and 6 and that the transcript contains no novel sequences or splices that would prevent the frameshift. UPCI:SCC066 produces two transcripts, a normal one that includes exons 4 and 5 and another that lacks them (Figure 4). RT-PCR of <span class="blue">human</span> fetal brain cDNA reveals very low levels of an RT-PCR product corresponding in size to that expected from the internally deleted transcript. This band is not readily visible at the exposure used for Figure 4A. We have not detected a similar sized PCR product in RNA from oropharyngeal epithelium but this may reflect the fact that CSMD1 transcripts are ~10x less abundant in oropharyngeal epithelium than they are in fetal brain (data not shown).
The transcripts lacking exons 4 and 5 appear to result from aberrant splicing rather than somatic deletion of these two exons or mutations of their splicing consensus sequences. Exons 4 and 5 can be amplified from PCI-13 genomic DNA and sequencing of those PCR products demonstrates that both their coding sequences and consensus splice sites are wildtype.<br><br>Activation of cryptic promoters in cancer cell lines
The RT-PCR survey revealed a second transcriptional anomaly exhibited by 4 cell lines: SCC9, 041, PCI-1 and PCI-2. Like PCI-100, these lines express low levels of the very 5' end of the transcript and even lower levels of more 3' exons within the first half of the transcript. However, these lines are distinct in expressing higher levels of the 3' half of the transcript, suggesting that alternative promoters in the middle of the gene may be used. SCC9 was chosen for further study because it expresses the 3' half of the transcript at a level dramatically higher than normal for oropharyngeal epithelium. Northern blotting detects a comparatively abundant 6.4 kb truncated transcript as well as smaller amounts of an 8.7 kb transcript in SCC9 (Figure 5A). The other three cell lines express their truncated CSMD1 transcripts at lower levels (data not shown). Sequence analysis of CSMD1 cDNA clones from SCC9 demonstrates that many transcripts are improperly spliced, resulting in retention of intronic sequences and/or deletion of exonic sequences (data not shown). In particular, retention of sequences from intron 40 is common. The high frequency of faulty splicing may explain the broadness of the 6.4 kb CSMD1 band in Figure 5A and suggests that the 8.7 kb transcript may also be incompletely or improperly spliced.
5' RACE [21] reveals that the SCC9 message is initiated just upstream of an Alu element in intron 36 (Figure 5B). Only the 5'-most 120 base pairs of the Alu element are present in the genome. RT-PCR using a forward primer specific for the novel sequences of the SCC9 transcript (prm1904, cgtttagttcgacacacttcatgt) demonstrates that cell lines 041, PCI-1 and PCI-2 do not initiate their CSMD1 transcripts at the same point, suggesting that other cryptic promoters are active in these lines. The sequence of this novel exon has been entered in Genbank as accession number DQ093422.<br><br>DNA methyltransferase inhibitors activate the same cryptic promoter used in cell line SCC9
Expression from epigenetically silenced promoters can sometimes be restored by treatment with inhibitors of DNA methyltransferase or histone deacetylase activity ([22]). We selected two low expressing cell lines with promoter methylation, UPCI:SCC104 and 094, for treatment with various concentrations of 5-azacytidine or 5-aza-2'-deoxycytidine (5-aza-dC) as well as combinations of either of those drugs with the histone deacetylase inhibitor trichostatin A. These treatments did not reactivate the silenced CSMD1 promoter. COBRA analysis of genomic DNA from the treated cells suggested that the drugs did not robustly affect methylation of the CSMD1 promoter even at levels high enough to be toxic to the cells. These experiments did however shed light on the cryptic promoter used in SCC9 cells and on the interpretation of experiments using methyltransferase inhibitors.
Treatment of cell line 094 with relatively high doses of 5aza-dC results in the expression of the 3' end of the CSMD1 transcript. This transcript was not detected in control 094 cells undergoing mock drug treatment (Figure 6) nor was it detected in 094 cells growing under normal culture conditions (data not shown). RT-PCR of cDNA from drug-treated 094 cells using the primer developed from the novel 5' exon of SCC9's truncated transcript (prm1904, see above) yielded a product identical in size to that amplified from SCC9 cDNA (Figure 6). The identity of the product was confirmed by DNA sequencing which also revealed that the drug-induced 094 transcript was more faithfully spliced than the transcript expressed in SCC9 (data not shown).<br><br>
Discussion
Our data clearly demonstrate that expression of normal CSMD1 transcripts is rare in head and neck squamous cell carcinoma cell lines. Of the HNSCC cell lines examined, only UPCI:SCC066 appears to express a normal transcript from the expected promoter. Even that cell line produces a second species of aberrantly spliced transcript lacking internal exons. Our data suggest that epigenetic modification of the DNA 5' of the transcription start site may contribute to the down-regulation of CSMD1. In addition, a low level of expression appears to be associated with production of prematurely terminated transcripts. This degree of complexity might be expected from a 2 megabase, 70 exon gene.
Methylation of a specific region of the CpG island, -395 to -112 bp relative to the transcriptional start site (amplicon 1), appears to be correlated with the activity of the normal CSMD1 promoter. In contrast, methylation of amplicon 2, located within the first exon, shows no such relationship. Our data suggest that the relationship between the amount of methylation in amplicon 1 and the level of expression may not be strictly quantitative. Differences between cell lines with amplicon 1 methylation could arise through a number of mechanisms, for example, variations in the levels of transcription factors between cell lines. In cases where there is considerable heterogeneity in the methylation pattern within a cell line like PCI-100, alleles with less methylation may be expressed at higher levels than those more heavily methylated (compare clones 51 and 52 to clone 58 in Figure 2C). Alternatively, the presence of methylation in amplicon 1 could be a qualitative but not strictly quantitative indicator of methylation of a critical segment of the promoter not discovered in this study.
The normal CSMD1 promoter was not reactivated by drugs that inhibit DNA methyltransferases and histone deacetylases, nor did the drugs abolish CSMD1 promoter methylation, even at toxic doses. Not all genes with promoter methylation respond to such treatments [23]. These drug treatments did, however, provide a potential explanation for the use of a normally cryptic promoter by cell line SCC9. The CSMD1 transcript in this line is initiated near a partial Alu element. 5-aza-dC treatment of cell line 094 activates the same cryptic promoter. This suggests that cryptic promoters may be naturally activated by general hypomethylation of the genome in cancer cells and the subsequent release of repetitive elements from epigenetic repression (reviewed by Ehrlich [17]). The SCC9 transcript does not appear to encode a functional protein but, with a very large gene like CSMD1, there is a potential for some abnormally initiated transcripts to encode truncated proteins with dominant negative properties.
The second ramification of this finding is for the interpretation of data obtained by treating cells with methyltransferase inhibitors. Detection of CSMD1 transcripts solely with primers mapping to the 3' end of the gene could have been erroneously interpreted as representing reactivation of the normal promoter. It seems imperative that such experiments demonstrate that transcripts detected after drug treatment are actually initiated at the normal promoter.
Aberrant splicing also appears to play a role in the production of defective CSMD1 transcripts. Loss of splicing fidelity has been proposed as a characteristic of cancer cells [24,25] and this would be consistent with the variety of misspliced transcripts we detected from SCC9. However, the removal of exons 4 and 5 from the CSMD1 transcript in PCI-13 may reflect a more specific phenomenon than a general inability to splice large introns; this line is still capable of splicing large introns as evidenced by its successful splicing of exon 3 to exon 6, eliminating an intron of over 666 kb. The failure to include exons 4 and 5 may be due to inactivation of a splicing enhancer in intron 3, or to less efficient splicing due to the fact that exons 4 and 5 do not begin with the consensus G residue (Figure 4B) [26].<br><br>Conclusion
Taken together, our data suggest that CSMD1 function is lost in head and neck squamous cell carcinoma cell lines through a variety of mechanisms other than point mutagenesis. Epigenetic modifications of amplicon 1 and defective splicing appear to be fruitful areas to explore in primary head and neck squamous cancers.<br><br>Methods
Cell lines and tissue samples
DNA from HNSCC cell lines UMSCC9, UMSCC35, UMSCC37, UMSCC38, UMSCC45, UMSCC49, UMSCC65, UMSCC68, and UMSCC76 was provided by Dr. Thomas Carey, University of Michigan [27]. Dr. Ruud Brakenhoff, Vrije Universitat, provided cell lines 040, 041, and 094 [28]; Dr. Theresa Whiteside, University of Pittsburgh, provided cell lines PCI-1, PCI-2, PCI-4B, PCI-13, PCI-30, PCI-50, PCI-51, PCI-52, PCI-100 [29], SCC4, SCC9 [30,31], and UPCI:SCC068, UPCI:SCC74, UPCI:SCC104, UPCI:SCC182, UPCI:SCC203, and UPCI:SCC220 [developed by Dr. Susanne Gollin, University of Pittsburgh, 32]. Dr. Gollin provided cell lines UPCI:SCC056, UPCI:SCC066, and UPCI:SCC114 [14,16,32]. The immortal but untransformed keratinocyte line OKF6-TERT1 was obtained from Dr. James Rheinwald, Harvard University [20].
Normal oropharyngeal epithelium was isolated from discarded tissue from uvulopalatopharyngoplasties (UPPP) collected anonymously with the approval of the Washington University <span class="yellow">Human</span> Studies Committee.<br><br>Cell Culture and Tissue Preparation
Squamous cell carcinoma cell lines were grown in DMEM or DMEM:F-12, 1:1 Mixture (BioWhittaker) containing 10% fetal <span class="yellow">bovine</span> serum (Sigma). DMEM medium was supplemented with 1X MEM Nonessential Amino Acids (BioWhittaker). Upper aerodigestive tract epithelium was separated from the rest of the UPPP specimen by digestion with Dispase II (Roche) using a protocol adapted from Oda and Watson [33].<br><br>Nucleic acid preparation and bisulfite conversion
Genomic DNA was isolated by using either Nucleospin Tissue kits (Clontech), QIAamp DNA Blood Mini kits (Qiagen) or Trizol (Invitrogen) according to the manufacturers' instructions. Total RNA isolation, synthesis of first strand cDNA, RT-PCR and 5' RACE PCR were performed essentially as previously described [1]. Poly-A+ RNA for Northern blotting was selected from total RNA using Oligotex beads (Qiagen). Northern blotting and hybridization were performed as previously described [1]. cDNA synthesis was primed using oligo dT or random primers and extended by either Thermoscript or Superscript III reverse transcriptase (Invitrogen). PCR was run in Perkin-Elmer 480 or Applied Biosystems 9700 thermal cyclers for 35 cycles unless otherwise noted. Images of ethidium bromide stained gels were captured with a Gel-Doc imaging station (Biorad). Quantitative PCR was run in an Applied Biosystems 5700 thermal cycler using SYBR Green Master Mix (Applied Biosystems). Primers prm2426 (gtgtggagtatctgcagacatga) and prm2427 (ctggactaagcctccacagttct) were used to amplify a 132 base segment spanning CSMD1's first and second exons. An amplicon from <span class="blue">human</span> 18S RNA was used as a basis for comparisons across cell lines (primers prm2396, ttcggaactgaggccatgat and prm2397, tttcgctctggtccgtcttg). Calculations were performed using the ΔΔCt method in GeneAmp 5700 SDS software (version 1.3) and Microsoft Excel. Quantitation was based on the average values obtained from duplicate reactions. The level of CSMD1 expression in normal oropharyngeal epithelium was determined from pooled cDNA from five UPPP specimens.
We used the CpGenome DNA Modification kit (Intergen) for bisulfite conversion of the genomic DNA according to the manufacturer's protocol, with the following exception. Incubation of the conversion reaction was carried out in a thermal cycler for six cycles each consisting of three minutes at 94°C followed by three hours at 50°C (Christina Menke and Paul Goodfellow, personal communication).<br><br>Analysis of CSMD1 Promoter Methylation
Methylation of three segments of the CSMD1 CpG island was examined using the Combined Bisulfite Restriction Analysis technique (COBRA) [19]. All three segments were amplified by using the nested primers and PCR conditions listed in Table 1. Amplicon 1 extends from -395 to -112 bp, amplicon 2 from +175 to +396 bp, and amplicon 3 from +398 to +718 bp relative to the first base of the transcript. A small region surrounding the transcription start site (-111 to +174 bp) could not be examined because no PCR primers could be designed from its extremely GC-rich sequence. The first round PCR used 2 μl of bisulfite converted genomic DNA in a final volume of 10 μl. Subsequent amplifications with nested primers used 4 μl of first round product as template in reactions with a final volume of 20 μl. All PCR was carried out for 35 cycles. A portion of the second round PCR product was run on a 1.5% agarose gel, stained with ethidium bromide, and quantified using the ImageQuant software package (v1.2 for Macintosh, Molecular Dynamics) so that equal amounts of each could be used in restriction digests.
Restriction digests for COBRA were performed with either BstU I or Taqα I (5 or 10 units per reaction, respectively; New England Biolabs) for 4 hours in a final volume of 10 μl. Taqα I digests were performed only when no methylation was detected with BstU I. BstU I digests also included an internal control DNA fragment to confirm complete digestion. This DNA fragment contains a single BstU I site and was amplified from a cloned CSMD1 cDNA using primers prm2020 (agatcccccagtgtctccctgtgt) and prm2021 (actgctggtgccgtggtaatgact). The control PCR product is 1019 bp long and is digested to two fragments of 605 and 414 bp by BstU I. Digestion products were fractionated on a 10% polyacrylamide gel, stained with ethidium bromide, and visualized with a Gel-Doc video imaging workstation (Bio-Rad).
High resolution analysis of methylation was performed by sequence analysis of individual clones from amplicons 1 and 2. DNA from amplicon 3 proved unclonable and gel electrophoresis suggests that its very AT rich sequence results in a bent DNA configuration. PCR products were purified using the Nucleospin Extraction columns (Clontech) and inserted into the pCR2.1-TOPO vector using the TOPO TA Cloning kit (Invitrogen) according to the manufacturer's instructions. Plasmid DNA from individual colonies was isolated using the Nucleospin Plus Plasmid Miniprep kit (Clontech) and sequenced with a reverse vector primer (agcggataacaatttcacacagga) using fluorescence based sequencing with Big Dye Terminator mix (Applied Biosystems).<br><br>Treatment of cultured cells with DNA methyltransferase inhibitors
Cell line 094 was treated with 5-aza-2'-deoxycytidine (5aza-dC) (Sigma) dissolved in DMSO. Two 100 mm cell culture dishes containing 5 × 105 cells were established for each of the drug concentrations tested. Cells were grown for 72 hours in media containing DMEM:F-12, 1:1 Mixture (BioWhittaker) with 1X MEM Nonessential Amino Acids (BioWhittaker) and 10% fetal <span class="yellow">bovine</span> serum and then switched to media containing 5aza-dC at concentrations of 0 μm, 5 μM, 25 μM, or 100 μM. Cells were fed daily for 4–5 days and then both plates were harvested in 3 ml of Trizol (Invitrogen) for isolation of RNA and DNA according to the manufacturer's instructions. RT-PCR used for detection of CSMD1 transcripts in these treated cells was run for 40 cycles.<br><br>
Abbreviations
5aza-dC = 5-aza-2'deoxycytidine, COBRA = Combined Bisulfite Restriction Analysis, CSMD1 = Cub and sushi multiple domains 1, HNSCC = head & neck squamous cell carcinoma, RT-PCR = reverse transcription – polymerase chain reaction, SCC = squamous cell carcinoma, UPPP = uvulopalatopharyngoplasty.<br><br>Competing interests
The author(s) declare that they have no competing interests<br><br>Authors' contributions
TMR performed the DNA methylation analysis, and parts of the transcript survey. BDT cloned and characterized the novel first exon expressed in cell line SCC9. SBS performed parts of the transcript survey and characterized the deletion of exons 4 and 5 in PCI-13. All three authors participated in the analysis of the data and in the writing of the manuscript.<br><br>
<h3>pmcA140510</h3>Enp1, a <span class="yellow">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="yellow">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="yellow">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="yellow">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="blue">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="blue">human</span> Enp1, expressed in <span class="yellow">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="blue">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="blue">human</span> homolog of <span class="yellow">yeast</span> Enp1 was PCR amplified from a <span class="blue">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="blue">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="yellow">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="yellow">donkey</span>-anti-<span class="yellow">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="yellow">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="yellow">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="yellow">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="blue">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="blue">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="yellow">yeast</span> Enp1 (19). In contrast, we identified a <span class="blue">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="blue">human</span> bystin (19) showed that they were encoded by the same gene on <span class="blue">human</span> chromosome 6. The reported sequence for <span class="blue">human</span> bystin is a fragment of the 437 amino acid <span class="blue">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="blue">human</span> bystin, as confirmed by available <span class="blue">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="yellow">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="yellow">Drosophila melanogaster</span> and <span class="yellow">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="blue">human</span> Enp1 homolog was cloned and expressed in <span class="yellow">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="blue">human</span> Enp1 homolog did not complement a <span class="yellow">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="blue">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="yellow">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="yellow">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="yellow">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="blue">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="blue">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="blue">human</span> homolog of Enp1 did not complement a <span class="yellow">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="blue">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA147033</h3>Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the ribonuclease A gene family.
Abstract
We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophil-associated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this 'mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing.
� 1997 Oxford University Press <br><br> Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> 4235�4239 <br><br> Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the Ribonuclease A gene family Dean Batten+, Kimberly D. Dyer, Joseph B. Domachowske� and Helene F. Rosenberg* The Laboratory of Host Defenses, Building 10, Room 11N104, National Institute of Allergy and Infectious Diseases, National Institutes of Health, 9000 Rockville Pike, Bethesda, MD 20892, USA Received August 4, 1997; Revised and Accepted September 17, 1997 DDBJ/EMBL/GenBank accession numbers: U72031, U72032, AF017258�AF017261 <br><br> ABSTRACT We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophilassociated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this `mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing. INTRODUCTION Ribonuclease A (RNase A, <span class="yellow">bovine</span> pancreatic ribonuclease) is the prototype of an enlarging family of proteins defined by distinctive elements of conserved primary structure and enzymatic activity (1,2). The features shared by all members of the RNase A family include six to eight cysteines and specific histidine and lysine residues that form the ribonuclease active site. To date, several distinct lineages within the RNase A superfamily have been identified. Pancreatic ribonucleases (RNases 1) have been isolated from an extensive array of mammalian species (3,4), and mRNA encoding <span class="blue">human</span> pancreatic ribonuclease has been detected in <br><br> numerous somatic tissues in addition to pancreas (5). RNases 2 and 3, eosinophil-derived neurotoxin (EDN) and eosinophil cationic protein (ECP), respectively, have been characterized primarily with respect to eosinophil function (6,7), although expression of EDN [also known as RNase Us (8) or <span class="blue">human</span> liver ribonuclease (9)] is also widespread (5). Angiogenin (RNase 5), a structurally atypical member of this family, was originally identified as an agent promoting blood vessel growth and development (10,11). RNase 4 (12,13) and RNase k6 (14) are also members of the RNase A superfamily, although their functions are currently obscure. Recently, Larson and colleagues (15) described cDNAs encoding two highly homologous <span class="yellow">murine</span> ribonucleases, the <span class="yellow">murine</span> eosinophil-associated ribonucleases (mEAR) 1 and 2, which were cloned and identified via tryptic peptides isolated from <span class="yellow">murine</span> eosinophil proteins. Although Southern analysis demonstrated that multiple copies of sequence homologous to mEAR-1 and mEAR-2 were present in <span class="yellow">murine</span> genomic DNA, the precise nature of these copies--as pseudogenes, polymorphisms, or distinct functional genes--was unclear. In this work, we have identified four of these homologous sequences, three encoding novel ribonucleases, and one encoding a pseudogene. Together with mEAR-1 and mEAR-2, these six ribonuclease genes form an `mR cluster,' whose members are more closely related to one another than they are to other <span class="yellow">murine</span> ribonucleases, yet whose position with respect to the defined RNase A lineages remains unclear. MATERIALS AND METHODS Isolation of genomic fragments encoding <span class="yellow">murine</span> ribonucleases by polymerase chain reaction (PCR) Genomic DNA was isolated from cells of the <span class="yellow">murine</span> 3T3 fibroblast cell line. The mR-3, mR-5 and mR-6P sequences were amplified using oligonucleotide primers derived from the published coding sequence of mEAR-1 as follows: 5-CAA ACC CTT TCC CAG AAG TTT GCC-3 (amino acids 28�33); 5-AAA TGT CCC ATC CAA GTG AAC TGG ACC-3 (amino acids 156�148) (15). The PCR reactions were performed as described previously (14), and the multiple products present in the single 400 bp product were identified by dideoxy<br><br> *To whom correspondence should be addressed. Tel: +1 301 402 9131; Fax: +1 301 402 4369; Email: hr2k@nih.gov Present addresses: +Duke University School of Medicine, Durham, NC 27701, USA and �Department of Pediatrics, State University of New York-Health Sciences Center at Syracuse, Syracuse, NY 13210, USA <br><br> 4236 Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> Figure 1. (A) Alignment of amino acid sequences encoded by six related <span class="yellow">murine</span> ribonuclease genes. Regions of amino acid sequence identity among the five functional genes are enclosed in boxes; the boxes enclosing the active site residues (His11, Lys35, His124) and eight cysteines conserved in all members of the RNase A superfamily are shaded. (B) Percent similarities between amino acid sequences determined via the BESTFIT algorithm of the Wisconsin Genetics Computer Group (WGCG) program on-line at the National Institutes of Health. GenBank accession numbers: mEAR (<span class="yellow">murine</span> eosinophil-associated ribonuclease)-1, U72032; mEAR-2, U72031; mR (<span class="yellow">murine</span> ribonuclease)-3, AF017258; mR-4, AF017259; mR-5, AF017260; mR-6P (pseudogene), AF017261. <br><br> sequencing of individual plasmids after subcloning into the pCR II TA cloning vector (Invitrogen, San Diego, CA). The complete open reading frames were obtained by extension in both 5 and 3 directions by uni-directional PCR (Genome Walker kit, Clontech, Palo Alto, CA), involving two amplifications with nested primers and Tth polymerase as described previously (14). The nested gene-specific primer pairs were as follows: for 5 extension of mR-3, 5-CTG GAA CCA CTG GAT ACG TGG GAC TGT CCT-3 and 5-ACG TGG GAC TGT CCT GTG GAG TTC TGG GTT-3; for 3 extension of mR-3, 5-ATG CTG TTG GTG TGT GTG GAA ATC CAA GTG-3 and 5-GCT TGT GCA GTG ACA ATA TAA GTC AAA ACT-3; for 5 extension of mR-5, 5-GGT GGG ACG GTC CTT TGG AGT TCT GGG GTT ACA-3 and 5-GGG GTT ACA GGC AAC TGT GTA GGA CTT CTT TCC-3; for 3 extension of mR-5, 5-GAT GTT GTC CGT GTG TGT CAC AAT CCA CCC-3 and 5-AAG ACT TGC AAA GAC GGG ACA AGT CCA AAT-3; for 5 extension of mR-6P, 5-TTG TTG TTT TGC ATC TGC ATT GTG CAT AAC TGC-3 and 5-TTC TGC ATA ACT GCT TGC TGA ACT TGT GAG TGA-3; for 3 extension of mR-6P, 5-ATG TGA TAA TGC AAT GCT GTC TCT TAG CAG TTA-3 and 5-TAC AAG <br><br> AGT ATG TAA GCC ATT GAA TCA TTT TCT-3. The products of these reactions were likewise subcloned into the PCR II TA vector and evaluated by dideoxy-sequencing. The mR-4 sequence was amplified with a 36 bp primer derived from sequence encoding the amino-terminal signal sequence of mEAR-1; 5-ATG GGT CCG AAG CTG CTT GAG TCC CGA CTT TGT CTC-3 with the carboxy-primer described above. <br><br> Northern analysis The <span class="yellow">murine</span> multi-tissue Northern blot was purchased from Clontech, and pre-hybridized and hybridized as per manufacturer's instructions with radiolabelled oligonucleotide probes. The membrane was washed with 5� SSPE with 0.1% SDS for 1 h at 37_C and the autoradiogram developed after exposure to film at �80_C. The mEAR-1/mEAR-2/mR-3 probe: 5-CTC TTG TCA CTG CAC AAG CCA CTT GGA TTT CC-3, the mR-5 probe: 5-GTC CCG TCT TTG CAA GTC TTG GGT GGA TTG TG-3, and the actin probe: 5-GCA CAT GCC GGA GCC GTT GTC GAC GAC GAG CGC GGC GAT ATC ATC ATC-3 (16). <br><br> 4237 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 4237 <br><br> Figure 2. (A) Dendrogram depicting relationships among the eight characterized <span class="yellow">murine</span> ribonucleases as determined by a modified UPGMA method (17). Abbreviations are as in Figure 1A, also mPR (<span class="yellow">murine</span> pancreatic ribonuclease, sw:rnp_<span class="yellow">mouse</span>), mANG (<span class="yellow">murine</span> angiogenin, U72672), hEDN (<span class="blue">human</span> eosinophil-derived neurotoxin, M24157), hECP (<span class="blue">human</span> eosinophil cationic protein, X15161), hRK6 (<span class="blue">human</span> ribonuclease k6, U64998), pRK6 (porcine kidney ribonuclease (24), and bRK6 (<span class="yellow">bovine</span> kidney ribonuclease, sw:rnkd_bovin). (B) Calculated isoelectric points from amino acid sequences in (A) as determined via the PEPTIDESORT algorithm of WGCG. Table 1. Amino acid sequence comparisons of <span class="yellow">murine</span> and <span class="blue">human</span> ribonucleases <br><br> Sequence analysis All DNA sequence analysis and comparisons were performed with the assistance of the Wisconsin Genetics Computer Group and Sequencher (Gene Codes Corporation) programs available at the National Institutes of Health. The dendrogram in Figure 2A was constructed by a modified version of the unweighted pair group method with arithmetic mean (UPGMA) (17), constructed using four initial pairings, mEAR-2 to mR-3, mR-5 to mR-6P, hEDN to hECP and bRK6 to pRK6. RESULTS Isolation of genomic fragments encoding novel <span class="yellow">murine</span> ribonucleases The alignment in Figure 1A displays the amino acid sequences encoded by the six related ribonuclease genes. <span class="yellow">Murine</span> eosinophil-associated ribonucleases-1 and -2 (mEAR-1 and mEAR-2) are the predicted amino acid sequences from two genes described previously by Larson and colleagues (15); <span class="yellow">murine</span> ribonucleases 3, 4, 5 and 6-pseudogene (mR-3, mR-4, mR-5 and mR-6P), are the predicted amino acid sequences encoded by novel DNA <br><br> hEDN mEAR-1 mEAR-2 mR-3 mR-4 mR-5 mANG mPR 56 56 54 60 60 43 53 <br><br> hECP 59 59 57 59 58 49 52 <br><br> hRK6 58 59 59 53 56 49 59 <br><br> hPR 47 48 49 48 57 55 79 <br><br> hR4 50 53 51 52 54 57 66 <br><br> hANG 49 48 51 47 47 80 60 <br><br> Values are expressed as percent similarity between pairs of amino acid sequences as determined by the BESTFIT algorithm of the Wisconsin Genetics Computer Group program on-line at the National Institutes of Health. Value representing the highest degree of sequence homology in each row is indicated in boldface. Abbreviations are as defined in Figures 1 and 2, and also include hR4 (<span class="blue">human</span> ribonuclease 4, sw:rnl4_<span class="blue">human</span>), hANG (<span class="blue">human</span> angiogenin, M11567), and hPR (<span class="blue">human</span> pancreatic ribonuclease, X62946). <br><br> 4238 Nucleic Acids Research, 1997, Vol. 25, No. 21 fragments amplified from <span class="yellow">murine</span> genomic DNA. All five functional genes encode amino acid sequences with eight cysteines and appropriately spaced catalytic histidines and lysine that are conserved among the members of the RNase A superfamily. The predicted coding sequence of the non-functional pseudogene (mR-6P) includes two aberrant stop codons (positions 19 and 100) as well as a point mutation resulting in the destruction of the cysteine at position 78. Each genomic fragment also encodes a 28 residue amino terminal signal sequence (not shown). The amino acid sequences are displayed in order of decreasing similarity to mEAR-1 (Fig. 1B). Genomic fragments encoding mEAR-1 and mEAR-2 were also detected. Relationships among the <span class="yellow">murine</span> ribonucleases A dendrogram depicting the relationships among several of the <span class="yellow">murine</span> and <span class="blue">human</span> ribonucleases is shown in Figure 2A. The mEAR-1, mEAR-2, and mR-3 sequences form a closely related sub-group, with only nine amino acid sequence differences noted between mEAR-2 and mR-3, and 12 and 17 between these two and mEAR-1, respectively. The mR-4, mR-5, and mR-6P sequences diverge from this subgroup and from one another as well. These six sequences are more closely related to one another than they are to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or to <span class="yellow">murine</span> angiogenin (mANG) or to any of the three <span class="blue">human</span> ribonucleases shown (hEDN, hECP or hRK6). Isoelectric points The isoelectric points of all six related ribonucleases range from 8�11, as is typical for members of the RNase A superfamily (Fig. 2B). Interestingly, the isoelectric points within the mEAR-1/mEAR-2/mR-3 subgroup vary by a full 1.0 unit (mEAR-1 at 9.2 to mEAR-2 at 10.2) in response to only 12 discrepancies in amino acid sequence. As increased charge has been associated with increased toxicity in other branches of the RNase A gene family (18), this finding has potential consequences with respect to the function of these proteins (see Discussion). Northern analysis Tissue-specific expression of mRNA encoding these <span class="yellow">murine</span> ribonucleases is shown in Figure 3. The probe complementary to all three sequences of the mEAR-1/mEAR-2/mR-3 subgroup hybridizes to an 1.4 kb mRNA isolated from renal tissue (Fig. 3A). As the probe cannot distinguish among the three genes, it is not possible to conclude whether the band detected represents mRNA encoding one, two, or all three of the ribonucleases of this subgroup. In contrast, the probe complementary to the mR-5 genomic fragment hybridizes to two prominent mRNA species in hepatic tissue, one at 1.4 kb, and a second at 2.0 kb. A prominent mRNA of 1.8 kb was also detected in mRNA isolated from <span class="yellow">murine</span> testicular tissue; fainter hybridizing mRNAs of varying size were detected in lung and skeletal muscle. The size variation may represent differential splicing of the mR-5-encoding mRNA, or may represent mRNAs encoding one or more as yet undiscovered <span class="yellow">murine</span> ribonucleases that are more closely related to mR-5 than those whose sequences are reported here. <br><br> Figure 3. (A) <span class="yellow">Murine</span> RNA probed with a 32 bp oligonucleotide complementary to mEAR-1, mEAR-2 and mR-3 coding sequences. Tissue source of each RNA is indicated above each lane. (B) Same as in (A), probed with a 32 bp oligonucleotide complementary to mR-5. (C) Same as in (A), probed with a 48 bp oligonucleotide complementary to <span class="blue">human</span> actin (16), demonstrating relative loading of each lane. <br><br> Relationship to <span class="blue">human</span> ribonucleases The data in Table 1 denote the amino acid sequence similarity between pairs of <span class="yellow">murine</span> and <span class="blue">human</span> ribonucleases of the RNase A superfamily. Overall, the similarities within the identified cluster range from 47 to 60%, with remarkably little variation among the pairs. Although the sequences of the mEAR-1/mEAR-2/mR-3 subgroup are slightly more similar to ECP and RNase k6, and mR-4 and mR-5, to EDN and ECP, the orthologous relationships of these <span class="yellow">murine</span> ribonucleases cannot be discerned from these data. This stands in direct contrast to mPR and mANG, whose <span class="blue">human</span> orthologs can be clearly distinguished on the basis of their amino acid sequence homologies. <br><br> DISCUSSION In this work, we have identified four novel <span class="yellow">murine</span> ribonuclease genes that, together with mEAR-1 and mEAR-2 defined by Larson and colleagues (15), form an unusual cluster within the RNase A gene superfamily. The six genes within this cluster have varying amino acid sequence homologies to one another, but are clearly more closely related to one another than to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or angiogenin (mANG), or to any of the <span class="blue">human</span> ribonucleases of the RNase A family. These results suggest that the `mR cluster' emerged via multiple duplications of a gene that had already diverged from those encoding the other <span class="yellow">murine</span> ribonucleases. Although this type of expansion is unusual in this gene family, it is actually not unique; similar expansion has occurred in <span class="yellow">bovine</span> species, with gene duplication resulting in the three closely-related <span class="yellow">bovine</span> pancreatic, <span class="yellow">bovine</span> brain (19,20) and <span class="yellow">bovine</span> seminal (21) ribonucleases. One interesting feature of the mR cluster is that its relationship to any of to the six known <span class="blue">human</span> RNase A-type genes cannot be <br><br> 4239 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 determined from their respective amino acid sequence homologies. Although the amino acid sequences encoded by mEAR-1 and mEAR-2 match those of tryptic peptides derived from <span class="yellow">murine</span> eosinophil proteins (15), the homology data do not stand in overwhelming support of a unique relationship between any of the mR cluster ribonucleases and the <span class="blue">human</span> eosinophil ribonucleases EDN and ECP. The existence of an additional, as yet unidentified <span class="blue">human</span> ribonuclease (or ribonucleases) more closely related to those of the mR cluster cannot be ruled out. Perhaps the most important issue raised by this work is the question of why so much evolutionary energy has been devoted to enlarging and diversifying the RNase A gene superfamily. To date, this superfamily now includes several distinct lineages, two species-limited clusters, and the two most rapidly evolving functional coding sequences known among primates (22). The question of evolutionary energy takes on particular significance here, where five functional ribonucleases have emerged in what appears to be a remarkably short period of evolutionary time (15). It is conceivable that each of these ribonucleases might serve a unique but related function. On this point, a number of studies have suggested a distinct role for the <span class="blue">human</span> ribonuclease ECP in eosinophil-mediated host defense (6,7); in addition, we have recently shown that eosinophil ribonucleases can inhibit retroviral transduction of <span class="blue">human</span> target cells (23). The possibility that certain ribonucleases have diverged to promote distinct and specific host defense-related activities remains an intriguing hypothesis. ACKNOWLEDGEMENTS We would like to thank Dr Jianzhi Zhang for helpful discussions, and Drs Harry L. Malech and John I. Gallin for their ongoing suppport of the work in our laboratory. D.B. is a Howard Hughes Medical Institute-National Institutes of Health Research Scholar REFERENCES 1 D'Alessio, G. and Riordan, J.F. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. 2 Sorrentino, S. and Libonati, M. (1997) FEBS Lett. 404, 1�5. 21 22 23 24 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br><br> 4239 <br><br> Beintema, J.J., Breukelman, H.J, Carsana, A., and Furia, A. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. Confalone, E., Beintema, J.J., Sasso, M.P. Carsana, A., Palmieria, M., Vento, M.T. and Furia A. (1995) J. Mol. Evol. 41, 850�858. Futami, J., Tsushima, Y., Murato, U, Tada, H., Sasaki, J., Seno, M., and Yamada, H. (1997) DNA Cell Biol. 16, 413�419. Ackerman, S.J. (1993) In Makino, S. and Fukuda, T. (eds) Eosinophils: biological and clinical aspects. CRC Press, Inc., Boca Raton, FL Spry, C.J.F. (1988) Eosinophils: a comprehensive review and guide to the scientific and medical literature. Oxford University Press, Oxford, UK. Beintema, J.J., Hofsteenge, J., Iwama, M., Morita, T., Ohgi, K., Irie, M., Sugiyama, R.H., Schieven G.L., Dekker, C.A., and Glitz, D.G. (1988) Biochemistry 27, 4530�4538. Sorrentino, S., Tucker, G.K., and Glitz, D.G. (1988) J. Biol. Chem. 263, 16125�16131. Vallee, B.L. and Riordan, J.F. (1988) Adv. Exp. Med. Biol. 234, 41�53. Moroianu, J. and Riordan, J.F. (1994) Proc. Natl. Acad. Sci. USA 91, 1677�1681. Seno, M., Futami, J., Tsushima, Y., Akutagawa, K., Kosaka, M., Tada, H. and Yamada, H. (1995) Bioch. Biophys. Acta 1261, 424�426. Rosenberg, H.F. and Dyer, K.D. (1995) Nucleic Acids Res. 23, 4290�4295. Rosenberg, H.F. and Dyer, K.D. (1996) Nucleic Acids Res. 24, 3507�3513. Larson, K.A., Olson, E.V., Madden, B.J., Gleich, G.J., Lee, N.A. and Lee, J.J. (1997) Proc. Natl. Acad. Sci. USA 93, 12370�12375. Nakajima-Iyima, S., Hamada, H., Reddy, P. and Kakanaga, T. (1985) Proc. Natl. Acad. Sci. USA 82, 6133�6137. Li, W.-H. and Graur, D. (1991) Fundamentals of Molecular Evolution. Sinauer Associates, Sunderland, MA. Rosenberg, H.F. and Dyer, K.D. (1995) J. Biol. Chem. 270, 21539�21544. Watanabe, H., Katoh, H, Ishii, M., Komoda, Y., Sanda, A., Takizawa, Y., Ohgi, K. and Irie, M. (1988) J. Biochem. 104, 939�945. Sasso, M.P., Carsana, A., Confalone, E., Cosi, C., Sorrentino, S., Viola, M., Palmieri, M., Russo, E. and Furia, A. (1991) Nucleic Acids Res. 19, 6469�6474. Palmieri, M., Carsana, A., Furia, A. and Libonati, M. (1985) Eur. J. Biochem. 152, 275�277. Rosenberg, H.F., Dyer, K.D., Tiffany, H.L. and Gonzalez, M. (1995) Nature Genet. 10, 219�223. Domachowske, J.B. and Rosenberg, H.F. (1997) J. Leukoc. Biol., 62, 363�368. Iwama, M., Sanda, A., Ohgi, K., Hofsteenge, J., and Irie, M. (1993) Biosci. Biotech. Biochem. 57, 2133�2138. <br><br>  <h3>pmcA1474674</h3>Physiological and chemical characterization of <span class="yellow">cyanobacterial</span> metallothioneins.
Abstract
Techniques have been developed for detection, quantitation, and isolation of bacterial metallothioneins (MTs) from <span class="yellow">cyanobacterial</span> species. These methods involve differential pulse polarography and reverse-phase high-performance liquid chromatography (HPLC) and have allowed detection of picomole quantities of these high sulfhydryl content proteins. The prokaryotic molecule was found to be induced in the presence of Cd or Zn salts with regulation at the level of transcription. Cu was not found to induce synthesis of the prokaryotic MT. Exposure to the former metals resulted in a growth lag followed by simultaneous induction of MT synthesis and onset of growth. Amino acid analysis and N-terminal sequence analysis indicated that the bacterial MTs from cyanobacteria are unique, having many aromatic and aliphatic residues and no apparent association of hydroxylated or basic amino acids with cysteines. Although the characteristic Cys-X-Cys sequences were present, no apparent amino acid sequence homology with the eukaryotic MTs was found in the first 42 residues.<br><br><br><br>
 Environmental Health Perspectives Vol. 65, pp. 71-75, 1986 <br><br> Physiological and Chemical <br><br> Characterization of Cyanobacterial Metallothioneins <br><br> by Robert W. Olafson* <br><br> Techniques have been developed for detection, quantitation, and isolation of bacterial metallothioneins (MTs) from <span class="yellow">cyanobacterial</span> species. These methods involve differential pulse polarography and reverse-phase high-performance liquid chromatography (HPLC) and have allowed detection of picomole quantities of these high sulfhydryl content proteins. The prokaryotic molecule was found to be induced in the presence of Cd or Zn salts with regulation at the level of transcription. Cu was not found to induce synthesis of the prokaryotic MT. Exposure to the former metals resulted in a growth lag followed by simultaneous induction of MT synthesis and onset of growth. Amino acid analysis and N-terminal sequence analysis indicated that the bacterial MTs from cyanobacteria are unique, having many aromatic and aliphatic residues and no apparent association of hydroxylated or basic amino acids with cysteines. Although the characteristic CysX-Cys sequences were present, no apparent amino acid sequence homology with the eukaryotic MTs was found in the first 42 residues. <br><br> Introduction <br><br> Metallothioneins have now been isolated and characterized from a large variety of eukaryotic organisms (1) and shown to be involved in heavy metal detoxification and/or homeostasis (2-4). These proteins are highly homologous with respect to amino acid sequence and complex metals in characteristic metal-thiolate cluster arrangements, the structures of which have been recently <br><br> investigated by 113Cd-NMR (6). <br><br> Although metallothionein (MT) is found throughout the eukaryotic world, few reports of the presence of this type of high sulfhydryl content metal-binding protein exist for prokaryotes. We earlier reported the presence of a prokaryotic MT in cyanobacteria (7,8) and have recently provided primary sequence evidence substantiating these data. This manuscript is intended to summarize the present state of knowledge regarding the physiological and chemical characterization of these bacterial MTs, with an emphasis on the techniques employed in such studies. <br><br> Analytical Procedure <br><br> In order to facilitate rapid detection, isolation, and quantitation of MT from various sources, we have routinely employed a differential pulse polarographic tech<br><br> *Department of Biochemistry and Microbiology, University of Victoria, Victoria, British Columbia V8W 2Y2, Canada. <br><br> nique first described by Brdicka (9-11). This procedure avoids use of radioisotopes and circumvents problems such as species specificity and metal stoichiometric assumptions associated with radioimmunoassays and metal binding assays. Figure 1 shows a typical polarographic wave for MT using the Brdicka procedure. Measurement of the wave height can be performed from either the <br><br> 0.20<br><br> glJ   MT-1 <br><br> O 0.15<br><br> E <br><br> 0 0.10 0 <br><br> E 0.05    _  _ <br><br> -1.35   -1.45    -1.55    -1.65 <br><br> V vs Agt/AgC1 <br><br> FIGURE 1. Differential pulse polarographic wave for <span class="yellow">murine</span> MT-1. <br><br> Wave heights were calculated from the broken tangent line between minima rather than the supporting electrolyte baseline at the bottom of the figure. <br><br> R. W OLAFSON <br><br> supporting electrolyte baseline or a tangent to the minima as shown in the preceding figure. In the cobalt hexaminechloride supporting electrolyte, the half-wave potential for all MTs studied to date is approximately -1.45 V versus a Ag/AgCl reference electrode. This electrochemical reaction results in highly reproducible and linear current responses which can be used with samples containing picomole levels of MT (Fig. 2). Since this is an exceedingly sensitive instrument, attention to detail is particularly important for successful use of the Brdicka procedure (11). Routine use of a reference standard MT is highly recommended to compensate for small day-today variations in response. For absolute values, standards must be identical to the species of protein quantitated; otherwise commercially available standard is adequate. Although certain tissues and cellular types can be shown to have no other polarographically active material other than MT, with samples from new species, it is necessary to evaluate this potential problem by gelperneation chromatography and, if necessary, take steps to remove contaminants before assaying. In most cases, <br><br> 0 <br><br> E i <br><br> m 0 <br><br> 0 E <br><br> 0.31 <br><br> 0.11 <br><br> O   0.1       0.3       0.5 <br><br> MT (nanomoles) <br><br> FIGURE 2. Standard curve for polarographic determination of me<br><br> tallothionein. <br><br> FIGURE 3. Cyclic voltamogram obtained by equilibration at -1.0 V <br><br> for 4 min followed by cyclic scanning to OV and -1.4 V at a scan rate of 200 mV/sec with a PAR Model 303 static mercury drop electrode in the hanging drop mode. The mercury drop electrode was set in the large drop position producing a drop of 0.0226 cm3. The supporting electrolyte was 20 mM HEPES buffer, pH 7.3. The dotted line is the equilibrium trace. Adapted Olafson and Sim (10). <br><br> this is easily managed by heat denaturation, if care is taken to assess the degree of MT losses due to coprecipitation. Such losses later can usually be minimized to less than 10% by adjustment of the tissue homogenate density prior to denaturation. <br><br> It should be indicated that additional utility can be found with a polarographic analyzer beyond application of the Brdicka procedure. For example, metal levels can be readily determined by using differential pulse or anodic stripping voltametry procedures (10), while Figure 3 shows a cyclic voltamogram of bacterial MT. The latter technique is potentially useful in assessing metal speciation and integrity of the protein tertiary structure near metal clusters, particularly after metal reconstitution studies. <br><br> E <br><br> 'C2A-                                    E   24 <br><br> 0           30                            40 E0 <br><br> *~~~~~~~~~~~~ <br><br> o                           C~~~~~~~~~~~~~~~~~~~~C <br><br> Fraction Number (18ml) <br><br> FIGURE 4. Sephadex G-75 chromatography profile of bacterial cell <br><br> lysate applied to a 5 x 100 cm column and eluted with 10 mM anunoium bicarbonate, 5 mM mercaptoethanol (7). <br><br> Elution  Time  (min) <br><br> FIGURE 5. Reverse-phase high-performance chromatography profile <br><br> of Sephadex G-75 fractionated bacterial MT. The 300 A pore size propyl column was eluted with 20 mM triethylamine phosphate, pH 7.0, by using an acetonitrile organic modifier (phase B). Column effluent was monitored at 250 nm with a sensitivity of 1.0 AUFS (12). <br><br> 72 <br><br> <span class="yellow">CYANOBACTERIAL</span> METALLOTHIONEINS <br><br> Isolation of Cyanobacterial MT <br><br> <span class="yellow">Synechococcus strain Tx-20</span> cyanobacteria were maintained in axenic culture on BG-11 medium (7) and harvested from aerated 20 L cultures grown at 280C under fluorescent light. Cells were broken in a French pressure cell at 0?C in 0.5 M Tris-HCI, pH 8.6, and the lysate exposed to 10 ,uCi of 109CdCl. The 40,000g supernatant was applied to a Sephadex G-75 column developed with 10 mM ammonium bicarbonate and 5 mM mercaptoethanol resulting in the profile shown in Figure 4. Polarographic activity and radionuclide binding were coincident with a 10,000 MW ultraviolet absorbing fraction. Further purification of this material could be undertaken by DEAE-cellulose chromatography or isoelectric focusing but maximum resolution of isoproteins was best attained by reverse-phase high-performance liquid chromatogra<br><br> 0              5 <br><br> DAYS <br><br> I0 <br><br> 5 <br><br> phy. Figure 5 shows such a separation performed on a 300 A pore size C-3 column (Beckman Instruments). Four baseline resolved MT peaks appear between 40 and 70 min. Peaks eluting at 80 min contain at least four additional MT components resolvable by adjustment of the applied organic modifier ramp. Preliminary evidence indicates that these isoproteins are microheterogeneous with respect to both amino acid composition and metal speciation resulting in the observed reverse-phase separation. <br><br> Physiological Characteristics of Cyanobacterial MT <br><br> Using the Brdicka polarographic procedure, it is possible to measure basal MT levels in Synechococcus strain <br><br> 5 <br><br> DAYS <br><br> 10 <br><br> 15 <br><br> FIGURE 6. Cell numbers and MT concentrations as a function of time after inoculation of wild-type cyanobacteria in the presence and absence <br><br> of (A) 22.5 ,uM CdCl2; (B) control; (C) 50 ,uM ZnSO4; (D) 50 ,uM CuCl2 (8). <br><br> 73 <br><br> 74                            ~~~~~~~~~~R. W  OLAFSON <br><br> Table 1. Amino acid compositions of <span class="yellow">cyanobacterial</span> <br><br> metallothioneins compared with eukaryotic metallothioneins. <br><br> Nearest integer residues per molecule <span class="yellow">Human</span> (MT-2)    Crab (MT-2)     Tx-20t CysA                  20            18            9.5 Asx                    4             6            5.7 Thr                    2             3            4.8 Ser                    8             5            3.8 Glx                    2             5            2.0 Gly                    5             3            8.0 Ala                    7              1           4.0 Val                    1             -            2.3 Met                    1                    -       <br><br> Ile                    1             -1.1 <br><br> Leu Tyr Phe His Lys Arg Pro <br><br> 8 2 <br><br> 8 2 6 <br><br> 2.9 2.3 <br><br> 2.8 2.6 1.0 1.9 <br><br> RRIMP NI cells cultured in the absence of added metal, as shown in Figure 6B (8). Comparison with cultures exposed to cadmium chloride (Fig. 6A) or zinc sulfate (Fig. 6C) showed that introduction of metal salts resulted in a growth lag and that resumption of growth occurred coincident with an increase in cellular levels of MT. In addition, it should be noted that, like mammalian MT, cadmium appears to be a more potent inducer than zinc. Twice as much cadmium-thionein was synthesized on exposure to half as much metal as was used in the zinc induction experiment. A further interesting finding was that copper exposure resulted in an even greater growth lag than observed with cadmium, but on resumption of growth in the presence copper, MT synthesis was not noted (Fig. 6D). Recent results suggest that this copper resistance is manifest via a membrane exclusion mechanism (unpublished results). <br><br> HOURS <br><br> FIGURE 7. Inhibition of cadmium induced synthesis by actinomycin <br><br> D (hatched bar) and chloramphenicol (solid bar). Control cells are shown by unmarked bars. <br><br> In order that the level of regulation of metal induction be determined for cyanobacteria, logarithmically growing cultures were exposed to cadmium and MT concentrations measured at several time intervals thereafter. Two cultures were exposed to either chloramphenicol or actinomycin D 30 min prior to introduction of metal, while a third culture was used as a control. The results of this experiment are shown in Figure 7 and indicate induction at the level of transcription, as was found in eukaryotic organisms. <br><br> The assumption at this stage in these investigations was that the growth lag observed in cultures exposed to <br><br> Table 2. Amino acid sequence analysis. <br><br> Amino acid <br><br> 1                      10                       20                        30 Cyanobacteria T S T T L V K C A C E P C L C N V D P S K A I D R N G L Y Y <span class="yellow">Scylla</span>        P DP C      C       NDK C DC K EG             EC KT GC K CT S C <span class="yellow">Human</span>    Ac MD P NC     S C AA G DS       CT     C  AG  S C KC     KE    C KC     T  S C <br><br> Neurospora <br><br> G D C G C S G A S S C N C G S G C S C S N <br><br> C G S K <br><br> Table 2. Continued. <br><br> Amino acid <br><br> 40           50          60 C C EAC A H GHT G G <br><br> R CP PC EQ C S SGC K CA NK E DC RK TC SK PC SCC P K K SCC S CCP V GC A KCA Q BC I C K GA SDK C S CC A <br><br> 74 <br><br> I <br><br> II <br><br> I I <br><br> <span class="yellow">CYANOBACTERIAL</span> METALLOTHIONEINS                       75 <br><br> cadmium or zinc was due to a rather protracted time of MT induction, perhaps associated with toxic burden. If the above cadmium-resistant cells were transferred into fresh media in the absence of cadmium and allowed to grow up between three successive transfers, levels of MT dropped to near basal values as synthesis was repressed in the absence of metal. However, when these cells were now transferred into cadmium-containing media, instead of the predicted growth lag, they grew immediately. Thus, these cells were truly cadmium resistant. Since 50 tubes of wild-type cells at a dilution of 104 grew, after the usual lag on exposure to cadmium-containing media, the acquisition of cadmium resistance in this strain of Synechococcus was therefore considered unlikely to be related to a chromosomal mutational event. Such a mutation frequency would be unreasonably high. Although no direct evidence exists at this time, this metal resistance phenomenon is best explained by the amplification of an extrachromosomal gene, especially since this strain is known to have plasmids. <br><br> Structure of Cyanobacterial MT <br><br> The structural determination of <span class="yellow">cyanobacterial</span> MT is not yet complete, although a substantial amount of information is now known. For example, Table 1 compares the amino acid composition of a <span class="yellow">Synechococcus TX-20</span> isoprotein with two eukaryotic MTs. These data indicate that the bacterial form is unique. While cysteine was still the predominant amino acid in this protein, levels of the amino acid were half that seen in the eukaryotes. In addition, the bacterial protein had two tyrosines, three histidines, and five long-chain aliphatics-all residues rarely found in eukaryotic MTs. Thus, the amino acid composition indicates that prokaryotic MTs are not as structurally homologous with eukaryotes as they are physiologically homologous. This was further exemplified on amino terminal amino acid sequence analysis (12). With the exception of the typical Cys-X-Cys sequences, the first 42 residues of the prokaryotic MT were essentially without homology on comparison with the <span class="yellow">crab</span> (<span class="yellow">Scylla</span>), <span class="blue">human</span>, or <span class="yellow">Neurospora crassa</span> MTs (Table 2). Of the above-mentioned uncommon MT residues, two tyrosines were found together in positions 29 and 30, all five of the long-chain aliphatic amino acids were located in the first 28 residues, and two histidines were situated at positions 37 and 39. Perhaps of greater interest was the complete lack of association of basic residues or hydroxylated residues with cysteines, as is found in eukaryotes (11). Instead, a block of four hydroxylated amino acids was found at the amino terminus. <br><br> The sequence of this unique MT is now nearing completion, opening the way for an X-ray crystallographic investigation. However, the degree of similarity between the metal cluster arrangement of these bacterial molecules with the eukaryotic structures will initially be undertaken spectroscopically. Reliable interpretation of the 113Cd-NMR data for this molecule will be dependent upon adequate isolation of isoproteins using RP-HPLC and assessment of conformational integrity of metal reconstituted molecules, using procedures such as cyclic voltametry. Studies along these lines are presently underway in this laboratory <br><br> The author gratefully acknowledges the financial support of the Australian Institute of Marine Science and the National Science and Engineering Research Council of Canada. Excellent technical assistance was provided by R. G. Sim and S. Kielland in carrying out the polarographic and sequence analyses, respectively. <br><br> REFERENCES <br><br> 1. Kagi, J. H. R., and Nordberg, M. (Eds.). Metallothionein. Birk<br><br> hauser Verlag, Basel, 1979, pp. 5-55. <br><br> 2. Olafson, R. W Differential pulse polarographic determination of <br><br> <span class="yellow">murine</span> metallothionein induction kinetics. J. Biol. Chem. 256:12631268 (1981). <br><br> 3. Olafson, R. W Intestinal metallothionein: effect of parenteral and <br><br> enteral zinc exposure on tissue levels of <span class="yellow">mice</span> on controlled zinc diets. J. Nutr. 113: 268-275 (1983). <br><br> 4. McCarter, J.A., and Roch, M. Chronic exposure of <span class="yellow">Coho salmon</span> <br><br> to sublethal concentrations of copper. III. Kinetics of metabolism of metallothionein. Comp. Biochem. Physiol. 77C: 83-87 (1984). <br><br> 5. Lerch, K., Ammer, D., and Olafson, R. W Crab metallothionein<br><br> primary structures of metallothioneins 1 and 2. J. Biol. Chem. 257: 2420-2426 (1982). <br><br> 6. Otvos, J. D., Olafson, R.W, and Armitage, I. M. Structure of an <br><br> invertebrate metallothionein from <span class="yellow">Scylla serrata</span>. J. Biol. Chem. 257: 2427-2431 (1982). <br><br> 7. Olafson, R. W, Abel, K., and Sim, R. G. Prokaryotic metallothi<br><br> onein: preliminary characteristics of a blue-green alga heavy metalbinding protein. Biochem. Biophys. Res. Commun. 89: 36-40 (1979). 8. Olafson, R. W, Loya, S., and Sim, R. G. Physiological parameters <br><br> of prokaryotic metallothionein induction. Biochem. Biophys. Res. Commun. 95: 1495-1503 (1980). <br><br> 9. Brdicka, R. Pblarographic studies with the dropping mercury ka<br><br> thode.-Part XXXI.-A new test for proteins in the presence of cobalt salts in ammoniacal solutions of ammonium chloride. Collect. Czech. Chem. Commun. 5: 112-128 (1933). <br><br> 10. Olafson, R. W, and Sim, R. G. An electrochemical approach to <br><br> quantitation and characterization of metallothionein. Anal. Biochem. 100: 343-351 (1979). <br><br> 11. Olafson, R. W Differential pulse polarographic determination of <br><br> <span class="yellow">murine</span> metallothionein induction kinetics. J. Biol. Chem. 256: 12631268 (1981). <br><br> 12. Olafson, R. W Prokaryotic metallothionein: amino terminal se<br><br> quence analysis of a unique metallothionein. Int. J. Peptide Protein Res. 24: 303-308 (1984). <h3>pmcA1521150</h3>Peculiarities of carcinogenesis under simultaneous oral administration of benzo(a)pyrene and o-cresol in <span class="yellow">mice</span>.
Abstract
A modifying influence of ortho-cresol (o-cresol) on the carcinogenic effect of benzo(a)pyrene (BaP) with combined oral administration to CC57Br <span class="yellow">mice</span> had been found. During simultaneous administration of o-cresol (1 mg) and BaP (1 mg), the incidence of tumors, the multiplicity of tumors, and the degree of malignancy all increased, but the latency was shortened. When o-cresol was administered before or after BaP (in identical doses), the carcinogenic effect was weakened. When o-cresol (10 mg) and BaP (5 mg) were administered simultaneously, the incidence of malignant tumors was similar to controls receiving BaP only (13.8%), indicating inhibition of carcinogenesis.<br><br><br><br>
 Environnmental Health Perspectives Supplements Vol. 101 (Suppl. 3): 341-344 (1993) <br><br> Peculiarities of Carcinogenesis under Simultaneous Oral Administration of <br><br> Benzo(a)Pyrene and o-Cresol in <span class="yellow">Mice</span> <br><br> by N. Ya. Yanysheva,' N. V. Balenko, 1. A. Chernichenko,1 and V. F. Babiy' <br><br> A modifying influence of ortho-cresol (o-cresol) on the carcinogenic effect of benzo(a)pyrene (BaP) with combined oral administration to CC57Br <span class="yellow">mice</span> had been found. During simultaneous administration of o-cresol (1 mg) and BaP (1 mg), the incidence of tumors, the multiplicity of tumors, and the degree of malignancy all increased, but the latency was shortened. When o-cresol was administered before or after BaP (in identical doses), the carcinogenic effect was weakened. When o-cresol (10 mg) and BaP (5 mg) were administered simultaneously, the incidence of malignant tumors was similar to controls receiving BaP only (13.8%), indicating inhibition of carcinogenesis. <br><br> Introduction <br><br> Numerous experiments have shown a modifying influence of different chemicals on benzo(a)pyrene (BaP)induced carcinogenic effects. Both enhancing or weakening effects have been seen (1-12). In some experiments ubiquitious environmental carcinogenic and toxic chemical pollutants were used (13-17). <br><br> The stimulating effect of phenol, nitrogen oxides, and sulfur dioxide upon BaP's blastogenic action on the respiratory tract and phenol on the digestive tract (forestomach) have been reported (12-18). We have established the relationship between the doses of carcinogenic and toxic agents and their modifying effects (17,18). The maximal enhancing effect was observed after BaP (2.5 mg) and NO2 (0.87 mg/m3) were exposed to <span class="yellow">rat</span> respiratory tract. The effect weakened with decreasing dose. At concentrations at the MPC level (MPC BaP, 1 ng/m3, MPC NO2, 0.04 mg/m3) no effect was seen. Phenol at 1 mg orally to CC57Br <span class="yellow">mice</span> enhanced the BaP (1 mg) effect, but no effect was seen at 0.02 mg. Phenolic compounds showed either enhancing or inhibiting effects on carcinogenesis depending on their chemical structure (16,18-25). <br><br> This report presents the results of combined oral administration of BaP and artho-cresol (o-cresol) in <span class="yellow">mice</span>, <br><br> 'Ukrainian State Medical Center for Environmental Health, Kiev, Ukraine. <br><br> Address reprint requests to N. Y. Yanysheva, Ukrainian State Medical Center for Environmental Health, Popudrenko St. 50, 253660 Kiev-94, Ukraine. <br><br> chemicals commonly found in ambient water because of their industrial use (coke chemistry, oil processing, shale processing, and other industries) (27-30). <br><br> Materials and Methods <br><br> The experiment was conducted on CC57Br female <span class="yellow">mice</span> weighing 12-14 g. Animals were divided in 15 groups. This experiment included three types of sequential combinations for the introduction of compounds: a) simultaneous BaP and o-cresol administration; b) BaP after o-cresol (stage 1); and c) BaP before o-cresol (stage 2). Appropriate controls were included (Table 1). <br><br> The chemicals were administered twice per week (for a total of 10, 20, and 40 doses) using a syringe and a needle with a soldered olive on its tip. BaP (1 or 5 mg) or o-cresol (0.02, 1, or 10 mg) was placed in triethylene glycol (TEG) and administered as 0.1-mL water solutions. The evaluation of o-cresol's modifying effect on the incidence of tumors, especially malignant tumors, the tumor latency period (tl), and the average time (ta) of the appearance of tumors, as well as the multiplicity index, M (the average number of tumors per animal for tumor-bearing animals) was recorded. Because the M could be identified at the initial stage of carcinogenesis by the third month after the beginning of the trial when neoplasms began to emerge, moribund animals were killed by ether inhalation if they did not die spontaneously. <br><br> The experiments with large chemical doses (BaP, 5 mg; o-cresol, 10 mg) lasted for 30 weeks. Some animals were killed after the 1st, 3rd, 5th, and 10th procedures for the <br><br> YANYSHEVA ET AL. <br><br> evaluation of initial stages of carcinogenesis; others were killed at 26 and 30 weeks. The stomachs were distended with formalin solution and the mucosa examined macroscopically. All tumors > 1 mm in diameter were recorded. Organs and tissues were fised in 10% neutral formalin solution, embedded in paraffin and routine histological slides prepared. Microscopic data were processed according to Mostkovoy (30). <br><br> Results <br><br> The data from this study demonstrate the modifying influence of o-cresol on BaP-induced carcinogenesis with combined oral exposure. The combined exposure and BaP alone caused benign and malignant epithelial tumors of the forestomach. The benign neoplasms were papillomas, and the malignant neoplasms were invasive carcinomas. <br><br> The combined administration of BaP and o-cresol showed different results depending on doses and sequential combinations of both chemicals. As shown in Table 2, the simultaneous administration of BaP (1 mg) and o-cresol (1 mg) affected all parameters of carcinogenesis that were measured. This included a significant increase in the incidence of tumors, shortening of the time to the appearance of the first tumor, and the mean time of tumor <br><br> development as compared to animals that received 1 mg of BaP alone and had tumors. One hundred percent of the <span class="yellow">mice</span> that received 5 mg of BaP alone had tumors. <br><br> Shortening of the time to appearance of malignant tumors was also observed: the <span class="yellow">mice</span> with forestomach cancer died between 23 and 26.2 weeks after the beginning of experiment, but the controls survived to 58.5 weeks. Muliple small metastases in the lungs and mediastinum were found in 42.8% of <span class="yellow">mice</span>, which signifies a high degree of malignancy. <br><br> o-Cresol at 0.02 mg did not modify carcinogenesis in comparison to the control. o-Cresol administration before or after BaP did not modify tumor incidence or the multiplicity index as compared to control, but the latency period was longer (Table 2). When o-cresol was administered after BaP (stage 2), there was an absence of malignant tumors. <br><br> Besides the quantitative aspects, other peculiarities of carcinogenesis in <span class="yellow">mice</span> simultaneously administered BaP and o-cresol should be noted. The combined chemical administration resulted in diffuse verrucosa vegetations over the forestomach surface, especially near the greater curvature. Highly aggressive malignant neoplasms developed earlier with more metastases as compared to control. <br><br> Many <span class="yellow">mice</span> were emaciated, and the tumors could be <br><br> Table 1. Scheme of BaP and o-cresol combined action under different regimes of oral administration. <br><br> Theatmenta <br><br> No. of <span class="yellow">mice</span>                        Stage 1                                            Stage 2 <br><br> Groups of        in each                                             No. of                                           No. of <br><br> animals          group        Substances           Dose, mg      applications    Substances         Dose, mg       applications <br><br> 1              55         BaP, o-cresol          5 +10            10          -                     -               2              45          BaP                     5              10          -                     -               3              40          TEG                     la             10          -                     -               4              45          TEG, o-cresol        1b + 10           10          -                     -               5              30          BaP, o-cresol         1+1              20          -                     -               6              30          BaP, o-cresol        1 + 0.02          20          -                     -               7              30          o-Cresol                1              20          BaP                   1               20 8              30          BaP                     1              20          o-Cresol              1               20 9              30          BaP                     1              20                                -               10              30          -                       -              -           BaP                   1               20 11              30         o-Cresol                 1              20                                -               12              30          -                      -               -           o-Cresol              1               20 13              30         o-Cresol + TEG         1+2'             20                                -               14              30         o-Cresol + TEG        0.02 +2b          20 <br><br> 15              30         TEG                     2b              20          -                     -               Abbreviations: BaP, benzo[a]pyrene; TEG, triethylene glycol. <br><br> Table 2. Occurrence of forestomach tumors in CC57Br <span class="yellow">mice</span> after combined oral administration of BaP and o-cresol. <br><br> Number of animals with forestomach tumors <br><br> Order of administration of substances             Total                         Benign                       Malignant <br><br> (dose, mg)                             Absolute       %        %        M         t,         ta         %        t,        ta BaP (5), o-cresol (10)                    29       100.0     86.2       ND        ND        ND        13.8      ND        ND BaP (1), o-cresol (1)                     19        95.0     60.0       9.6       10.7      16.3      35.0      23.0      25.2 BaP (1), o-cresol (0.02)                   7        35.0     30.0       1.6       13.5      19.8       5.0      56.8      56.8 BaP (1), stage 1, o-cresol (1), stage 2    7        31.8     31.8       1.4       16.2      31.1       0         0         0 <br><br> BaP (1), stage 2                           6        35.3     29.4       2.8       15.8      41.2       5.9      24.8      24.8 BaP (5), stage 1                          18       100.0     50.0       ND        ND        ND        50.0      ND        ND BaP (1), stage 1                          8         33.3     28.7       1.4       14.0      21.3       4.6      58.5      58.5 BaP (1), stage 2                           7        36.8     31.6       2.8       10.0      13.3       5.2      55.7      55.7 <br><br> Abbreviations: BaP, benzo[a]pyrene; 0, not observed; ND, not determined; t1, time of the first tumor appearance in weeks; ta, mean time of tumor development in weeks; M, multiplicity. <br><br> 342 <br><br> CARCINOGENESIS OF BaP AND o-CRESOL                           343 <br><br> palpated through the abdominal wall. At autopsy, enlargement of the stomach with tuberculous white superficial vegetations were observed. The forestomach and glandular part of the stomach was often obliterated by tumor masses. <br><br> The stomach was often adhered to the pancreas, liver, and mesentery. Hemmorrhage and inflammation were found in tumors foci. When o-cresol was introduced before BaP, the tumors were more frequently found closer to the small curvature of the stomach between the forestomach and esophageal entrance. Over the large curvature less prominent mucosal folds were observed. Microscopically, a decrease in mucosa papillae, epithelial atrophy (one to two cullular layers), decreased keratonization, and nuclear pycnosis were observed. With the simultaneous administration of large doses of o-cresol and BaP, the final carcinogenic effect (30 weeks after the first dose) was similar to BaP alone but differences were observed at the earlier stages (after the 1st, 3rd, 5th, and 10th exposures). <br><br> In the BaP control, multiple forestomach epithelial proliferative and hyperplastic changes were found after the third dose. Multiple papillomas occurred (9-15 in each <span class="yellow">mouse</span>). The stomach's mucosal folds appeared thickened diffusely, then papillomas appeared and finally merged together. <br><br> Approximately half of the tumors were malignant. The neoplasms filled almost the whole forestomach cavity and infiltrated the wall with tuberculous vegetations, which were visible on the serosal surface. Thmor infiltration in the liver, pancreas, and wide dissemination of peritoneum were also observed. In <span class="yellow">mice</span> simultaneously administered BaP and o-cresol, the proliferative alterations of forestomach epithelium were seen after the third dose. However, they occurred as the single foci at damaged mucosa and even in the later stages were not diffuse. Adjacent to the hyperplastic foci, the mucosa was atrophied with decreased keratonization. The epithelial cells also showed cytoplasm coagulation and nuclear pycnosis. Edema, inflammation of the mucosa, submucosa microabscesses, and erosion were seen. There were fewer papillomas per <span class="yellow">mouse</span> (four to eight in each <span class="yellow">mouse</span>) than in the control group. Even in the late stages the papillomas were isolated and elevated on the atrophied mucosal folds. <br><br> In the final experiment the papillomas prevailed as compared to the previous experiments. Only 4 out of 29 <span class="yellow">mice</span> (13.8%) developed malignant tumors. Neoplasms were smaller and appeared as single verrucosa vegetations 5-10 mm in diameter. Thus, the toxic dose of o-cresol inhibits the carcinogenic process of induced forestomach tumors by decreasing multiplicity, frequency, and percentage of malignant tumors. <br><br> Discussion <br><br> Our results and the literature suggest a hypothesis on modifying carcinogenesis mechanisms. The primary effect of the toxic agents, including carcinogenic agents, may relate to cellular membrane damage with the subsequent increased permeability (31-35), which may be the mechanism of the o-cresol. <br><br> With simultaneous introduction of o-cresol at low toxic doses and BaP there may be increased carcinogen penetration to the target cells. In addition, membrane damage may alter other cellular systems responsible for energy and xenobiotic detoxication. Damage of these processes may activate free-radical reactions, glycolysis, or alter carcinogenic metabolism, which promotes the oncogenic effect. The o-cresol effect on these systems was confirmed by our previous investigations on cytochrome P450, ferrosulfuric nonhemic proteins, and semiquinon radical content affected by the combined action of BaP and o-cresol (36). <br><br> Another effect was obtained with BaP exposure after o-cresol. The atrophic alterations in the stomach induced by 2.5 months of o-cresol administration might decrease the natural conditions of retention (muscosal folds, frontier torn), and thus shorten the contact of BaP with the forestomach. In our opinion, this explains the decreased carcinogenic effect seen after o-cresol exposure. <br><br> Considering the toxic effect of o-cresol at low toxic doses (1 mg) after BaP and the effect of a super toxic dose (10 mg) simultaneously with BaP administration, it is possible that the inhibition of carcinogenesis in both trials was related to the toxic effects of o-cresol. Cytotoxicity may not only hamper tumor growth, but also promote the regression of inducible and spontaneous neoplasms (37-39). There is also a possibility that BaP damages metabolic systems and decreases resistance to carcinogenesis. <br><br> Conclusions <br><br> The data obtained demonstrate that simultaneous administration of BaP modifies the induced carcinogenesis depending on the dose and the sequence of administration. Administered with BaP (1 mg), a low toxic o-cresol dose (about minimally effective) produces an enhanced cocarcinogenic effect reflected in the incidence, frequency, multiplicity, and degree of malignancy of forestomach tumors. <br><br> o-Cresol administration at low toxic doses before or after BaP at the same dose level (1 mg) and its administration at super toxic doses (10 mg) with the BaP optimal dose (5 mg) may inhibit carcinogenesis. Simultaneous introduction of a noneffective o-cresol dose (0.02 mg) with the BaP (1 mg) does not change its carcinogenic activity. Controlling both chemicals in the environment is the most effective measure of preventing potential risk and is undoubtedly of paramount significance. <br><br> REFERENCES <br><br> 1. Bingham, E., and Falk, H. L. Environmental carcinogenesis. The <br><br> modifying effect of cocarcinogens on the threshold response. Arch. Environ. Health 19: 779-783 (1969). <br><br> 2. Goldschmidt, B. M., Katz, C., and Van Duuren, B. L. The cocar<br><br> cinogenic activity of non-carcinogenic aromatic hydrocarbons. Proc. Am. Assoc. Cancer Res. 17: 84 (1973). <br><br> 3. Kaufman, D. C., and Madison, R. M. Synergistic effects of <br><br> benzo(a)pyrene and N-methyl-N-nitrosourea respirators carcinogenesis in <span class="yellow">Syrian golden hamsters</span>. In: Proceedings of the Sym<br><br> 344                                     YANYSHEVA ET AL. <br><br> posium on Epithelial Respiratory Carcinogenesis and Bioassays. Batelle Seattle Research Center, Seattle, WA, 1974, p. 17. <br><br> 4. Montesano, R., Saffiotti, U, Ferrero, A, and Kaufman, D. Synergistic <br><br> effects of benzo(a)pyrene and diethylnitrosamine on respiratory carcinogenesis in hamsters. J. Natl. Cancer. Inst. 5: 1395-1397 (1974). 5. Pfeiffer, E. H. Investigations of the carcinogenic burden by air <br><br> pollution in <span class="yellow">man</span>. VII. Studies on the ontogenetic interaction of polycyclic aromatic hydrocarbons. Zbl. Bakteriol. Parasitenk. Infektion. Hyg. Abt. B 158(1): 69-83 (1973). <br><br> 6. Slaga, T. J., Jecker, L., Bracken, W. M., and Weeks, C. E. The effects of <br><br> weak or non-carcinogenic polycyclic hydrocarbons on 7,12-dimethylbenz(a)anthracene and benzo(a)pyrene skin tumor-initiation. Cancer Lett. 7: 51-59 (1979). <br><br> 7. Topping, D. C., Martin, D. H., and Nettesheim, P. Determination of <br><br> cocarcinogenic activity of benzo(a)pyrene for respiratory tract mucosa. Cancer Lett. 11: 315-321 (1981). <br><br> 8. Ilnitsky, A. P., Voronin, V. M., and Solenova, M. G. Carcinogenic effect <br><br> of benzo(a)pyrene and its combinations with other carbohydrates at their oral intake [in Russian]. Voprosy Onkol. 10: 71-75 (1974). <br><br> 9. Kurljandsky, B. A., Medvedovsky, A. G., and Mashbits, F. D. On <br><br> cocarcinogenic effect of toxic factors of low intensity [in Russian]. Gig. Sanit. 10: 83-85 (1972). <br><br> 10. Thrusov, V. S., and Andrianov, L. A. On combined effect on the skin of <br><br> single application of 7,12-dimethylbenz(a)anthracene and repeated applications of benz(a)pyrene [in Russian]. Bull. Eksper. Biol. Med. 3: 80-83 (1972). <br><br> 11. Yanysheva, N. Ya., Chernichenko, I. A., Balenko, N. V, and Kireeva, <br><br> I. S. Carcinogenic substances and their hygienic standardization in the environment [in Russian]. Zdorovja, Kiev, 1977. <br><br> 12. Laskin, S., Kushner, M., and Drew, R. T. Studies in pulmonary <br><br> careinogenesis. In: Inhalation Carcinogenesis (M. G. Hanna, P. Nettesheim, and J. R. Gillbert, Eds.) U.S. Atomic Energy Commission Symposium, Series 18, 1970, p. 321. <br><br> 13. Laskin, S., Kuschner, M., and Drew, R. T. Inhalation exposure to <br><br> sulphur dioxide and benzo(a)pyrene. Oak Ridge National Laboratory, Gatlinburg, TN, 1970, pp. 322-351. <br><br> 14. Kuschner, M., and Laskin, S. Experimental models in environmental <br><br> carcinogenesis. Am. J. Pathol. 1: 183-196 (1971). <br><br> 15. Komine, T. Influences of sulfur dioxide and 3,4-benzpyrene on the <br><br> respiratory organ of <span class="yellow">rats</span>. Hokkaido Igaku Zasshi 3: 189-203 (1977). 16. Skvortsova, N. N. Role of some common atmospheric pollutions in <br><br> lung blastomogenesis [in Russian]. In: Handbook of Hygiene of Atmospheric Air (K. A. Bushtueva, Ed.), Meditsina, Moscow, 1976, pp. 384-391. <br><br> 17. Yanysheva, N. Ya., Balenko, N. V, Chernichenko, I. A, and Babiy, V. F. <br><br> Quantitation of modified effects of nitrogen oxides on carcinogenicity of benz(a)pyrene effect [in Rusian]. Gig. Sanit. 7: 7-9 (1986). <br><br> 18. Yanysheva, N. Ya., Balenko, N. V., Chernichenko, I. A., Babiy, V. F., <br><br> Bakanova, G. N., and Lemeshko, L. P. Peculiarities of manifestation of carcinogenesis at combined effect of benz(a)pyrene and phenol depending on regimen of intake into the organism [in Russian]. Gig. Sanit. 4: 29-33 (1988). <br><br> 19. Van Duuren, B. L., Katz, C., and Goldschmidt, B. M. Brief communica<br><br> tion: cocarcinogenic agents in <span class="yellow">tobacco</span> carcinogenesis. J. Natl. Cancer Inst. 51: 703-705 (1973). <br><br> 20. Van Duuren, B. L., and Goldschmidt, B. M. Cocarcinogenic and <br><br> tumor-promoting agents in <span class="yellow">tobacco</span> carcinogenesis. J. Natl. Cancer Inst. 56: 1237-1242 (1976). <br><br> 21. Wattenberg, L. W., Coccia, J. B., and Lam, L. K. Inhibitory effects of <br><br> phenolic compounds on benzo(a)pyrene-induced neoplasia. Cancer Res. 8: 2820-2823 (1980). <br><br> 22. Bogovski, P. A, and Mirme, H. Yu. Cocarcinogenicity of phenols from <br><br> Estonian shale tars (oils). Environ. Health Perspect. 30: 177-178 (1979). <br><br> 23. Gubergritz, M. Ya., and Kirso, U. E. Carcinogenic properties, struc<br><br> ture and reactive capacity of phenols [in Russian]. Voprosy Onkol. 8: 96-100 (1970). <br><br> 24. Mirme, H. Yu. Modifying effect of water-soluble shale phenols upon <br><br> carcinogenesis [in Russian]. In: Problems of Prevention of Pollution of Environment by Carcinogenic Substances (M. M. Shabad, Ed.), Tallinn, Valgus, 1972, pp. 16-18. <br><br> 25. Kirso, U. E., Pashin, Yu. V., Bakhitova, L. M., and Kjung, A. I. Effect <br><br> of antioxidants on carcinogenic and mutagenic activity of benzo(a)pyrene [in Russian]. Voprosy Onkol. 4: 70-75 (1985). <br><br> 26. Veldre, I. A. Comparative assessment of toxic effect of some water<br><br> soluble phenols [in Russian]. In: Proceedings of the Scientific Conference on Urgent Problems Related with Decrease of Infectious Diseases Morbidity and with Hygienic Problems. Tallinn, 1968, pp. 200-202. <br><br> 27. Substances possessing cocarcinogenic effect. In: Carcinogenic Sub<br><br> stances in the Environment of <span class="yellow">Man</span> (L. M. Shabad and A. P. Ilnitsky, Eds.), Budapest, 1979, pp. 111-113. <br><br> 28. Djatlovitskaya, F. G., and Maktas, E. D. Separate determination of <span class="yellow">fly</span> <br><br> phenols in water using thin-layer chromatography [in Russian]. Gig. Sanit. 6: 60-63 (1965). <br><br> 29. Kostovetsky, Ya. I., and Zholdakova, Z. I. About hygienic rate setting <br><br> of phenol in the water of waterpools [in Russian]. Gig. Sanit. 7: 7-10 (1971). <br><br> 30. Mostkovoy, M. I. Practicum on variational-statistical processing of <br><br> clinical material [in Russian]. Ashkhabad, 1954, p. 132. <br><br> 31. Weinstein, I. B. The scientific basis for carcinogen detection and <br><br> primary cancer prevention. Cancer 47: 1133-1141 (1981). <br><br> 32. Merkulov, A. I, and Skvortsova, R. I. About toxic effect of phenol [in <br><br> Russian]. Gig. Sanit. 1: 79-80 (1984). <br><br> 33. Weinstein, I. B., Mufson, R. A., Lee, L. S., et al. Membrane and other <br><br> biochemical effects of the phorbol esters and their relevance to tumor promotion. In: Carcinogenesis. Fundamental Mechanisms and Environmental Effects. London, 1980, pp. 543-563. <br><br> 34. Yamasaki, H., and Weinstein, I. B. Cellular and molecular mecha<br><br> nisms of tumor promotion and their implications for risk assessment. In: Methods for Estimating Risk of Chemical Injury: <span class="yellow">Human</span> and Non-<span class="blue">human</span> Biota and Ecosystems (V. B. Vouk, G. C. Butler, D. G. Hoel, and D. B. Peakall, Eds.), SCOPE, 1985, pp. 155-180. <br><br> 35. Karu, T, Kirso, U. E., and Andrianov, L. A. Dynamics of resorption of <br><br> 3,4-benzpyrene with phenols from <span class="yellow">mouse</span> skin [in Russian]. Voprosy Onkol. 5: 80-84 (1973). <br><br> 36. Yanysheva, N. Ya., Yurkovskaya, T. N., Beregovskaya, N. N., et al. <br><br> Metalloenzymic complexes of energetic and detoxicating systems of cell during benz(a)pyrene-induced carcinogenesis. In: Proceedings of the 4th National Congress of Oncology with International Participation. Sofia, 1985, p. 102. <br><br> 37. Mizell, M. Anuran (Lucke) tumor breakdown in regenerating anuran <br><br> tadpole tails. Anat. Res. 137: 382-383 (1960). <br><br> 38. Sheremetieva, E. N. Spontaneous melanoma in regenerating tails of <br><br> axolotis. J. Exp. Zool. 158: 101-122 (1965). <br><br> 39. Tsonis, P. A, and Eguchi, G. Carcinogens on regeneration. Effects on <br><br> N-methyl-N-nitro-N-nitrosoguanidine and 4-nitrogninoline-1-oxide on limb regeneration in adult newts. Differentiation 20: 52-60 (1981). <h3>pmcA1526545</h3>B lymphocyte stimulator (BLyS) isoforms in systemic lupus erythematosus: disease activity correlates better with blood leukocyte BLyS mRNA levels than with plasma BLyS protein levels
Abstract
Considerable evidence points to a role for B lymphocyte stimulator (BLyS) overproduction in <span class="yellow">murine</span> and <span class="blue">human</span> systemic lupus erythematosus (SLE). Nevertheless, the correlation between circulating levels of BLyS protein and disease activity in <span class="blue">human</span> SLE is modest at best. This may be due to an inadequacy of the former to reflect endogenous BLyS overproduction faithfully, in that steady-state protein levels are affected not just by production rates but also by rates of peripheral utilization and excretion. Increased levels of BLyS mRNA may better reflect increased in vivo BLyS production, and therefore they may correlate better with biologic and clinical sequelae of BLyS overexpression than do circulating levels of BLyS protein. Accordingly, we assessed peripheral blood leukocyte levels of BLyS mRNA isoforms (full-length BLyS and ΔBLyS) and plasma BLyS protein levels in <span class="yellow">patients</span> with SLE, and correlated these levels with laboratory and clinical features. BLyS protein, full-length BLyS mRNA, and ΔBLyS mRNA levels were greater in SLE <span class="yellow">patients</span> (n = 60) than in rheumatoid arthritis <span class="yellow">patients</span> (n = 60) or normal control individuals (n = 30). Although full-length BLyS and ΔBLyS mRNA levels correlated significantly with BLyS protein levels in the SLE cohort, BLyS mRNA levels were more closely associated with serum immunoglobulin levels and SLE Disease Activity Index scores than were BLyS protein levels. Moreover, changes in SLE Disease Activity Index scores were more closely associated with changes in BLyS mRNA levels than with changes in BLyS protein levels among the 37 SLE <span class="yellow">patients</span> from whom repeat blood samples were obtained. Thus, full-length BLyS and ΔBLyS mRNA levels are elevated in SLE and are more closely associated with disease activity than are BLyS protein levels. BLyS mRNA levels may be a helpful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>.<br><br>Introduction
B lymphocyte stimulator (BLyS; a trademark of <span class="yellow">Human</span> Genome Sciences, Inc., Rockville, MD, USA) is a 285-amino-acid member of the tumor necrosis factor ligand superfamily [1-3]. A causal relation between constitutive overproduction of BLyS and development of systemic lupus erythematosus (SLE)-like illness has incontrovertibly been established in <span class="yellow">mice</span>. BLyS-transgenic <span class="yellow">mice</span> often develop SLE-like features as they age [3-5], and SLE-prone (NZB × NZW)F1 (BWF1) and MRL-lpr/lpr <span class="yellow">mice</span> respond clinically to treatment with BLyS antagonists (decreased disease progression and improved survival) [3,6].
Considerable inferential evidence points to a role for BLyS overproduction in <span class="blue">human</span> SLE as well. Cross-sectional studies have demonstrated elevated circulating levels of BLyS in 20–30% of <span class="blue">human</span> SLE <span class="yellow">patients</span> tested at a single point in time [7,8]. Moreover, a 12-month longitudinal study documented persistently elevated serum BLyS levels in about 25% of SLE <span class="yellow">patients</span> and intermittently elevated serum BLyS levels in an additional 25% of <span class="yellow">patients</span> [9]. Remarkably, circulating BLyS levels did not correlate with disease activity (measured using the SLE Disease Activity Index [SLEDAI]) in these cross-sectional or longitudinal studies [7-9]. Although a statistically significant correlation between circulating BLyS levels and SLEDAI has been appreciated in a more recent 24-month longitudinal study of 245 SLE <span class="yellow">patients</span> (with >1,700 plasma samples analyzed) [10], the correlation remains weak.
The limited correlation between circulating BLyS protein levels and disease activity in these studies may have exposed an inadequacy of the former to reflect faithfully endogenous BLyS overproduction. In addition to the rate of BLyS protein production, several other factors (for example, utilization and excretion) can affect circulating BLyS protein levels. Although there are no practicable means of directly measuring in vivo BLyS production per se in <span class="yellow">humans</span>, the level of BLyS mRNA may serve as a better surrogate marker of in vivo BLyS production than does the level of BLyS protein. Candidate BLyS mRNA isoforms include the full-length BLyS mRNA isoform, which encodes the full-length protein, and the alternatively spliced ΔBLyS mRNA isoform, which encodes a protein with a small peptide deletion [11]. (ΔBLyS does not bind to cells expressing BLyS receptors, and therefore it has no agonistic activity. Moreover, ΔBLyS can form heterotrimers with full-length BLyS, thereby actually functioning as a dominant-negative antagonist of BLyS activity.)
In this report we demonstrate that peripheral blood leukocytes from SLE <span class="yellow">patients</span> express elevated mRNA levels of both full-length BLyS and ΔBLyS relative to those levels expressed by <span class="yellow">patients</span> with rheumatoid arthritis (RA) or by normal control individuals. In the SLE <span class="yellow">patients</span>, both full-length BLyS and ΔBLyS mRNA levels are more closely associated with disease activity (SLEDAI) than are BLyS protein levels. Accordingly, BLyS mRNA levels may be a helpful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>.<br><br>Materials and methods
General details
This study was approved by the institutional review boards of the University of Southern California and the Scripps Research Institute. All <span class="yellow">participants</span> gave their written informed consent before participation in this study.<br><br><span class="yellow">Participants</span>
<span class="yellow">Patients</span> receiving outpatient medical care at the rheumatology clinics of the Los Angeles County + University of Southern California Medical Center were recruited into the study. Diagnoses of SLE (n = 60) or RA (n = 60) were based on established clinical criteria [12]. Healthy control individuals (n = 30) were recruited from Los Angeles County + University of Southern California Medical Center and University of Southern California Keck School of Medicine personnel. No exclusions were made on any basis other than an inability to give informed consent. Each <span class="yellow">patient</span>'s sex, race, age, and medications at the time of the phlebotomy were recorded (Table 1).
Based solely on the <span class="yellow">patient</span>'s willingness to donate a second blood sample, repeat blood samples were collected from 37 of the SLE <span class="yellow">patients</span> 147–511 days (median 371 days) after collection of the first samples. These <span class="yellow">patients</span> were not selected on the basis of any demographic, clinical, or laboratory feature.
Clinical disease activity for the SLE <span class="yellow">patients</span> was assessed using the SLEDAI [13] and using a modified SLEDAI that excludes the contribution of anti-double-stranded DNA (anti-dsDNA) antibodies from the total score. Each <span class="yellow">patient</span>'s medical chart was reviewed for results of standard clinical laboratory tests within the previous or subsequent 1-month period.<br><br>Plasma BLyS determination
Whole venous blood was centrifuged to yield plasma and a buffy coat. The plasma was harvested, stored at -70°C, and assayed for BLyS levels by ELISA [8,14] using Fab fragments of the capture antibody rather than the whole antibody to reduce assay interference by rheumatoid factor. The lower limit of detection in this assay is 0.3 ng/ml. For statistical purposes, plasma samples with BLyS concentrations below the lower limit of detection were assigned a value of 0.25 ng/ml.<br><br>Blood BLyS mRNA determination
The buffy coat from centrifuged whole blood was harvested, added to RNAlater™ (Ambion, Austin, TX, USA) at a 1:4 vol/vol ratio for RNA stabilization, stored at -70°C, and assayed for full-length BLyS and ΔBLyS mRNA levels by real-time PCR. Total RNA was purified from buffy coat samples using RNAeasy miniprep kits (Qiagen, Valencia, CA, USA), and contaminating genomic DNA was removed by DNAse-I digestion. One-tenth volume of total RNA was used as template in the first-strand cDNA reaction using oligo-dT and the Superscript III first-strand synthesis system (Invitrogen, Carlsbad, CA, USA). Duplicate samples of cDNA were amplified with primers against β-actin, full-length BLyS, or ΔBLyS: β-actin sense 5'-CGAGAAGATGACCCAGATCATGT-3'; β-actin anti-sense 5'-GGCATACCCCTCGTAGATGG-3'; full-length BLyS sense 5'-GCAGACAGTGAAACACCAACTATAC-3'; ΔBLyS sense 5'-CAGAAGAAACAGGATCTTACACAT-3'; and full-length BLyS/ΔBLyS anti-sense 5'-TGCCAGCTGAATAGCAGGAATTAT-3'.
A 165 bp amplicon for β-actin was PCR-amplified using the 7900 HT ABI Prism machine (Qiagen) with annealing at 65°C. A 296 bp amplicon for full-length BLyS was PCR-amplified, with annealing at 64°C. A 270 bp amplicon for ΔBLyS was PCR-amplified with annealing at 61°C. The annealing conditions for full-length BLyS and ΔBLyS were determined so that each primer set remained specific to the respective BLyS isoform and yielded a PCR efficiency similar to those of cloned cDNA standards. Melting curve analysis revealed a single peak for each gene amplified. The threshold cycle (Ct) values for each reaction were determined using Sequence Detection System software (Applied Biosystems, Foster City, CA, USA). Results are presented as ratios of full-length BLyS or ΔBLyS mRNA to β-actin mRNA, which were calculated using the following formulae:
2 exp(Ctβ-actin - Ctfull-length BLyS)
2 exp(Ctβ-actin - CtΔBLyS)<br><br>Determination of anti-BLyS autoantibodies
BLyS was bound to microtiter plates by first coating the plates with streptavidin and then adding biotinylated recombinant BLyS. Using these plates as the capture reagent, plasma samples were incubated, and <span class="yellow">horseradish</span> peroxidase-conjugated anti-<span class="blue">human</span> IgA/IgM/IgG (Southern Biotechnology Associates, Birmingham, AL, USA; 1:20,000 final dilution) or <span class="yellow">horseradish</span> peroxidase-conjugated anti-<span class="blue">human</span> IgG (Southern Biotechnology; 1:10,000 final dilution) were used as the detector reagents.<br><br>Statistical analysis
All analyses were performed using SigmaStat software (SPSS, Chicago, IL, USA). Results that did not follow a normal distribution were log-transformed to achieve normality. Parametric testing between two matched or unmatched groups was performed using the paired or unpaired t test, respectively. Parametric testing among three or more groups was performed using one-way analysis of variance. When log-transformation failed to generate normally distributed data or the equal variance test was not satisfied, nonparametric testing was performed using the Mann–Whitney rank sum test between two groups and by Kruskal–Wallis one-way analysis of variance on ranks among three or more groups. Correlations were determined using Pearson product moment correlation for interval data and using Spearman rank order correlation for ordinal data or for interval data that did not follow a normal distribution. Nominal data were analyzed using χ2 analysis-of-contingency tables.<br><br>
Results
Elevated plasma BLyS levels and blood levels of full-length BLyS and ΔBLyS mRNA isoforms in systemic lupus erythematosus <span class="yellow">patients</span>
Previous reports of elevated circulating BLyS levels in SLE <span class="yellow">patients</span> were based on a BLyS ELISA that utilized a whole (unfragmented) capture anti-BLyS monoclonal antibody [7-9]. Since the publication of these reports, it has been recognized that the presence of rheumatoid factor can potentially interfere with the assay and lead to spurious overestimation of the true circulating BLyS levels (<span class="yellow">Human</span> Genome Sciences, Inc.; unpublished observations). To mitigate potential interference from rheumatoid factor, the BLyS ELISA has been modified and the capture anti-BLyS monoclonal antibody is now utilized as a Fab fragment. Despite the changes in the ELISA format, our findings are entirely consistent with those of the previous reports. Plasma BLyS levels were significantly greater in the SLE group than in either RA or normal control group (P < 0.001; Figure 1a). Arbitrary assignment of the 95th percentile value among the normal control individuals as the upper limit of 'normal' revealed that two of the 30 normal control individuals, 15 of the 60 RA <span class="yellow">patients</span>, and 29 of the 60 SLE <span class="yellow">patients</span> harbored elevated plasma BLyS levels (P < 0.001).
Overexpression of BLyS in SLE <span class="yellow">patients</span> was also established by measuring BLyS mRNA levels normalized to β-actin mRNA levels in peripheral blood leukocytes (buffy coats). The geometric mean full-length BLyS mRNA and ΔBLyS mRNA levels among the SLE <span class="yellow">patients</span> were each significantly greater than those among the RA <span class="yellow">patients</span> and normal control individuals, respectively (P < 0.001 for each; Figure 1b,c). Arbitrary assignment of the 95th percentile values for full-length BLyS and ΔBLyS mRNA levels among the normal control individuals as the upper limits of 'normal' revealed that two of the 30 normal control individuals, four of the 60 RA <span class="yellow">patients</span>, and 20 of the 60 SLE <span class="yellow">patients</span> had elevated full-length BLyS mRNA levels (P < 0.001), and that two of the 30 normal control individuals, three of the 60 RA <span class="yellow">patients</span>, and 19 of the 60 SLE <span class="yellow">patients</span> had elevated ΔBLyS mRNA levels (P < 0.001). Levels of full-length BLyS and ΔBLyS mRNA strongly correlated with each other (r = 0.703; P < 0.001) in the SLE cohort, and plasma BLyS levels also correlated significantly with levels of each BLyS isoform (r = 0.429, P < 0.001; and r = 0.290, P = 0.024, respectively). Among these SLE <span class="yellow">patients</span>, none of the measured BLyS parameters correlated with <span class="yellow">patient</span> age, sex, race, or daily dose of corticosteroids (data not shown). Because the racial composition of the normal cohort was not as predominantly Hispanic as were those of the RA and SLE cohorts, we assessed the BLyS parameters in the respective Hispanic subpopulations. As for the entire populations, values for SLE were significantly greater than those for either RA or normal controls (P ≤ 0.004; data not shown).<br><br>Correlations between BLyS parameters and plasma immunoglobulin levels
BLyS is a potent B cell survival factor [15-21], and administration of exogenous BLyS to <span class="yellow">mice</span> leads to B cell expansion and hypergammaglobulinemia [1]. Previous studies with numbers of SLE <span class="yellow">patients</span> greater than were included in the present study documented a modest but significant correlation between serum levels of BLyS and IgG [8,10]. In our SLE cohort of limited size, plasma BLyS levels failed to show significant correlations with plasma levels of total immunoglobulin, IgG, or IgA. In contrast, full-length BLyS and ΔBLyS mRNA levels correlated significantly with each (Figure 2). (None of the BLyS parameters correlated with plasma IgM levels.) The absence of significant correlation between plasma BLyS levels and the immunoglobulin parameters also persisted when just the 53 <span class="yellow">patients</span> with detectable plasma BLyS levels were considered (r = -0.133, P = 0.346 for total immunoglobulin; r = -0.048, P = 0.734 for IgG; and r = 0.033, P = 0.817 for IgA).<br><br>Correlations between BLyS parameters and disease activity
Previous studies either have failed to demonstrate a significant correlation between disease activity and circulating BLyS levels [7-9] or have detected only a weak correlation between the two [10]. Consonant with those studies, we identified no significant correlation between plasma BLyS levels and SLEDAI in our cohort of 60 SLE <span class="yellow">patients</span> (Figure 3a). The failure to demonstrate a significant correlation cannot be attributed to a skewing of the results by the <span class="yellow">patients</span> in whom plasma BLyS levels were below the limit of detection, because no significant correlation was detected among the 53 SLE <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range (r = 0.185, P = 0.183). In contrast, a significant correlation between SLEDAI and full-length BLyS mRNA levels was readily discernible (Figure 3b). A trend toward a correlation between SLEDAI and ΔBLyS mRNA levels was also observed, although it did not achieve statistical significance (Figure 3c).
A component of the SLEDAI is the presence of circulating anti-dsDNA antibodies. Because circulating BLyS levels may affect the presence and/or titers of circulating anti-dsDNA antibodies [7-10], we assessed correlations between the individual BLyS parameters and a modified SLEDAI that excludes any consideration of anti-dsDNA antibodies. As with the unmodified SLEDAI, the modified SLEDAI did not correlate with plasma BLyS levels (Figure 3d) either among the SLE cohort overall or among the 53 <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range (r = 0.160, P = 0.252), but it significantly correlated with full-length BLyS mRNA levels (Figure 3e) and exhibited a trend toward correlation with ΔBLyS mRNA levels (Figure 3f). Thus, the stronger correlations between BLyS mRNA levels and disease activity cannot solely be explained by any effects that BLyS may have on anti-dsDNA antibodies per se.
Moreover, among the 37 SLE <span class="yellow">patients</span> who were evaluated on two separate occasions, trends toward correlation were appreciated between changes in the unmodified or modified SLEDAI and changes in full-length BLyS or ΔBLyS mRNA levels but not changes in plasma BLyS levels (Figure 4). These results cannot be ascribed to changes in medications taken by the <span class="yellow">patients</span>, because changes in neither disease activity nor in any of the BLyS parameters correlated with changes in the doses of corticosteroids or cytotoxics taken by the <span class="yellow">patients</span> (data not shown). The failure to demonstrate a meaningful association between changes in SLEDAI score and changes in plasma BLyS protein levels cannot be attributed to a skewing of the results by the <span class="yellow">patients</span> in whom plasma BLyS levels were below the limit of detection, because the absence of association between the two persisted among the 27 SLE <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range in both samples (r = -0.069, P = 0.727 for plasma BLyS versus unmodified SLEDAI; r = -0.020, P = 0.919 for plasma BLyS versus modified SLEDAI).<br><br>Lack of correlation between levels of BLyS mRNA isoforms and percentages of individual leukocyte cell types
Among cells in peripheral blood, BLyS is predominantly expressed by cells of the myeloid lineage (monocytes and neutrophils) [1,14,22,23]. Accordingly, a shift in the differential leukocyte count away from lymphocytes to monocytes and/or neutrophils could substantially alter BLyS mRNA results. Because of the limited amount of blood we were permitted to obtain from the SLE <span class="yellow">patients</span> (consequent to the high prevalence of anemia among these <span class="yellow">patients</span>), we were unable to purify the individual leukocyte populations for BLyS mRNA analysis. Nevertheless, to demonstrate that the elevated BLyS mRNA levels in SLE did not simply reflect a shift in differential leukocyte count, we assessed the correlations between the individual BLyS parameters on the one hand and the percentages of blood neutrophils, monocytes, and lymphocytes on the other. No correlations were appreciated (Figure 5).<br><br>Presence of anti-BLyS autoantibodies in <span class="yellow">patients</span> with systemic lupus erythematosus
The poorer correlation between plasma BLyS protein levels and disease activity compared with that between BLyS mRNA levels and disease activity was striking. <span class="yellow">Patients</span> with SLE frequently develop autoantibodies against self-antigens, and so some of the SLE <span class="yellow">patients</span> might have harbored autoantibodies to BLyS. Such autoantibodies could have complexed with BLyS and enhanced its clearance, thereby masking BLyS overproduction. Alternatively, such autoantibodies might have sterically blocked the epitopes recognized by the detecting antibodies in the in vitro ELISA. In this case, measured BLyS levels would have been spuriously reduced, again masking BLyS overproduction.
In our cohort, IgA/IgM/IgG anti-BLyS antibodies were detected in six out of the 60 SLE <span class="yellow">patients</span>. Such autoantibodies were also detected in two out of 60 RA <span class="yellow">patients</span> and in one out of 30 normal control individuals, demonstrating that anti-BLyS autoantibodies are not restricted to SLE <span class="yellow">patients</span>. IgG anti-BLyS autoantibodies were detected in 3 SLE <span class="yellow">patients</span> but in no RA <span class="yellow">patients</span> or normal control individuals.<br><br>
Discussion
Elevated blood levels of BLyS protein and mRNA are well described features of <span class="blue">human</span> SLE [7-9]. We confirmed these observations in our study and extended them by documenting increases not just in levels of full-length BLyS mRNA but also in levels of ΔBLyS mRNA (Figure 1). Of note, BLyS mRNA levels were elevated in SLE but not in RA, raising the possibility that BLyS overproduction in SLE is systemic whereas BLyS overproduction in RA may be more focused to the affected arthritic joints [24]. The modestly elevated plasma BLyS protein levels in RA <span class="yellow">patients</span> may reflect, at least in part, release of locally overproduced BLyS into the circulation.
The relationship between circulating BLyS protein levels and disease activity was addressed in several previous studies, but significant correlations between the two measures did not emerge [7-9]. In the largest study to date, a 2-year longitudinal study of 245 <span class="yellow">patients</span> in which more than 1,700 plasma samples were analyzed, a significant but weak correlation between the two was appreciated [10]. In the present study, a significant correlation between plasma BLyS protein levels and disease activity was again not realized (Figure 3a,d).
The weak, at best, correlation between circulating BLyS levels and disease activity is seemingly rather surprising. There is a clear-cut association in BlyS transgenic <span class="yellow">mice</span> between BLyS overexpression and development of SLE-like features [3-5], and treatment of SLE-prone <span class="yellow">mice</span> with BLyS antagonists retards the progression of disease and improves survival [3,6]. Moreover, development of precocious glomerular pathology in autoimmune-prone <span class="yellow">mice</span> correlates strongly with circulating BLyS levels [25].
The likely explanation for the weak correlation between circulating BLyS levels and disease activity in <span class="blue">human</span> SLE is not that disease activity in SLE <span class="yellow">patients</span> is insensitive to the degree of BLyS overproduction. Rather, a more tenable explanation is that circulating BLyS levels in <span class="blue">human</span> SLE do not always accurately reflect excessive endogenous BLyS production. We can identify at least three nonmutually exclusive mechanisms to explain a dissociation between the two.
First, SLE <span class="yellow">patients</span> frequently develop autoantibodies to a myriad of self-targets (for example, erythrocytes, lymphocytes). Indeed, we detected circulating IgA/IgM/IgG anti-BLyS autoantibodies in 10% (6/60) of the tested SLE <span class="yellow">patients</span>, and we detected circulating IgG anti-BLyS autoantibodies in 5% (3/60). These percentages may be underestimates of the true prevalence of anti-BLyS autoantibodies, because some of these autoantibodies may be saturated in vivo with circulating BLyS, rendering them incapable of binding to BLyS in the in vitro detection assay. We do not yet know whether the anti-BLyS autoantibodies are functionally neutralizing but, regardless, such autoantibodies could enhance the clearance of BLyS and/or interfere with in vitro detection of BLyS, thereby masking endogenous BLyS overproduction.
Second, increased urinary excretion of BLyS has been reported in SLE <span class="yellow">patients</span>, especially among those with clinically overt renal involvement [26]. At least four of the <span class="yellow">patients</span> we studied manifested nephrotic-range proteinuria (≥3 g/24 hours), and so urinary loss of BLyS was probably substantial in at least these <span class="yellow">patients</span>. A validated assay for urinary BLyS detection has not yet been developed so we were unable to quantify urinary BLyS levels. Once an assay for urinary BLyS levels is validated, we should be able to assess the effect of urinary BLyS excretion on circulating BLyS levels.
Third, BLyS promotes in vivo expansion of B cells [1]. Freshly isolated SLE B cells, despite intact surface expression of BLyS receptors, bind less biotinylated BLyS ex vivo than do freshly isolated normal B cells [27]. Although other interpretations are possible, the most likely explanation is that BLyS receptors on B cells in SLE <span class="yellow">patients</span> are occupied in vivo by soluble BLyS. Accordingly, it is likely that BLyS receptors expressed by the expanded B cell population do bind BLyS and remove it from the circulation, resulting in a homeostatic pathway that modulates the effects of BLyS overproduction on circulating BLyS levels. Indeed, circulating levels of BLyS rise with peripheral blood B cell depletion and fall with re-emergence of peripheral blood B cells in rituximab-treated RA or SLE <span class="yellow">patients</span> [28,29], highlighting this inverse relationship between circulating BLyS levels and B cell load. Moreover, one of the hallmarks of active disease in <span class="blue">human</span> SLE is the increased percentages of activated B cells and plasma cells in peripheral blood [30-34], probably reflecting increased systemic numbers of activated B cells and plasma cells. Although not yet formally tested, differential BLyS receptor expression by these cells compared with expression by nonactivated B cells may result in increased peripheral BLyS utilization, further dampening the effects of BLyS overproduction on circulating protein levels.
To circumvent these confounding processes, we used BLyS mRNA levels as a surrogate marker of endogenous BLyS production. Overall, the correlations between disease activity and either full-length BLyS or ΔBLyS mRNA levels were much stronger than that between disease activity and BLyS protein levels (Figures 3 and 4). This was the case regardless of whether we used the standard SLEDAI or the modified SLEDAI as a measure of disease activity. These correlations were not spurious ones consequent to shifts in percentages of leukocyte subpopulations in peripheral blood, because BLyS mRNA levels did not correlate with percentages of blood neutrophils, monocytes, or lymphocytes (Figure 5).
A similar pattern was observed between plasma immunoglobulin levels and the BLyS parameters, with plasma levels of total immunoglobulin, IgG, and IgA correlating significantly with full-length BLyS and ΔBLyS mRNA levels but not with plasma BLyS levels (Figure 2). These significant correlations between full-length BLyS or ΔBLyS mRNA levels and plasma immunoglobulin levels again highlight the greater ability of BLyS mRNA levels, compared with plasma BLyS protein levels, to reflect ongoing BLyS overproduction.
At present, it is not known whether soluble ΔBLyS protein is present in the circulation of SLE <span class="yellow">patients</span> or of normal individuals. Although full-length BLyS protein is readily cleaved and released from cells transfected with a vector containing <span class="yellow">murine</span> full-length BLyS, ΔBLyS protein is not cleaved or released from <span class="yellow">murine</span> ΔBLyS transfectants [11]. Given the strong similarities between <span class="yellow">murine</span> and <span class="blue">human</span> full-length BLyS and ΔBLyS, it is likely that <span class="blue">human</span> soluble ΔBLyS protein is also not cleaved from the cell surface and released into the circulation. Moreover, soluble ΔBLyS protein is not released from cells transfected with a vector containing just the extracellular domain of <span class="blue">human</span> ΔBLyS (which encodes the soluble protein; A.L. Gavin, unpublished observations). Whether this reflects rapid intracellular degradation of soluble ΔBLyS or some other impediment to its release remains unknown. Regardless, if the inability to release soluble ΔBLyS in vitro faithfully recapitulates in vivo biology, then the stronger associations between SLE disease activity and full-length BLyS or ΔBLyS mRNA levels compared with that between SLE disease activity and BLyS protein levels could not be attributable to interference by biologically inactive (inhibitory) ΔBLyS protein in the BLyS protein detection ELISA. Importantly, even if soluble ΔBLyS protein is present in the circulation and is detected by the BLyS protein detection ELISA, then the stronger correlations between SLE disease activity and full-length BLyS or ΔBLyS mRNA levels than that between disease activity and total BLyS (including ΔBLyS) protein levels suggest that full-length BLyS and/or ΔBLyS mRNA levels may operationally serve as useful biomarkers of disease activity in SLE. Although the complexity and labor intensiveness associated with quantitative real-time PCR may render measurement of BLyS mRNA levels impracticable for routine clinical practice, such measurement could readily be incorporated into clinical trials and yield valuable information.
Longitudinal observations in large numbers of SLE <span class="yellow">patients</span> will be necessary to establish or refute the utility of full-length BLyS and/or ΔBLyS mRNA to subserve this clinically vital function.
Although expression of the two major BLyS isoforms was highly coordinate among SLE <span class="yellow">patients</span>, there were several <span class="yellow">patients</span> in whom ΔBLyS mRNA levels were markedly greater than or less than the expected values based on full-length BLyS mRNA levels (data not shown). This raises the possibility that dysregulation of ΔBLyS may contribute to overall BLyS dysregulation in at least some SLE <span class="yellow">patients</span>. It is known that interferon-γ, interleukin-10, interferon-α, and CD154 can upregulate full-length BLyS mRNA levels [14,22,35], but it is not known what effects these or other cytokines/cell-surface structures have on ΔBLyS expression. Further investigation of the regulation of ΔBLyS and the differential expression of BLyS isoforms is certainly warranted.
Although the associations between full-length BLyS and/or ΔBLyS mRNA levels and disease activity in SLE were usually strong when the SLE cohort was analyzed in aggregate, there were several SLE <span class="yellow">patients</span> in whom BLyS mRNA levels were quite high despite little objective ongoing disease activity, and there were several SLE <span class="yellow">patients</span> in whom BLyS mRNA levels were low despite considerable ongoing disease activity. One must recognize that the bulk of the pathogenic autoimmune responses probably takes place in the spleen and lymph nodes, rather than in the peripheral blood, where myeloid lineage cells (for example, dendritic cells) produce BLyS and support B cell survival and expansion [36]. Local BLyS production in the secondary lymphoid tissues will be more important to the development and maintenance of an autoimmune response than will remote BLyS levels in the circulation. Because at least some autoreactive B cells may be more sensitive to BLyS-mediated survival signals than non-autoreactive B cells [37,38], local increases in BLyS production could preferentially promote expansion of autoreactive B cells. These cells, in turn, could activate autoreactive T cells by presenting autoantigen to them, and some of the autoreactive B cells would respond to T cell derived signals and mature into (pathogenic) autoantibody secreting plasma cells. In contrast to <span class="yellow">murine</span> studies, in which investigators can readily harvest and analyze lymphoid and myeloid lineage cells from any site (for example, spleen, bone marrow), such is not the case for <span class="blue">human</span> studies. Peripheral blood is the only site readily accessible for <span class="blue">human</span> studies, and it is possible that, at least in some <span class="yellow">patients</span>, BLyS mRNA levels in circulating leukocytes do not reflect local BLyS production in the secondary lymphoid tissues.
One must also recognize that disease activity in SLE is not solely driven by B cells. Systemic inflammation and SLE flares can be triggered via B cell independent means. Not all SLE <span class="yellow">patients</span> treated with a B cell depleting course of rituximab experience clinical remission [39], strongly pointing to the importance of non-B cells in disease pathogenesis/maintenance. Conversely, not all pathogenic B cells necessarily require high levels of BLyS to effect their pathogenicity. <span class="yellow">Murine</span> studies have unequivocally documented B cell subpopulations that do not depend upon BLyS for their survival [40-42]. Although <span class="yellow">mice</span> completely devoid of BLyS have reduced numbers of mature B cells and harbor reduced levels of immunoglobulin, these reductions are incomplete. Thus, it is possible that some SLE <span class="yellow">patients</span> harbor pathogenic B cells that are relatively insensitive to BLyS and could drive considerable disease activity even in the context of low endogenous BLyS production. Conversely, <span class="yellow">patients</span> with high BLyS mRNA levels may be those <span class="yellow">patients</span> whose disease is strongly driven by BLyS and may be especially helped by BLyS antagonist therapy. Future clinical trials should be able to establish whether the BLyS mRNA levels are good predictors of response to such agents.<br><br>Conclusion
Plasma total immunoglobulin, IgG, and IgA levels and disease activity (as measured by SLEDAI) in SLE <span class="yellow">patients</span> correlate more closely with peripheral blood leukocyte levels of BLyS mRNA than with plasma levels of BLyS protein. These findings suggest that BLyS mRNA levels better reflect in vivo BLyS production than do circulating BLyS protein levels, and may be a useful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>. These findings also support the premise that BLyS overexpression not only promotes development of disease but also actively contributes to the ongoing maintenance of disease in SLE <span class="yellow">patients</span>. This reinforces the rationale underlying clinical trials with BLyS antagonists in SLE.<br><br>Abbreviations
anti-dsDNA = anti-double-stranded DNA; BLyS = B lymphocyte stimulator; bp = base pairs; Ct = threshold cycle; ELISA = enzyme-linked immunosorbent assay; PCR = polymerase chain reaction; RA = rheumatoid arthritis; SLE = systemic lupus erythematosus; SLEDAI = SLE Disease Activity Index.<br><br>Competing interests
TSM and DMH were employees of <span class="yellow">Human</span> Genome Sciences (HGS) at the time the investigation was conducted. (DMH has since left the company.) WS has received research support from HGS and has served as a consultant to HGS (<$10,000). CEC, ALG, and DN declare that they have no competing interests.<br><br>Authors' contributions
CEC identified and recruited all <span class="yellow">participants</span>; collected all the blood samples and reviewed all the medical charts; and wrote the initial working draft of this manuscript. ALG developed and performed all the real-time PCR assays and assisted in the interpretation of the results and in writing the final version of the manuscript. TSM performed the plasma BLyS protein and anti-BLyS assays and assisted in the interpretation of the results and in writing the final version of the manuscript. DMH assisted in the design in the study, in the interpretation of the results, and in writing the final version of the manuscript. DN assisted in the design in the study, in the interpretation of the results, and in writing the final version of the manuscript. WS conceived the study, supervised the recruitment of <span class="yellow">participants</span>, performed the statistical analyses, assisted in the interpretation of the results, and supervised the editing of the manuscript to its final form. All authors read and approved the final manuscript version.<br><br>
<h3>pmcA1859974</h3><span class="yellow">Human</span> growth hormone (GH1) gene polymorphism map in a normal-statured adult population
Abstract
Objective
GH1 gene presents a complex map of single nucleotide polymorphisms (SNPs) in the entire promoter, coding and noncoding regions. The aim of the study was to establish the complete map of GH1 gene SNPs in our control normal population and to analyse its association with adult height.<br><br>Design, subjects and measurements
A systematic GH1 gene analysis was designed in a control population of 307 adults of both sexes with height normally distributed within normal range for the same population: −2 standard deviation scores (SDS) to +2 SDS. An analysis was performed on individual and combined genotype associations with adult height.<br><br>Results
Twenty-five SNPs presented a frequency over 1%: 11 in the promoter (P1 to P11), three in the 5′UTR region (P12 to P14), one in exon 1 (P15), three in intron 1 (P16 to P18), two in intron 2 (P19 and P20), two in exon 4 (P21 and P22) and three in intron 4 (P23 to P25). Twenty-nine additional changes with frequencies under 1% were found in 29 subjects. P8, P19, P20 and P25 had not been previously described. P6, P12, P17 and P25 accounted for 6·2% of the variation in adult height (P = 0·0007) in this population with genotypes A/G at P6, G/G at P6 and A/G at P12 decreasing height SDS (−0·063 ± 0·031, −0·693 ± 0·350 and −0·489 ± 0·265, Mean ± SE) and genotypes A/T at P17 and T/G at P25 increasing height SDS (+1·094 ± 0·456 and +1·184 ± 0·432).<br><br>Conclusions
This study established the GH1 gene sequence variation map in a normal adult height control population confirming the high density of SNPs in a relatively small gene. Our study shows that the more frequent SNPs did not significantly contribute to height determination, while only one promoter and two intronic SNPs contributed significantly to it. Studies in larger populations will have to confirm the associations and in vitro functional studies will elucidate the mechanisms involved. Systematic GH1 gene analysis in <span class="yellow">patients</span> with growth delay and suspected GH deficiency/insufficiency will clarify whether different SNP frequencies and/or the presence of different sequence changes may be associated with phenotypes in them.<br><br><br><br>Introduction
<span class="yellow">Human</span> skeletal growth and final height attainment are a result of a multifactorial regulation involving systemic and local hormones, growth and nutritional factors, lifestyle and genetic factors. Heritability estimates1 and genome-wide linkage analysis2 have shown that genetic factors play a major role in determining stature. Among these factors, the GH-IGF-I axis plays an important role during postnatal life, and associations between structural variations in its genes and height are currently under study.3 Although growth hormone (GH) deficiency is a well-known cause of growth retardation, which responds to GH replacement therapy, the diagnosis and physiopathological mechanisms for the so-called ‘idiopathic isolated GH deficiency’ (IIGHD) require further clarification. In addition, GH secretion levels and markers of GH biological activity have been demonstrated to be specific and sensitive only in major deficiency states.4,5 Genetic causes of GH deficiency within the GH1 gene have been established; however, they are rarely recognized and only sought in major GH deficiency states during childhood and in family studies.3 GH1 gene, located at 17q22–24, is a component of the GH gene cluster in which five genes evolving from a common ancestor are 91–99% sequence conserved (paralogues).6 GH1 is more abundantly expressed in pituitary cells, while the other four genes are expressed in placental tissue. Large deletions within the GH1 gene cluster were described first followed by point mutations, the majority of which affect introns 3 or 4, provoke skipping of exon 3 product and exert a dominant effect.3,7,8 More recently, the presence of single nucleotide polymorphic points (SNPs) in the promoter region or in intron 4 of the GH1 gene have been described9–12 and associations with promoter allele activities or with GH secretion efficacy and circulating IGF-I levels in growth-retarded <span class="yellow">patients</span> have also been described.11,12 Other studies have analysed several GH1 gene SNP genotypes as related to the incidence of neoplasia, with a positive association with colorectal neoplasia for intron 4 SNP,13 a negative result for breast carcinoma14,15 or a positive one for breast cancer risk.16,17 In addition, a recent study in a cohort of adults over ages 60 years detected a significant association between genotypes at one SNP in the GH1 gene promoter region and at the intron 4 SNP described by Hasegawa et al.11 with baseline bone density and accelerated bone loss together with an interaction with weight at 1 year.18 Intron 4 SNP described by Hasegawa et al.11 has also been associated, in <span class="yellow">women</span>, with shorter body height and reduced mortality,19 whereas another intron 4 SNP (T1169A) has been associated in both sexes with a favourable metabolic profile.20 A systematic SNP study was conducted by Adkins et al.21 in GH1 promoter, coding and noncoding regions in DNAs from placental tissues, and analysis of associations between genotypes and birth weight revealed an association between an alternate nucleotide at −1 and +3 of translation initiation site and fetal growth restriction. However, no systematic GH1 gene analysis in the entire promoter, coding and noncoding regions has been conducted in adults to establish the map of structural variation and its possible association with height. The relatively short size of the entire gene permits a complete analysis which is, nevertheless, hampered by the need to avoid amplification of any other of the GH cluster genes (paralogues) and the high density of sequence variations.
To obtain normative data for subsequent analysis of GH1 gene contribution to IIGHD in <span class="yellow">children</span>, a systematic GH1 gene structural analysis was designed in a normal adult control population to establish the GH1 gene SNP map in adults from our population with heights within the normal range, determine the genotype frequencies and analyse possible associations between individual and combined SNPs with height.<br><br>Subjects and methods
Subjects
A total of 307 adult subjects of both sexes (164 <span class="yellow">women</span> and 143 <span class="yellow">men</span>) were recruited from hospital personnel and parents of <span class="yellow">patients</span> with no history of growth retardation. Subjects had to fulfil the following criteria: Iberian Peninsular (except Basque) family origin and no family history of pathological short stature. A single subject per family was included. The protocol was approved by the Hospital Vall d’Hebron Ethics Committee and written informed consent was obtained from each <span class="yellow">participant</span>. Height standard deviation scores (height SDS) were calculated according to sex-specific reference growth charts for the Spanish population (Carrascosa et al.22 charts were used for subjects under ages 30 years and Hernández et al.23 for subjects aged 30–50 years). Only individuals with height SDS between −2 and +2 SDS were included in the study (mean −0·016; 32 <span class="yellow">women</span> and 28 <span class="yellow">men</span> between −2·000 and −1·010; 99 <span class="yellow">women</span> and 80 <span class="yellow">men</span> between −1·000 and +0·910; 33 <span class="yellow">women</span> and 35 <span class="yellow">men</span> between +1·010 and +1·980) and sample size was adjusted for normal sex and height SDS distribution. Height and weight were recorded in the morning by a single observer. Height was measured with a Harpenden stadiometer. Four millilitres of peripheral venous blood were drawn into EDTA-containing tubes for molecular genetic analysis.<br><br>Genomic DNA study
Genomic DNA was obtained from peripheral blood following the method described by Lahiri and Nurnberger.24 DNA was amplified by polymerase chain reaction (PCR) using a nested strategy. Briefly, 50 ng of genomic DNA were added to a 10 µl reaction mixture of 1 mm Mg(OAc)2, 0·6 mm dNTPs, 0·3 µm of each primer, and 0·4 U r Tth DNA polymerase XL (Applied Biosystems, Foster City, CA, USA). The sense and antisense primers used corresponded to nucleotides 4156–5′ACGGTCCGCCACTACGCCCAGC-3′ and the complement of 6948–5′TGCAGTGAGCCAAGATTGTGCC-3′ of the GH gene cluster.6 The PCR reaction mix was denatured for 5 min at 94 °C and cycled 40 times (94 °C, 1 min; 72 °C, 3 min 30 s) followed by a 7-min extension at 72 °C. The resulting GH1 PCR products (2893 bp) were used as templates for five nested reactions (AN, BL, CK, DI, FP), carried out as follows: 1 µl of each GH1 PCR product was added to a 20 µl reaction mixture of 1·5 mm MgCl2, 0·2 mm dNTPs, 0·3 µm of each primer and 0·4 U Eco Taq DNA polymerase (Ecogen S.R.L., Barcelona, Spain). Reaction mixtures were denatured for 5 min at 94 °C, cycled 40 times (94 °C, 1 min; 58 °C, 1 min; and 72 °C, 1 min), followed by a 7-min extension at 72 °C. Sense and antisense primers were as follows:
Sequencing from both ends was performed by the dideoxy method using ABI PRISM BigDye Terminator version 3·1 Cycle Sequencing Kit (Applied Biosystems, Foster City, CA, USA). GH1 gene nucleotide sequence published by Chen et al.6 was used as control. For each DNA, the five segments from the nested PCR were assembled with the SeqEscape programme (Applied Biosystems) and interpretation was made visually and simultaneously by two observers. Antisense sequencing was performed to confirm each nucleotide sequence change up to the establishment of the more frequent SNP map (frequency over 1%), whereas less frequent single or multiple nucleotide changes were reconfirmed in each DNA by antisense sequence and resequencing after a new nested PCR from original DNA was performed.<br><br>Single nucleotide polymorphism (SNP) genotyping
The sequences for the five genes of the GH cluster identified by Chen et al.6 and reported as the GI sequence 183148 were aligned using the Multalin program.25 SNPs and other sequence changes identified were indicated using their position corresponding to GH1. Genotypes were deduced by the combination of genetic variation at the polymorphic positions.<br><br>Statistical analysis
Standardized height was investigated for normal distribution (Kolmogorov-Smirnov test: c2 = 2·882, P = 0·4733). Hardy–Weinberg equilibrium was tested for SNPs presenting three alternate genotypes according to standard procedures using χ2-analysis. anova test was applied to investigate individual and combined SNP association with adult height SDS; significance assessment was adjusted for multiple testing using Fisher's PLSD test setting Pcritical = 0·05 or the Bonferroni–Dunn test setting Pcritical = 0·05/n (n = number of comparisons carried out). Stepwise regression analysis was applied to predict the contribution of SNPs to adult height SDS. Statview 4·5 program (Abacus Concepts Inc., Berkeley, CA, USA) was used for statistical analyses.<br><br>
Results
GH1 gene sequence variation
GH1 gene sequence comparison with the GI-183148 sequence published by Chen et al.6 yielded a total of 54 single or multiple nucleotide changes. Twenty-five SNPs presented a frequency over 1% (genotypes and frequencies are listed in Table 1). SNPs which presented the three alternate genotypes (P2 to P4, P6, P7, P10 and P24) were in Hardy–Weinberg equilibrium (data not shown). Twenty-nine additional changes were found with a frequency under 1% or involving more than one nucleotide and thus could be considered as rare variant SNPs (R1 to R29) (Table 2). These changes were found in 29 of 307 subjects (9·4%), all in heterozygosity.<br><br>GH1-paralogue alignment
A sequence alignment was performed to study possible sequence recombinations among paralogues of the five GH1-gene cluster (Fig. 1). This alignment showed that 9 of 25 SNPs (36%) in the GH1 gene did not correspond to any of the paralogues.
Among the 29 rare SNPs found, six (20·7%) did not correspond to any of the paralogues: two were located in the 5′UTR region (R9 and R10), two in intron 1 (positions 5300 = R14 and 5302 = R17), one in intron 2 (position 5679 = R21) and one in intron 4 (position 6344 = R23) (Table 2).<br><br>Equivalence with previously reported GH1 changes
Equivalence with changes and SNPs previously reported by other authors are shown in Table 3. The majority found a high density of SNPs in the promoter and 5′UTR regions in control populations.10,12,21 Several sequence changes have been reported in <span class="yellow">patients</span> with familial or idiopathic short stature,11,26,27 whereas P8, P19, P20 and P25 (at positions 5165, 5681, 5686 and 6358, respectively, in the Genebank accession GI 183148) located in the promoter, intron 2 and intron 4 regions, respectively, had not been previously described.<br><br>GH1 genotypes and associations with height SDS
Associations between genotypes and standardized height were first studied in the subpopulation of 278 controls carrying only the 25 most frequent SNPs in the GH1 gene (c2 = 2·59; P = 0·5458 for normality of height distribution).
Three individual SNPs showed a statistically significant association with height SDS: at positions 5286 (P16), 5290 (P17) and 6358 (P25). Subjects with heterozygous genotypes presented statistically significant taller stature than the corresponding homozygous genotypes (P = 0·016 for P16, P = 0·015 for P17 and P = 0·023 for P25) (Fig. 2a,b,c). P16 and P17 were in linkage disequilibrium (LD) (r2 = 0·831), while P25 was carried by six subjects homozygous at P16 and P17.
GH1 gene genotypes were defined by genetic variation in the 25 polymorphic positions. We found 163 different combinations. Only two genotypes presented a frequency over 5% (Table 4). Height SDS in the two more frequent genotypes did not differ significantly and covered the whole height range. Genotype 1 presented four heterozygous variations and Genotype 2 was the corresponding homozygous genotype. Heterozygous positions corresponded to SNPs with the highest frequency variation (4886 (P4), 5107 (P7), 5157 (P10) and 6331 (P24)). In addition, DNAs exhibited a different genotype in each of 129 subjects (46·4%).
The 11 SNPs found in the promoter region (Table 1) were grouped in 94 genotypes and analysed for association with adult height SDS. The four more frequent combinations are listed in Table 4: height SDS of these four genotypes did not differ statistically although Genotype 3 tended to have a shorter height. Genotype 3 differed from Genotype 1 in the SNP located at position 5089 (P6), corresponding to Pit 1 proximal responsive element for GH1 gene promoter. Genotype 4 is heterozygous at positions 4856 (P2), 4863 (P3) and 5107 (P7). In addition, 19% of cases exhibited a genotype in the promoter region carried by only one subject.
Combination of the three SNPs in the 5′UTR region of GH1 gene resulted in five different genotypes. SNPs at positions 5178 (P12) and 5187 (P13) were in LD (r2 = 0·88). Mean height SDS comparison among these genotypes was not statistically significant, although mean height SDS of alternate nucleotide carriers at position 5178 (P12) tended to be shorter (Table 4).
An anova analysis was conducted to investigate the interaction between two or more SNPs and height SDS. SNPs at positions 5286 (P16) and 5290 (P17) were in LD (r2 = 0·83): the heterozygous genotype AG/AT for these SNPs was associated with taller stature (shown above). SNP at position 6358 (P25) increased the expected height SDS for individual carriers of the G allele at 5089 (P6) SNP as shown in Fig. 2(d): subjects heterozygous at 6358 (P25) were taller than the mean, and mean height SDS of subjects with GG/TG combined genotype was significantly higher than the corresponding GG/TT genotype (P = 0·0021), suggesting an interaction between these two SNPs as they were not in LD.
Analysis of height SDS association with the most frequent single and combined SNPs and with rare variant SNPs was performed in the 29 individuals carrying the rare SNPs (Table 2). None of them carried any of the three SNPs (P16, P17 and P25) related to taller stature in the population of 278 controls with only the frequent SNPs. In these 29, mean height SDS (0·000 ± 0·987, from −1·930 to +1·870) did not differ from that of the 278 controls (−0·018 ± 1·041, from −2·000 to +1·980). Analysis of associations between individual SNP genotypes and height SDS revealed that SNPs at positions 5089 (P6), 5178 (P12) and 5187 (P13) were associated with significantly shorter stature (Fig. 2e). Only two sequence changes considered as rare SNPs were carried by individuals in the lower normal height range (between −1·500 and −2·000 SDS) (Table 2): R4 (4979 C > T) in the promoter region and R14 (5300 C > T) in intron 1. Predicted single amino acid changes located in exon 5 (R25 to R27) were not associated with short stature.
In the entire population of 307 controls, stepwise regression analysis between height SDS and genotypes at the 25 SNPs showed that genotypes at 5089 (P6), 5178 (P12), 5290 (P17) and 6358 (P25) were significantly correlated with height SDS (r2 = 0·062, P = 0·0007) with genotypes A/G at P6, G/G at P6 and A/G at P12 decreasing height SDS (−0·063 ± 0·031, −0·693 ± 0·350 and −0·489 ± 0·265, respectively, Mean ± SE) and genotypes A/T at P17 and T/G at P25 increasing height SDS (+ 1·094 ± 0·456 and +1·184 ± 0·432, respectively).<br><br>
Discussion
Genetic variations within <span class="blue">human</span> GH1 gene have been described by several authors.9–12,21 The populations described to date comprised small numbers of normal-stature individuals,9 male adults with narrow height range12 or growth-retarded <span class="yellow">patients</span> with/without GHD before achievement of adult height.9,11,12 Our study was designed to characterize the GH1 gene sequence variation in individuals within the whole range of normal adult height (between −2 and +2 SDS) according to the standards for our population. Height was normally distributed, both sexes were equally represented and the GI-183148 homozygous sequence6 was used for comparison. A nested PCR with specific primers for GH1 gene was designed, thus avoiding amplification of any other GH gene paralogue of the GH gene cluster.
Our results establish a map of 25 SNPs as present in over 1% of individuals, whereas 29 other sequence changes (single or multiple nucleotide) are present in less than 1% of subjects. More than 50% (n = 14) of SNPs are located in the promoter and 5′UTR regions, thus confirming previous reports: Giordano et al.9 reported eight SNPs in the promoter and 5′UTR regions, Wagner et al.10 16 SNPs from the promoter to intron 1 and Horan et al.12 identified 36 haplotypes in control subjects of the British population, which would result from the combination of 15 of the previously reported SNPs. Our results confirm the presence of 13 of those points; SNP at 5165 (R11 in the present study and +3 in references9,12) was present in less than 1% of subjects and a new SNP is described (P8 at 5116 in the VDR/RA/T3 responsive element sequence). The remaining SNPs (from P15 to P25, n = 11) are distributed in introns 1, 2 and 4, and among coding regions only exon 1 and exon 4 bear a total of three SNPs, two of which predict an amino acid change (P15 and P21). These two latter SNPs had been described by Millar et al.26 and the more frequent SNP in intron 4 (P24) has been described by Hasegawa et al.11 together with the two more frequent SNPs in the promoter region (P4 and P7 in our map). Three new SNPs are described (P19 and P20 in intron 2 and P25 in intron 4), all outside the splice sites. Only SNPs presenting high frequency are present in homozygous alternate state and this accounts mostly for the majority of the promoter and 5′UTR SNPs and in the intron 4 more frequent SNP (P24) described by Hasegawa et al.11 In conclusion, in the entire coding and noncoding GH1 gene sequence, only P24 is present in homozygous alternate state. Our results show that the GI 183148 homozygous sequence is present in our population except for SNP P14 in the 5′UTR region which is only present as the alternate nucleotide in homozygous or heterozygous states.
As described by several authors9,10,12,21 several promoter SNPs affected functional sequences and P6 is located in the Pit 1 proximal responsive element, P7 and P8 in the VDR/RA/T3 responsive element and P9 (G del) in the TATA box.
The mechanisms by which the high density of SNPs in the GH1 gene is generated has been proposed to be recombination and gene conversion with any other(s) of the GH cluster genes.9,12,28 Alignment of the 25 SNPs with the other GH1 gene paralogues demonstrated in our results that this mechanism is possible for 64% of SNPs. Familial SNP transmission pattern analysis will be of interest to support the hypothesis of GH gene recombination.
In addition, 29 of 307 individuals (9·4%) bore additional GH1 sequence changes with frequencies under 1%. As for SNPs, they are located along the whole gene with higher density in the promoter and 5′UTR regions. Interestingly, intron 3 and exon 5 present several of these less frequent changes. Intron 3 has been shown to carry the majority of single nucleotide mutations causing the dominant form of GH deficiency.3 The two single nucleotide changes detected in intron 3 (R15 at 6056 and R16 at 6061) are in perfect LD (r2 = 1·0) and located within the enhancer splice site element (ESE) described by Ryther et al.29,30 Studies in additional normal or growth-retarded populations will permit description of their possible clinical implications. Five single nucleotide changes are located in exon 5; of these five, three predict an amino acid change, and one of the three (Ile179Met) has been described by Lewis et al.27 in a paediatric <span class="yellow">patient</span> with familial short stature and the other two, as yet undescribed, are contiguous in a single individual (Pro133Hys and Arg134Leu). Polynucleotide changes are mostly located in the promoter region corresponding to the VDR/RA/T3 response element. As for frequent SNPs, the majority of the sequence changes with frequencies under 1% may have been generated by recombination within the GH gene cluster as 19 of 24 (79%) may correspond to one or more of the GH1 gene paralogues.
Our results now show the diversity and complexity of SNP genotypes, as previously highlighted by other authors9,10,12,21 in a normal adult height control population. Our initial aim when designing the present study was to establish the map of GH1 gene SNPs in our adult control population with heights normally distributed within the entire normal range for further comparison with genotypes in our paediatric population with growth delay, variable response to GH secretion tests and adequate response to GH therapy. Analysis of SNP association with adult height was subsequently performed to establish a body of knowledge useful for comparing <span class="yellow">patient</span> genotypes and phenotypes. This analysis was first performed in controls bearing only frequent SNPs (90·5% of the total population). We demonstrate that the four SNPs with the highest allelic variation frequencies (P4, P7, P10 and P24) do not significantly contribute to adult height determination, with the heterozygous genotype being the most frequent followed by the corresponding homozygous genotype in the whole sequence, and heights are normally distributed over the entire height range. The third most frequent combined genotype in the promoter region in our population presented, in addition, in heterozygosity, the SNP at P6 in the sequence regulated by Pit 1 and although mean height of individuals (6·1%) bearing this genotype was around −0·5 SDS, this was not statistically significant.
Analysis of single SNP genotype association with adult height yielded few clues as to the contribution of GH1 gene variation to adult height determination. Only three SNPs (P16, P17 and P25), present with low frequency and only in heterozygous state, were individually significantly associated with taller stature and none was individually associated with shorter stature. P16 and P17 (in LD, r2 = 0·83) are located in intron 1 and P25 in intron 4. The resulting sequence for the presence of P16 and P17 corresponded to the paralogue GH2 and generated a responsive element for a core-binding protein (Matinspector Programme, Geometrix Software GmbH, München, Germany) with three Kruppel-type zinc fingers which could increase the efficacy of GH1 gene transcription;31,32 moreover, Kruppel-like proteins have recently been described in the brain.33 Stepwise regression analysis demonstrated that P17 and P25 contribute, separately, to an increase of almost 1·0 height SDS. P16 and P17 had been described by Adkins et al.21 although they found no association with fetal growth, whereas P25 had not previously been described. The mechanisms by which they may determine taller final height should be established by in vitro studies analysing GH1 gene transcription and GH protein translation efficiencies.
Analysis of interaction effect between SNPs detected that variation at P25 masked an effect of P6. Individuals homozygous at P25 (TT) present a significant association between P6 genotype and height with the homozygous alternate genotype at P6 (GG) being associated with shorter stature. This was further confirmed in the subpopulation of 29 individuals bearing rare SNPs who, in the absence of heterozygous change at P25, presented significantly shorter stature in the heterozygous alternate nucleotide change at P6 (AG). P6, located at Pit 1 proximal responsive element of the GH1 gene promoter, was first described by Wagner et al.10 and Giordano et al.9 and further by Horan et al.12 Six of nine GH1 gene promoter haplotypes bearing the alternate G at P6 presented lower transcriptional activities and electrophoretic mobility shift assays (EMSA) detected differential protein binding strength, although in vitro studies were unable to identify this SNP as a major determinant of GH1 gene expression level.12 A recent study from Giordano et al.34 has shown a twofold reduced luciferase activity for the G nucleotide bearing promoter haplotype in transfected <span class="yellow">rat</span> pituitary cells. Genotypes at P6 had also been associated with decreased breast cancer risk through its association with lower GH secretion and IGF-I circulating levels.16,17
In our results, GH1 gene polymorphic structural variation accounted for only 6·2% of adult height determination in the entire adult population studied and genome-wide linkage analysis of stature in multiple populations revealed no linkage with chromosome 17 GH gene cluster.2,35 As only some of the less frequent SNPs are statistically associated with height, and in view of the high density of SNPs, our study may be hampered by selection bias36 and would ideally have required a wider sampling of some 2 000 individuals; however, this was a highly laborious strategy when the complete sequencing technique is applied. The high density of SNPs and their proximity hamper other genotyping strategies for rapid determination of the complete GH1 SNP map in large control and <span class="yellow">patient</span> populations. Individual SNP associations with height or other GH secretion-related phenotypic traits will require further confirmation by studies in larger populations and by in vitro functional studies.
In conclusion, our study established the GH1 gene sequence variation map in an adult control population with heights normally distributed within the normal range. SNPs and other sequence change contributions to skeletal growth as observed at adult height demonstrated that, despite the high frequency of variation and diversity and complexity of combinations, only some of the less frequent SNPs were associated with taller stature (P17 in intron 1 and P25 in intron 4), even masking the SNP contribution to a shorter one (P6 in the promoter and P12 in the 5′UTR regions, respectively). Systematic GH1 gene analysis in <span class="yellow">patients</span> with growth delay and suspected GH deficiency/insufficiency will clarify whether different SNP frequencies and/or the presence of different sequence changes may be associated with phenotypes in them.<br><br>
<h3>pmcA1913570</h3>Elevated Blood Lead Levels of <span class="yellow">Children</span> in Guiyu, an Electronic Waste Recycling Town in China
Abstract
Background
Electronic waste (e-waste) recycling has remained primitive in Guiyu, China, and thus may contribute to the elevation of blood lead levels (BLLs) in <span class="yellow">children</span> living in the local environment.<br><br>Objectives
We compared the BLLs in <span class="yellow">children</span> living in the e-waste recycling town of Guiyu with those living in the neighboring town of Chendian.<br><br>Methods
We observed the processing of e-waste recycling in Guiyu and studied BLLs in a cluster sample of 226 <span class="yellow">children</span> < 6 years of age who lived in Guiyu and Chendian. BLLs were determined with atomic absorption spectrophotometry. Hemoglobin (Hgb) and physical indexes (height and weight, head and chest circumferences) were also measured.<br><br>Results
BLLs in 165 <span class="yellow">children</span> of Guiyu ranged from 4.40 to 32.67 μg/dL with a mean of 15.3 μg/dL, whereas BLLs in 61 <span class="yellow">children</span> of Chendian were from 4.09 to 23.10 μg/dL with a mean of 9.94 μg/dL. Statistical analyses showed that <span class="yellow">children</span> living in Guiyu had significantly higher BLLs compared with those living in Chendian (p < 0.01). Of <span class="yellow">children</span> in Guiyu, 81.8% (135 of 165) had BLLs > 10 μg/dL, compared with 37.7% of <span class="yellow">children</span> (23 of 61) in Chendian (p < 0.01). In addition, we observed a significant increasing trend in BLLs with increasing age in Guiyu (p < 0.01). It appeared that there was correlation between the BLLs in <span class="yellow">children</span> and numbers of e-waste workshops. However, no significant difference in Hgb level or physical indexes was found between the two towns.<br><br>Conclusions
The primitive e-waste recycling activities may contribute to the elevated BLLs in <span class="yellow">children</span> living in Guiyu.<br><br><br><br>Disposal of electronic waste, or e-waste, is an emerging global environmental issue, as these wastes have become the most rapidly growing segment of the municipal waste stream in the world [Dahl 2002; Halluite et al. 2005; Jang and Townsend 2003; Schmidt 2002; Silicon Valley Toxics Coalition (SVTC) 2001]. It is reported that approximately 500 million computers became obsolete between 1997 and 2007 in the United States (National Safety Council 1999). Up to 80% of e-waste from the United States has seeped into Asia and Africa (Johnson 2006; Puckett et al. 2002; Schmidt 2002, 2006; SVTC 2001). It is noteworthy that the United States is the only developed country today that has not ratified the United Nations Basel Convention, which bans the export of hazardous wastes to developing countries (United Nations Environment Programme 1992, 2006; USA Today 2002).
Together with New Delhi in India, Guiyu in Shantou, Guangdong Province, China (Figure 1), is one of the popular destinations of e-waste (Brigden et al. 2005; Puckett et al. 2002). Within a total area of 52 km2 and local population of 132,000 (in 2003), Guiyu has accommodated millions of tons of e-waste from overseas and domestic a year. Nearly 60–80% of families in the town have engaged in e-waste recycling operations conducted by small scale family-run workshops, with approximately 100,000 migrant workers employed in processing e-waste. Because the implementation of a clean and safe high-tech recovery process was very expensive (Allsopp et al. 2006), the processes and techniques used during the recycling activities in Guiyu were very primitive. The result was that many tons of e-waste material and process residues were dumped in workshops, yards, roadsides, open fields, irrigation canals, riverbanks, ponds, and rivers. Hazardous chemicals can be released from e-wastes through disposal or recycling processes, threatening the health of local residents. Several studies have reported the soaring levels of toxic heavy metals and organic contaminants in samples of dust, soil, river sediment, surface water, and groundwater of Guiyu (Brigden et al. 2005; Puckett et al. 2002; Wang and Guo 2006; Wang et al. 2005; Wong et al. 2006; Yu et al. 2006). Previously, we have shown that the residents in Guiyu had high incidence of skin damage, headaches, vertigo, nausea, chronic gastritis, and gastric and duodenal ulcers, all of which may be caused by the primitive recycling processing of e-waste (Qiu et al. 2004).
Of many toxic heavy metals, lead is the most widely used in electronic devices for various purposes, resulting in a variety of health hazards due to environmental contamination (Jang and Townsend 2003; Musson et al. 2006; Vann et al. 2006). Lead enters biological systems via food, water, air, and soil. <span class="yellow">Children</span> are particularly vulnerable to lead poisoning—more so than adults because they absorb more lead from their environments (Baghurst et al. 1992; Grigg 2004; Guilarte et al. 2003; Jain and Hu 2006; Needleman 2004; Safi et al. 2006; Wasserman et al. 1998). The U.S. Centers for Disease Control and Prevention (CDC) defined elevated blood lead levels (BLLs) as those ≥ 10 μg/dL in <span class="yellow">children</span> ≤ 6 years of age (CDC 1991). Nevertheless, studies have increasingly shown that low blood lead concentrations, even < 10 μg/dL, were inversely associated with <span class="yellow">children</span>’s IQ scores and academic skills (Canfield et al. 2003; Lanphear et al. 2000, 2005; Nevin 2000; Schnaas et al. 2006). Therefore, no safety margin at existing exposures has been identified (Chiodo et al. 2004; Koller et al. 2004).
Considering the potential heavy metal contamination in the local living environment of Guiyu, we hypothesized that <span class="yellow">children</span> living in Guiyu may have elevated BLLs and thus their physical and mental development may have been affected. In this study, we evaluated the mean BLLs in <span class="yellow">children</span> 1–6 years of age living in Guiyu and compared them with those living in the neighboring town of Chendian, where no e-waste processing was taken.
Materials and Methods
Geographic location and site description
There are 28 villages with a total area of 52 km2 and a resident population of 132,000 and around 100,000 migrant workers in Guiyu (Figure 1). We chose four villages for their differences in the scale and type of e-waste processing. Beilin village has dense e-waste workshops mainly involved in equipment dismantling, circuit board baking, and acid baths; Dutou village specializes in plastics sorting, including manually stripping plastic materials from electronic products and then crudely classifying them; Huamei village had workshops similar to those of Beilin, but they are fewer and scattered; and Longgang village was involved in plastic reprocessing in which plastics collected from Dutou and other villages were washed and smashed into tiny pieces of recycled plastic. We used the neighboring town of Chendian as a control because the local residents work mainly in the textiles industry, not in e-waste processing. The population, traffic density, lifestyle, and socioeconomic status were very similar to those of Guiyu.<br><br>Study population
The study population was composed of <span class="yellow">children</span> ≤ 6 years of age. No <span class="yellow">children</span> involved in the study had any occupational exposure to e-waste. A cluster sample of 165 <span class="yellow">children</span> with a median age of 5.0 years lived in the four villages of Guiyu (Figure 1). Sixty-one <span class="yellow">children</span> with a median age of 4.0 years resided in Chendian were included in the study for comparison. After written informed consent was obtained from the parents or guardians, blood samples were collected from the <span class="yellow">children</span> at village kindergartens. To facilitate the counseling process, advice on dietary and eating habits to minimize lead exposure were provided to the local residents. All <span class="yellow">children</span> found to have high BLLs were advised to get further hospital treatment. The study was approved by the <span class="yellow">Human</span> Ethics Committee of Shantou University Medical College.<br><br>Measurement of BLLs and hemoglobin
Venipuncture blood samples were obtained from each volunteer at the kindergarten, and collected in lead-free tubes by trained nurses. Lead in total blood was analyzed by graphite furnace atomic absorption spectrometry (GFAAS), which consisted of a Shimadzu AA-660 AAS and GFA-4B graphite furnace atomizer and an ACS-60G autosampler (Shimadzu Corporation, Kyoto, Japan). The main parameters used for the determination were a wavelength of 283.3 nm, current of 8 mA, a slit width of 1.00 nm, drying at 150°C, ashing at 325°C, and atomization at 1,400°C. The accuracy of the method was controlled by recoveries between 95% and 107% from the spiked blood samples. Repeated analyses of standard solutions confirmed the method’s precision. The BLLs were expressed in micrograms per deciliter (1 μg/dL = 0.0484 μmol/L). Meanwhile, we assessed hemoglobin (Hgb) levels by hemoglobin cyanide method with hemoglobinometer (XK-2, JiangSu, China).<br><br>Evaluation of physical developmental indexes
<span class="yellow">Children</span>’s physical growth and development, such as body height, weight, and head and chest circumferences were measured when blood samples were collected. Weight and height were measured using a weighing and height scale (TZ120; Yuyao Balance Instrument Factory, Yuyao, China) with maximum weight of 120 kg (minimum scale, 50 g) and minimum height of 70 cm (minimum scale, 0.5 cm). Head and chest circumferences were measured using graduated anthropometric tapes.<br><br>Statistical analyses
We performed statistical analyses using SPSS version 10.0 software (SPSS, Chicago, IL, USA). We used independent sample t-tests or covariance analyses for comparisons of mean, chi-square analyses for test of frequency data, and linear regression analysis for the association between BLLs and age. Differences were considered significant with a p-value < 0.05.<br><br>
Results
Observation of e-waste processing
The primitive e-waste recycling procedures in Guiyu were mainly as follows: a) Old electronic equipment was dismantled (Figure 2) with electric drill, cutter, hammer, and screwdriver into component parts such as monitor, hard drive, CD driver, wires, cables, circuit boards, transformer, charger, battery, and plastic or metal frame that are sold for reuse or to other workshops for further recycling. b) Circuit boards (Figure 3) of computers and other large appliances were heated over coal fires to melt the solder to release valuable electronic components, such as diodes, resistors, and microchips. c) Circuit boards of cell phones and other hand-held devices were taken apart by a electrothermal machine (Figure 4), which was a particular environmental and <span class="blue">human</span> health concern in the processing of e-waste in Guiyu. d) In acid baths (Figure 5), some microchips and computer parts were soaked to extract precious gold and palladium, from which the waste acids were discharged into nearby fields and streams. e) Wires and cables were stripped or simply burnt in open air to recover metals. f) Printer cartridges were ripped apart for their toner and recyclable aluminum, steel, and plastic parts. g) Plastic [e.g., polyvinyl chloride (PVC), acrylonitrile butadiene styrene copolymer (ABS), high-density polyethylene (HDPE)] was sorted by workers according to rigidity, color, and luster. Plastic scraps that cannot be sorted visually must be burned and classified by burning odor. Another way to sort different plastics was gravitational separation into ceramic jugs with brine (Figure 6), after which the pieces were spread on the sidewalk to dry; h) For reprocessing, after sorting plastic scraps were fed into grinders that spit out tiny pieces of plastic. i) For metals sorting and reprocessing, transformers, chargers, batteries, and cathode-ray tubes were separated and hammered open for recycling metals such as copper, steel, silver, aluminum, which were then reprocessed to raw material.
Although the methods for processing e-waste were primitive, the coordination of e-waste recycling in Guiyu was very well organized into specific tasks. Workshops specializing in dismantled equipment would not conduct circuit board baking or plastics and metals reprocessing. The chain of recycling components from each type of e-waste was well established in the town.<br><br>BLLs in <span class="yellow">children</span>
We collected blood from 165 <span class="yellow">children</span> in Guiyu and 61 <span class="yellow">children</span> in Chendian and measured the BLLs in these <span class="yellow">children</span>. Table 1 shows that the BLLs corresponded to the <span class="yellow">children</span>’s age, sex, and town of residence. As expected, BLLs among Guiyu <span class="yellow">children</span> were much higher than those in the <span class="yellow">children</span> of Chendian (p < 0.01). Among Guiyu <span class="yellow">children</span>, 135 (81.8%) had BLLs > 10 μg/dL, whereas 23 (37.7%) in Chendian (p < 0.01) had high levels. Among 135 (81.8%) Guiyu <span class="yellow">children</span> with elevated BLLs, 61.8% and 20% had BLLs > 10 μg/dL and 20 μg/dL respectively, but lead levels > 45 μg/dL were not found. And BLLs of Guiyu increased somewhat with age (p < 0.01); older <span class="yellow">children</span> tended to have higher BLLs than younger ones. We found no evidence for the association in lead concentrations or prevalence of elevated BLLs differentiated by sex (both p > 0.05).
Table 2 presents BLLs for 165 exposed <span class="yellow">children</span> in the four villages. The findings showed that BLLs from different villages were in the following descending order: Beilin, 19.34 μg/dL > Dutou, 17.86 μg/dL > Huamei, 14.23 μg/dL > Longgang, 13.13 μg/dL (Table 2). <span class="yellow">Children</span> living in Beilin, where the number of e-waste workshops specializing in equipment dismantling, circuit board baking, and acid baths, had the highest BLLs. Dutou, which had many workshops specializing in plastics sorting, including strip plastic materials from e-waste, had the second highest BLLs in <span class="yellow">children</span>. Huamei had e-waste workshops similar to those of Beilin, but fewer and less centralized; the BLLs of Huamei <span class="yellow">children</span> were much lower than those of Beilin and Dutou. Longgang, a village specializing in reprocessing plastics collected from other villages that had no workshops directly processing e-waste, had the lowest BLLs. There was a significant difference in BLLs among the <span class="yellow">children</span> of the four villages (p < 0.01). In Beilin and Dutou, 88.8% and 100% <span class="yellow">children</span> had elevated BLLs > 10 μg/dL, respectively.
As far as physical indexes and Hgb levels were concerned, there was no significant difference between Guiyu and Chendian (p > 0.05, Table 3).<br><br>
Discussion
In this study, we observed that the processing of e-waste in Guiyu was very primitive and the recycling industry depended mainly on manual processing methods. Despite the fact that the coordination of the e-waste recycling is well organized in family-based small business units, the manual processing methods and the deposition of the e-waste have contributed to the contamination by heavy metals in the living environment. Examination of the possible impact of the e-waste industry on the BLLs of <span class="yellow">children</span> living in Guiyu revealed that Guiyu <span class="yellow">children</span> had significantly higher BLLs than Chendian <span class="yellow">children</span>. Of <span class="yellow">children</span> tested in Guiyu, 81.8% had BLLs > 10 μg/dL, indicating a correlation between the BLLs in <span class="yellow">children</span> and the numbers of e-waste workshops. We speculated that the elevated BLLs in Guiyu <span class="yellow">children</span> may be directly caused by the contamination of the lead during e-waste recycling. However, further study should be conducted to determine the relationship between BLLs in <span class="yellow">Children</span> and the actual lead contamination in the environment.
Lead is considered one of the major heavy metal contaminants during the process of e-waste recycling. A cathode ray tube inside a television set or a computer monitor contains an average of 4–8 lb lead; monitor glass contains about 20% lead by weight; a typical battery weighs 36 lb and contains about 18 lb of lead. For decades, lead as a major component of solders has been used to attach electronic components to printed circuit boards. Lead compounds have also been used as stabilizers in some PVC cables and other products. Our study demonstrated in Guiyu a significant increasing trend in BLLs with increasing age; older <span class="yellow">children</span> tended to have higher BLLs than younger ones. This might be the result of increasing exposure risk because older <span class="yellow">children</span> might have more outdoor activities. In addition, it may also be attributed to the fact that the heaviest lead-contaminated zone in air after the burning of the e-waste was 75–100 cm above the ground (Wang and Zhang 2006), which was the height range for normal Chinese <span class="yellow">children</span> 5–6 years of age.
In China, the mean BLL of <span class="yellow">children</span> was 9.29 μg/dL, and 33.8% of the subjects had BLLs > 10 μg/dL; <span class="yellow">boys</span>’ mean BLL was 9.64 μg/dL, significantly higher than the <span class="yellow">girls</span>’ mean BLL of 8.94 μg/dL (p < 0.001) (Wang and Zhang 2006). Generally in China, BLLs of <span class="yellow">children</span> living in industrial and urban areas were significantly higher than those of <span class="yellow">children</span> in suburbs and rural areas (Wang and Zhang 2006). In Guiyu, the BLLs of <span class="yellow">children</span> were higher than the mean level in China, and there were no significant different between <span class="yellow">boys</span> and <span class="yellow">girls</span>. Although Guiyu is rural, the <span class="yellow">children</span>’s BLLs were nearly double those of a nearby urban area, Shantou City (7.9 μg/dL; Luo et al. 2003). Compared with results from studies conducted in some other part of Guangdong province, such as Zhongshan City (7.45 μg/dL; Huang et al. 2003) and Shenzhen City (9.06 μg/dL; Wang et al. 2003), we observed higher BLLs not only in Guiyu <span class="yellow">children</span>, but also in Chendian <span class="yellow">children</span> (9.94 μg/dL). The lead contamination may have spread from Guiyu to nearby Chendian by dust, river, and air and contributed to the elevation of Chendian <span class="yellow">children</span>’s BLLs.
In conclusion, elevated BLLs in Guiyu <span class="yellow">children</span> are common as a result of exposure to lead contamination caused by primitive e-waste recycling activities. Lead contamination from e-waste processing appears to have reached the level considered to be a serious threat to <span class="yellow">children</span>’s health around the e-waste recycling area. Based on these threats, it is necessary to increase public awareness about the effects of exposure to lead from e-waste and arouse local governments’ interest in public health and safety, so that an infrastructure for safe management of e-waste can be established. More important, responsible management strategies should be undertaken to minimize e-waste production and make e-waste components more easily recycled and reused.<br><br>Correction
In the Abstract and Discussion, the percentage of Guiyu <span class="yellow">children</span> with BLLs > 10 μg/dL has been corrected from 88% in the original manuscript published online to 81.8%.<br><br>
<h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="blue">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="blue">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="blue">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="blue">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="blue">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="blue">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="blue">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA2065882</h3>Identification of Two Independent Risk Factors for Lupus within the MHC in United Kingdom Families
Abstract
The association of the major histocompatibility complex (MHC) with SLE is well established yet the causal variants arising from this region remain to be identified, largely due to inadequate study design and the strong linkage disequilibrium demonstrated by genes across this locus. The majority of studies thus far have identified strong association with classical class II alleles, in particular HLA-DRB1*0301 and HLA-DRB1*1501. Additional associations have been reported with class III alleles; specifically, complement C4 null alleles and a tumor necrosis factor promoter SNP (TNF-308G/A). However, the relative effects of these class II and class III variants have not been determined. We have thus used a family-based approach to map association signals across the MHC class II and class III regions in a cohort of 314 complete United Kingdom Caucasian SLE trios by typing tagging SNPs together with classical typing of the HLA-DRB1 locus. Using TDT and conditional regression analyses, we have demonstrated the presence of two distinct and independent association signals in SLE: HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, OR = 2.3) and the T allele of SNP rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001, OR = 2.0) in intron 6 of the class III region gene SKIV2L. Assessment of genotypic risk demonstrates a likely dominant model of inheritance for HLA-DRB1*0301, while rs419788-T confers susceptibility in an additive manner. Furthermore, by comparing transmitted and untransmitted parental chromosomes, we have delimited our class II signal to a 180 kb region encompassing the alleles HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201 alone. Our class III signal importantly excludes independent association at the TNF promoter polymorphism, TNF-308G/A, in our SLE cohort and provides a potentially novel locus for future genetic and functional studies.
Systemic lupus erythematosus (SLE/lupus) is a complex autoimmune disease in which the body's immune system attacks its own tissues, causing inflammation in a variety of different organs such as the skin, joints, and kidneys. The cause of lupus is not known, but genes play a significant role in the predisposition to disease. The major histocompatibility complex (MHC) on Chromosome 6 contains at least 100 different genes that affect the immune system, including the genes with the strongest effect on lupus susceptibility. Despite the importance of the MHC in SLE, the identity of the actual genes in the MHC region that cause SLE has remained elusive. In the present study, we used the latest set of genetic markers present at the MHC in lupus families to identify the actual genes that affect the disease. To our knowledge, we have shown for the first time that two separate groups of genes are involved in SLE. One group of genes alters how the immune system may inappropriately target its own tissues in the disease. How the second set of genes predisposes to SLE is the subject of ongoing study.<br><br><br><br>Introduction
Since the early 1970s, the <span class="blue">human</span> major histocompatibility complex (MHC) has been shown to be associated with a number of autoimmune, inflammatory, and infectious diseases, and it continues to be the focus of intense research [1]. The recently defined extended MHC (xMHC) encompasses 7.6 Mb of genome on 6p21.3 and is divided into five subregions from telomere to centromere: extended class I, classical class I, classical class III, classical class II, and extended class II. In addition, the MHC contains two hypervariable regions, the RCCX module in class III (spanning complement C4) and the HLA-DRB genes in class II, that both exhibit copy number polymorphism. Examination of the sequence across the extended MHC reveals the presence of 421 genes, and over 252 (60%) are thought to be expressed [2]. Around 40% of genes expressed within the classical MHC encode proteins with putative immunomodulatory function [3]. The classical class I and class II loci encode the <span class="blue">human</span> leucocyte antigen (HLA) proteins involved in antigen presentation to T cells, initiating the adaptive immune response. The class III region contains the greatest density of genes in the genome (58 expressed genes), which are often found in functionally related clusters [2].
A major obstacle in the identification of disease-specific causal variants within the MHC has been the strong linkage disequilibrium (LD) exhibited by certain alleles in this region, resulting in the existence of long-range, conserved, extended haplotypes [4], also known as ancestral haplotypes [5], sometimes spanning more than 2 Mb [6]. Thus, for many MHC-associated diseases, it has only been possible to delimit association signals to a particular extended haplotype or segment of one.
Systemic lupus erythematosus (SLE/lupus, [Online Mendelian Inheritance in <span class="yellow">Man</span> 152700, http://www.ncbi.nlm.nih.gov/sites/entrez?db=OMIM&TabCmd=Limits]) is a chronic, multi-system autoimmune disease affecting young <span class="yellow">women</span> ten times more commonly than <span class="yellow">men</span>. The worldwide prevalence of SLE is estimated at between 12 and 124 cases per 100,000 individuals [7]. SLE is characterized by the presence of pathogenic autoantibodies to nuclear and cell-surface antigens that show affinity maturation. The consequent immune complexes deposit in tissues, causing inflammation and damage. It is well established that there is a complex genetic component to lupus aetiology, with hormonal and environmental influences also contributing to disease susceptibility [8,9].
The MHC has been the most consistently confirmed genetic risk factor for SLE, and multiple different genes within the region have been significantly implicated with disease susceptibility. For example, hereditary and acquired deficiencies of the early classical complement component C4, located within the MHC class III locus, leads to a lupus-like syndrome. A role for another class III gene, tumour necrosis factor alpha (TNF), in SLE was suggested following the observation that the lupus-prone New Zealand F1 <span class="yellow">mouse</span> hybrid exhibits constitutively low TNF expression [10]. Recently, the development of antinuclear antibodies in <span class="yellow">patients</span> treated with TNF antagonists has also stimulated interest in the possible role of TNF in SLE [11–13]. <span class="yellow">Murine</span> and <span class="blue">human</span> candidate gene studies, together with genome-wide linkage screens, provide further support that multiple genetic loci, including the <span class="yellow">mouse</span> MHC complex H2 and the <span class="blue">human</span> MHC locus, contribute to disease susceptibility [14–17].
It should be noted that the <span class="blue">human</span> MHC was first associated with SLE in 1971, when studies demonstrated that lupus probands were enriched for the class I alleles HL-A8 (now known as HLA-B8) and HLA-W15 (now known as HLA-B15) when compared with healthy controls [18,19]. Further case control association studies were small, performed in ethnically diverse populations, and tested only a small number of the classical HLA and complement C4 alleles. The most consistent findings reported to date are associations with the class II alleles HLA-DR2 (DRB1*1501) and HLA-DR3 (DRB1*0301) and their respective haplotypes in Caucasian populations. The complement C4A null allele (C4A*Q0) has shown inconsistent association with lupus in a number of studies—a situation that may reflect genetic heterogeneity in disease susceptibility [20–23]. In addition, a recent study has demonstrated that low C4A copy number is a risk factor for lupus in a European American cohort [24]. However, the C4A null allele lies on the lupus-associated DR3 “autoimmune” extended haplotype (AH8.1), which exhibits extremely strong LD [6]. It therefore remains to be definitively established whether this locus constitutes a distinct susceptibility allele to that of the class II association or is merely in LD with it. Similarly, certain TNF promoter polymorphisms, including the much-studied SNP TNF-308G/A, have shown association with SLE; but again, many of these variants are carried on the highly conserved 8.1 ancestral haplotype, thus restricting interpretation of these data.
In 2002, a family-based study employing microsatellites as surrogate markers for HLA-DRB1 haplotypes in Caucasian lupus families demonstrated association with DR3-, DR2-, and DR8 (DRB1*0801)-containing haplotypes. In that study, Graham and colleagues reported that, taking advantage of recombinant chromosomes, the disease risk region could be limited to a 1 Mb region encompassing classical class II and class III [25].
We have performed a medium resolution association mapping study of the MHC in lupus families, utilizing a combination of SNPs and four-digit typing at the HLA-DRB1 locus in order to anchor haplotypes. Sixty-eight SNPs were successfully genotyped across a 2.4 Mb region of the MHC, from the class I locus KIAA1949 to the class II gene HLA-DPB2, in 314 UK Caucasian SLE trios. We used these data to perform a family-based association study in an attempt to distinguish the relative effects of the class II and class III regions of the MHC in lupus susceptibility. In addition, we employed the long-range haplotype test to search for the presence of high-frequency, extended haplotypes indicative of recent positive selection [26]. We have also used family-based and case-control strategies to examine genotypic risk at HLA-DRB1 and rs419788.<br><br>Results
Association Testing of HLA-DRB1 and MHC Region SNPs
In order to define the causal variation within the MHC region, we typed 314 complete SLE trios for the HLA-DRB1 gene as well as for 86 SNPs across a 2.4 Mb region encompassing the HLA class I locus HLA-B to HLA-DPB2. High-quality genotype data was obtained for HLA-DRB1 and 68 MHC SNPs (see Table S1 for quality control data). Association testing of the HLA-DRB1 gene revealed a significant association with HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, T:U = 129:55) in our lupus cohort (Table 1). There was also a trend for under transmission of the HLA-DRB1*0701 allele (nominal p = 0.0013, T:U 42:77); however, this association was no longer significant after correction for multiple testing as determined by 10,000 permutations of the dataset (permuted p = 0.09). Furthermore, we did not find evidence of association with HLA-DRB1*1501 (nominal p = 1.0, T:U 70:70) or HLA-DRB1*0801 (nominal p = 1.0, T:U = 11:11) in our cohort (see Table S2 for complete HLA-DRB1 association data); alleles previously suggested by microsatellite typing of a US lupus cohort [25].
Association testing of the MHC region SNPs also identified significant evidence of association to SLE (Table 1 for associated markers and Table S3 for all MHC SNPs). The SNP with the most significant association, rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001) was of similar strength to that of the HLA-DRB1*0301 allele, with odds ratios (ORs) and 95% confidence intervals (CIs) of 2.0 (1.6–2.6) and 2.3 (1.7–3.2), respectively. This SNP is located within intron 6 of the class III gene, superkiller viralicidic activity 2-like (<span class="yellow">Saccharomyces cerevisiae</span>) (SKIV2L), and is located approximately 500 kb telomeric to the HLA-DRB1 gene. Of the other 12 SNPs that were significantly associated with SLE (nominal p = 4.0 × 10−4 to 2.5 × 10−7; permuted p = 0.03 to <0.0001), one was located in the class I region between HLA-B and MICA, seven were located in the class III region, and four were situated in the class II region (Table 1; Figure S1). Specifically, the seven associated class III SNPs were located in or close to the following genes: the TNF promoter, BAT3, SLC44A4, EHMT2, TNXB, GPSM3, and NOTCH4. One of the four class II associated SNPs was close to HLA-DRA, two were between HLA-DRB1 and HLA-DQA1 and one was in intron 1 of HLA-DQA1. The correlation between all 68 SNPs and HLA-DRB1 in our UK SLE cohort is illustrated in Figure 1. The markers showing significant association are highlighted.<br><br>Conditional Analyses Identify Two Independent Association Signals in the MHC
In order to establish whether the two most associated signals identified in this association-mapping experiment are likely to represent a single causal allele or independent risk factors, we first examined the association data conditioned upon the presence of the HLA-DRB1*0301 allele. We found that four of the 13 associated SNPs showed evidence of signals independent of HLA-DRB1*0301 in our dataset, the strongest of which was rs419788 (Table 1). We therefore conditioned the three remaining SNPs (rs2523589, rs1052486, and rs605203) on rs419788 to assess whether these signals are truly independent of each other or show association due to LD with rs419788. In addition, we included HLA-DRB1 in stepwise conditional regression analyses performed on the SNPs showing association independent of HLA-DRB1 (unpublished data). These analyses demonstrated that the observed association signals at rs2523589, rs1052486, and rs605203 were predominantly dependent upon the association at rs419788, and suggested that there are two major independent association signals in the MHC in UK SLE: HLA-DRB1 and rs419788. The independence of the association signals at HLA-DRB1 and rs419788 is further supported by the observation that there is only modest LD between these two (r2 = 0.24). There was no association with any other HLA-DRB1 allele and the four SNPs independent of HLA-DRB1*0301 (TRANSMIT, unpublished data).
The association of the tumour necrosis factor gene promoter SNP TNF-308G/A with SLE is lost after conditioning for HLA-DRB1*0301 in our cohort. If we perform the reverse analysis and condition HLA-DRB1*0301 on the presence of the TNF promoter SNP, we find that the association remains, confirming that our TNF association is secondary to that of HLA-DRB1*0301.<br><br>Genotypic Risk for Class II and Class III Association Signals
Having established independent association at the allelic level with HLA-DRB1*0301 and rs419788-T in our UK SLE cohort, we wanted to further determine the genotypic risk conferred by these variants and hence gain insight into their underlying mode of inheritance in lupus. We used case-control and family-based analyses to assess genotypic risk at HLA-DRB1, while the family-based test alone was used for rs419788. Common family-based tests of LD, such as those used in this study (Genehunter), measure transmission distortion based on allele counts rather than genotype counts; the former has been shown to be more powerful under additive models, while the latter has greater power under recessive or dominant genetic models [27]. The genotype-pedigree disequilibrium test (geno-PDT) determines LD between a locus genotype and disease by comparing genotypes that are transmitted from parent to proband with those that are not [27]. We used the geno-PDT to assess genotypic risk for our class II and class III association signals: HLA-DRB1 and the SNP rs419788. In the case-control analysis for HLA-DRB1, ORs with 95% CI were calculated and Fisher's exact test employed to assess statistically significant differences between HLA-DRB1 genotypes in lupus probands and healthy controls. For HLA-DRB1, the alleles were coded as follows: HLA-DRB1*0301, HLA-DRB1*1501, HLA-DRB1*X where X represents all HLA-DRB1 alleles other than HLA-DRB1*0301, and HLA-DRB1*1501. We included HLA-DRB1*1501 in the analysis, even though we find no allelic association in our cohort, because previous studies have shown a greater risk for lupus in individuals who are compound heterozygotes for HLA-DRB1*0301- and HLA-DRB1*1501-containing haplotypes [25,28]. Overall the results are consistent with a dominant effect from HLA-DRB1*0301 (Table 2) and a dose-dependent (additive) effect from rs419788-T (Table 3). Specifically, both case-control and geno-PDT demonstrate that there is no dose-dependent increase in disease risk for HLA-DRB1*0301. Rather, it appears that the presence of a single copy of HLA-DRB1*0301 alone is sufficient to increase susceptibility to disease. Moreover the 0301/X genotypes constitute the greatest risk in our cohort rather than the 0301/1501 heterozygotes. Genotypes containing HLA-DRB1*1501 in the absence of HLA-DRB1*0301 revealed no significant association in our cohort.
All three rs419788 genotypes demonstrated significant association in our lupus families (Table 3). The common CC genotype was significantly under transmitted, while the rare T allele displayed dose-dependent over transmission to lupus probands.<br><br>Characterization of HLA-DRB1*0301 Risk Haplotype
Next, we wanted to further delimit the MHC class II association signal that we have detected at HLA-DRB1. We used phased parental genotype data to compare the allelic composition of HLA-DRB1*0301-bearing haplotypes that were transmitted (T) to affected probands to those that were not transmitted (or untransmitted, UT) with the aim of identifying differences that could delineate the lupus susceptibility interval(s) arising from this haplotype (summarized in Figures 2A, 2B, and S2). We observed a striking difference between transmitted and untransmitted chromosomes within the class II region: nearly all transmitted HLA-DRB1*0301 haplotypes (99%) are identical across a 180 kb region defined by eight SNPs, whereas the corresponding region within untransmitted HLA-DRB1*0301 haplotypes exhibits significant recombination. These data strongly suggest the existence of a risk haplotype that, interestingly, contains only three expressed genes: HLA-DRB1, HLA-DQA1, and HLA-DQB1. Furthermore, we can confidently define the allelic composition of this risk haplotype, as these three genes are in strong LD and occur in one common haplotype in Caucasians: HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. Thus, we hypothesize that the specific combination of all three alleles is required to confer disease risk in lupus or that disease susceptibility lies with either HLA-DRB1*0301 or the HLA-DQ alleles. We do not have sufficient numbers of recombinant chromosomes in this risk region to further delimit this signal: 2/176 (1.1%) transmitted HLA-DRB1*0301 haplotypes are recombinant at HLA-DQA1-HLA-DQB1; 3/178 (1.7%) transmitted haplotypes identical across HLA-DQA1-HLA-DQB1 do not possess HLA-DRB1*0301.
The composite relative extended haplotype homozygosity (REHH) versus frequency plot for UK SLE; Utah residents with ancestry from northern and western Europe (CEPH); and Yoruba in Ibadan, Nigeria (Yoruba) populations is shown in Figure 3A. We can only comment on evidence for positive selection in CEPH individuals, as we have used this population alone to assess background variation on Chromosome 6. The SLE and Yoruba cohorts are shown for comparative purposes. We find no evidence of positive selection for HLA-DRB1*0301 in the CEPH population. However, this allele is enriched in our lupus cohort (21% of parental chromosomes) and displays greater extended homozygosity when compared with HLA-DRB1*0301-bearing haplotypes in CEPH and Yoruba. Hence, the HLA-DRB1*0301 allele in lupus is observed as an outlier on the plot when compared to background variation in CEPH. These data support our previous observations (outlined above) of the highly conserved nature of HLA-DRB1*0301 haplotypes in lupus. In addition, the haplotype bifurcation plots centered on HLA-DRB1*0301 for UK SLE, CEPH, and Yoruba populations in Figure 3B illustrate preservation of the common HLA-DRB1*0301 haplotype in CEPH and UK SLE, while that seen in the Yoruba is significantly different. The class II regions of all three populations are essentially identical across our chosen SNPs; the main differences lie in class III. The difference in African populations in the class III region is one possible explanation for the lack of evidence for an association between HLA-DRB1*0301 and SLE in African or African American populations. However, HLA-DRB1*0301 has a lower frequency (∼7%–10%) in African populations compared with Europeans (∼13%), and the number of HLA association studies conducted in African populations is very limited.<br><br>Characterization of Class III Region Risk Haplotype
Our data reveal a second independent signal at the MHC in SLE arising from the T allele of SNP rs419788 in intron 6 of the class III gene, SKIV2L. Further evidence supporting the independence of the rs419788-T and HLA-DRB1*0301 alleles is provided by the moderate LD between these two variants (r2 = 0.24) coupled with our data demonstrating that only 47% of rs419788-T allele-bearing haplotypes contain HLA-DRB1*0301.
The structure and composition of T and UT haplotypes anchored at rs419788-T were essentially identical (Figures 2C, 2D, and S2), and hence not informative in delimiting our class III signal. Therefore, we examined the LD structure around our associated class III SNP to better define our disease risk interval. In our lupus dataset the rs419788-T allele resides on three of seven haplotypes present within a large block of six SNPs exhibiting strong LD. This haplotype block encompasses roughly 270 kb containing class III genes from SLC44A4 to AGER, including the RCCX module. Next, we analyzed the haplotype block structure of this region in CEPH families using SNP data dumped from the International HapMap Project (http://www.hapmap.org/). The greater density of SNP typing available in the HapMap CEPH population compared to our current UK SLE map allowed us to potentially refine our signal by exploring correlations between our associated SNP and those surrounding it. Analysis of these data (Figure 4) suggests the presence of short-range LD around our associated variant, rs419788, in CEPH families, encompassing approximately 40 kb of the genome which includes the five genes: complement factor B (CFB), RD RNA binding protein (RDBP), SKIV2L, dom-3 homolg Z (<span class="yellow">C. elegans</span>) (DOM3Z), and serine/threonine kinase 19 (STK19), and does not include the complement C4 locus. Furthermore, assessment of marker association in our lupus dataset demonstrates that after conditioning for HLA-DRB1*0301, the only markers that retain association signals are telomeric of SKIV2L, suggesting that complement C4, which is centromeric to this gene, may not be responsible for our independent class III signal.<br><br>Subphenotype Analysis
In order to gain further insight into disease pathogenesis, we examined common lupus subphenotypes. Such subsets are more homogeneous than lupus per se and thus maybe enriched for specific predisposing variants. In addition, one might expect a close association between MHC class II alleles and autoantibody subsets in lupus if these are indeed causal variants, given their role in antigen presentation and subsequent humoral immunity. We therefore tested our two main MHC association signals, HLA-DRB1*0301 and rs419788, for association with renal disease and autoantibody subsets in our lupus cohort.
We found that HLA-DRB1*0301 was associated with the presence of anti-Ro and anti-La antibodies in our UK SLE cohort, with the latter showing the greatest evidence of association (anti-La nominal p < 0.001 compared with anti-Ro nominal p < 0.025). We found no association of HLA-DRB1*0301 with renal disease or any other autoantibody subsets in our dataset (see Table S4 for detailed results).
Genotypes of the SNP rs419788 were not associated with any of the tested lupus subphenotypes after controlling for the effect of HLA-DRB1*0301 (unpublished data).<br><br>
Discussion
We present the first family-based SNP association study of the MHC in SLE. We have genotyped 69 markers (HLA-DRB1 and 68 SNPs) across 2.4 Mb of the MHC, encompassing class III and class II, in a cohort of 314 UK Caucasian SLE trios. Transmission disequilibrium testing of these data has shown predominant association with the alleles HLA-DRB1*0301 and rs419788-T, together with 12 other MHC SNPs. Moreover, using conditional analyses, we have shown that the two primary signals of association at the MHC are independent of each other. Specifically, one signal arises from HLA-DRB1*0301 in class II and the other from the T allele of SNP rs419788 in the class III gene SKIV2L.
Examination of bifurcation plots for T and UT HLA-DRB1*0301-containing haplotypes has enabled delineation of our class II association signal to a 180 kb region encompassing HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. These data substantially refine that previously published by Graham et al. in 2002 [25], where the lupus susceptibility interval within HLA-DRB1*0301-containing haplotypes could only be delimited to a 1 Mb region encompassing class II and class III. The precise causal variant(s) within this region remains to be determined, as the three implicated alleles exhibit strong LD with few recombination events separating them (two out of 176 transmitted HLA-DRB1*0301 chromosomes in our dataset). However, all three allelic variants represent attractive functional candidates in lupus susceptibility for their role in antigen presentation and stimulation of the adaptive immune response.
Our association of HLA-DRB1*0301 with lupus concurs with published data in Caucasian cohorts and is well established [16]. While our lack of association with HLA-DRB1*1501 and HLA-DRB1*0801 is consistent with previous data from the UK [29], Spain [30], the Netherlands [31], Sweden [32], Mexico [33], and the US [34], it conflicts with that of other US groups [25,35]. Interestingly, we demonstrate a trend, though not statistically significant, for undertransmission of HLA-DRB1*0701—a result also observed in prior UK and Canadian lupus studies [29,36]. Moreover, a negative association of HLA-DRB1*0701 has been reported in other autoimmune diseases including Graves disease [37,38], type 1 diabetes [39], and rheumatoid arthritis [40].
It appears that the conflicting results between UK SLE and previous US (Minnesota [MN]) [25] SLE data stem from differences in HLA-DRB1 allele frequency in the probands of each cohort. The reason for this is unclear. A comparison between UK and MN SLE cohorts (Table 4) reveals that UK SLE cases are enriched for HLA-DRB1*0301 but not HLA-DRB1*0801 or HLA-DRB1*1501 when compared to a UK control population. In contrast, MN SLE cases are enriched for HLA-DRB1*0301-DQB1*0201, DRB1*0801-DQB1*0402, and DRB1*1501-DQB1*0602 inferred haplotypes when compared to MN controls [25]. There is no statistically significant difference in the aforementioned HLA class II alleles/haplotypes between UK and MN control populations that could account for the disparity seen in the respective lupus cohorts. Differences in disease severity and subphenotype frequency between the two populations could account for the observed discrepancy. From the limited data available we found that the presence of renal disease appears to be similar in both cohorts (UK SLE 36% compared with MN SLE 40%), while the gender ratios are significantly different (female: male UK SLE 11:1 compared with MN SLE 57:1, Chi square p value < 0.001). We were unable to compare other lupus subphenotypes. Furthermore, closer inspection of these data reveals that microsatellite-inference of HLA-DRB1 alleles in the MN SLE dataset may underestimate the frequency of HLA-DRB1*0301 and overestimate that of HLA-DRB1*1501, thus diminishing the effect of the former and enhancing that of the latter. It is also possible that the MN SLE cohort shows greater racial heterogeneity in comparison to our UK SLE cohort, despite both being characterized as Caucasian.
Previous studies have demonstrated increased risk for lupus in individuals carrying particular combinations of microsatellite-inferred HLA-DRB1-HLA-DQB1 haplotypes [25,28]. The highest risk genotype was found to be the compound heterozygote HLA-DRB1*0301-DQB1*0201/HLA-DRB1*1501-DQB1*0602, while HLA-DRB1*0301-DQB1*0201-containing genotypes demonstrated a dose-dependent effect in increasing lupus susceptibility [25,28]. In the present study, we have examined genotypic risk at the classically typed HLA-DRB1 locus and in contrast to the aforementioned data of Graham et al. [25,28] we have shown a likely dominant effect of the associated allele, HLA-DRB1*0301. The case-control and family-based analyses for HLA-DRB1 also show the greater power of the former to detect significant association (Table 2). Specifically, all genotypes containing HLA-DRB1*0301 show increased transmission to lupus probands; however, homozygotes show no greater risk compared with heterozygotes, as would be expected under additive or multiplicative models. Thus, a dominant model of inheritance, requiring the presence of a single copy of the disease-predisposing variant alone, likely underlies the susceptibility conferred by HLA-DRB1*0301 in UK SLE. Such a model would fit an antigen presentation hypothesis where susceptible individuals carrying an HLA-DRB1*0301 allele are able to present auto-antigens to CD4+ lymphocytes, thus stimulating an autoimmune response. The differences between our UK SLE and the previously published US SLE data may reflect disease, ethnic, and haplotypic heterogeneity.
Interestingly, analysis of genotypic risk at the associated class III marker, rs419788, suggests an additive (dose-dependent) pattern of inheritance for the rare T allele, where one copy confers a low risk of disease and two copies results in greater susceptibility. The different inheritance patterns for our class II and class III association signals provide further evidence for their independence.
A variety of HLA-DR and HLA-DQ alleles have been associated with autoantibody subsets in ethnically diverse populations of lupus. The strongest associations have been demonstrated between anti-Ro and anti-La antibodies and HLA-DR3 and HLA-DQ2 (HLA-DQB1*0201), which are in strong LD [41–45] in case-control studies. Here, we confirm the association of HLA-DRB1*0301 with anti-Ro and anti-La antibody production in our family-based cohort.
Examination of LD structure around our second independent association, rs419788-T in class III, coupled with the results of our conditional analysis, suggests that this signal could also be delimited to a relatively narrow genomic interval of about 40 kb given further SNP mapping in our cohort. This region includes the genes CFB, RDBP, SKIV2L, DOM3Z, and STK19, but does not include complement C4. Thus, complement C4 null alleles, which have been implicated in lupus pathogenesis, may not be responsible for our class III signal. We conclude, therefore, that our family-based mapping study has potentially revealed a hitherto unknown lupus susceptibility interval in the class III region of the MHC. However, we cannot conclusively exclude association at complement C4/RCCX without direct determination of C4 polymorphism/copy number in our cohort.
With respect to the genes implicated in our study, CFB is a vital component of the alternate complement pathway and disregulation may clearly affect the inflammatory response [46]. RD and Skiv2l are proteins potentially involved in RNA processing. The RD protein forms part of a negative elongation factor (NELF) complex that represses RNA polymerase II transcript elongation, while Skiv2l is a DEAD box protein with possible function as an RNA helicase. The function of Dom3z is currently unknown, although the homologous <span class="yellow">yeast</span> protein binds nuclear exoribonuclease. Moreover, its ubiquitous expression suggests a housekeeping role. STK19 is a protein kinase of unknown function with primary nuclear localization [47]. Interestingly, RDBP and SKIV2L are found to be highly expressed in T lymphocytes, B lymphocytes, and dendritic cells (SymAtlas, http://symatlas.gnf.org/SymAtlas/).
A number of studies have demonstrated conflicting evidence for and against association with various TNF locus polymorphisms in SLE [48]. A recent meta-analysis of the TNF-308G/A promoter polymorphism in SLE [48] revealed evidence of association for the minor allele (A) in European populations; however, this study did not account for LD with class II alleles. On conditioning our dataset for HLA-DRB1*0301, we find that the TNF promoter signal is lost, suggesting that this association is not independent and is due to LD with HLA-DRB1*0301 (or another variant in LD with HLA-DRB1*0301).
In summary, we have found association with two distinct and independent variants within the class II (HLA-DRB1*0301) and class III (SKIV2L) regions of the MHC in UK SLE trios. We can delimit our class II signal in lupus to three genetic variants (HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201) that may confer disease risk in combination or as separate signals. Our class III signal importantly excludes independent association at the TNF promoter polymorphism TNF-308G/A and potentially provides a novel locus for further study.<br><br>Materials and Methods
Study cohorts.
SLE families. The cohort comprises 314 complete SLE trios (that is, mother, father, and affected lupus proband) collected as previously described [49]. All study <span class="yellow">participants</span> are European Caucasian on the basis of grandparental origin. All 314 lupus probands (288 female, 26 male) fulfill the revised American College of Rheumatology (ACR) criteria for SLE [50], 36% of whom have a diagnosis of lupus nephritis. Written consent was obtained from all study <span class="yellow">participants</span> and ethical approval for this study was obtained from the Multi-Centre Research Ethics Committee (MREC 2 June 1998).<br><br>Healthy controls.
The control population for the HLA-DRB1 genotypic risk case-control analysis constitutes 1,667 healthy males of Northern European origin. The individuals are potential hematopoietic stem cell donors and were typed to four digits for HLA-DRB1 at the Anthony Nolan Trust, UK for this purpose. The level of resolution used for the typing of HLA-DRB1*15 alleles in these healthy controls resulted in the ambiguous allele string HLA-DRB1*1501/1502/1504/1506. However, it is likely that the great majority are HLA-DRB1*1501. There is no gender bias in HLA-DRB1 allele frequencies, so although we have used a male control cohort, the frequencies would be expected to be the same in a similar female cohort (Steven Marsh, personal communication).<br><br>SNP genotyping.
Eighty-six SNPs were chosen for genotyping in our mapping study. Specifically, we selected 40 MHC class II and class III haplotype tagging SNPs from a preliminary MHC SNP map [51] that had previously shown robust genotyping efficacy. In addition, we typed 36 MHC class II tag SNPs from a subsequent high-resolution MHC study [52]. We also included the TNF-308G/A promoter SNP, together with nine further SNPs in the region of HLA-B and MICA obtained from the database, dbSNP (http://www.ncbi.nlm.nih.gov/projects/SNP/). The latter SNPs had not been well characterized. All variants were typed in the entire cohort (n = 942). The SNPs span approximately 2.4 Mb of the MHC from the class I gene, KIAA1949 to the class II pseudogene, HLA-DPB2 and thus encompass MHC class III and class II. SNP genotyping was performed at the Broad Institute of MIT and Harvard and at Imperial College London by matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry using the Sequenom MassARRAY platform as previously described [53]. SNPs that failed Sequenom typing were retyped by KBiosciences (http://www.kbioscience.co.uk/) using their in-house SNP genotyping methodology, KASPar (http://www.kbioscience.co.uk/genotyping/index.htm), a competitive allele-specific PCR technique.<br><br>HLA-DRB1 genotyping.
HLA-DRB1 typing was performed at the Anthony Nolan Trust, UK. All samples (n = 942 UK SLE trios and n = 1,667 controls) were genotyped using LABType SSO (sequence-specific oligonucleotide) typing technology according to the manufacturer's written recommendations (http://www.onelambda.com). Briefly, a locus-specific biotinylated PCR amplicon is produced, denatured, and rehybridized to complementary oligonucleotide probes conjugated to fluorescently coded beads. The bound biotinylated PCR product can be detected using R-phycoerythrin-conjugated streptavidin. A flow analyzer identifies the fluorescent intensity of phycoerythrin on each bead. The assignment of HLA type is based on the reaction pattern compared to patterns associated with known sequences.
High resolution testing was performed using the Dynal AllSet+ SSP (sequence-specific primers) DRB1 assay according to the manufacturer's protocol (Invitrogen) for SLE families only. The presence or absence of PCR amplification was detected in a gel electrophoresis step using visualization by ethidium bromide incorporation. Genotypes were determined using SSPTool software.
Samples that could not be resolved to four digits using SSO and PCR-SSP were analyzed by DNA sequencing of exon 2 of HLA-DRB1. Primers, reagents, and protocols were provided by The Anthony Nolan Trust, UK (primer sequences are available on request). Specific HLA-DRB1 alleles were assigned by comparing the resultant sequence with reference sequence from the IMGT/HLA Database [54].<br><br>Data analysis.
Mendelian inconsistencies were removed using PedCheck [55]. Families in which more than eight markers demonstrated Mendel errors were removed from further analysis. Markers with less than 80% genotyping efficiency and markers where more than eight families showed Mendel errors were also excluded from analysis. Two markers located within the SLE associated class II region (rs2239802 in intron 4 of HLA-DRA and rs6457594 in the region between HLA-DRB9 and HLA-DRB5) show deviation from Hardy–Weinberg equilibrium (HWE), which may reflect an undetected SLE association or systematic genotyping error. HWE was assessed in parental samples in our cohort. There is currently no uniform opinion in the community regarding the inclusion or exclusion of SNPs that show deviation from HWE, hence we elected to include these markers in the final analysis.
Sixty-eight out of the total 86 SNPs passed our quality-control measures (see Table S1 for details). In summary, one SNP was monomorphic in our dataset, four SNPs yielded low genotyping efficiency, and 13 SNPs were excluded for unacceptable Mendel error rate. The mean call rate for all markers post-quality control was 94% (range 83% to 99%).
Family-based association testing was performed using Genehunter TDT (version 2.1) [56] and TRANSMIT (version 2.5) [57]. Haplotypes were constructed and permutation testing performed using Haploview (version 3.32) [58]. Significance of association signals in all analyses was based on permutation testing (10,000 permutations). The data are represented both as nominal and permuted p values. ORs with 95% CI for family-based analyses were calculated in PLINK (http://pngu.mgh.harvard.edu/purcell/plink/) [59]. Conditional regression analyses were undertaken using WHAP [60].
The geno-PDT was performed using PDT version 5.1 with default settings [27]. The HLA-DRB1 alleles were coded into three groups for the geno-PDT and the case-control analysis: HLA-DRB1*0301, HLA-DRB1*1501 and HLA-DRX where HLA-DRX includes all HLA-DRB1 alleles other than HLA-DRB1*0301 or HLA-DRB1*1501. The HLA-DRB1*1501 code in the healthy controls represents the allele string HLA-DRB1*1501/1502/1504/1506, as described previously. The HLA-DRB1*1501 code in the lupus probands represents the alleles HLA-DRB1*1501 (84 out of 91 *1501 and *1502 alleles) and HLA-DRB1*1502 (7/91 *1501 and *1502 alleles), as *1504 and *1506 were not present in this population. Fisher's exact test was used to assess significance of association in the case-control analysis.<br><br>Subphenotype analysis.
We looked for association of the HLA-DRB1*0301 allele with autoantibody subsets and renal disease in our cohort using the Chi-square test. We compared cases with and without the subphenotype of interest with DRB1*0301 homozygosity, heterozygosity, combined homozygosity and heterozygosity, and non-DRB1*0301 status. We performed the same analyses for homozygous and heterozygous genotypes of the associated SNP rs419788. The autoantibody subsets compared were anti-C1q, IgG, and IgM anti-cardiolipin antibodies (ACLG and ACLM), anti-Ro, anti-La, anti-RNP, anti-Sm, and anti-dsDNA.<br><br>Delineation of associated MHC haplotypes and evidence for positive selection.
We looked for positively selected alleles in our dataset using the long-range haplotype test as measured by extended haplotype homozygosity (EHH), previously described by Sabeti et al. [26]. Essentially, such an analysis allows assessment of positive selection by mining datasets for high frequency extended haplotypes in comparison to the other core haplotypes at a locus.
EHH is defined as the probability that two randomly chosen chromosomes carrying the core haplotype of interest will be identical by descent (homozygosity at all SNPs) for the entire interval from the core to a distance x. The REHH is the ratio of the EHH on the tested core haplotype compared with the combined EHH of all the other core haplotypes at the region excluding the tested core; as such, REHH accounts for local variation in recombination rate while EHH does not [26].
The program emphase was employed to assign the phase of parental genotype data and reconstruct missing information. Emphase is a simple phaser similar to the phaser of Excoffier and Slatkin [61]. It is very fast, especially on large datasets, and sufficiently accurate for most genetic applications. EHH analysis was performed on the phased parental data using the software program SWEEP (http://www.broad.mit.edu/mpg/sweep/index.html).<br><br>Haplotype bifurcation plots.
We represent the breakdown of LD on core haplotypes using haplotype bifurcation diagrams generated in the program TREE [62] (also explained in [52]).<br><br>REHH versus frequency plots.
Fifty-three SNPs (identified in Figure 1) are common to our dataset and the CEPH and Yoruba HapMap [63] populations. These three datasets, together with CEPH SNP data for Chromosome 6 in its entirety, were used to generate separate REHH versus frequency plots in SWEEP. The plots from the four cohorts were combined for visual, not statistical, comparison. Evidence for positive selection was quantitatively assessed in CEPH individuals, as this population alone was used to assess background variation on Chromosome 6. The UK SLE and Yoruba cohort data are shown for comparison. The 95th percentile based on total CEPH Chromosome 6 SNP data is shown.<br><br>
Supporting Information
Accession Numbers
The Online Mendelian Inheritance in <span class="yellow">Man</span> (OMIM, http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim) accession numbers for the genes described in this study are as follows: AGER, 600214; BAT3, 142590; C4A, 120810; C4B, 120820; CFB, 138470; DOM3Z, 605996; DOM3Z, 605996; EHMT2, 604599; HLA-B, 142830; HLA-DPB2, 120290; HLA-DQA1, 146880; HLA-DQB1, 604305; HLA-DRA, 142860; HLA-DRB1, 142857; KIAA1949, 610990; MICA, 600169; NOTCH4, 164951; RDBP, 154040; RDBP, 154040; SKIV2L, 600478; SLC44A4, 606107; STK19, 604977; STK19, 604977; TNF, 191160; and TNXB, 600985.<br><br><br><br><h3>pmcA2238978</h3>SuperCAT: a supertree database for combined and integrative multilocus sequence typing analysis of the <span class="yellow">Bacillus cereus group</span> of bacteria (including <span class="yellow">B. cereus</span>, <span class="yellow">B. anthracis</span> and <span class="yellow">B. thuringiensis</span>)
Abstract
The <span class="yellow">Bacillus cereus group</span> of bacteria is an important group including mammalian and insect pathogens, such as <span class="yellow">B. anthracis</span>, the <span class="yellow">anthrax</span> bacterium, <span class="yellow">B. thuringiensis</span>, used as a biological pesticide and <span class="yellow">B. cereus</span>, often involved in food poisoning incidents. To characterize the population structure and epidemiology of these bacteria, five separate multilocus sequence typing (MLST) schemes have been developed, which makes results difficult to compare. Therefore, we have developed a database that compiles and integrates MLST data from all five schemes for the <span class="yellow">B. cereus group</span>, accessible at http://mlstoslo.uio.no/. Supertree techniques were used to combine the phylogenetic information from analysis of all schemes and datasets, in order to produce an integrated view of the <span class="yellow">B. cereus group</span> population. The database currently contains strain information and sequence data for 1029 isolates and 26 housekeeping gene fragments, which can be searched by keywords, MLST scheme, or sequence similarity. Supertrees can be browsed according to various criteria such as species, isolate source, or genetic distance, and subtrees containing strains of interest can be extracted. Besides analysis of the available data, the user has the possibility to enter her/his own sequences and compare them to the database and/or include them into the supertree reconstructions.<br><br>INTRODUCTION
Multilocus sequence typing (MLST) is a tool that is widely used for phylogenetic typing of bacteria. MLST is based on polymerase chain reaction (PCR) amplification and sequencing of internal fragments of usually seven essential or housekeeping genes spread around the bacterial chromosome. The genetic relatedness among isolates is then determined by comparison of the nucleotide sequence types (1,2). MLST is thus a method that is unambiguous and truly portable among laboratories. Since the initial development of this technique for <span class="yellow">Neisseria meningitidis</span> in 1998, MLST schemes have been developed for about 30 species including some of the most important bacterial pathogens, e.g. <span class="yellow">Streptococcus pneumoniae</span>, <span class="yellow">Streptococcus pyogenes</span>, <span class="yellow">Haemophilus influenzae</span>, <span class="yellow">Staphylococcus aureus</span>, <span class="yellow">Campylobacter jejuni</span>, <span class="yellow">Enterococcus faecium</span>, <span class="yellow">Burkholderia pseudomallei</span>, <span class="yellow">Escherichia coli</span>, <span class="yellow">Salmonella enterica</span> and the <span class="yellow">Bacillus cereus group</span> (see (1) for a recent review). These MLST schemes have been used successfully to explore the population structure of bacteria, to study the evolution of their virulence properties, to identify antibiotic-resistant strains and epidemic clones, and for epidemiological surveillance.
The <span class="yellow">B. cereus group</span> includes bacterial species that are of medical and/or economic importance, such as <span class="yellow">B. anthracis</span>, an obligate mammalian pathogen causing the lethal disease <span class="yellow">anthrax</span>, <span class="yellow">B. cereus</span>, an opportunistic <span class="blue">human</span> pathogen involved in food-poisoning incidents and contaminations in hospitals, <span class="yellow">B. thuringiensis</span>, an insect pathogen and one of the world's most widely used biopesticide and <span class="yellow">B. weihenstephanensis</span>, a cold-tolerant species known for contaminating dairies. These species are genetically very closely related and may be considered as one species based on genetic and genomic evidence (3–5). Unlike other bacterial species that are typed using a single MLST scheme, five separate schemes have been developed for the <span class="yellow">B. cereus group</span>, based on different sets of genes and isolates (5–10). The Priest scheme (8) is currently the most widely used. Studies with the various schemes have independently indicated that the <span class="yellow">B. cereus group</span> population is divided into three main phylogenetic clusters and that species are usually intermixed within the groups. One cluster contains the monomorphic <span class="yellow">B. anthracis</span> isolates and a number of <span class="yellow">B. cereus</span> and <span class="yellow">B. thuringiensis</span> strains, many of which are from clinical sources. A second heterogeneous cluster includes <span class="yellow">B. cereus</span> and <span class="yellow">B. thuringiensis</span> isolates from various origins, while cold-tolerant <span class="yellow">B. weihenstephanensis</span> and <span class="yellow">B. cereus</span> isolates belong to the third group. The separate MLST analyses have also revealed that the <span class="yellow">B. cereus group</span> population is weakly clonal overall due to numerous clinical and virulent isolates emerging from different phylogenetic positions (5–8,11–14), with the exception of the ‘cold-tolerant’ cluster that seems to exhibit a panmictic (or sexual) population structure, i.e. with frequent genetic exchanges between strains (9).
Despite the overall congruence between the various MLST studies, the use of separate schemes with no gene overlap and very little strain overlap has produced a confusing situation and makes the results difficult to compare directly. Therefore, we recently proposed a combined scheme based on genes taken from three of the four schemes available by then and for which we created a web-based database accessible at the University of Oslo's MLST server, http://mlstoslo.uio.no/ (5). Here, in order to provide the <span class="yellow">B. cereus group</span> research community with a common MLST resource, we have developed on the same website a database, SuperCAT, that compiles and integrates MLST data from all the published <span class="yellow">B. cereus group</span> schemes. In addition, we applied supertree reconstruction methods to build an integrated view of the <span class="yellow">B. cereus group</span> population and phylogeny. Below we describe the content and main features of the new database as well as the process of supertree building.<br><br>DATABASE CONTENT AND IMPLEMENTATION
The SuperCAT database provides information, sequence and phylogenetic data for all bacterial isolates that have been typed using any of the five published MLST schemes for the <span class="yellow">B. cereus group</span> (Table 1). Strain information, when known, includes isolate description, source and geographical location of isolation, and the scheme(s) used for typing. The sequence data include the nucleotide sequences of the MLST loci examined in a given strain. SuperCAT also contains the phylogenetic supertree of the <span class="yellow">B. cereus group</span> reconstructed by combining the sequence data from all five schemes, as well as supertrees built for individual schemes. Information and sequences for isolates typed by the Priest and Tourasse–Helgason schemes were retrieved from the databases devoted to these schemes at http://pubmlst.org/bcereus and http://mlstoslo.uio.no/, respectively. MLST data for additional strains not available in the pubmlst.org repository (strains from (15) are missing therein) and for the Helgason, Ko, and Candelon–Sorokin schemes were taken from the published literature and the Genbank nucleotide sequence database (Table 1). In addition, sequences of all MLST loci were extracted from the complete genomes of the 21 sequenced <span class="yellow">B. cereus group</span> strains available in Genbank. Altogether, SuperCAT currently contains data for 1029 isolates and 26 gene fragments from 25 different genes. However, since most strains have been typed using only 6 or 7 of the 26 loci, about one-third of the complete set of sequences are included. The 26 loci, only available for the completely sequenced strains, sum up to 10 619 bp. All these genes are located on the chromosome, thus the database provides no information about extrachromosomal plasmids even though most of the strains do carry one or several small and/or large plasmids. Unlike scheme-specific MLST databases, SuperCAT does not contain allele and sequence type (ST) numbers. Since isolates in SuperCAT have been typed by different subsets of loci, complete allelic profiles are unavailable and therefore STs cannot be assigned for most strains, except the fully sequenced ones.
Table 1.The five MLST schemes designed for typing bacteria of the <span class="yellow">B. cereus group</span>SchemeGenesTotal sequence length (bp)Total number of isolateseUsed in (references)Helgasonadk, ccpA, ftsA, glpT, pyre, recF and sucC2 938120(6,12,46)Candelon–Sorokina,cclpC, dinB, gdpD, panC, purF and yhfL2 850149(9,10)Kob,cgyrB, mbl, mdh, mutS, pycA(1) and rpoB2 00265(7)Priesta,bglpF, gmk, ilvD, pta, purH, pycA(2) and tpi2 829721(8,11,13–15,46–48)Tourasse–Helgasona,b,dadk, ccpA, glpF, glpT, panC, pta and pycA(2)2 658172(5)aSpecific databases for the Priest and Tourasse–Helgason schemes are accessible at http://pubmlst.org/bcereus/ and http://mlstoslo.uio.no/, respectively. A BLAST database for the Candelon–Sorokin scheme is available at http://spock.jouy.inra.fr/cgi-bin/bacilliMLSopen.cgi.bWhile the Tourasse–Helgason and Priest schemes use the same gene fragment for the pycA gene, the Ko scheme is based on a different and non-overlapping gene region.cThe <span class="yellow">B. cereus group</span>-specific transcriptional regulator plcR was originally included in the Candelon–Sorokin and Ko schemes. However, plcR follows a phylogeny different from the other MLST loci (7,10) and is no longer used for MLST; therefore, it is not included in SuperCAT.dThe Tourasse–Helgason scheme is a combined scheme based on 3 genes from the Helgason scheme (adk, ccpA, and glpT), 3 genes from the Priest scheme (glpF, pta and pycA(2)), and the panC gene from the Candelon–Sorokin scheme.eIncluding strains with fully sequenced genomes.
SuperCAT is built as a relational database using the PostgreSQL management system, and data are accessible through a graphical web interface. User queries and results pages are processed and created on-the-fly via a highly modified version of the mlstdbNet software (16) written in PERL and based on the DataBase Interface (DBI) and Common Gateway Interface (CGI) modules. The database is implemented on a Linux Apache web server maintained through the facilities and support provided by the Norwegian EMBnet node. Some large supertree computations are run on a Linux supercomputer at the University of Oslo. The ATV (A Tree Viewer) Java applet is used for phylogenetic tree display (17). ATV notably supports horizontal and vertical zooming capabilities that are suitable for browsing large trees. The Jalview editor Java applet is also implemented in SuperCAT for advanced multiple sequence alignment display (18).<br><br>SUPERTREE RECONSTRUCTION
Supertree techniques allow to combine the phylogenetic information from different datasets into a common phylogenetic tree and several studies have shown that meaningful supertrees can be obtained even when taxon overlap is very sparse (see (19,20) for reviews). Supertree analysis has thus become increasingly popular for taking advantage and combining the massive amount of sequence data available in public databases for reconstructing large-scale organismal phylogenies with the ultimate goal of building the tree of life (21–24). In this study, the 21 <span class="yellow">B. cereus group</span> strains that have been completely sequenced, and for which the sequences at all 26 MLST loci are thus available, can be used to join all five schemes and provide the strain overlap necessary for supertree analysis. The global <span class="yellow">B. cereus group</span> supertree, containing 1029 isolates, was reconstructed according to the widely used matrix representation by parsimony (MRP) procedure (Figure 1; (19,25,26)). Scheme-specific supertrees were also reconstructed for each of the five MLST schemes by the same technique. Briefly, a phylogenetic tree is built for every gene separately by the maximum likelihood method with the PHYML_aLRT program (27). Then, each gene tree is recoded into a binary matrix representing the branching order (i.e. the phylogenetic groupings) following standard MRP coding using the SuperMRP.pl script (28). All gene tree matrices are concatenated into a supermatrix, in which isolates missing from a particular tree are coded using the ‘?’ character representing unknown data. In this supermatrix, the sequence of 0's, 1's and ?'s defines the branching profile of a strain. Closely related strains have similar branching profiles. Supertrees are then generated from the supermatrix by the maximum parsimony technique using the program MIX from the PHYLIP package (29) run with default parameters. The maximum parsimony step infers the trees that would require the minimum number of changes between the branching profiles of all isolates, where the unknown characters can take any of the two possible states 0 or 1 (they are not treated as missing gaps). As many trees were equally parsimonious, the final supertree was taken as the strict consensus of all parsimony trees with the CONSENSE program of PHYLIP. In order to obtain branch lengths that are proportional to the amount of nucleotide changes, we added an additional step in which branch lengths and statistical support for groupings are estimated from the concatenated sequences by the maximum likelihood method employing approximate likelihood-ratio tests (aLRTs) for branches using PHYML_aLRT with Shimodaira-Hasegawa-like support values (27,30). aLRTs provide a fast way of testing branch support without requirement of multiple replicates like traditional bootstrap procedures. The Felsenstein-1984 nucleotide substitution model supplemented with a gamma distribution (F84+Γ) was used in maximum likelihood computations for individual gene trees and the supertree (31). This model allows for unequal base frequencies, transition/transversion rate bias, and gamma-distributed substitution rate variation among sites. It was empirically chosen as a consensus from exploratory model testing using ModelTest (32,33), which indicated that models including these three factors were most appropriate for the MLST loci studied, although models for individual loci differed slightly. Note that the maximum likelihood technique also allows for uneven rates of nucleotide substitution between strains, which allows to accommodate slow- and fast-evolving isolates. To reduce the size of the binary supermatrix and speed up computations, individual gene trees and the supertree were built using only one representative from a set of strains having identical sequences. The remaining identical isolates were graphically added to the tree afterwards when drawing the final supertree.
Figure 1.Schematic overview of the <span class="yellow">B. cereus group</span> supertree reconstruction procedure using Matrix Representation by Parsimony (MRP). See text for details.
It should be noted that the global 1029-strain supertree retains the phylogenetic signals from the individual schemes and contains the three main clusters of the <span class="yellow">B. cereus group</span> population described in the section ‘Introduction’. The integrated SuperCAT system may also allow to infer new relationships between strains that were analyzed with different gene sets. Even though the 26 loci sequences are available for only 21 isolates, they apparently provide enough overlap information for building the main branches of the supertree. These 21 isolates cover all three clusters, although the majority of them are <span class="yellow">B. anthracis</span> strains or clinical strains closely related to <span class="yellow">B. anthracis</span> due to the focus of genome sequencing projects, making the part of the supertree containing these isolates likely to be more accurate than the rest of the tree. Furthermore, 111 other isolates have been typed by 10 genes or more, providing additional overlap (see the ‘Gene Distribution’ page). Although about two-thirds of the sequence data are missing overall, it has been shown for other organisms that relevant supertrees could be reconstructed with datasets containing more than 90% of missing data, especially when the characters that are present are informative (20,22,23,34). Empirical and simulation studies have indicated that this behavior may be due to the fact that the characters which are present are more important for the tree-building process than those which are absent (see (20,34) and references therein). Precise within-cluster groupings may contain more uncertainty, as indicated by the large number of unresolved multifurcations in the <span class="yellow">B. cereus group</span> supertree. Finally, it is also worth mentioning that the branching orders of the scheme-specific MRP supertrees are highly correlated to those of the published trees built with concatenated sequences and other phylogenetic algorithms.<br><br>DATA ACCESS AND MANIPULATION
The complete list of isolates included in SuperCAT (currently 1029) with strain description, source and country of origin is available at the database home page. By default all isolates in the database are used in the analysis tools provided, but the user can select strains of interest by keywords, MLST scheme, entering a list of strain identifiers, or choosing isolates individually via checkboxes. All subsequent analyses will be based on the selected strain subset and their loci. The keyword search will look for matches in any of the strain, description, source, location and scheme fields. Complex keyword queries with several logical operators can be formulated in the ‘advanced search’ page. Note that many isolates were referred by alternative names in different MLST schemes and publications, therefore synonyms have been included in the strain descriptions that allow a particular isolate to be looked up using any of its alternative identifiers. A sequence search is also possible using BLASTN (35), in order to select isolates that have allele sequences identical to user-entered query sequences.
Throughout SuperCAT, clicking on a strain name will pop up an isolate-specific window showing all relevant information and giving access to the nucleotide sequences of individual loci for that isolate. Detailed information about the MLST schemes (e.g. loci names and lengths, genomic coordinates, literature references) and their overlap, the distribution of available loci among the isolates, and the supertree reconstruction procedure can be obtained by clicking the relevant links in the header line present at the top of every page.
Apart from the basic functions for selecting and accessing strain information and sequence data for all five <span class="yellow">B. cereus group</span> MLST schemes, the main features of SuperCAT relate to the manipulation of the supertrees constructed by the MRP approach. The global supertree based on the combination of all five <span class="yellow">B. cereus group</span> MLST schemes as well as the five scheme-specific supertrees can be browsed according to various user-chosen criteria (Figure 2). Isolates in the supertrees can be colored by species or source of isolation. It is also possible to specifically mark in red the current subset of strains that has been selected by the user and to extract from the supertrees the subtree containing only those isolates. In the case of the multi-scheme supertree highlighting of the strains can be based on genetic distance. With this option the user can mark on and/or extract from the tree the isolates that are genetically closely related to strains of her/his choice. The user can either select strains that share one or several identical allele sequences with her/his query isolate(s) or that are at a specified genetic distance. Distances between isolates are computed by summing up the lengths of the branches (in average number of nucleotide substitutions per site) connecting the isolates in the supertree (known as patristic distances; (36)). The genetic relatedness search functions are also available in a separate page for the user to find closely related isolates without tree manipulation. SuperCAT allows to compare the scheme-specific MLST supertrees with each other and with the global supertree by using the subset of isolates that are common to all selected schemes. Common isolates can either be highlighted in red or be extracted as subtrees from each supertree, which can be used for comparing the positions of the common strains in the various MLST trees. For all supertree-related options, detailed tree navigation can be achieved using the various functions in the ATV tree window when the trees are displayed (17).
Figure 2.Examples of supertree browsing and manipulation in SuperCAT. A, supertree colored by species; B, specific highlighting of user-selected strains (in red); C, extracted subtree containing only the strains highlighted in B. Trees are displayed using ATV (17).
Besides the manipulation of the precomputed supertrees, SuperCAT offers the user the possibility to compute new supertrees by MRP using any combination of strains, schemes and genes. Supertree computations may be extremely time consuming, ranging from a few minutes to 2–3 days with the complete database. Users are therefore requested to enter their e-mail addresses and will receive a notification containing a link to the results page when the supertree is ready. Note that when building a supertree for a user-selected subset of strains, the computation will first include all database isolates. A subtree containing only the user-selected isolates will then be extracted from the supertree of all strains. Although more time consuming, this strategy allows: (i) to avoid sampling artefacts as phylogenies built with different isolate sets may vary and (ii) to obtain relationships even if the selected isolates have been typed using non-overlapping gene sets, as the supertree of all isolates can always be built owing to the completely sequenced strains that are common to all schemes.
Another main feature of the SuperCAT database is that the user can enter her/his own private sequences and conduct several sequence analyses (Figure 3). These analyses include: (a) building new supertrees containing user isolates and sequences; (b) finding database isolates having sequences most similar to the user's query sequences using an on-line BLASTN (35) service; and (c) aligning user sequences to database genes using the multiple sequence alignment program CLUSTALW (37). For the last option, the Jalview editor (18) is provided for advanced multiple alignment display. All user-entered data must be in FASTA format and can be either copied and pasted into the query forms or uploaded from text files stored locally on the user's computer.
Figure 3.Examples of query results in SuperCAT. A, multi-scheme BLAST search with sequence alignment; B, multi-scheme genetic search showing the list of isolates sharing one or more sequences with a query strain; C, multiple sequence alignment using Jalview (18).
All strain information data, sequences and phylogenetic trees, including user-made supertrees and extracted subtrees, can be saved and downloaded freely from the database. Users wishing to have their MLST data included as part of the SuperCAT release (and/or the Tourasse–Helgason scheme-specific database) are welcome to contact N.J.T. or A.-B.K. at the e-mail addresses given on the Oslo MLST server front page.<br><br>DISCUSSION AND FUTURE DEVELOPMENTS
SuperCAT is a newly created database devoted to the <span class="yellow">B. cereus group</span> of bacteria whose main objectives are to provide a common MLST repository and means for building a comprehensive genetic analysis of the group that has been typed by five separate schemes. The database is publicly and freely available at http://mlstoslo.uio.no/, along with the database specific for the combined scheme of (5). We plan to update SuperCAT quarterly.
Future developments of the database may deal with refining the supertree-building procedure. In particular, a new improved method has recently been developed for taking into account both nucleotide substitutions and recombination events in phylogenies, as part of the ClonalFrame software, which has been applied to the <span class="yellow">B. cereus group</span> and the Priest scheme (38). It would therefore be of interest to examine the suitability of ClonalFrame in the supertree context. It is also tempting to extend the supertree analysis beyond MLST data, by incorporating the phylogenies obtained previously from large-scale multilocus enzyme electrophoresis (MLEE; (3,39–41)) and amplified fragment length polymorphism (AFLP; (42–45)) studies. The MRP framework is ideal since it allows to integrate trees that can be built from different methods and data. As MLEE, AFLP and MLST have different levels of resolution, one can hope that combining them might provide an even more robust supertree for the <span class="yellow">B. cereus group</span>.<br><br>
<h3>pmcA2365205</h3>Cytotoxic Activity of Silyl- and Germyl-Substituted 4,4-Dioxo-3a,6a-Dihydrothieno[2,3−d]isoxazolines-2
Abstract
The [2+3] dipolar cycloaddition of nitrile oxides to the double C = C bonds of thiophene-1, 1-dioxides leads to formation of the fused isoxazolines-2 (1, 2). Tumor growth inhibition of these compounds strongly depends on the nature of group IV A element increasing from slightly active tert-butyl derivatives to silicon and germanium containing analogues. The products of benzonitrile oxide cycloaddition have greater cytotoxic effect than the compounds obtained from the cycloaddition reaction of 2, 5-disubstituted thiophene-1, 1-dioxides with acetonitrile oxide. Fused silyl substituted isoxazolines-2 are stronger NO-inducers than their germyl and tert-butyl analogues.
CYTOTOXIC ACTIVITY OF SILYL- AND GERMYL-SUBSTITUTED 4,4-DIOXO-3a,6a-DIHYDROTHIENO [2,3-d] ISOXAZOLINES-2 <br><br> E. Lukevics*, P. Arsenyan, I. Shestakova, O. Zharkova, I. Kanepe, R. Mezapuke, and O. Pudova Latvian Institute of Organic Synthesis, Aizkraukles 21, Riga, LV-1006, Latvia <br><br> ABSTRACT The [2+3] dipolar cycloaddition of nitrile oxides to the double (3=(3 bonds of thiophene-l,l-dioides leads to formation of the fused isoxazolines-2 (1, 2). Tumor growth inhibition of these compounds strongly depends on the nature of group IV A element increasing from slightly active ert-butyl derivatives to silicon and germanium containing analogues. The products of benzonitdle oxide cycloaddition have greater cytotoic effect than the compounds obtained from he cycloaddition reaction of 2,5-disubstituted thiophene-l,l-dioxides with acetonitrile oxide. Fused silyl substituted isoxazolines-2 are stronger NO-inducers than their germyl and tert-butyl analogues. <br><br> INTRODUCTION The interest in silyl substituted thiophene-l,l-dioxides stems from the fact that they are useful synthetic intermediates for the preparation of various types of organic compounds by Diels-Alder cycloaddition [1], amine induced ring-opening reaction [2], or coupling of bromothiophene-l,l-dioxides with thienyl stannanes in the presence of a palladium (0) catalyst [3]. It has been shown that unsubstituted thiophene-l,l-dioxide prepared in situ is a quite reactive dipolarophile in the [2+3] cycloaddition reactions with N,o-diphenylnitrone [4], benzonitrile [4, 5] and mesitonitrile [4, 5] oxides yielding mono- and diisoxazolines-2 and N-substituted isoxazolidines. Moreover, our recent studies indicate that silyl- and germylcontaining isoxazolines have gained a great deal of attention as compounds possessing a wide spectrum of the biological properties. The vasodilating, anticoagulant and cardioprotective activity of 5-Si-(Ge)substituted isoxazolines-2 has been studied in vitro and in vivo [6, 7]. The most active isoxazoline -3-(5"-triethylgermyl-3"-isoxazolino)pyridine hydrochloride protected the heart from rhythm disturbances and lethality during ischemiareperfusion [7]. It has been shown that silylisoxazolines-2 are more potent in protection against hypoxia and corazole convulsions than germanium analogues. However, germylisoxazolines-2 are stronger tumor growth inhibitors and NO-inducers than their silicon analogue [8]. This work presents the results of cytotoxic activity for fused isoxazolines-2 bearing a group 14 element as substituent (1, 2) in function of the nature of the group 14 element. <br><br> M <br><br> e3M.CH3 N <br><br> -0\ R <br><br> O\ M <br><br> e3M S <br><br> N <br><br> 02 <br><br> 02 2 <br><br> 1 <br><br> MATERIALS AND METHODS CHEMISTRY Seven tert-butyl-, trimethylsilyl-, and trimethylgermyl-substituted 4,4-dioxo-3a,6adihydrothieno[2,3-d]isoxazoline-2 1 and 2 (Table 1)were prepared by the [2+3] dipolar 63 <br><br> Vol. 7, No. 2, 2000 <br><br> Cytotoxic Activity ofSilyl-and Germyl-Substituted 4, 4-dioxo-3a ,6a-Dihydrothieno[2, 3-d]Isoxazolines-2 <br><br> cycloaddition of aceto- and benzonitrile oxides to 2,5-disubstituted thiophene-l,l-dioxides. Their synthesis and characterization are given in ref. [9]. <br><br> _,,/0\N Me3M M'Me3 <br><br> Me3M <br><br> 02 M, M'=C, Si, Ge; R=Me, Ph; R'=H, Me3Ge Table 1. Me3C, MeSi, Investigated dihydrothieno[2,3-d]isoxazolines-2 <br><br> 02 Me3Ge substituted <br><br> %, \R 4,4-dioxo-3a,6a<br><br> Compound <br><br> 'Type la <br><br> M <br><br> R H Me3Ge Me3Ge <br><br> Yield <br><br> 4,4-dioxo-3 methyl"5-tert-butyl-3a,6adihydrothieno[2,3-d]isoxazoline-2 <br><br> (3' C Si <br><br> (%) 80 <br><br> 4,4-dioxo-3-methyl-3a-trimethylgermyl-5-tert-butyl3a, 6a-d ihyd rothieno[2,3-d]isoxazoli ne-2 4,4-dioxo-3-methyl-3a-trimethyigermyl-5tri methylsilyl-3a, 6a-d hyd roth ien o[2,3d]isoxazoline-2 4,4-d io x o-3- meth y I-3a, 5- b s tri m ethy Ig e rmy I)3a, 6a-d hyd rothieno[2,3-d] isoxazo ne-2 4,4-d ioxo-3-ph en y I-5-tert-b uty I-3a, 6adihydrothieno[2,3-d]isoxazoline-2 4,4-d ioxo-3-p h e ny I-5-tri meth y si ly I-3a, 6adihydrothieno[2,3-d]isoxazoline-2 4,4-d ioxo-3-p h e n y I-5-tri methy Ig e rmy I-3a, 6adi,hydr0thieno[2,3, d]isoxazoline-2 <br><br> lb lc <br><br> 58 45 67 84 <br><br> ld 2a 2b <br><br> Ge <br><br> Me3Ge <br><br> C Si <br><br> 77 85 <br><br> 2c <br><br> Ge <br><br> IN VITRO CYTOTOXITY ASSAY Monolayer cells lines were cultivated for 72 h in DMEM standard medium without an indicator and antibiotics. After the ampoule was defreezed not more than four passa4ges were performed. The control cells and cells with tested substances in the range of 2-5 10 cell/mL concentration (depending on line nature)were placed on a separate 96 wells plates. Solutions containing test compounds were diluted and added in wells to give the final concentrations of 50, 25, 12.5, and 6.25 #g/mL Control cells were treated in the same manner only in the absence of test compounds. Plates were cultivated for 72 h. A quantity of survived cells was determined using crystal violet (CV) or 3-(4,5-dimethylthiazol-2-yl)-2,5diphenyltetrazolinium bromide (MTT) coloration which was assayed by multiscan spectrofotometer. The quantity of alive cells on control plate was taken in calculations for 100% [10, 11]. Concentration of NO was determined according to [10]. RESULTS AND DISCUSSION Potential cytotoxic activity of synthesized fused isoxazolines 1 and 2 was tested in vitro on four monolayer tumor cell lines: MG-22A (<span class="yellow">mouse</span> hepatoma), HT-1080 (<span class="blue">human</span> fibroblastoma), B16 (<span class="yellow">mouse</span> melanoma), Neuro 2A (<span class="yellow">mouse</span> neiroblastoma). Concentrations providing 50% of tumor death effect were determined according to the known procedure [12] using 96 well plates. The experimental evaluation of cytotoxicity properties is presented in Table 2. A preliminary analysis of the structure-activity relationship for the cytotoxic action clearly indicates the strong influence of the MeM (M=C, Si, Ge) group in position 5 of fused isoxazolines 1 and 2. Derivatives bearing tert-butyl substituent (la,b and 2a) have a slight cytotoxic effect (> 10 #g/mL). The substitution of the tert-butyl group by trimethylsilyl or 64 <br><br> E. Lukevics et al. <br><br> Metal-Based Drugs <br><br> A <br><br> oO <br><br> {D <br><br> 0 <br><br> 0 <br><br> 65 <br><br> Vol. 7, No. 2, 2000 <br><br> Cytotoxic Activity ofSilyl-and Germyl-Substituted 4, 4-dioxo-3a ,6a-Dihydrothieno[2, 3-d]Isoxazolines-2 <br><br> trimethylgermyl ones leads to considerable increase of cytotoxicity. It must be noted that the activity of silicon- and germanium-containing compounds (1 and l d) depends on the tumor type. 5-Trimethylsilyl-substituted fused isoxazoline le is more active than the germanium analogue l d in tests on HP-1080 and MG-22A cell lines. However, the germanium compound ld has greater cytotoxic effect on Neuro 2A and B16 cell lines than the silicon derivative lc. Comparison of the tumor growth inhibition for derivatives 1 and 2 shows a higher activity of the condensed isoxazolines 2 containing a phenyl group in position 3 with respect to 4,4-dioxo-3-methyl-3a-trimethylgermyl-5-MeM-3a,6a-dihyd rothieno[2,3d]isoxazolines l b-d. Silyl- and germyl-substituted fused isoxazolines have a medium NOinduction ability, 4,4-dioxo-3-phenyl-5-trimethylsilyl-3a,6a-dihydrothieno[2,3-d]isoxazoline-2 (2b) being the most active (250% in the MG-22A test). <br><br> ACKNOWLEGMENT We are grateful to Latvian Taiho Foundation for financial support. REFERENCES 1. A. R. M. Donovan, M. K. Shepherd, Tetrahedron Lett., 35 (1994)4425. 2. S. Gronowitz, A.-B. H0rnfeldt, E. Lukevics, O. Pudova, Synthesis, (1994)40. 3. G. Barbarella, L. Favaretto, G. Sotgiu, M. Zambianchi, L. Antolini, O. Pudova, A. Bongini, J. Org. Chem., 63 (1998) 5497. 4. A. Bened, R. Durand, D. Pioch, P. Geneste, J. P. Declerq, G. Germain, J. Rambaud, R. Roques, J. Org. Chem., 46 (1981) 3502. 5. F. M. Albini, P. Ceva, A. Mascherpa, E. Albini, P. Caramella, Tetrahedron, 38 (1982) <br><br> 6. 7. 8. <br><br> 9. 10. 11. 12. <br><br> 3629. E. Lukevics, M. Veveris, V. Dirnens, Appl. Organomet. Chem., 11 (1997) 805. E. Lukevics, P. Arsenyan, M. Veveris, Metal Based Drugs, 5 (1998) 251. E. Lukevics, P. Arsenyan, S. Germane, I. Shestakova, Applied Organomet. Chem., 13 (1999) 795. E. Lukevics, P. Arsenyan, S. Belyakov, J. Popelis, O. Pudova, Organometallics, 18 (1999) 3187. D.J. Fast, R.C. Lynch, R.W. Leu, J. Leuckocyt. Biol., 52 (1992) 255. P.J. Freshney, Culture of Animal Cells (A Manual of Basic Technique), Wiley-Liss, New York, 1994, pp. 296-297. R. J. Riddell, R. H. Clothier, M. Fd. Balls, Chem. Toxicol., 24 (1986) 469. <br><br> Received: January 21, 2000 Accepted: February 1, 2000 Received in revised camera-ready format" February 2, 2000 <br><br> 66 <br><br>  <h3>pmcA2365769</h3>IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome
Abstract
This study was undertaken to evaluate the role of IL-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum IL-5 levels that might be essential for eosinophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-IL-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of IL-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce IL-5.
Research Paper Mediators of Inflammation 5, 24-31 (1996) <br><br> Tins study was undertaken to evaluate the role of 11-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum 11-5 levels that might be essential for eosi<br><br> IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome L. H. Faccioli, 1"cA V. F. Mokwa, C. L. Silva, G. M. Rocha, J. I. Araujo, M. A. Nahori 2 and B. B. Vargaftig 2 <br><br> nophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-11-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of 11-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing 11-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce 11-5. <br><br> 1Department of Parasitology, Microbiology and Immunology, School of Medicine of Ribeir,5o Preto, 14049-900, Ribeiro Preto, SP, Brazil. Fax: (+55) 16 633 6631 2Unit de Pharmacologie Cellulaire, Unit Associe Institut Pasteur/INSERM n. 285, Paris, <br><br> France. <br><br> CACorresponding Author <br><br> Key words: Eosinophil, Eosinophilia by <span class="yellow">Toxocara canis</span>, IL-5 in eosinophilia, <span class="yellow">Toxocara canis</span> <br><br> Introduction <br><br> treatment in <span class="yellow">mice</span> infected with <br><br> Eosinophilia has been associated with parasitic diseases, particularly when the parasites invade the tissues or injure the mucosal surfaces. <span class="yellow">Toxocara canis</span> is an intestinal parasite of <span class="yellow">dogs</span>, and is the most common aetiologic agent of visceral larva migrans syndrome (VLMS). In <span class="yellow">humans</span>, VLMS results from the ingestion of embryonated eggs of <span class="yellow">T. canis</span>, that eclode in the small intesfine. The infective larvae invade the mucosa, move into the liver via the portal circulation, and from there to the lungs. 2 Beaver et al., 3 who were the first to describe this syndrome, noted the intense eosinophilia which reaches more than 90% of total leucocyte counts. However, there are few studies regarding the mechanisms involved in the blood and tissue eosinophilia obseeeed in VLMS. Several investigators have suggested a direct correlation between eosinophilia and interleukin5 (IL-5) in <span class="blue">human</span> helminth infections4'5 and in experimental animal models. 67 Inhibition of eosinophilia has been demonstrated by anti-IL-5 24 Mediators of Inflammation Vol 5 <br><br> brasiliensis, 8 $chistosoma mansoni, <span class="yellow">Toxocara canis</span>7 and <span class="yellow">Heligmosomoidespolygyrus</span>. 1 IL-5 has also been shown to support the terminal differentiation, proliferation of eosinophil precursors 11'12 and eosinophil activation. 13 Although IL-5 does not demonstrate eosinophil chemotactic activity in vivo14 there is some evidence suggesting that this cytokine may modulate a selective eosinophil accumulation at the site of inflammation. Moreover, Sehmi et a/.15 reported that IL-5 has a selective priming effect on eosinophil migratory response to nonselective chemoattractant mediators in vitro. Also, Moser et a/. 16 have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with IL-5, IL-3 and GM-CSF. Thus, the involvement of IL-5 in eosinophilia is not fully understood. In the present study we have used a <span class="yellow">guinea-pig</span> model of VLMS to investigate the involvement of IL-5 in eosinophil migration and in the maintenance of eosinophilia in blood, bone marrow, lung and peritoneal cavity. (C) 1996 Rapid Science Publishers <br><br> Nipostrongylus <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> Materials and Methods Animals: Outbred albino weighing 300-400g at the ments were obtained from the School of Medicine of versity of So Paulo, Brazil. <br><br> female <span class="yellow">guinea-pigs</span> start of the experi<br><br> were routinely processed, embedded in paraffin, sectioned at 4-61.tm, and stained with Chromothope 2R and haematoxylin, for examination by light microscopy. Determination oflL-5 in serum: The IL-5 level in the serum of <span class="yellow">guinea-pigs</span> was measured using an enzyme-linked immunosorbent assay (ELISA). Briefly, ELISA plates (96-well Immunoplate MaxiSorp, Nunc, Roskilde, Denmark) were coated with IL-5-specific monoclonal antibody (TRFK-5, 5 lag/ml in phosphate buffered saline, pH 7.4, PBS, 100 l.tl/well). After 2 h of incubation at 37C, the wells were washed four times with PBS containing 0.1% Tween 20 (PBS-T). Then, 100 btl of samples or recombinant <span class="yellow">murine</span> IL-5 standards (0.15-200ng/ml) in PBS-T and 2% BSA (PBSTBSA) were added to each well. After incubation for l h at 37C, the wells were washed three times and 100 l.tl of biotinylated <span class="yellow">rat</span> anti-<span class="yellow">murine</span> IL-5 (biotinylated-TRFK-5, 1 lag/ml in PBS-TBSA) was added. After incubation for 1 h at 37C, the wells were washed again three times and 100 of peroxidase-labelled streptavidin (1/1000, Kirkegaard & Perry Laboratories Inc., Maryland, USA) were added to each well. Following incubation for l h at 37C and further washing, the enzyme was developed using the TMB substrate peroxidase for 5 min. The reaction was stopped by adding 501.tl of 2.0 N HCl, and the optical densities were read at 490nm using an automated plate reader. The sensitivity of the assay was 0.15 ng/ml and the upper limit 100ng/ml. <br><br> the animal house of Ribeiro Preto, Uni<br><br> Infection of animals: <span class="yellow">T. canis</span> eggs were obtained by the methods of Olson and Schulz, 17 with minor modifications. Briefly, gravid female <span class="yellow">worms</span> were recovered from <span class="yellow">dogs</span>, and the eggs were rescued from the uterus, washed and allowed to develop to the infective stage in shallow dishes containing 0.5% formalin at 37C. Under light ether anaesthesia, the animals were infected with I ml saline containing 500 <span class="yellow">T. canis</span> eggs, by gastric intubation using a metal cannula. Blood cell counts: <span class="yellow">Guinea-pigs</span> were anaesthetized with sodium pentobarbitone (30mg/kg, i.v.) and blood samples were collected by cardiac puncture with 10% EDTA. Total cell counts were carried out using diluting fluid in a Neubauer chamber. Differential countings were obtained using Rosenfeld-stained cytocentrifuge preparations, <br><br> Bronchoalveolar lavage fluid. The <span class="yellow">guinea-pigs</span> were killed by an overdose of sodium pentobarbitone and 5ml of phosphate-buffered saline (PBS) containing 0.5% sodium citrate (PBS/SC), at room temperature, were instilled through a polyethylene cannula introduced into the trachea. The cells present in the bronchoalveolar lavage fluid (BALF) were recovered immediately. The procedure was repeated once. The leucocyte counts in the BAI_ were determined as described above, Peritoneal cells: The cells from the peritoneal cavities were harvested by injection of 10ml of PBS/SC into the peritoneum. Only 5-8 ml of the <br><br> fluid was withdrawn for cell counts, as described above, <br><br> Monoclonal antibodies: The <span class="yellow">rat</span> monoclonal antibody TRFK-5 was a generous gift from Dr P. Minoprio, Institut Pasteur, Paris. The neutralizing antibody was purified by precipitation with ammonium sulfate (45%) from ascites prepared in CD1 <span class="yellow">nude mice</span> (Charles River, St Aubin les Elbeuf, France) inoculated 1 week before the injection of hybridoma cells, with I ml of pristane (Sigma). After precipitation and dialysis of the ascite fluid overnight against PBS, the dialysate was further purified on a Protein G1 column (HiTrapTM, Pharmacia Upsala, Sweden). <br><br> Bone marrow cells: Bone marrow cells were collected by flushing the contents of the <span class="yellow">guinea-pig</span> femur with 10 ml of PBS/SC. Total cell numbers were determined as above. In the differential cell counts the cell populations were divided into mature neutrophils, mature eosinophils and others (mainly precursors and mononuclear <br><br> cells), Histopathological studies: Tissues were removed from <span class="yellow">guinea-pigs</span> at various times post-infection and immediately fixed in 10% formalin. Tissues <br><br> Eosinophil and cytokine depletion: <span class="yellow">Guinea-pigs</span> were injected i.p. with TRFK-5 or with the irrelevant antibody (<span class="yellow">rat</span> IgG against total anti-<span class="blue">human</span> IgG) once, 2mg/animal, at the time of infection or at different intervals (1, 3, 12 or 17 days) thereafter. The animals in this group were sacririced 18 days after infection. liver: One lobule of was used to determine the larval each liver counts from infected <span class="yellow">guinea-pigs</span>. Larval recovery <br><br> Recovery <br><br> of larvae from <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 25 <br><br> L. H. Faccioli et al. <br><br> was evaluated as described by Kayes and Oaks, 18 with minor modifications. Briefly, the tissue was chopped and digested with pepsin-HC1 (pH 1.5<br><br> 1.8) for 2h at 37C. Larval counts for each sample were performed after centrifugation and examination of three 100-l.tl samples under the light microscope. Statistical analysis: Data are presented as the mean _+ S.E.M. and were analysed statistically using the Mann-Whitney test for unpaired data. A p < 0.05 value was considered to be statistically <br><br> significant. <br><br> Results <br><br> of eosinophil counts in blood bone marrow, BALF and peritoneum: <span class="yellow">Guinea-pigs</span> infected with <span class="yellow">T. canis</span> eggs showed a timedependent blood, bone-marrow, BAUV and peritoneal eosinophilia (Fig. 1). The results represent the mean of nine animals obtained in three different experiments. The eosinophil number Kinetics 16<br><br> increased significantly from 0.55 +_ 0.37 x 105 at the beginning of experiment to 6.0 +__ 1.03 x 105 at 6 days post-infection, peaked at day 18 (12.0 _+ 2.31 x 105), and decreased by day 24 (8.11 2.85 x 105) (Fig. 1A). A rise in the percentage of mature eosinophils in bone marrow" was observed 12 days after infection (ranging from 6 2% to 14 2%) and peaked at 18 days (17 2%) (Fig. 1B). As in blood, the number of eosinophils in BALF increased significantly from 0.14 0.06 x 105 to 1.37 _+ 0.35 x 105 at 6 days after infection, reaching a peak at 18 days (10.23 2.62 x 105) with an increase in relative number of as much as 90% in eosinophil counts in relation to controls, and was still elevated at day 24 (9.07___ 3.47 x 10 s) (Fig. 1C). The remaining cells in the BALF were alveolar macrophages, lymphocytes, mast cells and ciliated cells. In contrast to blood and BALF, the number of eosinophils in the peritoneal cavity increased significantl only at day 12 post-infection (onset, 2.06 1.04 x 105; day 6, 3.68 _+ 0.82 x 105; day 12, 5.77 +_ 1.12 x 10>; and increased progres<br><br> __ _ _ _. (B) <br><br> (AI Blood <br><br> 24 20" <br><br> Bone Marrow <br><br> 1612 8 <br><br> 4 O0<br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 111 <br><br> 21 <br><br> 24 <br><br> 16<br><br> (c) BALF <br><br> 16<br><br> ID) Peritoneum <br><br> 12<br><br> __o <br><br> 12<br><br> 8 <br><br> 4 <br><br> 0 3 <br><br> O6 9 <br><br> 12 <br><br> 15 <br><br> 16 <br><br> 21 <br><br> 24 <br><br> Days postinfoction <br><br> FIG. 1. Number of eosinophils in blood, BALF and peritoneal cavity, and percentage, of eosinophils in bone marrow of <span class="yellow">T. canis</span>qnfected <span class="yellow">guinea-pigs</span>. Values are the mean -t-S.E.M. (n=8 to 9). Asterisks indicate a significant difference between infected and noninfected animals (n 5 6). *p < 0.05 and **p < 0.01. 26 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> sively until day 24, 12.44 4- 2.72 x 105) (Fig. 1D). The percentage of eosinophils in some animals reached 55% at the peak of infection. No increase in the number of mononuclear cells was seen in any compartment analysed. Larval counts: The percentage of inoculated <span class="yellow">T. canis</span> larvae recovered by peptic digestion of the liver of experimental animals 4 h and 1, 2, 3, 4, 9, 12 and 18 days after inoculation of 500 eggs per animal is shown in Fig. 2. Most of the larvae were recovered 2 to 4 days after infection and 10% recovery was also observed on day 18 in the liver of the animals. <br><br> 150 <br><br> 120 <br><br> IL-5 level in serum of infected animals: IL-5 was measured in the serum of infected and normal <span class="yellow">guinea-pigs</span>. Each time point in Fig. 3 represents the mean of results from three to five infected animals, and from six controls. Two peaks of IL5 were present in the serum of infected <span class="yellow">guineapigs</span> 1 day after infection (102 __+ 22 pg/ml), and 18 days later (59 7 pg/ml). The level of IL-5 in the controls was 31 4- 4 pg/ml. Eosinophil numbers in infected animals treated with TRFK-5: When <span class="yellow">guinea-pigs</span> received an i.p. injection of TRFK-5, the monoclonal antibody against IL-5, at the time of egg administration or 1 day later, the number of eosinophils in blood, BALF, peritoneal cavity and bone marrow was <br><br> _ <br><br> 30<br><br> 0 <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 12 <br><br> 18 <br><br> Days postinfection <br><br> FIG. 3. IL-5 concentration in serum of <span class="yellow">T. cani</span>infected <span class="yellow">guineapigs</span> (n 3-5). Basal IL-5 concentrations were of 31 4 pg/ml (n= 12). <br><br> _ <br><br> 24 <br><br> 60<br><br> 50<br><br> 40<br><br> 0 <br><br> "T" <br><br> 1" <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 18 <br><br> 21 <br><br> 24 <br><br> Days postinfection FIG. 2. Percentage of <span class="yellow">T. canis</span> larvae recovered from liver of <span class="yellow">guinea-pigs</span> studied at various times post-infection. Data obtained from five animals. <br><br> drastically reduced, even when determined 18 days after infection (Table 1). No inhibition of eosinophil counts was observed when the animals were inoculated with the irrelevant antibody at the time of infection (Table 1). Fig. 4 shows the comparative results of eosinophilia obtained when the antibody was given 3 days or 17 days after egg inoculation. The antibody given at 3 days after infection induced a high percentage of inhibition in eosinophil counts in all the compartments analysed 18 days after infection (Fig. 4A). However, when TRFK-5 was administered to the infected animals on day 17 post-infection (thus 1 day before sacrifice), a significant inhibition in number and percentage of eosinophils was observed only in the blood (p=0.030) (Fig. 4B). A small non-significant decrease was seen in BALF (p 0.790) and peritoneum (p= 0.222). Moreover, the number of mature eosinophils in bone marrow increased by 140% (p 0.038). As demonstrated in Fig. 4B, the behaviour of eosinotShilia in BALF was completely different from that observed in blood. Thus, to better understand the eosinophilia in the lungs of infected animals, we monitored eosinophil numbers in BALF after administration of TRFK-5 at the same time, or 1, 3, 12 or 17 days after infection. The animals were sacrificed 18 days after infection. In another group, TRFK-5 was administered 18 days post-infection and the animals were sacrificed 6 days later. When the mAb was administered at the same time or 1 or 3 days postinfection there was a significant inhibition in the number of eosinophils (Fig. 5). These data show Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 27 <br><br> L. H. Faccioli et al. Table 1. Eosinophils in T. caniinfected <span class="yellow">guinea-pigs</span> treated or untreated with TRFK-5 Compartment <br><br> Time of sacrifice <br><br> (days) <br><br> Non18 24 18 24 18 24 18 24 <br><br> treated <br><br> Blood <br><br> 12.18 5.28 10.08 -t- 2.85 16.50 4.42 6.57 +__ 1.68 <br><br> BALF Peritoneal cavity <br><br> 12.31 -I- 2.35 12.44 2.33 9 6 <br><br> Bone marrow <br><br> _ _ _ _ _ ___ _ __ __ _ __ Days of treatment with TRFK-5 after egg administration <br><br> Irrelevant Ab at the time of infection <br><br> 0 <br><br> 3 <br><br> 12 <br><br> 17 <br><br> 18 <br><br> (n 6) <br><br> (n=4) <br><br> (n=4) <br><br> (n 5) <br><br> (n 5) <br><br> (n=4) <br><br> 0.14 -t- 0.14" <br><br> 0.26 0.79 <br><br> 0.26* 0.24* <br><br> 0.17 _-t- 0.17" <br><br> 0.15 <br><br> 0.15" <br><br> 2.59 <br><br> 0.80* 4.36 2.21 <br><br> 13.32 <br><br> 4.54 <br><br> O. 15 <br><br> 0.15* <br><br> 0.36 <br><br> 0.10" <br><br> 1.22 -t- 0.50* <br><br> 7.07 -t- 2.39 <br><br> 15.24 <br><br> 29.31 _+ 15.90 16.09 <br><br> 10.33 _+ 4.10 3.79 <br><br> 1.47 -t- 0.76* 3 <br><br> 0.48 <br><br> _+ 0.35* <br><br> 0.46 2.6 <br><br> 0.27* 1.3" <br><br> 2.89 <br><br> 0.94* <br><br> 10.42 <br><br> 4.00 <br><br> 1.58" <br><br> -+ <br><br> 1" <br><br> 1.25 -I- 0.25* <br><br> + <br><br> 5 <br><br> 15 -t- 2 <br><br> 9_+1 <br><br> 2 <br><br> 0.7* <br><br> In blood BALF and peritoneal cavity the values represent mean -t-_ S.E.M. x 10 eosinophils. *p < 0.05. <br><br> ml-1 <br><br> and in bone marrow mean -t-_ S.E.M. of the percentage of mature <br><br> that the inhibition of the first peak of IL-5 which appeared at 1 to 3 days after infection as shown in Fig. 3, is also very important for the establishment of eosinophilia in the lungs. However, 24- (A) <br><br> when the mAb was administered 12, 17 or 18 days after infection there was no significant inhibition in the numbers of eosinophils in BALF (Fig. 5), showing that once established, the eosi18<br><br> 15x <br><br> 16<br><br> 12<br><br> ._ . 0 <br><br> 12 <br><br> o <br><br> BIo o d <br><br> BALF <br><br> P e rit oneum <br><br> Bone marrow <br><br> 24- (Is) <br><br> 1815<br><br> E x <br><br> 20<br><br> 16<br><br> o <br><br> 12///. <br><br> o = o <br><br> 12<br><br> 0 Blood <br><br> O" <br><br> BALF <br><br> Peritoneum <br><br> Bone marrow FIG. 4. Number of eosinophils in blood, BALF and peritoneum and percentage of eosinophils in bone marrow of <span class="yellow">T. cani</span>infected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) 2 mg/animal at 3 days post-infection; (B) 2 mg/animal 17 days post-infection. The treated and control animals were sacrificed 18 days after infection. Asterisks indicate a significant difference from infected controls (n=5-6) and from animals treated with TRFK-5 (n=4- 5). *p < 0.05 and **p < 0.01. 28 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> 24<br><br> E <br><br> 20<br><br> heart; data not shown) and muscle, as reported by other investigators, 8 were infiltrated. The factors responsible for in vivo eosinophil accumulation at inflammatory sites have been <br><br> 0 <br><br> TRFK-5 i.p. <br><br> 0 <br><br> 3 <br><br> 12 17 <br><br> A <br><br> B <br><br> FIG. 5. Number of eosinophils in BALF of T. caniinfected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) animals were sacrificed 18 days post-infection and (B) 24 days after infection. Asterisks indicate a significant difference from infected control and from animals treated with TRFK-5 (p< 0.01). <br><br> nophilia persists in lungs, probably by the secretion of IL-5 from cells localized in the lung microenvironment, <br><br> Histopathological analysis: The treatment of <span class="yellow">T. canis</span>-infected animals with irrelevant antibody showed a widespread eosinophilic infiltration as in untreated animals (Fig. 6A, B). However, the treatment of animals with TRFK-5 at the same time of infection, or 1 day or 3 days later ted to a complete inhibition of eosinophil infiltration in the lung parenchyma (Fig. 6C). By contrast, the mononuclear cell infiltration in the lungs was not modified. When the infected <span class="yellow">guinea-pigs</span> received TRFK-5 1 day before sacrifice (or 17 days post-infection), eosinophil infiltration in the lung parenchyma was also inhibited (Fig. 6D) but not to the same extent as observed in the group receiving TRFK-5 given at the time of infection or 3 days later. Thus, the histological determination of eosinophil infiltration in these lungs corroborates a reduction but not a sizeable inhibition of eosinophil numbers as observed in the BALF of the same infected animals, <br><br> poorly defined, although T lymphocytes and mast cells appear to be involved in eosinophilia. 9'2 IL-5, a T cell-derived factor that regulates B cell functions, is an eosinophil differentiation factor11 as well as a stimulating and survival-prolonging factor specific for eosinophils in vitro. 2 Also, several investigators have demonstrated that sTstemic eosinophilia in <span class="yellow">mice</span> infected with parasites is mediated by IL-5 produced in response to the infection. 2'22 In the present study, the i.p. administration of the TRFK-5 antibody markedly inhibited the widespread eosinophilia observed in <span class="yellow">T. canis</span>-infected <span class="yellow">guinea-pigs</span>, indicating that IL5 participated in a <span class="yellow">guinea-pig</span> model of VLMS eosinophJlia. Most of the <span class="yellow">T. canis</span> larvae which penetrated the intestinal wall had migrated into the liver within 72h after inoculation as demonstrated here and elsewhere. 2 It is apparently during this interval that the worm provides the signals to cytokine-producing cells, which in turn trigger increased serum levels of specific cytokine as demonstrated here for IL-5, 24 to 72 h after infection. The signals may be provided directly by the invading parasite or by cells in response to the parasite. The cytokine pattern that develops at this early stage, probably induced by a T-cell independent pathway, may also influence the pattern of T cell differentiation into a Th2 type, which may be responsible for the second peak of IL-5 observed in our experimental model (Fig. 3), although a second cycle of larval invasion (Fig. 2) with a rapid peak of IL-5 liberation cannot be ruled out. <br><br> Discussion The results of the present study show that in our experimental model widespread eosinophilia follows the infection of <span class="yellow">guinea-pigs</span> with second stage eggs from <span class="yellow">T. canis</span>, as also noted in <span class="yellow">humans</span> and in other experimental animals. 7'7 <span class="yellow">T. canis</span> is a potent stimulus for systemic eosinophilia, since blood, BALF, peritoneum and all tissues examined (kidney, eyes, spleen, thymus, <br><br> Thus, our results suggest that the eosinophilia against helminth larvae may be initiated by the release of IL-5 when the parasites migrate from the intestine to the liver by stimulation of specific cell populations. Then, an early release of IL-5 quickly induces eosinophil recruitment, probably first from the stored mature eosinophil pool from vascular endothelium or by the mobilization of eosinophils from extravascular sites to the blood. This fact could explain why we found increased eosinophils first in blood and later in other compartments. The early IL-5 release may also serve as a signal for eosinophil differentiation and maturation in bone marrow. The time <br><br> inteeeal observed between the first peak of IL-5 release and the increase of eosinophils in blood coincides with that reported to be necessary for eosinophil differentiation and maturation in vitro. 12Increased eosinophil production and liberation into blood and other tissues occurs Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 29 <br><br> L. H. Faccioli et al. <br><br> FIG. 6. (A) photomicrographs of lung parenchyma from <span class="yellow">guinea-pigs</span> infected for 18 days with <span class="yellow">T. cani</span>,, (B) infected animals which were treated with irrelevant antibody at the time of infection; (C) infected animals which were treated with the mAb TRFK-5 at 3 days after infection; (D) mAb administration 17 days after infection. The animals were sacrificed 18 days after infection. Note the intense eosinophil infiltration into the lung in A and B, the inhibition of eosinophils in C and the reduction of eosinophils in D. <br><br> thereafter. Thus, early and later IL-5 release provides a necessary level of this cytokine, which is involved in the maintenance of eosinophilia. We may assume that the inhibition of the first peak of IL-5 release by TRFK-5 does not permit the subsequent T cell stimulation and differentiation. This may explain the long-lasting effect of TRFK5 treatment observed here and also reported by others. 8 In agreement with our results, there is an important observation of Svetic et al. 24 showing that a specific and highly reproducible IL-5 gene expression pattern is detectable in Peyer's patches by 6 to 12h after <span class="yellow">Heligmosomoides polygyrus</span> infection. The early increase in IL-5 gene expression after infection was probably T cell-independent, inasmuch as it was obseeeed in Peyer's patches of congenitally athymic <span class="yellow">mice</span> and of conventional <span class="yellow">mice</span> treated with anti-CD4 30 Mediators of Inflammation Vol 5 <br><br> and anti-CD8 mAb. Moreover, Kusama eta/. 25 have observed two peaks of eosinophilia in normal and athymic <span class="yellow">mice</span>, and suggested that IL-5 observed in the first peak was produced by cells other than CD4 T cells, since anti-CD4 and anti-CD3 mAb reduced only the second peak of eosinophilia in normal <span class="yellow">mice</span> and slightly reduced the first peak of eosinophilia in both normal and nu/nu <span class="yellow">mice</span>. The local lung cells producing IL-5 may also help us to explain the reason why 12, 17 or 18 days post-infection TRFK-5 treatment only partially inhibits, or does not inhibit eosinophil infiltration into the lungs, as demonstrated in Figs 5 and 6. We may suggest that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK<br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> 5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens have already stimulated some cell populations to produce IL-5, as demonstrated by Kusama et aL25These results suggest that eosinophilia in lungs is under the control of different factors when compared to that observed in blood and the peritoneal cavity. One of the most important results obtained here was the inhibition of circulating eosinophil numbers by the different mAb treatments, even when the antibody was given at the peak of blood eosinophilia, which was accompanied by an increase of mature eosinophils in bone marrow. This suggests that IL-5, apart from being required for the terminal differentiation of eosinophils in bone marrow, 26 is also likely to drive eosinophils from the bone marrow to the blood and then to the tissues, probably by upregulating VLA-4 expression in eosinophils. Moser et aL have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with cytokines such as IL-5, IL-3 or GM-CSF for expression of adhesion molecules such as VI-4. Recently, Pretolani et al. 27 have indeed shown that an anti-VLA-4 antibody suppresses eosinophil recruitment to lung in the <span class="yellow">guinea-pig</span> and, as a consequence, inhibits the accompanying bronchopulmonary hyperresponsiveness. <br><br> tant in protective immunity to a gastrointestinal nematode infection in <span class="yellow">mice</span>. Proc Natl Acad Sci USA 1991; 88= 5513-5517. 11. Yamaguchi A, Suda T, Suda J, et al. Purified interleukin (IL-5) supports the terminal differentiation and proliferation of <span class="yellow">murine</span> eosinophilic <br><br> precursors. J Exp Med 1988; 16'7: 43-56. 12. Yamaguchi Y, Hayashi Y, Sugama Y, et al. Highly purified <span class="yellow">murine</span> interleukin (IL-5) stimulates eosinophil function and prolongs in vitro survival. IL-5 as an eosinophil chemotactic factor. J Exp Med 1988; 16'7: 17371752. 13. Rothenberg ME, Petersen J, Stevens RL, Silberstein DS, McKenzie DT, Austen KF, Owen WF. IL-5-dependent conversion of normodense <span class="blue">human</span> eosinophils to the hypodense phenotype uses 3T3 fibroblasts for enhanced viability, accelerated hypodensity, and sustained antibodydependent cytotoxicity. J Immuno11989; 143; 2311-2316. 14. Collins PD, Weg VB, Faccioli LH, Watson ML, Moqbel R, Williams TJ. Eosinophil accumulation induced by <span class="blue">human</span> interleukin-8 in the <span class="yellow">guinea pig</span> in vivo. Immunology 1993; '7}; 312-318. 15. Sehmi R, Wardlavo AJ, Cromwell O, Kurihara K, Waltmann P, Kay AB. Interleukin-5 selectively enhances the chemotactic response of eosinophils obtained from normal but not eosinophilic subjects. Blood 1992; '79; 2952-2959. 16. Moser R, Fehr J, Bruijnzeel PLB. IL-4 controls the selective endotheliumdriven transmigration of eosinophils from allergic individuals. J Immunol 1992; 149: 1432-1438. 17. Olson LJ, Schulz CW. Nematode induced hypersensitivity reactions in <span class="yellow">guinea pigs</span>: onset of eosinophilia and positive Schultz-Dale reactions following graded infection with <span class="yellow">Toxocara canis</span>. Ann N Y Acad Sci 1963; 113; 440-455. 18. Kayes SG, Oaks <br><br> JA. Development <br><br> of the granulomatous response in <br><br> <span class="yellow">murine</span> toxocariasis. I. Initial events. Am J Patho11978; ,}3; <br><br> 277-294. <br><br> 19. Basten A, Beeson PB. Mechanisms of eosinophilia. II. Role of the lymphocyte. J Exp Med 1970; 131; 1288-1305. 20. Plaut M, Pierce JH, Watson CJ, Hanley-Hyde J, Nordan RP, Paul WE. Mast cell lines produce lymphokines in response to cross-linkage of Fc epsilon RI or to calcium ionophoras. Nature 1989; 339: 64-67. is 21. Sher A, Coffman RL, Hieny S, Scott P, Cheever AW. Interleukin required for the blood and tissue eosinophilia but not granuloma formation induced by infection with <span class="yellow">Schistosoma mansoni</span>. Proc Natl Acad Sci USA 1990; 8'7; 61-65. 22. Herndon FJ, Kayes SG. Depletion of eosinophils by anti-IL-5 monoclonal antibody treatment of <span class="yellow">mice</span> infected with <span class="yellow">Trichinella spiralis</span> does not alter parasite burden or immunologic resistance to reinfection. J Immuno11992; 149: 3642-3647. 23. Oshima T. Standardization of techniques for infecting <span class="yellow">mice</span> with Tox24. ocara canis and observations on the normal migration routes of the larvae. J Parasito11961; 4'7= 652. Svetic A, Madden KB, Zhou XD, et al. A primary intestinal helminthic infection rapidly induces a gut-associated elevation of Th2-associated cytokines and IL-3. J Immuno11993; 150: 3434-3441. Kusama Y, Takamoto M, Kasahara T, Takatsu K, Nariuchi H, Sugane K. Mechanisms of eosinophilia in BALB/c-nu/+ and congenitally athymic BALB/c-nu/nu <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Immunology 1995; 84; 461-468. Rennick DM, Thompson-Snipes L, Coffman RL, Seymour BWP, Jackson JD, Hudak S. In vivo administration of antibody to interleukin-5 inhibits increased generation of eosinophils and their progenitors in bone marrow of parasitized <span class="yellow">mice</span>. Blood 1990; '76: 312-316. Petrolani M, Ruffle C, Lapa e Silva JR, Joseph D, Lobb RR, Boris Vargaftig B. Antibody to very late activation antigen 4 prevents antigen-induced bronchial hyperreactivity and cellular infiltration in <span class="yellow">guinea pig</span> airways. J Exp Med 1994; 180= 795-805. <br><br> References 25. 1. Nutman <br><br> 2. <br><br> 3. 4. 5. <br><br> 6. 7. <br><br> 8. <br><br> 9. <br><br> TB, Ottesen EA, Cohen SG. The eosinophil, eosinophilia, and eosinophil-related disorders. Allergy Proc 1989; 10; 47-62. Glickman LT, Schantz PM. Epidemiology and pathogenesis of zoonotic toxocariasis. Epidem Rev 1981; 3= 230-250. Beaver P, Snyder H, Carrera G, Dent J, Lafferty J. Chronic eosinophilia due to visceral larva migrans. Pediatrics 1952; 9: 7-19. Limaye AP, Abrams JS, Silver JE, Ottesen EA, Nutman TB. Regulation of parasite-induced eosinophilia: selectively increased interleukin production in helminth-infected <span class="yellow">patients</span>. J Exp Med 1990; l'7:a; 399-402. Steel C, Nutman TB. Regulation of IL-5 in onchocerciasis: a critical role for IL-2. J Immuno11993; 15{}; 5511-5518. Yamaguchi Y, Matsui T, Kasahara T, et al. In vivo changes of hemapoietic progenitors and the expression of the interleukin gene in eosinophilic <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Exp Hemato11990; 18; 1152-1157. Parsons JC, Coffman RL, Grieve RB. Antibody to interleukin prevents blood and tissue eosinophilia but not liver trapping in <span class="yellow">murine</span> larval toxocariasis. Parasite Immuno11993; 15: 501-508. Coffman RL, Seymour BWP, Hudak S, Jackson J, Rennick D. Antibody to interleukin-5 inhibits helminth-induced eosinophilia in <span class="yellow">mice</span>. Science 1989; 245; 308-310. Sher A, Coffman RL, Hieny S, Cheever AW. Ablation of eosinophil and IgE responses with anti-IL-5 or anti-IL-4 antibodies fails to affect immunity against <span class="yellow">Schistosoma mansoni</span> in the <span class="yellow">mouse</span>. J Immuno11990; 145: 3911<br><br> 26. <br><br> 27. <br><br> ACKNOWLEDGEMENTS. This work was funded by grant 300652/85-2 from Conselho Nacional de Desenvolvimento Cientifico e Tecnol6gico (CNPq) and Grant 92/5105-7 from Fundago de Amparo ft Pesquisa do Estado de So Paulo (FAPESP). We wish to thank Mrs M.A. Fernandes for technical assistance, Mrs M.M.O. Rossi for the histological sections, M. Costa Gongalves for the photography artwork, and Dr P. Minoprio, Institut Pasteur, for kindly providing TRFK-5. <br><br> 3916. 10. Urban <br><br> Jr JF, <br><br> Katona IM, Paul WE, Finkelman FD. Interleukin 4 is impor<br><br> Received 12 October 1995; accepted 17 November 1995 <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 31 <br><br>  <h3>pmcA2377325</h3>Purinergic signalling in the subretinal space: a role in the communication between the retina and the RPE
Abstract
The retinal pigment epithelium (RPE) is separated from the photoreceptor outer segments by the subretinal space. While the actual volume of this space is minimal, the communication that occurs across this microenvironment is important to the visual process, and accumulating evidence suggests the purines ATP and adenosine contribute to this communication. P1 and P2 receptors are localized to membranes on both the photoreceptor outer segments and on the apical membrane of the RPE which border subretinal space. ATP is released across the apical membrane of the RPE into this space in response to various triggers including glutamate and chemical ischemia. This ATP is dephosphorylated into adenosine by a series of ectoenzymes on the RPE apical membrane. Regulation of release and ectoenzyme activity in response to light-sensitive signals can alter the balance of purines in subretinal space, and thus coordinate communication across subretinal space with the visual process.<br><br>Introduction
The retinal pigment epithelium (RPE) lies between the outer segments of the photoreceptors and the choroidal blood supply (Fig. 1). The RPE combines the functions of epithelial and glial cells, providing a barrier while also supporting the neural photoreceptors and modulating their function. Tight communication between photoreceptors and the RPE is critical to coordinate the multiple levels of interaction, and the purinergic contribution to this communication is becoming apparent. The relevance of this purinergic input is emphasized by the many functional effects of P1 and P2 receptor stimulation and by the multiple mechanisms in place to regulate subretinal levels of purine agonists. As the dynamics of ATP release and extracellular conversion into adenosine will modify agonist availability, the modulation of these processes can exert a temporal control on purinergic signaling. The following review will outline the main interactions between the RPE and photoreceptors, describe the effects of stimulating purinergic receptors on both sides of subretinal space, and summarize how levels of ATP, ADP, and adenosine are manipulated in this microenvironment.
Fig. 1Schematic illustration of the key components of purinergic signaling in the subretinal microenvironment. Stimulation of P2 receptors on the RPE can enhance transepithelial fluid absorption while P1 receptors can modulate phagocytosis. ATP released through CFTR and other Cl− channels can stimulate P2 receptors or be converted to ADP, AMP, and adenosine (Ado) by a series of ectonucleotidases present on the apical membrane of the RPE. By controlling the balance of extracellular purines available to stimulate these receptors these mechanisms can control levels of endogenous purines available to activate the receptors. While theoretically possible, it remains to be determined whether these subretinal purines can actually stimulate photoreceptors
Purines and subretinal space
RPE-photoreceptor interactions across the subretinal space
The outer segments of the rods and cones are responsible for the initial stages of vision, converting photon energy into a series of enzymatic reactions that close the light-sensitive channels on the photoreceptor plasma membrane, hyperpolarize the cells, and reduce the release of glutamate from the synaptic terminals [1, 2]. Efficient photoreceptor function depends upon both short-term and long-term support from the RPE. The critical nature of these interactions is evident from the rapid degeneration of photoreceptors in the absence of a healthy RPE layer and by the RPE localization of defective gene product in some forms of hereditary photoreceptor degeneration [3].
The apical membrane of the RPE is separated from the plasma membrane of the outer segments by an extracellular space of only 10–20 nm [4]. Although small, this subretinal space contains a highly structured matrix which ensheathes the outer segments and extends to the RPE [5, 6]. The presence of enzymes within this interphotoreceptor matrix emphasizes that this extracellular space itself is functionally active [7, 8]. This intimate anatomical relationship between photoreceptors and the RPE reflects multiple functional interactions. For example, the RPE accepts, recycles, and exports central components of the phototransduction process [9]. The outer segments are continuously resynthesized, and the phagocytosis, degradation, and processing of shed outer segment tips by the RPE cells is central to this renewal [10]. The ion channels and transporters on the apical membrane of the RPE help regulate the ionic composition of the subretinal space [11]. As extracellular levels of ions can modify the ionic driving forces across the photoreceptor plasma membrane, these RPE transporters can influence the state of neural activity. The transport of fluid and ions from the apical membrane to basolateral membrane of the RPE is also one of the main forces keeping the retina attached [12].
The control of photoreceptor activity by light gives a rapid temporal dependence to some interactions between the photoreceptors and the RPE. The volume of subretinal space increases in response to light [13], linking small changes in the ionic composition of the subretinal space with activity of the RPE transport mechanisms which maintain this volume [14, 15]. Other processes are controlled on a diurnal cycle. The shed tips of the outer segments are phagocytosed by the RPE soon after the onset of light [16, 17]. These processes can both be modulated by purine levels in subretinal space, indicating purinergic regulation is important over multiple time scales.<br><br>Purinergic receptors on photoreceptors
A2 adenosine receptors were localized to both the inner and outer segments of photoreceptor outer segments over a decade ago by Blazynski and colleagues [18], with more recent reports emphasizing their functional role. A2 agonists inhibit the L-type Ca2+ channel on rod outer segments [19] and can inhibit the synaptic release of glutamate from rods, suggesting changes in adenosine levels in subretinal space could modulate light sensitivity [20]. The A2 agonist DPMA and the adenosine deaminase inhibitor EHNA reduce mRNA for opsin in rods, suggesting that endogenous levels of adenosine can downregulate opsin message at night [21]. EHNA and the A2A receptor agonist CGS21680 also increase the survival of <span class="yellow">chick</span> embryonic photoreceptors in culture [22], indicating a long-term neuroprotective role for adenosine.
P2 receptors are also present in the photoreceptors. mRNA for the P2X2 receptor is expressed in the photoreceptor cell bodies, with immunohistochemical localization of the protein to outer segments [23]. In situ hybridization indicates the photoreceptor layer has the highest level of P2Y2 receptor of any region in the <span class="yellow">rabbit</span> retina, although staining was not pronounced in monkey [24]. P2X7 receptors have recently been localized to photoreceptor synaptic terminals, with evidence for ecto-ATPase activity in the synapse, and functional evidence suggesting ATP augments transmission of the light response by rods [25]. It was suggested that ATP might be co-released from photoreceptors with glutamate, although this remains to be tested directly.<br><br>Purinergic receptors on the RPE
Stimulation of P1 receptors can have a considerable impact on RPE cells. A2 receptors have been recognized on cultured and fresh RPE cells for some time [26, 27], with in situ hybridization confirming the presence of A2A receptors in <span class="yellow">rat</span> RPE [28]. Stimulation of A2 receptors reduces the rate of rod outer segment phagocytosis by RPE cells [29], while application of adenosine to the apical membrane of <span class="yellow">chick</span> RPE cells increases the basolateral Cl- conductance, the transepithelial potential, and the c-wave, and decreases the hyperpolarization of the basal membrane in response to light [30]. Although adenosine alone does not increase intracellular Ca2+ levels [31], adenosine acts synergistically with ATP to elevate Ca2+ levels in <span class="blue">human</span> RPE cells by stimulating both A1 and A2A receptors [32, 33]. Stimulation of A1 receptors with high doses of NECA increases the active transport of fluorescein across the RPE, while activation of A2A receptors decreases this transport, and by extension, transport of the ions that underlie fluid movement [34]. Stimulation of A1 and A2A receptors produces analogous increases and decreases, respectively, in the absorption of subretinal fluid blebs. This is consistent with the negative coupling of the A1 receptor and the positive coupling of the A2 receptors to adenylate cyclase, as increasing cAMP inhibits the transport of fluid across the RPE towards the choroid [35–37]. The agonist 2-Cl adenosine reverses the deficit in phosphoinositide metabolism found in diabetic RPE cells [38], suggesting effects on metabolism in addition to transport and phagocytosis.
Multiple P2 receptors have been localized to the RPE. The P2Y2 receptor was initially characterized in cultured <span class="blue">human</span> RPE [31], with subsequent reports localizing transcript for P2Y1, P2Y2, P2Y4, and P2Y6 in the <span class="yellow">rat</span> RPE/choroid [39] and for P2Y1 and P2Y12 receptors in ARPE-19 cells [40], and functionally identifying a P2X receptor in <span class="yellow">rat</span> RPE cells [41]. ATP, ADP, and UTP induce numerous effects on RPE physiology [32, 33, 42, 43]. While these effects likely involve multiple different receptor types, the contributions of the P2Y2 receptor have been explored in most detail to date. The P2Y2 receptor has been specifically localized to the apical membrane of fresh <span class="yellow">bovine</span> RPE cells, and addition of ATP to this membrane transiently elevates Ca2+, activates a basolateral Cl- conductance, inhibits an apical K+ conductance, and increases the apical to basolateral flow of fluid [43]. This increased absorption of fluid from the subretinal space suggests P2Y2 receptor stimulation could reduce the excess fluid that accumulates in retinal edemas, and several reports have supported this theory. ATP, UTP, and the P2Y2 receptor agonist INS37217 decrease the size of subretinal fluid blebs when injected into subretinal space of <span class="yellow">rats</span> [44]. In both normal and rds +/- <span class="yellow">mice</span> with experimentally induced detachment, INS31217 improves the ERG recovery and decreased cell death [45]. INS37217 also reduces subretinal blebs in <span class="yellow">rabbits</span> [46]. Injection of another P2Y2 agonist, INS542, increases the active transport of fluorescein across the RPE, consistent with this upregulation of ion and fluid transport across the tissue [47]. Together these experiments emphasize the clinical potential of treating retinal edema with P2Y2 agonists.<br><br>
Regulation of purine levels in subretinal space
While synthesized purinergic agonists may prove useful in treating some ocular disorders, the endogenous activation of P1 and P2 receptors in the subretinal microenvironment will ultimately be determined by availability of agonists. These levels are largely controlled by the release of ATP into the subretinal space, its conversion into other purines including adenosine, and the manipulation of adenosine by enzymes or transporters. Recent work has increased our understanding of both the stimuli that initiate changes in subretinal purine levels and the mechanisms that mediate these changes.
Release of ATP by the RPE
At least some of the ATP capable of stimulating the purinergic receptors on RPE cells is released from the RPE itself. The resulting autocrine stimulation ensures local delivery, and control, of purines to initiate the physiologic changes in the RPE. The release of ATP by RPE cells is triggered by multiple stimuli including osmotic stress, bFGF, UTP, NMDA, glutamate, and ATP [39, 40, 48–51]. The ATP release following activation of NMDA receptors by glutamate may have the most interesting implications for communication across subretinal space, given that glutamate confers the light signal from photoreceptors to the rest of the visual system. Glutamate and the specific receptor agonist NMDA triggers ATP release from ARPE-19 cells, with the release inhibited by NMDA antagonist MK-801, and by DCKA, which inhibits the glycine B binding site on NMDA receptors [51, 52]. Although NMDA raises intracellular Ca2+ levels, this increase is prevented by eliminating ATP with apyrase, indicating autostimulation through released ATP is responsible for this Ca2+ signal. NMDA also triggers a release of ATP when applied to the intact <span class="yellow">bovine</span> RPE eyecup [51]. The NMDA receptors and the ATP release sites have been functionally identified to the apical membrane of the <span class="yellow">bovine</span> RPE, suggesting the neurotransmitter interactions could amplify the signal from any glutamate reaching subretinal space.
The ability of both UTP and ATP to stimulate release of ATP from the RPE supports the theory that the system acts to amplify signals. When applied at greater than 1 μM, ATP triggers a secondary release of ATP peaking 10 min after the initial stimuli [40]. UTP also initiates a release in extracellular ATP with a similar delay [48]. The rise in ATP triggered by UTP is inhibited by the Cl- channel blocker NPPB, and UDP is much less effective at triggering release than UTP; both observations suggest the P2Y2 receptor contributes to the increase in ATP more than diphosphokinase, although influence from the enzyme cannot be ruled out [53].
Recent evidence suggests that ischemia may lead to the release of ATP from RPE cells. Chemical ischemia triggers a substantial ATP release from cardiac myocytes [54], while changes in oxygen levels trigger ATP release in central chemoreceptors [55]. We found that exposure to sodium cyanide led to a rapid release of ATP from ARPE-19 cells (Fig. 2). As hypoxic and/or ischemic challenge may lead to changes in the expression of growth factors in RPE cells during certain ocular disorders such as macular degeneration [56], and as purines can induce expression of VEGF in other cells [57], this ATP release may contribute to growth factor signaling by the diseased RPE.
Fig. 2Chemical ischemia triggers ATP release from ARPE-19 cells. ATP release was measured in the bath directly from cells plated in 96-well plates to which the luciferin- luciferase reaction mixture was added [51]. Left Levels of ATP in the bath after addition of 5 mM NaCN to the cells. Measurement began 1 min after addition of NaCN or control solution to wells. Right Levels of ATP measured at the peak, 3 min after addition of NaCN (n = 12). Levels were normalized to concurrent levels in control (n = 14). Symbols and bars represent mean ± SE, *p < 0.001
The particular conduit for ATP release varies with the stimuli. The release in response to hypotonic challenge is largely dependent upon CFTR, as it was prevented by the specific CFTR inhibitor CFTR172 in addition to the more general blocker glybenclamide [50]. While the precise mechanisms by which CFTR contributes to this release are not yet known, a role for CFTR in ATP release into subretinal space is consistent with the reduction of certain ERG components in cftr -/- <span class="yellow">mice</span> [58] and with the ability of apical ATP to activate conductances associated with these ERG components [43]. The release of ATP is also largely blocked by the vesicular transport inhibitor brefeldin A, suggesting the two processes occurred in series whereby ATP efflux follows the insertion of vesicles containing CFTR into the plasma membrane. Although the Ca2+ chelator BAPTA blocks this ATP release [50], raising Ca2+ alone with ionophore ionomycin does not itself initiate release [48]. This necessary but not sufficient contribution of Ca2+ also supports a role for vesicular insertion.
In contrast to the release following hypotonic challenge, the ATP release in response to NMDA does not involve CFTR [51]. Release is blocked by NPPB, however, suggesting another type of anion channel could serve as a conduit for ATP release. The presence of parallel mechanisms coexisting on the same cell for ATP release triggered by either agonists or by cell swelling has also been reported in astrocytes [59] and may reflect the multiple roles of purinergic signaling within a given tissue. As both stimuli lead to release across the apical membrane into subretinal space, both are expected to influence signaling in the microenvironment.<br><br>Interconversion of purines in subretinal space
The interconversion of nucleotides and nucleosides each capable of stimulating distinct receptors makes the purinergic signaling system of particular interest in a confined region such as the subretinal space. The main enzymes responsible for dephosphorylating extracellular ATP on the RPE cells have been analyzed and a basic understanding of their regulation has begun. This section first describes the enzymes that act on ATP and ADP, followed by enzymes which convert AMP into adenosine.
The dephosphorylation of extracellular ATP by RPE cells involves enzymes from multiple families [40], as found in airway epithelial cells [60]. Degradation of ATP by the apical membrane of the fresh <span class="yellow">bovine</span> eyecup and by ARPE-19 cells is inhibited by ARL67156 or βγmATP. Message for eNPP1, eNPP2, and eNPP3 is present in ARPE-19 cells, consistent with the preference of βγmATP for members of the eNPP family [61]. The cells also express NTPDase2, and NTPDase3, although the intermittent presence of NTPDase1 likely reflects a regulated process [40]. Ecto-alkaline phosphatase has no effect on ATP degradation in RPE cells, in contrast to its considerable contribution in airway epithelium [62]. The putative contribution from diphosphokinases to interconversion of subretinal purines is presently unknown.
Extracellular AMP is rapidly dephosphorylated into adenosine in subretinal space. The production of adenosine from ATP at the apical membrane of the <span class="yellow">bovine</span> RPE eyecup is inhibited by the ecto-5′-nucleotidase inhibitor αβmADP, confirming a role for this enzyme [63]. The enzyme is localized to <span class="yellow">rat</span> RPE and ARPE-19 cells immunohistochemically. Degradation of 5′AMP is highest near the subretinal space of <span class="yellow">rat</span> retina [63], although localization in <span class="yellow">mouse</span> indicated larger amounts of ecto-5′-nucleotidase at the tips of adjacent Müller cells [64]. Levamisole does not inhibit the dephosphorylation of 5′AMP by the RPE, consistent with the absence of substantial ecto-alkaline phosphatase in subretinal space.
The presence of light may alter the levels of adenosine in subretinal space. Epinephrine is released at the onset of light [65] and stimulation of the RPE with epinephrine can decrease activity of ecto-5′-nucleotidase [63]. While norepinephrine and phenylephrine lead to similar decreases in enzyme activity, prazosin and corynanthine block the effects of norepinephrine, implicating the α1 epinephrine receptor in the inhibition of ecto-5′-nucleotidase [63]. The kinetics of inhibition are consistent with cleavage of the nucleotidase from its GPI anchor. The phagocytosis of rod outer segments is maximal shortly after light onset [16], and this phagocytosis is inhibited by adenosine [29]. The ability of epinephrine released by the illuminated retina to reduce ecto-5′-nucleotidase activity and consequently adenosine levels may relieve this inhibition and enhance the rate of phagocytosis at light onset.<br><br>
Physiologic effects of subretinal purines on the RPE and photoreceptors
The number of purinergic receptors on both photoreceptor and RPE membranes suggests purines make multiple contributions to the physiology of the outer retina. Our increased understanding of how agonist levels in subretinal space are controlled has begun to indicate how and when this contribution may occur. Future research will involve applying these findings from isolated systems to intact RPE-photoreceptor models, and pursuing the role of defective purinergic regulation in ocular disease. While it is unlikely that ATP released across the apical membrane of the RPE can diffuse to these P2 receptors in the outer plexiform layer given the ecto-ATPase activity in the synaptic clef [25], stimulation of receptors elsewhere on the photoreceptor membrane is possible. It would be interesting to determine whether ATP released from the RPE and converted to adenosine by ecto-nucleotidases can actually modulate the response to light by stimulating the A2A receptors on photoreceptor outer segments. The impact of purinergic signaling on chronic ocular diseases is also of interest, such as the role of ischemia-driven ATP release in VEGF production. While the small size of subretinal space can complicate pharmacologic manipulation within the intact RPE-photoreceptor complex, molecular approaches may provide new insight into how endogenous purines in subretinal space affect the physiology, and pathophysiology, of both RPE and photoreceptors.<br><br><br><br><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="yellow">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="yellow">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="yellow">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="yellow">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="yellow">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="yellow">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="yellow">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="yellow">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="blue">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="yellow">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="yellow">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="yellow">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="yellow">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="yellow">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="yellow">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="yellow">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="yellow">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="yellow">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="yellow">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="yellow">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
<h3>pmcA2430230</h3>Emergence of Delayed Methylmercury Toxicity after Perinatal Exposure in Metallothionein-Null and Wild-Type C57BL <span class="yellow">Mice</span>
Abstract
Background
Although a long latency period of toxicity after exposure to methylmercury (MeHg) is known to exist in <span class="yellow">humans</span>, few animal studies have addressed this issue. Substantiation of delayed MeHg toxicity in animals would affect the risk evaluation of MeHg.<br><br>Objectives
Our goal in this study was to demonstrate the existence of a latency period in a <span class="yellow">rodent</span> model in which the toxicity of perinatal MeHg exposure becomes apparent only later in life. Our study included metallothionein (MT) knockout <span class="yellow">mice</span> because studies have suggested the potential susceptibility of this strain to the neurodevelopmental toxicity of MeHg.<br><br>Methods
Pregnant MT-null and wild-type C57Bl/6J <span class="yellow">mice</span> were exposed to MeHg through their diet containing 5 μg Hg/g during gestation and early lactation. We examined behavioral functions of the offspring using frequently used paradigms, including open field behavior (OPF), passive avoidance (PA), and the Morris water maze (MM), at ages of 12–13 and 52–53 weeks.<br><br>Results
At 12 weeks of age, behavioral effects of MeHg were not detected, except for OPF performance in MeHg-exposed MT-null females. At 52 weeks of age, the MeHg-exposed groups showed poorer performance both in PA and MM, and their OPF activity differed from controls. These effects of MeHg appeared exaggerated in the MT-null strain. The brain Hg concentration had leveled off by 13 weeks of age.<br><br>Conclusions
The results suggest the existence of a long latency period after perinatal exposure to low-level MeHg, in which the behavioral effects emerged long after the leveling-off of brain Hg levels. Hence, the initial toxicologic event responsible for the late effects should have occurred before this leveling-off of brain Hg.<br><br><br><br>Methylmercury (MeHg) poses serious and practical concerns for <span class="blue">human</span> populations regarding perinatal exposure. Fish, especially large predator (carnivore) fish species, accumulate high concentrations of MeHg through the marine food chain, and exposure of pregnant <span class="yellow">women</span> to MeHg through the consumption of fish has evoked widespread concern due to potential effects on offspring. Two large-scale cohort studies in fish-eating populations of Seychelles and Faroe islanders are being conducted; although the former has not found consistent adverse developmental effects of MeHg (Myers et al. 2007), the latter has reported adverse effects (Debes et al. 2006). Fundamental reasons for this discrepancy have not been completely elucidated, and many questions remain regarding the neurotoxicity of MeHg, despite extensive study.
Among the unanswered questions is whether there is a long latency period for behavioral manifestations after exposure to MeHg (Clarkson and Magos 2006; Landrigan et al. 2005; Rice 1996; Weiss et al. 2005a, 2005b). Typical examples of latent toxicity in <span class="yellow">humans</span>, including both acute and chronic MeHg exposures, have been described in detail elsewhere (Weiss et al. 2005a). Davidson et al. (2006) recently suggested that effects of perinatal exposure to MeHg may emerge 9 years after birth in the Seychelles cohort. Consequently, risk assessments of MeHg exposure could be inaccurate because studies (<span class="blue">human</span> or animal) usually do not focus on later stages of life and therefore could miss delayed effects. The possibility of delayed toxicity is exemplified by the expanded Barker hypothesis, which posits that the origin of some neurodegenerative diseases such as Parkinson and Alzheimer diseases lies in early exposure to environmental chemicals (Landrigan et al. 2005). Although epidemiologic evidence would be ideal for exploring the possibility of delayed toxicity (and, indeed, data from epidemiologic studies form the basis of current risk assessment for developmental toxicity of MeHg), considering the complex effects of numerous potential confounders and the existence of multiple exposures in <span class="blue">human</span> populations, animal models would likely make important contributions to this field.
Although numerous animal studies have described the developmental neurotoxicity of MeHg (Watanabe and Satoh, 1996), few have addressed the latency issue. Few studies have evaluated the neurobehavioral effects in rodents longitudinally beyond 6 months after perinatal exposure. Spyker (1975) addressed this issue in her pioneering work, reporting the late development of behavioral toxicity in <span class="yellow">mice</span> prenatally exposed to MeHg; it appeared, however, that the substantial mortality and retarded growth among the exposed <span class="yellow">mice</span> were apparent before weaning, indicating that the doses used (even though some lower dose levels were included) exerted severe toxicity. Using a relatively complex schedule-controlled operant behavior method, <span class="yellow">rats</span> whose parents were exposed to MeHg (0.5 or 6.4 mg/L) from 4 weeks before mating and continuing to postnatal day (PND) 16 were shown to be less sensitive to a change in the reinforcement schedule than were their nonexposed counterparts at 28–32 months of age (Newland et al. 2004). <span class="yellow">Mice</span> that were perinatally exposed to 1 or 3 mg/L MeHg in drinking water did not show significant deviation from controls in behavioral performance (motor performance, memory, and learning) at 5, 15, or 26 months of age, but the lifetime-exposed groups did show a significant deviation (Weiss et al. 2005c). The existence of a latency period (i.e., the absence of effects earlier in life followed by the emergence of effects at a later stage of life) has not been demonstrated in any <span class="yellow">rodent</span> study. In nonhuman primates, delayed emergence of the signs of neurotoxicity was observed several years after the cessation of a 7-year postnatal exposure (Rice 1996).
Metallothionein (MT) protects against the toxicities of a variety of metals. We examined the neurotoxicity and developmental toxicity of metallic Hg in MT I/II-knockout <span class="yellow">mice</span> (Yoshida et al. 2004) and demonstrated the susceptibility of this genetically manipulated strain to the toxicity of metallic Hg. In contrast to metallic Hg, MeHg does not induce MT, and MT would not substantially influence the kinetics of MeHg (Yasutake et al. 1998). Several reports, however, have demonstrated protective effects of MT against MeHg toxicity, which was ascribed to the radical scavenging effect of MT (Yao et al. 2000). We also showed that perinatal exposure to MeHg results in altered metabolism of thyroid hormones in neonates that was more distinct in MT-null strains than their wild-type counterparts (Mori et al. 2006). The vulnerability of the MT-null strain suggests that delayed neurobehavioral toxicity due to MeHg, if it does exist, might be more distinctive in this strain.
By utilizing the MT-null strain, we aimed to answer the following two questions: First, could we generate a model in which the toxicity of MeHg would emerge or at least become exaggerated later in life as opposed to earlier in life (i.e., at 3–6 months, which was the timing for most of the earlier studies that used behavioral evaluations)? Second, would the MT-null strain be affected more than its parent C57BL strain? Answering either of these questions not only could influence the risk evaluation of MeHg, but it could also lead to a better understanding of the mechanism of toxicity for perinatal MeHg exposure. To address these issues, we used three behavioral paradigms, the open field (OPF), passive avoidance (PA), and Morris (water) maze (MM) tests, which are often used in this field and which we used in our previous studies on the effects of Hg vapor (Yoshida et al. 2004, 2006). Performances in the MM and PA are said to be the most sensitive to aging (Gower and Lamberty 1993).
We used a dose of 5 μg MeHg/g in the diet, which resulted in a brain Hg level relevant to <span class="blue">human</span> risk assessment. We evaluated the behavioral end points twice, once around 3 months of age and the other time around 1 year; the latter time roughly corresponds to the period when many behavioral performances, including OPF (Acevedoa et al. 2006; Carrie et al. 1999; Gower and Lamberty 1993), PA (Gower and Lamberty 1993), and MM (Bach et al. 1999; Carrie et al. 1999), show alterations in this <span class="yellow">mouse</span> strain.
Materials and Methods
Animals and MeHg exposure
OLA129/C57BL/6J strain <span class="yellow">mice</span> (wild type) and MT I/II-knockout <span class="yellow">mice</span> (MT-null) of this strain were provided by K.H. Choo of the Murdoch Institute, Parkville, Australia (Michalska and Choo 1993) and were of a mixed genetic background of 129/Ola and C57BL/6 strains. F1 hybrid <span class="yellow">mice</span> were mated with C57BL/6 <span class="yellow">mice</span> for six generations at the National Institute for Environmental Studies (Tsukuba, Japan). At 10 weeks of age, single male and female <span class="yellow">mice</span> were allowed to cohabit; every female <span class="yellow">mouse</span> was checked each morning for the presence of a vaginal plug. When a plug was confirmed, the day was designated as day 0 of gestation (GD0).
The diet, NIH-07PLD formula (CLEA Japan, Inc., Tokyo, Japan), contained vitamins and trace elements as follows (per kilogram diet): 3.2 mg CuSO4, 88 mg FeSO4, 149 mg MnSO4, 25 mg ZnCO3, 1.6 mg Ca(IO3)2, 11 mg vitamin B1, 4.7 mg vitamin B2, 1.9 mg vitamin B6, 44 mg vitamin E, in addition to 5 μg MeHg/g. This diet was fed to the pregnant <span class="yellow">mice</span> starting from GD0 through 10 days after delivery (i.e., PND10). Thereafter, we switched <span class="yellow">mice</span> to a diet that did not contain MeHg. We chose GD0 as the beginning of exposure because exposures that started before conception often resulted in fairly high Hg concentrations in fetal/neonatal brains (Kakita et al. 2003), and we chose PND10 to cover the early neonatal period, in which considerable brain growth occurs. In our experimental setting, the neonatal <span class="yellow">mice</span> started to eat from the diet bucket and drink from the water bottle from PND10 onward. Control <span class="yellow">mice</span> were kept on the same diet but without MeHg (< 0.01 μg Hg/g). On PND1, to avoid the confounding effects due to different litter size, we reduced each litter to six pups (three males and three females when possible), and on PND10, two males and two females from each litter were killed for chemical analyses.
The remaining male and female offspring per litter were weaned on PND28 and used for subsequent behavioral analyses (either at 12–13 weeks or 52–53 weeks, depending on the litter) as described below. We measured body weights of the weaned <span class="yellow">mice</span> every 2 weeks. Thus, four experimental groups were used (with or without MeHg exposure for two strains), and each group consisted of 12–13 litters. For half of the litters, the behavioral analyses were conducted at 12–13 weeks of age, and upon completion of the behavioral analyses, the animals were killed under ether anesthesia. We then dissected the organs (brain, liver, and kidneys) for Hg analyses. For the remaining half of the litters, we conducted the behavioral tests at 52–53 weeks of age. The <span class="yellow">mice</span> were treated humanely and with regard to alleviation of suffering according to the National Institute for Environmental Studies’ Guidelines for Animal Welfare and the guidelines of St. Marianna University.<br><br>Behavioral evaluations
The details of each behavioral procedure have been described elsewhere (Yoshida et al. 2006). Brief descriptions follow.
For the OPF test, we used an OPF apparatus (Ohara Co., Ltd., Tokyo, Japan) with a 60 × 60-cm floor surrounded by walls 60 cm high. The experimental room light was turned off, and a dim light of 80 lux was lit during the experiment. We placed a <span class="yellow">mouse</span> in the center of the floor and monitored its behavior for 10 min using a CCD camera connected to a computer. The position of the center of gravity was calculated by image-analyzing software, which was used to calculate the total distance traveled by the <span class="yellow">mouse</span> as well as the positional preference (either center or peripheral, where peripheral was defined as the area within 10 cm from the wall). We cleaned the OPF apparatus with 70% ethanol between trials.
The apparatus for the PA test (Ohara Co. Ltd.) consisted of a light compartment that illuminated by a 400-lux light and a dark compartment with black opaque walls and lids. The two compartments were separated by a mobile guillotine door. On the first day (training trial), we placed each <span class="yellow">mouse</span> in the light compartment facing away from the guillotine door, which was closed. After 30 sec of introduction, the door was opened; when the <span class="yellow">mouse</span> entered the dark compartment, a brief electric shock (4 mA for 2 sec) was delivered through the metal grid floor. This would force the <span class="yellow">mouse</span> back to the light component. The interval between the opening of the door and the entry to the dark room (in seconds) was recorded and defined as the latency. On the next day, the same procedure was repeated, but without the electric shock (retaining trail). In this PA paradigm, aversive learning was assumed to be established in the training trial, and we used its retention in the retaining trail as the index of learning. Between each individual trial, we cleaned the apparatus with ethanol. The cutoff time of the retention session was 300 sec.
The MM test apparatus (Ohara Co., Ltd.) was a round-shaped water pool with a diameter of 120 cm. A small platform was submerged in the water, which provided a place for <span class="yellow">mice</span> to escape from the water (i.e., an aversive stimulus, water temperature = 23 ± 1°C). The water was made opaque by adding white paint so that the <span class="yellow">mouse</span> could not see the submerged platform. In each trial, we released a <span class="yellow">mouse</span> into the pool from a determined position along the wall, and the performance of the <span class="yellow">mouse</span> was monitored by a CCD camera/image analyzer. The time required to reach the platform was recorded. If a <span class="yellow">mouse</span> could not find the platform within 60 sec after release, it was led to the platform and placed on it for 20 sec before being removed. In these cases, a latency of 60 sec was recorded. We conducted the trial once a day up to the fifth day for each <span class="yellow">mouse</span>, and the order of each <span class="yellow">mouse</span> was counterbalanced across the day. On the sixth day, a transfer test (or probe test), which is a trial without the platform, was conducted; in this procedure, we counted the number of times that the <span class="yellow">mouse</span> crossed the position where the platform had been.<br><br>Tissue Hg concentration
The tissue samples were homogenized (10% weight/volume) in distilled water using a Polytron homogenizer (Kinematica GmbH, Littau, Switzerland). We determined Hg levels in the homogenates by the oxygen combustion–gold amalgamation method (Ohkawa et al. 1977) using an atomic absorption Hg detector MD-1 (Nippon Instruments, Co. Ltd., Osaka, Japan). To ensure the accuracy of the measurement, we included reference material from a dogfish (DORM-2; National Research Council of Canada, Ottawa, Ontario, Canada) with a certified value of 4.64 ± 0.26 μg/g in the analyses; the observed values fell within the certified range. The detection limit of the measurement was 0.1 ng Hg.<br><br>Statistics
We analyzed behavioral data for OPF and PA by analysis of variance (ANOVA), taking sex, strain, and MeHg exposure as the factors. All the interactions among these factors were put into the model. When any of the interactions was highly significant, we analyzed the data separately in an appropriate way; for example, if sex × Hg was significant, the data for males and females were separately analyzed for the effects of strain and Hg. Whenever appropriate, ANOVAs were followed by Mann-Whitney U or Student’s t-tests, depending on the nature and distribution of the variables. We analyzed data for the MM by repeated-measures ANOVA, taking the exposure as between-group and trials as within-group variables. The test was performed for each of the four sex and strain combinations separately. The significance level was set at p < 0.05.<br><br>
Results
Body weight
Up to 20 weeks of age, the body weight values of the control and MeHg-exposed groups were not different, regardless of strain or sex. After 28 weeks, except for the wild-type female groups, the MeHg-exposed groups weighed significantly less than the controls (Figure 1).<br><br>OPF
At 12 weeks of age, a three-way ANOVA of the locomotion distance revealed that only strain was a significant factor (p < 0.001), reflecting the longer distance traveled by the MT-null <span class="yellow">mice</span> (Figure 2A). Strain was also a significant factor for the proportion of the central-area locomotion (Figure 3A), and Hg exposure marginally affected this outcome. In MT-null females, the proportion of central-area locomotion was higher in MeHg-exposed <span class="yellow">mice</span> than in the controls; this difference was not observed in any other strain–sex combination. At 52 weeks of age, the strain × Hg interaction was highly significant (p < 0.001) in an ANOVA of locomotion distance (Figure 2B); MeHg exposure was associated with decreased locomotion distance in wild-type <span class="yellow">mice</span> and with increased distance in MT-null <span class="yellow">mice</span>. A strain-wise two-way ANOVA (with sex and Hg as the factors) revealed that only Hg was significant in both strains (p < 0.01). Regarding the proportion of the central-area locomotion, the effects of MeHg appeared to depend on sex [i.e., sex × Hg was highly significant (p < 0.001) in the three-way ANOVA; Figure 3B]. Indeed, a sex-wise two-way ANOVA showed significant effects of Hg only in females (p < 0.001).<br><br>PA
At 12 weeks of age, all the groups showed prolonged latency in the second (retention) trial, and no consistent effect of MeHg was recognized regardless of strain or sex (Figure 4A).
At 52 weeks of age, a three-way ANOVA revealed a significant interaction between strain and Hg (p < 0.05); strain-wise two-way ANOVAs revealed a significant effect of Hg on learning in MT-null <span class="yellow">mice</span> of both sexes; these groups of <span class="yellow">mice</span> showed significantly shorter (less than half) latency times compared to control <span class="yellow">mice</span> (Figure 4B). A notable difference between the results at 52 weeks of age and those at 12 weeks was that many of the tested <span class="yellow">mice</span> exceeded the cutoff time in the retention trials at 52 weeks, except for the MT-null groups.<br><br>MM
At 13 weeks of age, repeated-measures ANOVA did not indicate any effects of MeHg (Figure 5A). At 52 weeks of age (Figure 5B), wild-type males and MT-null females shared the same statistical results; Hg as well as the Hg × trial interaction were statistically significant. Thus, in both cases, the MeHg groups showed a longer latency, hampering learning performance.<br><br>Tissue Hg concentration
At PND10, which was immediately after the exposure, brain Hg concentrations of the neonatal <span class="yellow">mice</span> were approximately ≤ 0.5 μg/g (Table 1). Although the MT-null <span class="yellow">mice</span> and females showed slightly higher brain Hg concentrations than the corresponding wild-type group and males, respectively, neither of these differences was significant. At 13 weeks, when the behavioral tests were completed, the brain Hg concentration was comparable to the control (nonexposed) level (approximately 5 ng/g in both the exposed and control brains). Interestingly, MT-null <span class="yellow">mice</span> had a significantly lower brain Hg concentration than the corresponding wild-type groups.<br><br>
Discussion
Results of the present study demonstrate the delayed emergence of neurobehavioral toxicity due to perinatal MeHg exposure, which presumably developed after brain MeHg concentrations had leveled off. This emergent toxicity was exaggerated in MT-null <span class="yellow">mice</span> and was more distinct in females. To our knowledge, our findings show the first clear-cut demonstrations of a long latency period of MeHg neurobehavioral toxicity in rodents and possible genetic susceptibility for the emergent toxicity.
The exposure level should be considered before discussing the end points. On PND10, immediately after the cessation of MeHg exposure, the brain Hg concentration was approximately 0.5 μg/g, regardless of the strain or sex. In a previous study, the brain Hg concentration in <span class="yellow">mice</span> perinatally exposed to 6 mg MeHg/L (via water) peaked between PND0 and PND4 and was approximately three times higher than on PND21 (Goulet et al. 2003). Therefore, the peak brain Hg concentration, which is presumably observed around birth, can be estimated as about 3-fold higher than that on PND10 and would be approximately 1.5 μg/g (0.5 μg/g × 3), which is one of the lowest levels among recent <span class="yellow">rodent</span> studies. As shown by Sakamoto et al. (2002) in their Figure 2, prenatal exposure of <span class="yellow">rats</span> to MeHg showed a peak brain Hg concentration around PND1 that was four to five times greater than the level on PND10. Also, Newland and Rasmussen (2000) reported a slight alteration of a complex operant behavior in <span class="yellow">rats</span> at ages < 2 years at brain Hg concentrations as low as 0.5 μg/g at birth, although statistical significance of this particular effect was not clear. It should be noted that <span class="yellow">rats</span> have different Hg kinetics (Hirayama and Yasutake 1986; Yasutake and Hirayama 1990) due to the high affinity of <span class="yellow">rat</span> hemoglobin for MeHg (Clarkson and Magos 2006).
The most important observation of the present study was that the effects of low-level MeHg exposure were detected only at later stages in the lives of the <span class="yellow">mice</span>. Except for the central-area occupancy in OPF in MT-null females, no statistically significant effects of MeHg were observed in any of the three behavioral tests at 12 weeks of age. In contrast, significant effects were observed in all three tests at 52 weeks of age. The brain Hg concentration of the exposed groups had leveled off and was not distinguishable from the non-exposed group at 13 weeks of age, immediately after the first phase of the behavioral testing. Therefore, in the present study, there was a latency period in which the dose and effects could not be detected, although effects were observed 9 months later. Another notable observation was that the emergent manifestation of toxicity was also recognized in the suppression of body weight (except for wild-type females), which only became apparent on or after 28 weeks of age.
The existence of a latency period of as long as several years after chronic (7 years from birth), low-level exposure to MeHg has been described in nonhuman primates (Rice 1996). In that case, however, the Hg concentration in the brain remained elevated, presumably as a result of the long exposure. Indeed, Rice (1996) argued that the minute amount of residual brain Hg could have caused the delayed toxicity. This was clearly not the case in the present study because the Hg concentration leveled off around the time of the first phase of the behavioral study. The absence of behavioral effects at 12 weeks of age ruled out the possibility that the residual behavioral effects were due to elevated Hg early in life. Therefore, the behavioral toxicity that surfaced at 52 weeks of age must have had its origin before the brain Hg concentration leveled off (at or before 13 weeks of age), although the redistribution of Hg to the brain from other sites of deposition, such as the liver, cannot be completely excluded. The long silent period before the manifestations of toxicity emerged suggests that a slow process plays a role in this latent toxicity.
Although an example of a slow process is aging, 52 weeks of age might not be sufficiently old for a <span class="yellow">mouse</span> to be considered aged in the physiologic sense because C57BL/6 <span class="yellow">mice</span> have a relatively long life span among <span class="yellow">mouse</span> strains [median survival of 27–31 months (Gower and Lamberty 1993)] and a survival rate at 18–19 months as high as 90% (Institute on Aging HP) (National Institute on Aging 2008). Nevertheless, various behavioral examinations have shown age-related changes in the performance of <span class="yellow">mice</span> at approximately 1 year of age in the OPF (Acevedoa et al. 2006; Carrie et al. 1999), MM (Bach et al. 1999; Carrie et al. 1999), and PA (Gower and Lamberty 1993) tests. The observed effects of MeHg, including the deterioration in the PA and MM and suppression in the OPF (in wild-type <span class="yellow">mice</span>), were consistent with these reported effects of aging on behavioral function (in the sense described above), except for the increased OPF activity in the MT-null <span class="yellow">mice</span>.
Regardless of its neural basis, the basis of neurobehavioral toxicity should be sought in early life stages when the brain Hg concentration is highly elevated (approximately 1.5 μg/g at its peak). Some in vivo experiments have demonstrated several candidate mechanisms of perinatal exposure to MeHg, including abnormal migration of neurons and/or glias (Kakita et al. 2003; Rodier et al. 1984), but at higher Hg concentrations. Using exactly the same exposure protocol as the present study, we found suppressed activity of type III iodo-thyronine deiodinase, a thyroid hormone-metabolizing enzyme, in the brains of PND10 <span class="yellow">mouse</span> neonates (Mori et al. 2006), consistent with our previous study of higher MeHg doses (Watanabe et al. 1999). This perturbation could be one of the candidate mechanisms responsible for the later anomalous behaviors because even a transient change in thyroid hormones during the critical period of perinatal life exerts long-term consequences (Auso et al. 2004).
The effects of MeHg at 52 weeks of age were influenced by two potential modifying factors, sex and strain. In the OPF, while the locomotion was affected in both strains (although the direction was opposite), center occupancy was significantly increased only in the MT-null <span class="yellow">mice</span>. In addition, the effects on PA were significant only in MT-null <span class="yellow">mice</span>, whereas MeHg at a higher dose was reported to worsen PA performance in <span class="yellow">rats</span> (6–8 weeks of age; Sakamoto et al. 2002). In addition, body weight gain was suppressed in both male and female MT-null <span class="yellow">mice</span>, whereas in wild-type <span class="yellow">mice</span> the suppression was observed only in males. Taken together, the MT-null strain appeared to be slightly more susceptible to the late-emergent effects of MeHg. Several lines of evidence have shown that MT-I,II is protective against the toxicity of MeHg (Leiva-Presa et al. 2004; Yao et al. 2000), and the present results were basically consistent with these reports. We have also reported the susceptibility of the MT-null strain to the neurotoxic effects of metallic Hg (Yoshida et al. 2004, 2006).
The difference in the susceptibility to MeHg between sexes is still debated (Clarkson and Magos 2006; National Research Council 2000; Vahter 2007). In the present study, some responses to MeHg were different between the sexes, including OPF performance at 12 and 52 weeks of age and MM performance at 52 weeks. The fact that the MT-null female group was the only group affected by MeHg at 12 weeks may suggest the particular susceptibility of females in this strain. This point needs to be clarified in further experiments.
The question remains of why MT-null <span class="yellow">mice</span> are susceptible to the delayed neurotoxicity of perinatal MeHg. Apparently kinetics play only a minor role because the strain did not show distinct effects on the brain Hg concentration at PND10. The significantly lower brain Hg concentration in MT-null <span class="yellow">mice</span> compared with corresponding wild-type <span class="yellow">mice</span> at 13 weeks of age indicated that MT-I,II might play a significant role in the retention of Hg (or MeHg). This is consistent with the results of studies of metallic Hg exposure (Yoshida et al. 2004, 2005); a lower brain Hg concentration may not guarantee lower toxicity, supporting the protective role of the protein. Earlier studies suggest that brain MT-I,II has an important role both in the response to oxidative injury (Potter et al. 2007) and in the process of aging (Kojima et al. 1999). Therefore, the lack of MT can exaggerate the toxicity of MeHg by enhancing the initial effects due to oxygen radicals and/or by accelerating functional aging. Apart from this, an intriguing possibility is that the brain-specific isoform, MT-III, might contribute to the results we obtained because the expression of MT-III, together with MT-I, is increased in the brain of old <span class="yellow">rats</span>, resulting in the low availability of free zinc for synapses (Mocchegiani et al. 2004). The age-dependent expressions of MT isoforms might be modified in MT-null <span class="yellow">mice</span>.
Results of the present study might allow the possibility of alternative interpretations due to some potentially confounding factors. For example, except for the wild-type females, we observed significant differences in body weight between MeHg-exposed and non-exposed groups. Because these differences only became clear later in life, they might be associated with the toxicity that also emerged later in life. Manipulation of body weight in rodents alters activity levels, although the reported results are not always consistent with each other (Harrison and Archer 1987; Samuelsson et al. 2008). Also, the differential performance in PA could be related with the potential effects of MeHg on (electric) shock sensitivity, which we did not examine. At least one high-dose study with adult <span class="yellow">rats</span> showed reduced electric sensitivity due to mercury exposure (Wu et al. 1985). These possibilities need to be addressed in future studies.
To summarize, the present results suggest that an initial (or triggering) toxicologic event occurs before the brain Hg concentration stabilizes and that the nature of this event should be either an acceleration of the aging process or interaction with the aging process. Thus, by identifying the physiologic events associated with the functional aging of the examined behavioral tasks, the fundamental toxicologic scar might be revealed.<br><br>
<h3>pmcA2453154</h3>A Novel Role of the NRF2 Transcription Factor in the Regulation of Arsenite-Mediated Keratin 16 Gene Expression in <span class="yellow">Human</span> Keratinocytes
Abstract
Background
Inorganic sodium arsenite (iAs) is a ubiquitous environmental contaminant and is associated with an increased risk of skin hyperkeratosis and cancer.<br><br>Objectives
We investigated the molecular mechanisms underlying the regulation of the keratin 16 (K16) gene by iAs in the <span class="blue">human</span> keratinocyte cell line HaCaT.<br><br>Methods
We performed reverse transcriptase polymerase chain reaction, luciferase assays, Western blots, and electrophoretic mobility shift assays to determine the transcriptional regulation of the K16 gene by iAs. We used gene overexpression approaches to elucidate the nuclear factor erythroid-derived 2 related factor 2 (NRF2) involved in the K16 induction.<br><br>Results
iAs induced the mRNA and protein expression of K16. We also found that the expression of K16 was transcriptionally induced by iAs through activator protein-1–like sites and an antioxidant response element (ARE) in its gene promoter region. Treatment with iAs also enhanced the production and translocation of the NRF2 transcription factor, an ARE-binding protein, into the nucleus without modification of its mRNA expression. In addition, iAs elongated the half-life of the NRF2 protein. When overexpressed in HaCaT cells, NRF2 was also directly involved in not only the up-regulation of the detoxification gene thioredoxin but also K16 gene expression.<br><br>Conclusions
Our data clearly indicate that the K16 gene is a novel target of NRF2. Furthermore, our findings also suggest that NRF2 has opposing roles in the cell—in the activation of detoxification pathways and in promoting the development of skin disorders.<br><br><br><br>Inorganic sodium arsenite (iAs), a ubiquitous element, is one of the most toxic metals present in the natural environment (Bagla and Kaiser 1996). Arsenicals are found as naturally occurring constituents of soil, food, and drinking water (Wu et al. 1989; Yoshida et al. 2004), and exposure to iAs has been associated with a variety of disease outcomes, including disorders of the skin, urinary bladder, liver, and lung (Tchounwou et al. 2004). In particular, skin hyperkeratosis is a characteristic dermatologic lesion associated with ingestion of arsenic from contaminated groundwater (McLellan 2002; Yoshida et al. 2004). There is also a significant association between hyperkeratosis, nonmelanoma skin cancer (e.g., basal cell carcinoma and squamous cell carcinoma), and Bowen disease (Col et al. 1999; Rossman et al. 2004). Furthermore, the pathologic features associated with arsenic-induced hyperkeratosis present as typical acanthotic types of psoriasis-like keratosis, characterized by the aberrant proliferation and terminal differentiation of epidermal keratinocytes (Lee et al. 2006). Many epidemiologic studies have shown that hyperkeratoses are the most frequent precursor lesions of some skin cancers (Bagla and Kaiser 1996; Col et al. 1999).
The keratins are the most prominent cytoskeletal proteins in keratinocytes and comprise a large family of proteins that form intermediate filament networks in all epithelial cell types (Moll et al. 1982). Keratin 16 (K16 ) and keratin 6 (K6) genes are constitutively expressed in a number of stratified epithelial levels, including the palmar and plantar epidermis (Moll et al. 1982). In skin diseases characterized by aberrant proliferation and differentiation, such as psoriasis and cancer, K16 is detectable at higher levels compared with normal tissue (Haider et al. 2006). Furthermore, the tissue-specific overexpression of wild-type K16 in the epidermis of transgenic <span class="yellow">mice</span> results in the hyperproliferation of keratinocytes and aberrant keratinization of cornified layers, leading to hyperkeratosis of the skin (Takahashi et al. 1994).
Nuclear factor erythoid-derived 2 related factor 2 (NRF2), a “cap ’n’ collar” basic leucine zipper transcription factor, regulates a transcriptional program that maintains cellular redox homeostasis and protects cells from oxidative stress and xenobiotic agents (Ishii et al. 2000; Moi et al. 1994). Several detoxifying and antioxidant genes, including glutathione-S-transferases (GSTs), heme oxygenase-1 (HMOX1), and thioredoxin (TXN), are regulated by NRF2 through the antioxidant responsive element (ARE) in the respective promoter regions of these genes (McMahon et al. 2001; Wakabayashi et al. 2004). NRF2 is held in the cytoplasm by a cytoskeletal-associated specific inhibitory protein (kelch-like ECH-associated protein 1; KEAP1) under normal cellular redox state conditions, where it is continuously targeted by the proteasomal degradation pathway (McMahon et al. 2003). Upon exposure of the cell to oxidative stress or electrophiles, NRF2 can escape this KEAP1-mediated repression, translocate to the nucleus, and activate the expression of its target genes (Dinkova-Kostova et al. 2002; McMahon et al. 2003).
Recently, studies of Keap1–/–<span class="yellow">mice</span> have shown that NRF2 accumulates in the nucleus and constitutively activates the transcription of its target genes, even in the absence of stress signals (Wakabayashi et al. 2003). Most interestingly, however, the skin, esophagus, and forestomach of Keap1-deficient <span class="yellow">mice</span> show cornified layer and hyperkeratosis phenotypes. In addition, previous studies have also shown that the expression of NRF2 and ARE-controlled genes is induced by iAs in some cell types (Pi et al. 2003; Sakurai et al. 2005). Furthermore, histochemical analyses have indicated that the expression of K16 is increased in Bowen disease, basal cell carcinoma, and squamous cell carcinoma induced by arsenicals (Yu et al. 1993). However, it remains to be determined whether NRF2 can regulate the transcriptional activation of K16 upon iAs exposure in <span class="blue">human</span> keratinocytes. Hence, these findings prompted us to investigate the molecular mechanisms underlying the regulation of the K16 gene by iAs-induced NRF2 mediation.
Materials and Methods
Chemicals and reagents
A purified preparation of inorganic sodium arsenite (iAs; NaAsO2; Merck, Darmstadt, Germany) was dissolved in phosphate-buffered saline (PBS) and added directly to the culture medium. A fresh iAs solution was prepared for each new experiment. Cycloheximide (CHX), dimethyl-sulfoxide (DMSO), and a protease inhibitor cocktail were purchased from Sigma (St. Louis, MO, USA). CHX was dissolved in DMSO and stored –20°C until use.<br><br>Cells and culture conditions
The <span class="blue">human</span> keratinocyte HaCaT cell line was obtained from N.E. Fusenig (German Cancer Research Center, Heidelberg, Germany). Cells were maintained in monolayer cultures in 95% air and 5% CO2 at 37°C in Dulbecco’s modified Eagles medium (DMEM) supplemented with 10% fetal <span class="yellow">bovine</span> serum (FBS), 50 U/mL penicillin and 50 mg/mL streptomycin and nonessential amino acids (Gibco BRL, Paisley, UK).<br><br>RNA preparation and semiquantitative reverse transcriptase-polymerase chain reaction (RT-PCR) analysis
We determined RNA expression levels by semiquantitative RT-PCR analysis as described previously (Sugioka et al. 2004). Total RNA was isolated from HaCaT cells using the GeneElute Mammalian Total RNA Kit (Sigma). The specific primers used for first-strand cDNA synthesis and PCR were as follows: K16 [forward, 5′-GAT GCT TGC TCT GAG AGG TC-3′, and reverse, 5′-CCA GCA AGA TCT GGT ACT CC-3′; Gene Bank accession no. NM_005557 (National Center for Biotechnology Information 2007)]; c-Jun (forward, 5′-CCT GTT GCG GCC CCG AAA CT-3′, and reverse, 5′-ACC ATG CCT GCC CCG TTG AC-3′; NM_002228); c-Fos (forward, 5′-TTT GCC TAA CCG CCA CGA TGA T-3′, and reverse, 5′-TTG CCG CTT TCT GCC ACC TC-3′; NM_005252); NRF2 (forward, 5′-AGA TTC ACA GGC CTT TCT CG-3′, and reverse, 5′-CAG CTC TCC CTA CCG TTG GA-3′; AF323119); KEAP1 (forward, 5′-CAG AGG TGG TGG TGT TGC TTA T-3′, and reverse, 5′-AGC TCG TTC ATG ATG CCA AAG-3′; NM_012289); TXN (forward, 5′-CAG GGG AAT GAA AGA AAG G-3′, and reverse, 5′-CAA GGT GAA GCA GAT CG-3′; NM_003329), and glyceraldehyde 3-phosphate dehydrogenase (GAPDH) as a loading control (forward, 5′-ACC ACA GTC CAT GCC ATC AC-3′, and reverse, 5′-TCC ACC ACC CTG TTG CTG TA-3′, NM_002046). PCR products were separated on a 1.8% agarose gel and stained with ethidium bromide.<br><br>Western blot analysis
We performed Western blot analysis as described previously (Sugioka et al. 2004). Briefly, nuclear and cytoplasmic proteins were extracted using the NE-PER nuclear and cytoplasmic extraction kit (Pierce, Rockford, IL, USA) according to the manufacturer’s protocol. For protein extraction, the cells were lysed in a buffer containing complete protease inhibitor cocktail. We measured protein concentrations using the DC Protein Assay Kit (Bio-Rad, Richmond, CA, USA). Equal amounts of protein were then resolved by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to a polyvinylidene fluoride membrane (Amersham Biosciences, Bucks, UK). Immunoblotting was carried out with specific antibodies in Tris-buffered saline with 0.05% Tween 20. The primary antibodies were as follows: K16 (Neomarkers, Fremont, CA, USA), NRF2 and KEAP1 (Santa Cruz Biotechnology, Santa Cruz, CA, USA), c-Jun (Cell Signaling, Beverly, MA, USA), and β-actin (Sigma). After washing, the membranes were probed with <span class="yellow">horseradish</span> peroxidase-conjugated secondary antibodies and developed by chemiluminescence using the ECL Plus Detection Kit (Amersham Biosciences).<br><br>Plasmids, transfections, and luciferase assays
<span class="yellow">Human</span> K16 promoter regions of varying lengths (pXK-1, 3, 4, 5-1, and 5-2) were provided by Y-N Wang (National Cheng Kung University, Taiwan). These DNA fragments were prepared from HaCaT cells and were ligated into the pXP-1 luciferase vector (Wang and Chang 2003). The p3xARE/Luc vector, harboring three tandem repeats of ARE, was donated by X.L. Chen (Discovery Research, AtheroGenics Inc., Alpharetta, GA, USA) (Chen et al. 2003). The wild-type NRF2 expression vector (WT-NRF2) was a gift from H.S. So (Wonkwang University School of Medicine, Korea) (So et al. 2006). NRF2 cDNA was subcloned into a pcDNA3.1(+) vector (Invitrogen, San Diego, CA, USA). For the transfection of reporter plasmids, we seeded HaCaT cells into six-well plates at a density of 80% the previous day. Cells were then transfected with a total of each luciferase reporter construct (2.5 μg) using LipofectAMINE plus (Invitrogen). To control for the efficiency of transfection, Renilla luciferase gene expression was monitored using either the pRL-CMV or pRL-TK vectors (Promega, Madison, WI, USA). For overexpression of WT-NRF2, we normalized the total plasmid concentration using the pcDNA3.1(+) empty vector. Thirty-six hours after transfection, the medium was replaced with fresh medium containing either vehicle (PBS) or iAs for 6 hr. After iAs exposure, we harvested cells and analyzed them for luciferase activity using a Dual-Luciferase Reporter Assay System (Promega).
For the investigation of the role of NRF2 in regulating K16 gene expression, transfection of an NRF2 expression plasmid into HaCaT cells was carried out using LipofectAMINE 2000 (Invitrogen). Cells were cultured in 100-mm plates 24 hr before transfection. The expression plasmid WT-NRF2 (15 μg) was then transfected into the cells for 48–60 hr. As a negative control, we used 15 μg of the pcDNA3.1(+) empty vector.<br><br>Electrophoretic mobility-shift assays (EMSA)
We extracted and measured nuclear proteins as described above. Nuclear protein/ DNA complexes were subjected to electrophoresis in nondenaturing 5% polyacrylamide gels containing 2% glycerol in 0.25% Tris-borate/EDTA buffer and transferred to Hybond-N+ nylon transfer membranes (Amersham Biosciences) for detection using the Light-Shift EMSA kit (Pierce) according to the manufacturer’s protocol, with minor modifications. We incubated 10-μg aliquots of nuclear extract with the DNA probe in a binding reaction buffer containing 10 mM Tris/HCl (pH 7.6), 50 mM KCl, 0.5 mM dithiothreitol, 0.25 mM EDTA, 5% glycerol, 2.5 mM MgCl2, 0.05% NP-40 detergent, and 2 μg of poly(dI-dC)·poly(dI-dC) for 30 min at room temperature. For supershift assays, 2 μg of either a polyclonal anti-NRF2 or an anti-c-Jun antibody (Santa Cruz Biotechnology) was added with the nuclear protein for 2 hr at 4°C before the labeled oligonucleotide probe was added. Biotin-labeled, double-stranded oligonucleotides WT-K16ARE (–157/–132, 5′-GGGGAACCTGGAGTCAGCAGT-TAGGA-3′), containing an ARE site (–148/ –140, underlined) in the <span class="blue">human</span> K16 promoter region, and Mut-K16ARE (5′-GGGGAA-CCTGGAGTCAaaAGTTAGGA-3′, mutated GC box in the ARE) were prepared by Fasmac (Kanagawa, Japan). A consensus ARE probe was purchased from Panomics, Inc. (Redwood City, CA, USA). For competition binding of the K16 ARE-complexes, we used an unlabeled AP-1 consensus oligonucleotide (5′-TATC-GATAAGCTATGAGTCATCCGGG-3′). The binding specificity was confirmed in each case by the addition of a 100-fold molar excess of unlabeled oligonucleotide.<br><br>CHX chase experiment
We investigated the posttranscriptional regulation of both the steady-state levels and half-life of the NRF2 protein by CHX chase analysis. Cells were incubated in serum-free medium in the absence or presence of iAs for 6 hr. The culture medium was then replaced with serum-free medium containing CHX (100 μg/mL). We prepared cell lysates at 0, 10, 30, 60, 120, and 240 min after iAs treatment. Whole-cell lysates were resolved by SDS-PAGE and immunoblotted with antibodies against NRF2.<br><br>Statistics
All the data generated from at least three independent experiments and expressed as the mean ± SD were analyzed by the Student’s t-test. Statistical comparisons were made by logarithmic transformation of the normalized values. We considered p-values < 0.01 to be statistically significant.<br><br>
Results
K16 expression is induced by iAs in HaCaT cells
We wanted to determine whether the K16 mRNA is transcriptionally regulated by iAs, and treated HaCaT cells with this compound for various time periods over a range of doses. After treatment of HaCaT cells with 1–20 μM iAs, the expression of K16 mRNA was increased compared with the control at 6 hr (Figure 1A) but had declined to basal levels at 24 hr. The increase in the K16 protein levels after 6 hr of iAs exposure was just detectable at 10–20 μM, but a dose-dependent increase was more evident at 10 hr (Figure 1B). This enhancement of K16 expression had declined to basal levels at 24 hr.<br><br>Identification of the iAs responsive region in the K16 gene promoter
To investigate the mechanisms underlying the transactivation of the K16 gene by iAs, we first examined the response of the K16 regulatory region to this compound using a luciferase reporter gene assay. The dose-dependent activation of K16 transcription after iAs treatment was observed with a construct containing a 515-bp fragment of the K16 promoter (Figure 2A). To further elucidate the region containing the iAs responsive element, we examined a series of deletions of the 5′-flanking region of K16 gene. The ARE sequence in the pXK-5–1 vector contains an activator protein-1 (AP-1)–like element followed by a GC box. As shown in Figure 2B, an enhancement in the reporter activity levels was observed for the promoter constructs, pXK-1, 3, 4, and 5-1, in response to 20 μM iAs. A decline in reporter activity, however, depended on the number of AP-1–like sites, and the results for the pXK-5-1 construct show also that ARE is activated by iAs. In contrast, no significant activation was observed using a pXK-5-2 construct in response to 20 μM iAs.<br><br>Expression of AP-1 transcriptions factor and c-Jun production following iAs treatment
We examined AP-1 transcription factors c-Jun and c-Fos expression in iAs-treated HaCaT cells by semiquantitative RT-PCR. iAs-induced c-Jun expression was observed during the first 3 hr after treatment (data not shown). An appreciable induction of c-Jun was also confirmed after 6 hr, but this was down-regulated by 24 hr after iAs treatment (Figure 3A). In contrast, the expression of c-Fos was only transiently detectable at 3 hr (data not shown) but was not observed during the 6–24 hr period of this experiment. As shown in Figure 3B, iAs-enhanced c-Jun production can be observed in a dose-dependent manner at 6 hr, but it declines from 10 to 24 hr.<br><br>iAs potently induces the translocation of NRF2 and activates the ARE of the K16 promoter
The results of our reporter assays suggested that iAs stimulates not only the AP-1–like sites but also the ARE site within the K16 gene promoter in HaCaT cells (Figure 3). In addition, several oxidative stress agents and toxic chemicals, including iAs, have been reported to induce the expression of ARE-dependent genes in several cell types (Pi et al. 2003; Sakurai et al. 2005). On the basis of our observations and some recent reports, we thus hypothesized that iAs would have the ability to activate the ARE of the K16 gene promoter directly, resulting in the induction of K16 expression in HaCaT cells. To confirm that the K16 ARE indeed functions as an iAs-responsive transcriptional control element, we performed transient transfections of HaCaT cells with a p3xARE/Luc construct and then subjected these cells to iAs for 6 hr. As shown in Figure 4A, treatment of HaCaT cells with iAs results in a dramatic increase in ARE-driven promoter activity. Likewise, EMSA using a consensus ARE probe show that iAs-induced ARE-binding complexes increase markedly, in a dose-dependent fashion (Figure 4B). These results indicate that iAs has the ability to activate the ARE-driven genes. We performed further EMSA experiments using an ARE probe specific to the K16 proximal promoter region (WT-K16ARE) and found that K16ARE–nuclear protein complexes formation is augmented by iAs in a dose-dependent manner (Figure 4C). Moreover, the formation of these complexes is specifically inhibited by the addition of excess unlabeled oligonucleotide competitor (Figure 4B,C), whereas an excess of an unlabeled AP-1 probe competes only marginally for K16ARE binding (Figure 4C).
The NRF2 transcription factor has been shown to bind to AREs upon translocation into the nucleus, resulting in the induction of ARE-mediated genes (Wakabayashi et al. 2004). To examine whether iAs induces and translocates NRF2 into the nucleus in HaCaT cells, we treated these cells with iAs for either 3 or 6 hr. As shown in Figure 4D, a dose-dependent accumulation of NRF2 protein was observed in the nucleus upon treatment with iAs for 6 hr. This was not observed in the parallel experiment performed over the 3-hr time course.
Supershift EMSA analysis using an NRF2 antibody showed that the iAs-induced and iAs-translocated NRF2 protein binds to the WT-K16ARE probe containing the ARE sequence of the K16 proximal promoter region (5′-GGAGTCAGC-3′) that comprises an AP-1–like site and a GC box, whereas the supershift of c-Jun was not observed (Figure 4E). To identify whether the GC box is dispensable for the iAs-stimulated binding activity of NRF2, we next performed EMSA analyses with either WT- or a Mut-K16ARE probe containing an intact AP-1–like element but a mutated GC box. As shown in Figure 4F, the K16ARE–nuclear protein complexes and supershifted bands that were enhanced by iAs treatment were largely abolished by the addition of the Mut-K16ARE probe.<br><br>iAs stabilizes the NRF2 protein
We examined the effects of iAs treatment on the function of KEAP1 in HaCaT cells. Treatment with iAs did not alter the expression levels of KEAP1 mRNA or protein over either a 3 or 6 hr time course (Figure 5A). Next, we examined the effects of iAs on the expression of NRF2 mRNA in HaCaT cells. Exposure to iAs did not significantly alter the steady-state levels of NRF2 mRNA (data not shown). Production of NRF2 protein, however, was observed to increase in both a dose-and time-dependent manner (Figure 4D). To further examine the stabilization of NRF2 protein by iAs, we monitored the decay of basal and iAs-induced NRF2 proteins after inhibition of protein synthesis by CHX (Figure 5B). The results of this analysis revealed that the NRF2 protein levels decrease by approximately 50% within 30 min of treatment with CHX in cells that had not been exposed to iAs. Only trace amounts of NRF2 are then detectable after 60 min of exposure to CHX in these cells. The HaCaT cells were then pretreated with iAs for 6 hr before their exposure to CHX in a similar timecourse experiment. The levels of NRF2 in these iAs-treated cells were again found to decrease by about 50%, but only after 120 min of CHX exposure.<br><br>NRF2 plays a crucial role in the regulation of K16 gene expression in HaCaT cells
To confirm the functional role of NRF2 in the induction of K16 gene expression by iAs, we investigated whether the expression of K16 mRNA is induced by the overexpression of NRF2 (WT-NRF2) in HaCaT cells. We also investigated the expression of the detoxification gene TXN, which is highly induced by a variety of oxidative stimuli through NRF2-mediated ARE transactivation (Kim et al. 2001). The expression of TXN gene in untransfected cells after treatment with iAs was stronger than that of the control cells (Figure 6A). When the cells were transfected with WT-NRF2 and then treated with or without iAs, the expression of TXN mRNA was augmented markedly compared with the empty-vector control. Similarly, the expression of K16 mRNA was also induced in cells transfected with WT-NRF2 in the absence or presence of iAs. We next performed a transient transfection of HaCaT cells with the pXK-5–1 luciferase vector together with the WT-NRF2 vector. The overexpression of NRF2 in increasing concentrations resulted in significant enhancement of the ARE-mediated K16 promoter activation (Figure 6B).<br><br>
Discussion
In the present study, we showed for the first time that iAs induces the transcriptional activation of K16 in the <span class="blue">human</span> keratinocyte cell line, HaCaT, through the ARE present in its gene promoter. It has been reported previously that treatment with iAs enhances the production and translocation of NRF2 into the nucleus in several cell types. However, until now it has remained uncertain whether the induction of NRF2 by iAs mediates the transcriptional activation of the K16 gene in keratinocytes. In our current experiments, we demonstrated that iAs elongates the half-life of the NRF2 protein, which results in its increased expression levels. Furthermore, this iAs-induced NRF2 protein was shown to bind to the ARE sequences in the promoter region of the K16 gene. Finally, by overexpressing NRF2, we have clarified that its induction is involved in not only the gene expression of the detoxification gene TXN, but also in the upregulation of K16 expression in HaCaT cells through the ARE in the K16 gene promoter. These experiments indicate an important and novel function for NRF2 in the regulation of K16 in keratinocytes and also help to further explain the molecular mechanisms underlying arsenic-mediated epidermal hyperkeratosis.
In our present experiments, the expression levels of K16 mRNA and protein were indeed found to be enhanced by iAs in a dose-dependent manner (Figure 1). In addition, luciferase assays of the K16 promoter revealed that iAs enhances its activity in a dose-dependent fashion, which is stimulated by AP-1–like sites and an ARE (Figure 2). The promoter of the <span class="blue">human</span> K16 gene was recently cloned and sequenced, and several AP-1–like sites were found within the –515-bp region of the gene (Wang and Chang 2003). AP-1 transcription factor can be formed by the dimerization of either Jun or Jun/Fos family members (Eferl and Wagner 2003). In the present study, the increased expression of c-Jun, but not c-Fos was evident in the nuclei of HaCaT cells after iAs treatment (Figure 3). Our findings thus suggest that the activation of c-Jun/AP-1 is one of the essential steps in the regulation of K16 gene expression by iAs exposure in HaCaT cells.
It has been well documented that the ARE core sequence includes an AP-1–like binding site (TGAC/GTCA), followed by a GC box (Rushmore et al. 1991; Xie et al. 1995). We have found in our current analyses that the AP-1–like site within the K16 promoter region from –148 to –140 bp (5′-GGAGTCAGC-3′) resembles a consensus ARE sequence. Recent studies have shown that AREs can be specifically bound by complexes of several basic-leucine zipper transcription factors, including NRF2 (Ishii et al. 2000; Moi et al. 1994). NRF2 heterodimerizes with either AP-1 or small MAF (MAFG, MAFK, and MAFF) proteins (MAF, v-maf musculoaponeurotic fibrosarcoma oncogene homolog) and binds to the ARE to induce the transcription of ARE-mediated genes (Motohashi et al. 2002). In the present investigation, EMSA and supershift assays showed that the NRF2 proteins in the nuclei bind to the ARE sequences of K16 promoter region after iAs exposure. iAs-induced c-Jun, however, does not bind to this ARE (Figure 4E). c-Jun may thus act on other AP-1 sites within the K16 promoter region. These results also suggest that other heterodimer partners of NRF2 are involved in the ARE regulation of K16 promoter region underlying iAs-mediated the K16 gene expression. Gel shifts with an K16Mut-ARE probe (harboring a mutation in the ARE GC box) clearly show that the ARE sequence in the K16 promoter, particularly the terminal GC dinucleotide, is essential for mediating iAs-induced K16 transactivation and NRF2 binding (Figure 4F). Several investigations have suggested that the GC nucleotides within the ARE are essential for both the basal and oxidative stress–induced activities of the ARE-related genes, NAD(P)H dehydrogenase quinone 1 (NQO1) and glutamate-cysteine ligase catalytic subunit (GCLC) (Wasserman and Fahl 1997; Wild et al. 1998). Our current results are consistent with these earlier studies in showing that the formation of the iAs-responsive NRF2/ARE complexes is reduced by a mutation in the GC box. Collectively, our present observations reveal a new molecular mechanism in which iAs-induced K16 gene expression is also regulated by activation NRF2/ARE pathways.
It has been widely accepted that oxidative stress disrupts sequestration of NRF2 by KEAP1, permits NRF2 translocation to the nucleus, and transactivates the expression of various NRF2-mediated genes (Dinkova-Kostova et al. 2002; McMahon et al. 2003). Our present study showed that iAs elongates the half-life of the NRF2 protein but has no effects upon KEAP1 expression (Figure 5). Other studies have also demonstrated that the production of NRF2 is increased by various inducers via posttranscriptional control (Nguyen et al. 2003; Stewart et al. 2003). Several earlier reports also indicated that either oxidative stress or antioxidant substances stabilize the expression of the NRF2 protein, either by directly modifying the cysteine residues on KEAP1 to disrupt the NRF2/KEAP1 complex (Dinkova-Kostova et al. 2002) or by facilitating the release of NRF2 through the phosphorylation of the NRF2/KEAP1 complex (Bloom and Jaiswal 2003). These findings are largely consistent with our present finding that iAs stabilizes the expression of NRF2 in HaCaT cells by elongating the protein half-life.
Recently, Wakabayashi et al. (2003) demonstrated that NRF2 accumulates in the nucleus at constitutively high levels and produces various cytoprotective genes in embryonic fibroblast- and liver-derived Keap1-null <span class="yellow">mice</span>. Surprisingly, these Keap1-deficient <span class="yellow">mice</span> also show a thicker stratum corneum epidermis, abnormal keratinization, and cornification in the esophagus and forestomach (hyperkeratosis). K6 was found to be strongly expressed in the esophageal epithelium of these <span class="yellow">mice</span>. These results indicate that K6 is also a target gene of NRF2. In addition, the promoter of the K6 gene bears a remarkable sequence similarity to the K16 promoter (Jiang et al. 1993). Therefore, we examined whether K16 gene expression is also regulated by NRF2. In the present study, the gene expression and transactivation of K16 were dramatically induced by transfection with WT-NRF2 via in HaCaT cells, clearly demonstrating that NRF2 acts as a direct transcriptional regulator of the K16 gene (Figure 6). In addition, we also showed that transfection of HaCaT cells with WT-NRF2 induces the expression of detoxification gene TXN (Figure 6A). NRF2 may thus have a major role to play in the development of hyperkeratosis, whereas the expression and induction of NRF2 is implicated in cell protection against a variety of genotoxic and cytotoxic effects. Hence, based on these results and on the findings from studies of Keap1 knockout <span class="yellow">mice</span>, iAs may both cause hyperkeratosis and induce detoxification enzymes via the modification of NRF2. Given that there are both beneficial and adverse effects of NRF2 activity, caution will therefore be needed when using antioxidants for prevention and therapy. Although further investigations are needed, we believe that our findings provide important clues for the design of future therapies for arsenic-mediated hyperkeratosis and for treatments involving the molecular targeting of NRF2.<br><br>Conclusion
Our findings clearly demonstrate that the induction of the K16 gene in <span class="blue">human</span> keratinocytes by iAs depends on NRF2 activation. Our results thus represent a valuable initial effort to elucidate the relationship between the K16 gene and the NRF2 transcription factor, which may be responsible for hyperkeratosis.<br><br>
<h3>pmcA2526157</h3>Identification and Analysis of Co-Occurrence Networks with NetCutter
Abstract
Background
Co-occurrence analysis is a technique often applied in text mining, comparative genomics, and promoter analysis. The methodologies and statistical models used to evaluate the significance of association between co-occurring entities are quite diverse, however.<br><br>Methodology/Principal Findings
We present a general framework for co-occurrence analysis based on a bipartite graph representation of the data, a novel co-occurrence statistic, and software performing co-occurrence analysis as well as generation and analysis of co-occurrence networks. We show that the overall stringency of co-occurrence analysis depends critically on the choice of the null-model used to evaluate the significance of co-occurrence and find that random sampling from a complete permutation set of the bipartite graph permits co-occurrence analysis with optimal stringency. We show that the Poisson-binomial distribution is the most natural co-occurrence probability distribution when vertex degrees of the bipartite graph are variable, which is usually the case. Calculation of Poisson-binomial P-values is difficult, however. Therefore, we propose a fast bi-binomial approximation for calculation of P-values and show that this statistic is superior to other measures of association such as the Jaccard coefficient and the uncertainty coefficient. Furthermore, co-occurrence analysis of more than two entities can be performed using the same statistical model, which leads to increased signal-to-noise ratios, robustness towards noise, and the identification of implicit relationships between co-occurring entities. Using NetCutter, we identify a novel protein biosynthesis related set of genes that are frequently coordinately deregulated in <span class="blue">human</span> cancer related gene expression studies. NetCutter is available at http://bio.ifom-ieo-campus.it/NetCutter/).<br><br>Conclusion
Our approach can be applied to any set of categorical data where co-occurrence analysis might reveal functional relationships such as clinical parameters associated with cancer subtypes or SNPs associated with disease phenotypes. The stringency of our approach is expected to offer an advantage in a variety of applications.<br><br><br><br>Introduction
Biological research has experienced a paradigm shift in the last decade catalyzed by the availability of genome sequences and the resulting development of high-throughput technologies. The large data volumes produced by these novel technologies are often published as supplementary material and/or stored in extensive data repositories [1]. Functional interpretation of these data is an ongoing challenge. Co-occurrence analysis, based on the hypothesis that co-occurring entities are functionally linked, is a technique that has been used in three main areas of biological research:
Co-occurrence of genes in sequenced genomes relies on the fact that proteins do not function in isolation and are dependent on other proteins, either as direct binding partners, or as catalysts of substrates. Thus, when two proteins significantly co-occur in a large number of genomes or can be observed as fusion proteins in a subset of genomes, they are likely to be binding partners or enzymes needed for a specific metabolic pathway. Examples of those studies have been reported by [2]–[7].
Text mining is a quickly evolving field that aims at developing technologies helping to cope with the functional interpretation of large volumes of publications. Co-occurrence of gene names in publication abstracts, entire publications, or other gene-related databases has been used to derive co-occurrence networks with clear evidence that edges in those networks are reflecting functionally relevant relationships [8]–[11]. Gene names have also been analyzed for co-occurrence with other entities such as mutations [12], chemical compounds [13], and disease related keywords [14]. From the resulting networks, hypotheses about candidate genes involved in inherited diseases and drug targets can be derived. Clustering of gene related publications using keywords has been applied to enhance the quality of gene expression clusters [15], [16]. More general (non gene-centric) approaches try to organize the literature into functional areas based on co-occurrence of MeSH terms, keywords, diseases, phenotypes, chemicals, and similar objects of biomedical research interest [17]–[21].
Co-occurrence analysis of transcription factor binding motifs has been carried out in a variety of slightly differing ways in a wide range of organisms, including <span class="yellow">humans</span>. [22]–[33]. The underlying hypothesis is that co-regulated genes, identified usually by gene expression studies, should contain specific combinations of transcription factor binding motifs in their upstream regulatory regions, the identification of which would allow the reverse-engineering of transcription regulatory networks [34].
We have recently applied co-occurrence analysis to studying published gene expression signatures and showed that co-occurrence patterns of genes reflect cancer signaling pathways [35]. Although co-occurrence analysis has a respectable history, the methodologies used in the studies mentioned above could not be easily applied to studying gene expression signatures. There are three main reasons that dictated the use of a different approach. First, gene expression signatures can vary in size by orders of magnitude. Obviously, the larger a signature the more likely it is to find two or more genes co-occurring in that signature. Thus, the significance of co-occurrences must be evaluated in the presence of considerable heterogeneity of co-occurrence probabilities among gene lists. As a consequence, the statistics used to evaluate the significance of co-occurrence events must reflect this heterogeneity. In particular, it must be based on list-specific co-occurrence probabilities. Second, in the vast majority of previous studies, co-occurrence is analyzed for pair-wise combinations of co-occurring entities. We found that the resulting stringency of this approach is not adequate for the analysis of published gene expression signatures [35]. Third, the null-model against which the significance of co-occurrences is tested does not work well for gene expression signatures. A common procedure is to use generic randomization of the entire data set under analysis or to select subsets of data entries randomly for comparison purposes. However, gene expression signatures are composed of distinct gene sets and the null-model must maintain this property, which is not guaranteed using these approaches. Furthermore, the list-specific nature of co-occurrence probabilities cannot be dealt with properly.
NetCutter was developed to address these challenges and to provide a generic tool for generating and analyzing co-occurrence networks. Although NetCutter has been developed for the analysis of gene expression signatures, it is based on abstract concepts that make it applicable to a wide variety of problems. The input is represented by a bipartite graph that is composed of list-entry pairs, which are stored in tab-separated text format. Co-occurrence of entries in lists is analyzed using pair-wise or higher order combinations of entries. The significance of co-occurrence is tested using a novel bi-binomial approximation of Poisson-binomial statistics (which is a binomial distribution with trial specific probabilities) that handles list-length-heterogeneity properly and provides a novel measure of association that is found to be superior to the Jaccard and the uncertainty coefficients. Occurrence probabilities are obtained from an edge-swapping procedure that maintains vertex degrees in the underlying bipartite graph and distinct sets of entries per list. As we shall see below, this procedure has a number of advantages over other possible null-models and permits co-occurrence analysis with near maximum stringency. Last but not least, NetCutter is equipped with a number of algorithms for the identification of network communities, vertex ranking, and convenience tools needed in the analysis of co-occurrence networks, or any undirected graph. We illustrate the utility of NetCutter in the identification of corresponding clusters of genes and publications from the PubLiME data set. PubLiME (Published Lists of Microarray Experiments) is a repository of published cancer related gene expression signatures (http://bio.ifom-ieo-campus.it/Publime). The concept of cluster correspondence follows from the bipartite graph representation of the data. Reversing the list-entry order in the bipartite graph permits identifying communities of entries as well as communities of lists. We show that communities of publications corresponding to communities of genes in the PubLiME data set can be used to generate hypotheses about the putative function of gene communities.<br><br>Results
The bipartite graph model of co-occurrence analysis
Co-occurrence analysis using NetCutter is based on the abstraction of list-entry pairs. Any entity that co-occurs with some other entity must be confined to some sort of container where co-occurrence is observed. For example, in the case of gene name co-occurrence in PubMed abstracts, the abstract is the container and the gene names are the co-occurring entities. Similarly, co-occurrence of transcription factor binding motifs is observed in gene promoters. The promoters are the containers where motif entities co-occur. The containers generally host more than one entity (otherwise co-occurrence would be impossible) and can be conveniently interpreted as lists. The co-occurring entities are the list entries. Lists and entries form a bipartite graph with one part of the graph representing lists and the other part representing entries. The presence of a given entry in a given list is indicated by an edge between the corresponding list and entry vertices. It is required that each entry can be linked to the same list only once. Without loss of generality, let's consider genes as entries and PubMedID_listIDs as lists in the following, unless otherwise specified (Fig. 1A). This interpretation of lists and entries has been applied in the co-occurrence analysis of published gene expression signatures [35].<br><br>Occurrence probabilities and null-models
A prerequisite for co-occurrence analysis is the availability of occurrence probabilities of genes per list. The occurrence probabilities can be derived from randomizing the bipartite graph and are dependent on the choice of the null-model. A null-model creates an occurrence probability matrix where the occurrence probability for each list–gene pair is listed. As a general property of this matrix, the sum of all matrix elements must equal the number of edges in the bipartite graph. This is because each edge is linked to either side of the bipartite graph with certainty and therefore the sum of occurrence probabilities over all lists (which can be calculated as the row sum if genes are listed vertically or as the column sum if genes are listed horizontally) followed by summing the results over all genes must be 1 for every edge. The number of matrix elements is given by #genes*#lists and therefore the average occurrence probability for any null-model must be #edges/(#genes*#lists). As a consequence, different null-models will only be distinguished by the way they attribute occurrence probabilities to vertices with different vertex degrees but not by the average occurrence probability.
We consider six different strategies to randomize the bipartite graph. First, we could reconnect all edges of the graph randomly. The probability of being connected by an edge for a given list-gene pair is given by (1/#genes)*(1/#lists). Since there are #edges edges to be reconnected, the occurrence probability for a single list-gene pair is #edges/(#genes*#lists), i.e. equal to the average occurrence probability. Thus, this model provides equal occurrence probabilities for all gene-list pairs and does not consider vertex degrees. We call this model the generic randomization (GR) model in the following.
Second, we could disconnect the edges on only the list side of the bipartite graph and reconnect them randomly. The occurrence probability of a gene vertex would be given by (gene vertex degree)/#lists. The sum of these probabilities over all lists is equal to the gene vertex degree and the sum of all gene vertex degrees is equal to the total number of edges. Thus, the sum of all matrix elements is equal to the number of edges, as required. Since this model considers gene vertex degrees, we call it the gene vertex degree (GVD) model.
Third, we disconnect the edges on the gene side of the bipartite graph and reconnect them randomly. The probability of a list vertex being connected to a gene would be given by (list vertex degree)/#genes. The sum of these probabilities over all genes is equal to the list vertex degree and the sum of all list vertex degrees is equal to the total number of edges. Again, the sum of all matrix elements is equal to the total number of edges. Since this model considers list vertex degrees, we call it the list vertex degree (LVD) model.
In model four and five, we reconnect edges considering both gene and list vertex degrees and allow multiple edges between list-gene pairs. The occurrence probabilities in model four are calculated according to the binomial distribution. We calculate the probability of a list-gene pair for being connected as the cumulative binomial probability of the list-gene pair being chosen at least once in the process of randomly reconnecting the edges. This can be achieved by setting the number of trials equal to the gene vertex degree, the probability of success equal to the list vertex degree divided by the total number of edges, and the number of successes equal to 0. The occurrence probability of a list-gene pair is then given by the complement of this probability. This model is called the binomial (BN) model. In model five, we calculate occurrence probabilities according to the hypergeometric distribution. The number of successes in the sample is equal to 0, the sample size is equal to the gene vertex degree, the number of successes in the population is set to the list vertex degree, and the population size is the total number of edges. Again, the occurrence probability of a list-gene pair is obtained as the complement of this probability. We call this model the hypergeometric (HG) model. Calculating occurrence probabilities in this manner does not guarantee that the matrix elements add up to the total number of edges. Therefore, the matrices are normalized such that this condition is satisfied by multiplying each matrix element with the factor #edges/(observed matrix sum), which is generally quite close to 1, however.
In model six, we again consider vertex degrees, but we require that each list is composed of distinct sets of genes. Thus, multiple edges are forbidden. This condition is satisfied by applying an edge-swapping procedure during graph randomization. Edge-swapping works by randomly choosing two list-gene pairs from the bipartite graph and prior to performing the edge-swap, a test is performed to ensure that the two genes are not already linked to the respective target lists. This procedure is performed a large number of times. To ensure complete randomization of the graph, the number of swaps performed should be significantly larger than the number of edges. After performing R randomizations of the graph and counting the number of times a gene has been linked to a particular list, division of this number by R gives the occurrence probability of a gene in a given list. As will be shown below, edge-swapping produces occurrence probabilities that closely approximate occurrence probabilities obtained by generating a complete permutation set of the bipartite graph, counting the number of times a gene is found part of a list, and dividing this number by the total number of permutations. In the permutation model, the sum of occurrence probabilities of a gene over all lists equals the gene vertex degree (see below) and thus the sum of all matrix elements is the number of edges. Since permutation sets of bipartite graphs are difficult to calculate, we use the edge-swapping procedure as a close approximation and call this model the edge-swapping (ES) model.
Fig. 1 shows the occurrence probabilities of the different null-models for the bipartite graph shown in Fig. 1A. The GR model yields identical occurrence probabilities for all list-gene pairs, which is equal to the average occurrence probability in all models. In the other models, the occurrence probabilities deviate to varying extent from the average occurrence probability as a function of vertex degrees. In the GVD model, the deviations are a function of gene vertex degree and in the LVD model the deviations are dependent on list vertex degrees. In the remaining models, the deviations are functions of both the gene and the list vertex degrees. In all cases, larger than average occurrence probabilities are obtained for larger vertex degrees at the expense of smaller than average occurrence probabilities for smaller vertex degrees. From these data, it is difficult to choose the most effective null-model. A hint can be gleaned from gene1, however. Gene1 is present in all lists. Therefore, the co-occurrence probability of gene1 with other genes, which is calculated by multiplying the occurrence probabilities of gene1 and geneX for every list under study, should depend only on the occurrence probability of this other gene. In other words, the occurrence probability of gene1 in all lists should be 1.0. Only two models satisfy this constraint: The GVD and the ES models. Since the GVD model does not consider list vertex degrees, it seems that the ES model is the preferred null-model.<br><br>Expected number of co-occurrences
As a general criterion for comparing the effectiveness of different null-models, we have to compare them for the number of expected co-occurrences. The most effective null-model will be the one that maximizes the expected number of co-occurrences. If the expected number of co-occurrences is larger, an observed number of co-occurrences in a real bipartite graph will be less significant and thus such a null-model permits co-occurrence analysis with higher stringency. The expected number of co-occurrences depends in an obvious fashion on the list vertex degree. If pair-wise co-occurrences are considered, the number of co-occurrences in a list of vertex degree N is given by the binomial coefficient N over 2. Larger lists will give rise to more co-occurrences and the number increases quickly with list vertex degree. The dependency of the expected number of co-occurrences on the gene vertex degree is less obvious and depends strongly on the null-model. A gene that is part of a list with vertex degree N will give rise to N-1 co-occurrences in that list. The null-model permits calculating the probability to find this gene in a given list. Thus, the expected number of co-occurrences of a gene is given by the sum of expected co-occurrences in all lists where for a single list the expected co-occurrences are given by (Nl−1)*pl. Nl is the list vertex degree and pl is the occurrence probability of the gene in that list as determined by the null-model.
We used the PubLiME data set [35] to calculate the expected number of co-occurrences with different null-models. The results are shown in Fig 2A. The expected number of co-occurrences was calculated for all genes in all lists using all null-models and the sum of expected co-occurrences per gene is shown as a scatter plot with the gene vertex degree on the x-axis and the expected number of co-occurrences on the y-axis. The results in Fig. 2A suggest the following ranking of null-models: GR<GVD<LVD<BN = HG<ES. The BN and the HG models perform in an essentially identical way. However, the ES model is the model that yields the largest estimates of expected co-occurrences. The results are also in line with the intuitive expectation that genes with higher vertex degree give rise to more co-occurrences. However, it can be seen that this is not true for all null-models. In particular, it is not true for the GR and the LVD models, which do not consider gene vertex degrees.
As outlined above, it is expected that the null-model that yields the highest estimates of expected co-occurrences should permit co-occurrence analysis with the highest stringency. In Fig. 2B, this hypothesis is tested directly again using the PubLiME data set [35]. For all null-models, co-occurrence analysis was carried out using module size 3 and support 5 (co-occurrence modules must be present in at least five publications). The choice of these parameters has been discussed in [35]. The number of co-occurrence modules was then determined that have a Z-score higher or equal than the cut-off shown in Fig. 2B. The Z-score is calculated from the mean and variance of the Poisson-binomial distribution as shown in the Materials and Methods section and published in [35]. More details on the probability distribution will be provided below. The GR and GVD models perform very poorly and identify large numbers of modules with high Z-scores. The LVD model performs a little better and approximates the BN and HG models at higher Z-score cut-offs. The BN and HG models give essentially identical results. However, the ES model is the model that yields the fewest number of significant co-occurrence modules and is thus the most stringent. The increased stringency of the ES model over the BN and HG models is also reflected in a higher signal-to-noise ratio calculated as the number of significant co-occurrence modules in the real bipartite graph divided by the number of modules found in a randomized bipartite graph (Fig. 2C).
The reason for the superior stringency of the ES model over all other models can be explained by examining the average occurrence probability per gene and list vertex degree. Fig. 2D and E show the average occurrence probability of genes with the same gene vertex degree as a function of the gene vertex degree. It can be seen that the ES model yields higher occurrence probability estimates for genes with higher vertex degrees as compared to the BN and HG models. In GR and LVD models, gene vertex degrees are ignored and occurrence probabilities for genes with large vertex degree are very small, which is compensated by larger occurrence probabilities for genes with small vertex degree. The GVD model is identical to the ES model in this setting. Fig. 2E shows the average occurrence probability of all lists with the same vertex degree as a function of list vertex degree. It can be seen that the ES model provides higher occurrence probability estimates for large lists as compared to the BN and HG models. In this setting, the LVD model performs like the ES model while the GR and GVD models yield small occurrence probabilities for large lists. Since it has been shown above that long lists and genes with high vertex degree are responsible for a large part of the total number of co-occurrences for the most stringent null-models, the null-model that provides larger occurrence probability estimates for genes and lists with high vertex degree at the expense of lower estimates for smaller degrees will be the most stringent because large occurrence probabilities make co-occurrence more likely and thus less significant. By these criteria, the ES model is the most stringent of all models tested.<br><br>The ES model as an approximation of the permutation null-model
The data shown above have revealed that the ES model is the best of the models tested. One may wonder, however, whether yet more effective null-models can be found. An obvious choice would be the permutation model. In the permutation model, a complete permutation set of the bipartite graph is created such that each list is composed of distinct sets of genes. The number of graphs where a gene is present in a given list divided by the total number of permutations then provides the occurrence probability estimate. The permutation model is the ideal null-model because it is exhaustive. The problem is that a complete permutation set of bipartite graphs of some complexity is very time consuming to calculate. For example, the simple bipartite graph from Fig. 1A is part of a permutation set of 455 graphs. The number of permutations is increasing quickly as the numbers of genes and lists grow. However, since edge-swapping ensures that gene lists are composed of distinct sets of genes, each edge-swap produces a graph that is part of the permutation set of the bipartite graph. Edge-swapping can thus be viewed as a random sampling procedure from the permutation set of the bipartite graph. Therefore, occurrence probability estimates derived by edge-swapping should approximate those obtained from the permutation model.
We generated a complete permutation set of the graph shown in Fig. 1A to verify this hypothesis. The results are shown in Fig. 3. Fig. 3A shows how the number of possible permutations can be calculated. Gene1 is present in all lists and does not have an impact on the total number of permutations. Gene2, having vertex degree two, is present in two out of three lists in one out of three possible ways. The remaining genes have vertex degree 1 and can be freely chosen to fill the empty slots. We can now count exactly how many times a gene is linked to a list and divide these counts by 455, the size of the permutation set, to obtain exact occurrence probabilities. These numbers are shown in graphical form in Fig. 3B and in numerical form in Fig. 3C. Fig. 3B also shows the occurrence probability estimates obtained by edge-swapping side-by-side to the exact occurrence probabilities. The graph in Fig. 1A was subjected to edge-swapping 1000 times and the number of times a gene was found present in a list was divided by 1000 to obtain the occurrence probability. At each run, 100 random edge swaps were performed to ensure complete randomization of the graph. This procedure was repeated 10 times and the mean and standard deviation of occurrence probability estimates for each gene in each list are shown. In all cases, the mean differs from the real probability by less than two standard deviations, in most cases by less than one standard deviation. Thus, edge-swapping provides reliable estimates of exact occurrence probabilities as determined from a complete permutation set.
As an interesting observation, we provide evidence that occurrence probabilities are non-linear functions of vertex degrees in the edge-swapping model. This is illustrated in Fig. 3C. Individual and average occurrence probabilities are shown as a function of gene and list vertex degrees. Non-linearity of individual occurrence probabilities can be verified from the counts table underneath the plots. However, the average occurrence probability is found to depend on vertex degrees in a linear fashion instead. This is a consequence of the fact that occurrence probabilities of a gene over all lists add up to the gene vertex degree and that the occurrence probabilities of all genes for a given list add up to the list vertex degree. At the same time, since the most stringent permutation based null-model predicts non-linear dependencies of individual occurrence probabilities on vertex degrees, assuming such linearity in statistical models of co-occurrence will be linked to loss of stringency.
We conclude that the ES null-model is the null-model that permits co-occurrence analysis with the highest stringency among the models tested and that it closely approximates occurrence probabilities derived from an ideal permutation model. The increased stringency of the ES model over other models is a consequence of higher occurrence probabilities for genes and list with high vertex degrees, which are giving rise to a large part of all co-occurrences in the bipartite graph. Since large occurrence probabilities make co-occurrence more likely, the analysis becomes more stringent.<br><br>Co-occurrence probabilities
Co-occurrence analysis can be thought of as a Bernoulli experiment with a binomial outcome (a given combination of entries is either present or not present in a given list). Thus, the Binomial distribution (BD) is a natural choice for judging the significance of the number of co-occurrences. However, the BD is defined for a probability of success which is equal in all trials. The list-specific nature of occurrence probabilities is not compatible with this condition (analysis of each list represents one trial), which means that co-occurrence analysis in the presence of list-length-heterogeneity is better described as a series of Poisson trials, where the probability of success varies from trial to trial. Therefore, the significance of co-occurrences must be evaluated using a binomial distribution with trial-specific probabilities, i.e. the Poisson-binomial distribution (PBD). The probability of success in a single Poisson trial can be calculated by multiplying the list-specific occurrence probabilities for the combination of genes under study. The number of occurrence probabilities that need to be multiplied is equal to the module size, i.e. the number of genes whose combination is studied. An observed number of co-occurrences for a combination of genes can then be evaluated using the PBD, which is given by the formula [36]:(18)
The structure of this formula is very similar to the structure of the formula used to calculate the binomial distribution, except that multiplication with a binomial coefficient is replaced by summation over individual terms, which makes calculation of P-values using (18) inefficient (note that equation numbering starts in the Materials and Methods section). Here, Ak denotes the kth set of indices of the i lists where genes are co-occurring (“success”). There are  possible sets and summation is carried out accordingly.  denotes the set of indices of N−i lists where genes are not co-occurring (“failure”). [36] have reported two fast procedures for calculating exact PBD P-values. However, both procedures work with probability ratios and suffer from numerical overflow/underflow problems for large numbers of trials. NetCutter uses two workarounds to circumvent this problem. One is based on using Poisson-binomial Z-scores, which can be calculated very easily instead (see below). The other relies on a fast approximation procedure for calculating Poisson-binomial P-values, which we call bi-binomial approximation (BBA) or bi-binomial distribution (BBD).<br><br>Z-scores and P-values of BBD
Given the mean μ (1) and variance σ2 (2) of PBD (see Materials and Methods), the Z-score associated with a given number of co-occurrences x is obtained as:(19)
Considering the structure of formulae (1) and (2) (Materials and Methods section), PBD Z-scores can be calculated very easily and provide a simple estimate of the significance of co-occurrence modules. However, in contrast to normally distributed Z-scores, binomial and Poisson-binomial Z-scores do not correspond to the same P-value for different sets of probabilities of success. To see this, calculate for example the probability of success in a series of 100 Bernoulli trials with success probability 0.1 and 0.9 for the expectation of 10 and 90 successes, respectively. The Z-score will be 0 in both cases but the corresponding cumulative P-values are 0.5832 and 0.5487. Therefore, exact levels of significance cannot be derived from Z-scores alone. Thus, a fast and reliable procedure for calculating Poisson-binomial P-values is needed. The BBD approximation was developed to solve this problem.
The BBD approximation of PBD P-values follows from the relationship between the variance of PBD and the population variance of trial-specific probabilities of success. This relationship is shown in Materials and Methods to be described by (4):(4)
This equation shows that there is an inverse linear relationship between the population variance S2 of the N trial probabilities and the variance of PBD σ2, which means that PBD becomes increasingly narrow as the variance of trial probabilities grows. It also shows that, for constant mean μ and number of trials N, the shape of PBD depends only on the variance of trial probabilities. Therefore, relationship (4) suggests an easy way to approximate PBD P-values. The P-value can be obtained by constructing a set of trial probabilities with equal variance as the original set of trial probabilities, which, however, are not all different. In other words, the series of Poisson trials can be replaced by two sets of Bernoulli trials with trial probabilities p1 and p2 constructed such that the variance is equal to the original set of trial probabilities. This strategy is illustrated in Fig. 4 and explains why this approximation is called bi-binomial. The details on how to obtain the values of the two sets of Bernoulli trial probabilities and the number of trials with p1 and p2 as probabilities of success are provided in the Materials and Methods section. The precision of the BBD approximation is discussed in supplementary material Simulation S1.
In order to evaluate whether BBD P-values as a significance measure of co-occurrence offer an advantage over other measures such as the Jaccard coefficient or the uncertainty coefficient, pair-wise co-occurrence of two genes in 200 lists with and without list-length-heterogeneity was studied (Fig. 5). Each gene is assumed to occur in 100 lists. Therefore, the occurrence probabilities of both genes over all 200 lists must add up to 100, regardless of list-length-heterogeneity. For simplicity, occurrence probabilities of both genes are assumed to be equal in any particular list. The co-occurrence probability in a list is then given by the square of the occurrence probability in that list. For all possible co-occurrences from 0 to 100, the Jaccard and uncertainty coefficients were calculated as detailed in the Materials and Methods section. In addition, cumulative BBD P-values were calculated using the co-occurrence probabilities as trial probabilities. To illustrate the advantage of BBD over BD as co-occurrence probability distribution, cumulative BD P-values of a BD with the same mean as BBD but constant trial probabilities is shown. These trial probabilities can be obtained by dividing the mean of BBD by the number of lists.
Three different cases of list-length-heterogeneity are considered in Fig. 5: No heterogeneity (standard deviation 0), heterogeneity with standard deviation 0.283 and heterogeneity with standard deviation 0.401 in the occurrence probabilities. The Jaccard and uncertainty coefficients are by definition insensitive to list-length-heterogeneity because differences in co-occurrence probabilities in a given list cannot be considered in their calculation. This is because both coefficients are defined by the counts of the four list classes: both genes absent, both genes present, first gene absent second gene present, and first gene present second gene absent, i.e. by the corresponding contingency table, which does not change with different list-length-heterogeneity. In the absence of list-length-heterogeneity, the cumulative P-values of BD and BBD (which are perfectly overlapping as expected) assume 0.5 at 50 co-occurrences, which corresponds to the expected number of co-occurrences calculated as (50 = 100 occurrences per gene/200 lists) ˆ2*200 lists. The uncertainty coefficient is found to be 0 and the Jaccard coefficient is 0.33333 at that point. When there is modest list-length-heterogeneity (standard deviation 0.283), the mean of BBD is shifting to the right. This is because the sum of squares of varying occurrence probabilities (i.e. the sum of co-occurrence probabilities used as trial probabilities, which is equal to the mean of BBD) is always larger than the sum of squares of constant occurrence probabilities with the same average occurrence probability (0.5). The corresponding BD in the presence of list-length-heterogeneity is obtained by dividing the expected number of co-occurrences by the total number of lists, which means assuming equal co-occurrences in all lists. This visualization is shown to illustrate how BBD (which is narrower than the corresponding BD) gives rise to a steeper cumulative distribution of P-values and as a consequence to more significant P-values for numbers of co-occurrence that are far from the expectation. As the level of list-length-heterogeneity grows (standard deviation of occurrence probabilities 0.401), the mean of BBD is shifted even further to the right and BBD P-values are distributed in a still steeper fashion as compared to corresponding BD P-values and the interval of non-significant co-occurrences is shrinking further. With modest list-length-heterogeneity, the expected number of co-occurrences is 66, which is associated with a Jaccard coefficient of 0.49 and an uncertainty coefficient of 0.075. In the case of large list-length-heterogeneity, the expected number of co-occurrences is 82 with J = 0.69 and UC = 0.32.
Taken together, these data show that the expected number of co-occurrences varies strongly with the level of list-length-heterogeneity and that the expected number of co-occurrences is associated with different values of UC and J. To complicate matters further, 66 co-occurrences (J = 0.49, UC = 0.075) represent significant positive association (PBBD = 0.996) with equal list lengths, no significant association with modest differences in list length (PBBD = 0.536) and strongly negative association (meaning one gene excludes the other) with strong list-length-heterogeneity (PBBD = 0.00016). Thus, the same J and UC association measure is obtained for positive, negative, and absence of association. Therefore, the meaning of these measures cannot be interpreted properly in the absence of knowledge about the occurrence probabilities of the co-occurring entities. Furthermore, the data in Fig. 5 also show that neither J nor UC can distinguish between positive and negative association while this is easy with cumulative BBD P-values: Large P-values mean positive association and low P-values mean negative association. In summary, we conclude that BBD provides a novel association measure that offers a number of advantages over the existing contingency table based association measures Jaccard coefficient and uncertainty coefficient. The results in Fig. 5 also show that significance of association depends critically on the specific distribution of co-occurring entities over lists of varying length (because this distribution determines the occurrence probabilities) and that contingency table based methods (which cannot capture this distribution) should be avoided in the presence of significant list-length-heterogeneity.<br><br>Generation of co-occurrence networks and the identification of communities
The procedures outlined above allow the identification of significant co-occurrence modules in any type of bipartite graph. Three user defined parameters have an impact on the stringency of co-occurrence analysis: The module size, the support, and the Z-score/P-value cutoff. The module size determines how many entries will be tested for co-occurrence, the support sets a lower boundary on the required number of co-occurrences, and the Z-score/P-value cutoff sets the significance threshold. In general, higher module size leads to more stringent co-occurrence analysis at the cost of computational complexity. The support parameter allows limiting this complexity by filtering out co-occurrence modules which co-occur less frequently than required by the support. The significance cutoff permits adjusting the signal-to-noise ratio, which is calculated as the number of co-occurrence modules observed in the real versus a randomized bipartite graph. The impact of these parameters on the stringency of co-occurrence analysis has been reported previously for the PubLiME data set [35] and is illustrated in a simulation study provided as supplementary material Simulation S1. From the set of significant co-occurrence modules, a co-occurrence network is generated by considering each entry a vertex and drawing an edge between any two vertices, which have been part of the same significant co-occurrence module [35].
An important question in the analysis of co-occurrence networks regards the presence of network communities. Communities can be understood as groups of vertices with the property that the number of edges running within groups is larger than expected by chance and that the number of edges running between groups is lower than expected by chance [37]. This problem of partitioning a graph is often referred to as the graph-cut problem (hence the name NetCutter). NetCutter is built on the Java Universal Network and Graph framework (JUNG) software package (http://jung.sourceforge.net), which provides algorithms for solving this problem. In particular, NetCutter implements the Bicomponent clustering algorithm [38], the Edge-Betweenness clustering algorithm [39], and the Exact Flow Community algorithm [40]. Furthermore, there is a clustering tool that is not part of the JUNG package, namely an algorithm identifying communities using eigenvectors of the modularity matrix [37]. The code for this algorithm was kindly provided by Mark Newman in C++ and ported to Java. In addition to these tools, NetCutter provides a number of convenience functions for the analysis of co-occurrence networks, such as testing the significance of lists reporting a set of entries making up a network community, ranking of vertices, random graph generators for topological analysis of co-occurrence networks, and others. Details on all functions are provided in the NetCutter software documentation.
One of the possible applications of NetCutter is illustrated below. This application is tightly linked to the bipartite graph representation of the data. Namely, NetCutter can be used to perform co-occurrence analysis of genes or list derived from the same bipartite graph. The network communities identified in each both reflect the same underlying structure of the bipartite graph. In the case of gene expression signatures stored in PubLiME, clusters of genes correspond to clusters of publications, which can reveal possible functions of gene clusters.<br><br>Cluster correspondence and association studies
The co-occurrence analysis of the PubLiME data set published previously [35] identified 5 major network communities of genes with consistent functional annotations that are deregulated in cancer related gene expression signatures. This analysis was performed by considering all genes mentioned in a particular publication as a single signature, even though they might have been part of different tables and cluster analyses. Here we present an advanced analysis of the PubLiME data set where each table and/or cluster identified in a given publication is considered as a separate signature. This brings the total number of signatures to be analyzed to 1015 comprising a total of 7358 differentially regulated genes derived from 233 publications reporting cancer related signatures derived from <span class="blue">human</span> samples. We use this analysis to illustrate three major points: First, the set of communities reported previously is reproduced by this more fine-grained analysis. Second, the set of gene communities corresponds to a set of publication communities. Third, associations between publications and gene communities can be calculated with higher stringency using the edge-swapping null-model in conjunction with bi-binomial P-values as compared to binomial or hypergeometric statistics.
The bipartite graph to be analyzed is composed of PubMedID_listID-gene pairs (see supplementary material Table S1). Co-occurrence analysis was carried out in two ways: First, gene co-occurrence was analyzed and communities of co-occurring genes were defined by edge-betweenness clustering as described in Materials and Methods. Second, co-occurrence of PubMedID_listIDs was analyzed. To this end, the order of PubMedID_listID-gene pairs was reversed to form GENE-PUBMEDID_LISTID pairs. Thus, the lists in the resulting bipartite graph are formed by genes and the entries are the PubMedID_listIDs where the genes are reported as differentially regulated. Occurrence probabilities for the reversed bipartite graph can be obtained by transposing the occurrence probability matrix of the original bipartite graph. Since the gene communities identified in gene co-occurrence analysis reflect the structure inherent in the bipartite graph (which is not affected by reversing the list-entry order), co-occurrence analysis of the reversed bipartite graph will result in PubMedID_listID communities that reflect the same underlying structure in the bipartite graph. In other words, PubMedID_listID communities correspond to gene communities. In less abstract terms, the PubMedID_listID communities should correspond to sets of publications that report similar sets of genes as differentially regulated. The identification of communities of publications can help the researcher to easily identify publications studying genes in a gene community that is of interest to the researcher.
The results of both types of co-occurrence analysis are displayed in Fig. 6. Fig. 6A shows the gene clusters identified. The clusters are named after significant enrichments of gene categories as determined by functional category enrichment using DAVID [41]. The P-values shown are Benjamini corrected for multiple testing as reported by DAVID. The clusters are very similar to the clusters published previously [35]. There is one new cluster that is strongly enriched for ribosomal proteins (“protein biosynthesis” cluster), which has not reached significance in our previous analysis. The “surface antigen” cluster contains many genes that had been reported as part of the “signal transduction” cluster. Altogether, however, these results strongly support the notion that the gene clusters in the PubLiME data set can be reproduced by the more fine-grained analysis that considers sublists in each publication as separate signatures.
The corresponding clusters of PubMedID_listIDs are shown in Fig. 6B. There are five clusters, which have been named after their corresponding gene cluster. Only one cluster (the “extracellular matrix-immune response cluster”) cannot be separated by edge-betweenness clustering at the point of maximal graph modularity. To see that this naming is indeed justified, we needed to investigate how strongly a given PubMedID_listID is associated with a given gene cluster, i.e. how significant is the overlap of the genes reported in a gene cluster and the genes reported in a PubMedID_listID. Binomial or hypergemetric statistics are generally used to calculate this significance. However, the bipartite graph model in conjunction with the edge-swapping null-model offers a more fine-grained approach based on bi-binomial statistics.
The edge-swapping null-model determines occurrence probabilities in such a way that the number of genes in a given PubMedID_listID is associated with insignificant P-values in the context of the complete bipartite graph. However, when a subset of genes is analyzed, e.g. all the genes that are reported in a particular list, the P-value associated with the number of genes contained in this list will likely be highly significant according to how unlikely it is to obtain all the genes contained in a given list in a random draw from all genes present in the bipartite graph. Thus, PubMedID_listID association with a set of genes in the bipartite graph model can be calculated in the following way: The set of genes that is used to analyze association is used to extract a subgraph from the original bipartite graph where occurrence probabilities for each gene-PubMedID_listID pair are identical to those in the original bipartite graph (i.e. they are not recalculated by edge-swapping). The vertex degree of the PubMedID_listID vertices in the subgraph indicates the number of genes contained in each PubMedID_listID overlapping with the set of genes used to extract the subgraph. From the occurrence probabilities of the genes in a given PubMedID_listID, the bi-binomial P-value can then be calculated for every list vertex degree observed in the subgraph. In Fig. 7, the significance of association of the PubMedID_listIDs (see Fig. 6B) with the cell cycle cluster of genes (Fig. 6A) is calculated. For comparison, binomial and hypergeometric P-values are also shown. It can be seen that the bi-binomial P-value is larger than the binomial and hypergeometric P-values, which means that the strength of association is evaluated in a more stringent manner using BBD statistics (see Discussion for an explanation of this observation).
The analysis of significant associations between PubMedID_listIDs and gene clusters now permit answering the question whether there is correspondence between gene clusters and PubMedID_listID clusters. The naming of PubMedID_listID clusters shown in Fig. 6B is based on the number of PubMedID_listID that are significantly associated with gene clusters shown in Fig. 6A. First, for each gene cluster, all the PubMedID_listIDs that are associated with that cluster with more than 95% confidence (i.e. cumulative bi-binomial P-values> = 0.95) were identified. Second, the number of significant PubMedID_listIDs in each PubMedID_listID cluster was counted for every gene cluster. The significance of this number was then calculated using binomial statistics. The results of this analysis are shown in Table 1. Negative decadic logarithms of the binomial P-value are displayed. It is apparent that each PubMedID_listID cluster is strongly associated with at least one gene cluster, except for the “extracellular matrix-immune response” cluster, which is associated with two gene clusters. The strength of these associations suggests that the PubMedID_listID clusters are indeed corresponding to the gene clusters and that both the gene and the PubMedID_listID clusters reflect the structure of the bipartite graph representing the PubLiME data set.
Details about all the lists analyzed are attached as supplementary material Table S2. Looking at these lists, some general conclusions about the gene clusters can be drawn. Cell cycle cluster genes have been found deregulated in a wide variety of tumor types such as colon cancer, breast cancer, in biliary tract cancer, pancreatic cancer, gastric cancer, prostate cancer, T-cell leukemia, glioma, acute lymphoblastic and myeloblastic leukemias, soft tissue sarcoma, neuroblastoma, as well as in a number of cellular model systems in response to different stimuli. Thus, the cell cycle cluster seems to consist of genes with a general role in oncogenesis. The surface antigen cluster instead seems to be derived preferentially from studies on leukemia. The interferon cluster genes are found deregulated in virus induced pathologies such as papilloma virus induced cervical cancer, and viral hepatitis. Immune response cluster genes were reported as differentially regulated in inflammatory conditions such as ulcerative colitis, Crohn's disease, and <span class="yellow">Helicobacter pylori</span> infections. Genes of the extracellular matrix cluster seem to be associated with cancer progression studies and metastatic potential. For the protein biosynthesis cluster, there are 15 signatures that are significantly enriched for those genes. The cancers studied comprise medulloblastoma, glioblastoma, pancreatic cancer, soft tissue sarcoma, lung carcinoma, breast carcinoma, prostate carcinoma, multiple myeloma, and lymphocytic leukemia. The genes are also found deregulated in response to DNA damage. Although the number of signatures is limited, the variation in conditions where the genes are deregulated is compatible with the hypothesis that protein biosynthesis genes, as cell cycle genes, are deregulated in many cancer types, which might reflect the general property of cancer cells to divide and grow in an uncontrolled fashion.<br><br>
Discussion
Here we have investigated basic aspects of co-occurrence analysis and present a software tool, NetCutter, which can be used to identify and analyze generic co-occurrence networks. In NetCutter, a co-occurrence data set is represented as a bipartite graph with one part representing lists and the other part list entries whose co-occurrence patterns are studied. The bipartite graph representation of co-occurrence data sets allows the efficacy of different null-models to be tested systematically. We have shown that an edge-swapping procedure used to randomize the bipartite graph generates a null-model that allows co-occurrence analysis with the highest stringency. The other null-models tested here tend to underestimate occurrence probabilities of entries per list for lists and genes with high vertex degrees, i.e. for lists and genes where most co-occurrences are observed. As a result, co-occurrences are judged more significant than they really are.
Co-occurrence data sets with exactly equal lists lengths are likely to be the exception from the rule. It can be assumed that some list-length-heterogeneity will be present in most circumstances. An important consequence of list-length-heterogeneity regards the co-occurrence probability distribution used to evaluate the significance of the observed number of co-occurrences. Co-occurrence analysis in the presence of list-length-heterogeneity is best performed using the Poisson-binomial distribution (a binomial distribution with trial specific probabilities). However, calculating Poisson-binomial P-values for large numbers of lists is difficult using existing procedures [36]. We have presented an approximation to the Poisson-binomial distribution, called bi-binomial distribution, which is based on replacing the set of Poisson trials by two sets of Bernoulli trials. The resulting distribution reproduces the Poisson-binomial distribution nearly exactly and its P-values can be calculated with ease even for thousands of lists (see also supplementary material Simulation S1 for details on the precision of BBD). Importantly, BBD provides a novel measure of association, which is shown to be superior to existing measures such as the Jaccard coefficient and the uncertainty coefficient, whose values cannot be interpreted properly in the absence of knowledge about the occurrence probabilities of co-occurring entities.
It is worth noting that Poisson-binomial Z-scores are distinguished from Gaussian Z-scores by the fact that they do not correspond to the same P-value for different PBDs, BBDs, and even BDs. This is because the Z-score is an explicit part of the function defining the normal probability density while it is not part of the definitions of BD, PBD, and BBD densities. As a consequence, the simple Poisson-binomial Z-score based approach to evaluating significance of co-occurrence must be complemented with the BBD to approximate Poisson-binomial P-values in order to enable multiple testing corrections and to allow calculation of confidence levels in association studies precisely. However, NetCutter is equipped with a bipartite graph randomization tool that permits measuring the number of false positives due to multiple testing directly by comparing the number of significant co-occurrence modules in the real bipartite graph to the corresponding number in a randomized version thereof. Randomization is performed by edge-swapping in order to preserve vertex degrees. The resulting signal-to-noise ratios that are plotted for each Z-score/P-value cutoff provide a highly reliable and visually intuitive defense mechanism against false positives (see also supplementary material Simulation S1).
In the vast majority of co-occurrence studies, pair-wise co-occurrences have been analyzed using different statistical models. We have observed that the stringency of pair-wise co-occurrence analysis is far below the stringency of co-occurrence analysis using higher order combinations of co-occurring entities [35]. In NetCutter, co-occurrence analysis is preceded by occurrence analysis, i.e. the occurrence probability of each entry in each list is determined. Starting from occurrence probabilities, co-occurrence probabilities for any size of co-occurrence modules under study can be obtained by multiplying the respective list-specific occurrence probabilities. Given the list-specific co-occurrence probabilities, bi-binomial P-values are then calculated in exactly the same way for any module size. As a consequence, NetCutter can perform co-occurrence analysis for higher order combinations of co-occurring entries (i.e. larger module sizes) using the same statistical model. One of the benefits of using higher module sizes is robustness of the analyses in the presence of noise. This is because each edge in the resulting co-occurrence network is evaluated many times since every pair of co-occurring entries can be part of many higher order co-occurrence modules [35]. Another advantage is that implicit relationships between entries, which have never occurred together [18], can be derived as a natural by-product of using module sizes larger than 2. As shown in a simulation study (supplementary material Simulation S1), the result is a dramatic reduction of misclassifications at higher module sizes.
NetCutter can be used to calculate the strength of association between a subset of entries and lists reporting those entries. In this case, the analysis is performed on a subgraph of the original bipartite graph. The subgraph can correspond to communities of entries in the co-occurrence network, or any set of entries of interest. NetCutter will then calculate the significance of observing a given number of occurrences of an entry in the user defined subset of lists using bi-binomal statistics. This analysis mode corresponds to association studies with the advantage that the structure of the underlying bipartite graph (i.e. list length heterogeneity) is considered and handled appropriately using the bi-binomial distribution. As a consequence, association studies can be performed with higher stringency.
This result can be understood by examining the occurrence probability matrix that is implicitly assumed in performing binomial or hypergeometric tests for the significance of overlaps. In both tests, a gene is assumed to have an equal opportunity to be present in a list. Therefore, the probability of success for a gene to be part of a list is given by the list vertex degree divided by the total number of genes. In other words, both tests are implicitly based on the list vertex degree model, which has been shown previously to underestimate the occurrence probability and the expected number of co-occurrences for genes with high vertex degree (see Fig. 2A). Since the BBD P-values are calculated from the ES-model, which assigns higher occurrence probabilities to genes with higher vertex degree and more expected co-occurrences, the observed number of overlaps between a set of genes of interest and the content of a given list (which can be viewed as co-occurrence of the overlapping genes in that list) will be judged less significant when the overlapping genes are of high vertex degree (and vice versa when the overlapping genes are of low vertex degree) as compared to binomial or hypergeometric tests. Since the BBD P-values are derived from the most stringent ES null-model, BBD P-values provide a more reliable estimate for the significance of overlap.
Co-occurrence analysis of data represented as bipartite graphs permits visualizing the structure of the bipartite graph either as communities of list entries (genes) or as communities of lists (PubMedID_ListID) in co-occurrence networks. We have analyzed the PubLiME data set for the presence of corresponding gene and list clusters. In addition to previously published clusters of genes, we describe a novel gene cluster that is composed of protein biosynthesis associated genes [35]. We found that the corresponding clusters of PubMedID_ListID (gene expression signatures) are in general strongly enriched for genes reported in the corresponding gene cluster and that interrogation of corresponding clusters can be used to deduct hypotheses about the putative function of gene clusters.
In addition to co-occurrence analysis, NetCutter offers a number of tools for the analysis of co-occurrence networks, or any undirected graph. In particular, community identification is supported by four different community identification algorithms. NetCutter also offers a range of convenience functions that are of help in network analysis. Worthy of mentioning are the random graph generators that can provide control graphs for topological studies. The complete set of options is described in the software documentation.
In summary, we present a general framework for co-occurrence analysis with many potential applications. We illustrate a number of advantages of using the bipartite graph representation of data and the associated statistics. In particular, the identification of corresponding clusters permits the identification of functional subunits such as gene clusters on the one hand, and the generation of hypotheses about the function of those units by analyzing the corresponding list clusters on the other hand. Future developments will be directed towards the analysis of data sets that are considerably larger than the data sets analyzed so far. For example, co-occurrence analysis might be of interest for the analysis of single nucleotide polymorphism (SNP) data sets and association studies of genome variability with disease. Each <span class="yellow">patient</span> is characterized by a specific range of SNPs. Co-occurrence patterns of <span class="yellow">patients</span> according to their SNPs could be compared to clinical parameters with the aim of identifying genomic regions associated with disease. The increased stringency of association studies offered by NetCutter may be of use in the analysis of polygenic diseases where conventional methods fail. For being useful in this setting, NetCutter must be capable of analyzing bipartite graphs with millions instead of thousands of vertices.<br><br>Materials and Methods
Implementation of NetCutter
NetCutter is written in Java using NetBeans6 software (http://www.netbeans.info/) and tested on the Java Runtime environment 1.6.0.0. on a Windows XP Professional computer. The Java Runtime environment, which can be downloaded from http://java.sun.com/, must be installed on a computer that is intended to run NetCutter. NetCutter is provided as a single jar file and should run by double clicking the jar file, provided that the Java runtime environment is properly installed. NetCutter makes use of the following software packages and classes: JUNG version 1.3 (http://jung.sourceforge.net/download.html), Apache Jakarta Commons Collections 3.1 (http://jakarta.apache.org/commons/collections/), Cern Colt Scientific Library 1.2.0 (http://dsd.lbl.gov/hoschek/colt/), Xerces (http://xerces.apache.org/xerces2-j/index.html), Jama (http://math.nist.gov/javanumerics/jama/), Netlib Java LAPACK (http://www.netlib.org/lapack/), JFreeChart (http://www.jfree.org/jfreechart/), partition.java (http://astro.u-strasbg.fr/fmurtagh/mda-sw/java/partition.java).<br><br>Bi-binomial approximation of Poisson-binomial distribution
The Poisson-binomial distribution (binomial distribution with trial specific probabilities) has recently been proposed as a statistic that properly handles largely differing sizes of gene expression signatures in meta-analysis of gene expression data [35]. Z-scores have been used to estimate the significance of co-occurrence because P-value calculation is cumbersome and error prone. Two methods reported by [36] suffer from numerical overflow/underflow problems when large numbers of Poisson trials with probabilities deviating significantly from 0.5 are being analyzed. Therefore, we propose a fast approximation of P-values based on a bi-binomial distribution. The bi-binomial distribution is a special case of the Poisson-binomial distribution where the probability of success can assume only two values. In order to achieve a good approximation of the underlying Poisson-binomial distribution, the values of these two probabilities and the number of trials where they are assumed must be determined carefully. As is shown in the following, the values of the two trial probabilities and their number of occurrences follow from the formula used to calculate the variance of the Poisson-binomial distribution and from the formula yielding the population variance of trial probabilities of the Poisson-binomial distribution to be approximated.
The mean μ and the variance σ2 of the Poisson-binomial distribution are given by equation (1) and (2), respectively.(1)(2)pi is the trial-specific probability of success and N is the total number of trials. For the sake of completeness, a formal proof of equation (1) is reported as supplementary material Proof S1 and the proof of equation (2) can be obtained in an analogous fashion.
The population variance S2 of trial probabilities pi is given by equation (3).(3)
Rearranging equation (3) considering (1) and (2) leads to (4) and (5), where pa denotes the average trial probability of success and qa its complement.(4)(5)Now let's define two trial probabilities p1 and p2, which are used N1 and N2 times during the Poisson trials, respectively. Thus, N1 and N2 add up to N.(6)
Considering (1), the average trial probability pa can then be obtained from (7).(7)
Using (7), p1 can thus be calculated as (8).(8)
Similarly, considering (2), the variance σ2 is given by (9).(9)
Substituting p1 in (9) using (8) followed by substituting σ2 in (5) by (9) leads to a quadratic equation for p2 as a function of pa, N, and S2, as shown in equation (10).(10)
The solution to (10) is given by (11).(11)
Setting p2 to(12)p1 can be obtained from (8) and shown to be given by formula (13):(13)Choosing p2 as(12a)leads to p1(13a)
Comparing (13a) to (12) and (12a) to (13), it can be seen that the formulae are identical except for the fact that N1 and N2 are reversed. Since the assignment of which set of trials is called N1 and which set of trials is called N2 is completely arbitrary, we can limit the remaining analysis on (12) and (13) without loss of generality.
Note that (12) and (13) do not guarantee that p1 and p2 are always confined between 0 and 1 for any combination of N1 and N2. While probabilities smaller than 0 or bigger than 1 would still result in a distribution with the same overall variance as the original distribution, P-value calculation will be imprecise because the tails of the distribution will deviate significantly from the original distribution. Thus, we need to define the values N1 and N2 in such a way that p2< = 1 and p1> = 0. This can be achieved by evaluating (12) and (13).
Evaluating (12) for the condition that p2< = 1, solving the resulting inequality for N2, and considering (5), which relates S2 and σ2, we obtain (14).(14)
Similarly, evaluating (13) for the condition p1> = 0, solving the resulting inequality for N2, considering (5), which relates S2 and σ2, and defining μf the expected number of failures as N * (1−pa) (15),(15)we obtain (16)(16)
The meaning of these boundaries is best illustrated by considering a Poisson-binomial distribution whose variance is 0, i.e. that assumes 1 at X = μ and 0 otherwise. In this case (14) requires N2< = μ while (16) requires N2> = μ. These conditions can only be fulfilled contemporaneously when N2 is set to μ. Intuitively, this means that there are μ trials with probability of success 1 and N−μ trials with probability of 0, resulting in a Poisson-binomial distribution with variance σ2 = 0 and mean μ. When σ2 is larger than 0, the choice of N1 and N2 is more flexible. However, since the choice of N2 = μ is valid for all possible values of σ2, this is how NetCutter determines N1 and N2. When μ is not an integer, N2 is set to the integer closest to μ.
Having determined p2 (12) and p1 (13) as well a N1 and N2 (14, 16, 6), we can now calculate the bi-binomial approximation of the Poisson-binomial distribution in a fashion that is very similar to calculate the binomial P-value. With q1 = 1−p1 and q2 = 1−p2 we obtain:(17)
The summation is necessary because i successes can be obtained from any combination of j p1 and i−j p2 trials, where j can assume any value from 0 to i.<br><br>Calculating Jaccard and uncertainty coefficients
For the purpose of comparing the efficacy of the bi-binomial distribution as a significance measure of co-occurrence, Jaccard and uncertainty coefficients (which are also called measures of association) were calculated using the formulae:
The Jaccard coefficient J is calculated as the number of times A and B occur together divided by the number of times A occurs without B plus the number of times B occurs without A plus the number of times A and B occur together [42].
The uncertainty coefficient [42] is calculated as:H is the entropy associated with A, B, and AB. For A, the entropy is calculated from the probabilities of A occurring in n1 out of N lists (n1/N) and A not occurring in n2 out of N lists (n2/N). Analogous calculations lead to the entropy associated with B. For H(A,B), the probabilities of A occurring without B, B occurring without A, A and B occurring together, and neither A nor B occurring in the lists are used.<br><br>Co-occurrence analysis of the PubLiME data set
The bipartite graph to be analyzed is composed of 27619 PubMedID_listID-gene pairs (see supplementary material Table S1). Edge-swapping (1000 simulations, see above) was used to determine occurrence probabilities and gene co-occurrence was analyzed using module size 3 (co-occurrence of three genes), bi-binomial Z-score cutoff 6, bi-binomial P-value cutoff 1.0E-5, and support 5. Supplementary material Simulation S1 provides details on why module size 3 is chosen. The support parameter ensures that each 3-gene co-occurrence module is present in at least 5 signatures. We identified 1654 significant modules in the test data compared to 5 modules in a randomized bipartite graph, corresponding to a signal-to-noise ratio of 331. The co-occurrence network was generated from the significant co-occurrence modules by drawing an edge between each pair wise combination of genes that are part of the same co-occurrence module. Gene communities were identified in this network by edge-betweenness clustering removing 4 edges, which resulted in a maximal network modularity of 0.63. Modularity is calculated as described by [43].
For the identification of PubMedID_listID clusters, the PubMedID_listID-gene pairs in the original bipartite graph were reversed to form gene-PubMedID_listID pairs. Occurrence probabilities were obtained by transposing the original occurrence probability matrix determined by edge-swapping as described above. PubMedID_listID co-occurrence was analyzed using module size 5, Z-score cutoff 6, bi-binomial P-value cutoff 1.0E-5, and support 3. Please note that the choice of these parameters is dictated by the parameters used in gene co-occurrence analysis. The reversal of the bipartite graph necessitates the support parameter used in gene co-occurrence analysis (5) to be used as module size in PubMedID_listID co-occurrence analysis and the module size used in gene co-occurrence analysis (3) to be used as the support parameter in PubMedID_listID co-occurrence analysis if the scope of the analysis is the identification of PubMedID_listID clusters that correspond to gene clusters identified before. The significance cutoffs remain unchanged. PubMedID_listID co-occurrence analysis revealed 448 significant co-occurrence modules in the real bipartite graph and 6 significant co-occurrence modules in the randomized bipartite graph with a signal-to-noise ratio of 75. Communities in the resulting co-occurrence network were identified by edge-betweenness clustering removing 130 edges. The resulting maximal network modularity was found to be 0.47.<br><br>
Supporting Information<br><br>
<h3>pmcA2575403</h3>Foamy Macrophages from Tuberculous <span class="yellow">Patients</span>' Granulomas Constitute a Nutrient-Rich Reservoir for <span class="yellow">M. tuberculosis</span> Persistence
Abstract
Tuberculosis (TB) is characterized by a tight interplay between <span class="yellow">Mycobacterium tuberculosis</span> and host cells within granulomas. These cellular aggregates restrict bacterial spreading, but do not kill all the bacilli, which can persist for years. In-depth investigation of <span class="yellow">M. tuberculosis</span> interactions with granuloma-specific cell populations are needed to gain insight into mycobacterial persistence, and to better understand the physiopathology of the disease. We have analyzed the formation of foamy macrophages (FMs), a granuloma-specific cell population characterized by its high lipid content, and studied their interaction with the tubercle bacillus. Within our in vitro <span class="blue">human</span> granuloma model, <span class="yellow">M. tuberculosis</span> long chain fatty acids, namely oxygenated mycolic acids (MA), triggered the differentiation of <span class="blue">human</span> monocyte-derived macrophages into FMs. In these cells, mycobacteria no longer replicated and switched to a dormant non-replicative state. Electron microscopy observation of <span class="yellow">M. tuberculosis</span>–infected FMs showed that the mycobacteria-containing phagosomes migrate towards host cell lipid bodies (LB), a process which culminates with the engulfment of the bacillus into the lipid droplets and with the accumulation of lipids within the microbe. Altogether, our results suggest that oxygenated mycolic acids from <span class="yellow">M. tuberculosis</span> play a crucial role in the differentiation of macrophages into FMs. These cells might constitute a reservoir used by the tubercle bacillus for long-term persistence within its <span class="blue">human</span> host, and could provide a relevant model for the screening of new antimicrobials against non-replicating persistent mycobacteria.<br><br>Introduction
Tuberculosis caused by <span class="yellow">Mycobacterium tuberculosis</span> (<span class="yellow">M.tb</span>) remains one of the leading causes of mortality in the world, with around 2 million deaths each year [1]. Most individuals remain asymptomatic after the primary infection with only 10% at risk of developing an active disease during their life [2]. In asymptomatic individuals, the bacilli are not cleared but rather persist in a dormant state, from which they may reactivate and induce clinical disease at later stages [3].
The prognosis of the disease depends on the host's efficiency to constrain the bacilli at the site of infection. When inhaled <span class="yellow">M.tb</span> reach the lungs, they are internalized by lung macrophages. The latter trigger the accumulation at the infectious site of macrophages, lymphocytes and dendritic cells, to form a granuloma, which is a major histo-pathological feature of TB. Within granulomas, macrophages differentiate into epithelioïd cells (differentiated macrophages), and/or fuse to form multinucleated giant cells (MGC). Macrophages with large numbers of lipid-free vacuoles, as well as macrophages filled with lipid-containing bodies, also called foamy macrophages (FM) are also found within granulomatous structures in both experimental animal models and <span class="blue">human</span> disease [4],[5]. The above cells are surrounded by a rim of lymphocytes, and at later stages, a tight coat of fibroblasts encloses the structure [6]. Although the structure and cell composition of granulomas are well known, the biology of these inflammatory structures and, more specifically, the role of granuloma-specific cell types, remain largely unknown.
We have previously developed an in vitro model of <span class="blue">human</span> tuberculous granulomas to gain insight into the survival strategies of the tubercle bacillus within its <span class="blue">human</span> host. This model now enables the characterization of granuloma-specific cell types, and their modulation by <span class="yellow">M.tb</span> [7]. The main advantage of this model over in vivo animal models or ex vivo <span class="blue">human</span> biopsy samples, is the availability of live granuloma cells which facilitates analysis of their cell biology. Using this model we have recently shown that, within granulomas, large multinucleated giant cells, also known as Langhans giant cells, result from the induction of granuloma macrophage fusion by <span class="yellow">M.tb</span> glycolipids [8]. We have shown that these cells have lost the ability to mediate bacterial uptake upon maturation, but have conserved their ability to mediate antigen presentation [9].
The differentiation of macrophages into FMs has been particularly well described in individuals developing a postprimary, also known as secondary or adult, TB. These postprimary infections are considered to be the result of re-infection or reactivation of a primary TB [10]. FMs have been described in leprosy <span class="yellow">patients</span> or <span class="yellow">M. avium</span>-infected AIDS (Acquired ImmunoDeficiency Syndrome) <span class="yellow">patients</span>, and in chronic stages of <span class="yellow">M.tb</span> infection in <span class="yellow">mice</span> [4],[11],[12]. The foamy aspect of these macrophages is the result of intracellular lipid accumulation within lipid bodies, also called lipid droplets or lipid vacuoles [13],[14],[15]. In an experimental model of leukocyte infection, it was recently suggested that BCG (Bacille Calmette Guerin) infection can induce, in a TLR2-dependent fashion, the rapid formation of lipid bodies carrying out part of the eicosanoïd biosynthesis that usually accompanies the infection, thus pointing to an active role for lipid bodies during the course of infection [16]. However, the mechanisms regulating this lipid accumulation during mycobacterial infection and their significance in the physiopathology of tuberculosis are not understood. Most of the studies on TB granulomas have focused on the contribution of host components, but very little is known about the role played by bacterial constituents in terms of granuloma formation and progression.
The present work was aimed at deciphering the role of FMs in <span class="yellow">M.tb</span> survival within <span class="blue">human</span> granulomas. To test our working hypothesis according to which FMs constitute a nutrient-rich reservoir for <span class="yellow">M.tb</span> persistence, we used our in vitro model of <span class="blue">human</span> granulomas to analyze the formation of FMs and their role during <span class="yellow">M.tb</span> infection. We showed that only highly virulent mycobacteria (<span class="yellow">M.tb</span>, <span class="yellow">M. avium</span>) and not saprophytic ones (<span class="yellow">M. smegmatis</span>) could induce the formation of FMs in mature granulomas. Moreover, we demonstrated that oxygenated mycolic acids specifically produced by the above pathogenic species were responsible for FMs formation. Once differentiated, FMs were unable to mediate phagocytosis of new bacilli and their microbicidal activity was reduced. <span class="yellow">M.tb</span> was not killed in FMs but instead persisted in a non-replicating state, and over-expressed dormancy genes. Noteworthy, in foamy macrophages, <span class="yellow">M.tb</span>-containing phagosomes were shown to migrate towards lipid bodies which they progressively surrounded and engulfed. As a result, bacteria were freed into lipid bodies, thus favoring the bacilli's access to nutrients. From these data, we propose that FMs could form a secure reservoir for the tubercle bacilli.<br><br>Methods
<span class="yellow">Human</span> samples
<span class="yellow">Human</span> blood samples, purchased from the French National Blood provider of Toulouse, were collected from fully anonymized non-tuberculous control donors, an ethical committee approval was, therefore, not necessary. This study was conducted according to the principles expressed in the Helsinki Declaration, with informed consent obtained from each donor.
We chose to work on lymph node samples rather than lung biopsies, which are usually only paraffin-embedded, because staining for lipids can only be performed on frozen samples. Lymph node biopsies were taken for diagnosis purposes, in ten non-<span class="yellow">HIV</span> <span class="yellow">patients</span>. For each biopsy, a fragment was sent to the microbiology laboratory, another was frozen in liquid nitrogen and the main part was fixed in formalin and paraffin-embedded for histological examination. <span class="yellow">M.tb</span> was identified in 9 lymph node biopsies and from the lung aspiration in the last <span class="yellow">patient</span>. This latter case showed no signs of necrosis, and no FMs were found in the lymph node biopsy. This study complies with the guidelines of the declaration of Helsinki.<br><br>Bacterial strains and culture conditions
Wild-type <span class="yellow">M. smegmatis</span> and <span class="yellow">M. smegmatis</span>/hma strains were previously described [17], <span class="yellow">M. tuberculosis</span>-GFP were a kind gift from Dr. C. Guilhot (CNRS-IPBS, Toulouse France). Bacilli were grown in Middlebrook 7H9 medium (Difco) supplemented with 10% albumin–dextrose–catalase (Difco). Fluorescent <span class="yellow">M. smegmatis</span> and <span class="yellow">M. smegmatis</span>/hma were obtained by FITC labelling as described in [18].<br><br>Isolation of RNA from intraphagosomal <span class="yellow">M.tb</span>
Six and 12 days post-infection (MOI 10), macrophages and FMs (5×106) were washed twice with PBS, scraped off the cell dishes and recovered by centrifugation. The cell pellets were lysed with lysis buffer (RNEasy mini kit, Quiagen) and transferred to 2 ml Eppendorf-tubes containing a 0.5 ml suspension of 0.1 mm-diameter glass beads (Biospec). Mycobacteria were disrupted using a bead beater (Retsch) followed by a 5 min centrifugation at 14 000g. RNA contained in the supernatant was then column-purified according to the manufacturer's conditions using the RNEasy mini kit (Qiagen) and quantified.<br><br>Quantitative Real-Time RT-PCR
In RNA samples DNA contamination was excluded by DNAse I treatment (Ambion). 1 µg total RNA was reverse-transcribed using random hexamer primers (Ambion) and Superscript III reverse transcriptase (Invitrogen). Real-time PCR was performed on cDNA using the SYBR green essay (Applied Biosystems). Reverse and forward primers used are listed below in Table 1. Fluorescence was measured by ABIPrism 7300 (Applied Biosystems). The calculated threshold cycle (Ct) value for each gene of interest was normalized to the Ct value for 16S and the fold expression was calculated using the formula: fold change = 2−Δ.(ΔCt) [19]. Real-time PCR conditions include initial activation at 94°C for 5 min, followed by 40 cycles of denaturation at 94°C for 30 sec, annealing and extension at 65°C for 1 min. The gene induction ratios were obtained by comparing gene expression levels in intracellular bacilli with those of log-phase in vitro-grown bacilli. RNAs were isolated from two independent macrophage infections.<br><br>In vitro <span class="blue">human</span> granuloma formation
In vitro granulomas were obtained as previously described [7]. Briefly, 1×106 freshly isolated Peripheral Blood Mononuclear Cells (PBMCs) were incubated with 1×104 viable <span class="yellow">M.tb</span>, or 1×103 viable <span class="yellow">M. smegmatis</span> or <span class="yellow">M. smegmatis</span>/hma. The culture medium was RPMI-1640+Glutamax (Difco), containing 7.5% <span class="blue">human</span> AB serum (Sigma-Aldrich).<br><br>Macrophage differentiation
2.5×106 PBMCs prepared in RPMI-1640+Glutamax (Difco) were plated over coverslips in 24-well plates. After 2 h culture at 37°C, cells were washed 3 times with PBS and then refed with RPMI-1640+Glutamax (Difco), containing 7.5% <span class="blue">human</span> AB serum. After 6 days of culturing, macrophages were differentiated.<br><br>Respiratory burst assay with Nitroblue tetrazolium (NBT)
<span class="yellow">Human</span> monocyte derived macrophages were stimulated with <span class="yellow">M. smegmatis</span>/hma for 2 h at 37°C, washed and reincubated in mycobacterium-free medium. After 2 days of differentiation into FMs, macrophages were co-stained with NBT (2 mg/ml Sigma-Aldrich) and Nile red (Sigma-Aldrich) for 30 min at 37°C. Stained cells were fixed and then analysed with an inverted microscope (Nikon TE 300).<br><br>Phagocytosis and Survival test
For phagocytosis assays, differentiated macrophages were incubated with <span class="yellow">M. smegmatis</span>/hma mycolic acids for 2 days and then infected with 1×108 labeled mycobacteria per well for 90 min, washed 3 times with PBS and chased for 3 h in fresh culture medium.
For survival experiments, macrophages were infected with <span class="yellow">M. tuberculosis</span>-GFP (10 bacteria/cell), washed 3 times with PBS (Gibco) and re-incubated in fresh culture medium. At selected time points thereafter (1, 3, 6, 10 and 14 days) cells were labeled with Nile red (Sigma-Aldrich), fixed and observed under a confocal microscope. The amount of mycobacteria per cells was evaluated. (100 cells were analyzed for each time point).<br><br>Lipid body staining and immunostaining
Granuloma cells were collected and plated onto glass coverslips with a cytospin (Thermo Shandon) fixed for 30 min in PBS-PFA 4% and stained with Oil red-O (Sigma-Aldrich) as described [20]. The slides were then counterstained with haematoxylin (Dako Cytomation) and observed under an inverted microscope (Nikon TE 300).
For fluorescence analysis, granuloma cells or macrophages were collected in PBS, lipid bodies were stained with Nile red (Sigma-Aldrich, 0.1 µg/ml, from a stock solution in methanol) for 15 min washed with PBS, fixed for 30 min in PBS-PFA 4%, mounted with the fluorescent mounting medium (Dako Cytomation) and observed under a confocal microscope.
In order to distinguish the lipids contained within lipid bodies, from those of the cell membrane, we used the fluorescent emission spectrum properties of Nile red which depend upon the kind of lipid associated with Nile red, i.e. for triacylglycerol: λmax em = 590 nm, for phospholipids: λmax em = 640 nm (Molecular Probes handbook). On confocal microscopy pictures, the phospholipid background of both macrophages and FM appears in red and the triacylglycerol-rich lipid bodies appear in white. Cells were considered to be positive for Nile red staining when more than 50% of the cell surface was stained (see Figure S2).<br><br>Mycolic acid isolation
Bacterial residues obtained after lipid extraction with organic solvents [17] were saponified with a mixture of 40% KOH aqueous solution and methoxyethanol (1∶7, v/v) at 110°C for 3 hours in a screw-capped tube. After acidification, fatty acids were extracted with diethylether, derivatised into methyl esters with diazomethane and analyzed by analytical thin-layer chromatography on silica Gel 60 (Silica Gel 60 Macherey-Nagel) using either dicholoromethane or petroleum ether/diethylether (9∶1, v/v, five runs). Visualization of lipid spots was performed by spraying the plates with molybdophosphoric acid (10% in ethanol), followed by charring.<br><br>Processing for electron microscopy
Granulomas were fixed for 1 hour at room temperature with 2.5% glutaraldehyde in 0.1 M cacodylate buffer, pH 7.2, containing 0.1 M sucrose, 5 mM CaCl2 and 5 mM MgCl2. After two successive 15-min washes with the same buffer, the granulomas were postfixed for 1 hour at RT with 1% osmium tetroxide (Electron Microscopy Science) in the same buffer devoid of sucrose. The granulomas were scraped off the culture dishes with a rubber policeman and concentrated in 1% agarose in the same buffer. After a one hour treatment at room temperature with 1% uranyl acetate in Veronal buffer, the samples were dehydrated in a graded series of ethanol and embedded in Spurr resin. Thin sections were stained with uranyl acetate and lead citrate.<br><br>Image acquisition in confocal microscopy
The images were obtained using a Leica confocal fluorescence microscope (SP2) equipped with a Plan Apo 40×1.4 Ph 6 objective (Olympus Optical) and CoolSNAP-Pro CF digital camera in conjunction with Image-Pro Plus version 4.5.1.3 software (Media Cybernetics). The images were edited using Adobe Photoshop CS2 9 software (Adobe Systems).<br><br>
Results
Foamy macrophages are strongly associated with necrotic lesions and often contain <span class="yellow">M. tuberculosis</span>
We analyzed lymph node biopsies from 10 tuberculous <span class="yellow">patients</span> as a first step for evaluating the role of FMs within tuberculous granulomas. A section through a representative biopsy is shown in Figure 1. Well-circumscribed and -differentiated granulomatous structures were observed in all the samples (Figure 1A). Classically, lesions display a necrotic center (N), an interface area between the necrotic center and the histiocytes (I), and some peripheral granulomas (G). Only seven out of ten <span class="yellow">patients</span> presented lesions displaying central necrosis (Table 2). Staining of the histology samples with Oil red-O, a classic lipid stain, confirmed the presence of FMs within the granulomatous structures in six out of seven samples presenting necrosis (Figure 1B), whereas no FMs were found in the three non-necrotic lesions (Table 2). Noteworthy, in samples with necrotic areas, FMs were always found in the interface region flanking the central necrosis (Figure 1B). These observations firstly confirmed the presence of FMs in most TB <span class="yellow">patients</span>' lesions thereby suggesting that these cells play an important role in the formation/maintenance of such lesions. Second, FMs seem to be associated with necrosis, which is a hallmark of TB lesions, since they were observed only in lesions with a necrotic center and preferentially located around the necrotic area.
Interestingly, staining, in parallel, of serial thin sections from a <span class="yellow">patient</span>'s lesion biopsy with Oil red-O (Figure 1C) and Ziehl Nielsen (Figure 1D), showed that most of the bacilli (arrow) were located in the same area as FMs, thus suggesting a strong association between the persisting tubercle bacilli and FMs within granulomas.<br><br><span class="yellow">M. tuberculosis</span> induces the formation of FMs within in vitro <span class="blue">human</span> tuberculous granulomas
To further characterize the role of FMs in the granulomatous response, we assessed whether FM formation in granulomatous structures was triggered only by pathogenic mycobacterial species (<span class="yellow">M.tb</span>), or by low virulent ones (<span class="yellow">M. smegmatis</span>) as well. PBMCs from non-tuberculous control individuals were infected with <span class="yellow">M.tb</span> or <span class="yellow">M. smegmatis</span>, following the procedure previously described for the induction of granulomatous structures ([7],[9] and Figure S1). Granuloma cells collected at days 3 and 11 were stained with Oil red-O to visualize the lipid droplets within FMs under the light microscope (Figure 2). At day 3, several <span class="yellow">M.tb</span>-induced granuloma cells already showed lipid bodies (Figure 2A). In contrast, the cells collected from <span class="yellow">M. smegmatis</span>-induced granulomas were seldom (5%) positively stained (Figure 2B). Interestingly, <span class="yellow">M. avium</span> induced FM formation in a similar way to <span class="yellow">M.tb</span> (not shown). By day 11, the amount of positively stained cells had increased in <span class="yellow">M.tb</span>-induced granulomas, but not in <span class="yellow">M. smegmatis</span>-induced ones. In addition, the number of lipid bodies per cell increased dramatically with time, as depited in the enlarged views (Figures 2A, C). The quantitative evaluation of the percentage of FMs within granulomas induced by both strains confirmed the differences observed under the light microscope, and showed a seven-fold difference (44% vs 6% respectively) between <span class="yellow">M.tb</span> and <span class="yellow">M. smegmatis</span> in terms of their ability to induce FM formation (Figure 2D). Our results therefore show that virulent species such as <span class="yellow">M.tb</span> and <span class="yellow">M. avium</span>, contrary to poorly or avirulent ones such as <span class="yellow">M. smegmatis</span>, are able to induce the formation of FMs within our experimental model.<br><br>Oxygenated mycolic acids induce the maturation of macrophages into FMs
Mycolic acids from <span class="yellow">M.tb</span> incorporated into liposomes were recently shown to trigger the differentiation of <span class="yellow">mice</span> peritoneal macrophages into foamy-like cells [21]. Interestingly, both <span class="yellow">M.tb</span> and <span class="yellow">M. avium</span>, which induce FM formation, express a family of oxygenated mycolic acids, especially ketomycolic acids, which are not produced by <span class="yellow">M. smegmatis</span> (Figure 3A). In this context, inactivation of the <span class="yellow">M.tb</span> hma gene (mmaA4-Rv0642c) was shown to abolish the synthesis of oxygenated keto- and hydroxyl-mycolic acid in the mutant strain [17]. Conversely, transforming <span class="yellow">M. smegmatis</span> with the hma gene induced the production of both keto- and hydroxyl-mycolic acids [22], (Figures 3B, C).
In the light of both data, we anticipated that oxygenated mycolic acids specifically produced by <span class="yellow">M.tb</span> and <span class="yellow">M. avium</span>, under the control of hma, are responsible for FM formation within <span class="blue">human</span> granulomas. To test this hypothesis, we compared FM formation after infection of PBMCs with either the wild-type, or the hma-expressing <span class="yellow">M. smegmatis</span> strain (<span class="yellow">M. smegmatis</span>/hma). Granulomas cells were collected 3 days later and stained with Oil red-O to visualize lipid bodies. Wild-type <span class="yellow">M. smegmatis</span>-induced granulomas displayed only 5.5% of FM, whereas the hma gene-expressing strain induced granulomas bearing a majority (67%) of brightly stained Oil red-O positive cells (Figures 4A, B, C). Induction of FM formation was even greater if isolated macrophages were directly infected with either strain. After only 4 hours of infection, <span class="yellow">M. smegmatis</span>/hma had already transformed 64% of the infected macrophages into lipid body-positive cells (see Figure S2 for Nile red positive cells), whereas only 9% of the cells contained lipid bodies after infection with the wild-type strain (Figures 4D, E, F). To confirm the specific role of hma-dependent oxygenated mycolic acids in FM formation, and to rule out a possible combined effect of oxygenated mycolic-acids with other mycobacterial components, mycolic acids isolated from wild-type <span class="yellow">M. smegmatis</span> or <span class="yellow">M. smegmatis</span>/hma were incubated with isolated macrophages. With mycolic acids isolated from the wild-type strain, only 13% of the macrophages were transformed into FM whereas 66% of the macrophages incubated with mycolic acids isolated from <span class="yellow">M. smegmatis</span>/hma were strongly stained for lipid bodies (Figures 4G, H, I).
These results therefore indicate that oxygenated mycolic acids play a leading role in <span class="yellow">M.tb</span>-induced FM formation.<br><br>The phagocytic and bactericidal activities are arrested in FMs
To assess the function of granuloma FM, we first evaluated the ability of such cells to mediate phagocytosis. For this purpose, macrophages isolated from PBMCs were exposed to <span class="yellow">M. smegmatis</span>/hma-derived mycolic acids to induce FM formation. Two days later, the cell population contained a mixture of FM (50–70%) and macrophages (30–50%), as assessed by Nile red staining (not shown). The mixed cell population was infected with FITC-labelled <span class="yellow">M. smegmatis</span>. Intracellular bacilli were found only within Nile red-negative macrophages thereby indicating that FMs are unable to ingest bacteria (Figure 5A). This result was reproduced using other mycobacterial strains, such as <span class="yellow">M.tb</span> and <span class="yellow">M. bovis BCG</span>, for infection (not shown). These results further suggest that the bacilli found in granuloma FMs were internalized by macrophages prior to their transformation into FMs.
To assess whether FMs are able to develop a respiratory burst, which is a major intracellular bactericidal activity, the ability of Nile red positive cells (i.e. FMs) to mediate NBT reduction was determined. As shown in Figure 5D, only 8% of the NBT-positive cells (Figure 5B) were Nile red positive FMs (Figure 5C). This strongly suggests that once macrophages have differentiated into FMs, they lose the ability to mediate intracellular bactericidal activity. We postulate that FMs could, therefore, form a secure reservoir for the tubercle bacilli.<br><br><span class="yellow">M. tuberculosis</span> persists in a dormant non-replicative state in FMs
To evaluate the validity of the above hypothesis, we analyzed the ability of <span class="yellow">M.tb</span> to replicate within FMs. For this purpose, isolated macrophages were infected with <span class="yellow">M.tb</span>. At selected intervals post-infection, the amount of bacilli per cell was compared in both non-differentiated macrophages and FMs (Figure 5E). Until day 6, <span class="yellow">M.tb</span> replicated in a similar fashion in both cell types. In contrast, after day 6 post-infection, the amount of bacilli remained stationary in FMs, whereas it continued to increase in macrophages. It is interesting to note that arrest of bacterial replication coincided with completion of macrophage differentiation into FM, i.e. starting from day 3 post-infection. Our data suggest, that the bacilli found in granuloma FMs were internalized by macrophages prior to their differentiation into FMs and also that bacilli can terminate their replication cycle while macrophages are being transformed into FM, but that replication comes to a halt as soon as the maturation process is complete.
We next determined whether the non-replicative bacilli observed in FMs were still alive. For this purpose, we analyzed the expression of a series of genes known to be up-regulated when bacilli are in a persistent non-replicating state [23],[24]. RNA was, therefore, prepared from both in vitro-grown <span class="yellow">M.tb</span> and intracellular bacilli at day 6 and 12 post-infection. The respective amounts of RNA corresponding to isocitrate lyase, α-cristallin, a very hypothetical 7.6 kDa protein, CHP and DosR proteins were then quantified by RT-PCR. As shown in Table 3, the dormancy genes were all strongly up-regulated in intracellular bacilli at day 12 post-infection. These results further demonstrate that the bacilli are not killed in FMs, but rather persist in a dormant, and therefore non-replicative stage [25]. Interestingly, the dormancy genes were not as strongly expressed at day 6, time at which bacteria were still able to replicate in macrophages undergoing differentiation into FMs.<br><br>Characterization of <span class="yellow">M.tb</span> survival within FMs and interaction with lipid bodies
To gain further insight into the morphological appearance of <span class="yellow">M.tb</span> within FM and into the interactions between FM lipid bodies (LB) and <span class="yellow">M.tb</span>-containing phagosomes, granulomas were fixed and processed for conventional electron microscopy at days 3 and 11 post-infection.
Whatever the time point at which granuloma cells were observed, bacteria were all enclosed in phagosomes, most of which contained a single bacterium. None of them (over a thousand which were examined under the electron microscope) were free in the cytoplasm and only one was enclosed in a classical autophagic vacuole. At day 3 post-infection, FMs profiles (thin sections) were scarce, representing at most 9% of the total population of macrophage profiles observed under the electron microscope (Figure 6A). In addition, 86% of the FM profiles displayed at most 5 small LBs (Figure 6B). At this stage of granuloma formation, bacteria were infrequent in FM, but were found in other types of macrophages. One of these displayed large numbers of vacuoles containing flocculent material and often one or two LBs. Over 95% of the bacteria located in the different granuloma macrophages were morphologically intact, and therefore alive [26]. Intact bacteria present no breaks in the cell wall or cytoplasmic membrane and their cytoplasm has preserved its ultrastructural organization and electron opacity. Furthermore, they display no electron translucent intracytoplasmic lipid inclusions (ILI). Bacteria were also observed in between cells, probably as a result of cell lysis within granulomas. These bacteria were also morphologically intact and devoid of ILI (not shown).
At day 11 post-infection, <span class="yellow">M.tb</span>-containing macrophages displaying large numbers of vacuoles with flocculent material were less frequently observed. Interestingly, the amount of such vacuoles had strongly decreased in most of these cells while the number of LBs had increased. The percentage of FM had increased to reach 41% of the total population of macrophage profiles within the granulomas (Figure 6A). From these observations, it is tempting to assume that the highly vesiculated macrophages give rise to FMs. Within FMs, the size and amount of LBs had also increased with time since 48% of the FM profiles now displayed more than 5 LB per FM thin section (Figure 6B), randomly distributed within the cells (Figure 6C). About 30% of the FM profiles displayed between 1 and 20 bacteria, which were morphologically intact and enclosed within phagosomes (Figure 6C, enlarged view).
The interaction between these bacteria and the cellular LBs was next examined at day 11. Sixty percent of the bacilli were scattered throughout the FMs and displayed no obvious signs of interaction with the cellular LBs. A small fraction of the bacteria (21%), however, were observed in the close vicinity of cellular LBs. The membrane of the phagosomes in which they were enclosed clearly interacted with cellular LBs (Figures 6C, 7A, 7B, arrows) and became tightly apposed to an increasingly larger surface area of the LB. As a result, the phagosomes started to surround LBs in a zippering fashion (Figures 7E, F). Ultimately, bacilli (19%) were translocated to cellular LBs (Figure 7C). From these observations, it is tempting to assume that <span class="yellow">M.tb</span>-containing phagosomes engulf cellular LBs rather than fusing with them. This process, which is reminiscent of autophagy, resulted in the transfer of free bacteria into the lumen of cellular LBs (Figure 7C), some of which displayed up to 21 bacteria (Figure 7G). Interestingly, only altered <span class="yellow">M.tb</span> found within FM lipid bodies exhibited electron translucent ILIs (Figures 7D, G), thereby suggesting that they are able to accumulate host cell lipids. In previous work, the term altered bacteria had been used to define live bacteria that had acquired ILIs [26]. Since the presence of ILIs within the cytoplasm of <span class="yellow">M.tb</span> is typical for non-replicating bacteria in a state of dormancy [27], this further confirms that these bacteria are dormant.<br><br>
Discussion
Studies carried out several decades ago suggested that postprimary tuberculosis starts as a lipid pneumonia [28],[29]. Indeed, following a first inflammatory process leading to exudates of mononuclear cells within alveolar spaces, early pathologists observed an accumulation of lipid droplets in alveolar macrophages of TB <span class="yellow">patients</span>. Tubercle bacilli were shown to reside in these lipid-rich macrophages which were named foamy macrophages (FMs) [30]. Recently, histo-pathological analysis of biopsies from <span class="yellow">patients</span> with untreated tuberculosis confirmed the century-old histological descriptions of postprimary tuberculosis [10]. In this study, Hunter et al showed that postprimary tuberculosis begins as a lipid pneumonia with the accumulation of large amounts of lipid-rich FMs, accompanied by bronchial obstruction. It was also shown that in alveolar foamy macrophages, the bacilli were mainly found within lipid droplets. All these observations underline the important, yet often neglected, role of lipid accumulation, and more precisely FM formation, at the infectious site in the physiopathology of TB.
In <span class="yellow">murine</span> experimental models, FMs accumulate within the outermost layer of granulomatous structures occupying the alveolar spaces during the chronic phase of infection. This strongly suggests that FMs could be involved in lesion cleaning via phagocytic uptake of cellular debris generated by the local inflammatory response. Once filled with debris, FMs would leave the parenchyma through the alveolar spaces up to the superior bronchial tree, to be finally swallowed and digested in the stomach [31]. In fact, this process is very well known, and is a crucial factor for TB diagnosis in <span class="yellow">children</span>. As <span class="yellow">infants</span> do not usually generate cavitary lesions, and because it is difficult to detect bacilli in the sputum, the diagnosis is linked to the detection of bacilli in the gastrointestinal lavage [32].
We show that <span class="yellow">M.tb</span>-induced the transformation of in vitro-grown <span class="blue">human</span> granuloma macrophages into FMs within 6 days, and even more rapidly (3–4 hours) in cultured macrophages. Although this event occurs more quickly than in vivo, or in animal models, our data are consistent with the above in vivo observations. Within FMs, bacilli and LBs were often tightly linked, to the point that a non-negligible amount of bacteria were ultimately observed within LBs. Interestingly, some of the bacilli transferred into lipid bodies displayed their own intracytoplasmic lipid inclusions, which are considered to be one of the hallmarks of non-replicating (dormant) <span class="yellow">M.tb</span> [27]. The recent observation of persistent ILI-containing tubercle bacilli within adipocyte LBs [33] is in good agreement with our observations. Since bacilli residing in phagosomes that do not interact with cellular lipid bodies do not display ILIs, it is tempting to assume that lipids within ILIs are of cellular origin. The accumulation of lipids within bacilli [47], via interaction with FM lipid bodies could, therefore, be crucial to <span class="yellow">M.tb</span> persistence. It is indeed known that <span class="yellow">M.tb</span> accumulates lipids, and more precisely triacylglycerols, during dormancy [34],[35] from which it derives both carbon and energy for its own metabolism. Intracellular persistence of <span class="yellow">M.tb</span> is also critically linked to the acquisition of host cholesterol through the Mce4 transporter system [36].
The question that arises is how do bacteria gain access to lipids from LB? Direct fusion of phagosomes with FM lipid bodies seems unlikely as the membranes of both structures are quite different from one another. Our observations suggest instead that once <span class="yellow">M.tb</span>-containing phagosomes have established close contact with a lipid body, they surround and engulf the latter by a process that remains to be deciphered. This phenomenon is somewhat reminiscent of autophagy, as observed under conditions of cholesterol depletion in macrophages infected with <span class="yellow">M. avium</span> [37]. After degradation of the resulting inner membrane, bacteria would be freed within the lipids of the engulfed LB, and therefore be in direct contact with cellular lipids. How bacilli translocate the cellular lipids to their own cytoplasm remains to be established.
Another important phenomenon underlined by our study is the strong correlation between the presence of FMs in the granulomatous structures and the development of necrosis within the lesion, as suggested by Pagel over 80 years ago [29]. Interestingly, FMs were systematically located at the interface region between the histiocytes and the central necrosis area of the biopsied lesions. Although necrosis formation could depend on an indirect effect of the global immune response, our data indicate that the formation of FMs is an important factor favoring the appearance of necrosis. Analysis of larger series of biopsy samples are, however, needed to definitely demonstrate our actual hypothesis according to which FMs play a direct and unique role in necrosis formation. Consistent with this hypothesis, we observed that FMs induced from <span class="yellow">M.tb</span>-infected macrophages displayed permanent TNF-α secretion, a potent pro-necrotic factor, whereas <span class="yellow">M. smegmatis</span>-infected macrophages were poor producers of TNF-α. At day 4 post-infection, TNF-α secretion was indeed twofold higher in macrophages (of which 70% had differentiated into FMs) infected with <span class="yellow">M.tb</span> than in those infected with <span class="yellow">M. smegmatis</span>, as measured both by ELISA and RNA quantification (Peyron, unpublished observations). However, one must keep in mind that the association of FMs and necrosis may be the consequence of the FM cleaning process of lipoproteins released into the necrotic tissue, as observed in atherosclerosis lesions [38]. It is thus tempting to propose that <span class="yellow">M.tb</span> mycolic acids may be responsible for the development of necrotic lesions, due to their ability to induce TNF-α production by FMs. Whether mycolic acids are directly involved in TNF-α production, or only indirectly by inducing FM formation, remains a matter of debate currently under study.
Our observations are strikingly similar to the phenomena described for postprimary tuberculosis, that seems to begin as localized foci of pneumonia followed by massive necrosis leading to the formation of pulmonary cavities [10]. If this proves to be the case, then the traditionally admitted phenomenon of cavitation arising from the erosion of caseating granulomatous structures from bronchi can be ruled out [39]. In our study, we successfully induced FM formation from isolated macrophages infected with <span class="yellow">M.tb</span>, i.e. outside a granulomatous structure, which is consistent with Hunter's recent proposal.
Until now, mycolic acids have been considered to be indirectly involved in virulence mechanisms as being part of complex molecules of the mycobacterial envelope. The most widely studied mycolic acid-containing mycobacterial compound trehalose 6,6′ dimycolate (TDM), has been extensively analyzed for its role in virulence since the mid-fifties [40]. Recently, it was shown to interfere with the host granulomatous response [41]. Overall, TDM was mainly shown to mediate macrophage activation and a Th1-type response to <span class="yellow">M.tb</span> infection (for review, see [25]).
Our results demonstrate a direct role of oxygenated mycolic acids for FM formation, independently from the appearance and stage of the disease. <span class="yellow">M.tb</span>-specific mycolic acids indeed trigger the transformation of both isolated and granuloma macrophages, into FM. Given the absence of FM formation in <span class="yellow">M. smegmatis</span>-induced granulomas, despite the induction of a comparative inflammatory response, ascertained by the similar induction of granulomas, this phenomenon clearly depends upon a direct contact with the bacilli, and not to the inflammatory response. Mycolic acids are major and hallmark components of the mycobacterial cell wall. They constitute 40–60% of dry weight of the envelope [42]. All members of the complex (e.g. <span class="yellow">M.tb</span>, <span class="yellow">Mycobacterium africanum</span>, <span class="yellow">Mycobacterium bovis</span> and <span class="yellow">Mycobacterium microti</span>) are able to synthesize the same combination of mycolic acids, i.e. cyclopropanated α-mycolic acids, ketomycolic and methoxymycolic acids [17], which are not synthesized by non-pathogenic mycobacterial species [43]. These structural specificities probably account for part of the pathogenicity of these species, as shown by the impaired virulence of mutant strains deprived of keto and methoxyl groups in experimental infections [17],[44],[45].
Our study, therefore, gives the first proof of a direct role of isolated mycolic acids in the interplay between <span class="yellow">M.tb</span> and host cells. Interestingly, this effect is expressed both by whole bacilli and isolated lipids, suggesting that oxygenated mycolic acids are either secreted by the bacilli, or exposed at the cell wall surface in a manner enabling their bioactivity. According to our results, mycolic acids trigger the formation, within granulomas, of FMs in which bacilli can hide and survive. Oxygenated mycolic acids, either free, as constituents of TDM [44], or linked to the cell wall arabinogalactan [46],[47], should, therefore, be considered as major virulence factors enabling <span class="yellow">M.tb</span> survival for long periods of time in a persistent state. Being an inducer of host lipid accumulation, and FM formation at the site of infection, these oxygenated mycolic acids could, therefore, also be responsible for the induction of necrosis within lesions, thus favoring <span class="yellow">M.tb</span> dissemination.
Interestingly, deletion of the mmaA4 (Rv0642c) gene also drastically decreased the ability of <span class="yellow">M.tb</span> to induce the differentiation of macrophages into FMs (data not shown). However, the residual ability of this mutant to induce FMs suggests that other mycobacterial factors might partially trigger the formation of FM. With regard to the mycolic acid methyltransferases, given that (i) mmaA2 (Rv0644c) and mmaA3 (Rv0643c) are pseudogenes in <span class="yellow">M. leprae</span> and (ii) mmaA4 KO present no trans cyclopropanation [45], thus excluding the involvement of the cmaA2 (Rv0503c) gene, we expect that at least pcaA (Rv0470c), which introduces cis-cyclopropane, may play the same role. Consistent with this hypothesis, a pcaA null mutant is unable to persist within infected <span class="yellow">mice</span> [46], thus demonstrating the role of a mycolic acid methyltransferase in the chronic stage of infection.
Overall, our study has shed light on a previously uncharacterized cell population participating in <span class="blue">human</span> tuberculous granulomas, namely foamy macrophages. We propose that the specific induction of FM by <span class="yellow">M.tb</span> would create a favourable environment for persistent bacteria. In our opinion, FMs could be a safe shelter because they preserve bacilli from a direct contact with granuloma lymphocytes and histiocytes, they lose one of the major macrophage bactericidal activities and they constitute an important source of nutrients for the bacilli thanks to the fatty acids accumulated in their lipid granules.<br><br>Supporting Information<br><br>
<h3>pmcA2610030</h3><span class="yellow">Human</span> genetic selection on the MTHFR 677C>T polymorphism
Abstract
Background
The prevalence of genotypes of the 677C>T polymorphism for the MTHFR gene varies among <span class="yellow">humans</span>. In previous studies, we found changes in the genotypic frequencies of this polymorphism in populations of different ages, suggesting that this could be caused by an increase in the intake of folate and multivitamins by <span class="yellow">women</span> during the periconceptional period. The aim was to analyze changes in the allelic frequencies of this polymorphism in a Spanish population, including samples from spontaneous abortions (SA).<br><br>Methods
A total of 1305 subjects born in the 20th century were genotyped for the 677C>T polymorphism using allele specific real-time PCR with Taqman® probes. A section of our population (n = 276) born in 1980–1989 was compared with fetal samples (n = 344) from SA of unknown etiology from the same period.<br><br>Results
An increase in the frequency of the T allele (0.38 vs 0.47; p < 0.001) and of the TT genotype (0.14 vs 0.24; p < 0.001) in subjects born in the last quarter of the century was observed. In the 1980–1989 period, the results show that the frequency of the wild type genotype (CC) is about tenfold lower in the SA samples than in the controls (0.03 vs 0.33; p < 0.001) and that the frequency of the TT genotype increases in the controls (0.19 to 0.27) and in the SA samples (0.20 to 0.33 (p < 0.01)); r = 0.98.<br><br>Conclusion
Selection in favor of the T allele has been detected. This selection could be due to the increased fetal viability in early stages of embryonic development, as is deduced by the increase of mutants in both living and SA populations.<br><br><br><br>Background
The methylenetetrahydrofolate reductase enzyme (MTHFR) catalyzes a reaction that produces 5-methyltetrahydrofolate (5-methylTHF), the methyl donor for homocysteine in the synthesis of methionine. The 677C>T mutation of the MTHFR gene has been associated with a thermolabile enzyme with decreased activity that may cause an increase in plasma homocysteine concentrations [1] when folate status is poor. This polymorphism is one of the most widely studied clinically relevant polymorphisms in <span class="yellow">humans</span>, as it is related to cardiovascular disease [2] and neural tube defects (NTD; 601634) [3].
A large number of studies have provided a broad overview of the prevalence of the 677C>T polymorphism in different <span class="blue">human</span> populations, showing that the distribution of frequencies is diverse [4]. These differences have been also observed between groups of different ages in the same Spanish population (older and younger than 24 years) [5] and in a Swiss population (older and younger than 60 years) [6], as well as in a Japanese population [7].
In some populations, such the Toscanians in Italy [8] and Mexicans [9], the homozygous mutated genotype (TT) has reached frequencies greater than 30%. On the other hand, in Africans the frequency of the TT genotype is very low (less than 1%) [10,11], but, in African-Americans, it has already reached 2% [12]. Studies based on the distribution of genotypic and allelic frequencies of the 677C>T polymorphism and the 1298A>C polymorphism in the MTHFR gene in Israeli, Japanese and Ghanaian Africans populations [13] concluded that the 677T mutation in the MTHFR gene emerged as a founder haplotype with some selective advantage. Recently, preliminary evidence of genetic selection of this polymorphism related to folate intake has been reported [14].
The aim of the present study is to analyze the changes in frequencies of the 677C>T polymorphism during the 20th century and particularly the evolution of the frequencies during the decade of 1980–1989, by comparing the genotype frequencies between living subjects born in this period versus samples of spontaneous abortions (SA) that occurred during in the same time period.<br><br>Methods
Subjects
This study was approved by the Ethics Committee at the University Hospital "Virgen de la Victoria" (Málaga). One of the study groups consisted of 344 fetal tissue samples from SA, obtained from the Department of Pathology of the University Hospital Carlos Haya (Málaga). These samples were selected after checking the clinical history and by the inclusion criteria of containing histologically confirmed fetal tissue collected in the 1980s from SA at less than 3 months (11 ± 1.70 week) and of unknown etiology. These fetal samples were compared with a control population of 276 subjects born in the 1980s with an average age of 22 ± 4.58.
Another population of subjects born in the south of Spain in the 20th century were genotyped (1305 subjects, 697 <span class="yellow">women</span> and 608 <span class="yellow">men</span>) and divided into four groups according to birth date: 1900 to 1925 (n = 206); 1926 to 1950 (n = 320), 1951 to 1975 (n = 408), 1976 to 2000 (n = 371). Individuals were selected randomly from different areas of the province of Malaga, in southern Spain, and from different social statuses to avoid a selection bias. All the selected individuals were Caucasian and residents of the study area. The parents of all subjects included in the study were also Caucasian and born in Spain. The possibility of a founder effect or genetic drift was investigated and rejected. All the selected individuals were also genotyped for an insertion/deletion polymorphism in the angiotensin converting enzyme (ACE) gene and/or the 2756A>G polymorphism in the methionine synthase gene (MTR), in order to determine whether or not our adult and young populations were genetically homogeneous. No significant differences were observed in allelic or genotypic frequencies for these genes between the different groups.
The population studied was randomly selected according to age. Subjects 0–12 years old were selected from dried blood spots from neonatal screening papers; subjects 10–24 years old were recruited from students in primary and secondary schools and in university; subjects 25–50 years old and >51 years old were recruited using their Andalusia Health Service identity cards. After approval by the University Hospital Ethical Committee, all the subjects were contacted, and, from those whose written consent was obtained, 10 ml of blood was taken. The investigation in this study conforms to the principles outlined in the Declaration of Helsinki.<br><br>Genetic analysis
The fetal samples were extracted from the archived formalin-fixed, paraffin-embedded tissue sections. Genomic DNA was extracted from fetal tissue using the method described by Coombs et al. (1999) [15]. Genomic DNA of the second and third groups was extracted from peripheral leukocytes using the AquaPure Genomic DNA Blood Kit (Bio-Rad).
Genotyping was performed using Real Time PCR with allele specific Taqman® probes and primers described by Ulvik et al. (2001) [16] and the following optimized protocol for 45 cycles: 10 s – 94°C, 40 s – 54°C, 15 s – 72°C. The PCR mix (25 μl total volume) consisted of 5 μl of genomic DNA, 0.5 μl of sense primer, 0.62 μl of anti-sense primer, 0.85 μl Taqman® probe FAM, 0.43 μl Taqman® probe TET, 20 μl PCR-buffer iQ-SupermixTM (Bio-Rad) (containing 100 mM KCl, 40 mM Tris-HCl, (pH 8.4) 1.6 mM dNTP (dATP, dCTP, dGTP and dTTP), iTaq® polymerase (50 units/mL) and 6 mM MgCl2) and 17.75 μl H2O.<br><br>Statistical and mathematical analysis
All samples were genotyped, and the allelic and genotypic frequencies were compared. Differences were analyzed statistically using the chi-square test or Fisher's exact test. Correlations are expressed using Pearson's coefficient (r).
Compliance of genotype distributions with Hardy-Weinberg (HW) equilibrium was evaluated by chi-square analysis. For all tests, a p-value < 0.05 was considered to be statistically significant. Values are expressed as the mean ± SD.
The genetic selection model was calculated for the evolution of the 677C>T genotypes. The genetic selection could be classified as codominant or incompletely dominant and directional with the heterozygous genotype having an intermediate fitness. For this kind of selection, the most appropriate mathematical model is dq = [spq(2hp+q-h)]/[p2 + 2pq(1-hs) + q2 × (1-s)], where dq is the change of frequency of the allele with lower fitness, s is the fraction of that genotype lost to selection, h is the degree of dominance (between 0, for no dominance and 1, for complete dominance), and p is the frequency of the allele with higher fitness.<br><br>
Results
We analyzed the genotype frequencies of the 677C>T polymorphism in a population born during the 20th century. A total of 1305 subjects were divided into four groups of 25 years according to birth date. The genotype frequencies were compared between the four quarters of the century and showed very significant changes (p < 0.001) in the group born in the last quarter of the 20th century (1976–2000), when compared to any of the other groups. The changes show a decrease of the CC genotype and an increase of the TT genotype in the last 25 years of the 20th century. (Table 1)
Considering that each 25 year period corresponds to a generation, allelic frequencies did not change during the first 75 years of the century (HW equilibrium). However, we found that allelic and genotypic frequencies for the 677C>T polymorphism in the last quarter of the century are significantly different compared to the previous generation (1951–1975). The genotype frequencies in the last quarter of the century are not the expected by a HW calculation using the allelic frequencies of the previous generation. This could be described as a consequence of genetic selection found in this population, in the absence of other causes. Applying the mathematical model described above to our population, the calculated fitness (s) is 0.5, and it can be predicted that both alleles will be approximately at a frequency of 50% in the next generation and allele T will be at 90% after seven generations (Figure 1A). Another possibility is that a scenario could be predicted in which both alleles will have frequencies of about 50% in the next generation and that they will maintain this stability while conditions remain unchanged. (Figure 1B)
The comparison of the genotype frequencies between a group of fetal samples from SA that occurred during the 1980–1989 decade and living subjects born in the same decade showed significant differences in genotype frequencies (p < 0.001). CC genotypes were almost absent in abortion samples compared to living subjects (0.03 vs 0.33), while CT and TT genotypes were overrepresented in the same group. When 3-year periods are studied in the decade, we detected a significant increase of the mutated subjects during the decade (CT p < 0.05; TT p < 0.01). Allele frequencies showed the same pattern (p < 0.05). Controls showed the same tendency but without statistical significance. (Table 2)
The evolution of genotype frequencies during the 1980–1989 decade of the TT genotypes correlates well in both living populations as well as fetal samples r = 0.98 (p = 0.11).<br><br>Discussion
Different reports show that the prevalence of the 677C>T polymorphism of the MTHFR gene differs dramatically among <span class="blue">human</span> populations. Evidence of this dynamism can be observed in many reports: frequency variations between populations that are geographically very close, even in the same country [8]; changes found in the same race or ethnic group such as Africans [10,11] and African-Americans [12]; the high prevalence of the 677C>T poymorphism in populations with special nutritional features such as Mexicans [9] and Japanese [13]; and changes in frequencies between generations of the same population, as has been observed in Spain [5], Switzerland [6] and Japan [7].
There are numerous interpretations of this great diversity, and most tend to be related to adaptation to external conditions such as climate or nutritional status. Dependence of folate degradation on skin pigmentation [17], nutritional habits or <span class="blue">human</span> intervention periconceptional periods could explain this genetic variation. Definitely, external factors in combination with different levels of MTHFR enzyme activity, conditioned by polymorphisms, could influence the fetal viability of certain genotypes.
In 1998, we suggested the possibility of genetic selection in Spain in favor of the mutants of the 677C>T polymorphism in the MTHFR gene based on the fact that treatment with vitamins and folates during pregnancy increased the viability of fetuses with the TT homozygous genotype. This hypothesis was based on the increase in the number of mutated individuals found in our population since the mid-1970s [5] and the coincident increased intake of vitamins and folate by pregnant <span class="yellow">women</span> in Spain [18,19]. In 2002, a new study found changes in genotype frequencies for the 677C>T and 1298A>C polymorphisms in different age groups. Total homocysteine (tHcy) levels in plasma were also analyzed according to the different genotype interactions [20]. That study hypothesized about fetal viability and about a genetic selection model on the basis of non-linkage disequilibrium between both polymorphisms. Recently, a study with fetal and control populations showed the strong influence of these polymorphisms, though mainly of the 677C>T polymorphism, on spontaneous early abortion [21]. In the present study, significant changes in allelic and genotypic frequencies are detected, as is Hardy-Weinberg disequilibrium, at the 677C>T polymorphism. We hypothesize that there is a dynamic process of genetic selection that favors the T allele. This process of selection started during the last quarter of the 20th century, during which the frequency for mutant homozygous (TT) rose significantly from 14% to 24%. We propose that this increase in mutants is due to the inclusion of an external factor that enhances mutant fetal viability.
If we apply the mathematical model for dynamic selection developed for diploid organisms with sexual reproduction, the T allele could reach to 90% in seven generations in our population (Figure 1A). However, this model assumes selection in a constant environment that applies to all individuals in the population studied. In our case, we suggest that the external factor is related to an increase in folate and vitamin intake in <span class="yellow">women</span> in periconceptional period and does not affect to all individuals [18,19]. We assume that prediction of a classic selection model in this case is only theoretical.
On the basis of a competition between alleles in which an environmental factor favors one allele versus the other, the final result would be that predicted by the previous mathematical model. However in this case, the environment is not selecting against the wild type allele but rather allowing the survival of more mutated alleles. Therefore, the expected result would be not a systematic increase of the mutated allele but the creation of an allelic balance dependent on vitamin and folate abundance conditions. In this case, the mutation would have a lower influence on fetal viability (Figure 1B).
The results showed an increase in mutated genotypes (CT and TT) and a strong protection against abortion by the wild type genotype (CC), which is practically non-existent in the SA group. The frequency of the CC genotype shows no change over the decade studied (1980–1989), which indicates that folate does not exert a visible effect on this genotype. However, the frequency of the mutated allele increases during this decade, especially in fetuses from abortions, and this increase correlated with the increase of the T allele in the control population. This finding suggests that the effect of folate is crucial to viability during the early stages of embryonic development, but, even with folate, not all embryos will survive until birth.
In this population, the mutant allele with lower enzymatic activity has higher fitness than the wild type. In the folate cycle, it can be observed that 5,10-methyleneTHF availability may be important. 5,10-methyleneTHF is the substrate for several reactions in the cycle, but two of them (5-methylTHF and thymidilate synthesis) might be essential for embryo development in folate deficiency conditions.
In both cases, complete or limited MTHFR activity will produce higher or lower 5,10-methyleneTHF availability, which might be an essential factor for embryo development, such that a greater folate levels can compensate the lower enzymatic activity of the mutant.
The implications of this polymorphism in nucleotide synthesis have not yet been determined, but certain data, such as high levels of uric acid found in mutated subjects [22,23], suggest that there are different turnover rates associated with different polymorphisms.<br><br>Conclusion
We suggest that there is genetic selection in our population for the T allele of the MTHFR – 677C>T polymorphism, whose origin could be an increase in fetal viability during the early stages of embryonic development because of an increase in folate and vitamin intake by <span class="yellow">women</span> in the periconceptional period that began to be established in Spain in the last quarter of the 20th century [18,19]. Higher frequencies for the T allele and TT genotype in our population are observed in the living and SA populations.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
AMO performed the statistical analysis, helped to draft the manuscript and revised it for publication. GC is the corresponding author, participated in the acquisition of samples and carried out the genotyping. ARP carried out the bibliographic search and helped to draft the manuscript. AJJ participated in the selection and the processing of samples. MJG coordinated the laboratory work and selected the genotyping method. AR selected the control subjects and designed the consent form. MR helped in the interpretation of data and tables performance. ARE conceived the study and is the guarantor of this work and the general coordinator. All authors read and approved the final manuscript.<br><br>Pre-publication history
The pre-publication history for this paper can be accessed here:<br><br><br><br><h3>pmcA2636785</h3>Identification of recruitment and retention strategies for rehabilitation professionals in Ontario, Canada: results from expert panels
Abstract
Background
Demand for rehabilitation services is expected to increase due to factors such as an aging population, workforce pressures, rise in chronic and complex multi-system disorders, advances in technology, and changes in interprofessional health service delivery models. However, health <span class="blue">human</span> resource (HHR) strategies for Canadian rehabilitation professionals are lagging behind other professional groups such as physicians and nurses. The objectives of this study were: 1) to identify recruitment and retention strategies of rehabilitation professionals including occupational therapists, physical therapists and speech language pathologists from the literature; and 2) to investigate both the importance and feasibility of the identified strategies using expert panels amongst HHR and education experts.<br><br>Methods
A review of the literature was conducted to identify recruitment and retention strategies for rehabilitation professionals. Two expert panels, one on Recruitment and Retention and the other on Education were convened to determine the importance and feasibility of the identified strategies. A modified-delphi process was used to gain consensus and to rate the identified strategies along these two dimensions.<br><br>Results
A total of 34 strategies were identified by the Recruitment and Retention and Education expert panels as being important and feasible for the development of a HHR plan for recruitment and retention of rehabilitation professionals. Seven were categorized under the Quality of Worklife and Work Environment theme, another seven in Financial Incentives and Marketing, two in Workload and Skill Mix, thirteen in Professional Development and five in Education and Training.<br><br>Conclusion
Based on the results from the expert panels, the three major areas of focus for HHR planning in the rehabilitation sector should include strategies addressing Quality of Worklife and Work Environment, Financial Incentives and Marketing and Professional Development.<br><br><br><br>Background
Demand for rehabilitation services is expected to increase within the next decade primarily due to factors such as an aging population, workforce pressures, rise in chronic and complex multi-system disorders, advances in technology, and changes in health service delivery models [1-4]. In Canada, rehabilitation personnel constitute the third largest health professional group after nurses and physicians. Despite the size of this workforce, studies have consistently reported ongoing shortages of physiotherapists (PTs), occupational therapists (OTs) and speech-language pathologists (SLPs) across all jurisdictions [5-7]. Similarly, recruitment and retention of rehabilitation professionals has been considered a challenge internationally, nationally and provincially. At the international level, the literature reports recruitment and retention difficulties of rehabilitation therapists in countries such as Australia, New Zealand, United Kingdom and the United States [8-13]. Provinces across Canada face similar issues; with Ontario projected to face the most difficulty due to its population growth rate [14].
Based on the Canadian Institute for Health Information's Health Personnel Trends in Canada from 1993 to 2002 report, numerous factors have been suggested to influence demand for physiotherapy and occupational therapy services. Factors that may influence increase demand for physiotherapy include: shift in health service delivery models from hospital to community care; earlier <span class="yellow">patient</span> discharge; increased expectations from aging Canadians concerning more active lifestyles; growing private practice sector and continued shortages for PTs in both private and public sectors in rural, remote and urban settings across Canada [2]. In 1993, an Ontario study stated that in order to meet demands of changing health care policy, medical technology and demographic changes in the population, the PT profession required an annual growth rate of 4.4% until the year 2000 [5]. However, the national health personnel databases revealed that the actual average annual growth rate of active PTs in Canada from 1995 to 2004 was only 2.5%, approximately half of the projected requirement suggested to meet demand [15].
Similarly, in Ontario in the early 1990s an increase in demand for OTs was projected because of the reported shortage in OTs and high attrition rate [16]. The shortage of OTs was explained by another Ontario study to be the result of the changing philosophies of care and management for the disabled, and a clearer understanding of the role of OT in the physical and mental well-being of the disabled [17]. In terms of actual shortages, some authors have reported ongoing vacancies and recruitment difficulties for OTs [11,18] while others have reported an increase in demand for both PTs [19] and OTs [20].
Speech language pathology is facing similar service demands. A report released in March 2003 by the College of Audiologists and Speech-Language Pathologists of Ontario (CASLPO) concluded that based on prevalence rates for Ontario residents with speech, language and related disorders, the demand for service would increase by 13% while the number of SLPs would decrease by 4% resulting in an overall reduction in service of 15% [21]. The American Speech-Language-Hearing Association (ASHA) has been tracking SLP vacancies. In their 2005 ASHA Speech-Language Pathology Health Care survey, 48% of respondents indicated that they had funded unfilled positions for SLPS in their agency [22]. The same survey also reported that 65% of respondents in home care indicated that job openings were more numerous than job seekers in their geographic area.
While labour market demand and supply are influential factors on recruitment and retention decisions, the development of strategies requires an understanding of conceptual frameworks or theories to categorise and explain how other underlying factors impact health worker's mobility. For example, Lehmann et al.'s model described that health worker's decisions to accept and stay in remote areas in the public sector depends on two interrelated aspects: the impact of the different environments (i.e. individual, local, work, national and international) and the location of decision-makers (i.e. local government, Ministry of Health, HR directorate, public service and other ministries)[23]. Behavioural and social science theories, such as those explained by Tett and Meyer, found that job satisfaction and organizational commitment each contribute independently to the prediction of the intention to resign (turnover), however job satisfaction was a stronger predictor than organizational commitment[24]. Based on this notion, considerable research has been devoted to identifying factors that affect job satisfaction among rehabilitation professionals. While there is no single, agreed upon model of job satisfaction, a variety of theoretical models have been studied to explain concepts and relationships associated with overall job satisfaction. The two most commonly used theories of job satisfaction for rehabilitation professionals are the Herzberg's Motivation-Hygiene Theory[25] and Mottaz's concepts of work values and work rewards [26].
A number of rehabilitation studies have used the Herzberg's Motivation-Hygiene Theory, also known as the two-factor theory of motivation to explain associations between motivation, job satisfaction and retention factors among OTs, PTs and SLPs [27-31]. Frederick Herzberg et al. explained that there were two independent incidents occurring at <span class="yellow">peoples</span>' jobs: one that made them feel good or satisfied, and another that made them feel bad or dissatisfied at work [25]. Intrinsic factors that motivate <span class="yellow">people</span> such as achievement, recognition, work itself, responsibility, advancement and personal growth were called the "motivators" which lead to feelings of satisfaction. Extrinsic factors such as work conditions, company policies, supervision, interpersonal work relations, salary and job security, known as "hygiene" factors, were claimed to prevent dissatisfaction. "Motivators" directly affect a <span class="yellow">person</span>'s motivational drive to do a good job, therefore they are believed to be more important than hygiene factors.
Mottaz on the other hand, accounted for individual differences in job satisfaction among workers and based his study on two dimensions: "work rewards" and "work values"[26]. "Work rewards" are perceived characteristics of the job and have three conceptual clusters which include task, social and organizational rewards [26]. Mottaz describes "task rewards" (intrinsic) as having five independent characteristics including: skill variety, task identity, task significance, autonomy and feedback. Examples include interesting and challenging work, self-direction and responsibility, creativity, opportunities to use one's skills and feedback. In the same study, Mottaz stated that "social rewards" (extrinsic) are derived from the interpersonal relationships established with others at work. Having supportive colleagues and supervisors is an example of this dimension. Lastly, "organizational rewards" (extrinsic) are tangible rewards that are provided by the employer/organization to facilitate performance. Such factors include working conditions, pay and fringe benefits, career advancement and security. The second dimension of job satisfaction is based on "work values", which is the importance that individuals place on their work rewards [26]. For example, some rehabilitation therapists may value extrinsic rewards such as pay and benefits as more important than intrinsic factors like clinical autonomy and challenging work. Although the Herzberg and Mottaz conceptual frameworks are organized differently, their job satisfaction variables are very similar (i.e. work conditions, pay, interpersonal relationships, etc.) and they both classify these factors as having intrinsic or extrinsic elements.
Despite the growing body of literature on recruitment and retention factors in various industries, there is a minimal amount of research studying these factors specifically among rehabilitation professionals. One published study however, did look at extrinsic and intrinsic job satisfaction factors on recruitment and retention of rehabilitation professionals (OTs, PTs and SLPs)[32]. Results from this study showed that intrinsic factors such as professional growth and having a work environment in line with personal values are more significant in predicting career satisfaction than extrinsic factors such as pay and continuing education. These same intrinsic factors are also significant in predicting retention in rehabilitation professionals. Another study looking at recruitment and retention of allied health professionals in the rural areas in New South Wales identified that the main reasons why <span class="yellow">people</span> liked working in rural areas were because of the attractive environment and helpful team members[33]. However 82% of employees reported that having their partner move away was the number one reason for leaving a rural job. A similar study was conducted among OTs and PTs in Northwestern Ontario[34]. Findings from this study indicated that factors contributing to initial decision on location of practice include availability of leisure/recreation activities, proximity of family origin and influences of spouse/partners. Study results also showed that the main reasons therapists left their job were to be closer to their family, lack of job opportunity and spousal influence.
Solely understanding factors that influence recruitment and retention decisions is not sufficient in the development of a HHR plan for rehabilitation professionals. In order for the plan to be effective and sustainable in addressing these factors, the most important and feasible workforce strategies needs to be identified.
There have been a number of reports on health <span class="blue">human</span> resources (HHR) planning, recruitment and retention strategies for physicians [35,36] and nurses [37], however information regarding rehabilitation professionals is lacking. Canadian reports indicate that the main reason for significant gaps in this field is the absence of current and reliable data available on supply, demand and labour force participation trends for rehabilitation therapists [38-40]. There is some research that has investigated theoretical models of job satisfaction on recruitment and retention [28,32]; however few studies have looked at how these models have been implemented. Other studies have examined the relationship of gender, workplace setting (i.e. hospital, ambulatory, rehabilitation, acute and long-term care) and geographical location (i.e. rural or urban) on job satisfaction and retention among rehabilitation professionals [41-43]. Furthermore, no empirical studies have examined conceptual frameworks for organizing recruitment and retention strategies for rehabilitation therapists. To address this gap, this research identified recruitment and retention strategies from the literature for rehabilitation professionals and determined their importance and feasibility using expert panels.<br><br>Methods
Phase 1: Literature Review
Identification of recruitment and retention strategies
A review of the literature was conducted to identify recruitment and retention strategies for rehabilitation therapists. In this study, rehabilitation professionals were defined as physical therapists (PTs), occupational therapists (OTs) and speech-language pathologists (SLPs). Both published and non-empirical literature was accessed in this review. Keywords used to search for relevant published studies in the Consolidated International Nursing and Allied Health Sciences Library (CINAHL) (1982 to 2005) and Medline (1996 to 2005) included: "health <span class="blue">human</span> resources or health manpower", "rehabilitation or rehabilitation professionals or vocational", "allied health professionals or personnel", "recruitment strategies", "retention strategies", "physical therapist or physiotherapist", "occupational therapy or occupational therapist", "speech-language pathologist or speech-language pathology". Non-empirical literature searches were made on international and national on-line catalogues and publications from health organizations, professional associations, and hospital and home care organizations. International reports were limited to developed countries since the purpose of this study was to identify strategies appropriate to the Ontario setting.<br><br>Organization and consolidation of strategies
There was a paucity of peer-reviewed studies obtained exploring rehabilitation HHR strategies, therefore the majority of strategies were selected from grey literature reports from international, national and provincial health organizations. From the literature review, 107 potential strategies were identified according to their relevance to HHR issues for rehabilitation in Ontario. These strategies were then categorized into two broad groups: A) Recruitment and Retention (n = 73), and B) Education (n = 34). The majority of strategies were not specific to rehabilitation professionals and they were reviewed by a group of three individuals collectively (rehabilitation researcher, manager, and clinician) for duplication, clarity, action focused properties and appropriateness to the Canadian or Ontario setting. When necessary, a small number of strategies were re-worded to be relevant to a rehabilitation context. This analysis resulted in the selection of 40 Recruitment and Retention and 24 Education strategies. Only 14 recruitment and retention and six education strategies were obtained from peer-reviewed articles. Since the majority of strategies were identified from the grey literature, it was not surprising that there was no apriori peer-reviewed conceptual framework that reflected the breadth of the strategies obtained from the literature review. As a result, the themes used by the Health and Community Services <span class="yellow">Human</span> Resources Sector Study in Newfoundland and Labrador [44] formed the basis for the organizational framework for this study since they aligned with the identified strategies. Each group was further categorized into the five themes (three for Recruitment and Retention and two for Education). The three Recruitment and Retention strategy themes were: (1) Quality of Worklife and Work Environment [n = 19]; (2) Workload and Skill Mix [n = 6]; and (3) Financial Incentives and Marketing [n = 15]. The two Education strategy themes were: (1) Education and Training [n = 11] and (2) Professional Development (n = 13).<br><br>
Phase 2: Expert Panel
<span class="yellow">Participant</span> Selection
Once this study was approved by the Research Ethics Board at the University of Toronto, key informants who participated in a previous study regarding rehabilitation supply and demand at the University of Toronto [21] were asked to nominate potential <span class="yellow">participants</span> for the panels. The selection criteria considered were acknowledged leadership in the panel member's specialty, expertise in recruitment and retention or education and training of rehabilitation professionals. Absence of conflicts of interest, geographic diversity, and diversity of practice setting were also considered. After purposefully selecting the initial list of candidates from among the nominations, each nominee was contacted to establish their interest and availability. Those who expressed an interest in participating were asked to send their curriculum vitae to help the research team evaluate their contributions to their field of expertise. Once candidate panelists were selected, each received a letter explaining the expert panel process and consent form. Two separate panels were constructed: one for Recruitment and Retention (n = 8) and the other for Education (n = 9) (Table 1). The size of the panel was large enough to permit diversity of representation while still being small enough to allow all <span class="yellow">participants</span> to be involved in the group discussion [45].<br><br>Expert Panel Process: Round 1 Survey
A modified-delphi technique was then used for the expert panel process, [46,47] based on the RAND/UCLA appropriateness method [45]. In round 1, members of the Recruitment and Retention panel were sent an electronic survey containing the 40 strategies identified from the literature review and the Education panelists were also sent an electronic survey with 24 strategies. For Round 1, each panel was asked to rate the strategies using a nine-point Likert-type rating scale that ranged from "none" (1) to "maximum" (9), on two key dimensions: Feasibility and Importance. Feasibility was defined as the practicality and cost implications of the strategy and was rated from the respondents' perspective. Importance was defined as how valuable, appropriate and useful the strategy could be for rehabilitation HHR planning in Ontario. At the end of the survey, panelists were given the opportunity to suggest additional strategies that they felt were appropriate to consider. Once completed, panelists were asked to return the survey to the study office by email or fax prior to the expert panel meeting in Round 2. Data from each questionnaire were entered into a spreadsheet and tabulated. Descriptive statistics were calculated for each strategy using frequency distributions and proportional percentages of respondents. Importance and feasibility rankings were based on the percentage of expert panelists' low, medium and high ratings.<br><br>Expert Panel Process Round 2: Expert Panel one-day meeting
After the independent completion of the survey, each panel was convened separately for a one-day meeting for final discussions, debates and consensus voting to decide on strategies [48]. A strategy that had been scored 7, 8 or 9 for both feasibility and importance by two-thirds of the panel was considered a high rating. Strategies that had a combination of medium and high scores between 4 and 9 in either of the two dimensions were considered medium rated strategies, while low rated strategies had scores between 1 and 3 for both dimensions.
On the day of the meeting, the panelists were given a copy of the aggregated survey results indicating the ratings of all of the strategies. High and low rated strategies were not discussed as there was already consensus, whereas all medium rated strategies were subject to discussion. Using a nominal group process [47], each strategy was discussed in turn, and panelists were given an opportunity to raise any issues or concerns regarding the clarity and wording of each strategy. Each of the strategies discussed were then individually rated a second time by each panelist in an attempt to reach further consensus.<br><br><br><br>Results
Selection of strategies for Round 1: Modified Delphi process
Following Round 1 rating of the 40 identified strategies, the Recruitment and Retention panel reached consensus on 12 strategies. However, 14 had a combination of high/medium importance and feasibility ratings and 14 had medium ratings on both dimensions, therefore it required further discussion. An additional strategy regarding family relocation programs was added by this panel.
The Education Panel ranked 16 of 24 strategies with high importance and feasibility after Round 1. Since there were only eight strategies with medium ratings, this expert panel decided to review all the strategies at the face-to-face meeting to discuss the rationale that would explain why some of the highly rated strategies were not already implemented and to come to a consensus on the other eight medium rated strategies. They also added an additional strategy for career paths.<br><br>Selection of strategies for Round 2: Face-to-face meeting
A total of 34 strategies were identified by both the Recruitment and Retention and Education expert panels as being important and feasible for the development of a HHR plan for recruitment and retention of rehabilitation professionals. Under the Recruitment and Retention theme, seven were categorized as Quality of Worklife and Work Environment; two were Workload and Skill Mix, and another seven were Financial Incentives and Marketing. As for the Education panel, five were categorized as Education and Training strategies while the other thirteen were related to Professional Development.
As indicated in Table 2, at the end of the second round of voting, the Recruitment and Retention panel had a total of 16 highly important and feasible strategies, 8 high/medium importance and feasibility, 8 medium and 9 low ratings for both dimensions. The Education panel on the other hand had a total of 18 high, 1 high/medium, 3 medium and 3 low rating strategies.<br><br>Recruitment and Retention Strategy Rankings
Table 3 provides a detailed description and ranking of each of the recruitment and retention strategies that were rated highly important and feasible. The importance and feasibility rankings were based on the largest proportion of panel members rating a strategy a 7, 8 or 9. The overall combined ranking was based on the average of the proportion of these two dimensions. Among these selected strategies, the majority were classified under Quality of Worklife and Work Environment (44%) and Financial Incentives and Marketing (44%), followed by Workload and Skill Mix (12%). It should be noted that some strategies had equal rankings; therefore the total number of rankings did not equal the total number of strategies.
Recruitment and retention strategies that had a combination of high and medium ratings in either of the two dimensions included: sense of empowerment in promoting healthy work environments; team-building exercises; developing participatory decision-making systems; improving rural working conditions; recognizing work-life balance; creating an environment where staff are valued; optimizing scope of practice and work-management autonomy. Strategies that had medium importance and feasibility ratings included: resolving concerns about liability and accountability in collaborative practice; recruiting international trained therapists; opportunity to work in different settings; interprofessional payment schemes; family leave; staff recognition and creating a position for a provincial health professional recruiter. Low importance and feasibility strategies included: word of mouth references; bursaries and retention bonuses; exchange employment opportunities; health promotion; retention workshop; 80–20 staffing model (80% clinical and 20% learning new skills or training others); using recruitment agencies and providing recruitment bonuses (Table 4).<br><br>Education Strategy Rankings
Education strategies that were rated highly important and feasible are described in Table 5. The majority of strategies in this group tend to be in the area of Professional Development (72%), more so than Education and Training (28%).
The medium rated education strategies included: expand interprofessional education; provide incentives for students interested in rural practice; summer mentorship programs for high school students; and aboriginal student support program. The strategies that were considered neither feasible nor important included: using return of service contracts after professional development; create a tiered pathway approach through modular education and laddered credentialing and in accreditation standards allow greater use of rural practice sites (Table 6).
Since the purpose of the panel was to identify recruitment and retention and education strategies that could inform the development of a HHR plan for rehabilitation professionals, there was also discussion about contextual factors that would influence a plan. Panelists commented that key factors to consider prior to implementation of these strategies should include workplace setting, geographical location (i.e. urban and rural) and gender issues.<br><br>
Discussion
The purpose of this study was to identify recruitment and retention strategies to inform the development of HHR planning for rehabilitation professionals in Ontario. This study highlights that Quality of Worklife and Work Environment, Financial Incentives and Marketing and Professional Development are the three major areas of focus when developing a competitive HR plan in the rehabilitation sector.
Quality of Worklife and Work Environment
Quality of Worklife and Work Environment strategies ranked among the top category for recruitment and retention of rehabilitation therapists. This has also been found among nurses [44] where it was reported that addressing such factors can affect the overall success of the program [49]. Specifically, our findings showed that the top ranked strategy for both importance and feasibility was improving and maintaining the safety of rehabilitation professionals in the workplace. Specific strategies that could reduce aggression, abuse and violence in the workplace include: zero tolerance policies, access to employee support programs and providing assistance to rehabilitation professionals who work alone (i.e. home care and rural and remote areas). Since there is less control over the environment in the home care setting, safety may become a greater concern in one practice setting over another. This might suggest that because there is less control in environments such as home care, remote areas, and psychiatric settings, maintaining safety will be more difficult and that solutions will need to be tailored to these settings in order to ensure retention of providers. Although no studies have looked at implementing personal safety strategies for home care therapists, written policies and procedures for home care nurses during inclement weather and for dealing with abusive or dangerous <span class="yellow">patients</span>, families and neighbourhoods have been reported [50].
In addition to the above, ensuring open and timely communication between employer and worker was also ranked highly among the recruitment and retention strategies. Examples of strategies include: open door policies, employee advisory committees and regular staff meetings and evaluations. Although these strategies were reported to be used among organizations providing services to <span class="yellow">persons</span> with developmental disabilities in Alberta, there was no description of the organizations, sample size or methodology [51]. Similar findings were found in a qualitative study among 16 nurses working from diverse practice settings (acute, long-term care, rehabilitation and community; from both urban and rural areas) in a health region in western Canada. From the semi-structured interviews, study <span class="yellow">participants</span> expressed a desire for improved consultation and communication with nurses regarding changes to the health care system [52].<br><br>Financial Incentives and Marketing
Another area that was ranked highly was marketing strategies to increase high school student and public awareness of rehabilitation careers. These specific strategies were also recommended by the Ontario Hospital Association (OHA) in order to establish a competitive position for Ontario hospitals in respect of recruitment and retention of health care professionals [53]. Similar strategies have been developed by the American Physical Therapy Association (APTA) in response to the declining number of students applying to Physical Therapy Education Programs [54]. To address this trend, APTA developed a campaign to promote Physical Therapy as the profession of choice to high school and college students across the United States. The potential components of their plan included: developing a "Recruiting Kit" for educators, students and various APTA members to be used in the high school and college settings to introduce Physical Therapy as a career; establishing public relations initiatives that demonstrate the role of Physical Therapy in the public arena targeting minority groups that are underrepresented in the profession; and creating alliances with professional associations of high school guidance counsellors and educators. Based on our finding, the APTA model may have applicability in Ontario.<br><br>Workload and Skill Mix
Of the six Workload and Skill Mix strategies only two were highly rated: implementing a caseload management database and using support personnel. Caseload management has been identified in the literature as an issue affecting all three rehabilitation professions. For example, in physiotherapy, Christie's study [55] found that caseload expectations tended to be significantly higher than the reality and that caseload varies across different programs. Similarly, the Canadian Association of Speech-Language Pathologists and Audiologists (CASPLA) survey indicated that factors affecting the workload of SLPs include delivery models, client disorder, severity and work setting [56]. A literature review and environmental scan undertaken by the Canadian Association of Occupational Therapists (CAOT) proposed that guiding principles for caseload management should include: evidence-based occupational therapy, cost-effectiveness, accountability, professional leadership and expert judgment, comprehensiveness and flexibility [57]. Therefore, upon implementation of a caseload management database for rehabilitation, key factors to consider include workplace setting and client service delivery models.
The other highly rated Workload and Skill Mix strategy was the use of support personnel (i.e. physiotherapy assistants or exercise therapists) to increase efficiency of utilization of scarcer and higher order rehabilitation competencies. Considerations for implementing this strategy include addressing key issues such supply, standards of education, standards of practice and accreditation. These are highlighted in an article by Salvatori [58] who reported that the actual number of OT personnel delivering OT services in Canada remains unknown and that there are no national standards of education nor accreditation process for OT assistants. CAOT believes that in order to utilize support personnel appropriately, studies on <span class="blue">human</span> resource needs for occupational therapy and support personnel are first needed with input from OTs, stakeholders, funders, decision makers and health policy planners [59]. Since there is a lack of competency profiles related to the role, responsibilities, and supervision of assistants, particularly with regards to delivering services in unsupervised community-based settings, the type of workplace setting where this strategy may be implemented should be considered [58].<br><br>Education and Training
Given that 60% of highly rated Education and Training strategies targeted rural and remote practices underscores the importance of specific strategies for rural and remote areas in the development of a HHR plan. The need to build on existing mechanisms to expand the availability of rural and remote clinical placements by providing financial and accommodation support was ranked among the top two most important and feasible education strategies for rehabilitation professionals. Not only has this strategy been used as a recruitment tool for rehabilitation students, it has also been reported by Solomon et al. [34] to be effective in retaining OTs and PTs in underserviced Northwestern Ontario communities. Respondents from Solomon et al.'s study reported that the top three benefits of supervising students were that it stimulates thinking, it provides opportunity to contribute to the profession and that it provides access to current information. The reported disadvantages however was that it was time-consuming and students contributed stress to the working environment. Similarly, a two-part study found substantial gaps between financial incentives students deem important in the creation of an appealing clinical placement opportunity and the actual provisions offered to them by Southeastern Ontario communities [60,61]. Although OT and PT students reported that they were more willing to complete a clinical placement in an underserviced community if provided travel stipends, rent-free housing and interprofessional education opportunities, the majority of these incentives were only available to medical students. In addition to training students for rural and remote practice, a longitudinal study reported that perceived opportunity for career development was the most significant factor related to job turnover and regional attrition among physiotherapists working in Northern Ontario [5]. Therefore developing workforce strategies for rehabilitation therapists working in these areas should be among one of the priority areas in HHR planning.<br><br>Professional Development
Our findings indicate that the theme with the largest number of strategies that were considered important and feasible to implement as part of HHR planning was professional development. Many were specific to rural and remote areas. Although the importance of continuous professional development (CPD) in recruitment and retention is well recognized, a Canadian study reported several barriers to its implementation [62]. In the case of OTs employed in public settings in Nova Scotia, Townsend et al. (2006) found that the most powerful deterrent for CPD was the lack of support from workplace policies. Based on their study results, the use of CPD as a recruitment and retention strategy was highly influenced by gender issues, work-life balance, career advancement, working conditions, geographical location, professional versus employer responsibility, and employee benefits. Although occupational therapy is a female-dominated profession, workplace policies did not address issues of gender. For example, therapists in this study indicated that CPD competes with family commitments, therefore these activities are "done largely during personal time, mainly at their own cost, and on top of childcare, eldercare, homemaking and other family responsibilities" [62]. In addition, heavy workloads, lack of salary and career incentives, and lack of policy and funding support are all barriers to CPD. These issues become more pronounced in rural and remote settings because smaller communities often only have one therapist; hence the systemic pressure of workload demands makes it difficult for the therapist to leave <span class="yellow">patient</span> care. OTs from this study also questioned who was responsible for CPD. Some felt that it was the professional's responsibility while others felt that it was the responsibility of the employer to provide CPD opportunities. The primary limitation employers faced was the lack of financial resources, however giving employees time off without pay was an alternative strategy utilized instead of funding professional development activities. Although there are professional and provincial variations in funding for CPD across Canada, these results are informative in that it highlights the need for employers to consider how workplace policies can affect recruitment and retention strategies.
Limitations of this study should be noted. First, the majority of the strategies were obtained from the grey literature that is not subject to the same scrutiny as the peer-reviewed literature. Second, almost none of the strategies were specifically developed for rehabilitation professionals and in many cases had to be re-worded to fit the rehabilitation context. There is a lack of research on rehabilitation clinicians' perspectives on recruitment and retention strategies; therefore future research should focus on investigating this area. Third, during the face-to-face meeting, bias could have resulted from panelists whose opinion may have influenced others significantly, especially if members came from similar workplace settings. The facilitator of the expert panels however, followed a strict process for managing the discussion and ensured that all panelists were given the opportunity to express their opinions.
Finally, although some strategies such as competitive wage packages, training/growth opportunities and professional development are viewed as both a recruitment and retention incentive, other strategies do not overlap and are appropriate for only one of the two tasks. For example, increasing public awareness of rehabilitation careers, providing rural and remote orientation packages and family relocation programs are only appropriate for attracting a worker while ensuring open and timely communication may be seen as a strategy only for retention. Future research should therefore consider studying recruitment and retention strategies separately so that a distinction between the two can be made.<br><br>
Conclusion
This study identified 34 strategies that should be considered as important and feasible for implementation as part of HHR planning for rehabilitation professionals. Although the highest ranked strategies focused on areas of Quality of Worklife and Work Environment, Financial Incentives and Marketing and Professional Development, key factors that need to be considered in the context of implementation include: workplace setting, geographical location and gender issues. While this is the first study to our knowledge that provides a comprehensive list of recruitment and retention strategies relevant to rehabilitation professionals, more information is needed for the development of a HHR plan. Information on trends in labour force participation as well as knowledge regarding the use and effectiveness of recruitment and retention strategies for rehabilitation professionals is needed. More importantly, the success of implementing and sustaining such strategies requires future research to validate these strategies from the perspective of rehabilitation clinicians and <span class="blue">human</span> resource decisions makers (i.e. local government, stakeholders, etc.) so that specific barriers and challenges can be identified.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
DT participated in the design of the study, conducted the literature review, analyzed and interpreted the results and drafted the manuscript.
LMH, AD, DB and KB were involved in providing feedback and editing the content of the manuscript.
MDL recommended <span class="yellow">participants</span> for the expert panel and was involved in providing feedback and editing the content of the manuscript.
SJ participated in the design of the study, recommended <span class="yellow">participants</span> for the expert panel, organized and consolidated strategies, interpreted the data and edited the content of the manuscript.<br><br>Pre-publication history
The pre-publication history for this paper can be accessed here:<br><br><br><br><h3>pmcA2652712</h3>Neurogenin2 Directs Granule Neuroblast Production and Amplification while NeuroD1 Specifies Neuronal Fate during Hippocampal Neurogenesis
Abstract
The specification and differentiation of dentate gyrus granule neurons in the hippocampus require temporally and spatially coordinated actions of both intrinsic and extrinsic molecules. The basic helix-loop-helix transcription factor Neurogenin2 (Ngn2) and NeuroD1 are key regulators in these processes. Based on existing classification, we analyzed the molecular events occurring during hippocampal neurogenesis, primarily focusing on juvenile animals. We found that Ngn2 is transiently expressed by late type-2a amplifying progenitors. The Ngn2 progenies mature into hippocampal granule neurons. Interestingly, the loss of Ngn2 at early stages of development leads to a robust reduction in neurogenesis, but does not disturb granule neuron maturation per se. We found that the role of Ngn2 is to maintain progenitors in an undifferentiated state, allowing them to amplify prior to their maturation into granule neurons upon NeuroD1 induction. When we overexpressed Ngn2 and NeuroD1 in vivo, we found NeuroD1 to exhibit a more pronounced neuron-inductive effect, leading to granule neuron commitment, than that displayed by Ngn2. Finally, we observed that all markers expressed during the transcriptional control of hippocampal neurogenesis in rodents are also present in the <span class="blue">human</span> hippocampus. Taken together, we demonstrate a critical role of for Ngn2 and NeuroD1 in controlling neuronal commitment and hippocampal granule neuroblast formation, both during embryonic development and in post-natal hippocampal granule neurogenesis.<br><br>Introduction
Neurons are born not only during development of the central nervous system, but neurogenesis also continues into adulthood. In both <span class="yellow">rodent</span> and <span class="blue">human</span> adult brain, neurogenesis is active in two distinct zones of the forebrain: the subventricular zone (SVZ) of the lateral ventricle and the subgranular zone (SGZ) of the hippocampal dentate gyrus (DG) [1]–[5]. The molecular mechanism underlying neurogenesis in the DG is not fully understood. Clearly, a cascade of transcriptional events controls the specification of neuronal identity in the DG [3], [6]–[8], but details of the expression pattern and function of each transcription factor remain elusive.
The paired-box homeodomain transcription factor Pax6 and the bHLH transcription factors Ngn2 and NeuroD1 are important when cells acquire a pan-neuronal character and a specific neuronal subtype [9], [10]. In the developing neocortex, Pax6 is expressed in dividing radial glial cells at the ventricular surface [9]. In the adult hippocampus, Pax6 is expressed in astrocytes in the SGZ and are considered to be the true stem cells [9], [11]. Loss- and gain-of-function studies indicate that Pax6 is involved in regulating the proliferation of neocortical and hippocampal progenitors [9], [12]–[16]. During neocorticogenesis, high concentrations of Pax6 induce the expression of the bHLH transcription factor Neurogenin2 (Ngn2) [17]. In turn, Ngn2 causes cell cycle exit [18], an event that takes place when NeuroD1 starts to be expressed [19]. In the developing hippocampus, the absence of Ngn2 leads to a reduced number of granule neurons [20]. NeuroD1, on the other hand, is essential for the differentiation and survival of hippocampal granule neurons [10]. Interestingly, in the absence of Ngn2, NeuroD1 can still be activated and neuronal differentiation can still take place [20]. This has led to the idea that the primary role of Ngn2 is not to direct neuronal differentiation.
Only a small number of studies have addressed the role of Ngn2 and NeuroD1 in hippocampal neurogenesis [10], [20], [21]. Therefore, we now re-examine their roles in hippocampal neurogenesis in detail, using gain- and loss-of-function experiments. We first establish a hierarchy of transcriptional events that occur during neurogenesis in the DG and then define the place that Ngn2, NeuroD1 and other transcription factors have in this cascade. We find that a lack of Ngn2 expression result in a markedly smaller hippocampus and an almost complete absence of the DG. We show that Ngn2 is required for granule neuroblasts production/amplification. Gain-of-function of NeuroD1 during development of the DG results in an efficient generation of granule neuroblasts, an effect that we do not observe when we overexpress Ngn2. Finally, we demonstrate that the same transcription factors and cellular markers seen in <span class="yellow">mouse</span> and <span class="yellow">rat</span> tissue are also present in the <span class="blue">human</span> hippocampus.<br><br>Results
Expression of transcription factors and cellular markers define different phases of postnatal hippocampal granule neuron maturation
We first analyzed in detail the chronology of expression of different transcription factors and cell-specific markers during DG granule neuron formation and correlated our findings to the previously established classification of adult hippocampal neurogenesis [3]. The current classification describes hippocampal stem cells and progenies into three categories (type-1, -2 and -3 cells) depending on the markers they express as they mature. In the adult <span class="yellow">mouse</span> DG, only a very small proportion of cells undergo mitosis at any one given time [22]. Consequently, in adult animals it is difficult to analyze which transcription factors are involved in the transition phase from one cell category to the next. Therefore, we studied 2 week-old rodents which have higher numbers of maturing granule neurons in the DG. We performed triple immunohistochemistry for all markers of interest and compared results from 2 week- and 2 month-old <span class="yellow">mice</span> (Figure 1 and figure S1).
We first confirmed that stem cells in the SGZ exhibit characteristics of radial glia [3], [11], [16]. They extend radial processes from the SGZ to the apex of the subgranular layer (SGL) and the basal molecular layer. Moreover, they express the intermediate filament protein nestin, glial fibrillary acidic protein (GFAP) and glial glutamate transporter (GLAST) [16] (Figure 1A and B and figure S1A and S1B). The GFAP- and GLAST+ cells are defined as type-1 cells [3]. They also express the paired-homeodomain transcription factor Pax6 (Figure 1B). These radial glia-like stem cells divide relatively infrequently, and are believed to undergo symmetric division (giving rise to two identical stem cells) under some conditions. They can also divide asymmetrically and give rise to a new stem cell and one neuronal progenitor, which usually is defined as a type-2 cell (Figure 2B) [3].
Type-2 cells are divided into two different populations: type-2a expressing nestin and type-2b co-expressing nestin and doublecortin (Dcx) [23]. Based on a previous report [24], we examined the expression pattern of the T-domain transcription factors Tbr1 and Tbr2. We found that Tbr2 is expressed in type-2a cells, in agreement with a recent report [25]. In the SGZ of 2 week-old <span class="yellow">mice</span>, we found an average of 50–70 Tbr2+ cells per 30 µm-thick section throughout the rostro-caudal axis of the dorsal hippocampus. This number was reduced by 35% in 2 month-old <span class="yellow">mice</span> (Figure 1C with figure S1C and data not shown). As opposed to Tbr1 (Figure 1D and figure S1D), Tbr2+ cells did not co-express Dcx or PSA-NCAM (Figure 1C and figure S1C). We observed that Tbr1-immuoreactivity gradually decreased as neurons matured and started to express NeuN (Figure 1H and figure S1H). Thus, Tbr2 labels type-2 cells, while Tbr1 is expressed by immature granule neurons.
We then examined the molecular phenotype of type-3 cells, a cell type that transiently expresses Calretinin [26]. We hypothesized that hippocampal type-3 cells might also express the bHLH transcription factor NeuroD1, known to partially overlap with Tbr2 and Tbr1 in different brain regions [24]. We found that NeuroD1 expression in the hippocampus starts in Tbr2+ cells and extends to post-mitotic Tbr1+ cells, which also express the hippocampal granule identity transcription factor Prox1 (Figure 1G and 1I and figure S1G and S1I). While Tbr1 expression ceases during granule maturation, that of NeuroD1 is weakly maintained when NeuN expression starts (Figure 1G and 1H and figure S1G and S1H). We found that calretinin expression decreased during granule neuron maturation, before NeuroD1 was reduced (Figure 1F). Thus NeuroD1 is expressed in type-2b and type-3 cells, as well as immature granule neurons. Finally, we found that NeuroD2 starts to be expressed just after NeuroD1 and, unlike NeuroD1, continues to be highly expressed in mature neurons (Figure 1J and data not shown).<br><br>Mash1 and Ngn2 define the early versus late type-2a stage in 2 weeks old <span class="yellow">mice</span>
Having established the hierarchy of transcription factors and cellular markers appearing during hippocampal neurogenesis, we set out to clarify the position of the two bHLH transcription factors Ngn2 and Mash1, in this process. We found that more than 90% of Ngn2-immunoreactive cells are Pax6 positive (Figure 2A), and almost all co-express Tbr2 (Figure 2B). Thus, Ngn2 appears to be expressed by type-2a cells. As our antibodies against Ngn2 and NeuroD1 were made in the same species, we could not determine if there was an overlap between these two proteins. However, by comparing their overlapping expression pattern with that of Tbr2, we propose that some Ngn2+ cells co-express NeuroD1, and that they appear at the onset of NeuroD1 expression and downregulation of Ngn2. We base this assumption on the facts that more than 50% of the Tbr2+ cells are Ngn2+ and more than 50% of Tbr2+ cells also express NeuroD1. The expression of Ngn2 in adult hippocampus was hard to detect using immunohistochemistry due to the low expression of the protein (data not shown). However, it is present, as previously described using reporter <span class="yellow">mice</span> and as revealed by in situ hybridization [21].
We next examined the expression of Mash1 and compared it with that of Ngn2, Pax6 and Tbr2. We found some Mash1+ cells (10–20 cells per section) in the dorsal hippocampus. Almost all of them co-expressed Pax6 (Figure 2C). Only few cells co-expressed Tbr2 (Figure 2D) or Ngn2 (Figure 2E), suggesting that Mash1 is downregulated when these two proteins are expressed. None of the Mash1-expressing cells were positive for NeuroD1 (Figure 2F). Mash1+ cells were still undergoing mitosis (Figure 2C) and consequently over 90% of them co-labeled with the cell-cycle marker Ki67 (Figure S1J).
Taken together, we show that Mash1 and Ngn2 are expressed in early and late stages of maturation of type2a progenitors, respectively, and that both transcription factors are co-expressed briefly when cells transit from early to late phase type2a cells.<br><br>Ngn2 progenies become hippocampal granule neurons
Previous work has shown that in Mash1 null mutant <span class="yellow">mice</span> hippocampal neurogenesis is not reduced and the hippocampus is not malformed [20]. Therefore we focused our initial analysis on the role of Ngn2 in hippocampal neurogenesis. We characterized the fate of Ngn2-expressing cells using Ngn2 Knock-in green fluorescent protein (GFP) <span class="yellow">mice</span> (Ngn2+/GFP) [27]. In the SGZ of 2 week-old hippocampi, we found bright GFP-expressing cells that were positive for Tbr2 (Figure 2H), the immature neuronal marker PSA-NCAM (Figure 2I) and NeuroD1 (Figure 2J). Furthermore, Prox1+, NeuroD2+ and calretinin+ cells were also weakly positive for GFP. We made similar observations in 2 month-old <span class="yellow">mice</span> (data not shown) confirming earlier published results [21]). We did not observe Mash1+/GFP+ and NeuN+/GFP+ cells (Figure 2G). These data from Ngn2+/GFP <span class="yellow">mice</span> are entirely consistent with our earlier observations using immunocytochemistry to label Ngn2-expressing cells and with GFP undergoing slow degradation after the expression of Ngn2 has ceased. Importantly, none of the GFP-expressing cells, including those only weakly fluorescent, residing in the SGL expressed the astrocytic marker GFAP or the oligodendrocytic markers CNPase (data not shown). Taken together, the data show that Ngn2 progenies become neurons and never generate astrocytes or oligodendrocytes.<br><br>Marked reduction of hippocampal granule neurons in absence of Ngn2
To determine the role of Ngn2 during the initiation of hippocampal granule neurogenesis, we analyzed the hippocampus of <span class="yellow">mice</span> lacking Ngn2 (Ngn2GFP/GFP). In contrast to the <span class="yellow">mice</span> use by Galichet and coworkers, our <span class="yellow">mice</span> have a lifespan of only 2 weeks after birth. Therefore, we analyzed 2 week-old and younger <span class="yellow">mice</span>. Consistent with previous observations [20], the hippocampus in Ngn2GFP/GFP mutants is clearly malformed (Figure 3A). In two week-old Ngn2GFP/GFP <span class="yellow">mice</span>, the ventral blade of the DG is completely absent. Already on postnatal day 1–2, the ventral blade of the DG is malformed along its whole rostro-caudal axis (Figure 3B).
As neither migration defects nor cell death cause the reduced number of hippocampal neurons in Ngn2 null <span class="yellow">mice</span> [20], we asked if Ngn2 is necessary for production of hippocampal granule neuroblasts. We first injected 2 day-old <span class="yellow">mouse</span> pups with BrdU 2 hours prior to sacrifice, allowing us to evaluate cell proliferation and ongoing neurogenesis. We also injected BrdU into female <span class="yellow">mice</span> on their final day of pregnancy and examined the brains of their pups 48 hours later (corresponding to P1–P2 old pups). This allowed us to assess the number of neurons derived from the last day of intrauterine development. Regardless of whether the <span class="yellow">mice</span> were WT or hetero/homozygous mutant pups, we found cells that had incorporated BrdU in the hippocampal subventricular zone (hSVZ), fimbria and the DG (Figure 4A and B; [28]), suggesting that progenitors originating in the hSVZ divide while migrating towards the DG. Most of them were organized in chains typical of migrating cells (Figure 4A and B and figure S2). In Ngn2GFP/GFP mutant <span class="yellow">mice</span> injected with BrdU 2 hours prior to sacrifice, we observed a decrease in the number of BrdU+ cells compared to heterozygous littermates. Thus, the numbers of newborn cells were reduced in the hSVZ/fimbria (133.3±7.5 for Ngn2+/GFP vs 69.7±5.0 for Ngn2GFP/GFP) and DG (132.0±9.1 for Ngn2+/GFP vs 42.6±2.2 for Ngn2GFP/GFP)(Figure 4C and D). We also examined the number of newborn cells differentiating into neurons, and identified them by the co-expression of BrdU and NeuroD1. As expected, they were relatively few in numbers because the short delay (2 hours) between BrdU administration and sacrifice in this first experimental paradigm. In Ngn2GFP/GFP null mutants, we found the number of cells differentiating into neurons to be reduced by 60% and 71% in the hSVZ/fimbria (8.6±1.4 for Ngn2+/GFP vs 3.4±0.5 for Ngn2GFP/GFP) and DG (9.7±1.1 for Ngn2+/GFP vs 2.8±0.6 for Ngn2GFP/GFP), respectively, (Figure 4E and F). As a result, the number of NeuroD1+ cells in the DG was reduced by 80% (580.3±27.7 for Ngn2+/GFP vs 116.2±7.3 for Ngn2GFP/GFP; Figure 4G).
We obtained similar results in the second paradigm, i.e. in <span class="yellow">mice</span> that we sacrificed 48 hours after BrdU administration and in which a larger number of the newborn cells had time to mature into neurons. In this case, the number of BrdU+/NeuroD1+ cells in the DG was reduced by 73% of Ngn2GFP/GFP mutant animals (91.5±24.1 for Ngn2+/GFP vs 26.4±3.3 for Ngn2GFP/GFP) (Figure 4J). We found no differences in the proportion of BrdU+ cells that expressed NeuroD1 in Ngn2+/GFP and Ngn2GFP/GFP <span class="yellow">mice</span> (14.2±1.0% and 17.4±1.8% respectively; Figure 4K). This shows that the few cells that manage to proliferate in the Ngn2 null <span class="yellow">mice</span> have the same ability to differentiate into neurons as those in <span class="yellow">mice</span> with one Ngn2 allele.
Interestingly, the absence of Ngn2 did not alter the identity of neurons in the DG granule layer in 2 week-old Ngn2 null <span class="yellow">mice</span>. The maturing granule progenitors sequentially expressed Pax6, Tbr2 (Figure S3A and S3B), NeuroD1 and Calretinin (Figure 4L), PSA-NCAM (Figure 4N), Tbr1 (data not shown) and Prox1 (Figure 4M). We confirmed that the same transcriptional cascade is active in the 20% (compared to <span class="yellow">mice</span> with one Ngn2 allele) residual granule neurons that are formed two day-old in Ngn2GFP/GFP <span class="yellow">mice</span> (Figure S3D–G and data not shown).
We next investigated if the cells that failed to develop into neurons in Ngn2GFP/GFP mutant <span class="yellow">mice</span>, became glial cells. We found that cells expressing GFP never co-labeled with the astrocyte marker GFAP (Figure 4N) or the oligodendrocyte marker CNPase (data not shown).
Altogether, our results show that Ngn2 plays an important role during the production/amplification of hippocampal granule neuroblasts, but not the acquisition of the granule neuron identity (Figure S7).<br><br>Ngn2 controls the amplification of granule neuron progenitors
We next monitored the mitogenic activity of Ngn2+ cells in the DG of 2 week-old WT <span class="yellow">mice</span>. We observed that over 50% of Ngn2+ cells in the DG of WT <span class="yellow">mice</span> co-expressed the mitosis marker Ki67 (Figure 5A). Inspired by this finding, we injected BrdU into 2 week-old WT and Ngn2GFP/GFP <span class="yellow">mice</span> and compared cell proliferation in the DG. We observed 92% reduction in number of proliferating cells in the SGZ of Ngn2 null mutant <span class="yellow">mice</span> (6.7±0.5) compared to WT (83.6±1.2) and heterozygotic (data not shown) littermates (Figure 5B–C). This data confirm the importance of Ngn2 during the amplification of granule progenitor.s.
To further explore whether Ngn2+ cells become post-mitotic or still proliferate after Ngn2 is downregulated, we performed immunohistochemistry on sections through the DG of Ngn2+/GFP <span class="yellow">mice</span>. We stained them with the mitosis marker Phospho-histone 3 (PH3) and NeuroD1, which is downstream of Ngn2 in the transcriptional cascade controlling neurogenesis. Thus, in the same sections we could identify whether cells that had initiated Ngn2 expression (GFP labeled), continued to divide (PH3+) or committed to neuronal differentiation (NeuroD1+). We observed some GFP+ cells that co-expressed PH3. They all exhibited morphological characteristics of one of the five mitotic phases: prophase, metaphase, anaphase, telophase and cytokinesis (Figure 5D and data not shown). Unexpectedly, cells that colabeled for GFP, PH3 and NeuroD1 (Figure 5E and figure S4A and S4B) were rare. This indicates that Ngn2+ cells undergo division/amplification and that they mature into post-mitotic neurons upon NeuroD1 expression. In <span class="yellow">mice</span> lacking Ngn2, we found the cells expressing GFP localized to the malformed ventral blade of the DG (Figure 5F and G). All of these cells were Ki67+ and they only very rarely expressed PH3 (Figure 5F and G, and figure S4C) in the absence of Ngn2 most of the cells are arrested in the cell cycle prior to entering the M phase and their mitosis is impaired. These data suggest that the mechanism of action of Ngn2 is conserved from development of the DG to postnatal hippocampal neurogenesis.
Our findings suggest that Ngn2 regulates amplification and cell cycle exit of DG granule progenitors. To examine this hypothesis, we compared the effects of Ngn2 and NeuroD1 on mitotic activity in embryonic cortico-hippocampal neurosphere-derived progenitors, 5 days upon transduction, in vitro. In cultures transduced with Ngn2 retrovirus, we found that 27.2% (±4.1%) of the PH3+ cells had been transduced (Figure I1–J). These cells were immunopositive for MAP2 (Figure 5I1 and I2) and therefore represented dividing neuroblasts. By contrast, in NeuroD1 transduced cultures, only 7.3% of PH3+ cells were GFP+ (Figure 5H1, H2 and J). All of the NeuroD1-transduced cells became MAP2+ (Figure 5H). To confirm this data we pulse-labeled transduced cultures with chlorodeoxyuridine (CldU) for 48 hours, five days after differentiation. In contrast to NeuroD1-transduced cultures, we observed dividing cells transduced with Ngn2 retrovirus that were positive for PH3 and that had incorporated CldU (Figure 5K–L2). As the number of cells PH3+/CldU+/GFP+ we observed was low, we did not quantify this finding.
Collectively, we have shown that Ngn2 is required for granule neuroblast production and amplification and that in the absence of Ngn2 the progenitors arrest in the cell cycle. NeuroD1, on the other hand, induces cell cycle exit and promotes rapid neuronal maturation.<br><br>NeuroD1 directs neuronal differentiation and maturation
Based on our previous observations we proposed that Ngn2 primarily controls amplification of granule neuroblasts and NeuroD1 directs neuronal differentiation. To test this hypothesis, we overexpressed Ngn2 or NeuroD1 in E14.5 cortico-hippocampal neurospheres and compared their effects after 5 days of differentiation of the progenitors. All cortico-hippocampal progenitors expressed Pax6 prior to differentiation (Figure S5). After 5 days, both factors suppressed Pax6 and Sox2 (Figure S5, figure 6C) and induced expression of Tbr1, Map2, NeuroD1 and PSA-NCAM (Figure 6D and E).
Interestingly, all Ngn2-overexpressing cells co-expressed NeuroD1. Based on these findings, we next explored whether Ngn2 overexpression induces NeuroD1 expression, and if NeuroD1 in turn directs neuronal differentiation. We compared the effects of both transcription factors in progenitors that either do or do not normally express them. Thus, we expressed Ngn2 in E14.5 neural progenitors isolated from three different brain regions: cortex/hippocampus, lateral ganglionic eminence (LGE) and ventral mesencephalon (VM). Ngn2 expression in the developing forebrain is normally limited to the neocortex, and it does not appear in LGE tissue [29]. In the developing VM, both Ngn2 and NeuroD1 are expressed, but not when neural progenitors from this region are cultured in vitro [30], [31]. In our experiments, cortico-hippocampal progenitors could differentiate into neurons, although they did not normally express NeuroD1 (Figure 6E). When we overexpressed Ngn2 in VM and LGE progenitors, the cells started to express both NeuroD1 and PSA-NCAM (Figure 6F and G). We then overexpressed NeuroD1 in the same types of cultured progenitors and found that the cells became immunoreactive for PSA-NCAM (Figure 6G and data not shown). Thus, NeuroD1 is sufficient to direct neuronal differentiation in cortico-hippocampal-, LGE- and VM-derived progenitors.
Our results from in vitro cultures and the analysis of the DG of Ngn2 mutant animals collectively show the neuron-inducing effect of NeuroD1 in hippocampal granule cell progenitors [32].<br><br>NeuroD1 directs exclusive neuronal differentiation of hippocampal granule neuron progenitors
As the next step, we tested the effects of NeuroD1 in vivo and compared them with those of Ngn2. We injected retroviruses carrying the gene for either Ngn2 [33] or NeuroD1, and the reporter gene eGFP into the ventricles E15.5 <span class="yellow">rat</span> embryos, in utero (Figure 7A). Three weeks later, when the <span class="yellow">rats</span> were about two weeks old, we examined their hippocampi and analyzed the eGFP+ cells. The <span class="yellow">rats</span> injected with control vector exhibited eGFP+ cells within the hippocampus that were either star-shaped, progenitor/glial-like cells (14.8±3.4%) or neuron-like cells with long neurites (85.3±3.4%) (Figure 7B). In <span class="yellow">rats</span> that we had injected with the vector encoding Ngn2, the eGFP-labeled, the transduced hippocampal cells were composed of 31.9% (±5.7%) progenitor/glia-like cells and 69.4% (±6.1%) neuron-like cells (Figure 7B). The glia-like subpopulation was immunopositive for the astrocytic marker GFAP (Figure 7C–D). In contrast, virtually all of the transduced cells in <span class="yellow">rats</span> injected with the NeuroD1 vector became neuron-like cells (99.9%±0.1; Figure 7B). None of these cells stained for GFAP (Figure 7E and F). They were positioned in the external layers of both the ventral and dorsal blades of the DG (Figure 7B3, E and I and figure S6A).
When we examined neuronal maturation of Ngn2- and NeuroD1-transduced cells, we observed that only a few of those transduced with the NeuroD1 vector expressed the early mature neuronal marker Tbr1 (3.1±1.4%; Figure 7I and J). They were located within the basal layers, near the SGL, of the ventral and dorsal blades of the DG. In contrast, a greater proportion of the cells transduced with Ngn2 still expressed Tbr1 (19.6%±3.3%; Figure 7H and J), indicating that they were less mature than the vast majority of the NeuroD1-transduced cells. Immunohistochemistry for NeuN confirmed these data (Figure 7M and O, and figure S6A). Indeed, when we overexpressed Ngn2 the proportion of hippocampal progenitors that became NeuN+ was no greater than in <span class="yellow">rats</span> transduced with the control virus (Figure 7F).
As a whole, our in utero injection experiments confirm that NeuroD1 has a stronger neuron-inducing effect than Ngn2 when overexpressed in hippocampal progenitors.<br><br>Involvement of Mash1, Ngn2, Tbr and NeuroD proteins during <span class="blue">human</span> hippocampal neurogenesis
Finally, we examined whether the transcription factors and cellular markers that are expressed in rodents are also expressed in <span class="blue">human</span> hippocampus. We performed immunohistochemistry on aged <span class="blue">human</span> hippocampal DG and found GFAP-, Sox2-, Pax6-, Nestin-, Prox1- and NeuN-immunopositive cells. This indicates that radial glia-like stem cells, neural progenitors and mature granule neurons are present in the aged <span class="blue">human</span> hippocampus (Figure 8A–E). While Prox1 is found mainly in granule neurons, Sox2 and Pax6 are exclusively localized to cells in the SGZ (Figure 8D and E).
As for aged rodents, we did not observe the presence of Mash1, Ngn2, Tbr2, Tbr1 and NeuroD1 proteins in aged <span class="blue">human</span> hippocampus, using immunohistochemistry. However, we could identify the presence of the transcripts, using RT-PCR (Figure 8F). In addition, we detected mRNA for Prox1, Calbindin1 and Calbindin2/Calretinin (Figure 8F). Interestingly, we also observed Sonic Hedgehog and Wnt-3A transcripts, indicating that they are present in the <span class="blue">human</span> DG and supporting previous claims that they are important for the regulation of adult hippocampal neurogenesis (Figure 8F; [34], [35]). Finally, we also found mRNA for GLAST, Pax6 and GFAP which suggests that radial glia-like stem cells are present in the adult <span class="blue">human</span> hippocampus, and may play a role in adult <span class="blue">human</span> hippocampal neurogenesis (Figure 8F; [8]).
Overall, our data show that different proteins known to play key roles in <span class="yellow">rodent</span> hippocampal granule neurogenesis are present in the adult <span class="blue">human</span> hippocampus.<br><br>
Discussion
In this study we determine the functions of Ngn2 and NeuroD1 during hippocampal neurogenesis. First, we map the hierarchy of molecular markers of neurogenesis in the DG. Second, we describe that Ngn2 is necessary for granule progenitor production/amplification. Third, we demonstrate that NeuroD1 directs neuronal differentiation of granule progenitors. Finally, we show that different cellular markers expressed during hippocampal neurogenesis in rodents are present in <span class="blue">human</span>.
Sequential expression of different transcription factors and cellular markers during hippocampal neurogenesis
We clarified the pattern of expression of various markers expressed during postnatal and adult hippocampal granule neurogenesis in detail. We found that the transcription factor Pax6 is initially expressed by both radial glia stem cells (type-1) and early amplifying progenitors (type-2a). The next transcription factor to appear in chronology is Mash1. Mash1 expression characterizes the early stage of hippocampal progenitor amplification (early type-2a). The role of Mash1 during hippocampal granule neurogenesis is still unclear. Mash1 null mutant <span class="yellow">mice</span> do not display any clear malformation of the DG [20]. However, overexpression of Mash1 alone in the DG leads to the generation of oligodendrocytes [36]. Based on these observations, we hypothesize that Mash1-expressing cells are not yet committed towards a granule cell fate. They may still have the potential to generate both neurons and oligodendrocytes, as is the case in the SVZ [37].
In agreement with different developing brain regions, we found that Ngn2 starts to be expressed in Mash1-positive transiently amplifying progenitors, i.e. “late type-2a cells” according to the classification we propose. We observed that Pax6 expression persisted longer than that of Mash1, in Ngn2-expressing cells (late type-2a cells), in juvenile but not adult DG (Fig. 1 and figure S1). Later on, Ngn2 is downregulated, whilst Tbr2 expression persists (type-2b cells). The transition from amplifying progenitor to neuroblast is defined by the expression of NeuroD1. Thus, Ngn2 is expressed at the beginning of the transiently amplifying progenitor phase while NeuroD1 marks the end of that period. This applies both to the juvenile and adult <span class="yellow">rat</span> brain [21], [25]. After NeuroD1 is turned on the progenitors leave the cell cycle, gradually mature, express PSA-NCAM, NeuroD2, Calretinin, Prox1, Tbr1 and, finally, NeuN. NeuroD1 expression persists at low levels in mature neurons (Figure 9; [10]). We found the sequential expression of these markers conserved in juvenile (P2 and 2 weeks) and adult <span class="yellow">rodent</span> brains. Importantly, we observed that the same transcription factors and neuronal markers are also present in the adult <span class="blue">human</span> hippocampus and arranged spatially in a manner reminiscent with what we saw in rodents. It is possible, however, that some of these markers are expressed for longer or shorter periods at postnatal and adult stages of hippocampal progenitor maturation. Moreover, one can ask to which extent the number of divisions occurring during neuroblasts maturation is conserved at postnatal and adult stages of neurogenesis.<br><br>A new function for Ngn2 in maintenance of a progenitor state for granule neuron production/amplification during embryonic and postnatal hippocampal neurogenesis
In agreement with earlier work, we confirm that Ngn2 is indispensable for hippocampal development and plays a vital role in postnatal hippocampal granule neurogenesis. Ngn2 null mutant animals display a reduced size of the cornu ammonis and a malformed DG (Figure 3 and 4; [20]). We show that the number of newborn neurons (incorporated BrdU) is decreased in absence of Ngn2, but hippocampal granule neuron subtype specification is not affected, post-nataly. Likewise, neurons from other brain regions, e.g. ventral midbrain dopamine neurons, can be generated in the absence of Ngn2 [30], [31]. The reduced numbers of mature neurons in the hippocampus and ventral midbrain of Ngn2 null mutant animals does not appear to be due to cell death or migration defects [20], [31]. Instead it may be due to a defect in the generation and/or amplification of neuronal progenitors. Indeed, in the hippocampus of postnatal animals, we observed that cells lacking Ngn2 arrest in the cell cycle, maintain Pax6 expression and cease to proliferate.
When overexpressed in neural progenitors in vitro, Ngn2 lead to up-regulation of NeuroD1 and caused neuronal differentiation (Figure 6), which was not always the case when we overexpressed the same factor in vivo (Figure 7L, N and O). We also saw that Ngn2 does not always efficiently induce cell cycle exit. Typically some mitotically active eGFP+ cells were present in the DG of <span class="yellow">rats</span> injected with the Ngn2 retrovirus (Figure 7P), and we observed that cultured Ngn2-transduced cells are still capable of dividing. Thus, Ngn2 does not always promote cell cycle exit and neuronal commitment, but depending on the state of the cells, Ngn2 may instead promote alternate cellular fates [38]–[40]. Indeed, an earlier study has shown that Ngn2-overexpressing progenitors generate oligodendrocytes when grafted to the adult spinal cord [40]. We have also seen that Ngn2-overexpressing neural progenitors from the embryonic midbrain form astrocytes when grafted to the striatum (unpublished observations). Probably, Ngn2 cannot influence already committed cells, but rather would control neuroblasts production. One could speculate that Ngn2 oscillates during granule neuron formation, as recently demonstrated during neocorticogenesis [41], [42].<br><br>NeuroD1, but not Ngn2, is obligatory for granule neuron progenitor differentiation
Hippocampal granule progenitors can mature and express Tbr1 and Prox1, both during the development and postnatally, even in the absence of Ngn2. Because neuronal differentiation still occurs in Ngn2 null <span class="yellow">mice</span>, it is clear that compensatory, Ngn2-independent mechanisms induce NeuroD1 expression. One candidate is Ngn1, which is expressed during neocorticogenesis, the specification of olfactory sensory neurons and during embryonic <span class="yellow">rat</span> hippocampal development [43]–[47]. Both Ngn1 and Ngn2 regulate NeuroD1 [19], [48]. The introduction of two mutant forms of Ngn1, a deletion of the basic region of Ngn1 and a substitution of two amino acids in the C-terminal basic region, prevents NeuroD1 expression and neuronal differentiation [48]. The double null Ngn1/Ngn2 mutant displays a more severe phenotype than single gene (Ngn1 or Ngn2) mutant <span class="yellow">mice</span>. For example, the total brain size of the double mutants is much smaller [10], [32]. If the role of Ngn1 during hippocampal neurogenesis is to activate NeuroD1 and that of Ngn2 is to control granule neuroblasts production/amplification, the DG of double Ngn1/Ngn2 knockout <span class="yellow">mice</span> should resemble that of NeuroD1 null mutant <span class="yellow">mice</span>.
We demonstrated that NeuroD1 directs neuronal differentiation both in vitro and in vivo. In progenitors isolated from different embryonic brain regions and cultured in vitro, we found that overexpression of NeuroD1 induced neuronal differentiation. The neurons generated expressed PSA-NCAM, Dcx and MAP2. After in utero retroviral vector-mediated gene delivery, virtually all cells transduced with NeuroD1 became neurons. In contrast, progenitors transduced with Ngn2 or control retroviruses adopted a neuron-like morphology in only 70–85% of cases. Under in vitro cell culture conditions, when we overexpressed Ngn2 in progenitors derived from the LGE we observed robust expression of NeuroD1 and neuronal differentiation. These in vitro results differ from those we obtained when we transduced the embryonic brain with a viral vector expressing Ngn2. In this latter case, Ngn2 did not induce neurons and the transduced cells did not express NeuroD1. Therefore, Ngn2 appears to direct non-neuronal cell type specification in vivo [38]–[40]. Taken together, we have confirmed that NeuroD1 plays a key role in neuronal differentiation in the hippocampus, both during development and in the adult brain.<br><br>Concluding remark
We present a detailed classification of different stages in hippocampal neurogenesis. Our detailed molecular mapping of hippocampal neurogenesis allows for a more accurate analysis of how new factors stimulate neurogenesis at different steps in the development of granule neurons. Thereby we hope to facilitate the development of new agents, which stimulate endogenous progenitors in the treatment of diseases.<br><br>
Materials and Methods
Animal tissue preparation
The creation of the Ngn2 transgenic <span class="yellow">mice</span> was reported elsewhere (ref guillemot). Heterozygote male and female <span class="yellow">mice</span> were crossed to obtain WT, heterozygote (Ngn2+/GFP) and null mutant (Ngn2GFP/GFP) animals. Tails of the Ngn2 offspring were used to obtain DNA for determination of the genotype using a polymerase chain reaction (PCR) assay as previously reported [31]. As null mutant Ngn2GFP/GFP do not survive longer than 2.5–3 weeks after birth, Ngn2GFP/GFP and their littermates were sacrificed at the postnatal ages of two days (P2) or two weeks for this study. Neurogenesis was assessed in two weeks or two months old WT <span class="yellow">mice</span>. Sprague Dawley pregnant <span class="yellow">rats</span> were ordered from B&K Universal Ltd, Sollentuna, Sweden (hppt://www.bku.com). All animals were housed in groups with ad libitum access to food and water at a 12-h light/dark cycle. All experimental procedures conducted in this study had been approved by the Ethical Committee at Lund University.
For immunohistochemical analysis, <span class="yellow">mice</span> (from two weeks old and adult stage) and juvenile <span class="yellow">rats</span> were sacrificed by transcardial perfusion with saline for 5–10 minutes, followed by 4% paraformaldehyde (PFA) for 10 minutes. Brains were kept in PFA overnight at 4°C and subsequently cryopreserved in a 20–30% sucrose/0.1 M phosphate buffer solution until sectioning on a microtome apparatus (30 µm thickness sections, Microm Zeiss). Seven series of coronal sections were cut throughout the brain. Free-floating sections were preserved in antifreeze solution until immunohistochemistry was performed. Heads of postanatal two days old <span class="yellow">mice</span> were decapitated and soaked in PFA 4% for 24 hours, at 4°C and transferred into sucrose solution until sectioning on cryostat apparatus (16 µm thickness; Leica CM3000). Sections were mounted on Superfrost glass slides and stored at −80°C until immunohistochemistry was performed.<br><br>Cloning, subcloning, virus production and titer measurement
The Moloney leukemia-derived retroviral vectors used in this study, pCMMP-IRES2eGFP-WPRE and pCMMP-Ngn2-IRES2eGFP-WPRE were previously described [40], [49]. To generate the construct pCMMP-NeuroD1-IRES2eGFP-WPRE, <span class="yellow">mouse</span> NeuroD1 cDNA was amplified from a pCS2+mtNeuroD1 plasmid (kindly provided by Professor Jackie Lee, Denver university, Boulder, USA) by PCR to introduce the restriction sites PmeI in 5′ and XhoI in 3′. Amplification of cDNA was performed as previously described [49]. The construction was verified by enzymatic restriction and by DNA sequencing using BigDye 3.1 (ABI). All infectious particles were produced using the producer cell line 293VSV-G and as previously described [49]. The titer of each retrovirus was measured by flow-cytometry based on eGFP expression, four days following infection of HT1080 cells and ranged from 0.5×109−2.1×109 TU/ml (All details on how to produce infectious particles can be provided upon request).<br><br>Neurosphere generation, transduction and differentiation
Pregnant female <span class="yellow">Sprague-Dawley rats</span> (B & K Universal, Sollentuna, Sweden) were terminally anesthetized by an overdose of sodium pentobarbital (i.p., 60 ng/ml). Embryos at stage embryonic day E14.5 (Plug day as day 0) were collected and cortical-hippocampal neurospheres were generated following dissection of the dorso-posterior part of the cortical tissue, and generated as previously described [49]. In this study, second passage (P2) neurospheres were used to study the effect of the overexpression of Ngn2 and NeuroD1 on neuronal differentiation. Each well, containing an equal starting population of 200,000 cells/ml, corresponding to 15–25 neurospheres, was transduced independently with each retrovirus at a multiplicity of infection (MOI) of 1, in proliferation medium supplemented by protamine-sulfate (4 mg/ml, Sigma). To induce differentiation, the medium was replaced with normal basic differentiation medium two days post-transduction and subsequently changed every other day until fixation.<br><br>Immunocytochemistry, immunohistochemistry and microscopy
The antibodies used in this study are: <span class="yellow">rabbit</span> anti-GFAP (1∶1000; DAKO), <span class="yellow">mouse</span> anti-Nestin (1∶100; BD PharMingen), <span class="yellow">guinea pig</span> anti-Glast (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Prox1 (1∶1000; Covance), <span class="yellow">goat</span> anti-Ngn2 (1∶20; Santa Cruz), <span class="yellow">goat</span> anti-NeuroD1 (1∶200; Santa Cruz), <span class="yellow">rabbit</span> anti-Pax6 (1∶150; Covance), <span class="yellow">rabbit</span> anti-Trb2 (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Trb1: (1∶1000; Chemicon), <span class="yellow">goat</span> anti-Dcx: (1∶500; Santa Cruz), <span class="yellow">mouse</span> anti-PSA-NCAM: (1∶500; Chemicon), <span class="yellow">mouse</span> anti-NeuN (1∶300; Chemicon), <span class="yellow">rabbit</span> anti-Calretinin (1∶500; Swant); <span class="yellow">mouse</span> anti-Sox2 (1∶100; R&D systems), <span class="yellow">mouse</span> anti-Mash1 (1∶100; BD PharMingen), <span class="yellow">rabbit</span> anti-NeuroD2 (1∶300; ABCAM), <span class="yellow">mouse</span> anti-MAP2 (1∶500; Sigma), <span class="yellow">rabbit</span> anti-Ki67 (1∶150; NovaCastra) and <span class="yellow">rabbit</span> anti-phospho-Histone H3. The secondary antibodies (1∶200) Cy2, FITC, Cy3 and Cy5 were from Jackson IR laboratories, Alexa-fluor 488, 568, 595 and 647 from Invitrogen-Molecular Probes. DAPI (1∶1000) was purchased from Sigma.
For immunocytochemistry, cultures were fixed in 4% paraformaldehyde at day five, rinsed with PBS three times prior to pre-incubation with a blocking solution (10% <span class="yellow">donkey</span> serum, 0.25% TritonX100 in PBS) for 1 hour. The remainder of the procedure was performed as previously described [49]. Specimen analyses were performed using a Leica confocal microscope (Leica software, equipped with a GreNe and a HeNe laser, using the following lines of excitation: 488 nm, 594 nm and 647 nm). Samples were analyzed using 20×, 40× and 63× objectives, sometimes zoomed. Figures were composed in CANVAS-X software.<br><br>Bromo-deoxyuridine (BrdU) and Cloro-deoxyuridine (CldU) pulse labeling and immunohisto- and immunocyto-chemistry
To assess cell proliferation and ongoing neurogenesis in vivo, animals were injected with BrdU (100 mg/kg, Sigma), two hours prior to sacrifice (for both P2 and two weeks old Ngn2 <span class="yellow">mice</span>). To assess neurogenesis, BrdU (100 mg/kg) was injected 48 hours prior to sacrifice (for P2 animals, BrdU was injected in pregnant dams half day prior to give birth). Immunohistochemistry was carried on as described above using a <span class="yellow">rat</span> anti-CldU/BrdU primary antibody (1∶200, monoclonal, Immunologicalsdirect, Oxfordshire, UK), with an additional denaturation in 1 M HCl for 30 minutes prior to pre-incubation with serum.
To assess the neuronal-inducing activity of Ngn2 and NeuroD1, transduced cultures were incubated with CldU (20 µM, Sigma) for 2 days, after a period of differentiation of five days. For immunocytochemistry, cultures were fixed with 4% paraformaldehyde at day 7, rinsed with PBS three times, treated with 1 M HCl at 65°C for 5–10 min, pre-incubated and then incubated with a <span class="yellow">rat</span> anti-CldU/BrdU antibody and other primary antibodies. The remainder of the procedure was performed according to the protocol for immunohistochemistry already mentioned.<br><br>In utero surgery
Timed pregnant female <span class="yellow">Sprague Dawley rats</span> with embryos at gestational age E15.5 were anesthetized with halothane. The mother was placed in the lower level of a two-level wooden stage. The abdomen was shaved with an electric razor and then cleaned with 70% alcohol. A 2–3 cm midline laparotomy was performed. Each uterine horn was carefully taken out individually and the number of embryos recorded. One horn was then placed back inside the mother, whilst the other horn was prepared for injection. The embryos were kept moist with constant application of warm saline to prevent dehydration. Approximately 2 µl of viral suspension (1×10∧9TU/ml) was injected into the lateral ventricle of each embryo, except for the embryo closest to the vagina. After the injections, the uterine horns were placed back into the abdomen. The abdominal wall and the overlying skin were then sutured. Care was taken not to damage abdominal muscles so that normal delivery of the pups was possible at term. The entire surgery generally took about 45 minutes to one hour. Each mother was allowed to recover in her cage before being returned to the animal stable. Shredded paper was added to each cage to encourage nesting and special care was taken not to stress the mothers. Following normal delivery, the pups were allowed to develop to adulthood up to two weeks.<br><br>Quantification
For the in vivo experiment, manual cell counting was performed on 18 and 30 µm thick brain sections for two days and two weeks old animals, respectively. The brain of P2 animals was cut into 10 series of coronal sections; the brain of two-weeks old animals was cut into 7–8 series of coronal sections. For each staining, one-two series from three to five different individuals were analyzed per genotype using a confocal microscope with 1, 2 or 3 lines of excitation, in sequential scanning in order to avoid false positives. When two series were analyzed, only one series was counted. The mean number of cells per hippocampus expressing the markers of interest, was calculated for each brain, based on the analysis of 6–8 consecutive dorsal hippocampal sections. The final mean number of cells per section was calculated by adding the mean number from each individual. We expressed the data as the mean number of positive cells±standard error of the mean (SEM). In this study, more than 25 Ngn2 null, >45 Ngn2 heterozygotes, and >60 WT individuals (including <span class="yellow">rats</span> injected with retroviruses) were analyzed, in total. Statistical comparison was performed using one-factor analysis of variance (ANOVA) with transcription factor, cellular marker, genotype or time as variables, followed by post-hoc analysis when significant differences were observed, using Statview 5.0 software (SAS institute Inc.). For cell culture experiments, three independent experiments were performed, each in duplicate. The counting was based on seven randomly chosen different fields of view. For each diagram, the level of significance (p-value) is represented as follows: P<0.05 = *; P<0.001 = **; P<0.0001 = ***. All data are expressed as±standard error of the mean (SEM).<br><br><span class="yellow">Human</span> sample and RT-PCR
The brain from a 64 year-old male was provided by the Harvard Brain Tissue Resource Center. The individual whose brain tissue was being analyzed gave written consent for storage and use of his tissue for research. Five millimeters thick fresh <span class="blue">human</span> hippocampal tissue sample was snap frozen. The frozen tissue block was cut into 20 series of coronal sections, using a cryostat. Every 10 sections, one section was placed in an Eppendorf tube, on dry ice. Pooled sections were used for RT-PCR. Total RNA was prepared using Trizol and RNAeasy, supplemented with RNAGuard as previously described [49]. The RNA was digested with <span class="yellow">shrimp</span> DNAse before cDNA synthesis, which was performed using a mix of oligo-dT and random hexamer primers and SuperscriptII reverse transcriptase. Advantage2 polymerase mix (Clontech/BRL) was used for PCR, with the following cycling conditions: 10 cycles of 94° C for 30 seconds, 68°C for 2 minutes, 30 cycles of 94°C for 30 seconds, 60°C for 1 minute, 68°C for 1 minute and 30 seconds, one cycle of 68° C for 2 minutes, soak at 16°C. Primers marked * were designed by PrimerBank. hRPS18 (sense) 5′-GCCTTTGCCATCACTGCCATT and hRPS18 (antisense) 5′-GCCAGTGGTCTTGGTGTGCT, hPAX6 (sense) 5′-GCCCTGGAGAAAGAGTTTGAGAGAACCCATT hPAX6 (antisense) 5′-GGGGAAATGAGTCCTGTTGAAGTGGTGC, hTBR1 (sense) 5′-GCGGACACCAATGTGCAAGGAAATCG and hTBR1 (antisense) 5′-CGAGGGGGTCAGGCGGTCCATGTCACAGC, hTBR2 (sense) 5′-GACCTGTGGCAAAGCCGACAATAACATGC and hTBR2 (antisense) 5′-GGGGGTGTCTCTATCCAAGAAGAGCCAAT, hPDHX ( = PROX1) (sense) 5′-GGGACACTACGGTTCCGTTTAAGTCCAGC and hPDHX (antisense) 5′-CTCTCCATCCCAGCTTACATTAACATCTGGCATTTGT, hNGN2 (sense) * 5′-CATCAAGAAGACCCGTAGACTGA and hNGN2 (antisense)* 5′-CAACACTGCCTCGGAGAAG, hMASH1 (sense) 5′-CCTGGATCCGCATGGAAAGCTCTGCCAAGATGGAG and hMASH1 (antisense) 5′-CCTGGATCCCCCCTCAGAACCAGTTGGTGAAGTCGA, hNEUROD1 (sense) 5′-GCTCAGGACCTACTAACAACAAAGGAAATCGAAACATG and hNEUROD1 (antisense) 5′-CAAAGCGTCTGAACGAAGGAGACCAGGT, hGLAST (sense) 5′-CATCAGGGAAGATGGGAATGCGAGCTGTAGTCTATTAT and hGLAST (antisense) 5′-CCACGGGGGCATACCACATTATTACTGCTACC, hNEST (sense) 5′-CAGGAGCGGCTGCGGGCTACTGAAAAGTTCC and hNEST (antisense) 5′-CAGGGCTGAGGGGTGGTGCCAAGGAGG, hGFAP (sense) 5′-CCACGAGGAGGAGGTTCGGGAACTCCAGGAGC and hGFAP (antisense) 5-GGAATGGTGATCCGGTTCTCCTCGCCCTCTAGC, hCALB2 ( = Calretinin) (sense) 5′-GGCTCTGGCATGATGTCAAAGAGTGACAACTTT and hCALB2 (antisense) 5′-GGGCATCCAGCTCATGCTCGTCAATGTAGCC, hCALB1 (sense) 5′-GCGAAAGAAGGCTGGATTGGAGTTATCACC and hCALB1 (antisense) 5′-CCCTCCATCCGACAAAGCCATTATGTTCTTCTTGTATG, hSOX2 (sense) 5′-GGAGAACCCCAAGATGCACAACTCGGAGAT and hSOX2 (antisense) 5′-GAGGAAGAGGTAACCACAGGGGGGCTGGAGC, hWNT3A (sense) 5′-CCGAGGGCATCAAGATTGGCATCCAGGAGTGC and hWNT3A (antisense) 5′-TCGGGTTGCGACCACCAGCATGTCTTCACCTC, hSHH* (sense) 5′-ACTCCGAGCGATTTAAGGAACT and hSHH* (antisense) 5′-CAGACGTGGTGATGTCCACTG.<br><br>
Supporting Information<br><br>
<h3>pmcA320464</h3>Organization and structure of the <span class="yellow">mouse</span> interleukin-2 gene.
Abstract
We have cloned a chromosomal DNA segment which covers the entire sequence for the <span class="yellow">murine</span> interleukin-2 gene and analysed the structure of the gene. The coding regions are separated into four blocks by three introns each of which is located similarly to the corresponding <span class="blue">human</span> gene. The exon sequences can be aligned perfectly with the previously cloned cDNA sequence. Of particular interests is the presence of sequences within the 5'-flanking region which are highly conserved between <span class="yellow">mouse</span> and <span class="yellow">man</span>. The conserved region which spans more than 400 base pairs may play a role in the regulation of IL-2 gene expression.Images<br><br><br><br><br><br>
 Volume 12 Number 24 1984                                   Nucleic Acids Research <br><br> Organization and structure of the <span class="yellow">mouse</span> interleukin-2 gene <br><br> Akira Fuse*, Takashi Fujita, Hidetaro Yasumitsu, Nobukazu Kashima+, Katsushige Hasegawa and Tadatsugu Taniguchi? <br><br> Department of Biochemistry, Cancer Institute, Japanese Foundation for Cancer Research, Toshimaku, Tokyo 170 and Institute for Molecular and Cellular Biology, Osaka University, Suita-shi, Osaka 565, Japan <br><br> Received 10 October 1984; Revised and Accepted 20 November 1984 <br><br> ABSTRACT <br><br> We have cloned a chromosomal DNA segment which covers the entire sequence for the <span class="yellow">murine</span> interleukin-2 gene and analysed the structure of the gene. The coding regions are separated into four blocks by three introns each of which is located similarly to the corresponding <span class="blue">human</span> gene.    The exon sequences can be aligned perfectly with the previously cloned cDNA sequence.  Of particular interests is the presence of sequences within the 5'flanking region which are highly conserved between <span class="yellow">mouse</span> and <span class="yellow">man</span>. The conserved region which spans more than 400 base pairs may play a role in the regulation of IL-2 gene expression. <br><br> INTRODUCTION <br><br> Interleukin-2 (IL-2) is a lymphokine produced by T cells upon antigenic or mitogenic stimulation and is required for the proliferation of T cells (1,2).      Several other biological activities of IL-2 which appear to be crucial in the immune regulation have also been reported (3, 4. 5. 6. 7.) .        We previously reported isolation and sequence analysis of the cDNA for <span class="blue">human</span> IL-2 (8), as well as the chromosomal gene (9). More recently, we have isolated a cDNA which encodes <span class="yellow">murine</span> IL-2 (Kashima et al., submitted for publication).  The cDNA contains a unique tandem repeat of CAG sequence which would encode 12 consecutive glutamine residues in the active IL-2 molecule. <br><br> In order to study the structure of the <span class="yellow">murine</span> IL-2 chromosomal gene and its controlling region, we isolated and analysed a A phage clone containing the gene and its flanking sequences. <br><br> MATERIALS AND METHODS <br><br> Southern blotting of total <span class="yellow">mouse</span> DNA <br><br> <span class="yellow">Mouse</span> chromosomal DNA was extracted from liver of BALB/c6 <br><br> ? I R L Press Limited, Oxford, England. <br><br> Nucleic Acids Research <br><br> Volume 12 Number 24 1984 <br><br> 9323 <br><br> Nucleic Acids Research <br><br> <span class="yellow">mouse</span> as described before (10). High molecular genomic DNA was digested with various restriction enzymes and electrophoresed on 0.8% agarose gel. Blotting analysis of DNA was carried out by the method of Southern (11). Hybridization was carried out as described previously and filters were washed either in 3 x SSC at 650C (lower stringent condition) or in 0.1 x SSC at 650C (higher stringent condition). <br><br> Screening of genomic DNA library <br><br> A <span class="yellow">bacteriophage</span> XCharon 4A/<span class="yellow">mouse</span> genomic DNA library prepared with partial EcoRI digests of <span class="yellow">mouse</span> DNA from MPC 11 plasmacytoma cells was kindly provided by Dr. T. Honjo. <span class="yellow">Mouse</span> IL-2-specific clones were screened by the method of Benton and Davis (12), using 700 bp PstI-AccI fragment of a cDNA clone, pMIL2-45   as  the probe   (Kashima   et  al.,   submitted   for publication).    Hybridization was performed as described previously(13). Positive clones were rescreened at least twice. Subcloninq and sequencing of the <span class="yellow">mouse</span> IL-2 gene <br><br> Two EcoRI fragments of 3.3 Kbp and 2.8 Kbp from the positive recombinant X phage were subcloned into EcoRI site of plasmid pBR322.  DNA segments derived from subcloned 3.3 Kbp and 2.8 Kbp fragments were labelled at either 3' end or 5' end, and subjected to sequence analysis by the chemical degradation method (14).  The 0.8 Kbp EcoRI fragment from the same X phage clone was directly subjected to sequence analysis by the dideoxy chain termination method (15). <br><br> RESULTS <br><br> Total DNA blotting analysis <br><br> In order to study structural organization of the <span class="yellow">mouse</span> IL-2 gene, we first subjected total <span class="yellow">mouse</span> DNA to the blotting analysis by using various probes specific for IL-2 gene. When <span class="yellow">mouse</span> DNA was digested with various restriction endonucleases and then probed with a 7 kb <span class="blue">human</span> chromosomal DNA segment which contains the <span class="blue">human</span> IL-2 gene and its flanking region (Fig. 1 lane 1-8, ref. 9.), single positive band appeared at lower but not at higher stringent condition for washing the filters (see Figure legend).    Additional bands corresponding to those observed by using <span class="yellow">mouse</span> IL-2 cDNA probes (lane 13-17) also appeared by longer <br><br> 9324 <br><br> Nucleic Acids Research <br><br> M 1 2 3 4     M 5 6 7 8      M 9 101112    M 13 141516     17 <br><br> a~ ~ 3 ..  <br><br> I pMIL2-45 SacI           Acc I <br><br> CZI~~1      1t L2- 20 <br><br> Acc I <br><br> 1 00 tp lX i <br><br> Fig. 1.  Blot hybridization analysis of <span class="yellow">mouse</span> chromosomal DNA. High molecular DNA prepared from Liver BALB/C6 <span class="yellow">mouse</span> was digested with various restriction endonucleases (BamHI for lanes 1, 5, 9, 13 ; EcoRI for lanes 2, 6, 10, 14, 17 ; HindIII for lanes 3, 7, 11, 15 ; XbaI for lanes 4, 8, 12, 16 ). The resulting digests were fractionated on 0.8 % agarose gel and transferred to a nitrocellulose filter.  Filters were hybridized by the published procedure (13) either with the nick-translated chromosomal DNA containing <span class="blue">human</span> IL-2 gene and its flanking region (total length, 7.0 kb, ref. 9) (lane 1-8) or with the nick-translated cDNA for <span class="yellow">mouse</span> IL-2 (see figure). Filters were then washed either in 3 x SSC at 65 % (lower stringent condition) (lane 1-4, 9-12) or 0.1 x SSC at 65 OC (higher stringent condition) (lane 5-8, 13-17). Lane M each contains 7 size markers with their size being 23.7 kb,  9.5 kb,   6.7 kb,  4.3 kb,   2.3 kb,  2.0 kb and 0.6 kb, respectively.  Brief restriction endonuclease cleavage map for the <span class="yellow">mouse</span> IL-2 cDNAs is presented in the lower part of the figure. <br><br> exposure of the film (data not shown).    Those results suggest the presence of highly conserved sequences between <span class="blue">human</span> and <span class="yellow">mouse</span> DNA either in the flanking regions or in the introns of the IL-2 gene, since the coding regions apparently show lower degree of sequence homology as evidenced in this series of blotting analysis (see below).  When the PstI insert of a <span class="yellow">mouse</span> IL-2 cDNA clone, pMIL2-20, was used as the probe, a simple pattern was <br><br> 9325 <br><br> Nucleic Acids Research <br><br> obtained at higher stringent condition (lane 13-17).   While the EcoRI-digested DNA gave rise to two positive bands (2.8 kb and 0.8 kb)(Fig. 1, lane 14) by this analysis, one additional band of 3.3 kb also appeared when the same DNA was probed with a longer cDNA insert from another clone, pMIL2-45 (Fig. 1, lane 17).  The 3.3 kb band was similar in its size with the positive band which became detectable by probing the same DNA with the 7.0 kb <span class="blue">human</span> DNA probe (Fig. 1, lane 2). Since this band appeared with the cDNA probe extending further upstream, it is likely that the 5' region of the gene is located within this DNA segment (see below). Indeed, this 3.3 kb band did not appear even after longer exposure of lane 14 (result not shown). BamHI digest of the <span class="yellow">mouse</span> DNA (Fig. 1, lane 1, 13) constantly gave a very faint signal which would correspond to a DNA larger than 15 kb.         Taken together, the results suggested the presence of a single copy gene for <span class="yellow">murine</span> IL-2.    On the other hand appearance of the multiple positive bands at lower stringent washing condition (lane 9-12) indicates the presence of IL-2 related sequences within the <span class="yellow">mouse</span> genome. <br><br> Screening of recombinant phaqe libraries <br><br> We next screened a gene library from partial EcoRIdigested DNA from MPC 11 cells and by using 0.8 Kbp SacI-AccI cDNA fragment as the probe and isolated 14 positive clones containing sequences specific to the <span class="yellow">mouse</span> IL-2 gene. Three of the clones analysed all contained three EcoRI fragments whose size is in agreement with the result of blotting analysis of the chromosomal DNA as shown in Fig. 1 (lane 17). One of these is designated MIL-2G70. <br><br> Nucleotide sequence analysis <br><br> Two DNA fragments of 3.3 Kbp and 2.8 Kbp were excised from the phage clone MIL-2G70 by EcoRI digestion and they were subcloned into pBR322.     DNA sequences were determined for selected regions of both inserts and compared to the known cDNA sequences (Kashima et al., submitted for publication).       The strategy used for sequence analysis of the genomic DNA is presented in Fig. 2.    Comparison of the <span class="yellow">mouse</span> genomic IL-2 sequence with <span class="yellow">mouse</span> IL-2 cDNA sequence revealed that, like the <span class="blue">human</span> gene, the gene is divided into four exons.     A putative <br><br> 9326 <br><br> Nucleic Acids Research <br><br> E               E           E    E   E <br><br> 500bp s    7S P HBH  H   \A <br><br> 200 bp  *     ,            -   4<br><br> I             4-4----*    *-    --4 <br><br> Fig. 2. Restriction map and sequencing strategy of <span class="yellow">mouse</span> IL-2 gene. Horizontal lines indicate the length of <span class="yellow">mouse</span> DNA inserted into the X phage Charon 4A or plasmid subclones.   Filled blocks, dashed blocks and open blocks indicate protein coding regions, untranslated regions and introns, respectively. Horizontal arrows indicate the direction and extent of sequence determination without ambiguity.   Dashed arrows:   determination was done by the chain termination method (15) after subcloning the       0.8 kb fragment into M13.       Rest of the sequence determination was      carried out by the method of Maxam and Gilbert (14).   A AccI site, B; BamHI site, E; EcoRI site, H; HindIII site, P; PstI site, S; SacI site. <br><br> capping site or the transcription initiation site was located 32 bp downstream from a TATAAA consensus promotor sequence (Fig. 3). The first ATG triplet was located 79 bp downstream from the TATA box. As seen also in the <span class="yellow">murine</span> IL-2 cDNA, there is an unusual repeat of CAG triplet coding for 12 glutamine residues in a row in the first exon. The second exon (60 bp) is separated from the first exon by a short intron consisting of 97 bp.     The second, the third and the fourth exons are interrupted by longer introns whose size is about 2.3 Kbp and 1.6 Kbp, respectively.  As far as the available sequence data are concerned, it seems that, despite their identical location, intron sequences are distinctly dissimilar except for the junction regions between the <span class="blue">human</span> and <span class="yellow">mouse</span> IL-2 genes.    There are two potential poly (A) addition signals within the <span class="yellow">mouse</span> gene (nucleotide positions 793 - 798 and 924 - 929 in Fig. 3 ) and, based on our sequence data for various cDNA clones, both signals seem to function and give rise to heterogeneous termini of the mRNA in the LBRM-33 cells (16). <br><br> We have also determined the sequence of about 500 bp of 5'-flanking region of <span class="yellow">mouse</span> IL-2 gene, since (i) promoter/regulatory sequences are located in this region in many other genes of eukaryotes and (ii) this region appeared to contain sequences which show strongest cross-hybridization between <span class="blue">human</span> and <span class="yellow">mouse</span> DNA around the IL-2 gene (Fig. 1). Comparison of the nucleotide sequences for the 5' flanking region <br><br> 9327 <br><br> Nucleic Acids Research <br><br> -4.0 <br><br> TAGGAGGTAAACCATCTCGAAACGGAAACCAATATCCTTCCTGTCTAATCAACAAATCTAAAG -400                                               -350 <br><br> GATTTATTCTTTTCATCTATCTCCTCTTGCGCCCGTCCACCACAACAGGCTGCTTACAGGTTCAGGATGGTTTTGACAAAGAGAACATTTT <br><br> -300                                               -250 <br><br> CATGAGTTACTTTTGTGTCTCCACCCCAAAGAGGAAAATTTTGTTCATACGAAAGGCGTTCATTGTATGAATTAAAACTGCCACCTAAGAG <br><br> -200                                               -150 <br><br> TGGGCTAACCCGACCAAGAGGGATTTCACCTAAATCCATTCAGTCAGTATATGGGGTTTAAACAAATTCCAGAGAGTCATCAGAAGAGGAA <br><br> -100                                                -50 <br><br> AAACAAAGGTAATACTTTCTGCCACACAGGTAGACTCTTTTGAAAATATGTGTAATATGTAAAACATCGTGACACCCCCATATTATTTTTC CAGCATTAACAGTATAAATTGCCTCCCATGCTGAAGAGCTGCC TCACCCTTGCTAATCACTCCTCACAGTGACCTCAAGTCCTGCAGGC <br><br> 50                                                                   100 <br><br> ATG TAC AGC ATG CAG CTC GCA TCC TGT GTC ACA TTG ACA CTT GTG CTC CTT GTC AAC AGC GCA CCC ACT Met Tyr Ser Met Gln Leu Ala Ser Cys Val Thr Leu Thr Leu Val Leu Leu Val Asn Ser Ala Pro Thr <br><br> 150 <br><br> TCA AGC TCC ACT TCA AGC TCT ACA GCG GAA GCA CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG Ser Ser Ser Thr Ser Ser Ser Thr Ala Glu Ala Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln <br><br> 200 <br><br> CAC CTG GAG CAG CTG TTG ATG GAC CTA CAG GAG CTC CTG AGC AGG ATG GAG    GTAAGTGCACAGCCATCCCATC His Leu Glu Gln Leu Leu Met Asp Leu Gln Glu Leu Leu Ser Arg Met Glu <br><br> TATAGGCAATACCTTTAGCTTTCTTGCCAAAGGCTGTGTTTAATAACCTTTAATAATAATGTTACGCTTTCTCAG AAT TAC AGG AAC <br><br> lAsn Tyr Arg Asn 250 <br><br> CTG AAA CTC CCC AGG ATG CTC ACC TTC AAA TTT TAC TTG CCC AAG CAG GTGAGTGAGTTTCTGTTTAACTGGTGC Leu Lys Leu Pro Arg Met Leu Thr Phe Lys Phe Tyr Leu Pro Lys Gln <br><br> TCTAATG----------------------------------------------------------__ <br><br> ?_______________----------------------- 2000 bp --------------------?---------------------------------------------------------- AATGTGAACCTTGTAGTTTCTTTGTAGATTGGAACAATAGTCTGAACTTGTGT TATGCATTGGTAGAGAAACACAGACTTTTTACAAAGTCTAATGAAGCCAGAAGAGCTCACTAGAGTGGCCTAACAGTTAAGGTGACCCTTA TGGTTGTGAGCTCTTGCTCTTCTAGATTTATGGCATCGATTACCTCAGTCCCCCTTTACAGAGGACAGGGAGTGGTAAAAGCTATGTGCTG CCTTCTCTTGATTAGAGAGACTGCAGACTAACTTTCTGGCTCTTCAGTATGTGGTGAGCTGAGCTGATGGTTAAGCTTATTACTCCTCTAG <br><br> 300                                                                 350 <br><br> GCC ACA GAA TTG AAA GAT CTT CAG TGC CTA GAA GAT GAA CTT GGA CCT CTG CGG CAT GTT CTG GAT TTG Ala Thr Glu Leu Lys Asp Leu Gln Cys Leu Glu Asp Glu Leu Gly Pro Leu Arg His Val Leu Asp Leu <br><br> 490 <br><br> ACT CAA AGC AAA AGC TTT CAA TTG GAA GAT GCT GAG AAT TTC ATC AGC AAT ATC AGA GTA ACT GTT GTA Thr Gln Ser Lys Ser Phe Gln Leu Glu Asp Ala Glu Asn Phe Ile Ser Asn Ile Arg Val Thr Val Val AAA CTA AAG GTAAGGTGTTGCTTTATTTGCTAATCTGGAAATAAAATAGAGAAGAAATGCATTTTTAAGTGGCTTGCCATTTCTGGT Lys Leu Lys <br><br> CTTTGATGGGTTCTGTGCATTTAGTCAACCAAAGTTTAAAGTCACTGTGCAAGTGAATCC --------------------------------------------------------------------    -1000   bp  ------ --------------------------------------------------------------- GAATTCTACAGAAGTGTTCAGTGTTCCCATCAAATGCTCGTTGGTAATCAACTCTGGAGAGCTT TATTTTTTATGCTTTACCATTTGATATCATAATAAAACATAACCCAATAATTGTGATCATTTCAGAAATGTGAATGTTCAATATATTTAAC <br><br> CAGTGTTAAAATAAATGCCTAAAAGCTCCGTTGAAGAATAATTATATGCAAAGTAAGCTACCTTAGCCTACAATTTTATATTCTTTTTTAG <br><br> 450                                                                 500 <br><br> GGC TCT GAC AAC ACA TTT GAG TGC CAA TTC GAT GAT GAG TCA GCA ACT GTG GTG GAC TTT CTG AGG AGA Gly Ser Asp Asn Thr Phe Glu Cys Gln Phe Asp Asp GlU Ser Ala Thr Val Val Asp Phe Leu Arg Arg <br><br> 550 <br><br> TGG ATA GCC TTC TGT CAA AGC ATC ATC TCA ACA AGC CCT CAA TAACTATGTACCTCCTGCTTACAACACATAAGGC Trp Ile Ala Phe Cys Gln Ser Ile Ile Ser Thr Ser Pro Gln <br><br> 600                                                650 <br><br> TCTCTATTTAT4TAAATATTTAACTTTAATTTATTTTTGGATGTATTGTTTACTATCTTTTGTAACTACTAGTCTTCAGATGATAAATATG <br><br> 700                                                750 <br><br> GATCTTTAAAGATTCTTTTTGTAAGCCCCAAGGGCTCAAAAATGTTTTAAACTATTTATCTGAAATTATT+ATTATATTGAATTGTTAAAT <br><br> < 4>800                                                 850 <br><br> ATCATGTGTAGGTAGACTcATTCA TAAAAGTAmAGATGATTCAAATATAAATAAGCTCAGATGTCTGTCATTTTTAGGACAGCACAAAG <br><br> 900                         0" <br><br> TAAGCGCTAAAATAACTTCTCAGTTATTCCTGTGAACTcTATGTTAATCAGTGTTTTCAAGAAATAAAGCTCTCCTCT           AAA GATGGCTTGTGGGAAAAGATCTCCTCTCCAGGGAGCTAACATCAGCTCAGAGTTTACTCAAGAATTC <br><br> Fig.      3.      Nucleotide sequence of <span class="yellow">mouse</span> IL-2 gene.                               Four exons are framed.          Numbers refer to nucleotide positions of exons from the presumed cap site. Dots and open circles indicate TATA box and poly A additional signals, respectively. <br><br> 9328 <br><br> Nucleic Acids Research <br><br> -4 5 0                               -400 <br><br> TAGGAGGT AAACCA T CTCGAAAC GGAAACCAATATCCTTCCTGTCTAATCAACAAATCTAAAGGATTTATTCTTTTCATCTATCTCC <br><br> .0     * .0 .- *. * ..* --.  -- .*-@ .. ............  --  -*-* * <br><br> TAAAAAGGTAAAACCAGTTCT GAAACAGGAAACCAATACACTTCCTGTTTAATCAACAAATCTAAA CATTTATTCTTTTCATCTGTTTAC <br><br> -350                         -300 <br><br> TCTTGCGCCCGTCCACCACAACAGGCTGCT T ACAGGTTCAGGATGGTTTTGACAAAGAGAACATTTTCATGAGTTACTTTTGTGTCTC TCTTGCTCTTGTTCACCACAATA  TGCTATTCACATGTTCAGTGTAGTTTTATGACAAAGAAAATTTTC TGAGTTACTTTTGTATCCC <br><br> -250 <br><br> CA CCCC  AAAG  AGGAAAATTTGTTTCATACGAAAGGCGTTCATTGTATGAATTAAAACT CCACCTAAGAGTGGGCTAACCCGA <br><br> *~~~ ~~   ..... *   .-- *..-  * .-- .**  . *-.   --.   *- *- -  -  *-   0 @  *-- 0 .. .   <br><br> CACCCCCTTAAAGAAAGGAGGAAAAACTGTTTCATACAGAAGGCGTTAATTGCATGAATTAGAGCTATCACCTAAGTGTGGGCTAATGTAA <br><br> -200                          -150 <br><br> CCAAGAGGGATTTCACCTAAATCCATTCAGTCAGTATAT GGGGTTTAAACAAATTCCAGAGAGTCATCAGAAGAGGAAAAACAAAGGTAA CAAAGAGGGATTTCACCTACATCCATTCAGTCAGTCTTTGGGGGTTTAAA AAATTCCAAAGAGTCATCAGAAGAGGAAAAATGAAGGTAA <br><br> -100                          -50 <br><br> TACTTTCTGCCACACAGGTAGACTCTTTTGAAAATATGTGTXATATTAAAACATCGTGACACCCCCATATiXrTCCAGCATTAACAG TGTTTTTT CAGACAGGTAAAGTC TTTGAAAATATGTGTAATATGTAAAACATTTTGACACCCCCATAATATTTTTCCAGAATTAACAG ATAAGCCTCCCATGCTGAAGAGCTGCCTATCACCCTTGCTAATCACTCCTCACAGTGACCTCAAGTCCTGCAGGCATG  <span class="yellow">Mouse</span> rTATGCATCTCTTGTTCAAGAGTTCCCTATCACTCTCTTTAATCACTACACACAGTAACCTCAACTCCTGCCACTATG  <span class="yellow">Human</span> <br><br> Fig.  4.   Comparison of 5'-flanking regions of <span class="yellow">mouse</span> and <span class="blue">human</span> IL-2 genes.   In aligning the sequences for both genes, gaps were introduced   to maximize   homology.    Dots   indicate  identical nucleotide sequences.   Number 1 indicates putative transcription initiation site. Inverted repeats are indicated by bars and dashed lines. TATA box is framed. <br><br> of both genes is illustrated in Fig. 4.    The nucleotide sequence homology from the TATA box to -470 is 85%. The highest region of homology was observed between the TATA box and position -97 (60 bp matches out of 64 bp) of both genes. DISCUSSION <br><br> We have isolated recombinant clones for <span class="yellow">mouse</span> IL-2 gene from a phage Charon 4A/<span class="yellow">mouse</span> genomic DNA library and determined the entire sequence of the gene except for the sequence of the internal portion of the second and third introns.         The <span class="yellow">mouse</span> IL-2 cDNA sequence was aligned with the genomic sequence and both sequences matched completely each other. The unusual CAG repeats encoding 12 glutamines which was found previously in the cloned cDNA was shown to be present also in the chromosomal gene. This finding further excludes the possibility that the unique repeat is generated by artifacts during the cDNA cloning process. Although we can not rule out the possibility for the deletion of this sequence in the <span class="blue">human</span> IL-2 gene, it is more likely that the CAG repeat has been generated in the <span class="yellow">mouse</span> genome rather recently.    The repeat could have been generated either by a direct insertion of the sequence or by the duplication after insertion of a unit sequence. <br><br> 9329 <br><br> Nucleic Acids Research <br><br> Organization of the <span class="yellow">mouse</span> IL-2 gene resembles to that of the <span class="blue">human</span> gene (Fig. 2., ref. 9).   There seems to be little sequence homology between corresponding introns of <span class="yellow">mouse</span> and <span class="blue">human</span> IL-2 genes, except for the intron-exon junctions part of which is thought to be necessary for the RNA splicing (17, 18). Dissimilarity of the intron sequences among the genes which are derived from a common ancestor has been reported in other genes (19, 20).   In spite of the divergence in sequence of introns, the size and position of the introns are very similar between the <span class="yellow">murine</span> and <span class="blue">human</span> IL-2 genes. <br><br> Of particular interests is the presence of highly conserved sequences in the 5' -flanking region of the <span class="blue">human</span> and <span class="yellow">mouse</span> IL-2 gene (Fig. 4).    Whereas the coding region shows nucleotide sequence homology of 72% between the two genes, the 5' upstream region spanning about 500 bp (Fig. 4) shows 85% homology which was readily detectable by the blotting analysis (Fig. 1, lane 1-4).  Since we have not yet determined the nucleotide sequence further upstream of the <span class="yellow">mouse</span> gene, we do not know whether or not this similarity extends further.  It is  likely that such sequences are involved in the controlled expression of the IL-2 genes in activated T-lymphocytes.  Work is in progress to identify such DNA sequences by introducing the cloned genes into various lymphocytic cell lines.   Our preliminary results indicate that the 5 '-flanking sequence of the <span class="blue">human</span> IL-2 gene mediates mitogen induced expression of the gene in T-lymphocytic cells (Fujita & Taniguchi, unpublished observation). <br><br> ACKNOWLEDGEMENTS <br><br> We thank Dr. T. Honjo for <span class="yellow">mouse</span> gene library.    We are also indepted to Ms. M. Nagatsuka for typing the manuscript. This work was supported in part by Grant-in-Aid for Special Project Research, Cancer-Bioscience from the Ministry of Education, Science and Culture, Japan. <br><br> ?To whom correspondence should be addressed <br><br> *Present address: Department of Microbiology, School of Medicine, Chiba University, Chiba 280, Japan <br><br> +Present address: Central Research Laboratory, Ajinomoto Co.Inc., Totsuka-ku, Yokohama 244, Japan <br><br> 9330 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> 1. Morgan, D. A., Ruscetti, F. W. and Gallo, R. (1976) Science, <br><br> 193, 1007-1008. <br><br> 2. Gil lis, S. Ferm, M. M., Ou, W. and Smith, K. (1 978) J. <br><br> Immunol., 120, 2023-2027. <br><br> 3. Chen, B. M. and Di Sabato, G. (1976) Cell. Immunol., 22, 211<br><br> 224. <br><br> 4. Henney, C. S., Kuribayashi, K., Kern, D. E. and Gillis, S. <br><br> (1981) Nature, 291, 335-338. <br><br> 5. Wagner, H., Hardt, C.,   Heeg, K., Rollinghoff, M. and <br><br> Pfizenmaier, K. (1980) Nature, 284, 278-280. <br><br> 6. Farrar, J. J., Benjamin, W. R., Hilfikr M. L., Howard, M., <br><br> Farrar,W. L. and Fuller-Farrar, J. (1982) Immunol. Rev. 63, 129-166. <br><br> 7. Pearlstein, K., Palladino, M. A., Welte, K. and Vilcek, J. <br><br> (1983) Cell. Immunol., 80, 1-9. <br><br> 8. Taniguchi, T., Matsui, H., Fujita, T., Takaoka, C., Kashima, <br><br> N., Yoshimoto, R. and Hamuro, J. (1983) Nature 302, 305-310. <br><br> 9. Fujita,T.,Takaoka,C., Matsui, H. and Taniguchi T. (1983) <br><br> Proc. Natl. Acad. Sci. USA 80, 7437-7411. <br><br> 10. Ohno, S. and Taniguchi, T. (1982) Nucleic Acids Res. 10, 967<br><br> 977. <br><br> 11. Southern, E. M. (1975) J. Mol. Biol. 98, 503-517. <br><br> 12. Benton, W. D. and Davis, R. W. (1977) Science 196, 180-182. <br><br> 13. Ohno, S. and Taniguchi, T. (1981) Proc. Natl. Acad. Sci. USA <br><br> 78, 5305-5309. <br><br> 14. Maxam, A. M. and Gilbert, W. (1980)  Meth. Enzymol. 65, 560<br><br> 580. <br><br> 15. Sanger, F., Nicklen, S. and Coulson, A. R. (1977) Proc. Natl. <br><br> Acad. Sci. USA, 74, 5463-5467. <br><br> 16. Gillis, S., Scheid, M. and Watson, J. D.(1980) J. Immunol. <br><br> ,125, 2570-2580. <br><br> 17. Breathnach, R., Benoist, C., O'Hare, K., Gannon, F. and <br><br> Chambon, P.(1978) Proc. Natl. Acad. Sci. USA, 75, 4853-4857. <br><br> 18. Lerner, M. R., Boyle, J. A., Mount, S. M., Wolin, S. L. and <br><br> Steitz, J. A. (1980) Nature, 283, 220-224. <br><br> 19. Van Ooyen, A., Van den Berg, J., Mantei, N. and Weissmann, <br><br> C.(1979) Science, 206, 337-344. <br><br> 20. Searle, P. F., Davison, B. L., Stuart, G. W., Wilkie, T. M., <br><br> Norstedt, G. and Palmiter, R. D. (1984) Mol. Cell. Biol. 4, 1221-1230. <br><br> 9331 <h3>pmcA327937</h3>Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a.
Abstract
Attempts to prevent the urea conversion of a 200-230,000 molecular weight DNA polymerase alpha to a 150-170,000 molecular weight form by the inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70,000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70,000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit.<br><br><br><br>
 Volume 6 Number 10 1979 <br><br> Nucleic Acids Research <br><br> Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a Keith McKune and Andrew M.Holmes* <br><br> Biochemistry Department, Strathclyde University, The Todd Centre, 31 Taylor Street, Glasgow, G4 ONR, UK <br><br> Received I June 1979 ABSTRACT <br><br> Attempts to prevent the urea conversion of a 200-230, 000 molecular weight DNA polymerase a to a 150-170, 000 molecular weight form by the <br><br> inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70, 000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70, 000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit. <br><br> INTRODUCTION <br><br> In mammals DNA polymerase a is thought to be the replicative <br><br> enzyme, but due to low levels of activity, even in tissues actively making DNA, and to enzyme heterogeneity it has been found difficult to purify <br><br> However, small samples of DNA polymerase a have been highly purified <br><br> 2, 3, 4, 5 from several sources and partially characterised <br><br> Heterogeneity has been observed in DNA polymerase a from a var<br><br> 6           7 <br><br> iety of sources, including <span class="yellow">rat</span> liver and spleen , Hela cells, baby <span class="yellow">hamster</span> <br><br> 8                9, 10                   11 <br><br> kidney cells , <span class="yellow">mouse</span> myeloma    , <span class="yellow">Drosophila</span> embryos  and <span class="yellow">calf</span> thymus <br><br> 1,6, 12, 13 . We have previously observed several species of <span class="yellow">calf</span> thymus <br><br> DNA polymerase a differing in size and charge  . In order of elution from DEAE cellulose they are enzyme A1 (200-230, 000 molecular weight), A2 (200-230, 000), B (100-110, 000) and C (150-170, 000). A poly (dA). oligo <br><br> (dT)10 preferring enzyme, enzyrrme D (140-150, 000 molecular weight) elutes just after enzyme B. The A enzymes seem identical in all properties except their charge. 5.0-5. 5S enzymes, analagous to B, have been observed to <br><br> C Information Retrieval Limited 1 Falconberg Court London Wl V 5FG England <br><br> 3341 <br><br> Nucleic Acids Research <br><br> arise as the result of proteolytic action  ' , similarly the B enzyme above 14 <br><br> ,but the relationship of the other observed species is not clear, nor is <br><br> the problem of which, if any, of these species is the replicative enzyme, <br><br> although circumstantial evidence has implicated a DNA polymerase a to be <br><br> 15 the replicative enzyme in <span class="yellow">adenovirus</span> infected KB cells <br><br> We have previously shown that several of these enzymes are inter<br><br> 16 <br><br> convertible . In particular, mild urea treatment can convert both A <br><br> enzymes to C enzyme, with the loss of a subunit or fragment of 50-70, 000 molecular weight. The C enzyme retains DNA polymerase activity, but <br><br> does show differences compared to A enzyme in heat sensitivity and sensitivity to N-ethylmaleimide. <br><br> Highly purified samples of A and C enzymes when subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis have shown a corre<br><br> lation of DNA polymerase activity with a polypeptide band 155, 000 molecular weight . Contaminating material of 50-70, 000 molecular weight appeared <br><br> to obscure the putative 50-70, 000 subunit in gels of A enzyme. The conclusions drawn were that DNA polymerase a is a 155, 000 molecular weight <br><br> polypeptide (C enzyrrme) which can and does associated with material of 50<br><br> 70, 000 molecular weight to give A enzyme, and that enzyme B is a proteol<br><br> 14 <br><br> ytic degradation product . The relationship of enzyme D to the others is <br><br> not clear. <br><br> It is possible that during the enzyme isolation procedure proteolytic action may have taken place on a 200-230, 000 molecular weight enzyme releasing 155, 000 and 50-70, 000 fragments which remain in association <br><br> until urea treatment separates them. It is also possible that urea treatment itself renders the enzyme susceptible to contaminating proteases. Further experiments to ascertain whether or not this is the case and, if so, to prevent it, have been carried out. <br><br> A enzyme can be reconstituted from C enzyme formed by the action <br><br> of urea on A enzyme by concentrating it with the 50-70, 000 molecular weight material. The A enzyrme obtained in this manner is highly purified, as is <br><br> the C enzyme formed by the urea treatment, and has been used in attempts to show differences in synthetic template-initiator complex utilisation <br><br> 3342 <br><br> Nucleic Acids Research <br><br> b y the se two enzyme s . <br><br> MATERIALS AND METHODS <br><br> <span class="yellow">Calf</span> thymus was obtained from 10-16 week old <span class="yellow">calves</span> and frozen at <br><br> -20?C until required. Chromatographic media and chemicals were obtained <br><br> 6 <br><br> from sources previously referred to . Radioactive deoxynucleoside tri<br><br> phosphates were obtained from the Radiochemical Centre, Amersham, Bucks. Synthetic oligo and polynucleotides were obtained from P. L. Biochemicals <br><br> Inc., except for poly (dC) which was a gift from Dr. I.R. Johnston and pre<br><br> 17 <br><br> pared from oligo d(C)5 as described  . N-a-p Tosyl-L-lysine chloromethy<br><br> lketone HCl and phenylmethylsulphonylfluoride were obtained from Sigma and Trasylol from Bayer. <br><br> Except where indicated all buffers contained 20% w/v glycerol and <br><br> 1 mM dithiothreitol. Standard linear phosphate gradients were run between 0.03 M and 0.25 M potassium phosphate, pH 7.8. Gradient salt concentr<br><br> 2 <br><br> ations were measured using a conductivity meter as described . Urea was <br><br> prepared as a 4. 8 M solution in 20% w/v glycerol, stirred with Amberlite <br><br> MB3 and filtered. Dithiothreitol was added to a final concentration of 1 mM before use. <br><br> 6 DNA polymerase was assayed using activated DNA as described <br><br> except that the buffer was 50 mM tris HC1, pH 7.8. One unit of DNA polymerase activity incorporates ln mol [ H] dTMP into an acid insoluble form in one hour at 370C. Assays using synthetic template-initiator complexes were carried out at 30?C in 0. 12 ml. The template-initiator complexes <br><br> 6 <br><br> were prepared and assays processed as described . All assays contained <br><br> 1 mM dithiothreitol, 62. 5 p.g <span class="yellow">bovine</span> serum albumin, 1 ,ug template-initiator complex, enzyme protein and the relevant [ H] deoxynucleoside triphosphate at 0.1 mM and 12-15 cpm/pmol. These assays were carried out at either pH 6.4 in 20 mM sodium-potassium phosphate, or at pH 7.8 in 50 mM tris HCl and contained either 10 mM MgCl2 or 1 mM MnCl2 as indicated. <br><br> Preliminary purification of DNA polymerase a to Fraction IV was as <br><br> 6 <br><br> described , the purification steps being phosphocellulose chromatography, <br><br> ammonium sulphate precipitation and gel filtration on Sepharose 6B. <br><br> 3343 <br><br> Nucleic Acids Research <br><br> Enzyme obtained from the DEAE cellulose step is referred to as Fraction <br><br> V enzyme. Samples were prepared for sodium dodecyl sulphate polyacrylamide gel electrophoresis and scanned as previously described2 <br><br> RESULTS AND DISCUSSION <br><br> (a)   Interconversion Studies <br><br> A enzyme was routinely converted to C enzyme by incubating 200500 units/ml of Fraction V A enzyme in 2.4 M urea in 0. 02 M potassium <br><br> phosphate, pH 7.8, for 60 minutes at 0?C. The mixture was then loaded on to a DEAE cellulose, washed with 0. 03 M potassium phosphate pH 7. 8, and enzyrme eluted either with the standard phosphate gradient or batchwise. Under these conditions usually about 50-60% of recovered activity was <br><br> enzyme C (Fig 1 a). Overall recovery was 70-80%. A enzyme was recon<br><br> 14 <br><br> stituted essentially as described . The flow through material from the <br><br> DEAE cellulose column after the urea treatment was loaded on to a 1 x 0. 8 cm phosphocellulose column, washed with 0.03 M potassium phosphate, pH 7.8, and the protein eluted with 0.25 M potassium phosphate, pH 7. 8. This material, the putative subunit, was vacuum dialysed with the C enzyme produced by the urea treatment, rechromatographed on DEAE cellulose and enzyme eluted batchwise (Fig 1 b). Recovery from this procedure was <br><br> usually 50-70% A enzyme. Overall recovery was 30-40% of the C enzyme dialysed. If the material eluted from the phosphocellulose by the 0. 25 M potassium phosphate was heated to 90?C for 5 minutes prior to vacuum <br><br> dialysis with the C enzyme the recovery from the DEAE cellulose column <br><br> was significantly higher (60-70% of the original C enzyme activity), but all recoverable DNA polymerase activity was C enzyme. This, together with the fact that the 60 minute treatment with urea has, on occasions, given <br><br> rise to a 50% increase in DNA polymerase activity prior to loading on to the DEAE cellulose column, would indicate that A enzyme is less active on <br><br> activated DNA than is C enzyme. If the DEAE cellulose flow through material came from urea treatment of A2 enzyme then A2 was produced on reconstitution; if from A1 then A1 was produced. <br><br> Although the mild urea treatment of A has been used to prepare C of <br><br> 3344 <br><br> Nucleic Acids Research <br><br> b <br><br> 120       180 C <br><br> I         I    ^ <br><br> Figure la  DEAE cellulose chromatography after 2.4 M urea treatment. <br><br> 4, 5000 units, 4.4 mg of Fraction II A2 enzyme were incubated with urea at a final concentration of 2.4 M for 60 minutes at 0?C, loaded on to a 5 x 1.4 cm DEAE cellulose column, the column was washed with 0. 03 M potassium phosphate, pH 7.8 and a 200 ml standard phosphate gradient applied. 2.3 ml fractions were collected and 10 Rl assayed for 5 minutes. (o-o) no <br><br> phenylmethylsulphonylfluoride (*-*) 3 mM phenylmethylsulphonylfluoride ( - ) phosphate gradient. <br><br> Figure lb  The reconstitution of A2 enzyme. 3000 units for C enzyme <br><br> derived by urea treatment of A2 were vacuum dialysed with the DEAE cellulose flow through material after phosphocellulose chromatography and <br><br> chromatographed on a 2 x 1. 2 cm DEAE cellulose column. After washing with 0.03 M potassium phosphate, pH 7.7, the enzymes were eluted batchwise with the above concentrations of potassium phosphate, pH 7.8. 1 ml fractions were collected and 10 .l assayed for 10 minutes. <br><br> 3345 <br><br> Nucleic Acids Research <br><br> high specific activity it is possible that conversion of the 200-230, 000 molecular weight enzymes to a 155- 170, 000 species maybe due to unfolding of the molecule to allow limited attack by contaminating proteases. Accord<br><br> ingly the urea conversion of A to C was investigated in the presence of certain protease inhibitors. The presence of the serine protease inhibitor <br><br> phenylmethylsulphonyl-fluoride in the incubation and chromatography buffers did not affect the conversion of A2to C (Fig 1 a). Likewise trasylol and Na-p Tosyl-L-lysine chloromethylketone HC1 had no effect. However, protease action could have occurred earlier in the purification procedure and the urea could be separating two fragments. Usually the DEAE cellulose profile shows that the majority of the enzyme activity is present as A <br><br> enzyme (Fig 2). In this instance A1 and A2 have not been separated. When the temperature of the material in the original blending procedure was kept below 0?C or phenylmethylsulphonylfluoride, N-a-p Tosyl-L-lysine chloromethylketone HCl or trasylol was included in the isolation buffers the DEAE cellulose profile was similar. The A enzymes were still capable of conversion by mild urea treatment to C enzyme, indicating that if protease activity is involved then it is not susceptible to these inhibitors. When the <span class="yellow">calf</span> <br><br> 20FI: Now ~ ~      ~      20 20Fr Na      06 <br><br> Figure 2  DEAE cellulose elution profile of <span class="yellow">calf</span> thymus DNA polymerase a. 47, 000 units, 97 mg of Fraction IV enzyme prepared from 415 g of <span class="yellow">calf</span> <br><br> thymus were loaded on to a 10 x 1.8 cm DEAE cellulose column, after washing with 0. 03 M potassium phosphate, pH 7. 8 a 400 ml standard phosphate gradient was applied. 5 ml fractions were collected and 10 ,ul assayed for 5 minutes. <br><br> 3346 <br><br> Nucleic Acids Research <br><br> thymus was allowed to warm up during the blending procedure, or the supernatant prior to phosphocellulose chromatography was heated to 37 ?C for 30 minutes, there was a marked decrease in the amount of A enzyme with a concomitant increase in the amount of B and C (unpublished observation). The presence of phenylmethylsulphonylfluoride, N- a-p T osyl- L-lysine <br><br> chloromethylketone HCl or trasylol under these conditions only had a marginal effect on the appearance of C enzyme, but did reduce the amount of B <br><br> enzyme. Heating the enzyme to 37?C after the phosphocellulose step had no effect on the DEAE cellulose elution profile. Attempts to convert A enzyme to C using trypsin have not been successful. A enzyme activity is lost without the appearance of any other species (unpublished observation), although <br><br> 14 the action of trypsin on C enzyme can give rise to small amounts of B <br><br> Although the conversion of A enzyme to C does not appear to be the result of serine protease action, proteases other than serine proteases <br><br> 18 <br><br> could have been responsible  . Also, the fact that A enzyme can be recon<br><br> stituted from C plus the flow through material from the DEAE cellulose after urea treatment does not necessarily mean we are dealing with two subunits as fragments produced by proteases mray be reassembled to give active <br><br> enzyme 9    . However, the fact that a 200-230, 000 molecular weight polypeptide band has never been observed in sodium dodecylsulphate polyacrylamride gels of highly purified A enzyme , or even cruder fractions of A <br><br> enzyrre (unpublished observation), may be significant. One might expect <br><br> some of the enzyme not to have been attacked by whatever is responsible for cleaving the molecule, if, indeed, this does happen. It would appear, therefore, that the A enzyme consists of subunits of 155, 000 and 50-70, 000 molecular weight with the small subunit having a slightly different charge in the <br><br> case of A  and A2. It has also been concluded that the heterogeneity in the <br><br> 1                                     ~~~~~~~~~~~~~~~~~3 <span class="yellow">mouse</span> myeloma DNA polymerase a fraction is not due to proteolysis . However, the 50-70, 000 molecular weight subunit has not yet been identified. <br><br> Sodiurrm dodecylsulphate polyacrylamide gels of reconstituted A enzyme have shown polypeptide bands at 150-160, 000 and 50-70, 000 molecular weight <br><br> (Fig. 3), but the ratio of staining intensity of the bands does not correspond to a 1: 1 relationship. The ratio of the two bands is variable but is usually between 1:2 and 1:3, indicating, perhaps, that more than one subunit of 50<br><br> 3347 <br><br> Nucleic Acids Research <br><br> TOP      123   4 5   6 <br><br> Figure 3  Scan of a 5% sodium dodecyl sulphate polyacrylamide gel of <br><br> reconstituted A2 enzyme. 1500 units of reconstituted A2 enzyme were subjected to polyacrylamide gel electrophoresis under non-denaturing conditions. The gels were sliced, enzyme extracted and assayed and the peak <br><br> fraction of DNA polymerase activity from three gels were pooled, subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis on a single <br><br> gel, stained and scanned at 2 volt sensitivity as described2. The molecular weight standards were: (1) <span class="yellow">bovine</span> serum albumin dimer (134, 000), (2) ,3 galactosidase (130, 000), (3) phosphorylase a (94, 000), (4) <span class="yellow">bovine</span> serum <br><br> albumin (68, 000), (5) pyruvate kinase (57, 000) and (6) lactate dehydrogenase (35, 000). <br><br> 70, 000 molecular weight can associate with the 155, 000 subunit. The proportion of the lower molecular weight polypeptide band is higher in the reconstituted A enzyrrme than in the C enzyme preparation from which it was formed, but even the C enzyme contained some material in the region. <br><br> Breakdown of material from 155, 000 to 50-70, 000 may be partly responsible for the contamination, but the presence of phenylmethylsulphonylfluoride in samples in preparation for sodium dodecylsulphate polyacrylamide gel electrophoresis has not been successful in preventing it. The <span class="blue">human</span> KB cell <br><br> DNA polymerase a, equivalent to the C enzyme, has been reported to consist <br><br> 4 <br><br> of subunits of 76, 000 and 66, 000 molecular weight , but at no time have we <br><br> observed polypeptide bands at 76, 000 and 66, 000 rising and falling with <br><br> enzyme activity in any gels of <span class="yellow">calf</span> thyrrmus enzymes A1, A  and C (and <br><br> 2 unpublished observations). (b)    Template Studies <br><br> Use has been made of the urea conversion of A to C and of the reconstitution of A enzyme to obtain samples of highly purified DNA polymerase, <br><br> specific activity in excess of 50, 000 units/mg, in order to study the effect of this 50-70, 000 molecular weight subunit on the DNA polymerase subunit. Previous results have indicated that A enzyrrme is stabler to heat and less <br><br> 3348 <br><br> Nucleic Acids Research <br><br> 14, 21 <br><br> susceptible to N-ethylmaleimide than C enzyme  . It has been reported <br><br> that highly purified DNA polymerase a can be associated with a protein capable of binding to DNA containing no 3' OH ends and capable of being released during the DNA polymerase assay  . Attempts to dissociate the 50<br><br> 70, 000 molecular weight subunit from the polymerase subunit by incubating A2 with DNA polymerase reaction mixes containing activated DNA, poly <br><br> (dA-T) and poly (dT). oligo (A)10 followed by ultracentrifugation in high salt have been unsuccessful. Similarly A absorbed and eluted from either native or denatured DNA cellulose remained A enzyme. Both A and C enzymes <br><br> were eluted from the DNA celluloses by less than 0. 1 M NaCl so it does not appear that the subunit enhances the binding of the DNA polymerase subunit to DNA. However, there are differences in the response of A and C <br><br> enzymes to synthetic template-initiator complexes (Table 1). Even if one takes into account the fact that A enzyme is less active on activated DNA <br><br> than C enzyme (the addition of the 50-70, 000 subunit to the polymerase subunit appears to result in a decrease of about 30% of polymerase activity on activated DNA) the A enzyme is still more active on these templates. <br><br> Although there is a variation in activity on these template-initiator comp<br><br> lexes each time assays are carried out on them depending on the method of preparing the complexes and the base ratio of template to initiator, A enzyme always seems to be significantly more active than C. The A2 <br><br> enzyme at pH 7.8, with extra subunit(s) is clearly more effective on the <br><br> oligoribonucleotide initiator, oligo(A)10 than is enzyme C. In view of the proposed RNA initiation of Okazaki pieces this may indicate a role of this subunit in Okazaki piece synthesis, in that it may aid the DNA polymerase to'take over' from the RNA polymerase. Using poly (dA). oligo (dT) 10 <br><br> (A:T=20: 1) and following the incorporation of [ H] dTMP as a function of time a short lag was observed for C enzyme, but not for A2 (Fig. 4). <br><br> Similar re sult s we re obtained when poly (dC). oligo (dG) 1 0 (C: G= 5: 1) wa s <br><br> used as temrrplate-initiator, but not when poly (dT). oligo (A)10 (T:A=1: 1) was used. Neither enzyme showed a lag on this template-initiator complex or on activated DNA. Incorporation versus enzyme concentration also showed this lag for C enzyme on poly (dA). oligo (dT)10. It is not certain what causes <br><br> 3349 <br><br> Nucleic Acids Research <br><br> TABLE 1 Template utilisation by reconstituted A and C derived from A. <br><br> 3H] dNTP Divalent cation  A2 at pH    C at pH <br><br> Template                          _      6.4   7.8    6.4   7.8 Activated DNA         dTTP         Mg++            100          100 Activated DNA         dATP         Mg++             91          85 Activated DNA         dGTP         Mg++             85          82 poly(dA).oligo(dT)10  dTTP         Mg      66.0    4.0    20    91 <br><br> (A:T = 20: 1) <br><br> poly(dA). oligo(dT)   dTTP         Mu      19.0    4. 5  6. 5  2.5 <br><br> poly(dA. oig~10 (A:T = 20:1) <br><br> poly(dT). oligo(dA)10  dATP        Mg       2.5   <1 (1       (1 <br><br> (T:A = 5:1) <br><br> poly(dT) .oligo(dA)1  dATP         Mn ++   15.5   23.0   9. 0  3. 0 <br><br> poly(dT). oligo(A)10  dATP         Mg      15.0  230.0  10.0  77. 5 <br><br> (T:A = 1:1) <br><br> poly(dT) * oligo(A) 10  dATP       Mn      41.5   75. 5 19.5  21.5 <br><br> (T:A = 1:1) <br><br> poly(dC). oligo(dG)   dGTP         Mg++     6. 0   15.5  4. 5  7. 5 <br><br> (C:G = 5:1)   10 <br><br> poly(dC). oligo(dG)10  dGTP        Mn +     6. 0  29.5   5. 0  6. 5 <br><br> (C:G  = 5: 1)                                 .    . <br><br> Values given are relative to incorporation of [ H] dTMP at pH 7. 8 on activated DNA. For A2 this was 176.5 pmol, for C 232 pmol. dATP, dCTP, dGTP and dTTP were included for assays using activated DNA, only the deoxynucleoside triphosphate stated was used in the synthetic template<br><br> initiator complex assays. Assays were for 10 minutes. The buffers and concentrations of the divalent cations were as in Materials and Methods. <br><br> this lag, but the annealing of template to initiator is only transient '  and the DNA polymerase subunit may have difficulty in stabilising the complex and the 50-70, 000 molecular weight subunit may be able to help the poly<br><br> merase subunit to overcome this. Addition of the 50-70, 000 subunit to the polymerase subunit prior to assaying with these templates had no effect on activity and it may be that a preincubation period is required before the two 3350 <br><br> Nucleic Acids Research <br><br> Figure 4 Activity of A2 and C enzymes on poly (dA). oligo (dT)jO (A:T = <br><br> 20:1) as a function of time. 50 ,ul samples were withdrawn at various times from a 0. 6 inl incubation mix at 30?C and added to 0. 5 ml 0. 1 M sodium <br><br> pyrophosphate containing 100 ,ug/ml native <span class="yellow">calf</span> thymus DNA and processed for counting in the usual manner6. The mix contained 1 mM dithiothreitol, 50 mM tris HC1, pH 7.8 10 mM MgC12, 0. 1 mM [3H] dTTP (15 cpm/pmol), 312.5 ,ug <span class="yellow">bovine</span> serum albumin, 5 Fg poly (dA). oligo (dT)  (A:T = 20:1) and enzyme protein (*-*) 3.3 units reconstituted A2, (o-o) 16. 0 units urea derived C enzyme. <br><br> subunits become fully associated. After the lag phase is over the C enzyme is still less active on these template-initiators than A enzyme . That is the A enzyme seems capable of elongating the initiator faster than the C enzyme. Experiments to determine whether the differences in rates of elongation of these template-initiator complexes are differences in processivity of the <br><br> enzymes under the different pH and divalent cation conditions are under way. <br><br> ACKNOWLEDGEMENTS <br><br> We thank the Medical Research Council for a research grant. <br><br> Communications concerning the paper should be sent to:<br><br> Dr. A. M. Holmes, Department of Biochemistry, University of Strathclyde, The Todd Centre, 31 Taylor Street, Glasgow G4 ONR, U.K. <br><br> 3351 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> (1)    Bollum, F.J. (1975). Prog. Nucleic Acid Res. Mol. Biol. 15, <br><br> 109-144. <br><br> (2)    Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1976). Eur. <br><br> J. Biochem. 62, 229-235. <br><br> (3)    Matsukage, A., Sivarajan, M. and Wilson, S.H. (1976). Biochem<br><br> istry 15, 5305-5314. <br><br> (4)    Fisher, P.A. and Korn, D. (1977). J. Biol. Chem. 252, 6528<br><br> 6535. <br><br> (5)    Fichot, O., Pascal, M., Mechali, M. and De Recondo, A, -M. <br><br> (1979). Biochim. Biophys. Acta, 561, 28-41. <br><br> (6)    Holmes, A.M., Hesslewood, I.R. and Johnston, I.R. (1974). Eur. <br><br> J. Biochemn. 43, 487-499. <br><br> (7)    Noy, G.P. and Weissbach, A. (1977). Biochim. Biophys. Acta, <br><br> 447, 70-83. <br><br> (8)    Craig, R.K. and Keir, H. M. (1975). Biochem. J. 145, 225-232. <br><br> (9)    Matsukage, A., Bohn, E.W. and Wilson, S.H. (1974). Proc. Natl. <br><br> Acad. Sci. U.S.A. 71, 578-582. <br><br> 110)   Hachmann, H.J. and Lezius, A.G. (1975). Eur. J. Biochem. 50, <br><br> 357- 366. <br><br> (11)   Brakel, C.L. and Blumenthal, A.B. (1977). Biochemistry, 16, <br><br> 3137-3143. <br><br> (12)   Momparler, R.L., Rossi, M. and Labitan, A. (1973). J. Biol. <br><br> Chem. 248, 285-293. <br><br> (13)   Yoshida, S., Konda, T. and Ando, T. (1974). Biochim. Biophys. <br><br> Acta, 353, 463-474. <br><br> (14)   Holmes, A.M., Hesslewood, I.P., Wickremasinghe, R.G. and <br><br> Johnston, I.R. (1977). Biochem. Soc. Symp. 42, 17-36. <br><br> (15)   De Jong, A., Van der Vliet, P. and Jansz, H.S. (1977). Biochim. <br><br> Biophys. Acta, 476, 156-165. <br><br> (16)   Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1975). <br><br> Nature (London) 255, 420-422. <br><br> (17)   Bollum, F. J. (1966). Procedures in Nucleic Acid Research <br><br> (Cantoni, G.L. and Davis, D.R. eds). pp 577-583, Harper and Row, New York. <br><br> (18)   Barrett, A.J. (1975). Proteases and Biological Control pp 467-482, <br><br> Cold Spring Harbor Laboratory, Cold Spring Harbor. <br><br> (19)   Richards, F. M. and Vithayathil, P.J. (1960). Brookhaven Symp. <br><br> Biol. 13, 115-134. <br><br> (20)   Lowe, P.A., and Malcolm, A.B.D. (1976). Eur. J. Biochem. 64, <br><br> 177- 188. <br><br> (21)   Hesslewood, I.P., Holmes, A.M., Wakeling, W.F. and Johnston, <br><br> I.R. (1978). Eur. J. Biochenr.. 84, 123-131. <br><br> (22)   Mechali, M. and De Recondo, A. -M. (1978). Biochim. Biophys. <br><br> Res. Comm, 82, 255-264. <br><br> (23)   Chang, L.M.S., Cassani, G.R. and Bollum, F.J. (1972). J. Biol. <br><br> Che. 247, 7718-7723. <br><br> (24)   Wickremasinghe, R.G. and Johnston, I.R. (1974). Biochim. <br><br> Biophys. Acta, 361, 37-52. <br><br> 3352 <h3>pmcA411032</h3>Down-regulation of the M6P/IGF-II receptor increases cell proliferation and reduces apoptosis in neonatal <span class="yellow">rat</span> cardiac myocytes
Abstract
Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a multi-functional protein that has been implicated in regulation of cell growth and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R, and cardiomyocyte apoptosis has been identified in a variety of cardiovascular disorders, such as myocardial infarction and heart failure. However, involvement of M6P/IGF2R in the pathogenesis of these conditions has not been determined. Thus, the objective of this study was to determine the role of M6P/IGF2R in regulation of cardiac myocyte growth and apoptosis.<br><br>Results
We down-regulated the expression of M6P/IGF2R in neonatal <span class="yellow">rat</span> cardiac myocytes and examined the effect on cell proliferation and apoptosis. Infection of neonatal cardiomyocytes with an <span class="yellow">adenovirus</span> expressing a ribozyme targeted against the M6P/IGF2R significantly reduced the level of M6P/IGF2R mRNA, as determined by RT-PCR and Ribonuclease Protection Assay (RPA). M6P-containing protein binding and endocytosis as well as the M6P/IGF2R-mediated internalization of 125I-IGF-II were lower in the ribozyme-treated cells than the control myocytes, indicating that the number of functional M6P/IGF2R in the ribozyme treated cells was reduced. Accordingly, a marked increase in cell proliferation and a reduced cell susceptibility to hypoxia- and TNF-induced apoptosis were observed in the ribozyme-treated cells.<br><br>Conclusions
These findings suggest that M6P/IGF2R may play a role in regulation of cardiac myocyte growth and apoptosis. Down regulation of this gene in cardiac tissues might be a new approach to prevention of cell death or promotion of mitogenesis for certain heart diseases.<br><br><br><br>Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a unique protein that interacts with multiple ligands, some of which are important growth regulatory factors [1]. The M6P/IGF2R participates in internalization and lysosomal degradation of IGF-II, a mitogen normally acting through the IGF-I receptor to stimulate cell proliferation [2]. The M6P/IGF2 receptor is required for the activation of TGF-β [3], a potent growth inhibitor for many cell types. This receptor is also involved in the binding, transport and activation of newly-synthesized lysosomal enzymes, such as cathepsins [4,5], which have been recently implicated in the induction of apoptosis [6]. On the basis of these functions, the M6P/IGF2R has been proposed to play a significant role in regulation of cell growth and apoptosis [7].
Apoptosis, or programmed cell death, is a tightly regulated process used to remove excess, hazardous or damaged somatic cells, and is crucial for the development, maintenance and survival of an organism. However, alterations in the control of apoptosis have also been shown to contribute to <span class="blue">human</span> diseases. In fact, morphological and biochemical markers of apoptosis have been identified in a wide variety of cardiovascular disorders, including myocardial infarction and heart failure. This suggests that activation of apoptotic pathways contributes to cardiomyocyte loss and subsequent cardiac dysfunction in these conditions. A number of factors involved in cardiomyocyte apoptosis are currently known and include insulin-like growth factor-I (IGF-I), stress-activated protein kinases (SAPKs) and the anti-apoptotic Bcl-2 family [8]. There are indications that other factors may be involved in induction and regulation of cardiac apoptosis. However, these potential factors and their corresponding mechanisms have not been identified.
Several lines of evidence point to the potential involvement of M6P/IGF2R in cardiac myocyte proliferation and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R and transgenic <span class="yellow">mice</span> containing a homologous deletion of the M6P/IGF2R gene manifest ventricular hyperplasia due to an increase in cell number [9,10], suggesting that the M6P/IGF2R normally acts to suppress cardiac myocyte cell growth. It has also been shown that TGF-β, a potent growth suppressor whose activation requires the binding of latent TGF-β to M6P/IGF2R [3], is commonly upregulated in chronic heart failure [11]. Additional evidence for the involvement of M6P/IGF2R in regulation of apoptosis comes from studies of tumorigenesis. It has been shown that M6P/IGF2R expression is significantly reduced in a variety of tumors and loss of heterozygocity (LOH) at the M6P/IGF2R gene locus 6q26 have been found in breast, liver cancers and squamous cell carcinoma of the lung [12-15]. Although several studies have examined the effect of M6P/IGF2R over-expression on cell growth [7], it is not known whether down-regulation of this receptor protein leads to cellular protection against apoptosis.
Ribozymes are catalytic RNA molecules that cleave a complementary mRNA sequence [16], thereby inactivating specific mRNAs and suppressing gene expression in vitro and in vivo [17,18]. Ribozymes have been shown to be highly specific, efficient and stable. They can be packaged into viral vectors to enhance transfer into cells and to achieve longer expression compared with naked oligonucleotides. In the present study, we employed ribozyme technology to study the role of M6P/IGF2R in regulation of cardiac myocyte cell growth. A hammerhead ribozyme against the M6P/IGF2R mRNA was constructed and packaged in an adenoviral vector. We then examined the effect of ribozyme-mediated down-regulation of M6P/IGF2R expression on cell growth and hypoxia- and TNF-induced apoptosis.<br><br>Results
Cleavage reaction of the ribozyme in vitro
The M6P/IGF2R ribozyme we constructed has 13-bp binding arms complementary to the target site of M6P/IGF2R mRNA, and a catalytic core (Fig. 1A). To evaluate the bioactivity of the ribozyme and the accessibility of the target site, a cleavage reaction was performed in vitro. The substrates, [α-32P] labeled RNA transcripts containing 45 bp of M6P/IGF2R mRNA or an unmatched sequence, were incubated with the ribozyme as described (see Materials and Methods). The ribozyme cleaved only the specific M6P/IGF2R mRNA into the expected products. In the assay of time course, the hammerhead ribozyme was able to cleave 24.2% of the M6P/IGF2R target within 10 minutes of incubation, 50.3% of the M6P/IGF2R target within 40 minutes of incubation, and by 640 minutes, 80.8% of the M6P/IGF2R target was converted to the expected products (Fig. 1B). This ribozyme did not digest the unmatched sequence (Fig. 1B). These results indicate a high efficiency and specificity of the ribozyme in vitro.<br><br>Ribozymes down-regulate M6P/IGF2R expression in cardiac myocytes
To examine the ability of the ribozyme to reduce levels of M6P/IGF2R mRNA in cultured cardiac myocytes, total RNA was extracted from cells infected with Ad-GFP/IGF2R-Rz or Ad-GFP, and subjected to RT-PCR using M6P/IGF2R-specific primers. Primers specific for β-actin were added to a parallel reaction to serve as an internal standard. Cells were used 4 days after infection, with average infection efficiency of 70–80% (for which a viral dose used had minimal cytotoxicity). The RT-PCR product of M6P/IGF2R was 856 bp, and the β-actin product was 285 bp. As shown in Fig. 2A, the Ad-GFP/IGF2R-Rz-infected cells exhibited a significantly lower level of M6P/IGF2R mRNA than Ad-GFP-infected cells, with a reduction of about 50%. This result was confirmed by ribonuclease protection assay (RPA), in which GAPDH was used as a control (Fig. 2C &2D). There was no significant difference in the level of M6P/IGF2R mRNA between Ad-GFP-infected cells and uninfected cells (data not shown), indicating that infection with the <span class="yellow">adenovirus</span> itself did not alter the endogenous M6P/IGF2R mRNA level. The results demonstrated that the ribozyme was highly effective in suppressing M6P/IGF2R expression in cultured cardiac myocytes.<br><br>Effect of ribozyme expression on the functional activity of M6P/IGF2R
To determine the effect of the ribozyme on the functional activity of M6P/IGF2R, binding and internalization of exogenous 125I-IGF-II was measured in cells infected with Ad-GFP/IGF2R-Rz. As shown in Fig. 3A, cells infected with Ad-GFP/IGF2R-Rz showed a 54% reduction in 125I-IGF-II internalization when compared with the control cells (infected with Ad-GFP). We also examined the effect of the ribozyme on the M6P-binding activity of the M6P/IGF2R using the M6P-bearing lysosomal enzyme, β-glucuronidase, as a probe. The results showed that the maximal M6P-binding capacity of cells treated with the ribozyme was about 50% less than that of controls (Fig. 3B). Furthermore, we assessed the ability of cells to internalize exogenous β-glucuronidase after treatment with ribozyme. Similarly, the M6P-inhibitable endocytosis of β-glucuronidase by ribozyme-treated cells was about 52% less than that of control cells (Fig. 3C). These results confirm that the number of functional M6P/IGF2R in ribozyme-treated cells was reduced.<br><br>Adenoviral delivery of ribozymes increases the proliferation of cardiac myocytes
We examined the effects of the ribozyme on the growth of cultured neonatal <span class="yellow">rat</span> cardiac myocytes. Morphological evaluation showed a remarkable difference in growth pattern between Ad-GFP/IGF2R-Rz-infected cells and the control cells: the ribozyme-expressing cells formed larger and more spread colonies (Fig. 4). Assessment of cell proliferative activity by the MTT assay and counts of viable cells showed that the number of cardiac myocytes in ribozyme-expressing cultures was significantly higher than in control cultures (Fig. 5). These results indicate that treatment with M6P/IGF2R-ribozyme can promote cardiac myocyte proliferation.<br><br>Effect of M6P/IGF2R-ribozyme expression on apoptosis of cardiac myocytes
We examined the effects of ribozyme expression on TNF-α and hypoxia-induced apoptosis of cultured cardiac myocytes. After a 24 hr challenge with hypoxia, the number of apoptotic cells in M6P/IGF2R-Rz expressing cultures was 38% lower than in control cultures as determined by Hoechst staining (which highlights the nuclei of apoptotic cells) and ELISA (Fig. 6A, 7A). MTT analysis showed that the number of viable cells in ribozyme-treated cultures was 40% higher than in control cultures (Fig. 7A).
After treatment with TNF-α, as shown in Fig. 6B, a large number of control cells underwent apoptosis, as indicated by morphological changes (small round shape) and bright blue nuclear staining. There were significantly more apoptotic cells in control cultures than in cultures expressing the Ad-GFP/IGF2R-Rz. The number of apoptotic cells, as measured by the cell death ELISA assay, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 40%) lower than in cultures infected with Ad-GFP (Fig. 7B). Accordingly, the number of viable cells, as measured by MTT analysis, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 45%) higher than in cultures infected with Ad-GFP (Fig. 7B). These results are consistent with the hypothesis that decreasing M6P/IGF2R expression by ribozyme treatment can reduce cell apoptosis.<br><br>
Discussion
Some 62,000,000 Americans have one or more types of cardiovascular disease (CVD) and CVD is the leading cause (40.1%) of death in the United States. Myocardial infarction and heart failure, conditions accompanied by cardiac myocyte apoptosis, represent 23% of all CVDs and are a growing clinical challenge in need of novel therapeutic strategies. In this study, we investigated the M6P/IGF2R as a potential new therapeutic target for reduction of cardiac apoptosis and cardiac injury in these conditions.
Using ribozyme technology we down-regulated the expression of the M6P/IGF2R in neonatal cardiac myocytes. We then examined cell proliferation and apoptosis under normal conditions and post challenge with either hypoxia, a model of ischemia-reperfusion, or TNF-α, a cytokine implicated in the pathogenesis of chronic heart failure [19]. Our results demonstrate an association of a decrease in the expression and function of the M6P/IGF2R with increased cell proliferation and decreased cell susceptibility to hypoxia- and TNF-induced apoptosis. Expression of the ribozyme targeted against the M6P/IGF2R in cardiomyocytes resulted in down-regulation of M6P/IGF2R expression, as measured by RT-PCR and RPA, and of M6P/IGF2R function, as indicated by a decrease in internalization of 125I-IGF-II, and β-glucuronidase binding and endocytosis.
MTT analysis and viable cell counts showed that ribozyme-mediated down-regulation of M6P/IGF2R resulted in a marked increase in cell proliferation of cardiomyocytes, which normally express high levels of M6P/IGF2R [20] and have limited proliferative capabilities [21]. These results are consistent with the findings of previous knockout studies [9,10]. Since the M6P/IGF2R has multiple actions on cell growth, its proliferative effect on the heart cells observed in this study might involve multiple mechanisms. However, it is likely that unchecked IGF-II stimulation plays a key role in the effect. Because the M6P/IGF2R is believed to sequester and degrade IGF-II [2], a decrease in M6P/IGF2R expression and function could result in decreased degradation and hence increased bioavailability of IGF-II to the IGF-I receptor, which mediates the growth-promoting effect of IGF-II. Supporting evidence for the involvement of IGF-II in the proliferative effect resulting from loss of M6P/IGF2R function comes from studies of M6P/IGF2R knock-out <span class="yellow">mice</span>. M6P/IGF2R-null <span class="yellow">mice</span> display global hyperplasia that coincides with elevated levels of IGF-II. Most importantly, however, the lethal nature of an M6P/IGF2R-null phenotype is reversed in an IGF-II-null background [9]. Our results showing that ribozyme-mediated down-regulation of M6P/IGF2R lead to a decrease in IGF-II internalization support the above possibility. However, further investigation to confirm this mechanism is warranted.
More importantly, our results also showed that M6P/IGF2R down-regulation resulted in decreased sensitivity of cardiomyocytes to hypoxia- and TNF-induced apoptosis. There is evidence that lysosomal enzymes, such as cathepsins B and D contribute to hypoxia- and TNF-induced apoptosis in vitro [22-25] and in vivo [26,27]. The M6P/IGF2R has been shown to be involved in binding, transport and activation of lysosomal enzymes, including cathepsins [4,5]. Therefore, it is possible that down-regulation of the M6P/IGF2R results in improper trafficking and activation of cathepsins. This, in turn would eliminate the apoptotic cascades triggered by these enzymes under hypoxia and TNF stimulation and result in decreased sensitivity of cardiomyocytes to apoptosis.
It has also been shown that TNF stimulation involves the activation of TGF-β [28-30], a ligand of M6P/IGF2R that has been implicated in the progression of chronic heart failure [11,31]. Therefore, down-regulation of M6P/IGF2R expression could also lead to a decreased bioavailability of activated TGF-β, thereby decreasing the sensitivity of cardiomyocytes to the TNF/TGF-β apoptotic pathway. The detailed mechanism of the observed effects is unknown and requires further investigation.<br><br>Conclusions
The present study demonstrates that ribozyme-mediated down-regulation of expression and functional activity of the M6P/IGF2R results in a decrease in the susceptibility of cardiac myocytes to apoptotic stimuli. These findings suggest that this receptor might be involved in cardiac cell growth and apoptosis. The ability of the M6P/IGF2R ribozyme to reduce M6P/IGF2R expression and function in transfected cells verifies the utility of the ribozyme in studying the role of M6P/IGF2R in cardiomyocyte growth and apoptosis. In addition to its utility as a research tool, the ribozyme, with further exploration and development, might have potential application as a therapeutic agent to prevent cell death or promote mitogenesis for certain clinical conditions, such as, myocardial infarction and chronic heart failure.<br><br>Methods
Construction of recombinant M6P/IGF2R-RZ adenoviral vector
The nucleotide numbers of the <span class="yellow">rat</span> M6P/IGF2R sequence targeted by the hammerhead ribozyme is 1147–1160 after coding site (exon 9). The structure of the M6P/IGF2R hammerhead ribozyme is shown in Fig. 1. A 49 bp M6P/IGF2R ribozyme oligonucleotide, 5'-GAATTCCCC ACACTG ATGAGCCGCTTCGGCGGCGAAACATTCAAC GCGT-3' and the corresponding reverse complementary strand were synthesized. The fragments were subcloned to produce a plasmid containing a ribozyme against M6P/IGF2R. For construction of the recombinant <span class="yellow">adenovirus</span> containing the M6P/IGF2R-ribozyme (pAd-GFP/IGF2R-Rz), the segments containing the ribozymes were amplified by PCR and cloned into a pAdTrack-CMV vector and then recombined homologously with an adenoviral backbone pAdEasy 1 vector to generate (pAd-GFP/IGF2R-Rz), following the protocol described by He et al. [32]. The pAd-GFP/IGF2R-Rz carries both the IGF2R-Rz and GFP (as reporter) genes, each under the control of separate cytomegalovirus (CMV) promoters. Another viral vector, pAd-GFP, which carries the GFP gene only under the control of the CMV promoter, was generated and used as a control vector. The adenoviral vector DNA were linerized with Pac I and transfected into the replication-permissive 293 cells (E1A transcomplementing cell line) by using Lipofectamine (Life Technologies) to produce E1-deleted, replication-defective recombinant <span class="yellow">adenovirus</span> as described previously [33]. Large-scale amplification of recombinant <span class="yellow">adenovirus</span> in 293 cells was followed by purification using a discontinuous CsCl gradient. The constructs were confirmed by enzymatic digestion and DNA sequencing.<br><br>Transcription and cleavage reaction of ribozyme in vitro
Plasmids containing the ribozyme or the substrate (either 45 bp of M6P/IGF2R mRNA or an unmatched sequence 5'-GTGCTGTCTGTATG-3') were linearized with MluI, respectively. All transcripts were generated with T7 RNA polymerase (Promega). Substrate transcripts were labeled by incorporation of [α-32P] UTP (NEN Life Science Products, Inc.). Specific activity of the [α-32P] UTP (10 μCi/μl) and the base composition of each substrate molecule were used to calculate the substrate concentration. Ribozyme transcripts were quantified spectrophotometrically. (The half-life of the M6P/IGF2R target is about 280 minutes).
Cleavage reaction mixture contained substrate RNA (40 nM), increasing amounts of ribozyme (60 nM), 20 mM MgCl2 and 20 mM Tris-HCl, pH8.0, in a final volume of 10 μl. The mixture was incubated at 37°C for a time-course of cleavage reaction from 0, 5, 10, 20, 40, 80, 160, 320, to 640 minutes and the cleavage reaction was stopped by addition of loading buffer (80% formamide, 10 mM Na2EDTA, pH 8.0, and 1 mg/ml each bromophenol blue and xylene cyanol). Cleavage products were analyzed on a 15% polyacrylamide and 8M urea gel. Product and substrate fragments were quantitated by using NIH Imager.<br><br>Cell cultures and infection with Ad-GFP/Rz-IGF2R and Ad-GFP
Cardiac myocytes were isolated from 1-day-old newborn <span class="yellow">rats</span> using the Neonatal Cardiomyocyte Isolation System (Worthington). The isolated cells were plated in 6-well plates and cultured in F-10 medium containing 5% (vol/vol) FBS and 10% (vol/vol) <span class="yellow">horse</span> serum at 37°C in a tissue culture incubator with 5% CO2 and 98% relative humidity. Cells were used for experiments after 2–3 days of culture. Viral infections were carried out by adding viral particles at various concentrations (usually, 2 × 108 virus particles/ml) to culture medium containing 2% (vol/vol) FBS. Initially, optimal viral concentration was determined by using Ad-GFP to achieve an optimal balance of high gene expression and low viral titer to minimize cytotoxicity. After 24 hours of incubation, the infection medium was replaced with normal (15% vol/vol serum) culture medium. For treatment with IGF-II, cells were incubated with 50 ng/ml IGF-II after 24 hours infection with Ad-GFP/IGF2R-Rz or Ad-GFP. Four days after infection, cells were used for analysis of gene expression of M6P/IGF2R and its effect on cell growth and apoptosis.<br><br>Analysis of gene expression in cardiac myocytes
The M6P/IGF2R transcripts were determined by both RT-PCR and Ribonuclease Protection Assay (RPA). RT-PCR was performed using the GeneAmp EZ rTth RNA PCR kit (Roche). Total RNA was extracted from cultured cells using an RNA isolation kit (Qiagen,), according to the manufacturer's protocol. M6P/IGF2R transcripts were amplified using the primers (5'-GACAGGCTCGTTCTGACTTA-3') and (5'-CTTCCACTCTTATCCACAGC-3') specific to the M6P/IGF2R. Each RT-PCR assay was performed in triplicate and product levels varied by less than 3.2% for each RNA sample. Primers specific for β-actin cDNA were added to a parallel reaction to standardize for variations in PCR between samples. PCR products were resolved on a 1.0% agarose gel, visualized under UV light and quantitated using NIH Imager.
RPA was performed using the RPA III kit (Ambion, Austin, TX). Briefly, total RNA was extracted from cultured cells using a total RNA isolation reagent (TRIzol, Gibco BRL) according to the manufacturer's protocol. The plasmid containing the <span class="yellow">rat</span> M6P/IGF2R gene was linearized and used as a transcription template. Antisense RNA probes were transcribed in vitro using [33P]-UTP, T7 polymerase (Riboprobea System T7 kit, Promega), hybridized with the total RNA extracted from the <span class="yellow">rat</span> cardiomyocytes, and digested with ribonuclease to remove non-hybridized RNA and probe. The protected RNA·RNA was resolved on a denaturing 5% sequence gel and subjected to autoradiography. A probe targeting the GAPDH gene was used as an internal control.<br><br>Measurement of 125I-IGF-II internalization
Cells were incubated at 37°C for 2 hrs in serum-free F-10 culture medium containing 125I-labeled IGF-II (0.5 ng/ml) with or without excess unlabeled IGF-II (2 μg/ml). Following the incubation, the cells were washed three times with ice-cold PBS, and cell-associated radioactivity was determined by a γ counter. Specific internalized 125I-IGF-II was calculated by subtracting the count of samples with excessive unlabeled IGF-II from that without unlabeled IGF-II, and normalized to protein contents.<br><br>Beta-glucuronidase binding assay
Binding of β-glucuronidase was assayed as described previously [34,35]. Briefly, cells were permeabilized with 0.25% saponin in 50 mM Hepes (pH 7.0), 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="blue">human</span> serum albumin, and 10 mM mannose-6-phosphate (M6P) for 30 minutes on ice. The cells were washed three times with ice-cold PBS containing 0.05% saponin. They were incubated with 20,000 units/ml β-glucuronidase from <span class="yellow">bovine</span> liver (Sigma) in 50 mM Hepes (pH 7.5) containing 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="blue">human</span> serum albumin, 0.5% saponin with or without 10 mM M6P overnight on ice. Cells were washed five times with ice-cold PBS containing 0.05% saponin and sonicated in 100 mM sodium acetate (pH 4.6). The protein concentration of solubilized cell extract was measured and enzyme activity was assayed as follows: for each reaction 50 ul cell extract were added to 500 ul of 100 mM sodium acetate (pH 4.0) containing 1 mM paranitrophenyl (PNP)-β-glucuronide (Sigma) as substrate. After an incubation period of 3 hours at 37°C, 500 ul 1 M Na2CO3 were added to each reaction and the absorbance was measured at 400 nm. Experimental values were compared to a standard curve that was constructed using 1–100 nM solutions of PNP (Sigma) in 500 ul 100 mM sodium acetate and 500 u1 1 M Na2CO3. Specific activity was calculated as nM of PNP produced/hour/mg of protein.<br><br>Beta-glucuronidase endocytosis assay
Beta-glucuronidase endocytosis assay was carried out as described previously [36]. Briefly, confluent cell cultures were washed twice with pre-warmed serum-free DMEM followed by incubation with DMEM containing 5 mg/ml <span class="blue">human</span> serum albumin and 10 mM M6P for 20 minutes. Following incubation cells were washed 3 times with pre-warmed DMEM. Cells were then incubated in DMEM containing 5 mg/ml <span class="blue">human</span> serum albumin alone or 4000 units β-glucuronidase with or without 10 mM M6P for 2 hours at 37°C. Following the incubation, the cells were washed 5 times with ice-cold PBS and subjected to enzyme activity assay as described above.<br><br>Cell proliferation assay (MTT assay and cell counts)
Cardiac myocytes were grown in culture plates (tissue culture grade, 12 wells, flat bottom) in a final volume of 1 ml serum-containing culture medium per well, in a humidified atmosphere (37°C and 5% C02) for 3 days. After infection with Ad-GFP/IGF2R-Rz or Ad-GFP, cells were incubated with or without 50 ng/ml IGF-II for 4 days. Following supplementation with IGF-II, 100 μl MTT labeling reagent (Roche) were added to each well and cells were incubated for 4 hours, followed by addition of 1 ml solubilization solution into each well. The plate was placed in an incubator at 37°C overnight. Spectrophotometrical absorbency of the samples was measured using an UV-visible Recording Spectrophotometer with wavelength of 550–690 nm. In addition, the total number of viable cells in each treatment was counted by trypan blue exclusion method using a hemocytometer.<br><br>Induction and analysis of cell apoptosis
Cells were infected with Ad-GFP or Ad-GFP/IGF2R-Rz. Seventy-two hours post infection, cells were treated with TNF (0.1 ng/ml) for 24 hrs or subjected to hypoxia. For induction of apoptosis by hypoxia, cell culture medium was changed to serum-free F-10 saturated with 95% N2/5% CO2 and cells were placed in a 37°C airtight box saturated with 95% N2/5% CO2 for 24 hrs. For normoxic controls, culture medium was changed to F-10/5%F BS/10% HS and cells were placed in a 37°C/5% CO2 incubator for 24 hrs before analysis.
Apoptotic cells were identified by Hoechst staining using the Vybrant™ Apoptosis Kit #5 (Molecular Probes) according to the manufacturer's protocol. In addition, after infection with Ad-GFP or Ad-GFP/IGF2R-Rz and challenge with either TNF or hypoxia, cell viability was assessed using the MTT assay Kit (Roche Molecular Biochemicals) and cell apoptosis was determined using the Cell Death Detection ELISA Kit assay (Roche Molecular Biochemicals) according to the manufacturer's protocol.<br><br>Statistical analysis
Students' t-test was used to evaluate the difference between two values. Each experiment was repeated at least three times. Statistical significance was accepted at the level of p < 0.05.<br><br>
List of abbreviations used
Ad-GFP, <span class="yellow">adenovirus</span> carrying GFP gene; Ad-GFP/IGF2R-Rz, <span class="yellow">adenovirus</span> carrying both the ribozyme against M6P/IGF2R and the GFP gene; GFP, green fluorescent protein; IGF-II, insulin-like growth factor II; M6P/IGF2R, mannose 6-phosphate/insulin-like growth factor II receptor; Rz, ribozyme.<br><br>Authors' contributions
ZC carried out construction of the ribozyme, production of the viruses, cellular experiments, biochemical assays and data analysis.
YG carried out the RPA assay and participated in the molecular biological studies.
JXK conceived of the study, participated in its design and coordination, and drafted the manuscript.<br><br>
<h3>pmcA1075922</h3>Tissue-dependent isoforms of mammalian Fox-1 homologs are associated with tissue-specific splicing activities
Abstract
An intronic hexanucleotide UGCAUG has been shown to play a critical role in the regulation of tissue-specific alternative splicing of pre-mRNAs in a wide range of tissues. Vertebrate Fox-1 has been shown to bind to this element, in a highly sequence-specific manner, through its RNA recognition motif (RRM). In mammals, there are at least two Fox-1-related genes, ataxin-2 binding protein 1 (A2BP1)/Fox-1 and Fxh/Rbm9, which encode an identical RRM. Here, we demonstrate that both <span class="yellow">mouse</span> Fxh and A2BP1 transcripts undergo tissue-specific alternative splicing, generating protein isoforms specific to brain and muscle. These tissue-specific isoforms are characterized for their abilities to regulate neural cell-specific alternative splicing of a cassette exon, N30, in the non-muscle myosin heavy chain II-B pre-mRNA, previously shown to be regulated through an intronic distal downstream enhancer (IDDE). All Fxh and A2BP1 isoforms with the RRM are capable of binding to the IDDE in vitro through the UGCAUG elements. Each isoform, however, shows quantitative differences in splicing activity and nuclear distribution in transfected cells. All Fxh isoforms and a brain isoform of A2BP1 show a predominant nuclear localization. Brain isoforms of both Fxh and A2BP1 promote N30 splicing much more efficiently than do the muscle-specific isoforms. Skeletal muscles express additional isoforms that lack a part of the RRM. These isoforms are incapable of activating neural cell-specific splicing and, moreover, can inhibit UGCAUG-dependent N30 splicing. These findings suggest that tissue-specific isoforms of Fxh and A2BP1 play an important role in determining tissue specificity of UGCAUG-mediated alternative splicing.<br><br>INTRODUCTION
Alternative splicing of pre-mRNA is one of the fundamental mechanisms for the regulation of gene expression in higher eukaryotes (1,2). Developmentally regulated, cell type- or tissue-specific, and signal-induced alternative splicing of pre-mRNAs takes place in multicellular organisms throughout their lifetimes. Misregulation or abnormalities in pre-mRNA splicing, in some instances, leads to cellular dysfunctions found in <span class="blue">human</span> and animal diseases (3–5). Using various model systems of regulated alternative splicing, a number of pre-mRNA features that influence alternative splice site selection have been defined (1,2). These include enhancer and repressor RNA sequences located in exons and introns. Identification of RNA-binding proteins targeting these cis-regulatory elements is currently in progress. In vertebrates, participation of the SR family proteins and hnRNP proteins, such as PTB and hnRNPA1, in alternative splicing regulation via binding to the cis-regulatory elements have been shown in many tissue-specific splicing models (6,7). Although these RNA-binding proteins are ubiquitously expressed, their different abundance in different cells, differences in their post-translational modifications in different cellular contexts and their different abilities to assemble multiprotein complexes in different pre-mRNA contexts are thought to contribute to the determination of cell type-specific patterns of alternative splicing. For the last few years, tissue-specific and tissue-enriched RNA-binding proteins have begun to be identified as splicing regulators. These include brain-specific (or enriched) Nova-1, nPTB (brPTB) and some of the CELF family proteins (8–12). Discovery of these proteins has had a great impact on studies aimed at understanding the molecular mechanisms of alternative splicing regulation.
One of the intronic cis-elements, which are involved in tissue-specific or differentiation stage-dependent regulation of alternative splicing, is the hexanucleotide UGCAUG. The importance of this element was originally recognized in fibronectin pre-mRNA by Huh and Hynes (13). Since then, alternative splicing specific to a variety of tissues or cell types, including neural cells, muscles, epithelial cells and erythrocytes, has been shown to be modulated via this element (14–21). Recently, Jin et al. (21) discovered that a <span class="yellow">zebrafish</span> homolog of <span class="yellow">Caenorhabditis elegans</span> Fox-1 (22) could bind specifically to the pentanucleotide GCAUG by in vitro selection from randomized RNA sequences. This pentanucleotide is almost identical to the hexanucleotide UGCAUG except for the first U. Moreover, the <span class="yellow">zebrafish</span> Fox-1 homolog, as well as the <span class="yellow">mouse</span> Fox-1 homolog, are capable of repressing the inclusion of an alternative cassette exon of the ATP synthase F1γ pre-mRNA via binding to GCAUG, which mimics muscle-specific exclusion of this exon. This <span class="yellow">mouse</span> homolog is identical to the ataxin-2 binding protein 1 (A2BP1), which has been previously cloned in <span class="yellow">humans</span> and <span class="yellow">mice</span> as the cDNA encoding a protein, which interacts with ataxin-2, the product of the causative gene for spinocerebellar ataxia type 2 (23,24). In addition to A2BP1/Fox-1, another <span class="yellow">mouse</span> homolog of <span class="yellow">C.elegans</span> Fox-1, Fxh, has been independently cloned as a cDNA, which is induced by androgen in motor neurons (25). Of note is that A2BP1/Fox-1 and Fxh share an identical RNA recognition motif (RRM) at the amino acid level. Therefore, two genes in the <span class="yellow">mouse</span> genome encode homologs of nematode Fox-1. According to the names given by the first cDNA cloning, we used the nomenclature of A2BP1 and Fxh in this report. A2BP1 and Fxh have been named A2BP1 and Rbm9, respectively, in the <span class="blue">human</span> and <span class="yellow">mouse</span> genomes.
We have been studying regulatory mechanisms of neural cell-specific alternative splicing using the non-muscle myosin heavy chain II-B (NMHC-B) gene as a model system (14,26). NMHC-B mRNA is expressed ubiquitously. However, an alternative exon, N30, which encodes a 30 nt coding sequence, is included in the mRNAs from some neural cells, but is skipped in those from all other cells in mammals and birds (27,28). In cultured cells, a switch in N30 splicing from exclusion to inclusion can be seen in neural retinoblastoma Y79 cells during the post-mitotic and differentiated stages triggered by butyrate treatment. We have previously defined an intronic distal downstream enhancer (IDDE), which confers neural cell specificity on N30 inclusion, using this cell line (14). The IDDE includes two copies of UGCAUG. Mutation of these hexanucleotides results in N30 skipping in post-mitotic differentiated Y79 cells.
In this study, we investigated the possible involvement of A2BP1 and Fxh in the regulation of N30 splicing. To this end, we have isolated cDNA clones for A2BP1 and Fxh from brain and muscles. cDNA cloning revealed the existence of tissue-specific (enriched) isoforms of both A2BP1 and Fxh. Of importance, different isoforms of A2BP1 and Fxh show different activities with respect to N30 splicing as well as different subcellular localizations.<br><br>MATERIALS AND METHODS
Database disposition
The sequences reported in this paper have been deposited in the GenBank database with accession numbers AY659951 (F011), AY659952 (F411), AY659953 (F402), AY659954 (A016), AY659955 (A030), AY659956 (A713), AY659957 (A715) and AY659958 (A704).<br><br>RNA preparation and RT–PCR
Total RNAs were isolated from <span class="yellow">mouse</span> tissues and cultured cells using an RNA isolation kit (Stratagene) or an RNeasy mini kit (Qiagen). To obtain the full-length coding regions of cDNAs for Fxh and A2BP1, RT–PCRs were performed using Superscript II RNase H− reverse transcriptase (Invitrogen) and Pfu Turbo DNA polymerase (Stratagene). The PCR primers used to obtain all Fxh cDNAs were 5′-ctcaggcctccactagttATGGAGAAAAAGAAAATGGTAACTC-3′ and 5′-ctcaggcctcctctagaaGTAGGGGGCAAATCGGCTGTA-3′. The upstream primers for the brain (A016 and A030) and muscle (A713, A715 and A704) A2BP1 cDNAs were 5′-ctcaggcctccactagtgATGAATTGTGAAAGAGAGCAGCT-3′ and 5′-ctcaggcctccactagtcATGTTGGCGTCGCAAGGAGTCC-3′, respectively, and the downstream primer for all A2BP1 cDNAs was 5′-ctcaggcctcctctagagATATGGAGCAAAACGGTTGTATCC-3′. Lower case letters represent adapter sequences including restriction enzyme sites. 5′ Rapid amplification of cDNA ends (RACE) was performed using Marathon-Ready cDNA (BD Biosciences Clontech). For the analysis of minigene and NMHC-B mRNAs, RT–PCRs were performed as described previously (14,26). Sequences of primers P1–P9 shown in Figures 1, 4 and 6 are as follows: P1, 5′-AATTCACCCAGCAACCAGAAT-3′; P2, 5′-TAGAGGGATGTAAGTGTTGATGCC-3′; P3, 5′-CAGAGGGCGGACAGTGTATGGT-3′; P4, 5′-GGCGGCAGGGGCGAGGGCAT-3′; P5, 5′-CCGTGGTCGCACCGTGTACAAC-3′; P6, 5′-CAGCGGCAGTGGCAGGGGTG-3′; P7, 5′-AGGAAGAAAGGACCATAATATTCC-3′; P8, 5′-CCTCCACCCAGCTCCAGTTGT-3′; and P9, 5′-CCTGTAGTTATTAAATCCTTCAAG-3′.<br><br>Preparation of expression constructs and minigenes
The cDNAs of Fxh and A2BP1 were introduced into a plasmid pCS3 + MT, which contains a myc-epitope, and its modified version pCS3 + MT + NLS, which in addition contains the nuclear localization signal (NLS) of the <span class="yellow">SV40</span> large T antigen (26). Minigenes G, J without the IDDE, and H with the wild-type IDDE are the same as minigenes W, D4 and Cm0, respectively in ref. (14). The 201 nt IDDEs with mutations were generated by recombinant PCR using the appropriate primers, which included mutated sequences. The hexanucleotide TGCATG sequences at the 5′ and 3′ sides were changed to GTTACT and ACCTAC, respectively.<br><br>Electrophoresis mobility shift assay
Template DNAs for in vitro RNA transcription were prepared by PCR using the wild-type and mutant IDDEs in the minigenes as templates and an upstream primer that included the T7 promoter sequence at the 5′ end. The probe and competitor RNAs were transcribed by T7 RNA polymerase in the presence of [α-32P]UTP and a trace amount of [35S]UTPαS, respectively, using a MAXIscript kit (Ambion). Mole concentrations of synthesized RNAs were estimated by radioactivities. Fxh and A2BP1 proteins with a myc tag were synthesized in vitro by using a TNT quick-coupled transcription/translation system (Promega) from pCS3 + MT constructs, which include the SP6 promoter. Binding reactions were carried out in a 10 μl mixture that contains 10 mM HEPES (pH 7.9), 2 mM MgCl2, 50 mM KCl, 5% glycerol, 0.5 mM DTT, 5 μg tRNA, 1 μl reticulocyte lysate reaction mixture and 15 fmol of probe, on ice for 20–30 min. An aliquot of 5 μg of heparin was added to the reaction 10 min before gel electrophoresis. The reaction mixtures were analyzed by electrophoresis in a 6% polyacrylamide gel using a 0.5× TBE buffer (Invitrogen).<br><br>Cell culture and transfection
The <span class="blue">human</span> retinoblastoma cell line Y79 was cultured and transfected with DNAs as described previously (14,26). Total amounts of transfected DNAs were adjusted to be constant by addition of the empty vector. Either Lipofectin (Invitrogen) or Effectene transfection reagent (Qiagen) was used for the transfection. For stable transfection, the pCS3+MT expression constructs were co-transfected with a plasmid carrying a neomycin resistant gene and selected by 0.2 mM geneticin (Invitrogen). For differentiation of Y79 cells, cells were plated on the poly-d-lysine-coated plates and then treated with 2.0–2.5 mM sodium butyrate for 4–5 days. HeLa cells were cultured as described and transfected with DNA using Effectene reagent (14,26).<br><br>Immunoblot analysis
Samples that required both protein and mRNA analysis were split upon harvesting. Total cell proteins were subjected to SDS–PAGE and blotted as described previously (26). The primary antibodies used are monoclonal antibodies to a myc-epitope (Invitrogen) and green fluorescent protein (GFP) (Clontech). Binding of antibodies was detected with the SuperSignal System (Pierce) or ECL (Amersham).<br><br>Immunofluorescent microscopy
HeLa or Y79 cells grown in a four-chamber glass slide were transfected as described above. Cells were fixed with 10% formaldehyde 24–48 h after transfection, and permealized with 0.5% Triton X-100 then blocked with 5% <span class="yellow">goat</span> serum. Primary antibodies used were <span class="yellow">mouse</span> anti-myc (Invitrogen), <span class="yellow">rabbit</span> anti-NMHC-B (29). Secondary antibodies were Alexa Fluor 594 <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG and Alexa Fluor 488 <span class="yellow">goat</span> anti-<span class="yellow">rabbit</span> IgG (Molecular Probes). DAPI was used for DNA staining. Specimens were mounted in ProLong antifed kit (Molecular Probe). The images were collected using Leica SP confocal microscopy (Leica).<br><br>
RESULTS
Tissue-dependent isoforms of Fxh and A2BP1 and their subcellular distribution
The A2BP1 mRNAs are detected almost exclusively in brain and striated muscles (heart and skeletal muscles) in adult <span class="yellow">mice</span>, whereas the Fxh mRNAs are expressed in a wide variety of tissues with the highest expression in brain and heart [(21,25) and S. Kawamoto, unpublished data]. These expression profiles prompted us to characterize the mRNAs of A2BP1 and Fxh in brain and muscles. We have cloned cDNAs for two gene transcripts from brain, heart and skeletal muscles by RT–PCR and 5′ RACE. The isolated cDNAs are schematically presented with genomic structures in Figure 1 (for amino acid sequences see also Figure 8). Both gene transcripts are found to undergo tissue-specific alternative splicing. In both cases, brain and striated muscles express unique splice variants generated by mutually exclusive splicing of exons B40 and M43, respectively, which provide different coding sequences in the middle of the carboxyl half of the molecules. Southern blot analysis of the RT–PCR products of the Fxh mRNAs from the different tissues probed by oligonucleotides corresponding to B40 and M43 shows that M43 is exclusively used in heart and skeletal muscles, while B40 is predominantly used in brain (Figure 2A). Digestion of the RT–PCR products of the A2BP1 mRNAs with restriction enzymes unique to B40 and M43 demonstrates that B40 and M43 are used almost exclusively in brain and muscles, respectively (Figure 2B). A2BP1 contains an additional cassette type alternative exon A53, consisting of 53 nt. Inclusion and exclusion of exon A53 results in two different amino acid sequences at the C-terminal region owing to a frame shift. In the case of the A2BP1 gene, moreover, it appears that brain and striated muscle utilize alternative promoters, resulting in different amino acid sequences at the N-terminus. The 5′ RACE of skeletal muscle mRNAs yielded essentially a single species of sequence with 29 unique N-terminal amino acids. The 5′ RACE using brain mRNAs; however, yielded multiple products with multiple deduced amino acid sequences, all of which differ from the muscle amino acid sequence at the very N-terminus. Here, we focus our analysis on a clone containing nine unique amino acids at the N-terminus, which was obtained most frequently. Exon–intron organization of Fxh and A2BP1 shows remarkable similarities and a RRM is encoded in four exons (Figure 1B). Of note is that the significant amounts of Fxh and A2BP1 mRNAs from skeletal muscles are missing a part of the RRM by exon skipping. Typically, as shown in Figure 2C, they lack the 93 nt exon that encodes RNP1, one of the two most critical motifs of the RRM (30). Some of them lack almost the entire RRM (e.g. F402 in Figure 1A).
To examine the subcellular distribution of each isoform of Fxh and A2BP1, myc-tagged proteins were transiently expressed in cultured cells and immunostained with an anti-myc antibody. Initially, HeLa cells were used to investigate subcellular localization, since these cells are more suited for these studies. Representative confocal images are shown in Figure 3A. DAPI and anti-NMHC-B antibodies serve as markers for nuclei and cytoplasm, respectively. As noted, the ratio of protein distributed between nuclei and cytoplasm differs among the proteins. All isoforms of Fxh have a predominant nuclear localization. The brain isoform of A2BP1 without the A53 exon (A016) localizes to both nuclei and cytoplasm, whereas the other brain isoform with the A53 exon (A030) localizes predominantly to cytoplasm with only a minimum being in the nuclei. The relative amounts of both muscle isoforms of A2BP1 (A713 and A715) are somewhat between the amounts of the two brain isoforms. These data are summarized in Figure 1A. The subcellular distribution of representative isoforms (F011, A016 and A030) was also examined in retinoblastoma Y79 cells, which were used as host cells for the transfection experiments in order to characterize splicing activities of Fxh and A2BP proteins (see below). Although Y79 cells have a spherical shape and have only thin cytoplasm, exogenously expressed isoforms of Fxh and A2BP1 show a similar subcellular distribution as in HeLa cells (Figure 3B). F011 localizes predominantly to nuclei, A016 to both nuclei and cytoplasm and A030 predominantly to cytoplasm.<br><br>Specific interaction of Fxh and A2BP1 with IDDE via a hexanucleotide UGCAUG
Fxh and A2BP1 share an identical RRM and A2BP1 has been reported to bind specifically to the pentanucleotide GCAUG through this RRM (21). We have previously reported that the IDDE of the NMHC-B transcript, which is indispensable for the regulation of neural cell-specific cassette type exon N30 splicing, has two copies of GCAUG (14). Therefore, we investigated whether Fxh and/or A2BP1 bound to the IDDE. Electrophoretic mobility shift assays (EMSAs) were carried out using labeled IDDE and in vitro transcribed and translated Fxh and A2BP1, which include a myc-epitope. Since all Fxh and A2BP1 isoforms, except F402 and A704, contain the identical RRM, representative isoforms were analyzed. The expression of F011 in reticulocyte lysate causes the formation of a RNA–protein complex whose migration shift distinguishes it from those of the control reticulocyte lysate (C in Figure 4B, lanes 3 and 10). The unlabeled wild-type IDDE competes with the probe efficiently for the formation of the specific complex C (Figure 4B, lane 4). On the other hand, the mutant mc (Figure 4A), which has a mutation in both copies of UGCAUG, does not (Figure 4B, lane 7). The mutant ma, which has a mutation in the hexanucleotide at the 5′ side, shows less efficient competition, compared with mb, which has a mutation in the 3′ side of the hexanucleotide (Figure 4B, lanes 5 and 6), indicating that the nucleotides at the 5′ side are more important than those at the 3′ side. The presence of an anti-myc antibody, but not a non-specific antibody, inhibits the formation of the specific complex (Figure 4B, lane 8). Synthesis of the full-length F011 protein using a reticulocyte lysate and specificity of the myc antibody for the expressed protein are verified by immunoblot analysis (Figure 4B, lanes 11 and 12). A016 and A030 show essentially identical results to those with F011 (data not shown). These results indicate that Fxh and A2BP1 can bind to the IDDE and that the hexanucleotide UGCAUG is required for their binding.<br><br>Fxh and A2BP1 enhance N30 inclusion in a UGCAUG-dependent manner
To study whether Fxh and A2BP1 regulate neural cell-specific splicing via binding to UGCAUG, the Fxh and A2BP1 expression constructs were co-transfected into retinoblastoma Y79 cells with a number of the reporter minigene constructs, which include the wild-type or a mutant version of UGCAUG in the IDDE (Figure 4A). Minigenes H and J consist of the exons E5, N30 and E6 and their flanking introns with some deletions in the introns. The IDDE with or without mutations in the hexanucleotide is included or excluded between N30 and E6. As described above, each isoform of Fxh and A2BP1 enters the nucleus to a different extent. To see the effects of different proteins on N30 splicing itself, independent of their differential properties in nuclear localization, an exogenous NLS was added to the expressed proteins. Since Fxh and A2BP1 proteins contain the identical RRM, and F011, A016 and A030 all show the same UGCAUG-dependent binding to the IDDE in vitro, F011 and A030 were used for these experiments. Host cells Y79 at the proliferating stage exclude the N30 exon in ∼90% of the endogenous NMHC-B mRNAs (e.g. see Figure 6A, lane 1).
As shown in Figure 4C, the mRNAs derived from minigene J exclude N30 without exogenous expression of Fxh or A2BP1 in Y79 cells, similar to the endogenous NMHC-B mRNAs (Figure 4C, upper panel, lanes 1–5). However, in the presence of exogenous expression of F011 and A030, the N30 inclusion is increased (Figure 4C, upper panel, lanes 7 and 12). The N30 inclusion is absolutely dependent on the presence of the IDDE (Figure 4C, upper panel, lanes 6 and 11). Moreover, mutation of either one of the two copies of UGCAUG (ma, mb) abolishes the inclusion of N30 (Figure 4C, upper panel, lanes 8–10, 13–15). In the context of minigene H, which contains shorter introns, the larger extent of N30 inclusion is induced by either F011 or A030 overexpression (Figure 4C, middle panel, lanes 7 and 12). The N30 inclusion of the minigene H mRNAs also depends on the presence of the IDDE (Figure 4C, middle panel, lanes 6 and 11). Mutation of both copies of UGCAUG (mc) results in a complete loss of N30 inclusion (Figure 4C, middle panel, lanes 10 and 15). The mutant ma shows stronger inhibition of N30 inclusion compared with mb (Figure 4C, middle panel, lanes 8 and 9). This observation is consistent with the competition experiments of the EMSA shown in Figure 4B, indicating that the 5′ hexanucleotide is more important than the 3′ hexanucleotide for binding of Fxh or A2BP1 to the IDDE as well as an activation of N30 splicing. Comparable amounts of F011 and A030 are expressed in each transfection as verified by immunoblots using an anti-myc antibody (Figure 4C, lower panel, lanes 6–17).
Since minigenes J and H lack a portion of the intron between exons N30 and E6 and, therefore, the IDDE is located ∼100 nt downstream of N30, instead of 1.5 kb as in the native gene, we also analyzed the effects of Fxh and A2BP1 on the N30 splicing of the wild-type minigene G, which includes full-length introns among E5, N30 and E6 (Figure 4A). As shown in Figure 4C (right panel), both F011 and A030 are capable of promoting N30 inclusion, with F011 showing a higher activity (Figure 4C, lanes 16 and 17). Although A030 can promote N30 inclusion as efficiently as F011 in the contexts of the minigene J and H transcripts, it can do so less efficiently than F011 in the context of the minigene G transcript. Interpretation of this observation will be discussed below.
Taken together, Fxh and A2BP1 can activate N30 inclusion in an IDDE-dependent manner. The hexanucleotide motif UGCAUG is indispensable for this activation.<br><br>Differential activities of alternatively spliced isoforms of Fxh and A2BP1 in promoting N30 inclusion
Both Fxh and A2BP1 mRNAs are expressed in brain and A2BP1 is also expressed in striated muscles and Fxh is expressed in an even wider variety of tissues. As demonstrated above, however, both Fxh and A2BP1 transcripts undergo tissue-dependent alternative splicing, producing muscle-specific and brain-enriched isoforms. Therefore, the relative activity of individual isoform of Fxh and A2BP1 in promoting N30 inclusion was compared using minigene G.
First, in order to evaluate the relative specific activity of each isoform in the splicing reaction separately from its ability to localize to nuclei, an exogenous NLS was included in the expressed protein to equalize the nuclear concentration of the expressed protein in these experiments. Essentially, all of the expressed proteins with the exogenous NLS are localized to the nucleus (data not shown). Therefore, the relative nuclear concentrations of the expressed proteins can be estimated easily by immunoblots. As shown in immunoblots in Figure 5A, similar quantities of proteins are expressed in a dose-dependent manner. Expression of the brain isoform F011 causes a dose-dependent increase in N30 inclusion and the extent of N30 inclusion reaches a plateau with ∼85% of the mRNAs including N30. In contrast, inclusion of N30 promoted by F411, the predominant isoform from skeletal muscles, reaches a plateau with only 40% of the mRNAs. With respect to A2BP1, the brain isoform A030 shows higher activity in N30 inclusion than the muscle isoform A715, which shows almost no activation, as shown in Figure 5A. Including additional isoforms, the activities of individual isoforms with an exogenous NLS in promoting N30 inclusion are shown in Figure 5B (lanes 8–13) and are also summarized in Figure 1A (nls). Special care was taken to ensure that a similar quantity of each protein was expressed and, if not, different amounts were tested to obtain comparable expression. In the presence of the exogenous NLS (nls), F011 and A016 show the highest activities among all isoforms tested. A30 shows a considerably lower activity than A016. Each of the muscle isoforms (F411, A713 and A715) has a lower activity than that of their brain counterparts (F011, A016 and A030, respectively). As expected, isoforms lacking a part or all of the RRM (F402 and A704) have no activity for N30 inclusion.
Next, the splicing activities of the wild-type proteins without an exogenous NLS were examined. Representative data are shown in Figure 5B (lanes 2–7) and the relative activities are summarized in Figure 1A (wt). The protein amounts detected by immunoblots in these experiments represent the total amounts of the proteins distributed to both the nuclei and the cytoplasm. The splicing activities of the wild-type proteins are consistent with the activities that combine the splicing activities of the proteins with the exogenous NLS and the activities of the native proteins to localize to nuclei. In the absence of the exogenous NLS (wt), the brain isoforms F011, and A016 to a lesser extent, are still capable of activating N30 inclusion efficiently. The muscle isoforms for both Fxh and A2BP1 (F411, A713 and A715), as well as the brain isoform A030, show only minimal activities. Therefore, F011 and A016 appear to have the most physiological relevance to N30 splicing activation.<br><br>Overexpression of Fxh and A2BP1 activates N30 inclusion of endogenous NMHC-B mRNAs
The <span class="blue">human</span> NMHC-B gene consists of 41 constitutive exons and 3 alternative exons. Its pre-mRNA is ∼156 kb in length and it is much more complex than the pre-mRNA from the minigenes. In addition, the minigene pre-mRNAs are driven by a heterologous promoter. Therefore, we next examined if Fxh and A2BP1 were capable of promoting N30 inclusion of the endogenous transcript. Y79 cells were stably transfected with the expression construct for F011 or A016. Both F011 and A016 are enriched in the brain and show higher activation of N30 inclusion in the minigene transcripts. mRNAs encoding endogenous NMHC-B were analyzed by using RT–PCR. As shown in Figure 6, the inclusion of exon N30 with and without another alternative exon, R18, in the endogenous mRNAs is markedly increased in the clones which were stably transfected with the construct for F011 or A016 containing an exogenous NLS (Figure 6, lanes 4 and 5). Although to a lesser extent, transfection of the wild-type constructs without an exogenous NLS also results in a significant increase in N30 inclusion (Figure 6, lanes 2 and 3). Thus, exogenously expressed Fxh and A2BP1 are capable of activating N30 inclusion not only in the transcripts from the minigenes, but also in those from the native NMHC-B gene in Y79 cells.<br><br>Fxh may cooperate with other factor(s) to promote N30 inclusion
To address the role of endogenous Fxh and its potential interaction with other proteins in promoting N30 inclusion, we made use of an isoform of Fxh, F402, which lacks the RRM. The mutant proteins lacking an RRM for other RNA-binding proteins have previously been reported to function in a dominant-negative fashion and inhibit the activities of the wild-type proteins (26,31). Therefore, the effects of F402 on the N30 inclusion of minigene H mRNAs were examined in the context of the Y79 cells treated with butyrate. Upon butyrate treatment, as reported previously, Y79 cells enter in a post-mitotic and differentiated stage, and importantly, the endogenous as well as the minigene mRNAs in those cells include N30 to a large extent, unlike those in the untreated cells that predominantly exclude N30 (14). As shown in Figure 7, in the absence of exogenous expression of F402, large quantities of the mRNAs derived from minigene H-wt, which contains the wild-type IDDE, include N30 (Figure 7, upper panel, lane 4). In contrast, only small quantities of N30 inclusion are detected in the mRNAs from minigene H-mc, which has mutations in both copies of UGCAUG in the IDDE (Figure 7, upper panel, lane 8). Therefore, the butyrate-treated Y79 cells contain factor(s), which are capable of activating the UGCAUG-dependent N30 inclusion. Co-transfection of the wild-type minigene H-wt with the F402 expression construct causes a dose-dependent inhibition of N30 inclusion (Figure 7, lanes 1–3), indicating that F402 has an antagonistic effect on the endogenous factors with respect to N30 inclusion. Notably, the UGCAUG-independent N30 inclusion seen in the mutant minigene H-mc is not affected by the co-expression of F402 (Figure 7, lanes 5–7), indicating that the inhibitory activity of F402 depends on the UGCAUG element. The parallel experiment with the F011 expression construct does not show a significant effect on N30 splicing in either the wild-type or mutant minigene (Figure 7, lanes 9–16). This may be due to the fact that N30 inclusion of the wild-type minigene mRNAs has already reached a maximal level and is consistent with the idea that butyrate-treated Y79 cells contain sufficient amounts of protein(s) functionally equivalent to F011, which can promote N30 inclusion in a UGCAUG-dependent manner. Since F402 does not bind to the UGCAUG element, and has a UGCAUG-dependent antagonistic effect on N30 splicing, it most probably disrupts protein–protein interactions of the endogenous proteins that are required for the UGCAUG-dependent activation of N30 splicing. Endogenous Fxh (and/or A2BP1) would be a good candidate whose function could be antagonized by exogenously expressed F402. Thus, this observation suggests that the endogenous Fxh has an effect on the activation of N30 splicing and that other proteins cooperate with Fxh for N30 activation. Since muscle cells express the RRM-defective isoforms to a significant extent (Figure 2C) and the wild-type F402 localize to nuclei efficiently, this also raises the possibility that the RRM-defective isoforms may have an inhibitory function on the N30 splicing in muscle cells.<br><br>
DISCUSSION
Two major findings are described in this report. First, Fxh and A2BP1 facilitate neural cell-specific inclusion of the cassette-type exon via binding to the specific intronic sequence UGCAUG. In addition to a minigene model system, Fxh and A2BP1 are capable of facilitating N30 inclusion of the endogenous pre-mRNA. This result provides an important demonstration of physiological relevance and supports the notion that the NMHC-B pre-mRNA is likely to be the true target for Fxh or A2BP1-mediated regulation. However, whether the endogenous Fxh or A2BP1 regulates endogenous NMHC-B pre-mRNA splicing needs to be determined in a future study. In vertebrates, small interfering RNAs and gene targeting strategies have recently been used successfully to address the roles of endogenous splicing regulators in alternative splicing of endogenous target pre-mRNAs (8,32–36). A second and more novel finding is the identification of tissue-specific isoforms of Fxh and A2BP1 with different splicing activities as well as different subcellular localizations. This finding raises the possibility that the products of the Fxh and A2BP1 genes can contribute to a mechanism as to how tissue specificity of alternative splicing is achieved.
Many splicing factors are detected not only in the nuclei, but also in the cytoplasm (37). They are shuttling between the nucleus and the cytoplasm and, in some instances, extracellular stimuli trigger changes in subcellular distribution of these proteins. Such translocations have been reported for hnRNPA1 and PTB (38,39). Moreover, a number of RNA-binding proteins have been demonstrated to play a role in multiple steps during gene expression in different subcellular compartments, such as pre-mRNA processing in nuclei, mRNA export from nuclei to cytoplasm and mRNA localization, stability and translation in cytoplasm (37,40). Therefore, not surprisingly, Fxh and A2BP1 isoforms were found to be distributed in both the nuclei and the cytoplasm in HeLa and Y79 cells. However, the relative ratios of proteins distributed between the two subcellular compartments at steady-state differ among the isoforms. In agreement with Jin et al. (21), substantial amounts of the brain isoform A016 are detected in nuclei. Other A2BP1 isoforms, the brain isoform A030 and the muscle isoforms A713 and A715, are only poorly detected in nuclei. This observation is consistent with the reports where endogenous A2BP1 in cerebellar Purkinje cells, hippocampus neurons and cardiac myocytes were shown to be localized essentially to the cytoplasm (23,24). Thus, inclusion and exclusion of A53 and differences in the very N-terminal sequences results in A2BP1 isoforms with a distinct subcellular localization. It is likely that A2BP1 proteins have multiple roles, involving both nuclear and cytoplasmic events. In contrast, all three Fxh isoforms predominantly localized to the nuclei. Therefore, in terms of their localization, Fxh proteins are better candidates for regulators of the pre-mRNA splicing that takes place in nuclei. Of note, however, our preliminary results of 5′ RACE, as well as the EST database, detect multiple 5′ end sequences for both Fxh and A2BP1 mRNAs, which are presumably generated by alternative promoters and alternative splicing. The diversity of the 5′ end cDNA sequences leads to the generation of a number of unique N-terminal amino acid sequences. Therefore, this study does not exclude the possible existence of other isoforms with different subcellular localizations for both Fxh and A2BP1. Our study also does not exclude the possibility that some of the isoforms translocate between the nucleus and the cytoplasm following stimuli.
The main aim of this study is to determine the relative activities of tissue-dependent isoforms of Fxh and A2BP1 in neural cell-specific and UGCAUG element-dependent alternative splicing. To obtain an indication of the relative specific activity of each isoform in transfected cells, the same amounts of the expressed proteins should be available for the splicing reaction in the nuclei. For this reason, an exogenous NLS was included in the expressed proteins. Essentially, all of the expressed proteins with the exogenous NLS localized to nuclei. Thus, the amounts of the expressed proteins determined by immunoblots represent the nuclear concentrations. The analysis using the proteins expressed with the exogenous NLS allowed us to compare directly the splicing activities of these proteins. Furthermore, this analysis also allowed us to define the critical regions of the proteins for splicing activation.
As shown in Figure 5, the splicing activities of the various isoforms of Fxh and A2BP1 are intrinsically different, regardless of the subcellular localization properties of the wild-type proteins. Among the isoforms tested in this study, F011 and A016, which include B40, are found to have higher activities in promoting N30 inclusion. When the primary amino acid sequences, outside of the RRM, of these two proteins are compared, the C-terminal regions (amino acids 190–377 of F011) show a higher homology with 71% identity, whereas the N-terminal regions (amino acids 1–112 of F011) show only 53% identity. The C-terminal region includes four subregions of nearly identical stretches of amino acids (Figure 8, I–IV). One subregion (II) includes 13 amino acids encoded by exon B40. Substitution of this subregion with exon M43 in Fxh causes substantial changes in amino acid sequences resulting in only a 21% identity in this region between F011 and F411. Another subregion (IV) is located at the C-terminal end and A030 lacks this homologous region by the inclusion of exon A53, which results in a frame shift. Since F411 and A030 show poor splicing activation compared with F011 and A016, respectively, these two subregions of F011 and A016 appear to serve as activation domains, presumably by interacting with other proteins. This notion is supported by the finding that the RRM-lacking isoform F402, which includes the same subregions II and IV as F011, functions apparently as a dominant-negative mutant to the wild-type Fxh, consistent with the interpretation that the mutant and wild-type are competing to interact with other protein(s). To date, Fyn tyrosine kinase and estrogen receptor-α have been reported to interact with Fxh, and ataxin-2 with A2BP1 (23,41,42). Whether these proteins participate in the regulation of pre-mRNA splicing is currently unknown. Of interest, A030 with the exogenous NLS activates N30 splicing as efficiently as F011 in the pre-mRNAs derived from minigenes J and H, which contain the shorter intron, whereas this isoform poorly activates N30 splicing in the pre-mRNA from minigene G, which contains the full-length intron. This observation implies that the interactions of A030 with different factors are required in the different pre-mRNA contexts. Therefore, the isoforms of Fxh and A2BP1 described here may have different effects on other UGCAUG-regulated alternative splicing.
The involvement of the hexanucleotide UGCAUG in regulated alternative splicing has been experimentally demonstrated in a number of neural cell-specific, as well as other tissue-specific, model systems (13–21). This hexanucleotide element plays a role, in most cases, as an enhancer in regulating alternative splicing of cassette-type exons as well as mutually exclusive exons. Furthermore, computational analysis has revealed that UGCAUG is over-represented in the introns in which splicing is regulated, compared with the constitutively spliced introns (43). This analysis also pointed to the UGCAUG element as playing a role in the regulation of tissue-specific alternative splicing in a wide range of tissues, but not in specific tissues.
To date, KH-type splicing regulatory protein (KSRP) (44), A2BP1 (21) and Fxh (this study) are known to be capable of binding to UGCAUG. KSRP is expressed ubiquitously and tissue-specific variants of this gene have not been described so far. In this study, we have described the existence of tissue-dependent isoforms of Fxh and A2BP1, which, while not identical in some areas of the molecule, may contain the same RRM. The physiological relevance of these isoforms is that they have different splicing activities and different subcellular localizations. The brain isoforms promote N30 inclusion more efficiently than the muscle isoforms of both Fxh and A2BP1. The isoforms lacking the RRM are normally expressed to a significant extent in skeletal muscles. This isoform is incapable of activating N30 splicing and, moreover, can inhibit N30 inclusion. The properties of these isoforms are consistent with Fxh and A2BP1 acting as regulators for N30 splicing, since N30 is included in neuronal cells, but excluded in muscles. Therefore, despite the tissue-independent occurrence of UGCAUG as a regulatory element, given the tissue-dependent isoforms of the UGCAUG-binding proteins (Fxh and A2BP1) with different activities, the hexanucleotide UGCAUG could confer tissue specificity on regulated splicing. One of the major problems in understanding the mechanisms responsible for alternative pre-mRNA splicing is the manner in which tissue specificity is determined. In vertebrates, to date, only a few tissue-specific proteins have been identified as splicing regulators (8–12). Here, we have shown that the tissue-dependent isoforms of the sequence-specific RNA-binding proteins, which themselves are generated by alternative splicing, have different activities in tissue-specific alternative splicing of target pre-mRNA. Therefore, these isoforms play a role in the determination of tissue specificity of target pre-mRNA splicing. The discovery of these tissue-dependent isoforms of the UGCAUG-binding proteins with different splicing activities adds an important new dimension to the molecular mechanisms responsible for regulating tissue-dependent alternative splicing mediated via UGCAUG.<br><br>
<h3>pmcA1079799</h3>Mutations of PIK3CA in gastric adenocarcinoma
Abstract
Background
Activation of the phosphatidylinositol 3-kinase (PI3K) through mutational inactivation of PTEN tumour suppressor gene is common in diverse cancer types, but rarely reported in gastric cancer. Recently, mutations in PIK3CA, which encodes the p110α catalytic subunit of PI3K, have been identified in various <span class="blue">human</span> cancers, including 3 of 12 gastric cancers. Eighty percent of these reported mutations clustered within 2 regions involving the helical and kinase domains. In vitro study on one of the "hot-spot" mutants has demonstrated it as an activating mutation.<br><br>Methods
Based on these data, we initiated PIK3CA mutation screening in 94 <span class="blue">human</span> gastric cancers by direct sequencing of the gene regions in which 80% of all the known PIK3CA mutations were found. We also examined PIK3CA expression level by extracting data from the previous large-scale gene expression profiling study. Using Significance Analysis of Microarrays (SAM), we further searched for genes that show correlating expression with PIK3CA.<br><br>Results
We have identified PIK3CA mutations in 4 cases (4.3%), all involving the previously reported hotspots. Among these 4 cases, 3 tumours demonstrated microsatellite instability and 2 tumours harboured concurrent KRAS mutation. Data extracted from microarray studies showed an increased expression of PIK3CA in gastric cancers when compared with the non-neoplastic gastric mucosae (p < 0.001). SAM further identified 2910 genes whose expression levels were positively associated with that of PIK3CA.<br><br>Conclusion
Our data suggested that activation of the PI3K signalling pathway in gastric cancer may be achieved through up-regulation or mutation of PIK3CA, in which the latter may be a consequence of mismatch repair deficiency.<br><br><br><br>Background
The phosphatidylinositol 3-kinase (PI3K)-AKT signalling pathway is involved in the regulation of diverse cellular processes, including cell growth, survival and motility. Abnormal activation of this pathway is frequently observed in various cancer types, leading to aberrant cell cycle progression, altered adhesion and motility, inhibition of apoptosis and induction of angiogenesis [1]. It has been previously reported that genetic alterations involving various members along this signalling pathway could lead to its activation in cancer. These include mutation, allelic loss or promoter methylation of the negative regulator PTEN [2]; or alternatively, chromosomal amplification or over-expression of the positive regulators PIK3CA [3-5] and the various AKT kinases [6,7]. Furthermore, changes in other related pathways that are commonly altered in cancer, such as those involved in growth factor stimulation via the G-protein-coupled receptors or through direct interaction with the activated form of small GTPase RAS, can also lead to PI3K-AKT pathway activation [8]. Activation of this pathway results in the phosphorylation of AKT at Thr-308/309 and Ser-473/474. These phosphorylated forms of AKT proteins have been detected by Western blot or immunohistochemistry in various cancer types, suggesting the frequent activation of PI3K-AKT pathway in the carcinogenic process [7,9].
Although genetic changes along the PI3K-AKT pathway have been repeatedly documented in brain, ovarian, endometrial, breast, prostate and thyroid cancers [1,2], reports on its mechanism of activation in gastric cancer are limited. Gastric cancer is the second most common cancer worldwide but its molecular basis of tumourigenesis is still poorly understood. Previous immunohistochemical study has demonstrated the presence of the phosphorylated form of AKT in 78% of gastric cancer [10], suggesting that activation of this pathway may also be common in gastric cancer. Though loss of heterozygosity (LOH) involving the PTEN locus has been demonstrated in 47% of gastric cancer in a recent study, mutation or promoter methylation was absent even in cases with LOH [11]. Thus data from this study could not support the two-hit inactivation of PTEN in gastric cancer, while the biological significance of PTEN haploinsufficiency remains controversial. Alternatively, amplification of AKT1 has been reported in a single case of gastric cancer [12], and amplification of PIK3CA associated with elevated mRNA levels has been found in 36% of gastric cancer [11]. More recently, Samuels et al. screened a diverse spectrum of <span class="blue">human</span> cancers for mutation in 16 PI3K or PI3K-like genes and found a high frequency of somatic mutation in PIK3CA, which encodes the p110α catalytic subunit. Major screening in colorectal cancer (CRC) identified PIK3CA mutations in 74 out of 234 (32%) cases, while mutations were also noted in 3 out of 12 (25%) gastric cancers. Reported mutations were mostly of missense type, and clustered within 2 regions in the helical and kinase domains. Expression of a "hot-spot" mutant, H1047R, conferred a significant up-regulation of lipid kinase activity of PIK3CA, suggesting it as an activating mutation [13]. In this study, we have examined a series of 94 <span class="blue">human</span> gastric adenocarcinomas for PIK3CA mutation. We have also examined PIK3CA expression level by extracting data from a large-scale gene expression profiling study previously performed for these cases [14,15]. Using SAM, genes with significant correlating expression with PIK3CA have also been identified.<br><br>Methods
<span class="yellow">Patient</span> samples preparation
DNA samples used for sequencing were prepared from frozen tumour and non-tumour gastric mucosae from 94 gastric cancer <span class="yellow">patients</span> who underwent gastrectomy in the Department of Surgery, Queen Mary Hospital, The University of Hong Kong, as previously described [16]. Majority of the frozen samples (n = 81) showed tumour component of over 70%, whereas in 13 cases a lower proportion between 50 to 70% was accepted due to the tumours' inherent diffuse infiltrative nature with entrapment of non-neoplastic components. Analysis for microsatellite instability (MSI), BRAF and KRAS mutation have been performed and reported previously [16]. RNA preparation and gene expression profiling using a cDNA microarray containing 44,500 cDNA clones, representing around 30,300 unique genes, has been performed and reported in 90 of these tumours in comparison to 22 non-tumour gastric mucosae [14,15]. This study was approved by the Ethics Committee of the University of Hong Kong.<br><br>Mutational screening
Mutation screening of PIK3CA was performed for exons 9 and 20, covering the mutational hotspots; and for exon 18, from which a mutation was found in a gastric cancer. Mutations in these 3 exons constituted 80% of all PIK3CA mutations detected in the previous study [13]. PIK3CA intron-specific external amplification primers and internal sequencing primers were designed according to the previous study [13] with some modifications [see Additional file 1]. In particular, primers for exon 9 have been modified to avoid amplification of homologous sequences located in other chromosomes. PCR products were generated using the external primers and directly sequenced using the internal primers with the DYEnamic™ ET Terminator Cycle Sequencing Kit (Amersham Biosciences, Freiburg, Germany) according to the manufacturer's instruction. Electrophoresis was performed in the ABI Prism® 3700 DNA Analyzer (Applied Biosystems, Foster City, CA, USA). For each exon, PCR products were generated from 2 independent PCR reactions for sequencing of the forward and reverse strands. For exon 9, 2 independent PCR followed by sequencing of the forward strand were performed. Analysis of the chromatograms was performed using the mutation analysis software Mutation Explorer™ (SoftGenetics, State College, PA, USA).<br><br>Extraction of expression data and statistical analysis
Gene expression data were extracted from the microarray database containing 126 samples (90 gastric cancers, 14 lymph node metastasis and 22 non-tumour gastric mucosae) based on a 3-fold signal above background ratio for either channel and with 80% good data [14]. Gene expression data from 20,336 cDNA clones satisfied this selection criteria and were extracted, which included a cDNA clone corresponding to PIK3CA (IMAGE clone number 345430, GenBank accession no. W72473). Expression data for PIK3CA was extracted and the differences in expression levels between tumour and non-tumour tissues were examined using the Student's t-Test. SAM was performed to identify genes with significant correlating expression with PIK3CA [17]. The missing values in the dataset were estimated by a K-nearest neighbours impute algorithm using 10 nearest neighbour [18] followed by 5000 permutations in the SAM analysis.<br><br>
Results
Among the 94 gastric adenocarcinoma analysed, we have detected PIK3CA mutation in 4 cases. Two cases harboured the mutation A3140G (H1047R) in exon 20, and the other 2 cases with mutations G1624A (E542K) and G1633A (E545K) in exon 9. Representative sequence chromatograms are shown in figure 1. All four mutations were absent in the corresponding non-neoplastic mucosae and thus were confirmed as somatic mutations. Though the overall mutation frequency (4.3%) was lower than that of the previous study, the nature of the 4 mutations found were consistent with those identified at the reported hotspots. In particular, the H1047R mutation has been reported in 2 gastric cancers and 15 colorectal cancers [13]. While the E542K and the E545K mutations were not found in gastric cancer in the previous series, a large number of colorectal tumours did harbour these 2 mutations.
PIK3CA mutation spectrum and their corresponding clinico-pathological features were listed in Table 1. We noted a higher tendency of high-level MSI in gastric cancers with PIK3CA mutations (3 in 4, 75%) than in those without (18 in 90, 20%). Moreover, though the overall incidence of KRAS mutation in the studied population was low (8 in 94), 2 of the 4 gastric cancers with PIK3CA mutation also harboured a KRAS mutation.
Since over-expression of PIK3CA has been reported in gastric cancer [11], we have also extracted PIK3CA expression data from our previous cDNA microarray study of these cases [14,15]. We have confirmed that expression level of PIK3CA was significantly higher in gastric cancers (n = 87, mean = 0.099, SD = 0.428) when compared with non-neoplastic gastric mucosae (n = 22, mean = -0.418, SD = 0.426; Student's t-Test, p < 0.001). Using PIK3CA expression level as a continuous variable for SAM analysis [17], we found 2910 cDNA clones (corresponding to about 2546 unique genes) whose expression associated positively with PIK3CA expression (median number of false significant = 0.372, Delta = 1.107) [see Additional file 2]. Interestingly, no gene was found to be negatively associated with PIK3CA expression.<br><br>Discussion
In this study, we have reported the presence of PIK3CA gene mutation in 4.3% of gastric cancer. A high tendency (3 in 4) of mismatch repair deficiency was noted in cases harbouring PIK3CA mutation. Though the small number of PIK3CA mutations in our study may not justify statistical claim of significance; suggestion of such, despite of its not being mentioned by the authors, can be found from a previous study in CRC by Samuels et al.. From their study of 33 MSI and 201 microsatellite stable (MSS) CRC cases, PIK3CA mutation was present in 48% of the MSI tumours, but only in 29% of the MSS tumours. A significant association would have been revealed if statistical analysis had been applied (Fisher's exact test, p = 0.014) [13]. Gastrointestinal tract cancers with MSI are known to have a different molecular pathway of tumour evolution compared with their MSS counterparts [19,20]. This can be attributed to their propensity for frameshift mutations in repeat sequences, resulting in selective disruption of genes with such sequences within their coding regions. With 2 poly-adenine tracts within its coding region, PTEN can be inactivated through frameshift mutations in MSI CRC, resulting in the selective targeting of the PI3K-AKT signalling pathway [21,22]. It is also known that mismatch repair deficiency would lead to an elevated rate of missense mutation due to impaired single nucleotide mismatch repair [23]. Thus, the observed higher incidence of PIK3CA missense mutation in MSI colorectal and gastric cancers suggests yet another mechanism for the activation of the PI3K-AKT signalling pathway through mismatch repair deficiency.
Our data also showed a higher tendency of KRAS mutation in cases with PIK3CA mutations (2 in 4) than in those without (6 in 90). Yet again due to the low incidence of both mutations in our samples, statistical significance may not be claimed. In the study by Samuels et al., some of the colorectal tumours with PIK3CA mutation also harboured KRAS or BRAF mutation [13]. The PI3K-AKT pathway is known to have a close association with the RAS-MEKK signalling pathway [8]. Constitutively active RAS can interact with the catalytic subunit of PI3K and lead to its activation. Ras-dependent PI3K activation contributes to the transforming phenotype by mediating anchorage-independent growth, cytoskeletal reorganisation and apoptosis evasion. It has been observed that genes involved in the same signalling pathway may manifest mutations in cancer cells in a mutually exclusive manner, presumably due to the lack of selective growth advantage in having a second hit in the already altered pathway. A prominent example is the mutually exclusive occurrence of the BRAF hotspot mutation (V600E) and KRAS mutations in colorectal cancer [24,25]. However, there exist other examples of alterations in multiple components of the same signalling pathway that may lead to a multi-level modulation of its activity. For example, non-V600E BRAF mutations tend to occur together with KRAS mutations [26], and inactivation of the secreted frizzled-related proteins (antagonists of WNT) by promoter methylation frequently coincides with mutations in the Adenomatous Polyposis Coli gene to achieve multi-level activation of the WNT signalling pathway in colorectal cancers [27]. Whether PIK3CA functions independently from RAS, or acts synergistically with RAS to produce additive effects on the activation of the same pathway awaits further clarification.
By extracting data from microarray, we have confirmed the up-regulation of PIK3CA expression in gastric cancer tissues compared with the non-neoplastic gastric mucosae and identified a large number of genes that showed a significant positive correlation in expression level with PIK3CA. These genes participate in diverse cellular processes with 177 as putative cell cycle-regulated genes [28] and 126 mapped to genes with known functions in cell cycle regulation, cell proliferation or DNA replication [see Additional file 2]. While some of these genes maybe induced by PIK3CA, others maybe co-ordinately regulated by common upstream signals. Expression data set at one point was limited in differentiating the above cause and consequence, yet it certainly revealed the complexity of the carcinogenic process and the intricate relationship of PIK3CA signalling with other cellular processes.
Contrary to our expectation, the incidence of PIK3CA mutation found in the current study (4%) is much lower compared with that observed by Samuel et al. (25%) [13]. The reason for discrepancy may simply be a result of sample bias as the previous study involved only a small number of gastric cancers (n = 12). However, ethnic differences can also be another possibility. The diverse pathological spectrum and aetiological factors of gastric cancers in different geographical locations may be paralleled by differences in molecular pathway of tumour development. Since our current study is only based on a Chinese population with an intermediate gastric cancer incidence, further studies involving <span class="yellow">patients</span> from different ethnic groups will be able to address this possibility.<br><br>Conclusion
Large-scale screening of gastric adenocarcinomas for PIK3CA mutations revealed a mutation incidence of 4.3%. Increased PIK3CA expression level was observed in gastric tumours compared with non-neoplastic mucosae. This increase in PIK3CA level was associated with the elevated expression of a large number of genes, which may constitute the upstream regulators or downstream targets of PIK3CA along the PI3K signalling pathway.<br><br>Competing interests
The author(s) declare that they have no competing interests.<br><br>Authors' contributions
VSWL carried out the molecular analysis, performed data analysis and drafted the manuscript. CWW, TLC, WZ assisted in the molecular analysis. KMC provided the clinical data. ASWC assisted in data analysis and edited the manuscript. SS and XC participated in the microarray study and data analysis. STY and SYL conceived of the study, participated in its design, coordination and data analysis, and edited the manuscript. All authors read and approved the final manuscript.<br><br>Pre-publication history
The pre-publication history for this paper can be accessed here:<br><br>
Supplementary Material<br><br>
<h3>pmcA1257442</h3>Identification of kinectin as a novel Behçet's disease autoantigen
Abstract
There has been some evidence that Behçet's disease (BD) has a significant autoimmune component but the molecular identity of putative autoantigens has not been well characterized. In the initial analysis of the autoantibody profile in 39 Chinese BD <span class="yellow">patients</span>, autoantibodies to cellular proteins were uncovered in 23% as determined by immunoblotting. We have now identified one of the major autoantibody specificities using expression cloning. Serum from a BD <span class="yellow">patient</span> was used as a probe to immunoscreen a λZAP expression cDNA library. Candidate autoantigen cDNAs were characterized by direct nucleotide sequencing and their expressed products were examined for reactivity to the entire panel of BD sera using immunoprecipitation. Reactivity was also examined with normal control sera and disease control sera from <span class="yellow">patients</span> with lupus and Sjögren's syndrome. Six independent candidate clones were isolated from the cDNA library screen and were identified as overlapping partial <span class="blue">human</span> kinectin cDNAs. The finding that kinectin was an autoantigen was verified in 9 out of 39 (23%) BD <span class="yellow">patient</span> sera by immunoprecipitation of the in vitro translation products. Sera from controls showed no reactivity. The significance of kinectin as a <span class="yellow">participant</span> in autoimmune pathogenesis in BD and the potential use of autoantibody to kinectin in serodiagnostics are discussed.<br><br>Introduction
Behçet's disease (BD) is a systemic vasculitic disease typified by a triad of symptoms including recurrent oral ulcers, genital ulcers and uveitis. In addition, skin, joint, large vessels, nervous system and gastrointestinal systems may be involved. BD is a global disease but has the highest prevalence in the region along the ancient 'Silk Road' in China. The etiopathogenesis of the disease remains unclear but microbial agent triggers, environmental factors, genetic predisposition, neutrophil hyperfunction, endothelial cell dysfunction and immunological abnormalities involving both T and B cells have been implicated. Increasing amounts of research evidence supports the possibility that it is an immune-mediated vasculitis, and that abnormal T-cell and B-cell reactions and autoantigen-driven autoimmunity play pivotal roles [1]. Systemic lupus erythematosus (SLE) is the prototypic systemic autoimmune rheumatic disease with autoantibodies against cellular (particularly nuclear) antigens, some of which are critically implicated in the autoimmune pathology while others provide valuable serodiagnostic markers for the disease. Unlike the picture in SLE and other related rheumatic diseases, in BD, antinuclear antibodies and antibodies to neutrophil cytoplasmic antigens etc. are not present. To date, since neither a specific autoantibody nor pathognomonic pathological index is available to help establish the diagnosis of BD, it is largely or solely based on clinical manifestations [2], and a dilemma in diagnosis is not a rare occurrence in clinical practice. Nevertheless, since the 1960s, there have been reports of autoantibodies against certain unknown components of <span class="blue">human</span> oral mucosa in sera of <span class="yellow">patients</span> with BD. Since then, sporadic reports on findings of autoantibodies in this disease have been described, such as antibodies to retinal antigen(s), heat shock protein (HSP) of some strains of <span class="yellow">Streptococcus sanguis</span> cross-reactive with <span class="blue">human</span> HSP polypeptide [3], antibodies to endothelial cell antigens (AECA) and antibodies to α-tropomyosin [4,5], attesting to the complicated humoral immune disorders in this disease.
This investigation was aimed at defining target cellular autoantigens using time-tested and well-established molecular techniques. Immunoscreening of expression libraries using BD sera was used since this approach has been successfully employed in the characterization of many clinically relevant antigens in systemic rheumatic diseases such as SS-A/Ro [6-9] and SS-B/La [10] antigens in Sjögren's syndrome (SjS) and centromere antigen CENP-B [11] in scleroderma. In addition, we have been successful in using this strategy to identify interesting autoantigens that have other biological significance. Examples of these include NOR90/hUBF [12], p80-coilin [13], Golgi autoantigens [14-16] and, more recently, GW182 [17].<br><br>Materials and methods
<span class="yellow">Patients</span> and sera
The currently used empirical criteria for the diagnosis of BD in this study were the criteria proposed by the International Study Group for BD (abbreviated as 'International Criteria') [2]. The study subjects of 39 Chinese BD <span class="yellow">patients</span> comprised 17 males and 22 females, mean age 37 ± 11.3 years old, who were divided into two subgroups: 25 typical BD <span class="yellow">patients</span> (Group I, satisfying the International Criteria) and 14 clinically diagnosed BD <span class="yellow">patients</span> who had recurrent oral ulcers and one of the symptoms of genital ulcers, eye symptoms or skin lesions as defined by the International Criteria, as well as additional symptom(s) closely related to BD as listed in the International Criteria, that is, gastrointestinal ulcerations, deep vein thrombosis or arthralgia/arthritis without evidence that the latter symptoms might be related to any other disease (Group II, defined as 'probable BD' in this study). Disease controls included 10 <span class="yellow">patients</span> with SLE and 10 with SjS, all satisfying corresponding international classification criteria. All BD <span class="yellow">patients</span> and disease controls involved in the study were <span class="yellow">patients</span> treated at the Rheumatology Department of Ren Ji Hospital, Shanghai, China, where their clinical data and serum samples were collected. Twenty normal control sera were randomly selected from healthy blood donors working in the same hospital. This study was approved by the institution review board of Ren Ji Hospital which is affiliated with Shanghai Second Medical University, and each <span class="yellow">patient</span> involved gave informed consent. All serum samples were preserved at -20°C or -70°C until use.<br><br>Cell lines and cell extracts
HeLa (ATCC CCL 2.2) and T24 (<span class="blue">human</span> transitional cell bladder carcinoma) were obtained from the American Type Culture Collection (Rockville, MD, USA). A <span class="yellow">bovine</span> aortic endothelial cell line was kindly provided by Dr Eugene G Levin from the Scripps Research Institute (La Jolla, CA, USA). Cells were cultured in DMEM containing 10% <span class="yellow">calf</span> serum, harvested and extracted in Buffer A (150 mM NaCl, 10 mM Tris-HCl, pH7.2, 0.5% Nonidet P-40) with protease inhibitor (Complete™; Boehringer Mannheim, Indianapolis, IN, USA). For the preparation of whole cell extract, 10 volumes of Laemmli gel sample buffer [18] were added to the cell pellet, boiled for 3 min and stored at -20°C until use.<br><br>Western blot
Whole cell lysates from <span class="yellow">bovine</span> aortic endothelial cell, HeLa and T24 cells were resolved individually by discontinuous 7.5% gel SDS-PAGE according to Laemmli's method [18]. Immunoblotting was performed as described by Towbin et al. [19] with modifications. Nitrocellulose strips were blocked with 3% nonfat milk in PBS containing 0.05% Tween-20 (PBS-T) and then incubated with BD <span class="yellow">patient</span> sera and normal control sera (1:100 dilution) at room temperature for 1 h. Filters were washed extensively with PBS-T to remove any unbound antibodies. Bound antibodies were detected with polyvalent, peroxidase-conjugated <span class="yellow">goat</span> anti-<span class="blue">human</span> Ig and visualized by incubating the nitrocellulose strips in chemiluminescent reagents (NEN Life Science Products Inc., Boston, MA, USA) and exposing to Kodak XAR-5 films.<br><br>Screening of phage cDNA expression library with antibody probes
Serum from a BD <span class="yellow">patient</span> showing the highest antibody titer in immunoblotting was selected as a probe and used at a dilution of 1:300 for initial immunoscreening of approximately 106 recombinants from a T24 cDNA expression library. The latter was constructed in λZAPExpress vector (Stratagene, La Jolla, CA, USA) and screened as previously described [20-22]. All screenings were performed on duplicate isopropyl β-D-thiogalactoside (IPTG) pre-impregnated nitrocellulose filters, and immunoreactive clones were detected by chemiluminescence. Positive phages were subsequently plaque purified to 100% by two repeated rounds of screening at low plaque densities. Before screening the cDNA library, the BD serum was extensively adsorbed against bacteria and wild-type λZAP phage mixture to reduce background binding.<br><br>Analysis of candidate cDNAs
Purified candidate plaques were subcloned in vivo into pBK-CMV plasmids using ExAssist™ helper phage as recommended in the manufacturer's instructions (Stratagene). The recombinant pBK-CMV plasmids were then purified using QIAprep Spin Minprep Kit (Qiagen, Valencia, CA, USA). Restriction enzyme digestion of plasmids with EcoRI and XhoI and electrophoresis in a standard 1.0% agarose gel was used to analyze the length of cDNA insert of each candidate plasmid. The complete nucleotide sequence was determined using Bigdye terminator sequencing and a semi-automated sequencer model 377 (ABI, Foster City, CA, USA). Both nucleotide and deduced amino acid sequences were analyzed for similarity with known sequences using BLAST search [23] and ExPASy Proteomics tools . Secondary structure analysis for coiled-coil motifs was conducted with the software program COILS [24].<br><br>Immunoprecipitation of in vitro translation products
Candidate cDNA clones were used as templates for in vitro transcription and translation and the products were used as substrates for immunoprecipitation to confirm the specificity of reaction with BD sera. In brief, 1 μg of the pBK-CMV plasmid identified in the screening outlined above was added as template in a 50-μl reaction for the coupled in vitro transcription and translation reaction with a <span class="yellow">rabbit</span> reticulocyte lysate system (Promega, Madison, WI, USA) in the presence of 35S-methionine (Trans-35S label; ICN Biochemicals, Costa Mesa, CA, USA) and RNasin® Ribonuclease Inhibitor (Stratagene) as recommended by the manufacturer (Promega). Translation was carried out at 30°C for 1.5 h. Products were analyzed in a 12.5% gel SDS-PAGE and stored at -80°C for further immunoprecipitation analysis. The in vitro translation proteins were examined for reactivity by sera using immunoprecipitation described [8,25].<br><br>
Results and discussion
Autoantibody detection in sera from BD <span class="yellow">patients</span>
Initial examination of a group of 39 BD <span class="yellow">patients</span> using indirect immunofluorescence (IIF) on a HEp-2 cell substrate did not yield any characteristic nuclear or cytoplasmic staining patterns. BD is thought by some to be a vasculitic disease involving pathophysiology of endothelial cells, and antibody to endothelial cell antigen (AECA) has been reported. Reports on the prevalence of AECA have varied largely and alpha-enolase was reported as one of the putative target antigens [26]. In this study, the use of <span class="yellow">bovine</span> aortic endothelial cells as substrate for IIF did not provide any additional data. However, Western blot analysis of the BD sera began to show some interesting autoreactivity using cell lysates from both HeLa and <span class="yellow">bovine</span> aortic endothelial cells. HeLa cells were initially used for this analysis because they are commonly used in the laboratory as Western blot substrate. Fig. 1 illustrates the common reactivity to 49 kDa and 120 kDa proteins in the endothelial cell lysates. These antigens were also detected in HeLa and T24 cells; the latter cell line was analyzed because our laboratory at The Scripps Research Institute has produced an excellent expression cDNA library from the T24 line and the positive result with the T24 cell extracts allowed us to screen the T24 library. Ig isotype analysis showed that all reactivity was largely IgG antibodies. Since the 49 kDa and 120 kDa bands were observed in cell extracts from <span class="yellow">bovine</span> as well as <span class="blue">human</span> cell lines, these autoantigens might be evolutionarily conserved.
In total, nine out of 39 BD sera (23%) had autoantibody to the 49 kDa antigen and eight (20%) to the 120 kDa antigen. Four BD sera (10%) reacted with both proteins. Additionally, sera that showed common reactivity to the 120 kDa protein also demonstrated a common band that migrated at ~150 kDa, although it appeared weaker than the 120 kDa band. These antigens appeared to have different molecular weights than those of the known autoantigens in systemic rheumatic diseases. In addition, other reactive bands were detected but they were not as commonly shared as the 49 kDa and 120 kDa bands. The 49 kDa protein was shown to be distinct from 48 kDa SS-B/La or 50 kDa Jo-1 proteins (Fig. 1). The 120 kDa antigen was also shown to migrate differently from alanyl tRNA synthetase in another Western blot analysis (data not shown) and did not share any apparent crossreactive epitopes with the 49 kDa antigen. Western blot analyses of 20 normal control sera did not show the reactivities observed with BD sera. In order to further characterize these autoreactivities, a serum sample from the Group I definitive BD <span class="yellow">patients</span> with the strongest reactivity to 49 kDa and 120 kDa antigens (Fig. 1, lane 2) was selected as antibody probe for expression library screening.<br><br>Kinectin identified as a novel BD autoantigen
After screening 500,000 clones from the T24 cell λZAPExpress expression library, seven immunoreactive clones were isolated and plaque purified in two to three rounds to achieve 100% homogeneity. The cDNA inserts were subcloned in vivo into pBK-CMV plasmids, analyzed by restriction digestion using EcoRI and XhoI enzymes, and submitted to direct nucleotide sequencing across the polylinker arms. The cDNA inserts represented six independent clones designated BD41 (identical to BD44), BD481, BD42, BD47, BD482 and BD49. Their identities were established as overlapping partial cDNAs of <span class="blue">human</span> kinectin, ranging from 1.9 kb to 3 kb (Fig. 2a). The full-length <span class="blue">human</span> kinectin (GenBank accession number NM_182926[27]) has 4,816 bases containing an open reading frame coding 1,357 amino acid residues with molecular mass 156 kDa. All six cDNAs lacked the 5' portion of the kinectin sequence to different degrees but spanned a sequence of kinectin that extended to the 3'-untranslated region. Secondary structure analysis of kinectin protein using the program COILS identified a long region of α-helical coiled-coil domain that extended from amino acid residue 327 to the C-terminus (Fig. 2a, hatched boxes). In vitro coupled transcription and translation of BD44 and BD42 clones directed the synthesis of [35S]-methionine-labeled polypeptides that migrated at 95 and 60 kDa, respectively, in addition to smaller polypeptides (Fig. 2b). These products had predicted molecular weights of 103 kDa and 75 kDa.
Kinectin was initially identified in <span class="yellow">chick</span> embryo brain microsome as an integral membrane protein anchored in endoplasmic reticulum and involved in kinesin-driven vesicle motility along microtubules [28,29]. Kinectin consists of a 120-kDa and a 160-kDa polypeptide interacting through the α-helical coiled-coil domain to form a heterodimer [30]. The full-length kinectin is the 160 kDa polypeptide containing an N-terminal transmembrane helix followed by a bipartite nuclear localization sequence and two C-terminal leucine zipper motifs. We presume that the 120 kDa polypeptide detected in Western blot (Fig. 1) is the truncated version of the 160-kDa polypeptide, lacking the N-terminal first 232 amino acids [30]. The N-terminus of the 160-kDa polypeptide consists of a transmembrane domain that anchors kinectin to endoplasmic reticulum [30,31]. This 120 kDa polypeptide is probably the predominant form detected in the Western blot analysis (Fig. 1) because of its preferential solubility due to the omission of the N-terminal transmembrane domain.
Other functions for kinectin have been reported. <span class="yellow">Yeast</span> two-hybrid screen studies from several laboratories have revealed the interaction of the Rho family of GTPase with kinectin, and have shown the functional links among RhoG, kinectin and kinesin, with kinectin as a key effector of RhoG microtubule-dependent cellular activity [32]. Kinectin was also identified as an important constituent of integrin-based adhesion complexes, which link integrins to the cytoskeleton and recruit signaling molecules [33]. A new study reported that a kinectin isoform lacking a major portion of the kinesin-binding domain is very probably the most conservative form of kinectin; it does not bind kinesin but act as a membrane anchor for the translation elongation factor-1 delta in the endoplasmic reticulum [34].<br><br>Prevalence and specificity of anti-kinectin autoantibodies
The in vitro [35S]-methionine-labeled translation product of BD44, representing the largest recombinant kinectin fragment available, was used as the antigen substrate in an immunoprecipitation assay. Out of 39 BD <span class="yellow">patient</span> sera, nine (23%) recognized the BD44 translation product (Fig. 3), whereas sera from 20 normal controls, 10 SLE and 10 SjS <span class="yellow">patients</span> did not show reactivity. Among the nine anti-kinectin positive <span class="yellow">patients</span>, six (6/25, 24%) were from Group I (definitive BD) including the BD <span class="yellow">patient</span> whose serum was used in the immunoscreening of expression cDNA library, and three (3/14, 21.4%) <span class="yellow">patients</span> were from the Group II (probable BD) in this study. According to the Fisher Exact Probability calculation (P = 1.00), there is no statistically significant difference for antibody to kinectin between the two groups. The combined data substantiated the finding that kinectin is an autoantigen that can be recognized by sera from 23% of Chinese BD <span class="yellow">patients</span> in this study with at least one immunoreactive region or autoepitope residing within the BD44 encoded polypeptide.
Currently, there are more than six diagnostic/classification criteria for BD, among which the International Criteria have been applied most extensively due to its relatively high sensitivity (91%) and specificity (96%) [2]. As discussed above, differential diagnosis of BD might be confusing in clinical practice since no specific laboratory test is available, and some <span class="yellow">patients</span> may have symptoms and signs strongly suggestive of BD but do not fully satisfy the International Criteria, as in the Group II (probable BD) <span class="yellow">patients</span> in our study group. A number of investigators have pointed out that a comprehensive analysis of the clinical data for a given <span class="yellow">patient</span> is very important for correct clinical diagnosis of BD, and that classification/diagnosis criteria, including the International Criteria, should be followed but should not be exclusive. The observation that three out of 14 <span class="yellow">patients</span> in the probable BD group also had antibody to kinectin and the similar percentage of positive reactors between this group and Group I (21.4% versus 24%) supports this notion. The further use of non-clinical parameters such as immunological biomarkers as adjuncts to identify BD <span class="yellow">patients</span> could be of help in the classification of this disease entity
While our work was ongoing, anti-kinectin antibodies were reported in sera from <span class="yellow">patients</span> with hepatocellular carcinoma (HCC) [35,36] and aplastic anemia [37,38]. The first HCC report [35] identified kinectin as a tumor-associated antigen from the screening of an autologous cDNA library constructed from the cancer of a 30-year-old <span class="yellow">patient</span> from Guangxi, China. This report stated that four out of five HCC <span class="yellow">patients</span> tested were positive for anti-kinectin antibody [35]. In 2004, another laboratory also reported the cloning of kinectin as a tumor-associated antigen from a (presumably) different 30-year-old Chinese HCC <span class="yellow">patient</span> [36]. In contrast, anti-kinectin antibodies were not detected in other studies of HCC <span class="yellow">patients</span> associated with our laboratory [39,40]. The reports of anti-kinectin antibodies in aplastic anemia are also very interesting [37,38]. The initial report by Hirano et al. identified kinectin by screening an aplastic anemia <span class="yellow">patient</span> for candidate antigens using a Clontech <span class="blue">human</span> fetal liver cDNA expression library and it was concluded that seven out of 18 aplastic anemia <span class="yellow">patients</span> were positive for anti-kinectin while none of the normal or disease controls had this antibody [37]. In their recent report, Hirano et al. reported that anti-kinectin antibodies were found in 39% of aplastic anemia <span class="yellow">patients</span> from the United States but only in three out of 30 (10%) cases in Japan [38]. In our study reported here, kinectin antibodies were only detected in BD <span class="yellow">patients</span> and not in normal controls and SLE and SjS disease controls. None of the BD <span class="yellow">patients</span> with anti-kinectin had signs of HCC or aplastic anemia at the time of diagnosis and at up to 4 years of follow-up. Mapping of epitope(s) recognized by anti-kinectin antibodies may shed light on the question of whether different autoepitopes reside within the kinectin molecule recognized by sera from different diseases.<br><br>Kinectin – a new member of coiled-coil cytoplasmic autoantigens
We have recently reviewed the literature on the growing number of cytoplasmic autoantigens rich in α-helical coiled-coil domains as typified from our study of Golgi autoantigens [41]. Golgi autoantigens are generally high molecular weight proteins between 100 and 350 kDa and rich in coiled-coil domains in the central region with non-coiled-coil or globular domains at both N and C termini. Golgi autoantigens are displayed on the cytoplasmic face of the Golgi complex and are not localized to apoptotic blebs during apoptosis [42]. Giantin, the highest molecular weight Golgi autoantigen reported, is the predominant target of <span class="blue">human</span> anti-Golgi complex antibodies and multiple non-cross-reactive epitopes have been mapped spanning the 350 kDa protein [43]. Other high molecular weight autoantigens with similar features have been reported in cytoplasmic and mitotic organelles suggesting that these selected proteins become autoimmunogenic based on their subcellular association and molecular features [41]. For example, in the endosomal compartment, the two known autoantigens are early endosomal protein EEA1 (180 kDa) [44] and CLIP-170 (170 kDa) [45]. There is also a series of centrosomal autoantigens identified as coiled-coil-rich proteins including pericentrin, a 220 kDa protein [46], ninein, a protein with alternatively spliced products of 245 and 249 kDa [47], Cep250 (250 kDa) and Cep110 (110 kDa) [48]. Centromere autoantigens have been described but the two interesting ones related to this discussion are CENP-E [49] and CENP-F [50]; both are high molecular weight proteins (312 to 400 kDa) and have the same type of overall structure as discussed above. NuMA is another large coiled-coil protein located at the mitotic spindle pole and is the most common target autoantigen in sera with mitotic spindle apparatus staining [51]. Non-muscle myosin (~200 kDa) is a cytoskeletal autoantigen [52] that falls in the same group of high molecular weight and coiled-coil-rich autoantigens. These endosomal, centrosomal, mitotic apparatus and intracellular autoantigens are, like the golgins, proteins with high molecular weights and an overall high content of coiled-coil domains. The combination of these two physical features in autoantigens may contribute to the induction and production of autoimmune antibodies in certain disease states. Kinectin is an integral membrane protein largely confined to the endoplasmic reticulum [28,31] and it fits into this new category of autoantigens that are large coiled-coil rich proteins (≥100 kDa) in the cytoplasm.<br><br>
Conclusion
Here we report the detection of kinectin autoantibody in 23% of Chinese <span class="yellow">patients</span> with BD. The identity of kinectin as a BD-related autoantigen has not been reported to date. Autoantibody reaction against kinectin in BD observed in this study further confirms the autoimmune involvement in BD and may provide new inroads into elucidating the immunopathogenesis of the disease. In an effort to clarify the association of BD with antibody to kinectin, it is essential to measure antibody to kinectin in larger <span class="yellow">patient</span> populations including both BD, probable BD and important autoimmune rheumatic diseases such as SLE, SjS, rheumatoid arthritis etc., as well as those diseases not easily differentiated from BD, such as recurrent aphthous oral ulcer, Reiter's syndrome, inflammatory bowel diseases etc. On the other hand, further analysis of the association of anti-kinectin antibody with different manifestations or disease 'subtypes' of BD is another important project. Anti-kinectin is clearly only one of the antigen-antibody systems identified because there were many other antibodies observed in the Western blot analysis of BD sera. Using other sera for immunoscreening would probably lead to the identification of other potentially important antigen-antibody systems.<br><br>Abbreviations
AECA = antibody to endothelial cell antigen; BD = Behçet's disease; DMEM = Dulbecco's modified Eagle's medium; HCC = hepatocellular carcinoma; HSP = heat shock protein; IIF = indirect immunofluorescence; PBS = phosphate buffered saline; SjS = Sjögren's syndrome; SLE = systemic lupus erythematosus.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YL performed the study and drafted the manuscript. PY provided technical help throughout the study. SLC and EMT conceived the study, participated in the design and helped in the analysis of the data. EKLC participated in the design of the study, interpreted data and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1347483</h3>EndoNet: an information resource about endocrine networks
Abstract
EndoNet is a new database that provides information about the components of endocrine networks and their relations. It focuses on the endocrine cell-to-cell communication and enables the analysis of intercellular regulatory pathways in <span class="yellow">humans</span>. In the EndoNet data model, two classes of components span a bipartite directed graph. One class represents the hormones (in the broadest sense) secreted by defined donor cells. The other class consists of the acceptor or target cells expressing the corresponding hormone receptors. The identity and anatomical environment of cell types, tissues and organs is defined through references to the CYTOMER® ontology. With the EndoNet user interface, it is possible to query the database for hormones, receptors or tissues and to combine several items from different search rounds in one complex result set, from which a network can be reconstructed and visualized. For each entity, a detailed characteristics page is available. Some well-established endocrine pathways are offered as showcases in the form of predefined result sets. These sets can be used as a starting point for a more complex query or for obtaining a quick overview. The EndoNet database is accessible at .<br><br>INTRODUCTION
Theoretical analyses in the post-sequencing era, in particular in the context of systems biology approaches, increasingly investigate the properties of all kinds of pathways and networks such as metabolic and signaling pathways. It is commonly accepted that we need formal descriptions of these networks to make systematic use of the overwhelming body of facts gathered over decades of laboratory work both in narrow or global scale.
Corresponding databases have been created and are available now for metabolic networks (KEGG) (1,2), protein interaction networks (BIND and DIP) (3,4) and signaling pathways (CSNDB, Patika and TRANSPATH®) (5–8), just to name a few. So far, however, their main focus is on intracellular processes. Intercellular signaling is addressed only insofar as usually the pathways modeled start with extracellular ligands, and as there exist catalogs and databases about secreted proteins, or the ‘secretome’, of certain systems (9–12).
This shortcoming is part of the more comprehensive problem, the genotype–phenotype gap: from a certain genotype, we are able to infer a ‘molecular phenotype’, but for correlating it with a more complex phenotype such as a biological process, or even a certain disease and its clinical appearance, we still depend largely on the mere description of an observed correlation. There is no way to infer such a phenotype through all the different layers of increasing complexity between genomic DNA sequences and the physiological function of whole organs and their interplay within an organism.
There may be principal barriers preventing such an inference across different complexity levels, but even to explore these limits, we have to make attempts to bridge the genotype–phenotype gap. We have to do the next step towards modeling intercellular networks that are inextricably linked to the physiology of multicellular organisms (13). Being one of the most complex constructs in the body, the endocrine system comprises numerous cells and tissues that secrete hormones which pass through the body, activate specific receptors of target cells and initiate there multiple intracellular signaling pathways.
Here, we present a new database, EndoNet, which provides information about the components of endocrine networks and their relations, and enables the analysis of intercellular regulatory pathways in <span class="yellow">humans</span>. The EndoNet database is accessible at .<br><br>RESULTS
The biological schema of endocrine actions
Development and function of different organs as well as the response of a whole multicellular organism to its environment is coordinated through a complex communication system between specialized cells that are part of its organs. This communication is mostly mediated by hormones. In a broader sense, this functional class of biomolecules also comprises growth factors, cytokines, chemokines and other signal transmitters. This generic view is supported, for instance, by the definition given for ‘hormone activity’ by Gene Ontology (GO) (14): ‘Any substance formed in very small amounts in one specialized organ or group of cells and carried (sometimes in the bloodstream) to another organ or group of cells, in the same organism, upon which it has a specific regulatory action’. This definition is also broad enough to include modes of hormonal actions as diverse as endocrine, paracrine and autocrine effects. Accordingly, with the term ‘hormone’ one might refer to any extracellular substance that induces specific responses in target cell and helps to coordinate growth, differentiation, gene expression and metabolic activities of various cells, tissues and organs in multicellular organisms (15).
Hormones can be classified based on their chemical nature, solubility, the distance over which the signal acts and so on (15,16). From the viewpoint of genome–phenotype relations, it is reasonable to distinguish between polypeptide, thus, genome-encoded hormones, on one hand, and those low-molecular weight hormones such as steroids, with only the machinery of their synthesis being genome-encoded, on the other hand. Another classification of hormones, which seems to be overlapping with the previous one refers to the intracellular location of their receptors and, thus, how the subsequent signal is further transduced: membrane-bound receptors usually trigger more or less complex signaling cascades towards the nucleus, whereas nuclear receptors, mainly bound by low-molecular weight hormones, have a very short signaling pathway downstream since they act as transcription factors themselves (15,16).
In intercellular communication, we can basically differentiate between two kinds of cells: donor cells which synthesize and secrete a hormone, and acceptor cells which express a hormone receptor (Figure 1a). Donor cells become active under the influence of an external, mostly environmental, stimulus. In the acceptor cell, binding of the hormone to a receptor triggers an intracellular signal transduction cascade with different kinds of end nodes and effects: transcription factors affecting the gene expression program of the acceptor cell, metabolic enzymes controlling the cell's metabolism, structural components which define the acceptor's morphological features, or components of the secretory apparatus regulating the release of other extracellular molecules. If synthesis and secretion of another hormone is among the effects exerted by receptor activation, the acceptor is turned into a donor cell, thus becoming an internal node of the organism's endocrine network. Acceptor cells which do not become producers of another hormone are called ‘terminal target cells’ of the endocrine network, but finally constitute the overall physiological effect of the respective hormonal pathway, or simply the phenotype (Figure 1a).<br><br>The EndoNet data model
In the EndoNet data model, two classes of entities—hormones (in the broadest sense) and their acceptor or target cells expressing the corresponding receptors—span a bipartite directed graph. Since one and the same hormone may be secreted by multiple cell types (donor cells), each such secretion event is represented by a hormone node on its own. Similarly, each cell type known to express a hormone receptor (acceptor cell) leads to an individual node. The graph's edges represent hormone transport and binding to a receptor (intercellular edges), on one hand, and triggering or inhibition of hormone secretion by a receptor activated by hormone binding (intracellular edges), on the other hand. Optionally, an edge representing the transport of a hormone can be subdivided by introducing the transport medium (usually blood) as an additional, intermediary node.
Thus, in the conceptual schema of the EndoNet database (Figure 1b), the links between cells/organs and hormone define donor cells (‘D’), those between cells/organs and receptors acceptor cells (‘A’). If an acceptor cell synthesizes another hormone in response to an incoming signal, it becomes an internal node in the emerging hormonal network.
In EndoNet, the pathway between a hormone receptor expressed in an acceptor cell and a hormone synthesized in the same cell (intracellular edge) is handled as a black box. In case of genome-encoded peptide hormones, cross-references to entries in the TRANSPATH® database, which describe the signaling cascade starting from the hormone's receptor and ending at the hormone's gene, are provided, if available. Datasets on non-peptide hormones will in future be enriched by a specific metabolic add-on which will include references to the databases KEGG (1) and BRENDA (17), allowing for further characterization of the steps performed during the hormone's synthesis and the regulation of both activity and expression of the enzymes involved.
By now, the EndoNet structure already allows for including descriptions of the physiological effects induced by hormone binding (see below, Future Developments).<br><br>The contents of EndoNet
In the present version of EndoNet, and as a first approach, we consider the endocrine (hormonal) network of the <span class="blue">human</span> body. For each molecule (hormone or receptor), a primary name and synonyms are given. In case of peptide hormones, the sequence of the processed polypeptide, rather than that of the protein precursor, is specified. For a multimeric protein hormone, the subunit composition as well as the sequences of all subunits are stored; the same holds true for hormone receptors.
Additionally, all peptide hormone and receptor datasets have links to HumanPSD™ (18) and to the Swiss-Prot database, The structures of non-peptide hormones can be accessed through the corresponding hyperlinks to the KEGG COMPOUND section. Finally, all molecules may have links to the TRANSPATH® database.
As described, EndoNet utilizes data about the tissues from which hormones are secreted and in which receptors are expressed to define donor and acceptor cells, respectively. The identity and anatomical environment of cell types, tissues and organs is defined through references to the CYTOMER® ontology (19,20); in the numerous cases where the receptors are ubiquitously expressed, just the root term ‘<span class="blue">human</span> body’ is linked.
Data on whether a hormone's synthesis is triggered or inhibited by another hormone through its respective receptor in a particular cell was obtained by manual selection from textbooks [e.g. (16), ], monographies (21), original literature, the EST library information and the linked databases (TRANSPATH®, HumanPSD™ and Swiss-Prot). The contents of EndoNet are summarized in Table 1.<br><br>Web interface, queries and visualization
EndoNet can be accessed through the WWW via a JSP-based web interface. Hormones, receptors and tissues can be queried for their names, and detailed information on all identified components is available through individual characteristics pages. Each hormone's individual entry page displays its source and target tissues (donor and acceptor cells) as well as its receptors, along with some molecular data. Similarly, each receptor entry exhibits the tissues in which the receptor is expressed, and the hormones it interacts with. Finally, each tissue entry lists the hormone receptors that are found in, as well as the hormones that are synthesized and secreted by the tissue. It is also indicated whether the corresponding tissue exerts gender-specific properties; additional information based on the CYTOMER® ontology is available through the corresponding link on the tissue detail page.
Instead of searching for a name of a hormone, one can also browse the hierarchical hormone classification featured by EndoNet (available at the ‘Search’ page).
Each query result can be used as starting point for an extended query. Different items of interest can be selected and added to a common result set. Since several search results can be combined, it is possible to create sets with multiple search parameters.
At any step of this incremental retrieval process, the sets of hormones, receptors and tissues obtained so far can be used as starting points for reconstructing a network by a depth-first graph traversal algorithm. The maximum number of steps can be selected separately for the upstream and downstream part of the reconstruction process. Subsequently, the graph will be displayed using a Graphviz-based visualization method [(22), ]. In the resulting image, hormones and receptors are represented as nodes grouped together into subgraphs representing the tissues (cells/organs) they are secreted from or expressed in, respectively (Figure 2). Autocrine loops (donor and acceptor cell being identical) are treated specially for visualization. On demand, the hormones' transport media can be included in the visualization of intercellular edges, enabling the user to choose between different complexities of output. Intracellular edges, which connect a receptor to a hormone, represent the influence of a receptor's activation on the secretion of a hormone from the same cell and are displayed differently, depending on whether this influence is triggering or inhibitory in nature.
The graph is displayed as a clickable image map, linking each entity to its detailed characteristics page, thus making the database entries accessible from the graphical overview of a network, too. The graph is available in two different formats: PNG and SVG. While virtually every browser supports PNG (a ‘pixel’ or ‘bitmap’ format), only a few of them provide a zoom functionality for bitmap pictures. Scalable vector graphics, (SVG) provides more functionality (including perfect image quality throughout all zoom factors), but until now most browsers do not support SVG natively and require an SVG plugin () for displaying this vector-based format.
Some well-established endocrine pathways are offered as showcases in the form of predefined result sets. For instance, sets representing the hypothalamic–hypophyseal axis with a focus on either thyroid hormones, adrenal hormones, growth hormones or prolactin are provided. These predefined sets can be used for obtaining a quick overview or as starting points for more complex queries.<br><br>
FUTURE DEVELOPMENTS
Among the important improvements of EndoNet which we are currently working on is the possibility to represent intercellular communication at different levels of the hierarchical organization of organs, tissues and cells in the organism, as well as to distinguish between such communications in male and female organisms. These options will be introduced by a tighter integration with the CYTOMER-based ontology (19,20).
The next step will be to expand the contents of EndoNet towards the details of the processes occurring in the transport medium, usually the blood. Since not all hormones are transported as free molecules and some hydrophobic hormones (e.g. steroids and thyroids) need to be bound to specific carrier proteins, proper description of such transporters and their interaction with the corresponding hormones will be required. It is planned to involve quantitative data about the regular or pathological levels of the hormones, the overall kinetics of each hormone in the blood (monotonous decay, oscillating concentrations, increase in response to certain stimuli, etc.), its turnover and metabolic products, etc. That will allow utilization of EndoNet contents for diagnostic purposes. In future, the EndoNet data model will be extended in order to incorporate external stimuli (e.g. light) and physiological states (stress, age, etc.) in a formalized manner, allowing to determine whether or not and in which quantity a hormone will be secreted under the given circumstances. EndoNet will then link the physiological effects of hormones with the intracellular molecular processes leading to its synthesis and secretion in the donor cells, and to the effects on its acceptor cells. At many places of these intracellular and intercellular networks, genetically determined aberrations may cause specific, sometimes pathological phenotypes. Thus, EndoNet will enable to bridge the gap between known genotypes and their molecular and clinical phenotypes in this area of medical research and its applications.<br><br>DISCUSSION
At its present state, EndoNet provides a high coverage of molecules that are conventionally considered as hormones as well as other molecules that are involved in intercellular communication, such as growth factors, lymphokines and chemokines and their known receptors. The aim of the database is to provide a useful resource for studying the principal features of hormonal networks in a comprehensive way, as it was done more exemplarily in the past for these kinds of networks (23), but was done globally for many other intracellular network types, such as metabolic, protein interaction and transcription networks [reviewed in (24,25)]. EndoNet database certainly is not yet complete but will grow rapidly.<br><br>
<h3>pmcA140144</h3>Gender differences in factors influencing insulin resistance in elderly hyperlipemic non-diabetic subjects
Abstract
Background
The increase in the prevalence of insulin resistance-related metabolic syndrome, a disorder that greatly increases the risk of diabetes, heart attack and stroke, is alarming. One of the most frequent and early symptoms of metabolic syndrome is hypertriglyceridemia. We examined the gender differences between various metabolic factors related to insulin resistance in elderly non-diabetic <span class="yellow">men</span> and postmenopausal <span class="yellow">women</span> of comparable age suffering from hypertriglyceridemia, and compared them with healthy subjects of equal age.<br><br>Results
The indexes of insulin resistance HOMA IR and QUICKI were significantly higher in both hyperlipemic <span class="yellow">men</span> and <span class="yellow">women</span> than in controls; 95% confidence limits of hyperlipemic subjects did not overlap with controls. In both normolipemic and hyperlipemic <span class="yellow">men</span> and <span class="yellow">women</span> serum leptin correlated significantly with insulin resistance, while HDL-cholesterol correlated inversely with HOMA-IR only in <span class="yellow">women</span> (both normo- and hyperlipemic), and serum tumor necrosis factor α (TNFα) only in hyperlipemic <span class="yellow">women</span>. According to results of multiple regression analysis with HOMA-IR as a dependent variable, leptin played a significant role in determining insulin resistance in both genders, but – aside from leptin – triglycerides, TNFα and decreased HDL-cholesterol were significant determinants in <span class="yellow">women</span>, while body mass index and decreased HDL-cholesterol were significant determinants in <span class="yellow">men</span>. The coefficient of determination (R2) of HOMA IR by above mentioned metabolic variables was in <span class="yellow">women</span> above 60%, in <span class="yellow">men</span> only about 40%.<br><br>Conclusion
The significant role of serum leptin in determination of insulin resistance in both elderly <span class="yellow">men</span> and postmenopausal <span class="yellow">women</span> of equal age was confirmed. However, the study also revealed significant gender differences : in <span class="yellow">women</span> a strong influence of triglycerides, TNFα and decreased HDL-cholesterol, in <span class="yellow">men</span> only a mild role of BMI and decreased HDL-cholesterol.<br><br><br><br>Background
In association with pandemic obesity the prevalence of the insulin resistance-related metabolic syndrome is constantly growing [1]. As a consequence of this fact, type 2 diabetes mellitus and cardiovascular mortality occurs in much younger age groups [2]. A typical hyperlipemia, consisting of an increase of serum triglycerides and a decrease of serum HDL-cholesterol, is a characteristic and an early symptom of this syndrome [3].
With increasing age, body mass index (BMI) and adiposity, insulin sensitivity declines and the number of cardiovascular risk factors increases in both genders [4-6]. It was repeatedly demonstrated that plasma concentration of leptin – a hormone produced mainly by adipose tissue – is substantially higher in all age groups of <span class="yellow">women</span> than in <span class="yellow">men</span> [7-10]. This may be caused by different size and/or distribution of fat tissue compartments influenced by hormones: estrogens stimulate, whereas testosterone inhibits leptin secretion. In <span class="yellow">women</span> subcutaneous fat mass prevails – and during augmentation of overweight it increases – while in <span class="yellow">men</span> intra-abdominal fat mass prevails [11-13]. Subcutaneous fat in particular serves as a substantial source of tumor necrosis factor α (TNFα), which represents one of the factors that interfere with insulin signal transduction into the cells [14-16]. Leptin, TNFα and some other factors are abundantly expressed in adipose tissue and contribute to the insulin resistance that accompanies overweight and obesity. Leptin correlates positively with hyperinsulinemia, BMI, fat mass and hypertriglyceridemia, respectively, and correlates inversely with HDL-cholesterol and lean body mass [17-25].
The incidence and mortality of ischemic heart disease and of other consequences of atherosclerosis increases with age in both genders, especially after the age of sixty. In premenopausal <span class="yellow">women</span>, however, the incidence of these disorders is considerably less frequent than in <span class="yellow">men</span> of appropriate age. After the menopause the prevalence of metabolic syndrome and cardiovascular mortality in <span class="yellow">women</span> gradually increases, attaining values comparable to <span class="yellow">men</span> at about the age of 70 [2,26]. Paradoxically, it takes place at the time when serum leptin concentration in <span class="yellow">women</span> has relatively decreased [27,28].
The aim of this study was to analyze the interrelations between several metabolic variables and factors related to insulin resistance in groups of both normal and hyperlipemic postmenopausal <span class="yellow">women</span> and <span class="yellow">men</span> of appropriate age, and to attempt to elucidate the gender differences and some pathophysiologic mechanisms of these differences. We compared homeostatic indexes of insulin resistance HOMA IR and QUICKI, serum lipid and insulin parameters, uric acid, leptin and TNFα between groups of subjects without apparent symptoms of metabolic syndrome, and groups showing mild hypertriglyceridemia with decreased HDL-cholesterol. In addition, serum concentration of the heart fraction of fatty acid binding protein (hFATP) was explored as a factor that might reflect the regulative role of PPAR gamma in lipid homeostasis [29,30], and serum IgG anticardiolipin (ACL-IgG) was investigated as an indirect indicator of oxidized lipid fractions related to atherosclerotic complications [31,32].<br><br>Methods
Subjects
The study was carried out on 70 out-<span class="yellow">patients</span> of the Metabolic Center at the hospital in Sternberk, Czech Republic. From these, 40 <span class="yellow">patients</span> (20 <span class="yellow">men</span> and 20 <span class="yellow">women</span>) were selected with mild hyperlipidemia, i.e. with plasma triglyceride concentration exceeding 2.0 mmol/l, total cholesterol exceeding 6.0 mmol/l, LDL cholesterol exceeding 4.0 mmol/l, and with HDL cholesterol concentration in <span class="yellow">men</span> under 1.0 mmol/l, and in <span class="yellow">women</span> under 1.2 mmol/l. These groups were denominated as "hyperlipemic". Two other groups (10 <span class="yellow">men</span> and 20 <span class="yellow">women</span>) with approximately normal serum values of these variables were taken as "controls". The average age in <span class="yellow">men</span> was 59.1 ± 10.6 y, and in <span class="yellow">women</span> 59.4 ± 10.1 y, respectively. The differences between lipid parameters of hyperlipemic and control groups were highly statistically significant, while the age differences were insignificant (see Table 1). None of the <span class="yellow">patients</span> had clinically apparent diabetes mellitus, but some of the hyperlipemic <span class="yellow">patients</span> exerted impaired glucose tolerance or impaired fasting glucose (values between 6.1 and 7.0 mmol/l, or between 6.1 and 7.8 mmol/l, respectively). None of the <span class="yellow">patients</span> was treated with insulin, peroral antidiabetics or antihyperlipemic drugs; some of them were treated with antihypertensive therapy. No signs of major clinical or laboratory symptoms of other diseases were present in any group of the explored <span class="yellow">patients</span>. Blood samples were obtained in the morning via a venipuncture after overnight fasting. After clotting the serum was separated and stored at -20° until used. An informed consent was obtained from all probands.
Body mass indexes (BMI), defined as weight in kilograms divided by the square of height in meters, were calculated.<br><br>Biochemical methods
Serum leptin concentrations were measured by a sandwich ELISA test kit (<span class="yellow">Human</span> Leptin ELISA, BioVendor Laboratory Medicine, Inc, Czech Republic). Its sensitivity limit was 0.2 ng/ml, intraassay CV 6.1% at the level of 7.5 ng/m, inter-assay CV 8.5% at the level of 4.8 ng/ml. Tetramethylbenzidine was used as a substrate; quality controls were <span class="blue">human</span> based. Several other hormones and peptides were estimated by routine immunochemical tests: insulin, C-peptide, TNFα (IMMULITE, Diagnostic Products Corporation, Los Angeles, CA, U.S.A.), proinsulin intact (DAKO, Denmark), IgG anticardiolipin (ACL-IgG, IMMCO Diagnostics, Buffalo, NY, U.S.A.) and heart fatty acid binding protein (hFABP, Hbt HUMAN H-FABP, HyCult Biotechnology, Uden, the Netherlands). Serum concentration of glucose, total cholesterol, triglycerides, HDL-cholesterol, LDL-cholesterol, Apoprotein B and uric acid were measured on a ILAB-600 biochemical analyzer (Instrumentation Laboratory, Lexington, Ma, U.S.A.) using BioVendor sets. All samples were processed and examined according to principles of good laboratory practice and under constant intralaboratory and external quality control.
The homeostatic indexes of insulin resistance (HOMA IR and QUICKI) were calculated according to the homeostasis model of assessment [33-35] as follows:
HOMA IR = fasting insulin (μU/ml) * fasting glucose (mmol/l) / 22.5;
QUICKI = 1 / [log fasting insulin (μU/ml) + log fasting glucose (mg/100 ml)].<br><br>Statistics
Statistical analysis was performed using the Version 6 SAS/STAT software (SAS Institute, Inc., Cary, NC, U.S.A.). The Shapiro-Wilks tests were used in testing the normality of distribution. Some of the data obtained were not normally distributed. The statistical significance of differences between the means in the hyperlipemic and control groups were evaluated using the unpaired Student's T-test in the case of normal distribution of data sets, and using the Kolmogorov-Smirnov test when at least in one of the data sets the normal distribution was excluded. Spearman's rank-order correlation was used for correlation analysis. Multiple regression analysis was performed using HOMA IR indexes of insulin resistance as dependent variables, and other metabolic and hormonal factors (lipid parameters, BMI, leptin, TNFα, hFABP, ACL-IgG) as independent variables. The so-called step-down regression model was used to select dominant independent variables. Various four-member groups of independent (explanatory) variables were used for the analysis and the non-zero intercept was taken into account. The independent variables were then dropped, one at a time; at each stage one variable making the least contribution to the dependent variable (i.e. that showed the least p-value in the test of the regression coefficient being zero) was excluded. The coefficient of determination R2, which can be viewed as a percentage explaining the total variance, was simultaneously monitored. A great drop in R2 after excluding some independent variable enabled selection of those independent variables that could be thought to be the most important determinants of the dependent variable.<br><br>
Results
Table 1 demonstrates mean parameters in individual groups of subjects matched according to sex, lipid parameters and age. While the age of all four groups did not differ substantially, the concentrations of total serum cholesterol, triglycerides, HDL-cholesterol and LDL-cholesterol differ very significantly in both male and female hyperlipemic groups as compared with controls. In addition, the concentration of triglycerides in control <span class="yellow">women</span> was significantly higher than in control <span class="yellow">men</span>, the concentration of triglycerides in hyperlipemic <span class="yellow">women</span> was lower than in hyperlipemic <span class="yellow">men</span>, and the concentration of HDL-cholesterol in hyperlipemic <span class="yellow">women</span> was very significantly higher when compared with hyperlipemic <span class="yellow">men</span>.
Table 2 shows the values of other metabolic and insulin parameters, factors related to insulin resistance and indexes of insulin resistance, respectively. Body mass indexes and uric acid concentration were significantly higher in hyperlipemic <span class="yellow">men</span> as compared to controls, but not in hyperlipemic <span class="yellow">women</span>. Uric acid concentration was substantially lower in hyperlipemic <span class="yellow">women</span> than in hyperlipemic <span class="yellow">men</span>. Plasma concentrations of glycemia, insulin and intact proinsulin were significantly higher in both hyperlipemic <span class="yellow">men</span> and <span class="yellow">women</span> as compared with controls of identical gender, while the concentration of leptin increased only in hyperlipemic <span class="yellow">men</span>. However, serum leptin concentrations of both control and hyperlipemic <span class="yellow">women</span> were significantly higher than in corresponding groups of <span class="yellow">men</span>. Serum concentrations of TNFα, hFABP and ACL-IgG in hyperlipemic groups of both <span class="yellow">men</span> and <span class="yellow">women</span> were not significantly different from control groups. On the other hand, the indexes of insulin resistance HOMA IR and QUICKI differed very significantly in hyperlipemic groups of both <span class="yellow">men</span> and <span class="yellow">women</span> as compared with corresponding control groups, more distinctly in <span class="yellow">women</span>.
From Fig. 1, presenting 95% confidence limits of insulin resistance indexes HOMA IR and QUICKI, we concluded that in groups of hyperlipemic <span class="yellow">patients</span> of both genders the insulin resistance was substantially higher than in control groups; the groups did not overlap each other.
In Table 3 the results of Spearman's correlations between insulin resistance index HOMA IR and various metabolic parameters are presented. In the control group of <span class="yellow">men</span>, positive significant correlation between HOMA IR and serum leptin concentration, and inverse significant correlation between HOMA IR and ACL IgG, respectively, were found. In the control group of <span class="yellow">women</span>, the significance of Spearman's correlation between HOMA IR and leptin was more expressive; inverse correlation between HOMA IR and HDL-cholesterol was also present.
In the hyperlipemic group of <span class="yellow">men</span>, the significance of the correlation between HOMA IR was more expressive in relation to the control group, and no significant correlation between HOMA IR and ACL IgG was found. In the hyperlipemic group of <span class="yellow">women</span>, however, the significance of Spearman's correlation between HOMA IR and serum leptin concentration weakened, the inverse correlation between HOMA IR and HDL-cholesterol remained approximately unchanged, and a positive correlation between HOMA IR and serum concentration of TNFα appeared.
Table 4 shows results of multiple regression analysis, when data from both control and hyperlipemic groups of each gender were judged together. HOMA IR was considered as a dependent variable and differently changed constellations of metabolic and other factors were taken as independent variables.
In <span class="yellow">men</span>, BMI and leptin seemed to play a main role in influencing the insulin resistance index HOMA IR, while TGL, ACL IgG and LDL-cholesterol didn't play any significant role (see left columns of Table 4). The decreasing of HDL-cholesterol concentration may also have some influence (see a significant drop of R2 after exclusion of this factor in Table 4A, 4B). But in the presence of leptin in the group of independent factors, the drop of R2 after exclusion of HDL-cholesterol from these factors was minimal (see Table 4C). On the other hand, after the exclusion of TNFα from the group of independent variables (see Table 4B, 4D) the value of R2 has unexpectedly risen, which could reflect the interference of TNFα with factors increasing the insulin resistance.
In <span class="yellow">women</span> (see right columns of Table 4), the maximal values of R2 were achieved with combination of independent variables containing TGL, leptin and HDL-cholesterol (about 60% influence on HOMA IR! – see Table 4A, 4B, 4C). TNFα seemed to play quite a different role than in <span class="yellow">men</span>: after exclusion of this factor from the group of independent factors R2significantly decreased (see Table 4B, 4D). In contrast to <span class="yellow">men</span>, the role of BMI seemed to be minimal. As in <span class="yellow">men</span>, the role of ACL IgG and LDL-cholesterol in influencing HOMA IR was negligible, but in contrast to <span class="yellow">men</span>, hFABP might play a certain role in this process (see Table 4D).
Generally, the insulin resistance (represented by HOMA IR) was in <span class="yellow">men</span> much less influenced by metabolic variables than in <span class="yellow">women</span>; while in <span class="yellow">women</span> in some combinations of dependent variables R2 reached 64 %, in <span class="yellow">men</span> the maximal value of R2 was only 39 %.<br><br>Discussion
In our previous paper [36], the mean value of HOMA IR in healthy subjects of both genders and of age comparable with our controls was 1.57 ± 0.87, and the mean value of index QUICKI 0.366 ± 0.029, respectively. These values, as well as the 95 % confidence limits, correspond to values found in controls in this study.
In accordance with many previous papers, serum concentrations of leptin in <span class="yellow">women</span> (both control and hyperlipemic) were substantially higher than in <span class="yellow">men</span>. In the control group of <span class="yellow">women</span> the correlation between leptin and HOMA IR was highly significant. However, in hyperlipemic <span class="yellow">women</span> the significance of this correlation lessened, because HOMA IR increased considerably (and significantly) but serum concentration of leptin only slightly (insignificantly). In <span class="yellow">men</span> the significance of correlations between serum leptin and HOMA IR was high and approximately the same in both the control and hyperlipemic groups, because the values of HOMA IR as well as serum leptin have nearly doubled in hyperlipemic in relation to control groups. In non-hyperlipemic postmenopausal <span class="yellow">women</span> the high concentration of serum leptin was not associated with higher insulin resistance: HOMA IR did not differ substantially from <span class="yellow">men</span>. A significant increase of insulin resistance in hyperlipemic <span class="yellow">women</span> was associated by only slight and insignificant increase of leptin concentration. According to Spearman's correlations, an increase of serum TNFα and/or a decrease of HDL-cholesterol might also play a distinct role in this respect. (see Table 3). In contrast to <span class="yellow">women</span>, in hyperlipemic <span class="yellow">men</span> the increase of insulin resistance index was approximately proportional with the increase of leptin concentration.
Multiple regression analysis affirmed the importance of leptin serum in increasing of insulin resistance in both genders. In <span class="yellow">men</span>, only BMI and HDL-cholesterol from other factors studied seemed to play a certain role, but the maximal values of influencing HOMA IR reached only 39%, with leptin and BMI being the more important factors. On the other hand, in <span class="yellow">women</span> the maximal determination of HOMA IR as high as 60% was registered in combination of serum leptin, TGL and decreased HDL-cholesterol as independent factors; the role of BMI was insignificant.
It is not known how leptin is regulated. A strong correlation between plasma leptin and fasting insulin undoubtedly exists, but hyperleptinemia in both obese and lean <span class="yellow">humans</span> is not likely the result of hyperinsulinemia [37]. A relationship between leptin and insulin dependent on sex or BMI was reported, but relationship between triglyceride concentrations and leptin was independent of sex, BMI, and insulin [18,24]. Hyperleptinemia, as an early sign of obesity, was closely linked to subcutaneous fat mass [39,40]. Percentage of body fat has been shown to be the strongest predictor of leptin levels even in lean <span class="yellow">women</span> [41]. Leptin was highly correlated with percentage of body fat and with fat mass in adults irrespective of gender and age; however, the mean determinant of leptin plasma concentration in <span class="yellow">men</span> and postmenopausal <span class="yellow">women</span> was BMI, while in premenopausal <span class="yellow">women</span> it was only the fat mass [42]. These findings contrast with our results showing minimal influence of BMI on HOMA IR in postmenopausal <span class="yellow">women</span>.
All factors mentioned are connected with fat tissue: leptin and TNFα are directly produced chiefly by adipocytes, BMI growth is obviously accompanied by fat mass increase, and the typical hypertriglyceridemia associated with a decrease of HDL-cholesterol goes along with obesity and fat mass growth. The gender differences in circulating leptin were best explained by percentage of body fat and – inversely – by lean body mass [25]. In both genders the intra-abdominal fat correlated with insulin resistance, while the subcutaneous fat correlated with circulating leptin [11,12]. In <span class="yellow">men</span> obesity led to a prevalent increase of intra-abdominal fat, while in <span class="yellow">women</span> of subcutaneous fat [13]. Influences of different compartments of adipose tissues could elucidate the variability of correlations between insulin resistance and high leptin concentrations in lean and obese subjects of both genders [43]. In our non-hyperlipemic postmenopausal <span class="yellow">women</span> the content of subcutaneous fat mass might be higher than in non-hyperlipemic <span class="yellow">men</span> of appropriate age, which indicated a higher serum concentration of leptin. However, the insulin resistance – related to intra-abdominal fat mass – did not differ from <span class="yellow">men</span>. The significant increase of insulin resistance and leptin concentration in hyperlipemic <span class="yellow">men</span> might reflect the growing content of both subcutaneous and intra-abdominal fat mass (see the significant increase of BMI). In hyperlipemic <span class="yellow">women</span> the significant increase of insulin resistance accompanying only minimal insignificant increase of leptin could be caused by prevalent growing of intra-abdominal fat mass.
In elderly postmenopausal <span class="yellow">women</span>, an association between leptin and plasma lipoprotein concentration was found which depended on adiposity [17], and inverse correlations between serum leptin and HDL-cholesterol were described [44]. In our study, insulin resistance in <span class="yellow">women</span> seemed to be more notably than in <span class="yellow">men</span> influenced by lipid disorders, i.e. positively by serum triglycerides and inversely by HDL-cholesterol. These findings might be important in considering the concept of treatment of insulin resistance-related disorders in postmenopausal <span class="yellow">women</span>.
The significant role of TNFα in insulin resistance, caused by inhibiting the transduction of insulin signaling and by down-regulation of glucose transporter GLUT-4 and insulin receptor substrate-1, has been repeatedly confirmed [45-48]. Our results supported these findings unambiguously only in <span class="yellow">women</span>, while in <span class="yellow">men</span> TNFα seemed paradoxically to interfere with other factors – mainly BMI and leptin – in influencing insulin resistance, thus playing a quite different role. Previously it was found [46] that correlation between serum TNFα on the one side, and insulin, HOMA IR, serum triglycerids, respectively, on the other side, was substantially more significant in <span class="yellow">women</span> than in <span class="yellow">men</span>. Serum concentration of TNFα in <span class="yellow">patients</span> with type 2 diabetes of both genders correlated only with the quantity of intra-abdominal fat compartment [50]. Visceral obesity correlated with plasmatic aldosterone and with insulin resistance only in premenopausal <span class="yellow">women</span>, but not in <span class="yellow">men</span> [51].
From all these data we might support our above mentioned conclusion – that rising of insulin resistance in hyperlipemic <span class="yellow">women</span> was associated with an increase of intra-abdominal fat, because this fat mass in particular is a source of TNFα, which interfered with insulin sensitivity only in <span class="yellow">women</span>. We came to this conclusion irrespective of the finding that the increase of serum TNFα in hyperlipemic <span class="yellow">women</span> was statistically insignificant; results of Spearman's correlation (Table 3) and multiple regression analysis confirm a distinct role of this factor. In hyperlipemic <span class="yellow">men</span> not only the serum concentration of TNFα has decreased instead of increasing, but according to multiple regression analysis it played a quite different role in influencing insulin sensitivity, interfering with factors that determined insulin resistance (leptin and BMI).
In the control group of <span class="yellow">men</span> IgG anticardiolipin was inversely correlated to HOMA IR. The significance of this finding is not clear. These antibodies indicate vascular and thrombotic complications and oxidative modification of lipoproteins [52,53] and may represent an increased risk of atherogenic and inflammatory complications. In this case, however, their growing might be connected with an increase in insulin sensitivity. Anyway, ACL IgG evidently did not participate significantly in influencing the increase of insulin resistance associated with hyperlipidemia, although other anti-cardiolipin correlations could be masked by the relatively large inter-individual variations in this parameter.
Neither serum concentration of hFABP, a factor ensuring transmembrane transport and oxidative metabolisation of long-chain fatty acids [54,55], was significantly changed in hyperlipemic and insulin resistant subjects of both genders. This factor was very weakly associated only with HOMA IR in <span class="yellow">women</span> (see Table 4D), indicating that enhanced metabolisation of fatty acids in cells might to some degree contribute to insulin resistance.<br><br>Conclusions
In postmenopausal <span class="yellow">women</span> as well as in <span class="yellow">men</span> of approximately equal age serum leptin plays a significant role as an important determinant of insulin resistance. In addition to this factor, in <span class="yellow">women</span> the grade of insulin resistance is very considerably influenced by serum triglycerides, tumor necrosis factor alpha, and by decreased concentration of HDL-cholesterol, while in <span class="yellow">men</span> only a mild influence of BMI and decreased HDL-cholesterol is observed. These findings are explained as a consequence of gender-related differences in adipose tissue composition and/or distribution in both normal-weight and over-weight subjects and should be taken into account in treatment of <span class="yellow">patients</span> with metabolic risk factors of cardiovascular diseases.<br><br>List of abbreviations
HDL-Cholesterol = high-density cholesterol
LDL-cholesterol = low-density cholesterol
HOMA IR = Homeostasis Assessment of Insulin Resistance
= fasting insulin (μU/ml) * fasting glucose (mmol/l) / 22,5
QUICKI = 1 / [log fasting insulin (μU/ml) + log fasting glucose (mg/100 ml) ]
TNFα = tumor necrosis factor alpha
BMI = body mass index
R2 = coefficient of determination
hFABP = heart fatty acid binding protein
ACL-IgG = IgG fraction of anticardiolipin
TGL = triglycerides
GLUT-4 = glucose transporter-4
PPARγ = Peroxisome Proliferator-Associated Receptor gamma
CV = coefficient of variation<br><br>Authors' contributions
Dr. Radka Lichnovská collected the clinical material, performed analysis of biochemical values and edited the manuscript.
Dr. Simona Gwozdziewiczová performed analysis of clinical and biochemical data and edited the manuscript.
Prof. Jirí Hrebícek initiated the study, participated in its design and coordination, and wrote and edited the manuscript.<br><br>
<h3>pmcA1779435</h3>Presence of antibodies against cyclic citrullinated peptides in <span class="yellow">patients</span> with 'rhupus': a cross-sectional study
Abstract
'Rhupus' is a rare condition sharing features of rheumatoid arthritis (RA) and systemic lupus erythematosus (SLE). If rhupus is a distinctive entity, an overlap between RA and SLE or a subset of SLE is currently debated. This study was performed to explore the prevalence of antibodies against cyclic citrullinated peptides (anti-CCP antibodies) in rhupus. <span class="yellow">Patients</span> meeting American College of Rheumatology criteria for RA, SLE, or both were included. Clinical and radiographic features were recorded and sera were searched for anti-CCP antibodies, rheumatoid factor, antinuclear antibodies, anti-extractable nuclear antigens, and antibodies against double-stranded DNA (anti-dsDNA antibodies). Seven <span class="yellow">patients</span> for each group were included. Clinical and serological features for RA or SLE were similar between rhupus and RA <span class="yellow">patients</span>, and between rhupus and SLE <span class="yellow">patients</span>, respectively. Values for anti-CCP antibodies obtained were significantly (p < 0.05) higher in RA (6/7) and rhupus (4/7) than in SLE <span class="yellow">patients</span> (0/7) and healthy subjects (0/7). Our data support the possibility that rhupus is an overlap between RA and SLE, because highly specific autoantibodies for RA (anti-CCP) and for SLE (anti-dsDNA and anti-Sm) are detected in coexistence.<br><br>Introduction
The clinical coexistence of rheumatoid arthritis (RA) and systemic lupus erythematosus (SLE) was first described in 1969 by Kantor and was termed 'rhupus syndrome' by Schur (both cited in [1]). Since then, fewer than 100 cases of rhupus have been published [1-3]. In an epidemiological study including about 7,000 new <span class="yellow">patients</span>, the prevalence of RA was 15% and for SLE it was 8.9%. The expected coincidence of RA and SLE by chance would therefore be 1.2%. However, the observed prevalence of rhupus was 0.09%, less than one-tenth of that expected [1].
Previous reports have shown that the <span class="yellow">patients</span> with rhupus display an array of autoantibodies including anti-double-stranded DNA (anti-dsDNA), anti-Sm (both highly specific for SLE), anti-SSA, anti-SSB, anti-ribonucleoprotein, antinuclear antibodies (ANA), anti-cardiolipins, and rheumatoid factor (RF) [1,2]. However, no study has yet been performed to investigate the presence of antibodies against cyclic citrullinated peptides (anti-CCP antibodies), which have a specificity for RA of 96 to 98% (for second-generation assays (anti-CCP2)) [4,5]. Recent data have confirmed that these antibodies are rarely if ever present in other autoimmune diseases such as SLE, Sjögren's syndrome (SS), scleroderma and myositis [6]. Nowadays, it is a matter of debate whether rhupus is a clinically and immunologically distinctive entity [2], a true overlap between SLE and RA [7], or a subgroup of <span class="yellow">patients</span> with lupus [8].
This descriptive, cross-sectional study was performed to investigate the frequency of anti-CCP antibodies in a cohort of <span class="yellow">patients</span> with rhupus.<br><br>Materials and methods
We included all <span class="yellow">patients</span> fulfilling American College of Rheumatology (ACR) classification criteria for both RA [9] and SLE [10] who belonged to our cohorts of <span class="yellow">patients</span> with RA and with SLE. Comparisons were made with age- and gender-matched <span class="yellow">patients</span> with RA and with SLE, and healthy subjects. The study was approved by the local ethics committee, and informed consent was obtained. Serum samples were obtained and stored at -75°C until use. Sera were analyzed for anti-CCP2 antibodies by ELISA (Inova Diagnostics, San Diego, CA, USA) with a cutoff value of 60 U/ml. Fine antinuclear reactivities (ELISA; Inova Diagnostics), RF (nephelometry), ANA (indirect immunofluorescence on HEp-2 slides), and anti-dsDNA (indirect immunofluorescence on Crithidia luciliae substrate) antibodies were also determined. Except for healthy individuals, standard radiographs of hands were available. For statistical analysis, ANOVA and the Mann–Whitney U test were performed as appropriate with GraphPad Prism 4.0 software (GraphPad Inc, San Diego, CA, USA).<br><br>Results
Seven female <span class="yellow">patients</span> with a median age of 44 years (range 25 to 64) met our inclusion criteria. The major clinical and laboratory findings are presented in Table 1. Healthy individuals and all <span class="yellow">patients</span> belonged to cohorts from the same ethnic group (Hispanic mestizo). No differences in demographic data were found between groups.
Mean ACR criteria for SLE were 5.57 (range 4 to 8) in the SLE group, and 5.57 (4 to 8) in the rhupus group. In the same way, mean ACR criteria for RA were 6 (4 to 7) in the RA group, and 5.14 (4 to 6) for the <span class="yellow">patients</span> with rhupus. In all <span class="yellow">patients</span> with rhupus, RA was presented as the initial disease, as has been described previously [2]. In accordance with another report, in two <span class="yellow">patients</span> the disease started during their childhood as juvenile chronic arthritis [1].
Anti-CCP antibodies were found in four of seven (57%) <span class="yellow">patients</span> with rhupus, and in six of seven (86%) <span class="yellow">patients</span> with RA, whereas neither <span class="yellow">patients</span> with SLE nor healthy individuals showed reactivity. When the concentrations in each group were compared, a statistical significant difference between groups was found (ANOVA, p < 0.05). The mean concentration of anti-CCP antibodies was 584 U/ml (range 0 to 1,237) in <span class="yellow">patients</span> with rhupus (Figure 1), 875 U/ml (0 to 1,178) in the RA group (not significant compared with rhupus), 1 U/ml (0 to 10) for SLE individuals (p < 0.05 compared with rhupus), and 0 U/ml (0 to 2) for healthy controls (p < 0.05 compared with rhupus). Two of three <span class="yellow">patients</span> with rhupus who were negative for anti-CCP antibodies were also negative for anti-dsDNA antibodies, RF and anti-extractable nuclear antigen antibodies, although both <span class="yellow">patients</span> met RA and SLE classification criteria, including ANA and erosive arthritis.
Differences in ANA, anti-dsDNA and anti-extractable nuclear antigen autoantibodies between <span class="yellow">patients</span> with rhupus and those with SLE were not found. We also found no difference in the prevalence of RF between <span class="yellow">patients</span> with rhupus and those with RA. Surprisingly, one healthy subject was positive for RF, ANA and anti-SSA antibodies, although she was asymptomatic and no features of any disease were found.<br><br>Discussion
The close association between different type II <span class="blue">human</span> leukocyte antigen (HLA) molecules and the risk of RA is well established. These major histocompatibility complex (MHC) class II molecules share the same amino acid sequence (QKRAA or QRRAA) in positions 69 to 74 of the β-chain, namely the 'shared epitope'. Recent works have demonstrated that this 'shared epitope' preferentially binds peptides containing the non-standard amino acid citrulline (deiminated arginine) [11]. In addition, an abnormally increased function of the enzyme peptidylarginine deiminase 4 (PAD4; responsible for the deimination of arginine) and an elevated anti-CCP autoantibody production in <span class="yellow">patients</span> with RA have been demonstrated [12]. These facts have built the first bridge between cellular and humoral autoimmunity in a major rheumatic disease, supporting a pathogenetic role for an abnormal metabolism of citrulline in the development of RA [13,14].
<span class="yellow">Patients</span> with SLE are often part of the control group when determining the specificity of anti-CCP antibodies for RA [15], although some studies have been performed specifically on <span class="yellow">patients</span> with SLE. These studies contribute some clues to the role of anti-CCP antibodies in rhupus. Mediwake and colleagues [16], in a study exploring the predictive value of anti-CCP antibodies to distinguish erosive arthritis in SLE, found ten <span class="yellow">patients</span> (out of 231) with erosive arthritis, two of whom had anti-CCP antibodies. In concord with this, Hoffman and colleagues [15] demonstrate that three <span class="yellow">patients</span> with erosive arthritis, included in a cohort of 235 <span class="yellow">patients</span> with SLE, were positive for anti-CCP antibodies. These authors suggest that the presence of anti-CCP antibodies can predispose for a chronic RA-like arthritis in <span class="yellow">patients</span> with SLE. Additionally Weissman and colleagues [17] demonstrated that <span class="yellow">patients</span> with SLE can display radiographic abnormalities similar to those of RA, although the presence of marginal erosions is a rare finding.
In the present study we demonstrate that the <span class="yellow">patients</span> with rhupus show a very similar arthritis pattern (including erosive disease) and similar autoantibody production (RF and anti-CCP antibodies) to those in <span class="yellow">patients</span> with RA. In addition, <span class="yellow">patients</span> with rhupus display a clinical and serological profile indistinguishable from <span class="yellow">patients</span> with SLE. Moreover, the presence of other coexistent autoimmune diseases was similar in all groups of <span class="yellow">patients</span> (two <span class="yellow">patients</span> with rhupus, three <span class="yellow">patients</span> with RA, and three <span class="yellow">patients</span> with SLE also had SS).
We found high titers of anti-CCP antibodies in four of seven (57%) <span class="yellow">patients</span> with rhupus, a frequency similar to that reported for RA [4]. This finding, together with the clinical similarity, supports the contention that rhupus belongs to the RA spectrum. The high prevalence of anti-CCP antibodies in RA found in our study could be explained by a selection bias because only <span class="yellow">patients</span> with RA with an aggressive disease (namely erosive arthritis and RF+) were included. In contrast, the mean ACR criterion for SLE was similar between <span class="yellow">patients</span> with rhupus and those with SLE, including the 'robust' features of SLE such as renal and neurological involvement, and anti-dsDNA and anti-Sm antibodies. These clinical and serological features shared between <span class="yellow">patients</span> with rhupus and those with SLE also place rhupus in the SLE spectrum.
Titration of anti-CCP antibodies in the rhupus group clearly shows a bimodal distribution, suggesting the existence of two different subpopulations. Because of the small number of <span class="yellow">patients</span>, we are unable to define the differential features underlying each subset. However, two of three <span class="yellow">patients</span> negative for anti-CCP antibodies were also negative for both RF and anti-dsDNA antibodies.<br><br>Conclusion
On the basis of the presence of shared clinical features of RA (mainly erosive arthritis) and SLE (including renal and neurological involvement) along with the presence of anti-dsDNA and anti-CCP autoantibodies in our <span class="yellow">patients</span> with rhupus, our findings strongly support the contention that rhupus is a true overlap between RA and SLE, not merely a part of the clinical spectrum of the articular involvement seen in SLE. Moreover, on the basis of the mean ACR criteria for both diseases, we have confirmed that <span class="yellow">patients</span> with rhupus have more RA-associated and less SLE-associated damage, an issue that has been suggested previously [2].
To our knowledge, this is the first report exploring the prevalence of anti-CCP antibodies specifically in <span class="yellow">patients</span> with rhupus. More studies are needed to expand the pathogenetic knowledge of this overlap syndrome.<br><br>Abbreviations
ANA = antinuclear antibodies; anti-CCP antibodies = antibodies against cyclic citrullinated peptides; anti-dsDNA antibodies = antibodies against double-stranded DNA; ELISA = enzyme-linked immunosorbent assay; RA = rheumatoid arthritis; RF = rheumatoid factor; SLE = systemic lupus erythematosus; SS = Sjögren's syndrome.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
LA-G participated in the conception and design of the experiments, in the acquisition, analysis and interpretation of data, and was involved in drafting the manuscript. RS performed the immunoassays. RM-V participated in the analysis and interpretation of data and performed the statistical analysis. LG-G participated in the analysis and interpretation of data. AV participated in the recruitment of <span class="yellow">patients</span> and the acquisition of data. RB participated in the interpretation of data, revising the manuscript for intellectual content and giving the final approval of the version to be published. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1779441</h3>Identification of a <span class="blue">human</span> peripheral blood monocyte subset that differentiates into osteoclasts
Abstract
Increased bone resorption mediated by osteoclasts causes various diseases such as osteoporosis and bone erosion in rheumatoid arthritis (RA). Osteoclasts are derived from the monocyte/macrophage lineage, but the precise origin remains unclear. In the present study, we show that the purified CD16- <span class="blue">human</span> peripheral blood monocyte subset, but not the CD16+ monocyte subset, differentiates into osteoclast by stimulation with receptor activator of NF-κB ligand (RANKL) in combination with macrophage colony-stimulating factor (M-CSF). Integrin-β3 mRNA and the integrin-αvβ3 heterodimer were only expressed on CD16- monocytes, when they were stimulated with RANKL + M-CSF. Downregulation of β3-subunit expression by small interfering RNA targeting β3 abrogated osteoclastogenesis from the CD16- monocyte subset. In contrast, the CD16+ monocyte subset expressed larger amounts of tumor necrosis factor alpha and IL-6 than the CD16- subset, which was further enhanced by RANKL stimulation. Examination of RA synovial tissue showed accumulation of both CD16+ and CD16- macrophages. Our results suggest that peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct responses to RANKL. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for osteoclastogenesis. Blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Introduction
Rheumatoid arthritis (RA) is an autoimmune disease characterized by chronic inflammation and proliferation of the synovium in multiple joints. A large number of inflammatory cells, including T cells, B cells, macrophages and dendritic cells, accumulate in the affected synovium, and these inflammatory cells, together with fibroblast-like synoviocytes, express various cytokines, such as tumor necrosis factor alpha (TNFα), IL-6 and receptor activator of NF-κB ligand (RANKL), which are known to induce differentiation and activation of osteoclasts. The inflammatory synovial tissue, known as pannus, invades the articular bone and causes focal bone erosion, which is the hallmark of RA. Histopathologically, osteoclasts are present at the interface of the pannus and bone. Interestingly, the deletion of RANKL or c-Fos gene, which is important for osteoclastogenesis, results in minimal bone destruction in <span class="yellow">mouse</span> models of arthritis [1,2]. Furthermore, other studies indicated that inhibition of osteoclastogenesis by osteoprotegerin, a decoy receptor for RANKL, limits bone destruction in experimental models of arthritis. These studies suggest that osteoclasts are involved in focal bone erosion in RA [3].
Osteoclasts are derived from the monocyte/macrophage lineage. It is reported that osteoclast precursors reside in <span class="blue">human</span> peripheral blood monocytes [4,5]. A marked increase of the circulating osteoclast precursors was demonstrated in <span class="yellow">patients</span> with erosive psoriatic arthritis as well as in arthritic TNFα transgenic <span class="yellow">mice</span> [6,7]. It was also shown that peripheral monocytes differentiate into osteoclasts when seeded on RANKL/osteoclast differentiation factor-producing RA synovial fibroblasts [8]. In addition, RA synovial macrophages thought to originate from peripheral blood monocytes were shown to differentiate into osteoclasts [9,10]. Monocytes are therefore involved not only in synovial inflammation, but also in bone remodeling as potential precursors for synovial macrophages and osteoclasts.
<span class="yellow">Human</span> peripheral blood monocytes consist of two major subsets, CD16+ and CD16-, comprising 5–10% and 90–95% of the monocytes, respectively. These two subsets exhibit different chemotaxis activities and potential of cytokine production [11,12]. Moreover, activation of the Toll-like receptor induces distinct subsets, CD1b+ dendritic cells and DC-SIGN+ (dendritic cell-specific C-type lectin ICAM-3-grabbing nonintegrin) macrophages from CD16+ and CD16- monocytes, respectively [13]. It has not been revealed, however, which monocyte subset develops into osteoclasts.
In the present study, we determined the <span class="blue">human</span> peripheral blood monocyte subset that differentiates into osteoclasts, and revealed that each subset exhibits a different response for osteoclastogenic stimuli.<br><br>Materials and methods
Purification of peripheral blood monocytes
Peripheral blood monocytes from healthy donors were collected using Ficoll-Conray (Imuuno-Biological Laboratories, Gunma, Japan) gradient centrifugation. Negative selection of monocytes was performed using MACS microbeads (Miltenyi Biotec, Auburn, CA, USA) according to the protocol supplied by the manufacturer.
The purified monocytes were separated into two subsets, CD16+ and CD16- monocytes, using CD16 MicroBeads (Miltenyi Biotec). Flow cytometry analysis using FITC-conjugated <span class="yellow">mouse</span> anti-CD14 mAb (MY4; Bechman Coulter, Fullerton, CA, USA) and phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8; BD Biosciences, San Jose, CA, USA) showed that the purities of the CD16+ and CD16- monocytes were more than 90% and 92%, respectively.
For the other experiment, monocytes were purified using CD14 MicroBeads (Miltenyi Biotec), and then stained either with FITC-conjugated <span class="yellow">mouse</span> anti-CD33 mAb (MY9; Bechman Coulter) or phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8). Cell sorting of the stained cells was performed using a FACS Vantage cytometer (BD Biosciences) or a MoFlo cell sorter (Dako, Glostrup, Denmark).<br><br>Osteoclast differentiation
Purified CD16+ and CD16- monocytes (5 × 104 cells/well) were incubated in 96-well plates in αMEM (Sigma, St Louis, MO, USA) with heat-inactivated 10% fetal <span class="yellow">bovine</span> serum (FBS) (Sigma) or with Ultra-Low IgG FBS (IgG < 5 μg/ml; Invitrogen, Carlsbad, CA, USA), and where indicated with M-CSF + RANKL (Peprotech, Rocky Hill, NJ, USA).
For the other experiments, varied numbers of CD16+ monocytes (1 × 103, 2.5 × 103, 5 × 103) were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured in 96-well plates in αMEM with heat-inactivated 10% FBS. The medium was replaced with fresh medium 3 days later, and after incubation for 7 days the cells were stained for tartrate-resistant acid phosphatase (TRAP) expression using a commercial kit (Hokudo, Sapporo, Japan). The number of TRAP-positive multinucleated cells (MNC) in three randomly selected fields examined at 100× magnification or the total number of TRAP-positive MNC per well was counted under light microscopy.<br><br>Resorption assay
Monocytes were seeded onto plates coated with calcium phosphate thin films (Biocoat Osteologic; BD Biosciences) and were incubated with RANKL (40 ng/ml) + M-CSF (25 ng/ml) for 7 days. The cells were then lysed in bleach solution (6% NaOCl, 5.2% NaCl). The resorption lacunae were examined under light microscopy.<br><br>Enzyme-linked immunosorbent assay
Purified monocytes were cultured in 96-well plates where indicated either with RANKL or M-CSF for 24 hours. Concentrations of TNFα and IL-6 in the culture supernatant were measured with an ELISA kit (BioSourse International, Camarillo, CA, USA). For experiments of matrix metalloproteinase (MMP)-9 and TRAP-5b, culture supernatants were collected on day 7 and the concentrations of these enzymes were measured using an MMP-9 ELISA kit (Amersham Biosciences, Piscataway, NJ, USA) or a TRAP-5b ELISA kit (Suomen, Turku, Finland).<br><br>Reverse transcriptase-polymerase chain reaction
Monocytes (1 × 106 cells/well) were cultured in six-well plates with M-CSF alone or with M-CSF + RANKL for 3 days. Total RNA was extracted using RNeasy Micro (Qiagen, Valencia, CA, USA). The RNA was then treated with DNase I (Qiagen). The oligo(dT)-primed cDNA was synthesized using Superscript II reverse transcriptase (Invitrogen). The amount of cDNA for amplification was adjusted by the amount of RNA measured by an optical density meter and also by β-actin or GAPDH PCR products. One microliter of cDNA was amplified in a 50 μl final volume containing 25 pmol appropriate primer pair, 10 pmol each of the four deoxynucleotide triphosphates, and 5 units FastStart Taq DNA Polymerase (Roche, Manheim, Germany) in a thermal cycler (PTC-200; MJ GeneWorks, Waltham, MA, USA).
The PCR conditions were 25–40 cycles of denaturation (95°C for 30 s), annealing (60–62°C for 1 min) and extension (72°C for 1 min). The sequences of the primers are presented in Table 1. The PCR products were separated by electrophoresis through 2% agarose gel.<br><br>Western immunoblot analysis
Purified monocytes were cultured for 3 days in the presence of 40 ng/ml M-CSF with or without 25 ng/ml RANKL. Cells were lysed in RIPA Lysis buffer (upstate, Lake Placid, NY, USA) containing protease inhibitors (Roche) for 15 min at 4°C. A total of 20 μg protein was boiled in the presence of 6 × sodium dodecyl sulfate sample buffer, and was separated on 7.5% or 10% sodium dodecyl sulfate-polyacrylamide gel (ATTO, Tokyo, Japan). Proteins were then electrotransferred to a polyvinylidene fluoride microporous membrane (Millipore, Billerica, MA, USA) in a semidry system. Membranes were incubated in 10% skim milk prepared in phosphate-buffered saline (PBS) containing 0.1% Tween 20, and were subjected to immunoblotting. Antibodies used were <span class="yellow">goat</span> anti-RANK antibody (Techne Corporation, Minneapolis, MN, USA), <span class="yellow">goat</span> anti-c-fms antibody (R&D systems, Minneapolis, MN, USA), and <span class="yellow">mouse</span> anti-β-actin mAb (AC-15; Sigma). Peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">goat</span> IgG antibody (Dako) or peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">mouse</span> IgG antibody (Dako) was used as the second antibody. The signals were visualized by chemiluminescence reagent (ECL; Amersham Biocsiences, Little Chalfont, UK).<br><br>Cell surface expression of c-fms
The following mAbs were used for analysis of c-fms expression: Alexa 647-conjugated anti-CD14 mAb (UCHM1; Serotec, Oxford, UK), FITC-conjugated anti-CD16 mAb (3G8; Bechman Coulter) and phycoerythin-conjugated anti-c-fms mAb (61708; R&D systems). Alexa 647-conjugated <span class="yellow">mouse</span> IgG2a (Serotec), FITC-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) and phycoerythin-conjugated <span class="yellow">mouse</span> IgG1 (Bechman Coulter) were used as isotype controls. Peripheral blood monocytes (1 × 105 cells) were incubated with 1 μg <span class="blue">human</span> IgG for 15 minutes, and were then stained with three fluorochrome-labeled mAbs for 45 minutes on ice. The stained cells were analyzed with a FACS Calibur (BD Biosciences).<br><br>Immunofluorescent staining
Monocytes (8 × 104 cells/well) were allowed to adhere on 96-well plates overnight or were cultured with M-CSF and RANKL for 2–4 days. The cells were fixed in acetone and then stained with anti-αvβ3 mAb (LM609; Chemicon, Temecula, CA, USA) or <span class="yellow">mouse</span> IgG1 (11711; R&D Systems) as an isotype-matched control. Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes, Eugene, OR, USA) was used as the second antibody. TOTO-3 (Molecular Probes) was used for nuclear staining.<br><br>Flow cytometric analysis of p38 MAPK and ERK1/2 phosphorylation
Purified monocytes were cultured in the presence of 25 ng/ml M-CSF for 3 days, and were either left unstimulated or were stimulated with 40 ng/ml RANKL at 37°C. Stimulations were stopped by adding an equal volume of PhosFlow Fix Buffer I solution (BD Biosciences) to the cell culture. After incubation for 10 minutes at 37°C, the cells were permeabilized by washing twice at room temperature in PhosFlow Perm/Wash Buffer I (BD Biosciences). A total of 1 × 105 cells was then Fc blocked with 1 μg <span class="blue">human</span> IgG for 15 minutes, and was stained with Alexa Fluor 647-conjugated mAb either to phospho-p38 MAPK (T180/Y182) or to phospho-ERK1/2 (T202/Y204) (BD Biosciences) for 30 minutes at room temperature. Alexa Fluor 647-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) was used as an isotype control. The cells were washed in PhosFlow Perm/Wash Buffer I, and were analyzed by flow cytometry, as already described.<br><br>RNA interference
RNA oligonucleotides (iGENE, Tsukuba, Japan) were designed based on the algorithm that incorporates single nucleotide polymorphism and homology screening to ensure a target-specific RNA interference effect. The following sense and antisense oligonucleotides were used: integrin β3, 5'-GCU UCA AUG AGG AAG UGA AGA AGC A-AG and 3'-UA-CGA AGU UAC UCC UUC ACU UCU UCG U; randomized control, 5'-CGA UUC GCU AGA CCG GCU UCA UUG C-AG and 3'-UA-GCU AAG CGA UCU GGC CGA AGU AAC G; and lamin, 5'-GAG GAA CUG GAC UUC CAG AAG AAC A-AG and 3'-UA-CUC CUU GAC CUG AAG GUC UUC UUG U.
CD16- monocytes (8 × 104 cells/well) were incubated in 96-well plates in optimem (Invitrogen). After 1 hour, siRNAs were transfected into the cells using oligofectamine (Qiagen) based on the method recommended by the manufacturer. After 2 hours, the cells were washed once with PBS, followed by the addition of αMEM supplemented with 10% FBS, M-CSF and RANKL. After a 2-day incubation, the β3 mRNA expression was analyzed by RT-PCR with different PCR cycles, as described earlier. Immunofluorescent staining for the αvβ3 heterodimer was also performed as described above, and numbers of αvβ3-positive cells were counted in randomly selected three fields at 100× magnification. Seven days after the transfection of siRNAs, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Inhibition of osteoclastogenesis with cyclic RGDfV peptide
CD16- monocytes were incubated in 96-well plates with M-CSF + RANKL for 2 days. A medium containing either cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) (Calbiochem, San Diego, CA, USA) or dimethyl sulfoxide was then added. After incubation for a further 5 days, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Immunohistochemistry
Synovial tissue samples were obtained during total knee joint replacement surgery from four RA <span class="yellow">patients</span>. Signed consent forms were obtained before the operation. The experimental protocol was approved by the ethics committee of the Tokyo Medical and Dental University. RA was diagnosed according to the American College of Rheumatology criteria [14].
Double immunofluorescent staining for CD68 and CD16 antigens was conducted on optimal cutting temperature-embedded sections of frozen synovial samples. Eight-micrometer-thick cryostat sections of RA synovium were fixed in acetone for 3 minutes and were then rehydrated in PBS for 5 minutes. The samples were incubated in 5 μg/ml proteinase K (Roche), 50 mM ethylenediamine tetraacetic acid, 100 mM Tris–HCl, pH 8.0, for 15 minutes at room temperature followed by a wash in PBS. The samples were then blocked with 10% <span class="yellow">goat</span> serum in PBS for 60 minutes at room temperature, and were incubated with anti-CD16 mAb (3G8; Immunotech, Marseille, France) or <span class="yellow">mouse</span> IgG1 (11711) as an isotype-matched control in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then washed three times in PBS, for 5 minutes each, and incubated with Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes) in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then sequentially stained for CD68 antigen in a manner similar to that used for CD16 staining. The samples were stained with anti-CD68 mAb (PGM1; Immunotech) or <span class="yellow">mouse</span> IgG3 (6A3; MBL, Nagoya, Japan) followed by labeling with Alexa fluor488-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG3 antibody (Molecular Probes). The samples were examined by confocal laser scanning microscope (Olympus, Tokyo, Japan).<br><br>Statistical analysis
Data are expressed as the mean ± standard error of the mean. A nonpaired Student's t test was used for comparison, using the StatView program (Abacus Concepts, Berkeley, CA, USA). P < 0.05 was considered statistically significant.<br><br>
Results
Induction of osteoclasts from CD16- peripheral blood monocytes
To identify the monocyte subset that differentiates into osteoclasts, we examined osteoclast formation from CD16+ and CD16- <span class="blue">human</span> peripheral blood monocytes. The monocyte subsets were purified using magnetic beads. Incubation with M-CSF alone did not induce osteoclast formation from either subset (Figure 1a). Culture with M-CSF + RANKL induced a significant number of TRAP-positive MNC from the CD16- subset, whereas only few CD16+ monocytes differentiated into TRAP-positive MNC (Figure 1a,b). We then assessed the bone resorptive ability by culturing cells on calcium phosphate-coated plates with M-CSF + RANKL. Resorption lacunae were detected only in the CD16- subset (Figure 1c), indicating the TRAP-positive CD16--derived MNC possessed the osteoclast phenotype.
Similar results were obtained using purified monocytes by FACS sorting (purities: CD16+, 96%; CD16-, 97%). The number of TRAP-positive MNC induced were 36 ± 3 cells/well and 348 ± 13 cells/well from CD16+ and CD16- monocytes, respectively. To exclude the possibility that the anti-CD16 antibody used for isolation of CD16+ monocytes inhibits osteoclast formation, we separated the two subsets, CD33low monocytes and CD33high monocytes, using anti-CD33 mAb and a fluorescent cell sorter, since it was reported that CD33low monocytes correspond to CD16+, and that CD33high correspond to CD16- monocytes [15]. On average, the CD33low population contained CD16- (10.2%)/CD16+ (89.8%) monocytes, and the CD33high population contained CD16- (86.3%)/CD16+ (13.7%) monocytes.
Culture with M-CSF + RANKL induced TRAP-positive MNC from CD33high monocytes, whereas no or few CD33low monocytes differentiated into TRAP-positive MNC (CD33low vs CD33high, 2 ± 1 vs 192 ± 71 cells/well; n = 3). TRAP-5b and MMP-9 in the culture supernatants, both of which are known to be produced by osteoclasts, were measured by ELISA. The concentrations of both enzymes were significantly higher in the culture supernatant of CD16- monocytes than in that of CD16+ monocytes (Figure 1d). These results suggest that the CD16- peripheral blood monocyte subset, but not the CD16+ subset, differentiate into osteoclasts by incubation with RANKL + M-CSF.<br><br>CD16+ monocytes do not affect the osteoclastogenesis from CD16- monocytes
To examine whether CD16+ monocytes affect osteoclastogenesis from CD16- monocytes, varied numbers of CD16+ monocytes were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured for 7 days in the presence of M-CSF + RANKL. The number of TRAP-positive MNC was not altered by the presence of CD16+ monocytes (Figure 2). The results indicated that CD16+ monocytes did not hamper or enhance the osteoclastogenesis from CD16- monocytes.<br><br>Differences in cytokine production by RANKL-stimulated or M-CSF-stimulated CD16+ and CD16- monocytes
To compare the biological response of CD16+ and CD16- subsets with either RANKL or M-CSF stimulation, we measured the amount of TNFα and IL-6 production after exposure of cells to various concentrations of RANKL or M-CSF with an ELISA. Without RANKL the CD16+ subset produced a significant amount of TNFα and IL-6, whereas the CD16- subset produced undetectable levels (Figure 3a). RANKL stimulation increased TNFα production from both subsets in a dose-dependent manner, although the CD16+ subset produced more TNFα than did the CD16- subset. RANKL stimulation also enhanced IL-6 production from the CD16+ subset, but not from the CD16- subset. M-CSF stimulation increased TNFα and IL-6 production from both subsets, although the CD16+ subset produced more than the CD16- subset (Figure 3b).
These results suggest that CD16+ monocytes also respond both to RANKL and M-CSF stimulation, although such stimulation does not result in differentiation into osteoclasts. CD16+ monocytes were also noted to express higher amounts of inflammatory cytokines compared with CD16- monocytes with or without RANKL or M-CSF stimulation.<br><br>Comparison of expression levels of molecules involved in osteoclastogenesis between CD16+ and CD16- monocytes
Diverse molecules are involved in RANKL/RANK and its costimulatory signal transduction pathways [16]. The different response to RANKL + M-CSF stimulation between the CD16+ monocyte subset and the CD16- monocytes subset might be explained by the expression profiles of these molecules. We therefore examined the mRNA levels of the following molecules: receptor activator of NF-κB (RANK), the receptor for RANKL; c-fms, the receptor for M-CSF; tumor necrosis factor receptor-associated factor 6 (TRAF-6), the adaptor protein for RANK; c-Fos and nuclear factor of activated T cells c1 (NFATc1), transcription factors that are essential for osteoclastogenesis; DNAX-activation protein 12 (DAP12) and Fc receptor γ chain (FcRγ), adaptor proteins known to deliver costimulatory signals in RANKL-induced osteoclastogenesis; signal regulatory protein β1 (SIRP-β1), triggering receptor expressed on myeloid cells 2 (TREM-2) and osteoclast-associated receptor (OSCAR), transmembrane receptors that associate with either DAP12 or FcRγ; and αv and β3, integrins known to be expressed as the αvβ3 heterodimer on osteoclasts.
The mRNA levels of RANK, c-fms, TRAF-6, DAP12 and SIRP-β1 under the baseline condition (no stimulation) varied between the donors; however, we did not find consistent differences in the mRNA levels of these molecules between the CD16+ monocyte subset and the CD16- monocyte subset among three to six donors (Figure 4a). The mRNA levels of other molecules, apart from integrin β3, were similar between the two subsets under the no-stimulation condition. Although the mRNA levels of RANK, c-fms, DAP12, FcRγ, TREM-2 and OSCAR increased in response to M-CSF alone or M-CSF + RANKL in both subsets, the expression levels were not significantly different between the two subsets. Expressions of TRAF-6, c-Fos and SIRP-β1 mRNA did not change following stimulation with M-CSF + RANKL. Of note, the expression of NFATc1 mRNA was enhanced by M-CSF + RANKL treatment only in the CD16- subset. Furthermore, expression of integrin αv in both subsets was enhanced by M-CSF with or without RANKL; however, the expression level was greater in the CD16- subset. It was noted that integrin-β3 mRNA was detected only in the CD16- subset and was increased by M-CSF + RANKL stimulation, but not by M-CSF alone. The protein expression of RANK under the baseline condition was weakly detected in both subsets, and the levels were varied between donors by western immunoblotting.
The protein expression of c-fms was weakly detected in unstimulated CD16+ monocytes, but not in CD16- monocytes (Figure 4b). Flow cytometry analysis of c-fms in fresh monocytes, however, showed that both subsets express the molecule on the cell surface (Figure 4c). Expressions of both RANK and c-fms were upregulated by M-CSF alone and by M-CSF + RANKL, and we did not find consistent differences in the protein levels of these molecules between the two monocyte subsets. The profiles of expression levels of molecules involved in RANKL/RANK and its costimulatory pathways are similar between the two subsets, except for NFATc1, integrin αv and integrin β3. We therefore assumed that the distinct induction of NFATc1, integrin αv and integrin β3 in response to RANKL stimulation among the two monocyte subsets might explain the differences in their abilities to differentiate into osteoclasts.<br><br>RANKL stimulation induces αvβ3 expression on CD16- monocytes
The integrin-β3 subunit binds to integrin αv only and is expressed as the heterodimeric protein αvβ3 on monocytes and osteoclasts [17]. We examined the expression of αvβ3 on CD16+ and CD16- monocytes by immunofluorescent staining. Neither unstimulated nor M-CSF-stimulated monocyte subsets expressed αvβ3 (Figure 4d and data not shown). After 48 and 72 hours of treatment with M-CSF + RANKL, αvβ3-positive mononuclear cells were observed in CD16- monocyte cultures but not in CD16+ monocyte cultures. At 96 hours, both αvβ3-positive mononuclear cells and multinucleated cells were present in the CD16- monocyte culture. The results indicated that αvβ3 was selectively expressed on CD16- monocytes in the presence of M-CSF + RANKL, and the expression was revealed before the cells differentiate into typical multinucleated osteoclasts.<br><br>RANKL activates ERK and p38 kinases only in CD16- monocytes
Since ERK and p38 MAPK are essential in RANKL-induced osteoclastogenesis [18-20], we next examined whether these kinases were activated differently in CD16+ monocytes and in CD16- monocytes. Purified monocytes were precultured with 25 ng/ml M-CSF for 3 days to enhance RANK expression, and were then treated with RANKL. The RANKL treatment induced phosphorylation of both ERK and p38 MAPK in CD16- monocytes at 5 minutes postexposure, although the p38 MAPK phosphorylation was weak. Both phosphorylations declined to a basal level within 20 minutes (Figure 5). In contrast, ERK and p38 MAPK were not detectably phosphorylated in CD16+ monocytes with RANKL.<br><br>siRNA targeting integrin β3 inhibits osteoclastogenesis from CD16- monocytes
The integrin-β3 cytoplasmic domain is essential for activation of intracellular signals from αvβ3 heterodimers [17]. We therefore examined the involvement of αvβ3 in RANKL + M-CSF-induced osteoclastogenesis in <span class="blue">human</span> CD16- monocytes using siRNA targeting the integrin-β3 subunit. The integrin-β3 siRNA or control randomized siRNA were transfected into CD16- monocytes. At 48 hours post-transfection, we determined the integrin-β3 mRNA level and αvβ3 heterodimer protein expression. The integrin-β3 mRNA level was reduced in the integrin-β3 siRNA-transfected monocytes compared with control siRNA-transfected monocytes (Figure 6a). The αvβ3 heterodimer expression was evaluated by immunofluorescent staining. The number of αvβ3-positive cells was significantly decreased in integrin-β3 siRNA-transfected monocytes compared with that in control siRNA (Figure 6b).
After 7 days of incubation, the number of TRAP-positive MNC was counted. Transfection with integrin-β3 siRNA significantly reduced the number of TRAP-positive MNC in a dose-dependent manner compared with control siRNA transfection (Figure 6c). In addition, the use of siRNA directed toward a different site of integrin-β3 mRNA also inhibited osteoclast formation from CD16- monocytes (data not shown). On the other hand, siRNA that targeted lamin, which was used as a negative control, did not inhibit the induction of osteoclasts (data not shown). These results indicate the importance of integrin β3 in RANKL-induced osteoclast formation from CD16- peripheral blood monocytes.<br><br>Cyclic RGDfV peptide inhibits the osteoclastogenesis from CD16- monocytes
Integrin αvβ3 recognizes a common tripeptide sequence, RGD (Arg-Gly-Asp), which is present in bone matrix proteins such as vitronectin and fibronectin [21]. Cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) inhibits binding of the RGD-containing molecules to αvβ3 [22]. To investigate the role of ligand binding to the αvβ3 heterodimer in the osteoclastogenesis, we examined whether cyclic RGDfV peptide inhibits the formation of osteoclasts. Cyclic RGDfV peptide significantly reduced the number of TRAP-positive MNC in a dose-dependent manner (Figure 6d). The results imply possible involvement of ligand bindings to αvβ3 in the osteoclastogenesis.<br><br>Knockdown of integrin β3 did not affect the expression of NFATc1 mRNA
In the next step, we determined whether integrin-β3-siRNA-induced inhibition of the osteoclastogenesis reflects downregulation of NFATc1, which is a key transcription factor in osteoclastogenesis [23]. For this purpose, we compared NFATc1 mRNA levels between integrin β3 and control siRNA-transfected monocytes. Interestingly, integrin-β3 knockdown did not alter the NFATc1 mRNA level (Figure 7), suggesting that signal transduction mediated by integrin β3 does not affect the expression of NFATc1.<br><br>Detection of CD16+ and CD16- macrophages in synovium of RA <span class="yellow">patients</span>
RA synovial macrophages are derived from peripheral blood monocytes, and their recruitment into the synovium is facilitated by various adhesion molecules and chemokines [24]. To analyze CD16 expression on synovial macrophages, RA synovial tissues were double-stained for CD16 and a macrophage marker, CD68. CD16-/CD68+ macrophages were widespread in the synovium. Although less frequent, CD16+/CD68+ macrophages were also observed both in the synovial intima and subintima (Figure 8). The presence of two subsets of macrophages, CD16+ and CD16-, in RA synovium indicates that both CD16+ and CD16- peripheral blood monocytes are recruited into the synovium.<br><br>
Discussion
<span class="yellow">Human</span> peripheral blood monocytes are a heterogeneous population, and they are divided into two subsets based on the expression of CD16. The CD16+ and CD16- monocyte subsets show functional differences in migration, cytokine production and differentiation into macrophages or dendritic cells [11-13,15]. We focused on the heterogeneity of the monocytes, and the primary question addressed in this study was which monocyte subset could be the source of osteoclasts. The results demonstrated that CD16- peripheral blood monocytes, but not CD16+ monocytes, differentiated in vitro into osteoclasts by treatment with RANKL + M-CSF.
To investigate the molecular mechanisms of the different response to RANKL and the differentiation into osteoclasts between CD16+ and CD16- monocytes, we examined the expression of molecules known to be involved in osteoclastogenesis. The expression profiles of integrin αv, integrin β3 and NFATc1 were different between the two subsets. Integrin αvβ3 heterodimer was expressed only on RANKL and M-CSF-stimulated CD16- monocytes. It is known that αvβ3 expressed on osteoclasts is important in bone resorption as well as in attachment of osteoclasts to the bone matrix [25].
It was recently reported that bone marrow macrophages of integrin-β3-deficient <span class="yellow">mice</span> could not differentiate into mature osteoclasts in vitro, suggesting that αvβ3 is involved not only in activation, but also in differentiation, of osteoclasts in <span class="yellow">mice</span> [26,27]. The authors also showed that αvβ3 and c-fms share a common intracellular signaling pathway, including the activation of ERK and the induction of c-Fos [27], both of which are essential for osteoclastogenesis [28,29]. In addition, it was reported that echistatin, an αvβ3 antagonist, inhibited osteoclast formation of <span class="yellow">mouse</span> bone marrow cells [30].
In accordance with these reports, our data showed that knockdown of integrin-β3 expression resulted in downregulation of the αvβ3 heterodimer, and abrogated osteoclastogenesis from <span class="blue">human</span> peripheral blood CD16- monocytes. We also showed that blocking of adhesive ligands to bind to αvβ3 by RGDfV peptide inhibited osteoclast formation from CD16- monocytes. Taken together, the process of ligand binding to αvβ3 may be involved in the osteoclastogenesis. Blockade of αvβ3 could therefore be a therapeutically beneficial approach to modulate osteoclastogenesis. Indeed, integrin αvβ3 antagonists effectively treated osteoporosis in <span class="yellow">mice</span>, <span class="yellow">rats</span> and <span class="yellow">humans</span>, and protected bone destruction in <span class="yellow">rat</span> adjuvant-induced arthritis in vivo [31-34]. Of note, it is reported that <span class="yellow">patients</span> with Iraqi-Jewish-type Glanzmann thrombasthenia who are deficient in integrin β3 do not develop osteopetrosis because of the upregulation of α2β1 expression on osteoclasts, although the bone-resorptive ability of the osteoclasts was decreased in vitro [35]. The function of αvβ3 in vivo in osteoclast formation and resorptive function could therefore be partially compensated by other integrins.
Although all the multinucleated osteoclasts expressed αvβ3 (Figure 4d) [36], a small number of M-CSF + RANKL-stimulated mononuclear CD16- monocytes expressed αvβ3 (Figure 4d). Multinucleated osteoclasts are formed by fusion of osteoclast precursor cells [37]. It was reported that αvβ3 is involved in the migration of osteoclast precursors [30]. The αvβ3-positive cells could therefore be forced to migrate by the ligands and may fuse with closed αvβ3-negative cells. Alternatively, only αvβ3-positive cells may be fused with each other.
It is possible to consider that signaling from CD16 by anti-CD16 mAb-coated magnetic beads, which were used for the cell separation, or by IgG contained in FBS might inhibit osteoclastogenesis from CD16+ monocytes. We therefore separated the two subsets using anti-CD33 mAb and a fluorescent cell sorter, and stimulated the cells with M-CSF + RANKL. The results showed that CD33low monocytes, which correspond to CD16+ monocytes, still could not differentiate into osteoclasts. CD16 is a heterodimer consisting of FcγIIIa and Fcγ, and has low affinity for the Fc region of IgG. Aggregation of CD16 by immune complexes leads to transmission of activating signals via the immunoreceptor tyrosine-based activation motif in the γ chain [38]. We also assessed osteoclastogenesis from the two monocyte subsets using IgG-depleted <span class="yellow">bovine</span> serum. Even in the IgG-free medium, CD16- monocytes but not CD16+ monocytes differentiated into osteoclasts (data not shown). We could therefore exclude the possibility that signal transduction through CD16 inhibits osteoclastogenesis from CD16+ monocytes.
NFATc1 is a key transcription factor in osteoclastogenesis [16]. In the present study, stimulation with M-CSF + RANKL increased the NFATc1 mRNA expression in the CD16- subset only, similar to integrin αv and integrin β3. The differences in NFATc1 induction might therefore also explain the difference in osteoclastogenesis between the two monocyte subsets. It is of interest that knockdown of integrin β3 did not lower the mRNA level of NFATc1. This result supports the notion that NFATc1 is located upstream of integrin-β3 expression [39]. It is also possible that parallel activation of two signaling pathways mediated by integrin β3 and NFATc1 contributes to osteoclastogenesis independently or cooperatively. Further studies are needed to determine the mechanisms of integrin β3 involvement in RANKL/RANK-mediated osteoclast differentiation.
It has been demonstrated that MAPK families, ERK and p38 MAPK, were activated by RANKL-induced intracellular signalings in osteoclasts and osteoclast precursors [18,19]. In addition, these kinases are involved in the differentiation of osteoclasts [20]. We showed that RANKL stimulation induced phosphorylation of ERK and p38 MAPK only in CD16- monocytes. It is suggested that differential activation of these kinases may partially explain the distinct properties of the two monocyte subsets upon RANKL stimulation.
Our results showed that CD16+ monocytes produce higher levels of inflammatory cytokines including TNFα and IL-6 compared with CD16- monocytes. These results are consistent with the previous report showing that CD16+ monocytes produced larger amounts of TNFα upon lipopolysaccharide or lipopeptide stimulation than did CD16- monocytes [40]. Interestingly, we showed that stimulation either with RANKL or M-CSF upregulated the TNFα and IL-6 production by CD16+ monocytes. A marked increase of CD16+ monocytes in peripheral blood is reported in inflammatory diseases, such as infection, malignancy, Kawasaki disease and RA [41-44]. Taken together, CD16+ monocytes may be an important source of inflammatory cytokines.
In <span class="yellow">mice</span>, peripheral blood Ly-6Chigh monocytes, which are thought to correspond to <span class="blue">human</span> CD16- monocytes, increase in inflammatory conditions, and these cells are recruited into sites of inflammation [45]. In contrast, Ly-6Clow monocytes, which are thought to correspond to <span class="blue">human</span> CD16+, migrate into noninflamed tissues [12]. These data on <span class="yellow">mouse</span> monocytes seem to be in contrast to the data on <span class="blue">human</span> monocytes, which show expansion of CD16+ monocytes in inflammatory conditions where they produce larger amounts of inflammatory cytokines. At present, it is not clear whether <span class="yellow">mouse</span> monocyte subsets, Ly-6Clow/Ly-6Chigh, represent <span class="blue">human</span> monocyte subsets, CD16+/CD16- monocytes, and whether the biologic functions of <span class="yellow">mouse</span> monocytes are analogous to those of <span class="blue">human</span> monocytes.
In <span class="yellow">mice</span>, blood monocytes newly released from the bone marrow are exclusively Ly-6Chigh and the level of Ly-6C is downregulated while in circulation [45]. It is thus suggested that in <span class="yellow">mice</span> the two monocyte subsets differing in Ly-6C expression represent different stages in the maturation pathway. In the <span class="blue">human</span>, transition from CD16- monocytes to CD16+ monocytes is observed upon culture with IL-10, M-CSF and transforming growth factor beta in vitro [42,46]. Similar to <span class="yellow">mouse</span> monocytes, therefore, <span class="blue">human</span> peripheral blood CD16- monocytes may also maturate into CD16+ monocytes.
It is reported that a significant number of RA synovial cells in the intima express CD16, suggesting that CD16+ cells are synovial macrophages [47]. We confirmed that both CD16+ and CD16- macrophages accumulate in the RA synovium by double-color immunohistochemical staining for CD68 and CD16. A number of chemokines are abundantly expressed in the RA synovium [24,48]. Among these cytokines, MCP-1, MIP-1α, SDF-1, RANTES and fractalkine can induce migration of CD16- monocytes in vitro ([11,12] and unpublished data). On the other hand, migration of CD16+ monocytes is induced only by fractalkine. These chemokines therefore seem to play an important role in recruitment of CD16+ and CD16- monocytes from the circulating pool into the RA synovium.
The osteoclast inducers are also produced in the RA synovium. RANKL is expressed by synovial fibroblasts and activated T cells [49-51], while M-CSF is expressed on RA synovial macrophages and fibroblasts [52,53].
TNFα and IL-6, which are mainly expressed on RA synovial macrophages and fibroblasts, respectively, could also enhance osteoclast differentiation [54]. Collectively, it is probable that the recruited CD16- monocytes/macrophages differentiate into osteoclasts in the RA synovium, and contribute to bone destruction. On the other hand, CD16+ monocytes/macrophages might also be involved in RA pathogenesis by producing inflammatory cytokines including TNFα and IL-6. Since TNFα and IL-6 enhance osteoclast formation [54,55], CD16+ monocytes/macrophages may also contribute to osteoclastogenesis in RA synovium.<br><br>Conclusion
We have shown that <span class="blue">human</span> peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct response to osteoclastogenic stimuli. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for the osteoclastogenesis. The blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Abbreviations
DAP = DNAX-activation protein; ELISA = enzyme-linked immunosorbent assay; FBS = fetal <span class="yellow">bovine</span> serum; FcRγ = Fc receptor γ chain; IL = interleukin; FITC = fluorescein isothiocianate; mAb, monoclonal antibody; M-CSF = macrophage colony-stimulating factor; MEM = modified Eagle's medium; MMP = matrix metalloproteinase; MNC = multinucleated cells; NF = nuclear factor; OSCAR = osteoclast-associated receptor; PBS = phosphate-buffered saline; PCR = polymerase chain reaction; RA = rheumatoid arthritis; RANK = receptor activator of NF-κB; RANKL = receptor activator of NF-κB ligand; RT = reverse transcriptase; siRNA = small interfering RNA; SIRP-β1 = signal regulatory protein-β1; TNFα = tumor necrosis factor alpha; TRAF = tumor necrosis factor receptor-associated factor; TRAP = tartrate-resistant acid phosphatase; TREM = triggering receptor expressed on myeloid cells.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YK participated in the design of the study, carried out the experiments and statistical analysis, and drafted the manuscript. KH and KT participated in the design of the study and its coordination. TN and NM conceived of the study, participated in its design and coordination, and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1891629</h3>A secondary structural common core in the ribosomal ITS2 (internal transcribed spacer) of
          Culexspecies from diverse geographical locations
Abstract
In the present study, sequence and structural analysis of ITS2 region (the spacer segment between 5.8S and 28S rRNA of 
          mature rRNA sequences) of 7 Culex species belonging to 5 different geographical locations was carried out. Alignment of the 
          ITS2 sequence from the 7 species revealed 8 homologous domains. Four species namely <span class="yellow">C. vishnui</span>, <span class="yellow">C. annulus</span>, <span class="yellow">C. pipiens</span>, 
          <span class="yellow">C. quiquefasciatus</span>showed high sequence (98­100%) and RNA secondary structure similarity. The ITS2 similarity among different 
          species is high despite their varying geographical locations. Several common features of secondary structure are shared among 
          these species, with some of them supported by compensatory changes, suggesting the significant role by ITS2 as an RNA domain 
          during ribosome biogenesis.<br><br>Background<br><br>				Culex mosquito species have been described from a wide range of environments and are involved in pathogen transmission from 
      <span class="blue">human</span> to reservoirs and vice-versa. [1] Sequence similarity have been 
      reported for ITS2 (the spacer segments between 5.8S and 28S rRNA of mature rRNA sequences) from 5 Culex species of diverse 
      geographic locations. [3–4–
      5–9–
      20] The ITS spacers are versatile as genetic markers and have been used for the determination of taxonomic 
      classification. [17] Recent functional analyses performed on <span class="yellow">yeast</span> ribosomal 
      RNA genes clearly show that the structural integrity of the transcribed spacer regions is an essential prerequisite for the correct 
      processing of mature rRNA and for the biogenesis of active ribosomal subunits. [2–
      3] The derivation of reliable secondary structure models for each transcribed spacer 
      region would represent a major step towards a detailed understanding of their biological roles.
Comparative sequence analysis provides a powerful tool for identifying biologically relevant folding patterns in RNA molecules. 
      [4] This involves collection of sequences exhibiting substantial nucleotide 
      differences while retaining unequivocal sequence similarity. However, a high degree of sequence variation in the transcribed 
      spacers is known even among closely related species. [5–6–7] Here, we analyze ITS2 from 7 
      Culex species characteristic of 5 different geographical locations to assemble common RNA structural features.<br><br>Methodology
Dataset
ITS2 nucleotide sequences from 7 Culex mosquitoes characteristic of 5 geographical locations (Italy, China, Africa, 
      America, and Japan) were downloaded from GenBank (www.ncbi.nlm.nih.gov/genbank
      ). The GenBank accession numbers for ITS2 sequences 
      from <span class="yellow">C. pipens</span> (North America), <span class="yellow">C. tritaeniorhynchus</span> (China), <span class="yellow">C. quinquefasciatus</span>
       (Africa), <span class="yellow">C. tarsalis</span> (Italy), <span class="yellow">C. annulus</span> (China), <span class="yellow">C. pseudovishnui</span> 
      (China), and <span class="yellow">C. vishnui</span> (Japan) were X75817, AF305558, Z48468, U33031, AF453488, AF453498 and AF165900, 
      respectively.<br><br>Sequence alignments
Multiple sequence alignments were performed using CLUSTALW with a gap opening penalty of 15 and gap extension penalty of 6.66. 
      [8]<br><br>Secondary structure prediction
The RNA secondary structures for ITS2 were predicted using RNADRAW. [21]
      RNADRAW predicts RNA structures by identifying suboptimal structures using the free energy optimization methodology at a default 
      temperature of 370°C.  In the current study, ITS2 and 5.8S regions (the first 170 nucleotides) were used for RNA structure 
      prediction. The minimum energy structure prediction algorithm in RNADRAW was ported from the RNAFOLD program included in the 
      Vienna RNA package. [24] The dynamic programming algorithm employed in 
      RNADRAW was based on the work of Zuker and Stiegler [25] and uses energy 
      parameters taken from Turner [26], Freier [27] and Jaeger. [28]<br><br>Phylogenetic analysis
The phylogenetic Genebee service was used for phylogenetic tree construction. [22
      ]<br><br>RNA fold
The Sribo program in Sfold (Statistical Folding and Rational Design of Nucleic Acids) was 
      used to predict the probable target accessibility sites (loops) for trans-cleaving ribozymes in ITS2. [24] The prediction of accessibility is based on a statistical sample of the Boltzmann 
      ensemble for secondary structures. Here, we assessed the likelihood of unpaired sites for potential ribozyme target. Each mRNA 
      exists as a population of different structures. Hence, stochastic approach to the evaluation of accessible sites was found 
      appropriate. [29] The probability profiling approach by Ding and Lawrence 
      [30] reveals target sites that are commonly accessible for 
      a large number of statistically representative structures in the target RNA. This novel approach bypasses the long-standing 
      difficulty in accessibility evaluation due to limited representation of probable structures due to high statistical confidence 
      in predictions. The probability profile for individual bases (W = 1) is produced for the region that 
      includes a triplet and two flanking sequences of 15 bases each in every site of the selected cleavage triplet (e.g., GUC).
      <br><br>
Results and Discussion
ITS2 sequences showed a taxonomic trend similar to that in phylogeny construction (Figure 1A
			). A multiple sequence alignment 
      of 5.8S rDNA and ITS2 showed in Figure 1B indicated that more distantly related species have 
      lower sequence similarity in the 
      ITS2 region. The predicted features of ITS2 using RNADRAW are given in Table 1. The stems 
      (double stranded paired regions) 
      stabilize RNA secondary structures and the number of stems present in each ITS2 is given (Table 1). ITS2 RNA structures from 
      <span class="yellow">C. pipiens</span> and <span class="yellow">C. quinquefasciatus</span> have the highest negative free energy (-149.38 Kcal and 
      -148.23 Kcal) followed by <span class="yellow">C. tarsalis</span> (-129.2), <span class="yellow">C. annulus</span> and then by <span class="yellow">C. vishnui</span>
      (-115.3), <span class="yellow">C. pseudovishnui</span> (-105.66) and <span class="yellow">C. tritaeniorhyncus</span> (- 82.57). Visual comparison shows 
      that this is related to the trend in the cladogram given in Figure 1A. A high degree of sequence similarity is observed at the 
      5' end compared to the 3' end (Figure 1A). This is due to factors such as genetic drift, the relative number and size of repeats, 
      rates of unequal crossover, gene conversion, immigration and the number of the loci influencing the length. [10] Furthermore, a high level of sequence similarity is found between C. 
      annulus and <span class="yellow">C. vishnui</span> as well as <span class="yellow">C.  quinquefasciatus</span> and <span class="yellow">C.  pipiens</span>.
      The simple tandem repeats shown in Figure 1B as bolded regions are found to be similar. 
      This similarity is seen in corresponding 
      RNA structures and computed energies.
The Culex species considered in this study were then grouped into three classes based on the similarity of 
      RNA stems and loops. Despite their different geographic locations with varying eco-climatic conditions, class I 
      (<span class="yellow">C. annulus</span>, <span class="yellow">C. pseudovishnui</span> and <span class="yellow">C. vishnui</span>) isolates (China and Japan) showed high similarity 
      in secondary structural features. Similar observations were seen in class II (<span class="yellow">C. pipiens</span>, <span class="yellow">C. quinquefasciatus</span> and 
      <span class="yellow">C. tarsalis</span>) isolates from North America and Africa. [19] 
      A high degree of similarity in the 5.8S region unlike the ITS2 is shown for different isolates due to relative evolutionary 
      selection. [9] The loops 11 and 12 having sequences UGUCG and CUUCGGUG, 
      respectively are highly conserved in all classes.
Figure 1C shows the distribution of different types of loops (hairpin, bulge, multi branched, interior and exterior) among 
      different isolates. The segments of the ITS2 having score ≥ 50 are further probed carefully for target site to assess the 
      likelihood of unpaired segments. Interestingly, the observed phylogenetic trend was identified with respect to the target 
      accessibility sites for the seven Culex isolates. The order of preference is interior loop, bulge loop, multiple 
      branched loop, hairpin loop and exterior loops in all the isolates.
Several homologous domains were observed in the ITS regions of Aedes mosquito species by Wesson 
			et al	 [11] and was shown that these domains base pair to 
			form a core region central to 
      several stem features. This suggested that conserved core region of rDNA is more important for a functional rRNA folding 
      pattern. Barker found that ITS2 is unique in the 16 populations of <span class="yellow">Boophilus microplus</span>, <span class="yellow">Boophilus decoloratus</span>, 
      <span class="yellow">Rhipicephalus appendiculatus</span>, <span class="yellow">Rhipicephalus zambesiensis</span>, <span class="yellow">Rhipicephalus evertsi</span>, <span class="yellow">Rhipicephalus sanguineus</span>, Rhipicephalus 
      turanicus, <span class="yellow">Rhipicephalus pumilio</span> and <span class="yellow">Rhipicephalus camicasi</span> from different geographical locations. 
      [12] These results suggest that the differences and similarity observed 
      in the ITS2 of different species are not simply accumulated due to random mutation and have evolved for functional selection 
      in ribosome biogenesis. However, it was shown in three related mosquito genera (Aedes, Psorophora and <span class="yellow">Haemogogus</span>)
      that the intra-spacer variable regions appear to co-evolve and that ITS2 variation is constrained by its secondary structure. 
      [11] Further studies have demonstrated that the ITS2 is essential 
      for the correct and efficient processing and maturation of 26S rRNA ribosomal units. [
      13] Furthermore, the information required for the efficient removal of ITS2 from its RNA precursor is not 
      localized but dispersed throughout the ITS2 region. Thus, insertions and deletions (indels) that affect secondary structures 
      alter rRNA processing. Critical changes in the rRNA folding pattern due to evolutionary sequence variation in the ITS spacer 
      regions may thus have an important role on the kinetics of precursor rRNA formation for the efficient functioning of rDNA clusters.
      <br><br>Conclusion
Comparison of ITS2 sequences from different isolates of Culex show similarity and variations. Surprisingly, species displaying 
      sequence similarity belong to different geographical locations with diverse climatic and ecological conditions. This implies that 
      the ITS2 regions have less selective pressure than the ribosomal regions. Several common structural folds were shared among the 
      selected mosquitoes for maintaining functional equivalents. Construction of an evolutionary tree using more isolates of 
      Culex will provide an understanding for their functional selection.<br><br>
<h3>pmcA1931593</h3>Integration in primary community care networks (PCCNs): examination of governance, clinical, marketing, financial, and information infrastructures in a national demonstration project in Taiwan
Abstract
Background
Taiwan's primary community care network (PCCN) demonstration project, funded by the Bureau of National Health Insurance on March 2003, was established to discourage hospital shopping behavior of <span class="yellow">people</span> and drive the traditional fragmented health care providers into cooperate care models. Between 2003 and 2005, 268 PCCNs were established. This study profiled the individual members in the PCCNs to study the nature and extent to which their network infrastructures have been integrated among the members (clinics and hospitals) within individual PCCNs.<br><br>Methods
The thorough questionnaire items, covering the network working infrastructures – governance, clinical, marketing, financial, and information integration in PCCNs, were developed with validity and reliability confirmed. One thousand five hundred and fifty-seven clinics that had belonged to PCCNs for more than one year, based on the 2003–2005 Taiwan Primary Community Care Network List, were surveyed by mail. Nine hundred and twenty-eight clinic members responded to the surveys giving a 59.6 % response rate.<br><br>Results
Overall, the PCCNs' members had higher involvement in the governance infrastructure, which was usually viewed as the most important for establishment of core values in PCCNs' organization design and management at the early integration stage. In addition, it found that there existed a higher extent of integration of clinical, marketing, and information infrastructures among the hospital-clinic member relationship than those among clinic members within individual PCCNs. The financial infrastructure was shown the least integrated relative to other functional infrastructures at the early stage of PCCN formation.<br><br>Conclusion
There was still room for better integrated partnerships, as evidenced by the great variety of relationships and differences in extent of integration in this study. In addition to provide how the network members have done for their initial work at the early stage of network forming in this study, the detailed surveyed items, the concepts proposed by the managerial and theoretical professionals, could be a guide for those health care providers who have willingness to turn their business into multi-organizations.<br><br><br><br>Background
Taiwan's National Health Insurance (NHI) under the control of the Bureau of National Health Insurance (BNHI), was launched in March 1995 to replace its social insurance system that was covering 59% of its population: government employees, labourers, farmers and servicemen [1]. By June 2003 the number of <span class="yellow">people</span> insured had reached 21,956,729 (99%). There were 17,259 medical providers (92%), including 575 hospitals and 16,684 clinics contracted with the BNHI for serving the enrolled population. The unique phenomenon characterized in Taiwan health care industry different from those in the western countries is the freedom of <span class="yellow">patients</span> to choose the health care providers they want, no matter what their disease severity is. Furthermore, Taiwan <span class="yellow">people</span> favor the larger scales of facilities and this fallacy leads to the phenomenon of big-hospital shopping. For example, <span class="yellow">people</span> choose the medical centers which are accredited as the highest level of medical science in Taiwan when they only suffer from a common cold.
In the spring of 2003, the SARS epidemic viciously attacked the health of Taiwan's <span class="yellow">people</span>. The <span class="yellow">people</span>'s freedom to choose medical providers caused the national health authority to barely control and traced the flow of epidemic. This event made Taiwan national health authorities rethink what happened and how it damaged under the traditional fragmented health care providers in Taiwan. One health reform launched was named the "Primary Community Care Network (PCCN) demonstration project", a nationwide health care financing program funded by the Bureau of National Health Insurance (BNHI) in March 2003 and it was a new model for the Taiwan government to redefine the role of family physicians in the health care delivery system. A PCCN in Taiwan consists of a group of clinic physicians whose medical jobs are viewed as family care and at least one hospital for secondary or tertiary care. The idea of member component design in PCCNs was aimed to lead the Taiwan citizens to choose one clinic physician as their personal family physician for health maintenance and this family physician also would have the responsibility of referring the <span class="yellow">patients</span> to specialty care if necessary. From a national health authority perspective, they expected the Taiwan <span class="yellow">people</span> to put an end to their fallacy that "bigger is better" for health care organizations and establish the idea of "<span class="blue">human</span> health", starting with prevention and primary care, followed by secondary or tertiary care, emphasizing health promotion and maintenance instead of disease curing. Furthermore, it could decrease the inappropriateness of medical usage, i.e., over-uses of secondary and tertiary medical services in the high-tech hospitals. In addition, the national health authority was expected to drive the traditional fragmented heath care providers into coordinated medical multidisciplinary teams and share the limited medical resources through the PCCN demonstration project. In summary, the PCCN demonstration project was aimed to: 1) change the traditional <span class="yellow">patients</span>' customs of freely choosing health care organizations and establish referral channels along the continuum of care, and 2) establish partnerships among the primary care clinics and hospitals to provide a continuum of health care services. It was also expected to establish the primary care system of family physicians to provide whole-<span class="yellow">people</span> health care and improve care quality [1].
Partnership structures in the PCCNs represent the virtual vertical (i.e., between the member clinics and hospitals) and virtual horizontal (i.e., among the member clinics) aspects of organizing, which designate the formal relationships between individuals and the total network and include organizational design to ensure effective communication, coordination, and integration across the total network. Each PCCN consists of five to ten clinics: half of them should offer the services of general medicine, internal medicine, surgery, obstetrics and gynecology, pediatric, or family medicine. And each PCCN has a central headquarters, usually in one of the clinic facilities, to coordinate and integrate the network. All the clinic physicians in a PCCN are assigned the roles of "family physicians" or "gatekeepers" who recruit <span class="yellow">people</span> from the local community, keep background and medical files on them, certify family physician education training programs, and hold office hours in the member hospital, where they serve as joint faculty members for further medical consultations or medical utilizations of labs and tests, if necessary. In addition, the hospital member is asked to help clinic physicians in their network to set up a medical information system, share hospital resources (medical equipment and library literature) with the clinic physicians in their network and establish referral channels among the network members. Furthermore, this new demonstration model tries to minimize the barriers to <span class="yellow">patient</span> access by setting up 24-hour a day, 7-day a week medical consultation telephone lines for providing urgent services onsite and for taking care of the <span class="yellow">patients</span> whose family physicians' practices are closed to assure seamless care channels. The BNHI funded these extra demonstration actions, at around one hundred thousand US dollars (i.e., NT$3,500,000) for each PCCN under the current fee-for-service payment system [1].
Figure 1 describes the organizational structure of individual PCCNs introduced in the demonstration project in Taiwan.
To date, the PCCN demonstration project has been in operation for more than three years. There have been 268 PCCNs formed in the period of 2003 to 2005 around Taiwan. The geographical distributions of PCCNs and their members were described in Table 1. Analyzing all 1,557 participating clinic members in the demonstration project in terms of medical specialties, they cover general medicine, internal medicine, surgeries, obstetrics and gynecology, pediatrics, family medicines, otolaryngology, ophthalmology, rehabilitation medicine, dermatology, and psychiatry, with 237 clinics providing more than two specialties. On the other hand, each PCCN recruits at least one district or regional accredited hospital for acute care demands (required for network members) and a medical center for tertiary care support (not required for network members). There are 6 medical centers, 52 regional hospitals, and 71 district hospitals joining in the demonstration project. See Table 1 for more detailed information about the PCCN members.
To date, there have been few empirical studies of the working relationships that have developed between members of the PCCN program. Partnership needs a method to determine at an early stage, to make sure whether they are making the most of collaboration [2] and the acceptance of the contracting networks in Taiwan as an organizational innovation worthy of greater diffusion deserves to be explored. Therefore, this study used a structured questionnaire to characterize the relationship among the members in the individual PCCNs, with regard to governance, clinical, marketing, financing, as well as information integration infrastructures. The results of this study provide descriptive analyses in detail to map the partnership developments, to enrich the body of knowledge of the partner relationships and to help policy makers understand the coordinated efforts of these health care providers which have developed under this system. It also provides the recommendations for heath policy decision-making and management of networks of health care providers for the future involvement.<br><br>Methods
This study was aimed at providing descriptive analyses to map the partnership development. To understand the actual integration actions done by network members, the theoretical concept employed by network partnerships were described and then the derived survey instrument was developed.
Theoretical framework for organization design of network integration
The rapid organizational changes in the health care industry have driven theorists from every discipline and across the world to seek an approach that allows organizations to flourish. Organization theory allows investigators to profile an organization from the aspect of patterns and regularities in organizational design and behavior. In the early 20th century, classical management theorists claimed that an organization has "a best way" to be organized and managed [3]. That implied that all organizations would own the "same" organizational styles or structures. In the 1960s, several theorists [4-8] challenged this assumption by applying a "contingency approach" to propose that there is no best way to organize an organization, and that the effectiveness of an organizational structure varies with the situation of an organization. Furthermore, it is proposed that the best way to organize an organization depends on the nature of the environment to which the organization relates.
Contingency theory delineates the concepts "organization's internal features," "the demands of organizational environments," "best adaptation," and, the most important and difficult of all, "best match" [9]. Lawrence & Lorsch [7] argued that environments characterized by uncertainty and rapid rates of change in market conditions or technology impose different demands, including constraints and opportunities, on organizations than do placid and stable environments. Similarly to Lawrence and Lorsch's views mentioned above, Galbraith [10,11] stressed the contingency perspective on information processing. The information-processing approach emphasizes that environment, size, and technology impose different information-processing requirements on organizations, and thus an organization must be designed to encourage information flow in both vertical and horizontal directions to achieve the overall tasks of the organization and, finally, organizational effectiveness [11-14].
Some theorists have criticized conventional contingency theorists who presume that organizational structure is driven by the environment. <span class="yellow">Child</span> [15], Miller [16], Van de Ven and Drazin [17], and Tushman and Romanelli [18] raised such criticisms; they argued that organizations become what they are not only because of the environment, but also because of choices made by members, especially choices about strategy and organizational design. As Thompson's words in the book Organizations in Action [8] put it, "organizations are not determined simply by their environments (p.27)." He also pointed out that "administration may innovate on any or all of the necessary dimensions, but only to the extent that innovations are acceptable to those on whom the organization can and must depend." Instead of assuming that administrators are highly constrained in their decisions, strategic contingency theorists emphasized "the importance of choice," that is, "the freedom of agency" [15]. Furthermore, Pfeffer [19] explicitly pointed out that "organizational structures are the outcomes of political contests within organizations (p.38)."
Daft [14] proposed a top management model to delineate how "a strategy is a plan for interacting with the competitive environment to achieve organizational goals." He stated that the major responsibility of top management is to determine the goals, strategy, and design of an organization to adapt to a changing environment. To assess the external and internal environments of an organization seems to be the first task for top managers in defining an organization's goals and missions. Then, guided by the goals and missions of the organization, top managers shape the design of the organization, including structural forms, information system, technology, <span class="blue">human</span> resources, organizational culture, and inter-organizational linkages, to achieve the final organizational performance.
Integration refers to the mechanisms of coordination, the ways guided to partnership goals to fit internal and external conditions [7,20,21]. In the early 1990s, proposals for US national health care reform recognized the need for integrating mechanisms to achieve both financial success and quality of care of a well-organized system of care [22,23]. Several researchers also viewed inter-organizational cooperation as resource exchanges, including client referrals, money, and staff [24-27]. From practical ways of viewing integration, the success of integration lies in the coordinative mechanisms and partnership working that support it [28], including an administrative organization that coordinates the operations of various health care services; a management information system that integrates clinical, utilization, and financial data and follows clients across different settings; a care coordination program such as case management or disease management that works with clients to arrange health care services; and a financial mechanism that enables pooling of funds across services [29-35]. Fox [36] suggested the success of integrated health networks should ensure that the new business link such aspects as technology, functional skills, customer access, management, or products that can be shared across both the core and the new business; to conduct market financial evaluation; to share the risk of vertical integration with outside entities, to develop the management structure that can reflect the degree of coordination necessary to support the core business activities; to ensure that the integration strategy meets the needs of customers, including medical treatment, the use of medical technology, and the preferred methods of purchase; and to measure the new business by its value to the enterprise as a whole, rather than by its profitability as a stand-alone entity.
In summary, the effects that integration in inter-organizational designs has on network management were substantial from a managerial perspective. Borrowing the ideas of strategic contingency perspective [8,15,19] and top management model [14], it could be imply that success (organization performance) in reengineering a network lies in the integration of process and services (see Figure 1), including leadership/governing structure, teamwork between disciplines and <span class="yellow">patient</span> care, financial planning, and information systems, characterized as the constructs of governance, clinical, financial, and information infrastructures, respectively, in this study. In addition, another construct, marketing infrastructure, was especially important and designed to explore for PCCNs in this study because of <span class="yellow">patients</span>' freedom of making healthcare choice and the traditional fragmented health care systems by individual health care organizations in Taiwan. One major reason for Taiwan <span class="yellow">people</span>'s hospital shopping preferences was that Taiwan <span class="yellow">people</span> usually believe the bigger the facility, the better capacities a facility has no matter on any aspect from medical professionals to tangible medical equipment and plants. And this fallacy made the public want to overuse the facility with high-tech medical services no matter if it fits their needs. From the health policy and management perspectives, therefore, the health care providers were encouraged to market their services as a new corporate identity and brand strategy [37], including offering tangible resources such as books, libraries, medical equipment, and intangible resources such as knowledge and information exchanges (education) and reputation sharing one another among PCCN members. Furthermore, through the process of marketing resource exchanges, therefore, each PCCN could establish the images of "one system, one brand and quality" for the public and for the health care providers. It also makes it be more visible to the public.
The five integration infrastructures of network management were constructed as a conceptual framework in this study to help to portray how the PCCN members have done. The survey instrument development was described in the following.<br><br>Survey instrument development: integration infrastructures and measurements of partnerships
Based on the five integration infrastructures of network management, the structured questionnaire were derived from extensive literature reviews.
Governance infrastructure
Governance assumes the broad responsibility for organizational goals and survival and involves the series process of setting and monitoring organizational goals and strategy development through a board of representatives [38]. Governance or administrative integration infrastructure in establishing network partnerships refers to administrative structures (or responsibilities) created to facilitate communication, clear lines of authority, accountability, and responsibility for <span class="yellow">patient</span> care services; to negotiate budgets and financial trade-offs; and to present a cohesive, consistent message in interactions with external agencies and the community [29,39-41] and most important for members in contract agreements, to manage participation [33]. From a multidisciplinary perspective, Mitchell and Shortell [42] applied the concepts of governance and management characteristics in effective community health partnerships. The construct of governance involved several tasks, including setting priorities for strategic goals, choosing the membership composition, obtaining the necessary financial resources, and setting up the accountability systems, and so on. The construct of the management refers to the tasks of engaging and maintaining organizational members' interest in a shared vision and mission, providing appropriate structures and coordination mechanisms for the specified strategies, promoting constructive conflicts and managing destructive conflicts, implementing information systems to monitor the dynamics, adjusting the leadership in the overall membership, and so on. The issues of governance and administrative integration in the PCCNs could include [2,38,40,41,43-47]:
• planning the shared visions and missions
• determining the shared service strategies, cooperation priorities, policies and principles
• identifying the information needed and how to get it
• organizing the network dynamics and member roles
• leading and managing the conflicts and communication
• designing and controlling the shared network performance systems, including indicator settings, feedbacks, and accountability.<br><br>Clinical infrastructure
The idea of care integration begins through such public programs that include social workers in public welfare departments, caseworkers in mental health, or nurses in public health departments. In the late 1980s, care integration was deemed necessary for the streamlining of care and negotiating the maze of long-term care services. At that time, it was referred to as service coordination or case management, or in other related terms [29]. The purpose of care integration is to work directly with <span class="yellow">patients</span> and their families over time to help them arrange and manage the complex resources that <span class="yellow">patients</span> may need to maintain health and independent functioning. At the same time, care integration is used to achieve the most cost-effective use possible of scarce resources, by steering <span class="yellow">patients</span> to the health, social, and support services most appropriate for them at a given time [29]. Conrad and Dowling [33] pointed out that to coordinate and integrate <span class="yellow">patient</span> care relies on connecting <span class="yellow">patient</span> services at the different stages of the <span class="yellow">patient</span> care processes. Care coordination in integrated networks can be achieved through integration of training programs and some clinical services, provision of complementary clinical capabilities, clinical geographic proximity design, clear role definition of each institution, commitment and flexibility of leaderships and medical staffs, and the support of a large referring physician groups embracing the affiliation concepts [48]. The issues of clinical integration in the PCCNs could include [48-50]:
• planning and differentiating target markets based on the clinical services of the network members
• uniting individual clinical professionals for clinical project planning
• designing <span class="yellow">patient</span>-centered care or case management teams
• establishing committees responsible for <span class="yellow">patient</span>-centered case report meetings, case referral, transfer, and tracing, file management (record and information exchanges), clinical quality management (quality assurance, improvement, risk and malpractice management, and utilization review), and medical continuing education and on-job education.<br><br>Marketing infrastructure
Marketing integration refers to how to work together as a whole both from the provider and <span class="yellow">patient</span> perspectives. One of the case reports interviewing developing integrated delivery system or networks realized that the most important thing is how an integrated system or network is promoted and what is promoted for the consumers [51], including focusing on product development, making sure the branding holds together, marketing directly to consumers, demonstrating values to consumers, and even conducting marketing research to make efforts for the long term. In a health care network with several organizational members and target <span class="yellow">patients</span>, the marketing infrastructure in PCCNs here refers to provider members' marketing, meaning the resource sharing and market development in a PCCN as a whole. The issues of the marketing integration in the PCCNs could include [37,52-54]:
• sharing the literature and facility publications among the network members
•uniting public promotions such as united activities, electronic and paper media for enhancing the network reputation as "one system, one brand and quality"
• differentiating target markets of the network for competing in the medical industry.<br><br>Financial infrastructure
Comprehensive, flexible, and adequate financing is a goal of the ideal continuum of care. That component is the most critical and challenging to manage under the changes in the health care delivery environment. Gillies et al. [30] suggested that integrating financial management across operating units adds the greatest value to systems or organizations. In one case study, Bramson et al. [55] also showed that reducing costs through joint purchasing by the radiology departments of a vertically integrated health system could yield substantial savings. The issues of the financial integration in the PCCNs could include:
• budgeting
• uniting equipment, medical materials, and drug purchasing and routine administrative stuff management
• pooling recruitment funds
• designing a financial risk and sharing mechanism.<br><br>Information infrastructure
Information is an essential component of an organization. A complete information system can help an organization to integrate its individual units and efficiently manage the continuum. The ideal information system for a continuum of care was conceived of and formed in the mid-1980s [56]. During the late 1980s, computer technology began to make an information system feasible and affordable through new computer chips with expanded capability and networking technology. In the 1990s, the individual services of the continuum upgraded their information systems to combine clinical, financial, and utilization data [29]. Some studies have argued that the quality of information systems can drive costs down, because a good information system can give physicians easy electronic access to complete the documentation of the <span class="yellow">patients</span>' clinical records, better inform them about reimbursement and capitation issues, help them easily associate and manage cases together, and achieve a higher level of professional satisfaction [57,58]. Using Inova Health System, an integrated delivery system in northern Virginia, as an example, Wager, Heda, and Austin [59] showed that by developing a health information network within an integrated delivery system, Inova can have a clinical transaction system for hospitals and other entities, a data repository for decision support and outcome management, a managed care information system to support managed care and capitation contracts, and greater capability to acquire physicians. The issues of information coordination include [60-68]:
• establishing an electronic medical record system, regional information network for <span class="yellow">patient</span> clinical and administrative data, clinical service arrangements and administrative work
• uniting the system information management and web pages.
The structured questionnaire was developed with the wording of practical managerial actions based on the five concepts just mentioned. There were 19 survey items on governance infrastructure, 25 on clinical infrastructure, 13 on marketing infrastructure, 20 on financial infrastructure, and 7 on information infrastructure. All 84 items were, simultaneously, applied to examine the relationships of the clinic's peer members and the relationship of clinic and hospital members in a PCCN, and it resulted in a total of 168 survey questions. The detailed information of the item questions was listed in Table 2, 3, 4, 5, 6. The structured questionnaires were drafted from previous literatures and then examined by two academic professors for theoretical accuracy. Then one pilot study was pre-tested for the PCCN pioneers (i.e., 92 network clinic members) and 116 hospital providers which have partner relationships with other health care organizations (i.e., hospitals, clinics, long-term care facilities). The wordings and meanings of each question item were revised to assure content validity. The Cronbach α values for the five integration constructs – governance, clinical, marketing, finance, and information infrastructure were 0.946, 0.958, 0.932, 0.944, and 0.898 for the measures of clinic-clinic member relationships; and 0.945, 0.949, 0.916, 0.948, and 0.896 for the measures of clinic-hospital member relationships.<br><br>
Study subjects
To find the member partnership, we sent questionnaires to 1,557 individual clinics which had belonged to PCCNs for at least one year, based on information contained in the Taiwan Primary Community Care Network List (Bureau of National Health Insurance 2003, 2004 and 2005).
We let clinic members in all PCCNs point out how they coordinate with their peer clinic members and hospital members within a PCCN because individual clinic members could be better informants than hospital members, which need to deal with multiple clinic relationships and therefore might find it hard to describe the coordination involvement one by one with clinic members. Moreover, networks form for various reasons and it might lead to the various involvements by individual network members (i.e., hospital and clinic members). Therefore, using the participating clinics as individual survey units, the results could portray the overall dynamics and processes more authentically and detailed throughout all PCCNs in the demonstration project.
Nine hundred and twenty-eight clinics responded (59.6 %), with 239 clinics in the Taipei region, 165 in the northern region, 241 in the central region, 108 in the southern region, 150 in the Kao-Ping region, and 15 in the eastern region of Taiwan. Ten clinics had not mentioned their practicing locations. There is no statistically significant difference in geographical distribution between the respondents and the study population (χ2 = 4.208, p > 0.05).<br><br>Analytical techniques
The data was first analyzed descriptively with frequency counts (percentage) for each survey item, instead of using mean as a statistical method, because the variation among the respondents may not represent the normal distribution and it might ignore the extreme values for the respondents' answers. To compare how the respondents perceived the strength of integration existing in clinic-clinic and clinic-hospital relationships, paired t-tests were performed for individual survey items, using the original numerical scores.<br><br>
Results
Profiling the partnerships in Taiwan PCCNs: governance infrastructure
With regard to the governance infrastructures, the frequency was counted for each survey item with recalculated scales: disagree (Likert scale 1 and 2), fair (Likert scale 3), and agree (Likert scale 4 and 5) with individual items. In clinic-clinic relationship (Table 2), the majority of clinic members agree that the determined deals were obeyed (Table 2, item 1: 88.69%), the goals and strategies of members were well-understood (Table 2, item 16: 79.74%), and the united principals for individual members were developed (Table 2, item 15: 79.42%). The higher percentages were also found in clinic-hospital relationship in the same items (Table 2). On the other hand, establishing fair coordination mechanism (Table 2, item 11: 27.91%), designing and employing the network performance indicators (Table 2, item 3: 21.23%), and establishing communication models and channels (Table 2, item 12: 19.94%) still occupied higher percentages not developed and deserved to been made the focus of more efforts in the future. Paired t-test analyses for all individual survey items of governance infrastructure showed that the deals obeyed (Table 2, item1) and plans and goals controlled (Table 2, item 2) were achieved more in clinic-clinic relationships than those in clinic-hospital relationships; however, the design of network performance indicators (Table 2, item 3), development of disintegration policy and principals (Table 2, item 8), and the establishment of fair coordination mechanism (Table 2, item 11) were reached more in clinic-hospital relationships than those in clinic-clinic relationships.<br><br>Profiling the partnerships in Taiwan PCCNs: clinical infrastructure
Examining the extent of clinical infrastructure for network members, establishing two-directed <span class="yellow">patient</span> referral systems and <span class="yellow">patient</span> referral information files (Table 3, items 35 & 37) and uniting medical continuing education and on-job education (Table 3, item 34) were shown at a highly implemented rate in clinic-clinic (more than 70%) and clinic-hospital (more than 80%) relationships. On the other hand, network members had higher percentages (more than 40%) not to think about the possible integration mechanisms including establishing committees to deal with medical malpractice (Table 3, item 44), planning and differentiating clinical market areas (Table 3, item 20), and designing <span class="yellow">patient</span>-centered case management teams (Table 3, item 22). Overall, there was better clinical integration involvement for all the described items in clinic-hospital relationships than those in clinic-clinic relationships within a network in this study (see Table 3, paired t-tests, p < 0.05).<br><br>Profiling the partnerships in Taiwan PCCNs: marketing infrastructure
For marketing planning, the clinics had better integrated marketing activities with their respective hospitals than with peer clinic members within PCCNs for all studied items (Table 4, paired t-test, p < 0.05). Examining the clinic-clinic relationships, uniting social activities (Table 4, item 53), sharing the individual facility reports for updated services (Table 4, item 46), public promotion (Table 4, item 54), and uniting and joining the facility activities (Table 4, item 51) were the top four marketing works done among clinic members (more than 60% implemented rate); and those items also showed a higher implemented rate (more than 70%) between clinic and hospital members.
On the other hand, facility assets such as reports (Table 4, item 49), and professional literatures and books (Table 4, item 45) were not well-shared among clinic members ("never-thinking" rate: 42.13%). In addition, uniting the network publication could make more efforts in the future ("never-thinking" rate in item 50: 39.98%). The room for clinic-hospital partnership to think about acting was kind of different from those in the clinic-clinic relationship. In addition to the uniting publication that can be encouraged to improve the clinic-hospital relationship (Table 4, item 50: 27.48%), cooperating in research projects (Table 4, item 52: 25.00%) and identifying and differentiating target markets (Table 4, item 57: 24.57%) had still more opportunities to be focused on in the future.<br><br>Profiling the partnerships in Taiwan PCCNs: financial infrastructure
The PCCN members were found to have a lower extent of financial integration as evidenced by higher percentage of ''never thinking'' scale about the survey items on almost all items (see Table 5). Slightly more integration (that is, ''acting'' rate) was found in only four items both in clinic-clinic relationships and in clinic-hospital relationships, including uniting budget planning (Table 5: item 58), sharing places, materials, and equipment (Table 5: item 68), uniting budgeting for certain services (Table 5: items 72), and designing the resource distribution principals based on the whole network goals (Table 5: item 77).
Further examining the financial infrastructure in clinic-clinic relationship and clinic-hospital relationship, paired-t tests revealed that clinic-hospital partnerships were involved more in places, materials, and equipment sharing and maintenance (Table 5, items 62 and 68) (p < 0.05) and higher financial infrastructure coordination exists in clinic-clinic relationships (Table 5, items 58, 59, 63–65, 72, 74, 75, and 77) (p < 0.05).<br><br>Profiling the partnerships in Taiwan PCCNs: information infrastructure
There was significantly greater integration of information in clinic-hospital than clinic-clinic relationships in all items in this category (Table 6, paired t-tests, p < 0.001). The greatest integration was found in electronic <span class="yellow">patient</span> records (Table 6: item 78), followed by information integration for <span class="yellow">patient</span> data (Table 6: item 79) and clinical service arrangements (Table 6: item 81). The lowest level of integration in information infrastructure was found in administrative works such as registration, billing and so on (Table 6: item 82, ''never-thinking'' rate more than 50%) within network members.<br><br>
Discussion
In this study, we surveyed 943 clinics that had belonged to Taiwan PCCNs for more than a year to understand the nature and extent of integration to which they and their associated PCCN members (clinics and hospitals) had in governance, clinical, marketing, financial, and information infrastructures. It was found a wide variance in the kind and degree of integration among them and a lot of room for better integration (Table 2, 3, 4, 5, 6).
From the governance perspective, we found lower integration was found in the establishment of fair coordination mechanism (Table 2: item 11) among member clinics and member hospitals. Coordination could be viewed from different perspectives, including the use of standardized languages and forms, organizational rules and procedures, the establishment of common rules, policies, and procedures, and the monitoring through memos, reports, and a computerized information system [69,70]. Facing the cumbersome integration processes, it suggests that each PCCN's headquarters should become actively involved and clarify the authority, responsibility and accountability of individual members, identify the potential conflict sources, and publicize the rules and regulation of network integration dynamics covering decision making processes, market planning, clinical teamwork designs, and financial reports of individual network members. These actions could enhance the trust and respect of network members one another and could improve the small extent of integration found in this study about the mechanisms for communication models and channels in the PCCNs (Table 2: item 12). From the network management perspective, communication could occur between the various entities such as between hospital and clinics, primary care physicians and specialists, managers and clinical professionals, and even among the clinical professionals in the network. To develop effective and timely communication channels was the key for the management of integrated organizations [30-32] and could alleviate the tensions that sometimes occur in the dynamics of the multi-organizations. In this study, it was found a low level of involvement of medical teams in medical projects, <span class="yellow">patient</span>-centered case management, and case report meetings among the network members (Table 3: item 21, 22, and 23) from a clinical integration perspective. Several researchers have addressed that clinical integration providing a process of medical management, care management, case management, and <span class="yellow">patient</span> management designed to transform the traditionally fragmented delivery system into a more cohesive system [71], and lead to higher service quality and assure financial objectives [72,73]. More attention could be paid to these activities in the future. In addition, it was also found less integration in planning and differentiating clinical market areas (Table 3, item 20) among the network members in the category of clinical infrastructure. This may result from the existing specialty diversities in individual PCCNs, which might not need to involve planning and differentiating market area based on the members' clinical services at the early stage of network development.
There was more involvement in marketing efforts in clinic-hospital relationships than in clinic-clinic relationships. Generally speaking, hospitals have more resources (i.e., money, <span class="blue">human</span> resources, materials, and physical assets) than clinics, which might explain the stronger marketing involvements between the clinic and hospital members, including the library sharing (books and literatures), facility brochure dissemination, public promoting, and medical research cooperation. These integration efforts also meet the expectation of the national health authority for resource sharing and medical quality image enhancement among the health care providers.
Financial infrastructure was found to be the least integrated, with most items never considered. Perhaps the only reason for the higher score of budget planning activities (Table 5: items 58 and 77) was that BHNI required each PCCN to design and determine its budgeting arrangement in advance before joining the demonstration project. While slightly more financial involvement was made among network members (Table 5: items 68, 72 & 73), possibly due to similar needs, there remains a lot of room for financial integration in the future.
There was a need for networks to develop electronic information systems, though creating and managing an integrated information system involves very detailed work. Most of the clinics surveyed have focused more on the individual public members' administrative works such as filing <span class="yellow">patient</span> medical records, collecting and managing network <span class="yellow">patient</span> clinical data, and scheduling clinical services, which were required by the BNHI. The factors for the health care managers to adopt the integrated clinical information systems include the decision of make or buy, adoption leadership, adoption objectives, implementation leadership, phased versus simultaneous implementation, parallel systems, information technology implementation policies and practices, use levels and resistance, and realized benefits and return on investment calculation [66], which might be very cumbersome and time-consuming. It suggests that the network partners might be engaged, firstly, more in simpler network cooperation such as the administrative systems for <span class="yellow">patient</span> admission to the network members and establishing united web pages for <span class="yellow">patients</span> to access their family physicians and network members for medical and public promotion purposes. And for further integrated information investments, efforts must be redirected for network members to work together to define the approach to specific classes of integration for the long term [74].<br><br>Conclusion
This study tried to portray and trace how the facility <span class="yellow">participants</span> were involved in the Taiwan PCCNs. It was found that Taiwan PCCNs' members had higher involvement in the governance infrastructure, which was usually viewed as the most important for establishment of core values in PCCNs' organization design and management. There existed a higher extent of integration of clinical, marketing, and information infrastructures among the hospital-clinic member relationship than those among clinic members within individual PCCNs. The financial infrastructure was shown the least integrated relative to other functional infrastructures at the early stage of PCCN formation. Page [43] argued that networks form and grow for various reasons, however, only some of them could be compatible with the iterative processes of collaboration. Some <span class="yellow">participants</span> in the PCCNs may simply seek short-term economic gains and have little interest in joint learning and continuous improvement. From an organizational design perspective, the old phrase proposed by the wisdom of the saying about developing the integrated organizations (networks) should be – "coming together is the beginning, and working together is the success." Page [43] examined the virtual provider organizations such as physician-hospital organizations and pointed out the issue of the provider attitudes and behaviors as the critically successful continuous improvements in the health care environments. A wide variance of degree of network integration in Taiwan PCCNs still leaves room to improve.
In this study, the thoroughly surveyed items, that is, the potential network design content, were employed. In addition to provide how the network members have done their initial work at the early stage of network forming in this study, the detailed surveyed items, the concepts proposed by the managerial and theoretical professionals, could be also a guide for those health care providers who have a willingness to join multi-organizations. It suggests that health care providers could take more detailed looks about those surveyed items and give some possible opportunities to create the potential actions. Further research could be empirically done to explore the relative influence of these integration mechanisms on the effectiveness of organizational partnerships.
The partnerships within each PCCN represent various relationships that depend on how much the members are engaged in the projects. In addition to the macro concepts including governance, clinical, marketing, financial, and information infrastructures explored in this study, other managerial issues for integrated organizations were also suggested such as formation of an integrated cultural atmosphere, <span class="blue">human</span> resources management, physician involvement, mission and commitment establishment, from micro organizational behavior perspective [30-32,34,36,75,76]. Micro managerial and longitudinal research designs could be employed to more precisely catch the never completing integration efforts in the future.<br><br>Abbreviations
primary community care network (PCCN); Bureau of National Health Insurance (BNHI)<br><br>Competing interests
The author(s) declare that they have no competing interests.<br><br>Authors' contributions
BYJL independently designed and conducted this study.<br><br>Pre-publication history
The pre-publication history for this paper can be accessed here:<br><br><br><br><h3>pmcA2151861</h3>Recombinant activated protein C in sepsis: endothelium protection or endothelium therapy?
Abstract
Endothelium dysfunction is one of the hallmarks of sepsis. Looney and Mattay, in the previous issue of Critical Care, highlight the role of activated protein C (APC) as a protective endothelial drug in septic situations. Nevertheless, the results of in vivo studies are less explicit and it remains uncertain whether these properties are relevant in <span class="blue">human</span> septic shock. Before considering recombinant APC (rAPC) as a therapeutic drug for the endothelium, we have to demonstrate its efficiency to protect or to reduce endothelium injury when infused a long time after the septic challenge. Nevertheless, if rAPC is efficient when infused in the early phase of septic challenge, we thus need to treat our <span class="yellow">patients</span> earlier. At the least, genetically engineered variants have been designed with greater anti-apoptotic activity and reduced anticoagulant activity relative to wild-type APC. Further studies are needed to demonstrate the usefulness of these variants in septic shock therapy.<br><br>
The use of recombinant activated protein C (rAPC) is one of the hottest topics in septic shock therapy. The pivotal phase 3 placebo-controlled Protein C Worldwide Evaluation in Severe Sepsis (PROWESS) clinical trial demonstrated a 19.4% relative risk reduction in 28-day mortality (6.1% absolute risk reduction) with an increased risk (3.5% versus 2.0%) of serious bleeding events compared with placebo.
Two recent and important articles have highlighted the role of APC as a protective endothelial drug [1] and as a cyto-protective drug [2]. Beneficial effects of rAPC in the PROWESS study were thought to be related to a reduction in coagulation and, to a lesser extent, to a reduction in inflammatory response to sepsis [3]. Post-PROWESS investigations have been associated with a myriad of cellular or animal studies demonstrating that rAPC, through reactions mediated by endothelial protein C receptor and the effector receptor, protease activated receptor-1, acts directly on cells to exert multiple cytoprotective effects including: down regulation of pro-inflammatory gene expression [4]; anti-inflammatory activities [5]; anti-apoptotic activity [6]; and protection of endothelial barrier function [1,2].
Endothelium dysfunction is one of the hallmarks of sepsis [7]. Sepsis, per se, may induce phenotypic modulations of the endothelium through direct or indirect interaction of the endothelial layer with components of the bacterial wall, inducing a myriad of host-derived factors from endothelial cells. Phenotypic modifications include changes in pro-coagulant and proadhesive properties, increased endothelial permeability, endothelial cell apoptosis and changes in vasomotor properties; the last of these is crucial since vasoplegia is directly related to septic shock mortality. Recent animal and <span class="blue">human</span> data have suggested that rAPC may improve both vascular and myocardial dysfunction and vascular reactivity to catecholamine during endotoxin and/or septic challenge [8,9].<br><br>From bench to bedside
Experimental evidence supports a role of APC in maintaining the integrity of the endothelium through both direct and indirect mechanisms. Nevertheless, the results of in vivo studies are less explicit. In a retrospective study of septic shock in <span class="yellow">humans</span>, Monnet and colleagues [9] demonstrated that APC infusion was associated with a decrease in the amount of delivered norepinephrine. Wiel and colleagues [10] demonstrated in a <span class="yellow">rabbit</span> model of endotoxin induced shock that APC decreased aorta endothelial injury. By contrast, in a lung model of endotoxin induced inflammation, Robriquet and colleagues [11] demonstrated a trend to an increased vascular permeability using high doses of <span class="blue">human</span> APC. This last result was in sharp contrast with the results obtained by Nick and colleagues [12] in a <span class="blue">human</span> model of pulmonary endotoxin administration. APC appears to improve mortality in septic shock with a high APACHE 2 score and is potentially detrimental in severe sepsis. In <span class="yellow">rats</span>, APC markedly decreased tumour necrosis factor concentrations whereas they remained unchanged in either <span class="blue">human</span> septic shock or endotoxemia. The question arises, therefore, as to whether it is truly possible to reconcile all these discrepancies? Moreover, can these stirring laboratory data be translated into clinical practice?<br><br>Limitations of experimental studies: endothelium protection versus endothelium therapy
Clearly, in cellular and animal models, rAPC has been given either as a pre-treatment or concurrent with septic challenge. This mode of administration favours the anti-inflammatory effects of rAPC, which are particularly efficient in <span class="yellow">murine</span> models in protecting the endothelium from cytokine-mediated apoptosis or upregulation of endothelial adhesion molecules. Thus, studies using post-injury treatment are needed in models that mimic septic shock, such as experimental pneumonia or peritonitis treated by antibiotics and volume resuscitation, and where the effects of rAPC would be investigated 16 to 24 hours after septic challenge. If we can demonstrate the efficiency of rAPC to protect or to reduce endothelium injury in these conditions, we can ultimately postulate that rAPC is also a therapeutic drug for the endothelium.<br><br>The earlier the better
If rAPC is efficient when infused in the early phase of septic challenge, we thus need to treat our <span class="yellow">patients</span> earlier. At least two studies suggest that treatment with rAPC within 24 hours may carry a larger survival advantage for <span class="yellow">patients</span> with severe sepsis, compared with those treated more than 24 hours after organ dysfunction [13]. Interventions directed at specific endpoints, when initiated early in the 'golden hours' of a patient's condition, seem to be promising [14]. The beneficial effects of earlier administration of rAPC to appropriate <span class="yellow">patients</span> may fit into this paradigm.<br><br>The future
Extensive in vivo and in vitro studies have focused on the cytoprotective effects of APC and most authors agree that its anticoagulant and cytoprotective effects are mediated by distinct APC structural features. Positively charged residues in surface loops in the APC protease domain have been identified as participating in the anticoagulant activity but not in cellular effects. Hence, variants have been designed with greater anti-apoptotic activity and reduced anticoagulant activity relative to wild-type APC [2]. Whether these genetically engineered variants actually provide superior pharmacological properties remains to be elucidated in vivo. Such investigations may allow the design of therapeutic APC variants with decreased anticoagulant activity to reduce the risk of bleeding on the one hand, but also with normal cytoprotective properties in order to retain full beneficial effects on sepsis outcome.<br><br>Abbreviations
PROWESS = Protein C Worldwide Evaluation in Severe Sepsis; rAPC = recombinant activated protein C.<br><br>Competing interests
BL has received reimbursements and funding from Eli Lilly, France.<br><br>
<h3>pmcA2365090</h3>Carbonic Anhydrase Inhibitors. Part 541: Metal Complexes of Heterocyclic Sulfonamides: A New Class of Antiglaucoma Agents
Abstract
Metal complexes of heterocyclic sulfonamides possessing carbonic anhydrase (CA) inhibitory properties were recently shown to be useful as intraocular pressure (IOP) lowering agents in experimental animals, and might be developed as a novel class of antiglaucoma drugs. Here we report the synthesis of a heterocyclic sulfonamide CA inhibitor and of the metal complexes containing main group metal ions, such as Be(II), Mg(II), Al(III), Zn(II), Cd(II) and Hg(II) and the new sulfonamide as well as 5-amino-1,3,4-thiadiazole-2-sulfonamide as ligands. The new complexes were characterized by standard physico-chemical procedures, and assayed as inhibitors of three CA isozymes, CA I, II and IV. Some of them (but not the parent sulfonamides) strongly lowered IOP in <span class="yellow">rabbits</span> when administered as a 2% solution into the eye.
CARBONIC ANHYDRASE INHIBITORS. Part 54 METAL COMPLEXES OF HETEROCYCLIC SULFONAMIDES: A NEW CLASS OF ANTIGLAUCOMA AGENTS Claudiu T. <br><br> Supuran*, Andrea Scozzafava <br><br> and Andrei Jitianu 2 <br><br> Universit& degli Studi, Dipartimento di Chimica, Laboratorio di Chimica Inorganica e Bioinorganica, Via Gino Capponi 7, 1-50121, Florence, Italy 2 "I.G. Murgulescu" Institute of Physical Chemistry, Academia Romana, Spl. Independentei 202, R-77208 Bucharest, Roumania Abstract: Metal complexes of heterocyclic sulfonamides possessing carbonic anhydrase (CA) inhibitory properties were recently shown to be useful as intraocular pressure (IOP) lowering agents in experimental animals, and might be developed as a novel class of antiglaucoma drugs. Here we report the synthesis of a heterocyclic sulfonamide CA inhibitor and of the metal complexes containing main group metal ions, such as Be(II), Mg(II), AI(III), Zn(II), Cd(II) and Hg(II) and the new sulfonamide as well as 5-amino-l,3,4thiadiazole-2-sulfonamide as ligands. The new complexes were characterized by standard physico-chemical procedures, and assayed as inhibitors of three CA isozymes, CA I, II and IV. Some of them (but not the parent sulfonamides) strongly lowered IOP in <span class="yellow">rabbits</span> when administered as a 2% solution into the eye. Introduction Sulfonamides possessing carbonic anhydrase (CA, EC 4.2.1.1) inhibitory properties [2] such as acetazolamide 1, methazolamide 2, ethoxzolamide 3 and dichlorophenamide 4 have been used for more than 40 years as pressure lowering systemic drugs in the treatment of open-angle glaucoma [3,4]. Their effect is due to inhibition of at least two CA isozymes present within cilliary processes of the eye, ie, CA II and CA IV, which is followed by lowered bicarbonate formation and reduction of aqueous humor secretion [5-7]. Their main drawback is constituted by side effects such as fatigue, augmented diuresis, or paresthesias, due to CA inhibition in other tissues/organs than the target one, ie, the eye [8]. <br><br> N--N <br><br> XN--N EtO S <br><br> NH 2 <br><br> O <br><br> NH O:S----O <br><br> NHEt <br><br> The above-mentioned side effects are absent in the case in which the inhibitor has topical activity, and is applied directly into the eye. This route has been demonstrated only in 1983 by Maren's group [9] and was followed by the development of the first clinical agent of this type, dorzolamide 5 [10,11 ]. Dorzolamide (Trusopt) has been introduced in clinical use in 1995 in USA and Europe and it constituted the beginning of a radically new treatment of glaucoma, devoid of the severe side effects observed with the systemic inhibitors [4-6]. The success of topical antiglaucoma CA inhibitors fostered much research in the synthesis and clinical evaluation of other types of such compounds [12-15]. 307 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides: A New Class ofAntiglaucoma Agents <br><br> On the other hand, metal complexes of heterocyclic sulfonamides of type 1-5 have been recently prepared by two groups 16-20], and it was proved that they possess much stronger CA inhibitory properties than the sulfonamides from which they were prepared 18-22]. Although the mechanism of CA inhibition of the metal complexes is presently unknown, it was hypothesized that their increased inhibitory power might be due to two processes, occurring separately or in concert, ie, (i) dissociation of the complex inhibitor in sulfonamide anions and metal ions (in diluted solution), which in turn both interact thereafter with the enzyme, at different binding sites, and (ii) direct interaction of the undissociated complex with the enzyme, and more specifically with the hydrophilic patch at the entrance of CA II active site [23], this being the isozyme most susceptible to inhibition with this class of compounds [2,22]. Whether initially the first mechanism of action mentioned above was favored by us [22], recent evidences suggested that the undissociated complex might be the inhibitory species, at least for some isozymes [24]. Since metal complexes are much more inhibitory than the parent sulfonamide from which they were prepared, it appeared of interest to test whether this property might be useful for their use in lowering IOP in experimental animals and whence as a possible glaucoma therapy. Recently we proved [25, 26] that some metal complexes of heterocyclic sulfonamides (which themselves do not possess IOP lowering properties) act as very powerful such agents when administered as diluted solutions directly into the eye of experimental animals, and would thus offer the possibility of developing such totally novel drugs. Here we report the synthesis of a heterocyclic sulfonamide possessing strong CA inhibitory properties, ie, 5-(chloroacetamido)-l,3,4-thiadiazole-2-sulfonamide, and of the metal complexes of this sulfonamide and of 5-amino-1,3,4-thiadiazole-2-sulfonamide, containing some main group metal ions. The new compounds have been characterized by standard physico-chemical procedures, and were assayed as inhibitors of three CA isozymes, hCA I, hCA II and bCA IV (h <span class="blue">human</span>; b <span class="yellow">bovine</span>; these are the isozymes considered to play a critical role in aqueous humour secretion within the eye of higher vertebrates [2-5]). Materials and Methods <br><br> Melting points were recorded with a heating plate microscope and are not corrected. IR spectra were recorded in KBr pellets with a Carl Zeiss IR-80 instrument. 1H-NMR spectra were recorded in DMSO-d6 as solvent, with a Bruker CPX200 instrument. Chemical shifts are reported as values, relative to Me4Si as internal standard. Conductimetric measurements were done at room temperature (1 mM concentration of complex) in DMSO solution with a Fisher conductimeter. Elemental analyses were done by combustion for C, H, N with an automated Carlo Erba analyzer, and gravimetrically for the metal ions, and were 0.4% of the theoretical values. Thermogravimetric measurements were done in air, at a heating rate of 10C/min., with a Perkin Elmer 3600 thermobalance. Sulfonamides used as standards in the enzymatic assay (except for 5), acetazolamide, pyridine, and chloroacetyl chloride used for the preparation of compound 7, solvents as well as inorganic reagents were from Sigma, Merck and Carlo Erba. 5-Amino-l,3,4-thiadiazole-2-sulfonamide 6 was prepared from acetazolamide by literature procedures [27], by desacetylation with concentrated hydrochloric acid, followed by neutralization with sodium bicarbonate of the corresponding hydrochloride (Scheme 1). Dorzolamide hydrochloride 5 was from Merck, Sharp and Dohme or was prepared as described by Ponticello et al 10,11 ]. <span class="yellow">Human</span> CA and CA II cDNAs were expressed in <span class="yellow">Escherichia coli</span> strain BL21 (DE3) from the plasmids pACA/hCA I and pACAdaCA II described by Forsman et al. [28] (the two plasmids were a gift from Prof. Sven Lindskog, Umea University, Sweden). Cell growth conditions were those described by Lindskog's group [29], and enzymes were purified by affinity chromatography according to the method of Khalifah et al [30]. Enzyme concentrations were determined spectrophotometrically at 280 nm, utilizing a molar absorptivity of 49 mM-l.cm-1 for hCA and 54 mM-l.cm-1 for hCA II, respectively, based on M 28.85 kDa for hCA I, and 29.3 kDa for hCA II, respectively [31,32]. bCA IV was isolated from <span class="yellow">bovine</span> lung microsomes as described by Maren et al, and its concentration has been determined by titration with ethoxzolamide [33]. Synthesis of 5-(chloroacetamido)- l, 3, 4-thiadiazole-2-sulfonamide 7 An amount of 1.80 g (10 mmol) of 5-amino-1,3,4-thiadiazole-2-sulfonamide 6 was suspended in 20 mL of anhydrous acetonitrile and 0.9 mL (0.87g, 11 mmol) of pyridine added. The mixture was magnetically stirred at 4 C for 10 minutes, then 10.5 mmol of monochloroacetyl chloride, dissolved in 3 mL acetonitrile, were added dropwise for 5 min, and stirring was continued for other 2 hours at room temperature. After an additional 30 min of refluxation, followed by cooling, the precipitated crystals were filtered and recrystallized from ethanol. Yield of 62% white crystals, mp 246-248 o lit [34] mp IR (KBr), cm-l: 590, 610, 660, 790, 935, 1090, 1115, 1170, 1350, 1400, 1550, 1650, 1720, 2870, 3280 3370 (broad); UV spectrum, ,max, nm (lg): 255 (3.50); 288 (4.37) H-NMR (DMSO-d6), i, ppm: 2.96 (s, 2H, CH2); 8.20 (s, 2H, SOzNH2); 12.22 (s, 1H, CONH). Analysis, found: C, 18.56; H, 1.88; N, 21.76; S, 24.62 %; C4HsC1N403S2 requires" C, 18.72; H, 1.96; N, 21.83; S, 24.98 %. <br><br> General procedure for the preparation of compounds 8-20 An amount of 6 mmol of sodium salt of sulfonamides 6 or 7 was prepared by reacting the corresponding sulfonamide with the required amount of an alcoholic 1N NaOH solution, in ethanol as solvent. To this 308 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> solution was added the aqueous metal salt (Zn(II), Mg(II), AI(III), Cd(II) chlorides, and Be(II), Pb(II) and Hg(II) nitrate) solution, working in molar ratios RSOzNH- Mn+ of 2:1 for the divalent cations and 3:1 for the trivalent cation, respectively. The aqueous-alcoholic reaction mixture was heated on a steam bath for one hour, adjusting the pH at 7 if necessary, and after being cooled at 0 C the precipitated complexes were filtered and thoroughly washed with alcohol-water 1:1 (v/v) and air dried. Yields were in the range of 85-90 %. The obtained white powders of compounds 8-20 melted with decomposition at temperatures higher than 350 C, and were poorly soluble in water and alcohol, but had good solubilities in DMSO, DMF as well as mixtures of DMSO-water, DMF-water. <br><br> Pharmacology Carbonic anhydrase inhibition Initial rates of 4-nitrophenyl acetate hydrolysis catalysed by different CA isozymes were monitored spectrophotometrically, at 400 rim, with a Cary 3 instrument interfaced with an IBM compatible PC [35]. Solutions of substrate were prepared in anhydrous acetonitrile; the substrate concentrations varied between 2.10-2 and 1.10-6 M, working at 25C. A molar absorption coefficient of 18,400 M-l.cm-1 was used for the 4-nitrophenolate formed by hydrolysis, in the conditions of the experiments (pH 7.40), as reported in the literature [35]. Non-enzymatic hydrolysis rates were always subtracted from the observed rates. Duplicate experiments were done for each inhibitor concentration, and the values reported throughout the paper are the mean of such results. Stock solutions of inhibitor (1 mM) were prepared in distilled-deionized water with 1020% (v/v) DMSO (which is not inhibitory at these concentrations [2]) and dilutions up to 0.01 nM were done thereafter with distilled-deionized water. Inhibitor and enzyme solutions were preincubated together for 10 min at room temperature prior to assay, in order to allow for the formation of the E-I complex. The inhibition constant KI was determined as described by Pocket and Stone [35]. Enzyme concentrations were 3.3 nM for hCA II, l0 nM for hCA and 34 nM for bCA IV (this isozyme has a decreased esterase activity [36] and higher concentrations had to be used for the-measurements). <br><br> Measurement of tonometric lOP Adult male New Zealand albino <span class="yellow">rabbits</span> weighing 2-3 kg were used in the experiments (three animals were used for each inhibitor studied). The experimental procedures conform to the Association for Research in Vision and Ophthalmology Resolution on the use of animals. The <span class="yellow">rabbits</span> were kept in individual cages with food and water provided ad libitum. The animals were maintained on a 12 h: 12 h light/dark cycle in a temperature controlled room, at 22-26 C. Solutions of inhibitors (2 %, by weight) were obtained in DMSOwater (2:3, v/v) due to the low water solubility of some of these derivatives. Control experiments with DMSO (at the same concentration as that used for obtaining the inhibitors solutions showed that it does not possess IOP lowering or increasing effects. IOP was measured using a Digilab 30R pneumatonometer (BioRad, Cambridge, MA, USA) as described by Maren's group [37-39]. The pressure readings were matched with two-point standard pressure measurements at least twice each day using a Digilab Calibration verifier. All IOP measurements were done by the same investigator with the same tonometer. One drop of 0.2 % oxybuprocaine hydrochloride (novesine, Sandoz) diluted 1"1 with saline was instilled in each eye immediately before each set of pressure measurements. IOP was measured three times at each time interval, and the means reported. IOP was measured first immediately before drug administration, then at 30 min after the instillation of the pharmacological agent, and then each 30 minutes for a period of several hours. For all IOP experiments drug was administered to only one eye, leaving the contralateral eye as an untreated control. The ocular hypotensive activity is expressed as the average difference in IOP between the treated and control eye, in this way minimizing the diurnal, seasonal and interindividual variations commonly observed in the <span class="yellow">rabbit</span> [37-39]. All data are expressed as mean SE, using a one-tailed test. Results and Discussion Reaction of 5-amino-l,3,4-thiadiazole-2-sulfonamide 6 [19b] with chloroacteyl chloride in the presence of pyridine afforded 5-(chloroacetamido)-l,3,4-thiadiazole-2-sulfonamide 7, by the procedure already reported by Young et al. [34] (Scheme 1). The sulfonamide 7 has been characterized by elemental analysis and spectroscopic methods which confirmed its structure (only its m.p. has been reported in ref. [34]). The sodium salt of sulfonamides 6 and 7, obtained in situ from the corresponding sulfonamide and sodium hydroxide, were then used for the preparation of coordination compounds, containing the following metal ions: Be(II), Mg(II), Al(III), Zn(II), Cd(II) and Hg(II). Mention should be made that although 5-amino-l,3,4-thiadiazole-2-sulfonamide 6 is the parent compound of important sulfonamide CA inhibitors, such as acetazolamide, benzolamide, methazolamide, etc., its coordination chemistry has been scarcely investigated up to now [22, 40]. The new complexes prepared in this work are shown in Table I. Both compounds containing the sulfonamide-deprotonated species of sulfonamide 7 (LH), as well as complexes in which the anion of 5amino-1,3,4-thiadiazole-2-sulfonamide (tda) act as ligands, have been prepared. In fact in another work [40] it was documented that in some cases, sulfonamides derived from this ring system may undergo hydrolysis to <br><br> 309 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides." A New Class ofAntiglaucoma Agents <br><br> the moiety substituting the 5 position, with the formation of 5-amino-l,3,4-thiadiazole-2-sulfonamide 6, which thereafter coordinates metal ions present in solution. <br><br> N CIH N <br><br> I <br><br> N +NaHCO 2 <br><br> N 2 <br><br> S' <br><br> -NaCl <br><br> I S <br><br> N <br><br> II <br><br> SO2NH <br><br> 1_12 O Py/MeCN <br><br> 6Htda +CIOCCH2CI <br><br> 0 Cl CH 2 Scheme 1 <br><br> HN <br><br> .I <br><br> N <br><br> N <br><br> S 7: LH <br><br> II <br><br> SqNH <br><br> Thus, the X-ray crystal structure of the complex [Zn(tda)2(NH3)].H20 prepared in this way has recently been reported by this group [40]. On the other hand, when the ligand 7 has not been hydrolyzed (during the preparation of the coordination compounds) in the presence of the metal ion to 5-amino-l,3,4-thiadiazole-2sulfonamide and chloroacetate, the metal complexes contining 6 as ligand have been prepared from the last (pure) compound (as sodium salt) and the corresponding metal salt, by the general procedure described in the Experimental part. Table I: Prepared complexes 8-20, containing the conjugate bases of sulfonamides 6 and 7 as ligands and their elemental analysis data. L stands for the sulfonamide deprotonated species of 7, whereas tda for the sulfonamide deprotonated species of 5-amino-1,3,4-thiadiazole-2-sulfonamide 6. <br><br> No. 8 9 10 11 12 13 14 15 16 17 18 19 20 <br><br> Complex <br><br> Yield <br><br> (%) <br><br> %Ma <br><br> Analysis (calculated/found) %H b %C b 13.0/13.1 11.0/10.8 11.3/11.4 10.2/10.1 8.5/8.1 7.9/7.9 12.7/12.8 18.4/18.1 18.1/17.9 16.6/16.2 15.3/14.9 13.4/13.3 12.7/12.5 1.6/1.3 2.7/2.3 1.4/1.1 1.2/1.2 1.0/1.2 1.6/1.3 1.6/1.6 1.5/1.5 1.5/1.3 1.3/1.4 1.2/1.1 1.1/1.2 1.0/1.0 <br><br> %N b 30.4/30.2 25.6/25.5 26.4/26.4 13.7/23.3 20.0/19.8 18.6/18.5 29.7/29.6 21.5/21.3 21.1/20.8 19.4/19.3 17.9/17.8 15.7/15.6 14.8/14.6 <br><br> [Be(tda)2] [Mg(tda)2].3 H20 [Zn(tda)2] [Cd(tda)2] [Hg(tda)2] [Pb(tda)z(OH2)2] [Al(tda)3] [BeL2] [ALL3] [ZnL2] [CdL2] [HgL2] [PbLz(OH2)2] <br><br> 78 76 83 90 95 84 72 75 59 87 88 92 95 <br><br> 2.4/2.5 5.5/5.1 15.4/15.0 23.8/24.1 35.8/35.7 34.4/34.7 4.7/4.4 1.7/1.6 3.4/3.5 11.3/11.5 18.0/18.1 28.1/28.3 27.4/27.2 <br><br> aBy gravimetry; bBy combustion. The new complexes have also been characterized by spectroscopic, conductimetric and thermogravimetric measurements (Table II). By comparing the IR spectra of the complexes and the corresponding ligands, the following observations should be made: (i) the shift of the two sulfonamido vibrations (both the symmetric as well as the the assymetric one), towards lower wavenumbers in the spectra of the complexes, as compared to the spectra of the corresponding ligand (Table II), as already documented previously for similar complexes [13-22]. This is a direct indication that the deprotonated sulfonamido 310 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> moieties of the ligands interacts with the metal ions in the newly prepared coordination compounds; (ii) the amide vibrations (the most intense such bands at 1670-1680 cm-1) of ligand 7 appear unchanged in the IR spectra of complexes 15-20 (data not shown), suggesting that these moieties do not participate in coordination of the metal ions; (iii) the C-N stretching vibration in the spectra of the prepared complexes is shifted with 5-20 cm-1 towards lower wavenumbers, as compared to the same vibration in the spectra of sulfonamides 6 and 7, indicating that one of the endocyclic nitrogens of the thiadiazolic ring (presumably N3) acts as donor atom, as already documented by X-ray crystallographic and spectroscopic determinations on complexes of other sulfonamides (such as 1-3) with divalent metal ions [13-22] (Table II); (iv) changes in the region 3100-3160 cm-, as the bands present in the spectra of sulfonamides 6, 7 are present in the spectra of complexes 8-20 too, but they are not well resolved, and have a smaller intensity. This is probably due to deprotonation of the SO2NH2 moiety and participation in the binding of cations; (v) the amino vibrations from 3320 cm- in the spectra of 6 appear unchanged in the spectra of its complexes 8-14 (data not s.hown). In the 1H-NMR spectra of compound 6 and its metal complexes, the signal of the amino group has been evidenced as a broad singlet centered at 4.54 ppm (Table II), which is not exchangeable by addition of D20 into the NMR tube, in contrast to the sulfonamido NH2 protons (which readily exchange). This proves that the 5-amino moiety is not involved in binding the metal ions, as already shown in the X-ray crystallographic work of the complex [Zn(tda)z(NH3)].H20 previously reported [40]. For sulfonamide 7 the CONH proton resonates as a singlet at 12.22 ppm. In complexes 15-20 only very minor shifts of this signal were evidenced (Table II), proving basically that the CONH moiety does not interact with the metal ions in these complexes. <br><br> Table II: Spectroscopic, thermogravimetric and conductimetric data for compounds 6-20. <br><br> 1H-NMR Spectrab Comp. IR Spectraa ,cm-1 as v(C=N) CONH, i (ppm) v(SO2)S; v(SO2) 6 8 9 10 11 <br><br> TG analysisc calc./found <br><br> Conductimetryd AM (-1 x cm2x mol-) 2 7 4 5 3 2 2 6 3 4 2 9 3 2 8 <br><br> 12 13 14 7 15 16 17 18 19 20 <br><br> 1170; 1150; 1150; 1145; 1145; 1140; 1145; 1150; 1170; 1130; 1140; 1140; 1150; 1140; 1145; <br><br> 1350 1300 1305 1300 1305 1300 1300 1300 1350 1335 1330 1330 1330 1335 1330 <br><br> 1610 1600 1600 1600 1600 1590 1605 1605 1610 1605 1610 1610 1605 1600 1600 <br><br> A A A A A A A <br><br> e e <br><br> 12.3/12.1f e e e 5.9/5.7g <br><br> A 12.22 (1H) 12.19 (2H) 12.18 (3H) 12.20 (2H) 12.18 (2H) 12.19 (2H) 12.21 (2H) <br><br> e e e e e e e 4.7/4.8g <br><br> a In KBr; bin DMSO-d6; A the signal of the 5-amino group of 6 (appearing in the ligand at 4.54 ppm as a broad singlet) appears at the same chemical shift (4.50 4.55 ppm) in complexes 8-14; cWeight loss between 70-250 C; d mM solution, in DMF, at 25C; e No weight loss seen under 250 C; fCorresponding to three lattice water molecules lost at 70-110C, and gCorresponding to two coordinated water molecules, lost at 160-180 C. <br><br> Thermogravimetric analysis showed the presence of uncoordinated water molecules in the molecule of complex 9 (the three waters were lost in a single step, between 70-110 C) and of coordinated water in the molecules of the lead(II) derivatives 13 and 20. All these compounds behaved as non-electrolytes in DMF as solvent (Table II). Mention should be made that the Mg(II) complex of sulfonamide 7 could not be isolated. Instead, only the correponding complex of 5-amino-l,3,4-thiadiazole has been obtained from reaction mixtures containing magnesium salts and the sodium salt of 7, probably due to a metal ion assisted hydrolysis of 7 to 6 and chloroacetate. Generally such hydrolytic processes involve highly acidic conditions and prolonged heating of the 5-alkylamido-1,3,4-thiadiazole-2-sulfonamide derivatives [41 ], but they might become milder by taking into account the putative catalytic effect of Mg2+ ions reported here. The data shown above lead to the conclusion that ligand 7 shares a common coordination chemistry with acetazolamide 1 with which it is structurally related, whereas 6 probably also behaves similarly to acetazolamide in the sense that the 5-amino group seems not to be involved in coordinating metal ions, at 311 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides: A New Class ofAntiglaucoma Agents <br><br> least in the complexes reported by us here (and also in the compound characterized by X-ray crystallography mentioned above [40]). Thus, in all complexes reported here these sulfonamides (as monodeprotonated species at the SO2NH 2 moieties) act as bidentate ligands, through the endocyclic N-3 and the NH- groups. The proposed formulae of the new complxes are shown below. Except for the two Pb(II) complexes 13 and 20, as well as the AI(III) derivatives 14 and 16, which presumably are pseudo-octahedral, the other derivatives are supposed to contain tetrahedral M(II) ions. <br><br> R <br><br> R 13"R = 20" R= <br><br> H2N <br><br> CICH2CONH <br><br> 14: R = 16: R = <br><br> H2N CICH2CONH <br><br> The compounds 6-20 together with the standard CA inhibitors 1-5 were assayed for inhibition against three isozymes, hCA I, hCA II and bCA IV (Table III). As seen from the above data, the chloracetamido derivative is more inhibitory than acetazolamide, methazolamide and dichlorophenamide, whereas the unacylated compound 6 is less inhibitory than the above sulfonamides. The metal complexes 820 are much more inhibitory than the sulfonamides from which they derive 6, 7 and than all other simple sulfonamides assayed. They behave similarly to the metal complexes of acetazolamide, methazolamide or dorzolamide previously reported by this group, which were all more inhibitory than the parent sulfonamide from which were prepared [16-22, 40]. Particularly strong inhibition was observed for the Zn(II), Hg(II), Pb(II) and Cd(II) complexes, especially against CA II and CA IV, the isozymes critical for aqueous humor formation. In vivo IOP lowering experiments were done in <span class="yellow">rabbits</span> with some of the new compounds prepared in the present work, such as the sulfonamides 6 and 7, and their Zn(II) complexes, which were among the strong CA II and CA IV inhibitors in the obtained series. Some of the IOP lowering data at half an hour and one hour after the instillation of one drop of 2 % solution of inhibitor within the <span class="yellow">rabbit</span> eye are shown in 312 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> Table IV, with dorzolamide (at the same concentration) as standard. In Fig. the time dependence of IOP lowering with dorzolamide 5 and the two Zn(II) complexes 10 and 17 is presented. Table III. CA inhibition data with the standard inhibitors 1-5, the sulfonamides 6 and 7, and their metal complexes 8-20. <br><br> No 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 a <br><br> Inhibitor <br><br> KI hCA Ia 900 780 25 1200 >50,000 1550 640 1050 350 50 40 12 80 240 120 80 40 40 9 15 <br><br> (nM) bCA IVb 220 240 13 380 43 780 24 540 220 25 19 10 26 110 12 16 9 10 5 10 <br><br> hCA IIa 12 14 8 38 9 230 5 190 110 15 14 7 10 76 5 4 3 3 2 5 <br><br> Acetazolamide Methazolamide Ethoxzolamide <br><br> Dichlorophenamide Dorzolamide <br><br> <span class="yellow">Human</span> (cloned) isozymes; b From <span class="yellow">bovine</span> lung microsomes. <br><br> Table IV: IOP lowering following topical application of CA inhibitors, half an hour and one hour after instillation into the eye of a drop (50 L) of 2 % solution of inhibitor. Inhibitor <br><br> lOP SE 1/2 h <br><br> a <br><br> (mm Hg) lh 4.1 0.15 0 0.09 0 0.09 5.0 0.12 8.1 0.21 Dorzolamide 5 6 7 10 17 a <br><br> 2.2 0.10 0 0.10 0 0.10 2.00.09 8.0 0.14 <br><br> IOP <br><br> IOP control eye" IOP treated eye (N 3). <br><br> As seen from the above data, the sulfonamides 6 and 7 are totally ineffective as lOP lowering agents, similarly to the classical clinically used inhibitors of type 1-5 [2,3]. On the other hand, dorzolamide, the first topical sulfonamide used clinically in the treatment of glaucoma is an effective such agent, with a decrease of lOP of around 4 mm Hg, one hour after administration directly into the eye (Table IV). From the data of this table, it is obvious that the metal complexes of heterocyclic sulfonamides investigated by us behave as much more effective IOP lowering agents than dorzolamide, and their effect is generally longerlasting (Fig. 1). A last remark should be made about the possible mechanism of action of the new class of IOP lowering agents. Obviously, their activity is due to inhibition of CA isozymes present in the cilliary processes within the eye, similarly to other topically active sulfonamides [2-6]. The fact that the sulfonamide per se is inactive via the topical route, whereas the metal complexes result much better than the drug <br><br> 313 <br><br> Vol. 4, No. 6, 1997 <br><br> Metal Complexes of Heterocyclic Sulfonamides." A New Class ofAntiglaucoma Agents <br><br> dorzolamide, indicates that the presence of metal ions in the molecules of these CA inhibitors is essential and confers them completely new properties. <br><br> Lowering of IOP (ram Hg) <br><br> "||, <br><br> / <br><br> ", , <br><br> '",., <br><br> ,< <br><br> .7, / <br><br> / <br><br> .6<br><br> -0 <br><br> -10 0 <br><br> 2 <br><br> Time (hours) <br><br> Fig l" Time dependence of IOP lowering with dorzolamide (curve 1); the zinc complex 10 (curve 3) and the zinc complex 17 (curve 3), after topical administration of one drop of 2 % solution of inhibitor in <span class="yellow">rabbit</span>. <br><br> Preliminary results from this laboratory indicate that the metal complexes of topically active sulfonamides show also increased lOP lowering effects with respect to the complexes prepared in the present study [42]. Our hypothesis is that the presence of the metal ion in the molecules of these complex inhibitors induces a dramatic change in their physico-chemical properties as compared to those of the parent sulfonamide. This phenomenon is certainly governed by the strong polarization induced by the metal ions. In this way, it is quite probable that the right balance between the lipo- and hydrosolubility of these compounds is achieved, which has been considered to be the critical factor for not observing topical activity in the classical CA inhibitors, such as acetazolamide, methazolamide and ethoxzolamide, which were either too lipophilic or too hy.drosoluble [2,3]. So, by choosing different metal ions and diverse sulfonamides, much larger possibilities arise to finely tune the pharmacological properties which strongly influence the value of a drug. In conclusion we describe here a novel class of lOP lowering agents, ie, the metal complexes of sulfonamide CA inhibitors. These derivatives appear to be very active and longer lasting than the drug dorzolamide, and might constitute the premises for a new generation of antiglaucoma drugs. <br><br> Acknowledgments. We are grateful to Prof. S. Lindskog (Umea Univ., Sweden) for the gift of the hCA and II plasmids. References Preceding part of this series" Supuran CT, Scozzafava A, Ilies MA, Iorga B, Cristea T, Chiraleu F, Banciu MD (1997) Eur J Med Chem, submitted. 2 Supuran CT (1994) "Carbonic anhydrase inhibitors" In Carbonic Anhydrase and Modulation of Physiologic and Pathologic Processes in the Organism, (Puscas I, Ed) Helicon, Timisoara, pp. 29-111. 3 Maren TH (1991) "The links among biochemistry, physiology and pharmacology in carbonic anhydrase mediated systems". In Carbonic Anhydrase From Biochemistry and Genetics to Physiology and Clinical Medicine, (Botr6 F, Gros G, Storey BT Eds) VCH, Weinheim, pp. 186-207. 4 Bayer A, Ferrari F, Maren TH, Erb C (1996) dFr Ophtalrnol 19, 3:57-362. :5 Maren TH, Conroy CW, Wynns GC, Levy NS (1997) d Ocul Pharrnacol Therapeut 13, 23-30. 6 Sugrue MF (1996) d Ocul Pharmacol Ther 12,363-376. 7 Bartlett JD, Jaanus SD (1989) "Carbonic anhydrase inhibitors". In Clinical Ocular Pharmacology, Second Edition, Butterworths Publishers, Boston, pp. 2:54-263. 8 Alward PD, Wilensky JT (1981) Arch Ophthalmo199, 1973-1976. <br><br> 314 <br><br> Claudiu T. Supuran et al. <br><br> Metal-Based Drugs <br><br> 9 Maren TH, Jankowska L, Sanyal G, Edelhauser HF (1983) Exp Eye Res 36, 457-480. 10 a) Ponticello GS, Freedman MB, Habecker CN, Lyle PA, Schwam H, Varga SL, Christy ME, Randall WC, Baldwin JJ(1987) J Med Chem 30, 591-597; b) Greer J, Erickson JW, Baldwin JJ, Varney MD (1994) J Med Chem 37, 1035-1054. 11 Baldwin JJ, Ponticello GS, Anderson GS, Christy ME, Murcko MA, Randall WC, Schwam H, Sugrue MF, Springer JP, Gautheron P, Grove J, Mallorga P, Viader MP, McKeever BM, Navia MA (1989) J Med Chem 32, 2510-2513. 12 Chow K, Lai R, Holmes JM, Wijono M, Wheeler LA, Garst ME (1996) EurJMed Chem 31, 175-186. 13 Supuran CT, Nicolae A, Popescu A (1996) Eur JMed Chem 31, 431-438. 14 Supuran CT, Popescu A, Ilisiu M, Costandache A, Banciu MD (1996) Eur JMed Chem 31, 439-448. 15 Supuran CT, Scozzafava A, Popescu A, Bobes-Tureac R, Banciu A, Creanga A, Bobes-Tureac G, Banciu MD (1997) Eur JMed Chem 32, 445-452. 16 Alzuet G, Casanova J, Ramirez JA, Borras J, Carugo O (1995) J Inorg Bioehem 57, 219-234. 17 Sumalan SL, Casanova J, Alzuet G, Borras J, Castifieiras A, Supuran CT (1996) J Inorg Biochem 62, 3139. 18 a) Supuran CT (1995) Metal Based Drugs 2, 327-330; b) Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 19-26. 19 a) Borras J, Cristea T, Supuran CT (1996), Main Group Met Chem 19, 339-346; b) Jitianu A, Ilies MA, Briganti F, Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 1-7. 20 a) Supuran CT, Scozzafava A (1997) J Enzyme Inhibition 12, 37-51; b) Supuran CT, Briganti F, Scozzafava A (1997) J Enzyme Inhibition 12, 175-190. 21 Mincione G, Scozzafava A, Supuran CT (1997) Metal Based Drugs 4, 27-34. 22 Alzuet G, Ferrer S, Borras J, Supuran CT (1994) Roum Chem Quart Rev 2, 283-300. 23 Briganti F, Mangani S, Orioli P, Scozzafava A, Vernaglione G, Supuran CT (1997) Biochemistry 36, 10384-10392. 24 Borras J, Casanova J, Cristea T, Gheorghe A, Scozzafava A, Supuran CT, Tudor V (1996) Metal Based Drugs 3, 143-148. 25 Supuran CT, Mincione F, Scozzafava A, Briganti F, Mincione G, Ilies MA (1997) Eur J Med Chem, in <br><br> press. 26 Supuran CT, Scozzafava A, Saramet I, Banciu MD (1997) J Enzyme Inhibition, in press. 27 Jitianu A, Ilies MA, Scozzafava A, Supuran CT (1997) Main Group Met Chem 20, 151-156. 28 Forsman C, Behravan G, Osterman A, Jonsson BH (1988) Acta Chem Scand B42, 314-318. 29 Behravan G, Jonasson P, Jonsson BH, Lindskog S (1991) Eur JBiochem 198, 589-592. 30 Khalifah RG, Strader DJ, Bryant SH, Gibson SM (1977) Biochemistry 16, 2241-2247. 31 Nyman PO, Lindskog S (1964) Biochim Biophys Acta 85, 141-151. 32 Henderson LE, Henriksson D, Nyman PO (1976)JBiol Chem 251, 5457-5463. 33 Maren TH, Wynns GC, Wistrand PJ (1993) Mol Pharmaco144, 901-906. 34 Young RW, Wood KH, Vaughan JR, Anderson GW (1956) J Am Chem Soc 78, 4649-4654. 35 Pocker Y, Stone JT (1967) Biochemistry 6, 668-678. 36 Baird TT, Waheed A, Okuyama T, Sly WS, Fierke CA (1997) Biochemistry 36, 2669-2678. 37 Maren TH, Bar-Ilan A, Conroy CW, Brechue WF (1990) Exp Eye Res 50, 27-36. 38 Maren TH, Brechue WF, Bar-Ilan A (1992) Exp Eye Res 55, 73-79. 39 Brechue WF, Maren TH (1993) Invest Ophthalmol Vis Sci 34, 2581-2587. 40 Borja P, Alzuet G, Server-Carri6 J, Borras J, Supuran CT (1997) J Biol Inorg Chem (JBIC), in press. 41 Supuran CT, Balaban AT, Gheorghiu MD, Schiketanz A, Dinculescu A, Puscas I (1990) Rev Roum Chim 35, 399-405. 42 Supuran CT, Scozzafava A (1997) unpublished results. <br><br> Received: September 24, 1997 Accepted: October 17, 1997 Received in revised camera-ready format" October 23, 1997 <br><br> 315 <br><br>  <h3>pmcA2365225</h3>Synthesis, Characterization and Antitumour Activity of Some Butyltin(IV) Cysteaminates and N,N-Dimethylcysteaminates
Abstract
The synthesis and characterization of four di- and tri-n-butyltin cysteaminates and N,N-dimethylcysteaminates and three protonated / quaternized derivatives are reported. They all exhibit moderate or high in vitro cytotoxic activity. Six of seven compounds presented in this work are more active than cisplatin, etoposide and 5-fluorouracil, but less active than methotrexate and doxorubicin.
Metal Based Drugs <br><br> Vol. 7, Nr. 5, 2000 <br><br> SYNTHESIS, CHARACTERIZATION AND ANTITUMOUR ACTITY OF SOME BUTYLTIN(IV) CYSTEAMINATES AND N,N-DIMETHYLCYSTEAMINATES Marcel Gielen Karel Handlir.2, Martin Hollein and Dick de Vos 2 3 <br><br> Departement of General and Organic Chemistry AOSC, Faculty of Applied Sciences, Free University of Brussels VUV, Pleinlaan 2, B-1050 Brussels, Belgium Departement of General and Inorganic Chemistry, Faculty of Chemical Technology, University of Pafdubice, Nam. Cs. legii 565, 53210 Pardubice, Czech Republic Pharmachemie B. V., Haarlem, the Netherlands Abstract. The synthesis and characterization of four di- and tri-n-butyltin cysteaminates and_ N,N-dimethylcysteaminates and three protonated / quaternized derivatives are reported. They all exhibit moderate or high in vitro cytotoxic activity. Six of seven compounds presented in this work are more active than cisplatin, etoposide and 5-fluorouracil, but less active than methotrexate and doxorubiein. Introduction. <br><br> Several organotin cysteaminates (aminoethylthiolates) have been studied 1-4 as compounds with potentially high biological activity, but little is known about their cytotoxic effect. The present paper reports the synthesis and characterization of di- and tri-n-butyltin cysteaminates and their N,N-dimethyl analogues and their in vitro cytotoxicity. It is shown that a correct evaluation of the biological activity of organotin compounds remains 5 often hampered by their low water solubility. Therefore, we focused on the synthesis of derivatives with potentially higher water solubility by introducing ionic groups into the molecule by protonation or quatemization of a nitrogen atom. Results and discussion. 3 <br><br> 2 <br><br> 4 <br><br> (n-Bu)n Sn [SCH2-CH2-NHm(CH3)2-m]4-n <br><br> n=3, n=3, n=2, n=2, <br><br> m=2 m=O m=2 m=O <br><br> (1) (2) (3) (4) <br><br> 3 <br><br> 2 <br><br> 4 <br><br> (n-Bu)3 Sn S-CH2-CH2-NHm(CH3)3.m X <br><br> m=3, x=cn3so3 (5) m 1, X=CH3SO3 (6) m 0, X I (7) <br><br> n-Bu CH3-CH2-CH2-CH2 <br><br> (2), di-nTri-n-butyltin cysteaminate (1), tri-n-butyltin N,N-dimethylcysteaminate butyltin bis(cysteaminate) (3) and di-n-butyltin bis(N,N-dimethylcysteaminate) (4) were prepared by the reaction of the sodium cysteaminate (prepared in situ by the reaction of sodium methylate with the hydrochloride of eysteamine) with the suitable di-or tri-n-butyltin chloride in chloroform following reference by the reactions of the The methanesulphonates 5 and 6 were respectively synthesized equimolar amount of methane sulphonic acid in unprotonated analogs 1 and 2 with an chloroform. After purification on a short column of alumina and evaporation of the solvent, colourless viscous oils were obtained. The quaternized methyl derivative of 2, the methiodide (7), was prepared by its reaction with methyl iodide in benzene and purified by recrystalization from chloroform. 233 <br><br> M. Gielen, K. Handlir, M. Hollein and D. de Vos Synthesis, Characterization and Antitumor Activity ofSome Butyltin(IV) Cysteaminates and N,N-Dimethylcysteaminates <br><br> 86 77 85 3 (C4H9)2Sn(SCH2CH2NH2)2 90 4 (C4Hg)zSn(SCH2CHzN(CH3)2)2 92 oil 5 (C4H9)3 SnSCH2CH2NH3 CH3SO3 94 6 oil (C4H9)3 SnSCH2CH2NH(CH3)2 CH3SO3 118-120 74 7 (C4H9).SnSCH.CH.N(CH3)31 The elemental analyses (C, H, N, Sn) agree satisfactorily with the proposed formul <br><br> Table 1. Melting points and yields for..c0mP0unds 1 ()rmula ,2 comp. 1 (C4H9)3SnSCHzCH2NH2 2 (C4H9)3SnSCHzCH2N(CH3)2 <br><br> 7. <br><br> m.p., ..b.p.., .C 120-123/40 Pa 129-131/35 Pa 167-169/35 Pa 172-174/40 Pa <br><br> 'o' <br><br> yield,.% <br><br> Compounds 1- 4, and 7, are poorly soluble in aqueous solutions (< mg/mL). Compounds 5 and 6 exhibit higher water solubilities (ca. 2 mg/mL). Compounds 1 7 are soluble in a therapeutic solution (0,103 M solution of NaCl in water/DMSO 1/9). A white turbidity appears in these solutions only after several hours due to slow hydrolysis. Compounds 1 7 were characterized by multinuclear NMR spectroscopy. The NMR parameters are given in Table 2. Table 2. Resonances of 13C, (ca 20% sol. v/v) 1' compound <br><br> SN and ll9Sn NMR spectra of 2 <br><br> the compounds 1-7 in CDCI3 5 86.3 13.45 <br><br> 6' 'S'n), "ppm 6(3c)a, ppm <br><br> x <br><br> 77.6 12.3 <br><br> 77.5 12.75 <br><br> 3 -28.9 22.38 <br><br> 38.' 18.78 <br><br> 4 <br><br> 6' '89.5 13.45 <br><br> 7 93.2 13.60 <br><br> (332) 27.93 <br><br> (331) 27.97 <br><br> (495) 28.27 <br><br> (438) 27.51 <br><br> (322) 28.30 <br><br> (350) 28.34 <br><br> (339) 28.47 <br><br> (21.3) 26.32 <br><br> (21.1 ) 26.38 <br><br> (28.4) 26.42 <br><br> (26.4) 25.90 <br><br> (21.4) 26.85 <br><br> (21.3) 26.81 <br><br> (22.0) 26.91 <br><br> (59.6) 12.89 <br><br> (59.8) 12.95 <br><br> (87.1) 13.35 <br><br> (83.8) 12.82 <br><br> (62.7) 13.45 <br><br> (61.4) 13.58 <br><br> 62.2) 14.00 <br><br> (-) 30.31 45.02 <br><br> (-) 23.59 63.12 44.71 <br><br> (-) 30.04 44.44 <br><br> (-) 23.84 61.84 44.46 <br><br> (-) 23.92 43.08 <br><br> (-; <br><br> (-) <br><br> 20.94 19.70 60.94 69.49 43.42 54.18 39.15 39.16 -360.4 -354.0 -355.3 -354.4 -349.4 -344.2 -325.8 1.1 6.2 1.5 1.5 12.7 11.7 30.1 13 a'values of nJ(n J 9 Sn, ,C) c0uplihg constants in Hz'for Carbon ato]ns in parentlee's difference btween (N) for the organo.i compound and free cysteamine, 6(N) 361.5 ppm; N,N-dimethylcysteamine, 5(N) -355.9 ppm) not observed <br><br> C resonances were assigned on the basis of the values of n J( 119 Sn, 13 C) coupling constants and standard 13 C- APT techniques utilization in agreement with ref.. The values of 119 Sn chemical shifts are found in the interval characteristic for fourcoordinated tin 6,7. The values of the coupling constants j(1 lqSn,13C) agree with the structure proposed. Compound 3 is characterized by a somewhat larger upfield shift, close to the upper limit of the above mentioned interval, together with a high value of 1j(l lqSn,13C) (495 Hz). 67 The C-Sn-C angle, est'mated, from this value of coupling constant is 124 According to ref. this behaviour can be due to intermolecular association increasing the coordinatitn number of tin in concentrated solutions, as suggested by the concentration dependence of tS( Sn) and cryoscopic measurements in benzene. The values of the 6(SN)chemical shifts of compounds 1, 2 and 4 only slightly differ from the (15N) values of the free cysteamine. The significantly larger i(15N) chemical shift of compound 3 is however much lower than those of compounds 5, 6 and 7 containing a N interaction in compounds 1 tetracoordinated nitrogen atom. It can be stated that the Sn 4 is negligible or only very weak in chlorofom solutions. This conclusion is not in contradiction with the proved Sn N interaction found in the solid state 8. 13 <br><br> . <br><br> , <br><br> <br><br> <br><br> 234 <br><br> Metal Based Drugs <br><br> Vol. 7, Nr. 5, 2000 <br><br> Antitumour activity. <br><br> The results of the in vitro antitumour tests of compounds 1 7 are given in Table 3 as the inhibition doses IDs0 observed against a panel of seven <span class="red">human</span> tumour cell lines, MCF-7 and EVSA-T, two breast cancers, WiDr, a colon cancer, IGROV, an ovarian cancer, M19 MEL, a melanoma, A248, a renal cancer, and H226, a non-small cell lung cancer. The antitumour tests results are compared with those obtained for clinically used reference compounds9 like cisplatin, doxorubicin, etoposide, 5-fluorouracil and methotrexate. Table 3. Inhibition doses <br><br> comp. 1 2 3 4 5 6 7 <br><br> MF-7 41 41 51 78 30 90 385 699 10 2 594 750 18 <br><br> EVSA-T 33 35 45 65 <3 57 367 <br><br> IDs0 of compounds WiDr 39 39 211 331 19 40 339 <br><br> 1- 7 and of five reference compounds. IGROV M19MEL A498 H226 67 44 85 39 46 78 88 39 68 70 118 72 110 129 108 112 31 29 36 42 83 133 77 111 327 758 576 411 169 60 580 297 7 558 16 505 442 23 <br><br> cisplatin doxorubicin <br><br> etoposide 5-fluorouracil <br><br> 422 8 317 475 5 <br><br> methotrexate <br><br> 967 11 150 225 <3 <br><br> 2 253 90 314 143 37 <br><br> 3 369 199 3 934 340 2 287 <br><br> It is evident from Table 3 that all studied compounds exhibit excellent cytostatic activities, except compound 7 showing only moderate activity. Compounds 1 6 are more active than cisplatin and, in most cases, than etoposide and 5-fluorouracil. Their activity is lower than that of doxorubicin and methotrexate in most cases for the studied cancer cell lines. The very limited set of acquired activity values show that the activity of tri-n-butyltin are higher than those of di-n-butyl analogues (1>3, 2>4) for all cell lines, especially compounds against WiDr. The N,N-dimethyl derivatives exhibit an activity comparable to or somewhat higher than that of unmethylated compounds. Compound 8, with a quaternized nitrogen, exhibits the lowest cytotoxic activities, approximatively 9 times lower than compound 2. It might be underlined that the ionic more water-soluble compounds 5 and 6 are not more active than the uncharged analogs 1-4, whereas the hydrophilic organotin polyoxacarboxylates exhibit much higher in vitro activities than other organotin carxylates 51 Compounds 5 and 6 (with a protonated nitrogen) exhibit activities comparable or even considerably higher (compound 5)against EVSA-T than the corresponding unprotonated compounds. Compound 7 (with a quaternized nitrogen) exhibits the lowest cytostatic activities, approximately 9 times should however be mentioned that another ionic organotin lower than compound 2. It compound, bis(dicyclohexyl)ammonium bis(2,6-pyridinecarboxyla,to)di-n-butylstannate, where the organotin moiety is this time anionic, is also as active as the uncharged analog l. All NMR spectra were recorded on a Bruker AMX 360 instrument using a 5 mm multinuclear tuneable probe. The residual CHCI3 resonance at 7.24 ppm was used as reference for the H soectra and the central 13CDCI3 resonance at 77.0 ppm, for the 13C spectra. The 119 Sn chemical shifts were refered to the external tetramethyltin [6( 119 Sn) 0.0]. The 15N NMR spectra were measur;l using the INEPT technique or in inverse-gated mode. (reference: Instrumentation. <br><br> external nitromethane, [5('N) 0.0]). The protocol followed for the in vitro antitumour screenings has been already reported 10. <br><br> Acknowledgements. K. H. and M H. thank the Grant Agency of the Czech Republic (Grant No. 203/00/0920) and the Ministry o" Education, Youth and Sport of the Czech republic, associated with EU in the COST 8.20 program for financial support of this work. We are grateful to Mr. R. G. Experimental Oostrum, Dr. J. verweij, Prof. Dr. G. Stoter, r.K. Nooter, Laboratory of Chemotherapy and Pharmacology, Department of Medical Oncology, Rotterdam Cancer 235 <br><br> M. Gielen, K. Handlir, M. HoHein and D. de Vos Synthesis, Characterization and Antitumor Activity ofSome Butyltin(IV) Cysteaminates and N,N-Dimethylcysteaminates <br><br> Institute, NL 3008 AE, Rotterdam, The Netherlands, for performing the in vitro tests. This research was supported by the Fund for Scientific Research Flanders (Belgium), grant nr G.0074.00, M. G.). <br><br> References. 1. B.S. Saraswat, J. Mason, Polyhedron(9), 5 (1986), 1449. 2. G. Domazetis, R. J. Magee, B. D. James, J. Organomet. Chem, 162 (1978), 239. 3. J.D. Cashion, G. Domazetis, B. D. James, J. Organomet. Chem, 185 (1980), 433. 4. G. Domazetis, R. J. Magee, B. D. James, J. Organomet. Chem, 148 (1978), 339. 5. G. Atassi, Rev. Si Ge Sn Pb Cpds, 8 (1985), 219; M. Kemmer, M. Gielen, M. Biesemans, D. de Vos, R. Willem, Metal-Based Drugs 5 (1998), 189; M. Gielen, M. Biesemans, D. de Vos, R. Willem, J. Inorg. Biochem., 79 (2000), 139. 6. J. Holecek, A. Lycka, Inorg. Chim. Acta, 118 (1986), L 15. 7. J. Holecek, M. Nadvomik, K. Handlir, A. Lycka, J. Organomet. Chem., 315 (1986), 299. 8. B.D. James, R. J. Magee, W. C. Patalinghug, B. W. Skelton, A. H. White, J. Organomet. Chem, 467 (1994), 51 9. P. Skehan, R. Storeng, D. Scudiero, A. Monks, J. McMahon, D. Vistica, J. T. Warren, H. Bokesh, S. Kenney, M. R. Boyd, J.. Natl. Cancer lnst., 82 (1990), 1107. 10. Y. P. Kepers, G. J. Peters, J. Van Ark-Otte, B. Winograd, H. M. Pinedo, Eur. J. Cancer, 27 (1991), 897. 11. M. Kemmer, L. Ghys, M. Gielen, M. Biesemans, E. R. T. Tiekink, R. Willem, J. Organomet. Chem. 582 (1999), 195. 12. S. W. Ng, V. G. Kumar Das, J. Holecek, A. Lycka, M. Gielen, M. G. B. Drew, Appl. Organomet. Chem. 11 (1997), 3 9. <br><br> Received" September 22, 2000 Accepted- October 19, 2000 Received in revised camera-ready format- October 20, 2000 <br><br> 236 <br><br>  <h3>pmcA2398786</h3>Genomic-Bioinformatic Analysis of Transcripts Enriched in the Third-Stage Larva of the Parasitic Nematode <span class="yellow">Ascaris suum</span>
Abstract
Differential transcription in <span class="yellow">Ascaris suum</span> was investigated using a genomic-bioinformatic approach. A cDNA archive enriched for molecules in the infective third-stage larva (L3) of <span class="yellow">A. suum</span> was constructed by suppressive-subtractive hybridization (SSH), and a subset of cDNAs from 3075 clones subjected to microarray analysis using cDNA probes derived from RNA from different developmental stages of <span class="yellow">A. suum</span>. The cDNAs (n = 498) shown by microarray analysis to be enriched in the L3 were sequenced and subjected to bioinformatic analyses using a semi-automated pipeline (ESTExplorer). Using gene ontology (GO), 235 of these molecules were assigned to ‘biological process’ (n = 68), ‘cellular component’ (n = 50), or ‘molecular function’ (n = 117). Of the 91 clusters assembled, 56 molecules (61.5%) had homologues/orthologues in the free-living nematodes <span class="yellow">Caenorhabditis elegans</span> and <span class="yellow">C. briggsae</span> and/or other organisms, whereas 35 (38.5%) had no significant similarity to any sequences available in current gene databases. Transcripts encoding protein kinases, protein phosphatases (and their precursors), and enolases were abundantly represented in the L3 of <span class="yellow">A. suum</span>, as were molecules involved in cellular processes, such as ubiquitination and proteasome function, gene transcription, protein–protein interactions, and function. In silico analyses inferred the <span class="yellow">C. elegans</span> orthologues/homologues (n = 50) to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%), or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway. Small numbers of these 50 molecules were predicted to be secreted (10%), anchored (2%), and/or transmembrane (12%) proteins. Functionally, 17 (34%) of them were predicted to be associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority being embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%). A genetic interaction network was predicted for these 17 <span class="yellow">C. elegans</span> orthologues, revealing highly significant interactions for nine molecules associated with embryonic and larval development (66.9%), information storage and processing (5.1%), cellular processing and signaling (15.2%), metabolism (6.1%), and unknown function (6.7%). The potential roles of these molecules in development are discussed in relation to the known roles of their homologues/orthologues in <span class="yellow">C. elegans</span> and some other nematodes. The results of the present study provide a basis for future functional genomic studies to elucidate molecular aspects governing larval developmental processes in <span class="yellow">A. suum</span> and/or the transition to parasitism.<br><br>Introduction
Parasitic nematodes are of major socio-economic importance in animals. For example, hundreds of millions of <span class="yellow">people</span> are infected with geohelminths (soil-transmitted worms), such as blood-feeding hookworms <span class="yellow">Ancylostoma duodenale</span> and/or <span class="yellow">Necator americanus</span>, <span class="yellow">Trichuris trichiura</span> and Ascaris spp. [1], causing serious adverse effects on <span class="blue">human</span> health, particularly in <span class="yellow">children</span>. Similarly, parasitic nematodes of livestock, such as <span class="yellow">pigs</span>, also cause substantial economic losses due to subclinical and clinical diseases, with billions of dollars spent annually on the treatment and control of gastro-intestinal nematodes. In addition to the socioeconomic impact that these parasites have, there is potential for the emergence of resistance in them against all of the main classes of (nematocidal) compounds used to treat the diseases they cause [2]–[5]. Therefore, there is a significant need to work toward discovering new compounds to control these parasites. Gaining an improved understanding of the molecular basis of parasite development provides such an avenue.
Compared with the free-living nematode <span class="yellow">Caenorhabditis elegans</span>, there is very little information on fundamental molecular aspects of development in parasitic nematodes [6]–[8]. Since the genome sequence of <span class="yellow">C. elegans</span> was published in 1998 [9], many aspects of the molecular biology of this nematode have been elucidated. For instance, microarray analyses have been used to examine developmental and gender-enriched gene expression [10],[11], and the functions of more than 96% of the <span class="yellow">C. elegans</span> genes have been assessed by double-stranded RNA interference (RNAi, or gene silencing; [12]) [13]–[18]. Comparative analyses of genetic data sets have shown that parasitic nematodes usually share ∼50–70% of genes with <span class="yellow">C. elegans</span> (e.g., [19],[20]). There is similarity in other features (such as basic body plan and moulting) between <span class="yellow">C. elegans</span> and parasitic nematodes, suggesting that some molecular pathways are relatively conserved [8],[21]. Understanding the pathways linked to basic nematode biology and development could have important implications for finding new ways of disrupting these pathways and thus facilitate the identification of new drug targets.
Despite the advances in genomic technologies [7], [22]–[29] and the study of <span class="yellow">C. elegans</span>, there is a paucity of information on the genomics of parasitic nematodes of animals, particularly in relation to development. Also considering the major socioeconomic impact of Ascaris and ascariasis in <span class="yellow">humans</span> and <span class="yellow">pigs</span> [30]–[32], several characteristics, including the large size of the adult worm (providing the opportunity of investigating individual organ systems and tissues), the ability to maintain Ascaris in the <span class="yellow">pig</span>, store eggs and culture larvae in vitro for relatively long periods of time (months to years) [32] as well as the discovery that RNAi achieves “cross-species” gene silencing for a selected number of genes [33],[34] and the imminent genome sequence (http://www.sanger.ac.uk/Projects/Helminths/) all indicate that Ascaris could serve as a powerful model system for investigating reproductive and developmental processes in nematodes.
In the present study, Ascaris from <span class="yellow">pigs</span> was used to study molecules abundantly transcribed in the infective third-stage larva (L3). Following the oral ingestion of Ascaris eggs by the host (<span class="blue">human</span> or <span class="yellow">pig</span>), L3s are released and then invade/penetrate predominantly the caecal wall [35] to then undergo hepato-pulmonary migration, after which ultimately the adult females and males establish and develop in the small intestine [36],[37]. The molecular mechanisms linked to host invasion and parasite development are largely unknown. Here, we constructed an L3-enriched cDNA library using the method of suppressive-subtractive hybridization (SSH), explored transcription of a representative subset of molecules by microarray analysis and conducted bioinformatic analyses to characterize these molecules, map them to biochemical pathways and predict genetic interactions based on comparisons with <span class="yellow">C. elegans</span> and/or other organisms.<br><br>Materials and Methods
Production of Different Developmental Stages of Ascaris
Experimental <span class="yellow">pigs</span> (8–12 weeks of age) were purchased from and maintained in the Experimental Animal Center of South China Agricultural University. These <span class="yellow">pigs</span> were treated humanely, according to the Animal Ethics procedures and guidelines of the People's Republic of China. Adult <span class="yellow">worms</span> (males and females) of <span class="yellow">A. suum</span> were collected from the small intestines of <span class="yellow">pigs</span> from an abattoir in Shenzhen, China. Infective eggs and infective L3s of <span class="yellow">A. suum</span> were produced according to the methods described previously [38]. In brief, eggs from the uteri of adult females of <span class="yellow">A. suum</span> were collected and incubated at 28°C for 28 days to allow them to develop to infective eggs (containing infective L3s). To obtain pure infective L3s, 7.5% v/v sodium hypochlorite was used to treat the larvated eggs at 37°C overnight and then the eggs were shaken with glass-beads; then, the exsheathed L3s and shells were separated by density gradient centrifugation using lymphocyte separating medium (LSM) [38]. Following the experimental infection of helminth-free <span class="yellow">pigs</span> with infective Ascaris eggs as described previously [39], the L3s from livers and in lungs as well as L4s in intestines were isolated according to an established method [40]. All parasite materials were snap-frozen in liquid nitrogen prior to storage at −70°C.<br><br>Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)
Total RNA was isolated from adult females and males, different larval stages or eggs of <span class="yellow">A. suum</span> using TriPure reagent (Roche) as recommended by the manufacturer. Equal amounts of total RNA from each stage or sex were pooled. The mRNA was isolated using the Oligotex mRNA Kit (Qiagen), following the manufacturer's protocol. SSH was carried out using the PCR-Select cDNA Subtraction kit (Clontech), according to the manufacturer's protocol. In brief, cDNA synthesized from mRNAs from infective L3s was subtracted against cDNA synthesized from the pooled mRNA from all other stages included herein. The SSH library was constructed using infective L3s as the tester and pooled cDNAs from all other stages as the driver. The effectiveness of this subtraction process has already been demonstrated in previous studies [41],[42]. The cDNA obtained following SSH was cloned into the pGEM-T Easy plasmid vector (Promega) and competent <span class="yellow">Escherichia coli</span> (JM109) transformed. Positive clones, picked randomly (based on blue/white selection), were grown overnight in Luria Bertani (LB) medium (shaking, 37°C). Individual inserts were PCR-amplified using “nested primers” 1 and 2R from the Subtraction kit (Clontech) and examined by agarose electrophoresis.<br><br>Preparation of Microarray Slides
Clones (n = 3075) from the subtracted library were picked and cultured overnight in LB containing ampicillin (1000 IU/ml) in sealed 96-well blocks. Five µl of culture suspension from each well were transferred into individual wells thermocycling (96-well) plates and the inserts PCR-amplified using primers 1 and 2R. Following a 10 min denaturation step at 94°C, the amplification proceeded for 25 cycles of 10 s at 94°C, 30 s at 68°C and 1.5 min at 72°C, with a final extension for 5 min at 72°C. Products were resolved in agarose gels, ethanol precipitated, re-suspended in 16 µl of “spotting solution” (Shanghai BioStar Genechip, Inc) to a final concentration of ∼500 ng per µl, before being printed on to glass slides (in duplicate) using a robotic arrayer. Sixteen blanks (using spotting solution only) and the same number of negative (irrelevant cDNAs with no relationship to Ascaris) were also printed on to slides and served as negative controls; β-actin of <span class="yellow">A. suum</span> served as a positive control to assess the efficiency of labeling and hybridization. The slides were air-dried for 2 h, and cDNA in the spots were cross-linked at 254 mJ. The printed slides were stored at 4°C.<br><br>Labeling of cDNA Probes with Fluorescent Dyes, and Microarray Analysis
The cDNAs produced from total RNA from <span class="yellow">A. suum</span> eggs, infective L3s, L3s isolated from <span class="yellow">pig</span> liver or lung, fourth-stage larvae (L4s), adult males or females [as described in the section ‘Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)’] were labeled with cyanine dyes. Cy3 or Cy5-dCTP was incorporated into cDNA produced from 30 µg of total RNA by direct labeling in a reverse transcription reaction using an oligo (dT) primer. Labeled cDNA was purified using DyeEx columns (Qiagen).
Microarray slides were incubated with a pre-hybridization solution [5×SSC, 1% <span class="yellow">bovine</span> serum albumin (BSA), 0.1% sodium dodecyl-sulphate (SDS)] for 6 h at 42°C. After pre-hybridization, the microarray slides were incubated with ‘pooled’ Cy3 and Cy5-labeled probes in hybridization solution (5×SSC, 1% BSA, 0.1% SDS), in the dark at 42°C for 18 h, and then washed in solution I (1×SSC, 0.2% SDS) for 10 min, followed by solution II (0.1×SSC, 0.2% SDS) for 10 min at 60°C, according to the protocols provided by Shanghai BioStar Genechip, Inc. A “dye flip” was carried out to control for any bias in hybridization signal between the Cy-labeled cDNA probes (produced for two distinct mRNA populations). The slides were dried and scanned (ScanArray 4000 scanner) using image acquisition software (Shanghai BioStar Genechip Inc.) and a range of laser power and photo-multiplier tube intensities. The mean hybridization signal (derived from four replicates of the same array) were corrected for background, normalized [43], log2-transformed and then subjected to statistical analysis employing the students t-test in a spreadsheet (Excel, Microsoft, USA). The microarray data were analysed for differential cDNA hybridization (>2.0-fold to 114.3-fold) between L3 and each of the other stages (eggs, lung and liver L3s, L4, adult female and adult male).<br><br>Verification of Differential Hybridization by Reverse Transcription-Coupled Polymerase Chain Reaction (RT-PCR) Analysis
For a subset (n = 17) of representative ESTs (rESTs), RT-PCR was used to verify the differential transcription recorded by microarray analysis. Double-stranded cDNA was synthesized from total RNA (separately) from each stage or sex of <span class="yellow">A. suum</span> using reverse transcriptase (Superscript III, Invitrogen). Briefly, 5 µg of total RNA were added to 14 µl of H2O and 1 µl of oligo d(T)n = 12–18 primer (0.5 µg/µl), heated to 70 °C for 10 min and chilled on ice. First- and second-strand cDNAs were synthesized via the addition of 4 µl of first-strand cDNA buffer (250 mM Tris-HCl, pH 8.3, 375 mM KCl and 15 mM MgCl2), 2 µl of 0.1 M dithiothreitol, and 1 µl of 10 mM of each dNTP, followed by an incubation at 25 °C (10 min), 42 °C (50 min) and 70 °C (15 min). One-tenth of each double-stranded cDNA produced was then used as a template in the PCR. The transcripts were amplified from individual cDNAs by PCR using oligonucleotide primers (sequences available upon request) designed to each EST. The PCR amplification of a portion (209 bp) of the β-actin gene (accession no. BI594141) using forward primer (5′-CTCGAAACAAGAATACGATG-3′) and reverse primer (5′- ACATGTGCCGTTGTATGATG-3′), previously determined to be present in all developmental stages and both sexes of <span class="yellow">A. suum</span> [44], served as a positive control. Samples without template (no-DNA controls) were included in each PCR run. The following cycling conditions were employed: one cycle at 94 °C (5 min), 94 °C (30 s), 60 °C (30 s) and 72 °C (30 s) for 30 cycles, followed by a final extension of 70 °C (7 min). Following the PCR, 5 µl of individual amplicons were resolved in ethidium bromide-stained agarose gels (2%) and then photographed upon transillumination. The relative band intensities were analyzed using UVIsoft Image Acquisition and Analysis software (UVITEC). The specificity and identity of individual amplicons were confirmed by direct sequencing using the same primers (separately) as employed for their amplification.<br><br>Sequencing and Bioinformatics Analyses
Clones from the SSH cDNA library with increased hybridization in microarray analysis to the infective L3 compared with other stages were sequenced using standard technology [45]. The nucleotide sequences have been deposited in the GenBank database under accession numbers ES290984-ES291074. Following the processing of the sequences (i.e., removal of vector sequences, quality assurance and clustering), contigs or singletons from individual clusters were subjected to BLASTx (NCBI: www.ncbi.nlm.nih.gov) and BLASTn (EMBL-EBI Parasite Genome Blast Server: www.ebi.ac.uk) analysis to identify putative homologues in <span class="yellow">C. elegans</span>, other nematodes and other organisms (e-value of ≤1e-05). Peptides inferred from ESTs were classified functionally using Interproscan (available at http://www.ebi.ac.uk/InterProScan/) employing the default search parameters. WormBase (www.wormbase.org) was interrogated extensively for relevant information on <span class="yellow">C. elegans</span> homologues/orthologues, including RNAi phenotypic, transcriptomic, proteomic and interactomic data. ESTs with homologues/orthologues in <span class="yellow">C. elegans</span> and other nematodes were also subjected to analysis employing the KEGG Orthology-Based Annotation System (KOBAS) (www.kobas.cbi.pku.edu.cn), which predicts the biochemical pathways in which molecules are involved. The open reading frames (ORFs) inferred from selected ESTs with orthologues in <span class="yellow">C. elegans</span> were also subjected to “secretome analysis” using the program SignalP v.2.0 www.cbs.dtu.dk/services/SignalP/), employing both the neural network and hidden Markov models to predict signal peptides and/or anchors [46]–[48]. Also, transmembrane domains were predicted using the program TMHMM (www.cbs.dtu.dk/services/TMHMM/; [49]–[51]), and subcellular localization inferred employing the program WoLF PSORT (http://wolfpsort.org/; [52]).
The method established by Zhong and Sternberg [53] was used to predict the interactions for <span class="yellow">C. elegans</span> orthologues of the L3-enriched molecules from Ascaris. In brief, interaction, phenotypic, expression and gene ontology data from <span class="yellow">fruitfly</span>, <span class="yellow">yeast</span>, <span class="yellow">mouse</span> and <span class="blue">human</span> were integrated using a naïve Bayesian model to predict genetic interactions among <span class="yellow">C. elegans</span> genes ([45],[53]; Zhong and Sternberg, unpublished). The predicted networks resulting from the analyses were saved in a graphic display file (gdf) format and examined using the graph exploration system available at http://graphexploration.cond.org/. Images were labeled and saved in the joint photographic experts group (jpeg) format.<br><br>
Results
To identify molecules transcribed abundantly in the L3 of <span class="yellow">A. suum</span>, an enriched cDNA library was constructed by SSH. From a total of 3075 clones from this library, 2921 (95%) were shown to contain an insert (which could be amplified by PCR). From 2671 (92%) of these clones, amplicons representing single bands of ∼400 to 600 bp in size were produced. These latter amplicons were arrayed (in duplicate) on to slides and then hybridized with Cy3-labeled L3-cDNA or with Cy5-labeled cDNA from eggs, liver/lung L3s, L4s, adult female or adult male of Ascaris. Dye flip was conducted to verify the hybridization data. Of the 2671 (duplicate) spots, 1526 had a significant difference in hybridization between infective L3 cDNA and cDNAs from all other stages or sexes of <span class="yellow">A. suum</span>, of which 515 had a >2.0-fold increased hybridization for the L3.
In order to independently verify the hybridization results in the microarray, a PCR-based analysis of a selected subset (n = 17) clones was conducted using specific primer pairs. Having verified the specificity and identity of individual amplicons by sequencing, PCR results were reproducible (based on multiple runs on different days) and ∼94% (16 of 17) concordant with those of the microarray analysis (not shown). There was complete concordance for representative clones associated with a differential signal of ≥3.0-fold in the microarray.
The clones linked to the 515 spots representing increased transcription (>2.0-fold) in infective L3 compared with the other developmental stages or sexes included were subjected to sequencing. The 498 sequences (length: 550±115 bp) determined were then subjected to detailed bioinformatic analyses. There were 91 unique clusters (accession numbers ES290984-ES291074), of which 55 were singletons (sequences determined once). Of 56 molecules (61.5%) with significant similarity to sequences other than <span class="yellow">A. suum</span> in the databases interrogated, 50 (54.9%) had <span class="yellow">C. elegans</span> or <span class="yellow">C. briggsae</span> homologues, and six had similarity to ESTs already sequenced from ascaridoid and/or other parasitic nematodes, and/or other organisms (Table 1). A significant proportion (38.4%) did not have any similarity in sequence to any organisms for which data are presently available. Comparative analysis specifically against <span class="yellow">A. suum</span> EST data sets (n∼42,000) available in public databases confirmed independently that the majority of molecules (>60%) were present exclusively in the infective L3 stage or were orphans.
As gene ontology (GO) provides a hierarchy that unifies the descriptions of biological, cellular and molecular functions [54], this approach was employed to predict the classification and gene function of molecules enriched in infective L3 of <span class="yellow">A. suum</span>. A summary of the GO categories of these molecules is displayed in Fig. 1. Of the 91 contigs, 32 (35%) could be functionally assigned to ‘biological process’ (n = 38), ‘cellular component’ (n = 17) and ‘molecular function’ (n = 64). The most common subcategories were gluconeogenesis (13%) and metabolic process (13%) within ‘biological process’, extracellular region (24%) within ‘cellular component’, and catalytic activity (11%) and phosphoenolpyruvate carboxykinase activity (8%) within ‘molecular function’ (Table S1).
A focused KOBAS analysis inferred the 50 <span class="yellow">C. elegans</span> orthologues/homologues to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%) or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway (Table 2). Of these 50 molecules, small numbers were predicted to be secreted (10%), anchored (2%) and/or transmembrane (12%) proteins (Table 2). Functionally, 17 (34%) of the 50 molecules were associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority displaying embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%) (Table 2).
Extending this analysis, a relatively complex genetic interaction network was predicted for the 17 <span class="yellow">C. elegans</span> orthologues (i.e., with non-wild-type RNAi phenotypes) (see Table S2). Statistically highly significant interactions were predicted for nine of the <span class="yellow">C. elegans</span> genes; the top five interactors are displayed in Fig. 2. The gene ontology categories for eight selected <span class="yellow">C. elegans</span> genes (F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2) included: embryonic development, egg hatching, larval development and/or growth. The other categories included: positive regulation of growth rate (F55A12.8, kin-2, mup-2, pab-1, rpl-22 and T21B10.2) and gamete generation and locomotory behaviour (kin-2, mup-2, pab-1 and F55A12.8, kin-2, mup-2, respectively). The <span class="yellow">C. elegans</span> homologue egl-3 was predicted to be involved in proteolysis (see www.wormbase.org). All nine <span class="yellow">C. elegans</span> orthologues were predicted to interact directly with a total of 296 (range: 5–75) other genes and, in particular, a direct genetic interaction was predicted between pab-1 and T21B10.2 (Fig. 2). The 296 interactors were associated with embryonic and larval development (n = 198; 66.9%), information storage and processing (n = 15; 5.1%), cellular processes and signalling (n = 45; 15.2%) and metabolism (n = 18; 6.1%); the precise function of some of the interactors (n = 20; 6.7%) is presently unknown (Table S2).<br><br>Discussion
The present study investigated transcripts in infective L3s of <span class="yellow">A. suum</span> using a genomic-bioinformatic platform. The focus was on comparisons with <span class="yellow">C. elegans</span> homologues/orthologues, because the entire genome sequence of this nematode is known [9] and because there is a wealth of information on the localization and functionality of its molecules (www.wormbase.org; http://elegans.bcgsc.bc.ca/knockout.shtml). The functions of most genes in <span class="yellow">C. elegans</span> have been assessed using RNAi (e.g., [14],[15],[17],[55],[56]) in the hermaphroditic stage, whereas there is a paucity of functional information available for Ascaris and other parasitic nematodes of animals [57],[58].
Following the microarray analysis of >2500 ESTs from the SSH library, 498 cDNAs inferred to be enriched in the L3, based on hybridization signal, were sequenced and subjected to comprehensive in silico analyses. Of the 91 clusters of molecules categorized, 50 (54.9%) had <span class="yellow">C. elegans</span> homologues/orthologues with loss-of-function phenotypes could be mapped to key pathways. The statistically significant genetic interactions predicted for 9 of the 50 <span class="yellow">C. elegans</span> orthologues [namely egl-3, F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2 ( = enol-1)] and the interaction network included genes encoding kinases, alpha-tubulins, enolases, troponin and other named and unnamed proteins. Eight of these molecules (enol-1, pab-1, F33D11.10, rpl-22, F55A12.8 mec-12, mup-2 and kin-2) have known or predicted roles in embryonic and larval growth and development, gamete generation, locomotory behaviour or other biological processes in <span class="yellow">C. elegans</span> (see www.wormbase.org).
The enolase encoded by enol-1 is predicted to play a role in glycolysis, gluconeogenesis, phenylalanine, tyrosine and tryptophan biosynthesis (cf. [59]). Since glucose is the main source for ATP production, the alteration in these key glycolytic enzymes may lead to cellular dysfunction, such as impaired ion-motive ATPase required to maintain potential gradients, operate pumps and maintain membrane lipid asymmetry [60]. Bioinformatic analysis for transmembrane helices (TMHMM) and peptide signal sequences (SignalP) predicted ENOL-1 to be a non-secreted protein localized to the cytoplasm (cf. Table 2). Nonetheless, enolases are often detected in the excretory/secretory (ES) products of parasitic helminths, including adult <span class="yellow">A. suum</span> [61], and appear to play a role in the triggering of nitric oxide production by host cells. The enol-1 orthologue of <span class="yellow">C. elegans</span> has been predicted to interact specifically with the polyadenylate binding protein gene, pab-1, inferred to be involved in coordinated gene transcription and expression during normal larval development [16]. Poly(A)-binding proteins (PABPs) are recognized to be central to the regulation of mRNA translation and stability [62]. Present evidence suggests that the expression of PAB-1 is regulated by an oligo-pyrimidine tract in response to cell growth and relates to coordinated growth regulation in <span class="yellow">C. elegans</span> [62]. Furthermore, gene silencing of pab-1 and its selected interactors (see Fig. 2) leads to embryonic lethal (Emb), slow growth (Slo) and sterile progeny (Stp) phenotypes (see www.wormbase.org).
Another gene (F33D11.10; EST code 4F10; see Table 2) which encodes an ATP-dependent RNA helicase and is associated with embryonic lethal (Emb) and larval lethal (Lvl) RNAi phenotypes, was shown to be highly transcribed in infective L3s of <span class="yellow">A. suum</span>. Helicases are involved in a variety of RNA metabolic processes, including translation initiation, pre-mRNA splicing, pre-rRNA processing, rRNA maturation and RNA degradation [63], and are crucial for life cycle progression, sex determination and early embryogenesis in <span class="yellow">C. elegans</span> [60]. The high transcription levels of a homologue/orthologue in the L3 of <span class="yellow">A. suum</span> might suggest a similar role in this ascaridoid. Similarly, the coordination of the expression of a large number of genes is required for normal growth and cell proliferation during larval development. The high transcription level for the ribosomal protein gene homologue rpl-22 (large subunit family member; EST code 26G12, see Table 2) in the infective L3 of <span class="yellow">A. suum</span> compared with other developmental stages is likely to reflect the substantial rate of cell growth in this stage [64].
The gene (F55A12.8, EST code 4G11; see Table 2) encoding an acetyl-transferase with a putative ATPase domain, shown to be enriched in the L3 of <span class="yellow">A. suum</span>, was predicted to interact with 75 other genes all involved in energy production and/or RNA processing (see Table S2). Several molecules involved in ATP synthesis and mitochondrial pathways (e.g., cytochrome oxidase c subunits 1, 2 and 3, ADP/ATP translocases, NADH dehydrogenases, ATPases and ATP synthetases) have been reported previously to be highly represented in the L3 stage of <span class="yellow">Anisakis simplex</span> [65], thus supporting the proposal that substantial energy is required for larval development as well as the transition from the free-living to the parasitic stage and the invasion of the host. There is also likely to be a substantial energy requirement for muscle contraction linked to larval motility in <span class="yellow">A. suum</span>, as the L3s penetrate the caecal wall in the porcine host, before undergoing hepato-pulmonary migration [35]. Accordingly, genes encoding a specialized tubulin expressed in mechanoreceptors (mec-12, EST code 13E09) and a troponin (mup-2, EST code 01G03; see Table 2), both predicted to interact with a total number of 32 tubulin- and myosin-encoding genes, also supported a link to extensive muscle contraction and motility in <span class="yellow">A. suum</span> L3s. Also, neuroactive peptides are required to regulate the responsiveness of nematode larvae to mechanical stimuli [66]. A homologue encoded by egl-3 was shown to be highly transcribed in the L3 of <span class="yellow">A. suum</span>; EGL-3 is predicted to be a pro-hormone convertase involved in the maturation of neuropeptides and could be associated with mechano-sensory responses and touch sensitivity linked to the host invasion.
A regulatory subunit of a cAMP-dependent protein kinase (kin-2, EST code 22H01; see Table 2) was predicted to interact with 72 other genes all involved in diverse cellular processes, such as nuclear trafficking, and DNA replication and repair (see Table S2). Based on gene ontology terms, kin-2 is implicated in gamete generation, growth, larval development, post-embryonic body morphogenesis, signal transduction and/or protein amino acid phosphorylation (see Table S2). Gene silencing of kin-2 in <span class="yellow">C. elegans</span> leads to phenotypes, such as larval lethal (Lvl), larval arrest (Lva), body morphology defect (Bmd), dumpy (Dpy), uncoordinated (Unc) and sterile progeny (Stp) (www.wormbase.org), suggesting that its homologue in <span class="yellow">A. suum</span> is central to larval maturation. The KOBAS analysis predicted the protein KIN-2 to be involved in the insulin-signaling pathway, previously implicated in controlling the exit from dauer in <span class="yellow">C. elegans</span> and the activation of L3s of the <span class="yellow">canine hookworm</span>, <span class="yellow">Ancylostoma caninum</span>, following exsheathment [67]. In a recent study, Brand and Hawdon [68] were able to inhibit (with a phosphoinositide-3-OH-kinase inhibitor) the activation of infective L3s of both of the hookworms <span class="yellow">Ancylostoma caninum</span> and <span class="yellow">Ancylostoma ceylanicum</span> via the insulin signaling pathway, thus lending some credence to the hypothesis that this pathway plays an critical role in regulating the transition from the free-living to the parasitic stage [68]. Recently, it has been proposed that transcriptional and feeding responses to serum-stimulation in <span class="yellow">Ancylostoma caninum</span> are regulated by parallel systems, with the insulin signaling pathway playing a significant role in the ‘resumption of feeding’ in activated larvae [69].
Protein kinases are also likely to be involved in pathways linked to sexual maturation in developing larvae. As already proposed for adult stages of <span class="yellow">H. contortus</span> [45], the protein kinase gene cdk-1 is predicted to play a pivotal role in the germline, oogenesis and spermiogenesis pathways of this parasitic nematode. Other protein kinases, such as PEPCK, and phosphatases, were shown herein to be transcribed at high levels in the L3 stage compared with other developmental stages of <span class="yellow">A. suum</span> (see Table 2), which is in accordance to findings reported recently for <span class="yellow">Anisakis simplex</span> [65]. Due to their major regulatory effects in eukaryotic signaling events and regulatory and sensory functions, protein kinases have been considered interesting targets for anti-parasitic drugs [70].
In conclusion, this study has given some interesting insights into early molecular processes in the L3 of <span class="yellow">A. suum</span>. Approximately 60% of the transcripts enriched in the L3 stage of <span class="yellow">A. suum</span> have homologues/orthologues in <span class="yellow">C. elegans</span>. The bioinformatic analyses of selected molecules suggest that a complex genetic network regulates or controls larval growth and development in <span class="yellow">A. suum</span> L3s, and some of these might be involved in or regulate the switch from the free-living to the parasitic stage. Some caution is warranted in drawing conclusions regarding molecular mechanisms regulating the transition to parasitism in parasitic nematodes from information on <span class="yellow">C. elegans</span>, as latter is a free-living nematode. Also, while the method of data integration is essential for the reliable prediction of genetic interactions, it might limit the capacity of the approach somewhat to infer nematode-specific interactions. As additional datasets of genes and gene functions become available for various parasitic nematodes, more informed inferences can be made regarding the functions of nematode-specific genes, particularly those involved in the transition to parasitism. The imminent genome sequence of <span class="yellow">A. suum</span> (http://www.sanger.ac.uk/Projects/Helminths/) should all assist in this endeavour. Also, functional analysis of selected molecules representing selected ESTs identified herein, utilizing gene silencing approaches established recently [33],[34], could provide some insights into developmental processes in Ascaris and related ascaridoid nematodes and provide avenues for the development of novel approaches for their control.<br><br>Supporting Information<br><br>
<h3>pmcA2553092</h3>TOPS++FATCAT: Fast flexible structural alignment using constraints derived from TOPS+ Strings Model
Abstract
Background
Protein structure analysis and comparison are major challenges in structural bioinformatics. Despite the existence of many tools and algorithms, very few of them have managed to capture the intuitive understanding of protein structures developed in structural biology, especially in the context of rapid database searches. Such intuitions could help speed up similarity searches and make it easier to understand the results of such analyses.<br><br>Results
We developed a TOPS++FATCAT algorithm that uses an intuitive description of the proteins' structures as captured in the popular TOPS diagrams to limit the search space of the aligned fragment pairs (AFPs) in the flexible alignment of protein structures performed by the FATCAT algorithm. The TOPS++FATCAT algorithm is faster than FATCAT by more than an order of magnitude with a minimal cost in classification and alignment accuracy. For beta-rich proteins its accuracy is better than FATCAT, because the TOPS+ strings models contains important information of the parallel and anti-parallel hydrogen-bond patterns between the beta-strand SSEs (Secondary Structural Elements). We show that the TOPS++FATCAT errors, rare as they are, can be clearly linked to oversimplifications of the TOPS diagrams and can be corrected by the development of more precise secondary structure element definitions.<br><br>Software Availability
The benchmark analysis results and the compressed archive of the TOPS++FATCAT program for Linux platform can be downloaded from the following web site: <br><br>Conclusion
TOPS++FATCAT provides FATCAT accuracy and insights into protein structural changes at a speed comparable to sequence alignments, opening up a possibility of interactive protein structure similarity searches.<br><br><br><br>Background
Structural biology is one of the most successful fields of modern biology. Over 50,000 solved protein structures illustrate details of many specific biological processes. The same data also provide us with information about the global features of protein structure space and can be studied to discover the evolutionary, physical, and mathematical rules governing them. How many fundamentally different protein shapes (folds) are there? How do protein structures evolve? How do new structural features appear, and if they are coupled with changes in function, how does this process occur? Such questions can be studied by classifying, comparing and analyzing known protein structures. Two different, but synergistic strategies are typically used for this purpose. In classification systems such as SCOP [1] or CATH [2], <span class="blue">human</span> intuition is used to simplify the description of protein structures to a manageable size, and a <span class="blue">human</span> eye, sometimes supported by automated analysis, can recognize patterns and types of structures. In the second approach, specialized comparison algorithms, such as DALI [3], CE [4], or FATCAT [5] can be used to calculate a distance-like metric in the protein structure space. This in turn can be used to cluster proteins into groups. Many such algorithms have been developed over the past few decades and have been mostly used for the classification of protein structures into families.
An exact solution of an alignment between two structures is formally equivalent to a threading problem and is therefore NP-complete [6]. However, a practical solution can be obtained by heuristics reducing the problem to a manageable size [7]. In <span class="blue">human</span> classification systems, the protein is usually reduced to a set of several structural elements, which obviously involve many arbitrary thresholds. Automated algorithms have the same problem and also suffer from inconsistencies between different numerical measures of protein structure similarity [8]. Interestingly, despite these problems, results of different approaches are broadly similar. They all identify approximately a few hundred general classes of protein structures, usually called folds [1] or topologies [2], distinguished by how the main chain of the protein folds around itself in the three-dimensional space. At the same time, the comparison of different approaches, both between and within the two classes, shows that fold/topologies (or cluster) definitions are somewhat fuzzy, with some proteins being occasionally difficult to classify and joining different groups depending on various assumptions. This lead some to question the concept of the fold [9], but practical application of protein structure comparison leaves little doubt that protein structure space has some natural granularity that overlaps well with the traditional fold classification.
Comparison and classification of protein structures is significantly simplified by the fact that proteins have naturally modular structures, being mostly composed of locally regular structures: alpha helices and beta strands. These two types of secondary structures constitute a little over 50% of an average protein's length. With the average length of a secondary structure being around 10 amino acids, this makes it possible to describe protein structure as an arrangement of a much smaller number of elements. Protein structures are often visualized in a simplified form, with the so-called ribbon diagram with secondary structures shown as helices and arrows being the most popular (see Figure 1). This picture can be simplified further by showing individual secondary structure elements as simple symbols (circles or boxes/triangles). These depictions, called fold diagrams, originally proposed in the 70s [10-12] are best captured by a TOPS (Topology of Protein Structures) algorithm, which attempts to automate the process of creation of the topology cartoon [13]. While useful in protein classification, such simplified descriptions are not used in the most popular automated protein structure comparison algorithms such as DALI [3] or CE [4]. Kleywegt and Jones developed a method for finding similar motifs based on comparing distance matrices that are constructed by representing protein as a set of SSEs with their directional vectors and angle between those vectors [14]. Programs that used SSEs either for structure comparison based on hierarchical superposition of both SSEs and atomic representation [15] or for finding common substructures in the comparison process based on subgraph isomorphism, such as [16,17] and recent applications of the TOPS diagram [18,19], usually struggle with translating the comparison results from the secondary structure to the individual residue level. Although the SSM method uses graph-matching procedures at the SSE level followed by an interactive 3D alignment of the protein C-alpha atom [20], it lacks the topological relationships between the SSEs, which are essential features in identifying common scaffolds in distantly related proteins. A TOPS pattern was used to guide the sequence alignment, for instance, to build multiple structural alignments of the distantly related family of beta-rich protein domains [21]. The Multiple Sequence Alignment Tool (MSAT) automates this approach, merging it with a popular ClustalW program [22]. DALI [3], CE [4] or FATCAT [5] introduce their own methods of decomposing the protein structure into smaller units, such as 7 × 7 dense distance map fragments (DALIs) or aligned fragment pairs (AFPs) (CE and FATCAT). The large number of such fragments and the combinations of the fragments that need to be evaluated by structure comparison programs is the main reason for the significant computational requirements of such algorithms. However, more importantly, TOPS+ method is used here to enable a structural comparison that takes into account flexibility in protein structures and not only classifies the differences, but also can recognize such rearrangements – which is a first such application using the SSEs language. In this contribution, we explore the question of whether it would be possible to combine insights provided by topology diagrams into automated protein structure alignment algorithms, focusing on the FATCAT program developed previously in our group.<br><br>Methods
Flexible structure alignment method FATCAT
FATCAT [5] is a unique structure alignment method that allows for flexibility in the structures being compared. It builds the alignment by chaining aligned fragment pairs (AFPs) [23] together using a unified scoring function where AFP extensions, gaps, and twists each have their specific scores (Figure 2). Introducing a twist into the alignment is penalized, but this penalty may be compensated for by the gain in the score of the resulting alignment (i.e., longer alignment and/or better RMSD). Rigid alignment can be treated as a special case, in which no twist is allowed in chaining AFPs. FATCAT program provides alignment in both, "rigid" mode and "flexible" (default) mode.
FATCAT, as well as most other protein structure comparison programs, is very slow when compared to sequence alignments. The computing time of FATCAT is determined by the size of the collection of AFPs detected between the two structures being compared. FATCAT is available from a server  with an option to search in SCOP or PDB databases for similar structures. This search typically takes between 8 to 16 hours of CPU time, and this is the main obstacle to broader use of this option. FATCAT has been used to construct a Flexible Structure Neighborhood (FSN) database that contains pre-computed results of structure similarity searches and it takes several weeks of CPU time to update the FSN database. Other protein structure comparison resources, such as DALI or CE have very similar problems.<br><br>TOPS cartoons and TOPS graph models
As discussed in the Background, TOPS cartoons capture the simplified, fold-level description of protein structure and at the same time can be automated [24]. The TOPS algorithm uses structural features such as hydrogen bonds and chirality of the beta strands to provide a scoring function to optimize the cartoon (see Figure 1(b)). In TOPS, the secondary structural elements (SSEs) are derived from the DSSP program [25]. Based on TOPS cartoons, a formal graph model and graph-based definitions of protein topology and pattern discovery and comparison methods were developed [26,27]. The TOPS database and comparison, pattern discovery and matching programs are accessible from .<br><br>Novel TOPS+ and TOPS+ strings models
The TOPS model was further enhanced to incorporate features such as protein-ligand interaction information and more detailed secondary structural segment information. This enhanced model is called TOPS+ model (see Figure 3a). This TOPS+ model can be described formally in a TOPS+ strings language (Figure 3b) at a reduced linear level. The enhanced TOPS+ strings models can be used in fast string-based structure matching and comparison, at the same time avoiding issues of NP-completeness associated with graph alignments.
In detail, each node (SSE segment) of the TOPS+ strings is described by its type, orientation, PDB start number, segment length, total number of incoming (InArc) and outgoing (OutArc) arcs (edges), total number of ArcTypes, and total number of ligand arcs (LigArc). The type of the segment (SSEType) could be one of [E, e, H, h, U, u], where, "E" and "e" represent the "up"- and "down"-oriented beta strands; "H" and "h" indicate the "up"- and "down"-oriented alpha helices; and "U" and "u" represent ligand-bound and ligand-free loops. The InArcType can be classified as an/a [R, L, P, A], where "R" and "L" represent right and left chiralities; and "P" and "A" represent parallel and anti-parallel hydrogen bonds, respectively. The OutArcType is represented in a similar manner by [R', L', P', A']. Ligand arcs are indicated by LT = AA, where LT is the ligand type and AA is the PDB number. For example, Figure 3(a) and 3(b) contain visual representations of TOPS+ and TOPS+ strings models, respectively, for the protein domain d1fnb_1. Here the triangles represent the beta strands; the red curve represents the alpha helix; gray ellipsoids indicate loops; and green arcs indicate hydrogen bonds between two beta strands, called anti-parallel beta sheets. The length of a TOPS+ strings model is defined by number of SSEs; thus, the length of d1fnb_1 is 19. For further details, see [28].<br><br>TOPS+ strings comparison method
TOPS+ is a comparison method that computes a distance between TOPS+ strings models of two proteins based on a dynamic programming approach and identifies the longest common subsequence (LCS), consisting of the list of the topologically equivalent SSEs between two proteins. For example, Figure 3(c) shows the TOPS+ strings alignment between Dihydropteridine reductase proteins from <span class="yellow">rat</span> (1dhr) and <span class="blue">human</span> (1hdr). The TOPS+ strings models for 1dhr and 1hdr are represented by a linear string-model, where a yellow triangle and red curves indicate the beta strands and alpha helices in their "up" or "down" orientations, respectively. The grey line and purple stubs represent the loop regions and the NAD ligand interactions, respectively. Note that the ligand-interaction information is optional and in this work we have not used it. The incoming and outgoing arcs are depicted in the SSEs (top and bottom of the beta strands), where red and green arcs represent the parallel and anti-parallel hydrogen-bond interactions that show beta-sheet information, while yellow and blue arcs indicate the right and left chirality relationships between the SSEs. A pink arrow between the TOPS+ strings elements indicates the conserved SSE. The dotted arrows indicate the conserved alpha helices and beta strands, while the plain arrows indicate the conserved loop regions.<br><br>TOPS++FATCAT method
In this work, we want to test the general idea of pruning the search space of the FATCAT comparison process using topological constraints derived from the TOPS+ strings alignment. Many of the AFPs considered in the FATCAT alignment could be easily eliminated from the comparison by constraining the alignment region. Here we explore constraints obtained from the TOPS+ strings alignment, which identifies topologically equivalent secondary structure elements (alpha helices, beta strands, and loops) for this purpose. Such equivalences define blocks that restrict the alignment region; AFPs that fall outside these regions are simply not considered (see Figure 4(b)). We introduce a parameter r to control the strictness of constraints by TOPS+ strings alignments; r equals 0 if the alignment region is strictly restrained by TOPS+ strings alignment, and r is set to 1 by default in our program to allow certain flexibility to the constrained alignment region (Figure 4(c)). We then can speed up the FATCAT alignment by considering only the AFPs within the constrained alignment area (Figure 4(d)). The rigid structural alignment can be treated as a special case of TOPS++FATCAT, in which no twist is allowed in chaining AFPs. However, the TOPS++FATCAT program provides alignment in both, "rigid" mode and "flexible" mode (default).<br><br>Benchmarking
For benchmarking and comparison, we have used the PDB40 dataset of 1,901 protein domain pairs (DP) corresponding to SCOP version 1.61 from the ASTRAL database [29]. Table 1 provides the SCOP superfamily level homolog versus non-homolog statistics for the four main SCOP classes i.e., all-alpha, all-beta, alpha/beta, alpha+beta, and all proteins regardless of their structural classes.<br><br>Evaluation Analyses
We performed the Receiver Operating Characteristics (ROC) curve and the AUC (Area Under the ROC Curve) analyses to compare the performance of the TOPS++FATCAT method with the original FATCAT method, using SCOP classification at the superfamily level as a standard of comparison [30].<br><br>
Results
ROC and AUC Analyses
We have compared the performance of the TOPS++FATCAT method against the original FATCAT method using the SCOP classification information at the superfamily level. We have plotted the ROC curves based on P-values obtained from the FATCAT and the TOPS++FATCAT methods. We have plotted the ROC curves separately for the main SCOP classes, i.e., all-alpha, all-beta, alpha/beta, alpha+beta, and all proteins regardless of the class they belong to (see Figure 5(a) to 5(e)). In the graph, the x- and y-axes represent the false positive and true positive rates of the performance of the comparison methods respectively. In the legend, rF-pvalue and fF-pvalue indicate results from the rigid and flexible FATCAT methods, respectively; similarly, rT2F-pvalue and fT2F-pvalue represent the rigid and flexible TOPS++FATCAT methods, respectively. We have calculated the AUC values for all the SCOP classes based on ROC curves obtained from the FATCAT and TOPS++FATCAT methods with the flexible/rigid options (see Table 2).
For all protein classes, the rigid FATCAT performs best, usually followed by the flexible FATCAT, the rigid TOPS++FATCAT, and the flexible TOPS++FATCAT. The performance of all four methods is best for all alpha and all beta proteins, and all four perform markedly worse (but similar to each other) for alpha/beta proteins. Only alpha+beta proteins show a clear difference between the FATCAT and TOPS++FATCAT methods. It is important to note that the TOPS+ strings models consider the parallel and anti-parallel properties of the beta-sheet information in the form of total number of incoming and outgoing arcs with their ArcTypes. Thus, the TOPS++FATCAT method discriminates the protein domain pairs more efficiently compared to the original FATCAT method. For example, in the all-beta protein domain pairs, both the flexible and the rigid TOPS++FATCAT methods perform well. The flexible TOPS++FATCAT method covers nearly 84% of protein domains with 0% false positives, but the flexible and rigid FATCAT methods cover only 76% and 49% of the true positives, respectively, with 0% false positives. The zoomed-in version of the ROC curves with up to 10% false positives for all-beta rich protein families is shown in Figure 5(f); where both the rigid TOPS++FATCAT (green) and flexible (red) TOPS++FATCAT methods have coverage rates of 82% and 84% true positives respectively with 0% false positives. The overall results for all protein classes show that TOPS++FATCAT performance is only slightly lower (3%–7% AUC value difference (see Table 2)) as compared to FATCAT while providing a significant, more than 10-fold speedup (see next section).<br><br>AFP and Runtime Analyses
We tested both the FATCAT and TOPS++FATCAT methods using the Mac OS X version 10.4.10 computer system with a 2 × 2.66-GHz Dual-Core Intel Xeon processor and 1-GB 667 MHz memory. We have performed runtime analysis on 1,901 protein domain pairs and counted the total number of AFPs and the corresponding runtime from both the FATCAT and the TOPS++FATCAT methods. The results show an exponential increase in AFPs (Figure 6(b)) and corresponding runtime (Figure 6(a)) for the FATCAT method as compared to the TOPS++FATCAT method (see Table 3) For example, the average number of AFPs for the TOPS++FATCAT method is 530, but the average number of AFPs for the FATCAT method is 15,019. This represents the number of average AFPs used by the FATCAT method is increased by a factor of 28 (see Table 3). This result leads to the conclusion that TOPS++FATCAT is 22 times faster compared to the FATCAT because this method must take into account more number of AFPs in the comparison process (see Table 3).<br><br>Case Studies
While the overall accuracy of both rigid and flexible FATCAT methods is better than their TOPS++FATCAT equivalents, an interesting example where the opposite is true lies in the comparison of two proteins, d2trxa_ (108 aa) from <span class="yellow">Escherichia coli</span> and d1kte__ (105 aa) from <span class="yellow">Sus scrofa</span> (<span class="yellow">pig</span>) from the thioredoxin-like superfamily. For this pair, the flexible_TOPS++FATCAT method provides an alignment with 88 equivalent positions with 1.67 Å chain RMSD and 3.06 Å of optimal RMSD without any twist, giving the alignment with 10% sequence identity (see Table 4). On the other hand, the flexible_FATCAT method provides an alignment with 86 aligned positions using a twist in the C-terminal region; it has a higher chain RMSD of 5.14 Å, and its optimal RMSD is 3.48 Å. For more information regarding the chain and optimal RMSDs refer [5]. The flexible_FATCAT method uses the twist to align a helix in the C-terminal region, which is positioned incorrectly with a beta-sheet core (see Table 4). Figure 7(a) shows the superposition of d2trxa_ (gray) and d1kte__ (orange) domains from the flexible_FATCAT method, where the blue color indicates the d1kte__ protein domain from the flexible_TOPS++FATCAT method. The incorrect alignment of the C-terminal domain alpha helix of the d1kte__ domain (orange) is visible in the core of the beta-sheet region. Figure 7(b) and 7(c) shows the AFPs from the flexible_FATCAT and flexible_TOPS++FATCAT methods, respectively. The hinge region provides a twist in the flexible_FATCAT method indicated by an arrow and the AFPs represented by a different color (see Figure 7(b)). In this case, the alignment constraints from the TOPS+ strings alignment allow the TOPS++FATCAT method to avoid a spurious alignment.
The Erythrocruorin protein domain d1eca__ (136 aa) from <span class="yellow">Chironomus thummi</span> and the Phycocyanin alpha subunit protein domain d1cpca_ (162 aa) from <span class="yellow">Fremyella diplosiphon</span> (Cyanobacterium) belong to the Globin-like superfamily. For these protein domain pairs, the FATCAT method provides a better alignment with 120 and 118 aligned positions with the chain RMSD of 4.02 Å based on the flexible and rigid options, respectively. The flexible_TOPS++FATCAT method gives an alignment of 63 aligned positions with the 3.23 Å optimal RMSD and the 6.28 Å chain RMSD. In this case, the flexible_TOPS++FATCAT method misses the N-terminal region helix and misaligns some helices. For example, Figure 8(a) shows the superposition of d1eca__ (gray) and d1cpca_ (orange) domains from the flexible_FATCAT method, while d1cpca_ (blue) domain is from the flexible_TOPS++FATCAT method. The AFP chaining alignment and the actual alignment from FATCAT are shown in Figure 8(b) and 8(e), respectively. Figure 8(c) shows the AFP alignment from TOPS++FATCAT, in which this method misses the N-terminal region and incorrectly aligns some of the C-terminal regions (see Figure 8(d)). However, the rigid_TOPS++FATCAT method produces an alignment of 108 aligned positions with optimal and chain RMSDs of 3.22 Å and 6.28 Å respectively. In general, TOPS comparison does not work well for alpha-rich proteins due to the lack of hydrogen bonds between SSEs [26]. The same is true for TOPS+ strings comparison to some extent; however, this method takes advantage of ligand-interaction information to compare protein domains more efficiently; for example the DNA binding motifs such as helix-turn-helix and helix-loop-helix can be easily recognized [28]. However, we have not explored that ligand pattern discovery option within the TOPS+ strings comparison in this paper. In addition, the TOPS+ strings alignment provides only a basic alignment; the scoring function to find the best alignment has not been optimized. These problems can be addressed in future development by considering the advanced TOPS+ and TOPS+ strings models based on helix-helix packing relationships and SSE-ligand interaction properties together with the right and left chiralities. Furthermore, the TOPS+ strings comparison can be optimized in both the comparison process as well as in the alignment process in order to take into account indels (insertion/deletion) of SSEs which exist in nature across the different members of the protein superfamilies [31].<br><br>
Discussion and conclusion
The overall results for all protein classes show that TOPS++FATCAT performance is only slightly lower (3%–7% AUC value difference) as compared to FATCAT while providing a significant, more than 10-fold speedup. The main reason for the discrepancies is that TOPS+ strings alignments occasionally misalign the secondary structure elements and subsequent FATCAT alignment, constrained by the TOPS+ strings alignment, cannot overcome the earlier errors. There is a clear trade-off between the runtime and the accuracy; limiting the pool of fragments being compared speeds up the algorithm but results in (slightly) lower accuracy. At the same time, these results offer clear suggestions for future development. Using a more advanced version of the TOPS+ strings comparison method would remove some of the false positives might be at a cost of significantly slowing the total performance of the TOPS++FATCAT method.<br><br>Authors' contributions
MV developed the TOPS++FATCAT algorithm, performed the calculations and prepared the figures, YY provided advice and oversight in the project, verified the code and provided FATCAT results for comparison, AG contributed to the original idea and to writing of the manuscript.<br><br>
<h3>pmcA2556924</h3>Are there sensitive subgroups for the effects of airborne particles?
Abstract
Recent studies have shown that particulate air pollution is a risk factor for hospitalization for heart and lung disease; however, little is known about what subpopulations are most sensitive to this pollutant. We analyzed Medicare hospital admissions for heart disease, chronic obstructive pulmonary disorders (COPD) and pneumonia in Chicago, Cook County, Illinois, between 1985 and 1994. We examined whether previous admissions or secondary diagnoses for selected conditions predisposed <span class="yellow">persons</span> to having a greater risk from air pollution. We also considered effect modification by age, sex, and race. We found that the air-pollution-associated increase in hospital admissions for cardiovascular diseases was almost doubled in subjects with concurrent respiratory infections. The risk was also increased by a previous admission for conduction disorders. For COPD and pneumonia admissions, diagnosis of conduction disorders or dysrhythmias increased the risk of particulate matter < 10 microm in aerodynamic diameter (PM(10))-associated admissions. <span class="yellow">Persons</span> with asthma had twice the risk of a PM(10)-associated pneumonia admission and <span class="yellow">persons</span> with heart failure had twice the risk of PM(10)-induced COPD admissions. The PM(10) effect did not vary by sex, age, and race. These results suggest that <span class="yellow">patients</span> with acute respiratory infections or defects in the electrical control of the heart are a risk group for particulate matter effects.
Articles <br><br> Are There Sensitive Subgroups for the Effects of Airborne Particles? Antonella Zanobetti,1 Joel Schwartz,1,2 and Diane Gold1,2 1Environmental <br><br> Epidemiology Program, Department of Environmental Health, Harvard School of Public Health, Boston, Massachusetts, USA; 2Channing Laboratory, Department of Medicine, Harvard Medical School and Brigham and <span class="yellow">Women</span>'s Hospital, Boston, Massachusetts, USA <br><br> Recent studies have shown that particulate air pollution is a risk factor for hospitalization for heart and lung disease; however, little is known about what subpopulations are most sensitive to this pollutant. We analyzed Medicare hospital admissions for heart disease, chronic obstructive pulmonary disorders (COPD) and pneumonia in Chicago, Cook County, Illinois, between 1985 and 1994. We examined whether previous admissions or secondary diagnoses for selected conditions predisposed <span class="yellow">persons</span> to having a greater risk from air pollution. We also considered effect modification by age, sex, and race. We found that the air-pollution-associated increase in hospital admissions for cardiovascular diseases was almost doubled in subjects with concurrent respiratory infections. The risk was also increased by a previous admission for conduction disorders. For COPD and pneumonia admissions, diagnosis of conduction disorders or dysrhythmias increased the risk of particulate matter < 10 �m in aerodynamic diameter (PM10)-associated admissions. <span class="yellow">Persons</span> with asthma had twice the risk of a PM10-associated pneumonia admission and <span class="yellow">persons</span> with heart failure had twice the risk of PM10-induced COPD admissions. The PM10 effect did not vary by sex, age, and race. These results suggest that <span class="yellow">patients</span> with acute respiratory infections or defects in the electrical control of the heart are a risk group for particulate matter effects. Key words: effect modification, hospital admissions, particulate air pollution. Environ Health Perspect 108:841�845 (2000). [Online 28 July 2000] http://ehpnet1.niehs.nih.gov/docs/2000/108p841-845zanobetti/abstract.html <br><br> Particulate air pollution has been associated with increases in daily deaths and hospital admissions in studies all over the world (1�15). These associations are now well documented but little is known, as yet, of the characteristics of <span class="yellow">persons</span> that put them at increased risk of adverse events related to particulate air pollution. This has been identified as a key data gap (16). Schwartz and Dockery (17) reported that <span class="yellow">persons</span> older than 65 years of age had a somewhat increased risk of death, and this has been confirmed in other studies (18). A more detailed examination of particulate matter-related risk by deciles of age (19) showed the risk beginning to increase at approximately 40 years of age and reaching its maximum for those 75 years of age and older. In addition to age, several studies suggest that <span class="yellow">persons</span> with respiratory illness are at increased risk for cardiovascular effects associated with air pollution. An examination of death certificates on high- and low-air pollution days reported a substantial difference in the proportion of deaths from cardiovascular causes that had respiratory disease as a contributing cause of death (19). A recent follow-up study of a cohort of <span class="yellow">persons</span> with chronic obstructive pulmonary disease (COPD) in Barcelona, Spain, found an association between particulate air pollution and all-cause mortality in the cohort (20). The magnitude of the risk per microgram per cubic meter of exposure was substantially greater than that for the general population. Environmental Health Perspectives <br><br> Controlled exposure of animals with chronic bronchitis and control animals to concentrated air particles also demonstrated a potentiating effect of chronic lung disease in the response to airborne particles (21). This has led to the hypothesis that the cardiovascular effects of air pollution are predominantly in <span class="yellow">persons</span> with chronic lung disease. There has been even less done to examine potential modifiers of the effects of airborne particles on hospital admissions. The existing literature on comorbidity shows that comorbidity per se seems to increase the risk of adverse outcomes (22�30). Little is known about the role of these comorbidities as effect modifiers for the effects of air pollution. This study uses data from the Medicare system to examine potential short-term and long-term medical conditions that may increase a <span class="yellow">person</span>'s risk of hospital admissions associated with particulate air pollution. In addition, we examine potential effect modification by age, race, and sex. <br><br> Materials and Methods Health data. The Health Care Financing Administration (Baltimore, MD) maintains records of every hospital admission for Medicare <span class="yellow">participants</span> in the United States. <span class="yellow">Persons</span> in this database have a unique identifier. Using this identifier, we traced every hospital admission for heart and lung disease for each <span class="yellow">person</span> in Cook County, Illinois, between 1985 and 1994. We chose Cook County because it is the most populous <br><br> county in the United States with daily monitoring for particulate matter with aerodynamic diameter < 10 �m (PM10). The data were then analyzed to look at effect modification by concurrent and preexisting conditions as well as by age, race, and sex. To establish a baseline risk, we computed daily counts of hospital admissions for cardiovascular disease (CVD) [International Classification of Disease, 9th edition, World Health Organization, Geneva (ICD-9) code 390�429], pneumonia (ICD-9 code 480�487), and COPD (ICD-9 code 490�496, excluding 493). The association between these daily counts and PM10 was examined for the years 1988�1994, when daily PM10 monitoring data were available in Chicago. Once our baseline risks were established, we examined three classes of potential effect modifiers. First, we looked at whether previous admissions for selected conditions predisposed <span class="yellow">persons</span> to having a greater risk from air pollution. For each of the three admission categories (CVD, pneumonia, and COPD), we considered 10 causes (defined by a previous admission) as effect modifiers: COPD (ICD-9 code 490�496 except 493), asthma (ICD-9 code 493), acute bronchitis (ICD-9 code 466), acute respiratory illness (ICD-9 code 460�466), pneumonia (ICD-9 code 480�487), CVD (ICD-9 code 390�429), myocardial infarction (ICD-9 code 410), congestive heart failure (ICD-9 code 428), conduction disorders (ICD-9 code 426), and dysrhythmias (ICD9 code 427). To test the hypothesis that <span class="yellow">persons</span> with these conditions had higher risks of subsequent PM10-related admissions, we computed separate daily counts of admissions for our three target causes, stratified by whether or not the <span class="yellow">person</span> admitted had been previously admitted for the hypothesized predisposing condition. Separate analyses were then performed within each strata to see if the effects of PM10 differed by strata. Address correspondence to A. Zanobetti, Department of Environmental Health, Environmental Epidemiology Program, Harvard School of Public Health, 665 Huntington Avenue, Boston, MA 02115 USA. Telephone: (617) 4324642. Fax: (617) 277-2382. E-mail: azanob@ sparc6a.harvard.edu Supported by NIEHS grant ES07937. Received 18 January 2000; accepted 18 April 2000. <br><br> � VOLUME 108 | NUMBER 9 | September 2000 <br><br> 841 <br><br> Articles <br><br> � <br><br> Zanobetti et al. <br><br> The second set of potential predisposing conditions included secondary diagnoses associated with the index admission. These could represent the presence of a chronic condition (e.g., COPD) that has not resulted in a previous hospital admission. They could also represent acute conditions that may have increased the subjects' sensitivity to air pollution. For example, if respiratory infections modified the effect of particulate matter on the cardiovascular health of <span class="yellow">persons</span> with underlying heart disease, then the risk of a hospital admission for heart disease might be different in <span class="yellow">persons</span> with infections. If this were true, then the risk ratio of a 10-�g/m3 increase of PM10 on cardiovascular admissions of <span class="yellow">persons</span> with a concurrent respiratory infection would be different from the ratio in <span class="yellow">persons</span> without respiratory infection. To test these hypotheses, we computed separate daily counts of admissions for events with and without the concurrent conditions hypothesized to increase sensitivity to air pollution. These were taken as the same 10 conditions in the first analysis with certain exclusions for pairing that would be illogical. That is, the concurrent diagnosis of a specific cardiac condition was not treated as an effect modifier for admissions for any cardiovascular condition. Likewise, pneumonia and COPD were not possible concurrent conditions for each other. The third set of predisposing conditions considered was being older than 75 years of age, nonwhite, and female. These were examined for all three outcomes. We obtained weather data for O'Hare Airport from the EarthInfo CD-ROM (EarthInfo CD NCDC Surface Airways, EarthInfo Inc., Boulder, CO), and we obtained air pollution data from the U.S. Environmental Protection Agency Aerometric Information Retrieval System network (31). <br><br> running-line smoother, loess (35), was chosen to estimate the smooth function. To control for weather variables and day of the week, we chose the smoothing parameter that minimized the Akaike's information criterion (36). To model seasonality we chose the smoothing parameter that minimized the sum of the autocorrelation of the residuals while removing seasonal patterns. Two autoregressive terms (37) were added in the model to eliminate the remaining serial correlation from the residuals. We used the mean of PM10 on the day of the admission and the day before the admission as our exposure variable. This gives results that are similar to those obtained fitting a full distributed lag model (38). PM10 was treated linearly. Our baseline models used the daily counts of CVD, pneumonia, and COPD admissions as outcomes. We then subdivided those counts by the presence or absence of the potential effect modifier and reestimated our regressions on those subgroups. We considered effect modification to be indicated when the estimates of PM10 in the group with the condition was outside of the 95% confidence interval (CI) of the effect estimate in <span class="yellow">persons</span> without the condition. <br><br> Results Table 1 shows the mean daily admissions for COPD, cardiovascular, and pneumonia both overall and in the presence of the potential effect modifiers. For some effect modifiers such as conduction disorders or myocardial infarctions, the counts in conjunction with our respiratory outcomes are <br><br> low, which limits power. In general, the numbers are lower for examining effect modification by previous admissions than for effect modification by concurrent diagnosis. This is as expected because many clinically relevant comorbidities may never have resulted in a hospital admission. Table 2 shows the 25th, 50th, and 75th percentile values for the environmental variables. The mean value for PM10 is 33 �g/m3. The daily values for PM10 were computed as the average of 10 monitors, two of which measured PM10 almost every day and the others less frequently (38). Table 3 shows the mean daily counts of CVD, COPD, and pneumonia by sex, age groups, and race. The distribution by sex is almost even, although the counts of admissions for males are generally lower (approximately 10%) than for females, particularly for cardiovascular diseases. The counts of CVD, COPD, and pneumonia admissions were similar for <span class="yellow">people</span> 65�75 or 75 years of age and older. Tables 4�6 show the results for the effect PM10 overall and stratifying by concurrent diagnosis and previous admissions. These are expressed as the percentage increase for 10 �g/m3 PM10. Table 4 shows the results for CVD. A 10-�g/m3 increase in PM10 was associated with a 1.31% (5% CI, 0.97%; 95% CI, 1.66%) increase in hospital admissions for heart disease in all elderly <span class="yellow">persons</span>. A concurrent (not previous) diagnosis of COPD modified the risk of PM10-associated admissions for heart disease. However, significant associations were still seen between PM10 <br><br> Table 1. Mean daily counts of admissions, Chicago 1986�1994, for COPD, CVD, and pneumonia overall and by concurrent diagnosis and by previous admissions. By concurrent diagnosis COPD CVD Pneumonia Overall Respiratory disease Acute bronchitis Acute respiratory infections Pneumonia Asthma COPD Cardiovascular disease CVD Conduction disorders Cardiac dysrhythmias Congestive heart failure Myocardial infarction NA, not applicable. <br><br> By previous admissions COPD CVD Pneumonia 7.8 0.8 0.9 1.6 0.9 2.7 2.1 0.0 0.4 0.9 0.3 102.1 1.6 1.8 7.3 1.5 2.0 54.7 1.0 9.9 24.2 11.4 26.5 0.9 1.0 6.4 0.7 1.4 7.2 0.2 1.5 3.1 1.0 <br><br> Methods We analyzed the data with a generalized additive robust Poisson regression model (32). This approach has become the norm in such studies (14,33,34). In the generalized additive model the outcome is assumed to depend on a sum of nonparametric smooth functions for each variable that models the potential nonlinear dependence of daily admission on weather and season. The model is of the form: log[E(Yt)] = 0+ S1 (X1 )+... + Sp (Xp) where E(Yt) is the expected value of the daily count of admissions Yt and Si are the smooth functions of the covariates Xi. We examined temperature, previous day's temperature, relative humidity, barometric pressure, and day of week covariates. The locally weighted <br><br> 7.8 0.1 0.3 0.4 0.1 NA 4.7 0.2 1.4 1.8 0.1 <br><br> 102.1 0.9 1.3 4.0 1.8 13.4 NA NA NA NA NA <br><br> 26.5 0.3 0.3 NA 0.9 6.9 14.7 0.6 4.6 7.3 0.4 <br><br> Table 2. 25th, 50th, and 75th percentile values for the environmental variables in Chicago, 1988�1994. Temperature (�F) 35 51 67 Relative humidity 62 70 79 Barometric pressure 29.2 29.3 29.4 PM10 (�g/m3) 23 33 46 <br><br> Table 3. Mean daily counts of admissions by sex, race, and age groups, Chicago, 1986�1994. Group Overall Female Nonwhite Age > 75 years COPD 7.8 4.2 1.6 3.7 CVD 102.1 59.4 21.0 55.1 Pneumonia 26.5 14.7 5.2 17.4 <br><br> 842 <br><br> VOLUME <br><br> 108 | NUMBER 9 | September 2000 � Environmental Health Perspectives <br><br> Articles <br><br> � <br><br> Effects of particles on sensitive subgroups <br><br> and heart disease admissions in <span class="yellow">persons</span> without COPD listed as either a comorbidity or a cause of previous admission (Table 4). A significant association was also seen in <span class="yellow">persons</span> without any respiratory disease as a concurrent diagnosis, although the risk is much lower than in <span class="yellow">persons</span> with respiratory disease. However, the risk associated with PM10 was roughly doubled in subjects with concurrent respiratory infections and the risk estimates in those subjects were outside the 95% CI of the risk in <span class="yellow">patients</span> without concurrent respiratory infections. A previous admission for conduction disorders (e.g., heart block) increased the risk of a PM10-related subsequent admission for any heart condition, and a weaker indication of effect modification was seen for <span class="yellow">persons</span> with previous admission for dysrhythmias. In contrast heart failure and previous myocardial infarctions were highly insignificant as effect modifiers. Table 5 shows the results for COPD. Overall, there is a 1.89% (95% CI, 0.8�3.0) increase in COPD admissions for a 10�g/m3 increase in PM10. The results of the stratified analysis suggest that preexisting heart disease modifies Table 4. Percentage increase in hospital admissions for CVD in all <span class="yellow">persons</span> and by concurrent diagnosis and previous admissions. PM10 2.5% CI 97.5% CI All <span class="yellow">persons</span> 1.31 By concurrent diagnosis Respiratory disease All respiratory disease With 1.65 Without 0.98 Acute bronchitis With 2.50 Without 1.07 Acute respiratory infections With 2.71 Without 1.06 Pneumonia With 1.95 Without 1.03 COPD With 1.59 Without 1.08 By previous admissions Respiratory disease All respiratory disease With 1.18 Without 1.08 COPD With 1.48 Without 1.09 Asthma With 1.71 Without 1.08 Cardiovascular disease Conduction disorders With 2.89 Without 1.07 Cardiac dyshrethmias With 1.61 Without 1.04 0.97 1.66 <br><br> the risk of COPD admissions on high particle days. Previous admissions for any cardiovascular disease increased the risk of a PM10associated COPD admission approximately 2.5-fold. A previous heart failure admission caused an even more striking increase in the PM10 effect. Previous admissions for dysrhythmias and conduction defects were rare (Table 1) with no power to examine effect modifications. Listings as concurrent diagnoses were more common and here they joined heart failure in increasing the risk of PM 10 -associated COPD admissions. For COPD there was also some indication that concurrent pneumonia or an acute respiratory infection admission in the last year increased risk. The low numbers made these estimates less precise, however. The percentage increase in pneumonia admission (Table 6) for 10 �g/m3 PM10 is higher than for COPD or CVD with an increase of 2.34% (95% CI, 1.66�3.0). As with COPD, <span class="yellow">persons</span> with heart disease appeared at higher risk of pneumonia hospital admissions associated with particulate air pollution. Here diagnoses suggestive of impaired autonomic control of the heart, such as conduction disorders or dysrhythmias, were associated with increased risk for PM 10 effects on pneumonia admissions. Unlike COPD, no difference was seen for congestive heart failure. <span class="yellow">Persons</span> with asthma Table 5. Percentage increase in hospital admissions for COPD in all <span class="yellow">persons</span> and by concurrent diagnosis and previous admissions. PM10 2.5% CI 97.5% CI All <span class="yellow">persons</span> 1.89 By concurrent diagnosis Respiratory disease Pneumonia With 4.00 Without 1.51 Cardiovascular disease Conduction disorders With 2.34 Without 1.60 Cardiac dysrhythmias With 3.09 Without 1.43 Congestive heart failure With 2.90 Without 1.39 By previous admissions Respiratory disease Acute respiratory infections With 3.20 Without 1.70 Cardiovascular disease CVD With 2.90 Without 1.18 Congestive heart failure With 4.37 Without 1.14 Within 1 year 6.04 0.80 2.99 <br><br> had twice the risk of a PM10-induced pneumonia admission as <span class="yellow">persons</span> without asthma. Table 7 shows the results by sex, age, and race. None of the effect size estimates for any of the stratification variables were outside of the 95% CI for the opposite strata. There was a tendency for the effect of PM10 on CVD admissions to be higher for females, whereas the effect on pneumonia admissions was higher for males. In general, we found somewhat larger effects on whites compared to nonwhites, and for <span class="yellow">persons</span> older than 75 years of age compared to younger <span class="yellow">persons</span>. <br><br> Discussion In this analysis we examined whether the effect of PM10 on the risk of hospital admission for heart and lung disease was different depending on the presence of comorbidities. We found that PM10 was associated with hospital admissions for all three causes (CVD, COPD, and pneumonia) and we found not a general increase in PM10 related risk with comorbidities, but a specific pattern that is suggestive of potential mechanisms and consistent with other recent epidemiologic and toxicologic findings. One major finding of this study is that preexisting cardiovascular disease, particularly impaired autonomic control (conduction defects and dysrhythmias) and heart failure, substantially increased the risk of respiratory admissions associated with airborne particles. In fact, recent <span class="blue">human</span> studies have shown that exposure to particulate air pollution is a risk factor for reduced heart rate variability (39�41). Reduced heart rate variability is an adverse response and a risk factor for arrhythmia. A new study of defibrillator discharges in <span class="yellow">patients</span> with implanted cardioverter defibrillators found that discharges were associated with air pollution (42). Exposure to combustion Table 6. Percentage increase in hospital admissions for pneumonia in all <span class="yellow">persons</span> and by concurrent diagnosis and previous admissions. PM10 All <span class="yellow">persons</span> By concurrent diagnosis Respiratory disease Asthma With Without Cardiovascular disease Conduction disorders With Without Cardiac dysrhythmias With Without By previous admissions Cardiovascular disease Cardiac dysrhythmias With Without 2.34 2.5% CI 97.5% CI 1.66 3.02 <br><br> 1.10 0.64 �0.47 0.76 0.18 0.76 0.55 0.72 0.85 0.75 <br><br> 2.20 1.33 5.55 1.37 5.30 1.37 3.36 1.35 2.34 1.41 <br><br> �0.45 0.47 �4.42 0.58 0.64 0.33 0.77 0.24 <br><br> 8.65 2.57 9.59 2.64 5.60 2.55 5.08 2.55 <br><br> 0.45 0.76 �0.40 0.78 �0.43 0.77 0.22 0.76 0.75 0.72 <br><br> 1.91 1.41 3.40 1.40 3.89 1.39 5.63 1.38 2.48 1.36 <br><br> 4.18 2.07 7.92 1.99 � � <br><br> 1.01 1.46 4.28 1.37 � � <br><br> 7.46 2.69 11.69 2.61 � � <br><br> �1.38 0.66 0.99 �0.01 1.43 0.05 2.10 <br><br> 8.01 2.76 4.85 2.39 7.40 2.24 10.14 <br><br> 3.47 2.08 <br><br> 1.21 1.45 <br><br> 5.79 2.71 <br><br> Increases are for a 10-�g/m3 increase in PM10. <br><br> Increases are for a 10-�g/m3 increase in PM10. <br><br> Increases are for a 10-�g/m3 increase in PM10. <br><br> Environmental Health Perspectives <br><br> � VOLUME 108 | NUMBER 9 | September 2000 <br><br> 843 <br><br> Articles <br><br> � <br><br> Zanobetti et al. <br><br> Table 7. Effect modification by sex, race, and age groups for 10 �g/m3 PM10. % All <span class="yellow">persons</span> Male Female White Non-white Age > 75 Age  75 1.89 1.34 2.19 1.65 1.07 2.20 1.33 COPD (95% CI) (0.80, 2.99) (�0.14, 2.84) (0.81, 3.59) (0.51, 2.81) (�1.11, 3.3) (0.72, 3.69) (0.03, 2.65) % 1.31 1.07 1.21 1.20 0.70 1.28 0.93 CVD (95% CI) (0.97, 1.66) (0.62, 1.51) (0.83, 1.6) (0.86, 1.55) (0.1, 1.3) (0.88, 1.69) (0.51, 1.35) % 2.34 2.65 1.91 2.45 1.91 2.12 2.52 Pneumonia (95% CI) (1.66, 3.02) (1.81, 3.5) (1.11, 2.72) (1.77, 3.14) (0.69, 3.14) (1.38, 2.86) (1.57, 3.48) <br><br> Figures shown are the percentage increase in admissions (95% CI). <br><br> particles has also been associated with arrhythmia in an animal model (43) and changes in ST segments have been noted as well (44). This is the first study to suggest <span class="yellow">persons</span> with defects in the electrical control of the heart are also at higher risk of respiratory illness after exposure to airborne particles. These data also suggest that <span class="yellow">persons</span> admitted to hospitals for pneumonia during an air pollution episode may be at high risk for clinically significant conduction disorders during that hospital admission. <span class="yellow">Patients</span> with congestive heart failure were at greater risk of hospital admissions for COPD in association with airborne particles. Heart failure and COPD is not an uncommon combination. The finding that these <span class="yellow">patients</span> are at higher risk for admissions associated with particulate air pollution is new but is also consistent with several other recent reports. The spontaneous hypertensive <span class="yellow">rat</span> develops a model of heart failure, and recent studies have reported greater sensitivity to particulate air pollution in these <span class="yellow">rats</span>. These include both electrocardiogram abnormalities (44) and pulmonary toxicity (45,46). Similarly, in an epidemiologic study, Hoek et al. (47), found a higher relative risk of death with an increase in PM10 for congestive heart failure deaths than other deaths. The potential role of COPD in those heart failure deaths was not examined. Another consistent pattern in our data is of acute respiratory infections increasing susceptibility to airborne particles. Acute bronchitis, or more generally acute upper respiratory illnesses, as well as pneumonia, increased susceptibility to particle-associated admissions for CVD and COPD. The notion that air pollution exacerbates acute respiratory infections is well supported by studies which report associations between airborne particles and hospital admissions for respiratory infections (48,49). Zelikoff et al. (50) exposed <span class="yellow">rats</span> infected with streptococcus to concentrated air particles and reported a significant increase in bacterial burdens and in the extent of pneumonia compared to animals exposed to filtrated air. This suggests an impaired immune response. Similarly, exposure to combustion <br><br> particles enhances influenza infections in <span class="yellow">mice</span> (51). An impaired defense to respiratory infection is a major reason that <span class="yellow">persons</span> with COPD require hospital admission. If airborne particles result in further impairment the effect modification we observe makes good sense. The effect modification for heart disease admissions is more relevant. This modification is consistent with the earlier report of Schwartz (19), who found greater reports of respiratory complications on death certificates with an underlying cause of heart disease if the death occurred on a day with high levels of airborne particles. Although airborne particle exposure has been associated with increased exacerbation of asthma (2,12,48,52�59), this paper is the first to suggest that asthmatics are more susceptible to PM10-induced pneumonia exacerbation or to cardiovascular effects. The effects on pneumonia admissions are plausible, given the impaired ability to fight off infections in asthmatics with mucus plugs and the evidence the airborne particles impair the lungs' ability to fight off bacterial and viral infections, as noted earlier. The increased cardiovascular sensitivity, albeit weaker, is interesting. If airborne particles affect the cardiovascular system via the role of the lung in autonomic control, it is possible that asthmatics would be more sensitive to those effects. Animal models of asthma showed that combustion particles enhance the asthmatic response to aeroallergen challenges (59). This suggests an enhancement of pulmonary response in asthmatics. On the other hand, the diagnosis of asthma is problematic in the elderly, and crossover with COPD is possible. The possibility that this explains our results is reduced by our failure to find previous hospital admission for COPD was an effect modifier for the effect of particles on cardiovascular admissions. We must acknowledge several potential limitations of this study. First, we considered only previous admissions that occurred within Cook County. Hence <span class="yellow">persons</span> with previous admissions elsewhere would be misclassified to our reference group. The effect of this would be to reduce the difference in PM 10 effect between the two groups. VOLUME <br><br> Nevertheless, we identified some interesting interactions. We cannot exclude the possibility that there are areas we missed for this reason. We also examined interactions in a log relative risk model, which is inherently multiplicative. Although we believe this is justified because doubling the population exposed would be expected to double the pollution associated admissions, it results in a more conservative definition of interaction than would an additive risk model. Finally, our exposure is clearly measured with error. Most of this error is Berkson error (60) and hence will introduce no bias, and Zeger et al. (60) showed that the remaining error would have to have pathologic correlations with other variables to result in an upward bias. Another important result from this study, of course, is an estimate of the magnitude of the effect of airborne particles on public health. The PM10 concentrations in Chicago during this period were associated with approximately 1,600 additional admissions per year for heart disease, 740 additional admissions per year for pneumonia, and 170 additional admissions per year for COPD. These are not trivial increases in serious morbidity. The results of our study should be replicated in additional cities, although they do begin to fill in some missing information about the effects of airborne particles on health. More generally airborne particles have been associated with a broad range of systemic changes including heart rate variability (39�41), increased peripheral neutrophils (61�63), increased plasma viscosity (64), an increase in blood pressure (65), and the outcomes mentioned previously. The role of these systemic changes as potential sources of the specific effect modifications we have seen should be an area of fruitful research in the future. REFERENCES AND NOTES 1. Katsouyanni K, Touloumi G, Spix C, Schwartz J, Balducci F, Medina S, Rossi G, Wojtyniak D, Sunyer J, Bacharova L, et al. Short term effects of ambient sulphur dioxide and particulate matter on mortality in 12 European cities: results from time series data from the APHEA project. Br Med J 314:1658�1663 (1997). Pope CA, Dockery DW, Schwartz J. Review of epidemiologic evidence of health effects of particulate air pollution. Inhal Toxicol 7:1�18 (1995). Schwartz J. Air pollution and daily mortality: a review and meta analysis. Environ Res 64:36�52 (1994). Dominici F, Samet J, Zeger SL. Combining evidence on air pollution and daily mortality from the largest 20 US cities: a hierarchical modeling strategy. R Stat Soc Ser A, in press. Burnett RT, Dales RE, Raizenne ME, Krewski D, Summers PW, Roberts GR, Raad-Young M, Dann T, Brooke T. Effects of low ambient levels of ozone and sulfates on the frequency of respiratory admissions to Ontario hospitals. Environ Res 65:172�194 (1994). Anderson HR, Spix C, Medina S, Schouten JP, Castellsague J, Rossi G, Zmirou D, Touloumi G, Wojtyniak B, Ponka A, et al. Air pollution and daily admissions for <br><br> 2. <br><br> 3. 4. <br><br> 5. <br><br> 6. <br><br> 844 <br><br> 108 | NUMBER 9 | September 2000 � Environmental Health Perspectives <br><br> Articles <br><br> � <br><br> Effects of particles on sensitive subgroups <br><br> 7. <br><br> 8. 9. <br><br> 10. <br><br> 11. <br><br> 12. <br><br> 13. <br><br> 14. <br><br> 15. <br><br> 16. <br><br> 17. <br><br> 18. <br><br> 19. 20. <br><br> 21. <br><br> 22. <br><br> 23. <br><br> 24. <br><br> 25. <br><br> 26. <br><br> 27. <br><br> 28. <br><br> 29. <br><br> 30. <br><br> chronic obstructive pulmonary disease in 6 European cities: results from the APHEA project. Eur Respir J 10:1064�1071 (1997). Schwartz J. Short term fluctuations in air pollution and hospital admissions of the elderly for respiratory disease. Thorax 50:531�538 (1995). Schwartz J. Air pollution and hospital admissions for heart disease in eight U.S. counties. Epidemiology 10:17�22 (1999). Schwartz J. Air pollution and hospital admissions for the elderly in Minneapolis. Arch Environ Health 49:366�374 (1994). Schwartz J. Air pollution and hospital admissions for the elderly in Birmingham, Alabama. Am J Epidemiol 139:589�598 (1994). Schwartz J. Air pollution and hospital admissions for the elderly in Detroit, MI. Am J Respir Crit Care Med 150:648�655 (1994). Pope CA III. Respiratory disease associated with community air pollution and a steel mill, Utah valley. Am J Public Health 79:623�628 (1989). Saldiva PH, Pope CA, Schwartz J, Dockery DW, Lichtenfels AJ, Salge JM, Barone I, Bohm GM. Air pollution and mortality in elderly <span class="yellow">people</span>: a time series study in Sao Paulo, Brazil. Arch Environ Health 50:159�163 (1995). Schwartz, J. Air pollution and hospital admissions for cardiovascular disease in Tucson. Epidemiology 8:371�177 (1997). Delfino RJ, Murphy Moulton AM, Becklake MR. Emergency room visits for respiratory illnesses among the elderly in Montreal: association with low level ozone exposure. Environ Res 76:67�77 (1998). National Research Council. Research Priorities for Airborne Particulate Matter. Washington, DC:National Academy Press, 1998. Schwartz J, Dockery DW. Increased mortality in Philadelphia associated with daily air pollution concentrations. Am Rev Respir Dis 145:600�604 (1992). Samet JM, Zeger SL, Berhane K. The association of mortality and particulate air pollution. In: Particulate Air Pollution and Daily Mortality. The Phase I Report of the Particle Epidemiology Evaluation Project. Boston, MA:Health Effects Institute, 1995. Schwartz J. What are <span class="yellow">people</span> dying of on high air pollution days? Environ Res 64:26�35 (1994). Sunyer J, Schwartz J, Tobias A, MacFarlane D, Garcia J, Anto JM. <span class="yellow">Patients</span> with chronic obstructive pulmonary disease are a susceptible population of dying due to urban particles. Am J Epidemiol 151(1):50�56 (2000). Godleski JJ, Sioutas C, Katler M, Koutrakis P. Death from inhalation of concentrated air particles in animal models of pulmonary disease. Am J Respir Crit Care Med 153:A15 (1996). Matsui K, Goldman L. Comorbidity as a correlate of length of stay for hospitalized <span class="yellow">patients</span> with acute chest pain. J Gen Intern Med 11:262�268 (1996). Charlson M, Szatrowshi TP, Peterson J, Gold J. Validation of a combined comorbidity index. J Clin Epidemiol 47:1245�1251 (1994). Monane M, Kanter DS, Glynn RJ, Avorn J. Variability in length of hospitalization for stroke. The role of managed care in an elderly population. Arch Neurol 53:848 (1996). Hallstrom AP, Cobb LA, Yu BH. Influence of comorbidity on the outcome of <span class="yellow">patients</span> treated for out-of-hospital ventricular fibrillation. Circulation 93:2019�2022 (1996). Malenka DJ, Mclerran D, Roos N, Fisher ES, Wennberg JE. Using administrative data to describe case-mix: a comparison with the medical record. J Clin Epidemiol 47:1027�1032 (1994). Romano PS, Roos LL, Jollis JG. Adapting a clinical comorbidity index for use with ICD-9-CM administrative data: differing perspectives. J Clin Epidemiol 46:1075�1079 (1993). Deyo RA, Cherkin DC, Ciol MA. Adapting a clinical comorbidity index for use with ICD-9CM administrative databases. J Clin Epidemiol 45:613�619 (1992). Charlson ME, Pompei P, Ales KL, MacKenzie CR. A new method of classifying prognostic comorbidity in longitudinal studies: development and validation. J Chronic Dis 40:373�383 (1987). Librero J, Peir� S, Ordi�ana R. Chronic comorbidity and outcomes of hospital care: length of stay, mortality, and readmission at 30 and 365 days. J Clin Epidemiol 52:171�179 (1999). <br><br> 31. Nehls GJ, Akland GG. Procedures for handling aerometric data. J Air Pollut Control Assoc 23:180�184 (1973). 32. Hastie T, Tibshirani R. Generalized Additive Models. London:Chapman and Hall, 1990. 33. Schwartz J. Generalized additive models in epidemiology. In: International Biometric Society, Invited Papers. 17th International Biometric Conference, 8�12 August 1994, Hamilton, Ontario, Canada. Washington, DC:International Biometric Society, 1994;55�80. 34. Rossi G, Vigotti MA, Zanobetti A, Repetto F, Giannelle V, Schwartz J. Air pollution and cause specific mortality in Milan, Italy, 1980�1989. Arch Environ Health 54:158�164 (1999). 35. Cleveland WS, Devlin SJ. Robust locally-weighted regression and smoothing scatterplots. J Am Stat Assoc 74:829�836 (1988). 36. Akaike H. Information theory and an extension of the maximum likelihood principal. In: 2nd International Symposium on Information Theory (Petrov BN, Csaki F, eds). Budapest:Akademiai Kaiado, 1973;267�281. 37. Brumback BA, Ryan LM, Schwartz J, Neas LM, Stark PC, Burge HA. Transitional regression models with application to environmental time series. J Acoust Soc Am 95(449):16�28 (2000). 38. Schwartz J. The distributed lag between air pollution and daily deaths. Epidemiology 11:320�326 (2000). 39. Pope CA III, Verrier RL, Lovett EG, Larson AC, Raizenne ME, Kanner RE, Schwartz J, Villegas GM, Dockery DW. Heart rate variability associated with particulate air pollution. Am Heart J 138:890�899 (1999). 40. Gold DR, Litonjua A, Schwartz J, Lovett E, Larson A, Nearing B, Allen G, Verrier M, Cherry R, Verrier R. Ambient pollution and heart rate variability. Circulation 101(11):1267�1273 (2000). 41. Liao D, Creason J, Shy C, Williams R, Watts R, Zweidinger R. Daily variation of particulate air pollution and poor cardiac autonomic control in the elderly. Environ Health Perspect 107:521�525 (1999). 42. Peters A, Liu E, Verrier RL, Schwartz J, Gold DR, Mittelman M, Baliff J, Allen G, Monahan K, Dockery DW. Air pollution and incidences of cardiac arrhythmia. Epidemiology 11(1):11�17 (2000). 43. Godleski JJ, Verrier RL, Koutrakis P, Catalano P. Mechanisms of Morbidity and Mortality from Exposure to Ambient Air Particles. Health Effects Institute Research Report 91. Cambridge, MA:Health Effects Institute, 2000. 44. Watkinson WP, Campen MJ, Kodavanti UP, Ledbetter AD, Costa DL. Effects of inhaled residual oil <span class="yellow">fly</span> ash particles on electrocardiographic and thermoregulatory parameters in normal and compromised <span class="yellow">rats</span> [Abstract]. Am J Respir Crit Care Med 157:A150 (1998). 45. Watkinson WP, Campen MJ, Costa DL. Cardiac arrhythmia induction after exposure to residual oil <span class="yellow">fly</span> ash particles in a <span class="yellow">rodent</span> model of pulmonary hypertension. Toxicol Sci 41:209�216 (1998). 46. Kodavanti UP, Jackson MC, Richards J, Ledbetter A, Costa DL. Differential pulmonary responses to inhaled emission particulate matter (PM) in systemically hypertensive vs. normotensive <span class="yellow">rats</span> [Abstract]. Am J Respir Crit Care Med 157:A260 (1998). 47. Hoek G, Brunekreef B, van Wijnen JH. Cardiovascular mortality response to air pollution is strongest for heart failure and thrombotic causes of death [Abstract]. Epidemiology 10:S177 (1999). 48. Bates DV, Szito R. Hospital admissions and air pollutants in southern Ontario: the acid summer haze effect. Environ Res 43:317�331 (1987). 49. Pope CA III. Respiratory disease associated with community air pollution and a steel mill, Utah valley. Am J Public Health 79:623�628 (1989). 50. Zelikoff JT, Nadziejko C, Fang T, Gordon C, Premdass C, Cohen MD. Short term, low-dose inhalation of ambient particulate matter exacerbates ongoing pneumococcal infections in Streptococcus Pneumoniae-infected rates. In: Proceedings of the Third Colloquium on Particulate Air Pollution and <span class="yellow">Human</span> Health (Phalen RF, Bell YM, eds). Irvine, CA:Air Pollution Health Effects Laboratory, University of California, 1999;8-94�8-101. 51. Clarke RW, Hemenway DR, Frank R, Kleeberger SR, Longphre MV, Jakab GJ. Particle associated sulfate exposure enhances <span class="yellow">murine</span> influenza mortality [Abstract]. Am J Respir Crit Care Med 155:A245 (1997). <br><br> 52. Pope CA, Dockery DW, Spengler JD, Raizenne ME. Respiratory health and PM10 pollution: a daily time series analysis. Am Rev Respir Dis 144:668�674 (1991). 53. Schwartz J, Koenig J, Slater D, Larson T. Particulate air pollution and hospital emergency visits for asthma in Seattle. Am Rev Respir Dis 147:826�831 (1993). 54. Thurston GD, Ito K, Lippman M, Hayes CG, Bates DV. Respiratory hospital admissions and summertime haze air pollution in Toronto, Ontario: consideration of the role of acid aerosols. Environ Res 65:271�290 (1994). 55. Norris G, YoungPong SN, Koenig JQ, Larson TV, Sheppard L, Stout JW. An association between fine particles and asthma emergency department visits for <span class="yellow">children</span> in Seattle. Environ Health Perspect 107:489�493 (1999). 56. Hamada K, Goldsmith CW, Kobzik L. Air pollutant aerosols allow airway sensitization to allergen in juvenile <span class="yellow">mice</span>. Am J Resp Crit Care Med A28 (1999). 57. Lambert AL, Selgrade M, Dong W, Winsett D, Gilmour M. Enhanced allergic sensitization by residual oil <span class="yellow">fly</span> ash particles is mediated by soluble metal constituents [Abstract]. Am J Respir Crit Care Med 159:A26 (1999). 58. Dailey LA, Madden MC, Devlin RB. Do airway epithelial cells from normal and asthmatic donors respond differently to an in vitro challenge with a particulate pollutant? [Abstract]. Am J Respir Crit Care Med 157:A598 (1998). 59. Gilmour MI, Winsett D, Selgrade MJ, Costa DL. Residual oil <span class="yellow">fly</span> ash exposure enhances allergic sensitization to house dust mite in <span class="yellow">rats</span> and augments immune-mediated inflammation [Abstract]. Am J Respir Crit Care Med 155:A244 (1997). 60. Zeger SL, Thomas D, Dominici F, Samet JM, Schwartz JM, Dockery D, Cohen A. Exposure measurement error in time�series studies of air pollution: concepts and consequences. Environ Health Perspect 108:419�426 (2000). 61. Salvi S, Blomberg A, Rudell B, Kelly F, Sandstrom T, Holgate ST, Frew A. Acute inflammatory responses in the airways and peripheral blood after short-term exposure to diesel exhaust in healthy <span class="blue">human</span> volunteers. Am J Respir Crit Care Med 159:702�709 (1999). 62. Tan WC, van Eeden S, Qiu DW, Liam BL, Dyachokova Y, Hogg JL. Particulate air pollution, bone marrow stimulation and the pathogenesis of excess cardiovascular and pulmonary deaths. Am J Respir Crit Care Med 155:1441�1447 (1997). 63. Gordon T, Nadziejko C, Schlesinger R, Chen LC. Pulmonary and cardiovascular effects of acute exposure to concentrated ambient particulate matter in <span class="yellow">rats</span>. Toxicol Lett 96�97:285�288 (1998). 64. Peters A, Doering A, Wichmann HE, Koenig W. Increased plasma viscosity during an air pollution episode: a link to mortality? Lancet 349(9065):1582�1587 (1997). 65. Peters A, Stieberv J, Doering A, Wichmann HE. Is systolic blood pressure associated with air pollution? [Abstract]. Epidemiology 10(4):S177 (1999). <br><br> Environmental Health Perspectives <br><br> � VOLUME 108 | NUMBER 9 | September 2000 <br><br> 845 <br><br>  <h3>pmcA2599109</h3>Suggestions Concerning the Use of the Subclavian which Arises from the Aorta in the Treatment of the Tetralogy of Fallot *
Abstract
Images<br><br><br><br><br><br>
 SUGGESTIONS CONCERNING THE USE OF THE SUBCLAVIAN <br><br> WHICH ARISES FROM THE AORTA IN THE TREATMENT <br><br> OF THE TETRALOGY OF FALLOT* <br><br> HARRIS B. SHUMACKER, JR.** <br><br> In Blalock's early experience with the operative treatment of the tetralogy of Fallot, the two subclavian, the innominate, and carotid arteries were all used for anastomosis to the pulmonary artery. Soon, however, it became evident that use of the innominate or carotid was followed by a relatively high incidence of complications resulting from cerebral ischemia. Blalock suggested, therefore, that the subclavian be used by preference. The subclavian branch of the innominate does not become kinked or badly angulated when it is turned down for the anastomosis, and a good functional result almost invariably follows the completion of a satisfactory anastomosis. The subclavian which arises directly from the aorta, on the other hand, tends to form a bad angle when it is brought down for the anastomosis, and, indeed, near its point of origin it may be so flattened out against the relatively rigid aortic wall as to obstruct all blood flow through it. These considerations led Blalock to recommend the use of the former except in <span class="yellow">infants</span> under two years of age in whom this artery may be too small and in adults or <span class="yellow">children</span> over twelve years old or five feet tall with a left aortic arch in whom it is often too short to permit a satisfactory anastomosis.7' <br><br> In contrast, Paine and Varco,' Lam,6 Holman,' Olim,7 and others have preferred to use the subclavian which arises from the aorta and have obtained generally excellent results. They point out that the tendency to kinking of the artery is more a theoretical than a practical disadvantage, that the exposure and dissection of both the pulmonary artery and the subclavian are easier on the side of the aortic arch, that both vessels are generally longer than on the opposite side and that, as a general rule, the anastomosis can be accomplished more readily. The practical usefulness of a third systemic vessel, namely the aorta, was demonstrated by the development of a technique for side-to-side aortic pulmonary anastomosis by Potts and his associates."0 In spite of the preferences held by individual operators for use of one vessel or another, the fact remains, as Blalock has emphasized, that there is always present a usable systemic vessel provided a suitable pulmonary artery is available. Nevertheless, the choice of the systemic artery in each individual <span class="yellow">patient</span> is important since in certain cases one or another is unsuitable. <br><br> * From the Department of Surgery, the Indiana University Medical Center, Indianapolis, Indiana. Aided by a contract between the Office of Naval Research, Department of the Navy, and Indiana University. <br><br> ** Resident Surgeon, New Haven Hospital, 1937-1938. Received for publication April 26, 1951. <br><br> TREATMENT OF TETRALOGY OF FALLOT <br><br> It has been my custom to follow a policy rather similar to that outlined by Blalock. The subclavian branch of the innominate has generally been preferred in <span class="yellow">patients</span> ranging between two and twelve years of age and in older individuals in whom there is a right aortic arch. In others the thoracotomy is usually made on the side of the aortic arch, and either the aorta itself or the subclavian arising from the aorta is used for the anastomosis. My experience with the use of the subclavian originating from the aorta is small and the results have not been as good as those which others have reported. One eighteen-month-old <span class="yellow">child</span> died the day after operation and was found at post-mortem examination to have an occluded anastomosis. In one three-year-old <span class="yellow">child</span> and one twenty-year-old <span class="yellow">woman</span> a poor result was obtained and a second operation upon the other side was necessary. In two additional cases the result was only fair. Excluding the <span class="yellow">patients</span> mentioned in the present report, only in two was an unquestionably good result obtained. <br><br> Recently, in several cases in which failure seemed evident a modification has been required in order to obtain a functioning shunt. Since these modifications proved successful and since these threats of failure tax one's ingenuiity at the time of operation, I have thought it might be helpful to describe the procedures employed and to illustrate them with case reports. One is a method which obviously has very limited applicability, while the otlher would seem to be rather generally applicable. <br><br> The first consists of the transplantation of the origin of the subclavian to a more suitable portion of the aorta. <br><br> Case report <br><br> The <span class="yellow">patient</span> was a 19-year-old <span class="yellow">girl</span> who had been cyanotic since the age of six months. She had developed normally but was always limited in exercise capacity. In her early years of school she was taken to school by her parents and carried to and from her seat in the classroom. She did reasonably well in high school, being driven to and from the school, but she got into severe difficulty when she attempted to attend college. The longer distances between classes and the necessity for climbing stairs precipitated a downhill course, with increasing dyspnea, more marked cyanosis and fatigue, and the onset of bouts of loss of consciousness. The latter occurred several times daily. One which was witnessed by her physician lasted 45 minutes; he feared it would prove fatal. Ordinarily, she could not walk more than half a block. <br><br> There was marked cyanosis of the nails and mucous membranes and to a lesser extent of the skin. Clubbing was very prominent. All the results of physical and laboratory examination fitted in with a diagnosis of tetralogy of Fallot. The hematocrit ranged between 83 and 90. Oxygen saturation of arterial blood was 65 per cent at complete rest. The aortic arch was on the left side. On October 26, 1949 a left lateral thoracotomy was performed, the pleural cavity being entered through the bed of the fifth rib. There was evident greatly increased collateral circulation in the mediastinum and hilum. The pulmonary artery appeared to have markedly reduced blood flow, was very short, small, and thin-walled. Its diameter was less than 5 mm. The aorta seemed to be bowed out laterally in an unusual fashion and the short pulmonary artery could not possibly be brought out over it. The subclavian artery was relatively small, having <br><br> 487 <br><br> YALE JOURNAL OF BIOLOGY AND MEDICINE <br><br> a diameter of only about 4 mm. but it seemed rather long. It was apparent that a Pott's procedure was impossible and that an end-to-side subclavian pulmonary artery anastomosis would be doomed to failure. Hence, the subclavian artery was divided at the level of its first branch, the pulmonary artery was divided proximally, and an end-to-end subclavian-pulmonary anastomosis was carried out. This was done with ease. When the clamps were removed, however, no blood flowed through it. The softwalled subclavian was completely flattened out against the relatively rigid aorta (Fig. 1). When the hilar structures were forcibly elevated in a cephalad direction the subclavian immediately filled and pulsated normally and a thrill could be felt. Simple expansion of the lungs, however, failed to achieve this result and I could conceive of no way in which the hilar region could be held in a more cephalad direction. The subclavian was then ligated at the point where it came off the aorta and its proximal end was anastomosed end-to-side to the descending aorta. It now pulsated vigorously as did the pulmonary artery. In spite of the good pulsation no thrill was palpable. The <span class="yellow">patient</span> had an uneventful but disappointing convalescence. She remained markedly cyanotic and no continuous bruit was audible. When she left the hospital 14 days after operation there was no improvement in her appearance, hematocrit, or arterial oxygen saturation. <br><br> Surprisingly enough she reported progress on each follow-up examination. By the end of seven weeks she was obviously less cyanotic, her hematocrit was 77, and she was able to walk a number of blocks and to climb a flight of stairs without difficulty. She reported that she had danced and roller-skated without much trouble. Shortly thereafter a continuous murmur was audible in the left chest. She continued to improve and in January entered a southern college. She did well. At least once a day, often twice, she walked without difficulty from the campus into town and back, a distance of ten blocks each way. She played a little tennis, learned to swim, and began to dance, including jitterbugging. When she xvas seen in June, her color was good, although there was still slight cyanosis of lips and nail beds. There was a very loud, continuous murmur in the left chest. The following fall she transferred to a midwestern university. She got along reasonably well but not as well as she had in a warmer climate and on more level terrain. She walked four or five blocks up and down hills between classes without difficulty in good weather but complained of some dyspnea and fatigue on cold, windy days. She attended dances and often danced each number throughout the evening without trouble. When seen in December she looked well. Her color was good and clubbing was definitely less marked than it had been previously. She had a severe cold at the time. Her oxygen saturation of arterial blood was 75 per cent at rest and it did not fall when she stood or still-walked. The hematocrit was 69. There was audible the same loud continuous murmur in the left chest. <br><br> Though the <span class="yellow">patient</span> has been markedly improved, it is recognized that the result is not as good as is commonly obtained when <span class="yellow">patients</span> with more adequate pulmonary arteries are treated by the conventional anastomosis of a systemic artery to the side of the pulmonary artery. Nevertheless, the <span class="yellow">patient</span> has thus far been given such good health and relatively normal capacity for ordinary activity that further operation has seemed unwarranted, though the possibility of some future attempt at creation of an additional shunt is being kept in mind. <br><br> The second procedure embodies the cephalad transplantation of the pulmonary artery by a plastic repair of the incision in the hilar and mediastinal pleural structures. <br><br> 488 <br><br> FIG. 1. Drawing illustrating the condition which existed in the first case after completion of the anastomosis (A) and its correction by transplantation of the origin of the subclavian (B). <br><br> FIG. 2. Drawing illustrating correction of obstruction to blood flow through the subclavian artery by inverted T or L plastic closure of the defect in the hilar and mediastinal structures. <br><br> TREATMENT OF TETRALOGY OF FALLOT <br><br> Case reports <br><br> The first <span class="yellow">patient</span> was a fully grown young <span class="yellow">man</span> of 17 with tetralogy of Fallot which caused considerable incapacity. He was admitted to the hospital on July 18, 1950 and was operated upon two days later. He was known to have a left aortic arch, and a left lateral thoracotomy was performed. The aorta was freed for a side-to-side anastomosis with the pulmonary artery. So much difficulty was encountered, however, in placing the aortic clamp in proper position for making a satisfactory incision in the aorta that the procedure was abandoned and an end-to-side subclavian-pulmonary artery anastomosis accomplished instead. The pulmonary artery was of fair size, having an estimated diameter of 1 cm. The subclavian artery appeared to be quite long and it was of very satisfactory size, having a diameter of about 6 mm. To my dismay, the first portion of the soft-walled subclavian artery was completely flattened out against the rather rigid wall of the aorta and no blood flow through it could be demonstrated, there being no subclavian pulsation nor thrill in the pulmonary artery. If one forcefully elevated the hilum of the lung in a cephalad direction, the obstruction disappeared, the subclavian artery began to pulsate, and a thrill could be palpated. When the lung was allowed to assume its usual position, the subclavian obstruction was again evident and could not be prevented by full expansion of the lung. It was found that the hilar region and the pulmonary artery could be maintained in a satisfactory cephalad position by traction upward upon the cuff of hilar pleura and the adjacent vascular sheath and that these structures were sufficiently strong so that traction could be maintained upon them with a small hemostat. The defect in the mediastinal and hilar tissues was then repaired using a sort of inverted T-plastic closure. By this maneuver success was achieved in elevating the pulmonary artery in a cephalad direction so that the subclavian obstruction was relieved and excellent pulsation was evident (Fig. 2B). A fairly good continuous thrill was palpable. The <span class="yellow">patient</span> had an uneventful convalescence. When last seen on January 16, 1951 he had excellent color without any visible cyanosis. The clubbing seemed to have decreased somewhat. A continuous murmur was audible. He stated that he noted no limitation of exercise capacity. In outlining his activities he said that, among other things, he was doing a great deal of ice-skating and was playing ice hockey regularly. He was planning to start college work the following month. <br><br> The second <span class="yellow">patient</span> was a 16-year-old <span class="yellow">boy</span> who had been cyanotic since birth. His physical development was somewhat retarded and he was slow in learning to sit and walk. Until he had grown old enough to be self-conscious about it he had always squatted when he was tired. By perseverance he had managed to do more than one would have suspected he could from the degree of his cyanosis. He could walk as much as five or six blocks at a slow pace. He was fond of drums and managed to play occasionally with an orchestra in a somewhat restricted fashion. For the past few months he had had more dyspnea, fatigability, and seemed to be going downhill generally. <br><br> The results of physical and laboratory studies were rather typical of the tetralogy of Fallot. Clubbing was marked, the nailbeds and mucous membranes were a rather deep purplish-blue color, and the skin had a dusky cyanotic tint. The aortic arch was determined to be on the left. On July 21, 1950 a left lateral thoracotomy was performed, the pleural cavity being entered through the bed of the fourth rib. There was a rather marked increase in the collateral circulation in the mediastinum and the hilar region. The pulmonary artery was easily dissected free. It was fairly long and was about 1 cm. in diameter. The subclavian artery seemed quite long and was of adequate size, having a diameter of about 5 mm. An end-to-side sub9lavian pulmonary anastomosis was performed. Again in this case, however, the first part of the subclavian was acutely angulated and completely flattened out against the aortic wall. There was no <br><br> 489 <br><br> YALE JOURNAL OF BIOLOGY AND MEDICINE <br><br> pulsation in the subclavian and no thrill in the pulmonary. In this instance also, good blood flow through the subclavian artery was evident whenever the hilar structures were elevated in a cephalad direction but no pulsation was demonstrable simply by expanding fully the lung. Again, a sort of inverted T-plastic closure of the defect in the hilum and mediastinum brought about a cephalad elevation of the pulmonary artery so that tension was released and there was excellent pulsation in the subclavian artery. A thrill was now palpable. Convalescence was uneventful. Improvement in color was evident within a few days and his color was excellent by the time he was discharged from the hospital on the thirteenth postoperative day. He rapidly found that he was now able to lead a quite normal sort of life. When he was seen on December 9 he stated that he had no limitation in exercise capacity. He could walk rapidly without fatigue. He was going to school and was playing the drums in a professional orchestra three or four nights each week. There was some diminution in the clubbing and his color was excellent. There was a loud continuous bruit audible in the chest. On March 27 the arterial oxygen saturation was 88.2 per cent. <br><br> When I first used this procedure I was rather surprised that such tissues would hold sutures and serve satisfactorily to elevate the hilum. On occasions I had previously toyed with the idea of suturing hilar pleura to the mediastinal pleura but had abandoned it as impractical because the sutures seemed to pull out whenever there was any tension whatsoever. If the procedure is to be successful, it is essential that the sutures encompass any adjacent areolar and fibrous tissue and especially the so-called vascular sheath which surrounds both pulmonary artery and aorta and is dissected free during the course of the operation. Fortunately, in its proximal portion the sheath about the pulmonary artery gains added strength from the extension into it of a reflection of the fibrous pericardium. Sutures through the mediastinal pleura in the region of the aortic arch purposely include the perivascular tissues which have been stripped off the aorta and subclavian artery and also any other available tissue which may lend strength, such as the divided ends of the supreme intercostal vein or other vessels which lhave been transected and ligated. The exact method of repair will vary from case to case. By placing the sutures properly it would seem possible sometimes to displace the pulmonary artery laterally as well as in a cephalad direction if such a maneuver was thought to be desirable (Fig. 2C). On occasions one would close the pleural defect fairly snugly, on others leave it wide open in places. <br><br> Discussion <br><br> Though there is always available some suitable systemic vessel and though the major concern in the operative treatment of the tetralogy of Fallot is the adequacy of the pulmonary artery, from time to time one may find the achievement of a satisfactory result thwarted by the local anatomical characteristics regardless of one's choice of procedure. Consequently, those modifications which may add to the likelihood of a successful outcome are important. Blalock"' pointed out the practicability of performing an end-toend subclavian-pulmonary artery anastomosis whenever the pulmonary artery is judged too small or the subclavian too short for a satisfactory end<br><br> 490 <br><br> TREATMENT OF TETRALOGY OF FALLOT             491 <br><br> to-side anastomosis. Holman' feels that a poorly functioning shunt after end-to-side anastomosis can usually be corrected by proximal division of the subclavian artery, thus in effect converting the procedure into an end-to-end anastomosis. If a satisfactory end-to-side suture of subclavian and pulmonary arteries or side-to-side aortic pulmonary anastomosis seems difficult to achieve, one may occasionally find it useful to divide the upper lobe branch of the pulmonary artery and carry out an end-to-end suture of the subclavian and the proximal end of the upper lobe branch. Potts and Smith' performed an anastomosis between the proximal end of the upper lobe branch and the side of the aorta in a case in which complete temporary occlusion of the main pulmonary artery was withstood poorly. I have found this principle of division of the upper lobe branch and use of its proximal end of value in obtaining a suitable subclavian pulmonary anastomosis when the subclavian seemed to have inadequate length. On occasions when the systemic vessel seems too short, one may elect to interpose a free vascular transplant,3"' a modification I first employed in 1946 though unfortunately not with success in this instance. <br><br> The operation performed in my first <span class="yellow">patient</span> constitutes in reality the use of the subclavian artery as an autogenous graft between the aorta and pulmonary artery. It will obviously not often be the procedure of choice but sometimes may be found a useful measure in converting an apparently inadequate functional shunt into a good one. If my initial experiences with the plastic repair of the defect in the mediastinal and hilar structures are characteristic of what may be expected of this procedure, it would seem to have wide applicability whenever a poorly functioning subclavian-pulmonary shunt seems correctable by cephalad transplantation of the hilar structures and the consequent release of tension. I was unaware of any reference in the literature to its use until belatedly I discovered that I had overlooked a statement in the legend of one of the excellent drawings in Blalock's paper on surgical procedures in pulmonic stenosis.' Here he states that suture of the pleura of the superior aspect of the hilum to the mediastinal pleura may effectively elevate a little the pulmonary artery. Perhaps our more detailed consideration of this maneuver may add to its general usefulness. <br><br> REFERENCES <br><br> 1 Blalock, A.: The technique of creation of an artificial ductus arteriosus in the <br><br> treatment of pulmonic stenosis. J. Thorac. Surg., 1947, 16, 244. <br><br> 2 Blalock, A.: Surgical procedures employed and anatomical variations encountered <br><br> in the treatment of congenital pulmonic stenosis. Surg., Gyn. Obst., 1948, 87, 385. <br><br> 3 Gross, R. E., Bill, A. H., Jr., and Pierce, E. C.: Methods for preservation and <br><br> transplantation of aortic grafts. Observation on arterial grafts in <span class="yellow">dogs</span>. Report of transplantation of preserved arterial grafts in nine <span class="blue">human</span> cases. Surg., Gyn. Obst., 1949, 88, 689. <br><br> 4 Holman, E.: The surgery of pulmonary stenosis. Experiences with left subclavian <br><br> to left pulmonary artery anastomosis. J. Thorac. Surg., 1949, 18, 827. <br><br> 5 Johnson, J., Kirby, C. K., Greifenstein, F. E., and Costillo, A.: The experimental <br><br> and clinical use of vein grafts to replace defects of large arteries. Surgery, 1949, 26, 945. <br><br> 492             YALE JOURNAL OF BIOLOGY AND MEDICINE <br><br> 6 Lam, C. R.: The choice of the side for approach in operations for pulmonary <br><br> stenosis. J. Thorac. Surg., 1949, 18, 661. <br><br> 7 Olim, C. B.: Experiences in the surgical treatment of congenital pulmonary <br><br> stenosis. American Surgeon, 1951, 17, 245. <br><br> 8 Paine, J. R. and Varco, R. C.: Experiences with the surgical treatment of pul<br><br> monic stenosis. Surgery, 1948, 24, 355. <br><br> 9 Potts, W. J. and Smith, S.: New surgical procedures in certain cases of congenital <br><br> pulmonary stenosis. Arch. Surg., 1949, 59, 491. <br><br> 10 Potts, W. J., Smith S., and Gibson, S.: Anastomosis of the aorta to a pulmonary <br><br> artery; certain types in congenital heart disease. J. Am. M. Ass., 1946, 132, 627. <h3>pmcA280693</h3>p8 inhibits the growth of <span class="blue">human</span> pancreatic cancer cells and its expression is induced through pathways involved in growth inhibition and repressed by factors promoting cell growth
Abstract
Background
p8 is a stress-induced protein with multiple functions and biochemically related to the architectural factor HMG-I/Y. We analyzed the expression and function of p8 in pancreatic cancer-derived cells.<br><br>Methods
Expression of p8 was silenced in the <span class="blue">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 by infection with a retrovirus expressing p8 RNA in the antisense orientation. Cell growth was measured in control and p8-silenced cells. Influence on p8 expression of the induction of intracellular pathways promoting cellular growth or growth arrest was monitored.<br><br>Results
p8-silenced cells grew more rapidly than control cells transfected with the empty retrovirus. Activation of the Ras→Raf→MEK→ERK and JNK intracellular pathways down-regulated p8 expression. In addition, the MEK1/2 inhibitor U0126 and the JNK inhibitor SP600125 up-regulates expression of p8. Conversely, p38 or TGFβ-1 induced p8 expression whereas the specific p38 inhibitor SB203580 down-regulated p8 expression. Finally, TGFβ-1 induction was in part mediated through p38.<br><br>Conclusions
p8 inhibits the growth of <span class="blue">human</span> pancreatic cancer cells. p8 expression is induced through pathways involved in growth inhibition and repressed by factors that promote cell growth. These results suggest that p8 belongs to a pathway regulating the growth of pancreatic cancer cells.<br><br><br><br>Background
While studying the molecular response of the injured pancreas, we identified a new gene, called p8, whose expression is strongly induced during the acute phase of pancreatitis [1]. Further experiments have shown that p8 mRNA is activated in almost all cells in response to several stresses [2], including minimal stresses such as after routine change of the culture medium in the absence of any added substance [3], indicating that p8 is a ubiquitous protein induced by cellular stress. The p8 gene was cloned in <span class="blue">human</span>, <span class="yellow">rat</span>, <span class="yellow">mouse</span>, and <span class="yellow">Xenopus laevis</span> [1,4-6], conceptually translated from the <span class="yellow">Drosophila melanogaster</span> genome or deduced from EST libraries (<span class="yellow">Bos taurus</span>, <span class="yellow">Xenopus tropicalis</span>, <span class="yellow">Zebrafish</span>, <span class="yellow">Orzzias latipes</span>, <span class="yellow">Bombyx mori</span> and <span class="yellow">Paralichthys olivaceous</span>). The overall degree of homology with <span class="blue">human</span> p8 ranged from 81 to 40%. Secondary structure prediction methods indicated that within the homologous region of the eleven proteins, there is a basic Helix-Loop-Helix secondary structure motif, characteristic of some classes of transcription factors [1]. Even though a small protein such as p8 would not need a nuclear localization signal (NLS) to be transported to the nucleus, a clear NLS can be predicted for the eleven proteins comprising a bipartite domain of positively charged aminoacids. In addition, a nuclear/cytoplasmic location has been demonstrated for <span class="blue">human</span> p8 upon overexpression of the recombinant protein and immunohistochemistry [4], and for recombinant <span class="yellow">Xenopus laevis</span> p8 fused to green fluorescent protein [6]. Homology searching in databases did not reveal significant similarity of p8 with other proteins of known function. However, biochemical properties of the mammalian p8 proteins are shared by some high mobility group proteins (HMG) [7], particularly by the HMG-I/Y family. The overall identity of <span class="blue">human</span> p8 with <span class="blue">human</span> HMG-I/Y is only about 35%, but the molecular mass, isoelectric point, hydrophilicity plot, the resistance to denaturation after heating at 100°C and the charge separation are very similar [8]. The p8 protein seems to bind DNA weakly, as shown by electrophoretic mobility shift assay, without preference for DNA sequences. Finally, <span class="blue">human</span> p8 has also been shown to be a substrate for protein kinase A in vitro and phosphorylated p8 has a higher content of secondary structure and binding to DNA is highly increased [8]. An architectural role in transcription has been proposed for this protein, in analogy with the HMG-I/Y proteins, and a recent work seems to confirm this hypothesis [9].
Functions of p8 appear to be multiple and complex. For example, p8 mRNA expression was strongly induced in 3T3 cells upon TGFβ-1 treatment which in turn enhances the Smad-transactivating function responsible for TGFβ-1 activity [10]. We also found that p8 is involved in cell cycle regulation since p8-deficient embryonic fibroblasts grew more rapidly and incorporated more [3H] thymidine and BrdU than p8-expressing cells [11]. Moreover, expression of p8 in breast cancer-derived cells seems to mediate the inhibition of cell growth induced by 1,25-Dihydroxyvitamin D3 [12]. On the contrary, we also reported that p8 may promote cell growth when overexpressed in Cos-7, AR42J and HeLa cells [1,4]. In addition, p8 seems to be involved in other intracellular functions such as apoptosis since p8-expressing fibroblasts are more sensitive than p8-deficient fibroblasts to the apoptosis induced by DNA damage. Also, p8 is required for endothelin-induced mesangial cell hypertrophy in diabetic kidney, in a mechanism involving ERK, JNK and PI3 kinase [13]. p8 seems to play a functional role in the initiation of LHβ gene expression during embryonic cell differentiation [14]. Moreover, the <span class="yellow">Drosophila melanogaster</span> p8 homologue is involved in response to starvation and might be activated to stop cell growth in case of nutrient deprivation [15]. Finally, a particularly attractive role in tumour progression was recently proposed for p8 [16]. Fibroblasts obtained from p8-expressing or p8-deficient animals were transformed with a retroviral vector expressing both the rasV12 mutated protein and the E1A adenoviral oncogene. In soft-agar assays, transformed p8-expressing cells formed colonies at high frequency, as expected, but p8-deficient transformed fibroblasts were unable to form colonies. Similarly, transformed p8-expressing cells produced tumours in all athymic <span class="yellow">nude mice</span> when injected subcutaneously or intraperitoneally, whereas transformed p8-deficient fibroblasts did not. On the other hand, studies by another laboratory revealed that expression of the Com1 protein [17], which is identical to <span class="blue">human</span> p8, mediates the growth of tumour cells after metastatic establishment in a secondary organ, indicating that activated expression of Com1/p8 in metastatic cells is required for tumour progression. These results strongly suggest that p8 is involved in the cellular pathway(s) required for tumour progression and metastasis.
Our aim is to check the relevance of p8 to cancer progression in <span class="blue">human</span>. As a first step, we investigated in the present study the function of p8 in two cell lines derived from <span class="blue">human</span> pancreatic cancer. We observed that inhibition of p8 expression increased the cells growth rate. In addition, activations of the Ras→Raf→MEK→ERK and JNK intracellular pathways, which promote the growth of pancreatic cells, down-regulated p8 expression, whereas activation of p38 or TGFβ-1, which inhibit cell growth, induced its expression. It was concluded that i/ p8 inhibits the growth of <span class="blue">human</span> pancreatic cancer cell lines, ii/ p8 expression is induced through pathways involved in growth inhibition and, conversely, repressed by factors that promote cell growth.<br><br>Results
p8 is silenced in pancreatic cancer cells by infection with a retrovirus expressing p8 RNA in the antisense orientation
Panc-1 and BxPc-3 pancreatic cells were chosen for this study because, on the one hand, both cells express higher level of p8 (Figure 1) and, on the other hand, because Panc-1 is wild-type for Smad4/DPC4 and mutated for K-ras, while BxPc-3 is Smad4/DPC4 mutated and K-ras wild-type [18,19], therefore representing different mechanisms of transformation and different genetic backgrounds. K-ras and Smad4/DPC4 mutations are the major mechanisms involved in pancreatic cancer development. We inhibited p8 expression in both Panc-1 and BxPc-3 pancreatic cells by infecting cells with a retrovirus expressing the p8 asRNA (antisense RNA) and carrying the puromycin resistance. The antibiotic-selected cells were analyzed by Western blotting to evaluate the intracellular amounts of p8 protein. As shown in Figure 2, the p8 protein was clearly visible in both Panc-1 and BxPc-3 pancreatic cells infected with the empty retrovirus but almost undetectable in cells infected with the retrovirus encoding the p8 asRNA showed, indicating that our anti-sense strategy is efficient to silence p8 gene expression in pancreatic cancer cells. Preliminary studies had been conducted to select the best strategy to inhibit p8 expression. We compared the efficacy of the stable transfection of a siRNA, using a retroviral expression vector to the asRNA strategy described above. In our hands, the antisense strategy worked best, as judged from Western blot assessment of p8 protein expression (data not shown).<br><br>p8-silenced pancreatic cells grow more rapidly
We compared in the two cell lines the influence on growth parameters of blocking p8 expression with the p8 asRNA. Figure 3 shows that both Panc-1 and BxPc-3 cells in which p8 has been silenced grew more rapidly than cells infected with the empty vector suggesting that inhibition by p8 of pancreatic cancer cell growth is independent from the mechanism of transformation and genetic background.<br><br>Serum-stimulated cellular growth down-regulates p8 expression
Fetal <span class="yellow">calf</span> serum, which contains a complex mix of growth factors, can be used as inductor of cell growth. As shown in Figure 4 expression of p8 mRNA was down-regulated in both Panc-1 and BxPc-3 when the cells were shifted from culture media containing 0.1% fetal <span class="yellow">calf</span> serum to media containing 10% FCS. p8 protein showed a similar behavior. These results show that p8 expression is down-regulated in growing pancreatic cells.<br><br>The Ras→Raf→MEK→ERK pathway down-regulates p8 expression in pancreatic cancer cells
Most <span class="blue">human</span> pancreatic cancers harbor mutations in the K-ras oncogene, which happens relatively early in pancreatic tumorigenesis [20]. The oncogenic mutation of the K-ras gene stabilizes the Ras protein in a GTP-bound form, which is constitutively active and make the cells grow more rapidly. Contrary to the activated Ras protein, p8 inhibits cell growth (Figure 3). We looked whether the Ras→Raf→MEK→ERK pathway was also involved in the regulation of p8 expression, and which step(s) were critical. Figure 5 shown that expression of a mutated form of the Ras protein (rasV12) in BxPc-3 cells, which are wild-type for ras, resulted in decreased p8 mRNA concentration and protein level suggesting that the activated ras inhibits p8 expression. Figure 6 shows that overexpression of Raf, but not of Raf301 (a negative mutant of Raf), and of ERK also inhibited the expression of the p8-CAT construct in Panc-1 as well as in BxPc-3. Finally, the MEK1/2 specific inhibitor U0126 [21] activated p8 mRNA expression in pancreatic cells whether they carry mutated ras (Panc-1) or wild-type (BxPc-3). Similar results were observed when expression of the p8 protein was monitored by Western blotting (Figure 7).<br><br>Activation of the JNK pathway down-regulates p8 expression in pancreatic cancer cells
c-Jun NH2-terminal kinase (JNK) is another major MAPK pathway which converts extracellular signals into expression of specific target genes through phosphorylation and activation of transcription factors. JNK activation has been implicated in various, often opposite cellular responses, such as cell proliferation, transformation and apoptosis. As shown in Figure 8, overexpression of JNK down-regulates the gene reporter activity of the p8-CAT construct in Panc-1 cells. Similar results were found in BxPc-3 cells. Treatment of these cells with the JNK specific inhibitor SP600125 [22] up-regulates expression of the p8 mRNA and p8 protein (Figure 9). These results show that the JNK pathway is involved in the regulation of p8 expression.<br><br>The p38 pathway up-regulates p8 expression in pancreatic cancer cells
The p38 signal transduction pathway also plays an essential role in regulating several cell functions including growth, response to inflammation, differentiation and apoptosis. In fact, in pancreatic cancer cells, p38 is a strong inhibitor of proliferation [23] contrary to the Ras→Raf→MEK→ERK and JNK pathways. We therefore analyzed the putative role of the p38 pathway in regulating p8 expression in pancreatic cancer cells. Figure 10 shows that over-expression of the plasmid encoding p38 significantly increases p8-CAT activity in Panc-1 as well as in BxPc-3 cells. Then, cells were treated with SB203580, a specific inhibitor of p38 [24], and p8 expression was measured. p8 mRNA as well as the encoded protein were down-regulated after inhibition of the p38 activity (Figure 11). These results indicate that the p38 pathway is a positive regulator of p8 expression in pancreatic cancer cells.<br><br>TGFβ-1 up-regulates p8 expression in pancreatic cancer cells
The most prominent biological activity of TGFβ-1 is its potent inhibition of cell growth in a wide variety of cell types including pancreatic cells. TGFβ-1 signals are sent through two types of transmembrane serine/threonine kinase receptors. In fact, TGFβ-1 binds and brings together the type I and type II receptors. In the resulting complex, the constitutively active TGFβ-1 type II receptor phosphorylates the type I receptor, which then plays a major role in transducing the signal to downstream components to affect gene expression through phosphorylation of SMAD proteins. Phosphorylated receptor-regulated SMADs then form heteromeric complexes with the common partner SMAD4. These heteromeric complexes then move to the nucleus, where SMAD4 will bind DNA and contribute to transcriptional activation. In general, pancreatic cancer cells present with defects in TGFβ-1 signaling and are resistant to TGFβ-1-mediated growth suppression. Since TGFβ-1 and p8 are inhibitors of pancreatic cell growth we analyzed whether p8 could mediate, at least in part, the effect of TGFβ-1. First, we found that treatment of Panc-1 cells with TGFβ-1 increased p8 mRNA levels and p8 protein as judged by Western blot (Figure 12). Then, to confirm that overexpression is regulated at the transcriptional level, we analyzed the effect of some constructs expressing constitutively activated type I TGFβ receptor, dominant negative type II TGFβ receptor, a dominant negative of Smad4 and the wild-type Smad4 on the p8-CAT activity. As expected, the constitutively activated type I TGFβ receptor but not the dominant negative type II TGFβ receptor increased CAT activity. Also, expression of the Smad4, contrary to that of the negative mutant, induced p8 transcription (Figure 13). Together, these results indicate that p8 is positively regulated by TGFβ-1.
Beside the Smad proteins, TGFβ-1 also activates the p38 MAPK pathway in pancreas-derived cells, which may play an important role in TGFβ-1 induced genes [25]. Therefore, we analyzed the p38-dependent effect of TGFβ-1 on p8 transcription. As shown in Figure 13, inhibition of p38 activity with the SB203580 specific inhibitor decreased about 40% the activity of TGFβ-1 on the p8 promoter indicating that the effect of TGFβ-1 on p8 promoter is mediated by both p38-dependent and p38-independent pathways.<br><br>
Discussion
Pancreatic adenocarcinoma is the fourth leading cause of death from malignant diseases [26]. The aggressive nature of the neoplasia, the lack of early detection, and the limited response to treatments such as chemotherapy and radiotherapy contribute to the high mortality rate of the disease. Therefore, a better understanding of the molecular mechanism leading to pancreatic cancer remains a major goal because it may help proposing strategies for earlier diagnosis and better treatment. The most commonly altered genes in pancreatic adenocarcinoma are K-ras (75 to 100%), p16INK4a (95%), p53 (50 to 75%) and DPC4 (50%) [27-31]. Whereas K-ras is a proto-oncogene all the others are tumour suppressor genes. Additional genes have been found altered at lower frequency. Panc-1 and BxPc-3 pancreatic cells were chosen for this study because they both express high levels of p8 (Figure 1) and because they present with different mechanisms of transformation and genetic backgrounds, Panc-1 being wild-type for Smad4/DPC4 but mutated for K-ras and BxPc-3 mutated for Smad4/DPC4 and wild-type for K-ras [18,19]. This work presents evidences that p8 inhibits the growth rate of pancreatic cancer-derived cells and that the intracellular pathways promoting cell growth down-regulate p8 expression whereas those promoting growth arrest up-regulate its expression. Together, these results suggest that p8 is downstream of some cell growth regulators and therefore regulation of p8 expression or its activity could be used as a target for treating pancreatic cancer.
Silencing p8 expression was able to strongly promote cell growth in both cell types, Panc-1 and BxPc-3, suggesting that p8 may act downstream of the ras- or Smad4/DPC4-dependent ways. Also, we found that stimulating cell growth by the complex combination of growth factors contained in fetal <span class="yellow">calf</span> serum down-regulated expression of p8 whereas, on the contrary, treating the cells with TGFβ-1, which promotes cell cycle arrest, stimulates p8 expression. Therefore, p8 gene expression seems to be regulated in opposite directions by mechanisms promoting cell growth or cell cycle arrest. It is interesting to note that while p8 expression is under the control of cell growth regulatory pathways such as Ras→Raf→MEK→ERK, JNK, p38 and TGFβ-1, p8 can affect cell cycle progression, suggesting that p8 is a target for factors regulating pancreatic cell growth.
A mechanism by which p8 could regulate cell cycle progression in embryonic fibroblasts was previously proposed [11]. In fact, p8 seems to take action upstream from cyclin-dependent kinases because the intracellular levels and activities of Cdk2 and Cdk4 are decreased when p8 is expressed. Concomitantly, the cyclin-dependent kinase inhibitor p27 is expressed at a low level in p8-deficient cells which may explain the increased activity of Cdk2 and Cdk4. The mechanism by which p8 regulates the intracellular level of those proteins remains to be determined. However, because p8 is a transcriptional cofactor, it is possible that regulation of expression of these molecules takes place, at least in part, at the transcription level.
Interestingly, expression of p8 mRNA seems to be regulated in a cell type- and stimulus-specific manner since, for example, p38 can induce p8 expression in response to stress in fibroblasts [3] but not in renal mesangial cells treated with endothelin [13]. In pancreatic cancer-derived cells p38 seems to play a major role since it is involved in p8 activation as judged by transient transfection assays and using a specific p38 inhibitor (Figures 10 and 11). In addition, p38 is also involved in TGFβ-1-induced p8 expression because about 40% of the TGFβ-1 effect was abolished when p38 activity was specifically blocked (Figure 13). On the other hand, ERK and JNK are inducers of p8 expression in mesangial cells treated with endothelin, but not involved in the activation of p8 in response to stress in fibroblasts [3], and even repressors in pancreatic cells (Figures 5, 6, 7, 8 and 9). Finally, PI3 kinase is an inducer of p8 expression in both endothelin-mediated p8 activation in mesangial cells [13] and pancreatic cells (data not shown).
Based on these observations, overexpression of p8 could be considered a possible goal for treating pancreatic tumours, in order to limit their growth. However, we previously reported that p8 repression would prevent rasV12/E1A transformed fibroblasts from evolving as tumours in <span class="yellow">nude mice</span> [16]. This apparent contradiction needs to be resolved before considering p8 as a target for treating cancer progression.<br><br>Conclusions
In conclusion (see Figure 14), we report in this paper that inhibition of p8 expression by an anti-sense strategy increases the growth rate of both Panc-1 and BxPc-3 pancreatic cancer-derived cells. Moreover, ERK- and JNK-mediated pathways down-regulate p8 expression, whereas p38 and TGFβ-1 pathways induce p8 expression. Also, cell growth triggered by expression of a RAS mutated protein or by 10% fetal <span class="yellow">calf</span> serum induces down-regulation of p8 expression. Together, these results indicate that p8 is an intracellular cell growth inhibitor and that it is oppositely regulated by growth-promoting or growth-inhibiting factors in pancreatic cancer-derived cells.<br><br>Material and Methods
Cell lines and cell culture conditions
The <span class="blue">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 were a kind gift of Dr C. Susini (INSERM U.531, Toulouse) and A. Hajri (IRCAD, Strasbourg) respectively. Panc-1 cells were grown in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal <span class="yellow">calf</span> serum, 2 mM L-glutamine, 100 IU/ml penicilin G and 100 μg/ml streptomycin. BxPc-3 were cultivated in RPMI 1640 medium in the presence of 2 mM L-glutamine, 4.5 g/L glucose, 10 mM Hepes, 1.0 mM sodium pyruvate, 10% fetal <span class="yellow">calf</span> serum and 100 IU/ml penicilin G and 100 μg/ml streptomycin. <span class="yellow">Human</span> recombinant TGFβ-1 was obtained from Sigma, and specific SB203580, U0126 and SP600125 inhibitors were from Calbiochem and utilized at 10 μM.<br><br>Expression plasmids
Expression plamids encoding p38 (pCEFL HA p38), Erk2 (pcDNAIII HA ERK2), JNK (pcDNAIIIB HA JNK), the wild-type Raf (pcDNA RAF BXB) and the Raf dominant negative (pcDNA RAF 301 K375W) were obtained from O Coso (University of Buenos Aires). Plamids encoding the constitutively activated type I TGFβ receptor (RI ACT), the dominant negative type II TGFβ receptor (RII DN) and the Smad4 dominant negative (DPC4 1–514 a.a.) were obtained from R Urrutia (Mayo Clinic, Rochester) and the wild type Smad4 was from C Heldin (Ludwig Institute, Uppsala).<br><br>Pancreatic p8-deficient cells
To silence p8 expression in pancreatic cells, we infected these cells with a retrovirus expressing <span class="blue">human</span> p8 in the antisense orientation. The retroviral vector was constructed as follows: <span class="blue">human</span> p8 cDNA was subcloned in HindIII and XhoI restriction sites of the pLPC plasmid (obtained from S. Lowe) in the antisense orientation. Amphotrope <span class="blue">human</span> p8 expressing retrovirus was then generated by transient transfection using Phoenix amphotrope packaging cells. Viral supernatant was used to infect Panc-1 and BxPc-3 pancreatic cells and the population of p8-silenced cells was isolated by selection in presence of puromycin (1 μg/ml). As control, cells were infected with the pLPC empty vector.<br><br>p8 expression in arrested and growing cells
One million of Panc-1 or BxPc-3 cells were cultivated on 10-cm Petri plates in standard conditions (with 10% FCS). After 48 h, culture media were changed for fresh media with FCS restricted to 0.1%, in order to stop growth. After 24 hours of growth arrest, culture medium was replaced either by medium with 10% fetal <span class="yellow">calf</span> serum to resume cell growth or, as control, by medium with 0.1% fetal <span class="yellow">calf</span> serum. Twenty four hours later cells were recovered and RNA and protein extracted.<br><br>p8 expression in TGFβ-1-treated Panc-1 cells
One million of Panc-1 cells were cultivated in 10-cm culture dishes for 48 hours under standard conditions before TGFβ-1 treatment. <span class="yellow">Human</span> recombinant TGFβ-1 (5 ng/ml) was added to cells, without changing the culture medium, and cells were collected 12 hours later for RNA and protein preparation.<br><br>BxPc-3 rasV12-expressing cells
pLPC-rasV12 and pLPC plasmids were obtained from S. Lowe. Phoenix amphotrope packaging cells (106) were plated in a 6-well plate, incubated for 24 hours, then transfected with PEI with 5 μg of retroviral plasmid. After 48 hours, the medium containing virus was filtered (0.45 μm filter, Millipore) to obtain the viral supernatant. Target BxPc-3 were plated at 2 × 105 cells per 35-mm dish and incubated overnight. For infections, the culture medium was replaced by an appropriate mix of the viral supernatant and culture medium (V/V), supplemented with 4 μg/ml polybrene (Sigma), and cells were incubated at 37°C. BxPc-3 rasV12-expressing cells were selected with puromycin (1 μg/ml). Cells infected with the pLPC empty vector were used as control.<br><br>Western-blot analyses
One hundred μg of total protein extracted from cells was separated with standard procedures on 15.0% SDS-PAGE using the Mini Protean System (Bio-Rad) and transferred to a nitrocellulose membrane (Sigma). The intracellular level of p8 was estimated by Western blot using a polyclonal antibody (1:1000) raised against <span class="blue">human</span> p8 [4].<br><br>Growth curves
One hundred thousand cells per well were plated in a series of 35-mm culture dishes. The cell number was estimated daily in triplicate, during 1 to 5 days, in a haemocytometer. Within experiments, each point was determined at least two times.<br><br>Cell transfection and gene reporter assays
Panc-1 and BxPc-3 (105) were cultivated in 30 mm diameter culture dishes for 24 hours then transiently transfected with 0.5 μg of p8-CAT reporter plasmid and 0.5 μg of pCMV/βgal plasmid (to control transfection efficiency) using the Fugene reagent in accordance with the manufacturer's protocol (Roche Molecular Biochemicals). The p8-CAT plasmid is the previously reported p-1471/+37p8-CAT promoter construct [5]. Reporter activities were measured as previously described [5]. Briefly, cell extracts were prepared with the reporter lysis buffer (Promega) 24 hours after transfection and CAT activity was determined by the phase extraction procedure [32] and β-galactosidase assay was performed essentially as described in Sambrook et al. [33]. CAT activity was normalized to β-galactosidase activity. Experiments were carried out in triplicate and repeated at least two times. Expression plasmids (0.5 μg) were co-transfected with p8-CAT and pCMV/βgal plasmids as indicated.<br><br>RT-PCR analysis
RNA was extracted using the Trizol (Life Technologies) procedure. Total RNA (1 μg) was analyzed by RT-PCR with the SuperScript™ One-step RT-PCR System and the Platinum Taq kit (Life Technologies). RT-PCR was performed using different numbers of cycles to verify that the conditions chosen were within the linear range. The mRNA coding for p8 was specifically amplified with sense (5' GAAGAGAGGCAGGGAAGACA 3') and antisense (5' CTGCCGTGCGTGTCTATTTA 3') primers, in positions 72 and 643 of the cDNA (accession # NM_012385), respectively. As control, the transcript coding for the ribosomal protein RL3 was specifically amplified for 22 cycles with sense (5'GAAAGAAGTCGTGGAGGCTG3') and antisense (5'ATCTCATCCTGCCCAAACAC3') primers, in positions 216 and 637 of the cDNA, respectively.<br><br>
Author's contributions
CM prepared cells and retrovirus, carried out RNA purification, RT-PCR, Western blots, and cell growth experiments, NL carried out CAT assays, SV participated in the design of the study and analysis of data, JLI participated in the analysis of data and wrote the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA328328</h3>Mapping irradiation hybrids to cosmid and <span class="yellow">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products.
Abstract
A direct hybridization protocol is described for screening cosmid and <span class="yellow">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual Alu-PCR product before library screening. A series of <span class="blue">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis.Images<br><br><br><br><br><br>
 Nucleic Acids Research, Vol. 19, No. 12 3315 <br><br> Mapping irradiation hybrids to cosmid and <span class="yellow">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products <br><br> Anthony P.Monaco*, Veronica M.S.Lam2, Gunther Zehetner, Gregory G.Lennon, Christal Douglas, Dean Nizetic, Peter N.Goodfellow1 and Hans Lehrach <br><br> Genome Analysis Laboratory and 'Molecular <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, 44 Lincoln's Inn Fields, London WC2A 3PX, UK and 2Department of Biochemistry, Li Shu Fan Building, Sassoon Road, University of Hong Kong, Hong Kong <br><br> Received March 12, 1991; Revised and Accepted May 16, 1991 <br><br> ABSTRACT <br><br> A direct hybridization protocol is described for screening cosmid and <span class="yellow">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual AluPCR product before library screening. A series of <span class="blue">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis. <br><br> INTRODUCTION <br><br> The generation of <span class="blue">human</span> DNA probes specific for individual chromosomes and subregions of chromosomes has been advanced with Alu-sequence primed polymerase chain reaction amplification (Alu-PCR, 1-3). This method specifically amplifies sequences between Alu repeats from <span class="blue">human</span> DNA in somatic cell hybrids and <span class="yellow">yeast</span> artificial chromosomes (YACs, 4). Individual Alu-PCR products can be purified from agarose gels or ligated into plasmid vectors to screen for single copy sequences. Unique Alu-PCR products are then localized to certain chromosome regions using DNA blots of somatic cell hybrid panels. Once localized, Alu-PCR fragments can be screened against genomic libraries to isolate longer DNA fragments from the region of interest. As an alternative to this multistep process we have developed a hybridization protocol for screening of cosmid and YAC libraries directly with pools of Alu-PCR products. <br><br> Two new <span class="blue">human</span> specific Alu primers were used to generate DNA probes from a series of irradiation-reduced hybrids containing multiple <span class="blue">human</span> X chromosome fragments of 1-2000 kb on a <span class="yellow">hamster</span> chromosome background (5; P.N.G., unpublished). The Alu-PCR products were hybridized as a pool of probes to X-specific cosmid and YAC libraries, after <br><br> competitive reassociation with an excess of <span class="blue">human</span> DNA to both the library filters and radioactively labelled Alu-PCR products. Comparisons were made between clones identified by overlapping irradiation hybrids and single copy DNA probes hybridized to the cosmid and YAC libraries. <br><br> METHODS <br><br> Two <span class="blue">human</span> Alu sequence primers were generated which were shown to be <span class="blue">human</span> specific; 3144 from the 3' end of Alu: 5'-GAGCGAGACTCCGTCTCAAA-3' and 2729 from the 5' end of Alu: 5'-GTGGATCACCTGAGGTCAGGAGTTC-3'. All PCR reactions were carried out with 100 ng of hybrid DNA and 0.7 ,^g of a single Alu primer in 100 d41 of 0.01 M Tris-HCl pH 8.3, 0.0015 M MgCl2, 0.05 M KCI, 200 AtM each of dNTPs, 10% dimethlysulfoxide, and 2.5 units of Cetus Taq polymerase. Reactions were 30 cycles of 94?C for 2 min, 57?C for 2 min, and 74?C for 4 min followed by a final extension time at 74?C for 9 min. Reactions products were analyzed on 1 % agarose gels and shown to contain between five and twenty fragments, with sizes ranging from 0.1 to 2.0 kb. <span class="yellow">Chinese hamster</span> DNA and no DNA PCR reactions were done to control for non-<span class="blue">human</span> products (data not shown). <br><br> Alu-PCR products were separated from Alu oligomers over Qiagen columns, and approximately 50-100 ng were labelled by random hexamer priming (6). The radioactively labelled pool of fragments was prehybridized with 37.5 ,^g of total <span class="blue">human</span> DNA and 12.5 tsg of <span class="yellow">hamster</span> DNA immobilized on a cellulose support matrix, prepared as previously described (7). Reactions were at 65?C in 1 ml of 0.75M NaCl, 0.05M sodium phosphate pH 7.2, 0.005M  EDTA, 0.1 % sodium dodecyl sulphate (SDS), 0.5 mg/ml heparin, and 100 jig/ml denatured <span class="yellow">salmon</span> sperm DNA. The cellulose was pelleted and the supernatant boiled for 2 min every 12-16 hours (three times in two days). <br><br> Cosmid and YAC library filters (Hybond N+, Amersham) were prehybridized at 42?C for 16 hours with 100 jig/ml <br><br> * To whom correspondence should be addressed at <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford OX3 9DU, UK <br><br> 1991 Oxford University Press <br><br> 3316 Nucleic Acids Research, Vol. 19, No. 12 <br><br> Chromosome X <br><br> .,.., ., < <br><br> _ ., . < ., <br><br> *2.' 1- 7-_ <br><br> _-_ 1. J <br><br> ,.1 .. - 1 , _ i - _ 1 . 1 <br><br> 1 ' 4 - -1 + <br><br> 1 1 L L . i <br><br> . 1 . _ 1  <br><br> 1 1 s. 1 i <br><br> 1 i ;. --r' F 1 -<br><br> 1 -1-l <br><br> j i-_1. 1 <br><br> -   _ 1 .. <br><br> s- S . i n -. <br><br> .. : ,1' ,. -1 _ E I i ? <br><br> ,. . _ _ _ <br><br> G-<br><br> . <br><br> *.-.,., z <br><br> _ <br><br> _ , <br><br> z ;- [X1 <br><br> 21 , In <br><br> 27  38 45   48  54  74   86 <br><br> I  1      I <br><br> I <br><br> I <br><br> 107 <br><br> MD <br><br> I <br><br> :I                                                       I  <br><br> I        I <br><br> I I <br><br> Figur 2a and 2b: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 48 to duplicate copies of 22 x22cm filters containing 9216 <span class="blue">human</span> X chromosome cosmids (8). 2c: Hybridization of Alu-PCR products generated with Alu primer 3144 from an independent hybrid (54) to a third identical cosmid filter. <br><br> Figure 1: A schematic diagram of the <span class="blue">human</span> X chromosome alongside the approximate cytogenetic location of fragments identified in nine irradiation hybrids (numbers across the top). The <span class="blue">human</span> X fragments were identified by hybridization of 27 known DNA markers (indicated by a black line; P.N.G, unpublished) or by cosmids in common with unique X chromosome probes in the reference library database (open boxes and Table 1). The size of the lines and boxes relate to the best cytogenetic location of the probes used according to <span class="yellow">Human</span> Gene Mapping 10.5 (14) and does not indicate the physical extent of the irradiation hybrid fragments. <br><br> denatured and sheared total <span class="blue">human</span> DNA in 50% formamide, 4XSSC, 0.05 M sodium phosphate pH 7.2, 0.001 M EDTA, 10% dextran sulphate, 1.0% SDS, 50 isg/ml denatured <span class="yellow">salmon</span> sperm DNA and OxDenhardt's solution. The radioactively labelled Alu-PCR products were denatured and added to fresh <br><br> hybridization solution without <span class="blue">human</span> DNA competitor at 1 x 106 <br><br> cpm/ml and hybridized at 42?C for 16 hours. Filters were washed in 0.1 xSSC and 1.0% SDS, twice at room temperature and twice at 65?C for 30 min each and exposed to Kodak X-OMAT film for 2-3 days at -70?C with an intensifying screen. <br><br> For each hybridization, two sets of duplicate cosmid filters were used from the ICRF reference library system (8), each containing 9216 flow-sorted <span class="blue">human</span> X chromosome cosmid clones or approximately 2 chromosome equivalents on a 22 x 22 cm filter (9). The coordinates of signals positive on duplicate cosmid filters were entered into the reference library database (G. Z, unpublished) using a digitizing tablet. For the X chromosome specific YAC library (A.P.M. and H.L., unpublished), about 420 YAC colonies were spotted manually onto filters from 96 well microtiter dishes using a 96 prong device. After growth on selective media for 3 days, YAC filters were processed for hybridization as previously described (10). <br><br> RESULTS <br><br> A panel of 195 X chromosome irradiation hybrids was constructed (50,000 rads) and characterized by DNA hybridization using 27 X chromosome markers and flourescence in situ hybridization using total <span class="blue">human</span> DNA as probe (Benham et al., 1989; P.N.G., unpublished). This analysis indicated that the hybrids contained multiple small fragments (4-10 fragments of 1000-5000 kb each) with a preferential retaining of centromere sequences (90%). From this panel, nine irradiation hybrids were chosen that contained less than five different regions by DNA probe hybridizations, mostly from the short arm of the X chromosome (Fig 1). All nine hybrids were used in PCR reactions with 3'-Alu primer 3144 and two were used with 5'-Alu <br><br> Figure 3: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 54 to a filter containing 420 YAC clones from the <span class="blue">human</span> X chromosome. The positive YAC was also identified in a separate hybridization with the DMD probe P20 (12). <br><br> primer 2729. Example hybridizations to a <span class="blue">human</span> X chromosome cosmid filter in Fig 2 shows the intensity and reliability of positive clones identified on duplicate filters with Alu-PCR products from the same irradiation hybrid (48) and the independence of clones identified with Alu-PCR products from a different hybrid (54). Fig 3 shows a single positive YAC clone after hybridization of Alu-PCR products from irradiation hybrid 54 to a filter containing about 420 YAC clones specific for the <span class="blue">human</span> X chromosome. <br><br> The total number of cosmids identified with each pool of AluPCR products for each hybrid is shown in Table 1. From the average number of cosmids identified (24) in four chromosome equivalents screened and the estimated average DNA content in each hybrid (3000-15000 kb), the Alu-PCR products generated by a single primer were calculated on average to be 300-1500 kb apart, similar to published estimates for this method (1,2). Only 3-4 cosmids were identified in common using Alu-PCR products generated with either 3' or 5' Alu primers (3144 or 2729) from two hybrids (38 and 45). This shows that separate products were amplified with the two Alu primers since they prime synthesis from opposite ends of the Alu consensus sequence and Alu sequences are oriented in the genome either head to head, <br><br> A <br><br> B <br><br> C <br><br> El <br><br> I0 <br><br> Nucleic Acids Research, Vol. 19, No. 12 3317 <br><br> Table 1. Cosmids identified by hybrids and unique probes <br><br> hybrids       21    27     38    38    45    45    48     54    74    86     107   unique <br><br> primer 3144  3144  3144  2729  3144  2729   3144  3144  3144  33144 3144   probes <br><br> 21     3144  59                                                                   1 27     3144   7    14                                                             2 38     3144   3     1    28                                                       2 38     2729   0     0     4     25                                                0 45     3144  13     2      2     0    32                                          3 45     2729   1     0      0     0     3    31                                    0 48     3144  10     2      4     0     2     0    28                              3 54     3144  11     3     0      0     5     0     5     30                       2 74     3144   5     0     0      0     4     0     5     12    20                 3 86     3144   3     1     0      0     4     1     3      4     3    17           0 107     3144   1     0      1     1     0     0     3     0      0     0    13    0 <br><br> tail to tail or head to tail relative to each other. Therefore, by using the 3'- and 5'-Alu primers in separate PCR reactions with the same hybrid DNA, the total number of products and cosmid clones identified was almost doubled. <br><br> Table 1 also indicates how many cosmids were identified by Alu-PCR products from other hybrids, and 16 cosmids previously identified with unique DNA probes in the reference library database. As can be seen in Fig 1 and previous irradiation hybrid analysis, there is a preferential retention of centromere sequences (2,11). However, there were no cosmids identified in common from all the hybrids positive with centromere sequences. This is probably due to a paucity of Alu repeats in the correct orientation in alphoid centromere sequences and thus few or no Alu-PCR products would be amplified from the centromere. <br><br> Cosmids identified in common by several irradiation hybrids (Table 1) were most likely from regions of overlap outside the centromere area as shown by the previous DNA probe characterization (Fig 1). The overlap regions between hybrids were also seen by 16 cosmids (Table 1, Fig 1) that were hybridization targets of unique DNA markers in the reference library database that mapped in independent experiments to the overlap region. At least for several cosmids this showed that the Alu-PCR products identified target cosmids that were definitely from the expected region contained in the hybrids. For example, hybrids 21 and 54 were both shown to contain part of the Duchenne muscular dystrophy (DMD) locus (Fig 1; P.N.G., unpublished) and had 11 cosmids in common, including one identified by the probe P20 from the deletion hotspot region of the DMD gene (12). In Fig 3 the hybridization of Alu-PCR products from hybrid 54 identified a YAC clone which was also positive for the DMD probe P20 (data not shown). This method also identified fragments in the hybrids that were not seen in the initial DNA characterization (Fig 1 and Table 1). Since the 27 DNA probes were not close enough to each other along the chromosome to detect all possible hybrid fragments (1000-5000 kb), many regions would have been untested. For example, AluPCR products from hybrids 45 and 48 identified several cosmids, also seen by the cDNA for chronic granulomatous disease gene (CYBB, 13) in Xp2 1.1, although this region was not tested in the original hybrid characterization. <br><br> DISCUSSION <br><br> The direct hybridization of Alu-PCR products from somatic cell hybrids to genomic libraries can bypass gel purification or ligation of fragments into plasmid vectors and individual analysis for single copy sequences. Hybridization of Alu-PCR products as <br><br> a pool to ordered array libraries such as the flow-sorted X chromosome cosmid library (9), allows the direct comparison of overlapping hybrids to pinpoint cosmids most likely to be from the region of interest. In conjunction with the reference library database (G.Z, unpublished) with 183 X chromosome probe hybridization entries, cosmids identified with both Alu-PCR products and uniquely mapped X probes immediately map them to the region of interest and proove that the method has worked. Similar hybridization experiments using Alu-PCR products from four overlapping irradiation hybrids identified four cosmids in common that mapped to the region of overlap by independent experiments (F.Muscatelli, A.P.M., P.N.G., H.L. and M.Fontes, in preparation). Since only 27 probes from the X chromosome were used to initially characterize the hybrids and the length of individual <span class="blue">human</span> fragments in the irradiation hybrids is about 1000-5000 kb, many regions could have been undetected in the original analysis. The direct hybridization of Alu-PCR products to the cosmid reference library detected such fragments since they identified cosmids in common with uniquely mapped probes in regions not tested originally (Table 1 and Fig 1). This should prove to be a sensitive and efficient method to determine content and overlap of irradiation hybrids in conjunction with DNA blot hybridization. However, since the exact length of the <span class="blue">human</span> DNA fragments for each hybrid and the spacing of Alu-PCR products along the chromosome is not known, it is difficult to directly correlate the number of target cosmids to the DNA content of the hybrids. <br><br> The direct hybridization of Alu-PCR products from irradiation or somatic cell hybrids to total genomic YAC libraries will be especially useful to construct long range YAC contigs from specific subregions of chromosomes. The dissection of a total genomic YAC library by this method may be more efficient than generating chromosome specific YAC libraries from somatic cell hybrids (usually a haploid <span class="blue">human</span> chromosome on a diploid or greater <span class="yellow">rodent</span> background) or flow-sorted chromosomes, because of the low transformation efficiency of <span class="yellow">yeast</span>. <br><br> ACKNOWLEDGEMENTS <br><br> We would like to thank Gert-Jan Van Ommen for the probe P20 and Stuart Orkin for the CYBB cDNA probe. A.P.M is supported in part by a research fellowship from the Muscular Dystrophy Association of America. V.M.S.L. is supported in part by the Medical Research Grant of the University of Hong Kong. Reference library filters and cosmids identified by Alu-PCR products can be requested from the Reference Library Database, ICRF, 44 Lincoln's Inn Fields, London WC2A 3PX, U.K. <br><br> 3318 Nucleic Acids Research, Vol. 19, No. 12 <br><br> REFERENCES <br><br> 1. Nelson, D.L., Ledbetter, S.A., Corbo, L., Victoria, M.F., Ramirez-Solis, <br><br> R., Webster, T.D., Ledbetter, D.H., and Caskey, C.T. (1989) Proc. Natl. Acad. Sci. USA, 86:6686-6690. <br><br> 2. Ledbetter, S.A., Nelson, D.L., Warren, S.T., and Ledbetter, D.H. (1990) <br><br> Genomics, 6:475-481. <br><br> 3. Cotter, F.E., Hampton, G.M., Nasipuri, S., Bodmer, W.F., and Young <br><br> B.D. (1990) Genomics, 7:257-263. <br><br> 4. Burke, D.T., Carle, G.F., and Olson, M.V. (1987) Science, 236:806-812. 5. Benham, F., Hart, K., Crolla, J., Bobrow, M., Francavilla, M., and <br><br> Goodfellow, P.N. (1989) Genomics, 4:509-517. <br><br> 6. Feinberg, A.P., and Vogelstein, B. (1984) Anal. Biochem., 137:266-267. 7. Hochgeschwender, U., Sutciffe, J.G., and Brennan, M.B. (1989) Proc. Natl. <br><br> Acad. Sci. USA, 86:8482-8486. <br><br> 8. Lehrach, H., Drmanac, R., Hoheisel, J., Larin, Z., Lennon, G., Monaco, <br><br> A.P., Nizetic, D., Zehetner, G., and Poustka, A. (1990) In Davies, K.E. & Tilghman, S.M. (eds.), Genome Analysis Volume 1: Genetic and Physical Mapping. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, pp. 39-81. <br><br> 9. Nizetic, D., Zehetner, G., Monaco, A.P., Gellen, L., Young, B.D., and <br><br> Lehrach, H. (1991) Proc. Natl. Acad. Sci. USA, 88:3233-3237. <br><br> 10. Larin, Z. and Lehrach, H. (1990) Genet. Res. Camb., 56:203-208. <br><br> 11. Cox, D,R., Pritchard, C.A., Uglum, E., Casher, D., Koborl. J. and Myers, <br><br> R.M. (1989) Genomics, 4:397-407. <br><br> 12. Wapenaar, M.C., Kievits, T., Hart, K.A., Abbs S., Blonden, L.A.J., <br><br> denDunnen, J.T., Grootscholten, P.M., Bakker, E., Verellen-Dumoulin, C., Bobrow, M., vanOmmen, G.J.B., and Pearson, P.L. (1988) Genomics, 2:101-108. <br><br> 13. Royer-Pokora, B., Kunkel, L.M., Monaco, A.P., Goff, S.C., Newburger, <br><br> P.E., Baehner, R.L., Cole F.S., Curnette J.T., and Orkin, S.A. (1986) Nature, 322: 32-38. <br><br> 14. Davies, K.E., Mandel, J.L., Monaco, A.P., Nussbaum, R.L. and Willard, <br><br> H.F. (1990) Cytogenet. Cell Genet. 55: 254-313. <h3>pmcA509286</h3>How can Health Behavior Theory be made more useful for intervention research?
Abstract
Background
The present paper expresses the author's views about the practical utility of Health Behavior Theory for health behavior intervention research. The views are skeptical and perhaps even a bit exaggerated. They are, however, also based on 20-plus years of in-the-trenches research focused on improving health behavior practice through research.<br><br>Discussion
The author's research has been theoretically driven and has involved measurement of varying variables considered to be important theoretical mediators and moderators of health behavior. Regretfully, much of this work has found these variables wanting in basic scientific merit. Health Behavior Theory as we have known it over the last 25 years or so has been dominated by conceptualizations of behavior change processes that highlight cognitive decision-making. Although much of health behavior practice targets what <span class="yellow">people</span> do rather than what they think, the logic of focusing on thoughts is that what <span class="yellow">people</span> think about is the key to what they will do in the future, and that interventions that can measure and harness those processes will succeed to a greater extent than those that do not. Unfortunately, in the author's experience, the premise of cognitive theories has fallen short empirically in a number of ways. The cognitive schemata favored by most health behavior theories are difficult to measure, they do not predict behavioral outcomes very well, there is little evidence that they cause behavior, and they are hard to change directly.<br><br>Summary
It is suggested that health behavior researchers reconsider their use of these theories in favor of models whose variables are more accessible to observation and experimental manipulation and that most importantly have strong empirical support.<br><br><br><br>Background
The author has been conducting research on behavioral treatment of obesity for about 25 years. During that time, the dominant conceptual models guiding intervention development have been cognitive behavior models that have their origin in psychological theory. Those most often cited include the Health Belief Model [1], Protection Motivation Theory [2], Subjective Expected Utility Theory [3], the Theory of Reasoned Action [4], Social Cognitive Theory [5], and the Transtheoretical Model [6]. All of these theories are concerned with how <span class="yellow">people</span> make behavioral choices and the general idea is that <span class="yellow">people</span> decide what to do based on the extent to which they expect that their choices will produce results that they value. Much of the content of the theories is concerned with factors that may affect value/expectancy calculations. As summarized by Weinstein in a comparative review of four social psychological theories [7], variables thought to influence value/expectancy judgments include such factors as perceived rewards of current behavior, self-efficacy, normative beliefs, motivation, and the perceived consequences of not changing behavior.
Weinstein's summary is illustrative of the fact that Health Behavior Theory has tended to be particularly interested in understanding <span class="yellow">people</span>'s motivation to change behavior rather than ability to change. Moreover, motivation is thought to be the result of a relatively complex, but logical, interpretation of large quantities of information about self and environment. The theories that Weinstein reviewed deal almost exclusively with behavioral decision processes in <span class="yellow">people</span>'s minds. They have few if any terms relating to how information gets into <span class="yellow">peoples</span> minds or how subsets of it receive more or less attention. Broader health behavior theories such as Social Cognitive Theory or the Transtheoretical model have addressed issues and variables outside the <span class="yellow">person</span> to a greater extent, but the fundamental interest in and belief in psychological variables as the key force in determining health behavior remains.
The implications of the focus of health behavior theory on psychological determinants of behavioral decision-making for my own research area of interest, obesity treatment, are several. One is the inclusion of measures of psychological characteristics in most research protocols (e.g., assessment of behavioral intentions, self-efficacy, perception of barriers to change, perception of social support, and outcome expectations). A second is the inclusion of treatment elements that specifically target psychological perceptions and processes independent of the diet and physical activity behaviors that actually produce weight change (e.g., how to deal with emotional eating, how to deal with the frustration of lapses and relapses, and how to talk to yourself to increase self-motivation). A third is the belief that psychological reactions to treatment experiences themselves are very important and deserve independent attention. Common behavioral prescriptions for weight-loss goals and frequency of self-weighing are exemplary (i.e., recommending infrequent weighing to prevent discouraging feedback about progress and encouraging smaller and thus "more attainable" behavior and weight-loss goals in the belief that they will be more motivating).
The problem with the emphasis on cognitive variables in weight-control research is that they have so far failed to meet fundamental scientific criteria for empirical verification. Thus, they also have not led to a better understanding of the weight-loss process, have not improved our ability to predict weight-loss outcomes, and have not led to improvement in treatment methods. In some cases it is even arguable that they have made treatment worse. I will illustrate these problems with results from my own research.<br><br>Discussion
Like most behavioral researchers in the obesity area, I have attempted to measure elements of health behavior theory in every obesity intervention project I have ever conducted. I have assessed weight-loss goals, behavioral and weight-loss self-efficacy, psychological well-being, perceived barriers to diet and physical activity change, stages-of-change, and perceived social support. How well have empirical examinations of these factors fared as predictors of success in weight control?
Self-efficacy
We have examined the predictive value of self-efficacy assessments in several of our studies and describe the results from three of these here in more detail [8-10]. In the first study, self-efficacy was assessed at baseline, posttreatment, and one year later in 85 <span class="yellow">men</span> participating in a 15-week weight-loss program [8]. The self-efficacy instrument had subscales for emotional states (e.g., anxiety) and situations (e.g., eating away from home). Higher baseline self-efficacy on both subscales was associated with greater weight loss in treatment and at 1- and 2-year follow-up. Emotional self-efficacy at posttreatment did not predict weight loss at 1- or 2-year follow-up. Situational self-efficacy at posttreatment predicted weight loss at 1-year but not 2-year follow-up.
The second study examined mood and situational self-efficacy in 55 <span class="yellow">men</span> and 58 <span class="yellow">women</span> before and after a 16-week weight-loss treatment with a 1-year follow-up [9]. <span class="yellow">Women</span> had lower pretreatment self-efficacy than <span class="yellow">men</span>. Self-efficacy was predictive of weight loss and maintenance in <span class="yellow">men</span> but not in <span class="yellow">women</span>. Change in self-efficacy over time was positively related to weight change in <span class="yellow">women</span> but not in <span class="yellow">men</span>.
The third study examined predictors of weight change over a 2-year period in 460 <span class="yellow">men</span> and 1172 <span class="yellow">women</span> who received a low-intensity weight-loss intervention delivered through their HMO [10]. The self-efficacy measure was the WEL questionnaire. <span class="yellow">Men</span> again were found to have higher baseline self-efficacy than <span class="yellow">women</span>. Self-efficacy did not predict weight change in <span class="yellow">men</span> but was positively, though weakly, related to weight change at 6 months only in <span class="yellow">women</span>.
Our overall conclusion from the analyses described above, as well as others not pursued in as great detail, is that self-efficacy is a weak predictor of weight loss and is inconsistent across study populations and gender. It tends to increase with weight loss. However, treatment-induced increases in efficacy are not predictive of longer-term weight-loss success.<br><br>Barriers to Adherence
We have also attempted to measure barriers to adherence to weight-control behaviors in many of our studies [11-14]. The instruments used for this have typically been formatted similarly to efficacy questionnaires in that <span class="yellow">people</span> are asked to indicate how difficult they find situational, knowledge, and motivational challenges to achieving diet and exercise changes. The findings in these studies have been quite consistent. Baseline assessments of perceived barriers to behavior change are not predictive of weight change. Weight loss is associated with reported decreases in perceived barriers. Treatment-induced change in perceived barriers are not predictive of future weight change. In other words, barrier perceptions as we have measured them do not appear to have pragmatic significance.<br><br>Weight Goals
Goal-setting has long been of interest to health behavior theory and in recent years has attracted attention in weight-loss research when it was realized that most <span class="yellow">people</span> who enter weight-loss treatments want to lose a lot more weight than is realistic given the potency of current weight-loss methodologies [15]. When asked to describe weight losses they deem to represent "dream, happy, acceptable, and disappointing," many individuals in treatment fail to reach even "disappointing" weight losses even though in objective medical terms the results are positive. Based on the argument that failure to reach gratifying weight-loss goals leads to psychological distress that lowers weight self-efficacy and undermines weight-loss efforts, it has become popular to recommend counseling in weight-loss treatments specifically targeting the lowering of weight-loss goals. The theoretical argument is that excessive outcome expectations undermine behavioral efforts. We have now completed three sets of formal analyses examining whether weight goals are predictive of weight-loss success. In one of these analyses the relationship between weight-loss goals, weight-loss goal attainment, and long-term (30 months) weight-loss attainment and psychological well-being were assessed in 69 <span class="yellow">men</span> and 61 <span class="yellow">women</span> participating in an intensive behavioral treatment program [16]. Results indicated that weight-loss goals were unrealistically high on average and that lower goals were more likely to be reached. Nevertheless, weight-loss goals did not predict either short- or long-term weight losses and were not associated with elevated psychological distress. Two more recent analyses we have conducted looking at weight-loss goals as predictors of success have produced similar results [Linde JA, Jeffery RW, Levy RL, Pronk NP and Boyle RG, unpublished data [17]]. Weight-loss goals either did not predict weight loss at all or were slightly positively related to weight-loss success.<br><br>Perceived Social Support
Perceived social support is another psychological factor thought to influence health behavior decision-making. We have measured social support in a variety of ways in our studies, ranging from single-item questions to multipaged assessments attempting to differentiate among informational, instrumental, and emotional support. The results, unfortunately, have closely paralleled those we have seen with other assessments of barriers to adherence. Assessments of social support prior to treatment do not predict weight loss. Average reports of social support tend to parallel weight loss itself. When <span class="yellow">people</span> lose weight they report more social support. When they regain, they report less. In other words, perceptions of social support are not predictive of success in weight-loss treatments.<br><br>Frequency Weight Self-monitoring
Self-monitoring of health behavior is incorporated into many health behavior theories, usually as part of a <span class="yellow">person</span>'s assessment of achieved outcomes. Although self-monitoring is usually considered a positive element in the adoption of health behavior, in obesity treatment frequent self-monitoring of weight has tended to be down-played or even discouraged on the grounds that disappointing results (i.e., less than desired weight change) may undermine motivation. This is another example in which health behavior theory may have indirectly led to incorrect treatment recommendations. In weight-loss treatments, active discouragement of frequent self-observation of weight has become popular based on the premise that more frequent weighting will cause psychological stress and lower self-efficacy. Recently, we have examined the relationship between frequency of self-weighing and body weight in both clinical and population samples and have found, somewhat to our surprise, that frequency of self-weighing is one of the strongest single predictors of body weight cross-sectionally, and change in the frequency of self-weighing is one of the strongest predictors of weight change [Linde JA, Jeffery RW and French SA, unpublished data]. The direction of predictions, however, is opposite that derived from theory. <span class="yellow">People</span> who weigh themselves more weigh less and are more successful in losing weight.<br><br>Stage-of-Change
A final failure of current health behavior theory to prove useful in weight-control research is a recent examination of the relationship between a stage-of-change measure adopted from Prochaska and short- and long-term weight loss [18]. Categories of precontemplation, contemplation, preparation, and action were defined based on questions about weight-loss intentions and recent weight-loss attempts. Despite a large sample size, excellent follow-up rates, and well-measured objective outcomes, we were unable to demonstrate that staging algorithms recommended by proponents of the Transtheoretical Model could predict weight-loss outcomes.<br><br>Experimental Modification of Expectations
Our most recent effort to utilize health behavior theory in obesity intervention research is a study that attempted to examine the effectiveness of experimentally-induced outcome expectancies on weight loss [Finch EA, Linde JA, Jeffery RW, Rothman AJ and King CM, unpublished data]. Obese <span class="yellow">men</span> and <span class="yellow">women</span> participated in an 8-week weight-loss program with 18-month follow-up in which they were assigned to one of two expectancy groups. The optimistic group was told that focusing exclusively on the positive benefits of weight loss would be valuable in ensuring that they remained motivated in their weight-loss efforts and was given assignments during weekly group sessions and homework between sessions to reinforce this optimistic mindset. A "balanced" expectancy group received the instructions that focusing on both the positive and negative aspects of weight loss, a balanced approach, would be most conducive to maintaining weight-loss motivation. This group also received assignments to reinforce their message. Results of this study indicated that the expectation induction was successful initially but difficult to maintain in the face of real weight-loss experience. We were also unable to show that experimentally-induced expectations influenced weight-loss success.<br><br>
Summary and Conclusion
To summarize the findings described above, I have had considerable difficulty over the last 25 years in confirming that the psychosocial variables favored by health behavior theory are of much value for obesity intervention research. They do not predict weight loss well, either as mediators or moderators. There is little evidence to support the idea that targeting them for intervention improves weight-loss outcomes. It is, of course, arguable that the weak findings relating to health behavior theory variables are due in large part to methodological weaknesses, either in measurement tools and/or their frequency of measurement. I would argue, however, that 25 years is long enough to wait for improved methods and that it is time to look elsewhere for variables that better predict weight-change outcomes and that, therefore, may form a better basis for improving future treatments.
Implication for Weight-Loss Treatment
Given the lack of success finding support for cognitive mediators of behavior change in weight loss, one might surmise that progress in improving weight-loss interventions over the last 20 years must have been dreary indeed. Somewhat surprisingly, however, that is not the case. In fact, the short-term (6 to 12 months) success of weight-loss treatments has approximately doubled over that time and several variables have been identified that reliably enhance treatment outcomes. It has been clearly shown experimentally that increasing treatment length [19], prescribing low-energy intakes [20], prescribing high-energy expenditure [21], using a deposit contract and group-based reward systems [22], and simplifying adherence to diet through meal substitutes [23] and exercise by providing exercise equipment [24] all improve initial weight loss. From a theoretical perspective, however, one thing is noteworthy about these successful innovations. Although not incompatible with health behavior theory, none of them are specifically derived from cognitive decision-making models. Indeed, health behavior theory does not include variables like these in its models.<br><br>Where Do We Go From Here?
The argument above about the practical limitations of many popular theories of health behavior is not meant to be a call to abandon theory. Behavior scientists have amassed much useful information about the principles underlying <span class="blue">human</span> behavior that should be valuable for health behavior interventions. Much is known about <span class="blue">human</span> perception, learning, motivation, and responsiveness to environmental opportunities and contingencies. Health behavior intervention lies at the interface between <span class="yellow">people</span> and their environment. Interventionists change aspects of the environment (cues, information, behavioral contingencies) with the intention of producing changes in how <span class="yellow">people</span> behave. What is needed to advance health behavior intervention is theory that addresses relationships between modifiable aspects of the environment and behavior. There is no doubt that cognitive processes are involved in these relationships. However, the extent to which current theories capture this is questionable. Data now available suggest that easily obtainable information about <span class="yellow">people</span>'s cognitive processes adds little to our ability to predict the results of interventions. Thus, it may be wise to pay more attention to applied theories like classical behavior theory [25], communications theory [26], and learning theory [27] than to those coming out of the social cognitive traditions.<br><br>
Competing interests
None declared.<br><br>
<h3>pmcA1538864</h3>QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences
Abstract
The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of growing evidence for their role in important biological processes and as therapeutic targets. G-quadruplex DNA has been suggested to regulate DNA replication and may control cellular proliferation. Sequences capable of forming G-quadruplexes in the RNA have been shown to play significant roles in regulation of polyadenylation and splicing events in mammalian transcripts. Whether quadruplex structure directly plays a role in regulating RNA processing requires investigation. Computational approaches to study G-quadruplexes allow detailed analysis of mammalian genomes. There are no known easily accessible user-friendly tools that can compute G-quadruplexes in the nucleotide sequences. We have developed a web-based server, QGRS Mapper, that predicts quadruplex forming G-rich sequences (QGRS) in nucleotide sequences. It is a user-friendly application that provides many options for defining and studying G-quadruplexes. It performs analysis of the user provided genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences. It is also useful for predicting G-quadruplex structures in oligonucleotides. The program provides options to search and retrieve desired gene/nucleotide sequence entries from NCBI databases for mapping G-quadruplexes in the context of RNA processing sites. This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing. In addition to providing data on composition and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper features interactive graphic representation of the data. The user can also use the graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen. QGRS Mapper can be accessed at .<br><br>INTRODUCTION
The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of increasing evidence for their role in important biological processes and as therapeutic targets (1–5). The G-quadruplex structure, also known as a G-quartet, is formed by repeated folding of either the single polynucleotide molecule or by association of two or four molecules. The structure consists of stacked G-tetrads, which are square co-planar arrays of four guanine bases each (6). G-quadruplex is stabilized with cyclic Hoogsteen hydrogen bonding between the four guanines within each tetrad. The present work focuses only on the unimolecular quadruplexes, since it is more likely to be encountered in physiological conditions (7,8).
Guanine-rich sequences capable of forming G-quadruplexes are found in telomeres, promoter regions, transcribed and other biologically important regions of the mammalian genomes. G-quadruplex DNA has been suggested to regulate DNA replication in retinoblastoma susceptibility gene (Rb) region (9). This structure may control cellular proliferation at telomeric level and by transcriptional regulation of oncogenes like c-myc (2,10,11) and c-kit (12). Formation of G-quadruplex seems to be regulated through interactions with cellular proteins. While some proteins help stabilize the structure (13), others are known to resolve it (1,4,14,15). Proteins and chemicals that stabilize the G-quadruplex structure can inhibit telomerase action and, therefore, are being evaluated as anticancer therapeutic agents (16–20). Chemical compounds that inhibit G-quadruplex helicase activity may also be capable of regulating cellular proliferation (4). G-quadruplexes are also being eyed as potential antimicrobial agents due to their ability to transport monovalent anions (21).
G-quadruplex motifs in the RNA have been shown to play significant roles in mRNA turnover (1), FMRP binding (22), translation initiation (23) as well as repression (24). We have shown previously that a G-rich sequence (GRS) can mediate 3′ end processing of mammalian pre-mRNAs by interacting with DSEF-1/hnRNPH/H′ protein (25–27). Members of the hnRNP H protein family recognize G-rich motifs capable of forming G-quadruplexes and are known to regulate polyadenylation and splicing events in mammalian transcripts (28–30). Regulated RNA processing is an essential component of differential gene expression which is central to many important biologically processes. More than half of <span class="blue">human</span> genes are known to have alternative polyadenylation (31). Over two-thirds of <span class="blue">human</span> genes are thought to undergo alternative splicing (32). Sequences capable of forming G-quadruplexes found in the vicinity of polyadenylation and splice sites act as regulators by interacting with hnRNP F and H proteins (25–27,30,33). Whether quadruplex structure directly plays a role in regulating RNA processing events requires investigation.
Computational approaches to study G-quadruplexes in the mammalian genomes allow large-scale and detailed analysis of mammalian genes. Although, G-quadruplexes have been surveyed in the <span class="blue">human</span> genome with such techniques (34,35), there are no known user-friendly computational tools easily accessible to the public. We had previously built a database of mapped G-quadruplex sequences in selected alternatively processed <span class="blue">human</span> and <span class="yellow">mouse</span> genes (36). Our preliminary analysis of the database suggests prevalence of such motifs near alternative splice and poly(A) sites. We have now developed a web-based server, QGRS Mapper, that generates detailed information on composition and distribution of putative quadruplex forming G-rich sequences (QGRS) in any NCBI nucleotide sequence identified or provided by the user. The program is also designed to handle the analysis of mammalian pre-mRNA sequences, including those that are alternatively processed (alternatively spliced or alternatively polyadenylated). Researchers interested in predicting the ability of a nucleotide sequence to form G-quadruplex structure will find QGRS Mapper to be a user-friendly application that provides many options for analysis. The user can define the minimum number of tetrads, maximum length of the G-quadruplex motif, and size as well as composition of the loops. The program can map unimolecular QGRS in the entire nucleotide sequence provided in the raw or FASTA format by the user. This method can be used for the analysis of genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences. It is also useful for predicting G-quadruplex structures in oligonucleotides. Alternatively, the program provides options to search the entire NCBI Gene Entrez, RefSeq and GenBank databases in order to retrieve the desired gene/nucleotide sequence entries for analysis of their transcribed regions. Furthermore, QGRS Mapper is a unique tool for mapping G-quadruplex forming sequences in the context of RNA processing sites. This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing. In addition to providing data on composition and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper offers interactive graphic representation of the data. The user can also use a graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen.<br><br>METHODS
QGRS definition
The main goal of the QGRS Mapper program is to predict the presence of QGRS in nucleotide entries. These putative G-quadruplexes are identified using the following motif.
GxNy1GxNy2GxNy3Gx
Here x = number of guanine tetrads in the G-quadruplex and y1,y2,y3 = length of gaps (i.e. the length of the loops connecting the guanine tetrads). The motif consists of four equal length sets of guanines (which we call G-groups), separated by arbitrary nucleotide sequences, with the following restrictions.
The sequence must contain at least two tetrads (i.e. x ≥ 2). Although structures with three or more G-tetrads are considered to be more stable, many nucleotide sequences are known to form quadruplexes with two G-tetrads (37,38). QGRS Mapper is meant to be a flexible and comprehensive tool for investigating G-quadruplexes; hence it considers sequences with two tetrads.By default, only QGRS of maximum length of 30 bases are considered. However, the program gives the user the option to search for sequences up to 45 bases. This restriction on the length of the sequences being considered is in agreement with recent literature (34,35). The maximum length of 30 bases restricts G-groups to a maximum size of 6.The gaps or loops between the G-groups may be arbitrary in composition or length (within the overall restrictions on the length of QGRS). The program gives the user the option to search for QGRS having loops with a specified length range (e.g. the user can search for QGRS with loops of lengths between 1 and 4). The user can also specify a string that one or more loops of each QGRS must contain. This string can be given as a regular expression. For example, entering the regular expression ‘T{3,5}’ will search for QGRS having one or more loops that contain three to five consecutive T's.Also, at most one of the gaps is allowed to be of zero length
Table 1 shows some examples of valid QGRS. The guanine groups which form the tetrads are underlined.
The first sequence has four tetrads and equal length gaps. This would seem to provide a G-quadruplex that is the most stable of the three sequences. The second sequence is notable for the significant differences in the size of its loops. The third sequence has two tetrads, even though three of the G-groups could have included another G (since all G-groups must be equal in size).<br><br>G-scores
We have devised a scoring system that evaluates a QGRS for its likelihood to form a stable G-quadruplex. Higher scoring sequences will make better candidates for G-quadruplexes. The scoring method uses the following principles which are based on previous studies (34,35,39–42).
Shorter loops are more common than longer loops.G-quadruplexes tend to have loops roughly equal in size.The greater the number of guanine tetrads, the more stable the quadruplex.
The computed G-scores are dependent on the user selected maximum QGRS length. The highest possible G-score, using the default maximum QGRS length of 30, is 105. Here is a sequence attaining that score:
GGGGGGTGGGGGGTGGGGGGTGGGGGG.<br><br>Eliminating QGRS overlaps
Two QGRS are said to overlap if their positions in the nucleotide sequence overlap. QGRS Mapper will start with a nucleotide sequence, find all QGRS occurring in the sequence and then produce a non-overlapping set of QGRS. Overlaps are eliminated by selecting the higher scoring QGRS. In the non-overlapping view, only this sequence will be displayed, although the user can request that all overlapping sequences be displayed.
Please see supplementary materials at NAR online for more details on the elimination of overlapping QGRS.<br><br>
FEATURES
Design and implementation
QGRS Mapper is a web-based program, written in PHP, with Java being used for some of its graphics. The program takes a nucleotide sequence from NCBI (or as provided by the user) and analyzes it for the presence of putative G-quadruplexes. The structure of QGRS Mapper is summarized in Table 2.<br><br>Search and analysis
QGRS Mapper allows the user to search for putative G-quadruplexes in a variety of ways. It is possible to enter a nucleotide sequence in raw or FASTA format for analysis. One can search and analyze gene sequences by Gene ID, Gene name or symbol, accession number or GI number for an NCBI nucleotide sequence entry. The user can opt to change the maximum length of QGRS that will be searched for (the default maximum length being 30) and change the minimum sized G-group (which is two by default). Also, the user can specify that the loops in the QGRS fall within a given size range and that one or more loops of the QGRS contain a given string (for which the user may enter a regular expression). The web page for QGRS analysis can be seen in Figure 1.
After entering a sequence in raw or FASTA format, QGRS Mapper will search the sequence for occurrences of QGRS. The user may enter any combination of the letters A, C, T, G, U, N.
The Gene ID field allows the user to search the NCBI Entrez Gene database. QGRS Mapper will connect to NCBI, download and parse the gene entry, and then analyze the transcribed region of its nucleotide sequence for the presence of QGRS. For example, entering the gene ID 403437 results in downloading the Brca1 gene sequence for <span class="yellow">Canis familiaris</span>. Using the default QGRS search parameters, QGRS Mapper finds 156 non-overlapping QGRS and 3394 overlapping QGRS in the transcribed region of this gene.
The Gene Name or Gene Symbol field also allows the user to search the NCBI databases for all such genes. Entering the gene name Bcl2 results in nine different hits which are displayed in Table 3.
All nine of these entries can be analyzed for the occurrence of QGRS. Clicking on the Gene ID takes the user to the respective Entrez Gene entry. Clicking on the last column initiates analysis of the selection by QGRS Mapper.
Similarly, the user can also enter an NCBI accession number to search for gene sequences. For example, searching the accession number AF312033 results in 12 hits being displayed for this GenBank nucleotide sequence entry.
The search phase of the program is followed by an analysis of the QGRS contained in the query sequence. In this phase of QGRS Mapper, the sequence data downloaded previously is analyzed to identify and map all QGRS relative to locations such as splice sites in exons/introns, and poly(A) site (if these locations are known). Furthermore the QGRS are scored by the method described above. The computed G-score is used to eliminate overlapping QGRS.
At times, QGRS Mapper must analyze a considerable amount of data. For example, the <span class="yellow">mouse</span> version of the gene PTPRU, which is 69822 bases long, contains 94681 QGRS of length up to 45 bases. QGRS Mapper will find, analyze and map all of these sequences. During this analysis a message is displayed indicating the estimated time left to completion.<br><br>QGRS Mapper output
After the analysis of overlaps is completed, QGRS Mapper displays a summary of its findings, in the Gene View. This summary includes basic gene information such as the gene ID, gene symbol, gene name, a link to the NCBI entry, organism name, chromosome number and number of products and poly(A) signals. Information is also given for each product, such as the number of exons and introns, number of QGRS (non-overlapping and overlapping), number of QGRS found near RNA processing sites, and a visual map of the product.
As an example, the Gene View for the <span class="blue">human</span> GREB1 is displayed in Figure 2, showing the table of gene information and product information for the first product (the output for all products may be seen in the supplementary material).
At this stage in the analysis the user can choose among three further displays: ‘Data View’, ‘Data View (with overlaps)’ and ‘Graphics View’. This can be done for the entire gene or for any particular product.
In the Data View, a table is displayed showing information for each of the set of non-overlapping QGRS. This table displays the position of the QGRS, which exon/intron it appears in, its distance from 3′ and 5′ splice sites, the QGRS sequence (with each G-group underlined) and the corresponding G-score. Similar display is also shown for each QGRS mapped to poly(A) region in the product. If the user requests the Data View for the entire gene, then the QGRS information is shown for each product. The ‘Data View (with overlaps)’ gives the same information but shows the locations of all QGRS. Figure 3 shows the Data View for product 1 of the GREB1 gene.
The user can also choose the Graphics View to give a visual display of the location of QGRS. This allows the user to see the location of QGRS relative to exons and introns (if that information is available). The Graphics View has the following components.
A graphic display of the entire gene (showing the location of the exons). This display includes a sliding window that can be used to focus on any particular segment of the gene. This window may be dragged to the left or right to change position within the gene.A magnified view of the fragment of the gene within the sliding window.A graph showing the location of QGRS within the fragment, with each QGRS being displayed by a bar whose height represents its G-score.A vertical slider that allows the user to change the size of the window. This allows the user to zoom in or out on any part of the gene. The sliding window on the gene expands or contracts as one zooms in or out. It is possible to see the nucleotide sequence of the product at maximum zoom levels.
The Graphics View for the entire gene shows the G-score graph together with an exon/intron map for each product. This allows the user to visually compare the location of QGRS for each product relative to that of splice sites. The Graphics View for the first product of the GREB1 gene is represented in Figure 4. The Graphics View for the entire GREB1 gene may be seen in the Supplementary Data.<br><br>
CONCLUSIONS
QGRS Mapper is a user-friendly web-based server that provides computational tools for prediction of Quadruplex forming G-rich sequences in the nucleotide sequences identified or provided by the user. The program offers many options, including user-defined composition of the quadruplex. It can analyze DNA or RNA sequence provided by the user in the raw or FASTA format. The application also provides tools for searching and retrieving gene/nucleotide entries from a variety of NCBI databases. There are several options for data output format, including an interactive graphic module.
Researchers interested in evaluating the ability of nucleotide sequences to form unimolecular G-quadruplexes will find QGRS Mapper to be very useful. Owing to the flexible and comprehensive nature of the design, it is expected to serve a variety of scientists. The application will be especially attractive to individuals interested in exploring the role of G-quadruplexes in regulated RNA processing. We are using the server to perform a large-scale analysis of alternatively processed mammalian transcripts. We are particularly interested in studying the composition and distribution patterns of G-quadruplexes in the transcribed regions of mammalian genes.<br><br>SUPPLEMENTARY DATA
Supplementary Data are available at NAR online.<br><br>
<h3>pmcA1590010</h3>Drug information resources used by nurse practitioners and collaborating physicians at the point of care in Nova Scotia, Canada: a survey and review of the literature
Abstract
Background
Keeping current with drug therapy information is challenging for health care practitioners. Technologies are often implemented to facilitate access to current and credible drug information sources. In the Canadian province of Nova Scotia, legislation was passed in 2002 to allow nurse practitioners (NPs) to practice collaboratively with physician partners. The purpose of this study was to determine the current utilization patterns of information technologies by these groups of practitioners.<br><br>Methods
Nurse practitioners and their collaborating physician partners in Nova Scotia were sent a survey in February 2005 to determine the frequency of use, usefulness, accessibility, credibility, and current/timeliness of personal digital assistant (PDA), computer, and print drug information resources. Two surveys were developed (one for PDA users and one for computer users) and revised based on a literature search, stakeholder consultation, and pilot-testing results. A second distribution to nonresponders occurred two weeks following the first. Data were entered and analysed with SPSS.<br><br>Results
Twenty-seven (14 NPs and 13 physicians) of 36 (75%) recipients responded. 22% (6) returned personal digital assistant (PDA) surveys. Respondents reported print, health professionals, and online/electronic resources as the most to least preferred means to access drug information, respectively. 37% and 35% of respondents reported using "both print and electronic but print more than electronic" and "print only", respectively, to search monograph-related drug information queries whereas 4% reported using "PDA only". Analysis of respondent ratings for all resources in the categories print,  health professionals and other, and online/electronic resources, indicated  that the Compendium of Pharmaceuticals and Specialties and pharmacists  ranked highly for frequency of use, usefulness, accessibility, credibility,  and current/timeliness by both groups of practitioners. Respondents' preferences and resource ratings were consistent with self-reported methods for conducting drug information queries. Few differences existed between NP and physician rankings of resources.<br><br>Conclusion
The use of computers and PDAs remains limited, which is also consistent with preferred and frequent use of print resources. Education for these practitioners regarding available electronic drug information resources may facilitate future computer and PDA use. Further research is needed to determine methods to increase computer and PDA use and whether these technologies affect prescribing and <span class="yellow">patient</span> outcomes.<br><br><br><br>Background
Challenges with knowledge management for health care professionals
In 1986, Haynes et al. published a series of 6 articles entitled "how to keep up with the medical literature" in an effort to help clinicians with information management, but this challenge has not decreased in last two decades [1-6]. Alper et al. suggest that maintaining currency with relevant literature in primary care would "require 627.5 hours per month, or about 29 hours per weekday, or 3.6 full-time equivalents of physician effort" [7]. The volume of information associated with keeping up to date is frequently cited as a barrier [8]. It is estimated that annually there are approximately 10,000 new randomized trials in MEDLINE and over 450,000 clinical trials identified by the Cochrane Collaboration [9,10]. Keeping up to date has been described with several analogies including clinicians attempting to drink water from a fire hose and swimming in rivers of clinical research with unprecedented depth, velocity, and turbulence [11,12].
Difficulties with dissemination of research evidence and keeping up to date on pharmacotherapeutic interventions are reported despite the development of tools such as clinical practice guidelines and systematic reviews that are intended to reduce the need for practitioners to evaluate original research [13]. To complicate matters further, there are often issues of credibility, timeliness, and volume of clinical practice guidelines and reviews. Many guidelines are criticized for their methodological development. Shaneyfelt et al. reviewed 279 guidelines for methodological standards from peer reviewed medical literature [14]. These authors found that only 51%, 33.6%, and 46% adhered to standards on guideline development and format, evidence identification and summary, and formulation of recommendations, respectively [14]. A Canadian review on drug therapy guidelines found significant variation in quality depending on the developer [13]. Approximately 25% of guidelines were not recommended for use in practice by the appraisers' criteria [13]. As an example of the volume of clinical practice guidelines available, eleven recent guidelines on community acquired pneumonia exist [15]. To add to the complexities involved with keeping current with pharmacotherapeutic management strategies, as of 2000, there were over 22,000 drug products approved for sale in Canada for <span class="blue">human</span> use [16].
There is also considerable debate regarding what constitutes "evidence" in practice, which contributes to confusion for clinicians [17,18]. Sim et al. succinctly describe the gap between evidence and action as difficulties with obtaining, systematically reviewing, applying in context, and measuring the outcome following application of evidence [19].<br><br>Maintaining competence – nurse practitioners as a new group of prescribers
Competencies for nurse practitioners (NPs) on a local and international level include critically appraising and applying literature and research findings in practice [20-23]. The Canadian Nurses Association (CNA) has developed the Canadian Nurse Practitioner Core Competency Framework that describes the knowledge, skills, judgment, and attributes required for practice. Evidence based practice is integral to pharmacotherapeutic interventions and prescribing competencies [23]. The National Prescribing Centre, an organization of the National Health Service in the UK, describes several competencies around information needs relevant to prescribing and emphasis is placed on using relevant and up to date information in various formats (e.g. print, electronic, verbal). Several related competencies include understanding advantages and disadvantages of information sources and the currency of resources [21]. Researchers in the US developed NP informatics competencies for integration into advanced nursing practice curricula [24]. Competencies related to informatics knowledge include critical analysis of data and information for use in evidence based practice, evaluating and applying relevant information, synthesizing best evidence, and using optimal search strategies to locate clinically sound and useful studies from information resources [24]. Achieving and maintaining competence in these domains as well as a solid foundation in pharmacology is necessary to support NPs in their relatively new role as a prescriber [25-27].<br><br>Knowledge management and information seeking behaviours among nurse practitioners and physicians
Information seeking behaviours of physicians are better documented than NPs [11]. Information related to diagnosis is important to both groups but drug therapy queries may occur more frequently with NPs [28-33]. Research on nurses' behaviours related to information seeking is available from the hospital setting [33-35] but the generalizability of these behaviours to NPs with a prescribing role is unclear. Differences in nursing roles, responsibilities, and legislation, including prescriptive authority, exist depending on the country of practice.<br><br>Nurse practitioners and their collaborating physician partners in Nova Scotia
Nova Scotia is a Canadian province with a population of approximately 942,000 [36]. The province is divided into six health zones that include nine district health authorities, one of which includes the provincial capital and is considered to be urban [37,38]. Health care service delivery is challenging due to many factors including the rural nature of the province, which is estimated to be 60% of population [37,39].
Starting in 1998, the Nova Scotia Department of Health led an initiative to explore different methods of delivering, managing, and funding primary care services. The Strengthening Primary Care in Nova Scotia Communities Initiative (SPCI) was established with the selection of four primary care demonstration sites where a primary health care NP was hired to practice collaboratively with one or more family/general physicians and other members of an interdisciplinary team. Each demonstration site adopted alternative (non fee-for-service) physician payment mechanisms and used electronic <span class="yellow">patient</span> records (EPRs) to support service delivery [41]. Demonstration sites participated in project evaluation components that included, but were not limited to, NP roles, alternative fee structures, consumer satisfaction, and implementation and integration of EPRs [41,42].
Legislation to allow NPs to practice collaboratively with physicians in Nova Scotia was passed in 2002, part way through the SPCI project [39]. Prescriptive authority granted through legislation authorizes NPs to prescribe from a schedule of drugs [43,44]. At the time of conducting this research project, 16 primary health care NPs were in active practice [43].
The EPR component of the SPCI project evaluation provided information on the use of technologies in the community context. Results from the implementation process indicated that considerable attention is required for technology literacy, time for training, and selection of software for EPRs [41]. Although the majority of community-based, non-institutional clinical practice settings in Nova Scotia primarily operate with paper-based charting systems, there is a movement toward integrating electronic technologies, including the EPR, in practice among health care providers, administrators, and the provincial government. In addition to recording <span class="yellow">patient</span> visit information, a component of the EPR package serves to provide drug information resources.
Drug therapy information resources for NPs and nurse prescribers have frequently been described as essential in supporting practice [25,28,29]. The role of NPs is relatively new in Canada [39] and there is limited information available to indicate the type of resources (e.g. print, electronic, EPR based) these prescribers use for drug and therapeutic information queries at the point of care. It is unknown as to whether differences exist regarding types of resources used, drug information needs, and utilization patterns among NPs and collaborating physician partners. Some research has suggested that the degree of multidisciplinary team functioning relates to the adoption of technology or innovations in practice but more research is required to determine the extent of these relationships [45,46].
The use of EPR technology is increasing in Nova Scotia but little information is available regarding the readiness of practitioners for use of specific features such as drug information resources. Based on the EPR related results of the SPCI evaluation, use of these functions could be challenging without proper facilitation. The purpose of the survey for this research was to describe drug information resources used by NPs and their collaborating physician partners at the point of care. The results of the survey will be used to guide further technology implementation strategies and stimulate further discussion around drug information resource usage at the point of care.<br><br>
Methods
Survey development
Survey development involved three stages including identification of important content areas, development of draft questions, and survey refinement.
Identifying important content areas for inclusion in the survey involved conducting a comprehensive English language literature search, consultation with relevant stakeholders (e.g. members of the Nova Scotia Department of Health), and input from subject matter experts at Dalhousie University. The literature review was conducted using the following bibliographic databases: PubMed, Cumulative Index to Nursing and Allied Health Literature (CINAHL), International Pharmaceutical Abstracts (IPA), and Web of Science Citation Databases. Hand and electronic searching of relevant journals was also conducted. Broad search terms were used without limits on publication date or place as nurse practitioner titles, roles and scopes of practice, and terminology regarding technology vary nationally and internationally. Some examples of terms used included nurse practitioner, nurse prescriber, nurse clinicians, district nurse, health visitor, drug information resources, drug information services, information needs, and information technology.
The draft survey was reviewed by the research team to reduce the number of items and improve clarity. The layout of the questionnaire was carefully examined to ensure that it was easy to follow and complete. Research results from a previous investigation of Nova Scotian physicians' behaviours regarding drug information were also used to further revise the survey [47]. This draft questionnaire was pilot tested by two out of province NPs and one physician. The results of the pilot were used to make final revisions to the survey. Based on pilot-testing feedback and investigator consensus, the final survey was divided into 2 versions, one for personal digital assistant (PDA) users and one for computer users.
The 10 page surveys for PDA and computer users had 5 or 6 sections, respectively, and 37 questions, many with multiple parts. The survey content included demographics, computer or PDA use and experience, drug and therapeutic resource use and preferences, PDA future use, perceived barriers and facilitators to PDA use, and technology training preferences.
Section one contained demographic questions such as gender, age, job title, volume of <span class="yellow">patients</span>, and EPR availability in the practice setting. Section two was designed to determine PDA or computer use and experience in the practice setting with questions regarding length of use, costs, and work versus home usage. This section also addressed usage and rating of different drug information resources. Resource ratings were based on the frequency of usage, usefulness, accessibility, credibility, and current/timeliness. Resources were grouped as print (i.e. books, journals, and clinical practice guidelines), online/electronic resources, and health professionals and other. Respondents used 5-point Likert scales (strongly agree to strongly disagree) for rating opinions related to resources. A rating of 6 (not applicable, I do not use this resource) was also included for respondents who did not use a particular resource. Frequency of searching for specific information was rated on a 3-point Likert scale (frequently to never). The final sections of the survey included categorical, open-ended, and Likert scale questions regarding preferred resources, technology barriers, PDA future use, and technology training preferences. Copies of the surveys are attached as an appendix in PDF format [see additional file 1 and 2] or can also be accessed from the Initiative for Medication Management, Policy Analysis, Research & Training (IMPART) website [48].
Ethics approval for the survey was granted through Dalhousie University Research Ethics Board on February 3, 2005.<br><br>Survey population
Licensed, actively practicing, primary health care NPs (n = 16) and their collaborating physician partners (n = 21) were eligible to participate.<br><br>Survey procedures
The survey recruitment procedures were based on the methods of Dillman [49] and Salant and Dillman [50]. Survey packages contained a cover letter, separate surveys for PDA and computer users, and a return self-addressed stamped envelope. The covering letter instructed respondents to self-select the appropriate survey (either PDA or computer) based on their drug information seeking behaviours. <span class="yellow">Participants</span> who had used a PDA at any time were instructed to complete the PDA version of the survey. Those who had never used a PDA for drug information were instructed to complete the computer version of the survey. Several strategies were used to optimize response rate and included: personalized cover letters, coloured paper for surveys, stamped return envelopes, follow-up mailing, and a priority post mailing [51]. The covering letter included coloured logos of Dalhousie University and the Nova Scotia Department of Health representing the investigator affiliations and endorsement of the project.
A master mailing list with names and addresses of NPs and their collaborating physician partners was created. To maintain confidentiality of respondents, a number placed on the bottom right corner of each survey corresponded to a name on the confidential master mailing sheet. The postage paid return envelopes were addressed to the research coordinator at the School of Nursing, Dalhousie University, who matched respondents to the mailing list from the first distribution. The cross-referenced mailing list was not accessible to those entering or analysing data. The research coordinator sent the second distribution to those who had not initially responded. A fluorescent coloured page was included in the second mailing to notify recipients of the second and final mailing status. The second mailing followed 2 weeks after the initial mailing (February 2005). The surveys were sent via Xpresspost™ through Canada Post.<br><br>Data analyses
Quantitative
Data were entered and analysed in Statistical Package for Social Sciences (SPSS) (version 11.5 for Windows). Five surveys were randomly selected as a check for accuracy of data entry. Descriptive statistics were used to describe resource usage by practitioners. Chi Square (Fisher's Exact when cell count less than 5) analyses were used to determine differences in computer or PDA use based on predetermined variables (e.g. high speed Internet connection, number of <span class="yellow">patients</span> per day). Mann Whitney U tests were used to compare physician and NPs Likert scale ratings (1 = strongly agree to 5 = strongly disagree) of resource use. Physician and NP rankings of all resources (print, online/electronic, and health professionals and other) were determined from means of Likert scale ratings (1 = strongly agree, 5 = strongly disagree) for each of the pre-specified characteristics (e.g. frequency of use, accessibility, etc.) and the frequency of use of the resources. The best rankings were assigned for the lowest mean scores and the largest number of the sample using a resource. These rankings (ranks based on mean and ranks based on sample) were then entered into a formula to calculate an overall rank. The formula includes: rank = [(rank according to % of sample using the resource + rank based on mean score) ÷ 2]. This formula was used to account for mean scores based on small samples as these numbers could potentially over or underestimate the value of a resource. Ratings of 6 (i.e. not applicable, I do not use this resource) were excluded from the analyses.<br><br>Qualitative
Comments were entered in a word-processing program and organized by type of respondent (PDA versus computer) and question number. The coded survey number and respondent type (NP or physician) were also included next to comments. Investigators determined themes and categorized comments based on previous experience, knowledge, and familiarity with the topic.<br><br><br><br>Results
Surveys were completed and returned by 75% of eligible <span class="yellow">participants</span> (27 of 36). One physician survey was undeliverable. The response rates from within the NP and physician samples were 88% and 65%, respectively. Complete demographic information is available in Table 1.
Methods for accessing resources and self-reported resource use
Resource use was similar amongst practitioners. Respondents indicated that print resources (mean 4.56, SD 0.80), health professionals (mean 3.26, SD 0.90), and online/electronic resources (mean 2.70, SD 1.20) were the preferred method (1 = least preferred to 5 = most preferred) for accessing drug information. Thirty-seven percent of respondents reported that searching for specific questions related to drug information (e.g. usual dosage, duration of therapy) was conducted using both print and electronic resources (but print use greater than electronic) (Table 2). The preferred means (i.e. print) to access resources was consistent with the most common means of conducting searches for specific drug information queries.
Respondents' ratings for pre-specified print, online/electronic, and professional resources and other, based on means from Likert scales and number of respondents using the resources, are presented in Tables 3, 4, and 5. Of all resources within the print, online/electronic, and health professionals or other categories, NPs and physicians rated the Compendium of Pharmaceuticals and Specialties (CPS) [52] and pharmacists as the top two most frequently used resources for providing drug and therapeutic information. Physicians rated other physicians as the third most frequently used resource. The book Therapeutic Choices [53] ranked third for NPs. Based on written feedback, physicians and NPs consulted pharmacists and other physicians most frequently. The CPS and pharmacists were also ranked as the top two resources overall in terms of usefulness, accessibility, credibility, and current/timeliness for physicians. Rankings by NPs were similar for usefulness, accessibility, and credibility. NPs ranked pharmacists, Therapeutic Choices, and academic detailing first and the CPS as second for current/timeliness.
Within the online/electronic category, electronic clinical practice guidelines (eCPGs) were rated the highest for all characteristics (e.g. usefulness, credibility). Although eCPGs were highly ranked, approximately 30% of the sample reported not using this resource. Other resources in this category were infrequently used based on respondents' self-reports.
Pharmaceutical industry representatives were used as a source of drug information by 85% and 86% of physicians and NPs, respectively (Table 5). This was higher than regional drug information services (used by 23% of physicians and 50% of NPs). After exclusion of traditional health professionals (i.e. physicians, nurses, pharmacists, allied health) in the health professionals and other category, pharmaceutical industry representatives received rankings for second or third for frequency of use, usefulness, accessibility, credibility, and current/timeliness, based on means and number of respondents using this resource (data not shown).<br><br>Differences between nurse practitioners and physicians
A series of Mann Whitney U tests were used to compare the responses of NPs and physicians on their use of print, online/electronic, and health professional resources. In total 95 statistical tests were conducted. The large number of tests increases the likelihood of a type I error as five significant differences would be expected by chance alone at an alpha threshold of 0.05. It is therefore important to treat these results with caution. A limited number of statistically significant (p < 0.05) differences were identified between physicians and NPs and are reported in Table 6. Therapeutic Choices differed significantly for frequency of use with more NPs making use of this resource. Allied health professionals significantly differed between NPs and physicians for accessibility and current/timeliness while NPs were more in agreement with these characteristics of the resource. Nurse colleague credibility and current/timeliness was rated significantly higher by NPs versus physicians.<br><br>Factors influencing electronic technology use at the point of care
Factors such as gender, age, practitioner type (NP vs physician), accessibility, technical support, Internet connection speed, <span class="yellow">patient</span> volume, presence of an EPR, and home computer use were examined to determine if they were associated with the use of a work computer to search for drug information at the point of care. No statistically significant associations were found (Fisher's Exact).<br><br>Additional resources from respondent comments
Respondents indicated other resources and programs, such as clinical calculators, that they would like to access from their computer or PDA. The top three resources that were desired included Canadian clinical practice guidelines, <span class="yellow">patient</span> education information, and ability to track clinical activities/statistics. Further comments from two NP computer survey respondents revealed that a resource on drug interactions and dosages would be desired. One other NP also indicated "up to date info [sic] on drugs to treat various illnesses ie doseage [sic], length of use etc."<br><br>Computer or personal digital assistant use in practice
Approximately 50% of computer survey respondents reported using their work computers for searching drug or therapeutic information related to <span class="yellow">patient</span> care. Of those respondents, just over half (54%) also reported using their home computer for this purpose. Sixty-seven and 17% of PDA survey respondents reported using their PDA for searching drug or therapeutic information related to <span class="yellow">patient</span> care at work and home, respectively.<br><br>Searching on a weekly basis for specific information related to drugs
Of the 24 specified categories of drug information included in the survey, the majority were reported as infrequently searched and a smaller percentage as never searched by respondents (data not shown). The top three categories rated as frequently searched were side effects, adult or usual drug dosage, and most appropriate drug for an indication. (Table 7)<br><br>Issues related to personal digital assistants
Respondents reported their level of agreement with statements related to how PDAs may influence their practice. The statements included aspects of workload (organization and paper work), convenience, and improving quality of care and <span class="yellow">patient</span> outcomes. (Table 8) Respondents agreed that PDAs are a convenient resource but indicated that PDAs would not decrease paperwork or improve <span class="yellow">patient</span> health outcomes.<br><br>Barriers and facilitators to personal digital assistants: themes from written comments
Peer support from colleagues, convenience, standardized usage, and financial and technical support were the main perceived facilitators to PDA use reported by respondents. The main perceived barrier to PDA use reported by respondents (n = 10) included cost. Other factors such as technology literacy, time, lack of peer support, no high speed internet for downloads, lack of needed resources, keeping up to date on resources, and searching speed were also reported.<br><br>Future use of personal digital assistants
Fifty-two percent, including current PDA users, reported that they would use a PDA in the future. Twenty two percent were  uncertain and 19% reported that they would not use a PDA in the future. Two <span class="yellow">people</span> did not respond.<br><br>Confidentiality
Fifty two percent of respondents indicated that <span class="yellow">patient</span> confidentiality with PDAs was no more concerning compared to use of other technologies. Forty-four percent did not know if they had a policy on <span class="yellow">patient</span> confidentiality with regard to technologies.<br><br>Technology training and reimbursement
Respondents rated (1 = least preferred to 5 = most preferred) one on one instruction and group learning led by an expert facilitator as the most preferred (mean 4.32, SD 0.99) means by which to receive instruction on a new technology. Least preferred methods included online discussions/chatrooms (mean 1.52, SD 1.04), internet videos (live: mean 1.70, SD 1.10, or static: mean 1.87, SD 1.14), video cassettes (mean 2.30, SD 1.55), trial and error learning (mean 2.32, SD 1.28), and written manuals (mean 2.92, SD 1.44). Paid leave for attendance at technology training sessions was the preferred means (mean 1.77, SD 0.86; 1 = strongly agree to 5 = strongly disagree) of remuneration for respondents. Respondents also indicated that if financial remuneration was to occur, it should correspond to the amount of time for training that is required (versus a flat rate) (mean 1.96, SD 1.08). Continuing education credits were not viewed as an incentive (mean 2.69, SD 1.44).<br><br>
Discussion
Preferred resources
In our study, printed materials (e.g. compendia, journals, textbook resources) and professionals (e.g. pharmacists) were the most preferred and frequently used means to access information. Physician reliance on text and compendia relative to online/electronic resources has been frequently reported [11]. In a study examining family doctors' use of information sources to answer clinical questions, <span class="blue">human</span> resources (e.g. doctor, pharmacist), non-prescribing print information (e.g. textbooks and journal articles), and prescribing texts were used 36%, 32%, and 25% of the time, respectively [54]. Books from the workplace were reported by approximately 79% of UK primary care nurses as a commonly used source of knowledge and information used to support practice [55]. Fewer than one-third (31%) reported using electronic resources (e.g. Internet, electronic journals) for this purpose [55]. Results of a postal questionnaire to NPs demonstrated that 61% and 51% of respondents reported using drug reference manuals and textbooks, respectively, a few times a week or more [29]. These frequencies were second and third only to consulting with their physician supervisor (63%). Data from structured interviews of a sample of 22 community nurse prescribers reported by Hall et al. revealed that the majority relied on print materials to access information, namely the British National Formulary [32]. A survey of a primary care practice-based research network in the US that included physicians, physician assistants, and nurse practitioners, revealed that interpersonal and rapidly accessed print resources were preferred. Sixty-one and 58% of respondents reported using drug reference sources such as the Physician's Desk Reference (PDR) and medical textbooks, respectively, a few times a day or daily [56].
The clinicians in our sample perceived the Canadian compendium, the CPS, to be useful, accessible, credible, and current/timely. The CPS, is described as "the Canadian drug reference for health professionals" and is intended to provide a central source of drug information on drug products available in Canada [52]. It is available in print (English and French) and became available online in June 2004. The CPS includes drug monographs for commonly used products approved for use in Canada, but it does not include all drugs available on the Canadian market [57]. The majority of these product monographs are based on monographs submitted by pharmaceutical manufacturers and approved by Health Canada. Some of the monographs are written by the Canadian Pharmacists Association and are described as being evidence-based [52]. The CPS also includes more than 100 pages of clinical tools [52]. The CPS has been criticized for including pharmaceutical company advertising and requiring manufacturer payment for inclusion of product monographs [58]. The accuracy of particular components of CPS monographs has also been investigated. A review of overdose management in 119 monographs from the 2001 CPS revealed considerable variability in the utility of information with 50% of the monographs containing misleading or dangerous advice [59]. Since 2004, the CPS has included an alert box in the overdose section of monographs notifying users to contact Poison Control Centres for overdose management information. Some authors have criticized references that are similar to the CPS as being inadequate with regard to inclusion of evidence based information [60].
The NPs in our sample also rated Therapeutic Choices highly for all characteristics.  This finding is most likely  attributable to the fact that it is a recommended resource for coursework  associated with the Dalhousie NP university program curriculum.  Therapeutic Choices is a concise therapeutics reference text published by the Canadian Pharmacists Association. The text contains approximately 120 extensively referenced chapters with a disease management approach including easy to use algorithms and tables. An editorial board is responsible for extensively reviewing the content to ensure unbiased and objective information is presented [53].<br><br>Health professionals
Reliance on other health professionals, especially pharmacists and physicians, as a resource for information was evident from our study and concurs with the findings of others [28,32,55,61]. Nurse practitioners have reported that collaborative relationships with pharmacists increase NP role satisfaction [61]. NPs frequently consult with allied health care professionals in their primary health care provider role and this is supported by written feedback from our sample regarding frequently consulted health professionals. Nursing colleagues are also likely to be rated highly by NPs due to their affiliation with peers from the same profession.
Some investigators have shown that non-<span class="blue">human</span> references (e.g. textbook) are sought for more technical aspects of prescribing (e.g. dose), whereas guidance regarding selection of agents (i.e. right drug for an indication) is sought from <span class="blue">human</span> resources (e.g. pharmacists or physicians) [62]. We were unable to determine what kinds of resources were used for specific purposes from our study.<br><br>Online and electronic resources, computers, and personal digital assistants
From our study, computer survey respondents ranked online/electronic resources third in preference following print and health professionals. Various barriers and facilitators to accessing information online/electronically or via the Internet have been described in the literature [55,63-66]. Variables that have been described by others as barriers such as accessibility, high speed internet access, <span class="yellow">patient</span> volume, age, practitioner type, and technology support did not appear to influence computer searching for information on drugs or therapeutics related to <span class="yellow">patient</span> care in our results. Some qualitative feedback does however support this notion. As an example, in response to a request for a rationale for not using computers one physician commented: "Retro tech [sic]/old fashion. I still like to use my mind and have always been a fan of pen and paper". Barriers that were identified with our sample regarding the use of handheld technologies such as PDAs included cost, time, and issues related to technology literacy. Several <span class="yellow">people</span> questioned the value of PDAs. One GP stated when referring to a PDA: "So far I have not discovered a use for one". Other respondents reinforced their preferences for other resources (e.g. books) and resistance to technology. When responding to barriers for the use of PDAs, one NP commented, "My huge dislike for machinery that frequently requires updating and patience". A physician responded, "as stated, I like to use my own mind, and can get all the info I need from books relatively quickly". Facilitators to the use of PDAs mainly included convenience factors such as having resources all in one place, faster means to get information, and portability. Our sample was not in agreement with some convenience factors in that they did not feel that PDAs would decrease paperwork. Practitioners from our sample felt relatively neutral about PDAs improving <span class="yellow">patient</span>'s health outcomes with 41% responding in this manner. Results from a sample of primary care practitioners in the US revealed that 76% agreed that the use of handheld devices for electronic prescribing would substantially reduce medical errors and improve the quality of health care [67].
Our study also suggests that resources such as the Cochrane Library and its Database of Systematic Reviews were not frequently used. This finding is similar to that of other investigators [30,35,64]. Despite the desire of some clinicians to use these resources, lack of confidence and ability to use them appropriately has been found [30,64,68,69]. Our study suggests that although this resource is perceived as credible, current/timely, and useful, it is also perceived to be somewhat inaccessible. The Cochrane Library is available to the health professionals (e.g. nurses, physicians, pharmacists, occupational therapists, physiotherapists, etc.) in our sample through professional bodies via the Atlantic Health Knowledge Partnership [70].<br><br>Technology training: preferences and incentives
With regard to receiving training for a new technology, our study demonstrates that in <span class="yellow">person</span> conferences or one on one training sessions are the preferred means to receive continuing education. <span class="yellow">Person</span> to <span class="yellow">person</span> interaction has been reported as the preferred and most frequently used means to access continuing education or training by other investigators [55,71].
Our study also indicates that this group of practitioners may benefit from accessing resources [72-80] that provide guidance on useful drug information resources available for devices such as PDAs. This is exemplified by one respondent's statement "knowledge regarding good software programs" as a barrier to the use of PDAs.<br><br>Pharmaceutical industry
The influence of the pharmaceutical industry on physician prescribing and research outcomes has been documented [81,82]. Although NP use of industry representatives as a source of pharmacological information has been documented, the influence on prescribing is largely uninvestigated [32,61,83-85]. The CNA competency framework includes a statement regarding prescribing and industry relations [23]. In our study, the physician and NP rankings of industry representatives were similar. Within the health professionals and other category, pharmaceutical representatives were used as a resource by more of the sample than regional drug information services and comparably to academic detailing services. Academic detailing is a form of continuing  medical education where a trained health professional visits prescribers for  a fifteen to twenty minute session to provide objective information  regarding a therapeutic topic based on best available evidence [86,87]. Following academic detailing, physician and NP rankings of pharmaceutical industry representatives were second or third for frequency of use, usefulness, accessibility, credibility, and current/timeliness.<br><br>Limitations
We do not have demographics or information regarding the reasons why survey recipients did not respond. As per ethical requirements to maintain confidentiality of respondents, we were not able to match respondents from their respective place of practice and therefore cannot conclude whether the practitioners within a practice setting influenced the others' responses. The sample size of the survey is small although it includes 88% response from community based NPs in Nova Scotia. The generalizability of the results is limited due to the variations in NP scopes of practice nationally and internationally. It is unknown whether the findings are generalizable to nonresponding physicians within Nova Scotia collaborating with NPs or to physicians not in collaborative practices with NPs as they were not included as a part of the sample. Due to multiple statistical comparisons (Mann Whitney U), the results comparing NP and physician ratings of results should be interpreted with caution.<br><br>
Conclusion
Respondent ratings of resources and preferences for resource use were consistent with self-reported means of conducting searches for specific drug information queries. The use of computers and PDAs remains limited and also matches preferences and resource ratings. Education to this group of practitioners regarding available drug information resources may facilitate use of computer and PDA resources. Further research is needed to determine methods to increase the use of computers and PDAs and if use of these technologies affects prescribing and <span class="yellow">patient</span> outcomes.<br><br>Competing interests
Ingrid Sketris holds a Chair from Canadian Institutes of Health Research (CIHR), Canadian Health Services Research Foundation (CHSRF) co-sponsored by the Nova Scotia Health Research Foundation (NSHRF). Andrea Murphy received salary support through this Chair as a research fellow at the time of conducting this research. The survey was performed in fulfillment of the requirements for the Drug Use Management and Policy Residency that Murphy participated in as a part of her fellowship. The residency was conducted with a decision making partner from the Nova Scotia Department of Health.
The opinions expressed in this paper are those of the authors and do not represent the opinions of the Nova Scotia Department of Health, CIHR/CHSRF or NSHRF.
MF, MM, RMM, and DG have no competing interests to declare.<br><br>Authors' contributions
AM conceptualized the design and composed the survey instruments, carried out the study, entered and analyzed the data, drafted the original manuscript, and modified subsequent drafts based on authors' and reviewers' feedback. MF, RMM, IS, MM, and DG reviewed and suggested revisions to the survey tools, covering letters, overall study design, and contributed to feedback on the analysis and manuscript revisions.<br><br>Pre-publication history
The pre-publication history for this paper can be accessed here:<br><br>
Supplementary Material<br><br>
<h3>pmcA1636465</h3>Expression of C-terminal deleted p53 isoforms in neuroblastoma
Abstract
The tumor suppressor gene, p53, is rarely mutated in neuroblastomas (NB) at the time of diagnosis, but its dysfunction could result from a nonfunctional conformation or cytoplasmic sequestration of the wild-type p53 protein. However, p53 mutation, when it occurs, is found in NB tumors with drug resistance acquired over the course of chemotherapy. As yet, no study has been devoted to the function of the specific p53 mutants identified in NB cells. This study includes characterization and functional analysis of p53 expressed in eight cell lines: three wild-type cell lines and five cell lines harboring mutations. We identified two transcription-inactive p53 variants truncated in the C-terminus, one of which corresponded to the p53β isoform recently identified in normal tissue by Bourdon et al. [J. C. Bourdon, K. Fernandes, F. Murray-Zmijewski, G. Liu, A. Diot, D. P. Xirodimas, M. K. Saville and D. P. Lane (2005) Genes Dev., 19, 2122–2137]. Our results show, for the first time, that the p53β isoform is the only p53 species to be endogenously expressed in the <span class="blue">human</span> NB cell line SK-N-AS, suggesting that the C-terminus truncated p53 isoforms may play an important role in NB tumor development.<br><br>INTRODUCTION
The p53 tumor suppressor gene remains the most frequently altered gene in <span class="blue">human</span> tumors. Several p53 mutation databases have been reported previously (1–3), and to date, more than 1500 different p53 mutants have been described (4). Functional inactivation of p53 is usually due to gene mutation, deletion or protein degradation. In general, the majority of p53 mutations in <span class="blue">human</span> neoplasia are missense mutations affecting the DNA-binding domain (DBD). Unlike other <span class="blue">human</span> cancers, p53 in neuroblastoma (NB) is rarely mutated in the primary tumor at diagnosis but high levels of wild-type p53 (wt p53) protein expression have been found in the cytoplasm of undifferentiated tumors (5,6). More recently, in normal unstressed cells, wt p53 protein was found to be retained in the cytoplasm as a latent form, in huge, p53-associated protein complexes known as ‘Parc’ (7). The steady-state concentration of p53 in normal unstressed cells is usually very low because of the short half-life of the wild-type (wt) protein. Overexpression of p53 in most of the transformed cells containing a missense mutation within the p53 gene appears to be due to the increased stability of mutated p53 (8). In unstressed NB cells, high wt p53 expression may reflect the embryonic origin of NBs, in which precursor cells fail to mature (9).
p53 mutations are unusual in <span class="blue">human</span> NB but, when they do occur, are found in post-chemotherapy tumors. In this respect, Tweddle et al. (10) described how two NB cell lines derived from the same <span class="yellow">patient</span> can elicit a different p53 status: wt p53 for SK-N-BE(1a) established before treatment, and mutated p53 for SK-N-BE(2c) established after relapse of the <span class="yellow">patient</span> under treatment with cytotoxic agents such as cyclophosphamide, doxorubicin, vincristine and radiotherapy. In NB cell lines, p53 mutation has been found in multidrug-resistant cells (11). Various types of p53 mutation have been detected in NB cells and can lead to inactivation either by shut-down of protein expression or production of aberrant p53 products. Indeed, in LAN-1 cells, p53 nonsense mutation at cysteine 182 in exon 5 leads to the absence of protein (9), whereas in SK-N-BE(2) cells, missense mutation at codon 135 (C135F) leads to stable overexpressed protein (11). By analyzing IGR-N-91, a cell line established in our laboratory from the bone marrow of a <span class="yellow">patient</span> with metastatic NB after unsuccessful Adriamycin–vincristine chemotherapy (12), we identified another type of aberrant protein that arises from the duplication of exons 7-8-9. This duplication spans from amino acids 225 to 331, which represent part of the DBD and part of the oligomerization domain (13). However, each p53 mutant has been described in the literature as a case report, and so far, no comparative study has been undertaken to link their biochemical features with functional properties.
In the present study we report two novel p53 C-terminus mutants identified in SK-N-AS and IGR-NB8 <span class="blue">human</span> NB cell lines. The biological properties of these two new variants were analyzed in comparison with p53 isolated from six other <span class="blue">human</span> NB lines: three [LAN-1, SK-N-BE(2) and IGR-N-91] expressing mutant p53 and three (SH-SY5Y, LAN-5 and IMR-32) expressing the wt protein. This characterization was done by using a range of functional assays: (i) the ability of the protein to bind with p53 consensus sequence using the functional analysis of separated allele in <span class="yellow">yeast</span> (FASAY); (ii) the ability of the protein to transactivate the p53-responsive element (RE) identified either in the promoter of p21/WAF1 or in the first intron of BAX, using luciferase reporter assay; (iii) the induction of endogenous p21/WAF1 gene expression under stress conditions.<br><br>MATERIALS AND METHODS
Neuroblastoma cell lines, culture and drug treatments
The parental <span class="blue">human</span> NB SH-SY5Y, SK-N-AS, IMR-32 and SK-N-BE(2) cell lines were purchased from the European Collection of Cell Cultures (ECACC, Wiltshire, UK). The <span class="blue">human</span> IGR-NB8 cells (a gift of Prof. Gilles Vassal, UPRES EA 3535, Institut Gustave Roussy, Villejuif) were derived from a previously untreated localized NB (14). The LAN-1 and LAN-5 cell lines were provided by Dr Nicole Gross (Pediatric Oncology Research, Lausanne, Switzerland). The <span class="blue">human</span> IGR-N-91 cell line was established in our laboratory from the bone marrow of a <span class="yellow">patient</span> with metastatic NB after unsuccessful adriamycin–vincristin chemotherapy (12). LAN-1, LAN-5 and IMR-32 were grown in RPMI medium supplemented with 2 mM l-glutamine and 10% fetal <span class="yellow">calf</span> serum and gentamicine 10 μg/ml. Others cell lines were cultured in DMEM.
For activation of endogenous p53, cells were treated with cis-platinum (Sigma) (10 μg/ml) for 24 h then lysed for western blot analysis.<br><br>Western blot analysis
This procedure was carried out as described previously (13). Protein lysates (50 μg) were submitted to 10% SDS–PAGE, and then transferred onto nitrocellulose filters. After saturation, the membranes were incubated with primary antibody diluted in 0.1% phosphate-buffered saline, Tween-20 and 3% skim milk. The primary antibodies used were anti-p53 monoclonal antibody (clone DO-7, 1/1000, DAKO), anti-p21/WAF1 monoclonal antibody (Ab-1, 1/200, Oncogene Research) and anti-β-actin monoclonal antibody (1/1000; Chemicon) as internal control. Protein bands were detected by ECL system (Amersham).<br><br>PCR, plasmids cloning
Genomic DNA was extracted using lysis buffer containing 20 mM Tris–HCl, pH 7.5; 0.4 M NaCl; 0.5% SDS; 10 mM EDTA, treated with proteinase K (200 μg/μl), purified with phenol/chloroform, precipitated with ethanol and dissolved in DNase free water. Total RNA were purified using RNAble reagent (Eurobio), precipitated with isopropanol and dissolved in RNase free water. cDNA was obtained by reverse transcription of 1 μg of total RNA using Superscript II™ RNase H-Reverse transcriptase (Invitrogen) and Oligo-d(T)16 in conditions specified by the manufacturer. Amplification of full-length p53 coding region from SH-SY5Y, IGR-N-91, IGR-NB8 and SK-N-BE(2) cell lines was performed using forward primer at position 152 and reverse primer at position 1583 (F1 and R7, respectively, Table 1; GenBank accession no. K03199). p53 cDNA from SK-N-AS cells for cloning was obtained from RT–PCR using F1 and reverse primer i9+: 5′-GCAAAGTCATAGAACCATTTTCAT-3′ (nucleotide position 14989, GenBank accession no. X54156) primers which encompass from exon 1 to exon i9+ first identified by Flaman et al. (15) included. The PCR was done in the presence of pfu Hotstart DNA polymerase (Stratagene) for 30 cycles of 1 min at 90°C, 1 min at 65°C and 2 min 30 s at 72°C using PTC-100 thermocycler (MJ-Research).
The p53 cDNA from SH-SY5Y, SK-N-AS, IGR-N-91, IGR-NB8 and SK-N-AS cells were then cloned into pcDNA3.1/V5-His-Topo vector (Invitrogen) according to the manufacturer's instruction. The p53 sequence of each cell line was investigated by sequencing of plasmids after cloning. Sequencing was performed by Genome Express (Meylan, France).
The pDDm-TO harboring p53 dominant negative form (p53DD) pGL3-E1bTATA and the pE1B-hWAF1 firefly luciferase reporter containing the p53-responsive element of the p21/WAF1 promoter were described previously (16,17) pE1B-BAXi contains the p53RE identified in the intron 1 of the BAX gene [(18) and D. Munsch, personal communication]. Oligonucleotides TCGAGGGCAGGCCCGGGCTTGTCG and CTAGCGACAAGCCCGGGCCTGCCC were annealed and cloned into pGL3-E1bTATA digested with NheI and XhoI to obtain pE1B-BAXi. The pcDNA3-ΔNp73α expression plasmid was a gift of Dr Daniel Caput (SANOFI, Labèges, France).<br><br>Fluorescent in situ hybridization (FISH)
Cytogenetic preparations
Metaphase spreads from healthy <span class="blue">human</span> male lymphocytes and tumor cell lines were prepared as described previously (19). BAC probe RP11-199F11, containing a 167 kb region spanning TP53 gene, was labeled by random priming in the presence of Alexa 594-dUTP (Molecular Probes). A commercial probe specific for chromosome 17 centromere, and labeled with spectrum green, was obtained from Vysis. After over-night cohybridization of the probes in the presence of Cot-1 DNA, the slides were washed and DNA counterstained with DAPI. The preparations were observed with an epifluorescence microscope and images captured with a Vysis imaging station. Between 3 and 14 metaphases spreads and 30–200 nuclei were examined for each cell line.<br><br>
Luciferase reporter assays
LAN-1 or SH-SY5Y cells were seeded in duplicates onto 6-well plates at a density of 2 × 104 cells per cm2 and cotransfected 24 h later with 0.5 μg (2.5 μg/ml) of pGL3 firefly luciferase reporter gene plasmid under the control of either pE1B-hWAF1 or pE1B-BAX using lipofectamine 2000 and 1 μg of either a p53 expressing plasmid or an empty vector. At 24 h after transfection, cells were lysed with 200 μl/well of passive lysis buffer provided with the ‘Luciferase assay kit’ (Promega). Luciferase activity was measured using Microlumat LB96P luminometer (EG & G Berthold Instrument).<br><br>Functional assay in <span class="yellow">yeast</span>
cDNA was obtained by RT of 1 μg of total RNA using Superscript II™ RNase H-Reverse transcriptase (Invitrogen) and random hexamers to prime the synthesis in conditions specified by the manufacturer. p53 cDNA was amplified by PCR and cotransformed into <span class="yellow">yeast</span>, IG397 Ade2 strain, together with either pRDI-22 vector for p53-standard assay or pFW35 and pFW34 plasmid for 5′ or 3′ split assay, respectively, carrying the ADE2 open reading frame under the control of a p53-responsive promoter (20). In a selective medium lacking leucine, wt-p53 activates transcription of ADE2 gene that encodes enzyme—phosphoribosylimidazole carboxylase—implicated in adenine biosynthesis. Therefore, a colony of cells that expresses ADE2 gene is white whereas the one composed of cells where ADE2 gene is not expressed owing p53 mutation is red.<br><br>
RESULTS
P53 status in SK-N-AS and IGR-NB8 cells
We first compared the migration profiles of p53 expressed in SK-N-AS and IGR-NB8 with those expressed in the other six NB cells, SH-SY5Y, LAN-5, LAN-1, IMR-32, SK-N-BE(2) and IGR-N-91. Western blots from 50 μg of total protein extracts were revealed with p53 monoclonal antibody (DO-7). A range of profiles was identified as shown in Figure 1A. As expected, p53 extracted from the three cell lines, SH-SY5Y, LAN-5 and IMR-32, expressing wt protein (13,21) migrated at the wt position. Of particular note in these three wt p53 cell lines was an additional faint band that migrated faster than the full-length protein. The LAN-1 cells were found to be p53 deficient (9). The SK-N-BE(2) cell line showed an intense band reflecting p53 stability due to a missense mutation at codon 135 (11). As expected, due to the previously identified duplication of exons 7-8-9 (13), p53 protein migration was delayed in the IGR-N-91 cells. In contrast, the p53 protein in the SK-N-AS cell line migrated noticeably faster than the wt protein, indicating that it was smaller in size. The p53 protein in the IGR-NB8 cell line was even smaller than that in SK-N-AS.
To analyze the coding region of each of the p53 variants, RT–PCR was performed using p53-specific F1-R7 primers (Table 1 and Figure 1B). The expected 1430 bp for full-length p53 was amplified from wt p53-expressing SH-SY5Y, LAN-5 and IMR-32 cell lines. As SK-N-BE(2) harbors a single point mutation at codon 135 (C135F), the amplified fragment analyzed by electrophoresis migrated as wt p53 (Figure 1B). The longer RT–PCR fragment from the mutated IGR-N-91 cell line resulted from the duplication of exons 7-8-9, as shown in our previous data (13), which corresponds to extra nucleic material of 321 nt. For the LAN-1 cells, no amplified fragment was observed in accordance with published data, which demonstrates the extremely low levels of p53 mRNA and the undetectable level of protein (9). An amplified fragment of the same length as the wt protein was observed in the IGR-NB8 cell line (Figure 1B). Indeed, complete gene sequencing revealed a point mutation E326STOP leading to a truncated protein at C-terminus. No fragment, however, was amplified from SK-N-AS with F1-R7 primers.
To further map the p53 mRNA transcribed in these cells, series of RT–PCR tests were performed using the forward primer, F2 (exon 8 position 1008th according to GenBank accession no. K03199), matched with different reverse primers, R3 (at the junction of exon 8/9, nt position 1124), R4, R5 (in exon 9 at positions 1154 and 1184, respectively), and R6 (in exon 10, at position 1230). The sequences of these primers are given in Table 1 and the results are presented in Figure 2A. SK-N-AS cDNA gave an amplified fragment of the same size as SH-SY5Y cDNA with the three primer pairs, F2/R3, F2/R4 and F3/R5. However, in contrast to SH-SY5Y, no fragment was obtained with SK-N-AS cDNA using the F2/R6 primer pair, which suggests the absence of exon 10 in SK-N-AS mRNA.<br><br>The p53 protein expressed in SK-N-AS is the p53β isoform
An alternatively spliced form of <span class="blue">human</span> p53 mRNA with an additional 133 bp exon derived from intron 9 has been detected in normal <span class="blue">human</span> lymphocytes (15). This spliced variant named ‘i9+’ encodes a truncated protein of 341 amino acids including 10 new amino acids derived from the novel exon, the p53β isoform according to Bourdon et al. nomenclature (22). This led us to hypothesize that the shorter protein expressed by the SK-N-AS cell line could be the p53β isoform. To test this hypothesis, RT–PCRs were performed using primer sets designed to amplify the 3′ region of p53 mRNA encoding either the specific C-terminal part of the wt protein (wt C-ter) or the specific C-terminal part of the β isoform (β C-ter). In parallel amplification with a primer pair amplifying the DBD was used as control. The sequences of these oligonucleotides are given in Table 1. The results presented in Figure 2B are consistent with the only expression of the p53β isoform in SK-N-AS as no band was observed in lane using specific C-ter primer located in exon 10. Interestingly, RT–PCR using SH-SY5Y (SH) gave an amplified fragment not only with the primer pair specific to the C-ter domain of wt p53 but also with the primer pair specific to the p53β isoform. This result, combined with the presence of an additional faint band migrating faster than wt p53 in denaturing polyacrylamide gel (Figure 1A), strongly suggests that both the p53 full-length protein and the β isoform were expressed in the SH-SY5Y cells.
The full-length SK-N-AS p53 cDNA was then amplified with the forward primer F1 and a reverse primer located within the novel exon i9+ (Table 1). This amplified fragment was cloned in pcDNA3/V5-His-Topo, as described in Materials and Methods. Its sequence analysis confirmed that the truncated p53 expressed in SK-N-AS was encoded by the i9+ splice variant described previously by Flaman et al. (15) that encodes the p53β isoform characterized by Bourdon et al. (22).
A series of genomic amplifications were performed to identify a possible deletion within the intron 9 that could account for the absence of normal size p53 in SK-N-AS cells. The primer sequences are given in Table 1. Amplifications were performed in parallel with total DNA extracted from SH-SY5Y and SK-N-AS cells. Results are presented in Figure 3. Normal size fragments that encompass the acceptor site of intron 9 were amplified with SH-SY5Y as well as with SK-N-AS DNA. On the contrary amplification fragments that encompass the intron 9 donor site were obtained only with SH-SY5Y but not with SK-N-AS DNA. These results identify a deletion of the intron 9/exon 10 junction within the SK-N-AS p53 gene.<br><br>A <span class="yellow">yeast</span> functional assay confirmed the absence of p53 full-length expression in SK-N-AS and IGR-NB8
It is possible to detect p53 mutation using a simple <span class="yellow">yeast</span> colony color assay as described by Flaman et al. (23). When the strain is transformed with a plasmid-encoding wt p53, the cells express the ADE2 gene and produce white colonies (Figure 4A, a, b2 and c1, and Figure 4B, dish a). Cells containing mutant p53 fail to express ADE2 and form small red colonies (Figure 4A, b and b1, and Figure 4B, dish c). When the p53 cDNA fragment is deleted, cells are unable to form a colony (Figure 4A, c and c2). As shown in Table 2, FASAY was performed as a p53-standard test with full-length cDNA or with the split version at the 5′ and 3′ end (15). The background of FASAY experiments is around 10%. p53 wt expressing SH-SY5Y and LAN-5, 2 wt cell lines, yielded ∼92–97% of white colonies (Table 2).
One hundred percent of the colonies carrying SK-N-BE(2) p53, which is homozygous for the C135F mutation, turned red with the standard or 5′ split assay, whereas 94% of the colonies turned white with the 3′ split assay since the missense mutation does not extend to the C′-terminus of the gene (Table 2 and Figure 4B, dish c). No colonies were observed with p53-deficient LAN-1 cells, (see also Figure 4A, c and c2). With SK-N-AS cells, the split 5′ assay gave 96% white colonies, while the p53-standard and split 3′ assay did not produce any colonies (Table 2). This means that the 5′ terminus was intact whereas the 3′ terminus had been deleted, as was confirmed by nucleotide sequence analysis. The IGR-NB8 colonies, however, were red both with the p53-standard assay and the split 3′ assay. In the IGR-N-91 cell line, where p53 harbors two contiguous sets of exons 7–9, spanning the DBD and oligomerization domain, it is interesting to note that the <span class="yellow">yeast</span> colonies were predominantly white (Figure 4B, dish b) with the split 5′ and the split 3′ assay (96 and 87% of white colonies, respectively) This suggests that the cells express a binding ability that is specific to wild-type p53 rather than mutated p53 (Table 2).<br><br>Transcription activity of SH-SY5Y, IGR-N-91, SK-N-BE(2), SK-N-AS and IGR-NB8 p53 variants in mammalian cells
To determine the transactivation ability of p53 variants in mammalian cells, we used a reporter gene strategy. The p53RE located within either the <span class="blue">human</span> p21/WAF1 promoter or the intron 1 of the <span class="yellow">mouse</span> and <span class="blue">human</span> BAX gene [(18) and D. Munsch, personnal communication) were cloned in a luciferase reporter gene plasmid upstream of the E1B minimal promoter as described in Materials and Methods. The p53-negative LAN-1 cells were cotransfected with the p53 vectors expressing the p53 cloned from either SH-SY5Y, IGR-N-91, SK-N-BE(2), SK-N-AS or IGR-NB8 and the luciferase reporter plasmids. Both p53RE were strongly stimulated in cells cotransfected with wt p53 cloned from SH-SY5Y, as compared to cells cotransfected with an empty plasmid. In contrast, none of the p53 variants was able to transactivate the expression of luciferase driven by either p21/WAF1 or BAX p53RE (Figure 5).
To test transactivation capability at the protein level, each variant was transfected into p53-negative LAN-1 cells and the stimulation of endogenous p21/Waf1 gene expression was analyzed by western blotting. As shown in Figure 6, in contrast to wt p53, none of the p53 variants was able to induce p21 protein accumulation.
We then tested for a possible dominant negative effect of these various mutants on wt p53-dependent transcriptional activity. To this end, SH-SY5Y cells were cotransfected with constructs encoding the luciferase gene driven by either the p21/Waf1 or BAX p53RE and the constructs expressing the various p53s cloned from IGR-NB8, SK-N-BE(2), IGR-N-91, SK-N-AS and IGR-NB8 NB cells or p53DD, a dominant negative mutant of wt p53 (24). The stress induced by transfection activated the transcriptional activity of the wt p53 expressed in SH-SY5Y, leading to a p53-dependent expression of luciferase as illustrated by the fact that coexpression of p53DD led to a substantial decrease in luciferase activity when compared to the luciferase activity of cells cotransfected with an empty plasmid (Figure 5). Compared to p53DD, mutants within the DBD isolated from IGR-N-91 or SK-N-BE(2) had only a moderate dominant negative effect on endogenous wt p53 transcriptional activity. More surprisingly, the transfection of the C-terminal truncated variant IGR-NB8 enhanced both BAX and p21/WAF1 p53RE activity. The coexpression of p53β cloned from SK-N-AS also enhanced BAX p53RE activity. A similar effect has been reported already for p53β by Bourdon et al. (22).
When combined, these results show that all the p53 variants isolated from the NB cells had lost the ability to specifically transactivate the p53 target genes. Their effect on the transcriptional activity of endogenous wt p53 expressed in SH-SY5Y cells, however, largely depended on the p53 domain affected by the modification.<br><br>All the identified p53 variants inhibited the induction of endogenous p21/WAF1 gene expression under stress conditions
We further examined whether the four p53 variants, SK-N-BE(2), IGR-N-91, SK-N-AS and IGR-NB8, had loss their ability to stimulate the endogenous expression of the p21/WAF1 gene, the archetypical cell cycle inhibitor and the true target of p53. To this end, cellular response to genotoxic stress was analyzed by western blot following treatment of the various cell lines with cisplatin, one of the most potent antitumor agents used in neuroblastoma. Results are presented in Figure 7. None of the mutant cells, regardless of the type of mutation, was able to induce p21/WAF1 protein accumulation, unlike the 3 p53 wild-type cells (SH-SY5Y, IMR-32 and LAN-5).<br><br>Genomic status of TP53 region in the various cell lines
According to Knudson's ‘two hit’ model of tumor suppressor gene functional inactivation, the mutation of one allele is supposed to be associated with a deletion of the second allele. To assess this genetic mechanism, we performed FISH experiments to search for deletions of one copy of the TP53 region, especially in cell lines with a mutated TP53 gene. For this purpose, metaphase preparations of the studied cell lines were cohybridized with a p53 DNA probe labeled in red (BAC clone RP11-199F11) as described previously (25) and a chromosome 17-specific centromeric probe labeled in green. The three cell lines shown previously to express a wt p53 protein (LAN5, IMR32 and SH-SY5Y), displayed as expected two signals with each probe, confirming the presence of both TP53 alleles in these cells (Figure 8 and Table 3). Conversely, IGR-N-91 and SK-N-AS cell lines displayed only one fluorescent signal for each probe, suggesting a whole chromosome 17 lost, or at least losses of the 17p arm and the centromeric region. The SK-N-BE(2) cell line has been described as containing only one chromosome 17 and one TP53 signal (10). In our analysis, only 10% of the cells displayed this characteristic, whilst most of the cells had two copies of both (Figure 8 and Table 3). p53 sequencing, however, confirmed the previously described mutation, and the absence of a normal allele, suggesting that the cells used in our study had acquired, during culture, an uniparental disomy for the TP53-mutated chromosome 17. Finally, the other two cell lines (LAN-1 and IGR-NB8) displayed highly variable genetic heterogeneity from one cell to the next (Table 3). Surprisingly, although p53 transcripts are extremely faintly expressed in LAN-1 cell line, all cells showed several FISH signals with the 167 kb BAC probe used here. To understand this apparent contradiction, an array-CGH experiment was performed on an oligo-array Agilent, which indicated a 133 kb interstitial deletion corresponding to the p53 coding region and the 97 kb upstream region. Accordingly, the fluorescent spots observed in FISH experiments on LAN-1 cells should be related to the hybridization of the 57 kb region downstream of p53 gene present in the BAC probe. IGRN-B8 cell line displayed a number of signals of both colors ranging from 0 to 4, with 87% of cells displaying a loss for one TP53 allele. Despite this genomic variability, analysis of the p53 protein showed a single shortened form in IGR-N-B8 cell line (Figure 1). Consequently, and as suggested for SK-N-BE(2) cell line, IGR-N-B8 cell line should contain a variable number of copies of chromosome 17 with mutated p53.<br><br>
DISCUSSION
The p53 gene, the ‘genome guardian’, is mutated in over 50% of <span class="blue">human</span> cancers, with the most common mutations being missense mutations (>2/3 of mutations) (26). In <span class="blue">human</span> neuroblastoma tumors, p53 mutations are rarely present at the time of diagnosis (5,27); however, oncogenic p53 mutations can be found in advanced neuroblastomas that often relapse following high-dose chemotherapy (10). In contrast, in breast cancers, it has been reported that p53 mutations might improve response to high-dose chemotherapy including therapy with epirubicin and cyclophosphamide (28).
An investigation into the p53 genomic status and functions of eight <span class="blue">human</span> NB lines revealed that all five of the mutated cell lines had distinct genetic characteristics as is schematically represented in Figure 9: SK-N-BE(2) with a single missense mutation in the p53 gene, encoding a highly stable full-length protein. SK-N-AS and IGR-NB8 proteins, although they have intact transactivation and DBDs, were truncated at the C-terminus generating 341 and 326 amino acids respectively; they therefore lack the tetramerization domain that is essential for an active conformation. Very recently, Bourdon et al. (22) reported the putative occurrence of β and γ isoforms from different tissues due to alternate splicing that indicates the similarity to those of p73 and p63 as identified previously by Daniel Caput and co-workers (29). In the p53 isoforms scheme proposed by Bourdon et al. (22), the SK-N-AS cell line that elicits p53i9 protein expression is consistent with the p53β isoform. Genomic analysis reveals that the only occurrence of the p53β isoform in SK-N-AS results from a deletion spanning the intron 9/exon 10 junction. Similar to the p53β isoform in SK-N-AS, the p53 in IGR-NB8 that lacks 67 amino acids at C-terminus was, alone, unable to induce p21/WAF1 promoter activation except with endogenous wt-p53 on SH-SY5Y cells where transfection with IGR-NB8 significantly augmented the transcriptional activation of the p21/Waf-1 promoter (Figure 5A). Studies by other authors have reported the interaction between the C-terminal domain and another region that impedes the active conformation of p53, suggesting an allosteric model for p53 activity regulation (30). Such events have been demonstrated for the 342-stop mutant, generated by mutagenesis, which can modulate transactivation, growth and apoptosis (31). Moreover, Harms and Chen (32) reported that the C-terminal basic domain inhibits induction of the proapoptotic target gene insulin-like growth factor binding protein 3, suggesting that IGR-NB8 might induce this gene. IGR-N-91 had an abnormally high molecular weight protein due to the duplication of wild-type exons 7-8-9, thus affecting the DBD and OD; and LAN-1, with a mutation at codon 182 (Cys→stop) concurred with an earlier report showing extremely low levels of mRNA and undetectable protein expression (9).
Notably, all the p53 variants, including SK-N-AS (β isoform) and IGR-NB8 (C-terminal truncated p53), elicited a total lack of p21 promoter activation. In particular, the p53β isoform was unable to induce endogenous p21 expression in SK-N-AS (Figure 6), concurring with data obtained from in vitro transfection experiments in H1299 cells by Bourdon et al. (22). For the IGR-N-91 cells, although p53 was mutated and unable to transactivate the p21/WAF1 promoter, the FASAY global test was not conclusive since ∼80% of colonies were white and nearly 20% (see also Table 2), though not enough, were red. Moreover, in this particular line, standard sequencing on cDNA using primers located within each exon as used for routine tumor analysis was unable to detect any anomalies in p53 genetic status (data not shown). These results enlighten the limit of the conventional tests to detect a transcription inactivation of p53 brought by duplication within the DBD.
Analysis of p53 genomic status was explored by FISH experiments, in search for a potential biallelic inactivation of p53, with a mutation of one allele and a deletion of the second one. This situation was indeed clearly observed in IGR-N-91 and SK-N-AS cell lines, with an unambiguous loss of one chromosome 17p arm in all cells of both. SK-N-BE(2), LAN-1 and IGR-NB-8 cell lines showed a more complex genomic situation which should be relevant of variable copy numbers of chromosome 17 bearing in most cases (LAN-1) or in all cases [SKN-BE(2), IGR-NB8] the mutated characteristic p53 allele. Our data therefore clearly demonstrate that each technique has a role and a combination of techniques is required in order to correctly define the p53 phenotype and genotype in tumor and particularly in NB cells.
Our data enlighten a high frequency of the C-terminal abnormalities (3/5 mutated) in NB cell lines. For SK-N-AS and IGR-NB8, a part of the oligomerization domain was lost and IGR-N-91 gained an extra oligomerization domain. According to FASAY assay the p53 expressed in IGR-N-91 still specifically bind DNA but not the p53 expressed in SK-N-AS and IGR-NB8 in agreement with previous published data obtained by electrophoretic mobility shift assay (33).
With regards to biological relevance, different mutants within the DBD vary in their oncogenicity. They are classified into two types depending of the location of the mutation, mutations of class I occur in the DNA contact areas, while class II mutations occur in areas important for the conformational stability of p53 protein (30). Although both class I and class II mutants have loss its ability to specifically bind DNA, class II mutations have been shown to be more oncogenic than class I. However, to our knowledge the oncogenicity of mutant affecting the C-terminal domain have not been studied. The biological role of the C-terminal mutants needs now to be thoroughly investigated in NB tumors.<br><br>
<h3>pmcA1779460</h3>MUC1 alters oncogenic events and transcription in <span class="blue">human</span> breast cancer cells
Abstract
Introduction
MUC1 is an oncoprotein whose overexpression correlates with aggressiveness of tumors and poor survival of cancer <span class="yellow">patients</span>. Many of the oncogenic effects of MUC1 are believed to occur through interaction of its cytoplasmic tail with signaling molecules. As expected for a protein with oncogenic functions, MUC1 is linked to regulation of proliferation, apoptosis, invasion, and transcription.<br><br>Methods
To clarify the role of MUC1 in cancer, we transfected two breast cancer cell lines (MDA-MB-468 and BT-20) with small interfering (si)RNA directed against MUC1 and analyzed transcriptional responses and oncogenic events (proliferation, apoptosis and invasion).<br><br>Results
Transcription of several genes was altered after transfection of MUC1 siRNA, including decreased MAP2K1 (MEK1), JUN, PDGFA, CDC25A, VEGF and ITGAV (integrin αv), and increased TNF, RAF1, and MMP2. Additional changes were seen at the protein level, such as increased expression of c-Myc, heightened phosphorylation of AKT, and decreased activation of MEK1/2 and ERK1/2. These were correlated with cellular events, as MUC1 siRNA in the MDA-MB-468 line decreased proliferation and invasion, and increased stress-induced apoptosis. Intriguingly, BT-20 cells displayed similar levels of apoptosis regardless of siRNA, and actually increased proliferation after MUC1 siRNA.<br><br>Conclusion
These results further the growing knowledge of the role of MUC1 in transcription, and suggest that the regulation of MUC1 in breast cancer may be more complex than previously appreciated. The differences between these two cell lines emphasize the importance of understanding the context of cell-specific signaling events when analyzing the oncogenic functions of MUC1, and caution against generalizing the results of individual cell lines without adequate confirmation in intact biological systems.<br><br><br><br>Introduction
MUC1 is the founding member of the mucin family: proteins characterized by heavy O-glycosylation centering around a variable number of tandem repeats that are rich in serine and threonine residues [1,2]. MUC1 is a transmembrane heterodimer with one subunit solely extracellular (MUC1-EX), and the other subunit composed of a short extracellular stem, a single transmembrane domain, and the cytoplasmic tail (together called the MUC1-CT). MUC1 possesses both pro- and anti-adhesive capacities, as the MUC1-EX provides binding sites for a variety of adhesion proteins, while its large size and extended structure prevents cell-cell contact [3-5].
Initially described as a tumor antigen overexpressed in >90% of breast cancers, MUC1 is now known to be an oncogene with roles in both tumor formation and progression [1,6]. <span class="yellow">Mouse</span> studies have been integral to the current understanding of MUC1 in cancer. Muc1 knockout <span class="yellow">mice</span> (Muc1-/-; MUC1 is <span class="blue">human</span>; Muc1 is <span class="yellow">mouse</span>) show a reduction in tumorigenic phenotype when crossed onto <span class="yellow">mice</span> overexpressing the Wnt-1 [7] or polyomavirus middle T antigen [8] oncogenes in the mammary gland. In contrast, MUC1 overexpression in the mammary gland drives tumor formation [9], indicating that MUC1 is a true oncogene.
Many of the oncogenic effects of MUC1 stem from its cytoplasmic tail, which binds to several proteins implicated in cancer, including c-Src [10,11] and the epidermal growth factor receptor (EGFR) family [12,13]. MUC1 stimulates mitogen activated protein kinase (MAPK) signaling through the extracellular signal regulated kinases (ERK1/2) [12,14]; this can occur through MUC1 association with Grb2 and son of sevenless to activate Ras [15]. ERK1/2 signaling is commonly stimulated by the Ras-Raf-MEK (MAPK and ERK kinase) cascade downstream of mitogens such as EGFR [16], and regulates transcription via factors like the activator protein-1 complex. Loss of MUC1 can reduce EGFR expression [17], providing another means of affecting MAPK signaling. Our results describe a novel mechanism by which MUC1 regulates the ERK1/2 pathway, through modulating transcription of the genes encoding MEK1, Raf-1, and c-Jun.
MUC1 expression correlates with increased survival in response to cytotoxic or oxidative agents [18-21], and can activate the phosphoinositol-3 kinase-AKT pathway as part of an anti-apoptotic response [18]. MUC1 has also recently been linked to transcription, as the MUC1-CT localizes to the nucleus [22] and affects transcription by β-catenin [22,23], FOXO3a [21], p53 [24], and estrogen receptor α [25]. However, there are indications that the role of MUC1 in oncogenesis is regulated by cell type and signaling context. For example, MUC1 can stimulate Fas-mediated apoptosis [26], while Muc1 is specifically down-regulated in c-neu-induced mammary tumors [27]. This report emphasizes the complexity of MUC1 signaling in breast cancer by contrasting results from two established breast cancer cell lines.
To understand MUC1 function in cells with high endogenous expression, that is, cells likely to have evolved with active MUC1 signaling, we used small interfering RNA (siRNA) to knock down MUC1 in MDA-MB-468 and BT-20 cells. We then analyzed transcription of 84 genes involved in cancer, as well as the effects upon cellular events linked to oncogenesis, such as apoptosis and proliferation. Though the cell lines show some similarity in transcriptional alterations after transfection with MUC1 siRNA, their phenotypes are quite dissimilar: MDA-MB-468 increases apoptosis and reduces proliferation and invasion, while BT-20 proliferates more rapidly after loss of MUC1. This last may reflect the striking amount of active AKT in BT-20; AKT activity is increased in both cell lines after MUC1 siRNA, which agrees with a previous study of MUC1 siRNA [21], but disagrees with results from 3Y1 fibroblasts [18]. Recent studies have emphasized the complex and context-specific regulation of even such classic oncogenes as AKT [28]. The differences between the two breast cancer cell lines in this study suggest that MUC1 oncogenic functions are also subject to cell-specific regulation, and stress the need for understanding the cellular signaling context when interpreting results.<br><br>Materials and methods
Cell culture and siRNA transfection
MDA-MB-468 and BT-20 cells (American Type Culture Collection) were cultured in Dulbecco's modified Eagle's medium (Invitrogen, Carlsbad, CA, USA) plus 10% fetal <span class="yellow">calf</span> serum, 1% Glutamax (Invitrogen) and 1% penicillin/streptomycin. Stable cell lines (468.Neo and 468.MUC1Δ8) were selected with 0.5 mg/ml G418. For epidermal growth factor (EGF) stimulation, MDA-MB-468 cells were serum-starved overnight and treated for 10 minutes at 37°C with 100 ng/ml EGF. Transient siRNA transfection was performed with Lipofectamine2000 (Invitrogen) and 100 nM siRNA oligonucleotides. The commercially available siRNA constructs (all from Dharmacon, Lafayette, CO, USA) were scrambled (siCONTROL non-targeting siRNA #1), or directed against firefly luciferase (siCONTROL non-targeting siRNA #2) or MUC1 (siGENOME smartpool). The independent oligonucleotides designed in our laboratory target sequences beginning at MUC1 codons 882 and 956, and have been described previously [29]. The scrambled siRNA construct was used only in BT-20 cells as it causes a non-specific knockdown of MUC1 in the MDA-MB-468 line.<br><br>Cloning of MUC1 WT vector and stable transfection
Two silent mutations (G891A and T894C) were introduced into the MUC1 cDNA (called MUC1Δ8) to make it resistant to the 882 siRNA that targets that region of the mRNA. The mutant cDNA was cloned into the pLNCX.1 vector with neomycin resistance (gift of Joseph Loftus, Mayo Clinic, Arizona, USA). Stable transfection was performed with Lipofectamine2000; cells were selected beginning 24 hours post-transfection and maintained as a polyclonal population.<br><br>Western blots and antibodies
Cells were lysed in buffer (20 mM HEPES pH 8.0, 150 mM sodium chloride, 1% Triton X-100, 2 mM EDTA) with commercial protease (Complete inhibitor cocktail, Roche, Pleasanton, CA, USA) and phosphatase inhibitors (10 mM sodium fluoride, 2 mM sodium vanadate, 50 μM ammonium molybdate). Protein concentration was determined by BCA (Pierce, Rockford, IL, USA); 50 μg of lysate were loaded on SDS-PAGE gels for each experiment, except for the pMEK1/2 blot in MDA-MB-468, where 150 μg were used. Non-commercial antibodies used were: BC2, a <span class="yellow">mouse</span> monoclonal to the MUC1-EX (gift of Dr McGuckin, Queensland University, Queensland, Australia), and CT2, an <span class="yellow">Armenian hamster</span> monoclonal to the MUC1-CT developed in our lab [12]. Antibodies to pMEK1/2, MEK1/2, ERK1/2, Myc, pAKT, AKT, β-tubulin (all Cell Signaling, Danvers, MA, USA), β-actin and dpERK1/2 (both Sigma, St. Louis, MO, USA) were used according to manufacturers' recommendations. All antibodies except β-actin (1:2,500) and dpERK1/2 (1:10,000) were used at 1:1,000 dilution for western blots. Flow cytometric analysis of MUC1 was done with HMPV-FITC, which recognizes the core peptide of the MUC1-EX tandem repeats (Pharmingen, San Diego, CA, USA). Bromodeoxyuridine (BrdU) staining was performed with a fluorescently conjugated antibody to BrdU (BrdU-PE, BD Biosciences, San Diego, CA, USA) as described below. Densitometry was performed using the public domain ImageJ program (developed at the NIH and available at [30]. Each band was measured in three places; the results were averaged and normalized to tubulin to control for loading.<br><br>Transwell invasion assays
Cells were serum-starved beginning 24 hours post-siRNA transfection. Cells were re-plated in serum-free medium 48 hours post-transfection at 50,000 cells per insert (sized for 24-well plates), with serum-containing medium in the bottom of the growth well as an attractant. Transwell inserts (BD Biosciences) pre-coated with laminin, fibronectin, collagen IV, or control (no matrix) were used, and cells were permitted to invade for 48 hours. At this point (96 hours post-transfection), visual inspection of the growth wells confirmed that negligible numbers of cells went through to the bottom of the plate. Non-invaded cells were swabbed from the tops of half of the inserts ('samples', containing only invaded cells), and retained in the others ('controls', all cells). Inserts were stained for 10 minutes with crystal violet (0.5% in 20% methanol) and washed with water. Membranes were cut out and destained for 10 minutes in 10% acetic acid in a 96-well plate; membranes were removed and absorbance was read at 570 nm. Percent invasion is defined as (absorbance of samples/absorbance of controls) × 100.<br><br>[3H]Thymidine incorporation assays
Cells were re-plated in quadruplicate 24 hours post-siRNA transfection at 15,000 cells/well (96-well plate) with [3H]thymidine (1 μCi/well), then incubated in normal conditions for 24 hours. At this time (48 hours post-siRNA transfection) excess radioactivity was washed off and the cells were harvested and read on a TopCount plate reader. Statistical analysis was performed using JMP 5.1.2 software (SAS Institute, Inc., Cary, NC, USA); the student's t test was used to determine p values and significance was confirmed with Wilcoxon rank sum and Pearson chi squared analyses.<br><br>BrdU incorporation
BrdU (50 μM) was given to cells 48 hours post-siRNA transfection and permitted to incorporate for 1.5 hours. Cells were then washed with PBS, trypsinized, and washed again. BrdU staining was performed according to an adaptation of the manufacturer's protocol: cells were re-suspended in PBS, mixed 1:1 with -20°C neat ethanol, and incubated 1 hour at -20°C to fix. Fixed cells were then washed gently and denatured in 2 M HCl for 20 minutes at room temperature. Following washing and 2 minute's incubation with 0.1 M Tris to neutralize the acid, cells were re-suspended in FACS buffer (0.5% fetal <span class="yellow">calf</span> serum in PBS) and stained with Phycoerythrin (PE)-conjugated anti-BrdU according to the manufacturer's protocol for flow cytometry analysis on a FACScan instrument.<br><br>Apoptosis and trypan blue staining
Apoptosis was measured using a kit (BD Biosciences) containing propidium iodide (PI) and FITC-conjugated annexin V. Cells were stained according to the manufacturer's protocol and the level of apoptosis determined by flow cytometry. Quadrants are: early apoptosis (annexin V+/PI-, lower right) late apoptosis (annexin V+/PI+, upper right) and non-apoptotic cell death (annexin V-/PI+, upper left). Treatments for the stress panel were: no treatment (control); DMSO as a control for celecoxib; 20 mM celecoxib, brand name Celebrex™ (dissolved in DMSO) [31]; 0.2 mM H2O2 [32]; or 1 mg/mL G418 (Pfizer, New York, NY, USA).<br><br>Real-time PCR arrays
Transcriptional analysis using Cancer PathwayFinder RT2 profiler arrays (SuperArray, Frederick, MD, USA) was performed according to the manufacturer's protocol. Briefly, total RNA was isolated using an RNeasy extraction kit (Qiagen, Valencia, CA, USA); 1 μg of RNA was reverse transcribed with the cDNA synthesis kit (SuperArray) and cDNA was subjected to real-time PCR using SYBR green to detect product. Arrays were performed independently at least twice for each cell line; all PCR products were checked on agarose gels. Values were obtained for the threshold cycle (Ct) for each gene and normalized using the average of four housekeeping genes on the same array (HPRT1, RPL13A, GAPD, ACTB). Ct values for housekeeping genes and a dilution series of ACTB were monitored for consistency between arrays. Change (ΔCt) between MUC1 siRNA and control siRNA was found by:
ΔCt = Ct(MUC1 siRNA) - Ct(control siRNA)
and fold change by:
Fold change = 2(-ΔCt)
Values are given as fold change; only genes showing two-fold or greater change were considered. Both luciferase and scrambled siRNA controls were used in BT-20; only genes showing consistent alteration with both controls were included in the results reported here. The scrambled siRNA could be not used in MDA-MB-468 as these cells decrease MUC1 expression in response to this construct.<br><br>
Results
siRNA transfection decreases MUC1 expression in breast cancer cell lines
Two <span class="blue">human</span> breast cancer cell lines, MDA-MB-468 and BT-20, were transiently transfected with a pool of four siRNA oligonucleotides directed against the MUC1 mRNA (468.siMUC1 and BT.siMUC1), or a control oligonucleotide directed against luciferase (468.siLuc and BT.siLuc). Both cell lines express high levels of MUC1, making them promising targets for this analysis. Western blots (Figure 1a) show successful knockdown of both the extracellular domain and cytoplasmic tail fragments of MUC1; luciferase siRNA does not substantially change the level of MUC1 compared to parental cells. 468.siMUC1 show a substantial decrease in the amount of MUC1-CT, while BT.siMUC1 show slightly less knockdown of MUC1-CT. Both MDA-MB-468 and BT-20 display a less dramatic decrease of MUC1 extracellular domain compared to MUC1-CT (Figure 1a); this likely represents protein synthesized prior to transfection, and may reflect differences in the turnover rates of the two subunits.
Analysis of the MUC1 extracellular domain by flow cytometry confirms that both cell lines substantially decrease MUC1 expression after siRNA (Figure 1b). By flow cytometry, 468.siMUC1 averaged 75% knockdown of MUC1 compared to 468.siLuc; and BT.siMUC1 averaged 50% knockdown relative to BT.siLuc. These effects could be titrated with the concentration of siRNA, were seen as early as 24 hours post-transfection (data not shown) and lasted to at least 96 h post-transfection (Figure 1b). All experiments were conducted within 48 to 96 hours after siRNA transfection. Similar results were obtained using two independent oligonucleotides designed in our lab (data not shown), designated '882' and '956' for the initial codon recognized by each.<br><br>Transcriptional changes are seen after MUC1 siRNA
Recent work indicates that MUC1 may affect transcription both directly via interaction with transcription factors and indirectly (for example, through modulating signaling). To study the effects of MUC1 knockdown in breast cancer cell lines, real-time PCR arrays were used to analyze transcription of 84 genes implicated in cancer. Only genes with greater than two-fold change were considered. Three genes (MAP2K1, VEGF, PDGFA) were altered two-fold or more after MUC1 siRNA in both MDA-MB-468 and BT-20 cells (Figure 2); two genes (ITGAV, MMP2) changed only in 468.siMUC1; and five genes (TIMP3, RAF1, JUN, TNF, CDC25A) only in BT.siMUC1. This list represents all genes affected greater than two-fold after MUC1 siRNA, rather than a select group. Three genes whose transcription was changed by less than two-fold are shown, two of which (PDGFB and ITGB1) are listed because they relate closely to genes altered by two-fold (PDGFA and ITGAV). The third, MYC, is included because western blots confirmed a substantial change at the protein level (Figure 3a) that may reflect both transcriptional and post-transcriptional regulation.
Interestingly, transcription of MAP2K1 was decreased in both cell lines after MUC1 siRNA. This gene encodes MEK1, one of the primary regulators of the ERK1/2 MAPK pathway [33], a network that has been linked several times to MUC1 [12,34-36]. We examined MEK1 and MEK2 levels by western blot to confirm decreased protein in MUC1 siRNA-treated cells (Figure 3a), and found that not only were total MEK1/2 levels lower in 468.siMUC1 and BT.siMUC1 compared to controls (0.48 and 0.68 relative to siLuc, respectively), but so were the basal amounts of active (phosphorylated) MEK1/2 (pMEK1/2; 0.12 and 0.42 relative to siLuc, respectively). Both 468.siMUC1 and BT.siMUC1 also showed reduced activation of ERK1/2 (dpERK1/2; 0.21 and 0.27 relative to siLuc, respectively), as would be expected with diminished signaling through MEK1/2; total ERK1/2 levels remain unchanged.
As both lines have high levels of EGFR and thus activate the MEK-ERK cascade intensely when stimulated with EGF [37], siRNA-transfected cells were treated with EGF. Notably, MUC1 siRNA impairs this important oncogenic pathway in MDA-MB-468 cells, as 468.siMUC1 display less pMEK1/2 in response to EGF than do 468.siLuc (Figure 3b). Interestingly, EGF treatment of BT-20 cells results in slightly higher pMEK1/2 levels in BT.siMUC1 compared to BT.siLuc. Though this result seems paradoxical in light of decreased MAP2K1 transcription in BT.siMUC1, it likely results from differential functions of Raf isoforms in combination with the increased RAF1 transcription (Figure 2) and protein level (Figure 3a) in these cells. Specifically, B-Raf is thought to be the main activator of MEK under normal conditions; Raf-1 activates MEK in response to stimulus [38]. Thus, it appears that basal pMEK1/2 levels are not greatly affected by Raf-1 overexpression in BT.siMUC1 cells, likely because MEK is regulated primarily by B-Raf under normal growth conditions. In contrast, when the cells are stimulated (EGF), increased Raf-1 levels in BT.siMUC1 leads to heightened pMEK1/2 (Figure 3b).<br><br>MUC1 siRNA increases apoptosis in MDA-MB-468 but not BT-20
We next examined whether MUC1 knockdown and its associated transcriptional alterations would affect overall cellular events. As several of the genes shown in Figure 2 are important in regulating proliferation and survival, and because of the recently described role of MUC1 in modulating apoptosis in response to cellular stresses [20,21,24], we first analyzed whether MUC1 siRNA would alter apoptosis in these lines. Although there was no change in basal apoptosis in either line (Figure 4a), we observed that the cell lines responded differently when trypsinized for re-plating 24 hours after transfection (Figure 4b). Interestingly, 468.siMUC1 cells show greater apoptosis after trypsinization than do 468.siLuc (49.8% versus 34.0%, respectively), while BT-20 cells from both siRNA treatments display similar levels of apoptosis (around 22%).
To examine whether this phenomenon is specific to trypsin treatment or part of a general stress response involving MUC1, we subjected cells to a panel of stresses and measured cell death. In agreement with the patterns seen with trypsinization, BT.siLuc and BT.siMUC1 respond similarly to all treatments (data not shown), while 468.siMUC1 die more readily than 468.siLuc in response to trypsin, G418, hydrogen peroxide, or celecoxib, a chemotherapeutic that targets the cyclooxygenase-2 (COX-2) pathway (Figure 4c); these data were confirmed with two independent siRNA constructs (data not shown).
Like the MAPK pathway, AKT signaling has been linked to MUC1 in cancer. Although transcription of AKT was not altered in MUC1 siRNA-treated cells, the results of our apoptosis studies prompted us to investigate levels of AKT further. As expected, the total AKT protein level is not greatly changed after MUC1 siRNA in either cell line, though the active form (pAKT) is increased in both 468.siMUC1 and BT.siMUC1 compared to controls (Figure 3a). This result disagrees with MUC1 activation of the AKT pathway in <span class="yellow">rat</span> 3Y1 cells [18], and may reflect regulation more appropriate to breast cancer cells; this is supported by activation of AKT in response to MUC1 siRNA in other lines [21]. In addition, there is a striking difference in the relative amounts of AKT and pAKT in the two cell lines (Figure 4d). When lysates from both lines are exposed to film for the same length of time (overexposure masks the differences between BT.siLuc and BT.siMUC1 that are apparent in Figure 3a), it is clear that pAKT levels are much higher in BT-20 than in MDA-MB-468, despite lower total AKT expression. This difference in AKT activation between MDA-MB-468 and BT-20 likely contributes to the disparity in their sensitivity to the increased apoptosis expected with loss of MUC1.<br><br>MUC1 siRNA alters proliferation and invasion
As MUC1 is involved in apoptosis, we next analyzed its effects on proliferation. BrdU and [3H]thymidine incorporation were used to analyze proliferation after MUC1 siRNA. 468.siMUC1 cells show a significant decrease in [3H]thymidine incorporation compared to 468.siLuc, while intriguingly, BT.siMUC1 cells show a significant increase in proliferation (Figure 5a). Growth curves mirror these results, as do experiments with the two independent MUC1 siRNA oligonucleotides (data not shown). Note that these assays require trypsinizing cells 24 hours post-transfection; therefore, the results in the MDA-MB-468 line could stem from the changes in apoptosis described in the previous section, rather than a true effect on proliferation. To control for this, we incubated non-trypsinized, siRNA-transfected cells at similar confluence with BrdU to measure incorporation. The 'clumped' profile of cells (contrast to Figure 4b) is likely a result of the acid denaturation (recommended by the antibody manufacturer), as it occurs uniformly in these experiments. BrdU incorporation (Figure 5b) confirms that the [3H]thymidine results are not solely due to alterations in apoptosis, as 468.siMUC1 cells incorporate less BrdU than 468.siLuc; once again, BT.siMUC1 cells show increased proliferation over BT.siLuc.
Given the role of MUC1 in adhesion, we examined whether MUC1 siRNA affects cellular invasion. In transwell assays, BT-20 cells invaded poorly, regardless of the siRNA used (data not shown). However, MDA-MB-468 cells invade more readily, and were analyzed on a panel of three different extracellular matrix proteins. Interestingly, 468.siMUC1 cells display somewhat decreased invasion on collagen IV, laminin, and fibronectin matrices, and on a no-matrix control (Figure 5c), which is in agreement with the trend towards decreased metastasis observed in Muc1-/- × <span class="yellow">MMTV</span>-PyV MT <span class="yellow">mice</span> [8].<br><br>Transfection of MUC1 rescues the 468.siMUC1 phenotype
To determine if the above effects are specific to MUC1, we created stable transfectants of the MDA-MB-468 line using empty vector (468.Neo) or a full-length MUC1 construct (468.MUC1Δ8) that is resistant to one of the independent MUC1-directed oligonucleotides ('882'). These cells were maintained in G418-containing medium to retain transgene selection. As expected, 468.MUC1Δ8 cells show higher levels of both the MUC1 extracellular domain and the MUC1-CT than do 468.Neo (Figure 6a). Note that 468.Neo have MUC1 expression comparable to parental MDA-MB-468; the exposures in Figure 6a are lighter than those in Figure 1a, in order to clearly show the relative levels of MUC1 in the stable transfectants. After MUC1 siRNA, 468.MUC1Δ8 lose some MUC1 (likely endogenous protein, which is not siRNA-resistant) but retain high-level expression, while 468.Neo show a decrease in MUC1 levels similar to parental 468.siMUC1 cells (Figures 6a,b). The difference in the amount of MUC1 knockdown between 468.Neo and 468.MUC1Δ8 is highlighted by the purple shading in Figure 6b.
BrdU incorporation (Figure 6c) indicates that 468.Neo show decreased nucleotide incorporation after MUC1 siRNA compared to control (3.3% versus 25.0%, respectively); this is not seen in 468.MUC1Δ8 cells, which show similar levels of BrdU incorporation regardless of the siRNA used (21.5% for luciferase, 23.9% for MUC1). 468.Neo cells display a more dramatic decrease in BrdU incorporation after MUC1 siRNA than what is seen in parental 468.siMUC1 cells, which may reflect the additional stress of being maintained in G418-containing medium. Similarly, analysis of apoptosis in trypsinized cells indicates that the increased apoptosis seen in parental 468.siMUC1 cells is also present in the 468.Neo line after MUC1 siRNA (Figure 6d; 43.6% in control versus 59.6% in MUC1 siRNA). However, in 468.MUC1Δ8 cells, the level of apoptosis after luciferase siRNA (34.1%) is lower than that in 468.Neo cells; MUC1 siRNA increases the amount of apoptosis slightly (42.8%), restoring it to a level similar to that seen in luciferase siRNA-treated 468.Neo cells. Together, these studies suggest that the above-described results are specific to MUC1, as stable transfection of an siRNA-resistant MUC1 rescues the phenotype seen in 468.siMUC1 cells.<br><br>
Discussion
This report describes both the transcriptional alterations seen after transfection with MUC1 siRNA in <span class="blue">human</span> breast cancer cells and the effects on events such as apoptosis and proliferation. The two cell lines used (MDA-MB-468 and BT-20) were chosen for high expression of MUC1 and a substantial (50% to 75%), consistent decrease in MUC1 expression after siRNA. Both lines have epithelial morphology, form tumors slowly in <span class="yellow">nude mice</span> [37], have mutant p53 [39,40], express EGFR [37], and lack estrogen receptor α [41]. One striking difference between these lines, however, is their response to MUC1 siRNA. MDA-MB-468 cells behave as expected for loss of an oncogene: MUC1 siRNA correlates with increased apoptosis in response to stress, decreased proliferation, and reduced invasion. In contrast, BT.siMUC1 cells proliferate more rapidly than BT.siLuc cells with little effect on apoptosis.
Much of the phenotype of these cells can be understood in light of protein levels and transcriptional activity after MUC1 siRNA. As mentioned, both MDA-MB-468 and BT-20 display increased pAKT after MUC1 siRNA, but the ratio of active to total AKT is considerably higher in the BT-20 line, which may help these cells resist the increased apoptosis expected with loss of MUC1. Myc levels are also higher in both cell lines after MUC1 siRNA, although the ability of Myc to promote proliferation and apoptosis in different cellular contexts [42] complicates the interpretation of this finding.
Both cell lines show reduced transcription of VEGF, PDGFA, PDGFB, and MAP2K1 (MEK1) after MUC1 siRNA. The genes encoding vascular endothelial growth factor (VEGF) and the A and B chains of platelet-derived growth factor (PDGF-A and PDGF-B) are interesting as these proteins have been heavily implicated in angiogenesis, suggesting a novel function for MUC1 in regulating this process. Vascular endothelial growth factor expression in cancer is linked to tumor growth and metastasis [43,44]; platelet-derived growth factor is also angiogenic, but has an additional role in stimulating desmoplasia [45]. Reduced transcription of these genes after MUC1 siRNA suggests that MUC1 may foster angiogenesis and stromal proliferation, although this must be confirmed in a more appropriate model system.
Decreased MAP2K1 (MEK1) transcription after MUC1 siRNA provides a novel mechanism by which MUC1 can affect the ERK1/2 MAPK pathway. MUC1 has often been linked to the Ras-Raf-MEK-ERK cascade [12,34-36,46], and at least two mechanisms by which MUC1 can alter MAPK signaling have been described: MUC1 interaction with and phosphorylation by the EGFR family [12,13], and MUC1 binding to the Grb2/Sos complex that activates Ras [46]. Reduction of MEK1 levels after MUC1 siRNA agrees with the role of MUC1 in strengthening MAPK signaling, and indicates that MUC1 can regulate both the transcription and activity of members of this pathway.
Two additional MAPK pathway members are altered specifically in BT.siMUC1, with no corresponding change in 468.siMUC1 cells. These genes are RAF1 and JUN which are increased and decreased, respectively, after MUC1 siRNA. Raf-1 and c-Jun both function outside of the ERK1/2 MAPK pathway, which may explain the seeming paradox of increased RAF1 transcription with simultaneous decreases in MAP2K1 and JUN. Specifically, Raf-1 can inhibit ASK1 (apoptosis signal-regulated kinase 1) upstream of p38 and JNK (Jun N-terminal kinase) [38]. ASK1 phosphorylates JNK in response to stress, resulting in activation of c-Jun and stimulation of apoptosis [47], indicating that the coordinate up-regulation of RAF1 and down-regulation of JUN may provide a potent anti-apoptotic effect in BT.siMUC1.
Regulation of life and death is also a hallmark of the CDC25A and TNF gene products. CDC25A is a phosphatase that stimulates cell cycle progression [48], thus the effects of its decrease in BT.siMUC1 are unclear in light of the increased proliferation of these cells. However, the CDC25 proteins (A, B, and C) were recently shown to have greater functional overlap than was previously thought [49], suggesting that the other two isoforms may compensate for reduced CDC25A levels. TNF encodes tumor necrosis factor (TNF)α, known for its potent, cell type-specific control of life and death. In tumor cells, TNFα expression can promote proliferation and inhibit apoptosis [50], suggesting that increased TNF transcription in BT.siMUC1 could contribute to the increased proliferation seen in these cells.
Interestingly, the increase in TNF is accompanied by decreased transcription of TIMP3, encoding tissue inhibitor of metalloproteinases (TIMP)3. The TIMP family disrupts the function of matrix metalloproteinases (MMPs), generally resulting in decreased invasion [51]. TIMP3 is unique in that it can also inhibit TNFα converting enzyme (TACE), which activates TNFα by cleaving it from the cell surface [50]. Reduced expression of TIMP3 would, therefore, foster signaling through TNFα by releasing inhibition of TNFα converting enzyme. In agreement with this, TIMP3 can promote apoptosis [52]; thus, its down-regulation in BT.siMUC1 provides another mechanism by which these cells are able to resist the increased apoptosis expected with loss of MUC1.
Another TIMP target responds to MUC1 siRNA, as 468.siMUC1 cells show significantly increased expression of MMP2 (encoding MMP-2/gelatinase A), the product of which degrades type IV collagen [52]. In breast cancer, the ratio of active to latent MMP-2 increases with tumor progression; MMP-2 may facilitate both angiogenesis and metastasis [52]. Its increase after loss of MUC1 is, therefore, unexpected, but at least two factors may clarify this result. First, though MMP-2 levels are increased in <span class="yellow">mouse</span> mammary tumors, its expression is confined to the stroma [53], suggesting that increased MMP2 transcription after loss of the epithelium-specific MUC1 might reflect a shift towards a more mesenchymal phenotype. Second, MMP-2 levels are increased by overexpression of erbB2 [52]; previous studies have shown that erbB2 and Muc1 expression are mutually exclusive in mammary tumors [27], implying that MMP2 might be part of a transcriptional profile linked to low MUC1 levels.
It is intriguing that, despite increased MMP2 transcription, invasion is decreased in 468.siMUC1 cells, even on collagen IV. This may reflect insufficient activation of MMP-2, as the precursor protein must be cleaved for enzymatic function [52]. Alternatively, the slowed invasion of these cells may relate to impaired adhesion resulting from decreased transcription of ITGAV and ITGB1 (αv and β1 integrins, respectively). Integrin signaling is tied to life-or-death decisions in epithelial cells, and integrin expression is vital for processes from wound healing to metastasis [54]. Integrin αvβ3 is implicated in facilitating metastasis of breast cancer cells to bone [55]; decreased transcription of ITGAV after MUC1 siRNA may, therefore, suggest that MUC1 is involved in this lethal process as well.
The MUC1 oncogene has been linked to apoptosis [18,20,26], proliferation [17], and transcription [21,23-25] in cancer. However, the two cell lines chosen for our study display very different responses to MUC1 siRNA, indicating that regulation of MUC1 in breast cancer is likely quite complex and cautioning against over-generalization of results from individual cell lines. Previous reports suggest that, though most studies outline a clearly oncogenic role for MUC1 in breast cancer, the exact details may vary depending on factors such as cell type and signaling context. For example, MUC1 stimulates Fas-mediated apoptosis in CHO cells [26], quite unlike the inhibition of apoptosis seen in other cell lines. Similarly, though MUC1 drives mammary oncogenesis in its own right [9] and facilitates tumorigenesis driven by other oncogenes [7,8], Muc1 is selectively down-regulated in c-neu-induced <span class="yellow">mouse</span> mammary tumors [27], indicating that the context of oncogenic signaling is vital to understanding the function of MUC1.
Thus, it is important to consider the relative levels of knockdown of MUC1 in the two cell lines: BT-20 cells reduce MUC1 expression after siRNA less strongly than do MDA-MB-468 (50% versus 75% knockdown, respectively). As MUC1 serves as a scaffold [11], overexpression of MUC1 relative to its associated signaling proteins might create a dilution effect, sequestering signal transducers away from each other; this would be relieved by MUC1 siRNA. Thus, enough MUC1 may be retained in BT.siMUC1 cells for its oncogenic effects, while signaling complex formation would be enhanced by lowering the amount of MUC1 relative to other signaling proteins.<br><br>Conclusion
The contrast between the MDA-MB-468 and BT-20 lines in response to MUC1 siRNA serves as a reminder that simplified models such as cell lines fail to encompass the complexity of intact biological systems. This report describes transcriptional alterations seen after MUC1 knockdown: decreased transcription of MAP2K1, VEGF, PDGFA, ITGAV, TIMP3, CDC25A, and JUN, and increased transcription of MMP2, TNF, and RAF1. The alterations in MAP2K1, RAF1, and JUN represent a novel means by which MUC1 can affect ERK1/2 signaling: transcriptional regulation of MAPK pathway members. Oncogenic events are also altered in both cell lines after MUC1 siRNA. These results strengthen the growing ties linking MUC1 and transcriptional regulation, and suggest that the role of MUC1 in breast cancer may be more complex than a direct correlation between MUC1 level and oncogenic function.<br><br>Abbreviations
BrdU = bromodeoxyuridine; EGF = epidermal growth factor; EGFR = epidermal growth factor receptor; ERK = extracellular signal regulated kinase; MAPK = mitogen activated protein kinase; MEK = MAPK and ERK kinase; MMP = matrix metalloproteinases; MUC1-CT = MUC1 cytoplasmic tail; MUC1-EX = MUC1 extracellular subunit; PBS = phosphate-buffered saline; PI = propidium iodide; siRNA = small interfering RNA; TIMP = tissue inhibitor of metalloproteinases; TNF = tumor necrosis factor.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
CLH performed all studies and composed the manuscript. SJG participated in the design and coordination of the studies and contributed strongly to the revision of the manuscript. Both authors have read and approved the manuscript.<br><br>
<h3>pmcA1885664</h3>The recombination-associated protein RdgC adopts a novel toroidal architecture for DNA binding
Abstract
RecA plays a central role in the nonmutagenic repair of stalled replication forks in bacteria. RdgC, a recombination-associated DNA-binding protein, is a potential negative regulator of RecA function. Here, we have determined the crystal structure of RdgC from <span class="yellow">Pseudomonas aeruginosa</span>. The J-shaped monomer has a unique fold and can be divided into three structural domains: tip domain, center domain and base domain. Two such monomers dimerize to form a ring-shaped molecule of approximate 2-fold symmetry. Of the two inter-subunit interfaces within the dimer, one interface (‘interface A’) between tip/center domains is more nonpolar than the other (‘interface B’) between base domains. The structure allows us to propose that the RdgC dimer binds dsDNA through the central hole of ∼30 Å diameter. The proposed model is supported by our DNA-binding assays coupled with mutagenesis, which indicate that the conserved positively charged residues on the protein surface around the central hole play important roles in DNA binding. The novel ring-shaped architecture of the RdgC dimer has significant implications for its role in homologous recombination.<br><br>INTRODUCTION
Homologous recombination systems contribute to the maintenance of genome integrity and are essential for the repair of stalled replication forks (1). Genetic studies of the recombination-dependent growth C (rdgC) gene in bacterial systems have suggested that it is involved in DNA replication and recombination (2,3). In <span class="yellow">Neisseria gonorrhoeae</span> (the gonococcus), RdgC is required for efficient pilin antigenic variation and plays some role in cell growth (4,5). The <span class="yellow">Escherichia coli</span> rdgC gene encodes a DNA-binding protein of 34 kDa, whose expression level was at its highest during exponential phase, reaching its maximum at ∼1000 dimers per cell (2). Its level decreased sharply to ∼50 dimers per cell in stationary phase. This profile suggests that RdgC might function during the period of DNA replication (2).
The RecA proteins and their homologs play a key role in recombinational DNA repair in bacteria and eukaryotes. Their aberrant reactions could result in gross chromosomal rearrangements that lead to <span class="blue">human</span> diseases, including cancer. Therefore, their functions must be tightly regulated, and understanding how the RecA family of recombinases is regulated is of utmost importance (1). The <span class="yellow">E. coli</span> RdgC protein is a potential negative regulator of the function of RecA (1). It inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). Sedimentation equilibrium data indicated that <span class="yellow">E. coli</span> RdgC exists in solution as a mixture of oligomeric states in equilibrium, most likely monomers, dimers and tetramers (1). This concentration-dependent change in the oligomeric state appears to affect its mode of binding to DNA and its capacity to inhibit RecA (1). The primary mechanism of RdgC inhibition appears to involve a simple competition for DNA-binding sites, especially on double-stranded DNA (dsDNA) (1).
Deletion of the <span class="yellow">E. coli</span> rdgC gene alone causes no obvious phenotype but is highly deleterious in strains lacking certain enzymes involved in recombination and replication restart (2,3). RdgC is essential for the growth of an <span class="yellow">E. coli</span> strain lacking PriA, indicating that it might affect replication fork progression or fork rescue (2). PriA provides a means to load the DnaB replicative helicase at DNA replication fork and D loop structures (2). RdgC is, therefore, a key factor in the rescue of stalled or broken forks and subsequent replication restart. dnaC suppressors of PriA can overcome this inviability, especially when RecF, RecO or RecR is inactivated, which indicates that RdgC avoids or counters the toxic effect of these proteins by limiting inappropriate RecA loading on SSB-coated ssDNA (2).
The DNA-binding ability of the RdgC protein has been demonstrated clearly using electrophoretic mobility shift assays (2). <span class="yellow">Escherichia coli</span> RdgC binds non-specifically to dsDNA and with higher affinity than ssDNA (2). RdgC from <span class="yellow">N. meningitidis</span> also binds DNA with little specificity for sequence or structure, like the <span class="yellow">E. coli</span> protein (4). RdgC exhibits no preference for DNA replication or recombination intermediates (2). However, no detectable domains or nuclease activity could be identified for the <span class="yellow">E. coli</span> RdgC protein (2,6). The complexes of <span class="yellow">E. coli</span> RdgC with both linear and supercoiled circular plasmid DNA were imaged using atomic force microscopy (7). RdgC was found to have an increased affinity to DNA ends and to promote bending of DNA (7). RdgC has the effect on DNA superstructure; the promotion of DNA condensation was observed at high protein concentrations (7). Recombination is largely enhanced by close contacts of distant regions along the DNA strands through condensation (7).
<span class="yellow">Pseudomonas aeruginosa</span> is a ubiquitous environmental Gram-negative bacterium that belongs to the γ subdivision of the Proteobacteria. Orthologs of the rdgC gene are found only in β and γ subdivisions of the Proteobacteria (2). The amino acid sequence of <span class="yellow">P. aeruginosa</span> RdgC, a 306-residue protein, shows 46 and 36% identities against those of <span class="yellow">E. coli</span> and <span class="yellow">N. meningitidis</span>, respectively. Despite the importance of DNA-binding ability of RdgC in its biological roles, the structural details of RdgC and the mechanism of its action remain unclear. Here, we report the crystal structure of <span class="yellow">P. aeruginosa</span> RdgC as determined by the multiwavelength anomalous diffraction (MAD) method of X-ray crystallography. It reveals that RdgC dimer is ring-shaped, with a central hole of ∼30 Å diameter. The inside surface around the central hole is rich in conserved, positively charged residues, which are implicated in DNA-binding by mutagenesis. We propose that dsDNA binds to RdgC through this central hole. Our structure provides a solid structural framework for a better understanding of the role of RdgC in homologous recombination.<br><br>MATERIALS AND METHODS
Protein expression and purification
The rdgC gene (PA3263) was amplified by the polymerase chain reaction (PCR) using the genomic DNA of <span class="yellow">P. aeruginosa</span> strain PAO1 as a template. The oligonucleotide primers designed using the published genome sequence (8) were 5′-G GAA TTC CAT ATG TGG TTC CGC AAT CTG CTC G-3′ (forward) and 5′-CCG CCG CTC GAG GAC GCC CTG GGG GAT TTC TTC-3′ (reverse). The bases in bold denote the NdeI and XhoI cleavage sites, respectively. The PCR product was digested with NdeI and XhoI, and was then inserted into the NdeI/XhoI-digested expression vector pET-21a(+) (Novagen). This vector construction adds an eight-residue tag (LEHHHHHH) to the carboxyl terminus of the gene product to facilitate protein purification.
The protein was expressed in <span class="yellow">E. coli</span> C41 (DE3) cells (9). The cells were grown at 37°C up to OD600 of 0.6 in Luria–Bertani medium that contained 50 μg ml−1 ampicillin, and the protein expression was induced by 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG). The cells were allowed to grow at 20°C for 22 h after IPTG induction and were harvested by centrifugation at 4200 g (6000 rev min−1; Sorvall GSA rotor) for 10 min at 4°C. The cell pellet was resuspended in an ice-cold lysis buffer [20 mM Tris-HCl (pH 7.9), 500 mM sodium chloride, 5 mM imidazole] and was then homogenized with an ultrasonic processor. The crude cell extract was centrifuged at 36 000 g (18 000 rev min−1; Hanil Supra 21 K rotor) for 60 min at 4°C, and the recombinant protein in the supernatant fraction was purified in three chromatographic steps.
The first purification step utilized the C-terminal hexahistidine-tag by Ni2+-chelated HiTrap chelating column (GE Healthcare). The eluent was diluted 3-fold with 50 mM Tris-HCl at pH 7.4. The diluted solution was applied to a 20-ml heparin-Sepharose column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol and 1 mM EDTA. The protein was eluted with a linear gradient of 0–2.0 M sodium chloride in the same buffer. The first peak corresponded to high molecular aggregates of RdgC and the second peak to RdgC dimers, which was also consistent with our dynamic light scattering measurements. When the aggregates were not separated from the dimers, the protein readily aggregated within a day. The dimer fractions were concentrated to ∼6 mg ml−1 concentration using an YM10 ultrafiltration membrane (Millipore-Amicon) and further purified by gel filtration on a HiLoad XK 16 Superdex 200 prep-grade column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 200 mM sodium chloride, 1 mM dithiothreitol and 1 mM EDTA. The first peak corresponding to the high molecular aggregates of RdgC was only a minor component, and the second peak corresponding to RdgC dimers was stable for several weeks when frozen at −70°C. The procedure for purifying the SeMet-substituted protein was the same, except for the presence of 10 mM dithiothreitol in all buffers used during purification steps. Dynamic light scattering experiments were performed at 24°C, with the protein (at 2 mg ml−1 concentration) dissolved in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride on a Model DynaPro-801 instrument (Wyatt, Santa Barbara, CA, USA). The protein concentration was estimated by measuring the absorbance at 280 nm, employing the calculated extinction coefficient of 20 910 M−1 cm−1 (SWISS-PROT; http://www.expasy.ch/).<br><br>Mutagenesis and electrophoretic mobility shift assay
Point mutants were prepared using the QuikChange Site-Directed Mutagenesis kit (Stratagene), and the mutations were confirmed by DNA sequencing. The seven mutants R4A, F120A, K146A, R198A, R208A, Q212A and R252A were well expressed and soluble. The three mutants K70A, R81A and R211A were not expressed, while R122A mutant was expressed but was not soluble. Therefore, we mutated Lys70, Arg81, Arg122 and Arg211 into aspartate. The K70D, R81D, R122D and R211D mutants were well expressed and soluble. The soluble mutants were expressed and purified under the conditions identical to those for the wild type, except that the heparin-Sepharose column step was omitted. All of the soluble mutants had similar elution profiles as the wild type upon gel filtration, and the dimer fractions were used for DNA-binding assays. We mixed 5 μl of RdgC (1.1 μg μl−1) and 5 μl of a linear dsDNA with 414 bp (63 ng μl−1), corresponding to the molar ratio of 64:1 for RdgC dimer to dsDNA. The reaction mixtures were incubated for 10 min at room temperature. RdgC–DNA complexes were analyzed by 1.5% (w/v) agarose gel electrophoresis in 1× TAE buffer (45 mM Tris-acetate at pH 8.3, 1 mM EDTA). DNA bands were visualized by ethidium bromide staining. For DNA-binding assays, the protein concentration was measured with a Bio-Rad protein assay kit.<br><br>Crystallization and X-ray data collection
Crystals were grown by the hanging-drop vapor diffusion method at 24°C by mixing equal volumes (2 μl each) of the protein solution (at 19 mg ml−1 concentration in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride) and the reservoir solution. To grow the native crystals, we used a reservoir solution consisting of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate and 1% (w/v) Anapoe®35 as the detergent. Native crystals grew to approximate dimensions of 0.2 mm × 0.1 mm × 0.1 mm within a few days. The SeMet-substituted protein was crystallized under conditions identical to those for the native crystals except for the presence of 10 mM dithiothreitol in the protein solution.
A crystal of the SeMet-substituted protein was dipped into a cryoprotectant solution for a few seconds and was flash-cooled in the cold nitrogen gas stream at 100 K. The cryoprotectant solution consisted of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate, 1% (w/v) Anapoe®35 as the detergent and 10% (v/v) glycerol. A set of Se MAD data was collected at 100 K at three different wavelengths using an Area Detector System Corporation Quantum 210 charge-coupled device detector at the beamline NW12A of Photon Factory, Tsukuba, Japan. The crystal was rotated through a total of 180° with a 1° oscillation range per frame. The raw data were processed and scaled using the program suit HKL2000 (10). The SeMet-substituted crystal belongs to the space group P21212, with unit cell parameters of a = 87.04 Å, b = 115.60 Å and c = 76.07 Å.
The native crystals were soaked for 10 min in 10 mM ethyl mercury thiosalicylate (EMTS) to collect a set of Hg MAD data at four different wavelengths. X-ray diffraction data from the EMTS-derivatized crystal, as well as a native crystal, were collected on a Bruker charge-coupled device detector at the beamline BL-6B of Pohang Light Source, Pohang, Korea. The EMTS-soaked crystal belongs to the space group P21212, with unit cell parameters of a = 87.16 Å, b = 116.02 Å and c = 74.97 Å. The native crystal belongs to the space group P21212, with unit cell parameters of a = 87.49 Å, b = 116.04 Å and c = 76.03 Å. If the presence of a dimeric molecule in the asymmetric unit is assumed, the calculated crystal volume per protein mass (VM) is 2.75 Å3 Da−1 and the solvent content is 55.3%. Supplementary Table 1 summarizes the statistics of X-ray diffraction data collection.<br><br>Structure solution and refinement
Heavy atom sites were located with the program SOLVE (11). Four Se sites from the SeMet-substituted crystal and two Hg sites from the EMTS-soaked crystal were located. Two sets of the initial Se and Hg MAD phases were separately improved using the program RESOLVE (12). Two electron density maps were independently interpreted, and the fragments of the two models were combined into a single model, since the two maps were complementary to each other. Non-crystallographic symmetry (NCS) matrices were found from a partial model built from the initial electron density map, and phases were further improved by the 2-fold NCS averaging, solvent flattening and histogram matching with the program DM (13). The model was built with O (14). The model was refined with the program CNS (15), including the bulk solvent correction. 10% of the data were randomly set aside as the test data for the calculation of Rfree (16). Several rounds of model building, simulated annealing, positional refinement and individual B-factor refinement were performed. Subsequently, this model was used to refine the structure of the native protein. The model has excellent stereochemistry, as evaluated by the program PROCHECK (17). Refinement statistics are summarized in Supplementary Table 1.<br><br>
RESULTS AND DISCUSSION
Model quality and monomer structure
The structure of <span class="yellow">P. aeruginosa</span> RdgC was solved using two sets of MAD data collected from a crystal of the selenomethionine (SeMet)-substituted protein and from a mercury derivative crystal of the native protein (Supplementary Table 1). The model has subsequently been refined using the 20.0–2.50 Å data from a native crystal to crystallographic Rwork and Rfree values of 23.4 and 27.9%, respectively, with no sigma cut-off. The refined model contains 612 residues of the two monomers in the asymmetric unit of the crystal (residues 1–306 for both chains A and B) and 87 water molecules. The C-terminal eight-residue fusion tag has no electron density in both subunits and has not been modeled. Conformations of the two monomers in the asymmetric unit are similar. The root-mean-square (r.m.s.) difference between the two monomers is 0.69 Å for 306 Cα atom pairs, with a maximum deviation of 8.8 Å for the Cα atom of Gly305. The residues giving r.m.s. differences larger than 2.0 Å are 14–16, 94–96, 201–202 and 303–306, which correspond to loops connecting secondary structure elements or the C-terminus. They are located on the surface of the dimer. Subunit A has a better map quality than subunit B, and thus has a lower mean B-factor than subunit B (40.3 Å2 versus. 56.1 Å2 for all 4788 atoms) (Supplementary Figure 1). Therefore, subunit A is chosen for discussions, unless otherwise stated.
The <span class="yellow">P. aeruginosa</span> RdgC monomer is J-shaped and has approximate dimensions of 72 Å × 60 Å × 40 Å (Figure 1). It can be divided into three structural domains: center domain, tip domain and base domain. Center domain (residues 1–73 and 121–167) includes a six-stranded anti-parallel β-sheet with two flanking α helices (α1 and α4). The helix α1 runs nearly parallel to the two outermost strands β2 and β3, which in turn lie adjacent to β4. Strands β2 and β3 are separated by a loop. The helix α4 is nearly orthogonal to the β-strands. Tip domain (residues 74–120) contains two α helices (α2 and α3) and is inserted between strands β4 and β5 of the center domain. It sticks out from the center domain. The two connections between the center domain and the tip domain are rich in conserved residues, hinting at an important functional role of these joints. They could act as a hinge for a possible conformational change. The C-terminal region of the polypeptide chain is folded into the base domain (residues 168–306), which consists of a five-stranded anti-parallel β-sheet with four α-helices, three of which (α5, α6 and α8) cover one side of the β-sheet. The long helix α8 is slightly bent.<br><br>
Structural similarity searches
When we searched for structural similarity against the Protein Data Bank database using the program DALI (18), the <span class="yellow">P. aeruginosa</span> RdgC monomer showed no significant similarity with a Z score above 5. Therefore, we conclude that the overall fold of the RdgC polypeptide chain is unique. When we elaborated the structural comparisons with individual domains of <span class="yellow">P. aeruginosa</span> RdgC, only the center domain gave a Z score above 5.
Using the center domain (residues 1–73 and 121–167) alone, the highest structural similarity is found with the <span class="blue">human</span> β2-adaptin appendage domain (PDB code 1E42, Z score = 7.1, r.m.s. deviation = 2.5 Å for 85 structurally aligned residues, residues 1–12, 14–30, 44–47, 58–72, 122–133 and 135–160 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 838–841, 848–880, 885–906 and 912–967 of the <span class="blue">human</span> β2-adaptin appendage domain chain A) (19). There does not appear to be any functional relatedness between these two proteins. The next highest similarity is found with the <span class="yellow">yeast</span> TATA-box-binding protein (PDB code 1YTB, Z score = 5.7, r.m.s. deviation = 2.6 Å for 79 structurally aligned residues, residues 4–12, 14–19, 21–27, 56–70, 124–132 and 134–166 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 69–96, 100–126 and 132–155 of TATA-box-binding protein, N-terminal half of chain A) (20). The TATA-box-binding protein is composed of a ten-stranded antiparallel β-sheet with four flanking α-helices on the convex side of the β-sheet (20). It has an internal quasi 2-fold symmetry, and its β-strands are involved in DNA binding. There is, however, no structural similarity of functional significance, because the two center domains of the RdgC dimer are well separated and do not associate into a single unit (Figure 1C and D).
Using the tip domain (residues 74–120) alone, the ATPase domain of <span class="yellow">bovine</span> Hsc70 chaperone shows highest structural similarity (PDB code 1BA1, Z score = 4.8, r.m.s. deviation = 1.7 Å for 45 structurally aligned residues, residues 76–120 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 230–253 and 256–276 of the Hsc70 ATPase domain) (21). There does not seem to be any significant functional relationship between these two proteins. The α-helix-loop-α-helix structure of the tip domain is reminiscent of the helix–hairpin–helix (HhH) DNA-binding motif that is present in a number of DNA repair enzymes (22–25). The HhH motifs are characterized by the conserved sequence motif hxxhxGhGxxxAxxxhh, where h is any hydrophobic residue (VILMWFYA) and x any residue (23). Two conserved glycines allow a tight turn between two α-helices. However, the tip domain of <span class="yellow">P. aeruginosa</span> RdgC protein does not have such a sequence motif, and the loop in the RdgC tip domain is much more extended than that of the HhH motifs. Therefore, the RdgC tip domain is uniquely folded.
With the base domain (residues 168–306) alone, the highest similarity is found with <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase (unpublished deposition; PDB code 2H9C, Z score = 4.0, r.m.s. deviation = 4.8 Å for 51 structurally aligned residues, residues 186–191, 237–240, 257–272 and 274–298 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 36–41, 50–53, 55–66 and 68–96 of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase chain A). There does not appear to be any functional relationship between the two proteins, and the base domain of RdgC is uniquely folded. The 101-residue chain of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase is folded into three α-helices, two of which overlap with two α-helices (α2 and α3) of the base domain of RdgC.<br><br>Ring-shaped dimer structure and inter-subunit interface
Our structure reveals that <span class="yellow">P. aeruginosa</span> RdgC forms a ring-shaped dimer of approximate 2-fold symmetry in the crystal (Figure 1). This observation is consistent with the results of previous gel filtration and sedimentation equilibrium experiments, which suggested the presence of dimeric species of <span class="yellow">E. coli</span> RdgC in solution (1,2). Depending on the concentration, RdgC could also exist in solution as monomers, tetramers and higher oligomers (1). In the crystal lattice, no monomeric, trimeric or tetrameric species exist. Our structure unequivocally rules out the monomer–trimer model, which fits the sedimentation equilibrium data approximately, as well as the monomer–dimer–tetramer model (1). The RdgC dimer has approximate dimensions of 80 Å × 60 Å × 40 Å, with a central hole of ∼30 Å diameter.
Only a moderate amount of surface area is buried upon the formation of the RdgC dimer. The monomer–monomer interface buries a solvent-accessible surface area of ∼1360 Å2 per monomer, corresponding to ∼8% of the total surface area of the monomer (Protein–Protein Interaction Server at http://www.biochem.ucl.ac.uk/bsm/PP/server/). Polar and nonpolar atoms in interface are 44 and 56%, respectively. The interface can be divided into two kinds: ‘interface A’ involving both the tip domains and center domains, and ‘interface B’ involving the base domains (Figure 2A). These two interfaces have significantly different characters.<br><br>The interface A is formed largely by the crossing over of the two tip domains in the dimer (Figure 1). This interface covers a surface area of ∼565 Å2 and is largely nonpolar, with the polar and nonpolar atoms in the interface being 37.5 and 2.5%, respectively. Phe120 contributes considerably to the nonpolar character of the interface between the tip domains, accounting for ∼150 Å2 of the interface area. Phe120 is part of a highly conserved sequence segment and lies at the boundary between the tip domain and the center domain. The aromatic ring of Phe120′ inserts into a hydrophobic pocket, which is formed by the hydrophobic residues Ile74, Leu75, Pro76, Val79, Leu115, Ala119 and Phe120 of the neighboring monomer (Figure 2B). The primed residue belongs to the adjacent subunit. These residues are well conserved in the RdgC family (indicated by green triangles below the aligned sequences in Figure 3). As these hydrophobic residues are located near the molecular 2-fold axis, the two clusters of hydrophobic side chains are merged into a single, large cluster in the dimer (Figure 2B). Of the 16 hydrogen bonds that exist between the two subunits, only one is made in this interface between Arg118 and Arg118′.<br><br>The interface B is contributed to exclusively by the two base domains. This interface covers a surface area of ∼795 Å2. The polar and nonpolar atoms in this interface are 50% each. The two β-sheets of the two base domains merge into a single ten-stranded antiparallel β-sheet in this interface (Figure 2A). Thus, 15 of 16 inter-subunit hydrogen bonds are clustered in this interface. The hydrogen bond networks can be divided into two types. The first type involves the main chain atoms of Val206, Arg208 and Lys210 in strand β9; this strand associates with the equivalent strand from the other subunit in an antiparallel manner (Figure 2C). The other type involves both the side chain and main chain atoms. The side chain of Arg211′ interacts with the backbone of Gly204, while the side chain of Lys227 hydrogen bonds to those of both Gln212′ and Glu218 (Figure 2D). In addition, the side chain of Gln212′ interacts with that of Asp199 and the backbone of Arg211′ (Figure 2D). Arg211, Gln212, Glu218 and Lys227 are highly conserved among the RdgC family members. The residues participating in the hydrogen bond networks are indicated by red circles below the aligned sequences in Figure 3. The observation that conserved residues play important roles in forming the dimer implies that dimerization of RdgC is critical for its function.
If one or both of interfaces A and B are to be broken for functional reasons such as for allowing dsDNA to enter into the central hole of the RdgC dimer, we can conceive of three scenarios. The first possibility would be a simultaneous breakage of the two interfaces A and B. The second would be the sequential disruption of the interfaces A and B. The third would be the disruption of only one of the two interfaces A and B. Sedimentation equilibrium indicated the presence of monomers in solution (1), which is consistent with the first two scenarios. If the third scenario holds, RdgC dimers could possibly exist in two conformations in solution, i.e. a closed form as observed in the present crystal structure, and an open form, in which one of the two interfaces A and B is broken.<br><br>A model for DNA binding by RdgC and DNA-binding assay
The net charge of <span class="yellow">P. aeruginosa</span> RdgC is highly negative, since each monomer contains 12 aspartate, 13 glutamate, 10 lysine, 9 arginine and 2 histidine residues. The net charge of a dimer would be −12 (or −8), if we assume none (or all) of histidines to be positively charged. The electrostatic potential at the molecular surface of RdgC dimer shows that the charge distribution on the protein surface is highly nonuniform (Figure 4A). Since the inner surface of the RdgC dimer is rich in conserved, positively charged residues and the diameter of the central hole (∼30 Å) is approximately similar to that of other toroidal DNA-binding proteins (26), it appears likely that RdgC dimers bind dsDNA through the central hole. Despite extensive co-crystallization and soaking trials with linear dsDNAs of different lengths (14-mer, 16-mer and 18-mer), we could not locate the bound DNA experimentally. We attribute this difficulty to the nature of sequence-nonspecific DNA binding by RdgC. Therefore, we built a crude model of a DNA complex of the RdgC dimer by fitting dsDNA into the central hole (Figure 4B). This model is supported by the DNA-binding assays coupled with mutagenesis, as described below. It is also consistent with the suggested binding site size of ∼16 nt for the <span class="yellow">E. coli</span> RdgC dimer (1) and the approximate thickness (∼40 Å) of the <span class="yellow">P. aeruginosa</span> RdgC dimer.<br><br>When we performed electrophoretic mobility shift assay for dsDNA binding, <span class="yellow">P. aeruginosa</span> RdgC did not require Mg2+ ions (Supplementary Figure 2), unlike <span class="yellow">Deinococcus radiodurans</span> RecR, which required high concentrations of Mg2+ ions (27). ATP was not required for dsDNA binding by <span class="yellow">P. aeruginosa</span> RdgC, either (Supplementary Figure 2). This is consistent with an apparent lack of an ATP-binding pocket in the RdgC dimer structure.
In order to identify the residues that are important for dsDNA binding, we carried out mutational analyses. The residues for mutagenesis were selected on the basis of the modeled complex between RdgC dimer and dsDNA, together with sequence conservation (Figure 4C). We prepared eleven point mutants (R4A, K70D, R81D, F120A, R122D, K146A, R198A, R208A, R211D, Q212A and R252A). These eleven mutated residues are well conserved among RdgC proteins. Eight mutants (R4A, K70D, R81D, R122D, K146A, R198A, R208A and R252A) had a lower affinity for dsDNA than the wild-type RdgC (Figure 4D). These eight mutated residues are indicated by blue squares below the aligned sequences in Figure 3. This result indicates that dsDNA binds to the inside surface around the central hole of the RdgC dimer, since the mutated residues are located on the inner surface of the RdgC dimer. It is worth mentioning that a single mutation introduces two alterations per RdgC dimer.
The three mutants (F120A, R211D and Q212A) were prepared to investigate whether dimerization of RdgC is crucial for dsDNA binding. Phe120 is a key residue located in the inter-subunit interface A (Figure 2B), while Arg211 makes hydrogen bonds in the interface B (Figure 2D) and Gln212 is involved in both intra- and inter-subunit interactions (Figure 2D). dsDNA binding by the F120A mutant is considerably weakened, compared to either the R211D or Q212A mutant (Figure 4D). This result suggests that destabilization of the ring-shaped architecture of the RdgC dimer is deleterious for tight dsDNA binding and that interface A is probably more important than interface B for dsDNA binding. dsDNA-binding affinity of the Q212A mutant was lower than the wild type (Figure 4D), whereas that of the R211D mutant was comparable to the wild type (Figure 4D). This result suggests that disruption of the inter-subunit interface B is also detrimental for dsDNA binding but to a smaller extent than disruption of the interface A. It further suggests that the side chain of Arg211 is not involved in recognizing dsDNA. It is interesting that the DNA complex for wild type forms a distinct band (lane 3 in Figure 4D), as opposed to a more diffuse band as one would expect if the complexes contained different numbers of RdgC dimers. Assuming a binding site size of ∼16 nt (1), the 414-bp duplex DNA would bind ∼25 RdgC dimers if it is fully saturated.<br><br>RdgC is unique among ring-shaped DNA-binding proteins
<span class="yellow">Pseudomonas aeruginosa</span> RdgC adds a unique member to the repertoire of toroidal DNA-binding proteins (26) such as <span class="yellow">D. radiodurans</span> RecR (27) and DNA polymerase processivity factors, including the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29) and PCNA of eukaryotic DNA polymerase δ (30,31) (Figure 5). These proteins do not have an overall sequence similarity among themselves, and their quaternary structures also vary. <span class="yellow">P. aeruginosa</span> RdgC and <span class="yellow">E. coli</span> β clamp are homodimers, T4 gp45 and eukaryotic PCNAs are homotrimers and <span class="yellow">D. radiodurans</span> RecR is a homotetramer.<br><br>In <span class="yellow">P. aeruginosa</span> RdgC, the inner surface of the ring-shaped dimer is largely formed by β-strands, while the outer surface of the ring is mostly formed by α-helices (Figure 5). The predicted DNA-binding site of <span class="yellow">P. aeruginosa</span> RdgC dimer is primarily formed by β-strands and loops. In contrast, in other ring-shaped DNA-binding proteins such as the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29), PCNA of eukaryotic DNA polymerases δ (30,31) and <span class="yellow">D. radiodurans</span> RecR (27), α-helices are located toward the center of the ring and are surrounded by β-strands in the outermost side of the ring (Figure 5). Despite a superficial resemblance of the overall ring-shaped architecture of the RdgC dimer with those of other toroidal DNA-binding proteins, the internal arrangement of the secondary structure elements in the RdgC dimer is strikingly different from those of other toroidal DNA-binding proteins. Thus, RdgC is unique among ring-shaped DNA-binding proteins.
A number of other proteins involved in DNA metabolism also adopt a ring-shaped structure despite diversity in the molecular functions (26). They include NAD+-dependent DNA ligase (25), λ exonuclease (32), hexameric helicases, topoisomerases, the trp RNA-binding attenuation protein, the <span class="yellow">bacteriophage</span> head-to-tail connector and translin (26). There are also many examples of the proteins in DNA recombination pathways that assemble into multi-subunit toroids containing a central channel (26). Electron microscopy showed that the <span class="yellow">E. coli</span> RuvB protein, in the presence of ATP, forms a dodecamer on double-stranded DNA in which two stacked hexameric rings encircle the DNA and are oriented in opposite directions with D6 symmetry (33). The <span class="blue">human</span> RAD52 protein was found to assemble into heptameric rings with a large funnel-shaped channel, 40–60 Å in diameter (34). The <span class="blue">human</span> DMC1 protein has so far been detected only as an octameric ring (35).<br><br>Inhibition mechanism of RecA function by RdgC
RdgC inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). It was observed that the effects of RdgC on RecA-promoted DNA strand exchange were much more pronounced than the effects of RdgC on other RecA functions, when RdgC was added to the reaction late (1). By examining the effects of the DinI protein on RdgC-mediated inhibition of RecA activities, it was concluded that the potent effect of RdgC in the inhibition of strand exchange did not primarily reflect a displacement of RecA in the nucleoprotein filament by RdgC, but instead a binding of RdgC to the dsDNA substrate so as to make it unavailable to the filaments for strand exchange (1).
Our work contributes to a better understanding of the inhibition mechanism of RecA function by RdgC. The ring-shaped architecture with a central hole that is suitable for the binding of dsDNA lends RdgC with a capacity to hold dsDNA very tightly. Therefore, we suggest that RdgC dimers probably function as a tight dsDNA gripper and physically block access to the target dsDNA by RecA filaments, because tight encircling of dsDNA by RdgC dimers, possibly as a linear aggregate, would prevent the two strands from dissociating. However, it remains unclear how the closed-ring structure of the RdgC dimer might allow binding of a large number of RdgC dimers on the duplex DNA. It may be conceivable that each RdgC dimer can open temporarily at one or both of its two inter-subunit interfaces so that dsDNA can enter the central hole.
RdgC was at its highest level during exponential phase, reaching its maximum of ∼1000 dimers per <span class="yellow">E. coli</span> cell. Its level decreased sharply to ∼50 dimers per cell in stationary phase (2). Under normal cellular conditions, RecA is maintained at ∼1000 molecules per cell (http://www.els.net/). Following LexA repressor cleavage, the level of RecA protein in the cell increases by as much as 20-fold. RdgC inhibition of the RecA function by a simple competition mechanism at the cellular concentration of RdgC clearly requires a much higher affinity for DNA by RdgC than RecA. Possible protein–protein interactions between RdgC dimers might have a profound effect on the affinity for DNA, when a large number of RdgC dimers are bound on a duplex DNA. If only one of the two inter-subunit interfaces can open up and the 2-fold symmetry axes of the RdgC dimers are misaligned sufficiently, the escape of DNA from a linear array of interacting RdgC dimers would be extremely difficult. Alternatively, both inter-subunit interfaces of each dimer may open up simultaneously. A line of indirect evidence for the possible protein–protein interactions between RdgC dimers is provided by the cooperative binding of RdgC to DNA (7). Cooperativity of RdgC binding may be caused by local and distant protein–protein interactions (7). A kind of side-by-side interaction between two adjacent molecules of RdgC dimer is present in the crystal, burying an accessible surface area of ∼750 Å2 at the interface. The interaction involves mostly the less polar side surface of the dimer (left side of Figure 4A), with 2-fold symmetry axes of the two interacting RdgC molecules making an angle of ∼90° when viewed down the central holes and ∼15° when viewed perpendicular to the holes. This may explain the observed DNA bending by RdgC, which was speculated to be a necessary step in the working mechanism of RdgC, if one can assume that the strong protein–protein interactions in the crystal were relevant for protein–protein interactions in solution (7).
RdgC protein was shown to have a potent effect on RecA protein-promoted DNA strand exchange in <span class="yellow">E. coli</span> (1). In a series of experiments, RdgC was added after RecA filaments had formed on the ssDNA and 5 min after the linear dsDNA was added. With 3 μM RecA, a sharp reduction in DNA strand exchange products was seen with 0.4 μM RdgC, and the generation of products was abolished at 0.8 μM RdgC (1). This suggests that the binding affinity of RdgC to dsDNA is considerably higher than that of RecA. The high affinity of RdgC could be explained by its ring shape and the suggested mode of DNA binding. Tetrameric species of <span class="yellow">E. coli</span> RdgC, which started to form at around 2 µM concentrations, were speculated to be the species with the most potent inhibitory effect, possibly by having a higher affinity for DNA (1). Higher affinity for DNA of the tetrameric species could be understood, if the formation of a tetramer in solution involves a protein–protein interaction similar to the side-by-side interaction between two adjacent RdgC dimers in the crystal, although there is no experimental evidence. Obviously, there are still many open questions that need to be addressed in future biochemical experiments.<br><br>
SUMMARY
The crystal structure of RdgC, a recombination-associated DNA-binding protein, determined in this study reveals that the polypeptide chain is uniquely folded and two monomers associate to form a ring-shaped dimer with a central hole of ∼30 Å diameter. The inner surface around the central hole is rich in positively charged residues. These structural features suggest that dsDNA runs through the central hole when it binds to RdgC dimers. This idea is supported by our mutational studies, which indicate that the conserved, positively charged residues around the central hole are important for dsDNA binding. The toroidal architecture of the RdgC dimer provides a sound framework for a better understanding of its role in homologous recombination.
Coordinates
Atomic coordinates and the structure factor data have been deposited in the Protein Data Bank (accession code 2OWY).<br><br>
SUPPLEMENTARY DATA
Supplementary Data are available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA2435603</h3>Potentiation of Nerve Growth Factor-Induced Neurite Outgrowth by Fluvoxamine: Role of Sigma-1 Receptors, IP3 Receptors and Cellular Signaling Pathways
Abstract
Background
Selective serotonin reuptake inhibitors (SSRIs) have been widely used and are a major therapeutic advance in psychopharmacology. However, their pharmacology is quite heterogeneous. The SSRI fluvoxamine, with sigma-1 receptor agonism, is shown to potentiate nerve-growth factor (NGF)-induced neurite outgrowth in PC 12 cells. However, the precise cellular and molecular mechanisms underlying potentiation by fluvoxamine are not fully understood. In this study, we examined the roles of cellular signaling pathways in the potentiation of NGF-induced neurite outgrowth by fluvoxamine and sigma-1 receptor agonists.<br><br>Methods and Findings
The effects of three SSRIs (fluvoxamine, sertraline, paroxetine) and three sigma-1 receptor agonists (SA4503, 4-phenyl-1-(4-phenylbutyl) piperidine (PPBP), and dehydroepiandrosterone (DHEA)-sulfate) on NGF-induced neurite outgrowth in PC12 cells were examined. Also examined were the effects of the sigma-1 receptor antagonist NE-100, inositol 1,4,5-triphosphate (IP3) receptor antagonist, and specific inhibitors of signaling pathways in the potentiation of NGF-induced neurite outgrowth by selective sigma-1 receptor agonist SA4503. Fluvoxamine (but not sertraline or paroxetine) and the sigma-1 receptor agonists SA4503, PPBP, and DHEA-sulfate significantly potentiated NGF-induced neurite outgrowth in PC12 cells in a concentration-dependent manner. The potentiation by fluvoxamine and the three sigma-1 receptor agonists was blocked by co-administration of the selective sigma-1 receptor antagonist NE-100, suggesting that sigma-1 receptors play a role in blocking the enhancement of NGF-induced neurite outgrowth. Moreover, the potentiation by SA4503 was blocked by co-administration of the IP3 receptor antagonist xestospongin C. In addition, the specific inhibitors of phospholipase C (PLC-γ), phosphatidylinositol 3-kinase (PI3K), p38MAPK, c-Jun N-terminal kinase (JNK), and the Ras/Raf/mitogen-activated protein kinase (MAPK) signaling pathways blocked the potentiation of NGF-induced neurite outgrowth by SA4503.<br><br>Conclusion
These findings suggest that stimulation of sigma-1 receptors and subsequent interaction with IP3 receptors, PLC-γ, PI3K, p38MAPK, JNK, and the Ras/Raf/MAPK signaling pathways are involved in the mechanisms of action of sigma-1 receptor agonists such as fluvoxamine and SA4503.<br><br><br><br>Introduction
Selective serotonin (5-HT; 5-hydroxytryptamine) reuptake inhibitors (SSRIs) have emerged as a major therapeutic advance in psychopharmacology. SSRIs are the treatment of choice for many indications, including major depressive disorder, dysthymia, panic disorder, obsessive-compulsive disorder, eating disorders, and premenstrual dysphoric disorder. In contrast, it is well known that their pharmacology is quite heterogeneous, although all of them block 5-HT transporters, thus increasing 5-HT levels throughout the central nervous system (CNS) [1]–[9].
Accumulating evidence suggests that sigma-1 receptors, which are intracellular endoplasmic reticulum (ER) proteins, are involved in both the neuroplasticity and pathophysiology of neuropsychiatric diseases such as major depressive disorder, anxiety, schizophrenia, and Alzheimer's disease [10]–[18]. Previously, we reported that some SSRIs possess high to moderate affinities for sigma-1 receptors in the <span class="yellow">rat</span> brain. The rank order of SSRIs affinities for sigma-1 receptors is fluvoxamine (Ki = 36 nM)>sertraline (Ki = 57 nM)>>paroxetine (Ki = 1893 nM) [19]. Recently, we reported that fluvoxamine, but not paroxetine, significantly ameliorated cognitive deficits in <span class="yellow">mice</span> after repeated phencyclidine administration, and that the effects of fluvoxamine were antagonized by co-administration of the selective sigma-1 receptor antagonist NE-100 [20], suggesting that sigma-1 receptors are involved in the mechanism of action of fluvoxamine [9]. Interestingly, it has been demonstrated that sigma-1 receptor agonists including fluvoxamine could potentiate nerve growth factor (NGF)-induced neurite outgrowth in PC12 cells, and that NE-100 blocked the potentiation by sigma-1 receptor agonists, suggesting sigma-1 receptors are involved in neuroplasticity [21]. However, the precise cellular mechanisms underlying the potentiation by sigma-1 receptor agonists are not fully understood [13], [21].
It is therefore of great interest to study the precise cellular mechanisms underlying the enhancement by fluvoxamine on NGF-induced neurite sprouting in PC12 cells. In the present study, we examined the effects of three SSRIs (fluvoxamine, sertraline, paroxetine), as well as the effects of a sigma-1 receptor agonist (4-phenyl-1-(4-phenylbutyl) piperidine (PPBP), dehydroepiandrosterone-sulphate (DHEA)-sulfate) [9], [22]–[28] and the selective sigma-1 receptor agonist SA4503 [29], [30], on NGF-induced neurite outgrowth in PC12 cells. Furthermore, it is also known that sigma-1 receptors have been shown to interact with IP3 receptors (17,18). Therefore, we examined the effects of NE-100 and xestospongin C (a selective inositol 1,4,5-triphosphate (IP3) receptor antagonist) [31] in order to investigate the roles of sigma-1 receptors and IP3 receptors in the mechanisms underlying the enhancement of NGF-induced neurite outgrowth by SA4503. Moreover, we examined the effects of specific inhibitors of several cellular signaling targets on the enhancement of NGF-induced neurite outgrowth by SA4503, since several signal transduction molecules have been implicated in NGF-induced neurite outgrowth [32].<br><br>Materials and Methods
Drugs
The drugs were obtained from the following sources: fluvoxamine maleate (Solvay Seiyaku K.K., Tokyo, Japan); paroxetine hydrochloride, dehydroepiandosterone-sulfate (DHEA-sulfate), LY294002 (Sigma-Aldrich, St Louis, MO, USA); sertraline (Toronto Research Chemicals Inc., North York, ON, Canada); SA4503 (M's Science Corporation, Kobe, Japan); NGF (Promega, Madison, WI); xestospongin C, lovastatin, PD98059, GW5074, SB203580, MEK 1/2 inhibitor (SL327), and SP600125 (Calbiochem-Novabiochem, San Diego, CA). The selective sigma-1 receptor antagonists NE-100 and 4-phenyl-1-(4-phenylbutyl) piperidine (PPBP) were synthesized in our laboratory. Other drugs were purchased from commercial sources.<br><br>Cell culture
PC12 sells (RIKEN Cell Bank, Tsukuba, Japan) were cultured at 37°C, 5% CO2 with Dulbecco's modified Eagle's medium (DMEM) supplemented with 5% heat-inactivated fetal <span class="yellow">bovine</span> serum (FBS), 10% heat-inactivated <span class="yellow">horse</span> serum, and 1% penicillin. The medium was changed two or three times a week. PC12 cells were plated onto 24-well tissue culture plates coated with poly-D-lysine/laminin. Cells were plated at relatively low density (0.25×104 cells/cm2) in DMEM medium containing 0.5% FBS, 1% penicillin streptomycin. Medium containing a minimal level of serum (0.5% FBS) was used, since serum is known to contain steroid hormones that bind to sigma-1 receptors [21]. First, we examined the optimal concentration of NGF for NGF-induced neurite outgrowth in PC12 cells. NGF (2.5, 5, 10, 20, 40 ng/ml) increased the number of cells with neurite outgrowth in PC12 cells in a concentration-dependent manner (Figure 1). In the subsequent studies, 2.5 ng/ml of NGF was used to study the potentiating effects of sigma-1 receptor agonists on NGF-induced neurite outgrowth. Twenty-four hours after plating, the medium was replaced with DMEM medium containing 0.5% FBS and 1% penicillin streptomycin with NGF (2.5 ng/ml) with or without several drugs.<br><br>Quantification of neurite sprouting
Five days after incubation with NGF (2.5 ng/ml) with or without the several drugs, morphometric analysis was performed on digitized images of live cells taken under phase-contrast illumination with an inverted microscope linked to a camera. Images of three fields per well were taken, with an average of 100 cells per field. Differentiated cells were counted by visual examination of the field; only cells that had at least one neurite with a length equal to the cell body diameter were counted, and were then expressed as a percentage of the total cells in the field. The counting was performed in a blinded manner.<br><br>Immunocytochemistry
Cells were fixed for 30 min at room temperature with 4% paraformaldehyde then permeabilized with 0.2% Triton and blocked with 1.5% normal <span class="yellow">goat</span> serum, 0.1% <span class="yellow">bovine</span> serum albumin (BSA) in 0.1 M phosphate-buffer saline for 1 h to reduce nonspecific binding. Cells were incubated overnight at 4°C with anti-microtubule-associated protein 2 (MAP-2) antibodies (1∶1000 dilution in blocking solution, Chemicon International, Temecula, CA, USA). The immunolabeling was visualized with secondary antibodies conjugated to Alexa-488 (1∶1000; Invitrogen, Carlsbad, CA, USA). MAP-2 immuncytochemistry was visualized with a fluorescence microscope (Axiovert 200, Carl Zeiss, Oberkocken, Germany).<br><br>Statistical analysis
Data are expressed as means±S.E.M. Statistical analysis was performed by using one-way analysis of variance (ANOVA) and the post hoc Bonferroni/Dunn test. P values less than 0.05 were considered statistically significant.<br><br>
Results
Effects of SSRIs on NGF-induced neurite outgrowth
Fluvoxamine (0.1, 1.0, or 10 µM) significantly increased the number of cells with neurite outgrowth by NGF (2.5 ng/ml) in PC12 cells, in a concentration-dependent manner (Figure 2 and 4). In contrast, sertraline (0.1, 1.0, or 10 µM) and paroxetine (0.1, 1.0, or 10 µM) did not increase the number of cells with neurite outgrowth by NGF (2.5 ng/ml). A higher concentration (10 µM) of paroxetine and sertraline significantly decreased the number of cells with NGF-induced neurite outgrowth, suggesting the cellular toxicity of these drugs in PC12 cells (Figure 2).<br><br>Role of sigma-1 receptors in the potentiation of NGF-induced neurite outgrowth by fluvoxamine and SA4503
The selective and potent sigma-1 receptor agonist SA4503 (0.01, 0.1, or 1.0 µM) significantly increased the number of cells with neurite outgrowth by NGF (2.5 ng/ml) in PC12 cells, in a concentration-dependent manner (Figure 3 and 4). In addition, other sigma-1 receptor agonists–PPBP (0.1, 1.0, or 10 µM) [22]–[26] and DHEA-sulphate (0.1, 1.0, or 10 µM) [9], [27], [28]–significantly increased the number of such cells, also in a concentration-dependent manner (Figure 3). In the absence of NGF, sigma-1 receptor agonists did not produce neurite outgrowth in PC12 cells (data not shown).
To investigate the role of sigma-1 receptors, we examined the effects of NE-100 (a selective sigma-1 receptor antagonist) on the potentiation of NGF-induced neurite outgrowth by fluvoxamine, SA4503, PPBP, and DHEA-sulphate. Co-administration of NE-100 (1.0 µM) significantly blocked such potentiation by fluvoxamine (10 µM), SA4503 (1.0 µM), PPBP (10 µM), or DHEA-sulphate (10 µM) (Figure 2 and 3). Furthermore, administration of NE-100 (1.0 µM) alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 2).<br><br>Role of IP3 receptors in the potentiation of NGF-induced neurite outgrowth by SA4503
Sigma-1 receptors have been shown to interact with IP3 receptors (17,18,33–35). To investigate the role of IP3 receptors in the effects of SA4503 on NGF-induced neurite outgrowth, we examined the effects of xestospongin C (a selective, reversible, and membrane-permeable inhibitor of IP3 receptors) [31] on the effects of SA4503 on NGF-induced neurite outgrowth. Co-administration of xestospongin C (1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 5). Furthermore, administration of xestospongin C (1.0 µM) alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 5).<br><br>Role of signaling molecules proximal to TrkA in the potentiation of NGF-induced neurite outgrowth by SA4503
Next, we examined the effects of the specific inhibitors of PLC-γ, PI3K, p38 MAPK, and c-Jun N-terminal kinase (JNK), since these signaling molecules are activated upon the addition of NGF [36]. The PLC-γ inhibitor (U73122; 1.0 µM), PI3K inhibitor (LY294002; 1.0 µM), p38 MAPK inhibitor (SB203580; 1.0 µM), and JNK inhibitor (SP600125; 1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 6). In contrast, these inhibitors alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 6).<br><br>Role of Raf/Ras/ERK/MAPK pathway in the potentiation of NGF-induced neurite outgrowth by SA4503
The Raf/Ras/ERK/MAPK pathway is known to be involved in NGF-induced outgrowth [32]. Therefore, we examined the effects of this pathway's specific inhibitors. The Ras inhibitor (GW5074; 1.0 µM), Raf inhibitor (lovastatin; 1.0 µM), MEK1/2 inhibitor (SL327; 1.0 µM), and MAPK inhibitor (PD98059; 1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 7). In contrast, these inhibitors alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 7).<br><br>
Discussion
In the present study, we found that fluvoxamine (but not sertraline or paroxetine) and the sigma-1 receptor agonists (SA4503, PPBP, DHEA-sulfate) could potentiate NGF-induced neurite outgrowth in PC12 cells, and that the effects of these drugs were blocked by co-incubation with the selective sigma-1 receptor antagonist NE-100. These findings suggest that agonism at sigma-1 receptors for these drugs is involved in the mechanisms underlying the drugs' potentiation of NGF-induced neurite outgrowth.
As shown in Figure 1, NGF increased the number of cells with neurite outgrowth in PC12, in a concentration dependent manner. To examine whether or not the NGF levels are altered by incubation with sigma-1 receptor agonists, we measured the levels of NGF in PC12 cells with or without sigma-1 receptor agonist SA4503. No differences of NGF levels were shown in the between control group and SA4503-treated group (data not shown). It is, therefore, unlikely that the potentaition of NGF-induced neurite outgrowth by sigma-1 receptor agonists might be due to increased levels of NGF.
Unlike fluvoxamine, sertraline, which has a moderate affinity for sigma-1 receptors, did not alter NGF-induced neurite outgrowth. The reasons underlying this discrepancy between these two SSRIs are currently unclear. One possibility may involve the difference in pharmacological actions (agonist vs. antagonist) between these SSRIs at sigma-1 receptors. Interestingly, we recently found that, in a novel object recognition test, phencyclidine-induced cognitive deficits could be significantly improved by subsequent subchronic (14 days) administration of fluvoxamine, but not of sertraline, suggesting that sigma-1 receptor agonism is involved in fluvoxamine's mechanism of action [9], Ishima et al., submitted. Taken together, these findings suggest that fluvoxamine and sertraline may function as an agonist and an antagonist at sigma-1 receptors, respectively, although further study is necessary. Another possibility may be that other pharmacological activities of sertraline mask the effects of sigma-1 receptor agonism. In this study, we also found that high concentration (10 µM) of paroxetine and sertraline, but not fluvoxamine, showed cytotoxicity in PC12 cells, suggesting that fluvoxamine may be a safe drug than paroxetine and sertraline.
Sigma-1 receptors have been shown to affect intracellular Ca2+ signaling, although the precise molecular and cellular mechanisms underlying this effect are unknown. Sigma-1 receptors bind to IP3 receptors in ER, and sigma-1 receptors regulate Ca2+ release from intracellular Ca2+ storage sites [12]. Very recently, Hayashi and Su [17] reported that sigma-1 receptors function as novel ligand-operated chaperones that specifically target mitochondrion-associated ER membrane. Furthermore, sigma-1 receptors form Ca2+-sensitive chaperone machinery with another chaperone, BiP, and prolong Ca2+ signaling from ER into mitochondria by stabilizing IP3 receptors at mitochondrion-associated ER membrane [17]. In this study, we found that the IP3 receptor antagonist xestospongin C significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503, suggesting the role of IP3 receptors on sigma-1 receptor-mediated potentiation of NGF-induced neurite outgrowth. Therefore, it is likely that stimulation at sigma-1 receptors by sigma-1 receptor agonists and subsequent interaction with IP3 receptors are involved in the mechanism underlying the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists.
NGF binds to the high-affinity tyrosine receptor TrkA, initiating several signaling pathways affecting both morphological and transcriptional targets [32], [36], [37]. The signaling molecules, including PLC-γ, PI3K, p38 MAPK, and JNK, are activated upon the addition of NGF [38]. PLC-γ catalyzes the hydrolysis of phosphatidylinositol-4,5-bisphosphate (PIP2) to diacylglycerol (DAG) and inositol triphosphate (IP3). DAG activates protein kinase C, and IP3 promotes transient release of Ca2+ from the ER [39]. The pathway via PLC-γ is responsible for NGF-induced cell differentiation [40] and neurite outgrowth [41]. Furthermore, stimulation of PI3K is reported to be involved in the promotion of neurite outgrowth in PC12 cells [42]. In this study, we found that the PLC-γ inhibitor U73122 and the PI3K inhibitor LY294002 significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503. Moreover, we found that both the p38MAPK inhibitor SB203580 and the JNK inhibitor SP600125 significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503. In addition, it is also interesting that neurite outgrowth induced by low concentration (2.5 ng/ml) of NGF was not blocked by these inihibitors, consistent with a previous report [43], [44], suggesting the existence of novel pathway(s) for NGF-induced neurite outgrowth. These findings suggest that the PLC-γ, PI3K, p38MAPK, and JNK signaling pathways are involved in the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists (Figure 8). In addition, we found that the specific inhibitors for the Raf/Ras/MEK/MAPK pathways significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503, suggesting that these pathways are involved in the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists (Figure 8).
Our recent positron emission tomography (PET) study demonstrated that, after a single oral administration, fluvoxamine bound to sigma-1 receptors in the living <span class="blue">human</span> brain [45]. This finding suggests that sigma-1 receptors are involved in the mechanism of action of fluvoxamine in the <span class="blue">human</span> brain [45]. Taken together, the past and present findings suggest that, unlike paroxetine and sertraline, the SSRI fluvoxamine, with its sigma-1 receptor agonistic activity, might be a unique therapeutic drug for neuropsychiatric diseases.
In conclusion, the present results suggest that, as a sigma-1 receptor agonist, fluvoxamine could potentiate the NGF-induced neurite outgrowth in PC12 cells. Furthermore, it is likely that interaction with IP3 receptors and several subsequent signaling molecules are involved in the mechanism underlying the pharmacological action of sigma-1 receptor agonists. Therefore, it is likely that sigma-1 receptor agonists such as fluvoxamine and DHEA-sulfate, which are available around the world, would be unique therapeutic drugs for neuropsychiatric diseases.<br><br>
<h3>pmcA2518116</h3>CHEK2 Mutations Affecting Kinase Activity Together With Mutations in TP53 Indicate a Functional Pathway Associated with Resistance to Epirubicin in Primary Breast Cancer
Abstract
Background
Chemoresistance is the main obstacle to cure in most malignant diseases. Anthracyclines are among the main drugs used for breast cancer therapy and in many other malignant conditions. Single parameter analysis or global gene expression profiles have failed to identify mechanisms causing in vivo resistance to anthracyclines. While we previously found TP53 mutations in the L2/L3 domains to be associated with drug resistance, some tumors harboring wild-type TP53 were also therapy resistant. The aim of this study was; 1) To explore alterations in the TP53 gene with respect to resistance to a regular dose epirubicin regimen (90 mg/m2 every 3 week) in <span class="yellow">patients</span> with primary, locally advanced breast cancer; 2) Identify critical mechanisms activating p53 in response to DNA damage in breast cancer; 3) Evaluate in vitro function of Chk2 and p14 proteins corresponding to identified mutations in the CHEK2 and p14(ARF) genes; and 4) Explore potential CHEK2 or p14(ARF) germline mutations with respect to family cancer incidence.<br><br>Methods and Findings
Snap-frozen biopsies from 109 <span class="yellow">patients</span> collected prior to epirubicin (as preoperative therapy were investigated for TP53, CHEK2 and p14(ARF) mutations by sequencing the coding region and p14(ARF) promoter methylations. TP53 mutastions were associated with chemoresistance, defined as progressive disease on therapy (p = 0.0358; p = 0.0136 for mutations affecting p53 loop domains L2/L3). Germline CHEK2 mutations (n = 3) were associated with therapy resistance (p = 0.0226). Combined, mutations affecting either CHEK2 or TP53 strongly predicted therapy resistance (p = 0.0101; TP53 mutations restricted to the L2/L3 domains: p = 0.0032). Two <span class="yellow">patients</span> progressing on therapy harbored the CHEK2 mutation, Arg95Ter, completely abrogating Chk2 protein dimerization and kinase activity. One <span class="yellow">patient</span> (Epi132) revealed family cancer occurrence resembling families harboring CHEK2 mutations in general, the other <span class="yellow">patient</span> (epi203) was non-conclusive. No mutation or promoter hypermethylation in p14(ARF) were detected.<br><br>Conclusion
This study is the first reporting an association between CHEK2 mutations and therapy resistance in <span class="blue">human</span> cancers and to document mutations in two genes acting direct up/down-stream to each other to cause therapy failure, emphasizing the need to investigate functional cascades in future studies.<br><br><br><br>Introduction
Chemoresistance is the main obstacle to cure in most malignancies, including breast cancer. While adjuvant chemotherapy may reduce the hazard rate of relapse by about one third in breast cancer <span class="yellow">patients</span> [1], the majority among <span class="yellow">patients</span> harboring micro- metastases are not cured by today's standards. Considering <span class="yellow">patients</span> harboring distant metastases, resistance and therapy failure inevitably occurs, in general over a time period of less than one year for each individual regimen [2].
Despite extensive experimental research [3], little data are available considering chemoresistance in vivo. For anthracycline therapy in breast cancer, topoisomerase-II amplifications have been associated with a dose-responsiveness different from what is observed in non-amplified tumors 4, 5. Several studies have tried to generate “prediction profiles” based on gene expression microarrays [6], [7], [8], however, none of the different profiles generated expressed a sensitivity suitable for clinical applications, or have been successfully reproduced by others (see references to original works in [9] and [10]).
p53 (the protein encoded by the TP53 gene) plays a key role in executing DNA-damage induced apoptosis and growth arrest [11]. Previously, our group reported mutations in the zink-binding domains L2 (codons 163–195) and L3 (codons 236–251) of p53 critical to DNA binding [12] to be associated with but not fully predictive for resistance to chemotherapy with a low-dose weekly anthracycline [13] or a mitomycin plus 5-fluoro-uracil containing [14] regimen. Similar findings were reported by another group [15]. In contrast, others reported TP53 mutations to predict sensitivity to a dose-dense epirubicin-cyclophosphamide regimen [16].
The finding that some tumors harboring wild-type TP53 may be resistant to anthracycline therapy lead us to postulate that other genes involved in the p53 pathway could be mutated in these tumors [3]. p53 is activated by post-translational modifications, and the protein is phosphorylated at multiple amino acids [17]. Phosphorylation at Ser 20 (Ser 23 in <span class="yellow">mice</span>) by the Chk2 protein (coded by the CHEK2 gene) in response to DNA damage activates p53 by inhibiting binding to, and deactivation by, the MDM2 (<span class="yellow">Mouse</span> Minute 2 homolog; HDM2) protein [18], [19], [20]. While experimental studies have suggested a critical role of Chk2 in activating p53 apoptotic response to genotoxic stress [21], [22], others claim Chk2 to be dispensable for p53 activation with respect to apoptosis as well as growth arrest [23]. Following an initial report of a CHEK2 germline mutation in a family filling the characteristics of a Li-Fraumeni syndrome (LFS) [24], recent papers have suggested germline mutations in CHEK2 to be associated with a moderately increased risk of breast and colon cancers (see references in [25]). Recently, we discovered a somatic, nonsense CHEK2 mutation in a single <span class="yellow">patient</span> expressing resistance to doxorubicin low dose therapy [26].
A second mechanism of p53 activation is through p14(ARF) (p19 in <span class="yellow">mice</span>) function. p14(ARF) does not phosphorylate p53, but inhibits MDM2 dependent p53 degradation through direct MDM2 binding. While p14(ARF)-mediated p53 activation has been linked to oncogene-induced p53 activation and, in general, considered not involved in response to DNA damage (see references in [27]), p14(ARF) may be activated through the E2F1/retinoblastoma pathway [28]. Importantly, two recent studies revealed lack of p19 (<span class="yellow">mouse</span> homologue of <span class="blue">human</span> p14(ARF)) function in <span class="yellow">mice</span> to inhibit p53 tumor suppressor function in response to ionizing radiation as well as DNA damaging agents [29], [30].
The aim of this study was 1) to explore alterations in the TP53 gene with respect to resistance to a regular dose epirubicin regimen (90 mg/m2 body surface every 3 week) in <span class="yellow">patient</span> with primary, locally advanced, breast cancer; 2) To explore defects in potential mechanisms activating p53 in response to DNA damage in breast cancer as a cause of drug resistance in wild-type tumors. To do so, we sequenced the complete coding regions for the CHEK2 and p14(ARF) genes and analyzed for p14(ARF) promoter hypermetylations; 3) Evaluate in vitro function of potential Chk2 and p14(ARF) protein translates corresponding to identified mutations in the CHEK2 and p14(ARF) genes; 4) Identify potential TP53, CHEK2 and p14(ARF) mutations to be germline, explore the incidence of different cancers among affected relatives with respect to specific mutations. By comparing in vitro characteristics of specific mutations to drug sensitivity and family cancer risk syndromes, this may add to our understanding of the importance of these gene cascades executing response to DNA damage versus tumor suppression activity.
Analyzing tumor samples from a total of 109 primary locally advanced breast cancer <span class="yellow">patients</span> treated with epirubicin 90mg/3 weekly, we found TP53 mutations affecting the L2/L3 domains or protein dimerization, as well as non-functional CHEK2 mutations abrogating dimerization and phosphorylation, to be associated with therapy resistance; no mutation or promoter hypermethylations of the p14(ARF) gene was discovered. Our findings suggest a critical role for Chk2 with respect to DNA-damage-dependent p53 activation and resistance to anthracycline therapy in <span class="blue">human</span> breast cancer.<br><br>Materials and Methods
<span class="yellow">Patients</span>
A total of 223 <span class="yellow">patients</span> with locally advanced non-inflammatory breast cancer (T3-4 and/or N2) were randomly allocated to primary treatment either with epirubicin 90 mg/m2 or paclitaxel 200 mg/m2. The primary aim of the study was identification of markers predicting drug resistance to the regimens. Thus, the reason for randomizing <span class="yellow">patients</span> was not for effect comparison, but to achieve similar <span class="yellow">patient</span> cohorts in the two arms. Based on the findings of a clinical lack of cross-resistance between anthracyclines and taxane therapies in breast cancer [31], we hypothesized the mechanisms of resistance to be different between the two compounds. While the analysis of tumor samples from the paclitaxel is ongoing, we here report our findings from the <span class="yellow">patients</span> allocated to the epirubicin arm.
The epirubicin arm included a total of 109 <span class="yellow">patients</span> (age 28 to 70 years, median 51 years). Two <span class="yellow">patients</span> were analyzed for gene mutations but omitted from statistical analysis as protocol violators; histopathological examination revealed one <span class="yellow">patient</span> (Epi089) to harbor a sarcomatoid tumor, while one <span class="yellow">patient</span> Epi232 was erroneously enrolled with stage II disease.
The study protocol was approved by the Regional Ethical Committee (Norwegian Health Region III), including formal Biobank registration in accordance to Norwegian law. The study and protocol is registered under the Norwegian Social Science Data services ((www.nsd/uib/personvern/database/), University of Bergen project no 16297 and Helse Bergen project no 13025). Each <span class="yellow">patient</span> gave written informed consent.<br><br>Tissue Sampling
Before commencing chemotherapy, each <span class="yellow">patient</span> had an incisional tumor biopsy as described previously [14]. All tissue samples were snap-frozen immediately on removal in the theatre.<br><br>Treatment Regime and Staging
Primary treatment consisted of epirubicin (90 mg/m2) administered as a 3-weekly schedule. Treatment was scheduled for four cycles unless progression occurred at an earlier stage. Clinical response was assessed before each treatment cycle, and the final response evaluated 3 weeks after the 4th cycle for overall response classification. Because the protocol was implemented by October 1997 with <span class="yellow">patients</span> enrolled between November 1997 and December 2003, responses were consistently graded by the UICC system [32] and not the more recently implemented “RECIST” criteria [33]. Thus, responses were classified as CR (Complete Response, complete disappearance of all tumor lesions), PR (Partial Response, reduction ≥50% in the sum of all tumor lesions, calculated for each as the product of the largest diameter and the one perpendicular to it), PD (Progressive Disease, increase in the diameter product of any individual tumor lesion by ≥25%), and SD (Stable Disease, anything between PR and PD). To analyze for the predictive value of the different parameters, similar to our previous studies [13], [14] we compared PD tumors (non responders) with the combined group of tumors classified as SD/PR/CR (responders); the reason for this approach is discussed in detail elsewhere [34]. Median follow-up time was defined from <span class="yellow">patient</span> inclusion in the study up to October 31, 2006. Deaths attributable to causes other than breast cancer were treated as censored observations.
All <span class="yellow">patient</span> records were subject to central audit for response classification (by E.L., B.Ø. and P.E.L.). Response classifications were completed and approved without any knowledge about result from laboratory analysis.<br><br>RNA Purification
Total RNA was purified by Trizol (Life Technologies, Inc.) extraction from snap-frozen tissue samples according to manufacturer's instructions. After extraction, the RNA was dissolved in 100 µl of DEPC treated ddH2O. cDNA was synthesized by reverse transcription using Transcriptor reverse transcriptase (Roche), according to the manufacturer's protocol.<br><br>DNA Purification
Genomic DNA from tumor biopsies and blood lymphocytes was isolated using QIAamp DNA Mini kit (Qiagen, Chatsworth, CA) according to the manufacturer's protocol.<br><br>Mutation Analysis
All mutational analysis was performed blinded to clinical data. Mutations in TP53, CHEK2 and p14(ARF) genes were analyzed by PCR (or nested PCR) amplification and sequencing of PCR product, or by cloning of PCR products and sequencing of the resulting plasmids (all primers described in Table 1). Cloning was performed using the TOPO TA Cloning kit (Invitrogen). Sequencing of clones was performed until at least 10 different sequences covered all parts of the CHEK2 coding sequence. DNA sequencing was carried out directly on 1 µl PCR product or plasmid using Big Dye terminator mix (Applied Biosystems). Capillary gel electrophoresis, data collection, and sequence analysis were done on an automated DNA sequencer (ABI 3700). When a mutation was detected, the relevant exon was amplified by PCR from genomic tumor DNA and DNA from blood lymphocytes and sequenced for verification and germline detection. (Primers described in Table 1).<br><br>Loss of Heterozygosity (LOH)
Loss of heterozygosity (LOH) in tumors with mutations in CHEK2 was assessed using the microsatellite marker, D22S275, which maps to intron 4 of CHEK2. LOH in tumors with mutation in TP53 was assessed using two markers, one variable number tandem repeat in intron 1 [35] and a CA repeat close to the TP53 gene [36]. Fluorescently end-labeled primers were used in the PCR, and the PCR products were analyzed on an ABI 3700. LOH was evaluated by comparing the allele peak-height ratios from blood DNA and tumor DNA. A sample was scored as having AI (Allelic Imbalance) when a reduction in peak height of one allele in tumor sample was at least 18% compared with that of blood DNA from the same <span class="yellow">patient</span> [37].<br><br>Analysis of p14(ARF) promoter methylation
Genomic DNA was subjected to bisulphate conversion using the CpGenome DNA Modification Kit (Intergen) according to the manufacturer's protocol. Both the unmethylated- and methylated-specific PCRs were performed in 50 µl reaction mixes containing 2.5 U AmpliTaq Gold DNA Polymerase (Applied Biosystems), 1× PCR buffer, 1.5 mM MgCl2, 0.1 mM of each deoxynucleotide triphosphate, 0.2 µM of each primer (Table 1) and 2 µl of modified genomic DNA. Thermocycling conditions for both the unmethylated- and methylated-specific PCRs were an initial step of 5 minutes at 95°C followed by 35 cycles of 30 sec. at 94°C, 30 sec. at 60.5°C and 60 sec. at 72°C before a final elongation step at 72°C for 7 min.<br><br>Chk2 Dimerisation
Chk2 mutant's ability to form dimers with the wild-type protein was investigated by immunoprecipitation. U-2-OS cells were co-transfected with expression vectors expressing wild-type Chk2 with N-terminal Xpr-tag (pcDNA4/HisMax, Invitrogen) and mutated Chk2 forms with C-terminal V5-tag (pcDNA3.1/V5-His, Invitrogen). Transfection was performed using FuGene 6.0 transfection reagent (Roche) according to the manufacturer's instructions. Cells were harvested in lysisbuffer (50 mM TrisHCl pH 8.0, 150 mM NaCl, 0.5% NP40, 5 mM EDTA pH 8.0) 48 hours after transfection. An aliquote of the cell lysate was harvested for subsequent Chk2-mutant-V5 transfection verification. Samples were further incubated with A/G Pluss Agarose beads (Santa Cruz Biotechnology) at 4°C for 25 minutes before the beads were removed by centrifugation at 5000g for 4 minutes and the samples were incubated with 1.5 µg anti-V5 (Invitrogen) at 4°C for 90 minutes. Fresh A/G Pluss Agarose beads were added and the samples were incubated for another 90 minutes at 4°C. The beads were washed three times with 1×PBS, before being separated on a 10% polyacrylamide gel and blotted on to a nitrocellulose membrane. Chk2-wild-type-Xpr co-precipitated with Chk2-mutant-V5 was detected through incubations with anti-Xpr antibody (Invitrogen), HRP-conjugated secondary antibody and ECL detection reagent (GE Healthcare).<br><br>Kinase Activity
Chk2 mutant's ability to function as kinases was investigated through an in vitro kinase assay. The V5 expression vectors used for the dimerisation study were also used to express Chk2 mutants in the kinase assay. U-2-OS cells were transfected using the FuGene 6.0 transfection reagent (Roche) according to the manufacturer's instructions. Cells were then incubated at 37°C in 5% CO2 and humidified atmosphere. After 24 hours doxorubicin (Nycomed Pharma) was added to the media to a final concentration of 50ng/ml and the cells were further incubated for 24 hours before harvest. 75 cm2 of 90% confluent cells were harvested in 500 µl lysis buffer (50 mM HEPES, 150 mM NaCl, 10% glycerol, 0.5% Triton X-100, 2 mM MgCl2, 5 mM EDTA), and the cytosol was incubated for 90 minutes at 4°C with 50 µl 50% Glutathione Sepharose beads (Amersham Biosciences) linked to anti-V5 antibody (Invitrogen). The beads were then washed twice with lysisbuffer containing 500 mM NaCl and twice with kinase assay buffer (50 mM HEPES, 10 mM MgCl2, 5 mM MnCl2, 2.5 mM EGTA). The beads received 30 µl kinase assay buffer with 7.5 µM cold ATP, 10 µCi 32P-gamma-ATP (GE Healthcare) and 2 µg isolated Cdc25C peptide, and was incubated at 30°C for 30 minutes. Samples were separated on a 12.5% polyacrylamide gel and blotted on to a nitrocellulose membrane. A radiosensitive imaging plate was exposed to the membrane and the plate was read in a FLA200 imager (Fuji).
The kinase assay described above was also used to determine the Chk2 mutants' kinase activity after co-transfection of each Chk2 mutant and wild-type Chk2 in equal amounts.<br><br>Statistical Analysis
Statistical analysis was performed using the Primer of Biostatistics system, version 5.0 [38]. The differences in the distribution of TP53 and CHEK2 mutations among <span class="yellow">patients</span> revealing a PD and the responders were analyzed with use of Fisher's exact test. P-values are reported as accumulated two-sided. Because of the limited time of the follow-up, no formal statistical assessment of overall survival was performed. Relapse-free survival was analyzed by the log-rank test. Details regarding outcome in individual <span class="yellow">patients</span> with mutations are shown in Table 2 and 3 to make them available to the reader.<br><br>
Results
TP53 Mutations and Response to Therapy
The TP53 mutations identified in the tumors of the <span class="yellow">patients</span> treated with epirubicin together with the clinical response to therapy and follow-up data are presented in Table 2. Somatic TP53 mutations were identified in 23 (21.5%) of the <span class="yellow">patients</span>. Normal tissue (WBC) was available from 18 of these for germline characterization, revealing none of the mutations identified to be germline alterations. Of the 23 mutations detected, 20 were missense and 3 were nonsense. One mutation (del483CAT) has not been reported previously either in breast cancer or in any other tumor type (IARC database: http://www.iarc.fr/p53/). Twelve of the mutations directly or indirectly affected the L2/L3 domains of the p53 protein (Table 2) previous found to predict a poor prognosis [39] and drug resistance [14], [40]. For statistical comparison, mutation Gly325Ter (<span class="yellow">patient</span> Epi215) located to the tetramerization domain is grouped together with the mutations affecting the L2/L3 domain, since this mutation leads to truncation of the protein and with loss of tetramerization and functional defects similar to L2/L3 mutations [41].
There was a statistical significant correlation between TP53 mutation status and lack of treatment response (PD) (Table 4; p = 0.0358; Fisher exact test). When tumors harboring TP53 mutations affecting the p53 L2/L3 DNA-binding domains were compared to those with wild-type TP53 or TP53 mutations outside the L2/L3 domains, this correlation was further strengthened (p = 0.0136).
The previously described TP53 polymorphism, Arg72Pro [42] was detected in 31 (29%) of our <span class="yellow">patients</span>. No correlation was found between this polymorphism and lack of treatment response (p = 0.2750; Fisher exact test) or TP53 mutational status (p = 0.2024).<br><br>CHEK2 Mutations and Response to Therapy
Table 3 presents the <span class="yellow">patients</span> with detected CHEK2 mutations together with a description of the clinical response and follow up-data. CHEK2 mutations were identified in three out of the 109 <span class="yellow">patients</span> (2.8%). Notably, each of the CHEK2 mutations identified was also present in <span class="yellow">patient</span> lymphocyte DNA, confirming a germline origin. The Arg95Ter (C283T) mutation is novel. This mutation was present in two <span class="yellow">patients</span> (Epi132 and Epi203) living in different parts of Norway with no known family relationship. However, linkage analysis using microsatellite markers (D22S275, D22S272, D22S1172 and D22S423) suggested a common founder mutation (data not shown). The C283T transition generates a novel stop codon in exon 1 of CHEK2, leading to truncation of the Chk2 protein. LOH analysis indicated loss of the wild-type CHEK2 allele in the both tumors from the two <span class="yellow">patients</span> harboring this mutation (Epi132 and Epi203). Both these tumors were non-responsive to epirubicin therapy (PD). In contrast, the third <span class="yellow">patient</span> with a germline CHEK2 mutation (<span class="yellow">patient</span> Epi151; point mutation at T1091C, Ile364Thr) had a partial response to epirubicin therapy. This tumor was non-informative with respect to LOH. Taking all CHEK2 mutations together, they predicted resistance to epirubicin (p = 0.0226).
The previously described silent Glu84Glu (A252G) polymorphism [24], [43] in exon 1 was detected in two (1.9%) <span class="yellow">patients</span>. No association between this polymorphism and treatment response was recorded.
One of the tumors (Epi203) harboring the C283T substitution (Arg95Ter) also harbored a somatic TP53 mutation in codon 175, Arg175His, located in the L2 domain of p53 (Table 2). This mutation was detected in another four of our <span class="yellow">patients</span> treated with epirubicin (Table 2). In addition, TP53 Arg175His mutation was recorded in one <span class="yellow">patient</span> of our previous study evaluating response to doxorubicin [13]. The fact that none of the Arg175His <span class="yellow">patients</span> presented here or in our previous study revealed resistance to therapy (PD) suggests this mutation may not cause resistance to anthracyclines in breast cancers in vivo. Omitting the tumor harboring both a CHEK2 and a TP53 mutation (<span class="yellow">patient</span> Epi203) from statistical analysis, Chk2 mutations (n = 2) were non-significantly associated with therapy resistance (p = 0.1633). In a previous study [26], however, we analyzed for CHEK2 mutation status in relation to therapy outcome in a cohort of <span class="yellow">patients</span> from doxorubicin study [13]. In that study [26], we detected the previously identified mutation Ile157Thr. In addition, we detected a novel nonsense somatic mutation (1368InsA). This mutation was associated with lack of function in vitro; moreover, it was associated with drug resistance in vivo. Analyzing our material and this cohort [26] together, (n = 160), CHEK2 mutations (n = 5 in total) predicted for resistance to doxorubicin and epirubicin therapy (p = 0.0123). Even though, excluding <span class="yellow">patient</span> Epi203 (harboring TP53 Arg175His and Arg95Ter CHEK2 mutation) as well as other <span class="yellow">patients</span> harboring TP53 L2/L3 mutations (n = 129), CHEK2 mutations (n = 4 in total) predicted for resistance to doxorubicin and epirubicin therapy (p = 0.030).<br><br>TP53 and CHEK2 Mutations Combined and Response to Therapy
Assuming that TP53 and CHEK2 mutations may substitute for each other, we analyzed for the predictive effect of mutations in both genes. The occurrence of a mutation affecting either CHEK2 or TP53 strongly predicted therapy resistance (p = 0.0101; Fisher exact test). When tumors harboring TP53-L2/L3 mutations and CHEK2 mutations were compared with those wild-type or TP53 mutations outside the L2/L3 domain, the correlation was further strengthened (p = 0.0032; Fisher exact test). The significance was preserved when comparing <span class="yellow">patients</span> with a PD to objective responders (CR and PR) excluding <span class="yellow">patients</span> with stable disease (SD) from the statistical analysis (Table 4).<br><br>p14(ARF) Mutations and Promoter Methylations
Neither mutations nor polymorphisms in the coding region of p14(ARF) were observed among the 107 <span class="yellow">patients</span> analyzed. Likewise, no promoter methylations were detected.<br><br>Influence of CHEK2 and TP53 Mutation Status on Relapse-Free Survival
Because of the limited time of the follow-up, no formal statistical assessment of overall survival was performed. Details regarding outcome for individual <span class="yellow">patients</span> with mutations are described in Table 2 and 3 to make these data available to the reader. Relapse-free survival is depicted in (Figure 1). Figure 1A shows relapse-free survival for the <span class="yellow">patients</span> with TP53 and CHEK2 mutations (all mutations found) compared to <span class="yellow">patients</span> without any TP53 or CHEK2 mutations, no difference in relapse-free survival was observed. Similar, no difference was seen when grouping TP53 mutations outside L2/L3 and CHEK2 mutation not affecting kinase function (Ile364Thr) as wild-type (Figure 1B). Grouping tumors harboring a mutation in L2/L3 together with CHEK2 mutations affecting kinase domain (Arg95Ter) in one group, mutations outside TP53 L2/L3 and Ile364Thr as one group and tumors without any found mutations in TP53 and CHEK2 separately, again no noticeably difference in relapse-free survival were seen (Figure 1C). Notably, in addition to a short median follow-up time, a total of 35 <span class="yellow">patients</span> with a sub-optimal response to epirubicin received subsequent treatment with paclitaxel, which may have influenced the outcome.<br><br>CHEK2 Mutant's Capability to Form Dimers
To investigate whether the identified CHEK2 mutations affect the ability of the Chk2 protein to form dimers, co-transfection and immunopresipitation of V5-tagged mutants and Xpress-tagged wild-type Chk2 were performed using CHEK2 low-expressing U-2-OS cells. As we identified the previously characterized CHEK2 germline mutants variants Arg117His (n = 2 and Ile157Thr (n = 1) among <span class="yellow">patients</span> allocated to primary treatment with paclitaxel in our ongoing study, these mutants were evaluated together with Arg95Ter and Ile364Thr. The results presented in Figure 2 show that all Chk2 variants carrying a point mutation were able to form dimers with wild-type Chk2, whereas the Arg95Ter variant was not.<br><br>Kinase Activity of CHEK2 Mutants
To investigate whether the identified CHEK2 mutants retained the wild-type kinase activity, an in vitro Chk2 kinase assay with respect to Chk2 autophosphorylation and Cdc25 substrate phosphorylation was performed. The U-2-OS cells were preferred for this assay because they were previously found to express only low levels of endogenous Chk2 [44]. This was confirmed by us using an antibody recognizing endogenous protein (data not shown). These cells have previously been used by other investigators to study Chk2 kinase activity [44], [45], [46].
The two mutants Arg117Gly and Ile157Thr were previously tested for in vitro kinase activity [47], but were both included here, together with wild-type CHEK2 as controls. Compared to wild-type Chk2, the Ile157Thr mutant retained wild-type kinase activity. The mutant Ile364Thr showed partially reduced kinase activity both in term of Cdc25-phosphorylation and autophosphorylation (Figure 3). In contrast, the mutant Arg117Gly showed strongly reduced kinase activity while the Arg95Ter mutant was totally devoid of any Chk2 kinase activity. The activity recorded for Ile157Thr and Arg117Gly was consistent with previously reported results for these two mutants [47]. Notably, there was an internal consistency with respect to percentage activity reduction comparing individual mutants with respect to autophosphorylation and phosphorylation of Cdc25 (Figure 3).
Since enzymatically active Chk2 exists as dimers, it was important to determine the effect of Chk2 mutants on wild-type/mutant heterodimer kinase activity. The effect on Chk2 kinase activities (Chk2 autophosphorylation and Cdc25 substrate phosphorylation) of the individual mutants were therefore determined after co-transfection with wild-type Chk2 as described in Materials and Methods. The results from this co-transfection-kinase assay (Figure 4) were similar to those of the single-transfection assay (Figure 3) except in the case of the Arg117Gly mutant, which expressed a substantial kinase activity when complexed with wild-type Chk2. This is consistent with previous data indicating that the Arg117Gly mutant has neglectable kinase activity itself but dimerizes efficiently to Chk2 wild-type without strongly affecting the wild-type Chk2 activity. Hence, the activity detected is probably caused by the co-transfected and co-precipitated wild-type protein.
To rule out the possibility that endogenously expressed wild-type Chk2 contributed to observed Arg117Gly kinase activity shown in Figure 4, we compared the Arg117Gly variant activity in the presence or absence of co-transfected wild-type Chk2 to the activities of Arg95Ter under the same conditions. The Arg95Ter variant does not form dimers with wild-type Chk2. As seen in Figure 5, Arg117Gly, which forms dimers with Chk2 wild-type, allows increased activity when co-transfected with wild-type as compared to the corresponding activity for the Arg95Ter mutant. The fact that Arg117Gly, when transfected alone, displays very similar activity as Arg95Ter or negative control (background levels), strongly indicates that the contribution of endogenous Chk2, which, similarly to exogenously expressed wild-type Chk2 co-precipitate with Arg117Gly is non-significant.<br><br>Family Cancer Incidence in Relation to CHEK2 Germline Mutations
Following an initial report of a family with a CHEK2 germline mutation expressing an increased cancer incidence resembling the Li-Fraumeni syndrome [24], recent studies have revealed the more common CHEK2 mutations to be associated with a moderately increased risk of breast and colorectal cancers. We hypothesized that CHEK2 mutations having a detrimental effect on drug sensitivity could be associated with a more aggressive, Li-Fraumeni or a Li-Fraumeni-like (LFL) cancer syndrome [48]. Except from the <span class="yellow">patient</span> harboring the Ile364Thr mutation who did not have any known congestion of cancer disease in the family, a detailed assessment of family cancer history was performed for each <span class="yellow">patient</span> harboring a germline CHEK2 mutation. The family cancer pedigrees are depicted in Figure 6.
While <span class="yellow">patients</span> harboring CHEK2 germline mutations revealed different types of cancers (mainly breast and tumors of the gastrointestinal area) in their family, surprisingly, no distinct pattern discriminating families harboring the Arg95Ter mutation from the other CHEK2 mutated families could be identified. One of them (Epi203), who inherited the mutation from her father's side of the family, had no accumulation of either breast or colorectal cancer on that side. It should be noted, however, that two brothers of her fathers mother had prostate cancer, and two siblings of his father having hepatocellular carcinoma and bladder cancer, respectively), while the other expressed a disease pattern resembling what has been seen with the more common CHEK2 mutations, like del1100C [25].<br><br>
Discussion
TP53 plays a key role as a tumor suppressor gene. Its protein product activates processes such as growth arrest, DNA repair, apoptosis and/or senescence in response to genotoxic damage as well as oncogene activity [49], [50]. Despite being extensively studied, critical issues regarding regulation of the p53 protein remain poorly understood, and conflicting evidence obtained in different experimental systems make the clinical relevance of experimental data questionable.
Chemoresistance is the main obstacle to cancer cure in most malignancies, including breast cancer. Previously, we found TP53 mutations affecting the L2/L3 DNA binding domain to be associated with lack of responsiveness to doxorubicin monotherapy [13] as well as mitomycin and 5-fluoro-uracil in concert [14]. However, some tumors revealed therapy resistance despite harboring wild-type TP53. Postulating that these tumors may harbor genetic disturbances in genes playing a key role in the p53 pathway, we here sequenced TP53 along with CHEK2 and p14(ARF), the latter two known to play a critical role as p53 activators, in tumors from 109 <span class="yellow">patients</span> treated with epirubicin monotherapy. Our results confirm TP53 mutations, in particular those affecting the L2/L3 domains, to be associated with drug resistance. Most importantly, we also found CHEK2 mutations generating a non-functional protein in our in vitro assays to be associated with drug resistance. In contrast, none of our tumors harbored either mutations or expressed promoter hypermethylations affecting the p14.
Based on in vitro assays, we were able to classify the different Chk2 mutants with respect to dimerization capability as well as kinase activity (Chk2 autophosphorylation and Cdc25 substrate phosphorylation). In addition, the kinase activities of the Chk2 wild-type/mutant complexes were monitored in co-transfection experiments. Notably, each point mutation (except for Arg117Gly) revealed similar relative kinase efficacy whether co-transfected with wild-type Chk2 or not (Figure 3 and 4). Cells co-transfected with Arg117Gly and wild-type Chk2 revealed kinase activity, probably due to the contribution of the wild type protein in Chk2 mutant – wild-type heterodimers. In contrast, cells transfected with Arg95Ter revealed no kinase activity whether co-transfected with wild-type Chk2 or not, clearly distinguishing this mutation from the others (Figure 3 and 5).
All in vitro assays were based on transfection of the U-2-OS cell line, a cell line known to express wild-type Chk2 at low levels, and previously used by other investigators to study Chk2 activity [44], [45], [46]. Since we were not able to obtain satisfactory technical quality of the kinase assay in cell lines negative for Chk2 (HCT 15 and HCT 116), we assessed potential background kinase activity due to endogenous Chk2 by performing western blot analysis revealing the endogenous levels of Chk2 in U-2-OS cells to be non-significant compared to the exogenously expressed Chk2 levels (data not shown). We also performed a separate kinase assay, directly comparing the effect of binding partners for the dimerizing Arg117Gly and the non-dimerizing Arg95Ter. This assay also revealed the contribution of endogenous Chk2 to be non-significant (Figure 5).
Taking our in vitro findings together with in vivo observations, our present data confirm that the functionally defective CHEK2 Arg95Ter mutation, together with LOH, is associated with resistance to anthracycline therapy. In contrast, the <span class="yellow">patient</span> harboring the Ile364Thr mutation, moderately reducing phosphorylation activity, responded well to therapy. The other missense mutations; Arg117Gly and Ile157Thr were observed among <span class="yellow">patients</span> receiving paclitaxel therapy only; thus, their influence on anthracycline sensitivity in vivo could not be addressed. Yet, based on the finding that the Arg117Gly mutant expressed no intrinsic activity, but readily dimerized to the wild-type protein without abolishing its activity, we hypothesize that this mutation and, probably, other yet unidentified CHEK2 mutations with a similar lack of intrinsic kinase activity, may cause resistance to anthracycline therapy if combined with LOH in breast cancer.
Our present findings have two major implications. First, we confirm that mutations in genes encoding proteins located within the same functional pathway may substitute for each other with respect to drug sensitivity, revealing for the first time a functional pathway critical to chemotherapy response in vivo. Second, the identification of mutations in the CHEK2 but not in the p14(ARF) gene in resistant tumors suggests that Chk2 mediated phosphorylation of p53 is a critical event in executing anti-tumor effect as a response to DNA damaging agents in breast cancer. This adds to our understanding not only of the function of p53 but Chk2 as well. p53 undergoes phosphorylation at multiple sites by different kinases, including Chk2 [51]. While activation of the ATM leading to direct (Ser 15) and Chk2-mediated (Ser 20) phosphorylation of p53 is considered an important mechanism for triggering p53 activation in response to DNA damage [52], some reports suggest ATM [53] and even Chk2 [23] to be redundant to this function. Importantly, Chk2 has been shown capable of inducing ATM-independent apoptosis in vitro [21]. While Chk2 phosphorylates p53 at Ser 20, thereby stabilizing p53 by preventing MDM2 binding [19], Chk2 also phosphorylates p53 at six additional sites, including Ser 313 and Ser 314 located in the nuclear localization signal domain of p53 [51]. In addition, Chk2 phosphorylates other important targets like BRCA1, Cdc25A and Cdc25C involved in DNA repair, G1 and G2 arrest, respectively [54]. Despite the wide range of known Chk2 substrates relevant for DNA repair and cell cycle control, our present findings that CHEK2 mutations leading to non-functional Chk2 protein may substitute for p53 mutations strongly advocate a role for Chk2 with respect to drug sensitivity executed through p53 activation.
Notably, one of the tumors (Epi203) with the Arg95Ter CHEK2 mutation in addition harbored a somatic TP53 mutation, Arg175His, with allelic imbalance for the TP53 gene (Table 2). Importantly, among another four <span class="yellow">patients</span> in this study (Epi063, Epi071, Epi087, Epi153) and one <span class="yellow">patient</span> from our previous doxorubicin protocol [13] harboring the Arg175His mutation together with allelic imbalance for TP53, all five of these <span class="yellow">patients</span> responded to anthracycline therapy either with a partial response or stable disease. In contrast, Epi132 and the only <span class="yellow">patient</span> for whom we previously identified a non-functional CHEK2 mutation (1368InsA; coding for a non-functional protein translate with cytoplasmic location [26]) expressed resistance to epirubicin and doxorubicin, respectively. Arg175His is a p53 “hot-spot” structural mutation reported to have defects with respect to transcriptional activation and also to negatively interact with wild-type p53 [55]. While this mutation has been shown to enhance chemoresistance upon transfection into p53 null Saos-2 cells [56], these osteosarcoma-derived cells may not necessarily be representative for breast cancers in vivo. Recent evidence strongly support p53 to be involved also in non-transcriptional mediated apoptosis by interacting with the Bcl-2/Bax system [57], and transcription-defect structural p53 mutants have been shown to execute non-transcriptional apoptosis in experimental systems [58]. Concomitant inactivation of Chk2 and p53 in breast cancer has been recorded by others [59], and the finding that a somatic mutation may generate a “growth advantage” in tumor cells already harboring a germline CHEK2 mutation may not implicate an effect on drug sensitivity in tumors not yet exposed to cytotoxic compounds. Rather, it may indicate a growth advantage, probably related to loss of p21 function. Notably, in a previous study we found the p21 polymorphism G251A to be associated with an increased risk of developing large breast cancers but to have no effect on drug sensitivity [60], indicating that growth rate and drug resistance may be regulated independently. Taken together, we believe our findings advocate a role for Chk2 in executing cellular response to anthracycline-induced DNA damage.
As mentioned above, removing TP53 mutated tumors including the double-mutated Epi203 from statistical analysis, CHEK2 mutation status still predicted for resistance to anthracycline therapy. In addition, removing the tumors harboring the Arg175His mutation from the p53 “L2/L3” group strengthened the correlation to lack of treatment response to epirubicin (p = 0.0005).
Comparing the effects of mutations in the CHEK2 gene to TP53 mutations indirectly underlines the importance of the role of Chk2 to chemoresistance. Our present findings as well as results from our previous studies [13], [14] revealed that about 50% of the <span class="yellow">patients</span> with tumors harboring TP53 L2/L3 mutations to be non-responders to primary therapy. In contrast, all our three <span class="yellow">patients</span> harboring a non-functional CHEK2 mutation (the two Arg95Ter mutated <span class="yellow">patients</span> here and our previous <span class="yellow">patient</span> harboring the 1368InsA) expressed primary resistance to therapy. We previously hypothesized that therapy response in tumors harboring TP53 L2/L3 mutations could be due to redundant pathways acting in concert [3]. Although no definite conclusion should be drawn from a limited number of observation, the fact that Chk2 not only phosphorylates p53 but also phosphorylates other substrates such as Cdc25A and Cdc25C [54] and E2F1 in response to etoposide-induced DNA damage [61] may indicate that inactivation of redundant pathways could take place in parallel.
The literature remains inconsistent with respect to whether the border amino acids 163, 195, 236 and 251 should be included in the p53 L2 and L3 domains [12]. Taking a conservative approach, we classified <span class="yellow">patient</span> Epi56, harboring a mutation in codon 163, as a L2/L3 mutant. The <span class="yellow">patient</span> harboring this mutation responded to therapy (PR). If this mutation was classified as outside the L2 domain, our p-value had been strengthened from p = 0.0136 to p = 0.0096.
Germline mutations in TP53 cause the Li-Fraumeni and Li-Fraumeni-like cancer disposition syndromes. However, while the germline and somatic mutations associated with these syndromes reveal a preference for the same codons [48], TP53 mutations affecting the DNA-binding domains seem associated with a poor prognosis [62], [63], [64] and, in particular, drug resistance [14], [40] in breast cancer. Thus, tumor suppression and tumor cell response to chemotherapeutics may involve different parts of p53 protein function. Following an initial report identifying a CHEK2 mutation in a family expressing characteristics of the Li-Fraumeni syndrome [65], recent evidence has linked CHEK2 founder mutations to a moderately increased risk of breast- and colorectal cancers with some additional disposition for other malignancies as well [66]. However, cancer incidence and phenotypes did not reveal an aggressive Li-Fraumeni or Li-Fraumeni-like tumor pattern. Similar to the two <span class="yellow">patients</span> in our paclitaxel treatment arm harboring the rare but previously characterized mutation Arg117Gly and the <span class="yellow">patient</span> with the Ile157Thr mutation, they expressed a moderately increased risk of breast and gastrointestinal cancers (Fig. 6). Thus, CHEK2 resembles TP53 in as much as there seems to be no direct correlation between effects of individual mutations with respect to tumor suppression and drug resistance.
Our finding that TP53 mutations located to the DNA-binding domains predicts drug resistance may indicate transcriptional mechanisms to be involved in drug-induced cell death. p53 induced apoptosis has been associated with transcriptional induction of genes including Puma and Noxa as well as Bax in experimental systems [55], [67], [68]. Yet, recent evidence has revealed p53 to induce apoptosis through non-transcriptional mechanisms by direct protein interactions with members of the Bcl-2/Bax system and mitochondrial release of cytochrom c [57], [69]. In deed, there is evidence that the DNA-binding domains, in particular the L3 part of the protein, may be critical also to transcriptional-independent apoptosis [70]. Of particular note is the finding that Chk2 may regulate transcriptional-independent p53-mediated apoptosis in response to DNA-damage created through ionizing irradiation [71]. Interestingly, Krajewski et al [72] reported low expression of Bax assessed by immunostaining to be associated with a low response to chemotherapy in metastatic breast cancer. Although no conclusion should be drawn at this stage, together these findings are consistent with the challenging hypothesis that transcription-independent activation of Bax following Chk2-phosphorylation may represent a key pathway in p53 dependent cell death in breast cancer in vivo.
p14 acts by releasing p53 from MDM2 binding, and has been related to oncogene-induced p53 activation [73]. Recently, p14 was shown to affect p53 by additional mechanisms, including acetylations [74], response to ionizing radiation in <span class="blue">human</span> fibroblasts [75], and tumor-suppression following ionizing radiation in <span class="yellow">mice</span> [76], [77]. These findings further links the retinoblastoma and p53 pathways [28]. As such, we believe the negative finding with respect to its role in chemoresistance adds important information.
Contrasting earlier findings by us and others [15], a recent study revealed TP53 mutations to be associated with increased likelihood of having a complete response to chemotherapy [16]. These results may not necessarily be at conflict. In the latter study, <span class="yellow">patients</span> received treatment with a “dose-dense” chemotherapy regimen; if confirmed, the combined data may outline a therapeutic indication for aggressive dose-dense therapy based on tumor TP53/CHEK2 status.
So far attempts to identify single markers and, more recently, gene expression arrays predicting chemoresistance have not proved successful (see refs in [9], [10]). The findings presented here reveal for the first time defects in a functional gene cascade to be associated with drug resistance in a <span class="blue">human</span> cancer in vivo. Moreover, the findings are made in breast cancer, the most frequent malignant disease among <span class="yellow">women</span> in the industrialized world, and relate to resistance to anthracyclines, the type of cytotoxic compounds most frequently employed for this malignancy.
While the only study we are aware of comparing TP53 mutation status in primaries and their distant metastases suggested an increasing fraction of tumors to express mutated TP53 during progression [78], we do not know the potential contribution of either TP53 or CHEK2 mutations to drug resistance in micrometastases or in metastatic disease. Yet the finding that one of our non-functional CHEK2 mutations associated with chemoresistance (1368InsA) occurred as a somatic, not germline mutation, suggest such mutations may be selected for during tumor progression. We propose the findings presented here provide important beacons identifying a functional pathway [3] likely to be disturbed through different mechanisms in relation to therapy resistance in advanced disease.
In conclusion, we believe our findings here that mutations in the TP53 and CHEK2 genes each may cause resistance to anthracycline therapy in primary tumors to have wide implications to future research in this area. While results from experimental systems are mandatory generating hypotheses, conflicting data from in vitro studies underlines the pivotal role of identifying defects associated with therapy resistance in vivo. Either through mutations of the genes themselves, or inactivation of this functional cascade through co-factors, we believe identification of the Chk2 – p53 axis as critical to anthracycline therapy response provides a functional clue for further investigations in this area.<br><br>
<h3>pmcA2562362</h3>Host immunity in the protective response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span>
Abstract
Background
We performed initial cell, cytokine and complement depletion studies to investigate the possible role of these effectors in response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span> in a susceptible BALB/c <span class="yellow">mouse</span> model of infection.<br><br>Results
While protection with heat-killed bacilli did not result in sterilizing immunity, limited protection was afforded against an otherwise lethal infection and provided insight into potential host protective mechanisms. Our results demonstrated that <span class="yellow">mice</span> depleted of either B cells, TNF-α or IFN-γ exhibited decreased survival rates, indicating a role for these effectors in obtaining partial protection from a lethal challenge by the intraperitoneal route. Additionally, complement depletion had no effect on immunoglobulin production when compared to non-complement depleted controls infected intranasally.<br><br>Conclusion
The data provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br><br><br>Background
<span class="yellow">Burkholderia mallei</span>, the etiologic agent of glanders, is a gram-negative, capsulated, non-motile, facultative intracelluar bacterium. Most known members of the Burkholderiaceae are resident in the soil; however, <span class="yellow">B. mallei</span> is thought to be an obligate mammalian pathogen. <span class="yellow">Horses</span> are highly susceptible to infection and considered the natural reservoir for infection, although <span class="yellow">mules</span> and <span class="yellow">donkeys</span> are also susceptible [1]. Clinically, glanders in solipeds can present as either a chronic (<span class="yellow">horses</span>) or acute (<span class="yellow">mules</span> and <span class="yellow">donkeys</span>) form. Naturally acquired <span class="blue">human</span> infection with <span class="yellow">B. mallei</span>, although not seen in the United States since 1945, has occurred rarely and sporadically among laboratory workers and those in direct contact with infected animals [2]. However, glanders is endemic among domestic animals in Africa, Asia, the Middle East, and Central and South America. The course of infection is dependent on the route of exposure. Direct contact with the skin can lead to a systemic infection. Inhalation of aerosol or dust containing <span class="yellow">B. mallei</span> can lead to septicemic, pulmonary, or chronic infections of the muscle, liver and spleen. The disease has a 95% case fatality rate for untreated septicemic infections and a 50% case fatality rate in antibiotic-treated individuals [3].
There is no <span class="blue">human</span> or animal vaccine available for glanders, and development of a partial or fully protective adaptive host response to the organism has not been well-defined. Previous studies with <span class="yellow">B. mallei</span> and the host response have shown that a mixed immune response consisting of both Th1 and Th2-associated cytokines with a predominant IgG1 subclass does not correlate with protection [4]. Additional studies with passive transfer of monoclonal antibodies specific for <span class="yellow">B. mallei</span> have correlated with early protection from infection [5]. Recent studies have also shown the Th1 cytokine IL-12 to mediate partial protection to non-viable <span class="yellow">B. mallei</span>-vaccinated <span class="yellow">mice</span> [6]. Thus, full correlates of protection mediated by the adaptive immune system against <span class="yellow">B. mallei</span> remain to be fully elucidated.
In this series of studies, we sought to address the impact of depletion of the major effector lymphoid cell populations (B220+ B cells, CD4+ or CD8+ T cells) and key pro-inflammatory/Type 1 cytokines (IFN-γ or TNF-α) on survival in BALB/c <span class="yellow">mice</span> vaccinated with heat killed (HK) bacilli followed by an intraperitoneal (i.p.) challenge with live organism. In addition, studies investigating the effect of complement on opsonization of organism and antibody production were assessed. Heat killed bacteria were used as a model of vaccination to allow evaluation of <span class="yellow">B. mallei</span> specific immune responses. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br>Results
Heat-killed <span class="yellow">B. mallei</span> vaccination mediates partial protection from lethal challenge
To begin to address this issue in an animal model of acute infection, we established that immunologically naive BALB/c <span class="yellow">mice</span> challenged i.p. with 2 × 107 CFU resulted in death by day 4–6, while i.p. immunization with 1 × 105 heat killed (HK) bacteria provided partial protection against a subsequent challenge. Two independent experiments resulted in similar findings of 40% survival for HK-vaccinated <span class="yellow">mice</span> with a mean survival time (MST) of 8 days versus 4 days in naïve <span class="yellow">mice</span> (Fig. 1). The administration of vaccines for <span class="yellow">B. mallei</span> during an outbreak would mandate relatively rapid onset of protection for <span class="blue">human</span> or veterinary use. Based on non-routine use and vaccine implementation in the course of an outbreak, a 14 day window was chosen for assessment of protection. Our results indicate that HK vaccination can afford partial protection to an otherwise lethal challenge of <span class="yellow">B. mallei</span> by the i.p. route.<br><br>Effects of cell depletion on HK-vaccinated survival
To dissect the cellular basis for protection mediated by HK vaccination, 13 days after immunization with HK bacteria (day -1), and at day of challenge, <span class="yellow">mice</span> were dosed with antibodies to deplete CD4+, CD8+ or B220+ cells. Antibody depletion of CD4+, CD8+, or B220+ cells in these <span class="yellow">mice</span> was confirmed by flow cytometric analysis with depletion efficiencies for CD4, CD8, and B220 populations at 99.7%, 96%, and 95%, respectively, relative to <span class="yellow">mice</span> treated with isotype control monoclonal antibodies (data not shown). Our results demonstrated decreased survival rates in B220 (p = 0.3418), CD4+ (p = 0.5417) and CD8+ (p = 0.4684) antibody depleted <span class="yellow">mice</span>, compared to isotype control antibody, a finding that indicated a possible role for vaccine induced antibody production. When challenged with 2 × 107 CFU/<span class="yellow">mouse</span> by the i.p. route, loss of T cells resulted in reduced survival (50%) relative to the non-specific isotype control (Fig. 2). In contrast to the loss of T cells, depletion of B220+ cells resulted in 100% mortality relative to the non-specific isotype control (Fig. 2). To further evaluate the necessity of these effector cells in providing protection following HK vaccination, relatively resistant C57BL/6 <span class="yellow">mice</span>, deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) were subjected to an identical HK vaccination and challenge regimen. Mature, B-cell-deficient <span class="yellow">mice</span> demonstrated a 50% decreased survival (p = 0.0888) compared to the wild-type <span class="yellow">mice</span> with an MST of 35.5 days (Fig. 3). CD4-/- and CD8-/- <span class="yellow">mice</span> exhibited a 60% (p = 0.1343) and 0% reduced survival, respectively (Fig. 3).<br><br>Effects of cytokine depletion on HK vaccination
Similar studies were performed to determine the role of IFN-γ or TNF-α in acute infection in BALB/c <span class="yellow">mice</span> immunized with HK bacteria. Six hours before challenge, <span class="yellow">mice</span> were dosed with antibodies that neutralize IFN-γ or TNF-α. Individual depletion of either TNF-α (p = 0.0145) or IFN-γ (p = 0.0446) resulted in 100% mortality with an MST of 3 and 2 days, respectively, compared to the HK-vaccinated isotype control <span class="yellow">mice</span> (Fig. 4). In contrast, 40% of HK-vaccinated, isotype control <span class="yellow">mice</span> survived to at least 12 days post-challenge (Fig 4). To further evaluate the host TNF-α response during an established <span class="yellow">B. mallei</span> chronic infection, we infected 12 BALB/c <span class="yellow">mice</span> by the i.p. route with 1 × 106 CFU <span class="yellow">B. mallei</span>. One animal was terminally ill on day 37 post-infection. On day 42 post-infection, the remaining 11 <span class="yellow">mice</span> were dosed with either anti-TNF-α (n = 6), or control mAb (AFRC Mac 49) (n = 5). No further deaths were observed in the control mAb-treated <span class="yellow">mice</span>. Rapid mortality was observed in the anti-TNF-α-treated group, with all <span class="yellow">mice</span> dying within 7 days of treatment (p = 0.0023) relative to the isotype-treated controls (Fig. 5).<br><br>J774A.1 uptake of serum treated <span class="yellow">B. mallei</span>
Complement mediated uptake assays were performed to evaluate opsonization. Results indicated enhanced bacterial uptake in J774A.1 phagocytes inoculated with serum treated <span class="yellow">B. mallei</span> (p = .0082), compared to <span class="yellow">B. mallei</span> alone, while heat-inactivated serum produced uptake percentages similar to those prior to serum addition (Fig. 6). Taken together, these results imply an active role for complement components in the uptake of organism by macrophages.<br><br>Immunoglobulin production in HK vaccinated BALB/c <span class="yellow">mice</span>
We further characterized the ability of HK vaccination to induce a predominant IgG isotype by determining IgG2a/IgG1 ratios in i.p. and i.n. vaccinated BALB/c <span class="yellow">mice</span>. Pre (day 14 post vaccination) and post (day 2 post infection) exposure serum samples were obtained and evaluated for IgG isotype concentrations (Table 1). No appreciable differences in IgG pre-exposure levels were seen when comparing i.n. to i.p. vaccination. In addition, cobra venom factor-treated animals showed no significant differences to non-cobra venom factor-treated animals in IgG pre-exposure (challenge) levels. Conversely, isotype switching in the cobra venom factor treated animals was enhanced in post-exposure serum IgG2a (Table 1).<br><br>
Discussion
Recent studies have shown a key role in protection from lethal challenge for IFN-γ in non-vaccinated <span class="yellow">mice</span> from either NK and/or NKT cells following experimental exposure to <span class="yellow">B. mallei</span> and <span class="yellow">B. pseudomallei</span> [7,8]. A similar protective role in the innate response to infection has been demonstrated for TNF-α in <span class="yellow">B. pseudomallei</span> infection [8]. The studies presented here are consistent with the essential role of these factors in the relative levels of protection conferred by vaccination with heat-killed <span class="yellow">B. pseudomallei</span> and would appear to be viable early markers for protection from lethal acute infection [9]. Currently, there are no fully protective vaccines against <span class="yellow">B. mallei</span> or <span class="yellow">B. pseudomallei</span> in a <span class="yellow">murine</span> model, particularly for the sensitive BALB/c versus C57BL6 models. Previous studies have also demonstrated that both the humoral and cell-mediated arms are essential for protection from <span class="yellow">B. pseudomallei</span> infection [10]. Thus, loss or reduction of TNF-α and IFN-γ levels result in significantly reduced survival rates, substantiating previous reports of the role of these factors in protection against <span class="yellow">B. mallei</span> [7]. Moreover, we demonstrate a role for sustained TNF-α production in the maintenance of host survival throughout the course of <span class="yellow">B. mallei</span> infection. <span class="yellow">Mice</span> with an established <span class="yellow">B. mallei</span> chronic infection rapidly lost the ability to control the growth of the bacillus upon neutralization of TNF-α. This would suggest a potential role for TNF-α in the maintenance of productive granulomas which may limit the spread of bacteria in chronically infected hosts, or, alternatively, in direct or indirect microbicidal or bacteriostatic activities at the sites of infection. Additional studies are underway to determine more precisely the role of TNF-α in host protection to <span class="yellow">B. mallei</span>.
Multiple innate and adaptive cell types may contribute to the production of IFN-γ in response to infection with <span class="yellow">B. mallei</span> following vaccination. Our results with individual depletion of CD4+ and CD8+ T cells suggests that both cell types may compensate for the functional loss of the other effector cell type in the production of this key cytokine. The effector role for IFN-γ in mediating protection against <span class="yellow">B. mallei</span> may include both immunoregulatory and non-regulatory functions. Regardless, the requirement of IFN-γ, as demonstrated by administration of neutralizing antibody prior to infection, indicates that stimulation of IFN-γ response is a desirable goal for a <span class="yellow">B. mallei</span> vaccine.
Similarly, B220-positive cells appear to play a role in protection following vaccination with heat-killed <span class="yellow">B. mallei</span>. Interestingly, this protective immunity, occurring in other intracellular pathogens, is not exclusively dependent on B cells [11]. Passive protection has been demonstrated against acute Burkholderia infection by monoclonal antibodies [5,12]. Protection against <span class="yellow">B. pseudomallei</span> infection by anti-LPS, capsular polysaccharide and proteins has been short-lived, suggesting that antibody production offers limited protection in the initial stages of infection by an as-yet-undefined mechanism [12]. We have shown that following depletion of B220+ cells, survival rates decreased as much as 100% relative to non-depleted controls and individual CD4/CD8-depleted <span class="yellow">mice</span> via the intraperitoneal route. Results from C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) substantiate the requirement for B-cell involvement by evidence of μMT and CD4-/- decreased survival. The lack of an effective CTL response to vaccination did not appear to alter survival in what would appear to be a CD4/B-cell (humoral)-driven response. In CD4-deficient <span class="yellow">mice</span>, we have the additional potential variable that a CD4-dependent antibody response might also be inhibited during the vaccination phase relative to <span class="yellow">mice</span> treated with antibody immediately prior to and during the early phases of infection. Although not statistically significant, we did observe a decrease in survival in μMT (mature B cell) deficient <span class="yellow">mice</span> as early as day 9 post challenge, whereas CD4-deficient <span class="yellow">mice</span> produced similar results at day 32 post challenge, indicating a role for B cells independent of CD4 T cell help, perhaps through a T-independent mechanism of antibody production. Although CD8-/- C57BL/6 demonstrated no decreased survival in our HK-vaccinated model, a lack of potential endogenous protein production by HK <span class="yellow">B. mallei</span> may have contributed to limited MHC-I presentation.
Complement associated studies revealed increased J774A.1 uptake of serum-treated <span class="yellow">B. mallei</span>. Complement-mediated uptake studies of <span class="yellow">B. pseudomallei</span> by polymorphonuclear leukocytes (PMNs) suggest that capsule production contributes to resistance of phagocytosis by reducing C3b bacterial deposition [13]. Previous studies have demonstrated that a polysaccharide capsule is present in <span class="yellow">B. mallei</span>, [14,15] although in the present study enhanced uptake with serum-treated <span class="yellow">B. mallei</span> was observed. Intracellular survival assays of complement mediated uptake of organisms were not performed in the present study, thus, the role of complement opsonization on intracellular survival is not fully resolved. Previous reports have demonstrated the ability of <span class="yellow">B. mallei</span> to survive within macrophage without the aid of serum coating organisms [16]. Conversely, the idea of antibody mediated opsonization to facilitate macrophage activation and clearance of intracellular organisms may offer support to the role of B cells in an effective immune response. A possible protective mechanism may include HK vaccination induced production of opsonizing antibodies which may aid in complement mediated uptake, thereby limiting the initial bacterial threshold below a lethal level.
Immunoglobulin responses to HK vaccination resulted in modest levels of IgG1 following 2 weeks post vaccination, while post-exposure levels were indicative of efficient class switching to a favorable IgG2a isotype. Importantly, cobra venom factor treatment of animals at time of vaccination did not alter their ability to produce immunoglobulin. In fact, cobra venom factor treated animals resulted in higher IgG2a levels when compared to non-treated. Complement activation can modulate both the primary and secondary immune responses and has been shown to enhance secondary immune responses to vaccination [17]. The current results suggest that cobra venom factor treatment may affect the modulation of the immune response to <span class="yellow">B. mallei</span> infection through B cell activation and/or memory B cell generation.<br><br>Conclusion
In summary, our results provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. Understanding and defining the role of B cells in adaptive <span class="yellow">B. mallei</span> immunity will likely be fundamental to the design of an efficacious vaccine and important goals of future research.<br><br>Methods
Bacterial strain and <span class="yellow">mice</span>
<span class="yellow">B. mallei</span> strain ATCC 23344 (China 7) was cultured on Luria-Bertani agar supplemented with 4% glycerol (LB+4%G) agar plates for 48 h at 37°C. Isolated colonies were sub-cultured to LB+4%G broth, and cultures were incubated at 37°C until optical density readings at 600 nm (OD600) reached an exponential phase of growth. Bacteria were pelleted by centrifugation, washed and re-suspended in sterile 1× phosphate-buffered saline (PBS, pH 7.4) to obtain the desired CFU/ml. To obtain HK inoculums, bacterial suspensions were incubated at 85°C for 3 h and stored at 4°C until use. The absence of live <span class="yellow">B. mallei</span> organisms in the HK preparations was confirmed after plating 10% of the total inoculums (v/v) and incubating these at 37°C for 48 h. All procedures were performed under a class II biosafety cabinet in a biosafety level 3 laboratory. Female, 6- to 8-week-old, BALB/c <span class="yellow">mice</span> (n = 5–7) were obtained from Harlan Sprague Dawley, Inc. (Indianapolis, Indiana). Female, 6- to 8-week-old, C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) and CD8 T-cells (CD8-/-) and wild-type <span class="yellow">mice</span> were obtained from The Jackson Laboratory (Bar Harbor, Maine).<br><br>Vaccination and challenge
BALB/c and C57BL/6 <span class="yellow">mice</span> were grouped and vaccinated with 0.5 μg of HK <span class="yellow">B. mallei</span> (without adjuvant) by i.p. injection using a 25-gauge syringe. Two weeks post HK vaccination <span class="yellow">mice</span> were injected i.p. with 2 × 107 CFU/100 μl of live <span class="yellow">B. mallei</span> (~20 LD50) [18]. Complement depleted animals were challenged with 2.5 × 104 CFU/50 μl (~0.25 LD50) by intranasal (i.n.) route. Aliquots from the inoculums were plated to confirm the infecting dose. All procedures and animal protocols used in this study were approved by the Biosafety and IACUC committees at UTMB and conducted in either BSL-3 or ABSL-3 laboratories.<br><br>Cell and cytokine depletions
Acute in vivo cell/cytokine depletion was performed with monoclonal <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD4 (GK1.5), CD8α (53-6.7) or B220 (RA3-6B2) obtained from R&D Systems, Inc. (Minneapolis, MN) by methods similar to those we have previously described [19]. Functional grade purified <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> interferon-gamma (IFN-γ, AN-18) was obtained from eBioscience (San Diego, CA) and purified anti-<span class="yellow">mouse</span> tumor necrosis factor (TNF-α, MP6-XT3) from BD Pharmingen (San Diego, CA). IFN-γ and TNF-α antibodies were injected i.p. 6 h prior to challenge, 200 μg per <span class="yellow">mouse</span> in 200 μl PBS or at later time points as indicated. <span class="yellow">Rat</span> IgG isotype control was obtained from Southern Biotech (Birmingham, AL) and administered i.p. on day of challenge, 200 μg/<span class="yellow">mouse</span>. <span class="yellow">Rat</span> anti-<span class="yellow">mouse</span> CD4, CD8α and B220 were injected i.p. twice, 1 day prior to challenge and on day of challenge, with an equivalent dosage sufficient to deplete T or B cells from 6 × 108 bone marrow cells per injection. The efficiency of depletion at time of infection for CD4+, CD8+, and B220+ cells was confirmed by flow cytometry analysis immediately prior to infection.<br><br>Complement depletion with cobra venom factor
<span class="yellow">Mice</span>, six to seven per group, were vaccinated i.p. with 1 × 105 CFU of nonviable <span class="yellow">B. mallei</span> cell preparation in a total volume of 0.1 ml. Two weeks later, 24 h and 1 h before challenge, complement depleted <span class="yellow">mice</span> were treated i.p. with 12.5 units total cobra venom factor (Quidel Corporation Speciality Products, San Diego, CA) in 0.1 ml of PBS. Complement depletion was confirmed prior to challenge by micro-titer hemolytic complement activity (CH50) assay as previously described [20].<br><br><span class="yellow">B. mallei</span> J774A.1 uptake assays
J774A.1 cells were seeded (5 × 105) onto Corning costar 24 well plates (Corning, NY) with DMEM and incubated overnight at 37°C with 5% CO2. Bacterial suspensions were incubated at 37°C for 45 minutes supplemented with 2% <span class="yellow">mouse</span> serum from Sigma-Aldrich (St. Louis, MO.), heat inactivated <span class="yellow">mouse</span> serum (56°C 30 minutes), or bacteria alone and then added at an MOI of 10:1 to J774A.1 cells in triplicate. Inoculated wells were centrifuged at 800 g for 2 minutes and incubated for 2 hours at 37°C with 5% CO2 followed by a PBS wash (×2) and 2 hour incubation with 250 μg/ml kanamycin. Wells were washed twice with PBS and lysed with 0.1% Triton X-100, followed by serial 10-fold dilutions plated on LBG plates and incubated at 37°C for 2 days. Colony forming units were enumerated and uptake expressed as a percentage of initial inoculating dose ± SEM.<br><br>Antibodies and flow cytometry
Flow cytometric analysis was performed on 0.1-ml blood samples transferred to micro centrifuge tubes containing 90 μl of acid citrate dextrose (ACD) solution. Red blood cells were lysed using ACK-lysing buffer (Biosource International, Inc., Camarillo, CA) according to the manufacturer's instruction. Antibodies used for analysis of surface markers included: FITC-conjugated <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD45R/B220 (RA3-6B2, BD Pharmingen San Diego, CA) for B cells; FITC-conjugated <span class="yellow">rat</span> anti-<span class="yellow">mouse</span> CD8α (53-6.7) and CD4 (GK1.5, BD Pharmingen, San Diego, CA) for CD8+ or CD4+ cells, respectively. Samples evaluated for CD4+ and CD8α+ cells were also incubated with biotin-conjugated <span class="yellow">hamster</span> anti-<span class="yellow">mouse</span> CD3e (145-2C11) monoclonal antibody (BD Pharmingen, San Diego, CA) and subsequently with streptavidin APC Cy7. Isotype-matched, non-specific controls were assayed in parallel (BD Pharmingen, San Diego, CA). Surface staining was performed according to previously published protocols [21]. Following cell staining, the samples were fixed with 2% buffered paraformaldehyde overnight prior to analysis by flow cytometry. Samples were analyzed using a FACSCalibur flow cytometer with BD CellQuest Pro software.<br><br>Antibody assays
Immunoglobulin subclass IgG1 and IgG2a titers in <span class="yellow">mice</span> were determined by a whole bacterial cell ELISA performed in 96-well, Immulon 2 HB, round-bottom plates (Dynex Technologies). <span class="yellow">B. mallei</span> antigen was diluted in 0.1 M carbonate buffer (pH 9.5) and 50 μl of diluted cells placed into wells. Plates were stored overnight at 4°C. The plates were washed with washing solution (1× PBS, 0.05% Tween 20), and incubated with 100 μl of blocking solution (1× PBS, 1% <span class="yellow">bovine</span> serum albumin, 0.05% Tween 20) for 1 h at 37°C. Dilutions of <span class="yellow">mouse</span> sera were made with blocking solution in duplicate and plates were incubated for 1 h at 37°C. Following incubation, plates were washed and 50 μl of anti-Ig-<span class="yellow">horseradish</span> peroxidase subclass conjugate, diluted accordingly to manufacturer's instructions (Southern Biotechnology Associates, Inc. Birmingham, Ala.), was added to each well and incubated for 1 h at 37°C. After washing, 50 μl of 2,2'-azino-di-(3-ethylbenzthizoline)-6-sulfonate (ABTS) peroxidase substrate (KPL, Inc., Gaithersburg, Maryland) was added to each well and plates incubated for 25 min at room temperature. The amount of bound antibody was determined colorimetrically by absorbance at 405 nm.<br><br>Statistical analysis
Survival curves were calculated by Kaplan Meier survival analysis with log-rank tests between groups using GraphPad Prism (V.4.03 for windows). Statistical analysis was generally performed with the paired Student's t-test. P value ≤ 0.05 was considered significant.<br><br>
Abbreviations
HK: Heat-killed; i.p.: intraperitoneal; i.n.: intranasal.<br><br>Authors' contributions
GCW designed and conducted experiments and drafted the manuscript. BMJ carried out the immunoassays and animal work. SP provided analysis of data and contributed to design and animal work. RAL participated in the generation and analysis of chronic TNF-α data. DME conceived the study, and participated in its design and coordination and helped to draft the manuscript. AGT participated in the bacterial work and drafting of the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2589670</h3>The ineffectiveness of tobramycin combination therapy in <span class="yellow">Streptococcus faecium</span> endocarditis.
Abstract
A <span class="yellow">patient</span> required mitral valve replacement following ineffective antibiotic treatment of enterococcal endocarditis caused by <span class="yellow">Streptococcus faecium</span>. Endocarditis had relapsed despite therapy with ampicillin and tobramycin for six weeks. A second relapse had occurred following treatment with penicillin and gentamicin. Initial failure of antibiotic therapy may be related to the known lack of in vitro and in vivo synergy between penicillin and tobramycin against <span class="yellow">S. faecium</span>. Effective therapy of enterococcal endocarditis requires considerations of bacterial speciation, determination of high-level aminoglycoside resistance, and preferably adequate antibiotic synergy studies to assure effective therapy.<br><br><br><br>
 THE YALE JOURNAL OF BIOLOGY AND MEDICINE 56 (1983), 243-249 <br><br> The Ineffectiveness of Tobramycin Combination Therapy <br><br> in <span class="yellow">Streptococcus Faecium</span> Endocarditis <br><br> JUDITH A. GOLDSTEIN, M.D.,a HOWARD COHEN, M.D.,a AND <br><br> FRANK J. BIA, M.D., M.P.H.a,b <br><br> aInfectious Disease Section of the Department of Medicine, and bThe Department of <br><br> Laboratory Medicine, Veterans Administration Medical Center, <br><br> West Haven, and Yale University School of Medicine, <br><br> New Haven, Connecticut <br><br> Received July 11, 1983 <br><br> A <span class="yellow">patient</span> required mitral valve replacement following ineffective antibiotic treatment of enterococcal endocarditis caused by <span class="yellow">Streptococcusfaecium</span>. Endocarditis had relapsed despite therapy with ampicillin and tobramycin for six weeks. A second relapse had occurred following treatment with penicillin and gentamicin. Initial failure of antibiotic therapy may be related to the known lack of in vitro and in vivo synergy between penicillin and tobramycin against <span class="yellow">S. faecium</span>. Effective therapy of enterococcal endocarditis requires considerations of bacterial speciation, determination of high-level aminoglycoside resistance, and preferably adequate antibiotic synergy studies to assure effective therapy. <br><br> INTRODUCTION <br><br> Enterococcal endocarditis requires special therapeutic considerations because the responsible organisms are relatively penicillin-resistant streptococci which require synergistic combinations of antibiotics to achieve acceptable cure rates [1,2]. The group D enterococci include three main species <span class="yellow">S. faecalis</span>, <span class="yellow">S. faecium</span>, and <span class="yellow">S. durans</span>. <span class="yellow">S. faecium</span> cause a minority of all cases of enterococcal endocarditis, in those instances in which enterococci have been speciated [3], but they have been more resistant both to penicillin and penicillin-aminoglycoside combinations than <span class="yellow">S. faecalis</span> [4,5,6]. <br><br> We describe a 64-year-old <span class="yellow">man</span> with <span class="yellow">S. faecium</span> endocarditis in whom a six-week course of ampicillin and tobramycin, followed by additional courses of penicillin and other aminoglycosides, failed to eradicate the organism from the <span class="yellow">patient</span>'s mitral valve. This case is of interest because therapeutic failure of ampicillin and tobramycin in <span class="yellow">S. faecium</span> endocarditis has not been reported previously, but might have been predicted on the basis of previous in vitro and in vivo studies [6]. Although the need for both a penicillin derivative and an aminoglycoside in the therapy of enterococcal endocarditis is widely known, it is important to distinguish between the differing efficacies of penicillin-aminoglycoside combinations for treating various species of enterococci such as <span class="yellow">S. faecium</span>. <br><br> 243 <br><br> Address reprint requests to: Judith A. Goldstein, M.D., Section of Infectious Diseases, Department of Medicine, Long Island Jewish-Hillside Medical Center, Queen's Hosp. Center Affiliate, 82-68 164th Street, Jamaica, NY 11432 <br><br> Copyright c 1983 by The Yale Journal of Biology and Medicine, Inc. All rights of reproduction in any form reserved. <br><br> GOLDSTEIN ET AL. <br><br> CASE REPORT <br><br> A 64-year-old male was in good health until December 1980, when he noted intermittent night sweats, malaise, fever, and fatigue. He received oral erythromycin for 14 days with transient improvement of symptoms. However, after completing therapy, symptoms reappeared and he noted a 15-pound weight loss with low-grade fever (99-100.5?F) during the two months preceding admission. There was no previous history of rheumatic or congenital heart disease. <br><br> In March 1981, the <span class="yellow">patient</span> was admitted to his community hospital where evaluation revealed a new apical systolic murmur radiating to the axilla. There were no petechiae, Janeway lesions, Osler's nodes, Roth spots, or splenomegaly. The hematocrit was 34.8 percent, WBC count 9,300 cells per cu mm with a differential count of 67 segmented forms, 9 bands, 15 lymphocytes, 7 monocytes, 1 eosinophil, and 1 basophil. The erythrocyte sedimentation rate (ESR) was 62 mm per hour (nl < 10 mm per hour) and the serum rheumatoid factor titer was 1:320. Chest films and electrocardiogram were reportedly normal. Group D streptococci grew from three sets of blood cultures. Enterococcal endocarditis was diagnosed and he was treated with six weeks of parenteral ampicillin (12 grams per day) and tobramycin (3 mg per kg per day). Resolution of symptoms occurred within several days after antibiotics were begun. Serum bactericidal titers against the organism, obtained during peak antibiotic levels, were 1:8 or greater on several occasions, and blood cultures were negative while the <span class="yellow">patient</span> was receiving antibiotics. M-mode echocardiography demonstrated left atrial enlargement but no definable abnormalities of the mitral or aortic valves. Intravenous pyelogram, oral cholecystogram, cystoscopy, liver-spleen scan, and upper and lower gastrointestinal series were normal except for a few sigmoid diverticuli. Flexible signoidoscopy demonstrated both a small perianal fissure and hemorrhoids. Blood cultures two weeks after therapy were negative. <br><br> Second Admission (June 19-August 4, 1981) <br><br> In June 1981, he again noted intermittent fever, night sweats, and fatigue. Group D streptococci grew from three sets of blood cultures and he was admitted to the West Haven VA Medical Center for recurrent endocarditis. On examination a somewhat louder apical systolic murmur was noted. The ESR was 50 mm/hour and rheumatoid factor titer was 1:320. Serum complement levels were normal and cryoglobulins were not detectable. Chest films, EKG, and urinalysis were normal. Cardiac M-mode echocardiogram revealed a globular mass attached to the posterior mitral valve leaflet with prolapse into a slightly enlarged left atrium. A twodimensional echocardiogram confirmed mitral valve prolapse and suggested a posterior leaflet vegetation. Gallium citrate scan was negative. <br><br> Three separate morphologic colony variants were isolated from blood, each identified as <span class="yellow">S. faecium</span> by Dr. R.R. Facklam (Center for Disease Control, Atlanta, Georgia). The <span class="yellow">patient</span> received intravenous penicillin (20 million units per day) and gentamicin (3 mg per kg per day) for six weeks with improvement. Peak serum bactericidal titers of 1:8 or greater were achieved against two of the colony variants; however, against the large colony morphotype, a titer of only 1:2 was obtained. An enlarged left atrium with intermittent fluttering and prolapse of the mitral valve was noted on echocardiography three weeks into therapy. Multiple blood cultures taken while the <span class="yellow">patient</span> was receiving antibiotics were negative, as were those obtained 48 and 72 hours after discontinuation of antibiotics. <br><br> 244 <br><br> <span class="yellow">S. FAECIUM</span> ENDOCARDITIS <br><br> Third Admission (August 12-October 4, 1981) <br><br> Withing a week following discharge, the <span class="yellow">patient</span> again developed fever, nocturnal sweats, and malaise. <span class="yellow">S. faecium</span> (large colony morphotype, and poorly growing small colony morphotype) grew from six sets of blood cultures obtained on admission. Penicillin (30 million units/day) and gentamicin (3 mg/kg/day) were again administered, initially achieving peak serum inhibitory and bactericidal dilutions of 1:16 and 1:8 against the organism, respectively. However, the organism had a lowlevel resistance to streptomycin (MIC < 125 /Ag/ml), and streptomycin (2 grams per day) was substituted for gentamicin two weeks into antibiotic therapy. Repeat echocardiograms showed irregular and shaggy densities of both mitral valve leaflets with partial prolapse. Cardiac catheterization demonstrated marked mitral valve prolapse with mitral regurgitation. A radiolucent filling defect was noted, suggesting a coronary artery embolus at the origin of the left anterior descending (LAD) artery, causing 75 percent occlusion of the lumen. The <span class="yellow">patient</span> underwent mitral valve replacement, receiving a number 31 porcine Hancock bioprosthesis and bypass graft to the midportion of the LAD. The mitral valve was thickened with several ruptured chordae of the posterior leaflet noted, but no vegetations. The aortic valve appeared normal, with no visible septal or ring abscesses. The occlusion in the LAD was not approached to avoid embolizing distal fragments. <br><br> Histopathologically the mitral valve showed mild fibrosis and myxoid degeneration without inflammatory changes. Bacterial and fungal stains were negative but <span class="yellow">S. faecium</span> grew from fragments of the resected valve. Following surgery the <span class="yellow">patient</span> received six additional weeks parenteral penicillin and streptomycin. Repeat blood cultures on this regimen and following therapy were negative. Evaluation six months following discontinuation of antibiotics showed no evidence of recurrent endocarditis. <br><br> LABORATORY EVALUATION <br><br> The minimum inhibitory and bactericidal concentrations of penicillin, ampicillin, and tobramycin against the <span class="yellow">S. faecium</span> isolated from the <span class="yellow">patient</span>'s blood cultures <br><br> TABLE 1 <br><br> Minimum Inhibitory and Bactericidal Concentrations of Antibiotics, <br><br> Against <span class="yellow">Streptococcus faecium</span> Isolates from Blood Cultures <br><br> Date       Penicillin     Ampicillin         Tobramycin <br><br> Organism         Isolated      MIC        MIC       MBC      MIC       MBC <span class="yellow">S. faecium</span>, prior to ampicillin/ <br><br> tobramycin therapy    3/17/81        2          1        2        >32       >32 <span class="yellow">S. faecium</span>, after ampicillin/ <br><br> tobramycin therapy <br><br> a. Large colony <br><br> variant           6/16/81        2          1        1        >32      >32 b. Medium colony <br><br> variant           6/16/81        2          1        2         32        32 c. Small colony <br><br> variant           6/16/81        4          2        4        >32      >32 aMIC and MBC are minimum inhibitory and bactericidal concentrations of antibiotics, respectively, in tig/ml. <br><br> 245 <br><br> GOLDSTEIN ET AL. <br><br> FIG. 1. Time-kill curve demonstrat8 3-  \   "''                                    ing the effects of various penicillinO                                                  aminoglycoside combinations on S. {73 2-  \         >,                             faecium obtained from the <span class="yellow">patient</span>'s <br><br> blood cultures immediately prior to mitral valve excision. Note the lack of 4u,                synergy between penicillin and tobra25kg/rm Streporrrychn t                  mycin against this organism when <br><br> /0 units/mI Penicillin _         compared to synergistic combinations 0     4     8    12    6    20    24            of penicillin-gentamicin and penicillin<br><br> HOURS                            streptomycin. <br><br> after initial oral erythromycin therapy, and prior to therapy for endocarditis, are shown in Table 1. Following combined therapy with ampicillin and tobramycin and relapse of endocarditis, three morphological variants were isolated from blood cultures and also evaluated. <br><br> Twenty-four hour time-kill curves for penicillin in combination with various aminoglycosides were performed by Dr. Robert Moellering on the <span class="yellow">S. faecium</span> isolated from the <span class="yellow">patient</span>'s blood immediately prior to mitral valve excision and replacement (Fig. 1). Synergy was readily demonstrable against this organism by penicillin-streptomycin and penicillin-gentamicin combinations in vitro, but not by penicillin-tobramycin. <br><br> DISCUSSION <br><br> Among streptococci, the enterococci are unusual in their relative resistance to a broad spectrum of antimicrobial agents, and single-agent therapy is rarely bactericidal against them [2]. Since Hunter's original observations in 1947, it has become increasingly clear that effective synergistic combinations of antibiotics are necessary to successfully treat enterococcal endocarditis [1]. <br><br> Although <span class="yellow">S. faecalis</span> represents the majority of clinical enterococcal isolates, <span class="yellow">S. faecium</span> nonetheless comprises 5-10 percent of these isolates in some series [3,4]. Moreover, major differences exist in antimicrobial susceptibility and resistance to penicillin-aminoglycoside synergism between these two enterococcal species. The MIC of penicillin against <span class="yellow">S. faecium</span> is higher and this organism is more resistant to a number of different combinations of penicillin and various aminoglycosides than is <span class="yellow">S. faecalis</span> [6]. <br><br> The mechanisms of resistance exhibited by enterococci to penicillin-aminoglycoside synergy have been investigated. Clinically achievable levels of amino<br><br> 246 <br><br> <span class="yellow">S. FAECIUM</span> ENDOCARDITIS <br><br> glycosides are generally ineffective against enterococci. This intrinsic low-level resistance (MIC c 250 isg/ml) is thought to be the result of poor antibiotic penetration of the bacterial cell wall. However, in the presence of antibiotics that interfere with cell wall synthesis, there is enhanced aminoglycoside uptake [7]. In concert, these events are the basis for penicillin-aminoglycoside synergism. Ribosomal resistance of the 30S subunit to streptomycin and defective uptake of gentamicin in the presence of penicillin have been reported mechanisms of resistance among enterococcal isolates [8,9]. However, in the majority of instances, failure of synergy involves plasmid-mediated production of aminoglycoside-modifying enzymes. For streptomycin and kanamycin, plasmid-mediated enzymatic inactivation confers high-level resistance (MIC > 2,000 Ag/ml) and correlates with failure of these aminoglycosides to exert a synergistic effect when combined with penicillin [10,11]. Plasmid-mediated modifying enzymes have been found in both <span class="yellow">S. faecalis</span> and <span class="yellow">S. faecium</span>' [12]. Currently, approximately one-half of clinical enterococcal isolates demonstrate high-level resistance to streptomycin and kanamycin [13]. <br><br> Combinations of penicillin with kanamycin, tobramycin, sisomicin, and netilmicin have consistently failed to demonstrate synergistic killing of <span class="yellow">S. faecium</span> in vitro [6]. This failure of synergy occurs even when high-level resistance to these aminoglycosides is not present. The mechanism of resistance appears to be related to the production of an inactivating enzyme that acetylates the aminoglycoside substrate at the 6' position1 [14]. The genetic basis for production of this enzyme has not been well-defined and plasmid transfer experiments have thus far been unsucessful in demonstrating the encodement of this enzyme by extrachromosomal DNA [14]. <br><br> Moellering et al. demonstrated in vivo, utilizing the <span class="yellow">rabbit</span> model of endocarditis, that penicillin and netilmicin were not efficacious in the treatment of endocarditis caused by a low-level aminoglycoside-resistant strain of <span class="yellow">S. faecium</span> [6]. Although combinations of penicillin with tobramycin, kanamycin, or sisomicin were not evaluated, the authors postulated that the same ineffectual result would have occurred. In the present case, the recurrence of <span class="yellow">S. faecium</span> endocarditis after six weeks of therapy with ampicillin and tobramycin confirms the therapeutic and clinical significance of their data, and emphasizes that tobramycin is not an aminoglycoside to be used for treatment of serious <span class="yellow">S. faecium</span> infections. <br><br> Bacterial tolerance has been suggested as a possible basis for therapeutic failures, particularly in the treatment of infections caused by <span class="yellow">Staphylococcus aureus</span> with defects in the production of autolysins. MBCs are generally several-fold higher than MICs and this phenomenon appears to be associated with the autolysin defect [5]. Lorian has also described the formation of numerous aberrant cross-walls by <span class="yellow">S. faecalis</span> grown in the presence of subinhibitory concentrations of penicillin [15]. MBCs were only slightly higher than MICs for the three <span class="yellow">S. faecium</span> variants obtained from our <span class="yellow">patient</span>, and they did not appear to be tolerant strains of <span class="yellow">S. faecium</span>. There was no evidence that any of the morphological variants isolated were unusually resistant to antibiotics. Therefore, subsequent failure of therapy could not be explained on the basis of antibiotic resistance patterns. The initial failure of ampicillin and tobramycin therapy may have allowed the infecting organism to become better established and more difficult to eradicate from the valve. Alternatively, a <br><br> 'Wennersten C, Moellering RC Jr: Mechanism of resistance to penicillin-aminoglycoside synergism in <span class="yellow">Streptococcus faecium</span>. Proceedings of the 1th International Congress of Chemotherapy and 19th Interscience Conference on Antimicrobial Agents and Chemotherapy 1:710-711, 1979 <br><br> 247 <br><br> 248                             GOLDSTEIN ET AL. <br><br> persistent focus of infection causing the LAD lesion seen by arteriography may have slowly resolved and accounted for the apparent failure to respond to synergistic combinations of antibiotics. <br><br> In summary, our <span class="yellow">patient</span> was treated for <span class="yellow">Streptococcusfaecium</span> endocarditis with both ampicillin and the aminoglycoside antibiotic, tobramycin. Relapse of endocarditis might have been anticipated on the basis of previous experimental data showing lack of synergy when tobramycin is used against this organism. This case graphically illustrates the relevance of synergy studies to therapeutic considerations in the treatment of endocarditis. However, subsequent therapy with synergistic combinations of antibiotics did not result in cure. The data do not implicate tolerant organisms as a cause for relapses. Failure of therapy may have been related to the presence of protected organisms in vegetations which were seen on echocardiograms, and also suggested by the presence of a possible coronary artery embolus seen on coronary arteriograms. <br><br> In conclusion, speciation of isolates suspected of causing endocarditis and adequate synergy studies of antibiotic combinations are indicated before a long and expensive course of therapy with antimicrobial agents is undertaken for this disease. However, as this case also illustrates, demonstration of synergism in vitro does not assure clinical cure. <br><br> ACKNOWLEDGEMENTS <br><br> Dr. Goldstein was supported by training grant Al 07033-05 from the National Institute of Allergy and Infectious Diseases. <br><br> The authors gratefully acknowledge the advice, laboratory assistance, and careful review of this manuscript by Dr. Robert C. Moellering, Jr. <br><br> The authors also wish to thank Dr. Howard S. Forster for referring this <span class="yellow">patient</span> to us, Ms. Mary Murray and Deborah Beauvais for manuscript preparation, and Ms. Gertrude Barden, MT (ASCP), MHS, for technical assistance and advice in evaluation of this <span class="yellow">patient</span>. <br><br> REFERENCES <br><br> 1. Hunter TH: Use of streptomycin in the treatment of bacterial endocarditis. Am J Med 2:436-442, <br><br> 1947 <br><br> 2. Moellering RC Jr, Krogstad DJ: Antibiotic resistance in enterococci. Microbiology 293-298, 1979 3. Facklam RR: Recognition of group D streptococcal species of <span class="blue">human</span> origin by biochemical and <br><br> physiological tests. Appl Microbiol 23:1131-1139, 1972 <br><br> 4. Toala P, McDonald A, Wilcox C, et al: Susceptibility of group D streptococcus (enterococcus) to 21 <br><br> antibiotics in vitro, with special reference to species differences. Am J Med Sci 258:416-430, 1969 5. Shungu DL, Cornett JB, Shockman GD: Morphological and physiological study of autolytic<br><br> defective <span class="yellow">Streptococcus faecium</span> strains. J Bacteriol 138:598-608, 1979 <br><br> 6. Moellering RC Jr, Korzeniowski OM, Sande MA, et al: Species-specific resistance to antimicrobial <br><br> synergism in <span class="yellow">Streptococcusfaecium</span> and <span class="yellow">Streptococcusfaecalis</span>. J Infect Dis 140:203-208, 1979 <br><br> 7. Moellering RC Jr, Weinberg AN: Studies on antibiotic synergism against enterococci. 11. Effect of <br><br> various antibiotics on the uptake of '4C-labelled streptomycin by enterococci. J Clin Invest 50:2580-2584, 1971 <br><br> 8. Zimmermann RA, Moellering RC Jr, Weinberg AN: Mechanisms of resistance to antibiotic <br><br> synergism in enterococci. J Bacteriol 105:873-879, 1971 <br><br> 9. Moellering RC Jr, Murray BE, Schoenbaum SC, et al: A novel mechanism of resistance to penicillin<br><br> gentamicin synergism in <span class="yellow">S. faecalis</span>. J Infect Dis 141:81-86, 1980 <br><br> 10. Krogstad DJ, Korfhagen TR, Moellering RC Jr, et al: Plasmid-mediated resistance to antibiotic <br><br> synergism in enterococci. J Clin Invest 61:1645-1653, 1978 <br><br> 11. Krogstad DJ, Korfhagen TR, Moellering RC Jr, et al: Aminoglycoside-inactivating enzymes in <br><br> clinical isolates of <span class="yellow">Streptococcusfaecalis</span>: An explanation for resistance to antibiotic synergism. J Clin Invest 62:480-486, 1978 <br><br> <span class="yellow">S. FAECIUM</span> ENDOCARDITIS                 249 <br><br> 12. Courvalin PM, Shaw WV, Jacob AE: Plasmid-mediated mechanisms of resistance to <br><br> aminoglycoside-aminocyclitol antibiotics and to chloramphenicol in group D streptococci. Antimicrob Agents Chemother 13:716-725, 1978 <br><br> 13. Calderwood SA, Wennersten C, Moellering RC Jr, et al: Resistance to six aminoglycosidic <br><br> aminocyclitol antibiotics among enterococci: Prevalence, evolution, and relationship to synergism with penicillin. Antimicrob Agents Chemother 12:401-405, 1977 <br><br> 14. Bisno AL: Treatment of infective endocarditis. New York, Grune and Stratton, 1981, p 90 <br><br> 15. Lorian V: Effects of subminimum inhibitory concentrations of antibiotics on bacteria. In Antibiotics <br><br> in Laboratory Medicine. Edited by V Lorian. Baltimore, Williams and Wilkins, 1980, p 342 </body></html>