
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>mouse</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="blue">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="blue">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1131934</h3>HP1 modulates the transcription of cell-cycle regulators in <span class="yellow">Drosophila melanogaster</span>
Abstract
Heterochromatin protein 1 (HP1) was originally described as a non-histone chromosomal protein and is required for transcriptional gene silencing and the formation of heterochromatin. Although it is localized primarily at pericentric heterochromatin, a scattered distribution over a large number of euchromatic loci is also evident. Here, we provide evidence that <span class="yellow">Drosophila</span> HP1 is essential for the maintenance of active transcription of euchromatic genes functionally involved in cell-cycle progression, including those required for DNA replication and mitosis. Depletion of HP1 in proliferating embryonic cells caused aberrant progression of the cell cycle at S phase and G2/M phase, linked to aberrant chromosome segregation, cytokinesis, and an increase in apoptosis. The chromosomal distribution of Aurora B, and the level of phosphorylation of histone H3 serine 10 were also altered in the absence of HP1. Using chromatin immunoprecipitation analysis, we further demonstrate that the promoters of a number of cell-cycle regulator genes are bound to HP1, supporting a direct role for HP1 in their active transcription. Overall, our data suggest that HP1 is essential for the maintenance of cell-cycle progression and the transcription of cell-cycle regulatory genes. The results also support the view that HP1 is a positive regulator of transcription in euchromatin.<br><br>INTRODUCTION
Chromatin in higher eukaryotes is subdivided into different functional compartments termed heterochromatin and euchromatin (1). Heterochromatin differs from euchromatin in its DNA composition, replication timing, condensation throughout the cell cycle, and its ability to silence euchromatic genes placed adjacent to or within its territory, often described as position-effect-variegation (PEV) (2).
Heterochromatin protein 1 (HP1) was the first protein identified in <span class="yellow">Drosophila melanogaster</span> as a heterochromatin-associated protein (3); the corresponding gene has been cloned from a number of organisms and is highly conserved from <span class="yellow">yeast</span> to <span class="yellow">human</span> (4). Polytene chromosome staining showed that, in <span class="yellow">Drosophila</span>, HP1 is distributed mainly in pericentric heterochromatin, telomeric heterochromatin, the banded small fourth chromosome (5–8), as well as ∼200 individual loci scattered throughout the euchromatic chromosomal arms (5). The gene encoding HP1 in <span class="yellow">D.melanogaster</span>, Su(var)2-5, was isolated as a suppressor of PEV (9–11). The protein contains a highly conserved motif, the chromo (chromatin organization modifier) domain, similar to Polycomb (Pc), a repressor of homeotic genes (12). The association between HP1 and pericentric heterochromatin is believed to occur via the chromo domain of HP1 and the N-terminal tail of histone H3 methylated at lysine 9 (13,14), generated by histone methyltransferase-Su(var)3-9, a partner of HP1 in pericentric heterochromatin (15). The C-terminal chromo ‘shadow’ domain of HP1 interacts with other silencing complexes to suppress local transcriptional activity (15–18). However, studies of HP1 chromosomal distribution also showed that HP1 does not always co-localize with lysine 9 methylated histone H3 or Su(var)3-9, especially in euchromatic regions (19–21); in some cases, HP1 is found directly bound to DNA (22,23). All these features argue for distinct roles for HP1 in chromatin and in epigenetic gene regulation.
HP1 is believed to be an essential structural protein protecting the integrity of chromosomes during cell division (8,24). Swi6, the homolog of HP1 in <span class="yellow">fission yeast</span>, is dispensable for survival, but its deletion results in lagging chromosomes during anaphase, and a high rate of chromosome loss (25,26). Mutations of HP1 in <span class="yellow">D.melanogaster</span> result in late larval lethality, chromosome breakages/loss, telomere fusion and a high frequency of cells with abnormal anaphase (8,27). Null alleles of the HP1 functional partner in <span class="yellow">mice</span> (SUVAR39) also showed various chromosomal defects (28), supporting a conserved role for heterochromatin proteins in the regulation of chromosome dynamics during cell-cycle progression. However, the mechanism(s) involved remains to be understood.
In this study, we utilized <span class="yellow">Drosophila</span> embryonic Kc cells and an RNA interference (RNAi)-based approach to demonstrate that HP1 plays an important role at S phase and G2/M phases during the cell cycle. We further show that nearly one-third of known/predicted cell-cycle regulators require HP1 to maintain their active transcription. These genes include MCMs, Orc4, CDC45L, INCENP, Aurora B, CAF1, Bub1, Bub3 and a few other cell-cycle regulators. ChIP analysis suggests that HP1 plays a direct role in their transcription. Therefore, the results of this study provide an alternative explanation for the specific role of HP1 in the regulation of chromatin dynamics and in cell-cycle progression.<br><br>MATERIALS AND METHODS
RNAi in Kc cells
<span class="yellow">Drosophila</span> Kc cells were routinely cultured at 25°C in Schneider <span class="yellow">Drosophila</span> medium (GIBCO) supplemented with 10% fetal <span class="yellow">calf</span> serum, 160 μg/ml penicillin, 250 μg/ml streptomycin, and 4 mM l-glutamine. Double-stranded RNA (dsRNA) of HP1 was generated by incubation of single-stranded RNA in annealing buffer (100 mM potassium acetate, 30 mM HEPES-KOH, pH 7.4, 2 mM magnesium acetate) for 3 min at 95°C and then placed in a beaker with water at 75°C and allowed to cool slowly to room temperature. The detailed procedure of RNAi was carried out according to the established protocols (). Briefly, Kc cells were seeded in a six-well dish using serum-free medium at 1 × 106 cells/ml. HP1 dsRNA (5 μg/ml) was added to the cultured Kc cells. After 60 min at room temperature, 2 ml of medium containing 10% serum was added to each well and the plates transferred to 25°C for up to 8 days. Western blotting and RT–PCR were carried out using the extract/total RNA isolated from control and dsRNA-treated cells on days 2, 6 and 8.<br><br>Cell-cycle and apoptosis analysis
The procedure for flow cytometric analysis of Kc cells followed that in the manual provided with the BrdU flow kit (BD PharMingen). The cells were fed with BrdU for 4 h, then scraped and collected. Fluorescence was measured using a FACSCalibur (Becton Dickinson). Data collection and analysis were performed using CellQuest software.<br><br>Electrophoresis and immunoblotting
Cell extracts (15 μg) were fractionated by 10% SDS–PAGE, then transferred to Hybond-P PVDF membranes (Amersham) and probed with primary antibodies (CIA9), and secondary antibodies (anti-<span class="yellow">rabbit</span> or anti-<span class="blue">mouse</span> <span class="yellow">horseradish</span> peroxidase-conjugated IgG), obtained from Jackson Immunoresearch Laboratories. Enhanced chemiluminescence reagents (Amersham Pharmacia Biotech) were used for signal detection.
For the analysis of H3 ser10 phosphorylation, we used whole-cell extracts from 700 000 Kc cells (control and RNAi at day 8). Western blotting was performed using polyclonal antibodies against ser10-phosphorylated histone H3 at a dilution of 1:1000 (Upstate). Kc control cells arrested in mitosis by incubation in 25 µM colchicine (Sigma) for 24 h were also analyzed for comparison.<br><br>Immunofluorescence
Kc cells were seeded onto polylysine slides, fixed with 4% formaldehyde for 15 min and permeabilized with 0.5% Triton X-100 for 5 min. The incubation with primary antibodies was carried out in blocking solution for 1 h.
For staining of mitotic cells, the cells were permeabilized using PBST (PBS containing 0.3% Triton X-100) and stained with polyclonal antibody against <span class="yellow">Drosophila</span> Aurora B at 1:200 dilution and monoclonal <span class="blue">mouse</span> at anti-β-tubulin 1:300 dilution (Chemicon International) as primary antibodies. Secondary antibodies were anti-<span class="yellow">rabbit</span> coupled with Alexa 488 (1:500) and anti-<span class="blue">mouse</span> coupled to Alexa 546 (1:500) (Molecular Probes, Eugene, Oregon). Images were acquired using a confocal LSM510 META microscope (Zeiss). Stacks of images were analyzed using the IMARIS 4.0 program (Media cybernetics, Carlsbad, CA).<br><br>Antibodies
Affinity-purified polyclonal antibodies of HP1 (<span class="yellow">rabbit</span> #192 and #187, 5 μg) and 5 μg of polyclonal anti-HA antibodies (Sigma) were used in each ChIP reaction. The specificity of the HP1 polyclonal antibodies was determined using various approaches, including western blotting assay, immunofluorescence staining and immunoprecipitation to pull down HP1 (data not shown). The monoclonal antibody HP1–CIA9 (5) was used at a dilution of 1:20 in immunoblotting assays.<br><br>Microarray analysis and RT–PCR
Total RNA was isolated from control and HP1-depleted Kc cells at day 8 using an RNeasy kit (Qiagen). RNA labeling and microarray data analysis followed the standard protocol from Affymetrix. We used ANOVA (P < 0.001) to assess the expression confidence for each gene.
For RT–PCR analysis, poly(A)+ mRNA was purified with the Oligotex Direct mRNA kit (Qiagen) according to the manufacturer's instructions. The purified poly(A)+ RNA was reverse transcribed using the Thermoscript kit (Invitrogen). The cDNA was then used for PCR amplification for 35 cycles with gene-specific primers. PCR products were scanned after electrophoretic separation with a Typhoon Scanner, quantified using ImageQuant software (Amersham Biosciences) and normalized for amplification of the Actin5c transcript. The sequence of primers used for RT–PCR and ChIP analysis are provided in the Supplementary Material.<br><br>ChIP
ChIP was performed according to Orlando et al. (29) and the protocol provided by Upstate () with some modifications. In brief, 1–2 × 108 Kc cells were prepared and fixed in 1% formaldehyde. Nuclei were isolated according to a standard procedure in Current Protocols (), then resuspended in 1.7 ml of lysis buffer (50 mM Tris, pH 8.0, 10 mM EDTA, 1% SDS and protease inhibitors) and sonicated using a Branson sonifier 250. Chromatin fractions in the size range 0.2–0.8 kb were used to perform immunoprecipitation experiments. We used 5 μg affinity-purified polyclonal antibodies (#192 and #182 for HP1; HA antibody for control) and 1 ml of <span class="yellow">salmon</span> sperm DNA/protein-A-agarose (Upstate) pre-cleared chromatin lysate in each reaction. The mixture was then rotated at 4°C overnight and the recovered beads were washed twice with 1 ml of Low salt buffer (Upstate), once with High salt buffer (Upstate), once with LiCl buffer (Upstate) and twice with TE at 4°C for 8 min. ChIP DNA was extracted according to the standard procedures (29).<br><br>
RESULTS
Depletion of HP1 in <span class="yellow">Drosophila</span> Kc cells
Various chromosomal defects in the cell cycle have been observed in embryos or larval tissues of <span class="yellow">Drosophila</span> HP1 mutants (8,27). However, the presence of maternally loaded HP1 in embryos and the lethality of HP1 mutants at late larval stages have so far precluded a systematic study of the role of HP1 in cell-cycle regulation. Therefore, we used <span class="yellow">Drosophila</span> Kc cells, a cell line derived from <span class="yellow">Drosophila</span> embryos, as a model system to address this problem. HP1 transcripts were depleted using an RNAi-based approach (see Materials and Methods). The reduction in HP1 expression was measured both by RT–PCR and by western blotting analysis (Figure 1A). A significant reduction in the HP1 expression was already evident after 2 days treatment with HP1 dsRNA. Cells at day 8 showed a reduction in HP1 of ∼90% (Figure 1A) and were therefore used in all subsequent experiments.<br><br>Cell-cycle progression at S and G2/M phase is altered in the absence of HP1
The impact of HP1 loss on the cell cycle of Kc cells was determined using cell-cycle profile analysis of HP1-depleted and control cells. The percentage of cells in S phase was determined by BrdU incorporation, and total DNA content by 7-amino-actinomycin (7-AAD). The results showed that the depletion of HP1 (day 8) caused a decrease in S-phase cells of at least 4-fold, and a 2-fold decrease in G2/M-phase cells (Figure 1B), although no significant effect was found at the G1 phase. In addition, depletion of HP1 caused a greater than 7-fold increase in the number of apoptotic cells. These results, therefore, confirm that HP1 is an important regulator during the cell cycle, especially at the S and G2/M phases.<br><br>Cell-cycle regulators require HP1 to maintain their active transcription
To ask whether the cell-cycle defects were due to changes in the transcription of genes functionally involved in S phase and the G2/M phase, we next assessed global changes in gene transcription following depletion of HP1. Expression profile analysis was performed using total RNA isolated from both HP1-depleted Kc cells and control Kc cells, and an Affymetrix <span class="yellow">Drosophila</span> chip. For each experiment, we used total RNA isolated from two independent HP1-depleted and control samples, and at least two independent experiments were performed.
The microarray analysis showed that loss of HP1 function in Kc cells resulted in alterations in transcription of >500 genes: ∼400 genes were down-regulated and ∼120 genes were up-regulated (>1.5-fold, ANOVA). The function of these genes ranged from cellular enzymes, signal transduction molecules, and membrane and cell structural proteins, to nucleic acid-binding proteins and cell-cycle regulators (Figure 2A). At the chromosomal level, the genes targeted by HP1 appeared to be distributed along all euchromatic chromosomal arms (data not shown), supporting a global role of HP1 in euchromatic gene regulation (20).
Among 60 known/predicted genes associated with DNA replication function, 15 were down-regulated in the absence of HP1 (Figure 2B). These included McM2, McM5, McM6 and CDC45L, which are required for processive DNA replication and correct chromosome condensation (30–32). Other genes involved in DNA replication, such as components of the origin recognition complex (Orc)—Orc4, Caf1, Gnf1, Dref1, DNA polymerase-γ and Tam—were also down-regulated (Figure 2B). Aurora B and inner centromere protein (INCENP), known to be required for kinetochore assembly, chromosome condensation and bipolar chromosome attachment during mitosis (33), also showed a reduction in transcription. A similar loss of transcription was observed in Bub1 and Bub3 (Figure 2B), encoding mitotic checkpoint control proteins (34,35). Loss function of Bub1 has been shown to cause chromatin bridges to extend between the two separating groups of chromosomes, and extensive chromosome fragmentation in anaphase cells (35).
We confirmed the changes in the transcription of cell-cycle regulators using semi-quantitative RT–PCR, which gave results consistent with the microarray analysis. In addition, cell-cycle regulator genes, such as McM3, McM7 and Asp (abnormal spindle), were also confirmed to be down-regulated (Figure 2C). Collectively, these results demonstrate that HP1 is indeed involved in the regulation of transcription of cell-cycle regulators.<br><br>HP1 is required for Aurora B distribution and histone H3 phosphorylation
INCENP is localized to the centromeric region of chromosomes at metaphase and the spindle midzone at anaphase, which then targets Aurora B, a kinase essential for histone H3 ser10 phosphorylation, to these sites (36). Loss of function of both these ‘chromosomal passenger’ proteins causes abnormal chromosomal segregation at metaphase, as well as certain cytokinesis defects (36,37). The loss of transcription of both INCENP and Aurora B after depletion of HP1, therefore, raised the possibility that localization of Aurora B (Figure 3 and data not shown) may be altered. Staining of HP1-depleted Kc cells with anti-Aurora B antibodies indeed revealed an altered localization of Aurora B and, in a number of cases, a complete loss of Aurora B (Figure 3A). Consistent with the loss function of Aurora B, the spindles in the metaphase cells were also disorganized, with a large number of cells showing an altered prometaphase chromosome alignment (Figure 3A). Some showed extensive chromosome fragmentation (Figure 3B), or the presence of a third spindle pole-like structure as indicated by beta-tubulin (Figure 3A). At telophase, we observed defective separating cells with an extra cell envelope-like structure without nuclei (Figure 3A). Chromatin bridges or lagging chromatids at telophase were also evident in some cells (Figure 3C); however, in some cases, localization of Aurora B appeared not to be affected, arguing that other pathways are possibly involved.
We next analyzed changes in histone H3 serine 10 phosphorylation, since the loss of transcription of INCENP is known to affect localization of Aurora B (33), which is essential in the regulation of histone H3 phosphorylation (36). Total cell extracts from HP1-depleted Kc cells were analyzed by western blotting (Figure 3D). The results indeed showed a severalfold reduction in H3 ser10 phosphorylation after depletion of HP1, consistent with the functional disruption of INCENP and Aurora B in the absence of HP1.<br><br>HP1 directly targets genes encoding cell-cycle regulators in euchromatin
To test whether the loss of transcription of genes involved in DNA replication and mitosis was a direct effect of the loss of HP1, we performed a ChIP analysis to determine whether HP1 is physically associated with these genes. Chromatin lysates from formaldehyde-fixed Kc cells were sonicated into small chromatin fragments (0.2–0.8 kb) and immunoprecipitated with polyclonal antibodies against <span class="yellow">Drosophila</span> HP1. As a control, we used a mock precipitation (beads only) and polyclonal antibodies against HA. Our ChIP results showed that known transposable elements distributed in heterochromatin, such as F-element, TART and 1360 (7,38), were all enriched in HP1 binding (Figure 4), which is also consistent with a previous study (20).
Using the same ChIP DNA material, we then attempted to determine whether HP1 was enriched in genes involved in DNA replication. Primers were designed to cover the promoter regions of selected genes. The results showed that McM3, McM5 and Tam were all enriched in HP1 binding (Figure 4). However, McM7 appeared to be HP1-negative, although its transcription was also affected by the loss of HP1 function. Genes essential for mitosis, such as Aurora B, were also HP1-positive (Figure 4). These results demonstrate that these cell-cycle regulator genes are directly targeted by HP1 in their promoter regions.<br><br>
DISCUSSION
In this study, we used microarray and RT–PCR techniques to demonstrate that transcription of cell-cycle regulators is misregulated in the absence of HP1. Certain defects in S phase may be a direct consequence of the loss of transcription of DNA replication genes such as McM2, McM5, McM6, CDC45L, Orc4 and others, since these genes have been functionally implicated in the initiation of DNA replication and/or the progression of replication forks (39). Depletion or mutation of these genes has been shown to result in DNA damage (32), the blockage of replication forks (39), increased chromosome loss/genome instability, and defective condensation (30).
The reduction in the number of cells in G2/M phase may be a consequence of the reduction in transcription or functional disruption of INCENP, Aurora B, Bub1, and Bub3 (34,36). Chromosome segregation defects, such as chromosome fragmentation and chromatin bridges in anaphase/telophase cells, and certain cytokinesis defects in HP1-depleted cells, mimic the phenotype of cells with loss function of INCENP, Aurora B or Bub1 (35–37). The mislocalization of Aurora B in the absence of HP1 is also consistent with the loss of transcription and functional disruption of INCENP (37), and the reduction in Aurora B transcription may be partially responsible for the observed chromosomal defects, including loss of histone H3 phosphorylation at serine 10.
HP1 is also known to physically interact with certain components of replication complexes such as ORCs and MCMs (30,40,41), with the inner centromere protein INCENP (42) and the chromatin assembly factor CAF1 (43) promoting delivery of HP1 to heterochromatin sites (44). Loss of HP1 is, therefore, expected to cause disruption to such HP1-associated complexes, and will partially contribute to the chromatin/chromosomal defects in HP1 mutants (8,27) and HP1-depleted Kc cells. It is therefore well possible that the loss of transcription of these cell-cycle regulator genes, and consequent disruption of HP1 functional complexes or heterochromatin structure, all contributed to the cell-cycle defects observed.
The ChIP assay supports the hypothesis that the loss of transcription of cell-cycle regulator genes is a direct effect of the lack of HP1. Aurora B, McM3, and McM5 were all bound by HP1 at their promoter regions, although other cell-cycle regulators, such as McM7, were HP1-negative, implying that the altered transcription in these genes might be a secondary effect of the loss of HP1.
A previous study in <span class="yellow">Drosophila</span> Kc cells (20) employed an approach based on the ectopic expression of a fusion protein of HP1 with a prokaryotic DNA adenine methyltransferase and identified a number of methylated targets in the genome. In this study, MCM3 and MCM5 were not found to be methylated, indicating lack of association with HP1. On the other hand, heterochromatin repeats, such F-element and 1360, were consistently found to be HP1-enriched both here and in the previous study. It remains to be determined whether these discrepancies are due to the different experimental systems used. However, we note that the previous study was performed using a cDNA array, while we observe binding of endogenous HP1 at the promoter of these genes. Similarly, another study using chromatin immunoprecipitation in larvae also showed few HP1-positive genes that were not detected in Kc cells by the Dam ID approach (21).
A large number of genes affected by the loss of <span class="yellow">D.melanogaster</span> HP1 in larval tissues (21) seem to be different from that in embryonic Kc cells. The change in the transcription of Aurora B and few cell-cycle regulators reported in this study is also not found among the HP1-affected genes at larval stage (21). This may be due to specific role(s) of HP1 in different stages of development. Alternatively, it is also possible that the impact of HP1 in the transcription of cell-cycle regulators in proliferating cells is underestimated when performing the analyses on larval tissues, and thus on mixed populations of both proliferating and differentiating/differentiated cells.
HP1 is generally known as a transcriptional repressor, as supported by several lines of evidence: silencing of a euchromatic reporter gene in heterochromatin requires HP1 (10,11), tethering of HP1 next to a euchromatic reporter gene causes silencing (45), and the repression of genes within euchromatic region 31 bound by HP1 is relieved in the absence of HP1 (46). In contrast, genes in heterochromatin, known as heterochromatic genes, such as light and rolled, seem to require HP1 to maintain their active transcription (47,48). The level of transcription of heterochromatic genes was dramatically reduced in a mutated HP1 background (47–49). It was therefore proposed that HP1 may function as a positive regulator of transcription of these genes (50,51), although the exact regulation mechanism remains unclear. A study of heat-shock genes found that HP1 is associated with RNA transcripts in the coding region, and is also a positive regulator of their transcription (52,53). The chromatin association of HP1 at the promoter region of active euchromatic genes demonstrated from this work and others, and its independence from histone H3K9 methylation (21), all suggest that mechanism whereby HP1 modulates transcription of euchromatic genes is potentially distinct from its role in heterochromatin formation.
Collectively, the results of this study demonstrate that HP1 plays an essential role in cell-cycle progression, and support the view that HP1, in addition to its role in heterochromatin, can act as a positive transcriptional regulator of euchromatic genes.<br><br>SUPPLEMENTARY MATERIAL
Supplementary Material is available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA1190194</h3>Structural organization and interactions of transmembrane domains in tetraspanin proteins
Abstract
Background
Proteins of the tetraspanin family contain four transmembrane domains (TM1-4) linked by two extracellular loops and a short intracellular loop, and have short intracellular N- and C-termini. While structure and function analysis of the larger extracellular loop has been performed, the organization and role of transmembrane domains have not been systematically assessed.<br><br>Results
Among 28 <span class="yellow">human</span> tetraspanin proteins, the TM1-3 sequences display a distinct heptad repeat motif (abcdefg)n. In TM1, position a is occupied by structurally conserved bulky residues and position d contains highly conserved Asn and Gly residues. In TM2, position a is occupied by conserved small residues (Gly/Ala/Thr), and position d has a conserved Gly and two bulky aliphatic residues. In TM3, three a positions of the heptad repeat are filled by two leucines and a glutamate/glutamine residue, and two d positions are occupied by either Phe/Tyr or Val/Ile/Leu residues. No heptad motif is apparent in TM4 sequences. Mutations of conserved glycines in <span class="yellow">human</span> CD9 (Gly25 and Gly32 in TM1; Gly67 and Gly74 in TM2) caused aggregation of mutant proteins inside the cell. Modeling of the TM1-TM2 interface in CD9, using a novel algorithm, predicts tight packing of conserved bulky residues against conserved Gly residues along the two helices. The homodimeric interface of CD9 was mapped, by disulfide cross-linking of single-cysteine mutants, to the vicinity of residues Leu14 and Phe17 in TM1 (positions g and c) and Gly77, Gly80 and Ala81 in TM2 (positions d, g and a, respectively). Mutations of a and d residues in both TM1 and TM2 (Gly25, Gly32, Gly67 and Gly74), involved in intramolecular TM1-TM2 interaction, also strongly diminished intermolecular interaction, as assessed by cross-linking of Cys80.<br><br>Conclusion
Our results suggest that tetraspanin intra- and intermolecular interactions are mediated by conserved residues in adjacent, but distinct regions of TM1 and TM2. A key structural element that defines TM1-TM2 interaction in tetraspanins is the specific packing of bulky residues against small residues.<br><br><br><br>Background
Tetraspanins constitute a large family of integral membrane proteins, characteristically containing 4, 6 or 8 conserved cysteine residues in the large extracellular loop (including the CCG and PxxCC motifs), which form disulfide bonds, and several conserved polar residues in the intracellular loop and transmembrane regions [1,2]. There are 32 putative tetraspanin family members in mammals, 37 in <span class="yellow">Drosophila melanogaster</span> and 20 in <span class="yellow">Caenorhabditis elegans</span>. Tetraspanins play diverse roles in cell adhesion, migration and fusion processes, cellular activation and signaling (reviewed in refs. [2-4]). Mammalian tetraspanins such as CD9, CD63, CD81, CD82, CD151, rds/peripherin, and uroplakins Ia and Ib have been most extensively studied, with <span class="blue">mouse</span> knock-out models available for CD9 [5-7], CD81 [8,9], CD151 [10] and a few others. However, the majority of tetraspanins are characterized very little, if at all, at genetic, biochemical or structural levels.
The large extracellular loop (LEL) of tetraspanins has received most attention, since it contains functionally important sites. Sequence QRD (194–196) in CD151 is important for association with integrins, which has functional consequences for integrin-dependent cell spreading and multicellular cable formation [11]. A site in the LEL of CD9, SFQ (residues 173–175), is essential for CD9 function in sperm-egg fusion [12]. The crystal structure of tetraspanin CD81 LEL revealed five α-helixes, A-E [13]. Helices A, B and E form a relatively conserved region in tetraspanins, whereas the region between helices B and E is the most variable [14]. Interestingly, the variable region contains most of the functionally important sites involved in tetraspanin protein-protein interactions.
A remarkable biochemical property of tetraspanin molecules is their ability to associate with a large number of other transmembrane proteins, including integrins, membrane-associated growth factors and receptors, MHC class II molecules, Ig superfamily proteins, and each other [2,3,15]. Several of these lateral associations of tetraspanins are detected in "mild" detergents (Brij series, CHAPS), but are disrupted by "strong" detergents such as Triton X-100 or SDS. Multiprotein complexes of tetraspanins and associated molecules, also called the "tetraspanin web" [16], may represent a distinct tetraspanin-enriched membrane microdomain [17,18]. The formation of this microdomain is influenced by palmitoylation of several conserved juxtamembrane cysteine residues in tetraspanins [19-21].
The transmembrane domains, encompassing nearly half of a tetraspanin protein, are the most conserved part of the molecule (Stipp et al. [1] and this study). However, very little functional information is available on these domains. The differential detergent sensitivity of tetraspanin-tetraspanin associations suggests that hydrophobic interactions between TM helices may play a role. Indeed, when the large extracellular loop (LEL) of CD151 is deleted, the molecule is still able to associate with other tetraspanins [22]. Thus, TM domains are strong candidates for mediating tetraspanin-tetraspanin interactions.
The importance of TM domain interactions in intramolecular organization was demonstrated in a study showing that CD82 fragment TM2-4, lacking TM1, was retained in the endoplasmic reticulum, but was transported to the cell surface upon co-expression of TM1 [23]. This in vivo reconstitution experiment demonstrated a strong interaction between TM1 and the rest of the molecule. Expression of a truncated CD9 molecule (TM3-LEL-TM4) results in intracellular accumulation of the protein and significant misfolding of the LEL, as judged by inappropriate disulfide formation and diminished antibody reactivity (our unpublished data). Similarly, a CD9 epitope in the LEL is lost in molecules lacking either TM2+TM3 or just TM4 [24]. Thus, TM domain interactions and packing are crucial for proper folding, stability and transport of tetraspanin molecules.
In a previous study, we showed that covalent cross-linking of membrane-proximal cysteine residues can be used as a tool for detection of tetraspanin-tetraspanin associations [25]. Inhibition of cysteine palmitoylation by 2-bromopalmitate (2-BP) made cysteines available for cross-linking and enabled demonstration of specific tetraspanin homodimerization and low levels of heterodimerization. We concluded that tetraspanin homodimers, formed in the Golgi, may be a fundamental structural unit within tetraspanin microdomains.
In this study, we carried out detailed sequence analysis of <span class="yellow">human</span> tetraspanin TM domains. We show that a heptad repeat containing conserved glycine, asparagine and large hydrophobic residues occurs in TM1 and TM2 domains, and predict tight intramolecular association of these two domains by packing of the large residues against the small residues. Moreover, by using cysteine cross-linking we map a dimerization interface in the <span class="yellow">human</span> CD9 protein, and show that conserved heptad motif glycine residues are also important for intermolecular CD9 associations.<br><br>Results
Sequence analysis of tetraspanin transmembrane domains: presence of the heptad repeat motif
We focused our attention on 28 <span class="yellow">human</span> tetraspanins identified from the SWISS-PROT and GenBank databases. All tetraspanins have in common four hydrophobic stretches (TM domains) of 20–25 residues, and contain highly conserved residues in the second extracellular loop, in particular the Cys-Cys-Gly (CCG) motif. Detailed analysis of the large extracellular loop sequences [14], and dendrograms based on full-length alignment can be found in earlier studies [26,27]. The length of each transmembrane domain was established based on previous sequence analysis of tetraspanin sequences [27,28], and on annotations to the database entries. Manual adjustments based on sequence homology and hydrophobicity profiles were done to fully delineate the TM domains. The resulting lengths of TM domains were: TM1 – 23 residues; TM2 – 21 residues; TM3 – 25 residues; TM4 – 25 residues. Two more residues could be added onto the N-terminal part of TM2; however, relatively small sequence conservation of these residues among tetraspanins and occurrence of polar/charged side chains in some tetraspanins precluded us from doing so for the global alignment.
Figures 1 and 2 show a multiple sequence alignment of four TM domains of 28 <span class="yellow">human</span> tetraspanins. For each position within the domains, consensus residues were determined and classified (with individual color code) in 4 categories: 1) large hydrophobic residues (including Val, Met, Leu, Ile, Phe, Tyr, Trp), 2) small residues (Gly, Ala, Ser and Thr), 3) Cys, and 4) Asn. When more than two types of residues occupied a given position in a TM, a dual-color pattern that reflected the prevalence of the particular residue type was used (Figure 1). Cysteine residues were shown separately due to their importance as palmitoylation target sites. The highly conserved asparagine residue in TM1 was considered separately. No proline residues are found in TM domains 1–3 of <span class="yellow">human</span> tetraspanins.
An inspection of the multiple sequence alignment reveals a repeating heptad amino acid pattern, (abcdefg)n, in TM1, 2 and 3 (Figure 1, 2). Heptad repeats promote helical coiled coil interactions in multiple soluble and membrane-spanning proteins [29-31]. In the heptad repeat, hydrophobic residues in positions a and d are of special importance, as they directly mediate interhelical contacts by creating a tight knobs-into-holes packing in the coiled coil structure [32]. For instance, in the leucine zipper of the <span class="yellow">yeast</span> transcription factor GCN4, positions a and d contain Val and Leu residues, respectively, with an Asn residue in a single a position forming a hydrogen bond across the GCN4 dimer interface [33].
In TM1 of tetraspanins, highly conserved Asn, Gly and Gly residues (numbers 18, 25 and 32 in the CD9 sequence) appear at d positions of the heptad repeats, and residues 14, 21 and 28 are at a positions (Figure 1). In TM2, residues 67, 74 and 81 (consensus Gly, Gly and Ala, respectively) occupy a positions, whereas residues 63, 70 and 77 are at d positions. Another highly conserved glycine, Gly80, occupies the 3rd g position in TM2. In TM3, the conserved pattern consists of two leucine residues (Leu89 and Leu96) and a glutamate/glutamine residue (Glu/Gln103) in a positions (Figure 2). Two d positions are also conserved – Phe/Tyr92 and Ile/Val/Leu99. TM4 lacks a conserved heptad pattern and has only a single conserved position, Glu/Gln209 (with four exceptions). These features of TM1-4 of tetraspanins are displayed on helical wheel diagrams (Figure 3).<br><br>Analysis of TM1 sequences
The conserved Asn-Gly-Gly motif, occupying designated d positions of the heptad repeat, is the most prominent structural feature of TM1. We also compared sequences of CD9 orthologs from 10 different organisms (the most available for any tetraspanin) to gain further insight into conservation and variability of the TM1 sequence. As shown in Figure 4, positions a, d and g in TM1 are among the most conserved (0, 1 and 1 substitution, respectively), while interspecies variability tends to occur in other positions: b (5 substitutions), c (4 substitutions), e (4 substitutions) and f (4 substitutions). Thus, the positions typically involved in coiled coil interactions (a and d) are the most conserved.
When residues of TM1 are plotted as a helical wheel, additional structural features are revealed (Figure 3). There are highly conserved aliphatic and aromatic residues in the first three a positions of the heptad motif (Phe15, Trp22 and Leu29 in CD9), as well as in g positions (Leu14, Phe21, Val28 in CD9). The "ridges" formed by these bulky residues are flanking the "groove"-forming Gly residues of the Asn-Gly-Gly position d motif. In contrast, b, c, e and f positions show an overall higher variability among tetraspanins, as also seen in the comparison of CD9 orthologs described above.<br><br>Analysis of TM2 sequences
A landmark feature of TM2 in tetraspanins is the presence of highly conserved glycine residues (Gly67, 74, 77 and 80 in CD9, Figure 1). Other substitutions at these positions are almost exclusively small residues, such as Ala or Ser. In addition, Ala, Ser or Thr occupy position 81. This residue, together with Gly67 and Gly74, forms face a of the helix. Residue Gly77 (position d) is preceded by conserved, chiefly large hydrophobic residues on the same helical face (Leu63 and Met70 in CD9). Extremely conserved Gly80 falls into heptad position g (Figure 3). Among CD9 orthologs, heptad positions a and d are absolutely conserved, whereas other positions have the following number of substitutions: b – 3; c – 2; e – 1; f – 3; g – 1 (Figure 4). Two of the f position residues in TM2 (65 and 79) also show higher variability among different tetraspanins (Figures 1, 3). Cysteine residues are frequently found near the cytoplasmic end of TM2 helix at positions 78 and 79; these cysteines are likely to be palmitoylated.<br><br>Analysis of TM3 and TM4 sequences
The TM3 domain provides another example of the heptad repeat pattern. Position a is occupied by two highly conserved leucine and a glutamate/glutamine residue (Leu89, Leu96 and Glu/Gln103 in CD9). Furthermore, two d positions are conserved – Phe/Tyr92 (aromatic residue) and Ile/Val99 (β-branched aliphatic residue; Figures 2, 3). In addition, residue 100 in position e is generally Phe or Leu. Among CD9 orthologs, position a has 1 substitution, positions b, c and f each have 6, positions d and e each have 2, and g has 4. Thus, as for TM1 and TM2, positions a and d are among the most conserved, but overall TM3 has more variable positions than TM1 or TM2 (Figure 3). Less than half of TM3 sequences contain cysteine residues, and those tend to occur at the internal positions of the helix (Figure 2).
TM4 shows less conservation among various tetraspanin family members than the other TM domains (Figures 2, 3). The only highly conserved feature is the glutamate/glutamine residue in position 209. In addition, one or two cysteine residues can be found at the C-terminal end of TM4 in some tetraspanins (e.g. CD9, CD81, CD151), and many sequences contain additional polar residues (Arg, Lys, His, Asn, Gln). No conserved heptad motif was identified in TM4, as also confirmed by analysis of substitutions in CD9 orthologs (data not shown).<br><br>Mutational analysis of conserved glycine residues in TM1 and TM2
The conserved nature of the Asn and Gly residues in TM1 and TM2 prompted an analysis of their functional role. To this end, we have probed whether mutations of these residues destabilize the protein molecule. We expressed a construct of the first and second TMs of CD9, connected by the small extracellular loop, and tagged with a C-terminal green fluorescent protein (TM(1+2)-GFP molecule). In <span class="yellow">human</span> rhabdomyosarcoma RD cells, the wild-type fusion protein localized mostly in a reticular, intracellular pattern, without forming any large aggregates (Figure 5, panel A). Remarkably, when double mutants Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu were expressed, the protein formed distinct large aggregates in a high proportion of cells (Figure 5, panels C and E). In contrast, double mutant Gly77Leu + Gly80Leu did not form such aggregates (Figure 5, panel G). Results with respective single mutants were similar to that with double mutants, with the aggregation being somewhat more pronounced for Leu substitutions of Gly67 and Gly74 compared to Gly25 and Gly32 mutations. No aggregation was observed for Asn18Ser and Asn18Tyr mutants (data not shown). Also, nearly identical results were obtained with <span class="yellow">human</span> HT1080 cells (data not shown).
We interpret these results as an indication that aggregating mutants are destabilized or misfolded while non-aggregating mutants retain the wild-type conformation. Intriguingly, mutations to the conserved GG7 motifs caused protein aggregation while the mutation of other glycines had no detectable effect. These results also suggest that wild-type GFP, which has weak tendency to self-associate, could enhance non-specific interactions of destabilized mutant TM(1+2) CD9 moieties, leading to their aggregation. Consistent with this hypothesis, the aggregation of Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu double mutants was suppressed when monomeric GFP molecule, Leu221Lys [34] was used (Figure 5, panels D and F). The use of monomeric GFP did not affect intercellular localization of wild-type CD9 TM(1+2) (Figure 5, panel B), or a Gly77Leu + Gly80Leu double mutant (Figure 5, panel H).
In summary, Leu substitutions of Gly residues that are part of the Asn-Gly-Gly (NGG7) motif in TM1, or Gly-Gly-Ala (GGA7) motif in TM2, resulted in destabilization and aggregation of GFP-fused TM(1+2) proteins, whereas substitutions of Gly77 or Gly80, which are not part of these motifs (Figure 3), failed to show such aggregation.<br><br>Prediction and modelling of interaction between TM1 and TM2
Consecutive helices in polytopic membrane proteins frequently interact [35]. Sequence analysis of TM1 and TM2 helices of tetraspanins reveals a remarkable complementarity in the distribution of large and small residues at heptad positions a and d along the helical axis (Figure 3), suggesting that these residues may interact. To further elucidate the potential for TM1-TM2 interaction, the putative interface was modeled using a novel algorithm that considers mutational data during each step of a Monte Carlo simulated annealing cycle (see Methods for details). Specifically, Gly25Leu, Gly32Leu, Gly67Leu and Gly74Leu were scored as disruptive mutations, while Asn18Ser, Gly77Leu and Gly80Leu were scored as silent mutations, based on their effects on protein stability (Figure 5 and data not shown).
The resulting model predicts left-handed crossing of TM1 and TM2 helices at an angle of +28°. The key element of the structure is the apposition of bulky and small heptad position a and d residues, as follows: Gly32-Leu63; Gly67-Leu29; Gly25-Met70; Gly74-Trp22; Asn18-Gly77; Ala81-Phe15 (Figure 6). Our model predicts that each of these residue pairs are in van der Waals contact. Additionally, two potential H-bonds are predicted in this model, indicating close packing: Gly67 Cα to Gly25 carbonyl oxygen, and Trp22 Cα to Met70 carbonyl oxygen. The packing is tighter in the ectodomain-proximal portion of the helices (Figure 6, panel B), as determined by Cα-Cα distances between interacting residue pairs.
The key elements of the model are corroborated by the presence of apparently complementary substitutions in TM1 and TM2 sequences of different tetraspanins (Figure 1, boxed residues). For example, Gly74 is predicted to interact with Trp22. In 8 of the 10 tetraspanins that contain a substitution for Gly74, a compensatory substitution occurs at the Trp22 position (Figure 1). Thus, a larger non-glycine side chain at position 74 may necessitate a less bulky non-Trp side chain in position 22. Likewise, the presence of a Cβ at position 25, typically occupied by glycine, necessitates a non-β-branched amino acid at position 70, which is occupied by a β-branched residue in nearly half of all cases. Indeed, we find that in each of 5 cases in which position 25 contains a Cβ, a leucine residue occurs in position 70. This analysis is consistent with our molecular model that suggests Leu70 will pack most favorably against a Cβ at position 25 than a β-branched residue or a methionine.<br><br>Role of TM1 and TM2 heptad motif residues in CD9 dimerization
To probe CD9 dimerization, we used a cysteine-mediated cross-linking approach. We established previously a simple and efficient method for cysteine-mediated cross-linking [25]. After cells are pre-treated with 2-BP for 16–24 hours to expose normally palmitoylated cysteines, the cysteines can be cross-linked using any of the following methods: a) Spontaneous oxidation in Brij97 lysates (a condition that preserves tetraspanin-tetraspanin associations), b) In situ cross-linking, by pre-lysis oxidation of cells with Cu2+-phenanthroline (CuP) to promote disulfide bond formation. c) In situ cross-linking with thiol-reactive cross-linking agents of defined length (e.g. DTME, BMB). The first two approaches produce in essence "zero-length" disulfides, indicative of close proximity of target cysteines and presumably high specificity of interaction. In contrast, chemical cross-linkers with 6–20 Å spacer arm may cross-link with higher efficiency, but not necessarily higher specificity. However, they provide advantages such as variable membrane permeability, and potential linkage cleavability. For tetraspanins such as CD9, membrane-permeable cross-linker DTME (13.3 Å-long, reducible) provides highly specific and efficient cross-linking [25]. Here we have used a cysteine cross-linking strategy, in combination with cysteine-scanning mutagenesis, to map the residues from TM1 and TM2 contributing to the CD9 dimerization interface.
For subsequent cross-linking experiments using CD9 TM(1+2)-GFP protein, the non-dimerizing form of GFP was used. This avoids potential GFP-dependent dimerization and aggregation that can be observed with wild-type GFP, especially when fusions with transmembrane proteins are studied [36]. Importantly, the Leu221Lys mutation in GFP prevented aggregation of mutant forms of CD9 TM(1+2), which was observed with wild-type GFP fusion (Figure 5). The TM(1+2) fragment of CD9 contains three native cysteines – Cys9, Cys78 and Cys79. Single-cysteine mutants of TM(1+2) were constructed, in which a cysteine was placed at various faces of TM1 or TM2 while all of the wild-type cysteines were simultaneously replaced by serines. The mutant proteins were transiently expressed in RD cells (having little endogenous CD9), which were then treated for 16–18 hours with 2-BP. To achieve maximal specificity in cross-linking we used a "zero-length" agent, CuP.
First, single-cysteine replacements were constructed for residues Leu14, Phe15, Gly16, Phe17 and Asn18, covering just over one complete helical turn at the beginning of TM1. While residue Asn18 is highly conserved, positions 14, 15 and 17 are occupied by bulky hydrophobic residues in most tetraspanins, whereas position 16 shows less conservation (Figures 1, 4). All of the single-cysteine mutants showed diffused pattern of protein localization, without any signs of aggregation. As shown in Figure 7A, the highest level of intermolecular cross-linking was observed for Leu14Cys and Phe17Cys mutants, a lower level for Phe15Cys and Gly16Cys mutants, and very little cross-linking for Asn18Cys substitution. These results indicate that: a) the first two transmembrane domains of CD9 alone can mediate its dimerization, and b) the g and c residues of TM1 (e.g. Leu14 and Phe17, Figure 3) are likely to be part of the intermolecular interface.
Similarly, single-cysteine substitutions were made for residues Gly77, Gly80 and Ala81 in TM2; in addition, proteins carrying a single wild-type cysteine, Cys9, Cys78 or Cys79, were tested. No protein aggregation was observed for any of these single-cysteine mutants. As shown in Figure 7B, the relatively low level of intermolecular cross-linking of wild-type CD9 TM(1+2)-GFP protein was enhanced dramatically in single-cysteine TM2 mutants Gly80Cys and Ala81Cys. The Gly77Cys mutant also had an elevated level of cross-linking. In contrast, any of the three native cysteines (9, 78 and 79) produced level of cross-linking not much greater than the wild-type TM(1+2) protein. Similar results were obtained with cysteine-reactive cross-linker BMB (data not shown). Likewise, comparable results were obtained with single-cysteine mutants of untagged, full-length CD9, using CuP (Figure 7C) as well as DTME cross-linker (data not shown).
These cross-linking results for TM1 and TM2 are consistent with our model that places residues Leu14, Phe17 and Gly80 on the same side of the TM1-TM2 pair (Figure 6, panel C). The strong cross-linking with Leu14Cys, Phe17Cys and Gly80Cys places the intermolecular interface toward the c and g phases of the TM1 helix, and the g phase of the TM2 helix, away from its e and f faces containing wild-type cysteines 78 and 79.
Critical residues at the TM1-TM2 interface also affect dimerization indirectly. To assess specific CD9 dimerization, we used a Gly80Cys substitution at the intermolecular interface for cross-linking. As shown in Figure 8A, single replacements of conserved heptad residues in positions 18, 25, 32, 67 and 74 (Asn18Ser, Gly25/32/67/74→Leu) strongly decreased the cross-linking mediated by Cys80. The effect was most pronounced for mutations of residues, Gly32 and Gly67, located in the tightly packed extracellular end of TM helices (Figure 6). In contrast, mutations of residues closer to the cytoplasmic end of TM2 (Gly74 and especially Ala81) had only modest to very little effect on cross-linking.
Relatively low efficiency of intermolecular cross-linking via native residues Cys9, 78, and 79 (Figures 7B,C) correlates well with the predicted location of Cys78 and 79 away from the dimeric interface (Figure 3), and suggests that the extramembrane N-terminal part of CD9 (residues 1–13) does not self-associate. We next examined whether mutations of conserved Asn and Gly residues in TM1 and TM2 decreased low-level background cross-linking via native cysteines. As expected, these mutations had virtually no effect on dimer formation of CD9 TM(1+2)-GFP (Figure 8B). The level of covalent dimer formed was not diminished for triple Asn18Ser + Gly25Leu + Gly32Leu and double Gly67Leu + Gly74Leu mutants, compared to wild-type TM(1+2) CD9 molecule. Similarly, the same triple and double mutations in the context of full-length CD9-GFP protein (with six cysteines) produced wild-type levels of cross-linking (Figure 8C). We interpret these findings as evidence for at least two types of associations between CD9 molecules: primary, involving residues 14, 17 and 80, and dependent on integrity of conserved heptad residues in TM1 and TM2, and less efficient secondary interactions, probably representing random collision events, and independent of the heptad residues (see Discussion for more details).<br><br>TM3 and TM4 cysteine residues in CD9 dimerization
After identifying the roles of conserved TM1 and TM2 residues in CD9 dimerization, we next probed whether residues proximal to TM domains 3 and 4 are also involved. To this end, disulfide cross-linking of full-length CD9 molecules containing 3 C-terminal cysteines (87, just before TM3; 218 and 219 in TM4) or 3 N-terminal cysteines (9, 78 and 79) was compared (Figure 9). We found that the C-terminal cysteines were only slightly better than N-terminal cysteines with respect to detection of CD9 dimers. However, markedly more trimers and tetramers were detected using C-terminal cysteines. Thus, residues 87, 218 and 219 at TM3 and TM4 in CD9 can together form contacts across the dimeric interface and also additional contacts with other neighboring CD9 molecules.<br><br>
Discussion
Here we provide the first detailed analysis of tetraspanin protein transmembrane domains. First, we show 1) the presence of a heptad repeat motif in TM1 and TM2, containing highly conserved Asn and Gly residues, 2) a leucine and glutamate/glutamine-containing heptad motif in TM3, and 3) high variability and absence of heptad repeats in TM4 sequences. Second, we provide evidence for a specific, intramolecular interaction between TM1 and TM2 domains, in which bulky hydrophobic residues pack against GG7 motif, and present a molecular model for this interaction. Third, experimental mapping of the CD9 dimerization interface firmly establishes an additional role for conserved TM1 and TM2 residues in dimeric intermolecular interactions. Fourth, preliminary evidence is provided to suggest that TM3 and TM4 domains contribute to expansion of CD9 dimers into higher order multimers.
Conserved residues in TM1 and TM2 of tetraspanins: role in intramolecular packing
We hypothesized that the first two transmembrane domains of tetraspanins might interact with each other because: a) consecutive TM domains frequently associate in known protein 3D structures [35], and b) they both contain a series of highly conserved amino acids – several Gly residues and an Asn residue (Figure 1). Conserved Gly residues are a frequent theme in the organization of interacting transmembrane domains. Analysis of 3D helix packing in polytopic membrane proteins reveals that Gly residues tend to localize in buried positions, especially at the helix-helix interfaces and helix crossing points [37,38]. Due to the absence of a side chain, Gly provides a flat surface for packing of a side chain from another residue, without loss of side-chain entropy upon interaction. The most common Gly-containing motif is GxxxG [39,40]. In glycophorin A (GpA), the major glycoprotein in erythrocyte cell membranes, Gly79 and Gly83 are part of the LIxxGVxxGVxxT sequence that promotes homodimerization of parallel transmembrane α-helixes [41,42]. In the GpA dimerization motif, Gly residues allow for tight packing in the right-handed helical crossing [43]. There are also examples of left-handed helical crossing in the context of a GxxxG motif [44]. Other membrane proteins that use the GxxxG motif for homo- or heterodimerization include <span class="yellow">bacteriophage M13</span> coat proteins [45], <span class="yellow">yeast</span> alpha factor receptor [46], integrin α IIb subunit [47], and ErbB1 receptor tyrosine kinase [48]. Other small residues, such as Ala and Ser, can substitute for Gly in this motif [49].
A protein motif in which Gly residues are separated by 6 other residues (GG7) is also common in transmembrane helices, especially in transporter/channel-like membrane proteins [50]. However, the structural features associated with this motif are not well known. In particular, it is unclear whether left-handed GG7 heptad repeat motif (as opposed to the "classic" right-handed GxxxG motif) can drive membrane helix association. In a recent work addressing this issue, Lear et al. [51] showed that a synthetic peptide containing Gly at heptad positions a and d could self-associate in vitro, likely in an antiparallel orientation. Heptad repeats containing conserved Gly residues occur in TM domains of α and β chains of MHC class II proteins, and mutations of the Gly residues disrupt the αβ heterodimer [52]. These examples demonstrate that Gly-based heptad motifs may be used for both intra- and intermolecular associations.
In this work, we identified a highly conserved GG7 motif in the first two tetraspanin TM domains. The GG7 sequence in tetraspanins is a part of a larger motif that also includes a conserved Asn residue in TM1 (NGG7) and an Ala/Ser/Thr residue in TM2 (GGA7). The seven-residue periodicity of these motifs strongly suggests their involvement in left-handed coiled coil packing reminiscent of the leucine zipper, rather than right-handed packing of the GpA-like GxxxG motif. For antiparallel helices, the left-handed crossing is in fact predominant over the right-handed in known TM domain structures [44].
In our model, heptad Gly residues in NGG7 and GGA7 sequences provide specific packing between antiparallel tetraspanin TM1 and TM2 helices by allowing tight van der Waals interactions with large hydrophobic residues (Figure 6). Highly efficient packing of bulky side chains against glycine residues is observed in known transmembrane protein 3D structures [38,53,54]. An example includes packing of helices M1 and M2 in potassium channel KcsA, where Val91 in M2 is paired with Gly43 in M1, and Leu36 in M1 contacts Ala98 and Gly99 in helix M2 [54,55]. In addition to facilitating helix-helix packing, Gly residues frequently provide additional CαH...O hydrogen bonds between two helices [44]. In our model, two Cα-backbone carbonyl H-bonds are predicted – between residues Gly27-Gly67, and Trp22-Met70.
Although polar and charged amino acid residues (such as Asn in the TM1 heptad motif) are infrequent in transmembrane domains, they are functionally important. Polar residues such as glutamine, glutamic acid, aspartic acid and asparagine can promote strong oligomerization of model membrane-associated helices [56-58]. Ruan et al. [59] used asparagine scanning mutagenesis to probe the interface of self-associating polyleucine helices by detecting their enhanced self-interaction in vitro and in the <span class="yellow">E. coli</span>-based ToxR assay. Thus, a hydrogen bond in an apolar environment can result in strong, though not necessarily specific, association of transmembrane helices. In fact, mutations to polar residues in transmembrane proteins are commonly associated with disease [60]. Because of this potential for non-specific interactions, polar residues tend to localize at buried positions in TM domains.
In our case, the conserved Asn18 residue in CD9 is predicted to be a part of the TM1-TM2 interface, though our model does not predict any electrostatic interaction between Asn18 and TM2 (Figure 6). Consistently, substitution such as Asn18Tyr (and Gly77Leu) in TM(1+2)-GFP protein was not destabilizing as analyzed by protein aggregation. Curiously, the full-length Asn18Ser CD9 migrated slightly slower on SDS-PAGE gel (data not shown), suggesting that Asn18 does play a role in maintaining conformation of the molecule. The Asn18Cys single-cysteine mutant shows very little intermolecular cross-linking (Figure 7A), supporting the proposed location of this residue at the intramolecular interface. It is tempting to speculate that the "pocket" between TM1 and TM2 lined by Asn18 and Gly77 might be important for accommodating palmitate moieties that target Cys78 and Cys79 residues, and/or important for access by the putative palmitoyl transferase to those residues. Understanding the exact role of these highly conserved Asn18 and Gly77 residues in tetraspanins awaits further investigation.
In summary, we identified conserved glycine residues of TM1 and TM2 of tetraspanins as key elements required for intramolecular packing. Mutations of these key residues (Gly25, Gly32, Gly67 and Gly74 in CD9) resulted in protein destabilization and aggregation. There is ample evidence in the literature for mutations in transmembrane proteins that lead to protein destabilization, misassembly and pathologic conditions [61]. Thus, we have identified conserved heptad Gly residues in TM1 and TM2 of tetraspanins as plausible targets of destabilizing mutations with potential functional consequences.<br><br>Intermolecular interactions in tetraspanins
Tetraspanin CD9 forms mostly homodimers, but also a low level of heterodimers with CD81 and CD151 [25]. Thus, mapping the dimerization interface is an important next step in structure-function analysis of tetraspanins. Disulfide-mediated cross-linking, often in combination with cysteine-scanning mutagenesis, is a common strategy to probe oligomerization or intersubunit interactions of transmembrane proteins such as histidine kinase EnvZ [62], M(3) muscarinic acetylcholine receptor [63], <span class="yellow">E. coli</span> lactose permease [64], synaptobrevin [65], integrins [66] and many others. In tetraspanins such as CD9, membrane-proximal cysteine residues are especially useful targets for disulfide trapping, as their linkage can be enhanced by pre-treating cells with 2-BP. While the ability of wild-type cysteines in CD9 to be cross-linked may indicate that they are close to the dimerization interface, more precise mapping was achieved here using cysteine-scanning mutagenesis.
Our data clearly identify regions, near the cytoplasmic face of TM1 and TM2, important for dimerization. Intermolecular zero-length cross-linking was highest when single cysteines were placed in positions 14, 17, 77, 80 and 81 in TM(1+2)-GFP molecule, or at positions 77, 80 or 81 in the full-length CD9 protein. Positions 14, 17 and 80 are distinct from the intramolecular interface and are on the same side of the TM1-TM2 pair (Figure 6). Thus, they are well-positioned to participate in an interaction with another molecule. At the same time, the model predicts that wild-type cysteines (Cys78, 79), which do not yield very efficient zero-length cross-linking, are on the other side of TM1-TM2 pair.
While using the cysteine at position 80 as the dimeric interface probe, mutations of conserved residues in TM1 and TM2 (especially Gly32 and Gly67 to Leu) clearly reduced intermolecular cross-linking. We do not suggest that those residues are directly involved in intermolecular interaction. Rather, we propose that destabilization of the intramolecular TM1-TM2 interaction by Gly to Leu substitutions (discussed above) causes an overall conformational change that reduces dimer formation.
An Ala81Leu mutation did not reduce cross-linking via Cys80, even though single-cysteine Ala81Cys molecules themselves produced a high level of cross-linking. These results, together with data on Gly32Leu and Gly67Leu mutations, are consistent with our model predicting that helices 1 and 2 interact more tightly near the extracellular end and less at the cytoplasmic end. This would give more flexibility to a cysteine at position 81 and also limit the effect of an Ala81Leu mutation. Location of this residue at the membrane/cytoplasmic border could also make it more accessible to CuP reagent as compared to residues buried in TM domain, thus elevating the efficiency of disulfide formation of the Ala81Cys mutant.<br><br>Multiple interfaces in tetraspanin molecules
In the full-length CD9 molecules, the 3 C-terminal cysteines (Cys87, 218 and 219) located at or in TM3 and TM4 promoted efficient dimer and even more efficient oligomer formation compared to the 3 N-terminal cysteines (Figure 9). Cys87 alone can be used to capture CD9 dimers [25]. These results suggest the existence of two dimeric interfaces in CD9 molecule – the TM 1-2/1-2 interface and the TM 3-4/3-4 interface (Figure 10). In a TM(1+2) molecule, the destabilization of 1–2 interaction, e.g. by Gly→Leu mutations, would affect the 1-2/1-2 interface, as discussed above. However, these mutations would not interfere with the 3-4/3-4 interface in a full-length molecule, which includes Cys87, 218 and 219. Thus, cross-linking of full-length molecules, containing all 6 cysteine residues, would be unaffected, as seen in Figure 8C. Furthermore, wild-type Cys9, Cys78 and Cys79 are apparently not at the primary 1-2/1-2 interface. Their relative inefficiency in cross-linking CD9 TM(1+2) protein likely reflects weak secondary contacts between the molecules, or possibly random collision events. Such events should be independent of mutations in the conserved Gly residues in TM1 and TM2, as was demonstrated in Figure 8B. The potential existence of two interfaces in tetraspanin molecules, 1-2/1-2 and 3-4/3-4, should provide enhanced flexibility for forming additional intermolecular contacts. Current understanding of tetraspanin microdomains assumes a few strong, primary homotypic and heterotypic tetraspanin complexes (e.g. CD9-CD9, CD9-CD81, CD151-α3 integrin, CD81-EWI2) that help bring together various other proteins, forming secondary-type associations. Such properties of tetraspanins may bring signaling molecules such as protein kinase C or phosphatidylinositol 4-kinase to the vicinity of integrins [67,68].
The organization of the TM3 domain points to a potential role in protein-protein interactions. A motif of Leu-Leu-Glu(Gln) spaced 7 residues apart (heptad positions a), with highly conserved residues in two consecutive positions d, poses as a likely interaction module. If responsible for heterologous protein-protein interactions, it would form another distinct interface of tetraspanin molecule. Our preliminary data indicate that replacing the Leu and Glu residues in TM3 of CD9 with Ala has no effect on cell surface expression of the protein and its dimerization (data not shown). It remains to be tested if interactions with other proteins will be affected. Similarly, the TM4 domain may provide additional contributions to lateral tetraspanin associations. Much higher sequence variability, and the lack of a distinct heptad pattern suggests that TM4 is a major contributor to diversity among tetraspanin complexes. Structure-function analysis of TM3 and TM4 domains in tetraspanins is the subject of ongoing investigation.<br><br>
Conclusion
We have defined the TM1-TM2 intramolecular interface in tetraspanin CD9, providing evidence for glycines (Gly25 and Gly32 in TM1, Gly67 and Gly74 in TM2) packing against apposing bulky aliphatic residues. Second, we mapped an intermolecular CD9 interface (involved in CD9 homodimer formation) to the vicinity of residues Leu14 and Phe17 in TM1 and Gly77, Gly80 and Ala81 in TM2. Finally, we provide preliminary evidence that TM3 and TM4 in CD9 may contribute to a second intermolecular interface. Key CD9 residues involved in intra- and intermolecular interactions are highly conserved throughout the tetraspanin family, thus suggesting that our findings will apply to most tetraspanins.<br><br>Methods
Materials
Cell culture reagents were from Invitrogen (Carlsbad, CA). 2-bromopalmitate was from Fluka (Milwaukee, WI), N-ethylmaleimide (NEM) and 1,10-phenanthroline were from Sigma-Aldrich (St. Louis, MO), and chemical cross-linkers dithio-bis-maleimidoethane (DTME) and 1,4-Bis-maleimidobutane (BMB) were purchased from Pierce Endogen (Rockford, IL). Triton X-100, protease inhibitor cocktail and FuGENE 6 transfection reagent were obtained from Roche (Indianapolis, IN). Restriction endonucleases and Pfu DNA polymerase were obtained New England Biolabs (Beverly, MA) and Stratagene (Carlsbad, CA), respectively. All other chemicals were purchased from Sigma-Aldrich or Fisher Scientific (Pittsburg, PA).<br><br>Sequence analysis
Tetraspanin sequences were obtained from SWISS-PROT and GenBank databases. Locus designations, accession numbers and the most commonly used protein names are summarized in Tables 1 and 2. TM segments were delineated by inspection of hydrophobicity profiles, using database annotations and previous analyses of TM sequences as a guide ([28], M. Hemler, unpublished), and aligned manually. Residue numbers in <span class="yellow">human</span> CD9 sequence are used a reference point throughout the study.<br><br>DNA cloning and mutagenesis
Sequence encoding CD9 protein was cloned into vector pcDNA3 (Invitrogen, Carlsbad, CA) and pEGFP-N1 (Clontech, Palo Alto, CA), for expression of untagged and C-terminally GFP-tagged CD9, respectively. pEGFP-N1 encoding CD9 TM(1+2) -GFP fusion protein was constructed by subcloning DNA for residues 1–83 of CD9 into HindIII and PstI sites of the vector; to introduce the PstI site, codon GTG for Val82 was changed to CTG (coding for Ala). In the resulting fusion protein, there is a 13-amino acid linker (with no cysteines) between CD9 and GFP. To minimize the low inherent ability of GFP to homodimerize, which could potentially influence the results of CD9 cross-linking, we used a monomeric GFP mutant, Leu221→Lys [34], for cross-linking experiments.
Mutations were introduced in full-length and TM(1+2) CD9 proteins by a PCR-based strategy using mutagenic primers and Pfu DNA polymerase. All mutations were confirmed by DNA sequencing.<br><br>Protein expression, microscopy, cysteine disulfide cross-linking and Western blotting
DNA constructs encoding TM(1+2)-GFP or full-length CD9 proteins were transfected into <span class="yellow">human</span> rhabdomyosarcoma RD cells using the FuGENE 6 reagent. Cells expressing GFP fusion proteins were analysed by fluorescence microscopy 18–28 hours post-transfection. Images were captured using Spot 1.4.0 camera (Diagnostic Instruments, Sterling Heights, MI) attached to Nikon Eclipse TE300 microscope.
For experiments involving cysteine-mediated cross-linking, cells were treated with 50 μM 2-BP starting 24–26 hours post-transfection and continuing for 16–18 hours. Cross-linking was carried out by incubating cells in HBSM buffer (25 mM Hepes-NaOH, pH 7.2, 150 mM NaCl, 2 mM MgCl2) containing either a) 0.6 mM CuSO4 and 1.8 mM 1,10-phenanthroline (CuP complex) or b) 0.2 mg/ml homobifunctional cysteine-reactive cross-linker (e.g. DTME), diluted from fresh 10 mg/ml solution in DMSO. After incubation for 10–15 minutes (with CuP) or 30–45 minutes (with cross-linker), cells were washed twice for 10 minutes with HBSM containing 10 mM NEM to block residual free cysteines. Cells were lysed in HBSM containing 1% Triton X-100, 0.1% SDS and a cocktail of protease inhibitors with 1 mM EDTA at 4°C for 45–60 minutes. Cell lysate was clarified by centrifugation at 14,000 × g for 15 minutes, an aliquot was removed, and proteins from it were precipitated by addition of trichloroacetic acid to 10% on ice followed by centrifugation at 14,000 × g for 10 minutes. After two washes with ice-cold acetone, protein pellet was solubilized in SDS-PAGE sample buffer without a reducing agent (50 mM Tris-HCl, pH 6.8, 1% SDS, 8% glycerol).
In some experiments, CD9 protein was immunoprecipitated using monoclonal antibody Alb6 (Immunotech, Marseille, France). Proteins were separated by SDS-PAGE and analyzed by Western blotting using monoclonal antibody JL-8 (Clontech) for GFP or Alb6 for CD9. Bands from X-ray films were quantitated using GeneTools™ software from Syngene (Frederick, MD).<br><br>Modeling of TM1-TM2 interaction
An atomic model of the CD9 TM1-TM2 dimer was constructed with a Monte Carlo-simulated annealing (MCSA) algorithm [69]. Two idealized α-helices corresponding to TM1 residues Tyr12 through Leu35 and TM2 residues Gly59 through Val82 were docked with six orthogonal parameters: three rigid body translations and three rotations. During each step of a MCSA cycle, there was an equal probability of changing either one parameter or all six parameters to random values. A conformation's energy was calculated in vacuo with the AMBER united-atom force field for van der Waals interactions [70]. The van der Waals term was modified as described by Kuhlman and Baker [71]. If a structure had favorable dimerization energy, the energies of select mutants were calculated. Structures were selected with a novel scoring function that maximizes the Boltzmann probability of dimerization for silent mutations while minimizing the probability for disruptive mutations. Asn18Ser, Gly77Leu, and Gly80Leu were scored as silent mutations while Gly25Leu, Gly32Leu, Gly67Leu, and Gly74Leu were considered to be disruptive. Each MCSA cycle consisted of 50,000 steps with an exponential temperature decay from 10,000 to 10 K.
Ten MCSA cycles through global sample space were used to restrict the search area. Parameters were restricted to ± 2 standard deviations from their mean value for structures within 10 kcal of the best structure. MCSA cycles were repeated as described above with additional optimization of χ values: rotamers at the protein-protein interface were optimized with Dead End Elimination [72], and χ values were further optimized with Monte Carlo. All MCSA cycles converged upon structures that were within a root mean squared deviation (RMSD) of 1.5 Å with the best structure, and structures that scored within 5 kcal of the best score had an RMSD of less than 0.5 Å with the best structure.<br><br>
List of abbreviations
2-BP, 2-bromopalmitate; CuP, Cu2+-phenanthroline; DTME, dithio-bis-maleimidoethane; LEL, large extracellular loop; NEM, N-ethylmaleimide; TM, transmembrane (domain).<br><br>Authors' contributions
OVK carried out sequence comparisons, mutational analysis and cross-linking experiments, and drafted the manuscript. DGM built the TM1-TM2 interaction model and contributed to the manuscript. WFD supervised DGM's work. MEH coordinated the whole study and prepared the final manuscript.<br><br>
<h3>pmcA140510</h3>Enp1, a <span class="yellow">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="yellow">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="yellow">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="yellow">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="yellow">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="yellow">human</span> Enp1, expressed in <span class="yellow">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="yellow">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="yellow">human</span> homolog of <span class="yellow">yeast</span> Enp1 was PCR amplified from a <span class="yellow">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="yellow">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="blue">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="yellow">donkey</span>-anti-<span class="blue">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="blue">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="yellow">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="yellow">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="yellow">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="yellow">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="yellow">yeast</span> Enp1 (19). In contrast, we identified a <span class="yellow">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="yellow">human</span> bystin (19) showed that they were encoded by the same gene on <span class="yellow">human</span> chromosome 6. The reported sequence for <span class="yellow">human</span> bystin is a fragment of the 437 amino acid <span class="yellow">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="yellow">human</span> bystin, as confirmed by available <span class="yellow">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="yellow">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="yellow">Drosophila melanogaster</span> and <span class="blue">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="yellow">human</span> Enp1 homolog was cloned and expressed in <span class="yellow">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="yellow">human</span> Enp1 homolog did not complement a <span class="yellow">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="yellow">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="yellow">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="yellow">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="yellow">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="yellow">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="yellow">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="yellow">human</span> homolog of Enp1 did not complement a <span class="yellow">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="yellow">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA147033</h3>Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the ribonuclease A gene family.
Abstract
We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophil-associated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this 'mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing.
� 1997 Oxford University Press <br><br> Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> 4235�4239 <br><br> Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the Ribonuclease A gene family Dean Batten+, Kimberly D. Dyer, Joseph B. Domachowske� and Helene F. Rosenberg* The Laboratory of Host Defenses, Building 10, Room 11N104, National Institute of Allergy and Infectious Diseases, National Institutes of Health, 9000 Rockville Pike, Bethesda, MD 20892, USA Received August 4, 1997; Revised and Accepted September 17, 1997 DDBJ/EMBL/GenBank accession numbers: U72031, U72032, AF017258�AF017261 <br><br> ABSTRACT We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophilassociated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this `mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing. INTRODUCTION Ribonuclease A (RNase A, <span class="yellow">bovine</span> pancreatic ribonuclease) is the prototype of an enlarging family of proteins defined by distinctive elements of conserved primary structure and enzymatic activity (1,2). The features shared by all members of the RNase A family include six to eight cysteines and specific histidine and lysine residues that form the ribonuclease active site. To date, several distinct lineages within the RNase A superfamily have been identified. Pancreatic ribonucleases (RNases 1) have been isolated from an extensive array of mammalian species (3,4), and mRNA encoding <span class="yellow">human</span> pancreatic ribonuclease has been detected in <br><br> numerous somatic tissues in addition to pancreas (5). RNases 2 and 3, eosinophil-derived neurotoxin (EDN) and eosinophil cationic protein (ECP), respectively, have been characterized primarily with respect to eosinophil function (6,7), although expression of EDN [also known as RNase Us (8) or <span class="yellow">human</span> liver ribonuclease (9)] is also widespread (5). Angiogenin (RNase 5), a structurally atypical member of this family, was originally identified as an agent promoting blood vessel growth and development (10,11). RNase 4 (12,13) and RNase k6 (14) are also members of the RNase A superfamily, although their functions are currently obscure. Recently, Larson and colleagues (15) described cDNAs encoding two highly homologous <span class="yellow">murine</span> ribonucleases, the <span class="yellow">murine</span> eosinophil-associated ribonucleases (mEAR) 1 and 2, which were cloned and identified via tryptic peptides isolated from <span class="yellow">murine</span> eosinophil proteins. Although Southern analysis demonstrated that multiple copies of sequence homologous to mEAR-1 and mEAR-2 were present in <span class="yellow">murine</span> genomic DNA, the precise nature of these copies--as pseudogenes, polymorphisms, or distinct functional genes--was unclear. In this work, we have identified four of these homologous sequences, three encoding novel ribonucleases, and one encoding a pseudogene. Together with mEAR-1 and mEAR-2, these six ribonuclease genes form an `mR cluster,' whose members are more closely related to one another than they are to other <span class="yellow">murine</span> ribonucleases, yet whose position with respect to the defined RNase A lineages remains unclear. MATERIALS AND METHODS Isolation of genomic fragments encoding <span class="yellow">murine</span> ribonucleases by polymerase chain reaction (PCR) Genomic DNA was isolated from cells of the <span class="yellow">murine</span> 3T3 fibroblast cell line. The mR-3, mR-5 and mR-6P sequences were amplified using oligonucleotide primers derived from the published coding sequence of mEAR-1 as follows: 5-CAA ACC CTT TCC CAG AAG TTT GCC-3 (amino acids 28�33); 5-AAA TGT CCC ATC CAA GTG AAC TGG ACC-3 (amino acids 156�148) (15). The PCR reactions were performed as described previously (14), and the multiple products present in the single 400 bp product were identified by dideoxy<br><br> *To whom correspondence should be addressed. Tel: +1 301 402 9131; Fax: +1 301 402 4369; Email: hr2k@nih.gov Present addresses: +Duke University School of Medicine, Durham, NC 27701, USA and �Department of Pediatrics, State University of New York-Health Sciences Center at Syracuse, Syracuse, NY 13210, USA <br><br> 4236 Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> Figure 1. (A) Alignment of amino acid sequences encoded by six related <span class="yellow">murine</span> ribonuclease genes. Regions of amino acid sequence identity among the five functional genes are enclosed in boxes; the boxes enclosing the active site residues (His11, Lys35, His124) and eight cysteines conserved in all members of the RNase A superfamily are shaded. (B) Percent similarities between amino acid sequences determined via the BESTFIT algorithm of the Wisconsin Genetics Computer Group (WGCG) program on-line at the National Institutes of Health. GenBank accession numbers: mEAR (<span class="yellow">murine</span> eosinophil-associated ribonuclease)-1, U72032; mEAR-2, U72031; mR (<span class="yellow">murine</span> ribonuclease)-3, AF017258; mR-4, AF017259; mR-5, AF017260; mR-6P (pseudogene), AF017261. <br><br> sequencing of individual plasmids after subcloning into the pCR II TA cloning vector (Invitrogen, San Diego, CA). The complete open reading frames were obtained by extension in both 5 and 3 directions by uni-directional PCR (Genome Walker kit, Clontech, Palo Alto, CA), involving two amplifications with nested primers and Tth polymerase as described previously (14). The nested gene-specific primer pairs were as follows: for 5 extension of mR-3, 5-CTG GAA CCA CTG GAT ACG TGG GAC TGT CCT-3 and 5-ACG TGG GAC TGT CCT GTG GAG TTC TGG GTT-3; for 3 extension of mR-3, 5-ATG CTG TTG GTG TGT GTG GAA ATC CAA GTG-3 and 5-GCT TGT GCA GTG ACA ATA TAA GTC AAA ACT-3; for 5 extension of mR-5, 5-GGT GGG ACG GTC CTT TGG AGT TCT GGG GTT ACA-3 and 5-GGG GTT ACA GGC AAC TGT GTA GGA CTT CTT TCC-3; for 3 extension of mR-5, 5-GAT GTT GTC CGT GTG TGT CAC AAT CCA CCC-3 and 5-AAG ACT TGC AAA GAC GGG ACA AGT CCA AAT-3; for 5 extension of mR-6P, 5-TTG TTG TTT TGC ATC TGC ATT GTG CAT AAC TGC-3 and 5-TTC TGC ATA ACT GCT TGC TGA ACT TGT GAG TGA-3; for 3 extension of mR-6P, 5-ATG TGA TAA TGC AAT GCT GTC TCT TAG CAG TTA-3 and 5-TAC AAG <br><br> AGT ATG TAA GCC ATT GAA TCA TTT TCT-3. The products of these reactions were likewise subcloned into the PCR II TA vector and evaluated by dideoxy-sequencing. The mR-4 sequence was amplified with a 36 bp primer derived from sequence encoding the amino-terminal signal sequence of mEAR-1; 5-ATG GGT CCG AAG CTG CTT GAG TCC CGA CTT TGT CTC-3 with the carboxy-primer described above. <br><br> Northern analysis The <span class="yellow">murine</span> multi-tissue Northern blot was purchased from Clontech, and pre-hybridized and hybridized as per manufacturer's instructions with radiolabelled oligonucleotide probes. The membrane was washed with 5� SSPE with 0.1% SDS for 1 h at 37_C and the autoradiogram developed after exposure to film at �80_C. The mEAR-1/mEAR-2/mR-3 probe: 5-CTC TTG TCA CTG CAC AAG CCA CTT GGA TTT CC-3, the mR-5 probe: 5-GTC CCG TCT TTG CAA GTC TTG GGT GGA TTG TG-3, and the actin probe: 5-GCA CAT GCC GGA GCC GTT GTC GAC GAC GAG CGC GGC GAT ATC ATC ATC-3 (16). <br><br> 4237 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 4237 <br><br> Figure 2. (A) Dendrogram depicting relationships among the eight characterized <span class="yellow">murine</span> ribonucleases as determined by a modified UPGMA method (17). Abbreviations are as in Figure 1A, also mPR (<span class="yellow">murine</span> pancreatic ribonuclease, sw:rnp_<span class="blue">mouse</span>), mANG (<span class="yellow">murine</span> angiogenin, U72672), hEDN (<span class="yellow">human</span> eosinophil-derived neurotoxin, M24157), hECP (<span class="yellow">human</span> eosinophil cationic protein, X15161), hRK6 (<span class="yellow">human</span> ribonuclease k6, U64998), pRK6 (porcine kidney ribonuclease (24), and bRK6 (<span class="yellow">bovine</span> kidney ribonuclease, sw:rnkd_bovin). (B) Calculated isoelectric points from amino acid sequences in (A) as determined via the PEPTIDESORT algorithm of WGCG. Table 1. Amino acid sequence comparisons of <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases <br><br> Sequence analysis All DNA sequence analysis and comparisons were performed with the assistance of the Wisconsin Genetics Computer Group and Sequencher (Gene Codes Corporation) programs available at the National Institutes of Health. The dendrogram in Figure 2A was constructed by a modified version of the unweighted pair group method with arithmetic mean (UPGMA) (17), constructed using four initial pairings, mEAR-2 to mR-3, mR-5 to mR-6P, hEDN to hECP and bRK6 to pRK6. RESULTS Isolation of genomic fragments encoding novel <span class="yellow">murine</span> ribonucleases The alignment in Figure 1A displays the amino acid sequences encoded by the six related ribonuclease genes. <span class="yellow">Murine</span> eosinophil-associated ribonucleases-1 and -2 (mEAR-1 and mEAR-2) are the predicted amino acid sequences from two genes described previously by Larson and colleagues (15); <span class="yellow">murine</span> ribonucleases 3, 4, 5 and 6-pseudogene (mR-3, mR-4, mR-5 and mR-6P), are the predicted amino acid sequences encoded by novel DNA <br><br> hEDN mEAR-1 mEAR-2 mR-3 mR-4 mR-5 mANG mPR 56 56 54 60 60 43 53 <br><br> hECP 59 59 57 59 58 49 52 <br><br> hRK6 58 59 59 53 56 49 59 <br><br> hPR 47 48 49 48 57 55 79 <br><br> hR4 50 53 51 52 54 57 66 <br><br> hANG 49 48 51 47 47 80 60 <br><br> Values are expressed as percent similarity between pairs of amino acid sequences as determined by the BESTFIT algorithm of the Wisconsin Genetics Computer Group program on-line at the National Institutes of Health. Value representing the highest degree of sequence homology in each row is indicated in boldface. Abbreviations are as defined in Figures 1 and 2, and also include hR4 (<span class="yellow">human</span> ribonuclease 4, sw:rnl4_<span class="yellow">human</span>), hANG (<span class="yellow">human</span> angiogenin, M11567), and hPR (<span class="yellow">human</span> pancreatic ribonuclease, X62946). <br><br> 4238 Nucleic Acids Research, 1997, Vol. 25, No. 21 fragments amplified from <span class="yellow">murine</span> genomic DNA. All five functional genes encode amino acid sequences with eight cysteines and appropriately spaced catalytic histidines and lysine that are conserved among the members of the RNase A superfamily. The predicted coding sequence of the non-functional pseudogene (mR-6P) includes two aberrant stop codons (positions 19 and 100) as well as a point mutation resulting in the destruction of the cysteine at position 78. Each genomic fragment also encodes a 28 residue amino terminal signal sequence (not shown). The amino acid sequences are displayed in order of decreasing similarity to mEAR-1 (Fig. 1B). Genomic fragments encoding mEAR-1 and mEAR-2 were also detected. Relationships among the <span class="yellow">murine</span> ribonucleases A dendrogram depicting the relationships among several of the <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases is shown in Figure 2A. The mEAR-1, mEAR-2, and mR-3 sequences form a closely related sub-group, with only nine amino acid sequence differences noted between mEAR-2 and mR-3, and 12 and 17 between these two and mEAR-1, respectively. The mR-4, mR-5, and mR-6P sequences diverge from this subgroup and from one another as well. These six sequences are more closely related to one another than they are to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or to <span class="yellow">murine</span> angiogenin (mANG) or to any of the three <span class="yellow">human</span> ribonucleases shown (hEDN, hECP or hRK6). Isoelectric points The isoelectric points of all six related ribonucleases range from 8�11, as is typical for members of the RNase A superfamily (Fig. 2B). Interestingly, the isoelectric points within the mEAR-1/mEAR-2/mR-3 subgroup vary by a full 1.0 unit (mEAR-1 at 9.2 to mEAR-2 at 10.2) in response to only 12 discrepancies in amino acid sequence. As increased charge has been associated with increased toxicity in other branches of the RNase A gene family (18), this finding has potential consequences with respect to the function of these proteins (see Discussion). Northern analysis Tissue-specific expression of mRNA encoding these <span class="yellow">murine</span> ribonucleases is shown in Figure 3. The probe complementary to all three sequences of the mEAR-1/mEAR-2/mR-3 subgroup hybridizes to an 1.4 kb mRNA isolated from renal tissue (Fig. 3A). As the probe cannot distinguish among the three genes, it is not possible to conclude whether the band detected represents mRNA encoding one, two, or all three of the ribonucleases of this subgroup. In contrast, the probe complementary to the mR-5 genomic fragment hybridizes to two prominent mRNA species in hepatic tissue, one at 1.4 kb, and a second at 2.0 kb. A prominent mRNA of 1.8 kb was also detected in mRNA isolated from <span class="yellow">murine</span> testicular tissue; fainter hybridizing mRNAs of varying size were detected in lung and skeletal muscle. The size variation may represent differential splicing of the mR-5-encoding mRNA, or may represent mRNAs encoding one or more as yet undiscovered <span class="yellow">murine</span> ribonucleases that are more closely related to mR-5 than those whose sequences are reported here. <br><br> Figure 3. (A) <span class="yellow">Murine</span> RNA probed with a 32 bp oligonucleotide complementary to mEAR-1, mEAR-2 and mR-3 coding sequences. Tissue source of each RNA is indicated above each lane. (B) Same as in (A), probed with a 32 bp oligonucleotide complementary to mR-5. (C) Same as in (A), probed with a 48 bp oligonucleotide complementary to <span class="yellow">human</span> actin (16), demonstrating relative loading of each lane. <br><br> Relationship to <span class="yellow">human</span> ribonucleases The data in Table 1 denote the amino acid sequence similarity between pairs of <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases of the RNase A superfamily. Overall, the similarities within the identified cluster range from 47 to 60%, with remarkably little variation among the pairs. Although the sequences of the mEAR-1/mEAR-2/mR-3 subgroup are slightly more similar to ECP and RNase k6, and mR-4 and mR-5, to EDN and ECP, the orthologous relationships of these <span class="yellow">murine</span> ribonucleases cannot be discerned from these data. This stands in direct contrast to mPR and mANG, whose <span class="yellow">human</span> orthologs can be clearly distinguished on the basis of their amino acid sequence homologies. <br><br> DISCUSSION In this work, we have identified four novel <span class="yellow">murine</span> ribonuclease genes that, together with mEAR-1 and mEAR-2 defined by Larson and colleagues (15), form an unusual cluster within the RNase A gene superfamily. The six genes within this cluster have varying amino acid sequence homologies to one another, but are clearly more closely related to one another than to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or angiogenin (mANG), or to any of the <span class="yellow">human</span> ribonucleases of the RNase A family. These results suggest that the `mR cluster' emerged via multiple duplications of a gene that had already diverged from those encoding the other <span class="yellow">murine</span> ribonucleases. Although this type of expansion is unusual in this gene family, it is actually not unique; similar expansion has occurred in <span class="yellow">bovine</span> species, with gene duplication resulting in the three closely-related <span class="yellow">bovine</span> pancreatic, <span class="yellow">bovine</span> brain (19,20) and <span class="yellow">bovine</span> seminal (21) ribonucleases. One interesting feature of the mR cluster is that its relationship to any of to the six known <span class="yellow">human</span> RNase A-type genes cannot be <br><br> 4239 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 determined from their respective amino acid sequence homologies. Although the amino acid sequences encoded by mEAR-1 and mEAR-2 match those of tryptic peptides derived from <span class="yellow">murine</span> eosinophil proteins (15), the homology data do not stand in overwhelming support of a unique relationship between any of the mR cluster ribonucleases and the <span class="yellow">human</span> eosinophil ribonucleases EDN and ECP. The existence of an additional, as yet unidentified <span class="yellow">human</span> ribonuclease (or ribonucleases) more closely related to those of the mR cluster cannot be ruled out. Perhaps the most important issue raised by this work is the question of why so much evolutionary energy has been devoted to enlarging and diversifying the RNase A gene superfamily. To date, this superfamily now includes several distinct lineages, two species-limited clusters, and the two most rapidly evolving functional coding sequences known among primates (22). The question of evolutionary energy takes on particular significance here, where five functional ribonucleases have emerged in what appears to be a remarkably short period of evolutionary time (15). It is conceivable that each of these ribonucleases might serve a unique but related function. On this point, a number of studies have suggested a distinct role for the <span class="yellow">human</span> ribonuclease ECP in eosinophil-mediated host defense (6,7); in addition, we have recently shown that eosinophil ribonucleases can inhibit retroviral transduction of <span class="yellow">human</span> target cells (23). The possibility that certain ribonucleases have diverged to promote distinct and specific host defense-related activities remains an intriguing hypothesis. ACKNOWLEDGEMENTS We would like to thank Dr Jianzhi Zhang for helpful discussions, and Drs Harry L. Malech and John I. Gallin for their ongoing suppport of the work in our laboratory. D.B. is a Howard Hughes Medical Institute-National Institutes of Health Research Scholar REFERENCES 1 D'Alessio, G. and Riordan, J.F. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. 2 Sorrentino, S. and Libonati, M. (1997) FEBS Lett. 404, 1�5. 21 22 23 24 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br><br> 4239 <br><br> Beintema, J.J., Breukelman, H.J, Carsana, A., and Furia, A. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. Confalone, E., Beintema, J.J., Sasso, M.P. Carsana, A., Palmieria, M., Vento, M.T. and Furia A. (1995) J. Mol. Evol. 41, 850�858. Futami, J., Tsushima, Y., Murato, U, Tada, H., Sasaki, J., Seno, M., and Yamada, H. (1997) DNA Cell Biol. 16, 413�419. Ackerman, S.J. (1993) In Makino, S. and Fukuda, T. (eds) Eosinophils: biological and clinical aspects. CRC Press, Inc., Boca Raton, FL Spry, C.J.F. (1988) Eosinophils: a comprehensive review and guide to the scientific and medical literature. Oxford University Press, Oxford, UK. Beintema, J.J., Hofsteenge, J., Iwama, M., Morita, T., Ohgi, K., Irie, M., Sugiyama, R.H., Schieven G.L., Dekker, C.A., and Glitz, D.G. (1988) Biochemistry 27, 4530�4538. Sorrentino, S., Tucker, G.K., and Glitz, D.G. (1988) J. Biol. Chem. 263, 16125�16131. Vallee, B.L. and Riordan, J.F. (1988) Adv. Exp. Med. Biol. 234, 41�53. Moroianu, J. and Riordan, J.F. (1994) Proc. Natl. Acad. Sci. USA 91, 1677�1681. Seno, M., Futami, J., Tsushima, Y., Akutagawa, K., Kosaka, M., Tada, H. and Yamada, H. (1995) Bioch. Biophys. Acta 1261, 424�426. Rosenberg, H.F. and Dyer, K.D. (1995) Nucleic Acids Res. 23, 4290�4295. Rosenberg, H.F. and Dyer, K.D. (1996) Nucleic Acids Res. 24, 3507�3513. Larson, K.A., Olson, E.V., Madden, B.J., Gleich, G.J., Lee, N.A. and Lee, J.J. (1997) Proc. Natl. Acad. Sci. USA 93, 12370�12375. Nakajima-Iyima, S., Hamada, H., Reddy, P. and Kakanaga, T. (1985) Proc. Natl. Acad. Sci. USA 82, 6133�6137. Li, W.-H. and Graur, D. (1991) Fundamentals of Molecular Evolution. Sinauer Associates, Sunderland, MA. Rosenberg, H.F. and Dyer, K.D. (1995) J. Biol. Chem. 270, 21539�21544. Watanabe, H., Katoh, H, Ishii, M., Komoda, Y., Sanda, A., Takizawa, Y., Ohgi, K. and Irie, M. (1988) J. Biochem. 104, 939�945. Sasso, M.P., Carsana, A., Confalone, E., Cosi, C., Sorrentino, S., Viola, M., Palmieri, M., Russo, E. and Furia, A. (1991) Nucleic Acids Res. 19, 6469�6474. Palmieri, M., Carsana, A., Furia, A. and Libonati, M. (1985) Eur. J. Biochem. 152, 275�277. Rosenberg, H.F., Dyer, K.D., Tiffany, H.L. and Gonzalez, M. (1995) Nature Genet. 10, 219�223. Domachowske, J.B. and Rosenberg, H.F. (1997) J. Leukoc. Biol., 62, 363�368. Iwama, M., Sanda, A., Ohgi, K., Hofsteenge, J., and Irie, M. (1993) Biosci. Biotech. Biochem. 57, 2133�2138. <br><br>  <h3>pmcA1521150</h3>Peculiarities of carcinogenesis under simultaneous oral administration of benzo(a)pyrene and o-cresol in <span class="yellow">mice</span>.
Abstract
A modifying influence of ortho-cresol (o-cresol) on the carcinogenic effect of benzo(a)pyrene (BaP) with combined oral administration to CC57Br <span class="yellow">mice</span> had been found. During simultaneous administration of o-cresol (1 mg) and BaP (1 mg), the incidence of tumors, the multiplicity of tumors, and the degree of malignancy all increased, but the latency was shortened. When o-cresol was administered before or after BaP (in identical doses), the carcinogenic effect was weakened. When o-cresol (10 mg) and BaP (5 mg) were administered simultaneously, the incidence of malignant tumors was similar to controls receiving BaP only (13.8%), indicating inhibition of carcinogenesis.<br><br><br><br>
 Environnmental Health Perspectives Supplements Vol. 101 (Suppl. 3): 341-344 (1993) <br><br> Peculiarities of Carcinogenesis under Simultaneous Oral Administration of <br><br> Benzo(a)Pyrene and o-Cresol in <span class="yellow">Mice</span> <br><br> by N. Ya. Yanysheva,' N. V. Balenko, 1. A. Chernichenko,1 and V. F. Babiy' <br><br> A modifying influence of ortho-cresol (o-cresol) on the carcinogenic effect of benzo(a)pyrene (BaP) with combined oral administration to CC57Br <span class="yellow">mice</span> had been found. During simultaneous administration of o-cresol (1 mg) and BaP (1 mg), the incidence of tumors, the multiplicity of tumors, and the degree of malignancy all increased, but the latency was shortened. When o-cresol was administered before or after BaP (in identical doses), the carcinogenic effect was weakened. When o-cresol (10 mg) and BaP (5 mg) were administered simultaneously, the incidence of malignant tumors was similar to controls receiving BaP only (13.8%), indicating inhibition of carcinogenesis. <br><br> Introduction <br><br> Numerous experiments have shown a modifying influence of different chemicals on benzo(a)pyrene (BaP)induced carcinogenic effects. Both enhancing or weakening effects have been seen (1-12). In some experiments ubiquitious environmental carcinogenic and toxic chemical pollutants were used (13-17). <br><br> The stimulating effect of phenol, nitrogen oxides, and sulfur dioxide upon BaP's blastogenic action on the respiratory tract and phenol on the digestive tract (forestomach) have been reported (12-18). We have established the relationship between the doses of carcinogenic and toxic agents and their modifying effects (17,18). The maximal enhancing effect was observed after BaP (2.5 mg) and NO2 (0.87 mg/m3) were exposed to <span class="yellow">rat</span> respiratory tract. The effect weakened with decreasing dose. At concentrations at the MPC level (MPC BaP, 1 ng/m3, MPC NO2, 0.04 mg/m3) no effect was seen. Phenol at 1 mg orally to CC57Br <span class="yellow">mice</span> enhanced the BaP (1 mg) effect, but no effect was seen at 0.02 mg. Phenolic compounds showed either enhancing or inhibiting effects on carcinogenesis depending on their chemical structure (16,18-25). <br><br> This report presents the results of combined oral administration of BaP and artho-cresol (o-cresol) in <span class="yellow">mice</span>, <br><br> 'Ukrainian State Medical Center for Environmental Health, Kiev, Ukraine. <br><br> Address reprint requests to N. Y. Yanysheva, Ukrainian State Medical Center for Environmental Health, Popudrenko St. 50, 253660 Kiev-94, Ukraine. <br><br> chemicals commonly found in ambient water because of their industrial use (coke chemistry, oil processing, shale processing, and other industries) (27-30). <br><br> Materials and Methods <br><br> The experiment was conducted on CC57Br female <span class="yellow">mice</span> weighing 12-14 g. Animals were divided in 15 groups. This experiment included three types of sequential combinations for the introduction of compounds: a) simultaneous BaP and o-cresol administration; b) BaP after o-cresol (stage 1); and c) BaP before o-cresol (stage 2). Appropriate controls were included (Table 1). <br><br> The chemicals were administered twice per week (for a total of 10, 20, and 40 doses) using a syringe and a needle with a soldered olive on its tip. BaP (1 or 5 mg) or o-cresol (0.02, 1, or 10 mg) was placed in triethylene glycol (TEG) and administered as 0.1-mL water solutions. The evaluation of o-cresol's modifying effect on the incidence of tumors, especially malignant tumors, the tumor latency period (tl), and the average time (ta) of the appearance of tumors, as well as the multiplicity index, M (the average number of tumors per animal for tumor-bearing animals) was recorded. Because the M could be identified at the initial stage of carcinogenesis by the third month after the beginning of the trial when neoplasms began to emerge, moribund animals were killed by ether inhalation if they did not die spontaneously. <br><br> The experiments with large chemical doses (BaP, 5 mg; o-cresol, 10 mg) lasted for 30 weeks. Some animals were killed after the 1st, 3rd, 5th, and 10th procedures for the <br><br> YANYSHEVA ET AL. <br><br> evaluation of initial stages of carcinogenesis; others were killed at 26 and 30 weeks. The stomachs were distended with formalin solution and the mucosa examined macroscopically. All tumors > 1 mm in diameter were recorded. Organs and tissues were fised in 10% neutral formalin solution, embedded in paraffin and routine histological slides prepared. Microscopic data were processed according to Mostkovoy (30). <br><br> Results <br><br> The data from this study demonstrate the modifying influence of o-cresol on BaP-induced carcinogenesis with combined oral exposure. The combined exposure and BaP alone caused benign and malignant epithelial tumors of the forestomach. The benign neoplasms were papillomas, and the malignant neoplasms were invasive carcinomas. <br><br> The combined administration of BaP and o-cresol showed different results depending on doses and sequential combinations of both chemicals. As shown in Table 2, the simultaneous administration of BaP (1 mg) and o-cresol (1 mg) affected all parameters of carcinogenesis that were measured. This included a significant increase in the incidence of tumors, shortening of the time to the appearance of the first tumor, and the mean time of tumor <br><br> development as compared to animals that received 1 mg of BaP alone and had tumors. One hundred percent of the <span class="yellow">mice</span> that received 5 mg of BaP alone had tumors. <br><br> Shortening of the time to appearance of malignant tumors was also observed: the <span class="yellow">mice</span> with forestomach cancer died between 23 and 26.2 weeks after the beginning of experiment, but the controls survived to 58.5 weeks. Muliple small metastases in the lungs and mediastinum were found in 42.8% of <span class="yellow">mice</span>, which signifies a high degree of malignancy. <br><br> o-Cresol at 0.02 mg did not modify carcinogenesis in comparison to the control. o-Cresol administration before or after BaP did not modify tumor incidence or the multiplicity index as compared to control, but the latency period was longer (Table 2). When o-cresol was administered after BaP (stage 2), there was an absence of malignant tumors. <br><br> Besides the quantitative aspects, other peculiarities of carcinogenesis in <span class="yellow">mice</span> simultaneously administered BaP and o-cresol should be noted. The combined chemical administration resulted in diffuse verrucosa vegetations over the forestomach surface, especially near the greater curvature. Highly aggressive malignant neoplasms developed earlier with more metastases as compared to control. <br><br> Many <span class="yellow">mice</span> were emaciated, and the tumors could be <br><br> Table 1. Scheme of BaP and o-cresol combined action under different regimes of oral administration. <br><br> Theatmenta <br><br> No. of <span class="yellow">mice</span>                        Stage 1                                            Stage 2 <br><br> Groups of        in each                                             No. of                                           No. of <br><br> animals          group        Substances           Dose, mg      applications    Substances         Dose, mg       applications <br><br> 1              55         BaP, o-cresol          5 +10            10          -                     -               2              45          BaP                     5              10          -                     -               3              40          TEG                     la             10          -                     -               4              45          TEG, o-cresol        1b + 10           10          -                     -               5              30          BaP, o-cresol         1+1              20          -                     -               6              30          BaP, o-cresol        1 + 0.02          20          -                     -               7              30          o-Cresol                1              20          BaP                   1               20 8              30          BaP                     1              20          o-Cresol              1               20 9              30          BaP                     1              20                                -               10              30          -                       -              -           BaP                   1               20 11              30         o-Cresol                 1              20                                -               12              30          -                      -               -           o-Cresol              1               20 13              30         o-Cresol + TEG         1+2'             20                                -               14              30         o-Cresol + TEG        0.02 +2b          20 <br><br> 15              30         TEG                     2b              20          -                     -               Abbreviations: BaP, benzo[a]pyrene; TEG, triethylene glycol. <br><br> Table 2. Occurrence of forestomach tumors in CC57Br <span class="yellow">mice</span> after combined oral administration of BaP and o-cresol. <br><br> Number of animals with forestomach tumors <br><br> Order of administration of substances             Total                         Benign                       Malignant <br><br> (dose, mg)                             Absolute       %        %        M         t,         ta         %        t,        ta BaP (5), o-cresol (10)                    29       100.0     86.2       ND        ND        ND        13.8      ND        ND BaP (1), o-cresol (1)                     19        95.0     60.0       9.6       10.7      16.3      35.0      23.0      25.2 BaP (1), o-cresol (0.02)                   7        35.0     30.0       1.6       13.5      19.8       5.0      56.8      56.8 BaP (1), stage 1, o-cresol (1), stage 2    7        31.8     31.8       1.4       16.2      31.1       0         0         0 <br><br> BaP (1), stage 2                           6        35.3     29.4       2.8       15.8      41.2       5.9      24.8      24.8 BaP (5), stage 1                          18       100.0     50.0       ND        ND        ND        50.0      ND        ND BaP (1), stage 1                          8         33.3     28.7       1.4       14.0      21.3       4.6      58.5      58.5 BaP (1), stage 2                           7        36.8     31.6       2.8       10.0      13.3       5.2      55.7      55.7 <br><br> Abbreviations: BaP, benzo[a]pyrene; 0, not observed; ND, not determined; t1, time of the first tumor appearance in weeks; ta, mean time of tumor development in weeks; M, multiplicity. <br><br> 342 <br><br> CARCINOGENESIS OF BaP AND o-CRESOL                           343 <br><br> palpated through the abdominal wall. At autopsy, enlargement of the stomach with tuberculous white superficial vegetations were observed. The forestomach and glandular part of the stomach was often obliterated by tumor masses. <br><br> The stomach was often adhered to the pancreas, liver, and mesentery. Hemmorrhage and inflammation were found in tumors foci. When o-cresol was introduced before BaP, the tumors were more frequently found closer to the small curvature of the stomach between the forestomach and esophageal entrance. Over the large curvature less prominent mucosal folds were observed. Microscopically, a decrease in mucosa papillae, epithelial atrophy (one to two cullular layers), decreased keratonization, and nuclear pycnosis were observed. With the simultaneous administration of large doses of o-cresol and BaP, the final carcinogenic effect (30 weeks after the first dose) was similar to BaP alone but differences were observed at the earlier stages (after the 1st, 3rd, 5th, and 10th exposures). <br><br> In the BaP control, multiple forestomach epithelial proliferative and hyperplastic changes were found after the third dose. Multiple papillomas occurred (9-15 in each <span class="blue">mouse</span>). The stomach's mucosal folds appeared thickened diffusely, then papillomas appeared and finally merged together. <br><br> Approximately half of the tumors were malignant. The neoplasms filled almost the whole forestomach cavity and infiltrated the wall with tuberculous vegetations, which were visible on the serosal surface. Thmor infiltration in the liver, pancreas, and wide dissemination of peritoneum were also observed. In <span class="yellow">mice</span> simultaneously administered BaP and o-cresol, the proliferative alterations of forestomach epithelium were seen after the third dose. However, they occurred as the single foci at damaged mucosa and even in the later stages were not diffuse. Adjacent to the hyperplastic foci, the mucosa was atrophied with decreased keratonization. The epithelial cells also showed cytoplasm coagulation and nuclear pycnosis. Edema, inflammation of the mucosa, submucosa microabscesses, and erosion were seen. There were fewer papillomas per <span class="blue">mouse</span> (four to eight in each <span class="blue">mouse</span>) than in the control group. Even in the late stages the papillomas were isolated and elevated on the atrophied mucosal folds. <br><br> In the final experiment the papillomas prevailed as compared to the previous experiments. Only 4 out of 29 <span class="yellow">mice</span> (13.8%) developed malignant tumors. Neoplasms were smaller and appeared as single verrucosa vegetations 5-10 mm in diameter. Thus, the toxic dose of o-cresol inhibits the carcinogenic process of induced forestomach tumors by decreasing multiplicity, frequency, and percentage of malignant tumors. <br><br> Discussion <br><br> Our results and the literature suggest a hypothesis on modifying carcinogenesis mechanisms. The primary effect of the toxic agents, including carcinogenic agents, may relate to cellular membrane damage with the subsequent increased permeability (31-35), which may be the mechanism of the o-cresol. <br><br> With simultaneous introduction of o-cresol at low toxic doses and BaP there may be increased carcinogen penetration to the target cells. In addition, membrane damage may alter other cellular systems responsible for energy and xenobiotic detoxication. Damage of these processes may activate free-radical reactions, glycolysis, or alter carcinogenic metabolism, which promotes the oncogenic effect. The o-cresol effect on these systems was confirmed by our previous investigations on cytochrome P450, ferrosulfuric nonhemic proteins, and semiquinon radical content affected by the combined action of BaP and o-cresol (36). <br><br> Another effect was obtained with BaP exposure after o-cresol. The atrophic alterations in the stomach induced by 2.5 months of o-cresol administration might decrease the natural conditions of retention (muscosal folds, frontier torn), and thus shorten the contact of BaP with the forestomach. In our opinion, this explains the decreased carcinogenic effect seen after o-cresol exposure. <br><br> Considering the toxic effect of o-cresol at low toxic doses (1 mg) after BaP and the effect of a super toxic dose (10 mg) simultaneously with BaP administration, it is possible that the inhibition of carcinogenesis in both trials was related to the toxic effects of o-cresol. Cytotoxicity may not only hamper tumor growth, but also promote the regression of inducible and spontaneous neoplasms (37-39). There is also a possibility that BaP damages metabolic systems and decreases resistance to carcinogenesis. <br><br> Conclusions <br><br> The data obtained demonstrate that simultaneous administration of BaP modifies the induced carcinogenesis depending on the dose and the sequence of administration. Administered with BaP (1 mg), a low toxic o-cresol dose (about minimally effective) produces an enhanced cocarcinogenic effect reflected in the incidence, frequency, multiplicity, and degree of malignancy of forestomach tumors. <br><br> o-Cresol administration at low toxic doses before or after BaP at the same dose level (1 mg) and its administration at super toxic doses (10 mg) with the BaP optimal dose (5 mg) may inhibit carcinogenesis. Simultaneous introduction of a noneffective o-cresol dose (0.02 mg) with the BaP (1 mg) does not change its carcinogenic activity. Controlling both chemicals in the environment is the most effective measure of preventing potential risk and is undoubtedly of paramount significance. <br><br> REFERENCES <br><br> 1. Bingham, E., and Falk, H. L. Environmental carcinogenesis. The <br><br> modifying effect of cocarcinogens on the threshold response. Arch. Environ. Health 19: 779-783 (1969). <br><br> 2. Goldschmidt, B. M., Katz, C., and Van Duuren, B. L. The cocar<br><br> cinogenic activity of non-carcinogenic aromatic hydrocarbons. Proc. Am. Assoc. Cancer Res. 17: 84 (1973). <br><br> 3. Kaufman, D. C., and Madison, R. M. Synergistic effects of <br><br> benzo(a)pyrene and N-methyl-N-nitrosourea respirators carcinogenesis in <span class="yellow">Syrian golden hamsters</span>. In: Proceedings of the Sym<br><br> 344                                     YANYSHEVA ET AL. <br><br> posium on Epithelial Respiratory Carcinogenesis and Bioassays. Batelle Seattle Research Center, Seattle, WA, 1974, p. 17. <br><br> 4. Montesano, R., Saffiotti, U, Ferrero, A, and Kaufman, D. Synergistic <br><br> effects of benzo(a)pyrene and diethylnitrosamine on respiratory carcinogenesis in hamsters. J. Natl. Cancer. Inst. 5: 1395-1397 (1974). 5. Pfeiffer, E. H. Investigations of the carcinogenic burden by air <br><br> pollution in <span class="yellow">man</span>. VII. Studies on the ontogenetic interaction of polycyclic aromatic hydrocarbons. Zbl. Bakteriol. Parasitenk. Infektion. Hyg. Abt. B 158(1): 69-83 (1973). <br><br> 6. Slaga, T. J., Jecker, L., Bracken, W. M., and Weeks, C. E. The effects of <br><br> weak or non-carcinogenic polycyclic hydrocarbons on 7,12-dimethylbenz(a)anthracene and benzo(a)pyrene skin tumor-initiation. Cancer Lett. 7: 51-59 (1979). <br><br> 7. Topping, D. C., Martin, D. H., and Nettesheim, P. Determination of <br><br> cocarcinogenic activity of benzo(a)pyrene for respiratory tract mucosa. Cancer Lett. 11: 315-321 (1981). <br><br> 8. Ilnitsky, A. P., Voronin, V. M., and Solenova, M. G. Carcinogenic effect <br><br> of benzo(a)pyrene and its combinations with other carbohydrates at their oral intake [in Russian]. Voprosy Onkol. 10: 71-75 (1974). <br><br> 9. Kurljandsky, B. A., Medvedovsky, A. G., and Mashbits, F. D. On <br><br> cocarcinogenic effect of toxic factors of low intensity [in Russian]. Gig. Sanit. 10: 83-85 (1972). <br><br> 10. Thrusov, V. S., and Andrianov, L. A. On combined effect on the skin of <br><br> single application of 7,12-dimethylbenz(a)anthracene and repeated applications of benz(a)pyrene [in Russian]. Bull. Eksper. Biol. Med. 3: 80-83 (1972). <br><br> 11. Yanysheva, N. Ya., Chernichenko, I. A., Balenko, N. V, and Kireeva, <br><br> I. S. Carcinogenic substances and their hygienic standardization in the environment [in Russian]. Zdorovja, Kiev, 1977. <br><br> 12. Laskin, S., Kushner, M., and Drew, R. T. Studies in pulmonary <br><br> careinogenesis. In: Inhalation Carcinogenesis (M. G. Hanna, P. Nettesheim, and J. R. Gillbert, Eds.) U.S. Atomic Energy Commission Symposium, Series 18, 1970, p. 321. <br><br> 13. Laskin, S., Kuschner, M., and Drew, R. T. Inhalation exposure to <br><br> sulphur dioxide and benzo(a)pyrene. Oak Ridge National Laboratory, Gatlinburg, TN, 1970, pp. 322-351. <br><br> 14. Kuschner, M., and Laskin, S. Experimental models in environmental <br><br> carcinogenesis. Am. J. Pathol. 1: 183-196 (1971). <br><br> 15. Komine, T. Influences of sulfur dioxide and 3,4-benzpyrene on the <br><br> respiratory organ of <span class="yellow">rats</span>. Hokkaido Igaku Zasshi 3: 189-203 (1977). 16. Skvortsova, N. N. Role of some common atmospheric pollutions in <br><br> lung blastomogenesis [in Russian]. In: Handbook of Hygiene of Atmospheric Air (K. A. Bushtueva, Ed.), Meditsina, Moscow, 1976, pp. 384-391. <br><br> 17. Yanysheva, N. Ya., Balenko, N. V, Chernichenko, I. A, and Babiy, V. F. <br><br> Quantitation of modified effects of nitrogen oxides on carcinogenicity of benz(a)pyrene effect [in Rusian]. Gig. Sanit. 7: 7-9 (1986). <br><br> 18. Yanysheva, N. Ya., Balenko, N. V., Chernichenko, I. A., Babiy, V. F., <br><br> Bakanova, G. N., and Lemeshko, L. P. Peculiarities of manifestation of carcinogenesis at combined effect of benz(a)pyrene and phenol depending on regimen of intake into the organism [in Russian]. Gig. Sanit. 4: 29-33 (1988). <br><br> 19. Van Duuren, B. L., Katz, C., and Goldschmidt, B. M. Brief communica<br><br> tion: cocarcinogenic agents in <span class="yellow">tobacco</span> carcinogenesis. J. Natl. Cancer Inst. 51: 703-705 (1973). <br><br> 20. Van Duuren, B. L., and Goldschmidt, B. M. Cocarcinogenic and <br><br> tumor-promoting agents in <span class="yellow">tobacco</span> carcinogenesis. J. Natl. Cancer Inst. 56: 1237-1242 (1976). <br><br> 21. Wattenberg, L. W., Coccia, J. B., and Lam, L. K. Inhibitory effects of <br><br> phenolic compounds on benzo(a)pyrene-induced neoplasia. Cancer Res. 8: 2820-2823 (1980). <br><br> 22. Bogovski, P. A, and Mirme, H. Yu. Cocarcinogenicity of phenols from <br><br> Estonian shale tars (oils). Environ. Health Perspect. 30: 177-178 (1979). <br><br> 23. Gubergritz, M. Ya., and Kirso, U. E. Carcinogenic properties, struc<br><br> ture and reactive capacity of phenols [in Russian]. Voprosy Onkol. 8: 96-100 (1970). <br><br> 24. Mirme, H. Yu. Modifying effect of water-soluble shale phenols upon <br><br> carcinogenesis [in Russian]. In: Problems of Prevention of Pollution of Environment by Carcinogenic Substances (M. M. Shabad, Ed.), Tallinn, Valgus, 1972, pp. 16-18. <br><br> 25. Kirso, U. E., Pashin, Yu. V., Bakhitova, L. M., and Kjung, A. I. Effect <br><br> of antioxidants on carcinogenic and mutagenic activity of benzo(a)pyrene [in Russian]. Voprosy Onkol. 4: 70-75 (1985). <br><br> 26. Veldre, I. A. Comparative assessment of toxic effect of some water<br><br> soluble phenols [in Russian]. In: Proceedings of the Scientific Conference on Urgent Problems Related with Decrease of Infectious Diseases Morbidity and with Hygienic Problems. Tallinn, 1968, pp. 200-202. <br><br> 27. Substances possessing cocarcinogenic effect. In: Carcinogenic Sub<br><br> stances in the Environment of <span class="yellow">Man</span> (L. M. Shabad and A. P. Ilnitsky, Eds.), Budapest, 1979, pp. 111-113. <br><br> 28. Djatlovitskaya, F. G., and Maktas, E. D. Separate determination of <span class="yellow">fly</span> <br><br> phenols in water using thin-layer chromatography [in Russian]. Gig. Sanit. 6: 60-63 (1965). <br><br> 29. Kostovetsky, Ya. I., and Zholdakova, Z. I. About hygienic rate setting <br><br> of phenol in the water of waterpools [in Russian]. Gig. Sanit. 7: 7-10 (1971). <br><br> 30. Mostkovoy, M. I. Practicum on variational-statistical processing of <br><br> clinical material [in Russian]. Ashkhabad, 1954, p. 132. <br><br> 31. Weinstein, I. B. The scientific basis for carcinogen detection and <br><br> primary cancer prevention. Cancer 47: 1133-1141 (1981). <br><br> 32. Merkulov, A. I, and Skvortsova, R. I. About toxic effect of phenol [in <br><br> Russian]. Gig. Sanit. 1: 79-80 (1984). <br><br> 33. Weinstein, I. B., Mufson, R. A., Lee, L. S., et al. Membrane and other <br><br> biochemical effects of the phorbol esters and their relevance to tumor promotion. In: Carcinogenesis. Fundamental Mechanisms and Environmental Effects. London, 1980, pp. 543-563. <br><br> 34. Yamasaki, H., and Weinstein, I. B. Cellular and molecular mecha<br><br> nisms of tumor promotion and their implications for risk assessment. In: Methods for Estimating Risk of Chemical Injury: <span class="yellow">Human</span> and Non-<span class="yellow">human</span> Biota and Ecosystems (V. B. Vouk, G. C. Butler, D. G. Hoel, and D. B. Peakall, Eds.), SCOPE, 1985, pp. 155-180. <br><br> 35. Karu, T, Kirso, U. E., and Andrianov, L. A. Dynamics of resorption of <br><br> 3,4-benzpyrene with phenols from <span class="blue">mouse</span> skin [in Russian]. Voprosy Onkol. 5: 80-84 (1973). <br><br> 36. Yanysheva, N. Ya., Yurkovskaya, T. N., Beregovskaya, N. N., et al. <br><br> Metalloenzymic complexes of energetic and detoxicating systems of cell during benz(a)pyrene-induced carcinogenesis. In: Proceedings of the 4th National Congress of Oncology with International Participation. Sofia, 1985, p. 102. <br><br> 37. Mizell, M. Anuran (Lucke) tumor breakdown in regenerating anuran <br><br> tadpole tails. Anat. Res. 137: 382-383 (1960). <br><br> 38. Sheremetieva, E. N. Spontaneous melanoma in regenerating tails of <br><br> axolotis. J. Exp. Zool. 158: 101-122 (1965). <br><br> 39. Tsonis, P. A, and Eguchi, G. Carcinogens on regeneration. Effects on <br><br> N-methyl-N-nitro-N-nitrosoguanidine and 4-nitrogninoline-1-oxide on limb regeneration in adult newts. Differentiation 20: 52-60 (1981). <h3>pmcA1851977</h3>Molecular Basis for a Lack of Correlation between Viral Fitness and Cell Killing Capacity
Abstract
The relationship between parasite fitness and virulence has been the object of experimental and theoretical studies often with conflicting conclusions. Here, we provide direct experimental evidence that viral fitness and virulence, both measured in the same biological environment provided by host cells in culture, can be two unrelated traits. A biological clone of <span class="yellow">foot-and-mouth disease virus</span> acquired high fitness and virulence (cell killing capacity) upon large population passages in cell culture. However, subsequent plaque-to-plaque transfers resulted in profound fitness loss, but only a minimal decrease of virulence. While fitness-decreasing mutations have been mapped throughout the genome, virulence determinants—studied here with mutant and chimeric viruses—were multigenic, but concentrated on some genomic regions. Therefore, we propose a model in which viral virulence is more robust to mutation than viral fitness. As a consequence, depending on the passage regime, viral fitness and virulence can follow different evolutionary trajectories. This lack of correlation is relevant to current models of attenuation and virulence in that virus de-adaptation need not entail a decrease of virulence.
Virulence expresses the harm that parasites inflict upon their hosts. Many studies have addressed the basis of virulence and its effect on host and parasite survival. It has generally been accepted that one of the components of parasite virulence is fitness, or the capacity of the parasite to multiply in its host. Some models have equated virulence with fitness. In the present study, we use <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) to document that virulence and fitness—measured in the same biological environment provided by cells in culture—can be unrelated traits. This has been achieved by multiplying the virus in a manner that mutations accumulated in its genome. Mutations decreased fitness dramatically, but not virulence. Chimeric and mutant viruses were constructed to show that virulence is influenced by only some of the <span class="yellow">FMDV</span> genes, while fitness is influenced by the entire genome. For this reason, virulence is more robust (“resistant”) than fitness to the effects of deleterious mutations. The fact that virulence can be unrelated to fitness has implications for the design of anti-viral vaccines because it suggests that it may be possible to design high fitness, low virulence strains to stimulate the host immune response. Furthermore, in modelling studies it cannot be assumed that virulence is equal to fitness.<br><br><br><br>Introduction
The relationship between fitness and virulence is an unsettled question, and sometimes fitness is considered a component of the virulence phenotype of parasites. RNA viruses are ideal systems to address this important question because of their high mutability and fecundity, which result in a potential for rapid evolution, and also because of the availability of quantitative fitness and virulence assays.
RNA viruses replicate as complex and dynamic mutant spectra, termed viral quasispecies. Key to quasispecies dynamics are mutation rates in the range of 10−3 to 10−5 substitutions per nucleotide copied, and competition among continuously arising variant genomes [1–4], which prompt rapid movements in sequence space, with corresponding changes of position in the fitness landscape [5]. Indeed, large population passages of RNA viruses in cell culture permit competitive optimization of mutant distributions that generally result in fitness gain [6,7], while repeated bottleneck events (experimentally realized as plaque-to-plaque transfers) lead to random accumulation of deleterious mutations (operation of Muller's ratchet [8]) and result in average fitness decreases [9–13]. Fitness recovery of low fitness <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) clones occurs mainly with introduction of mutations along the genome, with very few true reversions.
An understanding of the consequences of fitness variation for viral virulence is a key question for viral pathogenesis and evolution. Here, we approach this issue with <span class="yellow">FMDV</span>, an important viral pathogen in veterinary medicine [14], and one that fully participates of quasispecies dynamics. Our laboratory has characterized multiple <span class="yellow">FMDV</span> variants that derive from one original biological clone, and that occupy widely different fitness levels when replicating in a defined environment in cell culture. We define fitness as the replication capacity of a mutant <span class="yellow">FMDV</span>, relative to a reference <span class="yellow">FMDV</span>, in direct growth-competition upon coinfection of baby <span class="yellow">hamster</span> kidney 21 (BHK-21) cells [15–17]. Fitness of <span class="yellow">FMDV</span> in BHK-21 cells is a multigenic trait [7].
In the present study, we define virulence of <span class="yellow">FMDV</span> as the capacity of the virus to kill BHK-21 cells under a standard set of cell culture conditions [18]. Thus, the <span class="yellow">FMDV</span>–BHK-21 system offered a means to investigate in a direct and comparative fashion the relationship between fitness and virulence of a virus, measured in the same biological environment provided by BHK-21 cells. We describe the behavior of an <span class="yellow">FMDV</span> clone (
				), which has a history of repeated serial plaque-to-plaque transfers in BHK-21 cells [11], that attained a very low fitness value relative to its parental reference virus (C-S8c1), and yet, its virulence for BHK-21 cells was significantly higher than that of C-S8c1. A comparative study of the capacity to kill BHK-21 cells of chimeric <span class="yellow">FMDVs</span> constructed with cDNA copies of the two parental <span class="yellow">FMDVs</span> indicates that the enhanced virulence for BHK-21 cells of the low fitness clone is a polygenic trait, with participation of the regions encoding capsid proteins and non-structural proteins 2A, 2B, and 2C as virulence determinants. Three specific amino acid replacements in 2C have been identified as redundant virulence determinants of <span class="yellow">FMDV</span> for BHK-21 cells. Thus, while large population passages of the virus resulted in a gain of both fitness and virulence, subsequent bottleneck passages resulted in a decrease of fitness but not of virulence.
			
The results suggest that fitness is very vulnerable to mutation in any genomic region. In contrast, because of the involvement of several (but not all) viral genes in virulence, and the redundant effect of three 2C substitutions, virulence is a more robust phenotypic trait than fitness, and less vulnerable to accumulation of mutations. Therefore, we provide direct evidence that viral fitness and capacity to kill cells can (in some cases) be unrelated traits. Furthermore, the relationship between fitness and virulence, of being either linked or unrelated traits, depends on the evolutionary history of the virus. This observation has implications for viral pathogenesis, and sheds light on models of virulence proposed on the basis of theoretical and experimental studies with cellular organisms.<br><br>Results
Inability of <span class="yellow">FMDV</span> 
					 to Establish a Persistent Infection in BHK-21 Cells
				
Several biological clones and populations were obtained by passaging <span class="yellow">FMDV</span> biological clone C-S8c1 [19–22] in BHK-21 cells, either as large population passages or plaque-to-plaque transfers (Figure 1). The biological clones and populations differed up to 236-fold in relative fitness (Table 1). The fitness differences found are expected from previous results on fitness gain upon large population passages of RNA viruses [6,7] and fitness decrease upon plaque-to-plaque (bottleneck) transfers [9–13]. The initial experiment was aimed at testing whether 
					, because of its low fitness (0.11 times that of its parental C-S8c1 [12,23,24] [Table 1]), had an advantage in establishing a persistent non-cytopathic infection in BHK-21 cells as compared with its parental clone, C-S8c1 (Figure 1). A persistent <span class="yellow">FMDV</span> infection is established by growing the cells that survive a standard cytolytic infection with <span class="yellow">FMDV</span> [25]. Confluent monolayers of BHK-21 cells were infected either with C-S8c1 or with 
					 at a multiplicity of infection (MOI) of 0.02–0.1 plaque-forming units (PFU)/cell (2 × 106 cells infected with 4 × 104 −2 × 105 PFU). Unexpectedly, at 24 h postinfection, the cells infected with 
					 showed extensive cytopathology, and at 48 h postinfection, no surviving cells were observed. The frequency of surviving cells in parallel infections with C-S8c1 was 5 × 10−3–9 × 10−3, which is consistent with previous determinations [25]. No persistently infected BHK-21 cell cultures could be established with 
					, despite several attempts. Thus, C-S8c1, which displays a 9-fold higher relative fitness than 
					 in BHK-21 cells, showed a capacity to kill BHK-21 cells that was at least 103-fold lower than the killing capacity of 
					 in the infectivity assay intended to establish a persistent <span class="yellow">FMDV</span> infection.
				<br><br><span class="yellow">FMDV</span> Fitness and Capacity to Kill BHK-21 Cells May Not Be Positively Correlated
The capacity of 
					 to kill BHK-21 cells despite its low fitness in BHK-21 cells led us to quantitatively examine the relationship between fitness of <span class="yellow">FMDV</span> and its capacity to kill BHK-21 cells. To this aim, <span class="yellow">FMDV</span> clones or populations were compared in a cell killing assay, consisting in determining the time required to kill 104 BHK-21 cells as a function of the PFU added (described in Materials and Methods). The results (Figure 2A) indicate that over the time range of 12 h to 48 h postinfection, the number of PFUs needed to kill 104 BHK-21 cells varied logarithmically as a function of time. Similar quantifications of relative virulence were obtained by measuring the PFU needed to kill 104 cells in 24 h, and then by extrapolating the PFU values to 0 h postinfection (Tables 1 and S1). Virulence of 
					 was 29 to 35 times higher than virulence of C-S8c1, despite the latter displaying a 9-fold higher fitness (Tables 1 and S1). The high virulence of 
					 was not due to the plaque-to-plaque transfers, since a high virulence was also quantitated for its parental clone, 
					, and for population C-S8p113 (Figure 2B; Tables 1, 2, and S1). 
					 deviated from a line that correlated relative fitness of <span class="yellow">FMDV</span> and the logarithm of cell killing capacity, as reflected in the decrease of the regression coefficient (R2) (inset in Figure 2A). Probably, this deviation is due to the fact that 
					 lost fitness due to plaque-to-plaque transfers, and the other viruses were not subjected to plaque-to-plaque transfers. On the other hand, virulence determinants were acquired during the large population passages done between C-S8c1 and C-S8c1p113. The 29- to 35-fold higher virulence of 
					 with respect to C-S8c1 (Tables 1 and S1), despite its low fitness, indicates that viral fitness and virulence can be two unrelated traits.
				<br><br>Mapping Virulence Determinants in the 
					 Genome
				
The comparison of the consensus nucleotide sequence of the 
					 genome with that of C-S8c1 revealed a total of 47 mutations (Table S2), leading to 21 amino acid replacements affecting structural and non-structural proteins (Figure 3). To identify the genomic regions associated with the increased virulence of 
					 with respect to C-S8c1, we measured the BHK-21 cell killing capacity of nine chimeric viruses rescued from constructs obtained by introducing fragments of cDNA of the 
					 genome into plasmid pMT28, which encodes infectious C-S8c1 RNA [21] (Figure 4). The results (Figure 5; Tables 2 and S1) show that several genomic regions contribute to the virulence of 
					 for BHK-21 cells, and that the major contributors map within genomic positions 2046 to 3760 (residues encoding part of VP2, VP3, and part of VP1, Figure 5A) and 3760 to 5839 (residues encoding 2A, 2B, 2C, and 3A, Figure 5B). The results exclude the internal ribosome entry site and the 3C- and 3D-coding regions as significant virulence determinants of 
					 for BHK-21 cells (virulence of the relevant chimeric viruses ≤ 2.5, relative to C-S8c1; Tables 2 and S1). Infectious progeny production by each chimeric virus was intermediate between the production of the parental viruses pMT28 and 
					, with no significant differences that could be correlated with virulence (Table 2).
				<br><br>Non-Structural Protein 2C Is a Determinant of the Virulence of <span class="yellow">FMDV</span> for BHK-21 Cells
Amino acid substitutions in <span class="yellow">human rhinovirus</span> protein 2C promoted cytopathology for <span class="blue">mouse</span> L cells [26]. Remarkably, 
					 shares with other <span class="yellow">FMDV</span> clones and populations, notably, MARLS and C-S8p260p3d (the two viruses showing the highest virulence for BHK-21 cells; Figure 2A; Tables 1 and S1), three amino acid substitutions in 2C: S80N, T256A, and Q263H. In addition, MARLS and CS8p260p3d include replacement M283V in 2C, relative to C-S8c1 [27,28]. To test whether any (or a combination) of the three shared amino acid substitutions in 2C contributed to the increased virulence of <span class="yellow">FMDV</span>, each of the mutations was introduced individually into plasmid pMT28 by site-directed mutagenesis, as described in Materials and Methods. Transcripts of the three mutants, termed pMT28 (SN), pMT28 (TA), and pMT28 (QH) (Figure 4), were used to transfect BHK-21 cells, and the viruses obtained were tested with the BHK-21 cell killing assay. Viruses having any of the substitutions in 2C have a virulence intermediate between that of C-S8c1 and 
					 (Figure 6A). To test whether the combination of the three substitutions in 2C could produce an additional increase of virulence, the three mutations were introduced in pMT28 to rescue the triple mutant pMT28 (SN, TA, QH) (Figure 4). The results (Figure 6B) show that the virulence of the triple 2C mutant is similar to the virulence of the individual 2C mutants. The 2C mutations did not significantly affect the infectious progeny production (Table 2). A testable prediction of this result is that the introduction of the wild-type 2C-3A-coding region in the genetic background of 
					 should produce a virus with lower virulence than 
					. Indeed, the results with such a chimeric virus (Figure 5D) indicate that the presence of the 2C- and 3A-coding region as the only genetic region of the pMT28 in the genetic background of 
					 resulted in an <span class="yellow">FMDV</span> with a 2.4- to 4.8-fold lower virulence than 
					. We conclude that mutations in 2C contribute to virulence of <span class="yellow">FMDV</span> for BHK-21 cells.
				
Thus, a virus that evolves towards low fitness levels due to the operation of Muller's ratchet may nevertheless maintain its capacity to kill the same cells in which it displays low fitness. In <span class="yellow">FMDV</span>, the enhanced capacity to kill BHK-21 cells was multigenic, including participation of non-structural protein 2C with three amino acid substitutions acting in a redundant fashion. In conclusion, the results provide a molecular interpretation of why fitness and virulence of an animal virus can follow disparate evolutionary trajectories, culminating in two unrelated traits.<br><br>
Discussion
The capacity of a virus to kill cells is probably influenced by several steps in the virus life cycle, including receptor affinity (which may trigger signalling pathways and alter cell functions) and intracellular viral replication that may lead to metabolic alterations such as transcriptional or translational shut-off [29]. A parallel increase of virulence and fitness as a virus improves its adaptation to a host cell type is expected, since a key parameter that should contribute to the fitness level of a cytophatic virus is the accumulation of infectious particles and its release from cells, which are events often associated with cell killing. This expectation was fulfilled in our experiments (increase of both fitness and cell killing following large population passages of C-S8c1 [Figure 2]), and also in other virus–host systems. In a comparison of two genetically divergent isolates of the whispovirus white spot syndrome virus, whose virulence for the <span class="yellow">shrimp</span> host <span class="yellow">Penaeus monodon</span> was measured by in vivo cumulative mortality rates, virulence correlated with competitive fitness in vivo [30]. The onset of type 1 diabetes by coxsackievirus B strains was linked to the viral replication rate and to the infectious dose [31]. In engineered alphavirus replicons, a direct correlation between the level of RNA replication and cytopathogenicity was observed [32]. At an epidemiological level, a greater replicative fitness of historical versus current <span class="yellow">human immunodeficiency virus type 1</span> (<span class="yellow">HIV-1</span>) isolates was taken as evidence of <span class="yellow">HIV-1</span> attenuation over time, assuming a direct connection between fitness and virulence [33]. In vivo, viral fitness may vary among specific organs, and virulence may be affected only when fitness for some specific target tissues is affected [34].
Replicative fitness is, however, but one of several factors which influence the progression of a viral infection in vivo. In a comparative analysis, R5-tropic and X4-tropic clones of <span class="yellow">HIV-1</span> showed similar replication capacity in mitogen-activated T cells. However, X4 clones were transferred more efficiently than R5 clones from dendritic cells to CD4(+) T cells, a fact that can contribute to the competitive advantage of X4 viruses in AIDS <span class="yellow">patients</span> [35]. <span class="yellow">Simian immunodeficiency virus</span> <span class="yellow">SIVmac239</span> infects both the <span class="yellow">sooty mangabey</span> and the <span class="yellow">rhesus macaque</span>, reaching high viral loads in both hosts, yet it is only virulent for the <span class="yellow">rhesus macaque</span> [36]. Deviations of a positive correlation between viral fitness and virulence were observed also in the plant viruses <span class="yellow">cucumber mosaic virus</span> [37] and <span class="yellow">barley stripe mosaic virus</span> [38]. A study of the effect of lysis timing on <span class="yellow">bacteriophage</span> fitness revealed that a strain with an intermediate lysis time had the highest fitness [39]. The time of transmission may also affect virulence. Nuclear polyhedrosis virus transmitted early to its host, the <span class="yellow">moth</span> <span class="yellow">Lymantria dispar</span>, was more virulent than virus transmitted late, although the latter was more productive because the virus could use more host tissue for replication [40]. In a study of the susceptibility of North American and non–North American breeds of Lymantria to several isolates of the fungus <span class="yellow">Entomaphaga maimaiga</span>, mortality was scored in all cases. However, virulence of the fungus, quantitated by the time of death of Lymantria, was, in some cases, inversely proportional to fitness, quantitated by fungal reproduction in the <span class="yellow">moth</span> [41]. In all these cases, the molecular basis of the lack of positive correlation between fitness and virulence is not understood.
Model for a Lack of Correlation between Viral Fitness and Virulence
The results with <span class="yellow">FMDV</span> clones H5 have documented that both fitness-enhancing and virulence-enhancing mutations can be incorporated in the viral genome in such a fashion that subsequent fitness-decreasing mutations associated with bottleneck (plaque-to-plaque) transfers produce only minimal effects on virulence (Figure 2). The dissection of accompanying molecular events, achieved through quantification of virulence of recombinant and mutant genomes (Tables 2 and S1), provides an interpretation of the lack of positive correlation between virulence and fitness. Multiple fitness-decreasing mutations occur in the course of plaque-to-plaque transfers, distributed throughout the <span class="yellow">FMDV</span> genome [11]. In contrast, determinants of virulence for BHK-21 cells are multigenic, but concentrated mainly in some <span class="yellow">FMDV</span> genomic regions. Similar multigenic but discrete virulence determinants have been described also in other virus–host systems [42,43]. To decrease virulence, mutations occurring randomly in the course of plaque-to-plaque transfers should affect specific genomic sites, and this will occur with a lower probability than fitness-decreasing mutations, which can hit any of the multifunctional picornaviral proteins and regulatory regions [11]. This model is reinforced by the observation that three amino acid substitutions in 2C (S80N, T256A, and Q263H) had a similar effect in enhancing <span class="yellow">FMDV</span> virulence, and the three mutations in the same genome had an effect comparable to each mutation individually (Figure 6; Tables 2 and S1). It is not clear what the basis of the contribution of 2C to virulence for BHK-21 could be. 2C is involved in RNA synthesis and contains a nucleotide-binding domain, although none of the substitutions found in 
					 and 
					 lie within such a domain. An unlikely triple reversion would be required to eliminate the virulence-enhancing effect of the three mutations in 2C. We propose that a higher robustness of the <span class="yellow">FMDV</span> genome with regard to virulence for BHK-21 cells, rather than to replicative fitness in the same cells, underlies the different trajectories followed by fitness and virulence upon subjecting the virus to repeated bottleneck transfers. Obviously, we cannot exclude that parameters of the virus life cycle, other than fitness as measured in our experiments, could correlate with virulence for BHK-21.
				
The comparative analysis of <span class="yellow">FMDV</span> clones and populations shows that shifts in virulence can occur even through the evolution of a single viral clone (C-S8c1), with its restricted genetic diversity prompted by different replication regimes in the same host cells, which also have a clonal origin (see Materials and Methods). We conjecture that the demonstration that fitness and virulence can follow different evolutionary courses has been possible thanks to the consequences of the extreme passage regimes to which the viral populations were subjected: competitive evolution of an ample mutant spectra during repeated large population passages, and accumulation of deleterious (with regard to fitness, but not with regard to virulence) mutations upon plaque-to-plaque transfers (predominance of genetic drift and operation of Muller's ratchet) [12,15].<br><br>Implications for Models of Virulence
It must be emphasized that fitness and virulence are relative values that pertain to a defined physical and biological environment. Virulence determinants of <span class="yellow">FMDV</span>, identified here for BHK-21 cells, need not apply to virulence for the natural animal hosts of <span class="yellow">FMDV</span> [44]. However, the observation of a lack of correlation between fitness and virulence in a <span class="yellow">FMDV</span> clone is relevant to current models of attenuation and virulence, since it shows that more virulent forms of a virus need not have a reproductive advantage, and that viral virulence is not necessarily a byproduct of viral fitness. Even if virulence is regarded as an unavoidable consequence of parasite adaptation [45], virus de-adaptation (fitness loss) need not entail a decrease of virulence.
Most current definitions of virulence include both the ability of the pathogen to multiply and to cause harm to its host; some authors, however, assume a direct relationship between fitness and capacity to produce disease [46–48]. In relating the results with <span class="yellow">FMDV</span> to general models of virulence in host–parasite systems, it must be considered that in the <span class="yellow">FMDV</span> system, evolution of the host BHK-21 cells could not influence <span class="yellow">FMDV</span> evolution, because clonal cell populations with a controlled passage history were supplied in constant numbers at each infection event (see Materials and Methods). Therefore, changes in host density, or mobility, as well as pathogen survival in the external environment, all of which are relevant parameters in virulence models [48,49], cannot play a role in our system. A consistent finding in serial passage experiments is that virulence of a parasite increases with passage number in a new host [50]. The results with <span class="yellow">FMDV</span> infecting BHK-21 cells cytolytically imply that the increase of virulence can be conditioned to the history of passage regimes undergone by a virus.
The invariance of BHK-21 cells in the course of serial cytolytic passages of <span class="yellow">FMDV</span> is in contrast with the parallel system consisting of BHK-21 cells persistently infected with <span class="yellow">FMDV</span> C-S8c1 [25], in which the cells are passaged and coevolve with the resident virus [51]. Host–virus coevolution has generally favored a decrease of viral virulence in the field, a classical example being <span class="yellow">myxoma virus</span> and myxomatosis in <span class="yellow">rabbits</span> [52].
Our comparison of <span class="yellow">FMDV</span> clones did not provide evidence of clones with high fitness and low virulence, which, with regard to natural hosts, is an aim of biomedicine to obtain vaccine strains. Yet, the existence of specific mutations that differentially affect fitness and virulence opens the way to engineer candidate vaccine strains unable to kill the host, while maintaining replicative competence. Virulence is, however, a feature of the host–parasite relationship [46], and the mutations needed to impair virulence are expected to be host-dependent [53,54].<br><br>
Materials and Methods
Cells, viruses, and infections.
The BHK-21 cells used in the present study were cloned by end-point dilution, followed by preparation of a cell stock from a single cell; they were passaged a maximum of 30 times before being used for <span class="yellow">FMDV</span> infection [25,51]. Procedures for cell growth, infection of BHK-21 cell monolayers with <span class="yellow">FMDV</span> in liquid medium, and plaque assays in semi-solid agar medium were carried out as previously described [11,19,25,27]. Mock-infected cells were handled in parallel in all infectivity and plaque assays to monitor absence of viral contamination. The <span class="yellow">FMDVs</span> used in the present study (Figure 1) are (i) the reference clone C-S8c1, which has been assigned a relative fitness of 1.0 [11]. (ii) MARLS, a monoclonal antibody escape mutant isolated from population C-S8c1p213 [55]; MARLS has a fitness of 25 relative to C-S8c1 [24]. (iii) C-S8p260p3d, a standard <span class="yellow">FMDV</span> virus rescued by low MOI passage of C-S8p260. The latter is a virus that evolved by passage of C-S8c1 at a high MOI, which resulted in dominance of two defective <span class="yellow">FMDV</span> genomes (both including internal deletions) that were infectious by complementation, in the absence of standard virus [22,24,28]; C-S8p260p3d has a relative fitness of 20 [24]. (iv) REDpt60, obtained after 60 successive plaque-to-plaque transfers of RED (a monoclonal antibody escape mutant isolated from population C-S8c1p100) [20]; REDpt60 has a fitness of 1.9 relative to C-S8c1. (v) C-S8c1p113, a viral population obtained after 113 serial cytolytic passages of C-S8c1 at a high MOI in BHK-21 cells (2 × 106 BHK-21 cells infected with the virus contained in 200 μl of the supernatant from the previous infection). (vi) Clone 
					, a biological clone isolated from population C-S8c1p113 [11]; its relative fitness is 26 (unpublished data). (vii) Clone 
					, obtained after 95 successive plaque-to-plaque transfers of 
					 [11]; its relative fitness is 0.11 [23].
				<br><br>Cell killing assay.
The capacity of <span class="yellow">FMDV</span> to kill BHK-21 cells was measured as previously described [18,22]. The assay consists in determining the minimum number of PFU required to kill 104 BHK-21 cells after variable times of infection. The assay was performed in M96 multiwell plates with monolayers of 104 BHK-21 cells per well infected with serial dilutions of virus. At different times postinfection, cells were fixed with 2% formaldehyde and stained with 2% crystal violet in 2% formaldehyde. Results are expressed as the logarithm of the number of PFUs needed for complete cell killing (as judged by cell staining with crystal violet, with series of control wells with known numbers of cells) as a function of time postinfection [18,22].<br><br>Determination of relative fitness of 
					.
				
The relative fitness of <span class="yellow">FMDV</span> 
					 was determined by growth competition in BHK-21 cells as previously described [7,10,11,24,56]. <span class="yellow">FMDV</span> 
					 was mixed with appropriate proportions of 
					, which was used as reference virus. This virus has a fitness 8.5-fold higher than that of the reference clone of C-S8c1 in BHK-21 cells [7,10,11,24,56]. Four serial infections were carried out at MOI of 0.1 PFU/cell. The proportion of the two competing genomes at different passages was determined by real-time reverse transcription (RT)–PCR, employing primers 5531wtnew and 
					, which are able to discriminate <span class="yellow">FMDV</span> 
					 RNA from 
					 RNA. The nucleotide sequences of the primers will be provided upon request. The fitness vector obtained for 
					 corresponded to the equation y = 0,0206e1,1074x; R2 = 0.9507. The antilogarithm (base e) of the vector slope is the fitness of the assayed virus relative to the reference virus [56].
				<br><br>cDNA synthesis, molecular cloning, and nucleotide sequencing.
Viral RNA was extracted by treatment with Trizol as previously described [57]. Reverse transcription of <span class="yellow">FMDV</span> RNA was carried out with <span class="yellow">avian myeloblastosis virus</span> reverse transcriptase (Promega, http://www.promega.com) or Transcriptor reverse transcriptase (Roche, http://www.roche.com), and PCR amplification was performed by using either Ampli-Taq polymerase (PerkinElmer, http://las.perkinelmer.com) or an Expand High Fidelity polymerase system (Roche), as instructed by the manufacturers. The <span class="yellow">FMDV</span> genome-specific oligonucleotide primers used have been previously described [22,58]. In all RT-PCR amplifications, negative amplification controls, including all reaction components except template RNA, were run in parallel to monitor absence of contamination.
Chimeric viruses containing selected regions of 
					 in the genetic background of C-S8c1 (Figure 4) were obtained by replacing the corresponding DNA fragment of pMT28 by a cDNA copy of 
					 RNA, using specific restriction sites. To obtain pMT28/
					 (436-2046), a chimera that included nucleotides 436 to 2046 of 
					 (the residue numbering of the <span class="yellow">FMDV</span> genome is as in [11]), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28 DNA digested with the same enzymes. To obtain pMT28/
					 (2046–3760), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760). To obtain pMT28/
					 (3760–5839), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839). To obtain pMT28/
					 (5839–7427), 
					 RNA was amplified by RT-PCR using primers 5531 wt new and C-Not-Pol, and then the cDNA was digested with Rsr II (5839) and Bam HI (7427). To obtain pMT28/
					 (436-3760), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2, and then the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–3760) DNA digested with the same enzymes. To obtain pMT28/
					 (3760–7427), 
					 RNA was amplified by RT-PCR using primers 3R2New and 3CD1, and then the cDNA was digested with Avr II (3760) and Rsr II (5839), and ligated into pMT28/
					 (5839–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (2046–7427), 
					 RNA was amplified by RT-PCR using primers 2R1New and pU, and then the cDNA was digested with Xba I (2046) and Avr II (3760), and ligated into pMT28/
					 (3760–7427) DNA digested with the same enzymes. To obtain pMT28/
					 (436-7427), 
					 RNA was amplified by RT-PCR using primers NR2 and JH2; the cDNA was digested with Hpa I (position 436) and Xba I (2046), and ligated into pMT28/
					 (2046–7427) DNA digested with the same enzymes. To obtain 
					/2C-3A(pMT28), pMT28 was digested with Bgl II (4201) and Rsr II (5839), and a DNA fragment including wild-type 2C-3A-coding region was inserted into pMT28/
					 (436-7427) DNA digested with the same enzymes. DNA ligation, transformation of <span class="yellow">Escherichia coli</span> DH5α, isolation of DNA from bacterial colonies, and characterization of DNA by restriction enzyme digestion were performed by standard procedures [59]. The primers used for molecular cloning and site-directed mutagenesis are described in Table S3.
				
To obtain <span class="yellow">FMDV</span> C-S8c1 containing the mutations found in gene 2C of 
					, plasmid pMT28 was subjected to site-directed mutagenesis using an oligonucleotide including the required nucleotide replacement, and 3R2New or 3CD1 as external oligonucleotide primer (Table S3; Figure 4). A DNA fragment termed A was obtained by subjecting plasmid pMT28 to site-directed mutagenesis using primers (reverse) mutSNu, mutTAu, and mutQHu (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3R2New, forward). A DNA fragment termed B was obtained amplifying pMT28 with primers (forward) mutSNd, mutTAd, and mutQHd (to introduce mutations S80N, T256A, and Q263H, respectively) and an external oligonucleotide primer (3CD1, reverse). DNA fragments A and B, including the desired mutations, were recombined by shuffling PCR using equimolar amounts of DNA fragments and two external primers (3R2New and 3CD1). The DNA with the desired mutation(s) in the 2C gene was digested with Avr II (genomic position 3760) and Rsr II (position 5839), and cloned into pMT28 to generate pMT28 (SN), pMT28 (TA), and pMT28 (QH). To obtain pMT28 (SN, TA, QH), plasmid pMT28 (SN) was subjected to site-directed mutagenesis to introduce mutation T256A in a similar way as described above, and then, plasmid pMT28 (SN, TA) was subjected to site-directed mutagenesis to introduce mutation Q263H. All chimeric viruses and mutants were analyzed by nucleotide sequencing using Big Dye Terminator Cycle Sequencing kit (Abi Prism; PerkinElmer) and sequencer ABI373 as previously described [58]. Sequences were analyzed using DNASTAR 4.0 (http://www.dnastar.com), GeneDoc, and GCC (University of Wisconsin). Each sequence was determined at least twice, with products obtained using different oligonucleotide primers.
				
DNA from pMT28 or its recombinant and mutant derivatives was linearized with Nde I and transcribed with SP6 RNA polymerase as previously described [22,27]. Transcript RNA integrity and concentration were estimated by agarose gel electrophoresis, in parallel runs with known amounts of standard C-S8c1 RNA. BHK-21 cell monolayers (70% confluent, about 1 × 106 cells) were transfected with RNA transcripts (1 μg RNA) using lipofectin as previously described [59]. Virus was collected from the culture supernatant at 72 h post-transfection. The virus obtained by transfection was passaged twice before using it in biological studies. RNA was extracted and sequenced to ascertain that the virus maintained the genomic structure and mutations of the initial transcript.
Consensus genomic nucleotide sequences of <span class="yellow">FMDV</span> clones were obtained by RT-PCR amplification of virion RNA using specific primers [7,22,28].<br><br>
Supporting Information
Accession Numbers
The GenBank accession numbers for the C-S8c1, 
					, 
					, CS8p260p3d, and MARLS genomic sequences are AJ133357, AM409190, AM409325, DQ409185, and AF274010, respectively. Nucleotide and amino acid sequences for picornaviruses can be found at http://www.iah.bbsrc.ac.uk/virus/picornaviridae/SequenceDatabase/3Ddatabase/3D.HTM.
				<br><br><br><br><h3>pmcA2065882</h3>Identification of Two Independent Risk Factors for Lupus within the MHC in United Kingdom Families
Abstract
The association of the major histocompatibility complex (MHC) with SLE is well established yet the causal variants arising from this region remain to be identified, largely due to inadequate study design and the strong linkage disequilibrium demonstrated by genes across this locus. The majority of studies thus far have identified strong association with classical class II alleles, in particular HLA-DRB1*0301 and HLA-DRB1*1501. Additional associations have been reported with class III alleles; specifically, complement C4 null alleles and a tumor necrosis factor promoter SNP (TNF-308G/A). However, the relative effects of these class II and class III variants have not been determined. We have thus used a family-based approach to map association signals across the MHC class II and class III regions in a cohort of 314 complete United Kingdom Caucasian SLE trios by typing tagging SNPs together with classical typing of the HLA-DRB1 locus. Using TDT and conditional regression analyses, we have demonstrated the presence of two distinct and independent association signals in SLE: HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, OR = 2.3) and the T allele of SNP rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001, OR = 2.0) in intron 6 of the class III region gene SKIV2L. Assessment of genotypic risk demonstrates a likely dominant model of inheritance for HLA-DRB1*0301, while rs419788-T confers susceptibility in an additive manner. Furthermore, by comparing transmitted and untransmitted parental chromosomes, we have delimited our class II signal to a 180 kb region encompassing the alleles HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201 alone. Our class III signal importantly excludes independent association at the TNF promoter polymorphism, TNF-308G/A, in our SLE cohort and provides a potentially novel locus for future genetic and functional studies.
Systemic lupus erythematosus (SLE/lupus) is a complex autoimmune disease in which the body's immune system attacks its own tissues, causing inflammation in a variety of different organs such as the skin, joints, and kidneys. The cause of lupus is not known, but genes play a significant role in the predisposition to disease. The major histocompatibility complex (MHC) on Chromosome 6 contains at least 100 different genes that affect the immune system, including the genes with the strongest effect on lupus susceptibility. Despite the importance of the MHC in SLE, the identity of the actual genes in the MHC region that cause SLE has remained elusive. In the present study, we used the latest set of genetic markers present at the MHC in lupus families to identify the actual genes that affect the disease. To our knowledge, we have shown for the first time that two separate groups of genes are involved in SLE. One group of genes alters how the immune system may inappropriately target its own tissues in the disease. How the second set of genes predisposes to SLE is the subject of ongoing study.<br><br><br><br>Introduction
Since the early 1970s, the <span class="yellow">human</span> major histocompatibility complex (MHC) has been shown to be associated with a number of autoimmune, inflammatory, and infectious diseases, and it continues to be the focus of intense research [1]. The recently defined extended MHC (xMHC) encompasses 7.6 Mb of genome on 6p21.3 and is divided into five subregions from telomere to centromere: extended class I, classical class I, classical class III, classical class II, and extended class II. In addition, the MHC contains two hypervariable regions, the RCCX module in class III (spanning complement C4) and the HLA-DRB genes in class II, that both exhibit copy number polymorphism. Examination of the sequence across the extended MHC reveals the presence of 421 genes, and over 252 (60%) are thought to be expressed [2]. Around 40% of genes expressed within the classical MHC encode proteins with putative immunomodulatory function [3]. The classical class I and class II loci encode the <span class="yellow">human</span> leucocyte antigen (HLA) proteins involved in antigen presentation to T cells, initiating the adaptive immune response. The class III region contains the greatest density of genes in the genome (58 expressed genes), which are often found in functionally related clusters [2].
A major obstacle in the identification of disease-specific causal variants within the MHC has been the strong linkage disequilibrium (LD) exhibited by certain alleles in this region, resulting in the existence of long-range, conserved, extended haplotypes [4], also known as ancestral haplotypes [5], sometimes spanning more than 2 Mb [6]. Thus, for many MHC-associated diseases, it has only been possible to delimit association signals to a particular extended haplotype or segment of one.
Systemic lupus erythematosus (SLE/lupus, [Online Mendelian Inheritance in <span class="yellow">Man</span> 152700, http://www.ncbi.nlm.nih.gov/sites/entrez?db=OMIM&TabCmd=Limits]) is a chronic, multi-system autoimmune disease affecting young <span class="yellow">women</span> ten times more commonly than <span class="yellow">men</span>. The worldwide prevalence of SLE is estimated at between 12 and 124 cases per 100,000 individuals [7]. SLE is characterized by the presence of pathogenic autoantibodies to nuclear and cell-surface antigens that show affinity maturation. The consequent immune complexes deposit in tissues, causing inflammation and damage. It is well established that there is a complex genetic component to lupus aetiology, with hormonal and environmental influences also contributing to disease susceptibility [8,9].
The MHC has been the most consistently confirmed genetic risk factor for SLE, and multiple different genes within the region have been significantly implicated with disease susceptibility. For example, hereditary and acquired deficiencies of the early classical complement component C4, located within the MHC class III locus, leads to a lupus-like syndrome. A role for another class III gene, tumour necrosis factor alpha (TNF), in SLE was suggested following the observation that the lupus-prone New Zealand F1 <span class="blue">mouse</span> hybrid exhibits constitutively low TNF expression [10]. Recently, the development of antinuclear antibodies in <span class="yellow">patients</span> treated with TNF antagonists has also stimulated interest in the possible role of TNF in SLE [11–13]. <span class="yellow">Murine</span> and <span class="yellow">human</span> candidate gene studies, together with genome-wide linkage screens, provide further support that multiple genetic loci, including the <span class="blue">mouse</span> MHC complex H2 and the <span class="yellow">human</span> MHC locus, contribute to disease susceptibility [14–17].
It should be noted that the <span class="yellow">human</span> MHC was first associated with SLE in 1971, when studies demonstrated that lupus probands were enriched for the class I alleles HL-A8 (now known as HLA-B8) and HLA-W15 (now known as HLA-B15) when compared with healthy controls [18,19]. Further case control association studies were small, performed in ethnically diverse populations, and tested only a small number of the classical HLA and complement C4 alleles. The most consistent findings reported to date are associations with the class II alleles HLA-DR2 (DRB1*1501) and HLA-DR3 (DRB1*0301) and their respective haplotypes in Caucasian populations. The complement C4A null allele (C4A*Q0) has shown inconsistent association with lupus in a number of studies—a situation that may reflect genetic heterogeneity in disease susceptibility [20–23]. In addition, a recent study has demonstrated that low C4A copy number is a risk factor for lupus in a European American cohort [24]. However, the C4A null allele lies on the lupus-associated DR3 “autoimmune” extended haplotype (AH8.1), which exhibits extremely strong LD [6]. It therefore remains to be definitively established whether this locus constitutes a distinct susceptibility allele to that of the class II association or is merely in LD with it. Similarly, certain TNF promoter polymorphisms, including the much-studied SNP TNF-308G/A, have shown association with SLE; but again, many of these variants are carried on the highly conserved 8.1 ancestral haplotype, thus restricting interpretation of these data.
In 2002, a family-based study employing microsatellites as surrogate markers for HLA-DRB1 haplotypes in Caucasian lupus families demonstrated association with DR3-, DR2-, and DR8 (DRB1*0801)-containing haplotypes. In that study, Graham and colleagues reported that, taking advantage of recombinant chromosomes, the disease risk region could be limited to a 1 Mb region encompassing classical class II and class III [25].
We have performed a medium resolution association mapping study of the MHC in lupus families, utilizing a combination of SNPs and four-digit typing at the HLA-DRB1 locus in order to anchor haplotypes. Sixty-eight SNPs were successfully genotyped across a 2.4 Mb region of the MHC, from the class I locus KIAA1949 to the class II gene HLA-DPB2, in 314 UK Caucasian SLE trios. We used these data to perform a family-based association study in an attempt to distinguish the relative effects of the class II and class III regions of the MHC in lupus susceptibility. In addition, we employed the long-range haplotype test to search for the presence of high-frequency, extended haplotypes indicative of recent positive selection [26]. We have also used family-based and case-control strategies to examine genotypic risk at HLA-DRB1 and rs419788.<br><br>Results
Association Testing of HLA-DRB1 and MHC Region SNPs
In order to define the causal variation within the MHC region, we typed 314 complete SLE trios for the HLA-DRB1 gene as well as for 86 SNPs across a 2.4 Mb region encompassing the HLA class I locus HLA-B to HLA-DPB2. High-quality genotype data was obtained for HLA-DRB1 and 68 MHC SNPs (see Table S1 for quality control data). Association testing of the HLA-DRB1 gene revealed a significant association with HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, T:U = 129:55) in our lupus cohort (Table 1). There was also a trend for under transmission of the HLA-DRB1*0701 allele (nominal p = 0.0013, T:U 42:77); however, this association was no longer significant after correction for multiple testing as determined by 10,000 permutations of the dataset (permuted p = 0.09). Furthermore, we did not find evidence of association with HLA-DRB1*1501 (nominal p = 1.0, T:U 70:70) or HLA-DRB1*0801 (nominal p = 1.0, T:U = 11:11) in our cohort (see Table S2 for complete HLA-DRB1 association data); alleles previously suggested by microsatellite typing of a US lupus cohort [25].
Association testing of the MHC region SNPs also identified significant evidence of association to SLE (Table 1 for associated markers and Table S3 for all MHC SNPs). The SNP with the most significant association, rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001) was of similar strength to that of the HLA-DRB1*0301 allele, with odds ratios (ORs) and 95% confidence intervals (CIs) of 2.0 (1.6–2.6) and 2.3 (1.7–3.2), respectively. This SNP is located within intron 6 of the class III gene, superkiller viralicidic activity 2-like (<span class="yellow">Saccharomyces cerevisiae</span>) (SKIV2L), and is located approximately 500 kb telomeric to the HLA-DRB1 gene. Of the other 12 SNPs that were significantly associated with SLE (nominal p = 4.0 × 10−4 to 2.5 × 10−7; permuted p = 0.03 to <0.0001), one was located in the class I region between HLA-B and MICA, seven were located in the class III region, and four were situated in the class II region (Table 1; Figure S1). Specifically, the seven associated class III SNPs were located in or close to the following genes: the TNF promoter, BAT3, SLC44A4, EHMT2, TNXB, GPSM3, and NOTCH4. One of the four class II associated SNPs was close to HLA-DRA, two were between HLA-DRB1 and HLA-DQA1 and one was in intron 1 of HLA-DQA1. The correlation between all 68 SNPs and HLA-DRB1 in our UK SLE cohort is illustrated in Figure 1. The markers showing significant association are highlighted.<br><br>Conditional Analyses Identify Two Independent Association Signals in the MHC
In order to establish whether the two most associated signals identified in this association-mapping experiment are likely to represent a single causal allele or independent risk factors, we first examined the association data conditioned upon the presence of the HLA-DRB1*0301 allele. We found that four of the 13 associated SNPs showed evidence of signals independent of HLA-DRB1*0301 in our dataset, the strongest of which was rs419788 (Table 1). We therefore conditioned the three remaining SNPs (rs2523589, rs1052486, and rs605203) on rs419788 to assess whether these signals are truly independent of each other or show association due to LD with rs419788. In addition, we included HLA-DRB1 in stepwise conditional regression analyses performed on the SNPs showing association independent of HLA-DRB1 (unpublished data). These analyses demonstrated that the observed association signals at rs2523589, rs1052486, and rs605203 were predominantly dependent upon the association at rs419788, and suggested that there are two major independent association signals in the MHC in UK SLE: HLA-DRB1 and rs419788. The independence of the association signals at HLA-DRB1 and rs419788 is further supported by the observation that there is only modest LD between these two (r2 = 0.24). There was no association with any other HLA-DRB1 allele and the four SNPs independent of HLA-DRB1*0301 (TRANSMIT, unpublished data).
The association of the tumour necrosis factor gene promoter SNP TNF-308G/A with SLE is lost after conditioning for HLA-DRB1*0301 in our cohort. If we perform the reverse analysis and condition HLA-DRB1*0301 on the presence of the TNF promoter SNP, we find that the association remains, confirming that our TNF association is secondary to that of HLA-DRB1*0301.<br><br>Genotypic Risk for Class II and Class III Association Signals
Having established independent association at the allelic level with HLA-DRB1*0301 and rs419788-T in our UK SLE cohort, we wanted to further determine the genotypic risk conferred by these variants and hence gain insight into their underlying mode of inheritance in lupus. We used case-control and family-based analyses to assess genotypic risk at HLA-DRB1, while the family-based test alone was used for rs419788. Common family-based tests of LD, such as those used in this study (Genehunter), measure transmission distortion based on allele counts rather than genotype counts; the former has been shown to be more powerful under additive models, while the latter has greater power under recessive or dominant genetic models [27]. The genotype-pedigree disequilibrium test (geno-PDT) determines LD between a locus genotype and disease by comparing genotypes that are transmitted from parent to proband with those that are not [27]. We used the geno-PDT to assess genotypic risk for our class II and class III association signals: HLA-DRB1 and the SNP rs419788. In the case-control analysis for HLA-DRB1, ORs with 95% CI were calculated and Fisher's exact test employed to assess statistically significant differences between HLA-DRB1 genotypes in lupus probands and healthy controls. For HLA-DRB1, the alleles were coded as follows: HLA-DRB1*0301, HLA-DRB1*1501, HLA-DRB1*X where X represents all HLA-DRB1 alleles other than HLA-DRB1*0301, and HLA-DRB1*1501. We included HLA-DRB1*1501 in the analysis, even though we find no allelic association in our cohort, because previous studies have shown a greater risk for lupus in individuals who are compound heterozygotes for HLA-DRB1*0301- and HLA-DRB1*1501-containing haplotypes [25,28]. Overall the results are consistent with a dominant effect from HLA-DRB1*0301 (Table 2) and a dose-dependent (additive) effect from rs419788-T (Table 3). Specifically, both case-control and geno-PDT demonstrate that there is no dose-dependent increase in disease risk for HLA-DRB1*0301. Rather, it appears that the presence of a single copy of HLA-DRB1*0301 alone is sufficient to increase susceptibility to disease. Moreover the 0301/X genotypes constitute the greatest risk in our cohort rather than the 0301/1501 heterozygotes. Genotypes containing HLA-DRB1*1501 in the absence of HLA-DRB1*0301 revealed no significant association in our cohort.
All three rs419788 genotypes demonstrated significant association in our lupus families (Table 3). The common CC genotype was significantly under transmitted, while the rare T allele displayed dose-dependent over transmission to lupus probands.<br><br>Characterization of HLA-DRB1*0301 Risk Haplotype
Next, we wanted to further delimit the MHC class II association signal that we have detected at HLA-DRB1. We used phased parental genotype data to compare the allelic composition of HLA-DRB1*0301-bearing haplotypes that were transmitted (T) to affected probands to those that were not transmitted (or untransmitted, UT) with the aim of identifying differences that could delineate the lupus susceptibility interval(s) arising from this haplotype (summarized in Figures 2A, 2B, and S2). We observed a striking difference between transmitted and untransmitted chromosomes within the class II region: nearly all transmitted HLA-DRB1*0301 haplotypes (99%) are identical across a 180 kb region defined by eight SNPs, whereas the corresponding region within untransmitted HLA-DRB1*0301 haplotypes exhibits significant recombination. These data strongly suggest the existence of a risk haplotype that, interestingly, contains only three expressed genes: HLA-DRB1, HLA-DQA1, and HLA-DQB1. Furthermore, we can confidently define the allelic composition of this risk haplotype, as these three genes are in strong LD and occur in one common haplotype in Caucasians: HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. Thus, we hypothesize that the specific combination of all three alleles is required to confer disease risk in lupus or that disease susceptibility lies with either HLA-DRB1*0301 or the HLA-DQ alleles. We do not have sufficient numbers of recombinant chromosomes in this risk region to further delimit this signal: 2/176 (1.1%) transmitted HLA-DRB1*0301 haplotypes are recombinant at HLA-DQA1-HLA-DQB1; 3/178 (1.7%) transmitted haplotypes identical across HLA-DQA1-HLA-DQB1 do not possess HLA-DRB1*0301.
The composite relative extended haplotype homozygosity (REHH) versus frequency plot for UK SLE; Utah residents with ancestry from northern and western Europe (CEPH); and Yoruba in Ibadan, Nigeria (Yoruba) populations is shown in Figure 3A. We can only comment on evidence for positive selection in CEPH individuals, as we have used this population alone to assess background variation on Chromosome 6. The SLE and Yoruba cohorts are shown for comparative purposes. We find no evidence of positive selection for HLA-DRB1*0301 in the CEPH population. However, this allele is enriched in our lupus cohort (21% of parental chromosomes) and displays greater extended homozygosity when compared with HLA-DRB1*0301-bearing haplotypes in CEPH and Yoruba. Hence, the HLA-DRB1*0301 allele in lupus is observed as an outlier on the plot when compared to background variation in CEPH. These data support our previous observations (outlined above) of the highly conserved nature of HLA-DRB1*0301 haplotypes in lupus. In addition, the haplotype bifurcation plots centered on HLA-DRB1*0301 for UK SLE, CEPH, and Yoruba populations in Figure 3B illustrate preservation of the common HLA-DRB1*0301 haplotype in CEPH and UK SLE, while that seen in the Yoruba is significantly different. The class II regions of all three populations are essentially identical across our chosen SNPs; the main differences lie in class III. The difference in African populations in the class III region is one possible explanation for the lack of evidence for an association between HLA-DRB1*0301 and SLE in African or African American populations. However, HLA-DRB1*0301 has a lower frequency (∼7%–10%) in African populations compared with Europeans (∼13%), and the number of HLA association studies conducted in African populations is very limited.<br><br>Characterization of Class III Region Risk Haplotype
Our data reveal a second independent signal at the MHC in SLE arising from the T allele of SNP rs419788 in intron 6 of the class III gene, SKIV2L. Further evidence supporting the independence of the rs419788-T and HLA-DRB1*0301 alleles is provided by the moderate LD between these two variants (r2 = 0.24) coupled with our data demonstrating that only 47% of rs419788-T allele-bearing haplotypes contain HLA-DRB1*0301.
The structure and composition of T and UT haplotypes anchored at rs419788-T were essentially identical (Figures 2C, 2D, and S2), and hence not informative in delimiting our class III signal. Therefore, we examined the LD structure around our associated class III SNP to better define our disease risk interval. In our lupus dataset the rs419788-T allele resides on three of seven haplotypes present within a large block of six SNPs exhibiting strong LD. This haplotype block encompasses roughly 270 kb containing class III genes from SLC44A4 to AGER, including the RCCX module. Next, we analyzed the haplotype block structure of this region in CEPH families using SNP data dumped from the International HapMap Project (http://www.hapmap.org/). The greater density of SNP typing available in the HapMap CEPH population compared to our current UK SLE map allowed us to potentially refine our signal by exploring correlations between our associated SNP and those surrounding it. Analysis of these data (Figure 4) suggests the presence of short-range LD around our associated variant, rs419788, in CEPH families, encompassing approximately 40 kb of the genome which includes the five genes: complement factor B (CFB), RD RNA binding protein (RDBP), SKIV2L, dom-3 homolg Z (<span class="yellow">C. elegans</span>) (DOM3Z), and serine/threonine kinase 19 (STK19), and does not include the complement C4 locus. Furthermore, assessment of marker association in our lupus dataset demonstrates that after conditioning for HLA-DRB1*0301, the only markers that retain association signals are telomeric of SKIV2L, suggesting that complement C4, which is centromeric to this gene, may not be responsible for our independent class III signal.<br><br>Subphenotype Analysis
In order to gain further insight into disease pathogenesis, we examined common lupus subphenotypes. Such subsets are more homogeneous than lupus per se and thus maybe enriched for specific predisposing variants. In addition, one might expect a close association between MHC class II alleles and autoantibody subsets in lupus if these are indeed causal variants, given their role in antigen presentation and subsequent humoral immunity. We therefore tested our two main MHC association signals, HLA-DRB1*0301 and rs419788, for association with renal disease and autoantibody subsets in our lupus cohort.
We found that HLA-DRB1*0301 was associated with the presence of anti-Ro and anti-La antibodies in our UK SLE cohort, with the latter showing the greatest evidence of association (anti-La nominal p < 0.001 compared with anti-Ro nominal p < 0.025). We found no association of HLA-DRB1*0301 with renal disease or any other autoantibody subsets in our dataset (see Table S4 for detailed results).
Genotypes of the SNP rs419788 were not associated with any of the tested lupus subphenotypes after controlling for the effect of HLA-DRB1*0301 (unpublished data).<br><br>
Discussion
We present the first family-based SNP association study of the MHC in SLE. We have genotyped 69 markers (HLA-DRB1 and 68 SNPs) across 2.4 Mb of the MHC, encompassing class III and class II, in a cohort of 314 UK Caucasian SLE trios. Transmission disequilibrium testing of these data has shown predominant association with the alleles HLA-DRB1*0301 and rs419788-T, together with 12 other MHC SNPs. Moreover, using conditional analyses, we have shown that the two primary signals of association at the MHC are independent of each other. Specifically, one signal arises from HLA-DRB1*0301 in class II and the other from the T allele of SNP rs419788 in the class III gene SKIV2L.
Examination of bifurcation plots for T and UT HLA-DRB1*0301-containing haplotypes has enabled delineation of our class II association signal to a 180 kb region encompassing HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. These data substantially refine that previously published by Graham et al. in 2002 [25], where the lupus susceptibility interval within HLA-DRB1*0301-containing haplotypes could only be delimited to a 1 Mb region encompassing class II and class III. The precise causal variant(s) within this region remains to be determined, as the three implicated alleles exhibit strong LD with few recombination events separating them (two out of 176 transmitted HLA-DRB1*0301 chromosomes in our dataset). However, all three allelic variants represent attractive functional candidates in lupus susceptibility for their role in antigen presentation and stimulation of the adaptive immune response.
Our association of HLA-DRB1*0301 with lupus concurs with published data in Caucasian cohorts and is well established [16]. While our lack of association with HLA-DRB1*1501 and HLA-DRB1*0801 is consistent with previous data from the UK [29], Spain [30], the Netherlands [31], Sweden [32], Mexico [33], and the US [34], it conflicts with that of other US groups [25,35]. Interestingly, we demonstrate a trend, though not statistically significant, for undertransmission of HLA-DRB1*0701—a result also observed in prior UK and Canadian lupus studies [29,36]. Moreover, a negative association of HLA-DRB1*0701 has been reported in other autoimmune diseases including Graves disease [37,38], type 1 diabetes [39], and rheumatoid arthritis [40].
It appears that the conflicting results between UK SLE and previous US (Minnesota [MN]) [25] SLE data stem from differences in HLA-DRB1 allele frequency in the probands of each cohort. The reason for this is unclear. A comparison between UK and MN SLE cohorts (Table 4) reveals that UK SLE cases are enriched for HLA-DRB1*0301 but not HLA-DRB1*0801 or HLA-DRB1*1501 when compared to a UK control population. In contrast, MN SLE cases are enriched for HLA-DRB1*0301-DQB1*0201, DRB1*0801-DQB1*0402, and DRB1*1501-DQB1*0602 inferred haplotypes when compared to MN controls [25]. There is no statistically significant difference in the aforementioned HLA class II alleles/haplotypes between UK and MN control populations that could account for the disparity seen in the respective lupus cohorts. Differences in disease severity and subphenotype frequency between the two populations could account for the observed discrepancy. From the limited data available we found that the presence of renal disease appears to be similar in both cohorts (UK SLE 36% compared with MN SLE 40%), while the gender ratios are significantly different (female: male UK SLE 11:1 compared with MN SLE 57:1, Chi square p value < 0.001). We were unable to compare other lupus subphenotypes. Furthermore, closer inspection of these data reveals that microsatellite-inference of HLA-DRB1 alleles in the MN SLE dataset may underestimate the frequency of HLA-DRB1*0301 and overestimate that of HLA-DRB1*1501, thus diminishing the effect of the former and enhancing that of the latter. It is also possible that the MN SLE cohort shows greater racial heterogeneity in comparison to our UK SLE cohort, despite both being characterized as Caucasian.
Previous studies have demonstrated increased risk for lupus in individuals carrying particular combinations of microsatellite-inferred HLA-DRB1-HLA-DQB1 haplotypes [25,28]. The highest risk genotype was found to be the compound heterozygote HLA-DRB1*0301-DQB1*0201/HLA-DRB1*1501-DQB1*0602, while HLA-DRB1*0301-DQB1*0201-containing genotypes demonstrated a dose-dependent effect in increasing lupus susceptibility [25,28]. In the present study, we have examined genotypic risk at the classically typed HLA-DRB1 locus and in contrast to the aforementioned data of Graham et al. [25,28] we have shown a likely dominant effect of the associated allele, HLA-DRB1*0301. The case-control and family-based analyses for HLA-DRB1 also show the greater power of the former to detect significant association (Table 2). Specifically, all genotypes containing HLA-DRB1*0301 show increased transmission to lupus probands; however, homozygotes show no greater risk compared with heterozygotes, as would be expected under additive or multiplicative models. Thus, a dominant model of inheritance, requiring the presence of a single copy of the disease-predisposing variant alone, likely underlies the susceptibility conferred by HLA-DRB1*0301 in UK SLE. Such a model would fit an antigen presentation hypothesis where susceptible individuals carrying an HLA-DRB1*0301 allele are able to present auto-antigens to CD4+ lymphocytes, thus stimulating an autoimmune response. The differences between our UK SLE and the previously published US SLE data may reflect disease, ethnic, and haplotypic heterogeneity.
Interestingly, analysis of genotypic risk at the associated class III marker, rs419788, suggests an additive (dose-dependent) pattern of inheritance for the rare T allele, where one copy confers a low risk of disease and two copies results in greater susceptibility. The different inheritance patterns for our class II and class III association signals provide further evidence for their independence.
A variety of HLA-DR and HLA-DQ alleles have been associated with autoantibody subsets in ethnically diverse populations of lupus. The strongest associations have been demonstrated between anti-Ro and anti-La antibodies and HLA-DR3 and HLA-DQ2 (HLA-DQB1*0201), which are in strong LD [41–45] in case-control studies. Here, we confirm the association of HLA-DRB1*0301 with anti-Ro and anti-La antibody production in our family-based cohort.
Examination of LD structure around our second independent association, rs419788-T in class III, coupled with the results of our conditional analysis, suggests that this signal could also be delimited to a relatively narrow genomic interval of about 40 kb given further SNP mapping in our cohort. This region includes the genes CFB, RDBP, SKIV2L, DOM3Z, and STK19, but does not include complement C4. Thus, complement C4 null alleles, which have been implicated in lupus pathogenesis, may not be responsible for our class III signal. We conclude, therefore, that our family-based mapping study has potentially revealed a hitherto unknown lupus susceptibility interval in the class III region of the MHC. However, we cannot conclusively exclude association at complement C4/RCCX without direct determination of C4 polymorphism/copy number in our cohort.
With respect to the genes implicated in our study, CFB is a vital component of the alternate complement pathway and disregulation may clearly affect the inflammatory response [46]. RD and Skiv2l are proteins potentially involved in RNA processing. The RD protein forms part of a negative elongation factor (NELF) complex that represses RNA polymerase II transcript elongation, while Skiv2l is a DEAD box protein with possible function as an RNA helicase. The function of Dom3z is currently unknown, although the homologous <span class="yellow">yeast</span> protein binds nuclear exoribonuclease. Moreover, its ubiquitous expression suggests a housekeeping role. STK19 is a protein kinase of unknown function with primary nuclear localization [47]. Interestingly, RDBP and SKIV2L are found to be highly expressed in T lymphocytes, B lymphocytes, and dendritic cells (SymAtlas, http://symatlas.gnf.org/SymAtlas/).
A number of studies have demonstrated conflicting evidence for and against association with various TNF locus polymorphisms in SLE [48]. A recent meta-analysis of the TNF-308G/A promoter polymorphism in SLE [48] revealed evidence of association for the minor allele (A) in European populations; however, this study did not account for LD with class II alleles. On conditioning our dataset for HLA-DRB1*0301, we find that the TNF promoter signal is lost, suggesting that this association is not independent and is due to LD with HLA-DRB1*0301 (or another variant in LD with HLA-DRB1*0301).
In summary, we have found association with two distinct and independent variants within the class II (HLA-DRB1*0301) and class III (SKIV2L) regions of the MHC in UK SLE trios. We can delimit our class II signal in lupus to three genetic variants (HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201) that may confer disease risk in combination or as separate signals. Our class III signal importantly excludes independent association at the TNF promoter polymorphism TNF-308G/A and potentially provides a novel locus for further study.<br><br>Materials and Methods
Study cohorts.
SLE families. The cohort comprises 314 complete SLE trios (that is, mother, father, and affected lupus proband) collected as previously described [49]. All study <span class="yellow">participants</span> are European Caucasian on the basis of grandparental origin. All 314 lupus probands (288 female, 26 male) fulfill the revised American College of Rheumatology (ACR) criteria for SLE [50], 36% of whom have a diagnosis of lupus nephritis. Written consent was obtained from all study <span class="yellow">participants</span> and ethical approval for this study was obtained from the Multi-Centre Research Ethics Committee (MREC 2 June 1998).<br><br>Healthy controls.
The control population for the HLA-DRB1 genotypic risk case-control analysis constitutes 1,667 healthy males of Northern European origin. The individuals are potential hematopoietic stem cell donors and were typed to four digits for HLA-DRB1 at the Anthony Nolan Trust, UK for this purpose. The level of resolution used for the typing of HLA-DRB1*15 alleles in these healthy controls resulted in the ambiguous allele string HLA-DRB1*1501/1502/1504/1506. However, it is likely that the great majority are HLA-DRB1*1501. There is no gender bias in HLA-DRB1 allele frequencies, so although we have used a male control cohort, the frequencies would be expected to be the same in a similar female cohort (Steven Marsh, personal communication).<br><br>SNP genotyping.
Eighty-six SNPs were chosen for genotyping in our mapping study. Specifically, we selected 40 MHC class II and class III haplotype tagging SNPs from a preliminary MHC SNP map [51] that had previously shown robust genotyping efficacy. In addition, we typed 36 MHC class II tag SNPs from a subsequent high-resolution MHC study [52]. We also included the TNF-308G/A promoter SNP, together with nine further SNPs in the region of HLA-B and MICA obtained from the database, dbSNP (http://www.ncbi.nlm.nih.gov/projects/SNP/). The latter SNPs had not been well characterized. All variants were typed in the entire cohort (n = 942). The SNPs span approximately 2.4 Mb of the MHC from the class I gene, KIAA1949 to the class II pseudogene, HLA-DPB2 and thus encompass MHC class III and class II. SNP genotyping was performed at the Broad Institute of MIT and Harvard and at Imperial College London by matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry using the Sequenom MassARRAY platform as previously described [53]. SNPs that failed Sequenom typing were retyped by KBiosciences (http://www.kbioscience.co.uk/) using their in-house SNP genotyping methodology, KASPar (http://www.kbioscience.co.uk/genotyping/index.htm), a competitive allele-specific PCR technique.<br><br>HLA-DRB1 genotyping.
HLA-DRB1 typing was performed at the Anthony Nolan Trust, UK. All samples (n = 942 UK SLE trios and n = 1,667 controls) were genotyped using LABType SSO (sequence-specific oligonucleotide) typing technology according to the manufacturer's written recommendations (http://www.onelambda.com). Briefly, a locus-specific biotinylated PCR amplicon is produced, denatured, and rehybridized to complementary oligonucleotide probes conjugated to fluorescently coded beads. The bound biotinylated PCR product can be detected using R-phycoerythrin-conjugated streptavidin. A flow analyzer identifies the fluorescent intensity of phycoerythrin on each bead. The assignment of HLA type is based on the reaction pattern compared to patterns associated with known sequences.
High resolution testing was performed using the Dynal AllSet+ SSP (sequence-specific primers) DRB1 assay according to the manufacturer's protocol (Invitrogen) for SLE families only. The presence or absence of PCR amplification was detected in a gel electrophoresis step using visualization by ethidium bromide incorporation. Genotypes were determined using SSPTool software.
Samples that could not be resolved to four digits using SSO and PCR-SSP were analyzed by DNA sequencing of exon 2 of HLA-DRB1. Primers, reagents, and protocols were provided by The Anthony Nolan Trust, UK (primer sequences are available on request). Specific HLA-DRB1 alleles were assigned by comparing the resultant sequence with reference sequence from the IMGT/HLA Database [54].<br><br>Data analysis.
Mendelian inconsistencies were removed using PedCheck [55]. Families in which more than eight markers demonstrated Mendel errors were removed from further analysis. Markers with less than 80% genotyping efficiency and markers where more than eight families showed Mendel errors were also excluded from analysis. Two markers located within the SLE associated class II region (rs2239802 in intron 4 of HLA-DRA and rs6457594 in the region between HLA-DRB9 and HLA-DRB5) show deviation from Hardy–Weinberg equilibrium (HWE), which may reflect an undetected SLE association or systematic genotyping error. HWE was assessed in parental samples in our cohort. There is currently no uniform opinion in the community regarding the inclusion or exclusion of SNPs that show deviation from HWE, hence we elected to include these markers in the final analysis.
Sixty-eight out of the total 86 SNPs passed our quality-control measures (see Table S1 for details). In summary, one SNP was monomorphic in our dataset, four SNPs yielded low genotyping efficiency, and 13 SNPs were excluded for unacceptable Mendel error rate. The mean call rate for all markers post-quality control was 94% (range 83% to 99%).
Family-based association testing was performed using Genehunter TDT (version 2.1) [56] and TRANSMIT (version 2.5) [57]. Haplotypes were constructed and permutation testing performed using Haploview (version 3.32) [58]. Significance of association signals in all analyses was based on permutation testing (10,000 permutations). The data are represented both as nominal and permuted p values. ORs with 95% CI for family-based analyses were calculated in PLINK (http://pngu.mgh.harvard.edu/purcell/plink/) [59]. Conditional regression analyses were undertaken using WHAP [60].
The geno-PDT was performed using PDT version 5.1 with default settings [27]. The HLA-DRB1 alleles were coded into three groups for the geno-PDT and the case-control analysis: HLA-DRB1*0301, HLA-DRB1*1501 and HLA-DRX where HLA-DRX includes all HLA-DRB1 alleles other than HLA-DRB1*0301 or HLA-DRB1*1501. The HLA-DRB1*1501 code in the healthy controls represents the allele string HLA-DRB1*1501/1502/1504/1506, as described previously. The HLA-DRB1*1501 code in the lupus probands represents the alleles HLA-DRB1*1501 (84 out of 91 *1501 and *1502 alleles) and HLA-DRB1*1502 (7/91 *1501 and *1502 alleles), as *1504 and *1506 were not present in this population. Fisher's exact test was used to assess significance of association in the case-control analysis.<br><br>Subphenotype analysis.
We looked for association of the HLA-DRB1*0301 allele with autoantibody subsets and renal disease in our cohort using the Chi-square test. We compared cases with and without the subphenotype of interest with DRB1*0301 homozygosity, heterozygosity, combined homozygosity and heterozygosity, and non-DRB1*0301 status. We performed the same analyses for homozygous and heterozygous genotypes of the associated SNP rs419788. The autoantibody subsets compared were anti-C1q, IgG, and IgM anti-cardiolipin antibodies (ACLG and ACLM), anti-Ro, anti-La, anti-RNP, anti-Sm, and anti-dsDNA.<br><br>Delineation of associated MHC haplotypes and evidence for positive selection.
We looked for positively selected alleles in our dataset using the long-range haplotype test as measured by extended haplotype homozygosity (EHH), previously described by Sabeti et al. [26]. Essentially, such an analysis allows assessment of positive selection by mining datasets for high frequency extended haplotypes in comparison to the other core haplotypes at a locus.
EHH is defined as the probability that two randomly chosen chromosomes carrying the core haplotype of interest will be identical by descent (homozygosity at all SNPs) for the entire interval from the core to a distance x. The REHH is the ratio of the EHH on the tested core haplotype compared with the combined EHH of all the other core haplotypes at the region excluding the tested core; as such, REHH accounts for local variation in recombination rate while EHH does not [26].
The program emphase was employed to assign the phase of parental genotype data and reconstruct missing information. Emphase is a simple phaser similar to the phaser of Excoffier and Slatkin [61]. It is very fast, especially on large datasets, and sufficiently accurate for most genetic applications. EHH analysis was performed on the phased parental data using the software program SWEEP (http://www.broad.mit.edu/mpg/sweep/index.html).<br><br>Haplotype bifurcation plots.
We represent the breakdown of LD on core haplotypes using haplotype bifurcation diagrams generated in the program TREE [62] (also explained in [52]).<br><br>REHH versus frequency plots.
Fifty-three SNPs (identified in Figure 1) are common to our dataset and the CEPH and Yoruba HapMap [63] populations. These three datasets, together with CEPH SNP data for Chromosome 6 in its entirety, were used to generate separate REHH versus frequency plots in SWEEP. The plots from the four cohorts were combined for visual, not statistical, comparison. Evidence for positive selection was quantitatively assessed in CEPH individuals, as this population alone was used to assess background variation on Chromosome 6. The UK SLE and Yoruba cohort data are shown for comparison. The 95th percentile based on total CEPH Chromosome 6 SNP data is shown.<br><br>
Supporting Information
Accession Numbers
The Online Mendelian Inheritance in <span class="yellow">Man</span> (OMIM, http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim) accession numbers for the genes described in this study are as follows: AGER, 600214; BAT3, 142590; C4A, 120810; C4B, 120820; CFB, 138470; DOM3Z, 605996; DOM3Z, 605996; EHMT2, 604599; HLA-B, 142830; HLA-DPB2, 120290; HLA-DQA1, 146880; HLA-DQB1, 604305; HLA-DRA, 142860; HLA-DRB1, 142857; KIAA1949, 610990; MICA, 600169; NOTCH4, 164951; RDBP, 154040; RDBP, 154040; SKIV2L, 600478; SLC44A4, 606107; STK19, 604977; STK19, 604977; TNF, 191160; and TNXB, 600985.<br><br><br><br><h3>pmcA2327290</h3><span class="yellow">Borrelia burgdorferi</span> membranes are the primary targets of reactive oxygen species
Abstract
Spirochetes living in an oxygen-rich environment or when challenged by host immune cells are exposed to reactive oxygen species (ROS). These species can harm/destroy cysteinyl residues, iron-sulphur clusters, DNA and polyunsaturated lipids, leading to inhibition of growth or cell death. Because <span class="yellow">Borrelia burgdorferi</span> contains no intracellular iron, DNA is most likely not a major target for ROS via Fenton reaction. In support of this, growth of <span class="yellow">B. burgdorferi</span> in the presence of 5 mM H2O2 had no effect on the DNA mutation rate (spontaneous coumermycin A1 resistance), and cells treated with 10 mM t-butyl hydroperoxide or 10 mM H2O2 show no increase in DNA damage. Unlike most bacteria, <span class="yellow">B. burgdorferi</span> incorporates ROS-susceptible polyunsaturated fatty acids from the environment into their membranes. Analysis of lipoxidase-treated <span class="yellow">B. burgdorferi</span> cells by Electron Microscopy showed significant irregularities indicative of membrane damage. Fatty acid analysis of cells treated with lipoxidase indicated that host-derived linoleic acid had been dramatically reduced (50-fold) in these cells, with a corresponding increase in the levels of malondialdehyde by-product (fourfold). These data suggest that <span class="yellow">B. burgdorferi</span> membrane lipids are targets for attack by ROS encountered in the various stages of the infective cycle.<br><br>Introduction
<span class="yellow">Borrelia burgdorferi</span>, the causative agent of Lyme disease, survives and proliferates in distinctly different niches, including its arthropod vector and various mammalian hosts. These ‘micro’ environments provide their own distinct sets of advantages and challenges to <span class="yellow">B. burgdorferi</span>. For example, during the initial stages of infection of the mammalian host, immune cells attempt to prevent <span class="yellow">B. burgdorferi</span> from establishing an infection using several systems including those generating bacteriocidal reactive oxygen species (ROS) [e.g. superoxide radicals (O2-), hydrogen peroxide (H2O2) and hydroxyl radicals (OH)] and reactive nitrogen species (RNS) [e.g. nitric oxide (NO) and peroxynitrite] (Storz and Imlay, 1999). In order for <span class="yellow">B. burgdorferi</span> to successfully colonize a new host and cause disease, they must overcome the challenges posed by the innate immune system including the deleterious effects of ROS/RNS compounds.
The effects of ROS/RNS on cells have been extensively investigated. These highly reactive compounds have been shown to damage cellular macromolecules including DNA, proteins and cellular membranes. The damage to membranes can arise through either lipid or membrane protein damage. In eukaryotes, membrane lipids are a major target of ROS. Free radicals attack polyunsaturated fatty acids in membranes and initiate lipid peroxidation. A primary effect of this is a decrease in membrane fluidity which affects the physical properties of the membrane altering the function of membrane-associated proteins. Once lipid peroxides form, they react with adjacent polyunsaturated lipids causing an amplification of the damage. Lipid peroxides undergo further oxidation to a variety of products, including aldehydes, which subsequently react with and damage membrane proteins. However, in bacteria, it is assumed that lipids are not subject to the oxidative damage observed in eukaryotic cells. Only certain polyunsaturated lipids, such as linoleic acid and linolenic acid, are susceptible to oxidation (Gutteridge and Halliwell, 1990), and it is clear that most bacteria do not synthesize or incorporate these types of lipids in their cell membrane. Two notable exceptions are the photosynthetic bacteria which synthesize and incorporate significant levels of linoleic acid in their membrane (Tasaka et al., 1996) and <span class="yellow">Helicobacter pylori</span> membranes which contain between 0.5% and 3% linoleic acid (Ursini and Bindoli, 1987).
Instead, it has been shown that the most damaging effects of ROS in bacteria result from the interactions of H2O2 with ‘free’ Fe2+ (Imlay, 2003), generating very reactive OH (Fenton reaction). Because of this reactivity, the effect on any given biomolecule will depend largely upon proximity to the target. Because Fe2+ localizes along the phosphodiester backbone of nucleic acid, DNA is a major target of OH. This reactive species can pull electrons from either the base or sugar moieties, producing a variety of lesions including single- and double-stranded breaks in the backbone and chemical cross-links to other molecules. These strand breaks and other lesions block DNA replication and contribute to OH toxicity and cell death. Other base damage, which does not hinder replication, may result in a significant increase in mutation rates.
The intracellular biochemistry of <span class="yellow">B. burgdorferi</span> suggests that the primary cellular target of ROS may be distinct from that described in other bacteria such as <span class="yellow">Escherichia coli</span>. In <span class="yellow">E. coli</span>, the extent of DNA damage due to H2O2 and Fenton chemistry is directly proportional to Fe metabolism and the free Fe concentration within the cell (10 µM) (Keyer and Imlay, 1996). As the intracellular Fe concentrations of <span class="yellow">B. burgdorferi</span> are estimated to be < 10 atoms per cell (Posey and Gherardini, 2000), it seems unlikely that DNA is a primary target for ROS. Therefore, the purpose of this study is to determine the biochemical effects of ROS on <span class="yellow">B. burgdorferi</span>, including growth effects and biological/physical damage.<br><br>Results
Effect of ROS on <span class="yellow">B. burgdorferi</span> cells
In order to determine what the cellular targets of ROS in <span class="yellow">B. burgdorferi</span> are, we first needed to determine the sensitivity of Borrelia cells to various oxidants. Microaerobic cultures of <span class="yellow">B. burgdorferi</span> strain B31A3 were grown to a cell density of 5 × 107 cells ml−1, treated with varying concentrations of H2O2 (0–50 mM) or t-butyl hydroperoxide (0–50 mM) and the number of surviving cells determined by plating. The results are shown in Fig. 1. When cells were exposed to 10 mM t-butyl hydroperoxide, approximately 50% of the cells survive (Fig. 1A). In contrast, when <span class="yellow">E. coli</span> cells were exposed to 1 mM t-butyl hydroperoxide, approximately 1% of the cells survive. <span class="yellow">B. burgdorferi</span> cells were much more resistant to t-butyl hydroperoxide than <span class="yellow">E. coli</span> cells, with a survival rate of approximately 100% at 1 mM t-butyl hydroperoxide. This trend was observed when cells were exposed to H2O2. <span class="yellow">E. coli</span> cells exposed to 1 mM H2O2 have approximately 10% survivability, while 100% of the <span class="yellow">B. burgdorferi</span> cells survive at this concentration, and approximately 80% survive when exposed to 10 mM H2O2 (Fig. 1B). Similar results were obtained when cells were exposed in HN (Hepes-NaCl) buffer, suggesting that the high-level resistance to ROS was not due to the interaction of ROS with components of Barbour-Stoenner-Kelly (BSK-II) growth medium. Taken together, these data indicated that <span class="yellow">B. burgdorferi</span> strain B31A3 was highly resistant to exposure to both t-butyl peroxide and H2O2.<br><br><span class="yellow">Borrelia burgdorferi</span> DNA is not the primary target of ROS
For most bacteria, DNA is the major target of ROS causing a wide variety of DNA lesions. This is in part due to the localization of ‘free’ Fe2+ along the phosphodiester backbone of nucleic acids, putting the DNA in close proximity to the active species formed via the Fenton reaction. However, <span class="yellow">B. burgdorferi</span> has been shown to harbour few genes encoding orthologues of known iron-containing proteins, does not require Fe for growth and has intracellular Fe concentrations estimated to be < 10 atoms per cell (Posey and Gherardini, 2000). Taken together, these observations strongly suggest that <span class="yellow">B. burgdorferi</span> DNA is not a major target for ROS. To test this hypothesis, different techniques were used to measure DNA damage in <span class="yellow">B. burgdorferi</span> cells after exposure to ROS.
One reliable indicator of DNA damage by ROS in a cell is an increase in the spontaneous mutation rate. In <span class="yellow">B. burgdorferi</span>, mutations that confer resistance to coumermycin A1, which targets the β subunit of DNA gyrase, have been mapped to gyrB, the gene encoding DNA gyrase B (Samuels et al., 1994). In each case, a single point mutation correlated with this drug resistance. To determine if exposure to oxidants increases DNA damage by increasing point mutations, <span class="yellow">B. burgdorferi</span> B31A3 cells grown under microaerobic conditions were treated with 5 mM H2O2 and plated in the presence and absence of 250 ng ml−1 coumermycin A1. The mutation frequency was calculated as the number of colonies that are CouR per total number of cells plated. The spontaneous resistance frequency of treated cells was approximately equivalent to that of untreated cells, 8.8 × 10−8 and 1.33 × 10−7, respectively, indicating no increase in the number of point mutations (Fig. 2A). Also, no increase in point mutations was observed when cells were treated with higher concentrations of H2O2 (10 mM) or when treated with t-butyl hydroperoxide (5 and 10 mM) (data not shown).
Another effective way to determine DNA damage is by measuring the number of apurinic/apyrimidinic sites (AP). AP sites in DNA, where the DNA base is lost, can be generated spontaneously under physiological conditions by hydrolysis of the N-glycosylic bond, or can also be formed by DNA-damaging agents (Lindahl and Nyberg, 1972), such as UV, alkylating agents or OH. They are also intermediates in the base excision repair pathway (BER) (Weiss and Grossman, 1987; Friedberg and Hanawalt, 1988; Wallace, 1988). Thus, the cellular steady state level of AP sites would increase as a consequence of base modifications and their subsequent repair. As AP sites are bypassed inefficiently by DNA polymerase in bacterial cells, DNA lesions can result in a significant block in DNA replication. Therefore, the number of AP sites can serve as a sensitive indicator of DNA damage resulting from oxidative stress (Kubo et al., 1992). To determine whether ROS can damage <span class="yellow">B. burgdorferi</span> DNA, strain B31A3 cells grown under microaerobic conditions were treated with H2O2 (1 or 10 mM), t-butyl hydroperoxide (1 or 10 mM) or lipoxidase (an enzyme which specifically catalyses the hydroperoxidation of lipids containing a cis, cis-pentadiene structure, such as linoleic acid). The DNA was isolated and assayed for AP sites. The results are shown in Fig. 2B. In all cases, the numbers of AP sites per 105 bp DNA were equivalent, indicating that the addition of oxidants did not increase the number of DNA base lesions. In contrast, when <span class="yellow">E. coli</span> strain TA4315 cells (ahpCF) (Storz et al., 1989) were treated with 100 µM H2O2, the number of AP sites increased ∼10-fold over untreated cells. These data suggest that <span class="yellow">B. burgdorferi</span> DNA was not a major target for oxidative damage under these conditions.
It is of note to mention that the total number of AP sites per 105 bp DNA is approximately 10-fold higher in untreated Borrelia B31A3 DNA than in untreated <span class="yellow">E. coli</span> DNA. The Borrelia genome consists of a linear chromosome and multiple linear and circular plasmids. Numerous observations indicate that functional telomeres require interaction with DNA damage repair proteins, suggesting that the DNA damage repair machinery, including the BER, is involved in replication of telomeres and protection of functional chromosome ends (Verdun and Karlseder, 2007). An important intermediate in BER is the apurinic or abasic site. Therefore, it was possible that the higher numbers of AP sites in untreated Borrelia DNA was due to the number of telomeres present in the genome. The <span class="yellow">B. burgdorferi</span> strain B31A3 harbours 11 linear plasmids and a linear chromosome with ∼100 unpaired bases in the telomere loops per genome or 10 bases per 105 bp DNA (Hinnebusch and Barbour, 1991; Chaconas, 2005). Untreated Borrelia B31A3 contains 11 ± 3.8 AP sites per 105 bp DNA, suggesting that the elevated number of AP sites in Borrelia was due to the number of telomeres. To test this hypothesis, the AP sites were measured in DNA isolated from untreated <span class="yellow">B. burgdorferi</span> strain B314 which harbours no linear plasmids (Sadziene et al., 1993). The number of unpaired bases in the telomere loops per genome was estimated to be 10, which corresponds to 1 base per 105 bp DNA, and only 3.6 ± 0.2 AP sites per 105 bp DNA were detected, supporting the hypothesis that the high number of AP sites in untreated Borrelia DNA was due to the number of unpaired bases in the telomeres. The analyses of the AP sites strongly suggested that <span class="yellow">B. burgdorferi</span> DNA was not the major target for oxidative damage.
In addition to the generation of abasic sites, oxygen radicals often damage DNA through the formation of 8-oxoguanine lesions (Nakamura et al., 2000). 8-Oxoguanine, through its ability to mispair with bases other than cytosine, likely plays a role in DNA mutagenesis. Consequently, 8-oxoguanine is often used as a marker of oxidized DNA damage. To determine whether ROS can damage <span class="yellow">B. burgdorferi</span> DNA and cause 8-oxoguanine lesions, strain B31A3 cells grown under microaerobic conditions were treated with H2O2 (5 or 10 mM), t-butyl hydroperoxide (5 or 10 mM) or lipoxidase. The DNA was isolated and assayed for 8-oxoguanine using an Enzyme-Linked ImmunoSorbent Assay. The results are shown in Fig. 2C. In all cases, the amount of 8-oxoguanine in Borrelia cells was below the detection limit of the assay. In contrast, when a MutM (the specific glycosylase that removes the 8-oxoguanine)-deficient <span class="yellow">E. coli</span> strain was treated with 100 µM H2O2, the amount of 8-oxoguanine sites increased ∼fivefold over untreated cells. It is important to point out that no MutM homologue has been identified in the genome of <span class="yellow">B. burgdorferi</span>. Taken together, these data suggested that <span class="yellow">B. burgdorferi</span> DNA was not a major target for oxidative damage under these conditions.<br><br>The membranes of <span class="yellow">B. burgdorferi</span> are targeted during oxidative stress
In eukaryotes, membrane lipids are a major target of ROS. Free radicals can attack polyunsaturated fatty acids, such as linoleic acid and linolenic acid, in membranes and initiate lipid peroxidation. This reaction can cascade throughout the membrane to adjacent polyunsaturated fatty acids, decreasing membrane fluidity and generating more toxic products such as aldehydes (Imlay, 2003). Because most bacterial membranes contain saturated and monounsaturated fatty acids rather than ‘reactive’ polyunsaturated lipids, peroxidation of lipids in bacterial membranes is not considered a problem. As <span class="yellow">B. burgdorferi</span> cannot synthesize their own lipids, they must instead scavenge them. Therefore, it seems likely that their membrane composition would reflect the host's lipid profile or that of their growth medium (Barbour, 1984; Fraser et al., 1997) and would contain some polyunsaturated fatty acids. To determine if <span class="yellow">B. burgdorferi</span> contains polyunsaturated lipids, <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under anaerobic conditions and analysed for fatty acid composition by Lipid Technologies (Austin, MN). The results are shown in Table 1 and are reported as percentage of total fatty acid content. Linoleic acid comprised ∼10% of the total lipid content and the linolenic acid content was measured to be ∼1%, indicating that Borrelia cells do contain lipids that are susceptible to ROS damage. The amount of linoleic acid and linolenic acid present in Borrelia reflected the amount present in the media. Therefore, these results suggested that the amount of these fatty acids in the membranes would vary as availability varies.
To determine if Borrelia polyunsaturated lipids can undergo lipid peroxidation, <span class="yellow">B. burgdorferi</span> B31A3 cells grown microaerobically were treated with 1 mM t-butyl hydroperoxide or 250 mg of lipoxidase, and the cell pellets were analysed for fatty acid composition (Industrial Laboratory). The results are shown in Table 2 and are reported as the percentage of total cell mass. The per cent of linoleic acid in the total cell mass decreased with treatment, while the levels of oleic acid (c18:1n9) and pentadecanoic acid (c15:0) remained relatively constant. Untreated cells contained 0.04% linoleic acid, while cells treated with t-butyl hydroperoxide contained 10-fold less (0.004%) linoleic acid and cells treated with lipoxidase had no detectable linoleic acid present in the sample. These data indicated that the linoleic acid present in <span class="yellow">B. burgdorferi</span> membranes can be oxidized by ROS.
Malondialdehyde (MDA) is generated as a relatively stable end-product from the oxidative degradation of polyunsaturated fatty acids. MDA has thus been used as an indicator of lipid peroxidation (Gutteridge and Halliwell, 1990; Esterbauer et al., 1991). To further demonstrate that Borrelia lipids can undergo lipid peroxidation, B31A3 cells grown microaerobically were treated with 5 mM AAPH (free radical generator) or 250 mg of lipoxidase and MDA measured (Seljeskog et al., 2006). The results are shown in Fig. 3. Untreated cells contained ∼16.5 µM of MDA per mg of protein. When the cells were treated with AAPH or lipoxidase, the amount of MDA increased ∼1.5-fold (27.3 µM of MDA per mg of protein) and approximately twofold (33 µM of MDA per mg of protein) respectively (Fig. 3A). As a control, eukaryotic cells (<span class="blue">mouse</span> myeloma cells SP2) were treated with AAPH and MDA measured (Fig. 3B). <span class="yellow">Mouse</span> myeloma cells have been used as a model system for the determination of phospholipid hydroperoxides (Chotimarkorn et al., 2005). In this case, the amount of MDA increased approximately threefold (36.3 µM of MDA per mg of protein) in the treated cells versus untreated cells (11.7 µM of MDA per mg of protein). Taken together, these data suggested that, like eukaryotic membranes, Borrelia membrane lipids were capable of undergoing lipid peroxidation.
As shown in Fig. 3A, there is a measurable quantity of MDA present even in untreated cells, suggesting that the membrane lipids are damaged without the addition of exogenous ROS. Oxidative damage is an unavoidable by-product of growth in an oxygen environment because superoxide anions and H2O2 are formed whenever molecular oxygen chemically oxidizes electron carriers. To determine if the Borrelia lipids are damaged from growth in an oxygen environment, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under anaerobic, microaerobic and aerobic conditions and MDA measured. Figure 3C demonstrates that as the oxygen concentration increased, the amount of MDA increased. Cells grown under anaerobic conditions contained the lowest amount of MDA (7.6 µM of MDA per mg of protein), approximately twofold less than the measured amount in microaerobic cells (16.5 µM of MDA per mg of protein). Aerobically grown cells contained the highest amount of MDA, ∼1.5-fold greater than that observed in microaerobic cells (28.8 µM of MDA per mg of protein) and ∼3.7-fold greater than that observed in anaerobically grown cells. These data suggested that Borrelia lipids can be damaged during aerobic growth.
Our High Performance Liquid Chromatography analyses of the MDA present in anaerobically grown <span class="yellow">B. burgdorferi</span> cells showed a small peak with a retention time similar to that of the MDA standard. This was puzzling as little or no lipid peroxidation should occur under these conditions. To further characterize this ‘MDA’ peak, a three-dimensional diode array spectra was generated by scanning each sample during the elution of the peak (Fig. 4, lower sections). An authentic MDA standard was also scanned as a control (Fig. 4A, lower section). In the anaerobically grown cells, the spectrum shows that two compounds with different absorbance maximums (Fig. 4B, lower section) comprised the single retention time peak from the HPLC chromatogram (Fig. 4B, upper section). Based on this spectrum, the amount of actual MDA contributed < 15% of the total amount of material detected in the HPLC peak while the second contaminating peak contributed > 85%. Therefore, the amount of MDA in untreated anaerobically grown cells was considerably less than the 7.6 µM of MDA per mg of protein actually measured. However, in cells grown under microaerobic conditions, the MDA peak contributed more to the overall retention time peak when compared with the anaerobic spectrum, while the second contaminating peak stays relatively constant (Fig. 4C, lower section). These spectra demonstrated that the increase in the MDA retention time peak between anaerobically and aerobically grown cells was due to the increase in the amount of authentic MDA present.
The fluorescent probe diphenyl-1-pyrenylphophine (DPPP) has been used for detection of lipid hydroperoxides in cell membranes (Okimoto et al., 2000; Takahashi et al., 2001). In this method, the hydroperoxides are reduced with DPPP, resulting in the formation of the fluorescent arylphosphine oxide. To visualize the lipid hydroperoxides, <span class="yellow">B. burgdorferi</span> B31A3 cells were labelled with DPPP and observed by fluorescence microscopy (Fig. 5B). <span class="yellow">E. coli</span> Top10 cells and <span class="blue">mouse</span> myeloma SP2 cells were also labelled and visualized to serve as negative and positive controls respectively (Fig. 5A and C). Red Fluorescent dye was used to visualize the cell membranes. A strong fluorescence of DPPP was observed for B31A3 cells (Fig. 5B) and for the <span class="blue">mouse</span> myeloma cells (Fig. 5C), but not for the <span class="yellow">E. coli</span> cells (Fig. 5A). An overlay of the two dyes demonstrates that the DPPP fluorescence of both the Borrelia and myeloma cells corresponds to the areas of red fluorescence. This indicated that lipid hydroperoxides were present on the Borrelia cell membranes and suggested that the membranes were damaged.
To further demonstrate Borrelia membrane damage, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under anaerobic and microaerobic conditions and visualized by negative stain using an electron microscope. Additionally, cells grown under microaerobic conditions were treated with 250 mg of lipoxidase and visualized by Electron Microscopy. Intact membranes were observed in cultures of B31A3 grown under anaerobic and microaerobic conditions (Fig. 6A and B respectively). However, in cultures treated with lipoxidase (Fig. 6C), a significant number of membrane blebs were seen surrounding the spirochetes, indicating membrane damage. Taken together, these data indicated that Borrelia membranes were a target for oxidative stress.<br><br>
Discussion
Most bacterial pathogens are faced with the challenge of overcoming ROS generated by the host immune system. These radicals can cause a great deal of damage to biological molecules both in vitro and in vivo. The potential cellular targets for ROS damage in bacteria include DNA, RNA, proteins and lipids, and extensive work has been done to determine the cellular targets of ROS that affect bacterial survival. To date, the most definitive work has been done on <span class="yellow">E. coli</span>. It seems clear from several studies that the most physiologically relevant target of ROS in <span class="yellow">E. coli</span> is DNA. The exposure to µM concentrations of ROS (i.e. H2O2) is sufficient to cause DNA damage, inhibit DNA replication, increase the mutation rate and often lead to cell death. This process leading to DNA damage begins with formation of O2- and H2O2 from the oxidation of flavoproteins and/or the diffusion of these reactive species from the extracellular milieu. The subsequent oxidation of Fe-S proteins by ROS (e.g. O2-) leads to an increase in intracellular ‘free’ Fe2+ which triggers the reduction of H2O2 to OH (Fenton reaction). The highly reactive nature of OH limits its diffusion so that it generally reacts with molecules in close proximity to its origin (e.g. the Fe2+ associated with DNA). In DNA, OH oxidizes sugar and base moieties, producing radicals which ultimately generate lesions, including base modifications, strand breaks and chemical cross-links to other molecules. Base modifications lead to mismatching and increased mutation frequencies while more severe damage, such as strand breaks, prevents DNA replication, contributing to OH toxicity and cell death. Critical to this chemical process in vivo is the presence of iron. No other metal or non-metal electron carrier appears to be able to univalently reduce H2O2 to OH in vivo (Macomber et al., 2007).
These observations are critical in beginning to understand the oxidative damage/targets in <span class="yellow">B. burgdorferi</span>. It has been shown in <span class="yellow">E. coli</span> that the free-iron pool size determines the rate of oxidative DNA damage. For example, in wild-type <span class="yellow">E. coli</span>, free iron levels are estimated to be 10 µM, yet H2O2 is only mildly genotoxic (Keyer and Imlay, 1996). However, in <span class="yellow">E. coli</span> Fur- mutants, intracellular iron concentration increases eightfold while survival is 10-fold lower when cells are exposed to H2O2. Because <span class="yellow">B. burgdorferi</span> cells do not contain detectable levels of intracellular Fe, it seems unlikely that DNA is a major target for damage via the Fenton reaction in this bacterium (Posey and Gherardini, 2000). The experimental data present in this report suggested that this was the case. When <span class="yellow">B. burgdorferi</span> cells were exposed to high concentrations of ROS (e.g. H2O2), there was no effect on the spontaneous mutation rate (Fig. 2A) or the number of DNA lesions (AP sites or 8-oxoguanine) (Fig. 2B and C). It should be noted that in these experiments, a wild-type strain of <span class="yellow">B. burgdorferi</span> was used and presumably all of the endogenous oxidative stress enzymes were expressed. Therefore, it is possible that no DNA damage was observed because these enzymes are capable of detoxifying the cell and protecting nucleic acids from oxidation via the Fenton reaction. However, we do not believe this to be the case as the concentration of oxidants used in these experiments were significantly higher than concentrations known to cause DNA damage in <span class="yellow">E. coli</span> and other pathogenic bacteria.
The lack of detectable DNA damage in <span class="yellow">B. burgdorferi</span> cells under the conditions tested could be the result of very efficient DNA repair systems. Bacteria, such as <span class="yellow">E. coli</span>, harbour genes encoding repair enzymes (i.e. MutM, MutY, Ung, AlkA, MutS, MutL, ExoIII, EndoVIII, PolI, RecJ). Key enzymes in the repair of 8-oxoguanine lesions resulting from the oxidation of DNA are the bifunctional glycosylases, such as MutM, MutY or EndoVIII. The first activity of these enzymes is to remove oxidized or ring-saturated bases while the second activity is to remove the resulting deoxyribose residue, generating a 3′-phosphate end (Krwawicz et al., 2007). This 3′-P is converted to a 3′-OH by various enzymes/pathways and the lesion is repaired by enzymes (e.g. ExoIII, PolI, LigI etc.) in the short or long BER pathways (S-BER, L-BER). Interestingly, the <span class="yellow">B. burgdorferi</span> genome does not contain genes encoding homologues of MutM, MutY or EndoVIII, suggesting that it would be difficult for the cells to efficiently repair 8-oxoguanine sites using this pathway. <span class="yellow">B. burgdorferi</span> does harbour the genes encoding the enzymes Ung (monofunctional gylcosylase, BB0053), MutS (BB0797, BB0098), MutL (BB0211), ExoIII (BB0114), PolI (BB0548), LigI (BB0552) and RecJ (BB0254) (Fraser et al., 1997) which are involved in excision and repair, via S-BER or L-BER, of deaminated, alkylated, methylated or mismatched bases. Clearly, these repair systems in <span class="yellow">B. burgdorferi</span> do not seem as robust at defending against oxidation of DNA as those described in <span class="yellow">E. coli</span>. The lack of key repair enzymes in this system may indicate that <span class="yellow">B. burgdorferi</span> DNA is not subjected to the same challenge from ROS as is <span class="yellow">E. coli</span>.
In most bacteria, ROS-mediated damage to lipids (lipid peroxidation) is very unlikely because of the lack of polyunsaturated fatty acids (e.g. linoleic acid) (Imlay, 2003). When it does occur, lipid peroxidation is initiated by the attack of free radicals on polyunsaturated fatty acids which decreases the membrane fluidity and, if these reactions propagate, lipid peroxides and their degradation products (e.g. aldehydes) in turn could damage proteins (Gutteridge and Halliwell, 1990). This would dramatically affect the function of transmembrane proteins and membrane-bound lipoproteins involved in the maintenance of membrane potential and solute transport, decreasing cell survivability. In contrast to most bacteria, <span class="yellow">B. burgdorferi</span> membranes contained significant levels of unsaturated fatty acids, such as linoleic acid and linolenic acid, which were derived from the growth media (Table 1). Thus, it seemed possible that lipids and/or proteins rather than DNA are the primary targets of ROS in <span class="yellow">B. burgdorferi</span>. When Borrelia cells are treated with oxidants, the levels of linoleic acid decreased while other fatty acids remain unaffected (Table 2). In addition, HPLC-based assays demonstrated that, as linoleic acid concentrations decrease, MDA (a toxic lipid peroxide intermediate) increased (Fig. 3). When these cells were examined by electron microscopy, damage to the membranes (membrane ‘blebs’) was observed (Fig. 6). These data indicated that unlike most other bacteria, Borrelia membranes were damaged by oxygen radicals.
As <span class="yellow">B. burgdorferi</span> may be exposed to potentially harmful oxygen species at different stages of the infective cycle, the ability to protect its membrane lipids from ROS should be required for survival in the different host environments. In eukaryotic cells, where lipid peroxidation is a major consequence of oxidative attack, proteins which protect membranes have been well studied. For example, phospholipid hydroperoxide glutathione peroxidase, a member of the glutathione peroxidase family, has been identified in a variety of higher organisms. These enzymes are capable of reducing an assortment of hydroperoxy lipids, including oxidized phospholipids and cholesterol esters (Ursini and Bindoli, 1987), and protect complex membranes from oxidative damage. Much less is known about the enzyme(s) responsible for protecting unsaturated lipids from ROS in prokaryotes. In <span class="yellow">H. pylori</span>, lipid hydroperoxide levels in ahpC mutants are approximately three times higher than in wild-type cells, suggesting a role for AhpC in reducing organic peroxides (Wang et al., 2006a,b). In addition, purified AhpC has been shown to reduce linoleic acid hydroperoxide in vitro (Baker et al., 2001). Because these antioxidant enzymes promote the in vivo survival of cells when challenged with ROS, enzymes for the reduction of lipid peroxides in Borrelia need to be identified.
Interestingly, Borrelia cells grown aerobically showed signs of membrane damage similar to those observed in cells exposed to various oxidants. The amount of MDA in untreated aerobically grown cells was equivalent to that observed in treated cells (Fig. 3). Also, EM indicated that cells grown microaerobically or aerobically had significantly more membrane damage than cells grown anaerobically (Fig. 6). This indicated that Borrelia membranes can be damaged simply by exposure to physiologically relevant concentrations of dissolved oxygen. In exponentially growing <span class="yellow">E. coli</span>, Imlay and Fridovich (1991) have shown that both O2- and H2O2 are generated by the autoxidation of components of the respiratory chain during oxygen metabolism. In contrast, <span class="yellow">B. burgdorferi</span> is very metabolically limited and has no enzymes for the TCA cycle or respiration. Therefore, it seems more likely that sources of ROS are exogenous (e.g. innate immune response in the mammalian host) rather than endogenous. In addition, the current practice of growing <span class="yellow">B. burgdorferi</span> under atmospheric oxygen could itself be unintentionally compromising cell integrity during in vitro manipulations.
Analyses of the <span class="yellow">B. burgdorferi</span> genome indicates that only a few genes encoding putative oxidative stress/intracellular redox proteins (SodA, NapA, BosR, CoADR, Trx and TrxR) (Fraser et al., 1997) are present, compared with other bacterial pathogens, including other pathogenic spirochetes (e.g. Treponema, Leptospira). Despite this apparent ‘deficiency’ in the number of ROS-protective enzymes, <span class="yellow">B. burgdorferi</span> cells appear to be able to cope with physiologically relevant levels of ROS. There are several factors that would contribute to this phenomenon: (i) as <span class="yellow">B. burgdorferi</span> does not harbour the genes encoding respiratory enzymes nor metabolize oxygen, it seems unlikely that significant levels of ROS are generated via the incomplete reduction of O2 during cellular metabolism. This would suggest that potential ROS challenges to <span class="yellow">B. burgdorferi</span> would come almost completely from extracellular sources with little contribution from intracellularly generated ROS; (ii) owing to a lack of understanding of the physiological conditions in vector and host tissues infected with <span class="yellow">B. burgdorferi</span>, it is difficult to assess the levels and/or sites of the potential ROS challenge during the infective cycle and (iii) most importantly, analyses of the potential targets for ROS in <span class="yellow">B. burgdorferi</span> strongly suggested that the major targets of oxidative damage are different and perhaps less extensive in this bacterium than in other bacterial pathogens (e.g. <span class="yellow">E. coli</span>). Taken together, this suggests that <span class="yellow">B. burgdorferi</span> would be innately more resistant to ROS and require a less extensive repertoire of enzymes to protect the cells from oxidative damage.<br><br>Experimental procedures
Strains, growth conditions and reagents
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 and strain B314 (Sadziene et al., 1993) were grown in modified BSK-II (Barbour, 1984) medium at 34°C under an atmosphere of 0–20% O2 with 5% CO2 and the balance N2. Cells density was determined using a dark-field microscope. All reagents were purchased from Sigma Chemicals, St. Louis, MO unless stated otherwise. <span class="yellow">E. coli</span> strain Top10, strain TA4315 (ahpCF) (Storz et al., 1989) and CM1319 (mutM) (Bridges et al., 1996) were grown in Luria–Bertani (LB) at 37°C with shaking.<br><br>Per cent survivability assays
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 was grown to a cell density of 5 × 107 cells ml−1 in BSK II under microaerobic (3% O2) conditions, the culture split and the cells treated with varying concentrations of oxidants (0–50 mM t-butyl hydroperoxide or H2O2) at 34°C for 4 h. After the incubation, cells were diluted in fresh BSK II, plated on BSK plates and incubated 7–14 days at 34°C. Per cent survivability was calculated as the number of colonies on the treated plates versus the number of colonies on the untreated plates.<br><br>Determination of the spontaneous mutation rate
To determine spontaneous resistance to coumermycin A1, <span class="yellow">B. burgdorferi</span> B31A3 cells were grown under microaerobic conditions to a cell density of 5 × 107 cells ml−1 and treated with 5 mM H2O2 for 1 h at 34°C. The cells were plated on BSK plates containing 0 or 250 ng ml−1 coumermycin A1 and incubated 7–14 days at 34°C. The resistance frequency was calculated as the number of colonies that are CouR per total number of cells plated.<br><br>Measurement of DNA base lesions
<span class="yellow">Borrelia burgdorferi</span> B31A3 cells were grown in BSK-II under microaerobic conditions to a cell density of 5 × 107 cells ml−1, treated with various oxidants (1 or 10 mM t-butyl hydroperoxide, 1 or 10 mM H2O2, 10 mM paraquat or lipoxidase) for 4 h and total DNA was isolated using Wizard Genomic DNA Purification Kit (Promega Corp., Madison, WI). The number of base lesions was determined using the DNA Damage Quantification Colorimetric Assay kit (Oxford Biomedical Research, Oxford, MI) following the manufacturer's protocol. Briefly, 500 ng of DNA was mixed with an equal volume of 10 mM biotinylated aldehyde reactive probe (ARP) reagent and incubated for 1 h at 37°C. The DNA-ARP product was ethanol-precipitated using glycogen as a carrier, washed three times with 70% ethanol and resuspended in Tris-EDTA to give a final concentration of 0.5 µg ml−1. The DNA-ARP product was allowed to bind to the wells of 96-well microplate overnight at 37°C. After the binding, the wells were washed four times with TPBS (137 mM NaCl, 2.7 mM KCl, 10 mM Na3HPO4, 2 mM KH2PO4, 0.5% Tween 20, pH 7.4). The HRP-streptavidin conjugate was diluted to 0.5 µg ml−1 in Assay Buffer (0.15 M NaCl, 10 mM NasHPO4, 1.5 mM KH2PO4, 2.5 mM KCl, 5 mg ml−1 BSA, 0.1% Tween, pH 7.5), 100 µl was added to each well and the plate incubated at 100 r.p.m. for 1 h at room temperature (RT). After incubation, the wells were washed four times with TPBS, 100 µl of substrate was added to each well and incubated for 1 h at 37°C. The reaction was then quenched with 100 µl of 1 M sulphuric acid and the reaction was monitored at 450 nm. The number of aldehyde reactive probe (DNA base lesions) per 105 bp DNA was determined using a standard curve. As a control, <span class="yellow">E. coli</span> TA4315 cells were grown in minimal media to OD600 of 0.4, treated with 0 or 100 µM H2O2 for 30 min and DNA isolated. The number of base lesions was determined as described above. <span class="yellow">B. burgdorferi</span> strain B314 cells were grown under microaerobic conditions and DNA isolated. The number of DNA lesions was determined as described.
To determine the amount of 8-oxoguanine in <span class="yellow">B. burgdorferi</span> DNA, cells were grown and treated as described above and DNA isolated. The DNA was converted to single-stranded DNA by boiling the sample for 5 min followed by rapid chill on ice. The DNA was then digested with nuclease P1 for 2 h and then treated with alkaline phosphatase for 1 h following manufacturer's protocols. The resultant mixture was then centrifuged for 5 min at 6000 g and the supernatant used for the 8-oxoguanine ELISA assay (Oxford Biomedical Research, Oxford, MI). As a control, <span class="yellow">E. coli</span> CM1319 (mutM) cells were grown in LB to OD600 of 0.4, treated with 0 or 100 µM H2O2 for 1 h and DNA isolated. The concentration of 8-oxoguanine was determined as above.<br><br>Lipid analyses
To determine fatty acid content in <span class="yellow">B. burgdorferi</span> total membranes, <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under anaerobic conditions to a cell density of 5 × 107 cells ml−1, harvested by centrifugation (5000 g, 15 min, 4°C) and washed two times with HN (20 mM NaCl, 50 mM Hepes, pH 7.6) buffer. Cell pellets were analysed for fatty acid composition by fatty acid methyl ester (FAME) gas chromatography (Lipid Technologies, Austin, MN) and results are reported as percentage of total fatty acid content. To determine the effects of oxidants on fatty acid composition, a 1.5 l culture of <span class="yellow">B. burgdorferi</span> strain B31A3 was grown under microaerobic conditions to a cell density of 5 × 107 cells ml−1. The culture was split into 500 ml aliquots, the first was treated with 1 mM t-butyl hydroperoxide, the second with 0.25 mg of lipoxidase (17 700 units) and the third was untreated. All cultures were incubated for 12 h at 34°C. Cells were harvested by centrifugation (5000 g, 4°C, 15 min) and washed three times with HN buffer. The fatty acids present in the cell pellets were analysed by FAME gas chromatography (Industrial Laboratory, Wheat Ridge, CO). Fatty acids were reported as percentage of total cell mass.<br><br>Measurement of MDA
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 was grown aerobically, microaerobically and anaerobically as described above to a cell density of 5 × 107 cells ml−1. The microaerobic culture was split and treated with either 5 mM AAPH [2,2′-azobis(2-methylpropionamidine) dihydrochloride] or 250 mg of lipoxidase for 4 h at 34°C. To measure the amount of MDA, the cells were derivatized with thiobarbituric acid and analysed by HPLC as described by Seljeskog et al. (2006). After the incubation, all cells were harvested by centrifugation (1000 g, 5 min, 4°C) and washed three times with HN buffer. Each sample was resuspended in 50 µl of HN, mixed with 150 µl of 0.1 N perchloric acid, 150 µl of 40 mM thiobarbituric acid and 35 µl of 20% SDS, vortexed and heated at 97°C for 60 min. After cooling at −20°C for 20 min, 300 µl of methanol and 100 µl of 20% trichloroacetic acid was added and the samples were mixed vigorously and centrifuged (13 000 g, 6 min). The samples (10 µl) were then analysed with an Agilent Technologies 1200 series HPLC system using a C18 4.6 × 150 mm column with mobile phase 72:17:11 (50 mM KPO4, pH 6.8 : methanol : acetonitrile). Absorbance was monitored at 532 nm. Pure MDA standards (0–10 µM) were prepared in methanol for comparison. As a negative control, <span class="yellow">E. coli</span> Top10 cells were grown to OD600 of 0.4, treated with 0 or 5 mM AAPH for 30 min and MDA measured as above. As a positive control, <span class="blue">mouse</span> myeloma SP2/O cells were cultured with HYQ-CCM1 (HyClone) medium at 37°C in a humidified 5% CO2 atmosphere, treated with 1 mM AAPH at 37°C for 4 h (Chotimarkorn et al., 2005) and MDA measured as above.<br><br>Identification of lipid damage using Diphenyl-1-pyrenylphosphine fluorescent stain
<span class="yellow">Borrelia burgdorferi</span> strain B31A3 cells were grown microaerobically as described above until a cell density of 5 × 107 cells ml−1 was obtained. The culture was divided into two equal aliquots and the cells treated with 5 mM AAPH for 4 h at 34°C. After the incubation, all cells were harvested by centrifugation (1000 g, 5 min, 4°C) and washed three times with HN buffer. To visualize Borrelia cells, the cells were stained with PKH26 Red Fluorescent Cell Linker Dye (Sigma Aldrich) following the manufacturer's protocol. To visualize the lipid hydroperoxides in the cell membrane, the cells were counterstained with DPPP (Cayman Chemicals, MI) (Okimoto et al., 2000). Briefly, after cells were stained with the Red Fluorescent dye, 2 ml of <span class="yellow">rabbit</span> serum was added to stop the reaction and the mixture incubated for 1 min at RT. Next, 4 ml of HN buffer was added, the cells harvested by centrifugation (1000 g, 10 min, RT) and washed three times with HN buffer. The cells were then resuspended in 1 ml of HN buffer, incubated at 34°C for 5 min and 30 µl of 2.5 mM DPPP was added. The incubation was then continued at 34°C for 5 min in the dark. After incubation, the mixture was centrifuged (1000 g, 10 min, RT) and the cells washed three times with HN buffer. The cells were then resuspended in 100 µl HN buffer and observed by fluorescence microscopy with excitation wavelengths 551 nm (Red Fluorescent) and 351 nm (DPPP), and emission wavelengths 567 nm (Red Fluorescent) and 380 nm (DPPP). As a positive control, <span class="blue">mouse</span> myeloma SP2/O cells were cultured with HYQ-CCM1 (HyClone) medium at 37°C in a humidified 5% CO2 atmosphere, treated with 1 mM AAPH at 37°C for 4 h (Chotimarkorn et al., 2005) and stained as described above. As a negative control, <span class="yellow">E. coli</span> Top10 cells were grown to OD600 of 0.4, treated with 0 or 5 mM AAPH for 30 min and stained as described above.<br><br>Electron microscopy
<span class="yellow">Borrelia burgdorferi</span> cells were grown under microaerobic or anaerobic conditions to a cell density of 5 × 107 cells ml−1. The microaerobic cultures were split and treated with lipoxidase for 4 h at 34°C. Cells were harvested by centrifugation (5000 g, 10 min, 4°C), washed and resuspended in HBSS (Lonza Group Ltd, Switzerland). The cells were fixed with Karnovsky's phosphate for 5 min at RT adsorbed to Formvar/carbon-coated grids (Ted Pella, Redding, CA) for 5 min and washed three times in H2O. The grids were stained with 1% ammonium molybdate and allowed to air-dry. Samples were examined using a Hitachi H7500 electron microscope (Hitachi High Technologies America, Pleasanton, CA).<br><br><br><br><h3>pmcA2365205</h3>Cytotoxic Activity of Silyl- and Germyl-Substituted 4,4-Dioxo-3a,6a-Dihydrothieno[2,3−d]isoxazolines-2
Abstract
The [2+3] dipolar cycloaddition of nitrile oxides to the double C = C bonds of thiophene-1, 1-dioxides leads to formation of the fused isoxazolines-2 (1, 2). Tumor growth inhibition of these compounds strongly depends on the nature of group IV A element increasing from slightly active tert-butyl derivatives to silicon and germanium containing analogues. The products of benzonitrile oxide cycloaddition have greater cytotoxic effect than the compounds obtained from the cycloaddition reaction of 2, 5-disubstituted thiophene-1, 1-dioxides with acetonitrile oxide. Fused silyl substituted isoxazolines-2 are stronger NO-inducers than their germyl and tert-butyl analogues.
CYTOTOXIC ACTIVITY OF SILYL- AND GERMYL-SUBSTITUTED 4,4-DIOXO-3a,6a-DIHYDROTHIENO [2,3-d] ISOXAZOLINES-2 <br><br> E. Lukevics*, P. Arsenyan, I. Shestakova, O. Zharkova, I. Kanepe, R. Mezapuke, and O. Pudova Latvian Institute of Organic Synthesis, Aizkraukles 21, Riga, LV-1006, Latvia <br><br> ABSTRACT The [2+3] dipolar cycloaddition of nitrile oxides to the double (3=(3 bonds of thiophene-l,l-dioides leads to formation of the fused isoxazolines-2 (1, 2). Tumor growth inhibition of these compounds strongly depends on the nature of group IV A element increasing from slightly active ert-butyl derivatives to silicon and germanium containing analogues. The products of benzonitdle oxide cycloaddition have greater cytotoic effect than the compounds obtained from he cycloaddition reaction of 2,5-disubstituted thiophene-l,l-dioxides with acetonitrile oxide. Fused silyl substituted isoxazolines-2 are stronger NO-inducers than their germyl and tert-butyl analogues. <br><br> INTRODUCTION The interest in silyl substituted thiophene-l,l-dioxides stems from the fact that they are useful synthetic intermediates for the preparation of various types of organic compounds by Diels-Alder cycloaddition [1], amine induced ring-opening reaction [2], or coupling of bromothiophene-l,l-dioxides with thienyl stannanes in the presence of a palladium (0) catalyst [3]. It has been shown that unsubstituted thiophene-l,l-dioxide prepared in situ is a quite reactive dipolarophile in the [2+3] cycloaddition reactions with N,o-diphenylnitrone [4], benzonitrile [4, 5] and mesitonitrile [4, 5] oxides yielding mono- and diisoxazolines-2 and N-substituted isoxazolidines. Moreover, our recent studies indicate that silyl- and germylcontaining isoxazolines have gained a great deal of attention as compounds possessing a wide spectrum of the biological properties. The vasodilating, anticoagulant and cardioprotective activity of 5-Si-(Ge)substituted isoxazolines-2 has been studied in vitro and in vivo [6, 7]. The most active isoxazoline -3-(5"-triethylgermyl-3"-isoxazolino)pyridine hydrochloride protected the heart from rhythm disturbances and lethality during ischemiareperfusion [7]. It has been shown that silylisoxazolines-2 are more potent in protection against hypoxia and corazole convulsions than germanium analogues. However, germylisoxazolines-2 are stronger tumor growth inhibitors and NO-inducers than their silicon analogue [8]. This work presents the results of cytotoxic activity for fused isoxazolines-2 bearing a group 14 element as substituent (1, 2) in function of the nature of the group 14 element. <br><br> M <br><br> e3M.CH3 N <br><br> -0\ R <br><br> O\ M <br><br> e3M S <br><br> N <br><br> 02 <br><br> 02 2 <br><br> 1 <br><br> MATERIALS AND METHODS CHEMISTRY Seven tert-butyl-, trimethylsilyl-, and trimethylgermyl-substituted 4,4-dioxo-3a,6adihydrothieno[2,3-d]isoxazoline-2 1 and 2 (Table 1)were prepared by the [2+3] dipolar 63 <br><br> Vol. 7, No. 2, 2000 <br><br> Cytotoxic Activity ofSilyl-and Germyl-Substituted 4, 4-dioxo-3a ,6a-Dihydrothieno[2, 3-d]Isoxazolines-2 <br><br> cycloaddition of aceto- and benzonitrile oxides to 2,5-disubstituted thiophene-l,l-dioxides. Their synthesis and characterization are given in ref. [9]. <br><br> _,,/0\N Me3M M'Me3 <br><br> Me3M <br><br> 02 M, M'=C, Si, Ge; R=Me, Ph; R'=H, Me3Ge Table 1. Me3C, MeSi, Investigated dihydrothieno[2,3-d]isoxazolines-2 <br><br> 02 Me3Ge substituted <br><br> %, \R 4,4-dioxo-3a,6a<br><br> Compound <br><br> 'Type la <br><br> M <br><br> R H Me3Ge Me3Ge <br><br> Yield <br><br> 4,4-dioxo-3 methyl"5-tert-butyl-3a,6adihydrothieno[2,3-d]isoxazoline-2 <br><br> (3' C Si <br><br> (%) 80 <br><br> 4,4-dioxo-3-methyl-3a-trimethylgermyl-5-tert-butyl3a, 6a-d ihyd rothieno[2,3-d]isoxazoli ne-2 4,4-dioxo-3-methyl-3a-trimethyigermyl-5tri methylsilyl-3a, 6a-d hyd roth ien o[2,3d]isoxazoline-2 4,4-d io x o-3- meth y I-3a, 5- b s tri m ethy Ig e rmy I)3a, 6a-d hyd rothieno[2,3-d] isoxazo ne-2 4,4-d ioxo-3-ph en y I-5-tert-b uty I-3a, 6adihydrothieno[2,3-d]isoxazoline-2 4,4-d ioxo-3-p h e ny I-5-tri meth y si ly I-3a, 6adihydrothieno[2,3-d]isoxazoline-2 4,4-d ioxo-3-p h e n y I-5-tri methy Ig e rmy I-3a, 6adi,hydr0thieno[2,3, d]isoxazoline-2 <br><br> lb lc <br><br> 58 45 67 84 <br><br> ld 2a 2b <br><br> Ge <br><br> Me3Ge <br><br> C Si <br><br> 77 85 <br><br> 2c <br><br> Ge <br><br> IN VITRO CYTOTOXITY ASSAY Monolayer cells lines were cultivated for 72 h in DMEM standard medium without an indicator and antibiotics. After the ampoule was defreezed not more than four passa4ges were performed. The control cells and cells with tested substances in the range of 2-5 10 cell/mL concentration (depending on line nature)were placed on a separate 96 wells plates. Solutions containing test compounds were diluted and added in wells to give the final concentrations of 50, 25, 12.5, and 6.25 #g/mL Control cells were treated in the same manner only in the absence of test compounds. Plates were cultivated for 72 h. A quantity of survived cells was determined using crystal violet (CV) or 3-(4,5-dimethylthiazol-2-yl)-2,5diphenyltetrazolinium bromide (MTT) coloration which was assayed by multiscan spectrofotometer. The quantity of alive cells on control plate was taken in calculations for 100% [10, 11]. Concentration of NO was determined according to [10]. RESULTS AND DISCUSSION Potential cytotoxic activity of synthesized fused isoxazolines 1 and 2 was tested in vitro on four monolayer tumor cell lines: MG-22A (<span class="blue">mouse</span> hepatoma), HT-1080 (<span class="yellow">human</span> fibroblastoma), B16 (<span class="blue">mouse</span> melanoma), Neuro 2A (<span class="blue">mouse</span> neiroblastoma). Concentrations providing 50% of tumor death effect were determined according to the known procedure [12] using 96 well plates. The experimental evaluation of cytotoxicity properties is presented in Table 2. A preliminary analysis of the structure-activity relationship for the cytotoxic action clearly indicates the strong influence of the MeM (M=C, Si, Ge) group in position 5 of fused isoxazolines 1 and 2. Derivatives bearing tert-butyl substituent (la,b and 2a) have a slight cytotoxic effect (> 10 #g/mL). The substitution of the tert-butyl group by trimethylsilyl or 64 <br><br> E. Lukevics et al. <br><br> Metal-Based Drugs <br><br> A <br><br> oO <br><br> {D <br><br> 0 <br><br> 0 <br><br> 65 <br><br> Vol. 7, No. 2, 2000 <br><br> Cytotoxic Activity ofSilyl-and Germyl-Substituted 4, 4-dioxo-3a ,6a-Dihydrothieno[2, 3-d]Isoxazolines-2 <br><br> trimethylgermyl ones leads to considerable increase of cytotoxicity. It must be noted that the activity of silicon- and germanium-containing compounds (1 and l d) depends on the tumor type. 5-Trimethylsilyl-substituted fused isoxazoline le is more active than the germanium analogue l d in tests on HP-1080 and MG-22A cell lines. However, the germanium compound ld has greater cytotoxic effect on Neuro 2A and B16 cell lines than the silicon derivative lc. Comparison of the tumor growth inhibition for derivatives 1 and 2 shows a higher activity of the condensed isoxazolines 2 containing a phenyl group in position 3 with respect to 4,4-dioxo-3-methyl-3a-trimethylgermyl-5-MeM-3a,6a-dihyd rothieno[2,3d]isoxazolines l b-d. Silyl- and germyl-substituted fused isoxazolines have a medium NOinduction ability, 4,4-dioxo-3-phenyl-5-trimethylsilyl-3a,6a-dihydrothieno[2,3-d]isoxazoline-2 (2b) being the most active (250% in the MG-22A test). <br><br> ACKNOWLEGMENT We are grateful to Latvian Taiho Foundation for financial support. REFERENCES 1. A. R. M. Donovan, M. K. Shepherd, Tetrahedron Lett., 35 (1994)4425. 2. S. Gronowitz, A.-B. H0rnfeldt, E. Lukevics, O. Pudova, Synthesis, (1994)40. 3. G. Barbarella, L. Favaretto, G. Sotgiu, M. Zambianchi, L. Antolini, O. Pudova, A. Bongini, J. Org. Chem., 63 (1998) 5497. 4. A. Bened, R. Durand, D. Pioch, P. Geneste, J. P. Declerq, G. Germain, J. Rambaud, R. Roques, J. Org. Chem., 46 (1981) 3502. 5. F. M. Albini, P. Ceva, A. Mascherpa, E. Albini, P. Caramella, Tetrahedron, 38 (1982) <br><br> 6. 7. 8. <br><br> 9. 10. 11. 12. <br><br> 3629. E. Lukevics, M. Veveris, V. Dirnens, Appl. Organomet. Chem., 11 (1997) 805. E. Lukevics, P. Arsenyan, M. Veveris, Metal Based Drugs, 5 (1998) 251. E. Lukevics, P. Arsenyan, S. Germane, I. Shestakova, Applied Organomet. Chem., 13 (1999) 795. E. Lukevics, P. Arsenyan, S. Belyakov, J. Popelis, O. Pudova, Organometallics, 18 (1999) 3187. D.J. Fast, R.C. Lynch, R.W. Leu, J. Leuckocyt. Biol., 52 (1992) 255. P.J. Freshney, Culture of Animal Cells (A Manual of Basic Technique), Wiley-Liss, New York, 1994, pp. 296-297. R. J. Riddell, R. H. Clothier, M. Fd. Balls, Chem. Toxicol., 24 (1986) 469. <br><br> Received: January 21, 2000 Accepted: February 1, 2000 Received in revised camera-ready format" February 2, 2000 <br><br> 66 <br><br>  <h3>pmcA2365769</h3>IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome
Abstract
This study was undertaken to evaluate the role of IL-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum IL-5 levels that might be essential for eosinophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-IL-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of IL-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce IL-5.
Research Paper Mediators of Inflammation 5, 24-31 (1996) <br><br> Tins study was undertaken to evaluate the role of 11-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum 11-5 levels that might be essential for eosi<br><br> IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome L. H. Faccioli, 1"cA V. F. Mokwa, C. L. Silva, G. M. Rocha, J. I. Araujo, M. A. Nahori 2 and B. B. Vargaftig 2 <br><br> nophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-11-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of 11-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing 11-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce 11-5. <br><br> 1Department of Parasitology, Microbiology and Immunology, School of Medicine of Ribeir,5o Preto, 14049-900, Ribeiro Preto, SP, Brazil. Fax: (+55) 16 633 6631 2Unit de Pharmacologie Cellulaire, Unit Associe Institut Pasteur/INSERM n. 285, Paris, <br><br> France. <br><br> CACorresponding Author <br><br> Key words: Eosinophil, Eosinophilia by <span class="yellow">Toxocara canis</span>, IL-5 in eosinophilia, <span class="yellow">Toxocara canis</span> <br><br> Introduction <br><br> treatment in <span class="yellow">mice</span> infected with <br><br> Eosinophilia has been associated with parasitic diseases, particularly when the parasites invade the tissues or injure the mucosal surfaces. <span class="yellow">Toxocara canis</span> is an intestinal parasite of <span class="yellow">dogs</span>, and is the most common aetiologic agent of visceral larva migrans syndrome (VLMS). In <span class="yellow">humans</span>, VLMS results from the ingestion of embryonated eggs of <span class="yellow">T. canis</span>, that eclode in the small intesfine. The infective larvae invade the mucosa, move into the liver via the portal circulation, and from there to the lungs. 2 Beaver et al., 3 who were the first to describe this syndrome, noted the intense eosinophilia which reaches more than 90% of total leucocyte counts. However, there are few studies regarding the mechanisms involved in the blood and tissue eosinophilia obseeeed in VLMS. Several investigators have suggested a direct correlation between eosinophilia and interleukin5 (IL-5) in <span class="yellow">human</span> helminth infections4'5 and in experimental animal models. 67 Inhibition of eosinophilia has been demonstrated by anti-IL-5 24 Mediators of Inflammation Vol 5 <br><br> brasiliensis, 8 $chistosoma mansoni, <span class="yellow">Toxocara canis</span>7 and <span class="yellow">Heligmosomoidespolygyrus</span>. 1 IL-5 has also been shown to support the terminal differentiation, proliferation of eosinophil precursors 11'12 and eosinophil activation. 13 Although IL-5 does not demonstrate eosinophil chemotactic activity in vivo14 there is some evidence suggesting that this cytokine may modulate a selective eosinophil accumulation at the site of inflammation. Moreover, Sehmi et a/.15 reported that IL-5 has a selective priming effect on eosinophil migratory response to nonselective chemoattractant mediators in vitro. Also, Moser et a/. 16 have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with IL-5, IL-3 and GM-CSF. Thus, the involvement of IL-5 in eosinophilia is not fully understood. In the present study we have used a <span class="yellow">guinea-pig</span> model of VLMS to investigate the involvement of IL-5 in eosinophil migration and in the maintenance of eosinophilia in blood, bone marrow, lung and peritoneal cavity. (C) 1996 Rapid Science Publishers <br><br> Nipostrongylus <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> Materials and Methods Animals: Outbred albino weighing 300-400g at the ments were obtained from the School of Medicine of versity of So Paulo, Brazil. <br><br> female <span class="yellow">guinea-pigs</span> start of the experi<br><br> were routinely processed, embedded in paraffin, sectioned at 4-61.tm, and stained with Chromothope 2R and haematoxylin, for examination by light microscopy. Determination oflL-5 in serum: The IL-5 level in the serum of <span class="yellow">guinea-pigs</span> was measured using an enzyme-linked immunosorbent assay (ELISA). Briefly, ELISA plates (96-well Immunoplate MaxiSorp, Nunc, Roskilde, Denmark) were coated with IL-5-specific monoclonal antibody (TRFK-5, 5 lag/ml in phosphate buffered saline, pH 7.4, PBS, 100 l.tl/well). After 2 h of incubation at 37C, the wells were washed four times with PBS containing 0.1% Tween 20 (PBS-T). Then, 100 btl of samples or recombinant <span class="yellow">murine</span> IL-5 standards (0.15-200ng/ml) in PBS-T and 2% BSA (PBSTBSA) were added to each well. After incubation for l h at 37C, the wells were washed three times and 100 l.tl of biotinylated <span class="yellow">rat</span> anti-<span class="yellow">murine</span> IL-5 (biotinylated-TRFK-5, 1 lag/ml in PBS-TBSA) was added. After incubation for 1 h at 37C, the wells were washed again three times and 100 of peroxidase-labelled streptavidin (1/1000, Kirkegaard & Perry Laboratories Inc., Maryland, USA) were added to each well. Following incubation for l h at 37C and further washing, the enzyme was developed using the TMB substrate peroxidase for 5 min. The reaction was stopped by adding 501.tl of 2.0 N HCl, and the optical densities were read at 490nm using an automated plate reader. The sensitivity of the assay was 0.15 ng/ml and the upper limit 100ng/ml. <br><br> the animal house of Ribeiro Preto, Uni<br><br> Infection of animals: <span class="yellow">T. canis</span> eggs were obtained by the methods of Olson and Schulz, 17 with minor modifications. Briefly, gravid female <span class="yellow">worms</span> were recovered from <span class="yellow">dogs</span>, and the eggs were rescued from the uterus, washed and allowed to develop to the infective stage in shallow dishes containing 0.5% formalin at 37C. Under light ether anaesthesia, the animals were infected with I ml saline containing 500 <span class="yellow">T. canis</span> eggs, by gastric intubation using a metal cannula. Blood cell counts: <span class="yellow">Guinea-pigs</span> were anaesthetized with sodium pentobarbitone (30mg/kg, i.v.) and blood samples were collected by cardiac puncture with 10% EDTA. Total cell counts were carried out using diluting fluid in a Neubauer chamber. Differential countings were obtained using Rosenfeld-stained cytocentrifuge preparations, <br><br> Bronchoalveolar lavage fluid. The <span class="yellow">guinea-pigs</span> were killed by an overdose of sodium pentobarbitone and 5ml of phosphate-buffered saline (PBS) containing 0.5% sodium citrate (PBS/SC), at room temperature, were instilled through a polyethylene cannula introduced into the trachea. The cells present in the bronchoalveolar lavage fluid (BALF) were recovered immediately. The procedure was repeated once. The leucocyte counts in the BAI_ were determined as described above, Peritoneal cells: The cells from the peritoneal cavities were harvested by injection of 10ml of PBS/SC into the peritoneum. Only 5-8 ml of the <br><br> fluid was withdrawn for cell counts, as described above, <br><br> Monoclonal antibodies: The <span class="yellow">rat</span> monoclonal antibody TRFK-5 was a generous gift from Dr P. Minoprio, Institut Pasteur, Paris. The neutralizing antibody was purified by precipitation with ammonium sulfate (45%) from ascites prepared in CD1 <span class="yellow">nude mice</span> (Charles River, St Aubin les Elbeuf, France) inoculated 1 week before the injection of hybridoma cells, with I ml of pristane (Sigma). After precipitation and dialysis of the ascite fluid overnight against PBS, the dialysate was further purified on a Protein G1 column (HiTrapTM, Pharmacia Upsala, Sweden). <br><br> Bone marrow cells: Bone marrow cells were collected by flushing the contents of the <span class="yellow">guinea-pig</span> femur with 10 ml of PBS/SC. Total cell numbers were determined as above. In the differential cell counts the cell populations were divided into mature neutrophils, mature eosinophils and others (mainly precursors and mononuclear <br><br> cells), Histopathological studies: Tissues were removed from <span class="yellow">guinea-pigs</span> at various times post-infection and immediately fixed in 10% formalin. Tissues <br><br> Eosinophil and cytokine depletion: <span class="yellow">Guinea-pigs</span> were injected i.p. with TRFK-5 or with the irrelevant antibody (<span class="yellow">rat</span> IgG against total anti-<span class="yellow">human</span> IgG) once, 2mg/animal, at the time of infection or at different intervals (1, 3, 12 or 17 days) thereafter. The animals in this group were sacririced 18 days after infection. liver: One lobule of was used to determine the larval each liver counts from infected <span class="yellow">guinea-pigs</span>. Larval recovery <br><br> Recovery <br><br> of larvae from <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 25 <br><br> L. H. Faccioli et al. <br><br> was evaluated as described by Kayes and Oaks, 18 with minor modifications. Briefly, the tissue was chopped and digested with pepsin-HC1 (pH 1.5<br><br> 1.8) for 2h at 37C. Larval counts for each sample were performed after centrifugation and examination of three 100-l.tl samples under the light microscope. Statistical analysis: Data are presented as the mean _+ S.E.M. and were analysed statistically using the Mann-Whitney test for unpaired data. A p < 0.05 value was considered to be statistically <br><br> significant. <br><br> Results <br><br> of eosinophil counts in blood bone marrow, BALF and peritoneum: <span class="yellow">Guinea-pigs</span> infected with <span class="yellow">T. canis</span> eggs showed a timedependent blood, bone-marrow, BAUV and peritoneal eosinophilia (Fig. 1). The results represent the mean of nine animals obtained in three different experiments. The eosinophil number Kinetics 16<br><br> increased significantly from 0.55 +_ 0.37 x 105 at the beginning of experiment to 6.0 +__ 1.03 x 105 at 6 days post-infection, peaked at day 18 (12.0 _+ 2.31 x 105), and decreased by day 24 (8.11 2.85 x 105) (Fig. 1A). A rise in the percentage of mature eosinophils in bone marrow" was observed 12 days after infection (ranging from 6 2% to 14 2%) and peaked at 18 days (17 2%) (Fig. 1B). As in blood, the number of eosinophils in BALF increased significantly from 0.14 0.06 x 105 to 1.37 _+ 0.35 x 105 at 6 days after infection, reaching a peak at 18 days (10.23 2.62 x 105) with an increase in relative number of as much as 90% in eosinophil counts in relation to controls, and was still elevated at day 24 (9.07___ 3.47 x 10 s) (Fig. 1C). The remaining cells in the BALF were alveolar macrophages, lymphocytes, mast cells and ciliated cells. In contrast to blood and BALF, the number of eosinophils in the peritoneal cavity increased significantl only at day 12 post-infection (onset, 2.06 1.04 x 105; day 6, 3.68 _+ 0.82 x 105; day 12, 5.77 +_ 1.12 x 10>; and increased progres<br><br> __ _ _ _. (B) <br><br> (AI Blood <br><br> 24 20" <br><br> Bone Marrow <br><br> 1612 8 <br><br> 4 O0<br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 111 <br><br> 21 <br><br> 24 <br><br> 16<br><br> (c) BALF <br><br> 16<br><br> ID) Peritoneum <br><br> 12<br><br> __o <br><br> 12<br><br> 8 <br><br> 4 <br><br> 0 3 <br><br> O6 9 <br><br> 12 <br><br> 15 <br><br> 16 <br><br> 21 <br><br> 24 <br><br> Days postinfoction <br><br> FIG. 1. Number of eosinophils in blood, BALF and peritoneal cavity, and percentage, of eosinophils in bone marrow of <span class="yellow">T. canis</span>qnfected <span class="yellow">guinea-pigs</span>. Values are the mean -t-S.E.M. (n=8 to 9). Asterisks indicate a significant difference between infected and noninfected animals (n 5 6). *p < 0.05 and **p < 0.01. 26 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> sively until day 24, 12.44 4- 2.72 x 105) (Fig. 1D). The percentage of eosinophils in some animals reached 55% at the peak of infection. No increase in the number of mononuclear cells was seen in any compartment analysed. Larval counts: The percentage of inoculated <span class="yellow">T. canis</span> larvae recovered by peptic digestion of the liver of experimental animals 4 h and 1, 2, 3, 4, 9, 12 and 18 days after inoculation of 500 eggs per animal is shown in Fig. 2. Most of the larvae were recovered 2 to 4 days after infection and 10% recovery was also observed on day 18 in the liver of the animals. <br><br> 150 <br><br> 120 <br><br> IL-5 level in serum of infected animals: IL-5 was measured in the serum of infected and normal <span class="yellow">guinea-pigs</span>. Each time point in Fig. 3 represents the mean of results from three to five infected animals, and from six controls. Two peaks of IL5 were present in the serum of infected <span class="yellow">guineapigs</span> 1 day after infection (102 __+ 22 pg/ml), and 18 days later (59 7 pg/ml). The level of IL-5 in the controls was 31 4- 4 pg/ml. Eosinophil numbers in infected animals treated with TRFK-5: When <span class="yellow">guinea-pigs</span> received an i.p. injection of TRFK-5, the monoclonal antibody against IL-5, at the time of egg administration or 1 day later, the number of eosinophils in blood, BALF, peritoneal cavity and bone marrow was <br><br> _ <br><br> 30<br><br> 0 <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 12 <br><br> 18 <br><br> Days postinfection <br><br> FIG. 3. IL-5 concentration in serum of <span class="yellow">T. cani</span>infected <span class="yellow">guineapigs</span> (n 3-5). Basal IL-5 concentrations were of 31 4 pg/ml (n= 12). <br><br> _ <br><br> 24 <br><br> 60<br><br> 50<br><br> 40<br><br> 0 <br><br> "T" <br><br> 1" <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 18 <br><br> 21 <br><br> 24 <br><br> Days postinfection FIG. 2. Percentage of <span class="yellow">T. canis</span> larvae recovered from liver of <span class="yellow">guinea-pigs</span> studied at various times post-infection. Data obtained from five animals. <br><br> drastically reduced, even when determined 18 days after infection (Table 1). No inhibition of eosinophil counts was observed when the animals were inoculated with the irrelevant antibody at the time of infection (Table 1). Fig. 4 shows the comparative results of eosinophilia obtained when the antibody was given 3 days or 17 days after egg inoculation. The antibody given at 3 days after infection induced a high percentage of inhibition in eosinophil counts in all the compartments analysed 18 days after infection (Fig. 4A). However, when TRFK-5 was administered to the infected animals on day 17 post-infection (thus 1 day before sacrifice), a significant inhibition in number and percentage of eosinophils was observed only in the blood (p=0.030) (Fig. 4B). A small non-significant decrease was seen in BALF (p 0.790) and peritoneum (p= 0.222). Moreover, the number of mature eosinophils in bone marrow increased by 140% (p 0.038). As demonstrated in Fig. 4B, the behaviour of eosinotShilia in BALF was completely different from that observed in blood. Thus, to better understand the eosinophilia in the lungs of infected animals, we monitored eosinophil numbers in BALF after administration of TRFK-5 at the same time, or 1, 3, 12 or 17 days after infection. The animals were sacrificed 18 days after infection. In another group, TRFK-5 was administered 18 days post-infection and the animals were sacrificed 6 days later. When the mAb was administered at the same time or 1 or 3 days postinfection there was a significant inhibition in the number of eosinophils (Fig. 5). These data show Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 27 <br><br> L. H. Faccioli et al. Table 1. Eosinophils in T. caniinfected <span class="yellow">guinea-pigs</span> treated or untreated with TRFK-5 Compartment <br><br> Time of sacrifice <br><br> (days) <br><br> Non18 24 18 24 18 24 18 24 <br><br> treated <br><br> Blood <br><br> 12.18 5.28 10.08 -t- 2.85 16.50 4.42 6.57 +__ 1.68 <br><br> BALF Peritoneal cavity <br><br> 12.31 -I- 2.35 12.44 2.33 9 6 <br><br> Bone marrow <br><br> _ _ _ _ _ ___ _ __ __ _ __ Days of treatment with TRFK-5 after egg administration <br><br> Irrelevant Ab at the time of infection <br><br> 0 <br><br> 3 <br><br> 12 <br><br> 17 <br><br> 18 <br><br> (n 6) <br><br> (n=4) <br><br> (n=4) <br><br> (n 5) <br><br> (n 5) <br><br> (n=4) <br><br> 0.14 -t- 0.14" <br><br> 0.26 0.79 <br><br> 0.26* 0.24* <br><br> 0.17 _-t- 0.17" <br><br> 0.15 <br><br> 0.15" <br><br> 2.59 <br><br> 0.80* 4.36 2.21 <br><br> 13.32 <br><br> 4.54 <br><br> O. 15 <br><br> 0.15* <br><br> 0.36 <br><br> 0.10" <br><br> 1.22 -t- 0.50* <br><br> 7.07 -t- 2.39 <br><br> 15.24 <br><br> 29.31 _+ 15.90 16.09 <br><br> 10.33 _+ 4.10 3.79 <br><br> 1.47 -t- 0.76* 3 <br><br> 0.48 <br><br> _+ 0.35* <br><br> 0.46 2.6 <br><br> 0.27* 1.3" <br><br> 2.89 <br><br> 0.94* <br><br> 10.42 <br><br> 4.00 <br><br> 1.58" <br><br> -+ <br><br> 1" <br><br> 1.25 -I- 0.25* <br><br> + <br><br> 5 <br><br> 15 -t- 2 <br><br> 9_+1 <br><br> 2 <br><br> 0.7* <br><br> In blood BALF and peritoneal cavity the values represent mean -t-_ S.E.M. x 10 eosinophils. *p < 0.05. <br><br> ml-1 <br><br> and in bone marrow mean -t-_ S.E.M. of the percentage of mature <br><br> that the inhibition of the first peak of IL-5 which appeared at 1 to 3 days after infection as shown in Fig. 3, is also very important for the establishment of eosinophilia in the lungs. However, 24- (A) <br><br> when the mAb was administered 12, 17 or 18 days after infection there was no significant inhibition in the numbers of eosinophils in BALF (Fig. 5), showing that once established, the eosi18<br><br> 15x <br><br> 16<br><br> 12<br><br> ._ . 0 <br><br> 12 <br><br> o <br><br> BIo o d <br><br> BALF <br><br> P e rit oneum <br><br> Bone marrow <br><br> 24- (Is) <br><br> 1815<br><br> E x <br><br> 20<br><br> 16<br><br> o <br><br> 12///. <br><br> o = o <br><br> 12<br><br> 0 Blood <br><br> O" <br><br> BALF <br><br> Peritoneum <br><br> Bone marrow FIG. 4. Number of eosinophils in blood, BALF and peritoneum and percentage of eosinophils in bone marrow of <span class="yellow">T. cani</span>infected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) 2 mg/animal at 3 days post-infection; (B) 2 mg/animal 17 days post-infection. The treated and control animals were sacrificed 18 days after infection. Asterisks indicate a significant difference from infected controls (n=5-6) and from animals treated with TRFK-5 (n=4- 5). *p < 0.05 and **p < 0.01. 28 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> 24<br><br> E <br><br> 20<br><br> heart; data not shown) and muscle, as reported by other investigators, 8 were infiltrated. The factors responsible for in vivo eosinophil accumulation at inflammatory sites have been <br><br> 0 <br><br> TRFK-5 i.p. <br><br> 0 <br><br> 3 <br><br> 12 17 <br><br> A <br><br> B <br><br> FIG. 5. Number of eosinophils in BALF of T. caniinfected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) animals were sacrificed 18 days post-infection and (B) 24 days after infection. Asterisks indicate a significant difference from infected control and from animals treated with TRFK-5 (p< 0.01). <br><br> nophilia persists in lungs, probably by the secretion of IL-5 from cells localized in the lung microenvironment, <br><br> Histopathological analysis: The treatment of <span class="yellow">T. canis</span>-infected animals with irrelevant antibody showed a widespread eosinophilic infiltration as in untreated animals (Fig. 6A, B). However, the treatment of animals with TRFK-5 at the same time of infection, or 1 day or 3 days later ted to a complete inhibition of eosinophil infiltration in the lung parenchyma (Fig. 6C). By contrast, the mononuclear cell infiltration in the lungs was not modified. When the infected <span class="yellow">guinea-pigs</span> received TRFK-5 1 day before sacrifice (or 17 days post-infection), eosinophil infiltration in the lung parenchyma was also inhibited (Fig. 6D) but not to the same extent as observed in the group receiving TRFK-5 given at the time of infection or 3 days later. Thus, the histological determination of eosinophil infiltration in these lungs corroborates a reduction but not a sizeable inhibition of eosinophil numbers as observed in the BALF of the same infected animals, <br><br> poorly defined, although T lymphocytes and mast cells appear to be involved in eosinophilia. 9'2 IL-5, a T cell-derived factor that regulates B cell functions, is an eosinophil differentiation factor11 as well as a stimulating and survival-prolonging factor specific for eosinophils in vitro. 2 Also, several investigators have demonstrated that sTstemic eosinophilia in <span class="yellow">mice</span> infected with parasites is mediated by IL-5 produced in response to the infection. 2'22 In the present study, the i.p. administration of the TRFK-5 antibody markedly inhibited the widespread eosinophilia observed in <span class="yellow">T. canis</span>-infected <span class="yellow">guinea-pigs</span>, indicating that IL5 participated in a <span class="yellow">guinea-pig</span> model of VLMS eosinophJlia. Most of the <span class="yellow">T. canis</span> larvae which penetrated the intestinal wall had migrated into the liver within 72h after inoculation as demonstrated here and elsewhere. 2 It is apparently during this interval that the worm provides the signals to cytokine-producing cells, which in turn trigger increased serum levels of specific cytokine as demonstrated here for IL-5, 24 to 72 h after infection. The signals may be provided directly by the invading parasite or by cells in response to the parasite. The cytokine pattern that develops at this early stage, probably induced by a T-cell independent pathway, may also influence the pattern of T cell differentiation into a Th2 type, which may be responsible for the second peak of IL-5 observed in our experimental model (Fig. 3), although a second cycle of larval invasion (Fig. 2) with a rapid peak of IL-5 liberation cannot be ruled out. <br><br> Discussion The results of the present study show that in our experimental model widespread eosinophilia follows the infection of <span class="yellow">guinea-pigs</span> with second stage eggs from <span class="yellow">T. canis</span>, as also noted in <span class="yellow">humans</span> and in other experimental animals. 7'7 <span class="yellow">T. canis</span> is a potent stimulus for systemic eosinophilia, since blood, BALF, peritoneum and all tissues examined (kidney, eyes, spleen, thymus, <br><br> Thus, our results suggest that the eosinophilia against helminth larvae may be initiated by the release of IL-5 when the parasites migrate from the intestine to the liver by stimulation of specific cell populations. Then, an early release of IL-5 quickly induces eosinophil recruitment, probably first from the stored mature eosinophil pool from vascular endothelium or by the mobilization of eosinophils from extravascular sites to the blood. This fact could explain why we found increased eosinophils first in blood and later in other compartments. The early IL-5 release may also serve as a signal for eosinophil differentiation and maturation in bone marrow. The time <br><br> inteeeal observed between the first peak of IL-5 release and the increase of eosinophils in blood coincides with that reported to be necessary for eosinophil differentiation and maturation in vitro. 12Increased eosinophil production and liberation into blood and other tissues occurs Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 29 <br><br> L. H. Faccioli et al. <br><br> FIG. 6. (A) photomicrographs of lung parenchyma from <span class="yellow">guinea-pigs</span> infected for 18 days with <span class="yellow">T. cani</span>,, (B) infected animals which were treated with irrelevant antibody at the time of infection; (C) infected animals which were treated with the mAb TRFK-5 at 3 days after infection; (D) mAb administration 17 days after infection. The animals were sacrificed 18 days after infection. Note the intense eosinophil infiltration into the lung in A and B, the inhibition of eosinophils in C and the reduction of eosinophils in D. <br><br> thereafter. Thus, early and later IL-5 release provides a necessary level of this cytokine, which is involved in the maintenance of eosinophilia. We may assume that the inhibition of the first peak of IL-5 release by TRFK-5 does not permit the subsequent T cell stimulation and differentiation. This may explain the long-lasting effect of TRFK5 treatment observed here and also reported by others. 8 In agreement with our results, there is an important observation of Svetic et al. 24 showing that a specific and highly reproducible IL-5 gene expression pattern is detectable in Peyer's patches by 6 to 12h after <span class="yellow">Heligmosomoides polygyrus</span> infection. The early increase in IL-5 gene expression after infection was probably T cell-independent, inasmuch as it was obseeeed in Peyer's patches of congenitally athymic <span class="yellow">mice</span> and of conventional <span class="yellow">mice</span> treated with anti-CD4 30 Mediators of Inflammation Vol 5 <br><br> and anti-CD8 mAb. Moreover, Kusama eta/. 25 have observed two peaks of eosinophilia in normal and athymic <span class="yellow">mice</span>, and suggested that IL-5 observed in the first peak was produced by cells other than CD4 T cells, since anti-CD4 and anti-CD3 mAb reduced only the second peak of eosinophilia in normal <span class="yellow">mice</span> and slightly reduced the first peak of eosinophilia in both normal and nu/nu <span class="yellow">mice</span>. The local lung cells producing IL-5 may also help us to explain the reason why 12, 17 or 18 days post-infection TRFK-5 treatment only partially inhibits, or does not inhibit eosinophil infiltration into the lungs, as demonstrated in Figs 5 and 6. We may suggest that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK<br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> 5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens have already stimulated some cell populations to produce IL-5, as demonstrated by Kusama et aL25These results suggest that eosinophilia in lungs is under the control of different factors when compared to that observed in blood and the peritoneal cavity. One of the most important results obtained here was the inhibition of circulating eosinophil numbers by the different mAb treatments, even when the antibody was given at the peak of blood eosinophilia, which was accompanied by an increase of mature eosinophils in bone marrow. This suggests that IL-5, apart from being required for the terminal differentiation of eosinophils in bone marrow, 26 is also likely to drive eosinophils from the bone marrow to the blood and then to the tissues, probably by upregulating VLA-4 expression in eosinophils. Moser et aL have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with cytokines such as IL-5, IL-3 or GM-CSF for expression of adhesion molecules such as VI-4. Recently, Pretolani et al. 27 have indeed shown that an anti-VLA-4 antibody suppresses eosinophil recruitment to lung in the <span class="yellow">guinea-pig</span> and, as a consequence, inhibits the accompanying bronchopulmonary hyperresponsiveness. <br><br> tant in protective immunity to a gastrointestinal nematode infection in <span class="yellow">mice</span>. Proc Natl Acad Sci USA 1991; 88= 5513-5517. 11. Yamaguchi A, Suda T, Suda J, et al. Purified interleukin (IL-5) supports the terminal differentiation and proliferation of <span class="yellow">murine</span> eosinophilic <br><br> precursors. J Exp Med 1988; 16'7: 43-56. 12. Yamaguchi Y, Hayashi Y, Sugama Y, et al. Highly purified <span class="yellow">murine</span> interleukin (IL-5) stimulates eosinophil function and prolongs in vitro survival. IL-5 as an eosinophil chemotactic factor. J Exp Med 1988; 16'7: 17371752. 13. Rothenberg ME, Petersen J, Stevens RL, Silberstein DS, McKenzie DT, Austen KF, Owen WF. IL-5-dependent conversion of normodense <span class="yellow">human</span> eosinophils to the hypodense phenotype uses 3T3 fibroblasts for enhanced viability, accelerated hypodensity, and sustained antibodydependent cytotoxicity. J Immuno11989; 143; 2311-2316. 14. Collins PD, Weg VB, Faccioli LH, Watson ML, Moqbel R, Williams TJ. Eosinophil accumulation induced by <span class="yellow">human</span> interleukin-8 in the <span class="yellow">guinea pig</span> in vivo. Immunology 1993; '7}; 312-318. 15. Sehmi R, Wardlavo AJ, Cromwell O, Kurihara K, Waltmann P, Kay AB. Interleukin-5 selectively enhances the chemotactic response of eosinophils obtained from normal but not eosinophilic subjects. Blood 1992; '79; 2952-2959. 16. Moser R, Fehr J, Bruijnzeel PLB. IL-4 controls the selective endotheliumdriven transmigration of eosinophils from allergic individuals. J Immunol 1992; 149: 1432-1438. 17. Olson LJ, Schulz CW. Nematode induced hypersensitivity reactions in <span class="yellow">guinea pigs</span>: onset of eosinophilia and positive Schultz-Dale reactions following graded infection with <span class="yellow">Toxocara canis</span>. Ann N Y Acad Sci 1963; 113; 440-455. 18. Kayes SG, Oaks <br><br> JA. Development <br><br> of the granulomatous response in <br><br> <span class="yellow">murine</span> toxocariasis. I. Initial events. Am J Patho11978; ,}3; <br><br> 277-294. <br><br> 19. Basten A, Beeson PB. Mechanisms of eosinophilia. II. Role of the lymphocyte. J Exp Med 1970; 131; 1288-1305. 20. Plaut M, Pierce JH, Watson CJ, Hanley-Hyde J, Nordan RP, Paul WE. Mast cell lines produce lymphokines in response to cross-linkage of Fc epsilon RI or to calcium ionophoras. Nature 1989; 339: 64-67. is 21. Sher A, Coffman RL, Hieny S, Scott P, Cheever AW. Interleukin required for the blood and tissue eosinophilia but not granuloma formation induced by infection with <span class="yellow">Schistosoma mansoni</span>. Proc Natl Acad Sci USA 1990; 8'7; 61-65. 22. Herndon FJ, Kayes SG. Depletion of eosinophils by anti-IL-5 monoclonal antibody treatment of <span class="yellow">mice</span> infected with <span class="yellow">Trichinella spiralis</span> does not alter parasite burden or immunologic resistance to reinfection. J Immuno11992; 149: 3642-3647. 23. Oshima T. Standardization of techniques for infecting <span class="yellow">mice</span> with Tox24. ocara canis and observations on the normal migration routes of the larvae. J Parasito11961; 4'7= 652. Svetic A, Madden KB, Zhou XD, et al. A primary intestinal helminthic infection rapidly induces a gut-associated elevation of Th2-associated cytokines and IL-3. J Immuno11993; 150: 3434-3441. Kusama Y, Takamoto M, Kasahara T, Takatsu K, Nariuchi H, Sugane K. Mechanisms of eosinophilia in BALB/c-nu/+ and congenitally athymic BALB/c-nu/nu <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Immunology 1995; 84; 461-468. Rennick DM, Thompson-Snipes L, Coffman RL, Seymour BWP, Jackson JD, Hudak S. In vivo administration of antibody to interleukin-5 inhibits increased generation of eosinophils and their progenitors in bone marrow of parasitized <span class="yellow">mice</span>. Blood 1990; '76: 312-316. Petrolani M, Ruffle C, Lapa e Silva JR, Joseph D, Lobb RR, Boris Vargaftig B. Antibody to very late activation antigen 4 prevents antigen-induced bronchial hyperreactivity and cellular infiltration in <span class="yellow">guinea pig</span> airways. J Exp Med 1994; 180= 795-805. <br><br> References 25. 1. Nutman <br><br> 2. <br><br> 3. 4. 5. <br><br> 6. 7. <br><br> 8. <br><br> 9. <br><br> TB, Ottesen EA, Cohen SG. The eosinophil, eosinophilia, and eosinophil-related disorders. Allergy Proc 1989; 10; 47-62. Glickman LT, Schantz PM. Epidemiology and pathogenesis of zoonotic toxocariasis. Epidem Rev 1981; 3= 230-250. Beaver P, Snyder H, Carrera G, Dent J, Lafferty J. Chronic eosinophilia due to visceral larva migrans. Pediatrics 1952; 9: 7-19. Limaye AP, Abrams JS, Silver JE, Ottesen EA, Nutman TB. Regulation of parasite-induced eosinophilia: selectively increased interleukin production in helminth-infected <span class="yellow">patients</span>. J Exp Med 1990; l'7:a; 399-402. Steel C, Nutman TB. Regulation of IL-5 in onchocerciasis: a critical role for IL-2. J Immuno11993; 15{}; 5511-5518. Yamaguchi Y, Matsui T, Kasahara T, et al. In vivo changes of hemapoietic progenitors and the expression of the interleukin gene in eosinophilic <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Exp Hemato11990; 18; 1152-1157. Parsons JC, Coffman RL, Grieve RB. Antibody to interleukin prevents blood and tissue eosinophilia but not liver trapping in <span class="yellow">murine</span> larval toxocariasis. Parasite Immuno11993; 15: 501-508. Coffman RL, Seymour BWP, Hudak S, Jackson J, Rennick D. Antibody to interleukin-5 inhibits helminth-induced eosinophilia in <span class="yellow">mice</span>. Science 1989; 245; 308-310. Sher A, Coffman RL, Hieny S, Cheever AW. Ablation of eosinophil and IgE responses with anti-IL-5 or anti-IL-4 antibodies fails to affect immunity against <span class="yellow">Schistosoma mansoni</span> in the <span class="blue">mouse</span>. J Immuno11990; 145: 3911<br><br> 26. <br><br> 27. <br><br> ACKNOWLEDGEMENTS. This work was funded by grant 300652/85-2 from Conselho Nacional de Desenvolvimento Cientifico e Tecnol6gico (CNPq) and Grant 92/5105-7 from Fundago de Amparo ft Pesquisa do Estado de So Paulo (FAPESP). We wish to thank Mrs M.A. Fernandes for technical assistance, Mrs M.M.O. Rossi for the histological sections, M. Costa Gongalves for the photography artwork, and Dr P. Minoprio, Institut Pasteur, for kindly providing TRFK-5. <br><br> 3916. 10. Urban <br><br> Jr JF, <br><br> Katona IM, Paul WE, Finkelman FD. Interleukin 4 is impor<br><br> Received 12 October 1995; accepted 17 November 1995 <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 31 <br><br>  <h3>pmcA2377325</h3>Purinergic signalling in the subretinal space: a role in the communication between the retina and the RPE
Abstract
The retinal pigment epithelium (RPE) is separated from the photoreceptor outer segments by the subretinal space. While the actual volume of this space is minimal, the communication that occurs across this microenvironment is important to the visual process, and accumulating evidence suggests the purines ATP and adenosine contribute to this communication. P1 and P2 receptors are localized to membranes on both the photoreceptor outer segments and on the apical membrane of the RPE which border subretinal space. ATP is released across the apical membrane of the RPE into this space in response to various triggers including glutamate and chemical ischemia. This ATP is dephosphorylated into adenosine by a series of ectoenzymes on the RPE apical membrane. Regulation of release and ectoenzyme activity in response to light-sensitive signals can alter the balance of purines in subretinal space, and thus coordinate communication across subretinal space with the visual process.<br><br>Introduction
The retinal pigment epithelium (RPE) lies between the outer segments of the photoreceptors and the choroidal blood supply (Fig. 1). The RPE combines the functions of epithelial and glial cells, providing a barrier while also supporting the neural photoreceptors and modulating their function. Tight communication between photoreceptors and the RPE is critical to coordinate the multiple levels of interaction, and the purinergic contribution to this communication is becoming apparent. The relevance of this purinergic input is emphasized by the many functional effects of P1 and P2 receptor stimulation and by the multiple mechanisms in place to regulate subretinal levels of purine agonists. As the dynamics of ATP release and extracellular conversion into adenosine will modify agonist availability, the modulation of these processes can exert a temporal control on purinergic signaling. The following review will outline the main interactions between the RPE and photoreceptors, describe the effects of stimulating purinergic receptors on both sides of subretinal space, and summarize how levels of ATP, ADP, and adenosine are manipulated in this microenvironment.
Fig. 1Schematic illustration of the key components of purinergic signaling in the subretinal microenvironment. Stimulation of P2 receptors on the RPE can enhance transepithelial fluid absorption while P1 receptors can modulate phagocytosis. ATP released through CFTR and other Cl− channels can stimulate P2 receptors or be converted to ADP, AMP, and adenosine (Ado) by a series of ectonucleotidases present on the apical membrane of the RPE. By controlling the balance of extracellular purines available to stimulate these receptors these mechanisms can control levels of endogenous purines available to activate the receptors. While theoretically possible, it remains to be determined whether these subretinal purines can actually stimulate photoreceptors
Purines and subretinal space
RPE-photoreceptor interactions across the subretinal space
The outer segments of the rods and cones are responsible for the initial stages of vision, converting photon energy into a series of enzymatic reactions that close the light-sensitive channels on the photoreceptor plasma membrane, hyperpolarize the cells, and reduce the release of glutamate from the synaptic terminals [1, 2]. Efficient photoreceptor function depends upon both short-term and long-term support from the RPE. The critical nature of these interactions is evident from the rapid degeneration of photoreceptors in the absence of a healthy RPE layer and by the RPE localization of defective gene product in some forms of hereditary photoreceptor degeneration [3].
The apical membrane of the RPE is separated from the plasma membrane of the outer segments by an extracellular space of only 10–20 nm [4]. Although small, this subretinal space contains a highly structured matrix which ensheathes the outer segments and extends to the RPE [5, 6]. The presence of enzymes within this interphotoreceptor matrix emphasizes that this extracellular space itself is functionally active [7, 8]. This intimate anatomical relationship between photoreceptors and the RPE reflects multiple functional interactions. For example, the RPE accepts, recycles, and exports central components of the phototransduction process [9]. The outer segments are continuously resynthesized, and the phagocytosis, degradation, and processing of shed outer segment tips by the RPE cells is central to this renewal [10]. The ion channels and transporters on the apical membrane of the RPE help regulate the ionic composition of the subretinal space [11]. As extracellular levels of ions can modify the ionic driving forces across the photoreceptor plasma membrane, these RPE transporters can influence the state of neural activity. The transport of fluid and ions from the apical membrane to basolateral membrane of the RPE is also one of the main forces keeping the retina attached [12].
The control of photoreceptor activity by light gives a rapid temporal dependence to some interactions between the photoreceptors and the RPE. The volume of subretinal space increases in response to light [13], linking small changes in the ionic composition of the subretinal space with activity of the RPE transport mechanisms which maintain this volume [14, 15]. Other processes are controlled on a diurnal cycle. The shed tips of the outer segments are phagocytosed by the RPE soon after the onset of light [16, 17]. These processes can both be modulated by purine levels in subretinal space, indicating purinergic regulation is important over multiple time scales.<br><br>Purinergic receptors on photoreceptors
A2 adenosine receptors were localized to both the inner and outer segments of photoreceptor outer segments over a decade ago by Blazynski and colleagues [18], with more recent reports emphasizing their functional role. A2 agonists inhibit the L-type Ca2+ channel on rod outer segments [19] and can inhibit the synaptic release of glutamate from rods, suggesting changes in adenosine levels in subretinal space could modulate light sensitivity [20]. The A2 agonist DPMA and the adenosine deaminase inhibitor EHNA reduce mRNA for opsin in rods, suggesting that endogenous levels of adenosine can downregulate opsin message at night [21]. EHNA and the A2A receptor agonist CGS21680 also increase the survival of <span class="yellow">chick</span> embryonic photoreceptors in culture [22], indicating a long-term neuroprotective role for adenosine.
P2 receptors are also present in the photoreceptors. mRNA for the P2X2 receptor is expressed in the photoreceptor cell bodies, with immunohistochemical localization of the protein to outer segments [23]. In situ hybridization indicates the photoreceptor layer has the highest level of P2Y2 receptor of any region in the <span class="yellow">rabbit</span> retina, although staining was not pronounced in monkey [24]. P2X7 receptors have recently been localized to photoreceptor synaptic terminals, with evidence for ecto-ATPase activity in the synapse, and functional evidence suggesting ATP augments transmission of the light response by rods [25]. It was suggested that ATP might be co-released from photoreceptors with glutamate, although this remains to be tested directly.<br><br>Purinergic receptors on the RPE
Stimulation of P1 receptors can have a considerable impact on RPE cells. A2 receptors have been recognized on cultured and fresh RPE cells for some time [26, 27], with in situ hybridization confirming the presence of A2A receptors in <span class="yellow">rat</span> RPE [28]. Stimulation of A2 receptors reduces the rate of rod outer segment phagocytosis by RPE cells [29], while application of adenosine to the apical membrane of <span class="yellow">chick</span> RPE cells increases the basolateral Cl- conductance, the transepithelial potential, and the c-wave, and decreases the hyperpolarization of the basal membrane in response to light [30]. Although adenosine alone does not increase intracellular Ca2+ levels [31], adenosine acts synergistically with ATP to elevate Ca2+ levels in <span class="yellow">human</span> RPE cells by stimulating both A1 and A2A receptors [32, 33]. Stimulation of A1 receptors with high doses of NECA increases the active transport of fluorescein across the RPE, while activation of A2A receptors decreases this transport, and by extension, transport of the ions that underlie fluid movement [34]. Stimulation of A1 and A2A receptors produces analogous increases and decreases, respectively, in the absorption of subretinal fluid blebs. This is consistent with the negative coupling of the A1 receptor and the positive coupling of the A2 receptors to adenylate cyclase, as increasing cAMP inhibits the transport of fluid across the RPE towards the choroid [35–37]. The agonist 2-Cl adenosine reverses the deficit in phosphoinositide metabolism found in diabetic RPE cells [38], suggesting effects on metabolism in addition to transport and phagocytosis.
Multiple P2 receptors have been localized to the RPE. The P2Y2 receptor was initially characterized in cultured <span class="yellow">human</span> RPE [31], with subsequent reports localizing transcript for P2Y1, P2Y2, P2Y4, and P2Y6 in the <span class="yellow">rat</span> RPE/choroid [39] and for P2Y1 and P2Y12 receptors in ARPE-19 cells [40], and functionally identifying a P2X receptor in <span class="yellow">rat</span> RPE cells [41]. ATP, ADP, and UTP induce numerous effects on RPE physiology [32, 33, 42, 43]. While these effects likely involve multiple different receptor types, the contributions of the P2Y2 receptor have been explored in most detail to date. The P2Y2 receptor has been specifically localized to the apical membrane of fresh <span class="yellow">bovine</span> RPE cells, and addition of ATP to this membrane transiently elevates Ca2+, activates a basolateral Cl- conductance, inhibits an apical K+ conductance, and increases the apical to basolateral flow of fluid [43]. This increased absorption of fluid from the subretinal space suggests P2Y2 receptor stimulation could reduce the excess fluid that accumulates in retinal edemas, and several reports have supported this theory. ATP, UTP, and the P2Y2 receptor agonist INS37217 decrease the size of subretinal fluid blebs when injected into subretinal space of <span class="yellow">rats</span> [44]. In both normal and rds +/- <span class="yellow">mice</span> with experimentally induced detachment, INS31217 improves the ERG recovery and decreased cell death [45]. INS37217 also reduces subretinal blebs in <span class="yellow">rabbits</span> [46]. Injection of another P2Y2 agonist, INS542, increases the active transport of fluorescein across the RPE, consistent with this upregulation of ion and fluid transport across the tissue [47]. Together these experiments emphasize the clinical potential of treating retinal edema with P2Y2 agonists.<br><br>
Regulation of purine levels in subretinal space
While synthesized purinergic agonists may prove useful in treating some ocular disorders, the endogenous activation of P1 and P2 receptors in the subretinal microenvironment will ultimately be determined by availability of agonists. These levels are largely controlled by the release of ATP into the subretinal space, its conversion into other purines including adenosine, and the manipulation of adenosine by enzymes or transporters. Recent work has increased our understanding of both the stimuli that initiate changes in subretinal purine levels and the mechanisms that mediate these changes.
Release of ATP by the RPE
At least some of the ATP capable of stimulating the purinergic receptors on RPE cells is released from the RPE itself. The resulting autocrine stimulation ensures local delivery, and control, of purines to initiate the physiologic changes in the RPE. The release of ATP by RPE cells is triggered by multiple stimuli including osmotic stress, bFGF, UTP, NMDA, glutamate, and ATP [39, 40, 48–51]. The ATP release following activation of NMDA receptors by glutamate may have the most interesting implications for communication across subretinal space, given that glutamate confers the light signal from photoreceptors to the rest of the visual system. Glutamate and the specific receptor agonist NMDA triggers ATP release from ARPE-19 cells, with the release inhibited by NMDA antagonist MK-801, and by DCKA, which inhibits the glycine B binding site on NMDA receptors [51, 52]. Although NMDA raises intracellular Ca2+ levels, this increase is prevented by eliminating ATP with apyrase, indicating autostimulation through released ATP is responsible for this Ca2+ signal. NMDA also triggers a release of ATP when applied to the intact <span class="yellow">bovine</span> RPE eyecup [51]. The NMDA receptors and the ATP release sites have been functionally identified to the apical membrane of the <span class="yellow">bovine</span> RPE, suggesting the neurotransmitter interactions could amplify the signal from any glutamate reaching subretinal space.
The ability of both UTP and ATP to stimulate release of ATP from the RPE supports the theory that the system acts to amplify signals. When applied at greater than 1 μM, ATP triggers a secondary release of ATP peaking 10 min after the initial stimuli [40]. UTP also initiates a release in extracellular ATP with a similar delay [48]. The rise in ATP triggered by UTP is inhibited by the Cl- channel blocker NPPB, and UDP is much less effective at triggering release than UTP; both observations suggest the P2Y2 receptor contributes to the increase in ATP more than diphosphokinase, although influence from the enzyme cannot be ruled out [53].
Recent evidence suggests that ischemia may lead to the release of ATP from RPE cells. Chemical ischemia triggers a substantial ATP release from cardiac myocytes [54], while changes in oxygen levels trigger ATP release in central chemoreceptors [55]. We found that exposure to sodium cyanide led to a rapid release of ATP from ARPE-19 cells (Fig. 2). As hypoxic and/or ischemic challenge may lead to changes in the expression of growth factors in RPE cells during certain ocular disorders such as macular degeneration [56], and as purines can induce expression of VEGF in other cells [57], this ATP release may contribute to growth factor signaling by the diseased RPE.
Fig. 2Chemical ischemia triggers ATP release from ARPE-19 cells. ATP release was measured in the bath directly from cells plated in 96-well plates to which the luciferin- luciferase reaction mixture was added [51]. Left Levels of ATP in the bath after addition of 5 mM NaCN to the cells. Measurement began 1 min after addition of NaCN or control solution to wells. Right Levels of ATP measured at the peak, 3 min after addition of NaCN (n = 12). Levels were normalized to concurrent levels in control (n = 14). Symbols and bars represent mean ± SE, *p < 0.001
The particular conduit for ATP release varies with the stimuli. The release in response to hypotonic challenge is largely dependent upon CFTR, as it was prevented by the specific CFTR inhibitor CFTR172 in addition to the more general blocker glybenclamide [50]. While the precise mechanisms by which CFTR contributes to this release are not yet known, a role for CFTR in ATP release into subretinal space is consistent with the reduction of certain ERG components in cftr -/- <span class="yellow">mice</span> [58] and with the ability of apical ATP to activate conductances associated with these ERG components [43]. The release of ATP is also largely blocked by the vesicular transport inhibitor brefeldin A, suggesting the two processes occurred in series whereby ATP efflux follows the insertion of vesicles containing CFTR into the plasma membrane. Although the Ca2+ chelator BAPTA blocks this ATP release [50], raising Ca2+ alone with ionophore ionomycin does not itself initiate release [48]. This necessary but not sufficient contribution of Ca2+ also supports a role for vesicular insertion.
In contrast to the release following hypotonic challenge, the ATP release in response to NMDA does not involve CFTR [51]. Release is blocked by NPPB, however, suggesting another type of anion channel could serve as a conduit for ATP release. The presence of parallel mechanisms coexisting on the same cell for ATP release triggered by either agonists or by cell swelling has also been reported in astrocytes [59] and may reflect the multiple roles of purinergic signaling within a given tissue. As both stimuli lead to release across the apical membrane into subretinal space, both are expected to influence signaling in the microenvironment.<br><br>Interconversion of purines in subretinal space
The interconversion of nucleotides and nucleosides each capable of stimulating distinct receptors makes the purinergic signaling system of particular interest in a confined region such as the subretinal space. The main enzymes responsible for dephosphorylating extracellular ATP on the RPE cells have been analyzed and a basic understanding of their regulation has begun. This section first describes the enzymes that act on ATP and ADP, followed by enzymes which convert AMP into adenosine.
The dephosphorylation of extracellular ATP by RPE cells involves enzymes from multiple families [40], as found in airway epithelial cells [60]. Degradation of ATP by the apical membrane of the fresh <span class="yellow">bovine</span> eyecup and by ARPE-19 cells is inhibited by ARL67156 or βγmATP. Message for eNPP1, eNPP2, and eNPP3 is present in ARPE-19 cells, consistent with the preference of βγmATP for members of the eNPP family [61]. The cells also express NTPDase2, and NTPDase3, although the intermittent presence of NTPDase1 likely reflects a regulated process [40]. Ecto-alkaline phosphatase has no effect on ATP degradation in RPE cells, in contrast to its considerable contribution in airway epithelium [62]. The putative contribution from diphosphokinases to interconversion of subretinal purines is presently unknown.
Extracellular AMP is rapidly dephosphorylated into adenosine in subretinal space. The production of adenosine from ATP at the apical membrane of the <span class="yellow">bovine</span> RPE eyecup is inhibited by the ecto-5′-nucleotidase inhibitor αβmADP, confirming a role for this enzyme [63]. The enzyme is localized to <span class="yellow">rat</span> RPE and ARPE-19 cells immunohistochemically. Degradation of 5′AMP is highest near the subretinal space of <span class="yellow">rat</span> retina [63], although localization in <span class="blue">mouse</span> indicated larger amounts of ecto-5′-nucleotidase at the tips of adjacent Müller cells [64]. Levamisole does not inhibit the dephosphorylation of 5′AMP by the RPE, consistent with the absence of substantial ecto-alkaline phosphatase in subretinal space.
The presence of light may alter the levels of adenosine in subretinal space. Epinephrine is released at the onset of light [65] and stimulation of the RPE with epinephrine can decrease activity of ecto-5′-nucleotidase [63]. While norepinephrine and phenylephrine lead to similar decreases in enzyme activity, prazosin and corynanthine block the effects of norepinephrine, implicating the α1 epinephrine receptor in the inhibition of ecto-5′-nucleotidase [63]. The kinetics of inhibition are consistent with cleavage of the nucleotidase from its GPI anchor. The phagocytosis of rod outer segments is maximal shortly after light onset [16], and this phagocytosis is inhibited by adenosine [29]. The ability of epinephrine released by the illuminated retina to reduce ecto-5′-nucleotidase activity and consequently adenosine levels may relieve this inhibition and enhance the rate of phagocytosis at light onset.<br><br>
Physiologic effects of subretinal purines on the RPE and photoreceptors
The number of purinergic receptors on both photoreceptor and RPE membranes suggests purines make multiple contributions to the physiology of the outer retina. Our increased understanding of how agonist levels in subretinal space are controlled has begun to indicate how and when this contribution may occur. Future research will involve applying these findings from isolated systems to intact RPE-photoreceptor models, and pursuing the role of defective purinergic regulation in ocular disease. While it is unlikely that ATP released across the apical membrane of the RPE can diffuse to these P2 receptors in the outer plexiform layer given the ecto-ATPase activity in the synaptic clef [25], stimulation of receptors elsewhere on the photoreceptor membrane is possible. It would be interesting to determine whether ATP released from the RPE and converted to adenosine by ecto-nucleotidases can actually modulate the response to light by stimulating the A2A receptors on photoreceptor outer segments. The impact of purinergic signaling on chronic ocular diseases is also of interest, such as the role of ischemia-driven ATP release in VEGF production. While the small size of subretinal space can complicate pharmacologic manipulation within the intact RPE-photoreceptor complex, molecular approaches may provide new insight into how endogenous purines in subretinal space affect the physiology, and pathophysiology, of both RPE and photoreceptors.<br><br><br><br><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="yellow">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="yellow">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="yellow">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="yellow">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="yellow">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="yellow">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="yellow">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="yellow">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="yellow">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="yellow">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="yellow">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="yellow">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="yellow">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="blue">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="yellow">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="yellow">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="yellow">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="yellow">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="yellow">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="yellow">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
<h3>pmcA2430230</h3>Emergence of Delayed Methylmercury Toxicity after Perinatal Exposure in Metallothionein-Null and Wild-Type C57BL <span class="yellow">Mice</span>
Abstract
Background
Although a long latency period of toxicity after exposure to methylmercury (MeHg) is known to exist in <span class="yellow">humans</span>, few animal studies have addressed this issue. Substantiation of delayed MeHg toxicity in animals would affect the risk evaluation of MeHg.<br><br>Objectives
Our goal in this study was to demonstrate the existence of a latency period in a <span class="yellow">rodent</span> model in which the toxicity of perinatal MeHg exposure becomes apparent only later in life. Our study included metallothionein (MT) knockout <span class="yellow">mice</span> because studies have suggested the potential susceptibility of this strain to the neurodevelopmental toxicity of MeHg.<br><br>Methods
Pregnant MT-null and wild-type C57Bl/6J <span class="yellow">mice</span> were exposed to MeHg through their diet containing 5 μg Hg/g during gestation and early lactation. We examined behavioral functions of the offspring using frequently used paradigms, including open field behavior (OPF), passive avoidance (PA), and the Morris water maze (MM), at ages of 12–13 and 52–53 weeks.<br><br>Results
At 12 weeks of age, behavioral effects of MeHg were not detected, except for OPF performance in MeHg-exposed MT-null females. At 52 weeks of age, the MeHg-exposed groups showed poorer performance both in PA and MM, and their OPF activity differed from controls. These effects of MeHg appeared exaggerated in the MT-null strain. The brain Hg concentration had leveled off by 13 weeks of age.<br><br>Conclusions
The results suggest the existence of a long latency period after perinatal exposure to low-level MeHg, in which the behavioral effects emerged long after the leveling-off of brain Hg levels. Hence, the initial toxicologic event responsible for the late effects should have occurred before this leveling-off of brain Hg.<br><br><br><br>Methylmercury (MeHg) poses serious and practical concerns for <span class="yellow">human</span> populations regarding perinatal exposure. Fish, especially large predator (carnivore) fish species, accumulate high concentrations of MeHg through the marine food chain, and exposure of pregnant <span class="yellow">women</span> to MeHg through the consumption of fish has evoked widespread concern due to potential effects on offspring. Two large-scale cohort studies in fish-eating populations of Seychelles and Faroe islanders are being conducted; although the former has not found consistent adverse developmental effects of MeHg (Myers et al. 2007), the latter has reported adverse effects (Debes et al. 2006). Fundamental reasons for this discrepancy have not been completely elucidated, and many questions remain regarding the neurotoxicity of MeHg, despite extensive study.
Among the unanswered questions is whether there is a long latency period for behavioral manifestations after exposure to MeHg (Clarkson and Magos 2006; Landrigan et al. 2005; Rice 1996; Weiss et al. 2005a, 2005b). Typical examples of latent toxicity in <span class="yellow">humans</span>, including both acute and chronic MeHg exposures, have been described in detail elsewhere (Weiss et al. 2005a). Davidson et al. (2006) recently suggested that effects of perinatal exposure to MeHg may emerge 9 years after birth in the Seychelles cohort. Consequently, risk assessments of MeHg exposure could be inaccurate because studies (<span class="yellow">human</span> or animal) usually do not focus on later stages of life and therefore could miss delayed effects. The possibility of delayed toxicity is exemplified by the expanded Barker hypothesis, which posits that the origin of some neurodegenerative diseases such as Parkinson and Alzheimer diseases lies in early exposure to environmental chemicals (Landrigan et al. 2005). Although epidemiologic evidence would be ideal for exploring the possibility of delayed toxicity (and, indeed, data from epidemiologic studies form the basis of current risk assessment for developmental toxicity of MeHg), considering the complex effects of numerous potential confounders and the existence of multiple exposures in <span class="yellow">human</span> populations, animal models would likely make important contributions to this field.
Although numerous animal studies have described the developmental neurotoxicity of MeHg (Watanabe and Satoh, 1996), few have addressed the latency issue. Few studies have evaluated the neurobehavioral effects in rodents longitudinally beyond 6 months after perinatal exposure. Spyker (1975) addressed this issue in her pioneering work, reporting the late development of behavioral toxicity in <span class="yellow">mice</span> prenatally exposed to MeHg; it appeared, however, that the substantial mortality and retarded growth among the exposed <span class="yellow">mice</span> were apparent before weaning, indicating that the doses used (even though some lower dose levels were included) exerted severe toxicity. Using a relatively complex schedule-controlled operant behavior method, <span class="yellow">rats</span> whose parents were exposed to MeHg (0.5 or 6.4 mg/L) from 4 weeks before mating and continuing to postnatal day (PND) 16 were shown to be less sensitive to a change in the reinforcement schedule than were their nonexposed counterparts at 28–32 months of age (Newland et al. 2004). <span class="yellow">Mice</span> that were perinatally exposed to 1 or 3 mg/L MeHg in drinking water did not show significant deviation from controls in behavioral performance (motor performance, memory, and learning) at 5, 15, or 26 months of age, but the lifetime-exposed groups did show a significant deviation (Weiss et al. 2005c). The existence of a latency period (i.e., the absence of effects earlier in life followed by the emergence of effects at a later stage of life) has not been demonstrated in any <span class="yellow">rodent</span> study. In nonhuman primates, delayed emergence of the signs of neurotoxicity was observed several years after the cessation of a 7-year postnatal exposure (Rice 1996).
Metallothionein (MT) protects against the toxicities of a variety of metals. We examined the neurotoxicity and developmental toxicity of metallic Hg in MT I/II-knockout <span class="yellow">mice</span> (Yoshida et al. 2004) and demonstrated the susceptibility of this genetically manipulated strain to the toxicity of metallic Hg. In contrast to metallic Hg, MeHg does not induce MT, and MT would not substantially influence the kinetics of MeHg (Yasutake et al. 1998). Several reports, however, have demonstrated protective effects of MT against MeHg toxicity, which was ascribed to the radical scavenging effect of MT (Yao et al. 2000). We also showed that perinatal exposure to MeHg results in altered metabolism of thyroid hormones in neonates that was more distinct in MT-null strains than their wild-type counterparts (Mori et al. 2006). The vulnerability of the MT-null strain suggests that delayed neurobehavioral toxicity due to MeHg, if it does exist, might be more distinctive in this strain.
By utilizing the MT-null strain, we aimed to answer the following two questions: First, could we generate a model in which the toxicity of MeHg would emerge or at least become exaggerated later in life as opposed to earlier in life (i.e., at 3–6 months, which was the timing for most of the earlier studies that used behavioral evaluations)? Second, would the MT-null strain be affected more than its parent C57BL strain? Answering either of these questions not only could influence the risk evaluation of MeHg, but it could also lead to a better understanding of the mechanism of toxicity for perinatal MeHg exposure. To address these issues, we used three behavioral paradigms, the open field (OPF), passive avoidance (PA), and Morris (water) maze (MM) tests, which are often used in this field and which we used in our previous studies on the effects of Hg vapor (Yoshida et al. 2004, 2006). Performances in the MM and PA are said to be the most sensitive to aging (Gower and Lamberty 1993).
We used a dose of 5 μg MeHg/g in the diet, which resulted in a brain Hg level relevant to <span class="yellow">human</span> risk assessment. We evaluated the behavioral end points twice, once around 3 months of age and the other time around 1 year; the latter time roughly corresponds to the period when many behavioral performances, including OPF (Acevedoa et al. 2006; Carrie et al. 1999; Gower and Lamberty 1993), PA (Gower and Lamberty 1993), and MM (Bach et al. 1999; Carrie et al. 1999), show alterations in this <span class="blue">mouse</span> strain.
Materials and Methods
Animals and MeHg exposure
OLA129/C57BL/6J strain <span class="yellow">mice</span> (wild type) and MT I/II-knockout <span class="yellow">mice</span> (MT-null) of this strain were provided by K.H. Choo of the Murdoch Institute, Parkville, Australia (Michalska and Choo 1993) and were of a mixed genetic background of 129/Ola and C57BL/6 strains. F1 hybrid <span class="yellow">mice</span> were mated with C57BL/6 <span class="yellow">mice</span> for six generations at the National Institute for Environmental Studies (Tsukuba, Japan). At 10 weeks of age, single male and female <span class="yellow">mice</span> were allowed to cohabit; every female <span class="blue">mouse</span> was checked each morning for the presence of a vaginal plug. When a plug was confirmed, the day was designated as day 0 of gestation (GD0).
The diet, NIH-07PLD formula (CLEA Japan, Inc., Tokyo, Japan), contained vitamins and trace elements as follows (per kilogram diet): 3.2 mg CuSO4, 88 mg FeSO4, 149 mg MnSO4, 25 mg ZnCO3, 1.6 mg Ca(IO3)2, 11 mg vitamin B1, 4.7 mg vitamin B2, 1.9 mg vitamin B6, 44 mg vitamin E, in addition to 5 μg MeHg/g. This diet was fed to the pregnant <span class="yellow">mice</span> starting from GD0 through 10 days after delivery (i.e., PND10). Thereafter, we switched <span class="yellow">mice</span> to a diet that did not contain MeHg. We chose GD0 as the beginning of exposure because exposures that started before conception often resulted in fairly high Hg concentrations in fetal/neonatal brains (Kakita et al. 2003), and we chose PND10 to cover the early neonatal period, in which considerable brain growth occurs. In our experimental setting, the neonatal <span class="yellow">mice</span> started to eat from the diet bucket and drink from the water bottle from PND10 onward. Control <span class="yellow">mice</span> were kept on the same diet but without MeHg (< 0.01 μg Hg/g). On PND1, to avoid the confounding effects due to different litter size, we reduced each litter to six pups (three males and three females when possible), and on PND10, two males and two females from each litter were killed for chemical analyses.
The remaining male and female offspring per litter were weaned on PND28 and used for subsequent behavioral analyses (either at 12–13 weeks or 52–53 weeks, depending on the litter) as described below. We measured body weights of the weaned <span class="yellow">mice</span> every 2 weeks. Thus, four experimental groups were used (with or without MeHg exposure for two strains), and each group consisted of 12–13 litters. For half of the litters, the behavioral analyses were conducted at 12–13 weeks of age, and upon completion of the behavioral analyses, the animals were killed under ether anesthesia. We then dissected the organs (brain, liver, and kidneys) for Hg analyses. For the remaining half of the litters, we conducted the behavioral tests at 52–53 weeks of age. The <span class="yellow">mice</span> were treated humanely and with regard to alleviation of suffering according to the National Institute for Environmental Studies’ Guidelines for Animal Welfare and the guidelines of St. Marianna University.<br><br>Behavioral evaluations
The details of each behavioral procedure have been described elsewhere (Yoshida et al. 2006). Brief descriptions follow.
For the OPF test, we used an OPF apparatus (Ohara Co., Ltd., Tokyo, Japan) with a 60 × 60-cm floor surrounded by walls 60 cm high. The experimental room light was turned off, and a dim light of 80 lux was lit during the experiment. We placed a <span class="blue">mouse</span> in the center of the floor and monitored its behavior for 10 min using a CCD camera connected to a computer. The position of the center of gravity was calculated by image-analyzing software, which was used to calculate the total distance traveled by the <span class="blue">mouse</span> as well as the positional preference (either center or peripheral, where peripheral was defined as the area within 10 cm from the wall). We cleaned the OPF apparatus with 70% ethanol between trials.
The apparatus for the PA test (Ohara Co. Ltd.) consisted of a light compartment that illuminated by a 400-lux light and a dark compartment with black opaque walls and lids. The two compartments were separated by a mobile guillotine door. On the first day (training trial), we placed each <span class="blue">mouse</span> in the light compartment facing away from the guillotine door, which was closed. After 30 sec of introduction, the door was opened; when the <span class="blue">mouse</span> entered the dark compartment, a brief electric shock (4 mA for 2 sec) was delivered through the metal grid floor. This would force the <span class="blue">mouse</span> back to the light component. The interval between the opening of the door and the entry to the dark room (in seconds) was recorded and defined as the latency. On the next day, the same procedure was repeated, but without the electric shock (retaining trail). In this PA paradigm, aversive learning was assumed to be established in the training trial, and we used its retention in the retaining trail as the index of learning. Between each individual trial, we cleaned the apparatus with ethanol. The cutoff time of the retention session was 300 sec.
The MM test apparatus (Ohara Co., Ltd.) was a round-shaped water pool with a diameter of 120 cm. A small platform was submerged in the water, which provided a place for <span class="yellow">mice</span> to escape from the water (i.e., an aversive stimulus, water temperature = 23 ± 1°C). The water was made opaque by adding white paint so that the <span class="blue">mouse</span> could not see the submerged platform. In each trial, we released a <span class="blue">mouse</span> into the pool from a determined position along the wall, and the performance of the <span class="blue">mouse</span> was monitored by a CCD camera/image analyzer. The time required to reach the platform was recorded. If a <span class="blue">mouse</span> could not find the platform within 60 sec after release, it was led to the platform and placed on it for 20 sec before being removed. In these cases, a latency of 60 sec was recorded. We conducted the trial once a day up to the fifth day for each <span class="blue">mouse</span>, and the order of each <span class="blue">mouse</span> was counterbalanced across the day. On the sixth day, a transfer test (or probe test), which is a trial without the platform, was conducted; in this procedure, we counted the number of times that the <span class="blue">mouse</span> crossed the position where the platform had been.<br><br>Tissue Hg concentration
The tissue samples were homogenized (10% weight/volume) in distilled water using a Polytron homogenizer (Kinematica GmbH, Littau, Switzerland). We determined Hg levels in the homogenates by the oxygen combustion–gold amalgamation method (Ohkawa et al. 1977) using an atomic absorption Hg detector MD-1 (Nippon Instruments, Co. Ltd., Osaka, Japan). To ensure the accuracy of the measurement, we included reference material from a dogfish (DORM-2; National Research Council of Canada, Ottawa, Ontario, Canada) with a certified value of 4.64 ± 0.26 μg/g in the analyses; the observed values fell within the certified range. The detection limit of the measurement was 0.1 ng Hg.<br><br>Statistics
We analyzed behavioral data for OPF and PA by analysis of variance (ANOVA), taking sex, strain, and MeHg exposure as the factors. All the interactions among these factors were put into the model. When any of the interactions was highly significant, we analyzed the data separately in an appropriate way; for example, if sex × Hg was significant, the data for males and females were separately analyzed for the effects of strain and Hg. Whenever appropriate, ANOVAs were followed by Mann-Whitney U or Student’s t-tests, depending on the nature and distribution of the variables. We analyzed data for the MM by repeated-measures ANOVA, taking the exposure as between-group and trials as within-group variables. The test was performed for each of the four sex and strain combinations separately. The significance level was set at p < 0.05.<br><br>
Results
Body weight
Up to 20 weeks of age, the body weight values of the control and MeHg-exposed groups were not different, regardless of strain or sex. After 28 weeks, except for the wild-type female groups, the MeHg-exposed groups weighed significantly less than the controls (Figure 1).<br><br>OPF
At 12 weeks of age, a three-way ANOVA of the locomotion distance revealed that only strain was a significant factor (p < 0.001), reflecting the longer distance traveled by the MT-null <span class="yellow">mice</span> (Figure 2A). Strain was also a significant factor for the proportion of the central-area locomotion (Figure 3A), and Hg exposure marginally affected this outcome. In MT-null females, the proportion of central-area locomotion was higher in MeHg-exposed <span class="yellow">mice</span> than in the controls; this difference was not observed in any other strain–sex combination. At 52 weeks of age, the strain × Hg interaction was highly significant (p < 0.001) in an ANOVA of locomotion distance (Figure 2B); MeHg exposure was associated with decreased locomotion distance in wild-type <span class="yellow">mice</span> and with increased distance in MT-null <span class="yellow">mice</span>. A strain-wise two-way ANOVA (with sex and Hg as the factors) revealed that only Hg was significant in both strains (p < 0.01). Regarding the proportion of the central-area locomotion, the effects of MeHg appeared to depend on sex [i.e., sex × Hg was highly significant (p < 0.001) in the three-way ANOVA; Figure 3B]. Indeed, a sex-wise two-way ANOVA showed significant effects of Hg only in females (p < 0.001).<br><br>PA
At 12 weeks of age, all the groups showed prolonged latency in the second (retention) trial, and no consistent effect of MeHg was recognized regardless of strain or sex (Figure 4A).
At 52 weeks of age, a three-way ANOVA revealed a significant interaction between strain and Hg (p < 0.05); strain-wise two-way ANOVAs revealed a significant effect of Hg on learning in MT-null <span class="yellow">mice</span> of both sexes; these groups of <span class="yellow">mice</span> showed significantly shorter (less than half) latency times compared to control <span class="yellow">mice</span> (Figure 4B). A notable difference between the results at 52 weeks of age and those at 12 weeks was that many of the tested <span class="yellow">mice</span> exceeded the cutoff time in the retention trials at 52 weeks, except for the MT-null groups.<br><br>MM
At 13 weeks of age, repeated-measures ANOVA did not indicate any effects of MeHg (Figure 5A). At 52 weeks of age (Figure 5B), wild-type males and MT-null females shared the same statistical results; Hg as well as the Hg × trial interaction were statistically significant. Thus, in both cases, the MeHg groups showed a longer latency, hampering learning performance.<br><br>Tissue Hg concentration
At PND10, which was immediately after the exposure, brain Hg concentrations of the neonatal <span class="yellow">mice</span> were approximately ≤ 0.5 μg/g (Table 1). Although the MT-null <span class="yellow">mice</span> and females showed slightly higher brain Hg concentrations than the corresponding wild-type group and males, respectively, neither of these differences was significant. At 13 weeks, when the behavioral tests were completed, the brain Hg concentration was comparable to the control (nonexposed) level (approximately 5 ng/g in both the exposed and control brains). Interestingly, MT-null <span class="yellow">mice</span> had a significantly lower brain Hg concentration than the corresponding wild-type groups.<br><br>
Discussion
Results of the present study demonstrate the delayed emergence of neurobehavioral toxicity due to perinatal MeHg exposure, which presumably developed after brain MeHg concentrations had leveled off. This emergent toxicity was exaggerated in MT-null <span class="yellow">mice</span> and was more distinct in females. To our knowledge, our findings show the first clear-cut demonstrations of a long latency period of MeHg neurobehavioral toxicity in rodents and possible genetic susceptibility for the emergent toxicity.
The exposure level should be considered before discussing the end points. On PND10, immediately after the cessation of MeHg exposure, the brain Hg concentration was approximately 0.5 μg/g, regardless of the strain or sex. In a previous study, the brain Hg concentration in <span class="yellow">mice</span> perinatally exposed to 6 mg MeHg/L (via water) peaked between PND0 and PND4 and was approximately three times higher than on PND21 (Goulet et al. 2003). Therefore, the peak brain Hg concentration, which is presumably observed around birth, can be estimated as about 3-fold higher than that on PND10 and would be approximately 1.5 μg/g (0.5 μg/g × 3), which is one of the lowest levels among recent <span class="yellow">rodent</span> studies. As shown by Sakamoto et al. (2002) in their Figure 2, prenatal exposure of <span class="yellow">rats</span> to MeHg showed a peak brain Hg concentration around PND1 that was four to five times greater than the level on PND10. Also, Newland and Rasmussen (2000) reported a slight alteration of a complex operant behavior in <span class="yellow">rats</span> at ages < 2 years at brain Hg concentrations as low as 0.5 μg/g at birth, although statistical significance of this particular effect was not clear. It should be noted that <span class="yellow">rats</span> have different Hg kinetics (Hirayama and Yasutake 1986; Yasutake and Hirayama 1990) due to the high affinity of <span class="yellow">rat</span> hemoglobin for MeHg (Clarkson and Magos 2006).
The most important observation of the present study was that the effects of low-level MeHg exposure were detected only at later stages in the lives of the <span class="yellow">mice</span>. Except for the central-area occupancy in OPF in MT-null females, no statistically significant effects of MeHg were observed in any of the three behavioral tests at 12 weeks of age. In contrast, significant effects were observed in all three tests at 52 weeks of age. The brain Hg concentration of the exposed groups had leveled off and was not distinguishable from the non-exposed group at 13 weeks of age, immediately after the first phase of the behavioral testing. Therefore, in the present study, there was a latency period in which the dose and effects could not be detected, although effects were observed 9 months later. Another notable observation was that the emergent manifestation of toxicity was also recognized in the suppression of body weight (except for wild-type females), which only became apparent on or after 28 weeks of age.
The existence of a latency period of as long as several years after chronic (7 years from birth), low-level exposure to MeHg has been described in nonhuman primates (Rice 1996). In that case, however, the Hg concentration in the brain remained elevated, presumably as a result of the long exposure. Indeed, Rice (1996) argued that the minute amount of residual brain Hg could have caused the delayed toxicity. This was clearly not the case in the present study because the Hg concentration leveled off around the time of the first phase of the behavioral study. The absence of behavioral effects at 12 weeks of age ruled out the possibility that the residual behavioral effects were due to elevated Hg early in life. Therefore, the behavioral toxicity that surfaced at 52 weeks of age must have had its origin before the brain Hg concentration leveled off (at or before 13 weeks of age), although the redistribution of Hg to the brain from other sites of deposition, such as the liver, cannot be completely excluded. The long silent period before the manifestations of toxicity emerged suggests that a slow process plays a role in this latent toxicity.
Although an example of a slow process is aging, 52 weeks of age might not be sufficiently old for a <span class="blue">mouse</span> to be considered aged in the physiologic sense because C57BL/6 <span class="yellow">mice</span> have a relatively long life span among <span class="blue">mouse</span> strains [median survival of 27–31 months (Gower and Lamberty 1993)] and a survival rate at 18–19 months as high as 90% (Institute on Aging HP) (National Institute on Aging 2008). Nevertheless, various behavioral examinations have shown age-related changes in the performance of <span class="yellow">mice</span> at approximately 1 year of age in the OPF (Acevedoa et al. 2006; Carrie et al. 1999), MM (Bach et al. 1999; Carrie et al. 1999), and PA (Gower and Lamberty 1993) tests. The observed effects of MeHg, including the deterioration in the PA and MM and suppression in the OPF (in wild-type <span class="yellow">mice</span>), were consistent with these reported effects of aging on behavioral function (in the sense described above), except for the increased OPF activity in the MT-null <span class="yellow">mice</span>.
Regardless of its neural basis, the basis of neurobehavioral toxicity should be sought in early life stages when the brain Hg concentration is highly elevated (approximately 1.5 μg/g at its peak). Some in vivo experiments have demonstrated several candidate mechanisms of perinatal exposure to MeHg, including abnormal migration of neurons and/or glias (Kakita et al. 2003; Rodier et al. 1984), but at higher Hg concentrations. Using exactly the same exposure protocol as the present study, we found suppressed activity of type III iodo-thyronine deiodinase, a thyroid hormone-metabolizing enzyme, in the brains of PND10 <span class="blue">mouse</span> neonates (Mori et al. 2006), consistent with our previous study of higher MeHg doses (Watanabe et al. 1999). This perturbation could be one of the candidate mechanisms responsible for the later anomalous behaviors because even a transient change in thyroid hormones during the critical period of perinatal life exerts long-term consequences (Auso et al. 2004).
The effects of MeHg at 52 weeks of age were influenced by two potential modifying factors, sex and strain. In the OPF, while the locomotion was affected in both strains (although the direction was opposite), center occupancy was significantly increased only in the MT-null <span class="yellow">mice</span>. In addition, the effects on PA were significant only in MT-null <span class="yellow">mice</span>, whereas MeHg at a higher dose was reported to worsen PA performance in <span class="yellow">rats</span> (6–8 weeks of age; Sakamoto et al. 2002). In addition, body weight gain was suppressed in both male and female MT-null <span class="yellow">mice</span>, whereas in wild-type <span class="yellow">mice</span> the suppression was observed only in males. Taken together, the MT-null strain appeared to be slightly more susceptible to the late-emergent effects of MeHg. Several lines of evidence have shown that MT-I,II is protective against the toxicity of MeHg (Leiva-Presa et al. 2004; Yao et al. 2000), and the present results were basically consistent with these reports. We have also reported the susceptibility of the MT-null strain to the neurotoxic effects of metallic Hg (Yoshida et al. 2004, 2006).
The difference in the susceptibility to MeHg between sexes is still debated (Clarkson and Magos 2006; National Research Council 2000; Vahter 2007). In the present study, some responses to MeHg were different between the sexes, including OPF performance at 12 and 52 weeks of age and MM performance at 52 weeks. The fact that the MT-null female group was the only group affected by MeHg at 12 weeks may suggest the particular susceptibility of females in this strain. This point needs to be clarified in further experiments.
The question remains of why MT-null <span class="yellow">mice</span> are susceptible to the delayed neurotoxicity of perinatal MeHg. Apparently kinetics play only a minor role because the strain did not show distinct effects on the brain Hg concentration at PND10. The significantly lower brain Hg concentration in MT-null <span class="yellow">mice</span> compared with corresponding wild-type <span class="yellow">mice</span> at 13 weeks of age indicated that MT-I,II might play a significant role in the retention of Hg (or MeHg). This is consistent with the results of studies of metallic Hg exposure (Yoshida et al. 2004, 2005); a lower brain Hg concentration may not guarantee lower toxicity, supporting the protective role of the protein. Earlier studies suggest that brain MT-I,II has an important role both in the response to oxidative injury (Potter et al. 2007) and in the process of aging (Kojima et al. 1999). Therefore, the lack of MT can exaggerate the toxicity of MeHg by enhancing the initial effects due to oxygen radicals and/or by accelerating functional aging. Apart from this, an intriguing possibility is that the brain-specific isoform, MT-III, might contribute to the results we obtained because the expression of MT-III, together with MT-I, is increased in the brain of old <span class="yellow">rats</span>, resulting in the low availability of free zinc for synapses (Mocchegiani et al. 2004). The age-dependent expressions of MT isoforms might be modified in MT-null <span class="yellow">mice</span>.
Results of the present study might allow the possibility of alternative interpretations due to some potentially confounding factors. For example, except for the wild-type females, we observed significant differences in body weight between MeHg-exposed and non-exposed groups. Because these differences only became clear later in life, they might be associated with the toxicity that also emerged later in life. Manipulation of body weight in rodents alters activity levels, although the reported results are not always consistent with each other (Harrison and Archer 1987; Samuelsson et al. 2008). Also, the differential performance in PA could be related with the potential effects of MeHg on (electric) shock sensitivity, which we did not examine. At least one high-dose study with adult <span class="yellow">rats</span> showed reduced electric sensitivity due to mercury exposure (Wu et al. 1985). These possibilities need to be addressed in future studies.
To summarize, the present results suggest that an initial (or triggering) toxicologic event occurs before the brain Hg concentration stabilizes and that the nature of this event should be either an acceleration of the aging process or interaction with the aging process. Thus, by identifying the physiologic events associated with the functional aging of the examined behavioral tasks, the fundamental toxicologic scar might be revealed.<br><br>
<h3>pmcA2652712</h3>Neurogenin2 Directs Granule Neuroblast Production and Amplification while NeuroD1 Specifies Neuronal Fate during Hippocampal Neurogenesis
Abstract
The specification and differentiation of dentate gyrus granule neurons in the hippocampus require temporally and spatially coordinated actions of both intrinsic and extrinsic molecules. The basic helix-loop-helix transcription factor Neurogenin2 (Ngn2) and NeuroD1 are key regulators in these processes. Based on existing classification, we analyzed the molecular events occurring during hippocampal neurogenesis, primarily focusing on juvenile animals. We found that Ngn2 is transiently expressed by late type-2a amplifying progenitors. The Ngn2 progenies mature into hippocampal granule neurons. Interestingly, the loss of Ngn2 at early stages of development leads to a robust reduction in neurogenesis, but does not disturb granule neuron maturation per se. We found that the role of Ngn2 is to maintain progenitors in an undifferentiated state, allowing them to amplify prior to their maturation into granule neurons upon NeuroD1 induction. When we overexpressed Ngn2 and NeuroD1 in vivo, we found NeuroD1 to exhibit a more pronounced neuron-inductive effect, leading to granule neuron commitment, than that displayed by Ngn2. Finally, we observed that all markers expressed during the transcriptional control of hippocampal neurogenesis in rodents are also present in the <span class="yellow">human</span> hippocampus. Taken together, we demonstrate a critical role of for Ngn2 and NeuroD1 in controlling neuronal commitment and hippocampal granule neuroblast formation, both during embryonic development and in post-natal hippocampal granule neurogenesis.<br><br>Introduction
Neurons are born not only during development of the central nervous system, but neurogenesis also continues into adulthood. In both <span class="yellow">rodent</span> and <span class="yellow">human</span> adult brain, neurogenesis is active in two distinct zones of the forebrain: the subventricular zone (SVZ) of the lateral ventricle and the subgranular zone (SGZ) of the hippocampal dentate gyrus (DG) [1]–[5]. The molecular mechanism underlying neurogenesis in the DG is not fully understood. Clearly, a cascade of transcriptional events controls the specification of neuronal identity in the DG [3], [6]–[8], but details of the expression pattern and function of each transcription factor remain elusive.
The paired-box homeodomain transcription factor Pax6 and the bHLH transcription factors Ngn2 and NeuroD1 are important when cells acquire a pan-neuronal character and a specific neuronal subtype [9], [10]. In the developing neocortex, Pax6 is expressed in dividing radial glial cells at the ventricular surface [9]. In the adult hippocampus, Pax6 is expressed in astrocytes in the SGZ and are considered to be the true stem cells [9], [11]. Loss- and gain-of-function studies indicate that Pax6 is involved in regulating the proliferation of neocortical and hippocampal progenitors [9], [12]–[16]. During neocorticogenesis, high concentrations of Pax6 induce the expression of the bHLH transcription factor Neurogenin2 (Ngn2) [17]. In turn, Ngn2 causes cell cycle exit [18], an event that takes place when NeuroD1 starts to be expressed [19]. In the developing hippocampus, the absence of Ngn2 leads to a reduced number of granule neurons [20]. NeuroD1, on the other hand, is essential for the differentiation and survival of hippocampal granule neurons [10]. Interestingly, in the absence of Ngn2, NeuroD1 can still be activated and neuronal differentiation can still take place [20]. This has led to the idea that the primary role of Ngn2 is not to direct neuronal differentiation.
Only a small number of studies have addressed the role of Ngn2 and NeuroD1 in hippocampal neurogenesis [10], [20], [21]. Therefore, we now re-examine their roles in hippocampal neurogenesis in detail, using gain- and loss-of-function experiments. We first establish a hierarchy of transcriptional events that occur during neurogenesis in the DG and then define the place that Ngn2, NeuroD1 and other transcription factors have in this cascade. We find that a lack of Ngn2 expression result in a markedly smaller hippocampus and an almost complete absence of the DG. We show that Ngn2 is required for granule neuroblasts production/amplification. Gain-of-function of NeuroD1 during development of the DG results in an efficient generation of granule neuroblasts, an effect that we do not observe when we overexpress Ngn2. Finally, we demonstrate that the same transcription factors and cellular markers seen in <span class="blue">mouse</span> and <span class="yellow">rat</span> tissue are also present in the <span class="yellow">human</span> hippocampus.<br><br>Results
Expression of transcription factors and cellular markers define different phases of postnatal hippocampal granule neuron maturation
We first analyzed in detail the chronology of expression of different transcription factors and cell-specific markers during DG granule neuron formation and correlated our findings to the previously established classification of adult hippocampal neurogenesis [3]. The current classification describes hippocampal stem cells and progenies into three categories (type-1, -2 and -3 cells) depending on the markers they express as they mature. In the adult <span class="blue">mouse</span> DG, only a very small proportion of cells undergo mitosis at any one given time [22]. Consequently, in adult animals it is difficult to analyze which transcription factors are involved in the transition phase from one cell category to the next. Therefore, we studied 2 week-old rodents which have higher numbers of maturing granule neurons in the DG. We performed triple immunohistochemistry for all markers of interest and compared results from 2 week- and 2 month-old <span class="yellow">mice</span> (Figure 1 and figure S1).
We first confirmed that stem cells in the SGZ exhibit characteristics of radial glia [3], [11], [16]. They extend radial processes from the SGZ to the apex of the subgranular layer (SGL) and the basal molecular layer. Moreover, they express the intermediate filament protein nestin, glial fibrillary acidic protein (GFAP) and glial glutamate transporter (GLAST) [16] (Figure 1A and B and figure S1A and S1B). The GFAP- and GLAST+ cells are defined as type-1 cells [3]. They also express the paired-homeodomain transcription factor Pax6 (Figure 1B). These radial glia-like stem cells divide relatively infrequently, and are believed to undergo symmetric division (giving rise to two identical stem cells) under some conditions. They can also divide asymmetrically and give rise to a new stem cell and one neuronal progenitor, which usually is defined as a type-2 cell (Figure 2B) [3].
Type-2 cells are divided into two different populations: type-2a expressing nestin and type-2b co-expressing nestin and doublecortin (Dcx) [23]. Based on a previous report [24], we examined the expression pattern of the T-domain transcription factors Tbr1 and Tbr2. We found that Tbr2 is expressed in type-2a cells, in agreement with a recent report [25]. In the SGZ of 2 week-old <span class="yellow">mice</span>, we found an average of 50–70 Tbr2+ cells per 30 µm-thick section throughout the rostro-caudal axis of the dorsal hippocampus. This number was reduced by 35% in 2 month-old <span class="yellow">mice</span> (Figure 1C with figure S1C and data not shown). As opposed to Tbr1 (Figure 1D and figure S1D), Tbr2+ cells did not co-express Dcx or PSA-NCAM (Figure 1C and figure S1C). We observed that Tbr1-immuoreactivity gradually decreased as neurons matured and started to express NeuN (Figure 1H and figure S1H). Thus, Tbr2 labels type-2 cells, while Tbr1 is expressed by immature granule neurons.
We then examined the molecular phenotype of type-3 cells, a cell type that transiently expresses Calretinin [26]. We hypothesized that hippocampal type-3 cells might also express the bHLH transcription factor NeuroD1, known to partially overlap with Tbr2 and Tbr1 in different brain regions [24]. We found that NeuroD1 expression in the hippocampus starts in Tbr2+ cells and extends to post-mitotic Tbr1+ cells, which also express the hippocampal granule identity transcription factor Prox1 (Figure 1G and 1I and figure S1G and S1I). While Tbr1 expression ceases during granule maturation, that of NeuroD1 is weakly maintained when NeuN expression starts (Figure 1G and 1H and figure S1G and S1H). We found that calretinin expression decreased during granule neuron maturation, before NeuroD1 was reduced (Figure 1F). Thus NeuroD1 is expressed in type-2b and type-3 cells, as well as immature granule neurons. Finally, we found that NeuroD2 starts to be expressed just after NeuroD1 and, unlike NeuroD1, continues to be highly expressed in mature neurons (Figure 1J and data not shown).<br><br>Mash1 and Ngn2 define the early versus late type-2a stage in 2 weeks old <span class="yellow">mice</span>
Having established the hierarchy of transcription factors and cellular markers appearing during hippocampal neurogenesis, we set out to clarify the position of the two bHLH transcription factors Ngn2 and Mash1, in this process. We found that more than 90% of Ngn2-immunoreactive cells are Pax6 positive (Figure 2A), and almost all co-express Tbr2 (Figure 2B). Thus, Ngn2 appears to be expressed by type-2a cells. As our antibodies against Ngn2 and NeuroD1 were made in the same species, we could not determine if there was an overlap between these two proteins. However, by comparing their overlapping expression pattern with that of Tbr2, we propose that some Ngn2+ cells co-express NeuroD1, and that they appear at the onset of NeuroD1 expression and downregulation of Ngn2. We base this assumption on the facts that more than 50% of the Tbr2+ cells are Ngn2+ and more than 50% of Tbr2+ cells also express NeuroD1. The expression of Ngn2 in adult hippocampus was hard to detect using immunohistochemistry due to the low expression of the protein (data not shown). However, it is present, as previously described using reporter <span class="yellow">mice</span> and as revealed by in situ hybridization [21].
We next examined the expression of Mash1 and compared it with that of Ngn2, Pax6 and Tbr2. We found some Mash1+ cells (10–20 cells per section) in the dorsal hippocampus. Almost all of them co-expressed Pax6 (Figure 2C). Only few cells co-expressed Tbr2 (Figure 2D) or Ngn2 (Figure 2E), suggesting that Mash1 is downregulated when these two proteins are expressed. None of the Mash1-expressing cells were positive for NeuroD1 (Figure 2F). Mash1+ cells were still undergoing mitosis (Figure 2C) and consequently over 90% of them co-labeled with the cell-cycle marker Ki67 (Figure S1J).
Taken together, we show that Mash1 and Ngn2 are expressed in early and late stages of maturation of type2a progenitors, respectively, and that both transcription factors are co-expressed briefly when cells transit from early to late phase type2a cells.<br><br>Ngn2 progenies become hippocampal granule neurons
Previous work has shown that in Mash1 null mutant <span class="yellow">mice</span> hippocampal neurogenesis is not reduced and the hippocampus is not malformed [20]. Therefore we focused our initial analysis on the role of Ngn2 in hippocampal neurogenesis. We characterized the fate of Ngn2-expressing cells using Ngn2 Knock-in green fluorescent protein (GFP) <span class="yellow">mice</span> (Ngn2+/GFP) [27]. In the SGZ of 2 week-old hippocampi, we found bright GFP-expressing cells that were positive for Tbr2 (Figure 2H), the immature neuronal marker PSA-NCAM (Figure 2I) and NeuroD1 (Figure 2J). Furthermore, Prox1+, NeuroD2+ and calretinin+ cells were also weakly positive for GFP. We made similar observations in 2 month-old <span class="yellow">mice</span> (data not shown) confirming earlier published results [21]). We did not observe Mash1+/GFP+ and NeuN+/GFP+ cells (Figure 2G). These data from Ngn2+/GFP <span class="yellow">mice</span> are entirely consistent with our earlier observations using immunocytochemistry to label Ngn2-expressing cells and with GFP undergoing slow degradation after the expression of Ngn2 has ceased. Importantly, none of the GFP-expressing cells, including those only weakly fluorescent, residing in the SGL expressed the astrocytic marker GFAP or the oligodendrocytic markers CNPase (data not shown). Taken together, the data show that Ngn2 progenies become neurons and never generate astrocytes or oligodendrocytes.<br><br>Marked reduction of hippocampal granule neurons in absence of Ngn2
To determine the role of Ngn2 during the initiation of hippocampal granule neurogenesis, we analyzed the hippocampus of <span class="yellow">mice</span> lacking Ngn2 (Ngn2GFP/GFP). In contrast to the <span class="yellow">mice</span> use by Galichet and coworkers, our <span class="yellow">mice</span> have a lifespan of only 2 weeks after birth. Therefore, we analyzed 2 week-old and younger <span class="yellow">mice</span>. Consistent with previous observations [20], the hippocampus in Ngn2GFP/GFP mutants is clearly malformed (Figure 3A). In two week-old Ngn2GFP/GFP <span class="yellow">mice</span>, the ventral blade of the DG is completely absent. Already on postnatal day 1–2, the ventral blade of the DG is malformed along its whole rostro-caudal axis (Figure 3B).
As neither migration defects nor cell death cause the reduced number of hippocampal neurons in Ngn2 null <span class="yellow">mice</span> [20], we asked if Ngn2 is necessary for production of hippocampal granule neuroblasts. We first injected 2 day-old <span class="blue">mouse</span> pups with BrdU 2 hours prior to sacrifice, allowing us to evaluate cell proliferation and ongoing neurogenesis. We also injected BrdU into female <span class="yellow">mice</span> on their final day of pregnancy and examined the brains of their pups 48 hours later (corresponding to P1–P2 old pups). This allowed us to assess the number of neurons derived from the last day of intrauterine development. Regardless of whether the <span class="yellow">mice</span> were WT or hetero/homozygous mutant pups, we found cells that had incorporated BrdU in the hippocampal subventricular zone (hSVZ), fimbria and the DG (Figure 4A and B; [28]), suggesting that progenitors originating in the hSVZ divide while migrating towards the DG. Most of them were organized in chains typical of migrating cells (Figure 4A and B and figure S2). In Ngn2GFP/GFP mutant <span class="yellow">mice</span> injected with BrdU 2 hours prior to sacrifice, we observed a decrease in the number of BrdU+ cells compared to heterozygous littermates. Thus, the numbers of newborn cells were reduced in the hSVZ/fimbria (133.3±7.5 for Ngn2+/GFP vs 69.7±5.0 for Ngn2GFP/GFP) and DG (132.0±9.1 for Ngn2+/GFP vs 42.6±2.2 for Ngn2GFP/GFP)(Figure 4C and D). We also examined the number of newborn cells differentiating into neurons, and identified them by the co-expression of BrdU and NeuroD1. As expected, they were relatively few in numbers because the short delay (2 hours) between BrdU administration and sacrifice in this first experimental paradigm. In Ngn2GFP/GFP null mutants, we found the number of cells differentiating into neurons to be reduced by 60% and 71% in the hSVZ/fimbria (8.6±1.4 for Ngn2+/GFP vs 3.4±0.5 for Ngn2GFP/GFP) and DG (9.7±1.1 for Ngn2+/GFP vs 2.8±0.6 for Ngn2GFP/GFP), respectively, (Figure 4E and F). As a result, the number of NeuroD1+ cells in the DG was reduced by 80% (580.3±27.7 for Ngn2+/GFP vs 116.2±7.3 for Ngn2GFP/GFP; Figure 4G).
We obtained similar results in the second paradigm, i.e. in <span class="yellow">mice</span> that we sacrificed 48 hours after BrdU administration and in which a larger number of the newborn cells had time to mature into neurons. In this case, the number of BrdU+/NeuroD1+ cells in the DG was reduced by 73% of Ngn2GFP/GFP mutant animals (91.5±24.1 for Ngn2+/GFP vs 26.4±3.3 for Ngn2GFP/GFP) (Figure 4J). We found no differences in the proportion of BrdU+ cells that expressed NeuroD1 in Ngn2+/GFP and Ngn2GFP/GFP <span class="yellow">mice</span> (14.2±1.0% and 17.4±1.8% respectively; Figure 4K). This shows that the few cells that manage to proliferate in the Ngn2 null <span class="yellow">mice</span> have the same ability to differentiate into neurons as those in <span class="yellow">mice</span> with one Ngn2 allele.
Interestingly, the absence of Ngn2 did not alter the identity of neurons in the DG granule layer in 2 week-old Ngn2 null <span class="yellow">mice</span>. The maturing granule progenitors sequentially expressed Pax6, Tbr2 (Figure S3A and S3B), NeuroD1 and Calretinin (Figure 4L), PSA-NCAM (Figure 4N), Tbr1 (data not shown) and Prox1 (Figure 4M). We confirmed that the same transcriptional cascade is active in the 20% (compared to <span class="yellow">mice</span> with one Ngn2 allele) residual granule neurons that are formed two day-old in Ngn2GFP/GFP <span class="yellow">mice</span> (Figure S3D–G and data not shown).
We next investigated if the cells that failed to develop into neurons in Ngn2GFP/GFP mutant <span class="yellow">mice</span>, became glial cells. We found that cells expressing GFP never co-labeled with the astrocyte marker GFAP (Figure 4N) or the oligodendrocyte marker CNPase (data not shown).
Altogether, our results show that Ngn2 plays an important role during the production/amplification of hippocampal granule neuroblasts, but not the acquisition of the granule neuron identity (Figure S7).<br><br>Ngn2 controls the amplification of granule neuron progenitors
We next monitored the mitogenic activity of Ngn2+ cells in the DG of 2 week-old WT <span class="yellow">mice</span>. We observed that over 50% of Ngn2+ cells in the DG of WT <span class="yellow">mice</span> co-expressed the mitosis marker Ki67 (Figure 5A). Inspired by this finding, we injected BrdU into 2 week-old WT and Ngn2GFP/GFP <span class="yellow">mice</span> and compared cell proliferation in the DG. We observed 92% reduction in number of proliferating cells in the SGZ of Ngn2 null mutant <span class="yellow">mice</span> (6.7±0.5) compared to WT (83.6±1.2) and heterozygotic (data not shown) littermates (Figure 5B–C). This data confirm the importance of Ngn2 during the amplification of granule progenitor.s.
To further explore whether Ngn2+ cells become post-mitotic or still proliferate after Ngn2 is downregulated, we performed immunohistochemistry on sections through the DG of Ngn2+/GFP <span class="yellow">mice</span>. We stained them with the mitosis marker Phospho-histone 3 (PH3) and NeuroD1, which is downstream of Ngn2 in the transcriptional cascade controlling neurogenesis. Thus, in the same sections we could identify whether cells that had initiated Ngn2 expression (GFP labeled), continued to divide (PH3+) or committed to neuronal differentiation (NeuroD1+). We observed some GFP+ cells that co-expressed PH3. They all exhibited morphological characteristics of one of the five mitotic phases: prophase, metaphase, anaphase, telophase and cytokinesis (Figure 5D and data not shown). Unexpectedly, cells that colabeled for GFP, PH3 and NeuroD1 (Figure 5E and figure S4A and S4B) were rare. This indicates that Ngn2+ cells undergo division/amplification and that they mature into post-mitotic neurons upon NeuroD1 expression. In <span class="yellow">mice</span> lacking Ngn2, we found the cells expressing GFP localized to the malformed ventral blade of the DG (Figure 5F and G). All of these cells were Ki67+ and they only very rarely expressed PH3 (Figure 5F and G, and figure S4C) in the absence of Ngn2 most of the cells are arrested in the cell cycle prior to entering the M phase and their mitosis is impaired. These data suggest that the mechanism of action of Ngn2 is conserved from development of the DG to postnatal hippocampal neurogenesis.
Our findings suggest that Ngn2 regulates amplification and cell cycle exit of DG granule progenitors. To examine this hypothesis, we compared the effects of Ngn2 and NeuroD1 on mitotic activity in embryonic cortico-hippocampal neurosphere-derived progenitors, 5 days upon transduction, in vitro. In cultures transduced with Ngn2 retrovirus, we found that 27.2% (±4.1%) of the PH3+ cells had been transduced (Figure I1–J). These cells were immunopositive for MAP2 (Figure 5I1 and I2) and therefore represented dividing neuroblasts. By contrast, in NeuroD1 transduced cultures, only 7.3% of PH3+ cells were GFP+ (Figure 5H1, H2 and J). All of the NeuroD1-transduced cells became MAP2+ (Figure 5H). To confirm this data we pulse-labeled transduced cultures with chlorodeoxyuridine (CldU) for 48 hours, five days after differentiation. In contrast to NeuroD1-transduced cultures, we observed dividing cells transduced with Ngn2 retrovirus that were positive for PH3 and that had incorporated CldU (Figure 5K–L2). As the number of cells PH3+/CldU+/GFP+ we observed was low, we did not quantify this finding.
Collectively, we have shown that Ngn2 is required for granule neuroblast production and amplification and that in the absence of Ngn2 the progenitors arrest in the cell cycle. NeuroD1, on the other hand, induces cell cycle exit and promotes rapid neuronal maturation.<br><br>NeuroD1 directs neuronal differentiation and maturation
Based on our previous observations we proposed that Ngn2 primarily controls amplification of granule neuroblasts and NeuroD1 directs neuronal differentiation. To test this hypothesis, we overexpressed Ngn2 or NeuroD1 in E14.5 cortico-hippocampal neurospheres and compared their effects after 5 days of differentiation of the progenitors. All cortico-hippocampal progenitors expressed Pax6 prior to differentiation (Figure S5). After 5 days, both factors suppressed Pax6 and Sox2 (Figure S5, figure 6C) and induced expression of Tbr1, Map2, NeuroD1 and PSA-NCAM (Figure 6D and E).
Interestingly, all Ngn2-overexpressing cells co-expressed NeuroD1. Based on these findings, we next explored whether Ngn2 overexpression induces NeuroD1 expression, and if NeuroD1 in turn directs neuronal differentiation. We compared the effects of both transcription factors in progenitors that either do or do not normally express them. Thus, we expressed Ngn2 in E14.5 neural progenitors isolated from three different brain regions: cortex/hippocampus, lateral ganglionic eminence (LGE) and ventral mesencephalon (VM). Ngn2 expression in the developing forebrain is normally limited to the neocortex, and it does not appear in LGE tissue [29]. In the developing VM, both Ngn2 and NeuroD1 are expressed, but not when neural progenitors from this region are cultured in vitro [30], [31]. In our experiments, cortico-hippocampal progenitors could differentiate into neurons, although they did not normally express NeuroD1 (Figure 6E). When we overexpressed Ngn2 in VM and LGE progenitors, the cells started to express both NeuroD1 and PSA-NCAM (Figure 6F and G). We then overexpressed NeuroD1 in the same types of cultured progenitors and found that the cells became immunoreactive for PSA-NCAM (Figure 6G and data not shown). Thus, NeuroD1 is sufficient to direct neuronal differentiation in cortico-hippocampal-, LGE- and VM-derived progenitors.
Our results from in vitro cultures and the analysis of the DG of Ngn2 mutant animals collectively show the neuron-inducing effect of NeuroD1 in hippocampal granule cell progenitors [32].<br><br>NeuroD1 directs exclusive neuronal differentiation of hippocampal granule neuron progenitors
As the next step, we tested the effects of NeuroD1 in vivo and compared them with those of Ngn2. We injected retroviruses carrying the gene for either Ngn2 [33] or NeuroD1, and the reporter gene eGFP into the ventricles E15.5 <span class="yellow">rat</span> embryos, in utero (Figure 7A). Three weeks later, when the <span class="yellow">rats</span> were about two weeks old, we examined their hippocampi and analyzed the eGFP+ cells. The <span class="yellow">rats</span> injected with control vector exhibited eGFP+ cells within the hippocampus that were either star-shaped, progenitor/glial-like cells (14.8±3.4%) or neuron-like cells with long neurites (85.3±3.4%) (Figure 7B). In <span class="yellow">rats</span> that we had injected with the vector encoding Ngn2, the eGFP-labeled, the transduced hippocampal cells were composed of 31.9% (±5.7%) progenitor/glia-like cells and 69.4% (±6.1%) neuron-like cells (Figure 7B). The glia-like subpopulation was immunopositive for the astrocytic marker GFAP (Figure 7C–D). In contrast, virtually all of the transduced cells in <span class="yellow">rats</span> injected with the NeuroD1 vector became neuron-like cells (99.9%±0.1; Figure 7B). None of these cells stained for GFAP (Figure 7E and F). They were positioned in the external layers of both the ventral and dorsal blades of the DG (Figure 7B3, E and I and figure S6A).
When we examined neuronal maturation of Ngn2- and NeuroD1-transduced cells, we observed that only a few of those transduced with the NeuroD1 vector expressed the early mature neuronal marker Tbr1 (3.1±1.4%; Figure 7I and J). They were located within the basal layers, near the SGL, of the ventral and dorsal blades of the DG. In contrast, a greater proportion of the cells transduced with Ngn2 still expressed Tbr1 (19.6%±3.3%; Figure 7H and J), indicating that they were less mature than the vast majority of the NeuroD1-transduced cells. Immunohistochemistry for NeuN confirmed these data (Figure 7M and O, and figure S6A). Indeed, when we overexpressed Ngn2 the proportion of hippocampal progenitors that became NeuN+ was no greater than in <span class="yellow">rats</span> transduced with the control virus (Figure 7F).
As a whole, our in utero injection experiments confirm that NeuroD1 has a stronger neuron-inducing effect than Ngn2 when overexpressed in hippocampal progenitors.<br><br>Involvement of Mash1, Ngn2, Tbr and NeuroD proteins during <span class="yellow">human</span> hippocampal neurogenesis
Finally, we examined whether the transcription factors and cellular markers that are expressed in rodents are also expressed in <span class="yellow">human</span> hippocampus. We performed immunohistochemistry on aged <span class="yellow">human</span> hippocampal DG and found GFAP-, Sox2-, Pax6-, Nestin-, Prox1- and NeuN-immunopositive cells. This indicates that radial glia-like stem cells, neural progenitors and mature granule neurons are present in the aged <span class="yellow">human</span> hippocampus (Figure 8A–E). While Prox1 is found mainly in granule neurons, Sox2 and Pax6 are exclusively localized to cells in the SGZ (Figure 8D and E).
As for aged rodents, we did not observe the presence of Mash1, Ngn2, Tbr2, Tbr1 and NeuroD1 proteins in aged <span class="yellow">human</span> hippocampus, using immunohistochemistry. However, we could identify the presence of the transcripts, using RT-PCR (Figure 8F). In addition, we detected mRNA for Prox1, Calbindin1 and Calbindin2/Calretinin (Figure 8F). Interestingly, we also observed Sonic Hedgehog and Wnt-3A transcripts, indicating that they are present in the <span class="yellow">human</span> DG and supporting previous claims that they are important for the regulation of adult hippocampal neurogenesis (Figure 8F; [34], [35]). Finally, we also found mRNA for GLAST, Pax6 and GFAP which suggests that radial glia-like stem cells are present in the adult <span class="yellow">human</span> hippocampus, and may play a role in adult <span class="yellow">human</span> hippocampal neurogenesis (Figure 8F; [8]).
Overall, our data show that different proteins known to play key roles in <span class="yellow">rodent</span> hippocampal granule neurogenesis are present in the adult <span class="yellow">human</span> hippocampus.<br><br>
Discussion
In this study we determine the functions of Ngn2 and NeuroD1 during hippocampal neurogenesis. First, we map the hierarchy of molecular markers of neurogenesis in the DG. Second, we describe that Ngn2 is necessary for granule progenitor production/amplification. Third, we demonstrate that NeuroD1 directs neuronal differentiation of granule progenitors. Finally, we show that different cellular markers expressed during hippocampal neurogenesis in rodents are present in <span class="yellow">human</span>.
Sequential expression of different transcription factors and cellular markers during hippocampal neurogenesis
We clarified the pattern of expression of various markers expressed during postnatal and adult hippocampal granule neurogenesis in detail. We found that the transcription factor Pax6 is initially expressed by both radial glia stem cells (type-1) and early amplifying progenitors (type-2a). The next transcription factor to appear in chronology is Mash1. Mash1 expression characterizes the early stage of hippocampal progenitor amplification (early type-2a). The role of Mash1 during hippocampal granule neurogenesis is still unclear. Mash1 null mutant <span class="yellow">mice</span> do not display any clear malformation of the DG [20]. However, overexpression of Mash1 alone in the DG leads to the generation of oligodendrocytes [36]. Based on these observations, we hypothesize that Mash1-expressing cells are not yet committed towards a granule cell fate. They may still have the potential to generate both neurons and oligodendrocytes, as is the case in the SVZ [37].
In agreement with different developing brain regions, we found that Ngn2 starts to be expressed in Mash1-positive transiently amplifying progenitors, i.e. “late type-2a cells” according to the classification we propose. We observed that Pax6 expression persisted longer than that of Mash1, in Ngn2-expressing cells (late type-2a cells), in juvenile but not adult DG (Fig. 1 and figure S1). Later on, Ngn2 is downregulated, whilst Tbr2 expression persists (type-2b cells). The transition from amplifying progenitor to neuroblast is defined by the expression of NeuroD1. Thus, Ngn2 is expressed at the beginning of the transiently amplifying progenitor phase while NeuroD1 marks the end of that period. This applies both to the juvenile and adult <span class="yellow">rat</span> brain [21], [25]. After NeuroD1 is turned on the progenitors leave the cell cycle, gradually mature, express PSA-NCAM, NeuroD2, Calretinin, Prox1, Tbr1 and, finally, NeuN. NeuroD1 expression persists at low levels in mature neurons (Figure 9; [10]). We found the sequential expression of these markers conserved in juvenile (P2 and 2 weeks) and adult <span class="yellow">rodent</span> brains. Importantly, we observed that the same transcription factors and neuronal markers are also present in the adult <span class="yellow">human</span> hippocampus and arranged spatially in a manner reminiscent with what we saw in rodents. It is possible, however, that some of these markers are expressed for longer or shorter periods at postnatal and adult stages of hippocampal progenitor maturation. Moreover, one can ask to which extent the number of divisions occurring during neuroblasts maturation is conserved at postnatal and adult stages of neurogenesis.<br><br>A new function for Ngn2 in maintenance of a progenitor state for granule neuron production/amplification during embryonic and postnatal hippocampal neurogenesis
In agreement with earlier work, we confirm that Ngn2 is indispensable for hippocampal development and plays a vital role in postnatal hippocampal granule neurogenesis. Ngn2 null mutant animals display a reduced size of the cornu ammonis and a malformed DG (Figure 3 and 4; [20]). We show that the number of newborn neurons (incorporated BrdU) is decreased in absence of Ngn2, but hippocampal granule neuron subtype specification is not affected, post-nataly. Likewise, neurons from other brain regions, e.g. ventral midbrain dopamine neurons, can be generated in the absence of Ngn2 [30], [31]. The reduced numbers of mature neurons in the hippocampus and ventral midbrain of Ngn2 null mutant animals does not appear to be due to cell death or migration defects [20], [31]. Instead it may be due to a defect in the generation and/or amplification of neuronal progenitors. Indeed, in the hippocampus of postnatal animals, we observed that cells lacking Ngn2 arrest in the cell cycle, maintain Pax6 expression and cease to proliferate.
When overexpressed in neural progenitors in vitro, Ngn2 lead to up-regulation of NeuroD1 and caused neuronal differentiation (Figure 6), which was not always the case when we overexpressed the same factor in vivo (Figure 7L, N and O). We also saw that Ngn2 does not always efficiently induce cell cycle exit. Typically some mitotically active eGFP+ cells were present in the DG of <span class="yellow">rats</span> injected with the Ngn2 retrovirus (Figure 7P), and we observed that cultured Ngn2-transduced cells are still capable of dividing. Thus, Ngn2 does not always promote cell cycle exit and neuronal commitment, but depending on the state of the cells, Ngn2 may instead promote alternate cellular fates [38]–[40]. Indeed, an earlier study has shown that Ngn2-overexpressing progenitors generate oligodendrocytes when grafted to the adult spinal cord [40]. We have also seen that Ngn2-overexpressing neural progenitors from the embryonic midbrain form astrocytes when grafted to the striatum (unpublished observations). Probably, Ngn2 cannot influence already committed cells, but rather would control neuroblasts production. One could speculate that Ngn2 oscillates during granule neuron formation, as recently demonstrated during neocorticogenesis [41], [42].<br><br>NeuroD1, but not Ngn2, is obligatory for granule neuron progenitor differentiation
Hippocampal granule progenitors can mature and express Tbr1 and Prox1, both during the development and postnatally, even in the absence of Ngn2. Because neuronal differentiation still occurs in Ngn2 null <span class="yellow">mice</span>, it is clear that compensatory, Ngn2-independent mechanisms induce NeuroD1 expression. One candidate is Ngn1, which is expressed during neocorticogenesis, the specification of olfactory sensory neurons and during embryonic <span class="yellow">rat</span> hippocampal development [43]–[47]. Both Ngn1 and Ngn2 regulate NeuroD1 [19], [48]. The introduction of two mutant forms of Ngn1, a deletion of the basic region of Ngn1 and a substitution of two amino acids in the C-terminal basic region, prevents NeuroD1 expression and neuronal differentiation [48]. The double null Ngn1/Ngn2 mutant displays a more severe phenotype than single gene (Ngn1 or Ngn2) mutant <span class="yellow">mice</span>. For example, the total brain size of the double mutants is much smaller [10], [32]. If the role of Ngn1 during hippocampal neurogenesis is to activate NeuroD1 and that of Ngn2 is to control granule neuroblasts production/amplification, the DG of double Ngn1/Ngn2 knockout <span class="yellow">mice</span> should resemble that of NeuroD1 null mutant <span class="yellow">mice</span>.
We demonstrated that NeuroD1 directs neuronal differentiation both in vitro and in vivo. In progenitors isolated from different embryonic brain regions and cultured in vitro, we found that overexpression of NeuroD1 induced neuronal differentiation. The neurons generated expressed PSA-NCAM, Dcx and MAP2. After in utero retroviral vector-mediated gene delivery, virtually all cells transduced with NeuroD1 became neurons. In contrast, progenitors transduced with Ngn2 or control retroviruses adopted a neuron-like morphology in only 70–85% of cases. Under in vitro cell culture conditions, when we overexpressed Ngn2 in progenitors derived from the LGE we observed robust expression of NeuroD1 and neuronal differentiation. These in vitro results differ from those we obtained when we transduced the embryonic brain with a viral vector expressing Ngn2. In this latter case, Ngn2 did not induce neurons and the transduced cells did not express NeuroD1. Therefore, Ngn2 appears to direct non-neuronal cell type specification in vivo [38]–[40]. Taken together, we have confirmed that NeuroD1 plays a key role in neuronal differentiation in the hippocampus, both during development and in the adult brain.<br><br>Concluding remark
We present a detailed classification of different stages in hippocampal neurogenesis. Our detailed molecular mapping of hippocampal neurogenesis allows for a more accurate analysis of how new factors stimulate neurogenesis at different steps in the development of granule neurons. Thereby we hope to facilitate the development of new agents, which stimulate endogenous progenitors in the treatment of diseases.<br><br>
Materials and Methods
Animal tissue preparation
The creation of the Ngn2 transgenic <span class="yellow">mice</span> was reported elsewhere (ref guillemot). Heterozygote male and female <span class="yellow">mice</span> were crossed to obtain WT, heterozygote (Ngn2+/GFP) and null mutant (Ngn2GFP/GFP) animals. Tails of the Ngn2 offspring were used to obtain DNA for determination of the genotype using a polymerase chain reaction (PCR) assay as previously reported [31]. As null mutant Ngn2GFP/GFP do not survive longer than 2.5–3 weeks after birth, Ngn2GFP/GFP and their littermates were sacrificed at the postnatal ages of two days (P2) or two weeks for this study. Neurogenesis was assessed in two weeks or two months old WT <span class="yellow">mice</span>. Sprague Dawley pregnant <span class="yellow">rats</span> were ordered from B&K Universal Ltd, Sollentuna, Sweden (hppt://www.bku.com). All animals were housed in groups with ad libitum access to food and water at a 12-h light/dark cycle. All experimental procedures conducted in this study had been approved by the Ethical Committee at Lund University.
For immunohistochemical analysis, <span class="yellow">mice</span> (from two weeks old and adult stage) and juvenile <span class="yellow">rats</span> were sacrificed by transcardial perfusion with saline for 5–10 minutes, followed by 4% paraformaldehyde (PFA) for 10 minutes. Brains were kept in PFA overnight at 4°C and subsequently cryopreserved in a 20–30% sucrose/0.1 M phosphate buffer solution until sectioning on a microtome apparatus (30 µm thickness sections, Microm Zeiss). Seven series of coronal sections were cut throughout the brain. Free-floating sections were preserved in antifreeze solution until immunohistochemistry was performed. Heads of postanatal two days old <span class="yellow">mice</span> were decapitated and soaked in PFA 4% for 24 hours, at 4°C and transferred into sucrose solution until sectioning on cryostat apparatus (16 µm thickness; Leica CM3000). Sections were mounted on Superfrost glass slides and stored at −80°C until immunohistochemistry was performed.<br><br>Cloning, subcloning, virus production and titer measurement
The Moloney leukemia-derived retroviral vectors used in this study, pCMMP-IRES2eGFP-WPRE and pCMMP-Ngn2-IRES2eGFP-WPRE were previously described [40], [49]. To generate the construct pCMMP-NeuroD1-IRES2eGFP-WPRE, <span class="blue">mouse</span> NeuroD1 cDNA was amplified from a pCS2+mtNeuroD1 plasmid (kindly provided by Professor Jackie Lee, Denver university, Boulder, USA) by PCR to introduce the restriction sites PmeI in 5′ and XhoI in 3′. Amplification of cDNA was performed as previously described [49]. The construction was verified by enzymatic restriction and by DNA sequencing using BigDye 3.1 (ABI). All infectious particles were produced using the producer cell line 293VSV-G and as previously described [49]. The titer of each retrovirus was measured by flow-cytometry based on eGFP expression, four days following infection of HT1080 cells and ranged from 0.5×109−2.1×109 TU/ml (All details on how to produce infectious particles can be provided upon request).<br><br>Neurosphere generation, transduction and differentiation
Pregnant female <span class="yellow">Sprague-Dawley rats</span> (B & K Universal, Sollentuna, Sweden) were terminally anesthetized by an overdose of sodium pentobarbital (i.p., 60 ng/ml). Embryos at stage embryonic day E14.5 (Plug day as day 0) were collected and cortical-hippocampal neurospheres were generated following dissection of the dorso-posterior part of the cortical tissue, and generated as previously described [49]. In this study, second passage (P2) neurospheres were used to study the effect of the overexpression of Ngn2 and NeuroD1 on neuronal differentiation. Each well, containing an equal starting population of 200,000 cells/ml, corresponding to 15–25 neurospheres, was transduced independently with each retrovirus at a multiplicity of infection (MOI) of 1, in proliferation medium supplemented by protamine-sulfate (4 mg/ml, Sigma). To induce differentiation, the medium was replaced with normal basic differentiation medium two days post-transduction and subsequently changed every other day until fixation.<br><br>Immunocytochemistry, immunohistochemistry and microscopy
The antibodies used in this study are: <span class="yellow">rabbit</span> anti-GFAP (1∶1000; DAKO), <span class="blue">mouse</span> anti-Nestin (1∶100; BD PharMingen), <span class="yellow">guinea pig</span> anti-Glast (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Prox1 (1∶1000; Covance), <span class="yellow">goat</span> anti-Ngn2 (1∶20; Santa Cruz), <span class="yellow">goat</span> anti-NeuroD1 (1∶200; Santa Cruz), <span class="yellow">rabbit</span> anti-Pax6 (1∶150; Covance), <span class="yellow">rabbit</span> anti-Trb2 (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Trb1: (1∶1000; Chemicon), <span class="yellow">goat</span> anti-Dcx: (1∶500; Santa Cruz), <span class="blue">mouse</span> anti-PSA-NCAM: (1∶500; Chemicon), <span class="blue">mouse</span> anti-NeuN (1∶300; Chemicon), <span class="yellow">rabbit</span> anti-Calretinin (1∶500; Swant); <span class="blue">mouse</span> anti-Sox2 (1∶100; R&D systems), <span class="blue">mouse</span> anti-Mash1 (1∶100; BD PharMingen), <span class="yellow">rabbit</span> anti-NeuroD2 (1∶300; ABCAM), <span class="blue">mouse</span> anti-MAP2 (1∶500; Sigma), <span class="yellow">rabbit</span> anti-Ki67 (1∶150; NovaCastra) and <span class="yellow">rabbit</span> anti-phospho-Histone H3. The secondary antibodies (1∶200) Cy2, FITC, Cy3 and Cy5 were from Jackson IR laboratories, Alexa-fluor 488, 568, 595 and 647 from Invitrogen-Molecular Probes. DAPI (1∶1000) was purchased from Sigma.
For immunocytochemistry, cultures were fixed in 4% paraformaldehyde at day five, rinsed with PBS three times prior to pre-incubation with a blocking solution (10% <span class="yellow">donkey</span> serum, 0.25% TritonX100 in PBS) for 1 hour. The remainder of the procedure was performed as previously described [49]. Specimen analyses were performed using a Leica confocal microscope (Leica software, equipped with a GreNe and a HeNe laser, using the following lines of excitation: 488 nm, 594 nm and 647 nm). Samples were analyzed using 20×, 40× and 63× objectives, sometimes zoomed. Figures were composed in CANVAS-X software.<br><br>Bromo-deoxyuridine (BrdU) and Cloro-deoxyuridine (CldU) pulse labeling and immunohisto- and immunocyto-chemistry
To assess cell proliferation and ongoing neurogenesis in vivo, animals were injected with BrdU (100 mg/kg, Sigma), two hours prior to sacrifice (for both P2 and two weeks old Ngn2 <span class="yellow">mice</span>). To assess neurogenesis, BrdU (100 mg/kg) was injected 48 hours prior to sacrifice (for P2 animals, BrdU was injected in pregnant dams half day prior to give birth). Immunohistochemistry was carried on as described above using a <span class="yellow">rat</span> anti-CldU/BrdU primary antibody (1∶200, monoclonal, Immunologicalsdirect, Oxfordshire, UK), with an additional denaturation in 1 M HCl for 30 minutes prior to pre-incubation with serum.
To assess the neuronal-inducing activity of Ngn2 and NeuroD1, transduced cultures were incubated with CldU (20 µM, Sigma) for 2 days, after a period of differentiation of five days. For immunocytochemistry, cultures were fixed with 4% paraformaldehyde at day 7, rinsed with PBS three times, treated with 1 M HCl at 65°C for 5–10 min, pre-incubated and then incubated with a <span class="yellow">rat</span> anti-CldU/BrdU antibody and other primary antibodies. The remainder of the procedure was performed according to the protocol for immunohistochemistry already mentioned.<br><br>In utero surgery
Timed pregnant female <span class="yellow">Sprague Dawley rats</span> with embryos at gestational age E15.5 were anesthetized with halothane. The mother was placed in the lower level of a two-level wooden stage. The abdomen was shaved with an electric razor and then cleaned with 70% alcohol. A 2–3 cm midline laparotomy was performed. Each uterine horn was carefully taken out individually and the number of embryos recorded. One horn was then placed back inside the mother, whilst the other horn was prepared for injection. The embryos were kept moist with constant application of warm saline to prevent dehydration. Approximately 2 µl of viral suspension (1×10∧9TU/ml) was injected into the lateral ventricle of each embryo, except for the embryo closest to the vagina. After the injections, the uterine horns were placed back into the abdomen. The abdominal wall and the overlying skin were then sutured. Care was taken not to damage abdominal muscles so that normal delivery of the pups was possible at term. The entire surgery generally took about 45 minutes to one hour. Each mother was allowed to recover in her cage before being returned to the animal stable. Shredded paper was added to each cage to encourage nesting and special care was taken not to stress the mothers. Following normal delivery, the pups were allowed to develop to adulthood up to two weeks.<br><br>Quantification
For the in vivo experiment, manual cell counting was performed on 18 and 30 µm thick brain sections for two days and two weeks old animals, respectively. The brain of P2 animals was cut into 10 series of coronal sections; the brain of two-weeks old animals was cut into 7–8 series of coronal sections. For each staining, one-two series from three to five different individuals were analyzed per genotype using a confocal microscope with 1, 2 or 3 lines of excitation, in sequential scanning in order to avoid false positives. When two series were analyzed, only one series was counted. The mean number of cells per hippocampus expressing the markers of interest, was calculated for each brain, based on the analysis of 6–8 consecutive dorsal hippocampal sections. The final mean number of cells per section was calculated by adding the mean number from each individual. We expressed the data as the mean number of positive cells±standard error of the mean (SEM). In this study, more than 25 Ngn2 null, >45 Ngn2 heterozygotes, and >60 WT individuals (including <span class="yellow">rats</span> injected with retroviruses) were analyzed, in total. Statistical comparison was performed using one-factor analysis of variance (ANOVA) with transcription factor, cellular marker, genotype or time as variables, followed by post-hoc analysis when significant differences were observed, using Statview 5.0 software (SAS institute Inc.). For cell culture experiments, three independent experiments were performed, each in duplicate. The counting was based on seven randomly chosen different fields of view. For each diagram, the level of significance (p-value) is represented as follows: P<0.05 = *; P<0.001 = **; P<0.0001 = ***. All data are expressed as±standard error of the mean (SEM).<br><br><span class="yellow">Human</span> sample and RT-PCR
The brain from a 64 year-old male was provided by the Harvard Brain Tissue Resource Center. The individual whose brain tissue was being analyzed gave written consent for storage and use of his tissue for research. Five millimeters thick fresh <span class="yellow">human</span> hippocampal tissue sample was snap frozen. The frozen tissue block was cut into 20 series of coronal sections, using a cryostat. Every 10 sections, one section was placed in an Eppendorf tube, on dry ice. Pooled sections were used for RT-PCR. Total RNA was prepared using Trizol and RNAeasy, supplemented with RNAGuard as previously described [49]. The RNA was digested with <span class="yellow">shrimp</span> DNAse before cDNA synthesis, which was performed using a mix of oligo-dT and random hexamer primers and SuperscriptII reverse transcriptase. Advantage2 polymerase mix (Clontech/BRL) was used for PCR, with the following cycling conditions: 10 cycles of 94° C for 30 seconds, 68°C for 2 minutes, 30 cycles of 94°C for 30 seconds, 60°C for 1 minute, 68°C for 1 minute and 30 seconds, one cycle of 68° C for 2 minutes, soak at 16°C. Primers marked * were designed by PrimerBank. hRPS18 (sense) 5′-GCCTTTGCCATCACTGCCATT and hRPS18 (antisense) 5′-GCCAGTGGTCTTGGTGTGCT, hPAX6 (sense) 5′-GCCCTGGAGAAAGAGTTTGAGAGAACCCATT hPAX6 (antisense) 5′-GGGGAAATGAGTCCTGTTGAAGTGGTGC, hTBR1 (sense) 5′-GCGGACACCAATGTGCAAGGAAATCG and hTBR1 (antisense) 5′-CGAGGGGGTCAGGCGGTCCATGTCACAGC, hTBR2 (sense) 5′-GACCTGTGGCAAAGCCGACAATAACATGC and hTBR2 (antisense) 5′-GGGGGTGTCTCTATCCAAGAAGAGCCAAT, hPDHX ( = PROX1) (sense) 5′-GGGACACTACGGTTCCGTTTAAGTCCAGC and hPDHX (antisense) 5′-CTCTCCATCCCAGCTTACATTAACATCTGGCATTTGT, hNGN2 (sense) * 5′-CATCAAGAAGACCCGTAGACTGA and hNGN2 (antisense)* 5′-CAACACTGCCTCGGAGAAG, hMASH1 (sense) 5′-CCTGGATCCGCATGGAAAGCTCTGCCAAGATGGAG and hMASH1 (antisense) 5′-CCTGGATCCCCCCTCAGAACCAGTTGGTGAAGTCGA, hNEUROD1 (sense) 5′-GCTCAGGACCTACTAACAACAAAGGAAATCGAAACATG and hNEUROD1 (antisense) 5′-CAAAGCGTCTGAACGAAGGAGACCAGGT, hGLAST (sense) 5′-CATCAGGGAAGATGGGAATGCGAGCTGTAGTCTATTAT and hGLAST (antisense) 5′-CCACGGGGGCATACCACATTATTACTGCTACC, hNEST (sense) 5′-CAGGAGCGGCTGCGGGCTACTGAAAAGTTCC and hNEST (antisense) 5′-CAGGGCTGAGGGGTGGTGCCAAGGAGG, hGFAP (sense) 5′-CCACGAGGAGGAGGTTCGGGAACTCCAGGAGC and hGFAP (antisense) 5-GGAATGGTGATCCGGTTCTCCTCGCCCTCTAGC, hCALB2 ( = Calretinin) (sense) 5′-GGCTCTGGCATGATGTCAAAGAGTGACAACTTT and hCALB2 (antisense) 5′-GGGCATCCAGCTCATGCTCGTCAATGTAGCC, hCALB1 (sense) 5′-GCGAAAGAAGGCTGGATTGGAGTTATCACC and hCALB1 (antisense) 5′-CCCTCCATCCGACAAAGCCATTATGTTCTTCTTGTATG, hSOX2 (sense) 5′-GGAGAACCCCAAGATGCACAACTCGGAGAT and hSOX2 (antisense) 5′-GAGGAAGAGGTAACCACAGGGGGGCTGGAGC, hWNT3A (sense) 5′-CCGAGGGCATCAAGATTGGCATCCAGGAGTGC and hWNT3A (antisense) 5′-TCGGGTTGCGACCACCAGCATGTCTTCACCTC, hSHH* (sense) 5′-ACTCCGAGCGATTTAAGGAACT and hSHH* (antisense) 5′-CAGACGTGGTGATGTCCACTG.<br><br>
Supporting Information<br><br>
<h3>pmcA320464</h3>Organization and structure of the <span class="blue">mouse</span> interleukin-2 gene.
Abstract
We have cloned a chromosomal DNA segment which covers the entire sequence for the <span class="yellow">murine</span> interleukin-2 gene and analysed the structure of the gene. The coding regions are separated into four blocks by three introns each of which is located similarly to the corresponding <span class="yellow">human</span> gene. The exon sequences can be aligned perfectly with the previously cloned cDNA sequence. Of particular interests is the presence of sequences within the 5'-flanking region which are highly conserved between <span class="blue">mouse</span> and <span class="yellow">man</span>. The conserved region which spans more than 400 base pairs may play a role in the regulation of IL-2 gene expression.Images<br><br><br><br><br><br>
 Volume 12 Number 24 1984                                   Nucleic Acids Research <br><br> Organization and structure of the <span class="blue">mouse</span> interleukin-2 gene <br><br> Akira Fuse*, Takashi Fujita, Hidetaro Yasumitsu, Nobukazu Kashima+, Katsushige Hasegawa and Tadatsugu Taniguchi? <br><br> Department of Biochemistry, Cancer Institute, Japanese Foundation for Cancer Research, Toshimaku, Tokyo 170 and Institute for Molecular and Cellular Biology, Osaka University, Suita-shi, Osaka 565, Japan <br><br> Received 10 October 1984; Revised and Accepted 20 November 1984 <br><br> ABSTRACT <br><br> We have cloned a chromosomal DNA segment which covers the entire sequence for the <span class="yellow">murine</span> interleukin-2 gene and analysed the structure of the gene. The coding regions are separated into four blocks by three introns each of which is located similarly to the corresponding <span class="yellow">human</span> gene.    The exon sequences can be aligned perfectly with the previously cloned cDNA sequence.  Of particular interests is the presence of sequences within the 5'flanking region which are highly conserved between <span class="blue">mouse</span> and <span class="yellow">man</span>. The conserved region which spans more than 400 base pairs may play a role in the regulation of IL-2 gene expression. <br><br> INTRODUCTION <br><br> Interleukin-2 (IL-2) is a lymphokine produced by T cells upon antigenic or mitogenic stimulation and is required for the proliferation of T cells (1,2).      Several other biological activities of IL-2 which appear to be crucial in the immune regulation have also been reported (3, 4. 5. 6. 7.) .        We previously reported isolation and sequence analysis of the cDNA for <span class="yellow">human</span> IL-2 (8), as well as the chromosomal gene (9). More recently, we have isolated a cDNA which encodes <span class="yellow">murine</span> IL-2 (Kashima et al., submitted for publication).  The cDNA contains a unique tandem repeat of CAG sequence which would encode 12 consecutive glutamine residues in the active IL-2 molecule. <br><br> In order to study the structure of the <span class="yellow">murine</span> IL-2 chromosomal gene and its controlling region, we isolated and analysed a A phage clone containing the gene and its flanking sequences. <br><br> MATERIALS AND METHODS <br><br> Southern blotting of total <span class="blue">mouse</span> DNA <br><br> <span class="yellow">Mouse</span> chromosomal DNA was extracted from liver of BALB/c6 <br><br> ? I R L Press Limited, Oxford, England. <br><br> Nucleic Acids Research <br><br> Volume 12 Number 24 1984 <br><br> 9323 <br><br> Nucleic Acids Research <br><br> <span class="blue">mouse</span> as described before (10). High molecular genomic DNA was digested with various restriction enzymes and electrophoresed on 0.8% agarose gel. Blotting analysis of DNA was carried out by the method of Southern (11). Hybridization was carried out as described previously and filters were washed either in 3 x SSC at 650C (lower stringent condition) or in 0.1 x SSC at 650C (higher stringent condition). <br><br> Screening of genomic DNA library <br><br> A <span class="yellow">bacteriophage</span> XCharon 4A/<span class="blue">mouse</span> genomic DNA library prepared with partial EcoRI digests of <span class="blue">mouse</span> DNA from MPC 11 plasmacytoma cells was kindly provided by Dr. T. Honjo. <span class="yellow">Mouse</span> IL-2-specific clones were screened by the method of Benton and Davis (12), using 700 bp PstI-AccI fragment of a cDNA clone, pMIL2-45   as  the probe   (Kashima   et  al.,   submitted   for publication).    Hybridization was performed as described previously(13). Positive clones were rescreened at least twice. Subcloninq and sequencing of the <span class="blue">mouse</span> IL-2 gene <br><br> Two EcoRI fragments of 3.3 Kbp and 2.8 Kbp from the positive recombinant X phage were subcloned into EcoRI site of plasmid pBR322.  DNA segments derived from subcloned 3.3 Kbp and 2.8 Kbp fragments were labelled at either 3' end or 5' end, and subjected to sequence analysis by the chemical degradation method (14).  The 0.8 Kbp EcoRI fragment from the same X phage clone was directly subjected to sequence analysis by the dideoxy chain termination method (15). <br><br> RESULTS <br><br> Total DNA blotting analysis <br><br> In order to study structural organization of the <span class="blue">mouse</span> IL-2 gene, we first subjected total <span class="blue">mouse</span> DNA to the blotting analysis by using various probes specific for IL-2 gene. When <span class="blue">mouse</span> DNA was digested with various restriction endonucleases and then probed with a 7 kb <span class="yellow">human</span> chromosomal DNA segment which contains the <span class="yellow">human</span> IL-2 gene and its flanking region (Fig. 1 lane 1-8, ref. 9.), single positive band appeared at lower but not at higher stringent condition for washing the filters (see Figure legend).    Additional bands corresponding to those observed by using <span class="blue">mouse</span> IL-2 cDNA probes (lane 13-17) also appeared by longer <br><br> 9324 <br><br> Nucleic Acids Research <br><br> M 1 2 3 4     M 5 6 7 8      M 9 101112    M 13 141516     17 <br><br> a~ ~ 3 ..  <br><br> I pMIL2-45 SacI           Acc I <br><br> CZI~~1      1t L2- 20 <br><br> Acc I <br><br> 1 00 tp lX i <br><br> Fig. 1.  Blot hybridization analysis of <span class="blue">mouse</span> chromosomal DNA. High molecular DNA prepared from Liver BALB/C6 <span class="blue">mouse</span> was digested with various restriction endonucleases (BamHI for lanes 1, 5, 9, 13 ; EcoRI for lanes 2, 6, 10, 14, 17 ; HindIII for lanes 3, 7, 11, 15 ; XbaI for lanes 4, 8, 12, 16 ). The resulting digests were fractionated on 0.8 % agarose gel and transferred to a nitrocellulose filter.  Filters were hybridized by the published procedure (13) either with the nick-translated chromosomal DNA containing <span class="yellow">human</span> IL-2 gene and its flanking region (total length, 7.0 kb, ref. 9) (lane 1-8) or with the nick-translated cDNA for <span class="blue">mouse</span> IL-2 (see figure). Filters were then washed either in 3 x SSC at 65 % (lower stringent condition) (lane 1-4, 9-12) or 0.1 x SSC at 65 OC (higher stringent condition) (lane 5-8, 13-17). Lane M each contains 7 size markers with their size being 23.7 kb,  9.5 kb,   6.7 kb,  4.3 kb,   2.3 kb,  2.0 kb and 0.6 kb, respectively.  Brief restriction endonuclease cleavage map for the <span class="blue">mouse</span> IL-2 cDNAs is presented in the lower part of the figure. <br><br> exposure of the film (data not shown).    Those results suggest the presence of highly conserved sequences between <span class="yellow">human</span> and <span class="blue">mouse</span> DNA either in the flanking regions or in the introns of the IL-2 gene, since the coding regions apparently show lower degree of sequence homology as evidenced in this series of blotting analysis (see below).  When the PstI insert of a <span class="blue">mouse</span> IL-2 cDNA clone, pMIL2-20, was used as the probe, a simple pattern was <br><br> 9325 <br><br> Nucleic Acids Research <br><br> obtained at higher stringent condition (lane 13-17).   While the EcoRI-digested DNA gave rise to two positive bands (2.8 kb and 0.8 kb)(Fig. 1, lane 14) by this analysis, one additional band of 3.3 kb also appeared when the same DNA was probed with a longer cDNA insert from another clone, pMIL2-45 (Fig. 1, lane 17).  The 3.3 kb band was similar in its size with the positive band which became detectable by probing the same DNA with the 7.0 kb <span class="yellow">human</span> DNA probe (Fig. 1, lane 2). Since this band appeared with the cDNA probe extending further upstream, it is likely that the 5' region of the gene is located within this DNA segment (see below). Indeed, this 3.3 kb band did not appear even after longer exposure of lane 14 (result not shown). BamHI digest of the <span class="blue">mouse</span> DNA (Fig. 1, lane 1, 13) constantly gave a very faint signal which would correspond to a DNA larger than 15 kb.         Taken together, the results suggested the presence of a single copy gene for <span class="yellow">murine</span> IL-2.    On the other hand appearance of the multiple positive bands at lower stringent washing condition (lane 9-12) indicates the presence of IL-2 related sequences within the <span class="blue">mouse</span> genome. <br><br> Screening of recombinant phaqe libraries <br><br> We next screened a gene library from partial EcoRIdigested DNA from MPC 11 cells and by using 0.8 Kbp SacI-AccI cDNA fragment as the probe and isolated 14 positive clones containing sequences specific to the <span class="blue">mouse</span> IL-2 gene. Three of the clones analysed all contained three EcoRI fragments whose size is in agreement with the result of blotting analysis of the chromosomal DNA as shown in Fig. 1 (lane 17). One of these is designated MIL-2G70. <br><br> Nucleotide sequence analysis <br><br> Two DNA fragments of 3.3 Kbp and 2.8 Kbp were excised from the phage clone MIL-2G70 by EcoRI digestion and they were subcloned into pBR322.     DNA sequences were determined for selected regions of both inserts and compared to the known cDNA sequences (Kashima et al., submitted for publication).       The strategy used for sequence analysis of the genomic DNA is presented in Fig. 2.    Comparison of the <span class="blue">mouse</span> genomic IL-2 sequence with <span class="blue">mouse</span> IL-2 cDNA sequence revealed that, like the <span class="yellow">human</span> gene, the gene is divided into four exons.     A putative <br><br> 9326 <br><br> Nucleic Acids Research <br><br> E               E           E    E   E <br><br> 500bp s    7S P HBH  H   \A <br><br> 200 bp  *     ,            -   4<br><br> I             4-4----*    *-    --4 <br><br> Fig. 2. Restriction map and sequencing strategy of <span class="blue">mouse</span> IL-2 gene. Horizontal lines indicate the length of <span class="blue">mouse</span> DNA inserted into the X phage Charon 4A or plasmid subclones.   Filled blocks, dashed blocks and open blocks indicate protein coding regions, untranslated regions and introns, respectively. Horizontal arrows indicate the direction and extent of sequence determination without ambiguity.   Dashed arrows:   determination was done by the chain termination method (15) after subcloning the       0.8 kb fragment into M13.       Rest of the sequence determination was      carried out by the method of Maxam and Gilbert (14).   A AccI site, B; BamHI site, E; EcoRI site, H; HindIII site, P; PstI site, S; SacI site. <br><br> capping site or the transcription initiation site was located 32 bp downstream from a TATAAA consensus promotor sequence (Fig. 3). The first ATG triplet was located 79 bp downstream from the TATA box. As seen also in the <span class="yellow">murine</span> IL-2 cDNA, there is an unusual repeat of CAG triplet coding for 12 glutamine residues in a row in the first exon. The second exon (60 bp) is separated from the first exon by a short intron consisting of 97 bp.     The second, the third and the fourth exons are interrupted by longer introns whose size is about 2.3 Kbp and 1.6 Kbp, respectively.  As far as the available sequence data are concerned, it seems that, despite their identical location, intron sequences are distinctly dissimilar except for the junction regions between the <span class="yellow">human</span> and <span class="blue">mouse</span> IL-2 genes.    There are two potential poly (A) addition signals within the <span class="blue">mouse</span> gene (nucleotide positions 793 - 798 and 924 - 929 in Fig. 3 ) and, based on our sequence data for various cDNA clones, both signals seem to function and give rise to heterogeneous termini of the mRNA in the LBRM-33 cells (16). <br><br> We have also determined the sequence of about 500 bp of 5'-flanking region of <span class="blue">mouse</span> IL-2 gene, since (i) promoter/regulatory sequences are located in this region in many other genes of eukaryotes and (ii) this region appeared to contain sequences which show strongest cross-hybridization between <span class="yellow">human</span> and <span class="blue">mouse</span> DNA around the IL-2 gene (Fig. 1). Comparison of the nucleotide sequences for the 5' flanking region <br><br> 9327 <br><br> Nucleic Acids Research <br><br> -4.0 <br><br> TAGGAGGTAAACCATCTCGAAACGGAAACCAATATCCTTCCTGTCTAATCAACAAATCTAAAG -400                                               -350 <br><br> GATTTATTCTTTTCATCTATCTCCTCTTGCGCCCGTCCACCACAACAGGCTGCTTACAGGTTCAGGATGGTTTTGACAAAGAGAACATTTT <br><br> -300                                               -250 <br><br> CATGAGTTACTTTTGTGTCTCCACCCCAAAGAGGAAAATTTTGTTCATACGAAAGGCGTTCATTGTATGAATTAAAACTGCCACCTAAGAG <br><br> -200                                               -150 <br><br> TGGGCTAACCCGACCAAGAGGGATTTCACCTAAATCCATTCAGTCAGTATATGGGGTTTAAACAAATTCCAGAGAGTCATCAGAAGAGGAA <br><br> -100                                                -50 <br><br> AAACAAAGGTAATACTTTCTGCCACACAGGTAGACTCTTTTGAAAATATGTGTAATATGTAAAACATCGTGACACCCCCATATTATTTTTC CAGCATTAACAGTATAAATTGCCTCCCATGCTGAAGAGCTGCC TCACCCTTGCTAATCACTCCTCACAGTGACCTCAAGTCCTGCAGGC <br><br> 50                                                                   100 <br><br> ATG TAC AGC ATG CAG CTC GCA TCC TGT GTC ACA TTG ACA CTT GTG CTC CTT GTC AAC AGC GCA CCC ACT Met Tyr Ser Met Gln Leu Ala Ser Cys Val Thr Leu Thr Leu Val Leu Leu Val Asn Ser Ala Pro Thr <br><br> 150 <br><br> TCA AGC TCC ACT TCA AGC TCT ACA GCG GAA GCA CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG CAG Ser Ser Ser Thr Ser Ser Ser Thr Ala Glu Ala Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln Gln <br><br> 200 <br><br> CAC CTG GAG CAG CTG TTG ATG GAC CTA CAG GAG CTC CTG AGC AGG ATG GAG    GTAAGTGCACAGCCATCCCATC His Leu Glu Gln Leu Leu Met Asp Leu Gln Glu Leu Leu Ser Arg Met Glu <br><br> TATAGGCAATACCTTTAGCTTTCTTGCCAAAGGCTGTGTTTAATAACCTTTAATAATAATGTTACGCTTTCTCAG AAT TAC AGG AAC <br><br> lAsn Tyr Arg Asn 250 <br><br> CTG AAA CTC CCC AGG ATG CTC ACC TTC AAA TTT TAC TTG CCC AAG CAG GTGAGTGAGTTTCTGTTTAACTGGTGC Leu Lys Leu Pro Arg Met Leu Thr Phe Lys Phe Tyr Leu Pro Lys Gln <br><br> TCTAATG----------------------------------------------------------__ <br><br> ?_______________----------------------- 2000 bp --------------------?---------------------------------------------------------- AATGTGAACCTTGTAGTTTCTTTGTAGATTGGAACAATAGTCTGAACTTGTGT TATGCATTGGTAGAGAAACACAGACTTTTTACAAAGTCTAATGAAGCCAGAAGAGCTCACTAGAGTGGCCTAACAGTTAAGGTGACCCTTA TGGTTGTGAGCTCTTGCTCTTCTAGATTTATGGCATCGATTACCTCAGTCCCCCTTTACAGAGGACAGGGAGTGGTAAAAGCTATGTGCTG CCTTCTCTTGATTAGAGAGACTGCAGACTAACTTTCTGGCTCTTCAGTATGTGGTGAGCTGAGCTGATGGTTAAGCTTATTACTCCTCTAG <br><br> 300                                                                 350 <br><br> GCC ACA GAA TTG AAA GAT CTT CAG TGC CTA GAA GAT GAA CTT GGA CCT CTG CGG CAT GTT CTG GAT TTG Ala Thr Glu Leu Lys Asp Leu Gln Cys Leu Glu Asp Glu Leu Gly Pro Leu Arg His Val Leu Asp Leu <br><br> 490 <br><br> ACT CAA AGC AAA AGC TTT CAA TTG GAA GAT GCT GAG AAT TTC ATC AGC AAT ATC AGA GTA ACT GTT GTA Thr Gln Ser Lys Ser Phe Gln Leu Glu Asp Ala Glu Asn Phe Ile Ser Asn Ile Arg Val Thr Val Val AAA CTA AAG GTAAGGTGTTGCTTTATTTGCTAATCTGGAAATAAAATAGAGAAGAAATGCATTTTTAAGTGGCTTGCCATTTCTGGT Lys Leu Lys <br><br> CTTTGATGGGTTCTGTGCATTTAGTCAACCAAAGTTTAAAGTCACTGTGCAAGTGAATCC --------------------------------------------------------------------    -1000   bp  ------ --------------------------------------------------------------- GAATTCTACAGAAGTGTTCAGTGTTCCCATCAAATGCTCGTTGGTAATCAACTCTGGAGAGCTT TATTTTTTATGCTTTACCATTTGATATCATAATAAAACATAACCCAATAATTGTGATCATTTCAGAAATGTGAATGTTCAATATATTTAAC <br><br> CAGTGTTAAAATAAATGCCTAAAAGCTCCGTTGAAGAATAATTATATGCAAAGTAAGCTACCTTAGCCTACAATTTTATATTCTTTTTTAG <br><br> 450                                                                 500 <br><br> GGC TCT GAC AAC ACA TTT GAG TGC CAA TTC GAT GAT GAG TCA GCA ACT GTG GTG GAC TTT CTG AGG AGA Gly Ser Asp Asn Thr Phe Glu Cys Gln Phe Asp Asp GlU Ser Ala Thr Val Val Asp Phe Leu Arg Arg <br><br> 550 <br><br> TGG ATA GCC TTC TGT CAA AGC ATC ATC TCA ACA AGC CCT CAA TAACTATGTACCTCCTGCTTACAACACATAAGGC Trp Ile Ala Phe Cys Gln Ser Ile Ile Ser Thr Ser Pro Gln <br><br> 600                                                650 <br><br> TCTCTATTTAT4TAAATATTTAACTTTAATTTATTTTTGGATGTATTGTTTACTATCTTTTGTAACTACTAGTCTTCAGATGATAAATATG <br><br> 700                                                750 <br><br> GATCTTTAAAGATTCTTTTTGTAAGCCCCAAGGGCTCAAAAATGTTTTAAACTATTTATCTGAAATTATT+ATTATATTGAATTGTTAAAT <br><br> < 4>800                                                 850 <br><br> ATCATGTGTAGGTAGACTcATTCA TAAAAGTAmAGATGATTCAAATATAAATAAGCTCAGATGTCTGTCATTTTTAGGACAGCACAAAG <br><br> 900                         0" <br><br> TAAGCGCTAAAATAACTTCTCAGTTATTCCTGTGAACTcTATGTTAATCAGTGTTTTCAAGAAATAAAGCTCTCCTCT           AAA GATGGCTTGTGGGAAAAGATCTCCTCTCCAGGGAGCTAACATCAGCTCAGAGTTTACTCAAGAATTC <br><br> Fig.      3.      Nucleotide sequence of <span class="blue">mouse</span> IL-2 gene.                               Four exons are framed.          Numbers refer to nucleotide positions of exons from the presumed cap site. Dots and open circles indicate TATA box and poly A additional signals, respectively. <br><br> 9328 <br><br> Nucleic Acids Research <br><br> -4 5 0                               -400 <br><br> TAGGAGGT AAACCA T CTCGAAAC GGAAACCAATATCCTTCCTGTCTAATCAACAAATCTAAAGGATTTATTCTTTTCATCTATCTCC <br><br> .0     * .0 .- *. * ..* --.  -- .*-@ .. ............  --  -*-* * <br><br> TAAAAAGGTAAAACCAGTTCT GAAACAGGAAACCAATACACTTCCTGTTTAATCAACAAATCTAAA CATTTATTCTTTTCATCTGTTTAC <br><br> -350                         -300 <br><br> TCTTGCGCCCGTCCACCACAACAGGCTGCT T ACAGGTTCAGGATGGTTTTGACAAAGAGAACATTTTCATGAGTTACTTTTGTGTCTC TCTTGCTCTTGTTCACCACAATA  TGCTATTCACATGTTCAGTGTAGTTTTATGACAAAGAAAATTTTC TGAGTTACTTTTGTATCCC <br><br> -250 <br><br> CA CCCC  AAAG  AGGAAAATTTGTTTCATACGAAAGGCGTTCATTGTATGAATTAAAACT CCACCTAAGAGTGGGCTAACCCGA <br><br> *~~~ ~~   ..... *   .-- *..-  * .-- .**  . *-.   --.   *- *- -  -  *-   0 @  *-- 0 .. .   <br><br> CACCCCCTTAAAGAAAGGAGGAAAAACTGTTTCATACAGAAGGCGTTAATTGCATGAATTAGAGCTATCACCTAAGTGTGGGCTAATGTAA <br><br> -200                          -150 <br><br> CCAAGAGGGATTTCACCTAAATCCATTCAGTCAGTATAT GGGGTTTAAACAAATTCCAGAGAGTCATCAGAAGAGGAAAAACAAAGGTAA CAAAGAGGGATTTCACCTACATCCATTCAGTCAGTCTTTGGGGGTTTAAA AAATTCCAAAGAGTCATCAGAAGAGGAAAAATGAAGGTAA <br><br> -100                          -50 <br><br> TACTTTCTGCCACACAGGTAGACTCTTTTGAAAATATGTGTXATATTAAAACATCGTGACACCCCCATATiXrTCCAGCATTAACAG TGTTTTTT CAGACAGGTAAAGTC TTTGAAAATATGTGTAATATGTAAAACATTTTGACACCCCCATAATATTTTTCCAGAATTAACAG ATAAGCCTCCCATGCTGAAGAGCTGCCTATCACCCTTGCTAATCACTCCTCACAGTGACCTCAAGTCCTGCAGGCATG  <span class="yellow">Mouse</span> rTATGCATCTCTTGTTCAAGAGTTCCCTATCACTCTCTTTAATCACTACACACAGTAACCTCAACTCCTGCCACTATG  <span class="yellow">Human</span> <br><br> Fig.  4.   Comparison of 5'-flanking regions of <span class="blue">mouse</span> and <span class="yellow">human</span> IL-2 genes.   In aligning the sequences for both genes, gaps were introduced   to maximize   homology.    Dots   indicate  identical nucleotide sequences.   Number 1 indicates putative transcription initiation site. Inverted repeats are indicated by bars and dashed lines. TATA box is framed. <br><br> of both genes is illustrated in Fig. 4.    The nucleotide sequence homology from the TATA box to -470 is 85%. The highest region of homology was observed between the TATA box and position -97 (60 bp matches out of 64 bp) of both genes. DISCUSSION <br><br> We have isolated recombinant clones for <span class="blue">mouse</span> IL-2 gene from a phage Charon 4A/<span class="blue">mouse</span> genomic DNA library and determined the entire sequence of the gene except for the sequence of the internal portion of the second and third introns.         The <span class="blue">mouse</span> IL-2 cDNA sequence was aligned with the genomic sequence and both sequences matched completely each other. The unusual CAG repeats encoding 12 glutamines which was found previously in the cloned cDNA was shown to be present also in the chromosomal gene. This finding further excludes the possibility that the unique repeat is generated by artifacts during the cDNA cloning process. Although we can not rule out the possibility for the deletion of this sequence in the <span class="yellow">human</span> IL-2 gene, it is more likely that the CAG repeat has been generated in the <span class="blue">mouse</span> genome rather recently.    The repeat could have been generated either by a direct insertion of the sequence or by the duplication after insertion of a unit sequence. <br><br> 9329 <br><br> Nucleic Acids Research <br><br> Organization of the <span class="blue">mouse</span> IL-2 gene resembles to that of the <span class="yellow">human</span> gene (Fig. 2., ref. 9).   There seems to be little sequence homology between corresponding introns of <span class="blue">mouse</span> and <span class="yellow">human</span> IL-2 genes, except for the intron-exon junctions part of which is thought to be necessary for the RNA splicing (17, 18). Dissimilarity of the intron sequences among the genes which are derived from a common ancestor has been reported in other genes (19, 20).   In spite of the divergence in sequence of introns, the size and position of the introns are very similar between the <span class="yellow">murine</span> and <span class="yellow">human</span> IL-2 genes. <br><br> Of particular interests is the presence of highly conserved sequences in the 5' -flanking region of the <span class="yellow">human</span> and <span class="blue">mouse</span> IL-2 gene (Fig. 4).    Whereas the coding region shows nucleotide sequence homology of 72% between the two genes, the 5' upstream region spanning about 500 bp (Fig. 4) shows 85% homology which was readily detectable by the blotting analysis (Fig. 1, lane 1-4).  Since we have not yet determined the nucleotide sequence further upstream of the <span class="blue">mouse</span> gene, we do not know whether or not this similarity extends further.  It is  likely that such sequences are involved in the controlled expression of the IL-2 genes in activated T-lymphocytes.  Work is in progress to identify such DNA sequences by introducing the cloned genes into various lymphocytic cell lines.   Our preliminary results indicate that the 5 '-flanking sequence of the <span class="yellow">human</span> IL-2 gene mediates mitogen induced expression of the gene in T-lymphocytic cells (Fujita & Taniguchi, unpublished observation). <br><br> ACKNOWLEDGEMENTS <br><br> We thank Dr. T. Honjo for <span class="blue">mouse</span> gene library.    We are also indepted to Ms. M. Nagatsuka for typing the manuscript. This work was supported in part by Grant-in-Aid for Special Project Research, Cancer-Bioscience from the Ministry of Education, Science and Culture, Japan. <br><br> ?To whom correspondence should be addressed <br><br> *Present address: Department of Microbiology, School of Medicine, Chiba University, Chiba 280, Japan <br><br> +Present address: Central Research Laboratory, Ajinomoto Co.Inc., Totsuka-ku, Yokohama 244, Japan <br><br> 9330 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> 1. Morgan, D. A., Ruscetti, F. W. and Gallo, R. (1976) Science, <br><br> 193, 1007-1008. <br><br> 2. Gil lis, S. Ferm, M. M., Ou, W. and Smith, K. (1 978) J. <br><br> Immunol., 120, 2023-2027. <br><br> 3. Chen, B. M. and Di Sabato, G. (1976) Cell. Immunol., 22, 211<br><br> 224. <br><br> 4. Henney, C. S., Kuribayashi, K., Kern, D. E. and Gillis, S. <br><br> (1981) Nature, 291, 335-338. <br><br> 5. Wagner, H., Hardt, C.,   Heeg, K., Rollinghoff, M. and <br><br> Pfizenmaier, K. (1980) Nature, 284, 278-280. <br><br> 6. Farrar, J. J., Benjamin, W. R., Hilfikr M. L., Howard, M., <br><br> Farrar,W. L. and Fuller-Farrar, J. (1982) Immunol. Rev. 63, 129-166. <br><br> 7. Pearlstein, K., Palladino, M. A., Welte, K. and Vilcek, J. <br><br> (1983) Cell. Immunol., 80, 1-9. <br><br> 8. Taniguchi, T., Matsui, H., Fujita, T., Takaoka, C., Kashima, <br><br> N., Yoshimoto, R. and Hamuro, J. (1983) Nature 302, 305-310. <br><br> 9. Fujita,T.,Takaoka,C., Matsui, H. and Taniguchi T. (1983) <br><br> Proc. Natl. Acad. Sci. USA 80, 7437-7411. <br><br> 10. Ohno, S. and Taniguchi, T. (1982) Nucleic Acids Res. 10, 967<br><br> 977. <br><br> 11. Southern, E. M. (1975) J. Mol. Biol. 98, 503-517. <br><br> 12. Benton, W. D. and Davis, R. W. (1977) Science 196, 180-182. <br><br> 13. Ohno, S. and Taniguchi, T. (1981) Proc. Natl. Acad. Sci. USA <br><br> 78, 5305-5309. <br><br> 14. Maxam, A. M. and Gilbert, W. (1980)  Meth. Enzymol. 65, 560<br><br> 580. <br><br> 15. Sanger, F., Nicklen, S. and Coulson, A. R. (1977) Proc. Natl. <br><br> Acad. Sci. USA, 74, 5463-5467. <br><br> 16. Gillis, S., Scheid, M. and Watson, J. D.(1980) J. Immunol. <br><br> ,125, 2570-2580. <br><br> 17. Breathnach, R., Benoist, C., O'Hare, K., Gannon, F. and <br><br> Chambon, P.(1978) Proc. Natl. Acad. Sci. USA, 75, 4853-4857. <br><br> 18. Lerner, M. R., Boyle, J. A., Mount, S. M., Wolin, S. L. and <br><br> Steitz, J. A. (1980) Nature, 283, 220-224. <br><br> 19. Van Ooyen, A., Van den Berg, J., Mantei, N. and Weissmann, <br><br> C.(1979) Science, 206, 337-344. <br><br> 20. Searle, P. F., Davison, B. L., Stuart, G. W., Wilkie, T. M., <br><br> Norstedt, G. and Palmiter, R. D. (1984) Mol. Cell. Biol. 4, 1221-1230. <br><br> 9331 <h3>pmcA327937</h3>Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a.
Abstract
Attempts to prevent the urea conversion of a 200-230,000 molecular weight DNA polymerase alpha to a 150-170,000 molecular weight form by the inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70,000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70,000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit.<br><br><br><br>
 Volume 6 Number 10 1979 <br><br> Nucleic Acids Research <br><br> Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a Keith McKune and Andrew M.Holmes* <br><br> Biochemistry Department, Strathclyde University, The Todd Centre, 31 Taylor Street, Glasgow, G4 ONR, UK <br><br> Received I June 1979 ABSTRACT <br><br> Attempts to prevent the urea conversion of a 200-230, 000 molecular weight DNA polymerase a to a 150-170, 000 molecular weight form by the <br><br> inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70, 000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70, 000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit. <br><br> INTRODUCTION <br><br> In mammals DNA polymerase a is thought to be the replicative <br><br> enzyme, but due to low levels of activity, even in tissues actively making DNA, and to enzyme heterogeneity it has been found difficult to purify <br><br> However, small samples of DNA polymerase a have been highly purified <br><br> 2, 3, 4, 5 from several sources and partially characterised <br><br> Heterogeneity has been observed in DNA polymerase a from a var<br><br> 6           7 <br><br> iety of sources, including <span class="yellow">rat</span> liver and spleen , Hela cells, baby <span class="yellow">hamster</span> <br><br> 8                9, 10                   11 <br><br> kidney cells , <span class="blue">mouse</span> myeloma    , <span class="yellow">Drosophila</span> embryos  and <span class="yellow">calf</span> thymus <br><br> 1,6, 12, 13 . We have previously observed several species of <span class="yellow">calf</span> thymus <br><br> DNA polymerase a differing in size and charge  . In order of elution from DEAE cellulose they are enzyme A1 (200-230, 000 molecular weight), A2 (200-230, 000), B (100-110, 000) and C (150-170, 000). A poly (dA). oligo <br><br> (dT)10 preferring enzyme, enzyrrme D (140-150, 000 molecular weight) elutes just after enzyme B. The A enzymes seem identical in all properties except their charge. 5.0-5. 5S enzymes, analagous to B, have been observed to <br><br> C Information Retrieval Limited 1 Falconberg Court London Wl V 5FG England <br><br> 3341 <br><br> Nucleic Acids Research <br><br> arise as the result of proteolytic action  ' , similarly the B enzyme above 14 <br><br> ,but the relationship of the other observed species is not clear, nor is <br><br> the problem of which, if any, of these species is the replicative enzyme, <br><br> although circumstantial evidence has implicated a DNA polymerase a to be <br><br> 15 the replicative enzyme in <span class="yellow">adenovirus</span> infected KB cells <br><br> We have previously shown that several of these enzymes are inter<br><br> 16 <br><br> convertible . In particular, mild urea treatment can convert both A <br><br> enzymes to C enzyme, with the loss of a subunit or fragment of 50-70, 000 molecular weight. The C enzyme retains DNA polymerase activity, but <br><br> does show differences compared to A enzyme in heat sensitivity and sensitivity to N-ethylmaleimide. <br><br> Highly purified samples of A and C enzymes when subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis have shown a corre<br><br> lation of DNA polymerase activity with a polypeptide band 155, 000 molecular weight . Contaminating material of 50-70, 000 molecular weight appeared <br><br> to obscure the putative 50-70, 000 subunit in gels of A enzyme. The conclusions drawn were that DNA polymerase a is a 155, 000 molecular weight <br><br> polypeptide (C enzyrrme) which can and does associated with material of 50<br><br> 70, 000 molecular weight to give A enzyme, and that enzyme B is a proteol<br><br> 14 <br><br> ytic degradation product . The relationship of enzyme D to the others is <br><br> not clear. <br><br> It is possible that during the enzyme isolation procedure proteolytic action may have taken place on a 200-230, 000 molecular weight enzyme releasing 155, 000 and 50-70, 000 fragments which remain in association <br><br> until urea treatment separates them. It is also possible that urea treatment itself renders the enzyme susceptible to contaminating proteases. Further experiments to ascertain whether or not this is the case and, if so, to prevent it, have been carried out. <br><br> A enzyme can be reconstituted from C enzyme formed by the action <br><br> of urea on A enzyme by concentrating it with the 50-70, 000 molecular weight material. The A enzyrme obtained in this manner is highly purified, as is <br><br> the C enzyme formed by the urea treatment, and has been used in attempts to show differences in synthetic template-initiator complex utilisation <br><br> 3342 <br><br> Nucleic Acids Research <br><br> b y the se two enzyme s . <br><br> MATERIALS AND METHODS <br><br> <span class="yellow">Calf</span> thymus was obtained from 10-16 week old <span class="yellow">calves</span> and frozen at <br><br> -20?C until required. Chromatographic media and chemicals were obtained <br><br> 6 <br><br> from sources previously referred to . Radioactive deoxynucleoside tri<br><br> phosphates were obtained from the Radiochemical Centre, Amersham, Bucks. Synthetic oligo and polynucleotides were obtained from P. L. Biochemicals <br><br> Inc., except for poly (dC) which was a gift from Dr. I.R. Johnston and pre<br><br> 17 <br><br> pared from oligo d(C)5 as described  . N-a-p Tosyl-L-lysine chloromethy<br><br> lketone HCl and phenylmethylsulphonylfluoride were obtained from Sigma and Trasylol from Bayer. <br><br> Except where indicated all buffers contained 20% w/v glycerol and <br><br> 1 mM dithiothreitol. Standard linear phosphate gradients were run between 0.03 M and 0.25 M potassium phosphate, pH 7.8. Gradient salt concentr<br><br> 2 <br><br> ations were measured using a conductivity meter as described . Urea was <br><br> prepared as a 4. 8 M solution in 20% w/v glycerol, stirred with Amberlite <br><br> MB3 and filtered. Dithiothreitol was added to a final concentration of 1 mM before use. <br><br> 6 DNA polymerase was assayed using activated DNA as described <br><br> except that the buffer was 50 mM tris HC1, pH 7.8. One unit of DNA polymerase activity incorporates ln mol [ H] dTMP into an acid insoluble form in one hour at 370C. Assays using synthetic template-initiator complexes were carried out at 30?C in 0. 12 ml. The template-initiator complexes <br><br> 6 <br><br> were prepared and assays processed as described . All assays contained <br><br> 1 mM dithiothreitol, 62. 5 p.g <span class="yellow">bovine</span> serum albumin, 1 ,ug template-initiator complex, enzyme protein and the relevant [ H] deoxynucleoside triphosphate at 0.1 mM and 12-15 cpm/pmol. These assays were carried out at either pH 6.4 in 20 mM sodium-potassium phosphate, or at pH 7.8 in 50 mM tris HCl and contained either 10 mM MgCl2 or 1 mM MnCl2 as indicated. <br><br> Preliminary purification of DNA polymerase a to Fraction IV was as <br><br> 6 <br><br> described , the purification steps being phosphocellulose chromatography, <br><br> ammonium sulphate precipitation and gel filtration on Sepharose 6B. <br><br> 3343 <br><br> Nucleic Acids Research <br><br> Enzyme obtained from the DEAE cellulose step is referred to as Fraction <br><br> V enzyme. Samples were prepared for sodium dodecyl sulphate polyacrylamide gel electrophoresis and scanned as previously described2 <br><br> RESULTS AND DISCUSSION <br><br> (a)   Interconversion Studies <br><br> A enzyme was routinely converted to C enzyme by incubating 200500 units/ml of Fraction V A enzyme in 2.4 M urea in 0. 02 M potassium <br><br> phosphate, pH 7.8, for 60 minutes at 0?C. The mixture was then loaded on to a DEAE cellulose, washed with 0. 03 M potassium phosphate pH 7. 8, and enzyrme eluted either with the standard phosphate gradient or batchwise. Under these conditions usually about 50-60% of recovered activity was <br><br> enzyme C (Fig 1 a). Overall recovery was 70-80%. A enzyme was recon<br><br> 14 <br><br> stituted essentially as described . The flow through material from the <br><br> DEAE cellulose column after the urea treatment was loaded on to a 1 x 0. 8 cm phosphocellulose column, washed with 0.03 M potassium phosphate, pH 7.8, and the protein eluted with 0.25 M potassium phosphate, pH 7. 8. This material, the putative subunit, was vacuum dialysed with the C enzyme produced by the urea treatment, rechromatographed on DEAE cellulose and enzyme eluted batchwise (Fig 1 b). Recovery from this procedure was <br><br> usually 50-70% A enzyme. Overall recovery was 30-40% of the C enzyme dialysed. If the material eluted from the phosphocellulose by the 0. 25 M potassium phosphate was heated to 90?C for 5 minutes prior to vacuum <br><br> dialysis with the C enzyme the recovery from the DEAE cellulose column <br><br> was significantly higher (60-70% of the original C enzyme activity), but all recoverable DNA polymerase activity was C enzyme. This, together with the fact that the 60 minute treatment with urea has, on occasions, given <br><br> rise to a 50% increase in DNA polymerase activity prior to loading on to the DEAE cellulose column, would indicate that A enzyme is less active on <br><br> activated DNA than is C enzyme. If the DEAE cellulose flow through material came from urea treatment of A2 enzyme then A2 was produced on reconstitution; if from A1 then A1 was produced. <br><br> Although the mild urea treatment of A has been used to prepare C of <br><br> 3344 <br><br> Nucleic Acids Research <br><br> b <br><br> 120       180 C <br><br> I         I    ^ <br><br> Figure la  DEAE cellulose chromatography after 2.4 M urea treatment. <br><br> 4, 5000 units, 4.4 mg of Fraction II A2 enzyme were incubated with urea at a final concentration of 2.4 M for 60 minutes at 0?C, loaded on to a 5 x 1.4 cm DEAE cellulose column, the column was washed with 0. 03 M potassium phosphate, pH 7.8 and a 200 ml standard phosphate gradient applied. 2.3 ml fractions were collected and 10 Rl assayed for 5 minutes. (o-o) no <br><br> phenylmethylsulphonylfluoride (*-*) 3 mM phenylmethylsulphonylfluoride ( - ) phosphate gradient. <br><br> Figure lb  The reconstitution of A2 enzyme. 3000 units for C enzyme <br><br> derived by urea treatment of A2 were vacuum dialysed with the DEAE cellulose flow through material after phosphocellulose chromatography and <br><br> chromatographed on a 2 x 1. 2 cm DEAE cellulose column. After washing with 0.03 M potassium phosphate, pH 7.7, the enzymes were eluted batchwise with the above concentrations of potassium phosphate, pH 7.8. 1 ml fractions were collected and 10 .l assayed for 10 minutes. <br><br> 3345 <br><br> Nucleic Acids Research <br><br> high specific activity it is possible that conversion of the 200-230, 000 molecular weight enzymes to a 155- 170, 000 species maybe due to unfolding of the molecule to allow limited attack by contaminating proteases. Accord<br><br> ingly the urea conversion of A to C was investigated in the presence of certain protease inhibitors. The presence of the serine protease inhibitor <br><br> phenylmethylsulphonyl-fluoride in the incubation and chromatography buffers did not affect the conversion of A2to C (Fig 1 a). Likewise trasylol and Na-p Tosyl-L-lysine chloromethylketone HC1 had no effect. However, protease action could have occurred earlier in the purification procedure and the urea could be separating two fragments. Usually the DEAE cellulose profile shows that the majority of the enzyme activity is present as A <br><br> enzyme (Fig 2). In this instance A1 and A2 have not been separated. When the temperature of the material in the original blending procedure was kept below 0?C or phenylmethylsulphonylfluoride, N-a-p Tosyl-L-lysine chloromethylketone HCl or trasylol was included in the isolation buffers the DEAE cellulose profile was similar. The A enzymes were still capable of conversion by mild urea treatment to C enzyme, indicating that if protease activity is involved then it is not susceptible to these inhibitors. When the <span class="yellow">calf</span> <br><br> 20FI: Now ~ ~      ~      20 20Fr Na      06 <br><br> Figure 2  DEAE cellulose elution profile of <span class="yellow">calf</span> thymus DNA polymerase a. 47, 000 units, 97 mg of Fraction IV enzyme prepared from 415 g of <span class="yellow">calf</span> <br><br> thymus were loaded on to a 10 x 1.8 cm DEAE cellulose column, after washing with 0. 03 M potassium phosphate, pH 7. 8 a 400 ml standard phosphate gradient was applied. 5 ml fractions were collected and 10 ,ul assayed for 5 minutes. <br><br> 3346 <br><br> Nucleic Acids Research <br><br> thymus was allowed to warm up during the blending procedure, or the supernatant prior to phosphocellulose chromatography was heated to 37 ?C for 30 minutes, there was a marked decrease in the amount of A enzyme with a concomitant increase in the amount of B and C (unpublished observation). The presence of phenylmethylsulphonylfluoride, N- a-p T osyl- L-lysine <br><br> chloromethylketone HCl or trasylol under these conditions only had a marginal effect on the appearance of C enzyme, but did reduce the amount of B <br><br> enzyme. Heating the enzyme to 37?C after the phosphocellulose step had no effect on the DEAE cellulose elution profile. Attempts to convert A enzyme to C using trypsin have not been successful. A enzyme activity is lost without the appearance of any other species (unpublished observation), although <br><br> 14 the action of trypsin on C enzyme can give rise to small amounts of B <br><br> Although the conversion of A enzyme to C does not appear to be the result of serine protease action, proteases other than serine proteases <br><br> 18 <br><br> could have been responsible  . Also, the fact that A enzyme can be recon<br><br> stituted from C plus the flow through material from the DEAE cellulose after urea treatment does not necessarily mean we are dealing with two subunits as fragments produced by proteases mray be reassembled to give active <br><br> enzyme 9    . However, the fact that a 200-230, 000 molecular weight polypeptide band has never been observed in sodium dodecylsulphate polyacrylamride gels of highly purified A enzyme , or even cruder fractions of A <br><br> enzyrre (unpublished observation), may be significant. One might expect <br><br> some of the enzyme not to have been attacked by whatever is responsible for cleaving the molecule, if, indeed, this does happen. It would appear, therefore, that the A enzyme consists of subunits of 155, 000 and 50-70, 000 molecular weight with the small subunit having a slightly different charge in the <br><br> case of A  and A2. It has also been concluded that the heterogeneity in the <br><br> 1                                     ~~~~~~~~~~~~~~~~~3 <span class="blue">mouse</span> myeloma DNA polymerase a fraction is not due to proteolysis . However, the 50-70, 000 molecular weight subunit has not yet been identified. <br><br> Sodiurrm dodecylsulphate polyacrylamide gels of reconstituted A enzyme have shown polypeptide bands at 150-160, 000 and 50-70, 000 molecular weight <br><br> (Fig. 3), but the ratio of staining intensity of the bands does not correspond to a 1: 1 relationship. The ratio of the two bands is variable but is usually between 1:2 and 1:3, indicating, perhaps, that more than one subunit of 50<br><br> 3347 <br><br> Nucleic Acids Research <br><br> TOP      123   4 5   6 <br><br> Figure 3  Scan of a 5% sodium dodecyl sulphate polyacrylamide gel of <br><br> reconstituted A2 enzyme. 1500 units of reconstituted A2 enzyme were subjected to polyacrylamide gel electrophoresis under non-denaturing conditions. The gels were sliced, enzyme extracted and assayed and the peak <br><br> fraction of DNA polymerase activity from three gels were pooled, subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis on a single <br><br> gel, stained and scanned at 2 volt sensitivity as described2. The molecular weight standards were: (1) <span class="yellow">bovine</span> serum albumin dimer (134, 000), (2) ,3 galactosidase (130, 000), (3) phosphorylase a (94, 000), (4) <span class="yellow">bovine</span> serum <br><br> albumin (68, 000), (5) pyruvate kinase (57, 000) and (6) lactate dehydrogenase (35, 000). <br><br> 70, 000 molecular weight can associate with the 155, 000 subunit. The proportion of the lower molecular weight polypeptide band is higher in the reconstituted A enzyrrme than in the C enzyme preparation from which it was formed, but even the C enzyme contained some material in the region. <br><br> Breakdown of material from 155, 000 to 50-70, 000 may be partly responsible for the contamination, but the presence of phenylmethylsulphonylfluoride in samples in preparation for sodium dodecylsulphate polyacrylamide gel electrophoresis has not been successful in preventing it. The <span class="yellow">human</span> KB cell <br><br> DNA polymerase a, equivalent to the C enzyme, has been reported to consist <br><br> 4 <br><br> of subunits of 76, 000 and 66, 000 molecular weight , but at no time have we <br><br> observed polypeptide bands at 76, 000 and 66, 000 rising and falling with <br><br> enzyme activity in any gels of <span class="yellow">calf</span> thyrrmus enzymes A1, A  and C (and <br><br> 2 unpublished observations). (b)    Template Studies <br><br> Use has been made of the urea conversion of A to C and of the reconstitution of A enzyme to obtain samples of highly purified DNA polymerase, <br><br> specific activity in excess of 50, 000 units/mg, in order to study the effect of this 50-70, 000 molecular weight subunit on the DNA polymerase subunit. Previous results have indicated that A enzyrrme is stabler to heat and less <br><br> 3348 <br><br> Nucleic Acids Research <br><br> 14, 21 <br><br> susceptible to N-ethylmaleimide than C enzyme  . It has been reported <br><br> that highly purified DNA polymerase a can be associated with a protein capable of binding to DNA containing no 3' OH ends and capable of being released during the DNA polymerase assay  . Attempts to dissociate the 50<br><br> 70, 000 molecular weight subunit from the polymerase subunit by incubating A2 with DNA polymerase reaction mixes containing activated DNA, poly <br><br> (dA-T) and poly (dT). oligo (A)10 followed by ultracentrifugation in high salt have been unsuccessful. Similarly A absorbed and eluted from either native or denatured DNA cellulose remained A enzyme. Both A and C enzymes <br><br> were eluted from the DNA celluloses by less than 0. 1 M NaCl so it does not appear that the subunit enhances the binding of the DNA polymerase subunit to DNA. However, there are differences in the response of A and C <br><br> enzymes to synthetic template-initiator complexes (Table 1). Even if one takes into account the fact that A enzyme is less active on activated DNA <br><br> than C enzyme (the addition of the 50-70, 000 subunit to the polymerase subunit appears to result in a decrease of about 30% of polymerase activity on activated DNA) the A enzyme is still more active on these templates. <br><br> Although there is a variation in activity on these template-initiator comp<br><br> lexes each time assays are carried out on them depending on the method of preparing the complexes and the base ratio of template to initiator, A enzyme always seems to be significantly more active than C. The A2 <br><br> enzyme at pH 7.8, with extra subunit(s) is clearly more effective on the <br><br> oligoribonucleotide initiator, oligo(A)10 than is enzyme C. In view of the proposed RNA initiation of Okazaki pieces this may indicate a role of this subunit in Okazaki piece synthesis, in that it may aid the DNA polymerase to'take over' from the RNA polymerase. Using poly (dA). oligo (dT) 10 <br><br> (A:T=20: 1) and following the incorporation of [ H] dTMP as a function of time a short lag was observed for C enzyme, but not for A2 (Fig. 4). <br><br> Similar re sult s we re obtained when poly (dC). oligo (dG) 1 0 (C: G= 5: 1) wa s <br><br> used as temrrplate-initiator, but not when poly (dT). oligo (A)10 (T:A=1: 1) was used. Neither enzyme showed a lag on this template-initiator complex or on activated DNA. Incorporation versus enzyme concentration also showed this lag for C enzyme on poly (dA). oligo (dT)10. It is not certain what causes <br><br> 3349 <br><br> Nucleic Acids Research <br><br> TABLE 1 Template utilisation by reconstituted A and C derived from A. <br><br> 3H] dNTP Divalent cation  A2 at pH    C at pH <br><br> Template                          _      6.4   7.8    6.4   7.8 Activated DNA         dTTP         Mg++            100          100 Activated DNA         dATP         Mg++             91          85 Activated DNA         dGTP         Mg++             85          82 poly(dA).oligo(dT)10  dTTP         Mg      66.0    4.0    20    91 <br><br> (A:T = 20: 1) <br><br> poly(dA). oligo(dT)   dTTP         Mu      19.0    4. 5  6. 5  2.5 <br><br> poly(dA. oig~10 (A:T = 20:1) <br><br> poly(dT). oligo(dA)10  dATP        Mg       2.5   <1 (1       (1 <br><br> (T:A = 5:1) <br><br> poly(dT) .oligo(dA)1  dATP         Mn ++   15.5   23.0   9. 0  3. 0 <br><br> poly(dT). oligo(A)10  dATP         Mg      15.0  230.0  10.0  77. 5 <br><br> (T:A = 1:1) <br><br> poly(dT) * oligo(A) 10  dATP       Mn      41.5   75. 5 19.5  21.5 <br><br> (T:A = 1:1) <br><br> poly(dC). oligo(dG)   dGTP         Mg++     6. 0   15.5  4. 5  7. 5 <br><br> (C:G = 5:1)   10 <br><br> poly(dC). oligo(dG)10  dGTP        Mn +     6. 0  29.5   5. 0  6. 5 <br><br> (C:G  = 5: 1)                                 .    . <br><br> Values given are relative to incorporation of [ H] dTMP at pH 7. 8 on activated DNA. For A2 this was 176.5 pmol, for C 232 pmol. dATP, dCTP, dGTP and dTTP were included for assays using activated DNA, only the deoxynucleoside triphosphate stated was used in the synthetic template<br><br> initiator complex assays. Assays were for 10 minutes. The buffers and concentrations of the divalent cations were as in Materials and Methods. <br><br> this lag, but the annealing of template to initiator is only transient '  and the DNA polymerase subunit may have difficulty in stabilising the complex and the 50-70, 000 molecular weight subunit may be able to help the poly<br><br> merase subunit to overcome this. Addition of the 50-70, 000 subunit to the polymerase subunit prior to assaying with these templates had no effect on activity and it may be that a preincubation period is required before the two 3350 <br><br> Nucleic Acids Research <br><br> Figure 4 Activity of A2 and C enzymes on poly (dA). oligo (dT)jO (A:T = <br><br> 20:1) as a function of time. 50 ,ul samples were withdrawn at various times from a 0. 6 inl incubation mix at 30?C and added to 0. 5 ml 0. 1 M sodium <br><br> pyrophosphate containing 100 ,ug/ml native <span class="yellow">calf</span> thymus DNA and processed for counting in the usual manner6. The mix contained 1 mM dithiothreitol, 50 mM tris HC1, pH 7.8 10 mM MgC12, 0. 1 mM [3H] dTTP (15 cpm/pmol), 312.5 ,ug <span class="yellow">bovine</span> serum albumin, 5 Fg poly (dA). oligo (dT)  (A:T = 20:1) and enzyme protein (*-*) 3.3 units reconstituted A2, (o-o) 16. 0 units urea derived C enzyme. <br><br> subunits become fully associated. After the lag phase is over the C enzyme is still less active on these template-initiators than A enzyme . That is the A enzyme seems capable of elongating the initiator faster than the C enzyme. Experiments to determine whether the differences in rates of elongation of these template-initiator complexes are differences in processivity of the <br><br> enzymes under the different pH and divalent cation conditions are under way. <br><br> ACKNOWLEDGEMENTS <br><br> We thank the Medical Research Council for a research grant. <br><br> Communications concerning the paper should be sent to:<br><br> Dr. A. M. Holmes, Department of Biochemistry, University of Strathclyde, The Todd Centre, 31 Taylor Street, Glasgow G4 ONR, U.K. <br><br> 3351 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> (1)    Bollum, F.J. (1975). Prog. Nucleic Acid Res. Mol. Biol. 15, <br><br> 109-144. <br><br> (2)    Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1976). Eur. <br><br> J. Biochem. 62, 229-235. <br><br> (3)    Matsukage, A., Sivarajan, M. and Wilson, S.H. (1976). Biochem<br><br> istry 15, 5305-5314. <br><br> (4)    Fisher, P.A. and Korn, D. (1977). J. Biol. Chem. 252, 6528<br><br> 6535. <br><br> (5)    Fichot, O., Pascal, M., Mechali, M. and De Recondo, A, -M. <br><br> (1979). Biochim. Biophys. Acta, 561, 28-41. <br><br> (6)    Holmes, A.M., Hesslewood, I.R. and Johnston, I.R. (1974). Eur. <br><br> J. Biochemn. 43, 487-499. <br><br> (7)    Noy, G.P. and Weissbach, A. (1977). Biochim. Biophys. Acta, <br><br> 447, 70-83. <br><br> (8)    Craig, R.K. and Keir, H. M. (1975). Biochem. J. 145, 225-232. <br><br> (9)    Matsukage, A., Bohn, E.W. and Wilson, S.H. (1974). Proc. Natl. <br><br> Acad. Sci. U.S.A. 71, 578-582. <br><br> 110)   Hachmann, H.J. and Lezius, A.G. (1975). Eur. J. Biochem. 50, <br><br> 357- 366. <br><br> (11)   Brakel, C.L. and Blumenthal, A.B. (1977). Biochemistry, 16, <br><br> 3137-3143. <br><br> (12)   Momparler, R.L., Rossi, M. and Labitan, A. (1973). J. Biol. <br><br> Chem. 248, 285-293. <br><br> (13)   Yoshida, S., Konda, T. and Ando, T. (1974). Biochim. Biophys. <br><br> Acta, 353, 463-474. <br><br> (14)   Holmes, A.M., Hesslewood, I.P., Wickremasinghe, R.G. and <br><br> Johnston, I.R. (1977). Biochem. Soc. Symp. 42, 17-36. <br><br> (15)   De Jong, A., Van der Vliet, P. and Jansz, H.S. (1977). Biochim. <br><br> Biophys. Acta, 476, 156-165. <br><br> (16)   Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1975). <br><br> Nature (London) 255, 420-422. <br><br> (17)   Bollum, F. J. (1966). Procedures in Nucleic Acid Research <br><br> (Cantoni, G.L. and Davis, D.R. eds). pp 577-583, Harper and Row, New York. <br><br> (18)   Barrett, A.J. (1975). Proteases and Biological Control pp 467-482, <br><br> Cold Spring Harbor Laboratory, Cold Spring Harbor. <br><br> (19)   Richards, F. M. and Vithayathil, P.J. (1960). Brookhaven Symp. <br><br> Biol. 13, 115-134. <br><br> (20)   Lowe, P.A., and Malcolm, A.B.D. (1976). Eur. J. Biochem. 64, <br><br> 177- 188. <br><br> (21)   Hesslewood, I.P., Holmes, A.M., Wakeling, W.F. and Johnston, <br><br> I.R. (1978). Eur. J. Biochenr.. 84, 123-131. <br><br> (22)   Mechali, M. and De Recondo, A. -M. (1978). Biochim. Biophys. <br><br> Res. Comm, 82, 255-264. <br><br> (23)   Chang, L.M.S., Cassani, G.R. and Bollum, F.J. (1972). J. Biol. <br><br> Che. 247, 7718-7723. <br><br> (24)   Wickremasinghe, R.G. and Johnston, I.R. (1974). Biochim. <br><br> Biophys. Acta, 361, 37-52. <br><br> 3352 <h3>pmcA1075922</h3>Tissue-dependent isoforms of mammalian Fox-1 homologs are associated with tissue-specific splicing activities
Abstract
An intronic hexanucleotide UGCAUG has been shown to play a critical role in the regulation of tissue-specific alternative splicing of pre-mRNAs in a wide range of tissues. Vertebrate Fox-1 has been shown to bind to this element, in a highly sequence-specific manner, through its RNA recognition motif (RRM). In mammals, there are at least two Fox-1-related genes, ataxin-2 binding protein 1 (A2BP1)/Fox-1 and Fxh/Rbm9, which encode an identical RRM. Here, we demonstrate that both <span class="blue">mouse</span> Fxh and A2BP1 transcripts undergo tissue-specific alternative splicing, generating protein isoforms specific to brain and muscle. These tissue-specific isoforms are characterized for their abilities to regulate neural cell-specific alternative splicing of a cassette exon, N30, in the non-muscle myosin heavy chain II-B pre-mRNA, previously shown to be regulated through an intronic distal downstream enhancer (IDDE). All Fxh and A2BP1 isoforms with the RRM are capable of binding to the IDDE in vitro through the UGCAUG elements. Each isoform, however, shows quantitative differences in splicing activity and nuclear distribution in transfected cells. All Fxh isoforms and a brain isoform of A2BP1 show a predominant nuclear localization. Brain isoforms of both Fxh and A2BP1 promote N30 splicing much more efficiently than do the muscle-specific isoforms. Skeletal muscles express additional isoforms that lack a part of the RRM. These isoforms are incapable of activating neural cell-specific splicing and, moreover, can inhibit UGCAUG-dependent N30 splicing. These findings suggest that tissue-specific isoforms of Fxh and A2BP1 play an important role in determining tissue specificity of UGCAUG-mediated alternative splicing.<br><br>INTRODUCTION
Alternative splicing of pre-mRNA is one of the fundamental mechanisms for the regulation of gene expression in higher eukaryotes (1,2). Developmentally regulated, cell type- or tissue-specific, and signal-induced alternative splicing of pre-mRNAs takes place in multicellular organisms throughout their lifetimes. Misregulation or abnormalities in pre-mRNA splicing, in some instances, leads to cellular dysfunctions found in <span class="yellow">human</span> and animal diseases (3–5). Using various model systems of regulated alternative splicing, a number of pre-mRNA features that influence alternative splice site selection have been defined (1,2). These include enhancer and repressor RNA sequences located in exons and introns. Identification of RNA-binding proteins targeting these cis-regulatory elements is currently in progress. In vertebrates, participation of the SR family proteins and hnRNP proteins, such as PTB and hnRNPA1, in alternative splicing regulation via binding to the cis-regulatory elements have been shown in many tissue-specific splicing models (6,7). Although these RNA-binding proteins are ubiquitously expressed, their different abundance in different cells, differences in their post-translational modifications in different cellular contexts and their different abilities to assemble multiprotein complexes in different pre-mRNA contexts are thought to contribute to the determination of cell type-specific patterns of alternative splicing. For the last few years, tissue-specific and tissue-enriched RNA-binding proteins have begun to be identified as splicing regulators. These include brain-specific (or enriched) Nova-1, nPTB (brPTB) and some of the CELF family proteins (8–12). Discovery of these proteins has had a great impact on studies aimed at understanding the molecular mechanisms of alternative splicing regulation.
One of the intronic cis-elements, which are involved in tissue-specific or differentiation stage-dependent regulation of alternative splicing, is the hexanucleotide UGCAUG. The importance of this element was originally recognized in fibronectin pre-mRNA by Huh and Hynes (13). Since then, alternative splicing specific to a variety of tissues or cell types, including neural cells, muscles, epithelial cells and erythrocytes, has been shown to be modulated via this element (14–21). Recently, Jin et al. (21) discovered that a <span class="yellow">zebrafish</span> homolog of <span class="yellow">Caenorhabditis elegans</span> Fox-1 (22) could bind specifically to the pentanucleotide GCAUG by in vitro selection from randomized RNA sequences. This pentanucleotide is almost identical to the hexanucleotide UGCAUG except for the first U. Moreover, the <span class="yellow">zebrafish</span> Fox-1 homolog, as well as the <span class="blue">mouse</span> Fox-1 homolog, are capable of repressing the inclusion of an alternative cassette exon of the ATP synthase F1γ pre-mRNA via binding to GCAUG, which mimics muscle-specific exclusion of this exon. This <span class="blue">mouse</span> homolog is identical to the ataxin-2 binding protein 1 (A2BP1), which has been previously cloned in <span class="yellow">humans</span> and <span class="yellow">mice</span> as the cDNA encoding a protein, which interacts with ataxin-2, the product of the causative gene for spinocerebellar ataxia type 2 (23,24). In addition to A2BP1/Fox-1, another <span class="blue">mouse</span> homolog of <span class="yellow">C.elegans</span> Fox-1, Fxh, has been independently cloned as a cDNA, which is induced by androgen in motor neurons (25). Of note is that A2BP1/Fox-1 and Fxh share an identical RNA recognition motif (RRM) at the amino acid level. Therefore, two genes in the <span class="blue">mouse</span> genome encode homologs of nematode Fox-1. According to the names given by the first cDNA cloning, we used the nomenclature of A2BP1 and Fxh in this report. A2BP1 and Fxh have been named A2BP1 and Rbm9, respectively, in the <span class="yellow">human</span> and <span class="blue">mouse</span> genomes.
We have been studying regulatory mechanisms of neural cell-specific alternative splicing using the non-muscle myosin heavy chain II-B (NMHC-B) gene as a model system (14,26). NMHC-B mRNA is expressed ubiquitously. However, an alternative exon, N30, which encodes a 30 nt coding sequence, is included in the mRNAs from some neural cells, but is skipped in those from all other cells in mammals and birds (27,28). In cultured cells, a switch in N30 splicing from exclusion to inclusion can be seen in neural retinoblastoma Y79 cells during the post-mitotic and differentiated stages triggered by butyrate treatment. We have previously defined an intronic distal downstream enhancer (IDDE), which confers neural cell specificity on N30 inclusion, using this cell line (14). The IDDE includes two copies of UGCAUG. Mutation of these hexanucleotides results in N30 skipping in post-mitotic differentiated Y79 cells.
In this study, we investigated the possible involvement of A2BP1 and Fxh in the regulation of N30 splicing. To this end, we have isolated cDNA clones for A2BP1 and Fxh from brain and muscles. cDNA cloning revealed the existence of tissue-specific (enriched) isoforms of both A2BP1 and Fxh. Of importance, different isoforms of A2BP1 and Fxh show different activities with respect to N30 splicing as well as different subcellular localizations.<br><br>MATERIALS AND METHODS
Database disposition
The sequences reported in this paper have been deposited in the GenBank database with accession numbers AY659951 (F011), AY659952 (F411), AY659953 (F402), AY659954 (A016), AY659955 (A030), AY659956 (A713), AY659957 (A715) and AY659958 (A704).<br><br>RNA preparation and RT–PCR
Total RNAs were isolated from <span class="blue">mouse</span> tissues and cultured cells using an RNA isolation kit (Stratagene) or an RNeasy mini kit (Qiagen). To obtain the full-length coding regions of cDNAs for Fxh and A2BP1, RT–PCRs were performed using Superscript II RNase H− reverse transcriptase (Invitrogen) and Pfu Turbo DNA polymerase (Stratagene). The PCR primers used to obtain all Fxh cDNAs were 5′-ctcaggcctccactagttATGGAGAAAAAGAAAATGGTAACTC-3′ and 5′-ctcaggcctcctctagaaGTAGGGGGCAAATCGGCTGTA-3′. The upstream primers for the brain (A016 and A030) and muscle (A713, A715 and A704) A2BP1 cDNAs were 5′-ctcaggcctccactagtgATGAATTGTGAAAGAGAGCAGCT-3′ and 5′-ctcaggcctccactagtcATGTTGGCGTCGCAAGGAGTCC-3′, respectively, and the downstream primer for all A2BP1 cDNAs was 5′-ctcaggcctcctctagagATATGGAGCAAAACGGTTGTATCC-3′. Lower case letters represent adapter sequences including restriction enzyme sites. 5′ Rapid amplification of cDNA ends (RACE) was performed using Marathon-Ready cDNA (BD Biosciences Clontech). For the analysis of minigene and NMHC-B mRNAs, RT–PCRs were performed as described previously (14,26). Sequences of primers P1–P9 shown in Figures 1, 4 and 6 are as follows: P1, 5′-AATTCACCCAGCAACCAGAAT-3′; P2, 5′-TAGAGGGATGTAAGTGTTGATGCC-3′; P3, 5′-CAGAGGGCGGACAGTGTATGGT-3′; P4, 5′-GGCGGCAGGGGCGAGGGCAT-3′; P5, 5′-CCGTGGTCGCACCGTGTACAAC-3′; P6, 5′-CAGCGGCAGTGGCAGGGGTG-3′; P7, 5′-AGGAAGAAAGGACCATAATATTCC-3′; P8, 5′-CCTCCACCCAGCTCCAGTTGT-3′; and P9, 5′-CCTGTAGTTATTAAATCCTTCAAG-3′.<br><br>Preparation of expression constructs and minigenes
The cDNAs of Fxh and A2BP1 were introduced into a plasmid pCS3 + MT, which contains a myc-epitope, and its modified version pCS3 + MT + NLS, which in addition contains the nuclear localization signal (NLS) of the <span class="yellow">SV40</span> large T antigen (26). Minigenes G, J without the IDDE, and H with the wild-type IDDE are the same as minigenes W, D4 and Cm0, respectively in ref. (14). The 201 nt IDDEs with mutations were generated by recombinant PCR using the appropriate primers, which included mutated sequences. The hexanucleotide TGCATG sequences at the 5′ and 3′ sides were changed to GTTACT and ACCTAC, respectively.<br><br>Electrophoresis mobility shift assay
Template DNAs for in vitro RNA transcription were prepared by PCR using the wild-type and mutant IDDEs in the minigenes as templates and an upstream primer that included the T7 promoter sequence at the 5′ end. The probe and competitor RNAs were transcribed by T7 RNA polymerase in the presence of [α-32P]UTP and a trace amount of [35S]UTPαS, respectively, using a MAXIscript kit (Ambion). Mole concentrations of synthesized RNAs were estimated by radioactivities. Fxh and A2BP1 proteins with a myc tag were synthesized in vitro by using a TNT quick-coupled transcription/translation system (Promega) from pCS3 + MT constructs, which include the SP6 promoter. Binding reactions were carried out in a 10 μl mixture that contains 10 mM HEPES (pH 7.9), 2 mM MgCl2, 50 mM KCl, 5% glycerol, 0.5 mM DTT, 5 μg tRNA, 1 μl reticulocyte lysate reaction mixture and 15 fmol of probe, on ice for 20–30 min. An aliquot of 5 μg of heparin was added to the reaction 10 min before gel electrophoresis. The reaction mixtures were analyzed by electrophoresis in a 6% polyacrylamide gel using a 0.5× TBE buffer (Invitrogen).<br><br>Cell culture and transfection
The <span class="yellow">human</span> retinoblastoma cell line Y79 was cultured and transfected with DNAs as described previously (14,26). Total amounts of transfected DNAs were adjusted to be constant by addition of the empty vector. Either Lipofectin (Invitrogen) or Effectene transfection reagent (Qiagen) was used for the transfection. For stable transfection, the pCS3+MT expression constructs were co-transfected with a plasmid carrying a neomycin resistant gene and selected by 0.2 mM geneticin (Invitrogen). For differentiation of Y79 cells, cells were plated on the poly-d-lysine-coated plates and then treated with 2.0–2.5 mM sodium butyrate for 4–5 days. HeLa cells were cultured as described and transfected with DNA using Effectene reagent (14,26).<br><br>Immunoblot analysis
Samples that required both protein and mRNA analysis were split upon harvesting. Total cell proteins were subjected to SDS–PAGE and blotted as described previously (26). The primary antibodies used are monoclonal antibodies to a myc-epitope (Invitrogen) and green fluorescent protein (GFP) (Clontech). Binding of antibodies was detected with the SuperSignal System (Pierce) or ECL (Amersham).<br><br>Immunofluorescent microscopy
HeLa or Y79 cells grown in a four-chamber glass slide were transfected as described above. Cells were fixed with 10% formaldehyde 24–48 h after transfection, and permealized with 0.5% Triton X-100 then blocked with 5% <span class="yellow">goat</span> serum. Primary antibodies used were <span class="blue">mouse</span> anti-myc (Invitrogen), <span class="yellow">rabbit</span> anti-NMHC-B (29). Secondary antibodies were Alexa Fluor 594 <span class="yellow">goat</span> anti-<span class="blue">mouse</span> IgG and Alexa Fluor 488 <span class="yellow">goat</span> anti-<span class="yellow">rabbit</span> IgG (Molecular Probes). DAPI was used for DNA staining. Specimens were mounted in ProLong antifed kit (Molecular Probe). The images were collected using Leica SP confocal microscopy (Leica).<br><br>
RESULTS
Tissue-dependent isoforms of Fxh and A2BP1 and their subcellular distribution
The A2BP1 mRNAs are detected almost exclusively in brain and striated muscles (heart and skeletal muscles) in adult <span class="yellow">mice</span>, whereas the Fxh mRNAs are expressed in a wide variety of tissues with the highest expression in brain and heart [(21,25) and S. Kawamoto, unpublished data]. These expression profiles prompted us to characterize the mRNAs of A2BP1 and Fxh in brain and muscles. We have cloned cDNAs for two gene transcripts from brain, heart and skeletal muscles by RT–PCR and 5′ RACE. The isolated cDNAs are schematically presented with genomic structures in Figure 1 (for amino acid sequences see also Figure 8). Both gene transcripts are found to undergo tissue-specific alternative splicing. In both cases, brain and striated muscles express unique splice variants generated by mutually exclusive splicing of exons B40 and M43, respectively, which provide different coding sequences in the middle of the carboxyl half of the molecules. Southern blot analysis of the RT–PCR products of the Fxh mRNAs from the different tissues probed by oligonucleotides corresponding to B40 and M43 shows that M43 is exclusively used in heart and skeletal muscles, while B40 is predominantly used in brain (Figure 2A). Digestion of the RT–PCR products of the A2BP1 mRNAs with restriction enzymes unique to B40 and M43 demonstrates that B40 and M43 are used almost exclusively in brain and muscles, respectively (Figure 2B). A2BP1 contains an additional cassette type alternative exon A53, consisting of 53 nt. Inclusion and exclusion of exon A53 results in two different amino acid sequences at the C-terminal region owing to a frame shift. In the case of the A2BP1 gene, moreover, it appears that brain and striated muscle utilize alternative promoters, resulting in different amino acid sequences at the N-terminus. The 5′ RACE of skeletal muscle mRNAs yielded essentially a single species of sequence with 29 unique N-terminal amino acids. The 5′ RACE using brain mRNAs; however, yielded multiple products with multiple deduced amino acid sequences, all of which differ from the muscle amino acid sequence at the very N-terminus. Here, we focus our analysis on a clone containing nine unique amino acids at the N-terminus, which was obtained most frequently. Exon–intron organization of Fxh and A2BP1 shows remarkable similarities and a RRM is encoded in four exons (Figure 1B). Of note is that the significant amounts of Fxh and A2BP1 mRNAs from skeletal muscles are missing a part of the RRM by exon skipping. Typically, as shown in Figure 2C, they lack the 93 nt exon that encodes RNP1, one of the two most critical motifs of the RRM (30). Some of them lack almost the entire RRM (e.g. F402 in Figure 1A).
To examine the subcellular distribution of each isoform of Fxh and A2BP1, myc-tagged proteins were transiently expressed in cultured cells and immunostained with an anti-myc antibody. Initially, HeLa cells were used to investigate subcellular localization, since these cells are more suited for these studies. Representative confocal images are shown in Figure 3A. DAPI and anti-NMHC-B antibodies serve as markers for nuclei and cytoplasm, respectively. As noted, the ratio of protein distributed between nuclei and cytoplasm differs among the proteins. All isoforms of Fxh have a predominant nuclear localization. The brain isoform of A2BP1 without the A53 exon (A016) localizes to both nuclei and cytoplasm, whereas the other brain isoform with the A53 exon (A030) localizes predominantly to cytoplasm with only a minimum being in the nuclei. The relative amounts of both muscle isoforms of A2BP1 (A713 and A715) are somewhat between the amounts of the two brain isoforms. These data are summarized in Figure 1A. The subcellular distribution of representative isoforms (F011, A016 and A030) was also examined in retinoblastoma Y79 cells, which were used as host cells for the transfection experiments in order to characterize splicing activities of Fxh and A2BP proteins (see below). Although Y79 cells have a spherical shape and have only thin cytoplasm, exogenously expressed isoforms of Fxh and A2BP1 show a similar subcellular distribution as in HeLa cells (Figure 3B). F011 localizes predominantly to nuclei, A016 to both nuclei and cytoplasm and A030 predominantly to cytoplasm.<br><br>Specific interaction of Fxh and A2BP1 with IDDE via a hexanucleotide UGCAUG
Fxh and A2BP1 share an identical RRM and A2BP1 has been reported to bind specifically to the pentanucleotide GCAUG through this RRM (21). We have previously reported that the IDDE of the NMHC-B transcript, which is indispensable for the regulation of neural cell-specific cassette type exon N30 splicing, has two copies of GCAUG (14). Therefore, we investigated whether Fxh and/or A2BP1 bound to the IDDE. Electrophoretic mobility shift assays (EMSAs) were carried out using labeled IDDE and in vitro transcribed and translated Fxh and A2BP1, which include a myc-epitope. Since all Fxh and A2BP1 isoforms, except F402 and A704, contain the identical RRM, representative isoforms were analyzed. The expression of F011 in reticulocyte lysate causes the formation of a RNA–protein complex whose migration shift distinguishes it from those of the control reticulocyte lysate (C in Figure 4B, lanes 3 and 10). The unlabeled wild-type IDDE competes with the probe efficiently for the formation of the specific complex C (Figure 4B, lane 4). On the other hand, the mutant mc (Figure 4A), which has a mutation in both copies of UGCAUG, does not (Figure 4B, lane 7). The mutant ma, which has a mutation in the hexanucleotide at the 5′ side, shows less efficient competition, compared with mb, which has a mutation in the 3′ side of the hexanucleotide (Figure 4B, lanes 5 and 6), indicating that the nucleotides at the 5′ side are more important than those at the 3′ side. The presence of an anti-myc antibody, but not a non-specific antibody, inhibits the formation of the specific complex (Figure 4B, lane 8). Synthesis of the full-length F011 protein using a reticulocyte lysate and specificity of the myc antibody for the expressed protein are verified by immunoblot analysis (Figure 4B, lanes 11 and 12). A016 and A030 show essentially identical results to those with F011 (data not shown). These results indicate that Fxh and A2BP1 can bind to the IDDE and that the hexanucleotide UGCAUG is required for their binding.<br><br>Fxh and A2BP1 enhance N30 inclusion in a UGCAUG-dependent manner
To study whether Fxh and A2BP1 regulate neural cell-specific splicing via binding to UGCAUG, the Fxh and A2BP1 expression constructs were co-transfected into retinoblastoma Y79 cells with a number of the reporter minigene constructs, which include the wild-type or a mutant version of UGCAUG in the IDDE (Figure 4A). Minigenes H and J consist of the exons E5, N30 and E6 and their flanking introns with some deletions in the introns. The IDDE with or without mutations in the hexanucleotide is included or excluded between N30 and E6. As described above, each isoform of Fxh and A2BP1 enters the nucleus to a different extent. To see the effects of different proteins on N30 splicing itself, independent of their differential properties in nuclear localization, an exogenous NLS was added to the expressed proteins. Since Fxh and A2BP1 proteins contain the identical RRM, and F011, A016 and A030 all show the same UGCAUG-dependent binding to the IDDE in vitro, F011 and A030 were used for these experiments. Host cells Y79 at the proliferating stage exclude the N30 exon in ∼90% of the endogenous NMHC-B mRNAs (e.g. see Figure 6A, lane 1).
As shown in Figure 4C, the mRNAs derived from minigene J exclude N30 without exogenous expression of Fxh or A2BP1 in Y79 cells, similar to the endogenous NMHC-B mRNAs (Figure 4C, upper panel, lanes 1–5). However, in the presence of exogenous expression of F011 and A030, the N30 inclusion is increased (Figure 4C, upper panel, lanes 7 and 12). The N30 inclusion is absolutely dependent on the presence of the IDDE (Figure 4C, upper panel, lanes 6 and 11). Moreover, mutation of either one of the two copies of UGCAUG (ma, mb) abolishes the inclusion of N30 (Figure 4C, upper panel, lanes 8–10, 13–15). In the context of minigene H, which contains shorter introns, the larger extent of N30 inclusion is induced by either F011 or A030 overexpression (Figure 4C, middle panel, lanes 7 and 12). The N30 inclusion of the minigene H mRNAs also depends on the presence of the IDDE (Figure 4C, middle panel, lanes 6 and 11). Mutation of both copies of UGCAUG (mc) results in a complete loss of N30 inclusion (Figure 4C, middle panel, lanes 10 and 15). The mutant ma shows stronger inhibition of N30 inclusion compared with mb (Figure 4C, middle panel, lanes 8 and 9). This observation is consistent with the competition experiments of the EMSA shown in Figure 4B, indicating that the 5′ hexanucleotide is more important than the 3′ hexanucleotide for binding of Fxh or A2BP1 to the IDDE as well as an activation of N30 splicing. Comparable amounts of F011 and A030 are expressed in each transfection as verified by immunoblots using an anti-myc antibody (Figure 4C, lower panel, lanes 6–17).
Since minigenes J and H lack a portion of the intron between exons N30 and E6 and, therefore, the IDDE is located ∼100 nt downstream of N30, instead of 1.5 kb as in the native gene, we also analyzed the effects of Fxh and A2BP1 on the N30 splicing of the wild-type minigene G, which includes full-length introns among E5, N30 and E6 (Figure 4A). As shown in Figure 4C (right panel), both F011 and A030 are capable of promoting N30 inclusion, with F011 showing a higher activity (Figure 4C, lanes 16 and 17). Although A030 can promote N30 inclusion as efficiently as F011 in the contexts of the minigene J and H transcripts, it can do so less efficiently than F011 in the context of the minigene G transcript. Interpretation of this observation will be discussed below.
Taken together, Fxh and A2BP1 can activate N30 inclusion in an IDDE-dependent manner. The hexanucleotide motif UGCAUG is indispensable for this activation.<br><br>Differential activities of alternatively spliced isoforms of Fxh and A2BP1 in promoting N30 inclusion
Both Fxh and A2BP1 mRNAs are expressed in brain and A2BP1 is also expressed in striated muscles and Fxh is expressed in an even wider variety of tissues. As demonstrated above, however, both Fxh and A2BP1 transcripts undergo tissue-dependent alternative splicing, producing muscle-specific and brain-enriched isoforms. Therefore, the relative activity of individual isoform of Fxh and A2BP1 in promoting N30 inclusion was compared using minigene G.
First, in order to evaluate the relative specific activity of each isoform in the splicing reaction separately from its ability to localize to nuclei, an exogenous NLS was included in the expressed protein to equalize the nuclear concentration of the expressed protein in these experiments. Essentially, all of the expressed proteins with the exogenous NLS are localized to the nucleus (data not shown). Therefore, the relative nuclear concentrations of the expressed proteins can be estimated easily by immunoblots. As shown in immunoblots in Figure 5A, similar quantities of proteins are expressed in a dose-dependent manner. Expression of the brain isoform F011 causes a dose-dependent increase in N30 inclusion and the extent of N30 inclusion reaches a plateau with ∼85% of the mRNAs including N30. In contrast, inclusion of N30 promoted by F411, the predominant isoform from skeletal muscles, reaches a plateau with only 40% of the mRNAs. With respect to A2BP1, the brain isoform A030 shows higher activity in N30 inclusion than the muscle isoform A715, which shows almost no activation, as shown in Figure 5A. Including additional isoforms, the activities of individual isoforms with an exogenous NLS in promoting N30 inclusion are shown in Figure 5B (lanes 8–13) and are also summarized in Figure 1A (nls). Special care was taken to ensure that a similar quantity of each protein was expressed and, if not, different amounts were tested to obtain comparable expression. In the presence of the exogenous NLS (nls), F011 and A016 show the highest activities among all isoforms tested. A30 shows a considerably lower activity than A016. Each of the muscle isoforms (F411, A713 and A715) has a lower activity than that of their brain counterparts (F011, A016 and A030, respectively). As expected, isoforms lacking a part or all of the RRM (F402 and A704) have no activity for N30 inclusion.
Next, the splicing activities of the wild-type proteins without an exogenous NLS were examined. Representative data are shown in Figure 5B (lanes 2–7) and the relative activities are summarized in Figure 1A (wt). The protein amounts detected by immunoblots in these experiments represent the total amounts of the proteins distributed to both the nuclei and the cytoplasm. The splicing activities of the wild-type proteins are consistent with the activities that combine the splicing activities of the proteins with the exogenous NLS and the activities of the native proteins to localize to nuclei. In the absence of the exogenous NLS (wt), the brain isoforms F011, and A016 to a lesser extent, are still capable of activating N30 inclusion efficiently. The muscle isoforms for both Fxh and A2BP1 (F411, A713 and A715), as well as the brain isoform A030, show only minimal activities. Therefore, F011 and A016 appear to have the most physiological relevance to N30 splicing activation.<br><br>Overexpression of Fxh and A2BP1 activates N30 inclusion of endogenous NMHC-B mRNAs
The <span class="yellow">human</span> NMHC-B gene consists of 41 constitutive exons and 3 alternative exons. Its pre-mRNA is ∼156 kb in length and it is much more complex than the pre-mRNA from the minigenes. In addition, the minigene pre-mRNAs are driven by a heterologous promoter. Therefore, we next examined if Fxh and A2BP1 were capable of promoting N30 inclusion of the endogenous transcript. Y79 cells were stably transfected with the expression construct for F011 or A016. Both F011 and A016 are enriched in the brain and show higher activation of N30 inclusion in the minigene transcripts. mRNAs encoding endogenous NMHC-B were analyzed by using RT–PCR. As shown in Figure 6, the inclusion of exon N30 with and without another alternative exon, R18, in the endogenous mRNAs is markedly increased in the clones which were stably transfected with the construct for F011 or A016 containing an exogenous NLS (Figure 6, lanes 4 and 5). Although to a lesser extent, transfection of the wild-type constructs without an exogenous NLS also results in a significant increase in N30 inclusion (Figure 6, lanes 2 and 3). Thus, exogenously expressed Fxh and A2BP1 are capable of activating N30 inclusion not only in the transcripts from the minigenes, but also in those from the native NMHC-B gene in Y79 cells.<br><br>Fxh may cooperate with other factor(s) to promote N30 inclusion
To address the role of endogenous Fxh and its potential interaction with other proteins in promoting N30 inclusion, we made use of an isoform of Fxh, F402, which lacks the RRM. The mutant proteins lacking an RRM for other RNA-binding proteins have previously been reported to function in a dominant-negative fashion and inhibit the activities of the wild-type proteins (26,31). Therefore, the effects of F402 on the N30 inclusion of minigene H mRNAs were examined in the context of the Y79 cells treated with butyrate. Upon butyrate treatment, as reported previously, Y79 cells enter in a post-mitotic and differentiated stage, and importantly, the endogenous as well as the minigene mRNAs in those cells include N30 to a large extent, unlike those in the untreated cells that predominantly exclude N30 (14). As shown in Figure 7, in the absence of exogenous expression of F402, large quantities of the mRNAs derived from minigene H-wt, which contains the wild-type IDDE, include N30 (Figure 7, upper panel, lane 4). In contrast, only small quantities of N30 inclusion are detected in the mRNAs from minigene H-mc, which has mutations in both copies of UGCAUG in the IDDE (Figure 7, upper panel, lane 8). Therefore, the butyrate-treated Y79 cells contain factor(s), which are capable of activating the UGCAUG-dependent N30 inclusion. Co-transfection of the wild-type minigene H-wt with the F402 expression construct causes a dose-dependent inhibition of N30 inclusion (Figure 7, lanes 1–3), indicating that F402 has an antagonistic effect on the endogenous factors with respect to N30 inclusion. Notably, the UGCAUG-independent N30 inclusion seen in the mutant minigene H-mc is not affected by the co-expression of F402 (Figure 7, lanes 5–7), indicating that the inhibitory activity of F402 depends on the UGCAUG element. The parallel experiment with the F011 expression construct does not show a significant effect on N30 splicing in either the wild-type or mutant minigene (Figure 7, lanes 9–16). This may be due to the fact that N30 inclusion of the wild-type minigene mRNAs has already reached a maximal level and is consistent with the idea that butyrate-treated Y79 cells contain sufficient amounts of protein(s) functionally equivalent to F011, which can promote N30 inclusion in a UGCAUG-dependent manner. Since F402 does not bind to the UGCAUG element, and has a UGCAUG-dependent antagonistic effect on N30 splicing, it most probably disrupts protein–protein interactions of the endogenous proteins that are required for the UGCAUG-dependent activation of N30 splicing. Endogenous Fxh (and/or A2BP1) would be a good candidate whose function could be antagonized by exogenously expressed F402. Thus, this observation suggests that the endogenous Fxh has an effect on the activation of N30 splicing and that other proteins cooperate with Fxh for N30 activation. Since muscle cells express the RRM-defective isoforms to a significant extent (Figure 2C) and the wild-type F402 localize to nuclei efficiently, this also raises the possibility that the RRM-defective isoforms may have an inhibitory function on the N30 splicing in muscle cells.<br><br>
DISCUSSION
Two major findings are described in this report. First, Fxh and A2BP1 facilitate neural cell-specific inclusion of the cassette-type exon via binding to the specific intronic sequence UGCAUG. In addition to a minigene model system, Fxh and A2BP1 are capable of facilitating N30 inclusion of the endogenous pre-mRNA. This result provides an important demonstration of physiological relevance and supports the notion that the NMHC-B pre-mRNA is likely to be the true target for Fxh or A2BP1-mediated regulation. However, whether the endogenous Fxh or A2BP1 regulates endogenous NMHC-B pre-mRNA splicing needs to be determined in a future study. In vertebrates, small interfering RNAs and gene targeting strategies have recently been used successfully to address the roles of endogenous splicing regulators in alternative splicing of endogenous target pre-mRNAs (8,32–36). A second and more novel finding is the identification of tissue-specific isoforms of Fxh and A2BP1 with different splicing activities as well as different subcellular localizations. This finding raises the possibility that the products of the Fxh and A2BP1 genes can contribute to a mechanism as to how tissue specificity of alternative splicing is achieved.
Many splicing factors are detected not only in the nuclei, but also in the cytoplasm (37). They are shuttling between the nucleus and the cytoplasm and, in some instances, extracellular stimuli trigger changes in subcellular distribution of these proteins. Such translocations have been reported for hnRNPA1 and PTB (38,39). Moreover, a number of RNA-binding proteins have been demonstrated to play a role in multiple steps during gene expression in different subcellular compartments, such as pre-mRNA processing in nuclei, mRNA export from nuclei to cytoplasm and mRNA localization, stability and translation in cytoplasm (37,40). Therefore, not surprisingly, Fxh and A2BP1 isoforms were found to be distributed in both the nuclei and the cytoplasm in HeLa and Y79 cells. However, the relative ratios of proteins distributed between the two subcellular compartments at steady-state differ among the isoforms. In agreement with Jin et al. (21), substantial amounts of the brain isoform A016 are detected in nuclei. Other A2BP1 isoforms, the brain isoform A030 and the muscle isoforms A713 and A715, are only poorly detected in nuclei. This observation is consistent with the reports where endogenous A2BP1 in cerebellar Purkinje cells, hippocampus neurons and cardiac myocytes were shown to be localized essentially to the cytoplasm (23,24). Thus, inclusion and exclusion of A53 and differences in the very N-terminal sequences results in A2BP1 isoforms with a distinct subcellular localization. It is likely that A2BP1 proteins have multiple roles, involving both nuclear and cytoplasmic events. In contrast, all three Fxh isoforms predominantly localized to the nuclei. Therefore, in terms of their localization, Fxh proteins are better candidates for regulators of the pre-mRNA splicing that takes place in nuclei. Of note, however, our preliminary results of 5′ RACE, as well as the EST database, detect multiple 5′ end sequences for both Fxh and A2BP1 mRNAs, which are presumably generated by alternative promoters and alternative splicing. The diversity of the 5′ end cDNA sequences leads to the generation of a number of unique N-terminal amino acid sequences. Therefore, this study does not exclude the possible existence of other isoforms with different subcellular localizations for both Fxh and A2BP1. Our study also does not exclude the possibility that some of the isoforms translocate between the nucleus and the cytoplasm following stimuli.
The main aim of this study is to determine the relative activities of tissue-dependent isoforms of Fxh and A2BP1 in neural cell-specific and UGCAUG element-dependent alternative splicing. To obtain an indication of the relative specific activity of each isoform in transfected cells, the same amounts of the expressed proteins should be available for the splicing reaction in the nuclei. For this reason, an exogenous NLS was included in the expressed proteins. Essentially, all of the expressed proteins with the exogenous NLS localized to nuclei. Thus, the amounts of the expressed proteins determined by immunoblots represent the nuclear concentrations. The analysis using the proteins expressed with the exogenous NLS allowed us to compare directly the splicing activities of these proteins. Furthermore, this analysis also allowed us to define the critical regions of the proteins for splicing activation.
As shown in Figure 5, the splicing activities of the various isoforms of Fxh and A2BP1 are intrinsically different, regardless of the subcellular localization properties of the wild-type proteins. Among the isoforms tested in this study, F011 and A016, which include B40, are found to have higher activities in promoting N30 inclusion. When the primary amino acid sequences, outside of the RRM, of these two proteins are compared, the C-terminal regions (amino acids 190–377 of F011) show a higher homology with 71% identity, whereas the N-terminal regions (amino acids 1–112 of F011) show only 53% identity. The C-terminal region includes four subregions of nearly identical stretches of amino acids (Figure 8, I–IV). One subregion (II) includes 13 amino acids encoded by exon B40. Substitution of this subregion with exon M43 in Fxh causes substantial changes in amino acid sequences resulting in only a 21% identity in this region between F011 and F411. Another subregion (IV) is located at the C-terminal end and A030 lacks this homologous region by the inclusion of exon A53, which results in a frame shift. Since F411 and A030 show poor splicing activation compared with F011 and A016, respectively, these two subregions of F011 and A016 appear to serve as activation domains, presumably by interacting with other proteins. This notion is supported by the finding that the RRM-lacking isoform F402, which includes the same subregions II and IV as F011, functions apparently as a dominant-negative mutant to the wild-type Fxh, consistent with the interpretation that the mutant and wild-type are competing to interact with other protein(s). To date, Fyn tyrosine kinase and estrogen receptor-α have been reported to interact with Fxh, and ataxin-2 with A2BP1 (23,41,42). Whether these proteins participate in the regulation of pre-mRNA splicing is currently unknown. Of interest, A030 with the exogenous NLS activates N30 splicing as efficiently as F011 in the pre-mRNAs derived from minigenes J and H, which contain the shorter intron, whereas this isoform poorly activates N30 splicing in the pre-mRNA from minigene G, which contains the full-length intron. This observation implies that the interactions of A030 with different factors are required in the different pre-mRNA contexts. Therefore, the isoforms of Fxh and A2BP1 described here may have different effects on other UGCAUG-regulated alternative splicing.
The involvement of the hexanucleotide UGCAUG in regulated alternative splicing has been experimentally demonstrated in a number of neural cell-specific, as well as other tissue-specific, model systems (13–21). This hexanucleotide element plays a role, in most cases, as an enhancer in regulating alternative splicing of cassette-type exons as well as mutually exclusive exons. Furthermore, computational analysis has revealed that UGCAUG is over-represented in the introns in which splicing is regulated, compared with the constitutively spliced introns (43). This analysis also pointed to the UGCAUG element as playing a role in the regulation of tissue-specific alternative splicing in a wide range of tissues, but not in specific tissues.
To date, KH-type splicing regulatory protein (KSRP) (44), A2BP1 (21) and Fxh (this study) are known to be capable of binding to UGCAUG. KSRP is expressed ubiquitously and tissue-specific variants of this gene have not been described so far. In this study, we have described the existence of tissue-dependent isoforms of Fxh and A2BP1, which, while not identical in some areas of the molecule, may contain the same RRM. The physiological relevance of these isoforms is that they have different splicing activities and different subcellular localizations. The brain isoforms promote N30 inclusion more efficiently than the muscle isoforms of both Fxh and A2BP1. The isoforms lacking the RRM are normally expressed to a significant extent in skeletal muscles. This isoform is incapable of activating N30 splicing and, moreover, can inhibit N30 inclusion. The properties of these isoforms are consistent with Fxh and A2BP1 acting as regulators for N30 splicing, since N30 is included in neuronal cells, but excluded in muscles. Therefore, despite the tissue-independent occurrence of UGCAUG as a regulatory element, given the tissue-dependent isoforms of the UGCAUG-binding proteins (Fxh and A2BP1) with different activities, the hexanucleotide UGCAUG could confer tissue specificity on regulated splicing. One of the major problems in understanding the mechanisms responsible for alternative pre-mRNA splicing is the manner in which tissue specificity is determined. In vertebrates, to date, only a few tissue-specific proteins have been identified as splicing regulators (8–12). Here, we have shown that the tissue-dependent isoforms of the sequence-specific RNA-binding proteins, which themselves are generated by alternative splicing, have different activities in tissue-specific alternative splicing of target pre-mRNA. Therefore, these isoforms play a role in the determination of tissue specificity of target pre-mRNA splicing. The discovery of these tissue-dependent isoforms of the UGCAUG-binding proteins with different splicing activities adds an important new dimension to the molecular mechanisms responsible for regulating tissue-dependent alternative splicing mediated via UGCAUG.<br><br>
<h3>pmcA1779441</h3>Identification of a <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts
Abstract
Increased bone resorption mediated by osteoclasts causes various diseases such as osteoporosis and bone erosion in rheumatoid arthritis (RA). Osteoclasts are derived from the monocyte/macrophage lineage, but the precise origin remains unclear. In the present study, we show that the purified CD16- <span class="yellow">human</span> peripheral blood monocyte subset, but not the CD16+ monocyte subset, differentiates into osteoclast by stimulation with receptor activator of NF-κB ligand (RANKL) in combination with macrophage colony-stimulating factor (M-CSF). Integrin-β3 mRNA and the integrin-αvβ3 heterodimer were only expressed on CD16- monocytes, when they were stimulated with RANKL + M-CSF. Downregulation of β3-subunit expression by small interfering RNA targeting β3 abrogated osteoclastogenesis from the CD16- monocyte subset. In contrast, the CD16+ monocyte subset expressed larger amounts of tumor necrosis factor alpha and IL-6 than the CD16- subset, which was further enhanced by RANKL stimulation. Examination of RA synovial tissue showed accumulation of both CD16+ and CD16- macrophages. Our results suggest that peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct responses to RANKL. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for osteoclastogenesis. Blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Introduction
Rheumatoid arthritis (RA) is an autoimmune disease characterized by chronic inflammation and proliferation of the synovium in multiple joints. A large number of inflammatory cells, including T cells, B cells, macrophages and dendritic cells, accumulate in the affected synovium, and these inflammatory cells, together with fibroblast-like synoviocytes, express various cytokines, such as tumor necrosis factor alpha (TNFα), IL-6 and receptor activator of NF-κB ligand (RANKL), which are known to induce differentiation and activation of osteoclasts. The inflammatory synovial tissue, known as pannus, invades the articular bone and causes focal bone erosion, which is the hallmark of RA. Histopathologically, osteoclasts are present at the interface of the pannus and bone. Interestingly, the deletion of RANKL or c-Fos gene, which is important for osteoclastogenesis, results in minimal bone destruction in <span class="blue">mouse</span> models of arthritis [1,2]. Furthermore, other studies indicated that inhibition of osteoclastogenesis by osteoprotegerin, a decoy receptor for RANKL, limits bone destruction in experimental models of arthritis. These studies suggest that osteoclasts are involved in focal bone erosion in RA [3].
Osteoclasts are derived from the monocyte/macrophage lineage. It is reported that osteoclast precursors reside in <span class="yellow">human</span> peripheral blood monocytes [4,5]. A marked increase of the circulating osteoclast precursors was demonstrated in <span class="yellow">patients</span> with erosive psoriatic arthritis as well as in arthritic TNFα transgenic <span class="yellow">mice</span> [6,7]. It was also shown that peripheral monocytes differentiate into osteoclasts when seeded on RANKL/osteoclast differentiation factor-producing RA synovial fibroblasts [8]. In addition, RA synovial macrophages thought to originate from peripheral blood monocytes were shown to differentiate into osteoclasts [9,10]. Monocytes are therefore involved not only in synovial inflammation, but also in bone remodeling as potential precursors for synovial macrophages and osteoclasts.
<span class="yellow">Human</span> peripheral blood monocytes consist of two major subsets, CD16+ and CD16-, comprising 5–10% and 90–95% of the monocytes, respectively. These two subsets exhibit different chemotaxis activities and potential of cytokine production [11,12]. Moreover, activation of the Toll-like receptor induces distinct subsets, CD1b+ dendritic cells and DC-SIGN+ (dendritic cell-specific C-type lectin ICAM-3-grabbing nonintegrin) macrophages from CD16+ and CD16- monocytes, respectively [13]. It has not been revealed, however, which monocyte subset develops into osteoclasts.
In the present study, we determined the <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts, and revealed that each subset exhibits a different response for osteoclastogenic stimuli.<br><br>Materials and methods
Purification of peripheral blood monocytes
Peripheral blood monocytes from healthy donors were collected using Ficoll-Conray (Imuuno-Biological Laboratories, Gunma, Japan) gradient centrifugation. Negative selection of monocytes was performed using MACS microbeads (Miltenyi Biotec, Auburn, CA, USA) according to the protocol supplied by the manufacturer.
The purified monocytes were separated into two subsets, CD16+ and CD16- monocytes, using CD16 MicroBeads (Miltenyi Biotec). Flow cytometry analysis using FITC-conjugated <span class="blue">mouse</span> anti-CD14 mAb (MY4; Bechman Coulter, Fullerton, CA, USA) and phycoerythin-conjugated <span class="blue">mouse</span> anti-CD16 mAb (3G8; BD Biosciences, San Jose, CA, USA) showed that the purities of the CD16+ and CD16- monocytes were more than 90% and 92%, respectively.
For the other experiment, monocytes were purified using CD14 MicroBeads (Miltenyi Biotec), and then stained either with FITC-conjugated <span class="blue">mouse</span> anti-CD33 mAb (MY9; Bechman Coulter) or phycoerythin-conjugated <span class="blue">mouse</span> anti-CD16 mAb (3G8). Cell sorting of the stained cells was performed using a FACS Vantage cytometer (BD Biosciences) or a MoFlo cell sorter (Dako, Glostrup, Denmark).<br><br>Osteoclast differentiation
Purified CD16+ and CD16- monocytes (5 × 104 cells/well) were incubated in 96-well plates in αMEM (Sigma, St Louis, MO, USA) with heat-inactivated 10% fetal <span class="yellow">bovine</span> serum (FBS) (Sigma) or with Ultra-Low IgG FBS (IgG < 5 μg/ml; Invitrogen, Carlsbad, CA, USA), and where indicated with M-CSF + RANKL (Peprotech, Rocky Hill, NJ, USA).
For the other experiments, varied numbers of CD16+ monocytes (1 × 103, 2.5 × 103, 5 × 103) were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured in 96-well plates in αMEM with heat-inactivated 10% FBS. The medium was replaced with fresh medium 3 days later, and after incubation for 7 days the cells were stained for tartrate-resistant acid phosphatase (TRAP) expression using a commercial kit (Hokudo, Sapporo, Japan). The number of TRAP-positive multinucleated cells (MNC) in three randomly selected fields examined at 100× magnification or the total number of TRAP-positive MNC per well was counted under light microscopy.<br><br>Resorption assay
Monocytes were seeded onto plates coated with calcium phosphate thin films (Biocoat Osteologic; BD Biosciences) and were incubated with RANKL (40 ng/ml) + M-CSF (25 ng/ml) for 7 days. The cells were then lysed in bleach solution (6% NaOCl, 5.2% NaCl). The resorption lacunae were examined under light microscopy.<br><br>Enzyme-linked immunosorbent assay
Purified monocytes were cultured in 96-well plates where indicated either with RANKL or M-CSF for 24 hours. Concentrations of TNFα and IL-6 in the culture supernatant were measured with an ELISA kit (BioSourse International, Camarillo, CA, USA). For experiments of matrix metalloproteinase (MMP)-9 and TRAP-5b, culture supernatants were collected on day 7 and the concentrations of these enzymes were measured using an MMP-9 ELISA kit (Amersham Biosciences, Piscataway, NJ, USA) or a TRAP-5b ELISA kit (Suomen, Turku, Finland).<br><br>Reverse transcriptase-polymerase chain reaction
Monocytes (1 × 106 cells/well) were cultured in six-well plates with M-CSF alone or with M-CSF + RANKL for 3 days. Total RNA was extracted using RNeasy Micro (Qiagen, Valencia, CA, USA). The RNA was then treated with DNase I (Qiagen). The oligo(dT)-primed cDNA was synthesized using Superscript II reverse transcriptase (Invitrogen). The amount of cDNA for amplification was adjusted by the amount of RNA measured by an optical density meter and also by β-actin or GAPDH PCR products. One microliter of cDNA was amplified in a 50 μl final volume containing 25 pmol appropriate primer pair, 10 pmol each of the four deoxynucleotide triphosphates, and 5 units FastStart Taq DNA Polymerase (Roche, Manheim, Germany) in a thermal cycler (PTC-200; MJ GeneWorks, Waltham, MA, USA).
The PCR conditions were 25–40 cycles of denaturation (95°C for 30 s), annealing (60–62°C for 1 min) and extension (72°C for 1 min). The sequences of the primers are presented in Table 1. The PCR products were separated by electrophoresis through 2% agarose gel.<br><br>Western immunoblot analysis
Purified monocytes were cultured for 3 days in the presence of 40 ng/ml M-CSF with or without 25 ng/ml RANKL. Cells were lysed in RIPA Lysis buffer (upstate, Lake Placid, NY, USA) containing protease inhibitors (Roche) for 15 min at 4°C. A total of 20 μg protein was boiled in the presence of 6 × sodium dodecyl sulfate sample buffer, and was separated on 7.5% or 10% sodium dodecyl sulfate-polyacrylamide gel (ATTO, Tokyo, Japan). Proteins were then electrotransferred to a polyvinylidene fluoride microporous membrane (Millipore, Billerica, MA, USA) in a semidry system. Membranes were incubated in 10% skim milk prepared in phosphate-buffered saline (PBS) containing 0.1% Tween 20, and were subjected to immunoblotting. Antibodies used were <span class="yellow">goat</span> anti-RANK antibody (Techne Corporation, Minneapolis, MN, USA), <span class="yellow">goat</span> anti-c-fms antibody (R&D systems, Minneapolis, MN, USA), and <span class="blue">mouse</span> anti-β-actin mAb (AC-15; Sigma). Peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">goat</span> IgG antibody (Dako) or peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="blue">mouse</span> IgG antibody (Dako) was used as the second antibody. The signals were visualized by chemiluminescence reagent (ECL; Amersham Biocsiences, Little Chalfont, UK).<br><br>Cell surface expression of c-fms
The following mAbs were used for analysis of c-fms expression: Alexa 647-conjugated anti-CD14 mAb (UCHM1; Serotec, Oxford, UK), FITC-conjugated anti-CD16 mAb (3G8; Bechman Coulter) and phycoerythin-conjugated anti-c-fms mAb (61708; R&D systems). Alexa 647-conjugated <span class="blue">mouse</span> IgG2a (Serotec), FITC-conjugated <span class="blue">mouse</span> IgG1 (BD Biosciences) and phycoerythin-conjugated <span class="blue">mouse</span> IgG1 (Bechman Coulter) were used as isotype controls. Peripheral blood monocytes (1 × 105 cells) were incubated with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and were then stained with three fluorochrome-labeled mAbs for 45 minutes on ice. The stained cells were analyzed with a FACS Calibur (BD Biosciences).<br><br>Immunofluorescent staining
Monocytes (8 × 104 cells/well) were allowed to adhere on 96-well plates overnight or were cultured with M-CSF and RANKL for 2–4 days. The cells were fixed in acetone and then stained with anti-αvβ3 mAb (LM609; Chemicon, Temecula, CA, USA) or <span class="blue">mouse</span> IgG1 (11711; R&D Systems) as an isotype-matched control. Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="blue">mouse</span> IgG1 antibody (Molecular Probes, Eugene, OR, USA) was used as the second antibody. TOTO-3 (Molecular Probes) was used for nuclear staining.<br><br>Flow cytometric analysis of p38 MAPK and ERK1/2 phosphorylation
Purified monocytes were cultured in the presence of 25 ng/ml M-CSF for 3 days, and were either left unstimulated or were stimulated with 40 ng/ml RANKL at 37°C. Stimulations were stopped by adding an equal volume of PhosFlow Fix Buffer I solution (BD Biosciences) to the cell culture. After incubation for 10 minutes at 37°C, the cells were permeabilized by washing twice at room temperature in PhosFlow Perm/Wash Buffer I (BD Biosciences). A total of 1 × 105 cells was then Fc blocked with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and was stained with Alexa Fluor 647-conjugated mAb either to phospho-p38 MAPK (T180/Y182) or to phospho-ERK1/2 (T202/Y204) (BD Biosciences) for 30 minutes at room temperature. Alexa Fluor 647-conjugated <span class="blue">mouse</span> IgG1 (BD Biosciences) was used as an isotype control. The cells were washed in PhosFlow Perm/Wash Buffer I, and were analyzed by flow cytometry, as already described.<br><br>RNA interference
RNA oligonucleotides (iGENE, Tsukuba, Japan) were designed based on the algorithm that incorporates single nucleotide polymorphism and homology screening to ensure a target-specific RNA interference effect. The following sense and antisense oligonucleotides were used: integrin β3, 5'-GCU UCA AUG AGG AAG UGA AGA AGC A-AG and 3'-UA-CGA AGU UAC UCC UUC ACU UCU UCG U; randomized control, 5'-CGA UUC GCU AGA CCG GCU UCA UUG C-AG and 3'-UA-GCU AAG CGA UCU GGC CGA AGU AAC G; and lamin, 5'-GAG GAA CUG GAC UUC CAG AAG AAC A-AG and 3'-UA-CUC CUU GAC CUG AAG GUC UUC UUG U.
CD16- monocytes (8 × 104 cells/well) were incubated in 96-well plates in optimem (Invitrogen). After 1 hour, siRNAs were transfected into the cells using oligofectamine (Qiagen) based on the method recommended by the manufacturer. After 2 hours, the cells were washed once with PBS, followed by the addition of αMEM supplemented with 10% FBS, M-CSF and RANKL. After a 2-day incubation, the β3 mRNA expression was analyzed by RT-PCR with different PCR cycles, as described earlier. Immunofluorescent staining for the αvβ3 heterodimer was also performed as described above, and numbers of αvβ3-positive cells were counted in randomly selected three fields at 100× magnification. Seven days after the transfection of siRNAs, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Inhibition of osteoclastogenesis with cyclic RGDfV peptide
CD16- monocytes were incubated in 96-well plates with M-CSF + RANKL for 2 days. A medium containing either cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) (Calbiochem, San Diego, CA, USA) or dimethyl sulfoxide was then added. After incubation for a further 5 days, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Immunohistochemistry
Synovial tissue samples were obtained during total knee joint replacement surgery from four RA <span class="yellow">patients</span>. Signed consent forms were obtained before the operation. The experimental protocol was approved by the ethics committee of the Tokyo Medical and Dental University. RA was diagnosed according to the American College of Rheumatology criteria [14].
Double immunofluorescent staining for CD68 and CD16 antigens was conducted on optimal cutting temperature-embedded sections of frozen synovial samples. Eight-micrometer-thick cryostat sections of RA synovium were fixed in acetone for 3 minutes and were then rehydrated in PBS for 5 minutes. The samples were incubated in 5 μg/ml proteinase K (Roche), 50 mM ethylenediamine tetraacetic acid, 100 mM Tris–HCl, pH 8.0, for 15 minutes at room temperature followed by a wash in PBS. The samples were then blocked with 10% <span class="yellow">goat</span> serum in PBS for 60 minutes at room temperature, and were incubated with anti-CD16 mAb (3G8; Immunotech, Marseille, France) or <span class="blue">mouse</span> IgG1 (11711) as an isotype-matched control in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then washed three times in PBS, for 5 minutes each, and incubated with Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="blue">mouse</span> IgG1 antibody (Molecular Probes) in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then sequentially stained for CD68 antigen in a manner similar to that used for CD16 staining. The samples were stained with anti-CD68 mAb (PGM1; Immunotech) or <span class="blue">mouse</span> IgG3 (6A3; MBL, Nagoya, Japan) followed by labeling with Alexa fluor488-conjugated <span class="yellow">goat</span> anti-<span class="blue">mouse</span> IgG3 antibody (Molecular Probes). The samples were examined by confocal laser scanning microscope (Olympus, Tokyo, Japan).<br><br>Statistical analysis
Data are expressed as the mean ± standard error of the mean. A nonpaired Student's t test was used for comparison, using the StatView program (Abacus Concepts, Berkeley, CA, USA). P < 0.05 was considered statistically significant.<br><br>
Results
Induction of osteoclasts from CD16- peripheral blood monocytes
To identify the monocyte subset that differentiates into osteoclasts, we examined osteoclast formation from CD16+ and CD16- <span class="yellow">human</span> peripheral blood monocytes. The monocyte subsets were purified using magnetic beads. Incubation with M-CSF alone did not induce osteoclast formation from either subset (Figure 1a). Culture with M-CSF + RANKL induced a significant number of TRAP-positive MNC from the CD16- subset, whereas only few CD16+ monocytes differentiated into TRAP-positive MNC (Figure 1a,b). We then assessed the bone resorptive ability by culturing cells on calcium phosphate-coated plates with M-CSF + RANKL. Resorption lacunae were detected only in the CD16- subset (Figure 1c), indicating the TRAP-positive CD16--derived MNC possessed the osteoclast phenotype.
Similar results were obtained using purified monocytes by FACS sorting (purities: CD16+, 96%; CD16-, 97%). The number of TRAP-positive MNC induced were 36 ± 3 cells/well and 348 ± 13 cells/well from CD16+ and CD16- monocytes, respectively. To exclude the possibility that the anti-CD16 antibody used for isolation of CD16+ monocytes inhibits osteoclast formation, we separated the two subsets, CD33low monocytes and CD33high monocytes, using anti-CD33 mAb and a fluorescent cell sorter, since it was reported that CD33low monocytes correspond to CD16+, and that CD33high correspond to CD16- monocytes [15]. On average, the CD33low population contained CD16- (10.2%)/CD16+ (89.8%) monocytes, and the CD33high population contained CD16- (86.3%)/CD16+ (13.7%) monocytes.
Culture with M-CSF + RANKL induced TRAP-positive MNC from CD33high monocytes, whereas no or few CD33low monocytes differentiated into TRAP-positive MNC (CD33low vs CD33high, 2 ± 1 vs 192 ± 71 cells/well; n = 3). TRAP-5b and MMP-9 in the culture supernatants, both of which are known to be produced by osteoclasts, were measured by ELISA. The concentrations of both enzymes were significantly higher in the culture supernatant of CD16- monocytes than in that of CD16+ monocytes (Figure 1d). These results suggest that the CD16- peripheral blood monocyte subset, but not the CD16+ subset, differentiate into osteoclasts by incubation with RANKL + M-CSF.<br><br>CD16+ monocytes do not affect the osteoclastogenesis from CD16- monocytes
To examine whether CD16+ monocytes affect osteoclastogenesis from CD16- monocytes, varied numbers of CD16+ monocytes were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured for 7 days in the presence of M-CSF + RANKL. The number of TRAP-positive MNC was not altered by the presence of CD16+ monocytes (Figure 2). The results indicated that CD16+ monocytes did not hamper or enhance the osteoclastogenesis from CD16- monocytes.<br><br>Differences in cytokine production by RANKL-stimulated or M-CSF-stimulated CD16+ and CD16- monocytes
To compare the biological response of CD16+ and CD16- subsets with either RANKL or M-CSF stimulation, we measured the amount of TNFα and IL-6 production after exposure of cells to various concentrations of RANKL or M-CSF with an ELISA. Without RANKL the CD16+ subset produced a significant amount of TNFα and IL-6, whereas the CD16- subset produced undetectable levels (Figure 3a). RANKL stimulation increased TNFα production from both subsets in a dose-dependent manner, although the CD16+ subset produced more TNFα than did the CD16- subset. RANKL stimulation also enhanced IL-6 production from the CD16+ subset, but not from the CD16- subset. M-CSF stimulation increased TNFα and IL-6 production from both subsets, although the CD16+ subset produced more than the CD16- subset (Figure 3b).
These results suggest that CD16+ monocytes also respond both to RANKL and M-CSF stimulation, although such stimulation does not result in differentiation into osteoclasts. CD16+ monocytes were also noted to express higher amounts of inflammatory cytokines compared with CD16- monocytes with or without RANKL or M-CSF stimulation.<br><br>Comparison of expression levels of molecules involved in osteoclastogenesis between CD16+ and CD16- monocytes
Diverse molecules are involved in RANKL/RANK and its costimulatory signal transduction pathways [16]. The different response to RANKL + M-CSF stimulation between the CD16+ monocyte subset and the CD16- monocytes subset might be explained by the expression profiles of these molecules. We therefore examined the mRNA levels of the following molecules: receptor activator of NF-κB (RANK), the receptor for RANKL; c-fms, the receptor for M-CSF; tumor necrosis factor receptor-associated factor 6 (TRAF-6), the adaptor protein for RANK; c-Fos and nuclear factor of activated T cells c1 (NFATc1), transcription factors that are essential for osteoclastogenesis; DNAX-activation protein 12 (DAP12) and Fc receptor γ chain (FcRγ), adaptor proteins known to deliver costimulatory signals in RANKL-induced osteoclastogenesis; signal regulatory protein β1 (SIRP-β1), triggering receptor expressed on myeloid cells 2 (TREM-2) and osteoclast-associated receptor (OSCAR), transmembrane receptors that associate with either DAP12 or FcRγ; and αv and β3, integrins known to be expressed as the αvβ3 heterodimer on osteoclasts.
The mRNA levels of RANK, c-fms, TRAF-6, DAP12 and SIRP-β1 under the baseline condition (no stimulation) varied between the donors; however, we did not find consistent differences in the mRNA levels of these molecules between the CD16+ monocyte subset and the CD16- monocyte subset among three to six donors (Figure 4a). The mRNA levels of other molecules, apart from integrin β3, were similar between the two subsets under the no-stimulation condition. Although the mRNA levels of RANK, c-fms, DAP12, FcRγ, TREM-2 and OSCAR increased in response to M-CSF alone or M-CSF + RANKL in both subsets, the expression levels were not significantly different between the two subsets. Expressions of TRAF-6, c-Fos and SIRP-β1 mRNA did not change following stimulation with M-CSF + RANKL. Of note, the expression of NFATc1 mRNA was enhanced by M-CSF + RANKL treatment only in the CD16- subset. Furthermore, expression of integrin αv in both subsets was enhanced by M-CSF with or without RANKL; however, the expression level was greater in the CD16- subset. It was noted that integrin-β3 mRNA was detected only in the CD16- subset and was increased by M-CSF + RANKL stimulation, but not by M-CSF alone. The protein expression of RANK under the baseline condition was weakly detected in both subsets, and the levels were varied between donors by western immunoblotting.
The protein expression of c-fms was weakly detected in unstimulated CD16+ monocytes, but not in CD16- monocytes (Figure 4b). Flow cytometry analysis of c-fms in fresh monocytes, however, showed that both subsets express the molecule on the cell surface (Figure 4c). Expressions of both RANK and c-fms were upregulated by M-CSF alone and by M-CSF + RANKL, and we did not find consistent differences in the protein levels of these molecules between the two monocyte subsets. The profiles of expression levels of molecules involved in RANKL/RANK and its costimulatory pathways are similar between the two subsets, except for NFATc1, integrin αv and integrin β3. We therefore assumed that the distinct induction of NFATc1, integrin αv and integrin β3 in response to RANKL stimulation among the two monocyte subsets might explain the differences in their abilities to differentiate into osteoclasts.<br><br>RANKL stimulation induces αvβ3 expression on CD16- monocytes
The integrin-β3 subunit binds to integrin αv only and is expressed as the heterodimeric protein αvβ3 on monocytes and osteoclasts [17]. We examined the expression of αvβ3 on CD16+ and CD16- monocytes by immunofluorescent staining. Neither unstimulated nor M-CSF-stimulated monocyte subsets expressed αvβ3 (Figure 4d and data not shown). After 48 and 72 hours of treatment with M-CSF + RANKL, αvβ3-positive mononuclear cells were observed in CD16- monocyte cultures but not in CD16+ monocyte cultures. At 96 hours, both αvβ3-positive mononuclear cells and multinucleated cells were present in the CD16- monocyte culture. The results indicated that αvβ3 was selectively expressed on CD16- monocytes in the presence of M-CSF + RANKL, and the expression was revealed before the cells differentiate into typical multinucleated osteoclasts.<br><br>RANKL activates ERK and p38 kinases only in CD16- monocytes
Since ERK and p38 MAPK are essential in RANKL-induced osteoclastogenesis [18-20], we next examined whether these kinases were activated differently in CD16+ monocytes and in CD16- monocytes. Purified monocytes were precultured with 25 ng/ml M-CSF for 3 days to enhance RANK expression, and were then treated with RANKL. The RANKL treatment induced phosphorylation of both ERK and p38 MAPK in CD16- monocytes at 5 minutes postexposure, although the p38 MAPK phosphorylation was weak. Both phosphorylations declined to a basal level within 20 minutes (Figure 5). In contrast, ERK and p38 MAPK were not detectably phosphorylated in CD16+ monocytes with RANKL.<br><br>siRNA targeting integrin β3 inhibits osteoclastogenesis from CD16- monocytes
The integrin-β3 cytoplasmic domain is essential for activation of intracellular signals from αvβ3 heterodimers [17]. We therefore examined the involvement of αvβ3 in RANKL + M-CSF-induced osteoclastogenesis in <span class="yellow">human</span> CD16- monocytes using siRNA targeting the integrin-β3 subunit. The integrin-β3 siRNA or control randomized siRNA were transfected into CD16- monocytes. At 48 hours post-transfection, we determined the integrin-β3 mRNA level and αvβ3 heterodimer protein expression. The integrin-β3 mRNA level was reduced in the integrin-β3 siRNA-transfected monocytes compared with control siRNA-transfected monocytes (Figure 6a). The αvβ3 heterodimer expression was evaluated by immunofluorescent staining. The number of αvβ3-positive cells was significantly decreased in integrin-β3 siRNA-transfected monocytes compared with that in control siRNA (Figure 6b).
After 7 days of incubation, the number of TRAP-positive MNC was counted. Transfection with integrin-β3 siRNA significantly reduced the number of TRAP-positive MNC in a dose-dependent manner compared with control siRNA transfection (Figure 6c). In addition, the use of siRNA directed toward a different site of integrin-β3 mRNA also inhibited osteoclast formation from CD16- monocytes (data not shown). On the other hand, siRNA that targeted lamin, which was used as a negative control, did not inhibit the induction of osteoclasts (data not shown). These results indicate the importance of integrin β3 in RANKL-induced osteoclast formation from CD16- peripheral blood monocytes.<br><br>Cyclic RGDfV peptide inhibits the osteoclastogenesis from CD16- monocytes
Integrin αvβ3 recognizes a common tripeptide sequence, RGD (Arg-Gly-Asp), which is present in bone matrix proteins such as vitronectin and fibronectin [21]. Cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) inhibits binding of the RGD-containing molecules to αvβ3 [22]. To investigate the role of ligand binding to the αvβ3 heterodimer in the osteoclastogenesis, we examined whether cyclic RGDfV peptide inhibits the formation of osteoclasts. Cyclic RGDfV peptide significantly reduced the number of TRAP-positive MNC in a dose-dependent manner (Figure 6d). The results imply possible involvement of ligand bindings to αvβ3 in the osteoclastogenesis.<br><br>Knockdown of integrin β3 did not affect the expression of NFATc1 mRNA
In the next step, we determined whether integrin-β3-siRNA-induced inhibition of the osteoclastogenesis reflects downregulation of NFATc1, which is a key transcription factor in osteoclastogenesis [23]. For this purpose, we compared NFATc1 mRNA levels between integrin β3 and control siRNA-transfected monocytes. Interestingly, integrin-β3 knockdown did not alter the NFATc1 mRNA level (Figure 7), suggesting that signal transduction mediated by integrin β3 does not affect the expression of NFATc1.<br><br>Detection of CD16+ and CD16- macrophages in synovium of RA <span class="yellow">patients</span>
RA synovial macrophages are derived from peripheral blood monocytes, and their recruitment into the synovium is facilitated by various adhesion molecules and chemokines [24]. To analyze CD16 expression on synovial macrophages, RA synovial tissues were double-stained for CD16 and a macrophage marker, CD68. CD16-/CD68+ macrophages were widespread in the synovium. Although less frequent, CD16+/CD68+ macrophages were also observed both in the synovial intima and subintima (Figure 8). The presence of two subsets of macrophages, CD16+ and CD16-, in RA synovium indicates that both CD16+ and CD16- peripheral blood monocytes are recruited into the synovium.<br><br>
Discussion
<span class="yellow">Human</span> peripheral blood monocytes are a heterogeneous population, and they are divided into two subsets based on the expression of CD16. The CD16+ and CD16- monocyte subsets show functional differences in migration, cytokine production and differentiation into macrophages or dendritic cells [11-13,15]. We focused on the heterogeneity of the monocytes, and the primary question addressed in this study was which monocyte subset could be the source of osteoclasts. The results demonstrated that CD16- peripheral blood monocytes, but not CD16+ monocytes, differentiated in vitro into osteoclasts by treatment with RANKL + M-CSF.
To investigate the molecular mechanisms of the different response to RANKL and the differentiation into osteoclasts between CD16+ and CD16- monocytes, we examined the expression of molecules known to be involved in osteoclastogenesis. The expression profiles of integrin αv, integrin β3 and NFATc1 were different between the two subsets. Integrin αvβ3 heterodimer was expressed only on RANKL and M-CSF-stimulated CD16- monocytes. It is known that αvβ3 expressed on osteoclasts is important in bone resorption as well as in attachment of osteoclasts to the bone matrix [25].
It was recently reported that bone marrow macrophages of integrin-β3-deficient <span class="yellow">mice</span> could not differentiate into mature osteoclasts in vitro, suggesting that αvβ3 is involved not only in activation, but also in differentiation, of osteoclasts in <span class="yellow">mice</span> [26,27]. The authors also showed that αvβ3 and c-fms share a common intracellular signaling pathway, including the activation of ERK and the induction of c-Fos [27], both of which are essential for osteoclastogenesis [28,29]. In addition, it was reported that echistatin, an αvβ3 antagonist, inhibited osteoclast formation of <span class="blue">mouse</span> bone marrow cells [30].
In accordance with these reports, our data showed that knockdown of integrin-β3 expression resulted in downregulation of the αvβ3 heterodimer, and abrogated osteoclastogenesis from <span class="yellow">human</span> peripheral blood CD16- monocytes. We also showed that blocking of adhesive ligands to bind to αvβ3 by RGDfV peptide inhibited osteoclast formation from CD16- monocytes. Taken together, the process of ligand binding to αvβ3 may be involved in the osteoclastogenesis. Blockade of αvβ3 could therefore be a therapeutically beneficial approach to modulate osteoclastogenesis. Indeed, integrin αvβ3 antagonists effectively treated osteoporosis in <span class="yellow">mice</span>, <span class="yellow">rats</span> and <span class="yellow">humans</span>, and protected bone destruction in <span class="yellow">rat</span> adjuvant-induced arthritis in vivo [31-34]. Of note, it is reported that <span class="yellow">patients</span> with Iraqi-Jewish-type Glanzmann thrombasthenia who are deficient in integrin β3 do not develop osteopetrosis because of the upregulation of α2β1 expression on osteoclasts, although the bone-resorptive ability of the osteoclasts was decreased in vitro [35]. The function of αvβ3 in vivo in osteoclast formation and resorptive function could therefore be partially compensated by other integrins.
Although all the multinucleated osteoclasts expressed αvβ3 (Figure 4d) [36], a small number of M-CSF + RANKL-stimulated mononuclear CD16- monocytes expressed αvβ3 (Figure 4d). Multinucleated osteoclasts are formed by fusion of osteoclast precursor cells [37]. It was reported that αvβ3 is involved in the migration of osteoclast precursors [30]. The αvβ3-positive cells could therefore be forced to migrate by the ligands and may fuse with closed αvβ3-negative cells. Alternatively, only αvβ3-positive cells may be fused with each other.
It is possible to consider that signaling from CD16 by anti-CD16 mAb-coated magnetic beads, which were used for the cell separation, or by IgG contained in FBS might inhibit osteoclastogenesis from CD16+ monocytes. We therefore separated the two subsets using anti-CD33 mAb and a fluorescent cell sorter, and stimulated the cells with M-CSF + RANKL. The results showed that CD33low monocytes, which correspond to CD16+ monocytes, still could not differentiate into osteoclasts. CD16 is a heterodimer consisting of FcγIIIa and Fcγ, and has low affinity for the Fc region of IgG. Aggregation of CD16 by immune complexes leads to transmission of activating signals via the immunoreceptor tyrosine-based activation motif in the γ chain [38]. We also assessed osteoclastogenesis from the two monocyte subsets using IgG-depleted <span class="yellow">bovine</span> serum. Even in the IgG-free medium, CD16- monocytes but not CD16+ monocytes differentiated into osteoclasts (data not shown). We could therefore exclude the possibility that signal transduction through CD16 inhibits osteoclastogenesis from CD16+ monocytes.
NFATc1 is a key transcription factor in osteoclastogenesis [16]. In the present study, stimulation with M-CSF + RANKL increased the NFATc1 mRNA expression in the CD16- subset only, similar to integrin αv and integrin β3. The differences in NFATc1 induction might therefore also explain the difference in osteoclastogenesis between the two monocyte subsets. It is of interest that knockdown of integrin β3 did not lower the mRNA level of NFATc1. This result supports the notion that NFATc1 is located upstream of integrin-β3 expression [39]. It is also possible that parallel activation of two signaling pathways mediated by integrin β3 and NFATc1 contributes to osteoclastogenesis independently or cooperatively. Further studies are needed to determine the mechanisms of integrin β3 involvement in RANKL/RANK-mediated osteoclast differentiation.
It has been demonstrated that MAPK families, ERK and p38 MAPK, were activated by RANKL-induced intracellular signalings in osteoclasts and osteoclast precursors [18,19]. In addition, these kinases are involved in the differentiation of osteoclasts [20]. We showed that RANKL stimulation induced phosphorylation of ERK and p38 MAPK only in CD16- monocytes. It is suggested that differential activation of these kinases may partially explain the distinct properties of the two monocyte subsets upon RANKL stimulation.
Our results showed that CD16+ monocytes produce higher levels of inflammatory cytokines including TNFα and IL-6 compared with CD16- monocytes. These results are consistent with the previous report showing that CD16+ monocytes produced larger amounts of TNFα upon lipopolysaccharide or lipopeptide stimulation than did CD16- monocytes [40]. Interestingly, we showed that stimulation either with RANKL or M-CSF upregulated the TNFα and IL-6 production by CD16+ monocytes. A marked increase of CD16+ monocytes in peripheral blood is reported in inflammatory diseases, such as infection, malignancy, Kawasaki disease and RA [41-44]. Taken together, CD16+ monocytes may be an important source of inflammatory cytokines.
In <span class="yellow">mice</span>, peripheral blood Ly-6Chigh monocytes, which are thought to correspond to <span class="yellow">human</span> CD16- monocytes, increase in inflammatory conditions, and these cells are recruited into sites of inflammation [45]. In contrast, Ly-6Clow monocytes, which are thought to correspond to <span class="yellow">human</span> CD16+, migrate into noninflamed tissues [12]. These data on <span class="blue">mouse</span> monocytes seem to be in contrast to the data on <span class="yellow">human</span> monocytes, which show expansion of CD16+ monocytes in inflammatory conditions where they produce larger amounts of inflammatory cytokines. At present, it is not clear whether <span class="blue">mouse</span> monocyte subsets, Ly-6Clow/Ly-6Chigh, represent <span class="yellow">human</span> monocyte subsets, CD16+/CD16- monocytes, and whether the biologic functions of <span class="blue">mouse</span> monocytes are analogous to those of <span class="yellow">human</span> monocytes.
In <span class="yellow">mice</span>, blood monocytes newly released from the bone marrow are exclusively Ly-6Chigh and the level of Ly-6C is downregulated while in circulation [45]. It is thus suggested that in <span class="yellow">mice</span> the two monocyte subsets differing in Ly-6C expression represent different stages in the maturation pathway. In the <span class="yellow">human</span>, transition from CD16- monocytes to CD16+ monocytes is observed upon culture with IL-10, M-CSF and transforming growth factor beta in vitro [42,46]. Similar to <span class="blue">mouse</span> monocytes, therefore, <span class="yellow">human</span> peripheral blood CD16- monocytes may also maturate into CD16+ monocytes.
It is reported that a significant number of RA synovial cells in the intima express CD16, suggesting that CD16+ cells are synovial macrophages [47]. We confirmed that both CD16+ and CD16- macrophages accumulate in the RA synovium by double-color immunohistochemical staining for CD68 and CD16. A number of chemokines are abundantly expressed in the RA synovium [24,48]. Among these cytokines, MCP-1, MIP-1α, SDF-1, RANTES and fractalkine can induce migration of CD16- monocytes in vitro ([11,12] and unpublished data). On the other hand, migration of CD16+ monocytes is induced only by fractalkine. These chemokines therefore seem to play an important role in recruitment of CD16+ and CD16- monocytes from the circulating pool into the RA synovium.
The osteoclast inducers are also produced in the RA synovium. RANKL is expressed by synovial fibroblasts and activated T cells [49-51], while M-CSF is expressed on RA synovial macrophages and fibroblasts [52,53].
TNFα and IL-6, which are mainly expressed on RA synovial macrophages and fibroblasts, respectively, could also enhance osteoclast differentiation [54]. Collectively, it is probable that the recruited CD16- monocytes/macrophages differentiate into osteoclasts in the RA synovium, and contribute to bone destruction. On the other hand, CD16+ monocytes/macrophages might also be involved in RA pathogenesis by producing inflammatory cytokines including TNFα and IL-6. Since TNFα and IL-6 enhance osteoclast formation [54,55], CD16+ monocytes/macrophages may also contribute to osteoclastogenesis in RA synovium.<br><br>Conclusion
We have shown that <span class="yellow">human</span> peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct response to osteoclastogenic stimuli. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for the osteoclastogenesis. The blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Abbreviations
DAP = DNAX-activation protein; ELISA = enzyme-linked immunosorbent assay; FBS = fetal <span class="yellow">bovine</span> serum; FcRγ = Fc receptor γ chain; IL = interleukin; FITC = fluorescein isothiocianate; mAb, monoclonal antibody; M-CSF = macrophage colony-stimulating factor; MEM = modified Eagle's medium; MMP = matrix metalloproteinase; MNC = multinucleated cells; NF = nuclear factor; OSCAR = osteoclast-associated receptor; PBS = phosphate-buffered saline; PCR = polymerase chain reaction; RA = rheumatoid arthritis; RANK = receptor activator of NF-κB; RANKL = receptor activator of NF-κB ligand; RT = reverse transcriptase; siRNA = small interfering RNA; SIRP-β1 = signal regulatory protein-β1; TNFα = tumor necrosis factor alpha; TRAF = tumor necrosis factor receptor-associated factor; TRAP = tartrate-resistant acid phosphatase; TREM = triggering receptor expressed on myeloid cells.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YK participated in the design of the study, carried out the experiments and statistical analysis, and drafted the manuscript. KH and KT participated in the design of the study and its coordination. TN and NM conceived of the study, participated in its design and coordination, and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1805739</h3>Embryonic sympathoblasts transiently express TrkB in vivo and proliferate in response to brain-derived neurotrophic factor in vitro
Abstract
Background
Nerve growth factor and neurotrophin-3 are involved in the development of sympathetic neurons; however, whether brain derived neurotrophic factor also plays a role is not known. The purpose of this study was to determine whether BDNF and its receptor, TrkB, are expressed during the development of paravertebral sympathetic ganglia in vivo and to determine the effect of BDNF in vitro.<br><br>Results
As neural crest cells coalesce to form sympathetic ganglia, TrkB-positive cells are seen in both <span class="yellow">chicken</span> and <span class="blue">mouse</span> embryos. In <span class="yellow">chicken</span> embryos, TrkB-expressing cells first appear at Hamburger-Hamilton Stage (St) 27 and they co-express HNK-1, confirming that they are migrating neural crest cells. The TrkB-positive cells lack neural markers at this stage; however, they migrate with other neurally differentiating cells that are TrkA and TrkC-positive. By St. 29/30, TrkB-positive cells begin to express the neural specific markers Hu C/D and Islet-1; eventually, all TrkB positive cells commence neural differentiation. By St. 34, TrkB and TrkC staining are lost. BDNF transcript expression parallels that of TrkB. In the <span class="blue">mouse</span>, TrkB-positive cells surround newly formed sympathetic ganglia and a small number of TrkB positive cells that co-express tyrosine hydroxylase are seen within ganglia between E13.5-15. In cell culture, many cells from St. 29–30 <span class="yellow">chicken</span> lumbar sympathetic ganglia express neural markers and are dividing, indicating that they are sympathoblasts. Sympathoblasts and neurons require both nerve growth factor and neurotrophin-3 for survival. BDNF increases the number of cells expressing neural markers in culture by increasing number of cells that incorporate bromodeoxyuridine. In contrast, most TrkB-positive sympathetic cells in vivo are not actively proliferating between E6–E8.<br><br>Conclusion
Developing paravertebral sympathetic ganglia in avian and <span class="yellow">murine</span> embryos contain a subpopulation of sympathoblasts that transiently express TrkB and ultimately commence neuronal differentiation. These TrkB expressing sympathoblasts are not actively dividing in vivo; yet, when placed in vitro, will divide in response to BDNF. This suggests that the availability of BDNF in vivo fails to reach a threshold necessary to induce proliferation. We suggest that excess TrkB stimulation of sympathoblasts in vivo may lead to the genesis of neuroblastoma.<br><br><br><br>Background
Neural crest cells destined to become paravertebral sympathetic neurons proliferate and differentiate during migration and gangliogenesis. In <span class="yellow">chicken</span> embryos, migrating neural crest cells express catecholamines at Hamburger/Hamilton Stage (St.) 19, and these cells form the primary sympathetic chain dorsolateral to the aorta at St. 22 (E3.5) [1]. Between St. 23 (E4) and St. 28 (E6), these cells disperse and undergo a secondary migration to form the paravertebral sympathetic chain that resides ventral to the spinal cord and dorsal root ganglion [1]. After ganglia coalesce, sympathoblasts express markers of neuronal differentiation, such as Q211 and tyrosine hydroxylase (TH), at a time when they also incorporate [3H]-thymidine [2]. Time lapse photography has shown that cultured E15.5–E16.5 sympathetic neurons from <span class="yellow">rat</span> embryos extend axons while they divide [3-5]. Although proliferation appears to be an important process to expand the sympathetic neuron population during differentiation, the mechanisms that guide sympathoblast proliferation have not been identified.
The development of sympathetic neurons is guided by neurotrophins. Neurotrophin-3 (NT-3) binds to its receptor, TrkC, to promote the survival of cultured sympathoblasts from early lumbar paravertebral ganglia [6]. Nerve growth factor (NGF) signals through its receptor, TrkA, to promote the survival of sympathetic neurons upon target innervation [7]. There are severe sympathetic defects in the superior cervical ganglion of individual NT-3 and NGF knockout <span class="yellow">mice</span> [8-10]. Furthermore, there is no additional cell death in the superior cervical ganglion of NT-3 and NGF double knockout <span class="blue">mouse</span> embryos, suggesting that all of the neurons are dependent on both neurotrophins for survival [11]. There is also an increase in sympathetic neuron cell death in TrkA knockout <span class="yellow">mice</span> [12]. However, in TrkB and BDNF knockout <span class="yellow">mice</span>, there is no apparent phenotype in the superior cervical ganglion, and there is little evidence that TrkB or BDNF is expressed in sympathetic ganglia. Thus, it is generally thought that TrkB and BDNF have little or no roles in guiding the development of sympathetic neurons.
In addition to their developmental functions, neurotrophin receptors regulate cell behavior in neuroblastoma, a tumor found in sympathetic ganglia and adrenal medulla. Tumors that express TrkA often spontaneously regress, while those that express TrkB and its ligand, brain-derived neurotrophic factor (BDNF), grow aggressively, are invasive, and fail to respond to chemotherapeutic agents [13]. The presence of TrkA in neuroblastoma tumors is consistent with its expression in developing sympathetic neurons, and suggests that regressive neuroblastoma tumors arise from early sympathetic neurons that express TrkA. The function of TrkB in early sympathetic development is unknown, which makes understanding the etiology of aggressive neuroblastoma tumors difficult. Based on its function in neuroblastoma tumors, we hypothesize that BDNF and TrkB expression in differentiating sympathoblasts is responsible for expanding the neuronal population through proliferation.
We sought to determine whether BDNF and TrkB are involved in sympathetic development. We report that during early embryonic development, TrkB is expressed in a subset of differentiating sympathoblasts in both avian and <span class="yellow">murine</span> embryos. We also find that BDNF promotes the proliferation of TrkB-positive sympathoblasts in cell culture. However, the majority of TrkB positive cells in vivo fail to take up bromodeoxyuridine (BrdU) over a 24 hr period, suggesting that endogenous BDNF concentrations do not reach a threshold necessary to stimulate proliferation of sympathoblasts. Shortly after all of the TrkB positive cells commence neuronal differentiation, TrkB immunoreactivity is lost. These results suggest that prolonged expression and/or activation of TrkB signaling at these early stages may be an early event triggering the formation of neuroblastoma.<br><br>Results
TrkB is expressed during migration of neural crest cells to sympathetic ganglia
We first determined whether TrkB is expressed in neural crest-derived cells in the region ventral to the spinal cord and dorsal root ganglia where sympathetic ganglia coalesce between Hamburger/Hamilton Stages (St.) 25–28/29. To identify cells that have commenced neuronal differentiation, transverse sections of the lumbar spinal column region were stained with antibodies against Hu C/D [14], a neuronal-specific RNA-binding protein, or Islet-1, a transcription factor found in sympathetic neurons [15]. We found that Hu C/D and Islet-1 are expressed in the same cells both in vivo and in vitro throughout sympathetic development. In experiments done between St. 25 and 28, Islet-1 staining appeared weaker than Hu C/D staining, and thus we used Hu C/D to identify differentiating neurons at these stages. At later stages, Islet-1 was used to facilitate the identification of neurons because of the nuclear location of its immunoreactivity.
Cells expressing Hu C/D are first detected at St. 25 ventral to the spinal cord and dorsal root ganglion and lateral to the dorsal aorta (Figure 1A, 1B). By St. 26, the number of cells that express Hu C/D in this region increases dramatically (Figure 1C). TrkB-expressing cells first appear at St. 27 in the same region and are adjacent to Hu C/D-positive cells (Figure 1D, 2A, 2H). TrkB-positive cells co-localize with a neural crest marker, HNK-1 (Figures 2B, 2C, 2D). The Hu C/D-positive cells in this region are likely to be sympathetic neurons, since they appear in the region where sympathetic ganglia form and express tyrosine hydroxylase, a rate-limiting enzyme in the synthesis of catecholamines (Figures 2E, 2F, 2G). At St. 28/29, the cells begin to coalesce ventral to the dorsal root ganglion and the Hu C/D-positive cells and TrkB-positive cells remain as two separate cell populations (Figure 1E); however, shortly afterwards, all of the TrkB-positive cells begin to express Islet-1 (Figure 3B) and Hu C/D (data not shown).<br><br>Developmental regulation of TrkA, TrkB, TrkC, and BDNF expression
In contrast to TrkB, the other neurotrophin receptors, TrkA and TrkC, are co-expressed in both Hu C/D-positive and Hu C/D-negative cells at St. 27 (Figures 2I, 2J). We find that approximately 30% of the Islet-1-positive cells express TrkA (Figure 3A), while 50% express TrkB (Figure 3B) and 100% express TrkC (Figure 3C) at St. 29/30 (E6.5). Thus, all developing neurons express TrkC in combination with either TrkA or TrkB. By St. 31 (E7), the number of TrkA-positive, Islet-1-positive cells increases to 100% (Figure 3D) and immunoreactivities for both TrkB and TrkC appear dispersed (Figure 3E, 3F). By St. 34 (E8), TrkA expression is well-sustained (Figure 3G) and TrkB and TrkC immunoreactivities are lost (Figure 3H, 3I). We also examined the early development of <span class="yellow">murine</span> sympathetic ganglia (Figure 4). At E13, the newly formed lumbar sympathetic ganglia can be observed ventral to the spinal cord and notochord by their staining for Hu C/D and TH (Figure 4A). TrkB-positive cells can be seen surrounding developing ganglia, as well as in occasional cells within the ganglia (Figure 4C, D). These TrkB-positive cells within the ganglia co-express TH and are seen at a frequency of 1–2 cells per section starting at E13 (Figure 4A–D) and are still present at E15.5 (data not shown).
In neuroblastoma cells, BDNF is co-expressed with TrkB, suggesting that autocrine stimulation is a means by which proliferation is sustained in the transformed cells. To test whether BDNF, the ligand for TrkB, was present in embryonic <span class="yellow">chick</span> sympathetic ganglia, we used quantitative real-time PCR with TaqMan probes to determine the relative abundance of BDNF transcripts in total RNA extracted from lumbar sympathetic ganglia at St. 29/30 (E6.5), St. 31 (E7), St. 34 (E8), and E9. BDNF expression within the ganglia parallels that of TrkB: BDNF mRNA expression levels are highest at St 29/30 (E6.5), and these levels decrease 2-fold at St. 31 (E7) and St. 34 (E8; Figure 5). By E9, BDNF levels are 7 times lower than at St. 29/30 (E6.5; Figure 5).<br><br>NT3 and NGF promote survival of differentiating sympathetic neurons in culture
To determine the effect of neurotrophins, we cultured cells dispersed from lumbar sympathetic ganglia at St. 29/30 (E6.5) because, at this stage, ganglion formation is complete, the number of TrkA-, TrkB-, and TrkC-positive cells have peaked, and all Trk-expressing cells have initiated neural differentiation. First, we identified markers expressed by acutely isolated cells. As shown in Table 1, 80–91% of the cells are p75 neurotrophin receptor (NTR)-positive, indicating that most of the cells are neural crest-derived and little mesenchymal contamination is introduced by the isolation procedure. In addition, 28–33% of the cultured cells express the neural marker Hu C/D. Approximately half of these Hu C/D-positive cells express TrkB. Conversely, all of the TrkB positive cells express Hu C/D. These TrkB-positive cells comprise approximately 14–17% of the total cell population.
We then determined how many of the acutely isolated cells were proliferating by incubating them for 12 hrs in BrdU-containing medium. For these experiments, we identified differentiating neurons with the transcription factor Islet-1 because this marker labels nuclei, thus it co localizes with any BrdU that has been incorporated into the DNA, allowing us to determine whether the cell had undergone S-phase of the cell cycle. After 12 hrs in BrdU, 59% of Islet-1-positive nuclei stain for BrdU immunoreactivity. Thus, cultures of St. 29/30 sympathetic ganglia contain many cells that proliferate while exhibiting markers of neuronal differentiation, confirming previous observations [2]. We call these dividing neuronal precursors sympathoblasts. The remaining non-BrdU incorporating, Islet-1 positive cells are likely to be post mitotic neurons.
Finally, we determined the trophic requirements of St. 29/30 (E6.5) sympathetic neurons and sympathoblasts. We monitored cultures over a three day period after plating and counted the number of phase bright cells with neurites, a morphological feature of both neurons and sympathoblasts. In the absence of trophic factors, more than 2/3 of the cells die by 24 hours in culture and BDNF, NT-3, or NGF alone is not sufficient to promote survival (Figure 6). However, NGF together with NT-3 supports the survival of a significantly larger number of cells (Figure 6). For the subsequent experiments, all neurons were cultured with 25 ng/ml NT-3 and 1 μg/ml 7S NGF to optimize survival.<br><br>BDNF promotes proliferation of TrkB-positive sympathetic neurons in culture
To determine the effects of BDNF, cultures of cells from St. 29/30 (E6.5) sympathetic ganglia were supplemented with 200 ng/ml BDNF and the number of neurons and sympathoblasts were counted at 24, 48, and 72 hours using phase microscopy (Figure 7A). A 1.6-fold increase in the number of neurons due to BDNF is observed by 24 hours and this number does not increase further at 48 or 72 hours. This effect of BDNF is concentration-dependent with an EC50 of 75 ng/ml (Figure 7B).
To test whether the increase in the number of neurons and sympathoblasts caused by BDNF is due to the differentiation of pluripotent neural crest cells, we quantified the effects of BDNF on the number of neurally differentiating cells (Hu C/D-positive) versus the number of non-neuronal cells (Hu C/D-negative) after identifying all neural crest-derived cells by staining for p75NTR in St. 29/30 (E6.5) cultures. If BDNF increases the number of neurons and sympathoblasts by inducing a non-neuronal cell to express Hu C/D, then we expected that the total cell number would remain the same and that there would be a decrease in the number of non-neuronal cells as well as a corresponding increase in the number of neurons. After 24 hours, BDNF significantly increases the number of p75NTR-positive cells as well as the number of Hu C/D-positive cells (Figure 8A). However, there was no statistically significant change in the number of non-neuronal cells. Thus, it is unlikely that BDNF increases the number of neurons and sympathoblasts by inducing differentiation of non-neuronal cells.
To determine whether the increase in the total number of neurons and sympathoblasts is caused by BDNF-induced proliferation, control and BDNF-treated sympathetic cultures were exposed to BrdU for 12 hours after plating, and the number of cells that incorporated BrdU into their DNA was determined after 24 hours in culture. Even in the control condition, a number of cells in the culture are dividing, giving a high baseline of BrdU incorporation (Figure 8B). When BDNF is added, the total number of BrdU positive cells increases approximately 1.6-fold (Figure 8B). This BDNF-induced increase in the total number of BrdU positive cells occurs in sympathoblasts because the number of Islet-1-positive nuclei from control cultures that label with BrdU is 268 ± 59 and BDNF treatment raises this number to 424 ± 80, which corresponds to an increase of 1.6-fold. This accounts for the 1.6-fold increase in total neuron number and total BrdU-positive cells described above. We then confirmed that BDNF acts on TrkB-positive cells: BDNF increases the number of TrkB-expressing cells that incorporate BrdU 2.6 – 4-fold over control (Table 2) and it also increases the overall number of TrkB-positive cells 2 – 2.5-fold over control (Table 2). BDNF does not increase the number of BrdU-positive, TrkB-negative cells or the overall number of TrkB-negative cells (Table 2). In further support that BDNF acts directly on TrkB-expressing cells, an antibody directed against the extracellular domain of TrkB completely prevents the effect of BDNF in promoting proliferation of TrkB-positive, but not TrkB-negative cells (compare Figure 9A to 9B). Thus, the effect of BDNF is restricted to the population that expresses TrkB, which are developing sympathoblasts.
To determine whether TrkB-positive cells are actively proliferating in vivo, embryos were injected with BrdU at St. 27 and harvested at St. 29, approximately 24 hrs later. The majority (85–90%) of TrkB-positive cells do not incorporate BrdU into their nuclei under basal conditions in vivo (Figure 10), although a few TrkB positive cells with labeled nuclei could be observed (arrows). This contrasts with our observation that 71–76% of TrkB-positive cells incorporate BrdU in culture after treatment with BDNF (Table 2), suggesting that endogenous BDNF does not achieve a threshold sufficient to support a high level of sympathoblast proliferation in vivo.<br><br>
Discussion
We report that the neurotrophin receptor TrkB is expressed in a subset of embryonic sympathoblasts during the early development of lumbar paravertebral sympathetic ganglia in <span class="yellow">chicken</span> and <span class="blue">mouse</span> embryos. In the <span class="yellow">chicken</span>, TrkB expression is transient, and completely lost by St 34 (E8). Since BDNF induces the proliferation of sympathoblasts in cell culture, yet in vivo there is little proliferation observed in TrkB-positive cells in nascent ganglia, we propose that if TrkB activation becomes unregulated by excess BDNF or constitutive phosphorylation of TrkB [16], this transient population of TrkB-positive sympathoblasts may trigger the genesis of neuroblastoma, a childhood tumor found in the paravertebral chain and adrenal medulla.
The two populations of sympathoblasts that we observe support previous findings of heterogeneity among developing sympathetic neurons and neural crest cells. Early sympathetic ganglia contain at least two subpopulations: early differentiating neurons that lack TrkB expression and express TrkA and TrkC, and late differentiating sympathoblasts that express TrkB. Explant cultures of sympathetic ganglia from E16 <span class="yellow">chick</span> embryos give rise to two neuronal populations: one that remains close to the explant, and one that migrates away from the explant [1]. In addition, early neuronal subpopulations have been observed in cultures of neural crest cells from St. 13/14 quail embryos as evidenced by the expression of neuronal cell type-specific gangliosides [17]. Perhaps these different subpopulations will ultimately give rise to the two neurochemically distinct populations found in lumbar sympathetic ganglia: the noradrenergic, NPY-containing neurons that innervate internal organs and enteric ganglia and the cholinergic, VIP-containing neurons that innervate vasculature in the hind limbs.
The effects of BDNF and TrkB deletion and over expression have been studied on superior cervical ganglion and preganglionic neurons in thoracic segments of the spinal column, but not on paravertebral sympathetic neurons. In the superior cervical ganglion, an increase in the number of neurons of BDNF null <span class="yellow">mice</span> is likely due to apoptosis induced by BDNF via p75NTR [18]. In contrast, the responses of paravertebral sympathetic neurons to BDNF are complex and subtype dependent. Over expression of BDNF leads to an increase in the number of noradrenergic fibers innervating the erector pilli muscles of hair follicles, while noradrenergic fibers innervating blood vessels were unaffected [19]. If our results indicating that BDNF promotes proliferation of TrkB-positive sympathoblasts in the <span class="yellow">chicken</span> embryo can be extrapolated to the subset of TrkB-positive sympathoblasts in <span class="yellow">murine</span> ganglia, then these TrkB-positive cells may be neurons destined to innervate the erector pilli. In other studies, TrkB null <span class="yellow">mice</span> showed no changes in morphology or cell number in superior cervical ganglia [12] or in the intermediolateral column [20]; but this may not be predictive of a phenotype in the lumbar paravertebral chain. It is thus possible that BDNF/TrkB signaling could play a specific role in other regions of the paravertebral sympathetic chain, such as the lumbar region. However, if TrkB-positive cells are not normally actively proliferating in vivo, then it would not be surprising that the development of the paravertebral sympathetic chain is not disrupted in TrkB or BDNF null <span class="yellow">mice</span>. It may be more informative to examine <span class="yellow">mice</span> that over express BDNF on a promoter that targets expression to embryonic lumbar ganglia. Unfortunately, such <span class="yellow">mice</span> do not exist.
Our findings that the St. 29/30 (E6.5) sympathoblasts are dependent on both NT-3 and NGF for survival in culture are consistent with previous work on <span class="blue">mouse</span> sympathoblasts from the superior cervical ganglion [11]. In these studies, NT-3 and NGF deletion separately led to a decrease in the number of sympathetic neurons at E17.5 compared to control. Deletion of both NT-3 and NGF together did not enhance cell death. In contrast, cultured <span class="yellow">rat</span> superior cervical ganglion sympathetic neurons respond to NT-3 at E14.5 and then to NGF at E19.5, although time points in between were not analyzed [6].
In addition to promoting survival, NT-3, NGF, and BDNF also induce proliferation of various neuronal precursors at different stages of development. NT-3 can promote the incorporation of [3H]-thymidine into cultured quail neural crest cells from the trunk region [21,22], Later in <span class="yellow">rat</span> sympathetic development, NT-3 supports survival of neurons, but does not promote proliferation [6], which is consistent with our results. NGF promotes an increase in BrdU incorporation from 25% to 35% in the DRG cervical segment 2 in the <span class="yellow">chick</span> embryo [23]. In <span class="yellow">chicken</span> embryos that are treated with NGF in ovo at St. 18 and 21, there is an increase in BrdU uptake after formation of the primary sympathetic chain at St. 23 [24]. Since NGF does not appear to affect proliferation of St. 29/30 (E6.5) <span class="yellow">chick</span> sympathoblasts, NGF may only promote proliferation in primary, but not secondary chain sympathoblasts. Motor neuron progenitors in the ventral neural tube from the <span class="yellow">chick</span> embryo express TrkB and when ventral neural tube explants are treated with BDNF, there is an increase in the number of motor neurons produced and BrdU incorporation [25]. BDNF also promotes the proliferation of cultured neuroblastoma cells [13]. Taken together, these results are consistent with our findings that NT-3 and NGF do not promote proliferation of St. 29/30 (E6.5) sympathoblasts, and support the assertion that BDNF promotes proliferation of TrkB-positive sympathoblasts in culture.
Our observations suggest a transient function of TrkB during early sympathetic development in supporting proliferation of this early subpopulation of sympathoblasts. However, the in vivo labeling suggests that only a minority (10–20%) of this population is dividing during the window that TrkB is expressed. In light of the very strong proliferative effect produced in cell culture, these TrkB expressing cells could respond more strongly if endogenous BDNF rises to higher levels, or if the mechanism that down regulates TrkB expression becomes nonfunctional. Such events could trigger an early proliferative event that leads to a cascade of changes that initiates transformation of cells to neuroblastoma. Thus, these early TrkB expressing cells help solve the puzzle as to why TrkB is expressed in aggressive and invasive forms of neuroblastoma, particularly because BDNF induces cultured neuroblastoma cells to become more proliferative, invasive, angiogenic, and resistant to chemotherapeutic reagents than untreated cultures [13]. Future studies will determine whether constitutive expression of BDNF and TrkB in the <span class="yellow">chick</span> embryo sustains proliferation of differentiating sympathoblasts.<br><br>Conclusion
We have identified a time point during development when differentiating lumbar sympathetic neurons transiently express TrkB and proliferate in response to high concentrations of BDNF in culture. These studies suggest that elevated BDNF expression above basal levels and signaling through TrkB may be a mechanism that contributes to the onset of neuroblastoma. A further understanding of the two populations of sympathetic neurons and the fate of the TrkB-positive cells will provide additional insight into the development of paravertebral sympathetic ganglia and the genesis of neuroblastoma.<br><br>Methods
Preparation of tissue for immunohistochemistry
The lumbar spinal column and surrounding tissues were dissected from <span class="yellow">chicken</span> embryos at the indicated stages and placed in Zamboni's fixative (4% (w/v) paraformaldehyde, 15% (v/v) picric acid in 0.1 M sodium phosphate buffer, pH 7.4) for two hours at room temperature. <span class="yellow">Mouse</span> embryos at 13–15 days post-coitus were collected according to an IACUC-approved protocol to Dr. L. Sherman at the Oregon Health and Science University. The <span class="blue">mouse</span> embryos were immersion-fixed in Zamboni's fixative overnight at 4 degrees C then washed with phosphate buffered saline (PBS; 130 mM NaCl, 20 mM sodium phosphate buffer, pH 7.4). Fixed tissues were equilibrated in 30% sucrose in 1× phosphate-buffered saline (PBS). Fixed <span class="blue">mouse</span> embryos were shipped to Vermont in sucrose. Transverse 30 μM sections of the spinal columns were cut at on a Microm HM cryostat (knife temperature: 16°C; object temperature: 23°C) and collected on Superfrost Plus slides (Fisher). Sections were dried at room temperature, washed in 1× PBS and incubated overnight in blocking buffer (1× PBS consisting of 10% (v/v) heat-inactivated <span class="yellow">horse</span> serum (Invitrogen/Gibco), 0.5% Triton X-100 (Sigma), and 0.1% sodium azide (Fisher)).<br><br>Immunohistochemistry
Sections were incubated with primary antibodies overnight at 4°C, followed by incubation with secondary antibodies for 2 hours at room temperature. Primary antibodies used were: <span class="yellow">rabbit</span> anti-p75 (1:1500, generous gift from Louis Reichardt, UCSF [26]), <span class="blue">mouse</span> IgG2b anti-Hu C/D, (1:250, Molecular Probes); <span class="blue">mouse</span> IgG1 anti-Islet-1, (1:10, Developmental Studies Hybridoma Bank); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkA (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkB (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkC (1:500) (all Trk antibodies were generous gifts of Dr. Louis Reichardt, UCSF [26-28]); <span class="blue">mouse</span> anti-HNK-1 (1:50, Developmental Studies Hybridoma Bank); <span class="blue">mouse</span> IgG2a anti-tyrosine hydroxylase (1:10, Developmental Studies Hybridoma Bank), <span class="yellow">sheep</span> anti-BrdU (1:100, Biodesign International), <span class="yellow">rabbit</span> anti-tyrosine hydroxylase (1:100, Chemicon), and <span class="yellow">goat</span> anti-TrkB (1:1000, R&D Systems). Immunofluorescence was imaged using a Nikon C1 confocal mounted on a Nikon Eclipse E800 microscope with a 10× Plan Apo (NA 0.785) air objective or a 60× Plan Apo (NA 1.4) oil objective lens, E7-C1 software, and UV, Argon, and He/Ne lasers exciting at 408, 488, and 543 nm and emitting at 404 500–530, and 555–615 nm, respectively. A Nikon Eclipse E800 microscope in the nearby COBRE Molecular/Cellular Core Facility was used for counting immunofluorescent cells at 200× using epifluorescence optics.<br><br>RNA Extraction/cDNA synthesis
Sympathetic ganglia were removed from <span class="yellow">chick</span> embryos and RNA was isolated using TriReagent (Molecular Research Center), an acidified guanidinium with phenol extraction method [29]. RNA was transcribed to cDNA using oligo-dT with Superscript II Reverse Transcriptase (Invitrogen) at 42°C for 1 hour.<br><br>Real-time PCR
Relative RNA levels were determined using quantitative real-time PCR with an ABI 7500 Fast Real Time PCR System. TaqMan probes were used to quantify the progression of the PCR reaction and reactions were normalized using the constitutively expressed gene <span class="yellow">chick</span> ribosomal binding protein s17 (CHRPS). The sequences were used for primer/probes sets: for BDNF: forward: 5'-AGCCCAGTGAGGAAAACAAG-3', reverse: 5'-ACTCCTCGAGCAGAAAGAGC-3', probe: 5'-[6-FAM]-TACACATCCCGAGTCATGCTGAGCA-[BHQ]-3'; for CHRPS (<span class="yellow">chick</span> ribosomal binding protein S-17): 5'AACGACTTCCACACCAACAA3', reverse: 5'CTTCATCAGGTGGGTGACAT3', probe: 5'-[6-FAM]-CGCCATCATCCCCAGCAAGA [BHQ]-3'. Primers and probes were synthesized by Operon Technologies, Inc (Alameda, CA). The primers for BDNF were validated against primers for CHRPS according to an Applied BioSystems protocol by serially diluting the target cDNA 1:10, determining the cycle threshold (Ct) for each reaction, and plotting the Ct versus log concentration. Slopes of the resulting lines were calculated and primers were accepted if their Ct slopes were between -3.2 and -3.4 (a perfect efficiency of 1.0 yields a slope of -3.3). To analyze the data, the delta Ct method of relative quantification was used, where the Ct of Chrps was subtracted from the Ct of the gene of interest (Delta Ct) and the arbitrary units of mRNA were expressed as 10000/2^(Delta Ct).<br><br>Cell culture
Sympathetic neurons were cultured as previously described [30] with a few modifications. Sympathetic ganglia were removed from the lumbar region of the paravertebral chain of St. 29/30 (E6.5) <span class="yellow">chick</span> embryos and placed in Modified Puck's solution with glucose (MPG). The cells were dissociated by incubation of sympathetic ganglia with 0.1% trypsin in MPG at 37°C for 10 minutes followed by triturating with a fire polished 9" Pasteur pipette. Cells were then resuspended in Dulbecco's Modified Eagle Medium (DMEM) consisting of 10% <span class="yellow">horse</span> serum, 2% fetal <span class="yellow">calf</span> serum, and 10 mg/ml penicillin/streptomycin. For neurotrophin studies, the culture medium was supplemented with 25 ng/ml NT-3 (R & D Systems) and 1 μg/ml 7S NGF (Alomone Labs) upon plating, and 50 ng/ml, 100 ng/ml, or 200 ng/ml BDNF (R & D Systems) once the cells adhered to the wells. Cells were plated on poly-D-lysine/laminin coated wells or cover slips (Fisher) as previously described [30].<br><br>Quantification of neurons and sympathoblasts using phase microscopy
Embryonic sympathoblasts and neurons are small, phase bright cells with neurites. The total number of cells with neurites the length of two cell bodies were counted in 10 non-overlapping fields of view evenly spaced in a grid-like pattern across the bottom of a well from a 24 well plate at 200× using a Nikon Eclipse TE200 microscope.<br><br>BrdU labeling
For in vitro studies, approximately 2 hours after plating cells from St. 29/30 (E6.5) sympathetic ganglia, cells were labeled with 10 μM bromodeoxyuridine (BrdU, Sigma) for 12 hours at 37°C. Following this labeling period, cells were incubated in complete medium without BrdU for an additional 10 hrs. Cells were then fixed in Zamboni's fixative for 30 min at room temperature and rinsed with 1× PBS. For in vivo studies, 25 μg BrdU was injected into the amnion of <span class="yellow">chick</span> embryos at St. 27. The cells and sections were denatured with 2 N HCl at 37°C for 1 hr, and were then neutralized with 0.1 M borate buffer, pH 8.5, for 10 min at room temperature. Immunochemistry was performed as described above.<br><br>
Abbreviations
BDNF, brain-derived neurotrophic factor; BrdU, Bromodeoxyuridine; DA, dorsal aorta; DMEM, Dulbecco's Modified Eagle's Medium; DRG, dorsal root ganglion; E, embryonic day; HS, <span class="yellow">horse</span> serum; MPG, Modified Puck's solution with glucose; NGF, nerve growth factor; NC, notochord; NT, neural tube; NT-3, neurotrophin-3; NTR, neurotrophin receptor; PBS, phosphate-buffered saline; SC, spinal cord; SCG, superior cervical ganglion; SEM, standard error of the mean; SG, sympathetic ganglion; St., stage; w/v, weight/volume; v/v, volume/volume.<br><br>Authors' contributions
JAS designed the experiments, performed the experiments, analyzed the data, and wrote the manuscript. GLSS contributed intellectually to the conception and design of this study, and assisted in the interpretation of the results. RN supervised the study, participated in the design of experiments, edited the manuscript, and obtained funding for the project. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2398786</h3>Genomic-Bioinformatic Analysis of Transcripts Enriched in the Third-Stage Larva of the Parasitic Nematode <span class="yellow">Ascaris suum</span>
Abstract
Differential transcription in <span class="yellow">Ascaris suum</span> was investigated using a genomic-bioinformatic approach. A cDNA archive enriched for molecules in the infective third-stage larva (L3) of <span class="yellow">A. suum</span> was constructed by suppressive-subtractive hybridization (SSH), and a subset of cDNAs from 3075 clones subjected to microarray analysis using cDNA probes derived from RNA from different developmental stages of <span class="yellow">A. suum</span>. The cDNAs (n = 498) shown by microarray analysis to be enriched in the L3 were sequenced and subjected to bioinformatic analyses using a semi-automated pipeline (ESTExplorer). Using gene ontology (GO), 235 of these molecules were assigned to ‘biological process’ (n = 68), ‘cellular component’ (n = 50), or ‘molecular function’ (n = 117). Of the 91 clusters assembled, 56 molecules (61.5%) had homologues/orthologues in the free-living nematodes <span class="yellow">Caenorhabditis elegans</span> and <span class="yellow">C. briggsae</span> and/or other organisms, whereas 35 (38.5%) had no significant similarity to any sequences available in current gene databases. Transcripts encoding protein kinases, protein phosphatases (and their precursors), and enolases were abundantly represented in the L3 of <span class="yellow">A. suum</span>, as were molecules involved in cellular processes, such as ubiquitination and proteasome function, gene transcription, protein–protein interactions, and function. In silico analyses inferred the <span class="yellow">C. elegans</span> orthologues/homologues (n = 50) to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%), or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway. Small numbers of these 50 molecules were predicted to be secreted (10%), anchored (2%), and/or transmembrane (12%) proteins. Functionally, 17 (34%) of them were predicted to be associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority being embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%). A genetic interaction network was predicted for these 17 <span class="yellow">C. elegans</span> orthologues, revealing highly significant interactions for nine molecules associated with embryonic and larval development (66.9%), information storage and processing (5.1%), cellular processing and signaling (15.2%), metabolism (6.1%), and unknown function (6.7%). The potential roles of these molecules in development are discussed in relation to the known roles of their homologues/orthologues in <span class="yellow">C. elegans</span> and some other nematodes. The results of the present study provide a basis for future functional genomic studies to elucidate molecular aspects governing larval developmental processes in <span class="yellow">A. suum</span> and/or the transition to parasitism.<br><br>Introduction
Parasitic nematodes are of major socio-economic importance in animals. For example, hundreds of millions of <span class="yellow">people</span> are infected with geohelminths (soil-transmitted worms), such as blood-feeding hookworms <span class="yellow">Ancylostoma duodenale</span> and/or <span class="yellow">Necator americanus</span>, <span class="yellow">Trichuris trichiura</span> and Ascaris spp. [1], causing serious adverse effects on <span class="yellow">human</span> health, particularly in <span class="yellow">children</span>. Similarly, parasitic nematodes of livestock, such as <span class="yellow">pigs</span>, also cause substantial economic losses due to subclinical and clinical diseases, with billions of dollars spent annually on the treatment and control of gastro-intestinal nematodes. In addition to the socioeconomic impact that these parasites have, there is potential for the emergence of resistance in them against all of the main classes of (nematocidal) compounds used to treat the diseases they cause [2]–[5]. Therefore, there is a significant need to work toward discovering new compounds to control these parasites. Gaining an improved understanding of the molecular basis of parasite development provides such an avenue.
Compared with the free-living nematode <span class="yellow">Caenorhabditis elegans</span>, there is very little information on fundamental molecular aspects of development in parasitic nematodes [6]–[8]. Since the genome sequence of <span class="yellow">C. elegans</span> was published in 1998 [9], many aspects of the molecular biology of this nematode have been elucidated. For instance, microarray analyses have been used to examine developmental and gender-enriched gene expression [10],[11], and the functions of more than 96% of the <span class="yellow">C. elegans</span> genes have been assessed by double-stranded RNA interference (RNAi, or gene silencing; [12]) [13]–[18]. Comparative analyses of genetic data sets have shown that parasitic nematodes usually share ∼50–70% of genes with <span class="yellow">C. elegans</span> (e.g., [19],[20]). There is similarity in other features (such as basic body plan and moulting) between <span class="yellow">C. elegans</span> and parasitic nematodes, suggesting that some molecular pathways are relatively conserved [8],[21]. Understanding the pathways linked to basic nematode biology and development could have important implications for finding new ways of disrupting these pathways and thus facilitate the identification of new drug targets.
Despite the advances in genomic technologies [7], [22]–[29] and the study of <span class="yellow">C. elegans</span>, there is a paucity of information on the genomics of parasitic nematodes of animals, particularly in relation to development. Also considering the major socioeconomic impact of Ascaris and ascariasis in <span class="yellow">humans</span> and <span class="yellow">pigs</span> [30]–[32], several characteristics, including the large size of the adult worm (providing the opportunity of investigating individual organ systems and tissues), the ability to maintain Ascaris in the <span class="yellow">pig</span>, store eggs and culture larvae in vitro for relatively long periods of time (months to years) [32] as well as the discovery that RNAi achieves “cross-species” gene silencing for a selected number of genes [33],[34] and the imminent genome sequence (http://www.sanger.ac.uk/Projects/Helminths/) all indicate that Ascaris could serve as a powerful model system for investigating reproductive and developmental processes in nematodes.
In the present study, Ascaris from <span class="yellow">pigs</span> was used to study molecules abundantly transcribed in the infective third-stage larva (L3). Following the oral ingestion of Ascaris eggs by the host (<span class="yellow">human</span> or <span class="yellow">pig</span>), L3s are released and then invade/penetrate predominantly the caecal wall [35] to then undergo hepato-pulmonary migration, after which ultimately the adult females and males establish and develop in the small intestine [36],[37]. The molecular mechanisms linked to host invasion and parasite development are largely unknown. Here, we constructed an L3-enriched cDNA library using the method of suppressive-subtractive hybridization (SSH), explored transcription of a representative subset of molecules by microarray analysis and conducted bioinformatic analyses to characterize these molecules, map them to biochemical pathways and predict genetic interactions based on comparisons with <span class="yellow">C. elegans</span> and/or other organisms.<br><br>Materials and Methods
Production of Different Developmental Stages of Ascaris
Experimental <span class="yellow">pigs</span> (8–12 weeks of age) were purchased from and maintained in the Experimental Animal Center of South China Agricultural University. These <span class="yellow">pigs</span> were treated humanely, according to the Animal Ethics procedures and guidelines of the People's Republic of China. Adult <span class="yellow">worms</span> (males and females) of <span class="yellow">A. suum</span> were collected from the small intestines of <span class="yellow">pigs</span> from an abattoir in Shenzhen, China. Infective eggs and infective L3s of <span class="yellow">A. suum</span> were produced according to the methods described previously [38]. In brief, eggs from the uteri of adult females of <span class="yellow">A. suum</span> were collected and incubated at 28°C for 28 days to allow them to develop to infective eggs (containing infective L3s). To obtain pure infective L3s, 7.5% v/v sodium hypochlorite was used to treat the larvated eggs at 37°C overnight and then the eggs were shaken with glass-beads; then, the exsheathed L3s and shells were separated by density gradient centrifugation using lymphocyte separating medium (LSM) [38]. Following the experimental infection of helminth-free <span class="yellow">pigs</span> with infective Ascaris eggs as described previously [39], the L3s from livers and in lungs as well as L4s in intestines were isolated according to an established method [40]. All parasite materials were snap-frozen in liquid nitrogen prior to storage at −70°C.<br><br>Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)
Total RNA was isolated from adult females and males, different larval stages or eggs of <span class="yellow">A. suum</span> using TriPure reagent (Roche) as recommended by the manufacturer. Equal amounts of total RNA from each stage or sex were pooled. The mRNA was isolated using the Oligotex mRNA Kit (Qiagen), following the manufacturer's protocol. SSH was carried out using the PCR-Select cDNA Subtraction kit (Clontech), according to the manufacturer's protocol. In brief, cDNA synthesized from mRNAs from infective L3s was subtracted against cDNA synthesized from the pooled mRNA from all other stages included herein. The SSH library was constructed using infective L3s as the tester and pooled cDNAs from all other stages as the driver. The effectiveness of this subtraction process has already been demonstrated in previous studies [41],[42]. The cDNA obtained following SSH was cloned into the pGEM-T Easy plasmid vector (Promega) and competent <span class="yellow">Escherichia coli</span> (JM109) transformed. Positive clones, picked randomly (based on blue/white selection), were grown overnight in Luria Bertani (LB) medium (shaking, 37°C). Individual inserts were PCR-amplified using “nested primers” 1 and 2R from the Subtraction kit (Clontech) and examined by agarose electrophoresis.<br><br>Preparation of Microarray Slides
Clones (n = 3075) from the subtracted library were picked and cultured overnight in LB containing ampicillin (1000 IU/ml) in sealed 96-well blocks. Five µl of culture suspension from each well were transferred into individual wells thermocycling (96-well) plates and the inserts PCR-amplified using primers 1 and 2R. Following a 10 min denaturation step at 94°C, the amplification proceeded for 25 cycles of 10 s at 94°C, 30 s at 68°C and 1.5 min at 72°C, with a final extension for 5 min at 72°C. Products were resolved in agarose gels, ethanol precipitated, re-suspended in 16 µl of “spotting solution” (Shanghai BioStar Genechip, Inc) to a final concentration of ∼500 ng per µl, before being printed on to glass slides (in duplicate) using a robotic arrayer. Sixteen blanks (using spotting solution only) and the same number of negative (irrelevant cDNAs with no relationship to Ascaris) were also printed on to slides and served as negative controls; β-actin of <span class="yellow">A. suum</span> served as a positive control to assess the efficiency of labeling and hybridization. The slides were air-dried for 2 h, and cDNA in the spots were cross-linked at 254 mJ. The printed slides were stored at 4°C.<br><br>Labeling of cDNA Probes with Fluorescent Dyes, and Microarray Analysis
The cDNAs produced from total RNA from <span class="yellow">A. suum</span> eggs, infective L3s, L3s isolated from <span class="yellow">pig</span> liver or lung, fourth-stage larvae (L4s), adult males or females [as described in the section ‘Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)’] were labeled with cyanine dyes. Cy3 or Cy5-dCTP was incorporated into cDNA produced from 30 µg of total RNA by direct labeling in a reverse transcription reaction using an oligo (dT) primer. Labeled cDNA was purified using DyeEx columns (Qiagen).
Microarray slides were incubated with a pre-hybridization solution [5×SSC, 1% <span class="yellow">bovine</span> serum albumin (BSA), 0.1% sodium dodecyl-sulphate (SDS)] for 6 h at 42°C. After pre-hybridization, the microarray slides were incubated with ‘pooled’ Cy3 and Cy5-labeled probes in hybridization solution (5×SSC, 1% BSA, 0.1% SDS), in the dark at 42°C for 18 h, and then washed in solution I (1×SSC, 0.2% SDS) for 10 min, followed by solution II (0.1×SSC, 0.2% SDS) for 10 min at 60°C, according to the protocols provided by Shanghai BioStar Genechip, Inc. A “dye flip” was carried out to control for any bias in hybridization signal between the Cy-labeled cDNA probes (produced for two distinct mRNA populations). The slides were dried and scanned (ScanArray 4000 scanner) using image acquisition software (Shanghai BioStar Genechip Inc.) and a range of laser power and photo-multiplier tube intensities. The mean hybridization signal (derived from four replicates of the same array) were corrected for background, normalized [43], log2-transformed and then subjected to statistical analysis employing the students t-test in a spreadsheet (Excel, Microsoft, USA). The microarray data were analysed for differential cDNA hybridization (>2.0-fold to 114.3-fold) between L3 and each of the other stages (eggs, lung and liver L3s, L4, adult female and adult male).<br><br>Verification of Differential Hybridization by Reverse Transcription-Coupled Polymerase Chain Reaction (RT-PCR) Analysis
For a subset (n = 17) of representative ESTs (rESTs), RT-PCR was used to verify the differential transcription recorded by microarray analysis. Double-stranded cDNA was synthesized from total RNA (separately) from each stage or sex of <span class="yellow">A. suum</span> using reverse transcriptase (Superscript III, Invitrogen). Briefly, 5 µg of total RNA were added to 14 µl of H2O and 1 µl of oligo d(T)n = 12–18 primer (0.5 µg/µl), heated to 70 °C for 10 min and chilled on ice. First- and second-strand cDNAs were synthesized via the addition of 4 µl of first-strand cDNA buffer (250 mM Tris-HCl, pH 8.3, 375 mM KCl and 15 mM MgCl2), 2 µl of 0.1 M dithiothreitol, and 1 µl of 10 mM of each dNTP, followed by an incubation at 25 °C (10 min), 42 °C (50 min) and 70 °C (15 min). One-tenth of each double-stranded cDNA produced was then used as a template in the PCR. The transcripts were amplified from individual cDNAs by PCR using oligonucleotide primers (sequences available upon request) designed to each EST. The PCR amplification of a portion (209 bp) of the β-actin gene (accession no. BI594141) using forward primer (5′-CTCGAAACAAGAATACGATG-3′) and reverse primer (5′- ACATGTGCCGTTGTATGATG-3′), previously determined to be present in all developmental stages and both sexes of <span class="yellow">A. suum</span> [44], served as a positive control. Samples without template (no-DNA controls) were included in each PCR run. The following cycling conditions were employed: one cycle at 94 °C (5 min), 94 °C (30 s), 60 °C (30 s) and 72 °C (30 s) for 30 cycles, followed by a final extension of 70 °C (7 min). Following the PCR, 5 µl of individual amplicons were resolved in ethidium bromide-stained agarose gels (2%) and then photographed upon transillumination. The relative band intensities were analyzed using UVIsoft Image Acquisition and Analysis software (UVITEC). The specificity and identity of individual amplicons were confirmed by direct sequencing using the same primers (separately) as employed for their amplification.<br><br>Sequencing and Bioinformatics Analyses
Clones from the SSH cDNA library with increased hybridization in microarray analysis to the infective L3 compared with other stages were sequenced using standard technology [45]. The nucleotide sequences have been deposited in the GenBank database under accession numbers ES290984-ES291074. Following the processing of the sequences (i.e., removal of vector sequences, quality assurance and clustering), contigs or singletons from individual clusters were subjected to BLASTx (NCBI: www.ncbi.nlm.nih.gov) and BLASTn (EMBL-EBI Parasite Genome Blast Server: www.ebi.ac.uk) analysis to identify putative homologues in <span class="yellow">C. elegans</span>, other nematodes and other organisms (e-value of ≤1e-05). Peptides inferred from ESTs were classified functionally using Interproscan (available at http://www.ebi.ac.uk/InterProScan/) employing the default search parameters. WormBase (www.wormbase.org) was interrogated extensively for relevant information on <span class="yellow">C. elegans</span> homologues/orthologues, including RNAi phenotypic, transcriptomic, proteomic and interactomic data. ESTs with homologues/orthologues in <span class="yellow">C. elegans</span> and other nematodes were also subjected to analysis employing the KEGG Orthology-Based Annotation System (KOBAS) (www.kobas.cbi.pku.edu.cn), which predicts the biochemical pathways in which molecules are involved. The open reading frames (ORFs) inferred from selected ESTs with orthologues in <span class="yellow">C. elegans</span> were also subjected to “secretome analysis” using the program SignalP v.2.0 www.cbs.dtu.dk/services/SignalP/), employing both the neural network and hidden Markov models to predict signal peptides and/or anchors [46]–[48]. Also, transmembrane domains were predicted using the program TMHMM (www.cbs.dtu.dk/services/TMHMM/; [49]–[51]), and subcellular localization inferred employing the program WoLF PSORT (http://wolfpsort.org/; [52]).
The method established by Zhong and Sternberg [53] was used to predict the interactions for <span class="yellow">C. elegans</span> orthologues of the L3-enriched molecules from Ascaris. In brief, interaction, phenotypic, expression and gene ontology data from <span class="yellow">fruitfly</span>, <span class="yellow">yeast</span>, <span class="blue">mouse</span> and <span class="yellow">human</span> were integrated using a naïve Bayesian model to predict genetic interactions among <span class="yellow">C. elegans</span> genes ([45],[53]; Zhong and Sternberg, unpublished). The predicted networks resulting from the analyses were saved in a graphic display file (gdf) format and examined using the graph exploration system available at http://graphexploration.cond.org/. Images were labeled and saved in the joint photographic experts group (jpeg) format.<br><br>
Results
To identify molecules transcribed abundantly in the L3 of <span class="yellow">A. suum</span>, an enriched cDNA library was constructed by SSH. From a total of 3075 clones from this library, 2921 (95%) were shown to contain an insert (which could be amplified by PCR). From 2671 (92%) of these clones, amplicons representing single bands of ∼400 to 600 bp in size were produced. These latter amplicons were arrayed (in duplicate) on to slides and then hybridized with Cy3-labeled L3-cDNA or with Cy5-labeled cDNA from eggs, liver/lung L3s, L4s, adult female or adult male of Ascaris. Dye flip was conducted to verify the hybridization data. Of the 2671 (duplicate) spots, 1526 had a significant difference in hybridization between infective L3 cDNA and cDNAs from all other stages or sexes of <span class="yellow">A. suum</span>, of which 515 had a >2.0-fold increased hybridization for the L3.
In order to independently verify the hybridization results in the microarray, a PCR-based analysis of a selected subset (n = 17) clones was conducted using specific primer pairs. Having verified the specificity and identity of individual amplicons by sequencing, PCR results were reproducible (based on multiple runs on different days) and ∼94% (16 of 17) concordant with those of the microarray analysis (not shown). There was complete concordance for representative clones associated with a differential signal of ≥3.0-fold in the microarray.
The clones linked to the 515 spots representing increased transcription (>2.0-fold) in infective L3 compared with the other developmental stages or sexes included were subjected to sequencing. The 498 sequences (length: 550±115 bp) determined were then subjected to detailed bioinformatic analyses. There were 91 unique clusters (accession numbers ES290984-ES291074), of which 55 were singletons (sequences determined once). Of 56 molecules (61.5%) with significant similarity to sequences other than <span class="yellow">A. suum</span> in the databases interrogated, 50 (54.9%) had <span class="yellow">C. elegans</span> or <span class="yellow">C. briggsae</span> homologues, and six had similarity to ESTs already sequenced from ascaridoid and/or other parasitic nematodes, and/or other organisms (Table 1). A significant proportion (38.4%) did not have any similarity in sequence to any organisms for which data are presently available. Comparative analysis specifically against <span class="yellow">A. suum</span> EST data sets (n∼42,000) available in public databases confirmed independently that the majority of molecules (>60%) were present exclusively in the infective L3 stage or were orphans.
As gene ontology (GO) provides a hierarchy that unifies the descriptions of biological, cellular and molecular functions [54], this approach was employed to predict the classification and gene function of molecules enriched in infective L3 of <span class="yellow">A. suum</span>. A summary of the GO categories of these molecules is displayed in Fig. 1. Of the 91 contigs, 32 (35%) could be functionally assigned to ‘biological process’ (n = 38), ‘cellular component’ (n = 17) and ‘molecular function’ (n = 64). The most common subcategories were gluconeogenesis (13%) and metabolic process (13%) within ‘biological process’, extracellular region (24%) within ‘cellular component’, and catalytic activity (11%) and phosphoenolpyruvate carboxykinase activity (8%) within ‘molecular function’ (Table S1).
A focused KOBAS analysis inferred the 50 <span class="yellow">C. elegans</span> orthologues/homologues to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%) or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway (Table 2). Of these 50 molecules, small numbers were predicted to be secreted (10%), anchored (2%) and/or transmembrane (12%) proteins (Table 2). Functionally, 17 (34%) of the 50 molecules were associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority displaying embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%) (Table 2).
Extending this analysis, a relatively complex genetic interaction network was predicted for the 17 <span class="yellow">C. elegans</span> orthologues (i.e., with non-wild-type RNAi phenotypes) (see Table S2). Statistically highly significant interactions were predicted for nine of the <span class="yellow">C. elegans</span> genes; the top five interactors are displayed in Fig. 2. The gene ontology categories for eight selected <span class="yellow">C. elegans</span> genes (F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2) included: embryonic development, egg hatching, larval development and/or growth. The other categories included: positive regulation of growth rate (F55A12.8, kin-2, mup-2, pab-1, rpl-22 and T21B10.2) and gamete generation and locomotory behaviour (kin-2, mup-2, pab-1 and F55A12.8, kin-2, mup-2, respectively). The <span class="yellow">C. elegans</span> homologue egl-3 was predicted to be involved in proteolysis (see www.wormbase.org). All nine <span class="yellow">C. elegans</span> orthologues were predicted to interact directly with a total of 296 (range: 5–75) other genes and, in particular, a direct genetic interaction was predicted between pab-1 and T21B10.2 (Fig. 2). The 296 interactors were associated with embryonic and larval development (n = 198; 66.9%), information storage and processing (n = 15; 5.1%), cellular processes and signalling (n = 45; 15.2%) and metabolism (n = 18; 6.1%); the precise function of some of the interactors (n = 20; 6.7%) is presently unknown (Table S2).<br><br>Discussion
The present study investigated transcripts in infective L3s of <span class="yellow">A. suum</span> using a genomic-bioinformatic platform. The focus was on comparisons with <span class="yellow">C. elegans</span> homologues/orthologues, because the entire genome sequence of this nematode is known [9] and because there is a wealth of information on the localization and functionality of its molecules (www.wormbase.org; http://elegans.bcgsc.bc.ca/knockout.shtml). The functions of most genes in <span class="yellow">C. elegans</span> have been assessed using RNAi (e.g., [14],[15],[17],[55],[56]) in the hermaphroditic stage, whereas there is a paucity of functional information available for Ascaris and other parasitic nematodes of animals [57],[58].
Following the microarray analysis of >2500 ESTs from the SSH library, 498 cDNAs inferred to be enriched in the L3, based on hybridization signal, were sequenced and subjected to comprehensive in silico analyses. Of the 91 clusters of molecules categorized, 50 (54.9%) had <span class="yellow">C. elegans</span> homologues/orthologues with loss-of-function phenotypes could be mapped to key pathways. The statistically significant genetic interactions predicted for 9 of the 50 <span class="yellow">C. elegans</span> orthologues [namely egl-3, F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2 ( = enol-1)] and the interaction network included genes encoding kinases, alpha-tubulins, enolases, troponin and other named and unnamed proteins. Eight of these molecules (enol-1, pab-1, F33D11.10, rpl-22, F55A12.8 mec-12, mup-2 and kin-2) have known or predicted roles in embryonic and larval growth and development, gamete generation, locomotory behaviour or other biological processes in <span class="yellow">C. elegans</span> (see www.wormbase.org).
The enolase encoded by enol-1 is predicted to play a role in glycolysis, gluconeogenesis, phenylalanine, tyrosine and tryptophan biosynthesis (cf. [59]). Since glucose is the main source for ATP production, the alteration in these key glycolytic enzymes may lead to cellular dysfunction, such as impaired ion-motive ATPase required to maintain potential gradients, operate pumps and maintain membrane lipid asymmetry [60]. Bioinformatic analysis for transmembrane helices (TMHMM) and peptide signal sequences (SignalP) predicted ENOL-1 to be a non-secreted protein localized to the cytoplasm (cf. Table 2). Nonetheless, enolases are often detected in the excretory/secretory (ES) products of parasitic helminths, including adult <span class="yellow">A. suum</span> [61], and appear to play a role in the triggering of nitric oxide production by host cells. The enol-1 orthologue of <span class="yellow">C. elegans</span> has been predicted to interact specifically with the polyadenylate binding protein gene, pab-1, inferred to be involved in coordinated gene transcription and expression during normal larval development [16]. Poly(A)-binding proteins (PABPs) are recognized to be central to the regulation of mRNA translation and stability [62]. Present evidence suggests that the expression of PAB-1 is regulated by an oligo-pyrimidine tract in response to cell growth and relates to coordinated growth regulation in <span class="yellow">C. elegans</span> [62]. Furthermore, gene silencing of pab-1 and its selected interactors (see Fig. 2) leads to embryonic lethal (Emb), slow growth (Slo) and sterile progeny (Stp) phenotypes (see www.wormbase.org).
Another gene (F33D11.10; EST code 4F10; see Table 2) which encodes an ATP-dependent RNA helicase and is associated with embryonic lethal (Emb) and larval lethal (Lvl) RNAi phenotypes, was shown to be highly transcribed in infective L3s of <span class="yellow">A. suum</span>. Helicases are involved in a variety of RNA metabolic processes, including translation initiation, pre-mRNA splicing, pre-rRNA processing, rRNA maturation and RNA degradation [63], and are crucial for life cycle progression, sex determination and early embryogenesis in <span class="yellow">C. elegans</span> [60]. The high transcription levels of a homologue/orthologue in the L3 of <span class="yellow">A. suum</span> might suggest a similar role in this ascaridoid. Similarly, the coordination of the expression of a large number of genes is required for normal growth and cell proliferation during larval development. The high transcription level for the ribosomal protein gene homologue rpl-22 (large subunit family member; EST code 26G12, see Table 2) in the infective L3 of <span class="yellow">A. suum</span> compared with other developmental stages is likely to reflect the substantial rate of cell growth in this stage [64].
The gene (F55A12.8, EST code 4G11; see Table 2) encoding an acetyl-transferase with a putative ATPase domain, shown to be enriched in the L3 of <span class="yellow">A. suum</span>, was predicted to interact with 75 other genes all involved in energy production and/or RNA processing (see Table S2). Several molecules involved in ATP synthesis and mitochondrial pathways (e.g., cytochrome oxidase c subunits 1, 2 and 3, ADP/ATP translocases, NADH dehydrogenases, ATPases and ATP synthetases) have been reported previously to be highly represented in the L3 stage of <span class="yellow">Anisakis simplex</span> [65], thus supporting the proposal that substantial energy is required for larval development as well as the transition from the free-living to the parasitic stage and the invasion of the host. There is also likely to be a substantial energy requirement for muscle contraction linked to larval motility in <span class="yellow">A. suum</span>, as the L3s penetrate the caecal wall in the porcine host, before undergoing hepato-pulmonary migration [35]. Accordingly, genes encoding a specialized tubulin expressed in mechanoreceptors (mec-12, EST code 13E09) and a troponin (mup-2, EST code 01G03; see Table 2), both predicted to interact with a total number of 32 tubulin- and myosin-encoding genes, also supported a link to extensive muscle contraction and motility in <span class="yellow">A. suum</span> L3s. Also, neuroactive peptides are required to regulate the responsiveness of nematode larvae to mechanical stimuli [66]. A homologue encoded by egl-3 was shown to be highly transcribed in the L3 of <span class="yellow">A. suum</span>; EGL-3 is predicted to be a pro-hormone convertase involved in the maturation of neuropeptides and could be associated with mechano-sensory responses and touch sensitivity linked to the host invasion.
A regulatory subunit of a cAMP-dependent protein kinase (kin-2, EST code 22H01; see Table 2) was predicted to interact with 72 other genes all involved in diverse cellular processes, such as nuclear trafficking, and DNA replication and repair (see Table S2). Based on gene ontology terms, kin-2 is implicated in gamete generation, growth, larval development, post-embryonic body morphogenesis, signal transduction and/or protein amino acid phosphorylation (see Table S2). Gene silencing of kin-2 in <span class="yellow">C. elegans</span> leads to phenotypes, such as larval lethal (Lvl), larval arrest (Lva), body morphology defect (Bmd), dumpy (Dpy), uncoordinated (Unc) and sterile progeny (Stp) (www.wormbase.org), suggesting that its homologue in <span class="yellow">A. suum</span> is central to larval maturation. The KOBAS analysis predicted the protein KIN-2 to be involved in the insulin-signaling pathway, previously implicated in controlling the exit from dauer in <span class="yellow">C. elegans</span> and the activation of L3s of the <span class="yellow">canine hookworm</span>, <span class="yellow">Ancylostoma caninum</span>, following exsheathment [67]. In a recent study, Brand and Hawdon [68] were able to inhibit (with a phosphoinositide-3-OH-kinase inhibitor) the activation of infective L3s of both of the hookworms <span class="yellow">Ancylostoma caninum</span> and <span class="yellow">Ancylostoma ceylanicum</span> via the insulin signaling pathway, thus lending some credence to the hypothesis that this pathway plays an critical role in regulating the transition from the free-living to the parasitic stage [68]. Recently, it has been proposed that transcriptional and feeding responses to serum-stimulation in <span class="yellow">Ancylostoma caninum</span> are regulated by parallel systems, with the insulin signaling pathway playing a significant role in the ‘resumption of feeding’ in activated larvae [69].
Protein kinases are also likely to be involved in pathways linked to sexual maturation in developing larvae. As already proposed for adult stages of <span class="yellow">H. contortus</span> [45], the protein kinase gene cdk-1 is predicted to play a pivotal role in the germline, oogenesis and spermiogenesis pathways of this parasitic nematode. Other protein kinases, such as PEPCK, and phosphatases, were shown herein to be transcribed at high levels in the L3 stage compared with other developmental stages of <span class="yellow">A. suum</span> (see Table 2), which is in accordance to findings reported recently for <span class="yellow">Anisakis simplex</span> [65]. Due to their major regulatory effects in eukaryotic signaling events and regulatory and sensory functions, protein kinases have been considered interesting targets for anti-parasitic drugs [70].
In conclusion, this study has given some interesting insights into early molecular processes in the L3 of <span class="yellow">A. suum</span>. Approximately 60% of the transcripts enriched in the L3 stage of <span class="yellow">A. suum</span> have homologues/orthologues in <span class="yellow">C. elegans</span>. The bioinformatic analyses of selected molecules suggest that a complex genetic network regulates or controls larval growth and development in <span class="yellow">A. suum</span> L3s, and some of these might be involved in or regulate the switch from the free-living to the parasitic stage. Some caution is warranted in drawing conclusions regarding molecular mechanisms regulating the transition to parasitism in parasitic nematodes from information on <span class="yellow">C. elegans</span>, as latter is a free-living nematode. Also, while the method of data integration is essential for the reliable prediction of genetic interactions, it might limit the capacity of the approach somewhat to infer nematode-specific interactions. As additional datasets of genes and gene functions become available for various parasitic nematodes, more informed inferences can be made regarding the functions of nematode-specific genes, particularly those involved in the transition to parasitism. The imminent genome sequence of <span class="yellow">A. suum</span> (http://www.sanger.ac.uk/Projects/Helminths/) should all assist in this endeavour. Also, functional analysis of selected molecules representing selected ESTs identified herein, utilizing gene silencing approaches established recently [33],[34], could provide some insights into developmental processes in Ascaris and related ascaridoid nematodes and provide avenues for the development of novel approaches for their control.<br><br>Supporting Information<br><br>
<h3>pmcA2412862</h3>Minocycline attenuates lipopolysaccharide (LPS)-induced neuroinflammation, sickness behavior, and anhedonia
Abstract
Background
Activation of the peripheral innate immune system stimulates the secretion of CNS cytokines that modulate the behavioral symptoms of sickness. Excessive production of cytokines by microglia, however, may cause long-lasting behavioral and cognitive complications. The purpose of this study was to determine if minocycline, an anti-inflammatory agent and purported microglial inhibitor, attenuates lipopolysaccharide (LPS)-induced neuroinflammation, sickness behavior, and anhedonia.<br><br>Methods
In the first set of experiments the effect of minocycline pretreatment on LPS-induced microglia activation was assessed in BV-2 microglia cell cultures. In the second study, adult (3–6 m) BALB/c <span class="yellow">mice</span> received an intraperitoneal (i.p.) injection of vehicle or minocycline (50 mg/kg) for three consecutive days. On the third day, <span class="yellow">mice</span> were also injected (i.p.) with saline or <span class="yellow">Escherichia coli</span> LPS (0.33 mg/kg) and behavior (i.e., sickness and anhedonia) and markers of neuroinflammation (i.e., microglia activation and inflammatory cytokines) were determined. In the final study, adult and aged BALB/c <span class="yellow">mice</span> were treated with the same minocycline and LPS injection regimen and markers of neuroinflammation were determined. All data were analyzed using Statistical Analysis Systems General Linear Model procedures and were subjected to one-, two-, or three-way ANOVA to determine significant main effects and interactions.<br><br>Results
Minocycline blocked LPS-stimulated inflammatory cytokine secretion in the BV-2 microglia-derived cell line and reduced LPS-induced Toll-like-receptor-2 (TLR2) surface expression on brain microglia. Moreover, minocycline facilitated the recovery from sickness behavior (i.e., anorexia, weight loss, and social withdrawal) and prevented anhedonia in adult <span class="yellow">mice</span> challenged with LPS. Furthermore, the minocycline associated recovery from LPS-induced sickness behavior was paralleled by reduced mRNA levels of Interleukin (IL)-1β, IL-6, and indoleamine 2, 3 dioxygenase (IDO) in the cortex and hippocampus. Finally, in aged <span class="yellow">mice</span>, where exaggerated neuroinflammation was elicited by LPS, minocycline pretreatment was still effective in markedly reducing mRNA levels of IL-1β, TLR2 and IDO in the hippocampus.<br><br>Conclusion
These data indicate that minocycline mitigates neuroinflammation in the adult and aged brain and modulates the cytokine-associated changes in motivation and behavior.<br><br><br><br>Background
The bi-directional communication between the immune system and the central nervous system (CNS) is necessary for mounting the appropriate immunological, physiological, and behavioral responses to immune stimulation [1]. CNS innate immune cells including microglia and macrophages play integral roles in receiving and propagating inflammatory signals that are initiated at the periphery. Activation of peripheral innate immune cells elicits the secretion of inflammatory cytokines, including interleukin (IL)-1, IL-6, and tumor necrosis factor-α (TNFα.), that use neural [2,3], humoral [4] and blood brain barrier pathways [5] to relay this signal to the CNS. This inflammatory signal, in turn, induces CNS macrophages and microglia to produce the same cytokines [6], which target neuronal substrates and elicit a sickness behavior syndrome that is normally adaptive and beneficial to the host [1]. An amplified or excessive inflammatory cytokine response in the brain, however, is associated with a myriad of complications including cognitive dysfunction [7-10], prolonged sickness behavior [11-14], and depressive-like behavior [15].
Microglia are primarily involved in immune surveillance [16,17], but when activated have macrophage-like capabilities including phagocytosis, inflammatory cytokine production, and antigen presentation [18]. Normally these neuroinflammatory changes are transient with microglia returning to a resting state as the immune stimulus is resolved. Aging or neurological disease, however, may provide a brain environment where microglia are more "reactive or primed" to a peripheral immune challenge [19]. Recent findings indicate that several markers of glial activation such as major histocompatibility complex (MHC) class II, complement receptors, and scavenger receptors are increased in brain during normal aging [13,20-26]. Furthermore, we and others have reported that a biological consequence of this reactive glial profile is an exaggerated neuroinflammatory response to innate immune challenge [9,10,12-14,27,28].
Active microglia and CNS macrophages also contribute to the production of oxidative and neuroactive mediators that may influence behavior. For instance, inflammatory cytokines in the CNS upregulate the enzyme IDO [29,30], which metabolizes tryptophan (TRP) into L-kynurenine (KYN) [31]. TRP degradation to KYN can reduce TRP levels that are required for serotonin synthesis [32] and can lead to the production of neuroactive mediators including 3-hydroxykynurenine (3HK) and quinolinic acid (QUIN) [31]. High levels of 3HK and QUIN induce neuronal damage through oxidative stress [33] and over stimulation of N-methyl-D-aspartate (NMDA) receptors [34,35]. A recent study indicates that while several cell types in the CNS express IDO, only microglia maintain all the enzymes required to produce 3HK and QUIN [36]. Because IDO mediated TRP degradation impacts both serotonergic and glutamatergic pathways, this may be an important mechanism underlying mood and behavior complications concomitant with inflammation [37-39].
Because activated microglia are suspected to cause or exacerbate several neurodegenerative diseases, pharmacological strategies to suppress microglial activity are being explored as therapies. Minocycline is a tetracycline derived antibiotic that has anti-inflammatory properties in the CNS that are separate from its antimicrobial action [40]. Minocycline readily crosses the blood brain barrier and attenuates inflammation associated with microglial activation. For example, minocycline blocks the deleterious effects of neuroinflammation on neurogenesis, long-term potentiation, and neuronal survival [41-43]. The mechanism of action is unclear, but recent studies indicate that minocycline abrogates MAPkinase and NFκB dependent signaling pathways in primary microglia and microglia cell cultures [44]. Moreover, in the brain of <span class="yellow">rats</span>, minocycline abrogates microglial expression of CD11b and MHC II through a protein kinase-c dependent mechanism [45]. This is relevant because minocycline attenuates neuroinflammation in several <span class="yellow">rodent</span> models of disease including Amyotrophic Lateral Sclerosis [46], Experimental Autoimmune Encephalomyelitis (EAE) [45] and MPTP-induced Parkinson's disease [47]. However, the extent to which minocycline facilitates the recovery from cytokine-mediated sickness behavior is unknown.
The present study investigated the degree to which minocycline–an anti-inflammatory agent and purported microglial inhibitor–reduced LPS-induced neuroinflammation and sickness behavior. We show that minocycline blocked LPS-stimulated inflammatory cytokine secretion in the BV-2 microglia-derived cell line and reduced LPS-induced Toll-like-receptor-2 (TLR2) surface expression on brain microglia. Moreover, our data show that minocycline pretreatment attenuated LPS-induced weight loss, social withdrawal, and anhedonia in adult <span class="yellow">mice</span>. The attenuation of sickness behavior was paralleled with minocycline dependent decrease in markers of neuroinflammation (IL-1β, TLR2, and IDO) in adult and aged <span class="yellow">mice</span>. These findings support our hypothesis that the ability to mitigate cytokine expression in the brain during systemic inflammatory events may be useful in preventing cognitive and behavioral deficits.<br><br>Methods
Animals
Male BALB/c <span class="yellow">mice</span>, adults (3 month old) and juvenile (3–4 week old) were purchased from Harlan (Indianapolis, IN). For age comparisons, male BALB/c <span class="yellow">mice</span> (3–4 and 20–22 month old) were purchased from the National Institute on Aging specific pathogen free colony. Upon arrival, <span class="yellow">mice</span> were individually housed in polypropylene cages and maintained at 21°C under a 12 h light: 12 h dark cycle with ad libitum access to water and <span class="yellow">rodent</span> chow. At the end of each study, <span class="yellow">mice</span> were examined postmortem for gross signs of disease (e.g., splenomeglia or tumors). Data from <span class="yellow">mice</span> determined to be unhealthy were excluded from analysis (< 5%). All procedures were in accordance with the National Institute of Health Guidelines for the Care and Use of Laboratory Animals and were approved by The Ohio State University Institutional Laboratory Animal Care and Use Committee.<br><br>Cell culture
BV-2 microglia cell lines were cultured in growth medium (DMEM supplemented with 10% FBS, sodium bicarbonate 3.7 g/l, 200 mM glutamine, 100 U/ml penicillin G, 100 μg/ml streptomycin, 0.25 μg/ml fungizone) as previously described [12]. Cultures were maintained at 37°C with 95% humidity and 5% CO2 and growth medium was replenished every third day until confluence. Cultures were washed twice and supplemented with warm growth medium containing experimental treatments. Cell viability was measured by the MTS cell proliferation assay according to the manufacturer's instructions (Promega, Madison, WI).<br><br>CNS macrophage/microglia isolation
CNS macrophages and microglia were collected from whole brain homogenates as described previously [48], but with several modifications. <span class="yellow">Mice</span> were euthanized by CO2 asphyxiation and whole brains were collected. Brains were homogenized in Hank's balanced salt solution (HBSS) pH 7.4. Brain homogenates were passed through a 70 μm nylon cell strainer and centrifuged at 400 × g for 10 min. Supernatants were removed and cell pellets were re-suspended in 70% isotonic Percoll (GE-healthcare, Uppsala, Sweden) at room temperature. A discontinuous Percoll density gradient was set up as follows: 70%, 35%, and 0% isotonic Percoll. This suspension was centrifuged for 30 minutes at 400 × g. A mixed population of CNS macrophages and microglia was collected from the interphase between the 70% and 35% Percoll layers. Cells were washed and then re-suspended in sterile HBSS. The number of viable cells was determined using a hemacytometer and 0.2% trypan blue staining.<br><br>Flow cytometry
Flow cytometric analysis of microglial cell surface markers was performed as described previously, but with a few modifications [48]. In brief, Fc receptors on macrophages and microglia were blocked with anti-CD16/CD32 antibody (eBiosciences, CA). Next, cells were incubated with either Panel-1 (anti-CD11b APC, anti-CD45 FITC, and anti-MHC II PE from eBiosciences, CA) or Panel-2 antibodies (anti-CD11b APC, anti-CD45 FITC, and anti-TLR2 PE from eBiosciences, CA). Expression of these surface receptors was determined by flow cytometry using a Becton-Dickinson FACSCaliber four color Cytometer. Thirty thousand events were collected and microglia were differentiated from macrophages based on the levels of CD11b and CD45 surface expression. Microglia stain CD11b+/CD45low and macrophages stain CD11b+/CD45high [48,49]. Flow data were analyzed using FlowJo software (Tree Star, San Carlos, CA).<br><br>Behavior tests
Social exploratory behavior
To assess the motivation to engage in social exploratory behavior, a novel juvenile conspecific was introduced into the test subject's home cage for a 10-min period. Behavior was video taped and the cumulative amount of time the subject engaged in social investigation was determined from the video records by a trained observer who was blind to the experimental treatments. Baseline social behavior was measured at time 0 for all experimental treatments. Social behavior was determined as the amount of time that the experimental subject spent investigating (e.g., anogenital sniffing, trailing) the juvenile. Results are expressed as percent decrease in time engaged in social behavior compared to respective baseline measures.<br><br>Sucrose preference
To assess sucrose preference, <span class="yellow">mice</span> were provided two solutions, water or water supplemented with 2% sucrose, in 50 ml conical tubes with stoppers fitted with ball-type sipper tubes. Prior to testing conditions, all <span class="yellow">mice</span> were acclimated to the two bottle test choice. All <span class="yellow">mice</span> drank both the water and the 2% sucrose solution, but preferred drinking the sucrose over the water (data not shown). On the day of testing, <span class="yellow">mice</span> were fluid and food deprived for 2 h prior to testing [50]. At the start of the dark phase of the photoperiod, drinking water and the 2% sucrose solution were placed in the home cage overnight (15 h). At the end of each testing period the fluid content of the conical tubes was measured and sucrose preference was determined using the equation: Sucrose intake/Total fluid intake (water + sucrose intake) × 100 [51].<br><br>
Plasma cytokine measurement
IL-6 and IL-1β were measured in the plasma as previously described [52]. In brief, <span class="yellow">mice</span> were euthanized by CO2 asphyxiation and blood was collected by cardiac puncture into EDTA coated syringes. Samples were centrifuged (6000 × g for 15 min at 4°C) and plasma was collected and stored frozen (-80°C) until assaying. Plasma samples were assayed for IL-6 using a customized ELISA that we have described in detail [52] and for IL-1β using a commercial ELISA kit (R&D Systems, Minneapolis, MN). Assays were sensitive to 8 pg/ml of IL-6 and 1.5 pg/ml of IL-1β, and inter- and intra-assay coefficients of variation were less than 10%.<br><br>Real time PCR
Total RNA was isolated from brain using the Tri Reagent protocol (Sigma, St. Louis, MO). RNA samples were subjected to a DNase I digestion procedure and then reverse transcribed to cDNA using a RT RETROscript kit (Ambion, Austin, TX). Quantitative real time PCR was performed using the Applied Biosystems (Foster, CA) Assay-on Demand Gene Expression protocol as previously described [13]. In brief, cDNA was amplified by real time PCR where a target cDNA (IL-1β, IL-6, MHC II, TLR2, or IDO) and a reference cDNA (glyceraldehyde-3-phosphate dehydrogenase) were amplified simultaneously using an oligonucleotide probe with a 5' fluorescent reporter dye (6-FAM) and a 3' quencher dye (NFQ). Fluorescence was determined on an ABI PRISM 7300-sequence detection system (Applied Biosystems, CA). Data were analyzed using the comparative threshold cycle (Ct) method and results are expressed as fold difference.<br><br>Experimental protocols
For the cell culture studies, minocycline was prepared in dimethyl sulfoxide (DMSO) and BV-2 cells were washed and replenished with growth mediumsupplemented with 0, 25, 50, 100, 200, or 400 μg/ml minocycline. After 30 min, LPS at 10 ng/ml was added to the culture medium. Supernatants were collected 4 h later and IL-6 and IL-1β concentrations were determined by ELISA. Total proteins were determined from cell culture homogenates by the Bio-Rad Dc protein assay according to the manufacturer's instructions (Bio-Rad Lboratories, Hercules, CA). Each treatment was replicated a minimum of four times. Cell viability was confirmed by the MTS cell proliferation assay according to the manufacturer's instructions (Promega, Madison, WI).
For all <span class="blue">mouse</span> studies, minocycline (Sigma, St. Louis, MO) was dissolved in sterile water and sonicated to ensure complete solubilization. In the first <span class="blue">mouse</span> study, adult male BALB/c <span class="yellow">mice</span> received an intraperitoneal (i.p.) injection of vehicle or minocycline (50 mg/kg) for three consecutive days. On the 3rd day, <span class="yellow">mice</span> were also injected i.p. with saline or <span class="yellow">Escherichia coli</span> LPS (0.33 mg/kg; serotype 0127:B8, Sigma, St. Louis, MO) and were euthanized by CO2 asphyxiation 24 h later (n = 6). The LPS dosage was selected because it elicits a proinflammatory cytokine response in the brain resulting in mild transient sickness behavior in adult <span class="yellow">mice</span> [13,53]. Macrophage/microglial cells were isolated from whole brain homogenates and TLR2 and MHC II surface expression were determined by flow cytometry. The minocycline injection regimen and dosage was selected because a repeated pretreatment course with minocycline is necessary to attenuate neuroinflammation [41-43,45].
In the second study, adult male BALB/c <span class="yellow">mice</span> received an i.p. injection with vehicle or minocycline for three consecutive days. On the third day, motivation to engage in social behavior was determined immediately before i.p. injection of saline or LPS (0.33 mg/kg) and again 2, 4, 8, 12, and 24 h later (n = 8). Body weight and food intake were measured at each time point over the 24 h period. In a related, but separate study, adult <span class="yellow">mice</span> were treated with minocycline and LPS as described and anhedonia was assessed 24–39 h following i.p. injection of saline or LPS (0.33 mg/kg) (n = 15). Body weight, food intake, water intake, and sucrose intake were determined over the testing period.
In the third study, adult BALB/c <span class="yellow">mice</span> were treated with minocycline and then LPS as described. <span class="yellow">Mice</span> were euthanized by CO2 asphyxiation 4 later. Brains were removed and dissected to collect different brain regions. Brain regions were stored at -20°C in RNAlater (Qiagen, CA). Total RNA was isolated from brain samples and assayed using quantitative PCR (n = 8). Plasma was also collected and stored (-80°C) until assaying.
In a final study, adult (3–4 month old) or aged (20–22 month old) male BALB/c <span class="yellow">mice</span> were treated with minocycline and LPS as described and euthanized 4 h later. Brains were dissected to collect different brain regions and were stored at -20°C in RNAlater (Qiagen, CA). Total RNA was isolated from the hippocampus and assayed using quantitative PCR (n = 8).<br><br>Statistical analysis
All data were analyzed using Statistical Analysis Systems (SAS) General Linear Model procedures. Data were subjected to one, two- (Mino × LPS, Age × LPS, Mino × Age) or three-way (Mino × LPS × Time, Mino × LPS × Age) ANOVA to determine significant main effects and interactions between main factors. When appropriate, differences between treatment means were evaluated by an F-protected t-test using the Least-Significant Difference procedure of SAS. All data are expressed as treatment means ± standard error of the mean (SEM).<br><br>
Results
Minocycline attenuates LPS-induced cytokine production in BV-2 microglia
Minocycline is a tetracycline-type antibiotic that has anti-inflammatory properties in the CNS [41-43,45]. To determine the degree to which minocycline suppresses microglia activation, BV-2 microglia-derived cell lines were used. In the first experiment, BV-2 cells were treated with LPS and IL-6 production was determined 4 h later. Fig. 1A shows that LPS increased IL-6 production in a dose dependent manner F(5, 23) = 101, P < 0.001). In the second experiment, BV-2 cells were incubated with DMSO or minocycline and then stimulated with LPS. Minocycline reduced LPS-induced IL-6 secretion in a dose dependent manner (Mino × LPS interaction, F(4, 23) = 16.87, P < 0.001, Fig. 1B). Minocycline pretreatment had a similar anti-inflammatory effect on LPS-stimulated IL-1β secretion (Fig. 1C). In a third experiment, minocycline suppressed LPS-induced MHC II, TLR2, IL-1β, and IL-6 mRNA levels (P < 0.05, for each, Fig. 1D). The MTS assay verified that neither cell survival nor proliferation was affected by the experimental treatments (data not shown).<br><br>LPS-induced TLR2 surface expression on microglia is reduced by minocycline
Because minocycline attenuated LPS-induced cytokine secretion and TLR2 mRNA expression in BV-2 cells we next sought to determine if minocycline suppresses markers of microglial activation in the brain of <span class="yellow">mice</span>. <span class="yellow">Mice</span> were injected i.p. with vehicle or minocycline for 3 consecutive days then challenged with saline or LPS i.p. Markers of activation, TLR2 and MHC II, were determined on microglia collected 24 h later. The representative bivariate density plot in Fig. 2A shows that there were two populations of CD11b/CD45 positive cells and that more cells stained CD11b+/CD45low (microglia) than CD11b+/CD45high (CNS macrophages). ANOVA revealed that LPS injection increased TLR2 surface expression on microglia (F(1, 20) = 17.6, P < 0.004, Fig. 2B&D), but this induction was abrogated by minocycline pretreatment (Tendency for Mino × LPS interaction, F(1, 20) = 2.66, P = 0.10, Fig. 2C&D). It is important to note that because minocycline and saline controls did not differ in their TLR2 expression, these data were grouped together as the Control group (Fig. 2B&C). In addition, neither minocycline nor LPS treatment had a significant main effect on MHC class II surface expression on microglia (data not shown). These data indicate that minocycline attenuated LPS-induced TLR2 expression on microglia.<br><br>Minocycline facilitates the recovery from LPS-induced sickness behavior
CNS macrophages and microglia produce inflammatory cytokines and secondary messengers that modulate behavioral responses. Therefore, we next investigated if minocycline reduced the sickness response associated with peripheral LPS injection. In this experiment, adult <span class="yellow">mice</span> were treated with minocycline and LPS as described. Social exploratory behavior was measured before i.p. LPS injection and again 2, 4, 8, and 24 h later. Fig. 3A shows that LPS injection caused a reduction in social exploratory behavior (F(1,57) = 218, P < 0.001) that was time dependent (F(4,57) = 66.5, P < 0.001). Moreover, the LPS-associated reduction in social exploration was attenuated by minocycline (Mino × LPS interaction, F(1,57) = 7.5, P < 0.007). For example, at 8 h post LPS, social exploration was reduced by 35% in minocycline pretreated <span class="yellow">mice</span> given LPS compared to a 67% reduction in vehicle pretreated <span class="yellow">mice</span> given LPS (P < 0.001). While minocycline administration alone reduced food intake and body weight in control <span class="yellow">mice</span> (P < 0.05, for each), it also protected against LPS-associated anorexia (Mino × LPS interaction, F(1, 60) = 70.0, P < 0.001, Fig. 3B) and weight loss (Mino × LPS interaction, F(1, 60) = 29.7, P < 0.001, Fig. 3C).
Because sickness can also be associated with longer lasting changes in motivation [38], we next sought to determine if minocycline abrogated LPS-induced anhedonia [54,55]. In this experiment, <span class="yellow">mice</span> were subjected to the same minocycline injection regimen and LPS challenge as above and sucrose preference was assessed 24–39 h post LPS injection. By 24 h post LPS injection, food and water intake returned to baseline and LPS treated <span class="yellow">mice</span> still exhibited a marked reduction in sucrose preference from 24–39 h (F(1,59) = 14.3, P < 0.003). Moreover, this LPS-dependent reduction in sucrose preference was prevented by minocycline pretreatment (Mino × LPS interaction, F(1, 59) = 9.9, P < 0.004, Fig. 4). For example, minocycline pretreated <span class="yellow">mice</span> injected with LPS maintained the same strong preference for sucrose as saline and minocycline controls (i.e., approximately 85% preference). These data can be interpreted to indicate that minocycline blocks anhedonia associated with peripheral LPS challenge.<br><br>Minocycline reduces LPS-induced neuroinflammation
Pro-inflammatory cytokines in the CNS are partially responsible for the behavioral symptoms of sickness (e.g., anorexia, social withdrawal, and anhedonia) [1]. Therefore, we investigated the degree to which minocycline reduces neuroinflammation (IL-1β, IL-6, and IDO) after peripheral injection of LPS. In this experiment, <span class="yellow">mice</span> were subjected to the minocycline injection regimen and LPS challenge as above and cytokine mRNA levels were determined in the cortex and hippocampus 4 h after LPS injection. In <span class="yellow">mice</span> pretreated with vehicle, LPS markedly increased IL-1β mRNA levels in the hippocampus (F(1,31) = 62, P < 0.0001) and cortex (F(1,31) = 17.25, P < 0.0003). The LPS-induced IL-1β mRNA expression was reduced in both brain regions in <span class="yellow">mice</span> receiving minocycline prior to LPS injection: (hippocampus, F(1,31) = 9.63, P < 0.01) and cortex, F(1,31) = 7.23, P = 0.08, Fig. 5A). LPS caused a similar induction of IL-6 mRNA levels in the hippocampus (F(1,31) = 37.2, P < 0.001) and cortex (F(1,31) = 22.5, P < 0.001), but minocycline pretreatment only significantly attenuated LPS-induced IL-6 mRNA levels in the hippocampus (F(1,31) = 10.27, P < 0.004, Fig. 5B).
IDO mRNA levels were determined from the same RNA pool. Fig. 6D shows that LPS injection increased IDO mRNA expression in the hippocampus (F(1,31) = 11.69, P < 0.002) and cortex (F(1,31) = 5.26, P < 0.03). This LPS-induced IDO mRNA expression was attenuated by minocycline in the hippocampus (F(1,31) = 11.69, P < 0.002) and cortex (F(1,31) = 5.26, P < 0.03). It is important to note that IDO mRNA was undetected in saline treated <span class="yellow">mice</span>. Therefore, the fold IDO change was relative to the IDO mRNA levels in <span class="yellow">mice</span> receiving minocycline prior to LPS.<br><br>Minocycline reduces LPS-induced IL-6, but not IL-1β, in the plasma
Because cytokine signals can be relayed from the periphery to the brain by humoral pathways [56], plasma cytokine levels of IL-6 and IL-1β were determined 4 h post LPS injection. As expected, LPS injection caused a marked increase in plasma IL-1β (F(1,36) = 52.5, P < 0.001) and IL-6 levels (F(1,36) 34.01, P < 0.01). Minocycline pretreatment reduced LPS-induced IL-6 levels in the plasma (F(1,36) 6.68, P < 0.01) but had no significant main effect on LPS-induced IL-1β levels (Fig. 6).<br><br>Minocycline attenuates LPS-induced exaggerated neuroinflammation in aged <span class="yellow">mice</span>
Aged BALB/c <span class="yellow">mice</span> (22–24 m) have an exaggerated neuroinflammatory response to LPS injection [10,13,14]. Therefore, we next sought to determine if the heightened inflammatory response in the brain of aged <span class="yellow">mice</span> was reduced by minocycline. In this experiment, adult and aged <span class="yellow">mice</span> were subjected to the minocycline injection regimen and LPS challenge as above. As we have reported previously, MHC II mRNA expression was increased by age (P < 0.03, Fig. 7A)[13,14], but MHC II levels were unaffected by either LPS or minocycline treatment (not shown). Consistent with the data presented in Fig. 2, ANOVA revealed a significant main effect of LPS injection on TLR2 mRNA expression in the hippocampus (F(1,63) = 85.5, P < 0.001). Moreover, LPS caused a greater increase in TLR2 mRNA in the hippocampus of aged <span class="yellow">mice</span> compared to adults (LPS × Age interaction, F(1,63) = 12.70, P < 0.01). Furthermore, minocycline pretreatment attenuated LPS-induced TLR2 mRNA levels in both adult and aged <span class="yellow">mice</span> (Mino × LPS interaction, F(1,63) = 9.02, P < 0.004).
Parallel to the results for TLR2, LPS caused a greater increase in IL-1β and IDO mRNA levels in hippocampus of aged <span class="yellow">mice</span> compared to adults (Age × LPS, F(1,60) = 8.64, P < 0.01 for IL-1β and F(1,60) = 4.0, P < 0.05 for IDO). Minocycline pretreatment attenuated LPS-induced mRNA levels of IL-1β (Mino × LPS, F(1,60) = 8.76, P < 0.01, Fig. 7C) and IDO (Mino × LPS, F(1,60) = 9.7, P < 0.003, Fig. 7D). While LPS induced higher IL-6 mRNA levels in the hippocampus of both adult and aged <span class="yellow">mice</span> (F(1,59) = 44.5, P < 0.001), there was not an Age × LPS interaction. Minocycline pretreatment attenuated the LPS-induced increase in hippocampal IL-6 mRNA (Mino × LPS, F(1,59) = 5.4, P < 0.02, Fig. 7E). Taken together these data indicate that minocycline pretreatment was effective in attenuating the exaggerated neuroinflammation in aged <span class="yellow">mice</span>.<br><br>
Discussion
In the elderly, systemic infection is associated with an increased frequency of behavioral and cognitive complications [57,58]. We have reported that stimulation of the peripheral immune system in older (20–24 m) BALB/c <span class="yellow">mice</span> causes exaggerated neuroinflammation that is paralleled by prolonged sickness [13], impaired working memory [10], and depressive-like behaviors [15]. Therefore, it is important to understand the mechanisms that can modulate cytokine-mediated pathways in the brain. Here we show that minocycline treatment reduced LPS-induced TLR2 expression in BV-2 cells and on microglia isolated from adult <span class="yellow">mice</span>. Moreover, we demonstrate that minocycline was effective in facilitating the recovery from LPS-induced sickness and preventing anhedonia in adult <span class="yellow">mice</span>. Furthermore, we show that minocycline attenuated LPS-induced neuroinflammation in adults and normalized the exaggerated neuroinflammation in aged <span class="yellow">mice</span>.
Our findings, using cell culture and animal experiments, support the notion that minocycline attenuates microglial activation and limits production of inflammatory mediators. For instance, minocycline pretreatment of BV-2 cultures decreased LPS-stimulated cytokine production in a dose dependent manner (Fig. 1A). In BV-2 cells, minocycline also attenuated mRNA expression of inflammatory genes including IL-6, IL-1β, MHC II, and TLR2 (Fig. 1D). These data are consistent with other studies using minocycline and LPS in BV-2 cells [44,59]. Based on these data we next investigated if microglial activation could be attenuated in the brain. Because LPS increases brain cytokine production we expected that MHC II expression would also be increased. Contrary to our predictions, neither MHC II mRNA levels (Fig. 7) in the brain nor MHC II surface expression on microglia (CD11b+/CD45low) (data not shown) were increased by LPS injection. In an EAE model, minocycline reduced microglial expression of MHC II [45], but one key difference from our study is that the induction of EAE pathology requires functional antigen presentation on MHC II [60]. It is postulated that microglia have several activation states that depend on the specific inflammatory stimulus [61]. Thus, in situations of transient peripheral innate immune stimulation, markers in the CNS such as Toll-Like receptors [6] may be indicative of microglia activation. In support of this premise, our data show that LPS injection increases TLR2 surface expression on microglia (CD11b+/CD45low), which is inhibited by minocycline pretreatment (Fig. 2). These data are consistent with other studies showing that central or peripheral LPS challenge increases TLR2 mRNA in the brain [6,14]. Taken together our findings can be interpreted to suggest that minocycline attenuates pathways associated with microglia activation following peripheral LPS challenge.
One of the important findings of this study was that reduction of neuroinflammation by minocycline was associated with facilitated recovery from LPS-induced sickness behavior. These results are akin to our previous work with the anti-oxidant, α-tocopherol [52], and an NFKB decoy inhibitor [62]. Consistent with our previous studies [52,53,62,63], reductions in neuroinflammatory cytokines (Fig. 5) did not prevent the induction of the LPS-induced sickness response (2–4 h), but rather facilitated the recovery from sickness (8–24 h) (Fig. 3A). Recovery may be a critical issue because brain cytokines and the corresponding physiological and behavioral responses are beneficial to the host [1]. The potential risk for a maladaptive response occurs when the normally transient neuroinflammatory response is amplified or protracted [64]. Therefore pharmacological agents, similar to minocycline, that attenuate neuroinflammatory responses, but do not completely inhibit them, may be important in preventing the development of more severe and long-lasting cognitive and behavioral complications.
The results of the sucrose preference experiments support the idea that limiting exposure to neuroinflammation decreases the duration of behavioral responses. For example, while minocycline did not inhibit cytokine expression (Fig. 5) or the induction of sickness (Fig. 3A), minocycline pretreatment completely reversed the reduction in sucrose preference (i.e., anhedonia) associated with LPS injection (Fig. 4). It is also important to mention that while LPS-associated sickness and anhedonia are interrelated, these behaviors can be differentiated from one another. For instance, reduced social exploration was evident 2–24 h post injection (Fig. 3A), but only decreased sucrose preference was exhibited 24 to 39 h later (Fig. 4). This separation between behaviors is consistent with other studies investigating sickness and longer-lasting changes in motivation [15,65,66].
IDO mediated TRP metabolism represents a potential connection between activation of CNS innate immune cells and longer lasting behavioral responses. IDO mediated TRP metabolism in the brain may affect behavior by impacting both serotonin and glutamate pathways [39]. We have reported that IDO induction and activity is amplified in the brain of aged <span class="yellow">mice</span> and is associated with prolonged depressive-like behavior [15]. Here we show that IDO mRNA induction is blocked by minocycline in the brain of both adult and aged <span class="yellow">mice</span> (Figs. 5&7). These data are consistent with a recent report showing a causal relationship between IDO activity and acute depressive effects in adult CD-1 <span class="yellow">mice</span>. In this study, O'Connor et al. report that both 1-methyl tryptophan (a competitive inhibitor of IDO) and minocycline blocked IDO induction and prevented depressive-like immobility in the tail suspension and forced swimming tests [66]. Thus, in the present study, the minocycline blockade of IDO induction may explain the abrogation of LPS-induced anhedonia.
Another interesting finding was that while minocycline pretreatment in adult <span class="yellow">mice</span> attenuated LPS-induced brain IL-1β at 4 h (Fig. 5), it had no effect on plasma IL-1β levels (Fig. 6). Because IL-1β signals can be relayed from the periphery to the brain by humoral pathways [5], these findings suggest that minocycline has anti-inflammatory properties within the brain. These data are consistent with related findings that minocycline readily crosses the blood brain barrier to elicit an anti-inflammatory effect [41-43]. With regard to IL-6, minocycline pretreatment attenuated both brain and plasma levels at 4 h post LPS injection. Because circulating IL-6 levels can be increased by CNS mediated pathways including activation of the hypothalamus-pituitary-adrenal (HPA) axis [67] and the sympathetic nervous system [68], the specific reduction in plasma IL-6 by minocycline may reflect the reduction in brain inflammation at 4 h (Fig. 5). In support of this notion, we and others have reported that i.c.v. injection of LPS or IL-1β increase plasma IL-6 levels, but not IL-1β levels [14,68,69].
The final critical finding of this study was that minocycline was effective in attenuating neuroinflammation independent of age. Consistent with other aging and neuroinflammation studies, our data show that LPS caused exaggerated neuroinflammation in aged <span class="yellow">mice</span> compared to adults [10,13-15]. It is important to mention that while there was an age-related difference in MHC II expression in the hippocampus of saline treated <span class="yellow">mice</span> (Fig. 7A) there was not an age-related difference in IL-1β and IL-6 mRNA levels. These data differ from a previous report using BALB/c <span class="yellow">mice</span> showing an increase in IL-6 in older <span class="yellow">mice</span> [70]. This may be because the <span class="yellow">mice</span> used in the present study were approximately 4 months younger than the <span class="yellow">mice</span> used previously. Nonetheless, microglia can be primed or reactive with increased MHC II expression, but do not necessarily produce inflammatory cytokines in this state [19]. The key results are that peripheral LPS injection causes a greater induction of TLR2, IL-1β, and IDO mRNA in the aged brain than in the adult brain and that minocycline pretreatment normalizes this age-related exaggerated neuroinflammation (Fig. 7). These findings are also important because an amplified neuroinflammatory response in the aged brain is a precursor to complications including deficits in working memory, memory consolidation, and depressive-like behavior [9,10,15]. Based on the biochemical and behavioral data obtained from this study, we predict that minocycline will abrogate the prolonged LPS-induced sickness [13] and depressive-like behavior exhibited by aged BALB/c <span class="yellow">mice</span> [15]. We acknowledge, however, that future studies are necessary to test these predictions.<br><br>Conclusion
The present study demonstrates that minocycline reduces LPS-induced microglial activation, CNS cytokine production, and behavioral symptoms of sickness (e.g., social withdrawal and anhedonia). These findings are potentially important because they indicate that minocycline can be used to mitigate cytokine expression in the brain and have a beneficial affect on behavioral responses. Taken together, these data support the idea that pharmacological strategies aimed at decreasing neuroinflammation associated with microglial activation are important for improving recovery from sickness and reducing the frequency of neurobehavioral complications.<br><br>List of abbreviations
3-Hydroxy-L-Kynuriene (3HK), Allophycocyanin (APC), Analysis of variance (ANOVA), Central Nervous System (CNS), Dulbecco's Modified Eagle's Medium (DMEM), Dimethyl Sulfoxide (DMSO), Experimental Autoimmune Encephalomyelitis (EAE), Enzyme Linked Immmunosorbent Assay (ELISA), Fluorescein Isothiocyanate (FITC), Fetal <span class="yellow">Bovine</span> Serum (FBS), Hank's Balanced Salt Solution (HBSS), Indoleamine 2, 3 dioxygenase (IDO), Intraperitoneal (i.p.), Intracerebroventricular (i.c.v.), Interleukin (IL), Kynurenine (KYN), Lipopolysaccharide (LPS), Major Histocompatibility Complex class II (MHC II), Mitogen Activated Protein Kinase (MAP-kinase), Nuclear factor kappa B (NFκB), N-methyl-D-aspartate (NMDA), R-Phycoerthrin (PE), Quinolinic acid (QUIN), Statistical Analysis Systems (SAS), Standard Error of the Mean (SEM), Toll-like Receptor-2 (TLR2), and Tryptophan (TRP).<br><br>Competing interests
The authors of this manuscript declare that there are no actual or potential conflicts of interest. The authors affirm that there are no financial, personal or other relationships with other <span class="yellow">people</span> or organizations that have inappropriately influenced or biased their research.<br><br>Authors' contributions
CJH was involved in research experimentation, completion of statistical analysis, and writing of the manuscript. YH, AW, MH and JH assisted with experimentation and data analysis. MB and JFS contributed to the design of the experiments and assisted in editing the manuscript. JPG directed all aspects of this research project including the experimental design, research experimentation, completion of statistical analysis, and writing of the manuscript.<br><br>
<h3>pmcA2636784</h3>Tissue remodeling: a mating-induced differentiation program for the <span class="yellow">Drosophila</span> oviduct
Abstract
Background
In both vertebrates and invertebrates, the oviduct is an epithelial tube surrounded by visceral muscles that serves as a conduit for gamete transport between the ovary and uterus. While <span class="yellow">Drosophila</span> is a model system for tubular organ development, few studies have addressed the development of the <span class="yellow">fly</span>'s oviduct. Recent studies in <span class="yellow">Drosophila</span> have identified mating-responsive genes and proteins whose levels in the oviduct are altered by mating. Since many of these molecules (e.g. Muscle LIM protein 84B, Coracle, Neuroglian) have known roles in the differentiation of muscle and epithelia of other organs, mating may trigger similar differentiation events in the oviduct. This led us to hypothesize that mating mediates the last stages of oviduct differentiation in which organ-specific specializations arise.<br><br>Results
Using electron- and confocal-microscopy we identified tissue-wide post-mating changes in the oviduct including differentiation of cellular junctions, remodeling of extracellular matrix, increased myofibril formation, and increased innervation. Analysis of once- and twice-mated females reveals that some mating-responsive proteins respond only to the first mating, while others respond to both matings.<br><br>Conclusion
We uncovered ultrastructural changes in the mated oviduct that are consistent with the roles that mating-responsive proteins play in muscle and epithelial differentiation elsewhere. This suggests that mating triggers the late differentiation of the oviduct. Furthermore, we suggest that mating-responsive proteins that respond only to the first mating are involved in the final maturation of the oviduct while proteins that remain responsive to later matings are also involved in maintenance and ongoing function of the oviduct. Taken together, our results establish the oviduct as an attractive system to address mechanisms that regulate the late stages of differentiation and maintenance of a tubular organ.<br><br><br><br>Background
Most internal organs, including the vascular and respiratory systems and the gastro-intestinal and urinary-genital tracts are comprised of a single epithelial tube or a network of tubes. Tubular organs serve as conduits for the transport of gases, liquids, or solutes, and serve as barriers between biological compartments. To create tubes with specific flow and barrier properties, the morphology of the tube must be precisely specified during development and modulated by physiology. To accommodate specific physiological roles, tissue-specific programs for differentiation are employed at the last stages of development. While much is known about the molecular and cellular basis of tube formation [1-6], little is known about the mechanisms that regulate the late stages of differentiation in which organ-specific specializations arise.
The conservation of genes and similarity in tubular organ design across taxa make <span class="yellow">Drosophila</span> an excellent model for understanding organogenesis in higher animals. In <span class="yellow">Drosophila</span>, the best understood tubular organs from a developmental point of view are the trachea and salivary gland. Studies of these organs reveal a general program for tubular organ development, in which combinatorial expression of global patterning genes specifies positions within the embryo for the subsequent activation of tissue-specific early genes and transcription factors. This program results in the activation of downstream genes involved in terminal differentiation of organ-specific specializations such as the cuticle that lines the tracheal lumen [1,2,4,7-9].
The <span class="yellow">Drosophila</span> female reproductive tract is another tubular system, consisting of the uterus and a common oviduct (the main tube) that branches into two lateral oviducts. Regional differences in function are observed along the length of the tract, with egg activation occurring largely at the proximal end, in the lateral oviducts, and fertilization occurring at the distal end, in the uterus [10,11]. Unlike other tubular organs, little is known about the development of the female reproductive tract. However, regional differences in function suggest the presence of region-specific differentiation programs within the female reproductive tract.
In <span class="yellow">Drosophila</span>, mating induces changes in female behavior and physiology via molecules transmitted in the seminal fluid. These changes are rapid and lead to a mated female state which is profoundly different from the unmated female state. While an unmated female lays few eggs and readily accepts the courtship efforts of a male, a mated female exhibits increased egg-laying and actively rejects males [12-19]. Microarray studies of whole <span class="yellow">flies</span> reveal that the changes in egg-laying rate are accompanied by a change in gene expression. Within three hours of mating there is an increase in expression of a small number of genes [20,21]. Rapid changes in gene expression, as well as protein abundance, have also been observed in the female reproductive tract [22,23].
In the upper reproductive tract (lateral and common oviducts, hereafter, oviduct), mating induces an increase in immune related transcripts and down regulates transcription factors involved in cell growth and differentiation. At the protein level mating induces increased abundance of proteins associated with muscle assembly and function and cytoskeletal proteins associated with epithelial morphogenesis [23]. Since many of these mating-responsive proteins act in late differentiation pathways of muscle and epithelia elsewhere (e.g. Bent, Muscle LIM protein 84B(Mlp84B), Neuroglian (Nrg), Coracle (Cora)), we hypothesize that mating triggers similar differentiation in the oviduct. To test our hypothesis we characterized the ultrastructure of oviduct epithelia and muscle, as well as the pattern of innervation before and after mating. We then examined the effect of different mating regimes on oviduct mating-responsive cytoskeletal proteins and on female reproductive output. Our results suggest that active tissue remodeling takes place in the oviduct epithelia and musculature in response to mating. Furthermore, we found a striking increase in innervation of the oviduct after mating.
Our results show that the reproductive tract is an attractive system to address mechanisms that regulate the late stages of tissue differentiation in a tubular organ. Unlike other tubular organs, the last differentiation stage of the oviduct is triggered by an extrinsic cue (mating). This makes it possible to experimentally control the onset of differentiation, with an opportunity to independently examine the effects of mating and age. In addition, it allows us to examine processes essential for reproduction.<br><br>Results
Mating induces changes in oviduct lumen
Our previous molecular profiling showed that mating promotes changes in actin-based cytoskeletal molecules and suggests that mating triggers molecular changes and tissue remodeling in the female reproductive tract that mediate its progression to a mature functional stage [23]. To gain insight into the mechanisms that underlie this progression, we used light and electron microscopy to determine the morphological status of the oviduct in unmated and mated 3-day old females.
In nearly all mated reproductive tracts processed for microscopy (8/9), an egg was located in one of the lateral oviducts, whereas an egg was never observed in the oviduct of unmated reproductive tracts (5/5) (Figure 1). This observation is consistent with previous studies that report increased ovulation and egg-laying at 6 h post-mating [24]. In all unmated reproductive tracts examined, the region between the lateral oviducts and the middle of the common oviduct was either tapered or constricted, whereas this region appeared relaxed in the mated reproductive tract (Figure 1A and 1D). These observations raise the possibility that the lumen is narrow in the unmated oviduct and larger in the mated oviduct. To address this possibility, we collected serial 1 μm longitudinal sections through the reproductive tracts of unmated and mated females and stained these sections with toluidine blue to survey the appearance of the lumen along the entire length of the oviduct (Figure 1B and 1E). Our examination reveals that, in unmated reproductive tracts, the lateral oviduct lumen has an irregular shape, while the common oviduct lumen appears straight. Moreover, in all the unmated reproductive tracts sectioned, we detected patches of darkly stained material in the lumen of the lower common oviduct. In the mated reproductive tract, the lumen of the upper oviduct (defined as the lateral oviduct and upper part of the common oviduct) has an irregular shape, and the lumen of the lower oviduct (defined as the lower part of common oviduct) appears straight. In addition, the lumen of the lower oviduct appears wider in mated than unmated reproductive tracts (Figure 1C and 1F). Interestingly, darkly stained material was not detected in the oviduct lumen of mated females. This observation appears to be consistent with the description made by Mahowald et al. [25], who reported that the oviduct lumen of unmated females is nearly filled with an intima-like matrix and that this matrix is reduced after mating.
Because 3-day-old mated females lay eggs, it is unclear whether the lack of lumenal material and increase in lumen size in mated females occurred before or after the passage of eggs. We suggest that mating directly or indirectly induces morphological changes in the oviduct that facilitate egg passage through the duct. Taken together, our observations lead us to propose that the oviduct lumen is closed and/or obstructed in the unmated reproductive tract, and that mating induces changes in the epithelia and/or muscle that "open" the oviduct lumen.<br><br>Initial formation of cell-cell junctions in oviduct epithelia is mating-independent
To determine whether mating induces specific morphological changes in the oviduct epithelia post-mating, we next examined the ultrastructure of the oviduct epithelia in unmated and mated reproductive tracts. Since molecular profiling demonstrates that proteins associated with cellular junctions such as α- and β-Spectrin (Spec), Cora, and Nrg [23] increase post-mating, we first determined the status of the cellular junctions in the oviduct epithelia of unmated females, and whether these junctions change post-mating. In <span class="yellow">Drosophila</span>, most ectodermally derived epithelia (such as the epidermis and trachea), with a few exceptions, are joined apically by a belt-like adherens junction called the zonal adherens junction (ZA) followed basally by a septate junction (SJ) [26]. Our analysis reveals that the oviduct is lined, along its entire length, by a monolayered epithelium comprised of squamous-type cells. Although region-specific differences in morphology were observed, all oviduct epithelia examined, in both unmated and mated females, are joined along their lateral membranes by an extensive SJ and lack an apical ZA (Figure 2). SJs and ZAs form complete belts that surround the epithelial cell, thus making these junctions easily visible in transverse sections through the epithelium. Because ZAs were not detected in our transverse sections through the oviduct, this implies that ZAs never formed, or developed earlier and were lost (Figure 2D). Interestingly, we did not detect any ultrastructural differences in the SJs at 6 h post-mating, but we did uncover differences in SJ ultrastructure in different regions of the oviduct.
Based on their ultrastructure, two types of SJs, smooth and pleated, can be distinguished in <span class="yellow">Drosophila</span> [26]. Smooth SJs are distinguished by the lack of visible septae and the appearance of electron dense material in the intercellular space, while pleated SJs are distinguished by the ladder-like appearance of septae. In the lateral oviducts and upper common oviduct, septa were not detected in the SJ, thus these SJs represent smooth SJs or an immature stage of pleated SJ (Figure 2A, 2A', 2A"). In contrast, a ladder-like arrangement of septae was often visible in the SJs of the lower common oviduct (Figure 2C'), thus these SJs can be classified as pleated. Unlike the smooth-like SJs of the upper oviduct, the pleated SJs of the lower oviduct are followed basally by spot type adherens junction (spot AJs) (Figure 2B, 2B'; additional file 1). Further analysis is necessary to determine if the SJs of the upper and lower oviduct represent different types or different developmental stages. Our findings demonstrate that the initial formation of SJs, as well as spot AJs in the lower oviduct, are mating-independent. This raises an interesting question. Why are SJ proteins such as Cora and Nrg up-regulated post-mating if SJs are formed prior to mating? It is possible that the increased expression of SJ proteins is associated with functional changes in polarized secretion post-mating. Recent studies have shown that SJs play an unexpected role in regulating the apical secretion of specialized extracellular matrix molecules in the trachea [27,28], and that these molecules are important regulators of lumen size.<br><br>Mating modulates apical secretory activity in the oviduct
Given the presence of extensive SJs in the oviduct and the role of SJs in regulating apical secretion of extracellular matrix molecules in other epithelia (e.g. trachea), we asked if mating modulates apical secretion in the oviduct epithelia. Our ultrastructural analysis reveals that different regions of the oviduct display different apical membrane morphology (i.e microvilli or pleats) (see Figures 2 and additional file 2A and 2A), but all epithelia are covered by an electron dense apical extracellular matrix (AECM) and a thin layer of cuticle. We found that mating induces ultrastructural changes in the AECM and cuticle in both the upper and lower oviduct. In the upper oviduct of the unmated female, the AECM varies in thickness along the apical surface (Figure 3A). Some areas have little AECM, while other areas are covered by a distinct layer of AECM (~1–2 μm in thickness; Figure 3B). However, the AECM of mated females is more evenly distributed along the apical surface, (~2 μm in thickness; Figure 3E). Strikingly, the AECM and cuticle of mated females have a ruffled appearance, suggesting that the AECM and cuticle have increased in surface area (Figure 3F). Electron dense granules up to ~1.5 μm in diameter were occasionally observed in both the AECM and cell cytoplasm (Figure 3F). Although further analysis is needed to determine the role of these granules in the oviduct epithelia, it is possible that these granules participate in the secretion and deposition of the AECM. Taken together, our findings suggest that polarized secretion via the AECM, while ongoing in the upper oviduct of the unmated female is enhanced and/or modulated post-mating.
Post-mating changes in AECM ultrastructure are also observed in the lower common oviduct. However, unlike the AECM of the upper oviduct, the AECM of the lower oviduct is well developed prior to mating. In the lower oviduct of the unmated female, the AECM consists of an amorphous electron dense material and is unevenly distributed, forming a thick, bulbous layer above the plasma membrane in some regions (additional file 2). Matrix-like material is also observed in the lumen, but this material is more electron dense than the AECM (additional file 2A and 2A). In the mated female, the AECM is flattened against the plasma membrane and is uniformly distributed along the apical surface (additional file 2B and 2B). Matrix-like material was not observed in the center of the lumen, but small pools of very electron dense material were detected in the spaces between the epithelial folds (additional file 2C and 2C). This may explain why lumenal matrix was not detected at the light microscopic level in the mated female oviduct (see Figure 1E and 1F). Taken together, our observations suggest that the lower common oviduct is a site of active apical secretion in both mated and unmated females, and that matrix secretion, particularly in the lumen, is reduced post-mating. These findings raise the intriguing possibility that the AECM and lumenal matrix function as a plug in the lower oviduct, and that mating induces the breakdown of this plug.<br><br>Mating induces changes in hemi-adherens junctions in upper oviduct
In addition to modulating secretion at the apical membrane, mating induces changes at the basolateral membrane. In many epithelia, one of the last steps of differentiation is the development of a layer of extracellular matrix (ECM) called the basal lamina that covers the apical and/or basal membranes and the concomitant development of hemi-adherens junctions (HAJs). HAJs connect the cell cytoskeleton with the ECM and are formed at virtually all cell surfaces that contact an ECM. HAJs can be distinguished at the ultrastructural level as a patch-like, electron dense undercoat of the plasma membrane that opposes the basal lamina ([26]; Figure 3G, 3H). In the <span class="yellow">Drosophila</span> embryo, the HAJs and basal lamina are formed at the same time [26]. Because the basal lamina is established at a time when the majority of extracellular matrix molecules are actively secreted [26,29], this suggests that the formation of HAJs is tightly coordinated with the secretion of the ECM.
One of the most striking post-mating changes observed in the oviduct epithelia was the appearance of numerous HAJs along the basolateral membrane in the upper oviduct (Figure 3). The importance of HAJs, particularly in the upper oviduct, is underscored by the extensive infolding of the basolateral membrane that is observed in both unmated and mated females (Figure 3A and 3E). The infolded membrane gives rise to a highly branched intercellular space that is filled with an ECM (Figure 3C and 3G). This ECM is contiguous with the basal lamina that surrounds the epithelia. Few HAJs were observed in the upper oviduct of the unmated female, and these were largely restricted to the basal membrane, and not observed along the basolateral infolding (Figure 3C). In contrast, numerous HAJs appear along the basolateral infolding post-mating in this region of the oviduct (Figure 3F–3H). In addition, the intercellular space appears wider post-mating (Figure 3C–3D and 3G–3H), suggesting that mating induces increased secretion and/or deposition of the ECM in this cellular compartment and brings the ECM to a threshold concentration that can support the development of HAJs. HAJs were also detected along the basal membrane, but they were not detected along the apical membrane even though this membrane was covered by an ECM. Interestingly, while the basolateral membrane forms very shallow folds in the lower oviduct (see additional file 2), HAJs were observed along this membrane in unmated reproductive tracts (data not shown), thus suggesting that the epithelia is more differentiated in this region of the oviduct, and that the differentiation of the upper and lower oviduct may be under different control.<br><br>Muscle differentiation is enhanced post-mating
While we uncovered post-mating changes in the oviduct epithelia that might facilitate its transition to a high egg-laying state, this transition may also be mediated by changes in oviduct muscle properties and/or activity. The oviduct is lined by circular muscle fibers with supercontractile characteristics [30]. Our previous studies showed that mef2 and mlp84B genes that regulate muscle differentiation, are expressed and increased post-mating in the oviduct, as well as in the sperm storage regions of the reproductive tract [22,23]. This suggests that mating induces muscle differentiation in the reproductive tract. Muscle differentiation is characterized by the assembly of myofilaments into bundles called myofibrils. As muscles differentiate, myofibrils and z-bodies appear simultaneously, and increase in number until the cytoplasm is filled with myofibrils [31]. Like epithelia, one of the last steps of muscle differentiation is the secretion of a basal lamina that surrounds the muscle fiber. To determine if mating induces structural changes in muscles (such as increased myofibrils) we examined the ultrastructure of muscle fibers in the upper and lower parts of the oviduct. Our analysis revealed that, in both unmated and mated reproductive tracts, the muscles of the lower oviduct are highly differentiated as evidenced by the high density of myofibrils, well developed and aligned z-bodies, and secretion of a thick, electron dense basal lamina (Figure 4E and 4F). In contrast, muscle fibers in the lateral oviducts and upper common oviduct appear less differentiated than muscles in the lower common oviduct, as evidenced by the lesser density of myofibrils and z-bodies, and little or no basal lamina (Figure 4A and 4B). Moreover, the muscles of the upper oviduct appear more differentiated in the mated than in unmated reproductive tracts (Figure 4A–4D). Interestingly, we observed neighboring muscle fibers in different states of differentiation in the lateral oviducts in both unmated and mated reproductive tracts, but not elsewhere in the oviduct (Figure 4B). These results suggest that mating enhances the rate of muscle differentiation in the upper oviduct, and that muscle differentiation is delayed in the upper as compared to the lower oviduct. The increased muscle differentiation in the upper oviduct is not dramatic and likely reflects the short post-mating period examined in this study. The delayed differentiation of the upper oviduct muscles resembles the delay in the onset of development between the adult thoracic muscles and abdominal muscles during metamorphosis [32]. Since the ovaries and the other parts of the reproductive tract are known to have different segmental origins [33], we hypothesize that different parts of the oviduct develop at different rates or begin development at different times.<br><br>Increased innervation in the oviduct post-mating
Nerve-muscle interactions play an important role in regulating adult muscle development and refining the final pattern of innervation [34,35]. Given that oviduct muscle differentiation is enhanced post-mating, we predicted that mating either directly or indirectly induces changes in innervation. To address this prediction, we quantified the number of nerve terminals or boutons innervating the lateral oviducts and common oviduct in unmated and mated reproductive tracts. Studies of oviduct innervation in <span class="yellow">Drosophila</span> reveal that the <span class="yellow">fly</span>'s oviduct receives aminergic, peptidergic and glutamatergic input [30,36-39]. In both larval and adult <span class="yellow">Drosophila</span>, different types of boutons are formed by neurons that express different neurotransmitters and modulators [40-42]. By similarity to the boutons described at the larval and adult neuromuscular junction, Middleton et al. [30] report that the <span class="yellow">fly</span>'s oviduct is innervated by glutamatergic type I boutons and tyraminergic/octopaminergic type II boutons. Rodgriguez-Valentin et al. [43] further report that the oviduct type II boutons co-express octopamine and glutamate. The neurons that give rise to the type I innervation have not been identified. However, it is well established that type II innervation arises from octopaminergic neurons located in the abdominal ganglion [30,43]. In addition, it has been shown that some or all of these neurons express a GAL4 insertion line for the bullwinkle (bwk) gene [43]. Bwk encodes a HMG-box containing putative transcription factor [44]. To determine if mating induces any changes in the number of type I and II boutons innervating the oviduct muscles, we used the pan-neural marker, anti-HRP to label all oviduct boutons in unmated and mated females. To distinguish between type I and II boutons we used an antibody against the Disc Large (DLG) protein [45]. Type I boutons were identified by their DLG postsynaptic staining and large size (> 8 μm in diameter) (Figure 4G), while type II boutons were distinguished by their absence of DLG staining and smaller size (< 2 μm) (Figure 4H). We find that type I and II boutons innervate the lateral oviducts and common oviduct, and that the type I innervation is restricted to a few axons that run parallel to the length of the oviduct, while the type II innervation is more widespread. We quantified the number of boutons in the lateral oviducts and common oviduct and observed a 74% increase in bouton number in the lateral oviduct and a 66% increase in the common oviduct post-mating (Figure 4I). More over, we observed no significant change in the number of type I boutons in the lateral oviduct and common oviduct. However, we detected a 1.5-fold increase in the number of type II boutons in the lateral oviduct and a 1.8-fold increase in type II innervation in the common oviduct. Dramatic increases in bouton growth are also observed during development. For example, a ten-fold increase in bouton number is observed at the neuromuscular junction during the larval period [46]. To determine if the increase in type II innervation was specific to mating or reflected normal growth in 3 day-old females, we quantified type I and II innervation in the oviducts of 5 day-old unmated females. We found no significant difference in type I and II innervation in unmated 3 day-old and 5-day-old females, indicating that mating, either directly or indirectly, induces a dramatic increase in type II innervation (Figure 4I). To determine if the post-mating increase in innervation is unique to the oviduct, we asked if mating induces a global change in innervation. We quantified bouton number in the adult ventral midline muscles of the 5th abdominal segment. These muscles are innervated by boutons that increase in number during metamorphosis [47]. No significant difference in bouton number was detected at these muscles in unmated and mated females (additional file 3). Though further analysis is needed, this suggests that the post-mating increase in innervation is oviduct-specific. Because the type II boutons are octopaminergic, the increased type II innervation may result in increased octopamine (OA) release in the oviduct. In support of this possibility, we have preliminary evidence that OA is released in the oviduct post-mating (Heifetz and Wolfner, in preparation). Studies in locust and <span class="yellow">Drosophila</span> demonstrate that OA inhibits oviduct contraction, while glutamate activates oviduct contraction [30,43,48]. In <span class="yellow">Drosophila</span>, electrical stimulation of the posterior abdominal nerve gives rise to a series of muscle contractions in the oviduct followed by a period of muscle fatigue or relaxation [43]. This pattern of muscle contraction and relaxation may facilitate the proper movement of the egg through the oviduct. In their study of bwk expressing neurons that innervate the oviduct, Rodriguez-Valentin et al. [43] show that OA and glutamate interact to produce the pattern of oviduct contraction and relaxation described above. It is therefore possible that the post-mating increase in type II innervation in the oviduct plays an important role in the increased rate of ovulation and egg-laying observed post-mating.<br><br>Female mating history affects the enrichment of cytosekeletal proteins in the oviduct
To gain insights into the role of cytoskeletal protein enrichment ([23]; additional file 4) in mediating the morphological changes detected in this study, we examined the effect of different mating regimes on cytoskeletal protein abundance. We focused on a subset of mating-responsive cytoskeletal proteins with well established roles in the differentiation of muscle and epithelia. These include: (i) Mlp84B which regulates the late differentiation pathway of muscle [49]; (ii) Cora and Nrg which are required for the formation of septate junctions in epithelia [50], and (iii) Hu-li tai shao (Hts), also known as adducin-like protein, which functions in assembly of the cytoskeletal network. Na+ pump α subunit (ATPα), another protein associated with septate junctions in epithelia, is not a mating-responsive protein and was used as a control. Using western blots, we first determined the abundance of the cytoskeletal proteins in oviducts of 3-day-old unmated and mated females at 6 hrs post-mating. We confirmed the proteomic results of Kalpenikov et al. [23] and found that mating increases the abundance of all proteins, except ATPα in mated oviducts relative to their abundance in unmated oviducts (Figure 5A).
To determine whether the increased abundance of mating-responsive proteins persists for longer times post-mating, we examined oviducts of 10-day-old females that mated once at 3 days of age, and calculated the abundance of the mating-responsive proteins relative to their level in oviducts of 3-day-old unmated females. We found no change or a slight increase in the relative abundance of all cytoskeletal proteins except Mlp84B at 7 days post-mating (Figure 5A). Strikingly, the level of Mlp84B declines by 7 days post-mating to the level observed prior to mating. Thus Mlp84B levels rise and fall after mating, while the epithelial-related proteins rapidly rise and are maintained at a high level after mating. This raises the possibility that a second mating might trigger an increase in Mlp84B protein expression as observed in 3-day-old females at 6 h post-mating. To test this possibility, females were mated twice (once at day 3, and once on day 10 of age), and their oviducts were examined at 6 hrs after the second mating. We calculated the abundance of the cytoskeletal proteins in the twice mated oviducts relative to their abundance in oviducts of 3-day-old unmated females. Our results show that a second mating has little or no effect on Mlp84B abundance. Thus, Mlp84B may represent a class of mating-responsive proteins that is only needed after the first mating. Interestingly, the effect of the second mating on the epithelial-related mating-responsive proteins appears to be different for each protein. While Cora levels drop to the level observed in 3-day-old unmated females, Nrg and Hts are maintained at a high level.
To determine if the changes in cytoskeletal protein abundance are mating-dependent we measured their abundance in the oviducts of unmated 5- and 10-day-old females. We calculated their abundance relative to their level in oviducts of unmated 3-day-old females (Figure 5B). We rationalized that if the change in cytoskeletal protein abundance is mating-dependent we will not see similar changes in unmated females. We observed a slow increase in the relative abundance of all mating-responsive proteins with time post-eclosion (Figure 5B). Because unmated females lay more eggs as they age (see additional file 5C) one possible interpretation of the increased level of cytoskeletal proteins in unmated females is that these proteins are associated with an intrinsic program for oviduct maturation and that mating accelerates this process to maximize egg-laying efficacy. Alternatively, it is possible that the slow increase in protein abundance observed in unmated females is due to the passage of eggs through the oviduct.
Taken together, our results suggest that mating is essential to fine-tune the levels of the mating-responsive proteins examined in this study. Because the changes in cytoskeletal protein abundance are different in unmated and mated females, this suggests that the post-mating changes are mating-dependent. Furthermore, we suggest that these post-mating changes are linked to changes in oviduct function.<br><br>Early or prior mating increases fecundity
In <span class="yellow">Drosophila</span>, female fecundity decreases with age [51-54]. It has been proposed that this decrease is due, in part, to the loss of germline and somatic stem cells [55]. Since the expression of the oviduct cytoskeletal proteins examined in this study change with age and mating experience, the state of the oviduct may also play a role in fecundity. To separate the effects of age and mating, we measured the fecundity of females that mated twice, first at 3 days post-eclosion and again at 10 days, and compared that to the fecundity of females that mated once at 3 days and females that mated once at 10 days. Fecundity was measured as the number of eggs laid per day per female during the first three days after mating. Once-mated 3-day-old females laid nearly twice as many eggs as once-mated 10-day-old females during the three days examined (24.5 ± 0.7 versus 13.3 ± 0.9, p < 0.0001). Twice-mated 10-day-old females also laid about 50% more eggs than once-mated females of the same age (19.3 ± 0.9 versus 13.3 ± 0.9, p < 0. 0001), but about 20% fewer than laid by once-mated 3-day-old females during the three days examined (19.3 ± 0.9 versus 24.5 ± 0.7, p < 0.0001) (Figure 6A, see also additional file 5A and 5B). Thus, the difference in fecundity between once-mated 10-day-old and once-mated 3-day-old females is not a result of age alone. Rather the main determinant of fecundity at 10 days is whether there had been a prior mating at 3 days. We also calculated the fertility (number of adults eclosed) of once- and twice-mated females. Once-mated 3-day-old females are more fertile than once-mated 10-day-old females (69.5 ± 1.6% versus 56.1 ± 3.0%, p < 0.0001) and slightly more fertile than twice-mated 10-day-old females (69.5 ± 1.6% versus 62.7 ± 2.4%, p < 0.015) (Figure 6B, see also additional file 5D). Because there is no significant difference in fertility between once-mated and twice-mated 10-day-old females, this suggests that (1) a prior mating has no significant effect on the fertility of 10-day-old mated females and (2) fertility decreases with age. One intriguing interpretation of these results is that an early mating increases fecundity which partially compensates for the age-related decrease in fertility. Thus, cytoskeletal mating-responsive protein changes may be associated with structural changes in the oviduct that result in increased fecundity. This may counteract the effects of decreased fertility on the reproductive output.<br><br>
Discussion
Despite the use of <span class="yellow">Drosophila</span> as a model system for organ-level biology and the emerging parallels between mammalian and <span class="yellow">Drosophila</span> reproductive biology [56], this is the first integrative tissue-wide study of post-mating changes in the <span class="yellow">Drosophila</span> oviduct. Our results provide several lines of evidence at the molecular, morphological and physiological levels suggesting that mating induces tissue-wide differentiation in the oviduct. Moreover, we identify ultrastructural changes in the mated oviduct that are consistent with the roles that some of the mating-responsive proteins examined in this study (e.g. Mlp84B, Cora, Nrg) are reported to play in muscle and epithelial differentiation elsewhere. For example, the increased abundance of Mlp84B, a major regulator of the late differentiation pathway of muscle [49] is consistent with the increased muscle differentiation in the upper oviduct post-mating (Figure 4A–4F). Similarly, the increased abundance of Cora and Nrg, molecules that are essential for SJ development and function [50] is consistent with the observation that SJs in the upper oviduct are immature and/or that mating induces changes in the apical extracellular matrix whose secretion may be regulated, in part, by SJs as occurs in the trachea [27,28]. Other post-mating changes that indicate that mating induces tissue-wide differentiation include increased HAJs along the basolateral membrane and increased innervation.
Analysis of protein abundance following different mating regimes (unmated, once-mated, twice-mated) gave us further insights into the possible roles that mating-responsive proteins play in the oviduct. For example, Mlp84B is only responsive to the first mating, while the epithelial proteins examined (Cora, Nrg and Hts) are responsive to the first and second mating. Furthermore, the response to the second mating is different from the response to the first mating. Taken together, these results suggest that Mlp84B is required for the final maturation of the oviduct, while the epithelial proteins examined are required for both the maturation and maintenance of the oviduct at a high functional state. Moreover, the post-mating pattern of Mlp84B supports the idea that the first mating induces the final maturation of the oviduct. The rise and fall of Mlp84B abundance after the first mating (Figure 5) parallels the expression pattern of Mlp84B during development where peaks in Mlp84B transcription occur during periods of embryogenesis and metamorphosis when muscle is differentiating [49].
Using different mating regimes we tested whether the first/early mating is essential for maintenance of high reproductive output in the second mating. Our results suggest that mating at an early age is essential to achieve maximum reproductive output (i.e. high fecundity and fertility). Since the first mating increases reproductive output (evidence from our mating regime experiments), it is likely that the ultrastructural changes detected in mated 3-day-old females lead to a highly functional oviduct.
We suggest that the final maturation of the oviduct includes a mating-dependent stage. We propose that during the first few days post-eclosion, the oviduct undergoes the first phase of differentiation, after which the oviduct is developmentally poised for a rapid response to an extrinsic cue (mating). Mating then triggers the second phase of maturation (tissue remodeling and modulation) which is essential for proper oviduct function (Figure 7). We further propose that the second phase of maturation consists of processes that are mating-independent and -dependent, and that both of these pathways are essential to produce a functional oviduct. For example, initial formation of SJs occurs prior to mating (mating-independent) while the increased apical secretion and development of HAJs are mating-dependent. The oviduct musculature is an example where both mating-independent and -dependent processes play a role. Muscles are highly differentiated in the lower oviduct prior to mating, while muscle differentiation is ongoing in the upper oviduct and increases after mating. Although the onset of muscle differentiation in both regions is mating-independent, the further differentiation of muscle in the upper oviduct is mating-dependent. Another possible interpretation of the role of mating on oviduct maturation is that mating accelerates and synchronizes processes that are essential for the functional maturation of the oviduct. It will therefore be interesting to examine the oviducts of older females to determine the status of oviduct maturation.
What is the benefit of mating-induced differentiation of the oviduct tissues? Unmated females are capable of laying eggs, albeit at a reduced rate as compared to mated females of the same age. One possible interpretation is that reproduction is energetically costly, thus delaying oviduct maturation until sperm is available is advantageous to the female. This may reflect the evolution of a mechanism to optimize reproductive capacity in early adulthood in short-lived animals.
In summary, we have identified events at the cellular, molecular and physiological levels that are part of an efficient and specific program for reproduction. <span class="yellow">Drosophila</span> affords us the opportunity to uncover the signaling pathways that coordinate these events to produce a physiologically functional organ.<br><br>Methods
Flies
Wild-type Canton-S <span class="yellow">flies</span> were used for the fecundity/fertility experiments and confocal analysis. Wild-type Canton-S5 [57] <span class="yellow">flies</span> were used for electron microscopy. All <span class="yellow">flies</span> were kept in a 12 hrs light/dark cycle at 23 ± 2°C. Upon eclosion, females and males were collected on ice and held separately until 3 (females and males) or 10 (females) days of age.<br><br>Sample preparation
For all assays (unless described differently) unmated females were placed with 3-day-old unmated males and observed until mating initiated. At the end of mating, females were aspirated into fresh vials and held for 6 hrs. At 6 hrs after the start of mating, females were placed on ice for dissection. Previous molecular studies indicate changes in protein abundance at 3 hrs post-mating. We hypothesize that these molecular changes will translate into morphological changes in the next few hours. We chose to analyze the morphology of mated females at 6 h post-mating as opposed to later times post-mating because the changes observed at later times may be due, in part, to the high rate of eggs passing through the oviduct.<br><br>Electron microscopy
Reproductive tracts were dissected in Schneider's <span class="yellow">Drosophila</span> medium (Sigma) on ice and processed for electron microscopy as described in [42]. Tracts were flat-embedded between two sheets of Aclar (Electron Microscopy Sciences), which allowed us to image the entire tract at the light microscopic level prior to sectioning. Sections were cut on a Reichart Ultracut microtome. One-μm thick sections were stained with 1% toluidine blue, and viewed with a Zeiss Axoplan microscope. Ultrathin sections (~100 nm) were mounted on formvar grids, stained with lead citrate, and viewed with a Philips/FEI Morgagni 268 TEM at 80 kV. Our analysis is based on 4 unmated samples and 3 mated samples. Two unmated samples were cut in the longitudinal plane and two additional unmated samples were cut in the transverse plane, while one mated sample was cut in the longitudinal plane and two additional mated samples were cut in the transverse plane. For longitudinal sections, the entire tract was re-embedded and cut. For samples cut in the transverse plane, the flat-embedded reproductive tract was divided into three regions: (1) lateral oviducts and upper common oviducts, (2) middle common oviduct, and (3) lower common oviduct. Each region was re-embedded and sectioned. It is beyond the scope of this paper to describe all three regions, and our analysis focuses on the uppermost and lowermost regions.<br><br>Immunocytochemistry
Reproductive tracts were dissected in Yamamoto's Ringer (10 mM MOPS; 80 mM NaCl; 10 mM KCL; 0.2 mM MgCl2; 0.1 mM CaCl2) with 5% (w/v) sucrose on ice, fixed in 4% paraphormaldehyde in PBS (phosphate-buffered saline; 0.85% NaCl, 1.4 mM KH2 PO4, 8 mM Na2 HPO4, pH 7.4) for 45 min and then washed in PBS. The reproductive tracts were then incubated in blocking solution (0.5% Triton x-100, 3% NGS, 0.1% BSA) for 2 hrs at room temperature. The following primary antibodies, reagents, and dilutions were used: Cy3-conjugated <span class="yellow">goat</span> anti-HRP, 1:200 (Jackson Immunochemicals, West Grove, PA); <span class="blue">mouse</span> anti-Disc Large (DLG), 1:1000 (Developmental Hybridoma Bank), Alexa Fluor 488-phalloidin, 1:200 (Invitrogen, Molecular Probes, Scotland). Secondary antibodies were Alexa Flour 488-conjugated <span class="yellow">Goat</span> anti-<span class="blue">mouse</span>, 1:200 and Alexa Flour 546-conjugated <span class="yellow">Goat</span> anti-<span class="yellow">rabbit</span>, 1:200 (Invitrogen, Molecular Probes, Scotland). Reproductive tracts were incubated with the different primary antibodies (diluted in PBS + 0.2% Triton x-100) for 2 hr at room temperature, washed with PBST, incubated with secondary antibodies for 2 hrs at room temperature and washed with PBS. Reproductive tracts of the different treatments were mounted with Antifade media [58] on a multi-well glass slide (Hendley-Essex, UK). For each treatment (unmated, mated) and antibody/reagent (HRP, DLG, phalloidin) a minimum of ten reproductive tracts from at least two independent biological replicates were prepared.<br><br>Confocal microscopy
Reproductive tracts were viewed with a Zeiss 510 laser scanning confocal microscope using 20× and 60× objective with additional zooming. Optical sections from different focal plans of each reproductive tract region (lateral oviducts, common oviduct, uterus) were collected and projected as a reconstructed three-dimensional image using LSM image browser (version 3,5,0,376) software. Image collections were identical for each of the different reproductive tract regions analyzed.<br><br>Quantitation of bouton number
To quantify the number of boutons in the lateral and common oviducts we used ImageJ software (1.37b, National Institutes of Health) to analyze confocal images of anti-HRP and anti-DLG labeled boutons in the oviduct. The number of boutons per unit area was quantified with the Particle Analysis Tool. Briefly, to differentiate between the boutons, the particle analysis tool requires the image to be a "binary" image (i.e., black or white), thus we first converted the images to gray scale. We then set a "threshold" range so that pixels in the image whose value lies in this range are converted to black; pixels with values outside this range are converted to white. We next defined a region of interest (ROI) within the oviduct to count particles (i.e. count boutons). This ROI was saved and served to measure the number of boutons per unit area in each treatment. For each oviduct we counted the number of anti-HRP and anti-DLG labeled boutons in two ROIs within the lateral oviducts and two ROIs in the common oviduct. One-way ANOVA (SPSS 15.0) was used to measure the difference in bouton number per unit area in different regions of the oviducts, in both unmated and mated females.<br><br>Quantitation of cytoskeleton proteins
Sample preparation
To evaluate the effect of mating on the abundance of the cytoskeleton proteins tested, females were: (i) aged for 3 days, mated with 3-day-old unmated males and their oviducts were dissected after 6 hrs post-mating (Once3); (ii) aged for 3 days, mated with 3-day-old unmated males and their oviducts were dissected after 7 days (Once3 day10); (iii) aged for 3 days, mated first with 3-day-old unmated males and held singly for 7 days. At 10 days of age, female were mated again with 3-day-old unmated males. Oviducts were dissected at 6 hrs post-second mating (Twice3&10). We also examined 5-day-old and 10-day-old unmated females (UM5, UM10 respectively).<br><br>SDS polyacrylamide gel electrophoresis (SDS-PAGE) and Western blotting
For each mating regime, sixty oviducts were pooled and 30 μl of SDS-PAGE sample buffer was added as described in [59]. Samples were boiled, and then frozen at -20°C until loading. SDS-PAGE was performed on 12% polyacrylamide gels and western blotted as in [60]. Proteins were cross-linked to the filter. The following primary antibodies and dilutions were used: <span class="blue">mouse</span> anti-Neuroglian (kindly provided by M. Hortsch) 1:250; <span class="yellow">Guinea pig</span> anti-Coracle (kindly provided by R.G. Fehon) 1:2500; <span class="yellow">rabbit</span> anti-Mlp84B (kindly provided by M. Beckerle) 1:1000; <span class="blue">mouse</span> anti-hts (1B1, Developmental Studies Hybridoma Bank, DSHB) 1:75; <span class="blue">mouse</span> anti-Na, K-ATPase (α5, DSHB) 1:100. Secondary antibodies included: anti-<span class="yellow">Guinea pig</span> IgG (peroxidase conjugated), anti-<span class="yellow">Rabbit</span> IgG and anti-<span class="yellow">Mouse</span> IgG (developed in <span class="yellow">goat</span>, Sigma, Israel) 1:10,000. Proteins were visualized using an enhanced chemiluminescence (ECL) detection system (Amersham Piscataway, NJ).<br><br>Analysis
The developed film was scanned and the signal intensity (protein abundance) of each band was determined using ImageJ software (1.37d, National Institutes of Health). We evaluated protein abundance by measuring the mean gray value of a specific band and the background. The mean gray value of the background was then subtracted from that of the measured band. Relative protein abundance in mated oviduct vs. 3-day-old unmated oviduct was then calculated. Four independent biological replicates were prepared for each mating status. The reported abundance (see Figure 5) is the relative ratio (mated/unmated or unmated/unmated) of at least three replicates that showed the same trend.<br><br>
Examination of female reproductive output
Mating regimes
To evaluate the effect of mating on reproductive output females were treated as follows: (i) aged for 3 days and mated with 3-day-old unmated males (Once3); (ii) aged for 10 days and mated with 3-day-old unmated males (Once10); (iii) aged for 3 days, mated first with 3-day-old unmated males, held for 7 days and mated again with 3-day-old unmated males (Twice3&10). In all cases male and female pairs were observed to record mating initiation and termination.<br><br>Analysis
Following mating, females were aspirated into fresh vials, held singly and allowed to lay eggs for 6 hrs, then transferred daily (each 24 hrs) to fresh vials. The number of eggs laid and the number of eclosed adults were counted from vials created at 6 hrs, 1, 2 and 3 days post-mating. To ascertain the baseline of female egg-laying, we also included in our experiment unmated females that were kept in the same conditions as mated females. The number of eggs laid by unmated females was counted from vials created at 6 hrs, 1, 2 and 3 days after placing the females in the holding vials. In addition, we also recorded the pattern of unmated female egg-laying for 10 days.
To determine the effect of different mating regimes on female reproductive output (i.e. fecundity and fertility), we used One-way ANOVA (SPSS 15.0).<br><br><br><br>Abbreviations
Nrg: Neuroglian; Cora: Coracle; Spec: α- and β-Spectrin; SJ: septate junction; SSJ: smooth septate junction; PSJ: pleated septate junction; ZA: zonal adherens junction; AJ: adherens junction; AECM: apical extracellular matrix; ECM: extracellular matrix; HAJ: hemi-adherens junction; SAJ: spot adherens junction; OA: Octopamine; Hts: Hu-li tai shao; ATP α: Na+ pump α subunit; Mlp84B: Muscle LIM protein at 84B; DLG: Disc Large; HRP: <span class="yellow">horseradish</span> peroxidas; UM: unmated; M: mated.<br><br>Authors' contributions
AK and PKR contributed equally to this manuscript. AK, PKR and YH conceived and designed the project and analyzed the data. AK performed the confocal, Western blots and fertility assays. PKR and AK performed the light microscopy. PKR conducted the electron microscopy. AK, PKR and YH wrote the manuscript. RRH contributed to design of the study and revision of the manuscript. All authors participated in the discussion and approval of the final manuscript.<br><br>Supplementary Material<br><br>
<h3>pmcA280693</h3>p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells and its expression is induced through pathways involved in growth inhibition and repressed by factors promoting cell growth
Abstract
Background
p8 is a stress-induced protein with multiple functions and biochemically related to the architectural factor HMG-I/Y. We analyzed the expression and function of p8 in pancreatic cancer-derived cells.<br><br>Methods
Expression of p8 was silenced in the <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 by infection with a retrovirus expressing p8 RNA in the antisense orientation. Cell growth was measured in control and p8-silenced cells. Influence on p8 expression of the induction of intracellular pathways promoting cellular growth or growth arrest was monitored.<br><br>Results
p8-silenced cells grew more rapidly than control cells transfected with the empty retrovirus. Activation of the Ras→Raf→MEK→ERK and JNK intracellular pathways down-regulated p8 expression. In addition, the MEK1/2 inhibitor U0126 and the JNK inhibitor SP600125 up-regulates expression of p8. Conversely, p38 or TGFβ-1 induced p8 expression whereas the specific p38 inhibitor SB203580 down-regulated p8 expression. Finally, TGFβ-1 induction was in part mediated through p38.<br><br>Conclusions
p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells. p8 expression is induced through pathways involved in growth inhibition and repressed by factors that promote cell growth. These results suggest that p8 belongs to a pathway regulating the growth of pancreatic cancer cells.<br><br><br><br>Background
While studying the molecular response of the injured pancreas, we identified a new gene, called p8, whose expression is strongly induced during the acute phase of pancreatitis [1]. Further experiments have shown that p8 mRNA is activated in almost all cells in response to several stresses [2], including minimal stresses such as after routine change of the culture medium in the absence of any added substance [3], indicating that p8 is a ubiquitous protein induced by cellular stress. The p8 gene was cloned in <span class="yellow">human</span>, <span class="yellow">rat</span>, <span class="blue">mouse</span>, and <span class="yellow">Xenopus laevis</span> [1,4-6], conceptually translated from the <span class="yellow">Drosophila melanogaster</span> genome or deduced from EST libraries (<span class="yellow">Bos taurus</span>, <span class="yellow">Xenopus tropicalis</span>, <span class="yellow">Zebrafish</span>, <span class="yellow">Orzzias latipes</span>, <span class="yellow">Bombyx mori</span> and <span class="yellow">Paralichthys olivaceous</span>). The overall degree of homology with <span class="yellow">human</span> p8 ranged from 81 to 40%. Secondary structure prediction methods indicated that within the homologous region of the eleven proteins, there is a basic Helix-Loop-Helix secondary structure motif, characteristic of some classes of transcription factors [1]. Even though a small protein such as p8 would not need a nuclear localization signal (NLS) to be transported to the nucleus, a clear NLS can be predicted for the eleven proteins comprising a bipartite domain of positively charged aminoacids. In addition, a nuclear/cytoplasmic location has been demonstrated for <span class="yellow">human</span> p8 upon overexpression of the recombinant protein and immunohistochemistry [4], and for recombinant <span class="yellow">Xenopus laevis</span> p8 fused to green fluorescent protein [6]. Homology searching in databases did not reveal significant similarity of p8 with other proteins of known function. However, biochemical properties of the mammalian p8 proteins are shared by some high mobility group proteins (HMG) [7], particularly by the HMG-I/Y family. The overall identity of <span class="yellow">human</span> p8 with <span class="yellow">human</span> HMG-I/Y is only about 35%, but the molecular mass, isoelectric point, hydrophilicity plot, the resistance to denaturation after heating at 100°C and the charge separation are very similar [8]. The p8 protein seems to bind DNA weakly, as shown by electrophoretic mobility shift assay, without preference for DNA sequences. Finally, <span class="yellow">human</span> p8 has also been shown to be a substrate for protein kinase A in vitro and phosphorylated p8 has a higher content of secondary structure and binding to DNA is highly increased [8]. An architectural role in transcription has been proposed for this protein, in analogy with the HMG-I/Y proteins, and a recent work seems to confirm this hypothesis [9].
Functions of p8 appear to be multiple and complex. For example, p8 mRNA expression was strongly induced in 3T3 cells upon TGFβ-1 treatment which in turn enhances the Smad-transactivating function responsible for TGFβ-1 activity [10]. We also found that p8 is involved in cell cycle regulation since p8-deficient embryonic fibroblasts grew more rapidly and incorporated more [3H] thymidine and BrdU than p8-expressing cells [11]. Moreover, expression of p8 in breast cancer-derived cells seems to mediate the inhibition of cell growth induced by 1,25-Dihydroxyvitamin D3 [12]. On the contrary, we also reported that p8 may promote cell growth when overexpressed in Cos-7, AR42J and HeLa cells [1,4]. In addition, p8 seems to be involved in other intracellular functions such as apoptosis since p8-expressing fibroblasts are more sensitive than p8-deficient fibroblasts to the apoptosis induced by DNA damage. Also, p8 is required for endothelin-induced mesangial cell hypertrophy in diabetic kidney, in a mechanism involving ERK, JNK and PI3 kinase [13]. p8 seems to play a functional role in the initiation of LHβ gene expression during embryonic cell differentiation [14]. Moreover, the <span class="yellow">Drosophila melanogaster</span> p8 homologue is involved in response to starvation and might be activated to stop cell growth in case of nutrient deprivation [15]. Finally, a particularly attractive role in tumour progression was recently proposed for p8 [16]. Fibroblasts obtained from p8-expressing or p8-deficient animals were transformed with a retroviral vector expressing both the rasV12 mutated protein and the E1A adenoviral oncogene. In soft-agar assays, transformed p8-expressing cells formed colonies at high frequency, as expected, but p8-deficient transformed fibroblasts were unable to form colonies. Similarly, transformed p8-expressing cells produced tumours in all athymic <span class="yellow">nude mice</span> when injected subcutaneously or intraperitoneally, whereas transformed p8-deficient fibroblasts did not. On the other hand, studies by another laboratory revealed that expression of the Com1 protein [17], which is identical to <span class="yellow">human</span> p8, mediates the growth of tumour cells after metastatic establishment in a secondary organ, indicating that activated expression of Com1/p8 in metastatic cells is required for tumour progression. These results strongly suggest that p8 is involved in the cellular pathway(s) required for tumour progression and metastasis.
Our aim is to check the relevance of p8 to cancer progression in <span class="yellow">human</span>. As a first step, we investigated in the present study the function of p8 in two cell lines derived from <span class="yellow">human</span> pancreatic cancer. We observed that inhibition of p8 expression increased the cells growth rate. In addition, activations of the Ras→Raf→MEK→ERK and JNK intracellular pathways, which promote the growth of pancreatic cells, down-regulated p8 expression, whereas activation of p38 or TGFβ-1, which inhibit cell growth, induced its expression. It was concluded that i/ p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cell lines, ii/ p8 expression is induced through pathways involved in growth inhibition and, conversely, repressed by factors that promote cell growth.<br><br>Results
p8 is silenced in pancreatic cancer cells by infection with a retrovirus expressing p8 RNA in the antisense orientation
Panc-1 and BxPc-3 pancreatic cells were chosen for this study because, on the one hand, both cells express higher level of p8 (Figure 1) and, on the other hand, because Panc-1 is wild-type for Smad4/DPC4 and mutated for K-ras, while BxPc-3 is Smad4/DPC4 mutated and K-ras wild-type [18,19], therefore representing different mechanisms of transformation and different genetic backgrounds. K-ras and Smad4/DPC4 mutations are the major mechanisms involved in pancreatic cancer development. We inhibited p8 expression in both Panc-1 and BxPc-3 pancreatic cells by infecting cells with a retrovirus expressing the p8 asRNA (antisense RNA) and carrying the puromycin resistance. The antibiotic-selected cells were analyzed by Western blotting to evaluate the intracellular amounts of p8 protein. As shown in Figure 2, the p8 protein was clearly visible in both Panc-1 and BxPc-3 pancreatic cells infected with the empty retrovirus but almost undetectable in cells infected with the retrovirus encoding the p8 asRNA showed, indicating that our anti-sense strategy is efficient to silence p8 gene expression in pancreatic cancer cells. Preliminary studies had been conducted to select the best strategy to inhibit p8 expression. We compared the efficacy of the stable transfection of a siRNA, using a retroviral expression vector to the asRNA strategy described above. In our hands, the antisense strategy worked best, as judged from Western blot assessment of p8 protein expression (data not shown).<br><br>p8-silenced pancreatic cells grow more rapidly
We compared in the two cell lines the influence on growth parameters of blocking p8 expression with the p8 asRNA. Figure 3 shows that both Panc-1 and BxPc-3 cells in which p8 has been silenced grew more rapidly than cells infected with the empty vector suggesting that inhibition by p8 of pancreatic cancer cell growth is independent from the mechanism of transformation and genetic background.<br><br>Serum-stimulated cellular growth down-regulates p8 expression
Fetal <span class="yellow">calf</span> serum, which contains a complex mix of growth factors, can be used as inductor of cell growth. As shown in Figure 4 expression of p8 mRNA was down-regulated in both Panc-1 and BxPc-3 when the cells were shifted from culture media containing 0.1% fetal <span class="yellow">calf</span> serum to media containing 10% FCS. p8 protein showed a similar behavior. These results show that p8 expression is down-regulated in growing pancreatic cells.<br><br>The Ras→Raf→MEK→ERK pathway down-regulates p8 expression in pancreatic cancer cells
Most <span class="yellow">human</span> pancreatic cancers harbor mutations in the K-ras oncogene, which happens relatively early in pancreatic tumorigenesis [20]. The oncogenic mutation of the K-ras gene stabilizes the Ras protein in a GTP-bound form, which is constitutively active and make the cells grow more rapidly. Contrary to the activated Ras protein, p8 inhibits cell growth (Figure 3). We looked whether the Ras→Raf→MEK→ERK pathway was also involved in the regulation of p8 expression, and which step(s) were critical. Figure 5 shown that expression of a mutated form of the Ras protein (rasV12) in BxPc-3 cells, which are wild-type for ras, resulted in decreased p8 mRNA concentration and protein level suggesting that the activated ras inhibits p8 expression. Figure 6 shows that overexpression of Raf, but not of Raf301 (a negative mutant of Raf), and of ERK also inhibited the expression of the p8-CAT construct in Panc-1 as well as in BxPc-3. Finally, the MEK1/2 specific inhibitor U0126 [21] activated p8 mRNA expression in pancreatic cells whether they carry mutated ras (Panc-1) or wild-type (BxPc-3). Similar results were observed when expression of the p8 protein was monitored by Western blotting (Figure 7).<br><br>Activation of the JNK pathway down-regulates p8 expression in pancreatic cancer cells
c-Jun NH2-terminal kinase (JNK) is another major MAPK pathway which converts extracellular signals into expression of specific target genes through phosphorylation and activation of transcription factors. JNK activation has been implicated in various, often opposite cellular responses, such as cell proliferation, transformation and apoptosis. As shown in Figure 8, overexpression of JNK down-regulates the gene reporter activity of the p8-CAT construct in Panc-1 cells. Similar results were found in BxPc-3 cells. Treatment of these cells with the JNK specific inhibitor SP600125 [22] up-regulates expression of the p8 mRNA and p8 protein (Figure 9). These results show that the JNK pathway is involved in the regulation of p8 expression.<br><br>The p38 pathway up-regulates p8 expression in pancreatic cancer cells
The p38 signal transduction pathway also plays an essential role in regulating several cell functions including growth, response to inflammation, differentiation and apoptosis. In fact, in pancreatic cancer cells, p38 is a strong inhibitor of proliferation [23] contrary to the Ras→Raf→MEK→ERK and JNK pathways. We therefore analyzed the putative role of the p38 pathway in regulating p8 expression in pancreatic cancer cells. Figure 10 shows that over-expression of the plasmid encoding p38 significantly increases p8-CAT activity in Panc-1 as well as in BxPc-3 cells. Then, cells were treated with SB203580, a specific inhibitor of p38 [24], and p8 expression was measured. p8 mRNA as well as the encoded protein were down-regulated after inhibition of the p38 activity (Figure 11). These results indicate that the p38 pathway is a positive regulator of p8 expression in pancreatic cancer cells.<br><br>TGFβ-1 up-regulates p8 expression in pancreatic cancer cells
The most prominent biological activity of TGFβ-1 is its potent inhibition of cell growth in a wide variety of cell types including pancreatic cells. TGFβ-1 signals are sent through two types of transmembrane serine/threonine kinase receptors. In fact, TGFβ-1 binds and brings together the type I and type II receptors. In the resulting complex, the constitutively active TGFβ-1 type II receptor phosphorylates the type I receptor, which then plays a major role in transducing the signal to downstream components to affect gene expression through phosphorylation of SMAD proteins. Phosphorylated receptor-regulated SMADs then form heteromeric complexes with the common partner SMAD4. These heteromeric complexes then move to the nucleus, where SMAD4 will bind DNA and contribute to transcriptional activation. In general, pancreatic cancer cells present with defects in TGFβ-1 signaling and are resistant to TGFβ-1-mediated growth suppression. Since TGFβ-1 and p8 are inhibitors of pancreatic cell growth we analyzed whether p8 could mediate, at least in part, the effect of TGFβ-1. First, we found that treatment of Panc-1 cells with TGFβ-1 increased p8 mRNA levels and p8 protein as judged by Western blot (Figure 12). Then, to confirm that overexpression is regulated at the transcriptional level, we analyzed the effect of some constructs expressing constitutively activated type I TGFβ receptor, dominant negative type II TGFβ receptor, a dominant negative of Smad4 and the wild-type Smad4 on the p8-CAT activity. As expected, the constitutively activated type I TGFβ receptor but not the dominant negative type II TGFβ receptor increased CAT activity. Also, expression of the Smad4, contrary to that of the negative mutant, induced p8 transcription (Figure 13). Together, these results indicate that p8 is positively regulated by TGFβ-1.
Beside the Smad proteins, TGFβ-1 also activates the p38 MAPK pathway in pancreas-derived cells, which may play an important role in TGFβ-1 induced genes [25]. Therefore, we analyzed the p38-dependent effect of TGFβ-1 on p8 transcription. As shown in Figure 13, inhibition of p38 activity with the SB203580 specific inhibitor decreased about 40% the activity of TGFβ-1 on the p8 promoter indicating that the effect of TGFβ-1 on p8 promoter is mediated by both p38-dependent and p38-independent pathways.<br><br>
Discussion
Pancreatic adenocarcinoma is the fourth leading cause of death from malignant diseases [26]. The aggressive nature of the neoplasia, the lack of early detection, and the limited response to treatments such as chemotherapy and radiotherapy contribute to the high mortality rate of the disease. Therefore, a better understanding of the molecular mechanism leading to pancreatic cancer remains a major goal because it may help proposing strategies for earlier diagnosis and better treatment. The most commonly altered genes in pancreatic adenocarcinoma are K-ras (75 to 100%), p16INK4a (95%), p53 (50 to 75%) and DPC4 (50%) [27-31]. Whereas K-ras is a proto-oncogene all the others are tumour suppressor genes. Additional genes have been found altered at lower frequency. Panc-1 and BxPc-3 pancreatic cells were chosen for this study because they both express high levels of p8 (Figure 1) and because they present with different mechanisms of transformation and genetic backgrounds, Panc-1 being wild-type for Smad4/DPC4 but mutated for K-ras and BxPc-3 mutated for Smad4/DPC4 and wild-type for K-ras [18,19]. This work presents evidences that p8 inhibits the growth rate of pancreatic cancer-derived cells and that the intracellular pathways promoting cell growth down-regulate p8 expression whereas those promoting growth arrest up-regulate its expression. Together, these results suggest that p8 is downstream of some cell growth regulators and therefore regulation of p8 expression or its activity could be used as a target for treating pancreatic cancer.
Silencing p8 expression was able to strongly promote cell growth in both cell types, Panc-1 and BxPc-3, suggesting that p8 may act downstream of the ras- or Smad4/DPC4-dependent ways. Also, we found that stimulating cell growth by the complex combination of growth factors contained in fetal <span class="yellow">calf</span> serum down-regulated expression of p8 whereas, on the contrary, treating the cells with TGFβ-1, which promotes cell cycle arrest, stimulates p8 expression. Therefore, p8 gene expression seems to be regulated in opposite directions by mechanisms promoting cell growth or cell cycle arrest. It is interesting to note that while p8 expression is under the control of cell growth regulatory pathways such as Ras→Raf→MEK→ERK, JNK, p38 and TGFβ-1, p8 can affect cell cycle progression, suggesting that p8 is a target for factors regulating pancreatic cell growth.
A mechanism by which p8 could regulate cell cycle progression in embryonic fibroblasts was previously proposed [11]. In fact, p8 seems to take action upstream from cyclin-dependent kinases because the intracellular levels and activities of Cdk2 and Cdk4 are decreased when p8 is expressed. Concomitantly, the cyclin-dependent kinase inhibitor p27 is expressed at a low level in p8-deficient cells which may explain the increased activity of Cdk2 and Cdk4. The mechanism by which p8 regulates the intracellular level of those proteins remains to be determined. However, because p8 is a transcriptional cofactor, it is possible that regulation of expression of these molecules takes place, at least in part, at the transcription level.
Interestingly, expression of p8 mRNA seems to be regulated in a cell type- and stimulus-specific manner since, for example, p38 can induce p8 expression in response to stress in fibroblasts [3] but not in renal mesangial cells treated with endothelin [13]. In pancreatic cancer-derived cells p38 seems to play a major role since it is involved in p8 activation as judged by transient transfection assays and using a specific p38 inhibitor (Figures 10 and 11). In addition, p38 is also involved in TGFβ-1-induced p8 expression because about 40% of the TGFβ-1 effect was abolished when p38 activity was specifically blocked (Figure 13). On the other hand, ERK and JNK are inducers of p8 expression in mesangial cells treated with endothelin, but not involved in the activation of p8 in response to stress in fibroblasts [3], and even repressors in pancreatic cells (Figures 5, 6, 7, 8 and 9). Finally, PI3 kinase is an inducer of p8 expression in both endothelin-mediated p8 activation in mesangial cells [13] and pancreatic cells (data not shown).
Based on these observations, overexpression of p8 could be considered a possible goal for treating pancreatic tumours, in order to limit their growth. However, we previously reported that p8 repression would prevent rasV12/E1A transformed fibroblasts from evolving as tumours in <span class="yellow">nude mice</span> [16]. This apparent contradiction needs to be resolved before considering p8 as a target for treating cancer progression.<br><br>Conclusions
In conclusion (see Figure 14), we report in this paper that inhibition of p8 expression by an anti-sense strategy increases the growth rate of both Panc-1 and BxPc-3 pancreatic cancer-derived cells. Moreover, ERK- and JNK-mediated pathways down-regulate p8 expression, whereas p38 and TGFβ-1 pathways induce p8 expression. Also, cell growth triggered by expression of a RAS mutated protein or by 10% fetal <span class="yellow">calf</span> serum induces down-regulation of p8 expression. Together, these results indicate that p8 is an intracellular cell growth inhibitor and that it is oppositely regulated by growth-promoting or growth-inhibiting factors in pancreatic cancer-derived cells.<br><br>Material and Methods
Cell lines and cell culture conditions
The <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 were a kind gift of Dr C. Susini (INSERM U.531, Toulouse) and A. Hajri (IRCAD, Strasbourg) respectively. Panc-1 cells were grown in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal <span class="yellow">calf</span> serum, 2 mM L-glutamine, 100 IU/ml penicilin G and 100 μg/ml streptomycin. BxPc-3 were cultivated in RPMI 1640 medium in the presence of 2 mM L-glutamine, 4.5 g/L glucose, 10 mM Hepes, 1.0 mM sodium pyruvate, 10% fetal <span class="yellow">calf</span> serum and 100 IU/ml penicilin G and 100 μg/ml streptomycin. <span class="yellow">Human</span> recombinant TGFβ-1 was obtained from Sigma, and specific SB203580, U0126 and SP600125 inhibitors were from Calbiochem and utilized at 10 μM.<br><br>Expression plasmids
Expression plamids encoding p38 (pCEFL HA p38), Erk2 (pcDNAIII HA ERK2), JNK (pcDNAIIIB HA JNK), the wild-type Raf (pcDNA RAF BXB) and the Raf dominant negative (pcDNA RAF 301 K375W) were obtained from O Coso (University of Buenos Aires). Plamids encoding the constitutively activated type I TGFβ receptor (RI ACT), the dominant negative type II TGFβ receptor (RII DN) and the Smad4 dominant negative (DPC4 1–514 a.a.) were obtained from R Urrutia (Mayo Clinic, Rochester) and the wild type Smad4 was from C Heldin (Ludwig Institute, Uppsala).<br><br>Pancreatic p8-deficient cells
To silence p8 expression in pancreatic cells, we infected these cells with a retrovirus expressing <span class="yellow">human</span> p8 in the antisense orientation. The retroviral vector was constructed as follows: <span class="yellow">human</span> p8 cDNA was subcloned in HindIII and XhoI restriction sites of the pLPC plasmid (obtained from S. Lowe) in the antisense orientation. Amphotrope <span class="yellow">human</span> p8 expressing retrovirus was then generated by transient transfection using Phoenix amphotrope packaging cells. Viral supernatant was used to infect Panc-1 and BxPc-3 pancreatic cells and the population of p8-silenced cells was isolated by selection in presence of puromycin (1 μg/ml). As control, cells were infected with the pLPC empty vector.<br><br>p8 expression in arrested and growing cells
One million of Panc-1 or BxPc-3 cells were cultivated on 10-cm Petri plates in standard conditions (with 10% FCS). After 48 h, culture media were changed for fresh media with FCS restricted to 0.1%, in order to stop growth. After 24 hours of growth arrest, culture medium was replaced either by medium with 10% fetal <span class="yellow">calf</span> serum to resume cell growth or, as control, by medium with 0.1% fetal <span class="yellow">calf</span> serum. Twenty four hours later cells were recovered and RNA and protein extracted.<br><br>p8 expression in TGFβ-1-treated Panc-1 cells
One million of Panc-1 cells were cultivated in 10-cm culture dishes for 48 hours under standard conditions before TGFβ-1 treatment. <span class="yellow">Human</span> recombinant TGFβ-1 (5 ng/ml) was added to cells, without changing the culture medium, and cells were collected 12 hours later for RNA and protein preparation.<br><br>BxPc-3 rasV12-expressing cells
pLPC-rasV12 and pLPC plasmids were obtained from S. Lowe. Phoenix amphotrope packaging cells (106) were plated in a 6-well plate, incubated for 24 hours, then transfected with PEI with 5 μg of retroviral plasmid. After 48 hours, the medium containing virus was filtered (0.45 μm filter, Millipore) to obtain the viral supernatant. Target BxPc-3 were plated at 2 × 105 cells per 35-mm dish and incubated overnight. For infections, the culture medium was replaced by an appropriate mix of the viral supernatant and culture medium (V/V), supplemented with 4 μg/ml polybrene (Sigma), and cells were incubated at 37°C. BxPc-3 rasV12-expressing cells were selected with puromycin (1 μg/ml). Cells infected with the pLPC empty vector were used as control.<br><br>Western-blot analyses
One hundred μg of total protein extracted from cells was separated with standard procedures on 15.0% SDS-PAGE using the Mini Protean System (Bio-Rad) and transferred to a nitrocellulose membrane (Sigma). The intracellular level of p8 was estimated by Western blot using a polyclonal antibody (1:1000) raised against <span class="yellow">human</span> p8 [4].<br><br>Growth curves
One hundred thousand cells per well were plated in a series of 35-mm culture dishes. The cell number was estimated daily in triplicate, during 1 to 5 days, in a haemocytometer. Within experiments, each point was determined at least two times.<br><br>Cell transfection and gene reporter assays
Panc-1 and BxPc-3 (105) were cultivated in 30 mm diameter culture dishes for 24 hours then transiently transfected with 0.5 μg of p8-CAT reporter plasmid and 0.5 μg of pCMV/βgal plasmid (to control transfection efficiency) using the Fugene reagent in accordance with the manufacturer's protocol (Roche Molecular Biochemicals). The p8-CAT plasmid is the previously reported p-1471/+37p8-CAT promoter construct [5]. Reporter activities were measured as previously described [5]. Briefly, cell extracts were prepared with the reporter lysis buffer (Promega) 24 hours after transfection and CAT activity was determined by the phase extraction procedure [32] and β-galactosidase assay was performed essentially as described in Sambrook et al. [33]. CAT activity was normalized to β-galactosidase activity. Experiments were carried out in triplicate and repeated at least two times. Expression plasmids (0.5 μg) were co-transfected with p8-CAT and pCMV/βgal plasmids as indicated.<br><br>RT-PCR analysis
RNA was extracted using the Trizol (Life Technologies) procedure. Total RNA (1 μg) was analyzed by RT-PCR with the SuperScript™ One-step RT-PCR System and the Platinum Taq kit (Life Technologies). RT-PCR was performed using different numbers of cycles to verify that the conditions chosen were within the linear range. The mRNA coding for p8 was specifically amplified with sense (5' GAAGAGAGGCAGGGAAGACA 3') and antisense (5' CTGCCGTGCGTGTCTATTTA 3') primers, in positions 72 and 643 of the cDNA (accession # NM_012385), respectively. As control, the transcript coding for the ribosomal protein RL3 was specifically amplified for 22 cycles with sense (5'GAAAGAAGTCGTGGAGGCTG3') and antisense (5'ATCTCATCCTGCCCAAACAC3') primers, in positions 216 and 637 of the cDNA, respectively.<br><br>
Author's contributions
CM prepared cells and retrovirus, carried out RNA purification, RT-PCR, Western blots, and cell growth experiments, NL carried out CAT assays, SV participated in the design of the study and analysis of data, JLI participated in the analysis of data and wrote the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1538864</h3>QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences
Abstract
The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of growing evidence for their role in important biological processes and as therapeutic targets. G-quadruplex DNA has been suggested to regulate DNA replication and may control cellular proliferation. Sequences capable of forming G-quadruplexes in the RNA have been shown to play significant roles in regulation of polyadenylation and splicing events in mammalian transcripts. Whether quadruplex structure directly plays a role in regulating RNA processing requires investigation. Computational approaches to study G-quadruplexes allow detailed analysis of mammalian genomes. There are no known easily accessible user-friendly tools that can compute G-quadruplexes in the nucleotide sequences. We have developed a web-based server, QGRS Mapper, that predicts quadruplex forming G-rich sequences (QGRS) in nucleotide sequences. It is a user-friendly application that provides many options for defining and studying G-quadruplexes. It performs analysis of the user provided genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences. It is also useful for predicting G-quadruplex structures in oligonucleotides. The program provides options to search and retrieve desired gene/nucleotide sequence entries from NCBI databases for mapping G-quadruplexes in the context of RNA processing sites. This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing. In addition to providing data on composition and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper features interactive graphic representation of the data. The user can also use the graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen. QGRS Mapper can be accessed at .<br><br>INTRODUCTION
The quadruplex structures formed by guanine-rich nucleic acid sequences have received significant attention recently because of increasing evidence for their role in important biological processes and as therapeutic targets (1–5). The G-quadruplex structure, also known as a G-quartet, is formed by repeated folding of either the single polynucleotide molecule or by association of two or four molecules. The structure consists of stacked G-tetrads, which are square co-planar arrays of four guanine bases each (6). G-quadruplex is stabilized with cyclic Hoogsteen hydrogen bonding between the four guanines within each tetrad. The present work focuses only on the unimolecular quadruplexes, since it is more likely to be encountered in physiological conditions (7,8).
Guanine-rich sequences capable of forming G-quadruplexes are found in telomeres, promoter regions, transcribed and other biologically important regions of the mammalian genomes. G-quadruplex DNA has been suggested to regulate DNA replication in retinoblastoma susceptibility gene (Rb) region (9). This structure may control cellular proliferation at telomeric level and by transcriptional regulation of oncogenes like c-myc (2,10,11) and c-kit (12). Formation of G-quadruplex seems to be regulated through interactions with cellular proteins. While some proteins help stabilize the structure (13), others are known to resolve it (1,4,14,15). Proteins and chemicals that stabilize the G-quadruplex structure can inhibit telomerase action and, therefore, are being evaluated as anticancer therapeutic agents (16–20). Chemical compounds that inhibit G-quadruplex helicase activity may also be capable of regulating cellular proliferation (4). G-quadruplexes are also being eyed as potential antimicrobial agents due to their ability to transport monovalent anions (21).
G-quadruplex motifs in the RNA have been shown to play significant roles in mRNA turnover (1), FMRP binding (22), translation initiation (23) as well as repression (24). We have shown previously that a G-rich sequence (GRS) can mediate 3′ end processing of mammalian pre-mRNAs by interacting with DSEF-1/hnRNPH/H′ protein (25–27). Members of the hnRNP H protein family recognize G-rich motifs capable of forming G-quadruplexes and are known to regulate polyadenylation and splicing events in mammalian transcripts (28–30). Regulated RNA processing is an essential component of differential gene expression which is central to many important biologically processes. More than half of <span class="yellow">human</span> genes are known to have alternative polyadenylation (31). Over two-thirds of <span class="yellow">human</span> genes are thought to undergo alternative splicing (32). Sequences capable of forming G-quadruplexes found in the vicinity of polyadenylation and splice sites act as regulators by interacting with hnRNP F and H proteins (25–27,30,33). Whether quadruplex structure directly plays a role in regulating RNA processing events requires investigation.
Computational approaches to study G-quadruplexes in the mammalian genomes allow large-scale and detailed analysis of mammalian genes. Although, G-quadruplexes have been surveyed in the <span class="yellow">human</span> genome with such techniques (34,35), there are no known user-friendly computational tools easily accessible to the public. We had previously built a database of mapped G-quadruplex sequences in selected alternatively processed <span class="yellow">human</span> and <span class="blue">mouse</span> genes (36). Our preliminary analysis of the database suggests prevalence of such motifs near alternative splice and poly(A) sites. We have now developed a web-based server, QGRS Mapper, that generates detailed information on composition and distribution of putative quadruplex forming G-rich sequences (QGRS) in any NCBI nucleotide sequence identified or provided by the user. The program is also designed to handle the analysis of mammalian pre-mRNA sequences, including those that are alternatively processed (alternatively spliced or alternatively polyadenylated). Researchers interested in predicting the ability of a nucleotide sequence to form G-quadruplex structure will find QGRS Mapper to be a user-friendly application that provides many options for analysis. The user can define the minimum number of tetrads, maximum length of the G-quadruplex motif, and size as well as composition of the loops. The program can map unimolecular QGRS in the entire nucleotide sequence provided in the raw or FASTA format by the user. This method can be used for the analysis of genomic sequences, e.g. promoter and telomeric regions, as well as RNA sequences. It is also useful for predicting G-quadruplex structures in oligonucleotides. Alternatively, the program provides options to search the entire NCBI Gene Entrez, RefSeq and GenBank databases in order to retrieve the desired gene/nucleotide sequence entries for analysis of their transcribed regions. Furthermore, QGRS Mapper is a unique tool for mapping G-quadruplex forming sequences in the context of RNA processing sites. This feature is very useful for investigating the functional relevance of G-quadruplex structure, in particular its role in regulating the gene expression by alternative processing. In addition to providing data on composition and locations of QGRS relative to the processing sites in the pre-mRNA sequence, QGRS Mapper offers interactive graphic representation of the data. The user can also use a graphics module to visualize QGRS distribution patterns among all the alternative RNA products of a gene simultaneously on a single screen.<br><br>METHODS
QGRS definition
The main goal of the QGRS Mapper program is to predict the presence of QGRS in nucleotide entries. These putative G-quadruplexes are identified using the following motif.
GxNy1GxNy2GxNy3Gx
Here x = number of guanine tetrads in the G-quadruplex and y1,y2,y3 = length of gaps (i.e. the length of the loops connecting the guanine tetrads). The motif consists of four equal length sets of guanines (which we call G-groups), separated by arbitrary nucleotide sequences, with the following restrictions.
The sequence must contain at least two tetrads (i.e. x ≥ 2). Although structures with three or more G-tetrads are considered to be more stable, many nucleotide sequences are known to form quadruplexes with two G-tetrads (37,38). QGRS Mapper is meant to be a flexible and comprehensive tool for investigating G-quadruplexes; hence it considers sequences with two tetrads.By default, only QGRS of maximum length of 30 bases are considered. However, the program gives the user the option to search for sequences up to 45 bases. This restriction on the length of the sequences being considered is in agreement with recent literature (34,35). The maximum length of 30 bases restricts G-groups to a maximum size of 6.The gaps or loops between the G-groups may be arbitrary in composition or length (within the overall restrictions on the length of QGRS). The program gives the user the option to search for QGRS having loops with a specified length range (e.g. the user can search for QGRS with loops of lengths between 1 and 4). The user can also specify a string that one or more loops of each QGRS must contain. This string can be given as a regular expression. For example, entering the regular expression ‘T{3,5}’ will search for QGRS having one or more loops that contain three to five consecutive T's.Also, at most one of the gaps is allowed to be of zero length
Table 1 shows some examples of valid QGRS. The guanine groups which form the tetrads are underlined.
The first sequence has four tetrads and equal length gaps. This would seem to provide a G-quadruplex that is the most stable of the three sequences. The second sequence is notable for the significant differences in the size of its loops. The third sequence has two tetrads, even though three of the G-groups could have included another G (since all G-groups must be equal in size).<br><br>G-scores
We have devised a scoring system that evaluates a QGRS for its likelihood to form a stable G-quadruplex. Higher scoring sequences will make better candidates for G-quadruplexes. The scoring method uses the following principles which are based on previous studies (34,35,39–42).
Shorter loops are more common than longer loops.G-quadruplexes tend to have loops roughly equal in size.The greater the number of guanine tetrads, the more stable the quadruplex.
The computed G-scores are dependent on the user selected maximum QGRS length. The highest possible G-score, using the default maximum QGRS length of 30, is 105. Here is a sequence attaining that score:
GGGGGGTGGGGGGTGGGGGGTGGGGGG.<br><br>Eliminating QGRS overlaps
Two QGRS are said to overlap if their positions in the nucleotide sequence overlap. QGRS Mapper will start with a nucleotide sequence, find all QGRS occurring in the sequence and then produce a non-overlapping set of QGRS. Overlaps are eliminated by selecting the higher scoring QGRS. In the non-overlapping view, only this sequence will be displayed, although the user can request that all overlapping sequences be displayed.
Please see supplementary materials at NAR online for more details on the elimination of overlapping QGRS.<br><br>
FEATURES
Design and implementation
QGRS Mapper is a web-based program, written in PHP, with Java being used for some of its graphics. The program takes a nucleotide sequence from NCBI (or as provided by the user) and analyzes it for the presence of putative G-quadruplexes. The structure of QGRS Mapper is summarized in Table 2.<br><br>Search and analysis
QGRS Mapper allows the user to search for putative G-quadruplexes in a variety of ways. It is possible to enter a nucleotide sequence in raw or FASTA format for analysis. One can search and analyze gene sequences by Gene ID, Gene name or symbol, accession number or GI number for an NCBI nucleotide sequence entry. The user can opt to change the maximum length of QGRS that will be searched for (the default maximum length being 30) and change the minimum sized G-group (which is two by default). Also, the user can specify that the loops in the QGRS fall within a given size range and that one or more loops of the QGRS contain a given string (for which the user may enter a regular expression). The web page for QGRS analysis can be seen in Figure 1.
After entering a sequence in raw or FASTA format, QGRS Mapper will search the sequence for occurrences of QGRS. The user may enter any combination of the letters A, C, T, G, U, N.
The Gene ID field allows the user to search the NCBI Entrez Gene database. QGRS Mapper will connect to NCBI, download and parse the gene entry, and then analyze the transcribed region of its nucleotide sequence for the presence of QGRS. For example, entering the gene ID 403437 results in downloading the Brca1 gene sequence for <span class="yellow">Canis familiaris</span>. Using the default QGRS search parameters, QGRS Mapper finds 156 non-overlapping QGRS and 3394 overlapping QGRS in the transcribed region of this gene.
The Gene Name or Gene Symbol field also allows the user to search the NCBI databases for all such genes. Entering the gene name Bcl2 results in nine different hits which are displayed in Table 3.
All nine of these entries can be analyzed for the occurrence of QGRS. Clicking on the Gene ID takes the user to the respective Entrez Gene entry. Clicking on the last column initiates analysis of the selection by QGRS Mapper.
Similarly, the user can also enter an NCBI accession number to search for gene sequences. For example, searching the accession number AF312033 results in 12 hits being displayed for this GenBank nucleotide sequence entry.
The search phase of the program is followed by an analysis of the QGRS contained in the query sequence. In this phase of QGRS Mapper, the sequence data downloaded previously is analyzed to identify and map all QGRS relative to locations such as splice sites in exons/introns, and poly(A) site (if these locations are known). Furthermore the QGRS are scored by the method described above. The computed G-score is used to eliminate overlapping QGRS.
At times, QGRS Mapper must analyze a considerable amount of data. For example, the <span class="blue">mouse</span> version of the gene PTPRU, which is 69822 bases long, contains 94681 QGRS of length up to 45 bases. QGRS Mapper will find, analyze and map all of these sequences. During this analysis a message is displayed indicating the estimated time left to completion.<br><br>QGRS Mapper output
After the analysis of overlaps is completed, QGRS Mapper displays a summary of its findings, in the Gene View. This summary includes basic gene information such as the gene ID, gene symbol, gene name, a link to the NCBI entry, organism name, chromosome number and number of products and poly(A) signals. Information is also given for each product, such as the number of exons and introns, number of QGRS (non-overlapping and overlapping), number of QGRS found near RNA processing sites, and a visual map of the product.
As an example, the Gene View for the <span class="yellow">human</span> GREB1 is displayed in Figure 2, showing the table of gene information and product information for the first product (the output for all products may be seen in the supplementary material).
At this stage in the analysis the user can choose among three further displays: ‘Data View’, ‘Data View (with overlaps)’ and ‘Graphics View’. This can be done for the entire gene or for any particular product.
In the Data View, a table is displayed showing information for each of the set of non-overlapping QGRS. This table displays the position of the QGRS, which exon/intron it appears in, its distance from 3′ and 5′ splice sites, the QGRS sequence (with each G-group underlined) and the corresponding G-score. Similar display is also shown for each QGRS mapped to poly(A) region in the product. If the user requests the Data View for the entire gene, then the QGRS information is shown for each product. The ‘Data View (with overlaps)’ gives the same information but shows the locations of all QGRS. Figure 3 shows the Data View for product 1 of the GREB1 gene.
The user can also choose the Graphics View to give a visual display of the location of QGRS. This allows the user to see the location of QGRS relative to exons and introns (if that information is available). The Graphics View has the following components.
A graphic display of the entire gene (showing the location of the exons). This display includes a sliding window that can be used to focus on any particular segment of the gene. This window may be dragged to the left or right to change position within the gene.A magnified view of the fragment of the gene within the sliding window.A graph showing the location of QGRS within the fragment, with each QGRS being displayed by a bar whose height represents its G-score.A vertical slider that allows the user to change the size of the window. This allows the user to zoom in or out on any part of the gene. The sliding window on the gene expands or contracts as one zooms in or out. It is possible to see the nucleotide sequence of the product at maximum zoom levels.
The Graphics View for the entire gene shows the G-score graph together with an exon/intron map for each product. This allows the user to visually compare the location of QGRS for each product relative to that of splice sites. The Graphics View for the first product of the GREB1 gene is represented in Figure 4. The Graphics View for the entire GREB1 gene may be seen in the Supplementary Data.<br><br>
CONCLUSIONS
QGRS Mapper is a user-friendly web-based server that provides computational tools for prediction of Quadruplex forming G-rich sequences in the nucleotide sequences identified or provided by the user. The program offers many options, including user-defined composition of the quadruplex. It can analyze DNA or RNA sequence provided by the user in the raw or FASTA format. The application also provides tools for searching and retrieving gene/nucleotide entries from a variety of NCBI databases. There are several options for data output format, including an interactive graphic module.
Researchers interested in evaluating the ability of nucleotide sequences to form unimolecular G-quadruplexes will find QGRS Mapper to be very useful. Owing to the flexible and comprehensive nature of the design, it is expected to serve a variety of scientists. The application will be especially attractive to individuals interested in exploring the role of G-quadruplexes in regulated RNA processing. We are using the server to perform a large-scale analysis of alternatively processed mammalian transcripts. We are particularly interested in studying the composition and distribution patterns of G-quadruplexes in the transcribed regions of mammalian genes.<br><br>SUPPLEMENTARY DATA
Supplementary Data are available at NAR online.<br><br>
<h3>pmcA1636465</h3>Expression of C-terminal deleted p53 isoforms in neuroblastoma
Abstract
The tumor suppressor gene, p53, is rarely mutated in neuroblastomas (NB) at the time of diagnosis, but its dysfunction could result from a nonfunctional conformation or cytoplasmic sequestration of the wild-type p53 protein. However, p53 mutation, when it occurs, is found in NB tumors with drug resistance acquired over the course of chemotherapy. As yet, no study has been devoted to the function of the specific p53 mutants identified in NB cells. This study includes characterization and functional analysis of p53 expressed in eight cell lines: three wild-type cell lines and five cell lines harboring mutations. We identified two transcription-inactive p53 variants truncated in the C-terminus, one of which corresponded to the p53β isoform recently identified in normal tissue by Bourdon et al. [J. C. Bourdon, K. Fernandes, F. Murray-Zmijewski, G. Liu, A. Diot, D. P. Xirodimas, M. K. Saville and D. P. Lane (2005) Genes Dev., 19, 2122–2137]. Our results show, for the first time, that the p53β isoform is the only p53 species to be endogenously expressed in the <span class="yellow">human</span> NB cell line SK-N-AS, suggesting that the C-terminus truncated p53 isoforms may play an important role in NB tumor development.<br><br>INTRODUCTION
The p53 tumor suppressor gene remains the most frequently altered gene in <span class="yellow">human</span> tumors. Several p53 mutation databases have been reported previously (1–3), and to date, more than 1500 different p53 mutants have been described (4). Functional inactivation of p53 is usually due to gene mutation, deletion or protein degradation. In general, the majority of p53 mutations in <span class="yellow">human</span> neoplasia are missense mutations affecting the DNA-binding domain (DBD). Unlike other <span class="yellow">human</span> cancers, p53 in neuroblastoma (NB) is rarely mutated in the primary tumor at diagnosis but high levels of wild-type p53 (wt p53) protein expression have been found in the cytoplasm of undifferentiated tumors (5,6). More recently, in normal unstressed cells, wt p53 protein was found to be retained in the cytoplasm as a latent form, in huge, p53-associated protein complexes known as ‘Parc’ (7). The steady-state concentration of p53 in normal unstressed cells is usually very low because of the short half-life of the wild-type (wt) protein. Overexpression of p53 in most of the transformed cells containing a missense mutation within the p53 gene appears to be due to the increased stability of mutated p53 (8). In unstressed NB cells, high wt p53 expression may reflect the embryonic origin of NBs, in which precursor cells fail to mature (9).
p53 mutations are unusual in <span class="yellow">human</span> NB but, when they do occur, are found in post-chemotherapy tumors. In this respect, Tweddle et al. (10) described how two NB cell lines derived from the same <span class="yellow">patient</span> can elicit a different p53 status: wt p53 for SK-N-BE(1a) established before treatment, and mutated p53 for SK-N-BE(2c) established after relapse of the <span class="yellow">patient</span> under treatment with cytotoxic agents such as cyclophosphamide, doxorubicin, vincristine and radiotherapy. In NB cell lines, p53 mutation has been found in multidrug-resistant cells (11). Various types of p53 mutation have been detected in NB cells and can lead to inactivation either by shut-down of protein expression or production of aberrant p53 products. Indeed, in LAN-1 cells, p53 nonsense mutation at cysteine 182 in exon 5 leads to the absence of protein (9), whereas in SK-N-BE(2) cells, missense mutation at codon 135 (C135F) leads to stable overexpressed protein (11). By analyzing IGR-N-91, a cell line established in our laboratory from the bone marrow of a <span class="yellow">patient</span> with metastatic NB after unsuccessful Adriamycin–vincristine chemotherapy (12), we identified another type of aberrant protein that arises from the duplication of exons 7-8-9. This duplication spans from amino acids 225 to 331, which represent part of the DBD and part of the oligomerization domain (13). However, each p53 mutant has been described in the literature as a case report, and so far, no comparative study has been undertaken to link their biochemical features with functional properties.
In the present study we report two novel p53 C-terminus mutants identified in SK-N-AS and IGR-NB8 <span class="yellow">human</span> NB cell lines. The biological properties of these two new variants were analyzed in comparison with p53 isolated from six other <span class="yellow">human</span> NB lines: three [LAN-1, SK-N-BE(2) and IGR-N-91] expressing mutant p53 and three (SH-SY5Y, LAN-5 and IMR-32) expressing the wt protein. This characterization was done by using a range of functional assays: (i) the ability of the protein to bind with p53 consensus sequence using the functional analysis of separated allele in <span class="yellow">yeast</span> (FASAY); (ii) the ability of the protein to transactivate the p53-responsive element (RE) identified either in the promoter of p21/WAF1 or in the first intron of BAX, using luciferase reporter assay; (iii) the induction of endogenous p21/WAF1 gene expression under stress conditions.<br><br>MATERIALS AND METHODS
Neuroblastoma cell lines, culture and drug treatments
The parental <span class="yellow">human</span> NB SH-SY5Y, SK-N-AS, IMR-32 and SK-N-BE(2) cell lines were purchased from the European Collection of Cell Cultures (ECACC, Wiltshire, UK). The <span class="yellow">human</span> IGR-NB8 cells (a gift of Prof. Gilles Vassal, UPRES EA 3535, Institut Gustave Roussy, Villejuif) were derived from a previously untreated localized NB (14). The LAN-1 and LAN-5 cell lines were provided by Dr Nicole Gross (Pediatric Oncology Research, Lausanne, Switzerland). The <span class="yellow">human</span> IGR-N-91 cell line was established in our laboratory from the bone marrow of a <span class="yellow">patient</span> with metastatic NB after unsuccessful adriamycin–vincristin chemotherapy (12). LAN-1, LAN-5 and IMR-32 were grown in RPMI medium supplemented with 2 mM l-glutamine and 10% fetal <span class="yellow">calf</span> serum and gentamicine 10 μg/ml. Others cell lines were cultured in DMEM.
For activation of endogenous p53, cells were treated with cis-platinum (Sigma) (10 μg/ml) for 24 h then lysed for western blot analysis.<br><br>Western blot analysis
This procedure was carried out as described previously (13). Protein lysates (50 μg) were submitted to 10% SDS–PAGE, and then transferred onto nitrocellulose filters. After saturation, the membranes were incubated with primary antibody diluted in 0.1% phosphate-buffered saline, Tween-20 and 3% skim milk. The primary antibodies used were anti-p53 monoclonal antibody (clone DO-7, 1/1000, DAKO), anti-p21/WAF1 monoclonal antibody (Ab-1, 1/200, Oncogene Research) and anti-β-actin monoclonal antibody (1/1000; Chemicon) as internal control. Protein bands were detected by ECL system (Amersham).<br><br>PCR, plasmids cloning
Genomic DNA was extracted using lysis buffer containing 20 mM Tris–HCl, pH 7.5; 0.4 M NaCl; 0.5% SDS; 10 mM EDTA, treated with proteinase K (200 μg/μl), purified with phenol/chloroform, precipitated with ethanol and dissolved in DNase free water. Total RNA were purified using RNAble reagent (Eurobio), precipitated with isopropanol and dissolved in RNase free water. cDNA was obtained by reverse transcription of 1 μg of total RNA using Superscript II™ RNase H-Reverse transcriptase (Invitrogen) and Oligo-d(T)16 in conditions specified by the manufacturer. Amplification of full-length p53 coding region from SH-SY5Y, IGR-N-91, IGR-NB8 and SK-N-BE(2) cell lines was performed using forward primer at position 152 and reverse primer at position 1583 (F1 and R7, respectively, Table 1; GenBank accession no. K03199). p53 cDNA from SK-N-AS cells for cloning was obtained from RT–PCR using F1 and reverse primer i9+: 5′-GCAAAGTCATAGAACCATTTTCAT-3′ (nucleotide position 14989, GenBank accession no. X54156) primers which encompass from exon 1 to exon i9+ first identified by Flaman et al. (15) included. The PCR was done in the presence of pfu Hotstart DNA polymerase (Stratagene) for 30 cycles of 1 min at 90°C, 1 min at 65°C and 2 min 30 s at 72°C using PTC-100 thermocycler (MJ-Research).
The p53 cDNA from SH-SY5Y, SK-N-AS, IGR-N-91, IGR-NB8 and SK-N-AS cells were then cloned into pcDNA3.1/V5-His-Topo vector (Invitrogen) according to the manufacturer's instruction. The p53 sequence of each cell line was investigated by sequencing of plasmids after cloning. Sequencing was performed by Genome Express (Meylan, France).
The pDDm-TO harboring p53 dominant negative form (p53DD) pGL3-E1bTATA and the pE1B-hWAF1 firefly luciferase reporter containing the p53-responsive element of the p21/WAF1 promoter were described previously (16,17) pE1B-BAXi contains the p53RE identified in the intron 1 of the BAX gene [(18) and D. Munsch, personal communication]. Oligonucleotides TCGAGGGCAGGCCCGGGCTTGTCG and CTAGCGACAAGCCCGGGCCTGCCC were annealed and cloned into pGL3-E1bTATA digested with NheI and XhoI to obtain pE1B-BAXi. The pcDNA3-ΔNp73α expression plasmid was a gift of Dr Daniel Caput (SANOFI, Labèges, France).<br><br>Fluorescent in situ hybridization (FISH)
Cytogenetic preparations
Metaphase spreads from healthy <span class="yellow">human</span> male lymphocytes and tumor cell lines were prepared as described previously (19). BAC probe RP11-199F11, containing a 167 kb region spanning TP53 gene, was labeled by random priming in the presence of Alexa 594-dUTP (Molecular Probes). A commercial probe specific for chromosome 17 centromere, and labeled with spectrum green, was obtained from Vysis. After over-night cohybridization of the probes in the presence of Cot-1 DNA, the slides were washed and DNA counterstained with DAPI. The preparations were observed with an epifluorescence microscope and images captured with a Vysis imaging station. Between 3 and 14 metaphases spreads and 30–200 nuclei were examined for each cell line.<br><br>
Luciferase reporter assays
LAN-1 or SH-SY5Y cells were seeded in duplicates onto 6-well plates at a density of 2 × 104 cells per cm2 and cotransfected 24 h later with 0.5 μg (2.5 μg/ml) of pGL3 firefly luciferase reporter gene plasmid under the control of either pE1B-hWAF1 or pE1B-BAX using lipofectamine 2000 and 1 μg of either a p53 expressing plasmid or an empty vector. At 24 h after transfection, cells were lysed with 200 μl/well of passive lysis buffer provided with the ‘Luciferase assay kit’ (Promega). Luciferase activity was measured using Microlumat LB96P luminometer (EG & G Berthold Instrument).<br><br>Functional assay in <span class="yellow">yeast</span>
cDNA was obtained by RT of 1 μg of total RNA using Superscript II™ RNase H-Reverse transcriptase (Invitrogen) and random hexamers to prime the synthesis in conditions specified by the manufacturer. p53 cDNA was amplified by PCR and cotransformed into <span class="yellow">yeast</span>, IG397 Ade2 strain, together with either pRDI-22 vector for p53-standard assay or pFW35 and pFW34 plasmid for 5′ or 3′ split assay, respectively, carrying the ADE2 open reading frame under the control of a p53-responsive promoter (20). In a selective medium lacking leucine, wt-p53 activates transcription of ADE2 gene that encodes enzyme—phosphoribosylimidazole carboxylase—implicated in adenine biosynthesis. Therefore, a colony of cells that expresses ADE2 gene is white whereas the one composed of cells where ADE2 gene is not expressed owing p53 mutation is red.<br><br>
RESULTS
P53 status in SK-N-AS and IGR-NB8 cells
We first compared the migration profiles of p53 expressed in SK-N-AS and IGR-NB8 with those expressed in the other six NB cells, SH-SY5Y, LAN-5, LAN-1, IMR-32, SK-N-BE(2) and IGR-N-91. Western blots from 50 μg of total protein extracts were revealed with p53 monoclonal antibody (DO-7). A range of profiles was identified as shown in Figure 1A. As expected, p53 extracted from the three cell lines, SH-SY5Y, LAN-5 and IMR-32, expressing wt protein (13,21) migrated at the wt position. Of particular note in these three wt p53 cell lines was an additional faint band that migrated faster than the full-length protein. The LAN-1 cells were found to be p53 deficient (9). The SK-N-BE(2) cell line showed an intense band reflecting p53 stability due to a missense mutation at codon 135 (11). As expected, due to the previously identified duplication of exons 7-8-9 (13), p53 protein migration was delayed in the IGR-N-91 cells. In contrast, the p53 protein in the SK-N-AS cell line migrated noticeably faster than the wt protein, indicating that it was smaller in size. The p53 protein in the IGR-NB8 cell line was even smaller than that in SK-N-AS.
To analyze the coding region of each of the p53 variants, RT–PCR was performed using p53-specific F1-R7 primers (Table 1 and Figure 1B). The expected 1430 bp for full-length p53 was amplified from wt p53-expressing SH-SY5Y, LAN-5 and IMR-32 cell lines. As SK-N-BE(2) harbors a single point mutation at codon 135 (C135F), the amplified fragment analyzed by electrophoresis migrated as wt p53 (Figure 1B). The longer RT–PCR fragment from the mutated IGR-N-91 cell line resulted from the duplication of exons 7-8-9, as shown in our previous data (13), which corresponds to extra nucleic material of 321 nt. For the LAN-1 cells, no amplified fragment was observed in accordance with published data, which demonstrates the extremely low levels of p53 mRNA and the undetectable level of protein (9). An amplified fragment of the same length as the wt protein was observed in the IGR-NB8 cell line (Figure 1B). Indeed, complete gene sequencing revealed a point mutation E326STOP leading to a truncated protein at C-terminus. No fragment, however, was amplified from SK-N-AS with F1-R7 primers.
To further map the p53 mRNA transcribed in these cells, series of RT–PCR tests were performed using the forward primer, F2 (exon 8 position 1008th according to GenBank accession no. K03199), matched with different reverse primers, R3 (at the junction of exon 8/9, nt position 1124), R4, R5 (in exon 9 at positions 1154 and 1184, respectively), and R6 (in exon 10, at position 1230). The sequences of these primers are given in Table 1 and the results are presented in Figure 2A. SK-N-AS cDNA gave an amplified fragment of the same size as SH-SY5Y cDNA with the three primer pairs, F2/R3, F2/R4 and F3/R5. However, in contrast to SH-SY5Y, no fragment was obtained with SK-N-AS cDNA using the F2/R6 primer pair, which suggests the absence of exon 10 in SK-N-AS mRNA.<br><br>The p53 protein expressed in SK-N-AS is the p53β isoform
An alternatively spliced form of <span class="yellow">human</span> p53 mRNA with an additional 133 bp exon derived from intron 9 has been detected in normal <span class="yellow">human</span> lymphocytes (15). This spliced variant named ‘i9+’ encodes a truncated protein of 341 amino acids including 10 new amino acids derived from the novel exon, the p53β isoform according to Bourdon et al. nomenclature (22). This led us to hypothesize that the shorter protein expressed by the SK-N-AS cell line could be the p53β isoform. To test this hypothesis, RT–PCRs were performed using primer sets designed to amplify the 3′ region of p53 mRNA encoding either the specific C-terminal part of the wt protein (wt C-ter) or the specific C-terminal part of the β isoform (β C-ter). In parallel amplification with a primer pair amplifying the DBD was used as control. The sequences of these oligonucleotides are given in Table 1. The results presented in Figure 2B are consistent with the only expression of the p53β isoform in SK-N-AS as no band was observed in lane using specific C-ter primer located in exon 10. Interestingly, RT–PCR using SH-SY5Y (SH) gave an amplified fragment not only with the primer pair specific to the C-ter domain of wt p53 but also with the primer pair specific to the p53β isoform. This result, combined with the presence of an additional faint band migrating faster than wt p53 in denaturing polyacrylamide gel (Figure 1A), strongly suggests that both the p53 full-length protein and the β isoform were expressed in the SH-SY5Y cells.
The full-length SK-N-AS p53 cDNA was then amplified with the forward primer F1 and a reverse primer located within the novel exon i9+ (Table 1). This amplified fragment was cloned in pcDNA3/V5-His-Topo, as described in Materials and Methods. Its sequence analysis confirmed that the truncated p53 expressed in SK-N-AS was encoded by the i9+ splice variant described previously by Flaman et al. (15) that encodes the p53β isoform characterized by Bourdon et al. (22).
A series of genomic amplifications were performed to identify a possible deletion within the intron 9 that could account for the absence of normal size p53 in SK-N-AS cells. The primer sequences are given in Table 1. Amplifications were performed in parallel with total DNA extracted from SH-SY5Y and SK-N-AS cells. Results are presented in Figure 3. Normal size fragments that encompass the acceptor site of intron 9 were amplified with SH-SY5Y as well as with SK-N-AS DNA. On the contrary amplification fragments that encompass the intron 9 donor site were obtained only with SH-SY5Y but not with SK-N-AS DNA. These results identify a deletion of the intron 9/exon 10 junction within the SK-N-AS p53 gene.<br><br>A <span class="yellow">yeast</span> functional assay confirmed the absence of p53 full-length expression in SK-N-AS and IGR-NB8
It is possible to detect p53 mutation using a simple <span class="yellow">yeast</span> colony color assay as described by Flaman et al. (23). When the strain is transformed with a plasmid-encoding wt p53, the cells express the ADE2 gene and produce white colonies (Figure 4A, a, b2 and c1, and Figure 4B, dish a). Cells containing mutant p53 fail to express ADE2 and form small red colonies (Figure 4A, b and b1, and Figure 4B, dish c). When the p53 cDNA fragment is deleted, cells are unable to form a colony (Figure 4A, c and c2). As shown in Table 2, FASAY was performed as a p53-standard test with full-length cDNA or with the split version at the 5′ and 3′ end (15). The background of FASAY experiments is around 10%. p53 wt expressing SH-SY5Y and LAN-5, 2 wt cell lines, yielded ∼92–97% of white colonies (Table 2).
One hundred percent of the colonies carrying SK-N-BE(2) p53, which is homozygous for the C135F mutation, turned red with the standard or 5′ split assay, whereas 94% of the colonies turned white with the 3′ split assay since the missense mutation does not extend to the C′-terminus of the gene (Table 2 and Figure 4B, dish c). No colonies were observed with p53-deficient LAN-1 cells, (see also Figure 4A, c and c2). With SK-N-AS cells, the split 5′ assay gave 96% white colonies, while the p53-standard and split 3′ assay did not produce any colonies (Table 2). This means that the 5′ terminus was intact whereas the 3′ terminus had been deleted, as was confirmed by nucleotide sequence analysis. The IGR-NB8 colonies, however, were red both with the p53-standard assay and the split 3′ assay. In the IGR-N-91 cell line, where p53 harbors two contiguous sets of exons 7–9, spanning the DBD and oligomerization domain, it is interesting to note that the <span class="yellow">yeast</span> colonies were predominantly white (Figure 4B, dish b) with the split 5′ and the split 3′ assay (96 and 87% of white colonies, respectively) This suggests that the cells express a binding ability that is specific to wild-type p53 rather than mutated p53 (Table 2).<br><br>Transcription activity of SH-SY5Y, IGR-N-91, SK-N-BE(2), SK-N-AS and IGR-NB8 p53 variants in mammalian cells
To determine the transactivation ability of p53 variants in mammalian cells, we used a reporter gene strategy. The p53RE located within either the <span class="yellow">human</span> p21/WAF1 promoter or the intron 1 of the <span class="blue">mouse</span> and <span class="yellow">human</span> BAX gene [(18) and D. Munsch, personnal communication) were cloned in a luciferase reporter gene plasmid upstream of the E1B minimal promoter as described in Materials and Methods. The p53-negative LAN-1 cells were cotransfected with the p53 vectors expressing the p53 cloned from either SH-SY5Y, IGR-N-91, SK-N-BE(2), SK-N-AS or IGR-NB8 and the luciferase reporter plasmids. Both p53RE were strongly stimulated in cells cotransfected with wt p53 cloned from SH-SY5Y, as compared to cells cotransfected with an empty plasmid. In contrast, none of the p53 variants was able to transactivate the expression of luciferase driven by either p21/WAF1 or BAX p53RE (Figure 5).
To test transactivation capability at the protein level, each variant was transfected into p53-negative LAN-1 cells and the stimulation of endogenous p21/Waf1 gene expression was analyzed by western blotting. As shown in Figure 6, in contrast to wt p53, none of the p53 variants was able to induce p21 protein accumulation.
We then tested for a possible dominant negative effect of these various mutants on wt p53-dependent transcriptional activity. To this end, SH-SY5Y cells were cotransfected with constructs encoding the luciferase gene driven by either the p21/Waf1 or BAX p53RE and the constructs expressing the various p53s cloned from IGR-NB8, SK-N-BE(2), IGR-N-91, SK-N-AS and IGR-NB8 NB cells or p53DD, a dominant negative mutant of wt p53 (24). The stress induced by transfection activated the transcriptional activity of the wt p53 expressed in SH-SY5Y, leading to a p53-dependent expression of luciferase as illustrated by the fact that coexpression of p53DD led to a substantial decrease in luciferase activity when compared to the luciferase activity of cells cotransfected with an empty plasmid (Figure 5). Compared to p53DD, mutants within the DBD isolated from IGR-N-91 or SK-N-BE(2) had only a moderate dominant negative effect on endogenous wt p53 transcriptional activity. More surprisingly, the transfection of the C-terminal truncated variant IGR-NB8 enhanced both BAX and p21/WAF1 p53RE activity. The coexpression of p53β cloned from SK-N-AS also enhanced BAX p53RE activity. A similar effect has been reported already for p53β by Bourdon et al. (22).
When combined, these results show that all the p53 variants isolated from the NB cells had lost the ability to specifically transactivate the p53 target genes. Their effect on the transcriptional activity of endogenous wt p53 expressed in SH-SY5Y cells, however, largely depended on the p53 domain affected by the modification.<br><br>All the identified p53 variants inhibited the induction of endogenous p21/WAF1 gene expression under stress conditions
We further examined whether the four p53 variants, SK-N-BE(2), IGR-N-91, SK-N-AS and IGR-NB8, had loss their ability to stimulate the endogenous expression of the p21/WAF1 gene, the archetypical cell cycle inhibitor and the true target of p53. To this end, cellular response to genotoxic stress was analyzed by western blot following treatment of the various cell lines with cisplatin, one of the most potent antitumor agents used in neuroblastoma. Results are presented in Figure 7. None of the mutant cells, regardless of the type of mutation, was able to induce p21/WAF1 protein accumulation, unlike the 3 p53 wild-type cells (SH-SY5Y, IMR-32 and LAN-5).<br><br>Genomic status of TP53 region in the various cell lines
According to Knudson's ‘two hit’ model of tumor suppressor gene functional inactivation, the mutation of one allele is supposed to be associated with a deletion of the second allele. To assess this genetic mechanism, we performed FISH experiments to search for deletions of one copy of the TP53 region, especially in cell lines with a mutated TP53 gene. For this purpose, metaphase preparations of the studied cell lines were cohybridized with a p53 DNA probe labeled in red (BAC clone RP11-199F11) as described previously (25) and a chromosome 17-specific centromeric probe labeled in green. The three cell lines shown previously to express a wt p53 protein (LAN5, IMR32 and SH-SY5Y), displayed as expected two signals with each probe, confirming the presence of both TP53 alleles in these cells (Figure 8 and Table 3). Conversely, IGR-N-91 and SK-N-AS cell lines displayed only one fluorescent signal for each probe, suggesting a whole chromosome 17 lost, or at least losses of the 17p arm and the centromeric region. The SK-N-BE(2) cell line has been described as containing only one chromosome 17 and one TP53 signal (10). In our analysis, only 10% of the cells displayed this characteristic, whilst most of the cells had two copies of both (Figure 8 and Table 3). p53 sequencing, however, confirmed the previously described mutation, and the absence of a normal allele, suggesting that the cells used in our study had acquired, during culture, an uniparental disomy for the TP53-mutated chromosome 17. Finally, the other two cell lines (LAN-1 and IGR-NB8) displayed highly variable genetic heterogeneity from one cell to the next (Table 3). Surprisingly, although p53 transcripts are extremely faintly expressed in LAN-1 cell line, all cells showed several FISH signals with the 167 kb BAC probe used here. To understand this apparent contradiction, an array-CGH experiment was performed on an oligo-array Agilent, which indicated a 133 kb interstitial deletion corresponding to the p53 coding region and the 97 kb upstream region. Accordingly, the fluorescent spots observed in FISH experiments on LAN-1 cells should be related to the hybridization of the 57 kb region downstream of p53 gene present in the BAC probe. IGRN-B8 cell line displayed a number of signals of both colors ranging from 0 to 4, with 87% of cells displaying a loss for one TP53 allele. Despite this genomic variability, analysis of the p53 protein showed a single shortened form in IGR-N-B8 cell line (Figure 1). Consequently, and as suggested for SK-N-BE(2) cell line, IGR-N-B8 cell line should contain a variable number of copies of chromosome 17 with mutated p53.<br><br>
DISCUSSION
The p53 gene, the ‘genome guardian’, is mutated in over 50% of <span class="yellow">human</span> cancers, with the most common mutations being missense mutations (>2/3 of mutations) (26). In <span class="yellow">human</span> neuroblastoma tumors, p53 mutations are rarely present at the time of diagnosis (5,27); however, oncogenic p53 mutations can be found in advanced neuroblastomas that often relapse following high-dose chemotherapy (10). In contrast, in breast cancers, it has been reported that p53 mutations might improve response to high-dose chemotherapy including therapy with epirubicin and cyclophosphamide (28).
An investigation into the p53 genomic status and functions of eight <span class="yellow">human</span> NB lines revealed that all five of the mutated cell lines had distinct genetic characteristics as is schematically represented in Figure 9: SK-N-BE(2) with a single missense mutation in the p53 gene, encoding a highly stable full-length protein. SK-N-AS and IGR-NB8 proteins, although they have intact transactivation and DBDs, were truncated at the C-terminus generating 341 and 326 amino acids respectively; they therefore lack the tetramerization domain that is essential for an active conformation. Very recently, Bourdon et al. (22) reported the putative occurrence of β and γ isoforms from different tissues due to alternate splicing that indicates the similarity to those of p73 and p63 as identified previously by Daniel Caput and co-workers (29). In the p53 isoforms scheme proposed by Bourdon et al. (22), the SK-N-AS cell line that elicits p53i9 protein expression is consistent with the p53β isoform. Genomic analysis reveals that the only occurrence of the p53β isoform in SK-N-AS results from a deletion spanning the intron 9/exon 10 junction. Similar to the p53β isoform in SK-N-AS, the p53 in IGR-NB8 that lacks 67 amino acids at C-terminus was, alone, unable to induce p21/WAF1 promoter activation except with endogenous wt-p53 on SH-SY5Y cells where transfection with IGR-NB8 significantly augmented the transcriptional activation of the p21/Waf-1 promoter (Figure 5A). Studies by other authors have reported the interaction between the C-terminal domain and another region that impedes the active conformation of p53, suggesting an allosteric model for p53 activity regulation (30). Such events have been demonstrated for the 342-stop mutant, generated by mutagenesis, which can modulate transactivation, growth and apoptosis (31). Moreover, Harms and Chen (32) reported that the C-terminal basic domain inhibits induction of the proapoptotic target gene insulin-like growth factor binding protein 3, suggesting that IGR-NB8 might induce this gene. IGR-N-91 had an abnormally high molecular weight protein due to the duplication of wild-type exons 7-8-9, thus affecting the DBD and OD; and LAN-1, with a mutation at codon 182 (Cys→stop) concurred with an earlier report showing extremely low levels of mRNA and undetectable protein expression (9).
Notably, all the p53 variants, including SK-N-AS (β isoform) and IGR-NB8 (C-terminal truncated p53), elicited a total lack of p21 promoter activation. In particular, the p53β isoform was unable to induce endogenous p21 expression in SK-N-AS (Figure 6), concurring with data obtained from in vitro transfection experiments in H1299 cells by Bourdon et al. (22). For the IGR-N-91 cells, although p53 was mutated and unable to transactivate the p21/WAF1 promoter, the FASAY global test was not conclusive since ∼80% of colonies were white and nearly 20% (see also Table 2), though not enough, were red. Moreover, in this particular line, standard sequencing on cDNA using primers located within each exon as used for routine tumor analysis was unable to detect any anomalies in p53 genetic status (data not shown). These results enlighten the limit of the conventional tests to detect a transcription inactivation of p53 brought by duplication within the DBD.
Analysis of p53 genomic status was explored by FISH experiments, in search for a potential biallelic inactivation of p53, with a mutation of one allele and a deletion of the second one. This situation was indeed clearly observed in IGR-N-91 and SK-N-AS cell lines, with an unambiguous loss of one chromosome 17p arm in all cells of both. SK-N-BE(2), LAN-1 and IGR-NB-8 cell lines showed a more complex genomic situation which should be relevant of variable copy numbers of chromosome 17 bearing in most cases (LAN-1) or in all cases [SKN-BE(2), IGR-NB8] the mutated characteristic p53 allele. Our data therefore clearly demonstrate that each technique has a role and a combination of techniques is required in order to correctly define the p53 phenotype and genotype in tumor and particularly in NB cells.
Our data enlighten a high frequency of the C-terminal abnormalities (3/5 mutated) in NB cell lines. For SK-N-AS and IGR-NB8, a part of the oligomerization domain was lost and IGR-N-91 gained an extra oligomerization domain. According to FASAY assay the p53 expressed in IGR-N-91 still specifically bind DNA but not the p53 expressed in SK-N-AS and IGR-NB8 in agreement with previous published data obtained by electrophoretic mobility shift assay (33).
With regards to biological relevance, different mutants within the DBD vary in their oncogenicity. They are classified into two types depending of the location of the mutation, mutations of class I occur in the DNA contact areas, while class II mutations occur in areas important for the conformational stability of p53 protein (30). Although both class I and class II mutants have loss its ability to specifically bind DNA, class II mutations have been shown to be more oncogenic than class I. However, to our knowledge the oncogenicity of mutant affecting the C-terminal domain have not been studied. The biological role of the C-terminal mutants needs now to be thoroughly investigated in NB tumors.<br><br>
<h3>pmcA1779460</h3>MUC1 alters oncogenic events and transcription in <span class="yellow">human</span> breast cancer cells
Abstract
Introduction
MUC1 is an oncoprotein whose overexpression correlates with aggressiveness of tumors and poor survival of cancer <span class="yellow">patients</span>. Many of the oncogenic effects of MUC1 are believed to occur through interaction of its cytoplasmic tail with signaling molecules. As expected for a protein with oncogenic functions, MUC1 is linked to regulation of proliferation, apoptosis, invasion, and transcription.<br><br>Methods
To clarify the role of MUC1 in cancer, we transfected two breast cancer cell lines (MDA-MB-468 and BT-20) with small interfering (si)RNA directed against MUC1 and analyzed transcriptional responses and oncogenic events (proliferation, apoptosis and invasion).<br><br>Results
Transcription of several genes was altered after transfection of MUC1 siRNA, including decreased MAP2K1 (MEK1), JUN, PDGFA, CDC25A, VEGF and ITGAV (integrin αv), and increased TNF, RAF1, and MMP2. Additional changes were seen at the protein level, such as increased expression of c-Myc, heightened phosphorylation of AKT, and decreased activation of MEK1/2 and ERK1/2. These were correlated with cellular events, as MUC1 siRNA in the MDA-MB-468 line decreased proliferation and invasion, and increased stress-induced apoptosis. Intriguingly, BT-20 cells displayed similar levels of apoptosis regardless of siRNA, and actually increased proliferation after MUC1 siRNA.<br><br>Conclusion
These results further the growing knowledge of the role of MUC1 in transcription, and suggest that the regulation of MUC1 in breast cancer may be more complex than previously appreciated. The differences between these two cell lines emphasize the importance of understanding the context of cell-specific signaling events when analyzing the oncogenic functions of MUC1, and caution against generalizing the results of individual cell lines without adequate confirmation in intact biological systems.<br><br><br><br>Introduction
MUC1 is the founding member of the mucin family: proteins characterized by heavy O-glycosylation centering around a variable number of tandem repeats that are rich in serine and threonine residues [1,2]. MUC1 is a transmembrane heterodimer with one subunit solely extracellular (MUC1-EX), and the other subunit composed of a short extracellular stem, a single transmembrane domain, and the cytoplasmic tail (together called the MUC1-CT). MUC1 possesses both pro- and anti-adhesive capacities, as the MUC1-EX provides binding sites for a variety of adhesion proteins, while its large size and extended structure prevents cell-cell contact [3-5].
Initially described as a tumor antigen overexpressed in >90% of breast cancers, MUC1 is now known to be an oncogene with roles in both tumor formation and progression [1,6]. <span class="yellow">Mouse</span> studies have been integral to the current understanding of MUC1 in cancer. Muc1 knockout <span class="yellow">mice</span> (Muc1-/-; MUC1 is <span class="yellow">human</span>; Muc1 is <span class="blue">mouse</span>) show a reduction in tumorigenic phenotype when crossed onto <span class="yellow">mice</span> overexpressing the Wnt-1 [7] or polyomavirus middle T antigen [8] oncogenes in the mammary gland. In contrast, MUC1 overexpression in the mammary gland drives tumor formation [9], indicating that MUC1 is a true oncogene.
Many of the oncogenic effects of MUC1 stem from its cytoplasmic tail, which binds to several proteins implicated in cancer, including c-Src [10,11] and the epidermal growth factor receptor (EGFR) family [12,13]. MUC1 stimulates mitogen activated protein kinase (MAPK) signaling through the extracellular signal regulated kinases (ERK1/2) [12,14]; this can occur through MUC1 association with Grb2 and son of sevenless to activate Ras [15]. ERK1/2 signaling is commonly stimulated by the Ras-Raf-MEK (MAPK and ERK kinase) cascade downstream of mitogens such as EGFR [16], and regulates transcription via factors like the activator protein-1 complex. Loss of MUC1 can reduce EGFR expression [17], providing another means of affecting MAPK signaling. Our results describe a novel mechanism by which MUC1 regulates the ERK1/2 pathway, through modulating transcription of the genes encoding MEK1, Raf-1, and c-Jun.
MUC1 expression correlates with increased survival in response to cytotoxic or oxidative agents [18-21], and can activate the phosphoinositol-3 kinase-AKT pathway as part of an anti-apoptotic response [18]. MUC1 has also recently been linked to transcription, as the MUC1-CT localizes to the nucleus [22] and affects transcription by β-catenin [22,23], FOXO3a [21], p53 [24], and estrogen receptor α [25]. However, there are indications that the role of MUC1 in oncogenesis is regulated by cell type and signaling context. For example, MUC1 can stimulate Fas-mediated apoptosis [26], while Muc1 is specifically down-regulated in c-neu-induced mammary tumors [27]. This report emphasizes the complexity of MUC1 signaling in breast cancer by contrasting results from two established breast cancer cell lines.
To understand MUC1 function in cells with high endogenous expression, that is, cells likely to have evolved with active MUC1 signaling, we used small interfering RNA (siRNA) to knock down MUC1 in MDA-MB-468 and BT-20 cells. We then analyzed transcription of 84 genes involved in cancer, as well as the effects upon cellular events linked to oncogenesis, such as apoptosis and proliferation. Though the cell lines show some similarity in transcriptional alterations after transfection with MUC1 siRNA, their phenotypes are quite dissimilar: MDA-MB-468 increases apoptosis and reduces proliferation and invasion, while BT-20 proliferates more rapidly after loss of MUC1. This last may reflect the striking amount of active AKT in BT-20; AKT activity is increased in both cell lines after MUC1 siRNA, which agrees with a previous study of MUC1 siRNA [21], but disagrees with results from 3Y1 fibroblasts [18]. Recent studies have emphasized the complex and context-specific regulation of even such classic oncogenes as AKT [28]. The differences between the two breast cancer cell lines in this study suggest that MUC1 oncogenic functions are also subject to cell-specific regulation, and stress the need for understanding the cellular signaling context when interpreting results.<br><br>Materials and methods
Cell culture and siRNA transfection
MDA-MB-468 and BT-20 cells (American Type Culture Collection) were cultured in Dulbecco's modified Eagle's medium (Invitrogen, Carlsbad, CA, USA) plus 10% fetal <span class="yellow">calf</span> serum, 1% Glutamax (Invitrogen) and 1% penicillin/streptomycin. Stable cell lines (468.Neo and 468.MUC1Δ8) were selected with 0.5 mg/ml G418. For epidermal growth factor (EGF) stimulation, MDA-MB-468 cells were serum-starved overnight and treated for 10 minutes at 37°C with 100 ng/ml EGF. Transient siRNA transfection was performed with Lipofectamine2000 (Invitrogen) and 100 nM siRNA oligonucleotides. The commercially available siRNA constructs (all from Dharmacon, Lafayette, CO, USA) were scrambled (siCONTROL non-targeting siRNA #1), or directed against firefly luciferase (siCONTROL non-targeting siRNA #2) or MUC1 (siGENOME smartpool). The independent oligonucleotides designed in our laboratory target sequences beginning at MUC1 codons 882 and 956, and have been described previously [29]. The scrambled siRNA construct was used only in BT-20 cells as it causes a non-specific knockdown of MUC1 in the MDA-MB-468 line.<br><br>Cloning of MUC1 WT vector and stable transfection
Two silent mutations (G891A and T894C) were introduced into the MUC1 cDNA (called MUC1Δ8) to make it resistant to the 882 siRNA that targets that region of the mRNA. The mutant cDNA was cloned into the pLNCX.1 vector with neomycin resistance (gift of Joseph Loftus, Mayo Clinic, Arizona, USA). Stable transfection was performed with Lipofectamine2000; cells were selected beginning 24 hours post-transfection and maintained as a polyclonal population.<br><br>Western blots and antibodies
Cells were lysed in buffer (20 mM HEPES pH 8.0, 150 mM sodium chloride, 1% Triton X-100, 2 mM EDTA) with commercial protease (Complete inhibitor cocktail, Roche, Pleasanton, CA, USA) and phosphatase inhibitors (10 mM sodium fluoride, 2 mM sodium vanadate, 50 μM ammonium molybdate). Protein concentration was determined by BCA (Pierce, Rockford, IL, USA); 50 μg of lysate were loaded on SDS-PAGE gels for each experiment, except for the pMEK1/2 blot in MDA-MB-468, where 150 μg were used. Non-commercial antibodies used were: BC2, a <span class="blue">mouse</span> monoclonal to the MUC1-EX (gift of Dr McGuckin, Queensland University, Queensland, Australia), and CT2, an <span class="yellow">Armenian hamster</span> monoclonal to the MUC1-CT developed in our lab [12]. Antibodies to pMEK1/2, MEK1/2, ERK1/2, Myc, pAKT, AKT, β-tubulin (all Cell Signaling, Danvers, MA, USA), β-actin and dpERK1/2 (both Sigma, St. Louis, MO, USA) were used according to manufacturers' recommendations. All antibodies except β-actin (1:2,500) and dpERK1/2 (1:10,000) were used at 1:1,000 dilution for western blots. Flow cytometric analysis of MUC1 was done with HMPV-FITC, which recognizes the core peptide of the MUC1-EX tandem repeats (Pharmingen, San Diego, CA, USA). Bromodeoxyuridine (BrdU) staining was performed with a fluorescently conjugated antibody to BrdU (BrdU-PE, BD Biosciences, San Diego, CA, USA) as described below. Densitometry was performed using the public domain ImageJ program (developed at the NIH and available at [30]. Each band was measured in three places; the results were averaged and normalized to tubulin to control for loading.<br><br>Transwell invasion assays
Cells were serum-starved beginning 24 hours post-siRNA transfection. Cells were re-plated in serum-free medium 48 hours post-transfection at 50,000 cells per insert (sized for 24-well plates), with serum-containing medium in the bottom of the growth well as an attractant. Transwell inserts (BD Biosciences) pre-coated with laminin, fibronectin, collagen IV, or control (no matrix) were used, and cells were permitted to invade for 48 hours. At this point (96 hours post-transfection), visual inspection of the growth wells confirmed that negligible numbers of cells went through to the bottom of the plate. Non-invaded cells were swabbed from the tops of half of the inserts ('samples', containing only invaded cells), and retained in the others ('controls', all cells). Inserts were stained for 10 minutes with crystal violet (0.5% in 20% methanol) and washed with water. Membranes were cut out and destained for 10 minutes in 10% acetic acid in a 96-well plate; membranes were removed and absorbance was read at 570 nm. Percent invasion is defined as (absorbance of samples/absorbance of controls) × 100.<br><br>[3H]Thymidine incorporation assays
Cells were re-plated in quadruplicate 24 hours post-siRNA transfection at 15,000 cells/well (96-well plate) with [3H]thymidine (1 μCi/well), then incubated in normal conditions for 24 hours. At this time (48 hours post-siRNA transfection) excess radioactivity was washed off and the cells were harvested and read on a TopCount plate reader. Statistical analysis was performed using JMP 5.1.2 software (SAS Institute, Inc., Cary, NC, USA); the student's t test was used to determine p values and significance was confirmed with Wilcoxon rank sum and Pearson chi squared analyses.<br><br>BrdU incorporation
BrdU (50 μM) was given to cells 48 hours post-siRNA transfection and permitted to incorporate for 1.5 hours. Cells were then washed with PBS, trypsinized, and washed again. BrdU staining was performed according to an adaptation of the manufacturer's protocol: cells were re-suspended in PBS, mixed 1:1 with -20°C neat ethanol, and incubated 1 hour at -20°C to fix. Fixed cells were then washed gently and denatured in 2 M HCl for 20 minutes at room temperature. Following washing and 2 minute's incubation with 0.1 M Tris to neutralize the acid, cells were re-suspended in FACS buffer (0.5% fetal <span class="yellow">calf</span> serum in PBS) and stained with Phycoerythrin (PE)-conjugated anti-BrdU according to the manufacturer's protocol for flow cytometry analysis on a FACScan instrument.<br><br>Apoptosis and trypan blue staining
Apoptosis was measured using a kit (BD Biosciences) containing propidium iodide (PI) and FITC-conjugated annexin V. Cells were stained according to the manufacturer's protocol and the level of apoptosis determined by flow cytometry. Quadrants are: early apoptosis (annexin V+/PI-, lower right) late apoptosis (annexin V+/PI+, upper right) and non-apoptotic cell death (annexin V-/PI+, upper left). Treatments for the stress panel were: no treatment (control); DMSO as a control for celecoxib; 20 mM celecoxib, brand name Celebrex™ (dissolved in DMSO) [31]; 0.2 mM H2O2 [32]; or 1 mg/mL G418 (Pfizer, New York, NY, USA).<br><br>Real-time PCR arrays
Transcriptional analysis using Cancer PathwayFinder RT2 profiler arrays (SuperArray, Frederick, MD, USA) was performed according to the manufacturer's protocol. Briefly, total RNA was isolated using an RNeasy extraction kit (Qiagen, Valencia, CA, USA); 1 μg of RNA was reverse transcribed with the cDNA synthesis kit (SuperArray) and cDNA was subjected to real-time PCR using SYBR green to detect product. Arrays were performed independently at least twice for each cell line; all PCR products were checked on agarose gels. Values were obtained for the threshold cycle (Ct) for each gene and normalized using the average of four housekeeping genes on the same array (HPRT1, RPL13A, GAPD, ACTB). Ct values for housekeeping genes and a dilution series of ACTB were monitored for consistency between arrays. Change (ΔCt) between MUC1 siRNA and control siRNA was found by:
ΔCt = Ct(MUC1 siRNA) - Ct(control siRNA)
and fold change by:
Fold change = 2(-ΔCt)
Values are given as fold change; only genes showing two-fold or greater change were considered. Both luciferase and scrambled siRNA controls were used in BT-20; only genes showing consistent alteration with both controls were included in the results reported here. The scrambled siRNA could be not used in MDA-MB-468 as these cells decrease MUC1 expression in response to this construct.<br><br>
Results
siRNA transfection decreases MUC1 expression in breast cancer cell lines
Two <span class="yellow">human</span> breast cancer cell lines, MDA-MB-468 and BT-20, were transiently transfected with a pool of four siRNA oligonucleotides directed against the MUC1 mRNA (468.siMUC1 and BT.siMUC1), or a control oligonucleotide directed against luciferase (468.siLuc and BT.siLuc). Both cell lines express high levels of MUC1, making them promising targets for this analysis. Western blots (Figure 1a) show successful knockdown of both the extracellular domain and cytoplasmic tail fragments of MUC1; luciferase siRNA does not substantially change the level of MUC1 compared to parental cells. 468.siMUC1 show a substantial decrease in the amount of MUC1-CT, while BT.siMUC1 show slightly less knockdown of MUC1-CT. Both MDA-MB-468 and BT-20 display a less dramatic decrease of MUC1 extracellular domain compared to MUC1-CT (Figure 1a); this likely represents protein synthesized prior to transfection, and may reflect differences in the turnover rates of the two subunits.
Analysis of the MUC1 extracellular domain by flow cytometry confirms that both cell lines substantially decrease MUC1 expression after siRNA (Figure 1b). By flow cytometry, 468.siMUC1 averaged 75% knockdown of MUC1 compared to 468.siLuc; and BT.siMUC1 averaged 50% knockdown relative to BT.siLuc. These effects could be titrated with the concentration of siRNA, were seen as early as 24 hours post-transfection (data not shown) and lasted to at least 96 h post-transfection (Figure 1b). All experiments were conducted within 48 to 96 hours after siRNA transfection. Similar results were obtained using two independent oligonucleotides designed in our lab (data not shown), designated '882' and '956' for the initial codon recognized by each.<br><br>Transcriptional changes are seen after MUC1 siRNA
Recent work indicates that MUC1 may affect transcription both directly via interaction with transcription factors and indirectly (for example, through modulating signaling). To study the effects of MUC1 knockdown in breast cancer cell lines, real-time PCR arrays were used to analyze transcription of 84 genes implicated in cancer. Only genes with greater than two-fold change were considered. Three genes (MAP2K1, VEGF, PDGFA) were altered two-fold or more after MUC1 siRNA in both MDA-MB-468 and BT-20 cells (Figure 2); two genes (ITGAV, MMP2) changed only in 468.siMUC1; and five genes (TIMP3, RAF1, JUN, TNF, CDC25A) only in BT.siMUC1. This list represents all genes affected greater than two-fold after MUC1 siRNA, rather than a select group. Three genes whose transcription was changed by less than two-fold are shown, two of which (PDGFB and ITGB1) are listed because they relate closely to genes altered by two-fold (PDGFA and ITGAV). The third, MYC, is included because western blots confirmed a substantial change at the protein level (Figure 3a) that may reflect both transcriptional and post-transcriptional regulation.
Interestingly, transcription of MAP2K1 was decreased in both cell lines after MUC1 siRNA. This gene encodes MEK1, one of the primary regulators of the ERK1/2 MAPK pathway [33], a network that has been linked several times to MUC1 [12,34-36]. We examined MEK1 and MEK2 levels by western blot to confirm decreased protein in MUC1 siRNA-treated cells (Figure 3a), and found that not only were total MEK1/2 levels lower in 468.siMUC1 and BT.siMUC1 compared to controls (0.48 and 0.68 relative to siLuc, respectively), but so were the basal amounts of active (phosphorylated) MEK1/2 (pMEK1/2; 0.12 and 0.42 relative to siLuc, respectively). Both 468.siMUC1 and BT.siMUC1 also showed reduced activation of ERK1/2 (dpERK1/2; 0.21 and 0.27 relative to siLuc, respectively), as would be expected with diminished signaling through MEK1/2; total ERK1/2 levels remain unchanged.
As both lines have high levels of EGFR and thus activate the MEK-ERK cascade intensely when stimulated with EGF [37], siRNA-transfected cells were treated with EGF. Notably, MUC1 siRNA impairs this important oncogenic pathway in MDA-MB-468 cells, as 468.siMUC1 display less pMEK1/2 in response to EGF than do 468.siLuc (Figure 3b). Interestingly, EGF treatment of BT-20 cells results in slightly higher pMEK1/2 levels in BT.siMUC1 compared to BT.siLuc. Though this result seems paradoxical in light of decreased MAP2K1 transcription in BT.siMUC1, it likely results from differential functions of Raf isoforms in combination with the increased RAF1 transcription (Figure 2) and protein level (Figure 3a) in these cells. Specifically, B-Raf is thought to be the main activator of MEK under normal conditions; Raf-1 activates MEK in response to stimulus [38]. Thus, it appears that basal pMEK1/2 levels are not greatly affected by Raf-1 overexpression in BT.siMUC1 cells, likely because MEK is regulated primarily by B-Raf under normal growth conditions. In contrast, when the cells are stimulated (EGF), increased Raf-1 levels in BT.siMUC1 leads to heightened pMEK1/2 (Figure 3b).<br><br>MUC1 siRNA increases apoptosis in MDA-MB-468 but not BT-20
We next examined whether MUC1 knockdown and its associated transcriptional alterations would affect overall cellular events. As several of the genes shown in Figure 2 are important in regulating proliferation and survival, and because of the recently described role of MUC1 in modulating apoptosis in response to cellular stresses [20,21,24], we first analyzed whether MUC1 siRNA would alter apoptosis in these lines. Although there was no change in basal apoptosis in either line (Figure 4a), we observed that the cell lines responded differently when trypsinized for re-plating 24 hours after transfection (Figure 4b). Interestingly, 468.siMUC1 cells show greater apoptosis after trypsinization than do 468.siLuc (49.8% versus 34.0%, respectively), while BT-20 cells from both siRNA treatments display similar levels of apoptosis (around 22%).
To examine whether this phenomenon is specific to trypsin treatment or part of a general stress response involving MUC1, we subjected cells to a panel of stresses and measured cell death. In agreement with the patterns seen with trypsinization, BT.siLuc and BT.siMUC1 respond similarly to all treatments (data not shown), while 468.siMUC1 die more readily than 468.siLuc in response to trypsin, G418, hydrogen peroxide, or celecoxib, a chemotherapeutic that targets the cyclooxygenase-2 (COX-2) pathway (Figure 4c); these data were confirmed with two independent siRNA constructs (data not shown).
Like the MAPK pathway, AKT signaling has been linked to MUC1 in cancer. Although transcription of AKT was not altered in MUC1 siRNA-treated cells, the results of our apoptosis studies prompted us to investigate levels of AKT further. As expected, the total AKT protein level is not greatly changed after MUC1 siRNA in either cell line, though the active form (pAKT) is increased in both 468.siMUC1 and BT.siMUC1 compared to controls (Figure 3a). This result disagrees with MUC1 activation of the AKT pathway in <span class="yellow">rat</span> 3Y1 cells [18], and may reflect regulation more appropriate to breast cancer cells; this is supported by activation of AKT in response to MUC1 siRNA in other lines [21]. In addition, there is a striking difference in the relative amounts of AKT and pAKT in the two cell lines (Figure 4d). When lysates from both lines are exposed to film for the same length of time (overexposure masks the differences between BT.siLuc and BT.siMUC1 that are apparent in Figure 3a), it is clear that pAKT levels are much higher in BT-20 than in MDA-MB-468, despite lower total AKT expression. This difference in AKT activation between MDA-MB-468 and BT-20 likely contributes to the disparity in their sensitivity to the increased apoptosis expected with loss of MUC1.<br><br>MUC1 siRNA alters proliferation and invasion
As MUC1 is involved in apoptosis, we next analyzed its effects on proliferation. BrdU and [3H]thymidine incorporation were used to analyze proliferation after MUC1 siRNA. 468.siMUC1 cells show a significant decrease in [3H]thymidine incorporation compared to 468.siLuc, while intriguingly, BT.siMUC1 cells show a significant increase in proliferation (Figure 5a). Growth curves mirror these results, as do experiments with the two independent MUC1 siRNA oligonucleotides (data not shown). Note that these assays require trypsinizing cells 24 hours post-transfection; therefore, the results in the MDA-MB-468 line could stem from the changes in apoptosis described in the previous section, rather than a true effect on proliferation. To control for this, we incubated non-trypsinized, siRNA-transfected cells at similar confluence with BrdU to measure incorporation. The 'clumped' profile of cells (contrast to Figure 4b) is likely a result of the acid denaturation (recommended by the antibody manufacturer), as it occurs uniformly in these experiments. BrdU incorporation (Figure 5b) confirms that the [3H]thymidine results are not solely due to alterations in apoptosis, as 468.siMUC1 cells incorporate less BrdU than 468.siLuc; once again, BT.siMUC1 cells show increased proliferation over BT.siLuc.
Given the role of MUC1 in adhesion, we examined whether MUC1 siRNA affects cellular invasion. In transwell assays, BT-20 cells invaded poorly, regardless of the siRNA used (data not shown). However, MDA-MB-468 cells invade more readily, and were analyzed on a panel of three different extracellular matrix proteins. Interestingly, 468.siMUC1 cells display somewhat decreased invasion on collagen IV, laminin, and fibronectin matrices, and on a no-matrix control (Figure 5c), which is in agreement with the trend towards decreased metastasis observed in Muc1-/- × <span class="yellow">MMTV</span>-PyV MT <span class="yellow">mice</span> [8].<br><br>Transfection of MUC1 rescues the 468.siMUC1 phenotype
To determine if the above effects are specific to MUC1, we created stable transfectants of the MDA-MB-468 line using empty vector (468.Neo) or a full-length MUC1 construct (468.MUC1Δ8) that is resistant to one of the independent MUC1-directed oligonucleotides ('882'). These cells were maintained in G418-containing medium to retain transgene selection. As expected, 468.MUC1Δ8 cells show higher levels of both the MUC1 extracellular domain and the MUC1-CT than do 468.Neo (Figure 6a). Note that 468.Neo have MUC1 expression comparable to parental MDA-MB-468; the exposures in Figure 6a are lighter than those in Figure 1a, in order to clearly show the relative levels of MUC1 in the stable transfectants. After MUC1 siRNA, 468.MUC1Δ8 lose some MUC1 (likely endogenous protein, which is not siRNA-resistant) but retain high-level expression, while 468.Neo show a decrease in MUC1 levels similar to parental 468.siMUC1 cells (Figures 6a,b). The difference in the amount of MUC1 knockdown between 468.Neo and 468.MUC1Δ8 is highlighted by the purple shading in Figure 6b.
BrdU incorporation (Figure 6c) indicates that 468.Neo show decreased nucleotide incorporation after MUC1 siRNA compared to control (3.3% versus 25.0%, respectively); this is not seen in 468.MUC1Δ8 cells, which show similar levels of BrdU incorporation regardless of the siRNA used (21.5% for luciferase, 23.9% for MUC1). 468.Neo cells display a more dramatic decrease in BrdU incorporation after MUC1 siRNA than what is seen in parental 468.siMUC1 cells, which may reflect the additional stress of being maintained in G418-containing medium. Similarly, analysis of apoptosis in trypsinized cells indicates that the increased apoptosis seen in parental 468.siMUC1 cells is also present in the 468.Neo line after MUC1 siRNA (Figure 6d; 43.6% in control versus 59.6% in MUC1 siRNA). However, in 468.MUC1Δ8 cells, the level of apoptosis after luciferase siRNA (34.1%) is lower than that in 468.Neo cells; MUC1 siRNA increases the amount of apoptosis slightly (42.8%), restoring it to a level similar to that seen in luciferase siRNA-treated 468.Neo cells. Together, these studies suggest that the above-described results are specific to MUC1, as stable transfection of an siRNA-resistant MUC1 rescues the phenotype seen in 468.siMUC1 cells.<br><br>
Discussion
This report describes both the transcriptional alterations seen after transfection with MUC1 siRNA in <span class="yellow">human</span> breast cancer cells and the effects on events such as apoptosis and proliferation. The two cell lines used (MDA-MB-468 and BT-20) were chosen for high expression of MUC1 and a substantial (50% to 75%), consistent decrease in MUC1 expression after siRNA. Both lines have epithelial morphology, form tumors slowly in <span class="yellow">nude mice</span> [37], have mutant p53 [39,40], express EGFR [37], and lack estrogen receptor α [41]. One striking difference between these lines, however, is their response to MUC1 siRNA. MDA-MB-468 cells behave as expected for loss of an oncogene: MUC1 siRNA correlates with increased apoptosis in response to stress, decreased proliferation, and reduced invasion. In contrast, BT.siMUC1 cells proliferate more rapidly than BT.siLuc cells with little effect on apoptosis.
Much of the phenotype of these cells can be understood in light of protein levels and transcriptional activity after MUC1 siRNA. As mentioned, both MDA-MB-468 and BT-20 display increased pAKT after MUC1 siRNA, but the ratio of active to total AKT is considerably higher in the BT-20 line, which may help these cells resist the increased apoptosis expected with loss of MUC1. Myc levels are also higher in both cell lines after MUC1 siRNA, although the ability of Myc to promote proliferation and apoptosis in different cellular contexts [42] complicates the interpretation of this finding.
Both cell lines show reduced transcription of VEGF, PDGFA, PDGFB, and MAP2K1 (MEK1) after MUC1 siRNA. The genes encoding vascular endothelial growth factor (VEGF) and the A and B chains of platelet-derived growth factor (PDGF-A and PDGF-B) are interesting as these proteins have been heavily implicated in angiogenesis, suggesting a novel function for MUC1 in regulating this process. Vascular endothelial growth factor expression in cancer is linked to tumor growth and metastasis [43,44]; platelet-derived growth factor is also angiogenic, but has an additional role in stimulating desmoplasia [45]. Reduced transcription of these genes after MUC1 siRNA suggests that MUC1 may foster angiogenesis and stromal proliferation, although this must be confirmed in a more appropriate model system.
Decreased MAP2K1 (MEK1) transcription after MUC1 siRNA provides a novel mechanism by which MUC1 can affect the ERK1/2 MAPK pathway. MUC1 has often been linked to the Ras-Raf-MEK-ERK cascade [12,34-36,46], and at least two mechanisms by which MUC1 can alter MAPK signaling have been described: MUC1 interaction with and phosphorylation by the EGFR family [12,13], and MUC1 binding to the Grb2/Sos complex that activates Ras [46]. Reduction of MEK1 levels after MUC1 siRNA agrees with the role of MUC1 in strengthening MAPK signaling, and indicates that MUC1 can regulate both the transcription and activity of members of this pathway.
Two additional MAPK pathway members are altered specifically in BT.siMUC1, with no corresponding change in 468.siMUC1 cells. These genes are RAF1 and JUN which are increased and decreased, respectively, after MUC1 siRNA. Raf-1 and c-Jun both function outside of the ERK1/2 MAPK pathway, which may explain the seeming paradox of increased RAF1 transcription with simultaneous decreases in MAP2K1 and JUN. Specifically, Raf-1 can inhibit ASK1 (apoptosis signal-regulated kinase 1) upstream of p38 and JNK (Jun N-terminal kinase) [38]. ASK1 phosphorylates JNK in response to stress, resulting in activation of c-Jun and stimulation of apoptosis [47], indicating that the coordinate up-regulation of RAF1 and down-regulation of JUN may provide a potent anti-apoptotic effect in BT.siMUC1.
Regulation of life and death is also a hallmark of the CDC25A and TNF gene products. CDC25A is a phosphatase that stimulates cell cycle progression [48], thus the effects of its decrease in BT.siMUC1 are unclear in light of the increased proliferation of these cells. However, the CDC25 proteins (A, B, and C) were recently shown to have greater functional overlap than was previously thought [49], suggesting that the other two isoforms may compensate for reduced CDC25A levels. TNF encodes tumor necrosis factor (TNF)α, known for its potent, cell type-specific control of life and death. In tumor cells, TNFα expression can promote proliferation and inhibit apoptosis [50], suggesting that increased TNF transcription in BT.siMUC1 could contribute to the increased proliferation seen in these cells.
Interestingly, the increase in TNF is accompanied by decreased transcription of TIMP3, encoding tissue inhibitor of metalloproteinases (TIMP)3. The TIMP family disrupts the function of matrix metalloproteinases (MMPs), generally resulting in decreased invasion [51]. TIMP3 is unique in that it can also inhibit TNFα converting enzyme (TACE), which activates TNFα by cleaving it from the cell surface [50]. Reduced expression of TIMP3 would, therefore, foster signaling through TNFα by releasing inhibition of TNFα converting enzyme. In agreement with this, TIMP3 can promote apoptosis [52]; thus, its down-regulation in BT.siMUC1 provides another mechanism by which these cells are able to resist the increased apoptosis expected with loss of MUC1.
Another TIMP target responds to MUC1 siRNA, as 468.siMUC1 cells show significantly increased expression of MMP2 (encoding MMP-2/gelatinase A), the product of which degrades type IV collagen [52]. In breast cancer, the ratio of active to latent MMP-2 increases with tumor progression; MMP-2 may facilitate both angiogenesis and metastasis [52]. Its increase after loss of MUC1 is, therefore, unexpected, but at least two factors may clarify this result. First, though MMP-2 levels are increased in <span class="blue">mouse</span> mammary tumors, its expression is confined to the stroma [53], suggesting that increased MMP2 transcription after loss of the epithelium-specific MUC1 might reflect a shift towards a more mesenchymal phenotype. Second, MMP-2 levels are increased by overexpression of erbB2 [52]; previous studies have shown that erbB2 and Muc1 expression are mutually exclusive in mammary tumors [27], implying that MMP2 might be part of a transcriptional profile linked to low MUC1 levels.
It is intriguing that, despite increased MMP2 transcription, invasion is decreased in 468.siMUC1 cells, even on collagen IV. This may reflect insufficient activation of MMP-2, as the precursor protein must be cleaved for enzymatic function [52]. Alternatively, the slowed invasion of these cells may relate to impaired adhesion resulting from decreased transcription of ITGAV and ITGB1 (αv and β1 integrins, respectively). Integrin signaling is tied to life-or-death decisions in epithelial cells, and integrin expression is vital for processes from wound healing to metastasis [54]. Integrin αvβ3 is implicated in facilitating metastasis of breast cancer cells to bone [55]; decreased transcription of ITGAV after MUC1 siRNA may, therefore, suggest that MUC1 is involved in this lethal process as well.
The MUC1 oncogene has been linked to apoptosis [18,20,26], proliferation [17], and transcription [21,23-25] in cancer. However, the two cell lines chosen for our study display very different responses to MUC1 siRNA, indicating that regulation of MUC1 in breast cancer is likely quite complex and cautioning against over-generalization of results from individual cell lines. Previous reports suggest that, though most studies outline a clearly oncogenic role for MUC1 in breast cancer, the exact details may vary depending on factors such as cell type and signaling context. For example, MUC1 stimulates Fas-mediated apoptosis in CHO cells [26], quite unlike the inhibition of apoptosis seen in other cell lines. Similarly, though MUC1 drives mammary oncogenesis in its own right [9] and facilitates tumorigenesis driven by other oncogenes [7,8], Muc1 is selectively down-regulated in c-neu-induced <span class="blue">mouse</span> mammary tumors [27], indicating that the context of oncogenic signaling is vital to understanding the function of MUC1.
Thus, it is important to consider the relative levels of knockdown of MUC1 in the two cell lines: BT-20 cells reduce MUC1 expression after siRNA less strongly than do MDA-MB-468 (50% versus 75% knockdown, respectively). As MUC1 serves as a scaffold [11], overexpression of MUC1 relative to its associated signaling proteins might create a dilution effect, sequestering signal transducers away from each other; this would be relieved by MUC1 siRNA. Thus, enough MUC1 may be retained in BT.siMUC1 cells for its oncogenic effects, while signaling complex formation would be enhanced by lowering the amount of MUC1 relative to other signaling proteins.<br><br>Conclusion
The contrast between the MDA-MB-468 and BT-20 lines in response to MUC1 siRNA serves as a reminder that simplified models such as cell lines fail to encompass the complexity of intact biological systems. This report describes transcriptional alterations seen after MUC1 knockdown: decreased transcription of MAP2K1, VEGF, PDGFA, ITGAV, TIMP3, CDC25A, and JUN, and increased transcription of MMP2, TNF, and RAF1. The alterations in MAP2K1, RAF1, and JUN represent a novel means by which MUC1 can affect ERK1/2 signaling: transcriptional regulation of MAPK pathway members. Oncogenic events are also altered in both cell lines after MUC1 siRNA. These results strengthen the growing ties linking MUC1 and transcriptional regulation, and suggest that the role of MUC1 in breast cancer may be more complex than a direct correlation between MUC1 level and oncogenic function.<br><br>Abbreviations
BrdU = bromodeoxyuridine; EGF = epidermal growth factor; EGFR = epidermal growth factor receptor; ERK = extracellular signal regulated kinase; MAPK = mitogen activated protein kinase; MEK = MAPK and ERK kinase; MMP = matrix metalloproteinases; MUC1-CT = MUC1 cytoplasmic tail; MUC1-EX = MUC1 extracellular subunit; PBS = phosphate-buffered saline; PI = propidium iodide; siRNA = small interfering RNA; TIMP = tissue inhibitor of metalloproteinases; TNF = tumor necrosis factor.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
CLH performed all studies and composed the manuscript. SJG participated in the design and coordination of the studies and contributed strongly to the revision of the manuscript. Both authors have read and approved the manuscript.<br><br>
<h3>pmcA2518116</h3>CHEK2 Mutations Affecting Kinase Activity Together With Mutations in TP53 Indicate a Functional Pathway Associated with Resistance to Epirubicin in Primary Breast Cancer
Abstract
Background
Chemoresistance is the main obstacle to cure in most malignant diseases. Anthracyclines are among the main drugs used for breast cancer therapy and in many other malignant conditions. Single parameter analysis or global gene expression profiles have failed to identify mechanisms causing in vivo resistance to anthracyclines. While we previously found TP53 mutations in the L2/L3 domains to be associated with drug resistance, some tumors harboring wild-type TP53 were also therapy resistant. The aim of this study was; 1) To explore alterations in the TP53 gene with respect to resistance to a regular dose epirubicin regimen (90 mg/m2 every 3 week) in <span class="yellow">patients</span> with primary, locally advanced breast cancer; 2) Identify critical mechanisms activating p53 in response to DNA damage in breast cancer; 3) Evaluate in vitro function of Chk2 and p14 proteins corresponding to identified mutations in the CHEK2 and p14(ARF) genes; and 4) Explore potential CHEK2 or p14(ARF) germline mutations with respect to family cancer incidence.<br><br>Methods and Findings
Snap-frozen biopsies from 109 <span class="yellow">patients</span> collected prior to epirubicin (as preoperative therapy were investigated for TP53, CHEK2 and p14(ARF) mutations by sequencing the coding region and p14(ARF) promoter methylations. TP53 mutastions were associated with chemoresistance, defined as progressive disease on therapy (p = 0.0358; p = 0.0136 for mutations affecting p53 loop domains L2/L3). Germline CHEK2 mutations (n = 3) were associated with therapy resistance (p = 0.0226). Combined, mutations affecting either CHEK2 or TP53 strongly predicted therapy resistance (p = 0.0101; TP53 mutations restricted to the L2/L3 domains: p = 0.0032). Two <span class="yellow">patients</span> progressing on therapy harbored the CHEK2 mutation, Arg95Ter, completely abrogating Chk2 protein dimerization and kinase activity. One <span class="yellow">patient</span> (Epi132) revealed family cancer occurrence resembling families harboring CHEK2 mutations in general, the other <span class="yellow">patient</span> (epi203) was non-conclusive. No mutation or promoter hypermethylation in p14(ARF) were detected.<br><br>Conclusion
This study is the first reporting an association between CHEK2 mutations and therapy resistance in <span class="yellow">human</span> cancers and to document mutations in two genes acting direct up/down-stream to each other to cause therapy failure, emphasizing the need to investigate functional cascades in future studies.<br><br><br><br>Introduction
Chemoresistance is the main obstacle to cure in most malignancies, including breast cancer. While adjuvant chemotherapy may reduce the hazard rate of relapse by about one third in breast cancer <span class="yellow">patients</span> [1], the majority among <span class="yellow">patients</span> harboring micro- metastases are not cured by today's standards. Considering <span class="yellow">patients</span> harboring distant metastases, resistance and therapy failure inevitably occurs, in general over a time period of less than one year for each individual regimen [2].
Despite extensive experimental research [3], little data are available considering chemoresistance in vivo. For anthracycline therapy in breast cancer, topoisomerase-II amplifications have been associated with a dose-responsiveness different from what is observed in non-amplified tumors 4, 5. Several studies have tried to generate “prediction profiles” based on gene expression microarrays [6], [7], [8], however, none of the different profiles generated expressed a sensitivity suitable for clinical applications, or have been successfully reproduced by others (see references to original works in [9] and [10]).
p53 (the protein encoded by the TP53 gene) plays a key role in executing DNA-damage induced apoptosis and growth arrest [11]. Previously, our group reported mutations in the zink-binding domains L2 (codons 163–195) and L3 (codons 236–251) of p53 critical to DNA binding [12] to be associated with but not fully predictive for resistance to chemotherapy with a low-dose weekly anthracycline [13] or a mitomycin plus 5-fluoro-uracil containing [14] regimen. Similar findings were reported by another group [15]. In contrast, others reported TP53 mutations to predict sensitivity to a dose-dense epirubicin-cyclophosphamide regimen [16].
The finding that some tumors harboring wild-type TP53 may be resistant to anthracycline therapy lead us to postulate that other genes involved in the p53 pathway could be mutated in these tumors [3]. p53 is activated by post-translational modifications, and the protein is phosphorylated at multiple amino acids [17]. Phosphorylation at Ser 20 (Ser 23 in <span class="yellow">mice</span>) by the Chk2 protein (coded by the CHEK2 gene) in response to DNA damage activates p53 by inhibiting binding to, and deactivation by, the MDM2 (<span class="yellow">Mouse</span> Minute 2 homolog; HDM2) protein [18], [19], [20]. While experimental studies have suggested a critical role of Chk2 in activating p53 apoptotic response to genotoxic stress [21], [22], others claim Chk2 to be dispensable for p53 activation with respect to apoptosis as well as growth arrest [23]. Following an initial report of a CHEK2 germline mutation in a family filling the characteristics of a Li-Fraumeni syndrome (LFS) [24], recent papers have suggested germline mutations in CHEK2 to be associated with a moderately increased risk of breast and colon cancers (see references in [25]). Recently, we discovered a somatic, nonsense CHEK2 mutation in a single <span class="yellow">patient</span> expressing resistance to doxorubicin low dose therapy [26].
A second mechanism of p53 activation is through p14(ARF) (p19 in <span class="yellow">mice</span>) function. p14(ARF) does not phosphorylate p53, but inhibits MDM2 dependent p53 degradation through direct MDM2 binding. While p14(ARF)-mediated p53 activation has been linked to oncogene-induced p53 activation and, in general, considered not involved in response to DNA damage (see references in [27]), p14(ARF) may be activated through the E2F1/retinoblastoma pathway [28]. Importantly, two recent studies revealed lack of p19 (<span class="blue">mouse</span> homologue of <span class="yellow">human</span> p14(ARF)) function in <span class="yellow">mice</span> to inhibit p53 tumor suppressor function in response to ionizing radiation as well as DNA damaging agents [29], [30].
The aim of this study was 1) to explore alterations in the TP53 gene with respect to resistance to a regular dose epirubicin regimen (90 mg/m2 body surface every 3 week) in <span class="yellow">patient</span> with primary, locally advanced, breast cancer; 2) To explore defects in potential mechanisms activating p53 in response to DNA damage in breast cancer as a cause of drug resistance in wild-type tumors. To do so, we sequenced the complete coding regions for the CHEK2 and p14(ARF) genes and analyzed for p14(ARF) promoter hypermetylations; 3) Evaluate in vitro function of potential Chk2 and p14(ARF) protein translates corresponding to identified mutations in the CHEK2 and p14(ARF) genes; 4) Identify potential TP53, CHEK2 and p14(ARF) mutations to be germline, explore the incidence of different cancers among affected relatives with respect to specific mutations. By comparing in vitro characteristics of specific mutations to drug sensitivity and family cancer risk syndromes, this may add to our understanding of the importance of these gene cascades executing response to DNA damage versus tumor suppression activity.
Analyzing tumor samples from a total of 109 primary locally advanced breast cancer <span class="yellow">patients</span> treated with epirubicin 90mg/3 weekly, we found TP53 mutations affecting the L2/L3 domains or protein dimerization, as well as non-functional CHEK2 mutations abrogating dimerization and phosphorylation, to be associated with therapy resistance; no mutation or promoter hypermethylations of the p14(ARF) gene was discovered. Our findings suggest a critical role for Chk2 with respect to DNA-damage-dependent p53 activation and resistance to anthracycline therapy in <span class="yellow">human</span> breast cancer.<br><br>Materials and Methods
<span class="yellow">Patients</span>
A total of 223 <span class="yellow">patients</span> with locally advanced non-inflammatory breast cancer (T3-4 and/or N2) were randomly allocated to primary treatment either with epirubicin 90 mg/m2 or paclitaxel 200 mg/m2. The primary aim of the study was identification of markers predicting drug resistance to the regimens. Thus, the reason for randomizing <span class="yellow">patients</span> was not for effect comparison, but to achieve similar <span class="yellow">patient</span> cohorts in the two arms. Based on the findings of a clinical lack of cross-resistance between anthracyclines and taxane therapies in breast cancer [31], we hypothesized the mechanisms of resistance to be different between the two compounds. While the analysis of tumor samples from the paclitaxel is ongoing, we here report our findings from the <span class="yellow">patients</span> allocated to the epirubicin arm.
The epirubicin arm included a total of 109 <span class="yellow">patients</span> (age 28 to 70 years, median 51 years). Two <span class="yellow">patients</span> were analyzed for gene mutations but omitted from statistical analysis as protocol violators; histopathological examination revealed one <span class="yellow">patient</span> (Epi089) to harbor a sarcomatoid tumor, while one <span class="yellow">patient</span> Epi232 was erroneously enrolled with stage II disease.
The study protocol was approved by the Regional Ethical Committee (Norwegian Health Region III), including formal Biobank registration in accordance to Norwegian law. The study and protocol is registered under the Norwegian Social Science Data services ((www.nsd/uib/personvern/database/), University of Bergen project no 16297 and Helse Bergen project no 13025). Each <span class="yellow">patient</span> gave written informed consent.<br><br>Tissue Sampling
Before commencing chemotherapy, each <span class="yellow">patient</span> had an incisional tumor biopsy as described previously [14]. All tissue samples were snap-frozen immediately on removal in the theatre.<br><br>Treatment Regime and Staging
Primary treatment consisted of epirubicin (90 mg/m2) administered as a 3-weekly schedule. Treatment was scheduled for four cycles unless progression occurred at an earlier stage. Clinical response was assessed before each treatment cycle, and the final response evaluated 3 weeks after the 4th cycle for overall response classification. Because the protocol was implemented by October 1997 with <span class="yellow">patients</span> enrolled between November 1997 and December 2003, responses were consistently graded by the UICC system [32] and not the more recently implemented “RECIST” criteria [33]. Thus, responses were classified as CR (Complete Response, complete disappearance of all tumor lesions), PR (Partial Response, reduction ≥50% in the sum of all tumor lesions, calculated for each as the product of the largest diameter and the one perpendicular to it), PD (Progressive Disease, increase in the diameter product of any individual tumor lesion by ≥25%), and SD (Stable Disease, anything between PR and PD). To analyze for the predictive value of the different parameters, similar to our previous studies [13], [14] we compared PD tumors (non responders) with the combined group of tumors classified as SD/PR/CR (responders); the reason for this approach is discussed in detail elsewhere [34]. Median follow-up time was defined from <span class="yellow">patient</span> inclusion in the study up to October 31, 2006. Deaths attributable to causes other than breast cancer were treated as censored observations.
All <span class="yellow">patient</span> records were subject to central audit for response classification (by E.L., B.Ø. and P.E.L.). Response classifications were completed and approved without any knowledge about result from laboratory analysis.<br><br>RNA Purification
Total RNA was purified by Trizol (Life Technologies, Inc.) extraction from snap-frozen tissue samples according to manufacturer's instructions. After extraction, the RNA was dissolved in 100 µl of DEPC treated ddH2O. cDNA was synthesized by reverse transcription using Transcriptor reverse transcriptase (Roche), according to the manufacturer's protocol.<br><br>DNA Purification
Genomic DNA from tumor biopsies and blood lymphocytes was isolated using QIAamp DNA Mini kit (Qiagen, Chatsworth, CA) according to the manufacturer's protocol.<br><br>Mutation Analysis
All mutational analysis was performed blinded to clinical data. Mutations in TP53, CHEK2 and p14(ARF) genes were analyzed by PCR (or nested PCR) amplification and sequencing of PCR product, or by cloning of PCR products and sequencing of the resulting plasmids (all primers described in Table 1). Cloning was performed using the TOPO TA Cloning kit (Invitrogen). Sequencing of clones was performed until at least 10 different sequences covered all parts of the CHEK2 coding sequence. DNA sequencing was carried out directly on 1 µl PCR product or plasmid using Big Dye terminator mix (Applied Biosystems). Capillary gel electrophoresis, data collection, and sequence analysis were done on an automated DNA sequencer (ABI 3700). When a mutation was detected, the relevant exon was amplified by PCR from genomic tumor DNA and DNA from blood lymphocytes and sequenced for verification and germline detection. (Primers described in Table 1).<br><br>Loss of Heterozygosity (LOH)
Loss of heterozygosity (LOH) in tumors with mutations in CHEK2 was assessed using the microsatellite marker, D22S275, which maps to intron 4 of CHEK2. LOH in tumors with mutation in TP53 was assessed using two markers, one variable number tandem repeat in intron 1 [35] and a CA repeat close to the TP53 gene [36]. Fluorescently end-labeled primers were used in the PCR, and the PCR products were analyzed on an ABI 3700. LOH was evaluated by comparing the allele peak-height ratios from blood DNA and tumor DNA. A sample was scored as having AI (Allelic Imbalance) when a reduction in peak height of one allele in tumor sample was at least 18% compared with that of blood DNA from the same <span class="yellow">patient</span> [37].<br><br>Analysis of p14(ARF) promoter methylation
Genomic DNA was subjected to bisulphate conversion using the CpGenome DNA Modification Kit (Intergen) according to the manufacturer's protocol. Both the unmethylated- and methylated-specific PCRs were performed in 50 µl reaction mixes containing 2.5 U AmpliTaq Gold DNA Polymerase (Applied Biosystems), 1× PCR buffer, 1.5 mM MgCl2, 0.1 mM of each deoxynucleotide triphosphate, 0.2 µM of each primer (Table 1) and 2 µl of modified genomic DNA. Thermocycling conditions for both the unmethylated- and methylated-specific PCRs were an initial step of 5 minutes at 95°C followed by 35 cycles of 30 sec. at 94°C, 30 sec. at 60.5°C and 60 sec. at 72°C before a final elongation step at 72°C for 7 min.<br><br>Chk2 Dimerisation
Chk2 mutant's ability to form dimers with the wild-type protein was investigated by immunoprecipitation. U-2-OS cells were co-transfected with expression vectors expressing wild-type Chk2 with N-terminal Xpr-tag (pcDNA4/HisMax, Invitrogen) and mutated Chk2 forms with C-terminal V5-tag (pcDNA3.1/V5-His, Invitrogen). Transfection was performed using FuGene 6.0 transfection reagent (Roche) according to the manufacturer's instructions. Cells were harvested in lysisbuffer (50 mM TrisHCl pH 8.0, 150 mM NaCl, 0.5% NP40, 5 mM EDTA pH 8.0) 48 hours after transfection. An aliquote of the cell lysate was harvested for subsequent Chk2-mutant-V5 transfection verification. Samples were further incubated with A/G Pluss Agarose beads (Santa Cruz Biotechnology) at 4°C for 25 minutes before the beads were removed by centrifugation at 5000g for 4 minutes and the samples were incubated with 1.5 µg anti-V5 (Invitrogen) at 4°C for 90 minutes. Fresh A/G Pluss Agarose beads were added and the samples were incubated for another 90 minutes at 4°C. The beads were washed three times with 1×PBS, before being separated on a 10% polyacrylamide gel and blotted on to a nitrocellulose membrane. Chk2-wild-type-Xpr co-precipitated with Chk2-mutant-V5 was detected through incubations with anti-Xpr antibody (Invitrogen), HRP-conjugated secondary antibody and ECL detection reagent (GE Healthcare).<br><br>Kinase Activity
Chk2 mutant's ability to function as kinases was investigated through an in vitro kinase assay. The V5 expression vectors used for the dimerisation study were also used to express Chk2 mutants in the kinase assay. U-2-OS cells were transfected using the FuGene 6.0 transfection reagent (Roche) according to the manufacturer's instructions. Cells were then incubated at 37°C in 5% CO2 and humidified atmosphere. After 24 hours doxorubicin (Nycomed Pharma) was added to the media to a final concentration of 50ng/ml and the cells were further incubated for 24 hours before harvest. 75 cm2 of 90% confluent cells were harvested in 500 µl lysis buffer (50 mM HEPES, 150 mM NaCl, 10% glycerol, 0.5% Triton X-100, 2 mM MgCl2, 5 mM EDTA), and the cytosol was incubated for 90 minutes at 4°C with 50 µl 50% Glutathione Sepharose beads (Amersham Biosciences) linked to anti-V5 antibody (Invitrogen). The beads were then washed twice with lysisbuffer containing 500 mM NaCl and twice with kinase assay buffer (50 mM HEPES, 10 mM MgCl2, 5 mM MnCl2, 2.5 mM EGTA). The beads received 30 µl kinase assay buffer with 7.5 µM cold ATP, 10 µCi 32P-gamma-ATP (GE Healthcare) and 2 µg isolated Cdc25C peptide, and was incubated at 30°C for 30 minutes. Samples were separated on a 12.5% polyacrylamide gel and blotted on to a nitrocellulose membrane. A radiosensitive imaging plate was exposed to the membrane and the plate was read in a FLA200 imager (Fuji).
The kinase assay described above was also used to determine the Chk2 mutants' kinase activity after co-transfection of each Chk2 mutant and wild-type Chk2 in equal amounts.<br><br>Statistical Analysis
Statistical analysis was performed using the Primer of Biostatistics system, version 5.0 [38]. The differences in the distribution of TP53 and CHEK2 mutations among <span class="yellow">patients</span> revealing a PD and the responders were analyzed with use of Fisher's exact test. P-values are reported as accumulated two-sided. Because of the limited time of the follow-up, no formal statistical assessment of overall survival was performed. Relapse-free survival was analyzed by the log-rank test. Details regarding outcome in individual <span class="yellow">patients</span> with mutations are shown in Table 2 and 3 to make them available to the reader.<br><br>
Results
TP53 Mutations and Response to Therapy
The TP53 mutations identified in the tumors of the <span class="yellow">patients</span> treated with epirubicin together with the clinical response to therapy and follow-up data are presented in Table 2. Somatic TP53 mutations were identified in 23 (21.5%) of the <span class="yellow">patients</span>. Normal tissue (WBC) was available from 18 of these for germline characterization, revealing none of the mutations identified to be germline alterations. Of the 23 mutations detected, 20 were missense and 3 were nonsense. One mutation (del483CAT) has not been reported previously either in breast cancer or in any other tumor type (IARC database: http://www.iarc.fr/p53/). Twelve of the mutations directly or indirectly affected the L2/L3 domains of the p53 protein (Table 2) previous found to predict a poor prognosis [39] and drug resistance [14], [40]. For statistical comparison, mutation Gly325Ter (<span class="yellow">patient</span> Epi215) located to the tetramerization domain is grouped together with the mutations affecting the L2/L3 domain, since this mutation leads to truncation of the protein and with loss of tetramerization and functional defects similar to L2/L3 mutations [41].
There was a statistical significant correlation between TP53 mutation status and lack of treatment response (PD) (Table 4; p = 0.0358; Fisher exact test). When tumors harboring TP53 mutations affecting the p53 L2/L3 DNA-binding domains were compared to those with wild-type TP53 or TP53 mutations outside the L2/L3 domains, this correlation was further strengthened (p = 0.0136).
The previously described TP53 polymorphism, Arg72Pro [42] was detected in 31 (29%) of our <span class="yellow">patients</span>. No correlation was found between this polymorphism and lack of treatment response (p = 0.2750; Fisher exact test) or TP53 mutational status (p = 0.2024).<br><br>CHEK2 Mutations and Response to Therapy
Table 3 presents the <span class="yellow">patients</span> with detected CHEK2 mutations together with a description of the clinical response and follow up-data. CHEK2 mutations were identified in three out of the 109 <span class="yellow">patients</span> (2.8%). Notably, each of the CHEK2 mutations identified was also present in <span class="yellow">patient</span> lymphocyte DNA, confirming a germline origin. The Arg95Ter (C283T) mutation is novel. This mutation was present in two <span class="yellow">patients</span> (Epi132 and Epi203) living in different parts of Norway with no known family relationship. However, linkage analysis using microsatellite markers (D22S275, D22S272, D22S1172 and D22S423) suggested a common founder mutation (data not shown). The C283T transition generates a novel stop codon in exon 1 of CHEK2, leading to truncation of the Chk2 protein. LOH analysis indicated loss of the wild-type CHEK2 allele in the both tumors from the two <span class="yellow">patients</span> harboring this mutation (Epi132 and Epi203). Both these tumors were non-responsive to epirubicin therapy (PD). In contrast, the third <span class="yellow">patient</span> with a germline CHEK2 mutation (<span class="yellow">patient</span> Epi151; point mutation at T1091C, Ile364Thr) had a partial response to epirubicin therapy. This tumor was non-informative with respect to LOH. Taking all CHEK2 mutations together, they predicted resistance to epirubicin (p = 0.0226).
The previously described silent Glu84Glu (A252G) polymorphism [24], [43] in exon 1 was detected in two (1.9%) <span class="yellow">patients</span>. No association between this polymorphism and treatment response was recorded.
One of the tumors (Epi203) harboring the C283T substitution (Arg95Ter) also harbored a somatic TP53 mutation in codon 175, Arg175His, located in the L2 domain of p53 (Table 2). This mutation was detected in another four of our <span class="yellow">patients</span> treated with epirubicin (Table 2). In addition, TP53 Arg175His mutation was recorded in one <span class="yellow">patient</span> of our previous study evaluating response to doxorubicin [13]. The fact that none of the Arg175His <span class="yellow">patients</span> presented here or in our previous study revealed resistance to therapy (PD) suggests this mutation may not cause resistance to anthracyclines in breast cancers in vivo. Omitting the tumor harboring both a CHEK2 and a TP53 mutation (<span class="yellow">patient</span> Epi203) from statistical analysis, Chk2 mutations (n = 2) were non-significantly associated with therapy resistance (p = 0.1633). In a previous study [26], however, we analyzed for CHEK2 mutation status in relation to therapy outcome in a cohort of <span class="yellow">patients</span> from doxorubicin study [13]. In that study [26], we detected the previously identified mutation Ile157Thr. In addition, we detected a novel nonsense somatic mutation (1368InsA). This mutation was associated with lack of function in vitro; moreover, it was associated with drug resistance in vivo. Analyzing our material and this cohort [26] together, (n = 160), CHEK2 mutations (n = 5 in total) predicted for resistance to doxorubicin and epirubicin therapy (p = 0.0123). Even though, excluding <span class="yellow">patient</span> Epi203 (harboring TP53 Arg175His and Arg95Ter CHEK2 mutation) as well as other <span class="yellow">patients</span> harboring TP53 L2/L3 mutations (n = 129), CHEK2 mutations (n = 4 in total) predicted for resistance to doxorubicin and epirubicin therapy (p = 0.030).<br><br>TP53 and CHEK2 Mutations Combined and Response to Therapy
Assuming that TP53 and CHEK2 mutations may substitute for each other, we analyzed for the predictive effect of mutations in both genes. The occurrence of a mutation affecting either CHEK2 or TP53 strongly predicted therapy resistance (p = 0.0101; Fisher exact test). When tumors harboring TP53-L2/L3 mutations and CHEK2 mutations were compared with those wild-type or TP53 mutations outside the L2/L3 domain, the correlation was further strengthened (p = 0.0032; Fisher exact test). The significance was preserved when comparing <span class="yellow">patients</span> with a PD to objective responders (CR and PR) excluding <span class="yellow">patients</span> with stable disease (SD) from the statistical analysis (Table 4).<br><br>p14(ARF) Mutations and Promoter Methylations
Neither mutations nor polymorphisms in the coding region of p14(ARF) were observed among the 107 <span class="yellow">patients</span> analyzed. Likewise, no promoter methylations were detected.<br><br>Influence of CHEK2 and TP53 Mutation Status on Relapse-Free Survival
Because of the limited time of the follow-up, no formal statistical assessment of overall survival was performed. Details regarding outcome for individual <span class="yellow">patients</span> with mutations are described in Table 2 and 3 to make these data available to the reader. Relapse-free survival is depicted in (Figure 1). Figure 1A shows relapse-free survival for the <span class="yellow">patients</span> with TP53 and CHEK2 mutations (all mutations found) compared to <span class="yellow">patients</span> without any TP53 or CHEK2 mutations, no difference in relapse-free survival was observed. Similar, no difference was seen when grouping TP53 mutations outside L2/L3 and CHEK2 mutation not affecting kinase function (Ile364Thr) as wild-type (Figure 1B). Grouping tumors harboring a mutation in L2/L3 together with CHEK2 mutations affecting kinase domain (Arg95Ter) in one group, mutations outside TP53 L2/L3 and Ile364Thr as one group and tumors without any found mutations in TP53 and CHEK2 separately, again no noticeably difference in relapse-free survival were seen (Figure 1C). Notably, in addition to a short median follow-up time, a total of 35 <span class="yellow">patients</span> with a sub-optimal response to epirubicin received subsequent treatment with paclitaxel, which may have influenced the outcome.<br><br>CHEK2 Mutant's Capability to Form Dimers
To investigate whether the identified CHEK2 mutations affect the ability of the Chk2 protein to form dimers, co-transfection and immunopresipitation of V5-tagged mutants and Xpress-tagged wild-type Chk2 were performed using CHEK2 low-expressing U-2-OS cells. As we identified the previously characterized CHEK2 germline mutants variants Arg117His (n = 2 and Ile157Thr (n = 1) among <span class="yellow">patients</span> allocated to primary treatment with paclitaxel in our ongoing study, these mutants were evaluated together with Arg95Ter and Ile364Thr. The results presented in Figure 2 show that all Chk2 variants carrying a point mutation were able to form dimers with wild-type Chk2, whereas the Arg95Ter variant was not.<br><br>Kinase Activity of CHEK2 Mutants
To investigate whether the identified CHEK2 mutants retained the wild-type kinase activity, an in vitro Chk2 kinase assay with respect to Chk2 autophosphorylation and Cdc25 substrate phosphorylation was performed. The U-2-OS cells were preferred for this assay because they were previously found to express only low levels of endogenous Chk2 [44]. This was confirmed by us using an antibody recognizing endogenous protein (data not shown). These cells have previously been used by other investigators to study Chk2 kinase activity [44], [45], [46].
The two mutants Arg117Gly and Ile157Thr were previously tested for in vitro kinase activity [47], but were both included here, together with wild-type CHEK2 as controls. Compared to wild-type Chk2, the Ile157Thr mutant retained wild-type kinase activity. The mutant Ile364Thr showed partially reduced kinase activity both in term of Cdc25-phosphorylation and autophosphorylation (Figure 3). In contrast, the mutant Arg117Gly showed strongly reduced kinase activity while the Arg95Ter mutant was totally devoid of any Chk2 kinase activity. The activity recorded for Ile157Thr and Arg117Gly was consistent with previously reported results for these two mutants [47]. Notably, there was an internal consistency with respect to percentage activity reduction comparing individual mutants with respect to autophosphorylation and phosphorylation of Cdc25 (Figure 3).
Since enzymatically active Chk2 exists as dimers, it was important to determine the effect of Chk2 mutants on wild-type/mutant heterodimer kinase activity. The effect on Chk2 kinase activities (Chk2 autophosphorylation and Cdc25 substrate phosphorylation) of the individual mutants were therefore determined after co-transfection with wild-type Chk2 as described in Materials and Methods. The results from this co-transfection-kinase assay (Figure 4) were similar to those of the single-transfection assay (Figure 3) except in the case of the Arg117Gly mutant, which expressed a substantial kinase activity when complexed with wild-type Chk2. This is consistent with previous data indicating that the Arg117Gly mutant has neglectable kinase activity itself but dimerizes efficiently to Chk2 wild-type without strongly affecting the wild-type Chk2 activity. Hence, the activity detected is probably caused by the co-transfected and co-precipitated wild-type protein.
To rule out the possibility that endogenously expressed wild-type Chk2 contributed to observed Arg117Gly kinase activity shown in Figure 4, we compared the Arg117Gly variant activity in the presence or absence of co-transfected wild-type Chk2 to the activities of Arg95Ter under the same conditions. The Arg95Ter variant does not form dimers with wild-type Chk2. As seen in Figure 5, Arg117Gly, which forms dimers with Chk2 wild-type, allows increased activity when co-transfected with wild-type as compared to the corresponding activity for the Arg95Ter mutant. The fact that Arg117Gly, when transfected alone, displays very similar activity as Arg95Ter or negative control (background levels), strongly indicates that the contribution of endogenous Chk2, which, similarly to exogenously expressed wild-type Chk2 co-precipitate with Arg117Gly is non-significant.<br><br>Family Cancer Incidence in Relation to CHEK2 Germline Mutations
Following an initial report of a family with a CHEK2 germline mutation expressing an increased cancer incidence resembling the Li-Fraumeni syndrome [24], recent studies have revealed the more common CHEK2 mutations to be associated with a moderately increased risk of breast and colorectal cancers. We hypothesized that CHEK2 mutations having a detrimental effect on drug sensitivity could be associated with a more aggressive, Li-Fraumeni or a Li-Fraumeni-like (LFL) cancer syndrome [48]. Except from the <span class="yellow">patient</span> harboring the Ile364Thr mutation who did not have any known congestion of cancer disease in the family, a detailed assessment of family cancer history was performed for each <span class="yellow">patient</span> harboring a germline CHEK2 mutation. The family cancer pedigrees are depicted in Figure 6.
While <span class="yellow">patients</span> harboring CHEK2 germline mutations revealed different types of cancers (mainly breast and tumors of the gastrointestinal area) in their family, surprisingly, no distinct pattern discriminating families harboring the Arg95Ter mutation from the other CHEK2 mutated families could be identified. One of them (Epi203), who inherited the mutation from her father's side of the family, had no accumulation of either breast or colorectal cancer on that side. It should be noted, however, that two brothers of her fathers mother had prostate cancer, and two siblings of his father having hepatocellular carcinoma and bladder cancer, respectively), while the other expressed a disease pattern resembling what has been seen with the more common CHEK2 mutations, like del1100C [25].<br><br>
Discussion
TP53 plays a key role as a tumor suppressor gene. Its protein product activates processes such as growth arrest, DNA repair, apoptosis and/or senescence in response to genotoxic damage as well as oncogene activity [49], [50]. Despite being extensively studied, critical issues regarding regulation of the p53 protein remain poorly understood, and conflicting evidence obtained in different experimental systems make the clinical relevance of experimental data questionable.
Chemoresistance is the main obstacle to cancer cure in most malignancies, including breast cancer. Previously, we found TP53 mutations affecting the L2/L3 DNA binding domain to be associated with lack of responsiveness to doxorubicin monotherapy [13] as well as mitomycin and 5-fluoro-uracil in concert [14]. However, some tumors revealed therapy resistance despite harboring wild-type TP53. Postulating that these tumors may harbor genetic disturbances in genes playing a key role in the p53 pathway, we here sequenced TP53 along with CHEK2 and p14(ARF), the latter two known to play a critical role as p53 activators, in tumors from 109 <span class="yellow">patients</span> treated with epirubicin monotherapy. Our results confirm TP53 mutations, in particular those affecting the L2/L3 domains, to be associated with drug resistance. Most importantly, we also found CHEK2 mutations generating a non-functional protein in our in vitro assays to be associated with drug resistance. In contrast, none of our tumors harbored either mutations or expressed promoter hypermethylations affecting the p14.
Based on in vitro assays, we were able to classify the different Chk2 mutants with respect to dimerization capability as well as kinase activity (Chk2 autophosphorylation and Cdc25 substrate phosphorylation). In addition, the kinase activities of the Chk2 wild-type/mutant complexes were monitored in co-transfection experiments. Notably, each point mutation (except for Arg117Gly) revealed similar relative kinase efficacy whether co-transfected with wild-type Chk2 or not (Figure 3 and 4). Cells co-transfected with Arg117Gly and wild-type Chk2 revealed kinase activity, probably due to the contribution of the wild type protein in Chk2 mutant – wild-type heterodimers. In contrast, cells transfected with Arg95Ter revealed no kinase activity whether co-transfected with wild-type Chk2 or not, clearly distinguishing this mutation from the others (Figure 3 and 5).
All in vitro assays were based on transfection of the U-2-OS cell line, a cell line known to express wild-type Chk2 at low levels, and previously used by other investigators to study Chk2 activity [44], [45], [46]. Since we were not able to obtain satisfactory technical quality of the kinase assay in cell lines negative for Chk2 (HCT 15 and HCT 116), we assessed potential background kinase activity due to endogenous Chk2 by performing western blot analysis revealing the endogenous levels of Chk2 in U-2-OS cells to be non-significant compared to the exogenously expressed Chk2 levels (data not shown). We also performed a separate kinase assay, directly comparing the effect of binding partners for the dimerizing Arg117Gly and the non-dimerizing Arg95Ter. This assay also revealed the contribution of endogenous Chk2 to be non-significant (Figure 5).
Taking our in vitro findings together with in vivo observations, our present data confirm that the functionally defective CHEK2 Arg95Ter mutation, together with LOH, is associated with resistance to anthracycline therapy. In contrast, the <span class="yellow">patient</span> harboring the Ile364Thr mutation, moderately reducing phosphorylation activity, responded well to therapy. The other missense mutations; Arg117Gly and Ile157Thr were observed among <span class="yellow">patients</span> receiving paclitaxel therapy only; thus, their influence on anthracycline sensitivity in vivo could not be addressed. Yet, based on the finding that the Arg117Gly mutant expressed no intrinsic activity, but readily dimerized to the wild-type protein without abolishing its activity, we hypothesize that this mutation and, probably, other yet unidentified CHEK2 mutations with a similar lack of intrinsic kinase activity, may cause resistance to anthracycline therapy if combined with LOH in breast cancer.
Our present findings have two major implications. First, we confirm that mutations in genes encoding proteins located within the same functional pathway may substitute for each other with respect to drug sensitivity, revealing for the first time a functional pathway critical to chemotherapy response in vivo. Second, the identification of mutations in the CHEK2 but not in the p14(ARF) gene in resistant tumors suggests that Chk2 mediated phosphorylation of p53 is a critical event in executing anti-tumor effect as a response to DNA damaging agents in breast cancer. This adds to our understanding not only of the function of p53 but Chk2 as well. p53 undergoes phosphorylation at multiple sites by different kinases, including Chk2 [51]. While activation of the ATM leading to direct (Ser 15) and Chk2-mediated (Ser 20) phosphorylation of p53 is considered an important mechanism for triggering p53 activation in response to DNA damage [52], some reports suggest ATM [53] and even Chk2 [23] to be redundant to this function. Importantly, Chk2 has been shown capable of inducing ATM-independent apoptosis in vitro [21]. While Chk2 phosphorylates p53 at Ser 20, thereby stabilizing p53 by preventing MDM2 binding [19], Chk2 also phosphorylates p53 at six additional sites, including Ser 313 and Ser 314 located in the nuclear localization signal domain of p53 [51]. In addition, Chk2 phosphorylates other important targets like BRCA1, Cdc25A and Cdc25C involved in DNA repair, G1 and G2 arrest, respectively [54]. Despite the wide range of known Chk2 substrates relevant for DNA repair and cell cycle control, our present findings that CHEK2 mutations leading to non-functional Chk2 protein may substitute for p53 mutations strongly advocate a role for Chk2 with respect to drug sensitivity executed through p53 activation.
Notably, one of the tumors (Epi203) with the Arg95Ter CHEK2 mutation in addition harbored a somatic TP53 mutation, Arg175His, with allelic imbalance for the TP53 gene (Table 2). Importantly, among another four <span class="yellow">patients</span> in this study (Epi063, Epi071, Epi087, Epi153) and one <span class="yellow">patient</span> from our previous doxorubicin protocol [13] harboring the Arg175His mutation together with allelic imbalance for TP53, all five of these <span class="yellow">patients</span> responded to anthracycline therapy either with a partial response or stable disease. In contrast, Epi132 and the only <span class="yellow">patient</span> for whom we previously identified a non-functional CHEK2 mutation (1368InsA; coding for a non-functional protein translate with cytoplasmic location [26]) expressed resistance to epirubicin and doxorubicin, respectively. Arg175His is a p53 “hot-spot” structural mutation reported to have defects with respect to transcriptional activation and also to negatively interact with wild-type p53 [55]. While this mutation has been shown to enhance chemoresistance upon transfection into p53 null Saos-2 cells [56], these osteosarcoma-derived cells may not necessarily be representative for breast cancers in vivo. Recent evidence strongly support p53 to be involved also in non-transcriptional mediated apoptosis by interacting with the Bcl-2/Bax system [57], and transcription-defect structural p53 mutants have been shown to execute non-transcriptional apoptosis in experimental systems [58]. Concomitant inactivation of Chk2 and p53 in breast cancer has been recorded by others [59], and the finding that a somatic mutation may generate a “growth advantage” in tumor cells already harboring a germline CHEK2 mutation may not implicate an effect on drug sensitivity in tumors not yet exposed to cytotoxic compounds. Rather, it may indicate a growth advantage, probably related to loss of p21 function. Notably, in a previous study we found the p21 polymorphism G251A to be associated with an increased risk of developing large breast cancers but to have no effect on drug sensitivity [60], indicating that growth rate and drug resistance may be regulated independently. Taken together, we believe our findings advocate a role for Chk2 in executing cellular response to anthracycline-induced DNA damage.
As mentioned above, removing TP53 mutated tumors including the double-mutated Epi203 from statistical analysis, CHEK2 mutation status still predicted for resistance to anthracycline therapy. In addition, removing the tumors harboring the Arg175His mutation from the p53 “L2/L3” group strengthened the correlation to lack of treatment response to epirubicin (p = 0.0005).
Comparing the effects of mutations in the CHEK2 gene to TP53 mutations indirectly underlines the importance of the role of Chk2 to chemoresistance. Our present findings as well as results from our previous studies [13], [14] revealed that about 50% of the <span class="yellow">patients</span> with tumors harboring TP53 L2/L3 mutations to be non-responders to primary therapy. In contrast, all our three <span class="yellow">patients</span> harboring a non-functional CHEK2 mutation (the two Arg95Ter mutated <span class="yellow">patients</span> here and our previous <span class="yellow">patient</span> harboring the 1368InsA) expressed primary resistance to therapy. We previously hypothesized that therapy response in tumors harboring TP53 L2/L3 mutations could be due to redundant pathways acting in concert [3]. Although no definite conclusion should be drawn from a limited number of observation, the fact that Chk2 not only phosphorylates p53 but also phosphorylates other substrates such as Cdc25A and Cdc25C [54] and E2F1 in response to etoposide-induced DNA damage [61] may indicate that inactivation of redundant pathways could take place in parallel.
The literature remains inconsistent with respect to whether the border amino acids 163, 195, 236 and 251 should be included in the p53 L2 and L3 domains [12]. Taking a conservative approach, we classified <span class="yellow">patient</span> Epi56, harboring a mutation in codon 163, as a L2/L3 mutant. The <span class="yellow">patient</span> harboring this mutation responded to therapy (PR). If this mutation was classified as outside the L2 domain, our p-value had been strengthened from p = 0.0136 to p = 0.0096.
Germline mutations in TP53 cause the Li-Fraumeni and Li-Fraumeni-like cancer disposition syndromes. However, while the germline and somatic mutations associated with these syndromes reveal a preference for the same codons [48], TP53 mutations affecting the DNA-binding domains seem associated with a poor prognosis [62], [63], [64] and, in particular, drug resistance [14], [40] in breast cancer. Thus, tumor suppression and tumor cell response to chemotherapeutics may involve different parts of p53 protein function. Following an initial report identifying a CHEK2 mutation in a family expressing characteristics of the Li-Fraumeni syndrome [65], recent evidence has linked CHEK2 founder mutations to a moderately increased risk of breast- and colorectal cancers with some additional disposition for other malignancies as well [66]. However, cancer incidence and phenotypes did not reveal an aggressive Li-Fraumeni or Li-Fraumeni-like tumor pattern. Similar to the two <span class="yellow">patients</span> in our paclitaxel treatment arm harboring the rare but previously characterized mutation Arg117Gly and the <span class="yellow">patient</span> with the Ile157Thr mutation, they expressed a moderately increased risk of breast and gastrointestinal cancers (Fig. 6). Thus, CHEK2 resembles TP53 in as much as there seems to be no direct correlation between effects of individual mutations with respect to tumor suppression and drug resistance.
Our finding that TP53 mutations located to the DNA-binding domains predicts drug resistance may indicate transcriptional mechanisms to be involved in drug-induced cell death. p53 induced apoptosis has been associated with transcriptional induction of genes including Puma and Noxa as well as Bax in experimental systems [55], [67], [68]. Yet, recent evidence has revealed p53 to induce apoptosis through non-transcriptional mechanisms by direct protein interactions with members of the Bcl-2/Bax system and mitochondrial release of cytochrom c [57], [69]. In deed, there is evidence that the DNA-binding domains, in particular the L3 part of the protein, may be critical also to transcriptional-independent apoptosis [70]. Of particular note is the finding that Chk2 may regulate transcriptional-independent p53-mediated apoptosis in response to DNA-damage created through ionizing irradiation [71]. Interestingly, Krajewski et al [72] reported low expression of Bax assessed by immunostaining to be associated with a low response to chemotherapy in metastatic breast cancer. Although no conclusion should be drawn at this stage, together these findings are consistent with the challenging hypothesis that transcription-independent activation of Bax following Chk2-phosphorylation may represent a key pathway in p53 dependent cell death in breast cancer in vivo.
p14 acts by releasing p53 from MDM2 binding, and has been related to oncogene-induced p53 activation [73]. Recently, p14 was shown to affect p53 by additional mechanisms, including acetylations [74], response to ionizing radiation in <span class="yellow">human</span> fibroblasts [75], and tumor-suppression following ionizing radiation in <span class="yellow">mice</span> [76], [77]. These findings further links the retinoblastoma and p53 pathways [28]. As such, we believe the negative finding with respect to its role in chemoresistance adds important information.
Contrasting earlier findings by us and others [15], a recent study revealed TP53 mutations to be associated with increased likelihood of having a complete response to chemotherapy [16]. These results may not necessarily be at conflict. In the latter study, <span class="yellow">patients</span> received treatment with a “dose-dense” chemotherapy regimen; if confirmed, the combined data may outline a therapeutic indication for aggressive dose-dense therapy based on tumor TP53/CHEK2 status.
So far attempts to identify single markers and, more recently, gene expression arrays predicting chemoresistance have not proved successful (see refs in [9], [10]). The findings presented here reveal for the first time defects in a functional gene cascade to be associated with drug resistance in a <span class="yellow">human</span> cancer in vivo. Moreover, the findings are made in breast cancer, the most frequent malignant disease among <span class="yellow">women</span> in the industrialized world, and relate to resistance to anthracyclines, the type of cytotoxic compounds most frequently employed for this malignancy.
While the only study we are aware of comparing TP53 mutation status in primaries and their distant metastases suggested an increasing fraction of tumors to express mutated TP53 during progression [78], we do not know the potential contribution of either TP53 or CHEK2 mutations to drug resistance in micrometastases or in metastatic disease. Yet the finding that one of our non-functional CHEK2 mutations associated with chemoresistance (1368InsA) occurred as a somatic, not germline mutation, suggest such mutations may be selected for during tumor progression. We propose the findings presented here provide important beacons identifying a functional pathway [3] likely to be disturbed through different mechanisms in relation to therapy resistance in advanced disease.
In conclusion, we believe our findings here that mutations in the TP53 and CHEK2 genes each may cause resistance to anthracycline therapy in primary tumors to have wide implications to future research in this area. While results from experimental systems are mandatory generating hypotheses, conflicting data from in vitro studies underlines the pivotal role of identifying defects associated with therapy resistance in vivo. Either through mutations of the genes themselves, or inactivation of this functional cascade through co-factors, we believe identification of the Chk2 – p53 axis as critical to anthracycline therapy response provides a functional clue for further investigations in this area.<br><br>
<h3>pmcA2562362</h3>Host immunity in the protective response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span>
Abstract
Background
We performed initial cell, cytokine and complement depletion studies to investigate the possible role of these effectors in response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span> in a susceptible BALB/c <span class="blue">mouse</span> model of infection.<br><br>Results
While protection with heat-killed bacilli did not result in sterilizing immunity, limited protection was afforded against an otherwise lethal infection and provided insight into potential host protective mechanisms. Our results demonstrated that <span class="yellow">mice</span> depleted of either B cells, TNF-α or IFN-γ exhibited decreased survival rates, indicating a role for these effectors in obtaining partial protection from a lethal challenge by the intraperitoneal route. Additionally, complement depletion had no effect on immunoglobulin production when compared to non-complement depleted controls infected intranasally.<br><br>Conclusion
The data provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="blue">mouse</span> model. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br><br><br>Background
<span class="yellow">Burkholderia mallei</span>, the etiologic agent of glanders, is a gram-negative, capsulated, non-motile, facultative intracelluar bacterium. Most known members of the Burkholderiaceae are resident in the soil; however, <span class="yellow">B. mallei</span> is thought to be an obligate mammalian pathogen. <span class="yellow">Horses</span> are highly susceptible to infection and considered the natural reservoir for infection, although <span class="yellow">mules</span> and <span class="yellow">donkeys</span> are also susceptible [1]. Clinically, glanders in solipeds can present as either a chronic (<span class="yellow">horses</span>) or acute (<span class="yellow">mules</span> and <span class="yellow">donkeys</span>) form. Naturally acquired <span class="yellow">human</span> infection with <span class="yellow">B. mallei</span>, although not seen in the United States since 1945, has occurred rarely and sporadically among laboratory workers and those in direct contact with infected animals [2]. However, glanders is endemic among domestic animals in Africa, Asia, the Middle East, and Central and South America. The course of infection is dependent on the route of exposure. Direct contact with the skin can lead to a systemic infection. Inhalation of aerosol or dust containing <span class="yellow">B. mallei</span> can lead to septicemic, pulmonary, or chronic infections of the muscle, liver and spleen. The disease has a 95% case fatality rate for untreated septicemic infections and a 50% case fatality rate in antibiotic-treated individuals [3].
There is no <span class="yellow">human</span> or animal vaccine available for glanders, and development of a partial or fully protective adaptive host response to the organism has not been well-defined. Previous studies with <span class="yellow">B. mallei</span> and the host response have shown that a mixed immune response consisting of both Th1 and Th2-associated cytokines with a predominant IgG1 subclass does not correlate with protection [4]. Additional studies with passive transfer of monoclonal antibodies specific for <span class="yellow">B. mallei</span> have correlated with early protection from infection [5]. Recent studies have also shown the Th1 cytokine IL-12 to mediate partial protection to non-viable <span class="yellow">B. mallei</span>-vaccinated <span class="yellow">mice</span> [6]. Thus, full correlates of protection mediated by the adaptive immune system against <span class="yellow">B. mallei</span> remain to be fully elucidated.
In this series of studies, we sought to address the impact of depletion of the major effector lymphoid cell populations (B220+ B cells, CD4+ or CD8+ T cells) and key pro-inflammatory/Type 1 cytokines (IFN-γ or TNF-α) on survival in BALB/c <span class="yellow">mice</span> vaccinated with heat killed (HK) bacilli followed by an intraperitoneal (i.p.) challenge with live organism. In addition, studies investigating the effect of complement on opsonization of organism and antibody production were assessed. Heat killed bacteria were used as a model of vaccination to allow evaluation of <span class="yellow">B. mallei</span> specific immune responses. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br>Results
Heat-killed <span class="yellow">B. mallei</span> vaccination mediates partial protection from lethal challenge
To begin to address this issue in an animal model of acute infection, we established that immunologically naive BALB/c <span class="yellow">mice</span> challenged i.p. with 2 × 107 CFU resulted in death by day 4–6, while i.p. immunization with 1 × 105 heat killed (HK) bacteria provided partial protection against a subsequent challenge. Two independent experiments resulted in similar findings of 40% survival for HK-vaccinated <span class="yellow">mice</span> with a mean survival time (MST) of 8 days versus 4 days in naïve <span class="yellow">mice</span> (Fig. 1). The administration of vaccines for <span class="yellow">B. mallei</span> during an outbreak would mandate relatively rapid onset of protection for <span class="yellow">human</span> or veterinary use. Based on non-routine use and vaccine implementation in the course of an outbreak, a 14 day window was chosen for assessment of protection. Our results indicate that HK vaccination can afford partial protection to an otherwise lethal challenge of <span class="yellow">B. mallei</span> by the i.p. route.<br><br>Effects of cell depletion on HK-vaccinated survival
To dissect the cellular basis for protection mediated by HK vaccination, 13 days after immunization with HK bacteria (day -1), and at day of challenge, <span class="yellow">mice</span> were dosed with antibodies to deplete CD4+, CD8+ or B220+ cells. Antibody depletion of CD4+, CD8+, or B220+ cells in these <span class="yellow">mice</span> was confirmed by flow cytometric analysis with depletion efficiencies for CD4, CD8, and B220 populations at 99.7%, 96%, and 95%, respectively, relative to <span class="yellow">mice</span> treated with isotype control monoclonal antibodies (data not shown). Our results demonstrated decreased survival rates in B220 (p = 0.3418), CD4+ (p = 0.5417) and CD8+ (p = 0.4684) antibody depleted <span class="yellow">mice</span>, compared to isotype control antibody, a finding that indicated a possible role for vaccine induced antibody production. When challenged with 2 × 107 CFU/<span class="blue">mouse</span> by the i.p. route, loss of T cells resulted in reduced survival (50%) relative to the non-specific isotype control (Fig. 2). In contrast to the loss of T cells, depletion of B220+ cells resulted in 100% mortality relative to the non-specific isotype control (Fig. 2). To further evaluate the necessity of these effector cells in providing protection following HK vaccination, relatively resistant C57BL/6 <span class="yellow">mice</span>, deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) were subjected to an identical HK vaccination and challenge regimen. Mature, B-cell-deficient <span class="yellow">mice</span> demonstrated a 50% decreased survival (p = 0.0888) compared to the wild-type <span class="yellow">mice</span> with an MST of 35.5 days (Fig. 3). CD4-/- and CD8-/- <span class="yellow">mice</span> exhibited a 60% (p = 0.1343) and 0% reduced survival, respectively (Fig. 3).<br><br>Effects of cytokine depletion on HK vaccination
Similar studies were performed to determine the role of IFN-γ or TNF-α in acute infection in BALB/c <span class="yellow">mice</span> immunized with HK bacteria. Six hours before challenge, <span class="yellow">mice</span> were dosed with antibodies that neutralize IFN-γ or TNF-α. Individual depletion of either TNF-α (p = 0.0145) or IFN-γ (p = 0.0446) resulted in 100% mortality with an MST of 3 and 2 days, respectively, compared to the HK-vaccinated isotype control <span class="yellow">mice</span> (Fig. 4). In contrast, 40% of HK-vaccinated, isotype control <span class="yellow">mice</span> survived to at least 12 days post-challenge (Fig 4). To further evaluate the host TNF-α response during an established <span class="yellow">B. mallei</span> chronic infection, we infected 12 BALB/c <span class="yellow">mice</span> by the i.p. route with 1 × 106 CFU <span class="yellow">B. mallei</span>. One animal was terminally ill on day 37 post-infection. On day 42 post-infection, the remaining 11 <span class="yellow">mice</span> were dosed with either anti-TNF-α (n = 6), or control mAb (AFRC Mac 49) (n = 5). No further deaths were observed in the control mAb-treated <span class="yellow">mice</span>. Rapid mortality was observed in the anti-TNF-α-treated group, with all <span class="yellow">mice</span> dying within 7 days of treatment (p = 0.0023) relative to the isotype-treated controls (Fig. 5).<br><br>J774A.1 uptake of serum treated <span class="yellow">B. mallei</span>
Complement mediated uptake assays were performed to evaluate opsonization. Results indicated enhanced bacterial uptake in J774A.1 phagocytes inoculated with serum treated <span class="yellow">B. mallei</span> (p = .0082), compared to <span class="yellow">B. mallei</span> alone, while heat-inactivated serum produced uptake percentages similar to those prior to serum addition (Fig. 6). Taken together, these results imply an active role for complement components in the uptake of organism by macrophages.<br><br>Immunoglobulin production in HK vaccinated BALB/c <span class="yellow">mice</span>
We further characterized the ability of HK vaccination to induce a predominant IgG isotype by determining IgG2a/IgG1 ratios in i.p. and i.n. vaccinated BALB/c <span class="yellow">mice</span>. Pre (day 14 post vaccination) and post (day 2 post infection) exposure serum samples were obtained and evaluated for IgG isotype concentrations (Table 1). No appreciable differences in IgG pre-exposure levels were seen when comparing i.n. to i.p. vaccination. In addition, cobra venom factor-treated animals showed no significant differences to non-cobra venom factor-treated animals in IgG pre-exposure (challenge) levels. Conversely, isotype switching in the cobra venom factor treated animals was enhanced in post-exposure serum IgG2a (Table 1).<br><br>
Discussion
Recent studies have shown a key role in protection from lethal challenge for IFN-γ in non-vaccinated <span class="yellow">mice</span> from either NK and/or NKT cells following experimental exposure to <span class="yellow">B. mallei</span> and <span class="yellow">B. pseudomallei</span> [7,8]. A similar protective role in the innate response to infection has been demonstrated for TNF-α in <span class="yellow">B. pseudomallei</span> infection [8]. The studies presented here are consistent with the essential role of these factors in the relative levels of protection conferred by vaccination with heat-killed <span class="yellow">B. pseudomallei</span> and would appear to be viable early markers for protection from lethal acute infection [9]. Currently, there are no fully protective vaccines against <span class="yellow">B. mallei</span> or <span class="yellow">B. pseudomallei</span> in a <span class="yellow">murine</span> model, particularly for the sensitive BALB/c versus C57BL6 models. Previous studies have also demonstrated that both the humoral and cell-mediated arms are essential for protection from <span class="yellow">B. pseudomallei</span> infection [10]. Thus, loss or reduction of TNF-α and IFN-γ levels result in significantly reduced survival rates, substantiating previous reports of the role of these factors in protection against <span class="yellow">B. mallei</span> [7]. Moreover, we demonstrate a role for sustained TNF-α production in the maintenance of host survival throughout the course of <span class="yellow">B. mallei</span> infection. <span class="yellow">Mice</span> with an established <span class="yellow">B. mallei</span> chronic infection rapidly lost the ability to control the growth of the bacillus upon neutralization of TNF-α. This would suggest a potential role for TNF-α in the maintenance of productive granulomas which may limit the spread of bacteria in chronically infected hosts, or, alternatively, in direct or indirect microbicidal or bacteriostatic activities at the sites of infection. Additional studies are underway to determine more precisely the role of TNF-α in host protection to <span class="yellow">B. mallei</span>.
Multiple innate and adaptive cell types may contribute to the production of IFN-γ in response to infection with <span class="yellow">B. mallei</span> following vaccination. Our results with individual depletion of CD4+ and CD8+ T cells suggests that both cell types may compensate for the functional loss of the other effector cell type in the production of this key cytokine. The effector role for IFN-γ in mediating protection against <span class="yellow">B. mallei</span> may include both immunoregulatory and non-regulatory functions. Regardless, the requirement of IFN-γ, as demonstrated by administration of neutralizing antibody prior to infection, indicates that stimulation of IFN-γ response is a desirable goal for a <span class="yellow">B. mallei</span> vaccine.
Similarly, B220-positive cells appear to play a role in protection following vaccination with heat-killed <span class="yellow">B. mallei</span>. Interestingly, this protective immunity, occurring in other intracellular pathogens, is not exclusively dependent on B cells [11]. Passive protection has been demonstrated against acute Burkholderia infection by monoclonal antibodies [5,12]. Protection against <span class="yellow">B. pseudomallei</span> infection by anti-LPS, capsular polysaccharide and proteins has been short-lived, suggesting that antibody production offers limited protection in the initial stages of infection by an as-yet-undefined mechanism [12]. We have shown that following depletion of B220+ cells, survival rates decreased as much as 100% relative to non-depleted controls and individual CD4/CD8-depleted <span class="yellow">mice</span> via the intraperitoneal route. Results from C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) substantiate the requirement for B-cell involvement by evidence of μMT and CD4-/- decreased survival. The lack of an effective CTL response to vaccination did not appear to alter survival in what would appear to be a CD4/B-cell (humoral)-driven response. In CD4-deficient <span class="yellow">mice</span>, we have the additional potential variable that a CD4-dependent antibody response might also be inhibited during the vaccination phase relative to <span class="yellow">mice</span> treated with antibody immediately prior to and during the early phases of infection. Although not statistically significant, we did observe a decrease in survival in μMT (mature B cell) deficient <span class="yellow">mice</span> as early as day 9 post challenge, whereas CD4-deficient <span class="yellow">mice</span> produced similar results at day 32 post challenge, indicating a role for B cells independent of CD4 T cell help, perhaps through a T-independent mechanism of antibody production. Although CD8-/- C57BL/6 demonstrated no decreased survival in our HK-vaccinated model, a lack of potential endogenous protein production by HK <span class="yellow">B. mallei</span> may have contributed to limited MHC-I presentation.
Complement associated studies revealed increased J774A.1 uptake of serum-treated <span class="yellow">B. mallei</span>. Complement-mediated uptake studies of <span class="yellow">B. pseudomallei</span> by polymorphonuclear leukocytes (PMNs) suggest that capsule production contributes to resistance of phagocytosis by reducing C3b bacterial deposition [13]. Previous studies have demonstrated that a polysaccharide capsule is present in <span class="yellow">B. mallei</span>, [14,15] although in the present study enhanced uptake with serum-treated <span class="yellow">B. mallei</span> was observed. Intracellular survival assays of complement mediated uptake of organisms were not performed in the present study, thus, the role of complement opsonization on intracellular survival is not fully resolved. Previous reports have demonstrated the ability of <span class="yellow">B. mallei</span> to survive within macrophage without the aid of serum coating organisms [16]. Conversely, the idea of antibody mediated opsonization to facilitate macrophage activation and clearance of intracellular organisms may offer support to the role of B cells in an effective immune response. A possible protective mechanism may include HK vaccination induced production of opsonizing antibodies which may aid in complement mediated uptake, thereby limiting the initial bacterial threshold below a lethal level.
Immunoglobulin responses to HK vaccination resulted in modest levels of IgG1 following 2 weeks post vaccination, while post-exposure levels were indicative of efficient class switching to a favorable IgG2a isotype. Importantly, cobra venom factor treatment of animals at time of vaccination did not alter their ability to produce immunoglobulin. In fact, cobra venom factor treated animals resulted in higher IgG2a levels when compared to non-treated. Complement activation can modulate both the primary and secondary immune responses and has been shown to enhance secondary immune responses to vaccination [17]. The current results suggest that cobra venom factor treatment may affect the modulation of the immune response to <span class="yellow">B. mallei</span> infection through B cell activation and/or memory B cell generation.<br><br>Conclusion
In summary, our results provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="blue">mouse</span> model. Understanding and defining the role of B cells in adaptive <span class="yellow">B. mallei</span> immunity will likely be fundamental to the design of an efficacious vaccine and important goals of future research.<br><br>Methods
Bacterial strain and <span class="yellow">mice</span>
<span class="yellow">B. mallei</span> strain ATCC 23344 (China 7) was cultured on Luria-Bertani agar supplemented with 4% glycerol (LB+4%G) agar plates for 48 h at 37°C. Isolated colonies were sub-cultured to LB+4%G broth, and cultures were incubated at 37°C until optical density readings at 600 nm (OD600) reached an exponential phase of growth. Bacteria were pelleted by centrifugation, washed and re-suspended in sterile 1× phosphate-buffered saline (PBS, pH 7.4) to obtain the desired CFU/ml. To obtain HK inoculums, bacterial suspensions were incubated at 85°C for 3 h and stored at 4°C until use. The absence of live <span class="yellow">B. mallei</span> organisms in the HK preparations was confirmed after plating 10% of the total inoculums (v/v) and incubating these at 37°C for 48 h. All procedures were performed under a class II biosafety cabinet in a biosafety level 3 laboratory. Female, 6- to 8-week-old, BALB/c <span class="yellow">mice</span> (n = 5–7) were obtained from Harlan Sprague Dawley, Inc. (Indianapolis, Indiana). Female, 6- to 8-week-old, C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) and CD8 T-cells (CD8-/-) and wild-type <span class="yellow">mice</span> were obtained from The Jackson Laboratory (Bar Harbor, Maine).<br><br>Vaccination and challenge
BALB/c and C57BL/6 <span class="yellow">mice</span> were grouped and vaccinated with 0.5 μg of HK <span class="yellow">B. mallei</span> (without adjuvant) by i.p. injection using a 25-gauge syringe. Two weeks post HK vaccination <span class="yellow">mice</span> were injected i.p. with 2 × 107 CFU/100 μl of live <span class="yellow">B. mallei</span> (~20 LD50) [18]. Complement depleted animals were challenged with 2.5 × 104 CFU/50 μl (~0.25 LD50) by intranasal (i.n.) route. Aliquots from the inoculums were plated to confirm the infecting dose. All procedures and animal protocols used in this study were approved by the Biosafety and IACUC committees at UTMB and conducted in either BSL-3 or ABSL-3 laboratories.<br><br>Cell and cytokine depletions
Acute in vivo cell/cytokine depletion was performed with monoclonal <span class="yellow">rat</span> anti-<span class="blue">mouse</span> CD4 (GK1.5), CD8α (53-6.7) or B220 (RA3-6B2) obtained from R&D Systems, Inc. (Minneapolis, MN) by methods similar to those we have previously described [19]. Functional grade purified <span class="yellow">rat</span> anti-<span class="blue">mouse</span> interferon-gamma (IFN-γ, AN-18) was obtained from eBioscience (San Diego, CA) and purified anti-<span class="blue">mouse</span> tumor necrosis factor (TNF-α, MP6-XT3) from BD Pharmingen (San Diego, CA). IFN-γ and TNF-α antibodies were injected i.p. 6 h prior to challenge, 200 μg per <span class="blue">mouse</span> in 200 μl PBS or at later time points as indicated. <span class="yellow">Rat</span> IgG isotype control was obtained from Southern Biotech (Birmingham, AL) and administered i.p. on day of challenge, 200 μg/<span class="blue">mouse</span>. <span class="yellow">Rat</span> anti-<span class="blue">mouse</span> CD4, CD8α and B220 were injected i.p. twice, 1 day prior to challenge and on day of challenge, with an equivalent dosage sufficient to deplete T or B cells from 6 × 108 bone marrow cells per injection. The efficiency of depletion at time of infection for CD4+, CD8+, and B220+ cells was confirmed by flow cytometry analysis immediately prior to infection.<br><br>Complement depletion with cobra venom factor
<span class="yellow">Mice</span>, six to seven per group, were vaccinated i.p. with 1 × 105 CFU of nonviable <span class="yellow">B. mallei</span> cell preparation in a total volume of 0.1 ml. Two weeks later, 24 h and 1 h before challenge, complement depleted <span class="yellow">mice</span> were treated i.p. with 12.5 units total cobra venom factor (Quidel Corporation Speciality Products, San Diego, CA) in 0.1 ml of PBS. Complement depletion was confirmed prior to challenge by micro-titer hemolytic complement activity (CH50) assay as previously described [20].<br><br><span class="yellow">B. mallei</span> J774A.1 uptake assays
J774A.1 cells were seeded (5 × 105) onto Corning costar 24 well plates (Corning, NY) with DMEM and incubated overnight at 37°C with 5% CO2. Bacterial suspensions were incubated at 37°C for 45 minutes supplemented with 2% <span class="blue">mouse</span> serum from Sigma-Aldrich (St. Louis, MO.), heat inactivated <span class="blue">mouse</span> serum (56°C 30 minutes), or bacteria alone and then added at an MOI of 10:1 to J774A.1 cells in triplicate. Inoculated wells were centrifuged at 800 g for 2 minutes and incubated for 2 hours at 37°C with 5% CO2 followed by a PBS wash (×2) and 2 hour incubation with 250 μg/ml kanamycin. Wells were washed twice with PBS and lysed with 0.1% Triton X-100, followed by serial 10-fold dilutions plated on LBG plates and incubated at 37°C for 2 days. Colony forming units were enumerated and uptake expressed as a percentage of initial inoculating dose ± SEM.<br><br>Antibodies and flow cytometry
Flow cytometric analysis was performed on 0.1-ml blood samples transferred to micro centrifuge tubes containing 90 μl of acid citrate dextrose (ACD) solution. Red blood cells were lysed using ACK-lysing buffer (Biosource International, Inc., Camarillo, CA) according to the manufacturer's instruction. Antibodies used for analysis of surface markers included: FITC-conjugated <span class="yellow">rat</span> anti-<span class="blue">mouse</span> CD45R/B220 (RA3-6B2, BD Pharmingen San Diego, CA) for B cells; FITC-conjugated <span class="yellow">rat</span> anti-<span class="blue">mouse</span> CD8α (53-6.7) and CD4 (GK1.5, BD Pharmingen, San Diego, CA) for CD8+ or CD4+ cells, respectively. Samples evaluated for CD4+ and CD8α+ cells were also incubated with biotin-conjugated <span class="yellow">hamster</span> anti-<span class="blue">mouse</span> CD3e (145-2C11) monoclonal antibody (BD Pharmingen, San Diego, CA) and subsequently with streptavidin APC Cy7. Isotype-matched, non-specific controls were assayed in parallel (BD Pharmingen, San Diego, CA). Surface staining was performed according to previously published protocols [21]. Following cell staining, the samples were fixed with 2% buffered paraformaldehyde overnight prior to analysis by flow cytometry. Samples were analyzed using a FACSCalibur flow cytometer with BD CellQuest Pro software.<br><br>Antibody assays
Immunoglobulin subclass IgG1 and IgG2a titers in <span class="yellow">mice</span> were determined by a whole bacterial cell ELISA performed in 96-well, Immulon 2 HB, round-bottom plates (Dynex Technologies). <span class="yellow">B. mallei</span> antigen was diluted in 0.1 M carbonate buffer (pH 9.5) and 50 μl of diluted cells placed into wells. Plates were stored overnight at 4°C. The plates were washed with washing solution (1× PBS, 0.05% Tween 20), and incubated with 100 μl of blocking solution (1× PBS, 1% <span class="yellow">bovine</span> serum albumin, 0.05% Tween 20) for 1 h at 37°C. Dilutions of <span class="blue">mouse</span> sera were made with blocking solution in duplicate and plates were incubated for 1 h at 37°C. Following incubation, plates were washed and 50 μl of anti-Ig-<span class="yellow">horseradish</span> peroxidase subclass conjugate, diluted accordingly to manufacturer's instructions (Southern Biotechnology Associates, Inc. Birmingham, Ala.), was added to each well and incubated for 1 h at 37°C. After washing, 50 μl of 2,2'-azino-di-(3-ethylbenzthizoline)-6-sulfonate (ABTS) peroxidase substrate (KPL, Inc., Gaithersburg, Maryland) was added to each well and plates incubated for 25 min at room temperature. The amount of bound antibody was determined colorimetrically by absorbance at 405 nm.<br><br>Statistical analysis
Survival curves were calculated by Kaplan Meier survival analysis with log-rank tests between groups using GraphPad Prism (V.4.03 for windows). Statistical analysis was generally performed with the paired Student's t-test. P value ≤ 0.05 was considered significant.<br><br>
Abbreviations
HK: Heat-killed; i.p.: intraperitoneal; i.n.: intranasal.<br><br>Authors' contributions
GCW designed and conducted experiments and drafted the manuscript. BMJ carried out the immunoassays and animal work. SP provided analysis of data and contributed to design and animal work. RAL participated in the generation and analysis of chronic TNF-α data. DME conceived the study, and participated in its design and coordination and helped to draft the manuscript. AGT participated in the bacterial work and drafting of the manuscript. All authors read and approved the final manuscript.<br><br>
</body></html>