
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Murine</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA147033</h3>Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the ribonuclease A gene family.
Abstract
We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophil-associated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this 'mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing.
� 1997 Oxford University Press <br><br> Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> 4235�4239 <br><br> Molecular cloning of four novel <span class="yellow">murine</span> ribonuclease genes: unusual expansion within the Ribonuclease A gene family Dean Batten+, Kimberly D. Dyer, Joseph B. Domachowske� and Helene F. Rosenberg* The Laboratory of Host Defenses, Building 10, Room 11N104, National Institute of Allergy and Infectious Diseases, National Institutes of Health, 9000 Rockville Pike, Bethesda, MD 20892, USA Received August 4, 1997; Revised and Accepted September 17, 1997 DDBJ/EMBL/GenBank accession numbers: U72031, U72032, AF017258�AF017261 <br><br> ABSTRACT We have characterized four novel <span class="yellow">murine</span> ribonuclease genes that, together with the <span class="yellow">murine</span> eosinophilassociated ribonucleases 1 and 2, form a distinct and unusual cluster within the RNase A gene superfamily. Three of these genes (mR-3, mR-4, mR-5) include complete open reading frames, encoding ribonucleases with eight cysteines and appropriately spaced histidines (His11 and His124) and lysine (Lys35) that are characteristic of this enlarging protein family; the fourth sequence encodes a non-functional pseudogene (mR-6P). Although the amino acid sequence similarities among these <span class="yellow">murine</span> ribonucleases varies from 60 to 94%, they form a unique cluster, as each sequence is found to be more closely related to another of this group than to either <span class="yellow">murine</span> angiogenin or to <span class="yellow">murine</span> pancreatic ribonuclease. Interestingly, the relationship between the six genes in this `mR cluster' and the defined lineages of the RNase A gene family could not be determined by amino acid sequence homology, suggesting the possibility that there are one or more additional ribonuclease lineages that have yet to be defined. Although the nature of the evolutionary constraints promoting this unusual expansion and diversification remain unclear, the implications with respect to function are intriguing. INTRODUCTION Ribonuclease A (RNase A, <span class="yellow">bovine</span> pancreatic ribonuclease) is the prototype of an enlarging family of proteins defined by distinctive elements of conserved primary structure and enzymatic activity (1,2). The features shared by all members of the RNase A family include six to eight cysteines and specific histidine and lysine residues that form the ribonuclease active site. To date, several distinct lineages within the RNase A superfamily have been identified. Pancreatic ribonucleases (RNases 1) have been isolated from an extensive array of mammalian species (3,4), and mRNA encoding <span class="yellow">human</span> pancreatic ribonuclease has been detected in <br><br> numerous somatic tissues in addition to pancreas (5). RNases 2 and 3, eosinophil-derived neurotoxin (EDN) and eosinophil cationic protein (ECP), respectively, have been characterized primarily with respect to eosinophil function (6,7), although expression of EDN [also known as RNase Us (8) or <span class="yellow">human</span> liver ribonuclease (9)] is also widespread (5). Angiogenin (RNase 5), a structurally atypical member of this family, was originally identified as an agent promoting blood vessel growth and development (10,11). RNase 4 (12,13) and RNase k6 (14) are also members of the RNase A superfamily, although their functions are currently obscure. Recently, Larson and colleagues (15) described cDNAs encoding two highly homologous <span class="yellow">murine</span> ribonucleases, the <span class="yellow">murine</span> eosinophil-associated ribonucleases (mEAR) 1 and 2, which were cloned and identified via tryptic peptides isolated from <span class="yellow">murine</span> eosinophil proteins. Although Southern analysis demonstrated that multiple copies of sequence homologous to mEAR-1 and mEAR-2 were present in <span class="yellow">murine</span> genomic DNA, the precise nature of these copies--as pseudogenes, polymorphisms, or distinct functional genes--was unclear. In this work, we have identified four of these homologous sequences, three encoding novel ribonucleases, and one encoding a pseudogene. Together with mEAR-1 and mEAR-2, these six ribonuclease genes form an `mR cluster,' whose members are more closely related to one another than they are to other <span class="yellow">murine</span> ribonucleases, yet whose position with respect to the defined RNase A lineages remains unclear. MATERIALS AND METHODS Isolation of genomic fragments encoding <span class="yellow">murine</span> ribonucleases by polymerase chain reaction (PCR) Genomic DNA was isolated from cells of the <span class="yellow">murine</span> 3T3 fibroblast cell line. The mR-3, mR-5 and mR-6P sequences were amplified using oligonucleotide primers derived from the published coding sequence of mEAR-1 as follows: 5-CAA ACC CTT TCC CAG AAG TTT GCC-3 (amino acids 28�33); 5-AAA TGT CCC ATC CAA GTG AAC TGG ACC-3 (amino acids 156�148) (15). The PCR reactions were performed as described previously (14), and the multiple products present in the single 400 bp product were identified by dideoxy<br><br> *To whom correspondence should be addressed. Tel: +1 301 402 9131; Fax: +1 301 402 4369; Email: hr2k@nih.gov Present addresses: +Duke University School of Medicine, Durham, NC 27701, USA and �Department of Pediatrics, State University of New York-Health Sciences Center at Syracuse, Syracuse, NY 13210, USA <br><br> 4236 Nucleic Acids Research, 1997, Vol. 25, No. 21 <br><br> Figure 1. (A) Alignment of amino acid sequences encoded by six related <span class="yellow">murine</span> ribonuclease genes. Regions of amino acid sequence identity among the five functional genes are enclosed in boxes; the boxes enclosing the active site residues (His11, Lys35, His124) and eight cysteines conserved in all members of the RNase A superfamily are shaded. (B) Percent similarities between amino acid sequences determined via the BESTFIT algorithm of the Wisconsin Genetics Computer Group (WGCG) program on-line at the National Institutes of Health. GenBank accession numbers: mEAR (<span class="yellow">murine</span> eosinophil-associated ribonuclease)-1, U72032; mEAR-2, U72031; mR (<span class="yellow">murine</span> ribonuclease)-3, AF017258; mR-4, AF017259; mR-5, AF017260; mR-6P (pseudogene), AF017261. <br><br> sequencing of individual plasmids after subcloning into the pCR II TA cloning vector (Invitrogen, San Diego, CA). The complete open reading frames were obtained by extension in both 5 and 3 directions by uni-directional PCR (Genome Walker kit, Clontech, Palo Alto, CA), involving two amplifications with nested primers and Tth polymerase as described previously (14). The nested gene-specific primer pairs were as follows: for 5 extension of mR-3, 5-CTG GAA CCA CTG GAT ACG TGG GAC TGT CCT-3 and 5-ACG TGG GAC TGT CCT GTG GAG TTC TGG GTT-3; for 3 extension of mR-3, 5-ATG CTG TTG GTG TGT GTG GAA ATC CAA GTG-3 and 5-GCT TGT GCA GTG ACA ATA TAA GTC AAA ACT-3; for 5 extension of mR-5, 5-GGT GGG ACG GTC CTT TGG AGT TCT GGG GTT ACA-3 and 5-GGG GTT ACA GGC AAC TGT GTA GGA CTT CTT TCC-3; for 3 extension of mR-5, 5-GAT GTT GTC CGT GTG TGT CAC AAT CCA CCC-3 and 5-AAG ACT TGC AAA GAC GGG ACA AGT CCA AAT-3; for 5 extension of mR-6P, 5-TTG TTG TTT TGC ATC TGC ATT GTG CAT AAC TGC-3 and 5-TTC TGC ATA ACT GCT TGC TGA ACT TGT GAG TGA-3; for 3 extension of mR-6P, 5-ATG TGA TAA TGC AAT GCT GTC TCT TAG CAG TTA-3 and 5-TAC AAG <br><br> AGT ATG TAA GCC ATT GAA TCA TTT TCT-3. The products of these reactions were likewise subcloned into the PCR II TA vector and evaluated by dideoxy-sequencing. The mR-4 sequence was amplified with a 36 bp primer derived from sequence encoding the amino-terminal signal sequence of mEAR-1; 5-ATG GGT CCG AAG CTG CTT GAG TCC CGA CTT TGT CTC-3 with the carboxy-primer described above. <br><br> Northern analysis The <span class="yellow">murine</span> multi-tissue Northern blot was purchased from Clontech, and pre-hybridized and hybridized as per manufacturer's instructions with radiolabelled oligonucleotide probes. The membrane was washed with 5� SSPE with 0.1% SDS for 1 h at 37_C and the autoradiogram developed after exposure to film at �80_C. The mEAR-1/mEAR-2/mR-3 probe: 5-CTC TTG TCA CTG CAC AAG CCA CTT GGA TTT CC-3, the mR-5 probe: 5-GTC CCG TCT TTG CAA GTC TTG GGT GGA TTG TG-3, and the actin probe: 5-GCA CAT GCC GGA GCC GTT GTC GAC GAC GAG CGC GGC GAT ATC ATC ATC-3 (16). <br><br> 4237 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 4237 <br><br> Figure 2. (A) Dendrogram depicting relationships among the eight characterized <span class="yellow">murine</span> ribonucleases as determined by a modified UPGMA method (17). Abbreviations are as in Figure 1A, also mPR (<span class="yellow">murine</span> pancreatic ribonuclease, sw:rnp_<span class="yellow">mouse</span>), mANG (<span class="yellow">murine</span> angiogenin, U72672), hEDN (<span class="yellow">human</span> eosinophil-derived neurotoxin, M24157), hECP (<span class="yellow">human</span> eosinophil cationic protein, X15161), hRK6 (<span class="yellow">human</span> ribonuclease k6, U64998), pRK6 (porcine kidney ribonuclease (24), and bRK6 (<span class="yellow">bovine</span> kidney ribonuclease, sw:rnkd_bovin). (B) Calculated isoelectric points from amino acid sequences in (A) as determined via the PEPTIDESORT algorithm of WGCG. Table 1. Amino acid sequence comparisons of <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases <br><br> Sequence analysis All DNA sequence analysis and comparisons were performed with the assistance of the Wisconsin Genetics Computer Group and Sequencher (Gene Codes Corporation) programs available at the National Institutes of Health. The dendrogram in Figure 2A was constructed by a modified version of the unweighted pair group method with arithmetic mean (UPGMA) (17), constructed using four initial pairings, mEAR-2 to mR-3, mR-5 to mR-6P, hEDN to hECP and bRK6 to pRK6. RESULTS Isolation of genomic fragments encoding novel <span class="yellow">murine</span> ribonucleases The alignment in Figure 1A displays the amino acid sequences encoded by the six related ribonuclease genes. <span class="blue">Murine</span> eosinophil-associated ribonucleases-1 and -2 (mEAR-1 and mEAR-2) are the predicted amino acid sequences from two genes described previously by Larson and colleagues (15); <span class="yellow">murine</span> ribonucleases 3, 4, 5 and 6-pseudogene (mR-3, mR-4, mR-5 and mR-6P), are the predicted amino acid sequences encoded by novel DNA <br><br> hEDN mEAR-1 mEAR-2 mR-3 mR-4 mR-5 mANG mPR 56 56 54 60 60 43 53 <br><br> hECP 59 59 57 59 58 49 52 <br><br> hRK6 58 59 59 53 56 49 59 <br><br> hPR 47 48 49 48 57 55 79 <br><br> hR4 50 53 51 52 54 57 66 <br><br> hANG 49 48 51 47 47 80 60 <br><br> Values are expressed as percent similarity between pairs of amino acid sequences as determined by the BESTFIT algorithm of the Wisconsin Genetics Computer Group program on-line at the National Institutes of Health. Value representing the highest degree of sequence homology in each row is indicated in boldface. Abbreviations are as defined in Figures 1 and 2, and also include hR4 (<span class="yellow">human</span> ribonuclease 4, sw:rnl4_<span class="yellow">human</span>), hANG (<span class="yellow">human</span> angiogenin, M11567), and hPR (<span class="yellow">human</span> pancreatic ribonuclease, X62946). <br><br> 4238 Nucleic Acids Research, 1997, Vol. 25, No. 21 fragments amplified from <span class="yellow">murine</span> genomic DNA. All five functional genes encode amino acid sequences with eight cysteines and appropriately spaced catalytic histidines and lysine that are conserved among the members of the RNase A superfamily. The predicted coding sequence of the non-functional pseudogene (mR-6P) includes two aberrant stop codons (positions 19 and 100) as well as a point mutation resulting in the destruction of the cysteine at position 78. Each genomic fragment also encodes a 28 residue amino terminal signal sequence (not shown). The amino acid sequences are displayed in order of decreasing similarity to mEAR-1 (Fig. 1B). Genomic fragments encoding mEAR-1 and mEAR-2 were also detected. Relationships among the <span class="yellow">murine</span> ribonucleases A dendrogram depicting the relationships among several of the <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases is shown in Figure 2A. The mEAR-1, mEAR-2, and mR-3 sequences form a closely related sub-group, with only nine amino acid sequence differences noted between mEAR-2 and mR-3, and 12 and 17 between these two and mEAR-1, respectively. The mR-4, mR-5, and mR-6P sequences diverge from this subgroup and from one another as well. These six sequences are more closely related to one another than they are to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or to <span class="yellow">murine</span> angiogenin (mANG) or to any of the three <span class="yellow">human</span> ribonucleases shown (hEDN, hECP or hRK6). Isoelectric points The isoelectric points of all six related ribonucleases range from 8�11, as is typical for members of the RNase A superfamily (Fig. 2B). Interestingly, the isoelectric points within the mEAR-1/mEAR-2/mR-3 subgroup vary by a full 1.0 unit (mEAR-1 at 9.2 to mEAR-2 at 10.2) in response to only 12 discrepancies in amino acid sequence. As increased charge has been associated with increased toxicity in other branches of the RNase A gene family (18), this finding has potential consequences with respect to the function of these proteins (see Discussion). Northern analysis Tissue-specific expression of mRNA encoding these <span class="yellow">murine</span> ribonucleases is shown in Figure 3. The probe complementary to all three sequences of the mEAR-1/mEAR-2/mR-3 subgroup hybridizes to an 1.4 kb mRNA isolated from renal tissue (Fig. 3A). As the probe cannot distinguish among the three genes, it is not possible to conclude whether the band detected represents mRNA encoding one, two, or all three of the ribonucleases of this subgroup. In contrast, the probe complementary to the mR-5 genomic fragment hybridizes to two prominent mRNA species in hepatic tissue, one at 1.4 kb, and a second at 2.0 kb. A prominent mRNA of 1.8 kb was also detected in mRNA isolated from <span class="yellow">murine</span> testicular tissue; fainter hybridizing mRNAs of varying size were detected in lung and skeletal muscle. The size variation may represent differential splicing of the mR-5-encoding mRNA, or may represent mRNAs encoding one or more as yet undiscovered <span class="yellow">murine</span> ribonucleases that are more closely related to mR-5 than those whose sequences are reported here. <br><br> Figure 3. (A) <span class="blue">Murine</span> RNA probed with a 32 bp oligonucleotide complementary to mEAR-1, mEAR-2 and mR-3 coding sequences. Tissue source of each RNA is indicated above each lane. (B) Same as in (A), probed with a 32 bp oligonucleotide complementary to mR-5. (C) Same as in (A), probed with a 48 bp oligonucleotide complementary to <span class="yellow">human</span> actin (16), demonstrating relative loading of each lane. <br><br> Relationship to <span class="yellow">human</span> ribonucleases The data in Table 1 denote the amino acid sequence similarity between pairs of <span class="yellow">murine</span> and <span class="yellow">human</span> ribonucleases of the RNase A superfamily. Overall, the similarities within the identified cluster range from 47 to 60%, with remarkably little variation among the pairs. Although the sequences of the mEAR-1/mEAR-2/mR-3 subgroup are slightly more similar to ECP and RNase k6, and mR-4 and mR-5, to EDN and ECP, the orthologous relationships of these <span class="yellow">murine</span> ribonucleases cannot be discerned from these data. This stands in direct contrast to mPR and mANG, whose <span class="yellow">human</span> orthologs can be clearly distinguished on the basis of their amino acid sequence homologies. <br><br> DISCUSSION In this work, we have identified four novel <span class="yellow">murine</span> ribonuclease genes that, together with mEAR-1 and mEAR-2 defined by Larson and colleagues (15), form an unusual cluster within the RNase A gene superfamily. The six genes within this cluster have varying amino acid sequence homologies to one another, but are clearly more closely related to one another than to either <span class="yellow">murine</span> pancreatic ribonuclease (mPR) or angiogenin (mANG), or to any of the <span class="yellow">human</span> ribonucleases of the RNase A family. These results suggest that the `mR cluster' emerged via multiple duplications of a gene that had already diverged from those encoding the other <span class="yellow">murine</span> ribonucleases. Although this type of expansion is unusual in this gene family, it is actually not unique; similar expansion has occurred in <span class="yellow">bovine</span> species, with gene duplication resulting in the three closely-related <span class="yellow">bovine</span> pancreatic, <span class="yellow">bovine</span> brain (19,20) and <span class="yellow">bovine</span> seminal (21) ribonucleases. One interesting feature of the mR cluster is that its relationship to any of to the six known <span class="yellow">human</span> RNase A-type genes cannot be <br><br> 4239 Nucleic Acids Research, 1997, Vol. 25, No. Nucleic Acids Research, 1994, Vol. 22, No. 121 determined from their respective amino acid sequence homologies. Although the amino acid sequences encoded by mEAR-1 and mEAR-2 match those of tryptic peptides derived from <span class="yellow">murine</span> eosinophil proteins (15), the homology data do not stand in overwhelming support of a unique relationship between any of the mR cluster ribonucleases and the <span class="yellow">human</span> eosinophil ribonucleases EDN and ECP. The existence of an additional, as yet unidentified <span class="yellow">human</span> ribonuclease (or ribonucleases) more closely related to those of the mR cluster cannot be ruled out. Perhaps the most important issue raised by this work is the question of why so much evolutionary energy has been devoted to enlarging and diversifying the RNase A gene superfamily. To date, this superfamily now includes several distinct lineages, two species-limited clusters, and the two most rapidly evolving functional coding sequences known among primates (22). The question of evolutionary energy takes on particular significance here, where five functional ribonucleases have emerged in what appears to be a remarkably short period of evolutionary time (15). It is conceivable that each of these ribonucleases might serve a unique but related function. On this point, a number of studies have suggested a distinct role for the <span class="yellow">human</span> ribonuclease ECP in eosinophil-mediated host defense (6,7); in addition, we have recently shown that eosinophil ribonucleases can inhibit retroviral transduction of <span class="yellow">human</span> target cells (23). The possibility that certain ribonucleases have diverged to promote distinct and specific host defense-related activities remains an intriguing hypothesis. ACKNOWLEDGEMENTS We would like to thank Dr Jianzhi Zhang for helpful discussions, and Drs Harry L. Malech and John I. Gallin for their ongoing suppport of the work in our laboratory. D.B. is a Howard Hughes Medical Institute-National Institutes of Health Research Scholar REFERENCES 1 D'Alessio, G. and Riordan, J.F. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. 2 Sorrentino, S. and Libonati, M. (1997) FEBS Lett. 404, 1�5. 21 22 23 24 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 <br><br> 4239 <br><br> Beintema, J.J., Breukelman, H.J, Carsana, A., and Furia, A. (1997) In D'Alessio, G. and Riordan, J.F. (eds) Ribonucleases: structure and function. Academic Press, Inc., New York, NY. Confalone, E., Beintema, J.J., Sasso, M.P. Carsana, A., Palmieria, M., Vento, M.T. and Furia A. (1995) J. Mol. Evol. 41, 850�858. Futami, J., Tsushima, Y., Murato, U, Tada, H., Sasaki, J., Seno, M., and Yamada, H. (1997) DNA Cell Biol. 16, 413�419. Ackerman, S.J. (1993) In Makino, S. and Fukuda, T. (eds) Eosinophils: biological and clinical aspects. CRC Press, Inc., Boca Raton, FL Spry, C.J.F. (1988) Eosinophils: a comprehensive review and guide to the scientific and medical literature. Oxford University Press, Oxford, UK. Beintema, J.J., Hofsteenge, J., Iwama, M., Morita, T., Ohgi, K., Irie, M., Sugiyama, R.H., Schieven G.L., Dekker, C.A., and Glitz, D.G. (1988) Biochemistry 27, 4530�4538. Sorrentino, S., Tucker, G.K., and Glitz, D.G. (1988) J. Biol. Chem. 263, 16125�16131. Vallee, B.L. and Riordan, J.F. (1988) Adv. Exp. Med. Biol. 234, 41�53. Moroianu, J. and Riordan, J.F. (1994) Proc. Natl. Acad. Sci. USA 91, 1677�1681. Seno, M., Futami, J., Tsushima, Y., Akutagawa, K., Kosaka, M., Tada, H. and Yamada, H. (1995) Bioch. Biophys. Acta 1261, 424�426. Rosenberg, H.F. and Dyer, K.D. (1995) Nucleic Acids Res. 23, 4290�4295. Rosenberg, H.F. and Dyer, K.D. (1996) Nucleic Acids Res. 24, 3507�3513. Larson, K.A., Olson, E.V., Madden, B.J., Gleich, G.J., Lee, N.A. and Lee, J.J. (1997) Proc. Natl. Acad. Sci. USA 93, 12370�12375. Nakajima-Iyima, S., Hamada, H., Reddy, P. and Kakanaga, T. (1985) Proc. Natl. Acad. Sci. USA 82, 6133�6137. Li, W.-H. and Graur, D. (1991) Fundamentals of Molecular Evolution. Sinauer Associates, Sunderland, MA. Rosenberg, H.F. and Dyer, K.D. (1995) J. Biol. Chem. 270, 21539�21544. Watanabe, H., Katoh, H, Ishii, M., Komoda, Y., Sanda, A., Takizawa, Y., Ohgi, K. and Irie, M. (1988) J. Biochem. 104, 939�945. Sasso, M.P., Carsana, A., Confalone, E., Cosi, C., Sorrentino, S., Viola, M., Palmieri, M., Russo, E. and Furia, A. (1991) Nucleic Acids Res. 19, 6469�6474. Palmieri, M., Carsana, A., Furia, A. and Libonati, M. (1985) Eur. J. Biochem. 152, 275�277. Rosenberg, H.F., Dyer, K.D., Tiffany, H.L. and Gonzalez, M. (1995) Nature Genet. 10, 219�223. Domachowske, J.B. and Rosenberg, H.F. (1997) J. Leukoc. Biol., 62, 363�368. Iwama, M., Sanda, A., Ohgi, K., Hofsteenge, J., and Irie, M. (1993) Biosci. Biotech. Biochem. 57, 2133�2138. <br><br>  <h3>pmcA1526545</h3>B lymphocyte stimulator (BLyS) isoforms in systemic lupus erythematosus: disease activity correlates better with blood leukocyte BLyS mRNA levels than with plasma BLyS protein levels
Abstract
Considerable evidence points to a role for B lymphocyte stimulator (BLyS) overproduction in <span class="yellow">murine</span> and <span class="yellow">human</span> systemic lupus erythematosus (SLE). Nevertheless, the correlation between circulating levels of BLyS protein and disease activity in <span class="yellow">human</span> SLE is modest at best. This may be due to an inadequacy of the former to reflect endogenous BLyS overproduction faithfully, in that steady-state protein levels are affected not just by production rates but also by rates of peripheral utilization and excretion. Increased levels of BLyS mRNA may better reflect increased in vivo BLyS production, and therefore they may correlate better with biologic and clinical sequelae of BLyS overexpression than do circulating levels of BLyS protein. Accordingly, we assessed peripheral blood leukocyte levels of BLyS mRNA isoforms (full-length BLyS and ΔBLyS) and plasma BLyS protein levels in <span class="yellow">patients</span> with SLE, and correlated these levels with laboratory and clinical features. BLyS protein, full-length BLyS mRNA, and ΔBLyS mRNA levels were greater in SLE <span class="yellow">patients</span> (n = 60) than in rheumatoid arthritis <span class="yellow">patients</span> (n = 60) or normal control individuals (n = 30). Although full-length BLyS and ΔBLyS mRNA levels correlated significantly with BLyS protein levels in the SLE cohort, BLyS mRNA levels were more closely associated with serum immunoglobulin levels and SLE Disease Activity Index scores than were BLyS protein levels. Moreover, changes in SLE Disease Activity Index scores were more closely associated with changes in BLyS mRNA levels than with changes in BLyS protein levels among the 37 SLE <span class="yellow">patients</span> from whom repeat blood samples were obtained. Thus, full-length BLyS and ΔBLyS mRNA levels are elevated in SLE and are more closely associated with disease activity than are BLyS protein levels. BLyS mRNA levels may be a helpful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>.<br><br>Introduction
B lymphocyte stimulator (BLyS; a trademark of <span class="yellow">Human</span> Genome Sciences, Inc., Rockville, MD, USA) is a 285-amino-acid member of the tumor necrosis factor ligand superfamily [1-3]. A causal relation between constitutive overproduction of BLyS and development of systemic lupus erythematosus (SLE)-like illness has incontrovertibly been established in <span class="yellow">mice</span>. BLyS-transgenic <span class="yellow">mice</span> often develop SLE-like features as they age [3-5], and SLE-prone (NZB × NZW)F1 (BWF1) and MRL-lpr/lpr <span class="yellow">mice</span> respond clinically to treatment with BLyS antagonists (decreased disease progression and improved survival) [3,6].
Considerable inferential evidence points to a role for BLyS overproduction in <span class="yellow">human</span> SLE as well. Cross-sectional studies have demonstrated elevated circulating levels of BLyS in 20–30% of <span class="yellow">human</span> SLE <span class="yellow">patients</span> tested at a single point in time [7,8]. Moreover, a 12-month longitudinal study documented persistently elevated serum BLyS levels in about 25% of SLE <span class="yellow">patients</span> and intermittently elevated serum BLyS levels in an additional 25% of <span class="yellow">patients</span> [9]. Remarkably, circulating BLyS levels did not correlate with disease activity (measured using the SLE Disease Activity Index [SLEDAI]) in these cross-sectional or longitudinal studies [7-9]. Although a statistically significant correlation between circulating BLyS levels and SLEDAI has been appreciated in a more recent 24-month longitudinal study of 245 SLE <span class="yellow">patients</span> (with >1,700 plasma samples analyzed) [10], the correlation remains weak.
The limited correlation between circulating BLyS protein levels and disease activity in these studies may have exposed an inadequacy of the former to reflect faithfully endogenous BLyS overproduction. In addition to the rate of BLyS protein production, several other factors (for example, utilization and excretion) can affect circulating BLyS protein levels. Although there are no practicable means of directly measuring in vivo BLyS production per se in <span class="yellow">humans</span>, the level of BLyS mRNA may serve as a better surrogate marker of in vivo BLyS production than does the level of BLyS protein. Candidate BLyS mRNA isoforms include the full-length BLyS mRNA isoform, which encodes the full-length protein, and the alternatively spliced ΔBLyS mRNA isoform, which encodes a protein with a small peptide deletion [11]. (ΔBLyS does not bind to cells expressing BLyS receptors, and therefore it has no agonistic activity. Moreover, ΔBLyS can form heterotrimers with full-length BLyS, thereby actually functioning as a dominant-negative antagonist of BLyS activity.)
In this report we demonstrate that peripheral blood leukocytes from SLE <span class="yellow">patients</span> express elevated mRNA levels of both full-length BLyS and ΔBLyS relative to those levels expressed by <span class="yellow">patients</span> with rheumatoid arthritis (RA) or by normal control individuals. In the SLE <span class="yellow">patients</span>, both full-length BLyS and ΔBLyS mRNA levels are more closely associated with disease activity (SLEDAI) than are BLyS protein levels. Accordingly, BLyS mRNA levels may be a helpful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>.<br><br>Materials and methods
General details
This study was approved by the institutional review boards of the University of Southern California and the Scripps Research Institute. All <span class="yellow">participants</span> gave their written informed consent before participation in this study.<br><br><span class="yellow">Participants</span>
<span class="yellow">Patients</span> receiving outpatient medical care at the rheumatology clinics of the Los Angeles County + University of Southern California Medical Center were recruited into the study. Diagnoses of SLE (n = 60) or RA (n = 60) were based on established clinical criteria [12]. Healthy control individuals (n = 30) were recruited from Los Angeles County + University of Southern California Medical Center and University of Southern California Keck School of Medicine personnel. No exclusions were made on any basis other than an inability to give informed consent. Each <span class="yellow">patient</span>'s sex, race, age, and medications at the time of the phlebotomy were recorded (Table 1).
Based solely on the <span class="yellow">patient</span>'s willingness to donate a second blood sample, repeat blood samples were collected from 37 of the SLE <span class="yellow">patients</span> 147–511 days (median 371 days) after collection of the first samples. These <span class="yellow">patients</span> were not selected on the basis of any demographic, clinical, or laboratory feature.
Clinical disease activity for the SLE <span class="yellow">patients</span> was assessed using the SLEDAI [13] and using a modified SLEDAI that excludes the contribution of anti-double-stranded DNA (anti-dsDNA) antibodies from the total score. Each <span class="yellow">patient</span>'s medical chart was reviewed for results of standard clinical laboratory tests within the previous or subsequent 1-month period.<br><br>Plasma BLyS determination
Whole venous blood was centrifuged to yield plasma and a buffy coat. The plasma was harvested, stored at -70°C, and assayed for BLyS levels by ELISA [8,14] using Fab fragments of the capture antibody rather than the whole antibody to reduce assay interference by rheumatoid factor. The lower limit of detection in this assay is 0.3 ng/ml. For statistical purposes, plasma samples with BLyS concentrations below the lower limit of detection were assigned a value of 0.25 ng/ml.<br><br>Blood BLyS mRNA determination
The buffy coat from centrifuged whole blood was harvested, added to RNAlater™ (Ambion, Austin, TX, USA) at a 1:4 vol/vol ratio for RNA stabilization, stored at -70°C, and assayed for full-length BLyS and ΔBLyS mRNA levels by real-time PCR. Total RNA was purified from buffy coat samples using RNAeasy miniprep kits (Qiagen, Valencia, CA, USA), and contaminating genomic DNA was removed by DNAse-I digestion. One-tenth volume of total RNA was used as template in the first-strand cDNA reaction using oligo-dT and the Superscript III first-strand synthesis system (Invitrogen, Carlsbad, CA, USA). Duplicate samples of cDNA were amplified with primers against β-actin, full-length BLyS, or ΔBLyS: β-actin sense 5'-CGAGAAGATGACCCAGATCATGT-3'; β-actin anti-sense 5'-GGCATACCCCTCGTAGATGG-3'; full-length BLyS sense 5'-GCAGACAGTGAAACACCAACTATAC-3'; ΔBLyS sense 5'-CAGAAGAAACAGGATCTTACACAT-3'; and full-length BLyS/ΔBLyS anti-sense 5'-TGCCAGCTGAATAGCAGGAATTAT-3'.
A 165 bp amplicon for β-actin was PCR-amplified using the 7900 HT ABI Prism machine (Qiagen) with annealing at 65°C. A 296 bp amplicon for full-length BLyS was PCR-amplified, with annealing at 64°C. A 270 bp amplicon for ΔBLyS was PCR-amplified with annealing at 61°C. The annealing conditions for full-length BLyS and ΔBLyS were determined so that each primer set remained specific to the respective BLyS isoform and yielded a PCR efficiency similar to those of cloned cDNA standards. Melting curve analysis revealed a single peak for each gene amplified. The threshold cycle (Ct) values for each reaction were determined using Sequence Detection System software (Applied Biosystems, Foster City, CA, USA). Results are presented as ratios of full-length BLyS or ΔBLyS mRNA to β-actin mRNA, which were calculated using the following formulae:
2 exp(Ctβ-actin - Ctfull-length BLyS)
2 exp(Ctβ-actin - CtΔBLyS)<br><br>Determination of anti-BLyS autoantibodies
BLyS was bound to microtiter plates by first coating the plates with streptavidin and then adding biotinylated recombinant BLyS. Using these plates as the capture reagent, plasma samples were incubated, and <span class="yellow">horseradish</span> peroxidase-conjugated anti-<span class="yellow">human</span> IgA/IgM/IgG (Southern Biotechnology Associates, Birmingham, AL, USA; 1:20,000 final dilution) or <span class="yellow">horseradish</span> peroxidase-conjugated anti-<span class="yellow">human</span> IgG (Southern Biotechnology; 1:10,000 final dilution) were used as the detector reagents.<br><br>Statistical analysis
All analyses were performed using SigmaStat software (SPSS, Chicago, IL, USA). Results that did not follow a normal distribution were log-transformed to achieve normality. Parametric testing between two matched or unmatched groups was performed using the paired or unpaired t test, respectively. Parametric testing among three or more groups was performed using one-way analysis of variance. When log-transformation failed to generate normally distributed data or the equal variance test was not satisfied, nonparametric testing was performed using the Mann–Whitney rank sum test between two groups and by Kruskal–Wallis one-way analysis of variance on ranks among three or more groups. Correlations were determined using Pearson product moment correlation for interval data and using Spearman rank order correlation for ordinal data or for interval data that did not follow a normal distribution. Nominal data were analyzed using χ2 analysis-of-contingency tables.<br><br>
Results
Elevated plasma BLyS levels and blood levels of full-length BLyS and ΔBLyS mRNA isoforms in systemic lupus erythematosus <span class="yellow">patients</span>
Previous reports of elevated circulating BLyS levels in SLE <span class="yellow">patients</span> were based on a BLyS ELISA that utilized a whole (unfragmented) capture anti-BLyS monoclonal antibody [7-9]. Since the publication of these reports, it has been recognized that the presence of rheumatoid factor can potentially interfere with the assay and lead to spurious overestimation of the true circulating BLyS levels (<span class="yellow">Human</span> Genome Sciences, Inc.; unpublished observations). To mitigate potential interference from rheumatoid factor, the BLyS ELISA has been modified and the capture anti-BLyS monoclonal antibody is now utilized as a Fab fragment. Despite the changes in the ELISA format, our findings are entirely consistent with those of the previous reports. Plasma BLyS levels were significantly greater in the SLE group than in either RA or normal control group (P < 0.001; Figure 1a). Arbitrary assignment of the 95th percentile value among the normal control individuals as the upper limit of 'normal' revealed that two of the 30 normal control individuals, 15 of the 60 RA <span class="yellow">patients</span>, and 29 of the 60 SLE <span class="yellow">patients</span> harbored elevated plasma BLyS levels (P < 0.001).
Overexpression of BLyS in SLE <span class="yellow">patients</span> was also established by measuring BLyS mRNA levels normalized to β-actin mRNA levels in peripheral blood leukocytes (buffy coats). The geometric mean full-length BLyS mRNA and ΔBLyS mRNA levels among the SLE <span class="yellow">patients</span> were each significantly greater than those among the RA <span class="yellow">patients</span> and normal control individuals, respectively (P < 0.001 for each; Figure 1b,c). Arbitrary assignment of the 95th percentile values for full-length BLyS and ΔBLyS mRNA levels among the normal control individuals as the upper limits of 'normal' revealed that two of the 30 normal control individuals, four of the 60 RA <span class="yellow">patients</span>, and 20 of the 60 SLE <span class="yellow">patients</span> had elevated full-length BLyS mRNA levels (P < 0.001), and that two of the 30 normal control individuals, three of the 60 RA <span class="yellow">patients</span>, and 19 of the 60 SLE <span class="yellow">patients</span> had elevated ΔBLyS mRNA levels (P < 0.001). Levels of full-length BLyS and ΔBLyS mRNA strongly correlated with each other (r = 0.703; P < 0.001) in the SLE cohort, and plasma BLyS levels also correlated significantly with levels of each BLyS isoform (r = 0.429, P < 0.001; and r = 0.290, P = 0.024, respectively). Among these SLE <span class="yellow">patients</span>, none of the measured BLyS parameters correlated with <span class="yellow">patient</span> age, sex, race, or daily dose of corticosteroids (data not shown). Because the racial composition of the normal cohort was not as predominantly Hispanic as were those of the RA and SLE cohorts, we assessed the BLyS parameters in the respective Hispanic subpopulations. As for the entire populations, values for SLE were significantly greater than those for either RA or normal controls (P ≤ 0.004; data not shown).<br><br>Correlations between BLyS parameters and plasma immunoglobulin levels
BLyS is a potent B cell survival factor [15-21], and administration of exogenous BLyS to <span class="yellow">mice</span> leads to B cell expansion and hypergammaglobulinemia [1]. Previous studies with numbers of SLE <span class="yellow">patients</span> greater than were included in the present study documented a modest but significant correlation between serum levels of BLyS and IgG [8,10]. In our SLE cohort of limited size, plasma BLyS levels failed to show significant correlations with plasma levels of total immunoglobulin, IgG, or IgA. In contrast, full-length BLyS and ΔBLyS mRNA levels correlated significantly with each (Figure 2). (None of the BLyS parameters correlated with plasma IgM levels.) The absence of significant correlation between plasma BLyS levels and the immunoglobulin parameters also persisted when just the 53 <span class="yellow">patients</span> with detectable plasma BLyS levels were considered (r = -0.133, P = 0.346 for total immunoglobulin; r = -0.048, P = 0.734 for IgG; and r = 0.033, P = 0.817 for IgA).<br><br>Correlations between BLyS parameters and disease activity
Previous studies either have failed to demonstrate a significant correlation between disease activity and circulating BLyS levels [7-9] or have detected only a weak correlation between the two [10]. Consonant with those studies, we identified no significant correlation between plasma BLyS levels and SLEDAI in our cohort of 60 SLE <span class="yellow">patients</span> (Figure 3a). The failure to demonstrate a significant correlation cannot be attributed to a skewing of the results by the <span class="yellow">patients</span> in whom plasma BLyS levels were below the limit of detection, because no significant correlation was detected among the 53 SLE <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range (r = 0.185, P = 0.183). In contrast, a significant correlation between SLEDAI and full-length BLyS mRNA levels was readily discernible (Figure 3b). A trend toward a correlation between SLEDAI and ΔBLyS mRNA levels was also observed, although it did not achieve statistical significance (Figure 3c).
A component of the SLEDAI is the presence of circulating anti-dsDNA antibodies. Because circulating BLyS levels may affect the presence and/or titers of circulating anti-dsDNA antibodies [7-10], we assessed correlations between the individual BLyS parameters and a modified SLEDAI that excludes any consideration of anti-dsDNA antibodies. As with the unmodified SLEDAI, the modified SLEDAI did not correlate with plasma BLyS levels (Figure 3d) either among the SLE cohort overall or among the 53 <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range (r = 0.160, P = 0.252), but it significantly correlated with full-length BLyS mRNA levels (Figure 3e) and exhibited a trend toward correlation with ΔBLyS mRNA levels (Figure 3f). Thus, the stronger correlations between BLyS mRNA levels and disease activity cannot solely be explained by any effects that BLyS may have on anti-dsDNA antibodies per se.
Moreover, among the 37 SLE <span class="yellow">patients</span> who were evaluated on two separate occasions, trends toward correlation were appreciated between changes in the unmodified or modified SLEDAI and changes in full-length BLyS or ΔBLyS mRNA levels but not changes in plasma BLyS levels (Figure 4). These results cannot be ascribed to changes in medications taken by the <span class="yellow">patients</span>, because changes in neither disease activity nor in any of the BLyS parameters correlated with changes in the doses of corticosteroids or cytotoxics taken by the <span class="yellow">patients</span> (data not shown). The failure to demonstrate a meaningful association between changes in SLEDAI score and changes in plasma BLyS protein levels cannot be attributed to a skewing of the results by the <span class="yellow">patients</span> in whom plasma BLyS levels were below the limit of detection, because the absence of association between the two persisted among the 27 SLE <span class="yellow">patients</span> in whom plasma BLyS levels were in the detectable range in both samples (r = -0.069, P = 0.727 for plasma BLyS versus unmodified SLEDAI; r = -0.020, P = 0.919 for plasma BLyS versus modified SLEDAI).<br><br>Lack of correlation between levels of BLyS mRNA isoforms and percentages of individual leukocyte cell types
Among cells in peripheral blood, BLyS is predominantly expressed by cells of the myeloid lineage (monocytes and neutrophils) [1,14,22,23]. Accordingly, a shift in the differential leukocyte count away from lymphocytes to monocytes and/or neutrophils could substantially alter BLyS mRNA results. Because of the limited amount of blood we were permitted to obtain from the SLE <span class="yellow">patients</span> (consequent to the high prevalence of anemia among these <span class="yellow">patients</span>), we were unable to purify the individual leukocyte populations for BLyS mRNA analysis. Nevertheless, to demonstrate that the elevated BLyS mRNA levels in SLE did not simply reflect a shift in differential leukocyte count, we assessed the correlations between the individual BLyS parameters on the one hand and the percentages of blood neutrophils, monocytes, and lymphocytes on the other. No correlations were appreciated (Figure 5).<br><br>Presence of anti-BLyS autoantibodies in <span class="yellow">patients</span> with systemic lupus erythematosus
The poorer correlation between plasma BLyS protein levels and disease activity compared with that between BLyS mRNA levels and disease activity was striking. <span class="yellow">Patients</span> with SLE frequently develop autoantibodies against self-antigens, and so some of the SLE <span class="yellow">patients</span> might have harbored autoantibodies to BLyS. Such autoantibodies could have complexed with BLyS and enhanced its clearance, thereby masking BLyS overproduction. Alternatively, such autoantibodies might have sterically blocked the epitopes recognized by the detecting antibodies in the in vitro ELISA. In this case, measured BLyS levels would have been spuriously reduced, again masking BLyS overproduction.
In our cohort, IgA/IgM/IgG anti-BLyS antibodies were detected in six out of the 60 SLE <span class="yellow">patients</span>. Such autoantibodies were also detected in two out of 60 RA <span class="yellow">patients</span> and in one out of 30 normal control individuals, demonstrating that anti-BLyS autoantibodies are not restricted to SLE <span class="yellow">patients</span>. IgG anti-BLyS autoantibodies were detected in 3 SLE <span class="yellow">patients</span> but in no RA <span class="yellow">patients</span> or normal control individuals.<br><br>
Discussion
Elevated blood levels of BLyS protein and mRNA are well described features of <span class="yellow">human</span> SLE [7-9]. We confirmed these observations in our study and extended them by documenting increases not just in levels of full-length BLyS mRNA but also in levels of ΔBLyS mRNA (Figure 1). Of note, BLyS mRNA levels were elevated in SLE but not in RA, raising the possibility that BLyS overproduction in SLE is systemic whereas BLyS overproduction in RA may be more focused to the affected arthritic joints [24]. The modestly elevated plasma BLyS protein levels in RA <span class="yellow">patients</span> may reflect, at least in part, release of locally overproduced BLyS into the circulation.
The relationship between circulating BLyS protein levels and disease activity was addressed in several previous studies, but significant correlations between the two measures did not emerge [7-9]. In the largest study to date, a 2-year longitudinal study of 245 <span class="yellow">patients</span> in which more than 1,700 plasma samples were analyzed, a significant but weak correlation between the two was appreciated [10]. In the present study, a significant correlation between plasma BLyS protein levels and disease activity was again not realized (Figure 3a,d).
The weak, at best, correlation between circulating BLyS levels and disease activity is seemingly rather surprising. There is a clear-cut association in BlyS transgenic <span class="yellow">mice</span> between BLyS overexpression and development of SLE-like features [3-5], and treatment of SLE-prone <span class="yellow">mice</span> with BLyS antagonists retards the progression of disease and improves survival [3,6]. Moreover, development of precocious glomerular pathology in autoimmune-prone <span class="yellow">mice</span> correlates strongly with circulating BLyS levels [25].
The likely explanation for the weak correlation between circulating BLyS levels and disease activity in <span class="yellow">human</span> SLE is not that disease activity in SLE <span class="yellow">patients</span> is insensitive to the degree of BLyS overproduction. Rather, a more tenable explanation is that circulating BLyS levels in <span class="yellow">human</span> SLE do not always accurately reflect excessive endogenous BLyS production. We can identify at least three nonmutually exclusive mechanisms to explain a dissociation between the two.
First, SLE <span class="yellow">patients</span> frequently develop autoantibodies to a myriad of self-targets (for example, erythrocytes, lymphocytes). Indeed, we detected circulating IgA/IgM/IgG anti-BLyS autoantibodies in 10% (6/60) of the tested SLE <span class="yellow">patients</span>, and we detected circulating IgG anti-BLyS autoantibodies in 5% (3/60). These percentages may be underestimates of the true prevalence of anti-BLyS autoantibodies, because some of these autoantibodies may be saturated in vivo with circulating BLyS, rendering them incapable of binding to BLyS in the in vitro detection assay. We do not yet know whether the anti-BLyS autoantibodies are functionally neutralizing but, regardless, such autoantibodies could enhance the clearance of BLyS and/or interfere with in vitro detection of BLyS, thereby masking endogenous BLyS overproduction.
Second, increased urinary excretion of BLyS has been reported in SLE <span class="yellow">patients</span>, especially among those with clinically overt renal involvement [26]. At least four of the <span class="yellow">patients</span> we studied manifested nephrotic-range proteinuria (≥3 g/24 hours), and so urinary loss of BLyS was probably substantial in at least these <span class="yellow">patients</span>. A validated assay for urinary BLyS detection has not yet been developed so we were unable to quantify urinary BLyS levels. Once an assay for urinary BLyS levels is validated, we should be able to assess the effect of urinary BLyS excretion on circulating BLyS levels.
Third, BLyS promotes in vivo expansion of B cells [1]. Freshly isolated SLE B cells, despite intact surface expression of BLyS receptors, bind less biotinylated BLyS ex vivo than do freshly isolated normal B cells [27]. Although other interpretations are possible, the most likely explanation is that BLyS receptors on B cells in SLE <span class="yellow">patients</span> are occupied in vivo by soluble BLyS. Accordingly, it is likely that BLyS receptors expressed by the expanded B cell population do bind BLyS and remove it from the circulation, resulting in a homeostatic pathway that modulates the effects of BLyS overproduction on circulating BLyS levels. Indeed, circulating levels of BLyS rise with peripheral blood B cell depletion and fall with re-emergence of peripheral blood B cells in rituximab-treated RA or SLE <span class="yellow">patients</span> [28,29], highlighting this inverse relationship between circulating BLyS levels and B cell load. Moreover, one of the hallmarks of active disease in <span class="yellow">human</span> SLE is the increased percentages of activated B cells and plasma cells in peripheral blood [30-34], probably reflecting increased systemic numbers of activated B cells and plasma cells. Although not yet formally tested, differential BLyS receptor expression by these cells compared with expression by nonactivated B cells may result in increased peripheral BLyS utilization, further dampening the effects of BLyS overproduction on circulating protein levels.
To circumvent these confounding processes, we used BLyS mRNA levels as a surrogate marker of endogenous BLyS production. Overall, the correlations between disease activity and either full-length BLyS or ΔBLyS mRNA levels were much stronger than that between disease activity and BLyS protein levels (Figures 3 and 4). This was the case regardless of whether we used the standard SLEDAI or the modified SLEDAI as a measure of disease activity. These correlations were not spurious ones consequent to shifts in percentages of leukocyte subpopulations in peripheral blood, because BLyS mRNA levels did not correlate with percentages of blood neutrophils, monocytes, or lymphocytes (Figure 5).
A similar pattern was observed between plasma immunoglobulin levels and the BLyS parameters, with plasma levels of total immunoglobulin, IgG, and IgA correlating significantly with full-length BLyS and ΔBLyS mRNA levels but not with plasma BLyS levels (Figure 2). These significant correlations between full-length BLyS or ΔBLyS mRNA levels and plasma immunoglobulin levels again highlight the greater ability of BLyS mRNA levels, compared with plasma BLyS protein levels, to reflect ongoing BLyS overproduction.
At present, it is not known whether soluble ΔBLyS protein is present in the circulation of SLE <span class="yellow">patients</span> or of normal individuals. Although full-length BLyS protein is readily cleaved and released from cells transfected with a vector containing <span class="yellow">murine</span> full-length BLyS, ΔBLyS protein is not cleaved or released from <span class="yellow">murine</span> ΔBLyS transfectants [11]. Given the strong similarities between <span class="yellow">murine</span> and <span class="yellow">human</span> full-length BLyS and ΔBLyS, it is likely that <span class="yellow">human</span> soluble ΔBLyS protein is also not cleaved from the cell surface and released into the circulation. Moreover, soluble ΔBLyS protein is not released from cells transfected with a vector containing just the extracellular domain of <span class="yellow">human</span> ΔBLyS (which encodes the soluble protein; A.L. Gavin, unpublished observations). Whether this reflects rapid intracellular degradation of soluble ΔBLyS or some other impediment to its release remains unknown. Regardless, if the inability to release soluble ΔBLyS in vitro faithfully recapitulates in vivo biology, then the stronger associations between SLE disease activity and full-length BLyS or ΔBLyS mRNA levels compared with that between SLE disease activity and BLyS protein levels could not be attributable to interference by biologically inactive (inhibitory) ΔBLyS protein in the BLyS protein detection ELISA. Importantly, even if soluble ΔBLyS protein is present in the circulation and is detected by the BLyS protein detection ELISA, then the stronger correlations between SLE disease activity and full-length BLyS or ΔBLyS mRNA levels than that between disease activity and total BLyS (including ΔBLyS) protein levels suggest that full-length BLyS and/or ΔBLyS mRNA levels may operationally serve as useful biomarkers of disease activity in SLE. Although the complexity and labor intensiveness associated with quantitative real-time PCR may render measurement of BLyS mRNA levels impracticable for routine clinical practice, such measurement could readily be incorporated into clinical trials and yield valuable information.
Longitudinal observations in large numbers of SLE <span class="yellow">patients</span> will be necessary to establish or refute the utility of full-length BLyS and/or ΔBLyS mRNA to subserve this clinically vital function.
Although expression of the two major BLyS isoforms was highly coordinate among SLE <span class="yellow">patients</span>, there were several <span class="yellow">patients</span> in whom ΔBLyS mRNA levels were markedly greater than or less than the expected values based on full-length BLyS mRNA levels (data not shown). This raises the possibility that dysregulation of ΔBLyS may contribute to overall BLyS dysregulation in at least some SLE <span class="yellow">patients</span>. It is known that interferon-γ, interleukin-10, interferon-α, and CD154 can upregulate full-length BLyS mRNA levels [14,22,35], but it is not known what effects these or other cytokines/cell-surface structures have on ΔBLyS expression. Further investigation of the regulation of ΔBLyS and the differential expression of BLyS isoforms is certainly warranted.
Although the associations between full-length BLyS and/or ΔBLyS mRNA levels and disease activity in SLE were usually strong when the SLE cohort was analyzed in aggregate, there were several SLE <span class="yellow">patients</span> in whom BLyS mRNA levels were quite high despite little objective ongoing disease activity, and there were several SLE <span class="yellow">patients</span> in whom BLyS mRNA levels were low despite considerable ongoing disease activity. One must recognize that the bulk of the pathogenic autoimmune responses probably takes place in the spleen and lymph nodes, rather than in the peripheral blood, where myeloid lineage cells (for example, dendritic cells) produce BLyS and support B cell survival and expansion [36]. Local BLyS production in the secondary lymphoid tissues will be more important to the development and maintenance of an autoimmune response than will remote BLyS levels in the circulation. Because at least some autoreactive B cells may be more sensitive to BLyS-mediated survival signals than non-autoreactive B cells [37,38], local increases in BLyS production could preferentially promote expansion of autoreactive B cells. These cells, in turn, could activate autoreactive T cells by presenting autoantigen to them, and some of the autoreactive B cells would respond to T cell derived signals and mature into (pathogenic) autoantibody secreting plasma cells. In contrast to <span class="yellow">murine</span> studies, in which investigators can readily harvest and analyze lymphoid and myeloid lineage cells from any site (for example, spleen, bone marrow), such is not the case for <span class="yellow">human</span> studies. Peripheral blood is the only site readily accessible for <span class="yellow">human</span> studies, and it is possible that, at least in some <span class="yellow">patients</span>, BLyS mRNA levels in circulating leukocytes do not reflect local BLyS production in the secondary lymphoid tissues.
One must also recognize that disease activity in SLE is not solely driven by B cells. Systemic inflammation and SLE flares can be triggered via B cell independent means. Not all SLE <span class="yellow">patients</span> treated with a B cell depleting course of rituximab experience clinical remission [39], strongly pointing to the importance of non-B cells in disease pathogenesis/maintenance. Conversely, not all pathogenic B cells necessarily require high levels of BLyS to effect their pathogenicity. <span class="red">Murine</span> studies have unequivocally documented B cell subpopulations that do not depend upon BLyS for their survival [40-42]. Although <span class="yellow">mice</span> completely devoid of BLyS have reduced numbers of mature B cells and harbor reduced levels of immunoglobulin, these reductions are incomplete. Thus, it is possible that some SLE <span class="yellow">patients</span> harbor pathogenic B cells that are relatively insensitive to BLyS and could drive considerable disease activity even in the context of low endogenous BLyS production. Conversely, <span class="yellow">patients</span> with high BLyS mRNA levels may be those <span class="yellow">patients</span> whose disease is strongly driven by BLyS and may be especially helped by BLyS antagonist therapy. Future clinical trials should be able to establish whether the BLyS mRNA levels are good predictors of response to such agents.<br><br>Conclusion
Plasma total immunoglobulin, IgG, and IgA levels and disease activity (as measured by SLEDAI) in SLE <span class="yellow">patients</span> correlate more closely with peripheral blood leukocyte levels of BLyS mRNA than with plasma levels of BLyS protein. These findings suggest that BLyS mRNA levels better reflect in vivo BLyS production than do circulating BLyS protein levels, and may be a useful biomarker in the clinical monitoring of SLE <span class="yellow">patients</span>. These findings also support the premise that BLyS overexpression not only promotes development of disease but also actively contributes to the ongoing maintenance of disease in SLE <span class="yellow">patients</span>. This reinforces the rationale underlying clinical trials with BLyS antagonists in SLE.<br><br>Abbreviations
anti-dsDNA = anti-double-stranded DNA; BLyS = B lymphocyte stimulator; bp = base pairs; Ct = threshold cycle; ELISA = enzyme-linked immunosorbent assay; PCR = polymerase chain reaction; RA = rheumatoid arthritis; SLE = systemic lupus erythematosus; SLEDAI = SLE Disease Activity Index.<br><br>Competing interests
TSM and DMH were employees of <span class="yellow">Human</span> Genome Sciences (HGS) at the time the investigation was conducted. (DMH has since left the company.) WS has received research support from HGS and has served as a consultant to HGS (<$10,000). CEC, ALG, and DN declare that they have no competing interests.<br><br>Authors' contributions
CEC identified and recruited all <span class="yellow">participants</span>; collected all the blood samples and reviewed all the medical charts; and wrote the initial working draft of this manuscript. ALG developed and performed all the real-time PCR assays and assisted in the interpretation of the results and in writing the final version of the manuscript. TSM performed the plasma BLyS protein and anti-BLyS assays and assisted in the interpretation of the results and in writing the final version of the manuscript. DMH assisted in the design in the study, in the interpretation of the results, and in writing the final version of the manuscript. DN assisted in the design in the study, in the interpretation of the results, and in writing the final version of the manuscript. WS conceived the study, supervised the recruitment of <span class="yellow">participants</span>, performed the statistical analyses, assisted in the interpretation of the results, and supervised the editing of the manuscript to its final form. All authors read and approved the final manuscript version.<br><br>
<h3>pmcA2065882</h3>Identification of Two Independent Risk Factors for Lupus within the MHC in United Kingdom Families
Abstract
The association of the major histocompatibility complex (MHC) with SLE is well established yet the causal variants arising from this region remain to be identified, largely due to inadequate study design and the strong linkage disequilibrium demonstrated by genes across this locus. The majority of studies thus far have identified strong association with classical class II alleles, in particular HLA-DRB1*0301 and HLA-DRB1*1501. Additional associations have been reported with class III alleles; specifically, complement C4 null alleles and a tumor necrosis factor promoter SNP (TNF-308G/A). However, the relative effects of these class II and class III variants have not been determined. We have thus used a family-based approach to map association signals across the MHC class II and class III regions in a cohort of 314 complete United Kingdom Caucasian SLE trios by typing tagging SNPs together with classical typing of the HLA-DRB1 locus. Using TDT and conditional regression analyses, we have demonstrated the presence of two distinct and independent association signals in SLE: HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, OR = 2.3) and the T allele of SNP rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001, OR = 2.0) in intron 6 of the class III region gene SKIV2L. Assessment of genotypic risk demonstrates a likely dominant model of inheritance for HLA-DRB1*0301, while rs419788-T confers susceptibility in an additive manner. Furthermore, by comparing transmitted and untransmitted parental chromosomes, we have delimited our class II signal to a 180 kb region encompassing the alleles HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201 alone. Our class III signal importantly excludes independent association at the TNF promoter polymorphism, TNF-308G/A, in our SLE cohort and provides a potentially novel locus for future genetic and functional studies.
Systemic lupus erythematosus (SLE/lupus) is a complex autoimmune disease in which the body's immune system attacks its own tissues, causing inflammation in a variety of different organs such as the skin, joints, and kidneys. The cause of lupus is not known, but genes play a significant role in the predisposition to disease. The major histocompatibility complex (MHC) on Chromosome 6 contains at least 100 different genes that affect the immune system, including the genes with the strongest effect on lupus susceptibility. Despite the importance of the MHC in SLE, the identity of the actual genes in the MHC region that cause SLE has remained elusive. In the present study, we used the latest set of genetic markers present at the MHC in lupus families to identify the actual genes that affect the disease. To our knowledge, we have shown for the first time that two separate groups of genes are involved in SLE. One group of genes alters how the immune system may inappropriately target its own tissues in the disease. How the second set of genes predisposes to SLE is the subject of ongoing study.<br><br><br><br>Introduction
Since the early 1970s, the <span class="yellow">human</span> major histocompatibility complex (MHC) has been shown to be associated with a number of autoimmune, inflammatory, and infectious diseases, and it continues to be the focus of intense research [1]. The recently defined extended MHC (xMHC) encompasses 7.6 Mb of genome on 6p21.3 and is divided into five subregions from telomere to centromere: extended class I, classical class I, classical class III, classical class II, and extended class II. In addition, the MHC contains two hypervariable regions, the RCCX module in class III (spanning complement C4) and the HLA-DRB genes in class II, that both exhibit copy number polymorphism. Examination of the sequence across the extended MHC reveals the presence of 421 genes, and over 252 (60%) are thought to be expressed [2]. Around 40% of genes expressed within the classical MHC encode proteins with putative immunomodulatory function [3]. The classical class I and class II loci encode the <span class="yellow">human</span> leucocyte antigen (HLA) proteins involved in antigen presentation to T cells, initiating the adaptive immune response. The class III region contains the greatest density of genes in the genome (58 expressed genes), which are often found in functionally related clusters [2].
A major obstacle in the identification of disease-specific causal variants within the MHC has been the strong linkage disequilibrium (LD) exhibited by certain alleles in this region, resulting in the existence of long-range, conserved, extended haplotypes [4], also known as ancestral haplotypes [5], sometimes spanning more than 2 Mb [6]. Thus, for many MHC-associated diseases, it has only been possible to delimit association signals to a particular extended haplotype or segment of one.
Systemic lupus erythematosus (SLE/lupus, [Online Mendelian Inheritance in <span class="yellow">Man</span> 152700, http://www.ncbi.nlm.nih.gov/sites/entrez?db=OMIM&TabCmd=Limits]) is a chronic, multi-system autoimmune disease affecting young <span class="yellow">women</span> ten times more commonly than <span class="yellow">men</span>. The worldwide prevalence of SLE is estimated at between 12 and 124 cases per 100,000 individuals [7]. SLE is characterized by the presence of pathogenic autoantibodies to nuclear and cell-surface antigens that show affinity maturation. The consequent immune complexes deposit in tissues, causing inflammation and damage. It is well established that there is a complex genetic component to lupus aetiology, with hormonal and environmental influences also contributing to disease susceptibility [8,9].
The MHC has been the most consistently confirmed genetic risk factor for SLE, and multiple different genes within the region have been significantly implicated with disease susceptibility. For example, hereditary and acquired deficiencies of the early classical complement component C4, located within the MHC class III locus, leads to a lupus-like syndrome. A role for another class III gene, tumour necrosis factor alpha (TNF), in SLE was suggested following the observation that the lupus-prone New Zealand F1 <span class="yellow">mouse</span> hybrid exhibits constitutively low TNF expression [10]. Recently, the development of antinuclear antibodies in <span class="yellow">patients</span> treated with TNF antagonists has also stimulated interest in the possible role of TNF in SLE [11–13]. <span class="red">Murine</span> and <span class="yellow">human</span> candidate gene studies, together with genome-wide linkage screens, provide further support that multiple genetic loci, including the <span class="yellow">mouse</span> MHC complex H2 and the <span class="yellow">human</span> MHC locus, contribute to disease susceptibility [14–17].
It should be noted that the <span class="yellow">human</span> MHC was first associated with SLE in 1971, when studies demonstrated that lupus probands were enriched for the class I alleles HL-A8 (now known as HLA-B8) and HLA-W15 (now known as HLA-B15) when compared with healthy controls [18,19]. Further case control association studies were small, performed in ethnically diverse populations, and tested only a small number of the classical HLA and complement C4 alleles. The most consistent findings reported to date are associations with the class II alleles HLA-DR2 (DRB1*1501) and HLA-DR3 (DRB1*0301) and their respective haplotypes in Caucasian populations. The complement C4A null allele (C4A*Q0) has shown inconsistent association with lupus in a number of studies—a situation that may reflect genetic heterogeneity in disease susceptibility [20–23]. In addition, a recent study has demonstrated that low C4A copy number is a risk factor for lupus in a European American cohort [24]. However, the C4A null allele lies on the lupus-associated DR3 “autoimmune” extended haplotype (AH8.1), which exhibits extremely strong LD [6]. It therefore remains to be definitively established whether this locus constitutes a distinct susceptibility allele to that of the class II association or is merely in LD with it. Similarly, certain TNF promoter polymorphisms, including the much-studied SNP TNF-308G/A, have shown association with SLE; but again, many of these variants are carried on the highly conserved 8.1 ancestral haplotype, thus restricting interpretation of these data.
In 2002, a family-based study employing microsatellites as surrogate markers for HLA-DRB1 haplotypes in Caucasian lupus families demonstrated association with DR3-, DR2-, and DR8 (DRB1*0801)-containing haplotypes. In that study, Graham and colleagues reported that, taking advantage of recombinant chromosomes, the disease risk region could be limited to a 1 Mb region encompassing classical class II and class III [25].
We have performed a medium resolution association mapping study of the MHC in lupus families, utilizing a combination of SNPs and four-digit typing at the HLA-DRB1 locus in order to anchor haplotypes. Sixty-eight SNPs were successfully genotyped across a 2.4 Mb region of the MHC, from the class I locus KIAA1949 to the class II gene HLA-DPB2, in 314 UK Caucasian SLE trios. We used these data to perform a family-based association study in an attempt to distinguish the relative effects of the class II and class III regions of the MHC in lupus susceptibility. In addition, we employed the long-range haplotype test to search for the presence of high-frequency, extended haplotypes indicative of recent positive selection [26]. We have also used family-based and case-control strategies to examine genotypic risk at HLA-DRB1 and rs419788.<br><br>Results
Association Testing of HLA-DRB1 and MHC Region SNPs
In order to define the causal variation within the MHC region, we typed 314 complete SLE trios for the HLA-DRB1 gene as well as for 86 SNPs across a 2.4 Mb region encompassing the HLA class I locus HLA-B to HLA-DPB2. High-quality genotype data was obtained for HLA-DRB1 and 68 MHC SNPs (see Table S1 for quality control data). Association testing of the HLA-DRB1 gene revealed a significant association with HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, T:U = 129:55) in our lupus cohort (Table 1). There was also a trend for under transmission of the HLA-DRB1*0701 allele (nominal p = 0.0013, T:U 42:77); however, this association was no longer significant after correction for multiple testing as determined by 10,000 permutations of the dataset (permuted p = 0.09). Furthermore, we did not find evidence of association with HLA-DRB1*1501 (nominal p = 1.0, T:U 70:70) or HLA-DRB1*0801 (nominal p = 1.0, T:U = 11:11) in our cohort (see Table S2 for complete HLA-DRB1 association data); alleles previously suggested by microsatellite typing of a US lupus cohort [25].
Association testing of the MHC region SNPs also identified significant evidence of association to SLE (Table 1 for associated markers and Table S3 for all MHC SNPs). The SNP with the most significant association, rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001) was of similar strength to that of the HLA-DRB1*0301 allele, with odds ratios (ORs) and 95% confidence intervals (CIs) of 2.0 (1.6–2.6) and 2.3 (1.7–3.2), respectively. This SNP is located within intron 6 of the class III gene, superkiller viralicidic activity 2-like (<span class="yellow">Saccharomyces cerevisiae</span>) (SKIV2L), and is located approximately 500 kb telomeric to the HLA-DRB1 gene. Of the other 12 SNPs that were significantly associated with SLE (nominal p = 4.0 × 10−4 to 2.5 × 10−7; permuted p = 0.03 to <0.0001), one was located in the class I region between HLA-B and MICA, seven were located in the class III region, and four were situated in the class II region (Table 1; Figure S1). Specifically, the seven associated class III SNPs were located in or close to the following genes: the TNF promoter, BAT3, SLC44A4, EHMT2, TNXB, GPSM3, and NOTCH4. One of the four class II associated SNPs was close to HLA-DRA, two were between HLA-DRB1 and HLA-DQA1 and one was in intron 1 of HLA-DQA1. The correlation between all 68 SNPs and HLA-DRB1 in our UK SLE cohort is illustrated in Figure 1. The markers showing significant association are highlighted.<br><br>Conditional Analyses Identify Two Independent Association Signals in the MHC
In order to establish whether the two most associated signals identified in this association-mapping experiment are likely to represent a single causal allele or independent risk factors, we first examined the association data conditioned upon the presence of the HLA-DRB1*0301 allele. We found that four of the 13 associated SNPs showed evidence of signals independent of HLA-DRB1*0301 in our dataset, the strongest of which was rs419788 (Table 1). We therefore conditioned the three remaining SNPs (rs2523589, rs1052486, and rs605203) on rs419788 to assess whether these signals are truly independent of each other or show association due to LD with rs419788. In addition, we included HLA-DRB1 in stepwise conditional regression analyses performed on the SNPs showing association independent of HLA-DRB1 (unpublished data). These analyses demonstrated that the observed association signals at rs2523589, rs1052486, and rs605203 were predominantly dependent upon the association at rs419788, and suggested that there are two major independent association signals in the MHC in UK SLE: HLA-DRB1 and rs419788. The independence of the association signals at HLA-DRB1 and rs419788 is further supported by the observation that there is only modest LD between these two (r2 = 0.24). There was no association with any other HLA-DRB1 allele and the four SNPs independent of HLA-DRB1*0301 (TRANSMIT, unpublished data).
The association of the tumour necrosis factor gene promoter SNP TNF-308G/A with SLE is lost after conditioning for HLA-DRB1*0301 in our cohort. If we perform the reverse analysis and condition HLA-DRB1*0301 on the presence of the TNF promoter SNP, we find that the association remains, confirming that our TNF association is secondary to that of HLA-DRB1*0301.<br><br>Genotypic Risk for Class II and Class III Association Signals
Having established independent association at the allelic level with HLA-DRB1*0301 and rs419788-T in our UK SLE cohort, we wanted to further determine the genotypic risk conferred by these variants and hence gain insight into their underlying mode of inheritance in lupus. We used case-control and family-based analyses to assess genotypic risk at HLA-DRB1, while the family-based test alone was used for rs419788. Common family-based tests of LD, such as those used in this study (Genehunter), measure transmission distortion based on allele counts rather than genotype counts; the former has been shown to be more powerful under additive models, while the latter has greater power under recessive or dominant genetic models [27]. The genotype-pedigree disequilibrium test (geno-PDT) determines LD between a locus genotype and disease by comparing genotypes that are transmitted from parent to proband with those that are not [27]. We used the geno-PDT to assess genotypic risk for our class II and class III association signals: HLA-DRB1 and the SNP rs419788. In the case-control analysis for HLA-DRB1, ORs with 95% CI were calculated and Fisher's exact test employed to assess statistically significant differences between HLA-DRB1 genotypes in lupus probands and healthy controls. For HLA-DRB1, the alleles were coded as follows: HLA-DRB1*0301, HLA-DRB1*1501, HLA-DRB1*X where X represents all HLA-DRB1 alleles other than HLA-DRB1*0301, and HLA-DRB1*1501. We included HLA-DRB1*1501 in the analysis, even though we find no allelic association in our cohort, because previous studies have shown a greater risk for lupus in individuals who are compound heterozygotes for HLA-DRB1*0301- and HLA-DRB1*1501-containing haplotypes [25,28]. Overall the results are consistent with a dominant effect from HLA-DRB1*0301 (Table 2) and a dose-dependent (additive) effect from rs419788-T (Table 3). Specifically, both case-control and geno-PDT demonstrate that there is no dose-dependent increase in disease risk for HLA-DRB1*0301. Rather, it appears that the presence of a single copy of HLA-DRB1*0301 alone is sufficient to increase susceptibility to disease. Moreover the 0301/X genotypes constitute the greatest risk in our cohort rather than the 0301/1501 heterozygotes. Genotypes containing HLA-DRB1*1501 in the absence of HLA-DRB1*0301 revealed no significant association in our cohort.
All three rs419788 genotypes demonstrated significant association in our lupus families (Table 3). The common CC genotype was significantly under transmitted, while the rare T allele displayed dose-dependent over transmission to lupus probands.<br><br>Characterization of HLA-DRB1*0301 Risk Haplotype
Next, we wanted to further delimit the MHC class II association signal that we have detected at HLA-DRB1. We used phased parental genotype data to compare the allelic composition of HLA-DRB1*0301-bearing haplotypes that were transmitted (T) to affected probands to those that were not transmitted (or untransmitted, UT) with the aim of identifying differences that could delineate the lupus susceptibility interval(s) arising from this haplotype (summarized in Figures 2A, 2B, and S2). We observed a striking difference between transmitted and untransmitted chromosomes within the class II region: nearly all transmitted HLA-DRB1*0301 haplotypes (99%) are identical across a 180 kb region defined by eight SNPs, whereas the corresponding region within untransmitted HLA-DRB1*0301 haplotypes exhibits significant recombination. These data strongly suggest the existence of a risk haplotype that, interestingly, contains only three expressed genes: HLA-DRB1, HLA-DQA1, and HLA-DQB1. Furthermore, we can confidently define the allelic composition of this risk haplotype, as these three genes are in strong LD and occur in one common haplotype in Caucasians: HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. Thus, we hypothesize that the specific combination of all three alleles is required to confer disease risk in lupus or that disease susceptibility lies with either HLA-DRB1*0301 or the HLA-DQ alleles. We do not have sufficient numbers of recombinant chromosomes in this risk region to further delimit this signal: 2/176 (1.1%) transmitted HLA-DRB1*0301 haplotypes are recombinant at HLA-DQA1-HLA-DQB1; 3/178 (1.7%) transmitted haplotypes identical across HLA-DQA1-HLA-DQB1 do not possess HLA-DRB1*0301.
The composite relative extended haplotype homozygosity (REHH) versus frequency plot for UK SLE; Utah residents with ancestry from northern and western Europe (CEPH); and Yoruba in Ibadan, Nigeria (Yoruba) populations is shown in Figure 3A. We can only comment on evidence for positive selection in CEPH individuals, as we have used this population alone to assess background variation on Chromosome 6. The SLE and Yoruba cohorts are shown for comparative purposes. We find no evidence of positive selection for HLA-DRB1*0301 in the CEPH population. However, this allele is enriched in our lupus cohort (21% of parental chromosomes) and displays greater extended homozygosity when compared with HLA-DRB1*0301-bearing haplotypes in CEPH and Yoruba. Hence, the HLA-DRB1*0301 allele in lupus is observed as an outlier on the plot when compared to background variation in CEPH. These data support our previous observations (outlined above) of the highly conserved nature of HLA-DRB1*0301 haplotypes in lupus. In addition, the haplotype bifurcation plots centered on HLA-DRB1*0301 for UK SLE, CEPH, and Yoruba populations in Figure 3B illustrate preservation of the common HLA-DRB1*0301 haplotype in CEPH and UK SLE, while that seen in the Yoruba is significantly different. The class II regions of all three populations are essentially identical across our chosen SNPs; the main differences lie in class III. The difference in African populations in the class III region is one possible explanation for the lack of evidence for an association between HLA-DRB1*0301 and SLE in African or African American populations. However, HLA-DRB1*0301 has a lower frequency (∼7%–10%) in African populations compared with Europeans (∼13%), and the number of HLA association studies conducted in African populations is very limited.<br><br>Characterization of Class III Region Risk Haplotype
Our data reveal a second independent signal at the MHC in SLE arising from the T allele of SNP rs419788 in intron 6 of the class III gene, SKIV2L. Further evidence supporting the independence of the rs419788-T and HLA-DRB1*0301 alleles is provided by the moderate LD between these two variants (r2 = 0.24) coupled with our data demonstrating that only 47% of rs419788-T allele-bearing haplotypes contain HLA-DRB1*0301.
The structure and composition of T and UT haplotypes anchored at rs419788-T were essentially identical (Figures 2C, 2D, and S2), and hence not informative in delimiting our class III signal. Therefore, we examined the LD structure around our associated class III SNP to better define our disease risk interval. In our lupus dataset the rs419788-T allele resides on three of seven haplotypes present within a large block of six SNPs exhibiting strong LD. This haplotype block encompasses roughly 270 kb containing class III genes from SLC44A4 to AGER, including the RCCX module. Next, we analyzed the haplotype block structure of this region in CEPH families using SNP data dumped from the International HapMap Project (http://www.hapmap.org/). The greater density of SNP typing available in the HapMap CEPH population compared to our current UK SLE map allowed us to potentially refine our signal by exploring correlations between our associated SNP and those surrounding it. Analysis of these data (Figure 4) suggests the presence of short-range LD around our associated variant, rs419788, in CEPH families, encompassing approximately 40 kb of the genome which includes the five genes: complement factor B (CFB), RD RNA binding protein (RDBP), SKIV2L, dom-3 homolg Z (<span class="yellow">C. elegans</span>) (DOM3Z), and serine/threonine kinase 19 (STK19), and does not include the complement C4 locus. Furthermore, assessment of marker association in our lupus dataset demonstrates that after conditioning for HLA-DRB1*0301, the only markers that retain association signals are telomeric of SKIV2L, suggesting that complement C4, which is centromeric to this gene, may not be responsible for our independent class III signal.<br><br>Subphenotype Analysis
In order to gain further insight into disease pathogenesis, we examined common lupus subphenotypes. Such subsets are more homogeneous than lupus per se and thus maybe enriched for specific predisposing variants. In addition, one might expect a close association between MHC class II alleles and autoantibody subsets in lupus if these are indeed causal variants, given their role in antigen presentation and subsequent humoral immunity. We therefore tested our two main MHC association signals, HLA-DRB1*0301 and rs419788, for association with renal disease and autoantibody subsets in our lupus cohort.
We found that HLA-DRB1*0301 was associated with the presence of anti-Ro and anti-La antibodies in our UK SLE cohort, with the latter showing the greatest evidence of association (anti-La nominal p < 0.001 compared with anti-Ro nominal p < 0.025). We found no association of HLA-DRB1*0301 with renal disease or any other autoantibody subsets in our dataset (see Table S4 for detailed results).
Genotypes of the SNP rs419788 were not associated with any of the tested lupus subphenotypes after controlling for the effect of HLA-DRB1*0301 (unpublished data).<br><br>
Discussion
We present the first family-based SNP association study of the MHC in SLE. We have genotyped 69 markers (HLA-DRB1 and 68 SNPs) across 2.4 Mb of the MHC, encompassing class III and class II, in a cohort of 314 UK Caucasian SLE trios. Transmission disequilibrium testing of these data has shown predominant association with the alleles HLA-DRB1*0301 and rs419788-T, together with 12 other MHC SNPs. Moreover, using conditional analyses, we have shown that the two primary signals of association at the MHC are independent of each other. Specifically, one signal arises from HLA-DRB1*0301 in class II and the other from the T allele of SNP rs419788 in the class III gene SKIV2L.
Examination of bifurcation plots for T and UT HLA-DRB1*0301-containing haplotypes has enabled delineation of our class II association signal to a 180 kb region encompassing HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. These data substantially refine that previously published by Graham et al. in 2002 [25], where the lupus susceptibility interval within HLA-DRB1*0301-containing haplotypes could only be delimited to a 1 Mb region encompassing class II and class III. The precise causal variant(s) within this region remains to be determined, as the three implicated alleles exhibit strong LD with few recombination events separating them (two out of 176 transmitted HLA-DRB1*0301 chromosomes in our dataset). However, all three allelic variants represent attractive functional candidates in lupus susceptibility for their role in antigen presentation and stimulation of the adaptive immune response.
Our association of HLA-DRB1*0301 with lupus concurs with published data in Caucasian cohorts and is well established [16]. While our lack of association with HLA-DRB1*1501 and HLA-DRB1*0801 is consistent with previous data from the UK [29], Spain [30], the Netherlands [31], Sweden [32], Mexico [33], and the US [34], it conflicts with that of other US groups [25,35]. Interestingly, we demonstrate a trend, though not statistically significant, for undertransmission of HLA-DRB1*0701—a result also observed in prior UK and Canadian lupus studies [29,36]. Moreover, a negative association of HLA-DRB1*0701 has been reported in other autoimmune diseases including Graves disease [37,38], type 1 diabetes [39], and rheumatoid arthritis [40].
It appears that the conflicting results between UK SLE and previous US (Minnesota [MN]) [25] SLE data stem from differences in HLA-DRB1 allele frequency in the probands of each cohort. The reason for this is unclear. A comparison between UK and MN SLE cohorts (Table 4) reveals that UK SLE cases are enriched for HLA-DRB1*0301 but not HLA-DRB1*0801 or HLA-DRB1*1501 when compared to a UK control population. In contrast, MN SLE cases are enriched for HLA-DRB1*0301-DQB1*0201, DRB1*0801-DQB1*0402, and DRB1*1501-DQB1*0602 inferred haplotypes when compared to MN controls [25]. There is no statistically significant difference in the aforementioned HLA class II alleles/haplotypes between UK and MN control populations that could account for the disparity seen in the respective lupus cohorts. Differences in disease severity and subphenotype frequency between the two populations could account for the observed discrepancy. From the limited data available we found that the presence of renal disease appears to be similar in both cohorts (UK SLE 36% compared with MN SLE 40%), while the gender ratios are significantly different (female: male UK SLE 11:1 compared with MN SLE 57:1, Chi square p value < 0.001). We were unable to compare other lupus subphenotypes. Furthermore, closer inspection of these data reveals that microsatellite-inference of HLA-DRB1 alleles in the MN SLE dataset may underestimate the frequency of HLA-DRB1*0301 and overestimate that of HLA-DRB1*1501, thus diminishing the effect of the former and enhancing that of the latter. It is also possible that the MN SLE cohort shows greater racial heterogeneity in comparison to our UK SLE cohort, despite both being characterized as Caucasian.
Previous studies have demonstrated increased risk for lupus in individuals carrying particular combinations of microsatellite-inferred HLA-DRB1-HLA-DQB1 haplotypes [25,28]. The highest risk genotype was found to be the compound heterozygote HLA-DRB1*0301-DQB1*0201/HLA-DRB1*1501-DQB1*0602, while HLA-DRB1*0301-DQB1*0201-containing genotypes demonstrated a dose-dependent effect in increasing lupus susceptibility [25,28]. In the present study, we have examined genotypic risk at the classically typed HLA-DRB1 locus and in contrast to the aforementioned data of Graham et al. [25,28] we have shown a likely dominant effect of the associated allele, HLA-DRB1*0301. The case-control and family-based analyses for HLA-DRB1 also show the greater power of the former to detect significant association (Table 2). Specifically, all genotypes containing HLA-DRB1*0301 show increased transmission to lupus probands; however, homozygotes show no greater risk compared with heterozygotes, as would be expected under additive or multiplicative models. Thus, a dominant model of inheritance, requiring the presence of a single copy of the disease-predisposing variant alone, likely underlies the susceptibility conferred by HLA-DRB1*0301 in UK SLE. Such a model would fit an antigen presentation hypothesis where susceptible individuals carrying an HLA-DRB1*0301 allele are able to present auto-antigens to CD4+ lymphocytes, thus stimulating an autoimmune response. The differences between our UK SLE and the previously published US SLE data may reflect disease, ethnic, and haplotypic heterogeneity.
Interestingly, analysis of genotypic risk at the associated class III marker, rs419788, suggests an additive (dose-dependent) pattern of inheritance for the rare T allele, where one copy confers a low risk of disease and two copies results in greater susceptibility. The different inheritance patterns for our class II and class III association signals provide further evidence for their independence.
A variety of HLA-DR and HLA-DQ alleles have been associated with autoantibody subsets in ethnically diverse populations of lupus. The strongest associations have been demonstrated between anti-Ro and anti-La antibodies and HLA-DR3 and HLA-DQ2 (HLA-DQB1*0201), which are in strong LD [41–45] in case-control studies. Here, we confirm the association of HLA-DRB1*0301 with anti-Ro and anti-La antibody production in our family-based cohort.
Examination of LD structure around our second independent association, rs419788-T in class III, coupled with the results of our conditional analysis, suggests that this signal could also be delimited to a relatively narrow genomic interval of about 40 kb given further SNP mapping in our cohort. This region includes the genes CFB, RDBP, SKIV2L, DOM3Z, and STK19, but does not include complement C4. Thus, complement C4 null alleles, which have been implicated in lupus pathogenesis, may not be responsible for our class III signal. We conclude, therefore, that our family-based mapping study has potentially revealed a hitherto unknown lupus susceptibility interval in the class III region of the MHC. However, we cannot conclusively exclude association at complement C4/RCCX without direct determination of C4 polymorphism/copy number in our cohort.
With respect to the genes implicated in our study, CFB is a vital component of the alternate complement pathway and disregulation may clearly affect the inflammatory response [46]. RD and Skiv2l are proteins potentially involved in RNA processing. The RD protein forms part of a negative elongation factor (NELF) complex that represses RNA polymerase II transcript elongation, while Skiv2l is a DEAD box protein with possible function as an RNA helicase. The function of Dom3z is currently unknown, although the homologous <span class="yellow">yeast</span> protein binds nuclear exoribonuclease. Moreover, its ubiquitous expression suggests a housekeeping role. STK19 is a protein kinase of unknown function with primary nuclear localization [47]. Interestingly, RDBP and SKIV2L are found to be highly expressed in T lymphocytes, B lymphocytes, and dendritic cells (SymAtlas, http://symatlas.gnf.org/SymAtlas/).
A number of studies have demonstrated conflicting evidence for and against association with various TNF locus polymorphisms in SLE [48]. A recent meta-analysis of the TNF-308G/A promoter polymorphism in SLE [48] revealed evidence of association for the minor allele (A) in European populations; however, this study did not account for LD with class II alleles. On conditioning our dataset for HLA-DRB1*0301, we find that the TNF promoter signal is lost, suggesting that this association is not independent and is due to LD with HLA-DRB1*0301 (or another variant in LD with HLA-DRB1*0301).
In summary, we have found association with two distinct and independent variants within the class II (HLA-DRB1*0301) and class III (SKIV2L) regions of the MHC in UK SLE trios. We can delimit our class II signal in lupus to three genetic variants (HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201) that may confer disease risk in combination or as separate signals. Our class III signal importantly excludes independent association at the TNF promoter polymorphism TNF-308G/A and potentially provides a novel locus for further study.<br><br>Materials and Methods
Study cohorts.
SLE families. The cohort comprises 314 complete SLE trios (that is, mother, father, and affected lupus proband) collected as previously described [49]. All study <span class="yellow">participants</span> are European Caucasian on the basis of grandparental origin. All 314 lupus probands (288 female, 26 male) fulfill the revised American College of Rheumatology (ACR) criteria for SLE [50], 36% of whom have a diagnosis of lupus nephritis. Written consent was obtained from all study <span class="yellow">participants</span> and ethical approval for this study was obtained from the Multi-Centre Research Ethics Committee (MREC 2 June 1998).<br><br>Healthy controls.
The control population for the HLA-DRB1 genotypic risk case-control analysis constitutes 1,667 healthy males of Northern European origin. The individuals are potential hematopoietic stem cell donors and were typed to four digits for HLA-DRB1 at the Anthony Nolan Trust, UK for this purpose. The level of resolution used for the typing of HLA-DRB1*15 alleles in these healthy controls resulted in the ambiguous allele string HLA-DRB1*1501/1502/1504/1506. However, it is likely that the great majority are HLA-DRB1*1501. There is no gender bias in HLA-DRB1 allele frequencies, so although we have used a male control cohort, the frequencies would be expected to be the same in a similar female cohort (Steven Marsh, personal communication).<br><br>SNP genotyping.
Eighty-six SNPs were chosen for genotyping in our mapping study. Specifically, we selected 40 MHC class II and class III haplotype tagging SNPs from a preliminary MHC SNP map [51] that had previously shown robust genotyping efficacy. In addition, we typed 36 MHC class II tag SNPs from a subsequent high-resolution MHC study [52]. We also included the TNF-308G/A promoter SNP, together with nine further SNPs in the region of HLA-B and MICA obtained from the database, dbSNP (http://www.ncbi.nlm.nih.gov/projects/SNP/). The latter SNPs had not been well characterized. All variants were typed in the entire cohort (n = 942). The SNPs span approximately 2.4 Mb of the MHC from the class I gene, KIAA1949 to the class II pseudogene, HLA-DPB2 and thus encompass MHC class III and class II. SNP genotyping was performed at the Broad Institute of MIT and Harvard and at Imperial College London by matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry using the Sequenom MassARRAY platform as previously described [53]. SNPs that failed Sequenom typing were retyped by KBiosciences (http://www.kbioscience.co.uk/) using their in-house SNP genotyping methodology, KASPar (http://www.kbioscience.co.uk/genotyping/index.htm), a competitive allele-specific PCR technique.<br><br>HLA-DRB1 genotyping.
HLA-DRB1 typing was performed at the Anthony Nolan Trust, UK. All samples (n = 942 UK SLE trios and n = 1,667 controls) were genotyped using LABType SSO (sequence-specific oligonucleotide) typing technology according to the manufacturer's written recommendations (http://www.onelambda.com). Briefly, a locus-specific biotinylated PCR amplicon is produced, denatured, and rehybridized to complementary oligonucleotide probes conjugated to fluorescently coded beads. The bound biotinylated PCR product can be detected using R-phycoerythrin-conjugated streptavidin. A flow analyzer identifies the fluorescent intensity of phycoerythrin on each bead. The assignment of HLA type is based on the reaction pattern compared to patterns associated with known sequences.
High resolution testing was performed using the Dynal AllSet+ SSP (sequence-specific primers) DRB1 assay according to the manufacturer's protocol (Invitrogen) for SLE families only. The presence or absence of PCR amplification was detected in a gel electrophoresis step using visualization by ethidium bromide incorporation. Genotypes were determined using SSPTool software.
Samples that could not be resolved to four digits using SSO and PCR-SSP were analyzed by DNA sequencing of exon 2 of HLA-DRB1. Primers, reagents, and protocols were provided by The Anthony Nolan Trust, UK (primer sequences are available on request). Specific HLA-DRB1 alleles were assigned by comparing the resultant sequence with reference sequence from the IMGT/HLA Database [54].<br><br>Data analysis.
Mendelian inconsistencies were removed using PedCheck [55]. Families in which more than eight markers demonstrated Mendel errors were removed from further analysis. Markers with less than 80% genotyping efficiency and markers where more than eight families showed Mendel errors were also excluded from analysis. Two markers located within the SLE associated class II region (rs2239802 in intron 4 of HLA-DRA and rs6457594 in the region between HLA-DRB9 and HLA-DRB5) show deviation from Hardy–Weinberg equilibrium (HWE), which may reflect an undetected SLE association or systematic genotyping error. HWE was assessed in parental samples in our cohort. There is currently no uniform opinion in the community regarding the inclusion or exclusion of SNPs that show deviation from HWE, hence we elected to include these markers in the final analysis.
Sixty-eight out of the total 86 SNPs passed our quality-control measures (see Table S1 for details). In summary, one SNP was monomorphic in our dataset, four SNPs yielded low genotyping efficiency, and 13 SNPs were excluded for unacceptable Mendel error rate. The mean call rate for all markers post-quality control was 94% (range 83% to 99%).
Family-based association testing was performed using Genehunter TDT (version 2.1) [56] and TRANSMIT (version 2.5) [57]. Haplotypes were constructed and permutation testing performed using Haploview (version 3.32) [58]. Significance of association signals in all analyses was based on permutation testing (10,000 permutations). The data are represented both as nominal and permuted p values. ORs with 95% CI for family-based analyses were calculated in PLINK (http://pngu.mgh.harvard.edu/purcell/plink/) [59]. Conditional regression analyses were undertaken using WHAP [60].
The geno-PDT was performed using PDT version 5.1 with default settings [27]. The HLA-DRB1 alleles were coded into three groups for the geno-PDT and the case-control analysis: HLA-DRB1*0301, HLA-DRB1*1501 and HLA-DRX where HLA-DRX includes all HLA-DRB1 alleles other than HLA-DRB1*0301 or HLA-DRB1*1501. The HLA-DRB1*1501 code in the healthy controls represents the allele string HLA-DRB1*1501/1502/1504/1506, as described previously. The HLA-DRB1*1501 code in the lupus probands represents the alleles HLA-DRB1*1501 (84 out of 91 *1501 and *1502 alleles) and HLA-DRB1*1502 (7/91 *1501 and *1502 alleles), as *1504 and *1506 were not present in this population. Fisher's exact test was used to assess significance of association in the case-control analysis.<br><br>Subphenotype analysis.
We looked for association of the HLA-DRB1*0301 allele with autoantibody subsets and renal disease in our cohort using the Chi-square test. We compared cases with and without the subphenotype of interest with DRB1*0301 homozygosity, heterozygosity, combined homozygosity and heterozygosity, and non-DRB1*0301 status. We performed the same analyses for homozygous and heterozygous genotypes of the associated SNP rs419788. The autoantibody subsets compared were anti-C1q, IgG, and IgM anti-cardiolipin antibodies (ACLG and ACLM), anti-Ro, anti-La, anti-RNP, anti-Sm, and anti-dsDNA.<br><br>Delineation of associated MHC haplotypes and evidence for positive selection.
We looked for positively selected alleles in our dataset using the long-range haplotype test as measured by extended haplotype homozygosity (EHH), previously described by Sabeti et al. [26]. Essentially, such an analysis allows assessment of positive selection by mining datasets for high frequency extended haplotypes in comparison to the other core haplotypes at a locus.
EHH is defined as the probability that two randomly chosen chromosomes carrying the core haplotype of interest will be identical by descent (homozygosity at all SNPs) for the entire interval from the core to a distance x. The REHH is the ratio of the EHH on the tested core haplotype compared with the combined EHH of all the other core haplotypes at the region excluding the tested core; as such, REHH accounts for local variation in recombination rate while EHH does not [26].
The program emphase was employed to assign the phase of parental genotype data and reconstruct missing information. Emphase is a simple phaser similar to the phaser of Excoffier and Slatkin [61]. It is very fast, especially on large datasets, and sufficiently accurate for most genetic applications. EHH analysis was performed on the phased parental data using the software program SWEEP (http://www.broad.mit.edu/mpg/sweep/index.html).<br><br>Haplotype bifurcation plots.
We represent the breakdown of LD on core haplotypes using haplotype bifurcation diagrams generated in the program TREE [62] (also explained in [52]).<br><br>REHH versus frequency plots.
Fifty-three SNPs (identified in Figure 1) are common to our dataset and the CEPH and Yoruba HapMap [63] populations. These three datasets, together with CEPH SNP data for Chromosome 6 in its entirety, were used to generate separate REHH versus frequency plots in SWEEP. The plots from the four cohorts were combined for visual, not statistical, comparison. Evidence for positive selection was quantitatively assessed in CEPH individuals, as this population alone was used to assess background variation on Chromosome 6. The UK SLE and Yoruba cohort data are shown for comparison. The 95th percentile based on total CEPH Chromosome 6 SNP data is shown.<br><br>
Supporting Information
Accession Numbers
The Online Mendelian Inheritance in <span class="yellow">Man</span> (OMIM, http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim) accession numbers for the genes described in this study are as follows: AGER, 600214; BAT3, 142590; C4A, 120810; C4B, 120820; CFB, 138470; DOM3Z, 605996; DOM3Z, 605996; EHMT2, 604599; HLA-B, 142830; HLA-DPB2, 120290; HLA-DQA1, 146880; HLA-DQB1, 604305; HLA-DRA, 142860; HLA-DRB1, 142857; KIAA1949, 610990; MICA, 600169; NOTCH4, 164951; RDBP, 154040; RDBP, 154040; SKIV2L, 600478; SLC44A4, 606107; STK19, 604977; STK19, 604977; TNF, 191160; and TNXB, 600985.<br><br><br><br></body></html>