
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Mycobacterium tuberculosis</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA2257940</h3>Discovery of genes implicated in whirling disease infection and resistance in <span class="yellow">rainbow trout</span> using genome-wide expression profiling
Abstract
Background
Whirling disease, caused by the pathogen <span class="yellow">Myxobolus cerebralis</span>, afflicts several salmonid species. <span class="yellow">Rainbow trout</span> are particularly susceptible and may suffer high mortality rates. The disease is persistent and spreading in hatcheries and natural waters of several countries, including the U.S.A., and the economic losses attributed to whirling disease are substantial. In this study, genome-wide expression profiling using cDNA microarrays was conducted for resistant Hofer and susceptible Trout Lodge <span class="yellow">rainbow trout</span> strains following pathogen exposure with the primary objective of identifying specific genes implicated in whirling disease resistance.<br><br>Results
Several genes were significantly up-regulated in skin following pathogen exposure for both the resistant and susceptible <span class="yellow">rainbow trout</span> strains. For both strains, response to infection appears to be linked with the interferon system. Expression profiles for three genes identified with microarrays were confirmed with qRT-PCR. Ubiquitin-like protein 1 was up-regulated over 100 fold and interferon regulating factor 1 was up-regulated over 15 fold following pathogen exposure for both strains. Expression of metallothionein B, which has known roles in inflammation and immune response, was up-regulated over 5 fold in the resistant Hofer strain but was unchanged in the susceptible Trout Lodge strain following pathogen exposure.<br><br>Conclusion
The present study has provided an initial view into the genetic basis underlying immune response and resistance of <span class="yellow">rainbow trout</span> to the whirling disease parasite. The identified genes have allowed us to gain insight into the molecular mechanisms implicated in salmonid immune response and resistance to whirling disease infection.<br><br><br><br>Background
Whirling disease was first described among farmed <span class="yellow">rainbow trout</span> (<span class="yellow">Oncorhynchus mykiss</span>), a native North American salmonid species, introduced to Germany as a food fish in the late 1800s [1]. Whirling disease is associated with systemic infections by the myxozoan <span class="yellow">Myxobolus cerebralis</span>, a parasite with presumed origins among salmonid fish in Eurasia [2,3]. <span class="yellow">Rainbow trout</span> are highly susceptible to whirling disease and the introduction of the parasite to the U.S.A. in the 1950s had immediate economic impacts on salmonid hatcheries in both eastern and western states [2]. The parasite has a broad worldwide distribution and has been identified in 25 states in the U.S.A. where salmonid fish are present [4]. Salmonid hatcheries throughout the U.S.A have suffered drastic economic losses due to whirling disease outbreaks. The disease has more recently been recognized as the principal cause of major population declines among wild <span class="yellow">rainbow trout</span> populations in the intermountain region of the U.S.A with serious negative impacts on sportfishing and allied industries [5-7]. Additionally, concerns continue over the potential negative ecologic impacts of whirling disease on wild salmonid populations, particularly threatened or endangered salmonids such as <span class="yellow">bull trout</span> (<span class="yellow">Salvelinus confluentus</span>), <span class="yellow">cutthroat trout</span> (<span class="yellow">Oncorhynchus clarki</span>), and <span class="yellow">steelhead</span> (<span class="yellow">Oncorhynchus mykiss</span>) [8,9].
<span class="yellow">Myxobolus cerebralis</span> has a complex life cycle that includes two alternate hosts, a salmonid fish and an oligochaete worm, <span class="yellow">Tubifex tubifex</span> [10,11]. Infection in the salmonid host begins when microscopic waterborne actinospore stages of <span class="yellow">M. cerebralis</span> are released from the worm and contact the skin of the fish host. Actinospores, also referred to as triactinomyxons for <span class="yellow">M. cerebralis</span>, attach preferentially to fins and the buccal cavity where they release one or more of three coiled polar filaments which penetrate and then anchor them to the epidermis[12]. Within minutes the sporoplasm, which contains 64 internal cells, migrates from the triactinomxyon to deeper layers of the epidermis, an action that may be facilitated by parasite proteases [13-15]. Aggregates and single cells from the sporoplasm then begin mitotic replication within two h of initial infection, alternating between inter and intracellular locations, a process that may also depend upon parasite coded protease activity [15,16]. Over the next 10 h at water temperatures of 15Â°C, parasites within host epithelial cells further divide by the process of endogeny or cell within cell replication prior to release and then penetration of new host cells. Between 12 and 20 h post infection, the number of parasite cells present in the epidermis steadily declines until new stages are observed in the subcutis at 48 h. Degenerative stages observed in the epidermis between 12 and 20 h are suspected to be a result of the action of the host immune response, although the cellular and or humoral factors involved are not currently known. After a brief residence in the subcutis, parasite stages are presumed to migrate to proximal nervous tissues, initially in peripheral and then more central locations [16]. Migration and potential replication of parasite stages in nervous tissue ensues over the next 16 d with the first parasites exiting to invade cartilage observed at 20 d post infection [16]. Feeding on cartilage may induce a host inflammatory response that constricts the spinal cord, brain stem, and caudal nerves resulting in the erratic swimming behavior (whirling) and black tail observed among fish with acute whirling disease [17]. An additional impact of cartilage destruction are permanent deformities to the skeletal system that may increase vulnerability to predation and impair ability to forage for food [1]. The final developmental stages of the parasite in the fish are environmentally resistant spore stages (myxospores) which remained trapped in cartilage or bone [18]. Death of infected fish or ingestion by fish or avian predators releases myxospores from the fish tissues and they may be ingested by the second host, the benthic dwelling oligochaete <span class="yellow">T. tubifex</span> [19]. A second developmental cycle then occurs under the mucosal lining of the intestine that results in the release of thousands of the actinospore (triactinomyxon) stages potentially over the entire lifetime of the individual oligochaete [20].
Susceptibility to whirling disease in U.S. <span class="yellow">rainbow trout</span> strains is pervasive with only two of the tested native strains displaying any degree of resistance, which may be inconsistent and relatively moderate [21,22]. Hatchery <span class="yellow">rainbow trout</span> in Germany (Hofer strain), however, have acquired a degree of resistance to whirling disease that is consistently much higher than any domestic rainbow strains and comparable to that of <span class="yellow">brown trout</span> (<span class="yellow">Salmon trutta</span>), which are native to Europe and typically asymptomatic following infection [23]. Laboratory tests comparing <span class="yellow">rainbow trout</span> strains under the same environmental conditions and pathogen exposure indicate that the Hofer strain's ability to combat <span class="yellow">M. cerebralis</span> infection has a genetic basis. Recently, controlled crosses of the Hofer strain and a susceptible strain (Colorado River <span class="yellow">rainbow trout</span> [CRR]) demonstrated that resistance to whirling disease was inherited by progeny [24] and heritability estimates are currently underway.
The discovery of the resistant Hofer strain allowed us to conduct an intraspecific comparison of susceptible and resistant <span class="yellow">rainbow trout</span> in order to gain insight into the genetic basis underlying whirling disease susceptibility for this species. Gene expression profiling, through the use of microarrays, is an extremely high-throughput method to discover specific genes and pathways involved in a disease phenotype without the bias of a candidate gene approach.
In this study, microarray analysis was used to examine expression changes in a resistant and susceptible strain of <span class="yellow">rainbow trout</span> following exposure to <span class="yellow">M. cerebralis</span>, the pathogen causing whirling disease. We have found several genes significantly up-regulated in both the resistant and susceptible strain that appear to be involved in host response to infection. We have also found a gene which is significantly up-regulated in the resistant but remains unchanged in the susceptible <span class="yellow">rainbow trout</span> strain following pathogen exposure that is a likely candidate gene for involvement in conferring whirling disease resistance.<br><br>Results and Discussion
Quantitative PCR conducted on caudal fin tissues at two hours post exposure to <span class="yellow">M. cerebralis</span> demonstrated each fish strain had similar initial pathogen loads, although there was substantial variation between individual fish within each strain. The mean parasite copy numbers per host cell were 1.20 Ã 106 (SD 1.53 Ã 106) for the Hofer and 1.04 Ã 106 (SD 0.91 Ã 106) for the Trout Lodge. These mean values and standard deviations are similar to those obtained in additional studies of susceptible <span class="yellow">rainbow trout</span> when examined at early time points post TAM exposure (unpublished data).
In order to study genes involved in whirling disease response, resistant and susceptible <span class="yellow">rainbow trout</span> strains were exposed to <span class="yellow">M. cerebralis</span> and RNA from skin tissue was converted to cDNA and hybridized onto microarrays. Relative gene expression for exposed and unexposed controls for each strain was compared and the list of differentially expressed genes for both strains is found in Table 1. A combined total of 17 genes or features (14 annotated genes, 3 unknown features) were differentially expressed in one or both strains following pathogen exposure and are involved with <span class="yellow">rainbow trout</span> infection response to whirling disease exposure. Several of these genes were found in different locations on the array as unique expressed sequence tag (EST) clones and their repeated presence on the significance gene lists provides additional support for their involvement in the whirling disease phenotype. The small number of genes found potentially indicates that only a few genes contribute to the phenotypic differences found between resistant Hofer and susceptible Trout Lodge, at least in terms of differential gene expression, during early disease progression in the skin. In the microarray statistical analysis, when the delta value was adjusted even slightly lower, the FDR estimate increases from 0% to ~78%. Since increasing the FDR cut-off to such a high percentage would dramatically reduce power, we chose to leave the gene list small with an estimated FDR of 0%. This type of dramatic increase in FDR estimation is additional support that there are not many genes differentially expressed in response to whirling disease infection for our chosen tissue and time points.
Different salmonid microarray platforms, such as those available from Oregon State University and Michigan State University, or different tissues and time points may produce additional candidate genes. A recent time course study used a candidate gene approach to identify four genes (TGF-Î², IL-1Î²1, IL-1Î²2, and COX-2) that were significantly up-regulated by both Hofer and Trout Lodge in response to whirling disease infection [25]. These genes and their downstream effectors were not identified in the current microarray study, likely due to many differences in experimental design between the two studies (e.g., pathogen exposure levels, tissue types, water temperatures, age of fish at exposure, etc.). While downstream effectors of these genes were present on the microarray, only one of the four genes (COX-2) was actually present on the microarray. It is our hope that future genome sequencing will enable the construction of more comprehensive microarray platforms for economically important aquaculture species, such as <span class="yellow">Atlantic salmon</span> and <span class="yellow">rainbow trout</span>.
All significant genes identified by the current microarray study were up-regulated following pathogen exposure for one or both strains. Therefore, it appears that both strains are undergoing transcriptional activation to defend against whirling disease infection and thus, are exclusively employing positive regulation for the genes examined in skin during early disease progression.
The normal caveats that apply for microarray studies (gene discovery is limited by transcripts on arrays, differences at transcriptional level may not cause phenotypic differences, results are dependent upon tissue type and time point chosen, etc.) apply for this study. Additionally, the comparison of two <span class="yellow">rainbow trout</span> strains (i.e., resistant versus susceptible) added another layer of complexity to the analysis. We chose to not directly compare the two strains because there could be expression differences between them, due to divergence following strain isolation, that are unrelated to the whirling disease phenotype. With that in mind, the two strains were first compared entirely separately from each other to discover expression differences in response to pathogen exposure for each strain. Only the genes responding to infection, and therefore implicated in the whirling disease phenotype, were compared between the two strains for differential gene expression (Figure 1). A limitation of this approach to our study is that constitutively expressed transcripts which are differentially expressed between the two strains that contribute to the whirling disease phenotype cannot be identified.
Microarray analysis of genes differentially expressed in the resistant Hofer strain in response to pathogen exposure
A total of 16 genes or features (13 annotated genes, 3 unknown features) were up-regulated in the resistant Hofer strain following pathogen exposure. All 13 annotated genes have been previously implicated in host immune response for other infectious diseases. <span class="yellow">Viral Hemorrhagic Septicemia Virus</span> (<span class="yellow">VHSV</span>) induced protein and neighbor of COX-4 are the only annotated genes without known molecular functions.
A common link between the majority of annotated genes with known molecular functions is an involvement in the interferon system. The interferon system is one of the first lines of host defense against invading pathogens for vertebrates (for review see [26]), including <span class="yellow">teleost fish</span> (for review see [27]). Other economically important salmonid pathogens, such as <span class="yellow">infectious pancreatic necrosis virus</span> and <span class="yellow">infectious salmon anaemia virus</span> have been found to activate both type I and type II interferon (IFN) responses in the <span class="yellow">Atlantic salmon</span> host following infection [28]. Interferons are cytokine proteins that are secreted following infection and play a critical role in both innate and adaptive immunity. The IFN system has been most widely researched in mammals and studies have found that type I IFN (mammalian IFN-Î±/Î²) are secreted by the pathogen-infected cells as part of a rapid initial immune response while Type II IFN (mammalian IFN-Î³) is secreted by natural killer (NK) and T cells and plays a more central role in the second wave of immune response. To cope with the myriad of host infections, the interferon system is highly complex and involves the regulation of hundreds of genes [29,30]. Specifically, type I IFN acts to increase MHC class I expression for antigen presentation [31], promote T cell survival [32], inhibit cell proliferation [33], mediate apoptosis [26], and increase NK cell activity [34]. Type II IFN acts to increase both MHC class I and II expression for antigen presentation [29], stimulate macrophages to kill engulfed pathogens [35], induce apoptosis [36], and regulate leukocyte-endothelium interactions [37] in addition to many other immune-related activities.
It is informative to examine the functional roles of each gene's encoded protein specifically to better understand the part each plays, both individually and as interconnected components, in host immune response. Expression of the interferon-induced 35 kDa protein is induced by IFN and it is involved in cytokine signalling [38]. Interferon regulatory factor 1 (IRF-1) and interferon regulatory factor 7 (IRF-7) are transcription factors that induce expression of IFN responsive genes [39,40]. Additionally, IRF-1 is involved in apoptosis and cell cycle regulation related to tumor suppression [41]. Similarly, cyclin-dependent kinase 4 inhibitor B (p15-INK4b) plays a role in apoptosis[42], cell cycle regulation [42], and tumor suppression [43] and can be induced by the cytokine TGF-Î² [44]. Gig2 is an interferon-inducible protein that is likely part of the JAK-STAT signal transduction pathway [45]. Ubiquitin and the proteasome subunit beta type 8 precursor are both members of the ubiquitin-proteasome system (for review see [26]), which serves to degrade proteins via proteolysis. These degraded proteins can originate from an invading pathogen and are displayed on MHC class I proteins. The beta-2-microglobulin is an integral component of MHC class I proteins and is therefore involved in antigen processing and presentation to cytotoxic T cells [46]. Haptoglobin binds hemoglobin and limits its availability to infectious bacteria, thus preventing bacterial proliferation in a wound [47]. The PPAR-Î±-interacting complex protein 285 is a transcriptional co-activator with helicase activity [48] and has sequence similarity to a <span class="yellow">rainbow trout</span> <span class="yellow">VHSV</span>-induced protein. Gene expression of metallothionein B (MT-B) is induced by several metal ions [49], cytokines [50-52], and stress hormones [53-55]. MT proteins are believed to play diverse functional roles in inflammation, immune response, apoptosis, tumor suppression, and detoxification (for reviews see [55,56]).<br><br>Microarray analysis of genes differentially expressed in the susceptible Trout Lodge strain in response to pathogen exposure
A total of six genes or features (five annotated genes, one unknown feature) were up-regulated in the susceptible Trout Lodge strain following pathogen exposure. Only one of the significant genes for Trout Lodge, which has sequence similarity to CC chemokine SCYA113, was not also differentially expressed in Hofer in response to pathogen exposure. The CC chemokine SCYA113 gene is a member of the CC chemokine family, which guides leukocytes to sites of infection and inflammation (for review see [57]). The fewer number of significant genes found for Trout Lodge relative to Hofer may indicate a decrease in transcriptional activation for this susceptible strain. There is, however, likely some degree of overlap in both strains' response to pathogen exposure due to the fact that several genes were up-regulated in both Hofer and Trout Lodge (i.e., ubiquitin-like protein 1, IRF-1, and PPAR-Î±-interacting protein Gig2). A critical phase in the early stages of <span class="yellow">M. cerebralis</span> infection in trout is invasion and intracellular replication, processes that begin as early as one hour post exposure to triactinomyxons [16]. A role for accumulated ubiquinated proteins in the lysosome in the killing of <span class="blue">Mycobacterium tuberculosis</span> has recently been described that has implications for a range of intracellular infections [58] and some similar responses to infection may be occurring for both resistant and susceptible strains.<br><br>Microarray analysis of genes differentially expressed between resistant and susceptible strains in response to pathogen exposure
Of the genes differentially expressed in response to pathogen exposure for both strains, only metallothionein B shows a statistically significant difference in expression between the two strains (Table 2). MT-B was found to be up-regulated in the resistant Hofer strain following pathogen exposure but remained unchanged in the susceptible Trout Lodge strain.
As previously noted, metallothionein has been implicated in a broad range of functional capacities, including inflammatory and immune responses. Several cytokines can induce metallothionein expression including IFN [59-61], interleukin-1 [50], interleukin-6 [51], and tumor necrosis factor-Î± [52]. Metallothionein has been shown to mediate leukocyte chemotaxis and has been hypothesized to serve as an early "danger signal" during times of stress or infection to activate an immune response [62]. The functional similarities between metallothionein and CC chemokine SCYA113, at least in terms of leukocyte chemotaxis, are certainly of interest since these genes displayed quite distinct expression profiles. Metallothionein was up-regulated in the resistant Hofer strain and CC chemokine SCYA113 was up-regulated in the susceptible Trout Lodge strain (although CC chemokine SCYA113 did not pass the significance cut-off to be considered differentially expressed between the two strains). This distinction between two genes, capable of similar biological roles, may indicate that leukocyte movements to, and their activities once at, the infection site are key factors in determining resistance versus susceptibility to whirling disease. Evaluations by light microscopy and qPCR for <span class="yellow">M. cerebralis</span> genomic DNA of Hofer and Trout Lodge <span class="yellow">rainbow trout</span> exposed to triactinomyxons demonstrates Hofer more efficiently eliminates invading parasites in the skin (M. Adkison, pers. comm.). While the parasite effectively penetrates the epidermis in both strains, significantly fewer parasites survive the migration from the skin to the nerves as evaluated at 10 d post exposure. A role for host immune factors in the elimination of invading parasites, even in susceptible <span class="yellow">rainbow trout</span> strains, is suggested by several prior light and electron microscopy studies that demonstrate an increase in degenerative stages in the skin beginning as early as 12 h and then their elimination by 24 h post-exposure to triactinomyxons [12,16,63].
The difference in metallothionein expression may be due to an alternative immune response pathway since the protein has known involvement in diverse functional capacities. For instance, metallothionein's role as a zinc-finger transcriptional regulator [64] may dramatically alter the expression profiles between resistant and susceptible <span class="yellow">rainbow trout</span>. All biological roles of this diverse protein should be considered when examining the complexities of host immune response. Additionally, upstream regulators of metallothionein expression could be the true underlying cause of the whirling disease phenotype since a gene expression study alone cannot determine if a gene is directly contributing to a phenotype (i.e., cause versus downstream effect).<br><br>Validation of microarray results by qRT-PCR
Quantitative RT-PCR (qRT-PCR) confirmed the microarrays results for two of the genes up-regulated in both Hofer and Trout Lodge following infection, ubiquitin and IRF-1, and the metallothionein gene (MT-B), which was up-regulated in Hofer but remained unchanged in Trout Lodge following infection (Figure 2). The qRT-PCR results for IRF-1 and metallothionein were quite similar to the microarray results for each gene, in terms of relative expression changes in response to infection. MT-B was found to once again be significantly up-regulated in the resistant Hofer strain following pathogen exposure but remained unchanged in the susceptible Trout Lodge. This difference in MT-B gene expression between the two strains was statistically significant (P ~ 0.001). The relative degree of up-regulation for ubiquitin following pathogen exposure was considerably higher in the qRT-PCR (~9 â 17 fold greater up-regulation in qRT-PCR versus microarrays). Many other studies have also observed this pattern of greater sensitivity in qRT-PCR versus microarray results (for examples see [65,66], which is often attributed to the more gene-specific optimized conditions of the qRT-PCR approach.
Given the high degree of statistical support and biological relevance of the candidate genes, we believe this study provides initial insight into <span class="yellow">rainbow trout</span> genes and pathways responding to whirling disease infection and identifies the first candidate genes for whirling disease resistance.<br><br>Potential future studies
While the interferon system appears to be a likely candidate system for further study, many of the significant genes are found in alternative pathways and have distinct roles and functions in other systems. Furthermore, it is increasingly apparent that epistatic interactions and the interplay between pathways/networks previously classified as discrete can have enormous phenotypic effects on quantitative traits [67]. Multiple avenues of research should be examined in future studies, using the candidate genes presented here as an initial guide, due to the complex relationships between hosts and pathogens. For instance, the migration of leukocytes and their subsequent activity in the skin are likely a critical part of the early immune and inflammatory host response after pathogen infection. Additionally, it is quite feasible that the difference in metallothionein expression is due to an alternative immune response pathway since the protein has known involvement in diverse functional capacities. For instance, metallothionein's role as a zinc-finger transcriptional regulator [64] may dramatically alter the expression profiles between resistant and susceptible <span class="yellow">rainbow trout</span>. All biological roles of this diverse protein should be considered when examining the complexities of host immune response. Finally, upstream regulators of metallothionein expression could be the true underlying cause of the whirling disease phenotype since a gene expression study alone cannot determine if a gene is directly contributing to a phenotype (i.e., cause versus downstream effect). The expression profiles of a variety of metallothionein upstream regulators, such as cytokines and metal transcription factor (MTF-1), could be evaluated in a time course study during early disease progression to identify additional candidate genes. A QTL mapping approach could also be used to identify particular chromosomal regions directly contributing to the disease phenotype.<br><br>
Conclusion
The present study has provided the first examination into the genetic basis underlying <span class="yellow">rainbow trout</span>'s immune response and resistance to the whirling disease pathogen. Several genes were significantly up-regulated in skin following pathogen exposure for both the resistant Hofer and susceptible Trout Lodge <span class="yellow">rainbow trout</span> strains. For both strains, response to infection appears to be linked with the interferon system. Metallothionein B is differentially expressed between the resistant and susceptible strains and is a good candidate for future whirling disease resistance studies. The identified genes have allowed us to gain initial insight into the molecular mechanisms involved in a salmonid host's immune response and resistance to whirling disease infection.<br><br>Methods
Animal care, pathogen exposure, and RNA preparation
Hofer and Trout Lodge <span class="yellow">rainbow trout</span> strains were reared in 35 gallon aquaria with 15Â°C flow-through well water for nine weeks post-hatch, with each fish weighing approximately 6.5 grams prior to pathogen exposure. Individuals from each strain (n = 60) were exposed to 2,000 triactinomyxons (TAMs) per fish for one hour. Additional fish (n = 60) from both strains served as unexposed controls, which were treated identically to exposed fish at all experimental stages other than their lack of pathogen exposure. Fish were then kept under standard aquaculture conditions until euthanized. TaqMan PCR for the quantitative evaluation of genomic parasite DNA was employed to confirm that fish in both the Hofer and Trout Lodge groups received equal amounts of parasite exposure. At two hours post TAM exposure, 6 fish in each exposed group were removed and euthanized with an overdose of benzocaine at a concentration of 500 mg/L. Caudal fins were removed posterior to the peduncle and used as the tissue for a quantiative TaqMan assay following procedures described by Kelley et al. [68].
Microarray studies examining skin four hours after pathogen exposure did not identify any genes differentially expressed between Trout Lodge and Hofer strains (data not shown). Therefore, we chose a later time point (24 hours after exposure) so that early host immune response was more likely to be fully underway and significant expression changes could be detected. After the 24 hour incubation period, all fish were euthanized with an overdose of benzocaine at a concentration of 500 mg/L. Each fish was euthanized individually and the caudal fin (largely comprised of skin tissue) was removed posterior to the peduncle. The fin was immediately placed into 2Ã Nucleic Acid Purification Lysis Solution supplied with ABI's TransPrep Chemistry kit (Applied Biosystems, Foster City, CA) to stop further gene expression changes. Total RNA was extracted from the fin of each individual using the ABI Prismâ¢ TransPrep system with the ABI Prismâ¢ 6100 Nucleic Acid PrepStation according to manufacturer instructions. RNA quality was assessed by agarose gel electrophoresis and RNA concentrations were measured using a ND-1000 spectrophotometer (NanoDrop Technologies, Wilmington, DE).
Starting total RNA yields were not sufficient for microarray hybridizations due to the small amount of caudal fin tissue present on these young fish. Therefore, 250 â 1000 ng of total RNA was used as the starting material to create amplified RNA (aRNA) indirectly labeled with Cy3 or Cy5 fluorescent dyes (GE Healthcare, Buckinghamshire, UK) using the Amino Allyl MessageAmpâ¢ II aRNA Amplification kit according to manufacturer instructions (Ambion, Austin, TX).<br><br>Microarray hybridization and data analysis
Salmonid cDNA microarrays (GRASP16k v2.0) were obtained from consortium for Genomic Research on Atlantic Salmon (cGRASP) and details of microarray development and fabrication can be found in von Schalburg et al. [69]. These arrays contain 13,421 <span class="yellow">Atlantic salmon</span> and 2,576 <span class="yellow">rainbow trout</span> cDNA features and have been successfully used for several previous <span class="yellow">rainbow trout</span> gene expression studies [70-73]. For each <span class="yellow">rainbow trout</span> strain, competitive hybridization was conducted on every array using equal amounts (8 Î¼g) of differentially labeled aRNA from one control fish and one exposed fish. Four biological replicates were performed for each experimental condition and dye-sample coupling was swapped between biological replicates in a balanced block design.
Prehybridization washes for all microarrays included: 2 Ã 5 min in 0.1% SDS, 5 Ã 1 min in NANOpure H2O with 0.5 mM dithiothreotol, 1 min in near boiling nanopure H2O, centrifugation for 2 min at 1500 RPM. To reduce background, the microarrays were next incubated for 90 min in 5 Ã SSC, 0.1% SDS, 3% BSA (Fraction V) at 49Â°C, washed 3 Ã 20 s in nanopure H2O, and dried by centrifugation for 5 min at 1500 RPM. The labeled aRNA samples were competitively hybridized to microarrays prewarmed to 49Â°C for 16 hours in a formamide-based buffer (Genisphere, Hatfield, PA) with LNA dT blocker (Genisphere). Posthybridization washes for all microarrays included: 1 Ã 10 min in 2 Ã SSC, 0.1% SDS prewarmed to 49Â°C, 2 Ã 5 min in 2 Ã SSC, 0.1% SDS at room temperature, 2 Ã 5 min 1 Ã SSC at room temperature, 2 Ã 5 min 0.1 Ã SSC at room temperature. Slides were then dried by centrifugation and immediately scanned using an Agilent G2565BA Microarray Scanner (Agilent Technologies, Santa Clara, CA).
Data underwent local background subtraction and LOWESS normalization using Agilent's Feature Extraction software. Raw and processed gene expression data have been deposited into the NCBI Gene Expression Omnibus [74] (series GSE8631) and are in compliance with MIAME guidelines. The Significance Analysis of Microarrays (SAM) software package [75] was used to identify differentially expressed genes between exposed and unexposed control fish for each <span class="yellow">rainbow trout</span> strain. Both a Wilcoxon rank sum and a modified t-test were conducted with 1,000 permutations and the minimum fold change cut-off was set to 2.0 up- or down-regulated. A false discovery rate (FDR) of 0.00% was estimated for both strains. To determine statistically significant differences between the Hofer and Trout Lodge strains, a Welch's t-test (P-value < 0.01) was implemented in Microsoft Excel between the log ratios (exposed/control) for each strain for all genes that were significant for at least one strain in the SAM program.<br><br>Quantitative RT-PCR
Microarray expression results were validated by qRT-PCR for several identified genes. Prior to qRT-PCR, 80 ng of total RNA was reverse transcribed from each biological replicate used for the microarray study along with two additional samples (total n = 6 per experimental condition) using the QuantiScript Reverse Transcriptase kit (Qiagen, Valencia, CA) according to manufacturer instructions. In contrast to the microarray experiments, the template RNA was not amplified before cDNA synthesis. EST clone sequences from the cGRASP microarray were used to design primers for genes undergoing validation, along with a Î²-actin reference gene used for normalization, with Primer3 software [76] and the sequence for each primer pair is shown in Table 3. The Quantitectâ¢ SYBRÂ® Green RT-PCR kit (Qiagen) was used according to the manufacturer's instructions except the final PCR volume was reduced to 25 Î¼l. The PCR conditions used on a Chromo4 Real Time PCR Detection System (Bio-Rad, Hercules, CA) were as follows: HotStarTaq DNA polymerase activation at 95Â°C for 15 min, 45 cycles of 15 s denaturation at 94Â°C, 30 s annealing at 58Â°C, 30 s extension at 72Â°C, followed by a melting curve to ensure that a single PCR product was produced for each reaction. For each gene, the relative amount of gene expression was calculated using the ÎÎCT method [77] and significance was determined using a nonparametric Mann-Whitney U test and multiple linear regression in JMP.<br><br>
Authors' contributions
MRB participated in study conception and design, conducted gene expression experiments and data analysis, and drafted the manuscript. ABW participated in study design, data analysis, and manuscript revision. RPH and BPM participated in study conception and design, supervision of research activities, and manuscript revision. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2575403</h3>Foamy Macrophages from Tuberculous <span class="yellow">Patients</span>' Granulomas Constitute a Nutrient-Rich Reservoir for <span class="yellow">M. tuberculosis</span> Persistence
Abstract
Tuberculosis (TB) is characterized by a tight interplay between <span class="blue">Mycobacterium tuberculosis</span> and host cells within granulomas. These cellular aggregates restrict bacterial spreading, but do not kill all the bacilli, which can persist for years. In-depth investigation of <span class="yellow">M. tuberculosis</span> interactions with granuloma-specific cell populations are needed to gain insight into mycobacterial persistence, and to better understand the physiopathology of the disease. We have analyzed the formation of foamy macrophages (FMs), a granuloma-specific cell population characterized by its high lipid content, and studied their interaction with the tubercle bacillus. Within our in vitro <span class="yellow">human</span> granuloma model, <span class="yellow">M. tuberculosis</span> long chain fatty acids, namely oxygenated mycolic acids (MA), triggered the differentiation of <span class="yellow">human</span> monocyte-derived macrophages into FMs. In these cells, mycobacteria no longer replicated and switched to a dormant non-replicative state. Electron microscopy observation of <span class="yellow">M. tuberculosis</span>âinfected FMs showed that the mycobacteria-containing phagosomes migrate towards host cell lipid bodies (LB), a process which culminates with the engulfment of the bacillus into the lipid droplets and with the accumulation of lipids within the microbe. Altogether, our results suggest that oxygenated mycolic acids from <span class="yellow">M. tuberculosis</span> play a crucial role in the differentiation of macrophages into FMs. These cells might constitute a reservoir used by the tubercle bacillus for long-term persistence within its <span class="yellow">human</span> host, and could provide a relevant model for the screening of new antimicrobials against non-replicating persistent mycobacteria.<br><br>Introduction
Tuberculosis caused by <span class="blue">Mycobacterium tuberculosis</span> (<span class="yellow">M.tb</span>) remains one of the leading causes of mortality in the world, with around 2 million deaths each year [1]. Most individuals remain asymptomatic after the primary infection with only 10% at risk of developing an active disease during their life [2]. In asymptomatic individuals, the bacilli are not cleared but rather persist in a dormant state, from which they may reactivate and induce clinical disease at later stages [3].
The prognosis of the disease depends on the host's efficiency to constrain the bacilli at the site of infection. When inhaled <span class="yellow">M.tb</span> reach the lungs, they are internalized by lung macrophages. The latter trigger the accumulation at the infectious site of macrophages, lymphocytes and dendritic cells, to form a granuloma, which is a major histo-pathological feature of TB. Within granulomas, macrophages differentiate into epithelioÃ¯d cells (differentiated macrophages), and/or fuse to form multinucleated giant cells (MGC). Macrophages with large numbers of lipid-free vacuoles, as well as macrophages filled with lipid-containing bodies, also called foamy macrophages (FM) are also found within granulomatous structures in both experimental animal models and <span class="yellow">human</span> disease [4],[5]. The above cells are surrounded by a rim of lymphocytes, and at later stages, a tight coat of fibroblasts encloses the structure [6]. Although the structure and cell composition of granulomas are well known, the biology of these inflammatory structures and, more specifically, the role of granuloma-specific cell types, remain largely unknown.
We have previously developed an in vitro model of <span class="yellow">human</span> tuberculous granulomas to gain insight into the survival strategies of the tubercle bacillus within its <span class="yellow">human</span> host. This model now enables the characterization of granuloma-specific cell types, and their modulation by <span class="yellow">M.tb</span> [7]. The main advantage of this model over in vivo animal models or ex vivo <span class="yellow">human</span> biopsy samples, is the availability of live granuloma cells which facilitates analysis of their cell biology. Using this model we have recently shown that, within granulomas, large multinucleated giant cells, also known as Langhans giant cells, result from the induction of granuloma macrophage fusion by <span class="yellow">M.tb</span> glycolipids [8]. We have shown that these cells have lost the ability to mediate bacterial uptake upon maturation, but have conserved their ability to mediate antigen presentation [9].
The differentiation of macrophages into FMs has been particularly well described in individuals developing a postprimary, also known as secondary or adult, TB. These postprimary infections are considered to be the result of re-infection or reactivation of a primary TB [10]. FMs have been described in leprosy <span class="yellow">patients</span> or <span class="yellow">M. avium</span>-infected AIDS (Acquired ImmunoDeficiency Syndrome) <span class="yellow">patients</span>, and in chronic stages of <span class="yellow">M.tb</span> infection in <span class="yellow">mice</span> [4],[11],[12]. The foamy aspect of these macrophages is the result of intracellular lipid accumulation within lipid bodies, also called lipid droplets or lipid vacuoles [13],[14],[15]. In an experimental model of leukocyte infection, it was recently suggested that BCG (Bacille Calmette Guerin) infection can induce, in a TLR2-dependent fashion, the rapid formation of lipid bodies carrying out part of the eicosanoÃ¯d biosynthesis that usually accompanies the infection, thus pointing to an active role for lipid bodies during the course of infection [16]. However, the mechanisms regulating this lipid accumulation during mycobacterial infection and their significance in the physiopathology of tuberculosis are not understood. Most of the studies on TB granulomas have focused on the contribution of host components, but very little is known about the role played by bacterial constituents in terms of granuloma formation and progression.
The present work was aimed at deciphering the role of FMs in <span class="yellow">M.tb</span> survival within <span class="yellow">human</span> granulomas. To test our working hypothesis according to which FMs constitute a nutrient-rich reservoir for <span class="yellow">M.tb</span> persistence, we used our in vitro model of <span class="yellow">human</span> granulomas to analyze the formation of FMs and their role during <span class="yellow">M.tb</span> infection. We showed that only highly virulent mycobacteria (<span class="yellow">M.tb</span>, <span class="yellow">M. avium</span>) and not saprophytic ones (<span class="yellow">M. smegmatis</span>) could induce the formation of FMs in mature granulomas. Moreover, we demonstrated that oxygenated mycolic acids specifically produced by the above pathogenic species were responsible for FMs formation. Once differentiated, FMs were unable to mediate phagocytosis of new bacilli and their microbicidal activity was reduced. <span class="yellow">M.tb</span> was not killed in FMs but instead persisted in a non-replicating state, and over-expressed dormancy genes. Noteworthy, in foamy macrophages, <span class="yellow">M.tb</span>-containing phagosomes were shown to migrate towards lipid bodies which they progressively surrounded and engulfed. As a result, bacteria were freed into lipid bodies, thus favoring the bacilli's access to nutrients. From these data, we propose that FMs could form a secure reservoir for the tubercle bacilli.<br><br>Methods
<span class="yellow">Human</span> samples
<span class="yellow">Human</span> blood samples, purchased from the French National Blood provider of Toulouse, were collected from fully anonymized non-tuberculous control donors, an ethical committee approval was, therefore, not necessary. This study was conducted according to the principles expressed in the Helsinki Declaration, with informed consent obtained from each donor.
We chose to work on lymph node samples rather than lung biopsies, which are usually only paraffin-embedded, because staining for lipids can only be performed on frozen samples. Lymph node biopsies were taken for diagnosis purposes, in ten non-<span class="yellow">HIV</span> <span class="yellow">patients</span>. For each biopsy, a fragment was sent to the microbiology laboratory, another was frozen in liquid nitrogen and the main part was fixed in formalin and paraffin-embedded for histological examination. <span class="yellow">M.tb</span> was identified in 9 lymph node biopsies and from the lung aspiration in the last <span class="yellow">patient</span>. This latter case showed no signs of necrosis, and no FMs were found in the lymph node biopsy. This study complies with the guidelines of the declaration of Helsinki.<br><br>Bacterial strains and culture conditions
Wild-type <span class="yellow">M. smegmatis</span> and <span class="yellow">M. smegmatis</span>/hma strains were previously described [17], <span class="yellow">M. tuberculosis</span>-GFP were a kind gift from Dr. C. Guilhot (CNRS-IPBS, Toulouse France). Bacilli were grown in Middlebrook 7H9 medium (Difco) supplemented with 10% albuminâdextroseâcatalase (Difco). Fluorescent <span class="yellow">M. smegmatis</span> and <span class="yellow">M. smegmatis</span>/hma were obtained by FITC labelling as described in [18].<br><br>Isolation of RNA from intraphagosomal <span class="yellow">M.tb</span>
Six and 12 days post-infection (MOI 10), macrophages and FMs (5Ã106) were washed twice with PBS, scraped off the cell dishes and recovered by centrifugation. The cell pellets were lysed with lysis buffer (RNEasy mini kit, Quiagen) and transferred to 2 ml Eppendorf-tubes containing a 0.5 ml suspension of 0.1 mm-diameter glass beads (Biospec). Mycobacteria were disrupted using a bead beater (Retsch) followed by a 5 min centrifugation at 14 000g. RNA contained in the supernatant was then column-purified according to the manufacturer's conditions using the RNEasy mini kit (Qiagen) and quantified.<br><br>Quantitative Real-Time RT-PCR
In RNA samples DNA contamination was excluded by DNAse I treatment (Ambion). 1 Âµg total RNA was reverse-transcribed using random hexamer primers (Ambion) and Superscript III reverse transcriptase (Invitrogen). Real-time PCR was performed on cDNA using the SYBR green essay (Applied Biosystems). Reverse and forward primers used are listed below in Table 1. Fluorescence was measured by ABIPrism 7300 (Applied Biosystems). The calculated threshold cycle (Ct) value for each gene of interest was normalized to the Ct value for 16S and the fold expression was calculated using the formula: fold changeâ=â2âÎ.(ÎCt) [19]. Real-time PCR conditions include initial activation at 94Â°C for 5 min, followed by 40 cycles of denaturation at 94Â°C for 30 sec, annealing and extension at 65Â°C for 1 min. The gene induction ratios were obtained by comparing gene expression levels in intracellular bacilli with those of log-phase in vitro-grown bacilli. RNAs were isolated from two independent macrophage infections.<br><br>In vitro <span class="yellow">human</span> granuloma formation
In vitro granulomas were obtained as previously described [7]. Briefly, 1Ã106 freshly isolated Peripheral Blood Mononuclear Cells (PBMCs) were incubated with 1Ã104 viable <span class="yellow">M.tb</span>, or 1Ã103 viable <span class="yellow">M. smegmatis</span> or <span class="yellow">M. smegmatis</span>/hma. The culture medium was RPMI-1640+Glutamax (Difco), containing 7.5% <span class="yellow">human</span> AB serum (Sigma-Aldrich).<br><br>Macrophage differentiation
2.5Ã106 PBMCs prepared in RPMI-1640+Glutamax (Difco) were plated over coverslips in 24-well plates. After 2 h culture at 37Â°C, cells were washed 3 times with PBS and then refed with RPMI-1640+Glutamax (Difco), containing 7.5% <span class="yellow">human</span> AB serum. After 6 days of culturing, macrophages were differentiated.<br><br>Respiratory burst assay with Nitroblue tetrazolium (NBT)
<span class="yellow">Human</span> monocyte derived macrophages were stimulated with <span class="yellow">M. smegmatis</span>/hma for 2 h at 37Â°C, washed and reincubated in mycobacterium-free medium. After 2 days of differentiation into FMs, macrophages were co-stained with NBT (2 mg/ml Sigma-Aldrich) and Nile red (Sigma-Aldrich) for 30 min at 37Â°C. Stained cells were fixed and then analysed with an inverted microscope (Nikon TE 300).<br><br>Phagocytosis and Survival test
For phagocytosis assays, differentiated macrophages were incubated with <span class="yellow">M. smegmatis</span>/hma mycolic acids for 2 days and then infected with 1Ã108 labeled mycobacteria per well for 90 min, washed 3 times with PBS and chased for 3 h in fresh culture medium.
For survival experiments, macrophages were infected with <span class="yellow">M. tuberculosis</span>-GFP (10 bacteria/cell), washed 3 times with PBS (Gibco) and re-incubated in fresh culture medium. At selected time points thereafter (1, 3, 6, 10 and 14 days) cells were labeled with Nile red (Sigma-Aldrich), fixed and observed under a confocal microscope. The amount of mycobacteria per cells was evaluated. (100 cells were analyzed for each time point).<br><br>Lipid body staining and immunostaining
Granuloma cells were collected and plated onto glass coverslips with a cytospin (Thermo Shandon) fixed for 30 min in PBS-PFA 4% and stained with Oil red-O (Sigma-Aldrich) as described [20]. The slides were then counterstained with haematoxylin (Dako Cytomation) and observed under an inverted microscope (Nikon TE 300).
For fluorescence analysis, granuloma cells or macrophages were collected in PBS, lipid bodies were stained with Nile red (Sigma-Aldrich, 0.1 Âµg/ml, from a stock solution in methanol) for 15 min washed with PBS, fixed for 30 min in PBS-PFA 4%, mounted with the fluorescent mounting medium (Dako Cytomation) and observed under a confocal microscope.
In order to distinguish the lipids contained within lipid bodies, from those of the cell membrane, we used the fluorescent emission spectrum properties of Nile red which depend upon the kind of lipid associated with Nile red, i.e. for triacylglycerol: Î»max emâ=â590 nm, for phospholipids: Î»max emâ=â640 nm (Molecular Probes handbook). On confocal microscopy pictures, the phospholipid background of both macrophages and FM appears in red and the triacylglycerol-rich lipid bodies appear in white. Cells were considered to be positive for Nile red staining when more than 50% of the cell surface was stained (see Figure S2).<br><br>Mycolic acid isolation
Bacterial residues obtained after lipid extraction with organic solvents [17] were saponified with a mixture of 40% KOH aqueous solution and methoxyethanol (1â¶7, v/v) at 110Â°C for 3 hours in a screw-capped tube. After acidification, fatty acids were extracted with diethylether, derivatised into methyl esters with diazomethane and analyzed by analytical thin-layer chromatography on silica Gel 60 (Silica Gel 60 Macherey-Nagel) using either dicholoromethane or petroleum ether/diethylether (9â¶1, v/v, five runs). Visualization of lipid spots was performed by spraying the plates with molybdophosphoric acid (10% in ethanol), followed by charring.<br><br>Processing for electron microscopy
Granulomas were fixed for 1 hour at room temperature with 2.5% glutaraldehyde in 0.1 M cacodylate buffer, pH 7.2, containing 0.1 M sucrose, 5 mM CaCl2 and 5 mM MgCl2. After two successive 15-min washes with the same buffer, the granulomas were postfixed for 1 hour at RT with 1% osmium tetroxide (Electron Microscopy Science) in the same buffer devoid of sucrose. The granulomas were scraped off the culture dishes with a rubber policeman and concentrated in 1% agarose in the same buffer. After a one hour treatment at room temperature with 1% uranyl acetate in Veronal buffer, the samples were dehydrated in a graded series of ethanol and embedded in Spurr resin. Thin sections were stained with uranyl acetate and lead citrate.<br><br>Image acquisition in confocal microscopy
The images were obtained using a Leica confocal fluorescence microscope (SP2) equipped with a Plan Apo 40Ã1.4 Ph 6 objective (Olympus Optical) and CoolSNAP-Pro CF digital camera in conjunction with Image-Pro Plus version 4.5.1.3 software (Media Cybernetics). The images were edited using Adobe Photoshop CS2 9 software (Adobe Systems).<br><br>
Results
Foamy macrophages are strongly associated with necrotic lesions and often contain <span class="yellow">M. tuberculosis</span>
We analyzed lymph node biopsies from 10 tuberculous <span class="yellow">patients</span> as a first step for evaluating the role of FMs within tuberculous granulomas. A section through a representative biopsy is shown in Figure 1. Well-circumscribed and -differentiated granulomatous structures were observed in all the samples (Figure 1A). Classically, lesions display a necrotic center (N), an interface area between the necrotic center and the histiocytes (I), and some peripheral granulomas (G). Only seven out of ten <span class="yellow">patients</span> presented lesions displaying central necrosis (Table 2). Staining of the histology samples with Oil red-O, a classic lipid stain, confirmed the presence of FMs within the granulomatous structures in six out of seven samples presenting necrosis (Figure 1B), whereas no FMs were found in the three non-necrotic lesions (Table 2). Noteworthy, in samples with necrotic areas, FMs were always found in the interface region flanking the central necrosis (Figure 1B). These observations firstly confirmed the presence of FMs in most TB <span class="yellow">patients</span>' lesions thereby suggesting that these cells play an important role in the formation/maintenance of such lesions. Second, FMs seem to be associated with necrosis, which is a hallmark of TB lesions, since they were observed only in lesions with a necrotic center and preferentially located around the necrotic area.
Interestingly, staining, in parallel, of serial thin sections from a <span class="yellow">patient</span>'s lesion biopsy with Oil red-O (Figure 1C) and Ziehl Nielsen (Figure 1D), showed that most of the bacilli (arrow) were located in the same area as FMs, thus suggesting a strong association between the persisting tubercle bacilli and FMs within granulomas.<br><br><span class="yellow">M. tuberculosis</span> induces the formation of FMs within in vitro <span class="yellow">human</span> tuberculous granulomas
To further characterize the role of FMs in the granulomatous response, we assessed whether FM formation in granulomatous structures was triggered only by pathogenic mycobacterial species (<span class="yellow">M.tb</span>), or by low virulent ones (<span class="yellow">M. smegmatis</span>) as well. PBMCs from non-tuberculous control individuals were infected with <span class="yellow">M.tb</span> or <span class="yellow">M. smegmatis</span>, following the procedure previously described for the induction of granulomatous structures ([7],[9] and Figure S1). Granuloma cells collected at days 3 and 11 were stained with Oil red-O to visualize the lipid droplets within FMs under the light microscope (Figure 2). At day 3, several <span class="yellow">M.tb</span>-induced granuloma cells already showed lipid bodies (Figure 2A). In contrast, the cells collected from <span class="yellow">M. smegmatis</span>-induced granulomas were seldom (5%) positively stained (Figure 2B). Interestingly, <span class="yellow">M. avium</span> induced FM formation in a similar way to <span class="yellow">M.tb</span> (not shown). By day 11, the amount of positively stained cells had increased in <span class="yellow">M.tb</span>-induced granulomas, but not in <span class="yellow">M. smegmatis</span>-induced ones. In addition, the number of lipid bodies per cell increased dramatically with time, as depited in the enlarged views (Figures 2A, C). The quantitative evaluation of the percentage of FMs within granulomas induced by both strains confirmed the differences observed under the light microscope, and showed a seven-fold difference (44% vs 6% respectively) between <span class="yellow">M.tb</span> and <span class="yellow">M. smegmatis</span> in terms of their ability to induce FM formation (Figure 2D). Our results therefore show that virulent species such as <span class="yellow">M.tb</span> and <span class="yellow">M. avium</span>, contrary to poorly or avirulent ones such as <span class="yellow">M. smegmatis</span>, are able to induce the formation of FMs within our experimental model.<br><br>Oxygenated mycolic acids induce the maturation of macrophages into FMs
Mycolic acids from <span class="yellow">M.tb</span> incorporated into liposomes were recently shown to trigger the differentiation of <span class="yellow">mice</span> peritoneal macrophages into foamy-like cells [21]. Interestingly, both <span class="yellow">M.tb</span> and <span class="yellow">M. avium</span>, which induce FM formation, express a family of oxygenated mycolic acids, especially ketomycolic acids, which are not produced by <span class="yellow">M. smegmatis</span> (Figure 3A). In this context, inactivation of the <span class="yellow">M.tb</span> hma gene (mmaA4-Rv0642c) was shown to abolish the synthesis of oxygenated keto- and hydroxyl-mycolic acid in the mutant strain [17]. Conversely, transforming <span class="yellow">M. smegmatis</span> with the hma gene induced the production of both keto- and hydroxyl-mycolic acids [22], (Figures 3B, C).
In the light of both data, we anticipated that oxygenated mycolic acids specifically produced by <span class="yellow">M.tb</span> and <span class="yellow">M. avium</span>, under the control of hma, are responsible for FM formation within <span class="yellow">human</span> granulomas. To test this hypothesis, we compared FM formation after infection of PBMCs with either the wild-type, or the hma-expressing <span class="yellow">M. smegmatis</span> strain (<span class="yellow">M. smegmatis</span>/hma). Granulomas cells were collected 3 days later and stained with Oil red-O to visualize lipid bodies. Wild-type <span class="yellow">M. smegmatis</span>-induced granulomas displayed only 5.5% of FM, whereas the hma gene-expressing strain induced granulomas bearing a majority (67%) of brightly stained Oil red-O positive cells (Figures 4A, B, C). Induction of FM formation was even greater if isolated macrophages were directly infected with either strain. After only 4 hours of infection, <span class="yellow">M. smegmatis</span>/hma had already transformed 64% of the infected macrophages into lipid body-positive cells (see Figure S2 for Nile red positive cells), whereas only 9% of the cells contained lipid bodies after infection with the wild-type strain (Figures 4D, E, F). To confirm the specific role of hma-dependent oxygenated mycolic acids in FM formation, and to rule out a possible combined effect of oxygenated mycolic-acids with other mycobacterial components, mycolic acids isolated from wild-type <span class="yellow">M. smegmatis</span> or <span class="yellow">M. smegmatis</span>/hma were incubated with isolated macrophages. With mycolic acids isolated from the wild-type strain, only 13% of the macrophages were transformed into FM whereas 66% of the macrophages incubated with mycolic acids isolated from <span class="yellow">M. smegmatis</span>/hma were strongly stained for lipid bodies (Figures 4G, H, I).
These results therefore indicate that oxygenated mycolic acids play a leading role in <span class="yellow">M.tb</span>-induced FM formation.<br><br>The phagocytic and bactericidal activities are arrested in FMs
To assess the function of granuloma FM, we first evaluated the ability of such cells to mediate phagocytosis. For this purpose, macrophages isolated from PBMCs were exposed to <span class="yellow">M. smegmatis</span>/hma-derived mycolic acids to induce FM formation. Two days later, the cell population contained a mixture of FM (50â70%) and macrophages (30â50%), as assessed by Nile red staining (not shown). The mixed cell population was infected with FITC-labelled <span class="yellow">M. smegmatis</span>. Intracellular bacilli were found only within Nile red-negative macrophages thereby indicating that FMs are unable to ingest bacteria (Figure 5A). This result was reproduced using other mycobacterial strains, such as <span class="yellow">M.tb</span> and <span class="yellow">M. bovis BCG</span>, for infection (not shown). These results further suggest that the bacilli found in granuloma FMs were internalized by macrophages prior to their transformation into FMs.
To assess whether FMs are able to develop a respiratory burst, which is a major intracellular bactericidal activity, the ability of Nile red positive cells (i.e. FMs) to mediate NBT reduction was determined. As shown in Figure 5D, only 8% of the NBT-positive cells (Figure 5B) were Nile red positive FMs (Figure 5C). This strongly suggests that once macrophages have differentiated into FMs, they lose the ability to mediate intracellular bactericidal activity. We postulate that FMs could, therefore, form a secure reservoir for the tubercle bacilli.<br><br><span class="yellow">M. tuberculosis</span> persists in a dormant non-replicative state in FMs
To evaluate the validity of the above hypothesis, we analyzed the ability of <span class="yellow">M.tb</span> to replicate within FMs. For this purpose, isolated macrophages were infected with <span class="yellow">M.tb</span>. At selected intervals post-infection, the amount of bacilli per cell was compared in both non-differentiated macrophages and FMs (Figure 5E). Until day 6, <span class="yellow">M.tb</span> replicated in a similar fashion in both cell types. In contrast, after day 6 post-infection, the amount of bacilli remained stationary in FMs, whereas it continued to increase in macrophages. It is interesting to note that arrest of bacterial replication coincided with completion of macrophage differentiation into FM, i.e. starting from day 3 post-infection. Our data suggest, that the bacilli found in granuloma FMs were internalized by macrophages prior to their differentiation into FMs and also that bacilli can terminate their replication cycle while macrophages are being transformed into FM, but that replication comes to a halt as soon as the maturation process is complete.
We next determined whether the non-replicative bacilli observed in FMs were still alive. For this purpose, we analyzed the expression of a series of genes known to be up-regulated when bacilli are in a persistent non-replicating state [23],[24]. RNA was, therefore, prepared from both in vitro-grown <span class="yellow">M.tb</span> and intracellular bacilli at day 6 and 12 post-infection. The respective amounts of RNA corresponding to isocitrate lyase, Î±-cristallin, a very hypothetical 7.6 kDa protein, CHP and DosR proteins were then quantified by RT-PCR. As shown in Table 3, the dormancy genes were all strongly up-regulated in intracellular bacilli at day 12 post-infection. These results further demonstrate that the bacilli are not killed in FMs, but rather persist in a dormant, and therefore non-replicative stage [25]. Interestingly, the dormancy genes were not as strongly expressed at day 6, time at which bacteria were still able to replicate in macrophages undergoing differentiation into FMs.<br><br>Characterization of <span class="yellow">M.tb</span> survival within FMs and interaction with lipid bodies
To gain further insight into the morphological appearance of <span class="yellow">M.tb</span> within FM and into the interactions between FM lipid bodies (LB) and <span class="yellow">M.tb</span>-containing phagosomes, granulomas were fixed and processed for conventional electron microscopy at days 3 and 11 post-infection.
Whatever the time point at which granuloma cells were observed, bacteria were all enclosed in phagosomes, most of which contained a single bacterium. None of them (over a thousand which were examined under the electron microscope) were free in the cytoplasm and only one was enclosed in a classical autophagic vacuole. At day 3 post-infection, FMs profiles (thin sections) were scarce, representing at most 9% of the total population of macrophage profiles observed under the electron microscope (Figure 6A). In addition, 86% of the FM profiles displayed at most 5 small LBs (Figure 6B). At this stage of granuloma formation, bacteria were infrequent in FM, but were found in other types of macrophages. One of these displayed large numbers of vacuoles containing flocculent material and often one or two LBs. Over 95% of the bacteria located in the different granuloma macrophages were morphologically intact, and therefore alive [26]. Intact bacteria present no breaks in the cell wall or cytoplasmic membrane and their cytoplasm has preserved its ultrastructural organization and electron opacity. Furthermore, they display no electron translucent intracytoplasmic lipid inclusions (ILI). Bacteria were also observed in between cells, probably as a result of cell lysis within granulomas. These bacteria were also morphologically intact and devoid of ILI (not shown).
At day 11 post-infection, <span class="yellow">M.tb</span>-containing macrophages displaying large numbers of vacuoles with flocculent material were less frequently observed. Interestingly, the amount of such vacuoles had strongly decreased in most of these cells while the number of LBs had increased. The percentage of FM had increased to reach 41% of the total population of macrophage profiles within the granulomas (Figure 6A). From these observations, it is tempting to assume that the highly vesiculated macrophages give rise to FMs. Within FMs, the size and amount of LBs had also increased with time since 48% of the FM profiles now displayed more than 5 LB per FM thin section (Figure 6B), randomly distributed within the cells (Figure 6C). About 30% of the FM profiles displayed between 1 and 20 bacteria, which were morphologically intact and enclosed within phagosomes (Figure 6C, enlarged view).
The interaction between these bacteria and the cellular LBs was next examined at day 11. Sixty percent of the bacilli were scattered throughout the FMs and displayed no obvious signs of interaction with the cellular LBs. A small fraction of the bacteria (21%), however, were observed in the close vicinity of cellular LBs. The membrane of the phagosomes in which they were enclosed clearly interacted with cellular LBs (Figures 6C, 7A, 7B, arrows) and became tightly apposed to an increasingly larger surface area of the LB. As a result, the phagosomes started to surround LBs in a zippering fashion (Figures 7E, F). Ultimately, bacilli (19%) were translocated to cellular LBs (Figure 7C). From these observations, it is tempting to assume that <span class="yellow">M.tb</span>-containing phagosomes engulf cellular LBs rather than fusing with them. This process, which is reminiscent of autophagy, resulted in the transfer of free bacteria into the lumen of cellular LBs (Figure 7C), some of which displayed up to 21 bacteria (Figure 7G). Interestingly, only altered <span class="yellow">M.tb</span> found within FM lipid bodies exhibited electron translucent ILIs (Figures 7D, G), thereby suggesting that they are able to accumulate host cell lipids. In previous work, the term altered bacteria had been used to define live bacteria that had acquired ILIs [26]. Since the presence of ILIs within the cytoplasm of <span class="yellow">M.tb</span> is typical for non-replicating bacteria in a state of dormancy [27], this further confirms that these bacteria are dormant.<br><br>
Discussion
Studies carried out several decades ago suggested that postprimary tuberculosis starts as a lipid pneumonia [28],[29]. Indeed, following a first inflammatory process leading to exudates of mononuclear cells within alveolar spaces, early pathologists observed an accumulation of lipid droplets in alveolar macrophages of TB <span class="yellow">patients</span>. Tubercle bacilli were shown to reside in these lipid-rich macrophages which were named foamy macrophages (FMs) [30]. Recently, histo-pathological analysis of biopsies from <span class="yellow">patients</span> with untreated tuberculosis confirmed the century-old histological descriptions of postprimary tuberculosis [10]. In this study, Hunter et al showed that postprimary tuberculosis begins as a lipid pneumonia with the accumulation of large amounts of lipid-rich FMs, accompanied by bronchial obstruction. It was also shown that in alveolar foamy macrophages, the bacilli were mainly found within lipid droplets. All these observations underline the important, yet often neglected, role of lipid accumulation, and more precisely FM formation, at the infectious site in the physiopathology of TB.
In <span class="yellow">murine</span> experimental models, FMs accumulate within the outermost layer of granulomatous structures occupying the alveolar spaces during the chronic phase of infection. This strongly suggests that FMs could be involved in lesion cleaning via phagocytic uptake of cellular debris generated by the local inflammatory response. Once filled with debris, FMs would leave the parenchyma through the alveolar spaces up to the superior bronchial tree, to be finally swallowed and digested in the stomach [31]. In fact, this process is very well known, and is a crucial factor for TB diagnosis in <span class="yellow">children</span>. As <span class="yellow">infants</span> do not usually generate cavitary lesions, and because it is difficult to detect bacilli in the sputum, the diagnosis is linked to the detection of bacilli in the gastrointestinal lavage [32].
We show that <span class="yellow">M.tb</span>-induced the transformation of in vitro-grown <span class="yellow">human</span> granuloma macrophages into FMs within 6 days, and even more rapidly (3â4 hours) in cultured macrophages. Although this event occurs more quickly than in vivo, or in animal models, our data are consistent with the above in vivo observations. Within FMs, bacilli and LBs were often tightly linked, to the point that a non-negligible amount of bacteria were ultimately observed within LBs. Interestingly, some of the bacilli transferred into lipid bodies displayed their own intracytoplasmic lipid inclusions, which are considered to be one of the hallmarks of non-replicating (dormant) <span class="yellow">M.tb</span> [27]. The recent observation of persistent ILI-containing tubercle bacilli within adipocyte LBs [33] is in good agreement with our observations. Since bacilli residing in phagosomes that do not interact with cellular lipid bodies do not display ILIs, it is tempting to assume that lipids within ILIs are of cellular origin. The accumulation of lipids within bacilli [47], via interaction with FM lipid bodies could, therefore, be crucial to <span class="yellow">M.tb</span> persistence. It is indeed known that <span class="yellow">M.tb</span> accumulates lipids, and more precisely triacylglycerols, during dormancy [34],[35] from which it derives both carbon and energy for its own metabolism. Intracellular persistence of <span class="yellow">M.tb</span> is also critically linked to the acquisition of host cholesterol through the Mce4 transporter system [36].
The question that arises is how do bacteria gain access to lipids from LB? Direct fusion of phagosomes with FM lipid bodies seems unlikely as the membranes of both structures are quite different from one another. Our observations suggest instead that once <span class="yellow">M.tb</span>-containing phagosomes have established close contact with a lipid body, they surround and engulf the latter by a process that remains to be deciphered. This phenomenon is somewhat reminiscent of autophagy, as observed under conditions of cholesterol depletion in macrophages infected with <span class="yellow">M. avium</span> [37]. After degradation of the resulting inner membrane, bacteria would be freed within the lipids of the engulfed LB, and therefore be in direct contact with cellular lipids. How bacilli translocate the cellular lipids to their own cytoplasm remains to be established.
Another important phenomenon underlined by our study is the strong correlation between the presence of FMs in the granulomatous structures and the development of necrosis within the lesion, as suggested by Pagel over 80 years ago [29]. Interestingly, FMs were systematically located at the interface region between the histiocytes and the central necrosis area of the biopsied lesions. Although necrosis formation could depend on an indirect effect of the global immune response, our data indicate that the formation of FMs is an important factor favoring the appearance of necrosis. Analysis of larger series of biopsy samples are, however, needed to definitely demonstrate our actual hypothesis according to which FMs play a direct and unique role in necrosis formation. Consistent with this hypothesis, we observed that FMs induced from <span class="yellow">M.tb</span>-infected macrophages displayed permanent TNF-Î± secretion, a potent pro-necrotic factor, whereas <span class="yellow">M. smegmatis</span>-infected macrophages were poor producers of TNF-Î±. At day 4 post-infection, TNF-Î± secretion was indeed twofold higher in macrophages (of which 70% had differentiated into FMs) infected with <span class="yellow">M.tb</span> than in those infected with <span class="yellow">M. smegmatis</span>, as measured both by ELISA and RNA quantification (Peyron, unpublished observations). However, one must keep in mind that the association of FMs and necrosis may be the consequence of the FM cleaning process of lipoproteins released into the necrotic tissue, as observed in atherosclerosis lesions [38]. It is thus tempting to propose that <span class="yellow">M.tb</span> mycolic acids may be responsible for the development of necrotic lesions, due to their ability to induce TNF-Î± production by FMs. Whether mycolic acids are directly involved in TNF-Î± production, or only indirectly by inducing FM formation, remains a matter of debate currently under study.
Our observations are strikingly similar to the phenomena described for postprimary tuberculosis, that seems to begin as localized foci of pneumonia followed by massive necrosis leading to the formation of pulmonary cavities [10]. If this proves to be the case, then the traditionally admitted phenomenon of cavitation arising from the erosion of caseating granulomatous structures from bronchi can be ruled out [39]. In our study, we successfully induced FM formation from isolated macrophages infected with <span class="yellow">M.tb</span>, i.e. outside a granulomatous structure, which is consistent with Hunter's recent proposal.
Until now, mycolic acids have been considered to be indirectly involved in virulence mechanisms as being part of complex molecules of the mycobacterial envelope. The most widely studied mycolic acid-containing mycobacterial compound trehalose 6,6â² dimycolate (TDM), has been extensively analyzed for its role in virulence since the mid-fifties [40]. Recently, it was shown to interfere with the host granulomatous response [41]. Overall, TDM was mainly shown to mediate macrophage activation and a Th1-type response to <span class="yellow">M.tb</span> infection (for review, see [25]).
Our results demonstrate a direct role of oxygenated mycolic acids for FM formation, independently from the appearance and stage of the disease. <span class="yellow">M.tb</span>-specific mycolic acids indeed trigger the transformation of both isolated and granuloma macrophages, into FM. Given the absence of FM formation in <span class="yellow">M. smegmatis</span>-induced granulomas, despite the induction of a comparative inflammatory response, ascertained by the similar induction of granulomas, this phenomenon clearly depends upon a direct contact with the bacilli, and not to the inflammatory response. Mycolic acids are major and hallmark components of the mycobacterial cell wall. They constitute 40â60% of dry weight of the envelope [42]. All members of the complex (e.g. <span class="yellow">M.tb</span>, <span class="yellow">Mycobacterium africanum</span>, <span class="yellow">Mycobacterium bovis</span> and <span class="yellow">Mycobacterium microti</span>) are able to synthesize the same combination of mycolic acids, i.e. cyclopropanated Î±-mycolic acids, ketomycolic and methoxymycolic acids [17], which are not synthesized by non-pathogenic mycobacterial species [43]. These structural specificities probably account for part of the pathogenicity of these species, as shown by the impaired virulence of mutant strains deprived of keto and methoxyl groups in experimental infections [17],[44],[45].
Our study, therefore, gives the first proof of a direct role of isolated mycolic acids in the interplay between <span class="yellow">M.tb</span> and host cells. Interestingly, this effect is expressed both by whole bacilli and isolated lipids, suggesting that oxygenated mycolic acids are either secreted by the bacilli, or exposed at the cell wall surface in a manner enabling their bioactivity. According to our results, mycolic acids trigger the formation, within granulomas, of FMs in which bacilli can hide and survive. Oxygenated mycolic acids, either free, as constituents of TDM [44], or linked to the cell wall arabinogalactan [46],[47], should, therefore, be considered as major virulence factors enabling <span class="yellow">M.tb</span> survival for long periods of time in a persistent state. Being an inducer of host lipid accumulation, and FM formation at the site of infection, these oxygenated mycolic acids could, therefore, also be responsible for the induction of necrosis within lesions, thus favoring <span class="yellow">M.tb</span> dissemination.
Interestingly, deletion of the mmaA4 (Rv0642c) gene also drastically decreased the ability of <span class="yellow">M.tb</span> to induce the differentiation of macrophages into FMs (data not shown). However, the residual ability of this mutant to induce FMs suggests that other mycobacterial factors might partially trigger the formation of FM. With regard to the mycolic acid methyltransferases, given that (i) mmaA2 (Rv0644c) and mmaA3 (Rv0643c) are pseudogenes in <span class="yellow">M. leprae</span> and (ii) mmaA4 KO present no trans cyclopropanation [45], thus excluding the involvement of the cmaA2 (Rv0503c) gene, we expect that at least pcaA (Rv0470c), which introduces cis-cyclopropane, may play the same role. Consistent with this hypothesis, a pcaA null mutant is unable to persist within infected <span class="yellow">mice</span> [46], thus demonstrating the role of a mycolic acid methyltransferase in the chronic stage of infection.
Overall, our study has shed light on a previously uncharacterized cell population participating in <span class="yellow">human</span> tuberculous granulomas, namely foamy macrophages. We propose that the specific induction of FM by <span class="yellow">M.tb</span> would create a favourable environment for persistent bacteria. In our opinion, FMs could be a safe shelter because they preserve bacilli from a direct contact with granuloma lymphocytes and histiocytes, they lose one of the major macrophage bactericidal activities and they constitute an important source of nutrients for the bacilli thanks to the fatty acids accumulated in their lipid granules.<br><br>Supporting Information<br><br>
</body></html>