
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>rat</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1521150</h3>Peculiarities of carcinogenesis under simultaneous oral administration of benzo(a)pyrene and o-cresol in <span class="yellow">mice</span>.
Abstract
A modifying influence of ortho-cresol (o-cresol) on the carcinogenic effect of benzo(a)pyrene (BaP) with combined oral administration to CC57Br <span class="yellow">mice</span> had been found. During simultaneous administration of o-cresol (1 mg) and BaP (1 mg), the incidence of tumors, the multiplicity of tumors, and the degree of malignancy all increased, but the latency was shortened. When o-cresol was administered before or after BaP (in identical doses), the carcinogenic effect was weakened. When o-cresol (10 mg) and BaP (5 mg) were administered simultaneously, the incidence of malignant tumors was similar to controls receiving BaP only (13.8%), indicating inhibition of carcinogenesis.<br><br><br><br>
 Environnmental Health Perspectives Supplements Vol. 101 (Suppl. 3): 341-344 (1993) <br><br> Peculiarities of Carcinogenesis under Simultaneous Oral Administration of <br><br> Benzo(a)Pyrene and o-Cresol in <span class="yellow">Mice</span> <br><br> by N. Ya. Yanysheva,' N. V. Balenko, 1. A. Chernichenko,1 and V. F. Babiy' <br><br> A modifying influence of ortho-cresol (o-cresol) on the carcinogenic effect of benzo(a)pyrene (BaP) with combined oral administration to CC57Br <span class="yellow">mice</span> had been found. During simultaneous administration of o-cresol (1 mg) and BaP (1 mg), the incidence of tumors, the multiplicity of tumors, and the degree of malignancy all increased, but the latency was shortened. When o-cresol was administered before or after BaP (in identical doses), the carcinogenic effect was weakened. When o-cresol (10 mg) and BaP (5 mg) were administered simultaneously, the incidence of malignant tumors was similar to controls receiving BaP only (13.8%), indicating inhibition of carcinogenesis. <br><br> Introduction <br><br> Numerous experiments have shown a modifying influence of different chemicals on benzo(a)pyrene (BaP)induced carcinogenic effects. Both enhancing or weakening effects have been seen (1-12). In some experiments ubiquitious environmental carcinogenic and toxic chemical pollutants were used (13-17). <br><br> The stimulating effect of phenol, nitrogen oxides, and sulfur dioxide upon BaP's blastogenic action on the respiratory tract and phenol on the digestive tract (forestomach) have been reported (12-18). We have established the relationship between the doses of carcinogenic and toxic agents and their modifying effects (17,18). The maximal enhancing effect was observed after BaP (2.5 mg) and NO2 (0.87 mg/m3) were exposed to <span class="blue">rat</span> respiratory tract. The effect weakened with decreasing dose. At concentrations at the MPC level (MPC BaP, 1 ng/m3, MPC NO2, 0.04 mg/m3) no effect was seen. Phenol at 1 mg orally to CC57Br <span class="yellow">mice</span> enhanced the BaP (1 mg) effect, but no effect was seen at 0.02 mg. Phenolic compounds showed either enhancing or inhibiting effects on carcinogenesis depending on their chemical structure (16,18-25). <br><br> This report presents the results of combined oral administration of BaP and artho-cresol (o-cresol) in <span class="yellow">mice</span>, <br><br> 'Ukrainian State Medical Center for Environmental Health, Kiev, Ukraine. <br><br> Address reprint requests to N. Y. Yanysheva, Ukrainian State Medical Center for Environmental Health, Popudrenko St. 50, 253660 Kiev-94, Ukraine. <br><br> chemicals commonly found in ambient water because of their industrial use (coke chemistry, oil processing, shale processing, and other industries) (27-30). <br><br> Materials and Methods <br><br> The experiment was conducted on CC57Br female <span class="yellow">mice</span> weighing 12-14 g. Animals were divided in 15 groups. This experiment included three types of sequential combinations for the introduction of compounds: a) simultaneous BaP and o-cresol administration; b) BaP after o-cresol (stage 1); and c) BaP before o-cresol (stage 2). Appropriate controls were included (Table 1). <br><br> The chemicals were administered twice per week (for a total of 10, 20, and 40 doses) using a syringe and a needle with a soldered olive on its tip. BaP (1 or 5 mg) or o-cresol (0.02, 1, or 10 mg) was placed in triethylene glycol (TEG) and administered as 0.1-mL water solutions. The evaluation of o-cresol's modifying effect on the incidence of tumors, especially malignant tumors, the tumor latency period (tl), and the average time (ta) of the appearance of tumors, as well as the multiplicity index, M (the average number of tumors per animal for tumor-bearing animals) was recorded. Because the M could be identified at the initial stage of carcinogenesis by the third month after the beginning of the trial when neoplasms began to emerge, moribund animals were killed by ether inhalation if they did not die spontaneously. <br><br> The experiments with large chemical doses (BaP, 5 mg; o-cresol, 10 mg) lasted for 30 weeks. Some animals were killed after the 1st, 3rd, 5th, and 10th procedures for the <br><br> YANYSHEVA ET AL. <br><br> evaluation of initial stages of carcinogenesis; others were killed at 26 and 30 weeks. The stomachs were distended with formalin solution and the mucosa examined macroscopically. All tumors > 1 mm in diameter were recorded. Organs and tissues were fised in 10% neutral formalin solution, embedded in paraffin and routine histological slides prepared. Microscopic data were processed according to Mostkovoy (30). <br><br> Results <br><br> The data from this study demonstrate the modifying influence of o-cresol on BaP-induced carcinogenesis with combined oral exposure. The combined exposure and BaP alone caused benign and malignant epithelial tumors of the forestomach. The benign neoplasms were papillomas, and the malignant neoplasms were invasive carcinomas. <br><br> The combined administration of BaP and o-cresol showed different results depending on doses and sequential combinations of both chemicals. As shown in Table 2, the simultaneous administration of BaP (1 mg) and o-cresol (1 mg) affected all parameters of carcinogenesis that were measured. This included a significant increase in the incidence of tumors, shortening of the time to the appearance of the first tumor, and the mean time of tumor <br><br> development as compared to animals that received 1 mg of BaP alone and had tumors. One hundred percent of the <span class="yellow">mice</span> that received 5 mg of BaP alone had tumors. <br><br> Shortening of the time to appearance of malignant tumors was also observed: the <span class="yellow">mice</span> with forestomach cancer died between 23 and 26.2 weeks after the beginning of experiment, but the controls survived to 58.5 weeks. Muliple small metastases in the lungs and mediastinum were found in 42.8% of <span class="yellow">mice</span>, which signifies a high degree of malignancy. <br><br> o-Cresol at 0.02 mg did not modify carcinogenesis in comparison to the control. o-Cresol administration before or after BaP did not modify tumor incidence or the multiplicity index as compared to control, but the latency period was longer (Table 2). When o-cresol was administered after BaP (stage 2), there was an absence of malignant tumors. <br><br> Besides the quantitative aspects, other peculiarities of carcinogenesis in <span class="yellow">mice</span> simultaneously administered BaP and o-cresol should be noted. The combined chemical administration resulted in diffuse verrucosa vegetations over the forestomach surface, especially near the greater curvature. Highly aggressive malignant neoplasms developed earlier with more metastases as compared to control. <br><br> Many <span class="yellow">mice</span> were emaciated, and the tumors could be <br><br> Table 1. Scheme of BaP and o-cresol combined action under different regimes of oral administration. <br><br> Theatmenta <br><br> No. of <span class="yellow">mice</span>                        Stage 1                                            Stage 2 <br><br> Groups of        in each                                             No. of                                           No. of <br><br> animals          group        Substances           Dose, mg      applications    Substances         Dose, mg       applications <br><br> 1              55         BaP, o-cresol          5 +10            10          -                     -               2              45          BaP                     5              10          -                     -               3              40          TEG                     la             10          -                     -               4              45          TEG, o-cresol        1b + 10           10          -                     -               5              30          BaP, o-cresol         1+1              20          -                     -               6              30          BaP, o-cresol        1 + 0.02          20          -                     -               7              30          o-Cresol                1              20          BaP                   1               20 8              30          BaP                     1              20          o-Cresol              1               20 9              30          BaP                     1              20                                -               10              30          -                       -              -           BaP                   1               20 11              30         o-Cresol                 1              20                                -               12              30          -                      -               -           o-Cresol              1               20 13              30         o-Cresol + TEG         1+2'             20                                -               14              30         o-Cresol + TEG        0.02 +2b          20 <br><br> 15              30         TEG                     2b              20          -                     -               Abbreviations: BaP, benzo[a]pyrene; TEG, triethylene glycol. <br><br> Table 2. Occurrence of forestomach tumors in CC57Br <span class="yellow">mice</span> after combined oral administration of BaP and o-cresol. <br><br> Number of animals with forestomach tumors <br><br> Order of administration of substances             Total                         Benign                       Malignant <br><br> (dose, mg)                             Absolute       %        %        M         t,         ta         %        t,        ta BaP (5), o-cresol (10)                    29       100.0     86.2       ND        ND        ND        13.8      ND        ND BaP (1), o-cresol (1)                     19        95.0     60.0       9.6       10.7      16.3      35.0      23.0      25.2 BaP (1), o-cresol (0.02)                   7        35.0     30.0       1.6       13.5      19.8       5.0      56.8      56.8 BaP (1), stage 1, o-cresol (1), stage 2    7        31.8     31.8       1.4       16.2      31.1       0         0         0 <br><br> BaP (1), stage 2                           6        35.3     29.4       2.8       15.8      41.2       5.9      24.8      24.8 BaP (5), stage 1                          18       100.0     50.0       ND        ND        ND        50.0      ND        ND BaP (1), stage 1                          8         33.3     28.7       1.4       14.0      21.3       4.6      58.5      58.5 BaP (1), stage 2                           7        36.8     31.6       2.8       10.0      13.3       5.2      55.7      55.7 <br><br> Abbreviations: BaP, benzo[a]pyrene; 0, not observed; ND, not determined; t1, time of the first tumor appearance in weeks; ta, mean time of tumor development in weeks; M, multiplicity. <br><br> 342 <br><br> CARCINOGENESIS OF BaP AND o-CRESOL                           343 <br><br> palpated through the abdominal wall. At autopsy, enlargement of the stomach with tuberculous white superficial vegetations were observed. The forestomach and glandular part of the stomach was often obliterated by tumor masses. <br><br> The stomach was often adhered to the pancreas, liver, and mesentery. Hemmorrhage and inflammation were found in tumors foci. When o-cresol was introduced before BaP, the tumors were more frequently found closer to the small curvature of the stomach between the forestomach and esophageal entrance. Over the large curvature less prominent mucosal folds were observed. Microscopically, a decrease in mucosa papillae, epithelial atrophy (one to two cullular layers), decreased keratonization, and nuclear pycnosis were observed. With the simultaneous administration of large doses of o-cresol and BaP, the final carcinogenic effect (30 weeks after the first dose) was similar to BaP alone but differences were observed at the earlier stages (after the 1st, 3rd, 5th, and 10th exposures). <br><br> In the BaP control, multiple forestomach epithelial proliferative and hyperplastic changes were found after the third dose. Multiple papillomas occurred (9-15 in each <span class="yellow">mouse</span>). The stomach's mucosal folds appeared thickened diffusely, then papillomas appeared and finally merged together. <br><br> Approximately half of the tumors were malignant. The neoplasms filled almost the whole forestomach cavity and infiltrated the wall with tuberculous vegetations, which were visible on the serosal surface. Thmor infiltration in the liver, pancreas, and wide dissemination of peritoneum were also observed. In <span class="yellow">mice</span> simultaneously administered BaP and o-cresol, the proliferative alterations of forestomach epithelium were seen after the third dose. However, they occurred as the single foci at damaged mucosa and even in the later stages were not diffuse. Adjacent to the hyperplastic foci, the mucosa was atrophied with decreased keratonization. The epithelial cells also showed cytoplasm coagulation and nuclear pycnosis. Edema, inflammation of the mucosa, submucosa microabscesses, and erosion were seen. There were fewer papillomas per <span class="yellow">mouse</span> (four to eight in each <span class="yellow">mouse</span>) than in the control group. Even in the late stages the papillomas were isolated and elevated on the atrophied mucosal folds. <br><br> In the final experiment the papillomas prevailed as compared to the previous experiments. Only 4 out of 29 <span class="yellow">mice</span> (13.8%) developed malignant tumors. Neoplasms were smaller and appeared as single verrucosa vegetations 5-10 mm in diameter. Thus, the toxic dose of o-cresol inhibits the carcinogenic process of induced forestomach tumors by decreasing multiplicity, frequency, and percentage of malignant tumors. <br><br> Discussion <br><br> Our results and the literature suggest a hypothesis on modifying carcinogenesis mechanisms. The primary effect of the toxic agents, including carcinogenic agents, may relate to cellular membrane damage with the subsequent increased permeability (31-35), which may be the mechanism of the o-cresol. <br><br> With simultaneous introduction of o-cresol at low toxic doses and BaP there may be increased carcinogen penetration to the target cells. In addition, membrane damage may alter other cellular systems responsible for energy and xenobiotic detoxication. Damage of these processes may activate free-radical reactions, glycolysis, or alter carcinogenic metabolism, which promotes the oncogenic effect. The o-cresol effect on these systems was confirmed by our previous investigations on cytochrome P450, ferrosulfuric nonhemic proteins, and semiquinon radical content affected by the combined action of BaP and o-cresol (36). <br><br> Another effect was obtained with BaP exposure after o-cresol. The atrophic alterations in the stomach induced by 2.5 months of o-cresol administration might decrease the natural conditions of retention (muscosal folds, frontier torn), and thus shorten the contact of BaP with the forestomach. In our opinion, this explains the decreased carcinogenic effect seen after o-cresol exposure. <br><br> Considering the toxic effect of o-cresol at low toxic doses (1 mg) after BaP and the effect of a super toxic dose (10 mg) simultaneously with BaP administration, it is possible that the inhibition of carcinogenesis in both trials was related to the toxic effects of o-cresol. Cytotoxicity may not only hamper tumor growth, but also promote the regression of inducible and spontaneous neoplasms (37-39). There is also a possibility that BaP damages metabolic systems and decreases resistance to carcinogenesis. <br><br> Conclusions <br><br> The data obtained demonstrate that simultaneous administration of BaP modifies the induced carcinogenesis depending on the dose and the sequence of administration. Administered with BaP (1 mg), a low toxic o-cresol dose (about minimally effective) produces an enhanced cocarcinogenic effect reflected in the incidence, frequency, multiplicity, and degree of malignancy of forestomach tumors. <br><br> o-Cresol administration at low toxic doses before or after BaP at the same dose level (1 mg) and its administration at super toxic doses (10 mg) with the BaP optimal dose (5 mg) may inhibit carcinogenesis. Simultaneous introduction of a noneffective o-cresol dose (0.02 mg) with the BaP (1 mg) does not change its carcinogenic activity. Controlling both chemicals in the environment is the most effective measure of preventing potential risk and is undoubtedly of paramount significance. <br><br> REFERENCES <br><br> 1. Bingham, E., and Falk, H. L. Environmental carcinogenesis. The <br><br> modifying effect of cocarcinogens on the threshold response. Arch. Environ. Health 19: 779-783 (1969). <br><br> 2. Goldschmidt, B. M., Katz, C., and Van Duuren, B. L. The cocar<br><br> cinogenic activity of non-carcinogenic aromatic hydrocarbons. Proc. Am. Assoc. Cancer Res. 17: 84 (1973). <br><br> 3. Kaufman, D. C., and Madison, R. M. Synergistic effects of <br><br> benzo(a)pyrene and N-methyl-N-nitrosourea respirators carcinogenesis in <span class="yellow">Syrian golden hamsters</span>. In: Proceedings of the Sym<br><br> 344                                     YANYSHEVA ET AL. <br><br> posium on Epithelial Respiratory Carcinogenesis and Bioassays. Batelle Seattle Research Center, Seattle, WA, 1974, p. 17. <br><br> 4. Montesano, R., Saffiotti, U, Ferrero, A, and Kaufman, D. Synergistic <br><br> effects of benzo(a)pyrene and diethylnitrosamine on respiratory carcinogenesis in hamsters. J. Natl. Cancer. Inst. 5: 1395-1397 (1974). 5. Pfeiffer, E. H. Investigations of the carcinogenic burden by air <br><br> pollution in <span class="yellow">man</span>. VII. Studies on the ontogenetic interaction of polycyclic aromatic hydrocarbons. Zbl. Bakteriol. Parasitenk. Infektion. Hyg. Abt. B 158(1): 69-83 (1973). <br><br> 6. Slaga, T. J., Jecker, L., Bracken, W. M., and Weeks, C. E. The effects of <br><br> weak or non-carcinogenic polycyclic hydrocarbons on 7,12-dimethylbenz(a)anthracene and benzo(a)pyrene skin tumor-initiation. Cancer Lett. 7: 51-59 (1979). <br><br> 7. Topping, D. C., Martin, D. H., and Nettesheim, P. Determination of <br><br> cocarcinogenic activity of benzo(a)pyrene for respiratory tract mucosa. Cancer Lett. 11: 315-321 (1981). <br><br> 8. Ilnitsky, A. P., Voronin, V. M., and Solenova, M. G. Carcinogenic effect <br><br> of benzo(a)pyrene and its combinations with other carbohydrates at their oral intake [in Russian]. Voprosy Onkol. 10: 71-75 (1974). <br><br> 9. Kurljandsky, B. A., Medvedovsky, A. G., and Mashbits, F. D. On <br><br> cocarcinogenic effect of toxic factors of low intensity [in Russian]. Gig. Sanit. 10: 83-85 (1972). <br><br> 10. Thrusov, V. S., and Andrianov, L. A. On combined effect on the skin of <br><br> single application of 7,12-dimethylbenz(a)anthracene and repeated applications of benz(a)pyrene [in Russian]. Bull. Eksper. Biol. Med. 3: 80-83 (1972). <br><br> 11. Yanysheva, N. Ya., Chernichenko, I. A., Balenko, N. V, and Kireeva, <br><br> I. S. Carcinogenic substances and their hygienic standardization in the environment [in Russian]. Zdorovja, Kiev, 1977. <br><br> 12. Laskin, S., Kushner, M., and Drew, R. T. Studies in pulmonary <br><br> careinogenesis. In: Inhalation Carcinogenesis (M. G. Hanna, P. Nettesheim, and J. R. Gillbert, Eds.) U.S. Atomic Energy Commission Symposium, Series 18, 1970, p. 321. <br><br> 13. Laskin, S., Kuschner, M., and Drew, R. T. Inhalation exposure to <br><br> sulphur dioxide and benzo(a)pyrene. Oak Ridge National Laboratory, Gatlinburg, TN, 1970, pp. 322-351. <br><br> 14. Kuschner, M., and Laskin, S. Experimental models in environmental <br><br> carcinogenesis. Am. J. Pathol. 1: 183-196 (1971). <br><br> 15. Komine, T. Influences of sulfur dioxide and 3,4-benzpyrene on the <br><br> respiratory organ of <span class="yellow">rats</span>. Hokkaido Igaku Zasshi 3: 189-203 (1977). 16. Skvortsova, N. N. Role of some common atmospheric pollutions in <br><br> lung blastomogenesis [in Russian]. In: Handbook of Hygiene of Atmospheric Air (K. A. Bushtueva, Ed.), Meditsina, Moscow, 1976, pp. 384-391. <br><br> 17. Yanysheva, N. Ya., Balenko, N. V, Chernichenko, I. A, and Babiy, V. F. <br><br> Quantitation of modified effects of nitrogen oxides on carcinogenicity of benz(a)pyrene effect [in Rusian]. Gig. Sanit. 7: 7-9 (1986). <br><br> 18. Yanysheva, N. Ya., Balenko, N. V., Chernichenko, I. A., Babiy, V. F., <br><br> Bakanova, G. N., and Lemeshko, L. P. Peculiarities of manifestation of carcinogenesis at combined effect of benz(a)pyrene and phenol depending on regimen of intake into the organism [in Russian]. Gig. Sanit. 4: 29-33 (1988). <br><br> 19. Van Duuren, B. L., Katz, C., and Goldschmidt, B. M. Brief communica<br><br> tion: cocarcinogenic agents in <span class="yellow">tobacco</span> carcinogenesis. J. Natl. Cancer Inst. 51: 703-705 (1973). <br><br> 20. Van Duuren, B. L., and Goldschmidt, B. M. Cocarcinogenic and <br><br> tumor-promoting agents in <span class="yellow">tobacco</span> carcinogenesis. J. Natl. Cancer Inst. 56: 1237-1242 (1976). <br><br> 21. Wattenberg, L. W., Coccia, J. B., and Lam, L. K. Inhibitory effects of <br><br> phenolic compounds on benzo(a)pyrene-induced neoplasia. Cancer Res. 8: 2820-2823 (1980). <br><br> 22. Bogovski, P. A, and Mirme, H. Yu. Cocarcinogenicity of phenols from <br><br> Estonian shale tars (oils). Environ. Health Perspect. 30: 177-178 (1979). <br><br> 23. Gubergritz, M. Ya., and Kirso, U. E. Carcinogenic properties, struc<br><br> ture and reactive capacity of phenols [in Russian]. Voprosy Onkol. 8: 96-100 (1970). <br><br> 24. Mirme, H. Yu. Modifying effect of water-soluble shale phenols upon <br><br> carcinogenesis [in Russian]. In: Problems of Prevention of Pollution of Environment by Carcinogenic Substances (M. M. Shabad, Ed.), Tallinn, Valgus, 1972, pp. 16-18. <br><br> 25. Kirso, U. E., Pashin, Yu. V., Bakhitova, L. M., and Kjung, A. I. Effect <br><br> of antioxidants on carcinogenic and mutagenic activity of benzo(a)pyrene [in Russian]. Voprosy Onkol. 4: 70-75 (1985). <br><br> 26. Veldre, I. A. Comparative assessment of toxic effect of some water<br><br> soluble phenols [in Russian]. In: Proceedings of the Scientific Conference on Urgent Problems Related with Decrease of Infectious Diseases Morbidity and with Hygienic Problems. Tallinn, 1968, pp. 200-202. <br><br> 27. Substances possessing cocarcinogenic effect. In: Carcinogenic Sub<br><br> stances in the Environment of <span class="yellow">Man</span> (L. M. Shabad and A. P. Ilnitsky, Eds.), Budapest, 1979, pp. 111-113. <br><br> 28. Djatlovitskaya, F. G., and Maktas, E. D. Separate determination of <span class="yellow">fly</span> <br><br> phenols in water using thin-layer chromatography [in Russian]. Gig. Sanit. 6: 60-63 (1965). <br><br> 29. Kostovetsky, Ya. I., and Zholdakova, Z. I. About hygienic rate setting <br><br> of phenol in the water of waterpools [in Russian]. Gig. Sanit. 7: 7-10 (1971). <br><br> 30. Mostkovoy, M. I. Practicum on variational-statistical processing of <br><br> clinical material [in Russian]. Ashkhabad, 1954, p. 132. <br><br> 31. Weinstein, I. B. The scientific basis for carcinogen detection and <br><br> primary cancer prevention. Cancer 47: 1133-1141 (1981). <br><br> 32. Merkulov, A. I, and Skvortsova, R. I. About toxic effect of phenol [in <br><br> Russian]. Gig. Sanit. 1: 79-80 (1984). <br><br> 33. Weinstein, I. B., Mufson, R. A., Lee, L. S., et al. Membrane and other <br><br> biochemical effects of the phorbol esters and their relevance to tumor promotion. In: Carcinogenesis. Fundamental Mechanisms and Environmental Effects. London, 1980, pp. 543-563. <br><br> 34. Yamasaki, H., and Weinstein, I. B. Cellular and molecular mecha<br><br> nisms of tumor promotion and their implications for risk assessment. In: Methods for Estimating Risk of Chemical Injury: <span class="yellow">Human</span> and Non-<span class="yellow">human</span> Biota and Ecosystems (V. B. Vouk, G. C. Butler, D. G. Hoel, and D. B. Peakall, Eds.), SCOPE, 1985, pp. 155-180. <br><br> 35. Karu, T, Kirso, U. E., and Andrianov, L. A. Dynamics of resorption of <br><br> 3,4-benzpyrene with phenols from <span class="yellow">mouse</span> skin [in Russian]. Voprosy Onkol. 5: 80-84 (1973). <br><br> 36. Yanysheva, N. Ya., Yurkovskaya, T. N., Beregovskaya, N. N., et al. <br><br> Metalloenzymic complexes of energetic and detoxicating systems of cell during benz(a)pyrene-induced carcinogenesis. In: Proceedings of the 4th National Congress of Oncology with International Participation. Sofia, 1985, p. 102. <br><br> 37. Mizell, M. Anuran (Lucke) tumor breakdown in regenerating anuran <br><br> tadpole tails. Anat. Res. 137: 382-383 (1960). <br><br> 38. Sheremetieva, E. N. Spontaneous melanoma in regenerating tails of <br><br> axolotis. J. Exp. Zool. 158: 101-122 (1965). <br><br> 39. Tsonis, P. A, and Eguchi, G. Carcinogens on regeneration. Effects on <br><br> N-methyl-N-nitro-N-nitrosoguanidine and 4-nitrogninoline-1-oxide on limb regeneration in adult newts. Differentiation 20: 52-60 (1981). <h3>pmcA1564398</h3>The role of the muscarinic system in regulating estradiol secretion varies during the estrous cycle: the hemiovariectomized <span class="blue">rat</span> model
Abstract
There is evidence that one gonad has functional predominance. The present study analyzed the acute effects of unilateral ovariectomy (ULO) and blocking the cholinergic system, by injecting atropine sulfate (ATR), on estradiol (E2) serum concentrations during the estrous cycle. The results indicate that ULO effects on E2 concentrations are asymmetric, vary during the estrous cycle, and partially depend on the cholinergic innervation.<br><br>Background
Estradiol secretion is regulated by pituitary [follicle stimulating hormone (FSH) and luteinizing hormones (LH), prolactin, and adrenocorticotropin (ACTH)]. The effects of these hormones are modulated by neurotransmitters released by the intrinsic ovarian innervation near the follicular wall. Acetylcholine produced by the follicle may be one of the neurotransmitters participating in modulating the effects of pituitary hormones on the follicle [1-3].
Evidence suggesting that one gonad has functional predominance in mammals and birds have been published [1,4-8]. In previous studies we have shown that unilateral ovariectomy (ULO) modifies progesterone and/or testosterone serum concentrations, and that the effects of ULO depend on both, the stage of the estrous cycle when ULO was performed and the ovary (left or right) remaining in situ [9-11].
Asymmetry in ovarian functions has been explained by differences in the ovarian innervation participating in modulating the effects of gonadotropin on the ovarian follicles [1,6]. Kawakami et al [12] showed that electrical stimulation of the medial basal pre-chiasmatic area, the ventro-medial hypothalamus, and the areas in the mesencephalon of hypophysectomized and adrenalectomized female <span class="yellow">rats</span> resulted in a significant increase of estradiol (E2) and progesterone (P4) plasma concentrations in the contra-lateral ovarian venous blood. In turn, stimulating the dorsal hippocampus, the lateral amygdala, and the mesencephalic areas resulted in lower E2 and P4 concentrations. Ovarian denervation of <span class="yellow">rats</span> in proestrus stage blocks E2 secretion induced by stimulating the medial basal pre-chiasmatic area. In addition, the electrochemical stimulation in proestrus day of the medial basal pre-chiasmatic area of untreated <span class="yellow">rats</span> increased E2 and P4 concentrations in serum. This effect was not observed when stimulation was applied to the pre-optic supra-chiasmatic area. According to the authors' interpretation of the results, the efferent neural system connecting the brain and the ovaries is supplementary to the brain-pituitary-ovarian hormonal mechanisms regulating ovarian steroid secretion, and the system may be required for adjusting ovarian responsiveness and sensitivity to gonadotropins [12,13].
Gerendai et al. [14] described a multi-synaptic neural pathway between the central nervous system and the ovaries, with the vagus nerve being one of the main neural pathways. In ULO treated <span class="yellow">rats</span>, bi-lateral sectioning the vagus nerve (ventral or dorsal) results in lower compensatory ovarian hypertrophy. The effects of sectioning the left vagus nerve depend on the remaining ovary in situ: <span class="yellow">rats</span> with the left ovary in situ had a larger proportion of ovulating animals, compensatory ovarian hypertrophy and number of ova shed. In turn, <span class="yellow">rats</span> with the right ovary in situ showed a decrease in all parameters studied [15].
Based on available information, the present study aims to analyze if changes in E2 secretion by the left and right ovaries vary during the estrous cycle, using the unilateral ovariectomized animal as a model of study.
We also investigated if, throughout estrous cycle diestrus 1 (D1), diestrus 2 (D2) and proestrus (P), the cholinergic system modulates E2 secretion in an asymmetric way. For this purpose, we analyzed the effects of injecting ATR at 13.00 h to <span class="yellow">rats</span> on D1, D2 or P with or without unilateral or bilateral ovariectomy.<br><br>Materials and methods
The study was performed with virgin adult female <span class="yellow">rats</span> (195–225-g body weight) of the CIIZ-V strain from our own stock. Animals were kept under controlled lighting conditions (lights on from 05:00 to 19:00 h), with free access to food (Purina S.A., Mexico) and tap water; following NIH Guide parameters for the care of laboratory animals. The Committee of the Facultad de Estudios Superiores Zaragoza approved the experimental protocols.
Estrous cycles were monitored by daily vaginal smears. Only <span class="yellow">rats</span> showing at least two consecutive 4-day cycles were used in the experiment. All surgeries were performed under ether anesthesia, between 13:00–13:15 hours. <span class="yellow">Rats</span> were sacrificed by decapitation one hour after treatment.
Experimental groups
<span class="yellow">Rats</span> were randomly allotted to one of the experimental groups described below. Animals from different experimental groups were treated simultaneously and sacrificed one hour after surgery. The number of animals used in each experimental group is presented in Tables 1, 2 and 3.
Control group (N = 48). Non-treated cyclic <span class="yellow">rats</span> sacrificed at 14:00 h on D1 (17 <span class="yellow">rats</span>), D2 (19 <span class="yellow">rats</span>) and P (12 <span class="yellow">rats</span>).
Ether anesthesia (N = 24): Groups of <span class="yellow">rats</span>, on specific stages (D1, D2 or P) of the estrous cycle, were anesthetized for 10 min and sacrificed one hour later.
Unilateral peritoneal perforation (sham operation) (N = 53): A unilateral incision was performed 2-cm below the last rib; affecting skin, muscle, and peritoneum. The ovaries were not injured or manipulated. After surgical procedures the wound was sealed.
Bilateral peritoneal perforation (sham operation 2) (N = 27). A bilateral incision below the last rib, including skin and muscle, was performed. The ovaries were not injured or manipulated. After surgical procedures the wound was sealed.
Unilateral ovariectomy (ULO) (N = 50): A unilateral incision below the last rib, including skin and muscle was performed, and the right or left ovary was extirpated. The wound was subsequently sealed.
Bilateral ovariectomy (N = 23): A bilateral incision below the last rib, including skin and muscle was performed, and the ovaries removed. The wound was subsequently sealed.<br><br>Blocking the cholinergic system
To analyze the effects of blocking the cholinergic system, groups of animals were injected with atropine sulfate (ATR, Sigma Chem. Co. St. Louis, Mo.). ATR was injected one hour before surgery at doses known to block ovulation: in D1, 100 mg/kg body weight (b.w.); in D2, 300 mg/kg b.w.; and in P, 700 mg/kg b.w. [16].
One hour after ATR treatment, <span class="yellow">rats</span> were randomly allotted to one of the following treatments: unilateral peritoneal perforation, bilateral peritoneal perforation, ULO, or bilateral ovariectomy. All animals were sacrificed one hour after surgery. For control purposes, untreated <span class="yellow">rats</span>, on D1, D2 or P, were injected with ATR in the same dose as in their corresponding treatment group. The animals were sacrificed two hours after treatment.<br><br>Autopsy procedures
Animals were sacrificed by decapitation. The blood of the trunk was collected in a test tube, allowed to clot at room temperature for 30 minutes and centrifuged at 3,000 rpm for 15 minutes. Serum was stored at -20°C, until E2 concentrations were measured.<br><br>Hormone assay
Concentrations of E2 in serum were measured by Radio-Immuno-Assay (RIA); using kits purchased from Diagnostic Products (Los Angeles, CA). Results are expressed in pg/ml. The Intra- and inter-assay variation coefficients were 6.9% and 10.8 %, respectively.<br><br>Statistics
Data on hormonal concentrations in serum were analyzed using multivariate analysis of variance (MANOVA) followed by Tukey's test. Differences in serum hormone concentrations between two groups were analyzed by Student's t-test. A probability value of less than 5% was considered significant.<br><br>
Results
Effects of ether anesthesia and unilateral or bilateral perforation of the peritoneum
In the control group, animals sacrificed on P showed significantly higher E2 serum concentration than animals sacrificed on D1 or D2 (D1: 55.3 ± 8.0; D2: 59.1 ± 7.9; P: 158.4 ± 1.8). Compared to the control group, ether anesthesia treatment did not modify E2 serum concentrations (D1: 62.9 ± 8.4; D2: 69.5 ± 12.0; P: 164.1 ± 17.6). Since ether anesthesia did not modify E2 serum concentrations, treatment results are compared to their respective control group.
The effects on E2 serum concentrations of unilaterally or bilaterally perforating the peritoneum depended on the side of the peritoneum and the stage of the estrous cycle when perforation surgery was performed. Perforating the left peritoneum on D1 resulted in lower E2 serum concentrations (55%), while bilateral perforation, or perforating the right side of the peritoneum, had no apparent effects (Table 1).
Perforating the right side of the peritoneum on D2 day resulted in E2 concentration increases (184%), while perforating the left side resulted in a decrease (51%) of E2 serum concentrations. Bilateral perforation had no apparent effects on hormone concentrations. Perforating the peritoneum on P phase (left, right or bilateral) resulted in hormone serum concentration decreases (Left 30%; Right 50%; Bilateral 41%). Results are summarized in Table 1.<br><br>Effects of unilateral or bilateral ovariectomy
When surgery was performed on D1, no significant differences in E2 serum concentrations were observed between <span class="yellow">rats</span> with ULO (animals with intact left or right ovary in situ) or perforation of the peritoneum (Figure 1). Animals with the left intact ovary in situ showed significantly higher E2 serum concentrations than animals with the right intact ovary in situ (61.5 ± 9.4 vs. 17.3 ± 4.3, p < 0.05 Student's t test).
Compared to animals with unilateral perforation of the peritoneum, animals with right ULO (left ovary in situ) performed on D2 had lower E2 serum concentrations (55%). Such differences were not observed in <span class="yellow">rats</span> with left ULO (Figure 1). As in <span class="yellow">rats</span> treated on D1, E2 serum concentrations were significantly higher in animals treated on D2 with the left ovary in situ (right ULO) than in animals with the right ovary in situ (49.5 ± 10.8 vs. 26.0 ± 6.9).
In animals treated on P, right ULO (left ovary in situ) resulted in higher E2 (180%) serum concentrations than in animals with unilateral peritoneum perforation. ULO performed on the left side (right ovary in situ), resulted in significantly lower (45%) E2 serum concentrations compared to <span class="yellow">rats</span> with a unilateral perforation of the peritoneum (Figure 1). As observed in <span class="yellow">rats</span> treated on D1 or D2, when the intact left ovary remains in situ, estradiol serum concentrations were significantly higher than in animals with the intact right ovary in situ (142.0 ± 14.1 vs. 61.5 ± 6.0). Compared to animals with a bilateral perforation of the peritoneum, bilateral ovariectomy resulted in significantly lower E2 serum concentrations, regardless of the stage of the estrous cycle surgery performed (D1 74%; D2 73%; P 84%). Results are summarized in Table 2.<br><br>Effects of blocking the cholinergic system
Injecting ATR on D1 or P resulted in E2 serum concentrations decreases (84% and 67%, respectively), and had no apparent effects on E2 serum concentrations when injected on D2 (Table 3).
Figure 2 shows that the effects of blocking the cholinergic system of <span class="yellow">rats</span> with unilateral perforation of the peritoneum depended on both, the side (left or right) and the phase of the estrous when surgery was performed. Injecting ATR on D1 or D2 resulted in a significant drop in E2 serum concentrations in animals with sham treatment on the right side. Blocking the cholinergic system of <span class="yellow">rats</span> with left side peritoneum perforation on D1 or P resulted in a drop in E2 serum concentrations (52%; 47%, respectively), while the same treatment performed on D2 resulted in a significant E2 concentrations increase (157%).
Figure 3 shows the effects of blocking the cholinergic system of <span class="yellow">rats</span> with ULO. ATR treatment on D1 or P stages performed on <span class="yellow">rats</span> with the left ovary in situ resulted in a significant drop of E2 serum concentrations (65%; 62% respectively). Such effects were not observed in <span class="yellow">rats</span> treated on D2. When ATR treatment was performed on <span class="yellow">rats</span> with the right ovary in situ on D1, E2 serum concentrations were lower (48%) than in ULO animals. Blocking the cholinergic system on D2 resulted in E2 serum concentrations increase (159%). When the treatment was performed on P, no significant differences in E2 serum concentrations were observed.
Compared to bilateral treatment, perforation of the peritoneum or bilateral ovariectomy, ATR treatment on D1 resulted in significant E2 serum concentrations decreases, 90% in bilateral peritoneal perforation and 60% in bilateral ovariectomized animals.
Blocking the cholinergic system on D2, to <span class="yellow">rats</span> with bilateral perforation of the peritoneum or bilateral ovariectomy resulted in E2 serum concentrations increases (159% and 253% respectively), while injecting ATR to animals treated on P had no apparent effects (Figure 4).<br><br>
Discussion
The results obtained in the present study suggest that the ability to compensate the secretion of E2 by the missing ovary is different between the right and left ovaries and varies during the estrous cycle. Similarly, our results suggest that the cholinergic system participates in regulating E2 secretion by the ovary, and that such participation varies depending on the ovary remaining in situ and the stage of the estrous cycle when the surgical procedure was performed.
Previously, we suggested the existence of a neural pathway arising from the peritoneum that participates in regulating E2 [9], P4 [10] and testosterone secretion [11]. In the <span class="blue">rat</span>, the sensory information arising from the peritoneum is sent to the nucleus tractus solitarius and stimulates neurokinine-B receptors [17]. Since perforating the peritoneum unilaterally on each day of the estrous cycle changed E2 serum concentrations, we think that each side of the peritoneum sends different neural information through the superior ovarian nerve (SON) to the ovary and the central nervous system, perhaps reaching nuclei related to the vagus nerve.
A study analyzing the distribution of sensory neurons innervating the peritoneum showed that when tracer was placed on the area where the peritoneum covers the abdominal wall, labeled neurons were observed only in the ipsilateral dorsal root ganglia [18]. The authors suggest that most of the parietal peritoneum receives sensory nerves from dorsal root ganglia, and visceral peritoneum from both, the spinal and vagus nerves.
According to Stener-Victorin et al. (19) repeated electro-acupunture treatments in <span class="yellow">rats</span> with polycystic ovary syndrome (PCO), induced by a single injection of estradiol valerate, resulted in lower nerve growth factor (NGF) concentrations at the ovarian level than in non-electro-acupunture treated PCO <span class="yellow">rats</span>. In our experiments, perforating the peritoneum affected the same somatic segments employed by Stener-Victorin et. al. [19].
We presume that peritoneum surgery resulted in an increase of NGF concentrations at the ovarian level, which in turn induced hyper-androgenism, as observed in <span class="yellow">women</span> with PCO [20]. Previously, we showed that the unilateral perforation of the peritoneum results in a significant increase in testosterone serum concentrations [11]. Because E2 serum concentrations did not increase after left or bilateral perforation of the peritoneum, we suppose that the neural information originating from the peritoneum inhibits the mechanisms regulating aromatase activity within the follicle.
One of the ovaries' sources of catecholamines arrives through the SON. In the ovary, the SON fibers are mainly distributed in the peri-follicular theca layer, and in close relation with the cells of the theca interna [21,22]. Sectioning the SON of <span class="yellow">rats</span> in P results in a sudden drop of P4 and E2 concentrations in the ovarian vein effluent [23], while the same procedure on estrus did not modify E2 concentrations [24]. Therefore, it is possible that perforating the peritoneum modifies the type and/or rate of information arriving to the ovary via the SON. Another possibility is that perforating the left side of the peritoneum results in an increase release of ovarian gamma amino butyric acid (GABA), and a subsequent increase of E2 concentrations.
According to Erdö, et. al. [25] and Laszlo, et. al. [26], injecting GABA into pseudo-pregnant <span class="yellow">rats</span> increases E2 concentration in the blood. Present results indicate that injecting ATR, before unilateral or bilateral perforation of the peritoneum, to <span class="yellow">rats</span> in D1 or P, results in lower E2serum concentrations; leading us to think that some of the neural fibers present in the peritoneum are muscarinic. It is also possible that blocking the cholinergic innervation, by ATR treatment, results in lower adrenaline and norepinephrine release by the adrenal medulla.
Our results, and those of others, suggest that stimulating on D2 the sensory receptors located on the left side of the peritoneum triggers an E2 secretion inhibitory mechanism, that the sensory pathway arising from the right side has a stimulatory effect, and that both are mediated by the cholinergic muscarinic system.
Previously, we showed that injecting ATR to <span class="yellow">rats</span> in D2 results in increases of P4 serum concentrations originating from the adrenals [8], without having apparent effects on testosterone serum concentrations [11]. Since P4 and androgens are precursors in the synthesis of E2, we presume that this mechanism may explain the increase in E2 serum concentrations observed in <span class="yellow">rats</span> with peritoneum perforation previously injected with ATR.
Another possibility explaining the differences on E2 secretion regulation during the estrous cycle is that the effects of the cholinergic system take place through changes at the celiac ganglion level. According to Aguado and Ojeda [23], acetylcholine inhibits P4 secretion in the celiac ganglion-SON-ovary preparation obtained from <span class="yellow">rats</span> in D1 or D2, while the preparation obtained from <span class="yellow">rats</span> in P resulted in only a moderate stimulation. Since there is evidence that fibers from the vagus nerve innervate neurons in the celiac ganglion [27], we presume that the cholinergic system modulates the sympathetic post-ganglionar activity and the secretory ability of the ovaries through the SON.<br><br>Conclusion
Based on the differences in E2 serum concentrations in <span class="yellow">rats</span> with ULO, present results suggest that the capacity to release E2 by the left and right ovaries varies during the estrous cycle. We presume that the left ovary releases more E2 than the right one. As previously proposed, another possibility is that neural communication between the ovaries modulates E2 secretion.<br><br>
<h3>pmcA1859974</h3><span class="yellow">Human</span> growth hormone (GH1) gene polymorphism map in a normal-statured adult population
Abstract
Objective
GH1 gene presents a complex map of single nucleotide polymorphisms (SNPs) in the entire promoter, coding and noncoding regions. The aim of the study was to establish the complete map of GH1 gene SNPs in our control normal population and to analyse its association with adult height.<br><br>Design, subjects and measurements
A systematic GH1 gene analysis was designed in a control population of 307 adults of both sexes with height normally distributed within normal range for the same population: −2 standard deviation scores (SDS) to +2 SDS. An analysis was performed on individual and combined genotype associations with adult height.<br><br>Results
Twenty-five SNPs presented a frequency over 1%: 11 in the promoter (P1 to P11), three in the 5′UTR region (P12 to P14), one in exon 1 (P15), three in intron 1 (P16 to P18), two in intron 2 (P19 and P20), two in exon 4 (P21 and P22) and three in intron 4 (P23 to P25). Twenty-nine additional changes with frequencies under 1% were found in 29 subjects. P8, P19, P20 and P25 had not been previously described. P6, P12, P17 and P25 accounted for 6·2% of the variation in adult height (P = 0·0007) in this population with genotypes A/G at P6, G/G at P6 and A/G at P12 decreasing height SDS (−0·063 ± 0·031, −0·693 ± 0·350 and −0·489 ± 0·265, Mean ± SE) and genotypes A/T at P17 and T/G at P25 increasing height SDS (+1·094 ± 0·456 and +1·184 ± 0·432).<br><br>Conclusions
This study established the GH1 gene sequence variation map in a normal adult height control population confirming the high density of SNPs in a relatively small gene. Our study shows that the more frequent SNPs did not significantly contribute to height determination, while only one promoter and two intronic SNPs contributed significantly to it. Studies in larger populations will have to confirm the associations and in vitro functional studies will elucidate the mechanisms involved. Systematic GH1 gene analysis in <span class="yellow">patients</span> with growth delay and suspected GH deficiency/insufficiency will clarify whether different SNP frequencies and/or the presence of different sequence changes may be associated with phenotypes in them.<br><br><br><br>Introduction
<span class="yellow">Human</span> skeletal growth and final height attainment are a result of a multifactorial regulation involving systemic and local hormones, growth and nutritional factors, lifestyle and genetic factors. Heritability estimates1 and genome-wide linkage analysis2 have shown that genetic factors play a major role in determining stature. Among these factors, the GH-IGF-I axis plays an important role during postnatal life, and associations between structural variations in its genes and height are currently under study.3 Although growth hormone (GH) deficiency is a well-known cause of growth retardation, which responds to GH replacement therapy, the diagnosis and physiopathological mechanisms for the so-called ‘idiopathic isolated GH deficiency’ (IIGHD) require further clarification. In addition, GH secretion levels and markers of GH biological activity have been demonstrated to be specific and sensitive only in major deficiency states.4,5 Genetic causes of GH deficiency within the GH1 gene have been established; however, they are rarely recognized and only sought in major GH deficiency states during childhood and in family studies.3 GH1 gene, located at 17q22–24, is a component of the GH gene cluster in which five genes evolving from a common ancestor are 91–99% sequence conserved (paralogues).6 GH1 is more abundantly expressed in pituitary cells, while the other four genes are expressed in placental tissue. Large deletions within the GH1 gene cluster were described first followed by point mutations, the majority of which affect introns 3 or 4, provoke skipping of exon 3 product and exert a dominant effect.3,7,8 More recently, the presence of single nucleotide polymorphic points (SNPs) in the promoter region or in intron 4 of the GH1 gene have been described9–12 and associations with promoter allele activities or with GH secretion efficacy and circulating IGF-I levels in growth-retarded <span class="yellow">patients</span> have also been described.11,12 Other studies have analysed several GH1 gene SNP genotypes as related to the incidence of neoplasia, with a positive association with colorectal neoplasia for intron 4 SNP,13 a negative result for breast carcinoma14,15 or a positive one for breast cancer risk.16,17 In addition, a recent study in a cohort of adults over ages 60 years detected a significant association between genotypes at one SNP in the GH1 gene promoter region and at the intron 4 SNP described by Hasegawa et al.11 with baseline bone density and accelerated bone loss together with an interaction with weight at 1 year.18 Intron 4 SNP described by Hasegawa et al.11 has also been associated, in <span class="yellow">women</span>, with shorter body height and reduced mortality,19 whereas another intron 4 SNP (T1169A) has been associated in both sexes with a favourable metabolic profile.20 A systematic SNP study was conducted by Adkins et al.21 in GH1 promoter, coding and noncoding regions in DNAs from placental tissues, and analysis of associations between genotypes and birth weight revealed an association between an alternate nucleotide at −1 and +3 of translation initiation site and fetal growth restriction. However, no systematic GH1 gene analysis in the entire promoter, coding and noncoding regions has been conducted in adults to establish the map of structural variation and its possible association with height. The relatively short size of the entire gene permits a complete analysis which is, nevertheless, hampered by the need to avoid amplification of any other of the GH cluster genes (paralogues) and the high density of sequence variations.
To obtain normative data for subsequent analysis of GH1 gene contribution to IIGHD in <span class="yellow">children</span>, a systematic GH1 gene structural analysis was designed in a normal adult control population to establish the GH1 gene SNP map in adults from our population with heights within the normal range, determine the genotype frequencies and analyse possible associations between individual and combined SNPs with height.<br><br>Subjects and methods
Subjects
A total of 307 adult subjects of both sexes (164 <span class="yellow">women</span> and 143 <span class="yellow">men</span>) were recruited from hospital personnel and parents of <span class="yellow">patients</span> with no history of growth retardation. Subjects had to fulfil the following criteria: Iberian Peninsular (except Basque) family origin and no family history of pathological short stature. A single subject per family was included. The protocol was approved by the Hospital Vall d’Hebron Ethics Committee and written informed consent was obtained from each <span class="yellow">participant</span>. Height standard deviation scores (height SDS) were calculated according to sex-specific reference growth charts for the Spanish population (Carrascosa et al.22 charts were used for subjects under ages 30 years and Hernández et al.23 for subjects aged 30–50 years). Only individuals with height SDS between −2 and +2 SDS were included in the study (mean −0·016; 32 <span class="yellow">women</span> and 28 <span class="yellow">men</span> between −2·000 and −1·010; 99 <span class="yellow">women</span> and 80 <span class="yellow">men</span> between −1·000 and +0·910; 33 <span class="yellow">women</span> and 35 <span class="yellow">men</span> between +1·010 and +1·980) and sample size was adjusted for normal sex and height SDS distribution. Height and weight were recorded in the morning by a single observer. Height was measured with a Harpenden stadiometer. Four millilitres of peripheral venous blood were drawn into EDTA-containing tubes for molecular genetic analysis.<br><br>Genomic DNA study
Genomic DNA was obtained from peripheral blood following the method described by Lahiri and Nurnberger.24 DNA was amplified by polymerase chain reaction (PCR) using a nested strategy. Briefly, 50 ng of genomic DNA were added to a 10 µl reaction mixture of 1 mm Mg(OAc)2, 0·6 mm dNTPs, 0·3 µm of each primer, and 0·4 U r Tth DNA polymerase XL (Applied Biosystems, Foster City, CA, USA). The sense and antisense primers used corresponded to nucleotides 4156–5′ACGGTCCGCCACTACGCCCAGC-3′ and the complement of 6948–5′TGCAGTGAGCCAAGATTGTGCC-3′ of the GH gene cluster.6 The PCR reaction mix was denatured for 5 min at 94 °C and cycled 40 times (94 °C, 1 min; 72 °C, 3 min 30 s) followed by a 7-min extension at 72 °C. The resulting GH1 PCR products (2893 bp) were used as templates for five nested reactions (AN, BL, CK, DI, FP), carried out as follows: 1 µl of each GH1 PCR product was added to a 20 µl reaction mixture of 1·5 mm MgCl2, 0·2 mm dNTPs, 0·3 µm of each primer and 0·4 U Eco Taq DNA polymerase (Ecogen S.R.L., Barcelona, Spain). Reaction mixtures were denatured for 5 min at 94 °C, cycled 40 times (94 °C, 1 min; 58 °C, 1 min; and 72 °C, 1 min), followed by a 7-min extension at 72 °C. Sense and antisense primers were as follows:
Sequencing from both ends was performed by the dideoxy method using ABI PRISM BigDye Terminator version 3·1 Cycle Sequencing Kit (Applied Biosystems, Foster City, CA, USA). GH1 gene nucleotide sequence published by Chen et al.6 was used as control. For each DNA, the five segments from the nested PCR were assembled with the SeqEscape programme (Applied Biosystems) and interpretation was made visually and simultaneously by two observers. Antisense sequencing was performed to confirm each nucleotide sequence change up to the establishment of the more frequent SNP map (frequency over 1%), whereas less frequent single or multiple nucleotide changes were reconfirmed in each DNA by antisense sequence and resequencing after a new nested PCR from original DNA was performed.<br><br>Single nucleotide polymorphism (SNP) genotyping
The sequences for the five genes of the GH cluster identified by Chen et al.6 and reported as the GI sequence 183148 were aligned using the Multalin program.25 SNPs and other sequence changes identified were indicated using their position corresponding to GH1. Genotypes were deduced by the combination of genetic variation at the polymorphic positions.<br><br>Statistical analysis
Standardized height was investigated for normal distribution (Kolmogorov-Smirnov test: c2 = 2·882, P = 0·4733). Hardy–Weinberg equilibrium was tested for SNPs presenting three alternate genotypes according to standard procedures using χ2-analysis. anova test was applied to investigate individual and combined SNP association with adult height SDS; significance assessment was adjusted for multiple testing using Fisher's PLSD test setting Pcritical = 0·05 or the Bonferroni–Dunn test setting Pcritical = 0·05/n (n = number of comparisons carried out). Stepwise regression analysis was applied to predict the contribution of SNPs to adult height SDS. Statview 4·5 program (Abacus Concepts Inc., Berkeley, CA, USA) was used for statistical analyses.<br><br>
Results
GH1 gene sequence variation
GH1 gene sequence comparison with the GI-183148 sequence published by Chen et al.6 yielded a total of 54 single or multiple nucleotide changes. Twenty-five SNPs presented a frequency over 1% (genotypes and frequencies are listed in Table 1). SNPs which presented the three alternate genotypes (P2 to P4, P6, P7, P10 and P24) were in Hardy–Weinberg equilibrium (data not shown). Twenty-nine additional changes were found with a frequency under 1% or involving more than one nucleotide and thus could be considered as rare variant SNPs (R1 to R29) (Table 2). These changes were found in 29 of 307 subjects (9·4%), all in heterozygosity.<br><br>GH1-paralogue alignment
A sequence alignment was performed to study possible sequence recombinations among paralogues of the five GH1-gene cluster (Fig. 1). This alignment showed that 9 of 25 SNPs (36%) in the GH1 gene did not correspond to any of the paralogues.
Among the 29 rare SNPs found, six (20·7%) did not correspond to any of the paralogues: two were located in the 5′UTR region (R9 and R10), two in intron 1 (positions 5300 = R14 and 5302 = R17), one in intron 2 (position 5679 = R21) and one in intron 4 (position 6344 = R23) (Table 2).<br><br>Equivalence with previously reported GH1 changes
Equivalence with changes and SNPs previously reported by other authors are shown in Table 3. The majority found a high density of SNPs in the promoter and 5′UTR regions in control populations.10,12,21 Several sequence changes have been reported in <span class="yellow">patients</span> with familial or idiopathic short stature,11,26,27 whereas P8, P19, P20 and P25 (at positions 5165, 5681, 5686 and 6358, respectively, in the Genebank accession GI 183148) located in the promoter, intron 2 and intron 4 regions, respectively, had not been previously described.<br><br>GH1 genotypes and associations with height SDS
Associations between genotypes and standardized height were first studied in the subpopulation of 278 controls carrying only the 25 most frequent SNPs in the GH1 gene (c2 = 2·59; P = 0·5458 for normality of height distribution).
Three individual SNPs showed a statistically significant association with height SDS: at positions 5286 (P16), 5290 (P17) and 6358 (P25). Subjects with heterozygous genotypes presented statistically significant taller stature than the corresponding homozygous genotypes (P = 0·016 for P16, P = 0·015 for P17 and P = 0·023 for P25) (Fig. 2a,b,c). P16 and P17 were in linkage disequilibrium (LD) (r2 = 0·831), while P25 was carried by six subjects homozygous at P16 and P17.
GH1 gene genotypes were defined by genetic variation in the 25 polymorphic positions. We found 163 different combinations. Only two genotypes presented a frequency over 5% (Table 4). Height SDS in the two more frequent genotypes did not differ significantly and covered the whole height range. Genotype 1 presented four heterozygous variations and Genotype 2 was the corresponding homozygous genotype. Heterozygous positions corresponded to SNPs with the highest frequency variation (4886 (P4), 5107 (P7), 5157 (P10) and 6331 (P24)). In addition, DNAs exhibited a different genotype in each of 129 subjects (46·4%).
The 11 SNPs found in the promoter region (Table 1) were grouped in 94 genotypes and analysed for association with adult height SDS. The four more frequent combinations are listed in Table 4: height SDS of these four genotypes did not differ statistically although Genotype 3 tended to have a shorter height. Genotype 3 differed from Genotype 1 in the SNP located at position 5089 (P6), corresponding to Pit 1 proximal responsive element for GH1 gene promoter. Genotype 4 is heterozygous at positions 4856 (P2), 4863 (P3) and 5107 (P7). In addition, 19% of cases exhibited a genotype in the promoter region carried by only one subject.
Combination of the three SNPs in the 5′UTR region of GH1 gene resulted in five different genotypes. SNPs at positions 5178 (P12) and 5187 (P13) were in LD (r2 = 0·88). Mean height SDS comparison among these genotypes was not statistically significant, although mean height SDS of alternate nucleotide carriers at position 5178 (P12) tended to be shorter (Table 4).
An anova analysis was conducted to investigate the interaction between two or more SNPs and height SDS. SNPs at positions 5286 (P16) and 5290 (P17) were in LD (r2 = 0·83): the heterozygous genotype AG/AT for these SNPs was associated with taller stature (shown above). SNP at position 6358 (P25) increased the expected height SDS for individual carriers of the G allele at 5089 (P6) SNP as shown in Fig. 2(d): subjects heterozygous at 6358 (P25) were taller than the mean, and mean height SDS of subjects with GG/TG combined genotype was significantly higher than the corresponding GG/TT genotype (P = 0·0021), suggesting an interaction between these two SNPs as they were not in LD.
Analysis of height SDS association with the most frequent single and combined SNPs and with rare variant SNPs was performed in the 29 individuals carrying the rare SNPs (Table 2). None of them carried any of the three SNPs (P16, P17 and P25) related to taller stature in the population of 278 controls with only the frequent SNPs. In these 29, mean height SDS (0·000 ± 0·987, from −1·930 to +1·870) did not differ from that of the 278 controls (−0·018 ± 1·041, from −2·000 to +1·980). Analysis of associations between individual SNP genotypes and height SDS revealed that SNPs at positions 5089 (P6), 5178 (P12) and 5187 (P13) were associated with significantly shorter stature (Fig. 2e). Only two sequence changes considered as rare SNPs were carried by individuals in the lower normal height range (between −1·500 and −2·000 SDS) (Table 2): R4 (4979 C > T) in the promoter region and R14 (5300 C > T) in intron 1. Predicted single amino acid changes located in exon 5 (R25 to R27) were not associated with short stature.
In the entire population of 307 controls, stepwise regression analysis between height SDS and genotypes at the 25 SNPs showed that genotypes at 5089 (P6), 5178 (P12), 5290 (P17) and 6358 (P25) were significantly correlated with height SDS (r2 = 0·062, P = 0·0007) with genotypes A/G at P6, G/G at P6 and A/G at P12 decreasing height SDS (−0·063 ± 0·031, −0·693 ± 0·350 and −0·489 ± 0·265, respectively, Mean ± SE) and genotypes A/T at P17 and T/G at P25 increasing height SDS (+ 1·094 ± 0·456 and +1·184 ± 0·432, respectively).<br><br>
Discussion
Genetic variations within <span class="yellow">human</span> GH1 gene have been described by several authors.9–12,21 The populations described to date comprised small numbers of normal-stature individuals,9 male adults with narrow height range12 or growth-retarded <span class="yellow">patients</span> with/without GHD before achievement of adult height.9,11,12 Our study was designed to characterize the GH1 gene sequence variation in individuals within the whole range of normal adult height (between −2 and +2 SDS) according to the standards for our population. Height was normally distributed, both sexes were equally represented and the GI-183148 homozygous sequence6 was used for comparison. A nested PCR with specific primers for GH1 gene was designed, thus avoiding amplification of any other GH gene paralogue of the GH gene cluster.
Our results establish a map of 25 SNPs as present in over 1% of individuals, whereas 29 other sequence changes (single or multiple nucleotide) are present in less than 1% of subjects. More than 50% (n = 14) of SNPs are located in the promoter and 5′UTR regions, thus confirming previous reports: Giordano et al.9 reported eight SNPs in the promoter and 5′UTR regions, Wagner et al.10 16 SNPs from the promoter to intron 1 and Horan et al.12 identified 36 haplotypes in control subjects of the British population, which would result from the combination of 15 of the previously reported SNPs. Our results confirm the presence of 13 of those points; SNP at 5165 (R11 in the present study and +3 in references9,12) was present in less than 1% of subjects and a new SNP is described (P8 at 5116 in the VDR/RA/T3 responsive element sequence). The remaining SNPs (from P15 to P25, n = 11) are distributed in introns 1, 2 and 4, and among coding regions only exon 1 and exon 4 bear a total of three SNPs, two of which predict an amino acid change (P15 and P21). These two latter SNPs had been described by Millar et al.26 and the more frequent SNP in intron 4 (P24) has been described by Hasegawa et al.11 together with the two more frequent SNPs in the promoter region (P4 and P7 in our map). Three new SNPs are described (P19 and P20 in intron 2 and P25 in intron 4), all outside the splice sites. Only SNPs presenting high frequency are present in homozygous alternate state and this accounts mostly for the majority of the promoter and 5′UTR SNPs and in the intron 4 more frequent SNP (P24) described by Hasegawa et al.11 In conclusion, in the entire coding and noncoding GH1 gene sequence, only P24 is present in homozygous alternate state. Our results show that the GI 183148 homozygous sequence is present in our population except for SNP P14 in the 5′UTR region which is only present as the alternate nucleotide in homozygous or heterozygous states.
As described by several authors9,10,12,21 several promoter SNPs affected functional sequences and P6 is located in the Pit 1 proximal responsive element, P7 and P8 in the VDR/RA/T3 responsive element and P9 (G del) in the TATA box.
The mechanisms by which the high density of SNPs in the GH1 gene is generated has been proposed to be recombination and gene conversion with any other(s) of the GH cluster genes.9,12,28 Alignment of the 25 SNPs with the other GH1 gene paralogues demonstrated in our results that this mechanism is possible for 64% of SNPs. Familial SNP transmission pattern analysis will be of interest to support the hypothesis of GH gene recombination.
In addition, 29 of 307 individuals (9·4%) bore additional GH1 sequence changes with frequencies under 1%. As for SNPs, they are located along the whole gene with higher density in the promoter and 5′UTR regions. Interestingly, intron 3 and exon 5 present several of these less frequent changes. Intron 3 has been shown to carry the majority of single nucleotide mutations causing the dominant form of GH deficiency.3 The two single nucleotide changes detected in intron 3 (R15 at 6056 and R16 at 6061) are in perfect LD (r2 = 1·0) and located within the enhancer splice site element (ESE) described by Ryther et al.29,30 Studies in additional normal or growth-retarded populations will permit description of their possible clinical implications. Five single nucleotide changes are located in exon 5; of these five, three predict an amino acid change, and one of the three (Ile179Met) has been described by Lewis et al.27 in a paediatric <span class="yellow">patient</span> with familial short stature and the other two, as yet undescribed, are contiguous in a single individual (Pro133Hys and Arg134Leu). Polynucleotide changes are mostly located in the promoter region corresponding to the VDR/RA/T3 response element. As for frequent SNPs, the majority of the sequence changes with frequencies under 1% may have been generated by recombination within the GH gene cluster as 19 of 24 (79%) may correspond to one or more of the GH1 gene paralogues.
Our results now show the diversity and complexity of SNP genotypes, as previously highlighted by other authors9,10,12,21 in a normal adult height control population. Our initial aim when designing the present study was to establish the map of GH1 gene SNPs in our adult control population with heights normally distributed within the entire normal range for further comparison with genotypes in our paediatric population with growth delay, variable response to GH secretion tests and adequate response to GH therapy. Analysis of SNP association with adult height was subsequently performed to establish a body of knowledge useful for comparing <span class="yellow">patient</span> genotypes and phenotypes. This analysis was first performed in controls bearing only frequent SNPs (90·5% of the total population). We demonstrate that the four SNPs with the highest allelic variation frequencies (P4, P7, P10 and P24) do not significantly contribute to adult height determination, with the heterozygous genotype being the most frequent followed by the corresponding homozygous genotype in the whole sequence, and heights are normally distributed over the entire height range. The third most frequent combined genotype in the promoter region in our population presented, in addition, in heterozygosity, the SNP at P6 in the sequence regulated by Pit 1 and although mean height of individuals (6·1%) bearing this genotype was around −0·5 SDS, this was not statistically significant.
Analysis of single SNP genotype association with adult height yielded few clues as to the contribution of GH1 gene variation to adult height determination. Only three SNPs (P16, P17 and P25), present with low frequency and only in heterozygous state, were individually significantly associated with taller stature and none was individually associated with shorter stature. P16 and P17 (in LD, r2 = 0·83) are located in intron 1 and P25 in intron 4. The resulting sequence for the presence of P16 and P17 corresponded to the paralogue GH2 and generated a responsive element for a core-binding protein (Matinspector Programme, Geometrix Software GmbH, München, Germany) with three Kruppel-type zinc fingers which could increase the efficacy of GH1 gene transcription;31,32 moreover, Kruppel-like proteins have recently been described in the brain.33 Stepwise regression analysis demonstrated that P17 and P25 contribute, separately, to an increase of almost 1·0 height SDS. P16 and P17 had been described by Adkins et al.21 although they found no association with fetal growth, whereas P25 had not previously been described. The mechanisms by which they may determine taller final height should be established by in vitro studies analysing GH1 gene transcription and GH protein translation efficiencies.
Analysis of interaction effect between SNPs detected that variation at P25 masked an effect of P6. Individuals homozygous at P25 (TT) present a significant association between P6 genotype and height with the homozygous alternate genotype at P6 (GG) being associated with shorter stature. This was further confirmed in the subpopulation of 29 individuals bearing rare SNPs who, in the absence of heterozygous change at P25, presented significantly shorter stature in the heterozygous alternate nucleotide change at P6 (AG). P6, located at Pit 1 proximal responsive element of the GH1 gene promoter, was first described by Wagner et al.10 and Giordano et al.9 and further by Horan et al.12 Six of nine GH1 gene promoter haplotypes bearing the alternate G at P6 presented lower transcriptional activities and electrophoretic mobility shift assays (EMSA) detected differential protein binding strength, although in vitro studies were unable to identify this SNP as a major determinant of GH1 gene expression level.12 A recent study from Giordano et al.34 has shown a twofold reduced luciferase activity for the G nucleotide bearing promoter haplotype in transfected <span class="blue">rat</span> pituitary cells. Genotypes at P6 had also been associated with decreased breast cancer risk through its association with lower GH secretion and IGF-I circulating levels.16,17
In our results, GH1 gene polymorphic structural variation accounted for only 6·2% of adult height determination in the entire adult population studied and genome-wide linkage analysis of stature in multiple populations revealed no linkage with chromosome 17 GH gene cluster.2,35 As only some of the less frequent SNPs are statistically associated with height, and in view of the high density of SNPs, our study may be hampered by selection bias36 and would ideally have required a wider sampling of some 2 000 individuals; however, this was a highly laborious strategy when the complete sequencing technique is applied. The high density of SNPs and their proximity hamper other genotyping strategies for rapid determination of the complete GH1 SNP map in large control and <span class="yellow">patient</span> populations. Individual SNP associations with height or other GH secretion-related phenotypic traits will require further confirmation by studies in larger populations and by in vitro functional studies.
In conclusion, our study established the GH1 gene sequence variation map in an adult control population with heights normally distributed within the normal range. SNPs and other sequence change contributions to skeletal growth as observed at adult height demonstrated that, despite the high frequency of variation and diversity and complexity of combinations, only some of the less frequent SNPs were associated with taller stature (P17 in intron 1 and P25 in intron 4), even masking the SNP contribution to a shorter one (P6 in the promoter and P12 in the 5′UTR regions, respectively). Systematic GH1 gene analysis in <span class="yellow">patients</span> with growth delay and suspected GH deficiency/insufficiency will clarify whether different SNP frequencies and/or the presence of different sequence changes may be associated with phenotypes in them.<br><br>
<h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="blue">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA2365769</h3>IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome
Abstract
This study was undertaken to evaluate the role of IL-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum IL-5 levels that might be essential for eosinophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-IL-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of IL-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce IL-5.
Research Paper Mediators of Inflammation 5, 24-31 (1996) <br><br> Tins study was undertaken to evaluate the role of 11-5 in eosinophil migration and in the maintenance of eosinophilia in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome. The results show that the infection of animals with <span class="yellow">Toxocara canis</span> induced an early increase in serum 11-5 levels that might be essential for eosi<br><br> IL-5 drives eosinophils from bone marrow to blood and tissues in a <span class="yellow">guinea-pig</span> model of visceral larva migrans syndrome L. H. Faccioli, 1"cA V. F. Mokwa, C. L. Silva, G. M. Rocha, J. I. Araujo, M. A. Nahori 2 and B. B. Vargaftig 2 <br><br> nophil differentiation and proliferation and for the development of eosinophilia. When infected <span class="yellow">guinea-pigs</span> were treated with mAb anti-11-5 (TRFK-5) given at the same time or 1 or 3 days after infection, there was a high percentage of reduction of eosinophil counts 18 days after infection. However, when the mAb was administered during the peak of eosinophilia, there was high inhibition in blood, no inhibition in bronchoalveolar lavage fluid (BALF) or peritoneum and an increase in eosinophil numbers in bone marrow. Thus, a basic level of 11-5 may be essential to drive eosinophils from bone marrow to blood and tissues, and for the maintenance of eosinophilia in infected animals. We may also conclude that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing 11-5. In this way, only one later TRFK-5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens had already stimulated some cell populations to produce 11-5. <br><br> 1Department of Parasitology, Microbiology and Immunology, School of Medicine of Ribeir,5o Preto, 14049-900, Ribeiro Preto, SP, Brazil. Fax: (+55) 16 633 6631 2Unit de Pharmacologie Cellulaire, Unit Associe Institut Pasteur/INSERM n. 285, Paris, <br><br> France. <br><br> CACorresponding Author <br><br> Key words: Eosinophil, Eosinophilia by <span class="yellow">Toxocara canis</span>, IL-5 in eosinophilia, <span class="yellow">Toxocara canis</span> <br><br> Introduction <br><br> treatment in <span class="yellow">mice</span> infected with <br><br> Eosinophilia has been associated with parasitic diseases, particularly when the parasites invade the tissues or injure the mucosal surfaces. <span class="yellow">Toxocara canis</span> is an intestinal parasite of <span class="yellow">dogs</span>, and is the most common aetiologic agent of visceral larva migrans syndrome (VLMS). In <span class="yellow">humans</span>, VLMS results from the ingestion of embryonated eggs of <span class="yellow">T. canis</span>, that eclode in the small intesfine. The infective larvae invade the mucosa, move into the liver via the portal circulation, and from there to the lungs. 2 Beaver et al., 3 who were the first to describe this syndrome, noted the intense eosinophilia which reaches more than 90% of total leucocyte counts. However, there are few studies regarding the mechanisms involved in the blood and tissue eosinophilia obseeeed in VLMS. Several investigators have suggested a direct correlation between eosinophilia and interleukin5 (IL-5) in <span class="yellow">human</span> helminth infections4'5 and in experimental animal models. 67 Inhibition of eosinophilia has been demonstrated by anti-IL-5 24 Mediators of Inflammation Vol 5 <br><br> brasiliensis, 8 $chistosoma mansoni, <span class="yellow">Toxocara canis</span>7 and <span class="yellow">Heligmosomoidespolygyrus</span>. 1 IL-5 has also been shown to support the terminal differentiation, proliferation of eosinophil precursors 11'12 and eosinophil activation. 13 Although IL-5 does not demonstrate eosinophil chemotactic activity in vivo14 there is some evidence suggesting that this cytokine may modulate a selective eosinophil accumulation at the site of inflammation. Moreover, Sehmi et a/.15 reported that IL-5 has a selective priming effect on eosinophil migratory response to nonselective chemoattractant mediators in vitro. Also, Moser et a/. 16 have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with IL-5, IL-3 and GM-CSF. Thus, the involvement of IL-5 in eosinophilia is not fully understood. In the present study we have used a <span class="yellow">guinea-pig</span> model of VLMS to investigate the involvement of IL-5 in eosinophil migration and in the maintenance of eosinophilia in blood, bone marrow, lung and peritoneal cavity. (C) 1996 Rapid Science Publishers <br><br> Nipostrongylus <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> Materials and Methods Animals: Outbred albino weighing 300-400g at the ments were obtained from the School of Medicine of versity of So Paulo, Brazil. <br><br> female <span class="yellow">guinea-pigs</span> start of the experi<br><br> were routinely processed, embedded in paraffin, sectioned at 4-61.tm, and stained with Chromothope 2R and haematoxylin, for examination by light microscopy. Determination oflL-5 in serum: The IL-5 level in the serum of <span class="yellow">guinea-pigs</span> was measured using an enzyme-linked immunosorbent assay (ELISA). Briefly, ELISA plates (96-well Immunoplate MaxiSorp, Nunc, Roskilde, Denmark) were coated with IL-5-specific monoclonal antibody (TRFK-5, 5 lag/ml in phosphate buffered saline, pH 7.4, PBS, 100 l.tl/well). After 2 h of incubation at 37C, the wells were washed four times with PBS containing 0.1% Tween 20 (PBS-T). Then, 100 btl of samples or recombinant <span class="yellow">murine</span> IL-5 standards (0.15-200ng/ml) in PBS-T and 2% BSA (PBSTBSA) were added to each well. After incubation for l h at 37C, the wells were washed three times and 100 l.tl of biotinylated <span class="blue">rat</span> anti-<span class="yellow">murine</span> IL-5 (biotinylated-TRFK-5, 1 lag/ml in PBS-TBSA) was added. After incubation for 1 h at 37C, the wells were washed again three times and 100 of peroxidase-labelled streptavidin (1/1000, Kirkegaard & Perry Laboratories Inc., Maryland, USA) were added to each well. Following incubation for l h at 37C and further washing, the enzyme was developed using the TMB substrate peroxidase for 5 min. The reaction was stopped by adding 501.tl of 2.0 N HCl, and the optical densities were read at 490nm using an automated plate reader. The sensitivity of the assay was 0.15 ng/ml and the upper limit 100ng/ml. <br><br> the animal house of Ribeiro Preto, Uni<br><br> Infection of animals: <span class="yellow">T. canis</span> eggs were obtained by the methods of Olson and Schulz, 17 with minor modifications. Briefly, gravid female <span class="yellow">worms</span> were recovered from <span class="yellow">dogs</span>, and the eggs were rescued from the uterus, washed and allowed to develop to the infective stage in shallow dishes containing 0.5% formalin at 37C. Under light ether anaesthesia, the animals were infected with I ml saline containing 500 <span class="yellow">T. canis</span> eggs, by gastric intubation using a metal cannula. Blood cell counts: <span class="yellow">Guinea-pigs</span> were anaesthetized with sodium pentobarbitone (30mg/kg, i.v.) and blood samples were collected by cardiac puncture with 10% EDTA. Total cell counts were carried out using diluting fluid in a Neubauer chamber. Differential countings were obtained using Rosenfeld-stained cytocentrifuge preparations, <br><br> Bronchoalveolar lavage fluid. The <span class="yellow">guinea-pigs</span> were killed by an overdose of sodium pentobarbitone and 5ml of phosphate-buffered saline (PBS) containing 0.5% sodium citrate (PBS/SC), at room temperature, were instilled through a polyethylene cannula introduced into the trachea. The cells present in the bronchoalveolar lavage fluid (BALF) were recovered immediately. The procedure was repeated once. The leucocyte counts in the BAI_ were determined as described above, Peritoneal cells: The cells from the peritoneal cavities were harvested by injection of 10ml of PBS/SC into the peritoneum. Only 5-8 ml of the <br><br> fluid was withdrawn for cell counts, as described above, <br><br> Monoclonal antibodies: The <span class="blue">rat</span> monoclonal antibody TRFK-5 was a generous gift from Dr P. Minoprio, Institut Pasteur, Paris. The neutralizing antibody was purified by precipitation with ammonium sulfate (45%) from ascites prepared in CD1 <span class="yellow">nude mice</span> (Charles River, St Aubin les Elbeuf, France) inoculated 1 week before the injection of hybridoma cells, with I ml of pristane (Sigma). After precipitation and dialysis of the ascite fluid overnight against PBS, the dialysate was further purified on a Protein G1 column (HiTrapTM, Pharmacia Upsala, Sweden). <br><br> Bone marrow cells: Bone marrow cells were collected by flushing the contents of the <span class="yellow">guinea-pig</span> femur with 10 ml of PBS/SC. Total cell numbers were determined as above. In the differential cell counts the cell populations were divided into mature neutrophils, mature eosinophils and others (mainly precursors and mononuclear <br><br> cells), Histopathological studies: Tissues were removed from <span class="yellow">guinea-pigs</span> at various times post-infection and immediately fixed in 10% formalin. Tissues <br><br> Eosinophil and cytokine depletion: <span class="yellow">Guinea-pigs</span> were injected i.p. with TRFK-5 or with the irrelevant antibody (<span class="blue">rat</span> IgG against total anti-<span class="yellow">human</span> IgG) once, 2mg/animal, at the time of infection or at different intervals (1, 3, 12 or 17 days) thereafter. The animals in this group were sacririced 18 days after infection. liver: One lobule of was used to determine the larval each liver counts from infected <span class="yellow">guinea-pigs</span>. Larval recovery <br><br> Recovery <br><br> of larvae from <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 25 <br><br> L. H. Faccioli et al. <br><br> was evaluated as described by Kayes and Oaks, 18 with minor modifications. Briefly, the tissue was chopped and digested with pepsin-HC1 (pH 1.5<br><br> 1.8) for 2h at 37C. Larval counts for each sample were performed after centrifugation and examination of three 100-l.tl samples under the light microscope. Statistical analysis: Data are presented as the mean _+ S.E.M. and were analysed statistically using the Mann-Whitney test for unpaired data. A p < 0.05 value was considered to be statistically <br><br> significant. <br><br> Results <br><br> of eosinophil counts in blood bone marrow, BALF and peritoneum: <span class="yellow">Guinea-pigs</span> infected with <span class="yellow">T. canis</span> eggs showed a timedependent blood, bone-marrow, BAUV and peritoneal eosinophilia (Fig. 1). The results represent the mean of nine animals obtained in three different experiments. The eosinophil number Kinetics 16<br><br> increased significantly from 0.55 +_ 0.37 x 105 at the beginning of experiment to 6.0 +__ 1.03 x 105 at 6 days post-infection, peaked at day 18 (12.0 _+ 2.31 x 105), and decreased by day 24 (8.11 2.85 x 105) (Fig. 1A). A rise in the percentage of mature eosinophils in bone marrow" was observed 12 days after infection (ranging from 6 2% to 14 2%) and peaked at 18 days (17 2%) (Fig. 1B). As in blood, the number of eosinophils in BALF increased significantly from 0.14 0.06 x 105 to 1.37 _+ 0.35 x 105 at 6 days after infection, reaching a peak at 18 days (10.23 2.62 x 105) with an increase in relative number of as much as 90% in eosinophil counts in relation to controls, and was still elevated at day 24 (9.07___ 3.47 x 10 s) (Fig. 1C). The remaining cells in the BALF were alveolar macrophages, lymphocytes, mast cells and ciliated cells. In contrast to blood and BALF, the number of eosinophils in the peritoneal cavity increased significantl only at day 12 post-infection (onset, 2.06 1.04 x 105; day 6, 3.68 _+ 0.82 x 105; day 12, 5.77 +_ 1.12 x 10>; and increased progres<br><br> __ _ _ _. (B) <br><br> (AI Blood <br><br> 24 20" <br><br> Bone Marrow <br><br> 1612 8 <br><br> 4 O0<br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 111 <br><br> 21 <br><br> 24 <br><br> 16<br><br> (c) BALF <br><br> 16<br><br> ID) Peritoneum <br><br> 12<br><br> __o <br><br> 12<br><br> 8 <br><br> 4 <br><br> 0 3 <br><br> O6 9 <br><br> 12 <br><br> 15 <br><br> 16 <br><br> 21 <br><br> 24 <br><br> Days postinfoction <br><br> FIG. 1. Number of eosinophils in blood, BALF and peritoneal cavity, and percentage, of eosinophils in bone marrow of <span class="yellow">T. canis</span>qnfected <span class="yellow">guinea-pigs</span>. Values are the mean -t-S.E.M. (n=8 to 9). Asterisks indicate a significant difference between infected and noninfected animals (n 5 6). *p < 0.05 and **p < 0.01. 26 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> sively until day 24, 12.44 4- 2.72 x 105) (Fig. 1D). The percentage of eosinophils in some animals reached 55% at the peak of infection. No increase in the number of mononuclear cells was seen in any compartment analysed. Larval counts: The percentage of inoculated <span class="yellow">T. canis</span> larvae recovered by peptic digestion of the liver of experimental animals 4 h and 1, 2, 3, 4, 9, 12 and 18 days after inoculation of 500 eggs per animal is shown in Fig. 2. Most of the larvae were recovered 2 to 4 days after infection and 10% recovery was also observed on day 18 in the liver of the animals. <br><br> 150 <br><br> 120 <br><br> IL-5 level in serum of infected animals: IL-5 was measured in the serum of infected and normal <span class="yellow">guinea-pigs</span>. Each time point in Fig. 3 represents the mean of results from three to five infected animals, and from six controls. Two peaks of IL5 were present in the serum of infected <span class="yellow">guineapigs</span> 1 day after infection (102 __+ 22 pg/ml), and 18 days later (59 7 pg/ml). The level of IL-5 in the controls was 31 4- 4 pg/ml. Eosinophil numbers in infected animals treated with TRFK-5: When <span class="yellow">guinea-pigs</span> received an i.p. injection of TRFK-5, the monoclonal antibody against IL-5, at the time of egg administration or 1 day later, the number of eosinophils in blood, BALF, peritoneal cavity and bone marrow was <br><br> _ <br><br> 30<br><br> 0 <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 12 <br><br> 18 <br><br> Days postinfection <br><br> FIG. 3. IL-5 concentration in serum of <span class="yellow">T. cani</span>infected <span class="yellow">guineapigs</span> (n 3-5). Basal IL-5 concentrations were of 31 4 pg/ml (n= 12). <br><br> _ <br><br> 24 <br><br> 60<br><br> 50<br><br> 40<br><br> 0 <br><br> "T" <br><br> 1" <br><br> 0 <br><br> 3 <br><br> 6 <br><br> 9 <br><br> 12 <br><br> 15 <br><br> 18 <br><br> 21 <br><br> 24 <br><br> Days postinfection FIG. 2. Percentage of <span class="yellow">T. canis</span> larvae recovered from liver of <span class="yellow">guinea-pigs</span> studied at various times post-infection. Data obtained from five animals. <br><br> drastically reduced, even when determined 18 days after infection (Table 1). No inhibition of eosinophil counts was observed when the animals were inoculated with the irrelevant antibody at the time of infection (Table 1). Fig. 4 shows the comparative results of eosinophilia obtained when the antibody was given 3 days or 17 days after egg inoculation. The antibody given at 3 days after infection induced a high percentage of inhibition in eosinophil counts in all the compartments analysed 18 days after infection (Fig. 4A). However, when TRFK-5 was administered to the infected animals on day 17 post-infection (thus 1 day before sacrifice), a significant inhibition in number and percentage of eosinophils was observed only in the blood (p=0.030) (Fig. 4B). A small non-significant decrease was seen in BALF (p 0.790) and peritoneum (p= 0.222). Moreover, the number of mature eosinophils in bone marrow increased by 140% (p 0.038). As demonstrated in Fig. 4B, the behaviour of eosinotShilia in BALF was completely different from that observed in blood. Thus, to better understand the eosinophilia in the lungs of infected animals, we monitored eosinophil numbers in BALF after administration of TRFK-5 at the same time, or 1, 3, 12 or 17 days after infection. The animals were sacrificed 18 days after infection. In another group, TRFK-5 was administered 18 days post-infection and the animals were sacrificed 6 days later. When the mAb was administered at the same time or 1 or 3 days postinfection there was a significant inhibition in the number of eosinophils (Fig. 5). These data show Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 27 <br><br> L. H. Faccioli et al. Table 1. Eosinophils in T. caniinfected <span class="yellow">guinea-pigs</span> treated or untreated with TRFK-5 Compartment <br><br> Time of sacrifice <br><br> (days) <br><br> Non18 24 18 24 18 24 18 24 <br><br> treated <br><br> Blood <br><br> 12.18 5.28 10.08 -t- 2.85 16.50 4.42 6.57 +__ 1.68 <br><br> BALF Peritoneal cavity <br><br> 12.31 -I- 2.35 12.44 2.33 9 6 <br><br> Bone marrow <br><br> _ _ _ _ _ ___ _ __ __ _ __ Days of treatment with TRFK-5 after egg administration <br><br> Irrelevant Ab at the time of infection <br><br> 0 <br><br> 3 <br><br> 12 <br><br> 17 <br><br> 18 <br><br> (n 6) <br><br> (n=4) <br><br> (n=4) <br><br> (n 5) <br><br> (n 5) <br><br> (n=4) <br><br> 0.14 -t- 0.14" <br><br> 0.26 0.79 <br><br> 0.26* 0.24* <br><br> 0.17 _-t- 0.17" <br><br> 0.15 <br><br> 0.15" <br><br> 2.59 <br><br> 0.80* 4.36 2.21 <br><br> 13.32 <br><br> 4.54 <br><br> O. 15 <br><br> 0.15* <br><br> 0.36 <br><br> 0.10" <br><br> 1.22 -t- 0.50* <br><br> 7.07 -t- 2.39 <br><br> 15.24 <br><br> 29.31 _+ 15.90 16.09 <br><br> 10.33 _+ 4.10 3.79 <br><br> 1.47 -t- 0.76* 3 <br><br> 0.48 <br><br> _+ 0.35* <br><br> 0.46 2.6 <br><br> 0.27* 1.3" <br><br> 2.89 <br><br> 0.94* <br><br> 10.42 <br><br> 4.00 <br><br> 1.58" <br><br> -+ <br><br> 1" <br><br> 1.25 -I- 0.25* <br><br> + <br><br> 5 <br><br> 15 -t- 2 <br><br> 9_+1 <br><br> 2 <br><br> 0.7* <br><br> In blood BALF and peritoneal cavity the values represent mean -t-_ S.E.M. x 10 eosinophils. *p < 0.05. <br><br> ml-1 <br><br> and in bone marrow mean -t-_ S.E.M. of the percentage of mature <br><br> that the inhibition of the first peak of IL-5 which appeared at 1 to 3 days after infection as shown in Fig. 3, is also very important for the establishment of eosinophilia in the lungs. However, 24- (A) <br><br> when the mAb was administered 12, 17 or 18 days after infection there was no significant inhibition in the numbers of eosinophils in BALF (Fig. 5), showing that once established, the eosi18<br><br> 15x <br><br> 16<br><br> 12<br><br> ._ . 0 <br><br> 12 <br><br> o <br><br> BIo o d <br><br> BALF <br><br> P e rit oneum <br><br> Bone marrow <br><br> 24- (Is) <br><br> 1815<br><br> E x <br><br> 20<br><br> 16<br><br> o <br><br> 12///. <br><br> o = o <br><br> 12<br><br> 0 Blood <br><br> O" <br><br> BALF <br><br> Peritoneum <br><br> Bone marrow FIG. 4. Number of eosinophils in blood, BALF and peritoneum and percentage of eosinophils in bone marrow of <span class="yellow">T. cani</span>infected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) 2 mg/animal at 3 days post-infection; (B) 2 mg/animal 17 days post-infection. The treated and control animals were sacrificed 18 days after infection. Asterisks indicate a significant difference from infected controls (n=5-6) and from animals treated with TRFK-5 (n=4- 5). *p < 0.05 and **p < 0.01. 28 Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> 24<br><br> E <br><br> 20<br><br> heart; data not shown) and muscle, as reported by other investigators, 8 were infiltrated. The factors responsible for in vivo eosinophil accumulation at inflammatory sites have been <br><br> 0 <br><br> TRFK-5 i.p. <br><br> 0 <br><br> 3 <br><br> 12 17 <br><br> A <br><br> B <br><br> FIG. 5. Number of eosinophils in BALF of T. caniinfected <span class="yellow">guinea-pigs</span> submitted or not to treatment with TRFK-5. (A) animals were sacrificed 18 days post-infection and (B) 24 days after infection. Asterisks indicate a significant difference from infected control and from animals treated with TRFK-5 (p< 0.01). <br><br> nophilia persists in lungs, probably by the secretion of IL-5 from cells localized in the lung microenvironment, <br><br> Histopathological analysis: The treatment of <span class="yellow">T. canis</span>-infected animals with irrelevant antibody showed a widespread eosinophilic infiltration as in untreated animals (Fig. 6A, B). However, the treatment of animals with TRFK-5 at the same time of infection, or 1 day or 3 days later ted to a complete inhibition of eosinophil infiltration in the lung parenchyma (Fig. 6C). By contrast, the mononuclear cell infiltration in the lungs was not modified. When the infected <span class="yellow">guinea-pigs</span> received TRFK-5 1 day before sacrifice (or 17 days post-infection), eosinophil infiltration in the lung parenchyma was also inhibited (Fig. 6D) but not to the same extent as observed in the group receiving TRFK-5 given at the time of infection or 3 days later. Thus, the histological determination of eosinophil infiltration in these lungs corroborates a reduction but not a sizeable inhibition of eosinophil numbers as observed in the BALF of the same infected animals, <br><br> poorly defined, although T lymphocytes and mast cells appear to be involved in eosinophilia. 9'2 IL-5, a T cell-derived factor that regulates B cell functions, is an eosinophil differentiation factor11 as well as a stimulating and survival-prolonging factor specific for eosinophils in vitro. 2 Also, several investigators have demonstrated that sTstemic eosinophilia in <span class="yellow">mice</span> infected with parasites is mediated by IL-5 produced in response to the infection. 2'22 In the present study, the i.p. administration of the TRFK-5 antibody markedly inhibited the widespread eosinophilia observed in <span class="yellow">T. canis</span>-infected <span class="yellow">guinea-pigs</span>, indicating that IL5 participated in a <span class="yellow">guinea-pig</span> model of VLMS eosinophJlia. Most of the <span class="yellow">T. canis</span> larvae which penetrated the intestinal wall had migrated into the liver within 72h after inoculation as demonstrated here and elsewhere. 2 It is apparently during this interval that the worm provides the signals to cytokine-producing cells, which in turn trigger increased serum levels of specific cytokine as demonstrated here for IL-5, 24 to 72 h after infection. The signals may be provided directly by the invading parasite or by cells in response to the parasite. The cytokine pattern that develops at this early stage, probably induced by a T-cell independent pathway, may also influence the pattern of T cell differentiation into a Th2 type, which may be responsible for the second peak of IL-5 observed in our experimental model (Fig. 3), although a second cycle of larval invasion (Fig. 2) with a rapid peak of IL-5 liberation cannot be ruled out. <br><br> Discussion The results of the present study show that in our experimental model widespread eosinophilia follows the infection of <span class="yellow">guinea-pigs</span> with second stage eggs from <span class="yellow">T. canis</span>, as also noted in <span class="yellow">humans</span> and in other experimental animals. 7'7 <span class="yellow">T. canis</span> is a potent stimulus for systemic eosinophilia, since blood, BALF, peritoneum and all tissues examined (kidney, eyes, spleen, thymus, <br><br> Thus, our results suggest that the eosinophilia against helminth larvae may be initiated by the release of IL-5 when the parasites migrate from the intestine to the liver by stimulation of specific cell populations. Then, an early release of IL-5 quickly induces eosinophil recruitment, probably first from the stored mature eosinophil pool from vascular endothelium or by the mobilization of eosinophils from extravascular sites to the blood. This fact could explain why we found increased eosinophils first in blood and later in other compartments. The early IL-5 release may also serve as a signal for eosinophil differentiation and maturation in bone marrow. The time <br><br> inteeeal observed between the first peak of IL-5 release and the increase of eosinophils in blood coincides with that reported to be necessary for eosinophil differentiation and maturation in vitro. 12Increased eosinophil production and liberation into blood and other tissues occurs Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 29 <br><br> L. H. Faccioli et al. <br><br> FIG. 6. (A) photomicrographs of lung parenchyma from <span class="yellow">guinea-pigs</span> infected for 18 days with <span class="yellow">T. cani</span>,, (B) infected animals which were treated with irrelevant antibody at the time of infection; (C) infected animals which were treated with the mAb TRFK-5 at 3 days after infection; (D) mAb administration 17 days after infection. The animals were sacrificed 18 days after infection. Note the intense eosinophil infiltration into the lung in A and B, the inhibition of eosinophils in C and the reduction of eosinophils in D. <br><br> thereafter. Thus, early and later IL-5 release provides a necessary level of this cytokine, which is involved in the maintenance of eosinophilia. We may assume that the inhibition of the first peak of IL-5 release by TRFK-5 does not permit the subsequent T cell stimulation and differentiation. This may explain the long-lasting effect of TRFK5 treatment observed here and also reported by others. 8 In agreement with our results, there is an important observation of Svetic et al. 24 showing that a specific and highly reproducible IL-5 gene expression pattern is detectable in Peyer's patches by 6 to 12h after <span class="yellow">Heligmosomoides polygyrus</span> infection. The early increase in IL-5 gene expression after infection was probably T cell-independent, inasmuch as it was obseeeed in Peyer's patches of congenitally athymic <span class="yellow">mice</span> and of conventional <span class="yellow">mice</span> treated with anti-CD4 30 Mediators of Inflammation Vol 5 <br><br> and anti-CD8 mAb. Moreover, Kusama eta/. 25 have observed two peaks of eosinophilia in normal and athymic <span class="yellow">mice</span>, and suggested that IL-5 observed in the first peak was produced by cells other than CD4 T cells, since anti-CD4 and anti-CD3 mAb reduced only the second peak of eosinophilia in normal <span class="yellow">mice</span> and slightly reduced the first peak of eosinophilia in both normal and nu/nu <span class="yellow">mice</span>. The local lung cells producing IL-5 may also help us to explain the reason why 12, 17 or 18 days post-infection TRFK-5 treatment only partially inhibits, or does not inhibit eosinophil infiltration into the lungs, as demonstrated in Figs 5 and 6. We may suggest that when eosinophils have already migrated to the lungs, TRFK-5 has no power to inhibit eosinophilia, which is also under control of local lung cells producing IL-5. In this way, only one later TRFK<br><br> 1996 <br><br> IL-5 drives eosinophils in <span class="yellow">guinea-pig</span> <br><br> 5 treatment may not be sufficient to modify the lung parenchyma microenvironment, since <span class="yellow">T. canis</span> antigens have already stimulated some cell populations to produce IL-5, as demonstrated by Kusama et aL25These results suggest that eosinophilia in lungs is under the control of different factors when compared to that observed in blood and the peritoneal cavity. One of the most important results obtained here was the inhibition of circulating eosinophil numbers by the different mAb treatments, even when the antibody was given at the peak of blood eosinophilia, which was accompanied by an increase of mature eosinophils in bone marrow. This suggests that IL-5, apart from being required for the terminal differentiation of eosinophils in bone marrow, 26 is also likely to drive eosinophils from the bone marrow to the blood and then to the tissues, probably by upregulating VLA-4 expression in eosinophils. Moser et aL have demonstrated that in order to acquire the ability to transmigrate, eosinophils must be primed with cytokines such as IL-5, IL-3 or GM-CSF for expression of adhesion molecules such as VI-4. Recently, Pretolani et al. 27 have indeed shown that an anti-VLA-4 antibody suppresses eosinophil recruitment to lung in the <span class="yellow">guinea-pig</span> and, as a consequence, inhibits the accompanying bronchopulmonary hyperresponsiveness. <br><br> tant in protective immunity to a gastrointestinal nematode infection in <span class="yellow">mice</span>. Proc Natl Acad Sci USA 1991; 88= 5513-5517. 11. Yamaguchi A, Suda T, Suda J, et al. Purified interleukin (IL-5) supports the terminal differentiation and proliferation of <span class="yellow">murine</span> eosinophilic <br><br> precursors. J Exp Med 1988; 16'7: 43-56. 12. Yamaguchi Y, Hayashi Y, Sugama Y, et al. Highly purified <span class="yellow">murine</span> interleukin (IL-5) stimulates eosinophil function and prolongs in vitro survival. IL-5 as an eosinophil chemotactic factor. J Exp Med 1988; 16'7: 17371752. 13. Rothenberg ME, Petersen J, Stevens RL, Silberstein DS, McKenzie DT, Austen KF, Owen WF. IL-5-dependent conversion of normodense <span class="yellow">human</span> eosinophils to the hypodense phenotype uses 3T3 fibroblasts for enhanced viability, accelerated hypodensity, and sustained antibodydependent cytotoxicity. J Immuno11989; 143; 2311-2316. 14. Collins PD, Weg VB, Faccioli LH, Watson ML, Moqbel R, Williams TJ. Eosinophil accumulation induced by <span class="yellow">human</span> interleukin-8 in the <span class="yellow">guinea pig</span> in vivo. Immunology 1993; '7}; 312-318. 15. Sehmi R, Wardlavo AJ, Cromwell O, Kurihara K, Waltmann P, Kay AB. Interleukin-5 selectively enhances the chemotactic response of eosinophils obtained from normal but not eosinophilic subjects. Blood 1992; '79; 2952-2959. 16. Moser R, Fehr J, Bruijnzeel PLB. IL-4 controls the selective endotheliumdriven transmigration of eosinophils from allergic individuals. J Immunol 1992; 149: 1432-1438. 17. Olson LJ, Schulz CW. Nematode induced hypersensitivity reactions in <span class="yellow">guinea pigs</span>: onset of eosinophilia and positive Schultz-Dale reactions following graded infection with <span class="yellow">Toxocara canis</span>. Ann N Y Acad Sci 1963; 113; 440-455. 18. Kayes SG, Oaks <br><br> JA. Development <br><br> of the granulomatous response in <br><br> <span class="yellow">murine</span> toxocariasis. I. Initial events. Am J Patho11978; ,}3; <br><br> 277-294. <br><br> 19. Basten A, Beeson PB. Mechanisms of eosinophilia. II. Role of the lymphocyte. J Exp Med 1970; 131; 1288-1305. 20. Plaut M, Pierce JH, Watson CJ, Hanley-Hyde J, Nordan RP, Paul WE. Mast cell lines produce lymphokines in response to cross-linkage of Fc epsilon RI or to calcium ionophoras. Nature 1989; 339: 64-67. is 21. Sher A, Coffman RL, Hieny S, Scott P, Cheever AW. Interleukin required for the blood and tissue eosinophilia but not granuloma formation induced by infection with <span class="yellow">Schistosoma mansoni</span>. Proc Natl Acad Sci USA 1990; 8'7; 61-65. 22. Herndon FJ, Kayes SG. Depletion of eosinophils by anti-IL-5 monoclonal antibody treatment of <span class="yellow">mice</span> infected with <span class="yellow">Trichinella spiralis</span> does not alter parasite burden or immunologic resistance to reinfection. J Immuno11992; 149: 3642-3647. 23. Oshima T. Standardization of techniques for infecting <span class="yellow">mice</span> with Tox24. ocara canis and observations on the normal migration routes of the larvae. J Parasito11961; 4'7= 652. Svetic A, Madden KB, Zhou XD, et al. A primary intestinal helminthic infection rapidly induces a gut-associated elevation of Th2-associated cytokines and IL-3. J Immuno11993; 150: 3434-3441. Kusama Y, Takamoto M, Kasahara T, Takatsu K, Nariuchi H, Sugane K. Mechanisms of eosinophilia in BALB/c-nu/+ and congenitally athymic BALB/c-nu/nu <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Immunology 1995; 84; 461-468. Rennick DM, Thompson-Snipes L, Coffman RL, Seymour BWP, Jackson JD, Hudak S. In vivo administration of antibody to interleukin-5 inhibits increased generation of eosinophils and their progenitors in bone marrow of parasitized <span class="yellow">mice</span>. Blood 1990; '76: 312-316. Petrolani M, Ruffle C, Lapa e Silva JR, Joseph D, Lobb RR, Boris Vargaftig B. Antibody to very late activation antigen 4 prevents antigen-induced bronchial hyperreactivity and cellular infiltration in <span class="yellow">guinea pig</span> airways. J Exp Med 1994; 180= 795-805. <br><br> References 25. 1. Nutman <br><br> 2. <br><br> 3. 4. 5. <br><br> 6. 7. <br><br> 8. <br><br> 9. <br><br> TB, Ottesen EA, Cohen SG. The eosinophil, eosinophilia, and eosinophil-related disorders. Allergy Proc 1989; 10; 47-62. Glickman LT, Schantz PM. Epidemiology and pathogenesis of zoonotic toxocariasis. Epidem Rev 1981; 3= 230-250. Beaver P, Snyder H, Carrera G, Dent J, Lafferty J. Chronic eosinophilia due to visceral larva migrans. Pediatrics 1952; 9: 7-19. Limaye AP, Abrams JS, Silver JE, Ottesen EA, Nutman TB. Regulation of parasite-induced eosinophilia: selectively increased interleukin production in helminth-infected <span class="yellow">patients</span>. J Exp Med 1990; l'7:a; 399-402. Steel C, Nutman TB. Regulation of IL-5 in onchocerciasis: a critical role for IL-2. J Immuno11993; 15{}; 5511-5518. Yamaguchi Y, Matsui T, Kasahara T, et al. In vivo changes of hemapoietic progenitors and the expression of the interleukin gene in eosinophilic <span class="yellow">mice</span> infected with <span class="yellow">Toxocara canis</span>. Exp Hemato11990; 18; 1152-1157. Parsons JC, Coffman RL, Grieve RB. Antibody to interleukin prevents blood and tissue eosinophilia but not liver trapping in <span class="yellow">murine</span> larval toxocariasis. Parasite Immuno11993; 15: 501-508. Coffman RL, Seymour BWP, Hudak S, Jackson J, Rennick D. Antibody to interleukin-5 inhibits helminth-induced eosinophilia in <span class="yellow">mice</span>. Science 1989; 245; 308-310. Sher A, Coffman RL, Hieny S, Cheever AW. Ablation of eosinophil and IgE responses with anti-IL-5 or anti-IL-4 antibodies fails to affect immunity against <span class="yellow">Schistosoma mansoni</span> in the <span class="yellow">mouse</span>. J Immuno11990; 145: 3911<br><br> 26. <br><br> 27. <br><br> ACKNOWLEDGEMENTS. This work was funded by grant 300652/85-2 from Conselho Nacional de Desenvolvimento Cientifico e Tecnol6gico (CNPq) and Grant 92/5105-7 from Fundago de Amparo ft Pesquisa do Estado de So Paulo (FAPESP). We wish to thank Mrs M.A. Fernandes for technical assistance, Mrs M.M.O. Rossi for the histological sections, M. Costa Gongalves for the photography artwork, and Dr P. Minoprio, Institut Pasteur, for kindly providing TRFK-5. <br><br> 3916. 10. Urban <br><br> Jr JF, <br><br> Katona IM, Paul WE, Finkelman FD. Interleukin 4 is impor<br><br> Received 12 October 1995; accepted 17 November 1995 <br><br> Mediators of Inflammation Vol 5 <br><br> 1996 <br><br> 31 <br><br>  <h3>pmcA2377325</h3>Purinergic signalling in the subretinal space: a role in the communication between the retina and the RPE
Abstract
The retinal pigment epithelium (RPE) is separated from the photoreceptor outer segments by the subretinal space. While the actual volume of this space is minimal, the communication that occurs across this microenvironment is important to the visual process, and accumulating evidence suggests the purines ATP and adenosine contribute to this communication. P1 and P2 receptors are localized to membranes on both the photoreceptor outer segments and on the apical membrane of the RPE which border subretinal space. ATP is released across the apical membrane of the RPE into this space in response to various triggers including glutamate and chemical ischemia. This ATP is dephosphorylated into adenosine by a series of ectoenzymes on the RPE apical membrane. Regulation of release and ectoenzyme activity in response to light-sensitive signals can alter the balance of purines in subretinal space, and thus coordinate communication across subretinal space with the visual process.<br><br>Introduction
The retinal pigment epithelium (RPE) lies between the outer segments of the photoreceptors and the choroidal blood supply (Fig. 1). The RPE combines the functions of epithelial and glial cells, providing a barrier while also supporting the neural photoreceptors and modulating their function. Tight communication between photoreceptors and the RPE is critical to coordinate the multiple levels of interaction, and the purinergic contribution to this communication is becoming apparent. The relevance of this purinergic input is emphasized by the many functional effects of P1 and P2 receptor stimulation and by the multiple mechanisms in place to regulate subretinal levels of purine agonists. As the dynamics of ATP release and extracellular conversion into adenosine will modify agonist availability, the modulation of these processes can exert a temporal control on purinergic signaling. The following review will outline the main interactions between the RPE and photoreceptors, describe the effects of stimulating purinergic receptors on both sides of subretinal space, and summarize how levels of ATP, ADP, and adenosine are manipulated in this microenvironment.
Fig. 1Schematic illustration of the key components of purinergic signaling in the subretinal microenvironment. Stimulation of P2 receptors on the RPE can enhance transepithelial fluid absorption while P1 receptors can modulate phagocytosis. ATP released through CFTR and other Cl− channels can stimulate P2 receptors or be converted to ADP, AMP, and adenosine (Ado) by a series of ectonucleotidases present on the apical membrane of the RPE. By controlling the balance of extracellular purines available to stimulate these receptors these mechanisms can control levels of endogenous purines available to activate the receptors. While theoretically possible, it remains to be determined whether these subretinal purines can actually stimulate photoreceptors
Purines and subretinal space
RPE-photoreceptor interactions across the subretinal space
The outer segments of the rods and cones are responsible for the initial stages of vision, converting photon energy into a series of enzymatic reactions that close the light-sensitive channels on the photoreceptor plasma membrane, hyperpolarize the cells, and reduce the release of glutamate from the synaptic terminals [1, 2]. Efficient photoreceptor function depends upon both short-term and long-term support from the RPE. The critical nature of these interactions is evident from the rapid degeneration of photoreceptors in the absence of a healthy RPE layer and by the RPE localization of defective gene product in some forms of hereditary photoreceptor degeneration [3].
The apical membrane of the RPE is separated from the plasma membrane of the outer segments by an extracellular space of only 10–20 nm [4]. Although small, this subretinal space contains a highly structured matrix which ensheathes the outer segments and extends to the RPE [5, 6]. The presence of enzymes within this interphotoreceptor matrix emphasizes that this extracellular space itself is functionally active [7, 8]. This intimate anatomical relationship between photoreceptors and the RPE reflects multiple functional interactions. For example, the RPE accepts, recycles, and exports central components of the phototransduction process [9]. The outer segments are continuously resynthesized, and the phagocytosis, degradation, and processing of shed outer segment tips by the RPE cells is central to this renewal [10]. The ion channels and transporters on the apical membrane of the RPE help regulate the ionic composition of the subretinal space [11]. As extracellular levels of ions can modify the ionic driving forces across the photoreceptor plasma membrane, these RPE transporters can influence the state of neural activity. The transport of fluid and ions from the apical membrane to basolateral membrane of the RPE is also one of the main forces keeping the retina attached [12].
The control of photoreceptor activity by light gives a rapid temporal dependence to some interactions between the photoreceptors and the RPE. The volume of subretinal space increases in response to light [13], linking small changes in the ionic composition of the subretinal space with activity of the RPE transport mechanisms which maintain this volume [14, 15]. Other processes are controlled on a diurnal cycle. The shed tips of the outer segments are phagocytosed by the RPE soon after the onset of light [16, 17]. These processes can both be modulated by purine levels in subretinal space, indicating purinergic regulation is important over multiple time scales.<br><br>Purinergic receptors on photoreceptors
A2 adenosine receptors were localized to both the inner and outer segments of photoreceptor outer segments over a decade ago by Blazynski and colleagues [18], with more recent reports emphasizing their functional role. A2 agonists inhibit the L-type Ca2+ channel on rod outer segments [19] and can inhibit the synaptic release of glutamate from rods, suggesting changes in adenosine levels in subretinal space could modulate light sensitivity [20]. The A2 agonist DPMA and the adenosine deaminase inhibitor EHNA reduce mRNA for opsin in rods, suggesting that endogenous levels of adenosine can downregulate opsin message at night [21]. EHNA and the A2A receptor agonist CGS21680 also increase the survival of <span class="yellow">chick</span> embryonic photoreceptors in culture [22], indicating a long-term neuroprotective role for adenosine.
P2 receptors are also present in the photoreceptors. mRNA for the P2X2 receptor is expressed in the photoreceptor cell bodies, with immunohistochemical localization of the protein to outer segments [23]. In situ hybridization indicates the photoreceptor layer has the highest level of P2Y2 receptor of any region in the <span class="yellow">rabbit</span> retina, although staining was not pronounced in monkey [24]. P2X7 receptors have recently been localized to photoreceptor synaptic terminals, with evidence for ecto-ATPase activity in the synapse, and functional evidence suggesting ATP augments transmission of the light response by rods [25]. It was suggested that ATP might be co-released from photoreceptors with glutamate, although this remains to be tested directly.<br><br>Purinergic receptors on the RPE
Stimulation of P1 receptors can have a considerable impact on RPE cells. A2 receptors have been recognized on cultured and fresh RPE cells for some time [26, 27], with in situ hybridization confirming the presence of A2A receptors in <span class="blue">rat</span> RPE [28]. Stimulation of A2 receptors reduces the rate of rod outer segment phagocytosis by RPE cells [29], while application of adenosine to the apical membrane of <span class="yellow">chick</span> RPE cells increases the basolateral Cl- conductance, the transepithelial potential, and the c-wave, and decreases the hyperpolarization of the basal membrane in response to light [30]. Although adenosine alone does not increase intracellular Ca2+ levels [31], adenosine acts synergistically with ATP to elevate Ca2+ levels in <span class="yellow">human</span> RPE cells by stimulating both A1 and A2A receptors [32, 33]. Stimulation of A1 receptors with high doses of NECA increases the active transport of fluorescein across the RPE, while activation of A2A receptors decreases this transport, and by extension, transport of the ions that underlie fluid movement [34]. Stimulation of A1 and A2A receptors produces analogous increases and decreases, respectively, in the absorption of subretinal fluid blebs. This is consistent with the negative coupling of the A1 receptor and the positive coupling of the A2 receptors to adenylate cyclase, as increasing cAMP inhibits the transport of fluid across the RPE towards the choroid [35–37]. The agonist 2-Cl adenosine reverses the deficit in phosphoinositide metabolism found in diabetic RPE cells [38], suggesting effects on metabolism in addition to transport and phagocytosis.
Multiple P2 receptors have been localized to the RPE. The P2Y2 receptor was initially characterized in cultured <span class="yellow">human</span> RPE [31], with subsequent reports localizing transcript for P2Y1, P2Y2, P2Y4, and P2Y6 in the <span class="blue">rat</span> RPE/choroid [39] and for P2Y1 and P2Y12 receptors in ARPE-19 cells [40], and functionally identifying a P2X receptor in <span class="blue">rat</span> RPE cells [41]. ATP, ADP, and UTP induce numerous effects on RPE physiology [32, 33, 42, 43]. While these effects likely involve multiple different receptor types, the contributions of the P2Y2 receptor have been explored in most detail to date. The P2Y2 receptor has been specifically localized to the apical membrane of fresh <span class="yellow">bovine</span> RPE cells, and addition of ATP to this membrane transiently elevates Ca2+, activates a basolateral Cl- conductance, inhibits an apical K+ conductance, and increases the apical to basolateral flow of fluid [43]. This increased absorption of fluid from the subretinal space suggests P2Y2 receptor stimulation could reduce the excess fluid that accumulates in retinal edemas, and several reports have supported this theory. ATP, UTP, and the P2Y2 receptor agonist INS37217 decrease the size of subretinal fluid blebs when injected into subretinal space of <span class="yellow">rats</span> [44]. In both normal and rds +/- <span class="yellow">mice</span> with experimentally induced detachment, INS31217 improves the ERG recovery and decreased cell death [45]. INS37217 also reduces subretinal blebs in <span class="yellow">rabbits</span> [46]. Injection of another P2Y2 agonist, INS542, increases the active transport of fluorescein across the RPE, consistent with this upregulation of ion and fluid transport across the tissue [47]. Together these experiments emphasize the clinical potential of treating retinal edema with P2Y2 agonists.<br><br>
Regulation of purine levels in subretinal space
While synthesized purinergic agonists may prove useful in treating some ocular disorders, the endogenous activation of P1 and P2 receptors in the subretinal microenvironment will ultimately be determined by availability of agonists. These levels are largely controlled by the release of ATP into the subretinal space, its conversion into other purines including adenosine, and the manipulation of adenosine by enzymes or transporters. Recent work has increased our understanding of both the stimuli that initiate changes in subretinal purine levels and the mechanisms that mediate these changes.
Release of ATP by the RPE
At least some of the ATP capable of stimulating the purinergic receptors on RPE cells is released from the RPE itself. The resulting autocrine stimulation ensures local delivery, and control, of purines to initiate the physiologic changes in the RPE. The release of ATP by RPE cells is triggered by multiple stimuli including osmotic stress, bFGF, UTP, NMDA, glutamate, and ATP [39, 40, 48–51]. The ATP release following activation of NMDA receptors by glutamate may have the most interesting implications for communication across subretinal space, given that glutamate confers the light signal from photoreceptors to the rest of the visual system. Glutamate and the specific receptor agonist NMDA triggers ATP release from ARPE-19 cells, with the release inhibited by NMDA antagonist MK-801, and by DCKA, which inhibits the glycine B binding site on NMDA receptors [51, 52]. Although NMDA raises intracellular Ca2+ levels, this increase is prevented by eliminating ATP with apyrase, indicating autostimulation through released ATP is responsible for this Ca2+ signal. NMDA also triggers a release of ATP when applied to the intact <span class="yellow">bovine</span> RPE eyecup [51]. The NMDA receptors and the ATP release sites have been functionally identified to the apical membrane of the <span class="yellow">bovine</span> RPE, suggesting the neurotransmitter interactions could amplify the signal from any glutamate reaching subretinal space.
The ability of both UTP and ATP to stimulate release of ATP from the RPE supports the theory that the system acts to amplify signals. When applied at greater than 1 μM, ATP triggers a secondary release of ATP peaking 10 min after the initial stimuli [40]. UTP also initiates a release in extracellular ATP with a similar delay [48]. The rise in ATP triggered by UTP is inhibited by the Cl- channel blocker NPPB, and UDP is much less effective at triggering release than UTP; both observations suggest the P2Y2 receptor contributes to the increase in ATP more than diphosphokinase, although influence from the enzyme cannot be ruled out [53].
Recent evidence suggests that ischemia may lead to the release of ATP from RPE cells. Chemical ischemia triggers a substantial ATP release from cardiac myocytes [54], while changes in oxygen levels trigger ATP release in central chemoreceptors [55]. We found that exposure to sodium cyanide led to a rapid release of ATP from ARPE-19 cells (Fig. 2). As hypoxic and/or ischemic challenge may lead to changes in the expression of growth factors in RPE cells during certain ocular disorders such as macular degeneration [56], and as purines can induce expression of VEGF in other cells [57], this ATP release may contribute to growth factor signaling by the diseased RPE.
Fig. 2Chemical ischemia triggers ATP release from ARPE-19 cells. ATP release was measured in the bath directly from cells plated in 96-well plates to which the luciferin- luciferase reaction mixture was added [51]. Left Levels of ATP in the bath after addition of 5 mM NaCN to the cells. Measurement began 1 min after addition of NaCN or control solution to wells. Right Levels of ATP measured at the peak, 3 min after addition of NaCN (n = 12). Levels were normalized to concurrent levels in control (n = 14). Symbols and bars represent mean ± SE, *p < 0.001
The particular conduit for ATP release varies with the stimuli. The release in response to hypotonic challenge is largely dependent upon CFTR, as it was prevented by the specific CFTR inhibitor CFTR172 in addition to the more general blocker glybenclamide [50]. While the precise mechanisms by which CFTR contributes to this release are not yet known, a role for CFTR in ATP release into subretinal space is consistent with the reduction of certain ERG components in cftr -/- <span class="yellow">mice</span> [58] and with the ability of apical ATP to activate conductances associated with these ERG components [43]. The release of ATP is also largely blocked by the vesicular transport inhibitor brefeldin A, suggesting the two processes occurred in series whereby ATP efflux follows the insertion of vesicles containing CFTR into the plasma membrane. Although the Ca2+ chelator BAPTA blocks this ATP release [50], raising Ca2+ alone with ionophore ionomycin does not itself initiate release [48]. This necessary but not sufficient contribution of Ca2+ also supports a role for vesicular insertion.
In contrast to the release following hypotonic challenge, the ATP release in response to NMDA does not involve CFTR [51]. Release is blocked by NPPB, however, suggesting another type of anion channel could serve as a conduit for ATP release. The presence of parallel mechanisms coexisting on the same cell for ATP release triggered by either agonists or by cell swelling has also been reported in astrocytes [59] and may reflect the multiple roles of purinergic signaling within a given tissue. As both stimuli lead to release across the apical membrane into subretinal space, both are expected to influence signaling in the microenvironment.<br><br>Interconversion of purines in subretinal space
The interconversion of nucleotides and nucleosides each capable of stimulating distinct receptors makes the purinergic signaling system of particular interest in a confined region such as the subretinal space. The main enzymes responsible for dephosphorylating extracellular ATP on the RPE cells have been analyzed and a basic understanding of their regulation has begun. This section first describes the enzymes that act on ATP and ADP, followed by enzymes which convert AMP into adenosine.
The dephosphorylation of extracellular ATP by RPE cells involves enzymes from multiple families [40], as found in airway epithelial cells [60]. Degradation of ATP by the apical membrane of the fresh <span class="yellow">bovine</span> eyecup and by ARPE-19 cells is inhibited by ARL67156 or βγmATP. Message for eNPP1, eNPP2, and eNPP3 is present in ARPE-19 cells, consistent with the preference of βγmATP for members of the eNPP family [61]. The cells also express NTPDase2, and NTPDase3, although the intermittent presence of NTPDase1 likely reflects a regulated process [40]. Ecto-alkaline phosphatase has no effect on ATP degradation in RPE cells, in contrast to its considerable contribution in airway epithelium [62]. The putative contribution from diphosphokinases to interconversion of subretinal purines is presently unknown.
Extracellular AMP is rapidly dephosphorylated into adenosine in subretinal space. The production of adenosine from ATP at the apical membrane of the <span class="yellow">bovine</span> RPE eyecup is inhibited by the ecto-5′-nucleotidase inhibitor αβmADP, confirming a role for this enzyme [63]. The enzyme is localized to <span class="blue">rat</span> RPE and ARPE-19 cells immunohistochemically. Degradation of 5′AMP is highest near the subretinal space of <span class="blue">rat</span> retina [63], although localization in <span class="yellow">mouse</span> indicated larger amounts of ecto-5′-nucleotidase at the tips of adjacent Müller cells [64]. Levamisole does not inhibit the dephosphorylation of 5′AMP by the RPE, consistent with the absence of substantial ecto-alkaline phosphatase in subretinal space.
The presence of light may alter the levels of adenosine in subretinal space. Epinephrine is released at the onset of light [65] and stimulation of the RPE with epinephrine can decrease activity of ecto-5′-nucleotidase [63]. While norepinephrine and phenylephrine lead to similar decreases in enzyme activity, prazosin and corynanthine block the effects of norepinephrine, implicating the α1 epinephrine receptor in the inhibition of ecto-5′-nucleotidase [63]. The kinetics of inhibition are consistent with cleavage of the nucleotidase from its GPI anchor. The phagocytosis of rod outer segments is maximal shortly after light onset [16], and this phagocytosis is inhibited by adenosine [29]. The ability of epinephrine released by the illuminated retina to reduce ecto-5′-nucleotidase activity and consequently adenosine levels may relieve this inhibition and enhance the rate of phagocytosis at light onset.<br><br>
Physiologic effects of subretinal purines on the RPE and photoreceptors
The number of purinergic receptors on both photoreceptor and RPE membranes suggests purines make multiple contributions to the physiology of the outer retina. Our increased understanding of how agonist levels in subretinal space are controlled has begun to indicate how and when this contribution may occur. Future research will involve applying these findings from isolated systems to intact RPE-photoreceptor models, and pursuing the role of defective purinergic regulation in ocular disease. While it is unlikely that ATP released across the apical membrane of the RPE can diffuse to these P2 receptors in the outer plexiform layer given the ecto-ATPase activity in the synaptic clef [25], stimulation of receptors elsewhere on the photoreceptor membrane is possible. It would be interesting to determine whether ATP released from the RPE and converted to adenosine by ecto-nucleotidases can actually modulate the response to light by stimulating the A2A receptors on photoreceptor outer segments. The impact of purinergic signaling on chronic ocular diseases is also of interest, such as the role of ischemia-driven ATP release in VEGF production. While the small size of subretinal space can complicate pharmacologic manipulation within the intact RPE-photoreceptor complex, molecular approaches may provide new insight into how endogenous purines in subretinal space affect the physiology, and pathophysiology, of both RPE and photoreceptors.<br><br><br><br><h3>pmcA2430230</h3>Emergence of Delayed Methylmercury Toxicity after Perinatal Exposure in Metallothionein-Null and Wild-Type C57BL <span class="yellow">Mice</span>
Abstract
Background
Although a long latency period of toxicity after exposure to methylmercury (MeHg) is known to exist in <span class="yellow">humans</span>, few animal studies have addressed this issue. Substantiation of delayed MeHg toxicity in animals would affect the risk evaluation of MeHg.<br><br>Objectives
Our goal in this study was to demonstrate the existence of a latency period in a <span class="yellow">rodent</span> model in which the toxicity of perinatal MeHg exposure becomes apparent only later in life. Our study included metallothionein (MT) knockout <span class="yellow">mice</span> because studies have suggested the potential susceptibility of this strain to the neurodevelopmental toxicity of MeHg.<br><br>Methods
Pregnant MT-null and wild-type C57Bl/6J <span class="yellow">mice</span> were exposed to MeHg through their diet containing 5 μg Hg/g during gestation and early lactation. We examined behavioral functions of the offspring using frequently used paradigms, including open field behavior (OPF), passive avoidance (PA), and the Morris water maze (MM), at ages of 12–13 and 52–53 weeks.<br><br>Results
At 12 weeks of age, behavioral effects of MeHg were not detected, except for OPF performance in MeHg-exposed MT-null females. At 52 weeks of age, the MeHg-exposed groups showed poorer performance both in PA and MM, and their OPF activity differed from controls. These effects of MeHg appeared exaggerated in the MT-null strain. The brain Hg concentration had leveled off by 13 weeks of age.<br><br>Conclusions
The results suggest the existence of a long latency period after perinatal exposure to low-level MeHg, in which the behavioral effects emerged long after the leveling-off of brain Hg levels. Hence, the initial toxicologic event responsible for the late effects should have occurred before this leveling-off of brain Hg.<br><br><br><br>Methylmercury (MeHg) poses serious and practical concerns for <span class="yellow">human</span> populations regarding perinatal exposure. Fish, especially large predator (carnivore) fish species, accumulate high concentrations of MeHg through the marine food chain, and exposure of pregnant <span class="yellow">women</span> to MeHg through the consumption of fish has evoked widespread concern due to potential effects on offspring. Two large-scale cohort studies in fish-eating populations of Seychelles and Faroe islanders are being conducted; although the former has not found consistent adverse developmental effects of MeHg (Myers et al. 2007), the latter has reported adverse effects (Debes et al. 2006). Fundamental reasons for this discrepancy have not been completely elucidated, and many questions remain regarding the neurotoxicity of MeHg, despite extensive study.
Among the unanswered questions is whether there is a long latency period for behavioral manifestations after exposure to MeHg (Clarkson and Magos 2006; Landrigan et al. 2005; Rice 1996; Weiss et al. 2005a, 2005b). Typical examples of latent toxicity in <span class="yellow">humans</span>, including both acute and chronic MeHg exposures, have been described in detail elsewhere (Weiss et al. 2005a). Davidson et al. (2006) recently suggested that effects of perinatal exposure to MeHg may emerge 9 years after birth in the Seychelles cohort. Consequently, risk assessments of MeHg exposure could be inaccurate because studies (<span class="yellow">human</span> or animal) usually do not focus on later stages of life and therefore could miss delayed effects. The possibility of delayed toxicity is exemplified by the expanded Barker hypothesis, which posits that the origin of some neurodegenerative diseases such as Parkinson and Alzheimer diseases lies in early exposure to environmental chemicals (Landrigan et al. 2005). Although epidemiologic evidence would be ideal for exploring the possibility of delayed toxicity (and, indeed, data from epidemiologic studies form the basis of current risk assessment for developmental toxicity of MeHg), considering the complex effects of numerous potential confounders and the existence of multiple exposures in <span class="yellow">human</span> populations, animal models would likely make important contributions to this field.
Although numerous animal studies have described the developmental neurotoxicity of MeHg (Watanabe and Satoh, 1996), few have addressed the latency issue. Few studies have evaluated the neurobehavioral effects in rodents longitudinally beyond 6 months after perinatal exposure. Spyker (1975) addressed this issue in her pioneering work, reporting the late development of behavioral toxicity in <span class="yellow">mice</span> prenatally exposed to MeHg; it appeared, however, that the substantial mortality and retarded growth among the exposed <span class="yellow">mice</span> were apparent before weaning, indicating that the doses used (even though some lower dose levels were included) exerted severe toxicity. Using a relatively complex schedule-controlled operant behavior method, <span class="yellow">rats</span> whose parents were exposed to MeHg (0.5 or 6.4 mg/L) from 4 weeks before mating and continuing to postnatal day (PND) 16 were shown to be less sensitive to a change in the reinforcement schedule than were their nonexposed counterparts at 28–32 months of age (Newland et al. 2004). <span class="yellow">Mice</span> that were perinatally exposed to 1 or 3 mg/L MeHg in drinking water did not show significant deviation from controls in behavioral performance (motor performance, memory, and learning) at 5, 15, or 26 months of age, but the lifetime-exposed groups did show a significant deviation (Weiss et al. 2005c). The existence of a latency period (i.e., the absence of effects earlier in life followed by the emergence of effects at a later stage of life) has not been demonstrated in any <span class="yellow">rodent</span> study. In nonhuman primates, delayed emergence of the signs of neurotoxicity was observed several years after the cessation of a 7-year postnatal exposure (Rice 1996).
Metallothionein (MT) protects against the toxicities of a variety of metals. We examined the neurotoxicity and developmental toxicity of metallic Hg in MT I/II-knockout <span class="yellow">mice</span> (Yoshida et al. 2004) and demonstrated the susceptibility of this genetically manipulated strain to the toxicity of metallic Hg. In contrast to metallic Hg, MeHg does not induce MT, and MT would not substantially influence the kinetics of MeHg (Yasutake et al. 1998). Several reports, however, have demonstrated protective effects of MT against MeHg toxicity, which was ascribed to the radical scavenging effect of MT (Yao et al. 2000). We also showed that perinatal exposure to MeHg results in altered metabolism of thyroid hormones in neonates that was more distinct in MT-null strains than their wild-type counterparts (Mori et al. 2006). The vulnerability of the MT-null strain suggests that delayed neurobehavioral toxicity due to MeHg, if it does exist, might be more distinctive in this strain.
By utilizing the MT-null strain, we aimed to answer the following two questions: First, could we generate a model in which the toxicity of MeHg would emerge or at least become exaggerated later in life as opposed to earlier in life (i.e., at 3–6 months, which was the timing for most of the earlier studies that used behavioral evaluations)? Second, would the MT-null strain be affected more than its parent C57BL strain? Answering either of these questions not only could influence the risk evaluation of MeHg, but it could also lead to a better understanding of the mechanism of toxicity for perinatal MeHg exposure. To address these issues, we used three behavioral paradigms, the open field (OPF), passive avoidance (PA), and Morris (water) maze (MM) tests, which are often used in this field and which we used in our previous studies on the effects of Hg vapor (Yoshida et al. 2004, 2006). Performances in the MM and PA are said to be the most sensitive to aging (Gower and Lamberty 1993).
We used a dose of 5 μg MeHg/g in the diet, which resulted in a brain Hg level relevant to <span class="yellow">human</span> risk assessment. We evaluated the behavioral end points twice, once around 3 months of age and the other time around 1 year; the latter time roughly corresponds to the period when many behavioral performances, including OPF (Acevedoa et al. 2006; Carrie et al. 1999; Gower and Lamberty 1993), PA (Gower and Lamberty 1993), and MM (Bach et al. 1999; Carrie et al. 1999), show alterations in this <span class="yellow">mouse</span> strain.
Materials and Methods
Animals and MeHg exposure
OLA129/C57BL/6J strain <span class="yellow">mice</span> (wild type) and MT I/II-knockout <span class="yellow">mice</span> (MT-null) of this strain were provided by K.H. Choo of the Murdoch Institute, Parkville, Australia (Michalska and Choo 1993) and were of a mixed genetic background of 129/Ola and C57BL/6 strains. F1 hybrid <span class="yellow">mice</span> were mated with C57BL/6 <span class="yellow">mice</span> for six generations at the National Institute for Environmental Studies (Tsukuba, Japan). At 10 weeks of age, single male and female <span class="yellow">mice</span> were allowed to cohabit; every female <span class="yellow">mouse</span> was checked each morning for the presence of a vaginal plug. When a plug was confirmed, the day was designated as day 0 of gestation (GD0).
The diet, NIH-07PLD formula (CLEA Japan, Inc., Tokyo, Japan), contained vitamins and trace elements as follows (per kilogram diet): 3.2 mg CuSO4, 88 mg FeSO4, 149 mg MnSO4, 25 mg ZnCO3, 1.6 mg Ca(IO3)2, 11 mg vitamin B1, 4.7 mg vitamin B2, 1.9 mg vitamin B6, 44 mg vitamin E, in addition to 5 μg MeHg/g. This diet was fed to the pregnant <span class="yellow">mice</span> starting from GD0 through 10 days after delivery (i.e., PND10). Thereafter, we switched <span class="yellow">mice</span> to a diet that did not contain MeHg. We chose GD0 as the beginning of exposure because exposures that started before conception often resulted in fairly high Hg concentrations in fetal/neonatal brains (Kakita et al. 2003), and we chose PND10 to cover the early neonatal period, in which considerable brain growth occurs. In our experimental setting, the neonatal <span class="yellow">mice</span> started to eat from the diet bucket and drink from the water bottle from PND10 onward. Control <span class="yellow">mice</span> were kept on the same diet but without MeHg (< 0.01 μg Hg/g). On PND1, to avoid the confounding effects due to different litter size, we reduced each litter to six pups (three males and three females when possible), and on PND10, two males and two females from each litter were killed for chemical analyses.
The remaining male and female offspring per litter were weaned on PND28 and used for subsequent behavioral analyses (either at 12–13 weeks or 52–53 weeks, depending on the litter) as described below. We measured body weights of the weaned <span class="yellow">mice</span> every 2 weeks. Thus, four experimental groups were used (with or without MeHg exposure for two strains), and each group consisted of 12–13 litters. For half of the litters, the behavioral analyses were conducted at 12–13 weeks of age, and upon completion of the behavioral analyses, the animals were killed under ether anesthesia. We then dissected the organs (brain, liver, and kidneys) for Hg analyses. For the remaining half of the litters, we conducted the behavioral tests at 52–53 weeks of age. The <span class="yellow">mice</span> were treated humanely and with regard to alleviation of suffering according to the National Institute for Environmental Studies’ Guidelines for Animal Welfare and the guidelines of St. Marianna University.<br><br>Behavioral evaluations
The details of each behavioral procedure have been described elsewhere (Yoshida et al. 2006). Brief descriptions follow.
For the OPF test, we used an OPF apparatus (Ohara Co., Ltd., Tokyo, Japan) with a 60 × 60-cm floor surrounded by walls 60 cm high. The experimental room light was turned off, and a dim light of 80 lux was lit during the experiment. We placed a <span class="yellow">mouse</span> in the center of the floor and monitored its behavior for 10 min using a CCD camera connected to a computer. The position of the center of gravity was calculated by image-analyzing software, which was used to calculate the total distance traveled by the <span class="yellow">mouse</span> as well as the positional preference (either center or peripheral, where peripheral was defined as the area within 10 cm from the wall). We cleaned the OPF apparatus with 70% ethanol between trials.
The apparatus for the PA test (Ohara Co. Ltd.) consisted of a light compartment that illuminated by a 400-lux light and a dark compartment with black opaque walls and lids. The two compartments were separated by a mobile guillotine door. On the first day (training trial), we placed each <span class="yellow">mouse</span> in the light compartment facing away from the guillotine door, which was closed. After 30 sec of introduction, the door was opened; when the <span class="yellow">mouse</span> entered the dark compartment, a brief electric shock (4 mA for 2 sec) was delivered through the metal grid floor. This would force the <span class="yellow">mouse</span> back to the light component. The interval between the opening of the door and the entry to the dark room (in seconds) was recorded and defined as the latency. On the next day, the same procedure was repeated, but without the electric shock (retaining trail). In this PA paradigm, aversive learning was assumed to be established in the training trial, and we used its retention in the retaining trail as the index of learning. Between each individual trial, we cleaned the apparatus with ethanol. The cutoff time of the retention session was 300 sec.
The MM test apparatus (Ohara Co., Ltd.) was a round-shaped water pool with a diameter of 120 cm. A small platform was submerged in the water, which provided a place for <span class="yellow">mice</span> to escape from the water (i.e., an aversive stimulus, water temperature = 23 ± 1°C). The water was made opaque by adding white paint so that the <span class="yellow">mouse</span> could not see the submerged platform. In each trial, we released a <span class="yellow">mouse</span> into the pool from a determined position along the wall, and the performance of the <span class="yellow">mouse</span> was monitored by a CCD camera/image analyzer. The time required to reach the platform was recorded. If a <span class="yellow">mouse</span> could not find the platform within 60 sec after release, it was led to the platform and placed on it for 20 sec before being removed. In these cases, a latency of 60 sec was recorded. We conducted the trial once a day up to the fifth day for each <span class="yellow">mouse</span>, and the order of each <span class="yellow">mouse</span> was counterbalanced across the day. On the sixth day, a transfer test (or probe test), which is a trial without the platform, was conducted; in this procedure, we counted the number of times that the <span class="yellow">mouse</span> crossed the position where the platform had been.<br><br>Tissue Hg concentration
The tissue samples were homogenized (10% weight/volume) in distilled water using a Polytron homogenizer (Kinematica GmbH, Littau, Switzerland). We determined Hg levels in the homogenates by the oxygen combustion–gold amalgamation method (Ohkawa et al. 1977) using an atomic absorption Hg detector MD-1 (Nippon Instruments, Co. Ltd., Osaka, Japan). To ensure the accuracy of the measurement, we included reference material from a dogfish (DORM-2; National Research Council of Canada, Ottawa, Ontario, Canada) with a certified value of 4.64 ± 0.26 μg/g in the analyses; the observed values fell within the certified range. The detection limit of the measurement was 0.1 ng Hg.<br><br>Statistics
We analyzed behavioral data for OPF and PA by analysis of variance (ANOVA), taking sex, strain, and MeHg exposure as the factors. All the interactions among these factors were put into the model. When any of the interactions was highly significant, we analyzed the data separately in an appropriate way; for example, if sex × Hg was significant, the data for males and females were separately analyzed for the effects of strain and Hg. Whenever appropriate, ANOVAs were followed by Mann-Whitney U or Student’s t-tests, depending on the nature and distribution of the variables. We analyzed data for the MM by repeated-measures ANOVA, taking the exposure as between-group and trials as within-group variables. The test was performed for each of the four sex and strain combinations separately. The significance level was set at p < 0.05.<br><br>
Results
Body weight
Up to 20 weeks of age, the body weight values of the control and MeHg-exposed groups were not different, regardless of strain or sex. After 28 weeks, except for the wild-type female groups, the MeHg-exposed groups weighed significantly less than the controls (Figure 1).<br><br>OPF
At 12 weeks of age, a three-way ANOVA of the locomotion distance revealed that only strain was a significant factor (p < 0.001), reflecting the longer distance traveled by the MT-null <span class="yellow">mice</span> (Figure 2A). Strain was also a significant factor for the proportion of the central-area locomotion (Figure 3A), and Hg exposure marginally affected this outcome. In MT-null females, the proportion of central-area locomotion was higher in MeHg-exposed <span class="yellow">mice</span> than in the controls; this difference was not observed in any other strain–sex combination. At 52 weeks of age, the strain × Hg interaction was highly significant (p < 0.001) in an ANOVA of locomotion distance (Figure 2B); MeHg exposure was associated with decreased locomotion distance in wild-type <span class="yellow">mice</span> and with increased distance in MT-null <span class="yellow">mice</span>. A strain-wise two-way ANOVA (with sex and Hg as the factors) revealed that only Hg was significant in both strains (p < 0.01). Regarding the proportion of the central-area locomotion, the effects of MeHg appeared to depend on sex [i.e., sex × Hg was highly significant (p < 0.001) in the three-way ANOVA; Figure 3B]. Indeed, a sex-wise two-way ANOVA showed significant effects of Hg only in females (p < 0.001).<br><br>PA
At 12 weeks of age, all the groups showed prolonged latency in the second (retention) trial, and no consistent effect of MeHg was recognized regardless of strain or sex (Figure 4A).
At 52 weeks of age, a three-way ANOVA revealed a significant interaction between strain and Hg (p < 0.05); strain-wise two-way ANOVAs revealed a significant effect of Hg on learning in MT-null <span class="yellow">mice</span> of both sexes; these groups of <span class="yellow">mice</span> showed significantly shorter (less than half) latency times compared to control <span class="yellow">mice</span> (Figure 4B). A notable difference between the results at 52 weeks of age and those at 12 weeks was that many of the tested <span class="yellow">mice</span> exceeded the cutoff time in the retention trials at 52 weeks, except for the MT-null groups.<br><br>MM
At 13 weeks of age, repeated-measures ANOVA did not indicate any effects of MeHg (Figure 5A). At 52 weeks of age (Figure 5B), wild-type males and MT-null females shared the same statistical results; Hg as well as the Hg × trial interaction were statistically significant. Thus, in both cases, the MeHg groups showed a longer latency, hampering learning performance.<br><br>Tissue Hg concentration
At PND10, which was immediately after the exposure, brain Hg concentrations of the neonatal <span class="yellow">mice</span> were approximately ≤ 0.5 μg/g (Table 1). Although the MT-null <span class="yellow">mice</span> and females showed slightly higher brain Hg concentrations than the corresponding wild-type group and males, respectively, neither of these differences was significant. At 13 weeks, when the behavioral tests were completed, the brain Hg concentration was comparable to the control (nonexposed) level (approximately 5 ng/g in both the exposed and control brains). Interestingly, MT-null <span class="yellow">mice</span> had a significantly lower brain Hg concentration than the corresponding wild-type groups.<br><br>
Discussion
Results of the present study demonstrate the delayed emergence of neurobehavioral toxicity due to perinatal MeHg exposure, which presumably developed after brain MeHg concentrations had leveled off. This emergent toxicity was exaggerated in MT-null <span class="yellow">mice</span> and was more distinct in females. To our knowledge, our findings show the first clear-cut demonstrations of a long latency period of MeHg neurobehavioral toxicity in rodents and possible genetic susceptibility for the emergent toxicity.
The exposure level should be considered before discussing the end points. On PND10, immediately after the cessation of MeHg exposure, the brain Hg concentration was approximately 0.5 μg/g, regardless of the strain or sex. In a previous study, the brain Hg concentration in <span class="yellow">mice</span> perinatally exposed to 6 mg MeHg/L (via water) peaked between PND0 and PND4 and was approximately three times higher than on PND21 (Goulet et al. 2003). Therefore, the peak brain Hg concentration, which is presumably observed around birth, can be estimated as about 3-fold higher than that on PND10 and would be approximately 1.5 μg/g (0.5 μg/g × 3), which is one of the lowest levels among recent <span class="yellow">rodent</span> studies. As shown by Sakamoto et al. (2002) in their Figure 2, prenatal exposure of <span class="yellow">rats</span> to MeHg showed a peak brain Hg concentration around PND1 that was four to five times greater than the level on PND10. Also, Newland and Rasmussen (2000) reported a slight alteration of a complex operant behavior in <span class="yellow">rats</span> at ages < 2 years at brain Hg concentrations as low as 0.5 μg/g at birth, although statistical significance of this particular effect was not clear. It should be noted that <span class="yellow">rats</span> have different Hg kinetics (Hirayama and Yasutake 1986; Yasutake and Hirayama 1990) due to the high affinity of <span class="red">rat</span> hemoglobin for MeHg (Clarkson and Magos 2006).
The most important observation of the present study was that the effects of low-level MeHg exposure were detected only at later stages in the lives of the <span class="yellow">mice</span>. Except for the central-area occupancy in OPF in MT-null females, no statistically significant effects of MeHg were observed in any of the three behavioral tests at 12 weeks of age. In contrast, significant effects were observed in all three tests at 52 weeks of age. The brain Hg concentration of the exposed groups had leveled off and was not distinguishable from the non-exposed group at 13 weeks of age, immediately after the first phase of the behavioral testing. Therefore, in the present study, there was a latency period in which the dose and effects could not be detected, although effects were observed 9 months later. Another notable observation was that the emergent manifestation of toxicity was also recognized in the suppression of body weight (except for wild-type females), which only became apparent on or after 28 weeks of age.
The existence of a latency period of as long as several years after chronic (7 years from birth), low-level exposure to MeHg has been described in nonhuman primates (Rice 1996). In that case, however, the Hg concentration in the brain remained elevated, presumably as a result of the long exposure. Indeed, Rice (1996) argued that the minute amount of residual brain Hg could have caused the delayed toxicity. This was clearly not the case in the present study because the Hg concentration leveled off around the time of the first phase of the behavioral study. The absence of behavioral effects at 12 weeks of age ruled out the possibility that the residual behavioral effects were due to elevated Hg early in life. Therefore, the behavioral toxicity that surfaced at 52 weeks of age must have had its origin before the brain Hg concentration leveled off (at or before 13 weeks of age), although the redistribution of Hg to the brain from other sites of deposition, such as the liver, cannot be completely excluded. The long silent period before the manifestations of toxicity emerged suggests that a slow process plays a role in this latent toxicity.
Although an example of a slow process is aging, 52 weeks of age might not be sufficiently old for a <span class="yellow">mouse</span> to be considered aged in the physiologic sense because C57BL/6 <span class="yellow">mice</span> have a relatively long life span among <span class="yellow">mouse</span> strains [median survival of 27–31 months (Gower and Lamberty 1993)] and a survival rate at 18–19 months as high as 90% (Institute on Aging HP) (National Institute on Aging 2008). Nevertheless, various behavioral examinations have shown age-related changes in the performance of <span class="yellow">mice</span> at approximately 1 year of age in the OPF (Acevedoa et al. 2006; Carrie et al. 1999), MM (Bach et al. 1999; Carrie et al. 1999), and PA (Gower and Lamberty 1993) tests. The observed effects of MeHg, including the deterioration in the PA and MM and suppression in the OPF (in wild-type <span class="yellow">mice</span>), were consistent with these reported effects of aging on behavioral function (in the sense described above), except for the increased OPF activity in the MT-null <span class="yellow">mice</span>.
Regardless of its neural basis, the basis of neurobehavioral toxicity should be sought in early life stages when the brain Hg concentration is highly elevated (approximately 1.5 μg/g at its peak). Some in vivo experiments have demonstrated several candidate mechanisms of perinatal exposure to MeHg, including abnormal migration of neurons and/or glias (Kakita et al. 2003; Rodier et al. 1984), but at higher Hg concentrations. Using exactly the same exposure protocol as the present study, we found suppressed activity of type III iodo-thyronine deiodinase, a thyroid hormone-metabolizing enzyme, in the brains of PND10 <span class="yellow">mouse</span> neonates (Mori et al. 2006), consistent with our previous study of higher MeHg doses (Watanabe et al. 1999). This perturbation could be one of the candidate mechanisms responsible for the later anomalous behaviors because even a transient change in thyroid hormones during the critical period of perinatal life exerts long-term consequences (Auso et al. 2004).
The effects of MeHg at 52 weeks of age were influenced by two potential modifying factors, sex and strain. In the OPF, while the locomotion was affected in both strains (although the direction was opposite), center occupancy was significantly increased only in the MT-null <span class="yellow">mice</span>. In addition, the effects on PA were significant only in MT-null <span class="yellow">mice</span>, whereas MeHg at a higher dose was reported to worsen PA performance in <span class="yellow">rats</span> (6–8 weeks of age; Sakamoto et al. 2002). In addition, body weight gain was suppressed in both male and female MT-null <span class="yellow">mice</span>, whereas in wild-type <span class="yellow">mice</span> the suppression was observed only in males. Taken together, the MT-null strain appeared to be slightly more susceptible to the late-emergent effects of MeHg. Several lines of evidence have shown that MT-I,II is protective against the toxicity of MeHg (Leiva-Presa et al. 2004; Yao et al. 2000), and the present results were basically consistent with these reports. We have also reported the susceptibility of the MT-null strain to the neurotoxic effects of metallic Hg (Yoshida et al. 2004, 2006).
The difference in the susceptibility to MeHg between sexes is still debated (Clarkson and Magos 2006; National Research Council 2000; Vahter 2007). In the present study, some responses to MeHg were different between the sexes, including OPF performance at 12 and 52 weeks of age and MM performance at 52 weeks. The fact that the MT-null female group was the only group affected by MeHg at 12 weeks may suggest the particular susceptibility of females in this strain. This point needs to be clarified in further experiments.
The question remains of why MT-null <span class="yellow">mice</span> are susceptible to the delayed neurotoxicity of perinatal MeHg. Apparently kinetics play only a minor role because the strain did not show distinct effects on the brain Hg concentration at PND10. The significantly lower brain Hg concentration in MT-null <span class="yellow">mice</span> compared with corresponding wild-type <span class="yellow">mice</span> at 13 weeks of age indicated that MT-I,II might play a significant role in the retention of Hg (or MeHg). This is consistent with the results of studies of metallic Hg exposure (Yoshida et al. 2004, 2005); a lower brain Hg concentration may not guarantee lower toxicity, supporting the protective role of the protein. Earlier studies suggest that brain MT-I,II has an important role both in the response to oxidative injury (Potter et al. 2007) and in the process of aging (Kojima et al. 1999). Therefore, the lack of MT can exaggerate the toxicity of MeHg by enhancing the initial effects due to oxygen radicals and/or by accelerating functional aging. Apart from this, an intriguing possibility is that the brain-specific isoform, MT-III, might contribute to the results we obtained because the expression of MT-III, together with MT-I, is increased in the brain of old <span class="yellow">rats</span>, resulting in the low availability of free zinc for synapses (Mocchegiani et al. 2004). The age-dependent expressions of MT isoforms might be modified in MT-null <span class="yellow">mice</span>.
Results of the present study might allow the possibility of alternative interpretations due to some potentially confounding factors. For example, except for the wild-type females, we observed significant differences in body weight between MeHg-exposed and non-exposed groups. Because these differences only became clear later in life, they might be associated with the toxicity that also emerged later in life. Manipulation of body weight in rodents alters activity levels, although the reported results are not always consistent with each other (Harrison and Archer 1987; Samuelsson et al. 2008). Also, the differential performance in PA could be related with the potential effects of MeHg on (electric) shock sensitivity, which we did not examine. At least one high-dose study with adult <span class="yellow">rats</span> showed reduced electric sensitivity due to mercury exposure (Wu et al. 1985). These possibilities need to be addressed in future studies.
To summarize, the present results suggest that an initial (or triggering) toxicologic event occurs before the brain Hg concentration stabilizes and that the nature of this event should be either an acceleration of the aging process or interaction with the aging process. Thus, by identifying the physiologic events associated with the functional aging of the examined behavioral tasks, the fundamental toxicologic scar might be revealed.<br><br>
<h3>pmcA2652712</h3>Neurogenin2 Directs Granule Neuroblast Production and Amplification while NeuroD1 Specifies Neuronal Fate during Hippocampal Neurogenesis
Abstract
The specification and differentiation of dentate gyrus granule neurons in the hippocampus require temporally and spatially coordinated actions of both intrinsic and extrinsic molecules. The basic helix-loop-helix transcription factor Neurogenin2 (Ngn2) and NeuroD1 are key regulators in these processes. Based on existing classification, we analyzed the molecular events occurring during hippocampal neurogenesis, primarily focusing on juvenile animals. We found that Ngn2 is transiently expressed by late type-2a amplifying progenitors. The Ngn2 progenies mature into hippocampal granule neurons. Interestingly, the loss of Ngn2 at early stages of development leads to a robust reduction in neurogenesis, but does not disturb granule neuron maturation per se. We found that the role of Ngn2 is to maintain progenitors in an undifferentiated state, allowing them to amplify prior to their maturation into granule neurons upon NeuroD1 induction. When we overexpressed Ngn2 and NeuroD1 in vivo, we found NeuroD1 to exhibit a more pronounced neuron-inductive effect, leading to granule neuron commitment, than that displayed by Ngn2. Finally, we observed that all markers expressed during the transcriptional control of hippocampal neurogenesis in rodents are also present in the <span class="yellow">human</span> hippocampus. Taken together, we demonstrate a critical role of for Ngn2 and NeuroD1 in controlling neuronal commitment and hippocampal granule neuroblast formation, both during embryonic development and in post-natal hippocampal granule neurogenesis.<br><br>Introduction
Neurons are born not only during development of the central nervous system, but neurogenesis also continues into adulthood. In both <span class="yellow">rodent</span> and <span class="yellow">human</span> adult brain, neurogenesis is active in two distinct zones of the forebrain: the subventricular zone (SVZ) of the lateral ventricle and the subgranular zone (SGZ) of the hippocampal dentate gyrus (DG) [1]–[5]. The molecular mechanism underlying neurogenesis in the DG is not fully understood. Clearly, a cascade of transcriptional events controls the specification of neuronal identity in the DG [3], [6]–[8], but details of the expression pattern and function of each transcription factor remain elusive.
The paired-box homeodomain transcription factor Pax6 and the bHLH transcription factors Ngn2 and NeuroD1 are important when cells acquire a pan-neuronal character and a specific neuronal subtype [9], [10]. In the developing neocortex, Pax6 is expressed in dividing radial glial cells at the ventricular surface [9]. In the adult hippocampus, Pax6 is expressed in astrocytes in the SGZ and are considered to be the true stem cells [9], [11]. Loss- and gain-of-function studies indicate that Pax6 is involved in regulating the proliferation of neocortical and hippocampal progenitors [9], [12]–[16]. During neocorticogenesis, high concentrations of Pax6 induce the expression of the bHLH transcription factor Neurogenin2 (Ngn2) [17]. In turn, Ngn2 causes cell cycle exit [18], an event that takes place when NeuroD1 starts to be expressed [19]. In the developing hippocampus, the absence of Ngn2 leads to a reduced number of granule neurons [20]. NeuroD1, on the other hand, is essential for the differentiation and survival of hippocampal granule neurons [10]. Interestingly, in the absence of Ngn2, NeuroD1 can still be activated and neuronal differentiation can still take place [20]. This has led to the idea that the primary role of Ngn2 is not to direct neuronal differentiation.
Only a small number of studies have addressed the role of Ngn2 and NeuroD1 in hippocampal neurogenesis [10], [20], [21]. Therefore, we now re-examine their roles in hippocampal neurogenesis in detail, using gain- and loss-of-function experiments. We first establish a hierarchy of transcriptional events that occur during neurogenesis in the DG and then define the place that Ngn2, NeuroD1 and other transcription factors have in this cascade. We find that a lack of Ngn2 expression result in a markedly smaller hippocampus and an almost complete absence of the DG. We show that Ngn2 is required for granule neuroblasts production/amplification. Gain-of-function of NeuroD1 during development of the DG results in an efficient generation of granule neuroblasts, an effect that we do not observe when we overexpress Ngn2. Finally, we demonstrate that the same transcription factors and cellular markers seen in <span class="yellow">mouse</span> and <span class="blue">rat</span> tissue are also present in the <span class="yellow">human</span> hippocampus.<br><br>Results
Expression of transcription factors and cellular markers define different phases of postnatal hippocampal granule neuron maturation
We first analyzed in detail the chronology of expression of different transcription factors and cell-specific markers during DG granule neuron formation and correlated our findings to the previously established classification of adult hippocampal neurogenesis [3]. The current classification describes hippocampal stem cells and progenies into three categories (type-1, -2 and -3 cells) depending on the markers they express as they mature. In the adult <span class="yellow">mouse</span> DG, only a very small proportion of cells undergo mitosis at any one given time [22]. Consequently, in adult animals it is difficult to analyze which transcription factors are involved in the transition phase from one cell category to the next. Therefore, we studied 2 week-old rodents which have higher numbers of maturing granule neurons in the DG. We performed triple immunohistochemistry for all markers of interest and compared results from 2 week- and 2 month-old <span class="yellow">mice</span> (Figure 1 and figure S1).
We first confirmed that stem cells in the SGZ exhibit characteristics of radial glia [3], [11], [16]. They extend radial processes from the SGZ to the apex of the subgranular layer (SGL) and the basal molecular layer. Moreover, they express the intermediate filament protein nestin, glial fibrillary acidic protein (GFAP) and glial glutamate transporter (GLAST) [16] (Figure 1A and B and figure S1A and S1B). The GFAP- and GLAST+ cells are defined as type-1 cells [3]. They also express the paired-homeodomain transcription factor Pax6 (Figure 1B). These radial glia-like stem cells divide relatively infrequently, and are believed to undergo symmetric division (giving rise to two identical stem cells) under some conditions. They can also divide asymmetrically and give rise to a new stem cell and one neuronal progenitor, which usually is defined as a type-2 cell (Figure 2B) [3].
Type-2 cells are divided into two different populations: type-2a expressing nestin and type-2b co-expressing nestin and doublecortin (Dcx) [23]. Based on a previous report [24], we examined the expression pattern of the T-domain transcription factors Tbr1 and Tbr2. We found that Tbr2 is expressed in type-2a cells, in agreement with a recent report [25]. In the SGZ of 2 week-old <span class="yellow">mice</span>, we found an average of 50–70 Tbr2+ cells per 30 µm-thick section throughout the rostro-caudal axis of the dorsal hippocampus. This number was reduced by 35% in 2 month-old <span class="yellow">mice</span> (Figure 1C with figure S1C and data not shown). As opposed to Tbr1 (Figure 1D and figure S1D), Tbr2+ cells did not co-express Dcx or PSA-NCAM (Figure 1C and figure S1C). We observed that Tbr1-immuoreactivity gradually decreased as neurons matured and started to express NeuN (Figure 1H and figure S1H). Thus, Tbr2 labels type-2 cells, while Tbr1 is expressed by immature granule neurons.
We then examined the molecular phenotype of type-3 cells, a cell type that transiently expresses Calretinin [26]. We hypothesized that hippocampal type-3 cells might also express the bHLH transcription factor NeuroD1, known to partially overlap with Tbr2 and Tbr1 in different brain regions [24]. We found that NeuroD1 expression in the hippocampus starts in Tbr2+ cells and extends to post-mitotic Tbr1+ cells, which also express the hippocampal granule identity transcription factor Prox1 (Figure 1G and 1I and figure S1G and S1I). While Tbr1 expression ceases during granule maturation, that of NeuroD1 is weakly maintained when NeuN expression starts (Figure 1G and 1H and figure S1G and S1H). We found that calretinin expression decreased during granule neuron maturation, before NeuroD1 was reduced (Figure 1F). Thus NeuroD1 is expressed in type-2b and type-3 cells, as well as immature granule neurons. Finally, we found that NeuroD2 starts to be expressed just after NeuroD1 and, unlike NeuroD1, continues to be highly expressed in mature neurons (Figure 1J and data not shown).<br><br>Mash1 and Ngn2 define the early versus late type-2a stage in 2 weeks old <span class="yellow">mice</span>
Having established the hierarchy of transcription factors and cellular markers appearing during hippocampal neurogenesis, we set out to clarify the position of the two bHLH transcription factors Ngn2 and Mash1, in this process. We found that more than 90% of Ngn2-immunoreactive cells are Pax6 positive (Figure 2A), and almost all co-express Tbr2 (Figure 2B). Thus, Ngn2 appears to be expressed by type-2a cells. As our antibodies against Ngn2 and NeuroD1 were made in the same species, we could not determine if there was an overlap between these two proteins. However, by comparing their overlapping expression pattern with that of Tbr2, we propose that some Ngn2+ cells co-express NeuroD1, and that they appear at the onset of NeuroD1 expression and downregulation of Ngn2. We base this assumption on the facts that more than 50% of the Tbr2+ cells are Ngn2+ and more than 50% of Tbr2+ cells also express NeuroD1. The expression of Ngn2 in adult hippocampus was hard to detect using immunohistochemistry due to the low expression of the protein (data not shown). However, it is present, as previously described using reporter <span class="yellow">mice</span> and as revealed by in situ hybridization [21].
We next examined the expression of Mash1 and compared it with that of Ngn2, Pax6 and Tbr2. We found some Mash1+ cells (10–20 cells per section) in the dorsal hippocampus. Almost all of them co-expressed Pax6 (Figure 2C). Only few cells co-expressed Tbr2 (Figure 2D) or Ngn2 (Figure 2E), suggesting that Mash1 is downregulated when these two proteins are expressed. None of the Mash1-expressing cells were positive for NeuroD1 (Figure 2F). Mash1+ cells were still undergoing mitosis (Figure 2C) and consequently over 90% of them co-labeled with the cell-cycle marker Ki67 (Figure S1J).
Taken together, we show that Mash1 and Ngn2 are expressed in early and late stages of maturation of type2a progenitors, respectively, and that both transcription factors are co-expressed briefly when cells transit from early to late phase type2a cells.<br><br>Ngn2 progenies become hippocampal granule neurons
Previous work has shown that in Mash1 null mutant <span class="yellow">mice</span> hippocampal neurogenesis is not reduced and the hippocampus is not malformed [20]. Therefore we focused our initial analysis on the role of Ngn2 in hippocampal neurogenesis. We characterized the fate of Ngn2-expressing cells using Ngn2 Knock-in green fluorescent protein (GFP) <span class="yellow">mice</span> (Ngn2+/GFP) [27]. In the SGZ of 2 week-old hippocampi, we found bright GFP-expressing cells that were positive for Tbr2 (Figure 2H), the immature neuronal marker PSA-NCAM (Figure 2I) and NeuroD1 (Figure 2J). Furthermore, Prox1+, NeuroD2+ and calretinin+ cells were also weakly positive for GFP. We made similar observations in 2 month-old <span class="yellow">mice</span> (data not shown) confirming earlier published results [21]). We did not observe Mash1+/GFP+ and NeuN+/GFP+ cells (Figure 2G). These data from Ngn2+/GFP <span class="yellow">mice</span> are entirely consistent with our earlier observations using immunocytochemistry to label Ngn2-expressing cells and with GFP undergoing slow degradation after the expression of Ngn2 has ceased. Importantly, none of the GFP-expressing cells, including those only weakly fluorescent, residing in the SGL expressed the astrocytic marker GFAP or the oligodendrocytic markers CNPase (data not shown). Taken together, the data show that Ngn2 progenies become neurons and never generate astrocytes or oligodendrocytes.<br><br>Marked reduction of hippocampal granule neurons in absence of Ngn2
To determine the role of Ngn2 during the initiation of hippocampal granule neurogenesis, we analyzed the hippocampus of <span class="yellow">mice</span> lacking Ngn2 (Ngn2GFP/GFP). In contrast to the <span class="yellow">mice</span> use by Galichet and coworkers, our <span class="yellow">mice</span> have a lifespan of only 2 weeks after birth. Therefore, we analyzed 2 week-old and younger <span class="yellow">mice</span>. Consistent with previous observations [20], the hippocampus in Ngn2GFP/GFP mutants is clearly malformed (Figure 3A). In two week-old Ngn2GFP/GFP <span class="yellow">mice</span>, the ventral blade of the DG is completely absent. Already on postnatal day 1–2, the ventral blade of the DG is malformed along its whole rostro-caudal axis (Figure 3B).
As neither migration defects nor cell death cause the reduced number of hippocampal neurons in Ngn2 null <span class="yellow">mice</span> [20], we asked if Ngn2 is necessary for production of hippocampal granule neuroblasts. We first injected 2 day-old <span class="yellow">mouse</span> pups with BrdU 2 hours prior to sacrifice, allowing us to evaluate cell proliferation and ongoing neurogenesis. We also injected BrdU into female <span class="yellow">mice</span> on their final day of pregnancy and examined the brains of their pups 48 hours later (corresponding to P1–P2 old pups). This allowed us to assess the number of neurons derived from the last day of intrauterine development. Regardless of whether the <span class="yellow">mice</span> were WT or hetero/homozygous mutant pups, we found cells that had incorporated BrdU in the hippocampal subventricular zone (hSVZ), fimbria and the DG (Figure 4A and B; [28]), suggesting that progenitors originating in the hSVZ divide while migrating towards the DG. Most of them were organized in chains typical of migrating cells (Figure 4A and B and figure S2). In Ngn2GFP/GFP mutant <span class="yellow">mice</span> injected with BrdU 2 hours prior to sacrifice, we observed a decrease in the number of BrdU+ cells compared to heterozygous littermates. Thus, the numbers of newborn cells were reduced in the hSVZ/fimbria (133.3±7.5 for Ngn2+/GFP vs 69.7±5.0 for Ngn2GFP/GFP) and DG (132.0±9.1 for Ngn2+/GFP vs 42.6±2.2 for Ngn2GFP/GFP)(Figure 4C and D). We also examined the number of newborn cells differentiating into neurons, and identified them by the co-expression of BrdU and NeuroD1. As expected, they were relatively few in numbers because the short delay (2 hours) between BrdU administration and sacrifice in this first experimental paradigm. In Ngn2GFP/GFP null mutants, we found the number of cells differentiating into neurons to be reduced by 60% and 71% in the hSVZ/fimbria (8.6±1.4 for Ngn2+/GFP vs 3.4±0.5 for Ngn2GFP/GFP) and DG (9.7±1.1 for Ngn2+/GFP vs 2.8±0.6 for Ngn2GFP/GFP), respectively, (Figure 4E and F). As a result, the number of NeuroD1+ cells in the DG was reduced by 80% (580.3±27.7 for Ngn2+/GFP vs 116.2±7.3 for Ngn2GFP/GFP; Figure 4G).
We obtained similar results in the second paradigm, i.e. in <span class="yellow">mice</span> that we sacrificed 48 hours after BrdU administration and in which a larger number of the newborn cells had time to mature into neurons. In this case, the number of BrdU+/NeuroD1+ cells in the DG was reduced by 73% of Ngn2GFP/GFP mutant animals (91.5±24.1 for Ngn2+/GFP vs 26.4±3.3 for Ngn2GFP/GFP) (Figure 4J). We found no differences in the proportion of BrdU+ cells that expressed NeuroD1 in Ngn2+/GFP and Ngn2GFP/GFP <span class="yellow">mice</span> (14.2±1.0% and 17.4±1.8% respectively; Figure 4K). This shows that the few cells that manage to proliferate in the Ngn2 null <span class="yellow">mice</span> have the same ability to differentiate into neurons as those in <span class="yellow">mice</span> with one Ngn2 allele.
Interestingly, the absence of Ngn2 did not alter the identity of neurons in the DG granule layer in 2 week-old Ngn2 null <span class="yellow">mice</span>. The maturing granule progenitors sequentially expressed Pax6, Tbr2 (Figure S3A and S3B), NeuroD1 and Calretinin (Figure 4L), PSA-NCAM (Figure 4N), Tbr1 (data not shown) and Prox1 (Figure 4M). We confirmed that the same transcriptional cascade is active in the 20% (compared to <span class="yellow">mice</span> with one Ngn2 allele) residual granule neurons that are formed two day-old in Ngn2GFP/GFP <span class="yellow">mice</span> (Figure S3D–G and data not shown).
We next investigated if the cells that failed to develop into neurons in Ngn2GFP/GFP mutant <span class="yellow">mice</span>, became glial cells. We found that cells expressing GFP never co-labeled with the astrocyte marker GFAP (Figure 4N) or the oligodendrocyte marker CNPase (data not shown).
Altogether, our results show that Ngn2 plays an important role during the production/amplification of hippocampal granule neuroblasts, but not the acquisition of the granule neuron identity (Figure S7).<br><br>Ngn2 controls the amplification of granule neuron progenitors
We next monitored the mitogenic activity of Ngn2+ cells in the DG of 2 week-old WT <span class="yellow">mice</span>. We observed that over 50% of Ngn2+ cells in the DG of WT <span class="yellow">mice</span> co-expressed the mitosis marker Ki67 (Figure 5A). Inspired by this finding, we injected BrdU into 2 week-old WT and Ngn2GFP/GFP <span class="yellow">mice</span> and compared cell proliferation in the DG. We observed 92% reduction in number of proliferating cells in the SGZ of Ngn2 null mutant <span class="yellow">mice</span> (6.7±0.5) compared to WT (83.6±1.2) and heterozygotic (data not shown) littermates (Figure 5B–C). This data confirm the importance of Ngn2 during the amplification of granule progenitor.s.
To further explore whether Ngn2+ cells become post-mitotic or still proliferate after Ngn2 is downregulated, we performed immunohistochemistry on sections through the DG of Ngn2+/GFP <span class="yellow">mice</span>. We stained them with the mitosis marker Phospho-histone 3 (PH3) and NeuroD1, which is downstream of Ngn2 in the transcriptional cascade controlling neurogenesis. Thus, in the same sections we could identify whether cells that had initiated Ngn2 expression (GFP labeled), continued to divide (PH3+) or committed to neuronal differentiation (NeuroD1+). We observed some GFP+ cells that co-expressed PH3. They all exhibited morphological characteristics of one of the five mitotic phases: prophase, metaphase, anaphase, telophase and cytokinesis (Figure 5D and data not shown). Unexpectedly, cells that colabeled for GFP, PH3 and NeuroD1 (Figure 5E and figure S4A and S4B) were rare. This indicates that Ngn2+ cells undergo division/amplification and that they mature into post-mitotic neurons upon NeuroD1 expression. In <span class="yellow">mice</span> lacking Ngn2, we found the cells expressing GFP localized to the malformed ventral blade of the DG (Figure 5F and G). All of these cells were Ki67+ and they only very rarely expressed PH3 (Figure 5F and G, and figure S4C) in the absence of Ngn2 most of the cells are arrested in the cell cycle prior to entering the M phase and their mitosis is impaired. These data suggest that the mechanism of action of Ngn2 is conserved from development of the DG to postnatal hippocampal neurogenesis.
Our findings suggest that Ngn2 regulates amplification and cell cycle exit of DG granule progenitors. To examine this hypothesis, we compared the effects of Ngn2 and NeuroD1 on mitotic activity in embryonic cortico-hippocampal neurosphere-derived progenitors, 5 days upon transduction, in vitro. In cultures transduced with Ngn2 retrovirus, we found that 27.2% (±4.1%) of the PH3+ cells had been transduced (Figure I1–J). These cells were immunopositive for MAP2 (Figure 5I1 and I2) and therefore represented dividing neuroblasts. By contrast, in NeuroD1 transduced cultures, only 7.3% of PH3+ cells were GFP+ (Figure 5H1, H2 and J). All of the NeuroD1-transduced cells became MAP2+ (Figure 5H). To confirm this data we pulse-labeled transduced cultures with chlorodeoxyuridine (CldU) for 48 hours, five days after differentiation. In contrast to NeuroD1-transduced cultures, we observed dividing cells transduced with Ngn2 retrovirus that were positive for PH3 and that had incorporated CldU (Figure 5K–L2). As the number of cells PH3+/CldU+/GFP+ we observed was low, we did not quantify this finding.
Collectively, we have shown that Ngn2 is required for granule neuroblast production and amplification and that in the absence of Ngn2 the progenitors arrest in the cell cycle. NeuroD1, on the other hand, induces cell cycle exit and promotes rapid neuronal maturation.<br><br>NeuroD1 directs neuronal differentiation and maturation
Based on our previous observations we proposed that Ngn2 primarily controls amplification of granule neuroblasts and NeuroD1 directs neuronal differentiation. To test this hypothesis, we overexpressed Ngn2 or NeuroD1 in E14.5 cortico-hippocampal neurospheres and compared their effects after 5 days of differentiation of the progenitors. All cortico-hippocampal progenitors expressed Pax6 prior to differentiation (Figure S5). After 5 days, both factors suppressed Pax6 and Sox2 (Figure S5, figure 6C) and induced expression of Tbr1, Map2, NeuroD1 and PSA-NCAM (Figure 6D and E).
Interestingly, all Ngn2-overexpressing cells co-expressed NeuroD1. Based on these findings, we next explored whether Ngn2 overexpression induces NeuroD1 expression, and if NeuroD1 in turn directs neuronal differentiation. We compared the effects of both transcription factors in progenitors that either do or do not normally express them. Thus, we expressed Ngn2 in E14.5 neural progenitors isolated from three different brain regions: cortex/hippocampus, lateral ganglionic eminence (LGE) and ventral mesencephalon (VM). Ngn2 expression in the developing forebrain is normally limited to the neocortex, and it does not appear in LGE tissue [29]. In the developing VM, both Ngn2 and NeuroD1 are expressed, but not when neural progenitors from this region are cultured in vitro [30], [31]. In our experiments, cortico-hippocampal progenitors could differentiate into neurons, although they did not normally express NeuroD1 (Figure 6E). When we overexpressed Ngn2 in VM and LGE progenitors, the cells started to express both NeuroD1 and PSA-NCAM (Figure 6F and G). We then overexpressed NeuroD1 in the same types of cultured progenitors and found that the cells became immunoreactive for PSA-NCAM (Figure 6G and data not shown). Thus, NeuroD1 is sufficient to direct neuronal differentiation in cortico-hippocampal-, LGE- and VM-derived progenitors.
Our results from in vitro cultures and the analysis of the DG of Ngn2 mutant animals collectively show the neuron-inducing effect of NeuroD1 in hippocampal granule cell progenitors [32].<br><br>NeuroD1 directs exclusive neuronal differentiation of hippocampal granule neuron progenitors
As the next step, we tested the effects of NeuroD1 in vivo and compared them with those of Ngn2. We injected retroviruses carrying the gene for either Ngn2 [33] or NeuroD1, and the reporter gene eGFP into the ventricles E15.5 <span class="blue">rat</span> embryos, in utero (Figure 7A). Three weeks later, when the <span class="yellow">rats</span> were about two weeks old, we examined their hippocampi and analyzed the eGFP+ cells. The <span class="yellow">rats</span> injected with control vector exhibited eGFP+ cells within the hippocampus that were either star-shaped, progenitor/glial-like cells (14.8±3.4%) or neuron-like cells with long neurites (85.3±3.4%) (Figure 7B). In <span class="yellow">rats</span> that we had injected with the vector encoding Ngn2, the eGFP-labeled, the transduced hippocampal cells were composed of 31.9% (±5.7%) progenitor/glia-like cells and 69.4% (±6.1%) neuron-like cells (Figure 7B). The glia-like subpopulation was immunopositive for the astrocytic marker GFAP (Figure 7C–D). In contrast, virtually all of the transduced cells in <span class="yellow">rats</span> injected with the NeuroD1 vector became neuron-like cells (99.9%±0.1; Figure 7B). None of these cells stained for GFAP (Figure 7E and F). They were positioned in the external layers of both the ventral and dorsal blades of the DG (Figure 7B3, E and I and figure S6A).
When we examined neuronal maturation of Ngn2- and NeuroD1-transduced cells, we observed that only a few of those transduced with the NeuroD1 vector expressed the early mature neuronal marker Tbr1 (3.1±1.4%; Figure 7I and J). They were located within the basal layers, near the SGL, of the ventral and dorsal blades of the DG. In contrast, a greater proportion of the cells transduced with Ngn2 still expressed Tbr1 (19.6%±3.3%; Figure 7H and J), indicating that they were less mature than the vast majority of the NeuroD1-transduced cells. Immunohistochemistry for NeuN confirmed these data (Figure 7M and O, and figure S6A). Indeed, when we overexpressed Ngn2 the proportion of hippocampal progenitors that became NeuN+ was no greater than in <span class="yellow">rats</span> transduced with the control virus (Figure 7F).
As a whole, our in utero injection experiments confirm that NeuroD1 has a stronger neuron-inducing effect than Ngn2 when overexpressed in hippocampal progenitors.<br><br>Involvement of Mash1, Ngn2, Tbr and NeuroD proteins during <span class="yellow">human</span> hippocampal neurogenesis
Finally, we examined whether the transcription factors and cellular markers that are expressed in rodents are also expressed in <span class="yellow">human</span> hippocampus. We performed immunohistochemistry on aged <span class="yellow">human</span> hippocampal DG and found GFAP-, Sox2-, Pax6-, Nestin-, Prox1- and NeuN-immunopositive cells. This indicates that radial glia-like stem cells, neural progenitors and mature granule neurons are present in the aged <span class="yellow">human</span> hippocampus (Figure 8A–E). While Prox1 is found mainly in granule neurons, Sox2 and Pax6 are exclusively localized to cells in the SGZ (Figure 8D and E).
As for aged rodents, we did not observe the presence of Mash1, Ngn2, Tbr2, Tbr1 and NeuroD1 proteins in aged <span class="yellow">human</span> hippocampus, using immunohistochemistry. However, we could identify the presence of the transcripts, using RT-PCR (Figure 8F). In addition, we detected mRNA for Prox1, Calbindin1 and Calbindin2/Calretinin (Figure 8F). Interestingly, we also observed Sonic Hedgehog and Wnt-3A transcripts, indicating that they are present in the <span class="yellow">human</span> DG and supporting previous claims that they are important for the regulation of adult hippocampal neurogenesis (Figure 8F; [34], [35]). Finally, we also found mRNA for GLAST, Pax6 and GFAP which suggests that radial glia-like stem cells are present in the adult <span class="yellow">human</span> hippocampus, and may play a role in adult <span class="yellow">human</span> hippocampal neurogenesis (Figure 8F; [8]).
Overall, our data show that different proteins known to play key roles in <span class="yellow">rodent</span> hippocampal granule neurogenesis are present in the adult <span class="yellow">human</span> hippocampus.<br><br>
Discussion
In this study we determine the functions of Ngn2 and NeuroD1 during hippocampal neurogenesis. First, we map the hierarchy of molecular markers of neurogenesis in the DG. Second, we describe that Ngn2 is necessary for granule progenitor production/amplification. Third, we demonstrate that NeuroD1 directs neuronal differentiation of granule progenitors. Finally, we show that different cellular markers expressed during hippocampal neurogenesis in rodents are present in <span class="yellow">human</span>.
Sequential expression of different transcription factors and cellular markers during hippocampal neurogenesis
We clarified the pattern of expression of various markers expressed during postnatal and adult hippocampal granule neurogenesis in detail. We found that the transcription factor Pax6 is initially expressed by both radial glia stem cells (type-1) and early amplifying progenitors (type-2a). The next transcription factor to appear in chronology is Mash1. Mash1 expression characterizes the early stage of hippocampal progenitor amplification (early type-2a). The role of Mash1 during hippocampal granule neurogenesis is still unclear. Mash1 null mutant <span class="yellow">mice</span> do not display any clear malformation of the DG [20]. However, overexpression of Mash1 alone in the DG leads to the generation of oligodendrocytes [36]. Based on these observations, we hypothesize that Mash1-expressing cells are not yet committed towards a granule cell fate. They may still have the potential to generate both neurons and oligodendrocytes, as is the case in the SVZ [37].
In agreement with different developing brain regions, we found that Ngn2 starts to be expressed in Mash1-positive transiently amplifying progenitors, i.e. “late type-2a cells” according to the classification we propose. We observed that Pax6 expression persisted longer than that of Mash1, in Ngn2-expressing cells (late type-2a cells), in juvenile but not adult DG (Fig. 1 and figure S1). Later on, Ngn2 is downregulated, whilst Tbr2 expression persists (type-2b cells). The transition from amplifying progenitor to neuroblast is defined by the expression of NeuroD1. Thus, Ngn2 is expressed at the beginning of the transiently amplifying progenitor phase while NeuroD1 marks the end of that period. This applies both to the juvenile and adult <span class="blue">rat</span> brain [21], [25]. After NeuroD1 is turned on the progenitors leave the cell cycle, gradually mature, express PSA-NCAM, NeuroD2, Calretinin, Prox1, Tbr1 and, finally, NeuN. NeuroD1 expression persists at low levels in mature neurons (Figure 9; [10]). We found the sequential expression of these markers conserved in juvenile (P2 and 2 weeks) and adult <span class="yellow">rodent</span> brains. Importantly, we observed that the same transcription factors and neuronal markers are also present in the adult <span class="yellow">human</span> hippocampus and arranged spatially in a manner reminiscent with what we saw in rodents. It is possible, however, that some of these markers are expressed for longer or shorter periods at postnatal and adult stages of hippocampal progenitor maturation. Moreover, one can ask to which extent the number of divisions occurring during neuroblasts maturation is conserved at postnatal and adult stages of neurogenesis.<br><br>A new function for Ngn2 in maintenance of a progenitor state for granule neuron production/amplification during embryonic and postnatal hippocampal neurogenesis
In agreement with earlier work, we confirm that Ngn2 is indispensable for hippocampal development and plays a vital role in postnatal hippocampal granule neurogenesis. Ngn2 null mutant animals display a reduced size of the cornu ammonis and a malformed DG (Figure 3 and 4; [20]). We show that the number of newborn neurons (incorporated BrdU) is decreased in absence of Ngn2, but hippocampal granule neuron subtype specification is not affected, post-nataly. Likewise, neurons from other brain regions, e.g. ventral midbrain dopamine neurons, can be generated in the absence of Ngn2 [30], [31]. The reduced numbers of mature neurons in the hippocampus and ventral midbrain of Ngn2 null mutant animals does not appear to be due to cell death or migration defects [20], [31]. Instead it may be due to a defect in the generation and/or amplification of neuronal progenitors. Indeed, in the hippocampus of postnatal animals, we observed that cells lacking Ngn2 arrest in the cell cycle, maintain Pax6 expression and cease to proliferate.
When overexpressed in neural progenitors in vitro, Ngn2 lead to up-regulation of NeuroD1 and caused neuronal differentiation (Figure 6), which was not always the case when we overexpressed the same factor in vivo (Figure 7L, N and O). We also saw that Ngn2 does not always efficiently induce cell cycle exit. Typically some mitotically active eGFP+ cells were present in the DG of <span class="yellow">rats</span> injected with the Ngn2 retrovirus (Figure 7P), and we observed that cultured Ngn2-transduced cells are still capable of dividing. Thus, Ngn2 does not always promote cell cycle exit and neuronal commitment, but depending on the state of the cells, Ngn2 may instead promote alternate cellular fates [38]–[40]. Indeed, an earlier study has shown that Ngn2-overexpressing progenitors generate oligodendrocytes when grafted to the adult spinal cord [40]. We have also seen that Ngn2-overexpressing neural progenitors from the embryonic midbrain form astrocytes when grafted to the striatum (unpublished observations). Probably, Ngn2 cannot influence already committed cells, but rather would control neuroblasts production. One could speculate that Ngn2 oscillates during granule neuron formation, as recently demonstrated during neocorticogenesis [41], [42].<br><br>NeuroD1, but not Ngn2, is obligatory for granule neuron progenitor differentiation
Hippocampal granule progenitors can mature and express Tbr1 and Prox1, both during the development and postnatally, even in the absence of Ngn2. Because neuronal differentiation still occurs in Ngn2 null <span class="yellow">mice</span>, it is clear that compensatory, Ngn2-independent mechanisms induce NeuroD1 expression. One candidate is Ngn1, which is expressed during neocorticogenesis, the specification of olfactory sensory neurons and during embryonic <span class="blue">rat</span> hippocampal development [43]–[47]. Both Ngn1 and Ngn2 regulate NeuroD1 [19], [48]. The introduction of two mutant forms of Ngn1, a deletion of the basic region of Ngn1 and a substitution of two amino acids in the C-terminal basic region, prevents NeuroD1 expression and neuronal differentiation [48]. The double null Ngn1/Ngn2 mutant displays a more severe phenotype than single gene (Ngn1 or Ngn2) mutant <span class="yellow">mice</span>. For example, the total brain size of the double mutants is much smaller [10], [32]. If the role of Ngn1 during hippocampal neurogenesis is to activate NeuroD1 and that of Ngn2 is to control granule neuroblasts production/amplification, the DG of double Ngn1/Ngn2 knockout <span class="yellow">mice</span> should resemble that of NeuroD1 null mutant <span class="yellow">mice</span>.
We demonstrated that NeuroD1 directs neuronal differentiation both in vitro and in vivo. In progenitors isolated from different embryonic brain regions and cultured in vitro, we found that overexpression of NeuroD1 induced neuronal differentiation. The neurons generated expressed PSA-NCAM, Dcx and MAP2. After in utero retroviral vector-mediated gene delivery, virtually all cells transduced with NeuroD1 became neurons. In contrast, progenitors transduced with Ngn2 or control retroviruses adopted a neuron-like morphology in only 70–85% of cases. Under in vitro cell culture conditions, when we overexpressed Ngn2 in progenitors derived from the LGE we observed robust expression of NeuroD1 and neuronal differentiation. These in vitro results differ from those we obtained when we transduced the embryonic brain with a viral vector expressing Ngn2. In this latter case, Ngn2 did not induce neurons and the transduced cells did not express NeuroD1. Therefore, Ngn2 appears to direct non-neuronal cell type specification in vivo [38]–[40]. Taken together, we have confirmed that NeuroD1 plays a key role in neuronal differentiation in the hippocampus, both during development and in the adult brain.<br><br>Concluding remark
We present a detailed classification of different stages in hippocampal neurogenesis. Our detailed molecular mapping of hippocampal neurogenesis allows for a more accurate analysis of how new factors stimulate neurogenesis at different steps in the development of granule neurons. Thereby we hope to facilitate the development of new agents, which stimulate endogenous progenitors in the treatment of diseases.<br><br>
Materials and Methods
Animal tissue preparation
The creation of the Ngn2 transgenic <span class="yellow">mice</span> was reported elsewhere (ref guillemot). Heterozygote male and female <span class="yellow">mice</span> were crossed to obtain WT, heterozygote (Ngn2+/GFP) and null mutant (Ngn2GFP/GFP) animals. Tails of the Ngn2 offspring were used to obtain DNA for determination of the genotype using a polymerase chain reaction (PCR) assay as previously reported [31]. As null mutant Ngn2GFP/GFP do not survive longer than 2.5–3 weeks after birth, Ngn2GFP/GFP and their littermates were sacrificed at the postnatal ages of two days (P2) or two weeks for this study. Neurogenesis was assessed in two weeks or two months old WT <span class="yellow">mice</span>. Sprague Dawley pregnant <span class="yellow">rats</span> were ordered from B&K Universal Ltd, Sollentuna, Sweden (hppt://www.bku.com). All animals were housed in groups with ad libitum access to food and water at a 12-h light/dark cycle. All experimental procedures conducted in this study had been approved by the Ethical Committee at Lund University.
For immunohistochemical analysis, <span class="yellow">mice</span> (from two weeks old and adult stage) and juvenile <span class="yellow">rats</span> were sacrificed by transcardial perfusion with saline for 5–10 minutes, followed by 4% paraformaldehyde (PFA) for 10 minutes. Brains were kept in PFA overnight at 4°C and subsequently cryopreserved in a 20–30% sucrose/0.1 M phosphate buffer solution until sectioning on a microtome apparatus (30 µm thickness sections, Microm Zeiss). Seven series of coronal sections were cut throughout the brain. Free-floating sections were preserved in antifreeze solution until immunohistochemistry was performed. Heads of postanatal two days old <span class="yellow">mice</span> were decapitated and soaked in PFA 4% for 24 hours, at 4°C and transferred into sucrose solution until sectioning on cryostat apparatus (16 µm thickness; Leica CM3000). Sections were mounted on Superfrost glass slides and stored at −80°C until immunohistochemistry was performed.<br><br>Cloning, subcloning, virus production and titer measurement
The Moloney leukemia-derived retroviral vectors used in this study, pCMMP-IRES2eGFP-WPRE and pCMMP-Ngn2-IRES2eGFP-WPRE were previously described [40], [49]. To generate the construct pCMMP-NeuroD1-IRES2eGFP-WPRE, <span class="yellow">mouse</span> NeuroD1 cDNA was amplified from a pCS2+mtNeuroD1 plasmid (kindly provided by Professor Jackie Lee, Denver university, Boulder, USA) by PCR to introduce the restriction sites PmeI in 5′ and XhoI in 3′. Amplification of cDNA was performed as previously described [49]. The construction was verified by enzymatic restriction and by DNA sequencing using BigDye 3.1 (ABI). All infectious particles were produced using the producer cell line 293VSV-G and as previously described [49]. The titer of each retrovirus was measured by flow-cytometry based on eGFP expression, four days following infection of HT1080 cells and ranged from 0.5×109−2.1×109 TU/ml (All details on how to produce infectious particles can be provided upon request).<br><br>Neurosphere generation, transduction and differentiation
Pregnant female <span class="yellow">Sprague-Dawley rats</span> (B & K Universal, Sollentuna, Sweden) were terminally anesthetized by an overdose of sodium pentobarbital (i.p., 60 ng/ml). Embryos at stage embryonic day E14.5 (Plug day as day 0) were collected and cortical-hippocampal neurospheres were generated following dissection of the dorso-posterior part of the cortical tissue, and generated as previously described [49]. In this study, second passage (P2) neurospheres were used to study the effect of the overexpression of Ngn2 and NeuroD1 on neuronal differentiation. Each well, containing an equal starting population of 200,000 cells/ml, corresponding to 15–25 neurospheres, was transduced independently with each retrovirus at a multiplicity of infection (MOI) of 1, in proliferation medium supplemented by protamine-sulfate (4 mg/ml, Sigma). To induce differentiation, the medium was replaced with normal basic differentiation medium two days post-transduction and subsequently changed every other day until fixation.<br><br>Immunocytochemistry, immunohistochemistry and microscopy
The antibodies used in this study are: <span class="yellow">rabbit</span> anti-GFAP (1∶1000; DAKO), <span class="yellow">mouse</span> anti-Nestin (1∶100; BD PharMingen), <span class="yellow">guinea pig</span> anti-Glast (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Prox1 (1∶1000; Covance), <span class="yellow">goat</span> anti-Ngn2 (1∶20; Santa Cruz), <span class="yellow">goat</span> anti-NeuroD1 (1∶200; Santa Cruz), <span class="yellow">rabbit</span> anti-Pax6 (1∶150; Covance), <span class="yellow">rabbit</span> anti-Trb2 (1∶500; Chemicon), <span class="yellow">rabbit</span> anti-Trb1: (1∶1000; Chemicon), <span class="yellow">goat</span> anti-Dcx: (1∶500; Santa Cruz), <span class="yellow">mouse</span> anti-PSA-NCAM: (1∶500; Chemicon), <span class="yellow">mouse</span> anti-NeuN (1∶300; Chemicon), <span class="yellow">rabbit</span> anti-Calretinin (1∶500; Swant); <span class="yellow">mouse</span> anti-Sox2 (1∶100; R&D systems), <span class="yellow">mouse</span> anti-Mash1 (1∶100; BD PharMingen), <span class="yellow">rabbit</span> anti-NeuroD2 (1∶300; ABCAM), <span class="yellow">mouse</span> anti-MAP2 (1∶500; Sigma), <span class="yellow">rabbit</span> anti-Ki67 (1∶150; NovaCastra) and <span class="yellow">rabbit</span> anti-phospho-Histone H3. The secondary antibodies (1∶200) Cy2, FITC, Cy3 and Cy5 were from Jackson IR laboratories, Alexa-fluor 488, 568, 595 and 647 from Invitrogen-Molecular Probes. DAPI (1∶1000) was purchased from Sigma.
For immunocytochemistry, cultures were fixed in 4% paraformaldehyde at day five, rinsed with PBS three times prior to pre-incubation with a blocking solution (10% <span class="yellow">donkey</span> serum, 0.25% TritonX100 in PBS) for 1 hour. The remainder of the procedure was performed as previously described [49]. Specimen analyses were performed using a Leica confocal microscope (Leica software, equipped with a GreNe and a HeNe laser, using the following lines of excitation: 488 nm, 594 nm and 647 nm). Samples were analyzed using 20×, 40× and 63× objectives, sometimes zoomed. Figures were composed in CANVAS-X software.<br><br>Bromo-deoxyuridine (BrdU) and Cloro-deoxyuridine (CldU) pulse labeling and immunohisto- and immunocyto-chemistry
To assess cell proliferation and ongoing neurogenesis in vivo, animals were injected with BrdU (100 mg/kg, Sigma), two hours prior to sacrifice (for both P2 and two weeks old Ngn2 <span class="yellow">mice</span>). To assess neurogenesis, BrdU (100 mg/kg) was injected 48 hours prior to sacrifice (for P2 animals, BrdU was injected in pregnant dams half day prior to give birth). Immunohistochemistry was carried on as described above using a <span class="blue">rat</span> anti-CldU/BrdU primary antibody (1∶200, monoclonal, Immunologicalsdirect, Oxfordshire, UK), with an additional denaturation in 1 M HCl for 30 minutes prior to pre-incubation with serum.
To assess the neuronal-inducing activity of Ngn2 and NeuroD1, transduced cultures were incubated with CldU (20 µM, Sigma) for 2 days, after a period of differentiation of five days. For immunocytochemistry, cultures were fixed with 4% paraformaldehyde at day 7, rinsed with PBS three times, treated with 1 M HCl at 65°C for 5–10 min, pre-incubated and then incubated with a <span class="blue">rat</span> anti-CldU/BrdU antibody and other primary antibodies. The remainder of the procedure was performed according to the protocol for immunohistochemistry already mentioned.<br><br>In utero surgery
Timed pregnant female <span class="yellow">Sprague Dawley rats</span> with embryos at gestational age E15.5 were anesthetized with halothane. The mother was placed in the lower level of a two-level wooden stage. The abdomen was shaved with an electric razor and then cleaned with 70% alcohol. A 2–3 cm midline laparotomy was performed. Each uterine horn was carefully taken out individually and the number of embryos recorded. One horn was then placed back inside the mother, whilst the other horn was prepared for injection. The embryos were kept moist with constant application of warm saline to prevent dehydration. Approximately 2 µl of viral suspension (1×10∧9TU/ml) was injected into the lateral ventricle of each embryo, except for the embryo closest to the vagina. After the injections, the uterine horns were placed back into the abdomen. The abdominal wall and the overlying skin were then sutured. Care was taken not to damage abdominal muscles so that normal delivery of the pups was possible at term. The entire surgery generally took about 45 minutes to one hour. Each mother was allowed to recover in her cage before being returned to the animal stable. Shredded paper was added to each cage to encourage nesting and special care was taken not to stress the mothers. Following normal delivery, the pups were allowed to develop to adulthood up to two weeks.<br><br>Quantification
For the in vivo experiment, manual cell counting was performed on 18 and 30 µm thick brain sections for two days and two weeks old animals, respectively. The brain of P2 animals was cut into 10 series of coronal sections; the brain of two-weeks old animals was cut into 7–8 series of coronal sections. For each staining, one-two series from three to five different individuals were analyzed per genotype using a confocal microscope with 1, 2 or 3 lines of excitation, in sequential scanning in order to avoid false positives. When two series were analyzed, only one series was counted. The mean number of cells per hippocampus expressing the markers of interest, was calculated for each brain, based on the analysis of 6–8 consecutive dorsal hippocampal sections. The final mean number of cells per section was calculated by adding the mean number from each individual. We expressed the data as the mean number of positive cells±standard error of the mean (SEM). In this study, more than 25 Ngn2 null, >45 Ngn2 heterozygotes, and >60 WT individuals (including <span class="yellow">rats</span> injected with retroviruses) were analyzed, in total. Statistical comparison was performed using one-factor analysis of variance (ANOVA) with transcription factor, cellular marker, genotype or time as variables, followed by post-hoc analysis when significant differences were observed, using Statview 5.0 software (SAS institute Inc.). For cell culture experiments, three independent experiments were performed, each in duplicate. The counting was based on seven randomly chosen different fields of view. For each diagram, the level of significance (p-value) is represented as follows: P<0.05 = *; P<0.001 = **; P<0.0001 = ***. All data are expressed as±standard error of the mean (SEM).<br><br><span class="yellow">Human</span> sample and RT-PCR
The brain from a 64 year-old male was provided by the Harvard Brain Tissue Resource Center. The individual whose brain tissue was being analyzed gave written consent for storage and use of his tissue for research. Five millimeters thick fresh <span class="yellow">human</span> hippocampal tissue sample was snap frozen. The frozen tissue block was cut into 20 series of coronal sections, using a cryostat. Every 10 sections, one section was placed in an Eppendorf tube, on dry ice. Pooled sections were used for RT-PCR. Total RNA was prepared using Trizol and RNAeasy, supplemented with RNAGuard as previously described [49]. The RNA was digested with <span class="yellow">shrimp</span> DNAse before cDNA synthesis, which was performed using a mix of oligo-dT and random hexamer primers and SuperscriptII reverse transcriptase. Advantage2 polymerase mix (Clontech/BRL) was used for PCR, with the following cycling conditions: 10 cycles of 94° C for 30 seconds, 68°C for 2 minutes, 30 cycles of 94°C for 30 seconds, 60°C for 1 minute, 68°C for 1 minute and 30 seconds, one cycle of 68° C for 2 minutes, soak at 16°C. Primers marked * were designed by PrimerBank. hRPS18 (sense) 5′-GCCTTTGCCATCACTGCCATT and hRPS18 (antisense) 5′-GCCAGTGGTCTTGGTGTGCT, hPAX6 (sense) 5′-GCCCTGGAGAAAGAGTTTGAGAGAACCCATT hPAX6 (antisense) 5′-GGGGAAATGAGTCCTGTTGAAGTGGTGC, hTBR1 (sense) 5′-GCGGACACCAATGTGCAAGGAAATCG and hTBR1 (antisense) 5′-CGAGGGGGTCAGGCGGTCCATGTCACAGC, hTBR2 (sense) 5′-GACCTGTGGCAAAGCCGACAATAACATGC and hTBR2 (antisense) 5′-GGGGGTGTCTCTATCCAAGAAGAGCCAAT, hPDHX ( = PROX1) (sense) 5′-GGGACACTACGGTTCCGTTTAAGTCCAGC and hPDHX (antisense) 5′-CTCTCCATCCCAGCTTACATTAACATCTGGCATTTGT, hNGN2 (sense) * 5′-CATCAAGAAGACCCGTAGACTGA and hNGN2 (antisense)* 5′-CAACACTGCCTCGGAGAAG, hMASH1 (sense) 5′-CCTGGATCCGCATGGAAAGCTCTGCCAAGATGGAG and hMASH1 (antisense) 5′-CCTGGATCCCCCCTCAGAACCAGTTGGTGAAGTCGA, hNEUROD1 (sense) 5′-GCTCAGGACCTACTAACAACAAAGGAAATCGAAACATG and hNEUROD1 (antisense) 5′-CAAAGCGTCTGAACGAAGGAGACCAGGT, hGLAST (sense) 5′-CATCAGGGAAGATGGGAATGCGAGCTGTAGTCTATTAT and hGLAST (antisense) 5′-CCACGGGGGCATACCACATTATTACTGCTACC, hNEST (sense) 5′-CAGGAGCGGCTGCGGGCTACTGAAAAGTTCC and hNEST (antisense) 5′-CAGGGCTGAGGGGTGGTGCCAAGGAGG, hGFAP (sense) 5′-CCACGAGGAGGAGGTTCGGGAACTCCAGGAGC and hGFAP (antisense) 5-GGAATGGTGATCCGGTTCTCCTCGCCCTCTAGC, hCALB2 ( = Calretinin) (sense) 5′-GGCTCTGGCATGATGTCAAAGAGTGACAACTTT and hCALB2 (antisense) 5′-GGGCATCCAGCTCATGCTCGTCAATGTAGCC, hCALB1 (sense) 5′-GCGAAAGAAGGCTGGATTGGAGTTATCACC and hCALB1 (antisense) 5′-CCCTCCATCCGACAAAGCCATTATGTTCTTCTTGTATG, hSOX2 (sense) 5′-GGAGAACCCCAAGATGCACAACTCGGAGAT and hSOX2 (antisense) 5′-GAGGAAGAGGTAACCACAGGGGGGCTGGAGC, hWNT3A (sense) 5′-CCGAGGGCATCAAGATTGGCATCCAGGAGTGC and hWNT3A (antisense) 5′-TCGGGTTGCGACCACCAGCATGTCTTCACCTC, hSHH* (sense) 5′-ACTCCGAGCGATTTAAGGAACT and hSHH* (antisense) 5′-CAGACGTGGTGATGTCCACTG.<br><br>
Supporting Information<br><br>
<h3>pmcA327937</h3>Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a.
Abstract
Attempts to prevent the urea conversion of a 200-230,000 molecular weight DNA polymerase alpha to a 150-170,000 molecular weight form by the inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70,000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70,000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit.<br><br><br><br>
 Volume 6 Number 10 1979 <br><br> Nucleic Acids Research <br><br> Further studies on partially purified <span class="yellow">calf</span> thymus DNA polymerase a Keith McKune and Andrew M.Holmes* <br><br> Biochemistry Department, Strathclyde University, The Todd Centre, 31 Taylor Street, Glasgow, G4 ONR, UK <br><br> Received I June 1979 ABSTRACT <br><br> Attempts to prevent the urea conversion of a 200-230, 000 molecular weight DNA polymerase a to a 150-170, 000 molecular weight form by the <br><br> inclusion of protease inhibitors have not been successful. No other method has been found capable of dissociating a 50-70, 000 fragment or subunit from the DNA polymerase subunit. Addition of this 50-70, 000 subunit to the polymerase subunit does not aid the binding of the enzyme to DNA, but does have an effect on the utilisation of synthetic template-initiator complexes by the polymerase subunit. <br><br> INTRODUCTION <br><br> In mammals DNA polymerase a is thought to be the replicative <br><br> enzyme, but due to low levels of activity, even in tissues actively making DNA, and to enzyme heterogeneity it has been found difficult to purify <br><br> However, small samples of DNA polymerase a have been highly purified <br><br> 2, 3, 4, 5 from several sources and partially characterised <br><br> Heterogeneity has been observed in DNA polymerase a from a var<br><br> 6           7 <br><br> iety of sources, including <span class="blue">rat</span> liver and spleen , Hela cells, baby <span class="yellow">hamster</span> <br><br> 8                9, 10                   11 <br><br> kidney cells , <span class="yellow">mouse</span> myeloma    , <span class="yellow">Drosophila</span> embryos  and <span class="yellow">calf</span> thymus <br><br> 1,6, 12, 13 . We have previously observed several species of <span class="yellow">calf</span> thymus <br><br> DNA polymerase a differing in size and charge  . In order of elution from DEAE cellulose they are enzyme A1 (200-230, 000 molecular weight), A2 (200-230, 000), B (100-110, 000) and C (150-170, 000). A poly (dA). oligo <br><br> (dT)10 preferring enzyme, enzyrrme D (140-150, 000 molecular weight) elutes just after enzyme B. The A enzymes seem identical in all properties except their charge. 5.0-5. 5S enzymes, analagous to B, have been observed to <br><br> C Information Retrieval Limited 1 Falconberg Court London Wl V 5FG England <br><br> 3341 <br><br> Nucleic Acids Research <br><br> arise as the result of proteolytic action  ' , similarly the B enzyme above 14 <br><br> ,but the relationship of the other observed species is not clear, nor is <br><br> the problem of which, if any, of these species is the replicative enzyme, <br><br> although circumstantial evidence has implicated a DNA polymerase a to be <br><br> 15 the replicative enzyme in <span class="yellow">adenovirus</span> infected KB cells <br><br> We have previously shown that several of these enzymes are inter<br><br> 16 <br><br> convertible . In particular, mild urea treatment can convert both A <br><br> enzymes to C enzyme, with the loss of a subunit or fragment of 50-70, 000 molecular weight. The C enzyme retains DNA polymerase activity, but <br><br> does show differences compared to A enzyme in heat sensitivity and sensitivity to N-ethylmaleimide. <br><br> Highly purified samples of A and C enzymes when subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis have shown a corre<br><br> lation of DNA polymerase activity with a polypeptide band 155, 000 molecular weight . Contaminating material of 50-70, 000 molecular weight appeared <br><br> to obscure the putative 50-70, 000 subunit in gels of A enzyme. The conclusions drawn were that DNA polymerase a is a 155, 000 molecular weight <br><br> polypeptide (C enzyrrme) which can and does associated with material of 50<br><br> 70, 000 molecular weight to give A enzyme, and that enzyme B is a proteol<br><br> 14 <br><br> ytic degradation product . The relationship of enzyme D to the others is <br><br> not clear. <br><br> It is possible that during the enzyme isolation procedure proteolytic action may have taken place on a 200-230, 000 molecular weight enzyme releasing 155, 000 and 50-70, 000 fragments which remain in association <br><br> until urea treatment separates them. It is also possible that urea treatment itself renders the enzyme susceptible to contaminating proteases. Further experiments to ascertain whether or not this is the case and, if so, to prevent it, have been carried out. <br><br> A enzyme can be reconstituted from C enzyme formed by the action <br><br> of urea on A enzyme by concentrating it with the 50-70, 000 molecular weight material. The A enzyrme obtained in this manner is highly purified, as is <br><br> the C enzyme formed by the urea treatment, and has been used in attempts to show differences in synthetic template-initiator complex utilisation <br><br> 3342 <br><br> Nucleic Acids Research <br><br> b y the se two enzyme s . <br><br> MATERIALS AND METHODS <br><br> <span class="yellow">Calf</span> thymus was obtained from 10-16 week old <span class="yellow">calves</span> and frozen at <br><br> -20?C until required. Chromatographic media and chemicals were obtained <br><br> 6 <br><br> from sources previously referred to . Radioactive deoxynucleoside tri<br><br> phosphates were obtained from the Radiochemical Centre, Amersham, Bucks. Synthetic oligo and polynucleotides were obtained from P. L. Biochemicals <br><br> Inc., except for poly (dC) which was a gift from Dr. I.R. Johnston and pre<br><br> 17 <br><br> pared from oligo d(C)5 as described  . N-a-p Tosyl-L-lysine chloromethy<br><br> lketone HCl and phenylmethylsulphonylfluoride were obtained from Sigma and Trasylol from Bayer. <br><br> Except where indicated all buffers contained 20% w/v glycerol and <br><br> 1 mM dithiothreitol. Standard linear phosphate gradients were run between 0.03 M and 0.25 M potassium phosphate, pH 7.8. Gradient salt concentr<br><br> 2 <br><br> ations were measured using a conductivity meter as described . Urea was <br><br> prepared as a 4. 8 M solution in 20% w/v glycerol, stirred with Amberlite <br><br> MB3 and filtered. Dithiothreitol was added to a final concentration of 1 mM before use. <br><br> 6 DNA polymerase was assayed using activated DNA as described <br><br> except that the buffer was 50 mM tris HC1, pH 7.8. One unit of DNA polymerase activity incorporates ln mol [ H] dTMP into an acid insoluble form in one hour at 370C. Assays using synthetic template-initiator complexes were carried out at 30?C in 0. 12 ml. The template-initiator complexes <br><br> 6 <br><br> were prepared and assays processed as described . All assays contained <br><br> 1 mM dithiothreitol, 62. 5 p.g <span class="yellow">bovine</span> serum albumin, 1 ,ug template-initiator complex, enzyme protein and the relevant [ H] deoxynucleoside triphosphate at 0.1 mM and 12-15 cpm/pmol. These assays were carried out at either pH 6.4 in 20 mM sodium-potassium phosphate, or at pH 7.8 in 50 mM tris HCl and contained either 10 mM MgCl2 or 1 mM MnCl2 as indicated. <br><br> Preliminary purification of DNA polymerase a to Fraction IV was as <br><br> 6 <br><br> described , the purification steps being phosphocellulose chromatography, <br><br> ammonium sulphate precipitation and gel filtration on Sepharose 6B. <br><br> 3343 <br><br> Nucleic Acids Research <br><br> Enzyme obtained from the DEAE cellulose step is referred to as Fraction <br><br> V enzyme. Samples were prepared for sodium dodecyl sulphate polyacrylamide gel electrophoresis and scanned as previously described2 <br><br> RESULTS AND DISCUSSION <br><br> (a)   Interconversion Studies <br><br> A enzyme was routinely converted to C enzyme by incubating 200500 units/ml of Fraction V A enzyme in 2.4 M urea in 0. 02 M potassium <br><br> phosphate, pH 7.8, for 60 minutes at 0?C. The mixture was then loaded on to a DEAE cellulose, washed with 0. 03 M potassium phosphate pH 7. 8, and enzyrme eluted either with the standard phosphate gradient or batchwise. Under these conditions usually about 50-60% of recovered activity was <br><br> enzyme C (Fig 1 a). Overall recovery was 70-80%. A enzyme was recon<br><br> 14 <br><br> stituted essentially as described . The flow through material from the <br><br> DEAE cellulose column after the urea treatment was loaded on to a 1 x 0. 8 cm phosphocellulose column, washed with 0.03 M potassium phosphate, pH 7.8, and the protein eluted with 0.25 M potassium phosphate, pH 7. 8. This material, the putative subunit, was vacuum dialysed with the C enzyme produced by the urea treatment, rechromatographed on DEAE cellulose and enzyme eluted batchwise (Fig 1 b). Recovery from this procedure was <br><br> usually 50-70% A enzyme. Overall recovery was 30-40% of the C enzyme dialysed. If the material eluted from the phosphocellulose by the 0. 25 M potassium phosphate was heated to 90?C for 5 minutes prior to vacuum <br><br> dialysis with the C enzyme the recovery from the DEAE cellulose column <br><br> was significantly higher (60-70% of the original C enzyme activity), but all recoverable DNA polymerase activity was C enzyme. This, together with the fact that the 60 minute treatment with urea has, on occasions, given <br><br> rise to a 50% increase in DNA polymerase activity prior to loading on to the DEAE cellulose column, would indicate that A enzyme is less active on <br><br> activated DNA than is C enzyme. If the DEAE cellulose flow through material came from urea treatment of A2 enzyme then A2 was produced on reconstitution; if from A1 then A1 was produced. <br><br> Although the mild urea treatment of A has been used to prepare C of <br><br> 3344 <br><br> Nucleic Acids Research <br><br> b <br><br> 120       180 C <br><br> I         I    ^ <br><br> Figure la  DEAE cellulose chromatography after 2.4 M urea treatment. <br><br> 4, 5000 units, 4.4 mg of Fraction II A2 enzyme were incubated with urea at a final concentration of 2.4 M for 60 minutes at 0?C, loaded on to a 5 x 1.4 cm DEAE cellulose column, the column was washed with 0. 03 M potassium phosphate, pH 7.8 and a 200 ml standard phosphate gradient applied. 2.3 ml fractions were collected and 10 Rl assayed for 5 minutes. (o-o) no <br><br> phenylmethylsulphonylfluoride (*-*) 3 mM phenylmethylsulphonylfluoride ( - ) phosphate gradient. <br><br> Figure lb  The reconstitution of A2 enzyme. 3000 units for C enzyme <br><br> derived by urea treatment of A2 were vacuum dialysed with the DEAE cellulose flow through material after phosphocellulose chromatography and <br><br> chromatographed on a 2 x 1. 2 cm DEAE cellulose column. After washing with 0.03 M potassium phosphate, pH 7.7, the enzymes were eluted batchwise with the above concentrations of potassium phosphate, pH 7.8. 1 ml fractions were collected and 10 .l assayed for 10 minutes. <br><br> 3345 <br><br> Nucleic Acids Research <br><br> high specific activity it is possible that conversion of the 200-230, 000 molecular weight enzymes to a 155- 170, 000 species maybe due to unfolding of the molecule to allow limited attack by contaminating proteases. Accord<br><br> ingly the urea conversion of A to C was investigated in the presence of certain protease inhibitors. The presence of the serine protease inhibitor <br><br> phenylmethylsulphonyl-fluoride in the incubation and chromatography buffers did not affect the conversion of A2to C (Fig 1 a). Likewise trasylol and Na-p Tosyl-L-lysine chloromethylketone HC1 had no effect. However, protease action could have occurred earlier in the purification procedure and the urea could be separating two fragments. Usually the DEAE cellulose profile shows that the majority of the enzyme activity is present as A <br><br> enzyme (Fig 2). In this instance A1 and A2 have not been separated. When the temperature of the material in the original blending procedure was kept below 0?C or phenylmethylsulphonylfluoride, N-a-p Tosyl-L-lysine chloromethylketone HCl or trasylol was included in the isolation buffers the DEAE cellulose profile was similar. The A enzymes were still capable of conversion by mild urea treatment to C enzyme, indicating that if protease activity is involved then it is not susceptible to these inhibitors. When the <span class="yellow">calf</span> <br><br> 20FI: Now ~ ~      ~      20 20Fr Na      06 <br><br> Figure 2  DEAE cellulose elution profile of <span class="yellow">calf</span> thymus DNA polymerase a. 47, 000 units, 97 mg of Fraction IV enzyme prepared from 415 g of <span class="yellow">calf</span> <br><br> thymus were loaded on to a 10 x 1.8 cm DEAE cellulose column, after washing with 0. 03 M potassium phosphate, pH 7. 8 a 400 ml standard phosphate gradient was applied. 5 ml fractions were collected and 10 ,ul assayed for 5 minutes. <br><br> 3346 <br><br> Nucleic Acids Research <br><br> thymus was allowed to warm up during the blending procedure, or the supernatant prior to phosphocellulose chromatography was heated to 37 ?C for 30 minutes, there was a marked decrease in the amount of A enzyme with a concomitant increase in the amount of B and C (unpublished observation). The presence of phenylmethylsulphonylfluoride, N- a-p T osyl- L-lysine <br><br> chloromethylketone HCl or trasylol under these conditions only had a marginal effect on the appearance of C enzyme, but did reduce the amount of B <br><br> enzyme. Heating the enzyme to 37?C after the phosphocellulose step had no effect on the DEAE cellulose elution profile. Attempts to convert A enzyme to C using trypsin have not been successful. A enzyme activity is lost without the appearance of any other species (unpublished observation), although <br><br> 14 the action of trypsin on C enzyme can give rise to small amounts of B <br><br> Although the conversion of A enzyme to C does not appear to be the result of serine protease action, proteases other than serine proteases <br><br> 18 <br><br> could have been responsible  . Also, the fact that A enzyme can be recon<br><br> stituted from C plus the flow through material from the DEAE cellulose after urea treatment does not necessarily mean we are dealing with two subunits as fragments produced by proteases mray be reassembled to give active <br><br> enzyme 9    . However, the fact that a 200-230, 000 molecular weight polypeptide band has never been observed in sodium dodecylsulphate polyacrylamride gels of highly purified A enzyme , or even cruder fractions of A <br><br> enzyrre (unpublished observation), may be significant. One might expect <br><br> some of the enzyme not to have been attacked by whatever is responsible for cleaving the molecule, if, indeed, this does happen. It would appear, therefore, that the A enzyme consists of subunits of 155, 000 and 50-70, 000 molecular weight with the small subunit having a slightly different charge in the <br><br> case of A  and A2. It has also been concluded that the heterogeneity in the <br><br> 1                                     ~~~~~~~~~~~~~~~~~3 <span class="yellow">mouse</span> myeloma DNA polymerase a fraction is not due to proteolysis . However, the 50-70, 000 molecular weight subunit has not yet been identified. <br><br> Sodiurrm dodecylsulphate polyacrylamide gels of reconstituted A enzyme have shown polypeptide bands at 150-160, 000 and 50-70, 000 molecular weight <br><br> (Fig. 3), but the ratio of staining intensity of the bands does not correspond to a 1: 1 relationship. The ratio of the two bands is variable but is usually between 1:2 and 1:3, indicating, perhaps, that more than one subunit of 50<br><br> 3347 <br><br> Nucleic Acids Research <br><br> TOP      123   4 5   6 <br><br> Figure 3  Scan of a 5% sodium dodecyl sulphate polyacrylamide gel of <br><br> reconstituted A2 enzyme. 1500 units of reconstituted A2 enzyme were subjected to polyacrylamide gel electrophoresis under non-denaturing conditions. The gels were sliced, enzyme extracted and assayed and the peak <br><br> fraction of DNA polymerase activity from three gels were pooled, subjected to sodium dodecyl sulphate polyacrylamide gel electrophoresis on a single <br><br> gel, stained and scanned at 2 volt sensitivity as described2. The molecular weight standards were: (1) <span class="yellow">bovine</span> serum albumin dimer (134, 000), (2) ,3 galactosidase (130, 000), (3) phosphorylase a (94, 000), (4) <span class="yellow">bovine</span> serum <br><br> albumin (68, 000), (5) pyruvate kinase (57, 000) and (6) lactate dehydrogenase (35, 000). <br><br> 70, 000 molecular weight can associate with the 155, 000 subunit. The proportion of the lower molecular weight polypeptide band is higher in the reconstituted A enzyrrme than in the C enzyme preparation from which it was formed, but even the C enzyme contained some material in the region. <br><br> Breakdown of material from 155, 000 to 50-70, 000 may be partly responsible for the contamination, but the presence of phenylmethylsulphonylfluoride in samples in preparation for sodium dodecylsulphate polyacrylamide gel electrophoresis has not been successful in preventing it. The <span class="yellow">human</span> KB cell <br><br> DNA polymerase a, equivalent to the C enzyme, has been reported to consist <br><br> 4 <br><br> of subunits of 76, 000 and 66, 000 molecular weight , but at no time have we <br><br> observed polypeptide bands at 76, 000 and 66, 000 rising and falling with <br><br> enzyme activity in any gels of <span class="yellow">calf</span> thyrrmus enzymes A1, A  and C (and <br><br> 2 unpublished observations). (b)    Template Studies <br><br> Use has been made of the urea conversion of A to C and of the reconstitution of A enzyme to obtain samples of highly purified DNA polymerase, <br><br> specific activity in excess of 50, 000 units/mg, in order to study the effect of this 50-70, 000 molecular weight subunit on the DNA polymerase subunit. Previous results have indicated that A enzyrrme is stabler to heat and less <br><br> 3348 <br><br> Nucleic Acids Research <br><br> 14, 21 <br><br> susceptible to N-ethylmaleimide than C enzyme  . It has been reported <br><br> that highly purified DNA polymerase a can be associated with a protein capable of binding to DNA containing no 3' OH ends and capable of being released during the DNA polymerase assay  . Attempts to dissociate the 50<br><br> 70, 000 molecular weight subunit from the polymerase subunit by incubating A2 with DNA polymerase reaction mixes containing activated DNA, poly <br><br> (dA-T) and poly (dT). oligo (A)10 followed by ultracentrifugation in high salt have been unsuccessful. Similarly A absorbed and eluted from either native or denatured DNA cellulose remained A enzyme. Both A and C enzymes <br><br> were eluted from the DNA celluloses by less than 0. 1 M NaCl so it does not appear that the subunit enhances the binding of the DNA polymerase subunit to DNA. However, there are differences in the response of A and C <br><br> enzymes to synthetic template-initiator complexes (Table 1). Even if one takes into account the fact that A enzyme is less active on activated DNA <br><br> than C enzyme (the addition of the 50-70, 000 subunit to the polymerase subunit appears to result in a decrease of about 30% of polymerase activity on activated DNA) the A enzyme is still more active on these templates. <br><br> Although there is a variation in activity on these template-initiator comp<br><br> lexes each time assays are carried out on them depending on the method of preparing the complexes and the base ratio of template to initiator, A enzyme always seems to be significantly more active than C. The A2 <br><br> enzyme at pH 7.8, with extra subunit(s) is clearly more effective on the <br><br> oligoribonucleotide initiator, oligo(A)10 than is enzyme C. In view of the proposed RNA initiation of Okazaki pieces this may indicate a role of this subunit in Okazaki piece synthesis, in that it may aid the DNA polymerase to'take over' from the RNA polymerase. Using poly (dA). oligo (dT) 10 <br><br> (A:T=20: 1) and following the incorporation of [ H] dTMP as a function of time a short lag was observed for C enzyme, but not for A2 (Fig. 4). <br><br> Similar re sult s we re obtained when poly (dC). oligo (dG) 1 0 (C: G= 5: 1) wa s <br><br> used as temrrplate-initiator, but not when poly (dT). oligo (A)10 (T:A=1: 1) was used. Neither enzyme showed a lag on this template-initiator complex or on activated DNA. Incorporation versus enzyme concentration also showed this lag for C enzyme on poly (dA). oligo (dT)10. It is not certain what causes <br><br> 3349 <br><br> Nucleic Acids Research <br><br> TABLE 1 Template utilisation by reconstituted A and C derived from A. <br><br> 3H] dNTP Divalent cation  A2 at pH    C at pH <br><br> Template                          _      6.4   7.8    6.4   7.8 Activated DNA         dTTP         Mg++            100          100 Activated DNA         dATP         Mg++             91          85 Activated DNA         dGTP         Mg++             85          82 poly(dA).oligo(dT)10  dTTP         Mg      66.0    4.0    20    91 <br><br> (A:T = 20: 1) <br><br> poly(dA). oligo(dT)   dTTP         Mu      19.0    4. 5  6. 5  2.5 <br><br> poly(dA. oig~10 (A:T = 20:1) <br><br> poly(dT). oligo(dA)10  dATP        Mg       2.5   <1 (1       (1 <br><br> (T:A = 5:1) <br><br> poly(dT) .oligo(dA)1  dATP         Mn ++   15.5   23.0   9. 0  3. 0 <br><br> poly(dT). oligo(A)10  dATP         Mg      15.0  230.0  10.0  77. 5 <br><br> (T:A = 1:1) <br><br> poly(dT) * oligo(A) 10  dATP       Mn      41.5   75. 5 19.5  21.5 <br><br> (T:A = 1:1) <br><br> poly(dC). oligo(dG)   dGTP         Mg++     6. 0   15.5  4. 5  7. 5 <br><br> (C:G = 5:1)   10 <br><br> poly(dC). oligo(dG)10  dGTP        Mn +     6. 0  29.5   5. 0  6. 5 <br><br> (C:G  = 5: 1)                                 .    . <br><br> Values given are relative to incorporation of [ H] dTMP at pH 7. 8 on activated DNA. For A2 this was 176.5 pmol, for C 232 pmol. dATP, dCTP, dGTP and dTTP were included for assays using activated DNA, only the deoxynucleoside triphosphate stated was used in the synthetic template<br><br> initiator complex assays. Assays were for 10 minutes. The buffers and concentrations of the divalent cations were as in Materials and Methods. <br><br> this lag, but the annealing of template to initiator is only transient '  and the DNA polymerase subunit may have difficulty in stabilising the complex and the 50-70, 000 molecular weight subunit may be able to help the poly<br><br> merase subunit to overcome this. Addition of the 50-70, 000 subunit to the polymerase subunit prior to assaying with these templates had no effect on activity and it may be that a preincubation period is required before the two 3350 <br><br> Nucleic Acids Research <br><br> Figure 4 Activity of A2 and C enzymes on poly (dA). oligo (dT)jO (A:T = <br><br> 20:1) as a function of time. 50 ,ul samples were withdrawn at various times from a 0. 6 inl incubation mix at 30?C and added to 0. 5 ml 0. 1 M sodium <br><br> pyrophosphate containing 100 ,ug/ml native <span class="yellow">calf</span> thymus DNA and processed for counting in the usual manner6. The mix contained 1 mM dithiothreitol, 50 mM tris HC1, pH 7.8 10 mM MgC12, 0. 1 mM [3H] dTTP (15 cpm/pmol), 312.5 ,ug <span class="yellow">bovine</span> serum albumin, 5 Fg poly (dA). oligo (dT)  (A:T = 20:1) and enzyme protein (*-*) 3.3 units reconstituted A2, (o-o) 16. 0 units urea derived C enzyme. <br><br> subunits become fully associated. After the lag phase is over the C enzyme is still less active on these template-initiators than A enzyme . That is the A enzyme seems capable of elongating the initiator faster than the C enzyme. Experiments to determine whether the differences in rates of elongation of these template-initiator complexes are differences in processivity of the <br><br> enzymes under the different pH and divalent cation conditions are under way. <br><br> ACKNOWLEDGEMENTS <br><br> We thank the Medical Research Council for a research grant. <br><br> Communications concerning the paper should be sent to:<br><br> Dr. A. M. Holmes, Department of Biochemistry, University of Strathclyde, The Todd Centre, 31 Taylor Street, Glasgow G4 ONR, U.K. <br><br> 3351 <br><br> Nucleic Acids Research <br><br> REFERENCES <br><br> (1)    Bollum, F.J. (1975). Prog. Nucleic Acid Res. Mol. Biol. 15, <br><br> 109-144. <br><br> (2)    Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1976). Eur. <br><br> J. Biochem. 62, 229-235. <br><br> (3)    Matsukage, A., Sivarajan, M. and Wilson, S.H. (1976). Biochem<br><br> istry 15, 5305-5314. <br><br> (4)    Fisher, P.A. and Korn, D. (1977). J. Biol. Chem. 252, 6528<br><br> 6535. <br><br> (5)    Fichot, O., Pascal, M., Mechali, M. and De Recondo, A, -M. <br><br> (1979). Biochim. Biophys. Acta, 561, 28-41. <br><br> (6)    Holmes, A.M., Hesslewood, I.R. and Johnston, I.R. (1974). Eur. <br><br> J. Biochemn. 43, 487-499. <br><br> (7)    Noy, G.P. and Weissbach, A. (1977). Biochim. Biophys. Acta, <br><br> 447, 70-83. <br><br> (8)    Craig, R.K. and Keir, H. M. (1975). Biochem. J. 145, 225-232. <br><br> (9)    Matsukage, A., Bohn, E.W. and Wilson, S.H. (1974). Proc. Natl. <br><br> Acad. Sci. U.S.A. 71, 578-582. <br><br> 110)   Hachmann, H.J. and Lezius, A.G. (1975). Eur. J. Biochem. 50, <br><br> 357- 366. <br><br> (11)   Brakel, C.L. and Blumenthal, A.B. (1977). Biochemistry, 16, <br><br> 3137-3143. <br><br> (12)   Momparler, R.L., Rossi, M. and Labitan, A. (1973). J. Biol. <br><br> Chem. 248, 285-293. <br><br> (13)   Yoshida, S., Konda, T. and Ando, T. (1974). Biochim. Biophys. <br><br> Acta, 353, 463-474. <br><br> (14)   Holmes, A.M., Hesslewood, I.P., Wickremasinghe, R.G. and <br><br> Johnston, I.R. (1977). Biochem. Soc. Symp. 42, 17-36. <br><br> (15)   De Jong, A., Van der Vliet, P. and Jansz, H.S. (1977). Biochim. <br><br> Biophys. Acta, 476, 156-165. <br><br> (16)   Holmes, A.M., Hesslewood, I.P. and Johnston, I.R. (1975). <br><br> Nature (London) 255, 420-422. <br><br> (17)   Bollum, F. J. (1966). Procedures in Nucleic Acid Research <br><br> (Cantoni, G.L. and Davis, D.R. eds). pp 577-583, Harper and Row, New York. <br><br> (18)   Barrett, A.J. (1975). Proteases and Biological Control pp 467-482, <br><br> Cold Spring Harbor Laboratory, Cold Spring Harbor. <br><br> (19)   Richards, F. M. and Vithayathil, P.J. (1960). Brookhaven Symp. <br><br> Biol. 13, 115-134. <br><br> (20)   Lowe, P.A., and Malcolm, A.B.D. (1976). Eur. J. Biochem. 64, <br><br> 177- 188. <br><br> (21)   Hesslewood, I.P., Holmes, A.M., Wakeling, W.F. and Johnston, <br><br> I.R. (1978). Eur. J. Biochenr.. 84, 123-131. <br><br> (22)   Mechali, M. and De Recondo, A. -M. (1978). Biochim. Biophys. <br><br> Res. Comm, 82, 255-264. <br><br> (23)   Chang, L.M.S., Cassani, G.R. and Bollum, F.J. (1972). J. Biol. <br><br> Che. 247, 7718-7723. <br><br> (24)   Wickremasinghe, R.G. and Johnston, I.R. (1974). Biochim. <br><br> Biophys. Acta, 361, 37-52. <br><br> 3352 <h3>pmcA411032</h3>Down-regulation of the M6P/IGF-II receptor increases cell proliferation and reduces apoptosis in neonatal <span class="blue">rat</span> cardiac myocytes
Abstract
Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a multi-functional protein that has been implicated in regulation of cell growth and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R, and cardiomyocyte apoptosis has been identified in a variety of cardiovascular disorders, such as myocardial infarction and heart failure. However, involvement of M6P/IGF2R in the pathogenesis of these conditions has not been determined. Thus, the objective of this study was to determine the role of M6P/IGF2R in regulation of cardiac myocyte growth and apoptosis.<br><br>Results
We down-regulated the expression of M6P/IGF2R in neonatal <span class="blue">rat</span> cardiac myocytes and examined the effect on cell proliferation and apoptosis. Infection of neonatal cardiomyocytes with an <span class="yellow">adenovirus</span> expressing a ribozyme targeted against the M6P/IGF2R significantly reduced the level of M6P/IGF2R mRNA, as determined by RT-PCR and Ribonuclease Protection Assay (RPA). M6P-containing protein binding and endocytosis as well as the M6P/IGF2R-mediated internalization of 125I-IGF-II were lower in the ribozyme-treated cells than the control myocytes, indicating that the number of functional M6P/IGF2R in the ribozyme treated cells was reduced. Accordingly, a marked increase in cell proliferation and a reduced cell susceptibility to hypoxia- and TNF-induced apoptosis were observed in the ribozyme-treated cells.<br><br>Conclusions
These findings suggest that M6P/IGF2R may play a role in regulation of cardiac myocyte growth and apoptosis. Down regulation of this gene in cardiac tissues might be a new approach to prevention of cell death or promotion of mitogenesis for certain heart diseases.<br><br><br><br>Background
The mannose 6-phosphate/insulin-like growth factor-II receptor (M6P/IGF2R) is a unique protein that interacts with multiple ligands, some of which are important growth regulatory factors [1]. The M6P/IGF2R participates in internalization and lysosomal degradation of IGF-II, a mitogen normally acting through the IGF-I receptor to stimulate cell proliferation [2]. The M6P/IGF2 receptor is required for the activation of TGF-β [3], a potent growth inhibitor for many cell types. This receptor is also involved in the binding, transport and activation of newly-synthesized lysosomal enzymes, such as cathepsins [4,5], which have been recently implicated in the induction of apoptosis [6]. On the basis of these functions, the M6P/IGF2R has been proposed to play a significant role in regulation of cell growth and apoptosis [7].
Apoptosis, or programmed cell death, is a tightly regulated process used to remove excess, hazardous or damaged somatic cells, and is crucial for the development, maintenance and survival of an organism. However, alterations in the control of apoptosis have also been shown to contribute to <span class="yellow">human</span> diseases. In fact, morphological and biochemical markers of apoptosis have been identified in a wide variety of cardiovascular disorders, including myocardial infarction and heart failure. This suggests that activation of apoptotic pathways contributes to cardiomyocyte loss and subsequent cardiac dysfunction in these conditions. A number of factors involved in cardiomyocyte apoptosis are currently known and include insulin-like growth factor-I (IGF-I), stress-activated protein kinases (SAPKs) and the anti-apoptotic Bcl-2 family [8]. There are indications that other factors may be involved in induction and regulation of cardiac apoptosis. However, these potential factors and their corresponding mechanisms have not been identified.
Several lines of evidence point to the potential involvement of M6P/IGF2R in cardiac myocyte proliferation and apoptosis. Cardiac myocytes express relatively high levels of M6P/IGF2R and transgenic <span class="yellow">mice</span> containing a homologous deletion of the M6P/IGF2R gene manifest ventricular hyperplasia due to an increase in cell number [9,10], suggesting that the M6P/IGF2R normally acts to suppress cardiac myocyte cell growth. It has also been shown that TGF-β, a potent growth suppressor whose activation requires the binding of latent TGF-β to M6P/IGF2R [3], is commonly upregulated in chronic heart failure [11]. Additional evidence for the involvement of M6P/IGF2R in regulation of apoptosis comes from studies of tumorigenesis. It has been shown that M6P/IGF2R expression is significantly reduced in a variety of tumors and loss of heterozygocity (LOH) at the M6P/IGF2R gene locus 6q26 have been found in breast, liver cancers and squamous cell carcinoma of the lung [12-15]. Although several studies have examined the effect of M6P/IGF2R over-expression on cell growth [7], it is not known whether down-regulation of this receptor protein leads to cellular protection against apoptosis.
Ribozymes are catalytic RNA molecules that cleave a complementary mRNA sequence [16], thereby inactivating specific mRNAs and suppressing gene expression in vitro and in vivo [17,18]. Ribozymes have been shown to be highly specific, efficient and stable. They can be packaged into viral vectors to enhance transfer into cells and to achieve longer expression compared with naked oligonucleotides. In the present study, we employed ribozyme technology to study the role of M6P/IGF2R in regulation of cardiac myocyte cell growth. A hammerhead ribozyme against the M6P/IGF2R mRNA was constructed and packaged in an adenoviral vector. We then examined the effect of ribozyme-mediated down-regulation of M6P/IGF2R expression on cell growth and hypoxia- and TNF-induced apoptosis.<br><br>Results
Cleavage reaction of the ribozyme in vitro
The M6P/IGF2R ribozyme we constructed has 13-bp binding arms complementary to the target site of M6P/IGF2R mRNA, and a catalytic core (Fig. 1A). To evaluate the bioactivity of the ribozyme and the accessibility of the target site, a cleavage reaction was performed in vitro. The substrates, [α-32P] labeled RNA transcripts containing 45 bp of M6P/IGF2R mRNA or an unmatched sequence, were incubated with the ribozyme as described (see Materials and Methods). The ribozyme cleaved only the specific M6P/IGF2R mRNA into the expected products. In the assay of time course, the hammerhead ribozyme was able to cleave 24.2% of the M6P/IGF2R target within 10 minutes of incubation, 50.3% of the M6P/IGF2R target within 40 minutes of incubation, and by 640 minutes, 80.8% of the M6P/IGF2R target was converted to the expected products (Fig. 1B). This ribozyme did not digest the unmatched sequence (Fig. 1B). These results indicate a high efficiency and specificity of the ribozyme in vitro.<br><br>Ribozymes down-regulate M6P/IGF2R expression in cardiac myocytes
To examine the ability of the ribozyme to reduce levels of M6P/IGF2R mRNA in cultured cardiac myocytes, total RNA was extracted from cells infected with Ad-GFP/IGF2R-Rz or Ad-GFP, and subjected to RT-PCR using M6P/IGF2R-specific primers. Primers specific for β-actin were added to a parallel reaction to serve as an internal standard. Cells were used 4 days after infection, with average infection efficiency of 70–80% (for which a viral dose used had minimal cytotoxicity). The RT-PCR product of M6P/IGF2R was 856 bp, and the β-actin product was 285 bp. As shown in Fig. 2A, the Ad-GFP/IGF2R-Rz-infected cells exhibited a significantly lower level of M6P/IGF2R mRNA than Ad-GFP-infected cells, with a reduction of about 50%. This result was confirmed by ribonuclease protection assay (RPA), in which GAPDH was used as a control (Fig. 2C &2D). There was no significant difference in the level of M6P/IGF2R mRNA between Ad-GFP-infected cells and uninfected cells (data not shown), indicating that infection with the <span class="yellow">adenovirus</span> itself did not alter the endogenous M6P/IGF2R mRNA level. The results demonstrated that the ribozyme was highly effective in suppressing M6P/IGF2R expression in cultured cardiac myocytes.<br><br>Effect of ribozyme expression on the functional activity of M6P/IGF2R
To determine the effect of the ribozyme on the functional activity of M6P/IGF2R, binding and internalization of exogenous 125I-IGF-II was measured in cells infected with Ad-GFP/IGF2R-Rz. As shown in Fig. 3A, cells infected with Ad-GFP/IGF2R-Rz showed a 54% reduction in 125I-IGF-II internalization when compared with the control cells (infected with Ad-GFP). We also examined the effect of the ribozyme on the M6P-binding activity of the M6P/IGF2R using the M6P-bearing lysosomal enzyme, β-glucuronidase, as a probe. The results showed that the maximal M6P-binding capacity of cells treated with the ribozyme was about 50% less than that of controls (Fig. 3B). Furthermore, we assessed the ability of cells to internalize exogenous β-glucuronidase after treatment with ribozyme. Similarly, the M6P-inhibitable endocytosis of β-glucuronidase by ribozyme-treated cells was about 52% less than that of control cells (Fig. 3C). These results confirm that the number of functional M6P/IGF2R in ribozyme-treated cells was reduced.<br><br>Adenoviral delivery of ribozymes increases the proliferation of cardiac myocytes
We examined the effects of the ribozyme on the growth of cultured neonatal <span class="blue">rat</span> cardiac myocytes. Morphological evaluation showed a remarkable difference in growth pattern between Ad-GFP/IGF2R-Rz-infected cells and the control cells: the ribozyme-expressing cells formed larger and more spread colonies (Fig. 4). Assessment of cell proliferative activity by the MTT assay and counts of viable cells showed that the number of cardiac myocytes in ribozyme-expressing cultures was significantly higher than in control cultures (Fig. 5). These results indicate that treatment with M6P/IGF2R-ribozyme can promote cardiac myocyte proliferation.<br><br>Effect of M6P/IGF2R-ribozyme expression on apoptosis of cardiac myocytes
We examined the effects of ribozyme expression on TNF-α and hypoxia-induced apoptosis of cultured cardiac myocytes. After a 24 hr challenge with hypoxia, the number of apoptotic cells in M6P/IGF2R-Rz expressing cultures was 38% lower than in control cultures as determined by Hoechst staining (which highlights the nuclei of apoptotic cells) and ELISA (Fig. 6A, 7A). MTT analysis showed that the number of viable cells in ribozyme-treated cultures was 40% higher than in control cultures (Fig. 7A).
After treatment with TNF-α, as shown in Fig. 6B, a large number of control cells underwent apoptosis, as indicated by morphological changes (small round shape) and bright blue nuclear staining. There were significantly more apoptotic cells in control cultures than in cultures expressing the Ad-GFP/IGF2R-Rz. The number of apoptotic cells, as measured by the cell death ELISA assay, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 40%) lower than in cultures infected with Ad-GFP (Fig. 7B). Accordingly, the number of viable cells, as measured by MTT analysis, in cultures infected with Ad-GFP/IGF2R-Rz was significantly (about 45%) higher than in cultures infected with Ad-GFP (Fig. 7B). These results are consistent with the hypothesis that decreasing M6P/IGF2R expression by ribozyme treatment can reduce cell apoptosis.<br><br>
Discussion
Some 62,000,000 Americans have one or more types of cardiovascular disease (CVD) and CVD is the leading cause (40.1%) of death in the United States. Myocardial infarction and heart failure, conditions accompanied by cardiac myocyte apoptosis, represent 23% of all CVDs and are a growing clinical challenge in need of novel therapeutic strategies. In this study, we investigated the M6P/IGF2R as a potential new therapeutic target for reduction of cardiac apoptosis and cardiac injury in these conditions.
Using ribozyme technology we down-regulated the expression of the M6P/IGF2R in neonatal cardiac myocytes. We then examined cell proliferation and apoptosis under normal conditions and post challenge with either hypoxia, a model of ischemia-reperfusion, or TNF-α, a cytokine implicated in the pathogenesis of chronic heart failure [19]. Our results demonstrate an association of a decrease in the expression and function of the M6P/IGF2R with increased cell proliferation and decreased cell susceptibility to hypoxia- and TNF-induced apoptosis. Expression of the ribozyme targeted against the M6P/IGF2R in cardiomyocytes resulted in down-regulation of M6P/IGF2R expression, as measured by RT-PCR and RPA, and of M6P/IGF2R function, as indicated by a decrease in internalization of 125I-IGF-II, and β-glucuronidase binding and endocytosis.
MTT analysis and viable cell counts showed that ribozyme-mediated down-regulation of M6P/IGF2R resulted in a marked increase in cell proliferation of cardiomyocytes, which normally express high levels of M6P/IGF2R [20] and have limited proliferative capabilities [21]. These results are consistent with the findings of previous knockout studies [9,10]. Since the M6P/IGF2R has multiple actions on cell growth, its proliferative effect on the heart cells observed in this study might involve multiple mechanisms. However, it is likely that unchecked IGF-II stimulation plays a key role in the effect. Because the M6P/IGF2R is believed to sequester and degrade IGF-II [2], a decrease in M6P/IGF2R expression and function could result in decreased degradation and hence increased bioavailability of IGF-II to the IGF-I receptor, which mediates the growth-promoting effect of IGF-II. Supporting evidence for the involvement of IGF-II in the proliferative effect resulting from loss of M6P/IGF2R function comes from studies of M6P/IGF2R knock-out <span class="yellow">mice</span>. M6P/IGF2R-null <span class="yellow">mice</span> display global hyperplasia that coincides with elevated levels of IGF-II. Most importantly, however, the lethal nature of an M6P/IGF2R-null phenotype is reversed in an IGF-II-null background [9]. Our results showing that ribozyme-mediated down-regulation of M6P/IGF2R lead to a decrease in IGF-II internalization support the above possibility. However, further investigation to confirm this mechanism is warranted.
More importantly, our results also showed that M6P/IGF2R down-regulation resulted in decreased sensitivity of cardiomyocytes to hypoxia- and TNF-induced apoptosis. There is evidence that lysosomal enzymes, such as cathepsins B and D contribute to hypoxia- and TNF-induced apoptosis in vitro [22-25] and in vivo [26,27]. The M6P/IGF2R has been shown to be involved in binding, transport and activation of lysosomal enzymes, including cathepsins [4,5]. Therefore, it is possible that down-regulation of the M6P/IGF2R results in improper trafficking and activation of cathepsins. This, in turn would eliminate the apoptotic cascades triggered by these enzymes under hypoxia and TNF stimulation and result in decreased sensitivity of cardiomyocytes to apoptosis.
It has also been shown that TNF stimulation involves the activation of TGF-β [28-30], a ligand of M6P/IGF2R that has been implicated in the progression of chronic heart failure [11,31]. Therefore, down-regulation of M6P/IGF2R expression could also lead to a decreased bioavailability of activated TGF-β, thereby decreasing the sensitivity of cardiomyocytes to the TNF/TGF-β apoptotic pathway. The detailed mechanism of the observed effects is unknown and requires further investigation.<br><br>Conclusions
The present study demonstrates that ribozyme-mediated down-regulation of expression and functional activity of the M6P/IGF2R results in a decrease in the susceptibility of cardiac myocytes to apoptotic stimuli. These findings suggest that this receptor might be involved in cardiac cell growth and apoptosis. The ability of the M6P/IGF2R ribozyme to reduce M6P/IGF2R expression and function in transfected cells verifies the utility of the ribozyme in studying the role of M6P/IGF2R in cardiomyocyte growth and apoptosis. In addition to its utility as a research tool, the ribozyme, with further exploration and development, might have potential application as a therapeutic agent to prevent cell death or promote mitogenesis for certain clinical conditions, such as, myocardial infarction and chronic heart failure.<br><br>Methods
Construction of recombinant M6P/IGF2R-RZ adenoviral vector
The nucleotide numbers of the <span class="blue">rat</span> M6P/IGF2R sequence targeted by the hammerhead ribozyme is 1147–1160 after coding site (exon 9). The structure of the M6P/IGF2R hammerhead ribozyme is shown in Fig. 1. A 49 bp M6P/IGF2R ribozyme oligonucleotide, 5'-GAATTCCCC ACACTG ATGAGCCGCTTCGGCGGCGAAACATTCAAC GCGT-3' and the corresponding reverse complementary strand were synthesized. The fragments were subcloned to produce a plasmid containing a ribozyme against M6P/IGF2R. For construction of the recombinant <span class="yellow">adenovirus</span> containing the M6P/IGF2R-ribozyme (pAd-GFP/IGF2R-Rz), the segments containing the ribozymes were amplified by PCR and cloned into a pAdTrack-CMV vector and then recombined homologously with an adenoviral backbone pAdEasy 1 vector to generate (pAd-GFP/IGF2R-Rz), following the protocol described by He et al. [32]. The pAd-GFP/IGF2R-Rz carries both the IGF2R-Rz and GFP (as reporter) genes, each under the control of separate cytomegalovirus (CMV) promoters. Another viral vector, pAd-GFP, which carries the GFP gene only under the control of the CMV promoter, was generated and used as a control vector. The adenoviral vector DNA were linerized with Pac I and transfected into the replication-permissive 293 cells (E1A transcomplementing cell line) by using Lipofectamine (Life Technologies) to produce E1-deleted, replication-defective recombinant <span class="yellow">adenovirus</span> as described previously [33]. Large-scale amplification of recombinant <span class="yellow">adenovirus</span> in 293 cells was followed by purification using a discontinuous CsCl gradient. The constructs were confirmed by enzymatic digestion and DNA sequencing.<br><br>Transcription and cleavage reaction of ribozyme in vitro
Plasmids containing the ribozyme or the substrate (either 45 bp of M6P/IGF2R mRNA or an unmatched sequence 5'-GTGCTGTCTGTATG-3') were linearized with MluI, respectively. All transcripts were generated with T7 RNA polymerase (Promega). Substrate transcripts were labeled by incorporation of [α-32P] UTP (NEN Life Science Products, Inc.). Specific activity of the [α-32P] UTP (10 μCi/μl) and the base composition of each substrate molecule were used to calculate the substrate concentration. Ribozyme transcripts were quantified spectrophotometrically. (The half-life of the M6P/IGF2R target is about 280 minutes).
Cleavage reaction mixture contained substrate RNA (40 nM), increasing amounts of ribozyme (60 nM), 20 mM MgCl2 and 20 mM Tris-HCl, pH8.0, in a final volume of 10 μl. The mixture was incubated at 37°C for a time-course of cleavage reaction from 0, 5, 10, 20, 40, 80, 160, 320, to 640 minutes and the cleavage reaction was stopped by addition of loading buffer (80% formamide, 10 mM Na2EDTA, pH 8.0, and 1 mg/ml each bromophenol blue and xylene cyanol). Cleavage products were analyzed on a 15% polyacrylamide and 8M urea gel. Product and substrate fragments were quantitated by using NIH Imager.<br><br>Cell cultures and infection with Ad-GFP/Rz-IGF2R and Ad-GFP
Cardiac myocytes were isolated from 1-day-old newborn <span class="yellow">rats</span> using the Neonatal Cardiomyocyte Isolation System (Worthington). The isolated cells were plated in 6-well plates and cultured in F-10 medium containing 5% (vol/vol) FBS and 10% (vol/vol) <span class="yellow">horse</span> serum at 37°C in a tissue culture incubator with 5% CO2 and 98% relative humidity. Cells were used for experiments after 2–3 days of culture. Viral infections were carried out by adding viral particles at various concentrations (usually, 2 × 108 virus particles/ml) to culture medium containing 2% (vol/vol) FBS. Initially, optimal viral concentration was determined by using Ad-GFP to achieve an optimal balance of high gene expression and low viral titer to minimize cytotoxicity. After 24 hours of incubation, the infection medium was replaced with normal (15% vol/vol serum) culture medium. For treatment with IGF-II, cells were incubated with 50 ng/ml IGF-II after 24 hours infection with Ad-GFP/IGF2R-Rz or Ad-GFP. Four days after infection, cells were used for analysis of gene expression of M6P/IGF2R and its effect on cell growth and apoptosis.<br><br>Analysis of gene expression in cardiac myocytes
The M6P/IGF2R transcripts were determined by both RT-PCR and Ribonuclease Protection Assay (RPA). RT-PCR was performed using the GeneAmp EZ rTth RNA PCR kit (Roche). Total RNA was extracted from cultured cells using an RNA isolation kit (Qiagen,), according to the manufacturer's protocol. M6P/IGF2R transcripts were amplified using the primers (5'-GACAGGCTCGTTCTGACTTA-3') and (5'-CTTCCACTCTTATCCACAGC-3') specific to the M6P/IGF2R. Each RT-PCR assay was performed in triplicate and product levels varied by less than 3.2% for each RNA sample. Primers specific for β-actin cDNA were added to a parallel reaction to standardize for variations in PCR between samples. PCR products were resolved on a 1.0% agarose gel, visualized under UV light and quantitated using NIH Imager.
RPA was performed using the RPA III kit (Ambion, Austin, TX). Briefly, total RNA was extracted from cultured cells using a total RNA isolation reagent (TRIzol, Gibco BRL) according to the manufacturer's protocol. The plasmid containing the <span class="blue">rat</span> M6P/IGF2R gene was linearized and used as a transcription template. Antisense RNA probes were transcribed in vitro using [33P]-UTP, T7 polymerase (Riboprobea System T7 kit, Promega), hybridized with the total RNA extracted from the <span class="blue">rat</span> cardiomyocytes, and digested with ribonuclease to remove non-hybridized RNA and probe. The protected RNA·RNA was resolved on a denaturing 5% sequence gel and subjected to autoradiography. A probe targeting the GAPDH gene was used as an internal control.<br><br>Measurement of 125I-IGF-II internalization
Cells were incubated at 37°C for 2 hrs in serum-free F-10 culture medium containing 125I-labeled IGF-II (0.5 ng/ml) with or without excess unlabeled IGF-II (2 μg/ml). Following the incubation, the cells were washed three times with ice-cold PBS, and cell-associated radioactivity was determined by a γ counter. Specific internalized 125I-IGF-II was calculated by subtracting the count of samples with excessive unlabeled IGF-II from that without unlabeled IGF-II, and normalized to protein contents.<br><br>Beta-glucuronidase binding assay
Binding of β-glucuronidase was assayed as described previously [34,35]. Briefly, cells were permeabilized with 0.25% saponin in 50 mM Hepes (pH 7.0), 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="yellow">human</span> serum albumin, and 10 mM mannose-6-phosphate (M6P) for 30 minutes on ice. The cells were washed three times with ice-cold PBS containing 0.05% saponin. They were incubated with 20,000 units/ml β-glucuronidase from <span class="yellow">bovine</span> liver (Sigma) in 50 mM Hepes (pH 7.5) containing 150 mM NaCl, 5 mM β-glycerophosphate, 0.5% <span class="yellow">human</span> serum albumin, 0.5% saponin with or without 10 mM M6P overnight on ice. Cells were washed five times with ice-cold PBS containing 0.05% saponin and sonicated in 100 mM sodium acetate (pH 4.6). The protein concentration of solubilized cell extract was measured and enzyme activity was assayed as follows: for each reaction 50 ul cell extract were added to 500 ul of 100 mM sodium acetate (pH 4.0) containing 1 mM paranitrophenyl (PNP)-β-glucuronide (Sigma) as substrate. After an incubation period of 3 hours at 37°C, 500 ul 1 M Na2CO3 were added to each reaction and the absorbance was measured at 400 nm. Experimental values were compared to a standard curve that was constructed using 1–100 nM solutions of PNP (Sigma) in 500 ul 100 mM sodium acetate and 500 u1 1 M Na2CO3. Specific activity was calculated as nM of PNP produced/hour/mg of protein.<br><br>Beta-glucuronidase endocytosis assay
Beta-glucuronidase endocytosis assay was carried out as described previously [36]. Briefly, confluent cell cultures were washed twice with pre-warmed serum-free DMEM followed by incubation with DMEM containing 5 mg/ml <span class="yellow">human</span> serum albumin and 10 mM M6P for 20 minutes. Following incubation cells were washed 3 times with pre-warmed DMEM. Cells were then incubated in DMEM containing 5 mg/ml <span class="yellow">human</span> serum albumin alone or 4000 units β-glucuronidase with or without 10 mM M6P for 2 hours at 37°C. Following the incubation, the cells were washed 5 times with ice-cold PBS and subjected to enzyme activity assay as described above.<br><br>Cell proliferation assay (MTT assay and cell counts)
Cardiac myocytes were grown in culture plates (tissue culture grade, 12 wells, flat bottom) in a final volume of 1 ml serum-containing culture medium per well, in a humidified atmosphere (37°C and 5% C02) for 3 days. After infection with Ad-GFP/IGF2R-Rz or Ad-GFP, cells were incubated with or without 50 ng/ml IGF-II for 4 days. Following supplementation with IGF-II, 100 μl MTT labeling reagent (Roche) were added to each well and cells were incubated for 4 hours, followed by addition of 1 ml solubilization solution into each well. The plate was placed in an incubator at 37°C overnight. Spectrophotometrical absorbency of the samples was measured using an UV-visible Recording Spectrophotometer with wavelength of 550–690 nm. In addition, the total number of viable cells in each treatment was counted by trypan blue exclusion method using a hemocytometer.<br><br>Induction and analysis of cell apoptosis
Cells were infected with Ad-GFP or Ad-GFP/IGF2R-Rz. Seventy-two hours post infection, cells were treated with TNF (0.1 ng/ml) for 24 hrs or subjected to hypoxia. For induction of apoptosis by hypoxia, cell culture medium was changed to serum-free F-10 saturated with 95% N2/5% CO2 and cells were placed in a 37°C airtight box saturated with 95% N2/5% CO2 for 24 hrs. For normoxic controls, culture medium was changed to F-10/5%F BS/10% HS and cells were placed in a 37°C/5% CO2 incubator for 24 hrs before analysis.
Apoptotic cells were identified by Hoechst staining using the Vybrant™ Apoptosis Kit #5 (Molecular Probes) according to the manufacturer's protocol. In addition, after infection with Ad-GFP or Ad-GFP/IGF2R-Rz and challenge with either TNF or hypoxia, cell viability was assessed using the MTT assay Kit (Roche Molecular Biochemicals) and cell apoptosis was determined using the Cell Death Detection ELISA Kit assay (Roche Molecular Biochemicals) according to the manufacturer's protocol.<br><br>Statistical analysis
Students' t-test was used to evaluate the difference between two values. Each experiment was repeated at least three times. Statistical significance was accepted at the level of p < 0.05.<br><br>
List of abbreviations used
Ad-GFP, <span class="yellow">adenovirus</span> carrying GFP gene; Ad-GFP/IGF2R-Rz, <span class="yellow">adenovirus</span> carrying both the ribozyme against M6P/IGF2R and the GFP gene; GFP, green fluorescent protein; IGF-II, insulin-like growth factor II; M6P/IGF2R, mannose 6-phosphate/insulin-like growth factor II receptor; Rz, ribozyme.<br><br>Authors' contributions
ZC carried out construction of the ribozyme, production of the viruses, cellular experiments, biochemical assays and data analysis.
YG carried out the RPA assay and participated in the molecular biological studies.
JXK conceived of the study, participated in its design and coordination, and drafted the manuscript.<br><br>
<h3>pmcA1779441</h3>Identification of a <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts
Abstract
Increased bone resorption mediated by osteoclasts causes various diseases such as osteoporosis and bone erosion in rheumatoid arthritis (RA). Osteoclasts are derived from the monocyte/macrophage lineage, but the precise origin remains unclear. In the present study, we show that the purified CD16- <span class="yellow">human</span> peripheral blood monocyte subset, but not the CD16+ monocyte subset, differentiates into osteoclast by stimulation with receptor activator of NF-κB ligand (RANKL) in combination with macrophage colony-stimulating factor (M-CSF). Integrin-β3 mRNA and the integrin-αvβ3 heterodimer were only expressed on CD16- monocytes, when they were stimulated with RANKL + M-CSF. Downregulation of β3-subunit expression by small interfering RNA targeting β3 abrogated osteoclastogenesis from the CD16- monocyte subset. In contrast, the CD16+ monocyte subset expressed larger amounts of tumor necrosis factor alpha and IL-6 than the CD16- subset, which was further enhanced by RANKL stimulation. Examination of RA synovial tissue showed accumulation of both CD16+ and CD16- macrophages. Our results suggest that peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct responses to RANKL. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for osteoclastogenesis. Blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Introduction
Rheumatoid arthritis (RA) is an autoimmune disease characterized by chronic inflammation and proliferation of the synovium in multiple joints. A large number of inflammatory cells, including T cells, B cells, macrophages and dendritic cells, accumulate in the affected synovium, and these inflammatory cells, together with fibroblast-like synoviocytes, express various cytokines, such as tumor necrosis factor alpha (TNFα), IL-6 and receptor activator of NF-κB ligand (RANKL), which are known to induce differentiation and activation of osteoclasts. The inflammatory synovial tissue, known as pannus, invades the articular bone and causes focal bone erosion, which is the hallmark of RA. Histopathologically, osteoclasts are present at the interface of the pannus and bone. Interestingly, the deletion of RANKL or c-Fos gene, which is important for osteoclastogenesis, results in minimal bone destruction in <span class="yellow">mouse</span> models of arthritis [1,2]. Furthermore, other studies indicated that inhibition of osteoclastogenesis by osteoprotegerin, a decoy receptor for RANKL, limits bone destruction in experimental models of arthritis. These studies suggest that osteoclasts are involved in focal bone erosion in RA [3].
Osteoclasts are derived from the monocyte/macrophage lineage. It is reported that osteoclast precursors reside in <span class="yellow">human</span> peripheral blood monocytes [4,5]. A marked increase of the circulating osteoclast precursors was demonstrated in <span class="yellow">patients</span> with erosive psoriatic arthritis as well as in arthritic TNFα transgenic <span class="yellow">mice</span> [6,7]. It was also shown that peripheral monocytes differentiate into osteoclasts when seeded on RANKL/osteoclast differentiation factor-producing RA synovial fibroblasts [8]. In addition, RA synovial macrophages thought to originate from peripheral blood monocytes were shown to differentiate into osteoclasts [9,10]. Monocytes are therefore involved not only in synovial inflammation, but also in bone remodeling as potential precursors for synovial macrophages and osteoclasts.
<span class="yellow">Human</span> peripheral blood monocytes consist of two major subsets, CD16+ and CD16-, comprising 5–10% and 90–95% of the monocytes, respectively. These two subsets exhibit different chemotaxis activities and potential of cytokine production [11,12]. Moreover, activation of the Toll-like receptor induces distinct subsets, CD1b+ dendritic cells and DC-SIGN+ (dendritic cell-specific C-type lectin ICAM-3-grabbing nonintegrin) macrophages from CD16+ and CD16- monocytes, respectively [13]. It has not been revealed, however, which monocyte subset develops into osteoclasts.
In the present study, we determined the <span class="yellow">human</span> peripheral blood monocyte subset that differentiates into osteoclasts, and revealed that each subset exhibits a different response for osteoclastogenic stimuli.<br><br>Materials and methods
Purification of peripheral blood monocytes
Peripheral blood monocytes from healthy donors were collected using Ficoll-Conray (Imuuno-Biological Laboratories, Gunma, Japan) gradient centrifugation. Negative selection of monocytes was performed using MACS microbeads (Miltenyi Biotec, Auburn, CA, USA) according to the protocol supplied by the manufacturer.
The purified monocytes were separated into two subsets, CD16+ and CD16- monocytes, using CD16 MicroBeads (Miltenyi Biotec). Flow cytometry analysis using FITC-conjugated <span class="yellow">mouse</span> anti-CD14 mAb (MY4; Bechman Coulter, Fullerton, CA, USA) and phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8; BD Biosciences, San Jose, CA, USA) showed that the purities of the CD16+ and CD16- monocytes were more than 90% and 92%, respectively.
For the other experiment, monocytes were purified using CD14 MicroBeads (Miltenyi Biotec), and then stained either with FITC-conjugated <span class="yellow">mouse</span> anti-CD33 mAb (MY9; Bechman Coulter) or phycoerythin-conjugated <span class="yellow">mouse</span> anti-CD16 mAb (3G8). Cell sorting of the stained cells was performed using a FACS Vantage cytometer (BD Biosciences) or a MoFlo cell sorter (Dako, Glostrup, Denmark).<br><br>Osteoclast differentiation
Purified CD16+ and CD16- monocytes (5 × 104 cells/well) were incubated in 96-well plates in αMEM (Sigma, St Louis, MO, USA) with heat-inactivated 10% fetal <span class="yellow">bovine</span> serum (FBS) (Sigma) or with Ultra-Low IgG FBS (IgG < 5 μg/ml; Invitrogen, Carlsbad, CA, USA), and where indicated with M-CSF + RANKL (Peprotech, Rocky Hill, NJ, USA).
For the other experiments, varied numbers of CD16+ monocytes (1 × 103, 2.5 × 103, 5 × 103) were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured in 96-well plates in αMEM with heat-inactivated 10% FBS. The medium was replaced with fresh medium 3 days later, and after incubation for 7 days the cells were stained for tartrate-resistant acid phosphatase (TRAP) expression using a commercial kit (Hokudo, Sapporo, Japan). The number of TRAP-positive multinucleated cells (MNC) in three randomly selected fields examined at 100× magnification or the total number of TRAP-positive MNC per well was counted under light microscopy.<br><br>Resorption assay
Monocytes were seeded onto plates coated with calcium phosphate thin films (Biocoat Osteologic; BD Biosciences) and were incubated with RANKL (40 ng/ml) + M-CSF (25 ng/ml) for 7 days. The cells were then lysed in bleach solution (6% NaOCl, 5.2% NaCl). The resorption lacunae were examined under light microscopy.<br><br>Enzyme-linked immunosorbent assay
Purified monocytes were cultured in 96-well plates where indicated either with RANKL or M-CSF for 24 hours. Concentrations of TNFα and IL-6 in the culture supernatant were measured with an ELISA kit (BioSourse International, Camarillo, CA, USA). For experiments of matrix metalloproteinase (MMP)-9 and TRAP-5b, culture supernatants were collected on day 7 and the concentrations of these enzymes were measured using an MMP-9 ELISA kit (Amersham Biosciences, Piscataway, NJ, USA) or a TRAP-5b ELISA kit (Suomen, Turku, Finland).<br><br>Reverse transcriptase-polymerase chain reaction
Monocytes (1 × 106 cells/well) were cultured in six-well plates with M-CSF alone or with M-CSF + RANKL for 3 days. Total RNA was extracted using RNeasy Micro (Qiagen, Valencia, CA, USA). The RNA was then treated with DNase I (Qiagen). The oligo(dT)-primed cDNA was synthesized using Superscript II reverse transcriptase (Invitrogen). The amount of cDNA for amplification was adjusted by the amount of RNA measured by an optical density meter and also by β-actin or GAPDH PCR products. One microliter of cDNA was amplified in a 50 μl final volume containing 25 pmol appropriate primer pair, 10 pmol each of the four deoxynucleotide triphosphates, and 5 units FastStart Taq DNA Polymerase (Roche, Manheim, Germany) in a thermal cycler (PTC-200; MJ GeneWorks, Waltham, MA, USA).
The PCR conditions were 25–40 cycles of denaturation (95°C for 30 s), annealing (60–62°C for 1 min) and extension (72°C for 1 min). The sequences of the primers are presented in Table 1. The PCR products were separated by electrophoresis through 2% agarose gel.<br><br>Western immunoblot analysis
Purified monocytes were cultured for 3 days in the presence of 40 ng/ml M-CSF with or without 25 ng/ml RANKL. Cells were lysed in RIPA Lysis buffer (upstate, Lake Placid, NY, USA) containing protease inhibitors (Roche) for 15 min at 4°C. A total of 20 μg protein was boiled in the presence of 6 × sodium dodecyl sulfate sample buffer, and was separated on 7.5% or 10% sodium dodecyl sulfate-polyacrylamide gel (ATTO, Tokyo, Japan). Proteins were then electrotransferred to a polyvinylidene fluoride microporous membrane (Millipore, Billerica, MA, USA) in a semidry system. Membranes were incubated in 10% skim milk prepared in phosphate-buffered saline (PBS) containing 0.1% Tween 20, and were subjected to immunoblotting. Antibodies used were <span class="yellow">goat</span> anti-RANK antibody (Techne Corporation, Minneapolis, MN, USA), <span class="yellow">goat</span> anti-c-fms antibody (R&D systems, Minneapolis, MN, USA), and <span class="yellow">mouse</span> anti-β-actin mAb (AC-15; Sigma). Peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">goat</span> IgG antibody (Dako) or peroxidase-conjugated <span class="yellow">rabbit</span> anti-<span class="yellow">mouse</span> IgG antibody (Dako) was used as the second antibody. The signals were visualized by chemiluminescence reagent (ECL; Amersham Biocsiences, Little Chalfont, UK).<br><br>Cell surface expression of c-fms
The following mAbs were used for analysis of c-fms expression: Alexa 647-conjugated anti-CD14 mAb (UCHM1; Serotec, Oxford, UK), FITC-conjugated anti-CD16 mAb (3G8; Bechman Coulter) and phycoerythin-conjugated anti-c-fms mAb (61708; R&D systems). Alexa 647-conjugated <span class="yellow">mouse</span> IgG2a (Serotec), FITC-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) and phycoerythin-conjugated <span class="yellow">mouse</span> IgG1 (Bechman Coulter) were used as isotype controls. Peripheral blood monocytes (1 × 105 cells) were incubated with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and were then stained with three fluorochrome-labeled mAbs for 45 minutes on ice. The stained cells were analyzed with a FACS Calibur (BD Biosciences).<br><br>Immunofluorescent staining
Monocytes (8 × 104 cells/well) were allowed to adhere on 96-well plates overnight or were cultured with M-CSF and RANKL for 2–4 days. The cells were fixed in acetone and then stained with anti-αvβ3 mAb (LM609; Chemicon, Temecula, CA, USA) or <span class="yellow">mouse</span> IgG1 (11711; R&D Systems) as an isotype-matched control. Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes, Eugene, OR, USA) was used as the second antibody. TOTO-3 (Molecular Probes) was used for nuclear staining.<br><br>Flow cytometric analysis of p38 MAPK and ERK1/2 phosphorylation
Purified monocytes were cultured in the presence of 25 ng/ml M-CSF for 3 days, and were either left unstimulated or were stimulated with 40 ng/ml RANKL at 37°C. Stimulations were stopped by adding an equal volume of PhosFlow Fix Buffer I solution (BD Biosciences) to the cell culture. After incubation for 10 minutes at 37°C, the cells were permeabilized by washing twice at room temperature in PhosFlow Perm/Wash Buffer I (BD Biosciences). A total of 1 × 105 cells was then Fc blocked with 1 μg <span class="yellow">human</span> IgG for 15 minutes, and was stained with Alexa Fluor 647-conjugated mAb either to phospho-p38 MAPK (T180/Y182) or to phospho-ERK1/2 (T202/Y204) (BD Biosciences) for 30 minutes at room temperature. Alexa Fluor 647-conjugated <span class="yellow">mouse</span> IgG1 (BD Biosciences) was used as an isotype control. The cells were washed in PhosFlow Perm/Wash Buffer I, and were analyzed by flow cytometry, as already described.<br><br>RNA interference
RNA oligonucleotides (iGENE, Tsukuba, Japan) were designed based on the algorithm that incorporates single nucleotide polymorphism and homology screening to ensure a target-specific RNA interference effect. The following sense and antisense oligonucleotides were used: integrin β3, 5'-GCU UCA AUG AGG AAG UGA AGA AGC A-AG and 3'-UA-CGA AGU UAC UCC UUC ACU UCU UCG U; randomized control, 5'-CGA UUC GCU AGA CCG GCU UCA UUG C-AG and 3'-UA-GCU AAG CGA UCU GGC CGA AGU AAC G; and lamin, 5'-GAG GAA CUG GAC UUC CAG AAG AAC A-AG and 3'-UA-CUC CUU GAC CUG AAG GUC UUC UUG U.
CD16- monocytes (8 × 104 cells/well) were incubated in 96-well plates in optimem (Invitrogen). After 1 hour, siRNAs were transfected into the cells using oligofectamine (Qiagen) based on the method recommended by the manufacturer. After 2 hours, the cells were washed once with PBS, followed by the addition of αMEM supplemented with 10% FBS, M-CSF and RANKL. After a 2-day incubation, the β3 mRNA expression was analyzed by RT-PCR with different PCR cycles, as described earlier. Immunofluorescent staining for the αvβ3 heterodimer was also performed as described above, and numbers of αvβ3-positive cells were counted in randomly selected three fields at 100× magnification. Seven days after the transfection of siRNAs, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Inhibition of osteoclastogenesis with cyclic RGDfV peptide
CD16- monocytes were incubated in 96-well plates with M-CSF + RANKL for 2 days. A medium containing either cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) (Calbiochem, San Diego, CA, USA) or dimethyl sulfoxide was then added. After incubation for a further 5 days, the number of TRAP-positive MNC in five fields examined at 100× magnification was counted under light microscopy.<br><br>Immunohistochemistry
Synovial tissue samples were obtained during total knee joint replacement surgery from four RA <span class="yellow">patients</span>. Signed consent forms were obtained before the operation. The experimental protocol was approved by the ethics committee of the Tokyo Medical and Dental University. RA was diagnosed according to the American College of Rheumatology criteria [14].
Double immunofluorescent staining for CD68 and CD16 antigens was conducted on optimal cutting temperature-embedded sections of frozen synovial samples. Eight-micrometer-thick cryostat sections of RA synovium were fixed in acetone for 3 minutes and were then rehydrated in PBS for 5 minutes. The samples were incubated in 5 μg/ml proteinase K (Roche), 50 mM ethylenediamine tetraacetic acid, 100 mM Tris–HCl, pH 8.0, for 15 minutes at room temperature followed by a wash in PBS. The samples were then blocked with 10% <span class="yellow">goat</span> serum in PBS for 60 minutes at room temperature, and were incubated with anti-CD16 mAb (3G8; Immunotech, Marseille, France) or <span class="yellow">mouse</span> IgG1 (11711) as an isotype-matched control in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then washed three times in PBS, for 5 minutes each, and incubated with Alexa fluor546-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG1 antibody (Molecular Probes) in 1% <span class="yellow">bovine</span> serum albumin/PBS for 60 minutes at room temperature. The samples were then sequentially stained for CD68 antigen in a manner similar to that used for CD16 staining. The samples were stained with anti-CD68 mAb (PGM1; Immunotech) or <span class="yellow">mouse</span> IgG3 (6A3; MBL, Nagoya, Japan) followed by labeling with Alexa fluor488-conjugated <span class="yellow">goat</span> anti-<span class="yellow">mouse</span> IgG3 antibody (Molecular Probes). The samples were examined by confocal laser scanning microscope (Olympus, Tokyo, Japan).<br><br>Statistical analysis
Data are expressed as the mean ± standard error of the mean. A nonpaired Student's t test was used for comparison, using the StatView program (Abacus Concepts, Berkeley, CA, USA). P < 0.05 was considered statistically significant.<br><br>
Results
Induction of osteoclasts from CD16- peripheral blood monocytes
To identify the monocyte subset that differentiates into osteoclasts, we examined osteoclast formation from CD16+ and CD16- <span class="yellow">human</span> peripheral blood monocytes. The monocyte subsets were purified using magnetic beads. Incubation with M-CSF alone did not induce osteoclast formation from either subset (Figure 1a). Culture with M-CSF + RANKL induced a significant number of TRAP-positive MNC from the CD16- subset, whereas only few CD16+ monocytes differentiated into TRAP-positive MNC (Figure 1a,b). We then assessed the bone resorptive ability by culturing cells on calcium phosphate-coated plates with M-CSF + RANKL. Resorption lacunae were detected only in the CD16- subset (Figure 1c), indicating the TRAP-positive CD16--derived MNC possessed the osteoclast phenotype.
Similar results were obtained using purified monocytes by FACS sorting (purities: CD16+, 96%; CD16-, 97%). The number of TRAP-positive MNC induced were 36 ± 3 cells/well and 348 ± 13 cells/well from CD16+ and CD16- monocytes, respectively. To exclude the possibility that the anti-CD16 antibody used for isolation of CD16+ monocytes inhibits osteoclast formation, we separated the two subsets, CD33low monocytes and CD33high monocytes, using anti-CD33 mAb and a fluorescent cell sorter, since it was reported that CD33low monocytes correspond to CD16+, and that CD33high correspond to CD16- monocytes [15]. On average, the CD33low population contained CD16- (10.2%)/CD16+ (89.8%) monocytes, and the CD33high population contained CD16- (86.3%)/CD16+ (13.7%) monocytes.
Culture with M-CSF + RANKL induced TRAP-positive MNC from CD33high monocytes, whereas no or few CD33low monocytes differentiated into TRAP-positive MNC (CD33low vs CD33high, 2 ± 1 vs 192 ± 71 cells/well; n = 3). TRAP-5b and MMP-9 in the culture supernatants, both of which are known to be produced by osteoclasts, were measured by ELISA. The concentrations of both enzymes were significantly higher in the culture supernatant of CD16- monocytes than in that of CD16+ monocytes (Figure 1d). These results suggest that the CD16- peripheral blood monocyte subset, but not the CD16+ subset, differentiate into osteoclasts by incubation with RANKL + M-CSF.<br><br>CD16+ monocytes do not affect the osteoclastogenesis from CD16- monocytes
To examine whether CD16+ monocytes affect osteoclastogenesis from CD16- monocytes, varied numbers of CD16+ monocytes were mixed with CD16- monocytes (5 × 104 cells/well), and were cultured for 7 days in the presence of M-CSF + RANKL. The number of TRAP-positive MNC was not altered by the presence of CD16+ monocytes (Figure 2). The results indicated that CD16+ monocytes did not hamper or enhance the osteoclastogenesis from CD16- monocytes.<br><br>Differences in cytokine production by RANKL-stimulated or M-CSF-stimulated CD16+ and CD16- monocytes
To compare the biological response of CD16+ and CD16- subsets with either RANKL or M-CSF stimulation, we measured the amount of TNFα and IL-6 production after exposure of cells to various concentrations of RANKL or M-CSF with an ELISA. Without RANKL the CD16+ subset produced a significant amount of TNFα and IL-6, whereas the CD16- subset produced undetectable levels (Figure 3a). RANKL stimulation increased TNFα production from both subsets in a dose-dependent manner, although the CD16+ subset produced more TNFα than did the CD16- subset. RANKL stimulation also enhanced IL-6 production from the CD16+ subset, but not from the CD16- subset. M-CSF stimulation increased TNFα and IL-6 production from both subsets, although the CD16+ subset produced more than the CD16- subset (Figure 3b).
These results suggest that CD16+ monocytes also respond both to RANKL and M-CSF stimulation, although such stimulation does not result in differentiation into osteoclasts. CD16+ monocytes were also noted to express higher amounts of inflammatory cytokines compared with CD16- monocytes with or without RANKL or M-CSF stimulation.<br><br>Comparison of expression levels of molecules involved in osteoclastogenesis between CD16+ and CD16- monocytes
Diverse molecules are involved in RANKL/RANK and its costimulatory signal transduction pathways [16]. The different response to RANKL + M-CSF stimulation between the CD16+ monocyte subset and the CD16- monocytes subset might be explained by the expression profiles of these molecules. We therefore examined the mRNA levels of the following molecules: receptor activator of NF-κB (RANK), the receptor for RANKL; c-fms, the receptor for M-CSF; tumor necrosis factor receptor-associated factor 6 (TRAF-6), the adaptor protein for RANK; c-Fos and nuclear factor of activated T cells c1 (NFATc1), transcription factors that are essential for osteoclastogenesis; DNAX-activation protein 12 (DAP12) and Fc receptor γ chain (FcRγ), adaptor proteins known to deliver costimulatory signals in RANKL-induced osteoclastogenesis; signal regulatory protein β1 (SIRP-β1), triggering receptor expressed on myeloid cells 2 (TREM-2) and osteoclast-associated receptor (OSCAR), transmembrane receptors that associate with either DAP12 or FcRγ; and αv and β3, integrins known to be expressed as the αvβ3 heterodimer on osteoclasts.
The mRNA levels of RANK, c-fms, TRAF-6, DAP12 and SIRP-β1 under the baseline condition (no stimulation) varied between the donors; however, we did not find consistent differences in the mRNA levels of these molecules between the CD16+ monocyte subset and the CD16- monocyte subset among three to six donors (Figure 4a). The mRNA levels of other molecules, apart from integrin β3, were similar between the two subsets under the no-stimulation condition. Although the mRNA levels of RANK, c-fms, DAP12, FcRγ, TREM-2 and OSCAR increased in response to M-CSF alone or M-CSF + RANKL in both subsets, the expression levels were not significantly different between the two subsets. Expressions of TRAF-6, c-Fos and SIRP-β1 mRNA did not change following stimulation with M-CSF + RANKL. Of note, the expression of NFATc1 mRNA was enhanced by M-CSF + RANKL treatment only in the CD16- subset. Furthermore, expression of integrin αv in both subsets was enhanced by M-CSF with or without RANKL; however, the expression level was greater in the CD16- subset. It was noted that integrin-β3 mRNA was detected only in the CD16- subset and was increased by M-CSF + RANKL stimulation, but not by M-CSF alone. The protein expression of RANK under the baseline condition was weakly detected in both subsets, and the levels were varied between donors by western immunoblotting.
The protein expression of c-fms was weakly detected in unstimulated CD16+ monocytes, but not in CD16- monocytes (Figure 4b). Flow cytometry analysis of c-fms in fresh monocytes, however, showed that both subsets express the molecule on the cell surface (Figure 4c). Expressions of both RANK and c-fms were upregulated by M-CSF alone and by M-CSF + RANKL, and we did not find consistent differences in the protein levels of these molecules between the two monocyte subsets. The profiles of expression levels of molecules involved in RANKL/RANK and its costimulatory pathways are similar between the two subsets, except for NFATc1, integrin αv and integrin β3. We therefore assumed that the distinct induction of NFATc1, integrin αv and integrin β3 in response to RANKL stimulation among the two monocyte subsets might explain the differences in their abilities to differentiate into osteoclasts.<br><br>RANKL stimulation induces αvβ3 expression on CD16- monocytes
The integrin-β3 subunit binds to integrin αv only and is expressed as the heterodimeric protein αvβ3 on monocytes and osteoclasts [17]. We examined the expression of αvβ3 on CD16+ and CD16- monocytes by immunofluorescent staining. Neither unstimulated nor M-CSF-stimulated monocyte subsets expressed αvβ3 (Figure 4d and data not shown). After 48 and 72 hours of treatment with M-CSF + RANKL, αvβ3-positive mononuclear cells were observed in CD16- monocyte cultures but not in CD16+ monocyte cultures. At 96 hours, both αvβ3-positive mononuclear cells and multinucleated cells were present in the CD16- monocyte culture. The results indicated that αvβ3 was selectively expressed on CD16- monocytes in the presence of M-CSF + RANKL, and the expression was revealed before the cells differentiate into typical multinucleated osteoclasts.<br><br>RANKL activates ERK and p38 kinases only in CD16- monocytes
Since ERK and p38 MAPK are essential in RANKL-induced osteoclastogenesis [18-20], we next examined whether these kinases were activated differently in CD16+ monocytes and in CD16- monocytes. Purified monocytes were precultured with 25 ng/ml M-CSF for 3 days to enhance RANK expression, and were then treated with RANKL. The RANKL treatment induced phosphorylation of both ERK and p38 MAPK in CD16- monocytes at 5 minutes postexposure, although the p38 MAPK phosphorylation was weak. Both phosphorylations declined to a basal level within 20 minutes (Figure 5). In contrast, ERK and p38 MAPK were not detectably phosphorylated in CD16+ monocytes with RANKL.<br><br>siRNA targeting integrin β3 inhibits osteoclastogenesis from CD16- monocytes
The integrin-β3 cytoplasmic domain is essential for activation of intracellular signals from αvβ3 heterodimers [17]. We therefore examined the involvement of αvβ3 in RANKL + M-CSF-induced osteoclastogenesis in <span class="yellow">human</span> CD16- monocytes using siRNA targeting the integrin-β3 subunit. The integrin-β3 siRNA or control randomized siRNA were transfected into CD16- monocytes. At 48 hours post-transfection, we determined the integrin-β3 mRNA level and αvβ3 heterodimer protein expression. The integrin-β3 mRNA level was reduced in the integrin-β3 siRNA-transfected monocytes compared with control siRNA-transfected monocytes (Figure 6a). The αvβ3 heterodimer expression was evaluated by immunofluorescent staining. The number of αvβ3-positive cells was significantly decreased in integrin-β3 siRNA-transfected monocytes compared with that in control siRNA (Figure 6b).
After 7 days of incubation, the number of TRAP-positive MNC was counted. Transfection with integrin-β3 siRNA significantly reduced the number of TRAP-positive MNC in a dose-dependent manner compared with control siRNA transfection (Figure 6c). In addition, the use of siRNA directed toward a different site of integrin-β3 mRNA also inhibited osteoclast formation from CD16- monocytes (data not shown). On the other hand, siRNA that targeted lamin, which was used as a negative control, did not inhibit the induction of osteoclasts (data not shown). These results indicate the importance of integrin β3 in RANKL-induced osteoclast formation from CD16- peripheral blood monocytes.<br><br>Cyclic RGDfV peptide inhibits the osteoclastogenesis from CD16- monocytes
Integrin αvβ3 recognizes a common tripeptide sequence, RGD (Arg-Gly-Asp), which is present in bone matrix proteins such as vitronectin and fibronectin [21]. Cyclic RGDfV peptide (Arg-Gly-Asp-D-Phe-Val) inhibits binding of the RGD-containing molecules to αvβ3 [22]. To investigate the role of ligand binding to the αvβ3 heterodimer in the osteoclastogenesis, we examined whether cyclic RGDfV peptide inhibits the formation of osteoclasts. Cyclic RGDfV peptide significantly reduced the number of TRAP-positive MNC in a dose-dependent manner (Figure 6d). The results imply possible involvement of ligand bindings to αvβ3 in the osteoclastogenesis.<br><br>Knockdown of integrin β3 did not affect the expression of NFATc1 mRNA
In the next step, we determined whether integrin-β3-siRNA-induced inhibition of the osteoclastogenesis reflects downregulation of NFATc1, which is a key transcription factor in osteoclastogenesis [23]. For this purpose, we compared NFATc1 mRNA levels between integrin β3 and control siRNA-transfected monocytes. Interestingly, integrin-β3 knockdown did not alter the NFATc1 mRNA level (Figure 7), suggesting that signal transduction mediated by integrin β3 does not affect the expression of NFATc1.<br><br>Detection of CD16+ and CD16- macrophages in synovium of RA <span class="yellow">patients</span>
RA synovial macrophages are derived from peripheral blood monocytes, and their recruitment into the synovium is facilitated by various adhesion molecules and chemokines [24]. To analyze CD16 expression on synovial macrophages, RA synovial tissues were double-stained for CD16 and a macrophage marker, CD68. CD16-/CD68+ macrophages were widespread in the synovium. Although less frequent, CD16+/CD68+ macrophages were also observed both in the synovial intima and subintima (Figure 8). The presence of two subsets of macrophages, CD16+ and CD16-, in RA synovium indicates that both CD16+ and CD16- peripheral blood monocytes are recruited into the synovium.<br><br>
Discussion
<span class="yellow">Human</span> peripheral blood monocytes are a heterogeneous population, and they are divided into two subsets based on the expression of CD16. The CD16+ and CD16- monocyte subsets show functional differences in migration, cytokine production and differentiation into macrophages or dendritic cells [11-13,15]. We focused on the heterogeneity of the monocytes, and the primary question addressed in this study was which monocyte subset could be the source of osteoclasts. The results demonstrated that CD16- peripheral blood monocytes, but not CD16+ monocytes, differentiated in vitro into osteoclasts by treatment with RANKL + M-CSF.
To investigate the molecular mechanisms of the different response to RANKL and the differentiation into osteoclasts between CD16+ and CD16- monocytes, we examined the expression of molecules known to be involved in osteoclastogenesis. The expression profiles of integrin αv, integrin β3 and NFATc1 were different between the two subsets. Integrin αvβ3 heterodimer was expressed only on RANKL and M-CSF-stimulated CD16- monocytes. It is known that αvβ3 expressed on osteoclasts is important in bone resorption as well as in attachment of osteoclasts to the bone matrix [25].
It was recently reported that bone marrow macrophages of integrin-β3-deficient <span class="yellow">mice</span> could not differentiate into mature osteoclasts in vitro, suggesting that αvβ3 is involved not only in activation, but also in differentiation, of osteoclasts in <span class="yellow">mice</span> [26,27]. The authors also showed that αvβ3 and c-fms share a common intracellular signaling pathway, including the activation of ERK and the induction of c-Fos [27], both of which are essential for osteoclastogenesis [28,29]. In addition, it was reported that echistatin, an αvβ3 antagonist, inhibited osteoclast formation of <span class="yellow">mouse</span> bone marrow cells [30].
In accordance with these reports, our data showed that knockdown of integrin-β3 expression resulted in downregulation of the αvβ3 heterodimer, and abrogated osteoclastogenesis from <span class="yellow">human</span> peripheral blood CD16- monocytes. We also showed that blocking of adhesive ligands to bind to αvβ3 by RGDfV peptide inhibited osteoclast formation from CD16- monocytes. Taken together, the process of ligand binding to αvβ3 may be involved in the osteoclastogenesis. Blockade of αvβ3 could therefore be a therapeutically beneficial approach to modulate osteoclastogenesis. Indeed, integrin αvβ3 antagonists effectively treated osteoporosis in <span class="yellow">mice</span>, <span class="yellow">rats</span> and <span class="yellow">humans</span>, and protected bone destruction in <span class="red">rat</span> adjuvant-induced arthritis in vivo [31-34]. Of note, it is reported that <span class="yellow">patients</span> with Iraqi-Jewish-type Glanzmann thrombasthenia who are deficient in integrin β3 do not develop osteopetrosis because of the upregulation of α2β1 expression on osteoclasts, although the bone-resorptive ability of the osteoclasts was decreased in vitro [35]. The function of αvβ3 in vivo in osteoclast formation and resorptive function could therefore be partially compensated by other integrins.
Although all the multinucleated osteoclasts expressed αvβ3 (Figure 4d) [36], a small number of M-CSF + RANKL-stimulated mononuclear CD16- monocytes expressed αvβ3 (Figure 4d). Multinucleated osteoclasts are formed by fusion of osteoclast precursor cells [37]. It was reported that αvβ3 is involved in the migration of osteoclast precursors [30]. The αvβ3-positive cells could therefore be forced to migrate by the ligands and may fuse with closed αvβ3-negative cells. Alternatively, only αvβ3-positive cells may be fused with each other.
It is possible to consider that signaling from CD16 by anti-CD16 mAb-coated magnetic beads, which were used for the cell separation, or by IgG contained in FBS might inhibit osteoclastogenesis from CD16+ monocytes. We therefore separated the two subsets using anti-CD33 mAb and a fluorescent cell sorter, and stimulated the cells with M-CSF + RANKL. The results showed that CD33low monocytes, which correspond to CD16+ monocytes, still could not differentiate into osteoclasts. CD16 is a heterodimer consisting of FcγIIIa and Fcγ, and has low affinity for the Fc region of IgG. Aggregation of CD16 by immune complexes leads to transmission of activating signals via the immunoreceptor tyrosine-based activation motif in the γ chain [38]. We also assessed osteoclastogenesis from the two monocyte subsets using IgG-depleted <span class="yellow">bovine</span> serum. Even in the IgG-free medium, CD16- monocytes but not CD16+ monocytes differentiated into osteoclasts (data not shown). We could therefore exclude the possibility that signal transduction through CD16 inhibits osteoclastogenesis from CD16+ monocytes.
NFATc1 is a key transcription factor in osteoclastogenesis [16]. In the present study, stimulation with M-CSF + RANKL increased the NFATc1 mRNA expression in the CD16- subset only, similar to integrin αv and integrin β3. The differences in NFATc1 induction might therefore also explain the difference in osteoclastogenesis between the two monocyte subsets. It is of interest that knockdown of integrin β3 did not lower the mRNA level of NFATc1. This result supports the notion that NFATc1 is located upstream of integrin-β3 expression [39]. It is also possible that parallel activation of two signaling pathways mediated by integrin β3 and NFATc1 contributes to osteoclastogenesis independently or cooperatively. Further studies are needed to determine the mechanisms of integrin β3 involvement in RANKL/RANK-mediated osteoclast differentiation.
It has been demonstrated that MAPK families, ERK and p38 MAPK, were activated by RANKL-induced intracellular signalings in osteoclasts and osteoclast precursors [18,19]. In addition, these kinases are involved in the differentiation of osteoclasts [20]. We showed that RANKL stimulation induced phosphorylation of ERK and p38 MAPK only in CD16- monocytes. It is suggested that differential activation of these kinases may partially explain the distinct properties of the two monocyte subsets upon RANKL stimulation.
Our results showed that CD16+ monocytes produce higher levels of inflammatory cytokines including TNFα and IL-6 compared with CD16- monocytes. These results are consistent with the previous report showing that CD16+ monocytes produced larger amounts of TNFα upon lipopolysaccharide or lipopeptide stimulation than did CD16- monocytes [40]. Interestingly, we showed that stimulation either with RANKL or M-CSF upregulated the TNFα and IL-6 production by CD16+ monocytes. A marked increase of CD16+ monocytes in peripheral blood is reported in inflammatory diseases, such as infection, malignancy, Kawasaki disease and RA [41-44]. Taken together, CD16+ monocytes may be an important source of inflammatory cytokines.
In <span class="yellow">mice</span>, peripheral blood Ly-6Chigh monocytes, which are thought to correspond to <span class="yellow">human</span> CD16- monocytes, increase in inflammatory conditions, and these cells are recruited into sites of inflammation [45]. In contrast, Ly-6Clow monocytes, which are thought to correspond to <span class="yellow">human</span> CD16+, migrate into noninflamed tissues [12]. These data on <span class="yellow">mouse</span> monocytes seem to be in contrast to the data on <span class="yellow">human</span> monocytes, which show expansion of CD16+ monocytes in inflammatory conditions where they produce larger amounts of inflammatory cytokines. At present, it is not clear whether <span class="yellow">mouse</span> monocyte subsets, Ly-6Clow/Ly-6Chigh, represent <span class="yellow">human</span> monocyte subsets, CD16+/CD16- monocytes, and whether the biologic functions of <span class="yellow">mouse</span> monocytes are analogous to those of <span class="yellow">human</span> monocytes.
In <span class="yellow">mice</span>, blood monocytes newly released from the bone marrow are exclusively Ly-6Chigh and the level of Ly-6C is downregulated while in circulation [45]. It is thus suggested that in <span class="yellow">mice</span> the two monocyte subsets differing in Ly-6C expression represent different stages in the maturation pathway. In the <span class="yellow">human</span>, transition from CD16- monocytes to CD16+ monocytes is observed upon culture with IL-10, M-CSF and transforming growth factor beta in vitro [42,46]. Similar to <span class="yellow">mouse</span> monocytes, therefore, <span class="yellow">human</span> peripheral blood CD16- monocytes may also maturate into CD16+ monocytes.
It is reported that a significant number of RA synovial cells in the intima express CD16, suggesting that CD16+ cells are synovial macrophages [47]. We confirmed that both CD16+ and CD16- macrophages accumulate in the RA synovium by double-color immunohistochemical staining for CD68 and CD16. A number of chemokines are abundantly expressed in the RA synovium [24,48]. Among these cytokines, MCP-1, MIP-1α, SDF-1, RANTES and fractalkine can induce migration of CD16- monocytes in vitro ([11,12] and unpublished data). On the other hand, migration of CD16+ monocytes is induced only by fractalkine. These chemokines therefore seem to play an important role in recruitment of CD16+ and CD16- monocytes from the circulating pool into the RA synovium.
The osteoclast inducers are also produced in the RA synovium. RANKL is expressed by synovial fibroblasts and activated T cells [49-51], while M-CSF is expressed on RA synovial macrophages and fibroblasts [52,53].
TNFα and IL-6, which are mainly expressed on RA synovial macrophages and fibroblasts, respectively, could also enhance osteoclast differentiation [54]. Collectively, it is probable that the recruited CD16- monocytes/macrophages differentiate into osteoclasts in the RA synovium, and contribute to bone destruction. On the other hand, CD16+ monocytes/macrophages might also be involved in RA pathogenesis by producing inflammatory cytokines including TNFα and IL-6. Since TNFα and IL-6 enhance osteoclast formation [54,55], CD16+ monocytes/macrophages may also contribute to osteoclastogenesis in RA synovium.<br><br>Conclusion
We have shown that <span class="yellow">human</span> peripheral blood monocytes consist of two functionally heterogeneous subsets with distinct response to osteoclastogenic stimuli. Osteoclasts seem to originate from CD16- monocytes, and integrin β3 is necessary for the osteoclastogenesis. The blockade of accumulation and activation of CD16- monocytes could therefore be a beneficial approach as an anti-bone resorptive therapy, especially for RA.<br><br>Abbreviations
DAP = DNAX-activation protein; ELISA = enzyme-linked immunosorbent assay; FBS = fetal <span class="yellow">bovine</span> serum; FcRγ = Fc receptor γ chain; IL = interleukin; FITC = fluorescein isothiocianate; mAb, monoclonal antibody; M-CSF = macrophage colony-stimulating factor; MEM = modified Eagle's medium; MMP = matrix metalloproteinase; MNC = multinucleated cells; NF = nuclear factor; OSCAR = osteoclast-associated receptor; PBS = phosphate-buffered saline; PCR = polymerase chain reaction; RA = rheumatoid arthritis; RANK = receptor activator of NF-κB; RANKL = receptor activator of NF-κB ligand; RT = reverse transcriptase; siRNA = small interfering RNA; SIRP-β1 = signal regulatory protein-β1; TNFα = tumor necrosis factor alpha; TRAF = tumor necrosis factor receptor-associated factor; TRAP = tartrate-resistant acid phosphatase; TREM = triggering receptor expressed on myeloid cells.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
YK participated in the design of the study, carried out the experiments and statistical analysis, and drafted the manuscript. KH and KT participated in the design of the study and its coordination. TN and NM conceived of the study, participated in its design and coordination, and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1805739</h3>Embryonic sympathoblasts transiently express TrkB in vivo and proliferate in response to brain-derived neurotrophic factor in vitro
Abstract
Background
Nerve growth factor and neurotrophin-3 are involved in the development of sympathetic neurons; however, whether brain derived neurotrophic factor also plays a role is not known. The purpose of this study was to determine whether BDNF and its receptor, TrkB, are expressed during the development of paravertebral sympathetic ganglia in vivo and to determine the effect of BDNF in vitro.<br><br>Results
As neural crest cells coalesce to form sympathetic ganglia, TrkB-positive cells are seen in both <span class="yellow">chicken</span> and <span class="yellow">mouse</span> embryos. In <span class="yellow">chicken</span> embryos, TrkB-expressing cells first appear at Hamburger-Hamilton Stage (St) 27 and they co-express HNK-1, confirming that they are migrating neural crest cells. The TrkB-positive cells lack neural markers at this stage; however, they migrate with other neurally differentiating cells that are TrkA and TrkC-positive. By St. 29/30, TrkB-positive cells begin to express the neural specific markers Hu C/D and Islet-1; eventually, all TrkB positive cells commence neural differentiation. By St. 34, TrkB and TrkC staining are lost. BDNF transcript expression parallels that of TrkB. In the <span class="yellow">mouse</span>, TrkB-positive cells surround newly formed sympathetic ganglia and a small number of TrkB positive cells that co-express tyrosine hydroxylase are seen within ganglia between E13.5-15. In cell culture, many cells from St. 29–30 <span class="yellow">chicken</span> lumbar sympathetic ganglia express neural markers and are dividing, indicating that they are sympathoblasts. Sympathoblasts and neurons require both nerve growth factor and neurotrophin-3 for survival. BDNF increases the number of cells expressing neural markers in culture by increasing number of cells that incorporate bromodeoxyuridine. In contrast, most TrkB-positive sympathetic cells in vivo are not actively proliferating between E6–E8.<br><br>Conclusion
Developing paravertebral sympathetic ganglia in avian and <span class="yellow">murine</span> embryos contain a subpopulation of sympathoblasts that transiently express TrkB and ultimately commence neuronal differentiation. These TrkB expressing sympathoblasts are not actively dividing in vivo; yet, when placed in vitro, will divide in response to BDNF. This suggests that the availability of BDNF in vivo fails to reach a threshold necessary to induce proliferation. We suggest that excess TrkB stimulation of sympathoblasts in vivo may lead to the genesis of neuroblastoma.<br><br><br><br>Background
Neural crest cells destined to become paravertebral sympathetic neurons proliferate and differentiate during migration and gangliogenesis. In <span class="yellow">chicken</span> embryos, migrating neural crest cells express catecholamines at Hamburger/Hamilton Stage (St.) 19, and these cells form the primary sympathetic chain dorsolateral to the aorta at St. 22 (E3.5) [1]. Between St. 23 (E4) and St. 28 (E6), these cells disperse and undergo a secondary migration to form the paravertebral sympathetic chain that resides ventral to the spinal cord and dorsal root ganglion [1]. After ganglia coalesce, sympathoblasts express markers of neuronal differentiation, such as Q211 and tyrosine hydroxylase (TH), at a time when they also incorporate [3H]-thymidine [2]. Time lapse photography has shown that cultured E15.5–E16.5 sympathetic neurons from <span class="blue">rat</span> embryos extend axons while they divide [3-5]. Although proliferation appears to be an important process to expand the sympathetic neuron population during differentiation, the mechanisms that guide sympathoblast proliferation have not been identified.
The development of sympathetic neurons is guided by neurotrophins. Neurotrophin-3 (NT-3) binds to its receptor, TrkC, to promote the survival of cultured sympathoblasts from early lumbar paravertebral ganglia [6]. Nerve growth factor (NGF) signals through its receptor, TrkA, to promote the survival of sympathetic neurons upon target innervation [7]. There are severe sympathetic defects in the superior cervical ganglion of individual NT-3 and NGF knockout <span class="yellow">mice</span> [8-10]. Furthermore, there is no additional cell death in the superior cervical ganglion of NT-3 and NGF double knockout <span class="yellow">mouse</span> embryos, suggesting that all of the neurons are dependent on both neurotrophins for survival [11]. There is also an increase in sympathetic neuron cell death in TrkA knockout <span class="yellow">mice</span> [12]. However, in TrkB and BDNF knockout <span class="yellow">mice</span>, there is no apparent phenotype in the superior cervical ganglion, and there is little evidence that TrkB or BDNF is expressed in sympathetic ganglia. Thus, it is generally thought that TrkB and BDNF have little or no roles in guiding the development of sympathetic neurons.
In addition to their developmental functions, neurotrophin receptors regulate cell behavior in neuroblastoma, a tumor found in sympathetic ganglia and adrenal medulla. Tumors that express TrkA often spontaneously regress, while those that express TrkB and its ligand, brain-derived neurotrophic factor (BDNF), grow aggressively, are invasive, and fail to respond to chemotherapeutic agents [13]. The presence of TrkA in neuroblastoma tumors is consistent with its expression in developing sympathetic neurons, and suggests that regressive neuroblastoma tumors arise from early sympathetic neurons that express TrkA. The function of TrkB in early sympathetic development is unknown, which makes understanding the etiology of aggressive neuroblastoma tumors difficult. Based on its function in neuroblastoma tumors, we hypothesize that BDNF and TrkB expression in differentiating sympathoblasts is responsible for expanding the neuronal population through proliferation.
We sought to determine whether BDNF and TrkB are involved in sympathetic development. We report that during early embryonic development, TrkB is expressed in a subset of differentiating sympathoblasts in both avian and <span class="yellow">murine</span> embryos. We also find that BDNF promotes the proliferation of TrkB-positive sympathoblasts in cell culture. However, the majority of TrkB positive cells in vivo fail to take up bromodeoxyuridine (BrdU) over a 24 hr period, suggesting that endogenous BDNF concentrations do not reach a threshold necessary to stimulate proliferation of sympathoblasts. Shortly after all of the TrkB positive cells commence neuronal differentiation, TrkB immunoreactivity is lost. These results suggest that prolonged expression and/or activation of TrkB signaling at these early stages may be an early event triggering the formation of neuroblastoma.<br><br>Results
TrkB is expressed during migration of neural crest cells to sympathetic ganglia
We first determined whether TrkB is expressed in neural crest-derived cells in the region ventral to the spinal cord and dorsal root ganglia where sympathetic ganglia coalesce between Hamburger/Hamilton Stages (St.) 25–28/29. To identify cells that have commenced neuronal differentiation, transverse sections of the lumbar spinal column region were stained with antibodies against Hu C/D [14], a neuronal-specific RNA-binding protein, or Islet-1, a transcription factor found in sympathetic neurons [15]. We found that Hu C/D and Islet-1 are expressed in the same cells both in vivo and in vitro throughout sympathetic development. In experiments done between St. 25 and 28, Islet-1 staining appeared weaker than Hu C/D staining, and thus we used Hu C/D to identify differentiating neurons at these stages. At later stages, Islet-1 was used to facilitate the identification of neurons because of the nuclear location of its immunoreactivity.
Cells expressing Hu C/D are first detected at St. 25 ventral to the spinal cord and dorsal root ganglion and lateral to the dorsal aorta (Figure 1A, 1B). By St. 26, the number of cells that express Hu C/D in this region increases dramatically (Figure 1C). TrkB-expressing cells first appear at St. 27 in the same region and are adjacent to Hu C/D-positive cells (Figure 1D, 2A, 2H). TrkB-positive cells co-localize with a neural crest marker, HNK-1 (Figures 2B, 2C, 2D). The Hu C/D-positive cells in this region are likely to be sympathetic neurons, since they appear in the region where sympathetic ganglia form and express tyrosine hydroxylase, a rate-limiting enzyme in the synthesis of catecholamines (Figures 2E, 2F, 2G). At St. 28/29, the cells begin to coalesce ventral to the dorsal root ganglion and the Hu C/D-positive cells and TrkB-positive cells remain as two separate cell populations (Figure 1E); however, shortly afterwards, all of the TrkB-positive cells begin to express Islet-1 (Figure 3B) and Hu C/D (data not shown).<br><br>Developmental regulation of TrkA, TrkB, TrkC, and BDNF expression
In contrast to TrkB, the other neurotrophin receptors, TrkA and TrkC, are co-expressed in both Hu C/D-positive and Hu C/D-negative cells at St. 27 (Figures 2I, 2J). We find that approximately 30% of the Islet-1-positive cells express TrkA (Figure 3A), while 50% express TrkB (Figure 3B) and 100% express TrkC (Figure 3C) at St. 29/30 (E6.5). Thus, all developing neurons express TrkC in combination with either TrkA or TrkB. By St. 31 (E7), the number of TrkA-positive, Islet-1-positive cells increases to 100% (Figure 3D) and immunoreactivities for both TrkB and TrkC appear dispersed (Figure 3E, 3F). By St. 34 (E8), TrkA expression is well-sustained (Figure 3G) and TrkB and TrkC immunoreactivities are lost (Figure 3H, 3I). We also examined the early development of <span class="yellow">murine</span> sympathetic ganglia (Figure 4). At E13, the newly formed lumbar sympathetic ganglia can be observed ventral to the spinal cord and notochord by their staining for Hu C/D and TH (Figure 4A). TrkB-positive cells can be seen surrounding developing ganglia, as well as in occasional cells within the ganglia (Figure 4C, D). These TrkB-positive cells within the ganglia co-express TH and are seen at a frequency of 1–2 cells per section starting at E13 (Figure 4A–D) and are still present at E15.5 (data not shown).
In neuroblastoma cells, BDNF is co-expressed with TrkB, suggesting that autocrine stimulation is a means by which proliferation is sustained in the transformed cells. To test whether BDNF, the ligand for TrkB, was present in embryonic <span class="yellow">chick</span> sympathetic ganglia, we used quantitative real-time PCR with TaqMan probes to determine the relative abundance of BDNF transcripts in total RNA extracted from lumbar sympathetic ganglia at St. 29/30 (E6.5), St. 31 (E7), St. 34 (E8), and E9. BDNF expression within the ganglia parallels that of TrkB: BDNF mRNA expression levels are highest at St 29/30 (E6.5), and these levels decrease 2-fold at St. 31 (E7) and St. 34 (E8; Figure 5). By E9, BDNF levels are 7 times lower than at St. 29/30 (E6.5; Figure 5).<br><br>NT3 and NGF promote survival of differentiating sympathetic neurons in culture
To determine the effect of neurotrophins, we cultured cells dispersed from lumbar sympathetic ganglia at St. 29/30 (E6.5) because, at this stage, ganglion formation is complete, the number of TrkA-, TrkB-, and TrkC-positive cells have peaked, and all Trk-expressing cells have initiated neural differentiation. First, we identified markers expressed by acutely isolated cells. As shown in Table 1, 80–91% of the cells are p75 neurotrophin receptor (NTR)-positive, indicating that most of the cells are neural crest-derived and little mesenchymal contamination is introduced by the isolation procedure. In addition, 28–33% of the cultured cells express the neural marker Hu C/D. Approximately half of these Hu C/D-positive cells express TrkB. Conversely, all of the TrkB positive cells express Hu C/D. These TrkB-positive cells comprise approximately 14–17% of the total cell population.
We then determined how many of the acutely isolated cells were proliferating by incubating them for 12 hrs in BrdU-containing medium. For these experiments, we identified differentiating neurons with the transcription factor Islet-1 because this marker labels nuclei, thus it co localizes with any BrdU that has been incorporated into the DNA, allowing us to determine whether the cell had undergone S-phase of the cell cycle. After 12 hrs in BrdU, 59% of Islet-1-positive nuclei stain for BrdU immunoreactivity. Thus, cultures of St. 29/30 sympathetic ganglia contain many cells that proliferate while exhibiting markers of neuronal differentiation, confirming previous observations [2]. We call these dividing neuronal precursors sympathoblasts. The remaining non-BrdU incorporating, Islet-1 positive cells are likely to be post mitotic neurons.
Finally, we determined the trophic requirements of St. 29/30 (E6.5) sympathetic neurons and sympathoblasts. We monitored cultures over a three day period after plating and counted the number of phase bright cells with neurites, a morphological feature of both neurons and sympathoblasts. In the absence of trophic factors, more than 2/3 of the cells die by 24 hours in culture and BDNF, NT-3, or NGF alone is not sufficient to promote survival (Figure 6). However, NGF together with NT-3 supports the survival of a significantly larger number of cells (Figure 6). For the subsequent experiments, all neurons were cultured with 25 ng/ml NT-3 and 1 μg/ml 7S NGF to optimize survival.<br><br>BDNF promotes proliferation of TrkB-positive sympathetic neurons in culture
To determine the effects of BDNF, cultures of cells from St. 29/30 (E6.5) sympathetic ganglia were supplemented with 200 ng/ml BDNF and the number of neurons and sympathoblasts were counted at 24, 48, and 72 hours using phase microscopy (Figure 7A). A 1.6-fold increase in the number of neurons due to BDNF is observed by 24 hours and this number does not increase further at 48 or 72 hours. This effect of BDNF is concentration-dependent with an EC50 of 75 ng/ml (Figure 7B).
To test whether the increase in the number of neurons and sympathoblasts caused by BDNF is due to the differentiation of pluripotent neural crest cells, we quantified the effects of BDNF on the number of neurally differentiating cells (Hu C/D-positive) versus the number of non-neuronal cells (Hu C/D-negative) after identifying all neural crest-derived cells by staining for p75NTR in St. 29/30 (E6.5) cultures. If BDNF increases the number of neurons and sympathoblasts by inducing a non-neuronal cell to express Hu C/D, then we expected that the total cell number would remain the same and that there would be a decrease in the number of non-neuronal cells as well as a corresponding increase in the number of neurons. After 24 hours, BDNF significantly increases the number of p75NTR-positive cells as well as the number of Hu C/D-positive cells (Figure 8A). However, there was no statistically significant change in the number of non-neuronal cells. Thus, it is unlikely that BDNF increases the number of neurons and sympathoblasts by inducing differentiation of non-neuronal cells.
To determine whether the increase in the total number of neurons and sympathoblasts is caused by BDNF-induced proliferation, control and BDNF-treated sympathetic cultures were exposed to BrdU for 12 hours after plating, and the number of cells that incorporated BrdU into their DNA was determined after 24 hours in culture. Even in the control condition, a number of cells in the culture are dividing, giving a high baseline of BrdU incorporation (Figure 8B). When BDNF is added, the total number of BrdU positive cells increases approximately 1.6-fold (Figure 8B). This BDNF-induced increase in the total number of BrdU positive cells occurs in sympathoblasts because the number of Islet-1-positive nuclei from control cultures that label with BrdU is 268 ± 59 and BDNF treatment raises this number to 424 ± 80, which corresponds to an increase of 1.6-fold. This accounts for the 1.6-fold increase in total neuron number and total BrdU-positive cells described above. We then confirmed that BDNF acts on TrkB-positive cells: BDNF increases the number of TrkB-expressing cells that incorporate BrdU 2.6 – 4-fold over control (Table 2) and it also increases the overall number of TrkB-positive cells 2 – 2.5-fold over control (Table 2). BDNF does not increase the number of BrdU-positive, TrkB-negative cells or the overall number of TrkB-negative cells (Table 2). In further support that BDNF acts directly on TrkB-expressing cells, an antibody directed against the extracellular domain of TrkB completely prevents the effect of BDNF in promoting proliferation of TrkB-positive, but not TrkB-negative cells (compare Figure 9A to 9B). Thus, the effect of BDNF is restricted to the population that expresses TrkB, which are developing sympathoblasts.
To determine whether TrkB-positive cells are actively proliferating in vivo, embryos were injected with BrdU at St. 27 and harvested at St. 29, approximately 24 hrs later. The majority (85–90%) of TrkB-positive cells do not incorporate BrdU into their nuclei under basal conditions in vivo (Figure 10), although a few TrkB positive cells with labeled nuclei could be observed (arrows). This contrasts with our observation that 71–76% of TrkB-positive cells incorporate BrdU in culture after treatment with BDNF (Table 2), suggesting that endogenous BDNF does not achieve a threshold sufficient to support a high level of sympathoblast proliferation in vivo.<br><br>
Discussion
We report that the neurotrophin receptor TrkB is expressed in a subset of embryonic sympathoblasts during the early development of lumbar paravertebral sympathetic ganglia in <span class="yellow">chicken</span> and <span class="yellow">mouse</span> embryos. In the <span class="yellow">chicken</span>, TrkB expression is transient, and completely lost by St 34 (E8). Since BDNF induces the proliferation of sympathoblasts in cell culture, yet in vivo there is little proliferation observed in TrkB-positive cells in nascent ganglia, we propose that if TrkB activation becomes unregulated by excess BDNF or constitutive phosphorylation of TrkB [16], this transient population of TrkB-positive sympathoblasts may trigger the genesis of neuroblastoma, a childhood tumor found in the paravertebral chain and adrenal medulla.
The two populations of sympathoblasts that we observe support previous findings of heterogeneity among developing sympathetic neurons and neural crest cells. Early sympathetic ganglia contain at least two subpopulations: early differentiating neurons that lack TrkB expression and express TrkA and TrkC, and late differentiating sympathoblasts that express TrkB. Explant cultures of sympathetic ganglia from E16 <span class="yellow">chick</span> embryos give rise to two neuronal populations: one that remains close to the explant, and one that migrates away from the explant [1]. In addition, early neuronal subpopulations have been observed in cultures of neural crest cells from St. 13/14 quail embryos as evidenced by the expression of neuronal cell type-specific gangliosides [17]. Perhaps these different subpopulations will ultimately give rise to the two neurochemically distinct populations found in lumbar sympathetic ganglia: the noradrenergic, NPY-containing neurons that innervate internal organs and enteric ganglia and the cholinergic, VIP-containing neurons that innervate vasculature in the hind limbs.
The effects of BDNF and TrkB deletion and over expression have been studied on superior cervical ganglion and preganglionic neurons in thoracic segments of the spinal column, but not on paravertebral sympathetic neurons. In the superior cervical ganglion, an increase in the number of neurons of BDNF null <span class="yellow">mice</span> is likely due to apoptosis induced by BDNF via p75NTR [18]. In contrast, the responses of paravertebral sympathetic neurons to BDNF are complex and subtype dependent. Over expression of BDNF leads to an increase in the number of noradrenergic fibers innervating the erector pilli muscles of hair follicles, while noradrenergic fibers innervating blood vessels were unaffected [19]. If our results indicating that BDNF promotes proliferation of TrkB-positive sympathoblasts in the <span class="yellow">chicken</span> embryo can be extrapolated to the subset of TrkB-positive sympathoblasts in <span class="yellow">murine</span> ganglia, then these TrkB-positive cells may be neurons destined to innervate the erector pilli. In other studies, TrkB null <span class="yellow">mice</span> showed no changes in morphology or cell number in superior cervical ganglia [12] or in the intermediolateral column [20]; but this may not be predictive of a phenotype in the lumbar paravertebral chain. It is thus possible that BDNF/TrkB signaling could play a specific role in other regions of the paravertebral sympathetic chain, such as the lumbar region. However, if TrkB-positive cells are not normally actively proliferating in vivo, then it would not be surprising that the development of the paravertebral sympathetic chain is not disrupted in TrkB or BDNF null <span class="yellow">mice</span>. It may be more informative to examine <span class="yellow">mice</span> that over express BDNF on a promoter that targets expression to embryonic lumbar ganglia. Unfortunately, such <span class="yellow">mice</span> do not exist.
Our findings that the St. 29/30 (E6.5) sympathoblasts are dependent on both NT-3 and NGF for survival in culture are consistent with previous work on <span class="yellow">mouse</span> sympathoblasts from the superior cervical ganglion [11]. In these studies, NT-3 and NGF deletion separately led to a decrease in the number of sympathetic neurons at E17.5 compared to control. Deletion of both NT-3 and NGF together did not enhance cell death. In contrast, cultured <span class="blue">rat</span> superior cervical ganglion sympathetic neurons respond to NT-3 at E14.5 and then to NGF at E19.5, although time points in between were not analyzed [6].
In addition to promoting survival, NT-3, NGF, and BDNF also induce proliferation of various neuronal precursors at different stages of development. NT-3 can promote the incorporation of [3H]-thymidine into cultured quail neural crest cells from the trunk region [21,22], Later in <span class="blue">rat</span> sympathetic development, NT-3 supports survival of neurons, but does not promote proliferation [6], which is consistent with our results. NGF promotes an increase in BrdU incorporation from 25% to 35% in the DRG cervical segment 2 in the <span class="yellow">chick</span> embryo [23]. In <span class="yellow">chicken</span> embryos that are treated with NGF in ovo at St. 18 and 21, there is an increase in BrdU uptake after formation of the primary sympathetic chain at St. 23 [24]. Since NGF does not appear to affect proliferation of St. 29/30 (E6.5) <span class="yellow">chick</span> sympathoblasts, NGF may only promote proliferation in primary, but not secondary chain sympathoblasts. Motor neuron progenitors in the ventral neural tube from the <span class="yellow">chick</span> embryo express TrkB and when ventral neural tube explants are treated with BDNF, there is an increase in the number of motor neurons produced and BrdU incorporation [25]. BDNF also promotes the proliferation of cultured neuroblastoma cells [13]. Taken together, these results are consistent with our findings that NT-3 and NGF do not promote proliferation of St. 29/30 (E6.5) sympathoblasts, and support the assertion that BDNF promotes proliferation of TrkB-positive sympathoblasts in culture.
Our observations suggest a transient function of TrkB during early sympathetic development in supporting proliferation of this early subpopulation of sympathoblasts. However, the in vivo labeling suggests that only a minority (10–20%) of this population is dividing during the window that TrkB is expressed. In light of the very strong proliferative effect produced in cell culture, these TrkB expressing cells could respond more strongly if endogenous BDNF rises to higher levels, or if the mechanism that down regulates TrkB expression becomes nonfunctional. Such events could trigger an early proliferative event that leads to a cascade of changes that initiates transformation of cells to neuroblastoma. Thus, these early TrkB expressing cells help solve the puzzle as to why TrkB is expressed in aggressive and invasive forms of neuroblastoma, particularly because BDNF induces cultured neuroblastoma cells to become more proliferative, invasive, angiogenic, and resistant to chemotherapeutic reagents than untreated cultures [13]. Future studies will determine whether constitutive expression of BDNF and TrkB in the <span class="yellow">chick</span> embryo sustains proliferation of differentiating sympathoblasts.<br><br>Conclusion
We have identified a time point during development when differentiating lumbar sympathetic neurons transiently express TrkB and proliferate in response to high concentrations of BDNF in culture. These studies suggest that elevated BDNF expression above basal levels and signaling through TrkB may be a mechanism that contributes to the onset of neuroblastoma. A further understanding of the two populations of sympathetic neurons and the fate of the TrkB-positive cells will provide additional insight into the development of paravertebral sympathetic ganglia and the genesis of neuroblastoma.<br><br>Methods
Preparation of tissue for immunohistochemistry
The lumbar spinal column and surrounding tissues were dissected from <span class="yellow">chicken</span> embryos at the indicated stages and placed in Zamboni's fixative (4% (w/v) paraformaldehyde, 15% (v/v) picric acid in 0.1 M sodium phosphate buffer, pH 7.4) for two hours at room temperature. <span class="yellow">Mouse</span> embryos at 13–15 days post-coitus were collected according to an IACUC-approved protocol to Dr. L. Sherman at the Oregon Health and Science University. The <span class="yellow">mouse</span> embryos were immersion-fixed in Zamboni's fixative overnight at 4 degrees C then washed with phosphate buffered saline (PBS; 130 mM NaCl, 20 mM sodium phosphate buffer, pH 7.4). Fixed tissues were equilibrated in 30% sucrose in 1× phosphate-buffered saline (PBS). Fixed <span class="yellow">mouse</span> embryos were shipped to Vermont in sucrose. Transverse 30 μM sections of the spinal columns were cut at on a Microm HM cryostat (knife temperature: 16°C; object temperature: 23°C) and collected on Superfrost Plus slides (Fisher). Sections were dried at room temperature, washed in 1× PBS and incubated overnight in blocking buffer (1× PBS consisting of 10% (v/v) heat-inactivated <span class="yellow">horse</span> serum (Invitrogen/Gibco), 0.5% Triton X-100 (Sigma), and 0.1% sodium azide (Fisher)).<br><br>Immunohistochemistry
Sections were incubated with primary antibodies overnight at 4°C, followed by incubation with secondary antibodies for 2 hours at room temperature. Primary antibodies used were: <span class="yellow">rabbit</span> anti-p75 (1:1500, generous gift from Louis Reichardt, UCSF [26]), <span class="yellow">mouse</span> IgG2b anti-Hu C/D, (1:250, Molecular Probes); <span class="yellow">mouse</span> IgG1 anti-Islet-1, (1:10, Developmental Studies Hybridoma Bank); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkA (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkB (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkC (1:500) (all Trk antibodies were generous gifts of Dr. Louis Reichardt, UCSF [26-28]); <span class="yellow">mouse</span> anti-HNK-1 (1:50, Developmental Studies Hybridoma Bank); <span class="yellow">mouse</span> IgG2a anti-tyrosine hydroxylase (1:10, Developmental Studies Hybridoma Bank), <span class="yellow">sheep</span> anti-BrdU (1:100, Biodesign International), <span class="yellow">rabbit</span> anti-tyrosine hydroxylase (1:100, Chemicon), and <span class="yellow">goat</span> anti-TrkB (1:1000, R&D Systems). Immunofluorescence was imaged using a Nikon C1 confocal mounted on a Nikon Eclipse E800 microscope with a 10× Plan Apo (NA 0.785) air objective or a 60× Plan Apo (NA 1.4) oil objective lens, E7-C1 software, and UV, Argon, and He/Ne lasers exciting at 408, 488, and 543 nm and emitting at 404 500–530, and 555–615 nm, respectively. A Nikon Eclipse E800 microscope in the nearby COBRE Molecular/Cellular Core Facility was used for counting immunofluorescent cells at 200× using epifluorescence optics.<br><br>RNA Extraction/cDNA synthesis
Sympathetic ganglia were removed from <span class="yellow">chick</span> embryos and RNA was isolated using TriReagent (Molecular Research Center), an acidified guanidinium with phenol extraction method [29]. RNA was transcribed to cDNA using oligo-dT with Superscript II Reverse Transcriptase (Invitrogen) at 42°C for 1 hour.<br><br>Real-time PCR
Relative RNA levels were determined using quantitative real-time PCR with an ABI 7500 Fast Real Time PCR System. TaqMan probes were used to quantify the progression of the PCR reaction and reactions were normalized using the constitutively expressed gene <span class="yellow">chick</span> ribosomal binding protein s17 (CHRPS). The sequences were used for primer/probes sets: for BDNF: forward: 5'-AGCCCAGTGAGGAAAACAAG-3', reverse: 5'-ACTCCTCGAGCAGAAAGAGC-3', probe: 5'-[6-FAM]-TACACATCCCGAGTCATGCTGAGCA-[BHQ]-3'; for CHRPS (<span class="yellow">chick</span> ribosomal binding protein S-17): 5'AACGACTTCCACACCAACAA3', reverse: 5'CTTCATCAGGTGGGTGACAT3', probe: 5'-[6-FAM]-CGCCATCATCCCCAGCAAGA [BHQ]-3'. Primers and probes were synthesized by Operon Technologies, Inc (Alameda, CA). The primers for BDNF were validated against primers for CHRPS according to an Applied BioSystems protocol by serially diluting the target cDNA 1:10, determining the cycle threshold (Ct) for each reaction, and plotting the Ct versus log concentration. Slopes of the resulting lines were calculated and primers were accepted if their Ct slopes were between -3.2 and -3.4 (a perfect efficiency of 1.0 yields a slope of -3.3). To analyze the data, the delta Ct method of relative quantification was used, where the Ct of Chrps was subtracted from the Ct of the gene of interest (Delta Ct) and the arbitrary units of mRNA were expressed as 10000/2^(Delta Ct).<br><br>Cell culture
Sympathetic neurons were cultured as previously described [30] with a few modifications. Sympathetic ganglia were removed from the lumbar region of the paravertebral chain of St. 29/30 (E6.5) <span class="yellow">chick</span> embryos and placed in Modified Puck's solution with glucose (MPG). The cells were dissociated by incubation of sympathetic ganglia with 0.1% trypsin in MPG at 37°C for 10 minutes followed by triturating with a fire polished 9" Pasteur pipette. Cells were then resuspended in Dulbecco's Modified Eagle Medium (DMEM) consisting of 10% <span class="yellow">horse</span> serum, 2% fetal <span class="yellow">calf</span> serum, and 10 mg/ml penicillin/streptomycin. For neurotrophin studies, the culture medium was supplemented with 25 ng/ml NT-3 (R & D Systems) and 1 μg/ml 7S NGF (Alomone Labs) upon plating, and 50 ng/ml, 100 ng/ml, or 200 ng/ml BDNF (R & D Systems) once the cells adhered to the wells. Cells were plated on poly-D-lysine/laminin coated wells or cover slips (Fisher) as previously described [30].<br><br>Quantification of neurons and sympathoblasts using phase microscopy
Embryonic sympathoblasts and neurons are small, phase bright cells with neurites. The total number of cells with neurites the length of two cell bodies were counted in 10 non-overlapping fields of view evenly spaced in a grid-like pattern across the bottom of a well from a 24 well plate at 200× using a Nikon Eclipse TE200 microscope.<br><br>BrdU labeling
For in vitro studies, approximately 2 hours after plating cells from St. 29/30 (E6.5) sympathetic ganglia, cells were labeled with 10 μM bromodeoxyuridine (BrdU, Sigma) for 12 hours at 37°C. Following this labeling period, cells were incubated in complete medium without BrdU for an additional 10 hrs. Cells were then fixed in Zamboni's fixative for 30 min at room temperature and rinsed with 1× PBS. For in vivo studies, 25 μg BrdU was injected into the amnion of <span class="yellow">chick</span> embryos at St. 27. The cells and sections were denatured with 2 N HCl at 37°C for 1 hr, and were then neutralized with 0.1 M borate buffer, pH 8.5, for 10 min at room temperature. Immunochemistry was performed as described above.<br><br>
Abbreviations
BDNF, brain-derived neurotrophic factor; BrdU, Bromodeoxyuridine; DA, dorsal aorta; DMEM, Dulbecco's Modified Eagle's Medium; DRG, dorsal root ganglion; E, embryonic day; HS, <span class="yellow">horse</span> serum; MPG, Modified Puck's solution with glucose; NGF, nerve growth factor; NC, notochord; NT, neural tube; NT-3, neurotrophin-3; NTR, neurotrophin receptor; PBS, phosphate-buffered saline; SC, spinal cord; SCG, superior cervical ganglion; SEM, standard error of the mean; SG, sympathetic ganglion; St., stage; w/v, weight/volume; v/v, volume/volume.<br><br>Authors' contributions
JAS designed the experiments, performed the experiments, analyzed the data, and wrote the manuscript. GLSS contributed intellectually to the conception and design of this study, and assisted in the interpretation of the results. RN supervised the study, participated in the design of experiments, edited the manuscript, and obtained funding for the project. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2553092</h3>TOPS++FATCAT: Fast flexible structural alignment using constraints derived from TOPS+ Strings Model
Abstract
Background
Protein structure analysis and comparison are major challenges in structural bioinformatics. Despite the existence of many tools and algorithms, very few of them have managed to capture the intuitive understanding of protein structures developed in structural biology, especially in the context of rapid database searches. Such intuitions could help speed up similarity searches and make it easier to understand the results of such analyses.<br><br>Results
We developed a TOPS++FATCAT algorithm that uses an intuitive description of the proteins' structures as captured in the popular TOPS diagrams to limit the search space of the aligned fragment pairs (AFPs) in the flexible alignment of protein structures performed by the FATCAT algorithm. The TOPS++FATCAT algorithm is faster than FATCAT by more than an order of magnitude with a minimal cost in classification and alignment accuracy. For beta-rich proteins its accuracy is better than FATCAT, because the TOPS+ strings models contains important information of the parallel and anti-parallel hydrogen-bond patterns between the beta-strand SSEs (Secondary Structural Elements). We show that the TOPS++FATCAT errors, rare as they are, can be clearly linked to oversimplifications of the TOPS diagrams and can be corrected by the development of more precise secondary structure element definitions.<br><br>Software Availability
The benchmark analysis results and the compressed archive of the TOPS++FATCAT program for Linux platform can be downloaded from the following web site: <br><br>Conclusion
TOPS++FATCAT provides FATCAT accuracy and insights into protein structural changes at a speed comparable to sequence alignments, opening up a possibility of interactive protein structure similarity searches.<br><br><br><br>Background
Structural biology is one of the most successful fields of modern biology. Over 50,000 solved protein structures illustrate details of many specific biological processes. The same data also provide us with information about the global features of protein structure space and can be studied to discover the evolutionary, physical, and mathematical rules governing them. How many fundamentally different protein shapes (folds) are there? How do protein structures evolve? How do new structural features appear, and if they are coupled with changes in function, how does this process occur? Such questions can be studied by classifying, comparing and analyzing known protein structures. Two different, but synergistic strategies are typically used for this purpose. In classification systems such as SCOP [1] or CATH [2], <span class="yellow">human</span> intuition is used to simplify the description of protein structures to a manageable size, and a <span class="yellow">human</span> eye, sometimes supported by automated analysis, can recognize patterns and types of structures. In the second approach, specialized comparison algorithms, such as DALI [3], CE [4], or FATCAT [5] can be used to calculate a distance-like metric in the protein structure space. This in turn can be used to cluster proteins into groups. Many such algorithms have been developed over the past few decades and have been mostly used for the classification of protein structures into families.
An exact solution of an alignment between two structures is formally equivalent to a threading problem and is therefore NP-complete [6]. However, a practical solution can be obtained by heuristics reducing the problem to a manageable size [7]. In <span class="yellow">human</span> classification systems, the protein is usually reduced to a set of several structural elements, which obviously involve many arbitrary thresholds. Automated algorithms have the same problem and also suffer from inconsistencies between different numerical measures of protein structure similarity [8]. Interestingly, despite these problems, results of different approaches are broadly similar. They all identify approximately a few hundred general classes of protein structures, usually called folds [1] or topologies [2], distinguished by how the main chain of the protein folds around itself in the three-dimensional space. At the same time, the comparison of different approaches, both between and within the two classes, shows that fold/topologies (or cluster) definitions are somewhat fuzzy, with some proteins being occasionally difficult to classify and joining different groups depending on various assumptions. This lead some to question the concept of the fold [9], but practical application of protein structure comparison leaves little doubt that protein structure space has some natural granularity that overlaps well with the traditional fold classification.
Comparison and classification of protein structures is significantly simplified by the fact that proteins have naturally modular structures, being mostly composed of locally regular structures: alpha helices and beta strands. These two types of secondary structures constitute a little over 50% of an average protein's length. With the average length of a secondary structure being around 10 amino acids, this makes it possible to describe protein structure as an arrangement of a much smaller number of elements. Protein structures are often visualized in a simplified form, with the so-called ribbon diagram with secondary structures shown as helices and arrows being the most popular (see Figure 1). This picture can be simplified further by showing individual secondary structure elements as simple symbols (circles or boxes/triangles). These depictions, called fold diagrams, originally proposed in the 70s [10-12] are best captured by a TOPS (Topology of Protein Structures) algorithm, which attempts to automate the process of creation of the topology cartoon [13]. While useful in protein classification, such simplified descriptions are not used in the most popular automated protein structure comparison algorithms such as DALI [3] or CE [4]. Kleywegt and Jones developed a method for finding similar motifs based on comparing distance matrices that are constructed by representing protein as a set of SSEs with their directional vectors and angle between those vectors [14]. Programs that used SSEs either for structure comparison based on hierarchical superposition of both SSEs and atomic representation [15] or for finding common substructures in the comparison process based on subgraph isomorphism, such as [16,17] and recent applications of the TOPS diagram [18,19], usually struggle with translating the comparison results from the secondary structure to the individual residue level. Although the SSM method uses graph-matching procedures at the SSE level followed by an interactive 3D alignment of the protein C-alpha atom [20], it lacks the topological relationships between the SSEs, which are essential features in identifying common scaffolds in distantly related proteins. A TOPS pattern was used to guide the sequence alignment, for instance, to build multiple structural alignments of the distantly related family of beta-rich protein domains [21]. The Multiple Sequence Alignment Tool (MSAT) automates this approach, merging it with a popular ClustalW program [22]. DALI [3], CE [4] or FATCAT [5] introduce their own methods of decomposing the protein structure into smaller units, such as 7 × 7 dense distance map fragments (DALIs) or aligned fragment pairs (AFPs) (CE and FATCAT). The large number of such fragments and the combinations of the fragments that need to be evaluated by structure comparison programs is the main reason for the significant computational requirements of such algorithms. However, more importantly, TOPS+ method is used here to enable a structural comparison that takes into account flexibility in protein structures and not only classifies the differences, but also can recognize such rearrangements – which is a first such application using the SSEs language. In this contribution, we explore the question of whether it would be possible to combine insights provided by topology diagrams into automated protein structure alignment algorithms, focusing on the FATCAT program developed previously in our group.<br><br>Methods
Flexible structure alignment method FATCAT
FATCAT [5] is a unique structure alignment method that allows for flexibility in the structures being compared. It builds the alignment by chaining aligned fragment pairs (AFPs) [23] together using a unified scoring function where AFP extensions, gaps, and twists each have their specific scores (Figure 2). Introducing a twist into the alignment is penalized, but this penalty may be compensated for by the gain in the score of the resulting alignment (i.e., longer alignment and/or better RMSD). Rigid alignment can be treated as a special case, in which no twist is allowed in chaining AFPs. FATCAT program provides alignment in both, "rigid" mode and "flexible" (default) mode.
FATCAT, as well as most other protein structure comparison programs, is very slow when compared to sequence alignments. The computing time of FATCAT is determined by the size of the collection of AFPs detected between the two structures being compared. FATCAT is available from a server  with an option to search in SCOP or PDB databases for similar structures. This search typically takes between 8 to 16 hours of CPU time, and this is the main obstacle to broader use of this option. FATCAT has been used to construct a Flexible Structure Neighborhood (FSN) database that contains pre-computed results of structure similarity searches and it takes several weeks of CPU time to update the FSN database. Other protein structure comparison resources, such as DALI or CE have very similar problems.<br><br>TOPS cartoons and TOPS graph models
As discussed in the Background, TOPS cartoons capture the simplified, fold-level description of protein structure and at the same time can be automated [24]. The TOPS algorithm uses structural features such as hydrogen bonds and chirality of the beta strands to provide a scoring function to optimize the cartoon (see Figure 1(b)). In TOPS, the secondary structural elements (SSEs) are derived from the DSSP program [25]. Based on TOPS cartoons, a formal graph model and graph-based definitions of protein topology and pattern discovery and comparison methods were developed [26,27]. The TOPS database and comparison, pattern discovery and matching programs are accessible from .<br><br>Novel TOPS+ and TOPS+ strings models
The TOPS model was further enhanced to incorporate features such as protein-ligand interaction information and more detailed secondary structural segment information. This enhanced model is called TOPS+ model (see Figure 3a). This TOPS+ model can be described formally in a TOPS+ strings language (Figure 3b) at a reduced linear level. The enhanced TOPS+ strings models can be used in fast string-based structure matching and comparison, at the same time avoiding issues of NP-completeness associated with graph alignments.
In detail, each node (SSE segment) of the TOPS+ strings is described by its type, orientation, PDB start number, segment length, total number of incoming (InArc) and outgoing (OutArc) arcs (edges), total number of ArcTypes, and total number of ligand arcs (LigArc). The type of the segment (SSEType) could be one of [E, e, H, h, U, u], where, "E" and "e" represent the "up"- and "down"-oriented beta strands; "H" and "h" indicate the "up"- and "down"-oriented alpha helices; and "U" and "u" represent ligand-bound and ligand-free loops. The InArcType can be classified as an/a [R, L, P, A], where "R" and "L" represent right and left chiralities; and "P" and "A" represent parallel and anti-parallel hydrogen bonds, respectively. The OutArcType is represented in a similar manner by [R', L', P', A']. Ligand arcs are indicated by LT = AA, where LT is the ligand type and AA is the PDB number. For example, Figure 3(a) and 3(b) contain visual representations of TOPS+ and TOPS+ strings models, respectively, for the protein domain d1fnb_1. Here the triangles represent the beta strands; the red curve represents the alpha helix; gray ellipsoids indicate loops; and green arcs indicate hydrogen bonds between two beta strands, called anti-parallel beta sheets. The length of a TOPS+ strings model is defined by number of SSEs; thus, the length of d1fnb_1 is 19. For further details, see [28].<br><br>TOPS+ strings comparison method
TOPS+ is a comparison method that computes a distance between TOPS+ strings models of two proteins based on a dynamic programming approach and identifies the longest common subsequence (LCS), consisting of the list of the topologically equivalent SSEs between two proteins. For example, Figure 3(c) shows the TOPS+ strings alignment between Dihydropteridine reductase proteins from <span class="red">rat</span> (1dhr) and <span class="yellow">human</span> (1hdr). The TOPS+ strings models for 1dhr and 1hdr are represented by a linear string-model, where a yellow triangle and red curves indicate the beta strands and alpha helices in their "up" or "down" orientations, respectively. The grey line and purple stubs represent the loop regions and the NAD ligand interactions, respectively. Note that the ligand-interaction information is optional and in this work we have not used it. The incoming and outgoing arcs are depicted in the SSEs (top and bottom of the beta strands), where red and green arcs represent the parallel and anti-parallel hydrogen-bond interactions that show beta-sheet information, while yellow and blue arcs indicate the right and left chirality relationships between the SSEs. A pink arrow between the TOPS+ strings elements indicates the conserved SSE. The dotted arrows indicate the conserved alpha helices and beta strands, while the plain arrows indicate the conserved loop regions.<br><br>TOPS++FATCAT method
In this work, we want to test the general idea of pruning the search space of the FATCAT comparison process using topological constraints derived from the TOPS+ strings alignment. Many of the AFPs considered in the FATCAT alignment could be easily eliminated from the comparison by constraining the alignment region. Here we explore constraints obtained from the TOPS+ strings alignment, which identifies topologically equivalent secondary structure elements (alpha helices, beta strands, and loops) for this purpose. Such equivalences define blocks that restrict the alignment region; AFPs that fall outside these regions are simply not considered (see Figure 4(b)). We introduce a parameter r to control the strictness of constraints by TOPS+ strings alignments; r equals 0 if the alignment region is strictly restrained by TOPS+ strings alignment, and r is set to 1 by default in our program to allow certain flexibility to the constrained alignment region (Figure 4(c)). We then can speed up the FATCAT alignment by considering only the AFPs within the constrained alignment area (Figure 4(d)). The rigid structural alignment can be treated as a special case of TOPS++FATCAT, in which no twist is allowed in chaining AFPs. However, the TOPS++FATCAT program provides alignment in both, "rigid" mode and "flexible" mode (default).<br><br>Benchmarking
For benchmarking and comparison, we have used the PDB40 dataset of 1,901 protein domain pairs (DP) corresponding to SCOP version 1.61 from the ASTRAL database [29]. Table 1 provides the SCOP superfamily level homolog versus non-homolog statistics for the four main SCOP classes i.e., all-alpha, all-beta, alpha/beta, alpha+beta, and all proteins regardless of their structural classes.<br><br>Evaluation Analyses
We performed the Receiver Operating Characteristics (ROC) curve and the AUC (Area Under the ROC Curve) analyses to compare the performance of the TOPS++FATCAT method with the original FATCAT method, using SCOP classification at the superfamily level as a standard of comparison [30].<br><br>
Results
ROC and AUC Analyses
We have compared the performance of the TOPS++FATCAT method against the original FATCAT method using the SCOP classification information at the superfamily level. We have plotted the ROC curves based on P-values obtained from the FATCAT and the TOPS++FATCAT methods. We have plotted the ROC curves separately for the main SCOP classes, i.e., all-alpha, all-beta, alpha/beta, alpha+beta, and all proteins regardless of the class they belong to (see Figure 5(a) to 5(e)). In the graph, the x- and y-axes represent the false positive and true positive rates of the performance of the comparison methods respectively. In the legend, rF-pvalue and fF-pvalue indicate results from the rigid and flexible FATCAT methods, respectively; similarly, rT2F-pvalue and fT2F-pvalue represent the rigid and flexible TOPS++FATCAT methods, respectively. We have calculated the AUC values for all the SCOP classes based on ROC curves obtained from the FATCAT and TOPS++FATCAT methods with the flexible/rigid options (see Table 2).
For all protein classes, the rigid FATCAT performs best, usually followed by the flexible FATCAT, the rigid TOPS++FATCAT, and the flexible TOPS++FATCAT. The performance of all four methods is best for all alpha and all beta proteins, and all four perform markedly worse (but similar to each other) for alpha/beta proteins. Only alpha+beta proteins show a clear difference between the FATCAT and TOPS++FATCAT methods. It is important to note that the TOPS+ strings models consider the parallel and anti-parallel properties of the beta-sheet information in the form of total number of incoming and outgoing arcs with their ArcTypes. Thus, the TOPS++FATCAT method discriminates the protein domain pairs more efficiently compared to the original FATCAT method. For example, in the all-beta protein domain pairs, both the flexible and the rigid TOPS++FATCAT methods perform well. The flexible TOPS++FATCAT method covers nearly 84% of protein domains with 0% false positives, but the flexible and rigid FATCAT methods cover only 76% and 49% of the true positives, respectively, with 0% false positives. The zoomed-in version of the ROC curves with up to 10% false positives for all-beta rich protein families is shown in Figure 5(f); where both the rigid TOPS++FATCAT (green) and flexible (red) TOPS++FATCAT methods have coverage rates of 82% and 84% true positives respectively with 0% false positives. The overall results for all protein classes show that TOPS++FATCAT performance is only slightly lower (3%–7% AUC value difference (see Table 2)) as compared to FATCAT while providing a significant, more than 10-fold speedup (see next section).<br><br>AFP and Runtime Analyses
We tested both the FATCAT and TOPS++FATCAT methods using the Mac OS X version 10.4.10 computer system with a 2 × 2.66-GHz Dual-Core Intel Xeon processor and 1-GB 667 MHz memory. We have performed runtime analysis on 1,901 protein domain pairs and counted the total number of AFPs and the corresponding runtime from both the FATCAT and the TOPS++FATCAT methods. The results show an exponential increase in AFPs (Figure 6(b)) and corresponding runtime (Figure 6(a)) for the FATCAT method as compared to the TOPS++FATCAT method (see Table 3) For example, the average number of AFPs for the TOPS++FATCAT method is 530, but the average number of AFPs for the FATCAT method is 15,019. This represents the number of average AFPs used by the FATCAT method is increased by a factor of 28 (see Table 3). This result leads to the conclusion that TOPS++FATCAT is 22 times faster compared to the FATCAT because this method must take into account more number of AFPs in the comparison process (see Table 3).<br><br>Case Studies
While the overall accuracy of both rigid and flexible FATCAT methods is better than their TOPS++FATCAT equivalents, an interesting example where the opposite is true lies in the comparison of two proteins, d2trxa_ (108 aa) from <span class="yellow">Escherichia coli</span> and d1kte__ (105 aa) from <span class="yellow">Sus scrofa</span> (<span class="yellow">pig</span>) from the thioredoxin-like superfamily. For this pair, the flexible_TOPS++FATCAT method provides an alignment with 88 equivalent positions with 1.67 Å chain RMSD and 3.06 Å of optimal RMSD without any twist, giving the alignment with 10% sequence identity (see Table 4). On the other hand, the flexible_FATCAT method provides an alignment with 86 aligned positions using a twist in the C-terminal region; it has a higher chain RMSD of 5.14 Å, and its optimal RMSD is 3.48 Å. For more information regarding the chain and optimal RMSDs refer [5]. The flexible_FATCAT method uses the twist to align a helix in the C-terminal region, which is positioned incorrectly with a beta-sheet core (see Table 4). Figure 7(a) shows the superposition of d2trxa_ (gray) and d1kte__ (orange) domains from the flexible_FATCAT method, where the blue color indicates the d1kte__ protein domain from the flexible_TOPS++FATCAT method. The incorrect alignment of the C-terminal domain alpha helix of the d1kte__ domain (orange) is visible in the core of the beta-sheet region. Figure 7(b) and 7(c) shows the AFPs from the flexible_FATCAT and flexible_TOPS++FATCAT methods, respectively. The hinge region provides a twist in the flexible_FATCAT method indicated by an arrow and the AFPs represented by a different color (see Figure 7(b)). In this case, the alignment constraints from the TOPS+ strings alignment allow the TOPS++FATCAT method to avoid a spurious alignment.
The Erythrocruorin protein domain d1eca__ (136 aa) from <span class="yellow">Chironomus thummi</span> and the Phycocyanin alpha subunit protein domain d1cpca_ (162 aa) from <span class="yellow">Fremyella diplosiphon</span> (Cyanobacterium) belong to the Globin-like superfamily. For these protein domain pairs, the FATCAT method provides a better alignment with 120 and 118 aligned positions with the chain RMSD of 4.02 Å based on the flexible and rigid options, respectively. The flexible_TOPS++FATCAT method gives an alignment of 63 aligned positions with the 3.23 Å optimal RMSD and the 6.28 Å chain RMSD. In this case, the flexible_TOPS++FATCAT method misses the N-terminal region helix and misaligns some helices. For example, Figure 8(a) shows the superposition of d1eca__ (gray) and d1cpca_ (orange) domains from the flexible_FATCAT method, while d1cpca_ (blue) domain is from the flexible_TOPS++FATCAT method. The AFP chaining alignment and the actual alignment from FATCAT are shown in Figure 8(b) and 8(e), respectively. Figure 8(c) shows the AFP alignment from TOPS++FATCAT, in which this method misses the N-terminal region and incorrectly aligns some of the C-terminal regions (see Figure 8(d)). However, the rigid_TOPS++FATCAT method produces an alignment of 108 aligned positions with optimal and chain RMSDs of 3.22 Å and 6.28 Å respectively. In general, TOPS comparison does not work well for alpha-rich proteins due to the lack of hydrogen bonds between SSEs [26]. The same is true for TOPS+ strings comparison to some extent; however, this method takes advantage of ligand-interaction information to compare protein domains more efficiently; for example the DNA binding motifs such as helix-turn-helix and helix-loop-helix can be easily recognized [28]. However, we have not explored that ligand pattern discovery option within the TOPS+ strings comparison in this paper. In addition, the TOPS+ strings alignment provides only a basic alignment; the scoring function to find the best alignment has not been optimized. These problems can be addressed in future development by considering the advanced TOPS+ and TOPS+ strings models based on helix-helix packing relationships and SSE-ligand interaction properties together with the right and left chiralities. Furthermore, the TOPS+ strings comparison can be optimized in both the comparison process as well as in the alignment process in order to take into account indels (insertion/deletion) of SSEs which exist in nature across the different members of the protein superfamilies [31].<br><br>
Discussion and conclusion
The overall results for all protein classes show that TOPS++FATCAT performance is only slightly lower (3%–7% AUC value difference) as compared to FATCAT while providing a significant, more than 10-fold speedup. The main reason for the discrepancies is that TOPS+ strings alignments occasionally misalign the secondary structure elements and subsequent FATCAT alignment, constrained by the TOPS+ strings alignment, cannot overcome the earlier errors. There is a clear trade-off between the runtime and the accuracy; limiting the pool of fragments being compared speeds up the algorithm but results in (slightly) lower accuracy. At the same time, these results offer clear suggestions for future development. Using a more advanced version of the TOPS+ strings comparison method would remove some of the false positives might be at a cost of significantly slowing the total performance of the TOPS++FATCAT method.<br><br>Authors' contributions
MV developed the TOPS++FATCAT algorithm, performed the calculations and prepared the figures, YY provided advice and oversight in the project, verified the code and provided FATCAT results for comparison, AG contributed to the original idea and to writing of the manuscript.<br><br>
<h3>pmcA2556924</h3>Are there sensitive subgroups for the effects of airborne particles?
Abstract
Recent studies have shown that particulate air pollution is a risk factor for hospitalization for heart and lung disease; however, little is known about what subpopulations are most sensitive to this pollutant. We analyzed Medicare hospital admissions for heart disease, chronic obstructive pulmonary disorders (COPD) and pneumonia in Chicago, Cook County, Illinois, between 1985 and 1994. We examined whether previous admissions or secondary diagnoses for selected conditions predisposed <span class="yellow">persons</span> to having a greater risk from air pollution. We also considered effect modification by age, sex, and race. We found that the air-pollution-associated increase in hospital admissions for cardiovascular diseases was almost doubled in subjects with concurrent respiratory infections. The risk was also increased by a previous admission for conduction disorders. For COPD and pneumonia admissions, diagnosis of conduction disorders or dysrhythmias increased the risk of particulate matter < 10 microm in aerodynamic diameter (PM(10))-associated admissions. <span class="yellow">Persons</span> with asthma had twice the risk of a PM(10)-associated pneumonia admission and <span class="yellow">persons</span> with heart failure had twice the risk of PM(10)-induced COPD admissions. The PM(10) effect did not vary by sex, age, and race. These results suggest that <span class="yellow">patients</span> with acute respiratory infections or defects in the electrical control of the heart are a risk group for particulate matter effects.
Articles <br><br> Are There Sensitive Subgroups for the Effects of Airborne Particles? Antonella Zanobetti,1 Joel Schwartz,1,2 and Diane Gold1,2 1Environmental <br><br> Epidemiology Program, Department of Environmental Health, Harvard School of Public Health, Boston, Massachusetts, USA; 2Channing Laboratory, Department of Medicine, Harvard Medical School and Brigham and <span class="yellow">Women</span>'s Hospital, Boston, Massachusetts, USA <br><br> Recent studies have shown that particulate air pollution is a risk factor for hospitalization for heart and lung disease; however, little is known about what subpopulations are most sensitive to this pollutant. We analyzed Medicare hospital admissions for heart disease, chronic obstructive pulmonary disorders (COPD) and pneumonia in Chicago, Cook County, Illinois, between 1985 and 1994. We examined whether previous admissions or secondary diagnoses for selected conditions predisposed <span class="yellow">persons</span> to having a greater risk from air pollution. We also considered effect modification by age, sex, and race. We found that the air-pollution-associated increase in hospital admissions for cardiovascular diseases was almost doubled in subjects with concurrent respiratory infections. The risk was also increased by a previous admission for conduction disorders. For COPD and pneumonia admissions, diagnosis of conduction disorders or dysrhythmias increased the risk of particulate matter < 10 �m in aerodynamic diameter (PM10)-associated admissions. <span class="yellow">Persons</span> with asthma had twice the risk of a PM10-associated pneumonia admission and <span class="yellow">persons</span> with heart failure had twice the risk of PM10-induced COPD admissions. The PM10 effect did not vary by sex, age, and race. These results suggest that <span class="yellow">patients</span> with acute respiratory infections or defects in the electrical control of the heart are a risk group for particulate matter effects. Key words: effect modification, hospital admissions, particulate air pollution. Environ Health Perspect 108:841�845 (2000). [Online 28 July 2000] http://ehpnet1.niehs.nih.gov/docs/2000/108p841-845zanobetti/abstract.html <br><br> Particulate air pollution has been associated with increases in daily deaths and hospital admissions in studies all over the world (1�15). These associations are now well documented but little is known, as yet, of the characteristics of <span class="yellow">persons</span> that put them at increased risk of adverse events related to particulate air pollution. This has been identified as a key data gap (16). Schwartz and Dockery (17) reported that <span class="yellow">persons</span> older than 65 years of age had a somewhat increased risk of death, and this has been confirmed in other studies (18). A more detailed examination of particulate matter-related risk by deciles of age (19) showed the risk beginning to increase at approximately 40 years of age and reaching its maximum for those 75 years of age and older. In addition to age, several studies suggest that <span class="yellow">persons</span> with respiratory illness are at increased risk for cardiovascular effects associated with air pollution. An examination of death certificates on high- and low-air pollution days reported a substantial difference in the proportion of deaths from cardiovascular causes that had respiratory disease as a contributing cause of death (19). A recent follow-up study of a cohort of <span class="yellow">persons</span> with chronic obstructive pulmonary disease (COPD) in Barcelona, Spain, found an association between particulate air pollution and all-cause mortality in the cohort (20). The magnitude of the risk per microgram per cubic meter of exposure was substantially greater than that for the general population. Environmental Health Perspectives <br><br> Controlled exposure of animals with chronic bronchitis and control animals to concentrated air particles also demonstrated a potentiating effect of chronic lung disease in the response to airborne particles (21). This has led to the hypothesis that the cardiovascular effects of air pollution are predominantly in <span class="yellow">persons</span> with chronic lung disease. There has been even less done to examine potential modifiers of the effects of airborne particles on hospital admissions. The existing literature on comorbidity shows that comorbidity per se seems to increase the risk of adverse outcomes (22�30). Little is known about the role of these comorbidities as effect modifiers for the effects of air pollution. This study uses data from the Medicare system to examine potential short-term and long-term medical conditions that may increase a <span class="yellow">person</span>'s risk of hospital admissions associated with particulate air pollution. In addition, we examine potential effect modification by age, race, and sex. <br><br> Materials and Methods Health data. The Health Care Financing Administration (Baltimore, MD) maintains records of every hospital admission for Medicare <span class="yellow">participants</span> in the United States. <span class="yellow">Persons</span> in this database have a unique identifier. Using this identifier, we traced every hospital admission for heart and lung disease for each <span class="yellow">person</span> in Cook County, Illinois, between 1985 and 1994. We chose Cook County because it is the most populous <br><br> county in the United States with daily monitoring for particulate matter with aerodynamic diameter < 10 �m (PM10). The data were then analyzed to look at effect modification by concurrent and preexisting conditions as well as by age, race, and sex. To establish a baseline risk, we computed daily counts of hospital admissions for cardiovascular disease (CVD) [International Classification of Disease, 9th edition, World Health Organization, Geneva (ICD-9) code 390�429], pneumonia (ICD-9 code 480�487), and COPD (ICD-9 code 490�496, excluding 493). The association between these daily counts and PM10 was examined for the years 1988�1994, when daily PM10 monitoring data were available in Chicago. Once our baseline risks were established, we examined three classes of potential effect modifiers. First, we looked at whether previous admissions for selected conditions predisposed <span class="yellow">persons</span> to having a greater risk from air pollution. For each of the three admission categories (CVD, pneumonia, and COPD), we considered 10 causes (defined by a previous admission) as effect modifiers: COPD (ICD-9 code 490�496 except 493), asthma (ICD-9 code 493), acute bronchitis (ICD-9 code 466), acute respiratory illness (ICD-9 code 460�466), pneumonia (ICD-9 code 480�487), CVD (ICD-9 code 390�429), myocardial infarction (ICD-9 code 410), congestive heart failure (ICD-9 code 428), conduction disorders (ICD-9 code 426), and dysrhythmias (ICD9 code 427). To test the hypothesis that <span class="yellow">persons</span> with these conditions had higher risks of subsequent PM10-related admissions, we computed separate daily counts of admissions for our three target causes, stratified by whether or not the <span class="yellow">person</span> admitted had been previously admitted for the hypothesized predisposing condition. Separate analyses were then performed within each strata to see if the effects of PM10 differed by strata. Address correspondence to A. Zanobetti, Department of Environmental Health, Environmental Epidemiology Program, Harvard School of Public Health, 665 Huntington Avenue, Boston, MA 02115 USA. Telephone: (617) 4324642. Fax: (617) 277-2382. E-mail: azanob@ sparc6a.harvard.edu Supported by NIEHS grant ES07937. Received 18 January 2000; accepted 18 April 2000. <br><br> � VOLUME 108 | NUMBER 9 | September 2000 <br><br> 841 <br><br> Articles <br><br> � <br><br> Zanobetti et al. <br><br> The second set of potential predisposing conditions included secondary diagnoses associated with the index admission. These could represent the presence of a chronic condition (e.g., COPD) that has not resulted in a previous hospital admission. They could also represent acute conditions that may have increased the subjects' sensitivity to air pollution. For example, if respiratory infections modified the effect of particulate matter on the cardiovascular health of <span class="yellow">persons</span> with underlying heart disease, then the risk of a hospital admission for heart disease might be different in <span class="yellow">persons</span> with infections. If this were true, then the risk ratio of a 10-�g/m3 increase of PM10 on cardiovascular admissions of <span class="yellow">persons</span> with a concurrent respiratory infection would be different from the ratio in <span class="yellow">persons</span> without respiratory infection. To test these hypotheses, we computed separate daily counts of admissions for events with and without the concurrent conditions hypothesized to increase sensitivity to air pollution. These were taken as the same 10 conditions in the first analysis with certain exclusions for pairing that would be illogical. That is, the concurrent diagnosis of a specific cardiac condition was not treated as an effect modifier for admissions for any cardiovascular condition. Likewise, pneumonia and COPD were not possible concurrent conditions for each other. The third set of predisposing conditions considered was being older than 75 years of age, nonwhite, and female. These were examined for all three outcomes. We obtained weather data for O'Hare Airport from the EarthInfo CD-ROM (EarthInfo CD NCDC Surface Airways, EarthInfo Inc., Boulder, CO), and we obtained air pollution data from the U.S. Environmental Protection Agency Aerometric Information Retrieval System network (31). <br><br> running-line smoother, loess (35), was chosen to estimate the smooth function. To control for weather variables and day of the week, we chose the smoothing parameter that minimized the Akaike's information criterion (36). To model seasonality we chose the smoothing parameter that minimized the sum of the autocorrelation of the residuals while removing seasonal patterns. Two autoregressive terms (37) were added in the model to eliminate the remaining serial correlation from the residuals. We used the mean of PM10 on the day of the admission and the day before the admission as our exposure variable. This gives results that are similar to those obtained fitting a full distributed lag model (38). PM10 was treated linearly. Our baseline models used the daily counts of CVD, pneumonia, and COPD admissions as outcomes. We then subdivided those counts by the presence or absence of the potential effect modifier and reestimated our regressions on those subgroups. We considered effect modification to be indicated when the estimates of PM10 in the group with the condition was outside of the 95% confidence interval (CI) of the effect estimate in <span class="yellow">persons</span> without the condition. <br><br> Results Table 1 shows the mean daily admissions for COPD, cardiovascular, and pneumonia both overall and in the presence of the potential effect modifiers. For some effect modifiers such as conduction disorders or myocardial infarctions, the counts in conjunction with our respiratory outcomes are <br><br> low, which limits power. In general, the numbers are lower for examining effect modification by previous admissions than for effect modification by concurrent diagnosis. This is as expected because many clinically relevant comorbidities may never have resulted in a hospital admission. Table 2 shows the 25th, 50th, and 75th percentile values for the environmental variables. The mean value for PM10 is 33 �g/m3. The daily values for PM10 were computed as the average of 10 monitors, two of which measured PM10 almost every day and the others less frequently (38). Table 3 shows the mean daily counts of CVD, COPD, and pneumonia by sex, age groups, and race. The distribution by sex is almost even, although the counts of admissions for males are generally lower (approximately 10%) than for females, particularly for cardiovascular diseases. The counts of CVD, COPD, and pneumonia admissions were similar for <span class="yellow">people</span> 65�75 or 75 years of age and older. Tables 4�6 show the results for the effect PM10 overall and stratifying by concurrent diagnosis and previous admissions. These are expressed as the percentage increase for 10 �g/m3 PM10. Table 4 shows the results for CVD. A 10-�g/m3 increase in PM10 was associated with a 1.31% (5% CI, 0.97%; 95% CI, 1.66%) increase in hospital admissions for heart disease in all elderly <span class="yellow">persons</span>. A concurrent (not previous) diagnosis of COPD modified the risk of PM10-associated admissions for heart disease. However, significant associations were still seen between PM10 <br><br> Table 1. Mean daily counts of admissions, Chicago 1986�1994, for COPD, CVD, and pneumonia overall and by concurrent diagnosis and by previous admissions. By concurrent diagnosis COPD CVD Pneumonia Overall Respiratory disease Acute bronchitis Acute respiratory infections Pneumonia Asthma COPD Cardiovascular disease CVD Conduction disorders Cardiac dysrhythmias Congestive heart failure Myocardial infarction NA, not applicable. <br><br> By previous admissions COPD CVD Pneumonia 7.8 0.8 0.9 1.6 0.9 2.7 2.1 0.0 0.4 0.9 0.3 102.1 1.6 1.8 7.3 1.5 2.0 54.7 1.0 9.9 24.2 11.4 26.5 0.9 1.0 6.4 0.7 1.4 7.2 0.2 1.5 3.1 1.0 <br><br> Methods We analyzed the data with a generalized additive robust Poisson regression model (32). This approach has become the norm in such studies (14,33,34). In the generalized additive model the outcome is assumed to depend on a sum of nonparametric smooth functions for each variable that models the potential nonlinear dependence of daily admission on weather and season. The model is of the form: log[E(Yt)] = 0+ S1 (X1 )+... + Sp (Xp) where E(Yt) is the expected value of the daily count of admissions Yt and Si are the smooth functions of the covariates Xi. We examined temperature, previous day's temperature, relative humidity, barometric pressure, and day of week covariates. The locally weighted <br><br> 7.8 0.1 0.3 0.4 0.1 NA 4.7 0.2 1.4 1.8 0.1 <br><br> 102.1 0.9 1.3 4.0 1.8 13.4 NA NA NA NA NA <br><br> 26.5 0.3 0.3 NA 0.9 6.9 14.7 0.6 4.6 7.3 0.4 <br><br> Table 2. 25th, 50th, and 75th percentile values for the environmental variables in Chicago, 1988�1994. Temperature (�F) 35 51 67 Relative humidity 62 70 79 Barometric pressure 29.2 29.3 29.4 PM10 (�g/m3) 23 33 46 <br><br> Table 3. Mean daily counts of admissions by sex, race, and age groups, Chicago, 1986�1994. Group Overall Female Nonwhite Age > 75 years COPD 7.8 4.2 1.6 3.7 CVD 102.1 59.4 21.0 55.1 Pneumonia 26.5 14.7 5.2 17.4 <br><br> 842 <br><br> VOLUME <br><br> 108 | NUMBER 9 | September 2000 � Environmental Health Perspectives <br><br> Articles <br><br> � <br><br> Effects of particles on sensitive subgroups <br><br> and heart disease admissions in <span class="yellow">persons</span> without COPD listed as either a comorbidity or a cause of previous admission (Table 4). A significant association was also seen in <span class="yellow">persons</span> without any respiratory disease as a concurrent diagnosis, although the risk is much lower than in <span class="yellow">persons</span> with respiratory disease. However, the risk associated with PM10 was roughly doubled in subjects with concurrent respiratory infections and the risk estimates in those subjects were outside the 95% CI of the risk in <span class="yellow">patients</span> without concurrent respiratory infections. A previous admission for conduction disorders (e.g., heart block) increased the risk of a PM10-related subsequent admission for any heart condition, and a weaker indication of effect modification was seen for <span class="yellow">persons</span> with previous admission for dysrhythmias. In contrast heart failure and previous myocardial infarctions were highly insignificant as effect modifiers. Table 5 shows the results for COPD. Overall, there is a 1.89% (95% CI, 0.8�3.0) increase in COPD admissions for a 10�g/m3 increase in PM10. The results of the stratified analysis suggest that preexisting heart disease modifies Table 4. Percentage increase in hospital admissions for CVD in all <span class="yellow">persons</span> and by concurrent diagnosis and previous admissions. PM10 2.5% CI 97.5% CI All <span class="yellow">persons</span> 1.31 By concurrent diagnosis Respiratory disease All respiratory disease With 1.65 Without 0.98 Acute bronchitis With 2.50 Without 1.07 Acute respiratory infections With 2.71 Without 1.06 Pneumonia With 1.95 Without 1.03 COPD With 1.59 Without 1.08 By previous admissions Respiratory disease All respiratory disease With 1.18 Without 1.08 COPD With 1.48 Without 1.09 Asthma With 1.71 Without 1.08 Cardiovascular disease Conduction disorders With 2.89 Without 1.07 Cardiac dyshrethmias With 1.61 Without 1.04 0.97 1.66 <br><br> the risk of COPD admissions on high particle days. Previous admissions for any cardiovascular disease increased the risk of a PM10associated COPD admission approximately 2.5-fold. A previous heart failure admission caused an even more striking increase in the PM10 effect. Previous admissions for dysrhythmias and conduction defects were rare (Table 1) with no power to examine effect modifications. Listings as concurrent diagnoses were more common and here they joined heart failure in increasing the risk of PM 10 -associated COPD admissions. For COPD there was also some indication that concurrent pneumonia or an acute respiratory infection admission in the last year increased risk. The low numbers made these estimates less precise, however. The percentage increase in pneumonia admission (Table 6) for 10 �g/m3 PM10 is higher than for COPD or CVD with an increase of 2.34% (95% CI, 1.66�3.0). As with COPD, <span class="yellow">persons</span> with heart disease appeared at higher risk of pneumonia hospital admissions associated with particulate air pollution. Here diagnoses suggestive of impaired autonomic control of the heart, such as conduction disorders or dysrhythmias, were associated with increased risk for PM 10 effects on pneumonia admissions. Unlike COPD, no difference was seen for congestive heart failure. <span class="yellow">Persons</span> with asthma Table 5. Percentage increase in hospital admissions for COPD in all <span class="yellow">persons</span> and by concurrent diagnosis and previous admissions. PM10 2.5% CI 97.5% CI All <span class="yellow">persons</span> 1.89 By concurrent diagnosis Respiratory disease Pneumonia With 4.00 Without 1.51 Cardiovascular disease Conduction disorders With 2.34 Without 1.60 Cardiac dysrhythmias With 3.09 Without 1.43 Congestive heart failure With 2.90 Without 1.39 By previous admissions Respiratory disease Acute respiratory infections With 3.20 Without 1.70 Cardiovascular disease CVD With 2.90 Without 1.18 Congestive heart failure With 4.37 Without 1.14 Within 1 year 6.04 0.80 2.99 <br><br> had twice the risk of a PM10-induced pneumonia admission as <span class="yellow">persons</span> without asthma. Table 7 shows the results by sex, age, and race. None of the effect size estimates for any of the stratification variables were outside of the 95% CI for the opposite strata. There was a tendency for the effect of PM10 on CVD admissions to be higher for females, whereas the effect on pneumonia admissions was higher for males. In general, we found somewhat larger effects on whites compared to nonwhites, and for <span class="yellow">persons</span> older than 75 years of age compared to younger <span class="yellow">persons</span>. <br><br> Discussion In this analysis we examined whether the effect of PM10 on the risk of hospital admission for heart and lung disease was different depending on the presence of comorbidities. We found that PM10 was associated with hospital admissions for all three causes (CVD, COPD, and pneumonia) and we found not a general increase in PM10 related risk with comorbidities, but a specific pattern that is suggestive of potential mechanisms and consistent with other recent epidemiologic and toxicologic findings. One major finding of this study is that preexisting cardiovascular disease, particularly impaired autonomic control (conduction defects and dysrhythmias) and heart failure, substantially increased the risk of respiratory admissions associated with airborne particles. In fact, recent <span class="yellow">human</span> studies have shown that exposure to particulate air pollution is a risk factor for reduced heart rate variability (39�41). Reduced heart rate variability is an adverse response and a risk factor for arrhythmia. A new study of defibrillator discharges in <span class="yellow">patients</span> with implanted cardioverter defibrillators found that discharges were associated with air pollution (42). Exposure to combustion Table 6. Percentage increase in hospital admissions for pneumonia in all <span class="yellow">persons</span> and by concurrent diagnosis and previous admissions. PM10 All <span class="yellow">persons</span> By concurrent diagnosis Respiratory disease Asthma With Without Cardiovascular disease Conduction disorders With Without Cardiac dysrhythmias With Without By previous admissions Cardiovascular disease Cardiac dysrhythmias With Without 2.34 2.5% CI 97.5% CI 1.66 3.02 <br><br> 1.10 0.64 �0.47 0.76 0.18 0.76 0.55 0.72 0.85 0.75 <br><br> 2.20 1.33 5.55 1.37 5.30 1.37 3.36 1.35 2.34 1.41 <br><br> �0.45 0.47 �4.42 0.58 0.64 0.33 0.77 0.24 <br><br> 8.65 2.57 9.59 2.64 5.60 2.55 5.08 2.55 <br><br> 0.45 0.76 �0.40 0.78 �0.43 0.77 0.22 0.76 0.75 0.72 <br><br> 1.91 1.41 3.40 1.40 3.89 1.39 5.63 1.38 2.48 1.36 <br><br> 4.18 2.07 7.92 1.99 � � <br><br> 1.01 1.46 4.28 1.37 � � <br><br> 7.46 2.69 11.69 2.61 � � <br><br> �1.38 0.66 0.99 �0.01 1.43 0.05 2.10 <br><br> 8.01 2.76 4.85 2.39 7.40 2.24 10.14 <br><br> 3.47 2.08 <br><br> 1.21 1.45 <br><br> 5.79 2.71 <br><br> Increases are for a 10-�g/m3 increase in PM10. <br><br> Increases are for a 10-�g/m3 increase in PM10. <br><br> Increases are for a 10-�g/m3 increase in PM10. <br><br> Environmental Health Perspectives <br><br> � VOLUME 108 | NUMBER 9 | September 2000 <br><br> 843 <br><br> Articles <br><br> � <br><br> Zanobetti et al. <br><br> Table 7. Effect modification by sex, race, and age groups for 10 �g/m3 PM10. % All <span class="yellow">persons</span> Male Female White Non-white Age > 75 Age  75 1.89 1.34 2.19 1.65 1.07 2.20 1.33 COPD (95% CI) (0.80, 2.99) (�0.14, 2.84) (0.81, 3.59) (0.51, 2.81) (�1.11, 3.3) (0.72, 3.69) (0.03, 2.65) % 1.31 1.07 1.21 1.20 0.70 1.28 0.93 CVD (95% CI) (0.97, 1.66) (0.62, 1.51) (0.83, 1.6) (0.86, 1.55) (0.1, 1.3) (0.88, 1.69) (0.51, 1.35) % 2.34 2.65 1.91 2.45 1.91 2.12 2.52 Pneumonia (95% CI) (1.66, 3.02) (1.81, 3.5) (1.11, 2.72) (1.77, 3.14) (0.69, 3.14) (1.38, 2.86) (1.57, 3.48) <br><br> Figures shown are the percentage increase in admissions (95% CI). <br><br> particles has also been associated with arrhythmia in an animal model (43) and changes in ST segments have been noted as well (44). This is the first study to suggest <span class="yellow">persons</span> with defects in the electrical control of the heart are also at higher risk of respiratory illness after exposure to airborne particles. These data also suggest that <span class="yellow">persons</span> admitted to hospitals for pneumonia during an air pollution episode may be at high risk for clinically significant conduction disorders during that hospital admission. <span class="yellow">Patients</span> with congestive heart failure were at greater risk of hospital admissions for COPD in association with airborne particles. Heart failure and COPD is not an uncommon combination. The finding that these <span class="yellow">patients</span> are at higher risk for admissions associated with particulate air pollution is new but is also consistent with several other recent reports. The spontaneous hypertensive <span class="blue">rat</span> develops a model of heart failure, and recent studies have reported greater sensitivity to particulate air pollution in these <span class="yellow">rats</span>. These include both electrocardiogram abnormalities (44) and pulmonary toxicity (45,46). Similarly, in an epidemiologic study, Hoek et al. (47), found a higher relative risk of death with an increase in PM10 for congestive heart failure deaths than other deaths. The potential role of COPD in those heart failure deaths was not examined. Another consistent pattern in our data is of acute respiratory infections increasing susceptibility to airborne particles. Acute bronchitis, or more generally acute upper respiratory illnesses, as well as pneumonia, increased susceptibility to particle-associated admissions for CVD and COPD. The notion that air pollution exacerbates acute respiratory infections is well supported by studies which report associations between airborne particles and hospital admissions for respiratory infections (48,49). Zelikoff et al. (50) exposed <span class="yellow">rats</span> infected with streptococcus to concentrated air particles and reported a significant increase in bacterial burdens and in the extent of pneumonia compared to animals exposed to filtrated air. This suggests an impaired immune response. Similarly, exposure to combustion <br><br> particles enhances influenza infections in <span class="yellow">mice</span> (51). An impaired defense to respiratory infection is a major reason that <span class="yellow">persons</span> with COPD require hospital admission. If airborne particles result in further impairment the effect modification we observe makes good sense. The effect modification for heart disease admissions is more relevant. This modification is consistent with the earlier report of Schwartz (19), who found greater reports of respiratory complications on death certificates with an underlying cause of heart disease if the death occurred on a day with high levels of airborne particles. Although airborne particle exposure has been associated with increased exacerbation of asthma (2,12,48,52�59), this paper is the first to suggest that asthmatics are more susceptible to PM10-induced pneumonia exacerbation or to cardiovascular effects. The effects on pneumonia admissions are plausible, given the impaired ability to fight off infections in asthmatics with mucus plugs and the evidence the airborne particles impair the lungs' ability to fight off bacterial and viral infections, as noted earlier. The increased cardiovascular sensitivity, albeit weaker, is interesting. If airborne particles affect the cardiovascular system via the role of the lung in autonomic control, it is possible that asthmatics would be more sensitive to those effects. Animal models of asthma showed that combustion particles enhance the asthmatic response to aeroallergen challenges (59). This suggests an enhancement of pulmonary response in asthmatics. On the other hand, the diagnosis of asthma is problematic in the elderly, and crossover with COPD is possible. The possibility that this explains our results is reduced by our failure to find previous hospital admission for COPD was an effect modifier for the effect of particles on cardiovascular admissions. We must acknowledge several potential limitations of this study. First, we considered only previous admissions that occurred within Cook County. Hence <span class="yellow">persons</span> with previous admissions elsewhere would be misclassified to our reference group. The effect of this would be to reduce the difference in PM 10 effect between the two groups. VOLUME <br><br> Nevertheless, we identified some interesting interactions. We cannot exclude the possibility that there are areas we missed for this reason. We also examined interactions in a log relative risk model, which is inherently multiplicative. Although we believe this is justified because doubling the population exposed would be expected to double the pollution associated admissions, it results in a more conservative definition of interaction than would an additive risk model. Finally, our exposure is clearly measured with error. Most of this error is Berkson error (60) and hence will introduce no bias, and Zeger et al. (60) showed that the remaining error would have to have pathologic correlations with other variables to result in an upward bias. Another important result from this study, of course, is an estimate of the magnitude of the effect of airborne particles on public health. The PM10 concentrations in Chicago during this period were associated with approximately 1,600 additional admissions per year for heart disease, 740 additional admissions per year for pneumonia, and 170 additional admissions per year for COPD. These are not trivial increases in serious morbidity. The results of our study should be replicated in additional cities, although they do begin to fill in some missing information about the effects of airborne particles on health. More generally airborne particles have been associated with a broad range of systemic changes including heart rate variability (39�41), increased peripheral neutrophils (61�63), increased plasma viscosity (64), an increase in blood pressure (65), and the outcomes mentioned previously. The role of these systemic changes as potential sources of the specific effect modifications we have seen should be an area of fruitful research in the future. REFERENCES AND NOTES 1. Katsouyanni K, Touloumi G, Spix C, Schwartz J, Balducci F, Medina S, Rossi G, Wojtyniak D, Sunyer J, Bacharova L, et al. Short term effects of ambient sulphur dioxide and particulate matter on mortality in 12 European cities: results from time series data from the APHEA project. Br Med J 314:1658�1663 (1997). Pope CA, Dockery DW, Schwartz J. Review of epidemiologic evidence of health effects of particulate air pollution. Inhal Toxicol 7:1�18 (1995). Schwartz J. Air pollution and daily mortality: a review and meta analysis. Environ Res 64:36�52 (1994). Dominici F, Samet J, Zeger SL. Combining evidence on air pollution and daily mortality from the largest 20 US cities: a hierarchical modeling strategy. R Stat Soc Ser A, in press. Burnett RT, Dales RE, Raizenne ME, Krewski D, Summers PW, Roberts GR, Raad-Young M, Dann T, Brooke T. Effects of low ambient levels of ozone and sulfates on the frequency of respiratory admissions to Ontario hospitals. Environ Res 65:172�194 (1994). Anderson HR, Spix C, Medina S, Schouten JP, Castellsague J, Rossi G, Zmirou D, Touloumi G, Wojtyniak B, Ponka A, et al. Air pollution and daily admissions for <br><br> 2. <br><br> 3. 4. <br><br> 5. <br><br> 6. <br><br> 844 <br><br> 108 | NUMBER 9 | September 2000 � Environmental Health Perspectives <br><br> Articles <br><br> � <br><br> Effects of particles on sensitive subgroups <br><br> 7. <br><br> 8. 9. <br><br> 10. <br><br> 11. <br><br> 12. <br><br> 13. <br><br> 14. <br><br> 15. <br><br> 16. <br><br> 17. <br><br> 18. <br><br> 19. 20. <br><br> 21. <br><br> 22. <br><br> 23. <br><br> 24. <br><br> 25. <br><br> 26. <br><br> 27. <br><br> 28. <br><br> 29. <br><br> 30. <br><br> chronic obstructive pulmonary disease in 6 European cities: results from the APHEA project. Eur Respir J 10:1064�1071 (1997). Schwartz J. Short term fluctuations in air pollution and hospital admissions of the elderly for respiratory disease. Thorax 50:531�538 (1995). Schwartz J. Air pollution and hospital admissions for heart disease in eight U.S. counties. Epidemiology 10:17�22 (1999). Schwartz J. Air pollution and hospital admissions for the elderly in Minneapolis. Arch Environ Health 49:366�374 (1994). Schwartz J. Air pollution and hospital admissions for the elderly in Birmingham, Alabama. Am J Epidemiol 139:589�598 (1994). Schwartz J. Air pollution and hospital admissions for the elderly in Detroit, MI. Am J Respir Crit Care Med 150:648�655 (1994). Pope CA III. Respiratory disease associated with community air pollution and a steel mill, Utah valley. Am J Public Health 79:623�628 (1989). Saldiva PH, Pope CA, Schwartz J, Dockery DW, Lichtenfels AJ, Salge JM, Barone I, Bohm GM. Air pollution and mortality in elderly <span class="yellow">people</span>: a time series study in Sao Paulo, Brazil. Arch Environ Health 50:159�163 (1995). Schwartz, J. Air pollution and hospital admissions for cardiovascular disease in Tucson. Epidemiology 8:371�177 (1997). Delfino RJ, Murphy Moulton AM, Becklake MR. Emergency room visits for respiratory illnesses among the elderly in Montreal: association with low level ozone exposure. Environ Res 76:67�77 (1998). National Research Council. Research Priorities for Airborne Particulate Matter. Washington, DC:National Academy Press, 1998. Schwartz J, Dockery DW. Increased mortality in Philadelphia associated with daily air pollution concentrations. Am Rev Respir Dis 145:600�604 (1992). Samet JM, Zeger SL, Berhane K. The association of mortality and particulate air pollution. In: Particulate Air Pollution and Daily Mortality. The Phase I Report of the Particle Epidemiology Evaluation Project. Boston, MA:Health Effects Institute, 1995. Schwartz J. What are <span class="yellow">people</span> dying of on high air pollution days? Environ Res 64:26�35 (1994). Sunyer J, Schwartz J, Tobias A, MacFarlane D, Garcia J, Anto JM. <span class="yellow">Patients</span> with chronic obstructive pulmonary disease are a susceptible population of dying due to urban particles. Am J Epidemiol 151(1):50�56 (2000). Godleski JJ, Sioutas C, Katler M, Koutrakis P. Death from inhalation of concentrated air particles in animal models of pulmonary disease. Am J Respir Crit Care Med 153:A15 (1996). Matsui K, Goldman L. Comorbidity as a correlate of length of stay for hospitalized <span class="yellow">patients</span> with acute chest pain. J Gen Intern Med 11:262�268 (1996). Charlson M, Szatrowshi TP, Peterson J, Gold J. Validation of a combined comorbidity index. J Clin Epidemiol 47:1245�1251 (1994). Monane M, Kanter DS, Glynn RJ, Avorn J. Variability in length of hospitalization for stroke. The role of managed care in an elderly population. Arch Neurol 53:848 (1996). Hallstrom AP, Cobb LA, Yu BH. Influence of comorbidity on the outcome of <span class="yellow">patients</span> treated for out-of-hospital ventricular fibrillation. Circulation 93:2019�2022 (1996). Malenka DJ, Mclerran D, Roos N, Fisher ES, Wennberg JE. Using administrative data to describe case-mix: a comparison with the medical record. J Clin Epidemiol 47:1027�1032 (1994). Romano PS, Roos LL, Jollis JG. Adapting a clinical comorbidity index for use with ICD-9-CM administrative data: differing perspectives. J Clin Epidemiol 46:1075�1079 (1993). Deyo RA, Cherkin DC, Ciol MA. Adapting a clinical comorbidity index for use with ICD-9CM administrative databases. J Clin Epidemiol 45:613�619 (1992). Charlson ME, Pompei P, Ales KL, MacKenzie CR. A new method of classifying prognostic comorbidity in longitudinal studies: development and validation. J Chronic Dis 40:373�383 (1987). Librero J, Peir� S, Ordi�ana R. Chronic comorbidity and outcomes of hospital care: length of stay, mortality, and readmission at 30 and 365 days. J Clin Epidemiol 52:171�179 (1999). <br><br> 31. Nehls GJ, Akland GG. Procedures for handling aerometric data. J Air Pollut Control Assoc 23:180�184 (1973). 32. Hastie T, Tibshirani R. Generalized Additive Models. London:Chapman and Hall, 1990. 33. Schwartz J. Generalized additive models in epidemiology. In: International Biometric Society, Invited Papers. 17th International Biometric Conference, 8�12 August 1994, Hamilton, Ontario, Canada. Washington, DC:International Biometric Society, 1994;55�80. 34. Rossi G, Vigotti MA, Zanobetti A, Repetto F, Giannelle V, Schwartz J. Air pollution and cause specific mortality in Milan, Italy, 1980�1989. Arch Environ Health 54:158�164 (1999). 35. Cleveland WS, Devlin SJ. Robust locally-weighted regression and smoothing scatterplots. J Am Stat Assoc 74:829�836 (1988). 36. Akaike H. Information theory and an extension of the maximum likelihood principal. In: 2nd International Symposium on Information Theory (Petrov BN, Csaki F, eds). Budapest:Akademiai Kaiado, 1973;267�281. 37. Brumback BA, Ryan LM, Schwartz J, Neas LM, Stark PC, Burge HA. Transitional regression models with application to environmental time series. J Acoust Soc Am 95(449):16�28 (2000). 38. Schwartz J. The distributed lag between air pollution and daily deaths. Epidemiology 11:320�326 (2000). 39. Pope CA III, Verrier RL, Lovett EG, Larson AC, Raizenne ME, Kanner RE, Schwartz J, Villegas GM, Dockery DW. Heart rate variability associated with particulate air pollution. Am Heart J 138:890�899 (1999). 40. Gold DR, Litonjua A, Schwartz J, Lovett E, Larson A, Nearing B, Allen G, Verrier M, Cherry R, Verrier R. Ambient pollution and heart rate variability. Circulation 101(11):1267�1273 (2000). 41. Liao D, Creason J, Shy C, Williams R, Watts R, Zweidinger R. Daily variation of particulate air pollution and poor cardiac autonomic control in the elderly. Environ Health Perspect 107:521�525 (1999). 42. Peters A, Liu E, Verrier RL, Schwartz J, Gold DR, Mittelman M, Baliff J, Allen G, Monahan K, Dockery DW. Air pollution and incidences of cardiac arrhythmia. Epidemiology 11(1):11�17 (2000). 43. Godleski JJ, Verrier RL, Koutrakis P, Catalano P. Mechanisms of Morbidity and Mortality from Exposure to Ambient Air Particles. Health Effects Institute Research Report 91. Cambridge, MA:Health Effects Institute, 2000. 44. Watkinson WP, Campen MJ, Kodavanti UP, Ledbetter AD, Costa DL. Effects of inhaled residual oil <span class="yellow">fly</span> ash particles on electrocardiographic and thermoregulatory parameters in normal and compromised <span class="yellow">rats</span> [Abstract]. Am J Respir Crit Care Med 157:A150 (1998). 45. Watkinson WP, Campen MJ, Costa DL. Cardiac arrhythmia induction after exposure to residual oil <span class="yellow">fly</span> ash particles in a <span class="yellow">rodent</span> model of pulmonary hypertension. Toxicol Sci 41:209�216 (1998). 46. Kodavanti UP, Jackson MC, Richards J, Ledbetter A, Costa DL. Differential pulmonary responses to inhaled emission particulate matter (PM) in systemically hypertensive vs. normotensive <span class="yellow">rats</span> [Abstract]. Am J Respir Crit Care Med 157:A260 (1998). 47. Hoek G, Brunekreef B, van Wijnen JH. Cardiovascular mortality response to air pollution is strongest for heart failure and thrombotic causes of death [Abstract]. Epidemiology 10:S177 (1999). 48. Bates DV, Szito R. Hospital admissions and air pollutants in southern Ontario: the acid summer haze effect. Environ Res 43:317�331 (1987). 49. Pope CA III. Respiratory disease associated with community air pollution and a steel mill, Utah valley. Am J Public Health 79:623�628 (1989). 50. Zelikoff JT, Nadziejko C, Fang T, Gordon C, Premdass C, Cohen MD. Short term, low-dose inhalation of ambient particulate matter exacerbates ongoing pneumococcal infections in Streptococcus Pneumoniae-infected rates. In: Proceedings of the Third Colloquium on Particulate Air Pollution and <span class="yellow">Human</span> Health (Phalen RF, Bell YM, eds). Irvine, CA:Air Pollution Health Effects Laboratory, University of California, 1999;8-94�8-101. 51. Clarke RW, Hemenway DR, Frank R, Kleeberger SR, Longphre MV, Jakab GJ. Particle associated sulfate exposure enhances <span class="yellow">murine</span> influenza mortality [Abstract]. Am J Respir Crit Care Med 155:A245 (1997). <br><br> 52. Pope CA, Dockery DW, Spengler JD, Raizenne ME. Respiratory health and PM10 pollution: a daily time series analysis. Am Rev Respir Dis 144:668�674 (1991). 53. Schwartz J, Koenig J, Slater D, Larson T. Particulate air pollution and hospital emergency visits for asthma in Seattle. Am Rev Respir Dis 147:826�831 (1993). 54. Thurston GD, Ito K, Lippman M, Hayes CG, Bates DV. Respiratory hospital admissions and summertime haze air pollution in Toronto, Ontario: consideration of the role of acid aerosols. Environ Res 65:271�290 (1994). 55. Norris G, YoungPong SN, Koenig JQ, Larson TV, Sheppard L, Stout JW. An association between fine particles and asthma emergency department visits for <span class="yellow">children</span> in Seattle. Environ Health Perspect 107:489�493 (1999). 56. Hamada K, Goldsmith CW, Kobzik L. Air pollutant aerosols allow airway sensitization to allergen in juvenile <span class="yellow">mice</span>. Am J Resp Crit Care Med A28 (1999). 57. Lambert AL, Selgrade M, Dong W, Winsett D, Gilmour M. Enhanced allergic sensitization by residual oil <span class="yellow">fly</span> ash particles is mediated by soluble metal constituents [Abstract]. Am J Respir Crit Care Med 159:A26 (1999). 58. Dailey LA, Madden MC, Devlin RB. Do airway epithelial cells from normal and asthmatic donors respond differently to an in vitro challenge with a particulate pollutant? [Abstract]. Am J Respir Crit Care Med 157:A598 (1998). 59. Gilmour MI, Winsett D, Selgrade MJ, Costa DL. Residual oil <span class="yellow">fly</span> ash exposure enhances allergic sensitization to house dust mite in <span class="yellow">rats</span> and augments immune-mediated inflammation [Abstract]. Am J Respir Crit Care Med 155:A244 (1997). 60. Zeger SL, Thomas D, Dominici F, Samet JM, Schwartz JM, Dockery D, Cohen A. Exposure measurement error in time�series studies of air pollution: concepts and consequences. Environ Health Perspect 108:419�426 (2000). 61. Salvi S, Blomberg A, Rudell B, Kelly F, Sandstrom T, Holgate ST, Frew A. Acute inflammatory responses in the airways and peripheral blood after short-term exposure to diesel exhaust in healthy <span class="yellow">human</span> volunteers. Am J Respir Crit Care Med 159:702�709 (1999). 62. Tan WC, van Eeden S, Qiu DW, Liam BL, Dyachokova Y, Hogg JL. Particulate air pollution, bone marrow stimulation and the pathogenesis of excess cardiovascular and pulmonary deaths. Am J Respir Crit Care Med 155:1441�1447 (1997). 63. Gordon T, Nadziejko C, Schlesinger R, Chen LC. Pulmonary and cardiovascular effects of acute exposure to concentrated ambient particulate matter in <span class="yellow">rats</span>. Toxicol Lett 96�97:285�288 (1998). 64. Peters A, Doering A, Wichmann HE, Koenig W. Increased plasma viscosity during an air pollution episode: a link to mortality? Lancet 349(9065):1582�1587 (1997). 65. Peters A, Stieberv J, Doering A, Wichmann HE. Is systolic blood pressure associated with air pollution? [Abstract]. Epidemiology 10(4):S177 (1999). <br><br> Environmental Health Perspectives <br><br> � VOLUME 108 | NUMBER 9 | September 2000 <br><br> 845 <br><br>  <h3>pmcA280693</h3>p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells and its expression is induced through pathways involved in growth inhibition and repressed by factors promoting cell growth
Abstract
Background
p8 is a stress-induced protein with multiple functions and biochemically related to the architectural factor HMG-I/Y. We analyzed the expression and function of p8 in pancreatic cancer-derived cells.<br><br>Methods
Expression of p8 was silenced in the <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 by infection with a retrovirus expressing p8 RNA in the antisense orientation. Cell growth was measured in control and p8-silenced cells. Influence on p8 expression of the induction of intracellular pathways promoting cellular growth or growth arrest was monitored.<br><br>Results
p8-silenced cells grew more rapidly than control cells transfected with the empty retrovirus. Activation of the Ras→Raf→MEK→ERK and JNK intracellular pathways down-regulated p8 expression. In addition, the MEK1/2 inhibitor U0126 and the JNK inhibitor SP600125 up-regulates expression of p8. Conversely, p38 or TGFβ-1 induced p8 expression whereas the specific p38 inhibitor SB203580 down-regulated p8 expression. Finally, TGFβ-1 induction was in part mediated through p38.<br><br>Conclusions
p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells. p8 expression is induced through pathways involved in growth inhibition and repressed by factors that promote cell growth. These results suggest that p8 belongs to a pathway regulating the growth of pancreatic cancer cells.<br><br><br><br>Background
While studying the molecular response of the injured pancreas, we identified a new gene, called p8, whose expression is strongly induced during the acute phase of pancreatitis [1]. Further experiments have shown that p8 mRNA is activated in almost all cells in response to several stresses [2], including minimal stresses such as after routine change of the culture medium in the absence of any added substance [3], indicating that p8 is a ubiquitous protein induced by cellular stress. The p8 gene was cloned in <span class="yellow">human</span>, <span class="blue">rat</span>, <span class="yellow">mouse</span>, and <span class="yellow">Xenopus laevis</span> [1,4-6], conceptually translated from the <span class="yellow">Drosophila melanogaster</span> genome or deduced from EST libraries (<span class="yellow">Bos taurus</span>, <span class="yellow">Xenopus tropicalis</span>, <span class="yellow">Zebrafish</span>, <span class="yellow">Orzzias latipes</span>, <span class="yellow">Bombyx mori</span> and <span class="yellow">Paralichthys olivaceous</span>). The overall degree of homology with <span class="yellow">human</span> p8 ranged from 81 to 40%. Secondary structure prediction methods indicated that within the homologous region of the eleven proteins, there is a basic Helix-Loop-Helix secondary structure motif, characteristic of some classes of transcription factors [1]. Even though a small protein such as p8 would not need a nuclear localization signal (NLS) to be transported to the nucleus, a clear NLS can be predicted for the eleven proteins comprising a bipartite domain of positively charged aminoacids. In addition, a nuclear/cytoplasmic location has been demonstrated for <span class="yellow">human</span> p8 upon overexpression of the recombinant protein and immunohistochemistry [4], and for recombinant <span class="yellow">Xenopus laevis</span> p8 fused to green fluorescent protein [6]. Homology searching in databases did not reveal significant similarity of p8 with other proteins of known function. However, biochemical properties of the mammalian p8 proteins are shared by some high mobility group proteins (HMG) [7], particularly by the HMG-I/Y family. The overall identity of <span class="yellow">human</span> p8 with <span class="yellow">human</span> HMG-I/Y is only about 35%, but the molecular mass, isoelectric point, hydrophilicity plot, the resistance to denaturation after heating at 100°C and the charge separation are very similar [8]. The p8 protein seems to bind DNA weakly, as shown by electrophoretic mobility shift assay, without preference for DNA sequences. Finally, <span class="yellow">human</span> p8 has also been shown to be a substrate for protein kinase A in vitro and phosphorylated p8 has a higher content of secondary structure and binding to DNA is highly increased [8]. An architectural role in transcription has been proposed for this protein, in analogy with the HMG-I/Y proteins, and a recent work seems to confirm this hypothesis [9].
Functions of p8 appear to be multiple and complex. For example, p8 mRNA expression was strongly induced in 3T3 cells upon TGFβ-1 treatment which in turn enhances the Smad-transactivating function responsible for TGFβ-1 activity [10]. We also found that p8 is involved in cell cycle regulation since p8-deficient embryonic fibroblasts grew more rapidly and incorporated more [3H] thymidine and BrdU than p8-expressing cells [11]. Moreover, expression of p8 in breast cancer-derived cells seems to mediate the inhibition of cell growth induced by 1,25-Dihydroxyvitamin D3 [12]. On the contrary, we also reported that p8 may promote cell growth when overexpressed in Cos-7, AR42J and HeLa cells [1,4]. In addition, p8 seems to be involved in other intracellular functions such as apoptosis since p8-expressing fibroblasts are more sensitive than p8-deficient fibroblasts to the apoptosis induced by DNA damage. Also, p8 is required for endothelin-induced mesangial cell hypertrophy in diabetic kidney, in a mechanism involving ERK, JNK and PI3 kinase [13]. p8 seems to play a functional role in the initiation of LHβ gene expression during embryonic cell differentiation [14]. Moreover, the <span class="yellow">Drosophila melanogaster</span> p8 homologue is involved in response to starvation and might be activated to stop cell growth in case of nutrient deprivation [15]. Finally, a particularly attractive role in tumour progression was recently proposed for p8 [16]. Fibroblasts obtained from p8-expressing or p8-deficient animals were transformed with a retroviral vector expressing both the rasV12 mutated protein and the E1A adenoviral oncogene. In soft-agar assays, transformed p8-expressing cells formed colonies at high frequency, as expected, but p8-deficient transformed fibroblasts were unable to form colonies. Similarly, transformed p8-expressing cells produced tumours in all athymic <span class="yellow">nude mice</span> when injected subcutaneously or intraperitoneally, whereas transformed p8-deficient fibroblasts did not. On the other hand, studies by another laboratory revealed that expression of the Com1 protein [17], which is identical to <span class="yellow">human</span> p8, mediates the growth of tumour cells after metastatic establishment in a secondary organ, indicating that activated expression of Com1/p8 in metastatic cells is required for tumour progression. These results strongly suggest that p8 is involved in the cellular pathway(s) required for tumour progression and metastasis.
Our aim is to check the relevance of p8 to cancer progression in <span class="yellow">human</span>. As a first step, we investigated in the present study the function of p8 in two cell lines derived from <span class="yellow">human</span> pancreatic cancer. We observed that inhibition of p8 expression increased the cells growth rate. In addition, activations of the Ras→Raf→MEK→ERK and JNK intracellular pathways, which promote the growth of pancreatic cells, down-regulated p8 expression, whereas activation of p38 or TGFβ-1, which inhibit cell growth, induced its expression. It was concluded that i/ p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cell lines, ii/ p8 expression is induced through pathways involved in growth inhibition and, conversely, repressed by factors that promote cell growth.<br><br>Results
p8 is silenced in pancreatic cancer cells by infection with a retrovirus expressing p8 RNA in the antisense orientation
Panc-1 and BxPc-3 pancreatic cells were chosen for this study because, on the one hand, both cells express higher level of p8 (Figure 1) and, on the other hand, because Panc-1 is wild-type for Smad4/DPC4 and mutated for K-ras, while BxPc-3 is Smad4/DPC4 mutated and K-ras wild-type [18,19], therefore representing different mechanisms of transformation and different genetic backgrounds. K-ras and Smad4/DPC4 mutations are the major mechanisms involved in pancreatic cancer development. We inhibited p8 expression in both Panc-1 and BxPc-3 pancreatic cells by infecting cells with a retrovirus expressing the p8 asRNA (antisense RNA) and carrying the puromycin resistance. The antibiotic-selected cells were analyzed by Western blotting to evaluate the intracellular amounts of p8 protein. As shown in Figure 2, the p8 protein was clearly visible in both Panc-1 and BxPc-3 pancreatic cells infected with the empty retrovirus but almost undetectable in cells infected with the retrovirus encoding the p8 asRNA showed, indicating that our anti-sense strategy is efficient to silence p8 gene expression in pancreatic cancer cells. Preliminary studies had been conducted to select the best strategy to inhibit p8 expression. We compared the efficacy of the stable transfection of a siRNA, using a retroviral expression vector to the asRNA strategy described above. In our hands, the antisense strategy worked best, as judged from Western blot assessment of p8 protein expression (data not shown).<br><br>p8-silenced pancreatic cells grow more rapidly
We compared in the two cell lines the influence on growth parameters of blocking p8 expression with the p8 asRNA. Figure 3 shows that both Panc-1 and BxPc-3 cells in which p8 has been silenced grew more rapidly than cells infected with the empty vector suggesting that inhibition by p8 of pancreatic cancer cell growth is independent from the mechanism of transformation and genetic background.<br><br>Serum-stimulated cellular growth down-regulates p8 expression
Fetal <span class="yellow">calf</span> serum, which contains a complex mix of growth factors, can be used as inductor of cell growth. As shown in Figure 4 expression of p8 mRNA was down-regulated in both Panc-1 and BxPc-3 when the cells were shifted from culture media containing 0.1% fetal <span class="yellow">calf</span> serum to media containing 10% FCS. p8 protein showed a similar behavior. These results show that p8 expression is down-regulated in growing pancreatic cells.<br><br>The Ras→Raf→MEK→ERK pathway down-regulates p8 expression in pancreatic cancer cells
Most <span class="yellow">human</span> pancreatic cancers harbor mutations in the K-ras oncogene, which happens relatively early in pancreatic tumorigenesis [20]. The oncogenic mutation of the K-ras gene stabilizes the Ras protein in a GTP-bound form, which is constitutively active and make the cells grow more rapidly. Contrary to the activated Ras protein, p8 inhibits cell growth (Figure 3). We looked whether the Ras→Raf→MEK→ERK pathway was also involved in the regulation of p8 expression, and which step(s) were critical. Figure 5 shown that expression of a mutated form of the Ras protein (rasV12) in BxPc-3 cells, which are wild-type for ras, resulted in decreased p8 mRNA concentration and protein level suggesting that the activated ras inhibits p8 expression. Figure 6 shows that overexpression of Raf, but not of Raf301 (a negative mutant of Raf), and of ERK also inhibited the expression of the p8-CAT construct in Panc-1 as well as in BxPc-3. Finally, the MEK1/2 specific inhibitor U0126 [21] activated p8 mRNA expression in pancreatic cells whether they carry mutated ras (Panc-1) or wild-type (BxPc-3). Similar results were observed when expression of the p8 protein was monitored by Western blotting (Figure 7).<br><br>Activation of the JNK pathway down-regulates p8 expression in pancreatic cancer cells
c-Jun NH2-terminal kinase (JNK) is another major MAPK pathway which converts extracellular signals into expression of specific target genes through phosphorylation and activation of transcription factors. JNK activation has been implicated in various, often opposite cellular responses, such as cell proliferation, transformation and apoptosis. As shown in Figure 8, overexpression of JNK down-regulates the gene reporter activity of the p8-CAT construct in Panc-1 cells. Similar results were found in BxPc-3 cells. Treatment of these cells with the JNK specific inhibitor SP600125 [22] up-regulates expression of the p8 mRNA and p8 protein (Figure 9). These results show that the JNK pathway is involved in the regulation of p8 expression.<br><br>The p38 pathway up-regulates p8 expression in pancreatic cancer cells
The p38 signal transduction pathway also plays an essential role in regulating several cell functions including growth, response to inflammation, differentiation and apoptosis. In fact, in pancreatic cancer cells, p38 is a strong inhibitor of proliferation [23] contrary to the Ras→Raf→MEK→ERK and JNK pathways. We therefore analyzed the putative role of the p38 pathway in regulating p8 expression in pancreatic cancer cells. Figure 10 shows that over-expression of the plasmid encoding p38 significantly increases p8-CAT activity in Panc-1 as well as in BxPc-3 cells. Then, cells were treated with SB203580, a specific inhibitor of p38 [24], and p8 expression was measured. p8 mRNA as well as the encoded protein were down-regulated after inhibition of the p38 activity (Figure 11). These results indicate that the p38 pathway is a positive regulator of p8 expression in pancreatic cancer cells.<br><br>TGFβ-1 up-regulates p8 expression in pancreatic cancer cells
The most prominent biological activity of TGFβ-1 is its potent inhibition of cell growth in a wide variety of cell types including pancreatic cells. TGFβ-1 signals are sent through two types of transmembrane serine/threonine kinase receptors. In fact, TGFβ-1 binds and brings together the type I and type II receptors. In the resulting complex, the constitutively active TGFβ-1 type II receptor phosphorylates the type I receptor, which then plays a major role in transducing the signal to downstream components to affect gene expression through phosphorylation of SMAD proteins. Phosphorylated receptor-regulated SMADs then form heteromeric complexes with the common partner SMAD4. These heteromeric complexes then move to the nucleus, where SMAD4 will bind DNA and contribute to transcriptional activation. In general, pancreatic cancer cells present with defects in TGFβ-1 signaling and are resistant to TGFβ-1-mediated growth suppression. Since TGFβ-1 and p8 are inhibitors of pancreatic cell growth we analyzed whether p8 could mediate, at least in part, the effect of TGFβ-1. First, we found that treatment of Panc-1 cells with TGFβ-1 increased p8 mRNA levels and p8 protein as judged by Western blot (Figure 12). Then, to confirm that overexpression is regulated at the transcriptional level, we analyzed the effect of some constructs expressing constitutively activated type I TGFβ receptor, dominant negative type II TGFβ receptor, a dominant negative of Smad4 and the wild-type Smad4 on the p8-CAT activity. As expected, the constitutively activated type I TGFβ receptor but not the dominant negative type II TGFβ receptor increased CAT activity. Also, expression of the Smad4, contrary to that of the negative mutant, induced p8 transcription (Figure 13). Together, these results indicate that p8 is positively regulated by TGFβ-1.
Beside the Smad proteins, TGFβ-1 also activates the p38 MAPK pathway in pancreas-derived cells, which may play an important role in TGFβ-1 induced genes [25]. Therefore, we analyzed the p38-dependent effect of TGFβ-1 on p8 transcription. As shown in Figure 13, inhibition of p38 activity with the SB203580 specific inhibitor decreased about 40% the activity of TGFβ-1 on the p8 promoter indicating that the effect of TGFβ-1 on p8 promoter is mediated by both p38-dependent and p38-independent pathways.<br><br>
Discussion
Pancreatic adenocarcinoma is the fourth leading cause of death from malignant diseases [26]. The aggressive nature of the neoplasia, the lack of early detection, and the limited response to treatments such as chemotherapy and radiotherapy contribute to the high mortality rate of the disease. Therefore, a better understanding of the molecular mechanism leading to pancreatic cancer remains a major goal because it may help proposing strategies for earlier diagnosis and better treatment. The most commonly altered genes in pancreatic adenocarcinoma are K-ras (75 to 100%), p16INK4a (95%), p53 (50 to 75%) and DPC4 (50%) [27-31]. Whereas K-ras is a proto-oncogene all the others are tumour suppressor genes. Additional genes have been found altered at lower frequency. Panc-1 and BxPc-3 pancreatic cells were chosen for this study because they both express high levels of p8 (Figure 1) and because they present with different mechanisms of transformation and genetic backgrounds, Panc-1 being wild-type for Smad4/DPC4 but mutated for K-ras and BxPc-3 mutated for Smad4/DPC4 and wild-type for K-ras [18,19]. This work presents evidences that p8 inhibits the growth rate of pancreatic cancer-derived cells and that the intracellular pathways promoting cell growth down-regulate p8 expression whereas those promoting growth arrest up-regulate its expression. Together, these results suggest that p8 is downstream of some cell growth regulators and therefore regulation of p8 expression or its activity could be used as a target for treating pancreatic cancer.
Silencing p8 expression was able to strongly promote cell growth in both cell types, Panc-1 and BxPc-3, suggesting that p8 may act downstream of the ras- or Smad4/DPC4-dependent ways. Also, we found that stimulating cell growth by the complex combination of growth factors contained in fetal <span class="yellow">calf</span> serum down-regulated expression of p8 whereas, on the contrary, treating the cells with TGFβ-1, which promotes cell cycle arrest, stimulates p8 expression. Therefore, p8 gene expression seems to be regulated in opposite directions by mechanisms promoting cell growth or cell cycle arrest. It is interesting to note that while p8 expression is under the control of cell growth regulatory pathways such as Ras→Raf→MEK→ERK, JNK, p38 and TGFβ-1, p8 can affect cell cycle progression, suggesting that p8 is a target for factors regulating pancreatic cell growth.
A mechanism by which p8 could regulate cell cycle progression in embryonic fibroblasts was previously proposed [11]. In fact, p8 seems to take action upstream from cyclin-dependent kinases because the intracellular levels and activities of Cdk2 and Cdk4 are decreased when p8 is expressed. Concomitantly, the cyclin-dependent kinase inhibitor p27 is expressed at a low level in p8-deficient cells which may explain the increased activity of Cdk2 and Cdk4. The mechanism by which p8 regulates the intracellular level of those proteins remains to be determined. However, because p8 is a transcriptional cofactor, it is possible that regulation of expression of these molecules takes place, at least in part, at the transcription level.
Interestingly, expression of p8 mRNA seems to be regulated in a cell type- and stimulus-specific manner since, for example, p38 can induce p8 expression in response to stress in fibroblasts [3] but not in renal mesangial cells treated with endothelin [13]. In pancreatic cancer-derived cells p38 seems to play a major role since it is involved in p8 activation as judged by transient transfection assays and using a specific p38 inhibitor (Figures 10 and 11). In addition, p38 is also involved in TGFβ-1-induced p8 expression because about 40% of the TGFβ-1 effect was abolished when p38 activity was specifically blocked (Figure 13). On the other hand, ERK and JNK are inducers of p8 expression in mesangial cells treated with endothelin, but not involved in the activation of p8 in response to stress in fibroblasts [3], and even repressors in pancreatic cells (Figures 5, 6, 7, 8 and 9). Finally, PI3 kinase is an inducer of p8 expression in both endothelin-mediated p8 activation in mesangial cells [13] and pancreatic cells (data not shown).
Based on these observations, overexpression of p8 could be considered a possible goal for treating pancreatic tumours, in order to limit their growth. However, we previously reported that p8 repression would prevent rasV12/E1A transformed fibroblasts from evolving as tumours in <span class="yellow">nude mice</span> [16]. This apparent contradiction needs to be resolved before considering p8 as a target for treating cancer progression.<br><br>Conclusions
In conclusion (see Figure 14), we report in this paper that inhibition of p8 expression by an anti-sense strategy increases the growth rate of both Panc-1 and BxPc-3 pancreatic cancer-derived cells. Moreover, ERK- and JNK-mediated pathways down-regulate p8 expression, whereas p38 and TGFβ-1 pathways induce p8 expression. Also, cell growth triggered by expression of a RAS mutated protein or by 10% fetal <span class="yellow">calf</span> serum induces down-regulation of p8 expression. Together, these results indicate that p8 is an intracellular cell growth inhibitor and that it is oppositely regulated by growth-promoting or growth-inhibiting factors in pancreatic cancer-derived cells.<br><br>Material and Methods
Cell lines and cell culture conditions
The <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 were a kind gift of Dr C. Susini (INSERM U.531, Toulouse) and A. Hajri (IRCAD, Strasbourg) respectively. Panc-1 cells were grown in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal <span class="yellow">calf</span> serum, 2 mM L-glutamine, 100 IU/ml penicilin G and 100 μg/ml streptomycin. BxPc-3 were cultivated in RPMI 1640 medium in the presence of 2 mM L-glutamine, 4.5 g/L glucose, 10 mM Hepes, 1.0 mM sodium pyruvate, 10% fetal <span class="yellow">calf</span> serum and 100 IU/ml penicilin G and 100 μg/ml streptomycin. <span class="yellow">Human</span> recombinant TGFβ-1 was obtained from Sigma, and specific SB203580, U0126 and SP600125 inhibitors were from Calbiochem and utilized at 10 μM.<br><br>Expression plasmids
Expression plamids encoding p38 (pCEFL HA p38), Erk2 (pcDNAIII HA ERK2), JNK (pcDNAIIIB HA JNK), the wild-type Raf (pcDNA RAF BXB) and the Raf dominant negative (pcDNA RAF 301 K375W) were obtained from O Coso (University of Buenos Aires). Plamids encoding the constitutively activated type I TGFβ receptor (RI ACT), the dominant negative type II TGFβ receptor (RII DN) and the Smad4 dominant negative (DPC4 1–514 a.a.) were obtained from R Urrutia (Mayo Clinic, Rochester) and the wild type Smad4 was from C Heldin (Ludwig Institute, Uppsala).<br><br>Pancreatic p8-deficient cells
To silence p8 expression in pancreatic cells, we infected these cells with a retrovirus expressing <span class="yellow">human</span> p8 in the antisense orientation. The retroviral vector was constructed as follows: <span class="yellow">human</span> p8 cDNA was subcloned in HindIII and XhoI restriction sites of the pLPC plasmid (obtained from S. Lowe) in the antisense orientation. Amphotrope <span class="yellow">human</span> p8 expressing retrovirus was then generated by transient transfection using Phoenix amphotrope packaging cells. Viral supernatant was used to infect Panc-1 and BxPc-3 pancreatic cells and the population of p8-silenced cells was isolated by selection in presence of puromycin (1 μg/ml). As control, cells were infected with the pLPC empty vector.<br><br>p8 expression in arrested and growing cells
One million of Panc-1 or BxPc-3 cells were cultivated on 10-cm Petri plates in standard conditions (with 10% FCS). After 48 h, culture media were changed for fresh media with FCS restricted to 0.1%, in order to stop growth. After 24 hours of growth arrest, culture medium was replaced either by medium with 10% fetal <span class="yellow">calf</span> serum to resume cell growth or, as control, by medium with 0.1% fetal <span class="yellow">calf</span> serum. Twenty four hours later cells were recovered and RNA and protein extracted.<br><br>p8 expression in TGFβ-1-treated Panc-1 cells
One million of Panc-1 cells were cultivated in 10-cm culture dishes for 48 hours under standard conditions before TGFβ-1 treatment. <span class="yellow">Human</span> recombinant TGFβ-1 (5 ng/ml) was added to cells, without changing the culture medium, and cells were collected 12 hours later for RNA and protein preparation.<br><br>BxPc-3 rasV12-expressing cells
pLPC-rasV12 and pLPC plasmids were obtained from S. Lowe. Phoenix amphotrope packaging cells (106) were plated in a 6-well plate, incubated for 24 hours, then transfected with PEI with 5 μg of retroviral plasmid. After 48 hours, the medium containing virus was filtered (0.45 μm filter, Millipore) to obtain the viral supernatant. Target BxPc-3 were plated at 2 × 105 cells per 35-mm dish and incubated overnight. For infections, the culture medium was replaced by an appropriate mix of the viral supernatant and culture medium (V/V), supplemented with 4 μg/ml polybrene (Sigma), and cells were incubated at 37°C. BxPc-3 rasV12-expressing cells were selected with puromycin (1 μg/ml). Cells infected with the pLPC empty vector were used as control.<br><br>Western-blot analyses
One hundred μg of total protein extracted from cells was separated with standard procedures on 15.0% SDS-PAGE using the Mini Protean System (Bio-Rad) and transferred to a nitrocellulose membrane (Sigma). The intracellular level of p8 was estimated by Western blot using a polyclonal antibody (1:1000) raised against <span class="yellow">human</span> p8 [4].<br><br>Growth curves
One hundred thousand cells per well were plated in a series of 35-mm culture dishes. The cell number was estimated daily in triplicate, during 1 to 5 days, in a haemocytometer. Within experiments, each point was determined at least two times.<br><br>Cell transfection and gene reporter assays
Panc-1 and BxPc-3 (105) were cultivated in 30 mm diameter culture dishes for 24 hours then transiently transfected with 0.5 μg of p8-CAT reporter plasmid and 0.5 μg of pCMV/βgal plasmid (to control transfection efficiency) using the Fugene reagent in accordance with the manufacturer's protocol (Roche Molecular Biochemicals). The p8-CAT plasmid is the previously reported p-1471/+37p8-CAT promoter construct [5]. Reporter activities were measured as previously described [5]. Briefly, cell extracts were prepared with the reporter lysis buffer (Promega) 24 hours after transfection and CAT activity was determined by the phase extraction procedure [32] and β-galactosidase assay was performed essentially as described in Sambrook et al. [33]. CAT activity was normalized to β-galactosidase activity. Experiments were carried out in triplicate and repeated at least two times. Expression plasmids (0.5 μg) were co-transfected with p8-CAT and pCMV/βgal plasmids as indicated.<br><br>RT-PCR analysis
RNA was extracted using the Trizol (Life Technologies) procedure. Total RNA (1 μg) was analyzed by RT-PCR with the SuperScript™ One-step RT-PCR System and the Platinum Taq kit (Life Technologies). RT-PCR was performed using different numbers of cycles to verify that the conditions chosen were within the linear range. The mRNA coding for p8 was specifically amplified with sense (5' GAAGAGAGGCAGGGAAGACA 3') and antisense (5' CTGCCGTGCGTGTCTATTTA 3') primers, in positions 72 and 643 of the cDNA (accession # NM_012385), respectively. As control, the transcript coding for the ribosomal protein RL3 was specifically amplified for 22 cycles with sense (5'GAAAGAAGTCGTGGAGGCTG3') and antisense (5'ATCTCATCCTGCCCAAACAC3') primers, in positions 216 and 637 of the cDNA, respectively.<br><br>
Author's contributions
CM prepared cells and retrovirus, carried out RNA purification, RT-PCR, Western blots, and cell growth experiments, NL carried out CAT assays, SV participated in the design of the study and analysis of data, JLI participated in the analysis of data and wrote the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1779460</h3>MUC1 alters oncogenic events and transcription in <span class="yellow">human</span> breast cancer cells
Abstract
Introduction
MUC1 is an oncoprotein whose overexpression correlates with aggressiveness of tumors and poor survival of cancer <span class="yellow">patients</span>. Many of the oncogenic effects of MUC1 are believed to occur through interaction of its cytoplasmic tail with signaling molecules. As expected for a protein with oncogenic functions, MUC1 is linked to regulation of proliferation, apoptosis, invasion, and transcription.<br><br>Methods
To clarify the role of MUC1 in cancer, we transfected two breast cancer cell lines (MDA-MB-468 and BT-20) with small interfering (si)RNA directed against MUC1 and analyzed transcriptional responses and oncogenic events (proliferation, apoptosis and invasion).<br><br>Results
Transcription of several genes was altered after transfection of MUC1 siRNA, including decreased MAP2K1 (MEK1), JUN, PDGFA, CDC25A, VEGF and ITGAV (integrin αv), and increased TNF, RAF1, and MMP2. Additional changes were seen at the protein level, such as increased expression of c-Myc, heightened phosphorylation of AKT, and decreased activation of MEK1/2 and ERK1/2. These were correlated with cellular events, as MUC1 siRNA in the MDA-MB-468 line decreased proliferation and invasion, and increased stress-induced apoptosis. Intriguingly, BT-20 cells displayed similar levels of apoptosis regardless of siRNA, and actually increased proliferation after MUC1 siRNA.<br><br>Conclusion
These results further the growing knowledge of the role of MUC1 in transcription, and suggest that the regulation of MUC1 in breast cancer may be more complex than previously appreciated. The differences between these two cell lines emphasize the importance of understanding the context of cell-specific signaling events when analyzing the oncogenic functions of MUC1, and caution against generalizing the results of individual cell lines without adequate confirmation in intact biological systems.<br><br><br><br>Introduction
MUC1 is the founding member of the mucin family: proteins characterized by heavy O-glycosylation centering around a variable number of tandem repeats that are rich in serine and threonine residues [1,2]. MUC1 is a transmembrane heterodimer with one subunit solely extracellular (MUC1-EX), and the other subunit composed of a short extracellular stem, a single transmembrane domain, and the cytoplasmic tail (together called the MUC1-CT). MUC1 possesses both pro- and anti-adhesive capacities, as the MUC1-EX provides binding sites for a variety of adhesion proteins, while its large size and extended structure prevents cell-cell contact [3-5].
Initially described as a tumor antigen overexpressed in >90% of breast cancers, MUC1 is now known to be an oncogene with roles in both tumor formation and progression [1,6]. <span class="yellow">Mouse</span> studies have been integral to the current understanding of MUC1 in cancer. Muc1 knockout <span class="yellow">mice</span> (Muc1-/-; MUC1 is <span class="yellow">human</span>; Muc1 is <span class="yellow">mouse</span>) show a reduction in tumorigenic phenotype when crossed onto <span class="yellow">mice</span> overexpressing the Wnt-1 [7] or polyomavirus middle T antigen [8] oncogenes in the mammary gland. In contrast, MUC1 overexpression in the mammary gland drives tumor formation [9], indicating that MUC1 is a true oncogene.
Many of the oncogenic effects of MUC1 stem from its cytoplasmic tail, which binds to several proteins implicated in cancer, including c-Src [10,11] and the epidermal growth factor receptor (EGFR) family [12,13]. MUC1 stimulates mitogen activated protein kinase (MAPK) signaling through the extracellular signal regulated kinases (ERK1/2) [12,14]; this can occur through MUC1 association with Grb2 and son of sevenless to activate Ras [15]. ERK1/2 signaling is commonly stimulated by the Ras-Raf-MEK (MAPK and ERK kinase) cascade downstream of mitogens such as EGFR [16], and regulates transcription via factors like the activator protein-1 complex. Loss of MUC1 can reduce EGFR expression [17], providing another means of affecting MAPK signaling. Our results describe a novel mechanism by which MUC1 regulates the ERK1/2 pathway, through modulating transcription of the genes encoding MEK1, Raf-1, and c-Jun.
MUC1 expression correlates with increased survival in response to cytotoxic or oxidative agents [18-21], and can activate the phosphoinositol-3 kinase-AKT pathway as part of an anti-apoptotic response [18]. MUC1 has also recently been linked to transcription, as the MUC1-CT localizes to the nucleus [22] and affects transcription by β-catenin [22,23], FOXO3a [21], p53 [24], and estrogen receptor α [25]. However, there are indications that the role of MUC1 in oncogenesis is regulated by cell type and signaling context. For example, MUC1 can stimulate Fas-mediated apoptosis [26], while Muc1 is specifically down-regulated in c-neu-induced mammary tumors [27]. This report emphasizes the complexity of MUC1 signaling in breast cancer by contrasting results from two established breast cancer cell lines.
To understand MUC1 function in cells with high endogenous expression, that is, cells likely to have evolved with active MUC1 signaling, we used small interfering RNA (siRNA) to knock down MUC1 in MDA-MB-468 and BT-20 cells. We then analyzed transcription of 84 genes involved in cancer, as well as the effects upon cellular events linked to oncogenesis, such as apoptosis and proliferation. Though the cell lines show some similarity in transcriptional alterations after transfection with MUC1 siRNA, their phenotypes are quite dissimilar: MDA-MB-468 increases apoptosis and reduces proliferation and invasion, while BT-20 proliferates more rapidly after loss of MUC1. This last may reflect the striking amount of active AKT in BT-20; AKT activity is increased in both cell lines after MUC1 siRNA, which agrees with a previous study of MUC1 siRNA [21], but disagrees with results from 3Y1 fibroblasts [18]. Recent studies have emphasized the complex and context-specific regulation of even such classic oncogenes as AKT [28]. The differences between the two breast cancer cell lines in this study suggest that MUC1 oncogenic functions are also subject to cell-specific regulation, and stress the need for understanding the cellular signaling context when interpreting results.<br><br>Materials and methods
Cell culture and siRNA transfection
MDA-MB-468 and BT-20 cells (American Type Culture Collection) were cultured in Dulbecco's modified Eagle's medium (Invitrogen, Carlsbad, CA, USA) plus 10% fetal <span class="yellow">calf</span> serum, 1% Glutamax (Invitrogen) and 1% penicillin/streptomycin. Stable cell lines (468.Neo and 468.MUC1Δ8) were selected with 0.5 mg/ml G418. For epidermal growth factor (EGF) stimulation, MDA-MB-468 cells were serum-starved overnight and treated for 10 minutes at 37°C with 100 ng/ml EGF. Transient siRNA transfection was performed with Lipofectamine2000 (Invitrogen) and 100 nM siRNA oligonucleotides. The commercially available siRNA constructs (all from Dharmacon, Lafayette, CO, USA) were scrambled (siCONTROL non-targeting siRNA #1), or directed against firefly luciferase (siCONTROL non-targeting siRNA #2) or MUC1 (siGENOME smartpool). The independent oligonucleotides designed in our laboratory target sequences beginning at MUC1 codons 882 and 956, and have been described previously [29]. The scrambled siRNA construct was used only in BT-20 cells as it causes a non-specific knockdown of MUC1 in the MDA-MB-468 line.<br><br>Cloning of MUC1 WT vector and stable transfection
Two silent mutations (G891A and T894C) were introduced into the MUC1 cDNA (called MUC1Δ8) to make it resistant to the 882 siRNA that targets that region of the mRNA. The mutant cDNA was cloned into the pLNCX.1 vector with neomycin resistance (gift of Joseph Loftus, Mayo Clinic, Arizona, USA). Stable transfection was performed with Lipofectamine2000; cells were selected beginning 24 hours post-transfection and maintained as a polyclonal population.<br><br>Western blots and antibodies
Cells were lysed in buffer (20 mM HEPES pH 8.0, 150 mM sodium chloride, 1% Triton X-100, 2 mM EDTA) with commercial protease (Complete inhibitor cocktail, Roche, Pleasanton, CA, USA) and phosphatase inhibitors (10 mM sodium fluoride, 2 mM sodium vanadate, 50 μM ammonium molybdate). Protein concentration was determined by BCA (Pierce, Rockford, IL, USA); 50 μg of lysate were loaded on SDS-PAGE gels for each experiment, except for the pMEK1/2 blot in MDA-MB-468, where 150 μg were used. Non-commercial antibodies used were: BC2, a <span class="yellow">mouse</span> monoclonal to the MUC1-EX (gift of Dr McGuckin, Queensland University, Queensland, Australia), and CT2, an <span class="yellow">Armenian hamster</span> monoclonal to the MUC1-CT developed in our lab [12]. Antibodies to pMEK1/2, MEK1/2, ERK1/2, Myc, pAKT, AKT, β-tubulin (all Cell Signaling, Danvers, MA, USA), β-actin and dpERK1/2 (both Sigma, St. Louis, MO, USA) were used according to manufacturers' recommendations. All antibodies except β-actin (1:2,500) and dpERK1/2 (1:10,000) were used at 1:1,000 dilution for western blots. Flow cytometric analysis of MUC1 was done with HMPV-FITC, which recognizes the core peptide of the MUC1-EX tandem repeats (Pharmingen, San Diego, CA, USA). Bromodeoxyuridine (BrdU) staining was performed with a fluorescently conjugated antibody to BrdU (BrdU-PE, BD Biosciences, San Diego, CA, USA) as described below. Densitometry was performed using the public domain ImageJ program (developed at the NIH and available at [30]. Each band was measured in three places; the results were averaged and normalized to tubulin to control for loading.<br><br>Transwell invasion assays
Cells were serum-starved beginning 24 hours post-siRNA transfection. Cells were re-plated in serum-free medium 48 hours post-transfection at 50,000 cells per insert (sized for 24-well plates), with serum-containing medium in the bottom of the growth well as an attractant. Transwell inserts (BD Biosciences) pre-coated with laminin, fibronectin, collagen IV, or control (no matrix) were used, and cells were permitted to invade for 48 hours. At this point (96 hours post-transfection), visual inspection of the growth wells confirmed that negligible numbers of cells went through to the bottom of the plate. Non-invaded cells were swabbed from the tops of half of the inserts ('samples', containing only invaded cells), and retained in the others ('controls', all cells). Inserts were stained for 10 minutes with crystal violet (0.5% in 20% methanol) and washed with water. Membranes were cut out and destained for 10 minutes in 10% acetic acid in a 96-well plate; membranes were removed and absorbance was read at 570 nm. Percent invasion is defined as (absorbance of samples/absorbance of controls) × 100.<br><br>[3H]Thymidine incorporation assays
Cells were re-plated in quadruplicate 24 hours post-siRNA transfection at 15,000 cells/well (96-well plate) with [3H]thymidine (1 μCi/well), then incubated in normal conditions for 24 hours. At this time (48 hours post-siRNA transfection) excess radioactivity was washed off and the cells were harvested and read on a TopCount plate reader. Statistical analysis was performed using JMP 5.1.2 software (SAS Institute, Inc., Cary, NC, USA); the student's t test was used to determine p values and significance was confirmed with Wilcoxon rank sum and Pearson chi squared analyses.<br><br>BrdU incorporation
BrdU (50 μM) was given to cells 48 hours post-siRNA transfection and permitted to incorporate for 1.5 hours. Cells were then washed with PBS, trypsinized, and washed again. BrdU staining was performed according to an adaptation of the manufacturer's protocol: cells were re-suspended in PBS, mixed 1:1 with -20°C neat ethanol, and incubated 1 hour at -20°C to fix. Fixed cells were then washed gently and denatured in 2 M HCl for 20 minutes at room temperature. Following washing and 2 minute's incubation with 0.1 M Tris to neutralize the acid, cells were re-suspended in FACS buffer (0.5% fetal <span class="yellow">calf</span> serum in PBS) and stained with Phycoerythrin (PE)-conjugated anti-BrdU according to the manufacturer's protocol for flow cytometry analysis on a FACScan instrument.<br><br>Apoptosis and trypan blue staining
Apoptosis was measured using a kit (BD Biosciences) containing propidium iodide (PI) and FITC-conjugated annexin V. Cells were stained according to the manufacturer's protocol and the level of apoptosis determined by flow cytometry. Quadrants are: early apoptosis (annexin V+/PI-, lower right) late apoptosis (annexin V+/PI+, upper right) and non-apoptotic cell death (annexin V-/PI+, upper left). Treatments for the stress panel were: no treatment (control); DMSO as a control for celecoxib; 20 mM celecoxib, brand name Celebrex™ (dissolved in DMSO) [31]; 0.2 mM H2O2 [32]; or 1 mg/mL G418 (Pfizer, New York, NY, USA).<br><br>Real-time PCR arrays
Transcriptional analysis using Cancer PathwayFinder RT2 profiler arrays (SuperArray, Frederick, MD, USA) was performed according to the manufacturer's protocol. Briefly, total RNA was isolated using an RNeasy extraction kit (Qiagen, Valencia, CA, USA); 1 μg of RNA was reverse transcribed with the cDNA synthesis kit (SuperArray) and cDNA was subjected to real-time PCR using SYBR green to detect product. Arrays were performed independently at least twice for each cell line; all PCR products were checked on agarose gels. Values were obtained for the threshold cycle (Ct) for each gene and normalized using the average of four housekeeping genes on the same array (HPRT1, RPL13A, GAPD, ACTB). Ct values for housekeeping genes and a dilution series of ACTB were monitored for consistency between arrays. Change (ΔCt) between MUC1 siRNA and control siRNA was found by:
ΔCt = Ct(MUC1 siRNA) - Ct(control siRNA)
and fold change by:
Fold change = 2(-ΔCt)
Values are given as fold change; only genes showing two-fold or greater change were considered. Both luciferase and scrambled siRNA controls were used in BT-20; only genes showing consistent alteration with both controls were included in the results reported here. The scrambled siRNA could be not used in MDA-MB-468 as these cells decrease MUC1 expression in response to this construct.<br><br>
Results
siRNA transfection decreases MUC1 expression in breast cancer cell lines
Two <span class="yellow">human</span> breast cancer cell lines, MDA-MB-468 and BT-20, were transiently transfected with a pool of four siRNA oligonucleotides directed against the MUC1 mRNA (468.siMUC1 and BT.siMUC1), or a control oligonucleotide directed against luciferase (468.siLuc and BT.siLuc). Both cell lines express high levels of MUC1, making them promising targets for this analysis. Western blots (Figure 1a) show successful knockdown of both the extracellular domain and cytoplasmic tail fragments of MUC1; luciferase siRNA does not substantially change the level of MUC1 compared to parental cells. 468.siMUC1 show a substantial decrease in the amount of MUC1-CT, while BT.siMUC1 show slightly less knockdown of MUC1-CT. Both MDA-MB-468 and BT-20 display a less dramatic decrease of MUC1 extracellular domain compared to MUC1-CT (Figure 1a); this likely represents protein synthesized prior to transfection, and may reflect differences in the turnover rates of the two subunits.
Analysis of the MUC1 extracellular domain by flow cytometry confirms that both cell lines substantially decrease MUC1 expression after siRNA (Figure 1b). By flow cytometry, 468.siMUC1 averaged 75% knockdown of MUC1 compared to 468.siLuc; and BT.siMUC1 averaged 50% knockdown relative to BT.siLuc. These effects could be titrated with the concentration of siRNA, were seen as early as 24 hours post-transfection (data not shown) and lasted to at least 96 h post-transfection (Figure 1b). All experiments were conducted within 48 to 96 hours after siRNA transfection. Similar results were obtained using two independent oligonucleotides designed in our lab (data not shown), designated '882' and '956' for the initial codon recognized by each.<br><br>Transcriptional changes are seen after MUC1 siRNA
Recent work indicates that MUC1 may affect transcription both directly via interaction with transcription factors and indirectly (for example, through modulating signaling). To study the effects of MUC1 knockdown in breast cancer cell lines, real-time PCR arrays were used to analyze transcription of 84 genes implicated in cancer. Only genes with greater than two-fold change were considered. Three genes (MAP2K1, VEGF, PDGFA) were altered two-fold or more after MUC1 siRNA in both MDA-MB-468 and BT-20 cells (Figure 2); two genes (ITGAV, MMP2) changed only in 468.siMUC1; and five genes (TIMP3, RAF1, JUN, TNF, CDC25A) only in BT.siMUC1. This list represents all genes affected greater than two-fold after MUC1 siRNA, rather than a select group. Three genes whose transcription was changed by less than two-fold are shown, two of which (PDGFB and ITGB1) are listed because they relate closely to genes altered by two-fold (PDGFA and ITGAV). The third, MYC, is included because western blots confirmed a substantial change at the protein level (Figure 3a) that may reflect both transcriptional and post-transcriptional regulation.
Interestingly, transcription of MAP2K1 was decreased in both cell lines after MUC1 siRNA. This gene encodes MEK1, one of the primary regulators of the ERK1/2 MAPK pathway [33], a network that has been linked several times to MUC1 [12,34-36]. We examined MEK1 and MEK2 levels by western blot to confirm decreased protein in MUC1 siRNA-treated cells (Figure 3a), and found that not only were total MEK1/2 levels lower in 468.siMUC1 and BT.siMUC1 compared to controls (0.48 and 0.68 relative to siLuc, respectively), but so were the basal amounts of active (phosphorylated) MEK1/2 (pMEK1/2; 0.12 and 0.42 relative to siLuc, respectively). Both 468.siMUC1 and BT.siMUC1 also showed reduced activation of ERK1/2 (dpERK1/2; 0.21 and 0.27 relative to siLuc, respectively), as would be expected with diminished signaling through MEK1/2; total ERK1/2 levels remain unchanged.
As both lines have high levels of EGFR and thus activate the MEK-ERK cascade intensely when stimulated with EGF [37], siRNA-transfected cells were treated with EGF. Notably, MUC1 siRNA impairs this important oncogenic pathway in MDA-MB-468 cells, as 468.siMUC1 display less pMEK1/2 in response to EGF than do 468.siLuc (Figure 3b). Interestingly, EGF treatment of BT-20 cells results in slightly higher pMEK1/2 levels in BT.siMUC1 compared to BT.siLuc. Though this result seems paradoxical in light of decreased MAP2K1 transcription in BT.siMUC1, it likely results from differential functions of Raf isoforms in combination with the increased RAF1 transcription (Figure 2) and protein level (Figure 3a) in these cells. Specifically, B-Raf is thought to be the main activator of MEK under normal conditions; Raf-1 activates MEK in response to stimulus [38]. Thus, it appears that basal pMEK1/2 levels are not greatly affected by Raf-1 overexpression in BT.siMUC1 cells, likely because MEK is regulated primarily by B-Raf under normal growth conditions. In contrast, when the cells are stimulated (EGF), increased Raf-1 levels in BT.siMUC1 leads to heightened pMEK1/2 (Figure 3b).<br><br>MUC1 siRNA increases apoptosis in MDA-MB-468 but not BT-20
We next examined whether MUC1 knockdown and its associated transcriptional alterations would affect overall cellular events. As several of the genes shown in Figure 2 are important in regulating proliferation and survival, and because of the recently described role of MUC1 in modulating apoptosis in response to cellular stresses [20,21,24], we first analyzed whether MUC1 siRNA would alter apoptosis in these lines. Although there was no change in basal apoptosis in either line (Figure 4a), we observed that the cell lines responded differently when trypsinized for re-plating 24 hours after transfection (Figure 4b). Interestingly, 468.siMUC1 cells show greater apoptosis after trypsinization than do 468.siLuc (49.8% versus 34.0%, respectively), while BT-20 cells from both siRNA treatments display similar levels of apoptosis (around 22%).
To examine whether this phenomenon is specific to trypsin treatment or part of a general stress response involving MUC1, we subjected cells to a panel of stresses and measured cell death. In agreement with the patterns seen with trypsinization, BT.siLuc and BT.siMUC1 respond similarly to all treatments (data not shown), while 468.siMUC1 die more readily than 468.siLuc in response to trypsin, G418, hydrogen peroxide, or celecoxib, a chemotherapeutic that targets the cyclooxygenase-2 (COX-2) pathway (Figure 4c); these data were confirmed with two independent siRNA constructs (data not shown).
Like the MAPK pathway, AKT signaling has been linked to MUC1 in cancer. Although transcription of AKT was not altered in MUC1 siRNA-treated cells, the results of our apoptosis studies prompted us to investigate levels of AKT further. As expected, the total AKT protein level is not greatly changed after MUC1 siRNA in either cell line, though the active form (pAKT) is increased in both 468.siMUC1 and BT.siMUC1 compared to controls (Figure 3a). This result disagrees with MUC1 activation of the AKT pathway in <span class="blue">rat</span> 3Y1 cells [18], and may reflect regulation more appropriate to breast cancer cells; this is supported by activation of AKT in response to MUC1 siRNA in other lines [21]. In addition, there is a striking difference in the relative amounts of AKT and pAKT in the two cell lines (Figure 4d). When lysates from both lines are exposed to film for the same length of time (overexposure masks the differences between BT.siLuc and BT.siMUC1 that are apparent in Figure 3a), it is clear that pAKT levels are much higher in BT-20 than in MDA-MB-468, despite lower total AKT expression. This difference in AKT activation between MDA-MB-468 and BT-20 likely contributes to the disparity in their sensitivity to the increased apoptosis expected with loss of MUC1.<br><br>MUC1 siRNA alters proliferation and invasion
As MUC1 is involved in apoptosis, we next analyzed its effects on proliferation. BrdU and [3H]thymidine incorporation were used to analyze proliferation after MUC1 siRNA. 468.siMUC1 cells show a significant decrease in [3H]thymidine incorporation compared to 468.siLuc, while intriguingly, BT.siMUC1 cells show a significant increase in proliferation (Figure 5a). Growth curves mirror these results, as do experiments with the two independent MUC1 siRNA oligonucleotides (data not shown). Note that these assays require trypsinizing cells 24 hours post-transfection; therefore, the results in the MDA-MB-468 line could stem from the changes in apoptosis described in the previous section, rather than a true effect on proliferation. To control for this, we incubated non-trypsinized, siRNA-transfected cells at similar confluence with BrdU to measure incorporation. The 'clumped' profile of cells (contrast to Figure 4b) is likely a result of the acid denaturation (recommended by the antibody manufacturer), as it occurs uniformly in these experiments. BrdU incorporation (Figure 5b) confirms that the [3H]thymidine results are not solely due to alterations in apoptosis, as 468.siMUC1 cells incorporate less BrdU than 468.siLuc; once again, BT.siMUC1 cells show increased proliferation over BT.siLuc.
Given the role of MUC1 in adhesion, we examined whether MUC1 siRNA affects cellular invasion. In transwell assays, BT-20 cells invaded poorly, regardless of the siRNA used (data not shown). However, MDA-MB-468 cells invade more readily, and were analyzed on a panel of three different extracellular matrix proteins. Interestingly, 468.siMUC1 cells display somewhat decreased invasion on collagen IV, laminin, and fibronectin matrices, and on a no-matrix control (Figure 5c), which is in agreement with the trend towards decreased metastasis observed in Muc1-/- × <span class="yellow">MMTV</span>-PyV MT <span class="yellow">mice</span> [8].<br><br>Transfection of MUC1 rescues the 468.siMUC1 phenotype
To determine if the above effects are specific to MUC1, we created stable transfectants of the MDA-MB-468 line using empty vector (468.Neo) or a full-length MUC1 construct (468.MUC1Δ8) that is resistant to one of the independent MUC1-directed oligonucleotides ('882'). These cells were maintained in G418-containing medium to retain transgene selection. As expected, 468.MUC1Δ8 cells show higher levels of both the MUC1 extracellular domain and the MUC1-CT than do 468.Neo (Figure 6a). Note that 468.Neo have MUC1 expression comparable to parental MDA-MB-468; the exposures in Figure 6a are lighter than those in Figure 1a, in order to clearly show the relative levels of MUC1 in the stable transfectants. After MUC1 siRNA, 468.MUC1Δ8 lose some MUC1 (likely endogenous protein, which is not siRNA-resistant) but retain high-level expression, while 468.Neo show a decrease in MUC1 levels similar to parental 468.siMUC1 cells (Figures 6a,b). The difference in the amount of MUC1 knockdown between 468.Neo and 468.MUC1Δ8 is highlighted by the purple shading in Figure 6b.
BrdU incorporation (Figure 6c) indicates that 468.Neo show decreased nucleotide incorporation after MUC1 siRNA compared to control (3.3% versus 25.0%, respectively); this is not seen in 468.MUC1Δ8 cells, which show similar levels of BrdU incorporation regardless of the siRNA used (21.5% for luciferase, 23.9% for MUC1). 468.Neo cells display a more dramatic decrease in BrdU incorporation after MUC1 siRNA than what is seen in parental 468.siMUC1 cells, which may reflect the additional stress of being maintained in G418-containing medium. Similarly, analysis of apoptosis in trypsinized cells indicates that the increased apoptosis seen in parental 468.siMUC1 cells is also present in the 468.Neo line after MUC1 siRNA (Figure 6d; 43.6% in control versus 59.6% in MUC1 siRNA). However, in 468.MUC1Δ8 cells, the level of apoptosis after luciferase siRNA (34.1%) is lower than that in 468.Neo cells; MUC1 siRNA increases the amount of apoptosis slightly (42.8%), restoring it to a level similar to that seen in luciferase siRNA-treated 468.Neo cells. Together, these studies suggest that the above-described results are specific to MUC1, as stable transfection of an siRNA-resistant MUC1 rescues the phenotype seen in 468.siMUC1 cells.<br><br>
Discussion
This report describes both the transcriptional alterations seen after transfection with MUC1 siRNA in <span class="yellow">human</span> breast cancer cells and the effects on events such as apoptosis and proliferation. The two cell lines used (MDA-MB-468 and BT-20) were chosen for high expression of MUC1 and a substantial (50% to 75%), consistent decrease in MUC1 expression after siRNA. Both lines have epithelial morphology, form tumors slowly in <span class="yellow">nude mice</span> [37], have mutant p53 [39,40], express EGFR [37], and lack estrogen receptor α [41]. One striking difference between these lines, however, is their response to MUC1 siRNA. MDA-MB-468 cells behave as expected for loss of an oncogene: MUC1 siRNA correlates with increased apoptosis in response to stress, decreased proliferation, and reduced invasion. In contrast, BT.siMUC1 cells proliferate more rapidly than BT.siLuc cells with little effect on apoptosis.
Much of the phenotype of these cells can be understood in light of protein levels and transcriptional activity after MUC1 siRNA. As mentioned, both MDA-MB-468 and BT-20 display increased pAKT after MUC1 siRNA, but the ratio of active to total AKT is considerably higher in the BT-20 line, which may help these cells resist the increased apoptosis expected with loss of MUC1. Myc levels are also higher in both cell lines after MUC1 siRNA, although the ability of Myc to promote proliferation and apoptosis in different cellular contexts [42] complicates the interpretation of this finding.
Both cell lines show reduced transcription of VEGF, PDGFA, PDGFB, and MAP2K1 (MEK1) after MUC1 siRNA. The genes encoding vascular endothelial growth factor (VEGF) and the A and B chains of platelet-derived growth factor (PDGF-A and PDGF-B) are interesting as these proteins have been heavily implicated in angiogenesis, suggesting a novel function for MUC1 in regulating this process. Vascular endothelial growth factor expression in cancer is linked to tumor growth and metastasis [43,44]; platelet-derived growth factor is also angiogenic, but has an additional role in stimulating desmoplasia [45]. Reduced transcription of these genes after MUC1 siRNA suggests that MUC1 may foster angiogenesis and stromal proliferation, although this must be confirmed in a more appropriate model system.
Decreased MAP2K1 (MEK1) transcription after MUC1 siRNA provides a novel mechanism by which MUC1 can affect the ERK1/2 MAPK pathway. MUC1 has often been linked to the Ras-Raf-MEK-ERK cascade [12,34-36,46], and at least two mechanisms by which MUC1 can alter MAPK signaling have been described: MUC1 interaction with and phosphorylation by the EGFR family [12,13], and MUC1 binding to the Grb2/Sos complex that activates Ras [46]. Reduction of MEK1 levels after MUC1 siRNA agrees with the role of MUC1 in strengthening MAPK signaling, and indicates that MUC1 can regulate both the transcription and activity of members of this pathway.
Two additional MAPK pathway members are altered specifically in BT.siMUC1, with no corresponding change in 468.siMUC1 cells. These genes are RAF1 and JUN which are increased and decreased, respectively, after MUC1 siRNA. Raf-1 and c-Jun both function outside of the ERK1/2 MAPK pathway, which may explain the seeming paradox of increased RAF1 transcription with simultaneous decreases in MAP2K1 and JUN. Specifically, Raf-1 can inhibit ASK1 (apoptosis signal-regulated kinase 1) upstream of p38 and JNK (Jun N-terminal kinase) [38]. ASK1 phosphorylates JNK in response to stress, resulting in activation of c-Jun and stimulation of apoptosis [47], indicating that the coordinate up-regulation of RAF1 and down-regulation of JUN may provide a potent anti-apoptotic effect in BT.siMUC1.
Regulation of life and death is also a hallmark of the CDC25A and TNF gene products. CDC25A is a phosphatase that stimulates cell cycle progression [48], thus the effects of its decrease in BT.siMUC1 are unclear in light of the increased proliferation of these cells. However, the CDC25 proteins (A, B, and C) were recently shown to have greater functional overlap than was previously thought [49], suggesting that the other two isoforms may compensate for reduced CDC25A levels. TNF encodes tumor necrosis factor (TNF)α, known for its potent, cell type-specific control of life and death. In tumor cells, TNFα expression can promote proliferation and inhibit apoptosis [50], suggesting that increased TNF transcription in BT.siMUC1 could contribute to the increased proliferation seen in these cells.
Interestingly, the increase in TNF is accompanied by decreased transcription of TIMP3, encoding tissue inhibitor of metalloproteinases (TIMP)3. The TIMP family disrupts the function of matrix metalloproteinases (MMPs), generally resulting in decreased invasion [51]. TIMP3 is unique in that it can also inhibit TNFα converting enzyme (TACE), which activates TNFα by cleaving it from the cell surface [50]. Reduced expression of TIMP3 would, therefore, foster signaling through TNFα by releasing inhibition of TNFα converting enzyme. In agreement with this, TIMP3 can promote apoptosis [52]; thus, its down-regulation in BT.siMUC1 provides another mechanism by which these cells are able to resist the increased apoptosis expected with loss of MUC1.
Another TIMP target responds to MUC1 siRNA, as 468.siMUC1 cells show significantly increased expression of MMP2 (encoding MMP-2/gelatinase A), the product of which degrades type IV collagen [52]. In breast cancer, the ratio of active to latent MMP-2 increases with tumor progression; MMP-2 may facilitate both angiogenesis and metastasis [52]. Its increase after loss of MUC1 is, therefore, unexpected, but at least two factors may clarify this result. First, though MMP-2 levels are increased in <span class="yellow">mouse</span> mammary tumors, its expression is confined to the stroma [53], suggesting that increased MMP2 transcription after loss of the epithelium-specific MUC1 might reflect a shift towards a more mesenchymal phenotype. Second, MMP-2 levels are increased by overexpression of erbB2 [52]; previous studies have shown that erbB2 and Muc1 expression are mutually exclusive in mammary tumors [27], implying that MMP2 might be part of a transcriptional profile linked to low MUC1 levels.
It is intriguing that, despite increased MMP2 transcription, invasion is decreased in 468.siMUC1 cells, even on collagen IV. This may reflect insufficient activation of MMP-2, as the precursor protein must be cleaved for enzymatic function [52]. Alternatively, the slowed invasion of these cells may relate to impaired adhesion resulting from decreased transcription of ITGAV and ITGB1 (αv and β1 integrins, respectively). Integrin signaling is tied to life-or-death decisions in epithelial cells, and integrin expression is vital for processes from wound healing to metastasis [54]. Integrin αvβ3 is implicated in facilitating metastasis of breast cancer cells to bone [55]; decreased transcription of ITGAV after MUC1 siRNA may, therefore, suggest that MUC1 is involved in this lethal process as well.
The MUC1 oncogene has been linked to apoptosis [18,20,26], proliferation [17], and transcription [21,23-25] in cancer. However, the two cell lines chosen for our study display very different responses to MUC1 siRNA, indicating that regulation of MUC1 in breast cancer is likely quite complex and cautioning against over-generalization of results from individual cell lines. Previous reports suggest that, though most studies outline a clearly oncogenic role for MUC1 in breast cancer, the exact details may vary depending on factors such as cell type and signaling context. For example, MUC1 stimulates Fas-mediated apoptosis in CHO cells [26], quite unlike the inhibition of apoptosis seen in other cell lines. Similarly, though MUC1 drives mammary oncogenesis in its own right [9] and facilitates tumorigenesis driven by other oncogenes [7,8], Muc1 is selectively down-regulated in c-neu-induced <span class="yellow">mouse</span> mammary tumors [27], indicating that the context of oncogenic signaling is vital to understanding the function of MUC1.
Thus, it is important to consider the relative levels of knockdown of MUC1 in the two cell lines: BT-20 cells reduce MUC1 expression after siRNA less strongly than do MDA-MB-468 (50% versus 75% knockdown, respectively). As MUC1 serves as a scaffold [11], overexpression of MUC1 relative to its associated signaling proteins might create a dilution effect, sequestering signal transducers away from each other; this would be relieved by MUC1 siRNA. Thus, enough MUC1 may be retained in BT.siMUC1 cells for its oncogenic effects, while signaling complex formation would be enhanced by lowering the amount of MUC1 relative to other signaling proteins.<br><br>Conclusion
The contrast between the MDA-MB-468 and BT-20 lines in response to MUC1 siRNA serves as a reminder that simplified models such as cell lines fail to encompass the complexity of intact biological systems. This report describes transcriptional alterations seen after MUC1 knockdown: decreased transcription of MAP2K1, VEGF, PDGFA, ITGAV, TIMP3, CDC25A, and JUN, and increased transcription of MMP2, TNF, and RAF1. The alterations in MAP2K1, RAF1, and JUN represent a novel means by which MUC1 can affect ERK1/2 signaling: transcriptional regulation of MAPK pathway members. Oncogenic events are also altered in both cell lines after MUC1 siRNA. These results strengthen the growing ties linking MUC1 and transcriptional regulation, and suggest that the role of MUC1 in breast cancer may be more complex than a direct correlation between MUC1 level and oncogenic function.<br><br>Abbreviations
BrdU = bromodeoxyuridine; EGF = epidermal growth factor; EGFR = epidermal growth factor receptor; ERK = extracellular signal regulated kinase; MAPK = mitogen activated protein kinase; MEK = MAPK and ERK kinase; MMP = matrix metalloproteinases; MUC1-CT = MUC1 cytoplasmic tail; MUC1-EX = MUC1 extracellular subunit; PBS = phosphate-buffered saline; PI = propidium iodide; siRNA = small interfering RNA; TIMP = tissue inhibitor of metalloproteinases; TNF = tumor necrosis factor.<br><br>Competing interests
The authors declare that they have no competing interests.<br><br>Authors' contributions
CLH performed all studies and composed the manuscript. SJG participated in the design and coordination of the studies and contributed strongly to the revision of the manuscript. Both authors have read and approved the manuscript.<br><br>
<h3>pmcA2435603</h3>Potentiation of Nerve Growth Factor-Induced Neurite Outgrowth by Fluvoxamine: Role of Sigma-1 Receptors, IP3 Receptors and Cellular Signaling Pathways
Abstract
Background
Selective serotonin reuptake inhibitors (SSRIs) have been widely used and are a major therapeutic advance in psychopharmacology. However, their pharmacology is quite heterogeneous. The SSRI fluvoxamine, with sigma-1 receptor agonism, is shown to potentiate nerve-growth factor (NGF)-induced neurite outgrowth in PC 12 cells. However, the precise cellular and molecular mechanisms underlying potentiation by fluvoxamine are not fully understood. In this study, we examined the roles of cellular signaling pathways in the potentiation of NGF-induced neurite outgrowth by fluvoxamine and sigma-1 receptor agonists.<br><br>Methods and Findings
The effects of three SSRIs (fluvoxamine, sertraline, paroxetine) and three sigma-1 receptor agonists (SA4503, 4-phenyl-1-(4-phenylbutyl) piperidine (PPBP), and dehydroepiandrosterone (DHEA)-sulfate) on NGF-induced neurite outgrowth in PC12 cells were examined. Also examined were the effects of the sigma-1 receptor antagonist NE-100, inositol 1,4,5-triphosphate (IP3) receptor antagonist, and specific inhibitors of signaling pathways in the potentiation of NGF-induced neurite outgrowth by selective sigma-1 receptor agonist SA4503. Fluvoxamine (but not sertraline or paroxetine) and the sigma-1 receptor agonists SA4503, PPBP, and DHEA-sulfate significantly potentiated NGF-induced neurite outgrowth in PC12 cells in a concentration-dependent manner. The potentiation by fluvoxamine and the three sigma-1 receptor agonists was blocked by co-administration of the selective sigma-1 receptor antagonist NE-100, suggesting that sigma-1 receptors play a role in blocking the enhancement of NGF-induced neurite outgrowth. Moreover, the potentiation by SA4503 was blocked by co-administration of the IP3 receptor antagonist xestospongin C. In addition, the specific inhibitors of phospholipase C (PLC-γ), phosphatidylinositol 3-kinase (PI3K), p38MAPK, c-Jun N-terminal kinase (JNK), and the Ras/Raf/mitogen-activated protein kinase (MAPK) signaling pathways blocked the potentiation of NGF-induced neurite outgrowth by SA4503.<br><br>Conclusion
These findings suggest that stimulation of sigma-1 receptors and subsequent interaction with IP3 receptors, PLC-γ, PI3K, p38MAPK, JNK, and the Ras/Raf/MAPK signaling pathways are involved in the mechanisms of action of sigma-1 receptor agonists such as fluvoxamine and SA4503.<br><br><br><br>Introduction
Selective serotonin (5-HT; 5-hydroxytryptamine) reuptake inhibitors (SSRIs) have emerged as a major therapeutic advance in psychopharmacology. SSRIs are the treatment of choice for many indications, including major depressive disorder, dysthymia, panic disorder, obsessive-compulsive disorder, eating disorders, and premenstrual dysphoric disorder. In contrast, it is well known that their pharmacology is quite heterogeneous, although all of them block 5-HT transporters, thus increasing 5-HT levels throughout the central nervous system (CNS) [1]–[9].
Accumulating evidence suggests that sigma-1 receptors, which are intracellular endoplasmic reticulum (ER) proteins, are involved in both the neuroplasticity and pathophysiology of neuropsychiatric diseases such as major depressive disorder, anxiety, schizophrenia, and Alzheimer's disease [10]–[18]. Previously, we reported that some SSRIs possess high to moderate affinities for sigma-1 receptors in the <span class="blue">rat</span> brain. The rank order of SSRIs affinities for sigma-1 receptors is fluvoxamine (Ki = 36 nM)>sertraline (Ki = 57 nM)>>paroxetine (Ki = 1893 nM) [19]. Recently, we reported that fluvoxamine, but not paroxetine, significantly ameliorated cognitive deficits in <span class="yellow">mice</span> after repeated phencyclidine administration, and that the effects of fluvoxamine were antagonized by co-administration of the selective sigma-1 receptor antagonist NE-100 [20], suggesting that sigma-1 receptors are involved in the mechanism of action of fluvoxamine [9]. Interestingly, it has been demonstrated that sigma-1 receptor agonists including fluvoxamine could potentiate nerve growth factor (NGF)-induced neurite outgrowth in PC12 cells, and that NE-100 blocked the potentiation by sigma-1 receptor agonists, suggesting sigma-1 receptors are involved in neuroplasticity [21]. However, the precise cellular mechanisms underlying the potentiation by sigma-1 receptor agonists are not fully understood [13], [21].
It is therefore of great interest to study the precise cellular mechanisms underlying the enhancement by fluvoxamine on NGF-induced neurite sprouting in PC12 cells. In the present study, we examined the effects of three SSRIs (fluvoxamine, sertraline, paroxetine), as well as the effects of a sigma-1 receptor agonist (4-phenyl-1-(4-phenylbutyl) piperidine (PPBP), dehydroepiandrosterone-sulphate (DHEA)-sulfate) [9], [22]–[28] and the selective sigma-1 receptor agonist SA4503 [29], [30], on NGF-induced neurite outgrowth in PC12 cells. Furthermore, it is also known that sigma-1 receptors have been shown to interact with IP3 receptors (17,18). Therefore, we examined the effects of NE-100 and xestospongin C (a selective inositol 1,4,5-triphosphate (IP3) receptor antagonist) [31] in order to investigate the roles of sigma-1 receptors and IP3 receptors in the mechanisms underlying the enhancement of NGF-induced neurite outgrowth by SA4503. Moreover, we examined the effects of specific inhibitors of several cellular signaling targets on the enhancement of NGF-induced neurite outgrowth by SA4503, since several signal transduction molecules have been implicated in NGF-induced neurite outgrowth [32].<br><br>Materials and Methods
Drugs
The drugs were obtained from the following sources: fluvoxamine maleate (Solvay Seiyaku K.K., Tokyo, Japan); paroxetine hydrochloride, dehydroepiandosterone-sulfate (DHEA-sulfate), LY294002 (Sigma-Aldrich, St Louis, MO, USA); sertraline (Toronto Research Chemicals Inc., North York, ON, Canada); SA4503 (M's Science Corporation, Kobe, Japan); NGF (Promega, Madison, WI); xestospongin C, lovastatin, PD98059, GW5074, SB203580, MEK 1/2 inhibitor (SL327), and SP600125 (Calbiochem-Novabiochem, San Diego, CA). The selective sigma-1 receptor antagonists NE-100 and 4-phenyl-1-(4-phenylbutyl) piperidine (PPBP) were synthesized in our laboratory. Other drugs were purchased from commercial sources.<br><br>Cell culture
PC12 sells (RIKEN Cell Bank, Tsukuba, Japan) were cultured at 37°C, 5% CO2 with Dulbecco's modified Eagle's medium (DMEM) supplemented with 5% heat-inactivated fetal <span class="yellow">bovine</span> serum (FBS), 10% heat-inactivated <span class="yellow">horse</span> serum, and 1% penicillin. The medium was changed two or three times a week. PC12 cells were plated onto 24-well tissue culture plates coated with poly-D-lysine/laminin. Cells were plated at relatively low density (0.25×104 cells/cm2) in DMEM medium containing 0.5% FBS, 1% penicillin streptomycin. Medium containing a minimal level of serum (0.5% FBS) was used, since serum is known to contain steroid hormones that bind to sigma-1 receptors [21]. First, we examined the optimal concentration of NGF for NGF-induced neurite outgrowth in PC12 cells. NGF (2.5, 5, 10, 20, 40 ng/ml) increased the number of cells with neurite outgrowth in PC12 cells in a concentration-dependent manner (Figure 1). In the subsequent studies, 2.5 ng/ml of NGF was used to study the potentiating effects of sigma-1 receptor agonists on NGF-induced neurite outgrowth. Twenty-four hours after plating, the medium was replaced with DMEM medium containing 0.5% FBS and 1% penicillin streptomycin with NGF (2.5 ng/ml) with or without several drugs.<br><br>Quantification of neurite sprouting
Five days after incubation with NGF (2.5 ng/ml) with or without the several drugs, morphometric analysis was performed on digitized images of live cells taken under phase-contrast illumination with an inverted microscope linked to a camera. Images of three fields per well were taken, with an average of 100 cells per field. Differentiated cells were counted by visual examination of the field; only cells that had at least one neurite with a length equal to the cell body diameter were counted, and were then expressed as a percentage of the total cells in the field. The counting was performed in a blinded manner.<br><br>Immunocytochemistry
Cells were fixed for 30 min at room temperature with 4% paraformaldehyde then permeabilized with 0.2% Triton and blocked with 1.5% normal <span class="yellow">goat</span> serum, 0.1% <span class="yellow">bovine</span> serum albumin (BSA) in 0.1 M phosphate-buffer saline for 1 h to reduce nonspecific binding. Cells were incubated overnight at 4°C with anti-microtubule-associated protein 2 (MAP-2) antibodies (1∶1000 dilution in blocking solution, Chemicon International, Temecula, CA, USA). The immunolabeling was visualized with secondary antibodies conjugated to Alexa-488 (1∶1000; Invitrogen, Carlsbad, CA, USA). MAP-2 immuncytochemistry was visualized with a fluorescence microscope (Axiovert 200, Carl Zeiss, Oberkocken, Germany).<br><br>Statistical analysis
Data are expressed as means±S.E.M. Statistical analysis was performed by using one-way analysis of variance (ANOVA) and the post hoc Bonferroni/Dunn test. P values less than 0.05 were considered statistically significant.<br><br>
Results
Effects of SSRIs on NGF-induced neurite outgrowth
Fluvoxamine (0.1, 1.0, or 10 µM) significantly increased the number of cells with neurite outgrowth by NGF (2.5 ng/ml) in PC12 cells, in a concentration-dependent manner (Figure 2 and 4). In contrast, sertraline (0.1, 1.0, or 10 µM) and paroxetine (0.1, 1.0, or 10 µM) did not increase the number of cells with neurite outgrowth by NGF (2.5 ng/ml). A higher concentration (10 µM) of paroxetine and sertraline significantly decreased the number of cells with NGF-induced neurite outgrowth, suggesting the cellular toxicity of these drugs in PC12 cells (Figure 2).<br><br>Role of sigma-1 receptors in the potentiation of NGF-induced neurite outgrowth by fluvoxamine and SA4503
The selective and potent sigma-1 receptor agonist SA4503 (0.01, 0.1, or 1.0 µM) significantly increased the number of cells with neurite outgrowth by NGF (2.5 ng/ml) in PC12 cells, in a concentration-dependent manner (Figure 3 and 4). In addition, other sigma-1 receptor agonists–PPBP (0.1, 1.0, or 10 µM) [22]–[26] and DHEA-sulphate (0.1, 1.0, or 10 µM) [9], [27], [28]–significantly increased the number of such cells, also in a concentration-dependent manner (Figure 3). In the absence of NGF, sigma-1 receptor agonists did not produce neurite outgrowth in PC12 cells (data not shown).
To investigate the role of sigma-1 receptors, we examined the effects of NE-100 (a selective sigma-1 receptor antagonist) on the potentiation of NGF-induced neurite outgrowth by fluvoxamine, SA4503, PPBP, and DHEA-sulphate. Co-administration of NE-100 (1.0 µM) significantly blocked such potentiation by fluvoxamine (10 µM), SA4503 (1.0 µM), PPBP (10 µM), or DHEA-sulphate (10 µM) (Figure 2 and 3). Furthermore, administration of NE-100 (1.0 µM) alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 2).<br><br>Role of IP3 receptors in the potentiation of NGF-induced neurite outgrowth by SA4503
Sigma-1 receptors have been shown to interact with IP3 receptors (17,18,33–35). To investigate the role of IP3 receptors in the effects of SA4503 on NGF-induced neurite outgrowth, we examined the effects of xestospongin C (a selective, reversible, and membrane-permeable inhibitor of IP3 receptors) [31] on the effects of SA4503 on NGF-induced neurite outgrowth. Co-administration of xestospongin C (1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 5). Furthermore, administration of xestospongin C (1.0 µM) alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 5).<br><br>Role of signaling molecules proximal to TrkA in the potentiation of NGF-induced neurite outgrowth by SA4503
Next, we examined the effects of the specific inhibitors of PLC-γ, PI3K, p38 MAPK, and c-Jun N-terminal kinase (JNK), since these signaling molecules are activated upon the addition of NGF [36]. The PLC-γ inhibitor (U73122; 1.0 µM), PI3K inhibitor (LY294002; 1.0 µM), p38 MAPK inhibitor (SB203580; 1.0 µM), and JNK inhibitor (SP600125; 1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 6). In contrast, these inhibitors alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 6).<br><br>Role of Raf/Ras/ERK/MAPK pathway in the potentiation of NGF-induced neurite outgrowth by SA4503
The Raf/Ras/ERK/MAPK pathway is known to be involved in NGF-induced outgrowth [32]. Therefore, we examined the effects of this pathway's specific inhibitors. The Ras inhibitor (GW5074; 1.0 µM), Raf inhibitor (lovastatin; 1.0 µM), MEK1/2 inhibitor (SL327; 1.0 µM), and MAPK inhibitor (PD98059; 1.0 µM) significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503 (1.0 µM) (Figure 7). In contrast, these inhibitors alone did not alter NGF-induced neurite outgrowth in PC12 cells (Figure 7).<br><br>
Discussion
In the present study, we found that fluvoxamine (but not sertraline or paroxetine) and the sigma-1 receptor agonists (SA4503, PPBP, DHEA-sulfate) could potentiate NGF-induced neurite outgrowth in PC12 cells, and that the effects of these drugs were blocked by co-incubation with the selective sigma-1 receptor antagonist NE-100. These findings suggest that agonism at sigma-1 receptors for these drugs is involved in the mechanisms underlying the drugs' potentiation of NGF-induced neurite outgrowth.
As shown in Figure 1, NGF increased the number of cells with neurite outgrowth in PC12, in a concentration dependent manner. To examine whether or not the NGF levels are altered by incubation with sigma-1 receptor agonists, we measured the levels of NGF in PC12 cells with or without sigma-1 receptor agonist SA4503. No differences of NGF levels were shown in the between control group and SA4503-treated group (data not shown). It is, therefore, unlikely that the potentaition of NGF-induced neurite outgrowth by sigma-1 receptor agonists might be due to increased levels of NGF.
Unlike fluvoxamine, sertraline, which has a moderate affinity for sigma-1 receptors, did not alter NGF-induced neurite outgrowth. The reasons underlying this discrepancy between these two SSRIs are currently unclear. One possibility may involve the difference in pharmacological actions (agonist vs. antagonist) between these SSRIs at sigma-1 receptors. Interestingly, we recently found that, in a novel object recognition test, phencyclidine-induced cognitive deficits could be significantly improved by subsequent subchronic (14 days) administration of fluvoxamine, but not of sertraline, suggesting that sigma-1 receptor agonism is involved in fluvoxamine's mechanism of action [9], Ishima et al., submitted. Taken together, these findings suggest that fluvoxamine and sertraline may function as an agonist and an antagonist at sigma-1 receptors, respectively, although further study is necessary. Another possibility may be that other pharmacological activities of sertraline mask the effects of sigma-1 receptor agonism. In this study, we also found that high concentration (10 µM) of paroxetine and sertraline, but not fluvoxamine, showed cytotoxicity in PC12 cells, suggesting that fluvoxamine may be a safe drug than paroxetine and sertraline.
Sigma-1 receptors have been shown to affect intracellular Ca2+ signaling, although the precise molecular and cellular mechanisms underlying this effect are unknown. Sigma-1 receptors bind to IP3 receptors in ER, and sigma-1 receptors regulate Ca2+ release from intracellular Ca2+ storage sites [12]. Very recently, Hayashi and Su [17] reported that sigma-1 receptors function as novel ligand-operated chaperones that specifically target mitochondrion-associated ER membrane. Furthermore, sigma-1 receptors form Ca2+-sensitive chaperone machinery with another chaperone, BiP, and prolong Ca2+ signaling from ER into mitochondria by stabilizing IP3 receptors at mitochondrion-associated ER membrane [17]. In this study, we found that the IP3 receptor antagonist xestospongin C significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503, suggesting the role of IP3 receptors on sigma-1 receptor-mediated potentiation of NGF-induced neurite outgrowth. Therefore, it is likely that stimulation at sigma-1 receptors by sigma-1 receptor agonists and subsequent interaction with IP3 receptors are involved in the mechanism underlying the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists.
NGF binds to the high-affinity tyrosine receptor TrkA, initiating several signaling pathways affecting both morphological and transcriptional targets [32], [36], [37]. The signaling molecules, including PLC-γ, PI3K, p38 MAPK, and JNK, are activated upon the addition of NGF [38]. PLC-γ catalyzes the hydrolysis of phosphatidylinositol-4,5-bisphosphate (PIP2) to diacylglycerol (DAG) and inositol triphosphate (IP3). DAG activates protein kinase C, and IP3 promotes transient release of Ca2+ from the ER [39]. The pathway via PLC-γ is responsible for NGF-induced cell differentiation [40] and neurite outgrowth [41]. Furthermore, stimulation of PI3K is reported to be involved in the promotion of neurite outgrowth in PC12 cells [42]. In this study, we found that the PLC-γ inhibitor U73122 and the PI3K inhibitor LY294002 significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503. Moreover, we found that both the p38MAPK inhibitor SB203580 and the JNK inhibitor SP600125 significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503. In addition, it is also interesting that neurite outgrowth induced by low concentration (2.5 ng/ml) of NGF was not blocked by these inihibitors, consistent with a previous report [43], [44], suggesting the existence of novel pathway(s) for NGF-induced neurite outgrowth. These findings suggest that the PLC-γ, PI3K, p38MAPK, and JNK signaling pathways are involved in the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists (Figure 8). In addition, we found that the specific inhibitors for the Raf/Ras/MEK/MAPK pathways significantly blocked the potentiation of NGF-induced neurite outgrowth by SA4503, suggesting that these pathways are involved in the potentiation of NGF-induced neurite outgrowth by sigma-1 receptor agonists (Figure 8).
Our recent positron emission tomography (PET) study demonstrated that, after a single oral administration, fluvoxamine bound to sigma-1 receptors in the living <span class="yellow">human</span> brain [45]. This finding suggests that sigma-1 receptors are involved in the mechanism of action of fluvoxamine in the <span class="yellow">human</span> brain [45]. Taken together, the past and present findings suggest that, unlike paroxetine and sertraline, the SSRI fluvoxamine, with its sigma-1 receptor agonistic activity, might be a unique therapeutic drug for neuropsychiatric diseases.
In conclusion, the present results suggest that, as a sigma-1 receptor agonist, fluvoxamine could potentiate the NGF-induced neurite outgrowth in PC12 cells. Furthermore, it is likely that interaction with IP3 receptors and several subsequent signaling molecules are involved in the mechanism underlying the pharmacological action of sigma-1 receptor agonists. Therefore, it is likely that sigma-1 receptor agonists such as fluvoxamine and DHEA-sulfate, which are available around the world, would be unique therapeutic drugs for neuropsychiatric diseases.<br><br>
<h3>pmcA2562362</h3>Host immunity in the protective response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span>
Abstract
Background
We performed initial cell, cytokine and complement depletion studies to investigate the possible role of these effectors in response to vaccination with heat-killed <span class="yellow">Burkholderia mallei</span> in a susceptible BALB/c <span class="yellow">mouse</span> model of infection.<br><br>Results
While protection with heat-killed bacilli did not result in sterilizing immunity, limited protection was afforded against an otherwise lethal infection and provided insight into potential host protective mechanisms. Our results demonstrated that <span class="yellow">mice</span> depleted of either B cells, TNF-α or IFN-γ exhibited decreased survival rates, indicating a role for these effectors in obtaining partial protection from a lethal challenge by the intraperitoneal route. Additionally, complement depletion had no effect on immunoglobulin production when compared to non-complement depleted controls infected intranasally.<br><br>Conclusion
The data provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br><br><br>Background
<span class="yellow">Burkholderia mallei</span>, the etiologic agent of glanders, is a gram-negative, capsulated, non-motile, facultative intracelluar bacterium. Most known members of the Burkholderiaceae are resident in the soil; however, <span class="yellow">B. mallei</span> is thought to be an obligate mammalian pathogen. <span class="yellow">Horses</span> are highly susceptible to infection and considered the natural reservoir for infection, although <span class="yellow">mules</span> and <span class="yellow">donkeys</span> are also susceptible [1]. Clinically, glanders in solipeds can present as either a chronic (<span class="yellow">horses</span>) or acute (<span class="yellow">mules</span> and <span class="yellow">donkeys</span>) form. Naturally acquired <span class="yellow">human</span> infection with <span class="yellow">B. mallei</span>, although not seen in the United States since 1945, has occurred rarely and sporadically among laboratory workers and those in direct contact with infected animals [2]. However, glanders is endemic among domestic animals in Africa, Asia, the Middle East, and Central and South America. The course of infection is dependent on the route of exposure. Direct contact with the skin can lead to a systemic infection. Inhalation of aerosol or dust containing <span class="yellow">B. mallei</span> can lead to septicemic, pulmonary, or chronic infections of the muscle, liver and spleen. The disease has a 95% case fatality rate for untreated septicemic infections and a 50% case fatality rate in antibiotic-treated individuals [3].
There is no <span class="yellow">human</span> or animal vaccine available for glanders, and development of a partial or fully protective adaptive host response to the organism has not been well-defined. Previous studies with <span class="yellow">B. mallei</span> and the host response have shown that a mixed immune response consisting of both Th1 and Th2-associated cytokines with a predominant IgG1 subclass does not correlate with protection [4]. Additional studies with passive transfer of monoclonal antibodies specific for <span class="yellow">B. mallei</span> have correlated with early protection from infection [5]. Recent studies have also shown the Th1 cytokine IL-12 to mediate partial protection to non-viable <span class="yellow">B. mallei</span>-vaccinated <span class="yellow">mice</span> [6]. Thus, full correlates of protection mediated by the adaptive immune system against <span class="yellow">B. mallei</span> remain to be fully elucidated.
In this series of studies, we sought to address the impact of depletion of the major effector lymphoid cell populations (B220+ B cells, CD4+ or CD8+ T cells) and key pro-inflammatory/Type 1 cytokines (IFN-γ or TNF-α) on survival in BALB/c <span class="yellow">mice</span> vaccinated with heat killed (HK) bacilli followed by an intraperitoneal (i.p.) challenge with live organism. In addition, studies investigating the effect of complement on opsonization of organism and antibody production were assessed. Heat killed bacteria were used as a model of vaccination to allow evaluation of <span class="yellow">B. mallei</span> specific immune responses. The results of this study demonstrate participation of B220+ cells and pro-inflammatory cytokines IFN-γ and TNF-α in protection following HK vaccination.<br><br>Results
Heat-killed <span class="yellow">B. mallei</span> vaccination mediates partial protection from lethal challenge
To begin to address this issue in an animal model of acute infection, we established that immunologically naive BALB/c <span class="yellow">mice</span> challenged i.p. with 2 × 107 CFU resulted in death by day 4–6, while i.p. immunization with 1 × 105 heat killed (HK) bacteria provided partial protection against a subsequent challenge. Two independent experiments resulted in similar findings of 40% survival for HK-vaccinated <span class="yellow">mice</span> with a mean survival time (MST) of 8 days versus 4 days in naïve <span class="yellow">mice</span> (Fig. 1). The administration of vaccines for <span class="yellow">B. mallei</span> during an outbreak would mandate relatively rapid onset of protection for <span class="yellow">human</span> or veterinary use. Based on non-routine use and vaccine implementation in the course of an outbreak, a 14 day window was chosen for assessment of protection. Our results indicate that HK vaccination can afford partial protection to an otherwise lethal challenge of <span class="yellow">B. mallei</span> by the i.p. route.<br><br>Effects of cell depletion on HK-vaccinated survival
To dissect the cellular basis for protection mediated by HK vaccination, 13 days after immunization with HK bacteria (day -1), and at day of challenge, <span class="yellow">mice</span> were dosed with antibodies to deplete CD4+, CD8+ or B220+ cells. Antibody depletion of CD4+, CD8+, or B220+ cells in these <span class="yellow">mice</span> was confirmed by flow cytometric analysis with depletion efficiencies for CD4, CD8, and B220 populations at 99.7%, 96%, and 95%, respectively, relative to <span class="yellow">mice</span> treated with isotype control monoclonal antibodies (data not shown). Our results demonstrated decreased survival rates in B220 (p = 0.3418), CD4+ (p = 0.5417) and CD8+ (p = 0.4684) antibody depleted <span class="yellow">mice</span>, compared to isotype control antibody, a finding that indicated a possible role for vaccine induced antibody production. When challenged with 2 × 107 CFU/<span class="yellow">mouse</span> by the i.p. route, loss of T cells resulted in reduced survival (50%) relative to the non-specific isotype control (Fig. 2). In contrast to the loss of T cells, depletion of B220+ cells resulted in 100% mortality relative to the non-specific isotype control (Fig. 2). To further evaluate the necessity of these effector cells in providing protection following HK vaccination, relatively resistant C57BL/6 <span class="yellow">mice</span>, deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) were subjected to an identical HK vaccination and challenge regimen. Mature, B-cell-deficient <span class="yellow">mice</span> demonstrated a 50% decreased survival (p = 0.0888) compared to the wild-type <span class="yellow">mice</span> with an MST of 35.5 days (Fig. 3). CD4-/- and CD8-/- <span class="yellow">mice</span> exhibited a 60% (p = 0.1343) and 0% reduced survival, respectively (Fig. 3).<br><br>Effects of cytokine depletion on HK vaccination
Similar studies were performed to determine the role of IFN-γ or TNF-α in acute infection in BALB/c <span class="yellow">mice</span> immunized with HK bacteria. Six hours before challenge, <span class="yellow">mice</span> were dosed with antibodies that neutralize IFN-γ or TNF-α. Individual depletion of either TNF-α (p = 0.0145) or IFN-γ (p = 0.0446) resulted in 100% mortality with an MST of 3 and 2 days, respectively, compared to the HK-vaccinated isotype control <span class="yellow">mice</span> (Fig. 4). In contrast, 40% of HK-vaccinated, isotype control <span class="yellow">mice</span> survived to at least 12 days post-challenge (Fig 4). To further evaluate the host TNF-α response during an established <span class="yellow">B. mallei</span> chronic infection, we infected 12 BALB/c <span class="yellow">mice</span> by the i.p. route with 1 × 106 CFU <span class="yellow">B. mallei</span>. One animal was terminally ill on day 37 post-infection. On day 42 post-infection, the remaining 11 <span class="yellow">mice</span> were dosed with either anti-TNF-α (n = 6), or control mAb (AFRC Mac 49) (n = 5). No further deaths were observed in the control mAb-treated <span class="yellow">mice</span>. Rapid mortality was observed in the anti-TNF-α-treated group, with all <span class="yellow">mice</span> dying within 7 days of treatment (p = 0.0023) relative to the isotype-treated controls (Fig. 5).<br><br>J774A.1 uptake of serum treated <span class="yellow">B. mallei</span>
Complement mediated uptake assays were performed to evaluate opsonization. Results indicated enhanced bacterial uptake in J774A.1 phagocytes inoculated with serum treated <span class="yellow">B. mallei</span> (p = .0082), compared to <span class="yellow">B. mallei</span> alone, while heat-inactivated serum produced uptake percentages similar to those prior to serum addition (Fig. 6). Taken together, these results imply an active role for complement components in the uptake of organism by macrophages.<br><br>Immunoglobulin production in HK vaccinated BALB/c <span class="yellow">mice</span>
We further characterized the ability of HK vaccination to induce a predominant IgG isotype by determining IgG2a/IgG1 ratios in i.p. and i.n. vaccinated BALB/c <span class="yellow">mice</span>. Pre (day 14 post vaccination) and post (day 2 post infection) exposure serum samples were obtained and evaluated for IgG isotype concentrations (Table 1). No appreciable differences in IgG pre-exposure levels were seen when comparing i.n. to i.p. vaccination. In addition, cobra venom factor-treated animals showed no significant differences to non-cobra venom factor-treated animals in IgG pre-exposure (challenge) levels. Conversely, isotype switching in the cobra venom factor treated animals was enhanced in post-exposure serum IgG2a (Table 1).<br><br>
Discussion
Recent studies have shown a key role in protection from lethal challenge for IFN-γ in non-vaccinated <span class="yellow">mice</span> from either NK and/or NKT cells following experimental exposure to <span class="yellow">B. mallei</span> and <span class="yellow">B. pseudomallei</span> [7,8]. A similar protective role in the innate response to infection has been demonstrated for TNF-α in <span class="yellow">B. pseudomallei</span> infection [8]. The studies presented here are consistent with the essential role of these factors in the relative levels of protection conferred by vaccination with heat-killed <span class="yellow">B. pseudomallei</span> and would appear to be viable early markers for protection from lethal acute infection [9]. Currently, there are no fully protective vaccines against <span class="yellow">B. mallei</span> or <span class="yellow">B. pseudomallei</span> in a <span class="yellow">murine</span> model, particularly for the sensitive BALB/c versus C57BL6 models. Previous studies have also demonstrated that both the humoral and cell-mediated arms are essential for protection from <span class="yellow">B. pseudomallei</span> infection [10]. Thus, loss or reduction of TNF-α and IFN-γ levels result in significantly reduced survival rates, substantiating previous reports of the role of these factors in protection against <span class="yellow">B. mallei</span> [7]. Moreover, we demonstrate a role for sustained TNF-α production in the maintenance of host survival throughout the course of <span class="yellow">B. mallei</span> infection. <span class="yellow">Mice</span> with an established <span class="yellow">B. mallei</span> chronic infection rapidly lost the ability to control the growth of the bacillus upon neutralization of TNF-α. This would suggest a potential role for TNF-α in the maintenance of productive granulomas which may limit the spread of bacteria in chronically infected hosts, or, alternatively, in direct or indirect microbicidal or bacteriostatic activities at the sites of infection. Additional studies are underway to determine more precisely the role of TNF-α in host protection to <span class="yellow">B. mallei</span>.
Multiple innate and adaptive cell types may contribute to the production of IFN-γ in response to infection with <span class="yellow">B. mallei</span> following vaccination. Our results with individual depletion of CD4+ and CD8+ T cells suggests that both cell types may compensate for the functional loss of the other effector cell type in the production of this key cytokine. The effector role for IFN-γ in mediating protection against <span class="yellow">B. mallei</span> may include both immunoregulatory and non-regulatory functions. Regardless, the requirement of IFN-γ, as demonstrated by administration of neutralizing antibody prior to infection, indicates that stimulation of IFN-γ response is a desirable goal for a <span class="yellow">B. mallei</span> vaccine.
Similarly, B220-positive cells appear to play a role in protection following vaccination with heat-killed <span class="yellow">B. mallei</span>. Interestingly, this protective immunity, occurring in other intracellular pathogens, is not exclusively dependent on B cells [11]. Passive protection has been demonstrated against acute Burkholderia infection by monoclonal antibodies [5,12]. Protection against <span class="yellow">B. pseudomallei</span> infection by anti-LPS, capsular polysaccharide and proteins has been short-lived, suggesting that antibody production offers limited protection in the initial stages of infection by an as-yet-undefined mechanism [12]. We have shown that following depletion of B220+ cells, survival rates decreased as much as 100% relative to non-depleted controls and individual CD4/CD8-depleted <span class="yellow">mice</span> via the intraperitoneal route. Results from C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) or CD8 T-cells (CD8-/-) substantiate the requirement for B-cell involvement by evidence of μMT and CD4-/- decreased survival. The lack of an effective CTL response to vaccination did not appear to alter survival in what would appear to be a CD4/B-cell (humoral)-driven response. In CD4-deficient <span class="yellow">mice</span>, we have the additional potential variable that a CD4-dependent antibody response might also be inhibited during the vaccination phase relative to <span class="yellow">mice</span> treated with antibody immediately prior to and during the early phases of infection. Although not statistically significant, we did observe a decrease in survival in μMT (mature B cell) deficient <span class="yellow">mice</span> as early as day 9 post challenge, whereas CD4-deficient <span class="yellow">mice</span> produced similar results at day 32 post challenge, indicating a role for B cells independent of CD4 T cell help, perhaps through a T-independent mechanism of antibody production. Although CD8-/- C57BL/6 demonstrated no decreased survival in our HK-vaccinated model, a lack of potential endogenous protein production by HK <span class="yellow">B. mallei</span> may have contributed to limited MHC-I presentation.
Complement associated studies revealed increased J774A.1 uptake of serum-treated <span class="yellow">B. mallei</span>. Complement-mediated uptake studies of <span class="yellow">B. pseudomallei</span> by polymorphonuclear leukocytes (PMNs) suggest that capsule production contributes to resistance of phagocytosis by reducing C3b bacterial deposition [13]. Previous studies have demonstrated that a polysaccharide capsule is present in <span class="yellow">B. mallei</span>, [14,15] although in the present study enhanced uptake with serum-treated <span class="yellow">B. mallei</span> was observed. Intracellular survival assays of complement mediated uptake of organisms were not performed in the present study, thus, the role of complement opsonization on intracellular survival is not fully resolved. Previous reports have demonstrated the ability of <span class="yellow">B. mallei</span> to survive within macrophage without the aid of serum coating organisms [16]. Conversely, the idea of antibody mediated opsonization to facilitate macrophage activation and clearance of intracellular organisms may offer support to the role of B cells in an effective immune response. A possible protective mechanism may include HK vaccination induced production of opsonizing antibodies which may aid in complement mediated uptake, thereby limiting the initial bacterial threshold below a lethal level.
Immunoglobulin responses to HK vaccination resulted in modest levels of IgG1 following 2 weeks post vaccination, while post-exposure levels were indicative of efficient class switching to a favorable IgG2a isotype. Importantly, cobra venom factor treatment of animals at time of vaccination did not alter their ability to produce immunoglobulin. In fact, cobra venom factor treated animals resulted in higher IgG2a levels when compared to non-treated. Complement activation can modulate both the primary and secondary immune responses and has been shown to enhance secondary immune responses to vaccination [17]. The current results suggest that cobra venom factor treatment may affect the modulation of the immune response to <span class="yellow">B. mallei</span> infection through B cell activation and/or memory B cell generation.<br><br>Conclusion
In summary, our results provide a basis for future studies of protection via vaccination using either subunit or whole-organism vaccine preparations from lethal infection in the experimental BALB/c <span class="yellow">mouse</span> model. Understanding and defining the role of B cells in adaptive <span class="yellow">B. mallei</span> immunity will likely be fundamental to the design of an efficacious vaccine and important goals of future research.<br><br>Methods
Bacterial strain and <span class="yellow">mice</span>
<span class="yellow">B. mallei</span> strain ATCC 23344 (China 7) was cultured on Luria-Bertani agar supplemented with 4% glycerol (LB+4%G) agar plates for 48 h at 37°C. Isolated colonies were sub-cultured to LB+4%G broth, and cultures were incubated at 37°C until optical density readings at 600 nm (OD600) reached an exponential phase of growth. Bacteria were pelleted by centrifugation, washed and re-suspended in sterile 1× phosphate-buffered saline (PBS, pH 7.4) to obtain the desired CFU/ml. To obtain HK inoculums, bacterial suspensions were incubated at 85°C for 3 h and stored at 4°C until use. The absence of live <span class="yellow">B. mallei</span> organisms in the HK preparations was confirmed after plating 10% of the total inoculums (v/v) and incubating these at 37°C for 48 h. All procedures were performed under a class II biosafety cabinet in a biosafety level 3 laboratory. Female, 6- to 8-week-old, BALB/c <span class="yellow">mice</span> (n = 5–7) were obtained from Harlan Sprague Dawley, Inc. (Indianapolis, Indiana). Female, 6- to 8-week-old, C57BL/6 <span class="yellow">mice</span> deficient in mature B-cells (μMT), CD4 T-cells (CD4-/-) and CD8 T-cells (CD8-/-) and wild-type <span class="yellow">mice</span> were obtained from The Jackson Laboratory (Bar Harbor, Maine).<br><br>Vaccination and challenge
BALB/c and C57BL/6 <span class="yellow">mice</span> were grouped and vaccinated with 0.5 μg of HK <span class="yellow">B. mallei</span> (without adjuvant) by i.p. injection using a 25-gauge syringe. Two weeks post HK vaccination <span class="yellow">mice</span> were injected i.p. with 2 × 107 CFU/100 μl of live <span class="yellow">B. mallei</span> (~20 LD50) [18]. Complement depleted animals were challenged with 2.5 × 104 CFU/50 μl (~0.25 LD50) by intranasal (i.n.) route. Aliquots from the inoculums were plated to confirm the infecting dose. All procedures and animal protocols used in this study were approved by the Biosafety and IACUC committees at UTMB and conducted in either BSL-3 or ABSL-3 laboratories.<br><br>Cell and cytokine depletions
Acute in vivo cell/cytokine depletion was performed with monoclonal <span class="blue">rat</span> anti-<span class="yellow">mouse</span> CD4 (GK1.5), CD8α (53-6.7) or B220 (RA3-6B2) obtained from R&D Systems, Inc. (Minneapolis, MN) by methods similar to those we have previously described [19]. Functional grade purified <span class="blue">rat</span> anti-<span class="yellow">mouse</span> interferon-gamma (IFN-γ, AN-18) was obtained from eBioscience (San Diego, CA) and purified anti-<span class="yellow">mouse</span> tumor necrosis factor (TNF-α, MP6-XT3) from BD Pharmingen (San Diego, CA). IFN-γ and TNF-α antibodies were injected i.p. 6 h prior to challenge, 200 μg per <span class="yellow">mouse</span> in 200 μl PBS or at later time points as indicated. <span class="yellow">Rat</span> IgG isotype control was obtained from Southern Biotech (Birmingham, AL) and administered i.p. on day of challenge, 200 μg/<span class="yellow">mouse</span>. <span class="yellow">Rat</span> anti-<span class="yellow">mouse</span> CD4, CD8α and B220 were injected i.p. twice, 1 day prior to challenge and on day of challenge, with an equivalent dosage sufficient to deplete T or B cells from 6 × 108 bone marrow cells per injection. The efficiency of depletion at time of infection for CD4+, CD8+, and B220+ cells was confirmed by flow cytometry analysis immediately prior to infection.<br><br>Complement depletion with cobra venom factor
<span class="yellow">Mice</span>, six to seven per group, were vaccinated i.p. with 1 × 105 CFU of nonviable <span class="yellow">B. mallei</span> cell preparation in a total volume of 0.1 ml. Two weeks later, 24 h and 1 h before challenge, complement depleted <span class="yellow">mice</span> were treated i.p. with 12.5 units total cobra venom factor (Quidel Corporation Speciality Products, San Diego, CA) in 0.1 ml of PBS. Complement depletion was confirmed prior to challenge by micro-titer hemolytic complement activity (CH50) assay as previously described [20].<br><br><span class="yellow">B. mallei</span> J774A.1 uptake assays
J774A.1 cells were seeded (5 × 105) onto Corning costar 24 well plates (Corning, NY) with DMEM and incubated overnight at 37°C with 5% CO2. Bacterial suspensions were incubated at 37°C for 45 minutes supplemented with 2% <span class="yellow">mouse</span> serum from Sigma-Aldrich (St. Louis, MO.), heat inactivated <span class="yellow">mouse</span> serum (56°C 30 minutes), or bacteria alone and then added at an MOI of 10:1 to J774A.1 cells in triplicate. Inoculated wells were centrifuged at 800 g for 2 minutes and incubated for 2 hours at 37°C with 5% CO2 followed by a PBS wash (×2) and 2 hour incubation with 250 μg/ml kanamycin. Wells were washed twice with PBS and lysed with 0.1% Triton X-100, followed by serial 10-fold dilutions plated on LBG plates and incubated at 37°C for 2 days. Colony forming units were enumerated and uptake expressed as a percentage of initial inoculating dose ± SEM.<br><br>Antibodies and flow cytometry
Flow cytometric analysis was performed on 0.1-ml blood samples transferred to micro centrifuge tubes containing 90 μl of acid citrate dextrose (ACD) solution. Red blood cells were lysed using ACK-lysing buffer (Biosource International, Inc., Camarillo, CA) according to the manufacturer's instruction. Antibodies used for analysis of surface markers included: FITC-conjugated <span class="blue">rat</span> anti-<span class="yellow">mouse</span> CD45R/B220 (RA3-6B2, BD Pharmingen San Diego, CA) for B cells; FITC-conjugated <span class="blue">rat</span> anti-<span class="yellow">mouse</span> CD8α (53-6.7) and CD4 (GK1.5, BD Pharmingen, San Diego, CA) for CD8+ or CD4+ cells, respectively. Samples evaluated for CD4+ and CD8α+ cells were also incubated with biotin-conjugated <span class="yellow">hamster</span> anti-<span class="yellow">mouse</span> CD3e (145-2C11) monoclonal antibody (BD Pharmingen, San Diego, CA) and subsequently with streptavidin APC Cy7. Isotype-matched, non-specific controls were assayed in parallel (BD Pharmingen, San Diego, CA). Surface staining was performed according to previously published protocols [21]. Following cell staining, the samples were fixed with 2% buffered paraformaldehyde overnight prior to analysis by flow cytometry. Samples were analyzed using a FACSCalibur flow cytometer with BD CellQuest Pro software.<br><br>Antibody assays
Immunoglobulin subclass IgG1 and IgG2a titers in <span class="yellow">mice</span> were determined by a whole bacterial cell ELISA performed in 96-well, Immulon 2 HB, round-bottom plates (Dynex Technologies). <span class="yellow">B. mallei</span> antigen was diluted in 0.1 M carbonate buffer (pH 9.5) and 50 μl of diluted cells placed into wells. Plates were stored overnight at 4°C. The plates were washed with washing solution (1× PBS, 0.05% Tween 20), and incubated with 100 μl of blocking solution (1× PBS, 1% <span class="yellow">bovine</span> serum albumin, 0.05% Tween 20) for 1 h at 37°C. Dilutions of <span class="yellow">mouse</span> sera were made with blocking solution in duplicate and plates were incubated for 1 h at 37°C. Following incubation, plates were washed and 50 μl of anti-Ig-<span class="yellow">horseradish</span> peroxidase subclass conjugate, diluted accordingly to manufacturer's instructions (Southern Biotechnology Associates, Inc. Birmingham, Ala.), was added to each well and incubated for 1 h at 37°C. After washing, 50 μl of 2,2'-azino-di-(3-ethylbenzthizoline)-6-sulfonate (ABTS) peroxidase substrate (KPL, Inc., Gaithersburg, Maryland) was added to each well and plates incubated for 25 min at room temperature. The amount of bound antibody was determined colorimetrically by absorbance at 405 nm.<br><br>Statistical analysis
Survival curves were calculated by Kaplan Meier survival analysis with log-rank tests between groups using GraphPad Prism (V.4.03 for windows). Statistical analysis was generally performed with the paired Student's t-test. P value ≤ 0.05 was considered significant.<br><br>
Abbreviations
HK: Heat-killed; i.p.: intraperitoneal; i.n.: intranasal.<br><br>Authors' contributions
GCW designed and conducted experiments and drafted the manuscript. BMJ carried out the immunoassays and animal work. SP provided analysis of data and contributed to design and animal work. RAL participated in the generation and analysis of chronic TNF-α data. DME conceived the study, and participated in its design and coordination and helped to draft the manuscript. AGT participated in the bacterial work and drafting of the manuscript. All authors read and approved the final manuscript.<br><br>
</body></html>