
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>rice</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="blue">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="yellow">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="yellow">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="yellow">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="blue">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="yellow">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="yellow">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="yellow">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="yellow">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="yellow">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="yellow">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="yellow">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="yellow">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="yellow">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="yellow">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="yellow">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="blue">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="blue">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="yellow">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="yellow">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
<h3>pmcA1312363</h3>Gene expression studies in isolated mitochondria: <span class="yellow">Solanum tuberosum</span> rps10 is recognized by cognate <span class="yellow">potato</span> but not by the transcription, splicing and editing machinery of <span class="yellow">wheat</span> mitochondria
Abstract
The complex gene expression mechanisms that occur in plant mitochondria, such as RNA editing and splicing, are not yet well understood. RNA editing in higher plant mitochondria is a highly specific process which modifies mRNA sequences by C-to-U conversions. It has been suggested that in some cases this process is required for splicing. Here, we use an experimental model based on the introduction of DNA into isolated mitochondria by electroporation to study organellar gene expression events. Our aim was to compare processing and editing of <span class="yellow">potato</span> small ribosomal protein 10 gene (rps10) transcripts in heterologous (<span class="yellow">wheat</span> mitochondria) and homologous (<span class="yellow">potato</span> mitochondria) contexts. rps10 is a suitable model because it contains a group II intron, is absent in <span class="yellow">wheat</span> mitochondria but is actively expressed in <span class="yellow">potato</span> mitochondria, where transcripts are spliced and undergo five C-to-U editing events. For this purpose, conditions for electroporating isolated <span class="yellow">potato</span> mitochondria were established. rps10 was placed under the control of either <span class="yellow">potato</span> or <span class="yellow">wheat</span> cox2 promoters. We found that rps10 was only transcribed under the control of a cognate promoter. In <span class="yellow">wheat</span> mitochondria, rps10 transcripts were neither spliced nor edited while they are correctly processed in <span class="yellow">potato</span> mitochondria. Interestingly, a <span class="yellow">wheat</span> editing site grafted into rps10 was not recognized by <span class="yellow">wheat</span> mitochondria but was correctly edited in <span class="yellow">potato</span> mitochondria. Taken together, these results suggest that editing might occur only when the transcripts are engaged in processing and that they would not be available to editing factors outside of a putative RNA maturation machinery complex.<br><br>INTRODUCTION
Gene expression in plant mitochondria is a complex process involving multiple steps such as transcription, cis- and trans-splicing, RNA trimming and RNA editing on the way to translation (1). RNA editing has been found in a variety of organisms and occurs through different mechanisms such as insertion or deletion of nucleotides, or base conversions [for details see reference (2)]. In higher plant organelles, RNA editing is an important post-transcriptional event characterized by C-to-U changes via a deamination mechanism (3–5). In <span class="yellow">Arabidopsis thaliana</span> 456 C-to-U editing events have been described (6). RNA editing occurs mostly in the coding regions which alter the identity of the encoded amino acid, but some editing events occur in highly structured regions of introns (7,8). Like other maturation processes, RNA editing is an essential post-transcriptional event in plant mitochondrial gene expression (9–11) required for the synthesis of functional proteins (12,13).
The introduction of foreign DNA into isolated mitochondria is a novel experimental model which has provided important information on RNA editing and RNA splicing enabling the use of a site-directed mutagenesis approach (14,15). Using a cognate cox2 chimeric gene construct, the cis-recognition elements required for plant mitochondria RNA editing have been determined (16,17). Using the same approach, Staudinger and Kempken (18) have reported that transcripts from <span class="yellow">A.thaliana</span> cox2, but not <span class="yellow">Sorghum bicolor</span> atp6, are edited when the genes are introduced into <span class="yellow">maize</span> mitochondria.
Interestingly, higher plant mitochondrial genomes differ in their gene contents due to an evolutionary information transfer from the organelle to the nucleus (19). Of particular interest is the situation of the small ribosomal protein 10 gene (rps10), a group II intron-bearing gene which is encoded in <span class="yellow">Solanum tuberosum</span> mitochondrial DNA but is absent from the <span class="yellow">wheat</span> mitochondrial genome (20). Higher plant mitochondria contain group II introns either in cis- or trans- configuration (1) which can be folded in a characteristic secondary structure (21). Intron removal in plant mitochondrial mRNAs is not well documented because such introns are unable to self-splice. Previously, we described that electroporated cox2 constructs were a good model for the study of the splicing process. Using this model, we found that editing and splicing of cox2 transcripts were not linked in <span class="yellow">wheat</span> mitochondria (15).
To challenge the ability of mitochondrial gene expression machinery to recognize genetic information which has been lost during evolution, we decided to introduce the <span class="yellow">S.tuberosum</span> non-cognate rps10 gene into <span class="yellow">wheat</span> mitochondria. Five C residues are changed to U in <span class="yellow">potato</span> mitochondria rps10 transcripts by editing. Two of them have been postulated as being necessary for acquisition of a proper secondary and/or tertiary structure for splicing. To test this hypothesis, it was necessary to set up the conditions for electroporation of foreign DNA into <span class="yellow">S.tuberosum</span> mitochondria. Here, we show that a <span class="yellow">potato</span> rps10 construct is transcribed when introduced into <span class="yellow">wheat</span> mitochondria, but transcripts are not recognized by the post-transcriptional processing machinery. In contrast, a rps10 construct is correctly expressed and processed in cognate <span class="yellow">potato</span> mitochondria. Moreover, we present evidence that transcript editing might be linked to overall RNA processing. This is the first report on DNA electroporation into <span class="yellow">potato</span> mitochondria.<br><br>MATERIALS AND METHODS
Plasmids
All plasmids used in this study are based on the previously described pCOXII vector (14). An NsiI restriction site was introduced at the initiation codon of the <span class="yellow">wheat</span> cox2 open reading frame (ORF). Then, NsiI was used in combination with a SpeI restriction site present in the original vector after the stop codon, to produce the chimeric vectors.
<span class="yellow">S.tuberosum</span> cox2 gene, including a 727 bp non-coding upstream region, was isolated by PCR from total DNA using the primer A designed from partial sequences reported by Löessl et al. (22), accession no. AF096321, containing the KpnI restriction sequence, and primer B derived from <span class="yellow">Triticum timopheevi</span> cox2 (AF336134). A fragment of 2888 bp was obtained and cloned on pGEM-T vector. The complete sequence of the <span class="yellow">S.tuberosum</span> cox2 was determined (accession no. DQ18064). To generate the plasmid pCOXIISt containing the <span class="yellow">S.tuberosum</span> cox2 gene, a KpnI/SpeI fragment containing the 727 bp upstream region and the complete coding region was used to replace the <span class="yellow">wheat</span> gene from pCOXII. As for pCOXII, a NsiI site was inserted at the ATG start codon and a 23 bp fragment was inserted at position −20. This 23 bp insertion provides a specific sequence allowing isolation of <span class="yellow">potato</span> cox2 transcripts originating from the introduced DNA by RT–PCR. The chimeric <span class="yellow">wheat</span>/<span class="yellow">potato</span> cox2 gene was constructed by replacing 727 bp KpnI/NsiI promoter sequence from pCOXIISt with the 880 bp <span class="yellow">wheat</span> upstream region from pCOXII.
The vectors pRPS10W and the pRPS10St derivative, containing <span class="yellow">wheat</span> and <span class="yellow">potato</span> cox2 promoters, respectively, were constructed by inserting the 1178 bp rps10 sequence containing two exons separated by a 777 bp intronic sequence [(23), accession no. X74826]. The coding region was isolated from total <span class="yellow">S.tuberosum</span> DNA by PCR using primers D1 and D2 containing the restriction sites NsiI and SpeI, respectively. The fragment NsiI/SpeI was purified and cloned into pCOXII and pCOXIISt, replacing the respective cox2 coding regions.
Since all vectors used here were based on pCOXII, they contain the downstream region from the <span class="yellow">wheat</span> cob gene (Ir-cob) (accession no. AF337547) (14). This sequence, combined with the 23 bp upstream insert sequence served to distinguish, using primers 1a and 1b, foreign from endogenous transcripts. All mutants were obtained using QuickChange® Site-Directed Mutagenesis kit (Stratagene). PCR product purifications were carried out with the Wizard® Clean-Up System (Promega).<br><br>PCR primers used
Oligonucleotides sequences are in 5′ to 3′ orientation. (1a), GCGGTGCAGTCATACAGATCTGC; (1b), TATCCAGATTTGGTACCAAAC; (2a), GCAGTCATACAGATCTGCCAAAG; (2b), AGATTTGGTACCAAACCCGGG; (A), TATAGGTACCTCTCAGGTGTCAAAGTGTGGATTT; (B), TATAACTAGTTTAAGCTTCCCCG; (D1), TATAATGCATAGACAAAGGAGAGCACTTA; (D2), TATAACTAGTTCAGGAAAGGGTCAACGCAA. Restriction sites are underlined.<br><br>Mutagenesis primers (only sense primers in 5′–3′ orientation are indicated)
Single C2, C3 and C2+C3 double mutant plasmids were constructed with primers (C2) AAGAAGTTCTTTTGGTTAAAACGCC and (C3) CGCCGTGCGACTTGGAGGACATAAG. NsiI-pCOXII: GGAAATCCAATGCATCTTCGTTCATT and NsiI-pCOXIISt: CCAAACCAAATGCATGTTCTAGAATG. Construct pCOXIISt containing a 23 nt insertion in the promoter region, was carried out into two consecutive insertions using primers: TGGGGGGAGCAGAGCAGTGCGGTGCAGTCACAAAGAATGAACCAAACC and GCAGTGCGGTGCAGTCATACAGATCTGCCAAAGAATGAACCAAACC. For constructs MA and MAB containing the <span class="yellow">wheat</span> cox2 C259 editing site, two consecutive insertions were carried out using primers: GGAAGATTGGATTACTATCGAAATTGCCCTGAATCA and TACTATCGAAATTATTCGGACCATGCCCTGAATCA. For ME6 and ME6b constructs, primers CCGCGAGGAATCAACTACTATCGAAATTATTGCCGGTGCTGAC and CAACTACTATCGAAATTATTCGGACCATATTGCCGGTGCTGAC were used. Inserted nucleotides are underlined and the modified residues are indicated in bold.<br><br>Mitochondria purification
<span class="yellow">S.tuberosum</span> cv. Rosevalt tubers and <span class="yellow">T.aestivum var Fortal</span> seeds were used. <span class="yellow">Potato</span> mitochondria were prepared from 2 kg of tubers in batches of 200 g with 200 ml of a homogenization buffer containing 0.4 M mannitol, 25 mM MOPS (pH 7.8), 1 mM EGTA, 8 mM cysteine and 1 mg/ml fatty acid-free BSA. Homogenization was carried out for 15 s in a Waring blendor at full speed. Homogenate was centrifuged in a Sorvall GSA rotor at 1500 g for 10 min at 4°C. Supernatant was centrifuged in a GSA rotor at 12 000 g for 15 min. The mitochondrial pellet was resuspended in 50 ml of homogenization buffer and centrifuged at 1500 g. The supernatant was centrifuged in a Sorvall SS-34 rotor at 15 000 g for 10 min, the pellet was resuspended in 12 ml of homogenization buffer and mitochondria were purified by centrifugation on a sucrose gradient essentially as described for <span class="yellow">wheat</span> embryo mitochondria (14).<br><br>Electroporation
Electrotransfer experiments were carried out with 1 mg of purified <span class="yellow">wheat</span> embryo or <span class="yellow">potato</span> tuber mitochondria in 50 µl of 0.33 M sucrose and 1 µg of recombinant plasmid as described (14). The electroporated mitochondria were incubated for 18 h at 25°C, then recovered by centrifugation in a Sigma N° 12024 rotor (Sigma 1K15 refrigerated centrifuge) at 15 000 g for 15 min at 4°C. In the case of <span class="yellow">potato</span> mitochondria the incubation mixture was supplemented with 1 mg/ml of fatty acid-free BSA. RNA was purified with 200 µl of TRIzol™ reagent (Gibco-BRL) according to the supplier's protocol.<br><br>DNAse I protection assay and DNA purification
After electroporation and centrifugation, the mitochondrial pellet was resuspended in 100 µl of buffer [10 mM Tris–HCl (pH 7.5), 2 mM magnesium acetate and 0.33 M sucrose] containing 60 U of DNase I (Gibco-BRL) and incubated for 1 h at room temperature. The DNase reaction was stopped by adding 4 µl of 0.5 M EDTA and then heating for 10 min at 65°C. Mitochondria were incubated with 100 µg of Proteinase K (Merck) for 4 h at 37°C. One microliter of 20% SDS was added to achieve mitochondrial lysis and the DNA was extracted with phenol/chloroform, precipitated with 0.1 vol of 3 M Sodium Acetate (pH 5.2), 3 vol of 100% ethanol and 100 ng of carrier <span class="yellow">yeast</span> tRNA and left overnight at −20°C. After centrifugation, the DNA pellet was resuspended in TE buffer [10 mM Tris–HCl (pH 8) and 1 mM EDTA].<br><br>RT–PCR
RNA (1 µg) was treated with 2 U of Amplification grade DNase I (Promega). cDNA synthesis was performed with 200 U of Superscript II RT using 100 ng of random hexamers. The PCR were performed with primers 1a and 1b using Advantage® 2 Polymerase Mix (Clontech) as follows: 95°C, 1 min; 5 cycles at 95°C for 30 s and 68°C for 1 min; 30 cycles at 95°C for 30 s, 58°C for 30 s and 68°C for 30 s, and finally 68°C for 1 min. Primers 2a and 2b were used for nested PCR from 1 µl of the first PCR. No RT–PCR amplification products were obtained with RNA from non-electroporated mitochondria.<br><br>DNA sequencing
Sequence analyses were performed directly on the RT–PCR product using an automatic DNA sequencing equipment with the BigDye® Terminator Cycle Sequencing Kit (Applied Biosystem).<br><br>
RESULTS
<span class="yellow">S.tuberosum</span> rps10 transcripts are not processed in <span class="yellow">wheat</span> mitochondria
The <span class="yellow">S.tuberosum</span> rps10 construct under control of a <span class="yellow">wheat</span> cox2 promoter (Figure 1A) was incorporated into purified <span class="yellow">wheat</span> mitochondria by electroporation as indicated in Materials and Methods. After electroporation and incubation in expression medium, mitochondrial RNA was extracted and analysed by RT–PCR. The primers used allowed detection of the transcripts generated by the constructs introduced and excluded any product from endogenous cox2 (or rps10 in the <span class="yellow">potato</span> system). Two bands of 1204 and 427 bp were expected from precursor and mature rps10 mRNAs, respectively. Only precursor rps10 molecules were detected (Figure 1B). As a control, a cognate cox2 construct (16) was used. In this case, the 2048 and 827 bp RT–PCR bands representing the precursor and spliced cox2 products, respectively were observed (Figure 1B). In all cases, the PCR bands actually represent transcription products since when PCR was performed without the RT step we observed no amplification products.
Previously, five C residues, two in exon 1, one in the intron and two in exon 2 have been reported to be changed to U by editing in <span class="yellow">potato</span> mitochondria (23). To determine if the non-cognate transcript could be recognized by the <span class="yellow">wheat</span> RNA editing machinery, the 1204 bp RT–PCR product was sequenced. While <span class="yellow">wheat</span> cox2 editing sites were correctly edited in control as expected (Figure 1C, only site C77 is shown), all five editable residues in <span class="yellow">potato</span> rps10 transcript remain unchanged. cox2 C77 editing sites are identical in <span class="yellow">potato</span> and <span class="yellow">wheat</span>.<br><br>Failure of rps10 transcript splicing in <span class="yellow">wheat</span> mitochondria is not linked to the absence of editing
It has been suggested that residues C2 and C3 participate in the secondary structure of the intron necessary for splicing (23). A possible explanation to the failure observed in precursor rps10 splicing could be that the absence of edition of C2 and C3 prevent the intron from organizing itself in a catalytically competent conformation. To test this hypothesis, single C2 and C3 mutants and a C2+C3 double mutant were constructed by changing the C residues to T. Neither single nor double mutants were able to undergo splicing (Figure 2).<br><br>Electroporation of <span class="yellow">S.tuberosum</span> mitochondria
Since two important post-transcriptional processes, RNA editing and splicing, were inoperative when <span class="yellow">S.tuberosum</span> rps10 was expressed in <span class="yellow">wheat</span> mitochondria, we decided to verify whether the negative results were inherent to the rps10 chimeric constructs or due to the lack of trans-recognition elements in <span class="yellow">wheat</span> mitochondria. For this purpose, it was necessary to set up an electroporation protocol adapted to <span class="yellow">S.tuberosum</span> mitochondria. Purified organelles were prepared from <span class="yellow">potato</span> tubers as indicated in Materials and Methods. Electric pulses in the range between 8 and 20 kV were performed. Internalization of exogenous DNA was measured by DNAse protection assays (14). <span class="yellow">Potato</span> mitochondria show a broad range response with a maximum around 13 kV (Figure 3). This voltage setting was therefore used for further experiments.<br><br><span class="yellow">S.tuberosum</span> mitochondria does not recognize the <span class="yellow">wheat</span> cox2 promoter
To ascertain the ability of electroporated mitochondria to perform expression of the exogenous gene construct, we used a plasmid containing the <span class="yellow">potato</span> cox2 gene controlled either by <span class="yellow">T.aestivum</span> or <span class="yellow">S.tuberosum</span> promoters. As shown in Figure 4, the construct containing the <span class="yellow">wheat</span> or <span class="yellow">potato</span> promoters was transcribed only in cognate mitochondria. In <span class="yellow">potato</span>, the mature product is barely detectable. In contrast to <span class="yellow">wheat</span> mitochondria, the 821 nt mature transcript was only detected when the electroporated <span class="yellow">potato</span> mitochondria were incubated in the presence of fatty acid-free BSA.<br><br>A cognate rps10 construct is correctly expressed, edited and processed in <span class="yellow">potato</span> mitochondria
The construct expressing <span class="yellow">potato</span> rps10 under the control of <span class="yellow">potato</span> cox2 promoter was introduced into <span class="yellow">S.tuberosum</span> isolated mitochondria. After incubation, the precursor and mature transcripts were amplified by RT–PCR (Figure 5A) and sequenced. As shown in Figure 5B, the four editing sites C1, C2, C4 and C5, described previously in endogenous transcripts, were found edited in mature mRNA. The presence of spliced molecules containing unedited residues (Figure 5B) indicates that rps10 transcripts could be spliced before editing. To confirm this observation, the PCR product was cloned and sequenced. One half of the individual clones were found edited at sites C1 and C2; 65% were edited at site C4 and 25% were edited at site C5.<br><br>A cognate editing site in a non-native context is not recognized by <span class="yellow">wheat</span> mitochondria but is recognized in heterologous mitochondria
To test whether <span class="yellow">wheat</span> mitochondria are able to edit a cognate site when placed in the context of a <span class="yellow">potato</span> transcript, we introduced the C259 −16/+6 region from <span class="yellow">wheat</span> cox2 into <span class="yellow">potato</span> rps10 exon 1 or intron (Figure 6A, construct MA and ME6, respectively). As reported previously, the 23 nt region forming the C259 editing site from <span class="yellow">wheat</span> cox2 was efficiently edited when grafted into a different context in a <span class="yellow">wheat</span> transcript (17). Wild-type rps10 (pRPS10W) and the MA and ME6 mutants were expressed in <span class="yellow">wheat</span> mitochondria but no splicing was observed (Figure 6B). Unexpectedly, <span class="yellow">wheat</span> mitochondria appear to be unable to edit the cognate C259 site when the −16/+6 region is located on a <span class="yellow">potato</span> rps10 precursor. The control CM construct, containing the C259 site grafted in a different context but in its own cox2 gene, was expressed, spliced and edited as expected [(17), Figure 6B and D]. To compare the editing status of RNAs at the same stage of processing, the editing status of the inserted C259 site in precursor cox2 transcripts is shown (Figure 6D). The analogous MAb and ME6b constructs, containing the <span class="yellow">S.tuberosum</span> cox2 promoter and the C259 region inserted into rps10 exon 1 and intron were used to electroporate <span class="yellow">potato</span> mitochondria. As shown in Figure 6C, MAb and ME6b were expressed and spliced at the same level as the control in the homologous context, indicating that insertion does not affect the splicing process. Interestingly, while C259 insertion was not recognized in <span class="yellow">wheat</span> mitochondria (Figure 6D), the <span class="yellow">potato</span> RNA editing machinery edited the inserted C259 region (Figure 6E). The C259 region inserted in the rps10 intron (construct ME6b) was also edited, although at a very low level, showing that editing may precede intron removal (not shown).<br><br>
DISCUSSION
Plant mitochondria undergo complex expression mechanisms which are poorly understood. Most studies are based on analysis of in vivo mature or intermediate gene products giving clues on possible mechanisms and suggesting pathways operating in gene expression. Direct tests using in vitro approaches have been fruitful (4,5,24), but are hampered by the difficulty of obtaining active mitochondrial extracts. Previously, we devised an experimental model based on electroporation of isolated mitochondria that allows us to test gene expression of wild-type and mutant genes. This approach has been very useful in elucidating the process of RNA maturation in plant mitochondria, in particular splicing and editing (14,15). The aim of this work was to analyse different gene expression events when a foreign gene was expressed in a heterologous context. rps10 was chosen precisely because this genetic information is lacking in <span class="yellow">wheat</span> mitochondria but is active in <span class="yellow">potato</span> mitochondria (20,23,25). Moreover, rps10 transcripts undergo five C-to-U editing events in <span class="yellow">potato</span> mitochondria and rps10 contains an intron, thus facilitating the analysis of post-transcriptional processing.
To best evaluate the expression of rps10 in the non-cognate mitochondria, it was necessary to set up electroporation conditions for introducing foreign DNA into <span class="yellow">S.tuberosum</span> mitochondria. <span class="yellow">Potato</span> tuber mitochondria were able to incorporate DNA essentially under the same conditions described for <span class="yellow">wheat</span> embryo organelles (14), except that the optimal voltage range was larger. A major difference to <span class="yellow">wheat</span> was that isolated <span class="yellow">potato</span> mitochondria were viable for 3–4 h as measured by oxygen consumption (data not shown). Thus, to observe transcript maturation after electroporation the incubation mixture needed to be supplemented with fatty acid-free BSA (see Figure 4). This behaviour probably reflects the uncoupling of oxidative phosphorylation by fatty acids during incubation of <span class="yellow">potato</span> mitochondria (26). In fact, previously we found that proper gene expression in isolated mitochondria requires a functional organelle able to generate ATP from ADP and succinate (14).
An interesting observation was that neither the <span class="yellow">wheat</span> nor the <span class="yellow">potato</span> cox2 promoters were recognized in a heterologous context (Figure 4A and B). This situation might by explained by the fact that <span class="yellow">potato</span> and <span class="yellow">wheat</span> promoter sequences have no recognizable homologous motifs. Plant mitochondria promoters are characterized by a conserved core CRTA sequence with differences in the extent and the composition of sequences around the consensus motif (27,28). While the transcription initiation site in <span class="yellow">wheat</span> mitochondria is located at position −170 (29), the <span class="yellow">potato</span> promoter has not yet been described. Transcription may be initiated at numerous sites suggesting a relaxed promoter recognition by the transcription machinery (28). However, the lack of crossed recognition of cox2 promoters in <span class="yellow">wheat</span> and <span class="yellow">potato</span> is not a general situation since the <span class="yellow">A.thaliana</span> cox2 gene is expressed and spliced when introduced into <span class="yellow">maize</span> mitochondria, indicating that the <span class="yellow">Arabidopsis</span> gene shares some signals with <span class="yellow">maize</span> cox2 promoter that are sufficient for transcription (18). The sites required for transcript initiation have been recently described in <span class="yellow">A.thaliana</span> mitochondrial genes (28). Three regions were described as important for transcription initiation. We found no such homologous sequences in the <span class="yellow">potato</span> cox2 upstream region indicating that the two dicot promoters do not have the same origin. This in turn may reflect the natural history of this particular mitochondrial gene evolving in its own context. It should be mentioned that the presence of a conserved sequence is not sufficient for expression since a region that acts as promoter in <span class="yellow">Arabidopsis</span>, <span class="yellow">potato</span> and Oenothera is inactive in vivo in <span class="yellow">pea</span> (30). Further studies will be required to understand the transcriptional events in plant mitochondria. Electroporation of foreign DNA into isolated mitochondria provides an interesting functional model, complementary to in vitro transcription assay, for answering these questions.
The <span class="yellow">potato</span> rps10 gene controlled by a <span class="yellow">T.aestivum</span> promoter (Figure 1A) was transcribed as a 1204 nt precursor in <span class="yellow">wheat</span> mitochondria, but no traces of mature RNA were observed. Moreover, the five C residues reported as RNA editing targets in vivo (23) remain unchanged, indicating that rps10 transcript was not recognized by the <span class="yellow">wheat</span> splicing and editing machinery. A control using the cognate cox2 construct demonstrates that electroporated organelles were competent for splicing and editing (Figure 1B and C).
Some editing events occur in highly structured domains of introns. Because in some cases, editing corrects A-C mispairing improving conformation of the intron, it has been proposed that the C-to-U change might be necessary for efficient splicing (8,9,23,31). Based on the canonical structure of group II mitochondrial introns (21), two editing sites, C2 located at the Intron Binding Site 2 (IBS2) and C3 located in the intron, nine residues downstream from the end of exon 1 in <span class="yellow">S.tuberosum</span> rps10 are of particular interest. It has been predicted that both edited residues participate in base-pair interactions in the putative secondary structure. Of particular interest is the site C3 located in intron domain I (23). This position may be crucial for splicing as inferred from mutants of a <span class="yellow">yeast</span> mitochondrial intron (32). We addressed this question by introducing rps10 mutant genes presenting C2, C3 or both positions in the edited form into mitochondria. As shown in Figure 2, transcription was as efficient as for the cognate cox2 construct but neither wild-type rps10 nor the C-U mutants underwent splicing. These results clearly demonstrate that C2 and C3 editing is not sufficient for splicing of rps10 precursor in <span class="yellow">wheat</span> mitochondria and lead us to conclude that splicing failure is probably linked to the lack of trans-recognition elements. One may speculate that these trans-acting factors, for instance nuclear-encoded maturases, were lost after rps10 was transferred to the nucleus in monocots (20,25). The C-to-U mutants were also tested in <span class="yellow">potato</span> mitochondria; no significant differences were detected in splicing efficiency when compared to the unedited construct (D. Choury, unpublished data). It should be noted that the lack of splicing in <span class="yellow">wheat</span> mitochondria is not a general feature of <span class="yellow">potato</span> introns, since the <span class="yellow">potato</span> cox2 intron is removed efficiently (Figure 4B). This is consistent with the hypothesis that <span class="yellow">potato</span> and <span class="yellow">wheat</span> mitochondria have similar trans-acting factors for cox2 intron removal.
A possible link between intron removal and editing has been proposed for splicing of nad1e and nad5III trans-introns where domain six may require editing to be structured in a catalytic competent secondary conformation (8). In <span class="blue">rice</span>, failure of RNA maturation and editing has been correlated with cytoplasmic male sterile phenotype. In this model, the absence of the nuclear gene Rf-1 affects B-atp6 RNA cleavage and editing (33). In organello studies have shown that <span class="yellow">S.bicolor</span> atp6-1 gene was transcribed but not edited when introduced into <span class="yellow">maize</span> mitochondria (18). However, partially edited molecules were detected in <span class="yellow">sorghum</span>-<span class="yellow">maize</span> atp6 chimeric transcripts when they included the <span class="yellow">maize</span> atp6 5′-untranslated sequence, suggesting that the 5′ non-coding region provides a structural motif or binding site for a transcript-specific editing factor (34). Unfortunately, no data on atp6 processing was reported to determine whether this region is directly involved in editing or some other maturation event.
Since <span class="yellow">wheat</span> mitochondria do not have rps10 encoded in the mitochondrial genome (20), it was interesting to test whether <span class="yellow">wheat</span> mitochondria were able to recognize editing sites which had been lost during evolution. In other words, whether the mitochondrial trans-recognition elements are specific for each site or whether they are operating on a subset of editing sites. This question is important since to date the factors responsible for RNA editing in plant mitochondria remain unknown. Solving this issue may provide clues to uncover such factors. As described above, the rps10 transcript was not processed in <span class="yellow">wheat</span> but it was correctly spliced in cognate mitochondria (Figure 5A). Indeed, the mature transcript had identical exon1 and exon2 junction as found in endogenous rps10 mRNA (data not shown). More importantly, all editing sites, C1, C2, C4 and C5 were significantly converted to Us (Figure 5B). The fact that the four editing sites were found either edited or unedited in spliced rps10 mRNA is an indication that editing does not precede splicing as was previously found for cox2 in <span class="yellow">wheat</span> (15).
It may be argued that the absence of splicing precluded editing. This possibility can be discarded since <span class="yellow">potato</span> rps10 precursor mRNA was found to be edited in vivo (23). These data clearly show that splicing is not required for rps10 editing, similar to previous findings for cox2 mRNA and also in cox2 mutant derivatives unable to remove the intron (15). This led us to postulate that the inability of <span class="yellow">wheat</span> mitochondria to recognize rps10 editing sites is likely due to the fact that <span class="yellow">wheat</span> mitochondria have lost the editing trans-recognition elements which become dispensable after transfer of rps10 to the nucleus. To test this possibility, rps10 chimeric plasmids containing editing site C259 from <span class="yellow">wheat</span> cox2 inserted either in exon1 or the intron were constructed. Site C259 is formed by 23 nt corresponding to the −16/+6 sequence embedding the target C. Previously we found that this small region could be recognized by the RNA editing machinery when placed outside of its natural context (16,17). The <span class="yellow">wheat</span> C259 editing site in the chimeric construct was correctly edited by <span class="yellow">potato</span> mitochondria. This result is not unexpected since the corresponding region in endogenous <span class="yellow">potato</span> cox2 mRNA, which presents two differences at positions −3 (C instead of A) and −7 (G instead of A), is edited. Furthermore, these positions were not crucial for editing of C259 (17). Surprisingly, the C259 editing site grafted in rps10 was not recognized by the <span class="yellow">wheat</span> editing machinery. We cannot exclude the possibility that editing efficiency in precursor molecules is very low and so is undetectable when sequencing RT–PCR products representing a pool of transcripts. However, analysis of the same region in cox2 precursors indicates that this was not the case since significant C-to-U conversion was observed (Figure 6D). These observations lead us to postulate that editing is occurring only when the transcript is engaged in post-transcriptional processing, suggesting that rps10 transcripts are not available to editing factors independently of the RNA maturation machinery. The fact that the <span class="yellow">wheat</span> editing site C259 inserted in chimeric rps10 transcripts was not recognized by <span class="yellow">wheat</span> mitochondria is a strong argument for this hypothesis. One might speculate that in plant mitochondria, transcripts have to be engaged in a kind of multiprotein processing complex. A failure to be recognized at some early stage will lead to their accumulation as unmodified precursors.<br><br>
</body></html>