
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Saccharomyces cerevisiae</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="blue">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="blue">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA140510</h3>Enp1, a <span class="yellow">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="blue">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="yellow">yeast</span> <span class="blue">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="yellow">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="yellow">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="yellow">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="yellow">human</span> Enp1, expressed in <span class="yellow">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="yellow">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="yellow">human</span> homolog of <span class="yellow">yeast</span> Enp1 was PCR amplified from a <span class="yellow">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="yellow">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="yellow">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="yellow">donkey</span>-anti-<span class="yellow">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="yellow">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="yellow">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="yellow">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="yellow">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="yellow">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="yellow">yeast</span> Enp1 (19). In contrast, we identified a <span class="yellow">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="yellow">human</span> bystin (19) showed that they were encoded by the same gene on <span class="yellow">human</span> chromosome 6. The reported sequence for <span class="yellow">human</span> bystin is a fragment of the 437 amino acid <span class="yellow">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="yellow">human</span> bystin, as confirmed by available <span class="yellow">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="yellow">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="yellow">Drosophila melanogaster</span> and <span class="yellow">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="yellow">human</span> Enp1 homolog was cloned and expressed in <span class="yellow">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="yellow">human</span> Enp1 homolog did not complement a <span class="yellow">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="yellow">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="yellow">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="yellow">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="yellow">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="yellow">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="yellow">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="yellow">human</span> homolog of Enp1 did not complement a <span class="yellow">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="yellow">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="blue">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA2065882</h3>Identification of Two Independent Risk Factors for Lupus within the MHC in United Kingdom Families
Abstract
The association of the major histocompatibility complex (MHC) with SLE is well established yet the causal variants arising from this region remain to be identified, largely due to inadequate study design and the strong linkage disequilibrium demonstrated by genes across this locus. The majority of studies thus far have identified strong association with classical class II alleles, in particular HLA-DRB1*0301 and HLA-DRB1*1501. Additional associations have been reported with class III alleles; specifically, complement C4 null alleles and a tumor necrosis factor promoter SNP (TNF-308G/A). However, the relative effects of these class II and class III variants have not been determined. We have thus used a family-based approach to map association signals across the MHC class II and class III regions in a cohort of 314 complete United Kingdom Caucasian SLE trios by typing tagging SNPs together with classical typing of the HLA-DRB1 locus. Using TDT and conditional regression analyses, we have demonstrated the presence of two distinct and independent association signals in SLE: HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, OR = 2.3) and the T allele of SNP rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001, OR = 2.0) in intron 6 of the class III region gene SKIV2L. Assessment of genotypic risk demonstrates a likely dominant model of inheritance for HLA-DRB1*0301, while rs419788-T confers susceptibility in an additive manner. Furthermore, by comparing transmitted and untransmitted parental chromosomes, we have delimited our class II signal to a 180 kb region encompassing the alleles HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201 alone. Our class III signal importantly excludes independent association at the TNF promoter polymorphism, TNF-308G/A, in our SLE cohort and provides a potentially novel locus for future genetic and functional studies.
Systemic lupus erythematosus (SLE/lupus) is a complex autoimmune disease in which the body's immune system attacks its own tissues, causing inflammation in a variety of different organs such as the skin, joints, and kidneys. The cause of lupus is not known, but genes play a significant role in the predisposition to disease. The major histocompatibility complex (MHC) on Chromosome 6 contains at least 100 different genes that affect the immune system, including the genes with the strongest effect on lupus susceptibility. Despite the importance of the MHC in SLE, the identity of the actual genes in the MHC region that cause SLE has remained elusive. In the present study, we used the latest set of genetic markers present at the MHC in lupus families to identify the actual genes that affect the disease. To our knowledge, we have shown for the first time that two separate groups of genes are involved in SLE. One group of genes alters how the immune system may inappropriately target its own tissues in the disease. How the second set of genes predisposes to SLE is the subject of ongoing study.<br><br><br><br>Introduction
Since the early 1970s, the <span class="yellow">human</span> major histocompatibility complex (MHC) has been shown to be associated with a number of autoimmune, inflammatory, and infectious diseases, and it continues to be the focus of intense research [1]. The recently defined extended MHC (xMHC) encompasses 7.6 Mb of genome on 6p21.3 and is divided into five subregions from telomere to centromere: extended class I, classical class I, classical class III, classical class II, and extended class II. In addition, the MHC contains two hypervariable regions, the RCCX module in class III (spanning complement C4) and the HLA-DRB genes in class II, that both exhibit copy number polymorphism. Examination of the sequence across the extended MHC reveals the presence of 421 genes, and over 252 (60%) are thought to be expressed [2]. Around 40% of genes expressed within the classical MHC encode proteins with putative immunomodulatory function [3]. The classical class I and class II loci encode the <span class="yellow">human</span> leucocyte antigen (HLA) proteins involved in antigen presentation to T cells, initiating the adaptive immune response. The class III region contains the greatest density of genes in the genome (58 expressed genes), which are often found in functionally related clusters [2].
A major obstacle in the identification of disease-specific causal variants within the MHC has been the strong linkage disequilibrium (LD) exhibited by certain alleles in this region, resulting in the existence of long-range, conserved, extended haplotypes [4], also known as ancestral haplotypes [5], sometimes spanning more than 2 Mb [6]. Thus, for many MHC-associated diseases, it has only been possible to delimit association signals to a particular extended haplotype or segment of one.
Systemic lupus erythematosus (SLE/lupus, [Online Mendelian Inheritance in <span class="yellow">Man</span> 152700, http://www.ncbi.nlm.nih.gov/sites/entrez?db=OMIM&TabCmd=Limits]) is a chronic, multi-system autoimmune disease affecting young <span class="yellow">women</span> ten times more commonly than <span class="yellow">men</span>. The worldwide prevalence of SLE is estimated at between 12 and 124 cases per 100,000 individuals [7]. SLE is characterized by the presence of pathogenic autoantibodies to nuclear and cell-surface antigens that show affinity maturation. The consequent immune complexes deposit in tissues, causing inflammation and damage. It is well established that there is a complex genetic component to lupus aetiology, with hormonal and environmental influences also contributing to disease susceptibility [8,9].
The MHC has been the most consistently confirmed genetic risk factor for SLE, and multiple different genes within the region have been significantly implicated with disease susceptibility. For example, hereditary and acquired deficiencies of the early classical complement component C4, located within the MHC class III locus, leads to a lupus-like syndrome. A role for another class III gene, tumour necrosis factor alpha (TNF), in SLE was suggested following the observation that the lupus-prone New Zealand F1 <span class="yellow">mouse</span> hybrid exhibits constitutively low TNF expression [10]. Recently, the development of antinuclear antibodies in <span class="yellow">patients</span> treated with TNF antagonists has also stimulated interest in the possible role of TNF in SLE [11–13]. <span class="yellow">Murine</span> and <span class="yellow">human</span> candidate gene studies, together with genome-wide linkage screens, provide further support that multiple genetic loci, including the <span class="yellow">mouse</span> MHC complex H2 and the <span class="yellow">human</span> MHC locus, contribute to disease susceptibility [14–17].
It should be noted that the <span class="yellow">human</span> MHC was first associated with SLE in 1971, when studies demonstrated that lupus probands were enriched for the class I alleles HL-A8 (now known as HLA-B8) and HLA-W15 (now known as HLA-B15) when compared with healthy controls [18,19]. Further case control association studies were small, performed in ethnically diverse populations, and tested only a small number of the classical HLA and complement C4 alleles. The most consistent findings reported to date are associations with the class II alleles HLA-DR2 (DRB1*1501) and HLA-DR3 (DRB1*0301) and their respective haplotypes in Caucasian populations. The complement C4A null allele (C4A*Q0) has shown inconsistent association with lupus in a number of studies—a situation that may reflect genetic heterogeneity in disease susceptibility [20–23]. In addition, a recent study has demonstrated that low C4A copy number is a risk factor for lupus in a European American cohort [24]. However, the C4A null allele lies on the lupus-associated DR3 “autoimmune” extended haplotype (AH8.1), which exhibits extremely strong LD [6]. It therefore remains to be definitively established whether this locus constitutes a distinct susceptibility allele to that of the class II association or is merely in LD with it. Similarly, certain TNF promoter polymorphisms, including the much-studied SNP TNF-308G/A, have shown association with SLE; but again, many of these variants are carried on the highly conserved 8.1 ancestral haplotype, thus restricting interpretation of these data.
In 2002, a family-based study employing microsatellites as surrogate markers for HLA-DRB1 haplotypes in Caucasian lupus families demonstrated association with DR3-, DR2-, and DR8 (DRB1*0801)-containing haplotypes. In that study, Graham and colleagues reported that, taking advantage of recombinant chromosomes, the disease risk region could be limited to a 1 Mb region encompassing classical class II and class III [25].
We have performed a medium resolution association mapping study of the MHC in lupus families, utilizing a combination of SNPs and four-digit typing at the HLA-DRB1 locus in order to anchor haplotypes. Sixty-eight SNPs were successfully genotyped across a 2.4 Mb region of the MHC, from the class I locus KIAA1949 to the class II gene HLA-DPB2, in 314 UK Caucasian SLE trios. We used these data to perform a family-based association study in an attempt to distinguish the relative effects of the class II and class III regions of the MHC in lupus susceptibility. In addition, we employed the long-range haplotype test to search for the presence of high-frequency, extended haplotypes indicative of recent positive selection [26]. We have also used family-based and case-control strategies to examine genotypic risk at HLA-DRB1 and rs419788.<br><br>Results
Association Testing of HLA-DRB1 and MHC Region SNPs
In order to define the causal variation within the MHC region, we typed 314 complete SLE trios for the HLA-DRB1 gene as well as for 86 SNPs across a 2.4 Mb region encompassing the HLA class I locus HLA-B to HLA-DPB2. High-quality genotype data was obtained for HLA-DRB1 and 68 MHC SNPs (see Table S1 for quality control data). Association testing of the HLA-DRB1 gene revealed a significant association with HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, T:U = 129:55) in our lupus cohort (Table 1). There was also a trend for under transmission of the HLA-DRB1*0701 allele (nominal p = 0.0013, T:U 42:77); however, this association was no longer significant after correction for multiple testing as determined by 10,000 permutations of the dataset (permuted p = 0.09). Furthermore, we did not find evidence of association with HLA-DRB1*1501 (nominal p = 1.0, T:U 70:70) or HLA-DRB1*0801 (nominal p = 1.0, T:U = 11:11) in our cohort (see Table S2 for complete HLA-DRB1 association data); alleles previously suggested by microsatellite typing of a US lupus cohort [25].
Association testing of the MHC region SNPs also identified significant evidence of association to SLE (Table 1 for associated markers and Table S3 for all MHC SNPs). The SNP with the most significant association, rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001) was of similar strength to that of the HLA-DRB1*0301 allele, with odds ratios (ORs) and 95% confidence intervals (CIs) of 2.0 (1.6–2.6) and 2.3 (1.7–3.2), respectively. This SNP is located within intron 6 of the class III gene, superkiller viralicidic activity 2-like (<span class="blue">Saccharomyces cerevisiae</span>) (SKIV2L), and is located approximately 500 kb telomeric to the HLA-DRB1 gene. Of the other 12 SNPs that were significantly associated with SLE (nominal p = 4.0 × 10−4 to 2.5 × 10−7; permuted p = 0.03 to <0.0001), one was located in the class I region between HLA-B and MICA, seven were located in the class III region, and four were situated in the class II region (Table 1; Figure S1). Specifically, the seven associated class III SNPs were located in or close to the following genes: the TNF promoter, BAT3, SLC44A4, EHMT2, TNXB, GPSM3, and NOTCH4. One of the four class II associated SNPs was close to HLA-DRA, two were between HLA-DRB1 and HLA-DQA1 and one was in intron 1 of HLA-DQA1. The correlation between all 68 SNPs and HLA-DRB1 in our UK SLE cohort is illustrated in Figure 1. The markers showing significant association are highlighted.<br><br>Conditional Analyses Identify Two Independent Association Signals in the MHC
In order to establish whether the two most associated signals identified in this association-mapping experiment are likely to represent a single causal allele or independent risk factors, we first examined the association data conditioned upon the presence of the HLA-DRB1*0301 allele. We found that four of the 13 associated SNPs showed evidence of signals independent of HLA-DRB1*0301 in our dataset, the strongest of which was rs419788 (Table 1). We therefore conditioned the three remaining SNPs (rs2523589, rs1052486, and rs605203) on rs419788 to assess whether these signals are truly independent of each other or show association due to LD with rs419788. In addition, we included HLA-DRB1 in stepwise conditional regression analyses performed on the SNPs showing association independent of HLA-DRB1 (unpublished data). These analyses demonstrated that the observed association signals at rs2523589, rs1052486, and rs605203 were predominantly dependent upon the association at rs419788, and suggested that there are two major independent association signals in the MHC in UK SLE: HLA-DRB1 and rs419788. The independence of the association signals at HLA-DRB1 and rs419788 is further supported by the observation that there is only modest LD between these two (r2 = 0.24). There was no association with any other HLA-DRB1 allele and the four SNPs independent of HLA-DRB1*0301 (TRANSMIT, unpublished data).
The association of the tumour necrosis factor gene promoter SNP TNF-308G/A with SLE is lost after conditioning for HLA-DRB1*0301 in our cohort. If we perform the reverse analysis and condition HLA-DRB1*0301 on the presence of the TNF promoter SNP, we find that the association remains, confirming that our TNF association is secondary to that of HLA-DRB1*0301.<br><br>Genotypic Risk for Class II and Class III Association Signals
Having established independent association at the allelic level with HLA-DRB1*0301 and rs419788-T in our UK SLE cohort, we wanted to further determine the genotypic risk conferred by these variants and hence gain insight into their underlying mode of inheritance in lupus. We used case-control and family-based analyses to assess genotypic risk at HLA-DRB1, while the family-based test alone was used for rs419788. Common family-based tests of LD, such as those used in this study (Genehunter), measure transmission distortion based on allele counts rather than genotype counts; the former has been shown to be more powerful under additive models, while the latter has greater power under recessive or dominant genetic models [27]. The genotype-pedigree disequilibrium test (geno-PDT) determines LD between a locus genotype and disease by comparing genotypes that are transmitted from parent to proband with those that are not [27]. We used the geno-PDT to assess genotypic risk for our class II and class III association signals: HLA-DRB1 and the SNP rs419788. In the case-control analysis for HLA-DRB1, ORs with 95% CI were calculated and Fisher's exact test employed to assess statistically significant differences between HLA-DRB1 genotypes in lupus probands and healthy controls. For HLA-DRB1, the alleles were coded as follows: HLA-DRB1*0301, HLA-DRB1*1501, HLA-DRB1*X where X represents all HLA-DRB1 alleles other than HLA-DRB1*0301, and HLA-DRB1*1501. We included HLA-DRB1*1501 in the analysis, even though we find no allelic association in our cohort, because previous studies have shown a greater risk for lupus in individuals who are compound heterozygotes for HLA-DRB1*0301- and HLA-DRB1*1501-containing haplotypes [25,28]. Overall the results are consistent with a dominant effect from HLA-DRB1*0301 (Table 2) and a dose-dependent (additive) effect from rs419788-T (Table 3). Specifically, both case-control and geno-PDT demonstrate that there is no dose-dependent increase in disease risk for HLA-DRB1*0301. Rather, it appears that the presence of a single copy of HLA-DRB1*0301 alone is sufficient to increase susceptibility to disease. Moreover the 0301/X genotypes constitute the greatest risk in our cohort rather than the 0301/1501 heterozygotes. Genotypes containing HLA-DRB1*1501 in the absence of HLA-DRB1*0301 revealed no significant association in our cohort.
All three rs419788 genotypes demonstrated significant association in our lupus families (Table 3). The common CC genotype was significantly under transmitted, while the rare T allele displayed dose-dependent over transmission to lupus probands.<br><br>Characterization of HLA-DRB1*0301 Risk Haplotype
Next, we wanted to further delimit the MHC class II association signal that we have detected at HLA-DRB1. We used phased parental genotype data to compare the allelic composition of HLA-DRB1*0301-bearing haplotypes that were transmitted (T) to affected probands to those that were not transmitted (or untransmitted, UT) with the aim of identifying differences that could delineate the lupus susceptibility interval(s) arising from this haplotype (summarized in Figures 2A, 2B, and S2). We observed a striking difference between transmitted and untransmitted chromosomes within the class II region: nearly all transmitted HLA-DRB1*0301 haplotypes (99%) are identical across a 180 kb region defined by eight SNPs, whereas the corresponding region within untransmitted HLA-DRB1*0301 haplotypes exhibits significant recombination. These data strongly suggest the existence of a risk haplotype that, interestingly, contains only three expressed genes: HLA-DRB1, HLA-DQA1, and HLA-DQB1. Furthermore, we can confidently define the allelic composition of this risk haplotype, as these three genes are in strong LD and occur in one common haplotype in Caucasians: HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. Thus, we hypothesize that the specific combination of all three alleles is required to confer disease risk in lupus or that disease susceptibility lies with either HLA-DRB1*0301 or the HLA-DQ alleles. We do not have sufficient numbers of recombinant chromosomes in this risk region to further delimit this signal: 2/176 (1.1%) transmitted HLA-DRB1*0301 haplotypes are recombinant at HLA-DQA1-HLA-DQB1; 3/178 (1.7%) transmitted haplotypes identical across HLA-DQA1-HLA-DQB1 do not possess HLA-DRB1*0301.
The composite relative extended haplotype homozygosity (REHH) versus frequency plot for UK SLE; Utah residents with ancestry from northern and western Europe (CEPH); and Yoruba in Ibadan, Nigeria (Yoruba) populations is shown in Figure 3A. We can only comment on evidence for positive selection in CEPH individuals, as we have used this population alone to assess background variation on Chromosome 6. The SLE and Yoruba cohorts are shown for comparative purposes. We find no evidence of positive selection for HLA-DRB1*0301 in the CEPH population. However, this allele is enriched in our lupus cohort (21% of parental chromosomes) and displays greater extended homozygosity when compared with HLA-DRB1*0301-bearing haplotypes in CEPH and Yoruba. Hence, the HLA-DRB1*0301 allele in lupus is observed as an outlier on the plot when compared to background variation in CEPH. These data support our previous observations (outlined above) of the highly conserved nature of HLA-DRB1*0301 haplotypes in lupus. In addition, the haplotype bifurcation plots centered on HLA-DRB1*0301 for UK SLE, CEPH, and Yoruba populations in Figure 3B illustrate preservation of the common HLA-DRB1*0301 haplotype in CEPH and UK SLE, while that seen in the Yoruba is significantly different. The class II regions of all three populations are essentially identical across our chosen SNPs; the main differences lie in class III. The difference in African populations in the class III region is one possible explanation for the lack of evidence for an association between HLA-DRB1*0301 and SLE in African or African American populations. However, HLA-DRB1*0301 has a lower frequency (∼7%–10%) in African populations compared with Europeans (∼13%), and the number of HLA association studies conducted in African populations is very limited.<br><br>Characterization of Class III Region Risk Haplotype
Our data reveal a second independent signal at the MHC in SLE arising from the T allele of SNP rs419788 in intron 6 of the class III gene, SKIV2L. Further evidence supporting the independence of the rs419788-T and HLA-DRB1*0301 alleles is provided by the moderate LD between these two variants (r2 = 0.24) coupled with our data demonstrating that only 47% of rs419788-T allele-bearing haplotypes contain HLA-DRB1*0301.
The structure and composition of T and UT haplotypes anchored at rs419788-T were essentially identical (Figures 2C, 2D, and S2), and hence not informative in delimiting our class III signal. Therefore, we examined the LD structure around our associated class III SNP to better define our disease risk interval. In our lupus dataset the rs419788-T allele resides on three of seven haplotypes present within a large block of six SNPs exhibiting strong LD. This haplotype block encompasses roughly 270 kb containing class III genes from SLC44A4 to AGER, including the RCCX module. Next, we analyzed the haplotype block structure of this region in CEPH families using SNP data dumped from the International HapMap Project (http://www.hapmap.org/). The greater density of SNP typing available in the HapMap CEPH population compared to our current UK SLE map allowed us to potentially refine our signal by exploring correlations between our associated SNP and those surrounding it. Analysis of these data (Figure 4) suggests the presence of short-range LD around our associated variant, rs419788, in CEPH families, encompassing approximately 40 kb of the genome which includes the five genes: complement factor B (CFB), RD RNA binding protein (RDBP), SKIV2L, dom-3 homolg Z (<span class="yellow">C. elegans</span>) (DOM3Z), and serine/threonine kinase 19 (STK19), and does not include the complement C4 locus. Furthermore, assessment of marker association in our lupus dataset demonstrates that after conditioning for HLA-DRB1*0301, the only markers that retain association signals are telomeric of SKIV2L, suggesting that complement C4, which is centromeric to this gene, may not be responsible for our independent class III signal.<br><br>Subphenotype Analysis
In order to gain further insight into disease pathogenesis, we examined common lupus subphenotypes. Such subsets are more homogeneous than lupus per se and thus maybe enriched for specific predisposing variants. In addition, one might expect a close association between MHC class II alleles and autoantibody subsets in lupus if these are indeed causal variants, given their role in antigen presentation and subsequent humoral immunity. We therefore tested our two main MHC association signals, HLA-DRB1*0301 and rs419788, for association with renal disease and autoantibody subsets in our lupus cohort.
We found that HLA-DRB1*0301 was associated with the presence of anti-Ro and anti-La antibodies in our UK SLE cohort, with the latter showing the greatest evidence of association (anti-La nominal p < 0.001 compared with anti-Ro nominal p < 0.025). We found no association of HLA-DRB1*0301 with renal disease or any other autoantibody subsets in our dataset (see Table S4 for detailed results).
Genotypes of the SNP rs419788 were not associated with any of the tested lupus subphenotypes after controlling for the effect of HLA-DRB1*0301 (unpublished data).<br><br>
Discussion
We present the first family-based SNP association study of the MHC in SLE. We have genotyped 69 markers (HLA-DRB1 and 68 SNPs) across 2.4 Mb of the MHC, encompassing class III and class II, in a cohort of 314 UK Caucasian SLE trios. Transmission disequilibrium testing of these data has shown predominant association with the alleles HLA-DRB1*0301 and rs419788-T, together with 12 other MHC SNPs. Moreover, using conditional analyses, we have shown that the two primary signals of association at the MHC are independent of each other. Specifically, one signal arises from HLA-DRB1*0301 in class II and the other from the T allele of SNP rs419788 in the class III gene SKIV2L.
Examination of bifurcation plots for T and UT HLA-DRB1*0301-containing haplotypes has enabled delineation of our class II association signal to a 180 kb region encompassing HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. These data substantially refine that previously published by Graham et al. in 2002 [25], where the lupus susceptibility interval within HLA-DRB1*0301-containing haplotypes could only be delimited to a 1 Mb region encompassing class II and class III. The precise causal variant(s) within this region remains to be determined, as the three implicated alleles exhibit strong LD with few recombination events separating them (two out of 176 transmitted HLA-DRB1*0301 chromosomes in our dataset). However, all three allelic variants represent attractive functional candidates in lupus susceptibility for their role in antigen presentation and stimulation of the adaptive immune response.
Our association of HLA-DRB1*0301 with lupus concurs with published data in Caucasian cohorts and is well established [16]. While our lack of association with HLA-DRB1*1501 and HLA-DRB1*0801 is consistent with previous data from the UK [29], Spain [30], the Netherlands [31], Sweden [32], Mexico [33], and the US [34], it conflicts with that of other US groups [25,35]. Interestingly, we demonstrate a trend, though not statistically significant, for undertransmission of HLA-DRB1*0701—a result also observed in prior UK and Canadian lupus studies [29,36]. Moreover, a negative association of HLA-DRB1*0701 has been reported in other autoimmune diseases including Graves disease [37,38], type 1 diabetes [39], and rheumatoid arthritis [40].
It appears that the conflicting results between UK SLE and previous US (Minnesota [MN]) [25] SLE data stem from differences in HLA-DRB1 allele frequency in the probands of each cohort. The reason for this is unclear. A comparison between UK and MN SLE cohorts (Table 4) reveals that UK SLE cases are enriched for HLA-DRB1*0301 but not HLA-DRB1*0801 or HLA-DRB1*1501 when compared to a UK control population. In contrast, MN SLE cases are enriched for HLA-DRB1*0301-DQB1*0201, DRB1*0801-DQB1*0402, and DRB1*1501-DQB1*0602 inferred haplotypes when compared to MN controls [25]. There is no statistically significant difference in the aforementioned HLA class II alleles/haplotypes between UK and MN control populations that could account for the disparity seen in the respective lupus cohorts. Differences in disease severity and subphenotype frequency between the two populations could account for the observed discrepancy. From the limited data available we found that the presence of renal disease appears to be similar in both cohorts (UK SLE 36% compared with MN SLE 40%), while the gender ratios are significantly different (female: male UK SLE 11:1 compared with MN SLE 57:1, Chi square p value < 0.001). We were unable to compare other lupus subphenotypes. Furthermore, closer inspection of these data reveals that microsatellite-inference of HLA-DRB1 alleles in the MN SLE dataset may underestimate the frequency of HLA-DRB1*0301 and overestimate that of HLA-DRB1*1501, thus diminishing the effect of the former and enhancing that of the latter. It is also possible that the MN SLE cohort shows greater racial heterogeneity in comparison to our UK SLE cohort, despite both being characterized as Caucasian.
Previous studies have demonstrated increased risk for lupus in individuals carrying particular combinations of microsatellite-inferred HLA-DRB1-HLA-DQB1 haplotypes [25,28]. The highest risk genotype was found to be the compound heterozygote HLA-DRB1*0301-DQB1*0201/HLA-DRB1*1501-DQB1*0602, while HLA-DRB1*0301-DQB1*0201-containing genotypes demonstrated a dose-dependent effect in increasing lupus susceptibility [25,28]. In the present study, we have examined genotypic risk at the classically typed HLA-DRB1 locus and in contrast to the aforementioned data of Graham et al. [25,28] we have shown a likely dominant effect of the associated allele, HLA-DRB1*0301. The case-control and family-based analyses for HLA-DRB1 also show the greater power of the former to detect significant association (Table 2). Specifically, all genotypes containing HLA-DRB1*0301 show increased transmission to lupus probands; however, homozygotes show no greater risk compared with heterozygotes, as would be expected under additive or multiplicative models. Thus, a dominant model of inheritance, requiring the presence of a single copy of the disease-predisposing variant alone, likely underlies the susceptibility conferred by HLA-DRB1*0301 in UK SLE. Such a model would fit an antigen presentation hypothesis where susceptible individuals carrying an HLA-DRB1*0301 allele are able to present auto-antigens to CD4+ lymphocytes, thus stimulating an autoimmune response. The differences between our UK SLE and the previously published US SLE data may reflect disease, ethnic, and haplotypic heterogeneity.
Interestingly, analysis of genotypic risk at the associated class III marker, rs419788, suggests an additive (dose-dependent) pattern of inheritance for the rare T allele, where one copy confers a low risk of disease and two copies results in greater susceptibility. The different inheritance patterns for our class II and class III association signals provide further evidence for their independence.
A variety of HLA-DR and HLA-DQ alleles have been associated with autoantibody subsets in ethnically diverse populations of lupus. The strongest associations have been demonstrated between anti-Ro and anti-La antibodies and HLA-DR3 and HLA-DQ2 (HLA-DQB1*0201), which are in strong LD [41–45] in case-control studies. Here, we confirm the association of HLA-DRB1*0301 with anti-Ro and anti-La antibody production in our family-based cohort.
Examination of LD structure around our second independent association, rs419788-T in class III, coupled with the results of our conditional analysis, suggests that this signal could also be delimited to a relatively narrow genomic interval of about 40 kb given further SNP mapping in our cohort. This region includes the genes CFB, RDBP, SKIV2L, DOM3Z, and STK19, but does not include complement C4. Thus, complement C4 null alleles, which have been implicated in lupus pathogenesis, may not be responsible for our class III signal. We conclude, therefore, that our family-based mapping study has potentially revealed a hitherto unknown lupus susceptibility interval in the class III region of the MHC. However, we cannot conclusively exclude association at complement C4/RCCX without direct determination of C4 polymorphism/copy number in our cohort.
With respect to the genes implicated in our study, CFB is a vital component of the alternate complement pathway and disregulation may clearly affect the inflammatory response [46]. RD and Skiv2l are proteins potentially involved in RNA processing. The RD protein forms part of a negative elongation factor (NELF) complex that represses RNA polymerase II transcript elongation, while Skiv2l is a DEAD box protein with possible function as an RNA helicase. The function of Dom3z is currently unknown, although the homologous <span class="yellow">yeast</span> protein binds nuclear exoribonuclease. Moreover, its ubiquitous expression suggests a housekeeping role. STK19 is a protein kinase of unknown function with primary nuclear localization [47]. Interestingly, RDBP and SKIV2L are found to be highly expressed in T lymphocytes, B lymphocytes, and dendritic cells (SymAtlas, http://symatlas.gnf.org/SymAtlas/).
A number of studies have demonstrated conflicting evidence for and against association with various TNF locus polymorphisms in SLE [48]. A recent meta-analysis of the TNF-308G/A promoter polymorphism in SLE [48] revealed evidence of association for the minor allele (A) in European populations; however, this study did not account for LD with class II alleles. On conditioning our dataset for HLA-DRB1*0301, we find that the TNF promoter signal is lost, suggesting that this association is not independent and is due to LD with HLA-DRB1*0301 (or another variant in LD with HLA-DRB1*0301).
In summary, we have found association with two distinct and independent variants within the class II (HLA-DRB1*0301) and class III (SKIV2L) regions of the MHC in UK SLE trios. We can delimit our class II signal in lupus to three genetic variants (HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201) that may confer disease risk in combination or as separate signals. Our class III signal importantly excludes independent association at the TNF promoter polymorphism TNF-308G/A and potentially provides a novel locus for further study.<br><br>Materials and Methods
Study cohorts.
SLE families. The cohort comprises 314 complete SLE trios (that is, mother, father, and affected lupus proband) collected as previously described [49]. All study <span class="yellow">participants</span> are European Caucasian on the basis of grandparental origin. All 314 lupus probands (288 female, 26 male) fulfill the revised American College of Rheumatology (ACR) criteria for SLE [50], 36% of whom have a diagnosis of lupus nephritis. Written consent was obtained from all study <span class="yellow">participants</span> and ethical approval for this study was obtained from the Multi-Centre Research Ethics Committee (MREC 2 June 1998).<br><br>Healthy controls.
The control population for the HLA-DRB1 genotypic risk case-control analysis constitutes 1,667 healthy males of Northern European origin. The individuals are potential hematopoietic stem cell donors and were typed to four digits for HLA-DRB1 at the Anthony Nolan Trust, UK for this purpose. The level of resolution used for the typing of HLA-DRB1*15 alleles in these healthy controls resulted in the ambiguous allele string HLA-DRB1*1501/1502/1504/1506. However, it is likely that the great majority are HLA-DRB1*1501. There is no gender bias in HLA-DRB1 allele frequencies, so although we have used a male control cohort, the frequencies would be expected to be the same in a similar female cohort (Steven Marsh, personal communication).<br><br>SNP genotyping.
Eighty-six SNPs were chosen for genotyping in our mapping study. Specifically, we selected 40 MHC class II and class III haplotype tagging SNPs from a preliminary MHC SNP map [51] that had previously shown robust genotyping efficacy. In addition, we typed 36 MHC class II tag SNPs from a subsequent high-resolution MHC study [52]. We also included the TNF-308G/A promoter SNP, together with nine further SNPs in the region of HLA-B and MICA obtained from the database, dbSNP (http://www.ncbi.nlm.nih.gov/projects/SNP/). The latter SNPs had not been well characterized. All variants were typed in the entire cohort (n = 942). The SNPs span approximately 2.4 Mb of the MHC from the class I gene, KIAA1949 to the class II pseudogene, HLA-DPB2 and thus encompass MHC class III and class II. SNP genotyping was performed at the Broad Institute of MIT and Harvard and at Imperial College London by matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry using the Sequenom MassARRAY platform as previously described [53]. SNPs that failed Sequenom typing were retyped by KBiosciences (http://www.kbioscience.co.uk/) using their in-house SNP genotyping methodology, KASPar (http://www.kbioscience.co.uk/genotyping/index.htm), a competitive allele-specific PCR technique.<br><br>HLA-DRB1 genotyping.
HLA-DRB1 typing was performed at the Anthony Nolan Trust, UK. All samples (n = 942 UK SLE trios and n = 1,667 controls) were genotyped using LABType SSO (sequence-specific oligonucleotide) typing technology according to the manufacturer's written recommendations (http://www.onelambda.com). Briefly, a locus-specific biotinylated PCR amplicon is produced, denatured, and rehybridized to complementary oligonucleotide probes conjugated to fluorescently coded beads. The bound biotinylated PCR product can be detected using R-phycoerythrin-conjugated streptavidin. A flow analyzer identifies the fluorescent intensity of phycoerythrin on each bead. The assignment of HLA type is based on the reaction pattern compared to patterns associated with known sequences.
High resolution testing was performed using the Dynal AllSet+ SSP (sequence-specific primers) DRB1 assay according to the manufacturer's protocol (Invitrogen) for SLE families only. The presence or absence of PCR amplification was detected in a gel electrophoresis step using visualization by ethidium bromide incorporation. Genotypes were determined using SSPTool software.
Samples that could not be resolved to four digits using SSO and PCR-SSP were analyzed by DNA sequencing of exon 2 of HLA-DRB1. Primers, reagents, and protocols were provided by The Anthony Nolan Trust, UK (primer sequences are available on request). Specific HLA-DRB1 alleles were assigned by comparing the resultant sequence with reference sequence from the IMGT/HLA Database [54].<br><br>Data analysis.
Mendelian inconsistencies were removed using PedCheck [55]. Families in which more than eight markers demonstrated Mendel errors were removed from further analysis. Markers with less than 80% genotyping efficiency and markers where more than eight families showed Mendel errors were also excluded from analysis. Two markers located within the SLE associated class II region (rs2239802 in intron 4 of HLA-DRA and rs6457594 in the region between HLA-DRB9 and HLA-DRB5) show deviation from Hardy–Weinberg equilibrium (HWE), which may reflect an undetected SLE association or systematic genotyping error. HWE was assessed in parental samples in our cohort. There is currently no uniform opinion in the community regarding the inclusion or exclusion of SNPs that show deviation from HWE, hence we elected to include these markers in the final analysis.
Sixty-eight out of the total 86 SNPs passed our quality-control measures (see Table S1 for details). In summary, one SNP was monomorphic in our dataset, four SNPs yielded low genotyping efficiency, and 13 SNPs were excluded for unacceptable Mendel error rate. The mean call rate for all markers post-quality control was 94% (range 83% to 99%).
Family-based association testing was performed using Genehunter TDT (version 2.1) [56] and TRANSMIT (version 2.5) [57]. Haplotypes were constructed and permutation testing performed using Haploview (version 3.32) [58]. Significance of association signals in all analyses was based on permutation testing (10,000 permutations). The data are represented both as nominal and permuted p values. ORs with 95% CI for family-based analyses were calculated in PLINK (http://pngu.mgh.harvard.edu/purcell/plink/) [59]. Conditional regression analyses were undertaken using WHAP [60].
The geno-PDT was performed using PDT version 5.1 with default settings [27]. The HLA-DRB1 alleles were coded into three groups for the geno-PDT and the case-control analysis: HLA-DRB1*0301, HLA-DRB1*1501 and HLA-DRX where HLA-DRX includes all HLA-DRB1 alleles other than HLA-DRB1*0301 or HLA-DRB1*1501. The HLA-DRB1*1501 code in the healthy controls represents the allele string HLA-DRB1*1501/1502/1504/1506, as described previously. The HLA-DRB1*1501 code in the lupus probands represents the alleles HLA-DRB1*1501 (84 out of 91 *1501 and *1502 alleles) and HLA-DRB1*1502 (7/91 *1501 and *1502 alleles), as *1504 and *1506 were not present in this population. Fisher's exact test was used to assess significance of association in the case-control analysis.<br><br>Subphenotype analysis.
We looked for association of the HLA-DRB1*0301 allele with autoantibody subsets and renal disease in our cohort using the Chi-square test. We compared cases with and without the subphenotype of interest with DRB1*0301 homozygosity, heterozygosity, combined homozygosity and heterozygosity, and non-DRB1*0301 status. We performed the same analyses for homozygous and heterozygous genotypes of the associated SNP rs419788. The autoantibody subsets compared were anti-C1q, IgG, and IgM anti-cardiolipin antibodies (ACLG and ACLM), anti-Ro, anti-La, anti-RNP, anti-Sm, and anti-dsDNA.<br><br>Delineation of associated MHC haplotypes and evidence for positive selection.
We looked for positively selected alleles in our dataset using the long-range haplotype test as measured by extended haplotype homozygosity (EHH), previously described by Sabeti et al. [26]. Essentially, such an analysis allows assessment of positive selection by mining datasets for high frequency extended haplotypes in comparison to the other core haplotypes at a locus.
EHH is defined as the probability that two randomly chosen chromosomes carrying the core haplotype of interest will be identical by descent (homozygosity at all SNPs) for the entire interval from the core to a distance x. The REHH is the ratio of the EHH on the tested core haplotype compared with the combined EHH of all the other core haplotypes at the region excluding the tested core; as such, REHH accounts for local variation in recombination rate while EHH does not [26].
The program emphase was employed to assign the phase of parental genotype data and reconstruct missing information. Emphase is a simple phaser similar to the phaser of Excoffier and Slatkin [61]. It is very fast, especially on large datasets, and sufficiently accurate for most genetic applications. EHH analysis was performed on the phased parental data using the software program SWEEP (http://www.broad.mit.edu/mpg/sweep/index.html).<br><br>Haplotype bifurcation plots.
We represent the breakdown of LD on core haplotypes using haplotype bifurcation diagrams generated in the program TREE [62] (also explained in [52]).<br><br>REHH versus frequency plots.
Fifty-three SNPs (identified in Figure 1) are common to our dataset and the CEPH and Yoruba HapMap [63] populations. These three datasets, together with CEPH SNP data for Chromosome 6 in its entirety, were used to generate separate REHH versus frequency plots in SWEEP. The plots from the four cohorts were combined for visual, not statistical, comparison. Evidence for positive selection was quantitatively assessed in CEPH individuals, as this population alone was used to assess background variation on Chromosome 6. The UK SLE and Yoruba cohort data are shown for comparison. The 95th percentile based on total CEPH Chromosome 6 SNP data is shown.<br><br>
Supporting Information
Accession Numbers
The Online Mendelian Inheritance in <span class="yellow">Man</span> (OMIM, http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim) accession numbers for the genes described in this study are as follows: AGER, 600214; BAT3, 142590; C4A, 120810; C4B, 120820; CFB, 138470; DOM3Z, 605996; DOM3Z, 605996; EHMT2, 604599; HLA-B, 142830; HLA-DPB2, 120290; HLA-DQA1, 146880; HLA-DQB1, 604305; HLA-DRA, 142860; HLA-DRB1, 142857; KIAA1949, 610990; MICA, 600169; NOTCH4, 164951; RDBP, 154040; RDBP, 154040; SKIV2L, 600478; SLC44A4, 606107; STK19, 604977; STK19, 604977; TNF, 191160; and TNXB, 600985.<br><br><br><br><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="yellow">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="yellow">yeast</span> <span class="blue">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="yellow">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="yellow">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="yellow">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="yellow">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="yellow">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="yellow">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="yellow">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="yellow">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="yellow">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="yellow">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="yellow">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="yellow">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="yellow">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="yellow">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="yellow">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="yellow">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="yellow">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="yellow">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="yellow">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="yellow">yeast</span> <span class="yellow">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
<h3>pmcA150219</h3>Rap1p and other transcriptional regulators can function in defining distinct domains of gene expression
Abstract
Barrier elements that are able to block the propagation of transcriptional silencing in <span class="yellow">yeast</span> are functionally similar to chromatin boundary/insulator elements in metazoans that delimit functional chromosomal domains. We show that the upstream activating sequences of many highly expressed ribosome protein genes and glycolytic genes exhibit barrier activity. Analyses of these barriers indicate that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p may participate in barrier function. We also present evidence suggesting that Rap1p is directly involved in barrier activity, and its barrier function correlates with local changes in chromatin structure. We further demonstrate that tethering the transcriptional activation domain of Rap1p to DNA is sufficient to recapitulate barrier activity. Moreover, targeting the activation domain of Adr1p or Gcn4p also establishes a barrier to silencing. These results support the notion that transcriptional regulators could also participate in delimiting functional domains in the genome.<br><br>INTRODUCTION
The eukaryotic genome is organized into discrete functional domains in which gene expression is either permitted or repressed. Domains permissive for gene expression are usually defined as euchromatin or active chromatin, and domains that repress gene expression heterochromatin or silent chromatin. The HML and HMR loci in <span class="blue">Saccharomyces cerevisiae</span> are silent chromatin domains (1). Compared to active chromatin, silent chromatin is more compact and its histone components are hypoacetylated (2). The SIR complex consisting of Sir2p through Sir4p is an integral part of <span class="yellow">yeast</span> silent chromatin. Sir3p and Sir4p interact with the N-terminal tails of histones H3 and H4 providing the structural basis of silent chromatin (1). Sir2p was recently found to be an NAD-dependent histone deacetylase and was suggested to be responsible for histone hypoacetylation in silent chromatin (3). Silent chromatin is initiated at small cis-acting DNA elements known as the E and I silencers flanking each of the HM loci (4).
Increasing evidence indicates that histone deacetylation is key to the propagation of silent chromatin (4). In vertebrates and <span class="yellow">fission yeast</span><span class="yellow">yeast</span>, spread of silent chromatin involves a chain of events of histone H3 deacetylation → H3 methylation → binding of methylated H3 by HP1 (heterochromatin protein 1) or swi6 (4). In <span class="yellow">S.cerevisiae</span>, there is evidence that Sir3p (hence the SIR complex) has much higher affinity to unacetylated histone H4 than acetylated H4 (5). Based on this and the fact that Sir2p is a histone deacetylase, a refined model for the propagation of silent chromatin can be proposed. In this model, Sir2p recruited to the silencer deacetylates histones in an adjacent nucleosome enabling it to bind another SIR complex with high affinity. The nucleosome-bound SIR complex in turn deacetylates the neighboring nucleosome allowing it to recruit a new SIR complex. In this manner, the SIR complex promotes its own propagation along an array of nucleosomes (4).
The fact that transcriptionally silent chromatin can encroach upon active chromatin poses the question of how interspersed domains of silent and active chromatin are demarcated. Studies of the <span class="yellow">Drosophila</span> and vertebrate genomes demonstrated that some domains are delimited by boundary or insulator elements (6). These elements are specialized DNA sequences that act as barriers to enhancers and/or silent chromatin. One of the best characterized insulators is the <span class="yellow">chicken</span> HS4 insulator at the β-globin locus. Histones surrounding this insulator are hyperacetylated indicating the presence of histone acetyltransferase (HAT) activity at the insulator (6). Recently, sequences that could block the spread of silent chromatin have been discovered in <span class="yellow">S.cerevisiae</span> (7,8). These sequences (referred to as silent chromatin barriers) include sub-telomeric anti-silencing regions (STARs), the right boundary of HMR, and the upstream activating sequence (UAS) of the highly expressed TEF2 gene (9–11). The mechanisms underlying barrier functions in <span class="yellow">yeast</span> are not known.
TEF2-UAS contains three tandem repressor activator protein 1 (Rap1p)-binding sites that are necessary and sufficient for its barrier function (11). Rap1p is an abundant sequence-specific DNA-binding protein (12). Variants of the 13 bp consensus sequence for Rap1p binding (ACACC CRYACAYM) (13) lie not only in the UASs of numerous genes but also in the silencers of the HM loci and in telomeric C1–3A repeats. Accordingly, Rap1p functions as a global regulator of transcriptional activation and repression, silencing, as well as telomere length (12). Rap1p performs these diverse functions by interacting with different factors in respective contexts. As an activator, Rap1p binds to the promoters of 362 ORFs (13) and can function via at least three mechanisms. First, Rap1p binding can ‘open up’ local chromatin structure at a promoter to help another activator to bind (14). Secondly, Rap1p bound to DNA can help Gcr1p bind to an adjacent site in the promoter through physical interaction (15). Thirdly, Rap1p is involved in the recruitment of the NuA4 HAT complex and TFIID to ribosome protein genes (RPGs) (16,17). As a silencing factor, Rap1p binds to silencers or telomeric repeats and recruits Sir3p/Sir4p through direct interactions (18). Rap1p also interacts with telomere-specific proteins in executing its role in regulating telomere length (19). It is puzzling that a protein that helps establish silent chromatin could also act as a barrier to its propagation.
In this study, we found that in addition to TEF2-UAS, UASs of many <span class="yellow">yeast</span> genes exhibit barrier activity. Analyses of these UASs indicate that binding sites for transcriptional regulators Abf1p, Reb1p, Adr1p, Gcn4p, as well as Rap1p, may participate in barrier function. These UASs are dispersed across the genome and might play a role in defining functional chromosomal domains. Moreover, we demonstrated that targeting the activation domain of Rap1p, Adr1p or Gcn4p alone, led to barrier function. Since the activation domain of an activator was usually involved in interacting with co-activators and/or components of the transcriptional machinery, these data indicated that other factors might also be involved in barrier function.<br><br>MATERIALS AND METHODS
Plasmids and strains
Plasmids 1–53 were used to make strains 1–53, respectively (see Figs 1–6). Plasmid 1 was previously named pMB22-a that contained a HindIII–BamHI fragment of chromosome III (coordinates 14838–16263) with a URA3 gene inserted at its EcoRV site (20). Plasmid 2 was derived from 1 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SnaBI site. Plasmids 3–38 were identical to 2 except that different fragments from various promoters (see Figs 1 and 2) were inserted at the SnaBI site. Plasmid 29 had a sequence of chromosome III (295327–295713) containing the HMR-tRNA gene (21) inserted at the SnaBI site of plasmid 1. Plasmids 39 and 40 were previously named pYXB26 and pYXB59, respectively (11). Plasmid 41 was identical to pYXB59 except bearing mutations illustrated in Figure 3A. Plasmids 42–48 were previously described as pYXB29, 48, 28, 27, 59, 31 and 37, respectively (11). Plasmid 50 was derived from plasmid 1 by inserting two copies of a sequence bearing the consensus binding sequence of LexA (bold) (22), GGGGTACGTACTGTATGTACATACAGGATATCGGGG, at the SnaBI site. Plasmid 51 was identical to 50 except having only one copy of the LexA-binding sequence. Plasmid 52 was derived from pYXB26 (11) by inserting a SpeI-ADE2-AvrII sequence at the SpeI site. Plasmid 53 was derived from plasmid 52 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SpeI site.
Plasmid L1 (see Fig. 5A) was made by inserting LEU2 into pBTM116 (23) carrying the LexA gene flanked by the promoter and terminator of ADH1. L2 through L8 were derived from L1 by fusing various sequences to LexA (see Fig. 5A and B). Plasmid pRS425 (24) carried the 2 µm origin and LEU2.
Strain YXB76 was MATa ura3-52 leu2-3,112 ade2-1 lys1-1 his5-2 can1-100, E-HMLα-Iinverted (20). Y2047b was MATa HMRa HMLα EΔ79–113::SUP4-o IΔ242 LEU2-GAL10-FLP1 ura3-52 ade2-1 lys1-1 his5-1 can1-100 [cir0] (25). Strain 1 was made by transforming YXB76 to Ura+ with HindIII + BamHI digested plasmid 1. Strains 2–38, 50 and 51 were similarly made with corresponding plasmids, respectively (e.g. strain 2 was made with plasmid 2). Strains 39, 40 and 42–48 were previously described as YXB26, 48, 29, 48, 28, 27, 59, 31 and 37, respectively (11). Strains 41, 52 and 53 (see Figs 3C and 6) were made by transforming Y2047b to canavanine-resistant by BamHI-digested plasmids 41, 52 and 53, respectively. Strains 40′–48′ were sir3::URA3 derivatives of strains 40–48, respectively. The relevant genotypes of all the strains were confirmed by Southern blotting.<br><br>Quantitative mating assay
Quantitative mating was performed as described (11).<br><br>Electrophoresis mobility shift assay (EMSA)
Rap1p was expressed in <span class="yellow">Escherichia coli BL21 (DE3)</span> from pL3S5, a pET vector carrying the RAP1 gene downstream of a T7 promoter (26). Crude protein extracts were prepared from 40 ml cultures harvested after 3 h of IPTG induction and resuspended in 1 ml of lysis buffer (50 mM NaH2PO4, 300 mM NaCl, pH 8.0). A control extract was prepared from a culture without IPTG induction. Western blotting revealed that Rap1p protein was present in the induced extract but not the uninduced one (data not shown). DNA fragments of 26 bp in length (see Fig. 3A) were end-labeled with 32P-phosphate. Fifteen microliter binding reactions were prepared, each consisting of 20 000 c.p.m. radiolabeled DNA probe, 5 µg of crude protein extract, non-specific competitor DNAs (0.3 µg of <span class="yellow">yeast</span> tRNA, 0.3 µg of <span class="yellow">salmon</span> sperm DNA and 0.5 µg of <span class="yellow">E.coli</span> DNA), 10 µg of BSA, 10 mM Tris–HCl (pH 8.0), 10 mM MgCl2 and 8% glycerol. The reactions were incubated at 25°C for 20 min and loaded onto non-denaturing polyacrylamide gels (4–5%). Gels were run at 40 mA for 1 h in 0.5× TBE buffer.<br><br>Analysis of the supercoiling of DNA circles
Cells were grown in YPR (<span class="yellow">yeast</span> extract + bacto peptone + 2% raffinose) to early log phase. Galactose (2%) was then added to the culture to induce the expression of FLP1. After 2.5 h of incubation, nucleic acid was isolated using the glass bead method and fractionated on agarose gels with 30 µg/ml chloroquine. DNA circles were detected by Southern blotting.<br><br>
RESULTS
UASs of many highly expressed genes can function as barriers to the spread of silencing
We have previously shown that TEF2-UAS was able to block the spread of silencing in a silencer-blocking assay (11). Such an assay tested if a sequence had the ability to prevent a silencer from silencing a reporter gene when it was positioned between the silencer and the reporter. In this report we used a new silencer-blocking assay to identify more barrier elements. This assay employed the URA3 gene and an inverted HML-I that could silence sequences to the right of HML (20) (Fig. 1, strain 1). URA3 expression makes cells sensitive to 5-fluoroorotic acid (FOA) thus URA3 silencing could be measured by cell growth on FOA medium (Fig. 1, strain 1). As expected, TEF2-UAS and another known barrier element, HMR-tRNA gene (21) exhibited barrier activity in this assay as indicated by the lack of silencing of URA3 in strains 2 and 29 (Fig. 1).
TEF2 encoding translation elongation factor 1α is one of the most highly expressed genes in <span class="yellow">yeast</span> (27). Other highly expressed genes include ribosome protein genes (RPGs) and genes coding for glycolytic enzymes (27). The coordinated regulation of most of these genes requires the so-called general regulatory factors Rap1p, Abf1p and Reb1p (28,29). Using the above silencer-blocking assay, we tested if the UASs of other highly expressed genes could also block URA3 silencing (Fig. 1). Seventeen new barrier elements were identified from a total of 26 UASs tested. These 17 elements were UASs of eight RPGs (see Fig. 1, strains 3–10), eight glycolytic genes (strains 18–25) and the HIS3 gene (strain 26). All 17 elements except ADH2-UAS and HIS3-UAS exhibited very strong barrier activity as evidenced by the absence of URA3-silencing in strains 3–10 and 18–24 (Fig. 1).
Seven of the eight RPG-UASs that had barrier activity, contained a single or a pair of tandem Rap1p-binding sites (Fig. 1, strains 3–9). Three of them also contained an Abf1p or Reb1p site (strains 6, 8 and 9). One barrier RPG-UAS (strain 10) had only a predicted Abf1p site. Of the seven RPG-UASs that had no barrier activity (Fig. 1, 11–17), only two bore no site for Rap1p, Abf1p or Reb1p (11 and 12). The remaining five each contained one to three Rap1p sites (strains 13–17). RPL15A-UAS (17) also contained an Abf1p site. For most of the 15 RPG-UASs tested (3–17), the presence or absence of predicted Rap1p sites correlated with the presence or absence of Rap1p association in vivo as previously examined by chromatin immunoprecipitation assays (13). Exceptions were the UASs of RPS24B, RPS28A and RPL15A (13).
Of the eight UASs of glycolytic genes tested (Fig. 1, strains 18–25), seven exhibited strong barrier activity (18–24) and one had weaker but detectable barrier activity (strain 25). Glycolytic genes are among the most highly expressed genes and their high expression depends on the functions of Rap1p, Abf1p or Reb1p as well as Gcr1p (29,30). The seven strong glycolytic barriers all consist of one or two Rap1p sites plus one or two Reb1p or Abf1p sites (Fig. 1, strains 18–24). In addition, one or more Gcr1p-binding sites are found adjacent to each Rap1p site. It was proposed that the function of Rap1p was to facilitate the binding of Gcr1p to the UAS (15). Abf1p or Reb1p was shown to play a role similar to that of Rap1p in activating glycolytic genes (31–33).
Although the above data reinforced the notion that certain Rap1p-binding UASs could serve as barriers to silencing, they raised new questions about the essential components of a barrier element. For example, some of the UASs that contained only a single Rap1p site had barrier activity (e.g. RPL19B-UAS, strain 3) whereas others that contained two or three sites had no activity (e.g. RPL39, strain 16). One explanation was that sequences flanking the Rap1p site(s) in a particular UAS could positively or negatively influence barrier function. Moreover, the presence of Abf1p or Reb1p sites in 11 of the 17 new barriers made us wonder if they also participated in barrier function. We began to address these issues by analyzing three new barrier elements (Fig. 2). RPS10A-UAS containing an Abf1p-binding site and a pair of Rap1p sites was divided into three fragments that were tested in a silencer-blocking assay (Fig. 2, strains 30–32). It was clear that only the fragment containing Rap1p sites functioned as a barrier (strain 31). TPI1-UAS bearing a Reb1p site and a Rap1p site was divided into two parts with one containing the Rap1p site (Fig. 2, strain 34) and the other Reb1p site (strain 33). Neither of these fragments alone retained barrier activity indicating that concerted action of Reb1p and Rap1p was required for barrier activity. Duplicating the Reb1p-bearing part of TPI1-UAS didn’t restore barrier activity (strain 35). However, triplicating the Rap1p-containing part did re-create barrier function (strain 36). RPS28A-UAS bore, in addition to a Rap1p site, an Abf1p site and an adjacent T-rich region that are required for efficient transcription of RPS28A (34). A sequence of RPS28A-UAS deleted for a fragment bearing the Rap1p site lost the barrier activity (Fig. 2, strain 37). However, duplicating this sequence restored barrier activity (strain 38). The above results indicated again that Rap1p sites could form barriers (strains 31 and 36). On the other hand, they also suggested that synergistic actions of Abf1p sites (strain 38), Reb1p site + Rap1p site (strain 18), or Abf1p site + Rap1p site (strain 16) could all lead to barrier function. This was not very surprising since Rap1p, Abf1p and Reb1p could perform similar and sometimes interchangeable functions in gene regulation (16,35,36). Note, however, we haven’t ruled out the possibility that other factors (e.g. Gcr1p) or structure features of DNA (e.g. T-tracks) were also involved in barrier function.
It was interesting that ADH2-UAS and HIS3-UAS bearing no site for Rap1p, Abf1p or Reb1p exhibited detectable barrier activity (Fig. 1, strains 25 and 26). However, binding sites for other transcriptional activators existed in these UASs. The major regulator that binds to ADH2-UAS is Adr1p (37). On the other hand, Gcn4p binds to multiple sites in HIS3-UAS and activates HIS3 transcription (38). The possible involvement of Adr1p or Gcn4p in barrier function was examined below. Note Adr1p function was subject to glucose repression (37), which might explain why ADH2-UAS only exhibited limited barrier activity since glucose was used in the media used in our assay.<br><br>Effect of mutating Rap1p-binding sites in TEF2-UAS on barrier function
Although we had shown that the three Rap1p-binding sites in TEF2-UAS were necessary and sufficient for its barrier activity (11), we had not shown if binding of Rap1p to these sites was required. To address this we performed mutational analysis of TEF2-UAS. As shown in Figure 3A and B, the three Rap1p-binding sites in TEF2-UAS (designated as R1, R2 and R3) were all able to bind Rap1p in an EMSA (Fig. 3B). Single or double C→A mutations were introduced into R1, R2 and R3 resulting in m1, m2 and m3, respectively (Fig. 3A). The m2 and m3 sites had both lost the ability to bind Rap1p, whereas m1 still bound Rap1p (Fig. 3B), which could be predicted from the specific mutations in each site (Fig. 3A, compare m1, m2 and m3 to the consensus sequence). We then tested a mutated TEF2-UAS containing m1, m2 and m3 in a silencer-blocking assay using the HML-E silencer and the HMLα genes (Fig. 3C). In this assay, the HML-I silencer was deleted but the HML-E silencer was sufficient to silence the HMLα genes (Fig. 3C, strain 39) (11). Silencing in strain 39 (a-type) was measured by its ability to mate with an α-type strain (Fig. 3C). As predicted, the wild-type TEF2-UAS blocked HMLα silencing (11) (Fig. 3C, strain 40). On the other hand, the mutated TEF2-UAS containing m1, m2 and m3 had lost barrier activity (Fig. 3C, compare the mating efficiencies of strains 41 and 40). Therefore, mutations that prevented Rap1p from binding two of the three sites in TEF2-UAS abolished its barrier activity, indicating that association of Rap1p with TEF2-UAS at more than one site was required for barrier function.<br><br>The barrier TEF2-UAS alters local chromatin structure
In eukaryotic cells, the topology of local DNA reflects the chromatin structure in which it resides. We had previously developed a DNA topology-based assay to examine chromatin structure in vivo, which involved excising a chromosomal region of interest from its genomic location as a circle using site-specific recombination (39). Using this strategy, we demonstrated that DNA in silenced chromatin was more negatively supercoiled than that in active chromatin (39). In this report we used the topology assay to address if barrier elements altered local chromatin structure as <span class="yellow">Drosophila</span> or <span class="yellow">chicken</span> insulators did (40,41).
Various sequences (designated X) from UASs of TEF2, AgTEF (TEF gene from <span class="yellow">Ashbya gossypii</span>) and ADE2 were inserted between HML-E and HMLα at HMLΔI (HML locus deleted for the I silencer) flanked by the FRT sites for the Flp1p site-specific recombinase (Fig. 4A and B). The FLP1 gene was under the control of the inducible GAL10 promoter (39). The mating efficiency of each strain measured silencing of HMLα (Fig. 4C). In strains 40, 42, 44 and 47, silencing was restricted to the left of the X insertions and HMLα was derepressed (Fig. 4C). Consistently, the proportion of silent chromatin in the region flanked by the FRT sites in each of these strains was decreased, as indicated by the reduced negative supercoiling of the excised circle (11) (data not shown). Note that the above effect of a barrier on HML chromatin reflected both change in the scope of silencing caused by the barrier activity and alteration (if any) in chromatin structure at the barrier independent of the silencing state of the locus. To exclusively examine the latter, we analyzed the topology of HML DNA in a silencing-deficient background. The SIR3 gene in strains described in Figure 4B was disrupted rendering them defective in silencing (strains 40′–48′ in Fig. 4D). Recombination mediated by Flp1p in these strains led to the excision of a group of chromosomal circles that differed only in the X insertion. The supercoiling of these circles was analyzed as a function of the length of the insertion (Fig. 4D). If an insertion simply lengthened a circle without causing abnormal changes in nucleosomal structure and/or density, the topoisomers of this circle would migrate slower relative to those of a similar circle with a smaller size. Otherwise, migration of the topoisomers would deviate from that predicted based on the size of the circle. As shown in Figure 4D, the circle from strain 40′ bearing a 104 bp barrier element migrated faster rather than slower than the circle from strain 46′ bearing a 91 bp non-barrier element. Circles bearing other barrier elements also exhibited unexpected faster migration (compare strains 44′ to 45′, 47′ to 43′, 42′ to 48′, respectively). Therefore, these data indicated that barrier function correlated with altered topology of local DNA. This was further demonstrated by that mutations in TEF2-UAS that abolished its barrier activity also abolished its effect on DNA topology (Fig. 4D, compare 41′ to 40′; note that circles from both strains had the same size). Since in our gel assay more negatively supercoiled DNA migrate slower, we concluded that every fragment with barrier activity caused a reduction in negative supercoiling of approximately one to two turns in HMLΔI DNA (Fig. 4D, compare the centers of distributions of topoisomers in pairs of lanes, e.g. 40′ and 41′).
The topology of eukaryotic DNA reflects mainly the wrapping of DNA into nucleosomes (approximately one negative superhelical turn is associated with each nucleosome) (42). Therefore, removal of one nucleosome would eliminate approximately one turn. On the other hand, histone acetylation can reduce negative supercoiling associated with a nucleosome by approximately 0.2 turns (43). Therefore, acetylating five nucleosomes would reduce negative supercoiling by approximately one turn. Consequently, the linking number change of one to two brought about by TEF2-UAS can be accounted for by either the loss of one to two nucleosomes or acetylation of five to 10 nucleosomes. Further experiments are under way to clarify the nature of change in chromatin induced by a barrier.<br><br>Targeting the transcription activation domain of Rap1p recapitulates its barrier activity
The multifunctional Rap1p can be divided into at least four functional domains (Fig. 5A) (12). We were interested in defining which of these domains might be involved in the barrier activity of Rap1p. To this end, we fused various parts of Rap1p to the bacterial LexA protein and tested if targeting any of the fusion proteins to LexA-binding sites could re-create a barrier to silencing (Fig. 5A). The LexA-RAP1 fusion genes were carried on 2 µm based plasmids marked by LEU2 (designated L1–L6, Fig. 5A). When introduced into strain 50 in which two LexA-binding sites had been inserted between the inverted HML-I and URA3, LexA alone had no effect on URA3 silencing (Fig. 5A, plasmid L1). Targeting the DNA binding domain, DNA bending domain, silencing domain, or the activating + silencing domain of Rap1p also did not affect URA3 silencing (L2–L5). On the other hand, targeting the activation domain alone recapitulated the barrier activity of Rap1p (L6). These results indicated that the activation domain of Rap1p was involved in barrier activity. The inability of the Rap1p-silencing domain to block URA3 silencing in our assay did not necessarily mean that this domain had no barrier function, since such a function might be suppressed by much stronger silencing activity associated with the silencing domain.
Was the lack of barrier function of fusion constructs L2–L5 due to a lack of expression of these proteins? To address this possibility, we examined the levels of L1–L6 proteins as well as L7 and L8 proteins described in Figure 5B by western blotting using an antibody against LexA. As shown in Figure 5C, L5 was expressed at a level comparable to L6, and L4 was expressed at a higher level than L6. Therefore, the lack of barrier function of L4 and L6 was not the result of their lack of expression. On the other hand, the level of L2 or L3 was significantly lower than that of L6, hence the lack of barrier function of L2 and L3 might be due to their insufficient expression.
As an important control for the above targeting experiments, we showed that plasmids L1–L6 (Fig. 5A) as well as L7 and L8 (Fig. 5B) bearing LexA-fusion genes had no effect on URA3 silencing in strain 1 (Fig. 1) in which there was no binding site for LexA present between HML-I and URA3 (data not shown). Hence, the effect of the fusion proteins on URA3 silencing was the result of their targeting to specific loci, not the consequence of non-specific, indirect functions (if any) of these proteins.<br><br>Targeted activation domain of Adr1p or Gcn4p can also act as a barrier factor
We have shown above that ADH2-UAS bearing Adr1p-binding sites and HIS3-UAS bearing Gcn4p sites both had detectable barrier activity (Fig. 1, strains 25 and 26). Given that tethering the activation domain of Rap1p is sufficient for barrier function, we tested if targeting the activation domain of Adr1p or Gcn4p could also establish a barrier. Adr1p contains four separate activation domains (TADI–TADIV) that can contact Ada2p, Gcn5p and/or TFIIB (45). TAD III (415–467) interacts with only Gcn5p but not Ada2p or TFIIB. We fused this domain of Adr1p and the activation domain of Gcn4p (102–149) to LexA, respectively (Fig. 5B, L7 and L8). Targeting these fusion proteins to two LexA-binding sites between HML-I and URA3 in strain 50 dramatically reduced URA3 silencing (Fig. 5B, compare L7 and L8 to L1). Therefore, like LexA-Rap1p (627–695), both LexA-Adr1p (415–467) and LexA-Gcn4p (102–149) also had barrier activity. Barrier function of these fusion proteins was significantly decreased in strain 51 in which only one LexA site was inserted between HML-I and URA3 (compare strains 50 and 51 carrying plasmid L7), indicating that the potency of such barriers was dependent on the amount of tethered fusion proteins.<br><br>
DISCUSSION
We have shown in this report that UASs of many highly expressed genes could function as barriers to the spread of transcriptional silencing. These newly identified barrier elements together with previously described barrier elements (STARs, HMR-tRNA gene, TEF2-UAS and CHA1-UAS) (9–11,21) are scattered across the <span class="yellow">yeast</span> genome and may play a role in dividing the genome into functional domains. Analyses of the barrier UASs suggest that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p can participate in barrier function. Barrier activity often requires concerted actions of more than one factor-binding site. Our mutational analysis of TEF2-UAS suggests a direct involvement of Rap1p in its barrier function. It is noteworthy that not all UASs of highly expressed genes have barrier activity. Many non-barrier UASs also consist of multiple regulator binding sites, therefore other unknown factors might determine whether a UAS could act as a barrier.
How Rap1p and the other transcriptional regulators act to block silencing is not clear. Since they can all act as transcriptional activators, one natural explanation is that they may overcome the silencing effect of a silencer and directly activate the reporter gene. However, it has been well documented that Rap1p and the other transcriptional regulators usually act at relatively short distances (less than ∼600 bp upstream of the translation start codon) (28,29,46), but the binding sites for these regulators were >800 bp from the translation start codon in most of the silencing-blocking tests in this report (Figs 1 and 2). In the test shown in Figure 6, TEF2-UAS blocked ADE2 silencing when it was ∼2 kb downstream from the translation start codon of the gene. Moreover, Fourel et al. (44) demonstrated that the barrier function (or anti-silencing function) of an activator was independent of its role in activating transcription. Therefore, it is unlikely that the ability of a UAS to prevent gene silencing is due to its direct activation of the gene. In light of this, our results are in agreement with the notion that transcriptional regulators can also participate in defining the boundaries of functional chromosomal domains.
A few models have been proposed for the barrier functions of transcriptional regulators. One model proposed that binding of a regulator to a barrier led to the formation of a nucleosome-free region thereby disrupting a continuous nucleosome array (11). Since the SIR complexes spread by self-interaction and interacting with adjacent nucleosmes in a stepwise fashion (4), a nucleosome-free gap could present a barrier to the spreading SIR complex. This ‘nucleosome gap’ hypothesis is consistent with that certain barrier factors such as Rap1p and Reb1p could indeed prevent nucleosome formation around their binding sites (14) and the barrier function of TEF2-UAS correlated with its ability to alter local chromatin structure (this report). In fact, many insulator/boundary elements in higher organisms have also been shown to alter local chromatin structure. For instance, the <span class="yellow">chicken</span> β-globin insulator was linked to changes in nucleosome positioning, and the <span class="yellow">Drosophila</span> scs and scs′ boundary elements are associated with unusual chromatin structures that are hypersensitive to nucleases (40,41). However, it is not clear whether any chromatin change associated with barrier function in <span class="yellow">yeast</span> or insulator function in higher cells is the cause or effect of the function.
Another model for barrier action proposed by R. Kamakaka and colleagues (21) proposed that barrier factors recruit chromatin modifying and/or remodeling complexes to counteract the more compact, hypoacetylated silent chromatin. In agreement with this hypothesis, targeted HATs SAS2, GCN5 and ESA1 all had barrier activity (21; Y.-H. Chiu, Q. Yu and X. Bi, unpublished results). Interestingly, Rap1p, Adr1p, Gcn4p and Abf1p share the ability to recruit HAT complexes NuA4 and/or SAGA to target genes (16,45,47–49). Therefore, these factors that are capable of binding to the newly identified barriers may function as barrier factors by recruiting HAT complexes. In fact, recruitment of HAT activity has also been suggested to be the mechanism underlying the function of the <span class="yellow">chicken</span> β-globin insulator (6), although it was not known what HAT might be involved. Our demonstration that tethering the activation domain of Rap1p, Adr1p or Gcn4p alone recapitulates barrier activity is consistent with the above HAT model for barrier function, since at least the activation domain of Adr1p has been shown to directly interact with HAT complexes (45). However, we haven’t ruled out the possibility that the barrier function of the targeted activation domain of Rap1p, Adr1p or Gcn4p is not related to the barrier function of the native protein. Note that the ‘nucleosome gap’ model and the HAT model are not mutually exclusive since the nucleosome gap could result from destabilization of nucelosomes caused by HAT function. A recent intriguing study showed that targeting proteins of the nuclear pore complex (NPC) could also establish a boundary for silent chromatin, and that tethering to NPC was required for boundary activity (50). This led to a revisit to the looping model for the function of boundary/insulator elements (51). However, an alternative explanation was that the NPC compartment could simply be rich in transcriptional activating factors or poor in silencing factors like the SIR proteins (50).
In summary, results from this and other studies indicate that factors previously identified as positive and/or negative regulators of gene activation could also function in demarcating distinct domains of gene activity. How these factors carry out their boundary function is not resolved but recruitment of other factors (e.g. chromatin modifying complexes) is likely to be involved.<br><br>NOTE ADDED IN PROOF
After this work was completed and submitted for publication the first time, Fourel et al. (52) reported that targeted activation domains of Rap1p and Abf1p had insulating capacity.<br><br>
</body></html>