
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>salmon</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1131934</h3>HP1 modulates the transcription of cell-cycle regulators in <span class="yellow">Drosophila melanogaster</span>
Abstract
Heterochromatin protein 1 (HP1) was originally described as a non-histone chromosomal protein and is required for transcriptional gene silencing and the formation of heterochromatin. Although it is localized primarily at pericentric heterochromatin, a scattered distribution over a large number of euchromatic loci is also evident. Here, we provide evidence that <span class="yellow">Drosophila</span> HP1 is essential for the maintenance of active transcription of euchromatic genes functionally involved in cell-cycle progression, including those required for DNA replication and mitosis. Depletion of HP1 in proliferating embryonic cells caused aberrant progression of the cell cycle at S phase and G2/M phase, linked to aberrant chromosome segregation, cytokinesis, and an increase in apoptosis. The chromosomal distribution of Aurora B, and the level of phosphorylation of histone H3 serine 10 were also altered in the absence of HP1. Using chromatin immunoprecipitation analysis, we further demonstrate that the promoters of a number of cell-cycle regulator genes are bound to HP1, supporting a direct role for HP1 in their active transcription. Overall, our data suggest that HP1 is essential for the maintenance of cell-cycle progression and the transcription of cell-cycle regulatory genes. The results also support the view that HP1 is a positive regulator of transcription in euchromatin.<br><br>INTRODUCTION
Chromatin in higher eukaryotes is subdivided into different functional compartments termed heterochromatin and euchromatin (1). Heterochromatin differs from euchromatin in its DNA composition, replication timing, condensation throughout the cell cycle, and its ability to silence euchromatic genes placed adjacent to or within its territory, often described as position-effect-variegation (PEV) (2).
Heterochromatin protein 1 (HP1) was the first protein identified in <span class="yellow">Drosophila melanogaster</span> as a heterochromatin-associated protein (3); the corresponding gene has been cloned from a number of organisms and is highly conserved from <span class="yellow">yeast</span> to <span class="yellow">human</span> (4). Polytene chromosome staining showed that, in <span class="yellow">Drosophila</span>, HP1 is distributed mainly in pericentric heterochromatin, telomeric heterochromatin, the banded small fourth chromosome (5–8), as well as ∼200 individual loci scattered throughout the euchromatic chromosomal arms (5). The gene encoding HP1 in <span class="yellow">D.melanogaster</span>, Su(var)2-5, was isolated as a suppressor of PEV (9–11). The protein contains a highly conserved motif, the chromo (chromatin organization modifier) domain, similar to Polycomb (Pc), a repressor of homeotic genes (12). The association between HP1 and pericentric heterochromatin is believed to occur via the chromo domain of HP1 and the N-terminal tail of histone H3 methylated at lysine 9 (13,14), generated by histone methyltransferase-Su(var)3-9, a partner of HP1 in pericentric heterochromatin (15). The C-terminal chromo ‘shadow’ domain of HP1 interacts with other silencing complexes to suppress local transcriptional activity (15–18). However, studies of HP1 chromosomal distribution also showed that HP1 does not always co-localize with lysine 9 methylated histone H3 or Su(var)3-9, especially in euchromatic regions (19–21); in some cases, HP1 is found directly bound to DNA (22,23). All these features argue for distinct roles for HP1 in chromatin and in epigenetic gene regulation.
HP1 is believed to be an essential structural protein protecting the integrity of chromosomes during cell division (8,24). Swi6, the homolog of HP1 in <span class="yellow">fission yeast</span>, is dispensable for survival, but its deletion results in lagging chromosomes during anaphase, and a high rate of chromosome loss (25,26). Mutations of HP1 in <span class="yellow">D.melanogaster</span> result in late larval lethality, chromosome breakages/loss, telomere fusion and a high frequency of cells with abnormal anaphase (8,27). Null alleles of the HP1 functional partner in <span class="yellow">mice</span> (SUVAR39) also showed various chromosomal defects (28), supporting a conserved role for heterochromatin proteins in the regulation of chromosome dynamics during cell-cycle progression. However, the mechanism(s) involved remains to be understood.
In this study, we utilized <span class="yellow">Drosophila</span> embryonic Kc cells and an RNA interference (RNAi)-based approach to demonstrate that HP1 plays an important role at S phase and G2/M phases during the cell cycle. We further show that nearly one-third of known/predicted cell-cycle regulators require HP1 to maintain their active transcription. These genes include MCMs, Orc4, CDC45L, INCENP, Aurora B, CAF1, Bub1, Bub3 and a few other cell-cycle regulators. ChIP analysis suggests that HP1 plays a direct role in their transcription. Therefore, the results of this study provide an alternative explanation for the specific role of HP1 in the regulation of chromatin dynamics and in cell-cycle progression.<br><br>MATERIALS AND METHODS
RNAi in Kc cells
<span class="yellow">Drosophila</span> Kc cells were routinely cultured at 25°C in Schneider <span class="yellow">Drosophila</span> medium (GIBCO) supplemented with 10% fetal <span class="yellow">calf</span> serum, 160 μg/ml penicillin, 250 μg/ml streptomycin, and 4 mM l-glutamine. Double-stranded RNA (dsRNA) of HP1 was generated by incubation of single-stranded RNA in annealing buffer (100 mM potassium acetate, 30 mM HEPES-KOH, pH 7.4, 2 mM magnesium acetate) for 3 min at 95°C and then placed in a beaker with water at 75°C and allowed to cool slowly to room temperature. The detailed procedure of RNAi was carried out according to the established protocols (). Briefly, Kc cells were seeded in a six-well dish using serum-free medium at 1 × 106 cells/ml. HP1 dsRNA (5 μg/ml) was added to the cultured Kc cells. After 60 min at room temperature, 2 ml of medium containing 10% serum was added to each well and the plates transferred to 25°C for up to 8 days. Western blotting and RT–PCR were carried out using the extract/total RNA isolated from control and dsRNA-treated cells on days 2, 6 and 8.<br><br>Cell-cycle and apoptosis analysis
The procedure for flow cytometric analysis of Kc cells followed that in the manual provided with the BrdU flow kit (BD PharMingen). The cells were fed with BrdU for 4 h, then scraped and collected. Fluorescence was measured using a FACSCalibur (Becton Dickinson). Data collection and analysis were performed using CellQuest software.<br><br>Electrophoresis and immunoblotting
Cell extracts (15 μg) were fractionated by 10% SDS–PAGE, then transferred to Hybond-P PVDF membranes (Amersham) and probed with primary antibodies (CIA9), and secondary antibodies (anti-<span class="yellow">rabbit</span> or anti-<span class="yellow">mouse</span> <span class="yellow">horseradish</span> peroxidase-conjugated IgG), obtained from Jackson Immunoresearch Laboratories. Enhanced chemiluminescence reagents (Amersham Pharmacia Biotech) were used for signal detection.
For the analysis of H3 ser10 phosphorylation, we used whole-cell extracts from 700 000 Kc cells (control and RNAi at day 8). Western blotting was performed using polyclonal antibodies against ser10-phosphorylated histone H3 at a dilution of 1:1000 (Upstate). Kc control cells arrested in mitosis by incubation in 25 µM colchicine (Sigma) for 24 h were also analyzed for comparison.<br><br>Immunofluorescence
Kc cells were seeded onto polylysine slides, fixed with 4% formaldehyde for 15 min and permeabilized with 0.5% Triton X-100 for 5 min. The incubation with primary antibodies was carried out in blocking solution for 1 h.
For staining of mitotic cells, the cells were permeabilized using PBST (PBS containing 0.3% Triton X-100) and stained with polyclonal antibody against <span class="yellow">Drosophila</span> Aurora B at 1:200 dilution and monoclonal <span class="yellow">mouse</span> at anti-β-tubulin 1:300 dilution (Chemicon International) as primary antibodies. Secondary antibodies were anti-<span class="yellow">rabbit</span> coupled with Alexa 488 (1:500) and anti-<span class="yellow">mouse</span> coupled to Alexa 546 (1:500) (Molecular Probes, Eugene, Oregon). Images were acquired using a confocal LSM510 META microscope (Zeiss). Stacks of images were analyzed using the IMARIS 4.0 program (Media cybernetics, Carlsbad, CA).<br><br>Antibodies
Affinity-purified polyclonal antibodies of HP1 (<span class="yellow">rabbit</span> #192 and #187, 5 μg) and 5 μg of polyclonal anti-HA antibodies (Sigma) were used in each ChIP reaction. The specificity of the HP1 polyclonal antibodies was determined using various approaches, including western blotting assay, immunofluorescence staining and immunoprecipitation to pull down HP1 (data not shown). The monoclonal antibody HP1–CIA9 (5) was used at a dilution of 1:20 in immunoblotting assays.<br><br>Microarray analysis and RT–PCR
Total RNA was isolated from control and HP1-depleted Kc cells at day 8 using an RNeasy kit (Qiagen). RNA labeling and microarray data analysis followed the standard protocol from Affymetrix. We used ANOVA (P < 0.001) to assess the expression confidence for each gene.
For RT–PCR analysis, poly(A)+ mRNA was purified with the Oligotex Direct mRNA kit (Qiagen) according to the manufacturer's instructions. The purified poly(A)+ RNA was reverse transcribed using the Thermoscript kit (Invitrogen). The cDNA was then used for PCR amplification for 35 cycles with gene-specific primers. PCR products were scanned after electrophoretic separation with a Typhoon Scanner, quantified using ImageQuant software (Amersham Biosciences) and normalized for amplification of the Actin5c transcript. The sequence of primers used for RT–PCR and ChIP analysis are provided in the Supplementary Material.<br><br>ChIP
ChIP was performed according to Orlando et al. (29) and the protocol provided by Upstate () with some modifications. In brief, 1–2 × 108 Kc cells were prepared and fixed in 1% formaldehyde. Nuclei were isolated according to a standard procedure in Current Protocols (), then resuspended in 1.7 ml of lysis buffer (50 mM Tris, pH 8.0, 10 mM EDTA, 1% SDS and protease inhibitors) and sonicated using a Branson sonifier 250. Chromatin fractions in the size range 0.2–0.8 kb were used to perform immunoprecipitation experiments. We used 5 μg affinity-purified polyclonal antibodies (#192 and #182 for HP1; HA antibody for control) and 1 ml of <span class="red">salmon</span> sperm DNA/protein-A-agarose (Upstate) pre-cleared chromatin lysate in each reaction. The mixture was then rotated at 4°C overnight and the recovered beads were washed twice with 1 ml of Low salt buffer (Upstate), once with High salt buffer (Upstate), once with LiCl buffer (Upstate) and twice with TE at 4°C for 8 min. ChIP DNA was extracted according to the standard procedures (29).<br><br>
RESULTS
Depletion of HP1 in <span class="yellow">Drosophila</span> Kc cells
Various chromosomal defects in the cell cycle have been observed in embryos or larval tissues of <span class="yellow">Drosophila</span> HP1 mutants (8,27). However, the presence of maternally loaded HP1 in embryos and the lethality of HP1 mutants at late larval stages have so far precluded a systematic study of the role of HP1 in cell-cycle regulation. Therefore, we used <span class="yellow">Drosophila</span> Kc cells, a cell line derived from <span class="yellow">Drosophila</span> embryos, as a model system to address this problem. HP1 transcripts were depleted using an RNAi-based approach (see Materials and Methods). The reduction in HP1 expression was measured both by RT–PCR and by western blotting analysis (Figure 1A). A significant reduction in the HP1 expression was already evident after 2 days treatment with HP1 dsRNA. Cells at day 8 showed a reduction in HP1 of ∼90% (Figure 1A) and were therefore used in all subsequent experiments.<br><br>Cell-cycle progression at S and G2/M phase is altered in the absence of HP1
The impact of HP1 loss on the cell cycle of Kc cells was determined using cell-cycle profile analysis of HP1-depleted and control cells. The percentage of cells in S phase was determined by BrdU incorporation, and total DNA content by 7-amino-actinomycin (7-AAD). The results showed that the depletion of HP1 (day 8) caused a decrease in S-phase cells of at least 4-fold, and a 2-fold decrease in G2/M-phase cells (Figure 1B), although no significant effect was found at the G1 phase. In addition, depletion of HP1 caused a greater than 7-fold increase in the number of apoptotic cells. These results, therefore, confirm that HP1 is an important regulator during the cell cycle, especially at the S and G2/M phases.<br><br>Cell-cycle regulators require HP1 to maintain their active transcription
To ask whether the cell-cycle defects were due to changes in the transcription of genes functionally involved in S phase and the G2/M phase, we next assessed global changes in gene transcription following depletion of HP1. Expression profile analysis was performed using total RNA isolated from both HP1-depleted Kc cells and control Kc cells, and an Affymetrix <span class="yellow">Drosophila</span> chip. For each experiment, we used total RNA isolated from two independent HP1-depleted and control samples, and at least two independent experiments were performed.
The microarray analysis showed that loss of HP1 function in Kc cells resulted in alterations in transcription of >500 genes: ∼400 genes were down-regulated and ∼120 genes were up-regulated (>1.5-fold, ANOVA). The function of these genes ranged from cellular enzymes, signal transduction molecules, and membrane and cell structural proteins, to nucleic acid-binding proteins and cell-cycle regulators (Figure 2A). At the chromosomal level, the genes targeted by HP1 appeared to be distributed along all euchromatic chromosomal arms (data not shown), supporting a global role of HP1 in euchromatic gene regulation (20).
Among 60 known/predicted genes associated with DNA replication function, 15 were down-regulated in the absence of HP1 (Figure 2B). These included McM2, McM5, McM6 and CDC45L, which are required for processive DNA replication and correct chromosome condensation (30–32). Other genes involved in DNA replication, such as components of the origin recognition complex (Orc)—Orc4, Caf1, Gnf1, Dref1, DNA polymerase-γ and Tam—were also down-regulated (Figure 2B). Aurora B and inner centromere protein (INCENP), known to be required for kinetochore assembly, chromosome condensation and bipolar chromosome attachment during mitosis (33), also showed a reduction in transcription. A similar loss of transcription was observed in Bub1 and Bub3 (Figure 2B), encoding mitotic checkpoint control proteins (34,35). Loss function of Bub1 has been shown to cause chromatin bridges to extend between the two separating groups of chromosomes, and extensive chromosome fragmentation in anaphase cells (35).
We confirmed the changes in the transcription of cell-cycle regulators using semi-quantitative RT–PCR, which gave results consistent with the microarray analysis. In addition, cell-cycle regulator genes, such as McM3, McM7 and Asp (abnormal spindle), were also confirmed to be down-regulated (Figure 2C). Collectively, these results demonstrate that HP1 is indeed involved in the regulation of transcription of cell-cycle regulators.<br><br>HP1 is required for Aurora B distribution and histone H3 phosphorylation
INCENP is localized to the centromeric region of chromosomes at metaphase and the spindle midzone at anaphase, which then targets Aurora B, a kinase essential for histone H3 ser10 phosphorylation, to these sites (36). Loss of function of both these ‘chromosomal passenger’ proteins causes abnormal chromosomal segregation at metaphase, as well as certain cytokinesis defects (36,37). The loss of transcription of both INCENP and Aurora B after depletion of HP1, therefore, raised the possibility that localization of Aurora B (Figure 3 and data not shown) may be altered. Staining of HP1-depleted Kc cells with anti-Aurora B antibodies indeed revealed an altered localization of Aurora B and, in a number of cases, a complete loss of Aurora B (Figure 3A). Consistent with the loss function of Aurora B, the spindles in the metaphase cells were also disorganized, with a large number of cells showing an altered prometaphase chromosome alignment (Figure 3A). Some showed extensive chromosome fragmentation (Figure 3B), or the presence of a third spindle pole-like structure as indicated by beta-tubulin (Figure 3A). At telophase, we observed defective separating cells with an extra cell envelope-like structure without nuclei (Figure 3A). Chromatin bridges or lagging chromatids at telophase were also evident in some cells (Figure 3C); however, in some cases, localization of Aurora B appeared not to be affected, arguing that other pathways are possibly involved.
We next analyzed changes in histone H3 serine 10 phosphorylation, since the loss of transcription of INCENP is known to affect localization of Aurora B (33), which is essential in the regulation of histone H3 phosphorylation (36). Total cell extracts from HP1-depleted Kc cells were analyzed by western blotting (Figure 3D). The results indeed showed a severalfold reduction in H3 ser10 phosphorylation after depletion of HP1, consistent with the functional disruption of INCENP and Aurora B in the absence of HP1.<br><br>HP1 directly targets genes encoding cell-cycle regulators in euchromatin
To test whether the loss of transcription of genes involved in DNA replication and mitosis was a direct effect of the loss of HP1, we performed a ChIP analysis to determine whether HP1 is physically associated with these genes. Chromatin lysates from formaldehyde-fixed Kc cells were sonicated into small chromatin fragments (0.2–0.8 kb) and immunoprecipitated with polyclonal antibodies against <span class="yellow">Drosophila</span> HP1. As a control, we used a mock precipitation (beads only) and polyclonal antibodies against HA. Our ChIP results showed that known transposable elements distributed in heterochromatin, such as F-element, TART and 1360 (7,38), were all enriched in HP1 binding (Figure 4), which is also consistent with a previous study (20).
Using the same ChIP DNA material, we then attempted to determine whether HP1 was enriched in genes involved in DNA replication. Primers were designed to cover the promoter regions of selected genes. The results showed that McM3, McM5 and Tam were all enriched in HP1 binding (Figure 4). However, McM7 appeared to be HP1-negative, although its transcription was also affected by the loss of HP1 function. Genes essential for mitosis, such as Aurora B, were also HP1-positive (Figure 4). These results demonstrate that these cell-cycle regulator genes are directly targeted by HP1 in their promoter regions.<br><br>
DISCUSSION
In this study, we used microarray and RT–PCR techniques to demonstrate that transcription of cell-cycle regulators is misregulated in the absence of HP1. Certain defects in S phase may be a direct consequence of the loss of transcription of DNA replication genes such as McM2, McM5, McM6, CDC45L, Orc4 and others, since these genes have been functionally implicated in the initiation of DNA replication and/or the progression of replication forks (39). Depletion or mutation of these genes has been shown to result in DNA damage (32), the blockage of replication forks (39), increased chromosome loss/genome instability, and defective condensation (30).
The reduction in the number of cells in G2/M phase may be a consequence of the reduction in transcription or functional disruption of INCENP, Aurora B, Bub1, and Bub3 (34,36). Chromosome segregation defects, such as chromosome fragmentation and chromatin bridges in anaphase/telophase cells, and certain cytokinesis defects in HP1-depleted cells, mimic the phenotype of cells with loss function of INCENP, Aurora B or Bub1 (35–37). The mislocalization of Aurora B in the absence of HP1 is also consistent with the loss of transcription and functional disruption of INCENP (37), and the reduction in Aurora B transcription may be partially responsible for the observed chromosomal defects, including loss of histone H3 phosphorylation at serine 10.
HP1 is also known to physically interact with certain components of replication complexes such as ORCs and MCMs (30,40,41), with the inner centromere protein INCENP (42) and the chromatin assembly factor CAF1 (43) promoting delivery of HP1 to heterochromatin sites (44). Loss of HP1 is, therefore, expected to cause disruption to such HP1-associated complexes, and will partially contribute to the chromatin/chromosomal defects in HP1 mutants (8,27) and HP1-depleted Kc cells. It is therefore well possible that the loss of transcription of these cell-cycle regulator genes, and consequent disruption of HP1 functional complexes or heterochromatin structure, all contributed to the cell-cycle defects observed.
The ChIP assay supports the hypothesis that the loss of transcription of cell-cycle regulator genes is a direct effect of the lack of HP1. Aurora B, McM3, and McM5 were all bound by HP1 at their promoter regions, although other cell-cycle regulators, such as McM7, were HP1-negative, implying that the altered transcription in these genes might be a secondary effect of the loss of HP1.
A previous study in <span class="yellow">Drosophila</span> Kc cells (20) employed an approach based on the ectopic expression of a fusion protein of HP1 with a prokaryotic DNA adenine methyltransferase and identified a number of methylated targets in the genome. In this study, MCM3 and MCM5 were not found to be methylated, indicating lack of association with HP1. On the other hand, heterochromatin repeats, such F-element and 1360, were consistently found to be HP1-enriched both here and in the previous study. It remains to be determined whether these discrepancies are due to the different experimental systems used. However, we note that the previous study was performed using a cDNA array, while we observe binding of endogenous HP1 at the promoter of these genes. Similarly, another study using chromatin immunoprecipitation in larvae also showed few HP1-positive genes that were not detected in Kc cells by the Dam ID approach (21).
A large number of genes affected by the loss of <span class="yellow">D.melanogaster</span> HP1 in larval tissues (21) seem to be different from that in embryonic Kc cells. The change in the transcription of Aurora B and few cell-cycle regulators reported in this study is also not found among the HP1-affected genes at larval stage (21). This may be due to specific role(s) of HP1 in different stages of development. Alternatively, it is also possible that the impact of HP1 in the transcription of cell-cycle regulators in proliferating cells is underestimated when performing the analyses on larval tissues, and thus on mixed populations of both proliferating and differentiating/differentiated cells.
HP1 is generally known as a transcriptional repressor, as supported by several lines of evidence: silencing of a euchromatic reporter gene in heterochromatin requires HP1 (10,11), tethering of HP1 next to a euchromatic reporter gene causes silencing (45), and the repression of genes within euchromatic region 31 bound by HP1 is relieved in the absence of HP1 (46). In contrast, genes in heterochromatin, known as heterochromatic genes, such as light and rolled, seem to require HP1 to maintain their active transcription (47,48). The level of transcription of heterochromatic genes was dramatically reduced in a mutated HP1 background (47–49). It was therefore proposed that HP1 may function as a positive regulator of transcription of these genes (50,51), although the exact regulation mechanism remains unclear. A study of heat-shock genes found that HP1 is associated with RNA transcripts in the coding region, and is also a positive regulator of their transcription (52,53). The chromatin association of HP1 at the promoter region of active euchromatic genes demonstrated from this work and others, and its independence from histone H3K9 methylation (21), all suggest that mechanism whereby HP1 modulates transcription of euchromatic genes is potentially distinct from its role in heterochromatin formation.
Collectively, the results of this study demonstrate that HP1 plays an essential role in cell-cycle progression, and support the view that HP1, in addition to its role in heterochromatin, can act as a positive transcriptional regulator of euchromatic genes.<br><br>SUPPLEMENTARY MATERIAL
Supplementary Material is available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA328328</h3>Mapping irradiation hybrids to cosmid and <span class="yellow">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products.
Abstract
A direct hybridization protocol is described for screening cosmid and <span class="yellow">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual Alu-PCR product before library screening. A series of <span class="yellow">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis.Images<br><br><br><br><br><br>
 Nucleic Acids Research, Vol. 19, No. 12 3315 <br><br> Mapping irradiation hybrids to cosmid and <span class="yellow">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products <br><br> Anthony P.Monaco*, Veronica M.S.Lam2, Gunther Zehetner, Gregory G.Lennon, Christal Douglas, Dean Nizetic, Peter N.Goodfellow1 and Hans Lehrach <br><br> Genome Analysis Laboratory and 'Molecular <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, 44 Lincoln's Inn Fields, London WC2A 3PX, UK and 2Department of Biochemistry, Li Shu Fan Building, Sassoon Road, University of Hong Kong, Hong Kong <br><br> Received March 12, 1991; Revised and Accepted May 16, 1991 <br><br> ABSTRACT <br><br> A direct hybridization protocol is described for screening cosmid and <span class="yellow">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual AluPCR product before library screening. A series of <span class="yellow">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis. <br><br> INTRODUCTION <br><br> The generation of <span class="yellow">human</span> DNA probes specific for individual chromosomes and subregions of chromosomes has been advanced with Alu-sequence primed polymerase chain reaction amplification (Alu-PCR, 1-3). This method specifically amplifies sequences between Alu repeats from <span class="yellow">human</span> DNA in somatic cell hybrids and <span class="yellow">yeast</span> artificial chromosomes (YACs, 4). Individual Alu-PCR products can be purified from agarose gels or ligated into plasmid vectors to screen for single copy sequences. Unique Alu-PCR products are then localized to certain chromosome regions using DNA blots of somatic cell hybrid panels. Once localized, Alu-PCR fragments can be screened against genomic libraries to isolate longer DNA fragments from the region of interest. As an alternative to this multistep process we have developed a hybridization protocol for screening of cosmid and YAC libraries directly with pools of Alu-PCR products. <br><br> Two new <span class="yellow">human</span> specific Alu primers were used to generate DNA probes from a series of irradiation-reduced hybrids containing multiple <span class="yellow">human</span> X chromosome fragments of 1-2000 kb on a <span class="yellow">hamster</span> chromosome background (5; P.N.G., unpublished). The Alu-PCR products were hybridized as a pool of probes to X-specific cosmid and YAC libraries, after <br><br> competitive reassociation with an excess of <span class="yellow">human</span> DNA to both the library filters and radioactively labelled Alu-PCR products. Comparisons were made between clones identified by overlapping irradiation hybrids and single copy DNA probes hybridized to the cosmid and YAC libraries. <br><br> METHODS <br><br> Two <span class="yellow">human</span> Alu sequence primers were generated which were shown to be <span class="yellow">human</span> specific; 3144 from the 3' end of Alu: 5'-GAGCGAGACTCCGTCTCAAA-3' and 2729 from the 5' end of Alu: 5'-GTGGATCACCTGAGGTCAGGAGTTC-3'. All PCR reactions were carried out with 100 ng of hybrid DNA and 0.7 ,^g of a single Alu primer in 100 d41 of 0.01 M Tris-HCl pH 8.3, 0.0015 M MgCl2, 0.05 M KCI, 200 AtM each of dNTPs, 10% dimethlysulfoxide, and 2.5 units of Cetus Taq polymerase. Reactions were 30 cycles of 94?C for 2 min, 57?C for 2 min, and 74?C for 4 min followed by a final extension time at 74?C for 9 min. Reactions products were analyzed on 1 % agarose gels and shown to contain between five and twenty fragments, with sizes ranging from 0.1 to 2.0 kb. <span class="yellow">Chinese hamster</span> DNA and no DNA PCR reactions were done to control for non-<span class="yellow">human</span> products (data not shown). <br><br> Alu-PCR products were separated from Alu oligomers over Qiagen columns, and approximately 50-100 ng were labelled by random hexamer priming (6). The radioactively labelled pool of fragments was prehybridized with 37.5 ,^g of total <span class="yellow">human</span> DNA and 12.5 tsg of <span class="yellow">hamster</span> DNA immobilized on a cellulose support matrix, prepared as previously described (7). Reactions were at 65?C in 1 ml of 0.75M NaCl, 0.05M sodium phosphate pH 7.2, 0.005M  EDTA, 0.1 % sodium dodecyl sulphate (SDS), 0.5 mg/ml heparin, and 100 jig/ml denatured <span class="red">salmon</span> sperm DNA. The cellulose was pelleted and the supernatant boiled for 2 min every 12-16 hours (three times in two days). <br><br> Cosmid and YAC library filters (Hybond N+, Amersham) were prehybridized at 42?C for 16 hours with 100 jig/ml <br><br> * To whom correspondence should be addressed at <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford OX3 9DU, UK <br><br> 1991 Oxford University Press <br><br> 3316 Nucleic Acids Research, Vol. 19, No. 12 <br><br> Chromosome X <br><br> .,.., ., < <br><br> _ ., . < ., <br><br> *2.' 1- 7-_ <br><br> _-_ 1. J <br><br> ,.1 .. - 1 , _ i - _ 1 . 1 <br><br> 1 ' 4 - -1 + <br><br> 1 1 L L . i <br><br> . 1 . _ 1  <br><br> 1 1 s. 1 i <br><br> 1 i ;. --r' F 1 -<br><br> 1 -1-l <br><br> j i-_1. 1 <br><br> -   _ 1 .. <br><br> s- S . i n -. <br><br> .. : ,1' ,. -1 _ E I i ? <br><br> ,. . _ _ _ <br><br> G-<br><br> . <br><br> *.-.,., z <br><br> _ <br><br> _ , <br><br> z ;- [X1 <br><br> 21 , In <br><br> 27  38 45   48  54  74   86 <br><br> I  1      I <br><br> I <br><br> I <br><br> 107 <br><br> MD <br><br> I <br><br> :I                                                       I  <br><br> I        I <br><br> I I <br><br> Figur 2a and 2b: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 48 to duplicate copies of 22 x22cm filters containing 9216 <span class="yellow">human</span> X chromosome cosmids (8). 2c: Hybridization of Alu-PCR products generated with Alu primer 3144 from an independent hybrid (54) to a third identical cosmid filter. <br><br> Figure 1: A schematic diagram of the <span class="yellow">human</span> X chromosome alongside the approximate cytogenetic location of fragments identified in nine irradiation hybrids (numbers across the top). The <span class="yellow">human</span> X fragments were identified by hybridization of 27 known DNA markers (indicated by a black line; P.N.G, unpublished) or by cosmids in common with unique X chromosome probes in the reference library database (open boxes and Table 1). The size of the lines and boxes relate to the best cytogenetic location of the probes used according to <span class="yellow">Human</span> Gene Mapping 10.5 (14) and does not indicate the physical extent of the irradiation hybrid fragments. <br><br> denatured and sheared total <span class="yellow">human</span> DNA in 50% formamide, 4XSSC, 0.05 M sodium phosphate pH 7.2, 0.001 M EDTA, 10% dextran sulphate, 1.0% SDS, 50 isg/ml denatured <span class="red">salmon</span> sperm DNA and OxDenhardt's solution. The radioactively labelled Alu-PCR products were denatured and added to fresh <br><br> hybridization solution without <span class="yellow">human</span> DNA competitor at 1 x 106 <br><br> cpm/ml and hybridized at 42?C for 16 hours. Filters were washed in 0.1 xSSC and 1.0% SDS, twice at room temperature and twice at 65?C for 30 min each and exposed to Kodak X-OMAT film for 2-3 days at -70?C with an intensifying screen. <br><br> For each hybridization, two sets of duplicate cosmid filters were used from the ICRF reference library system (8), each containing 9216 flow-sorted <span class="yellow">human</span> X chromosome cosmid clones or approximately 2 chromosome equivalents on a 22 x 22 cm filter (9). The coordinates of signals positive on duplicate cosmid filters were entered into the reference library database (G. Z, unpublished) using a digitizing tablet. For the X chromosome specific YAC library (A.P.M. and H.L., unpublished), about 420 YAC colonies were spotted manually onto filters from 96 well microtiter dishes using a 96 prong device. After growth on selective media for 3 days, YAC filters were processed for hybridization as previously described (10). <br><br> RESULTS <br><br> A panel of 195 X chromosome irradiation hybrids was constructed (50,000 rads) and characterized by DNA hybridization using 27 X chromosome markers and flourescence in situ hybridization using total <span class="yellow">human</span> DNA as probe (Benham et al., 1989; P.N.G., unpublished). This analysis indicated that the hybrids contained multiple small fragments (4-10 fragments of 1000-5000 kb each) with a preferential retaining of centromere sequences (90%). From this panel, nine irradiation hybrids were chosen that contained less than five different regions by DNA probe hybridizations, mostly from the short arm of the X chromosome (Fig 1). All nine hybrids were used in PCR reactions with 3'-Alu primer 3144 and two were used with 5'-Alu <br><br> Figure 3: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 54 to a filter containing 420 YAC clones from the <span class="yellow">human</span> X chromosome. The positive YAC was also identified in a separate hybridization with the DMD probe P20 (12). <br><br> primer 2729. Example hybridizations to a <span class="yellow">human</span> X chromosome cosmid filter in Fig 2 shows the intensity and reliability of positive clones identified on duplicate filters with Alu-PCR products from the same irradiation hybrid (48) and the independence of clones identified with Alu-PCR products from a different hybrid (54). Fig 3 shows a single positive YAC clone after hybridization of Alu-PCR products from irradiation hybrid 54 to a filter containing about 420 YAC clones specific for the <span class="yellow">human</span> X chromosome. <br><br> The total number of cosmids identified with each pool of AluPCR products for each hybrid is shown in Table 1. From the average number of cosmids identified (24) in four chromosome equivalents screened and the estimated average DNA content in each hybrid (3000-15000 kb), the Alu-PCR products generated by a single primer were calculated on average to be 300-1500 kb apart, similar to published estimates for this method (1,2). Only 3-4 cosmids were identified in common using Alu-PCR products generated with either 3' or 5' Alu primers (3144 or 2729) from two hybrids (38 and 45). This shows that separate products were amplified with the two Alu primers since they prime synthesis from opposite ends of the Alu consensus sequence and Alu sequences are oriented in the genome either head to head, <br><br> A <br><br> B <br><br> C <br><br> El <br><br> I0 <br><br> Nucleic Acids Research, Vol. 19, No. 12 3317 <br><br> Table 1. Cosmids identified by hybrids and unique probes <br><br> hybrids       21    27     38    38    45    45    48     54    74    86     107   unique <br><br> primer 3144  3144  3144  2729  3144  2729   3144  3144  3144  33144 3144   probes <br><br> 21     3144  59                                                                   1 27     3144   7    14                                                             2 38     3144   3     1    28                                                       2 38     2729   0     0     4     25                                                0 45     3144  13     2      2     0    32                                          3 45     2729   1     0      0     0     3    31                                    0 48     3144  10     2      4     0     2     0    28                              3 54     3144  11     3     0      0     5     0     5     30                       2 74     3144   5     0     0      0     4     0     5     12    20                 3 86     3144   3     1     0      0     4     1     3      4     3    17           0 107     3144   1     0      1     1     0     0     3     0      0     0    13    0 <br><br> tail to tail or head to tail relative to each other. Therefore, by using the 3'- and 5'-Alu primers in separate PCR reactions with the same hybrid DNA, the total number of products and cosmid clones identified was almost doubled. <br><br> Table 1 also indicates how many cosmids were identified by Alu-PCR products from other hybrids, and 16 cosmids previously identified with unique DNA probes in the reference library database. As can be seen in Fig 1 and previous irradiation hybrid analysis, there is a preferential retention of centromere sequences (2,11). However, there were no cosmids identified in common from all the hybrids positive with centromere sequences. This is probably due to a paucity of Alu repeats in the correct orientation in alphoid centromere sequences and thus few or no Alu-PCR products would be amplified from the centromere. <br><br> Cosmids identified in common by several irradiation hybrids (Table 1) were most likely from regions of overlap outside the centromere area as shown by the previous DNA probe characterization (Fig 1). The overlap regions between hybrids were also seen by 16 cosmids (Table 1, Fig 1) that were hybridization targets of unique DNA markers in the reference library database that mapped in independent experiments to the overlap region. At least for several cosmids this showed that the Alu-PCR products identified target cosmids that were definitely from the expected region contained in the hybrids. For example, hybrids 21 and 54 were both shown to contain part of the Duchenne muscular dystrophy (DMD) locus (Fig 1; P.N.G., unpublished) and had 11 cosmids in common, including one identified by the probe P20 from the deletion hotspot region of the DMD gene (12). In Fig 3 the hybridization of Alu-PCR products from hybrid 54 identified a YAC clone which was also positive for the DMD probe P20 (data not shown). This method also identified fragments in the hybrids that were not seen in the initial DNA characterization (Fig 1 and Table 1). Since the 27 DNA probes were not close enough to each other along the chromosome to detect all possible hybrid fragments (1000-5000 kb), many regions would have been untested. For example, AluPCR products from hybrids 45 and 48 identified several cosmids, also seen by the cDNA for chronic granulomatous disease gene (CYBB, 13) in Xp2 1.1, although this region was not tested in the original hybrid characterization. <br><br> DISCUSSION <br><br> The direct hybridization of Alu-PCR products from somatic cell hybrids to genomic libraries can bypass gel purification or ligation of fragments into plasmid vectors and individual analysis for single copy sequences. Hybridization of Alu-PCR products as <br><br> a pool to ordered array libraries such as the flow-sorted X chromosome cosmid library (9), allows the direct comparison of overlapping hybrids to pinpoint cosmids most likely to be from the region of interest. In conjunction with the reference library database (G.Z, unpublished) with 183 X chromosome probe hybridization entries, cosmids identified with both Alu-PCR products and uniquely mapped X probes immediately map them to the region of interest and proove that the method has worked. Similar hybridization experiments using Alu-PCR products from four overlapping irradiation hybrids identified four cosmids in common that mapped to the region of overlap by independent experiments (F.Muscatelli, A.P.M., P.N.G., H.L. and M.Fontes, in preparation). Since only 27 probes from the X chromosome were used to initially characterize the hybrids and the length of individual <span class="yellow">human</span> fragments in the irradiation hybrids is about 1000-5000 kb, many regions could have been undetected in the original analysis. The direct hybridization of Alu-PCR products to the cosmid reference library detected such fragments since they identified cosmids in common with uniquely mapped probes in regions not tested originally (Table 1 and Fig 1). This should prove to be a sensitive and efficient method to determine content and overlap of irradiation hybrids in conjunction with DNA blot hybridization. However, since the exact length of the <span class="yellow">human</span> DNA fragments for each hybrid and the spacing of Alu-PCR products along the chromosome is not known, it is difficult to directly correlate the number of target cosmids to the DNA content of the hybrids. <br><br> The direct hybridization of Alu-PCR products from irradiation or somatic cell hybrids to total genomic YAC libraries will be especially useful to construct long range YAC contigs from specific subregions of chromosomes. The dissection of a total genomic YAC library by this method may be more efficient than generating chromosome specific YAC libraries from somatic cell hybrids (usually a haploid <span class="yellow">human</span> chromosome on a diploid or greater <span class="yellow">rodent</span> background) or flow-sorted chromosomes, because of the low transformation efficiency of <span class="yellow">yeast</span>. <br><br> ACKNOWLEDGEMENTS <br><br> We would like to thank Gert-Jan Van Ommen for the probe P20 and Stuart Orkin for the CYBB cDNA probe. A.P.M is supported in part by a research fellowship from the Muscular Dystrophy Association of America. V.M.S.L. is supported in part by the Medical Research Grant of the University of Hong Kong. Reference library filters and cosmids identified by Alu-PCR products can be requested from the Reference Library Database, ICRF, 44 Lincoln's Inn Fields, London WC2A 3PX, U.K. <br><br> 3318 Nucleic Acids Research, Vol. 19, No. 12 <br><br> REFERENCES <br><br> 1. Nelson, D.L., Ledbetter, S.A., Corbo, L., Victoria, M.F., Ramirez-Solis, <br><br> R., Webster, T.D., Ledbetter, D.H., and Caskey, C.T. (1989) Proc. Natl. Acad. Sci. USA, 86:6686-6690. <br><br> 2. Ledbetter, S.A., Nelson, D.L., Warren, S.T., and Ledbetter, D.H. (1990) <br><br> Genomics, 6:475-481. <br><br> 3. Cotter, F.E., Hampton, G.M., Nasipuri, S., Bodmer, W.F., and Young <br><br> B.D. (1990) Genomics, 7:257-263. <br><br> 4. Burke, D.T., Carle, G.F., and Olson, M.V. (1987) Science, 236:806-812. 5. Benham, F., Hart, K., Crolla, J., Bobrow, M., Francavilla, M., and <br><br> Goodfellow, P.N. (1989) Genomics, 4:509-517. <br><br> 6. Feinberg, A.P., and Vogelstein, B. (1984) Anal. Biochem., 137:266-267. 7. Hochgeschwender, U., Sutciffe, J.G., and Brennan, M.B. (1989) Proc. Natl. <br><br> Acad. Sci. USA, 86:8482-8486. <br><br> 8. Lehrach, H., Drmanac, R., Hoheisel, J., Larin, Z., Lennon, G., Monaco, <br><br> A.P., Nizetic, D., Zehetner, G., and Poustka, A. (1990) In Davies, K.E. & Tilghman, S.M. (eds.), Genome Analysis Volume 1: Genetic and Physical Mapping. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, pp. 39-81. <br><br> 9. Nizetic, D., Zehetner, G., Monaco, A.P., Gellen, L., Young, B.D., and <br><br> Lehrach, H. (1991) Proc. Natl. Acad. Sci. USA, 88:3233-3237. <br><br> 10. Larin, Z. and Lehrach, H. (1990) Genet. Res. Camb., 56:203-208. <br><br> 11. Cox, D,R., Pritchard, C.A., Uglum, E., Casher, D., Koborl. J. and Myers, <br><br> R.M. (1989) Genomics, 4:397-407. <br><br> 12. Wapenaar, M.C., Kievits, T., Hart, K.A., Abbs S., Blonden, L.A.J., <br><br> denDunnen, J.T., Grootscholten, P.M., Bakker, E., Verellen-Dumoulin, C., Bobrow, M., vanOmmen, G.J.B., and Pearson, P.L. (1988) Genomics, 2:101-108. <br><br> 13. Royer-Pokora, B., Kunkel, L.M., Monaco, A.P., Goff, S.C., Newburger, <br><br> P.E., Baehner, R.L., Cole F.S., Curnette J.T., and Orkin, S.A. (1986) Nature, 322: 32-38. <br><br> 14. Davies, K.E., Mandel, J.L., Monaco, A.P., Nussbaum, R.L. and Willard, <br><br> H.F. (1990) Cytogenet. Cell Genet. 55: 254-313. <h3>pmcA150219</h3>Rap1p and other transcriptional regulators can function in defining distinct domains of gene expression
Abstract
Barrier elements that are able to block the propagation of transcriptional silencing in <span class="yellow">yeast</span> are functionally similar to chromatin boundary/insulator elements in metazoans that delimit functional chromosomal domains. We show that the upstream activating sequences of many highly expressed ribosome protein genes and glycolytic genes exhibit barrier activity. Analyses of these barriers indicate that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p may participate in barrier function. We also present evidence suggesting that Rap1p is directly involved in barrier activity, and its barrier function correlates with local changes in chromatin structure. We further demonstrate that tethering the transcriptional activation domain of Rap1p to DNA is sufficient to recapitulate barrier activity. Moreover, targeting the activation domain of Adr1p or Gcn4p also establishes a barrier to silencing. These results support the notion that transcriptional regulators could also participate in delimiting functional domains in the genome.<br><br>INTRODUCTION
The eukaryotic genome is organized into discrete functional domains in which gene expression is either permitted or repressed. Domains permissive for gene expression are usually defined as euchromatin or active chromatin, and domains that repress gene expression heterochromatin or silent chromatin. The HML and HMR loci in <span class="yellow">Saccharomyces cerevisiae</span> are silent chromatin domains (1). Compared to active chromatin, silent chromatin is more compact and its histone components are hypoacetylated (2). The SIR complex consisting of Sir2p through Sir4p is an integral part of <span class="yellow">yeast</span> silent chromatin. Sir3p and Sir4p interact with the N-terminal tails of histones H3 and H4 providing the structural basis of silent chromatin (1). Sir2p was recently found to be an NAD-dependent histone deacetylase and was suggested to be responsible for histone hypoacetylation in silent chromatin (3). Silent chromatin is initiated at small cis-acting DNA elements known as the E and I silencers flanking each of the HM loci (4).
Increasing evidence indicates that histone deacetylation is key to the propagation of silent chromatin (4). In vertebrates and <span class="yellow">fission yeast</span><span class="yellow">yeast</span>, spread of silent chromatin involves a chain of events of histone H3 deacetylation → H3 methylation → binding of methylated H3 by HP1 (heterochromatin protein 1) or swi6 (4). In <span class="yellow">S.cerevisiae</span>, there is evidence that Sir3p (hence the SIR complex) has much higher affinity to unacetylated histone H4 than acetylated H4 (5). Based on this and the fact that Sir2p is a histone deacetylase, a refined model for the propagation of silent chromatin can be proposed. In this model, Sir2p recruited to the silencer deacetylates histones in an adjacent nucleosome enabling it to bind another SIR complex with high affinity. The nucleosome-bound SIR complex in turn deacetylates the neighboring nucleosome allowing it to recruit a new SIR complex. In this manner, the SIR complex promotes its own propagation along an array of nucleosomes (4).
The fact that transcriptionally silent chromatin can encroach upon active chromatin poses the question of how interspersed domains of silent and active chromatin are demarcated. Studies of the <span class="yellow">Drosophila</span> and vertebrate genomes demonstrated that some domains are delimited by boundary or insulator elements (6). These elements are specialized DNA sequences that act as barriers to enhancers and/or silent chromatin. One of the best characterized insulators is the <span class="yellow">chicken</span> HS4 insulator at the β-globin locus. Histones surrounding this insulator are hyperacetylated indicating the presence of histone acetyltransferase (HAT) activity at the insulator (6). Recently, sequences that could block the spread of silent chromatin have been discovered in <span class="yellow">S.cerevisiae</span> (7,8). These sequences (referred to as silent chromatin barriers) include sub-telomeric anti-silencing regions (STARs), the right boundary of HMR, and the upstream activating sequence (UAS) of the highly expressed TEF2 gene (9–11). The mechanisms underlying barrier functions in <span class="yellow">yeast</span> are not known.
TEF2-UAS contains three tandem repressor activator protein 1 (Rap1p)-binding sites that are necessary and sufficient for its barrier function (11). Rap1p is an abundant sequence-specific DNA-binding protein (12). Variants of the 13 bp consensus sequence for Rap1p binding (ACACC CRYACAYM) (13) lie not only in the UASs of numerous genes but also in the silencers of the HM loci and in telomeric C1–3A repeats. Accordingly, Rap1p functions as a global regulator of transcriptional activation and repression, silencing, as well as telomere length (12). Rap1p performs these diverse functions by interacting with different factors in respective contexts. As an activator, Rap1p binds to the promoters of 362 ORFs (13) and can function via at least three mechanisms. First, Rap1p binding can ‘open up’ local chromatin structure at a promoter to help another activator to bind (14). Secondly, Rap1p bound to DNA can help Gcr1p bind to an adjacent site in the promoter through physical interaction (15). Thirdly, Rap1p is involved in the recruitment of the NuA4 HAT complex and TFIID to ribosome protein genes (RPGs) (16,17). As a silencing factor, Rap1p binds to silencers or telomeric repeats and recruits Sir3p/Sir4p through direct interactions (18). Rap1p also interacts with telomere-specific proteins in executing its role in regulating telomere length (19). It is puzzling that a protein that helps establish silent chromatin could also act as a barrier to its propagation.
In this study, we found that in addition to TEF2-UAS, UASs of many <span class="yellow">yeast</span> genes exhibit barrier activity. Analyses of these UASs indicate that binding sites for transcriptional regulators Abf1p, Reb1p, Adr1p, Gcn4p, as well as Rap1p, may participate in barrier function. These UASs are dispersed across the genome and might play a role in defining functional chromosomal domains. Moreover, we demonstrated that targeting the activation domain of Rap1p, Adr1p or Gcn4p alone, led to barrier function. Since the activation domain of an activator was usually involved in interacting with co-activators and/or components of the transcriptional machinery, these data indicated that other factors might also be involved in barrier function.<br><br>MATERIALS AND METHODS
Plasmids and strains
Plasmids 1–53 were used to make strains 1–53, respectively (see Figs 1–6). Plasmid 1 was previously named pMB22-a that contained a HindIII–BamHI fragment of chromosome III (coordinates 14838–16263) with a URA3 gene inserted at its EcoRV site (20). Plasmid 2 was derived from 1 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SnaBI site. Plasmids 3–38 were identical to 2 except that different fragments from various promoters (see Figs 1 and 2) were inserted at the SnaBI site. Plasmid 29 had a sequence of chromosome III (295327–295713) containing the HMR-tRNA gene (21) inserted at the SnaBI site of plasmid 1. Plasmids 39 and 40 were previously named pYXB26 and pYXB59, respectively (11). Plasmid 41 was identical to pYXB59 except bearing mutations illustrated in Figure 3A. Plasmids 42–48 were previously described as pYXB29, 48, 28, 27, 59, 31 and 37, respectively (11). Plasmid 50 was derived from plasmid 1 by inserting two copies of a sequence bearing the consensus binding sequence of LexA (bold) (22), GGGGTACGTACTGTATGTACATACAGGATATCGGGG, at the SnaBI site. Plasmid 51 was identical to 50 except having only one copy of the LexA-binding sequence. Plasmid 52 was derived from pYXB26 (11) by inserting a SpeI-ADE2-AvrII sequence at the SpeI site. Plasmid 53 was derived from plasmid 52 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SpeI site.
Plasmid L1 (see Fig. 5A) was made by inserting LEU2 into pBTM116 (23) carrying the LexA gene flanked by the promoter and terminator of ADH1. L2 through L8 were derived from L1 by fusing various sequences to LexA (see Fig. 5A and B). Plasmid pRS425 (24) carried the 2 µm origin and LEU2.
Strain YXB76 was MATa ura3-52 leu2-3,112 ade2-1 lys1-1 his5-2 can1-100, E-HMLα-Iinverted (20). Y2047b was MATa HMRa HMLα EΔ79–113::SUP4-o IΔ242 LEU2-GAL10-FLP1 ura3-52 ade2-1 lys1-1 his5-1 can1-100 [cir0] (25). Strain 1 was made by transforming YXB76 to Ura+ with HindIII + BamHI digested plasmid 1. Strains 2–38, 50 and 51 were similarly made with corresponding plasmids, respectively (e.g. strain 2 was made with plasmid 2). Strains 39, 40 and 42–48 were previously described as YXB26, 48, 29, 48, 28, 27, 59, 31 and 37, respectively (11). Strains 41, 52 and 53 (see Figs 3C and 6) were made by transforming Y2047b to canavanine-resistant by BamHI-digested plasmids 41, 52 and 53, respectively. Strains 40′–48′ were sir3::URA3 derivatives of strains 40–48, respectively. The relevant genotypes of all the strains were confirmed by Southern blotting.<br><br>Quantitative mating assay
Quantitative mating was performed as described (11).<br><br>Electrophoresis mobility shift assay (EMSA)
Rap1p was expressed in <span class="yellow">Escherichia coli BL21 (DE3)</span> from pL3S5, a pET vector carrying the RAP1 gene downstream of a T7 promoter (26). Crude protein extracts were prepared from 40 ml cultures harvested after 3 h of IPTG induction and resuspended in 1 ml of lysis buffer (50 mM NaH2PO4, 300 mM NaCl, pH 8.0). A control extract was prepared from a culture without IPTG induction. Western blotting revealed that Rap1p protein was present in the induced extract but not the uninduced one (data not shown). DNA fragments of 26 bp in length (see Fig. 3A) were end-labeled with 32P-phosphate. Fifteen microliter binding reactions were prepared, each consisting of 20 000 c.p.m. radiolabeled DNA probe, 5 µg of crude protein extract, non-specific competitor DNAs (0.3 µg of <span class="yellow">yeast</span> tRNA, 0.3 µg of <span class="blue">salmon</span> sperm DNA and 0.5 µg of <span class="yellow">E.coli</span> DNA), 10 µg of BSA, 10 mM Tris–HCl (pH 8.0), 10 mM MgCl2 and 8% glycerol. The reactions were incubated at 25°C for 20 min and loaded onto non-denaturing polyacrylamide gels (4–5%). Gels were run at 40 mA for 1 h in 0.5× TBE buffer.<br><br>Analysis of the supercoiling of DNA circles
Cells were grown in YPR (<span class="yellow">yeast</span> extract + bacto peptone + 2% raffinose) to early log phase. Galactose (2%) was then added to the culture to induce the expression of FLP1. After 2.5 h of incubation, nucleic acid was isolated using the glass bead method and fractionated on agarose gels with 30 µg/ml chloroquine. DNA circles were detected by Southern blotting.<br><br>
RESULTS
UASs of many highly expressed genes can function as barriers to the spread of silencing
We have previously shown that TEF2-UAS was able to block the spread of silencing in a silencer-blocking assay (11). Such an assay tested if a sequence had the ability to prevent a silencer from silencing a reporter gene when it was positioned between the silencer and the reporter. In this report we used a new silencer-blocking assay to identify more barrier elements. This assay employed the URA3 gene and an inverted HML-I that could silence sequences to the right of HML (20) (Fig. 1, strain 1). URA3 expression makes cells sensitive to 5-fluoroorotic acid (FOA) thus URA3 silencing could be measured by cell growth on FOA medium (Fig. 1, strain 1). As expected, TEF2-UAS and another known barrier element, HMR-tRNA gene (21) exhibited barrier activity in this assay as indicated by the lack of silencing of URA3 in strains 2 and 29 (Fig. 1).
TEF2 encoding translation elongation factor 1α is one of the most highly expressed genes in <span class="yellow">yeast</span> (27). Other highly expressed genes include ribosome protein genes (RPGs) and genes coding for glycolytic enzymes (27). The coordinated regulation of most of these genes requires the so-called general regulatory factors Rap1p, Abf1p and Reb1p (28,29). Using the above silencer-blocking assay, we tested if the UASs of other highly expressed genes could also block URA3 silencing (Fig. 1). Seventeen new barrier elements were identified from a total of 26 UASs tested. These 17 elements were UASs of eight RPGs (see Fig. 1, strains 3–10), eight glycolytic genes (strains 18–25) and the HIS3 gene (strain 26). All 17 elements except ADH2-UAS and HIS3-UAS exhibited very strong barrier activity as evidenced by the absence of URA3-silencing in strains 3–10 and 18–24 (Fig. 1).
Seven of the eight RPG-UASs that had barrier activity, contained a single or a pair of tandem Rap1p-binding sites (Fig. 1, strains 3–9). Three of them also contained an Abf1p or Reb1p site (strains 6, 8 and 9). One barrier RPG-UAS (strain 10) had only a predicted Abf1p site. Of the seven RPG-UASs that had no barrier activity (Fig. 1, 11–17), only two bore no site for Rap1p, Abf1p or Reb1p (11 and 12). The remaining five each contained one to three Rap1p sites (strains 13–17). RPL15A-UAS (17) also contained an Abf1p site. For most of the 15 RPG-UASs tested (3–17), the presence or absence of predicted Rap1p sites correlated with the presence or absence of Rap1p association in vivo as previously examined by chromatin immunoprecipitation assays (13). Exceptions were the UASs of RPS24B, RPS28A and RPL15A (13).
Of the eight UASs of glycolytic genes tested (Fig. 1, strains 18–25), seven exhibited strong barrier activity (18–24) and one had weaker but detectable barrier activity (strain 25). Glycolytic genes are among the most highly expressed genes and their high expression depends on the functions of Rap1p, Abf1p or Reb1p as well as Gcr1p (29,30). The seven strong glycolytic barriers all consist of one or two Rap1p sites plus one or two Reb1p or Abf1p sites (Fig. 1, strains 18–24). In addition, one or more Gcr1p-binding sites are found adjacent to each Rap1p site. It was proposed that the function of Rap1p was to facilitate the binding of Gcr1p to the UAS (15). Abf1p or Reb1p was shown to play a role similar to that of Rap1p in activating glycolytic genes (31–33).
Although the above data reinforced the notion that certain Rap1p-binding UASs could serve as barriers to silencing, they raised new questions about the essential components of a barrier element. For example, some of the UASs that contained only a single Rap1p site had barrier activity (e.g. RPL19B-UAS, strain 3) whereas others that contained two or three sites had no activity (e.g. RPL39, strain 16). One explanation was that sequences flanking the Rap1p site(s) in a particular UAS could positively or negatively influence barrier function. Moreover, the presence of Abf1p or Reb1p sites in 11 of the 17 new barriers made us wonder if they also participated in barrier function. We began to address these issues by analyzing three new barrier elements (Fig. 2). RPS10A-UAS containing an Abf1p-binding site and a pair of Rap1p sites was divided into three fragments that were tested in a silencer-blocking assay (Fig. 2, strains 30–32). It was clear that only the fragment containing Rap1p sites functioned as a barrier (strain 31). TPI1-UAS bearing a Reb1p site and a Rap1p site was divided into two parts with one containing the Rap1p site (Fig. 2, strain 34) and the other Reb1p site (strain 33). Neither of these fragments alone retained barrier activity indicating that concerted action of Reb1p and Rap1p was required for barrier activity. Duplicating the Reb1p-bearing part of TPI1-UAS didn’t restore barrier activity (strain 35). However, triplicating the Rap1p-containing part did re-create barrier function (strain 36). RPS28A-UAS bore, in addition to a Rap1p site, an Abf1p site and an adjacent T-rich region that are required for efficient transcription of RPS28A (34). A sequence of RPS28A-UAS deleted for a fragment bearing the Rap1p site lost the barrier activity (Fig. 2, strain 37). However, duplicating this sequence restored barrier activity (strain 38). The above results indicated again that Rap1p sites could form barriers (strains 31 and 36). On the other hand, they also suggested that synergistic actions of Abf1p sites (strain 38), Reb1p site + Rap1p site (strain 18), or Abf1p site + Rap1p site (strain 16) could all lead to barrier function. This was not very surprising since Rap1p, Abf1p and Reb1p could perform similar and sometimes interchangeable functions in gene regulation (16,35,36). Note, however, we haven’t ruled out the possibility that other factors (e.g. Gcr1p) or structure features of DNA (e.g. T-tracks) were also involved in barrier function.
It was interesting that ADH2-UAS and HIS3-UAS bearing no site for Rap1p, Abf1p or Reb1p exhibited detectable barrier activity (Fig. 1, strains 25 and 26). However, binding sites for other transcriptional activators existed in these UASs. The major regulator that binds to ADH2-UAS is Adr1p (37). On the other hand, Gcn4p binds to multiple sites in HIS3-UAS and activates HIS3 transcription (38). The possible involvement of Adr1p or Gcn4p in barrier function was examined below. Note Adr1p function was subject to glucose repression (37), which might explain why ADH2-UAS only exhibited limited barrier activity since glucose was used in the media used in our assay.<br><br>Effect of mutating Rap1p-binding sites in TEF2-UAS on barrier function
Although we had shown that the three Rap1p-binding sites in TEF2-UAS were necessary and sufficient for its barrier activity (11), we had not shown if binding of Rap1p to these sites was required. To address this we performed mutational analysis of TEF2-UAS. As shown in Figure 3A and B, the three Rap1p-binding sites in TEF2-UAS (designated as R1, R2 and R3) were all able to bind Rap1p in an EMSA (Fig. 3B). Single or double C→A mutations were introduced into R1, R2 and R3 resulting in m1, m2 and m3, respectively (Fig. 3A). The m2 and m3 sites had both lost the ability to bind Rap1p, whereas m1 still bound Rap1p (Fig. 3B), which could be predicted from the specific mutations in each site (Fig. 3A, compare m1, m2 and m3 to the consensus sequence). We then tested a mutated TEF2-UAS containing m1, m2 and m3 in a silencer-blocking assay using the HML-E silencer and the HMLα genes (Fig. 3C). In this assay, the HML-I silencer was deleted but the HML-E silencer was sufficient to silence the HMLα genes (Fig. 3C, strain 39) (11). Silencing in strain 39 (a-type) was measured by its ability to mate with an α-type strain (Fig. 3C). As predicted, the wild-type TEF2-UAS blocked HMLα silencing (11) (Fig. 3C, strain 40). On the other hand, the mutated TEF2-UAS containing m1, m2 and m3 had lost barrier activity (Fig. 3C, compare the mating efficiencies of strains 41 and 40). Therefore, mutations that prevented Rap1p from binding two of the three sites in TEF2-UAS abolished its barrier activity, indicating that association of Rap1p with TEF2-UAS at more than one site was required for barrier function.<br><br>The barrier TEF2-UAS alters local chromatin structure
In eukaryotic cells, the topology of local DNA reflects the chromatin structure in which it resides. We had previously developed a DNA topology-based assay to examine chromatin structure in vivo, which involved excising a chromosomal region of interest from its genomic location as a circle using site-specific recombination (39). Using this strategy, we demonstrated that DNA in silenced chromatin was more negatively supercoiled than that in active chromatin (39). In this report we used the topology assay to address if barrier elements altered local chromatin structure as <span class="yellow">Drosophila</span> or <span class="yellow">chicken</span> insulators did (40,41).
Various sequences (designated X) from UASs of TEF2, AgTEF (TEF gene from <span class="yellow">Ashbya gossypii</span>) and ADE2 were inserted between HML-E and HMLα at HMLΔI (HML locus deleted for the I silencer) flanked by the FRT sites for the Flp1p site-specific recombinase (Fig. 4A and B). The FLP1 gene was under the control of the inducible GAL10 promoter (39). The mating efficiency of each strain measured silencing of HMLα (Fig. 4C). In strains 40, 42, 44 and 47, silencing was restricted to the left of the X insertions and HMLα was derepressed (Fig. 4C). Consistently, the proportion of silent chromatin in the region flanked by the FRT sites in each of these strains was decreased, as indicated by the reduced negative supercoiling of the excised circle (11) (data not shown). Note that the above effect of a barrier on HML chromatin reflected both change in the scope of silencing caused by the barrier activity and alteration (if any) in chromatin structure at the barrier independent of the silencing state of the locus. To exclusively examine the latter, we analyzed the topology of HML DNA in a silencing-deficient background. The SIR3 gene in strains described in Figure 4B was disrupted rendering them defective in silencing (strains 40′–48′ in Fig. 4D). Recombination mediated by Flp1p in these strains led to the excision of a group of chromosomal circles that differed only in the X insertion. The supercoiling of these circles was analyzed as a function of the length of the insertion (Fig. 4D). If an insertion simply lengthened a circle without causing abnormal changes in nucleosomal structure and/or density, the topoisomers of this circle would migrate slower relative to those of a similar circle with a smaller size. Otherwise, migration of the topoisomers would deviate from that predicted based on the size of the circle. As shown in Figure 4D, the circle from strain 40′ bearing a 104 bp barrier element migrated faster rather than slower than the circle from strain 46′ bearing a 91 bp non-barrier element. Circles bearing other barrier elements also exhibited unexpected faster migration (compare strains 44′ to 45′, 47′ to 43′, 42′ to 48′, respectively). Therefore, these data indicated that barrier function correlated with altered topology of local DNA. This was further demonstrated by that mutations in TEF2-UAS that abolished its barrier activity also abolished its effect on DNA topology (Fig. 4D, compare 41′ to 40′; note that circles from both strains had the same size). Since in our gel assay more negatively supercoiled DNA migrate slower, we concluded that every fragment with barrier activity caused a reduction in negative supercoiling of approximately one to two turns in HMLΔI DNA (Fig. 4D, compare the centers of distributions of topoisomers in pairs of lanes, e.g. 40′ and 41′).
The topology of eukaryotic DNA reflects mainly the wrapping of DNA into nucleosomes (approximately one negative superhelical turn is associated with each nucleosome) (42). Therefore, removal of one nucleosome would eliminate approximately one turn. On the other hand, histone acetylation can reduce negative supercoiling associated with a nucleosome by approximately 0.2 turns (43). Therefore, acetylating five nucleosomes would reduce negative supercoiling by approximately one turn. Consequently, the linking number change of one to two brought about by TEF2-UAS can be accounted for by either the loss of one to two nucleosomes or acetylation of five to 10 nucleosomes. Further experiments are under way to clarify the nature of change in chromatin induced by a barrier.<br><br>Targeting the transcription activation domain of Rap1p recapitulates its barrier activity
The multifunctional Rap1p can be divided into at least four functional domains (Fig. 5A) (12). We were interested in defining which of these domains might be involved in the barrier activity of Rap1p. To this end, we fused various parts of Rap1p to the bacterial LexA protein and tested if targeting any of the fusion proteins to LexA-binding sites could re-create a barrier to silencing (Fig. 5A). The LexA-RAP1 fusion genes were carried on 2 µm based plasmids marked by LEU2 (designated L1–L6, Fig. 5A). When introduced into strain 50 in which two LexA-binding sites had been inserted between the inverted HML-I and URA3, LexA alone had no effect on URA3 silencing (Fig. 5A, plasmid L1). Targeting the DNA binding domain, DNA bending domain, silencing domain, or the activating + silencing domain of Rap1p also did not affect URA3 silencing (L2–L5). On the other hand, targeting the activation domain alone recapitulated the barrier activity of Rap1p (L6). These results indicated that the activation domain of Rap1p was involved in barrier activity. The inability of the Rap1p-silencing domain to block URA3 silencing in our assay did not necessarily mean that this domain had no barrier function, since such a function might be suppressed by much stronger silencing activity associated with the silencing domain.
Was the lack of barrier function of fusion constructs L2–L5 due to a lack of expression of these proteins? To address this possibility, we examined the levels of L1–L6 proteins as well as L7 and L8 proteins described in Figure 5B by western blotting using an antibody against LexA. As shown in Figure 5C, L5 was expressed at a level comparable to L6, and L4 was expressed at a higher level than L6. Therefore, the lack of barrier function of L4 and L6 was not the result of their lack of expression. On the other hand, the level of L2 or L3 was significantly lower than that of L6, hence the lack of barrier function of L2 and L3 might be due to their insufficient expression.
As an important control for the above targeting experiments, we showed that plasmids L1–L6 (Fig. 5A) as well as L7 and L8 (Fig. 5B) bearing LexA-fusion genes had no effect on URA3 silencing in strain 1 (Fig. 1) in which there was no binding site for LexA present between HML-I and URA3 (data not shown). Hence, the effect of the fusion proteins on URA3 silencing was the result of their targeting to specific loci, not the consequence of non-specific, indirect functions (if any) of these proteins.<br><br>Targeted activation domain of Adr1p or Gcn4p can also act as a barrier factor
We have shown above that ADH2-UAS bearing Adr1p-binding sites and HIS3-UAS bearing Gcn4p sites both had detectable barrier activity (Fig. 1, strains 25 and 26). Given that tethering the activation domain of Rap1p is sufficient for barrier function, we tested if targeting the activation domain of Adr1p or Gcn4p could also establish a barrier. Adr1p contains four separate activation domains (TADI–TADIV) that can contact Ada2p, Gcn5p and/or TFIIB (45). TAD III (415–467) interacts with only Gcn5p but not Ada2p or TFIIB. We fused this domain of Adr1p and the activation domain of Gcn4p (102–149) to LexA, respectively (Fig. 5B, L7 and L8). Targeting these fusion proteins to two LexA-binding sites between HML-I and URA3 in strain 50 dramatically reduced URA3 silencing (Fig. 5B, compare L7 and L8 to L1). Therefore, like LexA-Rap1p (627–695), both LexA-Adr1p (415–467) and LexA-Gcn4p (102–149) also had barrier activity. Barrier function of these fusion proteins was significantly decreased in strain 51 in which only one LexA site was inserted between HML-I and URA3 (compare strains 50 and 51 carrying plasmid L7), indicating that the potency of such barriers was dependent on the amount of tethered fusion proteins.<br><br>
DISCUSSION
We have shown in this report that UASs of many highly expressed genes could function as barriers to the spread of transcriptional silencing. These newly identified barrier elements together with previously described barrier elements (STARs, HMR-tRNA gene, TEF2-UAS and CHA1-UAS) (9–11,21) are scattered across the <span class="yellow">yeast</span> genome and may play a role in dividing the genome into functional domains. Analyses of the barrier UASs suggest that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p can participate in barrier function. Barrier activity often requires concerted actions of more than one factor-binding site. Our mutational analysis of TEF2-UAS suggests a direct involvement of Rap1p in its barrier function. It is noteworthy that not all UASs of highly expressed genes have barrier activity. Many non-barrier UASs also consist of multiple regulator binding sites, therefore other unknown factors might determine whether a UAS could act as a barrier.
How Rap1p and the other transcriptional regulators act to block silencing is not clear. Since they can all act as transcriptional activators, one natural explanation is that they may overcome the silencing effect of a silencer and directly activate the reporter gene. However, it has been well documented that Rap1p and the other transcriptional regulators usually act at relatively short distances (less than ∼600 bp upstream of the translation start codon) (28,29,46), but the binding sites for these regulators were >800 bp from the translation start codon in most of the silencing-blocking tests in this report (Figs 1 and 2). In the test shown in Figure 6, TEF2-UAS blocked ADE2 silencing when it was ∼2 kb downstream from the translation start codon of the gene. Moreover, Fourel et al. (44) demonstrated that the barrier function (or anti-silencing function) of an activator was independent of its role in activating transcription. Therefore, it is unlikely that the ability of a UAS to prevent gene silencing is due to its direct activation of the gene. In light of this, our results are in agreement with the notion that transcriptional regulators can also participate in defining the boundaries of functional chromosomal domains.
A few models have been proposed for the barrier functions of transcriptional regulators. One model proposed that binding of a regulator to a barrier led to the formation of a nucleosome-free region thereby disrupting a continuous nucleosome array (11). Since the SIR complexes spread by self-interaction and interacting with adjacent nucleosmes in a stepwise fashion (4), a nucleosome-free gap could present a barrier to the spreading SIR complex. This ‘nucleosome gap’ hypothesis is consistent with that certain barrier factors such as Rap1p and Reb1p could indeed prevent nucleosome formation around their binding sites (14) and the barrier function of TEF2-UAS correlated with its ability to alter local chromatin structure (this report). In fact, many insulator/boundary elements in higher organisms have also been shown to alter local chromatin structure. For instance, the <span class="yellow">chicken</span> β-globin insulator was linked to changes in nucleosome positioning, and the <span class="yellow">Drosophila</span> scs and scs′ boundary elements are associated with unusual chromatin structures that are hypersensitive to nucleases (40,41). However, it is not clear whether any chromatin change associated with barrier function in <span class="yellow">yeast</span> or insulator function in higher cells is the cause or effect of the function.
Another model for barrier action proposed by R. Kamakaka and colleagues (21) proposed that barrier factors recruit chromatin modifying and/or remodeling complexes to counteract the more compact, hypoacetylated silent chromatin. In agreement with this hypothesis, targeted HATs SAS2, GCN5 and ESA1 all had barrier activity (21; Y.-H. Chiu, Q. Yu and X. Bi, unpublished results). Interestingly, Rap1p, Adr1p, Gcn4p and Abf1p share the ability to recruit HAT complexes NuA4 and/or SAGA to target genes (16,45,47–49). Therefore, these factors that are capable of binding to the newly identified barriers may function as barrier factors by recruiting HAT complexes. In fact, recruitment of HAT activity has also been suggested to be the mechanism underlying the function of the <span class="yellow">chicken</span> β-globin insulator (6), although it was not known what HAT might be involved. Our demonstration that tethering the activation domain of Rap1p, Adr1p or Gcn4p alone recapitulates barrier activity is consistent with the above HAT model for barrier function, since at least the activation domain of Adr1p has been shown to directly interact with HAT complexes (45). However, we haven’t ruled out the possibility that the barrier function of the targeted activation domain of Rap1p, Adr1p or Gcn4p is not related to the barrier function of the native protein. Note that the ‘nucleosome gap’ model and the HAT model are not mutually exclusive since the nucleosome gap could result from destabilization of nucelosomes caused by HAT function. A recent intriguing study showed that targeting proteins of the nuclear pore complex (NPC) could also establish a boundary for silent chromatin, and that tethering to NPC was required for boundary activity (50). This led to a revisit to the looping model for the function of boundary/insulator elements (51). However, an alternative explanation was that the NPC compartment could simply be rich in transcriptional activating factors or poor in silencing factors like the SIR proteins (50).
In summary, results from this and other studies indicate that factors previously identified as positive and/or negative regulators of gene activation could also function in demarcating distinct domains of gene activity. How these factors carry out their boundary function is not resolved but recruitment of other factors (e.g. chromatin modifying complexes) is likely to be involved.<br><br>NOTE ADDED IN PROOF
After this work was completed and submitted for publication the first time, Fourel et al. (52) reported that targeted activation domains of Rap1p and Abf1p had insulating capacity.<br><br>
</body></html>