
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>S. cerevisiae</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="blue">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="yellow">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="yellow">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="yellow">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="yellow">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="yellow">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="yellow">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="yellow">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="blue">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="yellow">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="yellow">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="yellow">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="yellow">yeast</span> <span class="blue">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="yellow">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="yellow">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="yellow">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="yellow">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="yellow">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="yellow">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="yellow">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="blue">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="yellow">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="yellow">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="yellow">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="yellow">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="yellow">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="yellow">yeast</span> <span class="blue">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="blue">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="yellow">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="yellow">yeast</span> <span class="blue">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
</body></html>