
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>sheep</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="blue">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1805739</h3>Embryonic sympathoblasts transiently express TrkB in vivo and proliferate in response to brain-derived neurotrophic factor in vitro
Abstract
Background
Nerve growth factor and neurotrophin-3 are involved in the development of sympathetic neurons; however, whether brain derived neurotrophic factor also plays a role is not known. The purpose of this study was to determine whether BDNF and its receptor, TrkB, are expressed during the development of paravertebral sympathetic ganglia in vivo and to determine the effect of BDNF in vitro.<br><br>Results
As neural crest cells coalesce to form sympathetic ganglia, TrkB-positive cells are seen in both <span class="yellow">chicken</span> and <span class="yellow">mouse</span> embryos. In <span class="yellow">chicken</span> embryos, TrkB-expressing cells first appear at Hamburger-Hamilton Stage (St) 27 and they co-express HNK-1, confirming that they are migrating neural crest cells. The TrkB-positive cells lack neural markers at this stage; however, they migrate with other neurally differentiating cells that are TrkA and TrkC-positive. By St. 29/30, TrkB-positive cells begin to express the neural specific markers Hu C/D and Islet-1; eventually, all TrkB positive cells commence neural differentiation. By St. 34, TrkB and TrkC staining are lost. BDNF transcript expression parallels that of TrkB. In the <span class="yellow">mouse</span>, TrkB-positive cells surround newly formed sympathetic ganglia and a small number of TrkB positive cells that co-express tyrosine hydroxylase are seen within ganglia between E13.5-15. In cell culture, many cells from St. 29–30 <span class="yellow">chicken</span> lumbar sympathetic ganglia express neural markers and are dividing, indicating that they are sympathoblasts. Sympathoblasts and neurons require both nerve growth factor and neurotrophin-3 for survival. BDNF increases the number of cells expressing neural markers in culture by increasing number of cells that incorporate bromodeoxyuridine. In contrast, most TrkB-positive sympathetic cells in vivo are not actively proliferating between E6–E8.<br><br>Conclusion
Developing paravertebral sympathetic ganglia in avian and <span class="yellow">murine</span> embryos contain a subpopulation of sympathoblasts that transiently express TrkB and ultimately commence neuronal differentiation. These TrkB expressing sympathoblasts are not actively dividing in vivo; yet, when placed in vitro, will divide in response to BDNF. This suggests that the availability of BDNF in vivo fails to reach a threshold necessary to induce proliferation. We suggest that excess TrkB stimulation of sympathoblasts in vivo may lead to the genesis of neuroblastoma.<br><br><br><br>Background
Neural crest cells destined to become paravertebral sympathetic neurons proliferate and differentiate during migration and gangliogenesis. In <span class="yellow">chicken</span> embryos, migrating neural crest cells express catecholamines at Hamburger/Hamilton Stage (St.) 19, and these cells form the primary sympathetic chain dorsolateral to the aorta at St. 22 (E3.5) [1]. Between St. 23 (E4) and St. 28 (E6), these cells disperse and undergo a secondary migration to form the paravertebral sympathetic chain that resides ventral to the spinal cord and dorsal root ganglion [1]. After ganglia coalesce, sympathoblasts express markers of neuronal differentiation, such as Q211 and tyrosine hydroxylase (TH), at a time when they also incorporate [3H]-thymidine [2]. Time lapse photography has shown that cultured E15.5–E16.5 sympathetic neurons from <span class="yellow">rat</span> embryos extend axons while they divide [3-5]. Although proliferation appears to be an important process to expand the sympathetic neuron population during differentiation, the mechanisms that guide sympathoblast proliferation have not been identified.
The development of sympathetic neurons is guided by neurotrophins. Neurotrophin-3 (NT-3) binds to its receptor, TrkC, to promote the survival of cultured sympathoblasts from early lumbar paravertebral ganglia [6]. Nerve growth factor (NGF) signals through its receptor, TrkA, to promote the survival of sympathetic neurons upon target innervation [7]. There are severe sympathetic defects in the superior cervical ganglion of individual NT-3 and NGF knockout <span class="yellow">mice</span> [8-10]. Furthermore, there is no additional cell death in the superior cervical ganglion of NT-3 and NGF double knockout <span class="yellow">mouse</span> embryos, suggesting that all of the neurons are dependent on both neurotrophins for survival [11]. There is also an increase in sympathetic neuron cell death in TrkA knockout <span class="yellow">mice</span> [12]. However, in TrkB and BDNF knockout <span class="yellow">mice</span>, there is no apparent phenotype in the superior cervical ganglion, and there is little evidence that TrkB or BDNF is expressed in sympathetic ganglia. Thus, it is generally thought that TrkB and BDNF have little or no roles in guiding the development of sympathetic neurons.
In addition to their developmental functions, neurotrophin receptors regulate cell behavior in neuroblastoma, a tumor found in sympathetic ganglia and adrenal medulla. Tumors that express TrkA often spontaneously regress, while those that express TrkB and its ligand, brain-derived neurotrophic factor (BDNF), grow aggressively, are invasive, and fail to respond to chemotherapeutic agents [13]. The presence of TrkA in neuroblastoma tumors is consistent with its expression in developing sympathetic neurons, and suggests that regressive neuroblastoma tumors arise from early sympathetic neurons that express TrkA. The function of TrkB in early sympathetic development is unknown, which makes understanding the etiology of aggressive neuroblastoma tumors difficult. Based on its function in neuroblastoma tumors, we hypothesize that BDNF and TrkB expression in differentiating sympathoblasts is responsible for expanding the neuronal population through proliferation.
We sought to determine whether BDNF and TrkB are involved in sympathetic development. We report that during early embryonic development, TrkB is expressed in a subset of differentiating sympathoblasts in both avian and <span class="yellow">murine</span> embryos. We also find that BDNF promotes the proliferation of TrkB-positive sympathoblasts in cell culture. However, the majority of TrkB positive cells in vivo fail to take up bromodeoxyuridine (BrdU) over a 24 hr period, suggesting that endogenous BDNF concentrations do not reach a threshold necessary to stimulate proliferation of sympathoblasts. Shortly after all of the TrkB positive cells commence neuronal differentiation, TrkB immunoreactivity is lost. These results suggest that prolonged expression and/or activation of TrkB signaling at these early stages may be an early event triggering the formation of neuroblastoma.<br><br>Results
TrkB is expressed during migration of neural crest cells to sympathetic ganglia
We first determined whether TrkB is expressed in neural crest-derived cells in the region ventral to the spinal cord and dorsal root ganglia where sympathetic ganglia coalesce between Hamburger/Hamilton Stages (St.) 25–28/29. To identify cells that have commenced neuronal differentiation, transverse sections of the lumbar spinal column region were stained with antibodies against Hu C/D [14], a neuronal-specific RNA-binding protein, or Islet-1, a transcription factor found in sympathetic neurons [15]. We found that Hu C/D and Islet-1 are expressed in the same cells both in vivo and in vitro throughout sympathetic development. In experiments done between St. 25 and 28, Islet-1 staining appeared weaker than Hu C/D staining, and thus we used Hu C/D to identify differentiating neurons at these stages. At later stages, Islet-1 was used to facilitate the identification of neurons because of the nuclear location of its immunoreactivity.
Cells expressing Hu C/D are first detected at St. 25 ventral to the spinal cord and dorsal root ganglion and lateral to the dorsal aorta (Figure 1A, 1B). By St. 26, the number of cells that express Hu C/D in this region increases dramatically (Figure 1C). TrkB-expressing cells first appear at St. 27 in the same region and are adjacent to Hu C/D-positive cells (Figure 1D, 2A, 2H). TrkB-positive cells co-localize with a neural crest marker, HNK-1 (Figures 2B, 2C, 2D). The Hu C/D-positive cells in this region are likely to be sympathetic neurons, since they appear in the region where sympathetic ganglia form and express tyrosine hydroxylase, a rate-limiting enzyme in the synthesis of catecholamines (Figures 2E, 2F, 2G). At St. 28/29, the cells begin to coalesce ventral to the dorsal root ganglion and the Hu C/D-positive cells and TrkB-positive cells remain as two separate cell populations (Figure 1E); however, shortly afterwards, all of the TrkB-positive cells begin to express Islet-1 (Figure 3B) and Hu C/D (data not shown).<br><br>Developmental regulation of TrkA, TrkB, TrkC, and BDNF expression
In contrast to TrkB, the other neurotrophin receptors, TrkA and TrkC, are co-expressed in both Hu C/D-positive and Hu C/D-negative cells at St. 27 (Figures 2I, 2J). We find that approximately 30% of the Islet-1-positive cells express TrkA (Figure 3A), while 50% express TrkB (Figure 3B) and 100% express TrkC (Figure 3C) at St. 29/30 (E6.5). Thus, all developing neurons express TrkC in combination with either TrkA or TrkB. By St. 31 (E7), the number of TrkA-positive, Islet-1-positive cells increases to 100% (Figure 3D) and immunoreactivities for both TrkB and TrkC appear dispersed (Figure 3E, 3F). By St. 34 (E8), TrkA expression is well-sustained (Figure 3G) and TrkB and TrkC immunoreactivities are lost (Figure 3H, 3I). We also examined the early development of <span class="yellow">murine</span> sympathetic ganglia (Figure 4). At E13, the newly formed lumbar sympathetic ganglia can be observed ventral to the spinal cord and notochord by their staining for Hu C/D and TH (Figure 4A). TrkB-positive cells can be seen surrounding developing ganglia, as well as in occasional cells within the ganglia (Figure 4C, D). These TrkB-positive cells within the ganglia co-express TH and are seen at a frequency of 1–2 cells per section starting at E13 (Figure 4A–D) and are still present at E15.5 (data not shown).
In neuroblastoma cells, BDNF is co-expressed with TrkB, suggesting that autocrine stimulation is a means by which proliferation is sustained in the transformed cells. To test whether BDNF, the ligand for TrkB, was present in embryonic <span class="yellow">chick</span> sympathetic ganglia, we used quantitative real-time PCR with TaqMan probes to determine the relative abundance of BDNF transcripts in total RNA extracted from lumbar sympathetic ganglia at St. 29/30 (E6.5), St. 31 (E7), St. 34 (E8), and E9. BDNF expression within the ganglia parallels that of TrkB: BDNF mRNA expression levels are highest at St 29/30 (E6.5), and these levels decrease 2-fold at St. 31 (E7) and St. 34 (E8; Figure 5). By E9, BDNF levels are 7 times lower than at St. 29/30 (E6.5; Figure 5).<br><br>NT3 and NGF promote survival of differentiating sympathetic neurons in culture
To determine the effect of neurotrophins, we cultured cells dispersed from lumbar sympathetic ganglia at St. 29/30 (E6.5) because, at this stage, ganglion formation is complete, the number of TrkA-, TrkB-, and TrkC-positive cells have peaked, and all Trk-expressing cells have initiated neural differentiation. First, we identified markers expressed by acutely isolated cells. As shown in Table 1, 80–91% of the cells are p75 neurotrophin receptor (NTR)-positive, indicating that most of the cells are neural crest-derived and little mesenchymal contamination is introduced by the isolation procedure. In addition, 28–33% of the cultured cells express the neural marker Hu C/D. Approximately half of these Hu C/D-positive cells express TrkB. Conversely, all of the TrkB positive cells express Hu C/D. These TrkB-positive cells comprise approximately 14–17% of the total cell population.
We then determined how many of the acutely isolated cells were proliferating by incubating them for 12 hrs in BrdU-containing medium. For these experiments, we identified differentiating neurons with the transcription factor Islet-1 because this marker labels nuclei, thus it co localizes with any BrdU that has been incorporated into the DNA, allowing us to determine whether the cell had undergone S-phase of the cell cycle. After 12 hrs in BrdU, 59% of Islet-1-positive nuclei stain for BrdU immunoreactivity. Thus, cultures of St. 29/30 sympathetic ganglia contain many cells that proliferate while exhibiting markers of neuronal differentiation, confirming previous observations [2]. We call these dividing neuronal precursors sympathoblasts. The remaining non-BrdU incorporating, Islet-1 positive cells are likely to be post mitotic neurons.
Finally, we determined the trophic requirements of St. 29/30 (E6.5) sympathetic neurons and sympathoblasts. We monitored cultures over a three day period after plating and counted the number of phase bright cells with neurites, a morphological feature of both neurons and sympathoblasts. In the absence of trophic factors, more than 2/3 of the cells die by 24 hours in culture and BDNF, NT-3, or NGF alone is not sufficient to promote survival (Figure 6). However, NGF together with NT-3 supports the survival of a significantly larger number of cells (Figure 6). For the subsequent experiments, all neurons were cultured with 25 ng/ml NT-3 and 1 μg/ml 7S NGF to optimize survival.<br><br>BDNF promotes proliferation of TrkB-positive sympathetic neurons in culture
To determine the effects of BDNF, cultures of cells from St. 29/30 (E6.5) sympathetic ganglia were supplemented with 200 ng/ml BDNF and the number of neurons and sympathoblasts were counted at 24, 48, and 72 hours using phase microscopy (Figure 7A). A 1.6-fold increase in the number of neurons due to BDNF is observed by 24 hours and this number does not increase further at 48 or 72 hours. This effect of BDNF is concentration-dependent with an EC50 of 75 ng/ml (Figure 7B).
To test whether the increase in the number of neurons and sympathoblasts caused by BDNF is due to the differentiation of pluripotent neural crest cells, we quantified the effects of BDNF on the number of neurally differentiating cells (Hu C/D-positive) versus the number of non-neuronal cells (Hu C/D-negative) after identifying all neural crest-derived cells by staining for p75NTR in St. 29/30 (E6.5) cultures. If BDNF increases the number of neurons and sympathoblasts by inducing a non-neuronal cell to express Hu C/D, then we expected that the total cell number would remain the same and that there would be a decrease in the number of non-neuronal cells as well as a corresponding increase in the number of neurons. After 24 hours, BDNF significantly increases the number of p75NTR-positive cells as well as the number of Hu C/D-positive cells (Figure 8A). However, there was no statistically significant change in the number of non-neuronal cells. Thus, it is unlikely that BDNF increases the number of neurons and sympathoblasts by inducing differentiation of non-neuronal cells.
To determine whether the increase in the total number of neurons and sympathoblasts is caused by BDNF-induced proliferation, control and BDNF-treated sympathetic cultures were exposed to BrdU for 12 hours after plating, and the number of cells that incorporated BrdU into their DNA was determined after 24 hours in culture. Even in the control condition, a number of cells in the culture are dividing, giving a high baseline of BrdU incorporation (Figure 8B). When BDNF is added, the total number of BrdU positive cells increases approximately 1.6-fold (Figure 8B). This BDNF-induced increase in the total number of BrdU positive cells occurs in sympathoblasts because the number of Islet-1-positive nuclei from control cultures that label with BrdU is 268 ± 59 and BDNF treatment raises this number to 424 ± 80, which corresponds to an increase of 1.6-fold. This accounts for the 1.6-fold increase in total neuron number and total BrdU-positive cells described above. We then confirmed that BDNF acts on TrkB-positive cells: BDNF increases the number of TrkB-expressing cells that incorporate BrdU 2.6 – 4-fold over control (Table 2) and it also increases the overall number of TrkB-positive cells 2 – 2.5-fold over control (Table 2). BDNF does not increase the number of BrdU-positive, TrkB-negative cells or the overall number of TrkB-negative cells (Table 2). In further support that BDNF acts directly on TrkB-expressing cells, an antibody directed against the extracellular domain of TrkB completely prevents the effect of BDNF in promoting proliferation of TrkB-positive, but not TrkB-negative cells (compare Figure 9A to 9B). Thus, the effect of BDNF is restricted to the population that expresses TrkB, which are developing sympathoblasts.
To determine whether TrkB-positive cells are actively proliferating in vivo, embryos were injected with BrdU at St. 27 and harvested at St. 29, approximately 24 hrs later. The majority (85–90%) of TrkB-positive cells do not incorporate BrdU into their nuclei under basal conditions in vivo (Figure 10), although a few TrkB positive cells with labeled nuclei could be observed (arrows). This contrasts with our observation that 71–76% of TrkB-positive cells incorporate BrdU in culture after treatment with BDNF (Table 2), suggesting that endogenous BDNF does not achieve a threshold sufficient to support a high level of sympathoblast proliferation in vivo.<br><br>
Discussion
We report that the neurotrophin receptor TrkB is expressed in a subset of embryonic sympathoblasts during the early development of lumbar paravertebral sympathetic ganglia in <span class="yellow">chicken</span> and <span class="yellow">mouse</span> embryos. In the <span class="yellow">chicken</span>, TrkB expression is transient, and completely lost by St 34 (E8). Since BDNF induces the proliferation of sympathoblasts in cell culture, yet in vivo there is little proliferation observed in TrkB-positive cells in nascent ganglia, we propose that if TrkB activation becomes unregulated by excess BDNF or constitutive phosphorylation of TrkB [16], this transient population of TrkB-positive sympathoblasts may trigger the genesis of neuroblastoma, a childhood tumor found in the paravertebral chain and adrenal medulla.
The two populations of sympathoblasts that we observe support previous findings of heterogeneity among developing sympathetic neurons and neural crest cells. Early sympathetic ganglia contain at least two subpopulations: early differentiating neurons that lack TrkB expression and express TrkA and TrkC, and late differentiating sympathoblasts that express TrkB. Explant cultures of sympathetic ganglia from E16 <span class="yellow">chick</span> embryos give rise to two neuronal populations: one that remains close to the explant, and one that migrates away from the explant [1]. In addition, early neuronal subpopulations have been observed in cultures of neural crest cells from St. 13/14 quail embryos as evidenced by the expression of neuronal cell type-specific gangliosides [17]. Perhaps these different subpopulations will ultimately give rise to the two neurochemically distinct populations found in lumbar sympathetic ganglia: the noradrenergic, NPY-containing neurons that innervate internal organs and enteric ganglia and the cholinergic, VIP-containing neurons that innervate vasculature in the hind limbs.
The effects of BDNF and TrkB deletion and over expression have been studied on superior cervical ganglion and preganglionic neurons in thoracic segments of the spinal column, but not on paravertebral sympathetic neurons. In the superior cervical ganglion, an increase in the number of neurons of BDNF null <span class="yellow">mice</span> is likely due to apoptosis induced by BDNF via p75NTR [18]. In contrast, the responses of paravertebral sympathetic neurons to BDNF are complex and subtype dependent. Over expression of BDNF leads to an increase in the number of noradrenergic fibers innervating the erector pilli muscles of hair follicles, while noradrenergic fibers innervating blood vessels were unaffected [19]. If our results indicating that BDNF promotes proliferation of TrkB-positive sympathoblasts in the <span class="yellow">chicken</span> embryo can be extrapolated to the subset of TrkB-positive sympathoblasts in <span class="yellow">murine</span> ganglia, then these TrkB-positive cells may be neurons destined to innervate the erector pilli. In other studies, TrkB null <span class="yellow">mice</span> showed no changes in morphology or cell number in superior cervical ganglia [12] or in the intermediolateral column [20]; but this may not be predictive of a phenotype in the lumbar paravertebral chain. It is thus possible that BDNF/TrkB signaling could play a specific role in other regions of the paravertebral sympathetic chain, such as the lumbar region. However, if TrkB-positive cells are not normally actively proliferating in vivo, then it would not be surprising that the development of the paravertebral sympathetic chain is not disrupted in TrkB or BDNF null <span class="yellow">mice</span>. It may be more informative to examine <span class="yellow">mice</span> that over express BDNF on a promoter that targets expression to embryonic lumbar ganglia. Unfortunately, such <span class="yellow">mice</span> do not exist.
Our findings that the St. 29/30 (E6.5) sympathoblasts are dependent on both NT-3 and NGF for survival in culture are consistent with previous work on <span class="yellow">mouse</span> sympathoblasts from the superior cervical ganglion [11]. In these studies, NT-3 and NGF deletion separately led to a decrease in the number of sympathetic neurons at E17.5 compared to control. Deletion of both NT-3 and NGF together did not enhance cell death. In contrast, cultured <span class="yellow">rat</span> superior cervical ganglion sympathetic neurons respond to NT-3 at E14.5 and then to NGF at E19.5, although time points in between were not analyzed [6].
In addition to promoting survival, NT-3, NGF, and BDNF also induce proliferation of various neuronal precursors at different stages of development. NT-3 can promote the incorporation of [3H]-thymidine into cultured quail neural crest cells from the trunk region [21,22], Later in <span class="yellow">rat</span> sympathetic development, NT-3 supports survival of neurons, but does not promote proliferation [6], which is consistent with our results. NGF promotes an increase in BrdU incorporation from 25% to 35% in the DRG cervical segment 2 in the <span class="yellow">chick</span> embryo [23]. In <span class="yellow">chicken</span> embryos that are treated with NGF in ovo at St. 18 and 21, there is an increase in BrdU uptake after formation of the primary sympathetic chain at St. 23 [24]. Since NGF does not appear to affect proliferation of St. 29/30 (E6.5) <span class="yellow">chick</span> sympathoblasts, NGF may only promote proliferation in primary, but not secondary chain sympathoblasts. Motor neuron progenitors in the ventral neural tube from the <span class="yellow">chick</span> embryo express TrkB and when ventral neural tube explants are treated with BDNF, there is an increase in the number of motor neurons produced and BrdU incorporation [25]. BDNF also promotes the proliferation of cultured neuroblastoma cells [13]. Taken together, these results are consistent with our findings that NT-3 and NGF do not promote proliferation of St. 29/30 (E6.5) sympathoblasts, and support the assertion that BDNF promotes proliferation of TrkB-positive sympathoblasts in culture.
Our observations suggest a transient function of TrkB during early sympathetic development in supporting proliferation of this early subpopulation of sympathoblasts. However, the in vivo labeling suggests that only a minority (10–20%) of this population is dividing during the window that TrkB is expressed. In light of the very strong proliferative effect produced in cell culture, these TrkB expressing cells could respond more strongly if endogenous BDNF rises to higher levels, or if the mechanism that down regulates TrkB expression becomes nonfunctional. Such events could trigger an early proliferative event that leads to a cascade of changes that initiates transformation of cells to neuroblastoma. Thus, these early TrkB expressing cells help solve the puzzle as to why TrkB is expressed in aggressive and invasive forms of neuroblastoma, particularly because BDNF induces cultured neuroblastoma cells to become more proliferative, invasive, angiogenic, and resistant to chemotherapeutic reagents than untreated cultures [13]. Future studies will determine whether constitutive expression of BDNF and TrkB in the <span class="yellow">chick</span> embryo sustains proliferation of differentiating sympathoblasts.<br><br>Conclusion
We have identified a time point during development when differentiating lumbar sympathetic neurons transiently express TrkB and proliferate in response to high concentrations of BDNF in culture. These studies suggest that elevated BDNF expression above basal levels and signaling through TrkB may be a mechanism that contributes to the onset of neuroblastoma. A further understanding of the two populations of sympathetic neurons and the fate of the TrkB-positive cells will provide additional insight into the development of paravertebral sympathetic ganglia and the genesis of neuroblastoma.<br><br>Methods
Preparation of tissue for immunohistochemistry
The lumbar spinal column and surrounding tissues were dissected from <span class="yellow">chicken</span> embryos at the indicated stages and placed in Zamboni's fixative (4% (w/v) paraformaldehyde, 15% (v/v) picric acid in 0.1 M sodium phosphate buffer, pH 7.4) for two hours at room temperature. <span class="yellow">Mouse</span> embryos at 13–15 days post-coitus were collected according to an IACUC-approved protocol to Dr. L. Sherman at the Oregon Health and Science University. The <span class="yellow">mouse</span> embryos were immersion-fixed in Zamboni's fixative overnight at 4 degrees C then washed with phosphate buffered saline (PBS; 130 mM NaCl, 20 mM sodium phosphate buffer, pH 7.4). Fixed tissues were equilibrated in 30% sucrose in 1× phosphate-buffered saline (PBS). Fixed <span class="yellow">mouse</span> embryos were shipped to Vermont in sucrose. Transverse 30 μM sections of the spinal columns were cut at on a Microm HM cryostat (knife temperature: 16°C; object temperature: 23°C) and collected on Superfrost Plus slides (Fisher). Sections were dried at room temperature, washed in 1× PBS and incubated overnight in blocking buffer (1× PBS consisting of 10% (v/v) heat-inactivated <span class="yellow">horse</span> serum (Invitrogen/Gibco), 0.5% Triton X-100 (Sigma), and 0.1% sodium azide (Fisher)).<br><br>Immunohistochemistry
Sections were incubated with primary antibodies overnight at 4°C, followed by incubation with secondary antibodies for 2 hours at room temperature. Primary antibodies used were: <span class="yellow">rabbit</span> anti-p75 (1:1500, generous gift from Louis Reichardt, UCSF [26]), <span class="yellow">mouse</span> IgG2b anti-Hu C/D, (1:250, Molecular Probes); <span class="yellow">mouse</span> IgG1 anti-Islet-1, (1:10, Developmental Studies Hybridoma Bank); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkA (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkB (1:500); <span class="yellow">rabbit</span> anti-<span class="yellow">chicken</span> TrkC (1:500) (all Trk antibodies were generous gifts of Dr. Louis Reichardt, UCSF [26-28]); <span class="yellow">mouse</span> anti-HNK-1 (1:50, Developmental Studies Hybridoma Bank); <span class="yellow">mouse</span> IgG2a anti-tyrosine hydroxylase (1:10, Developmental Studies Hybridoma Bank), <span class="blue">sheep</span> anti-BrdU (1:100, Biodesign International), <span class="yellow">rabbit</span> anti-tyrosine hydroxylase (1:100, Chemicon), and <span class="yellow">goat</span> anti-TrkB (1:1000, R&D Systems). Immunofluorescence was imaged using a Nikon C1 confocal mounted on a Nikon Eclipse E800 microscope with a 10× Plan Apo (NA 0.785) air objective or a 60× Plan Apo (NA 1.4) oil objective lens, E7-C1 software, and UV, Argon, and He/Ne lasers exciting at 408, 488, and 543 nm and emitting at 404 500–530, and 555–615 nm, respectively. A Nikon Eclipse E800 microscope in the nearby COBRE Molecular/Cellular Core Facility was used for counting immunofluorescent cells at 200× using epifluorescence optics.<br><br>RNA Extraction/cDNA synthesis
Sympathetic ganglia were removed from <span class="yellow">chick</span> embryos and RNA was isolated using TriReagent (Molecular Research Center), an acidified guanidinium with phenol extraction method [29]. RNA was transcribed to cDNA using oligo-dT with Superscript II Reverse Transcriptase (Invitrogen) at 42°C for 1 hour.<br><br>Real-time PCR
Relative RNA levels were determined using quantitative real-time PCR with an ABI 7500 Fast Real Time PCR System. TaqMan probes were used to quantify the progression of the PCR reaction and reactions were normalized using the constitutively expressed gene <span class="yellow">chick</span> ribosomal binding protein s17 (CHRPS). The sequences were used for primer/probes sets: for BDNF: forward: 5'-AGCCCAGTGAGGAAAACAAG-3', reverse: 5'-ACTCCTCGAGCAGAAAGAGC-3', probe: 5'-[6-FAM]-TACACATCCCGAGTCATGCTGAGCA-[BHQ]-3'; for CHRPS (<span class="yellow">chick</span> ribosomal binding protein S-17): 5'AACGACTTCCACACCAACAA3', reverse: 5'CTTCATCAGGTGGGTGACAT3', probe: 5'-[6-FAM]-CGCCATCATCCCCAGCAAGA [BHQ]-3'. Primers and probes were synthesized by Operon Technologies, Inc (Alameda, CA). The primers for BDNF were validated against primers for CHRPS according to an Applied BioSystems protocol by serially diluting the target cDNA 1:10, determining the cycle threshold (Ct) for each reaction, and plotting the Ct versus log concentration. Slopes of the resulting lines were calculated and primers were accepted if their Ct slopes were between -3.2 and -3.4 (a perfect efficiency of 1.0 yields a slope of -3.3). To analyze the data, the delta Ct method of relative quantification was used, where the Ct of Chrps was subtracted from the Ct of the gene of interest (Delta Ct) and the arbitrary units of mRNA were expressed as 10000/2^(Delta Ct).<br><br>Cell culture
Sympathetic neurons were cultured as previously described [30] with a few modifications. Sympathetic ganglia were removed from the lumbar region of the paravertebral chain of St. 29/30 (E6.5) <span class="yellow">chick</span> embryos and placed in Modified Puck's solution with glucose (MPG). The cells were dissociated by incubation of sympathetic ganglia with 0.1% trypsin in MPG at 37°C for 10 minutes followed by triturating with a fire polished 9" Pasteur pipette. Cells were then resuspended in Dulbecco's Modified Eagle Medium (DMEM) consisting of 10% <span class="yellow">horse</span> serum, 2% fetal <span class="yellow">calf</span> serum, and 10 mg/ml penicillin/streptomycin. For neurotrophin studies, the culture medium was supplemented with 25 ng/ml NT-3 (R & D Systems) and 1 μg/ml 7S NGF (Alomone Labs) upon plating, and 50 ng/ml, 100 ng/ml, or 200 ng/ml BDNF (R & D Systems) once the cells adhered to the wells. Cells were plated on poly-D-lysine/laminin coated wells or cover slips (Fisher) as previously described [30].<br><br>Quantification of neurons and sympathoblasts using phase microscopy
Embryonic sympathoblasts and neurons are small, phase bright cells with neurites. The total number of cells with neurites the length of two cell bodies were counted in 10 non-overlapping fields of view evenly spaced in a grid-like pattern across the bottom of a well from a 24 well plate at 200× using a Nikon Eclipse TE200 microscope.<br><br>BrdU labeling
For in vitro studies, approximately 2 hours after plating cells from St. 29/30 (E6.5) sympathetic ganglia, cells were labeled with 10 μM bromodeoxyuridine (BrdU, Sigma) for 12 hours at 37°C. Following this labeling period, cells were incubated in complete medium without BrdU for an additional 10 hrs. Cells were then fixed in Zamboni's fixative for 30 min at room temperature and rinsed with 1× PBS. For in vivo studies, 25 μg BrdU was injected into the amnion of <span class="yellow">chick</span> embryos at St. 27. The cells and sections were denatured with 2 N HCl at 37°C for 1 hr, and were then neutralized with 0.1 M borate buffer, pH 8.5, for 10 min at room temperature. Immunochemistry was performed as described above.<br><br>
Abbreviations
BDNF, brain-derived neurotrophic factor; BrdU, Bromodeoxyuridine; DA, dorsal aorta; DMEM, Dulbecco's Modified Eagle's Medium; DRG, dorsal root ganglion; E, embryonic day; HS, <span class="yellow">horse</span> serum; MPG, Modified Puck's solution with glucose; NGF, nerve growth factor; NC, notochord; NT, neural tube; NT-3, neurotrophin-3; NTR, neurotrophin receptor; PBS, phosphate-buffered saline; SC, spinal cord; SCG, superior cervical ganglion; SEM, standard error of the mean; SG, sympathetic ganglion; St., stage; w/v, weight/volume; v/v, volume/volume.<br><br>Authors' contributions
JAS designed the experiments, performed the experiments, analyzed the data, and wrote the manuscript. GLSS contributed intellectually to the conception and design of this study, and assisted in the interpretation of the results. RN supervised the study, participated in the design of experiments, edited the manuscript, and obtained funding for the project. All authors read and approved the final manuscript.<br><br>
<h3>pmcA324075</h3>Structural organization of a 17 KB segment of the alpha 2 collagen gene: evaluation by R loop mapping.
Abstract
A recombinant phage, SpC3, containing a 17 kb genomic DNA insert representing approximately 60% of the 3' portion of the <span class="blue">sheep</span> collagen alpha 2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 alpha 2 coding sequences (exons) were mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region required for coding the interspersed 3 kb of alpha 2 collagen genetic information was 5.6 fold longer than the corresponding alpha 2 mRNA coding sequences.Images<br><br><br><br><br><br>
 Nucleic Acids Research <br><br> Structural organization of a 17 KB segment of the a2 collagen gene: evaluation by R loop mapping Millie P.Schafer, Charles D.Boyd, Paul Tolstoshev and Ronald G.Crystal <br><br> Pulmonary Branch, National Heart, Lung and Blood Institute, National Institutes of Health, Bethesda, MD 20205, USA <br><br> Received 13 February 1980 <br><br> ABSTRACT <br><br> A recombinant phage, SpC3, containing a 17 kb genomic DNA insert rep<br><br> resenting approximately 60% of the 3' portion of the <span class="blue">sheep</span> collagen a2 gene, was evaluated by electron microscopic R loop analysis. A minimum of 17 intervening sequences (introns) and 18 a2 coding sequences (exons) were <br><br> mapped. With the exception of the 850 base pair exon located at the extreme 3' end of the insert, all exons contained 250 base pairs or less. The total length of all the exons in SpC3 was 3,014 base pairs. The length distribution of the 17 introns ranged from 300 to 1600 base pairs; together, all of the introns comprised 14,070 base pairs of SpC3 DNA. Thus, the DNA region <br><br> required for coding the interspersed 3 kb of a2 collagen genetic information was 5.6 fold longer than the corresponding a2 mRNA coding sequences. <br><br> INTRODUCTION <br><br> Type I collagen, composed of two al(I) and one a2 polypeptide chains, is the most abundant of the five known mammalian collagen types. It is a major extracellular constituent of tissues such as bone, tendon, skin and lung where, because of its great tensile strength, it plays an important <br><br> role in tissue structure and function. The polypeptides comprising type I <br><br> collagen are synthesized in long, precursor forms, referred to as pro al(I) and pro a2 chains,1 each containing approximately 1500 amino acid residues (1). <br><br> 1 The terminology for the primary translation products of the various <br><br> collagen messenger RNAs is still in a state of flux. It is known that al(I) and a2 chains are synthesized in precursor forms; these are currently termed pro al(I) and pro a2 chains (1). However, recent studies (43,44) have shown that the actual primary translation product of al(I) mRNA is somewhat larger <br><br> than the pro al(I) chain; this translation product has been termed a "pre-pro al(I) chain." Although the corresponding "pre-pro a2 chain" likely exists, it has not yet been characterized. For simplicity, therefore, until the <br><br> proper terminology for the primary translation product is clarified, we have <br><br> chosen to use the terms "al(I)" and "a2" to refer to the type I collagen mRNAs and collagen structural gene sequences. <br><br> ?) IRL Press Umited, 1 Falconberg Court, London W1V 5FG, U.K. <br><br> Volurne 8 Number 10 1980 <br><br> 2241 <br><br> Nucleic Acids Research <br><br> There is increasing evidence that the quantities of al(I) and a2 <br><br> chains produced by cells may be regulated, in part, at the genomic level <br><br> (1). However, to understand the processes which mediate their expression, <br><br> it is necessary to understand the structural organization of these collagen genes and the various enzymes that modify the collagen chains after its <br><br> synthesis (1). As an initial approach to this problem, we have recently <br><br> isolated a recombinant <span class="yellow">bacteriophage</span>, termed SpC3, containing approximately 60% of the a2 gene for type I collagen. Partial characterization of this <br><br> a2 gene demonstrated that, like most other structurally evaluated eukaryotic genes, the coding sequences are interspersed with intervening sequences (2). The present study further characterizes this portion of the a2 <br><br> gene by utilizing electron microscopic R loop analysis to map the coding regions (exons) and intervening sequences (introns) contained within the <span class="blue">sheep</span> genomic insert of SpC3. The data suggests that the structural <br><br> organization of this portion of the a2 collagen gene represents 3 kb of coding information that is interspersed in a complex fashion with 17 introns over 17 kb of genome. <br><br> MATERIALS AND METHODS <br><br> Isolation and characterization of a2 collagen recombinant clone, SpC3. <br><br> The <span class="blue">sheep</span> a2 collagen recombinant clone, SpC3, containing a portion of the <span class="blue">sheep</span> a2 gene, was isolated as previously described (2). Briefly, <br><br> high molecular weight fetal <span class="blue">sheep</span> liver DNA was extracted by the method of Blin and Stafford (3), and 15-20 kb DNA fragments, resulting from partial Eco RI digestion, were isolated and ligated to the left and right arms of Charon 4A. The recombinant DNA was then packaged in vitro and the <span class="blue">sheep</span> <br><br> genomic library amplified (4-6). Screening of the <span class="blue">sheep</span> genomic library was <br><br> conducted utilizing [32 P]-labeled fetal <span class="blue">sheep</span> tendon type I collagen cDNA and <br><br> several positive recombinant phages were isolated. One recombinant, containing a 17 kb <span class="blue">sheep</span> genomic insert, was demonstrated to have coding sequences corresponding to 60% of the 3' end of a2 mRNA. Prior studies showed that the insert in SpC3 was not due to ligation of noncontiguous restriction fragments and/or genetic rearrangement during the cloning process (2). Electron mi croscopy. <br><br> To visualize the recombinant <span class="yellow">bacteriophage</span> DNA structure complementary to a2 collagen mRNA, 1 iig of SpC3 DNA was lyophilized with 1 ig or 100 ng of mRNA <br><br> to give DNA:mRNA ratios of 1:1 and 10:1, respectively. These were then dissolved in 10 i1 of R loop buffer [70% (v/v) deionized formamide, 0.1 N-[tris (hydroxy<br><br> 2242 <br><br> Nucleic Acids Research <br><br> methyl)methyl] glycine (tricine)-NaOH, pH 8.0, 0.5 M NaCl, and 0.01 M ethyl<br><br> enediaminetetraacetate (EDTA)], incubated in sealed capillary tubes at 520 for 14 hr (7), and then diluted 1:10 with R loop buffer. Two spreading methods were used to subsequently prepare the hybrids for visualization in the electron microscope. <br><br> 70% formamide method. Immediately before spreading onto a deionized <br><br> water hypophase, the nucleic acid mixtures were diluted an additional 10-fold in R loop buffer and cytochrome c, 100 vg/ml, was added. <br><br> Urea-formamide method.   The nucleic acid solutions were spread using a modification of the method of Westphal and Lai (8,9). The hyperphase contained 55% formamide, 2.6 M urea, 0.009 M EDTA, 0.09 M tricine-NaOH, pH 8.0, and the nucleic acid mixture (approximately 0.1 to 1.0 ig/ml). <br><br> The mixture was heated at 40? for 30 sec, placed in ice water, and then allowed to reach room temperature. Cytochrome c, 100 pg/ml, was added immediately before spreading onto a deionized water hypophase. <br><br> The nucleic acid-protein films from both methods were absorbed onto <br><br> parlodion-coated grids, stained with uranyl acetate (10), dehydrated in 90% ethanol and rotary-shadowed with platinum-palladium (80:20) at an angle of 50. Micrographs were taken with a Siemens Elmiskop 101 electron microscope at an original magnification of 10,000 and an accelerating voltage of 60 kv. Nucleic acid lengths were measured at a final magnification of 43,000 with a <br><br> Hewlett-Packard 9810A calculator equipped with a 9864A digitizer using pBR322 and fX-174 double and single stranded DNA, respectively, as internal length standards. <br><br> Approximately 10,000 hybrid molecules were screened for these studies; approximately 200 represented molecules containing unambiguous regions <br><br> appropriate for quantitative analysis. The mean lengths of each intron and exon were determined for each method and the data expressed as mean ? <br><br> standard deviation. Nucleic acid lengths of < 50 base pairs could not be <br><br> accurately determined using the methods outlined above. Thus, in the cases <br><br> where introns or exons were of this length or less, accurate error estimates could not be made. <br><br> Bi ohazard precauti ons. <br><br> The construction and screening of the <span class="blue">sheep</span> genomic library together with amplification of pCg45 and preparation of high titre lysates of chimaeric <br><br> Charon 4A were performed under the physical and biological containment levels specified by the NIH guidelines for recombinant DNA research (11). <br><br> 2243 <br><br> Nucleic Acids Research <br><br> RESULTS <br><br> Location and orientation of the intervening and a2 mRNA coding sequences in the SpC3 recombinant DNA. <br><br> Hybridization of fetal <span class="blue">sheep</span> tendon type I collagen mRNA to SpC3 DNA <br><br> yielded a complex R loop pattern in the region where the <span class="blue">sheep</span> genomic insert was expected to be ligated to the arms of the lambda vector, Charon 4A. To <br><br> confirm that these hybridization events were restricted to the inserted <span class="blue">sheep</span> DNA fragment, duplex DNA strands on both sides of this hybridization area were measured. Using pBR322 as an internal length standard, the left arm of the phage was found to contain 21,780 ? 1500 bp, and the right arm, 11,950 ? <br><br> 1030 bp, corresponding to the report values for the left and right arms, respectively, of Charon 4A DNA (5,6). <br><br> The hybridization of SpC3 to a2 mRNA resulted in the destabilization of <br><br> the inserted helical duplex DNA fragment such that one DNA strand was displaced (Figure 1). Occasionally, the displaced DNA strand was mostly absent, probably due to fragmentation induced by accidental mechanical shearing. This event considerably enhanced the visualization and ordering of the introns. <br><br> As the a2 mRNA annealed to complementary DNA regions present in one <br><br> DNA strand, numerous single-stranded loops resulted. These loops, referred to as introns, represented DNA sequences not complementary to the mRNA. Although, by convention, introns are usually sequentially labeled 5' to 3', an opposite order had to be employed in this case as SpC3 does not contain the sequences coding for the 5' end of <span class="blue">sheep</span> a2 mRNA. Infrequently, all introns <br><br> were clearly observed within the same hybrid molecule (Figure 1). The largest introns were towards the left arm of Charon 4A and were more clustered than the introns at the opposite end of the hybrid. Thus, much of the genetic <br><br> information coding for the a2 mRNA was contained in that half of the hybrid attached to the right arm. <br><br> A small non-hybridized tail was consistently found in the region where <br><br> the right arm of the phage was ligated to the <span class="blue">sheep</span> genomic insert (Figure 1). This tail likely corresponded to the 3' poly A sequence of the a2 mRNA. At <br><br> the extreme 5' end of the insert, a variable length was often observed for the remaining unhybridized mRNA strand. This was most likely due to the size heterogeneity of the a2 mRNA used in these studies. <br><br> Size determination and comparative analysis of SpC3 introns. <br><br> Although localization of exon and intron sequences was possible with both the 70% formamide and urea-formamide spreading methods, the probability of <br><br> detecting unambiguous regions in a hybrid molecule was far greater with the <br><br> 2244 <br><br> Nucleic Acids Research <br><br> A                                B ',:  '14 .0. <br><br> 13 <br><br> 7 1 <br><br> -5 13 <br><br> .  .   . .   ,4                     x2%.   N1 <br><br> 9 <br><br> >   10          R       4 <br><br> Figure 1.    Electron microscopic visualization of all 17 introns and 18 exons <br><br> from hybrid molecules formed between recombinant clone SpC3 DNA and fetal <span class="blue">sheep</span> <br><br> tendon a2 mRNA. (A,B) Shown are the double-stranded segment of the right arm CR) of Charon 4A; double-stranded segment of the left arm CL) of Charon 4A; the <br><br> displaced single-stranded DNA segment of the insert (5); single-stranded DNA loops representing introns sequentially labeled 1 through 17 in the 3' to 5' direction; and 18 regions of insert DNA sequences hybridized to ct2 mRNA <br><br> ---      - ).  (C,D) Similar to (A,B) but the single-stranded DNA segment (5) of the insert is mostly absent. The urea-formamide spreading method, as described in Materials and Methods, was employed. <br><br> 2245 <br><br> Nucleic Acids Research <br><br> urea-formamide method than with the 70% formamide procedure. For this reason, most measurements were made with the urea-formamide method (Tables 1,2). For <br><br> example, the lowest probability event, the detection of intron 17 and the surrounding small exons, "q" and "r", was only observed by way of the ureaformamide spreading procedure. <br><br> The smallest introns, 7 and 12, were found to contain approximately <br><br> the same number of bases. Introns 1 through 8, representing the DNA sequences which split the a2 coding sequence corresponding to the 3' end of the a2 mRNA, varied in length over approximately a 2.4-fold range. With the exception of <br><br> Table 1. Electron microscopic R loop analysis <br><br> present in SpC3 DNA1. <br><br> of the <span class="blue">sheep</span> a2 introns <br><br> SpC3 was hybridized to <span class="blue">sheep</span> a2 mRNA as described in Materials and Methods; intron lengths were determined by comparison with *X-174 DNA lengths. <br><br> 2 Introns are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 Intron 17 was visualized only by the urea-formamide method. <br><br> 2246 <br><br> 70% Formamide Method              Urea-Formamide Method <br><br> Intron2         Number of       Intron length       Number of        Intron length <br><br> introns analyzed       (bases)       introns analyzed      (bases) <br><br> 1              10             819 t 063              35            806 t 76 2              12             771 ? 89               46            757 t 78 3              11             458 ? 58               47            464 ? 66 4               6             627? 53               41             611 ? 73 5               5             554 ? 51              34             562 ? 76 6               6             675 ? 19               40            684 ? 76 7               5             386 ? 48               50            342 ? 54 8                7            771 ? 84               49            757 ? 82 9               8           1,181 ? 94               52          1,124 ?122 10               8           1,133 ?106               53          1,124 ?137 11              10             964 t?79              61             952 ?105 12               9             362 ? 77              62             342 ? 71 13               4             723 ?222               53            806 ?105 14               4           1,085 ?142               53          1,075 ?105 15               3             795 ? 60              47             806 ?100 16               2           1,229 ? 48              43           1,319 ?146 174              -                 -                  12          1,539 ?166 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Table 2.   Electron microscopic R loop analysis of the <span class="blue">sheep</span> a2 exons <br><br> present in SpC3 DNA1. <br><br> 70% Formamide Method                Urea-Formamide Method <br><br> Exon2          Number of        Exon length       Number of       Exon length <br><br> exons analyzed     (base pairs)    exons analyzed    (base pairs) <br><br> a                 7             807 ? 883           26            883 ?132 b                10             252 ? 35            41            234 ? 49 c                11             151 ? 25            48            182 ? 42 d                 6             252 ? 60            44            234 ? 62 e                 5             126 ? 45            37            129 ? 47 f                 5             126 ? 38            35            104 ? 41 9                 5             176 ? 48            40            182 ? 54 h                 5             100 ? 43            47            104 ? 31 <br><br> 7             100 ? 35            50            104 ? 28 j                 7             126 ? 48            45            130 ? 47 k                 8              50 ? 48            51             52 ? 39 1                 9              75? 10             60             52 ? 44 m                 6             126 ? 43            51            182 ? 54 n                 4             100 ? 45            53             52 ? 42 o                 3             <504                50             78 ? 36 p                 2             126 ? 71            47            104 ? 36 q5                - _                               33            130?47 r5 r                                                 6             78?36 <br><br> SpC3 was hybridized to <span class="blue">sheep</span> a2 mRNA as described in M4aterials and <br><br> Methods; exon lengths were determined by comparison with pBR322 DNA lengths. <br><br> 2 Exons are designated sequentially 3' to 5'. <br><br> 3 All data is presented as mean ? standard deviation. <br><br> 4 5 <br><br> No statistical evaluation was possible because of the size of the exon (see (Materials and Methods). <br><br> Exons q and r were only analyzed by the urea-formamide method. <br><br> intron 1, the nucleotide size of these introns was less than 800 bases. In contrast, introns 9 through 17 varied over a 4.5-fold range. Except for intron 12, all of the introns 9 through 17 contained 800 bases or more. <br><br> Summation of introns 1 through 16 for the 70% formamide and urea<br><br> formamide spreading methods yielded 12,533 and 12,531 bases, respectively. <br><br> However, detection of intron 17 by the urea-formamide method indicates that <br><br> the actual summed value for the intervening sequences of SpC3 is 14,070 bases. <br><br> 2247 <br><br> 1 <br><br> Nucleic Acids Research <br><br> Size determination and the comparative analysis of SpC3 exons. <br><br> Eighteen exons, "a" through "r", were mapped (Table 2). The largest exon, "a", located on the extreme 3' end of the <span class="blue">sheep</span> genomic insert, contained <br><br> approximately 800 bp. With the exception of exon "m", exons larger than 150 <br><br> base pairs were confined to the genetic region corresponding to the far 3' end of the a2 mRNA. Exon "a" and exon "b", the two exons situated on either side of intron 1, coded for around 1000 bases of a2 collagen information; these two <br><br> exons contained one-third of the mapped 3,014 DNA bases complementary to a2 mRNA Length distribution and organization of the introns and exons in the <span class="blue">sheep</span> genomi c i nsert. <br><br> There was a sharp contrast in the size distribution of the exons and introns contained within SpC3 (Figure 2). While the intron sizes varied <br><br> considerably, the exons, in general, showed a very narrow distribution of <br><br> lengths, and thus tended to cluster at the lowest end of the length distri<br><br> A. <br><br> (I) <br><br> z 6 0 <br><br> z <br><br> ZL4 0 <br><br> 021               <br><br> z <br><br> 10 <br><br> B. <br><br> 8 <br><br> z 0 <br><br> 16 0 z <br><br> 2 <br><br> 20   400  600  800 1000 1200 1400 1600 <br><br> LENGTH IN NUCLEOTIDES <br><br> Figure 2. Length distribution of the intron and exon mean lengths measured from micrographs prepared by the urea-formamide method. (A) Introns; (b) Exons. See Tables 1 and 2 for details. <br><br> 2248 <br><br> Nucleic Acids Research bution graph. The exon in the 800-900 bp region was unique and was the only exon which exhibited length overlap with introns. <br><br> The maps of the a2 collagen gene obtained with both electron micro<br><br> scopic spreading methods were almost identical (Figure 3), except that the <br><br> region at the extreme 5' end of the insert could only be mapped with the ureaformamide method. From these measurements we concluded that the length of the a2 collagen genetic regi?on in SpC3 was 17,084 bases long with only 3,014 bases actually coding for a2 mRNA. Therefore, only 18% of the mapped 17 kb <span class="blue">sheep</span> genetic insert actually contained a2 structural information. <br><br> DISCUSSION <br><br> The organizational structure of the a2 collagen gene is complex. A minimum of 17 introns interrupt the 3 kb of ca2 genetic information interspersed as 18 exons over 17 kb of <span class="blue">sheep</span> DNA. With the exception of exon "a" located at the extreme 3' end of the inserted a2 gene, all other mapped exons are rather small, exhibiting a size distribution range of 50 to 250 bp. In <br><br> contrast, the intron sizes range from 340 to 1540 bases occupying a total <br><br> of 14 kb. Thus, the DNA region containing the 3 kb of a2 collagen genetic information is 5.6 fold longer than the corresponding co2 mRNA sequences. <br><br> Although the total number of introns present within the 60% 3' portion <br><br> of the a2 <span class="blue">sheep</span> gene was determined to be 17, variation in the precise number <br><br> TRANSRIPTlO <br><br> A                          5w                r <br><br> 1 "2a4  133  112s  11.17  0. 3O 2  715  6.2  313  04  4.2B  3.51  2.2  12 <br><br> EXON0            p  o    n         5  Ik         h     f     d 4  .  b   <br><br> WPM300       16   16  14   13  12  11  10  9   6  7  6  5  4  3   2   1 <br><br> B <br><br> 110   16     4.2  13111  112n  1113104  3.6    gm74  2 6.  4 4 2  3.2  22  in <br><br> 1  a1m03  133  1.2 11   10  01 341  6. 1   117  6.2 4  12  417  3I5'm 2  <br><br> DM00  1    q5    p  0    n   m  I  k.   j    i   h  g  f     d  c   b     a 147306   17   16   15  14  13  12  11  10  9   a  7  6  5  4  3   2   1 <br><br> 175 17.0 16.6& 16'5.5 15.0 14.5 14. 13.5 13.0 12.5 12.0 11.5 11 10.5M 10.0 9.5 9.0 8. 5.0 71 7.0 6.5 6. 5.5 50 4.5 4.0 3.5 3.0 2.5 2.0 1.5 1.0 <br><br> KILOBASE <br><br> Fiue3      Organization of 60% of the 3' end of the <span class="blue">sheep</span> a2 collagen gene. (A)}The exon ( ) and intron (CDJ) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules visualized by the electron microscopic 70% formamide spreading method.  (B) The exon Um) <br><br> and intron (Efl) order within the a2 gene as determined by R loop analysis of SpC3 DNA-a2 mRNA hybrid molecules evaluated by the electron microscopic <br><br> urea-formamide spreading method. Arrows represent the approximate location of intron-exon junctions. All values are presented as kilobase pairs. <br><br> 2249 <br><br> Nucleic Acids Research <br><br> of introns present in individual hybrid molecules was observed. The reason <br><br> for this may be due to size heterogeneity of the a2 mRNA used as a complementary probe. Although the a2 mRNA was extensively purified (2), some degradation was likely present. It is also possible that additional introns exist that are too small to be detected by electron microscopic R loop analysis. The most likely a2 genetic regions containing such introns are exons "g" and "m". Small <br><br> "protrusions" were sometimes observed within these exons that could represent small introns. Very small introns have been reported for tRNA genes (12,13), <br><br> and it will be necessary to sequence these regions of SpC3 to determine if such introns are present within the a2 <span class="blue">sheep</span> genome. <br><br> The existence of mRNAs comprised of sequences complementary to noncontiguous regions in a DNA genome was first described for <span class="yellow">adenovirus</span> (14,15) and <br><br> simian virus (16,17) genes. Similar observations were soon reported for eukaryotic genes, including serum albumin (18), conalbumin (19), fibroin (20), s<br><br> globin (21,22), growth hormones (23,24), immunoglobulin (25), lysozyme (26), <br><br> ovalbumin (27-29), ovomucoid (30), rRNA (31-34), tRNA (12,13), and vitellogenin (35). The precise number or occurrence of introns within these genes does not seem to follow any clear pattern. However, the genes for albumin, conalbumin, <br><br> lysozyme, ovalbumin, and ovomucoid exhibit a structural organization comparable to the a2 collagen gene. All of these genes are composed of numerous small <br><br> exons, usually less than 250 base pairs, and introns which show a large size variation within an individual gene. Within all of these genomes, the DNA <br><br> regions required for coding structural formation are 4 to 7 times longer than the corresponding mRNA coding sequences. <br><br> As with the a2 collagen gene, a large unique exon at the extreme 3' end <br><br> is also present in the ovalbumin gene (36,37). In the a2 gene, this exon was 800 to 900 nucleotides in length, while in the ovalbumin gene the exon contained 1,030 base pairs. Interestingly, this ovalbumin exon contained 634-650 base pairs ot DNA sequence to the 3' side of the terminator <br><br> triplet, suggesting the ovalbumin mRNA has a large untranslated region at the 3' end. <br><br> The consistent appearance of a small non-hybridized tail, approximately 100 nucleotides in length, in the region where the right arm of the phage was ligated to the <span class="blue">sheep</span> genomic insert of SpC3 very likely corresponded to the <br><br> 3' poly A sequence of the a2 mRNA. This would suggest that the a2 mRNA coding sequence terminates very near or at this point in the genome. Previously published biochemical evidence (2) obtained from restriction mapping and Southern blot analysis using [32P]-labeled <span class="yellow">chick</span> a2 cDNA probe supports <br><br> 2250 <br><br> Nucleic Acids Research <br><br> this observation. <br><br> It is unclear what role introns play in genetic expression. However, <br><br> it is known that these sequences are tranncribed as part of larger precursor mRNAs and the introns are then excised and the fragmented mRNA chains <br><br> covalently rejoined (38). Some evidence has recently accumulated demonstrating the existence of precursor a2 mRNAs (39,40). However, if the introns in the a2 gene are transcribed in their entirety, as has been <br><br> demonstrated for the introns in ovalbumin and globin (41,42), much larger precursor a2 mRNAs may exist than those reported to date. For if the re<br><br> mainder of the a2 collagen gene has a structural organization comparable to the 60% 3' portion that has been analyzed, it is probable that the a2 <br><br> collagen gene in its entirety is dispersed throughout a 30 kb DNA segment. <br><br> ACKNOWLEDGMENTS <br><br> We thank Dr. John Dahlberg and Frances Loebenstein, Laboratory of Cellular and Molecular Biology, National Cancer Institute, for the generous use of a <br><br> Siemens Elmiskop 101 electron microscope; Dr. Ursula Heine and Benjamin Elliott, Laboratory of Viral Carcinogenesis, National Cancer Institute, for the use of a Hewlett-Packard calculator equipped with a digitizer and a Denton vacuum <br><br> evaporator, and Drs. Victor Ferrans and Oichi Kawanami, Laboratory of Pathology, <br><br> National Heart, Lung, and Blood Institute, for expert assistance with photography. We are also grateful to Margery Sullivan, Laboratory of Molecular Genetics, <br><br> National Institute of <span class="yellow">Child</span> Health and <span class="yellow">Human</span> Development, for helpful discussions and to Dr. Helga Boedtker, Department of Chemistry, Harvard University, for making available the recombinant plasmid pCg45. <br><br> References <br><br> 1.   Prockop, D. J., Kivirikko, k. I., Tuderman, L., and Guzman, N. A. <br><br> (1979) New Engl. J. Med. 301, 13-23 and 77-85. <br><br> 2.   Boyd, C. D., Tolstoshev, P., Schafer, M. P., Trapnell, B. C., Coon, <br><br> H. C., Kretschmer, P. J., Nienhuis, A. W., and Crystal, R. G. (1980) J. Biol. Chem., in press. <br><br> 3.   Blin, N., and Stafford, D. W. (1976) Nucl. Acids Res. 3, 2303-2308. 4.   Kretschmer, P. J., Kaufman, R. E., Coon, H. C., Chen, M. JY., Geist <br><br> C. E., and Nienhuis, A. W. (1980) J. Biol. Chem., in press. <br><br> 5.   Maniatis, T., Hardison, R. C., Lacey, E., Laver, H., O'Connell, C., <br><br> Quon, D., Sim, G. K., and Efstratiadis, A. (1975) Cell 15, 687-701. <br><br> 6.   Blattner, F. R., Blechl, A. E., Denniston-Thompson, K., Faber, H. E., <br><br> Richards, J. E., Slightom, H. L., Tucker, P. W., and Smithies, 0. (1978) Science 202, 1279-1284. <br><br> 7.   Meyer, J., Neuwald, P. D., Lai, S., Maizel, J. V., Jr., and Westphal, <br><br> H. (1977) J. Virol. 21, 1010-1014. <br><br> 2251 <br><br> Nucleic Acids Research <br><br> 8.   Westphal, H., and Lai, S. P. (1977) J. Mol. Biol. 116, 525-548. 9.   Schafer, M. P., Rohrmann, G., Heine, U., and Beaudreau, G. S. <br><br> (1979) Virol. 95, 176-184. <br><br> 10. Davis, R. W., Simon, M. M., and Davison, N. (1971) in Methods in <br><br> Enzymology (Grossman, L. and Moldave, K., eds) Vol. 21, pp. 413428, Academic Press, New York. <br><br> 11.  Federal Register (1978) 43, 60108-60131. <br><br> 12. Goodman, H. M., Olson, M. V., Hall, B. D. (1977) Pro. Natl. Acad. <br><br> Sci. U.S.A. 74, 5453-5457. <br><br> 13. Valenzuela, P., Venegas, A., Weinberg, F., Bishop, R., Rutter, <br><br> W. J. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 190-194. <br><br> 14. Berget, S. M., Moore, C., and Sharp, P. A. (1977) Proc. Natl. Acad. <br><br> Sci. U.S.A. 74, 3171-3175. <br><br> 15. Chow, L. T., Gelinas, R. E., Broker, T., and Roberts, R. J. (1977) <br><br> Cell 12, 1-8. <br><br> 16. Berk, A. J., and Sharp, P. A. (1978) Proc. Natl. Acad. Sci. U.S.A. <br><br> 75, 1274-1278. <br><br> 17. Bratosin, S., Horowitz, M., Laub, 0., and Aloni, Y. (1978) <br><br> Cell 13, 783-790. <br><br> 18.  Sargent, T. D., Wu, J. R., Sala-Trepat, J. M., Wallace, R. B., <br><br> Reyes, A. A., and Bonner, J. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 3256-3260. <br><br> 19.  Perrin, F., Cochet, M., Gerlinger, P., Cami, B., LePennec, J. P., <br><br> and Chambon, P. (1979) Nucl. Acids Res. 6 2731-2748. <br><br> 20. Tsujimoto, Y., and Suzuki, Y. (1979) Cell 16 425-436. <br><br> 21. Jeffreys, A. J., and Flavell, R. A. (1977) Cell 12, 1097-1108. <br><br> 22. Tilghman, S. M., Tiemeier, D. C., Seidman, J. C., Matija Peterlin, B., <br><br> Sullivan, M., Maizel, J. V., and Leder, P. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 725-729. <br><br> 23. Fiddes, J. C., Seeburg, P. H., DeNoto, F. M., Hallewell, R. A., <br><br> Baxter, J. D., and Goodman, H. M. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 4294-4298. <br><br> 24.  Soreq, H., Harpold, M., Evans, R., Darnell, Jr., J. E., and <br><br> Bancroft, F. C. (1979) Nucl. Acids Res. 6, 2471-2482. <br><br> 25.  Tonegawa, S., Maxam, A. M., Tizard, R., Bernard, 0., and Gilbert, W. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1485-1489. <br><br> 26. Nguyen-Huu, M. C., Stratmann, M., Groner, B., Wurtz, T., Land, M., <br><br> Glesecke, K., Sippel, A. F., and Schutz, G. (1979) Proc. Natl. Acad. Sci. U.S.A. 76, 76-80. <br><br> 27. Breathnach, R., Mandel, J. L., and Chambon, P. (1977) Nature 270, <br><br> 314-319. <br><br> 28. Dugaiczyk, A., Woo, S. L. C., Lai, E. C., Mace, M. L., Jr., <br><br> McReynolds, L., and O'Malley, B. W. (1978) Nature 274, 328-333. 29. Weinstock, R., Sweet, R., Weiss, M., Cedar, M., and Axel, R. <br><br> (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1299-1303. <br><br> 30.  Catterall, J. F., Stein, J. P., Lai, E. C., Woo, S. L. C., <br><br> Dugaiczyk, Mace, M. L., Means, A. R., and O'Malley, B. W. (1979) Nature 278, 323-327. <br><br> 31. White, R. L., and Hogness, D. S. (1977) Cell 10, 177-192. <br><br> 32.  Glover, D. M., and Hogness, D. S. (1977) Cell 16, 167-176. 33. Wellauer, P. K., and David, I. B. (1977) Cell 10, 193-212. <br><br> 34. Pellegrini, M., Manning, J., and Davidson, N. (1977) Cell 10, <br><br> 213-214. <br><br> 35. Wahli, W., David, I. B., Wyler, T., Jaggi, R. B., Weber, R., and <br><br> Ryffel, G. V. (1979) Cell 16, 535-549. <br><br> 36. McReynolds, L., O'Malley, B. W., Nisbet, A. D., Fothergill, J. E., <br><br> 2252 <br><br> Nucleic Acids Research <br><br> Givol, D., Fields, S., Robertson, M., Brownlee, G. G. (1978) Nature 273, 723-728. <br><br> 37. O'Hare, K., Breathnach, R., Benoist, C., Chambon, P. (1979) <br><br> Nucl. Acids Res. 7, 321-334. <br><br> 38. Abelson, J. (1979) Ann. Rev. Biochem. 48, 1035-1069. <br><br> 39. Adams, S. L., Alwine, J. C., de Chrombrugghe, B., and Pastan, I. <br><br> (1979) J. Biol. Chem. 254, 4935-4938, <br><br> 40. Rave, N., Crkvenjakow, R., and Boedtker, H. (1979) Nucl. Acids Res. <br><br> 6, 3559-3567. <br><br> 41. Roop, D. R., Nordstrom, J. L., Tsai, S. Y., Tsai, M. J., and <br><br> O'Malley, B. W. (1978) Cell 15, 671-685. <br><br> 42. Tilghman, S. M., Curtis, P. J., Tiemeier, D. C., Leder, P., and <br><br> Weissmann, C. (1978) Proc. Natl. Acad. Sci. U.S.A. 75, 1309-1313. 43.  Palmiter, R. D., Davidson, J. M., Gagnon, H., Rowe, D. W., and <br><br> Bornstein, P. (1979) J. Biol. Chem. 254, 1433-1436. <br><br> 44. Graves, P. N., Olsen, B. R., Feitzig, P. P., Monson, J. M. and <br><br> Prockop, D. J. (1979) Fed. Proc. 38, 620. <br><br> 2253 </body></html>