
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Staphylococcus aureus</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA140510</h3>Enp1, a <span class="yellow">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="yellow">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="yellow">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="yellow">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="yellow">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="yellow">human</span> Enp1, expressed in <span class="yellow">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="yellow">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="yellow">human</span> homolog of <span class="yellow">yeast</span> Enp1 was PCR amplified from a <span class="yellow">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="yellow">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="yellow">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="yellow">donkey</span>-anti-<span class="yellow">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="yellow">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="yellow">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="blue">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="yellow">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="yellow">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="yellow">yeast</span> Enp1 (19). In contrast, we identified a <span class="yellow">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="yellow">human</span> bystin (19) showed that they were encoded by the same gene on <span class="yellow">human</span> chromosome 6. The reported sequence for <span class="yellow">human</span> bystin is a fragment of the 437 amino acid <span class="yellow">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="yellow">human</span> bystin, as confirmed by available <span class="yellow">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="yellow">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="yellow">Drosophila melanogaster</span> and <span class="yellow">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="yellow">human</span> Enp1 homolog was cloned and expressed in <span class="yellow">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="yellow">human</span> Enp1 homolog did not complement a <span class="yellow">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="yellow">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="yellow">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="yellow">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="yellow">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="yellow">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="yellow">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="yellow">human</span> homolog of Enp1 did not complement a <span class="yellow">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="yellow">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA2238978</h3>SuperCAT: a supertree database for combined and integrative multilocus sequence typing analysis of the <span class="yellow">Bacillus cereus group</span> of bacteria (including <span class="yellow">B. cereus</span>, <span class="yellow">B. anthracis</span> and <span class="yellow">B. thuringiensis</span>)
Abstract
The <span class="yellow">Bacillus cereus group</span> of bacteria is an important group including mammalian and insect pathogens, such as <span class="yellow">B. anthracis</span>, the <span class="yellow">anthrax</span> bacterium, <span class="yellow">B. thuringiensis</span>, used as a biological pesticide and <span class="yellow">B. cereus</span>, often involved in food poisoning incidents. To characterize the population structure and epidemiology of these bacteria, five separate multilocus sequence typing (MLST) schemes have been developed, which makes results difficult to compare. Therefore, we have developed a database that compiles and integrates MLST data from all five schemes for the <span class="yellow">B. cereus group</span>, accessible at http://mlstoslo.uio.no/. Supertree techniques were used to combine the phylogenetic information from analysis of all schemes and datasets, in order to produce an integrated view of the <span class="yellow">B. cereus group</span> population. The database currently contains strain information and sequence data for 1029 isolates and 26 housekeeping gene fragments, which can be searched by keywords, MLST scheme, or sequence similarity. Supertrees can be browsed according to various criteria such as species, isolate source, or genetic distance, and subtrees containing strains of interest can be extracted. Besides analysis of the available data, the user has the possibility to enter her/his own sequences and compare them to the database and/or include them into the supertree reconstructions.<br><br>INTRODUCTION
Multilocus sequence typing (MLST) is a tool that is widely used for phylogenetic typing of bacteria. MLST is based on polymerase chain reaction (PCR) amplification and sequencing of internal fragments of usually seven essential or housekeeping genes spread around the bacterial chromosome. The genetic relatedness among isolates is then determined by comparison of the nucleotide sequence types (1,2). MLST is thus a method that is unambiguous and truly portable among laboratories. Since the initial development of this technique for <span class="yellow">Neisseria meningitidis</span> in 1998, MLST schemes have been developed for about 30 species including some of the most important bacterial pathogens, e.g. <span class="yellow">Streptococcus pneumoniae</span>, <span class="yellow">Streptococcus pyogenes</span>, <span class="yellow">Haemophilus influenzae</span>, <span class="blue">Staphylococcus aureus</span>, <span class="yellow">Campylobacter jejuni</span>, <span class="yellow">Enterococcus faecium</span>, <span class="yellow">Burkholderia pseudomallei</span>, <span class="yellow">Escherichia coli</span>, <span class="yellow">Salmonella enterica</span> and the <span class="yellow">Bacillus cereus group</span> (see (1) for a recent review). These MLST schemes have been used successfully to explore the population structure of bacteria, to study the evolution of their virulence properties, to identify antibiotic-resistant strains and epidemic clones, and for epidemiological surveillance.
The <span class="yellow">B. cereus group</span> includes bacterial species that are of medical and/or economic importance, such as <span class="yellow">B. anthracis</span>, an obligate mammalian pathogen causing the lethal disease <span class="yellow">anthrax</span>, <span class="yellow">B. cereus</span>, an opportunistic <span class="yellow">human</span> pathogen involved in food-poisoning incidents and contaminations in hospitals, <span class="yellow">B. thuringiensis</span>, an insect pathogen and one of the world's most widely used biopesticide and <span class="yellow">B. weihenstephanensis</span>, a cold-tolerant species known for contaminating dairies. These species are genetically very closely related and may be considered as one species based on genetic and genomic evidence (3–5). Unlike other bacterial species that are typed using a single MLST scheme, five separate schemes have been developed for the <span class="yellow">B. cereus group</span>, based on different sets of genes and isolates (5–10). The Priest scheme (8) is currently the most widely used. Studies with the various schemes have independently indicated that the <span class="yellow">B. cereus group</span> population is divided into three main phylogenetic clusters and that species are usually intermixed within the groups. One cluster contains the monomorphic <span class="yellow">B. anthracis</span> isolates and a number of <span class="yellow">B. cereus</span> and <span class="yellow">B. thuringiensis</span> strains, many of which are from clinical sources. A second heterogeneous cluster includes <span class="yellow">B. cereus</span> and <span class="yellow">B. thuringiensis</span> isolates from various origins, while cold-tolerant <span class="yellow">B. weihenstephanensis</span> and <span class="yellow">B. cereus</span> isolates belong to the third group. The separate MLST analyses have also revealed that the <span class="yellow">B. cereus group</span> population is weakly clonal overall due to numerous clinical and virulent isolates emerging from different phylogenetic positions (5–8,11–14), with the exception of the ‘cold-tolerant’ cluster that seems to exhibit a panmictic (or sexual) population structure, i.e. with frequent genetic exchanges between strains (9).
Despite the overall congruence between the various MLST studies, the use of separate schemes with no gene overlap and very little strain overlap has produced a confusing situation and makes the results difficult to compare directly. Therefore, we recently proposed a combined scheme based on genes taken from three of the four schemes available by then and for which we created a web-based database accessible at the University of Oslo's MLST server, http://mlstoslo.uio.no/ (5). Here, in order to provide the <span class="yellow">B. cereus group</span> research community with a common MLST resource, we have developed on the same website a database, SuperCAT, that compiles and integrates MLST data from all the published <span class="yellow">B. cereus group</span> schemes. In addition, we applied supertree reconstruction methods to build an integrated view of the <span class="yellow">B. cereus group</span> population and phylogeny. Below we describe the content and main features of the new database as well as the process of supertree building.<br><br>DATABASE CONTENT AND IMPLEMENTATION
The SuperCAT database provides information, sequence and phylogenetic data for all bacterial isolates that have been typed using any of the five published MLST schemes for the <span class="yellow">B. cereus group</span> (Table 1). Strain information, when known, includes isolate description, source and geographical location of isolation, and the scheme(s) used for typing. The sequence data include the nucleotide sequences of the MLST loci examined in a given strain. SuperCAT also contains the phylogenetic supertree of the <span class="yellow">B. cereus group</span> reconstructed by combining the sequence data from all five schemes, as well as supertrees built for individual schemes. Information and sequences for isolates typed by the Priest and Tourasse–Helgason schemes were retrieved from the databases devoted to these schemes at http://pubmlst.org/bcereus and http://mlstoslo.uio.no/, respectively. MLST data for additional strains not available in the pubmlst.org repository (strains from (15) are missing therein) and for the Helgason, Ko, and Candelon–Sorokin schemes were taken from the published literature and the Genbank nucleotide sequence database (Table 1). In addition, sequences of all MLST loci were extracted from the complete genomes of the 21 sequenced <span class="yellow">B. cereus group</span> strains available in Genbank. Altogether, SuperCAT currently contains data for 1029 isolates and 26 gene fragments from 25 different genes. However, since most strains have been typed using only 6 or 7 of the 26 loci, about one-third of the complete set of sequences are included. The 26 loci, only available for the completely sequenced strains, sum up to 10 619 bp. All these genes are located on the chromosome, thus the database provides no information about extrachromosomal plasmids even though most of the strains do carry one or several small and/or large plasmids. Unlike scheme-specific MLST databases, SuperCAT does not contain allele and sequence type (ST) numbers. Since isolates in SuperCAT have been typed by different subsets of loci, complete allelic profiles are unavailable and therefore STs cannot be assigned for most strains, except the fully sequenced ones.
Table 1.The five MLST schemes designed for typing bacteria of the <span class="yellow">B. cereus group</span>SchemeGenesTotal sequence length (bp)Total number of isolateseUsed in (references)Helgasonadk, ccpA, ftsA, glpT, pyre, recF and sucC2 938120(6,12,46)Candelon–Sorokina,cclpC, dinB, gdpD, panC, purF and yhfL2 850149(9,10)Kob,cgyrB, mbl, mdh, mutS, pycA(1) and rpoB2 00265(7)Priesta,bglpF, gmk, ilvD, pta, purH, pycA(2) and tpi2 829721(8,11,13–15,46–48)Tourasse–Helgasona,b,dadk, ccpA, glpF, glpT, panC, pta and pycA(2)2 658172(5)aSpecific databases for the Priest and Tourasse–Helgason schemes are accessible at http://pubmlst.org/bcereus/ and http://mlstoslo.uio.no/, respectively. A BLAST database for the Candelon–Sorokin scheme is available at http://spock.jouy.inra.fr/cgi-bin/bacilliMLSopen.cgi.bWhile the Tourasse–Helgason and Priest schemes use the same gene fragment for the pycA gene, the Ko scheme is based on a different and non-overlapping gene region.cThe <span class="yellow">B. cereus group</span>-specific transcriptional regulator plcR was originally included in the Candelon–Sorokin and Ko schemes. However, plcR follows a phylogeny different from the other MLST loci (7,10) and is no longer used for MLST; therefore, it is not included in SuperCAT.dThe Tourasse–Helgason scheme is a combined scheme based on 3 genes from the Helgason scheme (adk, ccpA, and glpT), 3 genes from the Priest scheme (glpF, pta and pycA(2)), and the panC gene from the Candelon–Sorokin scheme.eIncluding strains with fully sequenced genomes.
SuperCAT is built as a relational database using the PostgreSQL management system, and data are accessible through a graphical web interface. User queries and results pages are processed and created on-the-fly via a highly modified version of the mlstdbNet software (16) written in PERL and based on the DataBase Interface (DBI) and Common Gateway Interface (CGI) modules. The database is implemented on a Linux Apache web server maintained through the facilities and support provided by the Norwegian EMBnet node. Some large supertree computations are run on a Linux supercomputer at the University of Oslo. The ATV (A Tree Viewer) Java applet is used for phylogenetic tree display (17). ATV notably supports horizontal and vertical zooming capabilities that are suitable for browsing large trees. The Jalview editor Java applet is also implemented in SuperCAT for advanced multiple sequence alignment display (18).<br><br>SUPERTREE RECONSTRUCTION
Supertree techniques allow to combine the phylogenetic information from different datasets into a common phylogenetic tree and several studies have shown that meaningful supertrees can be obtained even when taxon overlap is very sparse (see (19,20) for reviews). Supertree analysis has thus become increasingly popular for taking advantage and combining the massive amount of sequence data available in public databases for reconstructing large-scale organismal phylogenies with the ultimate goal of building the tree of life (21–24). In this study, the 21 <span class="yellow">B. cereus group</span> strains that have been completely sequenced, and for which the sequences at all 26 MLST loci are thus available, can be used to join all five schemes and provide the strain overlap necessary for supertree analysis. The global <span class="yellow">B. cereus group</span> supertree, containing 1029 isolates, was reconstructed according to the widely used matrix representation by parsimony (MRP) procedure (Figure 1; (19,25,26)). Scheme-specific supertrees were also reconstructed for each of the five MLST schemes by the same technique. Briefly, a phylogenetic tree is built for every gene separately by the maximum likelihood method with the PHYML_aLRT program (27). Then, each gene tree is recoded into a binary matrix representing the branching order (i.e. the phylogenetic groupings) following standard MRP coding using the SuperMRP.pl script (28). All gene tree matrices are concatenated into a supermatrix, in which isolates missing from a particular tree are coded using the ‘?’ character representing unknown data. In this supermatrix, the sequence of 0's, 1's and ?'s defines the branching profile of a strain. Closely related strains have similar branching profiles. Supertrees are then generated from the supermatrix by the maximum parsimony technique using the program MIX from the PHYLIP package (29) run with default parameters. The maximum parsimony step infers the trees that would require the minimum number of changes between the branching profiles of all isolates, where the unknown characters can take any of the two possible states 0 or 1 (they are not treated as missing gaps). As many trees were equally parsimonious, the final supertree was taken as the strict consensus of all parsimony trees with the CONSENSE program of PHYLIP. In order to obtain branch lengths that are proportional to the amount of nucleotide changes, we added an additional step in which branch lengths and statistical support for groupings are estimated from the concatenated sequences by the maximum likelihood method employing approximate likelihood-ratio tests (aLRTs) for branches using PHYML_aLRT with Shimodaira-Hasegawa-like support values (27,30). aLRTs provide a fast way of testing branch support without requirement of multiple replicates like traditional bootstrap procedures. The Felsenstein-1984 nucleotide substitution model supplemented with a gamma distribution (F84+Γ) was used in maximum likelihood computations for individual gene trees and the supertree (31). This model allows for unequal base frequencies, transition/transversion rate bias, and gamma-distributed substitution rate variation among sites. It was empirically chosen as a consensus from exploratory model testing using ModelTest (32,33), which indicated that models including these three factors were most appropriate for the MLST loci studied, although models for individual loci differed slightly. Note that the maximum likelihood technique also allows for uneven rates of nucleotide substitution between strains, which allows to accommodate slow- and fast-evolving isolates. To reduce the size of the binary supermatrix and speed up computations, individual gene trees and the supertree were built using only one representative from a set of strains having identical sequences. The remaining identical isolates were graphically added to the tree afterwards when drawing the final supertree.
Figure 1.Schematic overview of the <span class="yellow">B. cereus group</span> supertree reconstruction procedure using Matrix Representation by Parsimony (MRP). See text for details.
It should be noted that the global 1029-strain supertree retains the phylogenetic signals from the individual schemes and contains the three main clusters of the <span class="yellow">B. cereus group</span> population described in the section ‘Introduction’. The integrated SuperCAT system may also allow to infer new relationships between strains that were analyzed with different gene sets. Even though the 26 loci sequences are available for only 21 isolates, they apparently provide enough overlap information for building the main branches of the supertree. These 21 isolates cover all three clusters, although the majority of them are <span class="yellow">B. anthracis</span> strains or clinical strains closely related to <span class="yellow">B. anthracis</span> due to the focus of genome sequencing projects, making the part of the supertree containing these isolates likely to be more accurate than the rest of the tree. Furthermore, 111 other isolates have been typed by 10 genes or more, providing additional overlap (see the ‘Gene Distribution’ page). Although about two-thirds of the sequence data are missing overall, it has been shown for other organisms that relevant supertrees could be reconstructed with datasets containing more than 90% of missing data, especially when the characters that are present are informative (20,22,23,34). Empirical and simulation studies have indicated that this behavior may be due to the fact that the characters which are present are more important for the tree-building process than those which are absent (see (20,34) and references therein). Precise within-cluster groupings may contain more uncertainty, as indicated by the large number of unresolved multifurcations in the <span class="yellow">B. cereus group</span> supertree. Finally, it is also worth mentioning that the branching orders of the scheme-specific MRP supertrees are highly correlated to those of the published trees built with concatenated sequences and other phylogenetic algorithms.<br><br>DATA ACCESS AND MANIPULATION
The complete list of isolates included in SuperCAT (currently 1029) with strain description, source and country of origin is available at the database home page. By default all isolates in the database are used in the analysis tools provided, but the user can select strains of interest by keywords, MLST scheme, entering a list of strain identifiers, or choosing isolates individually via checkboxes. All subsequent analyses will be based on the selected strain subset and their loci. The keyword search will look for matches in any of the strain, description, source, location and scheme fields. Complex keyword queries with several logical operators can be formulated in the ‘advanced search’ page. Note that many isolates were referred by alternative names in different MLST schemes and publications, therefore synonyms have been included in the strain descriptions that allow a particular isolate to be looked up using any of its alternative identifiers. A sequence search is also possible using BLASTN (35), in order to select isolates that have allele sequences identical to user-entered query sequences.
Throughout SuperCAT, clicking on a strain name will pop up an isolate-specific window showing all relevant information and giving access to the nucleotide sequences of individual loci for that isolate. Detailed information about the MLST schemes (e.g. loci names and lengths, genomic coordinates, literature references) and their overlap, the distribution of available loci among the isolates, and the supertree reconstruction procedure can be obtained by clicking the relevant links in the header line present at the top of every page.
Apart from the basic functions for selecting and accessing strain information and sequence data for all five <span class="yellow">B. cereus group</span> MLST schemes, the main features of SuperCAT relate to the manipulation of the supertrees constructed by the MRP approach. The global supertree based on the combination of all five <span class="yellow">B. cereus group</span> MLST schemes as well as the five scheme-specific supertrees can be browsed according to various user-chosen criteria (Figure 2). Isolates in the supertrees can be colored by species or source of isolation. It is also possible to specifically mark in red the current subset of strains that has been selected by the user and to extract from the supertrees the subtree containing only those isolates. In the case of the multi-scheme supertree highlighting of the strains can be based on genetic distance. With this option the user can mark on and/or extract from the tree the isolates that are genetically closely related to strains of her/his choice. The user can either select strains that share one or several identical allele sequences with her/his query isolate(s) or that are at a specified genetic distance. Distances between isolates are computed by summing up the lengths of the branches (in average number of nucleotide substitutions per site) connecting the isolates in the supertree (known as patristic distances; (36)). The genetic relatedness search functions are also available in a separate page for the user to find closely related isolates without tree manipulation. SuperCAT allows to compare the scheme-specific MLST supertrees with each other and with the global supertree by using the subset of isolates that are common to all selected schemes. Common isolates can either be highlighted in red or be extracted as subtrees from each supertree, which can be used for comparing the positions of the common strains in the various MLST trees. For all supertree-related options, detailed tree navigation can be achieved using the various functions in the ATV tree window when the trees are displayed (17).
Figure 2.Examples of supertree browsing and manipulation in SuperCAT. A, supertree colored by species; B, specific highlighting of user-selected strains (in red); C, extracted subtree containing only the strains highlighted in B. Trees are displayed using ATV (17).
Besides the manipulation of the precomputed supertrees, SuperCAT offers the user the possibility to compute new supertrees by MRP using any combination of strains, schemes and genes. Supertree computations may be extremely time consuming, ranging from a few minutes to 2–3 days with the complete database. Users are therefore requested to enter their e-mail addresses and will receive a notification containing a link to the results page when the supertree is ready. Note that when building a supertree for a user-selected subset of strains, the computation will first include all database isolates. A subtree containing only the user-selected isolates will then be extracted from the supertree of all strains. Although more time consuming, this strategy allows: (i) to avoid sampling artefacts as phylogenies built with different isolate sets may vary and (ii) to obtain relationships even if the selected isolates have been typed using non-overlapping gene sets, as the supertree of all isolates can always be built owing to the completely sequenced strains that are common to all schemes.
Another main feature of the SuperCAT database is that the user can enter her/his own private sequences and conduct several sequence analyses (Figure 3). These analyses include: (a) building new supertrees containing user isolates and sequences; (b) finding database isolates having sequences most similar to the user's query sequences using an on-line BLASTN (35) service; and (c) aligning user sequences to database genes using the multiple sequence alignment program CLUSTALW (37). For the last option, the Jalview editor (18) is provided for advanced multiple alignment display. All user-entered data must be in FASTA format and can be either copied and pasted into the query forms or uploaded from text files stored locally on the user's computer.
Figure 3.Examples of query results in SuperCAT. A, multi-scheme BLAST search with sequence alignment; B, multi-scheme genetic search showing the list of isolates sharing one or more sequences with a query strain; C, multiple sequence alignment using Jalview (18).
All strain information data, sequences and phylogenetic trees, including user-made supertrees and extracted subtrees, can be saved and downloaded freely from the database. Users wishing to have their MLST data included as part of the SuperCAT release (and/or the Tourasse–Helgason scheme-specific database) are welcome to contact N.J.T. or A.-B.K. at the e-mail addresses given on the Oslo MLST server front page.<br><br>DISCUSSION AND FUTURE DEVELOPMENTS
SuperCAT is a newly created database devoted to the <span class="yellow">B. cereus group</span> of bacteria whose main objectives are to provide a common MLST repository and means for building a comprehensive genetic analysis of the group that has been typed by five separate schemes. The database is publicly and freely available at http://mlstoslo.uio.no/, along with the database specific for the combined scheme of (5). We plan to update SuperCAT quarterly.
Future developments of the database may deal with refining the supertree-building procedure. In particular, a new improved method has recently been developed for taking into account both nucleotide substitutions and recombination events in phylogenies, as part of the ClonalFrame software, which has been applied to the <span class="yellow">B. cereus group</span> and the Priest scheme (38). It would therefore be of interest to examine the suitability of ClonalFrame in the supertree context. It is also tempting to extend the supertree analysis beyond MLST data, by incorporating the phylogenies obtained previously from large-scale multilocus enzyme electrophoresis (MLEE; (3,39–41)) and amplified fragment length polymorphism (AFLP; (42–45)) studies. The MRP framework is ideal since it allows to integrate trees that can be built from different methods and data. As MLEE, AFLP and MLST have different levels of resolution, one can hope that combining them might provide an even more robust supertree for the <span class="yellow">B. cereus group</span>.<br><br>
<h3>pmcA2658668</h3>A novel DNA-binding protein modulating methicillin resistance in <span class="blue">Staphylococcus aureus</span>
Abstract
Background
Methicillin resistance in <span class="blue">Staphylococcus aureus</span> is conferred by the mecA-encoded penicillin-binding protein PBP2a. Additional genomic factors are also known to influence resistance levels in strain specific ways, although little is known about their contribution to resistance phenotypes in clinical isolates. Here we searched for novel proteins binding to the mec operator, in an attempt to identify new factor(s) controlling methicillin resistance phenotypes.<br><br>Results
Analysis of proteins binding to a DNA fragment containing the mec operator region identified a novel, putative helix-turn-helix DNA-binding protein, SA1665. Nonpolar deletion of SA1665, in heterogeneously methicillin resistant <span class="yellow">S. aureus</span> (<span class="yellow">MRSA</span>) of different genetic backgrounds, increased methicillin resistance levels in a strain dependent manner. This phenotype could be fully complemented by reintroducing SA1665 in trans. Northern and Western blot analyses, however, revealed that SA1665 had no visible influence on mecA transcription or amounts of PBP2a produced.<br><br>Conclusion
SA1665 is a new chromosomal factor which influences methicillin resistance in <span class="yellow">MRSA</span>. Although SA1665 bound to the mecA promoter region, it had no apparent influence on mecA transcription or translation, suggesting that this predicted DNA-binding protein modulates resistance indirectly, most likely through the control of other genomic factors which contribute to resistance.<br><br><br><br>Background
Methicillin resistant <span class="yellow">S. aureus</span> (<span class="yellow">MRSA</span>) are an ever increasing threat, both in clinical settings and more recently as an emerging community acquired pathogen. Their invasiveness and pathogenesis relies on a variable arsenal of virulence factors, paired with resistance to virtually all β-lactams and their derivatives. Their ability to rapidly generate resistance to other unrelated classes of antibiotics, or to take up additional resistance determinants, severely hampers therapy and eradication.
In <span class="yellow">S. aureus</span>, methicillin resistance is conferred by an acquired, β-lactam-insensitive penicillin-binding protein (PBP), PBP2a [1-4]. PBP2a is encoded by mecA, which is divergently transcribed from its cognate regulators, mecR1 (sensor/signal transducer) and mecI (repressor). If mecR1-mecI are absent or truncated, transcriptional control of mecA is taken over by the structurally similar blaZ (penicillinase) regulatory elements blaR1/blaI, if present. In the absence of both regulatory loci, mecA is constitutively transcribed [5,6]. In the presence of β-lactams, the transmembrane sensor/signal transducers BlaR1/MecR1, undergo a conformational change, followed by autoproteolytic cleavage of the n-terminal cytoplasmic domain, leading to the activation of the cytoplasmic peptidase and subsequent dissociation of the repressor due to proteolytic degradation [7-9]. However, the signal transduction cascade of this regulatory system has still not been completely elucidated.
Oxacillin resistance levels conferred by mecA are strain specific and can vary greatly, with oxacillin minimal inhibitory concentrations (MICs) of different strains ranging from phenotypically susceptible levels, as low as 1 μg/ml up to extremely high values of > 500 μg/ml. Methicillin resistance is also generally expressed heterogeneously. Heterogeneously resistant <span class="yellow">MRSA</span>, when exposed to β-lactam antibiotics, segregate highly resistant subpopulations, which are much more resistant than the majority of the cells [10]. The frequency of highly resistant subclones generated is often well above the spontaneous mutation frequency, and once selected high level resistance often remains stable, even in the absence of selective pressure. There is currently no satisfactory genetic model which explains how these higher resistance levels are triggered or selected and exactly what factors are functionally responsible for the increased resistance in clinical isolates. Methicillin resistance levels are known to not directly correlate with mecA transcription or levels of PBP2a produced [11,12]. However, resistance levels can be manipulated by environmental conditions, such as temperature, pH, osmolarity, and medium composition [13,14].
It has been shown experimentally, that in addition to mecA, methicillin resistance depends on the correct interplay of a multitude of genomic factors, termed fem/aux factors, including genes involved in peptidoglycan precursor formation, composition and turnover; teichoic acid synthesis; and genes of unknown or poorly characterised functions [15-18]. In addition to structural genes, many regulatory loci have also been shown to influence resistance levels, including global regulators of virulence factor production such as the quorum sensing agr system, the staphylococcal accessory regulator SarA and the alternate sigma factor σB [19,20]; regulators of metabolism, such as the catabolite control protein A (CcpA) [21]; and the VraSR two-component sensor transducer, which induces the cell wall stress stimulon in response to cell wall active antibiotic challenge [22].
The vast MIC differences between <span class="yellow">MRSA</span> strains, the population heterogeneity within single strains and the dependence of resistance levels on external factors are reflected in these many structural genes and global regulators, which can influence resistance levels.
While typically considered nosocomial pathogens, new faster growing and apparently more virulent <span class="yellow">MRSA</span> have begun spreading in the community. Interestingly, these emerging strains often express very low methicillin resistance, e.g. the <span class="yellow">MRSA</span> clone spreading amongst intravenous drug users in the Zurich area, which has an in vitro doubling time of 25 min, but oxacillin MICs of only 0.5 to 4 μg/ml [23]. This particular clone's low-level resistance is partially due to a promoter mutation, leading to tight repression of mecA, but resistance levels appear to be mainly restricted by unknown factors within its genomic background [12].
To identify potential factors involved in mecA regulation or methicillin resistance levels in such an extremely low level resistant <span class="yellow">MRSA</span>, we performed DNA-binding protein purification assays, using the mecA operator region as bait. A novel, uncharacterized protein, SA1665, was found to bind to this DNA fragment, and shown to increase methicillin resistance levels when deleted.<br><br>Results
Identification of SA1665
<span class="yellow">MRSA</span> strain CHE482 is the type strain for the so-called "drug clone" spreading amongst intravenous drug users in the Zurich area [12,23]. This strain carries mecA and expresses PBP2a, but appears phenotypically methicillin susceptible by conventional phenotypic tests. However, like most other low-level resistant <span class="yellow">MRSA</span>, it can segregate a small proportion of higher resistant subclones in the presence of β-lactams. We hypothesized that regulation of methicillin resistance in such low-level resistant clonal lineages may differ qualitatively from classical heterogeneously- or highly-resistant <span class="yellow">MRSA</span>.
A DNA-binding protein purification assay was performed to identify new potential factors involved in the regulation of mecA/PBP2a. The mecA/mecR1 intergenic DNA region, including the 5' 9 bp of mecR1 and the first 52 bp of mecA, was used as bait against crude protein extract from strain CHE482. Proteins binding to this DNA fragment were analysed by SDS-PAGE. Even though CHE482 contained BlaI, which is known to bind to the mec operator, this band could not be identified on gels due to co-migrating, non-specific bands the same size as BlaI (14.9 KDa) that bound to both the DNA-coated and uncoated control beads. The most prominent protein band of ~16–20 kDa, isolated from DNA-labelled but not from control beads, was identified as the hypothetical protein SA1665 (N315 genome annotation [BA000018]) (Figure 1A). SA1665 encodes a predicted 17-kDa protein with an n-terminal helix-turn-helix (HTH) motif characteristic of DNA-binding transcriptional regulators. The amino acid sequence of SA1665 showed 100% identity amongst <span class="yellow">S. aureus</span> database sequences and 97–98% identity amongst other staphylococci, including <span class="yellow">S. haemolyticus</span>, <span class="yellow">S. epidermidis</span> and <span class="yellow">S. saprophyticus</span>, indicating that SA1665 is highly conserved. Conversely, there were no orfs highly similar to SA1665 found in other bacterial species, with the most similar sequences found in <span class="yellow">Bacillus licheniformis</span> DSM13 and <span class="yellow">Desulfitobacterium hafniense Y51</span>, which shared only 64% and 59% similarity, respectively.<br><br>Electro mobility shift assays (EMSA)
EMSA was used to confirm binding of SA1665 to the mec operator region. Crude protein extracts of <span class="yellow">E. coli strain BL21</span>, carrying the empty plasmid (pET28nHis6) or pME20 (pET28nHis6-SA1665) which expressed nHis6-SA1665 upon induction with IPTG, were incubated with the 161-bp biotinylated-DNA fragment previously used as bait in the DNA-binding protein assay. A band shift was observed with extracts from the strain expressing recombinant nHis6-SA1665 but not from the control strain carrying the empty plasmid. Several bands resulted from the shift, which is most likely due to protein oligomerisation (Figure 2A). The specificity of the gel shift was also demonstrated by the addition of increasing concentrations of purified nHis6-SA1665 protein to the biotinylated-DNA fragment (Figure 2B). Band-shift of the biotinylated DNA was inhibited in the presence of specific competitor DNA but not by the presence of the non-specific competitor DNA, confirming that nHis6-SA1665 had a specific binding affinity for the 161-bp DNA fragment.<br><br>Effect of SA1665 deletion on β-lactam resistance
To analyse the effect of SA1665 inactivation on methicillin resistance, nonpolar markerless deletions of SA1665 (Figure 1B) were constructed in a selection of clinical <span class="yellow">MRSA</span> isolates, which varied in their genetic background, SCCmec type, and mecA regulation [24]. Strain CHE482, belongs to clonal complex CC45 and sequence type ST45, and contains a novel SCCmec (SCCmecN1 [23]); while strains ZH37 (CC45/ST45) and ZH73 (CC22/ST22) contain type IV SCCmecs. All three of these strains have truncated mecI/mecR1 regulatory loci but intact BlaI/BlaR1 loci controlling mecA expression. Strain ZH44 (CCT8/ST8) contained a type A mec complex (mecI-mecR1-mecA) within a type II SCCmec, and had no β-lactamase locus; so mecA was only under the control of its cognate regulators MecI/MecR1.
Deletion of SA1665 increased oxacillin resistance in all mutants compared to their corresponding parent strains, as demonstrated on oxacillin gradient plates (Figure 3A); with mutants ΔCHE482 and ΔZH37 approximately doubling in resistance and ΔZH44 and ΔZH73 expressing considerably higher resistance. Population analysis resistance profiles of the mutants showed a distinct shift at the top of the curve, indicating that the higher resistance was due to increased basal oxacillin resistance levels (Figure 3B). Strains CHE482/ΔCHE482 and ZH37/ΔZH37 had very similar resistance profiles, despite having different SCCmec elements, suggesting that it was their common clonal background (CC45) that determined their resistance levels and the extent of resistance increase upon SA1665 deletion.
Growth curve analyses showed that deletion of SA1665 slightly reduced the growth rate of all strains tested (Figure 3C). Wild type growth rates were restored upon complementation (data not shown).<br><br>Resistance complementation
Plasmids pME26 and pME27 were constructed for complementation of the deletion mutants. Both plasmids contained the SA1665 orf along with its own promoter and transcriptional terminator. Strains ΔCHE482, ΔZH37, and ΔZH73 were complemented with pME26, and intrinsically kanamycin resistant strain ΔZH44 was complemented with pME27. Wild type-like resistance levels were restored in all mutants by introduction of the complementing plasmids, as shown by gradient plates (Figure 3A).<br><br>Transcriptional analyses
Primer extension, using the 5'-biotinylated primer me97, identified two potential SA1665 transcriptional start sites (TSS), 76-nt and 139-nt upstream of the SA1665 ATG start codon (Figure 4A). Predicted σA promoter consensus -10/-35 box sequences were located upstream of both TSS (Figure 4B). Identical TSS were also identified using the downstream primer me98 (data not shown).
Northern blot analysis was used to investigate SA1665 expression and the influence of SA1665 deletion on mecA and mecR1 transcription. RNA samples taken from different time points over the growth curve of CHE482 showed that SA1665 was expressed strongly in early exponential phase at OD600 nm 0.25 and 0.5, then transcript levels decreased and were almost undetectable in early stationary phase at OD600 nm 4.0 (Figure 5A). In addition to the main transcript of ~0.46 kb, a weaker, larger transcript of ~0.6 kb was also visible, especially at later growth stages. Figure 5B shows the transcriptional behaviour of SA1665 when CHE482 cells were challenged with sub-inhibitory (4 μg/ml) and inhibitory (120 μg/ml) concentrations of cefoxitin. These results showed that low levels of cefoxitin, such as those used to induce mecA/mecR1 transcription, appeared to slightly decrease SA1665 transcription after 30 min exposure, while larger, inhibitory concentrations caused even more significant alterations in the SA1665 transcriptional profile, making it similar to that normally seen in stationary phase growth. These results indicate that transcription of SA1665 may respond in some way to cell wall stress, rather than in direct response to the presence of β-lactams. This observation is based on relatively subtle changes in SA1665 transcription, especially at low concentrations of cefoxitin such as those required for mecA/mecR1 induction. Since deletion of SA1665 has been shown to increase β-lactam resistance, reduced SA1665 transcription in the presence of β-lactams may also provide some protection against β-lactam exposure.
Northerns also showed that, as expected, the SA1665 transcripts were absent from the deletion mutant (Figure 5C), and additional experiments demonstrated that wild type SA1665 transcription patterns were restored by complementation of ΔCHE482 with pME26 (data not shown). The effects of SA1665 deletion on directly up- and down-stream genes were also investigated. Northern blots of the neighbouring genes SA1664, SA1666 and SA1667, showed that expression of all three genes was very weak compared to that of SA1665. A weak transcript of about 3 kb was present in hybridizations probed with orfs SA1665-SA1667. This band decreased in size in the SA1665 mutant when probed with SA1666 and SA1667. One of the transcripts hybridising to the SA1664 probe also decreased in size by ~0.5 kb in the SA1665 mutant, suggesting that SA1665 was present on several transcripts of different lengths, including a high abundance monocistronic transcript and low abundance polycistronic transcripts (Figure 5C). Transcript abundance of both the upstream SA1666-SA1667 operon and the downstream SA1664-specific transcript all appeared to increase slightly in ΔCHE482. The significance of these subtle increases in transcription are unknown, however, polar effects from SA1665 deletion seem unlikely, based on the facts that all genes were still transcribed, their transcription levels all remained extremely low and the transcriptional terminator of SA1665 remained intact in the deletion mutant (Figure 1B).
Expression of mecR1 and mecA were analysed from RNA of uninduced and induced cultures of CHE482 and ΔCHE482. Cells were induced at OD600 nm 0.25 (Figure 5D) and 1.0 (data not shown) with sub-inhibitory concentrations of cefoxitin, to relieve BlaI-repression of mecA. mecR1, although truncated in CHE482, was still transcribed and had the same expression pattern as mecA, as both became derepressed over time and had the highest transcript levels after 30 min of induction. In the mutant ΔCHE482, transcripts of both mecA and mecR1' were unaffected by SA1665 deletion, indicating that SA1665 had no influence on their expression at either OD 0.25 (Figure 5D) or OD 1.0 (data not shown). SA1665 deletion also had no effect on mecA transcription or induction in strains ZH37, ZH44 and ZH73 (data not shown).<br><br>Western blot analysis
Mutants of CHE482 and of ZH44 and ZH73, which had the largest differences in oxacillin resistance levels, were analysed by Western blot analysis to determine if SA1665 affected production of PBP2a from mecA. As shown in Figure 5E, all pairs of wild type and mutant strains had similar amounts of PBP2a present both before and after induction with cefoxitin, indicating that SA1665 deletion did not alter amounts of PBP2a produced. Therefore it seems that SA1665 exerts no direct control over mecA or PBP2a expression.<br><br>
Discussion
Methicillin resistance in <span class="yellow">MRSA</span> is primarily dependent on the presence of the mecA gene, however, resistance levels are generally governed by strain-specific factors including mecA regulatory elements and other chromosomal fem/aux factors which either enhance or repress the expression of resistance. For instance, the very low-level methicillin resistance of the Zurich drug clone CHE482, was shown to be controlled by its genetic background [12] suggesting that it either contained or lacked certain fem/aux factors involved in controlling resistance expression. Many of the currently known fem/aux factors are directly or indirectly involved in cell wall synthesis and turnover, or envelope biogenesis, however there still remain factors of unknown function. Most of the currently known fem/aux factors reduce methicillin resistance levels when inactivated. A few genes, such as lytH, dlt, norG, sarV and cidA increase resistance levels upon inactivation or mutation. All of these genes, except norG, which is an efflux pump regulator, play a role in either autolysis or are important for cell physiology and growth [25-30]. Other genes increase β-lactam resistance upon overexpression, such as hmrA coding for a putative amidohydrolase, hmrB coding for a putative acyl carrier protein [31], or the NorG-controlled abcA multidrug efflux pump [28].
SA1665, a predicted DNA-binding transcriptional regulator, was found to bind to a DNA fragment containing the mecA promoter region. However, although this protein shifted the mecA operator/5' coding sequence, it did not appear to directly control mecA or mecR1 transcription or PBP2a production. Therefore its binding to the mecA region may have no specific regulatory function. Such interactions have been noted before, such as the HTH protein NorG, which was shown to bind specifically to norA, norB and norC promoters, but only transcription of norB was increased when NorG was overexpressed [28]. We have to postulate therefore that SA1665 may modulate β-lactam resistance in a mecA-independent manner, by controlling cellular functions affecting resistance levels. Experiments to determine the SA1665 regulon are ongoing. The impact of deleting SA1665 in <span class="yellow">MRSA</span> was extremely strain specific, underlining the importance of the genetic background in governing the final methicillin resistance levels of <span class="yellow">MRSA</span>, and demonstrating the large genomic variability between different strain lineages.<br><br>Conclusion
SA1665 is a previously uncharacterised DNA-binding protein that has a negative effect on β-lactam resistance in <span class="yellow">MRSA</span>. The SA1665 protein was identified in a DNA-binding protein purification assay, in which it bound to a DNA fragment covering the mec operator region. However, while nonpolar deletion of SA1665 was shown to increase oxacillin resistance levels in several heterogeneously resistant <span class="yellow">MRSA</span>, its deletion had no effect on mecA transcription or PBP2a production. Therefore the negative impact of SA1665 on methicillin resistance is most likely to be through the regulation of other chromosomal factors or cellular functions required for methicllin resistance.<br><br>Methods
Strains and growth conditions
Strains and plasmids used in this study are listed in Table 1. Clinical isolates are from the IMM collection in Zurich, Switzerland. Strains were grown at 37°C in Luria Bertani (LB) broth, shaking at 180 rpm, or on LB agar. Media were supplemented with the following antibiotics when appropriate: 25 or 50 μg/ml kanamycin, 10 μg/ml chloramphenicol, 5 or 10 μg/ml tetracycline, 100 μg/ml ampicillin. Concentrations of cefoxitin used for transcriptional induction were either sub-inhibitory (4 μg/ml) or inhibitory (120 μg/ml).<br><br>Susceptibility testing
Oxacillin resistance levels were compared by swabbing 0.5 McFarland cell suspensions across agar plates containing appropriate concentration gradients of oxacillin. For population analysis profiles, appropriate dilutions of an overnight culture, ranging from 100 to 108, were plated on increasing concentrations of oxacillin. Plates were incubated at 35°C and colony forming units per ml (cfu/ml) were determined after 48 h.<br><br>Binding-protein purification
Crude protein extracts were isolated from CHE482, grown under normal culture conditions until OD600 nm 1.5. Cells were harvested, resuspended in PBS (pH 7.4) and mechanically lysed using Lysing Matrix B (BIO 101 Systems) tubes and a FastPrep FP120 (BIO 101 Systems). Suspensions were clarified by centrifugation and supernatants, containing soluble cytoplasmic proteins, were transferred to Amicon Ultra centrifugal filter devices (Millipore) with a pore cut-off size of 10 kDa. Proteins were then washed and concentrated in 1× binding buffer (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, and 1 mM DTT, 0.5 M NaCl). Protein concentrations were measured by Bradford assay (BioRad Laboratories GmbH) [32]. Primer pair me36F/me36Rbiot (Table 2) were used to amplify a biotinylated mecA promoter/operator fragment, which was bound to streptavidin coated magnetic beads (Dynabeads M-280 Streptavidin, DYNAL BIOTECH) according to the manufacturer's instructions. Binding reactions, containing DNA-coated beads mixed with 100 μg of crude protein extract in 1× protein binding buffer (20 mM Hepes, pH 7.6, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20 (w/v), 30 mM KCl), 0.02 μg/μl poly d(I-C) and 2 ng/μl poly L-lysine, were incubated at room temperature for 30 min with constant rotation. Beads were then washed and binding-proteins eluted in elution buffer (1× protein binding buffer containing 2 M KCl). Eluted proteins were dialysed against water, concentrated by evaporation, and run on 15% SDS polyacrylamide gels. Gels were silver stained using the Protein Silver Staining kit (Amersham Biosciences AB) without the addition of glutaraldehyde. Protein bands were excised from gels and analysed by mass spectrometry (LC/ESI/MS/MS) at the Functional Genomics Centre, Zurich. The SA1665 protein sequence [BAB42933] was analysed by Blast search  and motif scan .<br><br>Expression of recombinant SA1665 protein
SA1665 was amplified using primer pair me65BamHI/me65XhoI (Table 2) and cloned in-frame into pET28nHis6 (unpublished, D. Frey). The resulting plasmid, pME20, was transformed into <span class="yellow">E. coli BL21</span> for expression of recombinant nHis6-SA1665 protein. To maximise the abundance of soluble protein produced, cultures were grown in osmotic shock medium at 37°C (1 g/l NaCl, 16 g/l tryptone, 10 g/l <span class="yellow">yeast</span>, 1 M sorbitol, 10 mM betaine, modified from [33]) to an OD600 nm of 0.5, cooled briefly on ice, then induced by adding 100 μM IPTG and growing overnight at 22°C. Crude soluble proteins were extracted using CelLyticB 2× cell lysis reagent (SIGMA). HIS-Select Cobalt Affinity Gel (SIGMA) was used to purify recombinant nHis6-SA1665 according to the manufacturer's instructions.<br><br>Electro mobility shift assay
For gel shift assays, 6 ng aliquots of the biotinylated-DNA fragment used for binding-protein purification were incubated with 0–250 ng of purified nHis6-SA1665 protein in 1× binding buffer (20 mM Hepes pH 7.6, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20 (w/v), 30 mM KCl) containing 0.05 μg/μl poly d(I-C) (Roche) and 5 ng/μl poly L-lysine (Roche). For control binding reactions, 130 × unlabelled mec operator DNA (amplified using primers me36F/me36R, Table 2) was used as a specific binding competitor and 6 ng of herring sperm DNA was used as unspecific competitor DNA. Binding was carried out at 22°C for 30 min. Samples were run on 6% native polyacrylamide gels, contact blotted onto positively charged nylon membrane and detected with the Biotin Chromogenic Detection Kit (Fermentas).<br><br>Primer extension
RNA was extracted from CHE482 cultures that were grown to OD600 nm 0.5, as previously described [12]. Primer extension reactions were performed using 20 μg of total RNA and 3 pmol of the 5'-biotin-labelled primers me97 and me98 (Table 2) using Superscript II reverse transcriptase (Invitrogen), according to the manufacturers instructions. Sequencing reactions were performed using the Thermo Sequenase cycle sequencing kit (U.S. Biochemicals). The Biotin Chromogenic Detection Kit (Fermentas) was used for biotin detection.<br><br>Markerless deletion of SA1665
In frame markerless deletions of SA1665, from the chromosomes of CHE482, ZH37, ZH44, and ZH73, were constructed using the pKOR1 allelic replacement system, as described by Bae et al. [34]. Primer pairs used to amplify the DNA fragments flanking SA1665, for recombination into pKOR1 were: me62attB1/me51BamHI and me62BamHI/me62attB2 (Table 2). All deletion mutants were confirmed by nucleotide sequencing over the deleted region, as well as by Southern blot analysis [35] and pulsed field gel electrophoresis (PFGE) [36].<br><br>Cloning of SA1665 for complementation
A 1533-bp DNA fragment, containing SA1665 together with 690-bp of upstream and 379-bp of downstream DNA, was amplified from strain CHE482 using primers me94BamHI/me94Asp718 (Table 2) and cloned into the <span class="yellow">E. coli</span>/<span class="yellow">S. aureus</span> shuttle vectors pAW17 and pBUS1 [37], creating the complementing plasmids pME26 and pME27, respectively. Plasmids were electroporated into RN4220 [38] and then transduced into different strains using phage 80α.<br><br>Northern blot analysis
Strains were grown overnight in LB (Difco), diluted 1:200 and grown for another 3 h. This preculture was used to inoculate 150 ml (1:1000) of fresh prewarmed LB. Cells were then grown to OD600 nm 0.25 or 1.0 and either left uninduced or induced with cefoxitin 4 or 120 μg/ml. Cultures were sampled from both uninduced and induced cells at time point 0' before induction and at 10' and 30' (min) after induction. To monitor SA1665 expression over growth, separate cultures were also sampled at different growth stages corresponding to OD600 nm 0.25, 0.5, 1, 2, and 4. Total RNA was extracted as described by Cheung et al. [39]. RNA samples (10 μg) were separated in a 1.5% agarose-20 mM guanidine thiocyanate gel in 1× TBE running buffer [40], then transferred and detected as described previously [41]. Digoxigenin (DIG) labelled-probes were amplified using the PCR DIG Probe synthesis kit (Roche). Table 2 contains the list of primer pairs used for the amplification of SA1664, SA1665, SA1666, SA1667, mecR1 and mecA [42] probes. All Northern's were repeated at least two times, using independently isolated RNA samples.<br><br>Western blot analysis
Cells were cultured, as described for Northern blot analysis, to OD600 nm 1.0, then induced with cefoxitin 4 μg/ml. Samples were collected at time 0 (before induction), 10 and 30 min (after induction). Cells were harvested by centrifugation, resuspended in PBS pH 7.4 containing DNase, lysostaphin and lysozyme (150 μg/ml of each) and incubated for 1 h at 37°C. Suspensions were then sonicated and protein aliquots (15 μg) were separated on 7.5% SDS-polyacrylamide gels, blotted onto nitrocellulose membranes (Hybond) and stained with Ponceau to confirm equal protein loading. PBP2a detection was performed using monoclonal PBP2a antibody (1:20000) from the <span class="yellow">MRSA</span>-screen kit (Denka Seiken).<br><br>
Authors' contributions
ME carried out molecular genetic and microbiological studies and drafted the manuscript. BB participated in the design of the study and helped to draft the manuscript. NM participated in the design and coordination of the study, carried out molecular biological studies and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA2589670</h3>The ineffectiveness of tobramycin combination therapy in <span class="yellow">Streptococcus faecium</span> endocarditis.
Abstract
A <span class="yellow">patient</span> required mitral valve replacement following ineffective antibiotic treatment of enterococcal endocarditis caused by <span class="yellow">Streptococcus faecium</span>. Endocarditis had relapsed despite therapy with ampicillin and tobramycin for six weeks. A second relapse had occurred following treatment with penicillin and gentamicin. Initial failure of antibiotic therapy may be related to the known lack of in vitro and in vivo synergy between penicillin and tobramycin against <span class="yellow">S. faecium</span>. Effective therapy of enterococcal endocarditis requires considerations of bacterial speciation, determination of high-level aminoglycoside resistance, and preferably adequate antibiotic synergy studies to assure effective therapy.<br><br><br><br>
 THE YALE JOURNAL OF BIOLOGY AND MEDICINE 56 (1983), 243-249 <br><br> The Ineffectiveness of Tobramycin Combination Therapy <br><br> in <span class="yellow">Streptococcus Faecium</span> Endocarditis <br><br> JUDITH A. GOLDSTEIN, M.D.,a HOWARD COHEN, M.D.,a AND <br><br> FRANK J. BIA, M.D., M.P.H.a,b <br><br> aInfectious Disease Section of the Department of Medicine, and bThe Department of <br><br> Laboratory Medicine, Veterans Administration Medical Center, <br><br> West Haven, and Yale University School of Medicine, <br><br> New Haven, Connecticut <br><br> Received July 11, 1983 <br><br> A <span class="yellow">patient</span> required mitral valve replacement following ineffective antibiotic treatment of enterococcal endocarditis caused by <span class="yellow">Streptococcusfaecium</span>. Endocarditis had relapsed despite therapy with ampicillin and tobramycin for six weeks. A second relapse had occurred following treatment with penicillin and gentamicin. Initial failure of antibiotic therapy may be related to the known lack of in vitro and in vivo synergy between penicillin and tobramycin against <span class="yellow">S. faecium</span>. Effective therapy of enterococcal endocarditis requires considerations of bacterial speciation, determination of high-level aminoglycoside resistance, and preferably adequate antibiotic synergy studies to assure effective therapy. <br><br> INTRODUCTION <br><br> Enterococcal endocarditis requires special therapeutic considerations because the responsible organisms are relatively penicillin-resistant streptococci which require synergistic combinations of antibiotics to achieve acceptable cure rates [1,2]. The group D enterococci include three main species <span class="yellow">S. faecalis</span>, <span class="yellow">S. faecium</span>, and <span class="yellow">S. durans</span>. <span class="yellow">S. faecium</span> cause a minority of all cases of enterococcal endocarditis, in those instances in which enterococci have been speciated [3], but they have been more resistant both to penicillin and penicillin-aminoglycoside combinations than <span class="yellow">S. faecalis</span> [4,5,6]. <br><br> We describe a 64-year-old <span class="yellow">man</span> with <span class="yellow">S. faecium</span> endocarditis in whom a six-week course of ampicillin and tobramycin, followed by additional courses of penicillin and other aminoglycosides, failed to eradicate the organism from the <span class="yellow">patient</span>'s mitral valve. This case is of interest because therapeutic failure of ampicillin and tobramycin in <span class="yellow">S. faecium</span> endocarditis has not been reported previously, but might have been predicted on the basis of previous in vitro and in vivo studies [6]. Although the need for both a penicillin derivative and an aminoglycoside in the therapy of enterococcal endocarditis is widely known, it is important to distinguish between the differing efficacies of penicillin-aminoglycoside combinations for treating various species of enterococci such as <span class="yellow">S. faecium</span>. <br><br> 243 <br><br> Address reprint requests to: Judith A. Goldstein, M.D., Section of Infectious Diseases, Department of Medicine, Long Island Jewish-Hillside Medical Center, Queen's Hosp. Center Affiliate, 82-68 164th Street, Jamaica, NY 11432 <br><br> Copyright c 1983 by The Yale Journal of Biology and Medicine, Inc. All rights of reproduction in any form reserved. <br><br> GOLDSTEIN ET AL. <br><br> CASE REPORT <br><br> A 64-year-old male was in good health until December 1980, when he noted intermittent night sweats, malaise, fever, and fatigue. He received oral erythromycin for 14 days with transient improvement of symptoms. However, after completing therapy, symptoms reappeared and he noted a 15-pound weight loss with low-grade fever (99-100.5?F) during the two months preceding admission. There was no previous history of rheumatic or congenital heart disease. <br><br> In March 1981, the <span class="yellow">patient</span> was admitted to his community hospital where evaluation revealed a new apical systolic murmur radiating to the axilla. There were no petechiae, Janeway lesions, Osler's nodes, Roth spots, or splenomegaly. The hematocrit was 34.8 percent, WBC count 9,300 cells per cu mm with a differential count of 67 segmented forms, 9 bands, 15 lymphocytes, 7 monocytes, 1 eosinophil, and 1 basophil. The erythrocyte sedimentation rate (ESR) was 62 mm per hour (nl < 10 mm per hour) and the serum rheumatoid factor titer was 1:320. Chest films and electrocardiogram were reportedly normal. Group D streptococci grew from three sets of blood cultures. Enterococcal endocarditis was diagnosed and he was treated with six weeks of parenteral ampicillin (12 grams per day) and tobramycin (3 mg per kg per day). Resolution of symptoms occurred within several days after antibiotics were begun. Serum bactericidal titers against the organism, obtained during peak antibiotic levels, were 1:8 or greater on several occasions, and blood cultures were negative while the <span class="yellow">patient</span> was receiving antibiotics. M-mode echocardiography demonstrated left atrial enlargement but no definable abnormalities of the mitral or aortic valves. Intravenous pyelogram, oral cholecystogram, cystoscopy, liver-spleen scan, and upper and lower gastrointestinal series were normal except for a few sigmoid diverticuli. Flexible signoidoscopy demonstrated both a small perianal fissure and hemorrhoids. Blood cultures two weeks after therapy were negative. <br><br> Second Admission (June 19-August 4, 1981) <br><br> In June 1981, he again noted intermittent fever, night sweats, and fatigue. Group D streptococci grew from three sets of blood cultures and he was admitted to the West Haven VA Medical Center for recurrent endocarditis. On examination a somewhat louder apical systolic murmur was noted. The ESR was 50 mm/hour and rheumatoid factor titer was 1:320. Serum complement levels were normal and cryoglobulins were not detectable. Chest films, EKG, and urinalysis were normal. Cardiac M-mode echocardiogram revealed a globular mass attached to the posterior mitral valve leaflet with prolapse into a slightly enlarged left atrium. A twodimensional echocardiogram confirmed mitral valve prolapse and suggested a posterior leaflet vegetation. Gallium citrate scan was negative. <br><br> Three separate morphologic colony variants were isolated from blood, each identified as <span class="yellow">S. faecium</span> by Dr. R.R. Facklam (Center for Disease Control, Atlanta, Georgia). The <span class="yellow">patient</span> received intravenous penicillin (20 million units per day) and gentamicin (3 mg per kg per day) for six weeks with improvement. Peak serum bactericidal titers of 1:8 or greater were achieved against two of the colony variants; however, against the large colony morphotype, a titer of only 1:2 was obtained. An enlarged left atrium with intermittent fluttering and prolapse of the mitral valve was noted on echocardiography three weeks into therapy. Multiple blood cultures taken while the <span class="yellow">patient</span> was receiving antibiotics were negative, as were those obtained 48 and 72 hours after discontinuation of antibiotics. <br><br> 244 <br><br> <span class="yellow">S. FAECIUM</span> ENDOCARDITIS <br><br> Third Admission (August 12-October 4, 1981) <br><br> Withing a week following discharge, the <span class="yellow">patient</span> again developed fever, nocturnal sweats, and malaise. <span class="yellow">S. faecium</span> (large colony morphotype, and poorly growing small colony morphotype) grew from six sets of blood cultures obtained on admission. Penicillin (30 million units/day) and gentamicin (3 mg/kg/day) were again administered, initially achieving peak serum inhibitory and bactericidal dilutions of 1:16 and 1:8 against the organism, respectively. However, the organism had a lowlevel resistance to streptomycin (MIC < 125 /Ag/ml), and streptomycin (2 grams per day) was substituted for gentamicin two weeks into antibiotic therapy. Repeat echocardiograms showed irregular and shaggy densities of both mitral valve leaflets with partial prolapse. Cardiac catheterization demonstrated marked mitral valve prolapse with mitral regurgitation. A radiolucent filling defect was noted, suggesting a coronary artery embolus at the origin of the left anterior descending (LAD) artery, causing 75 percent occlusion of the lumen. The <span class="yellow">patient</span> underwent mitral valve replacement, receiving a number 31 porcine Hancock bioprosthesis and bypass graft to the midportion of the LAD. The mitral valve was thickened with several ruptured chordae of the posterior leaflet noted, but no vegetations. The aortic valve appeared normal, with no visible septal or ring abscesses. The occlusion in the LAD was not approached to avoid embolizing distal fragments. <br><br> Histopathologically the mitral valve showed mild fibrosis and myxoid degeneration without inflammatory changes. Bacterial and fungal stains were negative but <span class="yellow">S. faecium</span> grew from fragments of the resected valve. Following surgery the <span class="yellow">patient</span> received six additional weeks parenteral penicillin and streptomycin. Repeat blood cultures on this regimen and following therapy were negative. Evaluation six months following discontinuation of antibiotics showed no evidence of recurrent endocarditis. <br><br> LABORATORY EVALUATION <br><br> The minimum inhibitory and bactericidal concentrations of penicillin, ampicillin, and tobramycin against the <span class="yellow">S. faecium</span> isolated from the <span class="yellow">patient</span>'s blood cultures <br><br> TABLE 1 <br><br> Minimum Inhibitory and Bactericidal Concentrations of Antibiotics, <br><br> Against <span class="yellow">Streptococcus faecium</span> Isolates from Blood Cultures <br><br> Date       Penicillin     Ampicillin         Tobramycin <br><br> Organism         Isolated      MIC        MIC       MBC      MIC       MBC <span class="yellow">S. faecium</span>, prior to ampicillin/ <br><br> tobramycin therapy    3/17/81        2          1        2        >32       >32 <span class="yellow">S. faecium</span>, after ampicillin/ <br><br> tobramycin therapy <br><br> a. Large colony <br><br> variant           6/16/81        2          1        1        >32      >32 b. Medium colony <br><br> variant           6/16/81        2          1        2         32        32 c. Small colony <br><br> variant           6/16/81        4          2        4        >32      >32 aMIC and MBC are minimum inhibitory and bactericidal concentrations of antibiotics, respectively, in tig/ml. <br><br> 245 <br><br> GOLDSTEIN ET AL. <br><br> FIG. 1. Time-kill curve demonstrat8 3-  \   "''                                    ing the effects of various penicillinO                                                  aminoglycoside combinations on S. {73 2-  \         >,                             faecium obtained from the <span class="yellow">patient</span>'s <br><br> blood cultures immediately prior to mitral valve excision. Note the lack of 4u,                synergy between penicillin and tobra25kg/rm Streporrrychn t                  mycin against this organism when <br><br> /0 units/mI Penicillin _         compared to synergistic combinations 0     4     8    12    6    20    24            of penicillin-gentamicin and penicillin<br><br> HOURS                            streptomycin. <br><br> after initial oral erythromycin therapy, and prior to therapy for endocarditis, are shown in Table 1. Following combined therapy with ampicillin and tobramycin and relapse of endocarditis, three morphological variants were isolated from blood cultures and also evaluated. <br><br> Twenty-four hour time-kill curves for penicillin in combination with various aminoglycosides were performed by Dr. Robert Moellering on the <span class="yellow">S. faecium</span> isolated from the <span class="yellow">patient</span>'s blood immediately prior to mitral valve excision and replacement (Fig. 1). Synergy was readily demonstrable against this organism by penicillin-streptomycin and penicillin-gentamicin combinations in vitro, but not by penicillin-tobramycin. <br><br> DISCUSSION <br><br> Among streptococci, the enterococci are unusual in their relative resistance to a broad spectrum of antimicrobial agents, and single-agent therapy is rarely bactericidal against them [2]. Since Hunter's original observations in 1947, it has become increasingly clear that effective synergistic combinations of antibiotics are necessary to successfully treat enterococcal endocarditis [1]. <br><br> Although <span class="yellow">S. faecalis</span> represents the majority of clinical enterococcal isolates, <span class="yellow">S. faecium</span> nonetheless comprises 5-10 percent of these isolates in some series [3,4]. Moreover, major differences exist in antimicrobial susceptibility and resistance to penicillin-aminoglycoside synergism between these two enterococcal species. The MIC of penicillin against <span class="yellow">S. faecium</span> is higher and this organism is more resistant to a number of different combinations of penicillin and various aminoglycosides than is <span class="yellow">S. faecalis</span> [6]. <br><br> The mechanisms of resistance exhibited by enterococci to penicillin-aminoglycoside synergy have been investigated. Clinically achievable levels of amino<br><br> 246 <br><br> <span class="yellow">S. FAECIUM</span> ENDOCARDITIS <br><br> glycosides are generally ineffective against enterococci. This intrinsic low-level resistance (MIC c 250 isg/ml) is thought to be the result of poor antibiotic penetration of the bacterial cell wall. However, in the presence of antibiotics that interfere with cell wall synthesis, there is enhanced aminoglycoside uptake [7]. In concert, these events are the basis for penicillin-aminoglycoside synergism. Ribosomal resistance of the 30S subunit to streptomycin and defective uptake of gentamicin in the presence of penicillin have been reported mechanisms of resistance among enterococcal isolates [8,9]. However, in the majority of instances, failure of synergy involves plasmid-mediated production of aminoglycoside-modifying enzymes. For streptomycin and kanamycin, plasmid-mediated enzymatic inactivation confers high-level resistance (MIC > 2,000 Ag/ml) and correlates with failure of these aminoglycosides to exert a synergistic effect when combined with penicillin [10,11]. Plasmid-mediated modifying enzymes have been found in both <span class="yellow">S. faecalis</span> and <span class="yellow">S. faecium</span>' [12]. Currently, approximately one-half of clinical enterococcal isolates demonstrate high-level resistance to streptomycin and kanamycin [13]. <br><br> Combinations of penicillin with kanamycin, tobramycin, sisomicin, and netilmicin have consistently failed to demonstrate synergistic killing of <span class="yellow">S. faecium</span> in vitro [6]. This failure of synergy occurs even when high-level resistance to these aminoglycosides is not present. The mechanism of resistance appears to be related to the production of an inactivating enzyme that acetylates the aminoglycoside substrate at the 6' position1 [14]. The genetic basis for production of this enzyme has not been well-defined and plasmid transfer experiments have thus far been unsucessful in demonstrating the encodement of this enzyme by extrachromosomal DNA [14]. <br><br> Moellering et al. demonstrated in vivo, utilizing the <span class="yellow">rabbit</span> model of endocarditis, that penicillin and netilmicin were not efficacious in the treatment of endocarditis caused by a low-level aminoglycoside-resistant strain of <span class="yellow">S. faecium</span> [6]. Although combinations of penicillin with tobramycin, kanamycin, or sisomicin were not evaluated, the authors postulated that the same ineffectual result would have occurred. In the present case, the recurrence of <span class="yellow">S. faecium</span> endocarditis after six weeks of therapy with ampicillin and tobramycin confirms the therapeutic and clinical significance of their data, and emphasizes that tobramycin is not an aminoglycoside to be used for treatment of serious <span class="yellow">S. faecium</span> infections. <br><br> Bacterial tolerance has been suggested as a possible basis for therapeutic failures, particularly in the treatment of infections caused by <span class="blue">Staphylococcus aureus</span> with defects in the production of autolysins. MBCs are generally several-fold higher than MICs and this phenomenon appears to be associated with the autolysin defect [5]. Lorian has also described the formation of numerous aberrant cross-walls by <span class="yellow">S. faecalis</span> grown in the presence of subinhibitory concentrations of penicillin [15]. MBCs were only slightly higher than MICs for the three <span class="yellow">S. faecium</span> variants obtained from our <span class="yellow">patient</span>, and they did not appear to be tolerant strains of <span class="yellow">S. faecium</span>. There was no evidence that any of the morphological variants isolated were unusually resistant to antibiotics. Therefore, subsequent failure of therapy could not be explained on the basis of antibiotic resistance patterns. The initial failure of ampicillin and tobramycin therapy may have allowed the infecting organism to become better established and more difficult to eradicate from the valve. Alternatively, a <br><br> 'Wennersten C, Moellering RC Jr: Mechanism of resistance to penicillin-aminoglycoside synergism in <span class="yellow">Streptococcus faecium</span>. Proceedings of the 1th International Congress of Chemotherapy and 19th Interscience Conference on Antimicrobial Agents and Chemotherapy 1:710-711, 1979 <br><br> 247 <br><br> 248                             GOLDSTEIN ET AL. <br><br> persistent focus of infection causing the LAD lesion seen by arteriography may have slowly resolved and accounted for the apparent failure to respond to synergistic combinations of antibiotics. <br><br> In summary, our <span class="yellow">patient</span> was treated for <span class="yellow">Streptococcusfaecium</span> endocarditis with both ampicillin and the aminoglycoside antibiotic, tobramycin. Relapse of endocarditis might have been anticipated on the basis of previous experimental data showing lack of synergy when tobramycin is used against this organism. This case graphically illustrates the relevance of synergy studies to therapeutic considerations in the treatment of endocarditis. However, subsequent therapy with synergistic combinations of antibiotics did not result in cure. The data do not implicate tolerant organisms as a cause for relapses. Failure of therapy may have been related to the presence of protected organisms in vegetations which were seen on echocardiograms, and also suggested by the presence of a possible coronary artery embolus seen on coronary arteriograms. <br><br> In conclusion, speciation of isolates suspected of causing endocarditis and adequate synergy studies of antibiotic combinations are indicated before a long and expensive course of therapy with antimicrobial agents is undertaken for this disease. However, as this case also illustrates, demonstration of synergism in vitro does not assure clinical cure. <br><br> ACKNOWLEDGEMENTS <br><br> Dr. Goldstein was supported by training grant Al 07033-05 from the National Institute of Allergy and Infectious Diseases. <br><br> The authors gratefully acknowledge the advice, laboratory assistance, and careful review of this manuscript by Dr. Robert C. Moellering, Jr. <br><br> The authors also wish to thank Dr. Howard S. Forster for referring this <span class="yellow">patient</span> to us, Ms. Mary Murray and Deborah Beauvais for manuscript preparation, and Ms. Gertrude Barden, MT (ASCP), MHS, for technical assistance and advice in evaluation of this <span class="yellow">patient</span>. <br><br> REFERENCES <br><br> 1. Hunter TH: Use of streptomycin in the treatment of bacterial endocarditis. Am J Med 2:436-442, <br><br> 1947 <br><br> 2. Moellering RC Jr, Krogstad DJ: Antibiotic resistance in enterococci. Microbiology 293-298, 1979 3. Facklam RR: Recognition of group D streptococcal species of <span class="yellow">human</span> origin by biochemical and <br><br> physiological tests. Appl Microbiol 23:1131-1139, 1972 <br><br> 4. Toala P, McDonald A, Wilcox C, et al: Susceptibility of group D streptococcus (enterococcus) to 21 <br><br> antibiotics in vitro, with special reference to species differences. Am J Med Sci 258:416-430, 1969 5. Shungu DL, Cornett JB, Shockman GD: Morphological and physiological study of autolytic<br><br> defective <span class="yellow">Streptococcus faecium</span> strains. J Bacteriol 138:598-608, 1979 <br><br> 6. Moellering RC Jr, Korzeniowski OM, Sande MA, et al: Species-specific resistance to antimicrobial <br><br> synergism in <span class="yellow">Streptococcusfaecium</span> and <span class="yellow">Streptococcusfaecalis</span>. J Infect Dis 140:203-208, 1979 <br><br> 7. Moellering RC Jr, Weinberg AN: Studies on antibiotic synergism against enterococci. 11. Effect of <br><br> various antibiotics on the uptake of '4C-labelled streptomycin by enterococci. J Clin Invest 50:2580-2584, 1971 <br><br> 8. Zimmermann RA, Moellering RC Jr, Weinberg AN: Mechanisms of resistance to antibiotic <br><br> synergism in enterococci. J Bacteriol 105:873-879, 1971 <br><br> 9. Moellering RC Jr, Murray BE, Schoenbaum SC, et al: A novel mechanism of resistance to penicillin<br><br> gentamicin synergism in <span class="yellow">S. faecalis</span>. J Infect Dis 141:81-86, 1980 <br><br> 10. Krogstad DJ, Korfhagen TR, Moellering RC Jr, et al: Plasmid-mediated resistance to antibiotic <br><br> synergism in enterococci. J Clin Invest 61:1645-1653, 1978 <br><br> 11. Krogstad DJ, Korfhagen TR, Moellering RC Jr, et al: Aminoglycoside-inactivating enzymes in <br><br> clinical isolates of <span class="yellow">Streptococcusfaecalis</span>: An explanation for resistance to antibiotic synergism. J Clin Invest 62:480-486, 1978 <br><br> <span class="yellow">S. FAECIUM</span> ENDOCARDITIS                 249 <br><br> 12. Courvalin PM, Shaw WV, Jacob AE: Plasmid-mediated mechanisms of resistance to <br><br> aminoglycoside-aminocyclitol antibiotics and to chloramphenicol in group D streptococci. Antimicrob Agents Chemother 13:716-725, 1978 <br><br> 13. Calderwood SA, Wennersten C, Moellering RC Jr, et al: Resistance to six aminoglycosidic <br><br> aminocyclitol antibiotics among enterococci: Prevalence, evolution, and relationship to synergism with penicillin. Antimicrob Agents Chemother 12:401-405, 1977 <br><br> 14. Bisno AL: Treatment of infective endocarditis. New York, Grune and Stratton, 1981, p 90 <br><br> 15. Lorian V: Effects of subminimum inhibitory concentrations of antibiotics on bacteria. In Antibiotics <br><br> in Laboratory Medicine. Edited by V Lorian. Baltimore, Williams and Wilkins, 1980, p 342 </body></html>