
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Xenopus laevis</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="yellow">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="yellow">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="yellow">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="blue">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="yellow">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="yellow">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="yellow">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="yellow">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="yellow">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="yellow">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="yellow">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="yellow">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="yellow">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="yellow">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="yellow">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="yellow">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="yellow">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="yellow">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="yellow">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="yellow">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="yellow">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="yellow">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="yellow">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="yellow">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="yellow">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="yellow">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="yellow">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="yellow">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="yellow">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="yellow">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="yellow">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA280693</h3>p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells and its expression is induced through pathways involved in growth inhibition and repressed by factors promoting cell growth
Abstract
Background
p8 is a stress-induced protein with multiple functions and biochemically related to the architectural factor HMG-I/Y. We analyzed the expression and function of p8 in pancreatic cancer-derived cells.<br><br>Methods
Expression of p8 was silenced in the <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 by infection with a retrovirus expressing p8 RNA in the antisense orientation. Cell growth was measured in control and p8-silenced cells. Influence on p8 expression of the induction of intracellular pathways promoting cellular growth or growth arrest was monitored.<br><br>Results
p8-silenced cells grew more rapidly than control cells transfected with the empty retrovirus. Activation of the Ras→Raf→MEK→ERK and JNK intracellular pathways down-regulated p8 expression. In addition, the MEK1/2 inhibitor U0126 and the JNK inhibitor SP600125 up-regulates expression of p8. Conversely, p38 or TGFβ-1 induced p8 expression whereas the specific p38 inhibitor SB203580 down-regulated p8 expression. Finally, TGFβ-1 induction was in part mediated through p38.<br><br>Conclusions
p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cells. p8 expression is induced through pathways involved in growth inhibition and repressed by factors that promote cell growth. These results suggest that p8 belongs to a pathway regulating the growth of pancreatic cancer cells.<br><br><br><br>Background
While studying the molecular response of the injured pancreas, we identified a new gene, called p8, whose expression is strongly induced during the acute phase of pancreatitis [1]. Further experiments have shown that p8 mRNA is activated in almost all cells in response to several stresses [2], including minimal stresses such as after routine change of the culture medium in the absence of any added substance [3], indicating that p8 is a ubiquitous protein induced by cellular stress. The p8 gene was cloned in <span class="yellow">human</span>, <span class="yellow">rat</span>, <span class="yellow">mouse</span>, and <span class="blue">Xenopus laevis</span> [1,4-6], conceptually translated from the <span class="yellow">Drosophila melanogaster</span> genome or deduced from EST libraries (<span class="yellow">Bos taurus</span>, <span class="yellow">Xenopus tropicalis</span>, <span class="yellow">Zebrafish</span>, <span class="yellow">Orzzias latipes</span>, <span class="yellow">Bombyx mori</span> and <span class="yellow">Paralichthys olivaceous</span>). The overall degree of homology with <span class="yellow">human</span> p8 ranged from 81 to 40%. Secondary structure prediction methods indicated that within the homologous region of the eleven proteins, there is a basic Helix-Loop-Helix secondary structure motif, characteristic of some classes of transcription factors [1]. Even though a small protein such as p8 would not need a nuclear localization signal (NLS) to be transported to the nucleus, a clear NLS can be predicted for the eleven proteins comprising a bipartite domain of positively charged aminoacids. In addition, a nuclear/cytoplasmic location has been demonstrated for <span class="yellow">human</span> p8 upon overexpression of the recombinant protein and immunohistochemistry [4], and for recombinant <span class="blue">Xenopus laevis</span> p8 fused to green fluorescent protein [6]. Homology searching in databases did not reveal significant similarity of p8 with other proteins of known function. However, biochemical properties of the mammalian p8 proteins are shared by some high mobility group proteins (HMG) [7], particularly by the HMG-I/Y family. The overall identity of <span class="yellow">human</span> p8 with <span class="yellow">human</span> HMG-I/Y is only about 35%, but the molecular mass, isoelectric point, hydrophilicity plot, the resistance to denaturation after heating at 100°C and the charge separation are very similar [8]. The p8 protein seems to bind DNA weakly, as shown by electrophoretic mobility shift assay, without preference for DNA sequences. Finally, <span class="yellow">human</span> p8 has also been shown to be a substrate for protein kinase A in vitro and phosphorylated p8 has a higher content of secondary structure and binding to DNA is highly increased [8]. An architectural role in transcription has been proposed for this protein, in analogy with the HMG-I/Y proteins, and a recent work seems to confirm this hypothesis [9].
Functions of p8 appear to be multiple and complex. For example, p8 mRNA expression was strongly induced in 3T3 cells upon TGFβ-1 treatment which in turn enhances the Smad-transactivating function responsible for TGFβ-1 activity [10]. We also found that p8 is involved in cell cycle regulation since p8-deficient embryonic fibroblasts grew more rapidly and incorporated more [3H] thymidine and BrdU than p8-expressing cells [11]. Moreover, expression of p8 in breast cancer-derived cells seems to mediate the inhibition of cell growth induced by 1,25-Dihydroxyvitamin D3 [12]. On the contrary, we also reported that p8 may promote cell growth when overexpressed in Cos-7, AR42J and HeLa cells [1,4]. In addition, p8 seems to be involved in other intracellular functions such as apoptosis since p8-expressing fibroblasts are more sensitive than p8-deficient fibroblasts to the apoptosis induced by DNA damage. Also, p8 is required for endothelin-induced mesangial cell hypertrophy in diabetic kidney, in a mechanism involving ERK, JNK and PI3 kinase [13]. p8 seems to play a functional role in the initiation of LHβ gene expression during embryonic cell differentiation [14]. Moreover, the <span class="yellow">Drosophila melanogaster</span> p8 homologue is involved in response to starvation and might be activated to stop cell growth in case of nutrient deprivation [15]. Finally, a particularly attractive role in tumour progression was recently proposed for p8 [16]. Fibroblasts obtained from p8-expressing or p8-deficient animals were transformed with a retroviral vector expressing both the rasV12 mutated protein and the E1A adenoviral oncogene. In soft-agar assays, transformed p8-expressing cells formed colonies at high frequency, as expected, but p8-deficient transformed fibroblasts were unable to form colonies. Similarly, transformed p8-expressing cells produced tumours in all athymic <span class="yellow">nude mice</span> when injected subcutaneously or intraperitoneally, whereas transformed p8-deficient fibroblasts did not. On the other hand, studies by another laboratory revealed that expression of the Com1 protein [17], which is identical to <span class="yellow">human</span> p8, mediates the growth of tumour cells after metastatic establishment in a secondary organ, indicating that activated expression of Com1/p8 in metastatic cells is required for tumour progression. These results strongly suggest that p8 is involved in the cellular pathway(s) required for tumour progression and metastasis.
Our aim is to check the relevance of p8 to cancer progression in <span class="yellow">human</span>. As a first step, we investigated in the present study the function of p8 in two cell lines derived from <span class="yellow">human</span> pancreatic cancer. We observed that inhibition of p8 expression increased the cells growth rate. In addition, activations of the Ras→Raf→MEK→ERK and JNK intracellular pathways, which promote the growth of pancreatic cells, down-regulated p8 expression, whereas activation of p38 or TGFβ-1, which inhibit cell growth, induced its expression. It was concluded that i/ p8 inhibits the growth of <span class="yellow">human</span> pancreatic cancer cell lines, ii/ p8 expression is induced through pathways involved in growth inhibition and, conversely, repressed by factors that promote cell growth.<br><br>Results
p8 is silenced in pancreatic cancer cells by infection with a retrovirus expressing p8 RNA in the antisense orientation
Panc-1 and BxPc-3 pancreatic cells were chosen for this study because, on the one hand, both cells express higher level of p8 (Figure 1) and, on the other hand, because Panc-1 is wild-type for Smad4/DPC4 and mutated for K-ras, while BxPc-3 is Smad4/DPC4 mutated and K-ras wild-type [18,19], therefore representing different mechanisms of transformation and different genetic backgrounds. K-ras and Smad4/DPC4 mutations are the major mechanisms involved in pancreatic cancer development. We inhibited p8 expression in both Panc-1 and BxPc-3 pancreatic cells by infecting cells with a retrovirus expressing the p8 asRNA (antisense RNA) and carrying the puromycin resistance. The antibiotic-selected cells were analyzed by Western blotting to evaluate the intracellular amounts of p8 protein. As shown in Figure 2, the p8 protein was clearly visible in both Panc-1 and BxPc-3 pancreatic cells infected with the empty retrovirus but almost undetectable in cells infected with the retrovirus encoding the p8 asRNA showed, indicating that our anti-sense strategy is efficient to silence p8 gene expression in pancreatic cancer cells. Preliminary studies had been conducted to select the best strategy to inhibit p8 expression. We compared the efficacy of the stable transfection of a siRNA, using a retroviral expression vector to the asRNA strategy described above. In our hands, the antisense strategy worked best, as judged from Western blot assessment of p8 protein expression (data not shown).<br><br>p8-silenced pancreatic cells grow more rapidly
We compared in the two cell lines the influence on growth parameters of blocking p8 expression with the p8 asRNA. Figure 3 shows that both Panc-1 and BxPc-3 cells in which p8 has been silenced grew more rapidly than cells infected with the empty vector suggesting that inhibition by p8 of pancreatic cancer cell growth is independent from the mechanism of transformation and genetic background.<br><br>Serum-stimulated cellular growth down-regulates p8 expression
Fetal <span class="yellow">calf</span> serum, which contains a complex mix of growth factors, can be used as inductor of cell growth. As shown in Figure 4 expression of p8 mRNA was down-regulated in both Panc-1 and BxPc-3 when the cells were shifted from culture media containing 0.1% fetal <span class="yellow">calf</span> serum to media containing 10% FCS. p8 protein showed a similar behavior. These results show that p8 expression is down-regulated in growing pancreatic cells.<br><br>The Ras→Raf→MEK→ERK pathway down-regulates p8 expression in pancreatic cancer cells
Most <span class="yellow">human</span> pancreatic cancers harbor mutations in the K-ras oncogene, which happens relatively early in pancreatic tumorigenesis [20]. The oncogenic mutation of the K-ras gene stabilizes the Ras protein in a GTP-bound form, which is constitutively active and make the cells grow more rapidly. Contrary to the activated Ras protein, p8 inhibits cell growth (Figure 3). We looked whether the Ras→Raf→MEK→ERK pathway was also involved in the regulation of p8 expression, and which step(s) were critical. Figure 5 shown that expression of a mutated form of the Ras protein (rasV12) in BxPc-3 cells, which are wild-type for ras, resulted in decreased p8 mRNA concentration and protein level suggesting that the activated ras inhibits p8 expression. Figure 6 shows that overexpression of Raf, but not of Raf301 (a negative mutant of Raf), and of ERK also inhibited the expression of the p8-CAT construct in Panc-1 as well as in BxPc-3. Finally, the MEK1/2 specific inhibitor U0126 [21] activated p8 mRNA expression in pancreatic cells whether they carry mutated ras (Panc-1) or wild-type (BxPc-3). Similar results were observed when expression of the p8 protein was monitored by Western blotting (Figure 7).<br><br>Activation of the JNK pathway down-regulates p8 expression in pancreatic cancer cells
c-Jun NH2-terminal kinase (JNK) is another major MAPK pathway which converts extracellular signals into expression of specific target genes through phosphorylation and activation of transcription factors. JNK activation has been implicated in various, often opposite cellular responses, such as cell proliferation, transformation and apoptosis. As shown in Figure 8, overexpression of JNK down-regulates the gene reporter activity of the p8-CAT construct in Panc-1 cells. Similar results were found in BxPc-3 cells. Treatment of these cells with the JNK specific inhibitor SP600125 [22] up-regulates expression of the p8 mRNA and p8 protein (Figure 9). These results show that the JNK pathway is involved in the regulation of p8 expression.<br><br>The p38 pathway up-regulates p8 expression in pancreatic cancer cells
The p38 signal transduction pathway also plays an essential role in regulating several cell functions including growth, response to inflammation, differentiation and apoptosis. In fact, in pancreatic cancer cells, p38 is a strong inhibitor of proliferation [23] contrary to the Ras→Raf→MEK→ERK and JNK pathways. We therefore analyzed the putative role of the p38 pathway in regulating p8 expression in pancreatic cancer cells. Figure 10 shows that over-expression of the plasmid encoding p38 significantly increases p8-CAT activity in Panc-1 as well as in BxPc-3 cells. Then, cells were treated with SB203580, a specific inhibitor of p38 [24], and p8 expression was measured. p8 mRNA as well as the encoded protein were down-regulated after inhibition of the p38 activity (Figure 11). These results indicate that the p38 pathway is a positive regulator of p8 expression in pancreatic cancer cells.<br><br>TGFβ-1 up-regulates p8 expression in pancreatic cancer cells
The most prominent biological activity of TGFβ-1 is its potent inhibition of cell growth in a wide variety of cell types including pancreatic cells. TGFβ-1 signals are sent through two types of transmembrane serine/threonine kinase receptors. In fact, TGFβ-1 binds and brings together the type I and type II receptors. In the resulting complex, the constitutively active TGFβ-1 type II receptor phosphorylates the type I receptor, which then plays a major role in transducing the signal to downstream components to affect gene expression through phosphorylation of SMAD proteins. Phosphorylated receptor-regulated SMADs then form heteromeric complexes with the common partner SMAD4. These heteromeric complexes then move to the nucleus, where SMAD4 will bind DNA and contribute to transcriptional activation. In general, pancreatic cancer cells present with defects in TGFβ-1 signaling and are resistant to TGFβ-1-mediated growth suppression. Since TGFβ-1 and p8 are inhibitors of pancreatic cell growth we analyzed whether p8 could mediate, at least in part, the effect of TGFβ-1. First, we found that treatment of Panc-1 cells with TGFβ-1 increased p8 mRNA levels and p8 protein as judged by Western blot (Figure 12). Then, to confirm that overexpression is regulated at the transcriptional level, we analyzed the effect of some constructs expressing constitutively activated type I TGFβ receptor, dominant negative type II TGFβ receptor, a dominant negative of Smad4 and the wild-type Smad4 on the p8-CAT activity. As expected, the constitutively activated type I TGFβ receptor but not the dominant negative type II TGFβ receptor increased CAT activity. Also, expression of the Smad4, contrary to that of the negative mutant, induced p8 transcription (Figure 13). Together, these results indicate that p8 is positively regulated by TGFβ-1.
Beside the Smad proteins, TGFβ-1 also activates the p38 MAPK pathway in pancreas-derived cells, which may play an important role in TGFβ-1 induced genes [25]. Therefore, we analyzed the p38-dependent effect of TGFβ-1 on p8 transcription. As shown in Figure 13, inhibition of p38 activity with the SB203580 specific inhibitor decreased about 40% the activity of TGFβ-1 on the p8 promoter indicating that the effect of TGFβ-1 on p8 promoter is mediated by both p38-dependent and p38-independent pathways.<br><br>
Discussion
Pancreatic adenocarcinoma is the fourth leading cause of death from malignant diseases [26]. The aggressive nature of the neoplasia, the lack of early detection, and the limited response to treatments such as chemotherapy and radiotherapy contribute to the high mortality rate of the disease. Therefore, a better understanding of the molecular mechanism leading to pancreatic cancer remains a major goal because it may help proposing strategies for earlier diagnosis and better treatment. The most commonly altered genes in pancreatic adenocarcinoma are K-ras (75 to 100%), p16INK4a (95%), p53 (50 to 75%) and DPC4 (50%) [27-31]. Whereas K-ras is a proto-oncogene all the others are tumour suppressor genes. Additional genes have been found altered at lower frequency. Panc-1 and BxPc-3 pancreatic cells were chosen for this study because they both express high levels of p8 (Figure 1) and because they present with different mechanisms of transformation and genetic backgrounds, Panc-1 being wild-type for Smad4/DPC4 but mutated for K-ras and BxPc-3 mutated for Smad4/DPC4 and wild-type for K-ras [18,19]. This work presents evidences that p8 inhibits the growth rate of pancreatic cancer-derived cells and that the intracellular pathways promoting cell growth down-regulate p8 expression whereas those promoting growth arrest up-regulate its expression. Together, these results suggest that p8 is downstream of some cell growth regulators and therefore regulation of p8 expression or its activity could be used as a target for treating pancreatic cancer.
Silencing p8 expression was able to strongly promote cell growth in both cell types, Panc-1 and BxPc-3, suggesting that p8 may act downstream of the ras- or Smad4/DPC4-dependent ways. Also, we found that stimulating cell growth by the complex combination of growth factors contained in fetal <span class="yellow">calf</span> serum down-regulated expression of p8 whereas, on the contrary, treating the cells with TGFβ-1, which promotes cell cycle arrest, stimulates p8 expression. Therefore, p8 gene expression seems to be regulated in opposite directions by mechanisms promoting cell growth or cell cycle arrest. It is interesting to note that while p8 expression is under the control of cell growth regulatory pathways such as Ras→Raf→MEK→ERK, JNK, p38 and TGFβ-1, p8 can affect cell cycle progression, suggesting that p8 is a target for factors regulating pancreatic cell growth.
A mechanism by which p8 could regulate cell cycle progression in embryonic fibroblasts was previously proposed [11]. In fact, p8 seems to take action upstream from cyclin-dependent kinases because the intracellular levels and activities of Cdk2 and Cdk4 are decreased when p8 is expressed. Concomitantly, the cyclin-dependent kinase inhibitor p27 is expressed at a low level in p8-deficient cells which may explain the increased activity of Cdk2 and Cdk4. The mechanism by which p8 regulates the intracellular level of those proteins remains to be determined. However, because p8 is a transcriptional cofactor, it is possible that regulation of expression of these molecules takes place, at least in part, at the transcription level.
Interestingly, expression of p8 mRNA seems to be regulated in a cell type- and stimulus-specific manner since, for example, p38 can induce p8 expression in response to stress in fibroblasts [3] but not in renal mesangial cells treated with endothelin [13]. In pancreatic cancer-derived cells p38 seems to play a major role since it is involved in p8 activation as judged by transient transfection assays and using a specific p38 inhibitor (Figures 10 and 11). In addition, p38 is also involved in TGFβ-1-induced p8 expression because about 40% of the TGFβ-1 effect was abolished when p38 activity was specifically blocked (Figure 13). On the other hand, ERK and JNK are inducers of p8 expression in mesangial cells treated with endothelin, but not involved in the activation of p8 in response to stress in fibroblasts [3], and even repressors in pancreatic cells (Figures 5, 6, 7, 8 and 9). Finally, PI3 kinase is an inducer of p8 expression in both endothelin-mediated p8 activation in mesangial cells [13] and pancreatic cells (data not shown).
Based on these observations, overexpression of p8 could be considered a possible goal for treating pancreatic tumours, in order to limit their growth. However, we previously reported that p8 repression would prevent rasV12/E1A transformed fibroblasts from evolving as tumours in <span class="yellow">nude mice</span> [16]. This apparent contradiction needs to be resolved before considering p8 as a target for treating cancer progression.<br><br>Conclusions
In conclusion (see Figure 14), we report in this paper that inhibition of p8 expression by an anti-sense strategy increases the growth rate of both Panc-1 and BxPc-3 pancreatic cancer-derived cells. Moreover, ERK- and JNK-mediated pathways down-regulate p8 expression, whereas p38 and TGFβ-1 pathways induce p8 expression. Also, cell growth triggered by expression of a RAS mutated protein or by 10% fetal <span class="yellow">calf</span> serum induces down-regulation of p8 expression. Together, these results indicate that p8 is an intracellular cell growth inhibitor and that it is oppositely regulated by growth-promoting or growth-inhibiting factors in pancreatic cancer-derived cells.<br><br>Material and Methods
Cell lines and cell culture conditions
The <span class="yellow">human</span> pancreatic cancer cell lines Panc-1 and BxPc-3 were a kind gift of Dr C. Susini (INSERM U.531, Toulouse) and A. Hajri (IRCAD, Strasbourg) respectively. Panc-1 cells were grown in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal <span class="yellow">calf</span> serum, 2 mM L-glutamine, 100 IU/ml penicilin G and 100 μg/ml streptomycin. BxPc-3 were cultivated in RPMI 1640 medium in the presence of 2 mM L-glutamine, 4.5 g/L glucose, 10 mM Hepes, 1.0 mM sodium pyruvate, 10% fetal <span class="yellow">calf</span> serum and 100 IU/ml penicilin G and 100 μg/ml streptomycin. <span class="yellow">Human</span> recombinant TGFβ-1 was obtained from Sigma, and specific SB203580, U0126 and SP600125 inhibitors were from Calbiochem and utilized at 10 μM.<br><br>Expression plasmids
Expression plamids encoding p38 (pCEFL HA p38), Erk2 (pcDNAIII HA ERK2), JNK (pcDNAIIIB HA JNK), the wild-type Raf (pcDNA RAF BXB) and the Raf dominant negative (pcDNA RAF 301 K375W) were obtained from O Coso (University of Buenos Aires). Plamids encoding the constitutively activated type I TGFβ receptor (RI ACT), the dominant negative type II TGFβ receptor (RII DN) and the Smad4 dominant negative (DPC4 1–514 a.a.) were obtained from R Urrutia (Mayo Clinic, Rochester) and the wild type Smad4 was from C Heldin (Ludwig Institute, Uppsala).<br><br>Pancreatic p8-deficient cells
To silence p8 expression in pancreatic cells, we infected these cells with a retrovirus expressing <span class="yellow">human</span> p8 in the antisense orientation. The retroviral vector was constructed as follows: <span class="yellow">human</span> p8 cDNA was subcloned in HindIII and XhoI restriction sites of the pLPC plasmid (obtained from S. Lowe) in the antisense orientation. Amphotrope <span class="yellow">human</span> p8 expressing retrovirus was then generated by transient transfection using Phoenix amphotrope packaging cells. Viral supernatant was used to infect Panc-1 and BxPc-3 pancreatic cells and the population of p8-silenced cells was isolated by selection in presence of puromycin (1 μg/ml). As control, cells were infected with the pLPC empty vector.<br><br>p8 expression in arrested and growing cells
One million of Panc-1 or BxPc-3 cells were cultivated on 10-cm Petri plates in standard conditions (with 10% FCS). After 48 h, culture media were changed for fresh media with FCS restricted to 0.1%, in order to stop growth. After 24 hours of growth arrest, culture medium was replaced either by medium with 10% fetal <span class="yellow">calf</span> serum to resume cell growth or, as control, by medium with 0.1% fetal <span class="yellow">calf</span> serum. Twenty four hours later cells were recovered and RNA and protein extracted.<br><br>p8 expression in TGFβ-1-treated Panc-1 cells
One million of Panc-1 cells were cultivated in 10-cm culture dishes for 48 hours under standard conditions before TGFβ-1 treatment. <span class="yellow">Human</span> recombinant TGFβ-1 (5 ng/ml) was added to cells, without changing the culture medium, and cells were collected 12 hours later for RNA and protein preparation.<br><br>BxPc-3 rasV12-expressing cells
pLPC-rasV12 and pLPC plasmids were obtained from S. Lowe. Phoenix amphotrope packaging cells (106) were plated in a 6-well plate, incubated for 24 hours, then transfected with PEI with 5 μg of retroviral plasmid. After 48 hours, the medium containing virus was filtered (0.45 μm filter, Millipore) to obtain the viral supernatant. Target BxPc-3 were plated at 2 × 105 cells per 35-mm dish and incubated overnight. For infections, the culture medium was replaced by an appropriate mix of the viral supernatant and culture medium (V/V), supplemented with 4 μg/ml polybrene (Sigma), and cells were incubated at 37°C. BxPc-3 rasV12-expressing cells were selected with puromycin (1 μg/ml). Cells infected with the pLPC empty vector were used as control.<br><br>Western-blot analyses
One hundred μg of total protein extracted from cells was separated with standard procedures on 15.0% SDS-PAGE using the Mini Protean System (Bio-Rad) and transferred to a nitrocellulose membrane (Sigma). The intracellular level of p8 was estimated by Western blot using a polyclonal antibody (1:1000) raised against <span class="yellow">human</span> p8 [4].<br><br>Growth curves
One hundred thousand cells per well were plated in a series of 35-mm culture dishes. The cell number was estimated daily in triplicate, during 1 to 5 days, in a haemocytometer. Within experiments, each point was determined at least two times.<br><br>Cell transfection and gene reporter assays
Panc-1 and BxPc-3 (105) were cultivated in 30 mm diameter culture dishes for 24 hours then transiently transfected with 0.5 μg of p8-CAT reporter plasmid and 0.5 μg of pCMV/βgal plasmid (to control transfection efficiency) using the Fugene reagent in accordance with the manufacturer's protocol (Roche Molecular Biochemicals). The p8-CAT plasmid is the previously reported p-1471/+37p8-CAT promoter construct [5]. Reporter activities were measured as previously described [5]. Briefly, cell extracts were prepared with the reporter lysis buffer (Promega) 24 hours after transfection and CAT activity was determined by the phase extraction procedure [32] and β-galactosidase assay was performed essentially as described in Sambrook et al. [33]. CAT activity was normalized to β-galactosidase activity. Experiments were carried out in triplicate and repeated at least two times. Expression plasmids (0.5 μg) were co-transfected with p8-CAT and pCMV/βgal plasmids as indicated.<br><br>RT-PCR analysis
RNA was extracted using the Trizol (Life Technologies) procedure. Total RNA (1 μg) was analyzed by RT-PCR with the SuperScript™ One-step RT-PCR System and the Platinum Taq kit (Life Technologies). RT-PCR was performed using different numbers of cycles to verify that the conditions chosen were within the linear range. The mRNA coding for p8 was specifically amplified with sense (5' GAAGAGAGGCAGGGAAGACA 3') and antisense (5' CTGCCGTGCGTGTCTATTTA 3') primers, in positions 72 and 643 of the cDNA (accession # NM_012385), respectively. As control, the transcript coding for the ribosomal protein RL3 was specifically amplified for 22 cycles with sense (5'GAAAGAAGTCGTGGAGGCTG3') and antisense (5'ATCTCATCCTGCCCAAACAC3') primers, in positions 216 and 637 of the cDNA, respectively.<br><br>
Author's contributions
CM prepared cells and retrovirus, carried out RNA purification, RT-PCR, Western blots, and cell growth experiments, NL carried out CAT assays, SV participated in the design of the study and analysis of data, JLI participated in the analysis of data and wrote the manuscript. All authors read and approved the final manuscript.<br><br>
</body></html>