
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>LINNAEUS inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>yeast</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>pmcA102792</h3>Scp160p, a multiple KH-domain protein, is a component of mRNP complexes in <span class="blue">yeast</span>
Abstract
Scp160p is a 160 kDa protein in the <span class="blue">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> that contains 14 repeats of the hnRNP K-homology (KH) domain, and demonstrates significant sequence homology to a family of proteins collectively known as vigilins. As a first step towards defining the function of Scp160p, we have characterized the subcellular distribution and in vivo interactions of this protein. Using sucrose gradient fractionation studies we have demonstrated that Scp160p in cytoplasmic lysates is predominantly associated with polyribosomes. Furthermore, we have found that Scp160p is released from polyribosomes by EDTA in the form of a large complex of ≥1300 kDa that is sensitive both to RNase and NaCl. Using affinity-chromatography to isolate these complexes, we have identified two protein components other than Scp160p: poly(A) binding protein, Pab1p, and Bfr1p. The presence of Pab1p confirms these complexes to be mRNPs. The presence of Bfr1p is intriguing because the null phenotype for this gene is essentially the same as that reported for scp160-null cells: increased cell size and aberrant DNA content. These results demonstrate that Scp160p associates with polyribosome-bound mRNP complexes in vivo, implicating a role for this protein in one or more levels of mRNA metabolism in <span class="blue">yeast</span>.<br><br>INTRODUCTION
Scp160p is a 1222 amino acid <span class="yellow">Saccharomyces cerevisiae</span> protein that contains 14 copies of the hnRNP K homology (KH) domain, a highly conserved motif found in many proteins involved in RNA metabolism (1). KH domain-containing proteins appear to have diverse functions and have been identified in all kingdoms of life, including the ribosomal protein S3 from <span class="yellow">Escherichia coli</span> (1), Mer1p from <span class="yellow">S.cerevisiae</span>, a meiosis-specific splicing factor (1), MEX-3 from <span class="yellow">Caenorhabditis elegans</span>, presumably involved in mRNA localization during development (2), and FMRP, the fragile-X mental retardation protein in <span class="yellow">humans</span> (3). A partial clone of Scp160p, known as HX, was one of the first multiple-KH proteins identified (1).
Whole cell immunofluorescence studies have demonstrated that Scp160p localizes to the cytoplasm, with enrichment around the nuclear envelope, and what appears to be the endoplasmic reticulum (3). Deletion of the SCP160 locus in <span class="blue">yeast</span> is not lethal, but results in a complex phenotype, including increased DNA content per cell, missegregation of genetic markers during sporulation, and abnormal cell morphology, including increased size and irregular shape (4). Observation of this phenotype led to the hypothesis that Scp160p may function in regulating ploidy during cell division. More recently, Weber and colleagues demonstrated in vitro RNA binding activity of Scp160p using northwestern blot analyses; the protein was found to bind efficiently to ribohomopolymers and rRNA, but not to tRNA (5). Cell fractionation studies revealed that a large percentage of Scp160p associates with membrane-pellets, and is released by treatment with either 10 mM EDTA or 500 mM NaCl (5). While these authors interpreted these results to suggest that the nuclear envelope/ER localization of Scp160p was due primarily to interactions of the protein with membrane-bound polyribosomes, clear evidence of this association has not been reported (5). Currently, the relationship between the phenotype of scp160 null mutants and the RNA-binding activity of Scp160p remains unclear.
Scp160p demonstrates significant sequence homology (~23% identity and ~40% similarity at the amino acid level) to a class of vertebrate KH-domain proteins collectively known as vigilins. First identified in <span class="yellow">chicken</span>, vigilin homologues have now been found in <span class="yellow">human</span> (6), <span class="yellow">Xenopus laevis</span> (7), <span class="yellow">Drosophila melanogaster</span> (8), <span class="yellow">C.elegans</span> (5) and <span class="yellow">Schizosaccharomyces pombe</span>. While all of the vigilin proteins studied to date are reported to bind nucleic acid, both the type of nucleic acid bound and the functional significance of these interactions remain unclear. For example, Kruse and colleagues reported from their work with <span class="yellow">human</span> cells in culture that vigilin may be involved in the binding and transport of tRNA (9–11). In contrast, Dodson and Shapiro concluded from their work with Xenopus vigilin that, in response to estrogen, the protein bound specifically to the 3′ UTR of vitellogenin mRNA (7,12), potentially stabilizing the message (13). Lastly, DDP1, the <span class="yellow">Drosophila</span> homolog of vigilin, was reported recently to interact with the dodeca-satellite repeat regions of centromeric heterochromatin in embryonic and larval cell nuclei, suggesting a possible role for this protein in heterochromatin structure (8).
The goal of the present study was to begin elucidating the function of Scp160p in <span class="blue">yeast</span> by characterizing the subcellular distribution and macromolecular interactions of this protein. We have demonstrated that Scp160p in cytoplasmic lysates is associated predominantly with polyribosomes, and that following treatment with EDTA, Scp160p remains in an RNase/NaCl-sensitive complex of apparent molecular weight approximately ≥1300 kDa. Affinity purification of this complex revealed the presence of poly(A)-binding protein (Pab1p), a well-characterized component of eukaryotic mRNPs (14,15). Finally, a third abundant protein component of this complex was identified as Bfr1p, a protein not previously reported to associate with mRNPs. While the function of Bfr1p remains unknown, gene deletion reportedly leads to a phenotype remarkably similar to that of scp160 deletion (16). These results indicate a role for Scp160p in mRNA metabolism in <span class="blue">yeast</span>, and by extension, support results seen with Xenopus vigilin in its interactions with mRNA. To our knowledge, the data reported here demonstrate Scp160p to be the first example of a KH-domain protein that functions as a component of polyribosome-associated mRNP complexes in the <span class="blue">yeast</span>, <span class="yellow">S.cerevisiae</span>.<br><br>MATERIALS AND METHODS
Plasmids, <span class="blue">yeast</span> strains and culture conditions
All recombinant DNA manipulations were performed according to standard techniques and utilized <span class="yellow">E.coli</span> strain XL1-Blue (Stratagene). The <span class="blue">yeast</span> strain JFy1511, expressing FLAG–Scp160p, was derived by two-step gene replacement from strain yJJ52 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112; generously donated by Drs Mark Parthun and Judith Jaehning, University of Colorado Health Sciences Center), and confirmed by PCR. The wild-type SCP160 coding sequence was obtained by PCR amplification from a genomic DNA preparation from yJFK1 (MATα gal7Δ102 ura3-52 trp1-289 ade1 lys1 leu2-3, 112 ΔGAL80::URA3), using a 16:1 mixture of Taq (Fisher Biotech) and Pfu (Stratagene) DNA polymerases and the following primers: Scp160F1 (5′-GCCGGTCGA-CTAACTGCAATGTCTGAAGAACAAACCGCTATTG-3′) and Scp160R1 (5′-GCGCGTCGACGAGCTTGTCTATCTT-CTTAAGG-3′). A wild-type SCP160 genomic clone, containing 1 kb upstream and 300 bp downstream sequence was PCR amplified from yJFK1 genomic DNA using the primers Scp160F0 (5′-GCCGAGCTCACACCAGCTTTGTCCTGG-3′) and Scp160R2 (5′-GCGCAAGCTTGTGCGGTA-TCCCAGTCTATG-3′). The resultant clone was confirmed by dideoxy sequencing. The N-terminal FLAG and HA tags were added using PCR with the primers ScpFLAGF1(5′-CCAT-TATAACTGCAATGGACTACAAGGACGACGACGACGAC-AAGATGTCTGAAGAACAAACCGCTATTG-3′) and ScpHAF1 (5′-CCCCCTCCTGTCGACATTATAACTGCAATGCACCA-TCACCATCACCATTCTGAAGAACAAACCGCTATTG-3′) respectively. For integration, constructs containing the 1 kb upstream and 300 bp downstream sequence were subcloned into the plasmid YIplac211 (17) using SacI and HindIII restriction sites. The HA-tagged construct was subcloned into the low copy number plasmid, YCplac22 using SacI and HindIII restriction sites. The SCP160 deletion construct was made by removal of a 3.4 kb ApaI–KpnI fragment from the coding region of this construct, followed by treatment with Klenow fragment and re-ligation following attachment of BamHI linker oligonucleotides. Genomic integration of the plasmid sequences and subsequent removal of the endogenous SCP160 allele were achieved by standard two-step gene replacement techniques (18) and confirmed by PCR. The N-terminally HA-tagged allele of BFR1 was generated by PCR-amplifiction of the BFR1 locus from wild-type (W303) <span class="blue">yeast</span> genomic DNA using the primers BFR1HAF1 (5′-CCGCGGATCCATGTACC-CATACGACGTCCCAGACTACGCTATGTCCTCCCAAC-AACACAA-3′) and BFR1HINDR1 (5′-CCGCAAGCTTGTCG-ACTATTTCATATGCCACAGGAAACAG-3′), and subcloned into YIPlac211. The BFR1 promoter region was PCR-amplified in a similar manner using the primers BFR1SACF1 (5′-CCGCGAG-CTCAGCATTAAGCATTCACGAGC-3′) and BFR1BAMR1 (5′-CCGCGGATCCGGCAATGGCTGTGTTGTTAGA-3′) and subcloned into the appropriate position upstream of the HA–Bfr1p open reading frame in the plasmid backbone. The entire open reading frame was confirmed by dideoxy sequencing. Finally, the HA–BFR1 allele was substituted into the <span class="blue">yeast</span> genome in place of the native allele using linearization with SphI and standard two-step gene replacement techniques. All <span class="blue">yeast</span> transformations and culture manipulations were performed according to standard protocols as described elsewhere (19).<br><br>Confirmation of genomic integrations
All genomic integrations were confirmed by PCR amplifications from purified <span class="blue">yeast</span> genomic DNA. For scp160 deletion mutants, the primers Scp160PF (5′-GATTTCCTAACTTTCC-GTCTA-3′) and Scp160R5 (5′-GCGCAAGCTTCACCGCCTTATAACGAAGAC-3′) that flanked the deleted region were used.
Similarly, epitope tags on Scp160p were confirmed by PCR using primers that flanked the tag sequence. Positive clones were further confirmed by western blot analysis of crude cell lysates using the appropriate anti-tag antibodies. The presence of HA–Bfr1p in cells was confirmed by western blot analysis of soluble cell lysates using 12CA5 mAb (Boehringer Mannheim).<br><br>Polyribosome isolation
Polyribosomes were isolated using a combination of protocols described by Stansfield and colleagues (20) and by Dr Maurice Swanson (personal communication). In brief, a 100 ml culture of <span class="blue">yeast</span> was grown in either YEPD or Hartwell synthetic medium, to early-log phase (OD600 = 1.0), at which time cyclohexamide (Sigma) was added directly to the culture to a final concentration of 100 µg/ml. The culture was incubated on ice for 15 min, and cells were harvested by centrifugation (4000 r.p.m./10 min). Following two washes in 10 ml of water containing 100 µg/ml cyclohexamide, cells were lysed by vortex agitation with an equal volume of glass beads in 1 ml lysis buffer (25 mM Tris pH 7.2, 50 mM KCl, 30 mM MgCl2, 5 mM β-mercaptoethanol, 200 µg/ml cyclohexamide, 2 µg/ml aprotonin, 1 mM PMSF, 0.5 µg/ml leupeptin, 2.9 µg/ml E64, 1 µg/ml antipain, 0.2 µg/ml chymostatin). Lysate was transferred to a clean microfuge tube, and centrifuged 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and then assayed for absorbance at 260 nm.
Approximately 12 OD260 units were loaded onto a 11 ml 15–45% sucrose gradient made in 10 mM Tris, pH 7.4, 70 mM NH4Cl, 4 mM MgOAc, using a Gradient Master automatic system, and the gradient was centrifuged in a SW41ti rotor (Beckman) at 39 000 r.p.m. for 2.5 h. 0.5 ml fractions were collected using an Isco gradient fractionator, and gradient profiles were determined by monitoring absorbance at 254 nm. For EDTA controls, lysis buffer containing 5 mM MgCl2 was used, and 30 mM EDTA was added to the sample before loading onto the gradient. Where indicated, addition of 50 U/ml of RNase One (Promega) was performed prior to loading the sample onto the gradient.<br><br>Gel filtration chromatography
Gel filtration chromatography was performed using a 120 ml Hi-Prep S-300 Sephacryl column (Pharmacia) with a cut-off of 1300 kDa, attached to an FPLC system (Pharmacia). The column had been calibrated previously using the following molecular weight standards (Sigma): blue dextran (2000 kDa), thyroglobulin (669 kDa), apoferritin (443 kDa), alcohol dehydrogenase (150 kDa), and <span class="yellow">bovine</span> serum albumin (66 kDa). <span class="yellow">Yeast</span> were lysed as described for polyribosome analysis, with an additional clarification by passage through a 0.2 µm syringe-tip filter (Acrodisc) and run over the column at a rate of 0.5 ml/min in polyribosome lysis buffer following treatment with the indicated reagents. Fractions (2.0 ml) were collected, from which 12 µl were combined with sample buffer (2% SDS, 10% glycerol, 100 mM dithiothreitol, 60 mM Tris pH 6.8, 0.001% bromophenol blue) and analyzed by western blot using the indicated antibodies.<br><br>α-FLAG affinity chromatography
For most experiments, one liter <span class="blue">yeast</span> cultures were grown to early log-phase and harvested by centrifugation. Cells were washed twice in T75 buffer (25 mM Tris pH 7.5, 75 mM NaCl) and then were lysed by vortex agitation with an equal volume of glass beads in 4 ml T75 buffer containing 30 mM EDTA. Lysate was transferred to a clean microfuge tube, and centrifuged for 10 min at 3000 g at 4°C. The supernatant was again transferred to a clean microfuge tube and centrifuged at 12 000 g for 15 min at 4°C, and finally passed through a 0.2 µm syringe filter. Lysate was then pre-purified by running over the S-300 gel-filtration column in T75 buffer, with pooling of the void volume fractions (~10 ml total). For several experiments (Fig. 5A, B, E and F), a low concentration (<10 µg/ml) of FLAG peptide (N-DYKDDDDK-C) was added to this sample. This void material was then loaded onto a 1 ml M2 α-FLAG column (Sigma); column flow-through was passed over the column a second time. The column was then washed extensively with 75 ml T75 buffer; material bound to the column was eluted with 5 ml T75 containing 184 mg/ml FLAG peptide. The peptide solution was allowed to incubate on the column for 20 min prior to collection of the first 1 ml fraction; subsequent fractions were collected every 10 min. Samples of crude material, S-300 void, α-FLAG flow-through, first and last wash, and eluate fractions were analyzed by western blot using the indicated antibodies. For Figure 5A and E, samples of first and last wash and eluate fractions were concentrated, run on SDS–PAGE and stained with colloidal G250 Coomassie.<br><br>Concentration of samples
Where indicated, samples were concentrated using the method of Traub et al. (21). 400 µl of sample were transferred to a microfuge tube, and combined with 400 µl of methanol, and 100 µl of chloroform. Samples were then vortexed vigorously, and centrifuged for 5 min at 12 000 g. The supernatant was discarded, leaving the interface intact, and an additional 400 µl of methanol was added to each tube. Samples were then inverted several times, and centrifuged again for 5 min. The supernatant was discarded, and the protein pellet was air-dried and resuspended in 1× sample buffer.<br><br>Western blot analysis
Western blot analysis was performed essentially as described previously (19). Briefly, samples to be analyzed were mixed with sample buffer, boiled, electrophoresed through a 10% SDS–polyacrylamide gel, and electro-blotted onto nitrocellulose (Bio-Rad). FLAG–Scp160p fusion protein was detected by incubation of the filter with <span class="yellow">mouse</span> M2 anti-FLAG monoclonal antibody (10 µg/ml final concentration), followed by HRP-conjugated <span class="yellow">sheep</span> anti-<span class="yellow">mouse</span> secondary antibody (Amersham), diluted 1:5000 as per the manufacturer’s instructions, and ECL reagent (Amersham), followed by exposure to X-ray film. HA-tagged Scp160p and HA-tagged Bfr1p were detected using the 12CA5 mAb (Boehringer Mannheim) at a final concentration of 0.8 µg/ml; Pab1p and Pub1p were detected using 1G1 mAb at 1:5000, and 4C3 mAb at 1:1000, respectively, both generous gifts from Dr Maurice Swanson (22,23). Where appropriate, films were analyzed by scanning densitometry (Molecular Dynamics), and quantitated using ImageQuant software (Molecular Dynamics).<br><br>Colloidal G250 coomassie staining
Procedure used was that of Neuhoff (24). Briefly, a one liter stock of staining solution was prepared containing 1 g Coomassie brilliant blue G-250 (Sigma), 100 g ammonium sulfate (Sigma), and 11.76 ml 85% phosphoric acid (Fisher). Following SDS–PAGE, gels were fixed in 40% methanol, 10% acetic acid for 10 min. Gels were then rinsed several times in water, then stained using 40 ml of the stock staining solution mixed with 10 ml methanol, for 2 h at room temperature. Stain was then poured off, and residual stain was removed by rinsing in water. By performing a standard analysis using <span class="yellow">bovine</span> serum albumin, the stain was able to detect ~15 ng total protein per lane.<br><br>Pichia expression system
An N-terminally HIS6/FLAG-tagged allele of the SCP160 coding sequence was blunt-end sub-cloned into the BamHI/SnaBI sites of the Pichia expression vector pPIC3.5K (Invitrogen). The construct was then linearized using SalI and integrated into the genome of the Pichia strain GS115 (Invitrogen). High expression transformants were selected initially on histidine-deficient medium followed by selection on increasing concentrations of G418 (US Biological). Crude lysates of the resultant transformants were then confirmed by western blot analysis with the anti-FLAG antibody M2, and the best expressing strain was cultured in a fermenter, harvested, and lysed by agitation with glass beads. To purify Scp160p, 2 ml of crude lysate was first diluted to 10 ml with 25 mM Tris, pH 7.5, 1 M NaCl, then twice passed over a 1 ml α-FLAG affinity column. The column was then washed with 75 ml of 25 mM Tris, pH 7.5, 1 M NaCl, and then eluted with 184 µg/ml FLAG peptide prepared in the same buffer.<br><br>
RESULTS
Expression of tagged Scp160p in <span class="blue">yeast</span>
An N-terminal FLAG-tagged form of Scp160p was created to facilitate detection of the protein in cells and extracts (Fig. 1A). To probe functionality of this fusion protein, we used two-step gene replacement (18) to substitute the modified allele into the SCP160 locus of haploid <span class="blue">yeast</span>, and then tested the morphological phenotype of the resultant cells. All strains were confirmed by PCR analysis of the SCP160 locus with appropriate primers (Materials and Methods), and expression of the tagged protein was confirmed by western blot analysis with the appropriate antibody (M2αFLAG) (Fig. 1B). In all cases, <span class="blue">yeast</span> expressing tagged Scp160p in place of the native protein appeared indistinguishable from the corresponding wild-type strains (data not shown). As a negative control, we also deleted almost the entire SCP160 coding region from the genomes of these <span class="blue">yeast</span> strains (Fig. 1A), and confirmed the expected mutant phenotype (data not shown).<br><br>Scp160p associates with polyribosomes
To test the hypothesis that Scp160p associates with polyribosomes we used sucrose gradient ultracentrifugation to size-fractionate subcellular components of lysates prepared from <span class="blue">yeast</span> expressing the N-terminal FLAG-tagged Scp160p protein. Western blot analyses of gradient fractions with an α-FLAG antibody (M2, Boehringer Mannheim), revealed a 160 kDa band that was most abundant in the denser fractions, consistent with the location of polyribosomes (>80S) (Fig. 2A, solid arrow). A convenient internal control for these gradients was provided by an unknown endogenous <span class="blue">yeast</span> cross-reacting protein at ~100 kDa, that exemplified the migration pattern of a free protein, appearing only in the upper fractions of the gradient (Fig. 2A, open arrow).
To determine whether the migration pattern of Scp160p in these sucrose gradients truly reflected association with <span class="blue">yeast</span> polyribosomes, lysates were pre-treated with either 30 mM EDTA or 50 U/ml RNase One (Promega) immediately prior to sucrose gradient fractionation. EDTA chelates Mg2+ cations, resulting in the dissociation of the small and large ribosomal subunits, reflected in gradient profiles (OD254) by the disappearance of 80S monosomes and polyribosomes along with a marked increase in the abundance of free ribosomal subunits (Fig. 2B). Under these conditions, the greatest intensity of FLAG–Scp160p signal was seen only in the upper-most fractions of the gradient (Fig. 2B). Alternatively, pre-treatment of lysates with RNase, which results in inter-ribosomal severing of translating messages, gave rise to a large pool of single 80S ribosomes (Fig. 2C). Again, FLAG–Scp160p was shifted to the uppermost fractions of these gradients. Although some of the Scp160p signal was detected in fractions larger than 40S, the migration pattern of the 100 kDa cross-reacting protein in these experiments indicated that RNase treatment caused an apparent diffusion of material in the upper portion of the gradient.<br><br>Characterization of Scp160p-containing complexes following EDTA and RNase treatment
Gel-filtration chromatography was used to determine the apparent molecular weight of Scp160p following its release from polyribosomes by both EDTA and RNase treatment. Lysates were prepared as described above for sucrose gradient analysis, but were instead size fractionated over a Sephacryl S-300 Hi-Prep column, with an inclusion cut-off of 1300 kDa. As expected, Scp160p from untreated lysates eluted in the void volume, confirming its association with large complexes, ostensibly polyribosomes (Fig. 3A, solid arrow). In EDTA treated lysates, all Scp160p signal was still detected in the void volume, indicating it remained in a complex of >1300 kDa (Fig. 3B, solid arrow). In contrast, limited RNase treatment (10 min) resulted in the appearance of an Scp160p-containing species of ~450 kDa, in addition to a fraction still visible in the void (Fig. 3C, solid arrow). More extensive RNase treatment (30 min) led to complete conversion to the 450 kDa species (Fig. 3D, solid arrow). Sequential treatment, first with RNase, then with EDTA, also resulted in a 450 kDa species (data not shown), suggesting that all components of this apparent 450 kDa complex were also present following EDTA treatment alone. As before, the 100 kDa, endogenous α-FLAG cross-reacting protein (Fig. 3, open arrows) served as a convenient internal control, eluting from the column at a volume consistent with its expected monomeric size.
To characterize further the stability of the large (apparent molecular weight >1300 kDa), EDTA-resistant complex, lysates were treated with increasing concentrations of NaCl prior to size fractionation (Fig. 3, panels E, F and G). As shown in Figure 3E, at 75 mM NaCl, the large complex remained intact. However, at a NaCl concentration of 150 mM, the Scp160p complex was partially reduced to 450 kDa, with some signal still remaining in the void (Fig. 3F, solid arrow). Following treatment with 1 M NaCl, Scp160p was only visible as the 450 kDa species (Fig. 3G, solid arrow). The fact that RNase treatment and high salt both generated Scp160p species of similar apparent molecular weight suggests that these high salt concentrations resulted in the disassociation of RNA, and perhaps other components, from Scp160p.<br><br>Pab1p and Bfr1p are present in EDTA-resistant Scp160p-containing complexes
To determine if the >1300 kDa Scp160p-containing complexes remaining after EDTA treatment were mRNPs, we assayed for the presence of the <span class="blue">yeast</span> poly(A) binding protein, Pab1p, following α-FLAG affinity purification as illustrated in Figure 4. Pab1p is an abundant and well-characterized component of mRNP complexes in <span class="blue">yeast</span>, as well as higher eukaryotes (14,15,25). As seen in Figure 5A and B, Pab1p did co-purify with FLAG–Scp160p; moreover, treatment with RNase immediately prior to FLAG-purification completely abolished this interaction (Fig. 5C), indicating an RNA-dependant association between these two proteins.
We also probed the isolated complexes for the presence of another abundant mRNP-component protein, Pub1p (23,26). Pub1p, as opposed to Pab1p, is not reported to associate with polyribosomes, and is hypothesized to bind a pool of non-translatable mRNAs (23). As predicted, Pub1p did not co-purify with Scp160p (Fig. 5D). The absence of Pub1p in Scp160p-containing complexes served as a negative control, demonstrating that abundant RNA-binding proteins did not co-purify non-specifically by this protocol. In addition, we performed mock purifications using lysates from <span class="blue">yeast</span> expressing the wild-type allele of SCP160 without the FLAG epitope (Fig. 5F), or with an unrelated FLAG-tagged protein, <span class="yellow">human</span> galactose 1-phosphate uridylyltransferase (GALT) (Fig. 5G, lower panel). In neither case did Pab1p bind and elute from the FLAG affinity column. Finally, no bands were visible by colloidal G250 Coomassie staining of samples from the mock (wild-type) preparations (Fig. 5E), further demonstrating the specificity of this procedure.
By colloidal G250 Coomassie staining (Materials and Methods) of the FLAG-purified complex, we observed in addition to Scp160p, two major bands, at ~70 kDa and ~55 kDa, which ostensibly represent co-purifying proteins (Fig. 5A, bottom panel). Western blot analysis suggested that the ~70 kDa species is Pab1p. To identify the 55 kDa protein, that band was excised from the gel, and sent to the Keck microchemical facility at Yale University for analysis by in gel tryptic digestion, followed by MALDI-mass spectrometry. The results of these analyses clearly identified the unknown protein as Bfr1p. To confirm this result, an N-terminal HA tag was engineered onto the genomic BFR1 locus in strains expressing FLAG-tagged or wild-type Scp160p. Scp160p complexes were then isolated from both of these strains using the protocol described above. As seen in Figure 6, an HA-tagged protein of ~55 kDa (center panel) appears in α-FLAG column elutions only when FLAG–Scp160p is also present (top panel). No signal is visible in mock purifications of extracts where Scp160p is not FLAG-tagged (bottom panel).<br><br>Monomeric Scp160p migrates as a ~450 kDa protein under native conditions
As a first step to characterize the post RNase/NaCl 450 kDa Scp160p species we compared it with Scp160p over-expressed and purified from an exogenous host, the <span class="blue">yeast</span> <span class="yellow">Pichia pastoris</span>. The purified N-terminal FLAG/hexahistidine-tagged Scp160p was run over an S300 gel filtration column; 2 ml fractions were collected and analyzed by α-FLAG western blot. As seen in Figure 7A, purified Scp160p eluted at a volume consistent with a >443 kDa protein. To confirm that no other proteins were associated with the purified Scp160p, 12 µl of the protein was run on SDS–PAGE and subjected to colloidal G250 Coomassie staining before loading on the S-300 column (Fig. 7B). As shown, no bands other than Scp160p were visible.
Several KH-domain proteins, including Sam68 and FMRP, have been reported to form homodimers in vivo (27). Therefore, we explored the possibility that the ~450 kDa Scp160p species might contain two or more copies of Scp160p. To test this hypothesis, <span class="blue">yeast</span> expressing FLAG–Scp160p were transfected with a centromeric plasmid, YCplac22–HA–Scp160p, encoding a distinct, HA-tagged allele of the protein. The functionality of this tagged protein had previously been demonstrated by polyribosome association as well as complementation of the null morphological phenotype in cells expressing only HA–Scp160p (data not shown). As seen in Figure 7C, when <span class="blue">yeast</span> co-expressing the two alleles were lysed and the FLAG-tagged protein was isolated as described above, no HA–Scp160p co-purified with the FLAG–Scp160p. These data strongly suggest that Scp160p does not form homo-dimers or higher-order multimers in vivo, and that the ~450 kDa species of Scp160p observed is simply the monomeric protein.<br><br>
DISCUSSION
The data presented here demonstrate two main points regarding the biochemical associations and function of Scp160p in <span class="blue">yeast</span>. First, the sucrose gradient fractionation data clearly demonstrate that Scp160p exists primarily associated with large complexes, likely to be polyribosomes. To our knowledge, this is the first time a vigilin family member has been shown to associate with polyribosomes in this manner. Interestingly, Weber and colleagues did not observe sucrose gradient fractionation data consistent with polyribosome association of Scp160p (5). These authors hypothesized that Scp160p associates only with membrane-bound polyribosomes, and that Scp160p is not found complexed with cytosolic polyribosomes (5). Our data do not support this hypothesis, since unlike Weber and colleagues, we were able to detect significant amounts of Scp160p associated with polyribosomes without using detergents during preparation. There are several possible explanations for this discrepancy. First, the lysate buffer utilized by that group contained 100 mM NaCl, whereas our buffer contained 50 mM KCl. Our data shown in Figure 3 suggest that 100 mM NaCl may have caused an instability in the Scp160p–RNA interaction, leading to release from polyribosomes during their procedure. Secondly, Weber’s lysate buffer reportedly contained heparin to inhibit ribonucleases; a number of reports (28,29) suggest that heparin can disrupt RNA–protein interactions. Although we have not in the past included heparin in any of our experiments, we have observed disruption of the polyribosome association in buffer containing residual diethylpyrocarbonate, another inhibitor of ribonucleases (data not shown).
Second, our data provide compelling evidence that Scp160p is released from polyribosomes as a component of an mRNP complex. Since Scp160p does not remain associated with either single ribosomes or ribosomal subunits following treatment with EDTA or RNase, it seems unlikely that Scp160p is a constitutive component of the translational machinery. Partial purification of the EDTA-resistant Scp160p complexes allowed us to demonstrate the presence of Pab1p but not Pub1p in these preparations, which is consistent with the idea that Scp160p associates with polyribosomes as a component of mRNP complexes. Previously, Scp160p had been shown to bind ribohomopolymers and rRNA in vitro, although the in vivo nucleic acid targets of Scp160p were unknown (5). The presence of Pab1p in RNase-sensitive Scp160p complexes indicates that Scp160p is primarily bound to polyadenylated RNAs. Whether or not Scp160p binds only specific sets of mRNAs will be the subject of future studies.
In addition to Scp160p and Pab1p, we have identified a third component of the complex, the protein Bfr1p. The gene, BFR1, was originally identified in a screen for high-copy suppressors of Brefeldin-A induced lethality, suggesting a role in the secretory pathway (16). Interestingly, however, bfr1 null mutants do not demonstrate any defects in the secretory pathway, but rather display similar phenotypes to scp160 null mutants, most notably increased ploidy and increased cell size (16). It is therefore unclear whether Bfr1p is capable of functioning in both RNA metabolism and secretion directly, or if the Scp160p–Bfr1p complex regulates the expression of one or more secretory genes at the post-transcriptional level. Additionally, two-hybrid analysis indicated an interaction of Bfr1p with the protein Bbp1p, a component of the mitotic spindle apparatus (30). While Bbp1p is an essential gene, overexpression leads to a phenotype similar to both bfr1 and scp160 null strains (30). Future work will address the functional relationship between Scp160p and Bfr1p including the possibility that these proteins represent a regulatory mechanism connecting the translational machinery with cell division and/or the secretory pathway.
Following treatment with either RNase or >150 mM NaCl, Scp160p remained as an apparent ~450 kDa species. Considering that purified Scp160p alone also migrates at this size, we conclude that this apparent complex may consist only of Scp160p, although the presence of other small components cannot be ruled out at this time. Furthermore, by utilizing <span class="blue">yeast</span> co-expressing two distinct epitope-tagged versions of Scp160p, we have ruled out the possibility of self-association of Scp160p. Although both tagged proteins appear functional, it remains a formal possibility that the tags somehow prevented formation of FLAG–HA hetero-complexes. Nonetheless, it seems most likely that native Scp160p monomers may simply migrate aberrantly under native conditions due to an unusual shape or some other physical property.
In conclusion we have shown convincing evidence that Scp160p exists in <span class="blue">yeast</span> cytoplasmic extracts primarily associated with polyribosomes. We have purified Scp160p following disruption of polyribosomes with EDTA, and identified two associated proteins: Pab1p and Bfr1p. The presence of Pab1p suggests that Scp160p associates with polyribosomes as a component of an mRNP, demonstrating Scp160p to be the first <span class="yellow">S.cerevisiae</span> multiple-KH domain protein characterized to function in this way.<br><br>
<h3>pmcA1131934</h3>HP1 modulates the transcription of cell-cycle regulators in <span class="yellow">Drosophila melanogaster</span>
Abstract
Heterochromatin protein 1 (HP1) was originally described as a non-histone chromosomal protein and is required for transcriptional gene silencing and the formation of heterochromatin. Although it is localized primarily at pericentric heterochromatin, a scattered distribution over a large number of euchromatic loci is also evident. Here, we provide evidence that <span class="yellow">Drosophila</span> HP1 is essential for the maintenance of active transcription of euchromatic genes functionally involved in cell-cycle progression, including those required for DNA replication and mitosis. Depletion of HP1 in proliferating embryonic cells caused aberrant progression of the cell cycle at S phase and G2/M phase, linked to aberrant chromosome segregation, cytokinesis, and an increase in apoptosis. The chromosomal distribution of Aurora B, and the level of phosphorylation of histone H3 serine 10 were also altered in the absence of HP1. Using chromatin immunoprecipitation analysis, we further demonstrate that the promoters of a number of cell-cycle regulator genes are bound to HP1, supporting a direct role for HP1 in their active transcription. Overall, our data suggest that HP1 is essential for the maintenance of cell-cycle progression and the transcription of cell-cycle regulatory genes. The results also support the view that HP1 is a positive regulator of transcription in euchromatin.<br><br>INTRODUCTION
Chromatin in higher eukaryotes is subdivided into different functional compartments termed heterochromatin and euchromatin (1). Heterochromatin differs from euchromatin in its DNA composition, replication timing, condensation throughout the cell cycle, and its ability to silence euchromatic genes placed adjacent to or within its territory, often described as position-effect-variegation (PEV) (2).
Heterochromatin protein 1 (HP1) was the first protein identified in <span class="yellow">Drosophila melanogaster</span> as a heterochromatin-associated protein (3); the corresponding gene has been cloned from a number of organisms and is highly conserved from <span class="blue">yeast</span> to <span class="yellow">human</span> (4). Polytene chromosome staining showed that, in <span class="yellow">Drosophila</span>, HP1 is distributed mainly in pericentric heterochromatin, telomeric heterochromatin, the banded small fourth chromosome (5–8), as well as ∼200 individual loci scattered throughout the euchromatic chromosomal arms (5). The gene encoding HP1 in <span class="yellow">D.melanogaster</span>, Su(var)2-5, was isolated as a suppressor of PEV (9–11). The protein contains a highly conserved motif, the chromo (chromatin organization modifier) domain, similar to Polycomb (Pc), a repressor of homeotic genes (12). The association between HP1 and pericentric heterochromatin is believed to occur via the chromo domain of HP1 and the N-terminal tail of histone H3 methylated at lysine 9 (13,14), generated by histone methyltransferase-Su(var)3-9, a partner of HP1 in pericentric heterochromatin (15). The C-terminal chromo ‘shadow’ domain of HP1 interacts with other silencing complexes to suppress local transcriptional activity (15–18). However, studies of HP1 chromosomal distribution also showed that HP1 does not always co-localize with lysine 9 methylated histone H3 or Su(var)3-9, especially in euchromatic regions (19–21); in some cases, HP1 is found directly bound to DNA (22,23). All these features argue for distinct roles for HP1 in chromatin and in epigenetic gene regulation.
HP1 is believed to be an essential structural protein protecting the integrity of chromosomes during cell division (8,24). Swi6, the homolog of HP1 in <span class="yellow">fission yeast</span>, is dispensable for survival, but its deletion results in lagging chromosomes during anaphase, and a high rate of chromosome loss (25,26). Mutations of HP1 in <span class="yellow">D.melanogaster</span> result in late larval lethality, chromosome breakages/loss, telomere fusion and a high frequency of cells with abnormal anaphase (8,27). Null alleles of the HP1 functional partner in <span class="yellow">mice</span> (SUVAR39) also showed various chromosomal defects (28), supporting a conserved role for heterochromatin proteins in the regulation of chromosome dynamics during cell-cycle progression. However, the mechanism(s) involved remains to be understood.
In this study, we utilized <span class="yellow">Drosophila</span> embryonic Kc cells and an RNA interference (RNAi)-based approach to demonstrate that HP1 plays an important role at S phase and G2/M phases during the cell cycle. We further show that nearly one-third of known/predicted cell-cycle regulators require HP1 to maintain their active transcription. These genes include MCMs, Orc4, CDC45L, INCENP, Aurora B, CAF1, Bub1, Bub3 and a few other cell-cycle regulators. ChIP analysis suggests that HP1 plays a direct role in their transcription. Therefore, the results of this study provide an alternative explanation for the specific role of HP1 in the regulation of chromatin dynamics and in cell-cycle progression.<br><br>MATERIALS AND METHODS
RNAi in Kc cells
<span class="yellow">Drosophila</span> Kc cells were routinely cultured at 25°C in Schneider <span class="yellow">Drosophila</span> medium (GIBCO) supplemented with 10% fetal <span class="yellow">calf</span> serum, 160 μg/ml penicillin, 250 μg/ml streptomycin, and 4 mM l-glutamine. Double-stranded RNA (dsRNA) of HP1 was generated by incubation of single-stranded RNA in annealing buffer (100 mM potassium acetate, 30 mM HEPES-KOH, pH 7.4, 2 mM magnesium acetate) for 3 min at 95°C and then placed in a beaker with water at 75°C and allowed to cool slowly to room temperature. The detailed procedure of RNAi was carried out according to the established protocols (). Briefly, Kc cells were seeded in a six-well dish using serum-free medium at 1 × 106 cells/ml. HP1 dsRNA (5 μg/ml) was added to the cultured Kc cells. After 60 min at room temperature, 2 ml of medium containing 10% serum was added to each well and the plates transferred to 25°C for up to 8 days. Western blotting and RT–PCR were carried out using the extract/total RNA isolated from control and dsRNA-treated cells on days 2, 6 and 8.<br><br>Cell-cycle and apoptosis analysis
The procedure for flow cytometric analysis of Kc cells followed that in the manual provided with the BrdU flow kit (BD PharMingen). The cells were fed with BrdU for 4 h, then scraped and collected. Fluorescence was measured using a FACSCalibur (Becton Dickinson). Data collection and analysis were performed using CellQuest software.<br><br>Electrophoresis and immunoblotting
Cell extracts (15 μg) were fractionated by 10% SDS–PAGE, then transferred to Hybond-P PVDF membranes (Amersham) and probed with primary antibodies (CIA9), and secondary antibodies (anti-<span class="yellow">rabbit</span> or anti-<span class="yellow">mouse</span> <span class="yellow">horseradish</span> peroxidase-conjugated IgG), obtained from Jackson Immunoresearch Laboratories. Enhanced chemiluminescence reagents (Amersham Pharmacia Biotech) were used for signal detection.
For the analysis of H3 ser10 phosphorylation, we used whole-cell extracts from 700 000 Kc cells (control and RNAi at day 8). Western blotting was performed using polyclonal antibodies against ser10-phosphorylated histone H3 at a dilution of 1:1000 (Upstate). Kc control cells arrested in mitosis by incubation in 25 µM colchicine (Sigma) for 24 h were also analyzed for comparison.<br><br>Immunofluorescence
Kc cells were seeded onto polylysine slides, fixed with 4% formaldehyde for 15 min and permeabilized with 0.5% Triton X-100 for 5 min. The incubation with primary antibodies was carried out in blocking solution for 1 h.
For staining of mitotic cells, the cells were permeabilized using PBST (PBS containing 0.3% Triton X-100) and stained with polyclonal antibody against <span class="yellow">Drosophila</span> Aurora B at 1:200 dilution and monoclonal <span class="yellow">mouse</span> at anti-β-tubulin 1:300 dilution (Chemicon International) as primary antibodies. Secondary antibodies were anti-<span class="yellow">rabbit</span> coupled with Alexa 488 (1:500) and anti-<span class="yellow">mouse</span> coupled to Alexa 546 (1:500) (Molecular Probes, Eugene, Oregon). Images were acquired using a confocal LSM510 META microscope (Zeiss). Stacks of images were analyzed using the IMARIS 4.0 program (Media cybernetics, Carlsbad, CA).<br><br>Antibodies
Affinity-purified polyclonal antibodies of HP1 (<span class="yellow">rabbit</span> #192 and #187, 5 μg) and 5 μg of polyclonal anti-HA antibodies (Sigma) were used in each ChIP reaction. The specificity of the HP1 polyclonal antibodies was determined using various approaches, including western blotting assay, immunofluorescence staining and immunoprecipitation to pull down HP1 (data not shown). The monoclonal antibody HP1–CIA9 (5) was used at a dilution of 1:20 in immunoblotting assays.<br><br>Microarray analysis and RT–PCR
Total RNA was isolated from control and HP1-depleted Kc cells at day 8 using an RNeasy kit (Qiagen). RNA labeling and microarray data analysis followed the standard protocol from Affymetrix. We used ANOVA (P < 0.001) to assess the expression confidence for each gene.
For RT–PCR analysis, poly(A)+ mRNA was purified with the Oligotex Direct mRNA kit (Qiagen) according to the manufacturer's instructions. The purified poly(A)+ RNA was reverse transcribed using the Thermoscript kit (Invitrogen). The cDNA was then used for PCR amplification for 35 cycles with gene-specific primers. PCR products were scanned after electrophoretic separation with a Typhoon Scanner, quantified using ImageQuant software (Amersham Biosciences) and normalized for amplification of the Actin5c transcript. The sequence of primers used for RT–PCR and ChIP analysis are provided in the Supplementary Material.<br><br>ChIP
ChIP was performed according to Orlando et al. (29) and the protocol provided by Upstate () with some modifications. In brief, 1–2 × 108 Kc cells were prepared and fixed in 1% formaldehyde. Nuclei were isolated according to a standard procedure in Current Protocols (), then resuspended in 1.7 ml of lysis buffer (50 mM Tris, pH 8.0, 10 mM EDTA, 1% SDS and protease inhibitors) and sonicated using a Branson sonifier 250. Chromatin fractions in the size range 0.2–0.8 kb were used to perform immunoprecipitation experiments. We used 5 μg affinity-purified polyclonal antibodies (#192 and #182 for HP1; HA antibody for control) and 1 ml of <span class="yellow">salmon</span> sperm DNA/protein-A-agarose (Upstate) pre-cleared chromatin lysate in each reaction. The mixture was then rotated at 4°C overnight and the recovered beads were washed twice with 1 ml of Low salt buffer (Upstate), once with High salt buffer (Upstate), once with LiCl buffer (Upstate) and twice with TE at 4°C for 8 min. ChIP DNA was extracted according to the standard procedures (29).<br><br>
RESULTS
Depletion of HP1 in <span class="yellow">Drosophila</span> Kc cells
Various chromosomal defects in the cell cycle have been observed in embryos or larval tissues of <span class="yellow">Drosophila</span> HP1 mutants (8,27). However, the presence of maternally loaded HP1 in embryos and the lethality of HP1 mutants at late larval stages have so far precluded a systematic study of the role of HP1 in cell-cycle regulation. Therefore, we used <span class="yellow">Drosophila</span> Kc cells, a cell line derived from <span class="yellow">Drosophila</span> embryos, as a model system to address this problem. HP1 transcripts were depleted using an RNAi-based approach (see Materials and Methods). The reduction in HP1 expression was measured both by RT–PCR and by western blotting analysis (Figure 1A). A significant reduction in the HP1 expression was already evident after 2 days treatment with HP1 dsRNA. Cells at day 8 showed a reduction in HP1 of ∼90% (Figure 1A) and were therefore used in all subsequent experiments.<br><br>Cell-cycle progression at S and G2/M phase is altered in the absence of HP1
The impact of HP1 loss on the cell cycle of Kc cells was determined using cell-cycle profile analysis of HP1-depleted and control cells. The percentage of cells in S phase was determined by BrdU incorporation, and total DNA content by 7-amino-actinomycin (7-AAD). The results showed that the depletion of HP1 (day 8) caused a decrease in S-phase cells of at least 4-fold, and a 2-fold decrease in G2/M-phase cells (Figure 1B), although no significant effect was found at the G1 phase. In addition, depletion of HP1 caused a greater than 7-fold increase in the number of apoptotic cells. These results, therefore, confirm that HP1 is an important regulator during the cell cycle, especially at the S and G2/M phases.<br><br>Cell-cycle regulators require HP1 to maintain their active transcription
To ask whether the cell-cycle defects were due to changes in the transcription of genes functionally involved in S phase and the G2/M phase, we next assessed global changes in gene transcription following depletion of HP1. Expression profile analysis was performed using total RNA isolated from both HP1-depleted Kc cells and control Kc cells, and an Affymetrix <span class="yellow">Drosophila</span> chip. For each experiment, we used total RNA isolated from two independent HP1-depleted and control samples, and at least two independent experiments were performed.
The microarray analysis showed that loss of HP1 function in Kc cells resulted in alterations in transcription of >500 genes: ∼400 genes were down-regulated and ∼120 genes were up-regulated (>1.5-fold, ANOVA). The function of these genes ranged from cellular enzymes, signal transduction molecules, and membrane and cell structural proteins, to nucleic acid-binding proteins and cell-cycle regulators (Figure 2A). At the chromosomal level, the genes targeted by HP1 appeared to be distributed along all euchromatic chromosomal arms (data not shown), supporting a global role of HP1 in euchromatic gene regulation (20).
Among 60 known/predicted genes associated with DNA replication function, 15 were down-regulated in the absence of HP1 (Figure 2B). These included McM2, McM5, McM6 and CDC45L, which are required for processive DNA replication and correct chromosome condensation (30–32). Other genes involved in DNA replication, such as components of the origin recognition complex (Orc)—Orc4, Caf1, Gnf1, Dref1, DNA polymerase-γ and Tam—were also down-regulated (Figure 2B). Aurora B and inner centromere protein (INCENP), known to be required for kinetochore assembly, chromosome condensation and bipolar chromosome attachment during mitosis (33), also showed a reduction in transcription. A similar loss of transcription was observed in Bub1 and Bub3 (Figure 2B), encoding mitotic checkpoint control proteins (34,35). Loss function of Bub1 has been shown to cause chromatin bridges to extend between the two separating groups of chromosomes, and extensive chromosome fragmentation in anaphase cells (35).
We confirmed the changes in the transcription of cell-cycle regulators using semi-quantitative RT–PCR, which gave results consistent with the microarray analysis. In addition, cell-cycle regulator genes, such as McM3, McM7 and Asp (abnormal spindle), were also confirmed to be down-regulated (Figure 2C). Collectively, these results demonstrate that HP1 is indeed involved in the regulation of transcription of cell-cycle regulators.<br><br>HP1 is required for Aurora B distribution and histone H3 phosphorylation
INCENP is localized to the centromeric region of chromosomes at metaphase and the spindle midzone at anaphase, which then targets Aurora B, a kinase essential for histone H3 ser10 phosphorylation, to these sites (36). Loss of function of both these ‘chromosomal passenger’ proteins causes abnormal chromosomal segregation at metaphase, as well as certain cytokinesis defects (36,37). The loss of transcription of both INCENP and Aurora B after depletion of HP1, therefore, raised the possibility that localization of Aurora B (Figure 3 and data not shown) may be altered. Staining of HP1-depleted Kc cells with anti-Aurora B antibodies indeed revealed an altered localization of Aurora B and, in a number of cases, a complete loss of Aurora B (Figure 3A). Consistent with the loss function of Aurora B, the spindles in the metaphase cells were also disorganized, with a large number of cells showing an altered prometaphase chromosome alignment (Figure 3A). Some showed extensive chromosome fragmentation (Figure 3B), or the presence of a third spindle pole-like structure as indicated by beta-tubulin (Figure 3A). At telophase, we observed defective separating cells with an extra cell envelope-like structure without nuclei (Figure 3A). Chromatin bridges or lagging chromatids at telophase were also evident in some cells (Figure 3C); however, in some cases, localization of Aurora B appeared not to be affected, arguing that other pathways are possibly involved.
We next analyzed changes in histone H3 serine 10 phosphorylation, since the loss of transcription of INCENP is known to affect localization of Aurora B (33), which is essential in the regulation of histone H3 phosphorylation (36). Total cell extracts from HP1-depleted Kc cells were analyzed by western blotting (Figure 3D). The results indeed showed a severalfold reduction in H3 ser10 phosphorylation after depletion of HP1, consistent with the functional disruption of INCENP and Aurora B in the absence of HP1.<br><br>HP1 directly targets genes encoding cell-cycle regulators in euchromatin
To test whether the loss of transcription of genes involved in DNA replication and mitosis was a direct effect of the loss of HP1, we performed a ChIP analysis to determine whether HP1 is physically associated with these genes. Chromatin lysates from formaldehyde-fixed Kc cells were sonicated into small chromatin fragments (0.2–0.8 kb) and immunoprecipitated with polyclonal antibodies against <span class="yellow">Drosophila</span> HP1. As a control, we used a mock precipitation (beads only) and polyclonal antibodies against HA. Our ChIP results showed that known transposable elements distributed in heterochromatin, such as F-element, TART and 1360 (7,38), were all enriched in HP1 binding (Figure 4), which is also consistent with a previous study (20).
Using the same ChIP DNA material, we then attempted to determine whether HP1 was enriched in genes involved in DNA replication. Primers were designed to cover the promoter regions of selected genes. The results showed that McM3, McM5 and Tam were all enriched in HP1 binding (Figure 4). However, McM7 appeared to be HP1-negative, although its transcription was also affected by the loss of HP1 function. Genes essential for mitosis, such as Aurora B, were also HP1-positive (Figure 4). These results demonstrate that these cell-cycle regulator genes are directly targeted by HP1 in their promoter regions.<br><br>
DISCUSSION
In this study, we used microarray and RT–PCR techniques to demonstrate that transcription of cell-cycle regulators is misregulated in the absence of HP1. Certain defects in S phase may be a direct consequence of the loss of transcription of DNA replication genes such as McM2, McM5, McM6, CDC45L, Orc4 and others, since these genes have been functionally implicated in the initiation of DNA replication and/or the progression of replication forks (39). Depletion or mutation of these genes has been shown to result in DNA damage (32), the blockage of replication forks (39), increased chromosome loss/genome instability, and defective condensation (30).
The reduction in the number of cells in G2/M phase may be a consequence of the reduction in transcription or functional disruption of INCENP, Aurora B, Bub1, and Bub3 (34,36). Chromosome segregation defects, such as chromosome fragmentation and chromatin bridges in anaphase/telophase cells, and certain cytokinesis defects in HP1-depleted cells, mimic the phenotype of cells with loss function of INCENP, Aurora B or Bub1 (35–37). The mislocalization of Aurora B in the absence of HP1 is also consistent with the loss of transcription and functional disruption of INCENP (37), and the reduction in Aurora B transcription may be partially responsible for the observed chromosomal defects, including loss of histone H3 phosphorylation at serine 10.
HP1 is also known to physically interact with certain components of replication complexes such as ORCs and MCMs (30,40,41), with the inner centromere protein INCENP (42) and the chromatin assembly factor CAF1 (43) promoting delivery of HP1 to heterochromatin sites (44). Loss of HP1 is, therefore, expected to cause disruption to such HP1-associated complexes, and will partially contribute to the chromatin/chromosomal defects in HP1 mutants (8,27) and HP1-depleted Kc cells. It is therefore well possible that the loss of transcription of these cell-cycle regulator genes, and consequent disruption of HP1 functional complexes or heterochromatin structure, all contributed to the cell-cycle defects observed.
The ChIP assay supports the hypothesis that the loss of transcription of cell-cycle regulator genes is a direct effect of the lack of HP1. Aurora B, McM3, and McM5 were all bound by HP1 at their promoter regions, although other cell-cycle regulators, such as McM7, were HP1-negative, implying that the altered transcription in these genes might be a secondary effect of the loss of HP1.
A previous study in <span class="yellow">Drosophila</span> Kc cells (20) employed an approach based on the ectopic expression of a fusion protein of HP1 with a prokaryotic DNA adenine methyltransferase and identified a number of methylated targets in the genome. In this study, MCM3 and MCM5 were not found to be methylated, indicating lack of association with HP1. On the other hand, heterochromatin repeats, such F-element and 1360, were consistently found to be HP1-enriched both here and in the previous study. It remains to be determined whether these discrepancies are due to the different experimental systems used. However, we note that the previous study was performed using a cDNA array, while we observe binding of endogenous HP1 at the promoter of these genes. Similarly, another study using chromatin immunoprecipitation in larvae also showed few HP1-positive genes that were not detected in Kc cells by the Dam ID approach (21).
A large number of genes affected by the loss of <span class="yellow">D.melanogaster</span> HP1 in larval tissues (21) seem to be different from that in embryonic Kc cells. The change in the transcription of Aurora B and few cell-cycle regulators reported in this study is also not found among the HP1-affected genes at larval stage (21). This may be due to specific role(s) of HP1 in different stages of development. Alternatively, it is also possible that the impact of HP1 in the transcription of cell-cycle regulators in proliferating cells is underestimated when performing the analyses on larval tissues, and thus on mixed populations of both proliferating and differentiating/differentiated cells.
HP1 is generally known as a transcriptional repressor, as supported by several lines of evidence: silencing of a euchromatic reporter gene in heterochromatin requires HP1 (10,11), tethering of HP1 next to a euchromatic reporter gene causes silencing (45), and the repression of genes within euchromatic region 31 bound by HP1 is relieved in the absence of HP1 (46). In contrast, genes in heterochromatin, known as heterochromatic genes, such as light and rolled, seem to require HP1 to maintain their active transcription (47,48). The level of transcription of heterochromatic genes was dramatically reduced in a mutated HP1 background (47–49). It was therefore proposed that HP1 may function as a positive regulator of transcription of these genes (50,51), although the exact regulation mechanism remains unclear. A study of heat-shock genes found that HP1 is associated with RNA transcripts in the coding region, and is also a positive regulator of their transcription (52,53). The chromatin association of HP1 at the promoter region of active euchromatic genes demonstrated from this work and others, and its independence from histone H3K9 methylation (21), all suggest that mechanism whereby HP1 modulates transcription of euchromatic genes is potentially distinct from its role in heterochromatin formation.
Collectively, the results of this study demonstrate that HP1 plays an essential role in cell-cycle progression, and support the view that HP1, in addition to its role in heterochromatin, can act as a positive transcriptional regulator of euchromatic genes.<br><br>SUPPLEMENTARY MATERIAL
Supplementary Material is available at NAR Online.<br><br>Supplementary Material<br><br>
<h3>pmcA1190194</h3>Structural organization and interactions of transmembrane domains in tetraspanin proteins
Abstract
Background
Proteins of the tetraspanin family contain four transmembrane domains (TM1-4) linked by two extracellular loops and a short intracellular loop, and have short intracellular N- and C-termini. While structure and function analysis of the larger extracellular loop has been performed, the organization and role of transmembrane domains have not been systematically assessed.<br><br>Results
Among 28 <span class="yellow">human</span> tetraspanin proteins, the TM1-3 sequences display a distinct heptad repeat motif (abcdefg)n. In TM1, position a is occupied by structurally conserved bulky residues and position d contains highly conserved Asn and Gly residues. In TM2, position a is occupied by conserved small residues (Gly/Ala/Thr), and position d has a conserved Gly and two bulky aliphatic residues. In TM3, three a positions of the heptad repeat are filled by two leucines and a glutamate/glutamine residue, and two d positions are occupied by either Phe/Tyr or Val/Ile/Leu residues. No heptad motif is apparent in TM4 sequences. Mutations of conserved glycines in <span class="yellow">human</span> CD9 (Gly25 and Gly32 in TM1; Gly67 and Gly74 in TM2) caused aggregation of mutant proteins inside the cell. Modeling of the TM1-TM2 interface in CD9, using a novel algorithm, predicts tight packing of conserved bulky residues against conserved Gly residues along the two helices. The homodimeric interface of CD9 was mapped, by disulfide cross-linking of single-cysteine mutants, to the vicinity of residues Leu14 and Phe17 in TM1 (positions g and c) and Gly77, Gly80 and Ala81 in TM2 (positions d, g and a, respectively). Mutations of a and d residues in both TM1 and TM2 (Gly25, Gly32, Gly67 and Gly74), involved in intramolecular TM1-TM2 interaction, also strongly diminished intermolecular interaction, as assessed by cross-linking of Cys80.<br><br>Conclusion
Our results suggest that tetraspanin intra- and intermolecular interactions are mediated by conserved residues in adjacent, but distinct regions of TM1 and TM2. A key structural element that defines TM1-TM2 interaction in tetraspanins is the specific packing of bulky residues against small residues.<br><br><br><br>Background
Tetraspanins constitute a large family of integral membrane proteins, characteristically containing 4, 6 or 8 conserved cysteine residues in the large extracellular loop (including the CCG and PxxCC motifs), which form disulfide bonds, and several conserved polar residues in the intracellular loop and transmembrane regions [1,2]. There are 32 putative tetraspanin family members in mammals, 37 in <span class="yellow">Drosophila melanogaster</span> and 20 in <span class="yellow">Caenorhabditis elegans</span>. Tetraspanins play diverse roles in cell adhesion, migration and fusion processes, cellular activation and signaling (reviewed in refs. [2-4]). Mammalian tetraspanins such as CD9, CD63, CD81, CD82, CD151, rds/peripherin, and uroplakins Ia and Ib have been most extensively studied, with <span class="yellow">mouse</span> knock-out models available for CD9 [5-7], CD81 [8,9], CD151 [10] and a few others. However, the majority of tetraspanins are characterized very little, if at all, at genetic, biochemical or structural levels.
The large extracellular loop (LEL) of tetraspanins has received most attention, since it contains functionally important sites. Sequence QRD (194–196) in CD151 is important for association with integrins, which has functional consequences for integrin-dependent cell spreading and multicellular cable formation [11]. A site in the LEL of CD9, SFQ (residues 173–175), is essential for CD9 function in sperm-egg fusion [12]. The crystal structure of tetraspanin CD81 LEL revealed five α-helixes, A-E [13]. Helices A, B and E form a relatively conserved region in tetraspanins, whereas the region between helices B and E is the most variable [14]. Interestingly, the variable region contains most of the functionally important sites involved in tetraspanin protein-protein interactions.
A remarkable biochemical property of tetraspanin molecules is their ability to associate with a large number of other transmembrane proteins, including integrins, membrane-associated growth factors and receptors, MHC class II molecules, Ig superfamily proteins, and each other [2,3,15]. Several of these lateral associations of tetraspanins are detected in "mild" detergents (Brij series, CHAPS), but are disrupted by "strong" detergents such as Triton X-100 or SDS. Multiprotein complexes of tetraspanins and associated molecules, also called the "tetraspanin web" [16], may represent a distinct tetraspanin-enriched membrane microdomain [17,18]. The formation of this microdomain is influenced by palmitoylation of several conserved juxtamembrane cysteine residues in tetraspanins [19-21].
The transmembrane domains, encompassing nearly half of a tetraspanin protein, are the most conserved part of the molecule (Stipp et al. [1] and this study). However, very little functional information is available on these domains. The differential detergent sensitivity of tetraspanin-tetraspanin associations suggests that hydrophobic interactions between TM helices may play a role. Indeed, when the large extracellular loop (LEL) of CD151 is deleted, the molecule is still able to associate with other tetraspanins [22]. Thus, TM domains are strong candidates for mediating tetraspanin-tetraspanin interactions.
The importance of TM domain interactions in intramolecular organization was demonstrated in a study showing that CD82 fragment TM2-4, lacking TM1, was retained in the endoplasmic reticulum, but was transported to the cell surface upon co-expression of TM1 [23]. This in vivo reconstitution experiment demonstrated a strong interaction between TM1 and the rest of the molecule. Expression of a truncated CD9 molecule (TM3-LEL-TM4) results in intracellular accumulation of the protein and significant misfolding of the LEL, as judged by inappropriate disulfide formation and diminished antibody reactivity (our unpublished data). Similarly, a CD9 epitope in the LEL is lost in molecules lacking either TM2+TM3 or just TM4 [24]. Thus, TM domain interactions and packing are crucial for proper folding, stability and transport of tetraspanin molecules.
In a previous study, we showed that covalent cross-linking of membrane-proximal cysteine residues can be used as a tool for detection of tetraspanin-tetraspanin associations [25]. Inhibition of cysteine palmitoylation by 2-bromopalmitate (2-BP) made cysteines available for cross-linking and enabled demonstration of specific tetraspanin homodimerization and low levels of heterodimerization. We concluded that tetraspanin homodimers, formed in the Golgi, may be a fundamental structural unit within tetraspanin microdomains.
In this study, we carried out detailed sequence analysis of <span class="yellow">human</span> tetraspanin TM domains. We show that a heptad repeat containing conserved glycine, asparagine and large hydrophobic residues occurs in TM1 and TM2 domains, and predict tight intramolecular association of these two domains by packing of the large residues against the small residues. Moreover, by using cysteine cross-linking we map a dimerization interface in the <span class="yellow">human</span> CD9 protein, and show that conserved heptad motif glycine residues are also important for intermolecular CD9 associations.<br><br>Results
Sequence analysis of tetraspanin transmembrane domains: presence of the heptad repeat motif
We focused our attention on 28 <span class="yellow">human</span> tetraspanins identified from the SWISS-PROT and GenBank databases. All tetraspanins have in common four hydrophobic stretches (TM domains) of 20–25 residues, and contain highly conserved residues in the second extracellular loop, in particular the Cys-Cys-Gly (CCG) motif. Detailed analysis of the large extracellular loop sequences [14], and dendrograms based on full-length alignment can be found in earlier studies [26,27]. The length of each transmembrane domain was established based on previous sequence analysis of tetraspanin sequences [27,28], and on annotations to the database entries. Manual adjustments based on sequence homology and hydrophobicity profiles were done to fully delineate the TM domains. The resulting lengths of TM domains were: TM1 – 23 residues; TM2 – 21 residues; TM3 – 25 residues; TM4 – 25 residues. Two more residues could be added onto the N-terminal part of TM2; however, relatively small sequence conservation of these residues among tetraspanins and occurrence of polar/charged side chains in some tetraspanins precluded us from doing so for the global alignment.
Figures 1 and 2 show a multiple sequence alignment of four TM domains of 28 <span class="yellow">human</span> tetraspanins. For each position within the domains, consensus residues were determined and classified (with individual color code) in 4 categories: 1) large hydrophobic residues (including Val, Met, Leu, Ile, Phe, Tyr, Trp), 2) small residues (Gly, Ala, Ser and Thr), 3) Cys, and 4) Asn. When more than two types of residues occupied a given position in a TM, a dual-color pattern that reflected the prevalence of the particular residue type was used (Figure 1). Cysteine residues were shown separately due to their importance as palmitoylation target sites. The highly conserved asparagine residue in TM1 was considered separately. No proline residues are found in TM domains 1–3 of <span class="yellow">human</span> tetraspanins.
An inspection of the multiple sequence alignment reveals a repeating heptad amino acid pattern, (abcdefg)n, in TM1, 2 and 3 (Figure 1, 2). Heptad repeats promote helical coiled coil interactions in multiple soluble and membrane-spanning proteins [29-31]. In the heptad repeat, hydrophobic residues in positions a and d are of special importance, as they directly mediate interhelical contacts by creating a tight knobs-into-holes packing in the coiled coil structure [32]. For instance, in the leucine zipper of the <span class="blue">yeast</span> transcription factor GCN4, positions a and d contain Val and Leu residues, respectively, with an Asn residue in a single a position forming a hydrogen bond across the GCN4 dimer interface [33].
In TM1 of tetraspanins, highly conserved Asn, Gly and Gly residues (numbers 18, 25 and 32 in the CD9 sequence) appear at d positions of the heptad repeats, and residues 14, 21 and 28 are at a positions (Figure 1). In TM2, residues 67, 74 and 81 (consensus Gly, Gly and Ala, respectively) occupy a positions, whereas residues 63, 70 and 77 are at d positions. Another highly conserved glycine, Gly80, occupies the 3rd g position in TM2. In TM3, the conserved pattern consists of two leucine residues (Leu89 and Leu96) and a glutamate/glutamine residue (Glu/Gln103) in a positions (Figure 2). Two d positions are also conserved – Phe/Tyr92 and Ile/Val/Leu99. TM4 lacks a conserved heptad pattern and has only a single conserved position, Glu/Gln209 (with four exceptions). These features of TM1-4 of tetraspanins are displayed on helical wheel diagrams (Figure 3).<br><br>Analysis of TM1 sequences
The conserved Asn-Gly-Gly motif, occupying designated d positions of the heptad repeat, is the most prominent structural feature of TM1. We also compared sequences of CD9 orthologs from 10 different organisms (the most available for any tetraspanin) to gain further insight into conservation and variability of the TM1 sequence. As shown in Figure 4, positions a, d and g in TM1 are among the most conserved (0, 1 and 1 substitution, respectively), while interspecies variability tends to occur in other positions: b (5 substitutions), c (4 substitutions), e (4 substitutions) and f (4 substitutions). Thus, the positions typically involved in coiled coil interactions (a and d) are the most conserved.
When residues of TM1 are plotted as a helical wheel, additional structural features are revealed (Figure 3). There are highly conserved aliphatic and aromatic residues in the first three a positions of the heptad motif (Phe15, Trp22 and Leu29 in CD9), as well as in g positions (Leu14, Phe21, Val28 in CD9). The "ridges" formed by these bulky residues are flanking the "groove"-forming Gly residues of the Asn-Gly-Gly position d motif. In contrast, b, c, e and f positions show an overall higher variability among tetraspanins, as also seen in the comparison of CD9 orthologs described above.<br><br>Analysis of TM2 sequences
A landmark feature of TM2 in tetraspanins is the presence of highly conserved glycine residues (Gly67, 74, 77 and 80 in CD9, Figure 1). Other substitutions at these positions are almost exclusively small residues, such as Ala or Ser. In addition, Ala, Ser or Thr occupy position 81. This residue, together with Gly67 and Gly74, forms face a of the helix. Residue Gly77 (position d) is preceded by conserved, chiefly large hydrophobic residues on the same helical face (Leu63 and Met70 in CD9). Extremely conserved Gly80 falls into heptad position g (Figure 3). Among CD9 orthologs, heptad positions a and d are absolutely conserved, whereas other positions have the following number of substitutions: b – 3; c – 2; e – 1; f – 3; g – 1 (Figure 4). Two of the f position residues in TM2 (65 and 79) also show higher variability among different tetraspanins (Figures 1, 3). Cysteine residues are frequently found near the cytoplasmic end of TM2 helix at positions 78 and 79; these cysteines are likely to be palmitoylated.<br><br>Analysis of TM3 and TM4 sequences
The TM3 domain provides another example of the heptad repeat pattern. Position a is occupied by two highly conserved leucine and a glutamate/glutamine residue (Leu89, Leu96 and Glu/Gln103 in CD9). Furthermore, two d positions are conserved – Phe/Tyr92 (aromatic residue) and Ile/Val99 (β-branched aliphatic residue; Figures 2, 3). In addition, residue 100 in position e is generally Phe or Leu. Among CD9 orthologs, position a has 1 substitution, positions b, c and f each have 6, positions d and e each have 2, and g has 4. Thus, as for TM1 and TM2, positions a and d are among the most conserved, but overall TM3 has more variable positions than TM1 or TM2 (Figure 3). Less than half of TM3 sequences contain cysteine residues, and those tend to occur at the internal positions of the helix (Figure 2).
TM4 shows less conservation among various tetraspanin family members than the other TM domains (Figures 2, 3). The only highly conserved feature is the glutamate/glutamine residue in position 209. In addition, one or two cysteine residues can be found at the C-terminal end of TM4 in some tetraspanins (e.g. CD9, CD81, CD151), and many sequences contain additional polar residues (Arg, Lys, His, Asn, Gln). No conserved heptad motif was identified in TM4, as also confirmed by analysis of substitutions in CD9 orthologs (data not shown).<br><br>Mutational analysis of conserved glycine residues in TM1 and TM2
The conserved nature of the Asn and Gly residues in TM1 and TM2 prompted an analysis of their functional role. To this end, we have probed whether mutations of these residues destabilize the protein molecule. We expressed a construct of the first and second TMs of CD9, connected by the small extracellular loop, and tagged with a C-terminal green fluorescent protein (TM(1+2)-GFP molecule). In <span class="yellow">human</span> rhabdomyosarcoma RD cells, the wild-type fusion protein localized mostly in a reticular, intracellular pattern, without forming any large aggregates (Figure 5, panel A). Remarkably, when double mutants Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu were expressed, the protein formed distinct large aggregates in a high proportion of cells (Figure 5, panels C and E). In contrast, double mutant Gly77Leu + Gly80Leu did not form such aggregates (Figure 5, panel G). Results with respective single mutants were similar to that with double mutants, with the aggregation being somewhat more pronounced for Leu substitutions of Gly67 and Gly74 compared to Gly25 and Gly32 mutations. No aggregation was observed for Asn18Ser and Asn18Tyr mutants (data not shown). Also, nearly identical results were obtained with <span class="yellow">human</span> HT1080 cells (data not shown).
We interpret these results as an indication that aggregating mutants are destabilized or misfolded while non-aggregating mutants retain the wild-type conformation. Intriguingly, mutations to the conserved GG7 motifs caused protein aggregation while the mutation of other glycines had no detectable effect. These results also suggest that wild-type GFP, which has weak tendency to self-associate, could enhance non-specific interactions of destabilized mutant TM(1+2) CD9 moieties, leading to their aggregation. Consistent with this hypothesis, the aggregation of Gly25Leu + Gly32Leu and Gly67Leu + Gly74Leu double mutants was suppressed when monomeric GFP molecule, Leu221Lys [34] was used (Figure 5, panels D and F). The use of monomeric GFP did not affect intercellular localization of wild-type CD9 TM(1+2) (Figure 5, panel B), or a Gly77Leu + Gly80Leu double mutant (Figure 5, panel H).
In summary, Leu substitutions of Gly residues that are part of the Asn-Gly-Gly (NGG7) motif in TM1, or Gly-Gly-Ala (GGA7) motif in TM2, resulted in destabilization and aggregation of GFP-fused TM(1+2) proteins, whereas substitutions of Gly77 or Gly80, which are not part of these motifs (Figure 3), failed to show such aggregation.<br><br>Prediction and modelling of interaction between TM1 and TM2
Consecutive helices in polytopic membrane proteins frequently interact [35]. Sequence analysis of TM1 and TM2 helices of tetraspanins reveals a remarkable complementarity in the distribution of large and small residues at heptad positions a and d along the helical axis (Figure 3), suggesting that these residues may interact. To further elucidate the potential for TM1-TM2 interaction, the putative interface was modeled using a novel algorithm that considers mutational data during each step of a Monte Carlo simulated annealing cycle (see Methods for details). Specifically, Gly25Leu, Gly32Leu, Gly67Leu and Gly74Leu were scored as disruptive mutations, while Asn18Ser, Gly77Leu and Gly80Leu were scored as silent mutations, based on their effects on protein stability (Figure 5 and data not shown).
The resulting model predicts left-handed crossing of TM1 and TM2 helices at an angle of +28°. The key element of the structure is the apposition of bulky and small heptad position a and d residues, as follows: Gly32-Leu63; Gly67-Leu29; Gly25-Met70; Gly74-Trp22; Asn18-Gly77; Ala81-Phe15 (Figure 6). Our model predicts that each of these residue pairs are in van der Waals contact. Additionally, two potential H-bonds are predicted in this model, indicating close packing: Gly67 Cα to Gly25 carbonyl oxygen, and Trp22 Cα to Met70 carbonyl oxygen. The packing is tighter in the ectodomain-proximal portion of the helices (Figure 6, panel B), as determined by Cα-Cα distances between interacting residue pairs.
The key elements of the model are corroborated by the presence of apparently complementary substitutions in TM1 and TM2 sequences of different tetraspanins (Figure 1, boxed residues). For example, Gly74 is predicted to interact with Trp22. In 8 of the 10 tetraspanins that contain a substitution for Gly74, a compensatory substitution occurs at the Trp22 position (Figure 1). Thus, a larger non-glycine side chain at position 74 may necessitate a less bulky non-Trp side chain in position 22. Likewise, the presence of a Cβ at position 25, typically occupied by glycine, necessitates a non-β-branched amino acid at position 70, which is occupied by a β-branched residue in nearly half of all cases. Indeed, we find that in each of 5 cases in which position 25 contains a Cβ, a leucine residue occurs in position 70. This analysis is consistent with our molecular model that suggests Leu70 will pack most favorably against a Cβ at position 25 than a β-branched residue or a methionine.<br><br>Role of TM1 and TM2 heptad motif residues in CD9 dimerization
To probe CD9 dimerization, we used a cysteine-mediated cross-linking approach. We established previously a simple and efficient method for cysteine-mediated cross-linking [25]. After cells are pre-treated with 2-BP for 16–24 hours to expose normally palmitoylated cysteines, the cysteines can be cross-linked using any of the following methods: a) Spontaneous oxidation in Brij97 lysates (a condition that preserves tetraspanin-tetraspanin associations), b) In situ cross-linking, by pre-lysis oxidation of cells with Cu2+-phenanthroline (CuP) to promote disulfide bond formation. c) In situ cross-linking with thiol-reactive cross-linking agents of defined length (e.g. DTME, BMB). The first two approaches produce in essence "zero-length" disulfides, indicative of close proximity of target cysteines and presumably high specificity of interaction. In contrast, chemical cross-linkers with 6–20 Å spacer arm may cross-link with higher efficiency, but not necessarily higher specificity. However, they provide advantages such as variable membrane permeability, and potential linkage cleavability. For tetraspanins such as CD9, membrane-permeable cross-linker DTME (13.3 Å-long, reducible) provides highly specific and efficient cross-linking [25]. Here we have used a cysteine cross-linking strategy, in combination with cysteine-scanning mutagenesis, to map the residues from TM1 and TM2 contributing to the CD9 dimerization interface.
For subsequent cross-linking experiments using CD9 TM(1+2)-GFP protein, the non-dimerizing form of GFP was used. This avoids potential GFP-dependent dimerization and aggregation that can be observed with wild-type GFP, especially when fusions with transmembrane proteins are studied [36]. Importantly, the Leu221Lys mutation in GFP prevented aggregation of mutant forms of CD9 TM(1+2), which was observed with wild-type GFP fusion (Figure 5). The TM(1+2) fragment of CD9 contains three native cysteines – Cys9, Cys78 and Cys79. Single-cysteine mutants of TM(1+2) were constructed, in which a cysteine was placed at various faces of TM1 or TM2 while all of the wild-type cysteines were simultaneously replaced by serines. The mutant proteins were transiently expressed in RD cells (having little endogenous CD9), which were then treated for 16–18 hours with 2-BP. To achieve maximal specificity in cross-linking we used a "zero-length" agent, CuP.
First, single-cysteine replacements were constructed for residues Leu14, Phe15, Gly16, Phe17 and Asn18, covering just over one complete helical turn at the beginning of TM1. While residue Asn18 is highly conserved, positions 14, 15 and 17 are occupied by bulky hydrophobic residues in most tetraspanins, whereas position 16 shows less conservation (Figures 1, 4). All of the single-cysteine mutants showed diffused pattern of protein localization, without any signs of aggregation. As shown in Figure 7A, the highest level of intermolecular cross-linking was observed for Leu14Cys and Phe17Cys mutants, a lower level for Phe15Cys and Gly16Cys mutants, and very little cross-linking for Asn18Cys substitution. These results indicate that: a) the first two transmembrane domains of CD9 alone can mediate its dimerization, and b) the g and c residues of TM1 (e.g. Leu14 and Phe17, Figure 3) are likely to be part of the intermolecular interface.
Similarly, single-cysteine substitutions were made for residues Gly77, Gly80 and Ala81 in TM2; in addition, proteins carrying a single wild-type cysteine, Cys9, Cys78 or Cys79, were tested. No protein aggregation was observed for any of these single-cysteine mutants. As shown in Figure 7B, the relatively low level of intermolecular cross-linking of wild-type CD9 TM(1+2)-GFP protein was enhanced dramatically in single-cysteine TM2 mutants Gly80Cys and Ala81Cys. The Gly77Cys mutant also had an elevated level of cross-linking. In contrast, any of the three native cysteines (9, 78 and 79) produced level of cross-linking not much greater than the wild-type TM(1+2) protein. Similar results were obtained with cysteine-reactive cross-linker BMB (data not shown). Likewise, comparable results were obtained with single-cysteine mutants of untagged, full-length CD9, using CuP (Figure 7C) as well as DTME cross-linker (data not shown).
These cross-linking results for TM1 and TM2 are consistent with our model that places residues Leu14, Phe17 and Gly80 on the same side of the TM1-TM2 pair (Figure 6, panel C). The strong cross-linking with Leu14Cys, Phe17Cys and Gly80Cys places the intermolecular interface toward the c and g phases of the TM1 helix, and the g phase of the TM2 helix, away from its e and f faces containing wild-type cysteines 78 and 79.
Critical residues at the TM1-TM2 interface also affect dimerization indirectly. To assess specific CD9 dimerization, we used a Gly80Cys substitution at the intermolecular interface for cross-linking. As shown in Figure 8A, single replacements of conserved heptad residues in positions 18, 25, 32, 67 and 74 (Asn18Ser, Gly25/32/67/74→Leu) strongly decreased the cross-linking mediated by Cys80. The effect was most pronounced for mutations of residues, Gly32 and Gly67, located in the tightly packed extracellular end of TM helices (Figure 6). In contrast, mutations of residues closer to the cytoplasmic end of TM2 (Gly74 and especially Ala81) had only modest to very little effect on cross-linking.
Relatively low efficiency of intermolecular cross-linking via native residues Cys9, 78, and 79 (Figures 7B,C) correlates well with the predicted location of Cys78 and 79 away from the dimeric interface (Figure 3), and suggests that the extramembrane N-terminal part of CD9 (residues 1–13) does not self-associate. We next examined whether mutations of conserved Asn and Gly residues in TM1 and TM2 decreased low-level background cross-linking via native cysteines. As expected, these mutations had virtually no effect on dimer formation of CD9 TM(1+2)-GFP (Figure 8B). The level of covalent dimer formed was not diminished for triple Asn18Ser + Gly25Leu + Gly32Leu and double Gly67Leu + Gly74Leu mutants, compared to wild-type TM(1+2) CD9 molecule. Similarly, the same triple and double mutations in the context of full-length CD9-GFP protein (with six cysteines) produced wild-type levels of cross-linking (Figure 8C). We interpret these findings as evidence for at least two types of associations between CD9 molecules: primary, involving residues 14, 17 and 80, and dependent on integrity of conserved heptad residues in TM1 and TM2, and less efficient secondary interactions, probably representing random collision events, and independent of the heptad residues (see Discussion for more details).<br><br>TM3 and TM4 cysteine residues in CD9 dimerization
After identifying the roles of conserved TM1 and TM2 residues in CD9 dimerization, we next probed whether residues proximal to TM domains 3 and 4 are also involved. To this end, disulfide cross-linking of full-length CD9 molecules containing 3 C-terminal cysteines (87, just before TM3; 218 and 219 in TM4) or 3 N-terminal cysteines (9, 78 and 79) was compared (Figure 9). We found that the C-terminal cysteines were only slightly better than N-terminal cysteines with respect to detection of CD9 dimers. However, markedly more trimers and tetramers were detected using C-terminal cysteines. Thus, residues 87, 218 and 219 at TM3 and TM4 in CD9 can together form contacts across the dimeric interface and also additional contacts with other neighboring CD9 molecules.<br><br>
Discussion
Here we provide the first detailed analysis of tetraspanin protein transmembrane domains. First, we show 1) the presence of a heptad repeat motif in TM1 and TM2, containing highly conserved Asn and Gly residues, 2) a leucine and glutamate/glutamine-containing heptad motif in TM3, and 3) high variability and absence of heptad repeats in TM4 sequences. Second, we provide evidence for a specific, intramolecular interaction between TM1 and TM2 domains, in which bulky hydrophobic residues pack against GG7 motif, and present a molecular model for this interaction. Third, experimental mapping of the CD9 dimerization interface firmly establishes an additional role for conserved TM1 and TM2 residues in dimeric intermolecular interactions. Fourth, preliminary evidence is provided to suggest that TM3 and TM4 domains contribute to expansion of CD9 dimers into higher order multimers.
Conserved residues in TM1 and TM2 of tetraspanins: role in intramolecular packing
We hypothesized that the first two transmembrane domains of tetraspanins might interact with each other because: a) consecutive TM domains frequently associate in known protein 3D structures [35], and b) they both contain a series of highly conserved amino acids – several Gly residues and an Asn residue (Figure 1). Conserved Gly residues are a frequent theme in the organization of interacting transmembrane domains. Analysis of 3D helix packing in polytopic membrane proteins reveals that Gly residues tend to localize in buried positions, especially at the helix-helix interfaces and helix crossing points [37,38]. Due to the absence of a side chain, Gly provides a flat surface for packing of a side chain from another residue, without loss of side-chain entropy upon interaction. The most common Gly-containing motif is GxxxG [39,40]. In glycophorin A (GpA), the major glycoprotein in erythrocyte cell membranes, Gly79 and Gly83 are part of the LIxxGVxxGVxxT sequence that promotes homodimerization of parallel transmembrane α-helixes [41,42]. In the GpA dimerization motif, Gly residues allow for tight packing in the right-handed helical crossing [43]. There are also examples of left-handed helical crossing in the context of a GxxxG motif [44]. Other membrane proteins that use the GxxxG motif for homo- or heterodimerization include <span class="yellow">bacteriophage M13</span> coat proteins [45], <span class="blue">yeast</span> alpha factor receptor [46], integrin α IIb subunit [47], and ErbB1 receptor tyrosine kinase [48]. Other small residues, such as Ala and Ser, can substitute for Gly in this motif [49].
A protein motif in which Gly residues are separated by 6 other residues (GG7) is also common in transmembrane helices, especially in transporter/channel-like membrane proteins [50]. However, the structural features associated with this motif are not well known. In particular, it is unclear whether left-handed GG7 heptad repeat motif (as opposed to the "classic" right-handed GxxxG motif) can drive membrane helix association. In a recent work addressing this issue, Lear et al. [51] showed that a synthetic peptide containing Gly at heptad positions a and d could self-associate in vitro, likely in an antiparallel orientation. Heptad repeats containing conserved Gly residues occur in TM domains of α and β chains of MHC class II proteins, and mutations of the Gly residues disrupt the αβ heterodimer [52]. These examples demonstrate that Gly-based heptad motifs may be used for both intra- and intermolecular associations.
In this work, we identified a highly conserved GG7 motif in the first two tetraspanin TM domains. The GG7 sequence in tetraspanins is a part of a larger motif that also includes a conserved Asn residue in TM1 (NGG7) and an Ala/Ser/Thr residue in TM2 (GGA7). The seven-residue periodicity of these motifs strongly suggests their involvement in left-handed coiled coil packing reminiscent of the leucine zipper, rather than right-handed packing of the GpA-like GxxxG motif. For antiparallel helices, the left-handed crossing is in fact predominant over the right-handed in known TM domain structures [44].
In our model, heptad Gly residues in NGG7 and GGA7 sequences provide specific packing between antiparallel tetraspanin TM1 and TM2 helices by allowing tight van der Waals interactions with large hydrophobic residues (Figure 6). Highly efficient packing of bulky side chains against glycine residues is observed in known transmembrane protein 3D structures [38,53,54]. An example includes packing of helices M1 and M2 in potassium channel KcsA, where Val91 in M2 is paired with Gly43 in M1, and Leu36 in M1 contacts Ala98 and Gly99 in helix M2 [54,55]. In addition to facilitating helix-helix packing, Gly residues frequently provide additional CαH...O hydrogen bonds between two helices [44]. In our model, two Cα-backbone carbonyl H-bonds are predicted – between residues Gly27-Gly67, and Trp22-Met70.
Although polar and charged amino acid residues (such as Asn in the TM1 heptad motif) are infrequent in transmembrane domains, they are functionally important. Polar residues such as glutamine, glutamic acid, aspartic acid and asparagine can promote strong oligomerization of model membrane-associated helices [56-58]. Ruan et al. [59] used asparagine scanning mutagenesis to probe the interface of self-associating polyleucine helices by detecting their enhanced self-interaction in vitro and in the <span class="yellow">E. coli</span>-based ToxR assay. Thus, a hydrogen bond in an apolar environment can result in strong, though not necessarily specific, association of transmembrane helices. In fact, mutations to polar residues in transmembrane proteins are commonly associated with disease [60]. Because of this potential for non-specific interactions, polar residues tend to localize at buried positions in TM domains.
In our case, the conserved Asn18 residue in CD9 is predicted to be a part of the TM1-TM2 interface, though our model does not predict any electrostatic interaction between Asn18 and TM2 (Figure 6). Consistently, substitution such as Asn18Tyr (and Gly77Leu) in TM(1+2)-GFP protein was not destabilizing as analyzed by protein aggregation. Curiously, the full-length Asn18Ser CD9 migrated slightly slower on SDS-PAGE gel (data not shown), suggesting that Asn18 does play a role in maintaining conformation of the molecule. The Asn18Cys single-cysteine mutant shows very little intermolecular cross-linking (Figure 7A), supporting the proposed location of this residue at the intramolecular interface. It is tempting to speculate that the "pocket" between TM1 and TM2 lined by Asn18 and Gly77 might be important for accommodating palmitate moieties that target Cys78 and Cys79 residues, and/or important for access by the putative palmitoyl transferase to those residues. Understanding the exact role of these highly conserved Asn18 and Gly77 residues in tetraspanins awaits further investigation.
In summary, we identified conserved glycine residues of TM1 and TM2 of tetraspanins as key elements required for intramolecular packing. Mutations of these key residues (Gly25, Gly32, Gly67 and Gly74 in CD9) resulted in protein destabilization and aggregation. There is ample evidence in the literature for mutations in transmembrane proteins that lead to protein destabilization, misassembly and pathologic conditions [61]. Thus, we have identified conserved heptad Gly residues in TM1 and TM2 of tetraspanins as plausible targets of destabilizing mutations with potential functional consequences.<br><br>Intermolecular interactions in tetraspanins
Tetraspanin CD9 forms mostly homodimers, but also a low level of heterodimers with CD81 and CD151 [25]. Thus, mapping the dimerization interface is an important next step in structure-function analysis of tetraspanins. Disulfide-mediated cross-linking, often in combination with cysteine-scanning mutagenesis, is a common strategy to probe oligomerization or intersubunit interactions of transmembrane proteins such as histidine kinase EnvZ [62], M(3) muscarinic acetylcholine receptor [63], <span class="yellow">E. coli</span> lactose permease [64], synaptobrevin [65], integrins [66] and many others. In tetraspanins such as CD9, membrane-proximal cysteine residues are especially useful targets for disulfide trapping, as their linkage can be enhanced by pre-treating cells with 2-BP. While the ability of wild-type cysteines in CD9 to be cross-linked may indicate that they are close to the dimerization interface, more precise mapping was achieved here using cysteine-scanning mutagenesis.
Our data clearly identify regions, near the cytoplasmic face of TM1 and TM2, important for dimerization. Intermolecular zero-length cross-linking was highest when single cysteines were placed in positions 14, 17, 77, 80 and 81 in TM(1+2)-GFP molecule, or at positions 77, 80 or 81 in the full-length CD9 protein. Positions 14, 17 and 80 are distinct from the intramolecular interface and are on the same side of the TM1-TM2 pair (Figure 6). Thus, they are well-positioned to participate in an interaction with another molecule. At the same time, the model predicts that wild-type cysteines (Cys78, 79), which do not yield very efficient zero-length cross-linking, are on the other side of TM1-TM2 pair.
While using the cysteine at position 80 as the dimeric interface probe, mutations of conserved residues in TM1 and TM2 (especially Gly32 and Gly67 to Leu) clearly reduced intermolecular cross-linking. We do not suggest that those residues are directly involved in intermolecular interaction. Rather, we propose that destabilization of the intramolecular TM1-TM2 interaction by Gly to Leu substitutions (discussed above) causes an overall conformational change that reduces dimer formation.
An Ala81Leu mutation did not reduce cross-linking via Cys80, even though single-cysteine Ala81Cys molecules themselves produced a high level of cross-linking. These results, together with data on Gly32Leu and Gly67Leu mutations, are consistent with our model predicting that helices 1 and 2 interact more tightly near the extracellular end and less at the cytoplasmic end. This would give more flexibility to a cysteine at position 81 and also limit the effect of an Ala81Leu mutation. Location of this residue at the membrane/cytoplasmic border could also make it more accessible to CuP reagent as compared to residues buried in TM domain, thus elevating the efficiency of disulfide formation of the Ala81Cys mutant.<br><br>Multiple interfaces in tetraspanin molecules
In the full-length CD9 molecules, the 3 C-terminal cysteines (Cys87, 218 and 219) located at or in TM3 and TM4 promoted efficient dimer and even more efficient oligomer formation compared to the 3 N-terminal cysteines (Figure 9). Cys87 alone can be used to capture CD9 dimers [25]. These results suggest the existence of two dimeric interfaces in CD9 molecule – the TM 1-2/1-2 interface and the TM 3-4/3-4 interface (Figure 10). In a TM(1+2) molecule, the destabilization of 1–2 interaction, e.g. by Gly→Leu mutations, would affect the 1-2/1-2 interface, as discussed above. However, these mutations would not interfere with the 3-4/3-4 interface in a full-length molecule, which includes Cys87, 218 and 219. Thus, cross-linking of full-length molecules, containing all 6 cysteine residues, would be unaffected, as seen in Figure 8C. Furthermore, wild-type Cys9, Cys78 and Cys79 are apparently not at the primary 1-2/1-2 interface. Their relative inefficiency in cross-linking CD9 TM(1+2) protein likely reflects weak secondary contacts between the molecules, or possibly random collision events. Such events should be independent of mutations in the conserved Gly residues in TM1 and TM2, as was demonstrated in Figure 8B. The potential existence of two interfaces in tetraspanin molecules, 1-2/1-2 and 3-4/3-4, should provide enhanced flexibility for forming additional intermolecular contacts. Current understanding of tetraspanin microdomains assumes a few strong, primary homotypic and heterotypic tetraspanin complexes (e.g. CD9-CD9, CD9-CD81, CD151-α3 integrin, CD81-EWI2) that help bring together various other proteins, forming secondary-type associations. Such properties of tetraspanins may bring signaling molecules such as protein kinase C or phosphatidylinositol 4-kinase to the vicinity of integrins [67,68].
The organization of the TM3 domain points to a potential role in protein-protein interactions. A motif of Leu-Leu-Glu(Gln) spaced 7 residues apart (heptad positions a), with highly conserved residues in two consecutive positions d, poses as a likely interaction module. If responsible for heterologous protein-protein interactions, it would form another distinct interface of tetraspanin molecule. Our preliminary data indicate that replacing the Leu and Glu residues in TM3 of CD9 with Ala has no effect on cell surface expression of the protein and its dimerization (data not shown). It remains to be tested if interactions with other proteins will be affected. Similarly, the TM4 domain may provide additional contributions to lateral tetraspanin associations. Much higher sequence variability, and the lack of a distinct heptad pattern suggests that TM4 is a major contributor to diversity among tetraspanin complexes. Structure-function analysis of TM3 and TM4 domains in tetraspanins is the subject of ongoing investigation.<br><br>
Conclusion
We have defined the TM1-TM2 intramolecular interface in tetraspanin CD9, providing evidence for glycines (Gly25 and Gly32 in TM1, Gly67 and Gly74 in TM2) packing against apposing bulky aliphatic residues. Second, we mapped an intermolecular CD9 interface (involved in CD9 homodimer formation) to the vicinity of residues Leu14 and Phe17 in TM1 and Gly77, Gly80 and Ala81 in TM2. Finally, we provide preliminary evidence that TM3 and TM4 in CD9 may contribute to a second intermolecular interface. Key CD9 residues involved in intra- and intermolecular interactions are highly conserved throughout the tetraspanin family, thus suggesting that our findings will apply to most tetraspanins.<br><br>Methods
Materials
Cell culture reagents were from Invitrogen (Carlsbad, CA). 2-bromopalmitate was from Fluka (Milwaukee, WI), N-ethylmaleimide (NEM) and 1,10-phenanthroline were from Sigma-Aldrich (St. Louis, MO), and chemical cross-linkers dithio-bis-maleimidoethane (DTME) and 1,4-Bis-maleimidobutane (BMB) were purchased from Pierce Endogen (Rockford, IL). Triton X-100, protease inhibitor cocktail and FuGENE 6 transfection reagent were obtained from Roche (Indianapolis, IN). Restriction endonucleases and Pfu DNA polymerase were obtained New England Biolabs (Beverly, MA) and Stratagene (Carlsbad, CA), respectively. All other chemicals were purchased from Sigma-Aldrich or Fisher Scientific (Pittsburg, PA).<br><br>Sequence analysis
Tetraspanin sequences were obtained from SWISS-PROT and GenBank databases. Locus designations, accession numbers and the most commonly used protein names are summarized in Tables 1 and 2. TM segments were delineated by inspection of hydrophobicity profiles, using database annotations and previous analyses of TM sequences as a guide ([28], M. Hemler, unpublished), and aligned manually. Residue numbers in <span class="yellow">human</span> CD9 sequence are used a reference point throughout the study.<br><br>DNA cloning and mutagenesis
Sequence encoding CD9 protein was cloned into vector pcDNA3 (Invitrogen, Carlsbad, CA) and pEGFP-N1 (Clontech, Palo Alto, CA), for expression of untagged and C-terminally GFP-tagged CD9, respectively. pEGFP-N1 encoding CD9 TM(1+2) -GFP fusion protein was constructed by subcloning DNA for residues 1–83 of CD9 into HindIII and PstI sites of the vector; to introduce the PstI site, codon GTG for Val82 was changed to CTG (coding for Ala). In the resulting fusion protein, there is a 13-amino acid linker (with no cysteines) between CD9 and GFP. To minimize the low inherent ability of GFP to homodimerize, which could potentially influence the results of CD9 cross-linking, we used a monomeric GFP mutant, Leu221→Lys [34], for cross-linking experiments.
Mutations were introduced in full-length and TM(1+2) CD9 proteins by a PCR-based strategy using mutagenic primers and Pfu DNA polymerase. All mutations were confirmed by DNA sequencing.<br><br>Protein expression, microscopy, cysteine disulfide cross-linking and Western blotting
DNA constructs encoding TM(1+2)-GFP or full-length CD9 proteins were transfected into <span class="yellow">human</span> rhabdomyosarcoma RD cells using the FuGENE 6 reagent. Cells expressing GFP fusion proteins were analysed by fluorescence microscopy 18–28 hours post-transfection. Images were captured using Spot 1.4.0 camera (Diagnostic Instruments, Sterling Heights, MI) attached to Nikon Eclipse TE300 microscope.
For experiments involving cysteine-mediated cross-linking, cells were treated with 50 μM 2-BP starting 24–26 hours post-transfection and continuing for 16–18 hours. Cross-linking was carried out by incubating cells in HBSM buffer (25 mM Hepes-NaOH, pH 7.2, 150 mM NaCl, 2 mM MgCl2) containing either a) 0.6 mM CuSO4 and 1.8 mM 1,10-phenanthroline (CuP complex) or b) 0.2 mg/ml homobifunctional cysteine-reactive cross-linker (e.g. DTME), diluted from fresh 10 mg/ml solution in DMSO. After incubation for 10–15 minutes (with CuP) or 30–45 minutes (with cross-linker), cells were washed twice for 10 minutes with HBSM containing 10 mM NEM to block residual free cysteines. Cells were lysed in HBSM containing 1% Triton X-100, 0.1% SDS and a cocktail of protease inhibitors with 1 mM EDTA at 4°C for 45–60 minutes. Cell lysate was clarified by centrifugation at 14,000 × g for 15 minutes, an aliquot was removed, and proteins from it were precipitated by addition of trichloroacetic acid to 10% on ice followed by centrifugation at 14,000 × g for 10 minutes. After two washes with ice-cold acetone, protein pellet was solubilized in SDS-PAGE sample buffer without a reducing agent (50 mM Tris-HCl, pH 6.8, 1% SDS, 8% glycerol).
In some experiments, CD9 protein was immunoprecipitated using monoclonal antibody Alb6 (Immunotech, Marseille, France). Proteins were separated by SDS-PAGE and analyzed by Western blotting using monoclonal antibody JL-8 (Clontech) for GFP or Alb6 for CD9. Bands from X-ray films were quantitated using GeneTools™ software from Syngene (Frederick, MD).<br><br>Modeling of TM1-TM2 interaction
An atomic model of the CD9 TM1-TM2 dimer was constructed with a Monte Carlo-simulated annealing (MCSA) algorithm [69]. Two idealized α-helices corresponding to TM1 residues Tyr12 through Leu35 and TM2 residues Gly59 through Val82 were docked with six orthogonal parameters: three rigid body translations and three rotations. During each step of a MCSA cycle, there was an equal probability of changing either one parameter or all six parameters to random values. A conformation's energy was calculated in vacuo with the AMBER united-atom force field for van der Waals interactions [70]. The van der Waals term was modified as described by Kuhlman and Baker [71]. If a structure had favorable dimerization energy, the energies of select mutants were calculated. Structures were selected with a novel scoring function that maximizes the Boltzmann probability of dimerization for silent mutations while minimizing the probability for disruptive mutations. Asn18Ser, Gly77Leu, and Gly80Leu were scored as silent mutations while Gly25Leu, Gly32Leu, Gly67Leu, and Gly74Leu were considered to be disruptive. Each MCSA cycle consisted of 50,000 steps with an exponential temperature decay from 10,000 to 10 K.
Ten MCSA cycles through global sample space were used to restrict the search area. Parameters were restricted to ± 2 standard deviations from their mean value for structures within 10 kcal of the best structure. MCSA cycles were repeated as described above with additional optimization of χ values: rotamers at the protein-protein interface were optimized with Dead End Elimination [72], and χ values were further optimized with Monte Carlo. All MCSA cycles converged upon structures that were within a root mean squared deviation (RMSD) of 1.5 Å with the best structure, and structures that scored within 5 kcal of the best score had an RMSD of less than 0.5 Å with the best structure.<br><br>
List of abbreviations
2-BP, 2-bromopalmitate; CuP, Cu2+-phenanthroline; DTME, dithio-bis-maleimidoethane; LEL, large extracellular loop; NEM, N-ethylmaleimide; TM, transmembrane (domain).<br><br>Authors' contributions
OVK carried out sequence comparisons, mutational analysis and cross-linking experiments, and drafted the manuscript. DGM built the TM1-TM2 interaction model and contributed to the manuscript. WFD supervised DGM's work. MEH coordinated the whole study and prepared the final manuscript.<br><br>
<h3>pmcA140510</h3>Enp1, a <span class="blue">yeast</span> protein associated with U3 and U14 snoRNAs, is required for pre-rRNA processing and 40S subunit synthesis
Abstract
ENP1 is an essential <span class="yellow">Saccharomyces cerevisiae</span> gene encoding a 483 amino acid polypeptide. Enp1 protein is localized in the nucleus and concentrated in the nucleolus. An enp1-1 temperature-sensitive mutant inhibited 35S pre-rRNA early processing at sites A0, A1 and A2 as shown by northern analysis of steady state levels of rRNA precursors. Pulse-chase analysis further revealed that the enp1-1 strain was defective in the synthesis of 20S pre-rRNA and hence 18S rRNA, which led to reduced formation of 40S ribosomal subunits. Co-precipitation analysis revealed that Enp1 was associated with Nop1 protein, as well as with U3 and U14 RNAs, two snoRNAs implicated in early pre-rRNA processing steps. These results suggest a direct role for Enp1 in the early steps of rRNA processing.<br><br>INTRODUCTION
Ribosome biogenesis is one of the major cellular activities in eukaryotic cells. It takes place primarily in a specialized subnuclear compartment, the nucleolus (1,2). In the <span class="blue">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, each rRNA gene is transcribed by RNA polymerase I into a 35S rRNA precursor, consisting of 18S, 5.8S and 25S rRNA sequences flanked by two external transcribed spacers (ETS) and separated by two internal transcribed spacers (ITS) (Fig. 1). This 35S precursor goes through a series of modifications and processing steps to generate the mature 18S, 5.8S and 25S rRNAs. The processing occurs first at sites A0, A1 and A2, resulting in the 20S pre-rRNA and 27SA2 pre-rRNA. The 20S pre-rRNA is then cleaved, leading to the mature 18S rRNA found in the 40S ribosomal subunit. The 27SA2 pre-rRNA is processed through two alternative pathways. The majority of 27SA2 pre-rRNA is cleaved at sites A3 and B2 to form the 27SA3 pre-rRNA, which is subsequently processed to produce 27SBS. Alternatively, 27SA2 pre-rRNA can be processed at sites B1L and B2 to generate 27SBL pre-rRNA. Both 27SBS and 27SBL pre-rRNAs are then cleaved at sites C1 and C2 to generate the mature 25S rRNA, and 7SS or 7SL intermediates, which are then processed to mature 5.8SS or 5.8SL rRNAs (Fig. 1). The 25S rRNA and 5.8S rRNA are the RNA components of the 60S ribosomal subunit (3). During the course of rRNA modification and processing, many of the ribosomal proteins are assembled onto the rRNA molecules to form the ribosome complex.
Ribosome biogenesis needs a large number of trans-acting factors, including small nucleolar RNAs (snoRNAs), protein components of the snoRNP complexes, rRNA modifying enzymes, endo- and exonucleases, putative RNA helicases and other protein factors (3,4). In <span class="blue">yeast</span> cells there are more than 100 different snoRNAs playing important roles in rRNA modification and processing. On the basis of their structure, the snoRNAs can be divided into two groups: the box C/D family and box H/ACA family. Only one snoRNA, MRP RNA, belongs to neither family (5–7). While the majority of the snoRNAs participate in RNA pseudouridylation and 2′-O-ribose methylation, a few of them, including MRP snoRNA, box C/D snoRNAs U3 and U14, and box H/ACA snoRNAs snR10 and snR30, are required for processing of the pre-rRNA (8–12). Not only are U3, U14, snR10 and snR30 essential for early cleavages of 35S pre-rRNA to 18S rRNA, the proteins associated with them, including Nop1, Nop5, Gar1 and Nop10, have also been shown to be required for 18S rRNA synthesis (13–16).
The ENP1 (Essential Nuclear Protein 1) gene was identified in a genetic screen for suppressors of an ost4 mutation (oligosaccharide transferase 4) (17). However, in subsequent studies it was found not to be involved in OST4 function (18). ENP1 is an essential gene encoding a 483 amino acid polypeptide. The Enp1 protein is highly conserved and homologs are found in all eukaryotes. The <span class="blue">yeast</span> protein was localized to the nucleus in a previous study (18). On the other hand, an Enp1 <span class="yellow">human</span> homolog, called bystin, was reported to localize to the cytoplasm and was proposed to be involved in cell adhesion (19).
In this study, we report that the Enp1 protein not only is nuclear but is enriched in the nucleolus. We also found that Enp1 is required for the synthesis of 40S ribosomal subunits, and its presence is necessary for the pre-rRNA processing to form 20S pre-rRNA and 18S rRNA. An association between Enp1 and U3 and U14 snoRNAs, and with the nucleolar protein Nop1, was established. We also found that <span class="yellow">human</span> Enp1, expressed in <span class="blue">yeast</span>, was located in the nucleus and the nucleolus, suggesting that the function of this protein is conserved.<br><br>MATERIALS AND METHODS
<span class="yellow">Yeast</span> strains and media
The <span class="yellow">S.cerevisiae</span> strains used in this study are all derivatives of a wild-type diploid strain W303 (MATa/MATα ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 his3-11,15/his3-11,15 ade2-1/ade2-1 can1-100/can1-100) except for strain RS1938. Strain JBY45 (MATa/MATα ENP1/Δenp1::his5+) was constructed by replacing one copy of the ENP1 open reading frame (ORF) with the <span class="yellow">Schizosaccharomyces pombe</span> his5+ gene (18). Strain JBY46 [MATa Δenp1::his5+/pJB23 (ENP1, URA3, CEN6)] is a haploid strain derived from JBY45 with wild-type ENP1 on a low copy number plasmid. Strain JBY48 [MATa Δenp1::his5+/pJB19 (enp1-1, TRP1, CEN6)] and strain JBY49 [MATa Δenp1::his5+/pJB39 (enp1-2, TRP1, CEN6)] have plasmids with enp1 temperature-sensitive (ts) mutations. Strain JBY51 (MATa Δenp1::his5+ TRP1::enp1-1) is an enp1 ts strain generated by integrating the enp1-1 gene at the chromosomal trp1-1 locus. Strain CWY13 [MATa Δenp1::his5+/pJB24 (pMET25-GFP-ENP1, URA3, CEN6)] has GFP-ENP1 under control of the MET25 promoter. Strain CWY14 (MATa ENP1-TAP) was created by fusing sequences encoding a Tandem Affinity Purification (TAP) tag (20,21) to the 3′ end of ENP1. Strain YRH39 (MATα HST2-TAP) was created by fusing sequences encoding a TAP tag to the 3′end of HST2. Strain RS1938 (Δnop1::URA3 pUN100-ProtA-NOP1) was created by transforming RS1935 (MATa/α leu2/leu2 ura3/ura3 lys2/lys2 ade2/ade2 Δnop1::URA3/NOP1) with pUN100-ProtA-NOP1 (strain and plasmid supplied by T. Schafer), followed by tetrad dissection. Strain CWY15 [MATa/pCW109 (pMET25-GFP-hENP1, URA3, CEN6)] has a <span class="yellow">human</span> homolog of Enp1 expressed in W303-1a. The media used were prepared as described (22).<br><br>Cloning of ENP1
pRS426-MEG1 (The original name of ENP1) was a gift from Dr William J. Lennarz. It contains the ENP1 gene within a 2.6 kb EcoRI genomic fragment cloned into pRS426 (18). The EcoRI fragment was subsequently cloned into pRS314 (TRP1, CEN6), pRS316 (URA3, CEN6) and pRS424 (TRP1, 2µ) to create pJB20, pJB23 and pJB21, respectively. pGFP-N-FUS is a centromeric plasmid for fusing green fluorescence protein (GFP) to a polypeptide’s N-terminus under control of the MET25 promoter (23). Cloning the ENP1 ORF into pGFP-N-FUS via XbaI and SalI sites generated plasmid pJB24. pJB19 (enp1-1, TRP1, CEN6) contains the enp1-1 mutant gene. The <span class="yellow">human</span> homolog of <span class="blue">yeast</span> Enp1 was PCR amplified from a <span class="yellow">human</span> cDNA clone BC007340 (Research Genetics), then cloned into pGFP-N-FUS, p415-ADH, p415-GPD and p415-TEF (24) via XbaI and SalI sites to generate pCW109, pCW113, pCW115 and pCW117, respectively. The fragment of the <span class="yellow">human</span> Enp1 homolog (amino acids 152–437) was also cloned into these vectors to generate pCW110, pCW114, pCW116 and pCW118.<br><br>Random mutagenesis of ENP1 to generate enp1 ts mutants
The enp1 ts mutants were generated with a protocol introducing random mutations by PCR (25). The PCR was performed with oligonucleotides ENP1-MUT5′ (5′-GGTGGTGTCAGT AGGGGA-3′), ENP1-MUT3′ (5′-CAGTCTGCAATATA TGGAC-3′) and plasmid template pJB20 (see above). The nucleotide concentrations in the reaction were 1 mM each for dATP, dGTP, dCTP and dTTP. After strain JBY46 was transformed with the PCR product and with pJB20 gapped by NheI and NsiI, the transformants were replica-plated onto two plates containing 5-FOA synthetic medium without tryptophan; one replica was incubated at 23°C and the other at 37°C. Colonies growing at 23°C but not at 37°C were picked as candidates. Since approximately 10% of the colonies were inviable at both temperatures, we knew that 10% of the ENP1 PCR products lost their function after the PCR mutagenesis. This confirmed the effectiveness of the mutagenesis procedure. Two candidates showed good growth at 23°C but no growth at 37°C. They were named strains JBY48 and JBY49 and contained plasmids with the enp1-1 and enp1-2 mutations, respectively. The mutated enp1-1 gene on pJB19 was cloned into an integration vector, pRS304, as an EcoRI fragment. The plasmid was subsequently linearized with SnaBI within the TRP1 marker and was integrated at the chromosomal trp1-1 locus of strain JBY46. Selection on 5-FOA was used to remove plasmid pJB23, creating strain JBY51.<br><br>Sucrose gradient analysis
Polyribosome preparation and analysis were carried out essentially as described (26). Cells grown in YPD were collected at mid-log phase (OD600 0.8–1.0) and were broken with glass beads. The lysate was frozen immediately in liquid N2 and was stored at –80°C. Lysate (30 U of absorbance at OD260) was layered over a 7–47% (w/v) sucrose gradient, which was centrifuged at 28 000 r.p.m. for 5 h at 4°C in a SW28 rotor and was analyzed with an ISCO UA-5 gradient UV detection system on absorbency at 254 nm.<br><br>Immunofluorescence
Immunofluorescence analysis was carried out essentially as described (27). Cells were fixed with 3.7% formaldehyde at room temperature for 1.5 h. Antibodies included a <span class="yellow">mouse</span> monoclonal anti-Nop1 (a gift from John P. Aris, University of Florida, Gainesville, Florida) and a Texas-red-conjugated <span class="yellow">donkey</span>-anti-<span class="yellow">mouse</span> antibody (Jackson Lab), both used at 1:500 dilution. Images were taken on a Zeiss Axioplan2 microscope equipped with a Zeiss AxioCam camera.<br><br>Pulse-chase labeling analysis of rRNA
For [methyl-3H]methionine pulse-chase analysis, cells were grown in synthetic medium without methionine at room temperature or 37°C for 2 h. When the OD600 reached 1.0, 6 ml of the culture were pulse-labeled with 250 µCi [methyl-3H]methionine (Amersham Pharmacia) for 3 min and chased with cold methionine (500 µg/ml) for 2, 4 or 12 min. For each time point of the chase 1.25 ml of culture was mixed with ice and collected. The pellets were frozen immediately in liquid N2 and stored at –80°C before total RNA was purified using a hot phenol method (28). The RNAs were separated on a 1.2% agarose formaldehyde gel and transferred onto Hybond-N+ nylon membranes (Amersham Pharmacia). After being sprayed with EN3HANCE (Du Pont), the membranes were exposed to film at –80°C (29).<br><br>Northern analysis
Cells were grown inYPD at room temperature or 37°C for 2–4 h. When the OD600 reached 1.0, 10 ml of cells were collected and frozen immediately. Total RNA was extracted as described (28). Five micrograms of RNA were separated on 1.2% agarose formaldehyde gels (for high molecular weight RNA) or on 6% polyacrylamide 7 M urea denaturing gels (for low molecular weight RNA) for each sample. The RNA was then transferred to Zeta-Probe GT nylon membranes (Bio-Rad). Probes for hybridyzation were specific oligonucleotides end-labeled using [γ-32P]ATP. After hybridization and washes, membranes were exposed to phosphorimager screens or X-ray films (29). Oligonucleotides specific for various regions of 35S pre-rRNA are: 1, GGTCTCTCTGCTGCCG GAAATG; 3, AATGAGCCATTCGCAGTTTCACTG; 4, GCTCTCATGCTCTTGCCAAAAC; 5, TGTTTGTTACCT CTGGGCCCCG; 6, TCCAGTTACGAAAATTCTTG; 7, CGTATCGCATTTCGCTGCGTTC; 8, GTTCGCCTAGAC GCTCTCTCTTC; 9, GCGAGATTCCCCTACCCAC. The regions complementary to these oligonucleotides are shown in Figure 6A. The oligonucleotides probing box C/D snoRNAs are: U3, TTCGGTTTCTCACTCACTCTGGGGTAC; U14, GGAACCAGTCTTTCATCACCGTG. The oligonucleotides probing box H/ACA snoRNAs are: snoRNA10, CCTTG CAACGGTCCTCATCCGGG; snoRNA30, GTCCGAAGC GCCATCTAGATGA.<br><br>RNA and protein precipitation analysis
Cells were grown in YPD (1 l) to OD600 1.0, then collected, washed once in ice-cold PBS and broken using glass beads in 10 ml IPP150 buffer (10 mM Tris pH 8.0, 150 mM NaCl and 0.1% NP-40) with protease inhibitors. The lysates were mixed with 200 µl IgG agarose beads for 2 h at 4°C. After several washes with 50 ml IPP150 buffer, the IgG beads were collected and the RNA associated with the beads extracted by the hot phenol method (28). One-tenth of the precipitated RNA was then separated on 6% polyacrylamide 7 M urea denaturing gels and electro-transferred to Zeta-Probe GT nylon membranes. They were probed with [γ-32P]ATP labeled oligonucleotides. For the lanes showing total RNA, 2 µg RNA prepared from exponentially growing cultures were loaded. For co-precipitation analysis of proteins, the IgG beads were resuspended in 2× Laemmli sample buffer, boiled for 5 min and separated by SDS–polyacrylamide gel electrophoresis. Approximately 1% of the precipitate was loaded onto each lane. For the total protein, 0.1% of the extract prior to precipitation was loaded onto each lane. Following electrophoresis, the proteins were transferred to nitrocellulose membranes and detected using anti-Nop1 antibody at 1:3000 dilution (provided by J. Aris) and anti-L3 antibody (provided by J. Warner) at 1:3000 dilution followed by peroxidase-conjugated anti-<span class="yellow">mouse</span> secondary antibody at 1:5000 dilution. The antibody complexes were detected using ECL-Plus reagents (Amersham Pharmacia) as specified by the manufacturer.<br><br>
RESULTS
Construction and analysis of ENP1 temperature-sensitive alleles
ENP1 is an essential <span class="blue">yeast</span> gene conserved among eukaryotes (18). To study its functions, we first created a diploid strain, JBY45, heterozygous for the Δenp1 mutation. Then two enp1 ts mutant alleles (enp1-1 and enp1-2) were generated by random PCR mutagenesis, as described in Materials and Methods. Both mutant alleles were recessive to the wild-type ENP1 gene. The enp1-1 gene was integrated into the chromosomal trp1-1 locus to create strain JBY51 and all subsequent experiments were done with this enp1 allele. At 23°C the growth of JBY51 was comparable to that of W303-1a (ENP1), but at 37°C it did not grow (Fig. 2). Sequence analysis revealed that enp1-1 contains two point mutations, resulting in substitutions of two amino acids: W242→G and V415→A. No attempt was made to determine whether both mutations were required for the ts phenotype. Flow cytometry of strain JBY51 showed DNA content profiles similar to those of the wild-type strain at the non-permissive temperature (data not shown), suggesting that ENP1 is not involved in cell cycle regulation.<br><br>Enp1 is enriched in the nucleolus
Enp1 tagged at the C-terminus with the myc epitope was previously found to localize to the nucleus (18). To expand this analysis, we fused GFP to the N-terminus of Enp1, and expressed the tagged Enp1 protein under control of the MET25 promoter as the sole source of Enp1 protein in the cells. Cells of strain CWY13 (pMET25-GFP-ENP1) were grown in media with methionine (transcription partially repressed) or without methionine (transcription not repressed). Growth of CWY13 was comparable to that of wild-type cells in both media (data not shown), indicating that the GFP tagged Enp1 protein was functional. In cells cultured in medium without methionine, in which the ENP transcription was not repressed, the GFP-Enp1 protein was expressed at a high level and showed strong green fluorescence distributed throughout the nucleus (data not shown), consistent with the original observation (18). With methionine added to the medium, however, the fluorescent signal of GFP-Enp1 was weaker and surprisingly showed crescent or cap-like patterns typical of nucleolar proteins (Fig. 3A). This nucleolar enrichment was confirmed by its co-localization with the nucleolar protein Nop1 (Fig. 3B).<br><br>ENP1 mutation leads to reduced levels of 40S ribosomal subunits
The nucleolar enrichment suggested that Enp1 might play some role in ribosome synthesis. To test this possibility, cells of W303-1a (ENP1), JBY51 (enp1-1) and a top2 ts strain RS191 (30) were grown in YPD at 23 or 37°C and their ribosome profiles analyzed after separation on sucrose gradients. At 23°C, enp1-1 cells showed polysome profiles similar to those of the wild-type and top2-1 strains (Fig. 4A–C). In contrast, after incubation at the non-permissive temperature (37°C) for 40 min (data not shown) and 2 h, enp1-1 cells had reduced levels of 40S ribosomal subunits, 80S monosomes and polysomes, along with a dramatic increase in the free 60S subunit peak (Fig. 4F), while the wild-type and top2-1 cells showed little change in polysome profile at 37°C (Fig. 4D and E). These results demonstrate that the changes of the enp1-1 polysome profile were due to specific defects caused by the enp1-1 mutation, and not due simply to the shift to 37°C for a wild-type or ts strain.<br><br>The processing of pre-rRNA for 18S rRNA is impaired in enp1 mutants
In most cases reductions in ribosomal subunit levels are the results of defects in pre-rRNA processing or ribosome assembly or both (3). To study the mechanism by which Enp1 affects the 40S subunit, we analyzed the effects of enp1 mutations on processing of the pre-rRNA using pulse-chase labeling. [methyl-3H]methionine is preferred for labeling rRNAs in pulse-chase analysis because rRNAs are specifically methylated during the early steps of the processing. Cells of W303-1a (ENP1) and JBY51 (enp1-1) were grown at 23 or 37°C for 2 h before they were labeled with [methyl-3H]methionine for 3 min and chased with cold methionine for 2, 4 and 12 min. In wild-type cells, the labeled 35S rRNA precursor, 27S and 20S rRNA intermediates were rapidly chased into mature 25S and 18S rRNAs (Fig. 5), at 23 or 37°C. In contrast, enp1-1 cells showed dramatic changes after incubation at 37°C. Although at 23°C the synthesis and processing were comparable to those of wild-type cells, cells cultured at 37°C for 2 h had neither 20S pre-rRNA nor 18S rRNA, while 25S rRNA was generated at normal levels. The 37°C grown enp1-1 cells also had low levels of aberrant 23S and possibly 21S intermediates (Fig. 5). The defect in 18S rRNA synthesis was further supported by pulse-chase experiments carried out using [5,6-3H]uracil. The results obtained were essentially the same; no 20S pre-rRNA or 18S rRNA was produced, while processing to 25S rRNA was not affected (data not shown).
Taken together, these results suggested that the enp1 mutation leads to specific defects in the processing pathway for 20S pre-rRNA and 18S rRNA, resulting in reduced 40S subunit synthesis and a lowered level of the 40S subunit.<br><br>Enp1 is required for pre-rRNA processing at A0, A1 and A2 sites
To define the processing steps affected by the enp1 mutation, the steady state levels of rRNA precursors and mature RNAs were analyzed by northern blotting. Total RNAs were isolated from cells of W303-1a (ENP1) and JBY51 (enp1-1) grown at 37°C for 2 or 4 h. After separation on formaldehyde agarose, RNAs were transferred onto nylon membranes and probed with radiolabeled oligonucleotides specific to various regions of the 35S pre-rRNA (Fig. 6A). After incubation at 37°C, enp1-1 cells had wild-type levels of 25S rRNA, but reduced levels of 18S rRNA (Fig. 6B). A probe specific to ITS1 (P5) further revealed that enp1-1 cells incubated at the non-permissive temperature accumulated two aberrant rRNAs, 23S and 21S (Fig. 6C). Aberrant rRNA processing products of similar sizes have been described in numerous studies (14,31–34) and result from defects in processing at A0, A1 and A2. Additional probes were used to verify the origins of the 23S and 21S RNAs in the enp1-1 strain. The 23S product was detected by oligos 1, 2, 4 and 5, but not by oligos 6, 7 or 8 (Fig. 6D, E, C, F, G and data not shown). Thus this RNA extends from the 5′ end of the 35S rRNA to the A3 site. The 21S product was detected by oligos 4 and 5, but not by oligos 1, 2, 6, 7 and 8 (Fig. 6E, C, D, F, G and data not shown), indicating that it extends from the A1 to the A3 site. At 37°C the enp1-1 cells also had less 32S and 20S rRNA intermediates (Fig. 6B and E). These results suggest that the enp1 mutation led to complete or nearly complete inhibition of processing at site A0 and A2, and partial inhibition of processing at site A1. As a consequence, the 35S pre-rRNA was cleaved at site A3 instead, producing 23S and 21S rRNA products. Consistent with this theory, the 27SA2 rRNA intermediate level was greatly reduced in enp1 cells at 37°C (Fig. 6C) due to the inhibition of processing at A2, while the 27SB pre-rRNA level was similar to wild-type cells (Fig. 6G). Moreover, no difference in the 5.8 rRNA level was observed between the wild-type cells and enp1 cells (Fig. 6H).<br><br>Enp1 is associated with U3 and U14 snoRNAs and with Nop1
It has previously been shown that mutations in the U3, U14, snR10 and snR30 snoRNAs and in their associated proteins affect processing of 35S pre-rRNA at A0, A1 and A2. Because of the similarity with the processing defects of enp1-1 strains, we tested the association between Enp1 and snoRNAs. To do this, we created a strain in which a TAP tag was fused to the C-terminus of Enp1. The TAP tag contains <span class="yellow">Staphylococcus aureus</span> Protein A as well as calmodulin-binding peptide sequences, so the tagged Enp1 binds IgG beads with high specificity. Cells grew well with the TAP-tagged Enp1 as the only source of Enp1 protein, showing that it was functional. As a control, we used a TAP tag on an unrelated cytoplasmic protein, Hst2. Strain CWY14 (ENP1-TAP) and the Hst2-TAP tagged strain were grown in YPD to mid-exponential phase before being harvested. Extracts were prepared and mixed with IgG agarose beads to precipitate Enp1-TAP and associated RNAs. Total RNAs were extracted from washed IgG beads, separated on 6% polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides. Northern analysis revealed that Enp1-TAP precipitates were enriched with U3 and U14 snoRNAs relative to precipitates from the Hst2-tagged strain (Fig. 7A). The Enp1-TAP samples were not enriched with snR30 snoRNA (Fig. 7A), nor with U24 or U18 snoRNAs (data not shown). We found no change in the levels of U3, U14, snR10 and snR30 snoRNAs in the enp1 mutant, indicating that the mutation did not affect snoRNAs levels (Fig. 6I). The results indicate that Enp1 associates in vivo with U3 and U14 snoRNAs. Much less U3 RNA co-immunoprecipitated with Enp1-TAP than with Protein A-tagged Nop1, which is known to associate with U3 and U14 snoRNAs (Fig. 7A). However, Protein A-Nop1 was much more efficiently precipitated than was the Enp1-TAP protein (data not shown). Therefore, it is likely that similar amounts of RNAs are associated with the two proteins and that the proteins are part of the same complex. To address this further, we checked for the presence of Nop1 in the Enp1-TAP immunoprecipitates. Figure 7B shows that Nop1 indeed is in the Enp1 precipitate but not in the Hst2 control, whereas an abundant ribosomal protein, L3, is in neither precipitate.
U3 and U14 snoRNAs have been shown to interact with rRNA precursors and these interactions are required for pre-rRNA processing (35–37). Using an in vitro system, U3 was also shown to be associated with its rRNA substrate and processing product (38). This raised the possibility that Enp1 might also be associated with rRNAs since its function in rRNA processing appeared linked to those of U3 and U14 snoRNAs. To test this possibility, we further analyzed the RNAs that co-precipiate with TAP-tagged Enp1. The RNAs were separated on 1.2% agarose formaldehyde gels or on polyacrylamide denaturing gels, transferred to nylon membranes and probed with radiolabeled oligonucleotides for pre-rRNAs. A probe (P4; see Fig. 6) specific to 20s and its precursors revealed that pre-rRNAs 35S, 33S, 32S and 20S specifically co-precipitated with Enp1-TAP (Fig. 7C). No such enrichment was found using a probe (P8; see Fig. 6) specific for 27S RNAs or a probe for 5.8S RNA (Fig. 7C). These results clearly demonstrated that Enp1 is associated with substrates and products of the early steps of 18S rRNA processing.<br><br>Comparison of Enp1 with homologs in other organisms
Homologs of Enp1 protein in <span class="yellow">human</span>, <span class="yellow">Drosophila</span> and <span class="yellow">Caenorhabditis elegans</span> have been reported (18). A previously described <span class="yellow">human</span> homolog, bystin, was only 306 amino acids in length, and lacked sequences corresponding to the N-terminal 163 amino acids of <span class="blue">yeast</span> Enp1 (19). In contrast, we identified a <span class="yellow">human</span> expressed sequence tag (EST), BC007340 in a Blast search that revealed an ORF of 1311 nucleotides encoding a 437 amino acid polypeptide (39). Comparison of this 437 amino acid polypeptide with <span class="yellow">human</span> bystin (19) showed that they were encoded by the same gene on <span class="yellow">human</span> chromosome 6. The reported sequence for <span class="yellow">human</span> bystin is a fragment of the 437 amino acid <span class="yellow">human</span> Enp1 protein, due to a truncated cDNA sequence. Also, a sequence discrepancy at the C-terminus is due to inaccurate DNA sequence of the <span class="yellow">human</span> bystin, as confirmed by available <span class="yellow">human</span> genome and EST sequences. Searches of the database of other organisms identified Enp1 homologs in <span class="yellow">S.pombe</span>, <span class="yellow">Arabidopsis thaliana</span>, <span class="yellow">C.elegans</span>, <span class="yellow">Drosophila melanogaster</span> and <span class="yellow">mouse</span>. The alignment of the homologs showed that they shared homology from the N-terminus to the C-terminus, with the C-terminal half extremely well conserved, with close to 90% similarity (Fig. 8). One interesting observation is that the two amino acids changed in the enp1-1 mutant, W242 and V415, are conserved among all homologs.
The <span class="yellow">human</span> Enp1 homolog was cloned and expressed in <span class="blue">yeast</span> and tested for function. Expressed under the control of ADH, TEF or GPD promoter, the <span class="yellow">human</span> Enp1 homolog did not complement a <span class="blue">yeast</span> enp1 null mutant. However, a GFP fusion to the N-terminus of <span class="yellow">human</span> Enp1 homolog localized to the nucleus and was enriched in the nucleolus (data not shown).<br><br>
DISCUSSION
ENP1 is a <span class="blue">yeast</span> gene first identified in a genetic screen for complementation of mutations in ost4, which encodes a subunit of oligosaccharide transferase (17), although subsequent work showed that it is unlikely that Enp1 has any connection to oligosaccharide transferase (18). ENP1 was shown to be essential for viability and the Enp1 protein localized to the nucleus (18).
When we re-examined the localization of Enp1, we observed that the protein was enriched in the nucleolus. This finding led us to test for a role of Enp1 in ribosome synthesis. Using an enp1 ts mutant, we found that depletion of Enp1 caused a 40S ribosomal subunit deficiency. Further analysis revealed that this deficiency was not due to a change of the subunit’s stability, but to a defect in the synthesis of the subunit’s 18S rRNA component. Pulse-chase analysis of RNA synthesis in an enp1-1 strain revealed that no 20S pre-rRNA or 18S rRNA was made at the non-permissive temperature, while levels of 25S rRNA appeared normal. Low levels of precursors to 18S rRNA were detected in the mutants, and they were extremely unstable. Northern analysis of steady state RNA levels demonstrated that the enp1 mutation specifically inhibited the pre-rRNA early cleavages at sites A0, A1 and A2, which are required for the production of the 20S pre-rRNA and the 18S rRNA. These defects of the enp1 mutation on rRNA processing are very similar to those caused by mutation of KRR1, another essential nucleolar gene required for synthesis of the 40S, but not the 60S subunit (40).
Co-precipitation analyses provided strong evidence that Enp1 is directly involved in pre-rRNA processing. In these experiments, TAP-tagged Enp1 bound to IgG beads specifically co-precipitated with two snoRNAs, U3 and U14, and with the 35S, 33S, 32S and 20S pre-rRNAs. Among the more than 100 snoRNAs in <span class="blue">yeast</span> cells, U3, U14, snR10, snR30 and MRP RNA are the only ones required for rRNA processing. It has been shown that mutations in U3, U14, snR10 and snR30 snoRNAs and protein components of the snoRNPs affect processing at sites A0, A1 or A2 (15,16,41,42). Enp1’s interaction with U3 and U14 snoRNAs suggests that Enp1’s function in rRNA processing involves these two snoRNAs. The fact that the nucleolar protein, Nop1, known to bind to U3 and U14 RNAs, also was found in the Enp1 precipitate, provides additional evidence that Enp1 is part of a complex involved in processing of rRNA. Recently, a genome-wide study of <span class="blue">yeast</span> protein complexes, using a TAP tag method similar to ours, reported a number of proteins that co-immunoprecipitated with Enp1 (43). These proteins included Imp4, Kre31, Kre33, Mpp10, Nop1, Nop14 and Sof1, all of which have been implicated in 18S RNA processing or 40S biogenesis. Very recently (after the studies in this manuscript were concluded) Grandi et al. published an analysis of components of the 90s preribosomal particle, which include the 35S pre-rRNA, U3 snoRNP and rRNA processing factors for the 40S subunit (44). In agreement with our findings, they found that Enp1 is a component of this 90s complex and also of a smaller complex containing the 20S pre-rRNA. Surprisingly, none of the other proteins involved in processing of pre-rRNAs that were tested associated with the 20S rRNA, suggesting that those proteins, but not Enp1, are released prior to 20S pre-rRNA formation (44). The authors also showed co-precipitation of Enp1 with dimethylated 20S pre-rRNA, which is formed after export of 20S to the cytoplasm. These results of Grandi et al. suggest that Enp1 may also be involved in later steps of processing or nuclear export.
A previous study on the Enp1 <span class="yellow">human</span> homolog, bystin, reported that the protein was localized in the cytoplasm of mammalian cells and might be involved in cell adhesion (19). Our discovery of the 437 amino acid <span class="yellow">human</span> Enp1 homolog revealed that the bystin studied previously was from a truncated library cDNA encoding only the C-terminal 306 amino acids. Although the <span class="yellow">human</span> homolog of Enp1 did not complement a <span class="blue">yeast</span> Δenp1 mutant, we did show that it was localized to the nucleus and enriched in the nucleolus (data not shown). This strongly suggests that the conserved function of Enp1 is in rRNA processing. The cytoplasmic localization of bystin and its proposed function in cell adhesion are unlikely to reflect the actual function of the <span class="yellow">human</span> Enp1 homolog.
It will be important to study the nature of the associations between Enp1 and U3 and U14 snoRNPs, and to learn more about the exact role of Enp1 in ribosomal RNA processing.<br><br>
<h3>pmcA1937013</h3>Dynein Modifiers in <span class="yellow">C. elegans</span>: Light Chains Suppress Conditional Heavy Chain Mutants
Abstract
Cytoplasmic dynein is a microtubule-dependent motor protein that functions in mitotic cells during centrosome separation, metaphase chromosome congression, anaphase spindle elongation, and chromosome segregation. Dynein is also utilized during interphase for vesicle transport and organelle positioning. While numerous cellular processes require cytoplasmic dynein, the mechanisms that target and regulate this microtubule motor remain largely unknown. By screening a conditional <span class="yellow">Caenorhabditis elegans</span> cytoplasmic dynein heavy chain mutant at a semipermissive temperature with a genome-wide RNA interference library to reduce gene functions, we have isolated and characterized twenty dynein-specific suppressor genes. When reduced in function, these genes suppress dynein mutants but not other conditionally mutant loci, and twelve of the 20 specific suppressors do not exhibit sterile or lethal phenotypes when their function is reduced in wild-type <span class="yellow">worms</span>. Many of the suppressor proteins, including two dynein light chains, localize to subcellular sites that overlap with those reported by others for the dynein heavy chain. Furthermore, knocking down any one of four putative dynein accessory chains suppresses the conditional heavy chain mutants, suggesting that some accessory chains negatively regulate heavy chain function. We also identified 29 additional genes that, when reduced in function, suppress conditional mutations not only in dynein but also in loci required for unrelated essential processes. In conclusion, we have identified twenty genes that in many cases are not essential themselves but are conserved and when reduced in function can suppress conditionally lethal <span class="yellow">C. elegans</span> cytoplasmic dynein heavy chain mutants. We conclude that conserved but nonessential genes contribute to dynein function during the essential process of mitosis.
Microtubules and microtubule-dependent motor proteins segregate chromosomes during mitosis and also promote cellular organization in nondividing cells. An essential motor protein complex called cytoplasmic dynein powers many aspects of microtubule-dependent transport, but it is currently unclear how dynein is regulated such that it can execute different processes. We have performed a genome-wide screen to isolate genes that are involved in dynein-dependent processes. We determined that 20 of the 49 genes we identified specifically influenced the viability of dynein mutant strains but not the viability of other <span class="yellow">C. elegans</span> mutants. Many of the proteins that specifically influence dynein localized to subcellular sites where the dynein heavy chain has been reported by others to be found. Additionally, we identified four dynein components that appear to negatively regulate the force-generating dynein heavy chain. The identification and initial characterization of this group of genes represents a route to identify genes that are not themselves essential but do participate in essential processes.<br><br><br><br>Introduction
The microtubule motor called cytoplasmic dynein has roles in diverse cellular processes including meiotic and mitotic spindle assembly and function, neuronal transport, and organelle positioning [1]. Cytoplasmic dynein is composed of a dimer of heavy chains (HCs), along with several accessory chains (ACs: intermediate, light intermediate, and light chains). Other dynein-interacting proteins, such as dynactin and LIS1, are likely present at substoichiometric levels and further modulate dynein function. The HCs contain both ATPase and microtubule binding activities and are sufficient for microtubule-based motility in vitro, moving toward the minus, or slow-growing, end of microtubules [2]. The dynein ACs provide cargo docking sites and often are encoded by multigene families in any one species [reviewed in 1,3]. In <span class="yellow">C. elegans</span>, a single gene called dhc-1 encodes a cytoplasmic dynein 1 HC, while 11 other genes encode five classes of predicted dynein ACs [3,4].
The early <span class="yellow">C. elegans</span> embryo is an excellent system for investigating gene contributions for essential cellular processes, including cytoskeletal functions [5]. The <span class="yellow">C. elegans</span> dynein HC DHC-1 is essential and required for multiple microtubule-dependent events during early embryogenesis [6–9]. Depletion of DHC-1 by RNA interference (RNAi) in early <span class="yellow">C. elegans</span> embryos produces defects in female meiotic divisions, migration of the oocyte and sperm pronuclei after fertilization, and centrosome separation during mitotic spindle assembly [6]. Analysis of fast-acting dhc-1 temperature-sensitive (ts) mutants has further revealed that dynein is required for chromosome congression to the metaphase plate during mitosis, as well as for mitotic spindle positioning [10].
While many requirements for cytoplasmic dynein are known, our knowledge of the molecular mechanisms that target and regulate dynein remains limited. Clearly, the multiple ACs can couple the dynein HC to particular substrates [11], including vesicles, nuclei, viruses, kinetochores, and rhodopsin [see table in 1]. However, reducing the function of only four of the eleven dynein ACs in <span class="yellow">C. elegans</span> produces lethal phenotypes [12]. Thus, it remains unclear how ACs influence the different essential requirements for dynein. Another potential route for dynein regulation involves the phosphorylation state of the different dynein chains, which in some cases confers distinctive functional properties to the motor. While many examples of dynein phosphorylation exist, and cell cycle dependent changes in phosphorylation have been described [13–15], few if any studies have demonstrated a requirement for such modification during mitosis. Large-scale forward genetic screens have identified genes with requirements similar to those for dynein, but many of these encode core components of the microtubule cytoskeleton and few are known to directly influence dynein itself [12,16]. Genes that do influence dynein function might also have other essential roles, leading to pleiotropic mutant phenotypes that obscure their relationship to dynein [17–19]. Moreover, redundancy within the multigene dynein subunit families, and also perhaps between the different ACs, may complicate the identification of single gene requirements that are important for dynein function. Thus far, reducing the function of individual genes has not provided substantial insight into the mechanisms that regulate and mediate the many different requirements for cytoplasmic dynein during mitosis.
To identify potential regulators of cytoplasmic dynein, we have used a sensitized genetic background to conduct a genome-wide screen for modifiers of dynein function in <span class="yellow">C. elegans</span>. Other groups have successfully used RNAi modifier screens to identify genes that function in particular pathways [20–23]; we have used RNAi to screen for genes that, when reduced in function, suppress the embryonic lethality associated with a temperature-sensitive (ts) allele of the dhc-1 dynein HC gene. Using the dhc-1ts genetic background, we found 49 genes that, when depleted, suppress the partial loss of HC function. Twenty of these genes suppress conditional dynein HC mutants but not other conditional mutants with unrelated defects. Finally, we show that some of these dynein-specific suppressors encode proteins that may overlap with the dynein HC in subcellular localization.<br><br>Results
To identify dynein suppressors, we used three different conditional and recessive dhc-1 mutants that were identified previously [7]. These ts alleles of dhc-1 (or195, or283, and or352) produce similar defects at the restrictive temperature of 26 °C, including incomplete mitotic spindle assembly in one-cell embryos and embryonic lethality [7]. We sequenced the dhc-1 locus in the three mutants. The independently isolated or195 and or283 alleles each change a conserved serine to leucine at codon 3200, within the coiled-coil region of the microtubule-binding stalk domain (Figure 1A and 1C). The or352 allele replaces a conserved glycine with aspartic acid at codon 2158, in the ATP-binding walker A motif of the second AAA ATPase domain (Figure 1B and 1C). As both missense mutations affect conserved residues, they may prove useful for engineering ts alleles in other organisms. The temperature versus viability curves of the dynein ts mutants feature a steep central transition zone ideal for modifier screening because subtle changes in temperature produce large changes in embryonic viability (Figure 2A).
To identify genes that, when reduced in function, can suppress conditional dhc-1 mutants, we developed a high-throughput RNAi-based screen (Figure 2B). To reduce gene function we used a library of 16,757 bacterial strains that each express a dsRNA corresponding to exon-rich gene sequences [17,19]. We then tested over 99% of the bacterial strains in this library for RNAi-mediated suppression of dhc-1(or195) embryonic lethality at 23 °C, after raising synchronized L1 larvae to adulthood on dsRNA-expressing bacterial lawns in 48-well agar plates. This screening procedure should work to identify nonessential and essential suppressor genes, because RNAi does not always fully reduce gene function [24,25], and even if RNAi does produce lethality, cosuppression could restore viability. Nevertheless, essential genes may be missed due to earlier requirements that produce strong larval arrest, sterile, or embryonic lethal phenotypes.
Using this screening procedure, we identified 49 bacterial clones that consistently increased embryonic viability at the semipermissive temperature. The dsRNA-producing plasmids were then sequenced to verify gene identity. Quantification of embryonic viability using dhc-1(or195) animals showed that the RNAi-mediated depletion of suppressor gene function increased viability to 5%–100%, compared to less than 2% in unsuppressed controls (Figure 3A; Table S1). The proteins encoded by the suppressor genes we identified are summarized in Figure 4.
As a more direct assay for dynein activity in the suppressed dhc-1 embryos, we measured spindle length and cytokinesis success: dhc-1 mutant embryos have severe spindle assembly defects and subsequent cytokinesis failures [6,7,10]. We shifted dhc-1 adult hermaphrodites from 23 °C to the fully restrictive temperature of 26 °C for 3–5 hours and made time-lapse video micrographs using Nomarski optics to monitor the first embryonic cell division. This procedure results in dhc-1(or195) embryos with P0 spindles 30% the length of wild-type spindles (Figure S1). In the suppressed dhc-1(or195) backgrounds, spindle lengths ranged from 30%–83% of wild-type lengths (Figure S1). Similarly, cytokinesis failed in unsuppressed dhc-1(or195) embryos 89% of the time, but most of the suppressors rescued this phenotype (Figure S1). These results indicate that most of the suppressors influence dynein-dependent cellular processes, as expected given their ability to restore viability when reduced in function.
Specificity of Suppression
Because RNAi can reduce the function of unintended targets (so-called “off-target effects” [26,27]), we also used available mutations in some of the suppressor genes we identified to reduce their function. We constructed double mutant strains using dhc-1(or195) and viable deletion alleles for two suppressor genes, dylt-1(ok417) and ufd-2(tm1380), and examined embryonic viability (Figure 3B). The deletion alleles of dylt-1 (encoding a Tctex1-type dynein light chain), and ufd-2 (encoding a ubiquitin conjugating enzyme) both recapitulated the suppression produced by RNAi knockdown (Figure 3B). The dpy-3(e27) and dpy-10(e128) point mutation alleles [28] also suppressed embryonic lethality in double mutants (Figure 3B). Based on this small sampling, and because RNAi in <span class="yellow">C. elegans</span> appears to be highly gene specific in the absence of close paralogs [12,19], we conclude that many of the suppressors we have identified will prove to be suppressor locus specific. The dsRNA-expressing bacterial clones we used to deplete two of the dynein suppressors (tag-300 and ZK1127.10) probably also knock down expression of one close paralog for each locus [29].
We next asked whether the suppressors are specific for dynein function or if their depletion more generally stabilizes ts proteins. We tested for specificity using two conditional mutants with cell fate patterning defects unrelated to dynein function, lit-1(or131) and spn-4(or191). The lit-1 gene encodes a MAP kinase-related protein [30], while spn-4 encodes a protein with an RNA binding motif [31]. We found that ten of the dhc-1-interacting genes significantly increased embryonic viability in both lit-1 and spn-4 mutants, while 18 others suppressed one or the other of these two conditional mutants when reduced in function using the same RNAi protocol as that used for dhc-1ts mutants (Figure 4, right two columns; Table S1). Therefore, about half of the suppressors appear to act nonspecifically on multiple ts mutants to restore embryonic viability. From here on, we will refer to the suppressors that only acted on dhc-1, and not on lit-1 and spn-4 ts mutants, as dynein-specific suppressors.
Because many ts mutations exert their effect via protein assembly or unfolding mechanisms [32], suppressor genes reduced in function by RNAi might not be expected to exhibit allele specificity with most ts mutations. To determine if the dynein suppressors are either allele or strain specific, we tested the two other conditional dhc-1 strains (containing the or283 and or352 alleles). Although the or283 allele is identical to or195, it provides a useful control for the presence of background mutations because the two strains were isolated independently. In most cases, depletion of the dynein-specific suppressors also restored viability to the other two ts dhc-1 alleles. Y40B1B.5, a putative translation initiation factor, suppressed only one conditional dhc-1 strain, and we consider this as an example of a nonspecific interaction (Figure 4, left three columns; Table S1). Two dsRNAs that do not suppress lit-1 or spn-4 mutants produced suppression in the dhc-1(or195) and dhc-1(or283) strains, but not in the dhc-1(or352) strain, perhaps indicating allele specificity or variability in the RNAi treatments. We conclude that strain background differences are relatively rare, and that the majority of the suppressors are allele-independent. To summarize, we have identified 20 genes that when reduced in function specifically suppress multiple dynein ts strains but not unrelated ts loci.<br><br>Survey of Putative Dynein Accessory Chains
We were surprised to discover that depleting two predicted dynein ACs specifically suppressed the partial loss of HC function, because most dynein accessory subunits are presumed to promote dynein function by aiding dynein complex formation or mediating cargo attachment [3,11,33]. Depletion of either dylt-1 (encoding a Tctex1-type light chain) or dyrb-1 (encoding a roadblock-type light chain) suppressed embryonic lethality in all three ts dynein HC mutant strains (Figure 4). To extend this observation, we surveyed all genes encoding predicted dynein components for suppression of the three ts dhc-1 mutants (Figure 5A; Table S2). We reasoned that some dynein subunit genes could have been missed in the primary screening and several dynein AC genes were not represented in the <span class="yellow">E. coli</span> RNAi library. After using RNAi to reduce their function, we found that one of three Tctex1 homologs (dylt-1), one of four LC8 homologs (dlc-1), one of two light intermediate chains (dli-1), as well as the sole roadblock homolog (dyrb-1) each strongly suppressed the three conditional dynein mutants. Lower-level suppression was also seen for the second light intermediate chain, xbx-1, when its function was reduced. Thus, one gene of each of four subunit classes restores viability to the three dhc-1 mutant strains when depleted by RNAi.
The only subunit class not found to suppress was the intermediate chain, encoded by a single gene in <span class="yellow">C. elegans</span>, dyci-1. When reduced in function by RNAi, dyci-1 produces a larval arrest phenotype like that observed for dhc-1(RNAi); this phenotype precludes any suppression of the conditional embryonic lethality (shown as “la” in Figure 5A). In contrast, knockdown of either dlc-1 or dli-1 suppresses embryonic lethality in the dhc-1ts mutants, even though reducing their function in otherwise wild-type embryos produces dhc-1-like defects, including embryonic lethality [12,34] (see Figure 5B and Discussion). The suppressing cytoplasmic dynein subunits and DYCI-1 are shown in a putative complex in Figure 5E.
We performed several genetic assays to better understand how the suppressor genes may be operating. First, suppression of dhc-1 lethality by reducing AC function may indicate that our dhc-1 alleles express a neomorphic and toxic DHC-1 protein: if the suppressor dynein AC subunits positively function in dynein processes, depleting them might suppress any neomorphic effects. This explanation is perhaps unlikely, because the dhc-1ts alleles are all recessive, but remained a possibility in dhc-1 homozygotes. We therefore reduced dynein function using RNAi in animals that had passed through the larval arrest points for dhc-1(RNAi) and dyci-1(RNAi). Specifically, we transferred dhc-1(or195) L4 hermaphrodites to plates with bacteria expressing dhc-1 or dyci-1 dsRNA. As control we performed dylt-1(RNAi) using the same procedure. We observed substantial suppression with dylt-1 in this assay, but we saw no suppression with the heavy or intermediate chains (Figure 5C). This suggests that the DHC-1ts protein is not toxic and that dyci-1 acts more like dhc-1 than the other suppressing ACs because it does not suppress the heavy chain mutant.
To further examine the nature of the AC suppression, we asked if depletion of the suppressor chains could bypass the requirement for dhc-1. We transferred wild-type L4 larvae to plates with bacteria expressing dsRNA corresponding to both the suppressor ACs and dhc-1. We did not observe any suppression in these double RNAi assays (Figure 5D), suggesting that dhc-1ts suppression requires the residual activity of the defective DHC-1 protein. We conclude that the dynein AC suppressors inhibit or somehow oppose the function of the DHC-1ts protein, and that the dhc-1(or195ts) mutation does not produce a toxic gene product but simply reduces DHC-1 activity to a low, but non-null, level.<br><br>Localization of the Dynein Suppressor Proteins
To further explore how the suppressor proteins function, we examined the subcellular localization of nine of them as stably expressed N-terminal GFP::S fusions. We chose to first focus on the suppressor genes that were conserved but poorly characterized in any system, or were conserved but uncharacterized during early <span class="yellow">C. elegans</span> embryogenesis. Prior dynein immunocytochemistry-based localization studies serve as a comparison [6,10,35]. As in other species, <span class="yellow">C. elegans</span> DHC-1 is associated with mitotic spindles, centrosomes, the nuclear envelope, the cell cortex, the midbody, and throughout the cytoplasm. Most of the suppressor proteins we examined localized to sites where DHC-1 is known to act or localize (Figure 6). However, the nearly ubiquitous distribution of DHC-1 in early embryonic cells makes colocalization likely but not necessarily meaningful, and biochemical studies are needed to conclusively address any direct or indirect physical associations.
Four suppressor GFP fusion proteins localized to nuclear membranes and to spindle poles or pericentrosomal regions. The DYLT-1 and DYRB-1 dynein light chains were associated with nuclear envelopes and centrosomes, as well as meiotic and mitotic spindle poles (Figure 6A–6H; Videos S1 and S2). The potential coiled-coil protein K04F10.3 was present on the nuclear envelope and in a pericentrosomal position during mitosis, similar to endoplasmic reticulum proteins [36] (Figure 6I–6L; Video S3). K04F10.3 was also highly enriched at meiotic spindle poles (Figure 6I), which has been observed for other endoplasmic reticulum proteins [36]. The NPP-22 transmembrane nucleoporin was found at nuclear envelopes (Figure 6M–6P; Video S4), as previously reported for later stage embryos [37], and it also surrounded centrosomes during mitosis. Two splice isoforms of the pleckstrin homology domain–containing EFA-6/Y55D9A.1, an ARF guanine nucleotide exchange factor, were enriched cortically both in the anterior portion of the one-cell zygote and at the blastomere boundary in two-cell embryos (Figure 6Q–6T; Videos S5 and S6). The conserved Mo25 homolog MOP-25.2/Y53C12A.4 was found enriched in a single spot after cytokinesis that appears to correspond to the midbody (Figure 6U–6X; Video S7). F10E7.8, a highly conserved ortholog of <span class="yellow">S. cerevisiae</span> Far11, appears nuclear (Figure 6Y–6B′ and Video S8). Finally, the nonspecific suppressor protein STAR-2, a predicted RNA binding protein, appears to be associated with P-granules (like its homolog GLD-1), where dynein is neither localized nor known to function (Figure 6C′–6F′).<br><br>DLYT-1 and DYRB-1: Dynein Light Chain Localization
The <span class="yellow">C. elegans</span> dynein HC protein weakly localizes to spindle poles during early embryonic cell cycles [6,10], and so did DYRB-1 and DYLT-1 (Videos S9 and S10). However, ts mutant forms of the DHC-1 protein (including DHC-1 encoded by the or195 allele) strongly localize to centrosomes when shifted to the non-permissive temperature [10]. The mechanism underlying this enhanced localization is not known, but it may represent trapping of the defective protein at a normally transient location. We exploited this behavior of the mutant DHC-1 protein to determine whether redistribution of the putative DYRB-1 and DYLT-1 dynein light chains also occurred in the dhc-1(or195) background.
We found that the cellular distributions of DYRB-1 and DYLT-1 were dramatically altered in dhc-1(or195) mutant embryos. After shifting the parental <span class="yellow">worms</span> to the restrictive temperature for 3–5 h prior to collecting embryos, these two proteins were prominently localized to centrosomes and to spindle poles that did not separate in one-cell stage embryos (Figure 7; Videos S11 and S12). The spindle pole to cytoplasmic fluorescence ratio during late anaphase was 5-fold higher in both of the dhc-1 homozygous mutant strains when compared to wild-type embryos expressing the GFP fusions. We also assayed localization of the two putative dynein light chains after short temperature shifts to the nonpermissive temperature in the dhc-1(or195) mutant background, which yields mitotic spindles with an overall wild-type appearance and function. These short temperature shifts also resulted in robust localization of these two dynein light chains to centrosomes (unpublished data). Finally, we examined the localization of GFP::DYRB-1 and GFP::DYLT-1 in embryos from dhc-1(or195) −/+ <span class="yellow">worms</span> grown at the dhc-1(or195) permissive temperature of 15 °C. Even though embryos from mothers heterozygous for this recessive mutation are viable and develop normally, even at 26 °C [7], we observed a substantial increase in both GFP fusion proteins at the mitotic spindle poles in early embryos (Figure 7; Videos S13 and S14). Importantly, localization of DYLT-1 and DYRB-1 to centrosomes does not occur in embryos depleted for DHC-1 with RNAi (our unpublished results), indicating that these proteins require the mutant DHC-1 polypeptide for centrosomal targeting in the dhc-1(or195) embryos. In summary, the DYRB-1 and DYLT-1 proteins localize to sites where the DHC-1 HC is also found in wild-type embryos, and all three proteins respond similarly to mutational alterations in DHC-1.<br><br>Genetic Characterization of the DYLT-1 and DYRB-1 Dynein Light Chains
We obtained putative null alleles to determine if dylt-1 and dyrb-1 function in dynein-dependent processes. DYLT-1 is 40% identical to <span class="yellow">human</span> DYNLT3 and 38% identical to <span class="yellow">Drosophila</span> Dlc90F (see alignment in Figure 8A). Two other <span class="yellow">C. elegans</span> genes, dylt-2 and dylt-3, encode more divergent members of this protein family. DYRB-1 is 49% identical to both <span class="yellow">human</span> DYNLRB1 and <span class="yellow">Drosophila</span> Robl (see alignment in Figure 8A). There do not appear to be other Roadblock genes in the <span class="yellow">C. elegans</span> genome [3]. Deletion alleles for both dylt-1 and dyrb-1 have been isolated (Figure 8B). The dylt-1(ok417) deletion removes the entire DYLT-1 open reading frame and does not affect adjacent coding regions. The dyrb-1(tm2645) deletion removes 69% of the dyrb-1 coding region, leaving 29 predicted N-terminal amino acids, and does not affect adjacent coding regions.
Both deletions are currently annotated as homozygous viable [29]. However, we found that the dyrb-1(tm2645) strain was in fact heterozygous for the deletion and that most embryos produced by dyrb-1(tm2645) homozygous animals failed to hatch (Figure 8C). Homozygous dyrb-1(tm2645) <span class="yellow">worms</span> also showed an egg-laying defect and produced small broods (unpublished data). To determine if the dyrb-1 deletion was responsible for the embryonic lethality, we crossed the GFP::dyrb-1 transgene into the deletion background. The presence of the transgene fully rescued the embryonic lethality (Figure 8C), but not the egg-laying defect: the transgene is driven from a germline-specific promoter and so would not be expected to rescue zygotic phenotypes. The embryonic lethality exhibited by dyrb-1(tm2645) mutants is consistent with RNAi studies performed by injection or soaking [12,38]. In contrast, homozygous dylt-1 deletion mutants did not exhibit any larval or embryonic lethality (Figure 8C).
To determine if these dynein light chain mutants exhibit dynein HC-like phenotypes, we observed the completion of meiotic polar body extrusion and the first two mitotic cell divisions in mutant embryos (Figure 8D). The dylt-1 embryos appeared wild type for completion of meiosis, pronuclear migration, and spindle assembly and function. However, the dyrb-1 embryos occasionally contained extra female pronuclei (observed in four of 12 recordings, Figure 8D), suggesting that polar body extrusion was defective, and pronuclear migration was often slow compared to wild-type embryos. Once formed, spindles appeared functional using Nomarski optics, although they were frequently positioned improperly and had large spindle poles, as has also been observed after RNAi knockdown [12]. Thus, these two genes are not strictly essential, but the DYRB-1 protein clearly is required for dynein-dependent processes.<br><br>
Discussion
By using the suppressor screening method outlined in Figure 2B, we have isolated and characterized 49 genes that when reduced in function can suppress a partial loss of dynein HC function. This screening procedure takes advantage of sensitized genetic backgrounds using conditional mutants, can be completed for one mutant in less than 5 wk, and is scalable so that many mutants can be screened in parallel. In fact, we have performed 15 such screens in different sensitized backgrounds (unpublished data). By using three dhc-1ts mutant strains, we found that strain background differences and allele specificity are minimal because most of these genes suppress all three dynein mutants when reduced in function using RNAi. Furthermore, by using two unrelated ts mutants to assay for specificity, we found that 57% of the suppressor genes suppress multiple unrelated mutant loci. Thus, it is clear that assaying the specificity of suppression is critical for evaluating the functional significance of these RNAi interactions. Eliminating the analysis of these nonspecific genes in future screens will save time and resources. Most of the specific suppressor proteins we examined appear to overlap in subcellular localization with the dynein HC, based on previous studies of DHC-1, while one nonspecific suppressor protein, STAR-2, localized to germline P-granules, where dynein is not known to function.
Many of the 20 genes that specifically suppress multiple dhc-1ts alleles are nonessential in <span class="yellow">C. elegans</span> but well conserved nonetheless. Six of eight deletion alleles available for the 20 specific suppressor genes are homozygous viable, and six additional specific genes do not display lethal phenotypes when reduced in function by RNAi in wild-type <span class="yellow">worms</span> [29]. Thus, our genetic screening has identified roles in an essential process for at least 12 apparently nonessential genes. Fourteen of the specific dynein suppressor genes have <span class="yellow">human</span> orthologs as determined by best reciprocal BLAST hits (Table 1), while mop-25.2 has a conserved <span class="yellow">human</span> homolog but also a paralog in <span class="yellow">C. elegans</span>. Eleven of these conserved genes are nonessential in <span class="yellow">C. elegans</span>. Interestingly, eight of the conserved genes in Table 1 have been implicated in <span class="yellow">human</span> disease etiology, with three of them identified as the causative gene [39–41]. Thus, using sensitized genetic backgrounds for genome-wide modifier screens can identify roles for nonessential but conserved genes and thereby provide insights into <span class="yellow">human</span> disease.
Nonspecific Suppression of Conditional Mutants
We examined the predicted molecular functions of the suppressor proteins to better understand the basis for the nonspecific suppression phenomenon. Strikingly, many of the nonspecific suppressor genes encode proteins with predicted roles in mitochondrial, ribosomal, and collagen function (18 of 29 genes, or 62%), while only two such genes appeared to specifically suppress dhc-1 (2 of 20 genes or 10%). It is possible that stress produced by RNAi knockdown of these suppressor genes triggers the activity of molecular chaperones that can generally restore function to ts proteins. Indeed, mutation of dpy-10 is known to suppress three other ts mutants: glp-1, emb-5, and mup-1 [42–44]. Furthermore, RNAi reduction of dpy-10, star-2, osr-1, or C50D2.1 (all suppressors of dhc-1, lit-1, and spn-4 ts alleles) induces the glycerol biosynthetic gene gpdh-2, while dpy-10 and osr-1 mutants exhibit increased glycerol levels, a condition that promotes protein stability [45]. We suggest that partial loss of central metabolic processes can invoke stress responses that nonspecifically alleviate protein-folding problems in ts proteins. Filtering out these nonspecific interactions by testing unrelated conditional mutants increases the likelihood that the remaining suppressor genes are more directly involved with dynein function. However, ts mutants likely differ in their susceptibility to nonspecific suppression mechanisms, and some apparently unrelated ts mutants might share common cofactors such that both mutants are suppressed by depletion of the same cofactor. Nevertheless, we expect that more extensive testing for specificity will prove very useful for judging the significance of modifier interactions.<br><br>Possible Relevance of Suppressor Proteins to Dynein Function
We examined the localization of a number of GFP fusions to suppressor proteins to gain insight into their functional relationship to dynein. In several cases, the subcellular distribution of the suppressor proteins overlapped in different ways with the known and nearly ubiquitous distribution of cytoplasmic dynein in the early <span class="yellow">C. elegans</span> embryo. In fact, the only specific suppressor that did not display dynein-like localization was F10E7.8, a homolog of <span class="blue">yeast</span> Far11 of unknown function [46], which was nuclear. The one nonspecific suppressor protein we examined did not show any dynein-like localization patterns. The subcellular localizations of the GFP::suppressor protein fusions are intriguing. However, given the nearly ubiquitous distribution of dynein in early embryonic cells, biochemical tests for direct association are needed to address the significance of any colocalization detected using light microscopy.
We are particularly interested in suppressor proteins that localize to mitotic spindle poles: the association of the DYLT-1 and DYRB-1 predicted dynein light chains with centrosomes and spindle poles suggests that they may be components of cytoplasmic dynein in <span class="yellow">C. elegans</span>. Localization of cytoplasmic dynein to centrosomes and spindle poles is well established [47,48], and the inhibition of dynein function prevents centrosome separation, centrosome attachment to nuclei, and the formation of bipolar spindles [6,10,49,50]. Moreover, the centrosomal localization of DHC-1, DYLT-1, and DYRB-1 are all greatly enhanced in dhc-1ts mutant embryos: this dependence of the light chain distribution on the HC further suggests they reside in the same motor complex (Figure 7 and [10]). Furthermore, roadblock light chains are well-established components of dynein, and all of the roadblock protein in mammalian liver extracts is dynein associated [51,52]. Finally, a DYLT-1 homolog in vertebrates is a stoichiometric subunit of cytoplasmic dynein [53]. The presence of these two light chains in a dynein complex is consistent with them having either positive or negative roles in the regulation of HC function (see below).
Cytoplasmic dynein is found on the nuclear envelope where it is thought to regulate nuclear membrane breakdown during mitosis [54], and dynein plays roles during the trafficking of endoplasmic reticulum components [55,56]. Therefore, the nuclear envelope/endoplasmic reticulum proteins NPP-22 and K04F10.3 could couple dynein activity to either of these structures. The anc-1 gene was also isolated in our screening and ANC-1 is localized to the nuclear envelope where it maintains nuclear positioning in postembryonic cells [57]. Reducing the function of these three genes may suppress partial loss of dynein HC mutants by reducing the need for dynein during nuclear envelope breakdown, through constitutive partial destabilization of the nuclear envelope.
The distribution of the cytoplasmic dynein HC includes sites other than spindle poles and nuclear envelopes in <span class="yellow">C. elegans</span>, for example, at the cell cortex and at the cell division remnant called the midbody [6,10]. The MOP-25.2 protein was found at the midbody and faintly at spindle poles. The MOP-25.2 ortholog in <span class="yellow">S. pombe</span>, Pmo25, is also present at the cell division site and on spindle poles [58]. Mammalian MOP-25.2 homologs stimulate the kinase activity of the LKB1 tumor suppressor (the <span class="yellow">C. elegans</span> ortholog is PAR-4), which in turn activates MARK microtubule-destabilizing kinases [59,60]. The <span class="yellow">C. elegans</span> MARK ortholog, PAR-1, controls cell polarity during embryogenesis, and orthologs have been implicated in regulation of microtubule dynamics from <span class="blue">yeast</span> to <span class="yellow">humans</span> [61–64].
Lastly, the two splice isoforms of EFA-6 were associated with the anterior cell cortex in late one-cell embryos. Cortically localized dynein may have important roles in applying forces to astral microtubules that influence mitotic spindle positioning and chromosome separation during anaphase [10,65,66]. EFA6 ARF guanine nucleotide exchange factors require their pleckstrin homology domain for cortical targeting, and are known to regulate cortical actin dynamics in vertebrate cells by promoting guanine nucleotide exchange on ARF6 [67,68]. Our results identifying efa-6 as a dynein HC suppressor suggest a functional linkage of the actin and microtubule cytoskeletons at the cell cortex. Interestingly, two <span class="blue">yeast</span> pleckstrin homology domain proteins, Num1 and mcp5+, localize to the cell cortex and direct astral microtubule and dynein function, although they do not contain a Sec7 domain like EFA-6 does [69–71].<br><br>Function of Dynein Intermediate, Light Intermediate, and Light Chains
The dynein chains in <span class="yellow">C. elegans</span> exhibit strikingly different functional requirements. The DYRB-1 roadblock light chain is required for completion of meiosis and pronuclear migration, but an at least partially functional mitotic spindle forms in the absence of DYRB-1 (Figure 8). The DLI-1 light intermediate chain is required for multiple dynein-dependent functions: pronuclear migration, centrosome separation, and meiotic and mitotic spindle function [12,34]. DLI-1 may promote nuclear envelope targeting of both centrosomes and DHC-1 by interacting with the nuclear envelope protein ZYG-12 [35]. The second worm light intermediate chain gene, xbx-1, is required for cilia function but not early embryonic development [12,72]. RNAi knockdown of DLC-1, one of three LC8 proteins in <span class="yellow">C. elegans</span>, produces defects similar to dli-1 but knockdown of the other two LC8-related genes does not result in any phenotypes [12]. RNAi depletion of dyci-1 results in severe meiotic, pronuclear migration, and mitotic spindle assembly defects [12] and in our feeding RNAi regimen dyci-1(RNAi) produces a larval arrest phenotype similar to that observed for dhc-1. Finally, the three Tctex1 proteins in <span class="yellow">C. elegans</span>, DYLT-1, 2, and 3, are not essential for dynein-related functions [12]. As the dyrb-1, dlc-1, and dli-1 dynein AC genes display some dhc-1-like requirements, they positively influence dynein function. However, because reducing their function suppresses dhc-1ts mutants, they may also exert negative regulation (along with dylt-1) on the heavy chain.<br><br>Negative Regulation of Dynein HC by Light Chain Subunits
Finding that reducing the function of light and light intermediate dynein chains suppressed the partial loss of HC function was a striking result. One member of each of four subunit classes can suppress the embryonic lethality associated with three dhc-1 ts mutants (Figure 5). We have considered two different models to explain how RNAi-mediated depletion of these dynein ACs can suppress reduced HC function. First, these dynein subunits could be in functional complexes with, and exert negative regulation on, the DHC-1 HC (Figure 5B). The suppression mechanism in this case proposes that removal of the suppressing ACs increases residual mutant DHC-1 activity. The other, nonsuppressing, accessory subunits might then function in nonmitotic cellular processes such as neuronal transport or organelle positioning. In support of this view, physical removal of the intermediate chains of <span class="yellow">rat</span> cytoplasmic dynein increased HC ATPase activity by about 4-fold (light chains were not monitored in this study but were likely removed as well) [73]. Thus, at least with respect to ATPase activity, some dynein ACs do act as biochemical negative regulators of HC function.<br><br>An Assortment of Essential and Nonessential Dynein Complexes
Alternatively, an assortment of dynein complexes (with different ACs) could coexist within early embryonic cells, with only a subset required for the essential mitotic functions that require DHC-1. In this case, suppression might result from the release of DHC-1 HCs from less essential motor complexes, allowing more of the functionally compromised HCs to participate in the essential process of mitosis. We currently disfavor this hypothesis because two of the suppressing light chains (DYLT-1 and DYRB-1) can indeed localize to meiotic and mitotic spindles (Figures 6 and 7), sites where DHC-1 has been shown by others to localize and function. Furthermore, the distribution of DYRB-1 and DYLT-1 closely resembles the distribution of the HC in dhc-1(or195) embryos, suggesting that these two light chains associate with the HC during mitosis (Figure 7 and [10]). Finally, dhc-1-like phenotypes result from mutation or RNAi knockdown of three suppressing ACs in otherwise wild-type <span class="yellow">worms</span>, indicating that they share at least some common and essential requirements. Regardless of the suppression mechanism, our identification of ACs that genetically interact with the DHC-1 HC provides a basis for functionally classifying the paralogs of these dynein subunit gene families, and for further investigation of dynein composition and function.<br><br>Nonessential Dynein Subunits and Negative Regulation of the HC
Some ACs are nonessential, supporting the view that some cytoplasmic dynein subunits could function by exerting negative regulation on the HC, rather than positively influencing essential HC function. For example, DYRB-1 is not absolutely required for viability because <span class="yellow">worms</span> lacking this protein can be propagated, although they are extremely sick and do exhibit two dhc-1-like phenotypes (Figure 8). Also, homozygous dylt-1 deletion mutants appear fully viable (Figure 8). The two additional Tctex1 <span class="yellow">C. elegans</span> genes could be functionally redundant with DYLT-1, but simultaneously reducing the function of DYLT-2 and DYLT-3 by RNAi in the dylt-1 deletion strain did not cause lethality (unpublished data). Because RNAi does not always completely reduce function, the question of redundancy in the Tctex1 <span class="yellow">C. elegans</span> gene family remains unresolved. However, <span class="yellow">Drosophila</span> contains only a single Tctex1 gene, Dlc90F [74,75]. A Dlc90F null allele that deletes 80% of the open reading frame is essential only for sperm production but not for viability of male or female <span class="yellow">flies</span>, despite the wild-type protein being incorporated into dynein motors and expressed in various <span class="yellow">Drosophila</span> tissues [74]. Thus, at least in <span class="yellow">Drosophila</span>, the Tctex1 dynein light chain family is not required for cell division processes like the HC is. Interestingly, budding <span class="blue">yeast</span> does not possess genes for the Tctex1 or roadblock ACs, indicating that functional cytoplasmic dynein does not require these subunits that are conserved in many other organisms. The AC genes that <span class="blue">yeast</span> does posses are not required for HC motility in vitro because dynein purified from <span class="blue">yeast</span> with mutations in these genes remains fully active [2]. Thus, dynein function in several contexts does not require AC subunits, and we suggest that in some cases they may have negative regulatory roles. Negative regulation of cytoplasmic dynein may be redundant with other modes of HC regulation or only required during special circumstances. Further studies of subunit localization, and in vitro studies of <span class="yellow">C. elegans</span> dynein motility, may provide further insight into the modes of AC regulation and function.<br><br>
Materials and Methods
<span class="yellow">C. elegans</span> strains and culture.
Strains were cultured according to standard procedures [28]. ts mutants were maintained at 15 °C and GFP-expressing strains in a wild-type background were maintained in a 23 °C incubator. dhc-1(or195) was outcrossed six times to the N2 Bristol wild-type strain and the or283 and or352 dhc-1 mutants were each outcrossed four times with N2. For sequencing mutant dhc-1 loci, genomic DNA was amplified as overlapping ∼1-kb fragments from the start codon to the stop codon and sequenced at the University of Oregon DNA sequencing laboratory. For double mutant constructions, the dhc-1(or195) mutation was monitored by sequencing or by assaying a restriction fragment length polymorphism caused by the mutation with Hpy188I (New England Biolabs, http://www.neb.com), following PCR amplification of the mutated region. The dylt-1(ok417) and ufd-2(tm1380) alleles were monitored by PCR amplification of genomic sequence encompassing the deletions and assaying product size by agarose gel electrophoresis.<br><br>GFP imaging.
Visualization of GFP fusion protein localization was accomplished by mounting embryos on M9 + 3% agarose pads on microscope slides covered with a coverslip. Time-lapse videos were obtained on a spinning disk Nikon Eclipse TE2000-U microscope (Nikon Instruments, http://www.nikon.com) fitted with an ORCA-ER digital camera (Hamamatsu Photonics, http://www.hamamatsu.com) using a Nikon 60×, 1.4 NA Plan Apo oil objective lens. Videos were adjusted for contrast in ImageJ (National Institutes of Health, http://rsb.info.nih.gov/ij/) [76], images were adjusted for levels in Adobe Photoshop (http://www.adobe.com/).<br><br>RNAi screening.
After obtaining the <span class="yellow">E. coli</span> RNAi library from the MRC Geneservice (Cambridge, UK) [17,19], we rearrayed it into a 48-well microplate format using a liquid-handling Qiagen BioRobot 8000 (http://www.qiagen.com). <span class="yellow">E. coli</span> strains were thawed from −80 °C storage and inoculated into 1 ml of LB + 100 mg/ml ampicillin-containing 96-well growth plates (Whatman, http://www.whatman.com) and covered with microporous sealing film (USA Scientific, http://www.usascientific.com). Only 48 wells of the 96-well growth plates were filled with media, corresponding to the rearrayed <span class="yellow">E. coli</span> library. After overnight shaking incubation at 37 °C, 20 μl of the cultures were dispensed with a 24-channel electronic repeating pipette (Rainin, http://www.rainin.com) onto 48-well plates (Nunc, http://www.nuncbrand.com/) containing NGM agar, 100 μg/ml ampicillin, and 1 mM IPTG and allowed to dry and induce dsRNA at 37 °C overnight. The 48-well agar plates were filled with a Wheaton Unispense peristaltic pump (http://www.wheaton.com) equipped with a custom-made adaptor (University of Oregon Technical Science Administration) that allowed simultaneous filling of eight wells with the agar solution. Approximately 15 hypochlorite-synchronized L1 mutant larvae were pipetted into each well of the 48-well plates with a multichannel pipette and allowed to produce broods. Screening for F1 viability was performed by visual examination with a dissecting microscope. Phenotypes were recorded on an Excel spreadsheet (http://www.microsoft.com) and organized in a FileMaker Pro (http://www.filemaker.com) database. We qualitatively identified 295 initial positive suppressing <span class="yellow">E. coli</span> strains, for which we repeated the assay on 60-mm plates with <span class="yellow">E. coli</span> again thawed from the library (not streak purified). If the observed phenotypes reproduced, the assay was performed with three streak-purified <span class="yellow">E. coli</span> colonies, and a single isolate that again displayed the interaction was kept for further analysis.<br><br>Embryonic viability quantitation methods.
To quantitate embryonic viability we used the following procedure. Cultures of dsRNA-producing bacteria were grown overnight in LB + 100 μg/ml ampicillin. Cultures (0.2 ml) were seeded onto 60-mm NGM agar plates containing 100 μg/ml ampicillin and 1 mM IPTG and allowed to induce dsRNAs overnight at room temperature. The L4440 control vector-containing strain was used as the bacterial lawn for the experiments shown in Figures 2A and 3B. Approximately 80 synchronized L1 larvae (obtained from hypochlorite-treated <span class="yellow">worms</span>) were pipetted onto the plates and allowed to grow to young adulthood. Five gravid <span class="yellow">worms</span> were transferred to prepared NGM agar plates supplemented with 100 μg/ml ampicillin and 1 mM IPTG containing a small RNAi bacterial lawn produced from ∼5 μl of overnight <span class="yellow">E. coli</span> culture. After producing broods, the adult <span class="yellow">worms</span> were removed and the embryos were allowed to develop for at least 24 h. Embryos and larvae were then counted immediately or after storage at 4 °C. We considered only suppressor dsRNAs that increased embryonic viability greater than 3-fold above the background viability observed with the L4440 control vector (in the dhc-1(or195)) screen to be significant enough for continued study.<br><br>Molecular biology.
We introduced a polylinker site containing six unique restriction enzyme recognition sites into the pIC26 GFP-S protein plasmid by using phosphorylated and PAGE-purified synthetic oligonucleotides [77]. Following ligation, the new plasmid was sequence verified. The modified vector, pSO26, allows the use of additional restriction enzymes and directional cloning for inserting genes of interest (Figure S2). The SpeI site was recreated at the 5′ end of the polylinker but not at the 3′ end. We amplified N2 genomic DNA or cDNA (Invitrogen, http://www.invitrogen.com) with Pfu Turbo polymerase (Stratagene, http://www.stratagene.com), and cloned A-tailed PCR products into either pGEM-T or pGEM-T-easy shuttle vectors (Promega, http://www.promega.com). Inserted genes were sequence verified at the University of Oregon DNA sequencing laboratory prior to cleavage and ligation to pSO26 (see Table S3 for restriction sites and primer sequences used). All of the constructs used in this study were cloned as SpeI-AsiSI or AscI-AsiSI fragments, except for the STAR-2 gene, which was amplified as a SpeI fragment and cloned into pIC26.
To construct dynein subunit dsRNA-expressing plasmids not available in the RNAi library, gene fragments were amplified from N2 genomic DNA with the following primers: F41G4.1: 5′-AAGATATCACCCAAAATGGTCCAAAACAAAG-3′ and 5′-CGGATATCTCGACTGAAGCTGGTTCTGA-3′, xbx-1: 5′-AAGATATCTACGACGATGGAAGTTTGAAG-3′ and 5′-CGGATATCCGTGCCTCTGCAGC-3′, dlc-3: 5′-AAGATATCAATTTCAGGTGGACACTGGC-3′ and 5′-CGGATATCAGCACACTTGCATCATCTGAA-3′. The PCR products were cut with EcoRV, ligated to EcoRV-digested L4440, and sequence verified.<br><br>Isolation of transgenic <span class="yellow">worms</span>.
GFP fusion plasmids were bombarded into unc-119(ed3) <span class="yellow">worms</span> as previously described except with the following two changes [78]. Three milligrams of gold particles were used per hepta adaptor bombardment. Also, we briefly sonicated the gold particles (prior to DNA coating and while suspended in 50% glycerol) with a Branson sonifier 450 (http://www.sonifier.com/) fitted with a small tip set to power level 1, to disrupt gold aggregates. Non-Unc <span class="yellow">worms</span> were picked to new plates and allowed to produce broods, which were assayed for GFP fluorescence with a Zeiss axioskop microscope (http://www.zeiss.com/) fitted with an X-Cite 120 illumination system (EXFO life sciences, Mississauga, Ontario, Canada). For each fluorescent line, 12 GFP-positive <span class="yellow">worms</span> were singled to new plates to determine if the constructs were integrated or were carried as extrachromosomal arrays.<br><br>
Supporting Information
Accession Numbers
The National Center for Biotechnology Information (NCBI) database (http://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi?itool=toolbar) accession numbers for the dhc-1 homologs discussed in this paper are <span class="yellow">C. elegans</span>, NP_491363; <span class="yellow">Dictyostelium discoideum</span>, XP_643185; <span class="yellow">Drosophila melanogaster</span>, AAA60323; <span class="yellow">Homo sapiens</span>, NP_001367; <span class="yellow">Mus musculus</span>, NP_084514; <span class="yellow">Saccharomyces cerevisiae</span>, NP_012980; and <span class="yellow">Schizosaccharomyces pombe</span>, NP_001018285.
The NCBI accession numbers for the <span class="yellow">Drosophila</span> and <span class="yellow">human</span> DYLT-1 and DYRB-1 protein homologues, respectively, are Dlc90F, NP_477356; DYNLT3, NP_006511; DYNLRB1, NP_054902; and robl, NP_523771.<br><br><br><br><h3>pmcA2065882</h3>Identification of Two Independent Risk Factors for Lupus within the MHC in United Kingdom Families
Abstract
The association of the major histocompatibility complex (MHC) with SLE is well established yet the causal variants arising from this region remain to be identified, largely due to inadequate study design and the strong linkage disequilibrium demonstrated by genes across this locus. The majority of studies thus far have identified strong association with classical class II alleles, in particular HLA-DRB1*0301 and HLA-DRB1*1501. Additional associations have been reported with class III alleles; specifically, complement C4 null alleles and a tumor necrosis factor promoter SNP (TNF-308G/A). However, the relative effects of these class II and class III variants have not been determined. We have thus used a family-based approach to map association signals across the MHC class II and class III regions in a cohort of 314 complete United Kingdom Caucasian SLE trios by typing tagging SNPs together with classical typing of the HLA-DRB1 locus. Using TDT and conditional regression analyses, we have demonstrated the presence of two distinct and independent association signals in SLE: HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, OR = 2.3) and the T allele of SNP rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001, OR = 2.0) in intron 6 of the class III region gene SKIV2L. Assessment of genotypic risk demonstrates a likely dominant model of inheritance for HLA-DRB1*0301, while rs419788-T confers susceptibility in an additive manner. Furthermore, by comparing transmitted and untransmitted parental chromosomes, we have delimited our class II signal to a 180 kb region encompassing the alleles HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201 alone. Our class III signal importantly excludes independent association at the TNF promoter polymorphism, TNF-308G/A, in our SLE cohort and provides a potentially novel locus for future genetic and functional studies.
Systemic lupus erythematosus (SLE/lupus) is a complex autoimmune disease in which the body's immune system attacks its own tissues, causing inflammation in a variety of different organs such as the skin, joints, and kidneys. The cause of lupus is not known, but genes play a significant role in the predisposition to disease. The major histocompatibility complex (MHC) on Chromosome 6 contains at least 100 different genes that affect the immune system, including the genes with the strongest effect on lupus susceptibility. Despite the importance of the MHC in SLE, the identity of the actual genes in the MHC region that cause SLE has remained elusive. In the present study, we used the latest set of genetic markers present at the MHC in lupus families to identify the actual genes that affect the disease. To our knowledge, we have shown for the first time that two separate groups of genes are involved in SLE. One group of genes alters how the immune system may inappropriately target its own tissues in the disease. How the second set of genes predisposes to SLE is the subject of ongoing study.<br><br><br><br>Introduction
Since the early 1970s, the <span class="yellow">human</span> major histocompatibility complex (MHC) has been shown to be associated with a number of autoimmune, inflammatory, and infectious diseases, and it continues to be the focus of intense research [1]. The recently defined extended MHC (xMHC) encompasses 7.6 Mb of genome on 6p21.3 and is divided into five subregions from telomere to centromere: extended class I, classical class I, classical class III, classical class II, and extended class II. In addition, the MHC contains two hypervariable regions, the RCCX module in class III (spanning complement C4) and the HLA-DRB genes in class II, that both exhibit copy number polymorphism. Examination of the sequence across the extended MHC reveals the presence of 421 genes, and over 252 (60%) are thought to be expressed [2]. Around 40% of genes expressed within the classical MHC encode proteins with putative immunomodulatory function [3]. The classical class I and class II loci encode the <span class="yellow">human</span> leucocyte antigen (HLA) proteins involved in antigen presentation to T cells, initiating the adaptive immune response. The class III region contains the greatest density of genes in the genome (58 expressed genes), which are often found in functionally related clusters [2].
A major obstacle in the identification of disease-specific causal variants within the MHC has been the strong linkage disequilibrium (LD) exhibited by certain alleles in this region, resulting in the existence of long-range, conserved, extended haplotypes [4], also known as ancestral haplotypes [5], sometimes spanning more than 2 Mb [6]. Thus, for many MHC-associated diseases, it has only been possible to delimit association signals to a particular extended haplotype or segment of one.
Systemic lupus erythematosus (SLE/lupus, [Online Mendelian Inheritance in <span class="yellow">Man</span> 152700, http://www.ncbi.nlm.nih.gov/sites/entrez?db=OMIM&TabCmd=Limits]) is a chronic, multi-system autoimmune disease affecting young <span class="yellow">women</span> ten times more commonly than <span class="yellow">men</span>. The worldwide prevalence of SLE is estimated at between 12 and 124 cases per 100,000 individuals [7]. SLE is characterized by the presence of pathogenic autoantibodies to nuclear and cell-surface antigens that show affinity maturation. The consequent immune complexes deposit in tissues, causing inflammation and damage. It is well established that there is a complex genetic component to lupus aetiology, with hormonal and environmental influences also contributing to disease susceptibility [8,9].
The MHC has been the most consistently confirmed genetic risk factor for SLE, and multiple different genes within the region have been significantly implicated with disease susceptibility. For example, hereditary and acquired deficiencies of the early classical complement component C4, located within the MHC class III locus, leads to a lupus-like syndrome. A role for another class III gene, tumour necrosis factor alpha (TNF), in SLE was suggested following the observation that the lupus-prone New Zealand F1 <span class="yellow">mouse</span> hybrid exhibits constitutively low TNF expression [10]. Recently, the development of antinuclear antibodies in <span class="yellow">patients</span> treated with TNF antagonists has also stimulated interest in the possible role of TNF in SLE [11–13]. <span class="yellow">Murine</span> and <span class="yellow">human</span> candidate gene studies, together with genome-wide linkage screens, provide further support that multiple genetic loci, including the <span class="yellow">mouse</span> MHC complex H2 and the <span class="yellow">human</span> MHC locus, contribute to disease susceptibility [14–17].
It should be noted that the <span class="yellow">human</span> MHC was first associated with SLE in 1971, when studies demonstrated that lupus probands were enriched for the class I alleles HL-A8 (now known as HLA-B8) and HLA-W15 (now known as HLA-B15) when compared with healthy controls [18,19]. Further case control association studies were small, performed in ethnically diverse populations, and tested only a small number of the classical HLA and complement C4 alleles. The most consistent findings reported to date are associations with the class II alleles HLA-DR2 (DRB1*1501) and HLA-DR3 (DRB1*0301) and their respective haplotypes in Caucasian populations. The complement C4A null allele (C4A*Q0) has shown inconsistent association with lupus in a number of studies—a situation that may reflect genetic heterogeneity in disease susceptibility [20–23]. In addition, a recent study has demonstrated that low C4A copy number is a risk factor for lupus in a European American cohort [24]. However, the C4A null allele lies on the lupus-associated DR3 “autoimmune” extended haplotype (AH8.1), which exhibits extremely strong LD [6]. It therefore remains to be definitively established whether this locus constitutes a distinct susceptibility allele to that of the class II association or is merely in LD with it. Similarly, certain TNF promoter polymorphisms, including the much-studied SNP TNF-308G/A, have shown association with SLE; but again, many of these variants are carried on the highly conserved 8.1 ancestral haplotype, thus restricting interpretation of these data.
In 2002, a family-based study employing microsatellites as surrogate markers for HLA-DRB1 haplotypes in Caucasian lupus families demonstrated association with DR3-, DR2-, and DR8 (DRB1*0801)-containing haplotypes. In that study, Graham and colleagues reported that, taking advantage of recombinant chromosomes, the disease risk region could be limited to a 1 Mb region encompassing classical class II and class III [25].
We have performed a medium resolution association mapping study of the MHC in lupus families, utilizing a combination of SNPs and four-digit typing at the HLA-DRB1 locus in order to anchor haplotypes. Sixty-eight SNPs were successfully genotyped across a 2.4 Mb region of the MHC, from the class I locus KIAA1949 to the class II gene HLA-DPB2, in 314 UK Caucasian SLE trios. We used these data to perform a family-based association study in an attempt to distinguish the relative effects of the class II and class III regions of the MHC in lupus susceptibility. In addition, we employed the long-range haplotype test to search for the presence of high-frequency, extended haplotypes indicative of recent positive selection [26]. We have also used family-based and case-control strategies to examine genotypic risk at HLA-DRB1 and rs419788.<br><br>Results
Association Testing of HLA-DRB1 and MHC Region SNPs
In order to define the causal variation within the MHC region, we typed 314 complete SLE trios for the HLA-DRB1 gene as well as for 86 SNPs across a 2.4 Mb region encompassing the HLA class I locus HLA-B to HLA-DPB2. High-quality genotype data was obtained for HLA-DRB1 and 68 MHC SNPs (see Table S1 for quality control data). Association testing of the HLA-DRB1 gene revealed a significant association with HLA-DRB1*0301 (nominal p = 4.9 × 10−8, permuted p < 0.0001, T:U = 129:55) in our lupus cohort (Table 1). There was also a trend for under transmission of the HLA-DRB1*0701 allele (nominal p = 0.0013, T:U 42:77); however, this association was no longer significant after correction for multiple testing as determined by 10,000 permutations of the dataset (permuted p = 0.09). Furthermore, we did not find evidence of association with HLA-DRB1*1501 (nominal p = 1.0, T:U 70:70) or HLA-DRB1*0801 (nominal p = 1.0, T:U = 11:11) in our cohort (see Table S2 for complete HLA-DRB1 association data); alleles previously suggested by microsatellite typing of a US lupus cohort [25].
Association testing of the MHC region SNPs also identified significant evidence of association to SLE (Table 1 for associated markers and Table S3 for all MHC SNPs). The SNP with the most significant association, rs419788 (nominal p = 4.3 × 10−8, permuted p < 0.0001) was of similar strength to that of the HLA-DRB1*0301 allele, with odds ratios (ORs) and 95% confidence intervals (CIs) of 2.0 (1.6–2.6) and 2.3 (1.7–3.2), respectively. This SNP is located within intron 6 of the class III gene, superkiller viralicidic activity 2-like (<span class="yellow">Saccharomyces cerevisiae</span>) (SKIV2L), and is located approximately 500 kb telomeric to the HLA-DRB1 gene. Of the other 12 SNPs that were significantly associated with SLE (nominal p = 4.0 × 10−4 to 2.5 × 10−7; permuted p = 0.03 to <0.0001), one was located in the class I region between HLA-B and MICA, seven were located in the class III region, and four were situated in the class II region (Table 1; Figure S1). Specifically, the seven associated class III SNPs were located in or close to the following genes: the TNF promoter, BAT3, SLC44A4, EHMT2, TNXB, GPSM3, and NOTCH4. One of the four class II associated SNPs was close to HLA-DRA, two were between HLA-DRB1 and HLA-DQA1 and one was in intron 1 of HLA-DQA1. The correlation between all 68 SNPs and HLA-DRB1 in our UK SLE cohort is illustrated in Figure 1. The markers showing significant association are highlighted.<br><br>Conditional Analyses Identify Two Independent Association Signals in the MHC
In order to establish whether the two most associated signals identified in this association-mapping experiment are likely to represent a single causal allele or independent risk factors, we first examined the association data conditioned upon the presence of the HLA-DRB1*0301 allele. We found that four of the 13 associated SNPs showed evidence of signals independent of HLA-DRB1*0301 in our dataset, the strongest of which was rs419788 (Table 1). We therefore conditioned the three remaining SNPs (rs2523589, rs1052486, and rs605203) on rs419788 to assess whether these signals are truly independent of each other or show association due to LD with rs419788. In addition, we included HLA-DRB1 in stepwise conditional regression analyses performed on the SNPs showing association independent of HLA-DRB1 (unpublished data). These analyses demonstrated that the observed association signals at rs2523589, rs1052486, and rs605203 were predominantly dependent upon the association at rs419788, and suggested that there are two major independent association signals in the MHC in UK SLE: HLA-DRB1 and rs419788. The independence of the association signals at HLA-DRB1 and rs419788 is further supported by the observation that there is only modest LD between these two (r2 = 0.24). There was no association with any other HLA-DRB1 allele and the four SNPs independent of HLA-DRB1*0301 (TRANSMIT, unpublished data).
The association of the tumour necrosis factor gene promoter SNP TNF-308G/A with SLE is lost after conditioning for HLA-DRB1*0301 in our cohort. If we perform the reverse analysis and condition HLA-DRB1*0301 on the presence of the TNF promoter SNP, we find that the association remains, confirming that our TNF association is secondary to that of HLA-DRB1*0301.<br><br>Genotypic Risk for Class II and Class III Association Signals
Having established independent association at the allelic level with HLA-DRB1*0301 and rs419788-T in our UK SLE cohort, we wanted to further determine the genotypic risk conferred by these variants and hence gain insight into their underlying mode of inheritance in lupus. We used case-control and family-based analyses to assess genotypic risk at HLA-DRB1, while the family-based test alone was used for rs419788. Common family-based tests of LD, such as those used in this study (Genehunter), measure transmission distortion based on allele counts rather than genotype counts; the former has been shown to be more powerful under additive models, while the latter has greater power under recessive or dominant genetic models [27]. The genotype-pedigree disequilibrium test (geno-PDT) determines LD between a locus genotype and disease by comparing genotypes that are transmitted from parent to proband with those that are not [27]. We used the geno-PDT to assess genotypic risk for our class II and class III association signals: HLA-DRB1 and the SNP rs419788. In the case-control analysis for HLA-DRB1, ORs with 95% CI were calculated and Fisher's exact test employed to assess statistically significant differences between HLA-DRB1 genotypes in lupus probands and healthy controls. For HLA-DRB1, the alleles were coded as follows: HLA-DRB1*0301, HLA-DRB1*1501, HLA-DRB1*X where X represents all HLA-DRB1 alleles other than HLA-DRB1*0301, and HLA-DRB1*1501. We included HLA-DRB1*1501 in the analysis, even though we find no allelic association in our cohort, because previous studies have shown a greater risk for lupus in individuals who are compound heterozygotes for HLA-DRB1*0301- and HLA-DRB1*1501-containing haplotypes [25,28]. Overall the results are consistent with a dominant effect from HLA-DRB1*0301 (Table 2) and a dose-dependent (additive) effect from rs419788-T (Table 3). Specifically, both case-control and geno-PDT demonstrate that there is no dose-dependent increase in disease risk for HLA-DRB1*0301. Rather, it appears that the presence of a single copy of HLA-DRB1*0301 alone is sufficient to increase susceptibility to disease. Moreover the 0301/X genotypes constitute the greatest risk in our cohort rather than the 0301/1501 heterozygotes. Genotypes containing HLA-DRB1*1501 in the absence of HLA-DRB1*0301 revealed no significant association in our cohort.
All three rs419788 genotypes demonstrated significant association in our lupus families (Table 3). The common CC genotype was significantly under transmitted, while the rare T allele displayed dose-dependent over transmission to lupus probands.<br><br>Characterization of HLA-DRB1*0301 Risk Haplotype
Next, we wanted to further delimit the MHC class II association signal that we have detected at HLA-DRB1. We used phased parental genotype data to compare the allelic composition of HLA-DRB1*0301-bearing haplotypes that were transmitted (T) to affected probands to those that were not transmitted (or untransmitted, UT) with the aim of identifying differences that could delineate the lupus susceptibility interval(s) arising from this haplotype (summarized in Figures 2A, 2B, and S2). We observed a striking difference between transmitted and untransmitted chromosomes within the class II region: nearly all transmitted HLA-DRB1*0301 haplotypes (99%) are identical across a 180 kb region defined by eight SNPs, whereas the corresponding region within untransmitted HLA-DRB1*0301 haplotypes exhibits significant recombination. These data strongly suggest the existence of a risk haplotype that, interestingly, contains only three expressed genes: HLA-DRB1, HLA-DQA1, and HLA-DQB1. Furthermore, we can confidently define the allelic composition of this risk haplotype, as these three genes are in strong LD and occur in one common haplotype in Caucasians: HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. Thus, we hypothesize that the specific combination of all three alleles is required to confer disease risk in lupus or that disease susceptibility lies with either HLA-DRB1*0301 or the HLA-DQ alleles. We do not have sufficient numbers of recombinant chromosomes in this risk region to further delimit this signal: 2/176 (1.1%) transmitted HLA-DRB1*0301 haplotypes are recombinant at HLA-DQA1-HLA-DQB1; 3/178 (1.7%) transmitted haplotypes identical across HLA-DQA1-HLA-DQB1 do not possess HLA-DRB1*0301.
The composite relative extended haplotype homozygosity (REHH) versus frequency plot for UK SLE; Utah residents with ancestry from northern and western Europe (CEPH); and Yoruba in Ibadan, Nigeria (Yoruba) populations is shown in Figure 3A. We can only comment on evidence for positive selection in CEPH individuals, as we have used this population alone to assess background variation on Chromosome 6. The SLE and Yoruba cohorts are shown for comparative purposes. We find no evidence of positive selection for HLA-DRB1*0301 in the CEPH population. However, this allele is enriched in our lupus cohort (21% of parental chromosomes) and displays greater extended homozygosity when compared with HLA-DRB1*0301-bearing haplotypes in CEPH and Yoruba. Hence, the HLA-DRB1*0301 allele in lupus is observed as an outlier on the plot when compared to background variation in CEPH. These data support our previous observations (outlined above) of the highly conserved nature of HLA-DRB1*0301 haplotypes in lupus. In addition, the haplotype bifurcation plots centered on HLA-DRB1*0301 for UK SLE, CEPH, and Yoruba populations in Figure 3B illustrate preservation of the common HLA-DRB1*0301 haplotype in CEPH and UK SLE, while that seen in the Yoruba is significantly different. The class II regions of all three populations are essentially identical across our chosen SNPs; the main differences lie in class III. The difference in African populations in the class III region is one possible explanation for the lack of evidence for an association between HLA-DRB1*0301 and SLE in African or African American populations. However, HLA-DRB1*0301 has a lower frequency (∼7%–10%) in African populations compared with Europeans (∼13%), and the number of HLA association studies conducted in African populations is very limited.<br><br>Characterization of Class III Region Risk Haplotype
Our data reveal a second independent signal at the MHC in SLE arising from the T allele of SNP rs419788 in intron 6 of the class III gene, SKIV2L. Further evidence supporting the independence of the rs419788-T and HLA-DRB1*0301 alleles is provided by the moderate LD between these two variants (r2 = 0.24) coupled with our data demonstrating that only 47% of rs419788-T allele-bearing haplotypes contain HLA-DRB1*0301.
The structure and composition of T and UT haplotypes anchored at rs419788-T were essentially identical (Figures 2C, 2D, and S2), and hence not informative in delimiting our class III signal. Therefore, we examined the LD structure around our associated class III SNP to better define our disease risk interval. In our lupus dataset the rs419788-T allele resides on three of seven haplotypes present within a large block of six SNPs exhibiting strong LD. This haplotype block encompasses roughly 270 kb containing class III genes from SLC44A4 to AGER, including the RCCX module. Next, we analyzed the haplotype block structure of this region in CEPH families using SNP data dumped from the International HapMap Project (http://www.hapmap.org/). The greater density of SNP typing available in the HapMap CEPH population compared to our current UK SLE map allowed us to potentially refine our signal by exploring correlations between our associated SNP and those surrounding it. Analysis of these data (Figure 4) suggests the presence of short-range LD around our associated variant, rs419788, in CEPH families, encompassing approximately 40 kb of the genome which includes the five genes: complement factor B (CFB), RD RNA binding protein (RDBP), SKIV2L, dom-3 homolg Z (<span class="yellow">C. elegans</span>) (DOM3Z), and serine/threonine kinase 19 (STK19), and does not include the complement C4 locus. Furthermore, assessment of marker association in our lupus dataset demonstrates that after conditioning for HLA-DRB1*0301, the only markers that retain association signals are telomeric of SKIV2L, suggesting that complement C4, which is centromeric to this gene, may not be responsible for our independent class III signal.<br><br>Subphenotype Analysis
In order to gain further insight into disease pathogenesis, we examined common lupus subphenotypes. Such subsets are more homogeneous than lupus per se and thus maybe enriched for specific predisposing variants. In addition, one might expect a close association between MHC class II alleles and autoantibody subsets in lupus if these are indeed causal variants, given their role in antigen presentation and subsequent humoral immunity. We therefore tested our two main MHC association signals, HLA-DRB1*0301 and rs419788, for association with renal disease and autoantibody subsets in our lupus cohort.
We found that HLA-DRB1*0301 was associated with the presence of anti-Ro and anti-La antibodies in our UK SLE cohort, with the latter showing the greatest evidence of association (anti-La nominal p < 0.001 compared with anti-Ro nominal p < 0.025). We found no association of HLA-DRB1*0301 with renal disease or any other autoantibody subsets in our dataset (see Table S4 for detailed results).
Genotypes of the SNP rs419788 were not associated with any of the tested lupus subphenotypes after controlling for the effect of HLA-DRB1*0301 (unpublished data).<br><br>
Discussion
We present the first family-based SNP association study of the MHC in SLE. We have genotyped 69 markers (HLA-DRB1 and 68 SNPs) across 2.4 Mb of the MHC, encompassing class III and class II, in a cohort of 314 UK Caucasian SLE trios. Transmission disequilibrium testing of these data has shown predominant association with the alleles HLA-DRB1*0301 and rs419788-T, together with 12 other MHC SNPs. Moreover, using conditional analyses, we have shown that the two primary signals of association at the MHC are independent of each other. Specifically, one signal arises from HLA-DRB1*0301 in class II and the other from the T allele of SNP rs419788 in the class III gene SKIV2L.
Examination of bifurcation plots for T and UT HLA-DRB1*0301-containing haplotypes has enabled delineation of our class II association signal to a 180 kb region encompassing HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201. These data substantially refine that previously published by Graham et al. in 2002 [25], where the lupus susceptibility interval within HLA-DRB1*0301-containing haplotypes could only be delimited to a 1 Mb region encompassing class II and class III. The precise causal variant(s) within this region remains to be determined, as the three implicated alleles exhibit strong LD with few recombination events separating them (two out of 176 transmitted HLA-DRB1*0301 chromosomes in our dataset). However, all three allelic variants represent attractive functional candidates in lupus susceptibility for their role in antigen presentation and stimulation of the adaptive immune response.
Our association of HLA-DRB1*0301 with lupus concurs with published data in Caucasian cohorts and is well established [16]. While our lack of association with HLA-DRB1*1501 and HLA-DRB1*0801 is consistent with previous data from the UK [29], Spain [30], the Netherlands [31], Sweden [32], Mexico [33], and the US [34], it conflicts with that of other US groups [25,35]. Interestingly, we demonstrate a trend, though not statistically significant, for undertransmission of HLA-DRB1*0701—a result also observed in prior UK and Canadian lupus studies [29,36]. Moreover, a negative association of HLA-DRB1*0701 has been reported in other autoimmune diseases including Graves disease [37,38], type 1 diabetes [39], and rheumatoid arthritis [40].
It appears that the conflicting results between UK SLE and previous US (Minnesota [MN]) [25] SLE data stem from differences in HLA-DRB1 allele frequency in the probands of each cohort. The reason for this is unclear. A comparison between UK and MN SLE cohorts (Table 4) reveals that UK SLE cases are enriched for HLA-DRB1*0301 but not HLA-DRB1*0801 or HLA-DRB1*1501 when compared to a UK control population. In contrast, MN SLE cases are enriched for HLA-DRB1*0301-DQB1*0201, DRB1*0801-DQB1*0402, and DRB1*1501-DQB1*0602 inferred haplotypes when compared to MN controls [25]. There is no statistically significant difference in the aforementioned HLA class II alleles/haplotypes between UK and MN control populations that could account for the disparity seen in the respective lupus cohorts. Differences in disease severity and subphenotype frequency between the two populations could account for the observed discrepancy. From the limited data available we found that the presence of renal disease appears to be similar in both cohorts (UK SLE 36% compared with MN SLE 40%), while the gender ratios are significantly different (female: male UK SLE 11:1 compared with MN SLE 57:1, Chi square p value < 0.001). We were unable to compare other lupus subphenotypes. Furthermore, closer inspection of these data reveals that microsatellite-inference of HLA-DRB1 alleles in the MN SLE dataset may underestimate the frequency of HLA-DRB1*0301 and overestimate that of HLA-DRB1*1501, thus diminishing the effect of the former and enhancing that of the latter. It is also possible that the MN SLE cohort shows greater racial heterogeneity in comparison to our UK SLE cohort, despite both being characterized as Caucasian.
Previous studies have demonstrated increased risk for lupus in individuals carrying particular combinations of microsatellite-inferred HLA-DRB1-HLA-DQB1 haplotypes [25,28]. The highest risk genotype was found to be the compound heterozygote HLA-DRB1*0301-DQB1*0201/HLA-DRB1*1501-DQB1*0602, while HLA-DRB1*0301-DQB1*0201-containing genotypes demonstrated a dose-dependent effect in increasing lupus susceptibility [25,28]. In the present study, we have examined genotypic risk at the classically typed HLA-DRB1 locus and in contrast to the aforementioned data of Graham et al. [25,28] we have shown a likely dominant effect of the associated allele, HLA-DRB1*0301. The case-control and family-based analyses for HLA-DRB1 also show the greater power of the former to detect significant association (Table 2). Specifically, all genotypes containing HLA-DRB1*0301 show increased transmission to lupus probands; however, homozygotes show no greater risk compared with heterozygotes, as would be expected under additive or multiplicative models. Thus, a dominant model of inheritance, requiring the presence of a single copy of the disease-predisposing variant alone, likely underlies the susceptibility conferred by HLA-DRB1*0301 in UK SLE. Such a model would fit an antigen presentation hypothesis where susceptible individuals carrying an HLA-DRB1*0301 allele are able to present auto-antigens to CD4+ lymphocytes, thus stimulating an autoimmune response. The differences between our UK SLE and the previously published US SLE data may reflect disease, ethnic, and haplotypic heterogeneity.
Interestingly, analysis of genotypic risk at the associated class III marker, rs419788, suggests an additive (dose-dependent) pattern of inheritance for the rare T allele, where one copy confers a low risk of disease and two copies results in greater susceptibility. The different inheritance patterns for our class II and class III association signals provide further evidence for their independence.
A variety of HLA-DR and HLA-DQ alleles have been associated with autoantibody subsets in ethnically diverse populations of lupus. The strongest associations have been demonstrated between anti-Ro and anti-La antibodies and HLA-DR3 and HLA-DQ2 (HLA-DQB1*0201), which are in strong LD [41–45] in case-control studies. Here, we confirm the association of HLA-DRB1*0301 with anti-Ro and anti-La antibody production in our family-based cohort.
Examination of LD structure around our second independent association, rs419788-T in class III, coupled with the results of our conditional analysis, suggests that this signal could also be delimited to a relatively narrow genomic interval of about 40 kb given further SNP mapping in our cohort. This region includes the genes CFB, RDBP, SKIV2L, DOM3Z, and STK19, but does not include complement C4. Thus, complement C4 null alleles, which have been implicated in lupus pathogenesis, may not be responsible for our class III signal. We conclude, therefore, that our family-based mapping study has potentially revealed a hitherto unknown lupus susceptibility interval in the class III region of the MHC. However, we cannot conclusively exclude association at complement C4/RCCX without direct determination of C4 polymorphism/copy number in our cohort.
With respect to the genes implicated in our study, CFB is a vital component of the alternate complement pathway and disregulation may clearly affect the inflammatory response [46]. RD and Skiv2l are proteins potentially involved in RNA processing. The RD protein forms part of a negative elongation factor (NELF) complex that represses RNA polymerase II transcript elongation, while Skiv2l is a DEAD box protein with possible function as an RNA helicase. The function of Dom3z is currently unknown, although the homologous <span class="blue">yeast</span> protein binds nuclear exoribonuclease. Moreover, its ubiquitous expression suggests a housekeeping role. STK19 is a protein kinase of unknown function with primary nuclear localization [47]. Interestingly, RDBP and SKIV2L are found to be highly expressed in T lymphocytes, B lymphocytes, and dendritic cells (SymAtlas, http://symatlas.gnf.org/SymAtlas/).
A number of studies have demonstrated conflicting evidence for and against association with various TNF locus polymorphisms in SLE [48]. A recent meta-analysis of the TNF-308G/A promoter polymorphism in SLE [48] revealed evidence of association for the minor allele (A) in European populations; however, this study did not account for LD with class II alleles. On conditioning our dataset for HLA-DRB1*0301, we find that the TNF promoter signal is lost, suggesting that this association is not independent and is due to LD with HLA-DRB1*0301 (or another variant in LD with HLA-DRB1*0301).
In summary, we have found association with two distinct and independent variants within the class II (HLA-DRB1*0301) and class III (SKIV2L) regions of the MHC in UK SLE trios. We can delimit our class II signal in lupus to three genetic variants (HLA-DRB1*0301-HLA-DQA1*0501-HLA-DQB1*0201) that may confer disease risk in combination or as separate signals. Our class III signal importantly excludes independent association at the TNF promoter polymorphism TNF-308G/A and potentially provides a novel locus for further study.<br><br>Materials and Methods
Study cohorts.
SLE families. The cohort comprises 314 complete SLE trios (that is, mother, father, and affected lupus proband) collected as previously described [49]. All study <span class="yellow">participants</span> are European Caucasian on the basis of grandparental origin. All 314 lupus probands (288 female, 26 male) fulfill the revised American College of Rheumatology (ACR) criteria for SLE [50], 36% of whom have a diagnosis of lupus nephritis. Written consent was obtained from all study <span class="yellow">participants</span> and ethical approval for this study was obtained from the Multi-Centre Research Ethics Committee (MREC 2 June 1998).<br><br>Healthy controls.
The control population for the HLA-DRB1 genotypic risk case-control analysis constitutes 1,667 healthy males of Northern European origin. The individuals are potential hematopoietic stem cell donors and were typed to four digits for HLA-DRB1 at the Anthony Nolan Trust, UK for this purpose. The level of resolution used for the typing of HLA-DRB1*15 alleles in these healthy controls resulted in the ambiguous allele string HLA-DRB1*1501/1502/1504/1506. However, it is likely that the great majority are HLA-DRB1*1501. There is no gender bias in HLA-DRB1 allele frequencies, so although we have used a male control cohort, the frequencies would be expected to be the same in a similar female cohort (Steven Marsh, personal communication).<br><br>SNP genotyping.
Eighty-six SNPs were chosen for genotyping in our mapping study. Specifically, we selected 40 MHC class II and class III haplotype tagging SNPs from a preliminary MHC SNP map [51] that had previously shown robust genotyping efficacy. In addition, we typed 36 MHC class II tag SNPs from a subsequent high-resolution MHC study [52]. We also included the TNF-308G/A promoter SNP, together with nine further SNPs in the region of HLA-B and MICA obtained from the database, dbSNP (http://www.ncbi.nlm.nih.gov/projects/SNP/). The latter SNPs had not been well characterized. All variants were typed in the entire cohort (n = 942). The SNPs span approximately 2.4 Mb of the MHC from the class I gene, KIAA1949 to the class II pseudogene, HLA-DPB2 and thus encompass MHC class III and class II. SNP genotyping was performed at the Broad Institute of MIT and Harvard and at Imperial College London by matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry using the Sequenom MassARRAY platform as previously described [53]. SNPs that failed Sequenom typing were retyped by KBiosciences (http://www.kbioscience.co.uk/) using their in-house SNP genotyping methodology, KASPar (http://www.kbioscience.co.uk/genotyping/index.htm), a competitive allele-specific PCR technique.<br><br>HLA-DRB1 genotyping.
HLA-DRB1 typing was performed at the Anthony Nolan Trust, UK. All samples (n = 942 UK SLE trios and n = 1,667 controls) were genotyped using LABType SSO (sequence-specific oligonucleotide) typing technology according to the manufacturer's written recommendations (http://www.onelambda.com). Briefly, a locus-specific biotinylated PCR amplicon is produced, denatured, and rehybridized to complementary oligonucleotide probes conjugated to fluorescently coded beads. The bound biotinylated PCR product can be detected using R-phycoerythrin-conjugated streptavidin. A flow analyzer identifies the fluorescent intensity of phycoerythrin on each bead. The assignment of HLA type is based on the reaction pattern compared to patterns associated with known sequences.
High resolution testing was performed using the Dynal AllSet+ SSP (sequence-specific primers) DRB1 assay according to the manufacturer's protocol (Invitrogen) for SLE families only. The presence or absence of PCR amplification was detected in a gel electrophoresis step using visualization by ethidium bromide incorporation. Genotypes were determined using SSPTool software.
Samples that could not be resolved to four digits using SSO and PCR-SSP were analyzed by DNA sequencing of exon 2 of HLA-DRB1. Primers, reagents, and protocols were provided by The Anthony Nolan Trust, UK (primer sequences are available on request). Specific HLA-DRB1 alleles were assigned by comparing the resultant sequence with reference sequence from the IMGT/HLA Database [54].<br><br>Data analysis.
Mendelian inconsistencies were removed using PedCheck [55]. Families in which more than eight markers demonstrated Mendel errors were removed from further analysis. Markers with less than 80% genotyping efficiency and markers where more than eight families showed Mendel errors were also excluded from analysis. Two markers located within the SLE associated class II region (rs2239802 in intron 4 of HLA-DRA and rs6457594 in the region between HLA-DRB9 and HLA-DRB5) show deviation from Hardy–Weinberg equilibrium (HWE), which may reflect an undetected SLE association or systematic genotyping error. HWE was assessed in parental samples in our cohort. There is currently no uniform opinion in the community regarding the inclusion or exclusion of SNPs that show deviation from HWE, hence we elected to include these markers in the final analysis.
Sixty-eight out of the total 86 SNPs passed our quality-control measures (see Table S1 for details). In summary, one SNP was monomorphic in our dataset, four SNPs yielded low genotyping efficiency, and 13 SNPs were excluded for unacceptable Mendel error rate. The mean call rate for all markers post-quality control was 94% (range 83% to 99%).
Family-based association testing was performed using Genehunter TDT (version 2.1) [56] and TRANSMIT (version 2.5) [57]. Haplotypes were constructed and permutation testing performed using Haploview (version 3.32) [58]. Significance of association signals in all analyses was based on permutation testing (10,000 permutations). The data are represented both as nominal and permuted p values. ORs with 95% CI for family-based analyses were calculated in PLINK (http://pngu.mgh.harvard.edu/purcell/plink/) [59]. Conditional regression analyses were undertaken using WHAP [60].
The geno-PDT was performed using PDT version 5.1 with default settings [27]. The HLA-DRB1 alleles were coded into three groups for the geno-PDT and the case-control analysis: HLA-DRB1*0301, HLA-DRB1*1501 and HLA-DRX where HLA-DRX includes all HLA-DRB1 alleles other than HLA-DRB1*0301 or HLA-DRB1*1501. The HLA-DRB1*1501 code in the healthy controls represents the allele string HLA-DRB1*1501/1502/1504/1506, as described previously. The HLA-DRB1*1501 code in the lupus probands represents the alleles HLA-DRB1*1501 (84 out of 91 *1501 and *1502 alleles) and HLA-DRB1*1502 (7/91 *1501 and *1502 alleles), as *1504 and *1506 were not present in this population. Fisher's exact test was used to assess significance of association in the case-control analysis.<br><br>Subphenotype analysis.
We looked for association of the HLA-DRB1*0301 allele with autoantibody subsets and renal disease in our cohort using the Chi-square test. We compared cases with and without the subphenotype of interest with DRB1*0301 homozygosity, heterozygosity, combined homozygosity and heterozygosity, and non-DRB1*0301 status. We performed the same analyses for homozygous and heterozygous genotypes of the associated SNP rs419788. The autoantibody subsets compared were anti-C1q, IgG, and IgM anti-cardiolipin antibodies (ACLG and ACLM), anti-Ro, anti-La, anti-RNP, anti-Sm, and anti-dsDNA.<br><br>Delineation of associated MHC haplotypes and evidence for positive selection.
We looked for positively selected alleles in our dataset using the long-range haplotype test as measured by extended haplotype homozygosity (EHH), previously described by Sabeti et al. [26]. Essentially, such an analysis allows assessment of positive selection by mining datasets for high frequency extended haplotypes in comparison to the other core haplotypes at a locus.
EHH is defined as the probability that two randomly chosen chromosomes carrying the core haplotype of interest will be identical by descent (homozygosity at all SNPs) for the entire interval from the core to a distance x. The REHH is the ratio of the EHH on the tested core haplotype compared with the combined EHH of all the other core haplotypes at the region excluding the tested core; as such, REHH accounts for local variation in recombination rate while EHH does not [26].
The program emphase was employed to assign the phase of parental genotype data and reconstruct missing information. Emphase is a simple phaser similar to the phaser of Excoffier and Slatkin [61]. It is very fast, especially on large datasets, and sufficiently accurate for most genetic applications. EHH analysis was performed on the phased parental data using the software program SWEEP (http://www.broad.mit.edu/mpg/sweep/index.html).<br><br>Haplotype bifurcation plots.
We represent the breakdown of LD on core haplotypes using haplotype bifurcation diagrams generated in the program TREE [62] (also explained in [52]).<br><br>REHH versus frequency plots.
Fifty-three SNPs (identified in Figure 1) are common to our dataset and the CEPH and Yoruba HapMap [63] populations. These three datasets, together with CEPH SNP data for Chromosome 6 in its entirety, were used to generate separate REHH versus frequency plots in SWEEP. The plots from the four cohorts were combined for visual, not statistical, comparison. Evidence for positive selection was quantitatively assessed in CEPH individuals, as this population alone was used to assess background variation on Chromosome 6. The UK SLE and Yoruba cohort data are shown for comparison. The 95th percentile based on total CEPH Chromosome 6 SNP data is shown.<br><br>
Supporting Information
Accession Numbers
The Online Mendelian Inheritance in <span class="yellow">Man</span> (OMIM, http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim) accession numbers for the genes described in this study are as follows: AGER, 600214; BAT3, 142590; C4A, 120810; C4B, 120820; CFB, 138470; DOM3Z, 605996; DOM3Z, 605996; EHMT2, 604599; HLA-B, 142830; HLA-DPB2, 120290; HLA-DQA1, 146880; HLA-DQB1, 604305; HLA-DRA, 142860; HLA-DRB1, 142857; KIAA1949, 610990; MICA, 600169; NOTCH4, 164951; RDBP, 154040; RDBP, 154040; SKIV2L, 600478; SLC44A4, 606107; STK19, 604977; STK19, 604977; TNF, 191160; and TNXB, 600985.<br><br><br><br><h3>pmcA2409186</h3>Comparative Genome Analysis of Filamentous Fungi Reveals Gene Family Expansions Associated with Fungal Pathogenesis
Abstract
Fungi and oomycetes are the causal agents of many of the most serious diseases of plants. Here we report a detailed comparative analysis of the genome sequences of thirty-six species of fungi and oomycetes, including seven plant pathogenic species, that aims to explore the common genetic features associated with plant disease-causing species. The predicted translational products of each genome have been clustered into groups of potential orthologues using Markov Chain Clustering and the data integrated into the e-Fungi object-oriented data warehouse (http://www.e-fungi.org.uk/). Analysis of the species distribution of members of these clusters has identified proteins that are specific to filamentous fungal species and a group of proteins found only in plant pathogens. By comparing the gene inventories of filamentous, ascomycetous phytopathogenic and free-living species of fungi, we have identified a set of gene families that appear to have expanded during the evolution of phytopathogens and may therefore serve important roles in plant disease. We have also characterised the predicted set of secreted proteins encoded by each genome and identified a set of protein families which are significantly over-represented in the secretomes of plant pathogenic fungi, including putative effector proteins that might perturb host cell biology during plant infection. The results demonstrate the potential of comparative genome analysis for exploring the evolution of eukaryotic microbial pathogenesis.<br><br>Introduction
Fungi and oomycetes are responsible for many of the world's most devastating plant diseases including late blight disease of <span class="yellow">potato</span>, caused by the oomycete pathogen <span class="yellow">Phytophthora infestans</span> and <span class="yellow">rice</span> blast disease caused by the ascomycete fungus <span class="yellow">Magnaporthe grisea</span>, both of which are responsible for very significant harvest losses each year. The enormous diversity of crop diseases caused by these eukaryotic micro-organisms poses a difficult challenge to the development of durable disease control strategies. Identifying common underlying molecular mechanisms necessary for pathogenesis in a wide range of pathogenic species is therefore a major goal of current research. Approximately 100,000 species of fungi have so far been described, but only a very small proportion of these are pathogenic [1]. Phylogenetic studies have, meanwhile, shown that disease-causing pathogens are not necessarily closely-related to each other, and in fact are spread throughout all taxonomic groups of fungi, often showing a close evolutionary relationship to non-pathogenic species [2], [3]. It therefore seems likely that phytopathogenicity has evolved as a trait many times during fungal and oomycete evolution [1] and in some groups may be ancestral to the more recent emergence of saprotrophic species. A significant effort has gone into the identification of pathogenicity determinants– individual genes that are essential for a pathogen to invade a host plant successfully, but which are dispensable for saprophytic growth [4], [5]. However, far from being novel proteins encoded only by the genomes of pathogenic fungi, many of the genes identified so far encode components of conserved signalling pathways that are found in all species of fungi, such as the mitogen activated protein (MAP) kinases [6], adenylate cyclase [7] and G-protein subunits [8]. The MAP kinase pathways, for example, have been studied extensively in the budding <span class="blue">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span> and trigger morphological and biochemical changes in response to external stimuli such as starvation stress or hyperosmotic conditions [9]. In pathogenic fungi, components of these pathways have evolved instead to regulate the morphological changes associated with plant infection. For example, appressorium formation in the <span class="yellow">rice blast fungus</span> <span class="yellow">Magnaporthe grisea</span>, stimulated by hard, hydrophobic surfaces is regulated by a MAP kinase cascade [10]. This pathway deploys novel classes of G-protein coupled receptors not found in the genome of <span class="yellow">S. cerevisiae</span> [11], but the inductive signal is transmitted via a MAP kinase, Pmk1, that is a functional homologue of the <span class="blue">yeast</span> Fus3 MAP kinase where it serves a role in pheromone signalling [10]. Similarly, conserved metabolic pathways such as the glyoxylate cycle and amino acid biosynthesis are also important for pathogenesis [12]–[14]. This may in some cases reflect the nutritional environment the pathogen encounters when growing in the host plant tissue, and in others shows the importance of simple metabolites for pathogenic processes, such as the role of glycerol as a compatible solute for generating turgor pressure in the appressorium of <span class="yellow">M. grisea</span> [15]. It is undoubtedly the case, however, that identification of such genes has also been a consequence of the manner in which these studies have been carried out, often using <span class="blue">yeast</span> as a model organism to test hypotheses concerning the developmental biology and biochemistry of plant pathogenic species.
Other pathogenicity factors identified to date have been shown to be involved in functions associated with host infection, such as plant cell wall degradation, toxin biosynthesis and protection against plant defences [reviewed in 5]. Identification of a pathogenicity factor generally involves making a mutant fungal strain with a non-functioning version of the gene by targeted gene deletion and assaying the ability of the mutant to cause disease. Therefore, most pathogenicity factors identified so far, have been validated in only a small number of genetically tractable pathogenic fungi, such as <span class="yellow">M. grisea</span> and the <span class="yellow">corn</span> smut <span class="yellow">Ustilago maydis</span> and many of the advances in understanding the developmental biology of plant infection have occurred in these model pathogens [16], [17]. However, there are severe limitations to studying pathogenicity by mutating one gene at a time and working predominantly with a hypothesis-driven, reverse genetics approach. Many virulence-associated processes, for instance, such as the development of infection structures and haustoria, are likely to involve a large number of gene products and so there is likely to be redundancy in gene function. One example of this is cutinase, a type of methyl esterase that hydrolyses the protective cutin layer present on the outside of the plant epidermis. Cutinase was excluded as a pathogencity factor for <span class="yellow">M. grisea</span> on the basis that a mutant strain containing a non-functional cutinase-encoding gene was still able to cause <span class="yellow">rice</span> blast disease [18]. However, sequencing of the <span class="yellow">M. grisea</span> genome has shown the presence of eight potential cutinase-encoding genes implicated in virulence [19]. Additionally, targeted gene deletion is not feasible in many important pathogens and the normal definition of fungal pathogenicity cannot be applied in the case of obligate biotrophs, such as the powdery mildew fungus <span class="yellow">Blumeria graminis</span>, which cannot be cultured away from living host plants. Therefore, new approaches are needed to identify genes that are vital for the process of pathogenicity. These include high-throughput methods such as microarray analysis, serial analysis of gene expression (SAGE), insertional mutagenesis, proteomics and metabolomics [19], [20] and are dependent on the availability of genome sequence information.
After the initial release of the genome of the budding <span class="blue">yeast</span> <span class="yellow">S. cerevisiae</span> in 1996 [21], the number of publicly available sequenced fungal genomes has recently risen very quickly. A large number of fungal genome sequences are now publicly available, including those from several phytopathogenic fungi, including <span class="yellow">M. grisea</span> [22], <span class="yellow">Ustilago maydis</span> [23], <span class="yellow">Gibberella zeae</span> [24] (the causal agent of head blight of <span class="yellow">wheat</span> and <span class="yellow">barley</span>), <span class="yellow">Stagonospora nodorum</span> [25] (the causal agent of glume blotch of <span class="yellow">wheat</span>), the grey mould fungus <span class="yellow">Botrytis cinerea</span> and the white mould fungus <span class="yellow">Sclerotinia sclerotiorum</span> [reviewed in 19]. Comparison of gene inventories of pathogenic and non-pathogenic organisms offers the most direct means of providing new information concerning the mechanisms involved in fungal and oomycete pathogenicity. In this report, we have developed and utilized the e-Fungi object-oriented data warehouse [26], which contains data from 36 species of fungi and oomycetes and deploys a range of querying tools to allow interrogation of a significant amount of genome data in unparalleled detail. We report the identification of new gene families that are over represented in the genomes of filamentous ascomycete phytopathogens and define gene sets that are specific to diverse fungal pathogen species. We also report the putatively secreted protein sets which are produced by plant pathogenic fungi and which may play significant roles in plant infection.<br><br>Results
Identification of orthologous gene sets from fungal and oomcyete genomes
Genome sequences and sets of predicted proteins were analysed from 34 species of fungi and 2 species of oomycete (Table 1). In order to compare such a large number of genomes, an object-oriented data warehouse has been constructed known as e-Fungi [26] which integrates genomic data with a variety of functional data and has a powerful set of queries that enables sophisticated, whole-genome comparisons to be performed. To compare genome inventories, the entire set of predicted proteins from the 36 species (348,787 proteins) were clustered using Markov Chain Clustering [27] as described previously [28], [29]. A total of 282,061 predicted proteins were grouped into 23,724 clusters, each cluster representing a group of putative orthologues. The remaining 66,934 sequences were singletons, the products of unique genes. A total of 165 clusters contained proteins from all 36 species used in this study (Table S1). Not surprisingly, they included many proteins involved in basic cellular processes, such as ribosomal proteins, components of transcription, translation and DNA replication apparatus, cytoskeletal proteins, histones, proteins involved in the secretory pathway, protein folding, protein sorting and ubiquitin-mediated proteolysis and enzymes involved in primary metabolism. Only 16 clusters contained proteins that were found in all 34 species of fungi, but which were absent from the two species of oomycete (Table S2). This number of fungal-specific clusters is surprisingly low considering the phylogenetic distance between the oomycetes and fungi [30]. The list however, is consistent with the fundamental differences in biology between fungi and oomycetes and included proteins involved in fungal septation, glycosylation, transcriptional regulation, cell signalling, as well as two amino-acyl tRNA synthetases. The obligate mammalian pathogen <span class="yellow">Encephalitozoon cuniculi</span>, a microsporidian fungus, has a reduced genome that codes only for 1,997 proteins and lacks genes encoding enzymes of many primary metabolic pathways such as the tricarboxylic acid cycle, fatty acid β-oxidation, biosynthetic enzymes of the vast majority of amino acids, fatty acids and nucleotides, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. It also lacks mitochondria and peroxisomes [31]. Therefore, we reasoned that the inclusion of this species in the analysis of MCL clusters is likely to result in underestimation of the number of groups of conserved proteins. By discarding <span class="yellow">E. cuniculi</span>, there are 377 clusters that contained proteins from 35 species of fungi and oomycetes (Table S3). This relatively small number of fungal-conserved clusters reflects the large evolutionary distance between members of the fungal kingdom, as well as complex patterns of gene gains and losses during the evolution of fungi. Basidiomycetes and ascomycetes are thought to have diverged nearly 1,000 million years ago [32] and the Saccharomycotina alone are more evolutionarily diverged than the Chordate phylum of the animal kingdom [33]. Since the divergence of Saccharomycotina (hemiascomycetes) and Pezizomycotina (euascomycetes), the genomes of the latter have greatly increased in size, partly due to the appearance of novel genes related to the filamentous lifestyle. Lineage-specific gene losses have also been shown in a number of hemiascomycete species [34]. As well as the groups of proteins mentioned above (Table S1), the fungal-conserved clusters included those containing enzymes from primary metabolic pathways not present in <span class="yellow">E. cuniculi</span>, such as the tricarboxylic acid cycle, amino acid metabolism, fatty acid biosynthesis, cholesterol biosynthesis and nucleotide metabolism, as well as components of the respiratory electron transport chain and F1-F0 ATP synthase. The conserved protein clusters also include a number of transporters (including mitochondrial transporters), enzymes involved in haem biosynthesis, autophagy-related proteins, those involved in protein targeting to the peroxisome and vacuole and additional groups of proteins involved in signal transduction that are not present in <span class="yellow">E. cuniculi</span> (including those involved in inosine triphosphate and leukotriene metabolism). The analysis also showed there were 105 clusters that contained proteins from 33 species of fungi (excluding <span class="yellow">E. cuniculi</span>), but not from the two species of oomycete (see Table S4). As well as those mentioned previously (Table S2), the group includes a number of clusters of transporters that are conserved in fungi but not found in oomycetes, as well as proteins involved in fungal cell wall synthesis, and lipid metabolism. It may be the case that the genomes of oomycete species do not possess orthologues of the fungal genes in these clusters, or alternatively, the large evolutionary distance between the oomycetes and fungi mean that the corresponding orthologues from each Kingdom cluster separately.<br><br>Comparative analysis of yeasts and filamentous fungi
One striking difference in the morphology of species of fungi is between those that have a filamentous, multi-cellular growth habit and those that grow as single <span class="blue">yeast</span> cells. There is some overlap between these two groups; because some fungi are dimorphic or even pleiomorphic, switching between different growth forms depending on environmental conditions or the stage of their life cycle. For example, the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> can exist saprophytically as haploid <span class="blue">yeast</span>-like cells, but needs to form a dikaryotic filamentous growth form in order to infect the host plant [23]. Generally the genomes of the filamentous fungi contain more protein-encoding genes (9,000–17,000) than those from unicellular yeasts (5,000–7,000), perhaps reflecting their greater morphological complexity and secondary metabolic capacity. <span class="yellow">U. maydis</span>, however, has 6,522 protein encoding genes, perhaps reflecting its lack of extensive secondary metabolic pathways and its potential usefulness in defining the minimal gene sets associated with biotrophic growth [23]. The increase in proteome size in filamentous ascomycetes may be due to the expansion of certain gene families or the presence of novel genes that are essential for the filamentous lifestyle.
For the purposes of this study, the filamentous fungi were defined as the filamentous ascomycetes (subphylum Pezizomycotina), basidiomycetes and zygomycetes and the unicellular fungi were defined as the budding yeasts (order Saccharomycetales), the archiascomycete <span class="yellow">Schizosaccharomyces pombe</span> and the microsporidian fungus <span class="yellow">Encephalitozoon cuniculi</span>. A total of 37 MCL clusters contained proteins from all species of filamentous fungi, but no species of unicellular fungi (Table 2). Interestingly, eight of these clusters also contained proteins from both species of oomycete represented in e-Fungi. The filamentous-fungal specific clusters included a number of proteins that are involved in cytoskeletal rearrangements (dedicator of cytokinesis protein, integrin beta-1-binding protein, dynactin p62 family, dynein light intermediate chain 2), it seems likely that these are required for the complex morphological changes that filamentous fungi undergo during their lifecycle and the production of differentiated cells, such as spores, fruiting bodies and infection structures. The results also suggest that filamentous fungal species make a greater use of lipids as signalling molecules than <span class="blue">yeast</span> species. For example, the occurrence of filamentous fungal-specific clusters representing two groups of lysophospholipases, as well as ceramidases that are involved in sphingolipid signalling [35] and linoleate diol synthases that can catalyse the formation of leukotrienes [36]. Interestingly, one of the products of linoleate diol synthase has been shown to be a sporulation hormone in <span class="yellow">Aspergillus nidulans</span> [37]. There is also a cluster that represents homologues of a novel <span class="yellow">human</span> gene (LRP16) that acts downstream of a steroid receptor and promotes cell proliferation [38]. Two clusters of filamentous fungal-specific proteins represent enzymes involved in molypterin biosynthesis (MCL2420, MCL2581). Molypterin is a molybdenum-containing co-factor for nitrate reductase, an enzyme that is known to be absent from the species of <span class="blue">yeast</span> used in this study [39]. Both these clusters are also found in oomycetes. There are other clusters representing proteins important for activities specific to filamentous fungi, such as homologues of Pro11 (striatin) which regulates fruiting body formation in <span class="yellow">Sordaria macrospora</span> [40], the vegetatible incompatibility protein HET-E-1, which prevents the formation of heterokaryons between incompatible fungal strains in <span class="yellow">Podospora anserina</span> [41], anucleate primary sterigmata protein A from <span class="yellow">Aspergillus nidulans</span>, which is essential for nuclear migration and conidiophore development [42] and cytochrome P450 and polyketide synthase-encoding genes, both of which are involved in a number of secondary metabolic pathways including toxin biosynthesis [43].<br><br>Pathogenicity-associated gene functions in fungi
As the selected set of fungi includes both saprotrophic and pathogenic species, this allows us to compare the gene inventories of phytopathogenic and closely related non-pathogenic fungi to look for genes that are unique to phytopathogens. Analysis of MCL clusters showed that there were no clusters that contained proteins from all species of fungal phytopathogen in e-Fungi (namely <span class="yellow">B. cinerea</span>, <span class="yellow">Eremothecium gossypii</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> and <span class="yellow">U. maydis</span>) but did not contain proteins from non-pathogenic species. There were, however, four clusters that were exclusive to filamentous ascomycete phytopathogens (namely <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> as shown in Table 3). Significantly, none of the members of these clusters had homology to any known proteins or contained motifs from the Pfam database [44], so we were unable to predict their function, although two of the clusters (MCL4854 and MCL8229) consisted entirely of proteins that were predicted to be secreted. Taken together, the observations indicate that a battery of completely novel secreted proteins may be associated with ascomycete fungal pathogens.
Pathogenicity factors have been defined as genes that are essential for successful completion of the pathogen lifecycle but dispensable for saprophytic growth [4]. This is an experimental definition based on whether null mutations of a given gene reduce the virulence of the pathogen on its host. We wished to ascertain whether homologues of previously characterised and experimentally-validated pathogenicity factors were limited to the genomes of pathogenic species. A search was therefore made for pathogenicity factors that have been identified experimentally for the species of phytopathogens represented in e-Fungi using PHI-base, the plant-host interaction database [45]. The matching locus was identified for each pathogenicity factor in the corresponding genome sequence by comparing a published protein sequence with sets of predicted proteins for each genome using BLASTP. This produced a list of 105 pathogenicity factors, although corresponding loci could not be found in genome sequences for all the published genes (see Table S5). MCL clusters containing these proteins were identified (76 unique clusters) and the species distribution of members of these clusters analysed. In total, 29 of the MCL clusters contained pathogenicity factors with members from at least 34 of the 36 species represented in e-Fungi (Table 4). Not surprisingly, many of these clusters contain conserved components of signalling pathways such as protein kinases, adenylate cyclases, G-proteins and cell cycle regulators. Cellular morphogenesis is known to be important for infection of the host plant by many phytopathogens, for example, in appressorium formation in <span class="yellow">Magnaporthe grisea</span> [46] or the switch in the growth form of <span class="yellow">Ustilago maydis</span> from <span class="blue">yeast</span>-like growth to filamentous invasive growth [47]. Links between successful plant infection and cell cycle control have also been demonstrated [48]. It seems likely that conserved signalling pathways that control activities, such as mating and morphogenesis in all fungi, have evolved to control processes essential for pathogencity in phytopathogens. Other conserved pathogenicity factors encode enzymes of metabolic pathways that are present in nearly all fungi, but seem to be important for the life cycle of particular pathogenic species, for example, enzymes involved in beta-oxidation of fatty acids, the glyoxylate shunt, amino acid metabolism and the utilisation of stored sugars. When considered together, this may indicate that nutritional conditions which fungi encounter when invading host plant tissue require mobilisation of stored lipids prior to nutrition being extracted from the host plant. Seventeen of the MCL clusters containing pathogenicity factors were specific to filamentous ascomycetes (Table 5). These include a number of enzymes involved in secondary metabolism, such as those involved in the synthesis of the fungal toxin trichothecene in <span class="yellow">G. zeae</span> [43] and those involved in melanin biosynthesis [49], as well as structural proteins, some of which are components of differentiated cell types not seen in yeasts, for example, hydrophobins which are components of aerial structures such as fruiting bodies [50] but are also involved in pathogenicity [16]. There also seems to be a number of filamentous ascomycete specific receptor proteins (transducin beta-subunit, G-protein coupled receptor, tetraspanins) that have evolved in pathogens to be used in sensing environmental cues that are essential for successful infection of the host [51]. The Woronin body is a structure found only in filamentous ascomycetes, and has been shown to be essential for pathogenicity in <span class="yellow">M. grisea</span> [52]. A major constituent of the woronin body, encoded by MVP1, is a pathogenicity factor for <span class="yellow">M. grisea</span>, but also has homologues in nearly all species of filamentous ascomycetes. Two proteins that were initially discovered as being highly expressed in the appressoria of <span class="yellow">M. grisea</span> and essential for pathogenicity (Mas1 and Mas3) [53] also have homologues in a number of species of filamentous fungi (Table 5). Thus, many innovations that have allowed filamentous ascomycetes to have a more complex morphology than unicellular yeasts have also evolved to be essential for plant infection by phytopathogenic species. Interestingly, none of the MCL clusters containing known pathogenicity factors contained members only from phytopathogenic fungi, apart from those that were restricted to just one species. These are therefore likely to represent highly-specialised proteins that have evolved for the specific lifecycle of just one species of phytopathogen, for example the Pwl proteins involved in determining host range of different strains of <span class="yellow">M. grisea</span> [54]. Two of the proteins specific to <span class="yellow">M. grisea</span>, the metallothionein Mmt1 [55] and the hydrophobin Mpg1 [56] are small polypeptides and are members of highly divergent gene families, other members of which do not cluster together using BLASTP.<br><br>Comparative analysis of plant-pathogenic and saprotrophic filamentous ascomycetes
Based on the analysis reported, it is likely that in general there are a large number of differences in gene inventories between filamentous and <span class="blue">yeast</span>-like fungi. Therefore, in order to compare the genomes of phytopathogens and saprotrophs, we focused on filamentous ascomycetes in order to resolve in greater detail the distinct differences in gene sets between these two ecologically separate groups of fungi. In this way differences due to phylogeny between the species would be minimised. We compared the gene inventories of the phytopathogens <span class="yellow">B. cinerea</span>, <span class="yellow">G. zeae</span>, <span class="yellow">M. grisea</span>, <span class="yellow">S. sclerotiorum</span>, <span class="yellow">S. nodorum</span> with the non-pathogens <span class="yellow">Aspergillus nidulans</span>, <span class="yellow">Chaetomium globosum</span>, <span class="yellow">Neurospora crassa</span> and <span class="yellow">Trichoderma reesei</span>. Phylogenetic analysis suggests that the phytopathogenic species do not form a separate clade from the pathogenic species (Figure 1), [3] and we assumed that differences in gene inventory should therefore reflect lifestyle rather than evolutionary distance. In order for such a comparison to be considered valid, the completeness and quality of the fungal genome sequences used should, however, also be comparable. Table S6 summarises the available data about genome sequence coverage, genome size and the number of predicted proteins for each species. This shows that the genome coverage is greater than 5x and the number of predicted proteins in the range of 10,000–16,000 for all genomes used, suggesting a high level of equivalence between species with regard to sequence quality. From our work it seems unlikely that there are pathogenicity factors conserved in, and specific to, all species of phytopathogen. It may, for instance, be the case that differences in the gene inventories are due to the expansion of certain gene families in the genomes of phytopathogenic species associated with functions necessary for pathogenesis. To define protein families, we used the Pfam database which contains protein family models based on Hidden Markov Models [44], [57]. Sets of predicted proteins for each fungal species in e-Fungi were analysed for the occurrence of Pfam motifs and the number of proteins containing each domain across fungal species ascertained. The sets of predicted protein sequences used in this study have been automatically predicted as part of each individual genome project and are likely to contain a number of artefactual sequences. The use of Pfam motifs to define gene families in this study reduces the likelihood of such sequences affecting the data, since Pfam motifs are based on multiple sequence alignments of well-studied proteins.
A small number of Pfam motifs were not found in the proteomes of the filamentous ascomycete non-pathogens, but were found in the proteomes of at least three species of filamentous ascomycete phytopathogens (Table 6). These include the Cas1p-like motif (PF07779), found in 4 species of phytopathogen, including five copies in <span class="yellow">G. zeae</span>, and the <span class="yellow">Yeast</span> cell wall synthesis protein KRE9/KNH1 motif (PF05390), which was found in three species of phytopathogen. Cas1p is a membrane protein necessary for the O-acetylation of the capsular polysaccharide of the basidiomycete animal pathogen <span class="yellow">Cryptococcus neoformans</span> [58]. KRE9 and KNH1 are involved in the synthesis of cell surface polysaccharides in <span class="yellow">S. cerevisiae</span> [59]. Taken together this suggests that synthesis of cell surface polysaccharides is important for phytopathogens, perhaps helping to shroud the fungus from plant defences. The function of the YDG/SRA domain motif (PF02182) is unknown, but is found in a novel <span class="yellow">mouse</span> cell proliferation protein Np95, in which the domain is important both for the interaction with histones and for chromatin binding in vivo [60]. As well as domains of unknown function, the list of phytopathogen-specific Pfam motifs includes Allophanate hydrolase (PF02682) which is found in an enzyme involved in the ATP-dependent urea degradation pathway [61], a peptidase motif, an opioid growth receptor motif (PF04664) and Mnd1 (PF03962), which is involved in recombination and meiotic nuclear division [62].
To detect potential gene family expansion, we decided to identify Pfam motifs that were present in both phytopathogenic and non-pathogenic species of filamentous ascomycetes, but that were more common in the genomes of the former. The Pfam motifs were ranked on the ratio of the mean number of proteins containing each motif in phytopathogens, when compared to non-pathogens (Table 7). The tables only show ratios of greater than or equal to 2.5. Pfam motifs that were more common in the proteomes of pathogens, include some found in enzymes involved in secondary metabolic pathways. These include novel enzymes that have only previously been studied in non-fungal species, such as the chalcone synthases; type III polyketide synthases involved in the biosynthesis of flavonoids in plants [63] and lipoxygenases; components of metabolic pathways resulting in the synthesis of physiologically-active compounds such as eicosanoids in mammals [64] and jasmonic acid in plants [65] as well as antibiotic synthesis monooxygenases. It seems likely that secondary metabolism is essential in phytopathogenic species for the synthesis of mycotoxins, antibiotics, siderophores and pigments [66], but it may also offer fungal pathogens a distinct alternative means of perturbing host metabolism, cell signalling or plant defence, in contrast to bacterial pathogens that rely on protein secretion to achieve this. There also seems to be number of protease and peptidase domains that are more common in the genomes of phytopathogens as well as domains from two classes of cell-wall degrading enzymes: namely cutinase (PF01083) and Glycosyl hydrolase family 53 (PF07745) which is found in arabinogalactan endo-1,4-beta-galactosidases that hydrolyze the galactan side chains that form part of the complex carbohydrate structure of pectin [67]. Two other domains found in enzymes involved in pectin degradation, pectinesterase (PF01095) and Glycosyl hydrolases family 28 (PF00295) are both more than twice as common in the genomes of phytopathogens than saprotrophs. In contrast, domains found in cellulases have fairly equal distribution between the proteomes of phytopathogens and non-pathogens (data not shown). Therefore, for phytopathogens the most essential enzymes for pathogenesis may well be those that allow the fungus to penetrate the protective cutin layer of the plant epidermis and disrupt the pectin matrix of the plant cell wall in which cellulose fibrils are embedded. Pectin-degrading enzymes have already been shown to be pathogenicity factors in a number of fungi [68]. NPP1 motifs are characteristic of a group of proteins called NLPs (Nep1-like proteins) that trigger defence responses, necrosis and cell death in plants and may act as virulence factors [69]. The NLPs are more common in the genomes of phytopathogenic, when compared to non-pathogenic ascomycetes, but are even more numerous in the proteomes of the oomycetes (64 proteins in <span class="yellow">Phytophthora ramorum</span> and 75 in <span class="yellow">Phytophthora sojae</span>). Proteins containing the Chitin recognition protein domain (PF00187) are also very common in the proteomes of phytopathogens (18 in <span class="yellow">M. grisea</span> and 16 in <span class="yellow">S. nodorum</span>). A role for chitin-binding proteins has been proposed in protecting the fungal cell wall from chitinases produced by host plants [70]. There are also two other Pfam motifs, which are more common in the proteomes of phytopathogens, that are found in enzymes involved in the catabolism of toxic compounds, namely arylesterase (PF01731) and EthD protein (PF07110) which breakdown organophosphorus esters [71] and ethyl tert-butyl ether [72], respectively.<br><br>Comparative secretome analysis of phytopathogenic and saprotrophic filamentous ascomycetes
Studies in bacterial pathogens and oomycetes have shown that a range of secreted proteins known as effectors are important for establishing infection of the host plant [73], [74]. These secreted proteins may disable plant defences and subvert cellular processes to suit the needs of invading pathogens. Therefore, we decided also to compare gene family size in the secretomes of phytopathogens and non-pathogens. There are a number of programs available that predict whether a protein is likely to be secreted, although the predictions they give significantly differ from each other. Therefore we defined the secretome of each fungal species based on those proteins that are predicted to be secreted by two different programs: SignalP 3.0 [75] and WoLFPSORT [76]. The size of each secretome is summarised in Figure 2. Even when using two programs, the sizes of predicted secretomes can vary greatly. For example, a similar analysis for <span class="yellow">M. grisea</span> using SignalP and ProtComp (www.Softberry.com) predicted only 739 secreted proteins (out of a proteome of 11,109) compared to our prediction of 1,546 secreted proteins (out of a proteome of 12,841) [22]. The size of the secretomes for each species varied from 5%–12% of the total proteome. Overall, the size of the secretomes from phytopathogens did not differ greatly from that of non-pathogens.
Table 8 shows a list of Pfam motifs, not found in the secretomes of non-pathogenic filamentous ascomycetes, that were present in at least three phytopathogenic fungal species. The Isochorismatase motif (PF00857) was found in the secretomes of all five species of phytopathogen. Isochorismatase catalyses the conversion of isochorismate to 2,3-dihydroxybenzoate and pyruvate. It has been implicated in the synthesis of the anti-microbial compound phenazine by <span class="yellow">Pseudomonas aeruginosa</span> [77] and the siderophore, enterobactin, by <span class="yellow">Escherichia coli</span> [78]. The isochorismatase motif is also found in a number of hydrolases, such as nicotinamidase that converts nicotinamide to nicotinic acid [79]. Members of this family are found in all filamentous ascomycetes, but interestingly they are only secreted in phytopathogens. Salicylic acid is synthesised in plants in response to pathogen attack and mediates plant defences. As isochorismate is a precursor of salicyclic acid [80], it may be worth speculating that isochorismatases secreted by fungi could act to reduce salicylic acid accumulation in response to pathogen attack and thus inhibit plant defence responses. The secreted isochorismatases (apart from one of the proteins from <span class="yellow">S. nodorum</span>) all show sequence similarity to ycaC from <span class="yellow">E. coli</span>, an octameric hydrolase of unknown function [81]. Pfam motifs found in the secretomes of at least three species of phytopathogens, but not in any of the non-pathogens also include those found in enzymes potentially involved in detoxification, such as arylesterase and amidohydrolase, and also beta-ketoacyl synthase, which catalyses the condensation of malonyl-ACP with a growing fatty acid chain and is found as a component of a number of enzyme systems, including fatty acid synthases and polyketide synthases [82], [83].
Table 9 shows a list of Pfam motifs that are more common in the secretomes of phytopathogens as compared to saprotrophs. These include a number of secreted proteases, transcription factors and components of signal transduction pathways. The Kelch domain (PF01344) shows the most striking difference in distribution between phytopathogenic and non-pathogenic genomes. This 50-residue domain is found in a number of actin-binding proteins [84], as well as enzymes such as galactose oxidase and neuraminidase. The putative function of each secreted Kelch domain-containing protein was ascertained by performing a BLAST search against the NCBI non-redundant protein database (Table 10). A number of these seem to be galactose oxidases, enzymes which catalyse the oxidation of a range of primary alcohols, including galactose, to the corresponding aldehyde with the concomitant reduction of oxygen to hydrogen peroxide (H2O2) [85]. Galactose oxidase shares a copper radical oxidase motif with the hydrogen peroxide-generating glyoxal oxidases involved in lignin-degradation in <span class="yellow">Phanerochaete chrysosporium</span> [86]. H2O2-producing copper oxidases have been shown to have roles in morphogenesis, in the <span class="yellow">corn</span>-smut fungus <span class="yellow">Ustilago maydis</span> for example, a glyoxal oxidase is required for filamentous growth and pathogenicity [87] and a galactose oxidase is involved in fruiting body formation in the gram-negative bacterium <span class="yellow">Stigmatella aurantiaca</span> [88]. Interestingly, the list of Pfam motifs more common in the secretomes of phytopathogens also includes those found in copper amine oxidases, H2O2-generating enzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes [89] and peroxidases, haem-containing enzymes that use hydrogen peroxide as the electron acceptor to catalyse a number of oxidative reactions. Secreted fungal peroxidases include enzymes involved in lignin breakdown by the white rot fungus <span class="yellow">Phanerochaete chrysosporium</span> [90], but in plants they generate reactive oxygen species and are involved in defence responses and growth induction [91]. A number of other secreted Kelch domain-containing proteins have similarity to proteins of unknown function from species of the bacterial phytopathogen Xanthomonas. Many Kelch domain-containing proteins are involved in cytoskeletal rearrangement and cell morphology [92], [93]. It may be worth speculating that secreted Kelch domain-containing proteins could act as effectors, causing changes in the arrangement of the cytoskeleton of infected plants to aid the proliferation of fungal hyphae. It has recently been shown, for example, that <span class="yellow">M. grisea</span> co-opts plasmodesmata to move from cell to cell in infected <span class="yellow">rice</span> leaves [94] and would therefore need to peturb cytoskeletal organisation in <span class="yellow">rice</span> epidermal cells. There are other Pfam domains that are more common in the secretomes of phytopathogens that may potentially be found in effectors such as the PAN domain (PF00024), that mediates protein-protein and protein-carbohydrate interactions [95] and the F5/8 type C domain (PF00754), found in the discoidin family of proteins involved in cell-adhesion or developmental processes [96].<br><br>
Discussion
One of the most fundamental aims in plant pathology research is to define precisely the difference between pathogenic and non-pathogenic microorganisms. The answer cannot be one of simple phylogeny, because phytopathogenic species are found in all taxonomic divisions of fungi and are often closely related to non-pathogenic species [3]. Before the availability of genomic sequences and high throughput approaches to study gene function [20], research was concentrated on the search for single pathogenicity factors; genes that are dispensable for saprophytic growth but essential for successful infection of the host plant [4], [97]. However, rather than encoding novel proteins found only in phytopathogens, the majority of pathogenicity factors discovered in this way have been found to be involved in signalling cascades and metabolic pathways and hence are conserved in most species of fungi [5]. Components of signalling cascades that in the budding <span class="blue">yeast</span> <span class="yellow">S. cerevisiae</span> are responsible for responses to pheromones, nutritional starvation and osmotic stress [9] have in many cases evolved different roles in the life cycle of pathogens, such as controlling appressorium formation, dimorphism and growth [10]. Although the central components of signalling are conserved between phytopathogens and <span class="yellow">S. cerevisiae</span>, the receptors are often different, reflecting the different environmental cues to which the pathogen needs to respond [11], [98].
Analysis of all available genome sequences from a wider range of fungal species has for the first time allowed us to address the differences between phytopathogens and non-pathogens at a whole genome level. For this purpose, the e-Fungi data warehouse provides a means to interrogate the vast amounts of genomic and functional data available in a simple integrated manner [26]. Previous research, in which EST datasets were compared with genomic sequences, suggested that the expressed gene inventories of phytopathogenic species were not significantly more similar to one another than to those of saprotrophic filamentous fungi [99]. We clustered sets of predicted proteins from 36 different species of fungi and oomycetes into groups of potential orthologues and the species distribution of members of each cluster was ascertained. There were no clusters that were completely specific to phytopathogenic species across both fungi and oomycetes, suggesting that the presence of novel, universal pathogenicity factors in the genomes of phytopathogens is unlikely. This was confirmed by looking at clusters containing empirically defined pathogenicity factors, where homologues of many of these were found in all species studied and none were conserved in the genomes only of phytopathogens. A small number were only found in a single species of fungus and probably represented proteins that are highly specialised for a particular role in a specific pathogenic species, for example in host-plant recognition [54]. Previous research also suggested that the gene inventories of filamentous fungi were more similar to each other than to those of unicellular yeasts [99]. Analysis of the clusters of similar proteins show some clusters that are found in all species of filamentous fungi (including ascomycetes, basidiomycetes and zygomycetes) but are not present in the genomes of yeasts, consistent with the original conclusion. These contain a number of proteins that are likely to be involved in morphological changes associated with the more complex filamentous lifestyle, as well those involved in secondary metabolism and signalling cascades that are not found in yeasts. In particular, our results suggest that filamentous fungi use a wider variety of lipid molecules for the purpose of signalling. Some of these may act as pheromones, or hormones– chemical messengers diffusing from one cell to another to elicit a physiological or developmental response [37]. A number of these innovations to the filamentous lifestyle may serve important roles in pathogenesis as well, because homologues of a number of pathogenicity factors are found only in filamentous ascomycetes. The distribution of filamentous fungi-specific proteins, such as involved in those cytoskeletal rearrangements and fruiting body formation, throughout the fungal kingdom (and in some cases in oomycetes as well), suggests that the last common ancestral fungus may well have been multi-cellular and the evolution of uni-cellular fungi was likely associated with massive gene loss. For example, it has been shown that early in ascomycete evolution there was a proliferation of subtilase-type protease-encoding genes that have been retained in some filamentous ascomycete lineages, but lost in the <span class="blue">yeast</span> lineage [100].
It has previously been speculated that the evolution of phytopathogenesis was associated with the expansion of certain gene families [1]. Duplication of an ancestral gene, followed by mutation allows members of the family to take on new functions [101]. For example, genomes of the filamentous ascomycetes studied here have between 40 and 140 cytochrome P450-encoding genes (data not shown) that are involved in toxin biosynthesis, lipid metabolism, alkane assimilation and detoxification [102] and which probably arose via gene duplication and functional diversification. In contrast, the genome of the budding <span class="blue">yeast</span> <span class="yellow">S. cerevisiae</span> has only three cytochrome P450-encoding enzymes. We have shown here that there are likely to be large differences in the gene inventories of filamentous fungi compared to unicellular yeasts.
To study the differences between phytopathogenic and saprophytic fungi, we concentrated on the filamentous ascomycetes where there are a number of phytopathogenic species genomes have been sequenced along with closely related non-pathogens. Protein families were defined using Pfam motifs [57] and the predicted protein sets for each species analysed in order to identify domains that were specific to or more common in the genomes of phytopathogens. Not surprisingly, many of the protein families we identified are likely to be associated with pathogenic processes such as plant cell wall degradation, toxin biosynthesis, formation of reactive oxygen species and detoxification [5]. Studies of bacterial phytopathogens have shown the importance of effectors, secreted proteins that disable plant defences and subvert metabolic and morphological processes for the benefit of the invading pathogen and which require delivery via a type III secretion system that are often deployed during pathogenesis [73]. Bacterial type III secreted effectors (T3SEs) have been shown to target salicyclic acid and abscisic acid-dependent defences, host vesicle trafficking, transcription and RNA metabolism, and several components of the plant defence signalling networks [103]. Very recently, potential effector-encoding genes have been identified in the genomes of several species of oomycete pathogens and are defined by the presence of a conserved RXLR-EER motif downstream of the signal peptide sequence [74]. The RXLR-EER motif is necessary for delivery of effector proteins into host plant cells and is therefore critical to their biological activity [74].
To identify potential fungal effectors, we compared Pfam motif frequency between the secretomes of phytopathogens and non-pathogens. This analysis identified potential effector-encoding genes, including secreted proteases, transcription factors and proteins that may be involved in cytoskeletal rearrangements (such as Kelch-domain containing proteins) and protein-protein interactions, as well as a group of pathogen-specific secreted isochorisimatases that potentially could suppress salicyclic acid-dependent host plant defences. Bacterial T3SEs are injected directly into the host cytoplasm via the type III secretion injection apparatus [73]. In contrast, the potential fungal effectors identified in this study appear to be secreted by the normal cellular secretory pathway via the endoplasmic reticulum and the mechanism by which fungal effectors might be taken up by plant cells and enter into the host cytoplasm is currently unknown.
Although the evolution of phytopathogenicity is likely to have happened several times and the lifestyles of these fungi are diverse, a comparison of gene inventories of a number of species using a powerful resource, such as e-Fungi, has allowed us to pinpoint new gene families that may serve important roles in the virulence of phytopathogens, allowing their selection for gene functional studies, that are currently in progress. The analyses deployed here may also offer a blueprint for the types of larger, more comprehensive studies that will be necessary to interpret the large flow of genetic data that will result from next generation DNA sequence analysis utilizing both a much wider variety of fungal pathogen species and also large sets of individual isolates of existing species.<br><br>Materials and Methods
Clustering of sequences
Sets of predicted proteins were downloaded for each of the 36 genomes from respective sequencing project websites (Table 1). Proteins less than 40 amino acids in length were not included in this analysis. Proteins were clustered using “all against all” BLASTP [104] followed by Markov Chain Clustering (MCL) [27] with 2.5 as a moderate inflation value and 10−10 as an E-value cut-off, as described previously [28], [29]. Clusters were annotated based on best hit against Swiss-Prot protein database [105] of members of that cluster (e-value <10−20 using BLASTP), or Pfam motifs contained in proteins from the cluster in the absence of Swiss-Prot hits.<br><br>Identification of Pfam motifs
The Pfam-A library from release 18.0 of the Pfam database was downloaded from the Pfam website (http://www.sanger.ac.uk/Software/Pfam/). This library contains 7973 protein models constructed from manually curated multiple alignments and covers 75% of proteins in UniProt [44], [57]. This library was used to analyse the sequences of predicted proteins for all 36 fungal genomes to identify the Pfam motifs that each protein contains. The analysis was performed using the “pfam_scan” perl script (version 0.5) downloaded from the Pfam website and HMMER software (downloaded from http://hmmer.wustl.edu/). Default thresholds were used, which are hand-curated for every family and designed to minimise false positives [44].<br><br>Identification of secreted proteins
The N-terminal sequence of each predicted protein from the 36 fungal genomes used in this study was analysed for the presence of a signal peptide using SignalP 3.0 [75] and sub-cellular localisation was predicted using WoLF PSORT [76]. Both these programs were installed locally. SignalP 3.0 uses two different algorithms to identify signal sequences. The secretome for each fungal species was defined as containing those proteins that were predicted have a signal peptide by both prediction algorithms from SignalP 3.0 and also predicted to be extracellular by WoLF PSORT.<br><br>Data analysis
All the data produced, as described above, was stored in the e-Fungi data warehouse [26] from which it can be accessed via a web-interface (http://www.e-fungi.org.uk/). Analyses described in this study were performed using the e-Fungi database.<br><br>
Supporting Information<br><br>
<h3>pmcA2658668</h3>A novel DNA-binding protein modulating methicillin resistance in <span class="yellow">Staphylococcus aureus</span>
Abstract
Background
Methicillin resistance in <span class="yellow">Staphylococcus aureus</span> is conferred by the mecA-encoded penicillin-binding protein PBP2a. Additional genomic factors are also known to influence resistance levels in strain specific ways, although little is known about their contribution to resistance phenotypes in clinical isolates. Here we searched for novel proteins binding to the mec operator, in an attempt to identify new factor(s) controlling methicillin resistance phenotypes.<br><br>Results
Analysis of proteins binding to a DNA fragment containing the mec operator region identified a novel, putative helix-turn-helix DNA-binding protein, SA1665. Nonpolar deletion of SA1665, in heterogeneously methicillin resistant <span class="yellow">S. aureus</span> (<span class="yellow">MRSA</span>) of different genetic backgrounds, increased methicillin resistance levels in a strain dependent manner. This phenotype could be fully complemented by reintroducing SA1665 in trans. Northern and Western blot analyses, however, revealed that SA1665 had no visible influence on mecA transcription or amounts of PBP2a produced.<br><br>Conclusion
SA1665 is a new chromosomal factor which influences methicillin resistance in <span class="yellow">MRSA</span>. Although SA1665 bound to the mecA promoter region, it had no apparent influence on mecA transcription or translation, suggesting that this predicted DNA-binding protein modulates resistance indirectly, most likely through the control of other genomic factors which contribute to resistance.<br><br><br><br>Background
Methicillin resistant <span class="yellow">S. aureus</span> (<span class="yellow">MRSA</span>) are an ever increasing threat, both in clinical settings and more recently as an emerging community acquired pathogen. Their invasiveness and pathogenesis relies on a variable arsenal of virulence factors, paired with resistance to virtually all β-lactams and their derivatives. Their ability to rapidly generate resistance to other unrelated classes of antibiotics, or to take up additional resistance determinants, severely hampers therapy and eradication.
In <span class="yellow">S. aureus</span>, methicillin resistance is conferred by an acquired, β-lactam-insensitive penicillin-binding protein (PBP), PBP2a [1-4]. PBP2a is encoded by mecA, which is divergently transcribed from its cognate regulators, mecR1 (sensor/signal transducer) and mecI (repressor). If mecR1-mecI are absent or truncated, transcriptional control of mecA is taken over by the structurally similar blaZ (penicillinase) regulatory elements blaR1/blaI, if present. In the absence of both regulatory loci, mecA is constitutively transcribed [5,6]. In the presence of β-lactams, the transmembrane sensor/signal transducers BlaR1/MecR1, undergo a conformational change, followed by autoproteolytic cleavage of the n-terminal cytoplasmic domain, leading to the activation of the cytoplasmic peptidase and subsequent dissociation of the repressor due to proteolytic degradation [7-9]. However, the signal transduction cascade of this regulatory system has still not been completely elucidated.
Oxacillin resistance levels conferred by mecA are strain specific and can vary greatly, with oxacillin minimal inhibitory concentrations (MICs) of different strains ranging from phenotypically susceptible levels, as low as 1 μg/ml up to extremely high values of > 500 μg/ml. Methicillin resistance is also generally expressed heterogeneously. Heterogeneously resistant <span class="yellow">MRSA</span>, when exposed to β-lactam antibiotics, segregate highly resistant subpopulations, which are much more resistant than the majority of the cells [10]. The frequency of highly resistant subclones generated is often well above the spontaneous mutation frequency, and once selected high level resistance often remains stable, even in the absence of selective pressure. There is currently no satisfactory genetic model which explains how these higher resistance levels are triggered or selected and exactly what factors are functionally responsible for the increased resistance in clinical isolates. Methicillin resistance levels are known to not directly correlate with mecA transcription or levels of PBP2a produced [11,12]. However, resistance levels can be manipulated by environmental conditions, such as temperature, pH, osmolarity, and medium composition [13,14].
It has been shown experimentally, that in addition to mecA, methicillin resistance depends on the correct interplay of a multitude of genomic factors, termed fem/aux factors, including genes involved in peptidoglycan precursor formation, composition and turnover; teichoic acid synthesis; and genes of unknown or poorly characterised functions [15-18]. In addition to structural genes, many regulatory loci have also been shown to influence resistance levels, including global regulators of virulence factor production such as the quorum sensing agr system, the staphylococcal accessory regulator SarA and the alternate sigma factor σB [19,20]; regulators of metabolism, such as the catabolite control protein A (CcpA) [21]; and the VraSR two-component sensor transducer, which induces the cell wall stress stimulon in response to cell wall active antibiotic challenge [22].
The vast MIC differences between <span class="yellow">MRSA</span> strains, the population heterogeneity within single strains and the dependence of resistance levels on external factors are reflected in these many structural genes and global regulators, which can influence resistance levels.
While typically considered nosocomial pathogens, new faster growing and apparently more virulent <span class="yellow">MRSA</span> have begun spreading in the community. Interestingly, these emerging strains often express very low methicillin resistance, e.g. the <span class="yellow">MRSA</span> clone spreading amongst intravenous drug users in the Zurich area, which has an in vitro doubling time of 25 min, but oxacillin MICs of only 0.5 to 4 μg/ml [23]. This particular clone's low-level resistance is partially due to a promoter mutation, leading to tight repression of mecA, but resistance levels appear to be mainly restricted by unknown factors within its genomic background [12].
To identify potential factors involved in mecA regulation or methicillin resistance levels in such an extremely low level resistant <span class="yellow">MRSA</span>, we performed DNA-binding protein purification assays, using the mecA operator region as bait. A novel, uncharacterized protein, SA1665, was found to bind to this DNA fragment, and shown to increase methicillin resistance levels when deleted.<br><br>Results
Identification of SA1665
<span class="yellow">MRSA</span> strain CHE482 is the type strain for the so-called "drug clone" spreading amongst intravenous drug users in the Zurich area [12,23]. This strain carries mecA and expresses PBP2a, but appears phenotypically methicillin susceptible by conventional phenotypic tests. However, like most other low-level resistant <span class="yellow">MRSA</span>, it can segregate a small proportion of higher resistant subclones in the presence of β-lactams. We hypothesized that regulation of methicillin resistance in such low-level resistant clonal lineages may differ qualitatively from classical heterogeneously- or highly-resistant <span class="yellow">MRSA</span>.
A DNA-binding protein purification assay was performed to identify new potential factors involved in the regulation of mecA/PBP2a. The mecA/mecR1 intergenic DNA region, including the 5' 9 bp of mecR1 and the first 52 bp of mecA, was used as bait against crude protein extract from strain CHE482. Proteins binding to this DNA fragment were analysed by SDS-PAGE. Even though CHE482 contained BlaI, which is known to bind to the mec operator, this band could not be identified on gels due to co-migrating, non-specific bands the same size as BlaI (14.9 KDa) that bound to both the DNA-coated and uncoated control beads. The most prominent protein band of ~16–20 kDa, isolated from DNA-labelled but not from control beads, was identified as the hypothetical protein SA1665 (N315 genome annotation [BA000018]) (Figure 1A). SA1665 encodes a predicted 17-kDa protein with an n-terminal helix-turn-helix (HTH) motif characteristic of DNA-binding transcriptional regulators. The amino acid sequence of SA1665 showed 100% identity amongst <span class="yellow">S. aureus</span> database sequences and 97–98% identity amongst other staphylococci, including <span class="yellow">S. haemolyticus</span>, <span class="yellow">S. epidermidis</span> and <span class="yellow">S. saprophyticus</span>, indicating that SA1665 is highly conserved. Conversely, there were no orfs highly similar to SA1665 found in other bacterial species, with the most similar sequences found in <span class="yellow">Bacillus licheniformis</span> DSM13 and <span class="yellow">Desulfitobacterium hafniense Y51</span>, which shared only 64% and 59% similarity, respectively.<br><br>Electro mobility shift assays (EMSA)
EMSA was used to confirm binding of SA1665 to the mec operator region. Crude protein extracts of <span class="yellow">E. coli strain BL21</span>, carrying the empty plasmid (pET28nHis6) or pME20 (pET28nHis6-SA1665) which expressed nHis6-SA1665 upon induction with IPTG, were incubated with the 161-bp biotinylated-DNA fragment previously used as bait in the DNA-binding protein assay. A band shift was observed with extracts from the strain expressing recombinant nHis6-SA1665 but not from the control strain carrying the empty plasmid. Several bands resulted from the shift, which is most likely due to protein oligomerisation (Figure 2A). The specificity of the gel shift was also demonstrated by the addition of increasing concentrations of purified nHis6-SA1665 protein to the biotinylated-DNA fragment (Figure 2B). Band-shift of the biotinylated DNA was inhibited in the presence of specific competitor DNA but not by the presence of the non-specific competitor DNA, confirming that nHis6-SA1665 had a specific binding affinity for the 161-bp DNA fragment.<br><br>Effect of SA1665 deletion on β-lactam resistance
To analyse the effect of SA1665 inactivation on methicillin resistance, nonpolar markerless deletions of SA1665 (Figure 1B) were constructed in a selection of clinical <span class="yellow">MRSA</span> isolates, which varied in their genetic background, SCCmec type, and mecA regulation [24]. Strain CHE482, belongs to clonal complex CC45 and sequence type ST45, and contains a novel SCCmec (SCCmecN1 [23]); while strains ZH37 (CC45/ST45) and ZH73 (CC22/ST22) contain type IV SCCmecs. All three of these strains have truncated mecI/mecR1 regulatory loci but intact BlaI/BlaR1 loci controlling mecA expression. Strain ZH44 (CCT8/ST8) contained a type A mec complex (mecI-mecR1-mecA) within a type II SCCmec, and had no β-lactamase locus; so mecA was only under the control of its cognate regulators MecI/MecR1.
Deletion of SA1665 increased oxacillin resistance in all mutants compared to their corresponding parent strains, as demonstrated on oxacillin gradient plates (Figure 3A); with mutants ΔCHE482 and ΔZH37 approximately doubling in resistance and ΔZH44 and ΔZH73 expressing considerably higher resistance. Population analysis resistance profiles of the mutants showed a distinct shift at the top of the curve, indicating that the higher resistance was due to increased basal oxacillin resistance levels (Figure 3B). Strains CHE482/ΔCHE482 and ZH37/ΔZH37 had very similar resistance profiles, despite having different SCCmec elements, suggesting that it was their common clonal background (CC45) that determined their resistance levels and the extent of resistance increase upon SA1665 deletion.
Growth curve analyses showed that deletion of SA1665 slightly reduced the growth rate of all strains tested (Figure 3C). Wild type growth rates were restored upon complementation (data not shown).<br><br>Resistance complementation
Plasmids pME26 and pME27 were constructed for complementation of the deletion mutants. Both plasmids contained the SA1665 orf along with its own promoter and transcriptional terminator. Strains ΔCHE482, ΔZH37, and ΔZH73 were complemented with pME26, and intrinsically kanamycin resistant strain ΔZH44 was complemented with pME27. Wild type-like resistance levels were restored in all mutants by introduction of the complementing plasmids, as shown by gradient plates (Figure 3A).<br><br>Transcriptional analyses
Primer extension, using the 5'-biotinylated primer me97, identified two potential SA1665 transcriptional start sites (TSS), 76-nt and 139-nt upstream of the SA1665 ATG start codon (Figure 4A). Predicted σA promoter consensus -10/-35 box sequences were located upstream of both TSS (Figure 4B). Identical TSS were also identified using the downstream primer me98 (data not shown).
Northern blot analysis was used to investigate SA1665 expression and the influence of SA1665 deletion on mecA and mecR1 transcription. RNA samples taken from different time points over the growth curve of CHE482 showed that SA1665 was expressed strongly in early exponential phase at OD600 nm 0.25 and 0.5, then transcript levels decreased and were almost undetectable in early stationary phase at OD600 nm 4.0 (Figure 5A). In addition to the main transcript of ~0.46 kb, a weaker, larger transcript of ~0.6 kb was also visible, especially at later growth stages. Figure 5B shows the transcriptional behaviour of SA1665 when CHE482 cells were challenged with sub-inhibitory (4 μg/ml) and inhibitory (120 μg/ml) concentrations of cefoxitin. These results showed that low levels of cefoxitin, such as those used to induce mecA/mecR1 transcription, appeared to slightly decrease SA1665 transcription after 30 min exposure, while larger, inhibitory concentrations caused even more significant alterations in the SA1665 transcriptional profile, making it similar to that normally seen in stationary phase growth. These results indicate that transcription of SA1665 may respond in some way to cell wall stress, rather than in direct response to the presence of β-lactams. This observation is based on relatively subtle changes in SA1665 transcription, especially at low concentrations of cefoxitin such as those required for mecA/mecR1 induction. Since deletion of SA1665 has been shown to increase β-lactam resistance, reduced SA1665 transcription in the presence of β-lactams may also provide some protection against β-lactam exposure.
Northerns also showed that, as expected, the SA1665 transcripts were absent from the deletion mutant (Figure 5C), and additional experiments demonstrated that wild type SA1665 transcription patterns were restored by complementation of ΔCHE482 with pME26 (data not shown). The effects of SA1665 deletion on directly up- and down-stream genes were also investigated. Northern blots of the neighbouring genes SA1664, SA1666 and SA1667, showed that expression of all three genes was very weak compared to that of SA1665. A weak transcript of about 3 kb was present in hybridizations probed with orfs SA1665-SA1667. This band decreased in size in the SA1665 mutant when probed with SA1666 and SA1667. One of the transcripts hybridising to the SA1664 probe also decreased in size by ~0.5 kb in the SA1665 mutant, suggesting that SA1665 was present on several transcripts of different lengths, including a high abundance monocistronic transcript and low abundance polycistronic transcripts (Figure 5C). Transcript abundance of both the upstream SA1666-SA1667 operon and the downstream SA1664-specific transcript all appeared to increase slightly in ΔCHE482. The significance of these subtle increases in transcription are unknown, however, polar effects from SA1665 deletion seem unlikely, based on the facts that all genes were still transcribed, their transcription levels all remained extremely low and the transcriptional terminator of SA1665 remained intact in the deletion mutant (Figure 1B).
Expression of mecR1 and mecA were analysed from RNA of uninduced and induced cultures of CHE482 and ΔCHE482. Cells were induced at OD600 nm 0.25 (Figure 5D) and 1.0 (data not shown) with sub-inhibitory concentrations of cefoxitin, to relieve BlaI-repression of mecA. mecR1, although truncated in CHE482, was still transcribed and had the same expression pattern as mecA, as both became derepressed over time and had the highest transcript levels after 30 min of induction. In the mutant ΔCHE482, transcripts of both mecA and mecR1' were unaffected by SA1665 deletion, indicating that SA1665 had no influence on their expression at either OD 0.25 (Figure 5D) or OD 1.0 (data not shown). SA1665 deletion also had no effect on mecA transcription or induction in strains ZH37, ZH44 and ZH73 (data not shown).<br><br>Western blot analysis
Mutants of CHE482 and of ZH44 and ZH73, which had the largest differences in oxacillin resistance levels, were analysed by Western blot analysis to determine if SA1665 affected production of PBP2a from mecA. As shown in Figure 5E, all pairs of wild type and mutant strains had similar amounts of PBP2a present both before and after induction with cefoxitin, indicating that SA1665 deletion did not alter amounts of PBP2a produced. Therefore it seems that SA1665 exerts no direct control over mecA or PBP2a expression.<br><br>
Discussion
Methicillin resistance in <span class="yellow">MRSA</span> is primarily dependent on the presence of the mecA gene, however, resistance levels are generally governed by strain-specific factors including mecA regulatory elements and other chromosomal fem/aux factors which either enhance or repress the expression of resistance. For instance, the very low-level methicillin resistance of the Zurich drug clone CHE482, was shown to be controlled by its genetic background [12] suggesting that it either contained or lacked certain fem/aux factors involved in controlling resistance expression. Many of the currently known fem/aux factors are directly or indirectly involved in cell wall synthesis and turnover, or envelope biogenesis, however there still remain factors of unknown function. Most of the currently known fem/aux factors reduce methicillin resistance levels when inactivated. A few genes, such as lytH, dlt, norG, sarV and cidA increase resistance levels upon inactivation or mutation. All of these genes, except norG, which is an efflux pump regulator, play a role in either autolysis or are important for cell physiology and growth [25-30]. Other genes increase β-lactam resistance upon overexpression, such as hmrA coding for a putative amidohydrolase, hmrB coding for a putative acyl carrier protein [31], or the NorG-controlled abcA multidrug efflux pump [28].
SA1665, a predicted DNA-binding transcriptional regulator, was found to bind to a DNA fragment containing the mecA promoter region. However, although this protein shifted the mecA operator/5' coding sequence, it did not appear to directly control mecA or mecR1 transcription or PBP2a production. Therefore its binding to the mecA region may have no specific regulatory function. Such interactions have been noted before, such as the HTH protein NorG, which was shown to bind specifically to norA, norB and norC promoters, but only transcription of norB was increased when NorG was overexpressed [28]. We have to postulate therefore that SA1665 may modulate β-lactam resistance in a mecA-independent manner, by controlling cellular functions affecting resistance levels. Experiments to determine the SA1665 regulon are ongoing. The impact of deleting SA1665 in <span class="yellow">MRSA</span> was extremely strain specific, underlining the importance of the genetic background in governing the final methicillin resistance levels of <span class="yellow">MRSA</span>, and demonstrating the large genomic variability between different strain lineages.<br><br>Conclusion
SA1665 is a previously uncharacterised DNA-binding protein that has a negative effect on β-lactam resistance in <span class="yellow">MRSA</span>. The SA1665 protein was identified in a DNA-binding protein purification assay, in which it bound to a DNA fragment covering the mec operator region. However, while nonpolar deletion of SA1665 was shown to increase oxacillin resistance levels in several heterogeneously resistant <span class="yellow">MRSA</span>, its deletion had no effect on mecA transcription or PBP2a production. Therefore the negative impact of SA1665 on methicillin resistance is most likely to be through the regulation of other chromosomal factors or cellular functions required for methicllin resistance.<br><br>Methods
Strains and growth conditions
Strains and plasmids used in this study are listed in Table 1. Clinical isolates are from the IMM collection in Zurich, Switzerland. Strains were grown at 37°C in Luria Bertani (LB) broth, shaking at 180 rpm, or on LB agar. Media were supplemented with the following antibiotics when appropriate: 25 or 50 μg/ml kanamycin, 10 μg/ml chloramphenicol, 5 or 10 μg/ml tetracycline, 100 μg/ml ampicillin. Concentrations of cefoxitin used for transcriptional induction were either sub-inhibitory (4 μg/ml) or inhibitory (120 μg/ml).<br><br>Susceptibility testing
Oxacillin resistance levels were compared by swabbing 0.5 McFarland cell suspensions across agar plates containing appropriate concentration gradients of oxacillin. For population analysis profiles, appropriate dilutions of an overnight culture, ranging from 100 to 108, were plated on increasing concentrations of oxacillin. Plates were incubated at 35°C and colony forming units per ml (cfu/ml) were determined after 48 h.<br><br>Binding-protein purification
Crude protein extracts were isolated from CHE482, grown under normal culture conditions until OD600 nm 1.5. Cells were harvested, resuspended in PBS (pH 7.4) and mechanically lysed using Lysing Matrix B (BIO 101 Systems) tubes and a FastPrep FP120 (BIO 101 Systems). Suspensions were clarified by centrifugation and supernatants, containing soluble cytoplasmic proteins, were transferred to Amicon Ultra centrifugal filter devices (Millipore) with a pore cut-off size of 10 kDa. Proteins were then washed and concentrated in 1× binding buffer (10 mM Tris-HCl, pH 7.5, 1 mM EDTA, and 1 mM DTT, 0.5 M NaCl). Protein concentrations were measured by Bradford assay (BioRad Laboratories GmbH) [32]. Primer pair me36F/me36Rbiot (Table 2) were used to amplify a biotinylated mecA promoter/operator fragment, which was bound to streptavidin coated magnetic beads (Dynabeads M-280 Streptavidin, DYNAL BIOTECH) according to the manufacturer's instructions. Binding reactions, containing DNA-coated beads mixed with 100 μg of crude protein extract in 1× protein binding buffer (20 mM Hepes, pH 7.6, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20 (w/v), 30 mM KCl), 0.02 μg/μl poly d(I-C) and 2 ng/μl poly L-lysine, were incubated at room temperature for 30 min with constant rotation. Beads were then washed and binding-proteins eluted in elution buffer (1× protein binding buffer containing 2 M KCl). Eluted proteins were dialysed against water, concentrated by evaporation, and run on 15% SDS polyacrylamide gels. Gels were silver stained using the Protein Silver Staining kit (Amersham Biosciences AB) without the addition of glutaraldehyde. Protein bands were excised from gels and analysed by mass spectrometry (LC/ESI/MS/MS) at the Functional Genomics Centre, Zurich. The SA1665 protein sequence [BAB42933] was analysed by Blast search  and motif scan .<br><br>Expression of recombinant SA1665 protein
SA1665 was amplified using primer pair me65BamHI/me65XhoI (Table 2) and cloned in-frame into pET28nHis6 (unpublished, D. Frey). The resulting plasmid, pME20, was transformed into <span class="yellow">E. coli BL21</span> for expression of recombinant nHis6-SA1665 protein. To maximise the abundance of soluble protein produced, cultures were grown in osmotic shock medium at 37°C (1 g/l NaCl, 16 g/l tryptone, 10 g/l <span class="blue">yeast</span>, 1 M sorbitol, 10 mM betaine, modified from [33]) to an OD600 nm of 0.5, cooled briefly on ice, then induced by adding 100 μM IPTG and growing overnight at 22°C. Crude soluble proteins were extracted using CelLyticB 2× cell lysis reagent (SIGMA). HIS-Select Cobalt Affinity Gel (SIGMA) was used to purify recombinant nHis6-SA1665 according to the manufacturer's instructions.<br><br>Electro mobility shift assay
For gel shift assays, 6 ng aliquots of the biotinylated-DNA fragment used for binding-protein purification were incubated with 0–250 ng of purified nHis6-SA1665 protein in 1× binding buffer (20 mM Hepes pH 7.6, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20 (w/v), 30 mM KCl) containing 0.05 μg/μl poly d(I-C) (Roche) and 5 ng/μl poly L-lysine (Roche). For control binding reactions, 130 × unlabelled mec operator DNA (amplified using primers me36F/me36R, Table 2) was used as a specific binding competitor and 6 ng of herring sperm DNA was used as unspecific competitor DNA. Binding was carried out at 22°C for 30 min. Samples were run on 6% native polyacrylamide gels, contact blotted onto positively charged nylon membrane and detected with the Biotin Chromogenic Detection Kit (Fermentas).<br><br>Primer extension
RNA was extracted from CHE482 cultures that were grown to OD600 nm 0.5, as previously described [12]. Primer extension reactions were performed using 20 μg of total RNA and 3 pmol of the 5'-biotin-labelled primers me97 and me98 (Table 2) using Superscript II reverse transcriptase (Invitrogen), according to the manufacturers instructions. Sequencing reactions were performed using the Thermo Sequenase cycle sequencing kit (U.S. Biochemicals). The Biotin Chromogenic Detection Kit (Fermentas) was used for biotin detection.<br><br>Markerless deletion of SA1665
In frame markerless deletions of SA1665, from the chromosomes of CHE482, ZH37, ZH44, and ZH73, were constructed using the pKOR1 allelic replacement system, as described by Bae et al. [34]. Primer pairs used to amplify the DNA fragments flanking SA1665, for recombination into pKOR1 were: me62attB1/me51BamHI and me62BamHI/me62attB2 (Table 2). All deletion mutants were confirmed by nucleotide sequencing over the deleted region, as well as by Southern blot analysis [35] and pulsed field gel electrophoresis (PFGE) [36].<br><br>Cloning of SA1665 for complementation
A 1533-bp DNA fragment, containing SA1665 together with 690-bp of upstream and 379-bp of downstream DNA, was amplified from strain CHE482 using primers me94BamHI/me94Asp718 (Table 2) and cloned into the <span class="yellow">E. coli</span>/<span class="yellow">S. aureus</span> shuttle vectors pAW17 and pBUS1 [37], creating the complementing plasmids pME26 and pME27, respectively. Plasmids were electroporated into RN4220 [38] and then transduced into different strains using phage 80α.<br><br>Northern blot analysis
Strains were grown overnight in LB (Difco), diluted 1:200 and grown for another 3 h. This preculture was used to inoculate 150 ml (1:1000) of fresh prewarmed LB. Cells were then grown to OD600 nm 0.25 or 1.0 and either left uninduced or induced with cefoxitin 4 or 120 μg/ml. Cultures were sampled from both uninduced and induced cells at time point 0' before induction and at 10' and 30' (min) after induction. To monitor SA1665 expression over growth, separate cultures were also sampled at different growth stages corresponding to OD600 nm 0.25, 0.5, 1, 2, and 4. Total RNA was extracted as described by Cheung et al. [39]. RNA samples (10 μg) were separated in a 1.5% agarose-20 mM guanidine thiocyanate gel in 1× TBE running buffer [40], then transferred and detected as described previously [41]. Digoxigenin (DIG) labelled-probes were amplified using the PCR DIG Probe synthesis kit (Roche). Table 2 contains the list of primer pairs used for the amplification of SA1664, SA1665, SA1666, SA1667, mecR1 and mecA [42] probes. All Northern's were repeated at least two times, using independently isolated RNA samples.<br><br>Western blot analysis
Cells were cultured, as described for Northern blot analysis, to OD600 nm 1.0, then induced with cefoxitin 4 μg/ml. Samples were collected at time 0 (before induction), 10 and 30 min (after induction). Cells were harvested by centrifugation, resuspended in PBS pH 7.4 containing DNase, lysostaphin and lysozyme (150 μg/ml of each) and incubated for 1 h at 37°C. Suspensions were then sonicated and protein aliquots (15 μg) were separated on 7.5% SDS-polyacrylamide gels, blotted onto nitrocellulose membranes (Hybond) and stained with Ponceau to confirm equal protein loading. PBP2a detection was performed using monoclonal PBP2a antibody (1:20000) from the <span class="yellow">MRSA</span>-screen kit (Denka Seiken).<br><br>
Authors' contributions
ME carried out molecular genetic and microbiological studies and drafted the manuscript. BB participated in the design of the study and helped to draft the manuscript. NM participated in the design and coordination of the study, carried out molecular biological studies and helped to draft the manuscript. All authors read and approved the final manuscript.<br><br>
<h3>pmcA1891629</h3>A secondary structural common core in the ribosomal ITS2 (internal transcribed spacer) of
          Culexspecies from diverse geographical locations
Abstract
In the present study, sequence and structural analysis of ITS2 region (the spacer segment between 5.8S and 28S rRNA of 
          mature rRNA sequences) of 7 Culex species belonging to 5 different geographical locations was carried out. Alignment of the 
          ITS2 sequence from the 7 species revealed 8 homologous domains. Four species namely <span class="yellow">C. vishnui</span>, <span class="yellow">C. annulus</span>, <span class="yellow">C. pipiens</span>, 
          <span class="yellow">C. quiquefasciatus</span>showed high sequence (98­100%) and RNA secondary structure similarity. The ITS2 similarity among different 
          species is high despite their varying geographical locations. Several common features of secondary structure are shared among 
          these species, with some of them supported by compensatory changes, suggesting the significant role by ITS2 as an RNA domain 
          during ribosome biogenesis.<br><br>Background<br><br>				Culex mosquito species have been described from a wide range of environments and are involved in pathogen transmission from 
      <span class="yellow">human</span> to reservoirs and vice-versa. [1] Sequence similarity have been 
      reported for ITS2 (the spacer segments between 5.8S and 28S rRNA of mature rRNA sequences) from 5 Culex species of diverse 
      geographic locations. [3–4–
      5–9–
      20] The ITS spacers are versatile as genetic markers and have been used for the determination of taxonomic 
      classification. [17] Recent functional analyses performed on <span class="blue">yeast</span> ribosomal 
      RNA genes clearly show that the structural integrity of the transcribed spacer regions is an essential prerequisite for the correct 
      processing of mature rRNA and for the biogenesis of active ribosomal subunits. [2–
      3] The derivation of reliable secondary structure models for each transcribed spacer 
      region would represent a major step towards a detailed understanding of their biological roles.
Comparative sequence analysis provides a powerful tool for identifying biologically relevant folding patterns in RNA molecules. 
      [4] This involves collection of sequences exhibiting substantial nucleotide 
      differences while retaining unequivocal sequence similarity. However, a high degree of sequence variation in the transcribed 
      spacers is known even among closely related species. [5–6–7] Here, we analyze ITS2 from 7 
      Culex species characteristic of 5 different geographical locations to assemble common RNA structural features.<br><br>Methodology
Dataset
ITS2 nucleotide sequences from 7 Culex mosquitoes characteristic of 5 geographical locations (Italy, China, Africa, 
      America, and Japan) were downloaded from GenBank (www.ncbi.nlm.nih.gov/genbank
      ). The GenBank accession numbers for ITS2 sequences 
      from <span class="yellow">C. pipens</span> (North America), <span class="yellow">C. tritaeniorhynchus</span> (China), <span class="yellow">C. quinquefasciatus</span>
       (Africa), <span class="yellow">C. tarsalis</span> (Italy), <span class="yellow">C. annulus</span> (China), <span class="yellow">C. pseudovishnui</span> 
      (China), and <span class="yellow">C. vishnui</span> (Japan) were X75817, AF305558, Z48468, U33031, AF453488, AF453498 and AF165900, 
      respectively.<br><br>Sequence alignments
Multiple sequence alignments were performed using CLUSTALW with a gap opening penalty of 15 and gap extension penalty of 6.66. 
      [8]<br><br>Secondary structure prediction
The RNA secondary structures for ITS2 were predicted using RNADRAW. [21]
      RNADRAW predicts RNA structures by identifying suboptimal structures using the free energy optimization methodology at a default 
      temperature of 370°C.  In the current study, ITS2 and 5.8S regions (the first 170 nucleotides) were used for RNA structure 
      prediction. The minimum energy structure prediction algorithm in RNADRAW was ported from the RNAFOLD program included in the 
      Vienna RNA package. [24] The dynamic programming algorithm employed in 
      RNADRAW was based on the work of Zuker and Stiegler [25] and uses energy 
      parameters taken from Turner [26], Freier [27] and Jaeger. [28]<br><br>Phylogenetic analysis
The phylogenetic Genebee service was used for phylogenetic tree construction. [22
      ]<br><br>RNA fold
The Sribo program in Sfold (Statistical Folding and Rational Design of Nucleic Acids) was 
      used to predict the probable target accessibility sites (loops) for trans-cleaving ribozymes in ITS2. [24] The prediction of accessibility is based on a statistical sample of the Boltzmann 
      ensemble for secondary structures. Here, we assessed the likelihood of unpaired sites for potential ribozyme target. Each mRNA 
      exists as a population of different structures. Hence, stochastic approach to the evaluation of accessible sites was found 
      appropriate. [29] The probability profiling approach by Ding and Lawrence 
      [30] reveals target sites that are commonly accessible for 
      a large number of statistically representative structures in the target RNA. This novel approach bypasses the long-standing 
      difficulty in accessibility evaluation due to limited representation of probable structures due to high statistical confidence 
      in predictions. The probability profile for individual bases (W = 1) is produced for the region that 
      includes a triplet and two flanking sequences of 15 bases each in every site of the selected cleavage triplet (e.g., GUC).
      <br><br>
Results and Discussion
ITS2 sequences showed a taxonomic trend similar to that in phylogeny construction (Figure 1A
			). A multiple sequence alignment 
      of 5.8S rDNA and ITS2 showed in Figure 1B indicated that more distantly related species have 
      lower sequence similarity in the 
      ITS2 region. The predicted features of ITS2 using RNADRAW are given in Table 1. The stems 
      (double stranded paired regions) 
      stabilize RNA secondary structures and the number of stems present in each ITS2 is given (Table 1). ITS2 RNA structures from 
      <span class="yellow">C. pipiens</span> and <span class="yellow">C. quinquefasciatus</span> have the highest negative free energy (-149.38 Kcal and 
      -148.23 Kcal) followed by <span class="yellow">C. tarsalis</span> (-129.2), <span class="yellow">C. annulus</span> and then by <span class="yellow">C. vishnui</span>
      (-115.3), <span class="yellow">C. pseudovishnui</span> (-105.66) and <span class="yellow">C. tritaeniorhyncus</span> (- 82.57). Visual comparison shows 
      that this is related to the trend in the cladogram given in Figure 1A. A high degree of sequence similarity is observed at the 
      5' end compared to the 3' end (Figure 1A). This is due to factors such as genetic drift, the relative number and size of repeats, 
      rates of unequal crossover, gene conversion, immigration and the number of the loci influencing the length. [10] Furthermore, a high level of sequence similarity is found between C. 
      annulus and <span class="yellow">C. vishnui</span> as well as <span class="yellow">C.  quinquefasciatus</span> and <span class="yellow">C.  pipiens</span>.
      The simple tandem repeats shown in Figure 1B as bolded regions are found to be similar. 
      This similarity is seen in corresponding 
      RNA structures and computed energies.
The Culex species considered in this study were then grouped into three classes based on the similarity of 
      RNA stems and loops. Despite their different geographic locations with varying eco-climatic conditions, class I 
      (<span class="yellow">C. annulus</span>, <span class="yellow">C. pseudovishnui</span> and <span class="yellow">C. vishnui</span>) isolates (China and Japan) showed high similarity 
      in secondary structural features. Similar observations were seen in class II (<span class="yellow">C. pipiens</span>, <span class="yellow">C. quinquefasciatus</span> and 
      <span class="yellow">C. tarsalis</span>) isolates from North America and Africa. [19] 
      A high degree of similarity in the 5.8S region unlike the ITS2 is shown for different isolates due to relative evolutionary 
      selection. [9] The loops 11 and 12 having sequences UGUCG and CUUCGGUG, 
      respectively are highly conserved in all classes.
Figure 1C shows the distribution of different types of loops (hairpin, bulge, multi branched, interior and exterior) among 
      different isolates. The segments of the ITS2 having score ≥ 50 are further probed carefully for target site to assess the 
      likelihood of unpaired segments. Interestingly, the observed phylogenetic trend was identified with respect to the target 
      accessibility sites for the seven Culex isolates. The order of preference is interior loop, bulge loop, multiple 
      branched loop, hairpin loop and exterior loops in all the isolates.
Several homologous domains were observed in the ITS regions of Aedes mosquito species by Wesson 
			et al	 [11] and was shown that these domains base pair to 
			form a core region central to 
      several stem features. This suggested that conserved core region of rDNA is more important for a functional rRNA folding 
      pattern. Barker found that ITS2 is unique in the 16 populations of <span class="yellow">Boophilus microplus</span>, <span class="yellow">Boophilus decoloratus</span>, 
      <span class="yellow">Rhipicephalus appendiculatus</span>, <span class="yellow">Rhipicephalus zambesiensis</span>, <span class="yellow">Rhipicephalus evertsi</span>, <span class="yellow">Rhipicephalus sanguineus</span>, Rhipicephalus 
      turanicus, <span class="yellow">Rhipicephalus pumilio</span> and <span class="yellow">Rhipicephalus camicasi</span> from different geographical locations. 
      [12] These results suggest that the differences and similarity observed 
      in the ITS2 of different species are not simply accumulated due to random mutation and have evolved for functional selection 
      in ribosome biogenesis. However, it was shown in three related mosquito genera (Aedes, Psorophora and <span class="yellow">Haemogogus</span>)
      that the intra-spacer variable regions appear to co-evolve and that ITS2 variation is constrained by its secondary structure. 
      [11] Further studies have demonstrated that the ITS2 is essential 
      for the correct and efficient processing and maturation of 26S rRNA ribosomal units. [
      13] Furthermore, the information required for the efficient removal of ITS2 from its RNA precursor is not 
      localized but dispersed throughout the ITS2 region. Thus, insertions and deletions (indels) that affect secondary structures 
      alter rRNA processing. Critical changes in the rRNA folding pattern due to evolutionary sequence variation in the ITS spacer 
      regions may thus have an important role on the kinetics of precursor rRNA formation for the efficient functioning of rDNA clusters.
      <br><br>Conclusion
Comparison of ITS2 sequences from different isolates of Culex show similarity and variations. Surprisingly, species displaying 
      sequence similarity belong to different geographical locations with diverse climatic and ecological conditions. This implies that 
      the ITS2 regions have less selective pressure than the ribosomal regions. Several common structural folds were shared among the 
      selected mosquitoes for maintaining functional equivalents. Construction of an evolutionary tree using more isolates of 
      Culex will provide an understanding for their functional selection.<br><br>
<h3>pmcA2398786</h3>Genomic-Bioinformatic Analysis of Transcripts Enriched in the Third-Stage Larva of the Parasitic Nematode <span class="yellow">Ascaris suum</span>
Abstract
Differential transcription in <span class="yellow">Ascaris suum</span> was investigated using a genomic-bioinformatic approach. A cDNA archive enriched for molecules in the infective third-stage larva (L3) of <span class="yellow">A. suum</span> was constructed by suppressive-subtractive hybridization (SSH), and a subset of cDNAs from 3075 clones subjected to microarray analysis using cDNA probes derived from RNA from different developmental stages of <span class="yellow">A. suum</span>. The cDNAs (n = 498) shown by microarray analysis to be enriched in the L3 were sequenced and subjected to bioinformatic analyses using a semi-automated pipeline (ESTExplorer). Using gene ontology (GO), 235 of these molecules were assigned to ‘biological process’ (n = 68), ‘cellular component’ (n = 50), or ‘molecular function’ (n = 117). Of the 91 clusters assembled, 56 molecules (61.5%) had homologues/orthologues in the free-living nematodes <span class="yellow">Caenorhabditis elegans</span> and <span class="yellow">C. briggsae</span> and/or other organisms, whereas 35 (38.5%) had no significant similarity to any sequences available in current gene databases. Transcripts encoding protein kinases, protein phosphatases (and their precursors), and enolases were abundantly represented in the L3 of <span class="yellow">A. suum</span>, as were molecules involved in cellular processes, such as ubiquitination and proteasome function, gene transcription, protein–protein interactions, and function. In silico analyses inferred the <span class="yellow">C. elegans</span> orthologues/homologues (n = 50) to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%), or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway. Small numbers of these 50 molecules were predicted to be secreted (10%), anchored (2%), and/or transmembrane (12%) proteins. Functionally, 17 (34%) of them were predicted to be associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority being embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%). A genetic interaction network was predicted for these 17 <span class="yellow">C. elegans</span> orthologues, revealing highly significant interactions for nine molecules associated with embryonic and larval development (66.9%), information storage and processing (5.1%), cellular processing and signaling (15.2%), metabolism (6.1%), and unknown function (6.7%). The potential roles of these molecules in development are discussed in relation to the known roles of their homologues/orthologues in <span class="yellow">C. elegans</span> and some other nematodes. The results of the present study provide a basis for future functional genomic studies to elucidate molecular aspects governing larval developmental processes in <span class="yellow">A. suum</span> and/or the transition to parasitism.<br><br>Introduction
Parasitic nematodes are of major socio-economic importance in animals. For example, hundreds of millions of <span class="yellow">people</span> are infected with geohelminths (soil-transmitted worms), such as blood-feeding hookworms <span class="yellow">Ancylostoma duodenale</span> and/or <span class="yellow">Necator americanus</span>, <span class="yellow">Trichuris trichiura</span> and Ascaris spp. [1], causing serious adverse effects on <span class="yellow">human</span> health, particularly in <span class="yellow">children</span>. Similarly, parasitic nematodes of livestock, such as <span class="yellow">pigs</span>, also cause substantial economic losses due to subclinical and clinical diseases, with billions of dollars spent annually on the treatment and control of gastro-intestinal nematodes. In addition to the socioeconomic impact that these parasites have, there is potential for the emergence of resistance in them against all of the main classes of (nematocidal) compounds used to treat the diseases they cause [2]–[5]. Therefore, there is a significant need to work toward discovering new compounds to control these parasites. Gaining an improved understanding of the molecular basis of parasite development provides such an avenue.
Compared with the free-living nematode <span class="yellow">Caenorhabditis elegans</span>, there is very little information on fundamental molecular aspects of development in parasitic nematodes [6]–[8]. Since the genome sequence of <span class="yellow">C. elegans</span> was published in 1998 [9], many aspects of the molecular biology of this nematode have been elucidated. For instance, microarray analyses have been used to examine developmental and gender-enriched gene expression [10],[11], and the functions of more than 96% of the <span class="yellow">C. elegans</span> genes have been assessed by double-stranded RNA interference (RNAi, or gene silencing; [12]) [13]–[18]. Comparative analyses of genetic data sets have shown that parasitic nematodes usually share ∼50–70% of genes with <span class="yellow">C. elegans</span> (e.g., [19],[20]). There is similarity in other features (such as basic body plan and moulting) between <span class="yellow">C. elegans</span> and parasitic nematodes, suggesting that some molecular pathways are relatively conserved [8],[21]. Understanding the pathways linked to basic nematode biology and development could have important implications for finding new ways of disrupting these pathways and thus facilitate the identification of new drug targets.
Despite the advances in genomic technologies [7], [22]–[29] and the study of <span class="yellow">C. elegans</span>, there is a paucity of information on the genomics of parasitic nematodes of animals, particularly in relation to development. Also considering the major socioeconomic impact of Ascaris and ascariasis in <span class="yellow">humans</span> and <span class="yellow">pigs</span> [30]–[32], several characteristics, including the large size of the adult worm (providing the opportunity of investigating individual organ systems and tissues), the ability to maintain Ascaris in the <span class="yellow">pig</span>, store eggs and culture larvae in vitro for relatively long periods of time (months to years) [32] as well as the discovery that RNAi achieves “cross-species” gene silencing for a selected number of genes [33],[34] and the imminent genome sequence (http://www.sanger.ac.uk/Projects/Helminths/) all indicate that Ascaris could serve as a powerful model system for investigating reproductive and developmental processes in nematodes.
In the present study, Ascaris from <span class="yellow">pigs</span> was used to study molecules abundantly transcribed in the infective third-stage larva (L3). Following the oral ingestion of Ascaris eggs by the host (<span class="yellow">human</span> or <span class="yellow">pig</span>), L3s are released and then invade/penetrate predominantly the caecal wall [35] to then undergo hepato-pulmonary migration, after which ultimately the adult females and males establish and develop in the small intestine [36],[37]. The molecular mechanisms linked to host invasion and parasite development are largely unknown. Here, we constructed an L3-enriched cDNA library using the method of suppressive-subtractive hybridization (SSH), explored transcription of a representative subset of molecules by microarray analysis and conducted bioinformatic analyses to characterize these molecules, map them to biochemical pathways and predict genetic interactions based on comparisons with <span class="yellow">C. elegans</span> and/or other organisms.<br><br>Materials and Methods
Production of Different Developmental Stages of Ascaris
Experimental <span class="yellow">pigs</span> (8–12 weeks of age) were purchased from and maintained in the Experimental Animal Center of South China Agricultural University. These <span class="yellow">pigs</span> were treated humanely, according to the Animal Ethics procedures and guidelines of the People's Republic of China. Adult <span class="yellow">worms</span> (males and females) of <span class="yellow">A. suum</span> were collected from the small intestines of <span class="yellow">pigs</span> from an abattoir in Shenzhen, China. Infective eggs and infective L3s of <span class="yellow">A. suum</span> were produced according to the methods described previously [38]. In brief, eggs from the uteri of adult females of <span class="yellow">A. suum</span> were collected and incubated at 28°C for 28 days to allow them to develop to infective eggs (containing infective L3s). To obtain pure infective L3s, 7.5% v/v sodium hypochlorite was used to treat the larvated eggs at 37°C overnight and then the eggs were shaken with glass-beads; then, the exsheathed L3s and shells were separated by density gradient centrifugation using lymphocyte separating medium (LSM) [38]. Following the experimental infection of helminth-free <span class="yellow">pigs</span> with infective Ascaris eggs as described previously [39], the L3s from livers and in lungs as well as L4s in intestines were isolated according to an established method [40]. All parasite materials were snap-frozen in liquid nitrogen prior to storage at −70°C.<br><br>Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)
Total RNA was isolated from adult females and males, different larval stages or eggs of <span class="yellow">A. suum</span> using TriPure reagent (Roche) as recommended by the manufacturer. Equal amounts of total RNA from each stage or sex were pooled. The mRNA was isolated using the Oligotex mRNA Kit (Qiagen), following the manufacturer's protocol. SSH was carried out using the PCR-Select cDNA Subtraction kit (Clontech), according to the manufacturer's protocol. In brief, cDNA synthesized from mRNAs from infective L3s was subtracted against cDNA synthesized from the pooled mRNA from all other stages included herein. The SSH library was constructed using infective L3s as the tester and pooled cDNAs from all other stages as the driver. The effectiveness of this subtraction process has already been demonstrated in previous studies [41],[42]. The cDNA obtained following SSH was cloned into the pGEM-T Easy plasmid vector (Promega) and competent <span class="yellow">Escherichia coli</span> (JM109) transformed. Positive clones, picked randomly (based on blue/white selection), were grown overnight in Luria Bertani (LB) medium (shaking, 37°C). Individual inserts were PCR-amplified using “nested primers” 1 and 2R from the Subtraction kit (Clontech) and examined by agarose electrophoresis.<br><br>Preparation of Microarray Slides
Clones (n = 3075) from the subtracted library were picked and cultured overnight in LB containing ampicillin (1000 IU/ml) in sealed 96-well blocks. Five µl of culture suspension from each well were transferred into individual wells thermocycling (96-well) plates and the inserts PCR-amplified using primers 1 and 2R. Following a 10 min denaturation step at 94°C, the amplification proceeded for 25 cycles of 10 s at 94°C, 30 s at 68°C and 1.5 min at 72°C, with a final extension for 5 min at 72°C. Products were resolved in agarose gels, ethanol precipitated, re-suspended in 16 µl of “spotting solution” (Shanghai BioStar Genechip, Inc) to a final concentration of ∼500 ng per µl, before being printed on to glass slides (in duplicate) using a robotic arrayer. Sixteen blanks (using spotting solution only) and the same number of negative (irrelevant cDNAs with no relationship to Ascaris) were also printed on to slides and served as negative controls; β-actin of <span class="yellow">A. suum</span> served as a positive control to assess the efficiency of labeling and hybridization. The slides were air-dried for 2 h, and cDNA in the spots were cross-linked at 254 mJ. The printed slides were stored at 4°C.<br><br>Labeling of cDNA Probes with Fluorescent Dyes, and Microarray Analysis
The cDNAs produced from total RNA from <span class="yellow">A. suum</span> eggs, infective L3s, L3s isolated from <span class="yellow">pig</span> liver or lung, fourth-stage larvae (L4s), adult males or females [as described in the section ‘Construction of the cDNA Library by Subtractive-Suppressive Hybridization (SSH)’] were labeled with cyanine dyes. Cy3 or Cy5-dCTP was incorporated into cDNA produced from 30 µg of total RNA by direct labeling in a reverse transcription reaction using an oligo (dT) primer. Labeled cDNA was purified using DyeEx columns (Qiagen).
Microarray slides were incubated with a pre-hybridization solution [5×SSC, 1% <span class="yellow">bovine</span> serum albumin (BSA), 0.1% sodium dodecyl-sulphate (SDS)] for 6 h at 42°C. After pre-hybridization, the microarray slides were incubated with ‘pooled’ Cy3 and Cy5-labeled probes in hybridization solution (5×SSC, 1% BSA, 0.1% SDS), in the dark at 42°C for 18 h, and then washed in solution I (1×SSC, 0.2% SDS) for 10 min, followed by solution II (0.1×SSC, 0.2% SDS) for 10 min at 60°C, according to the protocols provided by Shanghai BioStar Genechip, Inc. A “dye flip” was carried out to control for any bias in hybridization signal between the Cy-labeled cDNA probes (produced for two distinct mRNA populations). The slides were dried and scanned (ScanArray 4000 scanner) using image acquisition software (Shanghai BioStar Genechip Inc.) and a range of laser power and photo-multiplier tube intensities. The mean hybridization signal (derived from four replicates of the same array) were corrected for background, normalized [43], log2-transformed and then subjected to statistical analysis employing the students t-test in a spreadsheet (Excel, Microsoft, USA). The microarray data were analysed for differential cDNA hybridization (>2.0-fold to 114.3-fold) between L3 and each of the other stages (eggs, lung and liver L3s, L4, adult female and adult male).<br><br>Verification of Differential Hybridization by Reverse Transcription-Coupled Polymerase Chain Reaction (RT-PCR) Analysis
For a subset (n = 17) of representative ESTs (rESTs), RT-PCR was used to verify the differential transcription recorded by microarray analysis. Double-stranded cDNA was synthesized from total RNA (separately) from each stage or sex of <span class="yellow">A. suum</span> using reverse transcriptase (Superscript III, Invitrogen). Briefly, 5 µg of total RNA were added to 14 µl of H2O and 1 µl of oligo d(T)n = 12–18 primer (0.5 µg/µl), heated to 70 °C for 10 min and chilled on ice. First- and second-strand cDNAs were synthesized via the addition of 4 µl of first-strand cDNA buffer (250 mM Tris-HCl, pH 8.3, 375 mM KCl and 15 mM MgCl2), 2 µl of 0.1 M dithiothreitol, and 1 µl of 10 mM of each dNTP, followed by an incubation at 25 °C (10 min), 42 °C (50 min) and 70 °C (15 min). One-tenth of each double-stranded cDNA produced was then used as a template in the PCR. The transcripts were amplified from individual cDNAs by PCR using oligonucleotide primers (sequences available upon request) designed to each EST. The PCR amplification of a portion (209 bp) of the β-actin gene (accession no. BI594141) using forward primer (5′-CTCGAAACAAGAATACGATG-3′) and reverse primer (5′- ACATGTGCCGTTGTATGATG-3′), previously determined to be present in all developmental stages and both sexes of <span class="yellow">A. suum</span> [44], served as a positive control. Samples without template (no-DNA controls) were included in each PCR run. The following cycling conditions were employed: one cycle at 94 °C (5 min), 94 °C (30 s), 60 °C (30 s) and 72 °C (30 s) for 30 cycles, followed by a final extension of 70 °C (7 min). Following the PCR, 5 µl of individual amplicons were resolved in ethidium bromide-stained agarose gels (2%) and then photographed upon transillumination. The relative band intensities were analyzed using UVIsoft Image Acquisition and Analysis software (UVITEC). The specificity and identity of individual amplicons were confirmed by direct sequencing using the same primers (separately) as employed for their amplification.<br><br>Sequencing and Bioinformatics Analyses
Clones from the SSH cDNA library with increased hybridization in microarray analysis to the infective L3 compared with other stages were sequenced using standard technology [45]. The nucleotide sequences have been deposited in the GenBank database under accession numbers ES290984-ES291074. Following the processing of the sequences (i.e., removal of vector sequences, quality assurance and clustering), contigs or singletons from individual clusters were subjected to BLASTx (NCBI: www.ncbi.nlm.nih.gov) and BLASTn (EMBL-EBI Parasite Genome Blast Server: www.ebi.ac.uk) analysis to identify putative homologues in <span class="yellow">C. elegans</span>, other nematodes and other organisms (e-value of ≤1e-05). Peptides inferred from ESTs were classified functionally using Interproscan (available at http://www.ebi.ac.uk/InterProScan/) employing the default search parameters. WormBase (www.wormbase.org) was interrogated extensively for relevant information on <span class="yellow">C. elegans</span> homologues/orthologues, including RNAi phenotypic, transcriptomic, proteomic and interactomic data. ESTs with homologues/orthologues in <span class="yellow">C. elegans</span> and other nematodes were also subjected to analysis employing the KEGG Orthology-Based Annotation System (KOBAS) (www.kobas.cbi.pku.edu.cn), which predicts the biochemical pathways in which molecules are involved. The open reading frames (ORFs) inferred from selected ESTs with orthologues in <span class="yellow">C. elegans</span> were also subjected to “secretome analysis” using the program SignalP v.2.0 www.cbs.dtu.dk/services/SignalP/), employing both the neural network and hidden Markov models to predict signal peptides and/or anchors [46]–[48]. Also, transmembrane domains were predicted using the program TMHMM (www.cbs.dtu.dk/services/TMHMM/; [49]–[51]), and subcellular localization inferred employing the program WoLF PSORT (http://wolfpsort.org/; [52]).
The method established by Zhong and Sternberg [53] was used to predict the interactions for <span class="yellow">C. elegans</span> orthologues of the L3-enriched molecules from Ascaris. In brief, interaction, phenotypic, expression and gene ontology data from <span class="yellow">fruitfly</span>, <span class="blue">yeast</span>, <span class="yellow">mouse</span> and <span class="yellow">human</span> were integrated using a naïve Bayesian model to predict genetic interactions among <span class="yellow">C. elegans</span> genes ([45],[53]; Zhong and Sternberg, unpublished). The predicted networks resulting from the analyses were saved in a graphic display file (gdf) format and examined using the graph exploration system available at http://graphexploration.cond.org/. Images were labeled and saved in the joint photographic experts group (jpeg) format.<br><br>
Results
To identify molecules transcribed abundantly in the L3 of <span class="yellow">A. suum</span>, an enriched cDNA library was constructed by SSH. From a total of 3075 clones from this library, 2921 (95%) were shown to contain an insert (which could be amplified by PCR). From 2671 (92%) of these clones, amplicons representing single bands of ∼400 to 600 bp in size were produced. These latter amplicons were arrayed (in duplicate) on to slides and then hybridized with Cy3-labeled L3-cDNA or with Cy5-labeled cDNA from eggs, liver/lung L3s, L4s, adult female or adult male of Ascaris. Dye flip was conducted to verify the hybridization data. Of the 2671 (duplicate) spots, 1526 had a significant difference in hybridization between infective L3 cDNA and cDNAs from all other stages or sexes of <span class="yellow">A. suum</span>, of which 515 had a >2.0-fold increased hybridization for the L3.
In order to independently verify the hybridization results in the microarray, a PCR-based analysis of a selected subset (n = 17) clones was conducted using specific primer pairs. Having verified the specificity and identity of individual amplicons by sequencing, PCR results were reproducible (based on multiple runs on different days) and ∼94% (16 of 17) concordant with those of the microarray analysis (not shown). There was complete concordance for representative clones associated with a differential signal of ≥3.0-fold in the microarray.
The clones linked to the 515 spots representing increased transcription (>2.0-fold) in infective L3 compared with the other developmental stages or sexes included were subjected to sequencing. The 498 sequences (length: 550±115 bp) determined were then subjected to detailed bioinformatic analyses. There were 91 unique clusters (accession numbers ES290984-ES291074), of which 55 were singletons (sequences determined once). Of 56 molecules (61.5%) with significant similarity to sequences other than <span class="yellow">A. suum</span> in the databases interrogated, 50 (54.9%) had <span class="yellow">C. elegans</span> or <span class="yellow">C. briggsae</span> homologues, and six had similarity to ESTs already sequenced from ascaridoid and/or other parasitic nematodes, and/or other organisms (Table 1). A significant proportion (38.4%) did not have any similarity in sequence to any organisms for which data are presently available. Comparative analysis specifically against <span class="yellow">A. suum</span> EST data sets (n∼42,000) available in public databases confirmed independently that the majority of molecules (>60%) were present exclusively in the infective L3 stage or were orphans.
As gene ontology (GO) provides a hierarchy that unifies the descriptions of biological, cellular and molecular functions [54], this approach was employed to predict the classification and gene function of molecules enriched in infective L3 of <span class="yellow">A. suum</span>. A summary of the GO categories of these molecules is displayed in Fig. 1. Of the 91 contigs, 32 (35%) could be functionally assigned to ‘biological process’ (n = 38), ‘cellular component’ (n = 17) and ‘molecular function’ (n = 64). The most common subcategories were gluconeogenesis (13%) and metabolic process (13%) within ‘biological process’, extracellular region (24%) within ‘cellular component’, and catalytic activity (11%) and phosphoenolpyruvate carboxykinase activity (8%) within ‘molecular function’ (Table S1).
A focused KOBAS analysis inferred the 50 <span class="yellow">C. elegans</span> orthologues/homologues to be involved in apoptosis and insulin signaling (2%), ATP synthesis (2%), carbon metabolism (6%), fatty acid biosynthesis (2%), gap junction (2%), glucose metabolism (6%) or porphyrin metabolism (2%), although 34 (68%) of them could not be mapped to a specific metabolic pathway (Table 2). Of these 50 molecules, small numbers were predicted to be secreted (10%), anchored (2%) and/or transmembrane (12%) proteins (Table 2). Functionally, 17 (34%) of the 50 molecules were associated with (non-wild-type) RNAi phenotypes in <span class="yellow">C. elegans</span>, the majority displaying embryonic lethality (Emb) (13 types; 58.8%), larval arrest (Lva) (23.5%) and larval lethality (Lvl) (47%) (Table 2).
Extending this analysis, a relatively complex genetic interaction network was predicted for the 17 <span class="yellow">C. elegans</span> orthologues (i.e., with non-wild-type RNAi phenotypes) (see Table S2). Statistically highly significant interactions were predicted for nine of the <span class="yellow">C. elegans</span> genes; the top five interactors are displayed in Fig. 2. The gene ontology categories for eight selected <span class="yellow">C. elegans</span> genes (F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2) included: embryonic development, egg hatching, larval development and/or growth. The other categories included: positive regulation of growth rate (F55A12.8, kin-2, mup-2, pab-1, rpl-22 and T21B10.2) and gamete generation and locomotory behaviour (kin-2, mup-2, pab-1 and F55A12.8, kin-2, mup-2, respectively). The <span class="yellow">C. elegans</span> homologue egl-3 was predicted to be involved in proteolysis (see www.wormbase.org). All nine <span class="yellow">C. elegans</span> orthologues were predicted to interact directly with a total of 296 (range: 5–75) other genes and, in particular, a direct genetic interaction was predicted between pab-1 and T21B10.2 (Fig. 2). The 296 interactors were associated with embryonic and larval development (n = 198; 66.9%), information storage and processing (n = 15; 5.1%), cellular processes and signalling (n = 45; 15.2%) and metabolism (n = 18; 6.1%); the precise function of some of the interactors (n = 20; 6.7%) is presently unknown (Table S2).<br><br>Discussion
The present study investigated transcripts in infective L3s of <span class="yellow">A. suum</span> using a genomic-bioinformatic platform. The focus was on comparisons with <span class="yellow">C. elegans</span> homologues/orthologues, because the entire genome sequence of this nematode is known [9] and because there is a wealth of information on the localization and functionality of its molecules (www.wormbase.org; http://elegans.bcgsc.bc.ca/knockout.shtml). The functions of most genes in <span class="yellow">C. elegans</span> have been assessed using RNAi (e.g., [14],[15],[17],[55],[56]) in the hermaphroditic stage, whereas there is a paucity of functional information available for Ascaris and other parasitic nematodes of animals [57],[58].
Following the microarray analysis of >2500 ESTs from the SSH library, 498 cDNAs inferred to be enriched in the L3, based on hybridization signal, were sequenced and subjected to comprehensive in silico analyses. Of the 91 clusters of molecules categorized, 50 (54.9%) had <span class="yellow">C. elegans</span> homologues/orthologues with loss-of-function phenotypes could be mapped to key pathways. The statistically significant genetic interactions predicted for 9 of the 50 <span class="yellow">C. elegans</span> orthologues [namely egl-3, F33D11.10, F55A12.8, kin-2, mec-12, mup-2, pab-1, rpl-22 and T21B10.2 ( = enol-1)] and the interaction network included genes encoding kinases, alpha-tubulins, enolases, troponin and other named and unnamed proteins. Eight of these molecules (enol-1, pab-1, F33D11.10, rpl-22, F55A12.8 mec-12, mup-2 and kin-2) have known or predicted roles in embryonic and larval growth and development, gamete generation, locomotory behaviour or other biological processes in <span class="yellow">C. elegans</span> (see www.wormbase.org).
The enolase encoded by enol-1 is predicted to play a role in glycolysis, gluconeogenesis, phenylalanine, tyrosine and tryptophan biosynthesis (cf. [59]). Since glucose is the main source for ATP production, the alteration in these key glycolytic enzymes may lead to cellular dysfunction, such as impaired ion-motive ATPase required to maintain potential gradients, operate pumps and maintain membrane lipid asymmetry [60]. Bioinformatic analysis for transmembrane helices (TMHMM) and peptide signal sequences (SignalP) predicted ENOL-1 to be a non-secreted protein localized to the cytoplasm (cf. Table 2). Nonetheless, enolases are often detected in the excretory/secretory (ES) products of parasitic helminths, including adult <span class="yellow">A. suum</span> [61], and appear to play a role in the triggering of nitric oxide production by host cells. The enol-1 orthologue of <span class="yellow">C. elegans</span> has been predicted to interact specifically with the polyadenylate binding protein gene, pab-1, inferred to be involved in coordinated gene transcription and expression during normal larval development [16]. Poly(A)-binding proteins (PABPs) are recognized to be central to the regulation of mRNA translation and stability [62]. Present evidence suggests that the expression of PAB-1 is regulated by an oligo-pyrimidine tract in response to cell growth and relates to coordinated growth regulation in <span class="yellow">C. elegans</span> [62]. Furthermore, gene silencing of pab-1 and its selected interactors (see Fig. 2) leads to embryonic lethal (Emb), slow growth (Slo) and sterile progeny (Stp) phenotypes (see www.wormbase.org).
Another gene (F33D11.10; EST code 4F10; see Table 2) which encodes an ATP-dependent RNA helicase and is associated with embryonic lethal (Emb) and larval lethal (Lvl) RNAi phenotypes, was shown to be highly transcribed in infective L3s of <span class="yellow">A. suum</span>. Helicases are involved in a variety of RNA metabolic processes, including translation initiation, pre-mRNA splicing, pre-rRNA processing, rRNA maturation and RNA degradation [63], and are crucial for life cycle progression, sex determination and early embryogenesis in <span class="yellow">C. elegans</span> [60]. The high transcription levels of a homologue/orthologue in the L3 of <span class="yellow">A. suum</span> might suggest a similar role in this ascaridoid. Similarly, the coordination of the expression of a large number of genes is required for normal growth and cell proliferation during larval development. The high transcription level for the ribosomal protein gene homologue rpl-22 (large subunit family member; EST code 26G12, see Table 2) in the infective L3 of <span class="yellow">A. suum</span> compared with other developmental stages is likely to reflect the substantial rate of cell growth in this stage [64].
The gene (F55A12.8, EST code 4G11; see Table 2) encoding an acetyl-transferase with a putative ATPase domain, shown to be enriched in the L3 of <span class="yellow">A. suum</span>, was predicted to interact with 75 other genes all involved in energy production and/or RNA processing (see Table S2). Several molecules involved in ATP synthesis and mitochondrial pathways (e.g., cytochrome oxidase c subunits 1, 2 and 3, ADP/ATP translocases, NADH dehydrogenases, ATPases and ATP synthetases) have been reported previously to be highly represented in the L3 stage of <span class="yellow">Anisakis simplex</span> [65], thus supporting the proposal that substantial energy is required for larval development as well as the transition from the free-living to the parasitic stage and the invasion of the host. There is also likely to be a substantial energy requirement for muscle contraction linked to larval motility in <span class="yellow">A. suum</span>, as the L3s penetrate the caecal wall in the porcine host, before undergoing hepato-pulmonary migration [35]. Accordingly, genes encoding a specialized tubulin expressed in mechanoreceptors (mec-12, EST code 13E09) and a troponin (mup-2, EST code 01G03; see Table 2), both predicted to interact with a total number of 32 tubulin- and myosin-encoding genes, also supported a link to extensive muscle contraction and motility in <span class="yellow">A. suum</span> L3s. Also, neuroactive peptides are required to regulate the responsiveness of nematode larvae to mechanical stimuli [66]. A homologue encoded by egl-3 was shown to be highly transcribed in the L3 of <span class="yellow">A. suum</span>; EGL-3 is predicted to be a pro-hormone convertase involved in the maturation of neuropeptides and could be associated with mechano-sensory responses and touch sensitivity linked to the host invasion.
A regulatory subunit of a cAMP-dependent protein kinase (kin-2, EST code 22H01; see Table 2) was predicted to interact with 72 other genes all involved in diverse cellular processes, such as nuclear trafficking, and DNA replication and repair (see Table S2). Based on gene ontology terms, kin-2 is implicated in gamete generation, growth, larval development, post-embryonic body morphogenesis, signal transduction and/or protein amino acid phosphorylation (see Table S2). Gene silencing of kin-2 in <span class="yellow">C. elegans</span> leads to phenotypes, such as larval lethal (Lvl), larval arrest (Lva), body morphology defect (Bmd), dumpy (Dpy), uncoordinated (Unc) and sterile progeny (Stp) (www.wormbase.org), suggesting that its homologue in <span class="yellow">A. suum</span> is central to larval maturation. The KOBAS analysis predicted the protein KIN-2 to be involved in the insulin-signaling pathway, previously implicated in controlling the exit from dauer in <span class="yellow">C. elegans</span> and the activation of L3s of the <span class="yellow">canine hookworm</span>, <span class="yellow">Ancylostoma caninum</span>, following exsheathment [67]. In a recent study, Brand and Hawdon [68] were able to inhibit (with a phosphoinositide-3-OH-kinase inhibitor) the activation of infective L3s of both of the hookworms <span class="yellow">Ancylostoma caninum</span> and <span class="yellow">Ancylostoma ceylanicum</span> via the insulin signaling pathway, thus lending some credence to the hypothesis that this pathway plays an critical role in regulating the transition from the free-living to the parasitic stage [68]. Recently, it has been proposed that transcriptional and feeding responses to serum-stimulation in <span class="yellow">Ancylostoma caninum</span> are regulated by parallel systems, with the insulin signaling pathway playing a significant role in the ‘resumption of feeding’ in activated larvae [69].
Protein kinases are also likely to be involved in pathways linked to sexual maturation in developing larvae. As already proposed for adult stages of <span class="yellow">H. contortus</span> [45], the protein kinase gene cdk-1 is predicted to play a pivotal role in the germline, oogenesis and spermiogenesis pathways of this parasitic nematode. Other protein kinases, such as PEPCK, and phosphatases, were shown herein to be transcribed at high levels in the L3 stage compared with other developmental stages of <span class="yellow">A. suum</span> (see Table 2), which is in accordance to findings reported recently for <span class="yellow">Anisakis simplex</span> [65]. Due to their major regulatory effects in eukaryotic signaling events and regulatory and sensory functions, protein kinases have been considered interesting targets for anti-parasitic drugs [70].
In conclusion, this study has given some interesting insights into early molecular processes in the L3 of <span class="yellow">A. suum</span>. Approximately 60% of the transcripts enriched in the L3 stage of <span class="yellow">A. suum</span> have homologues/orthologues in <span class="yellow">C. elegans</span>. The bioinformatic analyses of selected molecules suggest that a complex genetic network regulates or controls larval growth and development in <span class="yellow">A. suum</span> L3s, and some of these might be involved in or regulate the switch from the free-living to the parasitic stage. Some caution is warranted in drawing conclusions regarding molecular mechanisms regulating the transition to parasitism in parasitic nematodes from information on <span class="yellow">C. elegans</span>, as latter is a free-living nematode. Also, while the method of data integration is essential for the reliable prediction of genetic interactions, it might limit the capacity of the approach somewhat to infer nematode-specific interactions. As additional datasets of genes and gene functions become available for various parasitic nematodes, more informed inferences can be made regarding the functions of nematode-specific genes, particularly those involved in the transition to parasitism. The imminent genome sequence of <span class="yellow">A. suum</span> (http://www.sanger.ac.uk/Projects/Helminths/) should all assist in this endeavour. Also, functional analysis of selected molecules representing selected ESTs identified herein, utilizing gene silencing approaches established recently [33],[34], could provide some insights into developmental processes in Ascaris and related ascaridoid nematodes and provide avenues for the development of novel approaches for their control.<br><br>Supporting Information<br><br>
<h3>pmcA328328</h3>Mapping irradiation hybrids to cosmid and <span class="blue">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products.
Abstract
A direct hybridization protocol is described for screening cosmid and <span class="blue">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual Alu-PCR product before library screening. A series of <span class="yellow">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis.Images<br><br><br><br><br><br>
 Nucleic Acids Research, Vol. 19, No. 12 3315 <br><br> Mapping irradiation hybrids to cosmid and <span class="blue">yeast</span> artificial chromosome libraries by direct hybridization of Alu-PCR products <br><br> Anthony P.Monaco*, Veronica M.S.Lam2, Gunther Zehetner, Gregory G.Lennon, Christal Douglas, Dean Nizetic, Peter N.Goodfellow1 and Hans Lehrach <br><br> Genome Analysis Laboratory and 'Molecular <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, 44 Lincoln's Inn Fields, London WC2A 3PX, UK and 2Department of Biochemistry, Li Shu Fan Building, Sassoon Road, University of Hong Kong, Hong Kong <br><br> Received March 12, 1991; Revised and Accepted May 16, 1991 <br><br> ABSTRACT <br><br> A direct hybridization protocol is described for screening cosmid and <span class="blue">yeast</span> artificial chromosome libraries with pools of Alu-PCR products from somatic cell or irradiation hybrids. This method eliminates purification, cloning and analysis of each individual AluPCR product before library screening. A series of <span class="yellow">human</span> X chromosome irradiation hybrids were mapped by this method, using a cosmid reference library for comparisons between overlapping hybrids to identify interesting clones for further analysis. <br><br> INTRODUCTION <br><br> The generation of <span class="yellow">human</span> DNA probes specific for individual chromosomes and subregions of chromosomes has been advanced with Alu-sequence primed polymerase chain reaction amplification (Alu-PCR, 1-3). This method specifically amplifies sequences between Alu repeats from <span class="yellow">human</span> DNA in somatic cell hybrids and <span class="blue">yeast</span> artificial chromosomes (YACs, 4). Individual Alu-PCR products can be purified from agarose gels or ligated into plasmid vectors to screen for single copy sequences. Unique Alu-PCR products are then localized to certain chromosome regions using DNA blots of somatic cell hybrid panels. Once localized, Alu-PCR fragments can be screened against genomic libraries to isolate longer DNA fragments from the region of interest. As an alternative to this multistep process we have developed a hybridization protocol for screening of cosmid and YAC libraries directly with pools of Alu-PCR products. <br><br> Two new <span class="yellow">human</span> specific Alu primers were used to generate DNA probes from a series of irradiation-reduced hybrids containing multiple <span class="yellow">human</span> X chromosome fragments of 1-2000 kb on a <span class="yellow">hamster</span> chromosome background (5; P.N.G., unpublished). The Alu-PCR products were hybridized as a pool of probes to X-specific cosmid and YAC libraries, after <br><br> competitive reassociation with an excess of <span class="yellow">human</span> DNA to both the library filters and radioactively labelled Alu-PCR products. Comparisons were made between clones identified by overlapping irradiation hybrids and single copy DNA probes hybridized to the cosmid and YAC libraries. <br><br> METHODS <br><br> Two <span class="yellow">human</span> Alu sequence primers were generated which were shown to be <span class="yellow">human</span> specific; 3144 from the 3' end of Alu: 5'-GAGCGAGACTCCGTCTCAAA-3' and 2729 from the 5' end of Alu: 5'-GTGGATCACCTGAGGTCAGGAGTTC-3'. All PCR reactions were carried out with 100 ng of hybrid DNA and 0.7 ,^g of a single Alu primer in 100 d41 of 0.01 M Tris-HCl pH 8.3, 0.0015 M MgCl2, 0.05 M KCI, 200 AtM each of dNTPs, 10% dimethlysulfoxide, and 2.5 units of Cetus Taq polymerase. Reactions were 30 cycles of 94?C for 2 min, 57?C for 2 min, and 74?C for 4 min followed by a final extension time at 74?C for 9 min. Reactions products were analyzed on 1 % agarose gels and shown to contain between five and twenty fragments, with sizes ranging from 0.1 to 2.0 kb. <span class="yellow">Chinese hamster</span> DNA and no DNA PCR reactions were done to control for non-<span class="yellow">human</span> products (data not shown). <br><br> Alu-PCR products were separated from Alu oligomers over Qiagen columns, and approximately 50-100 ng were labelled by random hexamer priming (6). The radioactively labelled pool of fragments was prehybridized with 37.5 ,^g of total <span class="yellow">human</span> DNA and 12.5 tsg of <span class="yellow">hamster</span> DNA immobilized on a cellulose support matrix, prepared as previously described (7). Reactions were at 65?C in 1 ml of 0.75M NaCl, 0.05M sodium phosphate pH 7.2, 0.005M  EDTA, 0.1 % sodium dodecyl sulphate (SDS), 0.5 mg/ml heparin, and 100 jig/ml denatured <span class="yellow">salmon</span> sperm DNA. The cellulose was pelleted and the supernatant boiled for 2 min every 12-16 hours (three times in two days). <br><br> Cosmid and YAC library filters (Hybond N+, Amersham) were prehybridized at 42?C for 16 hours with 100 jig/ml <br><br> * To whom correspondence should be addressed at <span class="yellow">Human</span> Genetics Laboratory, Imperial Cancer Research Fund, Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford OX3 9DU, UK <br><br> 1991 Oxford University Press <br><br> 3316 Nucleic Acids Research, Vol. 19, No. 12 <br><br> Chromosome X <br><br> .,.., ., < <br><br> _ ., . < ., <br><br> *2.' 1- 7-_ <br><br> _-_ 1. J <br><br> ,.1 .. - 1 , _ i - _ 1 . 1 <br><br> 1 ' 4 - -1 + <br><br> 1 1 L L . i <br><br> . 1 . _ 1  <br><br> 1 1 s. 1 i <br><br> 1 i ;. --r' F 1 -<br><br> 1 -1-l <br><br> j i-_1. 1 <br><br> -   _ 1 .. <br><br> s- S . i n -. <br><br> .. : ,1' ,. -1 _ E I i ? <br><br> ,. . _ _ _ <br><br> G-<br><br> . <br><br> *.-.,., z <br><br> _ <br><br> _ , <br><br> z ;- [X1 <br><br> 21 , In <br><br> 27  38 45   48  54  74   86 <br><br> I  1      I <br><br> I <br><br> I <br><br> 107 <br><br> MD <br><br> I <br><br> :I                                                       I  <br><br> I        I <br><br> I I <br><br> Figur 2a and 2b: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 48 to duplicate copies of 22 x22cm filters containing 9216 <span class="yellow">human</span> X chromosome cosmids (8). 2c: Hybridization of Alu-PCR products generated with Alu primer 3144 from an independent hybrid (54) to a third identical cosmid filter. <br><br> Figure 1: A schematic diagram of the <span class="yellow">human</span> X chromosome alongside the approximate cytogenetic location of fragments identified in nine irradiation hybrids (numbers across the top). The <span class="yellow">human</span> X fragments were identified by hybridization of 27 known DNA markers (indicated by a black line; P.N.G, unpublished) or by cosmids in common with unique X chromosome probes in the reference library database (open boxes and Table 1). The size of the lines and boxes relate to the best cytogenetic location of the probes used according to <span class="yellow">Human</span> Gene Mapping 10.5 (14) and does not indicate the physical extent of the irradiation hybrid fragments. <br><br> denatured and sheared total <span class="yellow">human</span> DNA in 50% formamide, 4XSSC, 0.05 M sodium phosphate pH 7.2, 0.001 M EDTA, 10% dextran sulphate, 1.0% SDS, 50 isg/ml denatured <span class="yellow">salmon</span> sperm DNA and OxDenhardt's solution. The radioactively labelled Alu-PCR products were denatured and added to fresh <br><br> hybridization solution without <span class="yellow">human</span> DNA competitor at 1 x 106 <br><br> cpm/ml and hybridized at 42?C for 16 hours. Filters were washed in 0.1 xSSC and 1.0% SDS, twice at room temperature and twice at 65?C for 30 min each and exposed to Kodak X-OMAT film for 2-3 days at -70?C with an intensifying screen. <br><br> For each hybridization, two sets of duplicate cosmid filters were used from the ICRF reference library system (8), each containing 9216 flow-sorted <span class="yellow">human</span> X chromosome cosmid clones or approximately 2 chromosome equivalents on a 22 x 22 cm filter (9). The coordinates of signals positive on duplicate cosmid filters were entered into the reference library database (G. Z, unpublished) using a digitizing tablet. For the X chromosome specific YAC library (A.P.M. and H.L., unpublished), about 420 YAC colonies were spotted manually onto filters from 96 well microtiter dishes using a 96 prong device. After growth on selective media for 3 days, YAC filters were processed for hybridization as previously described (10). <br><br> RESULTS <br><br> A panel of 195 X chromosome irradiation hybrids was constructed (50,000 rads) and characterized by DNA hybridization using 27 X chromosome markers and flourescence in situ hybridization using total <span class="yellow">human</span> DNA as probe (Benham et al., 1989; P.N.G., unpublished). This analysis indicated that the hybrids contained multiple small fragments (4-10 fragments of 1000-5000 kb each) with a preferential retaining of centromere sequences (90%). From this panel, nine irradiation hybrids were chosen that contained less than five different regions by DNA probe hybridizations, mostly from the short arm of the X chromosome (Fig 1). All nine hybrids were used in PCR reactions with 3'-Alu primer 3144 and two were used with 5'-Alu <br><br> Figure 3: Hybridization of Alu-PCR products generated with Alu primer 3144 from irradiation hybrid 54 to a filter containing 420 YAC clones from the <span class="yellow">human</span> X chromosome. The positive YAC was also identified in a separate hybridization with the DMD probe P20 (12). <br><br> primer 2729. Example hybridizations to a <span class="yellow">human</span> X chromosome cosmid filter in Fig 2 shows the intensity and reliability of positive clones identified on duplicate filters with Alu-PCR products from the same irradiation hybrid (48) and the independence of clones identified with Alu-PCR products from a different hybrid (54). Fig 3 shows a single positive YAC clone after hybridization of Alu-PCR products from irradiation hybrid 54 to a filter containing about 420 YAC clones specific for the <span class="yellow">human</span> X chromosome. <br><br> The total number of cosmids identified with each pool of AluPCR products for each hybrid is shown in Table 1. From the average number of cosmids identified (24) in four chromosome equivalents screened and the estimated average DNA content in each hybrid (3000-15000 kb), the Alu-PCR products generated by a single primer were calculated on average to be 300-1500 kb apart, similar to published estimates for this method (1,2). Only 3-4 cosmids were identified in common using Alu-PCR products generated with either 3' or 5' Alu primers (3144 or 2729) from two hybrids (38 and 45). This shows that separate products were amplified with the two Alu primers since they prime synthesis from opposite ends of the Alu consensus sequence and Alu sequences are oriented in the genome either head to head, <br><br> A <br><br> B <br><br> C <br><br> El <br><br> I0 <br><br> Nucleic Acids Research, Vol. 19, No. 12 3317 <br><br> Table 1. Cosmids identified by hybrids and unique probes <br><br> hybrids       21    27     38    38    45    45    48     54    74    86     107   unique <br><br> primer 3144  3144  3144  2729  3144  2729   3144  3144  3144  33144 3144   probes <br><br> 21     3144  59                                                                   1 27     3144   7    14                                                             2 38     3144   3     1    28                                                       2 38     2729   0     0     4     25                                                0 45     3144  13     2      2     0    32                                          3 45     2729   1     0      0     0     3    31                                    0 48     3144  10     2      4     0     2     0    28                              3 54     3144  11     3     0      0     5     0     5     30                       2 74     3144   5     0     0      0     4     0     5     12    20                 3 86     3144   3     1     0      0     4     1     3      4     3    17           0 107     3144   1     0      1     1     0     0     3     0      0     0    13    0 <br><br> tail to tail or head to tail relative to each other. Therefore, by using the 3'- and 5'-Alu primers in separate PCR reactions with the same hybrid DNA, the total number of products and cosmid clones identified was almost doubled. <br><br> Table 1 also indicates how many cosmids were identified by Alu-PCR products from other hybrids, and 16 cosmids previously identified with unique DNA probes in the reference library database. As can be seen in Fig 1 and previous irradiation hybrid analysis, there is a preferential retention of centromere sequences (2,11). However, there were no cosmids identified in common from all the hybrids positive with centromere sequences. This is probably due to a paucity of Alu repeats in the correct orientation in alphoid centromere sequences and thus few or no Alu-PCR products would be amplified from the centromere. <br><br> Cosmids identified in common by several irradiation hybrids (Table 1) were most likely from regions of overlap outside the centromere area as shown by the previous DNA probe characterization (Fig 1). The overlap regions between hybrids were also seen by 16 cosmids (Table 1, Fig 1) that were hybridization targets of unique DNA markers in the reference library database that mapped in independent experiments to the overlap region. At least for several cosmids this showed that the Alu-PCR products identified target cosmids that were definitely from the expected region contained in the hybrids. For example, hybrids 21 and 54 were both shown to contain part of the Duchenne muscular dystrophy (DMD) locus (Fig 1; P.N.G., unpublished) and had 11 cosmids in common, including one identified by the probe P20 from the deletion hotspot region of the DMD gene (12). In Fig 3 the hybridization of Alu-PCR products from hybrid 54 identified a YAC clone which was also positive for the DMD probe P20 (data not shown). This method also identified fragments in the hybrids that were not seen in the initial DNA characterization (Fig 1 and Table 1). Since the 27 DNA probes were not close enough to each other along the chromosome to detect all possible hybrid fragments (1000-5000 kb), many regions would have been untested. For example, AluPCR products from hybrids 45 and 48 identified several cosmids, also seen by the cDNA for chronic granulomatous disease gene (CYBB, 13) in Xp2 1.1, although this region was not tested in the original hybrid characterization. <br><br> DISCUSSION <br><br> The direct hybridization of Alu-PCR products from somatic cell hybrids to genomic libraries can bypass gel purification or ligation of fragments into plasmid vectors and individual analysis for single copy sequences. Hybridization of Alu-PCR products as <br><br> a pool to ordered array libraries such as the flow-sorted X chromosome cosmid library (9), allows the direct comparison of overlapping hybrids to pinpoint cosmids most likely to be from the region of interest. In conjunction with the reference library database (G.Z, unpublished) with 183 X chromosome probe hybridization entries, cosmids identified with both Alu-PCR products and uniquely mapped X probes immediately map them to the region of interest and proove that the method has worked. Similar hybridization experiments using Alu-PCR products from four overlapping irradiation hybrids identified four cosmids in common that mapped to the region of overlap by independent experiments (F.Muscatelli, A.P.M., P.N.G., H.L. and M.Fontes, in preparation). Since only 27 probes from the X chromosome were used to initially characterize the hybrids and the length of individual <span class="yellow">human</span> fragments in the irradiation hybrids is about 1000-5000 kb, many regions could have been undetected in the original analysis. The direct hybridization of Alu-PCR products to the cosmid reference library detected such fragments since they identified cosmids in common with uniquely mapped probes in regions not tested originally (Table 1 and Fig 1). This should prove to be a sensitive and efficient method to determine content and overlap of irradiation hybrids in conjunction with DNA blot hybridization. However, since the exact length of the <span class="yellow">human</span> DNA fragments for each hybrid and the spacing of Alu-PCR products along the chromosome is not known, it is difficult to directly correlate the number of target cosmids to the DNA content of the hybrids. <br><br> The direct hybridization of Alu-PCR products from irradiation or somatic cell hybrids to total genomic YAC libraries will be especially useful to construct long range YAC contigs from specific subregions of chromosomes. The dissection of a total genomic YAC library by this method may be more efficient than generating chromosome specific YAC libraries from somatic cell hybrids (usually a haploid <span class="yellow">human</span> chromosome on a diploid or greater <span class="yellow">rodent</span> background) or flow-sorted chromosomes, because of the low transformation efficiency of <span class="blue">yeast</span>. <br><br> ACKNOWLEDGEMENTS <br><br> We would like to thank Gert-Jan Van Ommen for the probe P20 and Stuart Orkin for the CYBB cDNA probe. A.P.M is supported in part by a research fellowship from the Muscular Dystrophy Association of America. V.M.S.L. is supported in part by the Medical Research Grant of the University of Hong Kong. Reference library filters and cosmids identified by Alu-PCR products can be requested from the Reference Library Database, ICRF, 44 Lincoln's Inn Fields, London WC2A 3PX, U.K. <br><br> 3318 Nucleic Acids Research, Vol. 19, No. 12 <br><br> REFERENCES <br><br> 1. Nelson, D.L., Ledbetter, S.A., Corbo, L., Victoria, M.F., Ramirez-Solis, <br><br> R., Webster, T.D., Ledbetter, D.H., and Caskey, C.T. (1989) Proc. Natl. Acad. Sci. USA, 86:6686-6690. <br><br> 2. Ledbetter, S.A., Nelson, D.L., Warren, S.T., and Ledbetter, D.H. (1990) <br><br> Genomics, 6:475-481. <br><br> 3. Cotter, F.E., Hampton, G.M., Nasipuri, S., Bodmer, W.F., and Young <br><br> B.D. (1990) Genomics, 7:257-263. <br><br> 4. Burke, D.T., Carle, G.F., and Olson, M.V. (1987) Science, 236:806-812. 5. Benham, F., Hart, K., Crolla, J., Bobrow, M., Francavilla, M., and <br><br> Goodfellow, P.N. (1989) Genomics, 4:509-517. <br><br> 6. Feinberg, A.P., and Vogelstein, B. (1984) Anal. Biochem., 137:266-267. 7. Hochgeschwender, U., Sutciffe, J.G., and Brennan, M.B. (1989) Proc. Natl. <br><br> Acad. Sci. USA, 86:8482-8486. <br><br> 8. Lehrach, H., Drmanac, R., Hoheisel, J., Larin, Z., Lennon, G., Monaco, <br><br> A.P., Nizetic, D., Zehetner, G., and Poustka, A. (1990) In Davies, K.E. & Tilghman, S.M. (eds.), Genome Analysis Volume 1: Genetic and Physical Mapping. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, pp. 39-81. <br><br> 9. Nizetic, D., Zehetner, G., Monaco, A.P., Gellen, L., Young, B.D., and <br><br> Lehrach, H. (1991) Proc. Natl. Acad. Sci. USA, 88:3233-3237. <br><br> 10. Larin, Z. and Lehrach, H. (1990) Genet. Res. Camb., 56:203-208. <br><br> 11. Cox, D,R., Pritchard, C.A., Uglum, E., Casher, D., Koborl. J. and Myers, <br><br> R.M. (1989) Genomics, 4:397-407. <br><br> 12. Wapenaar, M.C., Kievits, T., Hart, K.A., Abbs S., Blonden, L.A.J., <br><br> denDunnen, J.T., Grootscholten, P.M., Bakker, E., Verellen-Dumoulin, C., Bobrow, M., vanOmmen, G.J.B., and Pearson, P.L. (1988) Genomics, 2:101-108. <br><br> 13. Royer-Pokora, B., Kunkel, L.M., Monaco, A.P., Goff, S.C., Newburger, <br><br> P.E., Baehner, R.L., Cole F.S., Curnette J.T., and Orkin, S.A. (1986) Nature, 322: 32-38. <br><br> 14. Davies, K.E., Mandel, J.L., Monaco, A.P., Nussbaum, R.L. and Willard, <br><br> H.F. (1990) Cytogenet. Cell Genet. 55: 254-313. <h3>pmcA1312363</h3>Gene expression studies in isolated mitochondria: <span class="yellow">Solanum tuberosum</span> rps10 is recognized by cognate <span class="yellow">potato</span> but not by the transcription, splicing and editing machinery of <span class="yellow">wheat</span> mitochondria
Abstract
The complex gene expression mechanisms that occur in plant mitochondria, such as RNA editing and splicing, are not yet well understood. RNA editing in higher plant mitochondria is a highly specific process which modifies mRNA sequences by C-to-U conversions. It has been suggested that in some cases this process is required for splicing. Here, we use an experimental model based on the introduction of DNA into isolated mitochondria by electroporation to study organellar gene expression events. Our aim was to compare processing and editing of <span class="yellow">potato</span> small ribosomal protein 10 gene (rps10) transcripts in heterologous (<span class="yellow">wheat</span> mitochondria) and homologous (<span class="yellow">potato</span> mitochondria) contexts. rps10 is a suitable model because it contains a group II intron, is absent in <span class="yellow">wheat</span> mitochondria but is actively expressed in <span class="yellow">potato</span> mitochondria, where transcripts are spliced and undergo five C-to-U editing events. For this purpose, conditions for electroporating isolated <span class="yellow">potato</span> mitochondria were established. rps10 was placed under the control of either <span class="yellow">potato</span> or <span class="yellow">wheat</span> cox2 promoters. We found that rps10 was only transcribed under the control of a cognate promoter. In <span class="yellow">wheat</span> mitochondria, rps10 transcripts were neither spliced nor edited while they are correctly processed in <span class="yellow">potato</span> mitochondria. Interestingly, a <span class="yellow">wheat</span> editing site grafted into rps10 was not recognized by <span class="yellow">wheat</span> mitochondria but was correctly edited in <span class="yellow">potato</span> mitochondria. Taken together, these results suggest that editing might occur only when the transcripts are engaged in processing and that they would not be available to editing factors outside of a putative RNA maturation machinery complex.<br><br>INTRODUCTION
Gene expression in plant mitochondria is a complex process involving multiple steps such as transcription, cis- and trans-splicing, RNA trimming and RNA editing on the way to translation (1). RNA editing has been found in a variety of organisms and occurs through different mechanisms such as insertion or deletion of nucleotides, or base conversions [for details see reference (2)]. In higher plant organelles, RNA editing is an important post-transcriptional event characterized by C-to-U changes via a deamination mechanism (3–5). In <span class="yellow">Arabidopsis thaliana</span> 456 C-to-U editing events have been described (6). RNA editing occurs mostly in the coding regions which alter the identity of the encoded amino acid, but some editing events occur in highly structured regions of introns (7,8). Like other maturation processes, RNA editing is an essential post-transcriptional event in plant mitochondrial gene expression (9–11) required for the synthesis of functional proteins (12,13).
The introduction of foreign DNA into isolated mitochondria is a novel experimental model which has provided important information on RNA editing and RNA splicing enabling the use of a site-directed mutagenesis approach (14,15). Using a cognate cox2 chimeric gene construct, the cis-recognition elements required for plant mitochondria RNA editing have been determined (16,17). Using the same approach, Staudinger and Kempken (18) have reported that transcripts from <span class="yellow">A.thaliana</span> cox2, but not <span class="yellow">Sorghum bicolor</span> atp6, are edited when the genes are introduced into <span class="yellow">maize</span> mitochondria.
Interestingly, higher plant mitochondrial genomes differ in their gene contents due to an evolutionary information transfer from the organelle to the nucleus (19). Of particular interest is the situation of the small ribosomal protein 10 gene (rps10), a group II intron-bearing gene which is encoded in <span class="yellow">Solanum tuberosum</span> mitochondrial DNA but is absent from the <span class="yellow">wheat</span> mitochondrial genome (20). Higher plant mitochondria contain group II introns either in cis- or trans- configuration (1) which can be folded in a characteristic secondary structure (21). Intron removal in plant mitochondrial mRNAs is not well documented because such introns are unable to self-splice. Previously, we described that electroporated cox2 constructs were a good model for the study of the splicing process. Using this model, we found that editing and splicing of cox2 transcripts were not linked in <span class="yellow">wheat</span> mitochondria (15).
To challenge the ability of mitochondrial gene expression machinery to recognize genetic information which has been lost during evolution, we decided to introduce the <span class="yellow">S.tuberosum</span> non-cognate rps10 gene into <span class="yellow">wheat</span> mitochondria. Five C residues are changed to U in <span class="yellow">potato</span> mitochondria rps10 transcripts by editing. Two of them have been postulated as being necessary for acquisition of a proper secondary and/or tertiary structure for splicing. To test this hypothesis, it was necessary to set up the conditions for electroporation of foreign DNA into <span class="yellow">S.tuberosum</span> mitochondria. Here, we show that a <span class="yellow">potato</span> rps10 construct is transcribed when introduced into <span class="yellow">wheat</span> mitochondria, but transcripts are not recognized by the post-transcriptional processing machinery. In contrast, a rps10 construct is correctly expressed and processed in cognate <span class="yellow">potato</span> mitochondria. Moreover, we present evidence that transcript editing might be linked to overall RNA processing. This is the first report on DNA electroporation into <span class="yellow">potato</span> mitochondria.<br><br>MATERIALS AND METHODS
Plasmids
All plasmids used in this study are based on the previously described pCOXII vector (14). An NsiI restriction site was introduced at the initiation codon of the <span class="yellow">wheat</span> cox2 open reading frame (ORF). Then, NsiI was used in combination with a SpeI restriction site present in the original vector after the stop codon, to produce the chimeric vectors.
<span class="yellow">S.tuberosum</span> cox2 gene, including a 727 bp non-coding upstream region, was isolated by PCR from total DNA using the primer A designed from partial sequences reported by Löessl et al. (22), accession no. AF096321, containing the KpnI restriction sequence, and primer B derived from <span class="yellow">Triticum timopheevi</span> cox2 (AF336134). A fragment of 2888 bp was obtained and cloned on pGEM-T vector. The complete sequence of the <span class="yellow">S.tuberosum</span> cox2 was determined (accession no. DQ18064). To generate the plasmid pCOXIISt containing the <span class="yellow">S.tuberosum</span> cox2 gene, a KpnI/SpeI fragment containing the 727 bp upstream region and the complete coding region was used to replace the <span class="yellow">wheat</span> gene from pCOXII. As for pCOXII, a NsiI site was inserted at the ATG start codon and a 23 bp fragment was inserted at position −20. This 23 bp insertion provides a specific sequence allowing isolation of <span class="yellow">potato</span> cox2 transcripts originating from the introduced DNA by RT–PCR. The chimeric <span class="yellow">wheat</span>/<span class="yellow">potato</span> cox2 gene was constructed by replacing 727 bp KpnI/NsiI promoter sequence from pCOXIISt with the 880 bp <span class="yellow">wheat</span> upstream region from pCOXII.
The vectors pRPS10W and the pRPS10St derivative, containing <span class="yellow">wheat</span> and <span class="yellow">potato</span> cox2 promoters, respectively, were constructed by inserting the 1178 bp rps10 sequence containing two exons separated by a 777 bp intronic sequence [(23), accession no. X74826]. The coding region was isolated from total <span class="yellow">S.tuberosum</span> DNA by PCR using primers D1 and D2 containing the restriction sites NsiI and SpeI, respectively. The fragment NsiI/SpeI was purified and cloned into pCOXII and pCOXIISt, replacing the respective cox2 coding regions.
Since all vectors used here were based on pCOXII, they contain the downstream region from the <span class="yellow">wheat</span> cob gene (Ir-cob) (accession no. AF337547) (14). This sequence, combined with the 23 bp upstream insert sequence served to distinguish, using primers 1a and 1b, foreign from endogenous transcripts. All mutants were obtained using QuickChange® Site-Directed Mutagenesis kit (Stratagene). PCR product purifications were carried out with the Wizard® Clean-Up System (Promega).<br><br>PCR primers used
Oligonucleotides sequences are in 5′ to 3′ orientation. (1a), GCGGTGCAGTCATACAGATCTGC; (1b), TATCCAGATTTGGTACCAAAC; (2a), GCAGTCATACAGATCTGCCAAAG; (2b), AGATTTGGTACCAAACCCGGG; (A), TATAGGTACCTCTCAGGTGTCAAAGTGTGGATTT; (B), TATAACTAGTTTAAGCTTCCCCG; (D1), TATAATGCATAGACAAAGGAGAGCACTTA; (D2), TATAACTAGTTCAGGAAAGGGTCAACGCAA. Restriction sites are underlined.<br><br>Mutagenesis primers (only sense primers in 5′–3′ orientation are indicated)
Single C2, C3 and C2+C3 double mutant plasmids were constructed with primers (C2) AAGAAGTTCTTTTGGTTAAAACGCC and (C3) CGCCGTGCGACTTGGAGGACATAAG. NsiI-pCOXII: GGAAATCCAATGCATCTTCGTTCATT and NsiI-pCOXIISt: CCAAACCAAATGCATGTTCTAGAATG. Construct pCOXIISt containing a 23 nt insertion in the promoter region, was carried out into two consecutive insertions using primers: TGGGGGGAGCAGAGCAGTGCGGTGCAGTCACAAAGAATGAACCAAACC and GCAGTGCGGTGCAGTCATACAGATCTGCCAAAGAATGAACCAAACC. For constructs MA and MAB containing the <span class="yellow">wheat</span> cox2 C259 editing site, two consecutive insertions were carried out using primers: GGAAGATTGGATTACTATCGAAATTGCCCTGAATCA and TACTATCGAAATTATTCGGACCATGCCCTGAATCA. For ME6 and ME6b constructs, primers CCGCGAGGAATCAACTACTATCGAAATTATTGCCGGTGCTGAC and CAACTACTATCGAAATTATTCGGACCATATTGCCGGTGCTGAC were used. Inserted nucleotides are underlined and the modified residues are indicated in bold.<br><br>Mitochondria purification
<span class="yellow">S.tuberosum</span> cv. Rosevalt tubers and <span class="yellow">T.aestivum var Fortal</span> seeds were used. <span class="yellow">Potato</span> mitochondria were prepared from 2 kg of tubers in batches of 200 g with 200 ml of a homogenization buffer containing 0.4 M mannitol, 25 mM MOPS (pH 7.8), 1 mM EGTA, 8 mM cysteine and 1 mg/ml fatty acid-free BSA. Homogenization was carried out for 15 s in a Waring blendor at full speed. Homogenate was centrifuged in a Sorvall GSA rotor at 1500 g for 10 min at 4°C. Supernatant was centrifuged in a GSA rotor at 12 000 g for 15 min. The mitochondrial pellet was resuspended in 50 ml of homogenization buffer and centrifuged at 1500 g. The supernatant was centrifuged in a Sorvall SS-34 rotor at 15 000 g for 10 min, the pellet was resuspended in 12 ml of homogenization buffer and mitochondria were purified by centrifugation on a sucrose gradient essentially as described for <span class="yellow">wheat</span> embryo mitochondria (14).<br><br>Electroporation
Electrotransfer experiments were carried out with 1 mg of purified <span class="yellow">wheat</span> embryo or <span class="yellow">potato</span> tuber mitochondria in 50 µl of 0.33 M sucrose and 1 µg of recombinant plasmid as described (14). The electroporated mitochondria were incubated for 18 h at 25°C, then recovered by centrifugation in a Sigma N° 12024 rotor (Sigma 1K15 refrigerated centrifuge) at 15 000 g for 15 min at 4°C. In the case of <span class="yellow">potato</span> mitochondria the incubation mixture was supplemented with 1 mg/ml of fatty acid-free BSA. RNA was purified with 200 µl of TRIzol™ reagent (Gibco-BRL) according to the supplier's protocol.<br><br>DNAse I protection assay and DNA purification
After electroporation and centrifugation, the mitochondrial pellet was resuspended in 100 µl of buffer [10 mM Tris–HCl (pH 7.5), 2 mM magnesium acetate and 0.33 M sucrose] containing 60 U of DNase I (Gibco-BRL) and incubated for 1 h at room temperature. The DNase reaction was stopped by adding 4 µl of 0.5 M EDTA and then heating for 10 min at 65°C. Mitochondria were incubated with 100 µg of Proteinase K (Merck) for 4 h at 37°C. One microliter of 20% SDS was added to achieve mitochondrial lysis and the DNA was extracted with phenol/chloroform, precipitated with 0.1 vol of 3 M Sodium Acetate (pH 5.2), 3 vol of 100% ethanol and 100 ng of carrier <span class="blue">yeast</span> tRNA and left overnight at −20°C. After centrifugation, the DNA pellet was resuspended in TE buffer [10 mM Tris–HCl (pH 8) and 1 mM EDTA].<br><br>RT–PCR
RNA (1 µg) was treated with 2 U of Amplification grade DNase I (Promega). cDNA synthesis was performed with 200 U of Superscript II RT using 100 ng of random hexamers. The PCR were performed with primers 1a and 1b using Advantage® 2 Polymerase Mix (Clontech) as follows: 95°C, 1 min; 5 cycles at 95°C for 30 s and 68°C for 1 min; 30 cycles at 95°C for 30 s, 58°C for 30 s and 68°C for 30 s, and finally 68°C for 1 min. Primers 2a and 2b were used for nested PCR from 1 µl of the first PCR. No RT–PCR amplification products were obtained with RNA from non-electroporated mitochondria.<br><br>DNA sequencing
Sequence analyses were performed directly on the RT–PCR product using an automatic DNA sequencing equipment with the BigDye® Terminator Cycle Sequencing Kit (Applied Biosystem).<br><br>
RESULTS
<span class="yellow">S.tuberosum</span> rps10 transcripts are not processed in <span class="yellow">wheat</span> mitochondria
The <span class="yellow">S.tuberosum</span> rps10 construct under control of a <span class="yellow">wheat</span> cox2 promoter (Figure 1A) was incorporated into purified <span class="yellow">wheat</span> mitochondria by electroporation as indicated in Materials and Methods. After electroporation and incubation in expression medium, mitochondrial RNA was extracted and analysed by RT–PCR. The primers used allowed detection of the transcripts generated by the constructs introduced and excluded any product from endogenous cox2 (or rps10 in the <span class="yellow">potato</span> system). Two bands of 1204 and 427 bp were expected from precursor and mature rps10 mRNAs, respectively. Only precursor rps10 molecules were detected (Figure 1B). As a control, a cognate cox2 construct (16) was used. In this case, the 2048 and 827 bp RT–PCR bands representing the precursor and spliced cox2 products, respectively were observed (Figure 1B). In all cases, the PCR bands actually represent transcription products since when PCR was performed without the RT step we observed no amplification products.
Previously, five C residues, two in exon 1, one in the intron and two in exon 2 have been reported to be changed to U by editing in <span class="yellow">potato</span> mitochondria (23). To determine if the non-cognate transcript could be recognized by the <span class="yellow">wheat</span> RNA editing machinery, the 1204 bp RT–PCR product was sequenced. While <span class="yellow">wheat</span> cox2 editing sites were correctly edited in control as expected (Figure 1C, only site C77 is shown), all five editable residues in <span class="yellow">potato</span> rps10 transcript remain unchanged. cox2 C77 editing sites are identical in <span class="yellow">potato</span> and <span class="yellow">wheat</span>.<br><br>Failure of rps10 transcript splicing in <span class="yellow">wheat</span> mitochondria is not linked to the absence of editing
It has been suggested that residues C2 and C3 participate in the secondary structure of the intron necessary for splicing (23). A possible explanation to the failure observed in precursor rps10 splicing could be that the absence of edition of C2 and C3 prevent the intron from organizing itself in a catalytically competent conformation. To test this hypothesis, single C2 and C3 mutants and a C2+C3 double mutant were constructed by changing the C residues to T. Neither single nor double mutants were able to undergo splicing (Figure 2).<br><br>Electroporation of <span class="yellow">S.tuberosum</span> mitochondria
Since two important post-transcriptional processes, RNA editing and splicing, were inoperative when <span class="yellow">S.tuberosum</span> rps10 was expressed in <span class="yellow">wheat</span> mitochondria, we decided to verify whether the negative results were inherent to the rps10 chimeric constructs or due to the lack of trans-recognition elements in <span class="yellow">wheat</span> mitochondria. For this purpose, it was necessary to set up an electroporation protocol adapted to <span class="yellow">S.tuberosum</span> mitochondria. Purified organelles were prepared from <span class="yellow">potato</span> tubers as indicated in Materials and Methods. Electric pulses in the range between 8 and 20 kV were performed. Internalization of exogenous DNA was measured by DNAse protection assays (14). <span class="yellow">Potato</span> mitochondria show a broad range response with a maximum around 13 kV (Figure 3). This voltage setting was therefore used for further experiments.<br><br><span class="yellow">S.tuberosum</span> mitochondria does not recognize the <span class="yellow">wheat</span> cox2 promoter
To ascertain the ability of electroporated mitochondria to perform expression of the exogenous gene construct, we used a plasmid containing the <span class="yellow">potato</span> cox2 gene controlled either by <span class="yellow">T.aestivum</span> or <span class="yellow">S.tuberosum</span> promoters. As shown in Figure 4, the construct containing the <span class="yellow">wheat</span> or <span class="yellow">potato</span> promoters was transcribed only in cognate mitochondria. In <span class="yellow">potato</span>, the mature product is barely detectable. In contrast to <span class="yellow">wheat</span> mitochondria, the 821 nt mature transcript was only detected when the electroporated <span class="yellow">potato</span> mitochondria were incubated in the presence of fatty acid-free BSA.<br><br>A cognate rps10 construct is correctly expressed, edited and processed in <span class="yellow">potato</span> mitochondria
The construct expressing <span class="yellow">potato</span> rps10 under the control of <span class="yellow">potato</span> cox2 promoter was introduced into <span class="yellow">S.tuberosum</span> isolated mitochondria. After incubation, the precursor and mature transcripts were amplified by RT–PCR (Figure 5A) and sequenced. As shown in Figure 5B, the four editing sites C1, C2, C4 and C5, described previously in endogenous transcripts, were found edited in mature mRNA. The presence of spliced molecules containing unedited residues (Figure 5B) indicates that rps10 transcripts could be spliced before editing. To confirm this observation, the PCR product was cloned and sequenced. One half of the individual clones were found edited at sites C1 and C2; 65% were edited at site C4 and 25% were edited at site C5.<br><br>A cognate editing site in a non-native context is not recognized by <span class="yellow">wheat</span> mitochondria but is recognized in heterologous mitochondria
To test whether <span class="yellow">wheat</span> mitochondria are able to edit a cognate site when placed in the context of a <span class="yellow">potato</span> transcript, we introduced the C259 −16/+6 region from <span class="yellow">wheat</span> cox2 into <span class="yellow">potato</span> rps10 exon 1 or intron (Figure 6A, construct MA and ME6, respectively). As reported previously, the 23 nt region forming the C259 editing site from <span class="yellow">wheat</span> cox2 was efficiently edited when grafted into a different context in a <span class="yellow">wheat</span> transcript (17). Wild-type rps10 (pRPS10W) and the MA and ME6 mutants were expressed in <span class="yellow">wheat</span> mitochondria but no splicing was observed (Figure 6B). Unexpectedly, <span class="yellow">wheat</span> mitochondria appear to be unable to edit the cognate C259 site when the −16/+6 region is located on a <span class="yellow">potato</span> rps10 precursor. The control CM construct, containing the C259 site grafted in a different context but in its own cox2 gene, was expressed, spliced and edited as expected [(17), Figure 6B and D]. To compare the editing status of RNAs at the same stage of processing, the editing status of the inserted C259 site in precursor cox2 transcripts is shown (Figure 6D). The analogous MAb and ME6b constructs, containing the <span class="yellow">S.tuberosum</span> cox2 promoter and the C259 region inserted into rps10 exon 1 and intron were used to electroporate <span class="yellow">potato</span> mitochondria. As shown in Figure 6C, MAb and ME6b were expressed and spliced at the same level as the control in the homologous context, indicating that insertion does not affect the splicing process. Interestingly, while C259 insertion was not recognized in <span class="yellow">wheat</span> mitochondria (Figure 6D), the <span class="yellow">potato</span> RNA editing machinery edited the inserted C259 region (Figure 6E). The C259 region inserted in the rps10 intron (construct ME6b) was also edited, although at a very low level, showing that editing may precede intron removal (not shown).<br><br>
DISCUSSION
Plant mitochondria undergo complex expression mechanisms which are poorly understood. Most studies are based on analysis of in vivo mature or intermediate gene products giving clues on possible mechanisms and suggesting pathways operating in gene expression. Direct tests using in vitro approaches have been fruitful (4,5,24), but are hampered by the difficulty of obtaining active mitochondrial extracts. Previously, we devised an experimental model based on electroporation of isolated mitochondria that allows us to test gene expression of wild-type and mutant genes. This approach has been very useful in elucidating the process of RNA maturation in plant mitochondria, in particular splicing and editing (14,15). The aim of this work was to analyse different gene expression events when a foreign gene was expressed in a heterologous context. rps10 was chosen precisely because this genetic information is lacking in <span class="yellow">wheat</span> mitochondria but is active in <span class="yellow">potato</span> mitochondria (20,23,25). Moreover, rps10 transcripts undergo five C-to-U editing events in <span class="yellow">potato</span> mitochondria and rps10 contains an intron, thus facilitating the analysis of post-transcriptional processing.
To best evaluate the expression of rps10 in the non-cognate mitochondria, it was necessary to set up electroporation conditions for introducing foreign DNA into <span class="yellow">S.tuberosum</span> mitochondria. <span class="yellow">Potato</span> tuber mitochondria were able to incorporate DNA essentially under the same conditions described for <span class="yellow">wheat</span> embryo organelles (14), except that the optimal voltage range was larger. A major difference to <span class="yellow">wheat</span> was that isolated <span class="yellow">potato</span> mitochondria were viable for 3–4 h as measured by oxygen consumption (data not shown). Thus, to observe transcript maturation after electroporation the incubation mixture needed to be supplemented with fatty acid-free BSA (see Figure 4). This behaviour probably reflects the uncoupling of oxidative phosphorylation by fatty acids during incubation of <span class="yellow">potato</span> mitochondria (26). In fact, previously we found that proper gene expression in isolated mitochondria requires a functional organelle able to generate ATP from ADP and succinate (14).
An interesting observation was that neither the <span class="yellow">wheat</span> nor the <span class="yellow">potato</span> cox2 promoters were recognized in a heterologous context (Figure 4A and B). This situation might by explained by the fact that <span class="yellow">potato</span> and <span class="yellow">wheat</span> promoter sequences have no recognizable homologous motifs. Plant mitochondria promoters are characterized by a conserved core CRTA sequence with differences in the extent and the composition of sequences around the consensus motif (27,28). While the transcription initiation site in <span class="yellow">wheat</span> mitochondria is located at position −170 (29), the <span class="yellow">potato</span> promoter has not yet been described. Transcription may be initiated at numerous sites suggesting a relaxed promoter recognition by the transcription machinery (28). However, the lack of crossed recognition of cox2 promoters in <span class="yellow">wheat</span> and <span class="yellow">potato</span> is not a general situation since the <span class="yellow">A.thaliana</span> cox2 gene is expressed and spliced when introduced into <span class="yellow">maize</span> mitochondria, indicating that the <span class="yellow">Arabidopsis</span> gene shares some signals with <span class="yellow">maize</span> cox2 promoter that are sufficient for transcription (18). The sites required for transcript initiation have been recently described in <span class="yellow">A.thaliana</span> mitochondrial genes (28). Three regions were described as important for transcription initiation. We found no such homologous sequences in the <span class="yellow">potato</span> cox2 upstream region indicating that the two dicot promoters do not have the same origin. This in turn may reflect the natural history of this particular mitochondrial gene evolving in its own context. It should be mentioned that the presence of a conserved sequence is not sufficient for expression since a region that acts as promoter in <span class="yellow">Arabidopsis</span>, <span class="yellow">potato</span> and Oenothera is inactive in vivo in <span class="yellow">pea</span> (30). Further studies will be required to understand the transcriptional events in plant mitochondria. Electroporation of foreign DNA into isolated mitochondria provides an interesting functional model, complementary to in vitro transcription assay, for answering these questions.
The <span class="yellow">potato</span> rps10 gene controlled by a <span class="yellow">T.aestivum</span> promoter (Figure 1A) was transcribed as a 1204 nt precursor in <span class="yellow">wheat</span> mitochondria, but no traces of mature RNA were observed. Moreover, the five C residues reported as RNA editing targets in vivo (23) remain unchanged, indicating that rps10 transcript was not recognized by the <span class="yellow">wheat</span> splicing and editing machinery. A control using the cognate cox2 construct demonstrates that electroporated organelles were competent for splicing and editing (Figure 1B and C).
Some editing events occur in highly structured domains of introns. Because in some cases, editing corrects A-C mispairing improving conformation of the intron, it has been proposed that the C-to-U change might be necessary for efficient splicing (8,9,23,31). Based on the canonical structure of group II mitochondrial introns (21), two editing sites, C2 located at the Intron Binding Site 2 (IBS2) and C3 located in the intron, nine residues downstream from the end of exon 1 in <span class="yellow">S.tuberosum</span> rps10 are of particular interest. It has been predicted that both edited residues participate in base-pair interactions in the putative secondary structure. Of particular interest is the site C3 located in intron domain I (23). This position may be crucial for splicing as inferred from mutants of a <span class="blue">yeast</span> mitochondrial intron (32). We addressed this question by introducing rps10 mutant genes presenting C2, C3 or both positions in the edited form into mitochondria. As shown in Figure 2, transcription was as efficient as for the cognate cox2 construct but neither wild-type rps10 nor the C-U mutants underwent splicing. These results clearly demonstrate that C2 and C3 editing is not sufficient for splicing of rps10 precursor in <span class="yellow">wheat</span> mitochondria and lead us to conclude that splicing failure is probably linked to the lack of trans-recognition elements. One may speculate that these trans-acting factors, for instance nuclear-encoded maturases, were lost after rps10 was transferred to the nucleus in monocots (20,25). The C-to-U mutants were also tested in <span class="yellow">potato</span> mitochondria; no significant differences were detected in splicing efficiency when compared to the unedited construct (D. Choury, unpublished data). It should be noted that the lack of splicing in <span class="yellow">wheat</span> mitochondria is not a general feature of <span class="yellow">potato</span> introns, since the <span class="yellow">potato</span> cox2 intron is removed efficiently (Figure 4B). This is consistent with the hypothesis that <span class="yellow">potato</span> and <span class="yellow">wheat</span> mitochondria have similar trans-acting factors for cox2 intron removal.
A possible link between intron removal and editing has been proposed for splicing of nad1e and nad5III trans-introns where domain six may require editing to be structured in a catalytic competent secondary conformation (8). In <span class="yellow">rice</span>, failure of RNA maturation and editing has been correlated with cytoplasmic male sterile phenotype. In this model, the absence of the nuclear gene Rf-1 affects B-atp6 RNA cleavage and editing (33). In organello studies have shown that <span class="yellow">S.bicolor</span> atp6-1 gene was transcribed but not edited when introduced into <span class="yellow">maize</span> mitochondria (18). However, partially edited molecules were detected in <span class="yellow">sorghum</span>-<span class="yellow">maize</span> atp6 chimeric transcripts when they included the <span class="yellow">maize</span> atp6 5′-untranslated sequence, suggesting that the 5′ non-coding region provides a structural motif or binding site for a transcript-specific editing factor (34). Unfortunately, no data on atp6 processing was reported to determine whether this region is directly involved in editing or some other maturation event.
Since <span class="yellow">wheat</span> mitochondria do not have rps10 encoded in the mitochondrial genome (20), it was interesting to test whether <span class="yellow">wheat</span> mitochondria were able to recognize editing sites which had been lost during evolution. In other words, whether the mitochondrial trans-recognition elements are specific for each site or whether they are operating on a subset of editing sites. This question is important since to date the factors responsible for RNA editing in plant mitochondria remain unknown. Solving this issue may provide clues to uncover such factors. As described above, the rps10 transcript was not processed in <span class="yellow">wheat</span> but it was correctly spliced in cognate mitochondria (Figure 5A). Indeed, the mature transcript had identical exon1 and exon2 junction as found in endogenous rps10 mRNA (data not shown). More importantly, all editing sites, C1, C2, C4 and C5 were significantly converted to Us (Figure 5B). The fact that the four editing sites were found either edited or unedited in spliced rps10 mRNA is an indication that editing does not precede splicing as was previously found for cox2 in <span class="yellow">wheat</span> (15).
It may be argued that the absence of splicing precluded editing. This possibility can be discarded since <span class="yellow">potato</span> rps10 precursor mRNA was found to be edited in vivo (23). These data clearly show that splicing is not required for rps10 editing, similar to previous findings for cox2 mRNA and also in cox2 mutant derivatives unable to remove the intron (15). This led us to postulate that the inability of <span class="yellow">wheat</span> mitochondria to recognize rps10 editing sites is likely due to the fact that <span class="yellow">wheat</span> mitochondria have lost the editing trans-recognition elements which become dispensable after transfer of rps10 to the nucleus. To test this possibility, rps10 chimeric plasmids containing editing site C259 from <span class="yellow">wheat</span> cox2 inserted either in exon1 or the intron were constructed. Site C259 is formed by 23 nt corresponding to the −16/+6 sequence embedding the target C. Previously we found that this small region could be recognized by the RNA editing machinery when placed outside of its natural context (16,17). The <span class="yellow">wheat</span> C259 editing site in the chimeric construct was correctly edited by <span class="yellow">potato</span> mitochondria. This result is not unexpected since the corresponding region in endogenous <span class="yellow">potato</span> cox2 mRNA, which presents two differences at positions −3 (C instead of A) and −7 (G instead of A), is edited. Furthermore, these positions were not crucial for editing of C259 (17). Surprisingly, the C259 editing site grafted in rps10 was not recognized by the <span class="yellow">wheat</span> editing machinery. We cannot exclude the possibility that editing efficiency in precursor molecules is very low and so is undetectable when sequencing RT–PCR products representing a pool of transcripts. However, analysis of the same region in cox2 precursors indicates that this was not the case since significant C-to-U conversion was observed (Figure 6D). These observations lead us to postulate that editing is occurring only when the transcript is engaged in post-transcriptional processing, suggesting that rps10 transcripts are not available to editing factors independently of the RNA maturation machinery. The fact that the <span class="yellow">wheat</span> editing site C259 inserted in chimeric rps10 transcripts was not recognized by <span class="yellow">wheat</span> mitochondria is a strong argument for this hypothesis. One might speculate that in plant mitochondria, transcripts have to be engaged in a kind of multiprotein processing complex. A failure to be recognized at some early stage will lead to their accumulation as unmodified precursors.<br><br>
<h3>pmcA150219</h3>Rap1p and other transcriptional regulators can function in defining distinct domains of gene expression
Abstract
Barrier elements that are able to block the propagation of transcriptional silencing in <span class="blue">yeast</span> are functionally similar to chromatin boundary/insulator elements in metazoans that delimit functional chromosomal domains. We show that the upstream activating sequences of many highly expressed ribosome protein genes and glycolytic genes exhibit barrier activity. Analyses of these barriers indicate that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p may participate in barrier function. We also present evidence suggesting that Rap1p is directly involved in barrier activity, and its barrier function correlates with local changes in chromatin structure. We further demonstrate that tethering the transcriptional activation domain of Rap1p to DNA is sufficient to recapitulate barrier activity. Moreover, targeting the activation domain of Adr1p or Gcn4p also establishes a barrier to silencing. These results support the notion that transcriptional regulators could also participate in delimiting functional domains in the genome.<br><br>INTRODUCTION
The eukaryotic genome is organized into discrete functional domains in which gene expression is either permitted or repressed. Domains permissive for gene expression are usually defined as euchromatin or active chromatin, and domains that repress gene expression heterochromatin or silent chromatin. The HML and HMR loci in <span class="yellow">Saccharomyces cerevisiae</span> are silent chromatin domains (1). Compared to active chromatin, silent chromatin is more compact and its histone components are hypoacetylated (2). The SIR complex consisting of Sir2p through Sir4p is an integral part of <span class="blue">yeast</span> silent chromatin. Sir3p and Sir4p interact with the N-terminal tails of histones H3 and H4 providing the structural basis of silent chromatin (1). Sir2p was recently found to be an NAD-dependent histone deacetylase and was suggested to be responsible for histone hypoacetylation in silent chromatin (3). Silent chromatin is initiated at small cis-acting DNA elements known as the E and I silencers flanking each of the HM loci (4).
Increasing evidence indicates that histone deacetylation is key to the propagation of silent chromatin (4). In vertebrates and <span class="yellow">fission yeast</span><span class="blue">yeast</span>, spread of silent chromatin involves a chain of events of histone H3 deacetylation → H3 methylation → binding of methylated H3 by HP1 (heterochromatin protein 1) or swi6 (4). In <span class="yellow">S.cerevisiae</span>, there is evidence that Sir3p (hence the SIR complex) has much higher affinity to unacetylated histone H4 than acetylated H4 (5). Based on this and the fact that Sir2p is a histone deacetylase, a refined model for the propagation of silent chromatin can be proposed. In this model, Sir2p recruited to the silencer deacetylates histones in an adjacent nucleosome enabling it to bind another SIR complex with high affinity. The nucleosome-bound SIR complex in turn deacetylates the neighboring nucleosome allowing it to recruit a new SIR complex. In this manner, the SIR complex promotes its own propagation along an array of nucleosomes (4).
The fact that transcriptionally silent chromatin can encroach upon active chromatin poses the question of how interspersed domains of silent and active chromatin are demarcated. Studies of the <span class="yellow">Drosophila</span> and vertebrate genomes demonstrated that some domains are delimited by boundary or insulator elements (6). These elements are specialized DNA sequences that act as barriers to enhancers and/or silent chromatin. One of the best characterized insulators is the <span class="yellow">chicken</span> HS4 insulator at the β-globin locus. Histones surrounding this insulator are hyperacetylated indicating the presence of histone acetyltransferase (HAT) activity at the insulator (6). Recently, sequences that could block the spread of silent chromatin have been discovered in <span class="yellow">S.cerevisiae</span> (7,8). These sequences (referred to as silent chromatin barriers) include sub-telomeric anti-silencing regions (STARs), the right boundary of HMR, and the upstream activating sequence (UAS) of the highly expressed TEF2 gene (9–11). The mechanisms underlying barrier functions in <span class="blue">yeast</span> are not known.
TEF2-UAS contains three tandem repressor activator protein 1 (Rap1p)-binding sites that are necessary and sufficient for its barrier function (11). Rap1p is an abundant sequence-specific DNA-binding protein (12). Variants of the 13 bp consensus sequence for Rap1p binding (ACACC CRYACAYM) (13) lie not only in the UASs of numerous genes but also in the silencers of the HM loci and in telomeric C1–3A repeats. Accordingly, Rap1p functions as a global regulator of transcriptional activation and repression, silencing, as well as telomere length (12). Rap1p performs these diverse functions by interacting with different factors in respective contexts. As an activator, Rap1p binds to the promoters of 362 ORFs (13) and can function via at least three mechanisms. First, Rap1p binding can ‘open up’ local chromatin structure at a promoter to help another activator to bind (14). Secondly, Rap1p bound to DNA can help Gcr1p bind to an adjacent site in the promoter through physical interaction (15). Thirdly, Rap1p is involved in the recruitment of the NuA4 HAT complex and TFIID to ribosome protein genes (RPGs) (16,17). As a silencing factor, Rap1p binds to silencers or telomeric repeats and recruits Sir3p/Sir4p through direct interactions (18). Rap1p also interacts with telomere-specific proteins in executing its role in regulating telomere length (19). It is puzzling that a protein that helps establish silent chromatin could also act as a barrier to its propagation.
In this study, we found that in addition to TEF2-UAS, UASs of many <span class="blue">yeast</span> genes exhibit barrier activity. Analyses of these UASs indicate that binding sites for transcriptional regulators Abf1p, Reb1p, Adr1p, Gcn4p, as well as Rap1p, may participate in barrier function. These UASs are dispersed across the genome and might play a role in defining functional chromosomal domains. Moreover, we demonstrated that targeting the activation domain of Rap1p, Adr1p or Gcn4p alone, led to barrier function. Since the activation domain of an activator was usually involved in interacting with co-activators and/or components of the transcriptional machinery, these data indicated that other factors might also be involved in barrier function.<br><br>MATERIALS AND METHODS
Plasmids and strains
Plasmids 1–53 were used to make strains 1–53, respectively (see Figs 1–6). Plasmid 1 was previously named pMB22-a that contained a HindIII–BamHI fragment of chromosome III (coordinates 14838–16263) with a URA3 gene inserted at its EcoRV site (20). Plasmid 2 was derived from 1 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SnaBI site. Plasmids 3–38 were identical to 2 except that different fragments from various promoters (see Figs 1 and 2) were inserted at the SnaBI site. Plasmid 29 had a sequence of chromosome III (295327–295713) containing the HMR-tRNA gene (21) inserted at the SnaBI site of plasmid 1. Plasmids 39 and 40 were previously named pYXB26 and pYXB59, respectively (11). Plasmid 41 was identical to pYXB59 except bearing mutations illustrated in Figure 3A. Plasmids 42–48 were previously described as pYXB29, 48, 28, 27, 59, 31 and 37, respectively (11). Plasmid 50 was derived from plasmid 1 by inserting two copies of a sequence bearing the consensus binding sequence of LexA (bold) (22), GGGGTACGTACTGTATGTACATACAGGATATCGGGG, at the SnaBI site. Plasmid 51 was identical to 50 except having only one copy of the LexA-binding sequence. Plasmid 52 was derived from pYXB26 (11) by inserting a SpeI-ADE2-AvrII sequence at the SpeI site. Plasmid 53 was derived from plasmid 52 by inserting TEF2-UAS (–511 to –407 bp relative to the translation start codon) at the SpeI site.
Plasmid L1 (see Fig. 5A) was made by inserting LEU2 into pBTM116 (23) carrying the LexA gene flanked by the promoter and terminator of ADH1. L2 through L8 were derived from L1 by fusing various sequences to LexA (see Fig. 5A and B). Plasmid pRS425 (24) carried the 2 µm origin and LEU2.
Strain YXB76 was MATa ura3-52 leu2-3,112 ade2-1 lys1-1 his5-2 can1-100, E-HMLα-Iinverted (20). Y2047b was MATa HMRa HMLα EΔ79–113::SUP4-o IΔ242 LEU2-GAL10-FLP1 ura3-52 ade2-1 lys1-1 his5-1 can1-100 [cir0] (25). Strain 1 was made by transforming YXB76 to Ura+ with HindIII + BamHI digested plasmid 1. Strains 2–38, 50 and 51 were similarly made with corresponding plasmids, respectively (e.g. strain 2 was made with plasmid 2). Strains 39, 40 and 42–48 were previously described as YXB26, 48, 29, 48, 28, 27, 59, 31 and 37, respectively (11). Strains 41, 52 and 53 (see Figs 3C and 6) were made by transforming Y2047b to canavanine-resistant by BamHI-digested plasmids 41, 52 and 53, respectively. Strains 40′–48′ were sir3::URA3 derivatives of strains 40–48, respectively. The relevant genotypes of all the strains were confirmed by Southern blotting.<br><br>Quantitative mating assay
Quantitative mating was performed as described (11).<br><br>Electrophoresis mobility shift assay (EMSA)
Rap1p was expressed in <span class="yellow">Escherichia coli BL21 (DE3)</span> from pL3S5, a pET vector carrying the RAP1 gene downstream of a T7 promoter (26). Crude protein extracts were prepared from 40 ml cultures harvested after 3 h of IPTG induction and resuspended in 1 ml of lysis buffer (50 mM NaH2PO4, 300 mM NaCl, pH 8.0). A control extract was prepared from a culture without IPTG induction. Western blotting revealed that Rap1p protein was present in the induced extract but not the uninduced one (data not shown). DNA fragments of 26 bp in length (see Fig. 3A) were end-labeled with 32P-phosphate. Fifteen microliter binding reactions were prepared, each consisting of 20 000 c.p.m. radiolabeled DNA probe, 5 µg of crude protein extract, non-specific competitor DNAs (0.3 µg of <span class="blue">yeast</span> tRNA, 0.3 µg of <span class="yellow">salmon</span> sperm DNA and 0.5 µg of <span class="yellow">E.coli</span> DNA), 10 µg of BSA, 10 mM Tris–HCl (pH 8.0), 10 mM MgCl2 and 8% glycerol. The reactions were incubated at 25°C for 20 min and loaded onto non-denaturing polyacrylamide gels (4–5%). Gels were run at 40 mA for 1 h in 0.5× TBE buffer.<br><br>Analysis of the supercoiling of DNA circles
Cells were grown in YPR (<span class="blue">yeast</span> extract + bacto peptone + 2% raffinose) to early log phase. Galactose (2%) was then added to the culture to induce the expression of FLP1. After 2.5 h of incubation, nucleic acid was isolated using the glass bead method and fractionated on agarose gels with 30 µg/ml chloroquine. DNA circles were detected by Southern blotting.<br><br>
RESULTS
UASs of many highly expressed genes can function as barriers to the spread of silencing
We have previously shown that TEF2-UAS was able to block the spread of silencing in a silencer-blocking assay (11). Such an assay tested if a sequence had the ability to prevent a silencer from silencing a reporter gene when it was positioned between the silencer and the reporter. In this report we used a new silencer-blocking assay to identify more barrier elements. This assay employed the URA3 gene and an inverted HML-I that could silence sequences to the right of HML (20) (Fig. 1, strain 1). URA3 expression makes cells sensitive to 5-fluoroorotic acid (FOA) thus URA3 silencing could be measured by cell growth on FOA medium (Fig. 1, strain 1). As expected, TEF2-UAS and another known barrier element, HMR-tRNA gene (21) exhibited barrier activity in this assay as indicated by the lack of silencing of URA3 in strains 2 and 29 (Fig. 1).
TEF2 encoding translation elongation factor 1α is one of the most highly expressed genes in <span class="blue">yeast</span> (27). Other highly expressed genes include ribosome protein genes (RPGs) and genes coding for glycolytic enzymes (27). The coordinated regulation of most of these genes requires the so-called general regulatory factors Rap1p, Abf1p and Reb1p (28,29). Using the above silencer-blocking assay, we tested if the UASs of other highly expressed genes could also block URA3 silencing (Fig. 1). Seventeen new barrier elements were identified from a total of 26 UASs tested. These 17 elements were UASs of eight RPGs (see Fig. 1, strains 3–10), eight glycolytic genes (strains 18–25) and the HIS3 gene (strain 26). All 17 elements except ADH2-UAS and HIS3-UAS exhibited very strong barrier activity as evidenced by the absence of URA3-silencing in strains 3–10 and 18–24 (Fig. 1).
Seven of the eight RPG-UASs that had barrier activity, contained a single or a pair of tandem Rap1p-binding sites (Fig. 1, strains 3–9). Three of them also contained an Abf1p or Reb1p site (strains 6, 8 and 9). One barrier RPG-UAS (strain 10) had only a predicted Abf1p site. Of the seven RPG-UASs that had no barrier activity (Fig. 1, 11–17), only two bore no site for Rap1p, Abf1p or Reb1p (11 and 12). The remaining five each contained one to three Rap1p sites (strains 13–17). RPL15A-UAS (17) also contained an Abf1p site. For most of the 15 RPG-UASs tested (3–17), the presence or absence of predicted Rap1p sites correlated with the presence or absence of Rap1p association in vivo as previously examined by chromatin immunoprecipitation assays (13). Exceptions were the UASs of RPS24B, RPS28A and RPL15A (13).
Of the eight UASs of glycolytic genes tested (Fig. 1, strains 18–25), seven exhibited strong barrier activity (18–24) and one had weaker but detectable barrier activity (strain 25). Glycolytic genes are among the most highly expressed genes and their high expression depends on the functions of Rap1p, Abf1p or Reb1p as well as Gcr1p (29,30). The seven strong glycolytic barriers all consist of one or two Rap1p sites plus one or two Reb1p or Abf1p sites (Fig. 1, strains 18–24). In addition, one or more Gcr1p-binding sites are found adjacent to each Rap1p site. It was proposed that the function of Rap1p was to facilitate the binding of Gcr1p to the UAS (15). Abf1p or Reb1p was shown to play a role similar to that of Rap1p in activating glycolytic genes (31–33).
Although the above data reinforced the notion that certain Rap1p-binding UASs could serve as barriers to silencing, they raised new questions about the essential components of a barrier element. For example, some of the UASs that contained only a single Rap1p site had barrier activity (e.g. RPL19B-UAS, strain 3) whereas others that contained two or three sites had no activity (e.g. RPL39, strain 16). One explanation was that sequences flanking the Rap1p site(s) in a particular UAS could positively or negatively influence barrier function. Moreover, the presence of Abf1p or Reb1p sites in 11 of the 17 new barriers made us wonder if they also participated in barrier function. We began to address these issues by analyzing three new barrier elements (Fig. 2). RPS10A-UAS containing an Abf1p-binding site and a pair of Rap1p sites was divided into three fragments that were tested in a silencer-blocking assay (Fig. 2, strains 30–32). It was clear that only the fragment containing Rap1p sites functioned as a barrier (strain 31). TPI1-UAS bearing a Reb1p site and a Rap1p site was divided into two parts with one containing the Rap1p site (Fig. 2, strain 34) and the other Reb1p site (strain 33). Neither of these fragments alone retained barrier activity indicating that concerted action of Reb1p and Rap1p was required for barrier activity. Duplicating the Reb1p-bearing part of TPI1-UAS didn’t restore barrier activity (strain 35). However, triplicating the Rap1p-containing part did re-create barrier function (strain 36). RPS28A-UAS bore, in addition to a Rap1p site, an Abf1p site and an adjacent T-rich region that are required for efficient transcription of RPS28A (34). A sequence of RPS28A-UAS deleted for a fragment bearing the Rap1p site lost the barrier activity (Fig. 2, strain 37). However, duplicating this sequence restored barrier activity (strain 38). The above results indicated again that Rap1p sites could form barriers (strains 31 and 36). On the other hand, they also suggested that synergistic actions of Abf1p sites (strain 38), Reb1p site + Rap1p site (strain 18), or Abf1p site + Rap1p site (strain 16) could all lead to barrier function. This was not very surprising since Rap1p, Abf1p and Reb1p could perform similar and sometimes interchangeable functions in gene regulation (16,35,36). Note, however, we haven’t ruled out the possibility that other factors (e.g. Gcr1p) or structure features of DNA (e.g. T-tracks) were also involved in barrier function.
It was interesting that ADH2-UAS and HIS3-UAS bearing no site for Rap1p, Abf1p or Reb1p exhibited detectable barrier activity (Fig. 1, strains 25 and 26). However, binding sites for other transcriptional activators existed in these UASs. The major regulator that binds to ADH2-UAS is Adr1p (37). On the other hand, Gcn4p binds to multiple sites in HIS3-UAS and activates HIS3 transcription (38). The possible involvement of Adr1p or Gcn4p in barrier function was examined below. Note Adr1p function was subject to glucose repression (37), which might explain why ADH2-UAS only exhibited limited barrier activity since glucose was used in the media used in our assay.<br><br>Effect of mutating Rap1p-binding sites in TEF2-UAS on barrier function
Although we had shown that the three Rap1p-binding sites in TEF2-UAS were necessary and sufficient for its barrier activity (11), we had not shown if binding of Rap1p to these sites was required. To address this we performed mutational analysis of TEF2-UAS. As shown in Figure 3A and B, the three Rap1p-binding sites in TEF2-UAS (designated as R1, R2 and R3) were all able to bind Rap1p in an EMSA (Fig. 3B). Single or double C→A mutations were introduced into R1, R2 and R3 resulting in m1, m2 and m3, respectively (Fig. 3A). The m2 and m3 sites had both lost the ability to bind Rap1p, whereas m1 still bound Rap1p (Fig. 3B), which could be predicted from the specific mutations in each site (Fig. 3A, compare m1, m2 and m3 to the consensus sequence). We then tested a mutated TEF2-UAS containing m1, m2 and m3 in a silencer-blocking assay using the HML-E silencer and the HMLα genes (Fig. 3C). In this assay, the HML-I silencer was deleted but the HML-E silencer was sufficient to silence the HMLα genes (Fig. 3C, strain 39) (11). Silencing in strain 39 (a-type) was measured by its ability to mate with an α-type strain (Fig. 3C). As predicted, the wild-type TEF2-UAS blocked HMLα silencing (11) (Fig. 3C, strain 40). On the other hand, the mutated TEF2-UAS containing m1, m2 and m3 had lost barrier activity (Fig. 3C, compare the mating efficiencies of strains 41 and 40). Therefore, mutations that prevented Rap1p from binding two of the three sites in TEF2-UAS abolished its barrier activity, indicating that association of Rap1p with TEF2-UAS at more than one site was required for barrier function.<br><br>The barrier TEF2-UAS alters local chromatin structure
In eukaryotic cells, the topology of local DNA reflects the chromatin structure in which it resides. We had previously developed a DNA topology-based assay to examine chromatin structure in vivo, which involved excising a chromosomal region of interest from its genomic location as a circle using site-specific recombination (39). Using this strategy, we demonstrated that DNA in silenced chromatin was more negatively supercoiled than that in active chromatin (39). In this report we used the topology assay to address if barrier elements altered local chromatin structure as <span class="yellow">Drosophila</span> or <span class="yellow">chicken</span> insulators did (40,41).
Various sequences (designated X) from UASs of TEF2, AgTEF (TEF gene from <span class="yellow">Ashbya gossypii</span>) and ADE2 were inserted between HML-E and HMLα at HMLΔI (HML locus deleted for the I silencer) flanked by the FRT sites for the Flp1p site-specific recombinase (Fig. 4A and B). The FLP1 gene was under the control of the inducible GAL10 promoter (39). The mating efficiency of each strain measured silencing of HMLα (Fig. 4C). In strains 40, 42, 44 and 47, silencing was restricted to the left of the X insertions and HMLα was derepressed (Fig. 4C). Consistently, the proportion of silent chromatin in the region flanked by the FRT sites in each of these strains was decreased, as indicated by the reduced negative supercoiling of the excised circle (11) (data not shown). Note that the above effect of a barrier on HML chromatin reflected both change in the scope of silencing caused by the barrier activity and alteration (if any) in chromatin structure at the barrier independent of the silencing state of the locus. To exclusively examine the latter, we analyzed the topology of HML DNA in a silencing-deficient background. The SIR3 gene in strains described in Figure 4B was disrupted rendering them defective in silencing (strains 40′–48′ in Fig. 4D). Recombination mediated by Flp1p in these strains led to the excision of a group of chromosomal circles that differed only in the X insertion. The supercoiling of these circles was analyzed as a function of the length of the insertion (Fig. 4D). If an insertion simply lengthened a circle without causing abnormal changes in nucleosomal structure and/or density, the topoisomers of this circle would migrate slower relative to those of a similar circle with a smaller size. Otherwise, migration of the topoisomers would deviate from that predicted based on the size of the circle. As shown in Figure 4D, the circle from strain 40′ bearing a 104 bp barrier element migrated faster rather than slower than the circle from strain 46′ bearing a 91 bp non-barrier element. Circles bearing other barrier elements also exhibited unexpected faster migration (compare strains 44′ to 45′, 47′ to 43′, 42′ to 48′, respectively). Therefore, these data indicated that barrier function correlated with altered topology of local DNA. This was further demonstrated by that mutations in TEF2-UAS that abolished its barrier activity also abolished its effect on DNA topology (Fig. 4D, compare 41′ to 40′; note that circles from both strains had the same size). Since in our gel assay more negatively supercoiled DNA migrate slower, we concluded that every fragment with barrier activity caused a reduction in negative supercoiling of approximately one to two turns in HMLΔI DNA (Fig. 4D, compare the centers of distributions of topoisomers in pairs of lanes, e.g. 40′ and 41′).
The topology of eukaryotic DNA reflects mainly the wrapping of DNA into nucleosomes (approximately one negative superhelical turn is associated with each nucleosome) (42). Therefore, removal of one nucleosome would eliminate approximately one turn. On the other hand, histone acetylation can reduce negative supercoiling associated with a nucleosome by approximately 0.2 turns (43). Therefore, acetylating five nucleosomes would reduce negative supercoiling by approximately one turn. Consequently, the linking number change of one to two brought about by TEF2-UAS can be accounted for by either the loss of one to two nucleosomes or acetylation of five to 10 nucleosomes. Further experiments are under way to clarify the nature of change in chromatin induced by a barrier.<br><br>Targeting the transcription activation domain of Rap1p recapitulates its barrier activity
The multifunctional Rap1p can be divided into at least four functional domains (Fig. 5A) (12). We were interested in defining which of these domains might be involved in the barrier activity of Rap1p. To this end, we fused various parts of Rap1p to the bacterial LexA protein and tested if targeting any of the fusion proteins to LexA-binding sites could re-create a barrier to silencing (Fig. 5A). The LexA-RAP1 fusion genes were carried on 2 µm based plasmids marked by LEU2 (designated L1–L6, Fig. 5A). When introduced into strain 50 in which two LexA-binding sites had been inserted between the inverted HML-I and URA3, LexA alone had no effect on URA3 silencing (Fig. 5A, plasmid L1). Targeting the DNA binding domain, DNA bending domain, silencing domain, or the activating + silencing domain of Rap1p also did not affect URA3 silencing (L2–L5). On the other hand, targeting the activation domain alone recapitulated the barrier activity of Rap1p (L6). These results indicated that the activation domain of Rap1p was involved in barrier activity. The inability of the Rap1p-silencing domain to block URA3 silencing in our assay did not necessarily mean that this domain had no barrier function, since such a function might be suppressed by much stronger silencing activity associated with the silencing domain.
Was the lack of barrier function of fusion constructs L2–L5 due to a lack of expression of these proteins? To address this possibility, we examined the levels of L1–L6 proteins as well as L7 and L8 proteins described in Figure 5B by western blotting using an antibody against LexA. As shown in Figure 5C, L5 was expressed at a level comparable to L6, and L4 was expressed at a higher level than L6. Therefore, the lack of barrier function of L4 and L6 was not the result of their lack of expression. On the other hand, the level of L2 or L3 was significantly lower than that of L6, hence the lack of barrier function of L2 and L3 might be due to their insufficient expression.
As an important control for the above targeting experiments, we showed that plasmids L1–L6 (Fig. 5A) as well as L7 and L8 (Fig. 5B) bearing LexA-fusion genes had no effect on URA3 silencing in strain 1 (Fig. 1) in which there was no binding site for LexA present between HML-I and URA3 (data not shown). Hence, the effect of the fusion proteins on URA3 silencing was the result of their targeting to specific loci, not the consequence of non-specific, indirect functions (if any) of these proteins.<br><br>Targeted activation domain of Adr1p or Gcn4p can also act as a barrier factor
We have shown above that ADH2-UAS bearing Adr1p-binding sites and HIS3-UAS bearing Gcn4p sites both had detectable barrier activity (Fig. 1, strains 25 and 26). Given that tethering the activation domain of Rap1p is sufficient for barrier function, we tested if targeting the activation domain of Adr1p or Gcn4p could also establish a barrier. Adr1p contains four separate activation domains (TADI–TADIV) that can contact Ada2p, Gcn5p and/or TFIIB (45). TAD III (415–467) interacts with only Gcn5p but not Ada2p or TFIIB. We fused this domain of Adr1p and the activation domain of Gcn4p (102–149) to LexA, respectively (Fig. 5B, L7 and L8). Targeting these fusion proteins to two LexA-binding sites between HML-I and URA3 in strain 50 dramatically reduced URA3 silencing (Fig. 5B, compare L7 and L8 to L1). Therefore, like LexA-Rap1p (627–695), both LexA-Adr1p (415–467) and LexA-Gcn4p (102–149) also had barrier activity. Barrier function of these fusion proteins was significantly decreased in strain 51 in which only one LexA site was inserted between HML-I and URA3 (compare strains 50 and 51 carrying plasmid L7), indicating that the potency of such barriers was dependent on the amount of tethered fusion proteins.<br><br>
DISCUSSION
We have shown in this report that UASs of many highly expressed genes could function as barriers to the spread of transcriptional silencing. These newly identified barrier elements together with previously described barrier elements (STARs, HMR-tRNA gene, TEF2-UAS and CHA1-UAS) (9–11,21) are scattered across the <span class="blue">yeast</span> genome and may play a role in dividing the genome into functional domains. Analyses of the barrier UASs suggest that binding sites for transcriptional regulators Rap1p, Abf1p, Reb1p, Adr1p and Gcn4p can participate in barrier function. Barrier activity often requires concerted actions of more than one factor-binding site. Our mutational analysis of TEF2-UAS suggests a direct involvement of Rap1p in its barrier function. It is noteworthy that not all UASs of highly expressed genes have barrier activity. Many non-barrier UASs also consist of multiple regulator binding sites, therefore other unknown factors might determine whether a UAS could act as a barrier.
How Rap1p and the other transcriptional regulators act to block silencing is not clear. Since they can all act as transcriptional activators, one natural explanation is that they may overcome the silencing effect of a silencer and directly activate the reporter gene. However, it has been well documented that Rap1p and the other transcriptional regulators usually act at relatively short distances (less than ∼600 bp upstream of the translation start codon) (28,29,46), but the binding sites for these regulators were >800 bp from the translation start codon in most of the silencing-blocking tests in this report (Figs 1 and 2). In the test shown in Figure 6, TEF2-UAS blocked ADE2 silencing when it was ∼2 kb downstream from the translation start codon of the gene. Moreover, Fourel et al. (44) demonstrated that the barrier function (or anti-silencing function) of an activator was independent of its role in activating transcription. Therefore, it is unlikely that the ability of a UAS to prevent gene silencing is due to its direct activation of the gene. In light of this, our results are in agreement with the notion that transcriptional regulators can also participate in defining the boundaries of functional chromosomal domains.
A few models have been proposed for the barrier functions of transcriptional regulators. One model proposed that binding of a regulator to a barrier led to the formation of a nucleosome-free region thereby disrupting a continuous nucleosome array (11). Since the SIR complexes spread by self-interaction and interacting with adjacent nucleosmes in a stepwise fashion (4), a nucleosome-free gap could present a barrier to the spreading SIR complex. This ‘nucleosome gap’ hypothesis is consistent with that certain barrier factors such as Rap1p and Reb1p could indeed prevent nucleosome formation around their binding sites (14) and the barrier function of TEF2-UAS correlated with its ability to alter local chromatin structure (this report). In fact, many insulator/boundary elements in higher organisms have also been shown to alter local chromatin structure. For instance, the <span class="yellow">chicken</span> β-globin insulator was linked to changes in nucleosome positioning, and the <span class="yellow">Drosophila</span> scs and scs′ boundary elements are associated with unusual chromatin structures that are hypersensitive to nucleases (40,41). However, it is not clear whether any chromatin change associated with barrier function in <span class="blue">yeast</span> or insulator function in higher cells is the cause or effect of the function.
Another model for barrier action proposed by R. Kamakaka and colleagues (21) proposed that barrier factors recruit chromatin modifying and/or remodeling complexes to counteract the more compact, hypoacetylated silent chromatin. In agreement with this hypothesis, targeted HATs SAS2, GCN5 and ESA1 all had barrier activity (21; Y.-H. Chiu, Q. Yu and X. Bi, unpublished results). Interestingly, Rap1p, Adr1p, Gcn4p and Abf1p share the ability to recruit HAT complexes NuA4 and/or SAGA to target genes (16,45,47–49). Therefore, these factors that are capable of binding to the newly identified barriers may function as barrier factors by recruiting HAT complexes. In fact, recruitment of HAT activity has also been suggested to be the mechanism underlying the function of the <span class="yellow">chicken</span> β-globin insulator (6), although it was not known what HAT might be involved. Our demonstration that tethering the activation domain of Rap1p, Adr1p or Gcn4p alone recapitulates barrier activity is consistent with the above HAT model for barrier function, since at least the activation domain of Adr1p has been shown to directly interact with HAT complexes (45). However, we haven’t ruled out the possibility that the barrier function of the targeted activation domain of Rap1p, Adr1p or Gcn4p is not related to the barrier function of the native protein. Note that the ‘nucleosome gap’ model and the HAT model are not mutually exclusive since the nucleosome gap could result from destabilization of nucelosomes caused by HAT function. A recent intriguing study showed that targeting proteins of the nuclear pore complex (NPC) could also establish a boundary for silent chromatin, and that tethering to NPC was required for boundary activity (50). This led to a revisit to the looping model for the function of boundary/insulator elements (51). However, an alternative explanation was that the NPC compartment could simply be rich in transcriptional activating factors or poor in silencing factors like the SIR proteins (50).
In summary, results from this and other studies indicate that factors previously identified as positive and/or negative regulators of gene activation could also function in demarcating distinct domains of gene activity. How these factors carry out their boundary function is not resolved but recruitment of other factors (e.g. chromatin modifying complexes) is likely to be involved.<br><br>NOTE ADDED IN PROOF
After this work was completed and submitted for publication the first time, Fourel et al. (52) reported that targeted activation domains of Rap1p and Abf1p had insulating capacity.<br><br>
<h3>pmcA1636465</h3>Expression of C-terminal deleted p53 isoforms in neuroblastoma
Abstract
The tumor suppressor gene, p53, is rarely mutated in neuroblastomas (NB) at the time of diagnosis, but its dysfunction could result from a nonfunctional conformation or cytoplasmic sequestration of the wild-type p53 protein. However, p53 mutation, when it occurs, is found in NB tumors with drug resistance acquired over the course of chemotherapy. As yet, no study has been devoted to the function of the specific p53 mutants identified in NB cells. This study includes characterization and functional analysis of p53 expressed in eight cell lines: three wild-type cell lines and five cell lines harboring mutations. We identified two transcription-inactive p53 variants truncated in the C-terminus, one of which corresponded to the p53β isoform recently identified in normal tissue by Bourdon et al. [J. C. Bourdon, K. Fernandes, F. Murray-Zmijewski, G. Liu, A. Diot, D. P. Xirodimas, M. K. Saville and D. P. Lane (2005) Genes Dev., 19, 2122–2137]. Our results show, for the first time, that the p53β isoform is the only p53 species to be endogenously expressed in the <span class="yellow">human</span> NB cell line SK-N-AS, suggesting that the C-terminus truncated p53 isoforms may play an important role in NB tumor development.<br><br>INTRODUCTION
The p53 tumor suppressor gene remains the most frequently altered gene in <span class="yellow">human</span> tumors. Several p53 mutation databases have been reported previously (1–3), and to date, more than 1500 different p53 mutants have been described (4). Functional inactivation of p53 is usually due to gene mutation, deletion or protein degradation. In general, the majority of p53 mutations in <span class="yellow">human</span> neoplasia are missense mutations affecting the DNA-binding domain (DBD). Unlike other <span class="yellow">human</span> cancers, p53 in neuroblastoma (NB) is rarely mutated in the primary tumor at diagnosis but high levels of wild-type p53 (wt p53) protein expression have been found in the cytoplasm of undifferentiated tumors (5,6). More recently, in normal unstressed cells, wt p53 protein was found to be retained in the cytoplasm as a latent form, in huge, p53-associated protein complexes known as ‘Parc’ (7). The steady-state concentration of p53 in normal unstressed cells is usually very low because of the short half-life of the wild-type (wt) protein. Overexpression of p53 in most of the transformed cells containing a missense mutation within the p53 gene appears to be due to the increased stability of mutated p53 (8). In unstressed NB cells, high wt p53 expression may reflect the embryonic origin of NBs, in which precursor cells fail to mature (9).
p53 mutations are unusual in <span class="yellow">human</span> NB but, when they do occur, are found in post-chemotherapy tumors. In this respect, Tweddle et al. (10) described how two NB cell lines derived from the same <span class="yellow">patient</span> can elicit a different p53 status: wt p53 for SK-N-BE(1a) established before treatment, and mutated p53 for SK-N-BE(2c) established after relapse of the <span class="yellow">patient</span> under treatment with cytotoxic agents such as cyclophosphamide, doxorubicin, vincristine and radiotherapy. In NB cell lines, p53 mutation has been found in multidrug-resistant cells (11). Various types of p53 mutation have been detected in NB cells and can lead to inactivation either by shut-down of protein expression or production of aberrant p53 products. Indeed, in LAN-1 cells, p53 nonsense mutation at cysteine 182 in exon 5 leads to the absence of protein (9), whereas in SK-N-BE(2) cells, missense mutation at codon 135 (C135F) leads to stable overexpressed protein (11). By analyzing IGR-N-91, a cell line established in our laboratory from the bone marrow of a <span class="yellow">patient</span> with metastatic NB after unsuccessful Adriamycin–vincristine chemotherapy (12), we identified another type of aberrant protein that arises from the duplication of exons 7-8-9. This duplication spans from amino acids 225 to 331, which represent part of the DBD and part of the oligomerization domain (13). However, each p53 mutant has been described in the literature as a case report, and so far, no comparative study has been undertaken to link their biochemical features with functional properties.
In the present study we report two novel p53 C-terminus mutants identified in SK-N-AS and IGR-NB8 <span class="yellow">human</span> NB cell lines. The biological properties of these two new variants were analyzed in comparison with p53 isolated from six other <span class="yellow">human</span> NB lines: three [LAN-1, SK-N-BE(2) and IGR-N-91] expressing mutant p53 and three (SH-SY5Y, LAN-5 and IMR-32) expressing the wt protein. This characterization was done by using a range of functional assays: (i) the ability of the protein to bind with p53 consensus sequence using the functional analysis of separated allele in <span class="blue">yeast</span> (FASAY); (ii) the ability of the protein to transactivate the p53-responsive element (RE) identified either in the promoter of p21/WAF1 or in the first intron of BAX, using luciferase reporter assay; (iii) the induction of endogenous p21/WAF1 gene expression under stress conditions.<br><br>MATERIALS AND METHODS
Neuroblastoma cell lines, culture and drug treatments
The parental <span class="yellow">human</span> NB SH-SY5Y, SK-N-AS, IMR-32 and SK-N-BE(2) cell lines were purchased from the European Collection of Cell Cultures (ECACC, Wiltshire, UK). The <span class="yellow">human</span> IGR-NB8 cells (a gift of Prof. Gilles Vassal, UPRES EA 3535, Institut Gustave Roussy, Villejuif) were derived from a previously untreated localized NB (14). The LAN-1 and LAN-5 cell lines were provided by Dr Nicole Gross (Pediatric Oncology Research, Lausanne, Switzerland). The <span class="yellow">human</span> IGR-N-91 cell line was established in our laboratory from the bone marrow of a <span class="yellow">patient</span> with metastatic NB after unsuccessful adriamycin–vincristin chemotherapy (12). LAN-1, LAN-5 and IMR-32 were grown in RPMI medium supplemented with 2 mM l-glutamine and 10% fetal <span class="yellow">calf</span> serum and gentamicine 10 μg/ml. Others cell lines were cultured in DMEM.
For activation of endogenous p53, cells were treated with cis-platinum (Sigma) (10 μg/ml) for 24 h then lysed for western blot analysis.<br><br>Western blot analysis
This procedure was carried out as described previously (13). Protein lysates (50 μg) were submitted to 10% SDS–PAGE, and then transferred onto nitrocellulose filters. After saturation, the membranes were incubated with primary antibody diluted in 0.1% phosphate-buffered saline, Tween-20 and 3% skim milk. The primary antibodies used were anti-p53 monoclonal antibody (clone DO-7, 1/1000, DAKO), anti-p21/WAF1 monoclonal antibody (Ab-1, 1/200, Oncogene Research) and anti-β-actin monoclonal antibody (1/1000; Chemicon) as internal control. Protein bands were detected by ECL system (Amersham).<br><br>PCR, plasmids cloning
Genomic DNA was extracted using lysis buffer containing 20 mM Tris–HCl, pH 7.5; 0.4 M NaCl; 0.5% SDS; 10 mM EDTA, treated with proteinase K (200 μg/μl), purified with phenol/chloroform, precipitated with ethanol and dissolved in DNase free water. Total RNA were purified using RNAble reagent (Eurobio), precipitated with isopropanol and dissolved in RNase free water. cDNA was obtained by reverse transcription of 1 μg of total RNA using Superscript II™ RNase H-Reverse transcriptase (Invitrogen) and Oligo-d(T)16 in conditions specified by the manufacturer. Amplification of full-length p53 coding region from SH-SY5Y, IGR-N-91, IGR-NB8 and SK-N-BE(2) cell lines was performed using forward primer at position 152 and reverse primer at position 1583 (F1 and R7, respectively, Table 1; GenBank accession no. K03199). p53 cDNA from SK-N-AS cells for cloning was obtained from RT–PCR using F1 and reverse primer i9+: 5′-GCAAAGTCATAGAACCATTTTCAT-3′ (nucleotide position 14989, GenBank accession no. X54156) primers which encompass from exon 1 to exon i9+ first identified by Flaman et al. (15) included. The PCR was done in the presence of pfu Hotstart DNA polymerase (Stratagene) for 30 cycles of 1 min at 90°C, 1 min at 65°C and 2 min 30 s at 72°C using PTC-100 thermocycler (MJ-Research).
The p53 cDNA from SH-SY5Y, SK-N-AS, IGR-N-91, IGR-NB8 and SK-N-AS cells were then cloned into pcDNA3.1/V5-His-Topo vector (Invitrogen) according to the manufacturer's instruction. The p53 sequence of each cell line was investigated by sequencing of plasmids after cloning. Sequencing was performed by Genome Express (Meylan, France).
The pDDm-TO harboring p53 dominant negative form (p53DD) pGL3-E1bTATA and the pE1B-hWAF1 firefly luciferase reporter containing the p53-responsive element of the p21/WAF1 promoter were described previously (16,17) pE1B-BAXi contains the p53RE identified in the intron 1 of the BAX gene [(18) and D. Munsch, personal communication]. Oligonucleotides TCGAGGGCAGGCCCGGGCTTGTCG and CTAGCGACAAGCCCGGGCCTGCCC were annealed and cloned into pGL3-E1bTATA digested with NheI and XhoI to obtain pE1B-BAXi. The pcDNA3-ΔNp73α expression plasmid was a gift of Dr Daniel Caput (SANOFI, Labèges, France).<br><br>Fluorescent in situ hybridization (FISH)
Cytogenetic preparations
Metaphase spreads from healthy <span class="yellow">human</span> male lymphocytes and tumor cell lines were prepared as described previously (19). BAC probe RP11-199F11, containing a 167 kb region spanning TP53 gene, was labeled by random priming in the presence of Alexa 594-dUTP (Molecular Probes). A commercial probe specific for chromosome 17 centromere, and labeled with spectrum green, was obtained from Vysis. After over-night cohybridization of the probes in the presence of Cot-1 DNA, the slides were washed and DNA counterstained with DAPI. The preparations were observed with an epifluorescence microscope and images captured with a Vysis imaging station. Between 3 and 14 metaphases spreads and 30–200 nuclei were examined for each cell line.<br><br>
Luciferase reporter assays
LAN-1 or SH-SY5Y cells were seeded in duplicates onto 6-well plates at a density of 2 × 104 cells per cm2 and cotransfected 24 h later with 0.5 μg (2.5 μg/ml) of pGL3 firefly luciferase reporter gene plasmid under the control of either pE1B-hWAF1 or pE1B-BAX using lipofectamine 2000 and 1 μg of either a p53 expressing plasmid or an empty vector. At 24 h after transfection, cells were lysed with 200 μl/well of passive lysis buffer provided with the ‘Luciferase assay kit’ (Promega). Luciferase activity was measured using Microlumat LB96P luminometer (EG & G Berthold Instrument).<br><br>Functional assay in <span class="blue">yeast</span>
cDNA was obtained by RT of 1 μg of total RNA using Superscript II™ RNase H-Reverse transcriptase (Invitrogen) and random hexamers to prime the synthesis in conditions specified by the manufacturer. p53 cDNA was amplified by PCR and cotransformed into <span class="blue">yeast</span>, IG397 Ade2 strain, together with either pRDI-22 vector for p53-standard assay or pFW35 and pFW34 plasmid for 5′ or 3′ split assay, respectively, carrying the ADE2 open reading frame under the control of a p53-responsive promoter (20). In a selective medium lacking leucine, wt-p53 activates transcription of ADE2 gene that encodes enzyme—phosphoribosylimidazole carboxylase—implicated in adenine biosynthesis. Therefore, a colony of cells that expresses ADE2 gene is white whereas the one composed of cells where ADE2 gene is not expressed owing p53 mutation is red.<br><br>
RESULTS
P53 status in SK-N-AS and IGR-NB8 cells
We first compared the migration profiles of p53 expressed in SK-N-AS and IGR-NB8 with those expressed in the other six NB cells, SH-SY5Y, LAN-5, LAN-1, IMR-32, SK-N-BE(2) and IGR-N-91. Western blots from 50 μg of total protein extracts were revealed with p53 monoclonal antibody (DO-7). A range of profiles was identified as shown in Figure 1A. As expected, p53 extracted from the three cell lines, SH-SY5Y, LAN-5 and IMR-32, expressing wt protein (13,21) migrated at the wt position. Of particular note in these three wt p53 cell lines was an additional faint band that migrated faster than the full-length protein. The LAN-1 cells were found to be p53 deficient (9). The SK-N-BE(2) cell line showed an intense band reflecting p53 stability due to a missense mutation at codon 135 (11). As expected, due to the previously identified duplication of exons 7-8-9 (13), p53 protein migration was delayed in the IGR-N-91 cells. In contrast, the p53 protein in the SK-N-AS cell line migrated noticeably faster than the wt protein, indicating that it was smaller in size. The p53 protein in the IGR-NB8 cell line was even smaller than that in SK-N-AS.
To analyze the coding region of each of the p53 variants, RT–PCR was performed using p53-specific F1-R7 primers (Table 1 and Figure 1B). The expected 1430 bp for full-length p53 was amplified from wt p53-expressing SH-SY5Y, LAN-5 and IMR-32 cell lines. As SK-N-BE(2) harbors a single point mutation at codon 135 (C135F), the amplified fragment analyzed by electrophoresis migrated as wt p53 (Figure 1B). The longer RT–PCR fragment from the mutated IGR-N-91 cell line resulted from the duplication of exons 7-8-9, as shown in our previous data (13), which corresponds to extra nucleic material of 321 nt. For the LAN-1 cells, no amplified fragment was observed in accordance with published data, which demonstrates the extremely low levels of p53 mRNA and the undetectable level of protein (9). An amplified fragment of the same length as the wt protein was observed in the IGR-NB8 cell line (Figure 1B). Indeed, complete gene sequencing revealed a point mutation E326STOP leading to a truncated protein at C-terminus. No fragment, however, was amplified from SK-N-AS with F1-R7 primers.
To further map the p53 mRNA transcribed in these cells, series of RT–PCR tests were performed using the forward primer, F2 (exon 8 position 1008th according to GenBank accession no. K03199), matched with different reverse primers, R3 (at the junction of exon 8/9, nt position 1124), R4, R5 (in exon 9 at positions 1154 and 1184, respectively), and R6 (in exon 10, at position 1230). The sequences of these primers are given in Table 1 and the results are presented in Figure 2A. SK-N-AS cDNA gave an amplified fragment of the same size as SH-SY5Y cDNA with the three primer pairs, F2/R3, F2/R4 and F3/R5. However, in contrast to SH-SY5Y, no fragment was obtained with SK-N-AS cDNA using the F2/R6 primer pair, which suggests the absence of exon 10 in SK-N-AS mRNA.<br><br>The p53 protein expressed in SK-N-AS is the p53β isoform
An alternatively spliced form of <span class="yellow">human</span> p53 mRNA with an additional 133 bp exon derived from intron 9 has been detected in normal <span class="yellow">human</span> lymphocytes (15). This spliced variant named ‘i9+’ encodes a truncated protein of 341 amino acids including 10 new amino acids derived from the novel exon, the p53β isoform according to Bourdon et al. nomenclature (22). This led us to hypothesize that the shorter protein expressed by the SK-N-AS cell line could be the p53β isoform. To test this hypothesis, RT–PCRs were performed using primer sets designed to amplify the 3′ region of p53 mRNA encoding either the specific C-terminal part of the wt protein (wt C-ter) or the specific C-terminal part of the β isoform (β C-ter). In parallel amplification with a primer pair amplifying the DBD was used as control. The sequences of these oligonucleotides are given in Table 1. The results presented in Figure 2B are consistent with the only expression of the p53β isoform in SK-N-AS as no band was observed in lane using specific C-ter primer located in exon 10. Interestingly, RT–PCR using SH-SY5Y (SH) gave an amplified fragment not only with the primer pair specific to the C-ter domain of wt p53 but also with the primer pair specific to the p53β isoform. This result, combined with the presence of an additional faint band migrating faster than wt p53 in denaturing polyacrylamide gel (Figure 1A), strongly suggests that both the p53 full-length protein and the β isoform were expressed in the SH-SY5Y cells.
The full-length SK-N-AS p53 cDNA was then amplified with the forward primer F1 and a reverse primer located within the novel exon i9+ (Table 1). This amplified fragment was cloned in pcDNA3/V5-His-Topo, as described in Materials and Methods. Its sequence analysis confirmed that the truncated p53 expressed in SK-N-AS was encoded by the i9+ splice variant described previously by Flaman et al. (15) that encodes the p53β isoform characterized by Bourdon et al. (22).
A series of genomic amplifications were performed to identify a possible deletion within the intron 9 that could account for the absence of normal size p53 in SK-N-AS cells. The primer sequences are given in Table 1. Amplifications were performed in parallel with total DNA extracted from SH-SY5Y and SK-N-AS cells. Results are presented in Figure 3. Normal size fragments that encompass the acceptor site of intron 9 were amplified with SH-SY5Y as well as with SK-N-AS DNA. On the contrary amplification fragments that encompass the intron 9 donor site were obtained only with SH-SY5Y but not with SK-N-AS DNA. These results identify a deletion of the intron 9/exon 10 junction within the SK-N-AS p53 gene.<br><br>A <span class="blue">yeast</span> functional assay confirmed the absence of p53 full-length expression in SK-N-AS and IGR-NB8
It is possible to detect p53 mutation using a simple <span class="blue">yeast</span> colony color assay as described by Flaman et al. (23). When the strain is transformed with a plasmid-encoding wt p53, the cells express the ADE2 gene and produce white colonies (Figure 4A, a, b2 and c1, and Figure 4B, dish a). Cells containing mutant p53 fail to express ADE2 and form small red colonies (Figure 4A, b and b1, and Figure 4B, dish c). When the p53 cDNA fragment is deleted, cells are unable to form a colony (Figure 4A, c and c2). As shown in Table 2, FASAY was performed as a p53-standard test with full-length cDNA or with the split version at the 5′ and 3′ end (15). The background of FASAY experiments is around 10%. p53 wt expressing SH-SY5Y and LAN-5, 2 wt cell lines, yielded ∼92–97% of white colonies (Table 2).
One hundred percent of the colonies carrying SK-N-BE(2) p53, which is homozygous for the C135F mutation, turned red with the standard or 5′ split assay, whereas 94% of the colonies turned white with the 3′ split assay since the missense mutation does not extend to the C′-terminus of the gene (Table 2 and Figure 4B, dish c). No colonies were observed with p53-deficient LAN-1 cells, (see also Figure 4A, c and c2). With SK-N-AS cells, the split 5′ assay gave 96% white colonies, while the p53-standard and split 3′ assay did not produce any colonies (Table 2). This means that the 5′ terminus was intact whereas the 3′ terminus had been deleted, as was confirmed by nucleotide sequence analysis. The IGR-NB8 colonies, however, were red both with the p53-standard assay and the split 3′ assay. In the IGR-N-91 cell line, where p53 harbors two contiguous sets of exons 7–9, spanning the DBD and oligomerization domain, it is interesting to note that the <span class="blue">yeast</span> colonies were predominantly white (Figure 4B, dish b) with the split 5′ and the split 3′ assay (96 and 87% of white colonies, respectively) This suggests that the cells express a binding ability that is specific to wild-type p53 rather than mutated p53 (Table 2).<br><br>Transcription activity of SH-SY5Y, IGR-N-91, SK-N-BE(2), SK-N-AS and IGR-NB8 p53 variants in mammalian cells
To determine the transactivation ability of p53 variants in mammalian cells, we used a reporter gene strategy. The p53RE located within either the <span class="yellow">human</span> p21/WAF1 promoter or the intron 1 of the <span class="yellow">mouse</span> and <span class="yellow">human</span> BAX gene [(18) and D. Munsch, personnal communication) were cloned in a luciferase reporter gene plasmid upstream of the E1B minimal promoter as described in Materials and Methods. The p53-negative LAN-1 cells were cotransfected with the p53 vectors expressing the p53 cloned from either SH-SY5Y, IGR-N-91, SK-N-BE(2), SK-N-AS or IGR-NB8 and the luciferase reporter plasmids. Both p53RE were strongly stimulated in cells cotransfected with wt p53 cloned from SH-SY5Y, as compared to cells cotransfected with an empty plasmid. In contrast, none of the p53 variants was able to transactivate the expression of luciferase driven by either p21/WAF1 or BAX p53RE (Figure 5).
To test transactivation capability at the protein level, each variant was transfected into p53-negative LAN-1 cells and the stimulation of endogenous p21/Waf1 gene expression was analyzed by western blotting. As shown in Figure 6, in contrast to wt p53, none of the p53 variants was able to induce p21 protein accumulation.
We then tested for a possible dominant negative effect of these various mutants on wt p53-dependent transcriptional activity. To this end, SH-SY5Y cells were cotransfected with constructs encoding the luciferase gene driven by either the p21/Waf1 or BAX p53RE and the constructs expressing the various p53s cloned from IGR-NB8, SK-N-BE(2), IGR-N-91, SK-N-AS and IGR-NB8 NB cells or p53DD, a dominant negative mutant of wt p53 (24). The stress induced by transfection activated the transcriptional activity of the wt p53 expressed in SH-SY5Y, leading to a p53-dependent expression of luciferase as illustrated by the fact that coexpression of p53DD led to a substantial decrease in luciferase activity when compared to the luciferase activity of cells cotransfected with an empty plasmid (Figure 5). Compared to p53DD, mutants within the DBD isolated from IGR-N-91 or SK-N-BE(2) had only a moderate dominant negative effect on endogenous wt p53 transcriptional activity. More surprisingly, the transfection of the C-terminal truncated variant IGR-NB8 enhanced both BAX and p21/WAF1 p53RE activity. The coexpression of p53β cloned from SK-N-AS also enhanced BAX p53RE activity. A similar effect has been reported already for p53β by Bourdon et al. (22).
When combined, these results show that all the p53 variants isolated from the NB cells had lost the ability to specifically transactivate the p53 target genes. Their effect on the transcriptional activity of endogenous wt p53 expressed in SH-SY5Y cells, however, largely depended on the p53 domain affected by the modification.<br><br>All the identified p53 variants inhibited the induction of endogenous p21/WAF1 gene expression under stress conditions
We further examined whether the four p53 variants, SK-N-BE(2), IGR-N-91, SK-N-AS and IGR-NB8, had loss their ability to stimulate the endogenous expression of the p21/WAF1 gene, the archetypical cell cycle inhibitor and the true target of p53. To this end, cellular response to genotoxic stress was analyzed by western blot following treatment of the various cell lines with cisplatin, one of the most potent antitumor agents used in neuroblastoma. Results are presented in Figure 7. None of the mutant cells, regardless of the type of mutation, was able to induce p21/WAF1 protein accumulation, unlike the 3 p53 wild-type cells (SH-SY5Y, IMR-32 and LAN-5).<br><br>Genomic status of TP53 region in the various cell lines
According to Knudson's ‘two hit’ model of tumor suppressor gene functional inactivation, the mutation of one allele is supposed to be associated with a deletion of the second allele. To assess this genetic mechanism, we performed FISH experiments to search for deletions of one copy of the TP53 region, especially in cell lines with a mutated TP53 gene. For this purpose, metaphase preparations of the studied cell lines were cohybridized with a p53 DNA probe labeled in red (BAC clone RP11-199F11) as described previously (25) and a chromosome 17-specific centromeric probe labeled in green. The three cell lines shown previously to express a wt p53 protein (LAN5, IMR32 and SH-SY5Y), displayed as expected two signals with each probe, confirming the presence of both TP53 alleles in these cells (Figure 8 and Table 3). Conversely, IGR-N-91 and SK-N-AS cell lines displayed only one fluorescent signal for each probe, suggesting a whole chromosome 17 lost, or at least losses of the 17p arm and the centromeric region. The SK-N-BE(2) cell line has been described as containing only one chromosome 17 and one TP53 signal (10). In our analysis, only 10% of the cells displayed this characteristic, whilst most of the cells had two copies of both (Figure 8 and Table 3). p53 sequencing, however, confirmed the previously described mutation, and the absence of a normal allele, suggesting that the cells used in our study had acquired, during culture, an uniparental disomy for the TP53-mutated chromosome 17. Finally, the other two cell lines (LAN-1 and IGR-NB8) displayed highly variable genetic heterogeneity from one cell to the next (Table 3). Surprisingly, although p53 transcripts are extremely faintly expressed in LAN-1 cell line, all cells showed several FISH signals with the 167 kb BAC probe used here. To understand this apparent contradiction, an array-CGH experiment was performed on an oligo-array Agilent, which indicated a 133 kb interstitial deletion corresponding to the p53 coding region and the 97 kb upstream region. Accordingly, the fluorescent spots observed in FISH experiments on LAN-1 cells should be related to the hybridization of the 57 kb region downstream of p53 gene present in the BAC probe. IGRN-B8 cell line displayed a number of signals of both colors ranging from 0 to 4, with 87% of cells displaying a loss for one TP53 allele. Despite this genomic variability, analysis of the p53 protein showed a single shortened form in IGR-N-B8 cell line (Figure 1). Consequently, and as suggested for SK-N-BE(2) cell line, IGR-N-B8 cell line should contain a variable number of copies of chromosome 17 with mutated p53.<br><br>
DISCUSSION
The p53 gene, the ‘genome guardian’, is mutated in over 50% of <span class="yellow">human</span> cancers, with the most common mutations being missense mutations (>2/3 of mutations) (26). In <span class="yellow">human</span> neuroblastoma tumors, p53 mutations are rarely present at the time of diagnosis (5,27); however, oncogenic p53 mutations can be found in advanced neuroblastomas that often relapse following high-dose chemotherapy (10). In contrast, in breast cancers, it has been reported that p53 mutations might improve response to high-dose chemotherapy including therapy with epirubicin and cyclophosphamide (28).
An investigation into the p53 genomic status and functions of eight <span class="yellow">human</span> NB lines revealed that all five of the mutated cell lines had distinct genetic characteristics as is schematically represented in Figure 9: SK-N-BE(2) with a single missense mutation in the p53 gene, encoding a highly stable full-length protein. SK-N-AS and IGR-NB8 proteins, although they have intact transactivation and DBDs, were truncated at the C-terminus generating 341 and 326 amino acids respectively; they therefore lack the tetramerization domain that is essential for an active conformation. Very recently, Bourdon et al. (22) reported the putative occurrence of β and γ isoforms from different tissues due to alternate splicing that indicates the similarity to those of p73 and p63 as identified previously by Daniel Caput and co-workers (29). In the p53 isoforms scheme proposed by Bourdon et al. (22), the SK-N-AS cell line that elicits p53i9 protein expression is consistent with the p53β isoform. Genomic analysis reveals that the only occurrence of the p53β isoform in SK-N-AS results from a deletion spanning the intron 9/exon 10 junction. Similar to the p53β isoform in SK-N-AS, the p53 in IGR-NB8 that lacks 67 amino acids at C-terminus was, alone, unable to induce p21/WAF1 promoter activation except with endogenous wt-p53 on SH-SY5Y cells where transfection with IGR-NB8 significantly augmented the transcriptional activation of the p21/Waf-1 promoter (Figure 5A). Studies by other authors have reported the interaction between the C-terminal domain and another region that impedes the active conformation of p53, suggesting an allosteric model for p53 activity regulation (30). Such events have been demonstrated for the 342-stop mutant, generated by mutagenesis, which can modulate transactivation, growth and apoptosis (31). Moreover, Harms and Chen (32) reported that the C-terminal basic domain inhibits induction of the proapoptotic target gene insulin-like growth factor binding protein 3, suggesting that IGR-NB8 might induce this gene. IGR-N-91 had an abnormally high molecular weight protein due to the duplication of wild-type exons 7-8-9, thus affecting the DBD and OD; and LAN-1, with a mutation at codon 182 (Cys→stop) concurred with an earlier report showing extremely low levels of mRNA and undetectable protein expression (9).
Notably, all the p53 variants, including SK-N-AS (β isoform) and IGR-NB8 (C-terminal truncated p53), elicited a total lack of p21 promoter activation. In particular, the p53β isoform was unable to induce endogenous p21 expression in SK-N-AS (Figure 6), concurring with data obtained from in vitro transfection experiments in H1299 cells by Bourdon et al. (22). For the IGR-N-91 cells, although p53 was mutated and unable to transactivate the p21/WAF1 promoter, the FASAY global test was not conclusive since ∼80% of colonies were white and nearly 20% (see also Table 2), though not enough, were red. Moreover, in this particular line, standard sequencing on cDNA using primers located within each exon as used for routine tumor analysis was unable to detect any anomalies in p53 genetic status (data not shown). These results enlighten the limit of the conventional tests to detect a transcription inactivation of p53 brought by duplication within the DBD.
Analysis of p53 genomic status was explored by FISH experiments, in search for a potential biallelic inactivation of p53, with a mutation of one allele and a deletion of the second one. This situation was indeed clearly observed in IGR-N-91 and SK-N-AS cell lines, with an unambiguous loss of one chromosome 17p arm in all cells of both. SK-N-BE(2), LAN-1 and IGR-NB-8 cell lines showed a more complex genomic situation which should be relevant of variable copy numbers of chromosome 17 bearing in most cases (LAN-1) or in all cases [SKN-BE(2), IGR-NB8] the mutated characteristic p53 allele. Our data therefore clearly demonstrate that each technique has a role and a combination of techniques is required in order to correctly define the p53 phenotype and genotype in tumor and particularly in NB cells.
Our data enlighten a high frequency of the C-terminal abnormalities (3/5 mutated) in NB cell lines. For SK-N-AS and IGR-NB8, a part of the oligomerization domain was lost and IGR-N-91 gained an extra oligomerization domain. According to FASAY assay the p53 expressed in IGR-N-91 still specifically bind DNA but not the p53 expressed in SK-N-AS and IGR-NB8 in agreement with previous published data obtained by electrophoretic mobility shift assay (33).
With regards to biological relevance, different mutants within the DBD vary in their oncogenicity. They are classified into two types depending of the location of the mutation, mutations of class I occur in the DNA contact areas, while class II mutations occur in areas important for the conformational stability of p53 protein (30). Although both class I and class II mutants have loss its ability to specifically bind DNA, class II mutations have been shown to be more oncogenic than class I. However, to our knowledge the oncogenicity of mutant affecting the C-terminal domain have not been studied. The biological role of the C-terminal mutants needs now to be thoroughly investigated in NB tumors.<br><br>
<h3>pmcA1885664</h3>The recombination-associated protein RdgC adopts a novel toroidal architecture for DNA binding
Abstract
RecA plays a central role in the nonmutagenic repair of stalled replication forks in bacteria. RdgC, a recombination-associated DNA-binding protein, is a potential negative regulator of RecA function. Here, we have determined the crystal structure of RdgC from <span class="yellow">Pseudomonas aeruginosa</span>. The J-shaped monomer has a unique fold and can be divided into three structural domains: tip domain, center domain and base domain. Two such monomers dimerize to form a ring-shaped molecule of approximate 2-fold symmetry. Of the two inter-subunit interfaces within the dimer, one interface (‘interface A’) between tip/center domains is more nonpolar than the other (‘interface B’) between base domains. The structure allows us to propose that the RdgC dimer binds dsDNA through the central hole of ∼30 Å diameter. The proposed model is supported by our DNA-binding assays coupled with mutagenesis, which indicate that the conserved positively charged residues on the protein surface around the central hole play important roles in DNA binding. The novel ring-shaped architecture of the RdgC dimer has significant implications for its role in homologous recombination.<br><br>INTRODUCTION
Homologous recombination systems contribute to the maintenance of genome integrity and are essential for the repair of stalled replication forks (1). Genetic studies of the recombination-dependent growth C (rdgC) gene in bacterial systems have suggested that it is involved in DNA replication and recombination (2,3). In <span class="yellow">Neisseria gonorrhoeae</span> (the gonococcus), RdgC is required for efficient pilin antigenic variation and plays some role in cell growth (4,5). The <span class="yellow">Escherichia coli</span> rdgC gene encodes a DNA-binding protein of 34 kDa, whose expression level was at its highest during exponential phase, reaching its maximum at ∼1000 dimers per cell (2). Its level decreased sharply to ∼50 dimers per cell in stationary phase. This profile suggests that RdgC might function during the period of DNA replication (2).
The RecA proteins and their homologs play a key role in recombinational DNA repair in bacteria and eukaryotes. Their aberrant reactions could result in gross chromosomal rearrangements that lead to <span class="yellow">human</span> diseases, including cancer. Therefore, their functions must be tightly regulated, and understanding how the RecA family of recombinases is regulated is of utmost importance (1). The <span class="yellow">E. coli</span> RdgC protein is a potential negative regulator of the function of RecA (1). It inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). Sedimentation equilibrium data indicated that <span class="yellow">E. coli</span> RdgC exists in solution as a mixture of oligomeric states in equilibrium, most likely monomers, dimers and tetramers (1). This concentration-dependent change in the oligomeric state appears to affect its mode of binding to DNA and its capacity to inhibit RecA (1). The primary mechanism of RdgC inhibition appears to involve a simple competition for DNA-binding sites, especially on double-stranded DNA (dsDNA) (1).
Deletion of the <span class="yellow">E. coli</span> rdgC gene alone causes no obvious phenotype but is highly deleterious in strains lacking certain enzymes involved in recombination and replication restart (2,3). RdgC is essential for the growth of an <span class="yellow">E. coli</span> strain lacking PriA, indicating that it might affect replication fork progression or fork rescue (2). PriA provides a means to load the DnaB replicative helicase at DNA replication fork and D loop structures (2). RdgC is, therefore, a key factor in the rescue of stalled or broken forks and subsequent replication restart. dnaC suppressors of PriA can overcome this inviability, especially when RecF, RecO or RecR is inactivated, which indicates that RdgC avoids or counters the toxic effect of these proteins by limiting inappropriate RecA loading on SSB-coated ssDNA (2).
The DNA-binding ability of the RdgC protein has been demonstrated clearly using electrophoretic mobility shift assays (2). <span class="yellow">Escherichia coli</span> RdgC binds non-specifically to dsDNA and with higher affinity than ssDNA (2). RdgC from <span class="yellow">N. meningitidis</span> also binds DNA with little specificity for sequence or structure, like the <span class="yellow">E. coli</span> protein (4). RdgC exhibits no preference for DNA replication or recombination intermediates (2). However, no detectable domains or nuclease activity could be identified for the <span class="yellow">E. coli</span> RdgC protein (2,6). The complexes of <span class="yellow">E. coli</span> RdgC with both linear and supercoiled circular plasmid DNA were imaged using atomic force microscopy (7). RdgC was found to have an increased affinity to DNA ends and to promote bending of DNA (7). RdgC has the effect on DNA superstructure; the promotion of DNA condensation was observed at high protein concentrations (7). Recombination is largely enhanced by close contacts of distant regions along the DNA strands through condensation (7).
<span class="yellow">Pseudomonas aeruginosa</span> is a ubiquitous environmental Gram-negative bacterium that belongs to the γ subdivision of the Proteobacteria. Orthologs of the rdgC gene are found only in β and γ subdivisions of the Proteobacteria (2). The amino acid sequence of <span class="yellow">P. aeruginosa</span> RdgC, a 306-residue protein, shows 46 and 36% identities against those of <span class="yellow">E. coli</span> and <span class="yellow">N. meningitidis</span>, respectively. Despite the importance of DNA-binding ability of RdgC in its biological roles, the structural details of RdgC and the mechanism of its action remain unclear. Here, we report the crystal structure of <span class="yellow">P. aeruginosa</span> RdgC as determined by the multiwavelength anomalous diffraction (MAD) method of X-ray crystallography. It reveals that RdgC dimer is ring-shaped, with a central hole of ∼30 Å diameter. The inside surface around the central hole is rich in conserved, positively charged residues, which are implicated in DNA-binding by mutagenesis. We propose that dsDNA binds to RdgC through this central hole. Our structure provides a solid structural framework for a better understanding of the role of RdgC in homologous recombination.<br><br>MATERIALS AND METHODS
Protein expression and purification
The rdgC gene (PA3263) was amplified by the polymerase chain reaction (PCR) using the genomic DNA of <span class="yellow">P. aeruginosa</span> strain PAO1 as a template. The oligonucleotide primers designed using the published genome sequence (8) were 5′-G GAA TTC CAT ATG TGG TTC CGC AAT CTG CTC G-3′ (forward) and 5′-CCG CCG CTC GAG GAC GCC CTG GGG GAT TTC TTC-3′ (reverse). The bases in bold denote the NdeI and XhoI cleavage sites, respectively. The PCR product was digested with NdeI and XhoI, and was then inserted into the NdeI/XhoI-digested expression vector pET-21a(+) (Novagen). This vector construction adds an eight-residue tag (LEHHHHHH) to the carboxyl terminus of the gene product to facilitate protein purification.
The protein was expressed in <span class="yellow">E. coli</span> C41 (DE3) cells (9). The cells were grown at 37°C up to OD600 of 0.6 in Luria–Bertani medium that contained 50 μg ml−1 ampicillin, and the protein expression was induced by 0.5 mM isopropyl-β-d-thiogalactopyranoside (IPTG). The cells were allowed to grow at 20°C for 22 h after IPTG induction and were harvested by centrifugation at 4200 g (6000 rev min−1; Sorvall GSA rotor) for 10 min at 4°C. The cell pellet was resuspended in an ice-cold lysis buffer [20 mM Tris-HCl (pH 7.9), 500 mM sodium chloride, 5 mM imidazole] and was then homogenized with an ultrasonic processor. The crude cell extract was centrifuged at 36 000 g (18 000 rev min−1; Hanil Supra 21 K rotor) for 60 min at 4°C, and the recombinant protein in the supernatant fraction was purified in three chromatographic steps.
The first purification step utilized the C-terminal hexahistidine-tag by Ni2+-chelated HiTrap chelating column (GE Healthcare). The eluent was diluted 3-fold with 50 mM Tris-HCl at pH 7.4. The diluted solution was applied to a 20-ml heparin-Sepharose column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol and 1 mM EDTA. The protein was eluted with a linear gradient of 0–2.0 M sodium chloride in the same buffer. The first peak corresponded to high molecular aggregates of RdgC and the second peak to RdgC dimers, which was also consistent with our dynamic light scattering measurements. When the aggregates were not separated from the dimers, the protein readily aggregated within a day. The dimer fractions were concentrated to ∼6 mg ml−1 concentration using an YM10 ultrafiltration membrane (Millipore-Amicon) and further purified by gel filtration on a HiLoad XK 16 Superdex 200 prep-grade column (GE Healthcare), which was previously equilibrated with a buffer of 50 mM Tris-HCl (pH 7.4), 200 mM sodium chloride, 1 mM dithiothreitol and 1 mM EDTA. The first peak corresponding to the high molecular aggregates of RdgC was only a minor component, and the second peak corresponding to RdgC dimers was stable for several weeks when frozen at −70°C. The procedure for purifying the SeMet-substituted protein was the same, except for the presence of 10 mM dithiothreitol in all buffers used during purification steps. Dynamic light scattering experiments were performed at 24°C, with the protein (at 2 mg ml−1 concentration) dissolved in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride on a Model DynaPro-801 instrument (Wyatt, Santa Barbara, CA, USA). The protein concentration was estimated by measuring the absorbance at 280 nm, employing the calculated extinction coefficient of 20 910 M−1 cm−1 (SWISS-PROT; http://www.expasy.ch/).<br><br>Mutagenesis and electrophoretic mobility shift assay
Point mutants were prepared using the QuikChange Site-Directed Mutagenesis kit (Stratagene), and the mutations were confirmed by DNA sequencing. The seven mutants R4A, F120A, K146A, R198A, R208A, Q212A and R252A were well expressed and soluble. The three mutants K70A, R81A and R211A were not expressed, while R122A mutant was expressed but was not soluble. Therefore, we mutated Lys70, Arg81, Arg122 and Arg211 into aspartate. The K70D, R81D, R122D and R211D mutants were well expressed and soluble. The soluble mutants were expressed and purified under the conditions identical to those for the wild type, except that the heparin-Sepharose column step was omitted. All of the soluble mutants had similar elution profiles as the wild type upon gel filtration, and the dimer fractions were used for DNA-binding assays. We mixed 5 μl of RdgC (1.1 μg μl−1) and 5 μl of a linear dsDNA with 414 bp (63 ng μl−1), corresponding to the molar ratio of 64:1 for RdgC dimer to dsDNA. The reaction mixtures were incubated for 10 min at room temperature. RdgC–DNA complexes were analyzed by 1.5% (w/v) agarose gel electrophoresis in 1× TAE buffer (45 mM Tris-acetate at pH 8.3, 1 mM EDTA). DNA bands were visualized by ethidium bromide staining. For DNA-binding assays, the protein concentration was measured with a Bio-Rad protein assay kit.<br><br>Crystallization and X-ray data collection
Crystals were grown by the hanging-drop vapor diffusion method at 24°C by mixing equal volumes (2 μl each) of the protein solution (at 19 mg ml−1 concentration in a buffer consisting of 50 mM Tris-HCl (pH 7.4), 1 mM dithiothreitol, 1 mM EDTA and 200 mM sodium chloride) and the reservoir solution. To grow the native crystals, we used a reservoir solution consisting of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate and 1% (w/v) Anapoe®35 as the detergent. Native crystals grew to approximate dimensions of 0.2 mm × 0.1 mm × 0.1 mm within a few days. The SeMet-substituted protein was crystallized under conditions identical to those for the native crystals except for the presence of 10 mM dithiothreitol in the protein solution.
A crystal of the SeMet-substituted protein was dipped into a cryoprotectant solution for a few seconds and was flash-cooled in the cold nitrogen gas stream at 100 K. The cryoprotectant solution consisted of 0.1 M sodium HEPES (pH 7.5), 1.0 M tri-sodium citrate, 1% (w/v) Anapoe®35 as the detergent and 10% (v/v) glycerol. A set of Se MAD data was collected at 100 K at three different wavelengths using an Area Detector System Corporation Quantum 210 charge-coupled device detector at the beamline NW12A of Photon Factory, Tsukuba, Japan. The crystal was rotated through a total of 180° with a 1° oscillation range per frame. The raw data were processed and scaled using the program suit HKL2000 (10). The SeMet-substituted crystal belongs to the space group P21212, with unit cell parameters of a = 87.04 Å, b = 115.60 Å and c = 76.07 Å.
The native crystals were soaked for 10 min in 10 mM ethyl mercury thiosalicylate (EMTS) to collect a set of Hg MAD data at four different wavelengths. X-ray diffraction data from the EMTS-derivatized crystal, as well as a native crystal, were collected on a Bruker charge-coupled device detector at the beamline BL-6B of Pohang Light Source, Pohang, Korea. The EMTS-soaked crystal belongs to the space group P21212, with unit cell parameters of a = 87.16 Å, b = 116.02 Å and c = 74.97 Å. The native crystal belongs to the space group P21212, with unit cell parameters of a = 87.49 Å, b = 116.04 Å and c = 76.03 Å. If the presence of a dimeric molecule in the asymmetric unit is assumed, the calculated crystal volume per protein mass (VM) is 2.75 Å3 Da−1 and the solvent content is 55.3%. Supplementary Table 1 summarizes the statistics of X-ray diffraction data collection.<br><br>Structure solution and refinement
Heavy atom sites were located with the program SOLVE (11). Four Se sites from the SeMet-substituted crystal and two Hg sites from the EMTS-soaked crystal were located. Two sets of the initial Se and Hg MAD phases were separately improved using the program RESOLVE (12). Two electron density maps were independently interpreted, and the fragments of the two models were combined into a single model, since the two maps were complementary to each other. Non-crystallographic symmetry (NCS) matrices were found from a partial model built from the initial electron density map, and phases were further improved by the 2-fold NCS averaging, solvent flattening and histogram matching with the program DM (13). The model was built with O (14). The model was refined with the program CNS (15), including the bulk solvent correction. 10% of the data were randomly set aside as the test data for the calculation of Rfree (16). Several rounds of model building, simulated annealing, positional refinement and individual B-factor refinement were performed. Subsequently, this model was used to refine the structure of the native protein. The model has excellent stereochemistry, as evaluated by the program PROCHECK (17). Refinement statistics are summarized in Supplementary Table 1.<br><br>
RESULTS AND DISCUSSION
Model quality and monomer structure
The structure of <span class="yellow">P. aeruginosa</span> RdgC was solved using two sets of MAD data collected from a crystal of the selenomethionine (SeMet)-substituted protein and from a mercury derivative crystal of the native protein (Supplementary Table 1). The model has subsequently been refined using the 20.0–2.50 Å data from a native crystal to crystallographic Rwork and Rfree values of 23.4 and 27.9%, respectively, with no sigma cut-off. The refined model contains 612 residues of the two monomers in the asymmetric unit of the crystal (residues 1–306 for both chains A and B) and 87 water molecules. The C-terminal eight-residue fusion tag has no electron density in both subunits and has not been modeled. Conformations of the two monomers in the asymmetric unit are similar. The root-mean-square (r.m.s.) difference between the two monomers is 0.69 Å for 306 Cα atom pairs, with a maximum deviation of 8.8 Å for the Cα atom of Gly305. The residues giving r.m.s. differences larger than 2.0 Å are 14–16, 94–96, 201–202 and 303–306, which correspond to loops connecting secondary structure elements or the C-terminus. They are located on the surface of the dimer. Subunit A has a better map quality than subunit B, and thus has a lower mean B-factor than subunit B (40.3 Å2 versus. 56.1 Å2 for all 4788 atoms) (Supplementary Figure 1). Therefore, subunit A is chosen for discussions, unless otherwise stated.
The <span class="yellow">P. aeruginosa</span> RdgC monomer is J-shaped and has approximate dimensions of 72 Å × 60 Å × 40 Å (Figure 1). It can be divided into three structural domains: center domain, tip domain and base domain. Center domain (residues 1–73 and 121–167) includes a six-stranded anti-parallel β-sheet with two flanking α helices (α1 and α4). The helix α1 runs nearly parallel to the two outermost strands β2 and β3, which in turn lie adjacent to β4. Strands β2 and β3 are separated by a loop. The helix α4 is nearly orthogonal to the β-strands. Tip domain (residues 74–120) contains two α helices (α2 and α3) and is inserted between strands β4 and β5 of the center domain. It sticks out from the center domain. The two connections between the center domain and the tip domain are rich in conserved residues, hinting at an important functional role of these joints. They could act as a hinge for a possible conformational change. The C-terminal region of the polypeptide chain is folded into the base domain (residues 168–306), which consists of a five-stranded anti-parallel β-sheet with four α-helices, three of which (α5, α6 and α8) cover one side of the β-sheet. The long helix α8 is slightly bent.<br><br>
Structural similarity searches
When we searched for structural similarity against the Protein Data Bank database using the program DALI (18), the <span class="yellow">P. aeruginosa</span> RdgC monomer showed no significant similarity with a Z score above 5. Therefore, we conclude that the overall fold of the RdgC polypeptide chain is unique. When we elaborated the structural comparisons with individual domains of <span class="yellow">P. aeruginosa</span> RdgC, only the center domain gave a Z score above 5.
Using the center domain (residues 1–73 and 121–167) alone, the highest structural similarity is found with the <span class="yellow">human</span> β2-adaptin appendage domain (PDB code 1E42, Z score = 7.1, r.m.s. deviation = 2.5 Å for 85 structurally aligned residues, residues 1–12, 14–30, 44–47, 58–72, 122–133 and 135–160 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 838–841, 848–880, 885–906 and 912–967 of the <span class="yellow">human</span> β2-adaptin appendage domain chain A) (19). There does not appear to be any functional relatedness between these two proteins. The next highest similarity is found with the <span class="blue">yeast</span> TATA-box-binding protein (PDB code 1YTB, Z score = 5.7, r.m.s. deviation = 2.6 Å for 79 structurally aligned residues, residues 4–12, 14–19, 21–27, 56–70, 124–132 and 134–166 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 69–96, 100–126 and 132–155 of TATA-box-binding protein, N-terminal half of chain A) (20). The TATA-box-binding protein is composed of a ten-stranded antiparallel β-sheet with four flanking α-helices on the convex side of the β-sheet (20). It has an internal quasi 2-fold symmetry, and its β-strands are involved in DNA binding. There is, however, no structural similarity of functional significance, because the two center domains of the RdgC dimer are well separated and do not associate into a single unit (Figure 1C and D).
Using the tip domain (residues 74–120) alone, the ATPase domain of <span class="yellow">bovine</span> Hsc70 chaperone shows highest structural similarity (PDB code 1BA1, Z score = 4.8, r.m.s. deviation = 1.7 Å for 45 structurally aligned residues, residues 76–120 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 230–253 and 256–276 of the Hsc70 ATPase domain) (21). There does not seem to be any significant functional relationship between these two proteins. The α-helix-loop-α-helix structure of the tip domain is reminiscent of the helix–hairpin–helix (HhH) DNA-binding motif that is present in a number of DNA repair enzymes (22–25). The HhH motifs are characterized by the conserved sequence motif hxxhxGhGxxxAxxxhh, where h is any hydrophobic residue (VILMWFYA) and x any residue (23). Two conserved glycines allow a tight turn between two α-helices. However, the tip domain of <span class="yellow">P. aeruginosa</span> RdgC protein does not have such a sequence motif, and the loop in the RdgC tip domain is much more extended than that of the HhH motifs. Therefore, the RdgC tip domain is uniquely folded.
With the base domain (residues 168–306) alone, the highest similarity is found with <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase (unpublished deposition; PDB code 2H9C, Z score = 4.0, r.m.s. deviation = 4.8 Å for 51 structurally aligned residues, residues 186–191, 237–240, 257–272 and 274–298 of <span class="yellow">P. aeruginosa</span> RdgC chain A and residues 36–41, 50–53, 55–66 and 68–96 of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase chain A). There does not appear to be any functional relationship between the two proteins, and the base domain of RdgC is uniquely folded. The 101-residue chain of <span class="yellow">P. aeruginosa</span> isochorismate-pyruvate lyase is folded into three α-helices, two of which overlap with two α-helices (α2 and α3) of the base domain of RdgC.<br><br>Ring-shaped dimer structure and inter-subunit interface
Our structure reveals that <span class="yellow">P. aeruginosa</span> RdgC forms a ring-shaped dimer of approximate 2-fold symmetry in the crystal (Figure 1). This observation is consistent with the results of previous gel filtration and sedimentation equilibrium experiments, which suggested the presence of dimeric species of <span class="yellow">E. coli</span> RdgC in solution (1,2). Depending on the concentration, RdgC could also exist in solution as monomers, tetramers and higher oligomers (1). In the crystal lattice, no monomeric, trimeric or tetrameric species exist. Our structure unequivocally rules out the monomer–trimer model, which fits the sedimentation equilibrium data approximately, as well as the monomer–dimer–tetramer model (1). The RdgC dimer has approximate dimensions of 80 Å × 60 Å × 40 Å, with a central hole of ∼30 Å diameter.
Only a moderate amount of surface area is buried upon the formation of the RdgC dimer. The monomer–monomer interface buries a solvent-accessible surface area of ∼1360 Å2 per monomer, corresponding to ∼8% of the total surface area of the monomer (Protein–Protein Interaction Server at http://www.biochem.ucl.ac.uk/bsm/PP/server/). Polar and nonpolar atoms in interface are 44 and 56%, respectively. The interface can be divided into two kinds: ‘interface A’ involving both the tip domains and center domains, and ‘interface B’ involving the base domains (Figure 2A). These two interfaces have significantly different characters.<br><br>The interface A is formed largely by the crossing over of the two tip domains in the dimer (Figure 1). This interface covers a surface area of ∼565 Å2 and is largely nonpolar, with the polar and nonpolar atoms in the interface being 37.5 and 2.5%, respectively. Phe120 contributes considerably to the nonpolar character of the interface between the tip domains, accounting for ∼150 Å2 of the interface area. Phe120 is part of a highly conserved sequence segment and lies at the boundary between the tip domain and the center domain. The aromatic ring of Phe120′ inserts into a hydrophobic pocket, which is formed by the hydrophobic residues Ile74, Leu75, Pro76, Val79, Leu115, Ala119 and Phe120 of the neighboring monomer (Figure 2B). The primed residue belongs to the adjacent subunit. These residues are well conserved in the RdgC family (indicated by green triangles below the aligned sequences in Figure 3). As these hydrophobic residues are located near the molecular 2-fold axis, the two clusters of hydrophobic side chains are merged into a single, large cluster in the dimer (Figure 2B). Of the 16 hydrogen bonds that exist between the two subunits, only one is made in this interface between Arg118 and Arg118′.<br><br>The interface B is contributed to exclusively by the two base domains. This interface covers a surface area of ∼795 Å2. The polar and nonpolar atoms in this interface are 50% each. The two β-sheets of the two base domains merge into a single ten-stranded antiparallel β-sheet in this interface (Figure 2A). Thus, 15 of 16 inter-subunit hydrogen bonds are clustered in this interface. The hydrogen bond networks can be divided into two types. The first type involves the main chain atoms of Val206, Arg208 and Lys210 in strand β9; this strand associates with the equivalent strand from the other subunit in an antiparallel manner (Figure 2C). The other type involves both the side chain and main chain atoms. The side chain of Arg211′ interacts with the backbone of Gly204, while the side chain of Lys227 hydrogen bonds to those of both Gln212′ and Glu218 (Figure 2D). In addition, the side chain of Gln212′ interacts with that of Asp199 and the backbone of Arg211′ (Figure 2D). Arg211, Gln212, Glu218 and Lys227 are highly conserved among the RdgC family members. The residues participating in the hydrogen bond networks are indicated by red circles below the aligned sequences in Figure 3. The observation that conserved residues play important roles in forming the dimer implies that dimerization of RdgC is critical for its function.
If one or both of interfaces A and B are to be broken for functional reasons such as for allowing dsDNA to enter into the central hole of the RdgC dimer, we can conceive of three scenarios. The first possibility would be a simultaneous breakage of the two interfaces A and B. The second would be the sequential disruption of the interfaces A and B. The third would be the disruption of only one of the two interfaces A and B. Sedimentation equilibrium indicated the presence of monomers in solution (1), which is consistent with the first two scenarios. If the third scenario holds, RdgC dimers could possibly exist in two conformations in solution, i.e. a closed form as observed in the present crystal structure, and an open form, in which one of the two interfaces A and B is broken.<br><br>A model for DNA binding by RdgC and DNA-binding assay
The net charge of <span class="yellow">P. aeruginosa</span> RdgC is highly negative, since each monomer contains 12 aspartate, 13 glutamate, 10 lysine, 9 arginine and 2 histidine residues. The net charge of a dimer would be −12 (or −8), if we assume none (or all) of histidines to be positively charged. The electrostatic potential at the molecular surface of RdgC dimer shows that the charge distribution on the protein surface is highly nonuniform (Figure 4A). Since the inner surface of the RdgC dimer is rich in conserved, positively charged residues and the diameter of the central hole (∼30 Å) is approximately similar to that of other toroidal DNA-binding proteins (26), it appears likely that RdgC dimers bind dsDNA through the central hole. Despite extensive co-crystallization and soaking trials with linear dsDNAs of different lengths (14-mer, 16-mer and 18-mer), we could not locate the bound DNA experimentally. We attribute this difficulty to the nature of sequence-nonspecific DNA binding by RdgC. Therefore, we built a crude model of a DNA complex of the RdgC dimer by fitting dsDNA into the central hole (Figure 4B). This model is supported by the DNA-binding assays coupled with mutagenesis, as described below. It is also consistent with the suggested binding site size of ∼16 nt for the <span class="yellow">E. coli</span> RdgC dimer (1) and the approximate thickness (∼40 Å) of the <span class="yellow">P. aeruginosa</span> RdgC dimer.<br><br>When we performed electrophoretic mobility shift assay for dsDNA binding, <span class="yellow">P. aeruginosa</span> RdgC did not require Mg2+ ions (Supplementary Figure 2), unlike <span class="yellow">Deinococcus radiodurans</span> RecR, which required high concentrations of Mg2+ ions (27). ATP was not required for dsDNA binding by <span class="yellow">P. aeruginosa</span> RdgC, either (Supplementary Figure 2). This is consistent with an apparent lack of an ATP-binding pocket in the RdgC dimer structure.
In order to identify the residues that are important for dsDNA binding, we carried out mutational analyses. The residues for mutagenesis were selected on the basis of the modeled complex between RdgC dimer and dsDNA, together with sequence conservation (Figure 4C). We prepared eleven point mutants (R4A, K70D, R81D, F120A, R122D, K146A, R198A, R208A, R211D, Q212A and R252A). These eleven mutated residues are well conserved among RdgC proteins. Eight mutants (R4A, K70D, R81D, R122D, K146A, R198A, R208A and R252A) had a lower affinity for dsDNA than the wild-type RdgC (Figure 4D). These eight mutated residues are indicated by blue squares below the aligned sequences in Figure 3. This result indicates that dsDNA binds to the inside surface around the central hole of the RdgC dimer, since the mutated residues are located on the inner surface of the RdgC dimer. It is worth mentioning that a single mutation introduces two alterations per RdgC dimer.
The three mutants (F120A, R211D and Q212A) were prepared to investigate whether dimerization of RdgC is crucial for dsDNA binding. Phe120 is a key residue located in the inter-subunit interface A (Figure 2B), while Arg211 makes hydrogen bonds in the interface B (Figure 2D) and Gln212 is involved in both intra- and inter-subunit interactions (Figure 2D). dsDNA binding by the F120A mutant is considerably weakened, compared to either the R211D or Q212A mutant (Figure 4D). This result suggests that destabilization of the ring-shaped architecture of the RdgC dimer is deleterious for tight dsDNA binding and that interface A is probably more important than interface B for dsDNA binding. dsDNA-binding affinity of the Q212A mutant was lower than the wild type (Figure 4D), whereas that of the R211D mutant was comparable to the wild type (Figure 4D). This result suggests that disruption of the inter-subunit interface B is also detrimental for dsDNA binding but to a smaller extent than disruption of the interface A. It further suggests that the side chain of Arg211 is not involved in recognizing dsDNA. It is interesting that the DNA complex for wild type forms a distinct band (lane 3 in Figure 4D), as opposed to a more diffuse band as one would expect if the complexes contained different numbers of RdgC dimers. Assuming a binding site size of ∼16 nt (1), the 414-bp duplex DNA would bind ∼25 RdgC dimers if it is fully saturated.<br><br>RdgC is unique among ring-shaped DNA-binding proteins
<span class="yellow">Pseudomonas aeruginosa</span> RdgC adds a unique member to the repertoire of toroidal DNA-binding proteins (26) such as <span class="yellow">D. radiodurans</span> RecR (27) and DNA polymerase processivity factors, including the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29) and PCNA of eukaryotic DNA polymerase δ (30,31) (Figure 5). These proteins do not have an overall sequence similarity among themselves, and their quaternary structures also vary. <span class="yellow">P. aeruginosa</span> RdgC and <span class="yellow">E. coli</span> β clamp are homodimers, T4 gp45 and eukaryotic PCNAs are homotrimers and <span class="yellow">D. radiodurans</span> RecR is a homotetramer.<br><br>In <span class="yellow">P. aeruginosa</span> RdgC, the inner surface of the ring-shaped dimer is largely formed by β-strands, while the outer surface of the ring is mostly formed by α-helices (Figure 5). The predicted DNA-binding site of <span class="yellow">P. aeruginosa</span> RdgC dimer is primarily formed by β-strands and loops. In contrast, in other ring-shaped DNA-binding proteins such as the β subunit of <span class="yellow">E. coli</span> DNA polymerase III (28), gp45 of T4 DNA polymerase (29), PCNA of eukaryotic DNA polymerases δ (30,31) and <span class="yellow">D. radiodurans</span> RecR (27), α-helices are located toward the center of the ring and are surrounded by β-strands in the outermost side of the ring (Figure 5). Despite a superficial resemblance of the overall ring-shaped architecture of the RdgC dimer with those of other toroidal DNA-binding proteins, the internal arrangement of the secondary structure elements in the RdgC dimer is strikingly different from those of other toroidal DNA-binding proteins. Thus, RdgC is unique among ring-shaped DNA-binding proteins.
A number of other proteins involved in DNA metabolism also adopt a ring-shaped structure despite diversity in the molecular functions (26). They include NAD+-dependent DNA ligase (25), λ exonuclease (32), hexameric helicases, topoisomerases, the trp RNA-binding attenuation protein, the <span class="yellow">bacteriophage</span> head-to-tail connector and translin (26). There are also many examples of the proteins in DNA recombination pathways that assemble into multi-subunit toroids containing a central channel (26). Electron microscopy showed that the <span class="yellow">E. coli</span> RuvB protein, in the presence of ATP, forms a dodecamer on double-stranded DNA in which two stacked hexameric rings encircle the DNA and are oriented in opposite directions with D6 symmetry (33). The <span class="yellow">human</span> RAD52 protein was found to assemble into heptameric rings with a large funnel-shaped channel, 40–60 Å in diameter (34). The <span class="yellow">human</span> DMC1 protein has so far been detected only as an octameric ring (35).<br><br>Inhibition mechanism of RecA function by RdgC
RdgC inhibits RecA-promoted DNA strand exchange, RecA-mediated ATPase activity and RecA-dependent LexA cleavage (1). It was observed that the effects of RdgC on RecA-promoted DNA strand exchange were much more pronounced than the effects of RdgC on other RecA functions, when RdgC was added to the reaction late (1). By examining the effects of the DinI protein on RdgC-mediated inhibition of RecA activities, it was concluded that the potent effect of RdgC in the inhibition of strand exchange did not primarily reflect a displacement of RecA in the nucleoprotein filament by RdgC, but instead a binding of RdgC to the dsDNA substrate so as to make it unavailable to the filaments for strand exchange (1).
Our work contributes to a better understanding of the inhibition mechanism of RecA function by RdgC. The ring-shaped architecture with a central hole that is suitable for the binding of dsDNA lends RdgC with a capacity to hold dsDNA very tightly. Therefore, we suggest that RdgC dimers probably function as a tight dsDNA gripper and physically block access to the target dsDNA by RecA filaments, because tight encircling of dsDNA by RdgC dimers, possibly as a linear aggregate, would prevent the two strands from dissociating. However, it remains unclear how the closed-ring structure of the RdgC dimer might allow binding of a large number of RdgC dimers on the duplex DNA. It may be conceivable that each RdgC dimer can open temporarily at one or both of its two inter-subunit interfaces so that dsDNA can enter the central hole.
RdgC was at its highest level during exponential phase, reaching its maximum of ∼1000 dimers per <span class="yellow">E. coli</span> cell. Its level decreased sharply to ∼50 dimers per cell in stationary phase (2). Under normal cellular conditions, RecA is maintained at ∼1000 molecules per cell (http://www.els.net/). Following LexA repressor cleavage, the level of RecA protein in the cell increases by as much as 20-fold. RdgC inhibition of the RecA function by a simple competition mechanism at the cellular concentration of RdgC clearly requires a much higher affinity for DNA by RdgC than RecA. Possible protein–protein interactions between RdgC dimers might have a profound effect on the affinity for DNA, when a large number of RdgC dimers are bound on a duplex DNA. If only one of the two inter-subunit interfaces can open up and the 2-fold symmetry axes of the RdgC dimers are misaligned sufficiently, the escape of DNA from a linear array of interacting RdgC dimers would be extremely difficult. Alternatively, both inter-subunit interfaces of each dimer may open up simultaneously. A line of indirect evidence for the possible protein–protein interactions between RdgC dimers is provided by the cooperative binding of RdgC to DNA (7). Cooperativity of RdgC binding may be caused by local and distant protein–protein interactions (7). A kind of side-by-side interaction between two adjacent molecules of RdgC dimer is present in the crystal, burying an accessible surface area of ∼750 Å2 at the interface. The interaction involves mostly the less polar side surface of the dimer (left side of Figure 4A), with 2-fold symmetry axes of the two interacting RdgC molecules making an angle of ∼90° when viewed down the central holes and ∼15° when viewed perpendicular to the holes. This may explain the observed DNA bending by RdgC, which was speculated to be a necessary step in the working mechanism of RdgC, if one can assume that the strong protein–protein interactions in the crystal were relevant for protein–protein interactions in solution (7).
RdgC protein was shown to have a potent effect on RecA protein-promoted DNA strand exchange in <span class="yellow">E. coli</span> (1). In a series of experiments, RdgC was added after RecA filaments had formed on the ssDNA and 5 min after the linear dsDNA was added. With 3 μM RecA, a sharp reduction in DNA strand exchange products was seen with 0.4 μM RdgC, and the generation of products was abolished at 0.8 μM RdgC (1). This suggests that the binding affinity of RdgC to dsDNA is considerably higher than that of RecA. The high affinity of RdgC could be explained by its ring shape and the suggested mode of DNA binding. Tetrameric species of <span class="yellow">E. coli</span> RdgC, which started to form at around 2 µM concentrations, were speculated to be the species with the most potent inhibitory effect, possibly by having a higher affinity for DNA (1). Higher affinity for DNA of the tetrameric species could be understood, if the formation of a tetramer in solution involves a protein–protein interaction similar to the side-by-side interaction between two adjacent RdgC dimers in the crystal, although there is no experimental evidence. Obviously, there are still many open questions that need to be addressed in future biochemical experiments.<br><br>
SUMMARY
The crystal structure of RdgC, a recombination-associated DNA-binding protein, determined in this study reveals that the polypeptide chain is uniquely folded and two monomers associate to form a ring-shaped dimer with a central hole of ∼30 Å diameter. The inner surface around the central hole is rich in positively charged residues. These structural features suggest that dsDNA runs through the central hole when it binds to RdgC dimers. This idea is supported by our mutational studies, which indicate that the conserved, positively charged residues around the central hole are important for dsDNA binding. The toroidal architecture of the RdgC dimer provides a sound framework for a better understanding of its role in homologous recombination.
Coordinates
Atomic coordinates and the structure factor data have been deposited in the Protein Data Bank (accession code 2OWY).<br><br>
SUPPLEMENTARY DATA
Supplementary Data are available at NAR Online.<br><br>Supplementary Material<br><br>
</body></html>