
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.linneaus_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Agrobacterium tumefaciens</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="linneaus_only">LINNEAUS only</span><br><hr><h3>21075924</h3>argC Orthologs from Rhizobiales show diverse profiles of transcriptional efficiency and functionality in <span class="annotated ">Sinorhizobium meliloti</span>. <br><br>Several factors can influence ortholog replacement between closely related species. We evaluated the transcriptional expression and metabolic performance of ortholog substitution complementing a <span class="annotated ">Sinorhizobium meliloti</span> argC mutant with argC from Rhizobiales (<span class="match annotated ">Agrobacterium tumefaciens</span>, <span class="annotated ">Rhizobium etli</span>, and <span class="annotated ">Mesorhizobium loti</span>). The argC gene is necessary for the synthesis of arginine, an amino acid that is central to protein and cellular metabolism. Strains were obtained carrying plasmids with argC orthologs expressed under the speB and argC (<span class="annotated ">S. meliloti</span>) and lac (<span class="annotated ">Escherichia coli</span>) promoters. Complementation analysis was assessed by growth, transcriptional activity, enzymatic activity, mRNA levels, specific detection of ArgC proteomic protein, and translational efficiency. The argC orthologs performed differently in each complementation, reflecting the diverse factors influencing gene expression and the ability of the ortholog product to function in a foreign metabolic background. Optimal complementation was directly related to sequence similarity with <span class="annotated ">S. meliloti</span>, and was inversely related to species signature, with <span class="annotated ">M. loti</span> argC showing the poorest performance, followed by <span class="annotated ">R. etli</span> and <span class="annotated ">A. tumefaciens</span>. Different copy numbers of genes and amounts of mRNA and protein were produced, even with genes transcribed from the same promoter, indicating that coding sequences play a role in the transcription and translation processes. These results provide relevant information for further genomic analyses and suggest that orthologous gene substitutions between closely related species are not completely functionally equivalent. 
<h3>21396477</h3><span class="annotated ">Agrobacterium tumefasciens</span>-mediated transformation of the aquatic fungus <span class="annotated ">Blastocladiella emersonii</span>. <br><br><span class="match annotated ">Agrobacterium tumefaciens</span> is widely used for plant DNA transformation and more recently, has also been used to transform <span class="annotated ">yeast</span>, filamentous fungi and even <span class="annotated ">human</span> cells. Using this technique, we developed the first transformation protocol for the saprobic aquatic fungus <span class="annotated ">Blastocladiella emersonii</span>, a Blastocladiomycete localized at the base of <span class="unannotated ">fungal</span> phylogenetic tree, which has been shown as a promising and interesting model of study of cellular function and differentiation. We constructed binary T-DNA vectors containing hygromycin phosphotransferase (hph) or enhanced green fluorescent protein (egfp) genes, under the control of <span class="annotated ">Aspergillus nidulans</span> trpC promoter and terminator sequences. 24 h of co-cultivation in induction medium (IM) agar plates, followed by transfer to PYG-agar plates containing cefotaxim to kill <span class="annotated ">Agrobacterium tumefsciens</span> and hygromycin to select transformants, resulted in growth and sporulation of resistant transformants. Genomic DNA from the pool o resistant zoospores were shown to contain T-DNA insertion as evidenced by PCR amplification of hph gene. Using a similar protocol we could also evidence the expression of enhanced green fluorescent protein (EGFP) in zoospores derived from transformed cells. This protocol can also open new perspectives for other non-transformable closely related fungi, like the Chytridiomycete class. 
</body></html>