
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Agrobacteriumtumefaciens</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>21362493</h3>One step construction of Agrobacterium-Recombination-ready-plasmids (OSCAR), an efficient and robust tool for ATMT based gene deletion construction in fungi. <br><br>Increasing availability of genomic data and sophistication of analytical methodology in fungi has elevated the need for functional genomics tools in these organisms. Previously we reported a method called DelsGate for rapid preparation of deletion constructs for protoplast-mediated <span class="unannotated ">fungal</span> transformation systems, which is based on Gateway(R) technology. However, over the past several years <span class="match annotated ">Agrobacteriumtumefaciens</span>-mediated transformation (ATMT) has become the preferred genetic transformation method for an increasing number of fungi. Therefore, we developed a method for One Step Construction of Agrobacterium-Recombination-ready-plasmids (OSCAR), to rapidly create deletion constructs for ATMT systems. The OSCAR methodology involves PCR amplification of the upstream and downstream flanks of the gene of interest, using gene specific primers each with a 5' extension containing one of four different attB recombination sites, modified from the Invitrogen MultiSite Gateway(R) system. Amplified gene flanks are then mixed with specifically designed marker and binary vectors and treated with BP clonase, generating the deletion construct in a single cloning step. The entire process of deletion construct preparation can be accomplished in just 2days. Using OSCAR we generated eight targeted deletion constructs and used two of them to generate deletion mutants in <span class="annotated ">Verticillium dahliae</span> by ATMT. In summary, OSCAR methodology combines PCR and Gateway(R) technology to rapidly and robustly generate precise deletion constructs for <span class="unannotated ">fungal</span> ATMT and homologous gene replacement. 
</body></html>