
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.linneaus_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Aspergillus nidulans</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="linneaus_only">LINNEAUS only</span><br><hr><h3>20433937</h3>Fungal secondary metabolites - strategies to activate silent gene clusters.<br><br>Filamentous fungi produce a multitude of low molecular weight bioactive compounds. The increasing number of <span class="unannotated ">fungal</span> genome sequences impressively demonstrated that their biosynthetic potential is far from being exploited. In fungi, the genes required for the biosynthesis of a secondary metabolite are clustered. Many of these bioinformatically newly discovered secondary metabolism gene clusters are silent under standard laboratory conditions. Consequently, no product can be found. This review summarizes the current strategies that have been successfully applied during the last years to activate these silent gene clusters in filamentous fungi, especially in the genus Aspergillus. The techniques take advantage of genome mining, vary from the simple search for compounds with bioinformatically predicted physicochemical properties up to methods that exploit a probable interaction of microorganisms. Until now, the majority of successful approaches have been based on molecular biology like the generation of gene &quot;knock outs&quot;, promoter exchange, overexpression of transcription factors or other pleiotropic regulators. Moreover, strategies based on epigenetics opened a new avenue for the elucidation of the regulation of secondary metabolite formation and will certainly continue to play a significant role for the elucidation of cryptic natural products. The conditions under which a given gene cluster is naturally expressed are largely unknown. One technique is to attempt to simulate the natural habitat by co-cultivation of microorganisms from the same ecosystem. This has already led to the activation of silent gene clusters and the identification of novel compounds in <span class="match annotated ">Aspergillus nidulans</span>. These simulation strategies will help discover new natural products in the future, and may also provide fundamental new insights into microbial communication. 
<h3>20633690</h3>UreA, the major urea/H+ symporter in <span class="match annotated ">Aspergillus nidulans</span>. <br><br>We report here the characterization of UreA, a high-affinity urea/H+ symporter of <span class="match annotated ">Aspergillus nidulans</span>. The deletion of the encoding gene abolishes urea transport at low substrate concentrations, suggesting that in these conditions UreA is the sole transport system specific for urea in <span class="annotated ">A. nidulans</span>. The ureA gene is not inducible by urea or its precursors, but responds to nitrogen metabolite repression, necessitating for its expression the AreA GATA factor. In contrast to what was observed for other transporters in <span class="annotated ">A. nidulans</span>, repression by ammonium is also operative during the isotropic growth phase. The activity of UreA is down-regulated post-translationally by ammonium-promoted endocytosis. <span class="unannotated ">A</span> number of homologues of UreA have been identified in <span class="annotated ">A. nidulans</span> and other Aspergilli, which cluster in four groups, two of which contain the urea transporters characterized so far in fungi and plants. This phylogeny may have arisen by gene duplication events, giving place to putative transport proteins that could have acquired novel, still unidentified functions. 
<h3>20654725</h3>The roles played by <span class="match annotated ">Aspergillus nidulans</span> apoptosis-inducing factor (AIF)-like mitochondrial oxidoreductase (AifA) and NADH-ubiquinone oxidoreductases (NdeA-B and NdiA) in farnesol resistance. <br><br>Farnesol (FOH) is a nonsterol isoprenoid produced by dephosphorylation of farnesyl pyrophosphate, a catabolite of the cholesterol biosynthetic pathway. These isoprenoids inhibit proliferation and induce apoptosis. Here, we show that <span class="match annotated ">Aspergillus nidulans</span> AifA encoding the apoptosis-inducing factor (AIF)-like mitochondrial oxidoreductase plays a role in the function of the mitochondrial Complex I. Additionally, we demonstrated that ndeA-B and ndiA encode external and internal alternative NADH dehydrogenases, respectively, that have a function in FOH resistance. When exposed to FOH, the DeltaaifA and DeltandeA strains have increased ROS production while DeltandeB, DeltandeA DeltandeB, and DeltandiA mutant strains showed the same ROS accumulation than in the absence of FOH. We observed several compensatory mechanisms affecting the differential survival of these mutants to FOH. 
<h3>20713166</h3>Characterization of the <span class="match annotated ">Aspergillus nidulans</span> biotin biosynthetic gene cluster and use of the bioDA gene as a new transformation marker. <br><br>The genes involved in the biosynthesis of biotin were identified in the hyphal fungus <span class="match annotated ">Aspergillus nidulans</span> through homology searches and complementation of <span class="annotated ">Escherichia coli</span> biotin-auxotrophic mutants. Whereas the 7,8-diaminopelargonic acid synthase and dethiobiotin synthetase are encoded by distinct genes in bacteria and the <span class="annotated ">yeast</span> <span class="annotated ">Saccharomyces cerevisiae</span>, both activities are performed in <span class="annotated ">A. nidulans</span> by a single enzyme, encoded by the bifunctional gene bioDA. Such a bifunctional bioDA gene is a genetic feature common to numerous members of the ascomycete filamentous fungi and basidiomycetes, as well as in plants and oomycota. However, unlike in other eukaryota, the three bio genes contributing to the four enzymatic steps from pimeloyl-CoA to biotin are organized in a gene cluster in pezizomycotina. The <span class="annotated ">A. nidulans</span> auxotrophic mutants biA1, biA2 and biA3 were all found to have mutations in the 7,8-diaminopelargonic acid synthase domain of the bioDA gene. Although biotin auxotrophy is an inconvenient marker in classical genetic manipulations due to cross-feeding of biotin, transformation of the biA1 mutant with the bioDA gene from either <span class="annotated ">A. nidulans</span> or <span class="annotated ">Aspergillus fumigatus</span> led to the recovery of well-defined biotin-prototrophic colonies. The usefulness of bioDA gene as a novel and robust transformation marker was demonstrated in co-transformation experiments with a green fluorescent protein reporter, and in the efficient deletion of the laccase (yA) gene via homologous recombination in a mutant lacking non-homologous end-joining activity. 
<h3>20797444</h3>Comparison of transcriptional and translational changes caused by long-term menadione exposure in <span class="match annotated ">Aspergillus nidulans</span>. <br><br>Under long-term oxidative stress caused by menadione sodium bisulfite, genome-wide transcriptional and proteome-wide translational changes were compared in <span class="match annotated ">Aspergillus nidulans</span> vegetative cells. The comparison of proteomic and DNA microarray expression data demonstrated that global gene expression changes recorded with either flip-flop or dendrimer cDNA labeling techniques supported proteome changes moderately with 40% and 34% coincidence coefficients, respectively. Enzyme levels in the glycolytic pathway were alternating, which was a direct consequence of fluctuating gene expression patterns. Surprisingly, enzymes in the vitamin B2 and B6 biosynthetic pathways were repressed concomitantly with the repression of some protein folding chaperones and nuclear transport elements. Under long-term oxidative stress, the peroxide-detoxifying peroxiredoxins and cytochrome c peroxidase were replaced by thioredoxin reductase, a nitroreductase and a flavohemoprotein, and protein degradation became predominant to eliminate damaged proteins. 
<h3>20816830</h3>Cross-talk between light and glucose regulation controls toxin production and morphogenesis in <span class="match annotated ">Aspergillus nidulans</span>. <br><br>Light is a major environmental stimulus that has a broad effect on organisms, triggering a cellular response that results in an optimal adaptation enhancing fitness and survival. In fungi, light affects growth, and causes diverse morphological changes such as those leading to reproduction. Light can also affect <span class="unannotated ">fungal</span> metabolism, including the biosynthesis of natural products. In this study we show that in <span class="match annotated ">Aspergillus nidulans</span> the effect of light on the production of the sterigmatocystin (ST) toxin depends on the glucose concentration. In cultures grown with 1% glucose and exposed to light, ST production was lower than when grown in the dark. This lower ST production coincided with an elevated rate of cellular damage with partial loss of nuclear integrity and vacuolated cytoplasm. However, in cultures grown with 2% glucose these effects were reversed and light enhanced ST production. Glucose abundance also affected the light-dependent subcellular localization of the VeA (velvet) protein, a key regulator necessary for normal light-dependent morphogenesis and secondary metabolism in Aspergilli and other <span class="unannotated ">fungal</span> genera. The role of other VeA-associated proteins, particularly the blue-light-sensing proteins LreA and LreB (WC-1 and WC-2 orthologs), on conidiation could also be modified by the abundance of glucose. We also show that LreA and LreB, as well as the phytochrome FphA, modulate not only the synthesis of sterigmatocystin, but also the production of the antibiotic penicillin. 
<h3>20817115</h3>Characterization of the developmental regulator FlbE in <span class="annotated ">Aspergillus fumigatus</span> and <span class="match annotated ">Aspergillus nidulans</span>. <br><br>Several upstream developmental activators control asexual development (conidiation) in Aspergillus. In this study, we characterize one of such activators called flbE in <span class="annotated ">Aspergillus fumigatus</span> and <span class="match annotated ">Aspergillus nidulans</span>. The predicted FlbE protein is composed of 222 and 201 aa in <span class="annotated ">A. fumigatus</span> and <span class="annotated ">A. nidulans</span>, respectively. While flbE is transiently expressed during early phase of growth in <span class="annotated ">A. nidulans</span>, it is somewhat constitutively expressed during the lifecycle of <span class="annotated ">A. fumigatus</span>. The deletion of flbE causes reduced conidiation and delayed expression of brlA and vosA in both species. Moreover, FlbE is necessary for salt-induced development in liquid submerged culture in <span class="annotated ">A. fumigatus</span>. The <span class="annotated ">A. nidulans</span> flbE null mutation is fully complemented by <span class="annotated ">A. fumigatus</span> flbE, indicating a functional conservancy of FlbE in Aspergillus. Both the deletion and overexpression of flbE in <span class="annotated ">A. nidulans</span> result in developmental defects, enhanced autolysis, precocious cell death, and delayed expression of brlA/vosA, suggesting that balanced activity of FlbE is crucial for proper growth and development. Importantly, the N-terminal portion of FlbE exhibits the trans-activation ability in <span class="annotated ">yeast</span>, whereas the C-terminal half negatively affects its activity. Site-directed mutagenesis of certain conserved N-terminal amino acids abolishes the ability of trans-activation, overexpression-induced autolysis, and complementing the null mutation. Finally, overexpression of flbD, but not flbB or flbC, restores conidiation in <span class="annotated ">A. nidulans</span> DeltaflbE, generally supporting the current genetic model for developmental regulation. 
<h3>20955810</h3>Novel mutations reveal two important regions in <span class="match annotated ">Aspergillus nidulans</span> transcriptional activator MetR. <br><br>Expression of the sulfur assimilation pathway in <span class="match annotated ">Aspergillus nidulans</span> is under control of sulfur metabolite repression, which is composed of scon genes encoding subunits of ubiquitin ligase and the metR gene coding for a transcriptional activator. In this paper we report three dominant suppressors of methionine requirement isolated from a metB3 diploid strain. All three mutations lead to the substitution of phenylalanine 48 by serine or leucine in the conserved N-terminal region of the MetR protein. Strains carrying the dominant suppressor mutations exhibit increased activities of homocysteine synthase and sulfur assimilation enzymes as well as elevated levels of the corresponding transcripts. These changes are observed even under conditions of methionine repression, which suggests that the mutated MetR protein may be resistant to inactivation or degradation mediated by sulfur metabolite repression. We also found that a mutant impaired in sulfite reductase activity, known until now as sG8, has a frameshift which changes 41 C-terminal amino acids. Therefore, it is now designated metR18. This mutant has elevated levels of MetR-regulated transcripts and of activities of sulfur assimilation enzymes (except sulfite reductase), which can be repressed to the wild type level by exogenous methionine. Thus, metR18 and the three dominant suppressors represent new types of mutations affecting different parts of the <span class="annotated ">A. nidulans</span> MetR protein. 
<h3>21396477</h3><span class="annotated ">Agrobacterium tumefasciens</span>-mediated transformation of the aquatic fungus <span class="annotated ">Blastocladiella emersonii</span>. <br><br><span class="annotated ">Agrobacterium tumefaciens</span> is widely used for plant DNA transformation and more recently, has also been used to transform <span class="annotated ">yeast</span>, filamentous fungi and even <span class="annotated ">human</span> cells. Using this technique, we developed the first transformation protocol for the saprobic aquatic fungus <span class="annotated ">Blastocladiella emersonii</span>, a Blastocladiomycete localized at the base of <span class="unannotated ">fungal</span> phylogenetic tree, which has been shown as a promising and interesting model of study of cellular function and differentiation. We constructed binary T-DNA vectors containing hygromycin phosphotransferase (hph) or enhanced green fluorescent protein (egfp) genes, under the control of <span class="match annotated ">Aspergillus nidulans</span> trpC promoter and terminator sequences. 24 h of co-cultivation in induction medium (IM) agar plates, followed by transfer to PYG-agar plates containing cefotaxim to kill <span class="annotated ">Agrobacterium tumefsciens</span> and hygromycin to select transformants, resulted in growth and sporulation of resistant transformants. Genomic DNA from the pool o resistant zoospores were shown to contain T-DNA insertion as evidenced by PCR amplification of hph gene. Using a similar protocol we could also evidence the expression of enhanced green fluorescent protein (EGFP) in zoospores derived from transformed cells. This protocol can also open new perspectives for other non-transformable closely related fungi, like the Chytridiomycete class. 
<h3>21419234</h3>Completing the purine utilisation pathway of <span class="match annotated ">Aspergillus nidulans</span>.<br><br>We have previously identified by classical genetics and biochemistry, all the genes of <span class="match annotated ">Aspergillus nidulans</span> predicted to be involved in purine utilisation, together with cognate regulatory genes and one gene encoding a novel xanthine hydroxylation activity. In this article we complete the description of the purine utilisation pathway with the identification of the two genes (uaX and uaW) encoding the enzymes catalysing the conversion of the product of urate oxidation by urate oxidase, 5-hydroxyisourate, to optically active allantoin. The identification of these additional genes confirms the complete absence of clustering of the genes involved in purine utilisation in <span class="annotated ">A. nidulans</span>. 
<h3>21807107</h3>Microtubule dynamics in mitosis in <span class="match annotated ">Aspergillus nidulans</span>. <br><br>Mitosis in <span class="match annotated ">Aspergillus nidulans</span> is very rapid, requiring less than 5 min at 37 ^0C in germlings (Bergen and Morris, 1983). In this time the cytoplasmic microtubules (MTs) must disassemble, the mitotic spindle assemble, function and disassemble, and cytoplasmic MTs reassemble. It follows that cytoplasmic MTs must be extremely dynamic in this period and we were interested, in particular, in examining the processes of MT disassembly in prophase and reassembly in anaphase and telophase. We observed a diploid strain that expressed GFP-alpha-tubulin. We used a spinning disk confocal microscope that allowed rapid image capture, which proved necessary because microtubule dynamics were extremely rapid. We found, for the first time, that microtubule severing occurs in prophase in a filamentous fungus and that catastrophe rather than nucleation limits astral microtubule growth. 
<h3>22001287</h3>Involvement of a helix-loop-helix transcription factor CHC-1 in CO(2)-mediated conidiation suppression in <span class="annotated ">Neurospora crassa</span>. <br><br>The morphological switch from vegetative growth to conidiation in filamentous fungi is highly regulated, but the understanding of the regulatory mechanisms is limited. In this study, by screening a set of knock-out mutants corresponding to 103 transcription factor encoding genes in <span class="annotated ">Neurospora crassa</span>, a mutant was found to produce abundant conidia in race tubes in which conidiation in the wild-type strain was suppressed. The corresponding gene NCU00749 encodes a protein containing a helix-loop-helix DNA binding region. Unlike enhanced conidiation in ras-1 and sod-1 mutants, which was completely suppressed by antioxidant N-acetyl cysteine, enhanced conidiation in the NCU00749 mutant was only slightly affected by N-acetyl cysteine. When grown on slants, the NCU00749 deletion mutant exhibited earlier conidial formation than the wild-type strain, and this was more evident at a higher (5%) CO(2) concentration. Therefore, we named NCU00749 as conidiation at high carbon dioxide-1 (chc-1). Genes that are highly expressed during conidial development, eas, con-6, con-8 and con-10, were transcribed at a higher rate in the chc-1 deletion mutant than the wild-type strain in response to conidiation induction. To determine the mechanisms by which CHC-1 regulates conidiation, we conducted a RNA sequencing analysis and found that 404 genes exhibited >= 2 fold changes in transcription in response to chc-1 deletion. Among them, fluffy and ada-6, two transcription factor genes that positively regulate conidiation in <span class="annotated ">N. crassa</span>, and rca-1, whose homolog flbD in <span class="match annotated ">Aspergillus nidulans</span> is essential for conidiation, were upregulated in the chc-1 deletion mutant. Results of RNA sequencing also suggest that signal transduction via the cAMP and the MAK-2 mediated signal pathways, and ROS generation and removal, mechanisms known to regulate conidiation, are not involved in chc-1 mediated control of conidiation. In addition, chc-1 also influences expression of genes involved in other important biological processes besides conidiation such as carbon metabolism, sphingolipid synthesis, cell wall synthesis, and calcium signaling. 
<h3>21220038</h3>acon-3, the <span class="annotated ">Neurospora crassa</span> ortholog of the developmental modifier, medA, complements the conidiation defect of the <span class="match annotated ">Aspergillus nidulans</span> mutant. <br><br><span class="match annotated ">Aspergillus nidulans</span> and <span class="annotated ">Neurospora crassa</span> are ascomycetes that produce asexual spores through morphologically distinct processes. MedA, a protein with unknown function, is required for normal asexual and sexual development in <span class="annotated ">A. nidulans</span>. We determined that the <span class="annotated ">N. crassa</span> ortholog of medA is acon-3, a gene required for early conidiophore development and female fertility. To test hypotheses about the evolutionary origins of asexual development in distinct <span class="unannotated ">fungal</span> lineages it is important to understand the degree of conservation of developmental regulators. The amino acid sequences of <span class="annotated ">A. nidulans</span> MedA and <span class="annotated ">N. crassa</span> ACON-3 shared 37% identity and 51% similarity. acon-3 is induced at late time points of conidiation. In contrast, medA is constitutively expressed and MedA protein localizes to nuclei in all tissue types. Nonetheless, expression of acon-3 using its native promoter complemented the conidiation defects of the <span class="annotated ">A. nidulans</span> DeltamedA and medA15 mutants. We conclude that the biochemical activity of the medA orthologs is conserved for conidiation. 
<h3>21277379</h3>Independent duplications of alpha-amylase in different strains of <span class="annotated ">Aspergillus oryzae</span>. <br><br><span class="annotated ">Aspergillus oryzae</span> is a filamentous fungus that has arisen through the ancient domestication of <span class="annotated ">Aspergillus flavus</span> for making traditional oriental foods and beverages. In the many centuries <span class="annotated ">A. oryzae</span> has been used for fermenting the starch in <span class="unannotated ">rice</span> to simple sugars, it has undergone selection for increased secretion of starch-degrading enzymes. In particular, all <span class="annotated ">A. oryzae</span> strains investigated thus far have two or more copies of a gene encoding alpha-amylase, whereas <span class="annotated ">A. flavus</span> has only one. Here we investigate the duplications leading to these copies in three <span class="annotated ">A. oryzae</span> strains. We find evidence of at least three separate duplications of alpha-amylase, an example of parallel evolution in a micro-organism under artificial selection. At least two of these duplications appear to be associated with activity of transposable elements of the Tc1/mariner class. Both involve a 9.1 kb element that terminates in inverted repeats, encodes a putative transposase and another putative protein of unknown function, and contains an unusual arrangement of four short internal imperfect repeats. Although &quot;unusual Mariners&quot; of this size have previously been identified in <span class="annotated ">A. oryzae</span>, <span class="annotated ">Aspergillus fumigatus</span> and <span class="match annotated ">Aspergillus nidulans</span>, this is the first evidence we know of that at least some of them are active in modern times and that their activity can contribute to beneficial genetic changes. 
</body></html>