
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Aspergillus niger</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>21176790</h3>Characterization of a polyketide synthase in <span class="match annotated ">Aspergillus niger</span> whose product is a precursor for both dihydroxynaphthalene (DHN) melanin and naphtho-gamma-pyrone. <br><br>The genome sequencing of the fungus <span class="match annotated ">Aspergillus niger</span> uncovered a large cache of genes encoding enzymes thought to be involved in the production of secondary metabolites yet to be identified. Identification and structural characterization of many of these predicted secondary metabolites are hampered by their low concentration relative to the known <span class="annotated ">A. niger</span> metabolites such as the naphtho-gamma-pyrone family of polyketides. We deleted a non-reducing PKS gene in <span class="annotated ">A. niger strain</span> ATCC 11414, a daughter strain of <span class="annotated ">A. niger ATCC strain 1015</span> whose genome was sequenced by the DOE Joint Genome Institute. This PKS encoding gene we name albA is a predicted ortholog of alb1 from <span class="annotated ">Aspergillus fumigatus</span> which is responsible for production of the naphtho-gamma-pyrone precursor for the 1,8-dihydroxynaphthalene (DHN) melanin/spore pigment. Our results show that the <span class="annotated ">A. nigeralbA</span> PKS is responsible for both the production of the spore pigment precursor and a family of naphtho-gamma-pyrones commonly found in significant quantity in <span class="annotated ">A. niger</span> culture extracts. The generation of an <span class="annotated ">A. niger</span> strain devoid of naphtho-gamma-pyrones will greatly facilitate the elucidation of cryptic biosynthetic pathways in this organism. 
<h3>21277986</h3>The molecular and genetic basis of conidial pigmentation in <span class="match annotated ">Aspergillus niger</span>.<br><br>A characteristic hallmark of <span class="match annotated ">Aspergillus niger</span> is the formation of <span class="unannotated ">black</span> conidiospores. We have identified four loci involved in spore pigmentation of <span class="annotated ">A. niger</span> by using a combined genomic and classical complementation approach. First, we characterized a newly isolated color mutant, colA, which lacked pigmentation resulting in white or colorless conidia. Pigmentation of the colA mutant was restored by a gene (An12g03950) which encodes a putative 4'phosphopantetheinyl transferase protein (PptA). 4'Phosphopantetheinyl transferase activity is required for the activation of Polyketide Synthases (PKSs) and/or Non-Ribosomal Peptide Synthases (NRPSs). The loci whose mutation resulted in fawn, olive, and brown color phenotypes were identified by complementation. The fawn phenotype was complemented by a PKS protein (FwnA, An09g05730), the ovlA mutant by An14g05350 (OlvA) and the brnA mutant by An14g05370 (BrnA), the respective homologs of alb1/pksP, ayg1 and abr1 in <span class="annotated ">A. fumigatus</span>. Targeted disruption of the pptA, fwnA, olvA and brnA genes confirmed the complementation results. Disruption of the pptA gene abolished synthesis of all polyketides and non-ribosomal peptides, while the naphtho-gamma-pyrone subclass of polyketides were specifically dependent on fwnA, and funalenone on fwnA, olvA and brnA. Thus, secondary metabolite profiling of the color mutants revealed a close relationship between polyketide synthesis and conidial pigmentation in <span class="annotated ">A. niger</span>. 
<h3>21324422</h3>A clone-based transcriptomics approach for the identification of genes relevant for itaconic acid production in Aspergillus. <br><br>Several Aspergillus species are well-known for the production of a variety of organic acids. In this study, a cloned based transcriptomics approach was used to identify genes crucial in the biosynthesis pathway for one of these acids, itaconic acid. From a number of different <span class="annotated ">Aspergillus terreus</span> controlled batch fermentations, those cultures with the largest difference in itaconic acid titer and productivity were selected for mRNA isolation. cDNAs derived from these mRNA samples were used for subsequent hybridization of glass slide arrays based on a collection of 5000 cDNA clones. <span class="unannotated ">A</span> selection of 13 cDNA clones resulting in the strongest (>10-fold) differential hybridization signals were identified and subsequently the inserts of these clones were sequenced. Sequence analysis revealed the presence of in total five different gene inserts among the sequenced clones. From one of these sequences, encoding a gene of the MmgE-PrpD family, the full length coding region was shown to encode one of the crucial itaconic acid pathway enzymes cis-aconitate decarboxylase, by heterologous expression in <span class="unannotated ">Escherichia coli</span>. Expression of this gene in <span class="match annotated ">Aspergillus niger</span>, which is a natural citric acid producer, resulted in itaconate production. Genome analysis suggests that in <span class="annotated ">A. terreus</span> the cis-aconitate decarboxylase gene is part of an itaconate acid related gene cluster including genes encoding two pathway specific transporters and a Zinc finger protein. Interestingly, this cluster is directly linked to the large lovastatin gene cluster. 
</body></html>