
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Bacillus subtilis</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>20173013</h3>Streptomyces lacticiproducens sp. nov., a lactic acid-producing streptomycete isolated from the rhizosphere of <span class="annotated ">tomato</span> plants. <br><br>A novel actinomycete, designated strain GIMN4.001(T), was isolated from the rhizosphere of <span class="annotated ">tomato</span> plants grown in Guangzhou, China. The strain produced greyish white aerial mycelia, lactic acid and a large quantity of double diamond-shaped crystals on <span class="annotated ">potato</span> dextrose agar and <span class="annotated ">yeast</span> extract-malt extract agar. The colour of the substrate mycelium was not sensitive to pH. Microscopic observations revealed that strain GIMN4.001(T) produced verticillate chains of cylindrical spores. Chemotaxonomic data confirmed that strain GIMN4.001(T) belonged to the genus Streptomyces. Melanin pigments were not produced. No antibacterial activity was observed against <span class="annotated ">Escherichia coli</span>, <span class="annotated ">Pseudomonas aeruginosa</span>, <span class="match annotated ">Bacillus subtilis</span> or <span class="annotated ">Candida albicans</span>, but inhibitory activity was observed against <span class="annotated ">Penicillium citrinum</span>. 16S rRNA gene sequence analysis revealed that strain GIMN4.001(T) was related most closely to <span class="annotated ">Streptomyces morookaense</span> <span class="annotated ">ATCC 19166(T)</span> (98.9 % similarity) and <span class="annotated ">Streptomyces lavenduligriseus</span> <span class="annotated ">ATCC 13306(T)</span> (98.7 %). Levels of DNA-DNA relatedness between strain GIMN4.001(T) and the type strains of these species were low (14-20 %). Furthermore, strain GIMN4.001(T) could be differentiated from <span class="annotated ">S. morookaense</span>, S. lavenduligriseus and other closely related species of the genus Streptomyces based on morphological, physiological and biochemical characteristics. On the basis of its physiological and molecular properties, strain GIMN4.001(T) is considered to represent a novel species of the genus Streptomyces, for which the name Streptomyces lacticiproducens sp. nov. is proposed. The type strain is GIMN4.001(T) (=CCTCC M208214(T)=NRRL B-24800(T)). 
<h3>20971907</h3>Mismatch repair modulation of MutY activity drives <span class="match annotated ">Bacillus subtilis</span> stationary-phase mutagenesis. <br><br>Stress-promoted mutations that occur in nondividing cells (adaptive mutations) have been implicated strongly in causing genetic variability as well as in species survival and evolutionary processes. Oxidative stress-induced DNA damage has been associated with generation of adaptive His(+) and Met(+) but not Leu(+) revertants in <span class="annotated ">strain Bacillus subtilis YB955 (hisC952 metB5 leuC427)</span>. Here we report that an interplay between MutY and MutSL (mismatch repair system [MMR]) plays a pivotal role in the production of adaptive Leu(+) revertants. Essentially, the genetic disruption of MutY dramatically reduced the reversion frequency to the leu allele in this model system. Moreover, the increased rate of adaptive Leu(+) revertants produced by a <span class="annotated ">MutSL knockout strain</span> was significantly diminished following mutY disruption. Interestingly, although the expression of mutY took place during growth and stationary phase and was not under the control of RecA, PerR, or sigma(B), a null mutation in the mutSL operon increased the expression of mutY several times. Thus, in starved cells, saturation of the MMR system may induce the expression of mutY, disturbing the balance between MutY and MMR proteins and aiding in the production of types of mutations detected by reversion to leucine prototrophy. In conclusion, our results support the idea that MMR regulation of the mutagenic/antimutagenic properties of MutY promotes stationary-phase mutagenesis in <span class="annotated ">B. subtilis</span> cells. 
<h3>21097612</h3>Differential responses of <span class="match annotated ">Bacillus subtilis</span> rRNA promoters to nutritional stress. <br><br>The in vivo expression levels of four rRNA promoter pairs (rrnp(1)p(2)) of <span class="match annotated ">Bacillus subtilis</span> were determined by employing single-copy lacZ fusions integrated at the amyE locus. The rrnO, rrnJ, rrnD, and rrnB promoters displayed unique growth rate regulation and stringent responses. Both lacZ activity and mRNA levels were highest for rrnO under all growth conditions tested, while rrnJ, rrnB, and rrnD showed decreasing levels of activity. During amino acid starvation induced by serine hydroxamate (SHX), only the strong rrnO and rrnJ promoters demonstrated stringent responses. Under the growth conditions used, the rrn promoters showed responses similar to the responses to carbon source limitation induced by alpha-methyl glucoside (alpha-MG). The ratio of P2 to P1 transcripts, determined by primer extension analysis, was high for the strong rrnO and rrnJ promoters, while only P2 transcripts were detected for the weak rrnD and rrnB promoters. Cloned P1 or P2 promoter fragments of rrnO or rrnJ were differentially regulated. In wild-type (relA(+)) and suppressor [relA(S)] strains under the conditions tested, only P2 responded to carbon source limitation by a decrease in RNA synthesis, correlating with an increase in (p)ppGpp levels and a decrease in the GTP concentration. The weak P1 promoter elements remain relaxed in the three genetic backgrounds [relA(+), relA, relA(S)] in the presence of alpha-MG. During amino acid starvation, P2 was stringently regulated in relA(+) and relA(S) cells, while only rrnJp(1) was also regulated, but to a lesser extent. Both the relA(+) and relA(S) strains showed (p)ppGpp accumulation after alpha-MG treatment but not after SHX treatment. These data reveal the complex nature of <span class="annotated ">B. subtilis</span> rrn promoter regulation in response to stress, and they suggest that the P2 promoters may play a more prominent role in the stringent response. 
<h3>21097613</h3>Primosomal proteins DnaD and DnaB are recruited to chromosomal regions bound by DnaA in <span class="match annotated ">Bacillus subtilis</span>. <br><br>The initiation of DNA replication requires the binding of the initiator protein, DnaA, to specific binding sites in the chromosomal origin of replication, oriC. DnaA also binds to many sites around the chromosome, outside oriC, and acts as a transcription factor at several of these. In low-G+C Gram-positive bacteria, the primosomal proteins DnaD and DnaB, in conjunction with loader ATPase DnaI, load the replicative helicase at oriC, and this depends on DnaA. DnaD and DnaB also are required to load the replicative helicase outside oriC during replication restart, independently of DnaA. Using chromatin immunoprecipitation, we found that DnaD and DnaB, but not the replicative helicase, are associated with many of the chromosomal regions bound by DnaA in <span class="match annotated ">Bacillus subtilis</span>. This association was dependent on DnaA, and the order of recruitment was the same as that at oriC, but it was independent of a functional oriC and suggests that DnaD and DnaB do not require open complex formation for the stable association with DNA. These secondary binding regions for DnaA could be serving as a reservoir for excess DnaA, DnaD, and DnaB to help properly regulate replication initiation and perhaps are analogous to the proposed function of the datA locus in <span class="annotated ">Escherichia coli</span>. Alternatively, DnaD and DnaB might modulate the activity of DnaA at the secondary binding regions. All three of these proteins are widely conserved and likely have similar functions in a range of organisms. 
<h3>21097616</h3>A LytM domain dictates the localization of proteins to the mother cell-forespore interface during bacterial endospore formation. <br><br>A large number of proteins are known to reside at specific subcellular locations in bacterial cells. However, the molecular mechanisms by which many of these proteins are anchored at these locations remains unclear. During endospore formation in <span class="match annotated ">Bacillus subtilis</span>, several integral membrane proteins are located specifically at the interface of the two adjacent cells of the developing sporangium, the mother cell and forespore. The mother cell membrane protein SpoIIIAH recognizes the cell-cell interface through an interaction with the forespore membrane protein SpoIIQ, and then the other proteins are positioned there by the SpoIIIAH-SpoIIQ complex. In this study, we investigated the molecular mechanisms underlying the formation of the SpoIIIAH-SpoIIQ complex. Using gel filtration chromatography and isothermal titration calorimetry, we measured the binding parameters that characterize the SpoIIIAH-SpoIIQ interaction in vitro. We also demonstrated that the interaction of SpoIIIAH and SpoIIQ is governed by their YscJ and degenerate LytM domains, respectively. Therefore, the LytM domain of SpoIIQ provides the positional cue that dictates the localization of mother cell membrane proteins to the mother cell-forespore interface. 
<h3>21097618</h3>Spatial regulation of histidine kinases governing biofilm formation in <span class="match annotated ">Bacillus subtilis</span>. <br><br><span class="match annotated ">Bacillus subtilis</span> is able to form architecturally complex biofilms on solid medium due to the production of an extracellular matrix. <span class="unannotated ">A</span> master regulator that controls the expression of the genes involved in matrix synthesis is Spo0A, which is activated by phosphorylation via a phosphorelay involving multiple histidine kinases. Here we report that four kinases, KinA, KinB, KinC, and KinD, help govern biofilm formation but that their contributions are partially masked by redundancy. We show that the kinases fall into two categories and that the members of each pair (one pair comprising KinA and KinB and the other comprising KinC and KinD) are partially redundant with each other. We also show that the kinases are spatially regulated: KinA and KinB are active principally in the older, inner regions of the colony, and KinC and KinD function chiefly in the younger, outer regions. These conclusions are based on the morphology of kinase mutants, real-time measurements of gene expression using luciferase as a reporter, and confocal microscopy using a fluorescent protein as a reporter. Our findings suggest that multiple signals from the older and younger regions of the colony are integrated by the kinases to determine the overall architecture of the biofilm community. 
<h3>21097623</h3>Contributions of multiple binding sites and effector-independent binding to CodY-mediated regulation in <span class="match annotated ">Bacillus subtilis</span>. <br><br>CodY is a branched-chain amino acid-responsive transcriptional regulator that controls, directly or indirectly, the expression of more than 100 genes and operons in <span class="match annotated ">Bacillus subtilis</span>. Using DNase I footprinting and gel-shift experiments, we identified two CodY-binding regions upstream of a <span class="annotated ">B. subtilis</span> gene (bcaP, previously known as yhdG) that encodes a transporter of branched-chain amino acids. Mutational analysis revealed that both CodY-binding regions contribute to repression in vivo and do so independently of each other. Thus, a single CodY-binding site is apparently sufficient for substantial CodY-dependent regulation. By analyzing affinities of wild-type and mutant CodY-binding sites for CodY and their regulation by wild-type CodY and forms of CodY with various levels of activation by branched-chain amino acids, we concluded that unliganded CodY cannot repress transcription in vivo and that the level of endogenously produced effectors is sufficient for CodY-mediated regulation of promoters with stronger sites. Because the sites with higher affinity apparently respond to lower concentrations of CodY effectors and saturate faster as the concentrations of effectors increase, having two sites of binding with different affinities for CodY permits a promoter to respond to a wider range of intracellular concentrations of effectors. 
<h3>21131488</h3><span class="annotated ">Bacillus anthracis</span> sin locus and regulation of secreted proteases.<br><br><span class="annotated ">Bacillus anthracis</span> shares many regulatory loci with the nonpathogenic Bacillus species <span class="match annotated ">Bacillus subtilis</span>. One such locus is sinIR, which in <span class="annotated ">B. subtilis</span> controls sporulation, biofilm formation, motility, and competency. As <span class="annotated ">B. anthracis</span> is not known to be motile, to be naturally competent, or to readily form biofilms, we hypothesized that the <span class="annotated ">B. anthracis</span> sinIR regulon is distinct from that of <span class="annotated ">B. subtilis</span>. <span class="unannotated ">A</span> genome-wide expression microarray analysis of <span class="annotated ">B. anthracis</span> parental and sinR mutant strains indicated limited convergence of the <span class="annotated ">B. anthracis</span> and <span class="annotated ">B. subtilis</span> SinR regulons. The <span class="annotated ">B. anthracis</span> regulon includes homologues of some <span class="annotated ">B. subtilis</span> SinR-regulated genes, including the signal peptidase gene sipW near the sinIR locus and the sporulation gene spoIIE. The <span class="annotated ">B. anthracis</span> SinR protein also negatively regulates transcription of genes adjacent to the sinIR locus that are unique to the <span class="annotated ">Bacillus cereus group species</span>. These include calY and inhA1, structural genes for the metalloproteases camelysin and immune inhibitor A1 (InhA1), which have been suggested to be associated with virulence in <span class="annotated ">B. cereus</span> and <span class="annotated ">B. anthracis</span>, respectively. Electrophoretic mobility shift assays revealed direct binding of <span class="annotated ">B. anthracis</span> SinR to promoter DNA from strongly regulated genes, such as calY and sipW, but not to the weakly regulated inhA1 gene. Assessment of camelysin and InhA1 levels in culture supernates from sinR-, inhA1-, and calY-null mutants showed that the concentration of InhA1 in the culture supernatant is inversely proportional to the concentration of camelysin. Our data are consistent with a model in which InhA1 protease levels are controlled at the transcriptional level by SinR and at the posttranslational level by camelysin. 
<h3>21036995</h3>Regulation of horizontal gene transfer in <span class="match annotated ">Bacillus subtilis</span> by activation of a conserved site-specific protease. <br><br>The mobile genetic element ICEBs1 is an integrative and conjugative element (a conjugative transposon) found in <span class="match annotated ">Bacillus subtilis</span>. The RecA-dependent SOS response and the RapI-PhrI cell sensory system activate ICEBs1 gene expression by stimulating cleavage of ImmR, the ICEBs1 immunity repressor, by the protease ImmA. We found that increasing the amount of wild-type ImmA in vivo caused partial derepression of ICEBs1 gene expression. However, during RapI-mediated derepression of ICEBs1 gene expression, ImmA levels did not detectably increase, indicating that RapI likely activates the protease ImmA by increasing its specific activity. We also isolated and characterized mutations in immA (immA(h)) that cause partial derepression of ICEBs1 gene expression in the absence of inducing signals. We obtained two types of immA(h) mutations: one type caused increased amounts of the mutant proteins in vivo but no detectable effect on specific activity in vitro; the other type had no detectable effect on the amount of the mutant protein in vivo but caused increased specific activity of the protein (as measured in vitro). Together, these findings indicate that derepression of ICEBs1 gene expression is likely caused by an increase in the specific activity of ImmA. Homologs of ImmA and ImmR are found in many mobile genetic elements, so the mechanisms that regulate ImmA-mediated cleavage of ImmR may be widely conserved. 
<h3>21037003</h3>A small protein required for the switch from {sigma}F to {sigma}G during sporulation in <span class="match annotated ">Bacillus subtilis</span>. <br><br>A cascade of alternative sigma factors governs the program of developmental gene expression during sporulation in <span class="match annotated ">Bacillus subtilis</span>. Little is known, however, about how the early-acting sigma factors are inactivated and replaced by the later-acting factors. Here we identify a small protein, Fin (formerly known as YabK), that is required for efficient switching from sigma(F)- to sigma(G)-directed gene expression in the forespore compartment of the developing sporangium. The fin gene, which is conserved among Bacillus species and species of related genera, is transcribed in the forespore under the control of both sigma(F) and sigma(G). Cells mutant for fin are unable to fully deactivate sigma(F) and, conversely, are unable to fully activate sigma(G). Consistent with their deficiency in sigma(G)-directed gene expression, fin cells are arrested in large numbers following the engulfment stage of sporulation, ultimately forming 50-fold fewer heat-resistant spores than the wild type. Based in part on the similarity of Fin to the anti-sigma(G) factor CsfB (also called Gin), we speculate that Fin is an anti-sigma(F) factor which, by disabling sigma(F), promotes the switch to late developmental gene expression in the forespore. 
</body></html>