
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>E. coli</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>21097626</h3>Altered regulation of the OmpF porin by Fis in <span class="annotated ">Escherichia coli</span> during an evolution experiment and between B and <span class="unannotated ">K-12</span> strains. <br><br>The phenotypic plasticity of global regulatory networks provides bacteria with rapid acclimation to a wide range of environmental conditions, while genetic changes in those networks provide additional flexibility as bacteria evolve across long time scales. We previously identified mutations in the global regulator-encoding gene fis that enhanced organismal fitness during a long-term evolution experiment with <span class="annotated ">Escherichia coli</span>. To gain insight into the effects of these mutations, we produced two-dimensional protein gels with strains carrying different fis alleles, including a beneficial evolved allele and one with an in-frame deletion. We found that Fis controls the expression of the major porin-encoding gene ompF in the <span class="match annotated ">E. coli</span> B-derived ancestral strain used in the evolution experiment, a relationship that has not been described before. We further showed that this regulatory connection evolved over two different time scales, perhaps explaining why it was not observed before. On the longer time scale, we showed that this regulation of ompF by Fis is absent from the more widely studied <span class="annotated ">K-12 strain</span> and thus is specific to the <span class="annotated ">B strain</span>. On a shorter time scale, this regulatory linkage was lost during 20,000 generations of experimental evolution of the <span class="annotated ">B strain</span>. Finally, we mapped the Fis binding sites in the ompF regulatory region, and we present a hypothetical model of ompF expression that includes its other known regulators. 
<h3>21097627</h3>Fur and the novel regulator YqjI control transcription of the ferric reductase gene yqjH in <span class="annotated ">Escherichia coli</span>. <br><br>Iron acquisition in aerobic habitats is complicated by the low solubility of ferric hydroxides. Siderophores that bind ferric iron with high affinity are used to mobilize iron. The reduction of ferric iron to the ferrous form can be coupled to the release of iron from siderophores. Iron is also stored intracellularly as a ferric mineral in proteins, such as ferritin, and must be reduced during release. In <span class="annotated ">Escherichia coli</span>, the yqjH gene encodes a putative ferric siderophore reductase that is also part of the Fur regulon. Here we show that YqjH has ferric reductase activity and is required for iron homeostasis in <span class="match annotated ">E. coli</span>. Divergently transcribed from yqjH is the yqjI gene, which encodes a novel member of the winged-helix family of transcriptional regulators and also contains an N-terminal extension similar to the Ni(2+)-binding C-terminal tail of SlyD. Deletion of yqjI leads to constitutive high-level activity of the yqjH and yqjI promoters. Purified YqjI binds inverted repeat target sequences within the yqjH and yqjI promoters. We also observed that YqjI-dependent transcriptional repression is reduced when cells are exposed to elevated nickel levels, resulting in increased expression of yqjH and yqjI. YqjI binding to nickel or iron reduces YqjI DNA-binding activity in vitro. Furthermore, we found that elevated nickel stress levels disrupt iron homeostasis in <span class="match annotated ">E. coli</span> and that deletion of yqjH increases nickel toxicity. Our results suggest that the YqjI protein controls expression of yqjH to help maintain iron homeostasis under conditions (such as elevated cellular nickel levels) that disrupt iron metabolism. 
<h3>21131485</h3>Regulation of nleA in Shiga toxin-producing <span class="annotated ">Escherichia coli O84:H4 strain 4795/97</span>. <br><br>Many Shiga toxin-producing <span class="annotated ">Escherichia coli</span> (<span class="annotated ">STEC</span>) strains express a type III secretion system (TTSS) encoded by the locus of enterocyte effacement (LEE). Using the TTSS, <span class="annotated ">STEC</span> is able to inject effector proteins directly into eukaryotic host cells, where they cause characteristic attaching and effacing (A/E) lesions. In addition to the LEE-encoded effectors, a number of non-LEE-encoded effectors, located on phage-associated elements, have been described. One of them, the non-LEE-encoded effector <span class="unannotated ">A</span> (NleA), is widely distributed among pathogenic <span class="match annotated ">E. coli</span>. In this study, we investigated the influence of environmental conditions on the expression of the phage-encoded effector nleA gene (designated nleA(4795)) present in <span class="annotated ">STEC O84:H4 strain 4795/97</span>. We demonstrated that a particular NaCl concentration and starvation stress increase the activity of the nleA(4795) promoter. Moreover, several regulators that control nleA(4795) expression were identified. The involvement of the LEE regulators Ler, GrlA, and GrlR show that nleA(4795) is integrated in the LEE regulation circuit. Furthermore, the binding of Ler to sequences upstream of nleA(4795) underlined these findings. 
<h3>21148732</h3>Functional and phylogenetic analysis of ureD in Shiga toxin-producing <span class="annotated ">Escherichia coli</span>. <br><br>Enterohemorrhagic <span class="annotated ">Escherichia coli</span> (<span class="annotated ">EHEC</span>) is a food-borne pathogen that can cause severe health complications and utilizes a much lower infectious dose than other <span class="match annotated ">E. coli</span> pathotypes. Despite having an intact ure locus, ureDABCEFG, the majority of <span class="annotated ">EHEC</span> strains are phenotypically urease negative under tested conditions. Urease activity potentially assists with survival fitness by enhancing acid tolerance during passage through the stomach or by aiding with colonization in either <span class="annotated ">human</span> or animal reservoirs. Previously, in the <span class="annotated ">EHEC O157:H7</span> Sakai strain, a point mutation in ureD, encoding a urease chaperone protein, was identified, resulting in a substitution of an amber stop codon for glutamine. This single nucleotide polymorphism (SNP) is observed in the majority of <span class="annotated ">EHEC O157:H7</span> isolates and correlates with a negative urease phenotype in vitro. We demonstrate that the lack of urease activity in vitro is not solely due to the amber codon in ureD. Our analysis has identified two additional SNPs in ureD affecting amino acid positions 38 and 205, in both cases determining whether the encoded amino acid is leucine or proline. Phylogenetic analysis based on Ure protein sequences from a variety of urease-encoding bacteria demonstrates that the proline at position 38 is highly conserved among Gram-negative bacteria. Experiments reveal that the L38P substitution enhances urease enzyme activity; however, the L205P substitution does not. Multilocus sequence typing analysis for a variety of Shiga toxin-producing <span class="match annotated ">E. coli</span> isolates combined with the ureD sequence reveals that except for a subset of the <span class="annotated ">O157:H7</span> strains, neither the in vitro urease-positive phenotype nor the ureD sequence is phylogenetically restricted. 
<h3>21169483</h3>Unexpected coregulator range for the global regulator Lrp of <span class="annotated ">Escherichia coli</span> and <span class="annotated ">Proteus mirabilis</span>. <br><br>The Lrp/AsnC family of transcription factors links gene regulation to metabolism in bacteria and archaea. Members of this family, collectively, respond to a wide range of amino acids as coregulators. In <span class="annotated ">Escherichia coli</span>, Lrp regulates over 200 genes directly and is well known to respond to leucine and, to a somewhat lesser extent, alanine. We focused on Lrp from <span class="annotated ">Proteus mirabilis</span> and <span class="match annotated ">E. coli</span>, orthologs with 98% identity overall and identical helix-turn-helix motifs, for which a previous study nevertheless found functional differences. Sequence differences between these orthologs, within and adjacent to the amino acid-responsive RAM domain, led us to test for differential sensitivity to coregulatory amino acids. In the course of this investigation, we found, via in vivo reporter fusion assays and in vitro electrophoretic mobility shift experiments, that <span class="match annotated ">E. coli</span> Lrp itself responded to a broader range of amino acids than was previously appreciated. In particular, for both the <span class="match annotated ">E. coli</span> and <span class="annotated ">P. mirabilis</span> orthologs, Lrp responsiveness to methionine was similar in magnitude to that to leucine. Both Lrp orthologs are also fairly sensitive to Ile, His, and Thr. These observations suggest that Lrp ties gene expression in the Enterobacteriaceae rather extensively to physiological status, as reflected in amino acid pools. These findings also have substantial implications for attempts to model regulatory architecture from transcriptome measurements or to infer such architecture from genome sequences, and they suggest that even well-studied regulators deserve ongoing exploration. 
<h3>21037010</h3>Inactivation of the RluD pseudouridine synthase has minimal effects on growth and ribosome function in wild-type <span class="annotated ">Escherichia coli</span> and <span class="annotated ">Salmonella enterica</span>. <br><br>The <span class="annotated ">Escherichia coli</span> rluD gene encodes a pseudouridine synthase responsible for the pseudouridine (Psi) modifications at positions 1911, 1915, and 1917 in helix 69 of 23S rRNA. It has been reported that deletion of rluD in <span class="annotated ">K-12 strains of E. coli</span> is associated with extremely slow growth, increased readthrough of stop codons, and defects in 50S ribosomal subunit assembly and 30S-50S subunit association. Suppressor mutations in the prfB and prfC genes encoding release factor 2 (RF2) and RF3 that restore the wild type-growth rate and also correct the ribosomal defects have now been isolated. These suppressors link helix 69 Psi residues with the termination phase of protein synthesis. However, further genetic analysis reported here also reveals that the slow growth and other defects associated with inactivation of rluD in <span class="annotated ">E. coli K-12</span> strains are due to a defective RF2 protein, with a threonine at position 246, which is present in all <span class="annotated ">K-12</span> strains. This is in contrast to the more typical alanine found at this position in most bacterial RF2s, including those of other <span class="match annotated ">E. coli</span> strains. Inactivation of rluD in <span class="match annotated ">E. coli</span> strains containing the prfB allele from <span class="annotated ">E. coli B</span> or in <span class="annotated ">Salmonella enterica</span>, both carrying an RF2 with Ala246, has negligible effects on growth, termination, or ribosome function. The results indicate that, in contrast to those in wild bacteria, termination functions in <span class="annotated ">E. coli K-12</span> strains carrying a partially defective RF2 protein are especially susceptible to perturbation of ribosome-RF interactions, such as that caused by loss of h69 Psi modifications. 
<h3>21037009</h3>Activation of the SoxR regulon in <span class="annotated ">Streptomyces coelicolor</span> by the extracellular form of the pigmented antibiotic actinorhodin. <br><br>The redox-sensitive transcription factor SoxR in enteric bacteria senses and regulates the cellular response to superoxide and nitric oxide. In other bacterial groups, however, it may respond to redox-active small molecules, as demonstrated for pyocyanin sensing in pseudomonads. The antibiotic-producing soil bacterium <span class="annotated ">Streptomyces coelicolor</span> contains a gene for an SoxR homologue (SCO1697) whose DNA recognition helix is identical to that of <span class="annotated ">Escherichia coli</span> SoxR. Using the <span class="match annotated ">E. coli</span> SoxR binding sequence, we predicted five candidate genes of the SoxR regulon and demonstrated that SoxR binds to their promoter regions and activates their expression concurrently with the production of the blue antibiotic actinorhodin (a benzoisochromanequinone). These genes encode a probable NADPH-dependent flavin reductase (SCO2478), an NADPH-dependent quinone reductase (SCO4266), an ABC transporter (SCO7008), a monooxygenase (SCO1909), and a hypothetical protein (SCO1178). Addition of actinorhodin to exponentially growing cells activated the expression of SoxR target genes in an SoxR-dependent manner. The secreted gamma-actinorhodin was over 10-fold more effective in activation than the intracellular form of actinorhodin, suggesting that SoxR is specified to respond more to exogenous signals than to intracellular metabolites. The DeltasoxR mutant was not compromised in resistance against oxidants but was slow in forming aerial mycelium on R2YE medium with reduced sporulation, and its production of actinorhodin and undecylprodigiosin was lowered by about 50% and 30%, respectively, compared to that of the wild type. These results support the proposal that SoxR senses redox-active molecules, such as actinorhodin in <span class="annotated ">S. coelicolor</span>, and induces a protective function against them. It also functions to ensure that cells undergo optimal differentiation and secondary metabolite production. 
</body></html>