
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Escherichia coli</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>20173013</h3><span class="annotated ">Streptomyces lacticiproducens</span> sp. nov., a lactic acid-producing streptomycete isolated from the rhizosphere of <span class="annotated ">tomato</span> plants. <br><br>A novel actinomycete, designated strain GIMN4.001(T), was isolated from the rhizosphere of <span class="annotated ">tomato</span> plants grown in Guangzhou, China. The strain produced greyish white aerial mycelia, lactic acid and a large quantity of double diamond-shaped crystals on <span class="annotated ">potato</span> dextrose agar and <span class="LINNAEUS_only ">yeast</span> extract-malt extract agar. The colour of the substrate mycelium was not sensitive to pH. Microscopic observations revealed that strain GIMN4.001(T) produced verticillate chains of cylindrical spores. Chemotaxonomic data confirmed that strain GIMN4.001(T) belonged to the genus Streptomyces. Melanin pigments were not produced. No antibacterial activity was observed against <span class="match annotated ">Escherichia coli</span>, <span class="annotated ">Pseudomonas aeruginosa</span>, <span class="annotated ">Bacillus subtilis</span> or <span class="annotated ">Candida albicans</span>, but inhibitory activity was observed against <span class="annotated ">Penicillium citrinum</span>. 16S rRNA gene sequence analysis revealed that strain GIMN4.001(T) was related most closely to <span class="annotated ">Streptomyces morookaense</span> <span class="annotated ">ATCC 19166</span>(T) (98.9 % similarity) and <span class="annotated ">Streptomyces lavenduligriseus</span> <span class="annotated ">ATCC 13306</span>(T) (98.7 %). Levels of DNA-DNA relatedness between strain GIMN4.001(T) and the type strains of these species were low (14-20 %). Furthermore, strain GIMN4.001(T) could be differentiated from <span class="annotated ">S. morookaense</span>, <span class="annotated ">S. lavenduligriseus</span> and other closely related species of the genus Streptomyces based on morphological, physiological and biochemical characteristics. On the basis of its physiological and molecular properties, strain GIMN4.001(T) is considered to represent a novel species of the genus Streptomyces, for which the name <span class="annotated ">Streptomyces lacticiproducens</span> sp. nov. is proposed. The type strain is GIMN4.001(T) (=CCTCC M208214(T)=NRRL B-24800(T)). 
<h3>20682355</h3>PRP8 intein in Ajellomycetaceae family pathogens: sequence analysis, splicing evaluation and homing endonuclease activity. <br><br>Inteins are intervening sequences that are transcribed and translated with flanking host protein sequences and then self-excised by protein splicing. Bi-functional inteins also contain a homing endonuclease responsible for their genetic mobility. The PRP8 intein, the most widespread among fungi, occurs in important pathogens such as <span class="annotated ">Histoplasma capsulatum</span> and <span class="annotated ">Paracoccidioides brasiliensis</span>, from the Ajellomycetaceae family. Herein, we describe the bi-functional PRP8 intein in two other Ajellomycetacean pathogens, <span class="annotated ">Blastomyces dermatitidis</span> and <span class="annotated ">Emmonsia parva</span>. Sequence analysis and experimental evidence suggest that the homing endonuclease from PbrPRP8 is inactive. The splicing activity of the PRP8 intein from the <span class="annotated ">B. dermatitidis</span>, <span class="annotated ">E. parva</span> and <span class="annotated ">P. brasiliensis</span> species complex was demonstrated in a non-native protein context in <span class="match annotated ">Escherichia coli</span>. Since the PRP8 intein is located in a functionally essential nuclear protein, it can be considered a promising therapeutic target for anti-fungal drugs, because inhibition of intein splicing should inhibit proliferation of intein-containing pathogens. 
<h3>20713166</h3>Characterization of the <span class="annotated ">Aspergillus nidulans</span> biotin biosynthetic gene cluster and use of the bioDA gene as a new transformation marker. <br><br>The genes involved in the biosynthesis of biotin were identified in the hyphal fungus <span class="annotated ">Aspergillus nidulans</span> through homology searches and complementation of <span class="match annotated ">Escherichia coli</span> biotin-auxotrophic mutants. Whereas the 7,8-diaminopelargonic acid synthase and dethiobiotin synthetase are encoded by distinct genes in bacteria and the <span class="LINNAEUS_only ">yeast</span> <span class="annotated ">Saccharomyces cerevisiae</span>, both activities are performed in <span class="annotated ">A. nidulans</span> by a single enzyme, encoded by the bifunctional gene bioDA. Such a bifunctional bioDA gene is a genetic feature common to numerous members of the ascomycete filamentous fungi and basidiomycetes, as well as in plants and oomycota. However, unlike in other eukaryota, the three bio genes contributing to the four enzymatic steps from pimeloyl-CoA to biotin are organized in a gene cluster in pezizomycotina. The <span class="annotated ">A. nidulans</span> auxotrophic mutants biA1, biA2 and biA3 were all found to have mutations in the 7,8-diaminopelargonic acid synthase domain of the bioDA gene. Although biotin auxotrophy is an inconvenient marker in classical genetic manipulations due to cross-feeding of biotin, transformation of the biA1 mutant with the bioDA gene from either <span class="annotated ">A. nidulans</span> or <span class="annotated ">Aspergillus fumigatus</span> led to the recovery of well-defined biotin-prototrophic colonies. The usefulness of bioDA gene as a novel and robust transformation marker was demonstrated in co-transformation experiments with a green fluorescent protein reporter, and in the efficient deletion of the laccase (yA) gene via homologous recombination in a mutant lacking non-homologous end-joining activity. 
<h3>20863856</h3>Expression of the C-terminal ORF2 protein of <span class="annotated ">duck astrovirus</span> for application in a serological test. <br><br><span class="annotated ">Duck astrovirus</span> (<span class="annotated ">DAstV</span>) is an important pathogen causing duck <span class="unannotated ">viral</span> <span class="unannotated ">hepatitis</span> (DVH), a highly contagious and fatal disease in young ducklings. To provide an antigen for a diagnostic serum test, the C-terminus of <span class="annotated ">DAstV</span> ORF2 protein was expressed in <span class="match annotated ">Escherichia coli</span>. Four positive and 30 negative sera were used to validate the purified ORF2 protein by developing an indirect enzyme-linked immunosorbent assay (ELISA). No cross-reactions were found against other duck pathogens, including <span class="annotated ">duck hepatitis A virus</span>, <span class="annotated ">duck plague herpesvirus</span>, <span class="annotated ">duck reovirus</span>, <span class="annotated ">Newcastle disease virus</span>, and <span class="annotated ">Riemerella anatipestifer 12/19</span><span class="annotated ">Riemerella anatipestifer</span> 12/19 (63.2%) and 26/51 (51%) sera samples from two flocks of ducks that survived <span class="annotated ">DAstV</span> infections in commercial duck farms were positive for <span class="annotated ">DAstV</span> by this method, respectively. Interestingly, <span class="annotated ">DAstV</span>-specific antibodies were also detected in 12 (28.6%) of 42 sera samples from a different flock without DVH, indicating a wide distribution of subclinical infections caused by <span class="annotated ">DAstV</span>. 
<h3>20870768</h3>Mutational analysis of the transmembrane helix 2-HAMP domain connection in the <span class="match annotated ">Escherichia coli</span> aspartate chemoreceptor tar. <br><br>Transmembrane helix 2 (TM2) of the Tar chemoreceptor undergoes an inward piston-like displacement of 1 to 3 <span class="unannotated ">A</span> upon binding aspartate. This signal is transmitted to the kinase-control module via the HAMP domain. Within Tar, the HAMP domain forms a parallel four-helix bundle consisting of a dimer of two amphipathic helices connected by a flexible linker. In the nuclear magnetic resonance structure of an archaeal HAMP domain, residues corresponding to the MLLT sequence between Arg-214 at the end of TM2 and Pro-219 of Tar are an N-terminal helical extension of AS1. We modified this region to test whether it behaves as a continuous helical connection between TM2 and HAMP. First, one to four Gly residues were inserted between Thr-218 and Pro-219. Second, the MLLT sequence was replaced with one to nine Gly residues. Third, the sequence was shortened or extended with residues compatible with helix formation. Cells expressing receptors in which the MLLT sequence was shortened to MLL or in which the MLLT sequence was replaced by four Gly residues performed good aspartate chemotaxis. Other mutant receptors supported diminished aspartate taxis. Most mutant receptors had biased signal outputs and/or abnormal patterns of adaptive methylation. We interpret these results to indicate that a strong, permanent helical connection between TM2 and the HAMP domain is not necessary for normal transmembrane signaling. 
<h3>20933014</h3>Monoclonal antibody-based triple antibody sandwich-enzyme-linked immunosorbent assay and immunocapture reverse transcription-polymerase chain reaction for <span class="annotated ">Odontoglossum ringspot virus</span> detection. <br><br><span class="annotated ">Odontoglossum ringspot virus</span> (<span class="annotated ">ORSV</span>) infects numerous commercially important orchids and causes significant losses worldwide. The coat protein (CP) gene of <span class="annotated ">ORSV</span> was cloned and expressed in <span class="match annotated ">Escherichia coli</span> by using the pET-32a expression vector, and the expression of recombinant protein was confirmed by Western blotting using anti-ORSV antibodies. The recombinant protein was purified using Ni-NTA agarose, and the purified protein was used as an immunogen to produce monoclonal antibodies (MAbs) and polyclonal antibodies (PAbs). Five <span class="LINNAEUS_only ">murine</span> MAbs against <span class="annotated ">ORSV</span> CP were obtained. Among them, two MAbs (6B4 and 1D1) also reacted with TMV CP. The triple antibody sandwich enzyme-linked immunosorbent assay (TAS-ELISA) and immunocapture reverse transcription-polymerase chain reaction (IC-RT-PCR) methods using the MAb (8A5) were then developed for sensitive, specific, and rapid detection of <span class="annotated ">ORSV</span>. TAS-ELISA and IC-RT-PCR could detect <span class="annotated ">ORSV</span> in the infected leaf saps with dilutions of 1:10,240 and 1:81,920 (w/v, g mL(-1)), respectively. TAS-ELISA and IC-RT-PCR detections indicated that <span class="annotated ">ORSV</span> was prevalent in orchids in the Zhejiang Province of China. 
<h3>20971898</h3>Nucleotide parasitism by <span class="annotated ">Simkania negevensis</span> (Chlamydiae).<br><br>Intracellular bacteria live in an environment rich in most essential metabolites but need special mechanisms to access these substrates. Nucleotide transport proteins (NTTs) catalyze the import of ATP and other nucleotides from the eukaryotic host into the bacterial cell and render de novo synthesis of these compounds dispensable. The draft genome sequence of <span class="annotated ">Simkania negevensis strain Z</span>, a chlamydial organism considered a newly emerging pathogen, revealed four genes encoding putative nucleotide transport proteins (SnNTT1 to SnNTT4), all of which are transcribed during growth of <span class="annotated ">S. negevensis</span> in Acanthamoeba host cells, as confirmed by reverse transcription-PCR. Using heterologous expression in <span class="match annotated ">Escherichia coli</span>, we could show that SnNTT1 functions as an ATP/ADP antiporter, SnNTT2 as a guanine nucleotide/ATP/H(+) symporter driven by the membrane potential, and SnNTT3 as a nucleotide triphosphate antiporter. In addition, SnNTT3 is able to transport dCTP, which has not been shown for a prokaryotic transport protein before. No substrate could be identified for SnNTT4. Taking these data together, <span class="annotated ">S. negevensis</span> employs a set of nucleotide transport proteins to efficiently tap its host's energy and nucleotide pools. Although similar to other chlamydiae, these transporters show distinct and unique adaptations with respect to substrate specificities and mode of transport. 
<h3>20971900</h3>Regulation and function of <span class="match annotated ">Escherichia coli</span> sugar efflux transporter <span class="unannotated ">A</span> (SetA) during glucose-phosphate stress. <br><br>Accumulation of certain nonmetabolizable sugar-phosphates (including alpha-methyl glucoside-6-phosphate) in <span class="match annotated ">Escherichia coli</span> is growth inhibitory and elicits the glucose-phosphate stress response. The transcription factor SgrR activates transcription of the small RNA SgrS under stress conditions. SgrS represses translation of mRNAs encoding sugar transporters. The sgrR and sgrS genes are located directly upstream of setA, and this gene organization is conserved in numerous enteric species, prompting the hypothesis that SetA contributes to the glucose-phosphate stress response. SetA is a proton motive force-driven efflux pump capable of transporting various sugars and sugar analogs in vitro. This study demonstrates that setA expression is induced in response to glucose-phosphate stress, and this requires SgrR. Under stress conditions, setA is cotranscribed with sgrS from the sgrS promoter. <span class="unannotated ">A</span> setA mutant exhibits a growth defect under stress conditions that can be complemented by setA in trans, suggesting that SetA contributes to the optimal cellular recovery from stress. Despite previous in vitro evidence that SetA can promote efflux of the stress-causing glucose analog alpha-methyl glucoside, in vivo data in this study indicate that SetA is not the major efflux pump responsible for removal of alpha-methyl glucoside under stress conditions. 
<h3>21037007</h3>X-ray structure and site-directed mutagenesis analysis of the <span class="match annotated ">Escherichia coli</span> colicin M immunity protein. <br><br>Colicin M (ColM), which is produced by some <span class="match annotated ">Escherichia coli</span> strains to kill competitor strains from the same or related species, was recently shown to inhibit cell wall peptidoglycan biosynthesis through enzymatic degradation of its lipid II precursor. ColM-producing strains are protected from the toxin that they produce by coexpression of a specific immunity protein, named Cmi, whose mode of action still remains to be identified. We report here the resolution of the crystal structure of Cmi, which is composed of four Beta strands and four alpha helices. This rather compact structure revealed a disulfide bond between residues Cys31 and Cys107. Interestingly, these two cysteines and several other residues appeared to be conserved in the sequences of several proteins of unknown function belonging to the YebF family which exhibit 25 to 35% overall sequence similarity with Cmi. Site-directed mutagenesis was performed to assess the role of these residues in the ColM immunity-conferring activity of Cmi, which showed that the disulfide bond and residues from the C-terminal extremity of the protein were functionally essential. The involvement of DsbA oxidase in the formation of the Cmi disulfide bond is also demonstrated. 
<h3>21057010</h3>Biochemical characterization of UDP-Gal:GlcNAc-pyrophosphate-lipid Beta-1,4-Galactosyltransferase WfeD, a new enzyme from <span class="annotated ">Shigella boydii</span> type 14 that catalyzes the second step in O-antigen repeating-unit synthesis. <br><br>The <span class="unannotated ">O</span> antigen is the outer part of the lipopolysaccharide (LPS) in the outer membrane of Gram-negative bacteria and contains many repeats of an oligosaccharide unit. It contributes to antigenic variability and is essential to the full function and virulence of bacteria. Shigella is a Gram-negative <span class="annotated ">human</span> pathogen that causes diarrhea in <span class="annotated ">humans</span>. The <span class="unannotated ">O</span> antigen of <span class="annotated ">Shigella boydii</span> type 14 consists of repeating oligosaccharide units with the structure [->6-d-Galpalpha1->4-d-GlcpABeta1->6-d-GalpBeta1->4-d-GalpBeta1->4-d-GlcpNAcBeta1->]n. The wfeD gene in the O-antigen gene cluster of <span class="annotated ">Shigella boydii</span> type 14 was proposed to encode a galactosyltransferase (GalT) involved in O-antigen synthesis. We confirmed here that the wfeD gene product is a Beta4-GalT that synthesizes the GalBeta1-4GlcNAcalpha-R linkage. WfeD was expressed in <span class="match annotated ">Escherichia coli</span>, and the activity was characterized by using UDP-[^3H]Gal as the donor substrate as well as the synthetic acceptor substrate GlcNAcalpha-pyrophosphate-(CH2)11-O-phenyl. The enzyme product was analyzed by liquid chromatography-mass spectrometry (LC-MS), high-performance liquid chromatography (HPLC), nuclear magnetic resonance (NMR), and galactosidase digestion. The enzyme was shown to be specific for the UDP-Gal donor substrate and required pyrophosphate in the acceptor substrate. Divalent metal ions such as Mn^2(+), Ni^2(+), and, surprisingly, also Pb^2(+) enhanced the enzyme activity. Mutational analysis showed that the Glu101 residue within a DxD motif is essential for activity, possibly by forming the catalytic nucleophile. The Lys211 residue was also shown to be required for activity and may be involved in the binding of the negatively charged acceptor substrate. Our study revealed that the Beta4-GalT WfeD is a novel enzyme that has virtually no sequence similarity to mammalian Beta4-GalT, although it catalyzes a similar reaction. 
<h3>21075924</h3>argC Orthologs from Rhizobiales show diverse profiles of transcriptional efficiency and functionality in <span class="annotated ">Sinorhizobium meliloti</span>. <br><br>Several factors can influence ortholog replacement between closely related species. We evaluated the transcriptional expression and metabolic performance of ortholog substitution complementing a <span class="annotated ">Sinorhizobium meliloti</span> argC mutant with argC from Rhizobiales (<span class="annotated ">Agrobacterium tumefaciens</span>, <span class="annotated ">Rhizobium etli</span>, and <span class="annotated ">Mesorhizobium loti</span>). The argC gene is necessary for the synthesis of arginine, an amino acid that is central to protein and cellular metabolism. Strains were obtained carrying plasmids with argC orthologs expressed under the speB and argC (<span class="annotated ">S. meliloti</span>) and lac (<span class="match annotated ">Escherichia coli</span>) promoters. Complementation analysis was assessed by growth, transcriptional activity, enzymatic activity, mRNA levels, specific detection of ArgC proteomic protein, and translational efficiency. The argC orthologs performed differently in each complementation, reflecting the diverse factors influencing gene expression and the ability of the ortholog product to function in a foreign metabolic background. Optimal complementation was directly related to sequence similarity with <span class="annotated ">S. meliloti</span>, and was inversely related to species signature, with <span class="annotated ">M. loti</span> argC showing the poorest performance, followed by <span class="annotated ">R. etli</span> and <span class="annotated ">A. tumefaciens</span>. Different copy numbers of genes and amounts of mRNA and protein were produced, even with genes transcribed from the same promoter, indicating that coding sequences play a role in the transcription and translation processes. These results provide relevant information for further genomic analyses and suggest that orthologous gene substitutions between closely related species are not completely functionally equivalent. 
<h3>21097613</h3>Primosomal proteins DnaD and DnaB are recruited to chromosomal regions bound by DnaA in <span class="annotated ">Bacillus subtilis</span>. <br><br>The initiation of DNA replication requires the binding of the initiator protein, DnaA, to specific binding sites in the chromosomal origin of replication, oriC. DnaA also binds to many sites around the chromosome, outside oriC, and acts as a transcription factor at several of these. In low-G+C Gram-positive bacteria, the primosomal proteins DnaD and DnaB, in conjunction with loader ATPase DnaI, load the replicative helicase at oriC, and this depends on DnaA. DnaD and DnaB also are required to load the replicative helicase outside oriC during replication restart, independently of DnaA. Using chromatin immunoprecipitation, we found that DnaD and DnaB, but not the replicative helicase, are associated with many of the chromosomal regions bound by DnaA in <span class="annotated ">Bacillus subtilis</span>. This association was dependent on DnaA, and the order of recruitment was the same as that at oriC, but it was independent of a functional oriC and suggests that DnaD and DnaB do not require open complex formation for the stable association with DNA. These secondary binding regions for DnaA could be serving as a reservoir for excess DnaA, DnaD, and DnaB to help properly regulate replication initiation and perhaps are analogous to the proposed function of the datA locus in <span class="match annotated ">Escherichia coli</span>. Alternatively, DnaD and DnaB might modulate the activity of DnaA at the secondary binding regions. All three of these proteins are widely conserved and likely have similar functions in a range of organisms. 
<h3>21097625</h3>Differences in MinC/MinD sensitivity between polar and internal Z rings in <span class="match annotated ">Escherichia coli</span>. <br><br>In <span class="match annotated ">Escherichia coli</span> the Z ring has the potential to assemble anywhere along the cell length but is restricted to midcell by the action of negative regulatory systems, including Min. In the current model for the Min system, the MinC/MinD division inhibitory complex is evenly distributed on the membrane and can disrupt Z rings anywhere in the cell; however, MinE spatially regulates MinC/MinD by restricting it to the cell poles, thus allowing Z ring formation at midcell. This model assumes that Z rings formed at different cellular locations have equal sensitivity to MinC/MinD in the absence of MinE. However, here we report evidence that differences in MinC/MinD sensitivity between polar and nonpolar Z rings exists even when there is no MinE. MinC/MinD at proper levels is able to block minicell production in Deltamin strains without increasing the cell length, indicating that polar Z rings are preferentially blocked. In the <span class="annotated ">FtsZ-I374V strain</span> (which is resistant to MinC(C)/MinD), wild-type morphology can be easily achieved with MinC/MinD in the absence of MinE. We also show that MinC/MinD at proper levels can rescue the lethal phenotype of a min slmA double deletion mutant, which we think is due to the elimination of polar Z rings (or FtsZ structures), which frees up FtsZ molecules for assembly of Z rings at internal sites to rescue division and growth. Taken together, these data indicate that polar Z rings are more susceptible to MinC/MinD than internal Z rings, even when MinE is absent. 
<h3>21097626</h3>Altered regulation of the OmpF porin by Fis in <span class="match annotated ">Escherichia coli</span> during an evolution experiment and between B and <span class="unannotated ">K-12</span> strains. <br><br>The phenotypic plasticity of global regulatory networks provides bacteria with rapid acclimation to a wide range of environmental conditions, while genetic changes in those networks provide additional flexibility as bacteria evolve across long time scales. We previously identified mutations in the global regulator-encoding gene fis that enhanced organismal fitness during a long-term evolution experiment with <span class="match annotated ">Escherichia coli</span>. To gain insight into the effects of these mutations, we produced two-dimensional protein gels with strains carrying different fis alleles, including a beneficial evolved allele and one with an in-frame deletion. We found that Fis controls the expression of the major porin-encoding gene ompF in the <span class="annotated ">E. coli</span> B-derived ancestral strain used in the evolution experiment, a relationship that has not been described before. We further showed that this regulatory connection evolved over two different time scales, perhaps explaining why it was not observed before. On the longer time scale, we showed that this regulation of ompF by Fis is absent from the more widely studied <span class="annotated ">K-12 strain</span> and thus is specific to the <span class="annotated ">B strain</span>. On a shorter time scale, this regulatory linkage was lost during 20,000 generations of experimental evolution of the <span class="annotated ">B strain</span>. Finally, we mapped the Fis binding sites in the ompF regulatory region, and we present a hypothetical model of ompF expression that includes its other known regulators. 
<h3>21097627</h3>Fur and the novel regulator YqjI control transcription of the ferric reductase gene yqjH in <span class="match annotated ">Escherichia coli</span>. <br><br>Iron acquisition in aerobic habitats is complicated by the low solubility of ferric hydroxides. Siderophores that bind ferric iron with high affinity are used to mobilize iron. The reduction of ferric iron to the ferrous form can be coupled to the release of iron from siderophores. Iron is also stored intracellularly as a ferric mineral in proteins, such as ferritin, and must be reduced during release. In <span class="match annotated ">Escherichia coli</span>, the yqjH gene encodes a putative ferric siderophore reductase that is also part of the Fur regulon. Here we show that YqjH has ferric reductase activity and is required for iron homeostasis in <span class="annotated ">E. coli</span>. Divergently transcribed from yqjH is the yqjI gene, which encodes a novel member of the winged-helix family of transcriptional regulators and also contains an N-terminal extension similar to the Ni(2+)-binding C-terminal tail of SlyD. Deletion of yqjI leads to constitutive high-level activity of the yqjH and yqjI promoters. Purified YqjI binds inverted repeat target sequences within the yqjH and yqjI promoters. We also observed that YqjI-dependent transcriptional repression is reduced when cells are exposed to elevated nickel levels, resulting in increased expression of yqjH and yqjI. YqjI binding to nickel or iron reduces YqjI DNA-binding activity in vitro. Furthermore, we found that elevated nickel stress levels disrupt iron homeostasis in <span class="annotated ">E. coli</span> and that deletion of yqjH increases nickel toxicity. Our results suggest that the YqjI protein controls expression of yqjH to help maintain iron homeostasis under conditions (such as elevated cellular nickel levels) that disrupt iron metabolism. 
<h3>21097632</h3>Acyl-homoserine lactone binding to and stability of the orphan <span class="annotated ">Pseudomonas aeruginosa</span> quorum-sensing signal receptor QscR. <br><br>The <span class="annotated ">Pseudomonas aeruginosa</span> transcription factor QscR responds to a variety of fatty acyl-homoserine lactones (HSLs), including N-3-oxododecanoyl-HSL (3OC12-HSL), which is produced and detected by the <span class="annotated ">P. aeruginosa</span> quorum-sensing circuit LasI and LasR. As is true for LasR and many other acyl-HSL-dependent transcription factors, production of soluble QscR in sufficient amounts for purification requires growth of recombinant bacteria in the presence of an appropriate acyl-HSL. QscR is thought to bind 3OC12-HSL relatively weakly compared to LasR, and unlike LasR, binding of purified QscR to target DNA was shown to strongly depend on exogenously added 3OC12-HSL. We show that purified QscR is dimeric at sufficiently high concentrations and monomeric at lower concentrations. Furthermore, QscR bound 3OC12-HSL more tightly than previously believed. Purified QscR retained 3OC12-HSL, and at sufficiently high concentrations, it bound target DNA in the absence of added 3OC12-HSL. We also obtained soluble QscR from recombinant <span class="match annotated ">Escherichia coli</span> grown in the presence of N-3-oxohexanoyl-HSL (3OC6-HSL) instead of 3OC12-HSL, and because 3OC6-HSL bound much more loosely to QscR than other acyl-HSLs tested, we were able to exchange 3OC6-HSL with other acyl-HSLs in vitro and then estimate binding affinities of QscR for different acyl-HSLs and for target DNA. Our data support a model whereby QscR polypeptides fold properly in the absence of an acyl-HSL, but soluble, acyl-HSL-free QscR does not accumulate because it is subject to rapid aggregation or proteolysis. 
<h3>21097634</h3>Role of the F1 region in the <span class="match annotated ">Escherichia coli</span> aerotaxis receptor Aer.<br><br>In <span class="match annotated ">Escherichia coli</span>, the aerotaxis receptor Aer is an atypical receptor because it senses intracellular redox potential. The Aer sensor is a cytoplasmic, N-terminal PAS domain that is tethered to the membrane by a 47-residue F1 linker. Here we investigated the function, topology, and orientation of F1 by employing random mutagenesis, cysteine scanning, and disulfide cross-linking. No native residue was obligatory for function, most deleterious substitutions had radically different side chain properties, and all F1 mutants but one were functionally rescued by the chemoreceptor Tar. Cross-linking studies were consistent with the predicted alpha-helical structure in the N-terminal F1 region and demonstrated trigonal interactions among the F1 linkers from three Aer monomers, presumably within trimer-of-dimer units, as well as binary interactions between subunits. Using heterodimer analyses, we also demonstrated the importance of arginine residues near the membrane interface, which may properly anchor the Aer protein in the membrane. By incorporating these data into a homology model of Aer, we developed a model for the orientation of the Aer F1 and PAS regions in an Aer lattice that is compatible with the known dimensions of the chemoreceptor lattice. We propose that the F1 region facilitates the orientation of PAS and HAMP domains during folding and thereby promotes the stability of the PAS and HAMP domains in Aer. 
<h3>21115656</h3>Novel members of the Cra regulon involved in carbon metabolism in <span class="match annotated ">Escherichia coli</span>. <br><br>Cra (catabolite repressor activator) is a global regulator of the genes for carbon metabolism in <span class="match annotated ">Escherichia coli</span>. To gain insights into the regulatory roles of Cra, attempts were made to identify the whole set of regulation targets using an improved genomic SELEX (systematic evolution of ligands by exponential enrichment) system. Surprisingly, a total of 164 binding sites were identified for Cra, 144 (88%) of which were newly identified. The majority of known targets were included in the SELEX chip pattern. The promoters examined by the lacZ reporter assay in vivo were all regulated by Cra. These two lines of evidence indicate that a total of as many as 178 promoters are under the control of Cra. The majority of Cra targets are the genes coding for the enzymes involved in central carbon metabolism, covering all the genes for the enzymes involved in glycolysis and metabolism downstream of glycolysis, including the tricarboxylic acid (TCA) cycle and aerobic respiration. Taken together, we propose that Cra plays a key role in balancing the levels of the enzymes for carbon metabolism. 
<h3>21131485</h3>Regulation of nleA in Shiga toxin-producing <span class="annotated ">Escherichia coli O84:H4 strain 4795/97</span>. <br><br>Many Shiga toxin-producing <span class="match annotated ">Escherichia coli</span> (<span class="annotated ">STEC</span>) strains express a type III secretion system (TTSS) encoded by the locus of enterocyte effacement (LEE). Using the TTSS, <span class="annotated ">STEC</span> is able to inject effector proteins directly into eukaryotic host cells, where they cause characteristic attaching and effacing (A/E) lesions. In addition to the LEE-encoded effectors, a number of non-LEE-encoded effectors, located on phage-associated elements, have been described. One of them, the non-LEE-encoded effector <span class="unannotated ">A</span> (NleA), is widely distributed among pathogenic <span class="annotated ">E. coli</span>. In this study, we investigated the influence of environmental conditions on the expression of the phage-encoded effector nleA gene (designated nleA(4795)) present in <span class="annotated ">STEC O84:H4 strain 4795/97</span>. We demonstrated that a particular NaCl concentration and starvation stress increase the activity of the nleA(4795) promoter. Moreover, several regulators that control nleA(4795) expression were identified. The involvement of the LEE regulators Ler, GrlA, and GrlR show that nleA(4795) is integrated in the LEE regulation circuit. Furthermore, the binding of Ler to sequences upstream of nleA(4795) underlined these findings. 
<h3>21131491</h3>The SMC-like protein complex SbcCD enhances DNA polymerase IV-dependent spontaneous mutation in <span class="match annotated ">Escherichia coli</span>. <br><br>In <span class="match annotated ">Escherichia coli</span>, RpoS, the general stress response sigma factor, regulates the activity of the specialized DNA polymerase DNA polymerase IV (Pol IV) both in stationary-phase and in exponential-phase cells. Because during exponential phase dinB, the gene encoding Pol IV, is transcribed independently of RpoS, RpoS must regulate Pol IV activity in growing cells indirectly via one or more intermediate factors. The results presented here show that one of these intermediate factors is SbcCD, an SMC-like protein and an ATP-dependent nuclease. By initiating or participating in double-strand break repair, SbcCD may provide DNA substrates for Pol IV polymerase activity. 
<h3>21148729</h3>Mapping functional domains of colicin M.<br><br>Colicin M (Cma) lyses <span class="match annotated ">Escherichia coli</span> cells by inhibiting murein biosynthesis through hydrolysis of the phosphate ester between C(55)-polyisoprenol and N-acetylmuramyl (MurNAc)-pentapeptide-GlcNAc in the periplasm. To identify Cma functional domains, we isolated 54 point mutants and small deletion mutants and examined their cytotoxicity levels. Activity and uptake mutants were distinguished by osmotic shock, which transfers Cma into the periplasm independent of the specific FhuA receptor and the Ton system. Deletion of the hydrophobic helix alpha1, which extends from the compact Cma structure, abolished interference with the antibiotic albomycin, which is transported across the outer membrane by the same system as Cma, thereby identifying alpha1 as the Cma site that binds to FhuA. Deletion of the C-terminal Lys-Arg strongly reduced Cma translocation across the outer membrane after binding to FhuA. Conversion of Asp226 to Glu, Asn, or Ala inactivated Cma. Asp226 is exposed at the Cma surface and is surrounded by Asp225, Asp229, His235, Tyr228, and Arg236; replacement of each with alanine inactivated Cma. We propose that Asp226 directly participates in phosphate ester hydrolysis and that the surrounding residues contribute to the active site. These residues are strongly conserved in Cma-like proteins of other species. Replacement of other conserved residues with alanine inactivated Cma; these mutations probably altered the Cma structure, as particularly apparent for mutants in the unique open Beta-barrel of Cma, which were isolated in lower yields. Our results identify regions in Cma responsible for uptake and activity and support the concept of a three-domain arrangement of Cma. 
<h3>21148732</h3>Functional and phylogenetic analysis of ureD in Shiga toxin-producing <span class="match annotated ">Escherichia coli</span>. <br><br>Enterohemorrhagic <span class="match annotated ">Escherichia coli</span> (<span class="annotated ">EHEC</span>) is a food-borne pathogen that can cause severe health complications and utilizes a much lower infectious dose than other <span class="annotated ">E. coli</span> pathotypes. Despite having an intact ure locus, ureDABCEFG, the majority of <span class="annotated ">EHEC</span> strains are phenotypically urease negative under tested conditions. Urease activity potentially assists with survival fitness by enhancing acid tolerance during passage through the stomach or by aiding with colonization in either <span class="annotated ">human</span> or animal reservoirs. Previously, in the <span class="annotated ">EHEC O157:H7 Sakai</span> strain, a point mutation in ureD, encoding a urease chaperone protein, was identified, resulting in a substitution of an amber stop codon for glutamine. This single nucleotide polymorphism (SNP) is observed in the majority of <span class="annotated ">EHEC O157:H7</span> isolates and correlates with a negative urease phenotype in vitro. We demonstrate that the lack of urease activity in vitro is not solely due to the amber codon in ureD. Our analysis has identified two additional SNPs in ureD affecting amino acid positions 38 and 205, in both cases determining whether the encoded amino acid is leucine or proline. Phylogenetic analysis based on Ure protein sequences from a variety of urease-encoding bacteria demonstrates that the proline at position 38 is highly conserved among Gram-negative bacteria. Experiments reveal that the L38P substitution enhances urease enzyme activity; however, the L205P substitution does not. Multilocus sequence typing analysis for a variety of Shiga toxin-producing <span class="annotated ">E. coli</span> isolates combined with the ureD sequence reveals that except for a subset of the <span class="annotated ">O157:H7</span> strains, neither the in vitro urease-positive phenotype nor the ureD sequence is phylogenetically restricted. 
<h3>21169483</h3>Unexpected coregulator range for the global regulator Lrp of <span class="match annotated ">Escherichia coli</span> and <span class="annotated ">Proteus mirabilis</span>. <br><br>The Lrp/AsnC family of transcription factors links gene regulation to metabolism in bacteria and archaea. Members of this family, collectively, respond to a wide range of amino acids as coregulators. In <span class="match annotated ">Escherichia coli</span>, Lrp regulates over 200 genes directly and is well known to respond to leucine and, to a somewhat lesser extent, alanine. We focused on Lrp from <span class="annotated ">Proteus mirabilis</span> and <span class="annotated ">E. coli</span>, orthologs with 98% identity overall and identical helix-turn-helix motifs, for which a previous study nevertheless found functional differences. Sequence differences between these orthologs, within and adjacent to the amino acid-responsive RAM domain, led us to test for differential sensitivity to coregulatory amino acids. In the course of this investigation, we found, via in vivo reporter fusion assays and in vitro electrophoretic mobility shift experiments, that <span class="annotated ">E. coli</span> Lrp itself responded to a broader range of amino acids than was previously appreciated. In particular, for both the <span class="annotated ">E. coli</span> and <span class="annotated ">P. mirabilis</span> orthologs, Lrp responsiveness to methionine was similar in magnitude to that to leucine. Both Lrp orthologs are also fairly sensitive to Ile, His, and Thr. These observations suggest that Lrp ties gene expression in the Enterobacteriaceae rather extensively to physiological status, as reflected in amino acid pools. These findings also have substantial implications for attempts to model regulatory architecture from transcriptome measurements or to infer such architecture from genome sequences, and they suggest that even well-studied regulators deserve ongoing exploration. 
<h3>21711401</h3>Identification and characterization of the cysteine protease inhibitor gene MdCPI from <span class="annotated ">Musca domestica</span>. <br><br>Cysteine proteinase inhibitors (CPIs) are involved in many vital cellular processes such as signalling pathways, apoptosis, immune response and development; however, no CPIs have yet been reported from the <span class="annotated ">housefly</span> <span class="annotated ">Musca domestica</span>. Here we report the isolation and characterization of a <span class="annotated ">housefly</span> CPI gene designated MdCPI. The gene contains an open reading frame of 357 bp encoding a protein of 118 amino acid residues with a putative signal peptide of 17 amino acid residues. Protein alignment demonstrated a high homology to that of <span class="annotated ">Sarcophaga crassipalpis</span> (identity = 51%). Phylogenetic analysis suggested that all CPIs from dipterans, including the <span class="annotated ">housefly</span>, belong to the I25A family and may be descended from a single common ancestor. The gene was expressed in and purified from <span class="match annotated ">Escherichia coli</span>. Biochemical studies showed that MdCPI exerts an inhibiting function on papain, which is a classical assay to confirm CPIs. Real-time quantitative PCR and immunolocalization analysis revealed that MdCPI is specifically expressed in haemocytes and fat bodies. It is highly down-regulated in larvae and markedly up-regulated in the pupal stage, suggesting that it may be related to development. 
<h3>20946420</h3>Association between nonsynonymous mutations of starch synthase IIa and starch quality in <span class="annotated ">rice</span> (<span class="annotated ">Oryza sativa</span>). <br><br>Starch quality is one of the most important agronomic traits in Asian <span class="annotated ">rice</span>, <span class="annotated ">Oryza sativa</span>. Starch synthase IIa (SsIIa) is a major candidate gene for starch quality variation. Within SsIIa, three nonsynonymous mutations in exon 8 have been shown to affect enzyme activity when expressed in <span class="match annotated ">Escherichia coli</span>. To search for the variation in SsIIa that is responsible for starch quality variation in <span class="annotated ">rice</span>, we sequenced the SsIIa exon 8 region and measured starch quality as starch disintegration in alkali for 289 accessions of cultivated <span class="annotated ">rice</span> and 57 accessions of its wild ancestor, <span class="annotated ">Oryza rufipogon</span>. <span class="unannotated ">A</span> general linear model and nested clade analysis were used to identify the associations between the three nonsynonymous single nucleotide polymorphisms (SNPs) and starch quality. Among the three nonsynonymous SNPs, we found strong evidence of association at one nucleotide site ('SNP 3'), corresponding to a Leu/Phe replacement at codon 781. <span class="unannotated ">A</span> second SNP, corresponding to a Val/Met replacement at codon 737, could potentially show an association with increased sample sizes. Variation in SsIIa enzyme activity is associated with the cohesiveness of <span class="annotated ">rice</span> grains when cooked, and our findings are consistent with selection for more cohesive grains during the domestication of tropical <span class="annotated ">japonica rice</span>. 
<h3>21037004</h3>Orientation of SecA and SecB in complex, derived from disulfide cross-linking.<br><br>SecA is the ATPase that acts as the motor for protein export in the general secretory, or Sec, system of <span class="match annotated ">Escherichia coli</span>. The tetrameric cytoplasmic chaperone SecB binds to precursors of exported proteins before they can become stably folded and delivers them to SecA. During this delivery step, SecB binds to SecA. The complex between SecA and SecB that is maximally active in translocation contains two protomers of SecA bound to a tetramer of SecB. The aminoacyl residues on each protein that are involved in binding the other have previously been identified by site-directed spin labeling and electron paramagnetic resonance (EPR) spectroscopy; however, that study provided no information concerning the relative orientation of the proteins within the complex. Here we used our extensive collection of single-cysteine variants of the two proteins and subjected pairwise combinations of SecA and SecB to brief oxidation to identify residues in close proximity. These data were used to generate a model for the orientation of the two proteins within the complex. 
<h3>21037010</h3>Inactivation of the RluD pseudouridine synthase has minimal effects on growth and ribosome function in wild-type <span class="match annotated ">Escherichia coli</span> and <span class="annotated ">Salmonella enterica</span>. <br><br>The <span class="match annotated ">Escherichia coli</span> rluD gene encodes a pseudouridine synthase responsible for the pseudouridine (Psi) modifications at positions 1911, 1915, and 1917 in helix 69 of 23S rRNA. It has been reported that deletion of rluD in <span class="annotated ">K-12</span> strains of <span class="annotated ">E. coli</span> is associated with extremely slow growth, increased readthrough of stop codons, and defects in 50S ribosomal subunit assembly and 30S-50S subunit association. Suppressor mutations in the prfB and prfC genes encoding release factor 2 (RF2) and RF3 that restore the wild type-growth rate and also correct the ribosomal defects have now been isolated. These suppressors link helix 69 Psi residues with the termination phase of protein synthesis. However, further genetic analysis reported here also reveals that the slow growth and other defects associated with inactivation of rluD in <span class="annotated ">E. coli K-12</span> strains are due to a defective RF2 protein, with a threonine at position 246, which is present in all <span class="annotated ">K-12</span> strains. This is in contrast to the more typical alanine found at this position in most bacterial RF2s, including those of other <span class="annotated ">E. coli</span> strains. Inactivation of rluD in <span class="annotated ">E. coli</span> strains containing the prfB allele from <span class="annotated ">E. coli B</span> or in <span class="annotated ">Salmonella enterica</span>, both carrying an RF2 with Ala246, has negligible effects on growth, termination, or ribosome function. The results indicate that, in contrast to those in wild bacteria, termination functions in <span class="annotated ">E. coli K-12</span> strains carrying a partially defective RF2 protein are especially susceptible to perturbation of ribosome-RF interactions, such as that caused by loss of h69 Psi modifications. 
<h3>21097628</h3>Promoter discrimination at <span class="unannotated ">class I</span> MarA regulon promoters mediated by glutamic acid 89 of the MarA transcriptional activator of <span class="match annotated ">Escherichia coli</span>. <br><br>Three paralogous transcriptional activators MarA, SoxS, and Rob, activate > 40 <span class="match annotated ">Escherichia coli</span> promoters. To understand why MarA does not activate certain promoters as strongly as SoxS, we compared MarA, MarA mutants, and SoxS for their abilities to activate 16 promoters and to bind their cognate marbox binding sites. Replacement of the MarA glutamic acid residue 89 with alanine greatly increased the marbox binding and activation of many <span class="unannotated ">class I</span> promoters. Like cells constitutive for SoxS, cells expressing the MarA with the E89A mutation were more resistant to superoxides than those harboring WT MarA. The activities of several other E89 substitutions ranked as follows: E89A > E89G > E89V > WT > E89D. Increased binding and activation occurred only at <span class="unannotated ">class I</span> promoters when the 12th base of the promoter's marbox (a position at which there is no known interaction between the marbox and MarA) was not a T residue. Furthermore, WT MarA binding to a synthetic marbox in vitro was enhanced when the phosphate group between positions 12 and 13 was eliminated on one strand. The results demonstrate that relatively minor changes in a single amino acid side chain (e.g., alanine to valine or glutamic acid to aspartic acid) can strongly influence activity despite any evidence that the side chain is involved in positive interactions with either DNA or RNA polymerase. We present a model which attributes the differences in binding and activation to the interference between the Beta- and gamma-carbons of the amino acid at position 89 and the phosphate group between positions 12 and 13. 
<h3>21115658</h3>Interbacterial macromolecular transfer by the <span class="annotated ">Campylobacter fetus subsp. venerealis</span> type IV secretion system. <br><br>We report here the first demonstration of intra- and interspecies conjugative plasmid DNA transfer for <span class="annotated ">Campylobacter fetus</span>. Gene regions carried by a <span class="annotated ">Campylobacter coli</span> plasmid were identified that are sufficient for conjugative mobilization to <span class="match annotated ">Escherichia coli</span> and <span class="annotated ">C. fetus</span> recipients. <span class="unannotated ">A</span> broader functional range is predicted. Efficient DNA transfer involves the virB9 and virD4 genes of the type IV bacterial secretion system encoded by a pathogenicity island of <span class="annotated ">C. fetus subsp. venerealis</span>. Complementation of these phenotypes from expression constructions based on the promoter of the <span class="annotated ">C. fetus</span> surface antigen protein (sap) locus was temperature dependent, and a temperature regulation of the sap promoter was subsequently confirmed under laboratory conditions. Gene transfer was sensitive to surface or entry exclusion functions in potential recipient cells carrying IncPalpha plasmid RP4 implying functional relatedness to <span class="annotated ">C. fetus</span> proteins. The virB/virD4 locus is also known to be involved in bacterial invasion and killing of cultured <span class="annotated ">human</span> cells in vitro. Whether specifically secreted effector proteins contribute to host colonization and infection activities is currently unknown. Two putative effector proteins carrying an FIC domain conserved in a few bacterial type III and type IV secreted proteins of pathogens were analyzed for secretion by the <span class="annotated ">C. fetus</span> or heterologous conjugative systems. No evidence for interbacterial translocation of the Fic proteins was found. 
<h3>21131487</h3>A caffeyl-coenzyme <span class="unannotated ">A</span> synthetase initiates caffeate activation prior to caffeate reduction in the acetogenic bacterium <span class="annotated ">Acetobacterium woodii</span>. <br><br>The anaerobic acetogenic bacterium <span class="annotated ">Acetobacterium woodii</span> couples the reduction of caffeate with electrons derived from hydrogen to the synthesis of ATP by a chemiosmotic mechanism using sodium ions as coupling ions, but the enzymes involved remain to be established. Previously, the electron transfer flavoproteins EtfA and EtfB were found to be involved in caffeate respiration. By inverse PCR, we identified three genes upstream of etfA and etfB: carA, carB, and carC. carA encodes a potential coenzyme <span class="unannotated ">A</span> (CoA) transferase, carB an acyl-CoA synthetase, and carC an acyl-CoA dehydrogenase. carA, -B, and -C are located together with etfA/carE and etfB/carD on one polycistronic message, indicating that CarA, CarB, and CarC are also part of the caffeate respiration pathway. The genetic data suggest an initial ATP-dependent activation of caffeate by CarB. To prove the proposed function of CarB, the protein was overproduced in <span class="match annotated ">Escherichia coli</span>, and the recombinant protein was purified. Purified CarB activates caffeate to caffeyl-CoA in an ATP- and CoA-dependent reaction. The enzyme has broad pH and temperature optima and requires K(+) for activity. In addition to caffeate, it can use p-coumarate, ferulate, and cinnamate as substrates, with 50, 15, and 9%, respectively, of the activity obtained with caffeate. Expression of the car operon is induced not only by caffeate, p-coumarate, ferulate, and cinnamate but also by sinapate. There is no induction by p-hydroxybenzoate or syringate. 
<h3>21324422</h3>A clone-based transcriptomics approach for the identification of genes relevant for itaconic acid production in Aspergillus. <br><br>Several Aspergillus species are well-known for the production of a variety of organic acids. In this study, a cloned based transcriptomics approach was used to identify genes crucial in the biosynthesis pathway for one of these acids, itaconic acid. From a number of different <span class="annotated ">Aspergillus terreus</span> controlled batch fermentations, those cultures with the largest difference in itaconic acid titer and productivity were selected for mRNA isolation. cDNAs derived from these mRNA samples were used for subsequent hybridization of glass slide arrays based on a collection of 5000 cDNA clones. <span class="unannotated ">A</span> selection of 13 cDNA clones resulting in the strongest (>10-fold) differential hybridization signals were identified and subsequently the inserts of these clones were sequenced. Sequence analysis revealed the presence of in total five different gene inserts among the sequenced clones. From one of these sequences, encoding a gene of the MmgE-PrpD family, the full length coding region was shown to encode one of the crucial itaconic acid pathway enzymes cis-aconitate decarboxylase, by heterologous expression in <span class="match annotated ">Escherichia coli</span>. Expression of this gene in <span class="annotated ">Aspergillus niger</span>, which is a natural citric acid producer, resulted in itaconate production. Genome analysis suggests that in <span class="annotated ">A. terreus</span> the cis-aconitate decarboxylase gene is part of an itaconate acid related gene cluster including genes encoding two pathway specific transporters and a Zinc finger protein. Interestingly, this cluster is directly linked to the large lovastatin gene cluster. 
<h3>21435060</h3>Characterization of an omega-class glutathione S-transferase in the stress response of the <span class="annotated ">silkmoth</span>. <br><br>The glutathione S-transferase (GST) superfamily is involved in detoxification of various xenobiotics. Using real-time PCR, mRNA encoding an omega-class GST of <span class="annotated ">Bombyx mori</span> (bmGSTO) was shown to be induced after exposure to various environmental stresses. <span class="unannotated ">A</span> soluble form of recombinant protein (rbmGSTO) was functionally overexpressed in <span class="match annotated ">Escherichia coli</span> cells and purified to homogeneity. Cys 38 and Pro 39 were found to be highly conserved in omega-class GSTs, and their roles were investigated by site-directed mutagenesis/kinetic analysis. Mutations of Cys 38 and Pro 39 residues affected the catalytic efficiency of enzymes, indicating that the presence of Cys 38 and Pro 39 residues is important for bmGSTO activity. Thus, bmGSTO could contribute to increasing the environmental stress resistance of lepidopteran insects. 
<h3>21906194</h3>Prolixicin: a novel antimicrobial peptide isolated from <span class="annotated ">Rhodnius prolixus</span> with differential activity against bacteria and <span class="annotated ">Trypanosoma cruzi</span>. <br><br>We identified and characterized the activity of prolixicin, a novel antimicrobial peptide (AMP) isolated from the hemipteran insect, <span class="annotated ">Rhodnius prolixus</span>. Sequence analysis reveals one region of prolixicin that may be related to the diptericin/attacin family of AMPs. Prolixicin is an 11-kDa peptide containing a putative 21 amino acid signal peptide, two putative phosphorylation sites and no glycosylation sites. It is produced by both adult fat body and midgut tissues in response to bacterial infection of the haemolymph or the midgut. Unlike most insect antibacterial peptides, the prolixicin gene does not seem to be regulated by NF-kappaB binding sites, but its promoter region contains several GATA sites. Recombinant prolixicin has strong activity against the Gram-negative bacterium <span class="match annotated ">Escherichia coli</span> and differential activity against several Gram-negative and Gram-positive bacteria. No significant toxicity was demonstrated against <span class="annotated ">Trypanosoma cruzi</span>, the <span class="annotated ">human</span> parasite transmitted by <span class="annotated ">R. prolixus</span>. 
<h3>21037009</h3>Activation of the SoxR regulon in <span class="annotated ">Streptomyces coelicolor</span> by the extracellular form of the pigmented antibiotic actinorhodin. <br><br>The redox-sensitive transcription factor SoxR in enteric bacteria senses and regulates the cellular response to superoxide and nitric oxide. In other bacterial groups, however, it may respond to redox-active small molecules, as demonstrated for pyocyanin sensing in pseudomonads. The antibiotic-producing soil bacterium <span class="annotated ">Streptomyces coelicolor</span> contains a gene for an SoxR homologue (SCO1697) whose DNA recognition helix is identical to that of <span class="match annotated ">Escherichia coli</span> SoxR. Using the <span class="annotated ">E. coli</span> SoxR binding sequence, we predicted five candidate genes of the SoxR regulon and demonstrated that SoxR binds to their promoter regions and activates their expression concurrently with the production of the blue antibiotic actinorhodin (a benzoisochromanequinone). These genes encode a probable NADPH-dependent flavin reductase (SCO2478), an NADPH-dependent quinone reductase (SCO4266), an ABC transporter (SCO7008), a monooxygenase (SCO1909), and a hypothetical protein (SCO1178). Addition of actinorhodin to exponentially growing cells activated the expression of SoxR target genes in an SoxR-dependent manner. The secreted gamma-actinorhodin was over 10-fold more effective in activation than the intracellular form of actinorhodin, suggesting that SoxR is specified to respond more to exogenous signals than to intracellular metabolites. The DeltasoxR mutant was not compromised in resistance against oxidants but was slow in forming aerial mycelium on R2YE medium with reduced sporulation, and its production of actinorhodin and undecylprodigiosin was lowered by about 50% and 30%, respectively, compared to that of the wild type. These results support the proposal that SoxR senses redox-active molecules, such as actinorhodin in <span class="annotated ">S. coelicolor</span>, and induces a protective function against them. It also functions to ensure that cells undergo optimal differentiation and secondary metabolite production. 
</body></html>