
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>H3</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>20659575</h3>Regulation of secondary metabolism by chromatin structure and epigenetic codes.<br><br>Chromatin, composed of DNA wrapped around an octamer of histones, is the relevant substrate for all genetic processes in eukaryotic nuclei. Changes in chromatin structure are associated with the activation and silencing of gene transcription and reversible post-translational modifications of histones are now known to direct chromatin structure transitions. Recent studies in several fungal species have identified a chromatin-based regulation of secondary metabolism (SM) gene clusters representing an upper-hierarchical level for the coordinated control of large chromosomal elements. Regulation by chromatin transition processes provides a mechanistic model to explain how different SM clusters located at dispersed genomic regions can be simultaneously silenced during primary metabolism. Activation of SM clusters has been shown to be associated with increased acetylation of histones <span class="match unannotated ">H3</span> and H4 and, consequently, inhibition of histone de-acetylase activities also leads to increased production of secondary metabolites. New findings suggest that SM clusters are silenced by heterochromatic histone marks and that the &quot;closed&quot; heterochromatic structures are reversed during SM activation. This process is mediated by the conserved activator of SM, LaeA. Despite the increase in knowledge about these processes, much remains to be learned from chromatin-level regulation of SM. For example, which proteins &quot;position&quot; the chromatin restructuring signal onto SM clusters or how exactly LaeA works to mediate the low level of heterochromatic marks inside different clusters remain open questions. Answers to these and other chromatin-related questions would certainly complete our understanding of SM gene regulation and signaling and, because for many predicted SM clusters corresponding products have not been identified so far, anti-silencing strategies would open new ways for the identification of novel bioactive substances. 
<h3>20980523</h3>T cell-mediated protection against lethal 2009 pandemic <span class="annotated ">H1N1 influenza virus</span> infection in a <span class="annotated ">mouse</span> model. <br><br>Genetic mutation and reassortment of <span class="annotated ">influenza virus</span> gene segments, in particular those of hemagglutinin (HA) and neuraminidase (NA), that lead to antigenic drift and shift are the major strategies for <span class="annotated ">influenza virus</span> to escape preexisting immunity. The most recent example of such phenomena is the first pandemic of <span class="annotated ">H1N1 influenza</span> of the 21st century, which started in 2009. Cross-reactive antibodies raised against <span class="annotated ">H1N1 viruses</span> circulating before 1930 show protective activity against the 2009 pandemic virus. Cross-reactive T-cell responses can also contribute to protection, but in vivo support of this view is lacking. To explore the protection mechanisms in vivo, we primed <span class="annotated ">mice</span> with <span class="annotated ">H1</span> and <span class="annotated ">H3 influenza virus</span> isolates and rechallenged them with a virus derived from the 2009 <span class="annotated ">H1N1 A/CA/04/09 virus</span>, named CA/E3/09. We found that priming with <span class="annotated ">influenza viruses</span> of both <span class="annotated ">H1</span> and <span class="match annotated ">H3</span> homo- and heterosubtypes protected against lethal CA/E3/09 virus challenge. Convalescent-phase sera from these primed <span class="annotated ">mice</span> conferred no neutralization activity in vitro and no protection in vivo. However, T-cell depletion studies suggested that both CD4 and CD8 T cells contributed to the protection. Taken together, these results indicate that cross-reactive T cells established after initial priming with distally related <span class="unannotated ">viruses</span> can be a vital component for prevention of disease and control of pandemic <span class="annotated ">H1N1 influenza virus</span> infection. Our results highlight the importance of establishing cross-reactive T-cell responses for protecting against existing or newly emerging pandemic <span class="annotated ">influenza viruses</span>. 
<h3>21029748</h3>One-step real-time reverse transcription-PCR assays for detecting and subtyping pandemic <span class="annotated ">influenza A/H1N1</span> 2009, seasonal <span class="annotated ">influenza A/H1N1</span>, and seasonal <span class="annotated ">influenza A/H3N2 viruses</span>. <br><br>Pandemic <span class="annotated ">influenza A/H1N1</span> 2009 (<span class="annotated ">A/H1N1pdm</span>) virus has caused significant outbreaks worldwide. <span class="unannotated ">A</span> previous one-step real-time reverse transcription-PCR (rRT-PCR) assay for detecting <span class="annotated ">A/H1N1pdm virus</span> (H1pdm rRT-PCR assay) was improved since the former probe had a low melting temperature and low tolerance to <span class="unannotated ">viral</span> mutation. To help with the screening of the <span class="annotated ">A/H1N1pdm virus</span>, rRT-PCR assays were also developed for detecting <span class="annotated ">human</span> seasonal <span class="annotated ">A/H1N1</span> (H1 rRT-PCR assay) and <span class="annotated ">A/H3N2 influenza viruses</span> (H3 rRT-PCR assay). H1pdm, <span class="unannotated ">H1</span>, and <span class="match unannotated ">H3</span> rRT-PCR assays were evaluated using in vitro-transcribed control RNA, isolated <span class="unannotated ">viruses</span>, and other respiratory pathogenic <span class="unannotated ">viruses</span>, and were shown to have high sensitivity, good linearity (R(2)=0.99), and high specificity. In addition, the improved H1pdm rRT-PCR assay could detect two <span class="annotated ">viral strains of A/H1N1pdm</span>, namely, <span class="annotated ">A/Aichi/472/2009 (H1N1)pdm</span> and <span class="annotated ">A/Sakai/89/2009 (H1N1)pdm</span>, which have mutation(s) in the probe-binding region of the hemagglutinin gene, without loss of sensitivity. Using the three rRT-PCR assays developed, 90 clinical specimens collected between May and October 2009 were then tested. Of these, 26, 20, and 2 samples were identified as positive for <span class="annotated ">A/H1pdm</span>, <span class="annotated ">A/H3</span>, and <span class="annotated ">A/H1</span>, respectively, while 42 samples were negative for <span class="annotated ">influenza A viruses</span>. The present results suggest that these highly sensitive and specific H1pdm, <span class="unannotated ">H1</span>, and <span class="match unannotated ">H3</span> rRT-PCR assays are useful not only for diagnosing <span class="annotated ">influenza viruses</span>, but also for the surveillance of <span class="annotated ">influenza viruses</span>. 
</body></html>