
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>man</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>20190019</h3><span class="annotated ">Auritidibacter ignavus gen. nov., sp. nov.</span>, of the family Micrococcaceae isolated from an ear swab of a <span class="match LINNAEUS_only ">man</span> with otitis externa, transfer of the members of the family Yaniellaceae Li et al. 2008 to the family Micrococcaceae and emended description of the suborder Micrococcineae. <br><br>A Gram-reaction-positive, aerobic, catalase-positive, non-spore-forming, rod-shaped bacterium designated <span class="annotated ">IMMIB L-1656(T)</span> was isolated from an ear swab of a <span class="match LINNAEUS_only ">man</span> and characterized using a polyphasic approach. 16S rRNA gene sequence analysis indicated that strain <span class="annotated ">IMMIB L-1656(T)</span> is related to members of the family Micrococcaceae (&lt;95.1 % sequence similarity). Anaylsis using different phylogenetic algorithms consistently grouped strain <span class="annotated ">IMMIB L-1656(T)</span> with members of the genus Yaniella. The organism posessed a cell-wall murein based on L-lysine (variation A4alpha, type L-Lys-Gly-L-Glu), MK-10 as the predominant menaquinone and long-chain cellular fatty acids of straight-chain and branched-chain saturated types (with iso-C(15 : 0) and anteiso-C(17 : 0) predominating). The polar lipids included diphosphatidylglycerol, phosphatidylglycerol and phosphatidylinositol in addition to unknown glycolipids. The DNA G+C content was 59.7 mol%. Based on its distinctive genotypic and phenotypic characteristics, strain <span class="annotated ">IMMIB L-1656(T)</span> represents a novel species in a novel genus, for which the name <span class="annotated ">Auritidibacter ignavus gen. nov., sp. nov.</span> is proposed. We also propose that members of the family Yaniellaceae be transferred to the family Micrococcaceae with amendments to the description of the suborder Micrococcineae. The type strain of <span class="annotated ">Auritidibacter ignavus</span> is <span class="annotated ">IMMIB L-1656(T)</span> (=<span class="annotated ">DSM 45359(T)</span> =<span class="annotated ">CCUG 57943(T)</span>). 
<h3>21084112</h3>Ebola haemorrhagic fever.<br><br><span class="annotated ">Ebola viruses</span> are the causative agents of a severe form of <span class="unannotated ">viral</span> haemorrhagic fever in <span class="match LINNAEUS_only ">man</span>, designated Ebola haemorrhagic fever, and are endemic in regions of central Africa. The exception is the species <span class="annotated ">Reston Ebola virus</span>, which has not been associated with <span class="annotated ">human</span> disease and is found in the Philippines. <span class="annotated ">Ebola virus</span> constitutes an important local public health threat in Africa, with a worldwide effect through imported infections and through the fear of misuse for biological terrorism. <span class="annotated ">Ebola virus</span> is thought to also have a detrimental effect on the great ape population in Africa. Case-fatality rates of the African species in <span class="match LINNAEUS_only ">man</span> are as high as 90%, with no prophylaxis or treatment available. <span class="annotated ">Ebola virus</span> infections are characterised by immune suppression and a systemic inflammatory response that causes impairment of the vascular, coagulation, and immune systems, leading to multiorgan failure and shock, and thus, in some ways, resembling septic shock. 
<h3>21126914</h3>Tracking a century of global expansion and evolution of <span class="annotated ">HIV</span> to drive understanding and to combat disease. <br><br>Since the isolation of <span class="annotated ">HIV</span>, multiple transmissions are thought to have occurred between <span class="match LINNAEUS_only ">man</span> and other old-world primates. Assessment of samples from apes and <span class="annotated ">human</span> beings with African equatorial forest ancestry has traced the origin of <span class="annotated ">HIV-1</span> to <span class="annotated ">chimpanzees</span>, and dated its most recent common ancestor to 1908. The evolution of <span class="annotated ">HIV-1</span> has been rapid, which has resulted in a complex classification, worldwide spread, and intermixing of strains; at least 48 circulating recombinant forms are currently identified. In addition to posing a nearly insurmountable challenge for diagnosis, treatment, vaccine development, and prevention, this extreme and divergent evolution has led to differences in virulence between <span class="annotated ">HIV-1</span> groups, subtypes, or both. Coincidental changes in <span class="annotated ">human</span> migration in the Congo river basin also affected spread of disease. Research over the past 25 years and advances in genomic sequencing methods, such as deep DNA sequencing, have greatly improved understanding and analysis of the thousands to millions of full infectious <span class="annotated ">HIV-1</span> genomes. 
</body></html>