
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Pseudomonas aeruginosa</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>20173013</h3>Streptomyces lacticiproducens sp. nov., a lactic acid-producing streptomycete isolated from the rhizosphere of <span class="annotated ">tomato</span> plants. <br><br>A novel actinomycete, designated strain GIMN4.001(T), was isolated from the rhizosphere of <span class="annotated ">tomato</span> plants grown in Guangzhou, China. The strain produced greyish white aerial mycelia, lactic acid and a large quantity of double diamond-shaped crystals on <span class="annotated ">potato</span> dextrose agar and <span class="annotated ">yeast</span> extract-malt extract agar. The colour of the substrate mycelium was not sensitive to pH. Microscopic observations revealed that strain GIMN4.001(T) produced verticillate chains of cylindrical spores. Chemotaxonomic data confirmed that strain GIMN4.001(T) belonged to the genus Streptomyces. Melanin pigments were not produced. No antibacterial activity was observed against <span class="annotated ">Escherichia coli</span>, <span class="match annotated ">Pseudomonas aeruginosa</span>, <span class="annotated ">Bacillus subtilis</span> or <span class="annotated ">Candida albicans</span>, but inhibitory activity was observed against <span class="annotated ">Penicillium citrinum</span>. 16S rRNA gene sequence analysis revealed that strain GIMN4.001(T) was related most closely to <span class="annotated ">Streptomyces morookaense</span> <span class="annotated ">ATCC 19166(T)</span> (98.9 % similarity) and <span class="annotated ">Streptomyces lavenduligriseus</span> <span class="annotated ">ATCC 13306(T)</span> (98.7 %). Levels of DNA-DNA relatedness between strain GIMN4.001(T) and the type strains of these species were low (14-20 %). Furthermore, strain GIMN4.001(T) could be differentiated from <span class="annotated ">S. morookaense</span>, S. lavenduligriseus and other closely related species of the genus Streptomyces based on morphological, physiological and biochemical characteristics. On the basis of its physiological and molecular properties, strain GIMN4.001(T) is considered to represent a novel species of the genus Streptomyces, for which the name Streptomyces lacticiproducens sp. nov. is proposed. The type strain is GIMN4.001(T) (=CCTCC M208214(T)=NRRL B-24800(T)). 
<h3>21097632</h3>Acyl-homoserine lactone binding to and stability of the orphan <span class="match annotated ">Pseudomonas aeruginosa</span> quorum-sensing signal receptor QscR. <br><br>The <span class="match annotated ">Pseudomonas aeruginosa</span> transcription factor QscR responds to a variety of fatty acyl-homoserine lactones (HSLs), including N-3-oxododecanoyl-HSL (3OC12-HSL), which is produced and detected by the <span class="annotated ">P. aeruginosa</span> quorum-sensing circuit LasI and LasR. As is true for LasR and many other acyl-HSL-dependent transcription factors, production of soluble QscR in sufficient amounts for purification requires growth of recombinant bacteria in the presence of an appropriate acyl-HSL. QscR is thought to bind 3OC12-HSL relatively weakly compared to LasR, and unlike LasR, binding of purified QscR to target DNA was shown to strongly depend on exogenously added 3OC12-HSL. We show that purified QscR is dimeric at sufficiently high concentrations and monomeric at lower concentrations. Furthermore, QscR bound 3OC12-HSL more tightly than previously believed. Purified QscR retained 3OC12-HSL, and at sufficiently high concentrations, it bound target DNA in the absence of added 3OC12-HSL. We also obtained soluble QscR from recombinant <span class="annotated ">Escherichia coli</span> grown in the presence of N-3-oxohexanoyl-HSL (3OC6-HSL) instead of 3OC12-HSL, and because 3OC6-HSL bound much more loosely to QscR than other acyl-HSLs tested, we were able to exchange 3OC6-HSL with other acyl-HSLs in vitro and then estimate binding affinities of QscR for different acyl-HSLs and for target DNA. Our data support a model whereby QscR polypeptides fold properly in the absence of an acyl-HSL, but soluble, acyl-HSL-free QscR does not accumulate because it is subject to rapid aggregation or proteolysis. 
<h3>21169488</h3>The sensor kinase CbrA is a global regulator that modulates metabolism, virulence, and antibiotic resistance in <span class="match annotated ">Pseudomonas aeruginosa</span>. <br><br><span class="match annotated ">Pseudomonas aeruginosa</span> is an opportunistic pathogen that possesses a large arsenal of virulence factors enabling the pathogen to cause serious infections in immunocompromised <span class="LINNAEUS_only ">patients</span>, burn victims, and cystic fibrosis <span class="LINNAEUS_only ">patients</span>. CbrA is a sensor kinase that has previously been implied to play a role with its cognate response regulator CbrB in the metabolic regulation of carbon and nitrogen utilization in <span class="annotated ">P. aeruginosa</span>. Here it is demonstrated that CbrA and CbrB play an important role in various virulence and virulence-related processes of the bacteria, including swarming, biofilm formation, cytotoxicity, and antibiotic resistance. The cbrA deletion mutant was completely unable to swarm while exhibiting an increase in biofilm formation, supporting the inverse regulation of swarming and biofilm formation in <span class="annotated ">P. aeruginosa</span>. The cbrA mutant also exhibited increased cytotoxicity to <span class="annotated ">human</span> lung epithelial cells as early as 4 and 6 h postinfection. Furthermore, the cbrA mutant demonstrated increased resistance toward a variety of clinically important antibiotics, including polymyxin B, ciprofloxacin, and tobramycin. Microarray analysis revealed that under swarming conditions, CbrA regulated the expression of many genes, including phoPQ, pmrAB, arnBCADTEF, dnaK, and pvdQ, consistent with the antibiotic resistance and swarming impairment phenotypes of the cbrA mutant. Phenotypic and real-time quantitative PCR (RT-qPCR) analyses of a PA14 cbrB mutant suggested that CbrA may be modulating swarming, biofilm formation, and cytotoxicity via CbrB and that the CrcZ small RNA is likely downstream of this two-component regulator. However, as CbrB did not have a resistance phenotype, CbrA likely modulates antibiotic resistance in a manner independent of CbrB. 
<h3>21075931</h3>MexT regulates the type III secretion system through MexS and PtrC in <span class="match unannotated ">Pseudomonas aeruginosa</span>. <br><br>The type III secretion system (T3SS) is the most important virulence factor in <span class="match unannotated ">Pseudomonas aeruginosa</span>, and its expression level varies in different isolates. We studied the molecular basis for such differences in two laboratory strains, PAK and PAO1. <span class="unannotated ">A</span> chromosomal clone library from the high-T3SS-producer strain PAK was introduced into the low-producer strain PAO1, and we found that a mexS gene from PAK confers high T3SS expression in the PAO1 background. Further tests demonstrated that both mexS and its neighboring mexT gene are required for the repression of the T3SS in PAO1, while the PAK genome encodes a defective MexS, accounting for the derepression of the T3SS in PAK and the dominant negative effect when it is introduced into PAO1. MexS is a probable oxidoreductase whose expression is dependent on MexT, a LysR-type transcriptional regulator. Various genetic data support the idea that MexS modulates the transcriptional regulator function of MexT. In searching for the MexT-dependent repressor of the T3SS, a small gene product of PA2486 (ptrC) was found effective in suppressing the T3SS upon overexpression. However, deletion of ptrC in the PAO1 background did not result in derepression of the T3SS, indicating the presence of another repressor for the T3SS. Interestingly, overexpression of functional mexS alone was sufficient to repress T3SS even in the absence of MexT, suggesting that MexS is another mediator of MexT-dependent T3SS repression. Overexpression of mexS alone had no effect on the well-known MexT-dependent genes, including those encoding MexEF efflux pump, elastase, and pyocyanin, indicating alternative regulatory mechanisms. <span class="unannotated ">A</span> model has been proposed for the MexS/MexT-mediated regulation of the T3SS, the MexEF efflux pump, and the production of elastase and pyocyanin. 
<h3>21148734</h3>Relating the physical properties of <span class="match annotated ">Pseudomonas aeruginosa</span> lipopolysaccharides to virulence by atomic force microscopy. <br><br>Lipopolysaccharides (LPS) are an important class of macromolecules that are components of the outer membrane of Gram-negative bacteria such as <span class="match annotated ">Pseudomonas aeruginosa</span>. <span class="annotated ">P. aeruginosa</span> contains two different sugar chains, the homopolymer common antigen (A band) and the heteropolymer <span class="unannotated ">O</span> antigen (B band), which impart serospecificity. The characteristics of LPS are generally assessed after isolation rather than in the context of whole bacteria. Here we used atomic force microscopy (AFM) to probe the physical properties of the LPS of <span class="annotated ">P. aeruginosa strain PA103</span> (serogroup O11) in situ. This strain contains a mixture of long and very long polymers of <span class="unannotated ">O</span> antigen, regulated by two different genes. For this analysis, we studied the wild-type strain and four mutants, DeltaWzz1 (producing only very long LPS), DeltaWzz2 (producing only long LPS), DDeltaM (with both the wzz1 and wzz2 genes deleted), and Wzy::GM (producing an LPS core oligosaccharide plus one unit of <span class="unannotated ">O</span> antigen). Forces of adhesion between the LPS on these strains and the silicon nitride AFM tip were measured, and the Alexander and de Gennes model of steric repulsion between a flat surface and a polymer brush was used to calculate the LPS layer thickness (which we refer to as length), compressibility, and spacing between the individual molecules. LPS chains were longest for the wild-type strain and DeltaWzz1, at 170.6 and 212.4 nm, respectively, and these values were not statistically significantly different from one another. Wzy::GM and DDeltaM have reduced LPS lengths, at 34.6 and 37.7 nm, respectively. Adhesion forces were not correlated with LPS length, but a relationship between adhesion force and bacterial pathogenicity was found in a <span class="annotated ">mouse</span> acute pneumonia model of infection. The adhesion forces with the AFM probe were lower for strains with LPS mutations, suggesting that the wild-type strain is optimized for maximal adhesion. Our research contributes to further understanding of the role of LPS in the adhesion and virulence of <span class="annotated ">P. aeruginosa</span>. 
<h3>21097635</h3>The peptidoglycan-binding protein FimV promotes assembly of the <span class="match annotated ">Pseudomonas aeruginosa</span> type IV pilus secretin. <br><br>The <span class="match annotated ">Pseudomonas aeruginosa</span> inner membrane protein FimV is among several proteins of unknown function required for type IV pilus-mediated twitching motility, arising from extension and retraction of pili from their site of assembly in the inner membrane. The pili transit the periplasm and peptidoglycan (PG) layer, ultimately exiting the cell through the PilQ secretin. Although fimV mutants are nonmotile, they are susceptible to killing by pilus-specific <span class="annotated ">bacteriophage</span>, a hallmark of retractable surface pili. Here we show that levels of recoverable surface pili were markedly decreased in fimV pilT retraction-deficient mutants compared with levels in the pilT control, demonstrating that FimV acts at the level of pilus assembly. Levels of inner membrane assembly subcomplex proteins PilM/N/O/P were decreased in fimV mutants, but supplementation of these components in trans did not restore pilus assembly or motility. Loss of FimV dramatically reduced the levels of the PilQ secretin multimer through which pili exit the cell, in part due to decreased levels of PilQ monomers, while PilF pilotin levels were unchanged. Expression of pilQ in trans in the wild type or fimV mutants increased total PilQ monomer levels but did not alter secretin multimer levels or motility. PG pulldown assays showed that the N terminus of FimV bound PG in a LysM motif-dependent manner, and a mutant with an in-frame chromosomal deletion of the LysM motif had reduced motility, secretin levels, and surface piliation. Together, our data show that FimV's role in pilus assembly is to promote secretin formation and that this function depends upon its PG-binding domain. 
</body></html>