
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>rice</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>20663713</h3>Conservation of POPs, the plant organellar DNA polymerases, in eukaryotes.<br><br>POPs, plant organellar DNA polymerases, have been isolated from various photosynthetic eukaryotes. Previously, we purified the native POP of <span class="annotated ">Cyanidioschyzon merolae</span> (CmPOP) from whole cellular extracts and showed that CmPOP has DNA polymerase activity with a high processivity and a 3'-5' exonuclease activity, and its expression is related to cell proliferation. In <span class="match annotated ">rice</span>, the recombinant protein of POP has activities found in CmPOP, and high fidelity of POP has also been demonstrated. These facts suggest that POPs are involved in the replication of organellar genomes. POPs are also conserved in most non-opisthokont eukaryotes, which lack DNA polymerase gamma (Polgamma), a mitochondrial replication enzyme in opisthokonts (fungi and animals). The ciliate <span class="annotated ">Tetrahymena thermophila</span> contains a single gene for a putative POP (TetPOP). Immunoblot analysis demonstrated that TetPOP is localized in mitochondria, and TetPOP has been purified from mitochondria through a column chromatography series. Sensitivity to phosphonoacetate and dideoxyTTP was examined in POPs (TetPOP and CmPOP) or POP-containing organelles (chloroplasts of Arabidopsis) and other polymerases (DNA polymerase I and mitochondria of <span class="annotated ">rat</span> liver, which contain Polgamma), and the results suggest that high sensitivity to phosphonoacetate is unique to POPs in Family-A DNA polymerases. Finally, we propose a model for the succession of organellar DNA polymerases. 
<h3>20825577</h3>OsRAN2, essential for mitosis, enhances cold tolerance in <span class="match annotated ">rice</span> by promoting export of intranuclear tubulin and maintaining cell division under cold stress. <br><br>With global climate change, abnormally low temperatures have affected the world's <span class="match annotated ">rice</span> production. Many genes have been shown to be essential for molecular improvement of <span class="match annotated ">rice</span> cold-tolerance traits. However, less is known about the molecular cellular mechanism of their response to cold stress. Here, we investigated OsRAN2 involved in regulation of cell division during cold stress in <span class="match annotated ">rice</span>. Expression of OsRAN2 was increased under cold treatment, but not during salt and drought stress. The mean root mitotic index was closely related to the expression level of OsRAN2. Knockdown transgenic <span class="match annotated ">rice</span> lines showed an aberrant organization of spindles during mitosis and stunted growth during development. Overexpression of OsRAN2 enhanced cold tolerance in <span class="match annotated ">rice</span>. The transgenic <span class="match annotated ">rice</span> overexpressing OsRAN2 showed maintained cell division, decreased proportion of cells with intranuclear tubulin and formation of a normal nuclear envelope under the cold condition. Our study suggests a mechanism for OsRAN2 in regulating cold resistance in <span class="match annotated ">rice</span> by maintaining cell division through promoting the normal export of intranuclear tubulin at the end of mitosis. This insight could help improve the cold-tolerance trait in <span class="match annotated ">rice</span>. 
<h3>20840506</h3>OsHMA3, a P1B-type of ATPase affects root-to-shoot cadmium translocation in <span class="match annotated ">rice</span> by mediating efflux into vacuoles. <br><br>* The cadmium (Cd) over-accumulating <span class="match annotated ">rice</span> (<span class="annotated ">Oryza sativa</span>) cv Cho-Ko-Koku was previously shown to have an enhanced rate of root-to-shoot Cd translocation. This trait is controlled by a single recessive allele located at qCdT7. * In this study, using positional cloning and transgenic strategies, heavy metal ATPase 3 (OsHMA3) was identified as the gene that controls root-to-shoot Cd translocation rates. The subcellular localization and Cd-transporting activity of the gene products were also investigated. * The allele of OsHMA3 that confers high root-to-shoot Cd translocation rates (OsHMA3mc) encodes a defective P(1B) -ATPase transporter. OsHMA3 fused to green fluorescent protein was localized to vacuolar membranes in plants and <span class="LINNAEUS_only ">yeast</span>. An OsHMA3 transgene complemented Cd sensitivity in a <span class="LINNAEUS_only ">yeast</span> mutant that lacks the ability to transport Cd into vacuoles. By contrast, OsHMA3mc did not complement the Cd sensitivity of this <span class="LINNAEUS_only ">yeast</span> mutant, indicating that the OsHMA3mc transport function was lost. * We propose that the root cell cytoplasm of Cd-overaccumulating <span class="match annotated ">rice</span> plants has more Cd available for loading into the xylem as a result of the lack of OsHMA3-mediated transportation of Cd to the vacuoles. This defect results in Cd translocation to the shoots in higher concentrations. These data demonstrate the importance of vacuolar sequestration for Cd accumulation in <span class="match annotated ">rice</span>. 
<h3>20840510</h3>Spatial distribution of arsenic and temporal variation of its concentration in <span class="match annotated ">rice</span>. <br><br>* In order to gain insights into the transport and distribution of arsenic (As) in intact <span class="match annotated ">rice</span> (<span class="annotated ">Oryza sativa</span>) plants and its unloading into the <span class="match annotated ">rice</span> grain, we investigated the spatial distribution of As and the temporal variation of As concentration in whole <span class="match annotated ">rice</span> plants at different growth stages. To the best of our knowledge, this is the first time that such a study has been performed. * Inductively coupled plasma mass spectroscopy (ICP-MS) and high-performance liquid chromatography (HPLC)-ICP-MS were used to analyze total As concentration and speciation. Moreover, synchrotron-based X-ray fluorescence (SXRF) was used to investigate in situ As distribution in the leaf, internode, node and grain. * Total As concentrations of vegetative tissues increased during the 2 wk after flowering. The concentration of dimethylarsinic acid (DMA) in the caryopsis decreased progressively with its development, whereas inorganic As concentration remained stable. The ratios of As content between neighboring leaves or between neighboring internodes were c. 0.6. SXRF revealed As accumulation in the center of the caryopsis during its early development and then in the ovular vascular trace. * These results indicate that there are different controls on the unloading of inorganic As and DMA; the latter accumulated mainly in the caryopsis before flowering, whereas inorganic As was mainly transported into the caryopsis during grain filling. Moreover, nodes appeared to serve as a check-point in As distribution in <span class="match annotated ">rice</span> shoots. 
<h3>20932726</h3><span class="annotated ">Oramoeba fumarolia</span> gen. nov., sp. nov., a new marine heterolobosean amoeboflagellate growing at 54 ^0C. <br><br>An amoeba strain was isolated from marine sediment taken from the beach near a fumarole in Italy. The trophozoites of this new marine species transforms into flagellates with variable numbers of flagella, from 2 to 10. The strain forms round to oval cysts. This thermophilic amoeboflagellate grows at temperatures up to 54^0C. Molecular phylogenetic analysis of the small subunit ribosomal DNA (SSU rDNA) places the amoeboflagellate in the Heterolobosea. The closest relatives are <span class="annotated ">Stachyamoeba sp. ATCC50324</span>, a strain isolated from an ocean sample, and <span class="annotated ">Vrihiamoeba italica</span>, a recent isolate from a <span class="match annotated ">rice</span> field. Like some other heterolobosean species, this new isolate has a group I intron in the SSU rDNA. Because of the unique place in the molecular phylogenetic tree, and because there is no species found in the literature with similar morphological and physiological characteristics, this isolate is considered to be a new genus and a new species, <span class="annotated ">Oramoeba fumarolia</span> gen. nov., sp. nov. 
<h3>21039565</h3>Aerenchyma formation in the <span class="match annotated ">rice</span> stem and its promotion by H2O2.<br><br>* Gas spaces (aerenchyma) form as an adaptation to submergence to facilitate gas exchange. In <span class="match annotated ">rice</span> (<span class="annotated ">Oryza sativa</span>), aerenchyma develop by cell death and lysis, which are poorly understood at the cellular level. * Aerenchyma formation was studied in <span class="match annotated ">rice</span> stems by light microscopy. It was analyzed in response to submergence, ethylene and hydrogen peroxide (H(2)O(2)) treatment, and in the MT2b::Tos17 mutant. O(2)*(-) was detected with nitroblue tetrazolium and an epinephrine assay. H(2)O(2) was detected with 3,3'-diaminobenzidine. * Aerenchyma develop constitutively in all internodes of the deep-water <span class="match annotated ">rice</span> variety Pin Gaew 56, but are absent from the nodes. Constitutive aerenchyma formation was also observed in two lowland <span class="match annotated ">rice</span> varieties, albeit to a lesser degree. <span class="unannotated ">A</span> larger number of aerenchyma are present in older internodes, and at the top of each internode, revealing developmental gradients. Submergence or treatment with the ethylene-releasing compound ethephon promoted aerenchyma formation in all genotypes analyzed. Pre-aerenchymal cells contain less starch, no chloroplasts, thinner cell walls and produce elevated levels of O(2)*(-) and H(2)O(2) compared with other parenchymal cells. Ethephon promotes O(2)*(-) formation and H(2)O(2) promotes aerenchyma formation in a dose-dependent manner. Further-more, genetic downregulation of the H(2)O(2) scavenger MT2b enhances aerenchyma formation. * Aerenchyma formation is mediated by reactive oxygen species. 
<h3>21039568</h3>Identification of a novel mitochondrial protein, short postembryonic roots 1 (SPR1), involved in root development and iron homeostasis in <span class="annotated ">Oryza sativa</span>. <br><br>* <span class="unannotated ">A</span> <span class="match annotated ">rice</span> mutant, <span class="annotated ">Oryza sativa short postembryonic roots 1</span> (<span class="annotated ">Osspr1</span>), has been characterized. It has short postembryonic roots, including adventitious and lateral roots, and a lower iron content in its leaves. * OsSPR1 was identified by map-based cloning. It encodes a novel mitochondrial protein with the Armadillo-like repeat domain. * <span class="annotated ">Osspr1</span> mutants exhibited decreased root cell elongation. The iron content of the mutant shoots was significantly altered compared with that of wild-type shoots. <span class="unannotated ">A</span> similar pattern of alteration of manganese and zinc concentrations in shoots was also observed. Complementation of the mutant confirmed that OsSPR1 is involved in post-embryonic root elongation and iron homeostasis in <span class="match annotated ">rice</span>. OsSPR1 was found to be ubiquitously expressed in various tissues throughout the plant. The transcript abundance of various genes involved in iron uptake and signaling via both strategies I and II was similar in roots of wild-type and mutant plants, but was higher in the leaves of mutant plants. * Thus, a novel mitochondrial protein that is involved in root elongation and plays a role in metal ion homeostasis has been identified. 
<h3>21668550</h3>Compatible interaction with its <span class="match annotated ">rice</span> host leads to enhanced expression of the gamma subunit of oligosaccharyl transferase in the <span class="annotated ">Asian rice gall midge</span>, <span class="annotated ">Orseolia oryzae</span>. <br><br>The <span class="annotated ">Asian rice gall midge</span>, <span class="annotated ">Orseolia oryzae</span>, is a fast evolving, damaging pest of <span class="match annotated ">rice</span>. Understanding the underlying molecular mechanism of interaction between the <span class="annotated ">gall midge</span> and <span class="match annotated ">rice</span> will help in devising strategies to control and manage the pest. The present study aims to identify <span class="match annotated ">rice</span>-responsive genes in the <span class="unannotated ">gall midge</span> that aid pest survival. The abundance of transcripts coding for enzymes related to glycosylation, in a cDNA library prepared from maggots of the <span class="annotated ">rice gall midge</span> feeding on susceptible hosts, indicated their probable involvement in the <span class="annotated ">gall midge</span>-<span class="match annotated ">rice</span> interaction. Hence, a full-length transcript for a gamma subunit of the oligosaccharyl transferase gene (OoOST) from the <span class="unannotated ">gall midge</span> was cloned and characterized. It has 72% similarity to its orthologue cloned from <span class="annotated ">Aedes aegypti</span>. Tissue-specific analysis of the expression of OoOST revealed an increase (&gt; sevenfold) in the transcripts of the gene in the salivary glands of maggots in susceptible plants when compared with the transcript level in the salivary glands of maggots feeding on resistant hosts. Using quantitative PCR, performed on different developmental stages of the maggots in two susceptible and two resistant hosts, we observed similar expression patterns (i.e. overexpression in the compatible interaction). These results indicate the involvement of OoOST in maggot survival and establishment in the susceptible host. In order to identify polymorphism in the gene, OoOST was cloned from three <span class="unannotated ">gall midge</span> biotypes GMB1, GMB4 and GMB4M. 
<h3>21929695</h3>Overexpression of a cytochrome P450 monooxygenase, CYP6ER1, is associated with resistance to imidacloprid in the <span class="annotated ">brown planthopper</span>, <span class="annotated ">Nilaparvata lugens</span>. <br><br>The <span class="annotated ">brown planthopper</span>, <span class="annotated ">Nilaparvata lugens</span>, is an economically significant pest of <span class="match annotated ">rice</span> throughout Asia and has evolved resistance to many insecticides including the neonicotinoid imidacloprid. The resistance of field populations of <span class="annotated ">N. lugens</span> to imidacloprid has been attributed to enhanced detoxification by cytochrome P450 monooxygenases (P450s), although, to date, the causative P450(s) has (have) not been identified. In the present study, biochemical assays using the model substrate 7-ethoxycoumarin showed enhanced P450 activity in several resistant <span class="annotated ">N. lugens</span> field strains when compared with a susceptible reference strain. Thirty three cDNA sequences encoding tentative unique P450s were identified from two recent sequencing projects and by degenerate PCR. The mRNA expression level of 32 of these was examined in susceptible, moderately resistant and highly resistant <span class="annotated ">N. lugens</span> strains using quantitative real-time PCR. <span class="unannotated ">A</span> single P450 gene (CYP6ER1) was highly overexpressed in all resistant strains (up to 40-fold) and the level of expression observed in the different <span class="annotated ">N. lugens</span> strains was significantly correlated with the resistance phenotype. These results provide strong evidence for a role of CYP6ER1 in the resistance of <span class="annotated ">N. lugens</span> to imidacloprid. 
<h3>20719251</h3>Diversification and evolution of the avirulence gene AVR-Pita1 in field isolates of <span class="annotated ">Magnaporthe oryzae</span>. <br><br><span class="annotated ">Rice</span> blast disease is the single most destructive plant disease that threatens stable <span class="match annotated ">rice</span> production worldwide. Race-specific resistance to the <span class="annotated ">rice blast pathogen</span> has not been durable and the mechanism by which the resistance is overcome remains largely unknown. Here we report the molecular mechanisms of diversification and the instability of the avirulence gene AVR-Pita1 in field strains of <span class="annotated ">Magnaporthe oryzae</span> interacting with the host resistance gene Pi-ta and triggering race-specific resistance. Two-base-pair insertions resulting in frame-shift mutations and partial and complete deletions of AVR-Pita1 were identified in virulent isolates. Moreover, a total of 38 AVR-Pita1 haplotypes encoding 27 AVR-Pita1 variants were identified among 151 avirulent isolates. Most DNA sequence variation was found to occur in the exon region resulting in amino acid substitution. These findings demonstrate that AVR-Pita1 is under positive selection and mutations of AVR-Pita1 are responsible for defeating race-specific resistance in nature. 
<h3>20946420</h3>Association between nonsynonymous mutations of starch synthase IIa and starch quality in <span class="match annotated ">rice</span> (<span class="annotated ">Oryza sativa</span>). <br><br>Starch quality is one of the most important agronomic traits in Asian <span class="match annotated ">rice</span>, <span class="annotated ">Oryza sativa</span>. Starch synthase IIa (SsIIa) is a major candidate gene for starch quality variation. Within SsIIa, three nonsynonymous mutations in exon 8 have been shown to affect enzyme activity when expressed in <span class="annotated ">Escherichia coli</span>. To search for the variation in SsIIa that is responsible for starch quality variation in <span class="match annotated ">rice</span>, we sequenced the SsIIa exon 8 region and measured starch quality as starch disintegration in alkali for 289 accessions of cultivated <span class="match annotated ">rice</span> and 57 accessions of its wild ancestor, <span class="annotated ">Oryza rufipogon</span>. <span class="unannotated ">A</span> general linear model and nested clade analysis were used to identify the associations between the three nonsynonymous single nucleotide polymorphisms (SNPs) and starch quality. Among the three nonsynonymous SNPs, we found strong evidence of association at one nucleotide site ('SNP 3'), corresponding to a Leu/Phe replacement at codon 781. <span class="unannotated ">A</span> second SNP, corresponding to a Val/Met replacement at codon 737, could potentially show an association with increased sample sizes. Variation in SsIIa enzyme activity is associated with the cohesiveness of <span class="match annotated ">rice</span> grains when cooked, and our findings are consistent with selection for more cohesive grains during the domestication of tropical <span class="annotated ">japonica rice</span>. 
<h3>21277379</h3>Independent duplications of alpha-amylase in different strains of <span class="annotated ">Aspergillus oryzae</span>. <br><br><span class="annotated ">Aspergillus oryzae</span> is a filamentous fungus that has arisen through the ancient domestication of <span class="annotated ">Aspergillus flavus</span> for making traditional oriental foods and beverages. In the many centuries <span class="annotated ">A. oryzae</span> has been used for fermenting the starch in <span class="match annotated ">rice</span> to simple sugars, it has undergone selection for increased secretion of starch-degrading enzymes. In particular, all <span class="annotated ">A. oryzae</span> strains investigated thus far have two or more copies of a gene encoding alpha-amylase, whereas <span class="annotated ">A. flavus</span> has only one. Here we investigate the duplications leading to these copies in three <span class="annotated ">A. oryzae</span> strains. We find evidence of at least three separate duplications of alpha-amylase, an example of parallel evolution in a micro-organism under artificial selection. At least two of these duplications appear to be associated with activity of transposable elements of the Tc1/mariner class. Both involve a 9.1 kb element that terminates in inverted repeats, encodes a putative transposase and another putative protein of unknown function, and contains an unusual arrangement of four short internal imperfect repeats. Although &quot;unusual Mariners&quot; of this size have previously been identified in <span class="annotated ">A. oryzae</span>, <span class="annotated ">Aspergillus fumigatus</span> and <span class="annotated ">Aspergillus nidulans</span>, this is the first evidence we know of that at least some of them are active in modern times and that their activity can contribute to beneficial genetic changes. 
<h3>21600998</h3>The cell cycle gene MoCDC15 regulates hyphal growth, asexual development and plant infection in the <span class="match annotated ">rice</span> blast pathogen <span class="annotated ">Magnaporthe oryzae</span>. <br><br><span class="annotated ">Rice</span> blast, caused by the pathogen <span class="annotated ">Magnaporthe oryzae</span>, is a serious hindrance to <span class="match annotated ">rice</span> production and has emerged as an important model for the characterization of molecular mechanisms relevant to pathogenic development in plants. Similar to other pathogenic fungi, conidiation plays a central role in initiation of <span class="annotated ">M.oryzae</span> infection and spread over a large area. However, relatively little is known regarding the molecular mechanisms that underlie conidiation in <span class="annotated ">M. oryzae</span>. To better characterize these mechanisms, we identified a conidiation-defective mutant, <span class="annotated ">ATMT0225B6</span> (<span class="annotated ">MoCDC15(T-DNA)</span>), in which a T-DNA insertion disrupted a gene that encodes a homolog of fission <span class="LINNAEUS_only ">yeast</span> cdc15, and generated a second strain containing a disruption in the same allele (DeltaMoCDC15(T-DNA)). The cdc15 gene has been shown to act as a coordinator of the cell cycle in <span class="LINNAEUS_only ">yeast</span>. Functional analysis of the <span class="unannotated ">MoCDC15(T-DNA)</span> and DeltaMoCDC15(T-DNA) mutants revealed that MoCDC15 is required for conidiation, preinfection development and pathogenicity in <span class="annotated ">M. oryzae</span>. Conidia from these mutants were viable, but failed to adhere to hydrophobic surface, a crucial step required for subsequent pathogenic development. All phenotypic defects observed in mutants were rescued in a strain complemented with <span class="annotated ">wild type MoCDC15</span>. Together, these data indicate that MoCDC15 functions as a coordinator of several biological processes important for pathogenic development in <span class="annotated ">M. oryzae</span>. 
<h3>20807375</h3>OsEDR1 negatively regulates <span class="match annotated ">rice</span> bacterial resistance via activation of ethylene biosynthesis. <br><br><span class="annotated ">Rice</span> OsEDR1 is a sequence ortholog of <span class="LINNAEUS_only ">Arabidopsis</span> EDR1. However, its molecular function is unknown. We show here that OsEDR1-suppressing/knockout (KO) plants, which developed spontaneous lesions on the leaves, have enhanced resistance to <span class="annotated ">Xanthomonas oryzae pv. oryzae</span> (<span class="annotated ">Xoo</span>) causing bacterial blight disease. This resistance was associated with increased accumulation of salicylic acid (SA) and jasmonic acid (JA), induced expression of SA- and JA-related genes and suppressed accumulation of 1-aminocyclopropane-1-carboxylic acid (ACC), the direct precursor of ethylene, and expression of ethylene-related genes. OsEDR1-KO plants also showed suppressed production of ethylene. Knockout of OsEDR1 suppressed the ACC synthase (ACS) gene family, which encodes the rate-limiting enzymes of ethylene biosynthesis by catalysing the formation of ACC. The lesion phenotype and enhanced bacterial resistance of the OsEDR1-KO plants was partly complemented by the treatment with ACC. ACC treatment was associated with decreased SA and JA biosynthesis in OsEDR1-KO plants. In contrast, aminoethoxyvinylglycine, the inhibitor of ethylene biosynthesis, promoted expression of SA and JA synthesis-related genes in OsEDR1-KO plants. These results suggest that ethylene is a negative signalling molecule in <span class="match annotated ">rice</span> bacterial resistance. In the <span class="match annotated ">rice</span>-<span class="annotated ">Xoo</span> interaction, OsEDR1 transcriptionally promotes the synthesis of ethylene that, in turn, suppresses SA- and JA-associated defence signalling. 
<h3>21406243</h3>Potassium and sodium uptake systems in fungi. The transporter diversity of <span class="annotated ">Magnaporthe oryzae</span>. <br><br>In this study, we report an inventory of the K(+) uptake systems in 62 fungal species for which the complete genome sequences are available. This inventory reveals that three types of K(+) uptake systems, TRK and HAK transporters and ACU ATPases, are widely present in several combinations across fungal species. PAT ATPases are less frequently present and are exceptional in Ascomycota. The genome of <span class="annotated ">Magnaporthe oryzae</span> contains four TRK, one HAK, and two ACU genes. The study of the expression of these genes at high K(+), K(+) starvation, and in infected <span class="match annotated ">rice</span> leaves revealed that the expression of four genes, ACU1, ACU2, HAK1, and TRK1 is much lower than that of TRK2, TRK3, and TRK4, except under K(+) starvation. The two ACU ATPases were cloned and functionally identified as high-affinity K(+) or Na(+) uptake systems. These two ATPases endow <span class="annotated ">Saccharomyces cerevisiae</span> with the capacity to grow for several generations in low Na(+) concentrations when K(+) was absent, which produces a dramatic increase of cellular Na(+)/K(+) ratio. 
</body></html>