
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>Salmonella enterica</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>21037010</h3>Inactivation of the RluD pseudouridine synthase has minimal effects on growth and ribosome function in wild-type <span class="annotated ">Escherichia coli</span> and <span class="match annotated ">Salmonella enterica</span>. <br><br>The <span class="annotated ">Escherichia coli</span> rluD gene encodes a pseudouridine synthase responsible for the pseudouridine (Psi) modifications at positions 1911, 1915, and 1917 in helix 69 of 23S rRNA. It has been reported that deletion of rluD in <span class="annotated ">K-12</span> strains of <span class="annotated ">E. coli</span> is associated with extremely slow growth, increased readthrough of stop codons, and defects in 50S ribosomal subunit assembly and 30S-50S subunit association. Suppressor mutations in the prfB and prfC genes encoding release factor 2 (RF2) and RF3 that restore the wild type-growth rate and also correct the ribosomal defects have now been isolated. These suppressors link helix 69 Psi residues with the termination phase of protein synthesis. However, further genetic analysis reported here also reveals that the slow growth and other defects associated with inactivation of rluD in <span class="annotated ">E. coli K-12</span> strains are due to a defective RF2 protein, with a threonine at position 246, which is present in all <span class="annotated ">K-12</span> strains. This is in contrast to the more typical alanine found at this position in most bacterial RF2s, including those of other <span class="annotated ">E. coli</span> strains. Inactivation of rluD in <span class="annotated ">E. coli</span> strains containing the prfB allele from <span class="annotated ">E. coli B</span> or in <span class="match annotated ">Salmonella enterica</span>, both carrying an RF2 with Ala246, has negligible effects on growth, termination, or ribosome function. The results indicate that, in contrast to those in wild bacteria, termination functions in <span class="annotated ">E. coli K-12</span> strains carrying a partially defective RF2 protein are especially susceptible to perturbation of ribosome-RF interactions, such as that caused by loss of h69 Psi modifications. 
<h3>21148731</h3>FAD binding by ApbE protein from <span class="match annotated ">Salmonella enterica</span>: a new class of FAD-binding proteins. <br><br>The periplasmic protein ApbE was identified through the analysis of several mutants defective in thiamine biosynthesis and was implicated as having a role in iron-sulfur cluster biosynthesis or repair. While mutations in apbE cause decreased activity of several iron-sulfur enzymes in vivo, the specific role of ApbE remains unknown. Members of the AbpE family include NosX and RnfF, which have been implicated in oxidation-reduction associated with nitrous oxide and nitrogen metabolism, respectively. In this work, we show that ApbE binds one FAD molecule per monomeric unit. The structure of ApbE in the presence of bound FAD reveals a new FAD-binding motif. Protein variants that are nonfunctional in vivo were generated by random and targeted mutagenesis. Each variant was substituted in the environment of the FAD and analyzed for FAD binding after reconstitution. The variant that altered a key tyrosine residue involved in FAD binding prevented reconstitution of the protein. 
</body></html>