
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.linneaus_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>viral</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="linneaus_only">LINNEAUS only</span><br><hr><h3>20863853</h3>Detection and quantitation of infectious <span class="annotated ">human adenoviruses</span> and <span class="annotated ">JC polyomaviruses</span> in water by immunofluorescence assay. <br><br>Human adenoviruses (HAdV) and <span class="unannotated ">JC polyomaviruses</span> (JCPyV) have been proposed as markers of fecal/urine contamination of <span class="unannotated ">human</span> origin. An indirect immunofluorescence assay has been developed to quantify infectious <span class="annotated ">human adenoviruses</span> types 2 and 41 and <span class="annotated ">JC polyomaviruses</span> strain Mad-4 in water samples. The immunofluorescence assay was compared with other quantitative techniques used commonly such as plaque assay, tissue culture infectious dose-50 and quantitative PCR (qPCR). The immunofluorescence assays showed to be specific for the detection of infectious viruses, obtaining negative results when UV or heat-inactivated viruses were analyzed. The assays required less time and showed higher sensitivity for the detection of infectious <span class="match unannotated ">viral</span> particles than other cell culture techniques (1 log(10) more) evaluated. River water samples spiked previously with <span class="annotated ">human</span> adenoviruses and raw sewage samples were also analyzed using the proposed immunofluorescence assay as well as by qPCR. The results show quantitations with 2 log(10) reduction in the numbers of infectious viruses compared with the number of genome copies detected by qPCR. The immunofluorescence assay developed is fast, sensitive, specific, and a standardizable technique for the quantitation and detection of infectious viruses in water samples. 
<h3>20863856</h3>Expression of the C-terminal ORF2 protein of <span class="annotated ">duck astrovirus</span> for application in a serological test. <br><br><span class="annotated ">Duck astrovirus</span> (<span class="annotated ">DAstV</span>) is an important pathogen causing duck <span class="match unannotated ">viral</span> hepatitis (DVH), a highly contagious and fatal disease in young ducklings. To provide an antigen for a diagnostic serum test, the C-terminus of <span class="unannotated ">DAstV</span> ORF2 protein was expressed in <span class="annotated ">Escherichia coli</span>. Four positive and 30 negative sera were used to validate the purified ORF2 protein by developing an indirect enzyme-linked immunosorbent assay (ELISA). No cross-reactions were found against other duck pathogens, including <span class="annotated ">duck hepatitis A virus</span>, <span class="annotated ">duck plague herpesvirus</span>, <span class="annotated ">duck reovirus</span>, <span class="annotated ">Newcastle disease virus</span>, and <span class="annotated ">Riemerella anatipestifer 12/19</span> (63.2%) and <span class="annotated ">26/51</span> (51%) sera samples from two flocks of ducks that survived <span class="annotated ">DAstV</span> infections in commercial duck farms were positive for <span class="annotated ">DAstV</span> by this method, respectively. Interestingly, <span class="annotated ">DAstV</span>-specific antibodies were also detected in 12 (28.6%) of 42 sera samples from a different flock without DVH, indicating a wide distribution of subclinical infections caused by <span class="annotated ">DAstV</span>. 
<h3>20962076</h3>Neutralizing antibody-resistant <span class="annotated ">hepatitis C virus</span> cell-to-cell transmission.<br><br>Hepatitis C <span class="unannotated ">virus</span> (<span class="annotated ">HCV</span>) can initiate infection by cell-free particle and cell-cell contact-dependent transmission. In this study we use a novel infectious coculture system to examine these alternative modes of infection. Cell-to-cell transmission is relatively resistant to <span class="annotated ">anti-HCV</span> glycoprotein monoclonal antibodies and polyclonal immunoglobulin isolated from infected individuals, providing an effective strategy for escaping host humoral immune responses. Chimeric viruses expressing the structural proteins representing the seven major <span class="annotated ">HCV</span> genotypes demonstrate neutralizing antibody-resistant cell-to-cell transmission. <span class="annotated ">HCV</span> entry is a multistep process involving numerous receptors. In this study we demonstrate that, in contrast to earlier reports, CD81 and the tight-junction components claudin-1 and occludin are all essential for both cell-free and cell-to-cell <span class="match unannotated ">viral</span> transmission. However, scavenger receptor BI (SR-BI) has a more prominent role in cell-to-cell transmission of the <span class="unannotated ">virus</span>, with SR-BI-specific antibodies and small-molecule inhibitors showing preferential inhibition of this infection route. These observations highlight the importance of targeting host cell receptors, in particular SR-BI, to control <span class="match unannotated ">viral</span> infection and spread in the liver. 
<h3>20962079</h3>Blood myeloid dendritic cells from <span class="annotated ">HIV-1</span>-infected individuals display a proapoptotic profile characterized by decreased Bcl-2 levels and by caspase-3+ frequencies that are associated with levels of plasma viremia and T cell activation in an exploratory study. <br><br>Reduced frequencies of myeloid and plasmacytoid dendritic cell (DC) subsets (mDCs and pDCs, respectively) have been observed in the peripheral blood of <span class="annotated ">HIV-1</span>-infected individuals throughout the course of disease. Accumulation of DCs in lymph nodes (LNs) may partly account for the decreased numbers observed in blood, but increased DC death may also be a contributing factor. We used multiparameter flow cytometry to evaluate pro- and antiapoptotic markers in blood mDCs and pDCs from untreated <span class="annotated ">HIV-1</span>-infected donors, from a subset of infected donors before and after receiving antiretroviral therapy (ART), and from uninfected control donors. Blood mDCs, but not pDCs, from untreated <span class="annotated ">HIV-1</span>-infected donors expressed lower levels of antiapoptotic Bcl-2 than DCs from uninfected donors. <span class="unannotated ">A</span> subset of <span class="annotated ">HIV-1</span>-infected donors had elevated frequencies of proapoptotic caspase-3(+) blood mDCs, and positive correlations were observed between caspase-3(+) mDC frequencies and plasma <span class="match unannotated ">viral</span> load and CD8(+) T-cell activation levels. Caspase-3(+) mDC frequencies, but not mDC Bcl-2 expression, were reduced with <span class="match unannotated ">viral</span> suppression on ART. Apoptosis markers on DCs in blood and LN samples from a cohort of untreated, <span class="annotated ">HIV-1</span>-infected donors with chronic disease were also evaluated. LN mDCs displayed higher levels of Bcl-2 and lower caspase-3(+) frequencies than did matched blood mDCs. Conversely, LN pDCs expressed lower Bcl-2 levels than their blood counterparts. In summary, blood mDCs from untreated <span class="annotated ">HIV-1</span>-infected subjects displayed a proapoptotic profile that was partially reversed with <span class="match unannotated ">viral</span> suppression, suggesting that DC death may be a factor contributing to blood DC depletion in the setting of chronic, untreated <span class="annotated ">HIV</span> disease. 
<h3>20962080</h3>Identification of binary interactions between <span class="annotated ">human cytomegalovirus</span> virion proteins. <br><br><span class="annotated ">Human cytomegalovirus</span> (<span class="annotated ">HCMV</span>) virions are composed of a DNA-containing nucleocapsid surrounded by a tegument layer and host-derived lipid envelope studded with virally encoded glycoproteins. These complex virions are estimated to be composed of more than 50 <span class="match unannotated ">viral</span> proteins. Assembly of <span class="annotated ">HCMV</span> virions is poorly understood, especially with respect to acquisition of the tegument; however, it is thought to involve the stepwise addition of virion components through protein-protein interactions. We sought to identify interactions among <span class="annotated ">HCMV</span> virion proteins using <span class="annotated ">yeast</span> two-hybrid analysis. Using 33 known capsid and tegument proteins, we tested 1,089 pairwise combinations for binary interaction in the two-hybrid assay. We identified 24 interactions among <span class="annotated ">HCMV</span> virion proteins, including 13 novel interactions among tegument proteins and one novel interaction between capsid proteins. Several of these novel interactions were confirmed by coimmunoprecipitation of protein complexes from transfected cells. In addition, we demonstrate three of these interactions in the context of <span class="annotated ">HCMV</span> infection. This study reveals several new protein-protein interactions among <span class="annotated ">HCMV</span> tegument proteins, some of which are likely important for <span class="annotated ">HCMV</span> replication and pathogenesis. 
<h3>20962086</h3><span class="annotated ">Poliovirus</span>-mediated disruption of cytoplasmic processing bodies.<br><br>Metazoan cells form cytoplasmic mRNA granules such as stress granules (SG) and processing bodies (P bodies) that are proposed to be sites of aggregated, translationally silenced mRNAs and mRNA degradation. <span class="annotated ">Poliovirus</span> (<span class="annotated ">PV</span>) is a plus-strand RNA <span class="unannotated ">virus</span> containing a genome that is a functional mRNA; thus, we investigated if <span class="annotated ">PV</span> antagonizes the processes that lead to formation of these structures. We have previously shown that <span class="annotated ">PV</span> infection inhibits the ability of cells to form stress granules by cleaving RasGAP-SH3-binding protein (G3BP). Here, we show that P bodies are also disrupted during <span class="annotated ">PV</span> infection in cells by 4 h postinfection. The disruption of P bodies is more rapid and more complete than disruption of stress granules. The kinetics of P body disruption correlated with production of <span class="match unannotated ">viral</span> proteinases and required substantial <span class="match unannotated ">viral</span> gene product expression. The organizing mechanism that forms P body foci in cells is unknown; however, potential scaffolding, aggregating, or other regulatory proteins found in P bodies were investigated for degradation. Two factors involved in 5'-end mRNA decapping and degradation, Xrn1 and Dcp1a, and the 3' deadenylase complex component Pan3 underwent accelerated degradation during infection, and Dcp1a may be a direct substrate of <span class="annotated ">PV</span> 3C proteinase. Several other key factors proposed to be essential for P body formation, GW182, Edc3, and Edc4, were unaffected by <span class="annotated ">poliovirus</span> infection. Since deadenylation has been reported to be required for P body formation, <span class="match unannotated ">viral</span> inhibition of deadenylation, through Pan3 degradation, is a potential mechanism of P body disruption. 
<h3>20962087</h3>The magnitude of local immunity in the lungs of <span class="annotated ">mice</span> induced by live attenuated <span class="unannotated ">influenza</span> vaccines is determined by local <span class="match unannotated ">viral</span> replication and induction of cytokines. <br><br>While live attenuated <span class="unannotated ">influenza</span> vaccines (LAIVs) have been shown to be efficacious and have been licensed for <span class="annotated ">human</span> use, the surface glycoproteins hemagglutinin (HA) and neuraminidase (NA) have to be updated for optimal protective efficacy. Little is known about the effect of different HA and NA proteins on the immunogenicity of LAIVs developed using the same backbone. <span class="unannotated ">A</span> panel of LAIVs that share the internal protein genes, with unique HA and NA gene segments from different <span class="unannotated ">influenza</span> subtypes, was rescued by reverse genetics, and a comparative study of immune responses induced by these vaccines was conducted in <span class="annotated ">mice</span>. The results suggest that the magnitude of lung immunity, including pulmonary IgA antibody and memory CD8(+) T lymphocytes, induced by the vaccines depends on the replication efficiency of the LAIVs, as well as the induction of cytokines/chemokines in the lungs. However, these factors are not important in determining systemic immunity such as serum antibody titers and memory CD8(+) T cells in the spleen. <span class="unannotated ">A</span> qualitative analysis of immune responses induced by a single dose of an <span class="annotated ">H5N1</span> LAIV revealed that the vaccine induced robust systemic and mucosal immunity in <span class="annotated ">mice</span>. In addition, antibodies and memory lymphocytes established in the lungs following vaccination were required for protection against lethal challenge with homologous and heterologous <span class="annotated ">H5N1 viruses</span>. Our results highlight the different requirements for inducing systemic and lung immunity that can be explored for the development of pulmonary immunity for protection against respiratory pathogens. 
<h3>20962090</h3><span class="annotated ">Marek's disease virus type 1</span> microRNA miR-M3 suppresses cisplatin-induced apoptosis by targeting Smad2 of the transforming growth factor beta signal pathway. <br><br>Viruses cause about 15% of the cancers that are still the leading causes of <span class="annotated ">human</span> mortality. The discovery of <span class="match unannotated ">viral</span> oncogenes has enhanced our understanding of <span class="match unannotated ">viral</span> oncogenesis. However, the underlying molecular mechanisms of virus-induced cancers are complex and require further investigation. The present study has attempted to investigate the effects of the microRNAs (miRNAs) encoded by <span class="annotated ">Marek's disease virus 1</span> (<span class="annotated ">MDV1</span>), a <span class="annotated ">chicken</span> herpesvirus causing acute T-cell lymphomas and solid visceral tumors in <span class="annotated ">chickens</span>, on anti-cancer drug-induced apoptosis and identify the targets of the miRNAs. The results showed that of the total 14 miRNAs encoded by <span class="annotated ">MDV1</span>, <span class="annotated ">MDV1</span>-miR-M3 significantly promoted cell survival under treatment with cisplatin, a widely used chemotherapy drug. <span class="annotated ">MDV1</span>-miR-M3 suppressed cisplatin-induced apoptosis by directly downregulating expression at the protein but not the mRNA level of Smad2, a critical component in the transforming growth factor Beta signal pathway. Our data suggest that latent/oncogenic viruses may encode miRNAs to directly target cellular factors involved in antiviral processes including apoptosis, thus proactively creating a cellular environment beneficial to <span class="match unannotated ">viral</span> latency and oncogenesis. Furthermore, the knowledge of the apoptosis resistance conferred by <span class="match unannotated ">viral</span> miRNAs has great practical implications for improving the efficacy of chemotherapies for treating cancers, especially those induced by oncogenic viruses. 
<h3>20962091</h3>Macaque long-term nonprogressors resist superinfection with multiple CD8+ T cell escape variants of <span class="annotated ">simian immunodeficiency virus</span>. <br><br><span class="annotated ">Human immunodeficiency virus</span> (<span class="annotated ">HIV</span>)-positive individuals can be superinfected with different <span class="unannotated ">virus</span> strains. Individuals who control an initial <span class="annotated ">HIV</span> infection are therefore still at risk for subsequent infection with divergent viruses, but the barriers to such superinfection remain unclear. Here we tested long-term nonprogressors' (LTNPs') susceptibility to superinfection using <span class="annotated ">Indian rhesus macaques</span> that express the major histocompatibility complex <span class="unannotated ">class I</span> (MHC-I) allele Mamu-B 17, which is associated with control of the pathogenic AIDS <span class="annotated ">virus SIVmac239</span>. The Mamu-B 17-restricted CD8(+) T cell repertoire is focused almost entirely on 5 epitopes. We engineered a series of <span class="unannotated ">SIVmac239</span> variants bearing mutations in 3, 4, or all 5 of these epitopes and used them to serially challenge 2 Mamu-B 17-positive LTNPs. None of the escape variants caused breakthrough replication in LTNPs, although they readily infected Mamu-B 17-negative naive macaques. In vitro competing coculture assays and examination of <span class="match unannotated ">viral</span> evolution in hosts lacking Mamu-B 17 suggested that the mutant viruses had negligible defects in replicative fitness. Both LTNPs maintained robust immune responses, including <span class="annotated ">simian immunodeficiency virus</span> (<span class="annotated ">SIV</span>)-specific CD8(+) and CD4(+) T cells and neutralizing antibodies. Our results suggest that escape mutations in epitopes bound by &quot;protective&quot; MHC-I molecules may not be sufficient to establish superinfection in LTNPs. 
<h3>20962095</h3>Role of RNase MRP in <span class="match unannotated ">viral</span> RNA degradation and RNA recombination.<br><br>RNA degradation, together with RNA synthesis, controls the steady-state level of <span class="match unannotated ">viral</span> RNAs in infected cells. The endoribonucleolytic cleavage of <span class="match unannotated ">viral</span> RNA is important not only for <span class="match unannotated ">viral</span> RNA degradation but for RNA recombination as well, due to the participation of some RNA degradation products in the RNA recombination process. To identify host endoribonucleases involved in degradation of <span class="annotated ">Tomato bushy stunt virus</span> (<span class="annotated ">TBSV</span>) in a <span class="annotated ">Saccharomyces cerevisiae</span> model host, we tested eight known endoribonucleases. Here we report that downregulation of SNM1, encoding a component of the RNase MRP, and a temperature-sensitive mutation in the NME1 gene, coding for the RNA component of RNase MRP, lead to reduced production of the endoribonucleolytically cleaved <span class="annotated ">TBSV</span> RNA in <span class="annotated ">yeast</span>. We also show that the highly purified <span class="annotated ">yeast</span> RNase MRP cleaves the <span class="annotated ">TBSV</span> RNA in vitro, resulting in <span class="annotated ">TBSV</span> RNA degradation products similar in size to those observed in <span class="annotated ">yeast</span> cells. Knocking down the NME1 homolog in <span class="annotated ">Nicotiana benthamiana</span> also led to decreased production of the cleaved <span class="annotated ">TBSV</span> RNA, suggesting that in plants, RNase MRP is involved in <span class="annotated ">TBSV</span> RNA degradation. Altogether, this work suggests a role for the host endoribonuclease RNase MRP in <span class="match unannotated ">viral</span> RNA degradation and recombination. 
<h3>20980498</h3>Template recognition mechanisms by replicase proteins differ between bipartite positive-strand genomic RNAs of a plant <span class="unannotated ">virus</span>. <br><br>Recognition of RNA templates by <span class="match unannotated ">viral</span> replicase proteins is one of the key steps in the replication process of all RNA viruses. However, the mechanisms underlying this phenomenon, including primary RNA elements that are recognized by the <span class="match unannotated ">viral</span> replicase proteins, are not well understood. Here, we used aptamer pulldown assays with membrane fractionation and protein-RNA coimmunoprecipitation in a cell-free <span class="match unannotated ">viral</span> translation/replication system to investigate how <span class="match unannotated ">viral</span> replicase proteins recognize the bipartite genomic RNAs of the <span class="annotated ">Red clover necrotic mosaic virus</span> (<span class="annotated ">RCNMV</span>). <span class="annotated ">RCNMV</span> replicase proteins bound specifically to a Y-shaped RNA element (YRE) located in the 3' untranslated region (UTR) of RNA2, which also interacted with the 480-kDa replicase complexes that contain <span class="match unannotated ">viral</span> and host proteins. The replicase-YRE interaction recruited RNA2 to the membrane fraction. Conversely, RNA1 fragments failed to interact with the replicase proteins supplied in trans. The results of protein-RNA coimmunoprecipitation assays suggest that RNA1 interacts with the replicase proteins coupled with their translation. Thus, the initial template recognition mechanisms employed by the replicase differ between <span class="annotated ">RCNMV</span> bipartite genomic RNAs and RNA elements are primary determinants of the differential replication mechanism. 
<h3>20980499</h3><span class="annotated ">Poliovirus</span> RNA is released from the capsid near a twofold symmetry axis.<br><br>After recognizing and binding to its host cell, <span class="annotated ">poliovirus</span> (like other nonenveloped viruses) faces the challenge of translocating its genome across a cellular membrane and into the cytoplasm. To avoid entanglement with the capsid, the RNA must exit via a single site on the virion surface. However, the mechanism by which a single site is selected (from among 60 equivalents) is unknown; and until now, even its location on the virion surface has been controversial. To help to elucidate the mechanism of infection, we have used single-particle cryo-electron microscopy and tomography to reconstruct conformationally altered intermediates that are formed by the poliovirion at various stages of the <span class="annotated ">poliovirus</span> infection process. Recently, we reported icosahedrally symmetric structures for two forms of the end-state 80S empty capsid particle. Surprisingly, RNA was frequently visible near the capsid; and in a subset of the virions, RNA was seen on both the inside and outside of the capsid, caught in the act of exiting. To visualize RNA exiting, we have now determined asymmetric reconstructions from that subset, using both single-particle cryo-electron microscopy and cryo-electron tomographic methods, producing independent reconstructions at -50-A resolution. Contrary to predictions in the literature, the footprint of RNA on the capsid surface is located close to a <span class="match unannotated ">viral</span> 2-fold axis, covering a slot-shaped area of reduced density that is present in both of the symmetrized 80S reconstructions and which extends by about 20 <span class="unannotated ">A</span> away from the 2-fold axis toward each neighboring 5-fold axis. 
<h3>20980500</h3>Differential regulation of <span class="annotated ">human papillomavirus type 8</span> by interferon regulatory factors 3 and 7. <br><br>The genus <span class="annotated ">Beta human papillomavirus (HPV) type 8</span> is associated with nonmelanoma skin cancer in <span class="linneaus_only ">patients</span> with epidermodysplasia verruciformis, and evidence for its protumorigenic potential in the general population increases. To date, strategies to suppress <span class="annotated ">genus Beta HPV</span> infections are limited. Interferon regulatory factors IRF-3 and IRF-7 play key roles in the activation of the innate immune response to <span class="match unannotated ">viral</span> infections. In this study, we show for the first time that both IRF-3 and IRF-7 regulate transcription of a papillomavirus, but with opposing effects. IRF-7, expressed in the suprabasal layers of <span class="annotated ">human</span> epidermis, increased <span class="annotated ">HPV8</span> late promoter activity via direct binding to <span class="match unannotated ">viral</span> DNA. UV-B light-induced activation of the <span class="annotated ">HPV8</span> promoter involved IRF-7 as a downstream effector. In contrast, IRF-3, expressed in all layers of <span class="annotated ">human</span> epidermis, induced strong <span class="annotated ">HPV8</span> suppression in primary keratinocytes. IRF-3-mediated suppression prevailed over IRF-7-induced <span class="annotated ">HPV8</span> transcription. Unlike the E6 oncoprotein of the mucosal high-risk <span class="annotated ">HPV16</span>, the <span class="annotated ">HPV8</span> E6 protein did not bind to IRF-3 and only weakly antagonized its activity. Strong antiviral activity was also observed, when keratinocytes were treated with potent IRF-3 activators, poly(I:C) or RNA bearing 5' phosphates. In conclusion, we show that IRF-3 activation induces a state of cell-autonomous immunity against HPV in primary <span class="annotated ">human</span> keratinocytes. Our study suggests that local application of IRF-3-activating compounds might constitute an attractive novel therapeutic strategy against <span class="annotated ">HPV8</span>-associated diseases, particularly in epidermodysplasia verruciformis <span class="linneaus_only ">patients</span>. 
<h3>20980502</h3>Cytomegalovirus microRNA expression is tissue specific and is associated with persistence. <br><br>MicroRNAs (miRNAs) are a class of small noncoding RNAs involved in posttranscriptional regulation. miRNAs are utilized in organisms ranging from plants to higher mammals, and data have shown that DNA viruses also use this method for host and <span class="match unannotated ">viral</span> gene regulation. Here, we report the sequencing of the small RNAs in <span class="annotated ">rat cytomegalovirus</span> (<span class="annotated ">RCMV</span>)-infected fibroblasts and persistently infected salivary glands. We identified 24 unique miRNAs that mapped to hairpin structures found within the <span class="match unannotated ">viral</span> genome. While most miRNAs were detected in both samples, four were detected exclusively in the infected fibroblasts and two were specific for the infected salivary glands. The <span class="annotated ">RCMV</span> miRNAs are distributed across the <span class="match unannotated ">viral</span> genome on both the positive and negative strands, with clusters of miRNAs at a number of locations, including near <span class="match unannotated ">viral</span> genes r1 and r111. The <span class="annotated ">RCMV</span> miRNAs have a genomic positional orientation similar to that of the miRNAs described for <span class="annotated ">mouse cytomegalovirus</span>, but they do not share any substantial sequence conservation. Similar to other reported miRNAs, the <span class="annotated ">RCMV</span> miRNAs had considerable variation at their 3' and 5' ends. Interestingly, we found a number of specific examples of differential isoform usage between the fibroblast and salivary gland samples. We determined by real-time PCR that expression of the <span class="annotated ">RCMV</span> miRNA miR-r111.1-2 is highly expressed in the salivary glands and that miR-R87-1 is expressed in most tissues during the acute infection phase. Our study identified the miRNAs expressed by <span class="annotated ">RCMV</span> in vitro and in vivo and demonstrated that expression is tissue specific and associated with a stage of <span class="match unannotated ">viral</span> infection. 
<h3>20980507</h3>Successful vaccination strategies that protect aged <span class="annotated ">mice</span> from lethal challenge from <span class="unannotated ">influenza virus</span> and heterologous <span class="annotated ">severe acute respiratory syndrome coronavirus</span>. <br><br>Newly emerging viruses often circulate as a heterogeneous swarm in wild animal reservoirs prior to their emergence in <span class="annotated ">humans</span>, and their antigenic identities are often unknown until an outbreak situation. The newly emerging <span class="annotated ">severe acute respiratory syndrome coronavirus</span> (<span class="annotated ">SARS-CoV</span>) and reemerging <span class="unannotated ">influenza virus</span> cause disproportionate disease in the aged, who are also notoriously difficult to successfully vaccinate, likely due to immunosenescence. To protect against future emerging strains, vaccine platforms should induce broad cross-reactive immunity that is sufficient to protect from homologous and heterologous challenge in all ages. From initial studies, we hypothesized that attenuated <span class="annotated ">Venezuelan equine encephalitis virus</span> (<span class="annotated ">VEE</span>) replicon particle (VRP) vaccine glycoproteins mediated vaccine failure in the aged. We then compared the efficacies of vaccines bearing attenuated (VRP(3014)) or wild-type <span class="annotated ">VEE</span> glycoproteins (VRP(3000)) in young and aged <span class="annotated ">mice</span> within novel models of severe <span class="annotated ">SARS-CoV</span> pathogenesis. Aged animals receiving VRP(3000)-based vaccines were protected from <span class="annotated ">SARS-CoV</span> disease, while animals receiving the VRP(3014)-based vaccines were not. The superior protection for the aged observed with VRP(3000)-based vaccines was confirmed in a lethal <span class="unannotated ">influenza virus</span> challenge model. While the VRP(3000) vaccine's immune responses in the aged were sufficient to protect against lethal homologous and heterologous challenge, our data suggest that innate defects within the VRP(3014) platform mediate vaccine failure. Exploration into the mechanism(s) of successful vaccination in the immunosenescent should aid in the development of successful vaccine strategies for other <span class="match unannotated ">viral</span> diseases disproportionately affecting the elderly, like <span class="annotated ">West Nile virus</span>, <span class="unannotated ">influenza virus</span>, norovirus, or other emerging viruses of the future. 
<h3>20980514</h3>Domain structure of <span class="annotated ">Lassa virus</span> L protein.<br><br>The 200-kDa L protein of arenaviruses plays a central role in <span class="match unannotated ">viral</span> genome replication and transcription. This study aimed at providing evidence for the domain structure of L protein by combining bioinformatics with a stepwise mutagenesis approach using the <span class="annotated ">Lassa virus</span> minireplicon system. Potential interdomain linkers were predicted using various algorithms. The prediction was challenged by insertion of flexible sequences into the predicted linkers. Insertion of 5 or 10 amino acid residues was tolerated at seven sites (S407, G446, G467, G774, G939, S1952, and V2074 in <span class="annotated ">Lassa virus AV</span>). At two of these sites, G467 and G939, L protein could be split into an N-terminal and a C-terminal part, which were able to trans-complement each other and reconstitute a functional complex upon coexpression. Coimmunoprecipitation studies revealed physical interaction between the N- and C-terminal domains, irrespective of whether L protein was split at G467 or G939. In confocal immunofluorescence microscopy, the N-terminal domains showed a dot-like, sometimes perinuclear, cytoplasmic distribution similar to that of full-length L protein, while the C-terminal domains were homogenously distributed in cytoplasm. The latter were redistributed into the dot-like structures upon coexpression with the corresponding N-terminal domain. In conclusion, this study demonstrates two interdomain linkers in <span class="annotated ">Lassa virus</span> L protein, at G467 and G939, suggesting that L protein is composed of at least three structural domains spanning residues 1 to 467, 467 to 939, and 939 to 2220. The first domain seems to mediate accumulation of L protein into cytoplasmic dot-like structures. 
<h3>20980524</h3>Association of TRIM22 with the type 1 interferon response and <span class="match unannotated ">viral</span> control during primary <span class="annotated ">HIV-1</span> infection. <br><br>Type 1 interferons (IFNs) induce the expression of the tripartite interaction motif (TRIM) family of E3 ligases, but the contribution of these antiviral factors to <span class="annotated ">HIV</span> pathogenesis is not completely understood. We hypothesized that the increased expression of select type 1 IFN and TRIM isoforms is associated with a significantly lower likelihood of <span class="annotated ">HIV-1</span> acquisition and <span class="match unannotated ">viral</span> control during primary <span class="annotated ">HIV-1</span> infection. We measured IFN-alpha, IFN-Beta, myxovirus resistance protein <span class="unannotated ">A</span> (MxA), <span class="annotated ">human</span> TRIM5alpha (huTRIM5alpha), and TRIM22 mRNA levels in peripheral blood mononuclear cells (PBMCs) of high-risk, <span class="annotated ">HIV-1</span>-uninfected <span class="linneaus_only ">participants</span> and <span class="annotated ">HIV-1</span>-positive study <span class="linneaus_only ">participants</span>. Samples were available for 32 uninfected subjects and 28 infected <span class="linneaus_only ">persons</span>, all within 1 year of infection. <span class="annotated ">HIV-1</span>-positive <span class="linneaus_only ">participants</span> had higher levels of IFN-Beta (P = 0.0005), MxA (P = 0.007), and TRIM22 (P = 0.01) and lower levels of huTRIM5alpha (P &lt; 0.001) than did <span class="annotated ">HIV-1</span>-negative <span class="linneaus_only ">participants</span>. TRIM22 but not huTRIM5alpha correlated positively with type 1 IFN (IFN-alpha, IFN-Beta, and MxA) (all P &lt; 0.0001). In a multivariate model, increased MxA expression showed a significant positive association with <span class="match unannotated ">viral</span> load (P = 0.0418). Furthermore, TRIM22 but not huTRIM5alpha, IFN-alpha, IFN-Beta, or MxA showed a negative correlation with plasma <span class="match unannotated ">viral</span> load (P = 0.0307) and a positive correlation with CD4(+) T-cell counts (P = 0.0281). In vitro studies revealed that <span class="annotated ">HIV</span> infection induced TRIM22 expression in PBMCs obtained from <span class="annotated ">HIV</span>-negative donors. Stable TRIM22 knockdown resulted in increased <span class="annotated ">HIV-1</span> particle release and replication in Jurkat reporter cells. Collectively, these data suggest concordance between type 1 IFN and TRIM22 but not huTRIM5alpha expression in PBMCs and that TRIM22 likely acts as an antiviral effector in vivo. 
<h3>20980525</h3>Determinants that specify the integration pattern of retrotransposon Tf1 in the fbp1 promoter of <span class="annotated ">Schizosaccharomyces pombe</span>. <br><br>Long terminal repeat (LTR) retrotransposons are closely related to retroviruses and, as such, are important models for the study of <span class="match unannotated ">viral</span> integration and target site selection. The transposon Tf1 of <span class="annotated ">Schizosaccharomyces pombe</span> integrates with a strong preference for the promoters of polymerase II (Pol II)-transcribed genes. Previous work in vivo with plasmid-based targets revealed that the patterns of insertion were promoter specific and highly reproducible. To determine which features of promoters are recognized by Tf1, we studied integration in a promoter that has been characterized. The promoter of fbp1 has two upstream activating sequences, UAS1 and UAS2. We found that integration was targeted to two windows, one 180 nucleotides (nt) upstream and the other 30 to 40 nt downstream of UAS1. <span class="unannotated ">A</span> series of deletions in the promoter showed that the integration activities of these two regions functioned autonomously. Integration assays of UAS2 and of a synthetic promoter demonstrated that strong promoter activity alone was not sufficient to direct integration. The factors that modulate the transcription activities of UAS1 and UAS2 include the activators Atf1p, Pcr1p, and Rst2p as well as the repressors Tup11p, Tup12p, and Pka1p. Strains lacking each of these proteins revealed that Atf1p alone mediated the sites of integration. These data indicate that Atf1p plays a direct and specific role in targeting integration in the promoter of fbp1. 
<h3>21029751</h3>Real time PCR method for simultaneous detection, quantitation and differentiation of capripoxviruses. <br><br>The genus Capripoxvirus (CaPV) comprises three members namely, <span class="annotated ">sheep poxvirus</span> (<span class="annotated ">SPPV</span>), <span class="annotated ">goat poxvirus</span> (<span class="annotated ">GTPV</span>) and <span class="annotated ">lumpy skin disease virus</span> (<span class="annotated ">LSDV</span>) affecting <span class="annotated ">sheep</span>, <span class="annotated ">goats</span> and <span class="annotated ">cattle</span>, respectively. CaPV infections produce similar symptoms in <span class="annotated ">sheep</span> and <span class="annotated ">goats</span>, and the three viruses cannot be distinguished serologically. Since there are conflicting opinions regarding the host specificity of CaPVs, particularly for <span class="annotated ">goatpox</span> and <span class="annotated ">sheeppox viruses</span>, the development of rapid genotyping tools will facilitate more accurate disease diagnosis and surveillance for better management of capripox outbreaks. This paper describes a species-specific, real time polymerase chain reaction (PCR), based on unique molecular markers that were found in the G-protein-coupled chemokine receptor (GPCR) gene sequences of CaPVs, that uses dual hybridization probes for their simultaneous detection, quantitation and genotyping. The assay can differentiate between CaPV strains based on differences in the melting point temperature (Tm) obtained after fluorescence melting curve analysis (FMCA). It is highly sensitive and presents low intra- and inter-run variation. This real time PCR assay will make a significant contribution to CaPV diagnosis and to the better understanding of the epidemiology of CaPVs by enabling rapid genotyping and gene-based classification of <span class="match unannotated ">viral</span> strains and unequivocal identification of isolates. 
<h3>21034775</h3>Cross-genotypic polyclonal anti-<span class="annotated ">HCV</span> antibodies from <span class="annotated ">human</span> ascitic fluid.<br><br>Many anti-<span class="annotated ">HCV</span> antibodies are available, but more are needed for research and clinical applications. This study examines whether ascitic fluid from cirrhotic <span class="linneaus_only ">patients</span> could be a source of reagent-grade antibodies. Ascitic fluid from 29 <span class="annotated ">HCV</span> <span class="linneaus_only ">patients</span> was screened by ELISA for anti-<span class="annotated ">HCV</span> antibodies against three <span class="match unannotated ">viral</span> proteins: core, NS4B, and NS5A. Significant patient-to-patient variability in anti-<span class="annotated ">HCV</span> antibody titers was observed. Total ascitic fluid IgG purified by Protein-A chromatography reacted with <span class="annotated ">HCV</span> proteins in immunoblots, cell extracts, and replicon-expressing cells. Affinity-purification using synthetic peptides as bait allowed the preparation of cross-genotypic antibodies directed against pre-selected regions of <span class="unannotated ">HCV</span> core, NS4B, and NS5A proteins. The performance of the polyclonal antibodies was comparable to that of monoclonal antibodies. Anti-NS4B antibody preparations reacted with genotype 1a, 1b, and 2a NS4B proteins in immunoblots and allowed NS4B to be localized in replicon-expressing cells. Ascitic fluid is an abundant source of <span class="annotated ">human</span> polyclonal cross-genotypic antibodies that can be used as an alternative to blood. This study shows the utility of selectively purifying <span class="annotated ">human</span> polyclonal antibodies from ascitic fluid. Affinity purification allows antibodies to be selected that are comparable to monoclonal antibodies in their ability to react with targeted regions of <span class="match unannotated ">viral</span> proteins. 
<h3>21034776</h3>A versatile vector for the production of pseudotyped viruses expressing gp120 antigens from different clades of primary <span class="annotated ">HIV-1</span> isolates. <br><br>A novel <span class="annotated ">HIV-1</span> Env expression vector (SF162-Z) was developed by introducing two new cloning sites on the backbone of an existing vector that produces a full length Env from <span class="annotated ">HIV-1</span> SF162 isolate. These sites facilitate the swapping of the gp120 portion of the SF162 Env with matching gp120 antigens from <span class="annotated ">HIV-1</span> isolates of different genetic clades. Final production of functional pseudotyped viruses will express chimeric Env antigens, including gp41 of the parental SF162 and gp120 from other primary isolates. This system is useful for testing the neutralizing sensitivity of partial env gene products frequently identified in <span class="match unannotated ">viral</span> quasi species in <span class="linneaus_only ">patients</span> infected with <span class="annotated ">HIV</span> or when only partial gp120 gene products are available. 
<h3>21039560</h3>The second face of a known player: <span class="annotated ">Arabidopsis</span> silencing suppressor AtXRN4 acts organ-specifically. <br><br>Plant viruses exploit the symplastic transport pathway provided by plasmodesmata by encoding for specialized movement proteins, which interact with host factors to enable <span class="match unannotated ">viral</span> intracellular and intercellular spread. Stable expression of the <span class="annotated ">Potato leaf roll virus</span> movement protein MP17 in <span class="annotated ">Arabidopsis</span> results in a carbohydrate export block and stunted growth. To identify host factors essential for <span class="match unannotated ">viral</span> infection, we screened a progeny population of EMS (ethyl methanesulfonate)-mutagenized <span class="annotated ">Arabidopsis</span> expressing a MP17:GFP fusion for suppressor mutants with restored wild type-like phenotype. Two suppressor mutants showed decreased susceptibility against <span class="annotated ">Turnip mosaic virus</span> and post-transcriptional silencing of MP17:GFP RNA in source leaves. Map based cloning identified in both lines mutations in XRN4 (Exoribonuclease 4), which was previously described as a suppressor of transgene silencing in source leaves. Importantly, silencing of MP17:GFP was not present in cotyledons and roots of the two suppressor mutants, which was confirmed in a third xrn4 T-DNA knock out line. Subsequent analysis of MP17:GFP transcript stability in xrn2 and xrn3 mutants indicated an essential role of AtXRN2 for silencing suppression in roots/cotyledons while AtXRN3 appears to act similar to AtXRN4 in source leaves, only. Overall, these findings point towards an organ-specific regulation of gene silencing in <span class="annotated ">Arabidopsis</span>. 
<h3>21047958</h3>Stepwise loss of fluorescent core protein V from <span class="annotated ">human adenovirus</span> during entry into cells. <br><br><span class="annotated ">Human adenoviruses (Ads)</span> replicate and assemble particles in the nucleus. They organize a linear double-strand DNA genome into a condensed core with about 180 nucleosomes, by the <span class="match unannotated ">viral</span> proteins VII (pVII), pX, and pV attaching the DNA to the capsid. Using reverse genetics, we generated a novel, nonconditionally replicating <span class="unannotated ">Ad</span> reporter by inserting green fluorescent protein (GFP) at the amino terminus of pV. Purified Ad2-GFP-pV virions had an oversized complete genome and incorporated about 38 GFP-pV molecules per virion, which is about 25% of the pV levels in Ad2. GFP-pV cofractionated with the DNA core, like pV, and newly synthesized GFP-pV had a subcellular localization indistinguishable from that of pV, indicating that GFP-pV is a valid reporter for pV. Ad2-GFP-pV completed the replication cycle, although at lower yields than Ad2. Incoming GFP-pV (or pV) was not imported into the nucleus. Virions lost GFP-pV at two points during the infection process: at entry into the cytosol and at the nuclear pore complex, where capsids disassemble. Disassembled capsids, positive for the conformation-specific antihexon antibody R70, were devoid of GFP-pV. The loss of GFP-pV was reduced by the macrolide antibiotic leptomycin B (LMB), which blocks nuclear export and <span class="annotated ">adenovirus</span> attachment to the nuclear pore complex. LMB inhibited the appearance of R70 epitopes on Ad2 and Ad2-GFP-pV, indicating that the loss of GFP-pV from Ad2-GFP-pV is an authentic step in the <span class="annotated ">adenovirus</span> uncoating program. Ad2-GFP-pV is genetically complete and hence enables detailed analyses of infection and spreading dynamics in cells and model organisms or assessment of oncolytic <span class="annotated ">adenoviral</span> potential. 
<h3>21067973</h3>Microbial and <span class="match unannotated ">viral</span> pathogens in colorectal cancer.<br><br>The heterogenetic and sporadic nature of colorectal cancer has led to many epidemiological associations with causes of this disease. As our understanding of the underlying molecular processes in colorectal-cancer develops, the concept of microbial-epithelial interactions as an oncogenic trigger might provide a plausible hypothesis for the pathogenesis of colorectal cancer. By contrast with other cancers of the gastrointestinal tract (gastric carcinoma, mucosa-associated lymphoid-tissue lymphoma), a direct causal link between microbial infection (bacteria and viruses) and colorectal carcinoma has not been established. Studies support the involvement of these organisms in oncogenesis, however, in colorectal cancer, clinical data are lacking. Here, we discuss current evidence (both in vitro and clinical studies), and focus on a putative role for bacterial and <span class="match unannotated ">viral</span> pathogens as a cause of colorectal cancer. 
<h3>21084112</h3>Ebola haemorrhagic fever.<br><br><span class="annotated ">Ebola viruses</span> are the causative agents of a severe form of <span class="match unannotated ">viral</span> haemorrhagic fever in <span class="linneaus_only ">man</span>, designated Ebola haemorrhagic fever, and are endemic in regions of central Africa. The exception is the species <span class="annotated ">Reston Ebola virus</span>, which has not been associated with <span class="annotated ">human</span> disease and is found in the Philippines. <span class="annotated ">Ebola virus</span> constitutes an important local public health threat in Africa, with a worldwide effect through imported infections and through the fear of misuse for biological terrorism. <span class="annotated ">Ebola virus</span> is thought to also have a detrimental effect on the great ape population in Africa. Case-fatality rates of the African species in <span class="linneaus_only ">man</span> are as high as 90%, with no prophylaxis or treatment available. <span class="annotated ">Ebola virus</span> infections are characterised by immune suppression and a systemic inflammatory response that causes impairment of the vascular, coagulation, and immune systems, leading to multiorgan failure and shock, and thus, in some ways, resembling septic shock. 
<h3>21435708</h3>Viral pneumonia.<br><br>About 200 million cases of <span class="match unannotated ">viral</span> community-acquired pneumonia occur every year-100 million in <span class="linneaus_only ">children</span> and 100 million in adults. Molecular diagnostic tests have greatly increased our understanding of the role of viruses in pneumonia, and findings indicate that the incidence of <span class="match unannotated ">viral</span> pneumonia has been underestimated. In <span class="linneaus_only ">children</span>, <span class="annotated ">respiratory syncytial virus</span>, rhinovirus, <span class="annotated ">human metapneumovirus</span>, <span class="annotated ">human bocavirus</span>, and parainfluenza viruses are the agents identified most frequently in both developed and developing countries. Dual <span class="match unannotated ">viral</span> infections are common, and a third of <span class="linneaus_only ">children</span> have evidence of viral-bacterial co-infection. In adults, viruses are the putative causative agents in a third of cases of community-acquired pneumonia, in particular <span class="unannotated ">influenza viruses</span>, rhinoviruses, and coronaviruses. Bacteria continue to have a predominant role in adults with pneumonia. Presence of <span class="match unannotated ">viral</span> epidemics in the community, patient's age, speed of onset of illness, symptoms, biomarkers, radiographic changes, and response to treatment can help differentiate <span class="match unannotated ">viral</span> from bacterial pneumonia. However, no clinical algorithm exists that will distinguish clearly the cause of pneumonia. No clear consensus has been reached about whether <span class="linneaus_only ">patients</span> with obvious <span class="match unannotated ">viral</span> community-acquired pneumonia need to be treated with antibiotics. Apart from neuraminidase inhibitors for pneumonia caused by <span class="unannotated ">influenza viruses</span>, there is no clear role for use of specific antivirals to treat <span class="match unannotated ">viral</span> community-acquired pneumonia. Influenza vaccines are the only available specific preventive measures. Further studies are needed to better understand the cause and pathogenesis of community-acquired pneumonia. Furthermore, regional differences in cause of pneumonia should be investigated, in particular to obtain more data from developing countries. 
<h3>21496127</h3>A role for endosomal proteins in alphavirus dissemination in mosquitoes.<br><br>Little is known about endosomal pathway proteins involved in arthropod-borne <span class="unannotated ">virus</span> (arbovirus) assembly and cell-to-cell spread in vector mosquitoes. UNC93A and synaptic vesicle-2 (SV2) proteins are involved in intracellular transport in mammals. They show amino acid sequence conservation from mosquitoes to <span class="annotated ">humans</span>, and their transcripts are highly enriched in <span class="annotated ">Aedes aegypti</span> during arbovirus infection. Transient gene silencing of SV2 or UNC93A in mosquitoes infected with the recombinant alphavirus <span class="annotated ">Sindbis</span> MRE16-enhanced green fluorescent protein (<span class="annotated ">SINV</span>; family Togaviridae) resulted in the accumulation of <span class="match unannotated ">viral</span> positive- and negative-strand RNA, congregation of <span class="unannotated ">virus</span> envelope antigen in intracellular networks, and reduced <span class="unannotated ">virus</span> dissemination outside of the midgut. Further, UNC93A silencing, but not SV2 silencing, resulted in a 10-fold reduction in <span class="match unannotated ">viral</span> titres at 4 days post-infection. Together, these data support a role for UNC93A and SV2 in <span class="unannotated ">virus</span> assembly or budding. Cis-regulatory elements (CREs) were identified at the 5'-ends of genes from the original data set in which SV2 and UNC93A were identified. Common CREs at the 5'-end genomic regions of a subset of enriched transcripts support the hypothesis that UNC93A transcription may be co-regulated with that of other ion transport and endosomal trafficking proteins. 
<h3>21511329</h3><span class="annotated ">Hepatitis delta virus</span>.<br><br><span class="annotated ">Hepatitis delta virus</span> (<span class="annotated ">HDV</span>) is a small, defective RNA <span class="unannotated ">virus</span> that can infect only individuals who have <span class="annotated ">hepatitis B virus</span> (<span class="annotated ">HBV</span>); worldwide more than 15 million <span class="linneaus_only ">people</span> are co-infected. There are eight reported genotypes of <span class="annotated ">HDV</span> with unexplained variations in their geographical distribution and pathogenicity. The <span class="annotated ">hepatitis D</span> virion is composed of a coat of <span class="annotated ">HBV</span> envelope proteins surrounding the nucleocapsid, which consists of a single-stranded, circular RNA genome complexed with delta antigen, the <span class="match unannotated ">viral</span> protein. <span class="annotated ">HDV</span> is clinically important because although it suppresses <span class="annotated ">HBV</span> replication, it causes severe liver disease with rapid progression to cirrhosis and hepatic decompensation. The range of clinical presentation is wide, varying from mild disease to fulminant liver failure. The prevalence of <span class="annotated ">HDV</span> is declining in some endemic areas but increasing in northern and central Europe because of immigration. Treatment of <span class="annotated ">HDV</span> is with pegylated interferon alfa; however, response rates are poor. Increased understanding of the molecular virology of <span class="annotated ">HDV</span> will identify novel therapeutic targets for this most severe form of chronic <span class="match unannotated ">viral</span> hepatitis. 
<h3>21699595</h3>Host translational control of a polydnavirus, <span class="annotated ">Cotesia plutellae bracovirus</span>, by sequestering host eIF4A to prevent formation of a translation initiation complex. <br><br>Host translational control is a <span class="match unannotated ">viral</span> strategy to exploit host cellular resources. Parasitization by some endoparasitoids containing polydnaviruses inhibits the synthesis of specific host proteins at post-transcriptional level. Two host translation inhibitory factors (HTIFs) have been proposed in <span class="annotated ">Cotesia plutellae bracovirus</span> (<span class="annotated ">CpBV</span>). Parasitization by <span class="annotated ">C. plutellae</span> inhibited storage protein 1 (SP1) synthesis of <span class="annotated ">Plutella xylostella</span> at post-transcriptional level. One HTIF, CpBV15Beta, inhibited the translation of SP1 mRNA in an in vitro translation assay using <span class="annotated ">rabbit</span> reticulocyte lysate, but did not inhibit its own mRNA. To further analyse the discrimination of target and nontarget mRNAs of the inhibitory effect of HTIF, 5' untranslated regions (UTRs) of SP1 and CpBV15Beta mRNA were reciprocally exchanged. In the presence of HTIFs, the chimeric CpBV15Beta mRNA that contained SP1 5' UTR was not translated, whereas the chimeric SP1 mRNA that contained CpBV15Beta 5' UTR was translated. There was a difference in the 5' UTR secondary structures between target (SP1) and nontarget (CpBV15alpha and CpBV15Beta) mRNAs in terms of thermal stability. Different mutant 5' UTRs of SP1 mRNA were prepared by point mutations to modify their secondary structures. The constructs containing 5' UTRs of high thermal stability in their secondary structures were inhibited by HTIF, but those of low thermal stability were not. Immunoprecipitation with CpBV15Beta antibody coprecipitated eIF4A, which would be required for unwinding the secondary structure of the 5' UTR. These results indicate that the <span class="match unannotated ">viral</span> HTIF discriminates between host mRNAs according to their dependency on eIF4A to form a functional initiation complex for translation. 
<h3>20962082</h3>The alphaherpesvirus US3/ORF66 protein kinases direct phosphorylation of the nuclear matrix protein matrin 3. <br><br>The protein kinase found in the short region of alphaherpesviruses, termed US3 in <span class="annotated ">herpes simplex virus type 1</span> (<span class="annotated ">HSV-1</span>) and <span class="annotated ">pseudorabies virus</span> (<span class="annotated ">PRV</span>) and ORF66 in <span class="annotated ">varicella-zoster virus</span> (<span class="annotated ">VZV</span>), affects several <span class="match unannotated ">viral</span> and host cell processes, and its specific targets remain an area of active investigation. Reports suggesting that <span class="annotated ">HSV-1</span> US3 substrates overlap with those of cellular protein kinase <span class="unannotated ">A</span> (PKA) prompted the use of an antibody specific for phosphorylated PKA substrates to identify US3/ORF66 targets. <span class="annotated ">HSV-1</span>, <span class="annotated ">VZV</span>, and <span class="annotated ">PRV</span> induced very different substrate profiles that were US3/ORF66 kinase dependent. The predominant <span class="annotated ">VZV</span>-phosphorylated 125-kDa species was identified as matrin 3, one of the major nuclear matrix proteins. Matrin 3 was also phosphorylated by <span class="annotated ">HSV-1</span> and <span class="annotated ">PRV</span> in a US3 kinase-dependent manner and by <span class="annotated ">VZV</span> ORF66 kinase at a novel residue (KRRRT150EE). Since <span class="annotated ">VZV</span>-directed T150 phosphorylation was not blocked by PKA inhibitors and was not induced by PKA activation, and since PKA predominantly targeted matrin 3 S188, it was concluded that phosphorylation by <span class="annotated ">VZV</span> was PKA independent. However, purified <span class="annotated ">VZV</span> ORF66 kinase did not phosphorylate matrin 3 in vitro, suggesting that additional cellular factors were required. In <span class="annotated ">VZV</span>-infected cells in the absence of the ORF66 kinase, matrin 3 displayed intranuclear changes, while matrin 3 showed a pronounced cytoplasmic distribution in late-stage cells infected with US3-negative <span class="annotated ">HSV-1</span> or <span class="annotated ">PRV</span>. This work identifies phosphorylation of the nuclear matrix protein matrin 3 as a new conserved target of this kinase group. 
<h3>20962083</h3>Small-molecule inhibition of <span class="annotated ">human immunodeficiency virus type 1</span> infection by <span class="unannotated ">virus</span> capsid destabilization. <br><br><span class="annotated ">Human immunodeficiency virus type 1</span> (<span class="annotated ">HIV-1</span>) infection is dependent on the proper disassembly of the <span class="match unannotated ">viral</span> capsid, or &quot;uncoating,&quot; in target cells. The <span class="annotated ">HIV-1</span> capsid consists of a conical multimeric complex of the <span class="match unannotated ">viral</span> capsid protein (CA) arranged in a hexagonal lattice. Mutations in CA that destabilize the <span class="match unannotated ">viral</span> capsid result in impaired infection owing to defects in reverse transcription in target cells. We describe here the mechanism of action of a small molecule <span class="annotated ">HIV-1</span> inhibitor, PF-3450074 (PF74), which targets CA. PF74 acts at an early stage of <span class="annotated ">HIV-1</span> infection and inhibits reverse transcription in target cells. We show that PF74 binds specifically to <span class="annotated ">HIV-1</span> particles, and substitutions in CA that confer resistance to the compound prevent binding. <span class="unannotated ">A</span> single point mutation in CA that stabilizes the <span class="annotated ">HIV-1</span> core also conferred strong resistance to the <span class="unannotated ">virus</span> without inhibiting compound binding. Treatment of <span class="annotated ">HIV-1</span> particles or purified cores with PF74 destabilized the <span class="match unannotated ">viral</span> capsid in vitro. Furthermore, the compound induced the rapid dissolution of the <span class="annotated ">HIV-1</span> capsid in target cells. PF74 antiviral activity was promoted by binding of the host protein cyclophilin <span class="unannotated ">A</span> to the <span class="annotated ">HIV-1</span> capsid, and PF74 and cyclosporine exhibited mutual antagonism. Our data suggest that PF74 triggers premature <span class="annotated ">HIV-1</span> uncoating in target cells, thereby mimicking the activity of the retrovirus restriction factor TRIM5alpha. This study highlights uncoating as a step in the <span class="annotated ">HIV-1</span> life cycle that is susceptible to small molecule intervention. 
<h3>20962098</h3>PB2 residue 158 is a pathogenic determinant of pandemic <span class="annotated ">H1N1</span> and <span class="annotated ">H5 influenza a viruses</span> in <span class="annotated ">mice</span>. <br><br><span class="annotated ">Influenza A viruses</span> are <span class="annotated ">human</span> and animal pathogens that cause morbidity and mortality, which range from mild to severe. The 2009 <span class="annotated ">H1N1</span> pandemic was caused by the emergence of a reassortant <span class="annotated ">H1N1</span> subtype (<span class="annotated ">H1N1pdm</span>) <span class="annotated ">influenza A virus</span> containing gene segments that originally circulated in <span class="annotated ">human</span>, avian, and <span class="annotated ">swine</span> <span class="unannotated ">virus</span> reservoirs. The molecular determinants of replication and pathogenesis of <span class="annotated ">H1N1pdm viruses</span> in <span class="annotated ">humans</span> and other mammals are poorly understood. Therefore, we set out to elucidate <span class="match unannotated ">viral</span> determinants critical to the pathogenesis of this novel reassortant using a <span class="annotated ">mouse</span> model. We found that a glutamate-to-glycine substitution at residue 158 of the PB2 gene (PB2-E158G) increased the morbidity and mortality of the parental <span class="annotated ">H1N1pdm virus</span>. Results from mini-genome replication assays in <span class="annotated ">human</span> cells and <span class="unannotated ">virus</span> titration in <span class="annotated ">mouse</span> tissues demonstrated that PB2-E158G is a pathogenic determinant, because it significantly increases <span class="match unannotated ">viral</span> replication rates. The <span class="unannotated ">virus</span> load in PB2-E158G-infected <span class="annotated ">mouse</span> lungs was 1,300-fold higher than that of the wild-type <span class="unannotated ">virus</span>. Our data also show that PB2-E158G had a much stronger influence on the RNA replication and pathogenesis of <span class="annotated ">H1N1pdm viruses</span> than PB2-E627K, which is a known pathogenic determinant. Remarkably, PB2-E158G substitutions also altered the pathotypes of two avian <span class="annotated ">H5 viruses</span> in <span class="annotated ">mice</span>, indicating that this residue impacts genetically divergent <span class="annotated ">influenza A viruses</span> and suggesting that this region of PB2 could be a new antiviral target. Collectively, the data presented in this study demonstrate that PB2-E158G is a novel pathogenic determinant of <span class="annotated ">influenza A viruses</span> in the <span class="annotated ">mouse</span> model. We speculate that PB2-E158G may be important in the adaptation of avian PB2 genes to other mammals, and BLAST sequence analysis identified a naturally occurring <span class="annotated ">human</span> <span class="annotated ">H1N1pdm</span> isolate that has this substitution. Therefore, future surveillance efforts should include scrutiny of this region of PB2 because of its potential impact on pathogenesis. 
<h3>20980511</h3>Dominant inhibition of Akt/protein kinase B signaling by the matrix protein of a negative-strand RNA <span class="unannotated ">virus</span>. <br><br><span class="annotated ">Vesicular stomatitis virus</span> (<span class="annotated ">VSV</span>) is a rhabdovirus that alters host nuclear and cytoplasmic function upon infection. We have investigated the effect of <span class="annotated ">VSV</span> infection on cellular signaling through the phosphatidylinositol-3 kinase (PI3k)/Akt signaling pathway. Akt phosphorylation at both threonine 308 (Thr308) and serine 473 (Ser473) was inhibited in cells infected with <span class="annotated ">VSV</span>. This inhibition was rapid (beginning within the first 2 to 3 h postinfection) and correlated with the dephosphorylation of downstream effectors of Akt, such as glycogen synthase kinase 3Beta (GSK3Beta) and mammalian target of rapamycin (mTOR). The dephosphorylation of Akt occurred in the presence of growth factor stimulation and was not overcome through constitutive membrane targeting of Akt or high levels of phosphatidylinositol-3,4,5-triphosphate (PIP3) accumulation in the membrane. Akt dephosphorylation was not a result of alterations in PDK1 phosphorylation or activity, changes in phosphatase and tensin homologue deleted on chromosome 10 (PTEN) levels, or the downregulation of PI3k signaling. Inactivation of Akt was caused by the expression of the <span class="match unannotated ">viral</span> M protein in the absence of other <span class="match unannotated ">viral</span> components, and an M protein mutant that does not inhibit RNA polymerase II (Pol II) transcription and nuclear/cytoplasmic transport was also defective in inhibiting Akt phosphorylation. These data illustrate that <span class="annotated ">VSV</span> utilizes a novel mechanism to alter this central player in cell signaling and oncogenesis. It also suggests an inside-out model of signal transduction where <span class="annotated ">VSV</span> interruption of nuclear events has a rapid and significant effect on membrane signaling events. 
<h3>20980526</h3>An in-depth analysis of original antigenic sin in <span class="annotated ">dengue virus</span> infection.<br><br>The evolution of <span class="annotated ">dengue viruses</span> has resulted in four antigenically similar yet distinct serotypes. Infection with one serotype likely elicits lifelong immunity to that serotype, but generally not against the other three. Secondary or sequential infections are common, as multiple <span class="match unannotated ">viral</span> serotypes frequently cocirculate. <span class="annotated ">Dengue</span> infection, although frequently mild, can lead to <span class="annotated ">dengue</span> hemorrhagic fever (DHF) which can be life threatening. DHF is more common in secondary <span class="annotated ">dengue</span> infections, implying a role for the adaptive immune response in the disease. There is currently much effort toward the design and implementation of a <span class="unannotated ">dengue</span> vaccine but these efforts are made more difficult by the challenge of inducing durable neutralizing immunity to all four viruses. Domain 3 of the <span class="annotated ">dengue virus</span> envelope protein (ED3) has been suggested as one such candidate because it contains neutralizing epitopes and it was originally thought that relatively few cross-reactive antibodies are directed to this domain. In this study, we performed a detailed analysis of the anti-ED3 response in a cohort of <span class="linneaus_only ">patients</span> suffering either primary or secondary <span class="annotated ">dengue</span> infections. The results show dramatic evidence of original antigenic sin in secondary infections both in terms of binding and enhancement activity. This has important implications for <span class="annotated ">dengue</span> vaccine design because heterologous boosting is likely to maintain the immunological footprint of the first vaccination. On the basis of these findings, we propose a simple in vitro enzyme-linked immunosorbent assay (ELISA) to diagnose the original <span class="annotated ">dengue</span> infection in secondary <span class="annotated ">dengue</span> cases. 
<h3>21029747</h3>A fluorescence resonance energy transfer-based fluorometer assay for screening anti-<span class="annotated ">coxsackievirus B3</span> compounds. <br><br>In view of the need to develop a simple and rapid method to screen for antiviral therapeutic agents, a fluorescence resonance energy transfer (FRET)-based reporter system consisting of engineered mammalian cells expressing a cyan fluorescent protein-yellow fluorescent protein (CFP-YFP) pair linked by a short peptide containing the cleavage site of <span class="match unannotated ">viral</span> protease 2A (2A(pro)) was developed. By detecting the 2A(pro) produced early during the <span class="unannotated ">virus</span> infection cycle, the CFP-YFP pair effectively identifies infectious <span class="annotated ">coxsackievirus B3</span> (<span class="annotated ">CVB3</span>), a picornavirus that causes <span class="match unannotated ">viral</span> myocarditis in <span class="annotated ">humans</span>. The reporter system was used to screen a library of 2000 drugs and natural products for potential antiviral compounds. The reporter cells were treated with the test compounds, challenged with <span class="annotated ">CVB3</span>, and then examined using a fluorometer at 24h post-infection. Sixty-four compounds, mostly therapeutic drugs, antimicrobial compounds and compounds with unknown functions, caused at least 50% inhibition of 2A(pro) activity. Three known antiviral compounds, cosmosiin, ribavirin and baicalein, were also identified in the screening. The developed method is an effective strategy for rapid screening, and identifies compounds that inhibit <span class="annotated ">CVB3</span> 2A(pro). This method should be a valuable aid in the antiviral drug discovery effort. 
<h3>21029748</h3>One-step real-time reverse transcription-PCR assays for detecting and subtyping <span class="annotated ">pandemic influenza A/H1N1 2009</span>, <span class="annotated ">seasonal influenza A/H1N1</span>, and <span class="annotated ">seasonal influenza A/H3N2 viruses</span>. <br><br><span class="annotated ">Pandemic influenza A/H1N1 2009 (A/H1N1pdm) virus</span> has caused significant outbreaks worldwide. <span class="unannotated ">A</span> previous one-step real-time reverse transcription-PCR (rRT-PCR) assay for detecting <span class="annotated ">A/H1N1pdm virus</span> (H1pdm rRT-PCR assay) was improved since the former probe had a low melting temperature and low tolerance to <span class="match unannotated ">viral</span> mutation. To help with the screening of the <span class="annotated ">A/H1N1pdm virus</span>, rRT-PCR assays were also developed for detecting <span class="annotated ">human</span> <span class="annotated ">seasonal A/H1N1</span> (H1 rRT-PCR assay) and <span class="annotated ">A/H3N2 influenza viruses</span> (H3 rRT-PCR assay). H1pdm, <span class="unannotated ">H1</span>, and H3 rRT-PCR assays were evaluated using in vitro-transcribed control RNA, isolated viruses, and other respiratory pathogenic viruses, and were shown to have high sensitivity, good linearity (R(2)=0.99), and high specificity. In addition, the improved H1pdm rRT-PCR assay could detect two <span class="annotated ">viral strains of A/H1N1pdm</span>, namely, <span class="annotated ">A/Aichi/472/2009 (H1N1)pdm</span> and <span class="annotated ">A/Sakai/89/2009 (H1N1)pdm</span>, which have mutation(s) in the probe-binding region of the hemagglutinin gene, without loss of sensitivity. Using the three rRT-PCR assays developed, 90 clinical specimens collected between May and October 2009 were then tested. Of these, 26, 20, and 2 samples were identified as positive for <span class="annotated ">A/H1pdm</span>, <span class="annotated ">A/H3</span>, and <span class="annotated ">A/H1</span>, respectively, while 42 samples were negative for <span class="annotated ">influenza A viruses</span>. The present results suggest that these highly sensitive and specific H1pdm, <span class="unannotated ">H1</span>, and H3 rRT-PCR assays are useful not only for diagnosing <span class="annotated ">influenza viruses</span>, but also for the surveillance of <span class="annotated ">influenza viruses</span>. 
<h3>21047957</h3>Modification of nonstructural protein 1 of <span class="annotated ">influenza A virus</span> by SUMO1.<br><br>Nonstructural protein 1 (NS1) is one of the major factors resulting in the efficient infection rate and high level of virulence of <span class="annotated ">influenza A virus</span>. Although consisting of only approximately 230 amino acids, NS1 has the ability to interfere with several systems of the host <span class="match unannotated ">viral</span> defense. In the present study, we demonstrate that NS1 of the highly pathogenic avian <span class="annotated ">influenza</span> <span class="annotated ">A/Duck/Hubei/L-1/2004 (H5N1) virus</span> interacts with <span class="annotated ">human</span> Ubc9, which is the E2 conjugating enzyme for sumoylation, and we show that SUMO1 is conjugated to <span class="annotated ">H5N1</span> NS1 in both transfected and infected cells. Furthermore, two lysine residues in the C terminus of NS1 were identified as SUMO1 acceptor sites. When the SUMO1 acceptor sites were removed by mutation, NS1 underwent rapid degradation. Studies of different <span class="annotated ">influenza A virus</span> strains of <span class="annotated ">human</span> and avian origin showed that the majority of viruses possess an NS1 protein that is modified by SUMO1, except for the recently emerged <span class="annotated ">swine-origin influenza A virus (S-OIV) (H1N1)</span>. Interestingly, growth of a sumoylation-deficient WSN <span class="unannotated ">virus</span> mutant was retarded compared to that of wild-type <span class="unannotated ">virus</span>. Together, these results indicate that sumoylation enhances NS1 stability and thus promotes rapid growth of <span class="annotated ">influenza A virus</span>. 
<h3>21097611</h3>Complete genome sequences of three <span class="annotated ">Erwinia amylovora</span> phages isolated in north america and a <span class="unannotated ">bacteriophage</span> induced from an <span class="annotated ">Erwinia tasmaniensis</span> strain. <br><br>Fire blight, a plant disease of economic importance caused by <span class="annotated ">Erwinia amylovora</span>, may be controlled by the application of bacteriophages. Here, we provide the complete genome sequences and the annotation of three <span class="annotated ">E. amylovora</span>-specific phages isolated in North America and genomic information about a <span class="unannotated ">bacteriophage</span> induced by mitomycin C treatment of an <span class="annotated ">Erwinia tasmaniensis</span> strain that is antagonistic for <span class="annotated ">E. amylovora</span>. The American phages resemble two already-described <span class="match unannotated ">viral</span> genomes, whereas the <span class="annotated ">E. tasmaniensis</span> phage displays a singular genomic sequence in BLAST searches. 
<h3>21585579</h3>Beta-Thymosin is upregulated by the steroid hormone 20-hydroxyecdysone and microorganisms. <br><br>Thymosins have diverse biological activities including actin-sequestering and tissue repair in vertebrates, however, there is little information about the function of thymosins in invertebrates. We isolated a Beta-thymosin gene in <span class="annotated ">Helicoverpa armigera</span>. It has two transcript variants, HaTHY1 and HaTHY2, encoding 19.0 kDa and 14.5 kDa peptides, respectively. HaTHY1 was mainly transcribed in the integument and midgut, while HaTHY2 was principally presented in the fat body and haemocytes. The transcript levels of HaTHY2 showed some fluctuation; there was an obvious increase at the metamorphic stage in the integument or fat body. HaTHY was able to be upregulated by 20-hydroxyecdysone or by bacterial and <span class="match unannotated ">viral</span> challenge. These data suggest that HaTHY is upregulated by the steroid hormone and by responses to microorganism infection. 
<h3>20962081</h3><span class="annotated ">Varicella-zoster virus</span> glycoprotein E is a critical determinant of virulence in the SCID mouse-<span class="annotated ">human</span> model of neuropathogenesis. <br><br><span class="annotated ">Varicella-zoster virus</span> (<span class="annotated ">VZV</span>) is a neurotropic alphaherpesvirus. <span class="annotated ">VZV</span> infection of <span class="annotated ">human</span> dorsal root ganglion (DRG) xenografts in immunodeficient <span class="unannotated ">mice</span> models the infection of sensory ganglia. We examined DRG infection with recombinant <span class="annotated ">VZV</span> (recombinant Oka [rOka]) and the following gE mutants: gEDelta27-90, gEDeltaCys, gE-AYRV, and gE-SSTT. gEDelta27-90, which lacks the gE domain that interacts with a putative receptor insulin-degrading enzyme (IDE), replicated as extensively as rOka, producing infectious virions and significant cytopathic effects within 14 days of inoculation. Since neural cells express IDE, the gE/IDE interaction was dispensable for <span class="annotated ">VZV</span> neurotropism. In contrast, gEDeltaCys, which lacks gE/gI heterodimer formation, was significantly impaired at early times postinfection; <span class="match unannotated ">viral</span> genome copy numbers increased slowly, and infectious <span class="unannotated ">virus</span> production was not detected until day 28. Delayed replication was associated with impaired cell-cell spread in ganglia, similar to the phenotype of a gI deletion mutant (rOkaDeltagI). However, at later time points, infection of satellite cells and other supportive nonneuronal cells resulted in extensive DRG tissue damage and cell loss such that cytopathic changes observed at day 70 were more severe than those for rOka-infected DRG. The replication of gE-AYRV, which is impaired for trans-Golgi network (TGN) localization, and the replication of gE-SSTT, which contains mutations in an acidic cluster, were equivalent to that of rOka, causing significant cytopathic effects and infectious <span class="unannotated ">virus</span> production by day 14; genome copy numbers were equivalent to those of rOka. These experiments suggest that the gE interaction with cellular IDE, gE targeting to TGN sites of virion envelopment, and phosphorylation at SSTT are dispensable for <span class="annotated ">VZV</span> DRG infection, whereas the gE/gI interaction is critical for <span class="unannotated ">VZV</span> neurovirulence. 
<h3>20962089</h3>Antiviral effects of a transgenic RNA-dependent RNA polymerase.<br><br>Transgenic expression of the RNA-dependent RNA polymerase 3D(pol) inhibited infection of <span class="annotated ">Theiler's murine encephalitis virus</span> (<span class="annotated ">TMEV</span>), a picornavirus from which it was derived. Here, we infected 3D(pol) transgenic <span class="annotated ">mice</span> with another picornavirus, as well as an alphaherpesvirus and a rhabdovirus. 3D(pol) transgenic FVB <span class="annotated ">mice</span> had significantly lower <span class="match unannotated ">viral</span> loads and survived longer after infection with all three types of viruses than nontransgenic FVB <span class="annotated ">mice</span>. Viral inhibition among three different types of <span class="unannotated ">virus</span> by transgenic 3D(pol) suggests that the mechanism of action is not the direct interference with picornaviral 3D(pol) but instead may be the changing of host cells to an antiviral state before or after <span class="match unannotated ">viral</span> infection occurs, as basal interferon levels were higher in 3D(pol) transgenic <span class="annotated ">mice</span> before infection. Further study of this mechanism may open new possibilities for future antiviral therapy. 
<h3>20980505</h3>Translational control of the abundance of cytoplasmic poly(A) binding protein in <span class="annotated ">human cytomegalovirus</span>-infected cells. <br><br>Irrespective of their effects on ongoing host protein synthesis, productive replication of the representative alphaherpesvirus <span class="annotated ">herpes simplex virus type 1</span>, the representative gammaherpesvirus <span class="annotated ">Kaposi's sarcoma herpesvirus</span>, and the representative betaherpesvirus <span class="annotated ">human cytomegalovirus</span> [<span class="annotated ">HCMV</span>] stimulates the assembly of the multisubunit, cap-binding translation factor eIF4F. However, only <span class="annotated ">HCMV</span> replication is associated with an increased abundance of eIF4F core components (eIF4E, eIF4G, eIF4A) and the eIF4F-associated factor poly(A) binding protein (PABP). Here, we demonstrate that the increase in translation factor concentration was readily detected in an asynchronous population of <span class="annotated ">HCMV</span>-infected primary <span class="annotated ">human</span> fibroblasts, abolished by prior UV inactivation of <span class="unannotated ">virus</span>, and genetically dependent upon <span class="match unannotated ">viral</span> immediate-early genes. Strikingly, while increased mRNA steady-state levels accompanied the rise in eIF4E and eIF4G protein levels, the overall abundance of PABP mRNA, together with the half-life of the polypeptide it encodes, remained relatively unchanged by <span class="annotated ">HCMV</span> infection. Instead, <span class="annotated ">HCMV</span>-induced PABP accumulation resulted from new protein synthesis and was sensitive to the mTORC1-selective inhibitor rapamycin, which interferes with phosphorylation of the mTORC1 substrate p70 S6K and the translational repressor 4E-BP1. While virus-induced PABP accumulation did not require p70 S6K, it was inhibited by the expression of a dominant-acting 4E-BP1 variant unable to be inactivated by mTORC1. Finally, unlike the situation in alpha- or gammaherpesvirus-infected cells, where PABP is redistributed to nuclei, PABP accumulated in the cytoplasm of <span class="annotated ">HCMV</span>-infected cells. Thus, cytoplasmic PABP accumulation is translationally controlled in <span class="annotated ">HCMV</span>-infected cells via a mechanism requiring mTORC1-mediated inhibition of the cellular 4E-BP1 translational repressor. 
<h3>20980506</h3>A new model of <span class="annotated ">Epstein-Barr virus</span> infection reveals an important role for earlylytic <span class="match unannotated ">viral</span> protein expression in the development of lymphomas.<br><br><span class="annotated ">Epstein-Barr virus</span> (<span class="annotated ">EBV</span>) infects cells in latent or lytic forms, but the roleof lytic infection in <span class="annotated ">EBV</span>-induced lymphomas is unclear. Here, we have used anew humanized <span class="annotated ">mouse</span> model, in which both <span class="annotated ">human</span> fetal CD34(+) hematopoietic stemcells and thymus/liver tissue are transplanted, to compare <span class="annotated ">EBV</span> pathogenesis andlymphoma formation following infection with a lytic replication-defectiveBZLF1-deleted (Z-KO) <span class="unannotated ">virus</span> or a lytically active BZLF1(+) control. Both thecontrol and Z-KO viruses established long-term <span class="match unannotated ">viral</span> latency in all infectedanimals. The infection appeared well controlled in some animals, but otherseventually developed CD20(+) diffuse large B cell lymphomas (DLBCL). Animalsinfected with the control <span class="unannotated ">virus</span> developed tumors more frequently than Z-KOvirus-infected animals. Specific immune responses against <span class="annotated ">EBV</span>-infected B cellswere generated in <span class="annotated ">mice</span> infected with either the control <span class="unannotated ">virus</span> or the Z-KOvirus. In both cases, forms of <span class="match unannotated ">viral</span> latency (type I and type IIB) wereobserved that are less immunogenic than the highly transforming form (type III)commonly found in tumors of immunocompromised hosts, suggesting that immunepressure contributed to the outcome of the infection. These results point to animportant role for lytic <span class="annotated ">EBV</span> infection in the development of B cell lymphomasin the context of an active host immune response.
<h3>20980513</h3>R5 <span class="annotated ">HIV</span> env and <span class="annotated ">vesicular stomatitis virus</span> G protein cooperate to mediate fusion to naive CD4+ T Cells. <br><br>Naive CD4(4) T cells are resistant to both <span class="annotated ">HIV</span> R5 env and <span class="annotated ">vesicular stomatitis virus</span> G protein (<span class="annotated ">VSV</span>-G)-mediated fusion. However, <span class="match annotated ">viral</span> particles carrying both <span class="annotated ">HIV</span> R5 env and <span class="annotated ">VSV</span>-G infect naive cells by an unexplained mechanism. We show that <span class="annotated ">VSV</span>-G-pseudotyped <span class="annotated ">virus</span> cannot fuse to unstimulated cells because the <span class="match annotated ">viral</span> particles cannot be endocytosed. However, virions carrying both <span class="annotated ">HIV</span> R5 env and <span class="annotated ">VSV</span>-G can fuse because CD4 binding allows <span class="match annotated ">viral</span> uptake. Our findings reveal a unique mechanism by which R5 <span class="annotated ">HIV</span> env and <span class="annotated ">VSV</span>-G cooperate to allow entry to naive CD4(+) T cells, providing a tool to target naive CD4(+) T cells with R5 <span class="annotated ">HIV</span> to study <span class="annotated ">HIV</span> coreceptor signaling and latency. 
</body></html>