
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>viruses</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr><h3>20863853</h3>Detection and quantitation of infectious <span class="unannotated ">human</span> adenoviruses and <span class="annotated ">JC polyomaviruses</span> in water by immunofluorescence assay. <br><br>Human adenoviruses (HAdV) and <span class="annotated ">JC polyomaviruses</span> (<span class="annotated ">JCPyV</span>) have been proposed as markers of fecal/urine contamination of <span class="annotated ">human</span> origin. An indirect immunofluorescence assay has been developed to quantify infectious <span class="annotated ">human adenoviruses types 2</span><span class="annotated ">human adenoviruses types</span> 2 and 41 and <span class="annotated ">JC polyomaviruses strain Mad-4</span> in water samples. The immunofluorescence assay was compared with other quantitative techniques used commonly such as plaque assay, tissue culture infectious dose-50 and quantitative PCR (qPCR). The immunofluorescence assays showed to be specific for the detection of infectious <span class="match unannotated ">viruses</span>, obtaining negative results when UV or heat-inactivated <span class="match unannotated ">viruses</span> were analyzed. The assays required less time and showed higher sensitivity for the detection of infectious <span class="unannotated ">viral</span> particles than other cell culture techniques (1 log(10) more) evaluated. River water samples spiked previously with <span class="unannotated ">human</span> adenoviruses and raw sewage samples were also analyzed using the proposed immunofluorescence assay as well as by qPCR. The results show quantitations with 2 log(10) reduction in the numbers of infectious <span class="match unannotated ">viruses</span> compared with the number of genome copies detected by qPCR. The immunofluorescence assay developed is fast, sensitive, specific, and a standardizable technique for the quantitation and detection of infectious <span class="match unannotated ">viruses</span> in water samples. 
<h3>20923688</h3>Universal primers for rapid detection of hytrosaviruses.<br><br>Hytrosaviridae is a proposed virus family encompassing <span class="match unannotated ">viruses</span> that cause salivary gland hypertrophy (SGH) syndrome in infected insects and reduce the fertility in their dipteran insect hosts. They contain a large, double stranded DNA genome of 120-190 kbp. To date, these <span class="match unannotated ">viruses</span> have been detected only in adult Diptera. These include hytrosaviruses detected in various tsetse <span class="LINNAEUS_only ">fly</span> species (Glossina spp.), the <span class="annotated ">narcissus bulb fly</span> <span class="annotated ">Merodon equestris</span> and the <span class="annotated ">house fly</span> <span class="annotated ">Musca domestica</span>. The limited number of hytrosaviruses reported to date may be a reflection of the frequent absence of external symptoms in infected adult <span class="LINNAEUS_only ">flies</span> and the fact that the virus does not cause rapid mortality. Based on the complete genome sequence of <span class="annotated ">Glossinia pallidipes</span> (GpSGHV) and <span class="annotated ">Musca domestica</span> (MdSGHV) salivary gland hypertrophy <span class="match unannotated ">viruses</span>, a PCR based methodology was developed to detect the <span class="match unannotated ">viruses</span> in these species. To be able to detect hytrosaviruses in other Diptera, five degenerate primer pairs were designed and tested on GpSGHV and MdSGHV DNA using gradient PCR with annealing temperatures from 37 to 61^0C. Two pairs of primers were selected from p74, two pairs from PIF-1 and one pair from ODV-e66 homologous proteins. Four primer pairs generated a virus specific PCR product on both MdSGHV and GpSGHV at all tested annealing temperatures, while the ODV-e66 based primers did not generate a virus specific product with annealing temperatures higher that 47^0C. No non-specific PCR product was found when using genomic DNA of infected <span class="LINNAEUS_only ">flies</span> as template DNA. These results offer new sets of primers that could be used to detect hytrosaviruses in other insects. 
<h3>20923689</h3>Simultaneous virus-specific detection of the two <span class="annotated ">cassava</span> brown streak-associated <span class="match unannotated ">viruses</span> by RT-PCR reveals wide distribution in East Africa, mixed infections, and infections in <span class="annotated ">Manihot glaziovii</span>. <br><br>The expanding <span class="annotated ">cassava</span> brown streak disease (CBSD) epidemic in East Africa is caused by two ipomoviruses (genus Ipomovirus; Potyviridae), namely, <span class="annotated ">Cassava brown streak virus</span> (<span class="annotated ">CBSV</span>), and <span class="annotated ">Ugandan cassava brown streak virus</span> (<span class="annotated ">UCBSV</span>) that was described recently. <span class="unannotated ">A</span> reverse transcription polymerase chain reaction (RT-PCR) based diagnostic method was developed in this study for simultaneous virus-specific detection of the two <span class="match unannotated ">viruses</span>. Results showed that <span class="annotated ">CBSV</span> and <span class="annotated ">UCBSV</span> are distributed widely in the highlands (> 1000 m above the sea level) of the Lake Victoria zone in Uganda and Tanzania and also in the Indian Ocean costal lowlands of Tanzania. Isolates of <span class="annotated ">UCBSV</span> from the Lake Victoria zone were placed to two phylogenetic clusters in accordance with their origin in Uganda or Tanzania, respectively. Mixed infections with <span class="annotated ">CBSV</span> and <span class="annotated ">UCBSV</span> were detected in many <span class="annotated ">cassava</span> plants in the areas surveyed. <span class="annotated ">CBSV</span> was also detected in the perennial species <span class="annotated ">Manihot glaziovii</span> (DNA-barcoded in this study) in Tanzania, which revealed the first virus reservoir other than <span class="annotated ">cassava</span>. The method for detection of <span class="annotated ">CBSV</span> and <span class="annotated ">UCBSV</span> described in this study has important applications for plant quarantine, resistance breeding of <span class="annotated ">cassava</span>, and studies on epidemiology and control of CBSD in East Africa. 
<h3>20933017</h3>Adaptation of a Madin-Darby <span class="annotated ">canine</span> kidney cell line to suspension growth in serum-free media and comparison of its ability to produce avian <span class="unannotated ">influenza virus</span> to Vero and BHK21 cell lines. <br><br>Madin-Darby <span class="annotated ">canine</span> kidney (MDCK) cells are currently considered for influenza vaccine manufacturing. <span class="unannotated ">A</span> drawback of these cells is their anchorage dependent growth, which greatly complicates process scale-up. In this paper a novel MDCK cell line (MDCK-SFS) is described that grows efficiently in suspension and retained high expression levels of both alpha-2,6 and alpha-2,3 sialic acid receptors, which bind preferably to <span class="unannotated ">human</span> and avian influenza <span class="match unannotated ">viruses</span>, respectively. The production of avian <span class="unannotated ">influenza virus</span> by BHK21, Vero and MDCK-SFS cell lines was compared. Although BHK21 cells consisted of two populations, one of which lacks the alpha-2,3 receptor, they supported the replication of two influenza strains to high titres. However, BHK21 cells are generally not applicable for influenza production since they supported the replication of six further strains poorly. MDCK-SFS cells yielded the highest infectious virus titres and virus genome equivalent concentration for five of the eight influenza strains analyzed and the highest hemagglutination activity for all eight virus strains. Taken together with their suitability for suspension growth this makes the MDCK-SFS cell line potentially useful for large scale <span class="unannotated ">influenza virus</span> production. 
<h3>20962076</h3>Neutralizing antibody-resistant <span class="annotated ">hepatitis C virus</span> cell-to-cell transmission.<br><br><span class="annotated ">Hepatitis C virus</span> (<span class="annotated ">HCV</span>) can initiate infection by cell-free particle and cell-cell contact-dependent transmission. In this study we use a novel infectious coculture system to examine these alternative modes of infection. Cell-to-cell transmission is relatively resistant to anti-HCV glycoprotein monoclonal antibodies and polyclonal immunoglobulin isolated from infected individuals, providing an effective strategy for escaping host humoral immune responses. Chimeric <span class="match unannotated ">viruses</span> expressing the structural proteins representing the seven major <span class="annotated ">HCV</span> genotypes demonstrate neutralizing antibody-resistant cell-to-cell transmission. <span class="annotated ">HCV</span> entry is a multistep process involving numerous receptors. In this study we demonstrate that, in contrast to earlier reports, CD81 and the tight-junction components claudin-1 and occludin are all essential for both cell-free and cell-to-cell <span class="unannotated ">viral</span> transmission. However, scavenger receptor BI (SR-BI) has a more prominent role in cell-to-cell transmission of the virus, with SR-BI-specific antibodies and small-molecule inhibitors showing preferential inhibition of this infection route. These observations highlight the importance of targeting host cell receptors, in particular SR-BI, to control <span class="unannotated ">viral</span> infection and spread in the liver. 
<h3>20962090</h3><span class="annotated ">Marek's disease virus type 1</span> microRNA miR-M3 suppresses cisplatin-induced apoptosis by targeting Smad2 of the transforming growth factor beta signal pathway. <br><br>Viruses cause about 15% of the cancers that are still the leading causes of <span class="annotated ">human</span> mortality. The discovery of <span class="unannotated ">viral</span> oncogenes has enhanced our understanding of <span class="unannotated ">viral</span> oncogenesis. However, the underlying molecular mechanisms of virus-induced cancers are complex and require further investigation. The present study has attempted to investigate the effects of the microRNAs (miRNAs) encoded by <span class="annotated ">Marek's disease virus 1</span> (<span class="annotated ">MDV1</span>), a <span class="unannotated ">chicken</span> herpesvirus causing acute T-cell lymphomas and solid visceral tumors in <span class="annotated ">chickens</span>, on anti-cancer drug-induced apoptosis and identify the targets of the miRNAs. The results showed that of the total 14 miRNAs encoded by <span class="annotated ">MDV1</span>, <span class="annotated ">MDV1</span>-miR-M3 significantly promoted cell survival under treatment with cisplatin, a widely used chemotherapy drug. <span class="annotated ">MDV1</span>-miR-M3 suppressed cisplatin-induced apoptosis by directly downregulating expression at the protein but not the mRNA level of Smad2, a critical component in the transforming growth factor Beta signal pathway. Our data suggest that latent/oncogenic <span class="match unannotated ">viruses</span> may encode miRNAs to directly target cellular factors involved in antiviral processes including apoptosis, thus proactively creating a cellular environment beneficial to <span class="unannotated ">viral</span> latency and oncogenesis. Furthermore, the knowledge of the apoptosis resistance conferred by <span class="unannotated ">viral</span> miRNAs has great practical implications for improving the efficacy of chemotherapies for treating cancers, especially those induced by oncogenic <span class="match unannotated ">viruses</span>. 
<h3>20962091</h3>Macaque long-term nonprogressors resist superinfection with multiple CD8+ T cell escape variants of <span class="annotated ">simian immunodeficiency virus</span>. <br><br><span class="annotated ">Human immunodeficiency virus</span> (<span class="annotated ">HIV</span>)-positive individuals can be superinfected with different virus strains. Individuals who control an initial <span class="annotated ">HIV</span> infection are therefore still at risk for subsequent infection with divergent <span class="match unannotated ">viruses</span>, but the barriers to such superinfection remain unclear. Here we tested long-term nonprogressors' (LTNPs') susceptibility to superinfection using <span class="annotated ">Indian rhesus macaques</span> that express the major histocompatibility complex <span class="unannotated ">class I</span> (MHC-I) allele Mamu-B 17, which is associated with control of the pathogenic AIDS virus <span class="annotated ">SIVmac239</span>. The Mamu-B 17-restricted CD8(+) T cell repertoire is focused almost entirely on 5 epitopes. We engineered a series of <span class="annotated ">SIVmac239</span> variants bearing mutations in 3, 4, or all 5 of these epitopes and used them to serially challenge 2 Mamu-B 17-positive LTNPs. None of the escape variants caused breakthrough replication in LTNPs, although they readily infected Mamu-B 17-negative naive macaques. In vitro competing coculture assays and examination of <span class="unannotated ">viral</span> evolution in hosts lacking Mamu-B 17 suggested that the mutant <span class="match unannotated ">viruses</span> had negligible defects in replicative fitness. Both LTNPs maintained robust immune responses, including <span class="annotated ">simian immunodeficiency virus</span> (<span class="annotated ">SIV</span>)-specific CD8(+) and CD4(+) T cells and neutralizing antibodies. Our results suggest that escape mutations in epitopes bound by &quot;protective&quot; MHC-I molecules may not be sufficient to establish superinfection in LTNPs. 
<h3>20962094</h3>Yeast-elicited cross-reactive antibodies to <span class="annotated ">HIV</span> Env glycans efficiently neutralize virions expressing exclusively high-mannose N-linked glycans. <br><br>The <span class="annotated ">HIV</span> envelope (Env) protein uses a dense coat of glycans to mask conserved domains and evade host humoral immune responses. The broadly neutralizing antibody 2G12, which binds a specific cluster of high-mannose glycans on <span class="annotated ">HIV</span> Env, shows that the glycan shield can also serve as a target for neutralizing antibodies. We have described a triple mutant <span class="annotated ">Saccharomyces cerevisiae</span> strain that expresses high-mannose glycoproteins that bind to 2G12. When used to immunize <span class="annotated ">rabbits</span>, this <span class="LINNAEUS_only ">yeast</span> elicits antibodies that bind to gp120-associated glycans but fail to neutralize virus. Here we sought to determine the reason for these discordant results. Affinity purification of sera over columns conjugated with three 2G12-reactive <span class="LINNAEUS_only ">yeast</span> glycoproteins showed that these proteins could adsorb 80% of the antibodies that bind to gp120 glycans. Despite binding to monomeric gp120, these mannose-specific antibodies failed to bind cell surface-expressed trimeric Env. However, when Env was expressed in the presence of the mannosidase inhibitor kifunensine to force retention of high-mannose glycans at all sites, the purified antibodies gained the abilities to bind trimeric Env and to strongly and broadly neutralize <span class="match unannotated ">viruses</span> produced under these conditions. Combined, these data show that the triple mutant <span class="LINNAEUS_only ">yeast</span> strain elicits antibodies that bind to high-mannose glycans presented on the <span class="annotated ">HIV</span> envelope, but only when they are displayed in a manner not found on native Env trimers. This implies that the underlying structure of the protein scaffold used to present the high-mannose glycans may be critical to allow elicitation of antibodies that recognize trimeric Env and neutralize virus. 
<h3>20962099</h3>Replicative and transcriptional activities of <span class="annotated ">hepatitis B virus</span> in <span class="LINNAEUS_only ">patients</span> coinfected with <span class="unannotated ">hepatitis</span> B and <span class="annotated ">hepatitis delta viruses</span><span class="match annotated ">viruses</span>. <br><br><span class="annotated ">Hepatitis B virus</span> (<span class="annotated ">HBV</span>) and <span class="annotated ">hepatitis delta virus</span> (<span class="annotated ">HDV</span>) interplay was investigated by examining liver and serum samples from 21 coinfected and 22 <span class="annotated ">HBV</span>-monoinfected <span class="LINNAEUS_only ">patients</span> with chronic liver disease. Different real-time PCR assays were applied to evaluate intrahepatic amounts of <span class="annotated ">HBV</span> DNA, covalently closed circular DNA (cccDNA), pregenomic RNA (pgRNA), pre-S/S RNAs, and <span class="annotated ">HDV</span> RNA. Besides <span class="annotated ">HBV</span> DNA and <span class="annotated ">HDV</span> RNA levels, HBsAg concentrations in the sera were also determined. <span class="annotated ">HDV</span>-coinfected cases showed significantly lower median levels of serum <span class="annotated ">HBV</span> DNA (-5 log), intrahepatic relaxed-circular DNA (-2 log), and cccDNA (-2 log) than those of <span class="annotated ">HBV</span>-monoinfected cases. Interestingly, pgRNA and pre-S/S RNA amounts were significantly lower (both -1 log) in <span class="annotated ">HDV</span>-positive <span class="LINNAEUS_only ">patients</span>, whereas serum HBsAg concentrations were comparable between the two <span class="LINNAEUS_only ">patient</span> groups. Pre-S/S RNA and HBsAg amounts per cccDNA molecule were higher in <span class="annotated ">HDV</span>-positive <span class="LINNAEUS_only ">patients</span> (3-fold and 1 log, respectively), showing that <span class="annotated ">HBV</span> replication was reduced, whereas synthesis of envelope proteins was not specifically decreased. The ratios of cccDNA to intracellular total <span class="annotated ">HBV</span> DNA showed a larger proportion of cccDNA molecules in <span class="annotated ">HDV</span>-positive cases. For these <span class="LINNAEUS_only ">patients</span>, both intrahepatic and serum <span class="annotated ">HDV</span> RNA amounts were associated with cccDNA but not with HBsAg or <span class="annotated ">HBV</span> DNA levels. Finally, <span class="annotated ">HBV</span> genomes with large deletions in the basal core promoter/precore region were detected in 5/21 <span class="annotated ">HDV</span>-positive <span class="LINNAEUS_only ">patients</span> but in no <span class="annotated ">HDV</span>-negative <span class="LINNAEUS_only ">patients</span> and were associated with lower viremia levels. These findings provide significant information about the interference exerted by <span class="annotated ">HDV</span> on <span class="annotated ">HBV</span> replication and transcription activities in the <span class="annotated ">human</span> liver. 
<h3>20980498</h3>Template recognition mechanisms by replicase proteins differ between bipartite positive-strand genomic RNAs of a plant virus. <br><br>Recognition of RNA templates by <span class="unannotated ">viral</span> replicase proteins is one of the key steps in the replication process of all RNA <span class="match unannotated ">viruses</span>. However, the mechanisms underlying this phenomenon, including primary RNA elements that are recognized by the <span class="unannotated ">viral</span> replicase proteins, are not well understood. Here, we used aptamer pulldown assays with membrane fractionation and protein-RNA coimmunoprecipitation in a cell-free <span class="unannotated ">viral</span> translation/replication system to investigate how <span class="unannotated ">viral</span> replicase proteins recognize the bipartite genomic RNAs of the <span class="annotated ">Red clover necrotic mosaic virus</span> (<span class="annotated ">RCNMV</span>). <span class="annotated ">RCNMV</span> replicase proteins bound specifically to a Y-shaped RNA element (YRE) located in the 3' untranslated region (UTR) of RNA2, which also interacted with the 480-kDa replicase complexes that contain <span class="unannotated ">viral</span> and host proteins. The replicase-YRE interaction recruited RNA2 to the membrane fraction. Conversely, RNA1 fragments failed to interact with the replicase proteins supplied in trans. The results of protein-RNA coimmunoprecipitation assays suggest that RNA1 interacts with the replicase proteins coupled with their translation. Thus, the initial template recognition mechanisms employed by the replicase differ between <span class="annotated ">RCNMV</span> bipartite genomic RNAs and RNA elements are primary determinants of the differential replication mechanism. 
<h3>20980502</h3>Cytomegalovirus microRNA expression is tissue specific and is associated with persistence. <br><br>MicroRNAs (miRNAs) are a class of small noncoding RNAs involved in posttranscriptional regulation. miRNAs are utilized in organisms ranging from plants to higher mammals, and data have shown that DNA <span class="match unannotated ">viruses</span> also use this method for host and <span class="unannotated ">viral</span> gene regulation. Here, we report the sequencing of the small RNAs in <span class="annotated ">rat cytomegalovirus</span> (<span class="annotated ">RCMV</span>)-infected fibroblasts and persistently infected salivary glands. We identified 24 unique miRNAs that mapped to hairpin structures found within the <span class="unannotated ">viral</span> genome. While most miRNAs were detected in both samples, four were detected exclusively in the infected fibroblasts and two were specific for the infected salivary glands. The <span class="annotated ">RCMV</span> miRNAs are distributed across the <span class="unannotated ">viral</span> genome on both the positive and negative strands, with clusters of miRNAs at a number of locations, including near <span class="unannotated ">viral</span> genes r1 and r111. The <span class="annotated ">RCMV</span> miRNAs have a genomic positional orientation similar to that of the miRNAs described for <span class="annotated ">mouse cytomegalovirus</span>, but they do not share any substantial sequence conservation. Similar to other reported miRNAs, the <span class="annotated ">RCMV</span> miRNAs had considerable variation at their 3' and 5' ends. Interestingly, we found a number of specific examples of differential isoform usage between the fibroblast and salivary gland samples. We determined by real-time PCR that expression of the <span class="annotated ">RCMV</span> miRNA miR-r111.1-2 is highly expressed in the salivary glands and that miR-R87-1 is expressed in most tissues during the acute infection phase. Our study identified the miRNAs expressed by <span class="annotated ">RCMV</span> in vitro and in vivo and demonstrated that expression is tissue specific and associated with a stage of <span class="unannotated ">viral</span> infection. 
<h3>20980507</h3>Successful vaccination strategies that protect aged <span class="annotated ">mice</span> from lethal challenge from <span class="annotated ">influenza virus</span> and heterologous <span class="annotated ">severe acute respiratory syndrome coronavirus</span>. <br><br>Newly emerging <span class="match unannotated ">viruses</span> often circulate as a heterogeneous swarm in wild animal reservoirs prior to their emergence in <span class="annotated ">humans</span>, and their antigenic identities are often unknown until an outbreak situation. The newly emerging <span class="annotated ">severe acute respiratory syndrome coronavirus</span> (<span class="annotated ">SARS-CoV</span>) and reemerging <span class="annotated ">influenza virus</span> cause disproportionate disease in the aged, who are also notoriously difficult to successfully vaccinate, likely due to immunosenescence. To protect against future emerging strains, vaccine platforms should induce broad cross-reactive immunity that is sufficient to protect from homologous and heterologous challenge in all ages. From initial studies, we hypothesized that attenuated <span class="annotated ">Venezuelan equine encephalitis virus</span> (<span class="annotated ">VEE</span>) replicon particle (VRP) vaccine glycoproteins mediated vaccine failure in the aged. We then compared the efficacies of vaccines bearing attenuated (VRP(3014)) or wild-type <span class="annotated ">VEE</span> glycoproteins (VRP(3000)) in young and aged <span class="annotated ">mice</span> within novel models of severe <span class="annotated ">SARS-CoV</span> pathogenesis. Aged animals receiving VRP(3000)-based vaccines were protected from <span class="annotated ">SARS-CoV</span> disease, while animals receiving the VRP(3014)-based vaccines were not. The superior protection for the aged observed with VRP(3000)-based vaccines was confirmed in a lethal <span class="annotated ">influenza virus</span> challenge model. While the VRP(3000) vaccine's immune responses in the aged were sufficient to protect against lethal homologous and heterologous challenge, our data suggest that innate defects within the VRP(3014) platform mediate vaccine failure. Exploration into the mechanism(s) of successful vaccination in the immunosenescent should aid in the development of successful vaccine strategies for other <span class="unannotated ">viral</span> diseases disproportionately affecting the elderly, like <span class="annotated ">West Nile virus</span>, <span class="annotated ">influenza virus</span>, norovirus, or other emerging <span class="match unannotated ">viruses</span> of the future. 
<h3>20980515</h3>Reverse genetics generation of chimeric infectious Junin/Lassa virus is dependent on interaction of homologous glycoprotein stable signal peptide and G2 cytoplasmic domains. <br><br>The Arenaviridae are a diverse and globally distributed collection of <span class="match unannotated ">viruses</span> that are maintained primarily by <span class="LINNAEUS_only ">rodent</span> reservoirs. <span class="annotated ">Junin virus</span> (<span class="annotated ">JUNV</span>) and <span class="annotated ">Lassa virus</span> (<span class="annotated ">LASV</span>) can both cause significant outbreaks of severe and often fatal <span class="annotated ">human</span> disease throughout their respective areas of endemicity. In an effort to improve upon the existing live attenuated <span class="annotated ">JUNV</span> Candid1 vaccine, we generated a genetically homogenous stock of this virus from cDNA copies of the virus S and L segments by using a reverse genetics system. Further, these cDNAs were used in combination with <span class="annotated ">LASV</span> cDNAs to successfully generate two recombinant Candid1 <span class="annotated ">JUNV</span>/<span class="annotated ">LASV</span> chimeric <span class="match unannotated ">viruses</span> (via envelope glycoprotein [GPC] exchange). It was found that while the GPC extravirion domains were readily exchangeable, homologous stable signal peptide (SSP) and G2 transmembrane and cytoplasmic tail domains were essential for correct GPC maturation and production of infectious chimeric <span class="match unannotated ">viruses</span>. The switching of the <span class="annotated ">JUNV</span> and <span class="annotated ">LASV</span> G1/G2 ectodomains within the Candid1 vaccine background did not alter the attenuated phenotype of the vaccine strain in a lethal <span class="annotated ">mouse</span> model. These recombinant chimeric <span class="match unannotated ">viruses</span> shed light on the fundamental requirements of arenavirus GPC maturation and may serve as a strategy for the development of bivalent <span class="annotated ">JUNV</span> and <span class="annotated ">LASV</span> vaccine candidates. 
<h3>20980521</h3>Characterization of cross-reactive CD8+ T-cell recognition of HLA-A2-restricted <span class="annotated ">HIV</span>-Gag (SLYNTVATL) and <span class="annotated ">HCV</span>-NS5b (ALYDVVSKL) epitopes in individuals infected with <span class="annotated ">human immunodeficiency</span> and <span class="annotated ">hepatitis C viruses</span>. <br><br>The immunologic mechanisms underlying the faster progression of <span class="annotated ">hepatitis C virus</span> (<span class="annotated ">HCV</span>) disease in the presence of <span class="annotated ">human immunodeficiency virus</span> (<span class="annotated ">HIV</span>) coinfection are not clearly understood. T-cell cross-reactivity between <span class="annotated ">HCV</span> and <span class="annotated ">influenza virus</span>-specific epitopes has been associated with rapid progression of <span class="annotated ">HCV</span> disease (S. Urbani, B. Amadei, P. Fisicaro, M. Pilli, G. Missale, <span class="unannotated ">A</span>. Bertoletti, and C. Ferrari, J. Exp. Med. 201:675-680, 2005). We asked whether T-cell cross-reactivity between <span class="annotated ">HCV</span> and <span class="annotated ">HIV</span> could exist during <span class="annotated ">HCV</span>/<span class="annotated ">HIV</span> coinfection and affect pathogenesis. Our search for amino acid sequence homology between the <span class="annotated ">HCV</span> and <span class="annotated ">HIV</span> proteomes revealed two similar HLA-A2-restricted epitopes, <span class="annotated ">HIV</span>-Gag (SLYNTVATL [HIV-SL9]) and <span class="annotated ">HCV</span>-NS5b (ALYDVVSKL [<span class="annotated ">HCV</span>-AL9]). We found that 4 out of 20 HLA-A2-positive (HLA-A2(+)) <span class="annotated ">HIV</span>-infected individuals had CD8(+) T cells that recognized both the <span class="annotated ">HIV</span>-SL9 and <span class="annotated ">HCV</span>-AL9 epitopes. However, the AL9 epitope was generally shown to be a weak agonist. Although <span class="annotated ">HCV</span>-monoinfected individuals in our study did not show AL9-specific responses, we found that about half of <span class="annotated ">HCV</span>/<span class="annotated ">HIV</span>-coinfected individuals had dual responses to both epitopes. High dual T-cell recognition among coinfected subjects was usually due to separate T-cell populations targeting each epitope, as determined by pentamer staining. The one individual demonstrating cross-reactive T cells to both epitopes showed the most advanced degree of liver disease. In coinfected individuals, we observed a positive correlation between the magnitudes of T-cell responses to both the SL9 and the AL9 epitopes, which was also positively associated with the clinical parameter of liver damage. Thus, we find that <span class="annotated ">HIV</span> infection induces T cells that can cross-react to heterologous <span class="match unannotated ">viruses</span> or prime for T cells that are closely related in sequence. However, the induction of cross-reactive T cells may not be associated with control of disease caused by the heterologous virus. This demonstrates that degeneracy of <span class="annotated ">HIV</span>-specific T cells may play a role in the immunopathology of <span class="annotated ">HCV</span>/<span class="annotated ">HIV</span> coinfection. 
<h3>20980523</h3>T cell-mediated protection against lethal 2009 pandemic <span class="annotated ">H1N1 influenza virus</span> infection in a <span class="annotated ">mouse</span> model. <br><br>Genetic mutation and reassortment of <span class="annotated ">influenza virus</span> gene segments, in particular those of hemagglutinin (HA) and neuraminidase (NA), that lead to antigenic drift and shift are the major strategies for <span class="annotated ">influenza virus</span> to escape preexisting immunity. The most recent example of such phenomena is the first pandemic of <span class="annotated ">H1N1 influenza</span> of the 21st century, which started in 2009. Cross-reactive antibodies raised against <span class="annotated ">H1N1 viruses</span> circulating before 1930 show protective activity against the 2009 pandemic virus. Cross-reactive T-cell responses can also contribute to protection, but in vivo support of this view is lacking. To explore the protection mechanisms in vivo, we primed <span class="annotated ">mice</span> with <span class="annotated ">H1</span> and <span class="annotated ">H3 influenza virus</span> isolates and rechallenged them with a virus derived from the 2009 <span class="annotated ">H1N1 A/CA/04/09 virus</span>, named CA/E3/09. We found that priming with <span class="annotated ">influenza viruses</span> of both <span class="annotated ">H1</span> and <span class="annotated ">H3</span> homo- and heterosubtypes protected against lethal CA/E3/09 virus challenge. Convalescent-phase sera from these primed <span class="annotated ">mice</span> conferred no neutralization activity in vitro and no protection in vivo. However, T-cell depletion studies suggested that both CD4 and CD8 T cells contributed to the protection. Taken together, these results indicate that cross-reactive T cells established after initial priming with distally related <span class="match unannotated ">viruses</span> can be a vital component for prevention of disease and control of pandemic <span class="annotated ">H1N1 influenza virus</span> infection. Our results highlight the importance of establishing cross-reactive T-cell responses for protecting against existing or newly emerging pandemic <span class="annotated ">influenza viruses</span>. 
<h3>21029750</h3>Detection of <span class="annotated ">infectious myonecrosis virus</span> using monoclonal antibody specific to N and C fragments of the capsid protein expressed heterologously. <br><br>The gene encoding the capsid protein in ORF1 of the genome of <span class="annotated ">infectious myonecrosis virus</span> (<span class="annotated ">IMNV</span>) (GenBank AY570982) was amplified into three parts named CP-N (nucleotides 2248-3045), CP-I (nucleotides 3046-3954) and CP-C (nucleotides 3955-4953). The CP-N fragment was inserted into expression vector pTYB1 while CP-I and CP-C were each inserted into expression vector pGEX-6P-1 for transformation of <span class="annotated ">BL21 E. coli</span> strain. After induction, intein-CP-N (84 kDa), glutathione-S-transferase (GST)-CP-I (60 kDa) and GST-CP-C (62 kDa) fusion proteins were produced. They were separated by SDS-PAGE and electroeluted before immunization of <span class="annotated ">Swiss mice</span> for monoclonal antibody (MAb) production. Two MAbs specific to CP-N and one MAb specific to CP-C were selected for use for detection of natural <span class="annotated ">IMNV</span> infections in <span class="annotated ">Penaeus vannamei</span> by dot blotting, Western blotting and immunohistochemistry. There was no cross-reaction with <span class="LINNAEUS_only ">shrimp</span> tissues or common <span class="LINNAEUS_only ">shrimp</span> <span class="match unannotated ">viruses</span> including <span class="annotated ">white spot syndrome virus</span> (<span class="annotated ">WSSV</span>), <span class="annotated ">yellow head virus</span> (<span class="annotated ">YHV</span>), <span class="annotated ">Taura syndrome virus</span> (<span class="annotated ">TSV</span>), <span class="annotated ">Penaeus monodon nucleopolyhedrovirus</span> (<span class="annotated ">PemoNPV</span>), <span class="annotated ">Penaeus stylirostris densovirus</span> (<span class="annotated ">PstDNV</span>) and <span class="annotated ">Penaeus monodon densovirus</span> (<span class="annotated ">PmDNV</span>). The detection sensitivities of the MAbs were approximately 6 fmol/spot of purified recombinant intein-CP-N protein and 8 fmol/spot of GST-CP-C as determined by dot blotting. <span class="unannotated ">A</span> combination of all three MAbs resulted in a twofold increase in sensitivity over use of any single MAb. However, this sensitivity was approximately 10 times lower than that of one-step RT-PCR using the same sample. Immunohistochemical analysis using MAbs specific to CP-N and CP-C in <span class="annotated ">IMNV</span>-infected <span class="LINNAEUS_only ">shrimp</span> revealed intense staining patterns in muscles, the lymphoid organ, gills, the heart, hemocytes and connective tissue. 
<h3>21029751</h3>Real time PCR method for simultaneous detection, quantitation and differentiation of capripoxviruses. <br><br>The genus Capripoxvirus (CaPV) comprises three members namely, <span class="annotated ">sheep poxvirus</span> (<span class="annotated ">SPPV</span>), <span class="annotated ">goat poxvirus</span> (<span class="annotated ">GTPV</span>) and <span class="annotated ">lumpy skin disease virus</span> (<span class="annotated ">LSDV</span>) affecting <span class="annotated ">sheep</span>, <span class="annotated ">goats</span> and <span class="annotated ">cattle</span>, respectively. CaPV infections produce similar symptoms in <span class="annotated ">sheep</span> and <span class="annotated ">goats</span>, and the three <span class="match unannotated ">viruses</span> cannot be distinguished serologically. Since there are conflicting opinions regarding the host specificity of CaPVs, particularly for <span class="annotated ">goatpox</span> and <span class="annotated ">sheeppox viruses</span>, the development of rapid genotyping tools will facilitate more accurate disease diagnosis and surveillance for better management of capripox outbreaks. This paper describes a species-specific, real time polymerase chain reaction (PCR), based on unique molecular markers that were found in the G-protein-coupled chemokine receptor (GPCR) gene sequences of CaPVs, that uses dual hybridization probes for their simultaneous detection, quantitation and genotyping. The assay can differentiate between CaPV strains based on differences in the melting point temperature (Tm) obtained after fluorescence melting curve analysis (FMCA). It is highly sensitive and presents low intra- and inter-run variation. This real time PCR assay will make a significant contribution to CaPV diagnosis and to the better understanding of the epidemiology of CaPVs by enabling rapid genotyping and gene-based classification of <span class="unannotated ">viral</span> strains and unequivocal identification of isolates. 
<h3>21034776</h3>A versatile vector for the production of pseudotyped <span class="match unannotated ">viruses</span> expressing gp120 antigens from different clades of primary <span class="annotated ">HIV-1</span> isolates. <br><br>A novel <span class="annotated ">HIV-1</span> Env expression vector (SF162-Z) was developed by introducing two new cloning sites on the backbone of an existing vector that produces a full length Env from <span class="annotated ">HIV-1</span> SF162 isolate. These sites facilitate the swapping of the gp120 portion of the SF162 Env with matching gp120 antigens from <span class="annotated ">HIV-1</span> isolates of different genetic clades. Final production of functional pseudotyped <span class="match unannotated ">viruses</span> will express chimeric Env antigens, including gp41 of the parental SF162 and gp120 from other primary isolates. This system is useful for testing the neutralizing sensitivity of partial env gene products frequently identified in <span class="unannotated ">viral</span> quasi species in <span class="LINNAEUS_only ">patients</span> infected with <span class="annotated ">HIV</span> or when only partial gp120 gene products are available. 
<h3>21039560</h3>The second face of a known player: <span class="LINNAEUS_only ">Arabidopsis</span> silencing suppressor AtXRN4 acts organ-specifically. <br><br>Plant <span class="match unannotated ">viruses</span> exploit the symplastic transport pathway provided by plasmodesmata by encoding for specialized movement proteins, which interact with host factors to enable <span class="unannotated ">viral</span> intracellular and intercellular spread. Stable expression of the <span class="annotated ">Potato leaf roll virus</span> movement protein MP17 in <span class="LINNAEUS_only ">Arabidopsis</span> results in a carbohydrate export block and stunted growth. To identify host factors essential for <span class="unannotated ">viral</span> infection, we screened a progeny population of EMS (ethyl methanesulfonate)-mutagenized <span class="LINNAEUS_only ">Arabidopsis</span> expressing a MP17:GFP fusion for suppressor mutants with restored wild type-like phenotype. Two suppressor mutants showed decreased susceptibility against <span class="annotated ">Turnip mosaic virus</span> and post-transcriptional silencing of MP17:GFP RNA in source leaves. Map based cloning identified in both lines mutations in XRN4 (Exoribonuclease 4), which was previously described as a suppressor of transgene silencing in source leaves. Importantly, silencing of MP17:GFP was not present in cotyledons and roots of the two suppressor mutants, which was confirmed in a third xrn4 T-DNA knock out line. Subsequent analysis of MP17:GFP transcript stability in xrn2 and xrn3 mutants indicated an essential role of AtXRN2 for silencing suppression in roots/cotyledons while AtXRN3 appears to act similar to AtXRN4 in source leaves, only. Overall, these findings point towards an organ-specific regulation of gene silencing in <span class="LINNAEUS_only ">Arabidopsis</span>. 
<h3>21371657</h3>Population biological principles of drug-resistance evolution in infectious diseases. <br><br>The emergence of resistant pathogens in response to selection pressure by drugs and their possible disappearance when drug use is discontinued are evolutionary processes common to many pathogens. Population biological models have been used to study the dynamics of resistance in <span class="match unannotated ">viruses</span>, bacteria, and eukaryotic microparasites both at the level of the individual treated host and of the treated host population. Despite the existence of generic features that underlie such evolutionary dynamics, different conclusions have been reached about the key factors affecting the rate of resistance evolution and how to best use drugs to minimise the risk of generating high levels of resistance. Improved understanding of generic versus specific population biological aspects will help to translate results between different studies, and allow development of a more rational basis for sustainable drug use than exists at present. 
<h3>21429803</h3>European guidelines on the clinical management of <span class="annotated ">HIV-1</span> tropism testing.<br><br>Viral tropism is the ability of <span class="match unannotated ">viruses</span> to enter and infect specific host cells and is based on the ability of <span class="match unannotated ">viruses</span> to bind to receptors on those cells. Testing for <span class="annotated ">HIV</span> tropism is recommended before prescribing a chemokine receptor blocker. In most European countries, <span class="annotated ">HIV</span> tropism is identified with tropism phenotype testing. New data support genotype analysis of the <span class="annotated ">HIV</span> third hypervariable loop (V3) for the identification of tropism. The European Consensus Group on clinical management of tropism testing was established to make recommendations to clinicians and clinical virologists. The panel recommends <span class="annotated ">HIV</span>-tropism testing for the following groups: drug-naive <span class="LINNAEUS_only ">patients</span> in whom toxic effects are anticipated or for whom few treatment options are available; <span class="LINNAEUS_only ">patients</span> who have poor tolerability to or toxic effects from current treatment or who have CNS pathology; and <span class="LINNAEUS_only ">patients</span> for whom therapy has failed and a change in treatment is considered. In general, an enhanced sensitivity Trofile assay and V3 population genotyping are the recommended methods. Genotypic methods are anticipated to be used more frequently in the clinical setting because of their greater accessibility, lower cost, and faster turnaround time than other methods. For the interpretation of V3 loop genotyping, clinically validated systems should be used when possible. Laboratories doing <span class="annotated ">HIV</span> tropism tests should have adequate quality assurance measures. Similarly, close collaboration between <span class="annotated ">HIV</span> clinicians and virologists is needed to ensure adequate diagnostic and treatment decisions. 
<h3>21435708</h3>Viral pneumonia.<br><br>About 200 million cases of <span class="unannotated ">viral</span> community-acquired pneumonia occur every year-100 million in <span class="LINNAEUS_only ">children</span> and 100 million in adults. Molecular diagnostic tests have greatly increased our understanding of the role of <span class="match unannotated ">viruses</span> in pneumonia, and findings indicate that the incidence of <span class="unannotated ">viral</span> pneumonia has been underestimated. In <span class="LINNAEUS_only ">children</span>, <span class="annotated ">respiratory syncytial virus</span>, rhinovirus, <span class="annotated ">human metapneumovirus</span>, <span class="annotated ">human bocavirus</span>, and parainfluenza <span class="match unannotated ">viruses</span> are the agents identified most frequently in both developed and developing countries. Dual <span class="unannotated ">viral</span> infections are common, and a third of <span class="LINNAEUS_only ">children</span> have evidence of viral-bacterial co-infection. In adults, <span class="match unannotated ">viruses</span> are the putative causative agents in a third of cases of community-acquired pneumonia, in particular <span class="annotated ">influenza viruses</span>, rhinoviruses, and coronaviruses. Bacteria continue to have a predominant role in adults with pneumonia. Presence of <span class="unannotated ">viral</span> epidemics in the community, patient's age, speed of onset of illness, symptoms, biomarkers, radiographic changes, and response to treatment can help differentiate <span class="unannotated ">viral</span> from bacterial pneumonia. However, no clinical algorithm exists that will distinguish clearly the cause of pneumonia. No clear consensus has been reached about whether <span class="LINNAEUS_only ">patients</span> with obvious <span class="unannotated ">viral</span> community-acquired pneumonia need to be treated with antibiotics. Apart from neuraminidase inhibitors for pneumonia caused by <span class="annotated ">influenza viruses</span>, there is no clear role for use of specific antivirals to treat <span class="unannotated ">viral</span> community-acquired pneumonia. Influenza vaccines are the only available specific preventive measures. Further studies are needed to better understand the cause and pathogenesis of community-acquired pneumonia. Furthermore, regional differences in cause of pneumonia should be investigated, in particular to obtain more data from developing countries. 
<h3>21840413</h3>The effects of dsRNA mycoviruses on growth and <span class="LINNAEUS_only ">murine</span> virulence of <span class="annotated ">Aspergillus fumigatus</span>. <br><br>Some isolates of the opportunistic <span class="annotated ">human</span> pathogenic fungus <span class="annotated ">Aspergillus fumigatus</span> are known to be infected with mycoviruses. The dsRNA genomes of two of these mycoviruses, which include a chrysovirus and a partitivirus, have been completely sequenced and an RT-PCR assay for the <span class="match unannotated ">viruses</span> has been developed. Through curing virus-infected <span class="annotated ">A. fumigatus</span> isolates by cycloheximide treatment and transfecting virus-free isolates with purified virus, as checked by RT-PCR, isogenic virus-free and virus-infected lines of the fungus were generated whose phenotypes and growth have been directly compared. Mycovirus infection of <span class="annotated ">A. fumigatus</span> with either the chrysovirus or the partitivirus resulted in significant aberrant phenotypic alterations and attenuation of growth of the fungus but had no effect on susceptibility to common antifungals. Chrysovirus infection of <span class="annotated ">A. fumigatus</span> caused no significant alterations to <span class="LINNAEUS_only ">murine</span> pathogenicity. 
<h3>20951165</h3>The validation of a real-time RT-PCR assay which detects <span class="annotated ">influenza A</span> and types simultaneously for <span class="annotated ">influenza A H1N1</span> (2009) and oseltamivir-resistant (H275Y) <span class="annotated ">influenza A H1N1</span> (2009). <br><br><span class="annotated ">Influenza A H1N1</span> (2009) was declared by the World Health Organisation (WHO) as the first influenza pandemic of the 21st century. Rapid detection of <span class="annotated ">influenza A</span> and differentiation of <span class="annotated ">influenza A H1N1</span> (2009) and seasonal <span class="annotated ">influenza A</span> is beneficial. In addition the rapid detection of antiviral resistant strains of <span class="annotated ">influenza A H1N1</span> (2009) would be useful for clinicians to allow for change to an effective treatment at a much earlier stage if resistance is found. It was the aim of this study to develop a real-time RT-PCR that can detect all <span class="annotated ">influenza A viruses</span> and type simultaneously for <span class="annotated ">influenza A H1N1</span> (2009) and oseltamivir resistant (H275Y) <span class="annotated ">influenza A H1N1</span> (2009). This multiplex assay will allow laboratories to screen respiratory samples for all types of <span class="annotated ">influenza A</span>, <span class="annotated ">influenza A H1N1</span> (2009) virus and oseltamivir resistant (H275Y) <span class="annotated ">influenza A H1N1</span> (2009) virus in a rapid and cost effective format, ensuring that typing methods for seasonal and avian <span class="match unannotated ">viruses</span> are used on a smaller subset of samples. Since most virology laboratories already offer a molecular service for <span class="annotated ">influenza A</span> this assay could easily be implemented into most areas at little cost therefore increasing local access to resistance testing. 
<h3>20970458</h3>Evaluation of the Virus Counter(R) for rapid <span class="unannotated ">baculovirus</span> quantitation.<br><br>The utility of a new instrument for rapid virus quantitation, the Virus Counter, was evaluated in a blind study conducted at three sites. This instrument is a substantially improved version of the original academic research instrument described previously by Stoffel and Rowlen (2005a). The addition of hydrodynamic focusing, a self-contained fluidics system and customized software for system control and data analysis has resulted in a commercially viable and available design. <span class="unannotated ">Baculovirus</span> samples were provided by Protein Sciences Corporation and blinded to InDevR and Baylor College of Medicine. Protein Sciences Corporation and Baylor College of Medicine analyzed the samples by plaque assay and InDevR analyzed the samples using the Virus Counter. Serial dilution of stock <span class="match unannotated ">viruses</span> into growth media and buffer allowed for comparison of measured versus intended concentrations. Direct log-scale comparison between pooled Virus Counter results and pooled plaque assay results indicated a linear relationship (slope=1.1+/-0.2, R(2)=0.86) with statistically significant Pearson correlation (r=0.93, p&lt;0.001). 
<h3>20980501</h3>A nuclear inhibitor of NF-kappaB encoded by a poxvirus.<br><br>Poxviruses have evolved various strategies to inhibit cytoplasmic events leading to activation of the nuclear factor kappaB (NF-kappaB) signaling pathway, with individual <span class="match unannotated ">viruses</span> often encoding multiple NF-kappaB inhibitors. Here, the novel <span class="annotated ">orf virus</span> (<span class="annotated ">ORFV</span>)-encoded protein ORFV002 was shown to inhibit nuclear events regulating NF-kappaB transcriptional activity. ORFV002 expression in cell cultures significantly decreased wild-type-virus-, tumor necrosis factor alpha (TNF-alpha)-, and lipopolysaccharide (LPS)-induced NF-kappaB-mediated gene expression. Expression of ORFV002 in cells, while not affecting phosphorylation or nuclear translocation of NF-kappaB-p65, markedly decreased TNF-alpha- and wild-type-virus-induced acetylation of NF-kappaB-p65, a p300-mediated nuclear modification of NF-kappaB-p65 that regulates its transactivating activity. ORFV002 was shown to colocalize and interact with NF-kappaB-p65, and expression of ORFV002 in cell cultures resulted in a reduced interaction of NF-kappaB-p65 with p300, suggesting that ORFV002 interferes with NF-kappaB-p65/p300 association. Deletion of ORFV002 from the OV-IA82 genome had no significant effect on <span class="annotated ">ORFV</span> pathogenesis in <span class="annotated ">sheep</span>, indicating that ORFV002 is nonessential for virus virulence in the natural host. This represents the first description of a nuclear inhibitor of NF-kappaB encoded by a poxvirus. 
<h3>20980526</h3>An in-depth analysis of original antigenic sin in <span class="annotated ">dengue virus</span> infection.<br><br>The evolution of <span class="annotated ">dengue viruses</span> has resulted in four antigenically similar yet distinct serotypes. Infection with one <span class="unannotated ">serotype</span> likely elicits lifelong immunity to that <span class="unannotated ">serotype</span>, but generally not against the other three. Secondary or sequential infections are common, as multiple <span class="unannotated ">viral</span> serotypes frequently cocirculate. <span class="annotated ">Dengue</span> infection, although frequently mild, can lead to <span class="annotated ">dengue</span> hemorrhagic fever (DHF) which can be life threatening. DHF is more common in secondary <span class="annotated ">dengue</span> infections, implying a role for the adaptive immune response in the disease. There is currently much effort toward the design and implementation of a <span class="annotated ">dengue</span> vaccine but these efforts are made more difficult by the challenge of inducing durable neutralizing immunity to all four <span class="match unannotated ">viruses</span>. Domain 3 of the <span class="annotated ">dengue virus</span> envelope protein (ED3) has been suggested as one such candidate because it contains neutralizing epitopes and it was originally thought that relatively few cross-reactive antibodies are directed to this domain. In this study, we performed a detailed analysis of the anti-ED3 response in a cohort of <span class="LINNAEUS_only ">patients</span> suffering either primary or secondary <span class="annotated ">dengue</span> infections. The results show dramatic evidence of original antigenic sin in secondary infections both in terms of binding and enhancement activity. This has important implications for <span class="annotated ">dengue</span> vaccine design because heterologous boosting is likely to maintain the immunological footprint of the first vaccination. On the basis of these findings, we propose a simple in vitro enzyme-linked immunosorbent assay (ELISA) to diagnose the original <span class="annotated ">dengue</span> infection in secondary <span class="annotated ">dengue</span> cases. 
<h3>21029748</h3>One-step real-time reverse transcription-PCR assays for detecting and subtyping pandemic <span class="annotated ">influenza A/H1N1</span> 2009, seasonal <span class="annotated ">influenza A/H1N1</span>, and seasonal <span class="annotated ">influenza A/H3N2 viruses</span>. <br><br>Pandemic <span class="annotated ">influenza A/H1N1</span> 2009 (<span class="annotated ">A/H1N1pdm</span>) virus has caused significant outbreaks worldwide. <span class="unannotated ">A</span> previous one-step real-time reverse transcription-PCR (rRT-PCR) assay for detecting <span class="annotated ">A/H1N1pdm virus</span> (H1pdm rRT-PCR assay) was improved since the former probe had a low melting temperature and low tolerance to <span class="unannotated ">viral</span> mutation. To help with the screening of the <span class="annotated ">A/H1N1pdm virus</span>, rRT-PCR assays were also developed for detecting <span class="annotated ">human</span> seasonal <span class="annotated ">A/H1N1</span> (H1 rRT-PCR assay) and <span class="annotated ">A/H3N2 influenza viruses</span> (H3 rRT-PCR assay). H1pdm, <span class="unannotated ">H1</span>, and <span class="unannotated ">H3</span> rRT-PCR assays were evaluated using in vitro-transcribed control RNA, isolated <span class="match unannotated ">viruses</span>, and other respiratory pathogenic <span class="match unannotated ">viruses</span>, and were shown to have high sensitivity, good linearity (R(2)=0.99), and high specificity. In addition, the improved H1pdm rRT-PCR assay could detect two <span class="annotated ">viral strains of A/H1N1pdm</span>, namely, <span class="annotated ">A/Aichi/472/2009 (H1N1)pdm</span> and <span class="annotated ">A/Sakai/89/2009 (H1N1)pdm</span>, which have mutation(s) in the probe-binding region of the hemagglutinin gene, without loss of sensitivity. Using the three rRT-PCR assays developed, 90 clinical specimens collected between May and October 2009 were then tested. Of these, 26, 20, and 2 samples were identified as positive for <span class="annotated ">A/H1pdm</span>, <span class="annotated ">A/H3</span>, and <span class="annotated ">A/H1</span>, respectively, while 42 samples were negative for <span class="annotated ">influenza A viruses</span>. The present results suggest that these highly sensitive and specific H1pdm, <span class="unannotated ">H1</span>, and <span class="unannotated ">H3</span> rRT-PCR assays are useful not only for diagnosing <span class="annotated ">influenza viruses</span>, but also for the surveillance of <span class="annotated ">influenza viruses</span>. 
<h3>21034773</h3>Comparison of commercial systems for extraction of nucleic acids from DNA/RNA respiratory pathogens. <br><br>This study compared six automated nucleic acid extraction systems and one manual kit for their ability to recover nucleic acids from <span class="annotated ">human</span> nasal wash specimens spiked with five respiratory pathogens, representing Gram-positive bacteria (<span class="annotated ">Streptococcus pyogenes</span>), Gram-negative bacteria (<span class="annotated ">Legionella pneumophila</span>), DNA <span class="match unannotated ">viruses</span> (<span class="annotated ">adenovirus</span>), segmented RNA <span class="match unannotated ">viruses</span> (<span class="annotated ">human influenza virus A</span>), and non-segmented RNA <span class="match unannotated ">viruses</span> (<span class="annotated ">respiratory syncytial virus</span>). The robots and kit evaluated represent major commercially available methods that are capable of simultaneous extraction of DNA and RNA from respiratory specimens, and included platforms based on magnetic-bead technology (KingFisher mL, Biorobot EZ1, easyMAG, KingFisher Flex, and MagNA Pure Compact) or glass fiber filter technology (Biorobot MDX and the manual kit Allprep). All methods yielded extracts free of cross-contamination and RT-PCR inhibition. All automated systems recovered <span class="annotated ">L. pneumophila</span> and <span class="annotated ">adenovirus</span> DNA equivalently. However, the MagNA Pure protocol demonstrated more than 4-fold higher DNA recovery from the <span class="annotated ">S. pyogenes</span> than other methods. The KingFisher mL and easyMAG protocols provided 1- to 3-log wider linearity and extracted 3- to 4-fold more RNA from the <span class="annotated ">human influenza virus</span> and <span class="annotated ">respiratory syncytial virus</span>. These findings suggest that systems differed in nucleic acid recovery, reproducibility, and linearity in a pathogen specific manner. 
<h3>21047957</h3>Modification of nonstructural protein 1 of <span class="annotated ">influenza A virus</span> by SUMO1.<br><br>Nonstructural protein 1 (NS1) is one of the major factors resulting in the efficient infection rate and high level of virulence of <span class="annotated ">influenza A virus</span>. Although consisting of only approximately 230 amino acids, NS1 has the ability to interfere with several systems of the host <span class="unannotated ">viral</span> defense. In the present study, we demonstrate that NS1 of the highly pathogenic avian <span class="annotated ">influenza A/Duck/Hubei/L-1/2004 (H5N1) virus</span> interacts with <span class="annotated ">human</span> Ubc9, which is the E2 conjugating enzyme for sumoylation, and we show that SUMO1 is conjugated to <span class="annotated ">H5N1</span> NS1 in both transfected and infected cells. Furthermore, two lysine residues in the C terminus of NS1 were identified as SUMO1 acceptor sites. When the SUMO1 acceptor sites were removed by mutation, NS1 underwent rapid degradation. Studies of different <span class="annotated ">influenza A virus</span> strains of <span class="annotated ">human</span> and avian origin showed that the majority of <span class="match unannotated ">viruses</span> possess an NS1 protein that is modified by SUMO1, except for the recently emerged <span class="annotated ">swine-origin influenza A virus</span> (<span class="annotated ">S-OIV</span>) (H1N1). Interestingly, growth of a sumoylation-deficient WSN virus mutant was retarded compared to that of wild-type virus. Together, these results indicate that sumoylation enhances NS1 stability and thus promotes rapid growth of <span class="annotated ">influenza A virus</span>. 
<h3>21529147</h3>Growth inhibition of <span class="annotated ">Beauveria bassiana</span> by bacteria isolated from the cuticular surface of the <span class="annotated ">corn leafhopper</span>, <span class="annotated ">Dalbulus maidis</span> and the planthopper, <span class="annotated ">Delphacodes kuscheli</span>, two important vectors of <span class="annotated ">maize</span> pathogens. <br><br>The phytosanitary importance of the <span class="annotated ">corn leafhopper</span>, <span class="annotated ">Dalbulus maidis</span> (De Long and Wolcott) (Hemiptera: Cicadellidae) and the planthopper, <span class="annotated ">Delphacodes kuscheli</span> Fennah (Hemiptera: Delphacidae) lies in their ability to transmit phloem-associated plant pathogens, mainly <span class="match unannotated ">viruses</span> and mollicutes, and to cause considerable mechanical damage to <span class="annotated ">corn</span> plants during feeding and oviposition. Fungi, particularly some members of the Ascomycota, are likely candidates for biocontrol agents against these insect pests, but several studies revealed their failure to invade the insect cuticle possibly because of the presence of inhibitory compounds such as phenols, quinones, and lipids and also by the antibiosis effect of the microbiota living on the cuticular surface of the host. The present work aims to understand interactions between the entomopathogenic fungus <span class="annotated ">Beauveria bassiana</span> (Balsamao-Crivelli) Vuillemin (Hypocreales: Cordycipitaceae) and bacterial antagonists isolated from the cuticular surface of <span class="annotated ">D. maidis</span> and <span class="annotated ">D. kuscheli</span>. <span class="unannotated ">A</span> total of 155 bacterial isolates were recovered from the insect's cuticle and tested against <span class="annotated ">B. bassiana</span>. Ninety-one out of 155 strains inhibited the growth of <span class="annotated ">B. bassiana</span>. Bacterial strains isolated from <span class="annotated ">D. maidis</span> were significantly more antagonistic against <span class="annotated ">B. bassiana</span> than those isolates from <span class="annotated ">D. kuscheli</span>. Among the most effective antagonistic strains, six isolates of <span class="annotated ">Bacillus thuringiensis</span> Berliner (Bacillales: Bacillaeae (after <span class="annotated ">B. subtilis</span>)), one isolate of <span class="annotated ">B. mycoides</span> Flugge, eight isolates of <span class="annotated ">B. megaterium</span> de Bary, five isolates of <span class="annotated ">B.pumilus</span> Meyer and Gottheil, one isolate of <span class="annotated ">B. licheniformis</span> (Weigmann) Chester, and four isolates of <span class="annotated ">B. subtilis</span> (Ehrenberg) Cohn were identified. 
<h3>20962089</h3>Antiviral effects of a transgenic RNA-dependent RNA polymerase.<br><br>Transgenic expression of the RNA-dependent RNA polymerase 3D(pol) inhibited infection of <span class="annotated ">Theiler's murine encephalitis virus</span> (<span class="annotated ">TMEV</span>), a picornavirus from which it was derived. Here, we infected 3D(pol) transgenic <span class="annotated ">mice</span> with another picornavirus, as well as an alphaherpesvirus and a rhabdovirus. 3D(pol) transgenic FVB <span class="annotated ">mice</span> had significantly lower <span class="unannotated ">viral</span> loads and survived longer after infection with all three types of <span class="match unannotated ">viruses</span> than nontransgenic FVB <span class="annotated ">mice</span>. Viral inhibition among three different types of virus by transgenic 3D(pol) suggests that the mechanism of action is not the direct interference with picornaviral 3D(pol) but instead may be the changing of host cells to an antiviral state before or after <span class="unannotated ">viral</span> infection occurs, as basal interferon levels were higher in 3D(pol) transgenic <span class="annotated ">mice</span> before infection. Further study of this mechanism may open new possibilities for future antiviral therapy. 
<h3>20980506</h3>A new model of <span class="annotated ">Epstein-Barr virus</span> infection reveals an important role for earlylytic <span class="unannotated ">viral</span> protein expression in the development of lymphomas.<br><br><span class="annotated ">Epstein-Barr virus</span> (<span class="annotated ">EBV</span>) infects cells in latent or lytic forms, but the roleof lytic infection in <span class="annotated ">EBV</span>-induced lymphomas is unclear. Here, we have used anew humanized <span class="annotated ">mouse</span> model, in which both <span class="annotated ">human</span> fetal CD34(+) hematopoietic stemcells and thymus/liver tissue are transplanted, to compare <span class="annotated ">EBV</span> pathogenesis andlymphoma formation following infection with a lytic replication-defectiveBZLF1-deleted (Z-KO) virus or a lytically active BZLF1(+) control. Both thecontrol and Z-KO <span class="match unannotated ">viruses</span> established long-term <span class="unannotated ">viral</span> latency in all infectedanimals. The infection appeared well controlled in some animals, but otherseventually developed CD20(+) diffuse large B cell lymphomas (DLBCL). Animalsinfected with the control virus developed tumors more frequently than Z-KOvirus-infected animals. Specific immune responses against <span class="annotated ">EBV</span>-infected B cellswere generated in <span class="annotated ">mice</span> infected with either the control virus or the Z-KOvirus. In both cases, forms of <span class="unannotated ">viral</span> latency (type I and type IIB) wereobserved that are less immunogenic than the highly transforming form (type III)commonly found in tumors of immunocompromised hosts, suggesting that immunepressure contributed to the outcome of the infection. These results point to animportant role for lytic <span class="annotated ">EBV</span> infection in the development of B cell lymphomasin the context of an active host immune response.
<h3>20980522</h3>RNA aptamers directed to <span class="annotated ">human immunodeficiency virus type 1</span> Gag polyprotein bind to the matrix and nucleocapsid domains and inhibit virus production. <br><br>Gag orchestrates the assembly and release of <span class="annotated ">human immunodeficiency virus type 1</span> (<span class="annotated ">HIV-1</span>) particles. We explored here the potential of anti-Gag RNA aptamers to inhibit <span class="annotated ">HIV-1</span> replication. In vitro, RNA aptamers raised against an <span class="annotated ">HIV-1</span> Gag protein, lacking the N-terminal myristate and the C-terminal p6 (DP6-Gag), could bind to matrix protein (MA), nucleocapsid protein (NC), or entire DP6-Gag protein. Upon cotransfection with pNL4-3.Luc molecular clone into 293T cells, six of the aptamers caused mild inhibition (2- to 3-fold) in the extracellular capsid levels, and one aptamer displayed 20-fold inhibition. The reduction was not due to a release defect but reflected Gag mRNA levels. We hypothesized that the aptamers influence genomic RNA levels via perturbation of specific Gag-genomic RNA interactions. Binding studies revealed that the &quot;NC-binders&quot; specifically compete with the packaging signal (psi) of <span class="annotated ">HIV-1</span> for binding to DP6-Gag. Therefore, we tested the ability of two NC-binders to inhibit <span class="match unannotated ">viruses</span> containing psi-region deletions (DeltaSL1 or DeltaSL3) and found that the NC-binders were no longer able to inhibit Gag synthesis. The inability of these aptamers to inhibit psi-deleted <span class="match unannotated ">viruses</span> correlated with the absence of competition with the corresponding psi transcripts lacking SL1 or SL3 for binding DP6-Gag in vitro. These results indicate that the NC-binding aptamers disrupt Gag-genomic RNA interaction and negatively affect genomic RNA transcription, processing, or stability. Our results reveal an essential interaction between <span class="annotated ">HIV-1</span> Gag and the psi-region that may be distinct from that which occurs during the encapsidation of genomic RNA. Thus, anti-Gag aptamers can be an effective tool to perturb Gag-genomic RNA interactions. 
</body></html>