
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>A. nidulans</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20633690</h3>UreA, the major urea/H+ symporter in <span class="yellow">Aspergillus nidulans</span>. <br><br>We report here the characterization of UreA, a high-affinity urea/H+ symporter of <span class="yellow">Aspergillus nidulans</span>. The deletion of the encoding gene abolishes urea transport at low substrate concentrations, suggesting that in these conditions UreA is the sole transport system specific for urea in <span class="blue">A. nidulans</span>. The ureA gene is not inducible by urea or its precursors, but responds to nitrogen metabolite repression, necessitating for its expression the AreA GATA factor. In contrast to what was observed for other transporters in <span class="blue">A. nidulans</span>, repression by ammonium is also operative during the isotropic growth phase. The activity of UreA is down-regulated post-translationally by ammonium-promoted endocytosis. <span class="yellow">A</span> number of homologues of UreA have been identified in <span class="blue">A. nidulans</span> and other Aspergilli, which cluster in four groups, two of which contain the urea transporters characterized so far in fungi and plants. This phylogeny may have arisen by gene duplication events, giving place to putative transport proteins that could have acquired novel, still unidentified functions. 
<h3>20713166</h3>Characterization of the <span class="yellow">Aspergillus nidulans</span> biotin biosynthetic gene cluster and use of the bioDA gene as a new transformation marker. <br><br>The genes involved in the biosynthesis of biotin were identified in the hyphal fungus <span class="yellow">Aspergillus nidulans</span> through homology searches and complementation of <span class="yellow">Escherichia coli</span> biotin-auxotrophic mutants. Whereas the 7,8-diaminopelargonic acid synthase and dethiobiotin synthetase are encoded by distinct genes in bacteria and the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, both activities are performed in <span class="blue">A. nidulans</span> by a single enzyme, encoded by the bifunctional gene bioDA. Such a bifunctional bioDA gene is a genetic feature common to numerous members of the ascomycete filamentous fungi and basidiomycetes, as well as in plants and oomycota. However, unlike in other eukaryota, the three bio genes contributing to the four enzymatic steps from pimeloyl-CoA to biotin are organized in a gene cluster in pezizomycotina. The <span class="blue">A. nidulans</span> auxotrophic mutants biA1, biA2 and biA3 were all found to have mutations in the 7,8-diaminopelargonic acid synthase domain of the bioDA gene. Although biotin auxotrophy is an inconvenient marker in classical genetic manipulations due to cross-feeding of biotin, transformation of the biA1 mutant with the bioDA gene from either <span class="blue">A. nidulans</span> or <span class="yellow">Aspergillus fumigatus</span> led to the recovery of well-defined biotin-prototrophic colonies. The usefulness of bioDA gene as a novel and robust transformation marker was demonstrated in co-transformation experiments with a green fluorescent protein reporter, and in the efficient deletion of the laccase (yA) gene via homologous recombination in a mutant lacking non-homologous end-joining activity. 
<h3>20817115</h3>Characterization of the developmental regulator FlbE in <span class="yellow">Aspergillus fumigatus</span> and <span class="yellow">Aspergillus nidulans</span>. <br><br>Several upstream developmental activators control asexual development (conidiation) in Aspergillus. In this study, we characterize one of such activators called flbE in <span class="yellow">Aspergillus fumigatus</span> and <span class="yellow">Aspergillus nidulans</span>. The predicted FlbE protein is composed of 222 and 201 aa in <span class="yellow">A. fumigatus</span> and <span class="blue">A. nidulans</span>, respectively. While flbE is transiently expressed during early phase of growth in <span class="blue">A. nidulans</span>, it is somewhat constitutively expressed during the lifecycle of <span class="yellow">A. fumigatus</span>. The deletion of flbE causes reduced conidiation and delayed expression of brlA and vosA in both species. Moreover, FlbE is necessary for salt-induced development in liquid submerged culture in <span class="yellow">A. fumigatus</span>. The <span class="blue">A. nidulans</span> flbE null mutation is fully complemented by <span class="yellow">A. fumigatus</span> flbE, indicating a functional conservancy of FlbE in Aspergillus. Both the deletion and overexpression of flbE in <span class="blue">A. nidulans</span> result in developmental defects, enhanced autolysis, precocious cell death, and delayed expression of brlA/vosA, suggesting that balanced activity of FlbE is crucial for proper growth and development. Importantly, the N-terminal portion of FlbE exhibits the trans-activation ability in <span class="yellow">yeast</span>, whereas the C-terminal half negatively affects its activity. Site-directed mutagenesis of certain conserved N-terminal amino acids abolishes the ability of trans-activation, overexpression-induced autolysis, and complementing the null mutation. Finally, overexpression of flbD, but not flbB or flbC, restores conidiation in <span class="blue">A. nidulans</span> DeltaflbE, generally supporting the current genetic model for developmental regulation. 
<h3>20955810</h3>Novel mutations reveal two important regions in <span class="yellow">Aspergillus nidulans</span> transcriptional activator MetR. <br><br>Expression of the sulfur assimilation pathway in <span class="yellow">Aspergillus nidulans</span> is under control of sulfur metabolite repression, which is composed of scon genes encoding subunits of ubiquitin ligase and the metR gene coding for a transcriptional activator. In this paper we report three dominant suppressors of methionine requirement isolated from a metB3 diploid strain. All three mutations lead to the substitution of phenylalanine 48 by serine or leucine in the conserved N-terminal region of the MetR protein. Strains carrying the dominant suppressor mutations exhibit increased activities of homocysteine synthase and sulfur assimilation enzymes as well as elevated levels of the corresponding transcripts. These changes are observed even under conditions of methionine repression, which suggests that the mutated MetR protein may be resistant to inactivation or degradation mediated by sulfur metabolite repression. We also found that a mutant impaired in sulfite reductase activity, known until now as sG8, has a frameshift which changes 41 C-terminal amino acids. Therefore, it is now designated metR18. This mutant has elevated levels of MetR-regulated transcripts and of activities of sulfur assimilation enzymes (except sulfite reductase), which can be repressed to the wild type level by exogenous methionine. Thus, metR18 and the three dominant suppressors represent new types of mutations affecting different parts of the <span class="blue">A. nidulans</span> MetR protein. 
<h3>21419234</h3>Completing the purine utilisation pathway of <span class="yellow">Aspergillus nidulans</span>.<br><br>We have previously identified by classical genetics and biochemistry, all the genes of <span class="yellow">Aspergillus nidulans</span> predicted to be involved in purine utilisation, together with cognate regulatory genes and one gene encoding a novel xanthine hydroxylation activity. In this article we complete the description of the purine utilisation pathway with the identification of the two genes (uaX and uaW) encoding the enzymes catalysing the conversion of the product of urate oxidation by urate oxidase, 5-hydroxyisourate, to optically active allantoin. The identification of these additional genes confirms the complete absence of clustering of the genes involved in purine utilisation in <span class="blue">A. nidulans</span>. 
<h3>21220038</h3>acon-3, the <span class="yellow">Neurospora crassa</span> ortholog of the developmental modifier, medA, complements the conidiation defect of the <span class="yellow">Aspergillus nidulans</span> mutant. <br><br><span class="yellow">Aspergillus nidulans</span> and <span class="yellow">Neurospora crassa</span> are ascomycetes that produce asexual spores through morphologically distinct processes. MedA, a protein with unknown function, is required for normal asexual and sexual development in <span class="blue">A. nidulans</span>. We determined that the <span class="yellow">N. crassa</span> ortholog of medA is acon-3, a gene required for early conidiophore development and female fertility. To test hypotheses about the evolutionary origins of asexual development in distinct <span class="yellow">fungal</span> lineages it is important to understand the degree of conservation of developmental regulators. The amino acid sequences of <span class="blue">A. nidulans</span> MedA and <span class="yellow">N. crassa</span> ACON-3 shared 37% identity and 51% similarity. acon-3 is induced at late time points of conidiation. In contrast, medA is constitutively expressed and MedA protein localizes to nuclei in all tissue types. Nonetheless, expression of acon-3 using its native promoter complemented the conidiation defects of the <span class="blue">A. nidulans</span> DeltamedA and medA15 mutants. We conclude that the biochemical activity of the medA orthologs is conserved for conidiation. 
</body></html>