
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.linneaus_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 

table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Aspergillus oryzae</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="linneaus_only">LINNEUS only</span><br><hr><h3>20849972</h3>Identification and characterization of genes responsible for biosynthesis of kojic acid, an industrially important compound from <span class="match annotated ">Aspergillus oryzae</span>. <br><br>Kojic acid is produced in large amounts by <span class="match annotated ">Aspergillus oryzae</span> as a secondary metabolite and is widely used in the cosmetic industry. Glucose can be converted to kojic acid, perhaps by only a few steps, but no genes for the conversion have thus far been revealed. Using a DNA microarray, gene expression profiles under three pairs of conditions significantly affecting kojic acid production were compared. All genes were ranked using an index parameter reflecting both high amounts of transcription and a high induction ratio under producing conditions. After disruption of nine candidate genes selected from the top of the list, two genes of unknown function were found to be responsible for kojic acid biosynthesis, one having an oxidoreductase motif and the other a transporter motif. These two genes are closely associated in the genome, showing typical characteristics of genes involved in secondary metabolism. 
<h3>22008745</h3>Identification and characterization of a putative basic helix-loop-helix transcription factor involved in the early stage of conidiophore development in <span class="match annotated ">Aspergillus oryzae</span>. <br><br>The helix-loop-helix (HLH) family of transcriptional factors is a key player in a wide range of developmental processes. HLH proteins form homo- and/or heterodimers with other HLH proteins and bind to E-box motifs. The regulation and functions of these proteins can be complex due to their bifunctional roles as activators and repressors of gene transcription. In this study, we isolated and characterized a novel predicted bHLH protein-encoding gene, AO090023000902, designated ecdR (early conidiophore development regulator), in <span class="match annotated ">Aspergillus oryzae</span>. The ecdR gene disruptant produced very few conidia. Conversely, the overexpression of ecdR resulted in the formation of a large number of conidia at an early stage, suggesting that the EcdR protein is required for early asexual development. Additionally, when serially diluted conidia were spread-cultivated onto malt agar medium, we found that conidial number of the control strain depended on the cultivated conidium density, while the ecdR-overexpressing strain showed no significant change in conidiation. These phenotypes of ecdR-disruptant and ecdR-overexpressing strains are partially similar to those of the sclR-overexpressing strain and sclR-disruptant, respectively. <span class="annotated ">Yeast</span> two-hybrid assays and sclR, ecdR-double deletion experiment indicated that EcdR plays a major role in conidiation, and SclR represses this function by competitively interacting with EcdR in <span class="annotated ">A. oryzae</span>. 
<h3>21277379</h3>Independent duplications of alpha-amylase in different strains of <span class="match annotated ">Aspergillus oryzae</span>. <br><br><span class="match annotated ">Aspergillus oryzae</span> is a filamentous fungus that has arisen through the ancient domestication of <span class="annotated ">Aspergillus flavus</span> for making traditional oriental foods and beverages. In the many centuries <span class="annotated ">A. oryzae</span> has been used for fermenting the starch in <span class="unannotated ">rice</span> to simple sugars, it has undergone selection for increased secretion of starch-degrading enzymes. In particular, all <span class="annotated ">A. oryzae</span> strains investigated thus far have two or more copies of a gene encoding alpha-amylase, whereas <span class="annotated ">A. flavus</span> has only one. Here we investigate the duplications leading to these copies in three <span class="annotated ">A. oryzae</span> strains. We find evidence of at least three separate duplications of alpha-amylase, an example of parallel evolution in a micro-organism under artificial selection. At least two of these duplications appear to be associated with activity of transposable elements of the Tc1/mariner class. Both involve a 9.1 kb element that terminates in inverted repeats, encodes a putative transposase and another putative protein of unknown function, and contains an unusual arrangement of four short internal imperfect repeats. Although &quot;unusual Mariners&quot; of this size have previously been identified in <span class="annotated ">A. oryzae</span>, <span class="annotated ">Aspergillus fumigatus</span> and <span class="annotated ">Aspergillus nidulans</span>, this is the first evidence we know of that at least some of them are active in modern times and that their activity can contribute to beneficial genetic changes. 
</body></html>