
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Aspergillus fumigatus</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20619350</h3>Characterization of the GPI-anchored endo Beta-1,3-glucanase Eng2 of <span class="blue">Aspergillus fumigatus</span>. <br><br>A GPI-anchored endo Beta-1,3-glucanase of <span class="blue">Aspergillus fumigatus</span> was characterized. The enzyme encoded by ENG2 (AFUA_2g14360) belongs to the glycoside hydrolase family 16 (GH16). The activity was characterized using a recombinant protein produced by <span class="yellow">Pichiapastoris</span>. The recombinant enzyme preferentially acts on soluble Beta-1,3-glucans. Enzymatic analysis of the endoglucanase activity using Carboxymethyl-Curdlan-Remazol Brilliant Blue (CM-Curdlan-RBB) as a substrate revealed a wide temperature optimum of 24-40^0C, a pH optimum of 5.0-6.5 and a K(m) of 0.8 mg ml(-1). HPAEC analysis of the products formed by Eng2 when acting on different oligo-Beta-1,3-glucans confirmed the predicted endoglucanase activity and also revealed a transferase activity for oligosaccharides of a low degree of polymerization. The growth phenotype of the Afeng2 mutant was identical to that of the wt strain. 
<h3>20713166</h3>Characterization of the <span class="yellow">Aspergillus nidulans</span> biotin biosynthetic gene cluster and use of the bioDA gene as a new transformation marker. <br><br>The genes involved in the biosynthesis of biotin were identified in the hyphal fungus <span class="yellow">Aspergillus nidulans</span> through homology searches and complementation of <span class="yellow">Escherichia coli</span> biotin-auxotrophic mutants. Whereas the 7,8-diaminopelargonic acid synthase and dethiobiotin synthetase are encoded by distinct genes in bacteria and the <span class="yellow">yeast</span> <span class="yellow">Saccharomyces cerevisiae</span>, both activities are performed in <span class="yellow">A. nidulans</span> by a single enzyme, encoded by the bifunctional gene bioDA. Such a bifunctional bioDA gene is a genetic feature common to numerous members of the ascomycete filamentous fungi and basidiomycetes, as well as in plants and oomycota. However, unlike in other eukaryota, the three bio genes contributing to the four enzymatic steps from pimeloyl-CoA to biotin are organized in a gene cluster in pezizomycotina. The <span class="yellow">A. nidulans</span> auxotrophic mutants biA1, biA2 and biA3 were all found to have mutations in the 7,8-diaminopelargonic acid synthase domain of the bioDA gene. Although biotin auxotrophy is an inconvenient marker in classical genetic manipulations due to cross-feeding of biotin, transformation of the biA1 mutant with the bioDA gene from either <span class="yellow">A. nidulans</span> or <span class="blue">Aspergillus fumigatus</span> led to the recovery of well-defined biotin-prototrophic colonies. The usefulness of bioDA gene as a novel and robust transformation marker was demonstrated in co-transformation experiments with a green fluorescent protein reporter, and in the efficient deletion of the laccase (yA) gene via homologous recombination in a mutant lacking non-homologous end-joining activity. 
<h3>20817115</h3>Characterization of the developmental regulator FlbE in <span class="blue">Aspergillus fumigatus</span> and <span class="yellow">Aspergillus nidulans</span>. <br><br>Several upstream developmental activators control asexual development (conidiation) in Aspergillus. In this study, we characterize one of such activators called flbE in <span class="blue">Aspergillus fumigatus</span> and <span class="yellow">Aspergillus nidulans</span>. The predicted FlbE protein is composed of 222 and 201 aa in <span class="yellow">A. fumigatus</span> and <span class="yellow">A. nidulans</span>, respectively. While flbE is transiently expressed during early phase of growth in <span class="yellow">A. nidulans</span>, it is somewhat constitutively expressed during the lifecycle of <span class="yellow">A. fumigatus</span>. The deletion of flbE causes reduced conidiation and delayed expression of brlA and vosA in both species. Moreover, FlbE is necessary for salt-induced development in liquid submerged culture in <span class="yellow">A. fumigatus</span>. The <span class="yellow">A. nidulans</span> flbE null mutation is fully complemented by <span class="yellow">A. fumigatus</span> flbE, indicating a functional conservancy of FlbE in Aspergillus. Both the deletion and overexpression of flbE in <span class="yellow">A. nidulans</span> result in developmental defects, enhanced autolysis, precocious cell death, and delayed expression of brlA/vosA, suggesting that balanced activity of FlbE is crucial for proper growth and development. Importantly, the N-terminal portion of FlbE exhibits the trans-activation ability in <span class="yellow">yeast</span>, whereas the C-terminal half negatively affects its activity. Site-directed mutagenesis of certain conserved N-terminal amino acids abolishes the ability of trans-activation, overexpression-induced autolysis, and complementing the null mutation. Finally, overexpression of flbD, but not flbB or flbC, restores conidiation in <span class="yellow">A. nidulans</span> DeltaflbE, generally supporting the current genetic model for developmental regulation. 
<h3>21073976</h3>FungiFun: a web-based application for functional categorization of <span class="yellow">fungal</span> genes and proteins. <br><br>FungiFun assigns functional annotations to <span class="yellow">fungal</span> genes or proteins and performs gene set enrichment analysis. Based on three different classification methods (FunCat, GO and KEGG), FungiFun categorizes genes and proteins for several <span class="yellow">fungal</span> species on different levels of annotation detail. It is web-based and accessible to users without any programming skills. FungiFun is the first tool offering gene set enrichment analysis including the FunCat categorization. Two biological datasets for <span class="blue">Aspergillus fumigatus</span> and <span class="yellow">Candida albicans</span> were analyzed using FungiFun, providing an overview of the usage and functions of the tool. FungiFun is freely accessible at https://www.omnifung.hki-jena.de/FungiFun/. 
<h3>21073977</h3>Cleavage of resveratrol in fungi: characterization of the enzyme Rco1 from <span class="yellow">Ustilago maydis</span>. <br><br><span class="yellow">Ustilago maydis</span>, the causative agent of <span class="yellow">corn smut</span> disease, contains two genes encoding members of the carotenoid cleavage oxygenase family, a group of enzymes that cleave double bonds in different substrates. One of them, Cco1, was formerly identified as a Beta-carotene cleaving enzyme. Here we elucidate the function of the protein encoded by the second gene, termed here as <span class="yellow">Ustilago maydis</span> Resveratrol cleavage oxygenase 1 (Um Rco1). In vitro incubations of heterologously expressed and purified UM Rco1 with different carotenoid and stilbene substrates demonstrate that it cleaves the interphenyl Calpha-CBeta double bond of the phytoalexin resveratrol and its derivative piceatannol. Um Rco1 exhibits a high degree of substrate specificity, as suggested by the lack of activity on carotenoids and the other resveratrol-related compounds tested. The activity of Um Rco1 was confirmed by incubation of <span class="yellow">U. maydis</span> rco1 deletion and over-expression strains with resveratrol. Furthermore, treatment with resveratrol resulted in striking alterations of cell morphology. However, pathogenicity assays indicated that Um rco1 is largely dispensable for biotrophic development. Our work reveals Um Rco1 as the first eukaryotic resveratrol cleavage enzyme identified so far. Moreover, Um Rco1 represents a subfamily of <span class="yellow">fungal</span> enzymes likely involved in the degradation of stilbene compounds, as suggested by the cleavage of resveratrol by homologs from <span class="blue">Aspergillus fumigatus</span>, <span class="yellow">Chaetomium globosum</span> and <span class="yellow">Botryotinia fuckeliana</span>. 
<h3>21145409</h3>Overexpression of the trichodiene synthase gene tri5 increases trichodermin production and antimicrobial activity in <span class="yellow">Trichoderma brevicompactum</span>. <br><br><span class="yellow">Trichoderma brevicompactum</span> produces trichodermin, a simple trichothecene-type toxin that shares the first steps of the sesquiterpene biosynthetic pathway with other phytotoxic trichothecenes from <span class="yellow">Fusarium spp</span>. Trichodiene synthase catalyses the conversion of farnesyl pyrophosphate to trichodiene and it is encoded by the tri5 gene that was cloned and analysed functionally by homologous overexpression in <span class="yellow">T. brevicompactum</span>. tri5 expression was up-regulated in media with glucose, H(2)O(2) or glycerol. tri5 repression was observed in cultures supplemented with the antioxidants ferulic acid and tyrosol. Acetone extracts of tri5-overexpressing transformants displayed higher antifungal activity than those from the wild-type. Chromatographic and spectroscopic analyses revealed that tri5 overexpression led to an increased production of trichodermin and tyrosol. Agar diffusion assays with these two purified metabolites from the tri5-overexpressing transformant <span class="yellow">T. brevicompactum</span> Tb41tri5 showed that only trichodermin had antifungal activity against <span class="yellow">Saccharomyces cerevisiae</span>, <span class="yellow">Kluyveromyces marxianus</span>, <span class="yellow">Candida albicans</span>, <span class="yellow">Candida glabrata</span>, <span class="yellow">Candida tropicalis</span> and <span class="blue">Aspergillus fumigatus</span>, in most cases such activity being higher than that observed for amphotericin B and hygromycin. Our results point to the significant role of tri5 in the production of trichodermin and in the antifungal activity of <span class="yellow">T. brevicompactum</span>. 
<h3>21176790</h3>Characterization of a polyketide synthase in <span class="yellow">Aspergillus niger</span> whose product is a precursor for both dihydroxynaphthalene (DHN) melanin and naphtho-gamma-pyrone. <br><br>The genome sequencing of the fungus <span class="yellow">Aspergillus niger</span> uncovered a large cache of genes encoding enzymes thought to be involved in the production of secondary metabolites yet to be identified. Identification and structural characterization of many of these predicted secondary metabolites are hampered by their low concentration relative to the known <span class="yellow">A. niger</span> metabolites such as the naphtho-gamma-pyrone family of polyketides. We deleted a non-reducing PKS gene in <span class="yellow">A. niger strain</span> ATCC 11414, a daughter strain of <span class="yellow">A. niger ATCC strain 1015</span> whose genome was sequenced by the DOE Joint Genome Institute. This PKS encoding gene we name albA is a predicted ortholog of alb1 from <span class="blue">Aspergillus fumigatus</span> which is responsible for production of the naphtho-gamma-pyrone precursor for the 1,8-dihydroxynaphthalene (DHN) melanin/spore pigment. Our results show that the <span class="yellow">A. nigeralbA</span> PKS is responsible for both the production of the spore pigment precursor and a family of naphtho-gamma-pyrones commonly found in significant quantity in <span class="yellow">A. niger</span> culture extracts. The generation of an <span class="yellow">A. niger</span><span class="yellow">A. niger strain</span> devoid of naphtho-gamma-pyrones will greatly facilitate the elucidation of cryptic biosynthetic pathways in this organism. 
<h3>21184840</h3>Functional analysis of the fungal/plant class chitinase family in <span class="blue">Aspergillus fumigatus</span>. <br><br>A quintuple mutant was constructed to delete the entire family of the fungal/plant (class III) chitinases of <span class="blue">Aspergillus fumigatus</span>. Only a limited reduction in the total chitinolytic activity was seen for the different chitinase mutants including the quintuple mutant. In spite of this reduction in chitinolytic activity, no growth or germination defects were observed in these chitinase mutants. This result demonstrated that the fungal/plant chitinases do not have an essential role in the morphogenesis of <span class="yellow">A. fumigatus</span>. <span class="yellow">A</span> slight diminution of the growth during autolysis was seen for the quintuple mutant suggesting that class III chitinases may play only a nutritional role during this phase of the cycle, retarding <span class="yellow">fungal</span> death. 
<h3>21195204</h3>Functional analysis of a mitochondrial phosphopantetheinyl transferase (PPTase) gene pptB in <span class="blue">Aspergillus fumigatus</span>. <br><br>The mitochondrial phosphopantetheinyl transferase gene pptB of the opportunistic pathogen <span class="blue">Aspergillus fumigatus</span> has been identified and characterised. Unlike pptA, which is required for lysine biosynthesis, secondary metabolism, and iron assimilation, pptB is essential for viability. PptB is located in the mitochondria. In vitro expression of pptA and pptB has shown that PptB is specific for the mitochondrial acyl carrier protein AcpA. 
<h3>21840411</h3>The metalloreductase FreB is involved in adaptation of <span class="blue">Aspergillus fumigatus</span> to iron starvation. <br><br><span class="blue">Aspergillus fumigatus</span> employs two high affinity iron uptake mechanisms, siderophore mediated iron uptake and reductive iron assimilation (RIA). The <span class="yellow">A. fumigatus</span> genome encodes 15 putative metalloreductases (MR) but the ferrireductasesinvolved in RIA remained elusive so far. Expression of the MR FreB was found to be transcriptionally repressed by iron via SreA, a repressor of iron acquisition during iron sufficiency, indicating a role in iron metabolism. FreB-inactivation by gene deletion was phenotypically largely inconspicuous unless combined with inactivation of the siderophore system, which then decreased growth rate, surface ferrireductase activity and oxidative stress resistance during iron starvation. This study also revealed that loss of copper-independent siderophore-mediated iron uptake increases sensitivity of <span class="yellow">A. fumigatus</span> to copper starvation due to copper-dependence of RIA. 
<h3>21840413</h3>The effects of dsRNA mycoviruses on growth and murine virulence of <span class="blue">Aspergillus fumigatus</span>. <br><br>Some isolates of the opportunistic <span class="yellow">human</span> pathogenic fungus <span class="blue">Aspergillus fumigatus</span> are known to be infected with mycoviruses. The dsRNA genomes of two of these mycoviruses, which include a chrysovirus and a partitivirus, have been completely sequenced and an RT-PCR assay for the viruses has been developed. Through curing virus-infected <span class="yellow">A. fumigatus</span> isolates by cycloheximide treatment and transfecting virus-free isolates with purified <span class="yellow">virus</span>, as checked by RT-PCR, isogenic virus-free and virus-infected lines of the fungus were generated whose phenotypes and growth have been directly compared. Mycovirus infection of <span class="yellow">A. fumigatus</span> with either the chrysovirus or the partitivirus resulted in significant aberrant phenotypic alterations and attenuation of growth of the fungus but had no effect on susceptibility to common antifungals. Chrysovirus infection of <span class="yellow">A. fumigatus</span> caused no significant alterations to murine pathogenicity. 
<h3>21907818</h3>The structure-function relationship of the <span class="yellow">Aspergillus fumigatuscyp51A</span> L98H conversion by site-directed mutagenesis: the mechanism of L98H azole resistance. <br><br>Since 1998, the rapid emergence of multi-azole-resistance (MAR) was observed in <span class="blue">Aspergillus fumigatus</span> in the Netherlands. Two dominant mutations were found in the cyp51A gene, a 34bp tandem repeat (TR) in the promoter region combined with a leucine to histidine substitution at codon 98 (L98H). In this study, we show that molecular dynamics simulations combined with site-directed mutagenesis of amino acid substitutions in the cyp51A gene, correlate to the structure-function relationship of the L98H substitution conferring to MAR in <span class="yellow">A. fumigatus</span>. Because of a L98H directed change in the flexibility of the loops, that comprise a gate-like structure in the protein, the capacity of the two ligand entry channels is modified by narrowing the diameter and thereby binding of azoles is obstructed. Moreover, the L98H induced relocation of tyrosine 121 and tyrosine 107 seems to be related to the MAR phenotype, without affecting the biological activity of the CYP51A protein. Site-directed mutagenesis showed that both the 34bp TR and the L98H mutation are required to obtain the MAR phenotype. Furthermore, the amino acid leucine in codon 98 in <span class="yellow">A. fumigatus</span> is highly conserved and important for maintaining the structure of the CYP51A protein that is essential for azole docking. 
<h3>21277379</h3>Independent duplications of alpha-amylase in different strains of <span class="yellow">Aspergillus oryzae</span>. <br><br><span class="yellow">Aspergillus oryzae</span> is a filamentous fungus that has arisen through the ancient domestication of <span class="yellow">Aspergillus flavus</span> for making traditional oriental foods and beverages. In the many centuries <span class="yellow">A. oryzae</span> has been used for fermenting the starch in <span class="yellow">rice</span> to simple sugars, it has undergone selection for increased secretion of starch-degrading enzymes. In particular, all <span class="yellow">A. oryzae</span> strains investigated thus far have two or more copies of a gene encoding alpha-amylase, whereas <span class="yellow">A. flavus</span> has only one. Here we investigate the duplications leading to these copies in three <span class="yellow">A. oryzae</span> strains. We find evidence of at least three separate duplications of alpha-amylase, an example of parallel evolution in a micro-organism under artificial selection. At least two of these duplications appear to be associated with activity of transposable elements of the Tc1/mariner class. Both involve a 9.1 kb element that terminates in inverted repeats, encodes a putative transposase and another putative protein of unknown function, and contains an unusual arrangement of four short internal imperfect repeats. Although &quot;unusual Mariners&quot; of this size have previously been identified in <span class="yellow">A. oryzae</span>, <span class="blue">Aspergillus fumigatus</span> and <span class="yellow">Aspergillus nidulans</span>, this is the first evidence we know of that at least some of them are active in modern times and that their activity can contribute to beneficial genetic changes. 
</body></html>