
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>cattle</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21029751</h3>Real time PCR method for simultaneous detection, quantitation and differentiation of capripoxviruses. 

The genus Capripoxvirus (CaPV) comprises three members namely, <span class="yellow">sheep poxvirus</span> (<span class="yellow">SPPV</span>), <span class="yellow">goat poxvirus</span> (<span class="yellow">GTPV</span>) and <span class="yellow">lumpy skin disease virus</span> (<span class="yellow">LSDV</span>) affecting <span class="yellow">sheep</span>, <span class="yellow">goats</span> and <span class="blue">cattle</span>, respectively. CaPV infections produce similar symptoms in <span class="yellow">sheep</span> and <span class="yellow">goats</span>, and the three viruses cannot be distinguished serologically. Since there are conflicting opinions regarding the host specificity of CaPVs, particularly for <span class="yellow">goatpox</span> and <span class="yellow">sheeppox viruses</span>, the development of rapid genotyping tools will facilitate more accurate disease diagnosis and surveillance for better management of capripox outbreaks. This paper describes a species-specific, real time polymerase chain reaction (PCR), based on unique molecular markers that were found in the G-protein-coupled chemokine receptor (GPCR) gene sequences of CaPVs, that uses dual hybridization probes for their simultaneous detection, quantitation and genotyping. The assay can differentiate between CaPV strains based on differences in the melting point temperature (Tm) obtained after fluorescence melting curve analysis (FMCA). It is highly sensitive and presents low intra- and inter-run variation. This real time PCR assay will make a significant contribution to CaPV diagnosis and to the better understanding of the epidemiology of CaPVs by enabling rapid genotyping and gene-based classification of <span class="yellow">viral</span> strains and unequivocal identification of isolates. 
<h3>21112814</h3>&quot;Slime molds&quot; among the Tubulinea (Amoebozoa): molecular systematics and taxonomy of Copromyxa. 

<span class="yellow">Copromyxa protea</span> is a dung-inhabiting amoeboid organism that aggregates to form simple macroscopic fruiting structures, sorocarps, which are composed of a single cell type. In a recent effort to find the phylogenetic positions of the less well-known sorocarpic protists considered to be &quot;cellular slime molds,&quot; or aggregatively fruiting amoebae, we isolated <span class="yellow">C. protea</span> and sequenced the nuclear-encoded small subunit ribosomal RNA gene from four samples collected from <span class="blue">cattle</span> farms in the central USA. Phylogenetic analyses of these data place <span class="yellow">C. protea</span> in the eukaryotic supergroup Amoebozoa together with the Tubulinea, in which there has been no previous report of an aggregative fruiting habit. This is consistent with the morphology of the trophozoites. In fact, <span class="yellow">Copromyxa protea</span> is found to be very closely related to <span class="yellow">Hartmannella cantabrigiensis</span> and to a since lost amoeba isolate, <span class="yellow">Hartmannella sp. 4/3Da/10</span>. This new grouping of Copromyxa+<span class="yellow">H. cantabrigiensis</span> is sister to Glaeseria, which together are sister to the Amoebidae (Amoeba+Chaos). We suggest renaming, <span class="yellow">H. cantabrigiensis</span> as <span class="yellow">C. cantabrigiensis</span> and designate isolate 4/3Da/10 as <span class="yellow">C. protea</span>. Future work is needed to see if these newly assigned members of the genus Copromyxa also show evidence of an ability to fruit. 
<h3>21097608</h3>African 2, a clonal complex of <span class="yellow">Mycobacterium bovis</span> epidemiologically important in East Africa. 

We have identified a clonal complex of <span class="yellow">Mycobacterium bovis</span> isolated at high frequency from <span class="blue">cattle</span> in Uganda, Burundi, Tanzania, and Ethiopia. We have named this related group of <span class="yellow">M. bovis</span> strains the African 2 (Af2) clonal complex of <span class="yellow">M. bovis</span>. Af2 strains are defined by a specific chromosomal deletion (RDAf2) and can be identified by the absence of spacers 3 to 7 in their spoligotype patterns. Deletion analysis of <span class="yellow">M. bovis</span> isolates from Algeria, Mali, Chad, Nigeria, Cameroon, South Africa, and Mozambique did not identify any strains of the Af2 clonal complex, suggesting that this clonal complex of <span class="yellow">M. bovis</span> is localized in East Africa. The specific spoligotype pattern of the Af2 clonal complex was rarely identified among isolates from outside Africa, and the few isolates that were found and tested were intact at the RDAf2 locus. We conclude that the Af2 clonal complex is localized to <span class="blue">cattle</span> in East Africa. We found that strains of the Af2 clonal complex of <span class="yellow">M. bovis</span> have, in general, four or more copies of the insertion sequence IS6110, in contrast to the majority of <span class="yellow">M. bovis</span> strains isolated from <span class="blue">cattle</span>, which are thought to carry only one or a few copies. 
<h3>21295520</h3>Increased sampling reveals novel lineages of Entamoeba: consequences of genetic diversity and host specificity for taxonomy and molecular detection. 

To expand the representation for phylogenetic analysis, ten additional complete Entamoeba small-subunit rRNA gene sequences were obtained from <span class="yellow">humans</span>, non-human primates, <span class="blue">cattle</span> and a tortoise. For some novel sequences no corresponding morphological data were available, and we suggest that these organisms should be referred to as ribosomal lineages (RL) rather than being assigned species names at present. To investigate genetic diversity and host specificity of selected Entamoeba species, a total of 91 new partial small subunit rRNA gene sequences were obtained, including 49 from <span class="yellow">Entamoeba coli</span>, 18 from <span class="yellow">Entamoeba polecki</span>, and 17 from <span class="yellow">Entamoeba hartmanni</span>. We propose a new nomenclature for significant variants within established Entamoeba species. Based on current data we propose that the uninucleated-cyst-producing Entamoeba infecting <span class="yellow">humans</span> is called <span class="yellow">Entamoeba polecki</span> and divided into four subtypes (ST1-ST4) and that <span class="yellow">Entamoeba coli</span> is divided into two subtypes (ST1-ST2). New hosts for several species were detected and, while host specificity and genetic diversity of several species remain to be clarified, it is clear that previous reliance on cultivated material has given us a misleading and incomplete picture of variation within the genus Entamoeba. 
</body></html>