
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Chaetomium globosum</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21073977</h3>Cleavage of resveratrol in fungi: characterization of the enzyme Rco1 from <span class="yellow">Ustilago maydis</span>. <br><br><span class="yellow">Ustilago maydis</span>, the causative agent of <span class="yellow">corn smut</span> disease, contains two genes encoding members of the carotenoid cleavage oxygenase family, a group of enzymes that cleave double bonds in different substrates. One of them, Cco1, was formerly identified as a Beta-carotene cleaving enzyme. Here we elucidate the function of the protein encoded by the second gene, termed here as <span class="yellow">Ustilago maydis</span> Resveratrol cleavage oxygenase 1 (Um Rco1). In vitro incubations of heterologously expressed and purified UM Rco1 with different carotenoid and stilbene substrates demonstrate that it cleaves the interphenyl Calpha-CBeta double bond of the phytoalexin resveratrol and its derivative piceatannol. Um Rco1 exhibits a high degree of substrate specificity, as suggested by the lack of activity on carotenoids and the other resveratrol-related compounds tested. The activity of Um Rco1 was confirmed by incubation of <span class="yellow">U. maydis</span> rco1 deletion and over-expression strains with resveratrol. Furthermore, treatment with resveratrol resulted in striking alterations of cell morphology. However, pathogenicity assays indicated that Um rco1 is largely dispensable for biotrophic development. Our work reveals Um Rco1 as the first eukaryotic resveratrol cleavage enzyme identified so far. Moreover, Um Rco1 represents a subfamily of <span class="yellow">fungal</span> enzymes likely involved in the degradation of stilbene compounds, as suggested by the cleavage of resveratrol by homologs from <span class="yellow">Aspergillus fumigatus</span>, <span class="blue">Chaetomium globosum</span> and <span class="yellow">Botryotinia fuckeliana</span>. 
<h3>20854921</h3>Genomic evidence of repeat-induced point mutation (RIP) in filamentous ascomycetes. <br><br>The genomes of 49 filamentous ascomycetes (subphylum Pezizomycotina) were examined by two independent methods for evidence of multiple C->T transitions typical of RIP. At least one transposable element or other repeat family was identified in each genome, and members were assessed for transition and transversion mutations relative to a model of their intact progenitor. Occurrence of RIP was indicated where family members differed by excess of directional transitions over transversions. Transition mutations were quantified by an algorithm taking double mutations in CpG and CpC dinucleotides into account. <span class="yellow">A</span> second method assessed dinucleotide frequency distribution anomalies in whole genomes, a procedure that allowed quantification of fractions of the non-coding genome that had been subject to extensive directional mutation. The results of both methods revealed that RIP-like activity varied greatly, both in extent of mutation and in dinucleotide context for C->T transitions. In the most extreme case, 75% of a <span class="yellow">Blastomyces dermatitidis</span> genome had suffered conspicuous GC-depletion, all of it in the non-coding fraction. Many genomes carried both intact repeats as well as others that had suffered heavily from transitions. Only one species, <span class="blue">Chaetomium globosum</span>, showed no evidence of directional mutation. 
</body></html>