
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>class I</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20962091</h3>Macaque long-term nonprogressors resist superinfection with multiple CD8+ T cell escape variants of <span class="yellow">simian immunodeficiency virus</span>. 

<span class="yellow">Human immunodeficiency virus</span> (<span class="yellow">HIV</span>)-positive individuals can be superinfected with different <span class="yellow">virus</span> strains. Individuals who control an initial <span class="yellow">HIV</span> infection are therefore still at risk for subsequent infection with divergent viruses, but the barriers to such superinfection remain unclear. Here we tested long-term nonprogressors' (LTNPs') susceptibility to superinfection using <span class="yellow">Indian rhesus macaques</span> that express the major histocompatibility complex <span class="red">class I</span> (MHC-I) allele Mamu-B 17, which is associated with control of the pathogenic AIDS <span class="yellow">virus SIVmac239</span>. The Mamu-B 17-restricted CD8(+) T cell repertoire is focused almost entirely on 5 epitopes. We engineered a series of <span class="yellow">SIVmac239</span> variants bearing mutations in 3, 4, or all 5 of these epitopes and used them to serially challenge 2 Mamu-B 17-positive LTNPs. None of the escape variants caused breakthrough replication in LTNPs, although they readily infected Mamu-B 17-negative naive macaques. In vitro competing coculture assays and examination of <span class="yellow">viral</span> evolution in hosts lacking Mamu-B 17 suggested that the mutant viruses had negligible defects in replicative fitness. Both LTNPs maintained robust immune responses, including <span class="yellow">simian immunodeficiency virus</span> (<span class="yellow">SIV</span>)-specific CD8(+) and CD4(+) T cells and neutralizing antibodies. Our results suggest that escape mutations in epitopes bound by &quot;protective&quot; MHC-I molecules may not be sufficient to establish superinfection in LTNPs. 
<h3>21034772</h3>Multiplex RT-PCR for rapid detection and differentiation of <span class="blue">class I</span> and <span class="yellow">class II Newcastle disease viruses</span>. 

A multiplex RT-PCR was developed for detection and differentiation of <span class="blue">class I</span> and <span class="yellow">class II strains of Newcastle disease virus (NDV)</span>. The method was shown to have high specificity and sensitivity. The results obtained from the multiplex RT-PCR for a total of 67 <span class="yellow">NDV</span> field isolates obtained in 2009 were consistent with those obtained by nucleotide sequencing and phylogenetic analysis. <span class="yellow">A</span> phylogenetic tree based on the partial sequences of the F gene revealed that the 67 field isolates of <span class="yellow">NDV</span> could be divided into two classes. Twenty-seven <span class="yellow">NDV</span><span class="yellow">NDV isolates</span> were grouped into <span class="blue">class I</span>, and two genotypes were identified. Most of the <span class="blue">class I</span> isolates were determined to be of genotype 3, with the exception of isolate <span class="yellow">NDV09-034</span>, which belonged to genotype 2. Forty <span class="yellow">class II NDV isolates</span> were divided into three genotypes, namely genotype VII (27 isolates), genotype I (2 isolates) and genotype II (11 isolates). Isolates of genotypes I and II in class II were shown to be related to commercial vaccine strains used commonly in China. All isolates of genotype VII were predicted to be virulent, on the basis of the sequence motif at the cleavage site of the F gene. This genotype has become predominantly responsible for most outbreaks of ND in China in recent years. In conclusion, this multiplex RT-PCR provides a new assay for rapid detection and differentiation of both classes of <span class="yellow">NDV isolates</span>. 
<h3>21097628</h3>Promoter discrimination at <span class="red">class I</span> MarA regulon promoters mediated by glutamic acid 89 of the MarA transcriptional activator of <span class="yellow">Escherichia coli</span>. 

Three paralogous transcriptional activators MarA, SoxS, and Rob, activate > 40 <span class="yellow">Escherichia coli</span> promoters. To understand why MarA does not activate certain promoters as strongly as SoxS, we compared MarA, MarA mutants, and SoxS for their abilities to activate 16 promoters and to bind their cognate marbox binding sites. Replacement of the MarA glutamic acid residue 89 with alanine greatly increased the marbox binding and activation of many <span class="red">class I</span> promoters. Like cells constitutive for SoxS, cells expressing the MarA with the E89A mutation were more resistant to superoxides than those harboring WT MarA. The activities of several other E89 substitutions ranked as follows: E89A > E89G > E89V > WT > E89D. Increased binding and activation occurred only at <span class="red">class I</span> promoters when the 12th base of the promoter's marbox (a position at which there is no known interaction between the marbox and MarA) was not a T residue. Furthermore, WT MarA binding to a synthetic marbox in vitro was enhanced when the phosphate group between positions 12 and 13 was eliminated on one strand. The results demonstrate that relatively minor changes in a single amino acid side chain (e.g., alanine to valine or glutamic acid to aspartic acid) can strongly influence activity despite any evidence that the side chain is involved in positive interactions with either DNA or RNA polymerase. We present a model which attributes the differences in binding and activation to the interference between the Beta- and gamma-carbons of the amino acid at position 89 and the phosphate group between positions 12 and 13. 
</body></html>