
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>HIV-1</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20962079</h3>Blood myeloid dendritic cells from <span class="blue">HIV-1</span>-infected individuals display a proapoptotic profile characterized by decreased Bcl-2 levels and by caspase-3+ frequencies that are associated with levels of plasma viremia and T cell activation in an exploratory study. <br><br>Reduced frequencies of myeloid and plasmacytoid dendritic cell (DC) subsets (mDCs and pDCs, respectively) have been observed in the peripheral blood of <span class="blue">HIV-1</span>-infected individuals throughout the course of disease. Accumulation of DCs in lymph nodes (LNs) may partly account for the decreased numbers observed in blood, but increased DC death may also be a contributing factor. We used multiparameter flow cytometry to evaluate pro- and antiapoptotic markers in blood mDCs and pDCs from untreated <span class="blue">HIV-1</span>-infected donors, from a subset of infected donors before and after receiving antiretroviral therapy (ART), and from uninfected control donors. Blood mDCs, but not pDCs, from untreated <span class="blue">HIV-1</span>-infected donors expressed lower levels of antiapoptotic Bcl-2 than DCs from uninfected donors. <span class="yellow">A</span> subset of <span class="blue">HIV-1</span>-infected donors had elevated frequencies of proapoptotic caspase-3(+) blood mDCs, and positive correlations were observed between caspase-3(+) mDC frequencies and plasma <span class="yellow">viral</span> load and CD8(+) T-cell activation levels. Caspase-3(+) mDC frequencies, but not mDC Bcl-2 expression, were reduced with <span class="yellow">viral</span> suppression on ART. Apoptosis markers on DCs in blood and LN samples from a cohort of untreated, <span class="blue">HIV-1</span>-infected donors with chronic disease were also evaluated. LN mDCs displayed higher levels of Bcl-2 and lower caspase-3(+) frequencies than did matched blood mDCs. Conversely, LN pDCs expressed lower Bcl-2 levels than their blood counterparts. In summary, blood mDCs from untreated <span class="blue">HIV-1</span>-infected subjects displayed a proapoptotic profile that was partially reversed with <span class="yellow">viral</span> suppression, suggesting that DC death may be a factor contributing to blood DC depletion in the setting of chronic, untreated <span class="yellow">HIV</span> disease. 
<h3>20962093</h3><span class="yellow">Human</span> pluripotent stem cells produce natural killer cells that mediate anti-<span class="blue">HIV-1</span> activity by utilizing diverse cellular mechanisms. <br><br>Cell-based therapies against HIV/AIDS have been gaining increased interest. Natural killer (NK) cells are a key component of the innate immune system with the ability to kill diverse tumor cells and virus-infected cells. While NK cells have been shown to play an important role in the control of <span class="blue">HIV-1</span> replication, their functional activities are often compromised in <span class="blue">HIV-1</span>-infected individuals. We have previously demonstrated the derivation of NK cells from <span class="yellow">human</span> embryonic stem cells (hESCs) with the ability to potently kill multiple types of tumor cells both in vitro and in vivo. We now demonstrate the derivation of functional NK cells from <span class="yellow">human</span> induced pluripotent stem cells (iPSCs). More importantly, both hESC- and iPSC-derived NK cells are able to inhibit <span class="blue">HIV-1</span> NL4-3 infection of CEM-GFP cells. Additional studies using <span class="blue">HIV-1</span>-infected <span class="yellow">human</span> primary CD4(+) T cells illustrated that hESC- and iPSC-derived NK cells suppress <span class="blue">HIV-1</span> infection by at least three distinct cellular mechanisms: killing of infected targets through direct lysis, antibody-dependent cellular cytotoxicity, and production of chemokines and cytokines. Our results establish the potential to utilize hESC- and iPSC-derived NK cells to better understand anti-<span class="blue">HIV-1</span> immunity and provide a novel cellular immunotherapeutic approach to treat HIV/AIDS. 
<h3>20962100</h3>Dynamics of two separate but linked <span class="yellow">HIV-1 CRF01_AE</span> outbreaks among injection drug users in Stockholm, Sweden, and Helsinki, Finland. <br><br>Detailed phylogenetic analyses were performed to characterize an <span class="blue">HIV-1</span> outbreak among injection drug users (IDUs) in Stockholm, Sweden, in 2006. This study investigated the source and dynamics of <span class="blue">HIV-1</span> spread during the outbreak as well as associated demographic and clinical factors. Seventy Swedish IDUs diagnosed during 2004 to 2007 were studied. Demographic, clinical, and laboratory data were collected, and the V3 region of the <span class="blue">HIV-1</span> envelope gene was sequenced to allow detailed phylogenetic analyses. The results showed that the Stockholm outbreak was caused by a <span class="yellow">CRF01_AE</span> variant imported from Helsinki, Finland, around 2003, which was quiescent until the outbreak started in 2006. Local Swedish <span class="yellow">subtype B</span> variants continued to spread at a lower rate. The number of new <span class="yellow">CRF01_AE</span> cases over a rooted phylogenetic tree accurately reflected the transmission dynamics and showed a temporary increase, by a factor of 12, in <span class="yellow">HIV</span> incidence during the outbreak. Virus levels were similar in <span class="yellow">CRF01_AE</span> and <span class="yellow">subtype B</span> infections, arguing against differences in contagiousness. Similarly, there were no major differences in other baseline characteristics. Instead, the outbreak in Stockholm (and Helsinki) was best explained by an introduction of <span class="yellow">HIV</span> into a standing network of previously uninfected IDUs. The combination of phylogenetics and epidemiological data creates a powerful tool for investigating outbreaks of <span class="yellow">HIV</span> and other infectious diseases that could improve surveillance and prevention. 
<h3>20980524</h3>Association of TRIM22 with the type 1 interferon response and <span class="yellow">viral</span> control during primary <span class="blue">HIV-1</span> infection. <br><br>Type 1 interferons (IFNs) induce the expression of the tripartite interaction motif (TRIM) family of E3 ligases, but the contribution of these antiviral factors to <span class="yellow">HIV</span> pathogenesis is not completely understood. We hypothesized that the increased expression of select type 1 IFN and TRIM isoforms is associated with a significantly lower likelihood of <span class="blue">HIV-1</span> acquisition and <span class="yellow">viral</span> control during primary <span class="blue">HIV-1</span> infection. We measured IFN-alpha, IFN-Beta, myxovirus resistance protein <span class="yellow">A</span> (MxA), <span class="yellow">human</span> TRIM5alpha (huTRIM5alpha), and TRIM22 mRNA levels in peripheral blood mononuclear cells (PBMCs) of high-risk, <span class="blue">HIV-1</span>-uninfected participants and <span class="blue">HIV-1</span>-positive study participants. Samples were available for 32 uninfected subjects and 28 infected persons, all within 1 year of infection. <span class="blue">HIV-1</span>-positive participants had higher levels of IFN-Beta (P = 0.0005), MxA (P = 0.007), and TRIM22 (P = 0.01) and lower levels of huTRIM5alpha (P &lt; 0.001) than did <span class="blue">HIV-1</span>-negative participants. TRIM22 but not huTRIM5alpha correlated positively with type 1 IFN (IFN-alpha, IFN-Beta, and MxA) (all P &lt; 0.0001). In a multivariate model, increased MxA expression showed a significant positive association with <span class="yellow">viral</span> load (P = 0.0418). Furthermore, TRIM22 but not huTRIM5alpha, IFN-alpha, IFN-Beta, or MxA showed a negative correlation with plasma <span class="yellow">viral</span> load (P = 0.0307) and a positive correlation with CD4(+) T-cell counts (P = 0.0281). In vitro studies revealed that <span class="yellow">HIV</span> infection induced TRIM22 expression in PBMCs obtained from <span class="yellow">HIV</span>-negative donors. Stable TRIM22 knockdown resulted in increased <span class="blue">HIV-1</span> particle release and replication in Jurkat reporter cells. Collectively, these data suggest concordance between type 1 IFN and TRIM22 but not huTRIM5alpha expression in PBMCs and that TRIM22 likely acts as an antiviral effector in vivo. 
<h3>21034776</h3>A versatile vector for the production of pseudotyped viruses expressing gp120 antigens from different clades of primary <span class="blue">HIV-1</span> isolates. <br><br>A novel <span class="blue">HIV-1</span> Env expression vector (SF162-Z) was developed by introducing two new cloning sites on the backbone of an existing vector that produces a full length Env from <span class="blue">HIV-1</span> SF162 isolate. These sites facilitate the swapping of the gp120 portion of the SF162 Env with matching gp120 antigens from <span class="blue">HIV-1</span> isolates of different genetic clades. Final production of functional pseudotyped viruses will express chimeric Env antigens, including gp41 of the parental SF162 and gp120 from other primary isolates. This system is useful for testing the neutralizing sensitivity of partial env gene products frequently identified in <span class="yellow">viral</span> quasi species in patients infected with <span class="yellow">HIV</span> or when only partial gp120 gene products are available. 
<h3>21429803</h3>European guidelines on the clinical management of <span class="blue">HIV-1</span> tropism testing.<br><br>Viral tropism is the ability of viruses to enter and infect specific host cells and is based on the ability of viruses to bind to receptors on those cells. Testing for <span class="yellow">HIV</span> tropism is recommended before prescribing a chemokine receptor blocker. In most European countries, <span class="yellow">HIV</span> tropism is identified with tropism phenotype testing. New data support genotype analysis of the <span class="yellow">HIV</span> third hypervariable loop (V3) for the identification of tropism. The European Consensus Group on clinical management of tropism testing was established to make recommendations to clinicians and clinical virologists. The panel recommends <span class="yellow">HIV</span>-tropism testing for the following groups: drug-naive patients in whom toxic effects are anticipated or for whom few treatment options are available; patients who have poor tolerability to or toxic effects from current treatment or who have CNS pathology; and patients for whom therapy has failed and a change in treatment is considered. In general, an enhanced sensitivity Trofile assay and V3 population genotyping are the recommended methods. Genotypic methods are anticipated to be used more frequently in the clinical setting because of their greater accessibility, lower cost, and faster turnaround time than other methods. For the interpretation of V3 loop genotyping, clinically validated systems should be used when possible. Laboratories doing <span class="yellow">HIV</span> tropism tests should have adequate quality assurance measures. Similarly, close collaboration between <span class="yellow">HIV</span> clinicians and virologists is needed to ensure adequate diagnostic and treatment decisions. 
<h3>20962083</h3>Small-molecule inhibition of <span class="yellow">human immunodeficiency virus type 1</span> infection by <span class="yellow">virus</span> capsid destabilization. <br><br><span class="yellow">Human immunodeficiency virus type 1</span> (<span class="blue">HIV-1</span>) infection is dependent on the proper disassembly of the <span class="yellow">viral</span> capsid, or &quot;uncoating,&quot; in target cells. The <span class="blue">HIV-1</span> capsid consists of a conical multimeric complex of the <span class="yellow">viral</span> capsid protein (CA) arranged in a hexagonal lattice. Mutations in CA that destabilize the <span class="yellow">viral</span> capsid result in impaired infection owing to defects in reverse transcription in target cells. We describe here the mechanism of action of a small molecule <span class="blue">HIV-1</span> inhibitor, PF-3450074 (PF74), which targets CA. PF74 acts at an early stage of <span class="blue">HIV-1</span> infection and inhibits reverse transcription in target cells. We show that PF74 binds specifically to <span class="blue">HIV-1</span> particles, and substitutions in CA that confer resistance to the compound prevent binding. <span class="yellow">A</span> single point mutation in CA that stabilizes the <span class="blue">HIV-1</span> core also conferred strong resistance to the <span class="yellow">virus</span> without inhibiting compound binding. Treatment of <span class="blue">HIV-1</span> particles or purified cores with PF74 destabilized the <span class="yellow">viral</span> capsid in vitro. Furthermore, the compound induced the rapid dissolution of the <span class="blue">HIV-1</span> capsid in target cells. PF74 antiviral activity was promoted by binding of the host protein cyclophilin <span class="yellow">A</span> to the <span class="blue">HIV-1</span> capsid, and PF74 and cyclosporine exhibited mutual antagonism. Our data suggest that PF74 triggers premature <span class="blue">HIV-1</span> uncoating in target cells, thereby mimicking the activity of the retrovirus restriction factor TRIM5alpha. This study highlights uncoating as a step in the <span class="blue">HIV-1</span> life cycle that is susceptible to small molecule intervention. 
<h3>20980495</h3>The B cell response is redundant and highly focused on V1V2 during early subtype C infection in a Zambian seroconverter. <br><br>High-titer autologous neutralizing antibody responses have been demonstrated during early subtype C <span class="yellow">human immunodeficiency virus type 1</span> (<span class="blue">HIV-1</span>) infection. However, characterization of this response against autologous <span class="yellow">virus</span> at the monoclonal antibody (MAb) level has only recently begun to be elucidated. Here we describe five monoclonal antibodies derived from a subtype C-infected seroconverter and their neutralizing activities against pseudoviruses that carry envelope glycoproteins from 48 days (0 month), 2 months, and 8 months after the estimated time of infection. Sequence analysis indicated that the MAbs arose from three distinct B cell clones, and their pattern of neutralization compared to that in patient plasma suggested that they circulated between 2 and 8 months after infection. Neutralization by MAbs representative of each B cell clone was mapped to two residues: position 134 in V1 and position 189 in V2. Mutational analysis revealed cooperative effects between glycans and residues at these two positions, arguing that they contribute to a single epitope. Analysis of the cognate gp120 sequence through homology modeling places this potential epitope near the interface between the V1 and V2 loops. Additionally, the escape mutation R189S in V2, which conferred resistance against all three MAbs, had no detrimental effect on <span class="yellow">virus</span> replication in vitro. Taken together, our data demonstrate that independent B cells repeatedly targeted a single structure in V1V2 during early infection. Despite this assault, a single amino acid change was sufficient to confer complete escape with minimal impact on replication fitness. 
<h3>21126914</h3>Tracking a century of global expansion and evolution of <span class="yellow">HIV</span> to drive understanding and to combat disease. <br><br>Since the isolation of <span class="yellow">HIV</span>, multiple transmissions are thought to have occurred between man and other old-world primates. Assessment of samples from apes and <span class="yellow">human</span> beings with African equatorial forest ancestry has traced the origin of <span class="blue">HIV-1</span> to <span class="yellow">chimpanzees</span>, and dated its most recent common ancestor to 1908. The evolution of <span class="blue">HIV-1</span> has been rapid, which has resulted in a complex classification, worldwide spread, and intermixing of strains; at least 48 circulating recombinant forms are currently identified. In addition to posing a nearly insurmountable challenge for diagnosis, treatment, vaccine development, and prevention, this extreme and divergent evolution has led to differences in virulence between <span class="blue">HIV-1</span> groups, subtypes, or both. Coincidental changes in <span class="yellow">human</span> migration in the Congo river basin also affected spread of disease. Research over the past 25 years and advances in genomic sequencing methods, such as deep DNA sequencing, have greatly improved understanding and analysis of the thousands to millions of full infectious <span class="blue">HIV-1</span> genomes. 
<h3>20962077</h3><span class="yellow">Simian immunodeficiency virus</span> from the <span class="yellow">sooty mangabey</span> and <span class="yellow">rhesus macaque</span> is modified with O-linked carbohydrate. <br><br>Although stretches of serine and threonine are sometimes sites for O-linked carbohydrate attachment, specific sequence and structural determinants for O-linked attachment remain ill defined. The gp120 envelope protein of <span class="yellow">SIVmac239</span> contains a serine-threonine-rich stretch of amino acids at positions 128 to 139. Here we show that lectin protein from <span class="yellow">jackfruit</span> seed (jacalin), which binds to non- and monosialylated core 1 O-linked carbohydrate, potently inhibited the replication of <span class="yellow">SIVmac239</span>. Selection of a jacalin-resistant <span class="yellow">SIVmac239</span> variant population resulted in <span class="yellow">virus</span> with specific substitutions within amino acids 128 to 139. Cloned <span class="yellow">simian immunodeficiency virus</span> (<span class="yellow">SIV</span>) variants with substitutions in the 128-to-139 region had infectivities equivalent to, or within 1 log unit of, that of <span class="yellow">SIVmac239</span> and were resistant to the inhibitory effects of jacalin. Characterization of the <span class="yellow">SIVmac239</span> gp120 O-linked glycome showed the presence of core 1 and core 2 O-linked carbohydrate; a 128-to-139-substituted variant gp120 from jacalin-resistant <span class="yellow">SIV</span> lacked O-linked carbohydrate. Unlike that of <span class="yellow">SIVmac239</span>, the replication of <span class="yellow">HIV-1 strain NL4-3</span> was resistant to inhibition by jacalin. Purified gp120s from four <span class="yellow">SIVmac</span> and <span class="yellow">SIVsm</span> strains bound jacalin strongly in an enzyme-linked immunosorbent assay, while nine different <span class="blue">HIV-1</span> gp120s, two <span class="yellow">SIVcpz</span> gp120s, and 128-to-139-substituted <span class="yellow">SIVmac239</span> gp120 did not bind jacalin. The ability or inability to bind jacalin thus correlated with the presence of the serine-threonine-rich stretch in the <span class="yellow">SIVmac</span> and <span class="yellow">SIVsm</span> gp120s and the absence of such stretches in the <span class="yellow">SIVcpz</span> and <span class="blue">HIV-1</span> gp120s. Consistent with sequence predictions, two <span class="yellow">HIV-2</span> gp120s bound jacalin, while one did not. These data demonstrate the presence of non- and monosialylated core 1 O-linked carbohydrate on the gp120s of <span class="yellow">SIVmac</span> and <span class="yellow">SIVsm</span> and the lack of these modifications on <span class="blue">HIV-1</span> and <span class="yellow">SIVcpz</span> gp120s. 
<h3>20980512</h3>Serine-threonine ubiquitination mediates downregulation of BST-2/tetherin and relief of restricted virion release by <span class="blue">HIV-1</span> Vpu. <br><br>The <span class="blue">HIV-1</span> protein Vpu counteracts the antiviral activity of the innate restriction factor BST-2/tetherin by a mechanism that partly depends on its interaction with Beta-TrCP, a substrate adaptor for an SCF (Skp-Cullin 1-F box) E3 ubiquitin ligase complex. This suggests that Vpu stimulates the ubiquitination of BST-2 and that this underlies the relief of restriction. Here, we show that Vpu stimulates ubiquitination of BST-2. Mutation of all potential ubiquitination sites in the cytoplasmic domain of BST-2, including lysines, cysteines, serines, and threonines, abrogates Vpu-mediated ubiquitination. However, a serine-threonine-serine sequence specifically mediates the downregulation of BST-2 from the cell surface and the optimal relief of restricted virion release. Serine-threonine ubiquitination of BST-2 is likely part of the mechanism by which Vpu counteracts innate defenses. 
<h3>20980522</h3>RNA aptamers directed to <span class="yellow">human immunodeficiency virus type 1</span> Gag polyprotein bind to the matrix and nucleocapsid domains and inhibit <span class="yellow">virus</span> production. <br><br>Gag orchestrates the assembly and release of <span class="yellow">human immunodeficiency virus type 1</span> (<span class="blue">HIV-1</span>) particles. We explored here the potential of anti-Gag RNA aptamers to inhibit <span class="blue">HIV-1</span> replication. In vitro, RNA aptamers raised against an <span class="blue">HIV-1</span> Gag protein, lacking the N-terminal myristate and the C-terminal p6 (DP6-Gag), could bind to matrix protein (MA), nucleocapsid protein (NC), or entire DP6-Gag protein. Upon cotransfection with pNL4-3.Luc molecular clone into 293T cells, six of the aptamers caused mild inhibition (2- to 3-fold) in the extracellular capsid levels, and one aptamer displayed 20-fold inhibition. The reduction was not due to a release defect but reflected Gag mRNA levels. We hypothesized that the aptamers influence genomic RNA levels via perturbation of specific Gag-genomic RNA interactions. Binding studies revealed that the &quot;NC-binders&quot; specifically compete with the packaging signal (psi) of <span class="blue">HIV-1</span> for binding to DP6-Gag. Therefore, we tested the ability of two NC-binders to inhibit viruses containing psi-region deletions (DeltaSL1 or DeltaSL3) and found that the NC-binders were no longer able to inhibit Gag synthesis. The inability of these aptamers to inhibit psi-deleted viruses correlated with the absence of competition with the corresponding psi transcripts lacking SL1 or SL3 for binding DP6-Gag in vitro. These results indicate that the NC-binding aptamers disrupt Gag-genomic RNA interaction and negatively affect genomic RNA transcription, processing, or stability. Our results reveal an essential interaction between <span class="blue">HIV-1</span> Gag and the psi-region that may be distinct from that which occurs during the encapsidation of genomic RNA. Thus, anti-Gag aptamers can be an effective tool to perturb Gag-genomic RNA interactions. 
</body></html>