
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>human adenovirus</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21047956</h3>Genetic analysis of B55alpha/Cdc55 protein phosphatase 2A subunits: association with the <span class="yellow">adenovirus</span> E4orf4 protein. <br><br>The <span class="yellow">human</span><span class="red">human adenovirus</span> E4orf4 protein is toxic in both <span class="yellow">human</span> tumor cells and <span class="yellow">Saccharomyces cerevisiae</span>. Previous studies indicated that most of this toxicity is dependent on an interaction of E4orf4 protein with the B55 class of regulatory subunits of protein phosphatase 2A (PP2A) and in <span class="yellow">yeast</span> with the B55 homolog Cdc55. We have found previously that E4orf4 inhibits PP2A activity against at least some substrates. In an attempt to understand the mechanism of this inhibition, we used a genetic approach to identify residues in the seven-bladed Beta-propeller proteins B55alpha and Cdc55 required for E4orf4 binding. In both cases, amino-terminal polypeptides composed only of blade 1 and at least part of blade 2 were found to bind E4orf4 and overexpression blocked E4orf4 toxicity in <span class="yellow">yeast</span>. Furthermore, certain amino acid substitutions in blades 1 and 2 within full-length B55alpha and Cdc55 resulted in loss of E4orf4 binding. Recent mutational analysis has suggested that segments of blades 1 and 2 present on the top face of B55alpha form part of the &quot;substrate-binding groove.&quot; Additionally, these segments are in close proximity to the catalytic C subunit of the PP2A holoenzyme. Thus, our results are consistent with the hypothesis that E4orf4 binding could affect the access of substrates, resulting in the failure to dephosphorylate some PP2A substrates. 
<h3>21047958</h3>Stepwise loss of fluorescent core protein V from <span class="blue">human adenovirus</span> during entry into cells. <br><br><span class="yellow">Human adenoviruses (Ads)</span> replicate and assemble particles in the nucleus. They organize a linear double-strand DNA genome into a condensed core with about 180 nucleosomes, by the <span class="yellow">viral</span> proteins VII (pVII), pX, and pV attaching the DNA to the capsid. Using reverse genetics, we generated a novel, nonconditionally replicating <span class="yellow">Ad</span> reporter by inserting green fluorescent protein (GFP) at the amino terminus of pV. Purified Ad2-GFP-pV virions had an oversized complete genome and incorporated about 38 GFP-pV molecules per virion, which is about 25% of the pV levels in Ad2. GFP-pV cofractionated with the DNA core, like pV, and newly synthesized GFP-pV had a subcellular localization indistinguishable from that of pV, indicating that GFP-pV is a valid reporter for pV. Ad2-GFP-pV completed the replication cycle, although at lower yields than Ad2. Incoming GFP-pV (or pV) was not imported into the nucleus. Virions lost GFP-pV at two points during the infection process: at entry into the cytosol and at the nuclear pore complex, where capsids disassemble. Disassembled capsids, positive for the conformation-specific antihexon antibody R70, were devoid of GFP-pV. The loss of GFP-pV was reduced by the macrolide antibiotic leptomycin B (LMB), which blocks nuclear export and <span class="yellow">adenovirus</span> attachment to the nuclear pore complex. LMB inhibited the appearance of R70 epitopes on Ad2 and Ad2-GFP-pV, indicating that the loss of GFP-pV from Ad2-GFP-pV is an authentic step in the <span class="yellow">adenovirus</span> uncoating program. Ad2-GFP-pV is genetically complete and hence enables detailed analyses of infection and spreading dynamics in cells and model organisms or assessment of oncolytic <span class="yellow">adenoviral</span> potential. 
</body></html>