
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>influenza</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20933017</h3>Adaptation of a Madin-Darby <span class="yellow">canine</span> kidney cell line to suspension growth in serum-free media and comparison of its ability to produce avian <span class="yellow">influenza virus</span> to Vero and BHK21 cell lines. 

Madin-Darby <span class="yellow">canine</span> kidney (MDCK) cells are currently considered for <span class="red">influenza</span> vaccine manufacturing. <span class="yellow">A</span> drawback of these cells is their anchorage dependent growth, which greatly complicates process scale-up. In this paper a novel MDCK cell line (MDCK-SFS) is described that grows efficiently in suspension and retained high expression levels of both alpha-2,6 and alpha-2,3 sialic acid receptors, which bind preferably to <span class="yellow">human</span> and avian <span class="yellow">influenza viruses</span>, respectively. The production of avian <span class="yellow">influenza virus</span> by BHK21, Vero and MDCK-SFS cell lines was compared. Although BHK21 cells consisted of two populations, one of which lacks the alpha-2,3 receptor, they supported the replication of two <span class="red">influenza</span> strains to high titres. However, BHK21 cells are generally not applicable for <span class="red">influenza</span> production since they supported the replication of six further strains poorly. MDCK-SFS cells yielded the highest infectious <span class="yellow">virus</span> titres and <span class="yellow">virus</span> genome equivalent concentration for five of the eight <span class="red">influenza</span> strains analyzed and the highest hemagglutination activity for all eight <span class="yellow">virus</span> strains. Taken together with their suitability for suspension growth this makes the MDCK-SFS cell line potentially useful for large scale <span class="yellow">influenza virus</span> production. 
<h3>20962084</h3>Modifications in the polymerase genes of a <span class="yellow">swine</span>-like triple-reassortant <span class="yellow">influenza virus</span> to generate live attenuated vaccines against 2009 pandemic <span class="yellow">H1N1 viruses</span>. 

On 11 June 2009, the World Health Organization (WHO) declared that the outbreaks caused by novel <span class="yellow">swine-origin influenza A (H1N1) virus</span> had reached pandemic proportions. The pandemic <span class="yellow">H1N1</span> (H1N1pdm) <span class="yellow">virus</span> is the predominant <span class="yellow">influenza virus</span> strain in the <span class="yellow">human</span> population. It has also crossed the species barriers and infected turkeys and <span class="yellow">swine</span> in several countries. Thus, the development of a vaccine that is effective in multiple animal species is urgently needed. We have previously demonstrated that the introduction of temperature-sensitive mutations into the PB2 and PB1 genes of an avian <span class="yellow">H9N2 virus</span>, combined with the insertion of a hemagglutinin (HA) tag in PB1, resulted in an attenuated (att) vaccine backbone for both <span class="yellow">chickens</span> and <span class="yellow">mice</span>. Because the new pandemic strain is a triple-reassortant (TR) <span class="yellow">virus</span>, we chose to introduce the double attenuating modifications into a <span class="yellow">swine</span>-like TR <span class="yellow">virus</span> isolate, <span class="yellow">A/turkey/OH/313053/04 (H3N2)</span> (ty/04), with the goal of producing live attenuated <span class="red">influenza</span> vaccines (LAIV). This genetically modified backbone had impaired polymerase activity and restricted <span class="yellow">virus</span> growth at elevated temperatures. In vivo characterization of two <span class="yellow">H1N1</span> vaccine candidates generated using the ty/04 att backbone demonstrated that this vaccine is highly attenuated in <span class="yellow">mice</span>, as indicated by the absence of signs of disease, limited replication, and minimum histopathological alterations in the respiratory tract. <span class="yellow">A</span> single immunization with the ty/04 att-based vaccines conferred complete protection against a lethal <span class="yellow">H1N1pdm</span><span class="yellow">H1N1pdm virus</span> infection in <span class="yellow">mice</span>. More importantly, vaccination of <span class="yellow">pigs</span> with a ty/04 att-H1N1 vaccine candidate resulted in sterilizing immunity upon an aggressive intratracheal challenge with the 2009 <span class="yellow">H1N1</span> pandemic <span class="yellow">virus</span>. Our studies highlight the safety of the ty/04 att vaccine platform and its potential as a master donor strain for the generation of live attenuated vaccines for <span class="yellow">humans</span> and livestock. 
<h3>20962087</h3>The magnitude of local immunity in the lungs of <span class="yellow">mice</span> induced by live attenuated <span class="red">influenza</span> vaccines is determined by local <span class="yellow">viral</span> replication and induction of cytokines. 

While live attenuated <span class="red">influenza</span> vaccines (LAIVs) have been shown to be efficacious and have been licensed for <span class="yellow">human</span> use, the surface glycoproteins hemagglutinin (HA) and neuraminidase (NA) have to be updated for optimal protective efficacy. Little is known about the effect of different HA and NA proteins on the immunogenicity of LAIVs developed using the same backbone. <span class="yellow">A</span> panel of LAIVs that share the internal protein genes, with unique HA and NA gene segments from different <span class="red">influenza</span> subtypes, was rescued by reverse genetics, and a comparative study of immune responses induced by these vaccines was conducted in <span class="yellow">mice</span>. The results suggest that the magnitude of lung immunity, including pulmonary IgA antibody and memory CD8(+) T lymphocytes, induced by the vaccines depends on the replication efficiency of the LAIVs, as well as the induction of cytokines/chemokines in the lungs. However, these factors are not important in determining systemic immunity such as serum antibody titers and memory CD8(+) T cells in the spleen. <span class="yellow">A</span> qualitative analysis of immune responses induced by a single dose of an <span class="yellow">H5N1</span> LAIV revealed that the vaccine induced robust systemic and mucosal immunity in <span class="yellow">mice</span>. In addition, antibodies and memory lymphocytes established in the lungs following vaccination were required for protection against lethal challenge with homologous and heterologous <span class="yellow">H5N1 viruses</span>. Our results highlight the different requirements for inducing systemic and lung immunity that can be explored for the development of pulmonary immunity for protection against respiratory pathogens. 
<h3>20980521</h3>Characterization of cross-reactive CD8+ T-cell recognition of HLA-A2-restricted HIV-Gag (SLYNTVATL) and HCV-NS5b (ALYDVVSKL) epitopes in individuals infected with <span class="yellow">human immunodeficiency</span> and <span class="yellow">hepatitis C viruses</span>. 

The immunologic mechanisms underlying the faster progression of <span class="yellow">hepatitis C virus</span> (<span class="yellow">HCV</span>) disease in the presence of <span class="yellow">human immunodeficiency virus</span> (<span class="yellow">HIV</span>) coinfection are not clearly understood. T-cell cross-reactivity between <span class="yellow">HCV</span> and <span class="red">influenza</span> virus-specific epitopes has been associated with rapid progression of <span class="yellow">HCV</span> disease (S. Urbani, B. Amadei, P. Fisicaro, M. Pilli, G. Missale, <span class="yellow">A</span>. Bertoletti, and C. Ferrari, J. Exp. Med. 201:675-680, 2005). We asked whether T-cell cross-reactivity between <span class="yellow">HCV</span> and <span class="yellow">HIV</span> could exist during <span class="yellow">HCV</span>/<span class="yellow">HIV</span> coinfection and affect pathogenesis. Our search for amino acid sequence homology between the <span class="yellow">HCV</span> and <span class="yellow">HIV</span> proteomes revealed two similar HLA-A2-restricted epitopes, HIV-Gag (SLYNTVATL [HIV-SL9]) and <span class="yellow">HCV</span>-NS5b (ALYDVVSKL [<span class="yellow">HCV</span>-AL9]). We found that 4 out of 20 HLA-A2-positive (HLA-A2(+)) <span class="yellow">HIV</span>-infected individuals had CD8(+) T cells that recognized both the <span class="yellow">HIV</span>-SL9 and HCV-AL9 epitopes. However, the AL9 epitope was generally shown to be a weak agonist. Although HCV-monoinfected individuals in our study did not show AL9-specific responses, we found that about half of HCV/<span class="yellow">HIV</span>-coinfected individuals had dual responses to both epitopes. High dual T-cell recognition among coinfected subjects was usually due to separate T-cell populations targeting each epitope, as determined by pentamer staining. The one individual demonstrating cross-reactive T cells to both epitopes showed the most advanced degree of liver disease. In coinfected individuals, we observed a positive correlation between the magnitudes of T-cell responses to both the SL9 and the AL9 epitopes, which was also positively associated with the clinical parameter of liver damage. Thus, we find that <span class="yellow">HIV</span> infection induces T cells that can cross-react to heterologous viruses or prime for T cells that are closely related in sequence. However, the induction of cross-reactive T cells may not be associated with control of disease caused by the heterologous <span class="yellow">virus</span>. This demonstrates that degeneracy of HIV-specific T cells may play a role in the immunopathology of <span class="yellow">HCV</span>/<span class="yellow">HIV</span> coinfection. 
<h3>21371656</h3>Seasonal <span class="red">influenza</span> epidemiology in sub-Saharan Africa: a systematic review.

Acute respiratory infection (ARI) is a leading cause of mortality worldwide, of which <span class="red">influenza</span> is an important cause that can be prevented with vaccination. We did a systematic review of research published from 1980 to 2009 on seasonal <span class="red">influenza</span> epidemiology in sub-Saharan Africa to identify data strengths and weaknesses that might affect policy decisions, to assess the state of knowledge on <span class="red">influenza</span> disease burden, and to ascertain unique features of <span class="red">influenza</span> epidemiology in the region. We assessed 1203 papers, reviewed 104, and included 49 articles. 1-25% of outpatient ARI visits were caused by <span class="red">influenza</span> (11 studies; mean 9*5%; median 10%), whereas 0*6-15*6% of children admitted to hospital for ARI had <span class="red">influenza</span> identified (15 studies; mean 6*6%; median 6*3%). Influenza was highly seasonal in southern Africa. Other data were often absent, particularly direct measurement of <span class="red">influenza</span> incidence rates for all ages, within different patient settings (outpatient, inpatient, community), and for all countries. Data from sub-Saharan Africa are insufficient to allow most countries to prioritise strategies for <span class="red">influenza</span> prevention and control. Key data gaps include incidence and case-fatality ratios for all ages, the contribution of <span class="red">influenza</span> towards admission of adults to hospital for ARI, representative seasonality data, economic burden, and the interaction of <span class="red">influenza</span> with prevalent disorders in Africa, such as malaria, <span class="yellow">HIV</span>, and malnutrition. 
<h3>20951165</h3>The validation of a real-time RT-PCR assay which detects <span class="yellow">influenza A</span><span class="yellow">A</span> and types simultaneously for <span class="yellow">influenza A H1N1</span> (2009) and oseltamivir-resistant (H275Y) <span class="yellow">influenza A H1N1</span><span class="yellow">A</span> <span class="yellow">H1N1</span> (2009). 

Influenza <span class="yellow">A</span> <span class="yellow">H1N1</span> (2009) was declared by the World Health Organisation (WHO) as the first <span class="red">influenza</span> pandemic of the 21st century. Rapid detection of <span class="yellow">influenza A</span><span class="yellow">A</span> and differentiation of <span class="yellow">influenza A H1N1</span><span class="yellow">A</span> <span class="yellow">H1N1</span> (2009) and seasonal <span class="yellow">influenza A</span><span class="yellow">A</span> is beneficial. In addition the rapid detection of antiviral resistant strains of <span class="yellow">influenza A H1N1</span><span class="yellow">A</span> <span class="yellow">H1N1</span> (2009) would be useful for clinicians to allow for change to an effective treatment at a much earlier stage if resistance is found. It was the aim of this study to develop a real-time RT-PCR that can detect all <span class="yellow">influenza A viruses</span> and type simultaneously for <span class="yellow">influenza A H1N1</span> (2009) and oseltamivir resistant (H275Y) <span class="yellow">influenza A H1N1</span> (2009). This multiplex assay will allow laboratories to screen respiratory samples for all types of <span class="yellow">influenza A</span>, <span class="yellow">influenza A H1N1</span> (2009) <span class="yellow">virus</span> and oseltamivir resistant (H275Y) <span class="yellow">influenza A H1N1</span> (2009) <span class="yellow">virus</span> in a rapid and cost effective format, ensuring that typing methods for seasonal and avian viruses are used on a smaller subset of samples. Since most virology laboratories already offer a molecular service for <span class="yellow">influenza A</span><span class="yellow">A</span> this assay could easily be implemented into most areas at little cost therefore increasing local access to resistance testing. 
<h3>21047957</h3>Modification of nonstructural protein 1 of <span class="yellow">influenza A virus</span> by SUMO1.

Nonstructural protein 1 (NS1) is one of the major factors resulting in the efficient infection rate and high level of virulence of <span class="yellow">influenza A virus</span>. Although consisting of only approximately 230 amino acids, NS1 has the ability to interfere with several systems of the host <span class="yellow">viral</span> defense. In the present study, we demonstrate that NS1 of the highly pathogenic avian <span class="blue">influenza</span> <span class="yellow">A/Duck/Hubei/L-1/2004 (H5N1) virus</span> interacts with <span class="yellow">human</span> Ubc9, which is the E2 conjugating enzyme for sumoylation, and we show that SUMO1 is conjugated to <span class="yellow">H5N1</span> NS1 in both transfected and infected cells. Furthermore, two lysine residues in the C terminus of NS1 were identified as SUMO1 acceptor sites. When the SUMO1 acceptor sites were removed by mutation, NS1 underwent rapid degradation. Studies of different <span class="yellow">influenza A virus</span> strains of <span class="yellow">human</span> and avian origin showed that the majority of viruses possess an NS1 protein that is modified by SUMO1, except for the recently emerged <span class="yellow">swine-origin influenza A virus (S-OIV) (H1N1)</span>. Interestingly, growth of a sumoylation-deficient WSN <span class="yellow">virus</span> mutant was retarded compared to that of wild-type <span class="yellow">virus</span>. Together, these results indicate that sumoylation enhances NS1 stability and thus promotes rapid growth of <span class="yellow">influenza A virus</span>. 
</body></html>