
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.linneaus_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 

table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>K-12</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21097626</h3>Altered regulation of the OmpF porin by Fis in <span class="annotated ">Escherichia coli</span> during an evolution experiment and between B and <span class="match unannotated ">K-12</span> strains. <br><br>The phenotypic plasticity of global regulatory networks provides bacteria with rapid acclimation to a wide range of environmental conditions, while genetic changes in those networks provide additional flexibility as bacteria evolve across long time scales. We previously identified mutations in the global regulator-encoding gene fis that enhanced organismal fitness during a long-term evolution experiment with <span class="annotated ">Escherichia coli</span>. To gain insight into the effects of these mutations, we produced two-dimensional protein gels with strains carrying different fis alleles, including a beneficial evolved allele and one with an in-frame deletion. We found that Fis controls the expression of the major porin-encoding gene ompF in the <span class="annotated ">E. coli</span> B-derived ancestral strain used in the evolution experiment, a relationship that has not been described before. We further showed that this regulatory connection evolved over two different time scales, perhaps explaining why it was not observed before. On the longer time scale, we showed that this regulation of ompF by Fis is absent from the more widely studied <span class="annotated ">K-12 strain</span> and thus is specific to the <span class="annotated ">B strain</span>. On a shorter time scale, this regulatory linkage was lost during 20,000 generations of experimental evolution of the <span class="annotated ">B strain</span>. Finally, we mapped the Fis binding sites in the ompF regulatory region, and we present a hypothetical model of ompF expression that includes its other known regulators. 
<h3>21037010</h3>Inactivation of the RluD pseudouridine synthase has minimal effects on growth and ribosome function in wild-type <span class="annotated ">Escherichia coli</span> and <span class="annotated ">Salmonella enterica</span>. <br><br>The <span class="annotated ">Escherichia coli</span> rluD gene encodes a pseudouridine synthase responsible for the pseudouridine (Psi) modifications at positions 1911, 1915, and 1917 in helix 69 of 23S rRNA. It has been reported that deletion of rluD in <span class="annotated ">K-12 strains of E. coli</span> is associated with extremely slow growth, increased readthrough of stop codons, and defects in 50S ribosomal subunit assembly and 30S-50S subunit association. Suppressor mutations in the prfB and prfC genes encoding release factor 2 (RF2) and RF3 that restore the wild type-growth rate and also correct the ribosomal defects have now been isolated. These suppressors link helix 69 Psi residues with the termination phase of protein synthesis. However, further genetic analysis reported here also reveals that the slow growth and other defects associated with inactivation of rluD in <span class="annotated ">E. coli K-12</span> strains are due to a defective RF2 protein, with a threonine at position 246, which is present in all <span class="match annotated ">K-12</span> strains. This is in contrast to the more typical alanine found at this position in most bacterial RF2s, including those of other <span class="annotated ">E. coli</span> strains. Inactivation of rluD in <span class="annotated ">E. coli</span> strains containing the prfB allele from <span class="annotated ">E. coli B</span> or in <span class="annotated ">Salmonella enterica</span>, both carrying an RF2 with Ala246, has negligible effects on growth, termination, or ribosome function. The results indicate that, in contrast to those in wild bacteria, termination functions in <span class="annotated ">E. coli K-12</span> strains carrying a partially defective RF2 protein are especially susceptible to perturbation of ribosome-RF interactions, such as that caused by loss of h69 Psi modifications. 
</body></html>