
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Magnaporthe oryzae</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20719251</h3>Diversification and evolution of the avirulence gene AVR-Pita1 in field isolates of <span class="blue">Magnaporthe oryzae</span>. 

Rice blast disease is the single most destructive plant disease that threatens stable <span class="yellow">rice</span> production worldwide. Race-specific resistance to the <span class="yellow">rice</span><span class="yellow">rice blast pathogen</span> has not been durable and the mechanism by which the resistance is overcome remains largely unknown. Here we report the molecular mechanisms of diversification and the instability of the avirulence gene AVR-Pita1 in field strains of <span class="blue">Magnaporthe oryzae</span> interacting with the host resistance gene Pi-ta and triggering race-specific resistance. Two-base-pair insertions resulting in frame-shift mutations and partial and complete deletions of AVR-Pita1 were identified in virulent isolates. Moreover, a total of 38 AVR-Pita1 haplotypes encoding 27 AVR-Pita1 variants were identified among 151 avirulent isolates. Most DNA sequence variation was found to occur in the exon region resulting in amino acid substitution. These findings demonstrate that AVR-Pita1 is under positive selection and mutations of AVR-Pita1 are responsible for defeating race-specific resistance in nature. 
<h3>21600998</h3>The cell cycle gene MoCDC15 regulates hyphal growth, asexual development and plant infection in the <span class="yellow">rice blast pathogen</span> <span class="blue">Magnaporthe oryzae</span>. 

Rice blast, caused by the pathogen <span class="blue">Magnaporthe oryzae</span>, is a serious hindrance to <span class="yellow">rice</span> production and has emerged as an important model for the characterization of molecular mechanisms relevant to pathogenic development in plants. Similar to other pathogenic fungi, conidiation plays a central role in initiation of <span class="yellow">M.oryzae</span> infection and spread over a large area. However, relatively little is known regarding the molecular mechanisms that underlie conidiation in <span class="yellow">M. oryzae</span>. To better characterize these mechanisms, we identified a conidiation-defective mutant, <span class="yellow">ATMT0225B6</span> (<span class="yellow">MoCDC15(T-DNA)</span>), in which a T-DNA insertion disrupted a gene that encodes a homolog of fission <span class="yellow">yeast</span> cdc15, and generated a second strain containing a disruption in the same allele (DeltaMoCDC15(T-DNA)). The cdc15 gene has been shown to act as a coordinator of the cell cycle in <span class="yellow">yeast</span>. Functional analysis of the <span class="yellow">MoCDC15(T-DNA)</span> and DeltaMoCDC15(T-DNA) mutants revealed that MoCDC15 is required for conidiation, preinfection development and pathogenicity in <span class="yellow">M. oryzae</span>. Conidia from these mutants were viable, but failed to adhere to hydrophobic surface, a crucial step required for subsequent pathogenic development. All phenotypic defects observed in mutants were rescued in a strain complemented with <span class="yellow">wild type MoCDC15</span>. Together, these data indicate that MoCDC15 functions as a coordinator of several biological processes important for pathogenic development in <span class="yellow">M. oryzae</span>. 
<h3>21241815</h3>Genetic and molecular characterization of a blue light photoreceptor MGWC-1 in <span class="yellow">Magnaporth oryzae</span>. 

Three key factors involved in successful plant disease development include the presence of a susceptible host, a virulent pathogen, and a disease-conducive environment. Our understanding of how environmental factors influence disease-conducive or disease-suppressive conditions, and how a pathogen advantageously capitalizes on them, is quite limited. Utilizing the model pathosystem <span class="blue">Magnaporthe oryzae</span>-<span class="yellow">Oryza sativa</span>, we found a significant light-dependent disease suppression. Our genetic data suggest that the blue-light receptor MGWC-1 in <span class="yellow">M. oryzae</span> is involved in light-dependent disease suppression during the dark-phase (disease-conducive light condition) immediately after pathogen-host contact. Sensing &quot;darkness&quot; is accomplished by MGWC-1, a blue-light receptor in <span class="yellow">M. oryzae</span>. To explore the potential molecular mechanisms of light-dependent disease suppression we performed a genome-wide microarray experiment and identified several groups of gene families that are differentially regulated during the light-to-dark transition. Our genetic and molecular data provide insights into how a <span class="yellow">fungal</span> pathogen utilizes ambient light signals for successful disease development. 
<h3>21406243</h3>Potassium and sodium uptake systems in fungi. The transporter diversity of <span class="blue">Magnaporthe oryzae</span>. 

In this study, we report an inventory of the K(+) uptake systems in 62 <span class="yellow">fungal</span> species for which the complete genome sequences are available. This inventory reveals that three types of K(+) uptake systems, TRK and HAK transporters and ACU ATPases, are widely present in several combinations across <span class="yellow">fungal</span> species. PAT ATPases are less frequently present and are exceptional in Ascomycota. The genome of <span class="blue">Magnaporthe oryzae</span> contains four TRK, one HAK, and two ACU genes. The study of the expression of these genes at high K(+), K(+) starvation, and in infected <span class="yellow">rice</span> leaves revealed that the expression of four genes, ACU1, ACU2, HAK1, and TRK1 is much lower than that of TRK2, TRK3, and TRK4, except under K(+) starvation. The two ACU ATPases were cloned and functionally identified as high-affinity K(+) or Na(+) uptake systems. These two ATPases endow <span class="yellow">Saccharomyces cerevisiae</span> with the capacity to grow for several generations in low Na(+) concentrations when K(+) was absent, which produces a dramatic increase of cellular Na(+)/K(+) ratio. 
</body></html>