
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Methanococcus maripaludis</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21075925</h3>Genetic and mass spectrometry analyses of the unusual type IV-like pili of the archaeon <span class="blue">Methanococcus maripaludis</span>. <br><br>The structure of pili from the archaeon <span class="blue">Methanococcus maripaludis</span> is unlike that of any bacterial pili. However, genetic analysis of the genes involved in the formation of these pili has been lacking until this study. Pili were isolated from a nonflagellated (DeltaflaK) mutant and shown by sodium dodecyl sulfate-polyacrylamide gel electrophoresis to consist primarily of subunits with an apparent molecular mass of 17 kDa. In-frame deletions were created in three genes, MMP0233, MMP0236, and MMP0237, which encode proteins with bacterial type IV pilin-like signal peptides previously identified by in silico methodology as likely candidates for pilus structural proteins. Deletion of MMP0236 or MMP0237 resulted in mutant cells completely devoid of pili on the cell surface, while deletion of the third pilin-like gene, MMP0233, resulted in cells greatly reduced in the number of pili on the surface. Complementation with the deleted gene in each case returned the cells to a piliated state. Surprisingly, mass spectrometry analysis of purified pili identified the major structural pilin as another type IV pilin-like protein, MMP1685, whose gene is located outside the first pilus locus. This protein was found to be glycosylated with an N-linked branched pentasaccharide glycan. Deletion and complementation analysis confirmed that MMP1685 is required for piliation. 
<h3>21097629</h3>Genome copy numbers and gene conversion in <span class="yellow">methanogenic archaea</span>.<br><br>Previous studies revealed that one species of <span class="yellow">methanogenic archaea</span>, <span class="yellow">Methanocaldococcus jannaschii</span>, is polyploid, while a second species, <span class="yellow">Methanothermobacter thermoautotrophicus</span>, is diploid. To further investigate the distribution of ploidy in <span class="yellow">methanogenic archaea</span>, species of two additional genera-<span class="yellow">Methanosarcina acetivorans</span> and <span class="blue">Methanococcus maripaludis</span>-were investigated. <span class="yellow">M. acetivorans</span> was found to be polyploid during fast growth (t(D) = 6 h; 17 genome copies) and oligoploid during slow growth (doubling time = 49 h; 3 genome copies). <span class="yellow">M. maripaludis</span> has the highest ploidy level found for any archaeal species, with up to 55 genome copies in exponential phase and ca. 30 in stationary phase. <span class="yellow">A</span> compilation of archaeal species with quantified ploidy levels reveals a clear dichotomy between Euryarchaeota and Crenarchaeota: none of seven euryarchaeal species of six genera is monoploid (haploid), while, in contrast, all six crenarchaeal species of four genera are monoploid, indicating significant genetic differences between these two kingdoms. Polyploidy in asexual species should lead to accumulation of inactivating mutations until the number of intact chromosomes per cell drops to zero (called &quot;Muller's ratchet&quot;). <span class="yellow">A</span> mechanism to equalize the genome copies, such as gene conversion, would counteract this phenomenon. Making use of a previously constructed heterozygous mutant strain of the polyploid <span class="yellow">M. maripaludis</span> we could show that in the absence of selection very fast equalization of genomes in <span class="yellow">M. maripaludis</span> took place probably via a gene conversion mechanism. In addition, it was shown that the velocity of this phenomenon is inversely correlated to the strength of selection. 
</body></html>