
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>M. loti</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20868395</h3>Role of lupeol synthase in <span class="yellow">Lotus japonicus</span> nodule formation.<br><br>* Triterpenes are plant secondary metabolites, derived from the cyclization of 2,3-oxidosqualene by oxidosqualene cyclases (OSCs). Here, we investigated the role of lupeol synthase, encoded by OSC3, and its product, lupeol, in developing roots and nodules of the model legume <span class="yellow">Lotus japonicus</span>. * The expression patterns of OSC3 in different developmental stages of uninfected roots and in roots infected with <span class="yellow">Mesorhizobium loti</span> were determined. The tissue specificity of OSC3 expression was analysed by in situ hybridization. Functional analysis, in which transgenic <span class="yellow">L. japonicus</span> roots silenced for OSC3 were generated, was performed. The absence of lupeol in the silenced plant lines was determined by GC-MS. * The expression of ENOD40, a marker gene for nodule primordia initiation, was increased significantly in the OSC3-silenced plant lines, suggesting that lupeol influences nodule formation. Silenced plants also showed a more rapid nodulation phenotype, consistent with this. Exogenous application of lupeol to <span class="blue">M. loti</span>-infected wild-type plants provided further evidence for a negative regulatory effect of lupeol on the expression of ENOD40. * The synthesis of lupeol in <span class="yellow">L. japonicus</span> roots and nodules can be solely attributed to OSC3. Taken together, our data suggest a role for lupeol biosynthesis in nodule formation through the regulation of ENOD40 gene expression. 
<h3>21075924</h3>argC Orthologs from Rhizobiales show diverse profiles of transcriptional efficiency and functionality in <span class="yellow">Sinorhizobium meliloti</span>. <br><br>Several factors can influence ortholog replacement between closely related species. We evaluated the transcriptional expression and metabolic performance of ortholog substitution complementing a <span class="yellow">Sinorhizobium meliloti</span> argC mutant with argC from Rhizobiales (<span class="yellow">Agrobacterium tumefaciens</span>, <span class="yellow">Rhizobium etli</span>, and <span class="yellow">Mesorhizobium loti</span>). The argC gene is necessary for the synthesis of arginine, an amino acid that is central to protein and cellular metabolism. Strains were obtained carrying plasmids with argC orthologs expressed under the speB and argC (<span class="yellow">S. meliloti</span>) and lac (<span class="yellow">Escherichia coli</span>) promoters. Complementation analysis was assessed by growth, transcriptional activity, enzymatic activity, mRNA levels, specific detection of ArgC proteomic protein, and translational efficiency. The argC orthologs performed differently in each complementation, reflecting the diverse factors influencing gene expression and the ability of the ortholog product to function in a foreign metabolic background. Optimal complementation was directly related to sequence similarity with <span class="yellow">S. meliloti</span>, and was inversely related to species signature, with <span class="blue">M. loti</span> argC showing the poorest performance, followed by <span class="yellow">R. etli</span> and <span class="yellow">A. tumefaciens</span>. Different copy numbers of genes and amounts of mRNA and protein were produced, even with genes transcribed from the same promoter, indicating that coding sequences play a role in the transcription and translation processes. These results provide relevant information for further genomic analyses and suggest that orthologous gene substitutions between closely related species are not completely functionally equivalent. 
</body></html>