
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Mycobacterium tuberculosis</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20971904</h3>Functional analysis of molybdopterin biosynthesis in mycobacteria identifies a fused molybdopterin synthase in <span class="blue">Mycobacterium tuberculosis</span>. <br><br>Most mycobacterial species possess a full complement of genes for the biosynthesis of molybdenum cofactor (MoCo). However, a distinguishing feature of members of the <span class="red">Mycobacterium tuberculosis</span> complex is their possession of multiple homologs associated with the first two steps of the MoCo biosynthetic pathway. <span class="yellow">A</span> mutant of <span class="yellow">M. tuberculosis</span> lacking the moaA1-moaD1 gene cluster and a derivative in which moaD2 was also deleted were significantly impaired for growth in media containing nitrate as a sole nitrogen source, indicating a reduced availability of MoCo to support the assimilatory function of the MoCo-dependent nitrate reductase, NarGHI. However, the double mutant displayed residual respiratory nitrate reductase activity, suggesting that it retains the capacity to produce MoCo. The <span class="yellow">M. tuberculosis</span> moaD and moaE homologs were further analyzed by expressing these genes in mutant strains of <span class="yellow">M. smegmatis</span> that lacked one or both of the sole molybdopterin (MPT) synthase-encoding genes, moaD2 and moaE2, and were unable to grow on nitrate, presumably as a result of the loss of MoCo-dependent nitrate assimilatory activity. Expression of <span class="yellow">M. tuberculosis</span> moaD2 in the <span class="yellow">M. smegmatis</span> moaD2 mutant and of <span class="yellow">M. tuberculosis</span> moaE1 or moaE2 in the <span class="yellow">M. smegmatis</span> moaE2 mutant restored nitrate assimilation, confirming the functionality of these genes in MPT synthesis. Expression of <span class="yellow">M. tuberculosis</span> moaX also restored MoCo biosynthesis in <span class="yellow">M. smegmatis</span> mutants lacking moaD2, moaE2, or both, thus identifying MoaX as a fused MPT synthase. By implicating multiple synthase-encoding homologs in MoCo biosynthesis, these results suggest that important cellular functions may be served by their expansion in <span class="yellow">M. tuberculosis</span>.
<h3>21148733</h3>Genome-wide identification of <span class="blue">Mycobacterium tuberculosis</span> exported proteins with roles in intracellular growth. <br><br>The exported proteins of <span class="blue">Mycobacterium tuberculosis</span> that are localized at the bacterial cell surface or secreted into the environment are ideally situated to interact with host factors and to function in virulence. In this study, we constructed a novel Beta-lactamase reporter transposon and used it directly in <span class="yellow">M. tuberculosis</span> for genome-wide identification of exported proteins. From 177 Beta-lactam-resistant transposon mutants, we identified 111 different exported proteins. The majority of these proteins have no known function, and for nearly half of the proteins, our demonstration that they are exported when fused to a Beta-lactamase reporter is the first experimental proof of their extracytoplasmic localization. The transposon mutants in our banked library were of further value as a collection of mutants lacking individual exported proteins. By individually testing each of 111 mutants for growth in macrophages, six attenuated mutants with insertions in mce1A, mce1B, mce2F, rv0199, ctaC, and lppX were identified. Given that much of the <span class="yellow">M. tuberculosis</span> genome encodes proteins of unknown function, our library of mapped transposon mutants is a valuable resource for efforts in functional genomics. This work also demonstrates the power of a Beta-lactamase reporter transposon that could be applied similarly to other bacterial pathogens. 
</body></html>