
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>N. crassa</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21840412</h3>Expression and functional characterisation of TNC, a high-affinity nickel transporter from <span class="yellow">Neurospora crassa</span>. <br><br>Our previous in silico studies identified a high-affinity nickel transporter,TNC, from the metal transportome of <span class="yellow">Neurospora crassa</span>. <span class="yellow">A</span> knockout mutant of the tnc gene in <span class="blue">N. crassa</span> failed to transport nickel, showed phenotypic growth defects and diminished urease activity under physiological levels of nickel. Transport assays conducted in wild type and knockout mutant strains showed that TNC transports nickel withhigh affinity but exhibits selectivity for other transition metal ions like cobalt. Heterologous complementation of <span class="yellow">Schizosaccharomyces pombe</span> nickel uptake mutant by TNC further substantiates its nickel transport function. Transcriptional analysis of the nickel transporter encoding gene, tnc in <span class="blue">N. crassa</span> by qRT-PCR showed its constitutive expression in various phases of its life cycle. However, levels of the corresponding protein TNC were down-regulated only by increasing the nickel, but not cobalt concentration in the media. Immunolocalisation data suggested that TNC is distributed in the plasma membrane of <span class="blue">N. crassa</span>. Thus, the present study establishes TNC as a functional plasma membrane nickel transporter necessary for physiological acquisition of nickel in the multicellular fungi <span class="blue">N. crassa</span>. 
<h3>22001287</h3>Involvement of a helix-loop-helix transcription factor CHC-1 in CO(2)-mediated conidiation suppression in <span class="yellow">Neurospora crassa</span>. <br><br>The morphological switch from vegetative growth to conidiation in filamentous fungi is highly regulated, but the understanding of the regulatory mechanisms is limited. In this study, by screening a set of knock-out mutants corresponding to 103 transcription factor encoding genes in <span class="yellow">Neurospora crassa</span>, a mutant was found to produce abundant conidia in race tubes in which conidiation in the wild-type strain was suppressed. The corresponding gene NCU00749 encodes a protein containing a helix-loop-helix DNA binding region. Unlike enhanced conidiation in ras-1 and sod-1 mutants, which was completely suppressed by antioxidant N-acetyl cysteine, enhanced conidiation in the NCU00749 mutant was only slightly affected by N-acetyl cysteine. When grown on slants, the NCU00749 deletion mutant exhibited earlier conidial formation than the wild-type strain, and this was more evident at a higher (5%) CO(2) concentration. Therefore, we named NCU00749 as conidiation at high carbon dioxide-1 (chc-1). Genes that are highly expressed during conidial development, eas, con-6, con-8 and con-10, were transcribed at a higher rate in the chc-1 deletion mutant than the wild-type strain in response to conidiation induction. To determine the mechanisms by which CHC-1 regulates conidiation, we conducted a RNA sequencing analysis and found that 404 genes exhibited >= 2 fold changes in transcription in response to chc-1 deletion. Among them, fluffy and ada-6, two transcription factor genes that positively regulate conidiation in <span class="blue">N. crassa</span>, and rca-1, whose homolog flbD in <span class="yellow">Aspergillus nidulans</span> is essential for conidiation, were upregulated in the chc-1 deletion mutant. Results of RNA sequencing also suggest that signal transduction via the cAMP and the MAK-2 mediated signal pathways, and ROS generation and removal, mechanisms known to regulate conidiation, are not involved in chc-1 mediated control of conidiation. In addition, chc-1 also influences expression of genes involved in other important biological processes besides conidiation such as carbon metabolism, sphingolipid synthesis, cell wall synthesis, and calcium signaling. 
<h3>21220038</h3>acon-3, the <span class="yellow">Neurospora crassa</span> ortholog of the developmental modifier, medA, complements the conidiation defect of the <span class="yellow">Aspergillus nidulans</span> mutant. <br><br><span class="yellow">Aspergillus nidulans</span> and <span class="yellow">Neurospora crassa</span> are ascomycetes that produce asexual spores through morphologically distinct processes. MedA, a protein with unknown function, is required for normal asexual and sexual development in <span class="yellow">A. nidulans</span>. We determined that the <span class="blue">N. crassa</span> ortholog of medA is acon-3, a gene required for early conidiophore development and female fertility. To test hypotheses about the evolutionary origins of asexual development in distinct <span class="yellow">fungal</span> lineages it is important to understand the degree of conservation of developmental regulators. The amino acid sequences of <span class="yellow">A. nidulans</span> MedA and <span class="blue">N. crassa</span> ACON-3 shared 37% identity and 51% similarity. acon-3 is induced at late time points of conidiation. In contrast, medA is constitutively expressed and MedA protein localizes to nuclei in all tissue types. Nonetheless, expression of acon-3 using its native promoter complemented the conidiation defects of the <span class="yellow">A. nidulans</span> DeltamedA and medA15 mutants. We conclude that the biochemical activity of the medA orthologs is conserved for conidiation. 
<h3>22001288</h3>Modulation of <span class="yellow">fungal</span> sensitivity to staurosporine by targeting proteins identified by transcriptional profiling. <br><br>An analysis of the time-dependent genetic response to the death-inducer staurosporine was performed in <span class="yellow">Neurospora crassa</span> by transcriptional profiling. Staurosporine induced two major genes encoding an ABC transporter and a protein with similarity to regulatory subunits of potassium channels. The transcriptional response is dependent on the activity of a novel transcription factor. Deletion mutants in differentially expressed genes displayed altered sensitivity to staurosporine, underscoring significant proteins involved in the response to the drug. <span class="yellow">A</span> null-mutant of the ABC transporter (abc3) is extremely sensitive to staurosporine, accumulates more staurosporine than the wild type strain and is defective in energy-dependent export of the drug, indicating that the ABC3 protein is the first described staurosporine transporter. It was located in the plasma membrane by immunofluorescence microscopy. The combination of inhibitors of ABC transporters or of potassium channels with staurosporine leads to an enhanced activity against <span class="blue">N. crassa</span> and pathogenic fungi paving the way to the development of more potent and specific antifungals. Our results highlight the general use of transcriptional profiling for the identification of novel proteins involved in cell death and their potential use as drug targets. 
</body></html>