
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Proteus mirabilis</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21169483</h3>Unexpected coregulator range for the global regulator Lrp of <span class="yellow">Escherichia coli</span> and <span class="blue">Proteus mirabilis</span>. 

The Lrp/AsnC family of transcription factors links gene regulation to metabolism in bacteria and archaea. Members of this family, collectively, respond to a wide range of amino acids as coregulators. In <span class="yellow">Escherichia coli</span>, Lrp regulates over 200 genes directly and is well known to respond to leucine and, to a somewhat lesser extent, alanine. We focused on Lrp from <span class="blue">Proteus mirabilis</span> and <span class="yellow">E. coli</span>, orthologs with 98% identity overall and identical helix-turn-helix motifs, for which a previous study nevertheless found functional differences. Sequence differences between these orthologs, within and adjacent to the amino acid-responsive RAM domain, led us to test for differential sensitivity to coregulatory amino acids. In the course of this investigation, we found, via in vivo reporter fusion assays and in vitro electrophoretic mobility shift experiments, that <span class="yellow">E. coli</span> Lrp itself responded to a broader range of amino acids than was previously appreciated. In particular, for both the <span class="yellow">E. coli</span> and <span class="yellow">P. mirabilis</span> orthologs, Lrp responsiveness to methionine was similar in magnitude to that to leucine. Both Lrp orthologs are also fairly sensitive to Ile, His, and Thr. These observations suggest that Lrp ties gene expression in the Enterobacteriaceae rather extensively to physiological status, as reflected in amino acid pools. These findings also have substantial implications for attempts to model regulatory architecture from transcriptome measurements or to infer such architecture from genome sequences, and they suggest that even well-studied regulators deserve ongoing exploration. 
</body></html>