
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>sheep</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>21029751</h3>Real time PCR method for simultaneous detection, quantitation and differentiation of capripoxviruses. <br><br>The genus Capripoxvirus (CaPV) comprises three members namely, <span class="yellow">sheep poxvirus</span> (<span class="yellow">SPPV</span>), <span class="yellow">goat poxvirus</span> (<span class="yellow">GTPV</span>) and <span class="yellow">lumpy skin disease virus</span> (<span class="yellow">LSDV</span>) affecting <span class="blue">sheep</span>, <span class="yellow">goats</span> and <span class="yellow">cattle</span>, respectively. CaPV infections produce similar symptoms in <span class="blue">sheep</span> and <span class="yellow">goats</span>, and the three viruses cannot be distinguished serologically. Since there are conflicting opinions regarding the host specificity of CaPVs, particularly for <span class="yellow">goatpox</span> and <span class="yellow">sheeppox viruses</span>, the development of rapid genotyping tools will facilitate more accurate disease diagnosis and surveillance for better management of capripox outbreaks. This paper describes a species-specific, real time polymerase chain reaction (PCR), based on unique molecular markers that were found in the G-protein-coupled chemokine receptor (GPCR) gene sequences of CaPVs, that uses dual hybridization probes for their simultaneous detection, quantitation and genotyping. The assay can differentiate between CaPV strains based on differences in the melting point temperature (Tm) obtained after fluorescence melting curve analysis (FMCA). It is highly sensitive and presents low intra- and inter-run variation. This real time PCR assay will make a significant contribution to CaPV diagnosis and to the better understanding of the epidemiology of CaPVs by enabling rapid genotyping and gene-based classification of <span class="yellow">viral</span> strains and unequivocal identification of isolates. 
<h3>20980501</h3>A nuclear inhibitor of NF-kappaB encoded by a poxvirus.<br><br>Poxviruses have evolved various strategies to inhibit cytoplasmic events leading to activation of the nuclear factor kappaB (NF-kappaB) signaling pathway, with individual viruses often encoding multiple NF-kappaB inhibitors. Here, the novel <span class="yellow">orf virus</span> (<span class="yellow">ORFV</span>)-encoded protein <span class="yellow">ORFV</span>002 was shown to inhibit nuclear events regulating NF-kappaB transcriptional activity. ORFV002 expression in cell cultures significantly decreased wild-type-virus-, tumor necrosis factor alpha (TNF-alpha)-, and lipopolysaccharide (LPS)-induced NF-kappaB-mediated gene expression. Expression of ORFV002 in cells, while not affecting phosphorylation or nuclear translocation of NF-kappaB-p65, markedly decreased TNF-alpha- and wild-type-virus-induced acetylation of NF-kappaB-p65, a p300-mediated nuclear modification of NF-kappaB-p65 that regulates its transactivating activity. ORFV002 was shown to colocalize and interact with NF-kappaB-p65, and expression of ORFV002 in cell cultures resulted in a reduced interaction of NF-kappaB-p65 with p300, suggesting that ORFV002 interferes with NF-kappaB-p65/p300 association. Deletion of ORFV002 from the OV-IA82 genome had no significant effect on <span class="yellow">ORFV</span> pathogenesis in <span class="red">sheep</span>, indicating that ORFV002 is nonessential for <span class="yellow">virus</span> virulence in the natural host. This represents the first description of a nuclear inhibitor of NF-kappaB encoded by a poxvirus. 
<h3>21641778</h3>Suitability of different media for in vitro cultivation of the ruminal protozoa species <span class="yellow">Entodinium caudatum</span>, <span class="yellow">Eudiplodinium maggii</span>, and <span class="yellow">Epidinium ecaudatum</span>. <br><br>Three protozoal cultivation media were tested to determine the medium which best facilitated growth and viability of key B-type ciliates isolated from the <span class="red">sheep</span> rumen. <span class="yellow">Entodinium caudatum</span> and <span class="yellow">Eudiplodinium maggii</span> were grown anaerobically in 50-ml flasks for 32 days in Caudatum-type (C), Kisidayova (K) or Dehority (M) medium. On day 32, in media K and M, <span class="yellow">E. caudatum</span> cell counts were high with 5.6x10(3) and 7.8x10(3) mL(-1), respectively, and the proportion of dead cells was low with 0.6 and 1.4%, respectively. <span class="yellow">E. maggii</span> concentrations when grown in medium M and C were 2.7x10(3) and 2.4x10(3) mL(-1), respectively, with 3.9 and 14.1% dead cells. Medium M, which favoured growth of both protozoa species, was tested again and <span class="yellow">Epidinium ecaudatum</span> was included. Protozoa were grown for a 4-month period and samples were taken in the last two months on days 1, 7, 35 and 57. Average cell concentrations were 10.0, 0.8 and 0.5x10(3) mL(-1) for <span class="yellow">E. caudatum</span>, <span class="yellow">E. maggii</span>, and <span class="yellow">E. ecaudatum</span>, respectively. In conclusion, medium M would appear to be the best choice for cultivating these three species in one medium. 
</body></html>