
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.linneaus_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 

table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>Sinorhizobium meliloti</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20964693</h3>Jasmonate biosynthesis in legume and actinorhizal nodules.<br><br>Jasmonic acid (JA) is a plant signalling compound that has been implicated in the regulation of mutualistic symbioses. In order to understand the spatial distribution of JA biosynthetic capacity in nodules of two actinorhizal species, <span class="annotated ">Casaurina glauca</span> and <span class="annotated ">Datisca glomerata</span>, and one legume, <span class="annotated ">Medicago truncatula</span>, we determined the localization of allene oxide cyclase (AOC) which catalyses a committed step in JA biosynthesis. In all nodule types analysed, AOC was detected exclusively in uninfected cells. The levels of JA were compared in the roots and nodules of the three plant species. The nodules and noninoculated roots of the two actinorhizal species, and the root systems of <span class="annotated ">M. truncatula</span>, noninoculated or nodulated with wild-type <span class="match annotated ">Sinorhizobium meliloti</span> or with mutants unable to fix nitrogen, did not show significant differences in JA levels. However, JA levels in all plant organs examined increased significantly on mechanical disturbance. To study whether JA played a regulatory role in the nodules of <span class="annotated ">M. truncatula</span>, composite plants containing roots expressing an MtAOC1-sense or MtAOC1-RNAi construct were inoculated with <span class="annotated ">S. meliloti</span>. Neither an increase nor reduction in AOC levels resulted in altered nodule formation. These data suggest that jasmonates are not involved in the development and function of root nodules. 
<h3>21057009</h3>Complex regulation of symbiotic functions is coordinated by MucR and quorum sensing in <span class="match unannotated ">Sinorhizobium meliloti</span>. <br><br>In <span class="match unannotated ">Sinorhizobium meliloti</span>, the production of exopolysaccharides such as succinoglycan and exopolysaccharide II (EPS II) enables the bacterium to invade root nodules on <span class="annotated ">Medicago sativa</span> and establish a nitrogen-fixing symbiosis. While extensive research has focused on succinoglycan, less is known concerning the regulation of EPS II or the mechanism by which it mediates entrance into the host plant. Previously, we reported that the ExpR/Sin quorum-sensing system is required to produce the symbiotically active low-molecular-weight fraction of this exopolysaccharide. Here, we show that this system induces EPS II production by increasing expression of the expG-expC operon, encoding both a transcriptional regulator (ExpG) and a glycosyl transferase (ExpC). ExpG derepresses EPS II production at the transcriptional level from MucR, a RosR homolog, while concurrently elevating expression of expC, resulting in the synthesis of the low-molecular-weight form. While the ExpR/Sin system abolishes the role of MucR on EPS II production, it preserves a multitude of other quorum-sensing-independent regulatory functions which promote the establishment of symbiosis. In planktonic <span class="unannotated ">S. meliloti</span>, MucR properly coordinates a diverse set of bacterial behaviors by repressing a variety of genes intended for expression during symbiosis and enhancing the bacterial ability to induce root nodule formation. Quorum sensing precisely modulates the functions of MucR to take advantage of both the production of symbiotically active EPS II as well as the proper coordination of bacterial behavior required to promote symbiosis. 
<h3>21075924</h3>argC Orthologs from Rhizobiales show diverse profiles of transcriptional efficiency and functionality in <span class="match annotated ">Sinorhizobium meliloti</span>. <br><br>Several factors can influence ortholog replacement between closely related species. We evaluated the transcriptional expression and metabolic performance of ortholog substitution complementing a <span class="match annotated ">Sinorhizobium meliloti</span> argC mutant with argC from Rhizobiales (<span class="annotated ">Agrobacterium tumefaciens</span>, <span class="annotated ">Rhizobium etli</span>, and <span class="annotated ">Mesorhizobium loti</span>). The argC gene is necessary for the synthesis of arginine, an amino acid that is central to protein and cellular metabolism. Strains were obtained carrying plasmids with argC orthologs expressed under the speB and argC (<span class="annotated ">S. meliloti</span>) and lac (<span class="annotated ">Escherichia coli</span>) promoters. Complementation analysis was assessed by growth, transcriptional activity, enzymatic activity, mRNA levels, specific detection of ArgC proteomic protein, and translational efficiency. The argC orthologs performed differently in each complementation, reflecting the diverse factors influencing gene expression and the ability of the ortholog product to function in a foreign metabolic background. Optimal complementation was directly related to sequence similarity with <span class="annotated ">S. meliloti</span>, and was inversely related to species signature, with <span class="annotated ">M. loti</span> argC showing the poorest performance, followed by <span class="annotated ">R. etli</span> and <span class="annotated ">A. tumefaciens</span>. Different copy numbers of genes and amounts of mRNA and protein were produced, even with genes transcribed from the same promoter, indicating that coding sequences play a role in the transcription and translation processes. These results provide relevant information for further genomic analyses and suggest that orthologous gene substitutions between closely related species are not completely functionally equivalent. 
<h3>20971905</h3>The nodulation of <span class="unannotated ">alfalfa</span> by the acid-tolerant <span class="annotated ">Rhizobium sp. strain LPU83</span> does not require sulfated forms of lipochitooligosaccharide nodulation signals. <br><br>The induction of root nodules by the majority of rhizobia has a strict requirement for the secretion of symbiosis-specific lipochitooligosaccharides (nodulation factors [NFs]). The nature of the chemical substitution on the NFs depends on the particular rhizobium and contributes to the host specificity imparted by the NFs. We present here a description of the genetic organization of the nod gene cluster and the characterization of the chemical structure of the NFs associated with the broad-host-range <span class="annotated ">Rhizobium sp. strain LPU83</span>, a bacterium capable of nodulating at least <span class="annotated ">alfalfa</span>, bean, and <span class="annotated ">Leucena leucocephala</span>. The nod gene cluster was located on the plasmid pLPU83b. The organization of the cluster showed synteny with those of the <span class="annotated ">alfalfa</span>-nodulating rhizobia, <span class="match annotated ">Sinorhizobium meliloti</span> and <span class="annotated ">Sinorhizobium medicae</span>. Interestingly, the strongest sequence similarity observed was between the partial nod sequences of <span class="annotated ">Rhizobium mongolense USDA 1844</span> and the corresponding <span class="annotated ">LPU83</span> nod genes sequences. The phylogenetic analysis of the intergenic region nodEG positions strain <span class="annotated ">LPU83</span> and the type strain <span class="annotated ">R. mongolense 1844</span> in the same branch, which indicates that <span class="annotated ">Rhizobium sp. strain LPU83</span> might represent an early <span class="annotated ">alfalfa</span>-nodulating genotype. The NF chemical structures obtained for the wild-type strain consist of a trimeric, tetrameric, and pentameric chitin backbone that shares some substitutions with both alfalfa- and bean-nodulating rhizobia. Remarkably, while in strain <span class="annotated ">LPU83</span> most of the NFs were sulfated in their reducing terminal residue, none of the NFs isolated from the nodH mutant <span class="annotated ">LPU83-H</span> were sulfated. The evidence obtained supports the notion that the sulfate decoration of NFs in <span class="annotated ">LPU83</span> is not necessary for <span class="annotated ">alfalfa</span> nodulation. 
</body></html>