
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>T. melanosporum</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20961294</h3>Isolation and characterization of MAT genes in the symbiotic ascomycete <span class="yellow">Tuber melanosporum</span>. <br><br>* The genome of <span class="yellow">Tuber melanosporum</span> has recently been sequenced. Here, we used this information to identify genes involved in the reproductive processes of this edible fungus. The sequenced strain (Mel28) possesses only one of the two master genes required for mating, that is, the gene that codes for the high mobility group (HMG) transcription factor (MAT1-2-1), whereas it lacks the gene that codes for the protein containing the alpha-box- domain (MAT1-1-1), suggesting that this fungus is heterothallic. * <span class="yellow">A</span> PCR-based approach was initially employed to screen truffles for the presence of the MAT1-2-1 gene and amplify the conserved regions flanking the mating type (MAT) locus. The MAT1-1-1 gene was finally identified using primers designed from the conserved regions of strains that lack the MAT1-2-1 gene. * Mating type-specific primer pairs were developed to screen asci and gleba from truffles of different origins and to genotype single ascospores within the asci. These analyses provided definitive evidence that <span class="blue">T. melanosporum</span> is a heterothallic species with a MAT locus that is organized similarly to those of ancient <span class="yellow">fungal</span> lineages. * <span class="yellow">A</span> greater understanding of the reproductive mechanisms that exist in Tuber spp. allows for optimization of <span class="yellow">truffle</span> plantation management strategies. 
<h3>20964691</h3><span class="yellow">Tuber melanosporum</span>: mating type distribution in a natural plantation and dynamics of strains of different mating types on the roots of nursery-inoculated host plants. <br><br>* In light of the recent finding that <span class="yellow">Tuber melanosporum</span>, the ectomycorrhizal ascomycete that produces the most highly prized <span class="yellow">black truffles</span>, is a heterothallic species, we monitored the spatial distribution of strains with opposite mating types (MAT) in a natural <span class="yellow">truffle</span> ground and followed strain dynamics in artificially inoculated host plants grown under controlled conditions. * In a natural <span class="yellow">truffle</span> ground, ectomycorrhizas (ECMs), soil samples and fruit bodies were sampled and genotyped to determine mating types. Simple sequence repeat (SSR) markers were also used to fingerprint ECMs and fruit bodies. The ECMs from nursery-inoculated host plants were analysed for mating type at 6 months and 19 months post-inoculation. * In open-field conditions, all ECMs from the same sampling site showed an identical mating type and an identical haploid genotype, based on SSR analysis. Interestingly, the gleba of fruit bodies always demonstrated the same genotype as the surrounding ECMs. Although root tips from nursery-grown plants initially developed ECMs of both mating types, a dominance of ECMs of the same MAT were found after several months. * The present study deepens our understanding of the vegetative and sexual propagation modes of <span class="blue">T. melanosporum</span>. These results are highly relevant for <span class="yellow">truffle</span> cultivation. 
<h3>20965267</h3>Distribution and localization of microsatellites in the <span class="yellow">Perigord black truffle</span> genome and identification of new molecular markers. <br><br>The level of genetic diversity and genetic structure in the <span class="yellow">Perigord black truffle</span> (<span class="yellow">Tuber melanosporum Vittad.</span>) has been debated for several years, mainly due to the lack of appropriate genetic markers. Microsatellites or simple sequence repeats (SSRs) are important for the genome organisation, phenotypic diversity and are one of the most popular molecular markers. In this study, we surveyed the <span class="blue">T. melanosporum</span> genome (1) to characterise its SSR pattern; (2) to compare it with SSR patterns found in 48 other <span class="yellow">fungal</span> and three oomycetes genomes and (3) to identify new polymorphic SSR markers for population genetics. The <span class="blue">T. melanosporum</span> genome is rich in SSRs with 22,425 SSRs with mono-nucleotides being the most frequent motifs. SSRs were found in all genomic regions although they are more frequent in non-coding regions (introns and intergenic regions). Sixty out of 135 PCR-amplified mono-, di-, tri-, tetra, penta, and hexa-nucleotides were polymorphic (44%) within <span class="yellow">black truffle</span> populations and 27 were randomly selected and analysed on 139 <span class="blue">T. melanosporum</span> isolates from France, Italy and Spain. The number of alleles varied from 2 to 18 and the expected heterozygosity from 0.124 to 0.815. One hundred and thirty-two different multilocus genotypes out of the 139 <span class="blue">T. melanosporum</span> isolates were identified and the genotypic diversity was high (0.999). Polymorphic SSRs were found in UTR regulatory regions of fruiting bodies and ectomycorrhiza regulated genes, suggesting that they may play a role in phenotypic variation. In conclusion, SSRs developed in this study were highly polymorphic and our results showed that <span class="blue">T. melanosporum</span> is a species with an important genetic diversity, which is in agreement with its recently uncovered heterothallic mating system. 
<h3>21039570</h3>Genomic profiling of carbohydrate metabolism in the ectomycorrhizal fungus <span class="yellow">Tuber melanosporum</span>.<br><br>* Primary carbohydrate metabolism plays a special role related to carbon/nitrogen exchange, as well as metabolic support of fruiting body development, in ectomycorrhizal macrofungi. In this study, we used information retrieved from the recently sequenced <span class="yellow">Tuber melanosporum</span> genome, together with transcriptome analysis data and targeted validation experiments, to construct the first genome-wide catalogue of the proteins supporting carbohydrate metabolism in a plant-symbiotic ascomycete. * More than 100 genes coding for enzymes of the glycolysis, pentose phosphate, tricarboxylic acid, glyoxylate and methylcitrate pathways, glycogen, trehalose and mannitol metabolism and cell wall precursor were annotated. Transcriptional regulation of these pathways in different stages of the <span class="blue">T. melanosporum</span> lifecycle was investigated using whole-genome oligoarray expression data together with real-time reverse transcription-polymerase chain reaction analysis of selected genes. * The most significant results were the identification of methylcitrate cycle genes and of an acid invertase, the first enzyme of this kind to be described in a plant-symbiotic filamentous fungus. * <span class="yellow">A</span> subset of transcripts coding for trehalose, glyoxylate and methylcitrate enzymes was up-regulated in fruiting bodies, whereas genes involved in mannitol and glycogen metabolism were preferentially expressed in mycelia and ectomycorrhizas, respectively. These data indicate a high degree of lifecycle stage specialization for particular branches of carbohydrate metabolism in <span class="blue">T. melanosporum</span>. 
</body></html>