
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>  
.yellow { background-color:rgba(50, 180, 180, 0.5); } 
.red { background-color:rgba(214, 75, 79, 0.5); } 
.blue { background-color:rgba(75, 75, 214, 0.5); } 
table { border-collapse: collapse; } 
th, td { border: 1px solid black; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }
</style></head><body>
<h2>virus</h2><span class="red">Not tagged name</span> - <span class="blue">Tagged name</span> - <span class="yellow">Other name</span><br><hr><h3>20850479</h3>Identification of the structural proteins of VP1 and VP2 of a novel <span class="yellow">mud crab dicistrovirus</span>. <br><br><span class="yellow">Mud crab dicistrovirus</span> (<span class="yellow">MCDV</span>), a newly identified single-stranded positive RNA <span class="red">virus</span>, is an important pathogen that causes serious economic losses to <span class="yellow">mud crab</span> aquaculture. In this study, <span class="yellow">MCDV</span> was purified, and three structural proteins of <span class="yellow">MCDV</span> were separated by SDS-PAGE. The N-terminal 15 amino acids were sequenced and aligned with the main structural proteins of other dicistrovirus. The three structural proteins were named VP1, VP2 and VP3. Monoclonal antibodies (MAbs) against the two main structural proteins, VP1 and VP2, were prepared, and the two structural proteins were then identified using these MAbs. The results of Western blot analyses demonstrated that five MAbs recognised VP1 and two recognised VP2. The results of immunogold transmission electron microscopy (IEM) revealed that the epitopes of the two structural proteins recognised by the MAbs were located at the outer surface of the virions, which suggested that the two structural proteins are <span class="yellow">MCDV</span> capsid proteins. The identification of the two structural proteins of <span class="yellow">MCDV</span> is useful for studying their functions, as well as the mechanism of infection and the pathogenesis of <span class="yellow">MCDV</span>. 
<h3>20923688</h3>Universal primers for rapid detection of hytrosaviruses.<br><br>Hytrosaviridae is a proposed <span class="red">virus</span> family encompassing viruses that cause salivary gland hypertrophy (SGH) syndrome in infected insects and reduce the fertility in their dipteran insect hosts. They contain a large, double stranded DNA genome of 120-190 kbp. To date, these viruses have been detected only in adult Diptera. These include hytrosaviruses detected in various tsetse fly species (Glossina spp.), the <span class="yellow">narcissus bulb fly</span> <span class="yellow">Merodon equestris</span> and the <span class="yellow">house fly</span> <span class="yellow">Musca domestica</span>. The limited number of hytrosaviruses reported to date may be a reflection of the frequent absence of external symptoms in infected adult flies and the fact that the <span class="red">virus</span> does not cause rapid mortality. Based on the complete genome sequence of <span class="yellow">Glossinia pallidipes</span> (GpSGHV) and <span class="yellow">Musca domestica</span> (MdSGHV) salivary gland hypertrophy viruses, a PCR based methodology was developed to detect the viruses in these species. To be able to detect hytrosaviruses in other Diptera, five degenerate primer pairs were designed and tested on GpSGHV and MdSGHV DNA using gradient PCR with annealing temperatures from 37 to 61^0C. Two pairs of primers were selected from p74, two pairs from PIF-1 and one pair from ODV-e66 homologous proteins. Four primer pairs generated a <span class="red">virus</span> specific PCR product on both MdSGHV and GpSGHV at all tested annealing temperatures, while the ODV-e66 based primers did not generate a <span class="red">virus</span> specific product with annealing temperatures higher that 47^0C. No non-specific PCR product was found when using genomic DNA of infected flies as template DNA. These results offer new sets of primers that could be used to detect hytrosaviruses in other insects. 
<h3>20923689</h3>Simultaneous virus-specific detection of the two <span class="yellow">cassava</span> brown streak-associated viruses by RT-PCR reveals wide distribution in East Africa, mixed infections, and infections in <span class="yellow">Manihot glaziovii</span>. <br><br>The expanding <span class="yellow">cassava</span> brown streak disease (CBSD) epidemic in East Africa is caused by two ipomoviruses (genus Ipomovirus; Potyviridae), namely, <span class="yellow">Cassava brown streak virus</span> (<span class="yellow">CBSV</span>), and <span class="yellow">Ugandan cassava brown streak virus</span> (<span class="yellow">UCBSV</span>) that was described recently. <span class="yellow">A</span> reverse transcription polymerase chain reaction (RT-PCR) based diagnostic method was developed in this study for simultaneous virus-specific detection of the two viruses. Results showed that <span class="yellow">CBSV</span> and <span class="yellow">UCBSV</span> are distributed widely in the highlands (> 1000 m above the sea level) of the Lake Victoria zone in Uganda and Tanzania and also in the Indian Ocean costal lowlands of Tanzania. Isolates of <span class="yellow">UCBSV</span> from the Lake Victoria zone were placed to two phylogenetic clusters in accordance with their origin in Uganda or Tanzania, respectively. Mixed infections with <span class="yellow">CBSV</span> and <span class="yellow">UCBSV</span> were detected in many <span class="yellow">cassava</span> plants in the areas surveyed. <span class="yellow">CBSV</span> was also detected in the perennial species <span class="yellow">Manihot glaziovii</span> (DNA-barcoded in this study) in Tanzania, which revealed the first <span class="red">virus</span> reservoir other than <span class="yellow">cassava</span>. The method for detection of <span class="yellow">CBSV</span> and <span class="yellow">UCBSV</span> described in this study has important applications for plant quarantine, resistance breeding of <span class="yellow">cassava</span>, and studies on epidemiology and control of CBSD in East Africa. 
<h3>20933015</h3>Detection and discrimination of members of the family Luteoviridae by real-time PCR and SYBR(R) GreenER(TM) melting curve analysis. <br><br>This study investigated the suitability of a two step real-time RT-PCR melting curve analysis as a tool for the detection and discrimination of nine species in the plant <span class="red">virus</span> family Luteoviridae, being <span class="yellow">Soybean dwarf virus</span> [<span class="yellow">SbDV</span>], <span class="yellow">Bean leafroll virus</span> [<span class="yellow">BLRV</span>], <span class="yellow">Beet chlorosis virus</span> [<span class="yellow">BChV</span>], <span class="yellow">Beet mild yellowing virus</span> [<span class="yellow">BMYV</span>], <span class="yellow">Beet western yellows virus</span> [<span class="yellow">BWYV</span>], <span class="yellow">Cereal yellow dwarf virus-RPV</span> [<span class="yellow">CYDV-RPV</span>], <span class="yellow">Cucurbit aphid-borne yellows virus</span> [<span class="yellow">CABYV</span>], <span class="yellow">Potato leafroll virus</span> [<span class="yellow">PLRV</span>] and <span class="yellow">Turnip yellows virus</span> [<span class="yellow">TuYV</span>]. Melting temperature and shape of the melting peak were analysed for 68 bp and 148 bp coat protein gene amplicons using SYBR(R) GreenER(TM) fluorescent dye. Specific melting peaks with unique melting temperature were observed for the various species of the family Luteoviridae using the 68 bp amplicon, but not with the 148 bp amplicon. Due to the high variability of sequences for some members of this family, different melting temperatures were also observed between different isolates of the species <span class="yellow">CYDV-RPV</span> and <span class="yellow">TuYV</span>. Nevertheless, discrimination between species was achieved for <span class="yellow">SbDV</span>, <span class="yellow">BLRV</span>, <span class="yellow">BChV</span>, <span class="yellow">BMYV</span>, <span class="yellow">CABYV</span> and either <span class="yellow">PLRV</span> or <span class="yellow">BWYV</span>. Melting curve analysis, in this study, is a faster and more discriminatory alternative to gel electrophoresis of end-point PCR products for the detection of Luteoviridae infection. 
<h3>20933016</h3>Detection and differentiation of <span class="yellow">tick-borne encephalitis virus</span> subtypes by a reverse transcription quantitative real-time PCR and pyrosequencing. <br><br><span class="yellow">Tick-borne encephalitis (TBE) virus</span> causes one of the most important flaviviral infections of the <span class="yellow">human</span> central nervous system in Europe and Asia. In recent years the rate of <span class="yellow">TBE</span> infection has been raising and the <span class="red">virus</span> has been spreading to new areas. Currently, the diagnosis of <span class="yellow">TBE</span> is based on detection of specific antibodies in patients' sera which appear as late as about 2 weeks post-infection. For a timely diagnosis of <span class="yellow">TBE virus</span> infections and epidemiological studies, a <span class="yellow">TBE virus</span>-specific reverse transcription quantitative real-time PCR (RT-qPCR) followed by pyrosequencing was developed. The assay is based on one degenerated primer pair detecting all three <span class="yellow">human</span>-pathogenic <span class="yellow">TBE virus</span> subtypes with a detection limit of 10 copies. Even though primers and probe are highly degenerated, the assay is specific for <span class="yellow">TBE virus</span> species and detects all subtypes with a comparable sensitivity. Furthermore, <span class="yellow">TBE virus</span> RT-qPCR could be carried out as one-step or two-step assay. RT-qPCR can be followed by pyrosequencing which allows a rapid subtyping of <span class="yellow">TBE viruses</span>. For detection purposes an internal control to monitor RNA extraction, cDNA synthesis and amplification is included. In summary, the method is sensitive, highly specific and easy-to-handle tool for the detection and differentiation of <span class="yellow">TBE virus</span> in the early phase of illness or in <span class="yellow">TBE</span> host animal species and ticks. 
<h3>20933017</h3>Adaptation of a Madin-Darby <span class="yellow">canine</span> kidney cell line to suspension growth in serum-free media and comparison of its ability to produce avian <span class="yellow">influenza virus</span> to Vero and BHK21 cell lines. <br><br>Madin-Darby <span class="yellow">canine</span> kidney (MDCK) cells are currently considered for <span class="yellow">influenza</span> vaccine manufacturing. <span class="yellow">A</span> drawback of these cells is their anchorage dependent growth, which greatly complicates process scale-up. In this paper a novel MDCK cell line (MDCK-SFS) is described that grows efficiently in suspension and retained high expression levels of both alpha-2,6 and alpha-2,3 sialic acid receptors, which bind preferably to <span class="yellow">human</span> and avian <span class="yellow">influenza viruses</span>, respectively. The production of avian <span class="yellow">influenza virus</span> by BHK21, Vero and MDCK-SFS cell lines was compared. Although BHK21 cells consisted of two populations, one of which lacks the alpha-2,3 receptor, they supported the replication of two <span class="yellow">influenza</span> strains to high titres. However, BHK21 cells are generally not applicable for <span class="yellow">influenza</span> production since they supported the replication of six further strains poorly. MDCK-SFS cells yielded the highest infectious <span class="red">virus</span> titres and <span class="red">virus</span> genome equivalent concentration for five of the eight <span class="yellow">influenza</span> strains analyzed and the highest hemagglutination activity for all eight <span class="red">virus</span> strains. Taken together with their suitability for suspension growth this makes the MDCK-SFS cell line potentially useful for large scale <span class="yellow">influenza virus</span> production. 
<h3>20943971</h3>Roles of <span class="yellow">vaccinia virus</span> genes E3L and K3L and host genes PKR and RNase L during intratracheal infection of C57BL/6 <span class="yellow">mice</span>. <br><br>The importance of the 2'-5' oligoadenylate synthetase (OAS)/RNase L and double-stranded RNA (dsRNA)-dependent protein kinase (PKR) pathways in host interferon induction resulting from <span class="red">virus</span> infection in response to dsRNA has been well documented. In poxvirus infections, the interactions between the <span class="yellow">vaccinia virus</span> (<span class="yellow">VV</span>) genes E3L and K3L, which target RNase L and PKR, respectively, serve to prevent the induction of the dsRNA-dependent induced interferon response in cell culture. To determine the importance of these host genes in controlling <span class="yellow">VV</span> infections, <span class="yellow">mouse</span> single-gene knockouts of RNase L and PKR and double-knockout <span class="yellow">mice</span> were studied following intratracheal infection with <span class="yellow">VV</span>, <span class="yellow">VV</span>DeltaK3L, or VVDeltaE3L. <span class="yellow">VV</span> caused lethal disease in all <span class="yellow">mouse</span> strains. The single-knockout animals were more susceptible than wild-type animals, while the RNase L(-/-) PKR(-/-) <span class="yellow">mice</span> were the most susceptible. VVDeltaE3L infections of wild-type <span class="yellow">mice</span> were asymptomatic, demonstrating that E3L plays a critical role in controlling the host immune response. RNase L(-/-) <span class="yellow">mice</span> showed no disease, whereas 20% of the PKR(-/-) <span class="yellow">mice</span> succumbed at a dose of 10(8) PFU. Lethal disease was routinely observed in RNase L(-/-) PKR(-/-) <span class="yellow">mice</span> inoculated with 10(8) PFU of VVDeltaE3L, with a distinct pathology. VVDeltaK3L infections exhibited no differences in virulence among any of the <span class="yellow">mouse</span> constructs, suggesting that PKR is not the exclusive target of K3L. Surprisingly, VVDeltaK3L did not disseminate to other tissues from the lung. Hence, the cause of death in this model is respiratory disease. These results also suggest that an unanticipated role of the K3L gene is to facilitate <span class="red">virus</span> dissemination. 
<h3>20951166</h3>Rapid detection of <span class="yellow">Newcastle disease virus</span> replication in embryonated <span class="yellow">chicken</span> eggs using quantitative real time polymerase chain reaction. <br><br><span class="yellow">Newcastle disease virus</span> (<span class="yellow">NDV</span>), an avian paramyxovirus, is an economically important disease of <span class="yellow">poultry</span> globally. Rapid methods to detect and differentiate the <span class="red">virus</span> are important to curtail the spread of this <span class="red">virus</span>. Nucleic acid based detection methods are routinely employed for diagnosis that suffer from the disadvantage of failure to discriminate viable <span class="red">virus</span> and non-infectious genome. However, <span class="red">virus</span> isolation remains the gold standard for diagnosis of field outbreaks. The sensitivity of <span class="red">virus</span> isolation was combined with nucleic acid based detection methods so that the time taken for confirmatory diagnosis could be considerably reduced while increasing sensitivity. Quantitative real time reverse transcription polymerase chain reaction (qRT-PCR) and conventional RT-PCR techniques were compared for the detection of <span class="yellow">NDV</span> genome replication in 9-11-day-old embryonated <span class="yellow">chicken</span> eggs (ECE) using the nucleoprotein (NP) gene of the <span class="red">virus</span> as a target. The results suggest that at least two to fourfold increase in cycle threshold (C(t)) values over the baseline C(t) value of samples lacking infectious <span class="red">virus</span>, would indicate live <span class="yellow">NDV</span> replication. The limit of detection of <span class="yellow">NDV</span> replication using qRT-PCR was 1x10(4.0) mean embryo infective doses (EID(50)). The earliest time point when live <span class="red">virus</span> replication was detectable by qRT-PCR or RT-PCR was 30h post-inoculation in ECE. 
<h3>20951743</h3>Development and validation of a lateral flow immunoassay using colloidal gold for the identification of serotype-specific <span class="yellow">foot-and-mouth disease virus O</span>, <span class="yellow">A</span> and <span class="yellow">Asia 1</span>. <br><br>A lateral flow immunoassay (LFI) was developed to identify and diagnose <span class="yellow">foot-and-mouth disease virus</span> (<span class="yellow">FMDV</span>) serotypes <span class="yellow">O</span>, <span class="yellow">A</span> and <span class="yellow">Asia 1</span>. Antibodies obtained from <span class="yellow">rabbits</span> and guinea <span class="yellow">pigs</span> immunized with cell-culture-adapted <span class="red">virus</span> strains (<span class="yellow">O/CHA/99</span>, <span class="yellow">A/GS/LX/66</span>, <span class="yellow">Asia 1/CHN/05</span>) and suckling-<span class="yellow">mouse</span> adapted <span class="red">virus</span> strains (<span class="yellow">O/AV99(L)</span>, <span class="yellow">A/AV88(L)</span>, <span class="yellow">Asia 1/YNBS/58</span>) were used as capture antibodies. The diagnostic kit included three immunochromatographic strips of types <span class="yellow">O</span>, <span class="yellow">A</span> and <span class="yellow">Asia 1</span>, and the type-specific results were confirmed by color on the test lines of the three strips. The LFI was evaluated using epithelial and vesicular samples (n=396) prepared from current and historical field samples (provide by the National Foot-and-Mouth Disease Reference Laboratory of China at Lanzhou Veterinary Research Institute). Negative samples (n=95) were collected from healthy animals. The diagnostic sensitivity of the LFI for <span class="yellow">FMDV</span> serotypes <span class="yellow">O</span>, <span class="yellow">A</span> and <span class="yellow">Asia 1</span> was 88.3% compared to 89.7% obtained by the reference method of indirect-sandwich ELISA. The sensitivity of the LFI for <span class="yellow">FMDV type Asia 1</span> was higher at 92.1% compared to 90.5% for the ELISA. The specificity of the LFI was 97.1% compared with 97.4%. 
<h3>20951744</h3>Implementation and validation of a sensitive PCR detection method in the eradication campaign against <span class="yellow">Aleutian mink disease virus</span>. <br><br><span class="yellow">Aleutian mink disease virus (AMDV)</span> is a severe progressive disease causing multiple different clinical syndromes in <span class="yellow">mink</span>. In Denmark, the disease is notifiable and under official control. The control programme, based on serological screening, has confined successfully <span class="yellow">AMDV</span> to the northern part of Denmark. However, re-infections and new introductions of <span class="red">virus</span> into farms require a confirmatory virological test to verify the positive test results of single animals and ultimately to investigate disease transmission. <span class="yellow">A</span> one step PCR amplifying a 374-base fragment of the NS1 gene of <span class="yellow">AMDV</span> was compared to the counter-current immune electrophoresis (CIE) routinely used in the serological screening programme. <span class="yellow">Mink</span> organs (n=299) obtained from 55 recently infected farms and 8 non-infected farms from 2008 to 2010 were tested by PCR, and the results were found to have a high correlation with the serological status of the <span class="yellow">mink</span>. The relative diagnostic sensitivity of the PCR was 94.7%, and the relative diagnostic specificity was 97.9% when read in parallel with the CIE. PCR positive samples were sequenced and phylogenetic analysis revealed high similarity within the analysed <span class="yellow">AMDV strains</span> and to <span class="yellow">AMDV strains</span> described previously. 
<h3>20962076</h3>Neutralizing antibody-resistant <span class="yellow">hepatitis C virus</span> cell-to-cell transmission.<br><br>Hepatitis C <span class="red">virus</span> (<span class="yellow">HCV</span>) can initiate infection by cell-free particle and cell-cell contact-dependent transmission. In this study we use a novel infectious coculture system to examine these alternative modes of infection. Cell-to-cell transmission is relatively resistant to <span class="yellow">anti-HCV</span> glycoprotein monoclonal antibodies and polyclonal immunoglobulin isolated from infected individuals, providing an effective strategy for escaping host humoral immune responses. Chimeric viruses expressing the structural proteins representing the seven major <span class="yellow">HCV</span> genotypes demonstrate neutralizing antibody-resistant cell-to-cell transmission. <span class="yellow">HCV</span> entry is a multistep process involving numerous receptors. In this study we demonstrate that, in contrast to earlier reports, CD81 and the tight-junction components claudin-1 and occludin are all essential for both cell-free and cell-to-cell <span class="yellow">viral</span> transmission. However, scavenger receptor BI (SR-BI) has a more prominent role in cell-to-cell transmission of the <span class="red">virus</span>, with SR-BI-specific antibodies and small-molecule inhibitors showing preferential inhibition of this infection route. These observations highlight the importance of targeting host cell receptors, in particular SR-BI, to control <span class="yellow">viral</span> infection and spread in the liver. 
<h3>20962078</h3><span class="yellow">Chikungunya virus</span> induces IPS-1-dependent innate immune activation and protein kinase R-independent translational shutoff. <br><br><span class="yellow">Chikungunya virus</span> (<span class="yellow">CHIKV</span>) is an arthritogenic mosquito-transmitted alphavirus that is undergoing reemergence in areas around the Indian Ocean. Despite the current and potential danger posed by this <span class="red">virus</span>, we know surprisingly little about the induction and evasion of <span class="yellow">CHIKV</span>-associated antiviral immune responses. With this in mind we investigated innate immune reactions to <span class="yellow">CHIKV</span> in <span class="yellow">human</span> fibroblasts, a demonstrable in vivo target of <span class="red">virus</span> replication and spread. We show that <span class="yellow">CHIKV</span> infection leads to activation of the transcription factor interferon regulatory factor 3 (IRF3) and subsequent transcription of IRF3-dependent antiviral genes, including beta interferon (IFN-Beta). IRF3 activation occurs by way of a virus-induced innate immune signaling pathway that includes the adaptor molecule interferon promoter stimulator 1 (IPS-1). Despite strong transcriptional upregulation of these genes, however, translation of the corresponding proteins is not observed. We further demonstrate that translation of cellular (but not viral) genes is blocked during infection and that although <span class="yellow">CHIKV</span> is found to trigger inactivation of the translational molecule eukaryotic initiation factor subunit 2alpha by way of the double-stranded RNA sensor protein kinase R, this response is not required for the block to protein synthesis. Furthermore, overall diminution of cellular RNA synthesis is also observed in the presence of <span class="yellow">CHIKV</span> and transcription of IRF3-dependent antiviral genes appears specifically blocked late in infection. We hypothesize that the observed absence of IFN-Beta and antiviral proteins during infection results from an evasion mechanism exhibited by <span class="yellow">CHIKV</span> that is dependent on widespread shutoff of cellular protein synthesis and a targeted block to late synthesis of antiviral mRNA transcripts. 
<h3>20962084</h3>Modifications in the polymerase genes of a <span class="yellow">swine</span>-like triple-reassortant <span class="yellow">influenza virus</span> to generate live attenuated vaccines against 2009 pandemic <span class="yellow">H1N1 viruses</span>. <br><br>On 11 June 2009, the World Health Organization (WHO) declared that the outbreaks caused by novel <span class="yellow">swine-origin influenza A (H1N1) virus</span> had reached pandemic proportions. The pandemic <span class="yellow">H1N1</span> (H1N1pdm) <span class="red">virus</span> is the predominant <span class="yellow">influenza virus</span> strain in the <span class="yellow">human</span> population. It has also crossed the species barriers and infected turkeys and <span class="yellow">swine</span> in several countries. Thus, the development of a vaccine that is effective in multiple animal species is urgently needed. We have previously demonstrated that the introduction of temperature-sensitive mutations into the PB2 and PB1 genes of an avian <span class="yellow">H9N2 virus</span>, combined with the insertion of a hemagglutinin (HA) tag in PB1, resulted in an attenuated (att) vaccine backbone for both <span class="yellow">chickens</span> and <span class="yellow">mice</span>. Because the new pandemic strain is a triple-reassortant (TR) <span class="red">virus</span>, we chose to introduce the double attenuating modifications into a <span class="yellow">swine</span>-like TR <span class="red">virus</span> isolate, <span class="yellow">A/turkey/OH/313053/04 (H3N2)</span> (ty/04), with the goal of producing live attenuated <span class="yellow">influenza</span> vaccines (LAIV). This genetically modified backbone had impaired polymerase activity and restricted <span class="red">virus</span> growth at elevated temperatures. In vivo characterization of two <span class="yellow">H1N1</span> vaccine candidates generated using the ty/04 att backbone demonstrated that this vaccine is highly attenuated in <span class="yellow">mice</span>, as indicated by the absence of signs of disease, limited replication, and minimum histopathological alterations in the respiratory tract. <span class="yellow">A</span> single immunization with the ty/04 att-based vaccines conferred complete protection against a lethal <span class="yellow">H1N1pdm</span><span class="yellow">H1N1pdm virus</span> infection in <span class="yellow">mice</span>. More importantly, vaccination of <span class="yellow">pigs</span> with a ty/04 att-H1N1 vaccine candidate resulted in sterilizing immunity upon an aggressive intratracheal challenge with the 2009 <span class="yellow">H1N1</span> pandemic <span class="red">virus</span>. Our studies highlight the safety of the ty/04 att vaccine platform and its potential as a master donor strain for the generation of live attenuated vaccines for <span class="yellow">humans</span> and livestock. 
<h3>20962086</h3><span class="yellow">Poliovirus</span>-mediated disruption of cytoplasmic processing bodies.<br><br>Metazoan cells form cytoplasmic mRNA granules such as stress granules (SG) and processing bodies (P bodies) that are proposed to be sites of aggregated, translationally silenced mRNAs and mRNA degradation. <span class="yellow">Poliovirus</span> (<span class="yellow">PV</span>) is a plus-strand RNA <span class="red">virus</span> containing a genome that is a functional mRNA; thus, we investigated if <span class="yellow">PV</span> antagonizes the processes that lead to formation of these structures. We have previously shown that <span class="yellow">PV</span> infection inhibits the ability of cells to form stress granules by cleaving RasGAP-SH3-binding protein (G3BP). Here, we show that P bodies are also disrupted during <span class="yellow">PV</span> infection in cells by 4 h postinfection. The disruption of P bodies is more rapid and more complete than disruption of stress granules. The kinetics of P body disruption correlated with production of <span class="yellow">viral</span> proteinases and required substantial <span class="yellow">viral</span> gene product expression. The organizing mechanism that forms P body foci in cells is unknown; however, potential scaffolding, aggregating, or other regulatory proteins found in P bodies were investigated for degradation. Two factors involved in 5'-end mRNA decapping and degradation, Xrn1 and Dcp1a, and the 3' deadenylase complex component Pan3 underwent accelerated degradation during infection, and Dcp1a may be a direct substrate of <span class="yellow">PV</span> 3C proteinase. Several other key factors proposed to be essential for P body formation, GW182, Edc3, and Edc4, were unaffected by <span class="yellow">poliovirus</span> infection. Since deadenylation has been reported to be required for P body formation, <span class="yellow">viral</span> inhibition of deadenylation, through Pan3 degradation, is a potential mechanism of P body disruption. 
<h3>20962091</h3>Macaque long-term nonprogressors resist superinfection with multiple CD8+ T cell escape variants of <span class="yellow">simian immunodeficiency virus</span>. <br><br><span class="yellow">Human immunodeficiency virus</span> (<span class="yellow">HIV</span>)-positive individuals can be superinfected with different <span class="red">virus</span> strains. Individuals who control an initial <span class="yellow">HIV</span> infection are therefore still at risk for subsequent infection with divergent viruses, but the barriers to such superinfection remain unclear. Here we tested long-term nonprogressors' (LTNPs') susceptibility to superinfection using <span class="yellow">Indian rhesus macaques</span> that express the major histocompatibility complex <span class="yellow">class I</span> (MHC-I) allele Mamu-B 17, which is associated with control of the pathogenic AIDS <span class="yellow">virus SIVmac239</span>. The Mamu-B 17-restricted CD8(+) T cell repertoire is focused almost entirely on 5 epitopes. We engineered a series of <span class="yellow">SIVmac239</span> variants bearing mutations in 3, 4, or all 5 of these epitopes and used them to serially challenge 2 Mamu-B 17-positive LTNPs. None of the escape variants caused breakthrough replication in LTNPs, although they readily infected Mamu-B 17-negative naive macaques. In vitro competing coculture assays and examination of <span class="yellow">viral</span> evolution in hosts lacking Mamu-B 17 suggested that the mutant viruses had negligible defects in replicative fitness. Both LTNPs maintained robust immune responses, including <span class="yellow">simian immunodeficiency virus</span> (<span class="yellow">SIV</span>)-specific CD8(+) and CD4(+) T cells and neutralizing antibodies. Our results suggest that escape mutations in epitopes bound by &quot;protective&quot; MHC-I molecules may not be sufficient to establish superinfection in LTNPs. 
<h3>20962094</h3><span class="yellow">Yeast</span>-elicited cross-reactive antibodies to <span class="yellow">HIV</span> Env glycans efficiently neutralize virions expressing exclusively high-mannose N-linked glycans. <br><br>The <span class="yellow">HIV</span> envelope (Env) protein uses a dense coat of glycans to mask conserved domains and evade host humoral immune responses. The broadly neutralizing antibody 2G12, which binds a specific cluster of high-mannose glycans on <span class="yellow">HIV</span> Env, shows that the glycan shield can also serve as a target for neutralizing antibodies. We have described a triple mutant <span class="yellow">Saccharomyces cerevisiae</span> strain that expresses high-mannose glycoproteins that bind to 2G12. When used to immunize <span class="yellow">rabbits</span>, this <span class="yellow">yeast</span> elicits antibodies that bind to gp120-associated glycans but fail to neutralize <span class="red">virus</span>. Here we sought to determine the reason for these discordant results. Affinity purification of sera over columns conjugated with three 2G12-reactive <span class="yellow">yeast</span> glycoproteins showed that these proteins could adsorb 80% of the antibodies that bind to gp120 glycans. Despite binding to monomeric gp120, these mannose-specific antibodies failed to bind cell surface-expressed trimeric Env. However, when Env was expressed in the presence of the mannosidase inhibitor kifunensine to force retention of high-mannose glycans at all sites, the purified antibodies gained the abilities to bind trimeric Env and to strongly and broadly neutralize viruses produced under these conditions. Combined, these data show that the triple mutant <span class="yellow">yeast</span> strain elicits antibodies that bind to high-mannose glycans presented on the <span class="yellow">HIV</span> envelope, but only when they are displayed in a manner not found on native Env trimers. This implies that the underlying structure of the protein scaffold used to present the high-mannose glycans may be critical to allow elicitation of antibodies that recognize trimeric Env and neutralize <span class="red">virus</span>. 
<h3>20962096</h3>Identification and structural characterization of the ALIX-binding late domains of <span class="yellow">simian immunodeficiency virus</span><span class="yellow">virus SIVmac239</span> and <span class="yellow">SIVagmTan-1</span>. <br><br><span class="yellow">Retroviral</span> Gag proteins contain short late-domain motifs that recruit cellular ESCRT pathway proteins to facilitate <span class="red">virus</span> budding. ALIX-binding late domains often contain the core consensus sequence YPX(n)L (where X(n) can vary in sequence and length). However, some <span class="yellow">simian immunodeficiency virus</span> (<span class="yellow">SIV</span>) Gag proteins lack this consensus sequence, yet still bind ALIX. We mapped divergent, ALIX-binding late domains within the p6(Gag) proteins of SIV(mac239) ((40)SREKPYKEVTEDLLHLNSLF(59)) and <span class="yellow">SIV(agmTan-1)</span> ((24)AAGAYDPARKLLEQYAKK(41)). Crystal structures revealed that anchoring tyrosines (in lightface) and nearby hydrophobic residues (underlined) contact the ALIX V domain, revealing how lentiviruses employ a diverse family of late-domain sequences to bind ALIX and promote <span class="red">virus</span> budding. 
<h3>20974179</h3>A combined method for rescue of modified enteroviruses by mutagenic primers, long PCR and T7 RNA polymerase-driven in vivo transcription. <br><br>The current methods for manipulation of enteroviral RNA genomes and production of modified <span class="red">virus</span> particles include stepwise subcloning procedures and in vitro transcription and RNA transfection steps that are both time-consuming and inefficient. Several enteroviral cDNA clones with 5'-terminal T7 promoter and <span class="yellow">coxsackievirus A9</span> (<span class="yellow">CAV9</span>) PCR product with the T7 promoter were transfected successfully into target cells expressing T7 RNA polymerase for the rescue of <span class="red">virus</span> particles. This demonstrated the overall feasibility of the in vivo transcription method. Furthermore, a rapid method using high-fidelity DNA polymerase, Phusion(TM), for amplification and mutagenesis of <span class="yellow">CAV9</span> cDNA was generated. <span class="yellow">A</span> long PCR method was employed together with mutagenic primers for direct introduction of a unique restriction enzyme site into the VP1-2A junction of the <span class="yellow">CAV9</span> cDNA clone during the PCR amplification process. Enhanced green fluorescent protein was subcloned to that site, and <span class="yellow">CAV9</span>-eGFP cDNA was transfected to the target cells for in vivo transcription and successful rescue of <span class="yellow">CAV9</span>-eGFP particles. The method allowed a straightforward mutagenesis and in vivo production of infectious enteroviral particles, and may be applicable routinely for rapid production of the modified picornaviruses over the use of the traditional subcloning protocols. 
<h3>20980497</h3>A <span class="yellow">vaccinia virus</span> deletion mutant reveals the presence of additional inhibitors of NF-kappaB. <br><br>The classical nuclear factor kappa B (NF-kappaB) signaling pathway is an important regulator of inflammation and innate immunity that is activated by a wide variety of stimuli, including <span class="red">virus</span> infection, tumor necrosis factor alpha (TNF-alpha), and interleukin 1Beta (IL-1Beta). Poxviruses, including <span class="yellow">vaccinia virus</span> (<span class="yellow">VV</span>) and <span class="yellow">ectromelia virus</span>, encode multiple proteins that function in immune evasion. Recently, a growing number of genes encoded by poxviruses have been shown to target and disrupt the NF-kappaB signaling pathway. To determine if additional gene products that interfere with NF-kappaB signaling existed, we used a <span class="yellow">vaccinia virus</span> deletion mutant, VV811, which is missing 55 open reading frames lacking all known inhibitors of TNF-alpha-induced NF-kappaB activation. Immunofluorescence analysis of HeLa cells treated with TNF-alpha and IL-1Beta revealed that NF-kappaB translocation to the nucleus was inhibited in VV811-infected cells. This was further confirmed through Western blotting of cytoplasmic and nuclear extracts for NF-kappaB. Additionally, VV811 infection inhibited TNF-alpha-induced IkappaBalpha degradation. In contrast to <span class="yellow">vaccinia virus strain Copenhagen</span> (<span class="yellow">VVCop</span>)-infected cells, VV811 infection resulted in the dramatic accumulation of phosphorylated IkappaBalpha. Correspondingly, coimmunoprecipitation assays demonstrated that the NF-kappaB-inhibitory IkappaBalpha-p65-p50 complex was intact in VV811-infected cells. Significantly, cells treated with 1-Beta-d-arabinofuranosylcytosine, an inhibitor of poxvirus late gene expression, demonstrated that an additional <span class="yellow">vaccinia virus</span> late gene was involved in the stabilization of IkappaBalpha. Overall, this work indicates that unidentified inhibitors of NF-kappaB exist in <span class="yellow">vaccinia virus</span>. The complex inhibition of NF-kappaB by <span class="yellow">vaccinia virus</span> illustrates the importance of NF-kappaB activation in the antiviral response. 
<h3>20980498</h3>Template recognition mechanisms by replicase proteins differ between bipartite positive-strand genomic RNAs of a plant <span class="red">virus</span>. <br><br>Recognition of RNA templates by <span class="yellow">viral</span> replicase proteins is one of the key steps in the replication process of all RNA viruses. However, the mechanisms underlying this phenomenon, including primary RNA elements that are recognized by the <span class="yellow">viral</span> replicase proteins, are not well understood. Here, we used aptamer pulldown assays with membrane fractionation and protein-RNA coimmunoprecipitation in a cell-free <span class="yellow">viral</span> translation/replication system to investigate how <span class="yellow">viral</span> replicase proteins recognize the bipartite genomic RNAs of the <span class="yellow">Red clover necrotic mosaic virus</span> (<span class="yellow">RCNMV</span>). <span class="yellow">RCNMV</span> replicase proteins bound specifically to a Y-shaped RNA element (YRE) located in the 3' untranslated region (UTR) of RNA2, which also interacted with the 480-kDa replicase complexes that contain <span class="yellow">viral</span> and host proteins. The replicase-YRE interaction recruited RNA2 to the membrane fraction. Conversely, RNA1 fragments failed to interact with the replicase proteins supplied in trans. The results of protein-RNA coimmunoprecipitation assays suggest that RNA1 interacts with the replicase proteins coupled with their translation. Thus, the initial template recognition mechanisms employed by the replicase differ between <span class="yellow">RCNMV</span> bipartite genomic RNAs and RNA elements are primary determinants of the differential replication mechanism. 
<h3>20980515</h3>Reverse genetics generation of chimeric infectious Junin/Lassa <span class="red">virus</span> is dependent on interaction of homologous glycoprotein stable signal peptide and G2 cytoplasmic domains. <br><br>The Arenaviridae are a diverse and globally distributed collection of viruses that are maintained primarily by rodent reservoirs. <span class="yellow">Junin virus</span> (<span class="yellow">JUNV</span>) and <span class="yellow">Lassa virus</span> (<span class="yellow">LASV</span>) can both cause significant outbreaks of severe and often fatal <span class="yellow">human</span> disease throughout their respective areas of endemicity. In an effort to improve upon the existing live attenuated <span class="yellow">JUNV</span> Candid1 vaccine, we generated a genetically homogenous stock of this <span class="red">virus</span> from cDNA copies of the <span class="red">virus</span> S and L segments by using a reverse genetics system. Further, these cDNAs were used in combination with <span class="yellow">LASV</span> cDNAs to successfully generate two recombinant Candid1 <span class="yellow">JUNV</span>/<span class="yellow">LASV</span> chimeric viruses (via envelope glycoprotein [GPC] exchange). It was found that while the GPC extravirion domains were readily exchangeable, homologous stable signal peptide (SSP) and G2 transmembrane and cytoplasmic tail domains were essential for correct GPC maturation and production of infectious chimeric viruses. The switching of the <span class="yellow">JUNV</span> and <span class="yellow">LASV</span> G1/G2 ectodomains within the Candid1 vaccine background did not alter the attenuated phenotype of the vaccine strain in a lethal <span class="yellow">mouse</span> model. These recombinant chimeric viruses shed light on the fundamental requirements of arenavirus GPC maturation and may serve as a strategy for the development of bivalent <span class="yellow">JUNV</span> and <span class="yellow">LASV</span> vaccine candidates. 
<h3>20980516</h3>B7-mediated costimulation of CD4 T cells constrains cytomegalovirus persistence. <br><br>Cytomegalovirus (CMV) utilizes multiple strategies to modulate immunity and promote lifelong, persistent/latent infection, including suppressing T cell activation pathways. Here we examined the role of B7 costimulatory ligands in establishing immune detente from both the host and <span class="red">virus</span> perspectives. <span class="yellow">Mice</span> lacking both B7.1 and B7.2 showed reduced early expansion of CMV-specific CD4 T cells, consequently allowing for enhanced levels of persistent <span class="red">virus</span> replication. In turn, a CMV mutant lacking expression of the m138 and m147.5 gene products, which restrict B7.1 and B7.2 expression in infected antigen-presenting cells, induced a more robust CD4 T cell response and showed decreased persistence. Together, these data reveal a requirement for B7-mediated signaling in regulating the CMV-specific CD4 T cell response and establishing host-virus equilibrium. 
<h3>20980518</h3>Upregulation of CXCL10 in <span class="yellow">human</span> dorsal root ganglia during experimental and natural <span class="yellow">varicella-zoster virus</span> infection. <br><br>Varicella-zoster <span class="red">virus</span> (<span class="yellow">VZV</span>) reactivation causes herpes zoster, which is accompanied by an influx of lymphocytes into affected ganglia, but the stimulus for this infiltrate is not known. We report that <span class="yellow">VZV</span> infection of ganglia leads to increased CXCL10 production in vitro, in an explant ganglion model and in naturally infected dorsal root ganglia (DRG) during herpes zoster. Lymphocytes expressing the receptor for CXCL10, CXCR3, were also observed throughout naturally infected ganglia during herpes zoster, including immediately adjacent to neurons. This study identifies <span class="yellow">VZV-induced</span> CXCL10 as a potential driver of T lymphocyte recruitment into DRG during herpes zoster. 
<h3>20980521</h3>Characterization of cross-reactive CD8+ T-cell recognition of HLA-A2-restricted HIV-Gag (SLYNTVATL) and HCV-NS5b (ALYDVVSKL) epitopes in individuals infected with <span class="yellow">human immunodeficiency</span> and <span class="yellow">hepatitis C viruses</span>. <br><br>The immunologic mechanisms underlying the faster progression of <span class="yellow">hepatitis C virus</span> (<span class="yellow">HCV</span>) disease in the presence of <span class="yellow">human immunodeficiency virus</span> (<span class="yellow">HIV</span>) coinfection are not clearly understood. T-cell cross-reactivity between <span class="yellow">HCV</span> and <span class="yellow">influenza</span> virus-specific epitopes has been associated with rapid progression of <span class="yellow">HCV</span> disease (S. Urbani, B. Amadei, P. Fisicaro, M. Pilli, G. Missale, <span class="yellow">A</span>. Bertoletti, and C. Ferrari, J. Exp. Med. 201:675-680, 2005). We asked whether T-cell cross-reactivity between <span class="yellow">HCV</span> and <span class="yellow">HIV</span> could exist during <span class="yellow">HCV</span>/<span class="yellow">HIV</span> coinfection and affect pathogenesis. Our search for amino acid sequence homology between the <span class="yellow">HCV</span> and <span class="yellow">HIV</span> proteomes revealed two similar HLA-A2-restricted epitopes, HIV-Gag (SLYNTVATL [HIV-SL9]) and <span class="yellow">HCV</span>-NS5b (ALYDVVSKL [<span class="yellow">HCV</span>-AL9]). We found that 4 out of 20 HLA-A2-positive (HLA-A2(+)) <span class="yellow">HIV</span>-infected individuals had CD8(+) T cells that recognized both the <span class="yellow">HIV</span>-SL9 and HCV-AL9 epitopes. However, the AL9 epitope was generally shown to be a weak agonist. Although HCV-monoinfected individuals in our study did not show AL9-specific responses, we found that about half of HCV/<span class="yellow">HIV</span>-coinfected individuals had dual responses to both epitopes. High dual T-cell recognition among coinfected subjects was usually due to separate T-cell populations targeting each epitope, as determined by pentamer staining. The one individual demonstrating cross-reactive T cells to both epitopes showed the most advanced degree of liver disease. In coinfected individuals, we observed a positive correlation between the magnitudes of T-cell responses to both the SL9 and the AL9 epitopes, which was also positively associated with the clinical parameter of liver damage. Thus, we find that <span class="yellow">HIV</span> infection induces T cells that can cross-react to heterologous viruses or prime for T cells that are closely related in sequence. However, the induction of cross-reactive T cells may not be associated with control of disease caused by the heterologous <span class="red">virus</span>. This demonstrates that degeneracy of HIV-specific T cells may play a role in the immunopathology of <span class="yellow">HCV</span>/<span class="yellow">HIV</span> coinfection. 
<h3>20980523</h3>T cell-mediated protection against <span class="yellow">lethal 2009 pandemic H1N1 influenza virus</span> infection in a <span class="yellow">mouse</span> model. <br><br>Genetic mutation and reassortment of <span class="yellow">influenza virus</span> gene segments, in particular those of hemagglutinin (HA) and neuraminidase (NA), that lead to antigenic drift and shift are the major strategies for <span class="yellow">influenza virus</span> to escape preexisting immunity. The most recent example of such phenomena is the first pandemic of <span class="yellow">H1N1 influenza</span> of the 21st century, which started in 2009. Cross-reactive antibodies raised against <span class="yellow">H1N1 viruses</span> circulating before 1930 show protective activity against the <span class="yellow">2009 pandemic virus</span>. Cross-reactive T-cell responses can also contribute to protection, but in vivo support of this view is lacking. To explore the protection mechanisms in vivo, we primed <span class="yellow">mice</span> with <span class="yellow">H1</span> and <span class="yellow">H3 influenza virus</span> isolates and rechallenged them with a <span class="red">virus</span> derived from the <span class="yellow">2009 H1N1 A/CA/04/09 virus</span>, named CA/E3/09. We found that priming with <span class="yellow">influenza viruses</span> of both <span class="yellow">H1</span> and H3 homo- and heterosubtypes protected against lethal CA/E3/09 <span class="red">virus</span> challenge. Convalescent-phase sera from these primed <span class="yellow">mice</span> conferred no neutralization activity in vitro and no protection in vivo. However, T-cell depletion studies suggested that both CD4 and CD8 T cells contributed to the protection. Taken together, these results indicate that cross-reactive T cells established after initial priming with distally related viruses can be a vital component for prevention of disease and control of <span class="yellow">pandemic H1N1 influenza virus</span> infection. Our results highlight the importance of establishing cross-reactive T-cell responses for protecting against existing or newly emerging pandemic <span class="yellow">influenza viruses</span>. 
<h3>21029752</h3>Multiplex PCR for rapid detection of <span class="yellow">serotype A foot-and-mouth disease virus</span> variants with amino acid deletion at position 59 of the capsid protein VP3. <br><br>In India, there has been co-circulation, extinction and emergence of genotypes/lineages within <span class="yellow">serotype A</span><span class="yellow">A</span> foot-and-mouth disease (<span class="yellow">FMD</span>) <span class="red">virus</span>. At present an antigenically heterogeneous, unique lineage within genotype VII dominates the field outbreaks. This genetic cluster has amino acid deletion at position 59 of VP3 (VP3(59)-deletion group), considered to be critical antigenically. The emergence of this group warrants rapid and accurate detection to facilitate early planning and implementation of an effective control policy. <span class="yellow">A</span> rapid multiplex PCR assay was developed for detection of the dominating VP3(59)-deletion group with 100% sensitivity and specificity, even before generating sequence data and confirmatory phylogenetic analysis. This development is important for surveillance of <span class="yellow">FMD</span> in India. 
<h3>21039559</h3>Systemic virus-induced gene silencing allows functional characterization of <span class="yellow">maize</span> genes during biotrophic interaction with <span class="yellow">Ustilago maydis</span>. <br><br>Infection of <span class="yellow">maize</span> (<span class="yellow">Zea mays</span>) plants with the <span class="yellow">corn</span><span class="yellow">corn smut</span> fungus <span class="yellow">Ustilago maydis</span> leads to the formation of large tumors on the stem, leaves and inflorescences. In this biotrophic interaction, plant defense responses are actively suppressed by the pathogen, and previous transcriptome analyses of infected <span class="yellow">maize</span> plants showed massive and stage-specific changes in host gene expression during disease progression. To identify <span class="yellow">maize</span> genes that are functionally involved in the interaction with <span class="yellow">U. maydis</span>, we adapted a virus-induced gene silencing (VIGS) system based on the brome mosaic <span class="red">virus</span> (BMV) for <span class="yellow">maize</span>. Conditions were established that allowed successful <span class="yellow">U. maydis</span> infection of BMV-preinfected <span class="yellow">maize</span> plants. This set-up enabled quantification of VIGS and its impact on <span class="yellow">U. maydis</span> infection using a quantitative real-time PCR (qRT-PCR)-based readout. In proof-of-principle experiments, an <span class="yellow">U. maydis</span>-induced terpene synthase was shown to negatively regulate disease development while a protein involved in cell death inhibition was required for full virulence of <span class="yellow">U. maydis</span>. The results suggest that this system is a versatile tool for the rapid identification of <span class="yellow">maize</span> genes that determine compatibility with <span class="yellow">U. maydis</span>. 
<h3>21047954</h3>Complement opsonization enhances friend <span class="red">virus</span> infection of B cells and thereby amplifies the virus-specific CD8+ T cell response. <br><br>B cells are one of the targets of <span class="yellow">Friend virus</span> (FV) infection, a well-established <span class="yellow">mouse</span> model often used to study <span class="yellow">retroviral</span> infections in vivo. Although B cells may be effective in stimulating cytotoxic T lymphocyte responses, studies involving their role in <span class="yellow">FV</span> infection have mainly focused on neutralizing antibody production. Here we show that polyclonal activation of B cells promotes their infection with <span class="yellow">FV</span> both in vitro and in vivo. Furthermore, we demonstrate that complement opsonization of Friend <span class="yellow">murine leukemia virus (F-MuLV)</span> enhances infection of B cells, which correlates with increased potency of B cells to activate FV-specific CD8(+) T cells. 
<h3>21199018</h3>Analysis of expressed sequence tags from <span class="yellow">Maize mosaic rhabdovirus</span>-infected gut tissues of <span class="yellow">Peregrinus maidis</span> reveals the presence of key components of insect innate immunity. <br><br>The <span class="yellow">corn planthopper</span>, <span class="yellow">Peregrinus maidis</span>, causes direct feeding damage to plants and transmits <span class="yellow">Maize mosaic rhabdovirus (MMV)</span> in a persistent-propagative manner. <span class="yellow">MMV</span> must cross several insect tissue layers for successful transmission to occur, and the gut serves as an important barrier for rhabdovirus transmission. In order to facilitate the identification of proteins that may interact with <span class="yellow">MMV</span> either by facilitating acquisition or responding to <span class="red">virus</span> infection, we generated and analysed the gut transcriptome of <span class="yellow">P. maidis</span>. From two normalized cDNA libraries, we generated a <span class="yellow">P. maidis</span> gut transcriptome composed of 20,771 expressed sequence tags (ESTs). Assembly of the sequences yielded 1860 contigs and 14,032 singletons, and biological roles were assigned to 5793 (36%). Comparison of <span class="yellow">P. maidis</span> ESTs with other insect amino acid sequences revealed that <span class="yellow">P. maidis</span> shares greatest sequence similarity with another hemipteran, the <span class="yellow">brown planthopper</span> <span class="yellow">Nilaparvata lugens</span>. We identified 202 <span class="yellow">P. maidis</span> transcripts with putative homology to proteins associated with insect innate immunity, including those implicated in the Toll, Imd, JAK/STAT, Jnk and the small-interfering RNA-mediated pathways. Sequence comparisons between our <span class="yellow">P. maidis</span> gut EST collection and the currently available National Center for Biotechnology Information EST database collection for <span class="yellow">Ni. lugens</span> revealed that a pathogen recognition receptor in the Imd pathway, peptidoglycan recognition protein-long class (PGRP-LC), is present in these two members of the family Delphacidae; however, these recognition receptors are lacking in the model hemipteran <span class="yellow">Acyrthosiphon pisum</span>. In addition, we identified sequences in the <span class="yellow">P. maidis</span> gut transcriptome that share significant amino acid sequence similarities with the rhabdovirus receptor molecule, acetylcholine receptor (AChR), found in other hosts. This EST analysis sheds new light on immune response pathways in hemipteran guts that will be useful for further dissecting innate defence response pathways to rhabdovirus infection. 
<h3>21496127</h3>A role for endosomal proteins in alphavirus dissemination in mosquitoes.<br><br>Little is known about endosomal pathway proteins involved in arthropod-borne <span class="red">virus</span> (arbovirus) assembly and cell-to-cell spread in vector mosquitoes. UNC93A and synaptic vesicle-2 (SV2) proteins are involved in intracellular transport in mammals. They show amino acid sequence conservation from mosquitoes to <span class="yellow">humans</span>, and their transcripts are highly enriched in <span class="yellow">Aedes aegypti</span> during arbovirus infection. Transient gene silencing of SV2 or UNC93A in mosquitoes infected with the recombinant alphavirus <span class="yellow">Sindbis</span> MRE16-enhanced green fluorescent protein (<span class="yellow">SINV</span>; family Togaviridae) resulted in the accumulation of <span class="yellow">viral</span> positive- and negative-strand RNA, congregation of <span class="red">virus</span> envelope antigen in intracellular networks, and reduced <span class="red">virus</span> dissemination outside of the midgut. Further, UNC93A silencing, but not SV2 silencing, resulted in a 10-fold reduction in <span class="yellow">viral</span> titres at 4 days post-infection. Together, these data support a role for UNC93A and SV2 in <span class="red">virus</span> assembly or budding. Cis-regulatory elements (CREs) were identified at the 5'-ends of genes from the original data set in which SV2 and UNC93A were identified. Common CREs at the 5'-end genomic regions of a subset of enriched transcripts support the hypothesis that UNC93A transcription may be co-regulated with that of other ion transport and endosomal trafficking proteins. 
<h3>21511329</h3><span class="yellow">Hepatitis delta virus</span>.<br><br><span class="yellow">Hepatitis delta virus</span> (<span class="yellow">HDV</span>) is a small, defective RNA <span class="red">virus</span> that can infect only individuals who have <span class="yellow">hepatitis B virus</span> (<span class="yellow">HBV</span>); worldwide more than 15 million people are co-infected. There are eight reported genotypes of <span class="yellow">HDV</span> with unexplained variations in their geographical distribution and pathogenicity. The <span class="yellow">hepatitis D</span> virion is composed of a coat of <span class="yellow">HBV</span> envelope proteins surrounding the nucleocapsid, which consists of a single-stranded, circular RNA genome complexed with delta antigen, the <span class="yellow">viral</span> protein. <span class="yellow">HDV</span> is clinically important because although it suppresses <span class="yellow">HBV</span> replication, it causes severe liver disease with rapid progression to cirrhosis and hepatic decompensation. The range of clinical presentation is wide, varying from mild disease to fulminant liver failure. The prevalence of <span class="yellow">HDV</span> is declining in some endemic areas but increasing in northern and central Europe because of immigration. Treatment of <span class="yellow">HDV</span> is with pegylated interferon alfa; however, response rates are poor. Increased understanding of the molecular virology of <span class="yellow">HDV</span> will identify novel therapeutic targets for this most severe form of chronic <span class="yellow">viral</span> hepatitis. 
<h3>21840413</h3>The effects of dsRNA mycoviruses on growth and murine virulence of <span class="yellow">Aspergillus fumigatus</span>. <br><br>Some isolates of the opportunistic <span class="yellow">human</span> pathogenic fungus <span class="yellow">Aspergillus fumigatus</span> are known to be infected with mycoviruses. The dsRNA genomes of two of these mycoviruses, which include a chrysovirus and a partitivirus, have been completely sequenced and an RT-PCR assay for the viruses has been developed. Through curing virus-infected <span class="yellow">A. fumigatus</span> isolates by cycloheximide treatment and transfecting virus-free isolates with purified <span class="red">virus</span>, as checked by RT-PCR, isogenic virus-free and virus-infected lines of the fungus were generated whose phenotypes and growth have been directly compared. Mycovirus infection of <span class="yellow">A. fumigatus</span> with either the chrysovirus or the partitivirus resulted in significant aberrant phenotypic alterations and attenuation of growth of the fungus but had no effect on susceptibility to common antifungals. Chrysovirus infection of <span class="yellow">A. fumigatus</span> caused no significant alterations to murine pathogenicity. 
<h3>20951165</h3>The validation of a real-time RT-PCR assay which detects <span class="yellow">influenza A</span><span class="yellow">A</span> and types simultaneously for <span class="yellow">influenza A H1N1</span> (2009) and oseltamivir-resistant (H275Y) <span class="yellow">influenza A H1N1</span><span class="yellow">A</span> <span class="yellow">H1N1</span> (2009). <br><br>Influenza <span class="yellow">A</span> <span class="yellow">H1N1</span> (2009) was declared by the World Health Organisation (WHO) as the first <span class="yellow">influenza</span> pandemic of the 21st century. Rapid detection of <span class="yellow">influenza A</span><span class="yellow">A</span> and differentiation of <span class="yellow">influenza A H1N1</span><span class="yellow">A</span> <span class="yellow">H1N1</span> (2009) and seasonal <span class="yellow">influenza A</span><span class="yellow">A</span> is beneficial. In addition the rapid detection of antiviral resistant strains of <span class="yellow">influenza A H1N1</span><span class="yellow">A</span> <span class="yellow">H1N1</span> (2009) would be useful for clinicians to allow for change to an effective treatment at a much earlier stage if resistance is found. It was the aim of this study to develop a real-time RT-PCR that can detect all <span class="yellow">influenza A viruses</span> and type simultaneously for <span class="yellow">influenza A H1N1</span> (2009) and oseltamivir resistant (H275Y) <span class="yellow">influenza A H1N1</span> (2009). This multiplex assay will allow laboratories to screen respiratory samples for all types of <span class="yellow">influenza A</span>, <span class="yellow">influenza A H1N1</span> (2009) <span class="red">virus</span> and oseltamivir resistant (H275Y) <span class="yellow">influenza A H1N1</span> (2009) <span class="red">virus</span> in a rapid and cost effective format, ensuring that typing methods for seasonal and avian viruses are used on a smaller subset of samples. Since most virology laboratories already offer a molecular service for <span class="yellow">influenza A</span><span class="yellow">A</span> this assay could easily be implemented into most areas at little cost therefore increasing local access to resistance testing. 
<h3>20962083</h3>Small-molecule inhibition of <span class="yellow">human immunodeficiency virus type 1</span> infection by <span class="red">virus</span> capsid destabilization. <br><br><span class="yellow">Human immunodeficiency virus type 1</span> (<span class="yellow">HIV-1</span>) infection is dependent on the proper disassembly of the <span class="yellow">viral</span> capsid, or &quot;uncoating,&quot; in target cells. The <span class="yellow">HIV-1</span> capsid consists of a conical multimeric complex of the <span class="yellow">viral</span> capsid protein (CA) arranged in a hexagonal lattice. Mutations in CA that destabilize the <span class="yellow">viral</span> capsid result in impaired infection owing to defects in reverse transcription in target cells. We describe here the mechanism of action of a small molecule <span class="yellow">HIV-1</span> inhibitor, PF-3450074 (PF74), which targets CA. PF74 acts at an early stage of <span class="yellow">HIV-1</span> infection and inhibits reverse transcription in target cells. We show that PF74 binds specifically to <span class="yellow">HIV-1</span> particles, and substitutions in CA that confer resistance to the compound prevent binding. <span class="yellow">A</span> single point mutation in CA that stabilizes the <span class="yellow">HIV-1</span> core also conferred strong resistance to the <span class="red">virus</span> without inhibiting compound binding. Treatment of <span class="yellow">HIV-1</span> particles or purified cores with PF74 destabilized the <span class="yellow">viral</span> capsid in vitro. Furthermore, the compound induced the rapid dissolution of the <span class="yellow">HIV-1</span> capsid in target cells. PF74 antiviral activity was promoted by binding of the host protein cyclophilin <span class="yellow">A</span> to the <span class="yellow">HIV-1</span> capsid, and PF74 and cyclosporine exhibited mutual antagonism. Our data suggest that PF74 triggers premature <span class="yellow">HIV-1</span> uncoating in target cells, thereby mimicking the activity of the retrovirus restriction factor TRIM5alpha. This study highlights uncoating as a step in the <span class="yellow">HIV-1</span> life cycle that is susceptible to small molecule intervention. 
<h3>20962098</h3>PB2 residue 158 is a pathogenic determinant of pandemic <span class="yellow">H1N1</span> and <span class="yellow">H5 influenza a viruses</span> in <span class="yellow">mice</span>. <br><br><span class="yellow">Influenza A viruses</span> are <span class="yellow">human</span> and animal pathogens that cause morbidity and mortality, which range from mild to severe. The 2009 <span class="yellow">H1N1</span> pandemic was caused by the emergence of a reassortant <span class="yellow">H1N1</span> subtype (<span class="yellow">H1N1pdm</span>) <span class="yellow">influenza A virus</span> containing gene segments that originally circulated in <span class="yellow">human</span>, avian, and <span class="yellow">swine</span> <span class="red">virus</span> reservoirs. The molecular determinants of replication and pathogenesis of <span class="yellow">H1N1pdm viruses</span> in <span class="yellow">humans</span> and other mammals are poorly understood. Therefore, we set out to elucidate <span class="yellow">viral</span> determinants critical to the pathogenesis of this novel reassortant using a <span class="yellow">mouse</span> model. We found that a glutamate-to-glycine substitution at residue 158 of the PB2 gene (PB2-E158G) increased the morbidity and mortality of the parental <span class="yellow">H1N1pdm virus</span>. Results from mini-genome replication assays in <span class="yellow">human</span> cells and <span class="red">virus</span> titration in <span class="yellow">mouse</span> tissues demonstrated that PB2-E158G is a pathogenic determinant, because it significantly increases <span class="yellow">viral</span> replication rates. The <span class="red">virus</span> load in PB2-E158G-infected <span class="yellow">mouse</span> lungs was 1,300-fold higher than that of the wild-type <span class="red">virus</span>. Our data also show that PB2-E158G had a much stronger influence on the RNA replication and pathogenesis of <span class="yellow">H1N1pdm viruses</span> than PB2-E627K, which is a known pathogenic determinant. Remarkably, PB2-E158G substitutions also altered the pathotypes of two avian <span class="yellow">H5 viruses</span> in <span class="yellow">mice</span>, indicating that this residue impacts genetically divergent <span class="yellow">influenza A viruses</span> and suggesting that this region of PB2 could be a new antiviral target. Collectively, the data presented in this study demonstrate that PB2-E158G is a novel pathogenic determinant of <span class="yellow">influenza A viruses</span> in the <span class="yellow">mouse</span> model. We speculate that PB2-E158G may be important in the adaptation of avian PB2 genes to other mammals, and BLAST sequence analysis identified a naturally occurring <span class="yellow">human</span> <span class="yellow">H1N1pdm</span> isolate that has this substitution. Therefore, future surveillance efforts should include scrutiny of this region of PB2 because of its potential impact on pathogenesis. 
<h3>20962101</h3><span class="yellow">Hepatitis C virus</span> NS2 protein serves as a scaffold for <span class="red">virus</span> assembly by interacting with both structural and nonstructural proteins. <br><br>Many aspects of the assembly of <span class="yellow">hepatitis C virus</span> (<span class="yellow">HCV</span>) remain incompletely understood. To characterize the role of NS2 in the production of infectious <span class="red">virus</span>, we determined NS2 interaction partners among other <span class="yellow">HCV</span> proteins during productive infection. Pulldown assays showed that NS2 forms complexes with both structural and nonstructural proteins, including E1, E2, p7, NS3, and NS5A. Confocal microscopy also demonstrated that NS2 colocalizes with E1, E2, and NS5A in dot-like structures near lipid droplets. However, NS5A did not coprecipitate with E2 and interacted only weakly with NS3 in pulldown assays. Also, there was no demonstrable interaction between p7 and E2 or NS3 in such assays. Therefore, NS2 is uniquely capable of interacting with both structural and nonstructural proteins. Among mutations in p7, NS2, and NS3 that prevent production of infectious <span class="red">virus</span>, only p7 mutations significantly reduced NS2-mediated protein interactions. These p7 mutations altered the intracellular distribution of NS2 and E2 and appeared to modulate the membrane topology of the C-terminal domain of NS2. These results suggest that NS2 acts to coordinate <span class="red">virus</span> assembly by mediating interactions between envelope proteins and NS3 and NS5A within replication complexes adjacent to lipid droplets, where <span class="red">virus</span> particle assembly is thought to occur. p7 may play an accessory role by regulating NS2 membrane topology, which is important for NS2-mediated protein interactions and therefore NS2 function. 
<h3>20970458</h3>Evaluation of the Virus Counter(R) for rapid <span class="yellow">baculovirus</span> quantitation.<br><br>The utility of a new instrument for rapid <span class="red">virus</span> quantitation, the Virus Counter, was evaluated in a blind study conducted at three sites. This instrument is a substantially improved version of the original academic research instrument described previously by Stoffel and Rowlen (2005a). The addition of hydrodynamic focusing, a self-contained fluidics system and customized software for system control and data analysis has resulted in a commercially viable and available design. <span class="yellow">Baculovirus</span> samples were provided by Protein Sciences Corporation and blinded to InDevR and Baylor College of Medicine. Protein Sciences Corporation and Baylor College of Medicine analyzed the samples by plaque assay and InDevR analyzed the samples using the Virus Counter. Serial dilution of stock viruses into growth media and buffer allowed for comparison of measured versus intended concentrations. Direct log-scale comparison between pooled Virus Counter results and pooled plaque assay results indicated a linear relationship (slope=1.1+/-0.2, R(2)=0.86) with statistically significant Pearson correlation (r=0.93, p&lt;0.001). 
<h3>20974180</h3>Non-structural protein 1 (NS1) antibody-based assays to differentiate <span class="yellow">West Nile</span> (WN) <span class="red">virus</span> from <span class="yellow">Japanese encephalitis virus</span> infections in horses: effects of <span class="yellow">WN virus</span> NS1 antibodies induced by inactivated <span class="yellow">WN</span> vaccine. <br><br>Antibodies to non-structural protein 1 (NS1) of <span class="yellow">West Nile virus</span> (WNV) have been used to differentiate <span class="yellow">WNV</span> infection from infection by serologically cross-reactive flaviviruses, including <span class="yellow">Japanese encephalitis virus</span> (JEV), in <span class="yellow">horses</span>. However, since the inactivated <span class="yellow">West Nile</span> (<span class="yellow">WN</span>) vaccine has been reported to induce NS1 antibodies, there is concern about the reliability of using NS1-based assays for testing vaccinated <span class="yellow">horses</span>. Therefore, the effect of inactivated <span class="yellow">WN</span> vaccine-induced antibodies on an epitope-blocking ELISA and complement-dependent cytotoxicity (CDC) assay were investigated. Both assays are based on NS1 antibodies and were established previously to differentiate <span class="yellow">WNV</span> from <span class="yellow">JEV</span> infections in <span class="yellow">horses</span>. Groups of three <span class="yellow">horses</span> were vaccinated with two or three doses of a commercial inactivated <span class="yellow">WN</span> vaccine and NS1 antibodies were detected by a conventional ELISA after the second vaccination. Vaccine-induced NS1 antibodies were also detected by blocking ELISA and a CDC assay and affected the ability of these assays to differentiate <span class="yellow">WNV</span> from <span class="yellow">JEV</span> infections. However, the effect was less significant in the CDC assay, where use of a low serum concentration ensured effective differentiation. The more efficient detection of infection-induced antibodies over vaccine-induced antibodies by the CDC assay was potentially attributable to the different IgG isotype profiles of these antibodies. 
<h3>20980495</h3>The B cell response is redundant and highly focused on V1V2 during early subtype C infection in a Zambian seroconverter. <br><br>High-titer autologous neutralizing antibody responses have been demonstrated during early subtype C <span class="yellow">human immunodeficiency virus type 1</span> (<span class="yellow">HIV-1</span>) infection. However, characterization of this response against autologous <span class="red">virus</span> at the monoclonal antibody (MAb) level has only recently begun to be elucidated. Here we describe five monoclonal antibodies derived from a subtype C-infected seroconverter and their neutralizing activities against pseudoviruses that carry envelope glycoproteins from 48 days (0 month), 2 months, and 8 months after the estimated time of infection. Sequence analysis indicated that the MAbs arose from three distinct B cell clones, and their pattern of neutralization compared to that in patient plasma suggested that they circulated between 2 and 8 months after infection. Neutralization by MAbs representative of each B cell clone was mapped to two residues: position 134 in V1 and position 189 in V2. Mutational analysis revealed cooperative effects between glycans and residues at these two positions, arguing that they contribute to a single epitope. Analysis of the cognate gp120 sequence through homology modeling places this potential epitope near the interface between the V1 and V2 loops. Additionally, the escape mutation R189S in V2, which conferred resistance against all three MAbs, had no detrimental effect on <span class="red">virus</span> replication in vitro. Taken together, our data demonstrate that independent B cells repeatedly targeted a single structure in V1V2 during early infection. Despite this assault, a single amino acid change was sufficient to confer complete escape with minimal impact on replication fitness. 
<h3>20980501</h3>A nuclear inhibitor of NF-kappaB encoded by a poxvirus.<br><br>Poxviruses have evolved various strategies to inhibit cytoplasmic events leading to activation of the nuclear factor kappaB (NF-kappaB) signaling pathway, with individual viruses often encoding multiple NF-kappaB inhibitors. Here, the novel <span class="yellow">orf virus</span> (<span class="yellow">ORFV</span>)-encoded protein <span class="yellow">ORFV</span>002 was shown to inhibit nuclear events regulating NF-kappaB transcriptional activity. ORFV002 expression in cell cultures significantly decreased wild-type-virus-, tumor necrosis factor alpha (TNF-alpha)-, and lipopolysaccharide (LPS)-induced NF-kappaB-mediated gene expression. Expression of ORFV002 in cells, while not affecting phosphorylation or nuclear translocation of NF-kappaB-p65, markedly decreased TNF-alpha- and wild-type-virus-induced acetylation of NF-kappaB-p65, a p300-mediated nuclear modification of NF-kappaB-p65 that regulates its transactivating activity. ORFV002 was shown to colocalize and interact with NF-kappaB-p65, and expression of ORFV002 in cell cultures resulted in a reduced interaction of NF-kappaB-p65 with p300, suggesting that ORFV002 interferes with NF-kappaB-p65/p300 association. Deletion of ORFV002 from the OV-IA82 genome had no significant effect on <span class="yellow">ORFV</span> pathogenesis in <span class="yellow">sheep</span>, indicating that ORFV002 is nonessential for <span class="red">virus</span> virulence in the natural host. This represents the first description of a nuclear inhibitor of NF-kappaB encoded by a poxvirus. 
<h3>20980511</h3>Dominant inhibition of Akt/protein kinase B signaling by the matrix protein of a negative-strand RNA <span class="red">virus</span>. <br><br><span class="yellow">Vesicular stomatitis virus</span> (<span class="yellow">VSV</span>) is a rhabdovirus that alters host nuclear and cytoplasmic function upon infection. We have investigated the effect of <span class="yellow">VSV</span> infection on cellular signaling through the phosphatidylinositol-3 kinase (PI3k)/Akt signaling pathway. Akt phosphorylation at both threonine 308 (Thr308) and serine 473 (Ser473) was inhibited in cells infected with <span class="yellow">VSV</span>. This inhibition was rapid (beginning within the first 2 to 3 h postinfection) and correlated with the dephosphorylation of downstream effectors of Akt, such as glycogen synthase kinase 3Beta (GSK3Beta) and mammalian target of rapamycin (mTOR). The dephosphorylation of Akt occurred in the presence of growth factor stimulation and was not overcome through constitutive membrane targeting of Akt or high levels of phosphatidylinositol-3,4,5-triphosphate (PIP3) accumulation in the membrane. Akt dephosphorylation was not a result of alterations in PDK1 phosphorylation or activity, changes in phosphatase and tensin homologue deleted on chromosome 10 (PTEN) levels, or the downregulation of PI3k signaling. Inactivation of Akt was caused by the expression of the <span class="yellow">viral</span> M protein in the absence of other <span class="yellow">viral</span> components, and an M protein mutant that does not inhibit RNA polymerase II (Pol II) transcription and nuclear/cytoplasmic transport was also defective in inhibiting Akt phosphorylation. These data illustrate that <span class="yellow">VSV</span> utilizes a novel mechanism to alter this central player in cell signaling and oncogenesis. It also suggests an inside-out model of signal transduction where <span class="yellow">VSV</span> interruption of nuclear events has a rapid and significant effect on membrane signaling events. 
<h3>20980519</h3>Activation of plasmacytoid dendritic cells by <span class="yellow">Kaposi's sarcoma-associated herpesvirus</span>. <br><br><span class="yellow">Kaposi's sarcoma-associated herpesvirus</span> (<span class="yellow">KSHV</span>) is associated with multiple <span class="yellow">human</span> malignancies, including Kaposi's sarcoma, primary effusion lymphoma, and multicentric Castleman's disease. Following primary infection, <span class="yellow">KSHV</span> typically goes through a brief period of lytic replication prior to the establishment of latency. Plasmacytoid dendritic cells (pDCs) are the major producers of type 1 interferon (IFN), primarily in response to <span class="red">virus</span> infection. Toll-like receptors (TLRs) are key components of the innate immune system, and they serve as pathogen recognition receptors that stimulate the host antiviral response. pDCs express exclusively TLR7 and TLR9, and it is through these TLRs that the type 1 interferon response is activated in pDCs. Currently, it is not known whether <span class="yellow">KSHV</span> is recognized by pDCs and whether activation of pDCs occurs in response to <span class="yellow">KSHV</span> infection. We now report evidence that <span class="yellow">KSHV</span> can infect <span class="yellow">human</span> pDCs and that pDCs are activated upon <span class="yellow">KSHV</span> infection, as measured by upregulation of CD83 and CD86 and by IFN-alpha secretion. We further show that induction of IFN-alpha occurs through activation of TLR9 signaling and that a TLR9 inhibitor diminishes the production and secretion of IFN-alpha by <span class="yellow">KSHV</span>-infected pDCs. 
<h3>21029747</h3>A fluorescence resonance energy transfer-based fluorometer assay for screening anti-<span class="yellow">coxsackievirus B3</span> compounds. <br><br>In view of the need to develop a simple and rapid method to screen for antiviral therapeutic agents, a fluorescence resonance energy transfer (FRET)-based reporter system consisting of engineered mammalian cells expressing a cyan fluorescent protein-yellow fluorescent protein (CFP-YFP) pair linked by a short peptide containing the cleavage site of <span class="yellow">viral</span> protease 2A (2A(pro)) was developed. By detecting the 2A(pro) produced early during the <span class="red">virus</span> infection cycle, the CFP-YFP pair effectively identifies infectious <span class="yellow">coxsackievirus B3</span> (<span class="yellow">CVB3</span>), a picornavirus that causes <span class="yellow">viral</span> myocarditis in <span class="yellow">humans</span>. The reporter system was used to screen a library of 2000 drugs and natural products for potential antiviral compounds. The reporter cells were treated with the test compounds, challenged with <span class="yellow">CVB3</span>, and then examined using a fluorometer at 24h post-infection. Sixty-four compounds, mostly therapeutic drugs, antimicrobial compounds and compounds with unknown functions, caused at least 50% inhibition of 2A(pro) activity. Three known antiviral compounds, cosmosiin, ribavirin and baicalein, were also identified in the screening. The developed method is an effective strategy for rapid screening, and identifies compounds that inhibit <span class="yellow">CVB3</span> 2A(pro). This method should be a valuable aid in the antiviral drug discovery effort. 
<h3>21047957</h3>Modification of nonstructural protein 1 of <span class="yellow">influenza A virus</span> by SUMO1.<br><br>Nonstructural protein 1 (NS1) is one of the major factors resulting in the efficient infection rate and high level of virulence of <span class="yellow">influenza A virus</span>. Although consisting of only approximately 230 amino acids, NS1 has the ability to interfere with several systems of the host <span class="yellow">viral</span> defense. In the present study, we demonstrate that NS1 of the highly pathogenic avian <span class="yellow">influenza</span> <span class="yellow">A/Duck/Hubei/L-1/2004 (H5N1) virus</span> interacts with <span class="yellow">human</span> Ubc9, which is the E2 conjugating enzyme for sumoylation, and we show that SUMO1 is conjugated to <span class="yellow">H5N1</span> NS1 in both transfected and infected cells. Furthermore, two lysine residues in the C terminus of NS1 were identified as SUMO1 acceptor sites. When the SUMO1 acceptor sites were removed by mutation, NS1 underwent rapid degradation. Studies of different <span class="yellow">influenza A virus</span> strains of <span class="yellow">human</span> and avian origin showed that the majority of viruses possess an NS1 protein that is modified by SUMO1, except for the recently emerged <span class="yellow">swine-origin influenza A virus (S-OIV) (H1N1)</span>. Interestingly, growth of a sumoylation-deficient WSN <span class="red">virus</span> mutant was retarded compared to that of wild-type <span class="red">virus</span>. Together, these results indicate that sumoylation enhances NS1 stability and thus promotes rapid growth of <span class="yellow">influenza A virus</span>. 
<h3>20962077</h3><span class="yellow">Simian immunodeficiency virus</span> from the <span class="yellow">sooty mangabey</span> and <span class="yellow">rhesus macaque</span> is modified with O-linked carbohydrate. <br><br>Although stretches of serine and threonine are sometimes sites for O-linked carbohydrate attachment, specific sequence and structural determinants for O-linked attachment remain ill defined. The gp120 envelope protein of <span class="yellow">SIVmac239</span> contains a serine-threonine-rich stretch of amino acids at positions 128 to 139. Here we show that lectin protein from <span class="yellow">jackfruit</span> seed (jacalin), which binds to non- and monosialylated core 1 O-linked carbohydrate, potently inhibited the replication of <span class="yellow">SIVmac239</span>. Selection of a jacalin-resistant <span class="yellow">SIVmac239</span> variant population resulted in <span class="red">virus</span> with specific substitutions within amino acids 128 to 139. Cloned <span class="yellow">simian immunodeficiency virus</span> (<span class="yellow">SIV</span>) variants with substitutions in the 128-to-139 region had infectivities equivalent to, or within 1 log unit of, that of <span class="yellow">SIVmac239</span> and were resistant to the inhibitory effects of jacalin. Characterization of the <span class="yellow">SIVmac239</span> gp120 O-linked glycome showed the presence of core 1 and core 2 O-linked carbohydrate; a 128-to-139-substituted variant gp120 from jacalin-resistant <span class="yellow">SIV</span> lacked O-linked carbohydrate. Unlike that of <span class="yellow">SIVmac239</span>, the replication of <span class="yellow">HIV-1 strain NL4-3</span> was resistant to inhibition by jacalin. Purified gp120s from four <span class="yellow">SIVmac</span> and <span class="yellow">SIVsm</span> strains bound jacalin strongly in an enzyme-linked immunosorbent assay, while nine different <span class="yellow">HIV-1</span> gp120s, two <span class="yellow">SIVcpz</span> gp120s, and 128-to-139-substituted <span class="yellow">SIVmac239</span> gp120 did not bind jacalin. The ability or inability to bind jacalin thus correlated with the presence of the serine-threonine-rich stretch in the <span class="yellow">SIVmac</span> and <span class="yellow">SIVsm</span> gp120s and the absence of such stretches in the <span class="yellow">SIVcpz</span> and <span class="yellow">HIV-1</span> gp120s. Consistent with sequence predictions, two <span class="yellow">HIV-2</span> gp120s bound jacalin, while one did not. These data demonstrate the presence of non- and monosialylated core 1 O-linked carbohydrate on the gp120s of <span class="yellow">SIVmac</span> and <span class="yellow">SIVsm</span> and the lack of these modifications on <span class="yellow">HIV-1</span> and <span class="yellow">SIVcpz</span> gp120s. 
<h3>20962081</h3><span class="yellow">Varicella-zoster virus</span> glycoprotein E is a critical determinant of virulence in the SCID mouse-<span class="yellow">human</span> model of neuropathogenesis. <br><br><span class="yellow">Varicella-zoster virus</span> (<span class="yellow">VZV</span>) is a neurotropic alphaherpesvirus. <span class="yellow">VZV</span> infection of <span class="yellow">human</span> dorsal root ganglion (DRG) xenografts in immunodeficient <span class="yellow">mice</span> models the infection of sensory ganglia. We examined DRG infection with recombinant <span class="yellow">VZV</span> (recombinant Oka [rOka]) and the following gE mutants: gEDelta27-90, gEDeltaCys, gE-AYRV, and gE-SSTT. gEDelta27-90, which lacks the gE domain that interacts with a putative receptor insulin-degrading enzyme (IDE), replicated as extensively as rOka, producing infectious virions and significant cytopathic effects within 14 days of inoculation. Since neural cells express IDE, the gE/IDE interaction was dispensable for <span class="yellow">VZV</span> neurotropism. In contrast, gEDeltaCys, which lacks gE/gI heterodimer formation, was significantly impaired at early times postinfection; <span class="yellow">viral</span> genome copy numbers increased slowly, and infectious <span class="red">virus</span> production was not detected until day 28. Delayed replication was associated with impaired cell-cell spread in ganglia, similar to the phenotype of a gI deletion mutant (rOkaDeltagI). However, at later time points, infection of satellite cells and other supportive nonneuronal cells resulted in extensive DRG tissue damage and cell loss such that cytopathic changes observed at day 70 were more severe than those for rOka-infected DRG. The replication of gE-AYRV, which is impaired for trans-Golgi network (TGN) localization, and the replication of gE-SSTT, which contains mutations in an acidic cluster, were equivalent to that of rOka, causing significant cytopathic effects and infectious <span class="red">virus</span> production by day 14; genome copy numbers were equivalent to those of rOka. These experiments suggest that the gE interaction with cellular IDE, gE targeting to TGN sites of virion envelopment, and phosphorylation at SSTT are dispensable for <span class="yellow">VZV</span> DRG infection, whereas the gE/gI interaction is critical for <span class="yellow">VZV</span> neurovirulence. 
<h3>20962089</h3>Antiviral effects of a transgenic RNA-dependent RNA polymerase.<br><br>Transgenic expression of the RNA-dependent RNA polymerase 3D(pol) inhibited infection of <span class="yellow">Theiler's murine encephalitis virus</span> (<span class="yellow">TMEV</span>), a picornavirus from which it was derived. Here, we infected 3D(pol) transgenic <span class="yellow">mice</span> with another picornavirus, as well as an alphaherpesvirus and a rhabdovirus. 3D(pol) transgenic FVB <span class="yellow">mice</span> had significantly lower <span class="yellow">viral</span> loads and survived longer after infection with all three types of viruses than nontransgenic FVB <span class="yellow">mice</span>. Viral inhibition among three different types of <span class="red">virus</span> by transgenic 3D(pol) suggests that the mechanism of action is not the direct interference with picornaviral 3D(pol) but instead may be the changing of host cells to an antiviral state before or after <span class="yellow">viral</span> infection occurs, as basal interferon levels were higher in 3D(pol) transgenic <span class="yellow">mice</span> before infection. Further study of this mechanism may open new possibilities for future antiviral therapy. 
<h3>20980505</h3>Translational control of the abundance of cytoplasmic poly(A) binding protein in <span class="yellow">human cytomegalovirus</span>-infected cells. <br><br>Irrespective of their effects on ongoing host protein synthesis, productive replication of the representative alphaherpesvirus <span class="yellow">herpes simplex virus type 1</span>, the representative gammaherpesvirus <span class="yellow">Kaposi's sarcoma herpesvirus</span>, and the representative betaherpesvirus <span class="yellow">human cytomegalovirus</span> [<span class="yellow">HCMV</span>] stimulates the assembly of the multisubunit, cap-binding translation factor eIF4F. However, only <span class="yellow">HCMV</span> replication is associated with an increased abundance of eIF4F core components (eIF4E, eIF4G, eIF4A) and the eIF4F-associated factor poly(A) binding protein (PABP). Here, we demonstrate that the increase in translation factor concentration was readily detected in an asynchronous population of <span class="yellow">HCMV</span>-infected primary <span class="yellow">human</span> fibroblasts, abolished by prior UV inactivation of <span class="red">virus</span>, and genetically dependent upon <span class="yellow">viral</span> immediate-early genes. Strikingly, while increased mRNA steady-state levels accompanied the rise in eIF4E and eIF4G protein levels, the overall abundance of PABP mRNA, together with the half-life of the polypeptide it encodes, remained relatively unchanged by <span class="yellow">HCMV</span> infection. Instead, <span class="yellow">HCMV</span>-induced PABP accumulation resulted from new protein synthesis and was sensitive to the mTORC1-selective inhibitor rapamycin, which interferes with phosphorylation of the mTORC1 substrate p70 S6K and the translational repressor 4E-BP1. While virus-induced PABP accumulation did not require p70 S6K, it was inhibited by the expression of a dominant-acting 4E-BP1 variant unable to be inactivated by mTORC1. Finally, unlike the situation in alpha- or gammaherpesvirus-infected cells, where PABP is redistributed to nuclei, PABP accumulated in the cytoplasm of <span class="yellow">HCMV</span>-infected cells. Thus, cytoplasmic PABP accumulation is translationally controlled in <span class="yellow">HCMV</span>-infected cells via a mechanism requiring mTORC1-mediated inhibition of the cellular 4E-BP1 translational repressor. 
<h3>20980506</h3>A new model of <span class="yellow">Epstein-Barr virus</span> infection reveals an important role for earlylytic <span class="yellow">viral</span> protein expression in the development of lymphomas.<br><br><span class="yellow">Epstein-Barr virus</span> (<span class="yellow">EBV</span>) infects cells in latent or lytic forms, but the roleof lytic infection in <span class="yellow">EBV</span>-induced lymphomas is unclear. Here, we have used anew humanized <span class="yellow">mouse</span> model, in which both <span class="yellow">human</span> fetal CD34(+) hematopoietic stemcells and thymus/liver tissue are transplanted, to compare <span class="yellow">EBV</span> pathogenesis andlymphoma formation following infection with a lytic replication-defectiveBZLF1-deleted (Z-KO) <span class="red">virus</span> or a lytically active BZLF1(+) control. Both thecontrol and Z-KO viruses established long-term <span class="yellow">viral</span> latency in all infectedanimals. The infection appeared well controlled in some animals, but otherseventually developed CD20(+) diffuse large B cell lymphomas (DLBCL). Animalsinfected with the control <span class="red">virus</span> developed tumors more frequently than Z-KOvirus-infected animals. Specific immune responses against <span class="yellow">EBV</span>-infected B cellswere generated in <span class="yellow">mice</span> infected with either the control <span class="red">virus</span> or the Z-KOvirus. In both cases, forms of <span class="yellow">viral</span> latency (type I and type IIB) wereobserved that are less immunogenic than the highly transforming form (type III)commonly found in tumors of immunocompromised hosts, suggesting that immunepressure contributed to the outcome of the infection. These results point to animportant role for lytic <span class="yellow">EBV</span> infection in the development of B cell lymphomasin the context of an active host immune response.
<h3>20980513</h3>R5 <span class="yellow">HIV</span> env and <span class="yellow">vesicular stomatitis virus</span> G protein cooperate to mediate fusion to naive CD4+ T Cells. <br><br>Naive CD4(4) T cells are resistant to both <span class="yellow">HIV</span> R5 env and <span class="yellow">vesicular stomatitis virus</span> G protein (<span class="yellow">VSV</span>-G)-mediated fusion. However, <span class="yellow">viral</span> particles carrying both <span class="yellow">HIV</span> R5 env and <span class="yellow">VSV</span>-G infect naive cells by an unexplained mechanism. We show that <span class="yellow">VSV</span>-G-pseudotyped <span class="blue">virus</span> cannot fuse to unstimulated cells because the <span class="yellow">viral</span> particles cannot be endocytosed. However, virions carrying both <span class="yellow">HIV</span> R5 env and <span class="yellow">VSV</span>-G can fuse because CD4 binding allows <span class="yellow">viral</span> uptake. Our findings reveal a unique mechanism by which R5 <span class="yellow">HIV</span> env and <span class="yellow">VSV</span>-G cooperate to allow entry to naive CD4(+) T cells, providing a tool to target naive CD4(+) T cells with R5 <span class="yellow">HIV</span> to study <span class="yellow">HIV</span> coreceptor signaling and latency. 
<h3>20980517</h3>A recombinant <span class="yellow">measles virus</span> unable to antagonize STAT1 function cannot control inflammation and is attenuated in <span class="yellow">rhesus monkeys</span>. <br><br>Measles remains a leading cause of death worldwide among children because it suppresses immune function. The <span class="yellow">measles virus</span> (<span class="yellow">MV</span>) P gene encodes three proteins (P, V, and C) that interfere with innate immunity, controlling STAT1, STAT2, mda5, and perhaps other key regulators of immune function. We identified here three residues in the shared domain of the P and V proteins-tyrosine 110, valine 112, and histidine 115-that function to retain STAT1 in the cytoplasm and inhibit interferon transcription. This information was used to generate a recombinant <span class="yellow">measles virus</span> unable to antagonize STAT1 function (STAT1-blind <span class="yellow">MV</span>) differing only in these three residues from a wild-type strain of well-defined virulence. This <span class="red">virus</span> was used to assess the relevance of P and V interactions with STAT1 for virulence in primates. When a group of six <span class="yellow">rhesus monkeys</span> (<span class="yellow">Macaca mulatta</span>) was inoculated intranasally with STAT1-blind <span class="yellow">MV</span>, viremia was short-lived, and the skin rash and other clinical signs observed with wild-type <span class="yellow">MV</span> were absent. The STAT1-blind <span class="red">virus</span> less efficiently controlled the inflammatory response, as measured by enhanced transcription of interleukin-6 and tumor necrosis factor alpha in peripheral blood mononuclear cells from infected hosts. Importantly, neutralizing antibody titers and <span class="yellow">MV</span>-specific T-cell responses were equivalent in hosts infected with either <span class="red">virus</span>. These findings indicate that efficient <span class="yellow">MV</span> interactions with STAT1 are required to sustain virulence in a natural host by controlling the inflammatory response against the <span class="red">virus</span>. They also suggest that selectively STAT1-blind <span class="yellow">MV</span> may have utility as vectors for targeted oncolysis and vaccination. 
<h3>20980522</h3>RNA aptamers directed to <span class="yellow">human immunodeficiency virus type 1</span> Gag polyprotein bind to the matrix and nucleocapsid domains and inhibit <span class="red">virus</span> production. <br><br>Gag orchestrates the assembly and release of <span class="yellow">human immunodeficiency virus type 1</span> (<span class="yellow">HIV-1</span>) particles. We explored here the potential of anti-Gag RNA aptamers to inhibit <span class="yellow">HIV-1</span> replication. In vitro, RNA aptamers raised against an <span class="yellow">HIV-1</span> Gag protein, lacking the N-terminal myristate and the C-terminal p6 (DP6-Gag), could bind to matrix protein (MA), nucleocapsid protein (NC), or entire DP6-Gag protein. Upon cotransfection with pNL4-3.Luc molecular clone into 293T cells, six of the aptamers caused mild inhibition (2- to 3-fold) in the extracellular capsid levels, and one aptamer displayed 20-fold inhibition. The reduction was not due to a release defect but reflected Gag mRNA levels. We hypothesized that the aptamers influence genomic RNA levels via perturbation of specific Gag-genomic RNA interactions. Binding studies revealed that the &quot;NC-binders&quot; specifically compete with the packaging signal (psi) of <span class="yellow">HIV-1</span> for binding to DP6-Gag. Therefore, we tested the ability of two NC-binders to inhibit viruses containing psi-region deletions (DeltaSL1 or DeltaSL3) and found that the NC-binders were no longer able to inhibit Gag synthesis. The inability of these aptamers to inhibit psi-deleted viruses correlated with the absence of competition with the corresponding psi transcripts lacking SL1 or SL3 for binding DP6-Gag in vitro. These results indicate that the NC-binding aptamers disrupt Gag-genomic RNA interaction and negatively affect genomic RNA transcription, processing, or stability. Our results reveal an essential interaction between <span class="yellow">HIV-1</span> Gag and the psi-region that may be distinct from that which occurs during the encapsidation of genomic RNA. Thus, anti-Gag aptamers can be an effective tool to perturb Gag-genomic RNA interactions. 
</body></html>