
<!doctype html><html lang="en"> <head><meta charset="utf-8"> 
<title>s800 inconsistencies</title> 
<style>

.gray { background-color:rgba(125, 125, 125, 0.5); } 
.orange { background-color:rgba(210, 180, 180, 0.5); } 

.match { text-decoration: underline; 
/*
  -webkit-text-decoration-color: red;  
  text-decoration-color: red; */
    -webkit-text-decoration-line: overline underline; /* Safari */
   text-decoration-line: overline underline; 
   color:yellow;
/*   font-weight:bold; */
  
  
  } 
.LINNAEUS_only { background-color: rgba(125, 125, 125, 0.5); }
.annotated { background-color:rgba(50, 180, 180, 0.5); } 
.unannotated { background-color:rgba(214, 75, 79, 0.5); } 

.match_species { text-decoration: underline; } 
/*
ul#menu li {
/*  display:inline; 
}
.horizontal { display: inline; border-left: 3px solid darkorange; padding-left: 0.3em; }
.first { border-left: none; padding-left: 0; }
*/
ul#menu li {
list-style-type: circle;
}
table { border-collapse: collapse; } 
th, td { border: 1px solid orange; padding: 4px; } 
th {  cursor: pointer; } 
th:hover { background: yellow; }

html, body, iframe { height: 100%; }

* {
  box-sizing: border-box;
}

.row {
  display: flex;
}


/* Create two equal columns that sits next to each other */
.column {
  flex: 50%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column1 {
  flex: 30%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
/* Create two equal columns that sits next to each other */
.column2 {
  flex: 70%;
  padding: 0px;
 /*height: 300px; /* Should be removed. Only for demonstration */
}
 
</style></head><body>
<h2>20980521</h2><span class="unannotated">not annotated</span> - <span class="annotated">annotated</span> - <span class="LINNAEUS_only">LINNAEUS only</span><br><hr>Characterization of cross-reactive CD8+ T-cell recognition of HLA-A2-restricted <span class="annotated ">HIV</span>-Gag (SLYNTVATL) and <span class="annotated ">HCV</span>-NS5b (ALYDVVSKL) epitopes in individuals infected with <span class="annotated ">human immunodeficiency</span> and <span class="annotated ">hepatitis C viruses</span>. <br><br>The immunologic mechanisms underlying the faster progression of <span class="annotated ">hepatitis C virus</span> (<span class="annotated ">HCV</span>) disease in the presence of <span class="annotated ">human immunodeficiency virus</span> (<span class="annotated ">HIV</span>) coinfection are not clearly understood. T-cell cross-reactivity between <span class="annotated ">HCV</span> and <span class="annotated ">influenza virus</span>-specific epitopes has been associated with rapid progression of <span class="annotated ">HCV</span> disease (S. Urbani, B. Amadei, P. Fisicaro, M. Pilli, G. Missale, <span class="unannotated ">A</span>. Bertoletti, and C. Ferrari, J. Exp. Med. 201:675-680, 2005). We asked whether T-cell cross-reactivity between <span class="annotated ">HCV</span> and <span class="annotated ">HIV</span> could exist during <span class="annotated ">HCV</span>/<span class="annotated ">HIV</span> coinfection and affect pathogenesis. Our search for amino acid sequence homology between the <span class="annotated ">HCV</span> and <span class="annotated ">HIV</span> proteomes revealed two similar HLA-A2-restricted epitopes, <span class="annotated ">HIV</span>-Gag (SLYNTVATL [HIV-SL9]) and <span class="annotated ">HCV</span>-NS5b (ALYDVVSKL [<span class="annotated ">HCV</span>-AL9]). We found that 4 out of 20 HLA-A2-positive (HLA-A2(+)) <span class="annotated ">HIV</span>-infected individuals had CD8(+) T cells that recognized both the <span class="annotated ">HIV</span>-SL9 and <span class="annotated ">HCV</span>-AL9 epitopes. However, the AL9 epitope was generally shown to be a weak agonist. Although <span class="annotated ">HCV</span>-monoinfected individuals in our study did not show AL9-specific responses, we found that about half of <span class="annotated ">HCV</span>/<span class="annotated ">HIV</span>-coinfected individuals had dual responses to both epitopes. High dual T-cell recognition among coinfected subjects was usually due to separate T-cell populations targeting each epitope, as determined by pentamer staining. The one individual demonstrating cross-reactive T cells to both epitopes showed the most advanced degree of liver disease. In coinfected individuals, we observed a positive correlation between the magnitudes of T-cell responses to both the SL9 and the AL9 epitopes, which was also positively associated with the clinical parameter of liver damage. Thus, we find that <span class="annotated ">HIV</span> infection induces T cells that can cross-react to heterologous <span class="unannotated ">viruses</span> or prime for T cells that are closely related in sequence. However, the induction of cross-reactive T cells may not be associated with control of disease caused by the heterologous virus. This demonstrates that degeneracy of <span class="annotated ">HIV</span>-specific T cells may play a role in the immunopathology of <span class="annotated ">HCV</span>/<span class="annotated ">HIV</span> coinfection. 
<br><br><br><hr><h3>Ann file</h3><pre>T1	Species 163 185;202 209	human immunodeficiency viruses<br><br>N1	Reference T1 Taxonomy:12721<br><br>T2	Species 190 209	hepatitis C viruses<br><br>N2	Reference T2 Taxonomy:11103<br><br>T3	Species 296 299	HCV<br><br>N3	Reference T3 Taxonomy:11103<br><br>T4	Species 328 356	human immunodeficiency virus<br><br>N4	Reference T4 Taxonomy:12721<br><br>T5	Species 358 361	HIV<br><br>N5	Reference T5 Taxonomy:12721<br><br>T6	Species 435 438	HCV<br><br>N6	Reference T6 Taxonomy:11103<br><br>T7	Species 523 526	HCV<br><br>N7	Reference T7 Taxonomy:11103<br><br>T8	Species 714 717	HIV<br><br>N8	Reference T8 Taxonomy:12721<br><br>T9	Species 737 740	HCV<br><br>N9	Reference T9 Taxonomy:11103<br><br>T10	Species 741 744	HIV<br><br>T11	Species 846 849	HIV<br><br>N11	Reference T11 Taxonomy:12721<br><br>T12	Species 943 946	HCV<br><br>N12	Reference T12 Taxonomy:11103<br><br>T13	Species 964 967	HCV<br><br>N13	Reference T13 Taxonomy:11103<br><br>T14	Species 1029 1032	HIV<br><br>N14	Reference T14 Taxonomy:12721<br><br>T15	Species 1098 1101	HIV<br><br>N15	Reference T15 Taxonomy:12721<br><br>T16	Species 1315 1318	HIV<br><br>N16	Reference T16 Taxonomy:12721<br><br>T17	Species 2271 2274	HCV<br><br>N17	Reference T17 Taxonomy:11103<br><br>T18	Species 2275 2278	HIV<br><br>N18	Reference T18 Taxonomy:12721<br><br>T19	Species 277 294	hepatitis C virus<br><br>N19	Reference T19 Taxonomy:11103	<br><br>T20	Species 104 107	HCV<br><br>N20	Reference T20 Taxonomy:11103	<br><br>N10	Reference T10 Taxonomy:12721	<br><br>T21	Species 80 83	HIV<br><br>N21	Reference T21 Taxonomy:12721	<br><br>T22	Species 1204 1207	HCV<br><br>N22	Reference T22 Taxonomy:11103	<br><br>T23	Species 443 458	influenza virus<br><br>N23	Reference T23 Taxonomy:11320	Influenza A virus<br><br>T24	Species 706 709	HCV<br><br>N24	Reference T24 Taxonomy:11103	<br><br>T25	Species 838 841	HCV<br><br>N25	Reference T25 Taxonomy:11103	<br><br>T26	Species 1311 1314	HCV<br><br>N26	Reference T26 Taxonomy:11103	<br><br>T27	Species 1908 1911	HIV<br><br>N27	Reference T27 Taxonomy:12721	<br><br>T28	Species 2208 2211	HIV<br><br>N28	Reference T28 Taxonomy:12721	<br><br>T29	Species 1110 1113	HCV<br><br>N29	Reference T29 Taxonomy:11103	<br><br>T30	Species 909 912	HIV<br><br>N30	Reference T30 Taxonomy:12721	<br><br></pre></body></html>